<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/1/1" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/1/1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/1/1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/1/1.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/1/1/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/1/1.atom" />
<meta name="HW.pisa" content="ggg;6/1/1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction Accounting for Residual Heteroskedasticity" />
<meta name="DC.Identifier" content="10.1534/g3.115.022897" />
<meta name="DC.Date" content="2016-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Ou et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Whole-genome prediction (WGP) models that use single-nucleotide polymorphism marker information to predict genetic merit of animals and plants typically assume homogeneous residual variance. However, variability is often heterogeneous across agricultural production systems and may subsequently bias WGP-based inferences. This study extends classical WGP models based on normality, heavy-tailed specifications and variable selection to explicitly account for environmentally-driven residual heteroskedasticity under a hierarchical Bayesian mixed-models framework. WGP models assuming homogeneous or heterogeneous residual variances were fitted to training data generated under simulation scenarios reflecting a gradient of increasing heteroskedasticity. Model fit was based on pseudo-Bayes factors and also on prediction accuracy of genomic breeding values computed on a validation data subset one generation removed from the simulated training dataset. Homogeneous vs. heterogeneous residual variance WGP models were also fitted to two quantitative traits, namely 45-min postmortem carcass temperature and loin muscle pH, recorded in a swine resource population dataset prescreened for high and mild residual heteroskedasticity, respectively. Fit of competing WGP models was compared using pseudo-Bayes factors. Predictive ability, defined as the correlation between predicted and observed phenotypes in validation sets of a five-fold cross-validation was also computed. Heteroskedastic error WGP models showed improved model fit and enhanced prediction accuracy compared to homoskedastic error WGP models although the magnitude of the improvement was small (less than two percentage points net gain in prediction accuracy). Nevertheless, accounting for residual heteroskedasticity did improve accuracy of selection, especially on individuals of extreme genetic merit." />
<meta name="DC.Contributor" content="Zhining Ou" />
<meta name="DC.Contributor" content="Robert J. Tempelman" />
<meta name="DC.Contributor" content="Juan P. Steibel" />
<meta name="DC.Contributor" content="Catherine W. Ernst" />
<meta name="DC.Contributor" content="Ronald O. Bates" />
<meta name="DC.Contributor" content="Nora M. Bello" />
<meta name="article:published_time" content="2016-01-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Prediction Accounting for Residual Heteroskedasticity" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Whole-genome prediction (WGP) models that use single-nucleotide polymorphism marker information to predict genetic merit of animals and plants typically assume homogeneous residual variance. However, variability is often heterogeneous across agricultural production systems and may subsequently bias WGP-based inferences. This study extends classical WGP models based on normality, heavy-tailed specifications and variable selection to explicitly account for environmentally-driven residual heteroskedasticity under a hierarchical Bayesian mixed-models framework. WGP models assuming homogeneous or heterogeneous residual variances were fitted to training data generated under simulation scenarios reflecting a gradient of increasing heteroskedasticity. Model fit was based on pseudo-Bayes factors and also on prediction accuracy of genomic breeding values computed on a validation data subset one generation removed from the simulated training dataset. Homogeneous &lt;i&gt;vs.&lt;/i&gt; heterogeneous residual variance WGP models were also fitted to two quantitative traits, namely 45-min postmortem carcass temperature and loin muscle pH, recorded in a swine resource population dataset prescreened for high and mild residual heteroskedasticity, respectively. Fit of competing WGP models was compared using pseudo-Bayes factors. Predictive ability, defined as the correlation between predicted and observed phenotypes in validation sets of a five-fold cross-validation was also computed. Heteroskedastic error WGP models showed improved model fit and enhanced prediction accuracy compared to homoskedastic error WGP models although the magnitude of the improvement was small (less than two percentage points net gain in prediction accuracy). Nevertheless, accounting for residual heteroskedasticity did improve accuracy of selection, especially on individuals of extreme genetic merit.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="ggg;6/1/1" />
<meta name="citation_id" content="6/1/1" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/1/1" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/1/1.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/1/1.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/1/1.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.022897" />
<meta name="citation_pmid" content="26564950" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1" />
<meta name="citation_lastpage" content="13" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhining Ou" />
<meta name="citation_author_institution" content="Department of Statistics, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author" content="Robert J. Tempelman" />
<meta name="citation_author_institution" content="Department of Animal Science, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Juan P. Steibel" />
<meta name="citation_author_institution" content="Department of Animal Science, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_institution" content="Department of Fisheries and Wildlife, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Catherine W. Ernst" />
<meta name="citation_author_institution" content="Department of Animal Science, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Ronald O. Bates" />
<meta name="citation_author_institution" content="Department of Animal Science, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Nora M. Bello" />
<meta name="citation_author_institution" content="Department of Statistics, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author_email" content="nbello@ksu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3902-1533" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=J. H. Albert;citation_title=Computational methods using a Bayesian hierarchical generalized linear model.;citation_pages=1037-1044;citation_volume=83;citation_year=1988;citation_doi=10.2307/2290133" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Y. M. Badke;citation_author=R. O. Bates;citation_author=C. W. Ernst;citation_author=C. Schwab;citation_author=J. P. Steibel;citation_title=Estimation of linkage disequilibrium in four US pig breeds.;citation_pages=24-24;citation_volume=13;citation_year=2012;citation_pmid=22252454;citation_doi=10.1186/1471-2164-13-24" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Y. Badke;citation_author=R. Bates;citation_author=C. Ernst;citation_author=J. Fix;citation_author=J. Steibel;citation_title=Accuracy of estimation of genomic breeding values in pigs using low-density genotypes and imputation.;citation_volume=4;citation_year=2014;citation_pmid=24531728;citation_doi=10.1534/g3.114.010504" />
<meta name="citation_reference" content="citation_journal_abbrev=Biom J;citation_author=N. Bello;citation_author=J. Steibel;citation_author=R. Tempelman;citation_title=Hierarchical Bayesian modeling of heterogeneous cluster- and subject-level associations between continuous and binary outcomes in dairy production.;citation_pages=230-248;citation_volume=54;citation_year=2012;citation_issue=2;citation_pmid=22522379;citation_doi=10.1002/bimj.201100055" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=P. Boddhireddy;citation_author=M. J. Kelly;citation_author=S. Northcutt;citation_author=K. C. Prayaga;citation_author=J. Rumph;citation_title=Genomic predictions in angus cattle: comparisons of sample size, response variables, and clustering methods for cross-validation.;citation_volume=92;citation_year=2014;citation_pmid=24431338;citation_doi=10.2527/jas.2013-6757" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. P. L. Calus;citation_author=T. H. E. Meuwissen;citation_author=A. P. W. de Roos;citation_author=R. F. Veerkamp;citation_title=Accuracy of Genomic Selection Using Different Methods to Define Haplotypes;citation_pages=553-561;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202394;citation_doi=10.1534/genetics.107.080838" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=F. F. Cardoso;citation_author=G. J. M. Rosa;citation_author=R. J. Tempelman;citation_title=Multiple-breed genetic inference using heavy-tailed structural models for heterogeneous residual variances;citation_pages=1766-1779;citation_volume=83;citation_year=2005;citation_issue=8;citation_pmid=16024695" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=F. F. Cardoso;citation_author=R. J. Tempelman;citation_author=G. J. M. Rosa;citation_title=Accounting for outliers and heteroskedasticity in multibreed genetic evaluations of postweaning gain of Nelore-Hereford cattle.;citation_volume=85;citation_year=2007;citation_pmid=17178804;citation_doi=10.2527/jas.2006-668" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=N. Cernicchiaro;citation_author=D. G. Renter;citation_author=S. Xiang;citation_author=B. J. White;citation_author=N. M. Bello;citation_title=Hierarchical Bayesian modeling of heterogeneous variances in average daily weight gain of commercial feedlot cattle.;citation_volume=91;citation_year=2013;citation_pmid=23482583;citation_doi=10.2527/jas.2012-5543" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=L. Chen;citation_author=C. Li;citation_author=S. Miller;citation_author=F. Schenkel;citation_title=Multi-population genomic prediction using a multi-task Bayesian learning model.;citation_pages=53;citation_volume=15;citation_year=2014;citation_pmid=24884927" />
<meta name="citation_reference" content="citation_title=Genetics of the Pig;citation_title=Genetics of meat quality and carcass traits;citation_author=D. C. Ciobanu;citation_author=S. M. Lonergan;citation_author=E. J. Huff-Lonergan;citation_pages=355-389;citation_year=2011" />
<meta name="citation_reference" content="citation_title=Bayesian Process Monitoring, Control and Optimization;citation_author=B. M. Colosimo;citation_author=E. Del Castillo;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Daetwyler;citation_author=M. P. L. Calus;citation_author=R. Pong Wong;citation_author=G. de Los Campos;citation_author=J. Hickey;citation_title=Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.;citation_volume=193;citation_year=2013;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. de los Campos;citation_author=Y. C. Klimentidis;citation_author=A. I. Vazquez;citation_author=D. B. Allison;citation_title=Prediction of expected years of life using whole-genome markers.;citation_pages=e40964-e40964;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22848416;citation_doi=10.1371/journal.pone.0040964" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=R. Fernando;citation_author=Y. C. Klimentidis;citation_author=D. Sorensen;citation_title=Prediction of complex human traits using the genomic best linear unbiased predictor.;citation_pages=e1003608;citation_volume=9;citation_year=2013;citation_pmid=23874214;citation_doi=10.1371/journal.pgen.1003608" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. W. de Roos;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Reliability of genomic predictions across multiple populations.;citation_volume=183;citation_year=2009;citation_pmid=19822733;citation_doi=10.1534/genetics.109.104935" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=D. B. Edwards;citation_author=C. W. Ernst;citation_author=R. J. Tempelman;citation_author=G. J. M. Rosa;citation_author=N. E. Raney;citation_title=Quantitative trait loci mapping in an F2 Duroc x Pietrain resource population: I. Growth traits.;citation_volume=86;citation_year=2008;citation_pmid=17965327;citation_doi=10.2527/jas.2006-625" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=D. B. Edwards;citation_author=C. W. Ernst;citation_author=N. E. Raney;citation_author=M. E. Doumit;citation_author=M. D. Hoge;citation_title=Quantitative trait locus mapping in an F2 Duroc x Pietrain resource population: II. Carcass and meat quality traits.;citation_volume=86;citation_year=2008;citation_pmid=17965326;citation_doi=10.2527/jas.2006-626" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. W. Edwards;citation_author=J. Jannink;citation_title=Bayesian Modeling of Heterogeneous Error and Genotype x Environment Interaction Variances;citation_pages=820-833;citation_volume=46;citation_year=2006;citation_issue=2;citation_doi=10.2135/cropsci2005.0164" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=J. L. Foulley;citation_author=D. Gianola;citation_title=Statistical analysis of ordered categorical data via a structural heteroskedastic threshold model.;citation_pages=249-273;citation_volume=28;citation_year=1996;citation_doi=10.1186/1297-9686-28-3-249" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=D. J. Garrick;citation_author=L. D. VanVleck;citation_title=Aspects of Selection for Performance in Several Environments with Heterogeneous Variances;citation_pages=409-421;citation_volume=65;citation_year=1987;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_title=Markov Chain Monte Carlo in Practice;citation_title=Model determination using sampling-based methods;citation_author=A. Gelfand;citation_pages=145-161;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Bayesian Anal.;citation_author=A. Gelman;citation_title=Prior distributions for variance parameters in hierarchical models.;citation_pages=515-533;citation_volume=1;citation_year=2006;citation_doi=10.1214/06-BA117A" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=A. Gelman;citation_author=D. B. Rubin;citation_title=Inference from iterative simulation using multiple sequences.;citation_pages=457-511;citation_volume=7;citation_year=1992;citation_doi=10.1214/ss/1177011136" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: the Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Anim. Biosci.;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_title=One hundred years of statistical developments in animal breeding.;citation_pages=19-56;citation_volume=3;citation_year=2015;citation_pmid=25387231;citation_doi=10.1146/annurev-animal-022114-110733" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=D. Gianola;citation_author=J. L. Foulley;citation_author=R. L. Fernando;citation_author=C. R. Henderson;citation_author=K. A. Weigel;citation_title=Estimation of Heterogeneous Variances Using Empirical Bayes Methods: Theoretical Considerations;citation_pages=2805-2823;citation_volume=75;citation_year=1992;citation_issue=10;citation_pmid=1430485;citation_doi=10.3168/jds.S0022-0302(92)78044-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=W. Hill;citation_author=E. Manfredi;citation_author=R. Fernando;citation_title=Additive genetic variability and the Bayesian alphabet.;citation_volume=183;citation_year=2009;citation_pmid=19620397;citation_doi=10.1534/genetics.109.103952" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=M. Goddard;citation_author=N. Wray;citation_author=K. Verbyla;citation_author=P. Visscher;citation_title=Estimating effects and making predictions from genome-wide marker data.;citation_pages=517-529;citation_volume=24;citation_year=2009;citation_doi=10.1214/09-STS306" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=J. Gualdron Duarte;citation_author=R. Bates;citation_author=C. Ernst;citation_author=N. Raney;citation_author=R. J. C. Cantet;citation_title=Genotype imputation accuracy in a F2 pig population using high density and low density SNP panels.;citation_pages=38;citation_volume=14;citation_year=2013;citation_pmid=23651538" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=J. Gualdron Duarte;citation_author=R. J. C. Cantet;citation_author=R. Bates;citation_author=C. Ernst;citation_author=N. Raney;citation_title=Rapid screening for phenotype-genotype associations by linear transformations of genomic evaluations.;citation_pages=246;citation_volume=15;citation_year=2014;citation_pmid=25038782;citation_doi=10.1186/1471-2105-15-246" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. Fernando;citation_author=K. Kizilkaya;citation_author=D. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=T. Hayashi;citation_author=H. Iwata;citation_title=EM algorithm for Bayesian estimation of genomic breeding values.;citation_pages=3-3;citation_volume=11;citation_year=2010;citation_pmid=20092655" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. Hayes;citation_author=M. E. Goddard;citation_title=The distribution of the effects of genes affecting quantitative traits in livestock.;citation_pages=209-229;citation_volume=33;citation_year=2001;citation_issue=3;citation_pmid=11403745;citation_doi=10.1051/gse:2001117" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Operations Research;citation_journal_abbrev=Operations Research;citation_author=P. Heidelberger;citation_author=P. Welch;citation_title=Simulation Run Length Control in the Presence of an Initial Transient;citation_pages=1109-1144;citation_volume=31;citation_year=1983;citation_issue=6;citation_doi=10.1287/opre.31.6.1109" />
<meta name="citation_reference" content="citation_journal_title=Anim. Prod.;citation_author=W. G. Hill;citation_title=On selection among groups with heterogeneous variance.;citation_pages=473-477;citation_volume=39;citation_year=1984;citation_doi=10.1017/S0003356100032220" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=H. A. Mulder;citation_title=Genetic analysis of environmental variation.;citation_pages=381-395;citation_volume=92;citation_year=2010;citation_issue=5-6;citation_pmid=21429270;citation_doi=10.1017/S0016672310000546" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquin;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. Du Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=R. Kass;citation_author=A. Raftery;citation_title=Bayes factors.;citation_pages=773-795;citation_volume=90;citation_year=1995;citation_pmid=10783804;citation_doi=10.2307/2291091" />
<meta name="citation_reference" content="citation_journal_title=Am. Stat.;citation_author=R. Kass;citation_author=B. Carlin;citation_author=A. Gelman;citation_author=R. Neal;citation_title=Markov chain Monte Carlo in practice: a roundtable discussion.;citation_pages=93-100;citation_volume=52;citation_year=1998;citation_doi=10.2307/2685466" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=K. Kizilkaya;citation_author=R. Tempelman;citation_title=A general approach to mixed effects modeling of residual variances in generalized linear mixed models.;citation_pages=31-56;citation_volume=37;citation_year=2005;citation_issue=1;citation_pmid=15588567;citation_doi=10.1186/1297-9686-37-1-31" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. T. Kuhn;citation_author=A. E. Freeman;citation_title=Biases in Predicted Transmitting Abilities of Sires When Daughters Receive Preferential Treatment;citation_pages=2067-2072;citation_volume=78;citation_year=1995;citation_issue=9;citation_doi=10.3168/jds.S0022-0302(95)76832-1" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. Lopez Cruz;citation_author=J. Crossa;citation_author=D. Bonnett;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_title=Increased prediction accuracy in wheat breeding trials using a marker x environment interaction genomic selection model.;citation_volume=5;citation_year=2015;citation_pmid=25660166;citation_doi=10.1534/g3.114.016097" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Anim. Biosci.;citation_author=T. Meuwissen;citation_author=B. Hayes;citation_author=M. Goddard;citation_title=Accelerating improvement of livestock with genomic selection.;citation_pages=221-237;citation_volume=1;citation_year=2013;citation_pmid=25387018;citation_doi=10.1146/annurev-animal-031412-103705" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. H. E. Meuwissen;citation_author=T. Solberg;citation_author=R. Shepherd;citation_author=J. Woolliams;citation_title=A fast algorithm for BayesB type of prediction of genome-wide estimates of genetic value.;citation_pages=2-2;citation_volume=41;citation_year=2009;citation_pmid=19284681;citation_doi=10.1186/1297-9686-41-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. Ostersen;citation_author=O. F. Christensen;citation_author=M. Henryon;citation_author=B. Nielsen;citation_author=G. Su;citation_title=Deregressed EBV as the response variable yield more reliable genomic predictions than traditional EBV in pure-bred pigs.;citation_pages=38-38;citation_volume=43;citation_year=2011;citation_pmid=22070746;citation_doi=10.1186/1297-9686-43-38" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_title=On the use of variance per genotype as a tool to identify quantitative trait interaction effects: a report from the Women’s Genome Health Study.;citation_author=G. Paré;citation_author=N. R. Cook;citation_author=P. M. Ridker;citation_author=D. I. Chasman;citation_volume=6;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=M. Plummer;citation_author=N. Best;citation_author=K. Cowles;citation_author=K. Vines;citation_title=CODA: convergence diagnostics and output analysis for MCMC.;citation_pages=7-11;citation_volume=6;citation_year=2006" />
<meta name="citation_reference" content="citation_title=Bayesian Statistics 4;citation_title=How many iterations in the Gibbs sampler;citation_author=A. E. Raftery;citation_author=S. Lewis;citation_pages=763-773;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Rönnegård;citation_author=W. Valdar;citation_title=Detecting major genetic loci controlling phenotypic variability in experimental crosses.;citation_volume=188;citation_year=2011;citation_pmid=21467569;citation_doi=10.1534/genetics.111.127068" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=L. Rönnegård;citation_author=W. Valdar;citation_title=Recent developments in statistical methods for detecting genetic loci affecting phenotypic variability.;citation_pages=63-63;citation_volume=13;citation_year=2012;citation_pmid=22827487" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. SanCristobal-Gaudy;citation_author=L. Bodin;citation_author=J. M. Elsen;citation_author=C. Chevalet;citation_title=Genetic components of litter size variability in sheep.;citation_pages=249-271;citation_volume=33;citation_year=2001;citation_issue=3;citation_pmid=11403747;citation_doi=10.1051/gse:2001101" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=M. T. See;citation_title=Heterogeneity of (co)variance among herds for backfat measures of swine;citation_pages=2568-2574;citation_volume=76;citation_year=1998;citation_issue=10;citation_pmid=9814895" />
<meta name="citation_reference" content="citation_journal_title=J. Muscle Foods;citation_author=P. Sellier;citation_author=G. Monin;citation_title=Genetics of pig meat quality: a review.;citation_pages=187-219;citation_volume=5;citation_year=1994;citation_doi=10.1111/j.1745-4573.1994.tb00530.x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=I. Stranden;citation_author=D. Gianola;citation_title=Mixed effects linear models with t-distributions for quantitative genetic analysis: a Bayesian approach.;citation_pages=25-42;citation_volume=31;citation_year=1999;citation_doi=10.1051/gse:19990102" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=M. V. Struchalin;citation_author=A. Dehghan;citation_author=J. C. Witteman;citation_author=C. van Duijn;citation_author=Y. S. Aulchenko;citation_title=Variance heterogeneity analysis for detection of potentially interacting genetic loci: method and its limitations.;citation_pages=92-92;citation_volume=11;citation_year=2010;citation_pmid=20942902" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. I. Vazquez;citation_author=G. de los Campos;citation_author=Y. C. Klimentidis;citation_author=G. J. M. Rosa;citation_author=D. Gianola;citation_title=A comprehensive genetic approach for improving prediction of skin cancer risk in humans.;citation_volume=192;citation_year=2012;citation_pmid=23051645;citation_doi=10.1534/genetics.112.141705" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. E. Wigginton;citation_author=D. J. Cutler;citation_author=G. R. Abecasis;citation_title=A note on exact tests of Hardy-Weinberg equilibrium.;citation_pages=887-893;citation_volume=76;citation_year=2005;citation_issue=5;citation_pmid=15789306;citation_doi=10.1086/429864" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Yang;citation_author=R. J. Tempelman;citation_title=A Bayesian antedependence model for whole genome prediction.;citation_volume=190;citation_year=2012;citation_pmid=22135352;citation_doi=10.1534/genetics.111.131540" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=W. Z. Yang;citation_author=C. Y. Chen;citation_author=R. J. Tempelman;citation_title=Improving the computational efficiency of fully Bayes inference and assessing the effect of misspecification of hyperparameters in whole-genome prediction models.;citation_pages=13;citation_volume=47;citation_year=2015;citation_pmid=25885894;citation_doi=10.1186/s12711-015-0092-x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=Y. Yang;citation_author=O. Christensen;citation_author=D. Sorensen;citation_title=Use of genomic models to study genetic control of environmental variance.;citation_pages=125-138;citation_volume=93;citation_year=2011;citation_doi=10.1017/S0016672311000012" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction Accounting for Residual Heteroskedasticity" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="twitter:description" content="Whole-genome prediction (WGP) models that use single-nucleotide polymorphism marker information to predict genetic merit of animals and plants typically assume homogeneous residual variance. However, variability is often heterogeneous across agricultural production systems and may subsequently bias WGP-based inferences. This study extends classical WGP models based on normality, heavy-tailed specifications and variable selection to explicitly account for environmentally-driven residual heteroskedasticity under a hierarchical Bayesian mixed-models framework. WGP models assuming homogeneous or heterogeneous residual variances were fitted to training data generated under simulation scenarios reflecting a gradient of increasing heteroskedasticity. Model fit was based on pseudo-Bayes factors and also on prediction accuracy of genomic breeding values computed on a validation data subset one generation removed from the simulated training dataset. Homogeneous vs. heterogeneous residual variance WGP models were also fitted to two quantitative traits, namely 45-min postmortem carcass temperature and loin muscle pH, recorded in a swine resource population dataset prescreened for high and mild residual heteroskedasticity, respectively. Fit of competing WGP models was compared using pseudo-Bayes factors. Predictive ability, defined as the correlation between predicted and observed phenotypes in validation sets of a five-fold cross-validation was also computed. Heteroskedastic error WGP models showed improved model fit and enhanced prediction accuracy compared to homoskedastic error WGP models although the magnitude of the improvement was small (less than two percentage points net gain in prediction accuracy). Nevertheless, accounting for residual heteroskedasticity did improve accuracy of selection, especially on individuals of extreme genetic merit." />
<meta name="og-title" property="og:title" content="Genomic Prediction Accounting for Residual Heteroskedasticity" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/1/1" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Whole-genome prediction (WGP) models that use single-nucleotide polymorphism marker information to predict genetic merit of animals and plants typically assume homogeneous residual variance. However, variability is often heterogeneous across agricultural production systems and may subsequently bias WGP-based inferences. This study extends classical WGP models based on normality, heavy-tailed specifications and variable selection to explicitly account for environmentally-driven residual heteroskedasticity under a hierarchical Bayesian mixed-models framework. WGP models assuming homogeneous or heterogeneous residual variances were fitted to training data generated under simulation scenarios reflecting a gradient of increasing heteroskedasticity. Model fit was based on pseudo-Bayes factors and also on prediction accuracy of genomic breeding values computed on a validation data subset one generation removed from the simulated training dataset. Homogeneous vs. heterogeneous residual variance WGP models were also fitted to two quantitative traits, namely 45-min postmortem carcass temperature and loin muscle pH, recorded in a swine resource population dataset prescreened for high and mild residual heteroskedasticity, respectively. Fit of competing WGP models was compared using pseudo-Bayes factors. Predictive ability, defined as the correlation between predicted and observed phenotypes in validation sets of a five-fold cross-validation was also computed. Heteroskedastic error WGP models showed improved model fit and enhanced prediction accuracy compared to homoskedastic error WGP models although the magnitude of the improvement was small (less than two percentage points net gain in prediction accuracy). Nevertheless, accounting for residual heteroskedasticity did improve accuracy of selection, especially on individuals of extreme genetic merit." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
    <title>Genomic Prediction Accounting for Residual Heteroskedasticity | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RyfGZXbLBSAg7YQEyrvugoz31EOwxB6nEkHHw8SNbRw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460245","apath":"\/ggg\/6\/1\/1.atom","pisa":"ggg;6\/1\/1","ac":{"\/ggg\/6\/1\/1.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/1\/1.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/1\/1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/1\/1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/1\/1.abstract":true,"\/content\/6\/1\/1":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dUoM\\\\\\\/Iq5VIiC3uDEQ\\\\\\\/z4H63d8+rLf2TRQAjoIetUw56MWHpY4Xsb8NzZ3SeMWr5bzTmvQb+P0bwDrrDjbU4FpfLtb7cG+PxIcZiC2YWq4XBqBYHaWMsTixgll\\\\\\\/fWgA+nqcqTm2+3wyLpl0oREYcxvfK4s\\\\\\\/Lrz2xO9AN4lJAW3qtsA3x+maZhgZ8n5wFb7EkEo3XrTetSSYLKsPUDYFW0CGTEnAfpDRmu9chv45b8ZwE+POe2IR6sSz4rkZ6m7P1Elc5K61eJ39OoNNJK9BXG\\\\\\\/N22scqKCY2NDsKlo1O96lHlqKjAPTTH2pEPV769KmF79MXgcWFZ5kgHhhJfQpO7CiH6z9kp245EybvEJTT53V2CGyoMIFLsXutegUp0BjmUcN5q9cQb8YtySxi87mrOZXqEtaC6eFI1n5VgzQMp13KDRiXxbnpJogOdGo5OLTPUQLtAWQSo4Vl77fpLnSVwqlP7y8DeovC5\\\\\\\/6gUE3vn\\\\\\\/zNDTA4O74pmpz6X34XGaSCsmbqM\\\\\\\/qdgRyQ8ELdGc7eNM73St8sgZAxELm\\\\\\\/6G7CM2290m4DCjRaTlIukgD27cFje9kqy1nvQk3TFnQp18J45i7UimHvpdQUV4M\\\\\\\/94nMezW1Ww\\\\\\\/H9lu4bnkAcyWxVNnWQJPQpR4go4ndAWn4zGnfSXzlcN4w6e1iGvsibuAbToCQOk87vZ3LOkgwf10q8qm4VVQpCuyER0n5V5ITdNB2vuor33xu1ymN1yf0rGAdZCEntXPKqw2T8qLLcka5JqdUFKmLSm9SrXdVWYZ9\\\\\\\/Gfx3oBsyPu1SeUkz2XrsWMaP8yUEhJwSYhfVdmuAmondMX\\\\\\\/m8X7buGsw071nUVH90AWuQ88ZoCpkLmMkDPLvebouPk3ic7wecfissYeXl2+440tIZURhcdlHUbcPg+KCkDoEzUvtHJZpDMaFT5BSUFPRehbRyR7Oqfk0oAExyQDjmlucv9FWy2g0wt4tN2oCJqEVJqvaqQSX9\\\\\\\/\\\\\\\/nko8AjAKWBkI6pkWUYtecE8njk6Ki8VuJkOHnu7RdOGaGGxfUJCAAIYCD+ToA8ZubZ5UGp5drfdmXMclGrBAH2uz0uWhpdYLcsYjjTDZOxRtJR4o7lFKayItTFTl2NVoGlFBGDw5IhUbVgA\\\\\\\/ugqnIjHOLdpguaJ6roLXjEZJffHuWN6k1vsBgBLXPCHNM1o+2QVN+c+VKWXWBFJjsH8jDYFx+jdH1NZ2Fn800zzEGHWncYRv7vyDL09DnOl7eaAN2WhFnJXeIsRGYfNMUtO4r8XvosOeza301\\\u0022,\\\u0022iv\\\u0022:\\\u0022WooYIsgW8L9RtZT1C4Gg8A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022183b992156f934fc2d70b037f114d98b4606babeb3a4df98dbf5eb4b9a376152\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qr7RMh0RxplgfL4RN0FT3OOQ5FVCdWX4cLIknlzqrCDRapdyzUH2FC0jULgyAGC1sOJ36IPHcIWz+uXE2tS015kSZT\\\\\\\/nxDBjdFhHae2XVK4T1qJS+V9z2d\\\\\\\/sLX5KAvQUQp4TRnnN5HgZHB9\\\\\\\/fYVSpiM+BeJRdtHrX19pCmgI7kOSKBHikBKtAf+CzrqYhL6Fm00lEpXKyFpXkIu6RZ9WzSG+IJIdoakzf3eRT2rbQMQ1FP5CTfQszkZfVKOP2LkrvqyJtBAYhzvST+PYRaoo4hFf8iE\\\\\\\/ukEz3EDV3S8l6i2zQlkCK3Alv3dPG7waHdnwcdv4sr4H3GqtEIY7Lz88+rIdoQiP5HHEP4cYge4NnIBe0mfk4iMqi51xH0F8xHEpKsxBYhJHhilZglybhLemQ9R51miCelyrapaFVgXJyI6sTe5JZrw25tP4pmOIcFhcO6gaptjD+QdxvJ4\\\\\\\/1KWenlsRdhu8sWPain7Je8J4HfmGqKuwrOskd0cKCZ\\\\\\\/VGBG0TyzUXdh9vDg7X4sM2LCPZbIjIvFB4zINHyK+A9a9Ary21M4vfczHCpTvNAdUMidbnpuQvnfRpNVNWQcl84azdjXVgGYSt0bWBmpE\\\\\\\/WQIHs68nmleXHL4+o5\\\\\\\/YUqD3X4x2ly\\\\\\\/dyC7\\\\\\\/zVTsdafiYhxN3nFxLo1juQq7fH7ZZjJ8ZKhP+tznYMKkDvrVnXjVeJWd1SEZFtkDVxOWe4PEu7gdWeiTciaRjkkoep3q+ZKRsVemwNj\\\\\\\/Sir7Tgz27d8OF+NUYnSW9rMtZsgN4Rc2IVToEk\\\\\\\/VsMIQa+foxn17O8DfHMG8Z6yb4uPZMg2DQA5vGMn2XcRcatFhJqtwabxfuJkXSneqqFYOYwTX65LhI+8l011shS2FzrC4OkDeEhhi\\\\\\\/kisyp\\\\\\\/0whZKslhPmC52V8zRDFzx1BWOQruwSRLhIJ8ycahr3Rp5LuHs+k6wmheWMXbS1l0gQJDwqGGavLWdt2xVs5NGl1jBj3NwZwlinID946lExbSM8tU8+Eev9ZjrecHHWDzATxBl7Hi82GTft7JJjtM5kmpbLalLFNd3p2MLwHpbqWfhNTvPVqno6Q8sHtgSR0Md17AST8\\\\\\\/tom7L7iwCwZiY+WwkwcGpbYpCm\\\\\\\/VqPNUk\\\\\\\/1YaO3RrZeS74Te5skXUsvplaHGV2Gtn93VOv87d5bsfNiQ9ic5MQsjz41+FPjBuLz2\\\\\\\/onRYi5FEU+DkvGl4lckBHi\\\\\\\/pmbbDnDyhETfivzhcfTkpJHFeHJg\\\\\\\/ZiEmEBifgpcHfV97Wn5UJtH4F3Pc8Veq4boNMEuRG6um+Ugr42tZl+YPiVocdUpvRWBtx82lowJSJ4jlUJlF5ulq7aqO5vQKtcJI2r3Qn3ylZCArWhJedND9g4EnJorQS\\\\\\\/f+B2wEQYER8q4bwNsR6WK49lWv0n24KmSffhYGsrWXXW02+FUsmvcdnKVR96QyjYFS7wcg5uLwGSNZ97fn7vFfNly5wq8ydtF53v0fvzAYU3aqj9wHuK0bYVnH3Ru6sDZn5Xe\\\\\\\/ascEAmoCVP9Cx8d6K1N06YtMUYwd2bP0JFsfhp2IoF\\\\\\\/bgA7oQVjVCVCiaNUBLGvEyRHilaGUFvqta1hx30XXXsicL04h4UCQ35AHtc4whZiIC0sLE99FBJcLqvoyojKihK3xREYR7T19xIkWBh4GrFHt9YctspD8LCy9wobhIHFtXJBlTQWb+IZ1pQWXvhv+spZ89A2usg789QQ+HfL3TV\\\\\\\/I4WxPzObWJvJuCCzzftZuphX8LX9Mk\\\\\\\/cASoGNxn7W3UPkfLcus4qEHkddWLCcENCfPm0I6fwDkruvdM\\\\\\\/YXoPiXJw5pmRBdq9A6CqfSUkFQKGDWmCZNYijsOG1BL4fI2TGVG9dGlKu2TFUuLeV1Z15wjO5m5acYVzuy4L5KR299fyD2eYFm8qYPDLruZqm35xW7TYBNSrnL9UEXX5FaZe1LEvettbqQJw0gBTUv6KHrzQLM+xVq\\\\\\\/7fX+XFsLaGbiH6VvSYN+IlZUPDFXkq5zD8E8gkApw9io+Cu2olLSs7\\\\\\\/JdU9NpIDYOWBumopUt\\\\\\\/FfbF9wBwmffltGXuGiI8kuW87inY+I5RogENIl\\\\\\\/4vmC3Wqai9xVbOZYPPaqghrF5IWNFwmlJKs90V\\\\\\\/Vdf\\\\\\\/QNXhfYyg\\\\\\\/8r59NJdx5ayZoQ7Il8CaWvHMwMxem\\\\\\\/CbCZJI7J7GQ6sAbhUIHi8IPQwYUS1wDh80eJ3yfUexQNVT+3Y9cQkV6hJJujsER36LSknm9xVWYvjEk1W6jLXwaz8sTIBXcpre6\\\\\\\/nXjZN0J868ao2f\\\\\\\/wx1hkU9o4I5WMo\\\\\\\/zBlwiqjuIKYzdUakRqB01Wqw5u+b38Ke5Qyhrm\\\\\\\/LVYS2FtmaJHAQBfz1z72cA1R4jTjRaIyNwyaGFlnlh\\\\\\\/hD0KRIbRw29I1KUFO54RFH01DNnYv\\\\\\\/gm6D8gJuSoeykhwjpZ1fvxdWz074IJUGdmuUtBO7md3Oo4NVhj8izIYpBrZogRDGj42miA8NW4qvuAMjNMKorXqzqFh0sP17Qe00hVLyKP9LMkMSkwdRwQZ1BYkWddSw5InOEfLG3yKNgsDnPFpsPG\\\\\\\/oC0WyFBDYEgApgro3kvtHHAkHJBt6YjCIODkBFrvoa9rDYsxgUOTDOnXvQCxgA8Dmq89r0r7zXcH7pT21ujNXYxn0pOmN\\\\\\\/V4FV00MeuI7FnIj2vE2yLkA2OIoVyd9VWYUqNKE+cUmWLVEvYepiaA9CRrHuBDmUgKHGb0jFuXNGdvU5gpFnLKsXcjPj30M9wIgsY9Uc7DwStskBnUxTW0mAAw9fl1bqmUUL3ewvPcGhZpCVpqNtTj7Bg8x2fbpIfww6tTkZGarp9gBkVBfYkFVo6kfs33hLBSKkTR6CEMLxf\\\\\\\/Iqnno2I6K7vLfnQG9vGVLligYJ9a1eCe\\\\\\\/bwz1wLXxpV8jnCrGigTJmvqvCzgvShPMYhJZkAIJyCg9T4auazQssMKTObH24zcoY\\\\\\\/dd\\\\\\\/Q7fdzmWkb4RzOC57xU+Af4WTtjaDe2qi1o+4L7hJuCjM6Jx0PRBXpeb7D+VVOTMm97mcmCK1GbMEe+NHjGG9zCe18Jt\\\\\\\/53Ewhu+71A9nqiTiWvjHH21tamamoj0Wp9OzuH8MMHV4g3BFQBuWByxHr3Kiw561+CpkejwQUgHgE\\\\\\\/ZR3\\\\\\\/eOuH+NFuDIOjDmpetLECkyPxoIzIQUQ0flMs1\\\\\\\/vJJ8kjxGe8tRxhoy1Mitx+8i12Mx6OI0SUMJXT+5wP435d\\\\\\\/Mux1Grs0bfjchQ3ZMA1e\\\\\\\/B8jfsMb9ugvewB4iwZa3qFmaK7WBLqWNT0CILRgbWzex8IM4Se7g104Lp+mm69S3\\\\\\\/JUTyY6UsUhlauln1UvBV+f3XRQPj7\\\\\\\/aUFU7MaqqbA4K9X4TXfi+LCannxfP7K19v3GMWXZ\\\\\\\/qpRsobwedRhiDX9+4LiB1v0l21\\\\\\\/XfysjE5vv\\\\\\\/+GmcYarUPiAtiVl1Asv5KkH4XuJvgeeOJKi\\\\\\\/7VAszUSyMmg4HiuaG5hMBJckrntX+sVJtoscxu1jZ90H+NEYIl7L7vfjs+bpI0nLGZSnZ7UR\\\\\\\/RfSbTrCeVwjVpXIGpkyyPWB9TyOwf\\\\\\\/BbVsxFEJqLF9hMjn0dctmZ8ieyczaQbsK56DUA9j\\\\\\\/8axJHrYTyhEoNwzmeAaABd2i7cyz8p7TcfvRsY\\\\\\\/6PQC3htcM5lWKLoLnfNqY5sbCy8zj2dll2KDOJUTM1Zoh\\\\\\\/aWmlvXCSSzrRcTdsTkA+0slEIEVqlPQTTzX3uR14MAACJv2DNSM0FOlWaUGKvv0yElwtk4k3UUAi4EfAesJC6uIuKFSiE\\\\\\\/1CktOODWb6Zin042v9sMY5d+nEgmIsPB+sqbAo4Vtdk7i9B70D3dd13XsqHY96FnyZYClQwIskDmDnhOzEH+M+YRlR2CCGmiY9spJNHzRfoVUzP\\\\\\\/Yuni+pGcqP8Bh5uUT\\\\\\\/sce1moO5IJdN8BJ8TPfzPs2j2DSxraeMR\\\\\\\/DkxSMBWpex1NXbx3d8fELh0z32KTFIjwa6V2vHp64CRt1SdRdR9knOUHBydNJEcAl87S44kUO6jt8xbBHJngLekiZ\\\\\\\/rXjAQYC4sgPENDNEjf0jluzFDl0sRTsimWvz9AdBoGanBYUxCUoxC2CLtZDwYUL\\\\\\\/u3uY53Ccs8LVebWbSUZkSBLbAm1AHpxX8GYg\\\\\\\/aMRvmzUrVxcicVxZtpp6+sc9eD5EvaYDN55CKr2r3qVaXiu4nm8L+KIEM8TJIMBUH+DSdIx4fzND4EWxFmUQdeOZqjVTbBo\\\\\\\/NHCsho7rR7PyrFGJnKia2tMaceIOdwJu\\\\\\\/TQsLDbnUTffZ6M9YGOtGfHe4yA8vkVGl3twhK1SO7eNrWrfr0b4NVCkYSsfNUUZxkcQDYCw00+epgEior8ufekTG1MGmb\\\\\\\/A68JGXCaVKeTwCy1uwB8kINPpAa964qAVlbZzKRy3Ahn8jaWHj2X+NHBwtcRo+8s2uI2I8GWVIOi+ViVXWJPTH9Jfyuo5wwz9pzW\\\\\\\/\\\\\\\/m5w4BApSdWRnnQmKhyMZtMFU64OfJSWZF1FztY9dq0kevzv89SFUVTfbYDyLa2llzZSXdO0isD1AqGoAbYm+TLTX0leublSRlVF9FeO1x4R78Pq8smTQ7WcYSx++GovSCR0kngSbngY6MHk0mtZntXB\\\\\\\/nhA8ZvpEwgq60k3cxdp6U2pYKZ0CBqVzOvrPtaHn4hkfvlsm\\\\\\\/pcopEIr51s\\\\\\\/wfRBZiD1EFq8INQYb+0AU\\\\\\\/dfQdkwLj0IA\\\\\\\/E4fEX50QRj3yiH71Uz\\\\\\\/34fkh5yvNhmScN9zEShPwZ0N8n0LRsVbPigT6pvkEGrO70DKWCU1PSCO2WqM4eKCD9HUVa1ZtBXr1JrfpQ0jl0Z+AszXOsDO3qNPclk5PFzBlWI\\\\\\\/yQbHvmlyO7PMbehb\\\\\\\/KzgslyGBGDn\\\\\\\/\\\\\\\/\\\\\\\/UuP+ER80rsBLmvtaVbN1RkUe4FpC4odd7dU4kBsE\\\\\\\/4OYfGhwC2f2jonyFAUc8DltBJtjG5FHXsu10tkjYRgjQQov8Agqi2rn6R\\\\\\\/N6iALZZSJhF33I5vy3V7nnguCKeMxTT6GENsuOAJ7Fc9YAuSLIElYjp2u7kJoMz1u\\\\\\\/yv4Fy5C4hJze7S\\\\\\\/Paoro3jQ4B0dA+3ShJYtMKYcK5L8fr6\\\\\\\/3kP0B37jKL66WKvfGb0RzU3yx4Ge4GEvLjfjUivm1MTp2J0\\\\\\\/PtIbFaxchJHHZ\\\\\\\/8IHwwfuSECXXq+UEzVr6WHNyDxJ\\\\\\\/gLZoCtzn7lTM2ttNlfojAkLYGVTDqrggkgQ\\\\\\\/K\\\\\\\/oTdd9kPxmyHDMxnW\\\\\\\/1M\\\\\\\/RHlhbxjgUXQHvQ\\\\\\\/jmuR4i8WmiPxBQuM1Wv8syFoAWbwj30PHUX\\\\\\\/PmG550aZKMQ5Upy3woRyFt+UdM4GdJr51G0LShIYrBeCU9IjHUI4PEG5d164GoRp4k\\\\\\\/NZ4PrT5HaaXe7Ba0kptUlPn4I02ZLeEF\\\\\\\/+e0ZbtRtkhKQh4TYgkNSq28YwNHWJLGSIFnB1qUaE6esDdJSA1q2l3OQS\\\\\\\/fR5us3RiOxB\\\\\\\/q\\\\\\\/mmYpnwQCX22W859vaKNhPXJNAJ9YRDhYELd9brhTl8WCsYsLE0ehMEMlp4AC9homk5\\\\\\\/\\\\\\\/ms0WPAhJcNneXaTE1IeSH9G6TLPivA5EPVTD1uxwlIJHfPEROluCWm6TxSBNxF7AYM6c0OgKbN0rHj473EWnz20U37ITRmNZEElUzQ3hVUgVU2EKlXFjYK4K1IROzRSkjfAjlPHtopiwrhW9kbz5HMWglA93v8v8T+5TJ+DEJ+Noi1uifkaRVhzmuap3Rs\\\\\\\/g0R7yAQjEJapX2acPHjCCf+m3CxcLL7hiLEg3dRJ5b7zcgxBYXIXIn5yVl5YbRTEffx37c4sE+oVmNWENEW89SEO2gZmAAmSi007Yo3wZSCFqTTUeyRz8\\\\\\\/c0nO+rC46Dl\\\\\\\/5rrpl9qi6UWJX5O2uTviTeoLS5n\\\\\\\/axiHH2Y9VgLWh4AYvwFIzfybvlDV1fmsdx1S4N1rp4JPeQdvNWZsf08LEKE3H9OThjsAs0NpDMQUjROCnN3B3smXrf+T2T+XE01qSnkUgdbNTXflrfTzgNpF4wxA+apoZ+PK8T\\\\\\\/8g+TAyUl2\\\\\\\/oc0MZaanbutEs2ShNLv\\\\\\\/IKUn5ZhsTyuVIhPEuXIfWPYSy3VN2q90BTRicW\\\\\\\/DwH60DN+Q4tq9qwzS95HR3tUWb3X7J5LV0JM9MBtbLUv8PhrlJD7BUa3qmtjEK4AmzHnNN0FxvIRrJVm93AlES6O45pMBeWFsCdjdm++0l4VTqvFE3dkG6fYdLKji6b2AlzzXodJcOETokxtW33H9PIOhgHKzn1leAaUVyKfOQ+bt\\\\\\\/WlHJ7YfOU6PdvtjoDoIIt126YjOmXe5te\\\\\\\/OGP6SGPaCT1i52VFIE3RP2JDqAGe2G0jpUDoT6iiK3Tvlm34dz42ih4Vincd3QHuF8sevvEBOjB\\\\\\\/WBiaBLHUoA5jWIrA6Ndl7kqMdHfNOHSX3I7e9qIH9IfUGv6aPPHnbw3tXawtBkdAJbSivcOlNNut6u3SRsDYn7MY+HYcMcVXsuRrYvcspkbft0HsLCtY8AcHejVmGPjuGDcmmLwS4UeazQ7+WpJQn3MYvKjgy313yyAsNR3wbcsOqO+RaIDtJcOzEBypTMlRM3sXsBe0Udm6qw\\\\\\\/m2EP32kG5T49Vp1GBjEJPnn8cQdcwNN9w9VFdsxx5TQfhhcfpe6lKiv536gg9bTFRCW5Fb1IlEds5dtbB7G93P5nmyqUyH6YPKfvX94UWqOHV35mMwu4KOwlBPYnjKpmZnYodcq4zcvqCiOsqyTvPZelIWD3gzsCAxHfDBhI5H7nolofYEwMiUIpVT+ftULybmGZefkhuk1WbDa1rrtkgnPKMTBd1tbBDxbzkH+OCCrBgT\\\\\\\/HGbJuAnqWU6yLtDN7K0pdhVoltNJeXow3dzrxmZOe5Vn\\\\\\\/J4VM8GW1bDHMuAgMpaeq2MUpZibAjzATp5DHXHlQKW3k1Qtv901783EFbLZCPFEuSLJx2c+dM5lcTufTS3qyDzGlvgpTkTjAq++A56Oy1m6vX4jhSsbbOQK\\\\\\\/N+qEPeZlmWjW2Ft40ALn9nzJ9mQCBua9HJ0VnyveQfiYGZcPTvJRO56\\\\\\\/LWCpeZBpbFrBKmIdDZlayCv2aPsfJIre8DioF3S6dfP6NQQKaMM0ON1XyYwYVZW98Tb5MuJbL7J9awuBqr1Iw5GPq27OT1p+SkjYTDoiaJWjUyfQ0QvxXGXVNmVz3uNV57GoqrGtQGTAXa1w0ViJzTdGfVuox8ASb+vQyIT\\\\\\\/RKYp2zGILIEJkNM7Sw5cewpdwA7h993CXrjl\\\\\\\/E3vVAJTDgxR7YAyytukkPEI9aBBdUZfswgBfUGDluK6xkbwcCXNvZDsFwItIFp8xq6Ffo0+FWVdUdukiI5GRKxkZaJdC\\\\\\\/ZEnXcO4lzfIHSMWnsL2W3oE13+Oem4fvs6abgVRS2hl5LotikzuJkHJOh3jcQIVW2Ahycnwyppx9PbFfJs2o03NGNgrk5jVl9yMb1evjZYKDkz3oyuEcIF+HdTSBOJ7ke5+jyIFxYyyJIlBTiIE8e\\\\\\\/7xt+87z7rr2pq1ighYQAqf8LGPLzrk7TkyD+WEHUicbnhBoVB3HstXOUFinmRzsCgHiLivtr6zg3cGpbfqbMZoFU2+3WBRwPslwMx4vC5ZN26sGv1nOT0FuJM91YXE\\\\\\\/oXKVLlKNtJWPZB6\\\\\\\/ZagAQ1gTvfWTBLUi8iLVFfgCx75\\\\\\\/bqVFT5lhPxiRCWUXHUoM\\\\\\\/HRRjA2KYlcfAAZ6QFCli1pNeLnhRIfXlKUDr+XQ1BVtxdOuFI3B1u+tB1q7g0YC3d2Ft8XBhKXg41NXt5eQ\\\\\\\/LWm5WsqILqje7mMQkfrrHKDNb2XB4faA0YbnWeAhH\\\\\\\/kAOpT8UIC4V5waRtxO2fHuTeSKCYfvrm2SpcNzmyr\\\\\\\/xELsnK81bL3dcYmNamFa4n6nT6Jm5Tun15\\\\\\\/SYbHFD3E9FJ5aNMKllwWCv1lC4+0STMH24G8EjyLp2k23sqTwBN4qUF34ivXUuLcOAAilv7QvEqu43h9FJRRAApnNpUi23Z18MHDYYKviwGsalDmyQqovpRytZsHhuztBskZiC5bSB2MeZOtlg1RrP3YvT55VPxNVNUXLHRyXG11+Ff3Yg7Yf1ssPlSlkbYorslF6BhO3++r6vqppRtFDaRSXFBgdKV47s91ni1mVmLk6HeOdr44iNgpY9mRBT3uC1fgB4EmjrMHtZhQP5UbBvxcaHbmGg6Hj\\\\\\\/Ecwuf1kHaguhsRttlDJfaabw\\\\\\\/Rf7uu3f1CPiTz4Md9mkhBH77+mEvAmS5wVAgy9wY3LbOsCNIBHwVuaQ23hTr2dJ\\\\\\\/L8uf8M\\\\\\\/It92G9lfFb77yD3oCckZKrrPdZFQUu326JzmVMFpTA7cZNTEVec7ksMY3aKEUOiGYIVshmLgHxC6A2atlhy3ugFiE83KrqR2T+X1DzOQM2uz\\\\\\\/dIvNwWwaGvNswrcrAJV9KB7Nw\\\\\\\/yhJjNnE9z9RDtYzvHISP3OumuyMOmAFrUly0JddTPO0t2PcnqEAxUezC5TsoXvlFwsIfgFZjVkT78zE2502GSwmUwIM5PsBhOczb6YIMZQTHm5Gjs1iB8KAAiH\\\\\\\/re1QczfS96S3JhoqfICceQNMCiRUk30cyW66aRAi24bAIc83ZPxhy8F0q\\\\\\\/EPXYIWq1YijfWMgYUYBkivK3X\\\\\\\/Loql29c5wGznmfxJuJehOTwWYu5B5IcuCWGy448YtCMeABHBMmIgpyVK3vr\\\\\\\/oFKF2qfZZ0bGMRhdmWbFSER1\\\\\\\/yquLEBJOOEuZnPz54PU\\\\\\\/TgVlTnX\\\\\\\/8kNW7eXzvFXn2ThpgIVEcExRyBORl0a\\\\\\\/3mQWC9WHxXX4\\\\\\\/KFrAxBeLGpecU6gD0MRNlxqwaeFjUKSm9XQcUQwTw+tE+I2UdIcUg+63HRxRJL\\\\\\\/rDcrOm4vHVc3J+1bN8EIXw8fWmMUABruSWC66FpMOACzmClqshNYoluPzNDB6S03CD1+XTJakBcXSsFZHVpO1VVbnGcplEXzdz6NqMqz1VMOJUU26dAnIM124EHD6cx67AP8G8ScNmE1q9ZGlY83bhArStFEX+\\\\\\\/zhp3hiDrTeihSy\\\\\\\/Vpy+u0vdAsWrO+oCvv1cU6SDoOOqiMSN8qcYNCm7Ceq35yacAPM6R4G0AInXh6Yh\\\\\\\/kNti469SeT5OaBJ3eWK\\\\\\\/sey9A0Xvxb14z\\\\\\\/uzTHiSqCtWjcQyj\\\\\\\/uwbs4aaOv+wu9Gqi4AeWdO0MnJQ8VO5E8ywCoaRpNBNGkFhK9OFkhgSC9+OwbLtjLbIO\\\\\\\/dYHnISc5Bul0t1A\\\\\\\/CbkdbEqWfzS8+YO7c6C4DpFwFsimRoR8Q5rUtkNmKch+artaUFWOjEaSNQnU3vyGruIBTLisQIcfE8N8KAry2zGTaS5+J8GmlRMteq+PE78Te9FvNj26vsNJkZjzkDUzmEjj2YwICvLSGIu5M\\\\\\\/hVYeg3qXDYO0u7JudYACPsHHzEdX\\\\\\\/z4PBLH3k1w+ORyA2CLMM+paNm017SHTEFvJ9153PGkg4uUhR68G3cYMYEyvcT5Pt\\\\\\\/HMZpy7A9paZif8GvV6IqPmE5M1wL30z1gQosuiyoRhgghxiq0tnJYP4k0b7Z5kPW9V\\\\\\\/1fMc+IAfsU9htZ98fe+bQNJz1BfoO1Y0A6SlF5bNbMDDz8ClmbU2dSd9Y8VfaUmBQ4XmHOpKyYP04JfYg1c2dk+\\\\\\\/qEy3bJMd+KtfqOrf96c+sKfvYbcO5JoWEf00VnNb3sQmTtILaaufjzFY562eMuGDr5OnV4Po34Z2XYwAmi+MUQVbQXj2Uzh1SBkEnbcgIEqsS4Coa5DqAakT+rmTOUD7TTTgawuxQ7flF5XMGp3LrLr6lo8c3LIAO\\\\\\\/xUqxc4SYAXPPyzhEeOgYWXRafgYJlVsewEy3Ls+NSVcunwiksqAKAlJyIo7uNAT\\\\\\\/NgHj3+q7fBVU2uIPk\\\\\\\/A+D5RhfZeAOev8yWMLa++6\\\\\\\/TnvzCmE\\\\\\\/nEqhUNOFjm2suTbcrE22qNpsvOkXLhrazVlZWlxHwhIvqlttMu+MSGyLW615Wt+4yxLz9mRsDleUhdr\\\\\\\/EUnZPVv+J19HfHlSaarwCDQ5MfRxIM1Bsb3Nn1+\\\\\\\/XVCPVHM6jAOaui+BDlCwOyfmFFMVnyibBKiBMucFmvTx3Xnf9Txgi1vWWyJQ+XMxRArVx+wJNZ1gSL99fyWC9+jayJ+EmBuvBzkuKCxkhYLx96i0eV2JxaVOm2nnF+DdzVihKl1HR1JfXOxI1qecmuEDvMLazVrkiNzBezfZI1E5rHkPMhUR8ZgBn0tmrmm4Ee+vk6A6h3z23FrmkQE8qjRGcLmC2FTRyTs8RddqMtWYkMq0+QGq1ZvAbp0e6GrYEV6xNQfvXPXfo7uUaRwjttBZKXkqjYN4qkwrqn3YiS0fwM202AhYkcO3NU606FSnDW+xVmJJtls57PN+HicP1Bb7zOUMlwXw32ribYYGPe5NWI3rJoDWAb9Ya2JJ\\\\\\\/m564V96q+eSueEuwQGXgKvOLu43dMmptiRGy7zXOVfr+dpqVG4WBT9GbDcSA25qJfGpXMjdCv9Su0apwr1x7f+2vmSWwNhpn1toBp6GuQdXnUNY18iXGr6iq2Sh0toOU7xyzvEbKbavum6c9rrDxaP9rQqPZHUh+bQYAc1NzBq6VdWy\\\\\\\/SeqpAY9PlNXomx+rzvVOsChXdjliyhZS4sADyF\\\\\\\/HZcc6Hln3nLw7qYGisLcRNi4wSLDNnI8rwhFVIfjDDxke5MXSQW6ZbDdH3IRbtzmc1qXoHr5xQ\\\\\\\/+rRpc5A+cWuiWSZ5QbFTU5XaPHqWDZUM7O0sNeULUX3yF8yP+vrl+3JCJc\\\\\\\/7zjmYBdQjCMOce7Fpr2wKB0Mg7tB7C\\\\\\\/1i8nnAkT1MlLiyuWuEL2Qh5kXVSiCQszmnb9fT5U+voK4c9ObsrYDdkqndBblmSfNdEDhHmS09jntQIgAYgpZxiJ\\\\\\\/1XFD6ULOvSuM4XAwNnZ7U9oB7ExjVMNVDC6GxRniJkzq6elCprn6CjVwfYUStNpEBonhMQQAN5275R4ytcc6N2Y6IaEZNRcYa9aYswA2UmbgMLr8gmjUAdAOYWqtw+1\\\\\\\/JfUTE8FG5NUp+d3\\\\\\\/KKh9gNReXoAe2ouLOt1daCD3qxx0Bs7Sill\\\\\\\/2Kt+inPGxIacS20dTvxpaPLKypjZxPeZAyOLAJJ\\\\\\\/8vHaa8cg\\\\\\\/4+VlG4a\\\\\\\/KXgPETdR9hI2TgbSc7Uj3z1bHw43rVVyKK6Igp+79YZU3\\\\\\\/1S3mKkYsHavadtfMB9gkpaTVk4p+0jvss9LFaiLP9uamwT0xmQO\\\\\\\/k9sIG92KOg3WG+Zv3apTA9jNP5L7BELAO0GsJ\\\\\\\/tvabJxkiEWOBBPbCNfZ+qdZata\\\\\\\/IUj6wx9VYyQENbAhWFp1z7H+Vyvc1\\\\\\\/t0PEJbkUmkuGO5f2QLLbJAQhODPfO6q+k+MxBGyfoVpAbZfEPxcSjerbckOw\\\\\\\/+zz1yZT6a40hHTdyfaLKsVF2zrafhJBpwF0GSVg26ohEmzJt3EeP1ZOS7zrJPdDVGzvWk4fkir67VMS3Io91LCovjYbzxC68PwpK2fPd84\\\\\\\/xaI0zfVzv1DFwGdSDbhZhOac4b+kz0bYEUk1mcvajKka9EiwkGQ6n46FeAkYzD5cOXAVLLboo4NSWLXBostzHc6\\\\\\\/OYnQxlzR\\\\\\\/gR08WJFuNRzlCUASaCWIAjTHaG1Z8dom5R9y+Wdtu4kQpnekxoCD54QHWXKxYM0PBJm9qmNOHiTzzQtKS\\\\\\\/8Nx106HH85DjRgSAD67pBGJsiFRTHEaXv1dryPyMScRHRleFdoRNT3VaR+3dAUgrHcoHfo3Od1KFUiCLPyfB08HLLi1aIyOxpu+yt\\\\\\\/M2srlGQMdiWmq7sYIwN+Ux\\\\\\\/Ufidzbf8oaFxJyUHeQgBmJ8EYfXymtxuC4ofY3DfmTv\\\\\\\/L0rRwoOnlHALGropqzgNXVxe0xD08lc9yxMX+CBqX1hJwYOvBM9dZDEUUJkOc4RFq34yejFARRaiHs+PwWjyQlnnpuGCEzhEy7GsHnoE5fj602Pg8PTXGIKswVgGLKpKrNLlRkpfIEM3BYDUNDpRNlRGbTYxLGAIMZ5SmsUah9HBiQ5YdPXZ73U1p9w\\\\\\\/815a1F1w2zKrZpY4eImGngoox52AWRf3p5mPFDgJfoMWEq3SoOuYMRQFYwQXY3IXN4qxbbWOHal9ucHipNIcrN1oOYjG1Q3\\\\\\\/gHpWlQXU3Pbv26o7cA4Z8nc7GDPHwSNQ2NpqaON6cRtJcJB1+aOdG3ijKo2jRkBZBykmD8VrupC6hGinLuYzNlGawyfbM670jLm2GGZCJdSI\\\\\\\/rbsWx2EfnARTY5\\\\\\\/bp6a\\\\\\\/IXmU12rHpG3JRR9ctstHrEoBzAPF+b7K6x0dKN8geVH31QLkBctXwJAofpm+Cs\\\\\\\/4FS\\\\\\\/5yVwIZxmLK0kXH7RPlGJKTQjYwbMw455myGKV9leNWvQm+2cRVj2n3lJXa7B3gmWEusfu3fxAgL6fBhuCbIbZN5us4G44gbShOFBDHFNojVhwaH5JYlUHbOis4\\\\\\\/WmOVqIVCNFdi3sGJhUBHerciSqfn5TRCSZEZhr56XIXZqyFap9Lmyn18jYgJxYpe8mLwxIdRmABz2kaSn07wGlXPOlFOp2uA8hP88c6Q7g6+QSdER+PbRzTJjlzQqjvGtP8RlGZu\\\\\\\/Q63O\\\\\\\/EC4DQu\\\\\\\/qCH5y8p3qmfqZfd+sOia0ntAKROW2F1H0MubYNknGYlEew1q9xjFjT+qbQM6bKQFD3WAfjMUQ\\\\\\\/52qWde+\\\\\\\/OVHJpx2G\\\\\\\/98zeLq+P\\\\\\\/V3gXDrimAPVznvPGvCKL9rbBXjRojkhyJm7XYifUzWwfYrItJcX8i\\\\\\\/mzkO6qgG3FvK43FxcS4\\\\\\\/0zct2fDXEYql6ho6C564Hc0QFWof0mCVeLw9PqvtjJ1m+QGZERTxVDgDNbhr9ygv+iGCSVwfvyDEfHwYH8YGOehBornwr0b5t2gA1eyJhG6GI1zXbiuCcGwtn\\\\\\\/YeoPcpO0A3KGsdU3f4ZYLTzD0m+uDrLM9mSXiZmJPnFegON2EugM1a\\\\\\\/3R2auilvMDR134jfN4y6f++9kC83BJzv\\\\\\\/6bqza2tw2M54j4qDGYEuDZVXdsgvxOfqPbBF6YxSk1f11y+xDQe+dvc\\\\\\\/t+GvDeLrin813JccPXfITQBg3uDJcxY2ear93VbGWrsfaG9rCfhKr8nPdEXDBf3Jo5BZDe0XSjiN\\\\\\\/7OYCxOd+jAswPMe8fk+MI67UbVHkmtY1iWhvGoXdVZNeRBZjLftGzPZ6ODSkTl1OGK5gWoM22J1gdRIKn+B6l36qmTUi8LOVpIeOkiYJcAzKY0IXjSt0VXfFLr2ZrFIeW1LNjV81FoHI5bCC8RWPYRfn\\\\\\\/vuvYlpecySS5gDLra24Dv5GLF1Guf0Z42VuGEc3D2NIZ\\\\\\\/hRME0OFewOslZ4J2cin2YETltZ6AWiUXefBpw8qyIIQnxOnpZjMrvXsnBZ6Z8g0yufsNAe8zQpIwXbzc6gO68JtM5xXY28lRZ\\\\\\\/3Q2PzRs+9B\\\\\\\/nH1b4ZMxpZv50JIdE8V04fR3cYUyYWCtrD42VS33CY6UXgwfOCJPmS+NItf0VG7LMfaWPqLh6jPd+R5eoY7uvNqL61t0c8RlvN1PV\\\\\\\/vBFG7J1GdSmBSrKn\\\\\\\/bsCY6JwOLwpA3mVCk0qpqPmFAkAyR\\\\\\\/e8ooQtlRZ4NljZKGrURRIH2oDpHTDobF6POTNy6rL3nIw5M4MQJ6ytojPX5S9w7lcNlHHZcPlDB7YL0TvIdvgZpjP+HuLlB4nUcDUWCAKmukMvZi7+d\\\\\\\/Y\\\\\\\/Mv2W2yuOzejV4ZOowWIcsg8Wzn82iA2Ln+SZMC2lkWNc90JcMeqwJsGUAwpOAO3AtNRe1N\\\\\\\/f1hiinxkXrVi81rqLQfhSIwLrp4QLooKX+aZRbbksDRsQSfYhRe7cK3qD7HeK2EFmkEIf3y7\\\\\\\/Bcve8TVHOeLl045BnVvmZpay+pmwKKcIQMsMV2il4e\\\\\\\/xT9zRdgbbAj88KsgHgeQdt35p4A51TK1lerJEWVdxk7PUp4l\\\\\\\/P0qh9PWif4fHwOoPslcX5bQoMLYZYsBmcGC33HgNcNM\\\\\\\/hiB7iL8mQrAyQXmxLztG0UVBDV8u\\\\\\\/5YpO4mWTEGMnmkQ20cSua05zdVr53ooApOcLuGGjbH7d42fNLyUjcJSOz8IikdGrPi612FSoqtoQteBgwF5cpviEDlEsnXsevL5ckGRL9X+1mkYWr1B2sj1pQRDNcJXTVA2TufkILqyGdwXe6Agnq6sOvLZdtNen6BNeTdFXqi8UpAY9js\\\\\\\/hhK9y9ISS6pXC8XcozjUuV3DxQw2VGLUyJWrM2scaUxsjwFqkSxsSJDInhpiJdChs4rCVvBUZcuHncMqF\\\\\\\/7rddvu+ZuzfAyQLZTy9x\\\\\\\/x9HqIsxt9ZEZgdWC8+4b8koJl3oSMK77\\\\\\\/wGMI0D0+g8KF8q9OB0m6TMrPI7DDlIFDG1\\\\\\\/brh4MAtWDb68DBT8mD\\\\\\\/SUkpO92P6KhlpgC8LcNuPz9pzpZ7WsRYLRk3O5ymSr8LhssofFQtN5KLknTzmY7sUhpxD0\\\\\\\/lk3N+ANmU3XSYOjGhMeXSIoI1bMjiy5ceMEYcbuAmoCH6qPlKioqyj4f2KWSadGMyQgjNv3MnFBN5ZiibmjRjI\\\\\\\/SMOOr5FVn8sjOneVSpRX5HZPBY6\\\\\\\/13Uj1KBxfOT0dz+DquNOuFAaMvwM4UFFGbBZQf4+0ZTwnIP3pyidLuHbPhnEBmA7EL5f5JNe\\\\\\\/BKrx9nJinXj8lf7say2ok2e8RMXapl2uUkc8NT\\\\\\\/pXknyUA9EOCv\\\\\\\/1PkGxWFO0eBOIJrEa3ohr1yJW3WtLGQJql6AX3nVBwR49JdIh0QvaIuD\\\\\\\/luIgsqoh1hg67eUcP3r0P7D1T6s69KEJlcdKEfLDL57cS1ngG9EeOoyjPyN6ifm2Vu0QHDcnusNuinRs5NJwJacoMrIAxuS\\\\\\\/1FYlHLXmbG9tRmwT28b9YuImrx7XdKBCfrN8JY8xx3mlBxP285sxipO4YwpxPhOjhaGM+6VVKN\\\\\\\/CJrFkJYUcsQzK0mSRxUlarb53bxv4HJtOVm\\\\\\\/kPjP8q6xUrlU2ohvY7Dn2RO004iAi0oEEsgIn1qD+4m7vO\\\\\\\/bCt0NQVhj4koIHp96xZlSo0YiH6NCAvMl+3+tWwQBUWgKjQH+U2QT50kaufHW+HOL\\\\\\\/aKK47IXL6\\\\\\\/TDYYYkksMe1RhFLSRBFdRtMQ0ffs\\\\\\\/0xCRuEriGzRS3PTv4W56V3hct4xBt4gxZHRRB0EzmbNLTrpACgTEp9Tc3Mt4ei7YQwyT\\\\\\\/fjRI+rIUqp+CArQ\\\\\\\/kMQrkWXXpGy+7hQNDu+I0vPNH22Jv\\\\\\\/Dc4Uf2eWBd1WpmJ\\\\\\\/J7nsuhAh3F8E4eGdXo+TQ5oM2+dh18P3gIAJqazpm3xW53mbxnu54YMOk+CJAW4NTNpmQs1u2ffLEVYXjWgW\\\\\\\/k5pLm1IenualOYDc1PTJgpRwennorreU3B9fbhPUHTmElfs1BcL6IF5WVm6jyhMSeluiDwY8o0XEDL7DSD4W8uZgJ+ObIqkdy\\\\\\\/3M\\\\\\\/W\\\\\\\/b9ITM44FzoZ\\\\\\\/ZilebBDVK7GALOmiifx6VW7qiPxte1WJpUci9jtQwoL7cYWJg6tYgBIEvhLdgkrhz5KKIghQqUaRB4v8biVJjGp7dfRFzUjMPHX6n7lC84NjfycXuN\\\\\\\/Ze2vzghIIo8t36HgyeY5QfJewOfQGOeh54P67\\\\\\\/4QfUAwEI+5XnI48V90dxSy5UE54KboQld0F9ak83lwH6btEMKZVtXoc0az8zjY0t3d1RVkVCoHTJKY4+pKRwrXSd82P6AJz1JpFzZOyZdBoEd5mBUyuIjRUOBIEoOnSlwCzew4zQQKdJ5mo8VMwQePiqNhLbOoXVIu55xRC5AOmNnOnR381+wAb2abEqjhmviEGS\\\\\\\/Darb39SQ1KTcsmrUMsNEn6iK0NF4TWXMZ0RQNy4h\\\\\\\/X9K6s2E98Nnh+Yq9\\\\\\\/\\\\\\\/9PuixXBaon9llJcaFFXvRXdnqwwnMmDJHIW7dIEFaX92nG3Me5JixCaS1EHaxpKOkJb9GiGtpoX1rY0IQk0MExuLtd0aTPPY15wlc6puK1xD3hP24o7YZoCEnkOoFAEavsJ9kzccv22ws5XcEt3iLdO9Lwfp0h052T0wiGpZH0Q2tjGwBruqBLJd6SlgsW82q2Ja2B7tXbGA\\\\\\\/GjqWBc2W\\\\\\\/f9lvTxhxglcS8fNGf9gkdC9i7XvkKEaCFED2Bd4kNk5o3GEwa9MidSBgf2rfBHJaut4ctOMm+8f7tLdrNRoOojbgJj2TGswhE6Opg8PCVwWytX06l2UDAbMVG6AQWyiPSmsllcvp\\\\\\\/r5qYSEztNxk66dSaIgTMxKjBzxlmhNGO\\\\\\\/ZMVXQJ0Ry2Fx1gaSD6Lifcu9CLU+zQdJ0Qh7MC\\\\\\\/7Nojz\\\\\\\/hNVxV8nV50Y4nKvAh2gJNebAsyjsIZbcx\\\\\\\/pBb4t8nX8e\\\\\\\/ij\\\\\\\/T2IwLfVUl2myXzGStkkj6gkS0uM++o9msUPnmYu8AjovOtGE4ZldLg4OqyIunmfDYRvhUFXsVEF32sFL4ZTTg5TEZtqE7n4U+119kz9cJ2U4cIa+7u8Wcc0kOlIXSwvMtkQGOWGpHVtTBBmby29xM9b233j7Pvuy6G0LVO1toghCMteCxove2ALGDy7MxKqQPdq7AKeGuGruKH+uBTpAcVNU9dx8xdxL39olhCXKLbryMBxMeqOEWdLlPffYgWsjadOmPQBt4OBAHAljE2+Pv5hTqmV4M3\\\\\\\/u7gSvsUfm+IJ59p\\\\\\\/lirClxVZahHDjXv4fmGzKKSk0P6KSPiX7jbepgXOc0ZVhrrE0dAiaoBX3jLgbWgVakncDqkXQJwYKRZiVL89A37LpHObj+WA5kUE5dVD01Lgn\\\\\\\/mpUel0R0qhlgmMpF03erzzCpeYhRvAQnjGlI2+MHVq6vbCoL2TfPMY40hHn7r+aZ6oWsJ36zPhKCIhfB4Cxky1mp91c36L0ZP8DCEYu5pH7Md+mdfxrrigKo+WlEo5\\\\\\\/H33ra2nqNbBrUmIK+\\\\\\\/3fgxYH8PxYWCbVEaQvFKQnZlieVpZ7768chhaOjPKNC3FGeUA1ASriWnMcp0cCd+xO65CoVHQ5onwv2jM+3T6+lodijE5Ad67ZfyuiLPBaOysGIPCYwCf6hcMCwV5634sxkxiMcOB3hnDypS9p+J8\\\\\\\/APwsCQ+VDdZI1IE5LI\\\\\\\/YXHzYmJfs7p+0X\\\\\\\/kIeJpXzZaoheVKvuvJCHmGb+zMnL8e47NLTua8ND4a9t4G78uBgGmseb2Uf9TO6ypMq50fPn+HViqO2LNsB+3151YPzJHWv34eXLYwpVYhqYWX1OE8X3lgkRcpZYz2SleQEDZegOejEfMJYyl4I\\\\\\\/86FY3YkovE9RoN8VyIIkWH0otGqnjFdcp8lgpO07BDuMkCJRhi7Ge8VxAk0RcTEa8M5nEJUg4E9Gg3o2m2bMAncXE2oDY0Uoj2rWHBGy7YTKPg047ae7Cb9DAPY\\\\\\\/d8NOCHnlaPq5rmzaAYhIKEgIr11mqMRcAcA4jpzdEmhrbc9Qa4Co21uiNt7+OFOeY+15TW2HWW5u8EkOawUf07oExHOyo0NNh\\\\\\\/nIHipxMPvXR8iQK\\\\\\\/MpnxtD+zo5wlLhPlT6tD9zYHJQoAhW0L4d2FPMi7bQkYfrFTOtdJDiN1fpkV8hb7DfwBPz6c6s4h091Q+GjR\\\\\\\/gmF8QRzjdY2N96AutdOFusywUTisiUF74xVxbpwiIlKeKxbySKpEOTP6gzgZo57YSI\\\\\\\/pO3pz3qEQ\\\\\\\/9wUtHsc7OY0N48R5cxsQdwUnf4E78\\\\\\\/xcCjYRsQ+5aEYkEjQf\\\\\\\/qOQmLl3O494kziBzcJ6NkOx0t0bYCTebYLTa358r3n0U6BlP97gpKQ52hwpq3eGFwwDEgA8\\\\\\\/0PdgY\\\\\\\/mY9RdPkBJFyLQLjGzBEWvt\\\\\\\/uCzNoX9df0chLy728YYYs2PEuXpVbDbBq1Hatsf6lTJ+TILUaK0Y19xDju3Vc8egGF3Bu1cZ+HuVGXGELVyTkyidhExHtS5\\\\\\\/EHnjwTzFelvXh4hJNouypfu5lmN\\\\\\\/LrSlGM2cRFbvlM7DaFUDbdY4LZRSF0IaMq1BLV7YSefgK9JpdN9g+a9qerpiZUvaeQLSLj3MPZW20ZVyheKHe\\\\\\\/OrarbBz9BiaIf\\\\\\\/IlvTdNbFydrbWTh6vwPEpYvsAmEZdH2PB2RGkC1dJOn98WJzoRc4Oi2WFcI+VyB1oH8CQoV3D5Afy9DZZjWrDLooPh5DpihCAdSU1nKGG+823RoYcM7aiRxtHadFBbtXJ\\\\\\\/65\\\\\\\/yWWySo0GuknvZRRP1TtfimpFaepCV3RhTeWXf1mTgF6kEzV9qX0jpjH9h5hrusQpVaP+5EW1FlhI9b5Edsa7VLAhaE0V1JVPLUZeJlyzvwt6mctbrxD2AN9OScjrOjbJ+k9XhhCkvjD+h6qTg3pI+Gwr4IT8gmS\\\\\\\/MWJ4OAwHwH\\\\\\\/ziWfAvA9evWopjIgEatBI\\\\\\\/tm++Ymri+Ln5tHH3+zikHF5b+Rpka0R++\\\\\\\/yAYkotNMQNXEzLemBUZgPImWScKGmwUE0ZfSA6mP\\\\\\\/Gzhoa5VBAGc6YquHECdRifajulAXwE5V5oQ1l9lo99HxmMRWfMG8GSd3f\\\\\\\/nmF3Co1ompKEofac99vy8KxTXN\\\\\\\/azA+mNSU1Il70FMqT+snPV5I73lHC6Szo8mMr3kjOqoXYlttI6HJgwpib7+Qjth2EhJuO1\\\\\\\/a6ESvIuUsqAMNYohfAYpBXPv3vqeNofokJnFOGwoRNbpcoj6dep5A3uvICWeHjhLKiGy+6ER\\\\\\\/yy7v7wPWsf3p\\\\\\\/J47cpOgqhpQcr0YVG2Icc1B3JLGJ7zOwkOAz9tPZNln2ErVO9WKde4y8jclJD2DLR97e\\\\\\\/uDWaSx0U4Jj4BqcrIftxFCnMA4eDXaW94ME7OZUtNldwkvSSoOPpbtEnmC0c5+EEwSaAoVAzWIB7jP1dmdOWIrNwFXULXxg0sovZGcybWDBWGThwmhzWz0xjjj7sar4sXB8dCPJhS4lOOMdk3FVLnm06W+0xZo8CDMgCpjDuiVjFCs+wy8wF+E4+zGMZwR6t\\\\\\\/I695P0hhUrg1x08we6kXSw7S+lv8u9h3KB8XnssGtVhxH\\\\\\\/tQb2g7tGTGgVOwlf\\\\\\\/B6dE8KncOkJ4awj6D8ADEAgv+FPvbGYBhfNo2E0wNZIndqBwgtz6J\\\\\\\/gVZ7Wxp6VCGGX3PpwwgXZz2ncPkwzz5aWKo\\\\\\\/PePXmQRHjzDFVtmIlziKL+AwmKI2Mr\\\\\\\/VWeDHxZu6X43ffqjUfoX4lTBJYJv73pe72rTYGz0Fz8xtmw2CKbUsTVEakdxIIiw6G2iBdAcMam3ojk1pCuGd5tabDiwGSx4hSYSbbpaki1Jtu83tLgv17TcfgixVJq8km3\\\\\\\/syPeXlP017DM8j+gwSDOa\\\\\\\/qEjd6QB4P9xTMOHj7qiawvvcCVm2gnu39OMoIUrVwWl48HfWCqc2Gv5SkC6Ieuo8txRxhSGx2z\\\\\\\/0ontTu02suyBFpC8YwPbYoCD929fGB09AGaPvkV3E1\\\\\\\/FbTszsMZs5rqrBa24W2y9i+XObb+cQw8KkpsO\\\\\\\/GKAEDHiWgjyqken5wK5wzmJwwJUr0MQTpnh\\\\\\\/dLddR3oVkr8VsO3A8kMJQFbYu+yQY4V5ax53Opolk4115dL5Ykf6KI2IbwT3hFfS64fIIBvRRDjoxLGM1J8eKctBqU+KR3glakZEfDpkAXpAJ5BzJpXrqVhrvrol974JB7OW0fqd3bcC5TMBc9CKDTMqpJWl7\\\\\\\/FuEXdTsItivW\\\\\\\/ivCILxufqaJ\\\\\\\/t2oXHBPHzPWCyNN9WUTh6McJk\\\\\\\/+sIZbgSQsLvXzWj+jh1bWSexl8yQ6oY69u8mUJewE8k0BBXoCTyoQkpCDzz5j\\\\\\\/KDhTF1aGe1uzFYoqBiY3CImPWPY6r1bJRWu\\\\\\\/+8rQMgmB3xE+0KBX\\\\\\\/+p+7+YP\\\\\\\/8bwt6+YUwpGT6elKRhN0lJ0JjoyhYLoqaqinte3o5BtK7DyXeJep3HOrhMRx3Kq6kPe1P+JwhFDm0kVjnr9vf8Br8iQyNkW3hFHh+gEaqrMT2g\\\\\\\/4GGB1D1wRO7\\\\\\\/rziplCJTJpniNfYfk9JAymBzHZBEtJRm8zzwTVOGaQ35erCeDcOOVfDLk+KQR5HWHyLWND1VjxMkKP3mDiMZSej0QtN600gd2znuH+PAMjlvS3WBrF2JkBIVuatahYdJuurymNlCoQGwPogByqot6KqQWwirY1O2vrtWN4yz6zhlGOtnrXC6L639vRz6Bk1oadeCLPzRbbfzZ1Q6Mk5SclTjF5du0VQY9r4Uj4+H0Gsg6n20YzQyDTLr+wYJjnbOxnJA5mx1IbDGNxKD4fIpxASYBWK9MednjEkfcQMtZIsNNgl+pgjuf\\\\\\\/qE\\\\\\\/8SmHT6dHv2naW1vhjiJjesl0Yo0oyFnwCEA6y\\\\\\\/RmAOJ3JAg4Zey\\\\\\\/DWGao5ksGeoBnzS1P0h3krH1lNW9\\\\\\\/5N671XXveJbPpZyiZY6qFjVaIklFE+DaMrBDcc4dj5KU+Z7VSxq6MlyVlGneri7FmSatEtPSQRw1pijcLUgAS2i03pS3ckxx4WrOSyeX6Sr+CooQJXwltokM8PLCoTzUU14gdLvibiJtFVX+etri6gojHOdn2O1B353zsXOX7AjqxmctVMlr02In9ffC\\\\\\\/4\\\\\\\/P\\\\\\\/3l3bbAfnyKtdrbLoQUCP7rznA2B+FxTVn7kuqPD62RIEyRGDSr2pmX6wYMFTqZZ1uo0I2B63ZFA+FY9YhHhl4bXgvELB\\\\\\\/FJbkguDJMwBtfPwPusmOFDviIizy1sGdKpVR6Xm1IgN\\\\\\\/eBvMqSRv\\\\\\\/2RBTyzrM1SPOMe8G3y+oa3mEycXqJ0Fcq6Ik9lFTPvacUucDfEANRllqS0v8aDFZeEHt9WrhQn+mnulEBj9OeULTm\\\\\\\/zQLQmi8TX1D+PvH7nwAgC2I2dPm2+Q9fNrXL0vqyeIeHA0YmGQkTge\\\\\\\/7rh0xdAikE9MZXi4fcyGI1SBEhM\\\\\\\/exoP0s6dk199n2iiWuyD8Y\\\\\\\/\\\\\\\/esdvOLsF\\\\\\\/xyuHPYUX97jLLAFmdLekcpXk\\\\\\\/7B3ApM5BM9rY9sLZWtHKJv0tcrmHHxDA8qBi2MR7dXjEg0e95cirA2PMYCDsNS9xsQ9rT7\\\\\\\/NyoYEkeCy3H6XBQTc78ZNwBhV8ftemMQLljb7nKaIDpDUWT2Dc1qjei+aybbSU0Da7bx7csJgi09iLHuifEwANwXakVIeeSRWV5E0IpheCGEmmzUXo0m5axbsAi1dt+8Ny9VOo6GoI2oZmtgso991yGhv\\\\\\\/tV0XyQs6MR0RzEnHRD+trciBcqaQiXNfg3cWtsRN+lzQMZ80IJ14NvuDNVXiRgW3Hb5ctbBo4ZZVFbCTqvnHcNYWxbJ72Qy6J9HGA5CCO30sEeZtNYO+uSOYZ5oMVaeynVI5D4bOuyxAcoVy1tWNtUhTfzDiA+FfrGMHKb+TkkPei9Pv6duPVkVQQFkTj+R5rCMcrC\\\\\\\/zXqCtYn2X1q0xw3xmtYN1Gq1sFu0BGEeJyILGbLd8Xdndej9Mha9NgMUtpd5NEPry42qPT3hKcKo7wQwsQmHeO\\\\\\\/VkAJaYFdLcmK59z7Fku\\\\\\\/5k4mRI0k7YX+hvUY4vDakIFh2UT3Qq9p6eeFxiyG5j8l2RpWiPgyBP94EHGqypF\\\\\\\/4ErxT7BXemrBBZrTBd8I1fm1KEMPNRibwKLGYqBS+xfXxNuzCE\\\\\\\/9hyaFHSeuZzz2A\\\\\\\/hlmahBwk249PL3y2JOzv1HOfJfqfJXi4PNGPOml7W5SCAdOc\\\\\\\/oY5DUkrXAchxQRRsBFHFW5zfzurI8cGs3NPf4sMHttqY\\\\\\\/cS4F2QFAgcOvzUBRZqS4iH46RfcKD0TgpZYaxoNgL92og4pi43YKp9m7+OA0dXoKaNmTtoJ\\\\\\\/R\\\\\\\/ueJEuhcqLjmtwo9nvkkkwdbv2AI\\\\\\\/1a6pn9hZSQ8+ccd6eMnusTzXvxPVPVBPDJaiUtHgxppTLNgsPA7r1HPPbJaicbk32CW9E3xBerMip0NvmpQJRD3ALIaRrDxY1VoltV1y+edkyK9mBLYtn\\\\\\\/ZaPlgIaRooXPFmhvNQ5VDa\\\\\\\/zMCpuP+1o9z1ArTnGWaT3VbMq3bAnCoyQoyyoJh6txW6cGa1rlw1cdgWQ+eEw\\\\\\\/Lx\\\\\\\/qJ8k1k9R5PXGNCVzlSEpXgGA0\\\\\\\/K2QpmxVeKyW62LiivmjbnZ9b5VtEUa9haZj81Q6OgSWEkU5pJzAnw1AC408PNSwgDBb9LMJud+Gjc+yYq2l16Q1i53aqlVOWoRj8aqWGnK+KdRXOpMj4C97GsaPv2F0mWuoli7+dVjw4XpwTNl3XuSEuW7y9IZ4jOI4uRpke2JHTzqRwwzffcpym31hCS9YcQ1\\\\\\\/wao4nZYEwJEOPvSzTgDMh8n06gFWM\\\\\\\/nsygcy2JXYOKSiPOQMEUXtRYSi1MHxNwl8l7\\\\\\\/zA0gc8p59xD3139QMgublZV4Vk8QRn7fm7tSfp64yKSLC8npY29Cqg9whGXJw4fE38zLJb6jWqwl1AI60r9qzytMut0EhHMlOIy1N1gk+X7cBeASH7dbXXSrnoIusTXU96gkJeetzPui0udiCG\\\\\\\/pSbMe84CCnQWb8KPmLVIEsC8POEYjBWjkRN8MygwGFOytbRRlNIeULoE3QbvESChauQ2EROxAjLbPa\\\\\\\/KMD3kYfUrFXsXxb7eJbNBuyitsSEW5woUDm1\\\\\\\/MUSzkL7wcEPNSmWxkJG41NwxIsHPO\\\\\\\/5ztdKcnNjPpJML8Ge7YgWrR\\\\\\\/1WRRzAb7EBdQUX8VZ4G7yJ+utz3Z6bF3nu67au5Fd19Q+N51ufJ7So+s7nFd9tLIPfgurW7s3nFZkRpR4LLuDfgMc9mEsAOONvSbIFCaGHJyLioAADD6xCcxuuXPWpI7gMFxYMUy1MMY6z\\\\\\\/gC01QIRJjXp6MNkBBYKAScv8Dr9AQP+pjIZqtAfQZxBE9ju\\\\\\\/fnkjEFbJoIsdPt7ipkPqJiN6ZqXCC9Uj3BDH\\\\\\\/nrSM9ugNAS4ZvR74s1pKtFP\\\\\\\/ag4XQziNRNhBfyrQ5Pvq6eJa\\\\\\\/KxwRh3g6M3shKtUk\\\\\\\/GQchZrkgeH2H3AqRwNVoPrbRKOIVcnYv8gJT8Ex5hK5ni+ubEiS9xYFA6oB\\\\\\\/FDgpX5oRdX97AxKQlgh\\\\\\\/MhwooytufzyGmLysLdOTvWQELagBEjS8A\\\\\\\/wyjcU06mAu9jFiUTSgi\\\\\\\/l7\\\\\\\/6b6bMNxmQqd0\\\\\\\/XMo4NJy3L+y4wCVW+5qJwABlalgLnBm4Rf1gc4Itl2UFMcqdnF5mxp1q5W9M2\\\\\\\/Jyk2PQD831ozoqHtEUrI2wNlBiSr++b7TnPvS73MicBsVlQPNmIYcFxYULiHfcT0vM+mQf\\\\\\\/AEzqX3qW9rAVp3nLLryFu7beJ9Ea8CS4zLGgQspZA6NYj\\\\\\\/kj6KPHrv2rTHScn0L\\\\\\\/\\\\\\\/XfmcbAy9cVjIwpXSxatNuWl9a3dT5ZA66AG1764CFfjd\\\\\\\/km3AZeSExRCeGCitISHxf4TVPew8avTfbekBr5lFUuqIdLx8s514\\\\\\\/iHFDpJisNg4GnimOLCeLhXsZNe1avAHMufTBU65RGT8soQtjWZ6ZHJNOrIhThR4nbJYkl0iys17SicWUGRTceDuD8AS9vOyOeEoMBJvePYbXFCCge80q2lSpOeH2NYhLVt3Xt1UIhD87Rbv4z76tj4tl4JzCLXBbftTPHp\\\\\\\/v5yu82itJ97Oy9dktdclhPtmcemAK3LyOobqocVLfZc7QCOMC6nu+aexD1DO9esceLn1vxvILQUCwBSqh3+W24d3E7XXPA+Bx+E6NnQ1pLozJ9xD7We6fiHDvSWMSz4rPiPfNFTW1daHwMr0WI7iv+jrMAPEcUlJcO0F72lRwIw3ZL8FPamB2U8eW6ONJ6vRLOdTGJSD+D8wvC4vxdMh8T8yvSgOiYUlPIwOJm\\\\\\\/j7A8c4IS2c4GVxlmrcNcHMW7tEi4w98ZLmo9\\\\\\\/p7ld4C8vBRNnDIozei0CtSWER+9+VioGqvO46hmXWjPdavDnwDOX51WmA7LCpBn2exiWVlqiQfsURdxvXnfyed0tbmaEsZ9jpQfuG2733zEDnQ1aGRlQHkXPBcuXIQIEccUm\\\\\\\/vYRmftnvrGP0E7j1KCU+f\\\\\\\/Q5kMyEyQpDJFdaBEzlzBrl2Y0UwPcbkQiVQq6fzwxI65SGkuzkmvPmNaAG6ZYsO2L9ep+9SgU0jN\\\\\\\/rz+Ic39M5pVoK9YKyGjYt+HOhPkV5IXXokK1ptV9MFNB3a+EUR2n3nJsZPP9wDmpBLvwNSG8DTHtaD1qp+ZTLTYpZEBbk+NmVX+NOZZS3gfJqaL97TpZUmXihG2nGiJa5eLSguhJm48CGIg3F8AnIprF\\\\\\\/x6uoQpxLadR2AypKhQQpFWvXZ6oHOHyRe8lA0f6K2gNJttrWzFES0n7NBhSiv4yQ4DFydjBYDvZM\\\\\\\/BrQucvyLe2TDWC45BVPTUbLLKdvG5wqM38wrgytJKlWlhKxhcjY1f6P\\\\\\\/T6Q9h1yv2eupZx1gilPA+I3hI2z00KJQ0W8oq3m9EenPGLdChqt9glqtVwkr\\\\\\\/52fR8pJus2cC8LT01nivy0nC87xHp6UMOjw7jrx1nxecY+hwHtFwh4FwG8vHiMlmjYQYMexUwilfNwtTsxadzJjXFhKktPbsxHQ9JLWFrbRNSSas4I1spoP6yu4bBilarzMyDP4\\\\\\\/JbbK569Wx9GJVgY4xj6VJEdF+XxRpq0QOI9Amo5o\\\\\\\/ZYoTW\\\\\\\/WBIT5pYZ98QtvBcTZhPu1m+1YUA894STx5ydy\\\\\\\/X9wLNyhxkZ7bNExKPr5NTBBeEMkSc69W09B9NQJKe2pp7dPbhmH7gUIohEoBMYDpDhirHHyeHnOvsLlDP0NVIhSykci7+3QxjekJWcbBD06IZMjOwkKcTDbTSDnHW9ZLUudt5+oybuNovh1VdMAvj\\\\\\\/5qRCU049nw94MQLvx3AuSbThCSCtuzxIHgl6FgmCqQfXr0+uzkZG9BNF9oeC0YDa3rucYXcabg9Joie6v4G9Ka4wXbjbotJRtsZDZHKyDmciGTPCilQxpgqGMdBbo5DNbGlahTp\\\\\\\/N6ijywjUEMmvgc8Mbr+BSN0itjEuTk9mdGG+gQKD6AdYsSKJDJO\\\\\\\/4GT0J0KzEKUREOXPdOypm\\\\\\\/+0w\\\\\\\/LCaphVCKz+FCx8gB6hBCkE51kC\\\\\\\/lEyae1r7YXhr1qbbuV5m9cpSADep\\\\\\\/rByHHFnMN6bCxP5YMJRhA5qmS29exovWmkoA7U7e4piNq2oipQvQqOXiZpwXBQP2HECaGFlV5rNQhZfGCm2DOPoagkgLfSXd8pnvH7VgVS2EdOEUtPgRpADYQ63AYJLPaIaMcMD0v\\\\\\\/L5VNEOoWDz3Hjd2Qp02\\\\\\\/Ujc77qQ6IlDjBy8ylW+wcQAPZrGNqeyyKq4rgmIYZZCyXIk67GCrYcGWU18gQ6schQgLXMdWm5cglcw\\\\\\\/Affb371DDpBCv8eeg1zmYJMcAIJcFpkAoxigDr4c9eDFWgQ8oMlpIxeUA5bMrUQw0WgBp16aPLV4vhFwBzmKkre8j8yCLP1paIU\\\\\\\/YhrKTqG+2ncgU1w\\\\\\\/7JrdXubL984Lc7Pn62bYuWcH0i3MTYbNMQfib3SV4XYWhfW+4Y0AVUVulvd1pn53eubMhlG7iU3xeIqw9qbliEYuQUHs3hVxYa0SsYhTj+I99Brn4IqDS4dgRxt\\\\\\\/o9ha3WsNMfDhfJUKLZTpIYt9fUHOUwn+lXcKwYuT4FGFGoRqWzEB3eOZilzhf+liSrcQLgsZUIROPXGM0XlFXHMah2dVqpVzOCYCThjZKMRtaI6Bm6I0j4OxCDBSN\\\\\\\/nSXsUd+xvWPh579eXQPno5LIHDtBZ8eRtqMhn4WaJlimdXntXvfHWuh80ExV0gXkoCAq8FATGYz\\\\\\\/ZqBOGyt9E10bc+0ZcKI3Hx0PZkzwZCZHjBzF62t12ZQ8oS\\\\\\\/0+3lhQ5x0Y\\\\\\\/i\\\\\\\/KoWxw5QSx66uHcMUFFQLQrryHoOsrzJWU5461w6Ub9YbnEirBJaDvXSAoeKRDtqjL9NtGR0PWGD7j+yItKH+ckjyPnfxokzkUDIOalnlf6wQuOm6Ef3bmT7UdH4OnvZLcnfL0SfjAVZWE8jCvJZyqaB1V3\\\\\\\/a+IISW2RjbepjYL4qiOU1GgvwusE\\\\\\\/+UlzHfrlh+IQqTpsZKqD0lNjXzRbnbrNoOeLtgn47nvHfXqAYRzq\\\\\\\/ohrDjDbn7LFdG\\\\\\\/L+HvowaB2LLQ9dm3P40t9TmTav5GRq6HSlgad\\\\\\\/ikGeWUOwKDuV+4BJ8DFdSBa4qaSs4zcGlqB25sD1lWI4w+G6z4+bwex05glwhiZlUa9EHl49wuxOb6vyW\\\\\\\/iNq3\\\\\\\/R3i5zuJepf5AApOawhJEc12Q6YV+XRDAdBD+0mzs98+4PBmAaY\\\\\\\/piVLdi8I5Id2b13STlmDc8SiqI2xJIq0L2hjHUamsd1yEamvO8SD+a1MFVTdgBYkImeiMhF\\\\\\\/CvohmC4zId8HKEj\\\\\\\/frOqJ8wDbsKuQra1SC\\\\\\\/wF6k1pirdIE\\\\\\\/PpSjroE2wwbY4a\\\\\\\/Hh77g72GaR56kn0P\\\\\\\/YPjduGFaziKs1hBP1zZtVqR+L0XQTCCTu0BG6U7usKTLD7heHswgaWJvGOAtLX0jGPGCivdYONZCAbT39fZBUsix9Zv5FtoZ8iIAHMAzlq0wORkontlPzojzhv2sWFkoDTwtQvDmlNJvJEYmYyaoaqnRzGtX4MZXvGWvm7EeDW1aB0c9cAkeA00n2Gnb9DoQOibfSiYkMWt8n51fkT5KYolvMKw6DrcLaP6tulEQya7N9fprPWWaxbzbY7bxyYOtVpjcdQ5jAxxnJc3YgI1RAynDyZ3XgUF4L3TttknI\\\\\\\/yHKIeeeDk9BRehuypUc7AelgC1vz0yneg1mlGB4X00q7Px5rTH8bCnZJn08MBr7PA2xMtER7WrysT7IkIInclrp2IEO5hkvrggkExYJDjRRgZFWrkqO9W7Dlk1ezIJgUIcI2z8ls1SjJdTi1UTLWETK2YLQ6y2NX1BEdS2MnOaldvxsPkiXBmsmeuZci1tr5ZHXhHPXmkAIQ2zV1ERJealx4tuSpKblpjwpSz\\\\\\\/HP6d73n8cOtQySwvx038UAv8ocZJfaVj58mvCGDtKPfxs1nydKiuCtr8y5tDOBe\\\\\\\/SeiTvl7PN3\\\\\\\/1fnyNO2kJoYYsxgdfpH2li43UqpRjH\\\\\\\/FZFMywQbThNktQ0MPZUI7yB20WXh29ie03DWqXx+kFZkv1C4vLmLMJKGWOFkQtwqLfb2jEDcYm\\\\\\\/knFKXJx+FwUz2jESqxWe08s8r8NyiSVq61OQpHjMNd4VCaGvt22FnaUCKlaju0KL2xKpl7BBMI5AXI6GoLW+L4T\\\\\\\/ypnpvYWtQ\\\\\\\/OPZbTyZCXdkBqip1V979ExDFf+64zorLirNI9sEhe4GcsLdmdFUWwFVwyFRcuOK9VAvltiEqm4BCzMjDBZu8jELHVRgdfLUqanTigeywLEjorB78Hhf8YEq6lQ3gUmkMhC8iTeDv6ck9NSnRswTiq8EWDXgeG4hUsJeJZVN1cj3uP62YHaeka9ylViLLQmygQ+IzVbOSq72RO2RS7SMaeFKlWP6YUNDXc1I7WRe3WDmBF6UgHubq0+NtF9lX0E2JoA+ek+cJZGmVDKGUquIKrULY5+h1aRBqhoWmvTvMd4oWiDEaBMd2PWIvfFgkiFXwG1q\\\\\\\/v49awKIGFW+x\\\\\\\/Zj\\\\\\\/L+q0LafVkRFjHgSbtVHPdIZlUNXnVn5Jj+RFVJ0uTBU3pTfwTPMgpthxoiCHpFxA9MZA0Ih72YHv3LT85sGioKGjFZ9CtY6FR9lHOL58S6Zt2aUVAA92zqJ95Lem9IteYAUzC72z1fqLD7Fj8Gv3GrxBWrFdN39L8G7WxJUIBEVZU8xXOlsr3FHqpvWMQgu474rq+P1uFCOnCFX1d69pgaXFaSauDRiSvRlOLVf574cIHCKqqiYEIltxHoIQ7Va6ElQkx0GvUfEAkzEa5fMOoFRepklDq7NnGuQkrT0mLG2q07\\\\\\\/pAbEYu5FkFU1FnoEVZ47K\\\\\\\/qC0EJITiDpvG+oRRyMcGFUKDiOmXzDkwRNKwDVOl\\\\\\\/7k\\\\\\\/c4BCY5qotIhYu119FJR1vY5vNJ2h7OSHCpt1n7VYx6UVXGvLTzTvMav1yChgUl0EveHXCEaBGkaazqAhItRNTvw+cxJJb\\\\\\\/wNBdqfjQ0Y91FrmeqqPlaZgr9MysusUJxk6p5vYQEgRHMXVY7mMdq+KzTVQ9f8yxjpH3u1LMqeGUTc2b0orNJsPp09NPb85zN+9VSJAuc2ItypYUseZmg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022b0GPW+8f6TBkOEJcfOBxSg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bdf9c077303a37a7974f3bb80f3360a202e2fdedc36c5217574d61bfece85a77\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hcZeK3sOTecFdk6yx+U8\\\\\\\/ZFGOJ9PLGnlHZ5rfhPw2d9h+oGudOoiDAFTgA\\\\\\\/i2BfJ1CxmKHKmD1RhzwnF6hCCsSR7wdVTvDYsxMW6A66Lf\\\\\\\/3i97Cp1o92t2r3oYn576NWzW60PWxn5Z+qFEPB2KPlVTJEHZa4CSDnSuABEwUHf7t4QB1kYZtQ3e8ijK+oMrG\\\\\\\/x\\\\\\\/JOHHHwooszhjPQN6hEWGTHDo0AcP2E\\\\\\\/IpF+r1hQ58P3U1JWiBqqVdb3virhjZ1ognWSEqS6XkvX+94Ov0NFidYNiMyplN4PqD91i8wkDcjRBjRun7S\\\\\\\/aHSo02i4N+v31FPCg8UyH8uF8\\\\\\\/\\\\\\\/WHrB0McP0495hQXZdzO2I6ziBmngIvs0zoN9TlusRN1fYj3zfHSgNm1CiNBbY\\\\\\\/oz8IBaOBLATEc4L4o8bmDQFCy9EEz7SG3qVWQrLoxM9b2efTj40oHzpl7JuF9vrBh6U3JwD5WjCoB\\\\\\\/t5KozmJq+\\\\\\\/7w39c72MmMKZHyKkQWBRERuFtCNweVTutlC74COx2KISEPZZiFYKEyhAzwTkjgZTlxTaC+mpFd2W0XI2VIOc75HepFnFLDahFTOMR7lYwLt1TWyL73wSmTDhoi\\\\\\\/eaMoS8phgWHhE4IbWIpInW5BYzL8UxulkXBxjh5RB6fXvMoGT6xwHihXU7txEiz\\\\\\\/ecA4UvGPN7mmdvtVAwO9PX21vMJGM4IIBSVcz2gpI718eDDS611LeyqRckSDM2IVha7wbt9OgbD+lSvfNsalCYB9PsY32HaX43a9qi\\\\\\\/RSv\\\\\\\/Bho4YNCca9KAhTvpkzlPvWBFSfBoyxDcDMRIGq\\\\\\\/5V4vP9NnjkNiadnKXIb1kqAWCKlqa8MMaHNIc4B3VjDuVDe1Gro\\\\\\\/frdLvSUGFgrGqBuvaSGW\\\\\\\/35Dp+ibXv5AAbPovAkHh0PlcjKvWyqktUYBM3TWG4cZUT4pmPq4xH0MFGlf5+ET8p3+nTJ6YKALWRFXqyA+UGsOLnzBYYMAW3BBXIBXD5iMcCcIf6LUzZ8ypYopizgPbiJ3Mpx\\\\\\\/H+0vZQYBLFb8xgnrnIzU3GW2FpTwoEnXfVAFq7MxfTEdDOA96F5TRxn2N07OhXswjIUlLE6tG9c9OUnaiYpEycKfioS8i8oVI0zDnQFhAY7a+5PNzBq96LHVqi\\\\\\\/QriYhDHMFGm0k+D7avw9nthoBXTpMj9V87dFqZ8sz1d4NU+Tr7jAHFCthSgmde9J1By8geK\\\\\\\/NG52V+REretZMRCNnoytn+fuC09fshV2TQZXPF76nIMMTL5PYFCmF8puD6TjI9e+E5WoyX8UQwz4w0DSLCxPww5Gjil12PCvVUdl7lV6XRwzCBlbLZF0BVvVNt5vFkGMcCzYPMnTMd9I3IFY60+6tpZAnWjB1dZWhjCuMNKbf6QAGjiBop5IF4WaUW0Rh9v\\\\\\\/w9fJO6BekYPUXjVqN\\\\\\\/a8b1EWbF71veK9aq8ia\\\\\\\/t\\\\\\\/j+PxgWbNYPTvxZy0+znCAq6+20Ckr6D2e6UivlcMyIA2Zbub\\\\\\\/BElo9wKBiDt9LmUE8VM9LG8Ctvx1QdbEKMKeCmSQr3md2dseaAXqjCSEMgLoMvNY1Fva5VQ4o6ubVZkrnRwY62QMDTfyI9w0YS9Ept9rxzFu7EIZVIzbPy4GNG3mwT2J1dFF\\\\\\\/IavLuXh6eOieEp1G1d5uyo7fG\\\\\\\/f\\\\\\\/0FWoDb9xKhjlc66KTNTZ3phSzizbWVwW8DXl3QfkL7iyHcnf6fpmMM4t2u9bUD3PzrKHHOGgWI8IzFVve\\\\\\\/RqvLZx7ADtt51VYAhVMtPmWoiKbweQDGaFoGpz0H6byucBYEiS4O7BtsiscFpJpJMrha\\\\\\\/A6Gd5qu\\\\\\\/G9AX08n+ZjdV4SYF3xvGlwnQ\\\\\\\/ogu9ulm6a+dgFiBECADlCmj6Zpbwyv+mK8RTCDpG+G9j9uYH7lD1IkHibjPa2vqGaKf1pZjuAoFN5AxaT+nyeQV01R\\\\\\\/Bs5pr99um6kCX18d32wYrIDrZMBKVJErLlDfIywi4uUjlWViRzqN3qIkiqDQPXd9QTw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022H5PkBQ6vzgkVoWuvyoke1w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a669c034e03c7b5a9cd8b4391c0a26e15f8020e6b3e002f6b89298057fbdc5d5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460245 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/1" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-W7eRI76tlsckKxnSRw6C1HPg_xGac6NCqVFfue1vNOM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hF2h62lyafESYPpy40Ey1zH3-zRiu60jxuD7T09QaWM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/12/2923" title="G3 Reviewer Index 2015" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/15" title="Comparative Genomics of an Emerging Amphibian Virus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460245" id="node460245" data-pisa="ggg;6/1/1" data-pisa-master="ggg;g3.115.022897" data-apath="/ggg/6/1/1.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction Accounting for Residual Heteroskedasticity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhining Ou</span>, <span class="highwire-citation-author" data-delta="1">Robert J. Tempelman</span>, <span class="highwire-citation-author" data-delta="2">Juan P. Steibel</span>, <span class="highwire-citation-author" data-delta="3">Catherine W. Ernst</span>, <span class="highwire-citation-author" data-delta="4">Ronald O. Bates</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-3902-1533" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nora M. Bello</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1-13;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022897 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460245" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhining Ou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhining%2BOu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ou%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhining%2BOu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert J. Tempelman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BTempelman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tempelman%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BTempelman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juan P. Steibel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Fisheries and Wildlife, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BP.%2BSteibel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steibel%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BP.%2BSteibel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Catherine W. Ernst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BW.%2BErnst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ernst%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BW.%2BErnst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ronald O. Bates </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronald%2BO.%2BBates%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bates%20RO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonald%2BO.%2BBates%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nora M. Bello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nora%2BM.%2BBello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bello%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANora%2BM.%2BBello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3902-1533" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nora M. Bello</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nbello@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">nbello@ksu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/1/1" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460245" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460245/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/1.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460245" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460245/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/1.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460245" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460245/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/1.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460245" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460245/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/1/1.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460245" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460245/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Whole-genome prediction (WGP) models that use single-nucleotide polymorphism marker information to predict genetic merit of animals and plants typically assume homogeneous residual variance. However, variability is often heterogeneous across agricultural production systems and may subsequently bias WGP-based inferences. This study extends classical WGP models based on normality, heavy-tailed specifications and variable selection to explicitly account for environmentally-driven residual heteroskedasticity under a hierarchical Bayesian mixed-models framework. WGP models assuming homogeneous or heterogeneous residual variances were fitted to training data generated under simulation scenarios reflecting a gradient of increasing heteroskedasticity. Model fit was based on pseudo-Bayes factors and also on prediction accuracy of genomic breeding values computed on a validation data subset one generation removed from the simulated training dataset. Homogeneous <em>vs.</em> heterogeneous residual variance WGP models were also fitted to two quantitative traits, namely 45-min postmortem carcass temperature and loin muscle pH, recorded in a swine resource population dataset prescreened for high and mild residual heteroskedasticity, respectively. Fit of competing WGP models was compared using pseudo-Bayes factors. Predictive ability, defined as the correlation between predicted and observed phenotypes in validation sets of a five-fold cross-validation was also computed. Heteroskedastic error WGP models showed improved model fit and enhanced prediction accuracy compared to homoskedastic error WGP models although the magnitude of the improvement was small (less than two percentage points net gain in prediction accuracy). Nevertheless, accounting for residual heteroskedasticity did improve accuracy of selection, especially on individuals of extreme genetic merit.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/whole-genome-prediction" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-prediction" rel="nofollow">whole-genome prediction</a></li><li class="kwd"><a href="/keyword/heteroskedastic-errors" class="hw-term hw-article-keyword hw-article-keyword-heteroskedastic-errors" rel="nofollow">heteroskedastic errors</a></li><li class="kwd"><a href="/keyword/genomic-breeding-values" class="hw-term hw-article-keyword hw-article-keyword-genomic-breeding-values" rel="nofollow">genomic breeding values</a></li><li class="kwd"><a href="/keyword/hierarchical-bayesian-model" class="hw-term hw-article-keyword hw-article-keyword-hierarchical-bayesian-model" rel="nofollow">hierarchical Bayesian model</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">genPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-3">Use of whole-genome prediction (WGP) models to predict individual genetic merit in complex traits is being increasingly utilized in modern animal, plant and human genetics. By incorporating genotypic information from single-nucleotide polymorphism (SNP) markers, WGP models can enhance accuracies on genetic merit prediction compared to the use of pedigree information alone (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">de los Campos <em>et al.</em> 2013a</a>). Currently popular WGP models include ridge-regression best linear unbiased prediction (RR-BLUP), BayesA and BayesB, all proposed by <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Meuwissen <em>et al.</em> (2001)</a>, and subsequently modified or extended to a wide array of models collectively dubbed as Bayesian alphabet models (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Habier <em>et al.</em> 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gianola 2013</a>). Typically, these Bayesian models specify either heavy-tailed distributions (<em>i.e.</em>, BayesA), variable selection (BayesCπ), or both (BayesB) on the distribution of the SNP effects.</p><p id="p-4">An often underappreciated, though pervasive, assumption underlying classical WGP models across the “Bayesian alphabet” (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gianola <em>et al.</em> 2009</a>) is that of homogeneous residual variances, often referred to as residual homoskedasticity. Yet, heterogeneity of residual variances across environments, or residual heteroskedasticity, is a well-documented phenomenon in livestock production systems (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cardoso <em>et al.</em> 2005</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">2007</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kizilkaya and Tempelman 2005</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bello <em>et al.</em> 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cernicchiaro <em>et al.</em> 2013</a>), thereby raising concern about the implications of the residual homoskedasticity assumption almost universally assumed in current WGP models (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gianola and Rosa 2015</a>). Indeed, the use of heteroskedastic models for genetic evaluations dates back to work by <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Foulley and Gianola (1996)</a>, who first modeled the logarithm of residual variances as a linear function of fixed effects. <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">SanCristobal-Gaudy <em>et al.</em> (2001)</a> presented further extensions to incorporate random effects, including correlated genetic effects, followed by a unifying framework for the structural modeling of heterogeneous variances proposed by <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Kizilkaya and Tempelman (2005)</a>. With the advent of genomic selection, <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Yang <em>et al.</em> (2011)</a> was among the first to propose a WGP model with heterogeneous residual variances for livestock populations, though only a genetic component was specified on both mean and residual variance.</p><p id="p-5">Environmentally-driven heteroskedasticity has been shown to have practical implications for the prediction of genetic merit. <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Hill (1984)</a> demonstrated that proportionally more individuals were likely to be selected from more variable groups if substantial heteroskedasticity was ignored using homoskedastic error models, especially if selection pressure was intense. Early attempts to remedy this problem included the preadjustment of phenotypes, <em>e.g.</em>, by centering and scaling (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Hill 1984</a>). More modern approaches include explicit specification and modeling of sources of heteroskedasticity. For instance, <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Kizilkaya and Tempelman (2005)</a> showed improved precision in estimated sire genetic merit for a birth weight trait when residual variance was specified as a function of sex and herds, finding that estimates of residual variances differed by as much as 20 times across herds.</p><p id="p-6">The objectives of this study were 1) to extend classical parametric WGP models, specifically RR-BLUP, BayesA, BayesB and BayesCπ, to explicitly account for residual heteroskedasticity, and 2) to assess potential gains in prediction accuracy by explicit modeling of residual variances as a function of various environmental or management factors in simulated and actual livestock performance data.</p><p id="p-7">We first introduce extensions to classical WGP models to accommodate heteroskedasticity, including a delineation of the criteria used for model performance. We then describe and present comparisons based on a simulation study and also an application to carcass traits data from pigs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Classical WGP models</h3><p id="p-8">The classical base WGP model expresses the phenotype <em>y<sub>i</sub></em> of an individual <em>i</em> (<em>i</em> <em>=</em> 1, 2,…,<em>n)</em> as a linear regression function of SNP marker effects, as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <strong>β</strong> is a vector of unknown fixed effects connected to the phenotypes via a known design vector <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-2.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-3.gif"/></noscript></span></span> is the unknown random effect of SNP marker <em>j</em> <em>=</em> 1, 2,…, <em>p</em> connected to <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-4.gif"/></noscript></span></span> via a known genotype <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-5.gif"/></noscript></span></span> coded as 0, 1 or 2 to represent the dosages of the minor allele, and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-6.gif"/></noscript></span></span> is the residual for animal <em>i</em>. Most current WGP models assume <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-7.gif"/></noscript></span></span>.</p><p id="p-9">Differences between WGP models RR-BLUP, BayesA, BayesB and BayesCπ are based upon the specification of the distribution of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-8.gif"/></noscript></span></span>. For RR-BLUP, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-9.gif"/></noscript></span></span>, whereas for BayesA <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-10.gif"/></noscript></span></span>, <em>i.e.</em>, independently and identically distributed scaled Student-<em>t</em> with common degrees of freedom <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-11.gif"/></noscript></span></span> and scale parameter <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-12.gif"/></noscript></span></span>. This prior specification on BayesA is marginally equivalent to <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-13.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-14.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-15.gif"/></noscript></span></span> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Yang and Tempelman 2012</a>). BayesB further extends BayesA and specifies the distribution of <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-16.gif"/></noscript></span></span> as a mixture of <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-17.gif"/></noscript></span></span> with probability <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-18.gif"/></noscript></span></span>and a point mass at 0 with probability <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-19.gif"/></noscript></span></span> (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Meuwissen <em>et al.</em> 2001</a>). BayesCπ is a particular special case of BayesB with <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-20.gif"/></noscript></span></span>such that the nonzero component of the mixture is <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-21.gif"/></noscript></span></span> (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Habier <em>et al.</em> 2011</a>).</p></div><div id="sec-3" class="subsection"><h3>Heteroskedastic extension of WGP models</h3><p id="p-10">Following <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Kizilkaya and Tempelman (2005)</a>, we extend WGP models to flexibly model the residual variance <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-22.gif"/></noscript></span></span> as a multiplicative function of both systematic and nonsystematic environmental components, thereby explicitly accounting for heteroskedasticity. Expressed in the natural logarithmic scale, this is equivalent to writing the following<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-23.gif"/></noscript></span> ,<span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-24.gif"/></noscript></span></span> is the residual variance corresponding to the environmental or management circumstances for individual <em>i</em>; <strong>τ</strong> is a <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-25.gif"/></noscript></span></span> vector of unknown fixed effect parameters connected to <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-26.gif"/></noscript></span></span> via a known design vector <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-27.gif"/></noscript></span></span> ; and similarly, <strong>v</strong> is a <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-28.gif"/></noscript></span></span> vector of unknown random effects connected to <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-29.gif"/></noscript></span></span>via a known design vector <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-30.gif"/></noscript></span></span>. Random effects on the residual variance may include environmental effects (<em>i.e.</em>, contemporary groups), genetic effects or both. <em>A priori</em>, elements of <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-31.gif"/></noscript></span></span> can be assumed independently distributed as inverted gamma <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-32.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-33.gif"/></noscript></span></span>&gt;2 such that <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-34.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-35.gif"/></noscript></span></span>. As a result, variation across <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-36.gif"/></noscript></span></span> can be characterized by defining a coefficient of variation <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-37.gif"/></noscript></span></span>. So specified, the magnitude of the heteroskedasticity across levels of the random effect factor <strong>v</strong> diminishes (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-38.gif"/></noscript></span></span>) with larger values of <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-39.gif"/></noscript></span></span>(<em>i.e.</em>, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-40.gif"/></noscript></span></span>) (<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Kizilkaya and Tempelman 2005</a>).</p></div><div id="sec-4" class="subsection"><h3>Prior specifications</h3><p id="p-11">We specify a flat prior on <strong>β</strong> such that <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-41.gif"/></noscript></span></span>constant. Here,<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-42.gif"/></noscript></span></span> in the homoskedastic error model, as well as elements of <strong>τ</strong> in <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a>, were specified with noninformative priors <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-43.gif"/></noscript></span></span> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gelman 2006</a>). The hyperparameter <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-44.gif"/></noscript></span></span> was assigned the vague, though proper, prior density <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-45.gif"/></noscript></span></span>, which is commonly used for strictly positive parameters (<a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">Kizilkaya and Tempelman 2005</a>). As previously shown by <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albert (1988)</a>, this prior defines a uniform prior density <em>U</em>(0,1) on the transformed variable <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-46.gif"/></noscript></span></span>. Then, by change of variables, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-47.gif"/></noscript></span></span> where <em>f</em> denotes the probability density function.</p><p id="p-12">For RR-BLUP and BayesCπ, we specify <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-48.gif"/></noscript></span></span> whereas for BayesA and BayesB, we specify <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-49.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-50.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-51.gif"/></noscript></span></span>. Finally, for BayesB and BayesCπ, π is assigned a Beta(10,1) prior to reflect a relatively weak assumption that most markers have null effects for any given trait.</p></div><div id="sec-5" class="subsection"><h3>Simulation study</h3><p id="p-13">We compared the performance of classical WGP models, namely RR-BLUP, BayesA, BayesB and BayesCπ, to that of their heteroskedastic error counterparts using a simulation study.</p><p id="p-14">Ten data set replicates were each generated from base populations of 150 unrelated individuals subjected to random mating for 6000 generations. Population size was kept constant until generation 6000, after which it was expanded 10 times to 1500 individuals. The genome was composed of three chromosomes, each of length 1 Morgan, and each containing a total of 10,000 equally-spaced monomorphic loci. The number of crossover events per meiosis was simulated from a Poisson distribution with mean 1 and the location of crossover was assumed uniformly distributed in a chromosome. The mutation rate for all loci was specified to be <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-52.gif"/></noscript></span></span>per locus per generation and to be recurrent so as to ensure biallelic loci.</p><p id="p-15">In Generation 6001, loci with minor allele frequency (MAF) &lt; 0.1 or loci that failed to meet Hardy-Weinberg equilibrium based on an exact test (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wigginton <em>et al.</em> 2005</a>) at a significant level of 0.0001 were discarded. For each dataset replicate, 60 loci were randomly selected to serve as quantitative trait loci (QTL), and an additional 3000 different loci were randomly selected to serve as SNP markers. For each of the 60 QTL, an allelic substitution effect <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-53.gif"/></noscript></span></span>(<em>k</em> = 1, 2,…, 60) was drawn from a <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-54.gif"/></noscript></span></span>(0,0.005), <em>i.e.</em>, a Student-<em>t</em> distribution with 0 mean and a scale of 0.005 based on five degrees of freedom. Our choice of a heavy-tailed distribution for the QTL effects is consistent with current notions of the genetic architecture of quantitative traits in livestock population, by which traits seem to be controlled by many genes of small effect and few of large effects (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hayes and Goddard 2001</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Goddard <em>et al.</em> 2009</a>). The total additive genetic variance <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-55.gif"/></noscript></span></span> was constructed from the weighted sum of genetic variances across the QTL effects, namely <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-56.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-57.gif"/></noscript></span></span> is the MAF at QTL <em>k</em>. The true breeding value (TBV) for an individual <em>i</em> was obtained as the aggregated allelic substitution effects <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-58.gif"/></noscript></span></span>over the selected 60 QTL loci, each weighted by its corresponding allelic dosage <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-59.gif"/></noscript></span></span>, such that <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-60.gif"/></noscript></span></span>. Trait heritability was set at <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-61.gif"/></noscript></span></span>.</p><p id="p-16">Within each data replicate, we considered five different simulation scenarios reflecting various degrees of residual heteroskedasticity. That is, the replicated datasets described in the previous paragraph were used as blocking factors to compare scenarios across a heteroskedastic error gradient. Simulation scenarios included the case of homoskedastic residuals whereby <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-62.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-63.gif"/></noscript></span></span> for all <em>l</em> =1, 2,…, 50 levels of a random effects factor, such that <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-64.gif"/></noscript></span></span>. In this study, specification of <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-65.gif"/></noscript></span></span>represents the homoskedastic error scenario, as <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-66.gif"/></noscript></span></span>. In turn, other scenarios were defined by increasing levels of residual heteroskedasticity; <em>i.e.</em>, <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-67.gif"/></noscript></span></span> = 50, 12, 5, and 3, such that the standard deviations <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-68.gif"/></noscript></span></span> of the relative variances (<span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-69.gif"/></noscript></span></span>) across these random effects were <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-70.gif"/></noscript></span></span>, respectively. In addition, all heteroskedastic error scenarios (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-71.gif"/></noscript></span></span>= 50, 12, 5, and 3) further incorporated systematic sources of heterogeneity whereby <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-72.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-73.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-74.gif"/></noscript></span></span>is a “fixed” reference residual variance. For data generation, the residual <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-75.gif"/></noscript></span></span> was sampled from <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-76.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-77.gif"/></noscript></span></span>was obtained as a function of <strong>τ</strong> and <strong>v</strong>, as described in <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a>. The phenotypic observation for individual <em>i</em> was generated as <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-78.gif"/></noscript></span></span>, with μ = 3 set as a common mean for all observations. Observations from Generation 6001 were used as a training set to fit the competing WGP models and to estimate SNP effects. For each simulated dataset, individuals from Generation 6001 were randomly mated to produce Generation 6002 consisting of additional 1500 animals. Genotypes and TBV from individuals in Generation 6002 were generated to be used for validation in the simulation study. The average level of linkage disequilibrium (LD) between adjacent markers in the simulation study ranged between 0.23 to 0.25 across all replicated datasets, to represent current livestock populations (<a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Calus <em>et al.</em> 2008</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Yang <em>et al.</em> 2011</a>).</p><p id="p-17">Each replicated dataset was fitted using homoskedastic and heteroskedastic error versions of the selected WGP models, namely RR-BLUP, BayesA, BayesB and BayesCπ. Programming code needed to implement these models is available in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022897/-/DC1/FileS1.zip">File S1</a>. Across models, Markov Chain Monte Carlo (MCMC) was implemented with burn-in lengths of 10,000 to 35,000, followed by subsequent saving of the next 140,000 to 480,000 cycles, depending on the WGP models and diagnostics as described subsequently.</p></div><div id="sec-6" class="subsection"><h3>Application to MSU swine resource population data</h3><p id="p-18">Data corresponding to a three-generation Duroc × Pietrain swine resource population developed at Michigan State University (MSU) was used in this study. A detailed description of the dataset is available in <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Edwards <em>et al.</em> (2008a</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2008b</a>). Briefly, a total of 19 F<sub>0</sub>, 55 F<sub>1</sub> and 928 F<sub>2</sub> pigs were included in the pedigree. All F<sub>0</sub> and F<sub>1</sub> animals as well as 336 F<sub>2</sub> animals were genotyped using the commercial Illumina PorcineSNP60 beadchip (GeneSeek a Neogen Co., Lincoln, NE) panel with a total of 62,163 SNP markers (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Gualdron Duarte <em>et al.</em> 2014</a>). Markers with more than 10% missing data, unknown physical positions, or with MAF &lt; 0.01 were removed from further analyses. Quality control procedures followed those described in <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Badke <em>et al.</em> (2012)</a>. Genotypes for the remaining 592 F<sub>2</sub> animals were obtained using a low-density panel of 9K tagSNP set referred to as the GeneSeek Genomic Profiler for Porcine LD (GGP-Porcine LD, GeneSeek a Neogen Company) consisting of a subset of the PorcineSNP60 panel. The F<sub>2</sub> animals genotyped with the 9K low density panel were imputed to 60K with imputation accuracy of approximately 0.99, as previously described (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Gualdron Duarte <em>et al.</em> 2013</a>). From the 60K SNP, a subset of 6210 markers was selected for this study. The selected SNP subset matched the panel of 10K tagSNP previously described by <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Badke <em>et al.</em> (2014)</a>. Phenotypes corresponding to 29 growth traits and 25 carcass and meat quality traits were obtained for F<sub>2</sub> animals, as described by <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Edwards <em>et al.</em> (2008a</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">2008b</a>). Traits were subjected to preliminary screening for heterogeneous residual variances using standard linear mixed models and approximately 80% of the traits showed some degree of residual heteroskedasticity. Two traits, namely carcass temperature at 45 min postmortem, and loin muscle pH at 45 min postmortem, were selected for further consideration based on potentially high and mild levels of heteroskedasticity, respectively, and were thus subjected to follow-up WGP analysis (see next section). Phenotypes for 921 and 908 F<sub>2</sub> individuals were available for 45 min postmortem carcass temperature and for loin muscle pH, respectively. Phenotypes of the selected traits, genotypes and pedigree of the available animals were contained in the Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022897/-/DC1/FileS2.zip">File S2</a>.</p><p id="p-19">Each of the two selected traits were fitted using RR-BLUP, BayesA, BayesB and BayesCπ WGP models in both their homoskedastic and heteroskedastic error specifications. For both traits, <strong>β</strong> included the fixed effect of sex and a regression coefficient on carcass weight. The general WGP model in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> was further expanded to incorporate clustering effects of slaughter dates <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-79.gif"/></noscript></span></span>as well as polygenic effects <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-80.gif"/></noscript></span></span>, where <strong>A</strong> is a known pedigree-based additive relationship matrix. Therefore, in our data application, the genomic expected breeding value (GEBV) for individual <em>i</em> (<em>i</em> = 1,2,…,<em>n</em>) was defined as <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-81.gif"/></noscript></span></span>. We modeled heterogeneous residual variances as presented in <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a>, with <strong>τ</strong> and <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-82.gif"/></noscript></span></span> specifying the fixed effects of sex and the random clustering effects of slaughter dates, respectively. Thus, the hyperparameter <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-83.gif"/></noscript></span></span>in <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-84.gif"/></noscript></span></span> reflects the magnitude of residual heteroskedasticity in the responses of interest due to slaughter dates clusters.</p><p id="p-20">Prior specifications were similar to those described for the simulation study, with the following exceptions due to problems with parameter identifiability. For BayesCπ, the prior hyperparameter <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-85.gif"/></noscript></span></span> was set at <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-86.gif"/></noscript></span></span> = 3 for both traits to maximize prior uncertainty while retaining a defined mean (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-87.gif"/></noscript></span></span> &gt; 2). Instead, the prior scale <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-88.gif"/></noscript></span></span> for BayesCπ was specified as <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-89.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-90.gif"/></noscript></span></span>for carcass temperature and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-91.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-92.gif"/></noscript></span></span>for loin muscle pH, based on the posterior medians of <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-93.gif"/></noscript></span></span> obtained in BayesA. For BayesB, the hyperparameter <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-94.gif"/></noscript></span></span> was assumed known and set at <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-95.gif"/></noscript></span></span> for carcass temperature and <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-96.gif"/></noscript></span></span> for loin muscle pH, whereby these values were obtained based on the posterior median of <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-97.gif"/></noscript></span></span> under BayesCπ with a homoskedastic error assumption. Sensitivity analyses were conducted to assess the influence of specifying <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-98.gif"/></noscript></span></span> on posterior inference of interest. Also due to parameter identifiability issues, the variance <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-99.gif"/></noscript></span></span> of the polygenic effects was first estimated from traditional (<em>i.e.</em>, non-WGP) animal models that either assumed residual homo- or heteroskedasticity. These estimates of <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-100.gif"/></noscript></span></span> under homo- and heteroskedastic assumptions were then specified as known constants when fitting homo- and heteroskedastic WGP models, respectively. Homoskedastic and heteroskedastic error specifications of the selected WGP models were fitted to each trait. In every case, a total of 20 parallel MCMC chains were run, each consisting of 12,000 to 27,000 burn-in cycles followed by 6,000 to 14,000 saved cycles. Post burn-in samples from the 20 parallel MCMC chains run on a given model can be considered samples from the joint posterior density of interest, and were thus combined for inference. Initial values of hyperparameters in each parallel chain were dispersed by an arbitrary small value while constraining them to fall within their allowable parameter space (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gelman and Rubin 1992</a>). Posterior inference on parameters of interest was summarized for the overall dataset.</p></div><div id="sec-7" class="subsection"><h3>Model comparison</h3><p id="p-21">For each of the WGP models considered, namely RR-BLUP, BayesA, BayesB and BayesCπ, the performance of the homoskedastic <em>vs.</em> its heteroskedastic error model counterparts was compared in both simulated data and real data using various criteria for model fit and for prediction, as follows.</p><p id="p-22">First, we compared quality of global model fit using pseudo-Bayes factor (PBF) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gelfand 1996</a>), defined as the ratio of the conditional likelihood function under each heteroskedastic error WGP model over its homoskedastic counterpart, expressed in logarithmic scale of base 10, as follows:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-101.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where the abbreviation HT and HO hereafter refer to the candidate heteroskedastic and homoskedastic models, respectively. Moreover, L(⋅) denotes the likelihood function of observation <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-102.gif"/></noscript></span></span> conditional on all remaining observations fitted with the corresponding WGP model. This conditional likelihood, also known as the conditional predictive ordinate (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Gelfand 1996</a>) for observation <em>i</em>, can be approximated by <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-103.gif"/></noscript></span></span>, where <em>B</em> is the number of post burn-in MCMC iterations; <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-104.gif"/></noscript></span></span> represents the posterior sample for model parameters <strong>θ</strong> after <em>b</em> iterations post burn-in (<em>b =</em>1,2,…,<em>B</em>). A positive value of <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-105.gif"/></noscript></span></span> indicates support for the heteroskedastic error model based on enhanced fit to the data relative to its homoskedastic error WGP counterpart, thereby indicating evidence for heterogeneity of residual variances.</p><p id="p-23">We further compared predictive performance of breeding values between competing homoskedastic and heteroskedastic error alternatives of each WGP model. For simulated data, we assess genomic prediction accuracy using the Pearson correlation between TBV in the simulated validation set and corresponding estimates <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-106.gif"/></noscript></span></span>, whereby <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-107.gif"/></noscript></span></span> were obtained by fitting the WGP model to the simulated training set. Within each WGP model, we compared homoskedastic <em>vs.</em> heteroskedastic error specifications across the various scenarios using a multifactorial ANOVA on genomic prediction accuracy, with the simulated replicated dataset as a random blocking factor.</p><p id="p-24">For the real data application, predictive performance was assessed using a five-fold cross-validation (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Daetwyler <em>et al.</em> 2013</a>), whereby animals within each slaughter dates cluster were randomly assigned to five nonoverlapping data partitions or folds of nearly equal size (175–191 animals). Each one of the five data folds was assigned to be a validation set exactly once. When a data fold was selected as a validation set, phenotypes of this particular fold were excluded from estimation of marker effects. Instead, phenotypes of the validation fold were predicted using estimates of SNP markers, polygenic and nongenetic effects obtained from fitting a model to the remaining data folds, referred to here at training folds. This procedure was repeated until each of the five data folds had served as a validation set once. Consequently, every phenotyped animal was excluded from estimation of marker effects once, in which case their phenotypes were predicted using estimates obtained from animals in corresponding training folds (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Meuwissen <em>et al.</em> 2013</a>). We defined cross-validation predictive ability as the Pearson correlation coefficient between observed phenotypes in the validation fold, and the corresponding predicted phenotypes from parameter estimates obtained from the training folds. That is, <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-108.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-109.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-110.gif"/></noscript></span></span> are the observed and predicted phenotypes, respectively, for animal <em>i</em> in the validation fold. The predicted phenotypes <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-111.gif"/></noscript></span></span> included estimated marker effects (weighted by their allelic frequencies) and estimated polygenic effects, as well as the estimated fixed effects of sex and carcass weight, and the random blocking effects of slaughter dates.</p><p id="p-25">We also characterized potential practical implications of heteroskedasticity in the context of breeding decisions based on WGP. More specifically, we computed the Spearman’s rank correlation coefficient between GEBV from homoskedastic <em>vs.</em> heteroskedastic error specifications for the top and bottom 10% ranked individuals. Relative ranking of top and bottom 10% individuals was assessed by fitting a linear mixed model to the estimated Spearman rank correlations obtained from data replicates, and testing for differences between simulation scenarios. For real data, rank correlations of GEBV for top and bottom 10% individuals in the validation sets were compared between homoskedastic and heteroskedastic WGP models.</p></div><div id="sec-8" class="subsection"><h3>MCMC diagnostics</h3><p id="p-26">Convergence diagnostics were implemented using the R package CODA (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Plummer <em>et al.</em> 2006</a>). We monitored convergence using trace plots. Diagnostic tests by <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Raftery and Lewis (1992)</a> and by <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Heidelberger and Welch (1983)</a> were conducted on the simulation study. For the data application, the Gelman and Rubin’s diagnostic on multiple MCMC chains produced a shrinkage factor &lt; 1.2 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Colosimo and Del Castillo 2007</a>). We also determined effective sample size (ESS) for key hyperparameters (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kass <em>et al.</em> 1998</a>). In each case, the number of MCMC cycles was adjusted to ensure that the ESS was greater than 100 for all hyperparameters.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-27">Data and R code are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022897/-/DC1/FileS1.zip">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022897/-/DC1/FileS2.zip">File S2</a> (refer to “README_data.txt” and “README_mcmc.txt”, respectively, for details).</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Simulation study</h3><p id="p-28">For each of the WGP models considered in this study, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows comparisons of global fit, expressed as <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-112.gif"/></noscript></span></span>, between homoskedastic and heteroskedastic model specifications for scenarios reflecting a gradient of increasing residual heteroskedasticity. Recall that positive values of the <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-113.gif"/></noscript></span></span> indicate support for the heteroskedastic, as opposed to the homoskedastic error version of the corresponding WGP model. When residual heteroskedasticity was high (<span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-114.gif"/></noscript></span></span>= 3 or 5), <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-115.gif"/></noscript></span></span>was estimated to be between 12.8 and 77.3 across MC replicates fitted with any of the WGP models. This supports a strong advantage in global fit for heteroskedastic, rather than homoskedastic, error specifications, regardless of the specific WGP model. As the amount of residual heteroskedasticity decreased (<span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-116.gif"/></noscript></span></span>= 12), so did the values of <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-117.gif"/></noscript></span></span> and thus the relative advantage of the heteroskedastic error WGP model over its homoskedastic error counterpart. Under scenarios of low heteroskedasticity (<span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-118.gif"/></noscript></span></span>= 50), or of homogeneous residual variance (<span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-119.gif"/></noscript></span></span>→ ∞), the values of <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-120.gif"/></noscript></span></span> under RR-BLUP and BayesA were closer to zero, thus indicating no apparent advantage of heteroskedastic WGP models over their homoskedastic error counterparts; in turn, BayesB and BayesCπ showed greater uncertainty in these conditions. Overall, we note that, when the amount of residual heteroskedasticity in the data was high (<span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-121.gif"/></noscript></span></span>= 3 or 5), <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-122.gif"/></noscript></span></span> consistently selected the appropriate heteroskedastic error specification for all WGP models; however, as mentioned above, the discriminatory capability of <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-123.gif"/></noscript></span></span> to detect random sources of residual heteroskedasticity was partially, though incrementally, impaired when heteroskedasticity was moderate (<span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-124.gif"/></noscript></span></span>= 12) or low (<span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-125.gif"/></noscript></span></span>= 50).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/1/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assessment of global fit, expressed as log10 pseudo-Bayes factor (PBF), between heteroskedastic and homoskedastic whole-genome prediction (WGP) models, namely RR-BLUP, BayesA, BayesB, and BayesC&#x3C0;, for 10 replicated data sets from each of five simulated scenarios defined by either residual homoskedasticity (&#x2192; &#x221E;) or a gradient of residual heteroskedasticity ranging from high (=&#xA0;3, 5) to moderate (=&#xA0;12) to low (=&#xA0;50). A horizontal reference line is provided at zero." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1253527945" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Assessment of global fit, expressed as log10 pseudo-Bayes factor (PBF), between heteroskedastic and homoskedastic whole-genome prediction (WGP) models, namely RR-BLUP, BayesA, BayesB, and BayesC&#x3C0;, for 10 replicated data sets from each of five simulated scenarios defined by either residual homoskedasticity (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-126&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&#x2192; &#x221E;) or a gradient of residual heteroskedasticity ranging from high (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-127&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;=&#xA0;3, 5) to moderate (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-128&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;=&#xA0;12) to low (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-129&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;=&#xA0;50). A horizontal reference line is provided at zero.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/1/F1.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/1/1/F1.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/1/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/1/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467487" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-29" class="first-child">Assessment of global fit, expressed as log10 pseudo-Bayes factor (PBF), between heteroskedastic and homoskedastic whole-genome prediction (WGP) models, namely RR-BLUP, BayesA, BayesB, and BayesCπ, for 10 replicated data sets from each of five simulated scenarios defined by either residual homoskedasticity (<span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F1/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F1/embed/mml-math-126.gif"/></noscript></span></span>→ ∞) or a gradient of residual heteroskedasticity ranging from high (<span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F1/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F1/embed/mml-math-127.gif"/></noscript></span></span>= 3, 5) to moderate (<span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F1/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F1/embed/mml-math-128.gif"/></noscript></span></span>= 12) to low (<span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F1/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F1/embed/mml-math-129.gif"/></noscript></span></span>= 50). A horizontal reference line is provided at zero.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> presents a summary of the posterior inference on the hyperparameter <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-130.gif"/></noscript></span></span>defining the degree of heterogeneity of residual variance across levels of the random effect factor for 10 MC replicates under each of the simulation scenarios fitted with the heteroskedastic WGP models. Coverage probabilities for <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-131.gif"/></noscript></span></span>, defined as proportion MC replicates for which the 95% highest posterior density (HPD) included the true parameter value, under WGP models RR-BLUP and BayesA were mutually identical at 100%, 90%, 100%,100% when true <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-132.gif"/></noscript></span></span>= 3, 5, 12, 50, respectively. In turn, coverage probabilities were 100%, 90%, 70%, 20% for BayesB and 100%, 90%, 80%, 20% for BayesCπ, when true <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-133.gif"/></noscript></span></span>= 3, 5, 12, 50, respectively. As might be expected, inferential precision on <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-134.gif"/></noscript></span></span>was maximized when heteroskedasticity was high, as indicated by the smallest difference between minimum and maximum values of the lower and upper boundaries of the 95% HPD on <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-135.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The reverse was also true, as inference on <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-136.gif"/></noscript></span></span>was most uncertain when heteroskedasticity was not present. That is, enhanced model fit of a heteroskedastic WGP model relative to its homoskedastic counterpart under conditions of heterogeneous variances may be explained by increased precision of inference on the hyperparameter<span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-137.gif"/></noscript></span></span>, and vice versa. These results on posterior inference of the heteroskedasticity parameter <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-138.gif"/></noscript></span></span> are consistent with those presented for overall goodness of fit in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467343/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467343/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Posterior inference on the hyperparameter <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/T1/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/T1/embed/mml-math-139.gif"/></noscript></span></span> for simulated datasets</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">To validate inferential performance on the fixed effect parameters <strong>τ</strong> specified on the residual variance, we considered the posterior density of the ratio of <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-148.gif"/></noscript></span></span> over <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-149.gif"/></noscript></span></span>. Coverage probability of the 95% HPD for the true value of the parameter ratio under heteroskedastic WGP models was 92% for both BayesB and BayesCπ, and 94% for both RR-BLUP and BayesA across simulation scenarios. In all cases, the observed coverage was within probabilistic expectation.</p><p id="p-33">Estimated genomic prediction accuracies (and corresponding standard errors) of heteroskedastic and homoskedastic error versions of WGP models are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. For all RR-BLUP, BayesA, BayesB and BayesCπ models, the heteroskedastic specification showed a gain on genomic prediction accuracy relative to the homoskedastic WGP counterpart whenever the amount of residual heteroskedasticity in the data were high (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-150.gif"/></noscript></span></span>= 3 or 5, <em>P</em> &lt; 0.001 in all cases). However, no evidence for any predictive advantage of heteroskedastic WGP specifications was apparent if the data had been generated under conditions of low or null heteroskedasticity (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-151.gif"/></noscript></span></span>= 50 or ∞; <em>P</em> &gt; 0.30 in all cases). For situations of moderate heteroskedasticity (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-152.gif"/></noscript></span></span>= 12) fitted with RR-BLUP or BayesA WGP models, the heteroskedastic error specification yielded greater (<em>P</em> &lt; 0.05) genomic predictive accuracy than its homoskedastic counterpart but this difference was not apparent using variable selection models like BayesB or BayesCπ. Despite the significant increase in genomic prediction accuracy by heteroskedastic WGP models when the data were highly heteroskedastic, we note that the gain in accuracy relative to the homoskedastic specification was of a relatively small magnitude (<em>i.e.</em>, range from 0.009 to 0.018 for <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-153.gif"/></noscript></span></span>= 3 and from 0.005 to 0.008 for <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-154.gif"/></noscript></span></span>= 5 across MC replicated data sets).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/1/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic prediction accuracy (least square mean estimates and 95% confidence intervals) for heteroskedastic (gray) and homoskedastic (white) specifications of WGP models considered in this study, namely RR-BLUP, BayesA, BayesB, and BayesC&#x3C0;, under simulation scenarios defined by either residual homoskedasticity (&#x2192; &#x221E;) or a gradient of residual heteroskedasticity ranging from high (=&#xA0;3, 5) to moderate (=&#xA0;12) to low (=&#xA0;50). Genomic prediction accuracy was defined as the Pearson correlation coefficient between true breeding value and expected breeding value. ** and * indicate differences between heteroskedastic and homoskedastic versions of each WGP model at P&#xA0;=&#xA0;0.001 and P&#xA0;=&#xA0;0.05, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1253527945" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Genomic prediction accuracy (least square mean estimates and 95% confidence intervals) for heteroskedastic (gray) and homoskedastic (white) specifications of WGP models considered in this study, namely RR-BLUP, BayesA, BayesB, and BayesC&#x3C0;, under simulation scenarios defined by either residual homoskedasticity (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-155&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&#x2192; &#x221E;) or a gradient of residual heteroskedasticity ranging from high (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-156&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;=&#xA0;3, 5) to moderate (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-157&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;=&#xA0;12) to low (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-158&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;=&#xA0;50). Genomic prediction accuracy was defined as the Pearson correlation coefficient between true breeding value and expected breeding value. ** and * indicate differences between heteroskedastic and homoskedastic versions of each WGP model at &lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;0.001 and &lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;0.05, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/1/F2.medium.gif" width="440" height="316"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/1/1/F2.medium.gif" width="440" height="316"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/1/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/1/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467333" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">Genomic prediction accuracy (least square mean estimates and 95% confidence intervals) for heteroskedastic (gray) and homoskedastic (white) specifications of WGP models considered in this study, namely RR-BLUP, BayesA, BayesB, and BayesCπ, under simulation scenarios defined by either residual homoskedasticity (<span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F2/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F2/embed/mml-math-155.gif"/></noscript></span></span>→ ∞) or a gradient of residual heteroskedasticity ranging from high (<span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F2/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F2/embed/mml-math-156.gif"/></noscript></span></span>= 3, 5) to moderate (<span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F2/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F2/embed/mml-math-157.gif"/></noscript></span></span>= 12) to low (<span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F2/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F2/embed/mml-math-158.gif"/></noscript></span></span>= 50). Genomic prediction accuracy was defined as the Pearson correlation coefficient between true breeding value and expected breeding value. ** and * indicate differences between heteroskedastic and homoskedastic versions of each WGP model at <em>P</em> = 0.001 and <em>P</em> = 0.05, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">To further characterize potential practical implications of heteroskedastic <em>vs.</em> homoskedastic WGP models in the context of breeding programs, we explored differences in the ranking of individuals of extreme genetic merit. We computed the Spearman correlation of the top 10% individuals whose GEBV had been estimated from homoskedastic and heteroskedastic WGP models across the simulated gradient of residual heteroskedasticity (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Results on the bottom 10% ranked individuals showed a similar pattern and are thus not discussed further. For homoskedastic scenarios (<span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-159.gif"/></noscript></span></span>→ ∞) or scenarios of low heteroskedasticity (<span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-160.gif"/></noscript></span></span>= 50), the Spearman rank correlation between homoskedastic- and heteroskedastic-based GEBVs from RR-BLUP, BayesA, BayesB or BayesCπ WGP models for the top 10% animals was close to 1, thus indicating minor concerns for selection purposes. However, as the amount of heteroskedasticity increased, the Spearman correlation between heteroskedastic-based GEBV and homoskedastic-assuming GEBV for the top 10% individuals decreased to an estimated value of 0.85. This result suggests nonnegligible reranking of top individuals for selection purposes. Given response to selection, this finding could potentially have direct implications for breeding programs despite the small magnitude of the overall gain on genomic prediction accuracy described before.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/1/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spearman&#x2019;s correlation coefficient (least square mean estimates and 95% confidence intervals) between top 10% homoskedastic-assuming predicted genomic breeding values (GEBV), and their heteroskedastic-assuming counterpart GEBV across WGP models under simulation scenarios defined by either residual homoskedasticity (&#x2192; &#x221E;) or a gradient of residual heteroskedasticity ranging from high (=&#xA0;3, 5) to moderate (=&#xA0;12) to low (=&#xA0;50). (a,b,c,d) Different letters indicate significant differences in ranking at &#x3B1;&#xA0;=&#xA0;0.05." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1253527945" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spearman&#x2019;s correlation coefficient (least square mean estimates and 95% confidence intervals) between top 10% homoskedastic-assuming predicted genomic breeding values (GEBV), and their heteroskedastic-assuming counterpart GEBV across WGP models under simulation scenarios defined by either residual homoskedasticity (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-161&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&#x2192; &#x221E;) or a gradient of residual heteroskedasticity ranging from high (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-162&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;=&#xA0;3, 5) to moderate (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-163&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;=&#xA0;12) to low (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-164&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;=&#xA0;50). (a,b,c,d) Different letters indicate significant differences in ranking at &#x3B1;&#xA0;=&#xA0;0.05.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/1/F3.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/1/1/F3.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/1/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/1/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467500" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Spearman’s correlation coefficient (least square mean estimates and 95% confidence intervals) between top 10% homoskedastic-assuming predicted genomic breeding values (GEBV), and their heteroskedastic-assuming counterpart GEBV across WGP models under simulation scenarios defined by either residual homoskedasticity (<span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F3/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F3/embed/mml-math-161.gif"/></noscript></span></span>→ ∞) or a gradient of residual heteroskedasticity ranging from high (<span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F3/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F3/embed/mml-math-162.gif"/></noscript></span></span>= 3, 5) to moderate (<span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F3/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F3/embed/mml-math-163.gif"/></noscript></span></span>= 12) to low (<span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F3/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/F3/embed/mml-math-164.gif"/></noscript></span></span>= 50). (a,b,c,d) Different letters indicate significant differences in ranking at α = 0.05.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>MSU swine resource population data</h3><p id="p-37">For carcass temperature at 45 min postmortem, the variance <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-165.gif"/></noscript></span></span> of polygenic effects were estimated at 0.022 and 0.036 for homoskedastic and heteroskedastic error specifications, respectively. For loin muscle pH at 45 min, the corresponding estimates of <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-166.gif"/></noscript></span></span> were 0.006 and 0.005, respectively.</p><p id="p-38">We first assessed evidence for residual heteroskedasticity on the traits carcass temperature and loin muscle pH 45 min postmortem selected for this study from the MSU resource population. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> summarizes the posterior inference for <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-167.gif"/></noscript></span></span> in the heteroskedastic specification of WGP models on the complete dataset. Recall that the hyperparameter <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-168.gif"/></noscript></span></span> defines the magnitude of non-systematic heterogeneity of residual variances for each of the selected traits as a function of slaughter dates clusters. For carcass temperature at 45 min postmortem, the posterior mean of <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-169.gif"/></noscript></span></span>was smaller than two in all cases. Similarly, the magnitude of the upper boundaries of the 95% HPD on <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-170.gif"/></noscript></span></span> did not exceed three under any of the heteroskedastic WGP models considered, thereby providing evidence for strong cluster-based heterogeneity of residual variances for this trait. In fact, the range of posterior means of slaughter date-specific <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-171.gif"/></noscript></span></span> for carcass temperature at 45 min postmortem was greater than eight-fold, thereby indicating that the residual variance for some slaughter dates clusters was estimated to be as large as eight times greater than the residual variance in other slaughter clusters. In turn, for loin muscle pH at 45 min postmortem, the posterior mean of <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-172.gif"/></noscript></span></span> was below six, and the corresponding upper boundaries of its 95% HPD was approximately 10 in all cases, thus indicating milder cluster-driven residual heteroskedasticity for this trait, whereby the range of posterior means of <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-173.gif"/></noscript></span></span>was close to three-fold across all 33 slaughter dates clusters. These results are consistent with our findings during preliminary data screening.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467378/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467378/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Posterior inference on the hyperparameter <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/T2/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/T2/embed/mml-math-174.gif"/></noscript></span></span> for two quantitative traits from the Michigan State University swine resource population</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">We also explored the effect of sex as an additional source of heteroskedasticity on the selected traits, as represented by parameter <strong>τ</strong> in <a id="xref-disp-formula-2-4" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a>. Based on the set-to-zero parameterization implemented in this study, the parameter <strong>τ</strong> may be interpreted as a ratio of female-to-male residual variances, whereby a ratio of one indicates homogeneous residual variances for both sexes. For carcass temperature at 45 min postmortem, the 95% HPD of <strong>τ</strong> fitted with any of the WGP models ranged from a lower boundary of 0.7 to an upper boundary of 1.2. For loin muscle pH, the corresponding 95% HPD range was 0.9 to 1.5. It then follows no evidence for sex-based heteroskedasticity of either trait regardless of WGP model.</p><p id="p-41">Next, we consider relative global fit of homoskedastic <em>vs.</em> heteroskedastic error WGP models to the actual data using PBF (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Gelfand 1996</a>), and use a threshold value of <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-179.gif"/></noscript></span></span> to conclude upon a decisive difference in fit between models (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kass and Raftery 1995</a>). For carcass temperature at 45 min postmortem, the range of <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-180.gif"/></noscript></span></span> across the five cross-validation folds was [45.7, 81.3] for RR-BLUP, [46.9, 81.0] for BayesA, [44.3, 77.5] for BayesB, and [43.8, 76.9] for BayesCπ WGP models. In turn, for loin muscle pH at 45 min postmortem, the range of <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-181.gif"/></noscript></span></span>was [9.7, 14.4], [9.8, 14.6], [8.8, 13.7] and [8.6, 13.8]. These results favor the use of heteroskedastic WGP error models for both traits, and under all WGP specifications considered here. The larger magnitude of <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-182.gif"/></noscript></span></span>, and hence greater evidence of residual heteroskedasticity for carcass temperature relative to loin muscle pH, is consistent both with our preliminary screening and with our posterior inference on <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-183.gif"/></noscript></span></span> as described previously.</p><p id="p-42">We also assessed predictive performance of homoskedastic and heteroskedastic WGP models. We first conducted a sensitivity analysis to evaluate the stability of cross-validation predictive ability using BayesCπ under different choices of <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-184.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-185.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-186.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-187.gif"/></noscript></span></span>for carcass temperature and <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-188.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-189.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-190.gif"/></noscript></span></span>for loin muscle pH trait. Similarly, sensitivity assessments were also conducted for BayesB looking at choices of <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-191.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-192.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-193.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-194.gif"/></noscript></span></span>for carcass temperature and <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-195.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-196.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-197.gif"/></noscript></span></span> for loin muscle pH trait. As expected, changes in the specification of <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-198.gif"/></noscript></span></span> were compensated with changes in the estimates of the proportion of SNP markers with nonzero effects (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Yang <em>et al.</em> 2015</a>). In turn, the estimated median cross-validation prediction accuracies for carcass temperature, and its corresponding standard deviation, across cross-validation folds at any choice of <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-199.gif"/></noscript></span></span> were 0.86 ± 0.05 for homoskedastic error BayesCπ or BayesB and 0.86 ± 0.04 for heteroskedastic error BayesCπ or BayesB. For loin muscle pH, cross-validation prediction accuracy based on homoskedastic error BayesCπ or BayesB ranged from 0.31 ± 0.06 to 0.29 ± 0.07 across prior specifications of <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-200.gif"/></noscript></span></span>. For heteroskedastic error BayesCπ or BayesB, cross-validation predictive ability ranged from 0.32 ± 0.07 to 0.29 ± 0.07 across values of <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-201.gif"/></noscript></span></span>. Overall, sensitivity analyses assessment indicated little reason to be concerned about specification of hyperparameters for the purpose of prediction accuracy.</p><p id="p-43"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> depicts estimated cross-validation predictive abilities across five folds for both carcass temperature and loin muscle pH at 45 min postmortem. Across WGP models, cross-validation predictive abilities for carcass temperature and loin muscle pH 45 min postmortem were estimated to be approximately 0.85 and 0.32, respectively. For neither trait did we find any evidence for differences in cross-validation predictive ability between homoskedastic <em>vs.</em> heteroskedastic specifications of any of the WGP models considered (<em>P</em> &gt; 0.25 in all cases for either trait).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/1/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated cross-validation predictive ability (and 95% confidence intervals) of genomic breeding values for (A) carcass temperature 45&#xA0;min postmortem and (B) loin muscle pH 45 min postmortem fitted with heteroskedastic (gray bars) and homoskedastic (white bars) specifications of WGP models considered in this study, namely RR-BLUP, BayesA, BayesB, and BayesC&#x3C0;. Cross-validation predictive ability is represented by the Pearson correlation coefficient between observed phenotypes in the validation fold, and their corresponding predictions based on estimates from the training folds in a five-fold cross-validation study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1253527945" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated cross-validation predictive ability (and 95% confidence intervals) of genomic breeding values for (A) carcass temperature 45&#xA0;min postmortem and (B) loin muscle pH 45 min postmortem fitted with heteroskedastic (gray bars) and homoskedastic (white bars) specifications of WGP models considered in this study, namely RR-BLUP, BayesA, BayesB, and BayesC&#x3C0;. Cross-validation predictive ability is represented by the Pearson correlation coefficient between observed phenotypes in the validation fold, and their corresponding predictions based on estimates from the training folds in a five-fold cross-validation study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/1/F4.medium.gif" width="440" height="141"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/1/1/F4.medium.gif" width="440" height="141"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/1/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/1/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467526" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-44" class="first-child">Estimated cross-validation predictive ability (and 95% confidence intervals) of genomic breeding values for (A) carcass temperature 45 min postmortem and (B) loin muscle pH 45 min postmortem fitted with heteroskedastic (gray bars) and homoskedastic (white bars) specifications of WGP models considered in this study, namely RR-BLUP, BayesA, BayesB, and BayesCπ. Cross-validation predictive ability is represented by the Pearson correlation coefficient between observed phenotypes in the validation fold, and their corresponding predictions based on estimates from the training folds in a five-fold cross-validation study.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">Often in animal breeding, greater interest is directed toward animals that exhibit extreme GEBV, as they are the ones likely to be selected as parents for the next generation. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> reports Spearman’s rank correlation coefficient between homoskedastic-error <em>vs.</em> heteroskedastic-error GEBV for animals of extreme genetic merit. For both traits, and regardless of WGP model, we observed considerable reranking of the top and bottom 10% individuals, particularly as the degree of residual variance heterogeneity in the data increased. In fact, for loin muscle pH at 45 min postmortem, the corresponding estimated median rank correlations ranged from 0.52 to 0.70 (top), and from 0.64 to 0.70 (bottom) across WGP models; in turn, for the more heteroskedastic trait (<em>i.e.</em>, carcass temperature at 45 min postmortem), the median rank correlation of GEBV for top and bottom 10% animals ranged from 0.05 to 0.38 (top) and from 0.43 to 0.54 (bottom) across WGP models. Such variability in rankings of GEBV may be partially due to the relatively small sample size (<em>i.e.</em>, only 10% animals within a cross-validation fold) used to estimate the rank correlation coefficient. This is further supported by the considerable variability observed among cross-validation folds in the reranking of individuals based on homoskedastic-based GEBV relative to their heteroskedastic counterpart, though this variability was particularly noticeable for carcass temperature. Again, this may be partially explained by the relatively larger magnitude of residual heteroskedasticity detected for this trait. We further note that reranking of extreme GEBV using homoskedastic <em>vs.</em> heteroskedastic errors seemed to be particularly extreme when the BayesB WGP specification was implemented.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467394/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467394/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Estimated Spearman’s rank correlation coefficient between homoskedastic-based and heteroskedastic-based estimated genomic breeding values corresponding to the top and bottom 10% individuals (approximately 17–20 within a cross-validation fold) for two quantitative traits of swine data</span><div class="sb-div caption-clear"></div></div></div><p id="p-47">For further illustration, we selected a cross-validation fold and depicted a scatterplot of homoskedastic-error <em>vs.</em> heteroskedastic-error GEBV for carcass temperature (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>) and for loin muscle pH (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>) under each WGP model. For both traits, individuals that showed extreme genetic merit under homoskedastic error assumptions had their GEBV considerably attenuated when residual heteroskedasticity was accounted for (<em>e.g.</em>, <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>, BayesB). In fact, an individual with extremely high GEBV inferred under a homoskedastic error model may not be considered as a viable selection candidate if its GEBV was estimated from a heteroskedastic error WGP model. This was indeed the case for the two individuals with top homoskedastic-based GEBV in the complete dataset. It is interesting to note that these top two individuals derived from one slaughter date cluster that had the largest posterior mean for the relative residual variance <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-207.gif"/></noscript></span></span> (data not shown). Conversely, candidate individuals with top or bottom genetic merit may be overlooked by using conventional homoskedastic error models (<em>e.g.</em>, <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>, BayesB).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/1/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustrative scatter plots of predicted genomic breeding values obtained from homoskedastic (HO) and heteroskedastic (HT) specifications of RR-BLUP, BayesA, BayesB, and BayesC&#x3C0; WGP models fitted to carcass temperature (A) and loin muscle pH (B) at 45 min postmortem. Symbols indicate individuals ranked among the top 10% (&#x25B4;), bottom 10% (&#x2666;) or remainder (&#x25CB;) based on the homoskedastic WGP model. Plots are based on a single cross-validation fold and are meant for illustration only." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1253527945" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Illustrative scatter plots of predicted genomic breeding values obtained from homoskedastic (HO) and heteroskedastic (HT) specifications of RR-BLUP, BayesA, BayesB, and BayesC&#x3C0; WGP models fitted to carcass temperature (A) and loin muscle pH (B) at 45 min postmortem. Symbols indicate individuals ranked among the top 10% (&#x25B4;), bottom 10% (&#x2666;) or remainder (&#x25CB;) based on the homoskedastic WGP model. Plots are based on a single cross-validation fold and are meant for illustration only.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/1/F5.medium.gif" width="267" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/1/1/F5.medium.gif" width="267" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/1/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/1/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467348" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-48" class="first-child">Illustrative scatter plots of predicted genomic breeding values obtained from homoskedastic (HO) and heteroskedastic (HT) specifications of RR-BLUP, BayesA, BayesB, and BayesCπ WGP models fitted to carcass temperature (A) and loin muscle pH (B) at 45 min postmortem. Symbols indicate individuals ranked among the top 10% (▴), bottom 10% (♦) or remainder (○) based on the homoskedastic WGP model. Plots are based on a single cross-validation fold and are meant for illustration only.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">Taken together, the observed reranking could have practical implications from a selection point of view. In support of this observation, we note that the mean of genomic breeding values in the top 10% individuals was between 1.2× (based on BayesA) to 10× (based on BayesCπ) greater in magnitude when estimated based on the heteroskedastic WGP model relative to the homoskedastic specification for either trait. Similar results were observed for the bottom 10% individuals with extreme genetic merit based on the hetero- <em>vs.</em> homoskedastic WGP specification. It should be acknowledged, though, that these comparisons are conditioned on the models used to predict mean genomic breeding values.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-50">In this study, we extend classical WGP models to account for potential heterogeneous residual variances across environments, and further assess whether explicit accounting for such heteroskedasticity may impact accuracy of prediction of genomic breeding values.</p><p id="p-51">Environmental residual heteroskedasticity is a rather common phenomenon across agricultural environments in livestock production. For instance, residual variance estimates for birth weight in an Italian Piemontese cattle population differed by approximately 10-fold across herds (<a id="xref-ref-45-7" class="xref-bibr" href="#ref-45">Kizilkaya and Tempelman 2005</a>), and that of average daily gains in feedlot cattle from the US Midwest differed by more than 12-fold across contemporary groups (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cernicchiaro <em>et al.</em> 2013</a>). Backfat thickness in pigs was shown to display considerable residual heteroskedasticity based on an animal model, whereby residual variances ranged by approximately eight-fold across herds (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">See 1998</a>). Similarly, in this study, we found evidence for considerable environmentally-driven heterogeneity of residual variances in other swine carcass traits, as indicated by the small magnitude of posterior means of <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-208.gif"/></noscript></span></span> for carcass temperature and loin muscle pH. These results indicate considerable departure from the residual homoskedasticity assumption commonly invoked by standard WGP models.</p><p id="p-52"><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gianola and Rosa (2015)</a> adverted that modeling heterogeneous residual variances across environments was likely to be important for reliable genomic selection, as further supported by our results. Unaccounted-for heteroskedastic errors can potentially impact breeding decisions as animals from the most diverse environments might then be disproportionally selected (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Hill 1984</a>). Indeed, previous studies have shown nonnegligible reranking of top and bottom 10% progenitors when heterogeneity of residual variances across environment or management groups is properly modeled (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cardoso <em>et al.</em> 2005</a>; <a id="xref-ref-45-8" class="xref-bibr" href="#ref-45">Kizilkaya and Tempelman 2005</a>). In fact, incorrectly assuming homogeneous residual variances could cause a substantial reduction in selection efficiency, particularly under conditions of low heritability (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Garrick and VanVleck 1987</a>). Heritabilities of most technological quality traits of meat in swine, such as the ones evaluated in this study, has been reported to range from low to moderate, as the average value for many studies fall into the range 0.10–0.30 (reviewed by <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Sellier and Monin 1994</a> and <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ciobanu <em>et al.</em> 2011</a>).</p><p id="p-53">Based on this evidence, we extended homoskedastic WGP models to allow for environmental heterogeneity of residual variances, and evaluated the relative performance of heteroskedastic and homoskedastic error WGP models in terms of both global fit and predictive performance. Our simulation study showed considerable improvements in global model fit when extreme residual heteroskedasticity was properly accounted for, though the advantage of heteroskedastic error WGP models seemed to dissipate quickly for even moderate amounts of environmental heterogeneity in residual variances, particularly under WGP models without variable selection (<em>i.e.</em>, RR-BLUP and BayesA). Furthermore, the observed advantage of heteroskedastic error WGP models in global fit translated into very small (∼1–2%), albeit significant, gains in genomic prediction accuracy under conditions of extreme data heteroskedasticity. As the amount of residual heteroskedasticity decreased, so did the power to detect differences in genomic prediction accuracy between heteroskedastic and homoskedastic model specification. This was particularly noticeable for WGP models with variable selection (<em>i.e.</em>, BayesB or BayesCπ) relative to those without variable selection (<em>i.e.</em>, RR-BLUP and BayesA). For the specific data application used in this study corresponding to the MSU swine resource population, there was no evidence of any gains in cross-validation predictive ability for selected carcass traits when heterogeneous residual variances across environments were explicitly modeled. This finding was somewhat unexpected given the extreme level of environmental heterogeneity of residual variances observed in at least one of those traits. The high level of environmental heteroskedasticity in the carcass temperature trait was recognized both by posterior inference on the hyperparameter <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-209.gif"/></noscript></span></span> and by improved global model fit of the heteroskedastic error model relative to the homoskedastic error model. Yet, it is possible that additional gains in prediction accuracy from specifying heteroskedasticity, either of environmental or genetic origin, may be difficult to observe due to the already large magnitude of “baseline” cross-validation predictive ability for this trait (∼0.85) based on standard homoskedastic error WGP models.</p><p id="p-54">It is unclear whether a genetic component might have contributed to the high level of residual heteroskedasticity observed in the carcass temperature trait. A recent study by <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Yang <em>et al.</em> (2011)</a> explored the use of parametric genomic models that specify genetic control of environmental variance in a swine production system. In particular, classical WGP models were extended to assess putative marker effects not only on GEBV but also on environmental variability. Consistent with our results, that study indicated enhanced fit of heteroskedastic error models. However, the gains in accuracy of prediction were either of small magnitude in simulated data or not at all apparent when applied to back fat thickness data in pigs, as was also observed in our application. Additional statistical methods for detecting genetic loci affecting phenotypic variability were recently introduced; proposed approaches range from fully-parametric (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Rönnegård and Valdar 2011</a>; <a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Yang <em>et al.</em> 2011</a>) through classical nonparametric (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Paré <em>et al.</em> 2010</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Struchalin <em>et al.</em> 2010</a>), including a two-stage semi-parametric approximation (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Hill and Mulder 2010</a>). Following a thorough review, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rönnegård and Valdar (2012)</a> highlighted the inferential importance of simultaneous estimation of effects on mean and variance as a strength of parametric methods for modeling variance-controlling QTL.</p><p id="p-55">Ideally, heterogeneous genetic and residual variances should be modeled simultaneously, for example with multiple breed studies (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">de Roos <em>et al.</em> 2009</a>) or genotype by environment interaction (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Edwards and Jannink 2006</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Jarquin <em>et al.</em> 2014</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lopez Cruz <em>et al.</em> 2015</a>) studies, both of which require the specification of heterogeneous genetic variances. A study by <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> (2014)</a> explored multi-population genomic prediction for milk production traits on two dairy breeds using a multi-task Bayesian learning model. When error variances and marker effects variances were explicitly specified as heterogeneous across breeds (as opposed to assumed homogeneous), gains in prediction accuracy across traits ranged from 0.04 to 0.14 using a 50K SNP panel, and from 0.02 to 0.11. Also, a yield variety trial of 40 oat genotypes across 34 environments reported “stabilized” genetic predictions (<em>i.e.</em>, higher repeatability) when genetic and environmental sources of heterogeneity were explicitly specified on residual and genotype-by-environment variance components (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Edwards and Jannink 2006</a>).</p><p id="p-56">It is often the case that deregressed expected breeding values are weighted and used as response variables in WGP models (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Garrick <em>et al.</em> 2009</a>) instead of actual phenotypes. In a way, modeling of weighted deregressed expected breeding values may be considered an approach to account of heterogeneous variance, in this case the variance of the expected breeding value. However, this approach may be considered <em>ad hoc</em> as its effectiveness depends on several factors such as the number of repeated measured observations, size of training data and the reliability of the breeding values (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Garrick <em>et al.</em> 2009</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ostersen <em>et al.</em> 2011</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boddhireddy <em>et al.</em> 2014</a>).</p><p id="p-57">Despite the observed lack of any appreciable gain in overall accuracy of prediction of carcass trait phenotypes by heteroskedastic WGP models, the differential ranking of animals with the 10% most extreme genetic merit suggest important practical implications for the assumption of homogeneous residual variance. Substantial reranking was apparent for these candidate animals depending on whether environmental residual heteroskedasticity was explicitly accounted for in obtaining their breeding values. In fact, noticeable differences in the mean of genomic breeding values for individuals of extreme genetic merit were apparent based on heteroskedastic <em>vs.</em> homoskedastic WGP specifications. To reconcile these results, we notice that the magnitude of the difference in GEBV between most extreme individuals and the rest seems to be rather small, and thus difficult to detect, particularly given the relatively narrow range of GEBV observed for the selected traits in this population. In turn, a localized performance metric, such as the Spearman correlation coefficient among the top and bottom 10% individuals, could detect regional patterns that may not be necessarily apparent from overall performance metrics, such as cross-validation predictive ability. However, we acknowledge that, given the low-to-moderate heritabilities of the traits evaluated in this study, it is not possible to discard nonnegligible sampling variability on the estimates of Spearman correlation coefficients or other statistical reasons related to unstable behavior of correlations within extreme tails.</p><p id="p-58">The very low magnitude of the posterior mean of <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/1/embed/mml-math-210.gif"/></noscript></span></span>observed in the swine data application indicates that residual variability is not homogeneous across environmental subclasses. However, extreme within-cluster residuals, for example, due to preferential treatment, may be a reasonable concern even after accounting for residual heteroskedasticity. Biased prediction of breeding values is a problem often encountered under conditions of preferential treatment (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Kuhn and Freeman 1995</a>). Our observation of substantial reranking of extreme breeding values suggests that heteroskedastic WGP models may, at least partially, offset prediction bias due to preferential treatment. Yet, preferential treatment is a concern that further motivates the need to extend heteroskedastic error WGP models to allow for outlier robustness, as advocated by <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Gianola and Rosa (2015)</a>. In a nongenomic application, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cardoso <em>et al.</em> (2007)</a> extended the univariate <em>t</em>-models proposed by <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Stranden and Gianola (1999)</a> to attenuate adverse effects of preferential treatment and specified Student-<em>t</em> distributed residual heteroskedasticity across environments to potentially accommodate a more robust analysis capable of muting the influence of extreme observations on inferences of cluster specific residual variances. Given the breeding objective of ranking candidates for selection, a heavy-tailed residual distribution combined with explicit modeling of environmental heteroskedasticity is likely to yield more robust genomic predictions in the sense of reducing influence from outlying identifiable clusters and extreme individual datapoints.</p><p id="p-59">Most recently, WGP models have been used to predict complex human traits, such as risk of disease and life expectancy (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">de los Campos <em>et al.</em> 2012</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Vazquez <em>et al.</em> 2012</a>). One can surmise that environmentally-driven heteroskedasticity is likely present in this context as well, though the full extent of it remains unclear. Predictive performance of WGP models for human traits can be low, mostly due to factors unique to human populations, such as unrelatedness of individuals and short LD patterns (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">de los Campos <em>et al.</em> 2013b</a>). Extending WGP models for complex human traits to explicitly model heterogeneous residual variances across environments could potentially help account for still-unexplained variance, and thus affect the extent of missing heritability in human populations.</p><p id="p-60">Finally, WGP models often require estimation of a large number of SNP marker effects and considerations for computing efficiency become paramount, particularly since MCMC inference can be computationally expensive. Computational enhancements for homoskedastic WGP models have been developed based on expectation-maximization (EM) based algorithms (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hayashi and Iwata 2010</a>), or analytically derived posterior densities of each marker effect (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Meuwissen <em>et al.</em> 2009</a>). Heteroskedastic error extensions to WGP models, such as those presented here, could also be further modified to enhance computational efficiency. For example, an EM-like algorithm such as that proposed by <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gianola <em>et al.</em> (1992)</a> may be adapted to obtain empirical Bayes estimates of environment-specific variances in a WGP context.</p><div id="sec-14" class="subsection"><h3>Conclusions</h3><p id="p-61">In this study, we describe extensions to classical whole-genome prediction models that incorporate modeling of heterogeneous residual variances across environments and evaluate potential impact of specifying heteroskedastic <em>vs.</em> homoskedastic error models on the accuracy of prediction of genomic breeding values. Heteroskedastic error models were overwhelmingly supported by improved global fit to the data. The advantages of heteroskedastic error WGP models on overall predictive ability of carcass traits in pigs was of small magnitude, if at all present; however, considerable reranking of individuals with extreme genetic merit was observed when heteroskedasticity was explicitly accounted for. Heteroskedastic error WGP modeling should be carefully considered in breeding programs as environmental residual heteroskedasticity seems prevalent and, if unaccounted, can have considerable practical implications for selection of individuals of extreme genetic merit. Additional work tackling simultaneous modeling of heterogeneous genetic variances jointly with heterogeneous error variances and potential outlier robustness extensions is warranted.</p></div></div><div class="section" id="sec-15"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">Wenzhao Yang and the MSU Department of Animal Science are gratefully acknowledged for facilitating partial R code and for preparing the data used in this study. This project was supported by Agriculture and Food Research Initiative Competitive Grant no. 2011-67015-30338 from the U.S. Department of Agriculture National Institute of Food and Agriculture. Computing for this project was performed on the Beocat Research Cluster at Kansas State University, which is funded in part by NSF grants CNS-1006860, EPS-1006860, and EPS-0919443. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-4"><p id="p-63">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022897/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022897/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-64"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-05-27"><span class="received-label">Received </span>May 27, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-10-14"><span class="accepted-label">Accepted </span>October 14, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Ou <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.1" data-doi="10.2307/2290133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Computational methods using a Bayesian hierarchical generalized linear model.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D83%26rft.spage%253D1037%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2290133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2290133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.2" data-doi="10.1186/1471-2164-13-24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badke</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwab</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steibel</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimation of linkage disequilibrium in four US pig breeds.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBadke%26rft.auinit1%253DY.%2BM.%26rft.volume%253D13%26rft.spage%253D24%26rft.epage%253D24%26rft.atitle%253DEstimation%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2Bfour%2BUS%2Bpig%2Bbreeds.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-24%26rft_id%253Dinfo%253Apmid%252F22252454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-24&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22252454&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.3" data-doi="10.1534/g3.114.010504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badke</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fix</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steibel</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Accuracy of estimation of genomic breeding values in pigs using low-density genotypes and imputation.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.010504%26rft_id%253Dinfo%253Apmid%252F24531728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvNC82MjMiO3M6NDoiYXRvbSI7czoxNToiL2dnZy82LzEvMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.4" data-doi="10.1002/bimj.201100055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bello</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steibel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hierarchical Bayesian modeling of heterogeneous cluster- and subject-level associations between continuous and binary outcomes in dairy production.</span> <abbr class="cit-jnl-abbrev">Biom. J.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">230</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiom%2BJ%26rft.aulast%253DBello%26rft.auinit1%253DN.%2BM.%26rft.volume%253D54%26rft.issue%253D2%26rft.spage%253D230%26rft.epage%253D248%26rft.atitle%253DHierarchical%2BBayesian%2Bmodeling%2Bof%2Bheterogeneous%2Bcluster-%2Band%2Bsubject-level%2Bassociations%2Bbetween%2Bcontinuous%2Band%2Bbinary%2Boutcomes%2Bin%2Bdairy%2Bproduction.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbimj.201100055%26rft_id%253Dinfo%253Apmid%252F22522379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bimj.201100055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22522379&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.5" data-doi="10.2527/jas.2013-6757"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boddhireddy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Northcutt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prayaga</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rumph</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic predictions in angus cattle: comparisons of sample size, response variables, and clustering methods for cross-validation.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2013-6757%26rft_id%253Dinfo%253Apmid%252F24431338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2013-6757&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24431338&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.6" data-doi="10.1534/genetics.107.080838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accuracy of genomic selection using different methods to define haplotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%2BL.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D553%26rft.epage%253D561%26rft.atitle%253DAccuracy%2Bof%2BGenomic%2BSelection%2BUsing%2BDifferent%2BMethods%2Bto%2BDefine%2BHaplotypes%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080838%26rft_id%253Dinfo%253Apmid%252F18202394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTUzIjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNi8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cardoso</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Multiple-breed genetic inference using heavy-tailed structural models for heterogeneous residual variances.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">1766</span>–<span class="cit-lpage">1779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DCardoso%26rft.auinit1%253DF.%2BF.%26rft.volume%253D83%26rft.issue%253D8%26rft.spage%253D1766%26rft.epage%253D1779%26rft.atitle%253DMultiple-breed%2Bgenetic%2Binference%2Busing%2Bheavy-tailed%2Bstructural%2Bmodels%2Bfor%2Bheterogeneous%2Bresidual%2Bvariances%26rft_id%253Dinfo%253Apmid%252F16024695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16024695&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.8" data-doi="10.2527/jas.2006-668"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cardoso</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Accounting for outliers and heteroskedasticity in multibreed genetic evaluations of postweaning gain of Nelore-Hereford cattle.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">909</span>–<span class="cit-lpage">918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2006-668%26rft_id%253Dinfo%253Apmid%252F17178804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2006-668&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17178804&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.9" data-doi="10.2527/jas.2012-5543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cernicchiaro</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Renter</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bello</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Hierarchical Bayesian modeling of heterogeneous variances in average daily weight gain of commercial feedlot cattle.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">2910</span>–<span class="cit-lpage">2919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2012-5543%26rft_id%253Dinfo%253Apmid%252F23482583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2012-5543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23482583&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schenkel</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multi-population genomic prediction using a multi-task Bayesian learning model.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D53%26rft_id%253Dinfo%253Apmid%252F24884927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24884927&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.1.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ciobanu</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonergan</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huff-Lonergan</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetics of meat quality and carcass traits</span>, pp. <span class="cit-fpage">355</span>–<span class="cit-lpage">389</span> in <span class="cit-source">Genetics of the Pig</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Ruvinsky</span>  <span class="cit-name-given-names">A. </span></span>, <span class="cit-ed"><span class="cit-name-surname">Rothschild</span>  <span class="cit-name-given-names">M. F.</span></span>. <span class="cit-publ-name">CABI</span>, <span class="cit-publ-loc">UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.1.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colosimo</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Del Castillo</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source"><em>Bayesian Process Monitoring</em>, <em>Control and Optimization</em></span>, <span class="cit-publ-name">Chapman and Hall/CRC</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.13" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNi8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.14" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNi8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.15" data-doi="10.1371/journal.pone.0040964"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allison</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Prediction of expected years of life using whole-genome markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e40964</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253Dde%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De40964%26rft.epage%253De40964%26rft.atitle%253DPrediction%2Bof%2Bexpected%2Byears%2Bof%2Blife%2Busing%2Bwhole-genome%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0040964%26rft_id%253Dinfo%253Apmid%252F22848416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0040964&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22848416&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.16" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNi8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.17" data-doi="10.1371/journal.pgen.1003608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Prediction of complex human traits using the genomic best linear unbiased predictor.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003608%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003608%26rft_id%253Dinfo%253Apmid%252F23874214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874214&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.18" data-doi="10.1534/genetics.109.104935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reliability of genomic predictions across multiple populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104935%26rft_id%253Dinfo%253Apmid%252F19822733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My80LzE1NDUiO3M6NDoiYXRvbSI7czoxNToiL2dnZy82LzEvMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.19" data-doi="10.2527/jas.2006-625"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raney</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">a</span> <span class="cit-article-title">Quantitative trait loci mapping in an F2 Duroc x Pietrain resource population: I. Growth traits.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2006-625%26rft_id%253Dinfo%253Apmid%252F17965327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2006-625&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17965327&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253443800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.20" data-doi="10.2527/jas.2006-626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raney</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doumit</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoge</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">b</span> <span class="cit-article-title">Quantitative trait locus mapping in an F2 Duroc x Pietrain resource population: II. Carcass and meat quality traits.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">254</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2006-626%26rft_id%253Dinfo%253Apmid%252F17965326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2006-626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17965326&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.21" data-doi="10.2135/cropsci2005.0164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Bayesian modeling of heterogeneous error and genotype × environment interaction variances.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">820</span>–<span class="cit-lpage">833</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DEdwards%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D820%26rft.epage%253D833%26rft.atitle%253DBayesian%2BModeling%2Bof%2BHeterogeneous%2BError%2Band%2BGenotype%2Bx%2BEnvironment%2BInteraction%2BVariances%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.0164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.0164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000235991100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.22" data-doi="10.1186/1297-9686-28-3-249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foulley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Statistical analysis of ordered categorical data via a structural heteroskedastic threshold model.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D28%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-28-3-249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-28-3-249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanVleck</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Aspects of selection for performance in several environments with heterogeneous variances.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D65%26rft.issue%253D2%26rft.spage%253D409%26rft.epage%253D421%26rft.atitle%253DAspects%2Bof%2BSelection%2Bfor%2BPerformance%2Bin%2BSeveral%2BEnvironments%2Bwith%2BHeterogeneous%2BVariances%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.24" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.1.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelfand</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Model determination using sampling-based methods</span>, pp. <span class="cit-fpage">145</span>–<span class="cit-lpage">161</span> in <span class="cit-source">Markov Chain Monte Carlo in Practice</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Gilks</span>  <span class="cit-name-given-names">W. R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Spiegelhalter</span>  <span class="cit-name-given-names">D.</span></span>. <span class="cit-publ-name">Chapman and Hall/CRC</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.26" data-doi="10.1214/06-BA117A"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Prior distributions for variance parameters in hierarchical models.</span> <abbr class="cit-jnl-abbrev">Bayesian Anal.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBayesian%2BAnal.%26rft.volume%253D1%26rft.spage%253D515%26rft_id%253Dinfo%253Adoi%252F10.1214%252F06-BA117A%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/06-BA117A&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000207447000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.27" data-doi="10.1214/ss/1177011136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Inference from iterative simulation using multiple sequences.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D7%26rft.spage%253D457%26rft_id%253Dinfo%253Adoi%252F10.1214%252Fss%252F1177011136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/ss/1177011136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.28" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: the Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNi8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.29" data-doi="10.1146/annurev-animal-022114-110733"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">One hundred years of statistical developments in animal breeding.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Anim. Biosci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BAnim.%2BBiosci.%26rft.volume%253D3%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-022114-110733%26rft_id%253Dinfo%253Apmid%252F25387231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-022114-110733&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25387231&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.30" data-doi="10.3168/jds.S0022-0302(92)78044-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimation of heterogeneous variances using empirical Bayes methods: theoretical considerations.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">2805</span>–<span class="cit-lpage">2823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D75%26rft.issue%253D10%26rft.spage%253D2805%26rft.epage%253D2823%26rft.atitle%253DEstimation%2Bof%2BHeterogeneous%2BVariances%2BUsing%2BEmpirical%2BBayes%2BMethods%253A%2BTheoretical%2BConsiderations%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252892%252978044-8%26rft_id%253Dinfo%253Apmid%252F1430485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(92)78044-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1430485&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.31" data-doi="10.1534/genetics.109.103952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Additive genetic variability and the Bayesian alphabet.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103952%26rft_id%253Dinfo%253Apmid%252F19620397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNi8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.32" data-doi="10.1214/09-STS306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimating effects and making predictions from genome-wide marker data.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D24%26rft.spage%253D517%26rft_id%253Dinfo%253Adoi%252F10.1214%252F09-STS306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/09-STS306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000277257000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gualdron Duarte</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raney</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantet</span>  <span class="cit-name-given-names">R. J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genotype imputation accuracy in a F2 pig population using high density and low density SNP panels.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D14%26rft.spage%253D38%26rft_id%253Dinfo%253Apmid%252F23651538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23651538&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.34" data-doi="10.1186/1471-2105-15-246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gualdron Duarte</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantet</span>  <span class="cit-name-given-names">R. J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raney</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Rapid screening for phenotype-genotype associations by linear transformations of genomic evaluations.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D246%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-246%26rft_id%253Dinfo%253Apmid%252F25038782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25038782&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.35" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the Bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">EM algorithm for Bayesian estimation of genomic breeding values.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DHayashi%26rft.auinit1%253DT.%26rft.volume%253D11%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DEM%2Balgorithm%2Bfor%2BBayesian%2Bestimation%2Bof%2Bgenomic%2Bbreeding%2Bvalues.%26rft_id%253Dinfo%253Apmid%252F20092655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20092655&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.37" data-doi="10.1051/gse:2001117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The distribution of the effects of genes affecting quantitative traits in livestock.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D209%26rft.epage%253D229%26rft.atitle%253DThe%2Bdistribution%2Bof%2Bthe%2Beffects%2Bof%2Bgenes%2Baffecting%2Bquantitative%2Btraits%2Bin%2Blivestock.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2001117%26rft_id%253Dinfo%253Apmid%252F11403745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2001117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11403745&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169502100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.38" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: Genomic selection in dairy cattle: progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.39" data-doi="10.1287/opre.31.6.1109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heidelberger</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Simulation run length control in the presence of an initial transient.</span> <abbr class="cit-jnl-abbrev">Oper. Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1109</span>–<span class="cit-lpage">1144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOperations%2BResearch%26rft.stitle%253DOperations%2BResearch%26rft.aulast%253DHeidelberger%26rft.auinit1%253DP.%26rft.volume%253D31%26rft.issue%253D6%26rft.spage%253D1109%26rft.epage%253D1144%26rft.atitle%253DSimulation%2BRun%2BLength%2BControl%2Bin%2Bthe%2BPresence%2Bof%2Ban%2BInitial%2BTransient%26rft_id%253Dinfo%253Adoi%252F10.1287%252Fopre.31.6.1109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1287/opre.31.6.1109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983SA75000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.40" data-doi="10.1017/S0003356100032220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">On selection among groups with heterogeneous variance.</span> <abbr class="cit-jnl-abbrev">Anim. Prod.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BProd.%26rft.volume%253D39%26rft.spage%253D473%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0003356100032220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0003356100032220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.41" data-doi="10.1017/S0016672310000546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic analysis of environmental variation.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D92%26rft.issue%253D5-6%26rft.spage%253D381%26rft.epage%253D395%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Benvironmental%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000546%26rft_id%253Dinfo%253Apmid%252F21429270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429270&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.42" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du Cheyron</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.43" data-doi="10.2307/2291091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kass</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raftery</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Bayes factors.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D90%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2291091%26rft_id%253Dinfo%253Apmid%252F10783804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2291091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10783804&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RA10400045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.44" data-doi="10.2307/2685466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kass</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neal</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Markov chain Monte Carlo in practice: a roundtable discussion.</span> <abbr class="cit-jnl-abbrev">Am. Stat.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BStat.%26rft.volume%253D52%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2685466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2685466&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000074022800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.45" data-doi="10.1186/1297-9686-37-1-31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A general approach to mixed effects modeling of residual variances in generalized linear mixed models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DKizilkaya%26rft.auinit1%253DK.%26rft.volume%253D37%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D56%26rft.atitle%253DA%2Bgeneral%2Bapproach%2Bto%2Bmixed%2Beffects%2Bmodeling%2Bof%2Bresidual%2Bvariances%2Bin%2Bgeneralized%2Blinear%2Bmixed%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-37-1-31%26rft_id%253Dinfo%253Apmid%252F15588567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-37-1-31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15588567&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225793700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.46" data-doi="10.3168/jds.S0022-0302(95)76832-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Biases in predicted transmitting abilities of sires when daughters receive preferential treatment.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">2067</span>–<span class="cit-lpage">2072</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DKuhn%26rft.auinit1%253DM.%2BT.%26rft.volume%253D78%26rft.issue%253D9%26rft.spage%253D2067%26rft.epage%253D2072%26rft.atitle%253DBiases%2Bin%2BPredicted%2BTransmitting%2BAbilities%2Bof%2BSires%2BWhen%2BDaughters%2BReceive%2BPreferential%2BTreatment%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252895%252976832-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(95)76832-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.47" data-doi="10.1534/g3.114.016097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Increased prediction accuracy in wheat breeding trials using a marker x environment interaction genomic selection model.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.016097%26rft_id%253Dinfo%253Apmid%252F25660166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvNC81NjkiO3M6NDoiYXRvbSI7czoxNToiL2dnZy82LzEvMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.48" data-doi="10.1146/annurev-animal-031412-103705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accelerating improvement of livestock with genomic selection.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Anim. Biosci.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BAnim.%2BBiosci.%26rft.volume%253D1%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-031412-103705%26rft_id%253Dinfo%253Apmid%252F25387018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-031412-103705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25387018&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNToiL2dnZy82LzEvMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.50" data-doi="10.1186/1297-9686-41-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shepherd</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A fast algorithm for BayesB type of prediction of genome-wide estimates of genetic value.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%26rft.volume%253D41%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DA%2Bfast%2Balgorithm%2Bfor%2BBayesB%2Btype%2Bof%2Bprediction%2Bof%2Bgenome-wide%2Bestimates%2Bof%2Bgenetic%2Bvalue.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-2%26rft_id%253Dinfo%253Apmid%252F19284681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19284681&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.51" data-doi="10.1186/1297-9686-43-38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henryon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Deregressed EBV as the response variable yield more reliable genomic predictions than traditional EBV in pure-bred pigs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DOstersen%26rft.auinit1%253DT.%26rft.volume%253D43%26rft.spage%253D38%26rft.epage%253D38%26rft.atitle%253DDeregressed%2BEBV%2Bas%2Bthe%2Bresponse%2Bvariable%2Byield%2Bmore%2Breliable%2Bgenomic%2Bpredictions%2Bthan%2Btraditional%2BEBV%2Bin%2Bpure-bred%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-38%26rft_id%253Dinfo%253Apmid%252F22070746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22070746&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paré</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ridker</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chasman</span>  <span class="cit-name-given-names">D. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">On the use of variance per genotype as a tool to identify quantitative trait interaction effects: a report from the Women’s Genome Health Study.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: e1000981.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plummer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowles</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vines</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">CODA: convergence diagnostics and output analysis for MCMC.</span> <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D6%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.1.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raftery</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">How many iterations in the Gibbs sampler</span>, pp. <span class="cit-fpage">763</span>–<span class="cit-lpage">773</span> in <span class="cit-source">Bayesian Statistics 4</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">J. O.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Dawid</span>  <span class="cit-name-given-names">A. P.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. F. M.</span></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford, U.K.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.55" data-doi="10.1534/genetics.111.127068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Detecting major genetic loci controlling phenotypic variability in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127068%26rft_id%253Dinfo%253Apmid%252F21467569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzIvNDM1IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNi8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recent developments in statistical methods for detecting genetic loci affecting phenotypic variability.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DRonnegard%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.spage%253D63%26rft.epage%253D63%26rft.atitle%253DRecent%2Bdevelopments%2Bin%2Bstatistical%2Bmethods%2Bfor%2Bdetecting%2Bgenetic%2Bloci%2Baffecting%2Bphenotypic%2Bvariability.%26rft_id%253Dinfo%253Apmid%252F22827487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22827487&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.57" data-doi="10.1051/gse:2001101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">SanCristobal-Gaudy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bodin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chevalet</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic components of litter size variability in sheep.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DSanCristobal-Gaudy%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D249%26rft.epage%253D271%26rft.atitle%253DGenetic%2Bcomponents%2Bof%2Blitter%2Bsize%2Bvariability%2Bin%2Bsheep.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2001101%26rft_id%253Dinfo%253Apmid%252F11403747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2001101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11403747&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">See</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Heterogeneity of (co)variance among herds for backfat measures of swine.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">2568</span>–<span class="cit-lpage">2574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DSee%26rft.auinit1%253DM.%2BT.%26rft.volume%253D76%26rft.issue%253D10%26rft.spage%253D2568%26rft.epage%253D2574%26rft.atitle%253DHeterogeneity%2Bof%2B%2528co%2529variance%2Bamong%2Bherds%2Bfor%2Bbackfat%2Bmeasures%2Bof%2Bswine%26rft_id%253Dinfo%253Apmid%252F9814895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9814895&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.59" data-doi="10.1111/j.1745-4573.1994.tb00530.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sellier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monin</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetics of pig meat quality: a review.</span> <abbr class="cit-jnl-abbrev">J. Muscle Foods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMuscle%2BFoods%26rft.volume%253D5%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1745-4573.1994.tb00530.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1745-4573.1994.tb00530.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.60" data-doi="10.1051/gse:19990102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stranden</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Mixed effects linear models with <em>t</em>-distributions for quantitative genetic analysis: a Bayesian approach.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D31%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19990102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19990102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Struchalin</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehghan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witteman</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Duijn</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>  <span class="cit-article-title">Variance heterogeneity analysis for detection of potentially interacting genetic loci: method and its limitations.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DStruchalin%26rft.auinit1%253DM.%2BV.%26rft.volume%253D11%26rft.spage%253D92%26rft.epage%253D92%26rft.atitle%253DVariance%2Bheterogeneity%2Banalysis%2Bfor%2Bdetection%2Bof%2Bpotentially%2Binteracting%2Bgenetic%2Bloci%253A%2Bmethod%2Band%2Bits%2Blimitations.%26rft_id%253Dinfo%253Apmid%252F20942902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20942902&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.62" data-doi="10.1534/genetics.112.141705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A comprehensive genetic approach for improving prediction of skin cancer risk in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1493</span>–<span class="cit-lpage">1502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141705%26rft_id%253Dinfo%253Apmid%252F23051645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE0OTMiO3M6NDoiYXRvbSI7czoxNToiL2dnZy82LzEvMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.63" data-doi="10.1086/429864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wigginton</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutler</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A note on exact tests of Hardy-Weinberg equilibrium.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">887</span>–<span class="cit-lpage">893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWigginton%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D76%26rft.issue%253D5%26rft.spage%253D887%26rft.epage%253D893%26rft.atitle%253DA%2Bnote%2Bon%2Bexact%2Btests%2Bof%2BHardy-Weinberg%2Bequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F429864%26rft_id%253Dinfo%253Apmid%252F15789306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/429864&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15789306&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228198300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.64" data-doi="10.1534/genetics.111.131540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A Bayesian antedependence model for whole genome prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1491</span>–<span class="cit-lpage">1501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131540%26rft_id%253Dinfo%253Apmid%252F22135352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE0OTEiO3M6NDoiYXRvbSI7czoxNToiL2dnZy82LzEvMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.65" data-doi="10.1186/s12711-015-0092-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Improving the computational efficiency of fully Bayes inference and assessing the effect of misspecification of hyperparameters in whole-genome prediction models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0092-x%26rft_id%253Dinfo%253Apmid%252F25885894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0092-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25885894&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.1.66" data-doi="10.1017/S0016672311000012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Use of genomic models to study genetic control of environmental variance.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D93%26rft.spage%253D125%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672311000012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672311000012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/1/1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/12/2923" title="G3 Reviewer Index 2015" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/15" title="Comparative Genomics of an Emerging Amphibian Virus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/1"><span>Volume 6 Issue 1, January 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/1.cover-source.jpg?itok=9zFiZvCK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/1.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qVfpvcYGAIPnY2h1zJKXSVLbqV-5_6qFOBVfJqyOtvA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections vert-borders vert-padding increased-bottom-margin" >
  
        <h2 class="pane-title">SUBJECTS</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomic-selection" class="highlight" data-icon-position="" data-hide-link-title="0">Genomic Selection<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/1/1" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/1/1" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/1/1" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction Accounting for Residual Heteroskedasticity</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460245" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-6KRkWfRQcsgnZgggJHlWkedw-yxPfDtqU1PaneNRQq0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/1.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/1/1" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-DaX1FRq2fhy2p84L2KFLBhyTIkBmwJS8XiJyDTmmA1Q" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/1/1" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/1.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460245" id="node460245--2" data-pisa="ggg;6/1/1" data-pisa-master="ggg;g3.115.022897" data-apath="/ggg/6/1/1.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction Accounting for Residual Heteroskedasticity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhining Ou</span>, <span class="highwire-citation-author" data-delta="1">Robert J. Tempelman</span>, <span class="highwire-citation-author" data-delta="2">Juan P. Steibel</span>, <span class="highwire-citation-author" data-delta="3">Catherine W. Ernst</span>, <span class="highwire-citation-author" data-delta="4">Ronald O. Bates</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-3902-1533" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nora M. Bello</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1-13;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022897 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460245--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhining Ou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhining%2BOu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ou%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhining%2BOu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert J. Tempelman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BTempelman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tempelman%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BTempelman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juan P. Steibel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Fisheries and Wildlife, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BP.%2BSteibel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steibel%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BP.%2BSteibel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Catherine W. Ernst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BW.%2BErnst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ernst%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BW.%2BErnst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ronald O. Bates </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronald%2BO.%2BBates%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bates%20RO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonald%2BO.%2BBates%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nora M. Bello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nora%2BM.%2BBello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bello%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANora%2BM.%2BBello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3902-1533" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nora M. Bello</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nbello@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">nbello@ksu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/1/1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/1/1&amp;title=Genomic%20Prediction%20Accounting%20for%20Residual%20Heteroskedasticity" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/1/1&amp;title=Genomic%20Prediction%20Accounting%20for%20Residual%20Heteroskedasticity" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/1/1&amp;title=Genomic%20Prediction%20Accounting%20for%20Residual%20Heteroskedasticity" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/1/1&amp;text=Genomic%20Prediction%20Accounting%20for%20Residual%20Heteroskedasticity" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/1/1&amp;title=Genomic%20Prediction%20Accounting%20for%20Residual%20Heteroskedasticity" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/1/1&amp;t=Genomic%20Prediction%20Accounting%20for%20Residual%20Heteroskedasticity" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/1/1&amp;title=Genomic%20Prediction%20Accounting%20for%20Residual%20Heteroskedasticity" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/1/1&amp;title=Genomic%20Prediction%20Accounting%20for%20Residual%20Heteroskedasticity" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460245/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460245" id="node460245--3" data-pisa="ggg;6/1/1" data-pisa-master="ggg;g3.115.022897" data-apath="/ggg/6/1/1.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction Accounting for Residual Heteroskedasticity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhining Ou</span>, <span class="highwire-citation-author" data-delta="1">Robert J. Tempelman</span>, <span class="highwire-citation-author" data-delta="2">Juan P. Steibel</span>, <span class="highwire-citation-author" data-delta="3">Catherine W. Ernst</span>, <span class="highwire-citation-author" data-delta="4">Ronald O. Bates</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-3902-1533" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nora M. Bello</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1-13;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022897 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460245--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhining Ou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhining%2BOu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ou%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhining%2BOu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert J. Tempelman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BTempelman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tempelman%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BTempelman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juan P. Steibel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Fisheries and Wildlife, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BP.%2BSteibel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steibel%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BP.%2BSteibel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Catherine W. Ernst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BW.%2BErnst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ernst%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BW.%2BErnst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ronald O. Bates </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronald%2BO.%2BBates%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bates%20RO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonald%2BO.%2BBates%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nora M. Bello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nora%2BM.%2BBello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bello%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANora%2BM.%2BBello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3902-1533" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nora M. Bello</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nbello@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">nbello@ksu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1&amp;rft.epage=13&amp;rft.atitle=Genomic%20Prediction%20Accounting%20for%20Residual%20Heteroskedasticity&amp;rft.volume=6&amp;rft.issue=1&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ou%2C+Zhining&amp;rft.au=Tempelman%2C+Robert+J.&amp;rft.au=Steibel%2C+Juan+P.&amp;rft.au=Ernst%2C+Catherine+W.&amp;rft.au=Bates%2C+Ronald+O.&amp;rft.au=Bello%2C+Nora+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460245/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460245/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460245/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460245/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460245/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460245/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460245/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460245/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460245/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460245/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460245/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460245/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/1/1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/1/1.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/1/1.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/1/1.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/1/1.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
