<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/1/79" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/1/79.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/1/79.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/1/79.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/1/79/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/1/79.atom" />
<meta name="HW.pisa" content="ggg;6/1/79" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mapping Quantitative Trait Loci Underlying Function-Valued Traits Using Functional Principal Component Analysis and Multi-Trait Mapping" />
<meta name="DC.Identifier" content="10.1534/g3.115.024133" />
<meta name="DC.Date" content="2016-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Kwak et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We previously proposed a simple regression-based method to map quantitative trait loci underlying function-valued phenotypes. In order to better handle the case of noisy phenotype measurements and accommodate the correlation structure among time points, we propose an alternative approach that maintains much of the simplicity and speed of the regression-based method. We overcome noisy measurements by replacing the observed data with a smooth approximation. We then apply functional principal component analysis, replacing the smoothed phenotype data with a small number of principal components. Quantitative trait locus mapping is applied to these dimension-reduced data, either with a multi-trait method or by considering the traits individually and then taking the average or maximum LOD score across traits. We apply these approaches to root gravitropism data on Arabidopsis recombinant inbred lines and further investigate their performance in computer simulations. Our methods have been implemented in the R package, funqtl." />
<meta name="DC.Contributor" content="Il-Youp Kwak" />
<meta name="DC.Contributor" content="Candace R. Moore" />
<meta name="DC.Contributor" content="Edgar P. Spalding" />
<meta name="DC.Contributor" content="Karl W. Broman" />
<meta name="article:published_time" content="2016-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Mapping Quantitative Trait Loci Underlying Function-Valued Traits Using Functional Principal Component Analysis and Multi-Trait Mapping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We previously proposed a simple regression-based method to map quantitative trait loci underlying function-valued phenotypes. In order to better handle the case of noisy phenotype measurements and accommodate the correlation structure among time points, we propose an alternative approach that maintains much of the simplicity and speed of the regression-based method. We overcome noisy measurements by replacing the observed data with a smooth approximation. We then apply functional principal component analysis, replacing the smoothed phenotype data with a small number of principal components. Quantitative trait locus mapping is applied to these dimension-reduced data, either with a multi-trait method or by considering the traits individually and then taking the average or maximum LOD score across traits. We apply these approaches to root gravitropism data on Arabidopsis recombinant inbred lines and further investigate their performance in computer simulations. Our methods have been implemented in the R package, funqtl.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="ggg;6/1/79" />
<meta name="citation_id" content="6/1/79" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/1/79" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/1/79.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/1/79.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/1/79.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.024133" />
<meta name="citation_pmid" content="26530421" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="79" />
<meta name="citation_lastpage" content="86" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Il-Youp Kwak" />
<meta name="citation_author_institution" content="Department of Statistics, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7117-7669" />
<meta name="citation_author" content="Candace R. Moore" />
<meta name="citation_author_institution" content="Department of Botany, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Edgar P. Spalding" />
<meta name="citation_author_institution" content="Department of Botany, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6890-4765" />
<meta name="citation_author" content="Karl W. Broman" />
<meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author_email" content="kbroman@biostat.wisc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4914-6671" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. L. Araus;citation_author=J. E. Cairns;citation_title=Field high-throughput phenotyping: the new crop breeding frontier.;citation_pages=52-61;citation_volume=19;citation_year=2014;citation_pmid=24139902;citation_doi=10.1016/j.tplants.2013.09.008" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=K. W. Broman;citation_author=T. P. Speed;citation_title=A model selection approach for the identification of quantitative trait loci in experimental crosses (with discussion).;citation_pages=641-656, 737–775;citation_volume=64;citation_year=2002;citation_doi=10.1111/1467-9868.00354" />
<meta name="citation_reference" content="citation_journal_abbrev=J Integr Plant Biol;citation_author=L. Cabrera-Bosquet;citation_author=J. Crossa;citation_author=J. von Zitzewitz;citation_author=M. D. Serret;citation_author=J. Luis Araus;citation_title=High-throughput phenotyping and genomic selection: the frontiers of crop breeding converge.;citation_pages=312-320;citation_volume=54;citation_year=2012;citation_issue=5;citation_pmid=22420640;citation_doi=10.1111/j.1744-7909.2012.01116.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=M. E. Ghanem;citation_author=H. Marrou;citation_author=T. R. Sinclair;citation_title=Physiological phenotyping of plants for crop improvement.;citation_pages=139-144;citation_volume=20;citation_year=2015;citation_pmid=25524213;citation_doi=10.1016/j.tplants.2014.11.006" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. S. Haley;citation_author=S. A. Knott;citation_title=A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.;citation_pages=315-324;citation_volume=69;citation_year=1992;citation_issue=4;citation_pmid=16718932;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. A. Knott;citation_author=C. S. Haley;citation_title=Multitrait Least Squares for Quantitative Trait Loci Detection;citation_pages=899-911;citation_volume=156;citation_year=2000;citation_issue=2;citation_pmid=11014835" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=I.-Y. Kwak;citation_author=C. R. Moore;citation_author=E. P. Spalding;citation_author=K. W. Broman;citation_volume=197;citation_year=2014;citation_pmid=24931408;citation_doi=10.1534/genetics.114.166306" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Li;citation_author=M. J. Sillanpää;citation_title=A Bayesian nonparametric approach for mapping dynamic quantitative traits.;citation_volume=194;citation_year=2013;citation_pmid=23770698;citation_doi=10.1534/genetics.113.152736" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Ma;citation_author=G. Casella;citation_author=R. L. Wu;citation_title=Functional Mapping of Quantitative Trait Loci Underlying the Character Process: A Theoretical Framework;citation_pages=1751-1762;citation_volume=161;citation_year=2002;citation_issue=4;citation_pmid=12196415" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Manichaikul;citation_author=J. Y. Moon;citation_author=S. Sen;citation_author=B. S. Yandell;citation_author=K. W. Broman;citation_title=A model selection approach for the identification of quantitative trait loci in experimental crosses, allowing epistasis.;citation_volume=181;citation_year=2009;citation_pmid=19104078;citation_doi=10.1534/genetics.108.094565" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=L. Min;citation_author=R. Yang;citation_author=X. Wang;citation_author=B. Wang;citation_title=Bayesian analysis for genetic architecture of dynamic traits.;citation_pages=124-133;citation_volume=106;citation_year=2011;citation_issue=1;citation_pmid=20332806;citation_doi=10.1038/hdy.2010.20" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. R. Moore;citation_author=L. S. Johnson;citation_author=I.-Y. Kwak;citation_author=M. Livny;citation_author=K. W. Broman;citation_title=High-throughput computer vision introduces the time axis to a quantitative trait map of a plant growth response.;citation_volume=195;citation_year=2013;citation_pmid=23979570;citation_doi=10.1534/genetics.113.153346" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;citation_year=2015" />
<meta name="citation_reference" content="citation_title=Functional data analysis;citation_author=J. Ramsay;citation_author=B. W. Silverman;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=M. J. Sillanpää;citation_author=P. Pikkuhookana;citation_author=S. Abrahamsson;citation_author=T. Knurr;citation_author=A. Fries;citation_title=Simultaneous estimation of multiple quantitative trait loci and growth curve parameters through hierarchical Bayesian modeling.;citation_pages=134-146;citation_volume=108;citation_year=2012;citation_issue=2;citation_pmid=21792229;citation_doi=10.1038/hdy.2011.56" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Xiong;citation_author=E. H. Goulding;citation_author=E. J. Carlson;citation_author=L. H. Tecott;citation_author=C. E. McCulloch;citation_title=A flexible estimating equations approach for mapping function-valued traits.;citation_volume=189;citation_year=2011;citation_pmid=21705752;citation_doi=10.1534/genetics.111.129221" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=J. Yang;citation_author=R. L. Wu;citation_author=G. Casella;citation_title=Nonparametric functional mapping of quantitative trait loci.;citation_pages=30-39;citation_volume=65;citation_year=2009;citation_pmid=18537950;citation_doi=10.1111/j.1541-0420.2008.01063.x" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=J. S. Yap;citation_author=J. Fan;citation_author=R. Wu;citation_title=Nonparametric modeling of longitudinal covariance structure in functional mapping of quantitative trait loci.;citation_pages=1068-1077;citation_volume=65;citation_year=2009;citation_pmid=19302406;citation_doi=10.1111/j.1541-0420.2009.01222.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. B. Zeng;citation_author=J. J. Liu;citation_author=L. F. Stam;citation_author=C. H. Kao;citation_author=J. M. Mercer;citation_title=Genetic Architecture of a Morphological Shape Difference Between Two Drosophila Species;citation_pages=299-310;citation_volume=154;citation_year=2000;citation_issue=1;citation_pmid=10628989" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mapping Quantitative Trait Loci Underlying Function-Valued Traits Using Functional Principal Component Analysis and Multi-Trait Mapping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="twitter:description" content="We previously proposed a simple regression-based method to map quantitative trait loci underlying function-valued phenotypes. In order to better handle the case of noisy phenotype measurements and accommodate the correlation structure among time points, we propose an alternative approach that maintains much of the simplicity and speed of the regression-based method. We overcome noisy measurements by replacing the observed data with a smooth approximation. We then apply functional principal component analysis, replacing the smoothed phenotype data with a small number of principal components. Quantitative trait locus mapping is applied to these dimension-reduced data, either with a multi-trait method or by considering the traits individually and then taking the average or maximum LOD score across traits. We apply these approaches to root gravitropism data on Arabidopsis recombinant inbred lines and further investigate their performance in computer simulations. Our methods have been implemented in the R package, funqtl." />
<meta name="og-title" property="og:title" content="Mapping Quantitative Trait Loci Underlying Function-Valued Traits Using Functional Principal Component Analysis and Multi-Trait Mapping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/1/79" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We previously proposed a simple regression-based method to map quantitative trait loci underlying function-valued phenotypes. In order to better handle the case of noisy phenotype measurements and accommodate the correlation structure among time points, we propose an alternative approach that maintains much of the simplicity and speed of the regression-based method. We overcome noisy measurements by replacing the observed data with a smooth approximation. We then apply functional principal component analysis, replacing the smoothed phenotype data with a small number of principal components. Quantitative trait locus mapping is applied to these dimension-reduced data, either with a multi-trait method or by considering the traits individually and then taking the average or maximum LOD score across traits. We apply these approaches to root gravitropism data on Arabidopsis recombinant inbred lines and further investigate their performance in computer simulations. Our methods have been implemented in the R package, funqtl." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
    <title>Mapping Quantitative Trait Loci Underlying Function-Valued Traits Using Functional Principal Component Analysis and Multi-Trait Mapping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ua5JGXsKX_Lbmt2FLTxyC64UhftVY0a4PO3U1rFp5TY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460249","apath":"\/ggg\/6\/1\/79.atom","pisa":"ggg;6\/1\/79","ac":{"\/ggg\/6\/1\/79.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/1\/79.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/1\/79"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/1\/79"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/1\/79":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022r+pdvqCXDst4hpJYGDl66cj9QndNnuGIL2C8TT9zX3XZrOoR5Hz0soezxOgmpQ5VodLs2tsk+Q6O0sV1mb\\\\\\\/mCHujISTLE8\\\\\\\/7btRGhVFRAlYxDWihMJ7MHH2qCbjmTrFyqYANfrhvg5b\\\\\\\/6agrkaGial8+bL6dmDrnsEQwZQ1fLqtJ1LjKWYosYw2d+uqnPHu6eLXGbpmHze3XVx3I7cx36n87IsdqxF7+4iu+n1dHWWwH0rTKtiNHR0OVc5+B4DMqZMn\\\\\\\/Bipky39h3n48EJCop7PRMRmaIbL5HdRJF0ZjykGyN78RYnhMJu9yw+gm1f5F6Q7uH7l8sX3G\\\\\\\/Te6c5iYRxCPtAp0uHBQC8++7zp78NUdF8JgWZ+GLcbB6rAyHjskUdj32VxXANiOZ2fSLXwehJaeH9SF\\\\\\\/XIQ7Szwbxr+wvQ+196YFZR9YN5YEnj9wRTPzOZ+smifsEaz8K+1awIjgsuIGo3v8sDD+3UzEIbjKsnq4vtgNZaugMfnTUBYQ43v3HtGLQNmV1kn3KOsZpByK307UCuKsCRqSFhdFVcjHH8Z1yQYuQCMBEmyOl8u7Snj9A\\\\\\\/mwOqbuIAJabnx4XWH+bjLs0yTNDWmIkH\\\\\\\/M4zBKWNw8QtJ1kV1G9jq+W8ct3Xwuo\\\\\\\/R12SsEWasn3IkuT4cNuYzHhWujRe9mUTnqaWldRKbqSGO\\\\\\\/F66u7JhX5HYy8CCpNPAYyfZPseuluWTp+n+\\\\\\\/D860zYaR5srCFplGThpXWwd+R9AewtaFojaYg3tN74UVv1PhxEH79pjm1DNRyS8Pk1QFC\\\\\\\/u0EA3UbBZhdZFePI32+GR1Ewerkgt3AQo54QVpi+Ro+zvL1k1WW9GBY\\\\\\\/bhn09I2X3fj6fsHuljirskPRxqYsjxN87k48VRHvVqoBmw3NNH90YGRvUI\\\\\\\/KVJA2CfAiXOrkkdLPiNvz6v3U6SARLwSUekrmqzeCe6VzMcoMOz5v84o1YU6Be3chxQ\\\\\\\/LgQdoxZX+YBrMz\\\\\\\/MwZTxo+Z87UYoDZ6xR72cxCa4do1cBnNVsCJsny85cKBstSaBbHEzSAsx0KtIhZRBTgZqF6lVR0mi9dBmJhY32DNO024gZ5fZriuPDCIsLsM2YRkAJpl8djKHUgSLkwNrYxjbofeRsuH0tLr5eDKKp2ivO28bz2HV+XRzFOBOR4JtIO+CFsZ+c3i+aohRQ7vBSQC+XnTYm0o6ZTdlwB6+Ubu0VmMx8\\\\\\\/gOOyb7t6W3rTGPN014FVEJzbA8PUpwQCyIKZXDM4tX9jDvAFG7VTn\\\\\\\/1QotCQ1UPQ\\\u0022,\\\u0022iv\\\u0022:\\\u0022d8jVfTndLADUxe+cn9jZ5A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d41ac63c65e7e1736aa1f54a5ffd376886cb7a18afd7af50db9bf036300f56d7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YurAJGtq3gHpg3B7+WYwqpL5nkk4WW6jdH38\\\\\\\/\\\\\\\/\\\\\\\/LW\\\\\\\/5tHag0AZduWUGkKoUvvnm+Pm421mh0omjYYKsSRaiH+9FwlqNiAtOsA\\\\\\\/umMQfFpxIdI37l35NViSKiIt26kJgOJ8G4QCkoOSshA7Y4kpnXwDA2SBRF6WXtg2AzhA4dZuUPqfv6x6vOKnndEc3BaX8SMZZQ1Q0TUo+nMevei27DUzLLD\\\\\\\/PXJ4GoDkh9ygtb1iwc5ocO4ngWJjzuZWbJhzDwh7srLNPezizzlG724xc8dOKaAV1Hgtg+GHYafIQ6uTbH5br7uKXnGn18NGiGkO5O7fqq2vdNj0LT8ozCLtV4qrq\\\\\\\/C6QhcQH\\\\\\\/LaixIfjP1OuyhrB931fq+7gXr8ZBArp3iZOZ9OtNIcm+MbQCiywN+kmZ2upIim83qUWiCzb\\\\\\\/hlQwhKCZA6bYJvgUTIJvFTKzqnCazuegG136CNO\\\\\\\/8hCqCFpeoSyiey7u9JarallthxBkadiu6MFeNwIpBuxoltVJXgmyLJ9kzS2OshQZd2cKNarfhOt6hxK\\\\\\\/E46DNjFn4uohVekL0ac8E2VwsImny6bbQDXF4J01Qll2r3x2tkRR\\\\\\\/Vm28iQ\\\\\\\/Ztig+xnurp1Ths\\\\\\\/\\\\\\\/fAZG6fS73zjHNYdggr3Wro+r8YTNTMY8KAcxYvTVm+wdikckz4N2S0HbdDvuio5F\\\\\\\/7vwAeiyvQhDyxwD+aqzCYCmuRUtsQqvABpe+t6iN6DCuRtm3MTJG6k6tHESoNyC42nN0dIhL+F\\\\\\\/5WViINSXwQUr5uEIZgiR0OHsuc8zZBzCj0IVHqr6iWDiLsHklWN0brF94ujzO1eb+jUMvd+0spY041yduZC1CvPSzhSo\\\\\\\/gYT5oqKIab\\\\\\\/cP0Qj2oaXHGdXaV9n2OadIXCHsRpJP+AA7uF5jAA7iPQajsnDX7nbBSqlKrjxdlBqGB65oVqme+bwP5wou1qd32WNDA7uifx29ME2xEGFjhIEBPH6hQYCJOWvg6wk4k3zuam+gECKNaiAvdTtcjK4MLA1s3QS98upmG9oloabvWctCjIj1Zne16YlG0rpccWqUZEkoKmLJC31Mi1ezZUE0lLdO7nldYNbTLnJ5HTaac6T+JzBScLQfJF8rqZ48BvPdrOQdK1ekB2OFNi4FWqWpk4Wb7h41+A9HuUiuqqHjA\\\\\\\/xG0gj9I+ayene1U1jEoisiSZMLSlK8ySOAJtcYsy+\\\\\\\/SvnZAoWO5nk+hgdP\\\\\\\/14Rl6bSWqwgfQjyosfxKgsmOvdzlEfBkZ7F8UZdhlihxI1IRs9GXh1SBjlGIQNzHAajDCzdGHSbdDyYHY6vWvrAXwwoj1GBckKyP\\\\\\\/TBEQ4fO3sR1Hw1WfFYSGT\\\\\\\/VF+5rcD7nWtaM8MFFUEzsUJr9e2CfVkn+KUiF+CDgPrhWN7WoSOJuoTa8qKO2w2t2VcdBbAnVvOHnjkKhzNW31upMHKbqvkv+c1QMZaqqGZWbMmKF8t6szH1DLgsgo8SVRO8tHxcha4\\\\\\\/g+QTtPez0CT6ql1nwQL+NZuP2warjdHyG2hqGSuRmQ7YpNYPeDyztIf3sYK23FeAyWbhW3WsBWov15lk+fnMMi2SoNehkdzeUuctnN2zZxs\\\\\\\/z9qHbVFUgnSTB+1lrBqXetc8fR1erpab0oeDEEl7gOgIj+LgtbTSj2LBT1E6d+XKFKE0akBOyINZnJKYYfxY+li6\\\\\\\/XIF3+LhcLcNK4XjCqf4QB5oqIyZ3E6Xb7QIy0eBbML0qcAIgRuq1WmLnYH4Ia2P6ce6va18KpLExScxxFLXs9snH902qXAF8mSsiIdOFGRII9T3\\\\\\\/1k9nuI2YOl3rWqkpuBowTwAjgC6DUIE8xaVqsxG2NCuSsQpdsbN6D4JccEcB+JWvFQrIsIMy\\\\\\\/Su0MzxodtlnqwG\\\\\\\/lr7NHi\\\\\\\/qJInsSAQ4lIyPXmfCpug4ewmPaUXfTT8lnhub9jcvWZuyNYyxMQYLody5PWX+izadSC1pdGGNp2o8xIfp0DQdNf98bT4bDPzrNWV9ItUuSEzFYO+IRLw9he7s0oHWn+hbNSVRCXDE3oboiht1zgpsLEb+fr7joRjX03PsSE6TbC1FIliw9TRjHqPdfkxS5ZJEcxf4xTC5GXCVgk47BxOjXRTgvjbYWFLHThhbkCypAmTIqe6n2jMnmcrnzESj+pnMEvfuiGe+KazaAXGNvwtrl\\\\\\\/FKe4A+vd8pUZF+EyVs9QH08GPQtI47vyHsawXzhJzqOnfxcXArE4H8jjG3NC6rcPGkXq9ktzUph\\\\\\\/SM6C5NPIc2vX2nMi\\\\\\\/Bc6cpXQHG3ZC1FLL3r6qJ0okVcJIwWLagK2rHq7e5mcNJwj7\\\\\\\/GGZKHK\\\\\\\/qFPNgYFHZrDWQwmYh5QNB\\\\\\\/qs3G1lhMVmyk4PSC5rvTWSbK9hOKsQ2KSeVuMc63m+K61GLam7UrVQTJqEXpfkB2bIm70CfK48\\\\\\\/SHtffH6FGPIcSqktA0+rvVMYsr4cZ6RMeuXx7qIwobuEUpOMwOSGfagBoEQ698zQWYLsCmJg4RWekkXZoHb4MZrJlvfe+8C2zJwfLTHWIpAMTJDfhjxONF2lQinY3gdeBabJJUJ7mMwr8S2UP0sMS4VNWa+gdp7u9lmSrF0\\\\\\\/Ij018KhJSB77bFvtGhy+R1jDcEQmTang9dBXT6CCg7AKv2jcT8uw2DeYLfiZj9pjz0BLllOYUMn4DTAbrDOvmteJJIF+EE7G8efgpL0W\\\\\\\/uCObp\\\\\\\/ecTLQIz5h+jwSnLP9sdAWxSZ2rdJ9aiP8AUrucb60+u5kGPrwt8B6FSnZ1aC6F1oKvb\\\\\\\/smgZNhFsS2C2ju7BePjrugP2kZa+fUhAw13CAaddHOpBqoaE1vY+kj0s4pdvszsuf5M+6YSPcQ6Sgr5k4eaMfA9+TiZWu6yP4Uo+ZchcLdzIv7GOiZSAKn225Kd9iO+bJ8jZyjLR1wsoVUSBMtVGGwvud23w3CjBj+a7JZSXORHtOQLttyWmYiAuEC2go\\\\\\\/AHe97XBza\\\\\\\/gwd6oFQ3tdCThyo9DC\\\\\\\/FRyHeu1UIu7mx26EOq\\\\\\\/Kv8ed+1YcarS51IgkHXiBzhek8NmpgoOZI1wAbfEI6zACO3UNAVcV3aVAoJrrv3y7uI9dw05db4WXJNSVIBkDCOGXbDMp8FCm\\\\\\\/QY9lWtMjyz8YPei9BxlM5QWZ3HJtFVvp5brOOgj1pCt+natfnIcLYFNGVGL6bVxxNq04RYUYNQwyp3zmX3DPROBXMojJj\\\\\\\/vwkFBIStZZzdJG4y58yJCNMx4APAz4kMyEFgsSewao4A8cFqULcX7UR+tOqzSBHFdA1Bl\\\\\\\/Y69O+bbgvD2ZiDzUV6l35rvC8mtqUjOc9pFQOeaE91jmQxwbYbuN5I53BNGvtJtMXNbtJGKjJmYE5m5Gi5FU3VdPbWqB0ZQl1ts\\\\\\\/9jtuP+lgYCw6Qz\\\\\\\/HLrOywfP6y\\\\\\\/u2Xc8q7TljD7t6rkh\\\\\\\/5ZIISziagZdVR1gaz\\\\\\\/N8t7wWlByZNebFpTwc05RKVygVPPKyQYKLObhkr2KO4J37Mqzy3YlkCra6vvmysK0eZBLeFp9OFbd2l7Ia\\\\\\\/fhOBEZhC\\\\\\\/lbj6oZdLPOasJUPmf1RoNcpnSUW\\\\\\\/djqWOTHxI1qX+B7mkcQXMP0jPg6jRwSNIB4iVuWHYhrs1ikG8NEgitL2bLJ8McUL0HWDPNwO3iGKgRB+Kxp0eJ0XWwJuLhLATZFkXMFkdgf1pkRPc\\\\\\\/F4dMV2TDdlnBg2psdvrFver96bHuSsSx0kzszWSVWq92Vr0Ba8M3muJbx6ioUEeGWL6Xh18KosxjpQqSFn53Emt6dE0tHmlLSFp4xKf5HWe7Cs6\\\\\\\/Pvj0QGF9DBLOh8fnl2seoxfgQETgtwUIMTuVT4Lh+Gg9Mgcv8l2ahTtPRGkv2gZoZMb8Y7BAWSNtuo9k10aJJwsw5NONH\\\\\\\/GJCOZEgNfdBd+k8HH+1k9Osf2yu23WoJTLcPVy2tX26iE4IOb8wfvrClPrjhG3l6gxtSknUj6tuBwZbVB4zJw0rIi0Mp9a\\\\\\\/5BroHuQvf1GqIXE4B36ihPAeeMl3jXvtVAS75gZ6wxB7xODJooKI2+GqruHvkn\\\\\\\/aPd9k+9+UALRq753\\\\\\\/IDxI\\\\\\\/OiOcZ\\\\\\\/4Eue7rOarroLbpkZeWxMDz2xSS8X0erXnE+eqG89anYFqQ22P1Sz79TXvmcGtZLxiw6uEQ9ufoQalbHeiwbaKVWJTeic+YlUL6NHWod5PXGdNebmdOjYNVGmiwn7YvotYWoAmtdThLt95FgYfqZ+gA9lklmCcYhIAMISt5cpIbYOmPrT6kSYJ6RyBzo2MEg59lCOdc47yqK+oItULhDqR6H+AkczzHRahlqqAKpBLBk80gnnzJYm9lmJiDIgOH4KKG0R4Xjgrs84Mhe7jTu6zHidnh8xNAotnhV84wOy2KT+xgyn1Wy0i3GQZlRr1cBOkB3bjTgpbrRyit7w3ffgU2eyyw0AYk2JMNRcNkYQLPqI5yfCeWVJfKYhptOdxlxA1iyVmZKW1bAWmodxs6o9Y+TQ3vmoGsWQSWxqdq5zloB2m3havbXaICkty5U3TfUHnMXDsEH6I948CyR2n5GAxR7OjhVBlD61ruwinKZlPWZeGsWcEBdia3x7JO3q2h7Kp4JJLWQCABzSDmno2pHpzrwcG+E4dSZC8Oltnv6XKDaJHO8O5h01qVCU1w1v4mUPiZDmmlC7AycDxvtDwF51\\\\\\\/GZQqPwlh4UmLZK6vUaBLVq2+ed+BE4IqpbONosEZFEFmpyaOEDs0Tm9JLXxP5ntMCp6lq5nNDRUpK3nGXgMj6yjc4Wbvaqe0mNyZaVU8BhBBAxQLO0wWwsDmFF4Ggr4wdRkJAGmBILbLn\\\\\\\/rRZ+G+jeK0BVQEGUWkFQo\\\\\\\/bdU0hdZGKtjKsvFBPI3q\\\\\\\/+ES9l6DC\\\\\\\/NexdNBoh3TuliQFbSp8CelyGwKQyCxDqQUJtDpyaima2XwCwozhGDbVusiPrzhGIEDlwCLFBs4i\\\\\\\/MCRmCU22VyPJQjg7TSrCAJMkK4LcE89wa6Wk6lYI\\\\\\\/qDOosN10Mtx2sFfcmTPIyb131HrbLNi0Z7s\\\\\\\/e1FoqNhYjYT3SheMldv8bEcqzBtK3xPJG6bXYpUbPDs9Sx93AIb+GzjcfvyZmQEVrstg4bBShGEkW5TjuG92Qo5S2rmfOJT1nj7gJ0+l4EoaHZQWQPFkOs36l17SMIGD3hbywF378fworOtyecR1c06zNRTOYVJkh3IBQvnBdPWMJ3PqjAm1bqRBZwCOutjZ7G3zom1u3avJ768Ktu3LYwY8rDpPkDUPJN7bfTLrqqfQZsuO4LrtPJJAn6qPU9Nonz4S60CapteR4XS23L1exs+Dc8+m8WNESxk0cFpcounqrDR70ciPdmXRbBxlugsz2Lbxgjy985RCIxK8fAFbb00I\\\\\\\/2wioSp0diCLl0SMLDlb7OCj6NQ1GS5J69IhuDKyU0dC8cwHSq+geB7ZJ4+UKs6r3V3GCXkaKx\\\\\\\/kehppileavUrNpzFR+9e3FQelTX0T0dVuEwTovJE42Uy+Ux+WFcZp4qmI1MRiVDfG0f6QHbSmx4rFapwG4hRjvPPeaM5I1SvIrSpg63MuUUNYVNSfWpHSM10edTSXGOgd5\\\\\\\/EQKL2H1ubPapdzc3cLCA2x1As00qilvo9qpC6s2s4EEW2sSf68jkcX03P7mTdzbocmADszajIjUXSqUbKsUau+Xj0JFwkeVVJTCTGBordevIHvvghoxmsuBF6gvV2OQ2y5hJl4I\\\\\\\/1Uejq9IxPkYqXaKhUXrmSg2Wz+TxCWV+ZWJca2Mv0BfqjSp2sHeNBt2aztRYMLUbleRvsDZhiMylYJLxCvNELihdyu2ggBVVlkim2geCV7kk1I4gns6swZxYzmbXynkVGiHCLO+SwvHovgsWDUhRSVeAtgP1xEZhlHi474zDGet9uNFPrKXjGm++K97bO1eNM20mlTKfBIeRvRLPpTRyQHMv7O\\\\\\\/MrRukQq9uTu2GuQzA2ASoT+qJCQ7X5MLQ5rIJfz810rtR0+X3oqaaISpNIztupnjW\\\\\\\/B\\\\\\\/Vnjdocxug\\\\\\\/pIy8Pd\\\\\\\/c0tMNQNgI+UOrL4u0pq1WEf7qVlerXbWQZQ8ASYlufWHv64GF6hmAGRXpZO7uHPiuj4jZEpSTmIkYoVBJjGZfTuLBi\\\\\\\/3z5w5NaP8qsDsgt9eWUfEkeGwQ3ofaiQ9TAodD3jhe57v8i4TeoJMevH0Ucb7FCHDJinQYsrUQSi\\\\\\\/aIc98lpi4rdrmv0GaeKwRVeNMPN0mQnqATJyWe1cpPc3LHheZxb0\\\\\\\/x0K9URNWYpbl2AEejFU3CgOkBmrg\\\\\\\/ISZC9ghh9rVVg+1bTWLlRWz1TlhuHndL6qGnukxL1\\\\\\\/g4Bf5N2AvZtWPsuFKy7PsKkQSvBolrXZzZE\\\\\\\/LlMBJBZs2q8hVfzjpwAyv1+SkL\\\\\\\/wEe0H5tFhZlzmr1J1lVlkv+NTcxHyfZ6n4CFYD71TDejjq6EywIzZOquE+C6vIMeCn2EDjzc9sK4CrH1LhrNfbC1QyoEwpuEekM5upDp2\\\\\\\/cgROESnnzkMuY2ne+ObviZtZw8SMmB4IcrV\\\\\\\/epp1fXwU1HZs1OuX6B96xA0MSFTyhZW+ofeDdFXKd\\\\\\\/oKL55DstHLZF7BoPMSnbng8Q2gJp5ytEN5SZdvpKKDKArV8wuE3V2FKDew6HbtBTf2jvYTeoEXBdKvs6HjA2Ltxt\\\\\\\/3H9yLn7Nb4TOUtXl58kOgUqg9HaB\\\\\\\/ShHCO6CM0llw2eo2fLRjeRXNkVAr11YaGnk7jU1B\\\\\\\/mBHrFCsyUly8b+pUwcFVzed+LKnE6+xN8o8ETIKpZ+7ELWBl0\\\\\\\/cWNs8TCt9J5+O+4jZVKvS419\\\\\\\/yK\\\\\\\/e3WFE3uT6mFyM9XJTxFYkYMtEYmtvaW9lbdAHYRBjDsbfuCAky1Y55tnDRh+P+tDI1RtOmc9gDapkNp7iz0+8is0QmQiU285KlSjNxIXhgglmNXlBm1DsgMZCqUUh\\\\\\\/v9\\\\\\\/uoEYKYJH+VVF4wGkSukXUkeoNQMysLsLj9eg+VjYr5z13\\\\\\\/2q7Za4rhKlF76Aa09ErzO6yaKN0L+XT0MaxGDYH35Q2QVjsDyJxuiTlJKKTdw72PJavGG9pCaMne9KQ\\\\\\\/lShV7ZkLxHsbJIiOj8eybdADtsGuSbX2+fCAYmOro2vookJTmkkBA8tJpUPqZMuajxBAk6bohlRNfcTN2Qe3\\\\\\\/P2Ey9o\\\\\\\/ArrPSRgI5L76aIGWEMsRdZIhf7YtWePDQQLzHOpY3wMjsh+yXs5\\\\\\\/qgGL\\\\\\\/H5lb0m2uSIfFfFYJGfebZu\\\\\\\/LJk7R7c+moKWyKEqvlmRv1a0\\\\\\\/XJtyWoLJjhHf8hOq7gKU7CYbeLn2eIg7kRw55\\\\\\\/BDMZo5AcatFWsFu\\\\\\\/Zl\\\\\\\/bCfE20XkeSGGH2TKbTroF+S89BL4LaoR7Zqq4H8sYW9OrqOu2qpTa8vshhAh\\\\\\\/fqJ6ZBqnVWAmbAR3fFVy9WpC4\\\\\\\/ojag09ysPLP3z2f\\\\\\\/kkMYFwbw4Z+mEGz302EVPCk2cEo9yXgzrQCEhHYEeAq8hU8Eiswjuy+YC31Oxrn9Wk9GM7s6V5qtSyxHQSsFIDFSud9VsyJ7+Kd37V5VHLDFqZZPk6UmOPWcYwJ4lmBDQaA+UZ6DoVC56+K2scq5kT6CJ+03z7qQAwTOqZY4H41jaVnVFgeH6TnFe54en9zndU+q4L5bN\\\\\\\/jiBQwkcwxz32S8ogaPUjMNTl6IJiwGIG9bFnKGvBkm5VnMRYokDkRFMlCgd5TDFXUXXUeRTf0VBWsZHZrl8t3w8Stj0Z0q8XkiUBbZ7vqNKtl1SjXtHiiCULc6nTw9t8yZzz17nEkuaNPdicz\\\\\\\/41at8egBNFYxuwWEFyo8tZuRizBWF6LFDCVfNzFdVruJZaAmVCL4oWz0gvT2Z8wMoK0f0wUouYlntPR8v+eJFKsVj8cVj0D4M8\\\\\\\/NYowlxRKqyAQTwMTxCW1VF\\\\\\\/7qICFsDdLzIxfdlB1L1xqIGpw0A9gcaVQxSxI5swGDO7igZppgpEnAlFEpdmXEqaYZAQAMj4PRvmvnl6QcMppqK80\\\\\\\/U4EFUAG+YRISi\\\\\\\/dunRG4bE\\\\\\\/1kxi2PgX7c5\\\\\\\/aro7rNQqTqPLx3kaTR51GjXjXyTKQBeGkb5bG\\\\\\\/MnMUXzjL15Cy8CbhHGO6yAuR9GZE7HD5X2OZixbe\\\\\\\/f78fE6WGDP\\\\\\\/QsZLUTex7GaDST8DPeKUzWcEkC0gn2v4Dz+j9zZ9XRDjPHCF4ELUuV\\\\\\\/aW0mngGHPkHzsaAu6Fr0fkDum6\\\\\\\/7KX8LPz00gYKjf3NypNisnxE\\\\\\\/Bg7871LykZE8NTGWbDFM1LqboXyzzf0DMrrB5QseSJI2WPptVLPH0vvuCw5sadC87QTWZaNasWwNXTuDaQBJh71x2fj8Ch+ZRD0aGjDB8NygdQjqdCr7GVygnxmsQX+CBPbP1XaTPhtN7Bp7NUWKVfo0puhDiHPpZm5S2oM+Ly0f\\\\\\\/yBNn2FCIuDL84GOmRj1ve54u86H6KC4xOYzcVJN\\\\\\\/kF\\\\\\\/byBrg6v6fRuXYW9W4nj8p2yIah2PI9grrY\\\\\\\/haDjV5aAugX4KT46uv7GlhmTebYXJBQLrhDxkdBs\\\\\\\/tUy58TpaBhodvcn82vNKXsbSAVGGnMlvGbMw7czLTyFRSnVP+Z5kU5EMe0U1aqzm1sExGZ0pfb1H8u72+mh4hhruGgtPRp+1ogpdYQwjipBxy4iLyoYiAWnzNX+QTP6tum387EwUyPGbzL7zoxiXXW693tf0SHB5Pb1HNvYs0vO2PcFfRHFRQadgkrkylU3IcfFbAbe2gR4FTvvNYrn8O+oDRR\\\\\\\/nRRdgSl\\\\\\\/sJOo2Lxt8ll6jWLovbRaea2vq0XlOhNL5Ki4zjM8ASolPv6N6fACcRqacbOuYHg5z4xbr7R8O20iz7nxol3Ceb3HBv31iIOTnxrk4xZpFmt\\\\\\\/WAud3GxJ6WGi24f8EfXlXoVDcnWjSsT2w9M2XqVXJRUkIuvtcbrOdc\\\\\\\/5oAqodJNkDgcJVIjMjNKbHB5MgKdY3h9+AwUn32HbsNIQcqtP4e1g4M2+\\\\\\\/3vp48N2CLFQujd3pWhWKoOLOBnxY2ASbtC71XSC0Elyg1du3N4OUCqxPfLbVxS3IiVWTvnjG9AJlA4RA1U7+tC9xnYbCZnMARmGthgfZ165kX9nLAY0LDHWjRaXe\\\\\\\/FI0yh1KKmRCSVyhYRbvfx4kmpoU7Q8dcEbvRwCOSvVh2AYICziL2jLXOh2FTNZ+S\\\\\\\/1xxB3WculRZ1A1miKnnCCoYS95eqnihXg5bwPSongJl38X72SbqOHxMBe6VTJwpANyFsoH2dNPv8fWN\\\\\\\/yhkSh+S1mQ\\\\\\\/AKP4EJMY8Qs0rS4SrSLD86LgwdB0Sq9bkQUB7imcv8opfyR8djuxUeQ5it\\\\\\\/xkMvJgP9x7aydArWySPaXti3cnnInh0yNKZY5A+6dLqm2xoDHZVxpy8zjJUQCHRUVmG5AzSiAEB52LIQSXvChg3mFsR+swphe7hVIC\\\\\\\/ScRP6E5asq8SRuiDZ3Y+NiHxkQtc9oPaDVR7\\\\\\\/XHd4RRfojqUsy7GMKdp+wfgB70fcXLnJCNf3oSUTWOTEO2pgwj0rDJk9pCUgVnPsJ40OIGrNNvX7NSzPsXI8TCP+qFcF4KIC\\\\\\\/x39ULPA0wKxt\\\\\\\/qrE39aOTczzj9GkpsI+aCPsKmHJN2dOrD6h01Wm\\\\\\\/huWnUN8EPrNJDD97mtb1SE3O6YRVvDZlsaO\\\\\\\/O8GgVnypsa9wqAfnFkrtfxQG8Ht\\\\\\\/Incdt3dtZPnOvjI\\\\\\\/VzrQBiI64R6nT9HHMKiKTMPW4JNDWXNW4Mx4PISnFp+2Gs58UtQ2droi\\\\\\\/eDnfk6S86S4P3nEuJM9Z68DHna87S8O8j3X+yUk4xvSS+yJI1X6aYbZp3\\\\\\\/Cyjg9ZWGR9vNQjuTBy8T9AOafHIottZSg7re\\\\\\\/ITq3+\\\\\\\/QrVrzf4o2fktbJ6B8f0UPB2Fdmm3bgKGUl1WFS5LJ\\\\\\\/3fqRL4r1WS0xdcWrf89gRaNvxfYAyVNtVpD5dnQU1vYkbUsNi7WitfhczgX1If4HfiW6jIdE3ClTtwN6aowYsXKJ6gP6Ttg6dsqG+3k6j8lJ+EaHBbXmPjBfo0hzNBxCR+asODjR1WanGTW+JHP6qdIskOr998YtbaA7ThBN2Yr9RZSntaLn5ygd+ZlVLDLxuEPaxLO0ZahGYfDTtFiCXXnqRciRTQYnGNHNzWPexQnMYgZ9mvxWFmiEYzJQ4q9awWsll6WwF2\\\\\\\/WslqP+BUslSDoYnIg9ffTEKDufAc28yl3AgFbqCraO6uPy1\\\\\\\/NYtMA9bqkaUY8Z270O9OQR7L8v9cCYawLzchsJQq7JpyPQpn6QLLGH+2PuAUzOxXiufIx+LdMoDa8DNH+qZnMsELyEKv\\\\\\\/Zfq4MTlxEhSaLoxd0h50aVLsENPivZE1NjQ02KdT4Cna4sDYDonKygK7zZPqjsa\\\\\\\/iaR+zY8gprxn7DiSaq\\\\\\\/z31hE2EfybDNyAFEGRM6TMPr9LosoP01lzakhmJ5HhpGsdLXfuNS+aOJjye4K+fLBa2OCkwlZxPe1LgefHk9MY91OA2KabO2YiRqHFq6gRDEpx5CJT5eCoZdoT2Xuz2YGef1SYIPZknG3KuX\\\\\\\/\\\\\\\/2MDNADG5OdXMQzsnRQaaAav+PBZ1fHdvUT0FD3DYxYmqn9wInkQ4eF4vDFt6cfVZyNdbYZ5N8GxgL\\\\\\\/PFmjj7\\\\\\\/u61CB2X\\\\\\\/UpuR78uCLHJgK3FJBOA3vokqxiWB+nLlUCekNQwQoPJtEkqZxn\\\\\\\/zR\\\\\\\/tAJhZl5c2zNYYJCyAAVZeX5z354BIpRjT9OeUrj\\\\\\\/dUvf+z25Zutwwd3IXgGpIMCZV206+VBaC01LK04cGmrvCXmTplPqTi+PKNEaLjaGK1D8cntEwfAinapDeclKj8cXqHh+gycNptBWCGJcISfnjKWc7zT6P1PhZaTT48447tVmuNZglm1jTHT25G23sa3xFBE9LvulA+8jnSKpKq8e0rb\\\\\\\/CrxROhsc09tY5AD2KApAwy9wM4CbB6SMlLouHUXH5DoPfKD4G1coTTdLYvQNHDfKm3NDet+jQuHosrFlweQx\\\\\\\/l\\\\\\\/WGQIOFsUJxx9pfLgO8z4qHr5DT2ilHlHDdp7ME2byrnaJE+nO0qMBOAlVCm+laE4FJwvyK+B7TjOoSuyKRpW8U1tgGR66zGpXiSe7YNFMinWc\\\\\\\/sVNX40MCXvWrtSe\\\\\\\/z1hzugiATMSg6iXWNQgVlYYkpyj73912FZ+WilGAf3w5ZOGwRFgx0\\\\\\\/b2IYvNEMLeY268fNqKbpTqtVE8MbY37dgy6\\\\\\\/mjQPKBKU1YtpaPcHYT8PysO9Z7PzSENZ6kQjScU6x2zoNaQ7S2\\\\\\\/yuUGV7RCNSQc0pyS0PpTOXZok6Tzp2wyIACUhfv982X4O2KRJ+Q4OCoiCMeIyQgMP87KVfINfNixtyX3alnagA0lbDL2J8bXRMkM1yA8+qlgNoZA5\\\\\\\/zdRvBi40kL+kX7KZUXQ9TVWVXQZJN7XKdYvORPLuR3k2FZlhR+THWlD7xJP2htWJ8sVy\\\\\\\/g2eoS8hKeaZmIthT0C+W1DggI1bOXljJ5G5UVRkExG3bQohCaOIS5eiMf+BHsYF6AE0VssEGBWvx456a61ydfsL1rIU8b38MCjJZ40op\\\\\\\/F7rCKZWl3ZwJYklXYVjr27CkJdA0UVTdrYDJwI3jaHMiWNP2y3Q3Z4gzzwnaKpRelpUvw0oaweSr+ywF2n2nsNP5YmGpdnD7P3p9Igqrs7GmVWE0uR41gR115Ocvb\\\\\\\/yR7SJGQywMaPxgChQVhKWkxq3+1xDSC3TrdijceIO8x+dGnmm0qxvewbq3TLCunQjBANtxVab1dCrgIf1zjJ4aEIPiulgSeRnx\\\\\\\/UyV4kZX9TOLquC0Ev7RgkT9C0RmM2CXeAtuJ9OeX6\\\\\\\/N\\\\\\\/14jaFLGo4gHJwqsbdBiiuBHP0nviTdJQdmQT\\\\\\\/zB0hCVsjh6ZLGUJNFtHrQO\\\\\\\/BAL5B7LLlqNxIm2u96PP+UHkCUQV6ri7mFBtxbOkfG355e8YOQ\\\\\\\/MdIO3ZhaPvDIx6R8palA99OqJ6geWT+JHlyah4PBBS9AxYFD9kKK4XlibSzZeP86CruWtUmiJR6RJtvxB09Y3nfS3bGNyconSXDMkySjIjqyaBpfNdRqxoZ\\\\\\\/H0+4oJuP3fDT\\\\\\\/S7iJkF84RlqXIRdeYPO+XpOwuyJgea+28G9NtIVF8dmUFT66oAoyrXlFXjvk51LF1LMi1LL3fzOVaJcY466QZc0ROhdp5hJ8BAEtFSa2iqdPk6van6sBv6uYcYaJH3ZFYfYI67wH0Ia\\\\\\\/5URwJx6gIWfp7vqFejLC4r6uz+Yh\\\\\\\/EgwtyAkXS3FiiwzhkxTXtw1jSHWTb7onzE1AIW+izK2yQRU4ooS5JdEr4LeeQOY50clwODPGKyGo\\\\\\\/DEjavEkxL1d9jyNvsdlzB5ABQ7QAdXErZGgCy7uWtsP2fvljqqaKUmm82uOa68Aa6oauzchI7XRm5t7IrxA16V4K4Hcln3jC3MfcS55OB70xQ530jxoXSgREbZoxJ7UU6CW7EZ\\\\\\\/P6MxE\\\\\\\/dlE1HBpysFyYVheXytwKcqtlivA10TfVND0PCt0bjE5D9WVzP\\\\\\\/Ac3YO3qUcO8m1kCDvxHmHkiU6u6CNrz9cw1P8ZaWAYsQuqII6YY26kAzMYyJkNZw+7P4cB7sqI40AswwxhnuOrIgUPEeMKHFA+BuRIiZfXnAo4ww8vCGaO9Y6LFD+rj4SRuYbb1Vl+XDId15xk8T6GnPPpPAD1XxVbiVKqHMnfwjWSqDsRECT8+bYrXIcUVCQ1Z+8EoJLESdIRX539c8X3V7IJ7XDyZUSNdgMDRefHfkiZbWsxPPE5eSqi+oWF+A3XPWAn1rM33IhgTy3eunKi\\\\\\\/XgGaR\\\\\\\/1fQsoSAUtmCcc8SHSjEyeNik5xMw+0H447ABkOwIiQ8fl8r2GcylIyDVT9RSiCSzkGnlU+XZGYr5sEWL6StJA4IU1OHJPd6R0WhaSBrrNEYxXb9q4rtFmi4D7Y3bsbSXGdKBR7kOMJsJf2hxAxRaQzt7H+oZbHKX1U0So3OvajhwNXzEdo+lLLaoPY\\\\\\\/EbMt6J\\\\\\\/v59N2IV\\\\\\\/lMgqPghZneHKOwgeU3owPzyy1b+Gltanh0rqQeNtxRc7n5nWi8yCxBd7nz+WPRewP4OiZexWuFUvE5svZNhCqWyEmEBAJvXX+u8Xi7RaTzPZHrTxRUwkJjEkR9i41RJSPxTtnUlQXQBWMPitC+vDfydVfgVpVP52BywvqmtrynCh4DNSZyzGFEYb4qVdYtL3uF2DCGe6\\\\\\\/R8AADZ\\\\\\\/FeFdHtdLPIjntiwsy3BypYqdcEKGyb1+xjYy+wz1s2\\\\\\\/k4N6a299NBFPTptwh+VDIRlMJG\\\\\\\/YIfw58CVpsvw\\\\\\\/SGsqg8WiJsBunO63ZTcmErstMwem8thl+SRrit92GZXuA0oHnt+M4KP0Wv5m5QqbsPHxkmfxqxxOknH6sVTjFUo9iU3cz1dm5lhVAl9xcsreOtNicRPpp\\\\\\\/oue4caMuvUULtYSfTjBz\\\\\\\/PpC2Gon4ilk2EWqWQp++wvF\\\\\\\/avtTS4PZQ7hub6gTo\\\\\\\/JlDb8h8ptGGH2lorQBWPcMynj7OlEoWZPeB6YVo22OAi7MrYA6\\\\\\\/QynDUOa9sPO+nqIf9vtvHHqRdfqZhx0ds04KbAaCCaQyEEFiYx\\\\\\\/ZXhFvQmUe8H8QocFYBh2FIlHtPscSa5ok9OF8ervjYQrnE8Ku12w2pupxhBKa\\\\\\\/PJ+rFK3MaQf0M+t37LghqEgVxrE7cQl1tlJV8twFfP6L9CBlaPKkfgSpX51qADFVatyjsY3h62KQEHMUrSc\\\\\\\/jCzM7tsNkroak0N8cT36PkAvZY\\\\\\\/zW45Xo7cjD365MrlQUSkzblB\\\\\\\/MUKd5+6Vz0UsgmC8Fi7yvDnTGH29MltODuomeIhGUfaoeuMcgcaF7BU1x29Et48CvCxUoC45RTwL8A2AoOiT8VU9bx\\\\\\\/r\\\\\\\/JjJunAIGi940HfmrNiZpIH9fZ2xVp9tUfeUWaD0FC9C\\\\\\\/lUOtzf4iZwXs8F5PQknDMGFwWkiqBY+OiMNr+UhpGreRXy6HJ3Ra4N9oIfeW6rdhWMQvEhZTbeprhHJfEpa13SPJwMY0odbBH3tg2pLbS2EsScygf71aWZgXhKZ3iSXQYs6Kbm9ABeh6tqfPWJ3\\\\\\\/i5gJSbZV\\\\\\\/9v3N\\\\\\\/x9J+gIOFPlEXsIWXPb4kB0bQPB9c6CYRnBDI2GY\\\\\\\/Y4WUOQ61NxWBkDFu2uOqf5D9lfmv0dcNrz0ddO\\\\\\\/\\\\\\\/7ciKD2Lj2O8lr48BzKRAbq2lSKT52\\\\\\\/SQGD2mDtLbDiS0YfQLHHeX4Rbn\\\\\\\/QGN8dXCDGZ1e3zN1eRN1Thpaqm9ESXJOjj8KNegkLyGYusbHPwNjhGO78DbexRW5gjUyHCJWxgG6liHcX9bkBK8mjH1Izz7\\\\\\\/bmklKUNOVeUpq1JAqVeEkyhyoAjrNw3NOexyAJPXHjpoHJNrj8WDOpcGt3GgHlI14FpplM3m68+HV6l0f49wUph3fWJavGk59QC7V4XPFaGAHms9eR5JF0+A64XkRM48aGe5rwsRKZ7tB9b+Bga6jCR+pAs+vNsc5rZ0WmRii2a0lbOh8IYUNj42QymORXwHzOk4OnmXmwnIkl2BW153yKaLsYoytTRhAvvoF2alAtOV8ZY\\\\\\\/e8vZWXhr7KcSbnDTE0PxLr3eErWJdlivZAEOU4zov7irOxTD8BPN\\\\\\\/Ra1eGCBXYOWjK1xwrtaorKeppsurgBYepc4jtO877smHwDKkPvNoKa7D+rO1Y9P3qey8OZK4PqkdSNrwo2ayv37gmJ9NlBNxm3FRmr6ERHu+XBHj4rKImgjMXtwTVSklXluUL8emBoblPmS9Q2BmvUw6vUEIgYQctn4uVl\\\\\\\/nWRAlVJhPwAwe\\\\\\\/N7anQNdgZ57fdzZ7U3inmRPSo5qL5Flg+upWmu39Ku2gBcTq4bji6IwgkqteiSSCnYxSukZ4e3b\\\\\\\/8lEY\\\\\\\/UCNwmJYyER4ZVYrnlI\\\\\\\/yPYOHvA474RlyaX2UAcTFN4DMsyfb69RY+v304LiXIKzAj3QL+SctLh7pXfzPT6n4GwVefdbXdWDT0bBpw0Ox9m3A4Thrjq0SPU37fmOBxNtWeDLPPJ2jLHoNZOjuu44TUq7XB\\\\\\\/XpqXHzFTMy0dgXkTM0EQJaTWia3bLn\\\\\\\/Puv+DqKJADUOZQY5Ka2idRv\\\\\\\/IncTL23ZIdA1mJeQyIXMidaCN0FCd217iVX9TsSQt3zU6p+9s3HACyLNX6nmoOrSFrTd0qqVdmMmc3h82wNvfSGYmjgH8W2+SkIOuGHd3N2zjmEzhxKLox32f0Jx9cI2yaXGwfn\\\\\\\/L+UMtfTFgC3Xo020SBmMHx7zZfbpEC4PjR0GhlC7Avf278GQIMoPYnBrLggNdXTZuWnVMTDMZ4IkP1cWygU3CSmHgfRJ2PyPvItmYdz\\\\\\\/iIWbx97zysCHypJb2mDBy\\\\\\\/SUAWMz1Byhe35gLNkTTgk19kNJWshpkZRpPSiy33Ubpuhx7dRc0Np\\\\\\\/M6O\\\\\\\/54RVmBizWA8BiDibLfENyd4ViMeL68yZJMfvNMl5+H+rgQcxjJl2VVPociDXoB+mN607zEtUGyzT\\\\\\\/5Ymob35m+vfgtAdShBYuHGJLmKGlsL\\\\\\\/QxDwX8XX0JdDeS7xq+11pTUjaWSfdDybjeHzjMPZeM7Dbz70WAh4\\\\\\\/N+63HLQXZ+RImlFAtgFwJqK\\\\\\\/\\\\\\\/PTr90s4fkkiwyBLRR3o4f4AkJCq43cUpvyCtSzAXogihPVYSt9HoAFvDKGW86aJZPi74gXUsBRRFMmm49KPLGG5gjUkI6z339b3x2qvL8AX6oQSyBwdyO+HZ04DYAGOhR6qnYCtVBuFOB8UEfwKJdOD8zphnnhd58V0bq8\\\\\\\/BVRpHtI3AvJgjW82nK2uyv2+MzvHUda\\\\\\\/1eB+ZrdynixNp2LeV2SL+1zSMfphXJvuUvWFrf\\\\\\\/UfmuDkETNEuwfRzLCVM8IE2nxYYFuyomC3vjy6B\\\\\\\/jJ3r\\\\\\\/A4G+SQajl5G97QXbWY+twP2gLcK90+Z7WpiZTKRKBjdbfNh5xuvD1wRWi9thX8uQQAzaurFYM+mCoRCMbDYl92xKHBAeLSVro6zevhrJrnPIi2agCuruwk5S62PzNWnqGjSzPyUDTPAmwXGxox8wIo8pBlzgmrzeSFZfORNJ2l0UHy+6qEgbJwuUU11pYvRpzto9o7qjIjiPMNE1FOIi0LORg4xbIkjFzGNr7tSEbJId+WsF9hE4+zOqj9F4CkDeoqIdJ4Vk4Z2yWhx8ajDLQNfQREkyZ4ImYTDvkBqZiHL\\\\\\\/NdrH3M\\\\\\\/tHFVksivUfSos6AjHaUlZvxomRHzJKEDaKfl8mf6z0i3wTGjeh\\\\\\\/KNau6SDg+gymynCuWPZn4n62Sqx0H1Sztyb3XBZtLmQiEuk0hWet\\\\\\\/JslTjCHnOI\\\\\\\/I7vWqnit4QnuMfkzwYxkr1xExVjP9OnwTtj7LLo5\\\\\\\/qbr8W7hfca70lGqIMCgiYVp00rHkXs7LoqZaltGEdy96D6sGPVF0wZnlSI3EamhYrL6Muiffq3LZg2XQmKkG67Y0oomcEynziKpmfUXrx2U0lkotYq8FZ8JwtW6bmmKntd3K2tWGY7iDEGj+p9K2d0Ny6BVBvkr89YA7txgozDZlUDurIbIqvB0QawnuxlVTlgTh0AXzqjo0miPwg8KkmRdqRBvssRZo\\\\\\\/UgoKCECgxXhncmCcg5VtW+dawbAbp00u2TI5Ej\\\\\\\/p+A93WZK+h5uZBnUWCkTz7GO9xL3YawUn6wIRIW0EEi6YrESewlx1oXdNpdjIEyLRgwVRIqtr+dCcsI8oDKofTpL6vgnkPFQXQUqbEazhXVLUMmf45Ofblyx7+lVshQqTccAbiv3QmI7oP2x2a+ya5DKLtpsZ66jBOdzI9qkd+99oFpvBFrcHTL5\\\\\\\/Q+CzTB1yUxZjdEa1cgZooI6UErqumBhXZnY479cQlr+BjWCnwl4qJp\\\\\\\/YgJHALP+A4UW3gZNTaRKDV3ff+3X0aI7nIT2XpqOSEF9oHPGgWI73oIK7J37POmu98XsWaq1GeSLYlEz4O0wA9SiYM1T8fn9ye2ghEvykrjOSxYa7\\\\\\\/sbkkl3I5UThOu30\\\\\\\/lBYI44lRd4ZP0SRx32ZI6EHgldDNrB81tqIHyArcnn6G5vmDtQ8uDC\\\\\\\/B873q+3V16JzBjAkoKswUBFw9D799RCEU5P8Jm30TSjqVD2dDLxjaZ2u2eW5Qhmnn3p0\\\\\\\/UYsES7v5LZB5ZKxfuzKGSJVgWCgksna8JAcuuKC\\\\\\\/YE\\\\\\\/DqBhYj598xgME\\\\\\\/nT6qxXz5S66Qhkjx4ifWYpfabNq1XlcA0AclWXiKxBDw4KyK9pUdYZEh+FBbn5GD9ieAmVORXqQEj1PBBvDMqCdAo14NiLrsT0OyyVfx8dGr9MDeBnNRVdcSe7R5XWKGxW5npcVGYCXjfy+04oi+9r2Orc236pwXpIfPlkatgu9I9OkMWPIfFENWgWJPwbh5WzWQKG8hJitJL0Lc0vkNPLYoWD+UKUWlOXhhDc\\\\\\\/gYJesV3NiCH1QWStMr59BWCDNNd7RfpmbKxpjpUpVKp+4J7pLIM57G2LLrLcogoNuc9MwonUQdzyeyBpub\\\\\\\/pf2SyBdEMNxUg+peuijhq5SauGSoXOtUS5V0TqV6E9C6D6yldAyS+3Z5LhFpMU4t5iwgtb5qcO60Pl3qaY2lyrWiiJUDsFttxRiuPvG\\\\\\\/p\\\\\\\/kme0Ibh0i+96uUvlL8vVYkMVMKv\\\\\\\/AMQzKHAr\\\\\\\/9Ql7OcOMRfPvvwdBu7051oStLZ3MB1EmgYzA\\\\\\\/0Wx1P2vyXIEBEZoE9NhuRH\\\\\\\/+eax4O1s+onI4Nf2HC1hn8BQv2QgBUMwMY3iwk4UMFKPF5HUL8cSOwrfDNSE9+oj2hGZZAvrcfLRA5uP9NrU0fm6uI5qMkOrwGPZh3s4yhKoCzaD3wjo9MtzFfphH\\\\\\\/YkkKKHMUrn1knWUXtktcFpaTXixOnjXSlrUnSMJFKJkXfRfwwerKUI9xK1ard4Si7bU\\\\\\\/XGH0gPsevf\\\\\\\/sGKPT2Lbf8zSZ7Kce+RHQfWNWuqULNSpvfmd4zlURnLf5q0WQruIHgTZlVatGJptCS1dHKk9LFH3eNFJRuqJSI3G9IOB+68Uka5Ak+a5bDFuKsK5k5vf0uwVaYrCpcL5fJDhsu7Jk5abIveObHhWnKdXhFNX\\\\\\\/OrgZ51TQFZqfB8aoM3qu72VwQ9CX\\\\\\\/MTslKk+a60X7dWhWDKDUyJTwcnfm6Dw9g4CJtfFlXNG38fycTaLH79\\\\\\\/ThjS6aEeOvHTv2VWKFUJeT5UCMKPm00P5cMqSNYqvI1StJhyNSQiQv16AYdGHsSqqsq+HBPaX\\\\\\\/4WtyV27wO7gqSrqyw9d4XxjaB4uT2s\\\\\\\/q1me37bSrBqrXfIURRL5G8Tzb7YiS0HfIKWSLAgtvvEPg2SkwjE5b3fTwzKRlRcZIUK6t9ucg8kdLncagmZu5eEe5LtJKZxr7OL5mNnUKDWHpJdk+ABvDWuuSTASqcpa6+RaEdnaAlLgn7PP4p+godbhOOjwEkh5DVl6msn4D+IjYqqPJvQI8vpGiqL65NUz2Xp4Myi7i3IBOfpRkwPL0VjZ4FDSjTK2TpEkEps79YHG0oN3Gx3Qg\\\\\\\/4\\\\\\\/EBqHGgmVGlGLxvFTHuH2W9q4okRvPcecgleNSAU57vmgCdqRKtIWhKS3kFpdPqnWutPdb53xqChADe07JOQaTBwjZ6+HLsJeCSnqmoXb+r\\\\\\\/UCZpPyjMpfDXUJuVO7mWYaR6bzMTbfb7fQ5nQ0l9WXSTq8jD6HTAofm+Pes+lgXqc5sgj\\\\\\\/3hG6j9eTSoFOdPoeF7lA76tBnUp7ieG5wgkUbCYbrjMVcuIpQx4j0jEt4f6o8fBXmCdnIoA7+ffhqg9d8e+e4k6DnmP55u5LKHJHWWm23M\\\\\\\/TLQNXH7m6B1LYOkJyy5+higz\\\\\\\/10BVXlwiZuDkzA7CRM3ro69Kk3fzL9Y3HsHAE8ovZwfieJI9ZHMbO4iVzTJob\\\\\\\/n4FgGEXbMePjfhqtY4CE0eVg0G84MysVL2qZ3YD58WVkEaKXiFtHjuP3npLm7L3WSgzqGeqZl7sYhlHwwpegl2Xjd16FtnkWUbCfv6Xu6O9EWH8NqotawWySA0HXin\\\\\\\/7\\\\\\\/s1VIK2+3kcsIlxCIPXcqp\\\\\\\/LKBe0yEa0ktgfdmh+N\\\\\\\/nIvyqy17MdzGuaXuxs9fVeqcezqsfDvCgTT0mrPQ8Js9j6oaSvGFoLpJVPfjGZHaTwWZhxVD3nblu\\\\\\\/Sg6uDcJoSsl6XiuVmM6OTz04qZ4F+5lPHmhOJX3vRQXqxIG0J65TVGWFJ2fPcMCFNx4lbZAxtykXnoqhLmMIuKdOG\\\\\\\/\\\\\\\/E\\\\\\\/iH4v7xi9d92\\\\\\\/Cv8nlCxeQ0dz6R4LJGzYzZ0WDho6F0Ks7GKU2TENxw4WRJBumpGf3gfQXwzsbSVb9tOeMbBLf5pnShYr\\\\\\\/OOZwj8aMaQku+MRSX9tesJrGykDcg\\\\\\\/sCRr1xzY2\\\\\\\/7bq5M\\\\\\\/xIZgTxLjTtGh6Sppue04Ixe5vtkRqVzpejimknp8ojW9985uzC0nSP1rb5+N6V\\\\\\\/J01\\\\\\\/jw3lfCj+l+FcvZjxMT+\\\\\\\/beYcG9HchtBKTBRZQyOArPwGAZM\\\\\\\/NxdxPIV1cKRROdlhQeHMoSZodjMRsa\\\\\\\/J7fDbUBih30+WGBc0\\\\\\\/MaiWltqonHZQOpCnnK2MsagtoKtwPH6xs46P2x7zjirs+tZeAVbncwHt2rdmk3iP2hVrUarG7C4YBvSw6BzDhStqfqDQ60JJSf054MNawa95Fn8r0VIZONkaVr1BqP4e4m0gPmscMh\\\\\\\/lC1eTocsrSIapXEqZWfkK6zcsgCitZf7t938q6yaxnJz51ahkP3Ut+KedxVaXq05THdLaSpJwblZLSUcqepUfEeejDydQxZ08B8lEzFxUJRuCf7b7o1cf9mkeinwc6B2Q9T8dTvCUJbzOJ7n9+HF\\\\\\\/Fx\\\\\\\/Qs9sCTdDnZrBeEWOMrxSRJ9osYOCa0BHV6TBV9dJzA+j0RCjDR7xK4midGuJ+Dk3ebTIJ4AlIavdubnskZSX8x0H+fUch\\\\\\\/z+qcTk1eX1kOWGSjAd52UBHQ56xvrv5BpgFmrJ2StVIXzvlQM8XKsdTfmx3CTg5iqCNK19YZfhSqI9Eg9Mvy9x\\\\\\\/aCvsvQkkxfy8raQYgtUzoLoEg5Q6DoANbmNDlHTocWpocF7S8tmoRheI\\\\\\\/JUYoNKZ4nvy1PvH8XnX0ysR3G7I8L7l9Uc0Vygiu\\\\\\\/QPFA8uR885tuZ\\\\\\\/Q0IXEO+3AkyTi7aW4WLqRyGcJ+w7Yn1kSCz2I5NcQq2G+H\\\\\\\/hoR5yA2ZPMuejh+4zT6Lyde+j5h0dxHQ+4H1IqNN0pR65wX+OLsoiURjBgn5CrFENOcruoSt2Lm0bq1Niz\\\\\\\/uM9fTkZMPyzhIc9ts2EBp8AGDyBAw9HzEBU8rtgRI9lR0uBOfMBovH9cK85RF0+pMiIYRZS5w9N2Obb0TzvWo9Lls\\\\\\\/3Tayn1xOJOMhL7t7gGP2YePKVw+yOZeF4WfOjT\\\\\\\/W1a0mZxMwZuu4Njh6IIJEHNwOPaMWQgbsvPj40\\\\\\\/NVrLp9fUVELyDIVKoIeJY+Ijb1GJ1eYS5EEyMtGWj3MrbQ6DechHelMqhd3USBJmXx2\\\\\\\/+aNYNzd+6pmFn1NPX3VZ9TnfUTU8Xbp6ozU3MzW8BY7xVj36xpgFSxXTj2tJ5UTl2VznFPWTtDbiGadn8vEOdvYwrQDJApu3gg\\\\\\\/qHy9fmDr2IYb6K0jKH98fC7XB3xHhA4Jwh1jM3ptI3juVwuLmDyzs1mva4\\\\\\\/G2zJ\\\\\\\/McN0b2YKmoXGdoLVYBRbi\\\\\\\/BGFmaMf9aT7LnVoHI3Yvg1ugRSFJIt23ZyOvhUIHdSRCNYzmK2W1t9eWqOJN3qQPFMhZGgF+q0voO9ejzMQ2\\\\\\\/s4OD+hFI7QEd7+UtxCbt2n40ESo1Kht3k6PsKCzVvsusQ0\\\\\\\/1WdSqZI\\\\\\\/ofgxj1P2+y01nb56xG2eIBsCJfFvOgKlO7Cx5Mfi1tTwj3ocLiAVCvLzwHs48wxPVuChcc+qPNv\\\\\\\/XEBN4hWy2QUVDjBpSCBgg+Rlzt6Hp9dZ8TYqydUacHTUxABTS968hCxgnrQdT3AKcXOrpqjYS3ShOnJivIj07UQseTENtn\\\\\\\/uAbRRaIU68ispC9GyUluEW8Ts54yMUW\\\\\\\/8ykaYUNB1tLjDkoeuNXunAaX\\\\\\\/tumacMDayzo48C98lytYBk3hWNl2VvdrY5E6bGJB710c\\\\\\\/5GH4uZ1aUTe08MVjmYURc\\\\\\\/Tfai5BEFka6RkCHeH3qrVsqrTuTMdVaQ8znOf9K1v\\\\\\\/mnfk5hs4rOSSHAE1p+0pI+cz0qDrQSa2NlTXBti\\\\\\\/oKDsYNXKJPHboLbZGYjARSDhZmVUp5tQKOXeO8NqJxqWpJwzahA0CHmJ\\\\\\\/X4umtDcJ3fsqUosGF6+xl9IRpCiO4EGoq1kGWOwnyqYJvGjuyC1sb6KnWQ0aSVt1S53UfilJGAQc5XHn\\\\\\\/Xq8p1rkXQbkBIunW1b4l7O+Gz1phCaQchck1KxZBS+M14iOhr0LGmjWNY80OC1eIvrWqrugfk5D8tEVv9gZdc0WKFWi5MSoTb6voew1BPjq31oyXSohgDRiGkXWmzCzqTFIUK22njkrGqd6r6pTiW6C8DTELub6UL0HcY9y7SzZhWAtdrH0tygTtbx7OnxXHlugDVP59XuqhRjVTY288f6\\\\\\\/2Lzu2yOBhqotKPnBtWUzp+2G1q0StYr40v5AdrzcEhS8Ee6KKxsQo0HSXjmMnppFqXAmwh+vksklCV73wUMgG6OjaIweQ8iKH86GR86xfzfu2i7ePjvpweXoJnfzP87DDSKhM9BoEjGgplvAlzI94XQfaKFP2QpWrC\\\\\\\/+LlHdVWYkC1tBXAhwJILIy7JGqOeEnsp2JGHxZP9xKyXE0kP+pnoApw8RSOAvxNiNbowiKrQ1apECio6ay2uL7W6\\\\\\\/AXzH31p\\\\\\\/AxHSyqHbwwk6K\\\\\\\/znuncUNPmyYcB8VNNJ0FKfgf74dimxlVCzwaXvC2zKKWbIGmJPEYsUdKaT+OU67uPtNbxmOXhk9bwlnL\\\\\\\/ygKHMnOcSTEPQCMBZgx5fplw0MyiFFxb2Qh60kgOp5MY43V2YM049HXS5Xr0+elmtoeXFnidr6fQHHWgWk+PpeMY+IoIgFvqUehIr3t8u2KxIMEwZhCf79okTUZlz2kqY4JZB08sFNvr4N8m6PVPw4+gJg7BaeLGJpUj0o7+\\\\\\\/ingDyiLm16nKCD6oy2Bgqp6e1ZX7k7UQXzKb1r96fCb916EXv+KkzfLzay0lFmi\\\\\\\/P9t2zUGJC2BN4u0iKfLUmYm8P18y4VxOaH9fJRXm2EO2Nb42FpeNF7cKv6ALbTNPKqOouxaJ4iZBL7sQeLFt0YZ02AbTGH0JDEGckE7FRs42nVSK04EGNPCesB\\\\\\\/kOczSKjiv6F5D05P1LJtaeys4c9IAKAfWF+LAQGIWZ+Xs64NKMu8WGM7piHkcu3S9iBYJ6iukoItYWeUaKv8G\\\\\\\/BAm+fmDd8mE\\\\\\\/m0pctWY6QHYl3bHLxsKGqEBLi4DCJOCV4o3DHDZc3ve0L2RiMKazXPjp+fMdvxJJA\\\\\\\/Lif3+Es3Oy5A\\\\\\\/mKSPb7AUtjZ8GenAQ\\\\\\\/awXVNgVRcIY1DRrqgKB4s7NaTDojd+W\\\\\\\/QwgkZ3qdoHSalKZUHhoB6hr33ybq3IBw\\\\\\\/ejE2q6jcO1Jz\\\\\\\/vH4sLcGkCPtfoebuts4mIiKekXpK8bK59le\\\\\\\/MG\\\\\\\/UVBqqX1cuXBAHQwkbBeg8yS85x2ES0j+\\\\\\\/uqxujZ\\\\\\\/iwUXfesOr3mIozla+omzq+mVduXkkh0bwJbPBBx45xsjBWbHJJJbw\\\\\\\/vBQ0R8fl8z9VuI4ZMIxCs2zSzLEUvFzEz95Cmt0Vr3zu+Icw79BHqN0OMIphMwUXx+BDerrGWyRytJC6aazPHd3RdiYbCsVy5ateW+Uean9e+CJKUExXvJc4iyNZHl4g0KyNxasE+OlK3\\\\\\\/M7OHzIgd6oJTk0HQUC9CWuyGHI\\\\\\\/BAxuwbPe4A\\\\\\\/NBU9GsTRuD1d0nyxUbWWfrsn1o710Uorv4LqRTAI5pdmTMoOFBSnPXQyG71h4+jeHFKu5J100WnKDIq8GBK\\\\\\\/DNwKBLNxAz7TEI6HCrGdebhMv8Hy9yNE1K3phA42rpeLJz4JdpHATZcCSoxpw5XSKoiL1P+q+Ti7pgQ718lw5tX3U85HZH5tTyua\\\\\\\/CWKXwDFL3gdU8MitYuUNM3GiWsiMZtqAA77U8cL8Qcu82\\\\\\\/UvSSJwiHMl4iUtS+XEm+24reJuLtaUAKxoZ6nOwPqYVaDJMIxNewFcHLyhkhVM3nsvDqDWylRMv12DifGuJp6zNYF0XjlM1ncjx3IXFnkR6IydCxiT15MD1mlwdw469CyonfRsnksOMIXq7kO9IGc\\\\\\\/AWsdBfLO67iuhUJwcWJFI+MCNpANX0thbG2R8e9H6Knb+azSegNZM1Dj+pLLpMRfgD27mtC2wCZkYlXUDFnInyLHrle00XZ0m5mgHpz8ZG5IW6mA6+sGfmRbgqVO6q14pKDLs3AvmwBLGSkDSRN2wsmFFeB\\\\\\\/wHYt6vrtdic\\\\\\\/d9dt5H2sDs6AjyzrfPA2TkC3mtfOROrXthxrW1xaL+5KctZFqp8p9UPmMy\\\\\\\/JjUEUW4+TfL0XK+CbeSQQf6JRBJOnAUXgv7mjG6JhScvqWcIBWMI1908XeO3ohNnhLGzSAtj4\\\\\\\/ZZ9ti0OVtNQ9+TYd0VVtGgfOc6hF8+EypLRijzGY6ltnZZ\\\\\\\/HGnQzG0YOFw+K8VzGM5p\\\\\\\/yfORnEU\\\\\\\/\\\\\\\/wRPg+zuAfiXt9WiD0LxmgbFvsf4sOKv\\\\\\\/LYlU4NZxPiiu5tS6enMZwbUGXVgtlHYMdHasjApdkPGCUtxQSovi1WO8tyK8eJZ+Nzs0kSBoWPWLhSAkF5SY5VoGSaHkyLjBK3hPxBKnjTbQNhKMUdTQDDO1Ls8LVtoK6bOl\\\\\\\/SWyuAmV9OutISTcrHJgPVhWnpYNo\\\\\\\/VlCTne2XYnN6Mh15hhkliukzmvkEbmIX5r0pJuX1+PYOMFK94M0cl4PmVx3+WRlJmVquJUxsyLh+r9TZaPB9CmTEmCnJodK9aegbgplju9UyLqCR9M8kp5aboNPZGc8UcCwTOLkV9vDyLFP0AgyvtHmVEj1K+g28Bo9NR\\\\\\\/WdRB1esHcjcpRTF4ToLx11yU9ZqHNgxhc9i3C3v5PmFNpDll0hGgCAe5\\\\\\\/UDEerSEm1GknkbghxQcKOKjQzLdtIuX8wO1itHPGQvyZMK4Ynm+7XicpK+4R1rpmX5Di5IPFA9w2YFrlBUSuBWsnLtm3OMZlvGTD5+cbtv1wMPiFlWgXE9gZCxeqhOTQ3+8cRII6iEeKtDgaWi6rADH0sxQRBeTS2v+XJTo3XUnEIKVkHs6wE6UvLJBVqAecstPM6zERXabLmDE9jTHp2UCDu5Mx4flCkWTMB5NqSPBrSNIa7fKVBXLodI8SWTHMb9XkYKjoGjJo7JCf+ixlqMMzjX8QU9JD7F5zVWXBCgKsOAI3LhdGPvrQSXyp9LCb\\\\\\\/CSZ8YgqgwvO2IXD2hdHyHp6Tfsp5izDTJvWJ2Acv1GPHDzh8RHd62BPusGh2ECU5muPWi6ugaYzyuipRcQPQyFv5P0173vUFVfzAUlmWuqxhBgqgv8QupaXMvP21oEQAWcOaMY4yFzy6K83qa4SLUf++91GZuWzASFTsZE6SkjOfoRzI\\\\\\\/iPWdLOJruYwOTOY+JHr5ngjcE6Ri6xJ6OwT5ykYWIf43bDGYpzUhCPBdaYIuHf7mmclwX+4Pv3IAONu9M2Jg8bJUJd5mRaOR0xRCPqmPP15sb7eSXAo\\\\\\\/JQbRG1x5Ea2oZQivfcipLZBDibTnJQeARV2Uz4jJB8DNy3wYi9GLxHXkyEWyx2Mw4Kt3OvSwQPJvb8YiWzVPWxCGFLb93BqHdOcRraGlUPuoZFpO5NZBjNzygUb8nHu3gfvNXGWW+HyvTWfADOmk0Lf9z\\\\\\\/SAHrz2iosZPJSVRHTsAAhueRjnpdolrC3AV\\\\\\\/wZ+1bsALy7dYHeVJ7JdcwINmIzbs6YVDA\\\\\\\/UI83f7iezKfeDOjq+XLsgM8V2EHJXvbSCs4IxeJ2\\\\\\\/RqvRLNgiu87qaf7S5SJ\\\\\\\/nY+dzFAG62FsAwqlrnKS12lptI\\\\\\\/4w0rbTvhfgNzwpy+A0\\\\\\\/5crQ09LMwJlRWbpA1oolY7fpQu1dp6bRErAU9xjBxNCwlkwB+t7N6saCosWWIbwdDy0MgaAVQVId+PMpaM0ZTP8xMjMdxGB33qAIKFigoYXjvhz8QyQNlOc8J40phcJMxXDGvHQyriOIwPkhQ13Gne45W+rftGR54VicZYkXSeVV8fKPt7uHAUtX\\\\\\\/o9EAhNzh9RGTwZD\\\\\\\/bVDg\\\\\\\/Eyuw+rkuUFfbcuZLP+AdY\\\\\\\/URFPDH0JONiaz4AinDwI5LVjzUpxOjXBsjSNy1PB81TMsMPZNcHQdYwugqq9a5rehRjmprEaTzJtwv20KGXZUd1BpLhIqXPYlVDXafcGwevnV109xwYYyjAZ4nOojOcbmkz1Jjxv0y\\\\\\\/5jPTgu5uWGQO0a4mkJU7DkhrBeZT81C21zeEmweVCXCVcrxSDtseYFaG+has6i0fEyI9OYrEeo4AP1IbeLOjyhTeKZP3ZNFg5vLnz9b0pjsuu1yG5i5OI4Tc9fyMSSSNBDOJnLGj23jQufi7qQmF2Abz92hW4o\\\\\\\/7DqDm3PE6Zb+M5Pe2kyp5k68C4PhBjkjq5K5C0y8hLAl9hxTOIoBAkE7K63ZLFyUYtEG2y8E7\\\\\\\/qbCptOJsEG0nBaOXGZMncQ5QJdz6WePzBuDcor32hkiM3XRveQsG7WCXAp3k6zA7AXbwX9dGZI2jtO8Euc4EeTVDtgoK9H2hVfFGXI85Z4a04E7Uttd1fOk3b4OQR3C\\\\\\\/pgO+hO47QrM84DYYnNpY3GIPSajZqnAlYV+ZqzDAFKtESbN6RIkBd2xQ6g7zI\\\\\\\/Sxnba90gCV05bIB5tPUpiKKGR0yRTl2RZJ3ROMzDrbYEHAvJ8nlYL3LjlLxm1OJ4vTYcSJv30+lAbPPWbCbOvpJus9Q8z2NbsQ+1fkq+AelYdlzJssSNVU2nhCiTZOR6tNEogBtZur5zJ3Q5KSSOEWMEWRw+p\\\\\\\/WWtAf6YfqTlpTUYW\\\\\\\/i\\\\\\\/9KkH8LEYpcOlkOL9FQdCQrYZjjrCKBrvHktH0CqfvLef3Iq9AYA\\\\\\\/2vvc8iLh27m2GDC2AGfVC0z69E7exfEGMZyEa859MRQxKGj8YNoZJU3\\\\\\\/Eccxb1ZTi7JKJ7QezDqrt+7IbW4sa\\\\\\\/wMqoy4fBqMF7JRq\\\\\\\/\\\\\\\/u12GrFrF6cyRSNmXDOc64MJlAmQbjaeaJAiiC\\\\\\\/+b\\\\\\\/fgYP3\\\\\\\/Nto7Hwwd2ZzrdEYlAwBIPssBgXwoCTxCC6PAMPlbSGUB99sNVIZkrDbB56iq0evv91Iz+DhTeN7hifw\\\\\\\/cjLiRexM6fgmYSmpNiSK9TQuX08pFm9Fvyjw5Is\\\\\\\/DO+WfS\\\\\\\/hyyokRu7DQ3vHFLHYve4FQgOQzkX3zlOgr1aa+VsMr93gAWCm27FX76XxiFnBRkJ5DN1FZxiPyDov335lI+0btdbqRP23XOzRD0SxBrybrROATJ4jQkMDGZUhP5rlFbCQPwNHeOl233qkRaam1oVzWXZU3D+mzt1AeFzBDxT8dUswbzZX\\\\\\\/5ZKqs3y7hZKYZJkuJfpXPtUE9YgPtDSVqv7uppbi1BDWV9Jjl+mPlKHcIKwzkYy1Iq0kQ0LZQPoD62X7tOl5wNOEKv0QH9\\\\\\\/w5b9X0bKEal6s9cVehZj3OiC6Xr0inqoFHs3l6DQc6DsFSY7SDEwez+6m+as8rL9uQUTV532h9wO8pRNzZ3ttegMJCLvRQUp\\\\\\\/ELnqaIasf4mxW9SN\\\\\\\/9TKZUJNDwzd1NQYmYdES13+1YPbzurR2+95THkxGO2m2xIHahnY0dkWd8Jlxm5rxopGJCm+lNnK+P8h3NznHYcMFj59AS24mfqacccY0eQTZaOjJ\\\\\\\/FkUmajP0GKT8B7AcybRTHKPJ4PvDvVtzsNA\\\\\\\/EJ81U4lGgjgtw98+JQufwh5XU+edxRiLj02bhwQJ8ULrjTfH5MycL2oynJjHrwH30TcKXPu9jGVXYG01HBZesw8OCBzso\\\\\\\/7EsNApZCnqvyQpOjdtQPJWkd5cCC6bWLGwidjR2KaFXUz97szJiZnjWX2awgTMzIS7CGMWaWQ5GwAYpGdMnXgGqv0T7JnCpHCGsU9Jt4sxyV1k4yenL0UoFvFyr9+Qs\\\\\\\/+TAzQL1xdILJivvE8\\\\\\\/pcfjffU3A\\\\\\\/Uj2J2hHrecKtK5yb1LHi2r2b89GkJYkviLgQS8Yar1obgVJeyxWTUaHfHeC8erTzOU1IOMkXlAu0xAsyHdPys60rmeWV8jkHj9EV4KY4AILI2Poz4UTNnQgXNrLz\\\\\\\/Wm9kfYeNbJmAhuG5LiY69MAtOVIUg2+dYWdGGqsZXL3FSeH2vuhJ789Oq6kKluufgdCPgMskCUK2Nk6kARowMyB+vN8rhktmArc7YeC75luUKB2rfgIGRPPjYJ7cdRoZCRDVdmJLFIXKlEASw0nLxQkWnDE012s4VjR8\\\\\\\/dQA0LOXlJIbiQiFtFtI4BKW5ixVdHiKAVrwu6e47A7vfQM1rYyo0OZOz\\\\\\\/l9W7X+Lb6KYhszW5qEjzs1pyuAoxgeLQ+Ksi1DphjPG7iL59ripzRFpxRP\\\\\\\/KcBeRMFWBA1OrcHFSDSeX1Fauk5zA1iSvlDmdQJKZAxzMei\\\\\\\/yr2PTTroFV+igcbJYu7J4pS+ggF9O6aId9urQZ7xsWKwgHCNfooD7NZIRhw3BNXhQWsq6Nm7CmYNmSnR5ICy+wTT30YXyaV+lt2g1nyW7fnNThRUDqr77IL\\\\\\\/YDUBeUlR4E98OeUbNmJTEB8j2\\\\\\\/BDS8hboAji9TIq44w1N+4PqVjvYKxeckKwqMw1Lka5xOHn+cIYf9j5TuegBNzY7Q+n\\\\\\\/uYyVhPOEwk180I0asLs9MVvkhjTv2TijUoYx20uRI3IAyrYQbedX5KeJtD0wdMUUl3CeV22H2RidX12A+nwZOcWZfHO7wfUF+3zm6DlX6FMUflaZoUi76L1StL13D2jsjiYPY6DtTcEn8cZuYvdQeZFZ35d0PngAeCnchL6J7cW\\\\\\\/oCtbkd\\\\\\\/X2p7DSFjr9YsEg6gGuYQlXBGSFJgDkcvl4WjyMNpg8gV9RJS4u3jsLQxeJ6FJactNRJOagBzSjmwaryL5LsgzOoH\\\\\\\/XRdH+SIwWB+IZHbvyA3i3E9fcUtlIIqNcMHf4NlVru9rvZeyuuzJWDZaO\\\\\\\/NBuwQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022SWlub+A7Bl9DanDkjrqtJQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229ba477600c04175df06b0c9a3eceb926b25b35cc6a86c2786f67e1697e2ece4b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229Kn6fRqnDfdjLmdAOAMbn6J0qz\\\\\\\/jFLxPAmwcYPN4TxSS2z\\\\\\\/hr17WuASfDgkrYvR6mJWtwMxz65AKS+Boa91GtbOIyNQxxi41EqzNBbFhlM13kMZJD8kB\\\\\\\/4olYCExSdnoCGEeaCIYG5xsAI8kXTsC4nezPAer9AG0MrIpwagxmiT49NrsSGHJy1r2zx0Jqyo6DMJpBe5MrII2FHA1zG4WWOje2Hl1RD7+LbAKTb5oA9hgr7EWokAu2+1XTbJsNy62K9g7nJBSfPGo0HOhvD84fdZtUtxtxbPsrjpEKLYBiQ1e1c9Vco0unk9WncyybI+beObXYDsa04hmonOGfETCJcpheYifNw854HjEfnvdWqlM+X+0EaFHZ+X8g7A3\\\\\\\/ewOQUwQo6SAUNpuhk64JCqsQKBNNbgdsdNl\\\\\\\/9UFtIa9UtZWp8yfdyQ\\\\\\\/cOuQEPREIeRuz2aBj6wCkoL5w9c7YijIpEPxdkC7cJPuI9MKSoaffDMDRoIkaJ8Ibo9kUpyFOUllJXdmFhr\\\\\\\/yxuV1ZRkLlUEZ+7MUUzqyrwBMiHAE3aq1AM1rhL7f8+WHlYNn0DIDiywF1L94jJzgUDR48uHZz3ZX0B7x6rBOQsvtl2dNfzhjBk+2wShkxrbmLh7IcVSYWUYlQwEoXoTREElu4zssOwq5akBwHCjU1XnIl2QqU7wTmaLasJrug2vz6YSL5xeFBb4Pv0mGdTGnkW3Fe6H8tN2NCzP4H0PXg4s0+EKGrL7webgPEFbzKXkiQgwbZK3deKR5zy+xMPmacjxd1FARd\\\\\\\/WzKKjLbSM0ubkoFZ0au+OXoZIpEWO83m9DSccpgm+ylGnX8OMW4CYOte8M0Qp6UHElGnM8uFlQDqb3hXVLYqG0ct748SEi0QvKFsYLOUs0bwSZV06KFrFQkSV60o43l0lEK6jgeaFDCfxBrEhnrTRue\\\\\\\/6h7tHoaY20bY0THIeaMwwvj9IiAOKmmZ+e\\\\\\\/y\\\\\\\/mXbzeqVGesUjWL6+vComVW9ueQetbIanszDs4EZ2dl3+bsM3lSuerjqYejmH+xpNmROaxRh2s3IFsVyGkKgUSHpZ5oU82Q8XSlJTpCpGLYvkh29nGMQ2Ksk1zSZudLl2RF7ROH4gKw6uodR5rMqRAXJ8wzyUk9lwoZPmMcQ4HP1cL\\\\\\\/IGaJ4vooJ+rrxrlfTBGMMgA5Eak7nfhmQAf0tJgxH16x7MEWBdjiJ77RQqupE+nH4yom1p1VzYMRJxc5NnJxxismfuF0+V69XJLnBiZFvt5lfCe94MawrXigYalfi0p8E086dwelS5CW6yuXwVnHousfy8XsUqjuiqhMY+0a70lYxUlkg7cT8oTnM+HTd9IKv02z6PIdnCj2wu8zGyUz6ZeougVU5olAo+RXv6OAmpD99LWN5b6fDkmHiC6d+8CBioita72Un1oBwz\\\\\\\/9ZYFfunBcsCdwWqWPWJpy4sVFrrs18uD4xYljwMBSY21WJe8H+NpFytY6N1UYtArEV25DPlYERI9K+GUQsz2C3v3lFjOtmVkyg\\\\\\\/P\\\\\\\/9det\\\\\\\/7KNhVUFVZHY413ZAH0+WjCjLNhg3bEQPgBVhUZG2YI+qFJ1yA4O9KcMz7\\\\\\\/5E3xXZGAlNidPwvn0f2qZy2Tog\\\\\\\/hI4VsI9aijyaK7vRDFIIPfazwTImWr0ws\\\\\\\/onrSMvJ+SqrS+yY2y+5zFIjSGKRqKzj8Nw0LWtJXTa4YCez1iGI8HG\\\\\\\/yLkIkS30lW2Xmsft0Uyfw4WKD\\\\\\\/bZwbSzoGsYOVjQavp1CZho9gymG5SaB8Kn7ML92CLXvVj4CWS2I7segpfehORgvOtYVd5UXiAQmyEHZrk\\\\\\\/J8Tgq3BOKN54jyJff++2OSCqqGegAnVTivd6\\\\\\\/+13prLShUpNiayG8D7yN78MVgq\\\\\\\/2489jKzLV1Q\\\\\\\/TvmSfBDSEPnto0+PsM\\\\\\\/cRazknouJprlo+Z0q2opHX74q4\\\\\\\/SM73Khb5MrpeiPbkqd9czpWD0jipnnFoxxMPpnOF9\\\\\\\/0\\\\\\\/0fzDpgDhF5NLG4AOzvnNeqIUgSLerDw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PTICMqdwtFkcYQ0E\\\\\\\/ZU9pA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002206dba4fb34eb10fb998b8c2e2c3da892d05510a09ff564cd2b07186ba61a4ff7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460249 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/79" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lCTwQFgsnF5yMDNqGIOzzh4cOB4jzLdWJpjtOG0oZr8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/79" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IXqvYmnNgGPhB1mIsR7bA_BEtxi_6b54kFJ6ndgHGJA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/1/67" title="Target Enrichment Improves Mapping of Complex Traits by Deep Sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/87" title="Identification of a Novel Nematotoxic Protein by Challenging the Model Mushroom Coprinopsis cinerea with a Fungivorous Nematode" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460249" id="node460249" data-pisa="ggg;6/1/79" data-pisa-master="ggg;g3.115.024133" data-apath="/ggg/6/1/79.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Quantitative Trait Loci Underlying Function-Valued Traits Using Functional Principal Component Analysis and Multi-Trait Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7117-7669" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Il-Youp Kwak</span>, <span class="highwire-citation-author" data-delta="1">Candace R. Moore</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6890-4765" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Edgar P. Spalding</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl W. Broman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">79-86;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024133 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460249" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Il-Youp Kwak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Il-Youp%2BKwak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kwak%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIl-Youp%2BKwak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7117-7669" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Il-Youp Kwak</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Candace R. Moore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Candace%2BR.%2BMoore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moore%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACandace%2BR.%2BMoore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Edgar P. Spalding </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edgar%2BP.%2BSpalding%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spalding%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdgar%2BP.%2BSpalding%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6890-4765" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Edgar P. Spalding</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl W. Broman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kbroman@biostat.wisc.edu" class="" data-icon-position="" data-hide-link-title="0">kbroman@biostat.wisc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/1/79" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460249" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460249/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/79.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460249" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460249/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/79.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460249" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460249/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/79.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460249" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460249/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/1/79.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460249" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460249/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">We previously proposed a simple regression-based method to map quantitative trait loci underlying function-valued phenotypes. In order to better handle the case of noisy phenotype measurements and accommodate the correlation structure among time points, we propose an alternative approach that maintains much of the simplicity and speed of the regression-based method. We overcome noisy measurements by replacing the observed data with a smooth approximation. We then apply functional principal component analysis, replacing the smoothed phenotype data with a small number of principal components. Quantitative trait locus mapping is applied to these dimension-reduced data, either with a multi-trait method or by considering the traits individually and then taking the average or maximum LOD score across traits. We apply these approaches to root gravitropism data on Arabidopsis recombinant inbred lines and further investigate their performance in computer simulations. Our methods have been implemented in the R package, funqtl.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/function-valued-traits" class="hw-term hw-article-keyword hw-article-keyword-function-valued-traits" rel="nofollow">function-valued traits</a></li><li class="kwd"><a href="/keyword/model-selection" class="hw-term hw-article-keyword hw-article-keyword-model-selection" rel="nofollow">model selection</a></li><li class="kwd"><a href="/keyword/growth-curves" class="hw-term hw-article-keyword hw-article-keyword-growth-curves" rel="nofollow">growth curves</a></li><li class="kwd"><a href="/keyword/multivariate-analysis" class="hw-term hw-article-keyword hw-article-keyword-multivariate-analysis" rel="nofollow">multivariate analysis</a></li></ul><p id="p-4">Technology developments have enabled the automated acquisition of numerous phenotypes, included function-valued traits, such as phenotypes measured over time. High-dimensional phenotype data are increasingly considered as part of efforts to map the genetic loci (quantitative trait loci, QTL) that influence quantitative traits.</p><p id="p-5">Numerous methods are available for QTL mapping with function-valued traits. <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ma <em>et al.</em> (2002)</a> considered parametric models such as the logistic growth model, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-1.gif"/></noscript></span></span>. The high-dimensional phenotype is reduced to a few parameters. This works well if the parametric model is approximately correct but in many cases the correct functional form is not clear. <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Yang <em>et al.</em> (2009)</a> proposed a nonparametric functional QTL mapping method, with a selected number of basis functions to fit the function-valued phenotype. <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Min <em>et al.</em> (2011)</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Sillanpää <em>et al.</em> (2012)</a>, and <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Li and Sillanpää (2013)</a> extended this method for multiple-QTL models. <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Min <em>et al.</em> (2011)</a> used Markov chain Monte Carlo (MCMC), <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Sillanpää <em>et al.</em> (2012)</a> used hierarchical modeling, and <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Li and Sillanpää (2013)</a> used a multivariate regression method. <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Xiong <em>et al.</em> (2011)</a> proposed an additional nonparametric functional mapping method based on estimating equations. An important barrier to these methods is the long computation time required for the analysis.</p><p id="p-6">In <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>, we proposed two simple regression-based methods to map QTL underlying function-valued phenotypes, based on the results from the individual analysis of the phenotypes at each time point. With the SLOD score, we take the average of the LOD scores across time points, and with the MLOD score, we take the maximum. These approaches are fast to compute, work well when the trait data are smooth, and provide results that are easily interpreted. Another important advantage is the ability to consider multiple QTL, which can improve power and enable the separation of linked QTL. However, the approaches do not work as well when the trait data are not smooth, and they do not take account of the correlation among time points.</p><p id="p-7">In the present paper, we describe methods to overcome these weaknesses. First, we replace the observed trait data with a smooth approximation. Second, we apply functional principal component analysis (PCA) as a dimension-reduction technique, and replace the smoothed phenotype data with a small number of principal components (PCs).</p><p id="p-8">QTL analysis is then performed on these dimension-reduced data. We consider either the multivariate QTL mapping method of <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Knott and Haley (2000)</a>, or use the SLOD or MLOD scores, as in <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>. These methods all have analogs for multiple-QTL models, by extending the penalized LOD scores of <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Broman and Speed (2002)</a> and <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Manichaikul <em>et al.</em> (2009)</a>.</p><p id="p-9">We illustrate these methods by application to the root gravitropism data of <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Moore <em>et al.</em> (2013)</a>, measured by automated image analysis over a time course of 8 hr across a population of <em>Arabidopsis thaliana</em> recombinant inbred lines (RIL). We further investigate the performance of these approaches in computer simulations.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-10">We will focus on the case of recombinant inbred lines, with genotypes AA or BB. Consider <em>n</em> lines, with function-valued phenotypes measured at <em>T</em> discrete time points (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-2.gif"/></noscript></span></span>).</p><div id="sec-2" class="subsection"><h3>Smoothing</h3><p id="p-11">We first smooth the phenotype data for each individual. Let <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-3.gif"/></noscript></span></span> denote the observed phenotype for individual <em>i</em> at time <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-4.gif"/></noscript></span></span>. We assume underlying smooth curves <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-5.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-6.gif"/></noscript></span></span>.</p><p id="p-12">We approximate the functional form of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-7.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-8.gif"/></noscript></span></span> for a set of basis functions <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-9.gif"/></noscript></span></span>, where the number of basis functions, <em>K</em>, is generally much smaller than the number of time points, <em>T</em>. There are many possible choices of basis functions. We are using B-splines (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ramsay and Silverman 2005</a>, pp. 49–53). <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Xiong <em>et al.</em> (2011)</a> and <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Li and Sillanpää (2013)</a> also used B-splines for QTL analysis with functional traits.</p><p id="p-13">Define the <em>K</em> by <em>T</em> matrix Φ, with <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-10.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-11.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-12.gif"/></noscript></span></span>. We estimate the coefficients, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-13.gif"/></noscript></span></span> by minimizing the least squares criterion (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Ramsay and Silverman 2005</a>):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-14.gif"/></noscript></span></span>This gives the solution <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-15.gif"/></noscript></span></span>. We then obtain the smoothed phenotypes <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-16.gif"/></noscript></span></span>, which are used in all subsequent analyses.</p><p id="p-14">A key issue is the choice of the number of basis functions. We use ten-fold cross-validation and choose the number of basis functions that minimizes the estimated sum of squared errors.</p></div><div id="sec-3" class="subsection"><h3>Functional principal component analysis</h3><p id="p-15">Having replaced the phenotype data with a smooth approximation, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-17.gif"/></noscript></span></span>, we use functional PCA (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Ramsay and Silverman 2005</a>) to reduce the dimensionality with little loss of information.</p><p id="p-16">In functional PCA, we seek a sequence of orthonormal functions, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-18.gif"/></noscript></span></span>, that take the role of PCs but for functional data. By orthonormal, we mean <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-19.gif"/></noscript></span></span>, for all <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-20.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-21.gif"/></noscript></span></span> for all <em>j</em>.</p><p id="p-17">With our smoothed functional data, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-22.gif"/></noscript></span></span>, we first find the function <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-23.gif"/></noscript></span></span> that maximizes <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-24.gif"/></noscript></span></span>, subject to the constraint <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-25.gif"/></noscript></span></span>. The computations make use of the B-spline basis representation of the <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-26.gif"/></noscript></span></span>.</p><p id="p-18">Conceptually, the procedure then proceeds inductively. Having identified <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-27.gif"/></noscript></span></span>, we choose <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-28.gif"/></noscript></span></span> that maximizes <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-29.gif"/></noscript></span></span> subject to the constraint that <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-30.gif"/></noscript></span></span> are orthonormal.</p><p id="p-19">We focus on a small number, <em>p</em>, of functional PCs that explain 99% of the data variation and consider the coefficients <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-31.gif"/></noscript></span></span>, as derived traits.</p></div><div id="sec-4" class="subsection"><h3>Single-QTL analysis</h3><p id="p-20">Having smoothed and dimension-reduced the phenotype data, we then use the <em>p</em> PCs as derived traits for QTL analysis, using one of three methods. First, we apply the multivariate QTL mapping method of <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Knott and Haley (2000)</a>. Our second and third approaches are to analyze the <em>p</em> derived traits individually and take the SLOD or MLOD score, respectively, at each putative QTL position, as in <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>.</p><div id="sec-5" class="subsection"><h4>HKLOD score:</h4><p id="p-21">In the method of <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Knott and Haley (2000)</a>, we take the <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-32.gif"/></noscript></span></span> matrix of derived phenotypes and scan the genome, and, at each position, λ, we fit a multivariate regression model with a single QTL. The basic model is <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-33.gif"/></noscript></span></span>, where <em>Y</em> is the <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-34.gif"/></noscript></span></span> matrix of phenotypes, <em>X</em> is an <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-35.gif"/></noscript></span></span> matrix of QTL genotype probabilities, and <em>B</em> is a <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-36.gif"/></noscript></span></span> matrix of QTL effects. The rows of the <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-37.gif"/></noscript></span></span> matrix of errors, <em>E</em>, are assumed to be independent and identically distributed draws from a multivariate normal distribution.</p><p id="p-22">The maximum likelihood estimate of the coefficients is <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-38.gif"/></noscript></span></span>, the same as if the traits were analyzed separately. A key component of the likelihood is the matrix of sums of squares and cross-products of residuals, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-39.gif"/></noscript></span></span>, and particularly its determinant, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-40.gif"/></noscript></span></span>. The log<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-41.gif"/></noscript></span></span> likelihood ratio comparing the model with a single QTL at position λ, to the null model of no QTL, is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-42.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-43.gif"/></noscript></span></span> is for the null model with no QTL, and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-44.gif"/></noscript></span></span> is for the model with a single QTL at λ.</p></div><div id="sec-6" class="subsection"><h4>SL and ML scores:</h4><p id="p-23">As further approaches, we apply the method of <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a> to the <em>p</em>-derived traits. We perform a genome scan by Haley-Knott regression (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Haley and Knott 1992</a>) with each trait separately, giving <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-45.gif"/></noscript></span></span> for trait <em>j</em> at position λ.</p><p id="p-24">Our second criterion is to take the average LOD score across traits: <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-46.gif"/></noscript></span></span>. We call this the SL score, to distinguish it from the SLOD of <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>, calculated using the original phenotype data.</p><p id="p-25">Our third criterion is to take the maximum LOD score, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-47.gif"/></noscript></span></span>. We call this the ML score, to distinguish it from the MLOD score of <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>.</p></div></div><div id="sec-7" class="subsection"><h3>Multiple-QTL analysis</h3><p id="p-26">As in <a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>, we use the penalized LOD score criterion of <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Broman and Speed (2002)</a> to extend each of the LOD-type statistics defined above, for use with multiple-QTL models. The penalized LOD score is <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-48.gif"/></noscript></span></span>, where γ denotes a multiple-QTL model with strictly additive QTL, and <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-49.gif"/></noscript></span></span> is the number of QTL in the model γ<em>. T</em> is a penalty on model size, chosen as the <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-50.gif"/></noscript></span></span> quantile of the genome-wide maximum LOD score under the null hypothesis of no QTL, derived from a permutation test (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Churchill and Doerge 1994</a>). We may replace the LOD score in the above equation with any of the HKLOD, SL and ML scores.</p><p id="p-27">To search the space of models, we use the stepwise model search algorithm of <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Broman and Speed (2002)</a>: we use forward selection up to a model of fixed size (<em>e.g.</em>, ten QTL), followed by backward elimination to the null model. The selected model <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-51.gif"/></noscript></span></span> is that which maximizes the penalized LOD score criterion, among all models visited.</p><p id="p-28">The selected model is of the form <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-52.gif"/></noscript></span></span>, where <em>Z</em> contains the derived traits (the coefficients from the functional PCA) and <em>Q</em> contains an intercept column and genotype probabilities at the inferred QTL. The derived phenotypes, <em>Z</em>, are linearly related to the smoothed phenotypes, <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-53.gif"/></noscript></span></span>, by the equation <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-54.gif"/></noscript></span></span>, where Ψ is a matrix with (<em>i</em>,<em>j</em>)th element <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-55.gif"/></noscript></span></span>. Thus we have <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-56.gif"/></noscript></span></span>, and so <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-57.gif"/></noscript></span></span> are the estimated QTL effects, translated back to the time domain (see <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, below).</p></div></div><div class="section other1" id="sec-8"><h2 class="">Application</h2><p id="p-29">As an illustration of our approaches, we considered data from <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Moore <em>et al.</em> (2013)</a> on gravitropism in Arabidopsis RIL, Cape Verde Islands (Cvi) × Landsberg erecta (Ler). For each of 162 RIL, 8–20 replicate seeds per line were germinated and then rotated 90 degrees, to change the orientation of gravity. The growth of the seedlings was captured on video, over the course of 8 hr, and a number of phenotypes were derived by automated image analysis.</p><p id="p-30">We focus on the angle of the root tip, in degrees, over time (averaged across replicates within a RIL), and consider only the first of two replicate data sets examined in <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Moore <em>et al.</em> (2013)</a>. There is genotype data at 234 markers on five chromosomes; the function-valued root tip angle trait was measured at 241 time points (every 2 min for 8 hr).</p><p id="p-31">The data are available at the QTL Archive, which is now part of the Mouse Phenome Database, as the <em>Moore1b</em> data set: <a href="http://phenome.jax.org/db/q?rtn=projects/projdet&amp;reqprojid=282">http://phenome.jax.org/db/q?rtn=projects/projdet&amp;reqprojid=282</a>.</p><div id="sec-9" class="subsection"><h3>Single-QTL analysis</h3><p id="p-32">We first performed genome scans with a single-QTL model by multiple methods: SLOD and MLOD from <a id="xref-ref-8-8" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>, EE(Wald) and EE(Residual) from <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Xiong <em>et al.</em> (2011)</a>, and the HKLOD, SL, and ML methods described above (and using four PCs). We used a permutation test (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Churchill and Doerge 1994</a>) with 1000 permutation replicates to estimate 5% significance thresholds, which are shown in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024133/-/DC1/TableS1.pdf">Table S1</a>.</p><p id="p-33">The results are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The SLOD method (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>) gave similar results to the EE(Residual) method (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a>), with significant evidence for QTL on chromosomes 1, 4, and 5. The MLOD method (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>) also showed evidence for a QTL on chromosome 3. The EE(Wald), HKLOD, and SL methods (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, C, E, and F</a>) all gave similar results, with significant evidence for QTL on each chromosome. The ML method (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1G</a>) is different, with significant evidence for QTL on chromosomes 1, 2, and 4.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/79/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The SLOD, MLOD, EE(Wald) and EE(Residual), HKLOD, SL, and ML curves for the root tip angle data. A red horizontal line indicates the calculated 5% permutation-based threshold." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1291384687" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The SLOD, MLOD, EE(Wald) and EE(Residual), HKLOD, SL, and ML curves for the root tip angle data. A red horizontal line indicates the calculated 5% permutation-based threshold.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/79/F1.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/1/79/F1.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/79/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/79/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467466" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-34" class="first-child">The SLOD, MLOD, EE(Wald) and EE(Residual), HKLOD, SL, and ML curves for the root tip angle data. A red horizontal line indicates the calculated 5% permutation-based threshold.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Multiple-QTL analysis</h3><p id="p-35">We further applied multiple-QTL analysis, extending the HKLOD, SL, and ML methods to use the penalized LOD score criterion of <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Broman and Speed (2002)</a> for function-valued traits. We focused on additive QTL models, and used the 5% permutation-based significance thresholds (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024133/-/DC1/TableS1.pdf">Table S1</a>) as the penalties.</p><p id="p-36">The penalized-HKLOD and penalized-SL criteria each indicated a five-QTL model, with a QTL on each chromosome. The inferred positions of the QTL showed only slight differences. The penalized-SL criterion indicated a two-QTL model, with QTL on chromosomes 1 and 4.</p><p id="p-37">LOD profiles for these models are displayed in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. These curves, which visualize both the evidence and localization of each QTL in the context of a multiple-QTL model, are calculated following an approach developed by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Zeng <em>et al.</em> (2000)</a>: the position of each QTL was varied one at a time, and, at each location for a given QTL, we derived a LOD-type score comparing the multiple-QTL model with the QTL under consideration at a particular position and the locations of all other QTL fixed, to the model with the given QTL omitted. For the SL (or ML) method, the profile is calculated for the four derived traits, individually, and then the SL (or ML) profiles are obtained by averaging (or maximizing) across traits. For the HKLOD method, the profiles are calculated using the multivariate LOD test statistic.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/79/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="HKLOD, SL, and ML profiles for a multiple-QTL model with the root tip angle data set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1291384687" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;HKLOD, SL, and ML profiles for a multiple-QTL model with the root tip angle data set.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/79/F2.medium.gif" width="356" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/1/79/F2.medium.gif" width="356" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/79/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/79/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467407" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-38" class="first-child">HKLOD, SL, and ML profiles for a multiple-QTL model with the root tip angle data set.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">In <a id="xref-ref-8-9" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>, we applied, to these same data, the analogous multiple-QTL modeling approach, with the penalized-SLOD and penalized-MLOD criteria (that is, using the original phenotype data, without the smoothing and dimension-reduction steps). The result (see Figure 3 in <a id="xref-ref-8-10" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> 2014</a>) with the penalized-SLOD criterion was the same two-QTL model identified by ML (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2C</a>), while the penalized-MLOD criterion gave a three-QTL model, with a QTL on chromosome 3, similar to that inferred by single-QTL analysis with MLOD (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1B</a>).</p><p id="p-40">The estimated QTL effects, translated back to the time domain, are displayed in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. The red curves are for the five-QTL model identified with the penalized-HKLOD and penalized-SL criteria. The blue dashed curves are for the two-QTL model identified with the penalized-ML criterion. The estimated QTL effects in panels B–F are for the difference between the Cvi allele and the Ler allele.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/79/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The estimated QTL effects for the root tip angle data set. The red curves are for the five-QTL model (from the penalized-HKLOD and penalized-SL criteria) and the blue dashed curves are for the two-QTL model (from the penalized-ML criterion). Positive values for the QTL effects indicate that the Cvi allele increases the tip angle phenotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1291384687" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The estimated QTL effects for the root tip angle data set. The red curves are for the five-QTL model (from the penalized-HKLOD and penalized-SL criteria) and the blue dashed curves are for the two-QTL model (from the penalized-ML criterion). Positive values for the QTL effects indicate that the Cvi allele increases the tip angle phenotype.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/79/F3.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/1/79/F3.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/79/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/79/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467376" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-41" class="first-child">The estimated QTL effects for the root tip angle data set. The red curves are for the five-QTL model (from the penalized-HKLOD and penalized-SL criteria) and the blue dashed curves are for the two-QTL model (from the penalized-ML criterion). Positive values for the QTL effects indicate that the Cvi allele increases the tip angle phenotype.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">The effects of the QTL on chromosomes 1 and 4 are approximately the same, whether or not the chromosome 2, 3, and 5 QTL are included in the model. The chromosome 1 QTL has greatest effect at later time points, while the chromosome 4 QTL has greatest effect earlier and over a wider interval of time. For both QTL, the Cvi allele increases the root tip angle phenotype.</p><p id="p-43">In summary, the HKLOD and SL methods gave similar results. For these data, the HKLOD and SL methods indicate evidence for a QTL on each chromosome. The results suggest that these approaches, with the initial dimension-reduction via functional PCA, may have higher power to detect QTL, as the multiple-QTL analysis with SLOD and MLOD, without dimension-reduction, indicated fewer QTL.</p></div></div><div class="section other2" id="sec-11"><h2 class="">Simulations</h2><p id="p-44">In order to investigate the performance of our proposed approaches and compare them to existing methods, we performed computer simulation studies with both single-QTL models and models with multiple QTL. For the simulations with a single-QTL model, we compared the HKLOD, SL, and ML methods to the SLOD and MLOD methods of <a id="xref-ref-8-11" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a> and the estimating equation approaches of <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Xiong <em>et al.</em> (2011)</a>. For the simulations with multiple QTL, we omitted the methods of <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Xiong <em>et al.</em> (2011)</a>, as they have not yet been implemented for multiple QTL.</p><div id="sec-12" class="subsection"><h3>Single-QTL models</h3><p id="p-45">To compare approaches in the context of a single QTL, we considered the simulation setting described in <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Yap <em>et al.</em> (2009)</a>, though exploring a range of QTL effects.</p><p id="p-46">We simulated an intercross with sample sizes of 100, 200, or 400, and a single chromosome of length 100 cM, with six equally spaced markers and with a QTL at 32 cM. The associated phenotypes were sampled from a multivariate normal distribution with the mean curve following a logistic function, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-58.gif"/></noscript></span></span>. The AA genotype had <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-59.gif"/></noscript></span></span>; the AB genotype had <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-60.gif"/></noscript></span></span>; and the BB genotype had <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-61.gif"/></noscript></span></span>. The shape of the growth curve with this parameter was shown in Figure S3 of <a id="xref-ref-8-12" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>. The phenotype data were simulated at ten time points.</p><p id="p-47">The residual error was assumed to follow a multivariate normal distribution with covariance structure <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-62.gif"/></noscript></span></span>. The constant <em>c</em> controls the overall error variance, and Σ was chosen to have one of three forms: (1) auto-regressive with <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-63.gif"/></noscript></span></span>, (2) equicorrelated with <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-64.gif"/></noscript></span></span>, or (3) an “unstructured” covariance matrix, as given in <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Yap <em>et al.</em> (2009)</a> (also shown in Table S2 of <a id="xref-ref-8-13" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> 2014</a>).</p><p id="p-48">The parameter <em>c</em> was given a range of values, which define the percent phenotypic variance explained by the QTL (the heritability). The effect of the QTL varies with time; we took the mean heritability across time as an overall summary. For the auto-regressive and equicorrelated covariance structures, we used <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-65.gif"/></noscript></span></span>; for the unstructured covariance matrix, we took <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-66.gif"/></noscript></span></span>.</p><p id="p-49">For each of 10,000 simulation replicates, we applied the previous SLOD and MLOD methods of <a id="xref-ref-8-14" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>, the EE(Wald) and EE(Residual) methods of <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Xiong <em>et al.</em> (2011)</a>, and the HKLOD, SL, and ML methods proposed here. For all seven approaches, we fit a three-parameter QTL model (that is, allowing for dominance).</p><p id="p-50">The estimated power to detect the QTL as a function of heritability due to the QTL, for <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-67.gif"/></noscript></span></span> and for the three different covariance structures, is shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. With the autocorrelated variance structure, all methods worked well, though HKLOD had noticeably lower power. With the equicorrelated variance structure, EE(Wald), SL and ML methods had higher power than the other four methods. The HKLOD method also worked reasonably well, but the EE(Residual), SLOD and MLOD methods had low power. In the unstructured variance setting, EE(Wald), MLOD, SL and ML methods worked better than the other three methods. EE(Residual) did not work well in this setting.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/79/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Power as a function of the percent phenotypic variance explained by a single QTL. The three columns correspond to sample sizes of n&#xA0;=&#xA0;100, 200, and 400, respectively. The three rows correspond to the covariance structures (autocorrelated, equicorrelated, and unstructured). In the legends on the right, the methods are sorted by their overall average power for the corresponding covariance structure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1291384687" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Power as a function of the percent phenotypic variance explained by a single QTL. The three columns correspond to sample sizes of &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;100, 200, and 400, respectively. The three rows correspond to the covariance structures (autocorrelated, equicorrelated, and unstructured). In the legends on the right, the methods are sorted by their overall average power for the corresponding covariance structure.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/79/F4.medium.gif" width="405" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/1/79/F4.medium.gif" width="405" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/79/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/79/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467480" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-51" class="first-child">Power as a function of the percent phenotypic variance explained by a single QTL. The three columns correspond to sample sizes of <em>n</em> = 100, 200, and 400, respectively. The three rows correspond to the covariance structures (autocorrelated, equicorrelated, and unstructured). In the legends on the right, the methods are sorted by their overall average power for the corresponding covariance structure.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">The precision of QTL mapping, measured by the root mean square (RMS) error in the estimated QTL position, is displayed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024133/-/DC1/FigureS1.pdf">Figure S1</a>. Performance, in terms of precision, corresponds quite closely to performance in terms of power: when power is high, the RMS error of the estimated QTL position is low, and vice versa.</p><p id="p-53">A possible weakness of the SLOD and MLOD approaches was that they do not make use of the function-valued form of the phenotypes. The methods may further suffer lower power in the case of noisy phenotypes. The methods proposed in this paper use smoothing to handle the measurement error. We repeated the same simulations in <a id="xref-ref-8-15" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a> with <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-68.gif"/></noscript></span></span>, adding independent, normally distributed errors (with standard deviation 1 or 2) at each time point.</p><p id="p-54">The estimated power to detect the QTL as a function of heritability due to the QTL, for added noise with SD <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-69.gif"/></noscript></span></span> and the three different covariance structures, is shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. The power of the SLOD and MLOD were greatly affected by the introduction of noise. However, the SL and ML methods, which used both smoothing and dimension-reduction, work well in the presence of noise. The EE(Wald) method performed well in this case, as well. The EE(Residual) method did not work well compared to the other six methods. Overall, the EE(Wald) method continued to perform best.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/79/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Power as a function of the percent phenotypic variance explained by a single QTL, with additional noise added to the phenotypes. The first column has no additional noise; the second and third columns have independent normally distributed noise added at each time point, with standard deviation 1 and 2, respectively. The percent variance explained by the QTL on the x-axis refers, in each case, to the variance explained in the case of no added noise. The three rows correspond to the covariance structures (autocorrelated, equicorrelated, and unstructured). In the legends on the right, the methods are sorted by their overall average power for the corresponding covariance structure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1291384687" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Power as a function of the percent phenotypic variance explained by a single QTL, with additional noise added to the phenotypes. The first column has no additional noise; the second and third columns have independent normally distributed noise added at each time point, with standard deviation 1 and 2, respectively. The percent variance explained by the QTL on the &lt;em&gt;x&lt;/em&gt;-axis refers, in each case, to the variance explained in the case of no added noise. The three rows correspond to the covariance structures (autocorrelated, equicorrelated, and unstructured). In the legends on the right, the methods are sorted by their overall average power for the corresponding covariance structure.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/79/F5.medium.gif" width="405" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/1/79/F5.medium.gif" width="405" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/79/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/79/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467434" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-55" class="first-child">Power as a function of the percent phenotypic variance explained by a single QTL, with additional noise added to the phenotypes. The first column has no additional noise; the second and third columns have independent normally distributed noise added at each time point, with standard deviation 1 and 2, respectively. The percent variance explained by the QTL on the <em>x</em>-axis refers, in each case, to the variance explained in the case of no added noise. The three rows correspond to the covariance structures (autocorrelated, equicorrelated, and unstructured). In the legends on the right, the methods are sorted by their overall average power for the corresponding covariance structure.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows the average computation time for SLOD/MLOD, EE(Wald), EE(Residual), HKLOD and SL/ML methods. For the single-QTL simulations, the computation time for the SLOD/MLOD and HKLOD methods are similar, while the SL/ML methods are somewhat slower. This is because, in the simulation data set, the phenotype was measured at ten time points. In the application, with 241 time points, the functional PCA-based methods (HKLOD and SL/ML) are faster than the SLOD/MLOD methods. The EE(Wald) method requires considerably longer computation time; on the other hand, it provided the highest power to detect QTL.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467468/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467468/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Average computation time for each method, in the single-QTL simulations and in the application</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Multiple-QTL models</h3><p id="p-57">To investigate the performance of the penalized-HKLOD, penalized-SL and penalized-ML criteria in the context of multiple QTL, we used the same setting as in <a id="xref-ref-8-16" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>. We simulated data from a three-QTL model modeled after that estimated, with the penalized-MLOD criterion, for the root tip angle data of <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Moore <em>et al.</em> (2013)</a> considered in the <em>Application</em> section.</p><p id="p-58">We assumed that the mean curve for the root tip angle phenotype followed a cubic polynomial, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-70.gif"/></noscript></span></span>, and assumed that the effect of each QTL also followed such a cubic polynomial. The four parameters for a given individual were drawn from a multivariate normal distribution with mean defined by the QTL genotypes and variance matrix estimated from the root tip angle data. Details on the parameter values used in the simulations appear in <a id="xref-ref-8-17" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>.</p><p id="p-59">Normally distributed measurement error (with mean 0 and variance 1) was added to the phenotype at each time point for each individual. Phenotypes are taken at 241 equally spaced time points in the interval of 0 to 1. We considered two sample sizes: <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-71.gif"/></noscript></span></span> [as in the <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Moore <em>et al.</em> (2013)</a> data] and twice that, <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/79/embed/mml-math-72.gif"/></noscript></span></span>.</p><p id="p-60">We performed 100 simulation replicates. For each replicate, we applied a stepwise model selection approach with each of the penalized-HKLOD, penalized-SL, penalized-ML, penalized-SLOD, and penalized-MLOD criteria. The simulation results are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467505/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467505/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Power to detect QTL, for a three-QTL model modeled after the data of <a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Moore <em>et al.</em> (2013)</a></span><div class="sb-div caption-clear"></div></div></div><p id="p-61">The penalized-SL and penalized-HKLOD criteria had higher power to detect all three QTL. In particular, the power to detect the QTL on chromosome 3 was greatly increased, in comparison to the penalized-SLOD and penalized-MLOD criteria.</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-62">We have described two techniques for improving the regression-based methods of <a id="xref-ref-8-18" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a> for QTL mapping with function-valued phenotypes: smoothing and dimensional-reduction. Smoothing leads to better performance in the case of noisy phenotype measurements (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Li and Sillanpää 2013</a>), and dimension-reduction improves power. The particular methods we used (smoothing via B-splines, and dimensional reduction via functional PCA) are not the only possibilities, but they are natural choices widely used in functional data analysis (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Ramsay and Silverman 2005</a>).</p><p id="p-63">Following smoothing and dimensional-reduction, we applied QTL analysis to the small number of derived traits, either by analyzing the traits individually and then combining the log likelihoods, as in <a id="xref-ref-8-19" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>, or by applying the multivariate QTL mapping method of <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Knott and Haley (2000)</a>. The latter approach cannot be applied directly to the original phenotypes, due to the large number of time points at which the traits were measured, but it can work well with the dimension-reduced derived traits.</p><p id="p-64">Key advantages of our proposed methods include speed of computation, and the ability to consider multiple-QTL models. The EE(Wald) method of <a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Xiong <em>et al.</em> (2011)</a>, based on estimating equations, was seen to be most powerful for QTL detection in our simulation study, but it is orders of magnitude slower and has not yet been implemented for multiple-QTL models.</p><p id="p-65">Many other methods have been developed for QTL mapping with function-valued traits. However, most focus on single-QTL models (<em>e.g.</em>, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Ma <em>et al.</em> 2002</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Yang <em>et al.</em> 2009</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Yap <em>et al.</em> 2009</a>). Bayesian methods for multiple-QTL mapping with function-valued traits have been proposed (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Min <em>et al.</em> 2011</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Sillanpää <em>et al.</em> 2012</a>; <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Li and Sillanpää 2013</a>), but these methods are computationally intensive, and software is not available.</p><p id="p-66">In considering multiple-QTL models, we have focused on strictly additive QTL. <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Manichaikul <em>et al.</em> (2009)</a> extended the work of <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Broman and Speed (2002)</a> by considering pairwise interactions among QTL. Our approaches may be similarly extended to handle interactions.</p><p id="p-67">The enormous recent growth in capabilities for high-throughput phenotyping, including images and time series, particularly in plants (see, for example, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cabrera-Bosquet <em>et al.</em> 2012</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Araus and Cairns 2014</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ghanem <em>et al.</em> 2015</a>), is accompanied by a growth in interest in the genetic analysis of such phenotype data. Speed of computation will be particularly important in the analysis of such high-dimensional data, as will the joint consideration of multiple loci. The methods we have proposed can meet many of these challenges.</p><p id="p-68">Our efforts on this problem were inspired by the data of <a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Moore <em>et al.</em> (2013)</a>, in which the phenotype was measured at a large number of time points. While our approaches do not require such a high density of time points, we expect that the use of smoothing and functional PCA will be most suitable in the case of at least eight to ten time points. In the case of a smaller number of time points, we would recommend the direct use of multivariate QTL analysis, or of the SLOD and MLOD methods of <a id="xref-ref-8-20" class="xref-bibr" href="#ref-8">Kwak <em>et al.</em> (2014)</a>.</p><p id="p-69">We implemented our methods as a package, funqtl, for the general statistical software R (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">R Core Team 2015</a>). Our package makes use of the fda package (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ramsay <em>et al.</em> 2014</a>) for smoothing and functional PCA. It is available at <a href="https://github.com/ikwak2/funqtl">https://github.com/ikwak2/funqtl</a>.</p></div><div class="section" id="sec-15"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-70">The authors thank Nathan Miller and two anonymous reviewers for suggestions to improve the manuscript. This work was supported in part by grant IOS-1031416 from the National Science Foundation Plant Genome Research Program to E.P.S. and by National Institutes of Health grant R01GM074244 to K.W.B.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-2"><p id="p-71">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024133/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024133/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-72"><em>Communicating editor: L. M. Steinmetz</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-08-25"><span class="received-label">Received </span>August 25, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-10-28"><span class="accepted-label">Accepted </span>October 28, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Kwak <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.1" data-doi="10.1016/j.tplants.2013.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araus</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Field high-throughput phenotyping: the new crop breeding frontier.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2013.09.008%26rft_id%253Dinfo%253Apmid%252F24139902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2013.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24139902&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330013700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.2" data-doi="10.1111/1467-9868.00354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A model selection approach for the identification of quantitative trait loci in experimental crosses (with discussion).</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">656, 737–775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D64%26rft.spage%253D641%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.3" data-doi="10.1111/j.1744-7909.2012.01116.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cabrera-Bosquet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Zitzewitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serret</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luis Araus</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-throughput phenotyping and genomic selection: The frontiers of crop breeding converge.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">312</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BIntegr%2BPlant%2BBiol%26rft.aulast%253DCabrera-Bosquet%26rft.auinit1%253DL.%26rft.volume%253D54%26rft.issue%253D5%26rft.spage%253D312%26rft.epage%253D320%26rft.atitle%253DHigh-throughput%2Bphenotyping%2Band%2Bgenomic%2Bselection%253A%2Bthe%2Bfrontiers%2Bof%2Bcrop%2Bbreeding%2Bconverge.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7909.2012.01116.x%26rft_id%253Dinfo%253Apmid%252F22420640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7909.2012.01116.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22420640&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.5" data-doi="10.1016/j.tplants.2014.11.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghanem</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marrou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinclair</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Physiological phenotyping of plants for crop improvement.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D20%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.11.006%26rft_id%253Dinfo%253Apmid%252F25524213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.11.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25524213&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.6" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHaley%26rft.auinit1%253DC.%2BS.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D324%26rft.atitle%253DA%2Bsimple%2Bregression%2Bmethod%2Bfor%2Bmapping%2Bquantitative%2Btrait%2Bloci%2Bin%2Bline%2Bcrosses%2Busing%2Bflanking%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F16718932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1992.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16718932&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JQ49200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Multitrait least squares for quantitative trait loci detection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">899</span>–<span class="cit-lpage">911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKnott%26rft.auinit1%253DS.%2BA.%26rft.volume%253D156%26rft.issue%253D2%26rft.spage%253D899%26rft.epage%253D911%26rft.atitle%253DMultitrait%2BLeast%2BSquares%2Bfor%2BQuantitative%2BTrait%2BLoci%2BDetection%26rft_id%253Dinfo%253Apmid%252F11014835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU2LzIvODk5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-other" id="cit-6.1.79.8" data-doi="10.1534/genetics.114.166306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kwak</span>  <span class="cit-name-given-names">I.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spalding</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> A simple regression-based method to map quantitative trait loci underlying function-valued phenotypes. <span class="cit-source">Genetics</span> <span class="cit-vol">197</span>: <span class="cit-fpage">1409</span>–<span class="cit-lpage">1416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166306%26rft_id%253Dinfo%253Apmid%252F24931408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny80LzE0MDkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.9" data-doi="10.1534/genetics.113.152736"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A Bayesian nonparametric approach for mapping dynamic quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152736%26rft_id%253Dinfo%253Apmid%252F23770698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvOTk3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional mapping of quantitative trait loci underlying the character process: a theoretical framework.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">1751</span>–<span class="cit-lpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMa%26rft.auinit1%253DC.-X.%26rft.volume%253D161%26rft.issue%253D4%26rft.spage%253D1751%26rft.epage%253D1762%26rft.atitle%253DFunctional%2BMapping%2Bof%2BQuantitative%2BTrait%2BLoci%2BUnderlying%2Bthe%2BCharacter%2BProcess%253A%2BA%2BTheoretical%2BFramework%26rft_id%253Dinfo%253Apmid%252F12196415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MS80LzE3NTEiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.11" data-doi="10.1534/genetics.108.094565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manichaikul</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moon</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A model selection approach for the identification of quantitative trait loci in experimental crosses, allowing epistasis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.094565%26rft_id%253Dinfo%253Apmid%252F19104078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MS8zLzEwNzciO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.12" data-doi="10.1038/hdy.2010.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Min</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bayesian analysis for genetic architecture of dynamic traits.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">124</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMin%26rft.auinit1%253DL.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D124%26rft.epage%253D133%26rft.atitle%253DBayesian%2Banalysis%2Bfor%2Bgenetic%2Barchitecture%2Bof%2Bdynamic%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.20%26rft_id%253Dinfo%253Apmid%252F20332806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.20&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20332806&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285336100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.13" data-doi="10.1534/genetics.113.153346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwak</span>  <span class="cit-name-given-names">I.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Livny</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-throughput computer vision introduces the time axis to a quantitative trait map of a plant growth response.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.153346%26rft_id%253Dinfo%253Apmid%252F23979570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzEwNzciO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.79.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source">R: A language and environment for statistical computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.79.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Functional data analysis</span>, <span class="cit-edition">Ed. 2nd</span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-web" id="cit-6.1.79.16"><div class="cit-metadata unstructured">Ramsay, J. O., H. Wickham, S. Graves, and G. Hooker, 2014 <em>fda: Functional data analysis</em>. R package version 2.4.4, <a href="http://cran.r-project.org/package=fda">http://cran.r-project.org/package=fda</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.17" data-doi="10.1038/hdy.2011.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pikkuhookana</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrahamsson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knurr</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fries</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Simultaneous estimation of multiple quantitative trait loci and growth curve parameters through hierarchical Bayesian modeling.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">134</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DSillanpaa%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D108%26rft.issue%253D2%26rft.spage%253D134%26rft.epage%253D146%26rft.atitle%253DSimultaneous%2Bestimation%2Bof%2Bmultiple%2Bquantitative%2Btrait%2Bloci%2Band%2Bgrowth%2Bcurve%2Bparameters%2Bthrough%2Bhierarchical%2BBayesian%2Bmodeling.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.56%26rft_id%253Dinfo%253Apmid%252F21792229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21792229&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.18" data-doi="10.1534/genetics.111.129221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goulding</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tecott</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCulloch</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A flexible estimating equations approach for mapping function-valued traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.129221%26rft_id%253Dinfo%253Apmid%252F21705752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzEvMzA1IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.19" data-doi="10.1111/j.1541-0420.2008.01063.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Nonparametric functional mapping of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D65%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1541-0420.2008.01063.x%26rft_id%253Dinfo%253Apmid%252F18537950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1541-0420.2008.01063.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18537950&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.20" data-doi="10.1111/j.1541-0420.2009.01222.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yap</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Nonparametric modeling of longitudinal covariance structure in functional mapping of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D65%26rft.spage%253D1068%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1541-0420.2009.01222.x%26rft_id%253Dinfo%253Apmid%252F19302406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1541-0420.2009.01222.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19302406&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272128200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.79.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercer</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetic architecture of a morphological shape difference between two Drosophila species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZeng%26rft.auinit1%253DZ.-B.%26rft.volume%253D154%26rft.issue%253D1%26rft.spage%253D299%26rft.epage%253D310%26rft.atitle%253DGenetic%2BArchitecture%2Bof%2Ba%2BMorphological%2BShape%2BDifference%2BBetween%2BTwo%2BDrosophila%2BSpecies%26rft_id%253Dinfo%253Apmid%252F10628989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzEvMjk5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/1/79.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/1/67" title="Target Enrichment Improves Mapping of Complex Traits by Deep Sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/87" title="Identification of a Novel Nematotoxic Protein by Challenging the Model Mushroom Coprinopsis cinerea with a Fungivorous Nematode" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/1"><span>Volume 6 Issue 1, January 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/1.cover-source.jpg?itok=9zFiZvCK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/79" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jqEthSg8MTnqd9Xz7DjsOSagL7pIaKdzldyrx7WVju8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/1/79" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/1/79" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/1/79" class="active" data-icon-position="" data-hide-link-title="0">Mapping Quantitative Trait Loci Underlying Function-Valued Traits Using Functional Principal Component Analysis and Multi-Trait Mapping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460249" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9q7HYI8QX3SiTnCMgODr7z1jgSiPprvQZ_PaG_q2fy8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/79.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/1/79" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-N_lgtrxu38rQSGRRuccoCg50IR6qIMIikxIFrhORuSc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/1/79" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/79.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460249" id="node460249--2" data-pisa="ggg;6/1/79" data-pisa-master="ggg;g3.115.024133" data-apath="/ggg/6/1/79.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Quantitative Trait Loci Underlying Function-Valued Traits Using Functional Principal Component Analysis and Multi-Trait Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7117-7669" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Il-Youp Kwak</span>, <span class="highwire-citation-author" data-delta="1">Candace R. Moore</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6890-4765" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Edgar P. Spalding</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl W. Broman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">79-86;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024133 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460249--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Il-Youp Kwak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Il-Youp%2BKwak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kwak%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIl-Youp%2BKwak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7117-7669" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Il-Youp Kwak</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Candace R. Moore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Candace%2BR.%2BMoore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moore%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACandace%2BR.%2BMoore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Edgar P. Spalding </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edgar%2BP.%2BSpalding%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spalding%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdgar%2BP.%2BSpalding%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6890-4765" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Edgar P. Spalding</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl W. Broman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kbroman@biostat.wisc.edu" class="" data-icon-position="" data-hide-link-title="0">kbroman@biostat.wisc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/1/79" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/1/79&amp;title=Mapping%20Quantitative%20Trait%20Loci%20Underlying%20Function-Valued%20Traits%20Using%20Functional%20Principal%20Component%20Analysis%20and%20Multi-Trait%20Mapping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/1/79&amp;title=Mapping%20Quantitative%20Trait%20Loci%20Underlying%20Function-Valued%20Traits%20Using%20Functional%20Principal%20Component%20Analysis%20and%20Multi-Trait%20Mapping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/1/79&amp;title=Mapping%20Quantitative%20Trait%20Loci%20Underlying%20Function-Valued%20Traits%20Using%20Functional%20Principal%20Component%20Analysis%20and%20Multi-Trait%20Mapping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/1/79&amp;text=Mapping%20Quantitative%20Trait%20Loci%20Underlying%20Function-Valued%20Traits%20Using%20Functional%20Principal%20Component%20Analysis%20and%20Multi-Trait%20Mapping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/1/79&amp;title=Mapping%20Quantitative%20Trait%20Loci%20Underlying%20Function-Valued%20Traits%20Using%20Functional%20Principal%20Component%20Analysis%20and%20Multi-Trait%20Mapping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/1/79&amp;t=Mapping%20Quantitative%20Trait%20Loci%20Underlying%20Function-Valued%20Traits%20Using%20Functional%20Principal%20Component%20Analysis%20and%20Multi-Trait%20Mapping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/1/79&amp;title=Mapping%20Quantitative%20Trait%20Loci%20Underlying%20Function-Valued%20Traits%20Using%20Functional%20Principal%20Component%20Analysis%20and%20Multi-Trait%20Mapping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/1/79&amp;title=Mapping%20Quantitative%20Trait%20Loci%20Underlying%20Function-Valued%20Traits%20Using%20Functional%20Principal%20Component%20Analysis%20and%20Multi-Trait%20Mapping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460249/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460249" id="node460249--3" data-pisa="ggg;6/1/79" data-pisa-master="ggg;g3.115.024133" data-apath="/ggg/6/1/79.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Quantitative Trait Loci Underlying Function-Valued Traits Using Functional Principal Component Analysis and Multi-Trait Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7117-7669" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Il-Youp Kwak</span>, <span class="highwire-citation-author" data-delta="1">Candace R. Moore</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6890-4765" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Edgar P. Spalding</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl W. Broman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">79-86;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024133 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460249--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Il-Youp Kwak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Il-Youp%2BKwak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kwak%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIl-Youp%2BKwak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7117-7669" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Il-Youp Kwak</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Candace R. Moore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Candace%2BR.%2BMoore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moore%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACandace%2BR.%2BMoore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Edgar P. Spalding </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Botany, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edgar%2BP.%2BSpalding%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spalding%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdgar%2BP.%2BSpalding%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6890-4765" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Edgar P. Spalding</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl W. Broman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kbroman@biostat.wisc.edu" class="" data-icon-position="" data-hide-link-title="0">kbroman@biostat.wisc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=79&amp;rft.epage=86&amp;rft.atitle=Mapping%20Quantitative%20Trait%20Loci%20Underlying%20Function-Valued%20Traits%20Using%20Functional%20Principal%20Component%20Analysis%20and%20Multi-Trait%20Mapping&amp;rft.volume=6&amp;rft.issue=1&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kwak%2C+Il-Youp&amp;rft.au=Moore%2C+Candace+R.&amp;rft.au=Spalding%2C+Edgar+P.&amp;rft.au=Broman%2C+Karl+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460249/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460249/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460249/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460249/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460249/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460249/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460249/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460249/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460249/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460249/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460249/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460249/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/1/79" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Application</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Simulations</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/1/79.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/1/79.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/1/79.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/1/79.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
