<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/8/2577" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/8/2577.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/8/2577.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/8/2577.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/8/2577.atom" />
<meta name="HW.pisa" content="ggg;7/8/2577" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Transcriptional Complexity and Distinct Expression Patterns of auts2 Paralogs in Danio rerio" />
<meta name="DC.Identifier" content="10.1534/g3.117.042622" />
<meta name="DC.Date" content="2017-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Kondrychyn et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Several genes that have been implicated in autism spectrum disorders (ASDs) have multiple transcripts. Therefore, comprehensive transcript annotation is critical for determining the respective gene function. The autism susceptibility candidate 2 ( AUTS2 ) gene is associated with various neurological disorders, including autism and brain malformation. AUTS2 is important for activation of transcription of neural specific genes, neuronal migration, and neurite outgrowth. Here, we present evidence for significant transcriptional complexity in the auts2 gene locus in the zebrafish genome, as well as in genomic loci of auts2 paralogous genes fbrsl1 and fbrs . Several genes that have been implicated in ASDs are large and have multiple transcripts. Neurons are especially enriched with longer transcripts compared to nonneural cell types. The human autism susceptibility candidate 2 ( AUTS2 ) gene is ∼1.2 Mb long and is implicated in a number of neurological disorders including autism, intellectual disability, addiction, and developmental delay. Recent studies show AUTS2 to be important for activation of transcription of neural specific genes, neuronal migration, and neurite outgrowth. However, much remains to be understood regarding the transcriptional complexity and the functional roles of AUTS2 in neurodevelopment. Zebrafish provide an excellent model system for studying both these questions. We undertook genomic identification and characterization of auts2 and its paralogous genes in zebrafish. There are four auts2 family genes in zebrafish: auts2a , auts2b , fbrsl1 , and fbrs . The absence of complete annotation of their structures hampers functional studies. We present evidence for transcriptional complexity of these four genes mediated by alternative splicing and alternative promoter usage. Furthermore, the expression of the various paralogs is tightly regulated both spatially and developmentally. Our findings suggest that auts2 paralogs serve distinct functions in the development and functioning of target tissues." />
<meta name="DC.Contributor" content="Igor Kondrychyn" />
<meta name="DC.Contributor" content="Lena Robra" />
<meta name="DC.Contributor" content="Vatsala Thirumalai" />
<meta name="article:published_time" content="2017-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Transcriptional Complexity and Distinct Expression Patterns of auts2 Paralogs in Danio rerio" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Several genes that have been implicated in autism spectrum disorders (ASDs) have multiple transcripts. Therefore, comprehensive transcript annotation is critical for determining the respective gene function. The autism susceptibility candidate 2 (&lt;i&gt;AUTS2&lt;/i&gt;) gene is associated with various neurological disorders, including autism and brain malformation. AUTS2 is important for activation of transcription of neural specific genes, neuronal migration, and neurite outgrowth. Here, we present evidence for significant transcriptional complexity in the &lt;i&gt;auts2&lt;/i&gt; gene locus in the zebrafish genome, as well as in genomic loci of &lt;i&gt;auts2&lt;/i&gt; paralogous genes &lt;i&gt;fbrsl1&lt;/i&gt; and &lt;i&gt;fbrs&lt;/i&gt;. Several genes that have been implicated in ASDs are large and have multiple transcripts. Neurons are especially enriched with longer transcripts compared to nonneural cell types. The human autism susceptibility candidate 2 (&lt;i&gt;AUTS2&lt;/i&gt;) gene is ∼1.2 Mb long and is implicated in a number of neurological disorders including autism, intellectual disability, addiction, and developmental delay. Recent studies show AUTS2 to be important for activation of transcription of neural specific genes, neuronal migration, and neurite outgrowth. However, much remains to be understood regarding the transcriptional complexity and the functional roles of AUTS2 in neurodevelopment. Zebrafish provide an excellent model system for studying both these questions. We undertook genomic identification and characterization of &lt;i&gt;auts2&lt;/i&gt; and its paralogous genes in zebrafish. There are four &lt;i&gt;auts2&lt;/i&gt; family genes in zebrafish: &lt;i&gt;auts2a&lt;/i&gt;, &lt;i&gt;auts2b&lt;/i&gt;, &lt;i&gt;fbrsl1&lt;/i&gt;, and &lt;i&gt;fbrs&lt;/i&gt;. The absence of complete annotation of their structures hampers functional studies. We present evidence for transcriptional complexity of these four genes mediated by alternative splicing and alternative promoter usage. Furthermore, the expression of the various paralogs is tightly regulated both spatially and developmentally. Our findings suggest that auts2 paralogs serve distinct functions in the development and functioning of target tissues.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/08/01" />
<meta name="citation_mjid" content="ggg;7/8/2577" />
<meta name="citation_id" content="7/8/2577" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/8/2577" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/8/2577.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/8/2577.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/8/2577.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.042622" />
<meta name="citation_pmid" content="28626003" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2577" />
<meta name="citation_lastpage" content="2593" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Igor Kondrychyn" />
<meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bangalore, Karnataka 560065, India" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2268-7291" />
<meta name="citation_author" content="Lena Robra" />
<meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bangalore, Karnataka 560065, India" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3510-9228" />
<meta name="citation_author" content="Vatsala Thirumalai" />
<meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bangalore, Karnataka 560065, India" />
<meta name="citation_author_email" content="vatsala@ncbs.res.in" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2318-5023" />
<meta name="citation_reference" content="citation_journal_abbrev=Comp Funct Genomics;citation_author=P. R. Araujo;citation_author=K. Yoon;citation_author=D. Ko;citation_author=A. D. Smith;citation_author=M. Qiao;citation_title=Before It Gets Started: Regulating Translation at the 5' UTR.;citation_pages=475731-475731;citation_volume=2012;citation_year=2012;citation_pmid=22693426;citation_doi=10.1155/2012/475731" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. Bakkaloglu;citation_author=B. J. O’Roak;citation_author=A. Louvi;citation_author=A. R. Gupta;citation_author=J. F. Abelson;citation_title=Molecular cytogenetic analysis and resequencing of contactin associated protein-like 2 in autism spectrum disorders.;citation_pages=165-173;citation_volume=282;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. G. Baltz;citation_author=M. Munschauer;citation_author=B. Schwanhäusser;citation_author=A. Vasile;citation_author=Y. Murakawa;citation_title=The mRNA-bound proteome and its global occupancy profile on protein-coding transcripts.;citation_pages=674-690;citation_volume=46;citation_year=2012;citation_issue=5;citation_pmid=22681889;citation_doi=10.1016/j.molcel.2012.05.021" />
<meta name="citation_reference" content="citation_journal_title=Brain research. Gene expression patterns;citation_journal_abbrev=Brain Res Gene Expr Patterns;citation_author=F. Bedogni;citation_author=R. D. Hodge;citation_author=B. R. Nelson;citation_author=E. A. Frederick;citation_author=N. Shiba;citation_title=Autism susceptibility candidate 2 (Auts2) encodes a nuclear protein expressed in developing brain regions implicated in autism neuropathology.;citation_pages=9-15;citation_volume=10;citation_year=2010;citation_issue=1;citation_pmid=19948250;citation_doi=10.1016/j.gep.2009.11.005" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Hum. Genet.;citation_author=G. Beunders;citation_author=S. A. de Munnik;citation_author=N. Van der Aa;citation_author=B. Ceulemans;citation_author=E. Voorhoeve;citation_title=Two male adults with pathogenic AUTS2 variants, including a two-base pair deletion, further delineate the AUTS2 syndrome.;citation_pages=803-807;citation_volume=23;citation_year=2015;citation_pmid=25205402;citation_doi=10.1038/ejhg.2014.173" />
<meta name="citation_reference" content="citation_journal_title=Lab. Med.;citation_author=M. Boroumand;citation_author=S. Ziaee;citation_author=N. Zarghami;citation_author=M. S. Anvari;citation_author=S. Cheraghi;citation_title=The Kalirin gene rs9289231 polymorphism as a novel predisposing marker for coronary artery disease.;citation_volume=45;citation_year=2014;citation_pmid=25316661;citation_doi=10.1309/LMLS813ZDPHRFLUU" />
<meta name="citation_reference" content="citation_journal_title=PLoS Curr.;citation_title=Multi-locus phylogenetic analysis reveals the pattern and tempo of bony fish evolution.;citation_author=R. E. Broughton;citation_author=R. Betancur-R;citation_author=C. Li;citation_author=G. Arratia;citation_author=G. Orti;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. Carninci;citation_author=A. Sandelin;citation_author=B. Lenhard;citation_author=S. Katayama;citation_author=K. Shimokawa;citation_title=Genome-wide analysis of mammalian promoter architecture and evolution.;citation_pages=626-635;citation_volume=38;citation_year=2006;citation_issue=6;citation_pmid=16645617;citation_doi=10.1038/ng1789" />
<meta name="citation_reference" content="citation_journal_title=Drug Alcohol Depend.;citation_author=Y.-H. Chen;citation_author=D. L. Liao;citation_author=C. H. Lai;citation_author=C. H. Chen;citation_title=Genetic analysis of AUTS2 as a susceptibility gene of heroin dependence.;citation_pages=238-242;citation_volume=128;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=I. Chojnicka;citation_author=K. Gajos;citation_author=K. Strawa;citation_author=G. Broda;citation_author=S. Fudalej;citation_title=Possible association between suicide committed under influence of ethanol and a variant in the AUTS2 gene.;citation_pages=e57199;citation_volume=8;citation_year=2013;citation_pmid=23437340;citation_doi=10.1371/journal.pone.0057199" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Christoffels;citation_author=E. G. Koh;citation_author=J. M. Chia;citation_author=S. Brenner;citation_author=S. Aparicio;citation_title=Fugu genome analysis provides evidence for a whole-genome duplication early during the evolution of ray-finned fishes.;citation_volume=21;citation_year=2004;citation_pmid=15014147;citation_doi=10.1093/molbev/msh114" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. E. Collins;citation_author=S. White;citation_author=S. M. J. Searle;citation_author=D. L. Stemple;citation_title=Incorporating RNA-seq data into the zebrafish Ensembl genebuild.;citation_volume=22;citation_year=2012;citation_pmid=22798491;citation_doi=10.1101/gr.137901.112" />
<meta name="citation_reference" content="citation_journal_title=Transl. Psychiatry;citation_author=H. Coon;citation_author=T. Darlington;citation_author=R. Pimentel;citation_author=K. R. Smith;citation_author=C. D. Huff;citation_title=Genetic risk factors in two Utah pedigrees at high risk for suicide.;citation_pages=325e;citation_volume=3;citation_year=2013;citation_doi=10.1038/tp.2013.100" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Neurosci.;citation_author=W. Dang;citation_author=Q. Zhang;citation_author=Y. S. Zhu;citation_author=X. Y. Lu;citation_title=The evidence for the contribution of the autism susceptibility candidate 2 (AUTS2) gene in heroin dependence susceptibility.;citation_pages=811-819;citation_volume=54;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=R. V. Davuluri;citation_author=Y. Suzuki;citation_author=S. Sugano;citation_author=C. Plass;citation_author=T. H. Huang;citation_title=The functional consequences of alternative promoter use in mammalian genomes.;citation_pages=167-177;citation_volume=24;citation_year=2008;citation_issue=4;citation_pmid=18329129;citation_doi=10.1016/j.tig.2008.01.008" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=Z. Gao;citation_author=J. Zhang;citation_author=R. Bonasio;citation_author=F. Strino;citation_author=A. Sawai;citation_title=PCGF homologs, CBX proteins, and RYBP define functionally distinct PRC1 family complexes.;citation_pages=344-356;citation_volume=45;citation_year=2012;citation_issue=3;citation_pmid=22325352;citation_doi=10.1016/j.molcel.2012.01.002" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=Z. Gao;citation_author=P. Lee;citation_author=J. M. Stafford;citation_author=M. von Schimmelmann;citation_author=A. Schaefer;citation_title=An AUTS2-Polycomb complex activates gene expression in the CNS.;citation_pages=349-354;citation_volume=516;citation_year=2014;citation_pmid=25519132;citation_doi=10.1038/nature13921" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. E. Green;citation_author=J. Krause;citation_author=A. W. Briggs;citation_author=T. Maricic;citation_author=U. Stenzel;citation_title=A Draft Sequence of the Neandertal Genome;citation_pages=710-722;citation_volume=328;citation_year=2010;citation_issue=5979;citation_pmid=20448178;citation_doi=10.1126/science.1188021;citation_georef=2010072445" />
<meta name="citation_reference" content="citation_journal_title=J. Physiol.;citation_author=S. Gustincich;citation_author=A. Sandelin;citation_author=C. Plessy;citation_author=S. Katayama;citation_author=R. Simone;citation_title=The complexity of the mammalian transcriptome.;citation_volume=575;citation_year=2006;citation_pmid=16857706;citation_doi=10.1113/jphysiol.2006.115568" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Hiller;citation_author=K. Huse;citation_author=K. Szafranski;citation_author=N. Jahn;citation_author=J. Hampe;citation_title=Widespread occurrence of alternative splicing at NAGNAG acceptors contributes to proteome plasticity.;citation_pages=1255-1257;citation_volume=36;citation_year=2004;citation_issue=12;citation_pmid=15516930;citation_doi=10.1038/ng1469" />
<meta name="citation_reference" content="citation_journal_title=Neuroscience;citation_journal_abbrev=Neuroscience;citation_author=K. Hinsch;citation_author=G. K. H. Zupanc;citation_title=Generation and long-term persistence of new neurons in the adult zebrafish brain: a quantitative analysis.;citation_pages=679-696;citation_volume=146;citation_year=2007;citation_issue=2;citation_pmid=17395385" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=S. Hoegg;citation_author=H. Brinkmann;citation_author=J. S. Taylor;citation_author=A. Meyer;citation_title=Phylogenetic timing of the fish-specific genome duplication correlates with the diversification of teleost fish.;citation_pages=190-203;citation_volume=59;citation_year=2004;citation_issue=2;citation_pmid=15486693;citation_doi=10.1007/s00239-004-2613-z" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep.;citation_author=K. Hori;citation_author=T. Hagai;citation_author=W. Shan;citation_author=A. Sakamoto;citation_author=S. Taya;citation_title=Cytoskeletal regulation by AUTS2 in neuronal migration and neuritogenesis.;citation_pages=2166-2179;citation_volume=9;citation_year=2014;citation_pmid=25533347;citation_doi=10.1016/j.celrep.2014.11.045" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=B. D. Horne;citation_author=E. R. Hauser;citation_author=L. Wang;citation_author=J. B. Muhlestein;citation_author=J. L. Anderson;citation_title=Validation study of genetic associations with coronary artery disease on chromosome 3q13-21 and potential effect modification by smoking.;citation_pages=551-558;citation_volume=73;citation_year=2009;citation_issue=Pt 6;citation_pmid=19706030;citation_doi=10.1111/j.1469-1809.2009.00540.x" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=V. M. Kalscheuer;citation_author=D. FitzPatrick;citation_author=N. Tommerup;citation_author=M. Bugge;citation_author=E. Niebuhr;citation_title=Mutations in autism susceptibility candidate 2 (AUTS2) in patients with mental retardation.;citation_pages=501-509;citation_volume=121;citation_year=2007;citation_issue=3-4;citation_pmid=17211639;citation_doi=10.1007/s00439-006-0284-0" />
<meta name="citation_reference" content="citation_journal_title=Hum. Genet.;citation_author=M. Kapoor;citation_author=J. C. Wang;citation_author=L. Wetherill;citation_author=N. Le;citation_author=S. Bertelsen;citation_title=A meta-analysis of two genome-wide association studies to identify novel loci for maximum number of alcoholic drinks.;citation_pages=1141-1151;citation_volume=132;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. Kimura;citation_author=A. Wakamatsu;citation_author=Y. Suzuki;citation_author=T. Ota;citation_author=T. Nishikawa;citation_title=Diversification of transcriptional modulation: large-scale identification and characterization of putative alternative promoters of human genes.;citation_volume=16;citation_year=2006;citation_pmid=16344560;citation_doi=10.1101/gr.4039406" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. R. Landry;citation_author=D. L. Mager;citation_author=B. T. Wilhelm;citation_title=Complex controls: the role of alternative promoters in mammalian genomes.;citation_pages=640-648;citation_volume=19;citation_year=2003;citation_issue=11;citation_pmid=14585616;citation_doi=10.1016/j.tig.2003.09.014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. J. Langham;citation_author=J. Walsh;citation_author=M. Dunn;citation_author=C. Ko;citation_author=S. A. Goff;citation_title=Genomic Duplication, Fractionation and the Origin of Regulatory Novelty;citation_pages=935-945;citation_volume=166;citation_year=2004;citation_issue=2;citation_pmid=15020478;citation_doi=10.1534/genetics.166.2.935" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Liu;citation_title=Increasing alternative promoter repertories is positively associated with differential expression and disease susceptibility.;citation_pages=e9482-e9482;citation_volume=5;citation_year=2010;citation_issue=3;citation_pmid=20208995;citation_doi=10.1371/journal.pone.0009482" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Lynch;citation_author=A. Force;citation_title=The Probability of Duplicate Gene Preservation by Subfunctionalization;citation_pages=459-473;citation_volume=154;citation_year=2000;citation_issue=1;citation_pmid=10629003" />
<meta name="citation_reference" content="citation_journal_title=Circ. Res.;citation_author=J. C. Makielski;citation_author=B. Ye;citation_author=C. R. Valdivia;citation_author=M. D. Pagel;citation_author=J. Pu;citation_title=A ubiquitous splice variant and a common polymorphism affect heterologous expression of recombinant human SCN5A heart sodium channels.;citation_volume=93;citation_year=2003;citation_pmid=14500339;citation_doi=10.1161/01.RES.0000096652.14509.96" />
<meta name="citation_reference" content="citation_journal_title=Mol. Psychiatry;citation_author=S. E. McCarthy;citation_author=J. Gillis;citation_author=M. Kramer;citation_author=J. Lihm;citation_author=S. Yoon;citation_title=De novo mutations in schizophrenia implicate chromatin remodeling and support a genetic overlap with autism and intellectual disability.;citation_pages=652-658;citation_volume=19;citation_year=2014;citation_pmid=24776741;citation_doi=10.1038/mp.2014.29" />
<meta name="citation_reference" content="citation_author=S. Ohno;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=N. Oksenberg;citation_author=N. Ahituv;citation_title=The role of AUTS2 in neurodevelopment and human evolution.;citation_pages=600-608;citation_volume=29;citation_year=2013;citation_pmid=24008202;citation_doi=10.1016/j.tig.2013.08.001" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. Oksenberg;citation_author=L. Stevison;citation_author=J. D. Wall;citation_author=N. Ahituv;citation_title=Function and regulation of AUTS2, a gene implicated in autism and human evolution.;citation_pages=e1003221;citation_volume=9;citation_year=2013;citation_pmid=23349641;citation_doi=10.1371/journal.pgen.1003221" />
<meta name="citation_reference" content="citation_journal_title=Biological psychiatry;citation_journal_abbrev=Biol Psychiatry;citation_author=M. J. Parsons;citation_author=U. M. D’Souza;citation_author=M. J. Arranz;citation_author=R. W. Kerwin;citation_author=A. J. Makoff;citation_title=The -1438A/G polymorphism in the 5-hydroxytryptamine type 2A receptor gene affects promoter activity.;citation_pages=406-410;citation_volume=56;citation_year=2004;citation_issue=6;citation_pmid=15364038;citation_doi=10.1016/j.biopsych.2004.06.020" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Prabhakar;citation_author=J. P. Noonan;citation_author=S. Paabo;citation_author=E. M. Rubin;citation_title=Accelerated Evolution of Conserved Noncoding Sequences in Humans;citation_pages=786-786;citation_volume=314;citation_year=2006;citation_issue=5800;citation_pmid=17082449;citation_doi=10.1126/science.1130738" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Redon;citation_author=S. Ishikawa;citation_author=K. R. Fitch;citation_author=L. Feuk;citation_author=G. H. Perry;citation_title=Global variation in copy number in the human genome.;citation_pages=444-454;citation_volume=444;citation_year=2006;citation_issue=7118;citation_pmid=17122850;citation_doi=10.1038/nature05329" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. Salichs;citation_author=A. Ledda;citation_author=L. Mularoni;citation_author=M. M. Alba;citation_author=S. de la Luna;citation_title=Genome-wide analysis of histidine repeats reveals their role in the localization of human proteins to the nuclear speckles compartment.;citation_pages=e1000397-e1000397;citation_volume=5;citation_year=2009;citation_issue=3;citation_pmid=19266028;citation_doi=10.1371/journal.pgen.1000397" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. Schumann;citation_author=L. J. Coin;citation_author=A. Lourdusamy;citation_author=P. Charoen;citation_author=K. H. Berger;citation_title=Genome-wide association and genetic functional studies identify autism susceptibility candidate 2 gene (AUTS2) in the regulation of alcohol consumption.;citation_volume=108;citation_year=2011;citation_pmid=21471458;citation_doi=10.1073/pnas.1017288108" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=R. Sultana;citation_author=C. Yu;citation_author=J. Yu;citation_author=J. Munson;citation_author=D. Chen;citation_title=Identification of a novel gene on chromosome 7q11.2 interrupted by a translocation breakpoint in a pair of autistic twins.;citation_pages=129-134;citation_volume=80;citation_year=2002;citation_issue=2;citation_pmid=12160723;citation_doi=10.1006/geno.2002.6810" />
<meta name="citation_reference" content="citation_journal_title=EMBO reports;citation_journal_abbrev=EMBO Rep.;citation_author=Y. Suzuki;citation_author=H. Taira;citation_author=T. Tsunoda;citation_author=J. Mizushima-Sugano;citation_author=J. Sese;citation_title=Diverse transcriptional initiation revealed by fine, large-scale mapping of mRNA start sites;citation_pages=388-393;citation_volume=2;citation_year=2001;citation_issue=5;citation_pmid=11375929;citation_doi=10.1093/embo-reports/kve085" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=H. Takeda;citation_author=T. Matsuzaki;citation_author=T. Oki;citation_author=T. Miyagawa;citation_author=H. Amanuma;citation_title=A novel POU domain gene, zebrafish pou2: expression and roles of two alternatively spliced twin products in early development.;citation_pages=45-59;citation_volume=8;citation_year=1994;citation_issue=1;citation_pmid=8288127;citation_doi=10.1101/gad.8.1.45" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=C. Thisse;citation_author=B. Thisse;citation_title=High-resolution in situ hybridization to whole-mount zebrafish embryos.;citation_pages=59-69;citation_volume=3;citation_year=2008;citation_issue=1;citation_pmid=18193022;citation_doi=10.1038/nprot.2007.514" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_journal_abbrev=Neuron;citation_author=T. Timmusk;citation_author=K. Palm;citation_author=M. Metsis;citation_author=T. Reintam;citation_author=V. Paalme;citation_title=Multiple promoters direct tissue-specific expression of the rat BDNF gene.;citation_pages=475-489;citation_volume=10;citation_year=1993;citation_issue=3;citation_pmid=8461137;citation_doi=10.1016/0896-6273(93)90335-O" />
<meta name="citation_reference" content="citation_journal_title=Neuropsychologia;citation_author=S. D. Vann;citation_title=Re-evaluating the role of the mammillary bodies in memory.;citation_pages=2316-2327;citation_volume=48;citation_year=2010;citation_pmid=19879886;citation_doi=10.1016/j.neuropsychologia.2009.10.019" />
<meta name="citation_reference" content="citation_journal_title=Exp. Eye Res.;citation_author=A. H. Wagner;citation_author=V. N. Anand;citation_author=W. H. Wang;citation_author=J. E. Chatterton;citation_author=D. Sun;citation_title=Exon-level expression profiling of ocular tissues.;citation_pages=105-111;citation_volume=111;citation_year=2013;citation_pmid=23500522;citation_doi=10.1016/j.exer.2013.03.004" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=L. Wang;citation_author=E. R. Hauser;citation_author=S. H. Shah;citation_author=M. A. Pericak-Vance;citation_author=C. Haynes;citation_title=Peakwide mapping on chromosome 3q13 identifies the kalirin gene as a novel candidate gene for coronary artery disease.;citation_pages=650-663;citation_volume=80;citation_year=2007;citation_issue=4;citation_pmid=17357071;citation_doi=10.1086/512981" />
<meta name="citation_reference" content="citation_author=M. Westerfield;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Mol. Sci.;citation_author=B. Zhang;citation_author=Y. H. Xu;citation_author=S. G. Wei;citation_author=H. B. Zhang;citation_author=D. K. Fu;citation_title=Association study identifying a new susceptibility gene (AUTS2) for schizophrenia.;citation_pages=19406-19416;citation_volume=15;citation_year=2014;citation_pmid=25347278;citation_doi=10.3390/ijms151119406" />
<meta name="citation_reference" content="citation_journal_title=The Journal of comparative neurology;citation_journal_abbrev=J Comp Neurol;citation_author=G. K. H. Zupanc;citation_author=I. Horschke;citation_title=Proliferation zones in the brain of adult gymnotiform fish: a quantitative mapping study.;citation_pages=213-233;citation_volume=353;citation_year=1995;citation_issue=2;citation_pmid=7745132;citation_doi=10.1002/cne.903530205" />
<meta name="citation_reference" content="citation_journal_title=The Journal of comparative neurology;citation_journal_abbrev=J Comp Neurol;citation_author=G. K. H. Zupanc;citation_author=K. Hinsch;citation_author=F. H. Gage;citation_title=Proliferation, migration, neuronal differentiation, and long-term survival of new cells in the adult zebrafish brain.;citation_pages=290-319;citation_volume=488;citation_year=2005;citation_issue=3;citation_pmid=15952170;citation_doi=10.1002/cne.20571" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Transcriptional Complexity and Distinct Expression Patterns of auts2 Paralogs in Danio rerio" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="twitter:description" content="Several genes that have been implicated in autism spectrum disorders (ASDs) have multiple transcripts. Therefore, comprehensive transcript annotation is critical for determining the respective gene function. The autism susceptibility candidate 2 ( AUTS2 ) gene is associated with various neurological disorders, including autism and brain malformation. AUTS2 is important for activation of transcription of neural specific genes, neuronal migration, and neurite outgrowth. Here, we present evidence for significant transcriptional complexity in the auts2 gene locus in the zebrafish genome, as well as in genomic loci of auts2 paralogous genes fbrsl1 and fbrs . Several genes that have been implicated in ASDs are large and have multiple transcripts. Neurons are especially enriched with longer transcripts compared to nonneural cell types. The human autism susceptibility candidate 2 ( AUTS2 ) gene is ∼1.2 Mb long and is implicated in a number of neurological disorders including autism, intellectual disability, addiction, and developmental delay. Recent studies show AUTS2 to be important for activation of transcription of neural specific genes, neuronal migration, and neurite outgrowth. However, much remains to be understood regarding the transcriptional complexity and the functional roles of AUTS2 in neurodevelopment. Zebrafish provide an excellent model system for studying both these questions. We undertook genomic identification and characterization of auts2 and its paralogous genes in zebrafish. There are four auts2 family genes in zebrafish: auts2a , auts2b , fbrsl1 , and fbrs . The absence of complete annotation of their structures hampers functional studies. We present evidence for transcriptional complexity of these four genes mediated by alternative splicing and alternative promoter usage. Furthermore, the expression of the various paralogs is tightly regulated both spatially and developmentally. Our findings suggest that auts2 paralogs serve distinct functions in the development and functioning of target tissues." />
<meta name="og-title" property="og:title" content="Transcriptional Complexity and Distinct Expression Patterns of auts2 Paralogs in Danio rerio" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/8/2577" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Several genes that have been implicated in autism spectrum disorders (ASDs) have multiple transcripts. Therefore, comprehensive transcript annotation is critical for determining the respective gene function. The autism susceptibility candidate 2 ( AUTS2 ) gene is associated with various neurological disorders, including autism and brain malformation. AUTS2 is important for activation of transcription of neural specific genes, neuronal migration, and neurite outgrowth. Here, we present evidence for significant transcriptional complexity in the auts2 gene locus in the zebrafish genome, as well as in genomic loci of auts2 paralogous genes fbrsl1 and fbrs . Several genes that have been implicated in ASDs are large and have multiple transcripts. Neurons are especially enriched with longer transcripts compared to nonneural cell types. The human autism susceptibility candidate 2 ( AUTS2 ) gene is ∼1.2 Mb long and is implicated in a number of neurological disorders including autism, intellectual disability, addiction, and developmental delay. Recent studies show AUTS2 to be important for activation of transcription of neural specific genes, neuronal migration, and neurite outgrowth. However, much remains to be understood regarding the transcriptional complexity and the functional roles of AUTS2 in neurodevelopment. Zebrafish provide an excellent model system for studying both these questions. We undertook genomic identification and characterization of auts2 and its paralogous genes in zebrafish. There are four auts2 family genes in zebrafish: auts2a , auts2b , fbrsl1 , and fbrs . The absence of complete annotation of their structures hampers functional studies. We present evidence for transcriptional complexity of these four genes mediated by alternative splicing and alternative promoter usage. Furthermore, the expression of the various paralogs is tightly regulated both spatially and developmentally. Our findings suggest that auts2 paralogs serve distinct functions in the development and functioning of target tissues." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
    <title>Transcriptional Complexity and Distinct Expression Patterns of auts2 Paralogs in Danio rerio | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__34Arr8QetTzezDWW4Etyv7gbgH-x-ZTNo3yO15Q-gHU__Mk-lyyG5WoK4SRy56eUq5eomxN7OGXjpDv-Ob1a1Q1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"nNC4mgxxhkdXoqU3hey5jF-XkLUp0wDydRDQxKIQXA4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485839","apath":"\/ggg\/7\/8\/2577.atom","pisa":"ggg;7\/8\/2577","ac":{"\/ggg\/7\/8\/2577.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/8\/2577.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2577"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2577"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/8\/2577.figures-only":true,"\/content\/7\/8\/2577":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BOWV+PUmBiM8g2if6EXenfIoxgacRpsS3uK6vkys6fkDDiulqVFhJb5ywbB\\\\\\\/K57JUxyfHvWY9pJR2VLRhgCEambo+7EgYNQTuP9hl2Y4DyV1GiRbnv6TXkbQXhqkOmcpuQgXrZuw76GyOicWHKAKk+04ppKT3Ssd6DWDX7AupP2k4cE55nBFLtYtFRYbjKDZPlzD2b29vNaO+72ZcDDcYk3pIl6yFiyWFG7gdyIn03vM6\\\\\\\/btAj296ip2l6bFExbkMILww0i4AxcWsctckMfR\\\\\\\/tSIHI1SvuRW783fy1beFuFlbfW8PZjyO84+HBqMXKESlCyEeKBE9RmbwndQhsG2YXsj2EA\\\\\\\/JsgxVm5BZmkmA0PLOKb2JhqJSMueibL5kTRJRNOR+Ydb3RYutGljqRTiJJBAsHHiRAgW\\\\\\\/oiQZJIqAzqxP0xSfomNHMelw9MdgnSpin7UWXNcB3xM8HhU\\\\\\\/ITUj9LP\\\\\\\/z\\\\\\\/InCTr5VVH6MwGbXSiu58YbhQWO56TJsV7USVUm+tE2e9f4\\\\\\\/gI+rnvzPt2Pzun7AuRl7LxLWxwxWD7+iMOfR4AzK3OetqsUK3vWK0t4ZZC7Hw8GVgjyu0oLXwLvLXFIH96Oda\\\\\\\/Y4wSHYxAb9eY5BV1tPGmya1t1OLkV1WYhnmdFpObDQIlyeummnaJsHoVFM+7OMJ+D6MzoayU4NUFChzetg3NmjejgcPe4u3tIO95nqWqyCg56RPUbWJ1NcRD8fBuBgSza6b3Vv2\\\\\\\/x3LqH\\\\\\\/PkaEbxgsTql9+ZxjA9z2uHL6iqw4ggUCyju2ach5YZKbYMdV1Kp580ccahDH78eElgp8hrTLeLD8mSNnhzNLUy3ypvpWV1C5A7TkoWgq6+yoUW5snyjAOZ\\\\\\\/lV0saziOStZM6kJzJvq2Jo\\\\\\\/U6nGf9b6dvmzPQkLUofBH2vcdofOVnsdgXRDIApfbVrejQxYIV99oWiOnV+wgubaasgBCaUNYCMWagJDXfQrVNzawxQh1VtrLPL+WERQH8faeDlFJrIRfbULcYGEK4+RU+YEu2jEm\\\\\\\/LjiywDlcyC4SkOHG5H\\\\\\\/fIjqZa6EKI7t5Ys5XMG2glq209UImc7srE2\\\\\\\/YtW55DwRMDofZUBq1\\\\\\\/Tw+ShMW62KPluAR8pbeilxQqsubZrlFCxAebeePxO8dtVDdeCFilt5WhOVokYtnBRgnWR7J1oEZ2J6jdCcRUXPMGP\\\\\\\/Cfz5OlQCDU3g2SQR9jaP3PxCw0hScQwa9Ldgp5uT2gt1zdUx8VyOlzUSd0sVJVNZ4FifoM1I9pFhrVc5100cwRz\\\u0022,\\\u0022iv\\\u0022:\\\u0022i1enaQPWEmr8MD9FdeuSwQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002281b9fab7785a46404fe14f483089d395907a242526a0c6db417d1321ab6b0cc7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Zn8qIjw5GSVY\\\\\\\/mUWS\\\\\\\/QX89WRVQ4+fbh6EL4faHPcK267sJ0tLZSQEYwASzklAmKeStSGMbDpYP7esY9RGq6ShTqjDKSuvHSdTPmm4yR5lfaPNAUlaR6XZ+6aQDXUIOHELgDQDZr54zGTdqleUdgYFz24URf1M2+2Vx9OA4SpyrKxRG5WkwIPttEViIMPP0utWtJTaIaTGFOSRgPakqD0k\\\\\\\/ZREGh+l2Me5fU6FthMhc9cKLFPMwcelucNmkxXGaoRJlGT8CZBB+SeLx0Pj5Uob+HeRAd5+s3ekcdHXImckpHEj52zATtRWIjee0lAehKGDWvl16yO1rlCSKgOrC0Xwlh7JVs4gIfuHCJ1hY6OmfABVVnSWR09uE4mATIgdifFClDwz\\\\\\\/66i\\\\\\\/kF0zcXzpxtinm2YREtp3L\\\\\\\/RxIWCjwyjl1\\\\\\\/sPYfJRpV82x93EY2Ps5IyqdAI8wJd5FinMHzejf\\\\\\\/UwZ4zVY5qm16ukU3eN5OHeu5FFO8KcmmuTeapnnl73Xh\\\\\\\/c9FqiZrHNzLDQCrahKn1NkgS8A\\\\\\\/s9CImnRV8j9CGAvDzhYUes8ySLEi9RKbgoEJKBwCU6kg31nKZq4oKH4GNxYIO0d3SpvpdmXFe8nbAqJ0mU\\\\\\\/db8fbHyDt8mFcgMM980fSGgVHhApCJp5IdqJmO6SyRq+T1WjzpNwUmyCEkGjgD90unK0J7z2juEJu9cYuCX\\\\\\\/pucwatWu\\\\\\\/g1mok4k+csnC6QMlEZlcsU261MYQmaHFhmkeJpDzSHJG9HQT+L5D09f\\\\\\\/sqDBiKlfxx1cvtv3ItxVf7e\\\\\\\/Li2j7RX1fyJ4F9V9woPP2bf17HhWO8PpQbjPNeqsokn9JRgzTGQMx5qYwGVb1MB71TMCaJvq1MOvucIbX7TX4lCfWPQUxeoutUwZW3+0p1uPpMJmJ+4q+0+1ZXwqwCiTyaIUczxr\\\\\\\/8Guzaz8GM6WZKF9clRl2K6v3T7khyWzIiuFF8yB8Nc2PIK1u6OVlCq9rcpbprFVLAhISldB7QJetnBEsS5uwWNIZ\\\\\\\/biCql9MHWQuNftGbSK44w0x31+4iy+1JrAwHtziE0dSzagd+Kysaml\\\\\\\/VL5UEMDGrt06yOgI8UeBbK6BOf3DCJpiakS6k00\\\\\\\/5\\\\\\\/\\\\\\\/UM\\\\\\\/nGFw3CauYQVLUaFuMBGVD86Ie61HazK36dVwCBsENhtM\\\\\\\/EBdzqdsxpCOo1irzZ9Kt4FgT+pm3UaGTcmz3qjGqEat3eaX2uAAkLPLZm3RwdK52BVL97CddE+u0hoqNZtyIn267UI5XY8bLuZYd1BQACWjhaB+LDiKbMIF940McFENeM5SmEghMf8pxWUGGBmSqjrZFYJfOLv+eivbzGRdx5Cft9aA7nsQ25yp6oC+7kx54zno65aacLEDWrColT6OD9tWZ70s3ndDjoqLZED8XYjMMuku7qFCNz03QG5IkZ0SwctFe2vmfVu7vqeBLrE480S8qGOdTDskwqiXg14rROrLKalhhiioWYTmtjWPSOKpWnc4Y8X71DPdBoywvC3gUhI0a\\\\\\\/McBkOiNELLGZq64OXHcAUyD1fHs\\\\\\\/2uqsGgDkm94+jlgXd+ue73a+7lU\\\\\\\/GUkrdZy12MVMXFcx3rgRdj4oQVusIb7LD50+5CI6FPL4g4a+Mr4ufKnUViHmDrrWa3k\\\\\\\/CDq7yZi4j24vA1fFvF69HKmssZnXvVFtIYLfEwjP87bbJ02TD6HgfzR8+prRTgl6yxgYmoZyApJ52DoR9tYU+VKSorSR0MwmJ+9P8MZBRErDky8qvaVa3UznaCroKbjMs7fJibuos9TxPYIQz+dX0h4GiQKf8imNSZFSgcx8fTHMVKvrQHSmGJhGAuJ\\\\\\\/0uA\\\\\\\/SMvA3lFtM9FkkwDAJ9CM0IWcvcJZES36ademC3pQa8p4TeHeSQpt0bRSRCaJAeYoxmnZFnK\\\\\\\/33fFMe1EJGI+FIOeg2dk+3+33n\\\\\\\/xxqUoR9ri\\\\\\\/ufTOl1YfxHLhPNCjUjny8xE0M23TgHbb02Cw0O2CY\\\\\\\/sxpUWe\\\\\\\/ehuo35+Rv2vvRCAoHjFU3bHjXDjgqfhudEg6RewjzpH+tvdApUsTdgnhnD62RKKlOpdYwZjPjV1REVvXEVt9aw6wX0ykDyxluTh\\\\\\\/Bp8Wj\\\\\\\/57bKwXvMdFdF50ucVMHzXOVZN29RQPeggWqVrOR3x2\\\\\\\/mHVK5bVAum4sco41Zhhj\\\\\\\/70S1ZV1NRgYcO2wBbARbXP6qjyqv\\\\\\\/WgvpEAK+LVGcoYAhyShqfAdbuFXZmtvO1Pa2Io7uCnfxcXpmpa4z2mYK1VkM1zhTeflUsXeXugbIqb2f2t9XQpKSQlvfm3bEYJ7kF2qmHjG9GkQRFh8OPUgdM65GF+JFcIvHw4mcFWAjoJ431beZd7wse+GS7M+TgLtkAKwSeRHU58Eicy+HfqqsW79tnS\\\\\\\/dv5nB3ZyeH3MsXkTlP9r9O3Ia4ZbeI6qLRHruxQXJSNc4zNR3qNvnBFso6dsxgMgwgbwB5SvRNSAAJp7lzVmFsRLRFLADe37LTRISZggb\\\\\\\/gtyzsUHItnW1OXleg7YtU7VkSsNq7jWQ0u7cYvTx8tVLMYBP4kgdV0t0UTJ9TgYjRnHWssAmvpWi7tCR+oaJAki4ROGRczHZTqmz7KKQtA0TYjsg2Ok7zqixRYPaYD8xK4cGWXbsgpc7zIxHBdJODQLGe90k3vOZJuI1LrW3+biEjQiaRyYvG76Ll10R\\\\\\\/1Oy6DvS3yuYHusj1nkGXVln8hLUxMQwGxSd6rwDhX9TYB\\\\\\\/7ZH5HSxSjQD+20IrjyN\\\\\\\/8ev+8cIiwVd9mVlO53Z5n7oPUeeoLnr4xvBE7QU0qmLCdWn9LuV89tLE8rezopRx3tV\\\\\\\/TbRcAG2jegrFltWsW6fCPFa2tyxxKnnL12N0GsGQBdEfe5uLEXUUh8kKMJlDZYIvVH0mDaQBBoqjYAHoJvjMR6zqX6iswWh6ouPja8\\\\\\\/Bb+UYuf08ARVDpErPDGi4cn\\\\\\\/EbfSF9Ukt2qenZtR+99VSgT7\\\\\\\/HhmymJSOJ+OvE4e88TzqBsuaQjOemRV9r6ZZyD2Fsz9h+QQyouQg0m9mXLnswRJBT6ZSESWOUAddU6TRjKgKx8w2K5uyKltDfC9dqLAEF70h6zq1lRq3SPospziZ8jE402qIOOuEA4T7TjMLO8TezPaVLiQOsVK4mYm6rCnWrqaTWOiDq\\\\\\\/00JCXu05fSBMvDqUjaWv2t\\\\\\\/9AQwoZMUnP8pq8hVmaEKq8GPePE0T25jTaQmuSYEoigcSoYiftr1rmOx9XY79TiL954BXUEpZ0KSTdg6TniHCri\\\\\\\/hzhq9EQZsafMd5lQnmQu1Xl3TyeHcdj940vUAndohPBzg51PqICY\\\\\\\/QBHdbbUCg3M0KizHc6y8KDhmOkIpjxtrrAaCqqBW4yQ8EeG0QC+1bJn\\\\\\\/1srDK0SFodxeTqlUBa5m3iWaz\\\\\\\/SVGNQ7m356I8N1AhhLD9petF4oRcLC9tw4OcR7lyGnCUiCOYS1V3wxrxEpUDyFRbXPOODfSzw+KNDQwhrI9E\\\\\\\/zA7vWOVIiWRgrTxDouZ4z+YJj7SjHI\\\\\\\/jidUaod98yWknuZ\\\\\\\/xUgKmouVorZwSefp56bUJfN5eQpmDxx7I0Z23Xc6uKB8Q+XTBl2XfmxNm5EeWLdKutGl9bQ4gVW0c2AV6Er41Ehs5wGDknLojevnwra0CC+lJkfy4aY14y98T+O\\\\\\\/GR7ko0O+roSCmj72iXWSmMIQUk9UUHWOBFwNYcBw0kIZAkkNxmDCkrIw3zJNUOuG2qEjoN7eS2qTFXCtrnDtEMyNg\\\\\\\/rTwu\\\\\\\/NNOs84q\\\\\\\/5xtRZtDKqAu+M\\\\\\\/X+d9TtJ6Dt1Ce2AI+y2DHz92KeVXnCsEwagKOLyDsysE7YmK05D6rVuDX1xX8jI6bsY9SePv3G6RiBkkjlb7cLnKjnPczCEKclDjCFYhCkF\\\\\\\/0bovRBvKqxqV1ad+kiBnjJh3045oR4mftXvLmNd\\\\\\\/i0eMK8lOAaSD0GNWHOuyf7Ei1cp5a0nbyWwkEXr8LlZPjLeyxIZbhoHz8Ad5EPsfAfVtrFTLK\\\\\\\/KKuz1TOnU1QX+OwGGvsKGstnaxHG+7MvLljM0uXmux4jhCt0R+94QYGosgT7u+PXPK1w8YdxH4iQBVJukPbnBwxWZSvi57lU8lXUJTOyy623iB2zfiCpQkmLkG7ZHpY7S+IRDtufkFmFDVyUuGEiE10A9KqSCoBdJhdRP2NE+B93VTbyXN6bGRawkY06sC903oqu2YA4UEUf3evBgeDClOjbfzOHeCmhmUCNlxkpwIpzbretE8+T2bUoJNaLuraGTv2M3Xep4VUit3lce5CusabUdSLzFWiv3mEV7dwfD6V3hnoF4OmwXS2dnBn1W1i4voC5mHo4LhJpmyyJ\\\\\\\/UoX9YwbKjP5lR9x7WcREWU5XaF1CGTRNWwbA0KEHwl63YAGHhOQm3rzDU8ScdBPj1IUoXyNnnN2rqLDraFjaDjAQnxQsAKIcixhoruSvdgSw0RcF8yB2IMXty96uIbOGPNtYoWcqjYOuy39l5bTY5KNM+U82zgPdnmUuMocrlxjGOpSB8tq41fMQPXu8eZTBE2QlMeO04LKk3DGuErp+UDnvo0KrcrWmhSQJJiaP2HxnsJ4Nf+0MNSayoXiwmyc6qua+7RgldxcOCfmwtQihYGZPrTM+Erk5fbGpsnc0CA0drCCKilcB\\\\\\\/2Sv\\\\\\\/Co76dWPAA85q2pO2WolY6geIo3VcCbhpSKI+9dF6947sCqtXevkUsgJnZDJ2mjT5ay5BK6vj7pkAISJ1weLEjfIqFUGqzO6txTiu5lOwOcpmWIDP7fES\\\\\\\/8NUXUoVtNRzElMqx3yBzvFnNZlnNCiW6Z5cFDPEsDP+Gbc79BxcFD\\\\\\\/\\\\\\\/uuraKnPgnOekLsfDVB8FM6fQHT5xa8do7+DKH9cRov97idenpciHm9eVcTsLQK4\\\\\\\/mjQAR9GYENNCu2MHTq9N4oTwbTHOGtg5KHCe8mfN\\\\\\\/5rkmwdoDEg7wSmW\\\\\\\/8m5tKGv5xBTfTJeUkI5DQgyPlsBQ9OJX0iyh17q0f6ZYGNlmc1gxu8LO0Afma0n+a8keeAw7JItnzBrk6JqBIHu21G9qdcGoruchAMgAAynhmXSRiqa\\\\\\\/S8mNwwA2ZD3LOdi9N6yk8IUROUMlaMvwIMUZl8kLHKuVTF1jISeVTAEM+gHDfAX3l1VZwQn0j6UJ6sTga4UdyZdFurjLuTFycMCm37kGCUXxxxYG2XHkK\\\\\\\/LSYjjGCyj9xujP4pDepJWxrt90gQl3\\\\\\\/S+H1CIWtSf8qkynGhjBiMa0X\\\\\\\/vDESY5M0LF3UsrPZgFaaPnd+Xs3Xzbxx2+kv8rjpSfNl\\\\\\\/8uxU6W+cVjjcaVRGlMmWF\\\\\\\/KiqRHW2gSGFPSbvJ5exXVhklK5lQi2q3gVZ5t7Is9rA4gwF0K9kNXoSLfjRWDjBheG2ThHrRsfHMjkyynSVJYR9ZXv22QNXBlzYzOENCD8OqXE3pGZ0on4gLq4bXaZhSOUgOUyyrsElumkm0nSMDyIM8xEYXqgVZzPXjBrxIH7\\\\\\\/ccFCHXRIjZseXppeGcgLNHsRAbBRUQK4BK8061rxpwk0P4YydwjUWEqWCqALVRf+G\\\\\\\/vjqH98tPYB8QiIijgU1T7kYYEdQIIP0mJYXpd8IiAYe1a6Y2ZfiC0kJTscft+RO+ScqdCrI2IlRmbtoVFnRCeP4FAJVWcB6tMQHE3KNjg+2CC5TwycEDBMSzaRuDNtLE\\\\\\\/S9cuCQ6Kzv5Aa477L64ngq2AvIL05oFpHow+EmpEg\\\\\\\/ZIxuh612jqo4CAZc4mIZ3GnWCZcRDceKY0szldNKafJQJxzylmpZ9uYoRcS57aU+NabPEkPaW6MWTmXEPJuu30V96UCTqczwI0xpS20zgeW9ZHBpnXvOUL3zc0s\\\\\\\/6AITuj8Ogoq0yU1D0dXIPqEstDpB\\\\\\\/02TKMf1Mjsr6ICAfkvKYPMI5dlAsVCdnKW3\\\\\\\/hMT62fyy+Vdu2KyFVq79MjJHUG\\\\\\\/jXa81HIdXxGjlXuLWw3zWENx2Cd3S2yS3FNC\\\\\\\/+R+1Ih+3IEcddCHFtW4P+DobpnXEQjQQ5BazB25bnupi3yYemWh0EJBmp0BQQhpMjv2MRXsKg7alQ1va4rgRYGFeioUSbWKdu548dWsjlvNHub+sYqPhttOr4hQsdytJXAqjsN6wwURhCKPpotrTKeWKVTT7nxLXb3uhj+HtJMJUKPlIS2Oc4rqKsa5ZhaItioY1WXcUXy1GSEjB5ys+HHBqd+WAg+pCYgm1BG08vvZ\\\\\\\/hoMpP\\\\\\\/UTTHf0wK\\\\\\\/3+jRsOhD93sV1gq97YQIG6YHxlPvI8l\\\\\\\/3LlqieUJDnVqPOTmM5NhvKdihk525ccZj7ntIofE+178HK2uZi1+iBc2R+bIajn\\\\\\\/zjwXbv1LWKvsPMHsk4uEoacOBwqXBTzybkSUhQcMYhYxOt8As\\\\\\\/sbqLDR75Y+Yqda9w7UOD2h2wluvwCPUpXL2VPNZhdkvvZvY7iP+hKS5251K\\\\\\\/bWxQP283bZaqP4kiUMDhYul5Yvru4GCfzq\\\\\\\/VaAMvTcYbLhS2oltxkjCsiwds62Wlm7omP+PGfa51TekQ7ACP0LW90I8kqJw\\\\\\\/urXF+GcEkenO5\\\\\\\/zMyAT0qWs2PqJF2V3RMeDK53z1905BZvwOdVwV1JzrU2Fn4OrEHP7MTMv0VJU5CaKsoTwyCKchonuDKzoNV0WY85tGMJlhq8DaHKMnq+0bx0aOAwiwTmiieu3rYTiZNuCWLC9SsUicsfnSsmJnqy3SDHukFfBz\\\\\\\/p6yXQ0uMVcB+KsA8Sq\\\\\\\/4n8dc6IOGaZViMmfVzMjPine1QSg5bT8M7cnq3QwNIfvbzyPQ4URvefbg1NmCPysaf6pe6Uty1fMw2RymCaYOg5amGkdZxVaw0Cjj00TG35RPkA8NxFM2gzdOkBEDovJUhyWstQ5F2x9zFkCDdoIGkErWaWqtWdoJ1VuZX99p4x6Q1N1yuTqHxngMrgPQ6tW9mi3glcaJz7r4DOjz\\\\\\\/28Jg\\\\\\\/rQT\\\\\\\/nlH91Zh0hKpYS25\\\\\\\/YrKI2gJa6tB0Aeazc9QRAvABfLRCd1BY0DdRQDbesmDg3rcKHF8a0mom4e5qGHCbqDiarEkbm7\\\\\\\/A88t3izNyhxgh2ApuoZHdO6ZpTJfBrY025+Qw0tKK\\\\\\\/xxhf8zWib5NrlkkFuMdY88UMUH\\\\\\\/r99UROljLKO5UHtGL74mCsdGjrKISodDvSV8ifLkiBWE6RGP7Cn0qIY3A89d2qsK3rOkH+FveWhd4Oio4VKDQlUkOqHOtN2HYXoGd7rW+oD4FTDKGVWsKI16UbPGe\\\\\\\/yazRO\\\\\\\/pCAR5s3W78dV5UpKfhW2FpywTKUwva7A5NidulmDDqpbH7Kxpd5vnJ72jYlIHiZcc03YyIEJf\\\\\\\/HoeFf+MbzbFFsMLGIRcM0j5jc84fln7C7HwoCo4ET4x2UquIJOSVUZ4qybTwNoL1\\\\\\\/rvf4JeIITsFdDkLgEnBLi\\\\\\\/pblLvGwYCpvUL0bPeqUQ\\\\\\\/tJio\\\\\\\/WG1j+bzGxfLmjcch888HpEMvE\\\\\\\/cNSDi47kFErLQvIZjwIynM6Ar20CRblRT7WieDwBnNf7hwQFoyAGJYbxZ7D1PKyQXqQIPXr95CYTJFCQ\\\\\\\/sJ\\\\\\\/TFCxRI+\\\\\\\/SEuIdm5lvfkJ64hEDN3cgHRJMZCVukH+PS72zDrXjaBvgVPqnjFxYF2VotwDHR0mOxerLAR+SaumHQc2UCKKbuLazIUWQ+kvcz4zauKkiMOBhEjAvkOE2UJvy75PcfscGyMtfGd1tC0hYdKfqfypDBrC\\\\\\\/sN0Onx\\\\\\\/OZL88qUFn57vj95\\\\\\\/RGlj4OMb1ZQUvA6aPf9xSlgjPiYUCML6eGKOw6L7zcYCwuLJOAEcDpSvDVfOSJ3RAtB6y\\\\\\\/3k02IHqnQNuZLcn7LyDj2198ikTtkVpgL4RCto\\\\\\\/9S+VySPIoteFPkaIBOk1BVXHs0bWFgkNXlf5Oekg4oD2e9Y67537y\\\\\\\/voo+xtrN18Xqd3CVKZzMrq\\\\\\\/D7KCwGMkFuwK8xMXsglbyHxABVBbi6gqewFpu16qRHQGZjRw2vRI4z4yoDvLQgZ3RsviqhNQ2Mo55Jybu3kQFk8JKOyQfYtb9DMUvGDkFZDrlgSNX99+O94uGMxHruYXGhPvf5oQG6EqAXvguu3OgkE1ApiTlsfRxL2\\\\\\\/8FkTsLiCG1Gmzy76sEcZ7oX35V0tDrIX\\\\\\\/u7ogqf4uV7LudrILZHIo5EUNdmqGXaKMIoOUKbTIopn9jekN7T+NV8UlcKDGzpqvfRh1yyBwfeabF5xNo88441TROlrODVYJOslcokpsrWxt6yxc7zfog07Ym4eiLVWjzy5Wi6GljxswObKfPs1b0e2tHhxQJWsVvvwrqSPLCpZhYzq1bqRsTQu9UokwmkWi0L0saoLb70rFbGzNF93avJZsJgejyypdIpknn8j\\\\\\\/\\\\\\\/20Oqj87pL2MikTTJMh4b6rMrqA\\\\\\\/6YrfZVcXPN5wwx0H0xczwnIbcQ1xbtDa7MRgRBPpEp0ARBuUCcSxCqg3p0d4El06et0l78icgW2jM\\\\\\\/RO7vp6+mfBn9ICTIY6UwLa\\\\\\\/0dMzckK2KlOs\\\\\\\/9lxpL\\\\\\\/ZFcZS+lw4X\\\\\\\/W2B4O\\\\\\\/LquZ30rCBU4eZdn0MH+2Y7F+\\\\\\\/o8rIz5vvn6PORLMYyT3RJfjOIqK3swIg4IX\\\\\\\/T3BDrG+2djYp\\\\\\\/48gA8jRJ\\\\\\\/8CNQZbi2cFO15P5Nquhd6Chp3fPirZ4zpyUg7byl1ZcPRZzrfDcNHNXxVxgY+ZLRLUROvKlrycsQGIRUg\\\\\\\/5POQLJWrdKJZTcCUjybyh6zaq8vmU4xXR6NsXFl+WJtd5us6tG7+bGAm5wdOpuwSHQqEyiiUvnJy67mJ5y3Pi+GhNLvlR61RFLcEq3gAEvHHwqkh\\\\\\\/IZetZ+w2yHy8FQnZTEWSp65lULudvvRbH2yj3LROl56TYSiwL1szWb3UieHCEXsBYq65AgkfGR7r\\\\\\\/1pBY5wJlcBJSXcF8+Mo3lIih40lYSAzAspAXOHWbX7QcIEJOnEg5ZQKxrPJ2DErfgqhoW3ASJoVKkfgtB6pup\\\\\\\/3rO5EnDOXt9YgNEEkbGqi2zGpAUI9HIZkZ672DhkKUik9v7AfnS993iOuzSFi+6y2JXO5h6PVPsb\\\\\\\/oVsVeS3rtGZm1S8hNy2wtjMfpRnsawazwsSsvfTaRG9KP9A9x8FOjZghMypwaq+FACSyClFuFIQmKoj1VyXH\\\\\\\/IvaX6rYdw6afDPYYYjj8hLvzVZ5fAQDjtLQB2krs9oC2e21e9xpEllPZkoieIyUjX8AR\\\\\\\/yEL97pjgS1olAQchIJg09mxoe\\\\\\\/JxClw5IQLNc7IarfpXehm\\\\\\\/fNKMlSmNJiSAkwvAuLVqb5hZQCfbN9cY67UMIzVFfKX5aVm1jNC62qo3phXMwSTSIAn+ez9tFgIL2TP1DZzeeRkHZwHtDjnHRaSuSMv+sl11Fcp60+NPIL6RrbIH32o+25xF1LhKI8Gukufua7e8C\\\\\\\/VdH7WeCFmHkPgE5FMt2V00lFNyJxtlsqXfAwrrDa5yr53S5fbhfYlNQu5eyt8lMjZIylB30E7tPUjUnzT3XCaupJNDZ9GJn1J\\\\\\\/9lR\\\\\\\/2+QJADzIYhVsdVEiVqtXoxUvyL7gMY0prtu6HyaHVW+GBlP\\\\\\\/oCTN7Vq1xvEsh+1hka1UaFeUsX0lUD3JsJbRKFbJWlDNec0\\\\\\\/rHt2ashydZRNYUZgBHsn82c6jZ1h0s4YUT\\\\\\\/K+cHs7s2kw+M\\\\\\\/5\\\\\\\/LoFw3RNLHmz629KFA5cqsPhVxuvABov4DYQ3LLCDN2rZfIh7M9yZvpB9YgyVUI7\\\\\\\/FXux7YHhqTGhmeWXnSmPH3zGnEWwluloV\\\\\\\/b3OIscgBUj8JGlMOOrOzj3Uiow7uVsuRWu+wJION+ZU3aS09hIZzVSKh4iSFhrD2IZocPk70e\\\\\\\/n2A7EMBiU9SWtZdpe737ZN6BkB8iYDv3S6oZZtmWoXCbiLia1pMgWb5DYHAuU5w\\\\\\\/qLO8asQPwXkvfublZilbIuSvVANuKAPe3Alu1WpWx4wdwqTTvq9PvlZlxpFaKIs+unBOhWbK73cIJJ3aOhyL\\\\\\\/TDcbOgJqjohHYwChFhOD1Vb62zjfJ6V7FeffcKP6pm9mO8XqVH7L\\\\\\\/iKgQrs0UPWTSUnNhFA+CbTPEJVOuUmOHhtRALJtHVgfAw5bZ3i7tPLHF5uZQOiZyjBL8fvlIct7BkpFIu6C0q0vTYmUq\\\\\\\/BpomvDzXRdCZKSc8sBJ0\\\\\\\/Tf\\\\\\\/j8by5FGLK1smT4UeCODyTe6OSXl2OaCdq95qDhnv0L5EKWZvIGLiEhwSO8tbMGbb2Zxt6f5IlZz9HFXsf89XPTDM+TFPlygnsSYdKJGUtilVdcZNFvUuvOC63IGw8DppUSi5LZCYKF64ybDVv8ivGD3GUTytpVbCjPMHLV1fhJZ0vlpiSh+pdLsaQLBB6ESC9UtuUiqD3qaR1v3cw6x50eEx4le\\\\\\\/n18MmdiSrbOHSp+CTrtv7NdriHz+6rt5N2SQn3TlOCNHbucUZYnKjUvwadK5acr1P+m8uerhwEbrkfPrb9hCaumX6RN6\\\\\\\/mgm6NBc7MKqi7Qwh35I2a+uxn\\\\\\\/0rS+\\\\\\\/8HhTpEFFHbccPlLwEM+g+00QwCYAAE+Ie4fdSTKA+Sq7yX5C3425eo2txrCVb\\\\\\\/NLAcTgoodAysUNgVJB3Dw8nwnx\\\\\\\/Qyq9ZDm0+\\\\\\\/oGRThzXG0lTu7YYwRDnJE+KRZ4LUmdFiQLOMsWVEvPcqEl4b2dZBanyVK0ECRDxE8c84AMwratrqUwc8pIS9PU\\\\\\\/mhoWsdPKc5MzeCAXLkGww\\\\\\\/275+mfuT1Msbs\\\\\\\/IzSouo2sb8RqhC1BwRqwQAGIcjSh856fDZ0pA7goKmKn1EyaedU7sI304qxLM\\\\\\\/bDn\\\\\\\/k5qksgwe7k2PrRavRS+2wb0uGQkUA2b7sL\\\\\\\/gJ6uEeVmeGJIpGgs0R2SL6dIkdv4jADrJ+mnQeSjmYc1aGNAY6XDUergO1m\\\\\\\/HHypYyGnWPtU4VTeyv6AieVHeaLs+6t+0X0ZNStxAnzfePEDHe9H62eNB7RYR7Cfbx+LHopxTFeqb3WTekJmoIFAC5AcNnr0JBRSl8B3qD05zWgChJ5VdSFXx0QCz2+JhegoWHGvK2rC21BC33cEDsatcPDIhrKZgUJBaTAVOQ3UktiSNLUCfWDQSdpEmM9FqLe2oKL+Ku6JuyqTom5EUKZRnR54MZH5jPXmQ8VYsc4\\\\\\\/13aAf7bPHbOHf62xSHShVUiidrKjFwfMUoPIHPiY6A5vyDQUoo5CbC9dJsHdtgmF0QJR2DJog6trnvo39H5OcTFeEGcGDDxGfD8o6\\\\\\\/rCqQvc0OfZkiapp+\\\\\\\/tFFWsJenYSXKvusuu81j\\\\\\\/uDaRBgD1o0iMCiNTNTtoVZxYW8kidrs7xPN\\\\\\\/DfJayAy5udeqqmwKK0zM7EeTd26h2RWx38\\\\\\\/RoE7blX0QaZd+3o8CYtcCIujDSYF43Ju2+cFpZXMUI4hnuCYMe8rh7E+VsAg5Se2beaCH8P7xCfS8f8Un5jGEAATDiem36so0ON7Zw7uFaC2nh8Do0aBxU9tby1SFBr3iSAtfyKc24zkHTlHTfeLLo3QNx6+ShZt\\\\\\\/NnUXQGzVOesfpFpTBpN9VjeiNF6I+uDtUPhNa8zqDgsTPrYYS0l92hIwPNVoKFm\\\\\\\/asri3hu1U22+Ei2YVRykIcMfxs1RYOk2I7V4zyHqE\\\\\\\/x6olNWb6GnEwje352LcQOgwHdDra4BhBTsdjsSUdgZdJvrAGQ38SX6FjQz6WtdpiscnuQ7fR3OQ7ATmHr2OSBcYTwOAFZwvkAja1rcICsoYtVjAm93fDPg6gV0O9VL3KfBc4Wz9mTvW8PMFstSUfnm0DQIdSUwsd6D806tlyy7qWdI5wWcFrZ5Ny6jxwHmSI3Qr27XlnPU9RmOE\\\\\\\/RW4aYf3DOMmwO1PC+sQs9wOACh8qXqNqeukW2R0N0WJU4+M2TStXWoztEFYsvucO3TIDAmNwfmzU7TgQUZ3P1Dle3z2xepC6C8WiyKnuDtusmUSdSxtO+gDqRFTPhOThyFLX2npOx5VBKGMko3dGTied5qgnUwR36bfotLak3ca\\\\\\\/KPVtrNkEazvnUyjmUFfLmz54EhpBNlBUJF\\\\\\\/4tnApushkfU0EdU+e3MiQ3b8nDhnU0QHa2O\\\\\\\/KI6ibcHzXOgO7TKlE6RVugmBoyx\\\\\\\/6UkMwBZkIgUODZJX4EY7+h9BHUAqawMwct5D+tMe5hXudpWqj1MFExydr2fx9viK+9I+HIowJvD40B3BeVvOGIuAFhJtYtPVZelCNWEcMpdBGl3ey75S63WJ6BCuGSLhEO7BsEeUE3pwkVUlZcT0a03EG8u28daDRGq\\\\\\\/IEevd9y7\\\\\\\/BvmpIxpfOd35kUD2AWewLCw+z3afcEGtYikCMWSbpSc+r1+4zGKFS4BeAP0yAcFO3mp0eWM0LMT3duwkRz2etyvQmOzCTkU+KqWH1N6RIknJNXmXnFSF2237ZCdE2ke\\\\\\\/NTH0uYVVlQmS2qsoQuPJRMokSwxxyfkcLe0kH5SqiVUo\\\\\\\/pApOHpwz1+tUchVuausvrnYM5P94kOJc5JE78sPnVYCSJx9rmrnFuzrS6mb8U1pRtWyRBcX0rsf+IjlfM+bhwZqkLA+XlY09A5GP5CT24V0DE27dU8sF5RsaJyLsIqDqAvzAH4D5PnOnIb88v0H67B0QxBLf7tNpQIb9V6616Xpp+xHmIUH6XbUDQ0wNzJEOlWNlG\\\\\\\/IIlN+gdhyuM4qVzTrUU48V3553WGa8ML0CBrmybBQJbNZtBTNlfSWpxBjqcIymj0yY41mtSal0fkVU\\\\\\\/uihtcRBll43ItSSO2d8hRqWWTUbyHANC8\\\\\\\/BTKOgqelmGsv6drof2JAOyzefGt\\\\\\\/zPf67da8fYbOeV3yP\\\\\\\/FKlgBm+cQFJ\\\\\\\/4MvNg0zuqqIqi9vnQ3wiZKMP9VG6mvpIgF1lz6qcWbrgJenrZaJdNbRBAf3\\\\\\\/6dMYFwE+r5PXHeNMzgwpDDix9pdeE8cSrLg0BWvua3UXTWy7ROlnpaDSt\\\\\\\/Ld1CYYrkAnGEC9Kiqx8hI+yDk3WzZn3i3MyS0NpY2AxgCirZ4lbcVvNk2iuLiY0Vh7a09VowqGRFUfeb8mRy7effA3NMbRFot6V6Bbt0lD1if0yeEPu3zL9PoWXRY\\\\\\\/b4RXhtMZFbveoabZ5BmMn8N1IX9v17nSNi6AvQqzjgbk\\\\\\\/MIJRPWTIAGJblHFP9PgU6mPrz3+YUmmHq1Ze+BY4Hgi6advJp6gliV9drx9MZbMmbgK8rZR7+nL0pVeNt6d4cmp\\\\\\\/c\\\\\\\/KkFOo8nzX7R0K0x1vn67+0jZ5LCGS1wMWuF9xGGdwAdgFsiX52RcEC+tEcGUNK4wIqpXcISChfhpeXHiR3DFruIH7G3\\\\\\\/uK67dv7rqaunVdQlSsBBuSEPRgFGSqLMcmAXG1rJdZ4\\\\\\\/7TMDrB84ev7Sga5D36cOL54aNmBBtb5mpPUKOLjM+Se04NZOw0QTLDabU4yhRdMxvdk88aDEb9USLm9tzqdFjWx+\\\\\\\/w+v5ngVxU1H9v8lyxPWgdHuoLvB0u1Wc5js6\\\\\\\/xqIgTJqO6tYNHyDy2+gr6vNkAkRH+kqkArRFFBfh2GAhybhdSxWMIcDICumGVhKrC4B+jUacsNvZyCoonCZvo6asMMHvUIjwl05xGUZtQ847BUPtfOkDnvmKmYeZTcmPmuaeHsG8g4RyBD8SqDR993te+AiDkSIjJYS9AlfGS1SDJ67OByQ9KRJhJsiHHhOuemF3gEI0AQtxlxPNnI8KIqvVNrm8H+3ltajoPETabrhcnCfu+A4A0xcP7aJkQ34IpkTswBwvqTJxu4mReF9I6gxT9NPLL6e9EjcjeoPswQlZRUwhGtGkwAye5oEQmkMq64oqIKPW2O\\\\\\\/CaaFaQmKpqPe4m4t1IF5Lw8zzfSSG8qDAK7QF3B3Bqu3Hp80I71n1hXKdoFQzrRxtfWGGklZN5u6Y09FdRlyxWvsU8gSCDTl\\\\\\\/B5dEjA4WIadYCX1fRBpVEuaeClbZB60MO\\\\\\\/NYgg7ghl1vktIz\\\\\\\/PaTnNPvz8PKrMoggobgxXfSY21EAqxO597jSWX+qlGX18C\\\\\\\/poKK3hzacWphJ9q1+AjN0uQAutgwy\\\\\\\/FBLK1ZXJ\\\\\\\/mKVAyJakX+QRe7fGD6sq\\\\\\\/Dmh7kbHJUvhQszU9t6Wnh0+U6gsYpYaauTicfC\\\\\\\/MowOH\\\\\\\/WpT8QsnsMuZKJ9\\\\\\\/ShzyfOT5ZgBcMDgiI4dVg2okk8fXDv9KUcKTThij37Joo6iwwfXhMZM2Gkrr918L3WzFNTvXb0hrtGmf4iaqae5ErYGZ9LA\\\\\\\/8w1RjXUFcpuCbbbIBnnWIt7eSIMOYgH0573asLqs3VTjx5BepFLDilNv6nBIud+c6CncroVrqwksmFX+jjgRSvlcjIzBtiLNRIXbxliChR6RWNhH0bPLJtuf9gjZrgidDO8O4adXP+vd42atXO1qhPXfSpQOONEhqGswafgGtXGPhCRhTD1y2kRTMndQIsGR6Ue+lwHXdojoJkR+sh5VUycvU4zWx4hWS81vebjKn9PGyoTdQ04b4DAkOnWQNqPt2XH47Ki35ZSyD5iNCH62gnxl9JPzytGbKr+N+cC\\\\\\\/lhwAoEQc6i26OxthbkQX888\\\\\\\/GHxIrthvFMacEruiECLjULPld2sism5hDH8GuJuTY\\\\\\\/c3xKCAXBwlxa00JI1OkcS1VQCr3HyCjIEEk4+d5kgQUd2WpsCYXxWIOsWHXdld5sqSf5N7wX9buYZV+edVMhQqPRYnz6YnVvVltezpqbu17ROUh0B8FXOJf6XXDHpOW+R2DOp0jVzfVlnJsQ9xwN0p54E\\\\\\\/sEubmGCPLM6nIVN6PorQzylL8xCrgAyCmrbkcfFnigLhmkRGqfFtDr7iTAB15V1muR+foVVGiDgcqUeINimi6W0oLe2RnQ0llUvoU+DP3k3pBXAjrue7yonOxaiDFB5faUJ4GF6jn9dPfkavKwsQnKmDKEgrTTIT\\\\\\\/j\\\\\\\/rzaW4mBW5d7p5GuDcngIuJa2DHuoJ6ETXhniF0I293FE5aeqZUYeMWMwScpU17B98K1a9qsbq4lmPqC922n3A2Y65rMSTJT94pGfCUXBLad8X8FhaqqmDaITbWyyv6y4\\\\\\\/iT97aPnwpn4DN7ZedrGqN9whLrqHTv1UcO\\\\\\\/FxQc31TMX5R5gVElk9surdOCVP8QaR5ZSUrpOjyI29WJZhWMKGkPMVFBzFkTYmNYtP9W33LIlDyNyba5sVs4vJaroB+evp1Y2qW06PpPRPSc1H\\\\\\\/ktMYRheFDafvLDPURIUz\\\\\\\/8FbvaxCgkZrIwnbfmiUAbfMrKSBYEZB+462Zu38bDP9R7tKF9LT33Ty2IgI6a0Kw\\\\\\\/JkES1LDN4rBc1C944TjnUJ0vaNs0MZ6bV5YTI1zTOEZtgEtYSzLuTJwCAo1CvfSmKkJ1uvtZtmPxdGUc1cu0Fy\\\\\\\/j3sstjWPtSiGkIXMwkTH4yY+KGaPTRqbtWlgUfVsELrAcEhneh48U9vIYPe5lrwAElZDWJ0Wkx7CzS4FLVrfWnEeXJijrYPq1He2AK1Z+cLJfWGMb0fWtxebxnos81pmz0fwFTpiggQfQgHchFIMe+Zh1PMc3\\\\\\\/+RCDuJMDNVY+u0owk2J59D8Fm8mPEsFOFWqUizIfauvTqQocP5+SxlVPavi3ozK\\\\\\\/LDzQAa0QKCiWWx4wJnCyP5Y98zIl81eQqUUF0++7rPyPb2XEHqcOS1SCRbUr4FaEdRLqKCMUsDX6\\\\\\\/j69s+CYybs9STvxVHCTA6600Td+LP2tWJEWWEkdLYUYLtuxvt9AurwwqLo2TsOaMhloTDciTxYYh1kHcrATqDNpTZ\\\\\\\/euRW1vtOdWz9rbdOTHNvVnTHBv82K1eSG\\\\\\\/509\\\\\\\/qjv0Fawy0fFwzb41oe8Gq99Jnm6\\\\\\\/EmhDHvmZEDW1p4ZCd5AN+abyWQDC+xiTOHEEeBuj5duGfC4AGfhKCTpiei0vEFVgwFnis4kuWUeXwU\\\\\\\/Dqr7tfiaxrw\\\\\\\/YIOOkjTWHOjy8HD3RmziFxnEPc4t3ouSAGiBb68xhAVobjEXfakyibpfqfiMV+6r2IeTozp3lWFJE7NoQExdS+gFbEYemF3hX41y\\\\\\\/FgXon9WI0qIZCiJwwOeOGiI2xt+ylOGSfvopt6KRwEJzNo1uBb5Hhb5STytnDVa6TaCkj3P2PVFoGrJLY0n6ZV1EBlz1t0Cj0IlrdUhsOBgm+zjBXTzx+MfXhfbO5hnFUQ8Vz8h0nKEcpmvLNn4IqxHdkd97VwtVedQG\\\\\\\/gYaEVL8xsEnTE8+Gf9LSPPF3xDk8faGgUzQB3n91gSr3criE27BEoBOJ5fpnC+LyOFwaOVzzNofGqJ\\\\\\\/\\\\\\\/GmKvQcgksx\\\\\\\/7lRzCSDh4MIvdpHWVNOVKfvHWZmwL3gBawIHJCACUF0zcBijcWx6+hO+z+nFGm2P0t72L\\\\\\\/Cg9quM+PeU8CkYgAf8PLvaxoRgriGW29lcz57\\\\\\\/4oOvd44Wbthxh70PNcoYOnfEhbRc3xImeN3eJqH0fMQE35cEPCPPmAf1OyBa9qfaCagYWC4QbCoHreW1PXK9OorQ6A60G30ulJRgeSXVFJ1P2lZjITlVolDiu8wvsuiNF+V0qDJWT2PgsPIhKu\\\\\\\/1jKAxyOjtUwl3zenhy2V13qg7Q7Mg4ajNeoQ5JyArwc5eYfxVjG9OWq+K0FAu\\\\\\\/jbU8gYFILS4v0rGVCGnO83R7TCTYjkyT2isR4+ret8Ema5i2n4JjP\\\\\\\/7DKToRfx8uRFejgZehdl9cCJBXG471jST4Ak3dHmPz9E3uGKTtBjJcwXEumN4WU9SIDNOJzN1yluTAhYeTgEXuIPKpXyp4mEwUOstJiKan9SgiEO3AV+hteigq8AxND6ThDA5YItqi1kCqqJMcMXR1x2dY2MzX4qvWaDREOl1uwRluho31MylwQlozRdn6c0yCkgvJ5FuPqu6PB3If0hJBY38njypvHS9C+PvUFv7tn9oVturTBHl9RFU9Wfql\\\\\\\/EzBA4fVulKP9\\\\\\\/4lx5cXNt1tkjkC00l+p7NjeKunT\\\\\\\/WosISXj+AFQC9FMdEcLZqnEW4BJ1O6BIsmVdbigvTGMfS4U4VFaiThuGS2wvRO33KkMUDvSwSYHX6\\\\\\\/J\\\\\\\/GXAuwF8ty4mUSz6OZyGux5T3ANf3dRvd4z5t4DP10tr05rukbTERjzjofNhzdDnfpEypt1WDImoA9k41j3DZK4GW3LyuE3h3hMBS3gLq9tuNP5HNOhn3CbSQXmbzQ49UhB4tGaG1mQozz154K4PLjUkAEa9wGkDzDA6\\\\\\\/z7BdfsQ5HGGGoHxOR8hzIVfTYnOgF\\\\\\\/BC6Vrgrv0w06FYova6UDQ\\\\\\\/AjO2KAQSsovgvqYUzUIHdrej9jMouqXIBHxyBo236LSffn+THoQxEU1kzrCdVi2vgXiYOBFdslrejmALs1\\\\\\\/QhfSoAz0UEPktlQxp1z9zZoQr0ofh5ZxnYZBCef327N9Ro0ulEvjynYAAPeBobB\\\\\\\/j14qeEHZgrxpRcQP5NTj2B+gQdpattiCyw6oiZB7IraC44Q3DdB\\\\\\\/pEme23op\\\\\\\/C\\\\\\\/JqXSJpxBwgn+\\\\\\\/gniCFyrz6kkGE5JwdDQB7G3zjC5NKW+PsubDpfJQbDumUk323hT2SyE7Qx1Y6ASgoueh3bfWDhHVm3\\\\\\\/+SFnkTDv5ey93K+1yySSD+TYwuDiShyL4RZzTtXXIXo35rf3nl3U8hdGcy7WmJjpx8es0dOv\\\\\\\/+gFh4O8+Ofp7wvLdT7JjCpakPLi+OLWTNnYZcY6SkqUIlu\\\\\\\/TNHKJ\\\\\\\/SyAxN7ejcBgiTK1c5LWXMfoOohz7eiB9PjdGKLBl52QCfxVPiborMJkbQUSNWe1rqV5zYJeaeGQq0h6Zd1kQhqA8knyVsKmiFxYJZ2PHV9DqeAb+nkYsyPr5TSeXiJZuYP8L9Sg0BCdsAVCBXHERJqz4tijbMQUrc0NvK31tSCvQrttRFtelkff4Yy2zeCM3sF79RMaNpodH1+5+5HDT0J7xmMUVJTILABAWuFd8B3GnrgULbs\\\\\\\/IxYKeZfg3jVLxoxVt3yipo2l1UOXVS9gFAeO6NPpZr9eQXitZF7TUUzM2IxAra6cze0dgmBAynHXLQMsWDMwlsQXgu+UG8\\\\\\\/lIF71htt0FiYAqDWk0HJztIasNhm+e9t0qS24GLffY2plf+eA9h8GRPFHsE+KXB1M1erHdQmmW86dVIwUmjH7BdBXd5kfz4kYHaSIowbLc7saqmusMZR+3iJN2W+CKLOimx3KZsp95mTm1uyHrph0Np3W8qdTjWb4x18WGeaG\\\\\\\/\\\\\\\/DXCvh24Wd8+IWDqCUrFjG\\\\\\\/B\\\\\\\/oyxFCVq7RlmrsQvnTEVqqTSoXQ+Tr+q5s8geAvBsdpI1Kvky6AELbNzTz91\\\\\\\/RqibP+K3ipBSPZB\\\\\\\/bRZkO3zS3wcovvQYfaexYlN6g6zPf\\\\\\\/0khJ9OSiFXM8JZCCu+8izaz8fJWXjc0gZ8shWWx2BI69H00FEi0E771xu41J1bScU4rP3J2GvuIlx5KIjBxjENezmqYPO2SPuSKnPmYy\\\\\\\/Enyu67RIlONjIaYWZZ+SQnUor7lSzGVCcqBAd90hf1OjheAjM\\\\\\\/cDMKKSY9NpInnzFU9OdlpKhwf3TrOIH9fmvYijR1+0thQaUtlog++rCCP\\\\\\\/x4Sa6pY79WgoA7UKXvEsz6sKn+2HXMlEKwz4uZPLEnR2aW307+t53nYb0ngR1QSquA2fz6cv\\\\\\\/y9oCEYy7RDS04R0EDuVeIXk+dVWuXph\\\\\\\/zfzkQBuU2pyPnBcMTx3OfHF8cuPHep+gcYiqooDB+7BI1h8Q4JeA78b+aW8NF+wGENb7+9s2aP26DcY6ArqMUCG+chPseBhqjrrQncfIqnCaU4eAmI9f9CFmCqXm7zNUTnjHE+eBX1487F9unE38zEIBLgKO9cPI8RsSpoX88Mp19B9VI9c6H1P4Pobq1ST2F\\\\\\\/FppDLHdICva2meegpqh4tqS1YrQQWKp5Ydw0YXGicwSm3WH5wzMgEEvkjSq\\\\\\\/\\\\\\\/n1QSLGkNdqJum0xDUHPU6Bt4x+46PWHcfKVIqeL816CZfCTPPhTVvSWBmdMs\\\\\\\/Nn4X9HmCISWB4XX4T\\\\\\\/py23bKAPWL\\\\\\\/+Gg1B\\\\\\\/xS6iVNdQ\\\\\\\/VZSYHsGy2rsDr84W9NrHVF5bt2B9phANizMxvSj7uwaejyFayClwkvkK+Kr5S6h4P+NhIwNEQlFtiqKcXHjBSPEfVY9uUeJhFInvLcQavrNoMKR387jYyIGOK5QVlm82xZAPNXitfka6GQwr0L42+PUywdlUFb2PPNGm9+wnEabFcXl8maDxIdmbWqQl8Vgh5ZSW7Tsy2EwXsbvE9V2AtoZJ6SfoMlkmAlQ3nHK7p25x4wCFY5hCyiBUWXHfZZW8j\\\\\\\/7np\\\\\\\/5XWdrbcN09DCN0v81\\\\\\\/ofR03HGnsTAwfMh8rAJk0vhNDaH8zqlD\\\\\\\/QBnj9v+I\\\\\\\/f0i2g0QsO5ZMgol1tNPLVrZm+XUO3kj1AfIH1ISYNXl6HTnHLL1ZPBh19vU+B6YO6Jr8y1i1sHuxmKVluFTFSy4F3nYQpQCazVxwHyG3O5BjX\\\\\\\/y1KPx89JHdBMnmgWALUfrDiPyvRInRJm+1DMTVtRP8ovbnMHvcO3bvkT5kgv7AblvwSjGYwlykiINLA\\\\\\\/9ZTamXAYxBbENa1uAU5jjcc67ixuV2AnHECP633xpMQ0DfVUjR65XS1uaYemyTk5vTB6zeoxr3AXywTBDgTak5MHNBQO6pSuhjJIms5woOOaSZ6RR3T9LlR8EGihWlX7qqrj1oUbztHcoWZkWt\\\\\\\/LSIQ1NGDkBr2Nd4SlcwLe\\\\\\\/DR93maokIB+NM79EfgzQz1GldMjD29tlW69\\\\\\\/NzO2WoCp\\\\\\\/mWodeAh3UtypK2R1jxOq8cGGNzV3XoBUm1DBFlUlKDLCBrq+RU7GaRDTkLJZzni3tSxi9qJ+CPcnpbkRFbTmbR8BH3EJGd3zHIMcI9FIQZSTbLPmK3imggCpqqJl59jFRW7LVnvAfDWfjWg5pRlufzUs6WdvCK8ZD+lkBjzTZCbeagopf2xeRMD0+zRsQgR3k+0nTigEbFK0\\\\\\\/wopXSJZH5MKBfPEpd+pcPzBgrzRDEJRL2YI4sSatww27GrZwCl4OhSWrq+WVe3a+zy29d5LPKogptjwsoAZvnYbFMqT1rFgafOP6\\\\\\\/3D4Kg+aK\\\\\\\/pLOyD\\\\\\\/Vewmp+WPmbKssUlzaQOjm4Tb\\\\\\\/51rK4ZQ0\\\\\\\/hhqaaITBO7uyD2zOjNtzHUt\\\\\\\/4RllQVs4zpIQYjVYSW\\\\\\\/2wRWClWclX3YN04WEc049eecLK1lyCnQDAXj87l7EWeK07ylSvG3+Y7dGubmrOr2vrKb2lVMtjBC6LWlw92pwRLw9NJmRha5Wpp0jQAyr+Elha7gvvneZb8GNofZWBfSiDz7aUIVQiE64TUUKmc2TKWQI6TfCUR9pg\\\\\\\/CGqDVwIa6YhbS+45xTT5fKgnhN8vJQoiddUvkFpWjGctyF92FHj3hVmpIPJTEfftUBJ\\\\\\\/vud4xAZKgmZNuu\\\\\\\/u9\\\\\\\/Xk+N88cEgaGtmk3BUt0+rOLUh8oxdmgIv4mj2LpkOzJcKcgl8vLPHrX2R05zKXIHisU+aajdv5Eq+TP9dY58+I\\\\\\\/OjAgCqqP9qbu8W8YnKYWdmFJ7RIGsULqu1ot7HyD1UdlxZLFYM7yd8FdmoybuYRSQuWRdavQL0gRZo4\\\\\\\/+8jDPq+3cSe8zdxhevqK3D62\\\\\\\/iurqCH4FP7n5CI+IycYeCU0cToRjcYqneD3mlhik7OSmXSdmYnIX2R4gt5CnFicKhW0LF3amQHtaown90PGBO4G4nrXOl\\\\\\\/1H4IR5qRvJSkCd7z5Eb2DFIHZv4wXhL7gEDxJ7ENSSZX2sN7QGJ5otSRmeemPy3Lwf3bHhOWhjHLlc9gBPq3XlF1L5SfC3MGvpcQtDjZZWvaHnDbREotaXE+Tyo+qcJbbkMokk0P8ichJ3HSecVxs5wHQvNHQXKQZMsm\\\\\\\/3gHfBlYjGm4oZNDDRk\\\\\\\/GIjn0fd1LD5LTWD+YpSL4cZskfQ7AT65q8Uk+fSW2w6L1dMpiw9aRm12GvGhgrTknsbw8XzQk5YwLbybI42DuxnXO3W\\\\\\\/D1pQatAp30rSGonHK9sIOA9\\\\\\\/V3plK+KWYvjtw5yQg2QEnIySe\\\\\\\/bYHe8k\\\\\\\/v0wVNviDx65QkbQ11o6eOziUxMO126xFhM2TcBO+GqCFj9n1hM4hGk\\\\\\\/7bk8wjVE7LUFj6lNpfWvZdB9xPjOgmTiInJDWdVI0MaGjVt7rP2LTRtOh+quX2RIROXpChWfEF+IAN\\\\\\\/N9rkM8JcoVdIZfzht\\\\\\\/lyRENt34yg5NB+OeIo0m9K13omhd1790BgH\\\\\\\/pJIGXLwImCv2vRYv8ObGZNmeGi7mndVVcTpRitiehqbId2\\\\\\\/\\\\\\\/Vf\\\\\\\/qdgkYRUDpide2epYsXHH94imG0Z4iOeDx6+gDIuPPICM3iVIl3unJ3vNpMiOtPnzULH6wj6Z+yZqK\\\\\\\/mQQZLy+ffEmqigG4B0lZEzlyUQQh7JIkr00f728dFbwjDjxu1SjTnd1IphYt2C+nGSobi76Ub\\\\\\\/dMWyLXLCcJwVvCvrFObK1cEOyktaG1sJ9ifB\\\\\\\/U0m\\\\\\\/U+\\\\\\\/Mm+IO1MTUDsMcskSRsfjPZhPb9NxOENTTNSLZ6eIqmjniowBL0kpRbWABryNaQ3DfB5npAR4Kse4KGO7nDbam8wKXimqvdT\\\\\\\/1xXjJoQHbQGxF93xidjkdfJJzIHDI1bpTwovkrlJ5jEAK2wdB7sxEXqhH+8+wadV02GnGGgTQ3HRFWR\\\\\\\/qSMPKZCpzwx2WHtiwn4Y7mFyjZQItz7oOIIWh4W3S4kik8i1+Yfo9wV0BYRnajzApVeKRjC\\\\\\\/D0QXifZ00x9gFJ9p\\\\\\\/Zbw5\\\\\\\/mV75DuKsYVIH+dMsMqAPUakx2ELVzQIvN8vkK8cZBPU2rCcImORtR+oozDB42he5Jtv0ZgDgCeOdnFoJvimnD5VfQdXSTh0w2MdGPkfw\\\\\\\/JDQPqlI70KQ6RneO7wJUwyDWf2+ffpHXJF+L1+dvGPBK+ieKiPp8YZjWKR9p8plK58YTN6wyLSx4Jedl74nfBP\\\\\\\/hPtNAlU5lAgW6qfpUSq6ndcSVbGYQoU0YR10qLVTPk3izkQBX2TIk1ng301gHeB\\\\\\\/4l8YtEC1dQ+PcVmc+jAvxn4glOl903BCukMYpRk0IWl3FHTWY5LYZC1uDOkaz2+7tQ8Tlk194LcXyeWHVoG4rqZjM53HojBoOr4IIvK5WR45fZzNdzJmMdMiKszFINaO7RGDG5j4QuuuOwr6M8c+nL6D3z05NEnp\\\\\\\/+vxuUIz0UukYWG8W9R69ZuYcWy3DaKZ5sWyJyQoNJ7Tzg5cgEmAVDo6MTkpTQKgAJGgcVmgG\\\\\\\/ptPmEnGyWVJtbMMJqCi1jucbQoTXbKg4lQ7R\\\\\\\/LxW9gpqZmWIQ1hRWKharFN3XcD49Y2Xjdrr3ucBgzJ86mcL7MoOp0zx8dFldV4bETehU53sLL7ZNFx+8iSACvamDb7+ZRxaOoq13dx6Hr3qMefLOPzGHS8GiD60c7mTUcTLY84Dui9A804\\\\\\\/rduk72YinkDVjmPgs47IalPm4A5eJkGd+ZEAMduhL4L2hBdxYuuACvpvjRJKy7GvbbRi63Od2lUmX6cUqc3WOJ7mi0NFo5j5\\\\\\\/s+lmQQbj8e88JZuNtHVaPTpXPceZ7QaUAoKfOkOslmkxhbRAU0fTnFRAvJ3HwODu20PJx6yUhMELDBlUsT2G5ATdmjGt7Adl69yCpAw9t0sYP8CDTyYjl+QBSduYfmzB1Rvl1j6C2aeUy7K2jU+QTa\\\\\\\/NdruEoiuchKCNWSpGQh9ub60tmS0AxVUjwlorIdN6Ohwl3oBuZzfkSWRV5RCU\\\\\\\/oEveP+dadlQNhR2JwhPvhp0RtdD5ep8YkKHRrMQai4KxFVwW9i0XussX0R148Odhj4Kt\\\\\\\/sG34iHhbiID1pXV52tZ2cCZeBq1eqNH0uLJmtu6fTwOcaIAJqQJP23R67DnP9Wt39y+2JyISNVps6uG1as7Pseej0Y0imiQhMoSk1PbeyUDELfg08E0qkE8bror0xyI6tcQXpvlxiSQJSo8i+GfGhfF5JjrIBuOfLbCSBJio6WtuIx2wX3OxFzVHOBhp5\\\\\\\/0n5v6B3gOkX7kyfPwGh8umM\\\\\\\/OC\\\\\\\/iNiXBc\\\\\\\/BHaysTezj2bOVBbBNDjcoBceoiskBGalUpHkQ\\\\\\\/VnQvepCKPhVWzU051yEGSzpsY9Wn+BVJ9PyShXIH5exlwvOMBVKdG8IDazLk4W3grUyL29lcnuQxvtSTC6GEOs2g09tt8ALle1zFeSsNjzYBBe\\\\\\\/DSslv+xsEACJXO\\\\\\\/NQnaDUiQWlUJW6WElSz2E+\\\\\\\/NC\\\\\\\/cqGXA5v6ZqZMMhB8o47MDK6NI8fOKLzdIZ1JeahuCzdImsTz4L\\\\\\\/UR9DzmHJiryuklR+GIqdE+gHCOaOkKl+MeljHxtdILpSyQknCFPXxT9bDPXQlA8t4JmumpCMEZuP7eS5oT02tjqy06CTy6ce\\\\\\\/I7Ugve6vHeCxdQ7\\\\\\\/vlN\\\\\\\/bmLKCxr0oY+rJmLZrrvFsuAXDtGvjE1mnISmORHwl3uHHS2wivZglFphYh8A2KdjJWlpOW2a6l8RKnuciQJcTcmzfGvtomzXik\\\\\\\/+FHUgYtqUSa2l32gohlCaUF76fjyHDGZGYPlBpOXsPW+yJueFJ5oIa3c028y9K6bvxjp80vBGcUJ9r9FHotLtH5kazkvKMypZ7K8bPMyMdEe0zPlnLGfitL5VoegjVDMuMIl1gW1L0pKvu53vgsLGBeYD0B6PwwF\\\\\\\/kl18lHeX\\\\\\\/HKti1lu4PDSDAotJ0Q4cccP1\\\\\\\/kqFZ3Q7ex+pGFUGRTJKEGjEq26pF1fvHBqAZZwOUj4vc\\\\\\\/r\\\\\\\/Z2M4awnNLcNdSVGn0rwgpvSL3x75C0NxNiHshGw30kFLbrLP6lhLnUeCZASnvmdndLrrKotz4e+RQgDge2nddDcr6KgOYJW6TC4uuRpEHZ0HgD76Pz4jS2hyD\\\\\\\/iWIj+Jh7aex3AxMXMfJirNylx2kYCzx9DAduKnyoyfgz4jedcfSkQLRetNy7HsTKtJD6\\\\\\\/RnjeIucYDRMREp2XiEbOLyv95cJSrOLKQIz95yk4pXyYwGNu49k2LWl\\\\\\\/1\\\\\\\/rBjbU8a03QKrS07+NNym57DTJOWxd93CZGAmigLAxrmOCF6znqdrHayoOGxaiyOEGpbDUqYx9wdxwxn\\\\\\\/lIjpvkKuqL+Wq2anb9ycJ9gd27MIzp5YEK8aQa4URUgxUmV1VkaQRvEpZ1086yoQs+BDaBYtAa54Gtbuk2b7y1Z+OqMRBdyeKXmpsH3DSU3RrPyE+PFhRwTj39Yrz0ykde4zIr3y4W1lfWfjWKxCBzvGD2uzfWrJHqWAokjepux7Q09FbC0GkR9JO0UWss678f0OhKHMnK3jXya8f9pqp8YfOysZCISHlixM4wr+dP1k3I5Y8cTXn0BiKj7lnIpxN6PBC1fU\\\\\\\/Q24islykisGaW2Kw8I18WV3Z3o+OdRXUI\\\\\\\/LojHdXZ86e9cFt1ccedROA4lLucMi5kx2O7+ID8wqTY+PxewYDYBdvwk\\\\\\\/j01tG\\\\\\\/b\\\\\\\/nvofCfEX1o\\\\\\\/qqP3eBEjM5kx9TpNtaGWKgFHIYQ53+nAfTRGZAWw35XLVU754BRq9h0fmmoTP4V1SCAPTZkOQ6gB3dzJb0ttg6\\\\\\\/94TSp2KMGf0OA1BRiwCMNTKHRt9HuByBZBxa6Bsndz+wk2xPTQpf4aGSNbQyMsreQ5vhCAaYkp1JSYiIgEJi3nw5fu+bfWE0gWS3SmRORuethO7FMeiuehwY9ptcqL1ElrMtxL0lQnmPV\\\\\\\/6wJZtCAv8pulARyVg7Xg5cuZMHQz4TxQXdyX4cKdyb7MIkg2LMEruAf4ZoAWWsh2zScr7BPV\\\\\\\/vRfFXZ8M3gBa8qr08HUq5ZeTEccbP\\\\\\\/B11TQHhY8p7cjrtcV\\\\\\\/9k0WOzi6JgrWCOn14bHTlmhP9ilGXGaJWMuemais75zU8XTUAD3ui6YF1X42MGwnyaVQLz7TZTaoIsu+DLR4XSQNWg9FdW6cScOktJ7tJZyamDp5V\\\\\\\/Bk8QR7inYbRsXvomuf4L\\\\\\\/5EGQyAAapimr9aHsbBY5ARLfgcWaoWrRfjuANfeBUOEI3tk1xzfl\\\\\\\/gMvI7IU3rj\\\\\\\/truaoW3OP1e+g+dZoCEv2Mz0hTTXsLeT4HycAiVRvIfQnFFb\\\\\\\/inXdB3ixC2DyUJST7epyBDKfCK49OlVc8yn2oxCC6CXhaDkijGFDjsYh7M\\\\\\\/TujDg2ABnFsOuCkATqUYUiA7mWzPxEfekT2GukPW26YGCDTP\\\\\\\/xdDjUO84RIwcvky\\\\\\\/af83jG1XC4bszXe9MisGp1ki37uVZ9l0PWcRlFWAXBRDLRrKXxgkS5pe6kf\\\\\\\/dlYZrkMgZ5ZESjeXJENI4eCrUC0jFq2YdOA1JjOOI9rzej\\\\\\\/Kh6lo9OJBbZwQg08mgc0uiwLauSBi9SIIkaAkuNKm8bxRwGJs9fGUFdB06EE0zgzFl0nB0B8SfVE7BKjxQfJE+EsItfYRRjiwdrgpDWClI6EQt+Tk0gWUKupe8MDmPIigq0Gj6KE4ZoOojo\\\\\\\/4bRuRfmowmJX7NOfwgSvXQlUY97p2UESVZWg0+xQo1Pdlvvpyjz04F2dnII8Cehw2LJjNbxJfUwb6Nv6LrQq42cC+Ljth8YfKmp0kyiUOyXKw9AjqA9cBO5n5t7baw3amhqkEeRxNlqN8rq4KqzKscutq4qXZ66w8gBU1JT\\\\\\\/ET\\\\\\\/0KwnIZoBqWzqkV310pjTxX4Vt8Cmr9TO8B4nsyFPO8YSSFBepZpGwPvJ8I7pvlUuLVOMj846g\\\\\\\/Q2thi4iM+DW\\\\\\\/s3a8m+aHgt0VF1WyTSsVEyV1g6v4vl9WweQi2Cw2SOhvWh\\\\\\\/ssUyXarPuhwbAinWvPKWVnZV6GWKogChJBlg4OXmn2UE44\\\\\\\/rKycC\\\\\\\/rhoxsO0mJPnZy4DVPmbyavgUVJB1GCuagSFnH82kjwf9FlgEoUxR4DTKp\\\\\\\/u5dnnYsuzzf8hLWMZ6tDW9ixKJ7tXi7mxtF4VLA\\\\\\\/GVtoyzRcvVaN7Tsn+1QJVig0GDWp\\\\\\\/lPrWKzT83p\\\\\\\/Afjs0Oqqq4\\\\\\\/TE9q4unawrbfV7070e1z+a1sx89y3KbY5pEh2h6OQwziBgazickbWgj1oIBFOe3SFiqc9reKa\\\\\\\/mqDIveA6D80UHq3pFkmRrpErvvrZmjCL+mZF\\\\\\\/Yvw9lS+23qwaNqdchrdhoK7RDDY8YT+MVW7lQCP1n7L1LmWp8YqCGp7YgWpTTJ5snZ+v1gKgjLHd3e\\\\\\\/EK1EKu1rFknRL6GMueSVmpssL8cSkFHlkJObJ4PGTLJWZ4H6esEwSUuaKUVhc0W2Wh4w1fphhyD0afoB+bxl+qUMh96TK5tZCoBi\\\\\\\/wkBwXnTUW1d7bpsyOTfv9sM3BM\\\\\\\/cbHQqgCnrE6zIN8Z0JalyYmdEWw968Xqkkm02qa0ebHnx7mOx09oN11zbo4gY3ayUF3\\\\\\\/QqYFcQVldmeFba48P7tYBYsYjnH8MA7JodYfA\\\\\\\/W19GkvsXqRqUNyh+Qv+bNy48UW7sOeyfDMqyqjYAsbEQu\\\\\\\/MFalxLn6amj8bxcIwoFZBAl15TB5Ic\\\\\\\/DEEFG\\\\\\\/mK7mfueq9YP\\\\\\\/CIsII4epxNNmK4\\\\\\\/y6p+g6kE2Odfn\\\\\\\/9UksSVoCaQRMpqhU7Rru2NLzZyPE8xpByJIXxbTs9vifVyK5yDqZD\\\\\\\/6gQvrhQAnT4YmasUMYgATv986GlhBC9ln9lyUBbVYwDl508eyFUiG8BqYtsWa8eZAx3Q44vvb1fMhZKB1VU+mQ62yTg2k7\\\\\\\/VPotXCMBYFTD9jvjqGTpwtgYApqsG4OUI00by4t65NZtmMaLF9dhpZcyo7\\\\\\\/abG+lvgTzN0llT8trj\\\\\\\/1bQ4hpDQ51vOftOgC7X33iATRG9trxWXky5OmJBU7Q6Vlq1Et4Momzb1s1K96kHroT6kULI8EqirGh7uQAsy+XU3c9qmt1HgE9lpaYmQTOC+TrHpfui3HC8l0F1BwApqOvTYupIjgU6asZES1gyNxAF2fwJgJVNw2eluZN7KjvRMcZ1yFDlZk5NME6Szww1oSxq5rMvyoNByx7\\\\\\\/IUSqcmdbuHWKpBD6x6CYV8E41noxYn+ymFvLYpkBU1gP2Uc4YTEWvKNSRqx9dIgdWtPc7Ru47dFt8v8A5KtY4D0fJXHgRCEEESGjsZv8RZ8GNweSKj4ONFOjBayXuXtY+zKo3n9VjT8rTwYqiPeeBY9MQ9NaITpIHEEi4tx7WIP1rb89y5mJPy4TRlj6tf4jTwomJKFNgw3qPR3HMWVPJdaKepq5OncQvr99EuKMKG5\\\\\\\/GjWNunKWln7bJzYhdpzN0rK5Z7lnJhosI0RweJy8wApWJ6c2zkBt0AVqCuG9Oa4jCWNix5XH5v9uHLHATKBk9CdYYaexO4gkqFh39rIniabaeMfUdYsY5rddpFmjtP3k6GAWXKIgYOENR0h090\\\\\\\/mI9BDcB4C5wWIRr8ouD3zi1\\\\\\\/aRZwUKNMOGxobHrzQp\\\\\\\/H+lrxLkniB058ijT0XC7G2c2PWO\\\\\\\/ibk+td17rkN8fMMMpxBndijaz+H5T9PN5AEjuqjA+lzOQRD\\\\\\\/QuuHHYLXZyv\\\\\\\/0Y\\\\\\\/Sd4pqEfbSs\\\\\\\/cu3WX920zU++om0CitWRUbZMuGlzXoYKM6bQzYI374b5lFjb4OLMzALHyfprflGzkB09ltzWRGkymkom0xn2t4IZNYAYcaIcCf+HFxhE1wjz+ftalaB0gjwWmCHZR9OklBewdlc\\\\\\\/7TTAlNaR1S683n8kkZidnobPVdHcwxiNAXYugqxkz12RaTFddwgOW4RbkwkplRpTuwsSIreegG\\\\\\\/5OJp77HGfFqg5eUt0Np\\\\\\\/TN3MHWXvXd8H3AWeAZ+Mr12bci2I5UxIEUayQgwkaS7Jo+PFp7olI2BOtbMq8pYjruvSMVMeTPqkDcseF+dofSettRMXXjPP+bCww4faGjwaS0OGhnYKR4SWkhrWuD3TwxvlWHT9FVdHuNsqFnmJPotJdYQQzai7xlAcNHqUyPfJ82bG6oy55bt8TbEhSxSCCnDS77QmCKzY0JHlRT+evxZRB2vYR\\\\\\\/6mswBXLHmsc+kf2du1Ak4PtbQst+9uaadDKtwCFdnmRL\\\\\\\/0H0GfnpdMhcEk4hKplkEVCe8+1+XsrwwdsXIQqJQqI9EV66zjIi1fGqSJnop1byICEWOei0sFISVLOxM3bGdEX3b9z98yO0n63QHwNxQPeqfInv2JfMP+9VTDGcAW362mDmZEQvxdQ0cD8LlV94H0Y1fYMLB3YKD\\\\\\\/ghkbh1C2WxTUgTOban6IysA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lKheda1ubW9Sm1GOQtRDog==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222950c81cccd3857b24ab7700b6de8e4b4a94388a18c9a31bf75d7dd04c35e637\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kK8kT4RgmNWURpTnKGaKQPC0ty4dRz08pTBxKuRKjNWua5+HwzA5JlAhFc7VHINWS5L7fAOaMk4FJAy\\\\\\\/3+WFR7O79LCUQm1AWLammhaAGIm0XDrlviFU31upHvLR4SiZf6brMjxhFfkiG5jhHFhE6c6\\\\\\\/XbzncCJ5ed0H8lnS0d7SoYEnd5XAADp0hlznupzW34doARhrPPfxIh0WB0o5IlaAnQq0EDhUONkwGZ8xl7Se0MeVZ1OrPV\\\\\\\/\\\\\\\/TVayLWfD+AW7gzwkJ9eiHoupLxLa72HfkII1LuZMDvl7KDUZsVt5HB5YjJf3ZT0fXLOKUTi\\\\\\\/UV9sOfV\\\\\\\/tQqQ+wvXQoipv6l9pKLN0\\\\\\\/Z0dzbnTsjp\\\\\\\/rOeZ\\\\\\\/W8XD822fV\\\\\\\/nOTj57HIKWyWlYa\\\\\\\/nq1I6TN1FnleM8cB6kGhTsFsiPHCpOAGp5xAtCQ9BdJ9+oGSGSqn3r6nKBJY8WlsEV5EivEKkLrK3oMpe6Mu8pyzjreK3XIMxg9mmBrtfJKJM8jd8FEfxia7E\\\\\\\/odvIlFTQA70FycKWyCXG+4lOWaR5kAIiRBB\\\\\\\/xNujHkNjnF545G1VulQ8FJ1vXfqfGbBh1E1FSfIHpKYK83zj8C+Mpu55fhWMaWuc7H6VMR5JAeyz\\\\\\\/sGdeq6aax+eDEYWqWvC8l3d9xuvoOtEbfO0WwfDuTn4cwrQTkqw4wA+jZIPp7ghPLOZ6WwerE9XbmZo78Wi4rbbEi65sET0r9HNN42ycu6NCsVr4cj8Uyeyu9JL53CdEEeaU9nXBwidjECAGS9EfcaMHOFfLblynYE6TBlERQZOo7MzOy\\\\\\\/DvHV8Rd\\\\\\\/DfSm4NZ7Bkow5PqMTM\\\\\\\/vvQXU9hIbxNo7RuxN\\\\\\\/fhfgdSzeNrTHH7qI0Sj+J1k0C+h7ZHF+rCczbztBqFgBjdk+MMNk8MJK2zHYS5SS710CZJSkTc6\\\\\\\/oc8oFp6omUIYbwi4EbVQ5OYSR3\\\\\\\/alkt99O66ZIsyTwcZx7FhhKR\\\\\\\/fQoMP8w5zIpLTIgv2qbbRvsEJHRyn9zbEgrrdLC8SjarjBEqs6b\\\\\\\/RWzPcGwmi9qZG7rFIjEhEXR2CHYVzs4CMltLtUerPsyJR4wFBcl6gzoaZo5oIncohrEGYiilAFb3P5EPcBJanQchxFeoGv3nH+3zfyhLKTrIGzpYfpO2ayUVBHRl0K2UlWrX\\\\\\\/x4nHV4K7iJJ0G8GdhZ893Aw2HQbY1lSMNo\\\\\\\/lFqh6l5xn7jQ+6FpCVL0VjmgP8D+wY8CK23D1cpCT99zyUZBtAt9R8PFINjmBsz393YNMgXjoOsBXVsgiW1qXY1+6rQzuvyIpuFZOo4M9WKVv6rQGaB\\\\\\\/I0khYrcKzLULM9kTXF5sQj2q1inQpPAo\\\\\\\/yyHdE5n82wcbk5wrxymSMIV9B4iLD0adgOSBBmQ\\\\\\\/lh4O4wV+b1gl1zSdjrvfFGNEAxE775\\\\\\\/jJWhaPk6JRSAX1QctXnvke+S4x1ajbRC2gTNxUf0ZJShm6tbD5rQKxrcqhENZqkCZN+u\\\\\\\/mv8tD4O0FAfQhiBfwf6Qqm+THwz+64TTBlWuvT5h7ZIGdKqESqhZuEeTmOgZcXBVCz6w2fM2oMH2fo1gYBdCMcH0z5KGtBXTrpjBKGNpEeYdSj7yI0oBblA+9mRPvbFC+2igMdb+TgkR2icF4alahmnwnOVJpt1RPCMwJSbdCxv+i150BbgA3O9HPBv3k6Lkyhx5\\\\\\\/hPwuAEeeAz349O5yXdX5\\\\\\\/ln1AVPPx60pM0KSOmdaQhgkXSIiyKD08xfXpdTYqV5f\\\\\\\/9bV5+4urGcBGRLhDHd0MeXUPUeK4E7BghTvaWPc\\\\\\\/GptYX+rqM00ESjWfll2sVwvGzR1By56niNPUn51ByIvBPEc4+uZIvFZJOl3ioCvHSh65OARlGbRf9BrrYiVpAv0QFEdQsY+pDqDM6NqMx2MaKWjE8ucAj2RpSdP29vDv+W\\\\\\\/5pHQ0fyGbF0lq+LY6\\\\\\\/xjTfA3sXLKHdHYQ12VMgbYEtUrTSqMvlrm\\\\\\\/ww\\\\\\\/bNeRpp1uhQiZx9xuVj27kG9qTXpQmg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lXSK2wYo+s\\\\\\\/eTfRA1+aUUQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002247315f834723470e377a7280019222b071788defd72910774923ffc2b21ff7bd\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485839 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2577" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ga_hD_TyvOn89_Ve2Xv7vL-KUKhmKM-LrlZ5rLr9tzk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2577" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-O_e3xRy9HZ3xwCSxCEimLy4p1E0uZEDHKE3WRjBJJrM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2565" title="Duplication and Loss of Function of Genes Encoding RNA Polymerase III Subunit C4 Causes Hybrid Incompatibility in Rice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2595" title="Unique Allelic eQTL Clusters in Human MHC Haplotypes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485839" id="node485839" data-pisa="ggg;7/8/2577" data-pisa-master="ggg;g3.117.042622" data-apath="/ggg/7/8/2577.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptional Complexity and Distinct Expression Patterns of <em>auts2</em> Paralogs in <em>Danio rerio</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2268-7291" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Igor Kondrychyn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3510-9228" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lena Robra</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2318-5023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vatsala Thirumalai</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2577-2593;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042622 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485839" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Igor Kondrychyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bangalore, Karnataka 560065, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Igor%2BKondrychyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kondrychyn%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIgor%2BKondrychyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2268-7291" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Igor Kondrychyn</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lena Robra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bangalore, Karnataka 560065, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lena%2BRobra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robra%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALena%2BRobra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3510-9228" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lena Robra</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Vatsala Thirumalai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bangalore, Karnataka 560065, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vatsala%2BThirumalai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thirumalai%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVatsala%2BThirumalai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2318-5023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vatsala Thirumalai</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vatsala@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">vatsala@ncbs.res.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/8/2577" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485839" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2577.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485839" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2577.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485839" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2577.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485839" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/8/2577.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485839" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Several genes that have been implicated in autism spectrum disorders (ASDs) have multiple transcripts. Therefore, comprehensive transcript annotation is critical for determining the respective gene function. The autism susceptibility candidate 2 (<em>AUTS2</em>) gene is associated with various neurological disorders, including autism and brain malformation. AUTS2 is important for activation of transcription of neural specific genes, neuronal migration, and neurite outgrowth. Here, we present evidence for significant transcriptional complexity in the <em>auts2</em> gene locus in the zebrafish genome, as well as in genomic loci of <em>auts2</em> paralogous genes <em>fbrsl1</em> and <em>fbrs</em>. Several genes that have been implicated in ASDs are large and have multiple transcripts. Neurons are especially enriched with longer transcripts compared to nonneural cell types. The human autism susceptibility candidate 2 (<em>AUTS2</em>) gene is ∼1.2 Mb long and is implicated in a number of neurological disorders including autism, intellectual disability, addiction, and developmental delay. Recent studies show AUTS2 to be important for activation of transcription of neural specific genes, neuronal migration, and neurite outgrowth. However, much remains to be understood regarding the transcriptional complexity and the functional roles of AUTS2 in neurodevelopment. Zebrafish provide an excellent model system for studying both these questions. We undertook genomic identification and characterization of <em>auts2</em> and its paralogous genes in zebrafish. There are four <em>auts2</em> family genes in zebrafish: <em>auts2a</em>, <em>auts2b</em>, <em>fbrsl1</em>, and <em>fbrs</em>. The absence of complete annotation of their structures hampers functional studies. We present evidence for transcriptional complexity of these four genes mediated by alternative splicing and alternative promoter usage. Furthermore, the expression of the various paralogs is tightly regulated both spatially and developmentally. Our findings suggest that auts2 paralogs serve distinct functions in the development and functioning of target tissues.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/alternative-promoters" class="hw-term hw-article-keyword hw-article-keyword-alternative-promoters" rel="nofollow">alternative promoters</a></li><li class="kwd"><a href="/keyword/alternative-splicing" class="hw-term hw-article-keyword hw-article-keyword-alternative-splicing" rel="nofollow">alternative splicing</a></li><li class="kwd"><a href="/keyword/auts2" class="hw-term hw-article-keyword hw-article-keyword-auts2" rel="nofollow">auts2</a></li><li class="kwd"><a href="/keyword/fbrsl1" class="hw-term hw-article-keyword hw-article-keyword-fbrsl1" rel="nofollow">fbrsl1</a></li><li class="kwd"><a href="/keyword/fbrs" class="hw-term hw-article-keyword hw-article-keyword-fbrs" rel="nofollow">fbrs</a></li></ul><p id="p-4">The <em>AUTS2</em> gene was originally identified in monozygotic twins concordant for autism who shared a balanced translocation from chromosome 7 to 20 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sultana <em>et al.</em> 2002</a>). Since that initial discovery, various structural variants disrupting the <em>AUTS2</em> locus have been identified in unrelated individuals with ASDs as well as other neuropathological conditions, including epilepsy and brain malformations [reviewed in <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Oksenberg and Ahituv (2013)</a>]. Although, disruption of <em>AUTS2</em> is most often reported to be associated with neurological phenotypes, copy number variations at the <em>AUTS2</em> locus have also been observed in unaffected individuals (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Redon <em>et al.</em> 2006</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bakkaloglu <em>et al.</em> 2008</a>), indicating that structural polymorphisms are tolerated in some cases. In addition, a noncoding single nucleotide polymorphism (SNP) within <em>AUTS2</em> has been associated with schizophrenia, alcohol consumption, heroin addiction, and suicidal behavior (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chen <em>et al.</em> 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chojnicka <em>et al.</em> 2013</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Coon <em>et al.</em> 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dang <em>et al.</em> 2014</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kapoor <em>et al.</em> 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">McCarthy <em>et al.</em> 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Schumann <em>et al.</em> 2011</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zhang <em>et al.</em> 2014</a>). <em>AUTS2</em> has also been implicated as an important gene in human-specific evolution (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Prabhakar <em>et al.</em> 2006</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Green <em>et al.</em> 2010</a>).</p><p id="p-5">AUTS2 is a highly conserved protein, primarily expressed in the brain in various neuronal cell types including glutamatergic, GABAergic, and dopaminergic neurons, as well as in regions implicated in autism neuropathology, such as the cerebral cortex and cerebellum (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Sultana <em>et al.</em> 2002</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kalscheuer <em>et al.</em> 2007</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bedogni <em>et al.</em> 2010</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wagner <em>et al.</em> 2013</a>). Although several features, such as putative nuclear localization sequences (NLS), two proline-rich regions (PR1 and PR2), the PY motif (PPPY), and histidine repeats, can be predicted in the human AUTS2 protein (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Sultana <em>et al.</em> 2002</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Oksenberg and Ahituv 2013</a>), it does not contain any regions of homology to other proteins, except a region called the Auts2 family domain, with homology to the fibrosin (FBRS) and fibrosin-like 1 (FBRSL1) proteins. <em>AUTS2</em>, <em>FBRSL1</em>, and <em>FBRS</em> genes thus form a paralog group. Paralogs are known to be created by a duplication event within a genome and may evolve new functions. The functional roles of Auts2 family proteins and whether they are functionally diversified are not yet entirely clear.</p><p id="p-6">Recently, proteomic analysis revealed that AUTS2, FBRS, and FBRSL1 are associated with the same subset of Polycomb Repressive Complex 1 (PRC1) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gao <em>et al.</em> 2012</a>) and, in particular, AUTS2 renders PRC1 capable of transcription activation (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gao <em>et al.</em> 2014</a>). Though initially thought to be a nuclear protein (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bedogni <em>et al.</em> 2010</a>), AUTS2 was shown to be also present in the cytoplasm, where it regulates Rho family GTPases to control neurite outgrowth and neuronal migration (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hori <em>et al.</em> 2014</a>). Thus, it seems that AUTS2 is a multifunctional protein regulating distinct pathways in neural development. The roles of FBRS and FBRSL1 in neural development have not yet been explored.</p><p id="p-7">We wanted to study the functions of <em>auts2</em> family genes in neural development using the zebrafish model system. Zebrafish provide multiple advantages for addressing these questions because of the ease of genetics, the transparency in early life stages, and the ability to image and record neural activity during nervous system development. As a first step toward this goal, in this manuscript, we describe the transcriptional complexity and the distinct expression patterns of <em>auts2</em> family genes in zebrafish.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Zebrafish use and care</h3><p id="p-8">Zebrafish (<em>Danio rerio</em>) of Indian wild-type strain were purchased from local suppliers and housed in aquarium tanks at 28° with a 14:10 hr light:dark cycle. Fish were maintained according to established protocols (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Westerfield 2000</a>) in agreement with the Institutional Animal Ethics Committee and the Institutional Biosafety Committee, National Centre for Biological Sciences.</p></div><div id="sec-3" class="subsection"><h3>Sequence collection and gene structure annotation</h3><p id="p-9"><em>auts2</em> paralogs were identified using the Ensembl genome browser (GRCz10, Ensembl release 86). We manually curated sequences to extract intron–exon structural information. Gene models were retrieved from the Reference Sequence (RefSeq) database (NCBI <em>D. rerio</em> Annotation Release 105) and were validated experimentally by the sequence analysis of cDNA clones from our 5′-RACE (rapid amplification of cDNA ends) and RT-PCR experiments (see below). The putative alternative transcription start sites (TSSs) were identified using our 5′-RACE data and RNASeq gene models, generated from the Wellcome Trust Sanger Institute Zebrafish Transcriptome Sequencing Project, Ref: ERP000016 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Collins <em>et al.</em> 2012</a>). The annotated RNASeq gene models are incorporated into the Ensembl genome browser (<a href="http://www.ensembl.org/Danio_rerio/">www.ensembl.org/Danio_rerio/</a>). In our analysis, we assumed that if the position of the first nucleotide in RNASeq transcript is annotated in an intron, it can be considered as a putative TSS.</p></div><div id="sec-4" class="subsection"><h3>Rapid amplification of 5′ cDNA ends (5′-RACE), RT-PCR, and cloning of the full-length cDNAs of auts2 paralogs</h3><p id="p-10">For 5′-RACE experiments, one microgram of a total RNA isolated from 24 hr embryos was utilized as a template for synthesis of first-strand cDNA using a SMARTer RACE cDNA amplification kit (Clontech) according to the manufacturer’s instructions. The 5′-RACE reactions were performed using the Advantage 2 Polymerase Mix (Clontech). Final PCR products were cloned into pCRII-TOPO vector (Invitrogen) and sequenced.</p><p id="p-11">For RT-PCR analysis, total RNA was isolated from zebrafish embryos at different developmental stages using an RNeasy Mini kit (QIAGEN) and first-strand cDNAs were synthesized from 1 μg of a total RNA by oligo(dT) priming using SMARTScribe Reverse Transcriptase (Clontech) according to the manufacturer’s protocol. Amplification of cDNA was performed using Herculase II Fusion DNA polymerase (Agilent). Identity of amplified PCR products was verified by direct sequencing. The same batch of cDNAs was used to profile expression of <em>auts2</em> paralogs during development.</p><p id="p-12">Full-length cDNAs of <em>auts2</em> paralogs were amplified using Q5 Hot Start High-Fidelity DNA Polymerase (New England Biolabs) and the resulting PCR products were cloned into pCR-Blunt vector (Invitrogen). Positive clones were verified by sequencing. Sequences of all primers used in this study will be provided upon request.</p></div><div id="sec-5" class="subsection"><h3>RNA probe synthesis</h3><p id="p-13">The cDNA-containing vectors were linearized with appropriate restriction enzymes (detailed maps of vectors will be provided upon request) and used as a template for RNA probe synthesis. Sense and antisense RNA probes were synthesized using MEGAscript SP6 or T7 kits (Ambion) and either digoxigenin-labeled or fluorescein-labeled rNTPs (Roche).</p></div><div id="sec-6" class="subsection"><h3>Whole-mount in situ hybridization (WISH)</h3><p id="p-14">Colorimetric WISH was conducted according to standard protocol (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Thisse and Thisse 2008</a>) with minor modifications. The protocol was identical for both embryos and whole brains. Briefly, embryos at different developmental stages were fixed in 4% paraformaldehyde in phosphate-buffered saline (PBS, pH 7.4) at 4° overnight. For expression analysis in the juvenile brain, 45-d-old fish were anesthetized in 0.02% Tricaine (ethyl 3-aminobenzoate methanesulfonate, Sigma) and then decapitated. The brains were dissected out from the skull, immediately immersed in cold 4% paraformaldehyde, and fixed at 4° for 24 hr. After fixation, embryos and brains were dehydrated in 100% methanol and stored at −20° until use. The methanol-stored embryos and brains were rehydrated in PBS containing 0.1% Tween-20 (PBST) and permeabilized by proteinase K treatment (Ambion, 10 μg/ml; incubation time was 10 min for brains and 18–24 hr embryos, and 20 min for 48 hr embryos; embryos younger than 18 hr were not treated with proteinase K). Hybridization was carried out in buffer [50% formamide, 5× SSC, 50 μg/ml heparin, 500 μg/ml tRNA (Roche), and 0.1% Tween-20] containing 5% dextran sulfate at 69° overnight. After stringency wash, the specimens were blocked with 2% blocking reagent (Cat. No. 11096176001; Roche) in maleic acid buffer (100 mM maleic acid, 150 mM NaCl, and 0.1% Tween-20, pH 7.5) and incubated overnight with anti-digoxigenin antibody, conjugated with alkaline phosphatase (1:5000; Roche) at 4°. For the colorimetric detection of alkaline phosphatase, specimens were incubated in staining buffer (100 mM Tris-HCl, 100 mM NaCl, 50 mM MgCl<sub>2</sub>, and 0.1% Tween-20, pH 9.5) containing 375 μg/ml nitro-blue tetrazolium chloride (Roche) and 175 μg/ml 5-bromo-4-chloro-3′-indolyl-phosphate (Roche).</p><p id="p-15">The following antisense and sense digoxigenin-labeled riboprobes were generated: (i) <em>auts2a</em>, 4.2 kb long probe corresponds to <em>auts2a-i3</em> isoform (structure of isoform can be found in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1B</a>), probe comprises 1508 bp of 5′-UTR, 2433 bp of open reading frame (ORF), and 174 bp of 3′-UTR; (ii) <em>fbrsl1</em>, 5.2 kb long probe corresponds to <em>fbrsl1-i2b</em> isoform (structure of isoform can be found in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6B</a>), probe comprises 1252 bp of 5′-UTR, 2529 bp of ORF, and 1291 bp of 3′-UTR; (iii) <em>fbrs</em>, 5.2 kb long probe comprises 780 bp of 5′-UTR, 3648 bp of ORF, and 677 bp of 3′-UTR; and (iv) <em>auts2b</em>, 2.5 kb long probe comprises 144 bp of 5′-UTR, 2247 bp of ORF, and 25 bp of 3′-UTR. Sense probes showed no specific or unspecific staining (see Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS1.jpeg">Figure S1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene structure and transcript complexity in the zebrafish auts2a gene locus. (A) Schematic presentation of the 560 kb long auts2a gene locus on chromosome 10 (not to scale). Exons are shown as bars. Mutually exclusive first exons (1A, 1B, 1C, 1D, and 1E) are in red. Arrows mark TSSs supported by either 5&#x2032;-RACE and RT-PCR (blue) or RNASeq data (purple). TSSs are associated with PAPs. TSS3 was mapped inside of the coding region of exon 2. (B) Overview of transcripts generated in the auts2a gene locus. RNAs identified in this study and RNASeq transcripts from a public database are shown in blue and purple, respectively. Introns are shown as a line. Noncoding and coding exons are depicted as open and filled bars, respectively. RNASeq transcript&#x2019;s IDs are shown in Table S2. Exons 2L, 6L, 7L, 8L, 16L, and 17L are 5&#x2032; extensions of the corresponding constitutive exons. (C) RT-PCR analysis of auts2a isoform expression during zebrafish development. Primers amplifying the full-length transcripts were used for analysis. Letters a, b, and c stand for the respective isoforms. (D) Partial DNA sequence of coding exons undergoing alternative splicing. Positions of alternative 5&#x2032; donor (exon 9) or 3&#x2032; acceptor (exons 3, 8, and 15) splice sites are highlighted in orange. Constitutive splice sites are underlined. Splicing at these sites leads to in-frame deletion of either single or seven amino acids (red) in the Auts2a protein. Exonic sequence is shown in upper case. Alternatively spliced isoforms identified in this study correspond to RefSeq transcripts (GenBank accession numbers are shown). ID, identifier; PAP, putative alternative promoter; RACE, rapid amplification of cDNA ends; RNASeq, RNA sequencing; RT-PCR, reverse transcription-polymerase chain reaction; TSS, transcription start site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1861302268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gene structure and transcript complexity in the zebrafish &lt;em&gt;auts2a&lt;/em&gt; gene locus. (A) Schematic presentation of the 560 kb long &lt;em&gt;auts2a&lt;/em&gt; gene locus on chromosome 10 (not to scale). Exons are shown as bars. Mutually exclusive first exons (1A, 1B, 1C, 1D, and 1E) are in red. Arrows mark TSSs supported by either 5&#x2032;-RACE and RT-PCR (blue) or RNASeq data (purple). TSSs are associated with PAPs. TSS3 was mapped inside of the coding region of exon 2. (B) Overview of transcripts generated in the &lt;em&gt;auts2a&lt;/em&gt; gene locus. RNAs identified in this study and RNASeq transcripts from a public database are shown in blue and purple, respectively. Introns are shown as a line. Noncoding and coding exons are depicted as open and filled bars, respectively. RNASeq transcript&#x2019;s IDs are shown in Table S2. Exons 2L, 6L, 7L, 8L, 16L, and 17L are 5&#x2032; extensions of the corresponding constitutive exons. (C) RT-PCR analysis of &lt;em&gt;auts2a&lt;/em&gt; isoform expression during zebrafish development. Primers amplifying the full-length transcripts were used for analysis. Letters &lt;em&gt;a&lt;/em&gt;, &lt;em&gt;b&lt;/em&gt;, and &lt;em&gt;c&lt;/em&gt; stand for the respective isoforms. (D) Partial DNA sequence of coding exons undergoing alternative splicing. Positions of alternative 5&#x2032; donor (exon 9) or 3&#x2032; acceptor (exons 3, 8, and 15) splice sites are highlighted in orange. Constitutive splice sites are underlined. Splicing at these sites leads to in-frame deletion of either single or seven amino acids (red) in the Auts2a protein. Exonic sequence is shown in upper case. Alternatively spliced isoforms identified in this study correspond to RefSeq transcripts (GenBank accession numbers are shown). ID, identifier; PAP, putative alternative promoter; RACE, rapid amplification of cDNA ends; RNASeq, RNA sequencing; RT-PCR, reverse transcription-polymerase chain reaction; TSS, transcription start site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2577/F1.medium.gif" width="425" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/8/2577/F1.medium.gif" width="425" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485871" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">Gene structure and transcript complexity in the zebrafish <em>auts2a</em> gene locus. (A) Schematic presentation of the 560 kb long <em>auts2a</em> gene locus on chromosome 10 (not to scale). Exons are shown as bars. Mutually exclusive first exons (1A, 1B, 1C, 1D, and 1E) are in red. Arrows mark TSSs supported by either 5′-RACE and RT-PCR (blue) or RNASeq data (purple). TSSs are associated with PAPs. TSS3 was mapped inside of the coding region of exon 2. (B) Overview of transcripts generated in the <em>auts2a</em> gene locus. RNAs identified in this study and RNASeq transcripts from a public database are shown in blue and purple, respectively. Introns are shown as a line. Noncoding and coding exons are depicted as open and filled bars, respectively. RNASeq transcript’s IDs are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS2.xslx">Table S2</a>. Exons 2L, 6L, 7L, 8L, 16L, and 17L are 5′ extensions of the corresponding constitutive exons. (C) RT-PCR analysis of <em>auts2a</em> isoform expression during zebrafish development. Primers amplifying the full-length transcripts were used for analysis. Letters <em>a</em>, <em>b</em>, and <em>c</em> stand for the respective isoforms. (D) Partial DNA sequence of coding exons undergoing alternative splicing. Positions of alternative 5′ donor (exon 9) or 3′ acceptor (exons 3, 8, and 15) splice sites are highlighted in orange. Constitutive splice sites are underlined. Splicing at these sites leads to in-frame deletion of either single or seven amino acids (red) in the Auts2a protein. Exonic sequence is shown in upper case. Alternatively spliced isoforms identified in this study correspond to RefSeq transcripts (GenBank accession numbers are shown). ID, identifier; PAP, putative alternative promoter; RACE, rapid amplification of cDNA ends; RNASeq, RNA sequencing; RT-PCR, reverse transcription-polymerase chain reaction; TSS, transcription start site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">cDNAs encoding <em>egr2a</em> (<em>krox20</em>) and <em>otx2</em> were kindly donated by Vladimir Korzh (International Institute of Molecular and Cell Biology, Warsaw); cDNAs encoding <em>wnt1</em> and <em>sim1a</em> were amplified and cloned into pGEM-TEasy (Promega) and pCRII-TOPO (Invitrogen) vectors, respectively. These cDNAs were used to generate fluorescein-labeled antisense riboprobes.</p></div><div id="sec-7" class="subsection"><h3>Double in situ hybridization</h3><p id="p-18">To detect two riboprobes simultaneously, we used a two-color <em>in situ</em> hybridization protocol combining colorimetric and tyramide signal amplification-based fluorescent detection systems. The procedure for double <em>in situ</em> hybridization on embryos was essentially the same as that for colorimetric WISH, with the following modifications. (1) Prior to proteinase K treatment, embryos were incubated in 3% H<sub>2</sub>O<sub>2</sub> in PBST for 30 min to quench endogenous peroxidase activity. (2) Digoxigenin-labeled and fluorescein-labeled riboprobes were mixed and used simultaneously during hybridization. (3) We first detected the fluorescein-labeled riboprobe using anti-fluorescein antibody, conjugated with horse-radish peroxidase (1:500; Roche). After washes, embryos were incubated for 30 min in the dark with the tyramide-Alexa Fluor 488 (Cat. No. T20948; Molecular Probes) working solution, prepared according to the manufacturer’s instructions. (4) The detection of the digoxigenin-labeled riboprobe was performed as for the colorimetric WISH using anti-digoxigenin antibody, conjugated with alkaline phosphatase. In case of fluorescent detection of the digoxigenin-labeled riboprobe, we first quenched peroxidase conjugated with anti-fluorescein antibody followed by incubation of embryos with anti-digoxigenin antibody conjugated with peroxidase (1:1000; Roche). After washes, embryos were incubated with the tyramide-Alexa Fluor 594 (Cat. No. T20950; Molecular Probes) working solution.</p></div><div id="sec-8" class="subsection"><h3>Microscopy and imaging</h3><p id="p-19">Embryos were mounted in 70% glycerol for microscopy and imaging. The brains were embedded in 1.5% agar, blocks were saturated with 30% sucrose in PBS at 4° overnight, and then cut at 12 μm using a cryostat (Leica CM1850 UV). Light and fluorescent images were acquired using a stereomicroscope (OLYMPUS SZX16) equipped with a mercury lamp and digital camera (Jenoptik ProgRes C3). The images were processed using ImageJ (NIH) and Adobe Photoshop CS3.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-20">All constructs, primers, and raw data are available upon request. Nucleotide sequences are deposited to GenBank under accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KY492367&amp;atom=%2Fggg%2F7%2F8%2F2577.atom">KY492367</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KY492385&amp;atom=%2Fggg%2F7%2F8%2F2577.atom">KY492385</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Structure of the zebrafish auts2a gene locus</h3><p id="p-21">We initially performed bioinformatic analysis to evaluate the structure of the zebrafish <em>auts2a</em> gene locus. The current RefSeq <em>auts2a</em> gene model (NCBI Gene ID: 368890) defines 19 exons annotated in this genomic locus (exons 1A–19 in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>) and incorporates six computationally annotated transcripts. The RefSeq transcript XM_009305336 begins with untranslated exon 1A (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>) and codes for the long protein isoform with 1278 amino acid residues (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS2.pptx">Figure S2A</a>). Transcript XM_009305341 begins with the mutually exclusive exon 1B located in intron 3 and spliced to exon 4 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). This transcript codes for a protein of 1128 amino acid residues with 16 unique residues at the N-terminal end and lacking 166 N-terminal amino acid residues present in the long isoform (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS2.pptx">Figure S2B</a>). The other four RefSeq transcripts represent alternatively spliced variants of the long isoform XM_009305336. Alternative splicing occurs at tandem splice acceptors with a NAGNAG motif found at 3′ acceptor splice sites of exon 3 (transcript XM_017358057), exon 8 (transcript XM_017358056), and exon 15 (transcript XM_017358058). Alternative tandem splicing of mRNA leads to the exclusion of 3 nt (nucleotides) and, as a result, deletion of a single amino acid in the protein (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>). Splicing at an alternative 5′ donor splice site of exon 9 (transcript XM_017358059) leads to the exclusion of 21 nt from the spliced mRNA and, as a result, in-frame deletion of seven amino acids in the auts2a protein (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a>).</p><p id="p-22">In addition to the RefSeq transcripts, numerous RNASeq transcripts generated from the Wellcome Trust Sanger Institute Zebrafish Transcriptome Sequencing Project (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Collins <em>et al.</em> 2012</a>) are currently incorporated into the <em>auts2a</em> gene locus (for details visit <a href="http://www.ensembl.org/Danio_rerio/">www.ensembl.org/Danio_rerio/</a>). The majority of RNASeq transcripts are apparently transcribed from unique TSSs and detected at specific developmental time points (summarized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS2.xslx">Table S2</a>). Analysis of RNASeq data revealed two additional mutually exclusive first exons, 1D and 1E, located in introns 5 and 6 and spliced to exons 6 and 7, respectively (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1A</a>). Transcript RNASEQT00000015232, identified from an olfactory epithelium RNA library (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS2.xslx">Table S2</a>), begins with exon 1E and encodes a protein of 1083 amino acid residues with 32 unique residues at the N-terminus. It lacks 227 residues in the N-terminus that are present in the long protein isoform (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS2.pptx">Figure S2B</a>). In the zebrafish genome assembly (GRCz10, Ensembl release 86), transcript ENSDART00000078920 corresponds to RNASEQT00000015232.</p><p id="p-23">In addition, several alternative first exons that represent 5′ extensions of the corresponding constitutive exons, exon 2L (5′ extension of exon 2), exon 6L, exon 7L, exon 8L, exon 16L, and exon 17L, were identified from RNASeq data (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS2.xslx">Table S2</a>). TSSs associated with these exons are located in the flanking introns: intron 1 (TSS2), intron 5 (TSS7), intron 6 (TSS9), intron 7 (TSS10), intron 15 (TSS12), and intron 17 (TSS13) (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1A</a>, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS3.docx">Figure S3</a> for details). The exon beginning with TSS11 encompasses sequences of exon 13, intron 13, and exon 14 (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS3.docx">Figure S3J</a> for details). The majority of mRNAs transcribed from these TSSs have alternative 3′-ends in introns and either encode small proteins or represent putative noncoding RNAs (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS2.xslx">Table S2</a>).</p><p id="p-24">From <em>in silico</em> data analysis, we could define the presence of multiple putative TSSs that are apparently used to generate alternative <em>auts2a</em> mRNAs. Assuming that different TSSs are supposed to be associated with the corresponding putative alternative promoters (PAPs), we could predict, in addition to the major 5′ promoter associated with TSS1, the existence of multiple PAPs in the <em>auts2a</em> gene locus (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1A</a>).</p></div><div id="sec-12" class="subsection"><h3>Identification of novel transcripts in the zebrafish auts2a gene locus</h3><p id="p-25">To clarify <em>in silico</em> data and to identify novel mRNAs transcribed from the <em>auts2a</em> gene locus, we performed 5′-RACE and RT-PCR analyses followed by isolation of full-length cDNAs encoding zebrafish <em>auts2a</em> mRNAs.</p><p id="p-26">For 5′-RACE, we designed gene-specific primers in the 5′-part of the gene (in exons 4 and 6). Total RNA isolated from 24 hr embryos was used as a template in 5′-RACE experiments. We amplified 11 5′-RACE products. The majority of the 5′-ends were mapped in close proximity to the annotated positions of the first nucleotides of RNASeq or RefSeq transcripts (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS3.docx">Figure S3</a> for details). Sequence analysis of 5′-RACE products confirmed previously annotated alternative first exons 1A, 1B, and 1D, and identified a novel mutually exclusive first exon 1C located in intron 4, placed 51,016 nt downstream from exon 1B (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1A</a>). Exon 1C is spliced to exon 4, similar to exon 1B, and no overlapping RefSeq or RNASeq annotations were found for this exon. When we used a reverse primer designed in the 3′-UTR of exon 19, we could not amplify cDNA beginning with exon 1C. However, the transcript corresponding to the 5′-RACE product could be detected by RT-PCR during analysis of <em>auts2a</em> expression through zebrafish development (TSS5 in <a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1C</a>), suggesting that RNA transcribed from TSS5 has an alternative 3′-end.</p><p id="p-27">We noticed a few interesting features when we analyzed the 5′-RACE results. First, multiple 5′-ends of different lengths were mapped into exons 1A (five 5′-ends) and 1D (four 5′-ends) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS3.docx">Figure S3, A and E</a>). The mapping was in close proximity to the annotated positions of the first nucleotides of RNASeq and/or RefSeq transcripts. Remarkably, all TSSs in exon 1A (annotated and experimentally determined) were clustered within 58 nt, while TSSs in exon 1D were spread over 300 nt. From genome-wide analysis of mammalian promoters, it was found that sharp starting sites are generally associated with promoters having TATA-boxes, while promoters associated with CpG islands do not show an accurate TSS, but instead a broad distribution of TSSs generally spread over 100 nt (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carninci <em>et al.</em> 2006</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gustincich <em>et al.</em> 2006</a>). We found a TATA-box motif in the promoter associated with TSS1 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS3.docx">Figure S3A</a>), but not in the promoter associated with TSS6. Except for the brain, where TSSs are surprisingly enriched in CpG islands, TATA-box-promoted transcripts tend to be tissue-specific (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Gustincich <em>et al.</em> 2006</a>). Despite this fact, the presence of multiple TSSs mapped to a single exon could also be accounted for by random transcription initiation events from the same promoter; even a small extension in the 5′-end sequence may also play an important role, for example, by harboring upstream ATGs (and upstream ORFs) that throttle the translation from the downstream authentic ATG.</p><p id="p-28">Second, in one case we mapped a 5′-end into the coding region of exon 2 (TSS3 in <a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1A</a>), 603-bp downstream from the first nucleotide of transcript RNASEQT00000008207 annotated in intron 1 (this transcript begins with alternative exon 2L, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS3.docx">Figure S3B</a>). It is generally assumed that the range over which the TSSs are scattered is on average 62 bp (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Suzuki <em>et al.</em> 2001</a>) and that two independent TSS clusters, associated with distinct PAPs, are separated with &gt; 500 bp intervals (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kimura <em>et al.</em> 2006</a>). Since TSS3 was mapped at a distance &gt; 500 bp from the first nucleotide of the RNASeq transcript, it is likely that TSS3 is associated with a different PAP, perhaps overlapping with that of TSS2. Although the location of TSS within the internal exon could be accounted for by the truncated cDNAs, it could also be a genuine “internal” putative promoter with unique features different from canonical promoters since the corresponding genomic regions serve as both exon and promoter. Interestingly, genes having TATA-box promoters are also preferentially associated with the presence of unusual transcripts, originating from exons (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Carninci <em>et al.</em> 2006</a>).</p><p id="p-29">To amplify full-length cDNAs that begin with alternative first exons, we designed a set of exon-specific forward primers that bind in close proximity to the annotated TSSs (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS3.docx">Figure S3</a> for positions of individual primers). Since reverse primers were bound to the 3′-UTR of exon 19, we could only amplify the population of cDNAs with complete ORFs that differ in the 5′-end but were similar in the 3′-end. With such an approach, the presence of alternative 3′-ends was not explored. The same set of primers was used for RT-PCR analysis of <em>auts2a</em> expression during zebrafish development.</p><p id="p-30">We cloned four variants of the long isoform <em>auts2a-i1</em>, all beginning with exon 1A and corresponding to the RefSeq transcripts: (1) <em>auts2a-i1a</em> (XM_009305336), (2) <em>auts2a-i1b</em> (XM_017358057), and (3) <em>auts2a-i1c</em> (XM_017358056). In the fourth transcript variant <em>auts2a-i1d</em>, noncoding exon 1A was spliced to exon 2 by utilizing a cryptic 5′ donor splice site located in the intron, 132 nt downstream from the constitutive splice site (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS3.docx">Figure S3A</a>). This leads to the formation of a transcript with different 5′-UTR length but does not affect the coding region of <em>auts2a</em>. Transcript <em>auts2a-i1d</em> also has an alternatively spliced exon 8, similar to isoform <em>auts2a-i1c</em>. We did not clone transcript variants corresponding to XM_017358059 (alternative exon 15) and XM_017358058 (alternative exon 9). However, direct sequencing of PCR products revealed coamplification of transcripts with constitutively and alternatively spliced exons 3, 8, and 15. RT-PCR analysis showed that the <em>auts2a-i1</em> isoform is maternally supplied and is also present from 9 to 72 hr (the last examined time point; TSS1 in <a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1C</a>).</p><p id="p-31">Transcript isoform <em>auts2a-i2</em> beginning with exon 1B corresponds to XM_009305341 (<a id="xref-fig-1-16" class="xref-fig" href="#F1">Figure 1B</a>), with the exception that, in contrast to the RefSeq transcript, cDNA of <em>auts2a-i2</em> has both alternatively spliced exons 8 and 15. During zebrafish development, <em>auts2a-i2</em> was detected only at 24, 48, and 72 hr (TSS4 in <a id="xref-fig-1-17" class="xref-fig" href="#F1">Figure 1C</a>).</p><p id="p-32">Transcript isoform <em>auts2a-i3</em> begins with untranslated exon 1D and contains alternatively spliced exons 8 and 15 (<a id="xref-fig-1-18" class="xref-fig" href="#F1">Figure 1B</a>). Direct sequencing of PCR products revealed coamplification of transcripts with constitutively and alternatively spliced exons 8 and 15, similar to the long isoform <em>auts2a-i1</em>. Exon 8 in isoform <em>auts2a-i3</em> consists of 181 nt of the 5′-UTR and 94 nt of CDS, and splicing at an alternative 3′ acceptor splice site in this isoform does not affect protein sequence as it does in isoform <em>auts2a-i2</em> (deletion of serine). The ORF of this transcript is identical to that of the long isoform except that it lacks 467 amino acid residues in the N-terminus (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS2.pptx">Figure S2A</a>). Similar to isoform <em>auts2a-i1</em>, <em>auts2a-i3</em> is maternally supplied and then is detected from 6 to 72 hr during zebrafish development (TSS6 in <a id="xref-fig-1-19" class="xref-fig" href="#F1">Figure 1C</a>).</p><p id="p-33">Two novel transcript isoforms, <em>auts2a-i4a</em> and <em>auts2a-i4b</em> beginning with exon 1E, were isolated during RT-PCR analysis. In contrast to the annotated transcript RNASEQT00000015232, exon 1E in isoform <em>auts2a-i4a</em> has 85 nt of 3′ extension due to the usage of alternative 5′ donor splice site (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS3.docx">Figure S3G</a>). Isoform <em>auts2a-i4a</em> encodes a protein of 1082 amino acid residues with a unique stretch of nine residues at the N-terminal end <em>vs.</em> 32 unique residues in protein coded by RNASEQT00000015232 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS2.pptx">Figure S2B</a>). In isoform <em>auts2a-i4b</em>, the first exon retains 1786 nt of intronic sequence and extends through exon 7. Despite this intron retention, the ORF of <em>auts2a-i4b</em> is identical to that of the <em>auts2a-i3</em> isoform. However, the sequence of the 5′-UTR (2880 nt long) in this isoform differs from that in isoform <em>auts2a-i3</em> (5′-UTR of 1684 nt long). During zebrafish development, isoform <em>auts2a-i4a</em> was detected exclusively at 9 hr, while <em>auts2a-i4b</em> could be detected at 12, 16, and 48 hr (TSS8 in <a id="xref-fig-1-20" class="xref-fig" href="#F1">Figure 1C</a>). We could not isolate cDNA with the structure of exon 1E corresponding to RNASEQT00000015232. Since this RNASeq transcript was found in sample prepared from olfactory epithelium of adult fish (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS2.xslx">Table S2</a>), it may explain why we could not amplify it during our RT-PCR analysis.</p><p id="p-34">RNASeq data show two transcripts beginning with exon 8L (5′ extension of exon 8) having alternative 3′-ends and encoding small proteins of 219 and 196 amino acid residues with unique N- and C-terminal ends (see <a id="xref-fig-1-21" class="xref-fig" href="#F1">Figure 1B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS2.xslx">Table S2</a>). During RT-PCR analysis, we could isolate three novel isoforms beginning with exon 8L that share their 3′-ends with the long isoform <em>auts2a-i1</em>. Isoform <em>auts2a-i5a</em> encodes a protein of 888 amino acid residues with 15 unique residues at the N-terminal end (<a id="xref-fig-1-22" class="xref-fig" href="#F1">Figure 1B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS2.pptx">Figure S2B</a>). Two other isoforms represent alternatively spliced transcripts with retention of intronic sequences that lead to premature stop codons. Isoform <em>auts2a-i5b</em> retains 38 nt from intron 15, while isoform <em>auts2a-i5c</em> retains 745 nt from introns 12 and 13, including exon 13. During zebrafish development, isoform <em>auts2a-i5a</em> was detected at 12, 16, 48, and 72 hr; isoform <em>auts2a-i5b</em> at 9 and 12 hr; and isoform <em>auts2a-i5c</em> at 24 hr only (TSS10 in <a id="xref-fig-1-23" class="xref-fig" href="#F1">Figure 1C</a>).</p></div><div id="sec-13" class="subsection"><h3>Expression of auts2a mRNA during zebrafish development</h3><p id="p-35">We conducted WISH to examine how the spatial expression patterns of <em>auts2a</em> transcripts vary during development. Two isoforms, <em>auts2a-i1</em> and <em>auts2a-i3</em>, were expressed through all developmental stages according to RT-PCR analysis (<a id="xref-fig-1-24" class="xref-fig" href="#F1">Figure 1C</a>). Since the usage of an antisense RNA probe detecting full-length <em>auts2a-i1</em> isoform showed ubiquitous staining (data not shown), we used an antisense probe detecting full-length <em>auts2a-i3</em> isoform for analysis, which showed more restricted staining. For probe details see <em>Materials and Methods</em>.</p><p id="p-36">Weak expression of <em>auts2a-i3</em> is first detected at the 1-somite stage (10.3 hr) in a medial part of the presumptive forebrain and more intensely in the presumptive hindbrain. In the hindbrain region, <em>auts2a-i3</em> expression extends into the mesoderm laterally to the neural plate. It is also expressed in the somite (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and A</a>’). At the 3-somite stage (11 hr), <em>auts2a-i3</em> continues to be weakly expressed in the presumptive forebrain and more intensely in the presumptive midbrain and hindbrain. It is also expressed outside of the neural plate, in two longitudinal stripes lateral to the hindbrain, presumably the primordial cranial ganglia, and in somites (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2, B and B</a>’). During neural keel formation (12–14 hr), weak expression of <em>auts2a-i3</em> is continued in the forebrain and more intensely in midbrain and hindbrain, with stronger expression in presumed rhombomere 4 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2, C–C</a>” and D–D”). To demarcate <em>auts2a</em> expression in the brain at this developmental stage, we performed double <em>in situ</em> hybridization using an <em>egr2a</em> (<em>krox20</em>) probe to mark rhombomeres 3 and 5 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2E</a>’), and a <em>wnt1</em> probe to mark the midbrain–hindbrain boundary (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2, F</a>’ and G’). Indeed, <em>auts2a-i3</em> was expressed in the telencephalon with more intense expression in the dorsal part, and ventrally in the rostral diencephalon (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2G</a>), posterior midbrain, rhombomeres 1 and 4, and the spinal cord (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2, E, F, and G</a>). At 18 hr, <em>auts2a-i3</em> is expressed in the telencephalon, diencephalon, midbrain, hindbrain, and spinal cord. It is also expressed in somites (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2, H and H</a>’). At 24 hr, <em>auts2a-i3</em> is expressed in the brain with stronger expression in the tectum. It is also expressed in the spinal cord, somites, and pectoral fin buds (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2, I and I</a>’).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression of auts2a mRNA during zebrafish development. Whole-mount in situ hybridization analysis of auts2a-i3 transcript expression in wild-type embryos at different developmental stages: 10.3 hr (A and A&#x2019;), 11 hr (B and B&#x2019;), 12 hr (C&#x2013;C&#x201D;), 14 hr (D&#x2013;D&#x201D;), 18 hr (H and H&#x2019;), and 24 hr (I and I&#x2019;). (E&#x2013;G&#x2019;) Double in situ hybridization with egr2a (krox20) (E and E&#x2019;) and wnt1 (F&#x2019; and G&#x2019;) as second probes. In (G), dotted lines approximately demarcate rhombomere borders and midbrain. (K&#x2013;Q) Transverse sections at the levels indicated by the vertical lines in (J) show auts2a-i3 mRNA expression in different regions of the juvenile brain: telencephalon (K and L), diencephalon and midbrain (M&#x2013;O), and hindbrain (P and Q). Abbreviations used to label areas in brain sections can be found in Table S1. cb, cerebellum; cg, cranial ganglia; de, diencephalon; dte, dorsal telencephalon; fb, forebrain; hb, hindbrain; mb, midbrain; mhb, midbrain-hindbrain boundary; ms, mesoderm; r, rhombomere; s, somite; te, telencephalon; vde, ventral diencephalon." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1861302268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Expression of &lt;em&gt;auts2a&lt;/em&gt; mRNA during zebrafish development. Whole-mount &lt;em&gt;in situ&lt;/em&gt; hybridization analysis of &lt;em&gt;auts2a-i3&lt;/em&gt; transcript expression in wild-type embryos at different developmental stages: 10.3 hr (A and A&#x2019;), 11 hr (B and B&#x2019;), 12 hr (C&#x2013;C&#x201D;), 14 hr (D&#x2013;D&#x201D;), 18 hr (H and H&#x2019;), and 24 hr (I and I&#x2019;). (E&#x2013;G&#x2019;) Double &lt;em&gt;in situ&lt;/em&gt; hybridization with &lt;em&gt;egr2a&lt;/em&gt; (&lt;em&gt;krox20&lt;/em&gt;) (E and E&#x2019;) and &lt;em&gt;wnt1&lt;/em&gt; (F&#x2019; and G&#x2019;) as second probes. In (G), dotted lines approximately demarcate rhombomere borders and midbrain. (K&#x2013;Q) Transverse sections at the levels indicated by the vertical lines in (J) show &lt;em&gt;auts2a-i3&lt;/em&gt; mRNA expression in different regions of the juvenile brain: telencephalon (K and L), diencephalon and midbrain (M&#x2013;O), and hindbrain (P and Q). Abbreviations used to label areas in brain sections can be found in Table S1. cb, cerebellum; cg, cranial ganglia; de, diencephalon; dte, dorsal telencephalon; fb, forebrain; hb, hindbrain; mb, midbrain; mhb, midbrain-hindbrain boundary; ms, mesoderm; r, rhombomere; s, somite; te, telencephalon; vde, ventral diencephalon.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2577/F2.medium.gif" width="363" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/8/2577/F2.medium.gif" width="363" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485950" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-37" class="first-child">Expression of <em>auts2a</em> mRNA during zebrafish development. Whole-mount <em>in situ</em> hybridization analysis of <em>auts2a-i3</em> transcript expression in wild-type embryos at different developmental stages: 10.3 hr (A and A’), 11 hr (B and B’), 12 hr (C–C”), 14 hr (D–D”), 18 hr (H and H’), and 24 hr (I and I’). (E–G’) Double <em>in situ</em> hybridization with <em>egr2a</em> (<em>krox20</em>) (E and E’) and <em>wnt1</em> (F’ and G’) as second probes. In (G), dotted lines approximately demarcate rhombomere borders and midbrain. (K–Q) Transverse sections at the levels indicated by the vertical lines in (J) show <em>auts2a-i3</em> mRNA expression in different regions of the juvenile brain: telencephalon (K and L), diencephalon and midbrain (M–O), and hindbrain (P and Q). Abbreviations used to label areas in brain sections can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS1.xslx">Table S1</a>. cb, cerebellum; cg, cranial ganglia; de, diencephalon; dte, dorsal telencephalon; fb, forebrain; hb, hindbrain; mb, midbrain; mhb, midbrain-hindbrain boundary; ms, mesoderm; r, rhombomere; s, somite; te, telencephalon; vde, ventral diencephalon.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">We also performed <em>in situ</em> hybridization analysis of <em>auts2a-i3</em> expression in the juvenile zebrafish brain (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2, J–Q</a>). In the forebrain, <em>auts2a-i3</em> expression is detected in the ventral telencephalic area with strong expression in the ventral nucleus (Vv) and anterior part of the parvocellular preoptic nucleus (PPa), as well as in the dorsal telencephalic area with strong expression in posterior (Dp) and medial parts (Dm) (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2, K and L</a>). Expression of <em>auts2a-i3</em> is also detected in the mammillary body (CM) (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2P</a>), a part of the hypothalamus that is important for recollective memory in rodents (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Vann 2010</a>), and in the periventricular zone of the hypothalamus (Hv) (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2M</a>). In the midbrain, <em>auts2a-i3</em> expression is primarily detected in layer three of the periventricular gray zone of the optic tectum (PGZ) (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2, M–O</a>). In the cerebellum, <em>auts2a-i3</em> is expressed in the Purkinje cell layer (PCL) (<a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2P</a>); in the hindbrain, expression is detected dorsally in facial (LVII) and glossopharyngeal (LIX) lobes, the caudal octavolateralis nucleus (CON), around the rhombencephalic ventricle (RV); and ventrally, in the inferior reticular formation (IRF) and inferior olive (IO) (<a id="xref-fig-2-16" class="xref-fig" href="#F2">Figure 2Q</a>).</p><p id="p-39">Since multiple transcript isoforms were identified in the <em>auts2a</em> gene locus, we wanted to determine if these isoforms are differentially expressed during development. To evaluate isoform-specific expression of <em>auts2a</em> transcripts, we designed riboprobes that recognize alternative first exons: mutually exclusive exons (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS4.jpeg">Figure S4A</a>). For <em>in situ</em> hybridization, we chose 24 hr embryos as this was the earliest stage at which three of the <em>auts2a</em> isoforms were expressed (<a id="xref-fig-1-25" class="xref-fig" href="#F1">Figure 1C</a>). We could detect reliable expression signals for <em>auts2a</em> transcripts generated from TSS1 and TSS6, but not for that from TSS4 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS4.jpeg">Figure S4B</a>), probably because of the short length of the probe (204 bp). The expression pattern of <em>auts2a</em> isoforms generated from TSS1 and TSS6 was quite similar in the brain, with stronger expression of the TSS6 isoform in the telencephalon and midbrain (tectum) in comparison to the TSS1 isoform (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS4.jpeg">Figure S4B</a>).</p></div><div id="sec-14" class="subsection"><h3>Duplication of the auts2 gene in teleost genomes</h3><p id="p-40">During evolution, teleost genomes experienced an additional round of whole-genome duplication (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Christoffels <em>et al.</em> 2004</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hoegg <em>et al.</em> 2004</a>). As a result, the zebrafish genome contains two <em>auts2</em> genes, <em>auts2a</em> on chromosome 10 and <em>auts2b</em> on chromosome 15 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3C</a>). The current RefSeq <em>auts2b</em> gene model (NCBI Gene ID: 100150849) defines 14 exons present in transcript XM_001921276 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). In addition, transcript XM_017351168 represents an alternatively spliced variant with a mutually exclusive first exon 1B, which is located in intron 2 and spliced to exon 3 (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). Analysis of RNASeq data allowed the prediction of an additional three alternative first exons representing 5′ extensions of constitutive exons: exon 2L (TSS2), exon 6L (TSS4), and exon 11L (TSS5) (summarized in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS3.xslx">Table S3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene structure and transcript complexity in the zebrafish auts2b gene locus. (A) Schematic presentation of the 20 kb long auts2b genomic region on Chr15 (not to scale). Exons are shown as bars. Mutually exclusive first exons (1A and 1B) are in red. Arrows show TSSs identified either in this study (blue) or annotated based on RNASeq data (purple). Structures of RNASeq transcripts transcribed from alternative TSSs are shown. RNASeq transcript IDs are shown in Table S3. Exons 2L and 11L are 5&#x2032; extensions of the corresponding exons. Noncoding and coding exons are depicted as open and filled bars, respectively. (B) Schematic presentation of the structure of auts2b genes in genomes of different teleosts. Conserved exon 9 is depicted in red. (C) Schematic showing synteny between auts2a and auts2b genomic regions in the zebrafish genome. (D) Alignment of amino acid sequence coded by conserved exons 9 (Auts2b) and 14 (Auts2a) from different teleosts (coding exon is highlighted in gray). Amino acids that are highly conserved among Auts2 paralogs are shown in red. Chr, chromosome; ID, identifier; RNA sequencing; TSS, transcription start site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1861302268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gene structure and transcript complexity in the zebrafish &lt;em&gt;auts2b&lt;/em&gt; gene locus. (A) Schematic presentation of the 20 kb long &lt;em&gt;auts2b&lt;/em&gt; genomic region on Chr15 (not to scale). Exons are shown as bars. Mutually exclusive first exons (1A and 1B) are in red. Arrows show TSSs identified either in this study (blue) or annotated based on RNASeq data (purple). Structures of RNASeq transcripts transcribed from alternative TSSs are shown. RNASeq transcript IDs are shown in Table S3. Exons 2L and 11L are 5&#x2032; extensions of the corresponding exons. Noncoding and coding exons are depicted as open and filled bars, respectively. (B) Schematic presentation of the structure of &lt;em&gt;auts2b&lt;/em&gt; genes in genomes of different teleosts. Conserved exon 9 is depicted in red. (C) Schematic showing synteny between &lt;em&gt;auts2a&lt;/em&gt; and &lt;em&gt;auts2b&lt;/em&gt; genomic regions in the zebrafish genome. (D) Alignment of amino acid sequence coded by conserved exons 9 (Auts2b) and 14 (Auts2a) from different teleosts (coding exon is highlighted in gray). Amino acids that are highly conserved among Auts2 paralogs are shown in red. Chr, chromosome; ID, identifier; RNA sequencing; TSS, transcription start site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2577/F3.medium.gif" width="383" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/8/2577/F3.medium.gif" width="383" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485957" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-41" class="first-child">Gene structure and transcript complexity in the zebrafish <em>auts2b</em> gene locus. (A) Schematic presentation of the 20 kb long <em>auts2b</em> genomic region on Chr15 (not to scale). Exons are shown as bars. Mutually exclusive first exons (1A and 1B) are in red. Arrows show TSSs identified either in this study (blue) or annotated based on RNASeq data (purple). Structures of RNASeq transcripts transcribed from alternative TSSs are shown. RNASeq transcript IDs are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS3.xslx">Table S3</a>. Exons 2L and 11L are 5′ extensions of the corresponding exons. Noncoding and coding exons are depicted as open and filled bars, respectively. (B) Schematic presentation of the structure of <em>auts2b</em> genes in genomes of different teleosts. Conserved exon 9 is depicted in red. (C) Schematic showing synteny between <em>auts2a</em> and <em>auts2b</em> genomic regions in the zebrafish genome. (D) Alignment of amino acid sequence coded by conserved exons 9 (Auts2b) and 14 (Auts2a) from different teleosts (coding exon is highlighted in gray). Amino acids that are highly conserved among Auts2 paralogs are shown in red. Chr, chromosome; ID, identifier; RNA sequencing; TSS, transcription start site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">We performed 5′-RACE with gene-specific primers designed in exons 3 and 4. We isolated a single 5′-RACE product and mapped it in exon 1, 588 nt downstream from the first nucleotides of RNASeq transcript RNASEQT00000017723 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS5.docx">Figure S5</a>), which is not annotated in the current Ensembl release, but was included in the previous genome build (Zv9, Ensembl release 75). With a forward primer, located 217 nt downstream from the first nucleotide of RNASEQT00000017723, we isolated full-length cDNA structurally corresponding to XM_001921276 and ENSDART00000161379. During the sequence analysis of clones, we also identified a novel transcript with intron retention (106 bp of intron 5) leading to a premature stop codon.</p><p id="p-43">Based on structure comparison between <em>auts2a</em> and <em>auts2b</em> genes, we assumed that, after whole-genome duplication, the 5′ genomic region of the ancestral gene corresponding to exons 1A–6 of <em>auts2a</em> was subsequently lost from the <em>auts2b</em> gene locus. Surprisingly, <em>in silico</em> analysis revealed the absence of the full-length <em>auts2b</em> gene in other teleosts whose genomes were sequenced (amazon molly, cod, fugu, medaka, platyfish, stickleback, tetraodon, and tilapia) with one exception. The genome of cave fish (<em>Astyanax mexicanus</em>) contains the <em>auts2b</em> gene but lacks the <em>auts2a</em> gene. It is not clear if <em>auts2a</em> gene is absent in the cave fish genome or it is simply not annotated in a current version of the cave fish whole-genome assembly due to incomplete sequencing. Protein sequence comparison and analysis of synteny strongly supports the identification of the cave fish <em>auts2</em> gene as an <em>auts2b</em> ortholog. We analyzed genomic regions in other teleost genomes that are syntenic to the <em>auts2b</em> locus in zebrafish genome. With the exception of the fugu genome, the other loci possess the predicted protein coding gene in the position of the <em>auts2b</em> gene. Protein sequence and gene structure analysis revealed a single conserved exon. This exon corresponds to exon 9 in the <em>auts2b</em> gene (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>). Exon 9 encodes 24 highly conserved amino acids within the Auts2 family domain (a single amino acid substitution in the Auts2a protein among teleosts and 100% identity in the Auts2b protein between zebrafish and cave fish) (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3D</a>). The rest of the amino acid sequence is highly diverse with low sequence similarity between analyzed proteins. At least in some teleosts this “gene” is transcribed as it is supported by RNASeq data from amazon molly, platyfish, and tilapia.</p></div><div id="sec-15" class="subsection"><h3>Expression of the auts2b gene during zebrafish development</h3><p id="p-44">RT-PCR analysis shows that <em>auts2b</em> is a zygotic gene in contrast to maternally supplied <em>auts2a</em> (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). At 90% of epiboly (9 hr), <em>auts2b</em> is expressed in the rostral neural plate, the prospective forebrain territory (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, B and B</a>’). At the 1-somite stage (10.3 hr), <em>auts2b</em> is expressed in the presumptive forebrain, presumptive hindbrain, and spinal cord (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, C and C</a>’). At the 3-somite stage (11 hr), <em>auts2b</em> is strongly expressed in the forebrain, hindbrain, and faintly in the midbrain (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4, D and D</a>’). During neural keel formation (12–14 hr), <em>auts2b</em> is expressed in the telencephalon, ventral diencephalon, hindbrain (more strongly in rhombomere 2 and less in rhombomeres 1 and 4), spinal cord, and somites (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4, E–F</a>”). The identity of rhombomeres was confirmed by using second probes, <em>egr2a</em> (<em>krox20</em>) to mark rhombomeres 3 and 5 (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4, G–G</a>”), and <em>otx2</em> to mark the midbrain territory (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4, H–H</a>”). Expression of <em>auts2b</em> was not detected in the midbrain (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4, F, F</a>’, and G). At 18 hr, expression of <em>auts2b</em> continued in the telencephalon, ventral diencephalon, hindbrain, spinal cord, and somites (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4, I and I</a>’). Expression in the ventral diencephalon at this stage was suggested by double <em>in situ</em> hybridization with <em>sim1a</em> as a marker for the ventral thalamus (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4, J and J</a>’). At 24 hr, <em>auts2b</em> is expressed in the telencephalon, ventral diencephalon, and hindbrain (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4, K and K</a>’), and in the roof plate of the spinal cord (<a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4K</a> inset). At 48 hr, <em>auts2b</em> is mainly expressed in the dorsal hindbrain and cerebellum (<a id="xref-fig-4-13" class="xref-fig" href="#F4">Figure 4, L and L</a>’), and in the roof plate (<a id="xref-fig-4-14" class="xref-fig" href="#F4">Figure 4L</a> inset).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression of auts2b mRNA during zebrafish development. (A) RT-PCR analysis of auts2b expression during development (amplicon size is 3.25 kb). Numbers shown below each lane indicate sample age in hours post fertilization. (B&#x2013;F&#x201D;, I, I&#x2019;, and K&#x2013;L&#x2019;): Whole-mount in situ hybridization analysis of auts2b transcript expression in wild-type embryos at different developmental stages: 9 hr (B and B&#x2019;), 10.3 hr (C and C&#x2019;), 11 hr (D and D&#x2019;), 12 hr (E&#x2013;E&#x201D;), 14 hr (F&#x2013;F&#x201D;), 18 hr (I and I&#x2019;), 24 hr (K and K&#x2019;), and 48 hr (L and L&#x2019;). (G&#x2013;H&#x201D;, J, and J&#x2019;) Double in situ hybridization with egr2a (krox20) (G&#x2013;G&#x201D;), otx2 (H&#x2013;H&#x201D;), and sim1a (J and J&#x2019;) as second probes. de, diencephalon; dth, dorsal thalamus; fb, forebrain; hb, hindbrain; mb, midbrain; mhb, midbrain-hindbrain boundary; r, rhombomere; rp, roof plate; RT-PCR, reverse transcription-polymerase chain reaction; s, somites; te, telencephalon; vde, ventral diencephalon; vth, ventral thalamus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1861302268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Expression of &lt;em&gt;auts2b&lt;/em&gt; mRNA during zebrafish development. (A) RT-PCR analysis of &lt;em&gt;auts2b&lt;/em&gt; expression during development (amplicon size is 3.25 kb). Numbers shown below each lane indicate sample age in hours post fertilization. (B&#x2013;F&#x201D;, I, I&#x2019;, and K&#x2013;L&#x2019;): Whole-mount &lt;em&gt;in situ&lt;/em&gt; hybridization analysis of &lt;em&gt;auts2b&lt;/em&gt; transcript expression in wild-type embryos at different developmental stages: 9 hr (B and B&#x2019;), 10.3 hr (C and C&#x2019;), 11 hr (D and D&#x2019;), 12 hr (E&#x2013;E&#x201D;), 14 hr (F&#x2013;F&#x201D;), 18 hr (I and I&#x2019;), 24 hr (K and K&#x2019;), and 48 hr (L and L&#x2019;). (G&#x2013;H&#x201D;, J, and J&#x2019;) Double &lt;em&gt;in situ&lt;/em&gt; hybridization with &lt;em&gt;egr2a&lt;/em&gt; (&lt;em&gt;krox20&lt;/em&gt;) (G&#x2013;G&#x201D;), &lt;em&gt;otx2&lt;/em&gt; (H&#x2013;H&#x201D;), and &lt;em&gt;sim1a&lt;/em&gt; (J and J&#x2019;) as second probes. de, diencephalon; dth, dorsal thalamus; fb, forebrain; hb, hindbrain; mb, midbrain; mhb, midbrain-hindbrain boundary; r, rhombomere; rp, roof plate; RT-PCR, reverse transcription-polymerase chain reaction; s, somites; te, telencephalon; vde, ventral diencephalon; vth, ventral thalamus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2577/F4.medium.gif" width="421" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/8/2577/F4.medium.gif" width="421" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486157" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-45" class="first-child">Expression of <em>auts2b</em> mRNA during zebrafish development. (A) RT-PCR analysis of <em>auts2b</em> expression during development (amplicon size is 3.25 kb). Numbers shown below each lane indicate sample age in hours post fertilization. (B–F”, I, I’, and K–L’): Whole-mount <em>in situ</em> hybridization analysis of <em>auts2b</em> transcript expression in wild-type embryos at different developmental stages: 9 hr (B and B’), 10.3 hr (C and C’), 11 hr (D and D’), 12 hr (E–E”), 14 hr (F–F”), 18 hr (I and I’), 24 hr (K and K’), and 48 hr (L and L’). (G–H”, J, and J’) Double <em>in situ</em> hybridization with <em>egr2a</em> (<em>krox20</em>) (G–G”), <em>otx2</em> (H–H”), and <em>sim1a</em> (J and J’) as second probes. de, diencephalon; dth, dorsal thalamus; fb, forebrain; hb, hindbrain; mb, midbrain; mhb, midbrain-hindbrain boundary; r, rhombomere; rp, roof plate; RT-PCR, reverse transcription-polymerase chain reaction; s, somites; te, telencephalon; vde, ventral diencephalon; vth, ventral thalamus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">We also performed <em>in situ</em> hybridization analysis of <em>auts2b</em> expression in the juvenile zebrafish brain (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). In the forebrain, expression of <em>auts2b</em> was detected in Dp and Dm zones of the dorsal telencephalic area, the anterior part of the PPa, the ventral entopeduncular nucleus (vENT), the lateral nucleus of ventral telencephalic area (Vl) (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, A and B</a>), and the ventral zone of the periventricular hypothalamus (Hv) (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). In the midbrain, <em>auts2b</em> expression was not detected (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5, C and D</a>). In the cerebellum, <em>auts2b</em> expression was observed in the granule cell layer of the corpus cerebelli (CCe) and eminentia granularis (EG) (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5E</a>). More caudally, <em>auts2b</em> is expressed in the caudal lobe of the cerebellum (LCa), a granular layer of EG (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5F</a>). In the hindbrain, expression of <em>auts2b</em> is detected in LVII and vagal (LX) lobes (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5G</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression of auts2b mRNA in juvenile brain. (A&#x2013;G) Transverse sections at the levels indicated by the vertical lines in (H) show auts2b mRNA expression in different regions of the juvenile brain: telencephalon (A and B), diencephalon and midbrain (C and D), and hindbrain (E&#x2013;G). Abbreviations used to label areas in brain sections can be found in Table S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1861302268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Expression of &lt;em&gt;auts2b&lt;/em&gt; mRNA in juvenile brain. (A&#x2013;G) Transverse sections at the levels indicated by the vertical lines in (H) show &lt;em&gt;auts2b&lt;/em&gt; mRNA expression in different regions of the juvenile brain: telencephalon (A and B), diencephalon and midbrain (C and D), and hindbrain (E&#x2013;G). Abbreviations used to label areas in brain sections can be found in Table S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2577/F5.medium.gif" width="440" height="192"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/8/2577/F5.medium.gif" width="440" height="192"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485916" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-47" class="first-child">Expression of <em>auts2b</em> mRNA in juvenile brain. (A–G) Transverse sections at the levels indicated by the vertical lines in (H) show <em>auts2b</em> mRNA expression in different regions of the juvenile brain: telencephalon (A and B), diencephalon and midbrain (C and D), and hindbrain (E–G). Abbreviations used to label areas in brain sections can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS1.xslx">Table S1</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Identification of the 5′-ends of the zebrafish fbrsl1 gene and isolation of novel fbrsl1 transcripts</h3><p id="p-48">In contrast to the <em>auts2a</em> gene, the RefSeq <em>fbrsl1</em> gene model (NCBI Gene ID: 557358) lacks a completely annotated 5′-end. Nevertheless, similar to the <em>auts2a</em> gene locus, several RNASeq transcripts coding for short polypeptides or representing noncoding RNAs are annotated in the <em>fbrsl1</em> gene locus (summarized in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>, for more details see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS4.xslx">Table S4</a> and the Ensembl browser). To define the 5′-end of the <em>fbrsl1</em> gene, we performed 5′-RACE analysis with gene-specific primers designed to exons 4, 5, and 7. We isolated 11 different 5′-RACE products and annotated two novel exons associated with them (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS6.docx">Figure S6, A–C</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene structure and transcript complexity in the zebrafish fbrsl1 gene locus. (A) Schematic presentation of the 580 kb long fbrsl1 genomic region on chromosome 5 (not to scale). Exons are shown as bars. Mutually exclusive first exons (1A and 1B) are in red. Exons 9a and 9b are mutually exclusive internal exons. Arrows show TSSs identified either in this study (blue) or annotated based on RNASeq data (purple). TSS2 is found inside of the coding part of exon 1A. (B) Schematic presentation of the structure of fbrsl1 transcripts identified in this study. Noncoding and coding exons are depicted as open and filled bars, respectively. Structure of RNASeq transcripts transcribed from alternative TSSs are shown. RNASeq transcript IDs are shown in Table S4. Exons 2L, 6L, 10L, 11L, 15L, and 17L are 5&#x2032; extensions of the corresponding exons. (C) RT-PCR analysis of fbrsl1 isoforms expression during zebrafish development. Primers amplifying the full-length transcripts were used for analysis. The second band (pointed by arrow) is a reverse transcriptase template switching artifact. The 0 hr lane for TSS4 was taken from a different gel. (D) Partial DNA sequence of coding exons undergoing alternative splicing. Position of alternative 5&#x2032; donor splice sites is highlighted in orange. Constitutive splice sites are underlined. Exonic sequence is shown in upper case. Deleted amino acids are shown in red. ID, identifier; RNASeq, RNA sequencing; RT-PCR, reverse transcription-polymerase chain reaction; TSS, transcription start site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1861302268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gene structure and transcript complexity in the zebrafish &lt;em&gt;fbrsl1&lt;/em&gt; gene locus. (A) Schematic presentation of the 580 kb long &lt;em&gt;fbrsl1&lt;/em&gt; genomic region on chromosome 5 (not to scale). Exons are shown as bars. Mutually exclusive first exons (1A and 1B) are in red. Exons 9a and 9b are mutually exclusive internal exons. Arrows show TSSs identified either in this study (blue) or annotated based on RNASeq data (purple). TSS2 is found inside of the coding part of exon 1A. (B) Schematic presentation of the structure of &lt;em&gt;fbrsl1&lt;/em&gt; transcripts identified in this study. Noncoding and coding exons are depicted as open and filled bars, respectively. Structure of RNASeq transcripts transcribed from alternative TSSs are shown. RNASeq transcript IDs are shown in Table S4. Exons 2L, 6L, 10L, 11L, 15L, and 17L are 5&#x2032; extensions of the corresponding exons. (C) RT-PCR analysis of &lt;em&gt;fbrsl1&lt;/em&gt; isoforms expression during zebrafish development. Primers amplifying the full-length transcripts were used for analysis. The second band (pointed by arrow) is a reverse transcriptase template switching artifact. The 0 hr lane for TSS4 was taken from a different gel. (D) Partial DNA sequence of coding exons undergoing alternative splicing. Position of alternative 5&#x2032; donor splice sites is highlighted in orange. Constitutive splice sites are underlined. Exonic sequence is shown in upper case. Deleted amino acids are shown in red. ID, identifier; RNASeq, RNA sequencing; RT-PCR, reverse transcription-polymerase chain reaction; TSS, transcription start site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2577/F6.medium.gif" width="440" height="335"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/8/2577/F6.medium.gif" width="440" height="335"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486147" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-49" class="first-child">Gene structure and transcript complexity in the zebrafish <em>fbrsl1</em> gene locus. (A) Schematic presentation of the 580 kb long <em>fbrsl1</em> genomic region on chromosome 5 (not to scale). Exons are shown as bars. Mutually exclusive first exons (1A and 1B) are in red. Exons 9a and 9b are mutually exclusive internal exons. Arrows show TSSs identified either in this study (blue) or annotated based on RNASeq data (purple). TSS2 is found inside of the coding part of exon 1A. (B) Schematic presentation of the structure of <em>fbrsl1</em> transcripts identified in this study. Noncoding and coding exons are depicted as open and filled bars, respectively. Structure of RNASeq transcripts transcribed from alternative TSSs are shown. RNASeq transcript IDs are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS4.xslx">Table S4</a>. Exons 2L, 6L, 10L, 11L, 15L, and 17L are 5′ extensions of the corresponding exons. (C) RT-PCR analysis of <em>fbrsl1</em> isoforms expression during zebrafish development. Primers amplifying the full-length transcripts were used for analysis. The second band (pointed by arrow) is a reverse transcriptase template switching artifact. The 0 hr lane for TSS4 was taken from a different gel. (D) Partial DNA sequence of coding exons undergoing alternative splicing. Position of alternative 5′ donor splice sites is highlighted in orange. Constitutive splice sites are underlined. Exonic sequence is shown in upper case. Deleted amino acids are shown in red. ID, identifier; RNASeq, RNA sequencing; RT-PCR, reverse transcription-polymerase chain reaction; TSS, transcription start site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">Exon 1A contains 850 nt of 5′-UTR and 273 nt of CDS (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS6.docx">Figure S6A</a>). Using BLAST, we could only map 1–548 nt to contig CU929333.5 (chromosome 5: 16,669,728–16,670,275 in Ensembl genomic coordinates), 135,534 nt upstream from the first nucleotide of RefSeq transcript XM_009301264. However, when we searched against the NCBI whole-genome shotgun database, this exon could be completely mapped to Zebrafish Genome Assembly WGS32 contig_000055998 (GenBank: CZQB01055999.1) at position 9394–10,516 (contig length is 54,890 bp). This contig is overlapped by contigs CABZ01050979.1 and CU929333.5 from Ensembl, although the gaps in sequence still remain (overlapping nucleotides 1–9395 and 9964–54,890).</p><p id="p-51">From 5′-RACE experiments, six 5′-ends were mapped to the untranslated region of exon 1A and one 5′-end into the translated region of exon 1A, 214 nt downstream from ATG (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS6.docx">Figure S6A</a>). Mapped TSSs were clustered in two groups (five 5′-ends in one group and two 5′-ends in the other) separated by 593 bp. Based on the idea that two independent TSS clusters associated with distinct PAPs are separated with &gt; 500-bp intervals (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kimura <em>et al.</em> 2006</a>), we predict that these two groups of TSSs are linked to distinct PAPs, likely overlapping, and can be recognized as separate TSSs: TSS1 and TSS2 (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>).</p><p id="p-52">Using a forward primer designed in close proximity to the most distant 5′-end (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS6.docx">Figure S6A</a>), we could isolate two transcripts, both beginning with exon 1A and including 19 exons. The difference between them is that the two isoforms, <em>fbrsl1-i1a</em> and <em>fbrsl1-i1b</em>, contain mutually exclusive exons 9a and 9b, respectively (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>). Transcripts <em>fbrsl1-i1a</em> and <em>fbrsl1-i1b</em> code protein isoforms of 1215 and 1249 amino acid residues, respectively (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS6.docx">Figure S6</a>). The RefSeq transcript XM_009301264, although lacking exon 1A, apparently represents an alternative splice variant of <em>fbrsl1-i1b</em> with exon 9b being spliced at an alternative 5′ donor splice site leading to the exclusion of 21 nt from the spliced mRNA and, hence, an in-frame deletion of seven amino acids in the protein (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6D</a>). Transcript XM_009301264 also has an alternative exon 16, which is spliced at an alternative 5′ donor splice site leading to the exclusion of 6 nt from mRNA and therefore an in-frame deletion of two amino acids (glycine and lysine) and substitution of tyrosine with aspartic acid (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6D</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS6.docx">Figure S6</a>). Notably, splicing at the alternative 5′ donor splice site of exon 9b is exactly the same as that in exon 9 of the XM_017358059 transcript from the <em>auts2a</em> gene locus (with deletion of identical amino acids).</p><p id="p-53">The second novel exon, noncoding exon 1B identified in intron 5, is spliced to exon 6 (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6A</a>). We mapped three 5′-ends of different lengths to this exon (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS6.docx">Figure S6C</a>). In a previous zebrafish genome build (Zv9, Ensembl release 75), RNASeq transcript RNASEQT00000006457 was annotated with exon 1B and its first nucleotide was 145 nt upstream from the TSS mapped in this study (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS6.docx">Figure S6C</a>). In the current zebrafish genome assembly (GRC10) this transcript is not included. However, we could isolate cDNA <em>fbrsl1-i2b</em> corresponding to RNASEQT00000006457 (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6B</a>). Transcript <em>fbrsl1-i2b</em> includes mutually exclusive exon 9b and codes for a short protein isoform. The ORF of this short transcript is identical to that of the long isoform except that it lacks 405 N-terminal amino acids present in the long isoform, and is predicted to encode a polypeptide of 842 amino acid residues (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS7.pptx">Figure S7</a>). Ensembl transcript ENSDART00000081064 and RNASeq transcript RNASEQT00000054612, which lack the annotated exon 1B, apparently represent alternative splice isoforms of the <em>fbrsl1-i2a</em> transcript with mutually exclusive exon 9a and encode a protein of 810 amino acid residues (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS7.pptx">Figure S7</a>).</p><p id="p-54">With a similar approach as was used for the <em>auts2a</em> gene locus, by using RNASeq data we defined additional 10 alternative first exons representing 5′ extensions of annotated exons: exon 2L (5′ extension of exon 2), exon 6L, exon 7L, exon 8L, exon 10L, exon 11L, exon 15L, exon 16L, exon 17L, and exon 18L (<a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6, A and B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS6.docx">Figure S6</a>). Several RNASeq transcripts have the first nucleotides annotated inside of mutually exclusive exons 9a and 9b, so the transcripts begin with ATG (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS4.xslx">Table S4</a> for details). We did not consider such candidates as putative TSSs because it could be accounted for by truncated cDNAs. However, we cannot rule out that they are genuine TSSs. In our 5′-RACE experiments, a single 5′-end was mapped inside of exon 2, 66 nt downstream from the first nucleotide of transcript RNASEQT00000061379 (TSS3 in <a id="xref-fig-6-10" class="xref-fig" href="#F6">Figure 6A</a>, see also <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS6.docx">Figure S6B</a> for details). In this case, it is likely that the 5′-RACE product represents the truncated form transcribed from TSS3.</p><p id="p-55">RT-PCR analysis of <em>fbrsl1</em> expression during zebrafish development revealed that both isoforms are maternally supplied (<a id="xref-fig-6-11" class="xref-fig" href="#F6">Figure 6C</a>). Isoforms <em>fbrsl1-i1</em> (TSS1) and <em>fbrsl1-i2</em> (TSS4) are detected again from 9 to 6 hr, respectively (weak signal at 9 hr for TSS1, <a id="xref-fig-6-12" class="xref-fig" href="#F6">Figure 6C</a>). Direct sequencing of PCR products revealed coamplification of transcripts with and without exons 4 or 17 (skipped exons), and coamplification of transcripts with mutually exclusive exons 9a and 9b.</p></div><div id="sec-17" class="subsection"><h3>Expression of the fbrsl1 gene during zebrafish development</h3><p id="p-56">We conducted WISH to examine the spatial expression of the <em>fbrsl1</em> gene during zebrafish development. Since we identified two populations of <em>fbrsl1</em> isoforms, <em>fbrsl1-i1</em> and <em>fbrsl1-i2</em>, that are generated from TSS1 and TSS4, respectively, we asked if these isoforms are expressed differentially during development. We used a similar approach as for <em>auts2a</em> isoforms. We designed two isoform-specific riboprobes that recognize alternative mutually exclusive exons 1A and 1B in <em>fbrsl1</em> transcripts (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS8.jpeg">Figure S8A</a>). Analysis of <em>in situ</em> hybridization data showed that although expression was quite similar between the two probes, the isoform transcribed from TSS4 was expressed in hypothalamus and ventral diencephalon, which was not detected for the TSS1 probe, and more strongly in the telencephalon (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS8.jpeg">Figure S8B</a>). For further analysis of <em>fbrsl1</em> expression during development, we used a probe designed against full-length isoform <em>fbrsl1-i2b</em> containing exon 9b (for probe details see <em>Materials and Methods</em>).</p><p id="p-57">Despite the detection of the <em>fbrsl1-i2</em> transcript at 6 hr by RT-PCR (TSS4 in <a id="xref-fig-6-13" class="xref-fig" href="#F6">Figure 6C</a>), weak expression in the neural plate becomes evident only at the 1-somite stage (10.3 hr) (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7, A and A</a>’). At the 3-somite stage (11 hr), <em>fbrsl1-i2b</em> is expressed in the presumptive diencephalon, midbrain, and rhombomere 4 (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7, B and B</a>’). During neural keel formation (12–14 hr), <em>fbrsl1-i2b</em> expression continues in the forebrain, midbrain, and rhombomere 4. Expression in rhombomere 4 was confirmed using double <em>in situ</em> hybridization with <em>egr2a</em> (<em>krox20</em>) as a second probe to mark rhombomeres 3 and 5 (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7, E and E</a>’). <em>Fbrsl1-i2b</em> is also expressed in the spinal cord, cranial ganglia, and Kupffer’s vesicle (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7, C–D</a>”). At 18 hr, expression in the brain is broad with the strongest signals in the telencephalon and hypothalamus (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7, I–I</a>”), and expression continues in somites (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7F</a>). At 24 hr, <em>fbrsl1-i2b</em> is strongly expressed in the telencephalon, hypothalamus, dorsal thalamus (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7, J–J</a>”), and somites (<a id="xref-fig-7-8" class="xref-fig" href="#F7">Figure 7G</a>). At 48 hr, <em>fbrsl1-i2b</em> is strongly expressed in the dorsal forebrain, optic tectum, cerebellum, dorsal hindbrain, and spinal cord (<a id="xref-fig-7-9" class="xref-fig" href="#F7">Figure 7, H and K–K</a>”).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression of fbrsl1 mRNA during zebrafish development. Whole-mount in situ hybridization analysis of fbrsl1-i2b transcript expression in wild-type embryos at different developmental stages: 10.3 hr (A and A&#x2019;), 11 hr (B and B&#x2019;), 12 hr (C&#x2013;C&#x201D;), 14 hr (D&#x2013;D&#x201D;), 18 hr (F, I, and I&#x2019;), 24 hr (G, J, and J&#x201D;), and 48 hr (H, K, and K&#x2019;). (E and E&#x2019;) Double in situ hybridization with egr2a (krox20) as a second probe. (I, I&#x2019;&#x2013;K, and K&#x2019;). Flat mount prep of embryonic brain with schematic presentation of brain subdivision at these developmental stages (I&#x201D;&#x2013;K&#x201D;). Eyes were removed in (I&#x2019;&#x2013;K&#x2019;). c, cerebellum; cg, cranial ganglia; de, diencephalon; dt, dorsal thalamus; e, epiphysis; fb, forebrain; hb, hindbrain; hy, hypothalamus; kv, Kupffer&#x2019;s vesicle; mb, midbrain; pt, posterior tuberculum; r, rhombomere; t, telencephalon; tc, tectum; tg, tegmentum; vt, ventral thalamus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1861302268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Expression of &lt;em&gt;fbrsl1&lt;/em&gt; mRNA during zebrafish development. Whole-mount &lt;em&gt;in situ&lt;/em&gt; hybridization analysis of &lt;em&gt;fbrsl1-i2b&lt;/em&gt; transcript expression in wild-type embryos at different developmental stages: 10.3 hr (A and A&#x2019;), 11 hr (B and B&#x2019;), 12 hr (C&#x2013;C&#x201D;), 14 hr (D&#x2013;D&#x201D;), 18 hr (F, I, and I&#x2019;), 24 hr (G, J, and J&#x201D;), and 48 hr (H, K, and K&#x2019;). (E and E&#x2019;) Double &lt;em&gt;in situ&lt;/em&gt; hybridization with &lt;em&gt;egr2a&lt;/em&gt; (&lt;em&gt;krox20&lt;/em&gt;) as a second probe. (I, I&#x2019;&#x2013;K, and K&#x2019;). Flat mount prep of embryonic brain with schematic presentation of brain subdivision at these developmental stages (I&#x201D;&#x2013;K&#x201D;). Eyes were removed in (I&#x2019;&#x2013;K&#x2019;). c, cerebellum; cg, cranial ganglia; de, diencephalon; dt, dorsal thalamus; e, epiphysis; fb, forebrain; hb, hindbrain; hy, hypothalamus; kv, Kupffer&#x2019;s vesicle; mb, midbrain; pt, posterior tuberculum; r, rhombomere; t, telencephalon; tc, tectum; tg, tegmentum; vt, ventral thalamus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2577/F7.medium.gif" width="427" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/8/2577/F7.medium.gif" width="427" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486041" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-58" class="first-child">Expression of <em>fbrsl1</em> mRNA during zebrafish development. Whole-mount <em>in situ</em> hybridization analysis of <em>fbrsl1-i2b</em> transcript expression in wild-type embryos at different developmental stages: 10.3 hr (A and A’), 11 hr (B and B’), 12 hr (C–C”), 14 hr (D–D”), 18 hr (F, I, and I’), 24 hr (G, J, and J”), and 48 hr (H, K, and K’). (E and E’) Double <em>in situ</em> hybridization with <em>egr2a</em> (<em>krox20</em>) as a second probe. (I, I’–K, and K’). Flat mount prep of embryonic brain with schematic presentation of brain subdivision at these developmental stages (I”–K”). Eyes were removed in (I’–K’). c, cerebellum; cg, cranial ganglia; de, diencephalon; dt, dorsal thalamus; e, epiphysis; fb, forebrain; hb, hindbrain; hy, hypothalamus; kv, Kupffer’s vesicle; mb, midbrain; pt, posterior tuberculum; r, rhombomere; t, telencephalon; tc, tectum; tg, tegmentum; vt, ventral thalamus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">We also performed <em>in situ</em> hybridization analysis of <em>fbrsl1-i2b</em> expression in the juvenile zebrafish brain (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). In the forebrain, expression of <em>fbrsl1-i2b</em> is detected in the dorsal telencephalic area (D), the ventral nucleus of ventral telencephalic area (Vv), the ventral (Hv), caudal (Hc), and dorsal (Hd) zones of the periventricular hypothalamus, the mammillary body (CM), and the posterior part of the parvocellular preoptic nucleus (PPp) (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8, A–D</a>). In the midbrain, <em>fbrsl1-i2b</em> is strongly expressed in the PGZ and also detected in the medial preglomerular nucleus (PGm) and dorsomedial optical tract (DOT) (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8, B–D</a>). In the cerebellum, <em>fbrsl1-i2b</em> is expressed in the PCL, similar to <em>auts2a</em>, and in the LCa (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8E</a>). In the hindbrain, <em>fbrsl1-i2b</em> is expressed in the LVII and CON (<a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8F</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression of fbrsl1 mRNA in juvenile brain. (A&#x2013;F) Transverse sections at the levels indicated by the vertical lines in (G) show fbrsl1-i2b mRNA expression in different regions of the juvenile brain: telencephalon (A), diencephalon and midbrain (B and C), and hindbrain (D&#x2013;F). Abbreviations used to label areas in brain sections can be found in Table S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1861302268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Expression of &lt;em&gt;fbrsl1&lt;/em&gt; mRNA in juvenile brain. (A&#x2013;F) Transverse sections at the levels indicated by the vertical lines in (G) show &lt;em&gt;fbrsl1-i2b&lt;/em&gt; mRNA expression in different regions of the juvenile brain: telencephalon (A), diencephalon and midbrain (B and C), and hindbrain (D&#x2013;F). Abbreviations used to label areas in brain sections can be found in Table S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2577/F8.medium.gif" width="440" height="235"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/8/2577/F8.medium.gif" width="440" height="235"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485975" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-60" class="first-child">Expression of <em>fbrsl1</em> mRNA in juvenile brain. (A–F) Transverse sections at the levels indicated by the vertical lines in (G) show <em>fbrsl1-i2b</em> mRNA expression in different regions of the juvenile brain: telencephalon (A), diencephalon and midbrain (B and C), and hindbrain (D–F). Abbreviations used to label areas in brain sections can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS1.xslx">Table S1</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>In silico identification of transcripts in the fbrs gene locus and expression of fbrs during development</h3><p id="p-61">The current RefSeq <em>fbrs</em> gene model (NCBI Gene ID: 100535921) defines 19 exons present in transcript XM_003199613 (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9A</a>). Three RefSeq predicted transcripts, XM_005156304, XM_005156306, and XM_017358534, represent alternatively spliced mRNAs that are transcribed from the same TSS1 (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9B</a>). Alternative splicing occurs at the 3′ acceptor splice site of exon 12 (XM_005156304), which leads to the exclusion of 15 nt from the spliced mRNA and results in an in-frame deletion of five amino acids in the protein (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9D</a>). Skipped exons are found in transcripts XM_005156306 (exons 16 and 17 are spliced out together), XM_017358534 (skipped exon 16), and in Ensembl transcript ENSDART00000153054 (skipped exon 5). Transcript ENSDART00000153054 also has an alternatively spliced exon 12 similar to that in XM_005156304.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene structure, transcript complexity, and expression of the fbrs gene during zebrafish development and in the juvenile brain. (A) Schematic presentation of the 20.5 kb long fbrs genomic region on chromosome 12 (not to scale). Exons are shown as bars. Arrows show TSSs identified either in this study (blue) or annotated based on RNASeq data (purple). (B) Schematic presentation of structure of fbrs transcript identified in this study. Structures of RNASeq transcripts transcribed from alternative TSSs are shown in purple. RNASeq transcript IDs are shown in Table S5. Noncoding and coding exons are depicted as open and filled bars, respectively. (C) RT-PCR analysis of fbrs expression during zebrafish development. Primers amplifying the full-length transcripts were used for analysis. The second faint bands are reverse transcriptase template switching artifacts. (D) Partial DNA sequence of alternatively spliced exon 12. Position of alternative 3&#x2032; acceptor splice site is highlighted in orange. Constitutive splice site is underlined. Deleted amino acids are shown in red. (E&#x2013;J) Whole mount in situ hybridization analysis of fbrs transcript expression in wild-type embryos at different developmental stages, from 1-somite (10.3 hr) to long-pec (48 hr) stages. Arrows point to expression in Kupffer&#x2019;s vesicle. (L&#x2013;O) Expression of fbrs mRNA in juvenile brain. Transverse sections were made at the levels indicated by the vertical lines in (K). Abbreviations used to label areas in brain sections can be found in Table S1. ID, identifier; RNASeq, RNA sequencing; RT-PCR, reverse transcription-polymerase chain reaction; TSS, transcription start site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1861302268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gene structure, transcript complexity, and expression of the &lt;em&gt;fbrs&lt;/em&gt; gene during zebrafish development and in the juvenile brain. (A) Schematic presentation of the 20.5 kb long &lt;em&gt;fbrs&lt;/em&gt; genomic region on chromosome 12 (not to scale). Exons are shown as bars. Arrows show TSSs identified either in this study (blue) or annotated based on RNASeq data (purple). (B) Schematic presentation of structure of &lt;em&gt;fbrs&lt;/em&gt; transcript identified in this study. Structures of RNASeq transcripts transcribed from alternative TSSs are shown in purple. RNASeq transcript IDs are shown in Table S5. Noncoding and coding exons are depicted as open and filled bars, respectively. (C) RT-PCR analysis of &lt;em&gt;fbrs&lt;/em&gt; expression during zebrafish development. Primers amplifying the full-length transcripts were used for analysis. The second faint bands are reverse transcriptase template switching artifacts. (D) Partial DNA sequence of alternatively spliced exon 12. Position of alternative 3&#x2032; acceptor splice site is highlighted in orange. Constitutive splice site is underlined. Deleted amino acids are shown in red. (E&#x2013;J) Whole mount &lt;em&gt;in situ&lt;/em&gt; hybridization analysis of &lt;em&gt;fbrs&lt;/em&gt; transcript expression in wild-type embryos at different developmental stages, from 1-somite (10.3 hr) to long-pec (48 hr) stages. Arrows point to expression in Kupffer&#x2019;s vesicle. (L&#x2013;O) Expression of &lt;em&gt;fbrs&lt;/em&gt; mRNA in juvenile brain. Transverse sections were made at the levels indicated by the vertical lines in (K). Abbreviations used to label areas in brain sections can be found in Table S1. ID, identifier; RNASeq, RNA sequencing; RT-PCR, reverse transcription-polymerase chain reaction; TSS, transcription start site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2577/F9.medium.gif" width="440" height="433"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/7/8/2577/F9.medium.gif" width="440" height="433"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2577/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485897" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-62" class="first-child">Gene structure, transcript complexity, and expression of the <em>fbrs</em> gene during zebrafish development and in the juvenile brain. (A) Schematic presentation of the 20.5 kb long <em>fbrs</em> genomic region on chromosome 12 (not to scale). Exons are shown as bars. Arrows show TSSs identified either in this study (blue) or annotated based on RNASeq data (purple). (B) Schematic presentation of structure of <em>fbrs</em> transcript identified in this study. Structures of RNASeq transcripts transcribed from alternative TSSs are shown in purple. RNASeq transcript IDs are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS5.xslx">Table S5</a>. Noncoding and coding exons are depicted as open and filled bars, respectively. (C) RT-PCR analysis of <em>fbrs</em> expression during zebrafish development. Primers amplifying the full-length transcripts were used for analysis. The second faint bands are reverse transcriptase template switching artifacts. (D) Partial DNA sequence of alternatively spliced exon 12. Position of alternative 3′ acceptor splice site is highlighted in orange. Constitutive splice site is underlined. Deleted amino acids are shown in red. (E–J) Whole mount <em>in situ</em> hybridization analysis of <em>fbrs</em> transcript expression in wild-type embryos at different developmental stages, from 1-somite (10.3 hr) to long-pec (48 hr) stages. Arrows point to expression in Kupffer’s vesicle. (L–O) Expression of <em>fbrs</em> mRNA in juvenile brain. Transverse sections were made at the levels indicated by the vertical lines in (K). Abbreviations used to label areas in brain sections can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS1.xslx">Table S1</a>. ID, identifier; RNASeq, RNA sequencing; RT-PCR, reverse transcription-polymerase chain reaction; TSS, transcription start site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-63">We performed 5′-RACE analysis with gene-specific primers designed against exon 2. We could identify only a single TSS, which was mapped 55 nt upstream from the first nucleotide of RefSeq transcripts (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS9.docx">Figure S9</a>). Using a forward primer, designed in close proximity to TSS1, we isolated cDNA corresponding to XM_017358534. From RNASeq data, two additional alternative first exons, representing 5′ extensions of exon 2 (2L, TSS2) and exon 18 (18L, TSS3), were predicted (<a id="xref-fig-9-4" class="xref-fig" href="#F9">Figure 9B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS9.docx">Figure S9</a>). Analysis of RNASeq data also revealed transcription from TSS1 that generates isoforms with alternative 3′-ends (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/TableS5.xslx">Table S5</a>).</p><p id="p-64">RT-PCR and <em>in situ</em> analysis revealed that <em>fbrs</em> mRNA is maternally supplied and ubiquitously expressed in neural tissues during development (<a id="xref-fig-9-5" class="xref-fig" href="#F9">Figure 9, C and E–J</a>). It is also expressed in Kupffer’s vesicle (<a id="xref-fig-9-6" class="xref-fig" href="#F9">Figure 9, F, G, and G</a>”) and later in the eyes and somites (<a id="xref-fig-9-7" class="xref-fig" href="#F9">Figure 9, H–J</a>). In the juvenile brain, expression of <em>fbrs</em> is detected in the Hv, Hc, and Hd zones of the periventricular hypothalamus, the PGZ, and the granular layer of the CCe (<a id="xref-fig-9-8" class="xref-fig" href="#F9">Figure 9, K–O</a>).</p></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-65">Eukaryotes employ a range of mechanisms to generate multiple mRNA isoforms. These mechanisms include usage of alternative TSSs, alternative splice sites, and polyadenylation. Here, we show that, in zebrafish, the <em>auts2</em> gene family has four paralogs: <em>auts2a</em>, <em>auts2b</em>, <em>fbrsl1</em>, and <em>fbrs</em>. All four paralogs exhibit multiple TSSs and alternative splicing, with <em>auts2a</em> and <em>fbrsl1</em> giving rise to more mRNA diversity than <em>auts2b</em> or <em>fbrs</em>. However, whether all mRNA isoforms are translated into proteins cannot be easily determined and the interesting question is whether there is a biological function underlying such complexity. Our study was limited to the usage of 5′-RACE analysis followed by RT-PCR and cloning. We have not examined the presence of alternative polyadenylation sites, although available RNASeq data support the presence of RNAs with alternative 3′-ends, with some of them encoding short polypeptides or representing ncRNAs.</p><div id="sec-20" class="subsection"><h3>Transcriptional complexity in the auts2a gene locus</h3><p id="p-66">Among <em>auts2</em> paralogs, the highest level of transcriptional complexity was found in <em>auts2a</em>. Here, the complexity is achieved mainly through the usage of alternative promoters rather than through alternative splicing. In this case, transcription from alternative promoters generates <em>auts2a</em> mRNAs that differ in 5′-UTRs and encode the N-terminally truncated protein isoforms, while the C-terminal portion of Auts2a protein remains the same. Differences in 5′-UTRs may have an impact on translation efficiency since cap-dependent ribosomal scanning is severely hampered in 5′-UTRs containing upstream AUGs (uAUGs), uORFs, and secondary structures (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Araujo <em>et al.</em> 2012</a>).</p><p id="p-67">The N-terminally truncated protein isoforms may differ in intracellular localization and/or trafficking. In the human AUTS2 protein, several regions were predicted that may be functionally important: the putative NLS in the N-terminal part, two proline-rich regions (PR1 and PR2), and the PY motif in the middle of the protein (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Sultana <em>et al.</em> 2002</a>; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS2.pptx">Figure S2</a> for zebrafish Auts2a protein annotation). The relevance of the predicted NLS is not clear, since an AUTS2 protein isoform lacking NLS was localized exclusively in the nucleus (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hori <em>et al.</em> 2014</a>). In our experiments, we also observed nuclear localization after transfection of the HEK293T cell line with either the N-terminal end of Auts2a or an Auts2a-i3 isoform that lacks 467 N-terminal amino acids that are present in the long protein isoform (data not shown). It is clear that Auts2 protein may contain another NLS, different from the one predicted in the N-terminal part. Alternatively, the nuclear localization of Auts2 protein might result from its interaction with another nuclear protein.</p><p id="p-68">The PR1 region was shown to be important for the regulation of actin remodelling, and for neuronal migration and neuritogenesis in particular (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Hori <em>et al.</em> 2014</a>). The C-terminal portion of AUTS2, comprising the Auts2 family domain and His repeats, is important for mediating transcriptional activation (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gao <em>et al.</em> 2014</a>) and deletion of this part in the human causes severe phenotypes (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beunders <em>et al.</em> 2015</a>). His repeats have been shown to be associated with protein localization at nuclear speckles (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Salichs <em>et al.</em> 2009</a>). The zebrafish Auts2a protein does not contain His repeats at the C-terminal end.</p><p id="p-69">Splicing of <em>auts2a</em> pre-mRNA at tandem splice acceptors with a NAGNAG motif leads to very minor changes in protein sequence (deletion of a single amino acid). It has been proposed that splicing between the intron-proximal and intron-distal AG is achieved by a competition mechanism (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hiller <em>et al.</em> 2004</a>). In many cases, the selection of such AGs can be highly regulated under specific spatiotemporal conditions or external stimuli. Tandem splicing of mRNAs can lead to the production of functionally different proteins, for example, the <em>SCN5A</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Makielski <em>et al.</em> 2003</a>) and <em>pou5f3</em> genes (formerly <em>pou2</em>) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Takeda <em>et al.</em> 1994</a>). In the first example, cells expressing voltage-dependent sodium channel α-subunit protein SCN5A, which contains Q1077, showed a reduced inward sodium current in comparison to SCN5A variants lacking Q1077 (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Makielski <em>et al.</em> 2003</a>). In the second example, the usage of either the distal or proximal 3′ splice acceptor sites leads to the generation of two proteins that possess distinct functions; one is a transcription factor, while the other is a non-DNA-binding protein (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Takeda <em>et al.</em> 1994</a>). In the case of <em>auts2a</em>, alternative splicing at these particular exons is evolutionarily conserved; the positions and identity of deleted amino acids are conserved among either all vertebrates (exons 3 and 8) or only among ray-finned fish, sharks, and coelacanth (exon 15), supporting the functional significance of such minor changes in protein sequence. Splicing at an alternative 5′ donor site of exon 9 leading to an in-frame deletion of seven amino acids is highly conserved among jawed vertebrates. Moreover, splicing at this exon is also conserved between <em>auts2a</em> and <em>fbrsl1</em> genes: identical amino acids that are present in the full-length Fbrsl1 protein are also removed during alternative splicing of the <em>fbrsl1</em> pre-mRNA (<a id="xref-fig-1-26" class="xref-fig" href="#F1">Figure 1D</a> and <a id="xref-fig-6-14" class="xref-fig" href="#F6">Figure 6D</a>).</p></div><div id="sec-21" class="subsection"><h3>Transcriptional complexity in the fbrsl1 gene locus</h3><p id="p-70">Transcriptional complexity in the <em>fbrsl1</em> gene is achieved through the usage of both alternative promoters and alternative splicing. Three modes of alternative splicing were found in the <em>fbrsl1</em> gene: exon skipping (exon 17), splicing at alternative 5′ donor splice sites of exons 9b and 16, and mutually exclusive exons 9a and 9b. Interestingly, from RNASeq data, two potential TSSs are annotated inside of exons 9a and 9b. Although the importance of exonic promoters is highly speculative, exonic TSSs might have some relationship to so-called exonic splicing enhancers influencing the recruitment of SR proteins (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Carninci <em>et al.</em> 2006</a>). Some have speculated that RNAs generated from these TSSs may regulate the decision of which exon will be used in the protein-coding transcript. Differential usage of these mutually exclusive exons leads to protein isoforms that differ substantially in sequence. Amino acid sequence coded by exon 9b is highly conserved even between paralogs. Exclusion of exon 9b may have a drastic effect on protein function. Currently, there is no experimental evidence for a functional role of FBRSL1, except that in RNA-bound proteome analysis FBRSL1 was identified as a candidate RNA-binding protein (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baltz <em>et al.</em> 2012</a>). However, the identity of the particular FBRSL1 protein isoform involved was not reported.</p></div><div id="sec-22" class="subsection"><h3>Implications of transcriptional complexity for disease</h3><p id="p-71">Although transcriptional complexity allows greater flexibility and control in complex systems, it is more likely to be misregulated, particularly in systems that depend heavily on alternative splicing. Aberrant promoter usage has been associated with several human cancers including colon cancer, ovarian cancer, and neuroblastomas (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Landry <em>et al.</em> 2003</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Davuluri <em>et al.</em> 2008</a>), suggesting that genes with alternative promoters are more likely to be associated with disease (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Davuluri <em>et al.</em> 2008</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Liu 2010</a>). <em>BDNF</em> utilizes multiple promoters in a tissue-specific manner, and promoter usage is altered after kainite-induced seizures (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Timmusk <em>et al.</em> 1993</a>). A SNP in the promoter region of the 5-HT2A receptor affects promoter activity and is associated with psychiatric disorders (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Parsons <em>et al.</em> 2004</a>). Several SNPs, identified within the promoter region of the <em>Kalirin</em> gene are associated with coronary artery disease (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wang <em>et al.</em> 2007</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Horne <em>et al.</em> 2009</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boroumand <em>et al.</em> 2014</a>). In this context, the transcriptional complexity in <em>auts2</em> family genes that we describe here provides an exciting opportunity to understand how misregulation of transcription at these loci leads to the disease conditions that they are associated with.</p></div><div id="sec-23" class="subsection"><h3>Expression patterns of auts2 paralogs</h3><p id="p-72">Differential patterns of gene expression among paralogs is widely believed to play a prominent role in morphological diversification. Duplicated genes are considered to diverge through neofunctionalization (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ohno 1970</a>) and/or subfunctionalization (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lynch and Force 2000</a>), but both processes can occur through evolution of the CDSs and/or the regulatory sequences, giving distinct and/or novel sites of expression.</p><p id="p-73">Previously, it was shown that <em>auts2a</em> is ubiquitously expressed in the central nervous system beginning from 24 hr (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Oksenberg <em>et al.</em> 2013</a>). Our data show that, already at very early stages, the expression of <em>auts2a</em> becomes restricted to the neural plate. Except for the <em>fbrs</em> gene, which is expressed ubiquitously through development, other <em>auts2</em> paralogs—<em>auts2a</em>, <em>auts2b</em>, and <em>fbrsl1</em>—show distinct expression patterns, particularly in the hindbrain, suggesting their role in patterning the hindbrain. For example, during neural keel formation, expression of <em>fbrsl1</em> in the hindbrain is mainly detected in rhombomere 4, while <em>auts2a</em> is expressed in rhombomeres 1, 2, and 4, and <em>auts2b</em> is expressed more broadly, with the strongest expression in rhombomere 2 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1/FigureS10.jpeg">Figure S10</a>). Expression of <em>fbrsl1</em> and <em>fbrs</em> genes in the Kupffer’s vesicle suggests their potential involvement in the establishment or maintenance of left–right asymmetry.</p><p id="p-74">Analysis of expression of <em>auts2</em> paralogs in juvenile brains revealed the presence of these transcripts in proliferation zones, suggesting their role in adult neurogenesis. In contrast to mammalian brains, teleostean brains have a tremendous number of proliferation zones. Many of these zones are found at or near the surfaces of ventricles (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zupanc <em>et al.</em> 2005</a>). Three paralogs—<em>auts2a</em>, <em>fbrsl1</em> and <em>fbrs</em>—are localized in the PGZ of the optic tectum, a site known for mitotic activity in the midbrain. In the cerebellum, proliferation zones are located in regions distant from any ventricle. Quantitative analysis has shown that the majority of the new brain cells are generated in the cerebellum; the proliferation zones are located in specific areas within the molecular layers of the cerebellar corpus and the valvula cerebelli (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Zupanc and Horschke 1995</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hinsch and Zupanc 2007</a>). <em>Auts2</em> paralogs are localized either in the granular layer (<em>auts2b</em> and <em>fbrs</em>) or Purkinje cell layer (<em>auts2a</em> and <em>fbrsl1</em>), sites where mitotic activity is minimal. Interestingly two <em>auts2</em> paralogs, <em>auts2b</em> and <em>fbrsl1</em>, are also expressed in the caudal lobe of the cerebellum, a granular layer of the eminentia granularis, the other site of mitotic activity in the cerebellum (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Zupanc and Horschke 1995</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hinsch and Zupanc 2007</a>).</p></div><div id="sec-24" class="subsection"><h3>Evolution of the auts2 gene locus</h3><p id="p-75">To our surprise, only cave fish and zebrafish possess a full-length copy of the <em>auts2b</em> gene. The other teleosts (where sequenced genomes are available) retain a highly reduced copy of the <em>auts2b</em> gene. Moreover, in the cave fish genome we could not find an <em>auts2a</em> gene and it is currently unclear if the absence of <em>auts2a</em> gene in the cave fish genome is simply due to incomplete sequencing and assembly, or it has indeed evolved beyond recognition. Phylogenetically, cave fish and zebrafish belong to the <em>Otomorpha</em> group. <em>Otomorpha</em> and <em>Euteleosteomorpha</em> (all other teleosts with sequenced genomes) split ∼245 MYA (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Broughton <em>et al.</em> 2013</a>). The most common fate of duplicated genes is that while one of the duplicated genes continues to be under selective pressure and retains the ancestral function, the other gene diverges and becomes nonfunctional through the accumulation of deleterious mutations (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Langham <em>et al.</em> 2004</a>). Less frequently, both genes are retained, which is the case for the <em>auts2a</em> and <em>auts2b</em> genes in the zebrafish genome. Although in other teleost genomes <em>auts2b</em> has evolved almost beyond recognition, transcription from this genomic locus could still be detected, as suggested by RNASeq data derived from amazon molly, tilapia, and platyfish transcriptomes. It will be interesting to examine how these transcripts are spatially expressed.</p></div><div id="sec-25" class="subsection"><h3>Conclusions</h3><p id="p-76">Taken together, our results show the existence of multiple <em>auts2</em> paralogs in zebrafish and the usage of alternative promoters and alternative splice sites for generating huge diversity in mRNA transcripts. The expression of these gene products is also tightly regulated developmentally and across multiple brain regions. Such complexity in regulation of these loci is bound to have significant functional roles and our future studies will be aimed at deciphering them.</p></div></div><div class="section" id="sec-26"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-77">We thank the National Centre of Biological Sciences (NCBS) DNA sequencing facility for the sequencing and Manjunath for maintaining the fish facility. This work was supported by a Wellcome Trust-DBT (Department of Biotechnology) India Alliance Intermediate Fellowship and a DBT research grant (awarded to V.T.), a NCBS Career Development Fellowship (awarded to I.K.), and an NCBS graduate student fellowship (L.R.). Additional funding was received from NCBS-TIFR core research support (V.T.). The authors declare no competing financial interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-78">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042622/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-79"><em>Communicating editor: D. J. Grunwald</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-04-20"><span class="received-label">Received </span>April 20, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-06-02"><span class="accepted-label">Accepted </span>June 2, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Kondrychyn <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.1" data-doi="10.1155/2012/475731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araujo</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ko</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Before it gets started: regulating translation at the 5′ UTR.</span> <abbr class="cit-jnl-abbrev">Comp. Funct. Genomics</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">475731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DComp%2BFunct%2BGenomics%26rft.aulast%253DAraujo%26rft.auinit1%253DP.%2BR.%26rft.volume%253D2012%26rft.spage%253D475731%26rft.epage%253D475731%26rft.atitle%253DBefore%2BIt%2BGets%2BStarted%253A%2BRegulating%2BTranslation%2Bat%2Bthe%2B5%2527%2BUTR.%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2012%252F475731%26rft_id%253Dinfo%253Apmid%252F22693426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2012/475731&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22693426&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bakkaloglu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Roak</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louvi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abelson</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular cytogenetic analysis and resequencing of contactin associated protein-like 2 in autism spectrum disorders.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D282%26rft.spage%253D165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.3" data-doi="10.1016/j.molcel.2012.05.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baltz</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munschauer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwanhäusser</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasile</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murakawa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The mRNA-bound proteome and its global occupancy profile on protein-coding transcripts.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">674</span>–<span class="cit-lpage">690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DBaltz%26rft.auinit1%253DA.%2BG.%26rft.volume%253D46%26rft.issue%253D5%26rft.spage%253D674%26rft.epage%253D690%26rft.atitle%253DThe%2BmRNA-bound%2Bproteome%2Band%2Bits%2Bglobal%2Boccupancy%2Bprofile%2Bon%2Bprotein-coding%2Btranscripts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.05.021%26rft_id%253Dinfo%253Apmid%252F22681889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.05.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22681889&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305095400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.4" data-doi="10.1016/j.gep.2009.11.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bedogni</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodge</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frederick</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiba</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Autism susceptibility candidate 2 (Auts2) encodes a nuclear protein expressed in developing brain regions implicated in autism neuropathology.</span> <abbr class="cit-jnl-abbrev">Gene Expr. Patterns</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrain%2Bresearch.%2BGene%2Bexpression%2Bpatterns%26rft.stitle%253DBrain%2BRes%2BGene%2BExpr%2BPatterns%26rft.aulast%253DBedogni%26rft.auinit1%253DF.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D15%26rft.atitle%253DAutism%2Bsusceptibility%2Bcandidate%2B2%2B%2528Auts2%2529%2Bencodes%2Ba%2Bnuclear%2Bprotein%2Bexpressed%2Bin%2Bdeveloping%2Bbrain%2Bregions%2Bimplicated%2Bin%2Bautism%2Bneuropathology.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gep.2009.11.005%26rft_id%253Dinfo%253Apmid%252F19948250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gep.2009.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19948250&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.5" data-doi="10.1038/ejhg.2014.173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beunders</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Munnik</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van der Aa</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceulemans</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorhoeve</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Two male adults with pathogenic AUTS2 variants, including a two-base pair deletion, further delineate the AUTS2 syndrome.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D23%26rft.spage%253D803%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2014.173%26rft_id%253Dinfo%253Apmid%252F25205402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2014.173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25205402&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.6" data-doi="10.1309/LMLS813ZDPHRFLUU"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boroumand</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziaee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zarghami</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anvari</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheraghi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Kalirin gene rs9289231 polymorphism as a novel predisposing marker for coronary artery disease.</span> <abbr class="cit-jnl-abbrev">Lab. Med.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">302</span>–<span class="cit-lpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLab.%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1309%252FLMLS813ZDPHRFLUU%26rft_id%253Dinfo%253Apmid%252F25316661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1309/LMLS813ZDPHRFLUU&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25316661&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broughton</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betancur-R</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arratia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orti</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multi-locus phylogenetic analysis reveals the pattern and tempo of bony fish evolution.</span> <abbr class="cit-jnl-abbrev">PLoS Curr.</abbr> DOI: 10.1371/currents.tol.2ca8041495ffafd0c92756e75247483e.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.8" data-doi="10.1038/ng1789"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carninci</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandelin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenhard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katayama</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimokawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide analysis of mammalian promoter architecture and evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">626</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCarninci%26rft.auinit1%253DP.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D626%26rft.epage%253D635%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Bmammalian%2Bpromoter%2Barchitecture%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1789%26rft_id%253Dinfo%253Apmid%252F16645617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1789&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16645617&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237954800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic analysis of AUTS2 as a susceptibility gene of heroin dependence.</span> <abbr class="cit-jnl-abbrev">Drug Alcohol Depend.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">238</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDrug%2BAlcohol%2BDepend.%26rft.volume%253D128%26rft.spage%253D238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.10" data-doi="10.1371/journal.pone.0057199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chojnicka</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gajos</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broda</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fudalej</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Possible association between suicide committed under influence of ethanol and a variant in the AUTS2 gene.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e57199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De57199%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0057199%26rft_id%253Dinfo%253Apmid%252F23437340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0057199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23437340&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.11" data-doi="10.1093/molbev/msh114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christoffels</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aparicio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Fugu genome analysis provides evidence for a whole-genome duplication early during the evolution of ray-finned fishes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1146</span>–<span class="cit-lpage">1151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh114%26rft_id%253Dinfo%253Apmid%252F15014147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msh114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15014147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221599300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.12" data-doi="10.1101/gr.137901.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">S. M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stemple</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Incorporating RNA-seq data into the zebrafish Ensembl genebuild.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2067</span>–<span class="cit-lpage">2078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.137901.112%26rft_id%253Dinfo%253Apmid%252F22798491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMC8yMDY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.13" data-doi="10.1038/tp.2013.100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darlington</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pimentel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huff</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic risk factors in two Utah pedigrees at high risk for suicide.</span> <abbr class="cit-jnl-abbrev">Transl. Psychiatry</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransl.%2BPsychiatry%26rft.volume%253D3%26rft.spage%253D325e%26rft_id%253Dinfo%253Adoi%252F10.1038%252Ftp.2013.100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/tp.2013.100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The evidence for the contribution of the autism susceptibility candidate 2 (AUTS2) gene in heroin dependence susceptibility.</span> <abbr class="cit-jnl-abbrev">J. Mol. Neurosci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">811</span>–<span class="cit-lpage">819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BNeurosci.%26rft.volume%253D54%26rft.spage%253D811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.15" data-doi="10.1016/j.tig.2008.01.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davuluri</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plass</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The functional consequences of alternative promoter use in mammalian genomes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DDavuluri%26rft.auinit1%253DR.%2BV.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D167%26rft.epage%253D177%26rft.atitle%253DThe%2Bfunctional%2Bconsequences%2Bof%2Balternative%2Bpromoter%2Buse%2Bin%2Bmammalian%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2008.01.008%26rft_id%253Dinfo%253Apmid%252F18329129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2008.01.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18329129&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255346300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.16" data-doi="10.1016/j.molcel.2012.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonasio</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strino</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sawai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">PCGF homologs, CBX proteins and RYBP define functionally distinct PRC1 family complexes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DGao%26rft.auinit1%253DZ.%26rft.volume%253D45%26rft.issue%253D3%26rft.spage%253D344%26rft.epage%253D356%26rft.atitle%253DPCGF%2Bhomologs%252C%2BCBX%2Bproteins%252C%2Band%2BRYBP%2Bdefine%2Bfunctionally%2Bdistinct%2BPRC1%2Bfamily%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.01.002%26rft_id%253Dinfo%253Apmid%252F22325352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22325352&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300481100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.17" data-doi="10.1038/nature13921"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stafford</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Schimmelmann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaefer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An AUTS2-Polycomb complex activates gene expression in the CNS.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">516</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D516%26rft.spage%253D349%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13921%26rft_id%253Dinfo%253Apmid%252F25519132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13921&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25519132&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.18" data-doi="10.1126/science.1188021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maricic</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stenzel</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A draft sequence of the Neandertal genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">328</span>: <span class="cit-fpage">710</span>–<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGreen%26rft.auinit1%253DR.%2BE.%26rft.volume%253D328%26rft.issue%253D5979%26rft.spage%253D710%26rft.epage%253D722%26rft.atitle%253DA%2BDraft%2BSequence%2Bof%2Bthe%2BNeandertal%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1188021%26rft_id%253Dinfo%253Apmid%252F20448178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjgvNTk3OS83MTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjU3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.19" data-doi="10.1113/jphysiol.2006.115568"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gustincich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandelin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plessy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katayama</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simone</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The complexity of the mammalian transcriptome.</span> <abbr class="cit-jnl-abbrev">J. Physiol.</abbr> <span class="cit-vol">575</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1113%252Fjphysiol.2006.115568%26rft_id%253Dinfo%253Apmid%252F16857706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1113/jphysiol.2006.115568&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16857706&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240007000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.20" data-doi="10.1038/ng1469"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hiller</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huse</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szafranski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahn</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampe</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Widespread occurrence of alternative splicing at NAGNAG acceptors contributes to proteome plasticity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1255</span>–<span class="cit-lpage">1257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHiller%26rft.auinit1%253DM.%26rft.volume%253D36%26rft.issue%253D12%26rft.spage%253D1255%26rft.epage%253D1257%26rft.atitle%253DWidespread%2Boccurrence%2Bof%2Balternative%2Bsplicing%2Bat%2BNAGNAG%2Bacceptors%2Bcontributes%2Bto%2Bproteome%2Bplasticity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1469%26rft_id%253Dinfo%253Apmid%252F15516930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1469&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15516930&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225354100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinsch</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zupanc</span>  <span class="cit-name-given-names">G. K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Generation and long-term persistence of new neurons in the adult zebrafish brain: a quantitative analysis.</span> <abbr class="cit-jnl-abbrev">Neuroscience</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuroscience%26rft.stitle%253DNeuroscience%26rft.aulast%253DHinsch%26rft.auinit1%253DK.%26rft.volume%253D146%26rft.issue%253D2%26rft.spage%253D679%26rft.epage%253D696%26rft.atitle%253DGeneration%2Band%2Blong-term%2Bpersistence%2Bof%2Bnew%2Bneurons%2Bin%2Bthe%2Badult%2Bzebrafish%2Bbrain%253A%2Ba%2Bquantitative%2Banalysis.%26rft_id%253Dinfo%253Apmid%252F17395385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17395385&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.22" data-doi="10.1007/s00239-004-2613-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoegg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brinkmann</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Phylogenetic timing of the fish-specific genome duplication correlates with the diversification of teleost fish.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">190</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DHoegg%26rft.auinit1%253DS.%26rft.volume%253D59%26rft.issue%253D2%26rft.spage%253D190%26rft.epage%253D203%26rft.atitle%253DPhylogenetic%2Btiming%2Bof%2Bthe%2Bfish-specific%2Bgenome%2Bduplication%2Bcorrelates%2Bwith%2Bthe%2Bdiversification%2Bof%2Bteleost%2Bfish.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-004-2613-z%26rft_id%253Dinfo%253Apmid%252F15486693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-004-2613-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15486693&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223424800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.23" data-doi="10.1016/j.celrep.2014.11.045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hori</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cytoskeletal regulation by AUTS2 in neuronal migration and neuritogenesis.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2166</span>–<span class="cit-lpage">2179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D9%26rft.spage%253D2166%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2014.11.045%26rft_id%253Dinfo%253Apmid%252F25533347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2014.11.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25533347&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.24" data-doi="10.1111/j.1469-1809.2009.00540.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horne</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muhlestein</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Validation study of genetic associations with coronary artery disease on chromosome 3q13–21 and potential effect modification by smoking.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DHorne%26rft.auinit1%253DB.%2BD.%26rft.volume%253D73%26rft.issue%253DPt%2B6%26rft.spage%253D551%26rft.epage%253D558%26rft.atitle%253DValidation%2Bstudy%2Bof%2Bgenetic%2Bassociations%2Bwith%2Bcoronary%2Bartery%2Bdisease%2Bon%2Bchromosome%2B3q13-21%2Band%2Bpotential%2Beffect%2Bmodification%2Bby%2Bsmoking.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.2009.00540.x%26rft_id%253Dinfo%253Apmid%252F19706030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.2009.00540.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19706030&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.25" data-doi="10.1007/s00439-006-0284-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalscheuer</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">FitzPatrick</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tommerup</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bugge</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niebuhr</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mutations in autism susceptibility candidate 2 (AUTS2) in patients with mental retardation.</span> <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DKalscheuer%26rft.auinit1%253DV.%2BM.%26rft.volume%253D121%26rft.issue%253D3-4%26rft.spage%253D501%26rft.epage%253D509%26rft.atitle%253DMutations%2Bin%2Bautism%2Bsusceptibility%2Bcandidate%2B2%2B%2528AUTS2%2529%2Bin%2Bpatients%2Bwith%2Bmental%2Bretardation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-006-0284-0%26rft_id%253Dinfo%253Apmid%252F17211639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-006-0284-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17211639&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245494200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kapoor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wetherill</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertelsen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A meta-analysis of two genome-wide association studies to identify novel loci for maximum number of alcoholic drinks.</span> <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BGenet.%26rft.volume%253D132%26rft.spage%253D1141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.27" data-doi="10.1101/gr.4039406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakamatsu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ota</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishikawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Diversification of transcriptional modulation: large-scale identification and characterization of putative alternative promoters of human genes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4039406%26rft_id%253Dinfo%253Apmid%252F16344560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjE2LzEvNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjU3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.28" data-doi="10.1016/j.tig.2003.09.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mager</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>  <span class="cit-name-given-names">B. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Complex controls: the role of alternative promoters in mammalian genomes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">640</span>–<span class="cit-lpage">648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLandry%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D19%26rft.issue%253D11%26rft.spage%253D640%26rft.epage%253D648%26rft.atitle%253DComplex%2Bcontrols%253A%2Bthe%2Brole%2Bof%2Balternative%2Bpromoters%2Bin%2Bmammalian%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2003.09.014%26rft_id%253Dinfo%253Apmid%252F14585616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2003.09.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14585616&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186530600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.29" data-doi="10.1534/genetics.166.2.935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langham</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ko</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genomic duplication, fractionation and the origin of regulatory novelty.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLangham%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D166%26rft.issue%253D2%26rft.spage%253D935%26rft.epage%253D945%26rft.atitle%253DGenomic%2BDuplication%252C%2BFractionation%2Band%2Bthe%2BOrigin%2Bof%2BRegulatory%2BNovelty%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.2.935%26rft_id%253Dinfo%253Apmid%252F15020478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY2LzIvOTM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.30" data-doi="10.1371/journal.pone.0009482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Increasing alternative promoter repertories is positively associated with differential expression and disease susceptibility.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e9482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLiu%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9482%26rft.epage%253De9482%26rft.atitle%253DIncreasing%2Balternative%2Bpromoter%2Brepertories%2Bis%2Bpositively%2Bassociated%2Bwith%2Bdifferential%2Bexpression%2Band%2Bdisease%2Bsusceptibility.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009482%26rft_id%253Dinfo%253Apmid%252F20208995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208995&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Force</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The probability of duplicated gene preservation by subfunctionalization.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D154%26rft.issue%253D1%26rft.spage%253D459%26rft.epage%253D473%26rft.atitle%253DThe%2BProbability%2Bof%2BDuplicate%2BGene%2BPreservation%2Bby%2BSubfunctionalization%26rft_id%253Dinfo%253Apmid%252F10629003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzEvNDU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.32" data-doi="10.1161/01.RES.0000096652.14509.96"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makielski</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdivia</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pagel</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A ubiquitous splice variant and a common polymorphism affect heterologous expression of recombinant human SCN5A heart sodium channels.</span> <abbr class="cit-jnl-abbrev">Circ. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">828</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCirc.%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1161%252F01.RES.0000096652.14509.96%26rft_id%253Dinfo%253Apmid%252F14500339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImNpcmNyZXNhaGEiO3M6NToicmVzaWQiO3M6ODoiOTMvOS84MjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjU3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.33" data-doi="10.1038/mp.2014.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lihm</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title"><em>De novo</em> mutations in schizophrenia implicate chromatin remodeling and support a genetic overlap with autism and intellectual disability.</span> <abbr class="cit-jnl-abbrev">Mol. Psychiatry</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">652</span>–<span class="cit-lpage">658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPsychiatry%26rft.volume%253D19%26rft.spage%253D652%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmp.2014.29%26rft_id%253Dinfo%253Apmid%252F24776741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/mp.2014.29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24776741&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336663100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2577.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <em>Evolution by Gene and Genome Duplication</em>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.35" data-doi="10.1016/j.tig.2013.08.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oksenberg</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahituv</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The role of AUTS2 in neurodevelopment and human evolution.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">600</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D29%26rft.spage%253D600%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2013.08.001%26rft_id%253Dinfo%253Apmid%252F24008202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2013.08.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24008202&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.36" data-doi="10.1371/journal.pgen.1003221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oksenberg</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevison</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahituv</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Function and regulation of AUTS2, a gene implicated in autism and human evolution.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003221%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003221%26rft_id%253Dinfo%253Apmid%252F23349641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003221&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349641&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.37" data-doi="10.1016/j.biopsych.2004.06.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Souza</span>  <span class="cit-name-given-names">U. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arranz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerwin</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makoff</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The -1438A/G polymorphism in the 5-hydroxytryptamine type 2A receptor gene affects promoter activity.</span> <abbr class="cit-jnl-abbrev">Biol. Psychiatry</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">406</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2Bpsychiatry%26rft.stitle%253DBiol%2BPsychiatry%26rft.aulast%253DParsons%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D56%26rft.issue%253D6%26rft.spage%253D406%26rft.epage%253D410%26rft.atitle%253DThe%2B-1438A%252FG%2Bpolymorphism%2Bin%2Bthe%2B5-hydroxytryptamine%2Btype%2B2A%2Breceptor%2Bgene%2Baffects%2Bpromoter%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biopsych.2004.06.020%26rft_id%253Dinfo%253Apmid%252F15364038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biopsych.2004.06.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15364038&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223855200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.38" data-doi="10.1126/science.1130738"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prabhakar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noonan</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paabo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Accelerated evolution of conserved noncoding sequences in humans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">314</span>: <span class="cit-fpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPrabhakar%26rft.auinit1%253DS.%26rft.volume%253D314%26rft.issue%253D5800%26rft.spage%253D786%26rft.epage%253D786%26rft.atitle%253DAccelerated%2BEvolution%2Bof%2BConserved%2BNoncoding%2BSequences%2Bin%2BHumans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1130738%26rft_id%253Dinfo%253Apmid%252F17082449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTQvNTgwMC83ODYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjU3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.39" data-doi="10.1038/nature05329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishikawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitch</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Global variation in copy number in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">444</span>: <span class="cit-fpage">444</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRedon%26rft.auinit1%253DR.%26rft.volume%253D444%26rft.issue%253D7118%26rft.spage%253D444%26rft.epage%253D454%26rft.atitle%253DGlobal%2Bvariation%2Bin%2Bcopy%2Bnumber%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05329%26rft_id%253Dinfo%253Apmid%252F17122850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17122850&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242215700038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.40" data-doi="10.1371/journal.pgen.1000397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salichs</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ledda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mularoni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alba</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de la Luna</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide analysis of histidine repeats reveals their role in the localization of human proteins to the nuclear speckles compartment.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSalichs%26rft.auinit1%253DE.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De1000397%26rft.epage%253De1000397%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Bhistidine%2Brepeats%2Breveals%2Btheir%2Brole%2Bin%2Bthe%2Blocalization%2Bof%2Bhuman%2Bproteins%2Bto%2Bthe%2Bnuclear%2Bspeckles%2Bcompartment.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000397%26rft_id%253Dinfo%253Apmid%252F19266028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000397&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19266028&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.41" data-doi="10.1073/pnas.1017288108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schumann</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coin</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lourdusamy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charoen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association and genetic functional studies identify autism susceptibility candidate 2 gene (AUTS2) in the regulation of alcohol consumption.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">7119</span>–<span class="cit-lpage">7124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1017288108%26rft_id%253Dinfo%253Apmid%252F21471458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE3LzcxMTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjU3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.42" data-doi="10.1006/geno.2002.6810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sultana</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Identification of a novel gene on chromosome 7q11.2 interrupted by a translocation breakpoint in a pair of autistic twins.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DSultana%26rft.auinit1%253DR.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D129%26rft.epage%253D134%26rft.atitle%253DIdentification%2Bof%2Ba%2Bnovel%2Bgene%2Bon%2Bchromosome%2B7q11.2%2Binterrupted%2Bby%2Ba%2Btranslocation%2Bbreakpoint%2Bin%2Ba%2Bpair%2Bof%2Bautistic%2Btwins.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fgeno.2002.6810%26rft_id%253Dinfo%253Apmid%252F12160723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/geno.2002.6810&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12160723&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177393500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.43" data-doi="10.1093/embo-reports/kve085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taira</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsunoda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mizushima-Sugano</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sese</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Diverse transcriptional initiation revealed by fine, large-scale mapping of mRNA start sites.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">388</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2Breports%26rft.stitle%253DEMBO%2BRep.%26rft.aulast%253DSuzuki%26rft.auinit1%253DY.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D388%26rft.epage%253D393%26rft.atitle%253DDiverse%2Btranscriptional%2Binitiation%2Brevealed%2Bby%2Bfine%252C%2Blarge-scale%2Bmapping%2Bof%2BmRNA%2Bstart%2Bsites%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fembo-reports%252Fkve085%26rft_id%253Dinfo%253Apmid%252F11375929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6NzoiMi81LzM4OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNTc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.44" data-doi="10.1101/gad.8.1.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuzaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyagawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amanuma</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">A novel POU domain gene, zebrafish <em>pou2</em>: expression and roles of two alternatively spliced twin products in early development.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DTakeda%26rft.auinit1%253DH%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D45%26rft.epage%253D59%26rft.atitle%253DA%2Bnovel%2BPOU%2Bdomain%2Bgene%252C%2Bzebrafish%2Bpou2%253A%2Bexpression%2Band%2Broles%2Bof%2Btwo%2Balternatively%2Bspliced%2Btwin%2Bproducts%2Bin%2Bearly%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.8.1.45%26rft_id%253Dinfo%253Apmid%252F8288127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NjoiOC8xLzQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1NzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.45" data-doi="10.1038/nprot.2007.514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thisse</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thisse</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution <em>in situ</em> hybridization to whole-mount zebrafish embryos.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DThisse%26rft.auinit1%253DC.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D69%26rft.atitle%253DHigh-resolution%2Bin%2Bsitu%2Bhybridization%2Bto%2Bwhole-mount%2Bzebrafish%2Bembryos.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2007.514%26rft_id%253Dinfo%253Apmid%252F18193022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2007.514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18193022&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253140400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.46" data-doi="10.1016/0896-6273(93)90335-O"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Timmusk</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palm</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metsis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reintam</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paalme</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Multiple promoters direct tissue-specific expression of the rat BDNF gene.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.stitle%253DNeuron%26rft.aulast%253DTimmusk%26rft.auinit1%253DT.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D475%26rft.epage%253D489%26rft.atitle%253DMultiple%2Bpromoters%2Bdirect%2Btissue-specific%2Bexpression%2Bof%2Bthe%2Brat%2BBDNF%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0896-6273%252893%252990335-O%26rft_id%253Dinfo%253Apmid%252F8461137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0896-6273(93)90335-O&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8461137&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KU90100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.47" data-doi="10.1016/j.neuropsychologia.2009.10.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vann</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Re-evaluating the role of the mammillary bodies in memory.</span> <abbr class="cit-jnl-abbrev">Neuropsychologia</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">2316</span>–<span class="cit-lpage">2327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuropsychologia%26rft.volume%253D48%26rft.spage%253D2316%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuropsychologia.2009.10.019%26rft_id%253Dinfo%253Apmid%252F19879886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuropsychologia.2009.10.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19879886&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280322000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.48" data-doi="10.1016/j.exer.2013.03.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anand</span>  <span class="cit-name-given-names">V. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatterton</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Exon-level expression profiling of ocular tissues.</span> <abbr class="cit-jnl-abbrev">Exp. Eye Res.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExp.%2BEye%2BRes.%26rft.volume%253D111%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.exer.2013.03.004%26rft_id%253Dinfo%253Apmid%252F23500522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.exer.2013.03.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23500522&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319788700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.49" data-doi="10.1086/512981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pericak-Vance</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haynes</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Peakwide mapping on chromosome 3q13 identifies the kalirin gene as a novel candidate gene for coronary artery disease.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">650</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DL.%26rft.volume%253D80%26rft.issue%253D4%26rft.spage%253D650%26rft.epage%253D663%26rft.atitle%253DPeakwide%2Bmapping%2Bon%2Bchromosome%2B3q13%2Bidentifies%2Bthe%2Bkalirin%2Bgene%2Bas%2Ba%2Bnovel%2Bcandidate%2Bgene%2Bfor%2Bcoronary%2Bartery%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F512981%26rft_id%253Dinfo%253Apmid%252F17357071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/512981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17357071&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245465200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2577.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westerfield</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <em>The Zebrafish Book: A Guide for the Laboratory Use of Zebrafish (Danio rerio)</em>, <span class="cit-edition">Ed. 4</span>. <span class="cit-publ-name">University of Oregon Press</span>, <span class="cit-publ-loc">Eugene, OR</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.51" data-doi="10.3390/ijms151119406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Association study identifying a new susceptibility gene (AUTS2) for schizophrenia.</span> <abbr class="cit-jnl-abbrev">Int. J. Mol. Sci.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">19406</span>–<span class="cit-lpage">19416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BMol.%2BSci.%26rft.volume%253D15%26rft.spage%253D19406%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fijms151119406%26rft_id%253Dinfo%253Apmid%252F25347278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/ijms151119406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25347278&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.52" data-doi="10.1002/cne.903530205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zupanc</span>  <span class="cit-name-given-names">G. K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horschke</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Proliferation zones in the brain of adult gymnotiform fish: a quantitative mapping study.</span> <abbr class="cit-jnl-abbrev">J. Comp. Neurol.</abbr> <span class="cit-vol">353</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bcomparative%2Bneurology%26rft.stitle%253DJ%2BComp%2BNeurol%26rft.aulast%253DZupanc%26rft.auinit1%253DG.%2BK.%26rft.volume%253D353%26rft.issue%253D2%26rft.spage%253D213%26rft.epage%253D233%26rft.atitle%253DProliferation%2Bzones%2Bin%2Bthe%2Bbrain%2Bof%2Badult%2Bgymnotiform%2Bfish%253A%2Ba%2Bquantitative%2Bmapping%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcne.903530205%26rft_id%253Dinfo%253Apmid%252F7745132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cne.903530205&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7745132&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QF23100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2577.53" data-doi="10.1002/cne.20571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zupanc</span>  <span class="cit-name-given-names">G. K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinsch</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gage</span>  <span class="cit-name-given-names">F. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Proliferation, migration, neuronal differentiation, and long-term survival of new cells in the adult zebrafish brain.</span> <abbr class="cit-jnl-abbrev">J. Comp. Neurol.</abbr> <span class="cit-vol">488</span>: <span class="cit-fpage">290</span>–<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bcomparative%2Bneurology%26rft.stitle%253DJ%2BComp%2BNeurol%26rft.aulast%253DZupanc%26rft.auinit1%253DG.%2BK.%26rft.volume%253D488%26rft.issue%253D3%26rft.spage%253D290%26rft.epage%253D319%26rft.atitle%253DProliferation%252C%2Bmigration%252C%2Bneuronal%2Bdifferentiation%252C%2Band%2Blong-term%2Bsurvival%2Bof%2Bnew%2Bcells%2Bin%2Bthe%2Badult%2Bzebrafish%2Bbrain.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcne.20571%26rft_id%253Dinfo%253Apmid%252F15952170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cne.20571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15952170&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2577.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230218200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/8/2577.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2565" title="Duplication and Loss of Function of Genes Encoding RNA Polymerase III Subunit C4 Causes Hybrid Incompatibility in Rice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2595" title="Unique Allelic eQTL Clusters in Human MHC Haplotypes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/8"><span>Volume 7 Issue 8, August 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/8.cover-source.jpg?itok=apH_YDuu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2577.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-M1qLSgBxMil7Dl5BZKYCM1HK_rVqjB5R5hoju6I8fiA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/8/2577" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/8/2577" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/8/2577" class="active" data-icon-position="" data-hide-link-title="0">Transcriptional Complexity and Distinct Expression Patterns of auts2 Paralogs in Danio rerio</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485839" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-iRQOPtuo4Z_kBfeTQMYm4JDDNsI8TwDoPA54Xsohld4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2577.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/8/2577" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form--76X-B7lcWzODKt1-IytHpO-09RH5IEuFIMqb8hpelw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/8/2577" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2577.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485839" id="node485839--2" data-pisa="ggg;7/8/2577" data-pisa-master="ggg;g3.117.042622" data-apath="/ggg/7/8/2577.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptional Complexity and Distinct Expression Patterns of <em>auts2</em> Paralogs in <em>Danio rerio</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2268-7291" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Igor Kondrychyn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3510-9228" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lena Robra</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2318-5023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vatsala Thirumalai</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2577-2593;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042622 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485839--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Igor Kondrychyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bangalore, Karnataka 560065, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Igor%2BKondrychyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kondrychyn%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIgor%2BKondrychyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2268-7291" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Igor Kondrychyn</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lena Robra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bangalore, Karnataka 560065, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lena%2BRobra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robra%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALena%2BRobra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3510-9228" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lena Robra</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Vatsala Thirumalai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bangalore, Karnataka 560065, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vatsala%2BThirumalai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thirumalai%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVatsala%2BThirumalai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2318-5023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vatsala Thirumalai</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vatsala@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">vatsala@ncbs.res.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/8/2577" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/8/2577&amp;title=Transcriptional%20Complexity%20and%20Distinct%20Expression%20Patterns%20of%20auts2%20Paralogs%20in%20Danio%20rerio" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/8/2577&amp;title=Transcriptional%20Complexity%20and%20Distinct%20Expression%20Patterns%20of%20auts2%20Paralogs%20in%20Danio%20rerio" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/8/2577&amp;title=Transcriptional%20Complexity%20and%20Distinct%20Expression%20Patterns%20of%20auts2%20Paralogs%20in%20Danio%20rerio" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/8/2577&amp;text=Transcriptional%20Complexity%20and%20Distinct%20Expression%20Patterns%20of%20auts2%20Paralogs%20in%20Danio%20rerio" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/8/2577&amp;title=Transcriptional%20Complexity%20and%20Distinct%20Expression%20Patterns%20of%20auts2%20Paralogs%20in%20Danio%20rerio" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/8/2577&amp;t=Transcriptional%20Complexity%20and%20Distinct%20Expression%20Patterns%20of%20auts2%20Paralogs%20in%20Danio%20rerio" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/8/2577&amp;title=Transcriptional%20Complexity%20and%20Distinct%20Expression%20Patterns%20of%20auts2%20Paralogs%20in%20Danio%20rerio" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/8/2577&amp;title=Transcriptional%20Complexity%20and%20Distinct%20Expression%20Patterns%20of%20auts2%20Paralogs%20in%20Danio%20rerio" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485839/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485839" id="node485839--3" data-pisa="ggg;7/8/2577" data-pisa-master="ggg;g3.117.042622" data-apath="/ggg/7/8/2577.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptional Complexity and Distinct Expression Patterns of <em>auts2</em> Paralogs in <em>Danio rerio</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2268-7291" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Igor Kondrychyn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3510-9228" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lena Robra</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2318-5023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vatsala Thirumalai</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2577-2593;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042622 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485839--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Igor Kondrychyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bangalore, Karnataka 560065, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Igor%2BKondrychyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kondrychyn%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIgor%2BKondrychyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2268-7291" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Igor Kondrychyn</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lena Robra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bangalore, Karnataka 560065, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lena%2BRobra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robra%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALena%2BRobra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3510-9228" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lena Robra</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Vatsala Thirumalai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bangalore, Karnataka 560065, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vatsala%2BThirumalai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thirumalai%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVatsala%2BThirumalai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2318-5023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vatsala Thirumalai</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vatsala@ncbs.res.in" class="" data-icon-position="" data-hide-link-title="0">vatsala@ncbs.res.in</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2577&amp;rft.epage=2593&amp;rft.atitle=Transcriptional%20Complexity%20and%20Distinct%20Expression%20Patterns%20of%20auts2%20Paralogs%20in%20Danio%20rerio&amp;rft.volume=7&amp;rft.issue=8&amp;rft.date=2017-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kondrychyn%2C+Igor&amp;rft.au=Robra%2C+Lena&amp;rft.au=Thirumalai%2C+Vatsala"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485839/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485839/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485839/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485839/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485839/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485839/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485839/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485839/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485839/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485839/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485839/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485839/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/8/2577" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/8/2577.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/8/2577.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/8/2577.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/8/2577.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
