<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/8/2859" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/8/2859.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/8/2859.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/8/2859.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/8/2859/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/8/2859.atom" />
<meta name="HW.pisa" content="ggg;7/8/2859" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A High-Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (Cervus elaphus)" />
<meta name="DC.Identifier" content="10.1534/g3.117.044198" />
<meta name="DC.Date" content="2017-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Johnston et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="High-density linkage maps are an important tool to gain insight into the genetic architecture of traits of evolutionary and economic interest, and provide a resource to characterize variation in recombination landscapes. Here, we used information from the cattle genome and the 50 K Cervine Illumina BeadChip to inform and refine a high-density linkage map in a wild population of red deer ( Cervus elaphus ). We constructed a predicted linkage map of 38,038 SNPs and a skeleton map of 10,835 SNPs across 34 linkage groups. We identified several chromosomal rearrangements in the deer lineage relative to sheep and cattle, including six chromosome fissions, one fusion, and two large inversions. Otherwise, our findings showed strong concordance with map orders in the cattle genome. The sex-averaged linkage map length was 2739.7 cM and the genome-wide autosomal recombination rate was 1.04 cM/Mb. The female autosomal map length was 1.21 longer than that of males (2767.4 cM vs. 2280.8 cM, respectively). Sex differences in map length were driven by high female recombination rates in peri-centromeric regions, a pattern that is unusual relative to other mammal species. This effect was more pronounced in fission chromosomes that would have had to produce new centromeres. We propose two hypotheses to explain this effect: (1) that this mechanism may have evolved to counteract centromeric drive associated with meiotic asymmetry in oocyte production; and/or (2) that sequence and structural characteristics suppressing recombination in close proximity to the centromere may not have evolved at neo-centromeres. Our study provides insight into how recombination landscapes vary and evolve in mammals, and will provide a valuable resource for studies of evolution, genetic improvement, and population management in red deer and related species." />
<meta name="DC.Contributor" content="Susan E. Johnston" />
<meta name="DC.Contributor" content="Jisca Huisman" />
<meta name="DC.Contributor" content="Philip A. Ellis" />
<meta name="DC.Contributor" content="Josephine M. Pemberton" />
<meta name="article:published_time" content="2017-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A High-Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (Cervus elaphus)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;High-density linkage maps are an important tool to gain insight into the genetic architecture of traits of evolutionary and economic interest, and provide a resource to characterize variation in recombination landscapes. Here, we used information from the cattle genome and the 50 K Cervine Illumina BeadChip to inform and refine a high-density linkage map in a wild population of red deer (&lt;i&gt;Cervus elaphus&lt;/i&gt;). We constructed a predicted linkage map of 38,038 SNPs and a skeleton map of 10,835 SNPs across 34 linkage groups. We identified several chromosomal rearrangements in the deer lineage relative to sheep and cattle, including six chromosome fissions, one fusion, and two large inversions. Otherwise, our findings showed strong concordance with map orders in the cattle genome. The sex-averaged linkage map length was 2739.7 cM and the genome-wide autosomal recombination rate was 1.04 cM/Mb. The female autosomal map length was 1.21 longer than that of males (2767.4 cM &lt;i&gt;vs.&lt;/i&gt; 2280.8 cM, respectively). Sex differences in map length were driven by high female recombination rates in peri-centromeric regions, a pattern that is unusual relative to other mammal species. This effect was more pronounced in fission chromosomes that would have had to produce new centromeres. We propose two hypotheses to explain this effect: (1) that this mechanism may have evolved to counteract centromeric drive associated with meiotic asymmetry in oocyte production; and/or (2) that sequence and structural characteristics suppressing recombination in close proximity to the centromere may not have evolved at neo-centromeres. Our study provides insight into how recombination landscapes vary and evolve in mammals, and will provide a valuable resource for studies of evolution, genetic improvement, and population management in red deer and related species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/08/01" />
<meta name="citation_mjid" content="ggg;7/8/2859" />
<meta name="citation_id" content="7/8/2859" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/8/2859" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/8/2859.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/8/2859.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/8/2859.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.044198" />
<meta name="citation_pmid" content="28667018" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2859" />
<meta name="citation_lastpage" content="2870" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Susan E. Johnston" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom" />
<meta name="citation_author_email" content="Susan.Johnston@ed.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5623-8902" />
<meta name="citation_author" content="Jisca Huisman" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9744-7196" />
<meta name="citation_author" content="Philip A. Ellis" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom" />
<meta name="citation_author" content="Josephine M. Pemberton" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0075-1504" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. S. Aulchenko;citation_author=S. Ripke;citation_author=A. Isaacs;citation_author=C. M. van Duijn;citation_title=GenABEL: an R library for genome-wide association analysis.;citation_volume=23;citation_year=2007;citation_pmid=17384015;citation_doi=10.1093/bioinformatics/btm108" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=M. R. Band;citation_author=J. H. Larson;citation_author=M. Rebeiz;citation_author=C. A. Green;citation_author=D. W. Heyen;citation_title=An Ordered Comparative Map of the Cattle and Human Genomes;citation_pages=1359-1368;citation_volume=10;citation_year=2000;citation_issue=9;citation_pmid=10984454;citation_doi=10.1101/gr.145900" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. Bérénos;citation_author=P. A. Ellis;citation_author=J. G. Pilkington;citation_author=S. H. Lee;citation_author=J. Gratten;citation_title=Heterogeneity of genetic architecture of body size traits in a free-living population.;citation_pages=1810-1830;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13146" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=I. R. Bradbury;citation_author=S. Hubert;citation_author=B. Higgins;citation_author=S. Bowman;citation_author=T. Borza;citation_title=Genomic islands of divergence and their consequences for the resolution of spatial structure in an exploited marine fish.;citation_pages=450-461;citation_volume=6;citation_year=2013;citation_pmid=23745137;citation_doi=10.1111/eva.12026" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Brandvain;citation_author=G. Coop;citation_title=Scrambling eggs: meiotic drive and the evolution of female recombination rates.;citation_volume=190;citation_year=2012;citation_pmid=22143919;citation_doi=10.1534/genetics.111.136721" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. S. O. Brieuc;citation_author=C. D. Waters;citation_author=J. E. Seeb;citation_author=K. A. Naish;citation_title=A dense linkage map for chinook salmon (Oncorhynchus tshawytscha) reveals variable chromosomal divergence after an ancestral whole genome duplication event.;citation_pages=447-460;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=A. Burt;citation_title=Perspective: sex, recombination, and the efficacy of selection--was Weismann right?;citation_pages=337-351;citation_volume=54;citation_year=2000;citation_issue=2;citation_pmid=10937212;citation_doi=10.1554/0014-3820(2000)054[0337:PSRATE]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Burt;citation_author=G. Bell;citation_title=Mammalian chiasma frequencies as a test of two theories of recombination.;citation_pages=803-805;citation_volume=326;citation_year=1987;citation_issue=6115;citation_pmid=3574451;citation_doi=10.1038/326803a0" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=Regularized gene selection in cancer microarray meta-analysis.;citation_pages=1-1;citation_volume=10;citation_year=2009;citation_pmid=19118496;citation_doi=10.1186/1471-2105-10-1" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. A. Capra;citation_author=M. J. Hubisz;citation_author=D. Kostka;citation_author=K. S. Pollard;citation_author=A. Siepel;citation_title=A model-based analysis of GC-biased gene conversion in the human and chimpanzee genomes.;citation_pages=1-15;citation_volume=9;citation_year=2013;citation_doi=10.1371/journal.pgen.1003529" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=L. Carbone;citation_author=S. G. Nergadze;citation_author=E. Magnani;citation_author=D. Misceo;citation_author=M. F. Cardone;citation_title=Evolutionary movement of centromeres in horse, donkey, and zebra.;citation_pages=777-782;citation_volume=87;citation_year=2006;citation_issue=6;citation_pmid=16413164;citation_doi=10.1016/j.ygeno.2005.11.012" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. H. Chan;citation_author=P. A. Jenkins;citation_author=Y. S. Song;citation_title=Genome-wide fine-scale recombination rate variation in Drosophila melanogaster.;citation_pages=1-28;citation_volume=8;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Charlesworth;citation_author=N. H. Barton;citation_title=Recombination load associated with selection for increased recombination.;citation_pages=27-41;citation_volume=67;citation_year=1996;citation_issue=1;citation_pmid=8919888;citation_doi=10.1017/S0016672300033450" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=L. Chmátal;citation_author=S. Gabriel;citation_author=G. Mitsainas;citation_author=J. Martínez-Vargas;citation_author=J. Ventura;citation_title=Centromere strength provides the cell biological basis for meiotic drive and karyotype evolution in mice.;citation_pages=2295-2300;citation_volume=24;citation_year=2014;citation_pmid=25242031;citation_doi=10.1016/j.cub.2014.08.017" />
<meta name="citation_reference" content="citation_title=Red Deer. Behaviour and Ecology of Two Sexes;citation_author=T. Clutton-Brock;citation_author=F. Guinness;citation_author=S. Albon;citation_year=1982" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=G. Coop;citation_author=M. Przeworski;citation_title=An evolutionary view of human recombination.;citation_pages=23-34;citation_volume=8;citation_year=2007;citation_issue=1;citation_pmid=17146469;citation_doi=10.1038/nrg1947" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=H. Ellegren;citation_title=GENOME SEQUENCING AND POPULATION GENOMICS IN NON-MODEL ORGANISMS;citation_pages=51-63;citation_volume=29;citation_year=2014;citation_pmid=24139972;citation_doi=10.1016/j.tree.2013.09.008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Felsenstein;citation_title=THE EVOLUTIONARY ADVANTAGE OF RECOMBINATION;citation_pages=737-756;citation_volume=78;citation_year=1974;citation_issue=2;citation_pmid=4448362" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. L. Fierst;citation_title=Using linkage maps to correct and scaffold de novo genome assemblies: methods, challenges, and computational tools.;citation_pages=220;citation_volume=6;citation_year=2015;citation_pmid=26150829" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Fledel-Alon;citation_author=D. J. Wilson;citation_author=K. Broman;citation_author=X. Wen;citation_author=C. Ober;citation_title=Broad-scale recombination patterns underlying proper disjunction in humans.;citation_pages=e1000658-e1000658;citation_volume=5;citation_year=2009;citation_issue=9;citation_pmid=19763175;citation_doi=10.1371/journal.pgen.1000658" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=T. Fountain;citation_author=M. Ravinet;citation_author=R. Naylor;citation_author=K. Reinhardt;citation_author=R. K. Butlin;citation_title=A linkage map and QTL analysis for pyrethroid resistance in the bed bug Cimex lectularius.;citation_pages=4059-4066;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. C. Frantz;citation_author=J. T. Pourtois;citation_author=M. Heuertz;citation_author=L. Schley;citation_author=M. C. Flamand;citation_title=Genetic structure and assignment tests demonstrate illegal translocation of red deer (Cervus elaphus) into a continuous population.;citation_pages=3191-3203;citation_volume=15;citation_year=2006;citation_issue=11;citation_pmid=16968264;citation_doi=10.1111/j.1365-294X.2006.03022.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Cytogenet;citation_author=A. S. Graphodatsky;citation_author=V. A. Trifonov;citation_author=R. Stanyon;citation_title=The genome diversity and karyotype evolution of mammals.;citation_pages=22-22;citation_volume=4;citation_year=2011;citation_pmid=21992653;citation_doi=10.1186/1755-8166-4-22" />
<meta name="citation_reference" content="citation_title=Documentation for CRIMAP, version 2.4;citation_author=P. Green;citation_author=K. Falls;citation_author=S. Crooks;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=I. Gustavsson;citation_author=C. O. Sundt;citation_title=Karyotypes in five species of deer (Alces alces L., Capreolus capreolus L., Cervus elaphus L., Cervus nippon nippon Temm. and Dama dama L.).;citation_pages=233-248;citation_volume=60;citation_year=1968;citation_issue=1;citation_pmid=5721324" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=D. Haig;citation_author=A. Grafen;citation_title=Genetic scrambling as a defence against meiotic drive.;citation_pages=531-558;citation_volume=153;citation_year=1991;citation_issue=4;citation_pmid=1806752;citation_doi=10.1016/S0022-5193(05)80155-9" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. Hassold;citation_author=P. Hunt;citation_title=To err (meiotically) is human: the genesis of human aneuploidy.;citation_pages=280-291;citation_volume=2;citation_year=2001;citation_issue=4;citation_pmid=11283700;citation_doi=10.1038/35066065" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. B. Hedges;citation_author=J. Marin;citation_author=M. Suleski;citation_author=M. Paymer;citation_author=S. Kumar;citation_title=Tree of life reveals clock-like speciation and diversification.;citation_volume=32;citation_year=2015;citation_pmid=25739733;citation_doi=10.1093/molbev/msv037" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. Huisman;citation_title=Pedigree reconstruction for SNP data: parentage assignment, sibship clustering and beyond.;citation_year=2017;citation_doi=10.1111/1755-0998.12665" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Huisman;citation_author=L. E. B. Kruuk;citation_author=P. A. Ellis;citation_author=T. Clutton-Brock;citation_author=J. M. Pemberton;citation_title=Inbreeding depression across the lifespan in a wild mammal population.;citation_volume=113;citation_year=2016;citation_pmid=26979959;citation_doi=10.1073/pnas.1518046113" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. E. Johnston;citation_author=C. Bérénos;citation_author=J. Slate;citation_author=J. M. Pemberton;citation_title=Conserved genetic architecture underlying individual recombination rate variation in a wild population of soay sheep (Ovis aries).;citation_volume=203;citation_year=2016;citation_pmid=27029733;citation_doi=10.1534/genetics.115.185553" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=M. Kardos;citation_author=H. R. Taylor;citation_author=H. Ellegren;citation_author=G. Luikart;citation_author=F. W. Allendorf;citation_title=Genomics advances the study of inbreeding depression in the wild.;citation_pages=1205-1218;citation_volume=9;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=T. Kawakami;citation_author=L. Smeds;citation_author=N. Backström;citation_author=A. Husby;citation_author=A. Qvarnström;citation_title=A high-density linkage map enables a second-generation collared flycatcher genome assembly and reveals the patterns of avian recombination rate variation and chromosomal evolution.;citation_pages=4035-4058;citation_volume=23;citation_year=2014;citation_pmid=24863701;citation_doi=10.1111/mec.12810" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Kong;citation_author=G. Thorleifsson;citation_author=D. F. Gudbjartsson;citation_author=G. Masson;citation_author=A. Sigurdsson;citation_title=Fine-scale recombination rate differences between sexes, populations and individuals.;citation_pages=1099-1103;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981099;citation_doi=10.1038/nature09525" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=L. E. B. Kruuk;citation_author=J. Slate;citation_author=J. M. Pemberton;citation_author=S. Brotherstone;citation_author=F. Guinness;citation_title=Antler size in red deer: heritability and selection but no evolution.;citation_pages=1683-1695;citation_volume=56;citation_year=2002;citation_issue=8;citation_pmid=12353761;citation_doi=10.1554/0014-3820(2002)056[1683:ASIRDH]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=E. S. Lander;citation_author=N. J. Schork;citation_title=Genetic dissection of complex traits.;citation_pages=5181;citation_volume=265;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. Leitwein;citation_author=B. Guinand;citation_author=J. Pouzadoux;citation_author=E. Desmarais;citation_author=P. Berrebi;citation_title=A dense brown trout (Salmo trutta) linkage map reveals recent chromosomal rearrangements in the salmo genus and the impact of selection on linked neutral diversity.;citation_volume=7;citation_year=2017;citation_pmid=28235829;citation_doi=10.1534/g3.116.038497" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=T. Lenormand;citation_author=J. Dutheil;citation_title=Recombination difference between sexes: a role for haploid selection.;citation_pages=e63-e63;citation_volume=3;citation_year=2005;citation_issue=3;citation_pmid=15736976;citation_doi=10.1371/journal.pbio.0030063" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Y. Liu;citation_author=A. P. Morgan;citation_author=E. J. Chesler;citation_author=W. Wang;citation_author=G. A. Churchill;citation_title=High-resolution sex-specific linkage maps of the mouse reveal polarized distribution of crossovers in male germline.;citation_volume=197;citation_year=2014;citation_pmid=24578350;citation_doi=10.1534/genetics.114.161653" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. Ma;citation_author=J. R. O’Connell;citation_author=P. M. VanRaden;citation_author=B. Shen;citation_author=A. Padhi;citation_title=Cattle sex-specific recombination and genetic control from a large pedigree analysis.;citation_pages=e1005387;citation_volume=11;citation_year=2015;citation_pmid=26540184;citation_doi=10.1371/journal.pgen.1005387" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. E. Mank;citation_title=The evolution of heterochiasmy: the role of sexual selection and sperm competition in determining sex-specific recombination rates in eutherian mammals.;citation_pages=355-363;citation_volume=91;citation_year=2009;citation_issue=5;citation_pmid=19922699;citation_doi=10.1017/S0016672309990255" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=G. J. McKinney;citation_author=L. W. Seeb;citation_author=W. A. Larson;citation_author=D. Gomez-Uchida;citation_author=M. T. Limborg;citation_title=An integrated linkage map reveals candidate genes underlying adaptive variation in chinook salmon (Oncorhynchus tshawytscha).;citation_pages=769-783;citation_volume=16;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Reproduction;citation_journal_abbrev=Reproduction;citation_author=M. A. Morelli;citation_author=P. E. Cohen;citation_title=Not all germ cells are created equal: Aspects of sexual dimorphism in mammalian meiosis;citation_pages=761-781;citation_volume=130;citation_year=2005;citation_issue=6;citation_pmid=16322537;citation_doi=10.1530/rep.1.00865" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res.;citation_author=H. Muller;citation_title=The relation of recombination to mutational advance.;citation_pages=2-9;citation_volume=1;citation_year=1964;citation_pmid=14195748;citation_doi=10.1016/0027-5107(64)90047-8" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. I. Nagaoka;citation_author=T. J. Hassold;citation_author=P. A. Hunt;citation_title=Human aneuploidy: mechanisms and new insights into an age-old problem.;citation_pages=493-504;citation_volume=13;citation_year=2012;citation_issue=7;citation_pmid=22705668;citation_doi=10.1038/nrg3245" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Pardo-Manuel de Villena;citation_author=C. Sapienza;citation_title=Female Meiosis Drives Karyotypic Evolution in Mammals;citation_pages=1179-1189;citation_volume=159;citation_year=2001;citation_issue=3;citation_pmid=11729161" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=N. Phadnis;citation_author=R. W. Hyppa;citation_author=G. R. Smith;citation_title=New and old ways to control meiotic recombination.;citation_pages=411-421;citation_volume=27;citation_year=2011;citation_issue=10;citation_pmid=21782271;citation_doi=10.1016/j.tig.2011.06.007" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=P. Rastas;citation_author=F. C. F. Calboli;citation_author=B. Guo;citation_author=T. Shikano;citation_author=J. Merilä;citation_title=Construction of ultradense linkage maps with Lep-MAP2: stickleback F2 recombinant crosses as an example.;citation_volume=8;citation_year=2016;citation_pmid=26668116;citation_doi=10.1093/gbe/evv250" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=M. Rocchi;citation_author=N. Archidiacono;citation_author=W. Schempp;citation_author=O. Capozzi;citation_author=R. Stanyon;citation_title=Centromere repositioning in mammals.;citation_pages=59-67;citation_volume=108;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=L. F. Rosin;citation_author=B. G. Mellone;citation_title=Centromeres drive a hard bargain.;citation_pages=101-117;citation_volume=33;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=H. V. Senn;citation_author=J. M. Pemberton;citation_title=Variable extent of hybridization between invasive sika (Cervus nippon) and native red deer (C. elaphus) in a small geographical area.;citation_pages=862-876;citation_volume=18;citation_year=2009;citation_issue=5;citation_pmid=19175500;citation_doi=10.1111/j.1365-294X.2008.04051.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Slate;citation_author=T. C. Van Stijn;citation_author=R. M. Anderson;citation_author=K. M. McEwan;citation_author=N. J. Maqbool;citation_title=A Deer (Subfamily Cervinae) Genetic Linkage Map and the Evolution of Ruminant Genomes;citation_pages=1587-1597;citation_volume=160;citation_year=2002;citation_issue=4;citation_pmid=11973312" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. S. Smukowski;citation_author=M. A. F. Noor;citation_title=Recombination rate variation in closely related species.;citation_pages=496-508;citation_volume=107;citation_year=2011;citation_issue=6;citation_pmid=21673743;citation_doi=10.1038/hdy.2011.44" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Stapley;citation_author=T. R. Birkhead;citation_author=T. Burke;citation_author=J. Slate;citation_title=A Linkage Map of the Zebra Finch Taeniopygia guttata Provides New Insights Into Avian Genome Evolution;citation_pages=651-667;citation_volume=179;citation_year=2008;citation_issue=1;citation_pmid=18493078;citation_doi=10.1534/genetics.107.086264" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Zool.;citation_author=A. H. Sturtevant;citation_title=The linear arrangement of six sex-linked factors in Drosophila, as shown by their mode of association.;citation_pages=43-59;citation_volume=14;citation_year=1913;citation_doi=10.1002/jez.1400140104" />
<meta name="citation_reference" content="citation_title=The Evolution of Sex;citation_title=Sex differences in rates of recombination and sexual selection;citation_author=R. Trivers;citation_pages=270-286;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_author=N. Vincenten;citation_author=L.-M. Kuhl;citation_author=I. Lam;citation_author=A. Oke;citation_author=A. R. Kerr;citation_title=The kinetochore prevents centromere-proximal crossover recombination during meiosis.;citation_pages=e10850;citation_volume=4;citation_year=2015;citation_pmid=26653857" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=S. N. Wood;citation_title=Fast stable restricted maximum likelihood and marginal likelihood estimation of semiparametric generalized linear models.;citation_pages=3-36;citation_volume=73;citation_year=2011;citation_doi=10.1111/j.1467-9868.2010.00749.x" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. V. Zimin;citation_author=A. L. Delcher;citation_author=L. Florea;citation_author=D. R. Kelley;citation_author=M. C. Schatz;citation_title=A whole-genome assembly of the domestic cow, Bos taurus.;citation_pages=R42-R42;citation_volume=10;citation_year=2009;citation_issue=4;citation_pmid=19393038;citation_doi=10.1186/gb-2009-10-4-r42" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A High-Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (Cervus elaphus)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="twitter:description" content="High-density linkage maps are an important tool to gain insight into the genetic architecture of traits of evolutionary and economic interest, and provide a resource to characterize variation in recombination landscapes. Here, we used information from the cattle genome and the 50 K Cervine Illumina BeadChip to inform and refine a high-density linkage map in a wild population of red deer ( Cervus elaphus ). We constructed a predicted linkage map of 38,038 SNPs and a skeleton map of 10,835 SNPs across 34 linkage groups. We identified several chromosomal rearrangements in the deer lineage relative to sheep and cattle, including six chromosome fissions, one fusion, and two large inversions. Otherwise, our findings showed strong concordance with map orders in the cattle genome. The sex-averaged linkage map length was 2739.7 cM and the genome-wide autosomal recombination rate was 1.04 cM/Mb. The female autosomal map length was 1.21 longer than that of males (2767.4 cM vs. 2280.8 cM, respectively). Sex differences in map length were driven by high female recombination rates in peri-centromeric regions, a pattern that is unusual relative to other mammal species. This effect was more pronounced in fission chromosomes that would have had to produce new centromeres. We propose two hypotheses to explain this effect: (1) that this mechanism may have evolved to counteract centromeric drive associated with meiotic asymmetry in oocyte production; and/or (2) that sequence and structural characteristics suppressing recombination in close proximity to the centromere may not have evolved at neo-centromeres. Our study provides insight into how recombination landscapes vary and evolve in mammals, and will provide a valuable resource for studies of evolution, genetic improvement, and population management in red deer and related species." />
<meta name="og-title" property="og:title" content="A High-Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (Cervus elaphus)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/8/2859" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="High-density linkage maps are an important tool to gain insight into the genetic architecture of traits of evolutionary and economic interest, and provide a resource to characterize variation in recombination landscapes. Here, we used information from the cattle genome and the 50 K Cervine Illumina BeadChip to inform and refine a high-density linkage map in a wild population of red deer ( Cervus elaphus ). We constructed a predicted linkage map of 38,038 SNPs and a skeleton map of 10,835 SNPs across 34 linkage groups. We identified several chromosomal rearrangements in the deer lineage relative to sheep and cattle, including six chromosome fissions, one fusion, and two large inversions. Otherwise, our findings showed strong concordance with map orders in the cattle genome. The sex-averaged linkage map length was 2739.7 cM and the genome-wide autosomal recombination rate was 1.04 cM/Mb. The female autosomal map length was 1.21 longer than that of males (2767.4 cM vs. 2280.8 cM, respectively). Sex differences in map length were driven by high female recombination rates in peri-centromeric regions, a pattern that is unusual relative to other mammal species. This effect was more pronounced in fission chromosomes that would have had to produce new centromeres. We propose two hypotheses to explain this effect: (1) that this mechanism may have evolved to counteract centromeric drive associated with meiotic asymmetry in oocyte production; and/or (2) that sequence and structural characteristics suppressing recombination in close proximity to the centromere may not have evolved at neo-centromeres. Our study provides insight into how recombination landscapes vary and evolve in mammals, and will provide a valuable resource for studies of evolution, genetic improvement, and population management in red deer and related species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
    <title>A High-Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (Cervus elaphus) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"oFkG3GSLLWFO1CMzhl3X0EroHAg1JpFP_We_uhhlIOI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485843","apath":"\/ggg\/7\/8\/2859.atom","pisa":"ggg;7\/8\/2859","ac":{"\/ggg\/7\/8\/2859.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/8\/2859.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2859"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2859"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/8\/2859":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MIJMH\\\\\\\/\\\\\\\/8P5PLDFKf\\\\\\\/g4MCRQJAWKnyMiwCwDAqe4EouFpl2RepcALlxkfJAWDFOKpj5YCZaoMYM8wSzOBtLV0ucVFL6LTuBaWgsn5IyGDVPFzj9GSJ8myeHaU\\\\\\\/xb+o33Y9bc370lzXjiJO8lBQcfzzLN7Co+y3PxVPtE8\\\\\\\/CbRI5F1BWb8MpteBEsBlh75ZWfQAr2LlKXOZk3Ah31gB8irnVvpk1Ip7hxqP5d2TTueCiRT6HttFNL7gGlrasuefRcbePBfrxSlyPwHfxuX7U6jmg\\\\\\\/Z\\\\\\\/g8N2rL2QwXAx4hm2D+sj8nCKRDNUUnwnisWYeyIHtmw6oEIPN4co4Kf7wi5TdIKPhwn+ual8QQ9opYPWoJIYd7R8wLYvf4O7jprLOjxxl36vWxgqvOeUjh2kKqLFY9kDM9lsBwXqxgwj\\\\\\\/MHjoRHD513JfYHlHRS71+Yo689cwJuu63Mzhv5idlSqfnGH7bve34F\\\\\\\/LFuPiaoutu9eL\\\\\\\/ljCqwY20aur1YCypvXg3xAFwYQ2Ppus2hLrZwfotSNbUpMJ3EDE5BYE7s87UR22+FwMe6AZLnzmXmGMwJYA83vQGPzHYV41KXayGR0kjsu53Xh4Spd7PgJqFwq3Iu6y9q3ZFYaII9da7F6jb3RkXgsPcvlAdrJ\\\\\\\/baP5QNE9ZY9wFP6SLUmyTfrVQ85hKn7VQYvmPNRApvtHmwbeVUZYPS\\\\\\\/lXZu80Scpm\\\\\\\/cAZ5exgBC3t2hEnCaeBdqNo4Kdccg9aUvrDBn2ne7XOmVHU5MdUlQ7JWvtw034LMxpHmn93Win5Ajz5jnoJbE5OFefzOmPAJbPdqpEoB34pY3F7NCi9mQlbfOBGbS91rPuOgzWVRzqVK8f9GFa\\\\\\\/sN55xYaC290fqnJ+iARfLdNHFC\\\\\\\/VLi0ESgzT8ZVSzUz9ScKNQebicK82mOcsKQNQ3rHy5Bx2kZPTAWpbkrZsWxtWpxLDkh+TSKLUWH+2wwqR7oGwexTI456T+QbN2AkLqUTqPbxboBgRyZyfQS2uJJTdhJHOP2lef6JB4edDVeiNBOysM6\\\\\\\/FluyRADnWfSlurJjslt+k0KzIhxl+xz\\\\\\\/+QUgBqff4lx7\\\\\\\/ZM\\\\\\\/wS+VQ8aQ1sOis+ToyYAdSrERGwiELFUwLnoiGAA4lhp2RdG5YsooDuAZ5D9oEugTMQqasJePEavXVwTQG3QeXNeBYuRTnSN488PdfijacVqJCoB+SHAfviQsRG5mZUaz1OipCjsmtiUAf1JdEYlGD0sdvWA5VVnx5mBsTduhkaqoIbTqGQWHrpYtd\\\\\\\/JgZl\\\u0022,\\\u0022iv\\\u0022:\\\u0022DKgi74N3KAVEk\\\\\\\/r0Hne+3A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022279091ea187302e5fd5d16fd9dabe70f059ca16a6b510f1d14e9f0ee2a03f11f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bKVy6vlrhIkOEkYG16MYQoCV9pV+HjRLxc\\\\\\\/yTCoammYcNEOsuSToRYwP1H1cVd3tgZKO3KZaCiz5QeEGQ9oyd5q8upHMOlgnZ1kEjeatz0mE6mrT8tSmYpPiYOx6\\\\\\\/WmGOpYTkSlo7HRilAFfo\\\\\\\/UHdd48eXosvmwdlfvSEYnKolWuURHDVve2+0L+FwJuCZsL+nbpaQPk1LDMithHZ7Hv5gGMffQcyklC8PRgAEvX4\\\\\\\/ijVkddkrvbeMlu5pMG4bhBx4ULClEA6p2EP1jOsEJ4Lnx3t4bmNPlD3j5qGZkViIguKdrV3eDDmlq581LehUkIB6Dz0pxvXumdrsHwzoS46PUnUh1Km2004Ze1izvdg3DTqvH2OAh5DrZSspr8NcP1Cm0hH5+UWCqP0RNlAbItwlvLrE0vKJC2YFdoQf+czxSWKWcfL8GZLWWDzoVlpfZxsB8JYJ4qy3gPF467loJxo172TUeNFBi4\\\\\\\/jeVc98G7VXAPGGYutUf8UwVFmeRRWuuzc9RzyxCKjUd6KG6qz5vpoR7UxNtbRy3aDuzZfEs5XAhQHc3WxLhAo3VDe9SFVQBL9JCEjeuLk1v+HO0UP\\\\\\\/DFkAsXNPm3om1zGxUF1UaZX8LydsywPPN3O8OtpFEpu8ok9sva+q+lWN1ZYpRMVLHmh0Z1ZDaiCcdbFwbsaNr9ndCWKBOnNv7QSdgys+RMbo3ToZy4BMWwxo3Y2ETl17Kb6xvcjxHFtixu7D+NNf4mu\\\\\\\/B1GxcAVc\\\\\\\/qcoYQ\\\\\\\/SFyaXtvH4ifLHzvq\\\\\\\/WhyY1NzC5gHl\\\\\\\/ug1vMSGtgd9A+XP1L8iJNirhhYRxR8X3fjNSpPhplyDtEFvtoKCn9N039OdwStFAQ8BRZemoxOKL7qXX0XsM1DbhrPU+c3TDLMSwaXgFX8X0a8PFzeqgf\\\\\\\/Vgq2aUpVCcDB5sJqg7A1iBUKFDn5wlDXJPPtsrT8dt0ed0TlEquQPDk9\\\\\\\/Z5PcqAk6wfJK16PTUJgCksLfE8OTDksUOyhuU20l4EOl60a9b5Lfx3WUmUN15NvjtcIeQnozeJax3A3S0gBmztF0x9eg7P0bOQ164FLxnPoT8sqI6oBJbt1WhVvl9KYOpB2nO79MSHquhZU\\\\\\\/QMkPShLk+H\\\\\\\/iQ5HKMZ6PFNKzvOEf0Q5WTAq8LniIGRo6e2UdMVp6WF0da7lCXahMAUdGT8MMmcmVxZbhzjGZzk4jSyQ7xnR3V1nNp36LckMyLGUn2vq9zKjQ2rjmgev6no3wdMgAfjbjshyx4PnmG0jB1PwgR3XimMr7iRbeYRc\\\\\\\/ObKLUqIjZteL8ARiUZImc3DTZgWhrcHATm2lnq7JpdeH8eaCAknTQFEL\\\\\\\/Tv6B9xV7iqugbjJbLGDkfTqiI+oik2lPNy6Tpq5rwmwXaAbtD+x\\\\\\\/LzgPHniohsUmrdrdYaGfh0ovYFy6VN3ZHpfSSSxy0phstXa8jpBZzzCbz0Gv0O4WfcKkv7rxqhGWXunShiVBHTlORQLJnHxreeki9cl4N8tIOQBmiGWUCa9IXRu3Tu5EatjIxCSaEkTiIZhtlbqCmaEumjZAl2Xyxu5MGfFTQtPKTOHUFkVoul1I0Az8tkHJ1ATXIma1xL3ILp6aeu0fjKYdjB9POGxNpXK3Mh6Jy0qHS4\\\\\\\/zffvaJnEbrhmNih+Fr2+PG6ZOIjQuE1dl1MVrD7oHrRxlZaMX2\\\\\\\/BtXAfNUNfV66J6ZCE6qei15sIv2CBew\\\\\\\/SARSayf8QO35IE5cFD6Lxtj3fvUHPLWbovu1uA+9C7v3\\\\\\\/YuJnOEw\\\\\\\/p0VcREkXAcjNAD1kjV\\\\\\\/DF6ZZjsRD3ICM50v3jva95\\\\\\\/65CifFFbFW4TNjzv9NKtxpgV5sOkYVSXl2f6Vu137uTqlqI9iXFflLDRB6Inup0abTGa3uJwzghGiFGAonD9YIYs4LgYIx9Wz3o3lssyaeGyJIJLrwN5kvoF2p2ucCS1vZJjfiLrb8Gj\\\\\\\/AoHjaZKcRlbN3TKOmwle5cHCex9gUnE+4y1xNt5lr7E17bccKG61eNdV2g03sIMQBVMYDJ7A2vJWT5+8CWl0wy5o7l3tYuDJOp+WdwPrXf+LPUnbzf0qLB\\\\\\\/bs9IbG0Qvc7AKcHd4VFdHX+NYLF86w7+v51zrmP7FjL+CwuALbdIBiTO7jBquW6f3Qxnwvm0LhuXE4a1\\\\\\\/yFMUnTiNv4FBbo2v6t1UVW8z5qxj4nexQUEpCAILxoeKG8cLdiRk17XbnCpLXY\\\\\\\/ydkUnA9e+qbiPqWWZ5BP521ject7kzEtLEZHgnRaOjAuj7d+SI0MREMOadR\\\\\\\/07+tyi+v3mYrJzo86FaaRVgBRz+RGKnNlL+2mmy7u+ASUTX+26TN7XN2fos4VENrMsCRhkI6nPwyCZTpXIY\\\\\\\/czLKWZyfXfCe2gFX3KcFd52Vfa5Ge4+rRs3zTw+eixy86BUU7kPkL6Zk2Ob3hEkK+RAzWD98jxM+aIwnff\\\\\\\/dqSzM\\\\\\\/rqO0rfZEp7Bwptiu7c5RnStfSJTxzkOfGYC+R41TUyAZG1t9dgVrZPElGinSKoWNdtOfJTH+Xlw8AKMtprP0WTAH7zogpB2+HhTboFZcrmTpEyxyfqbXpbHc4ilY0CvwSlz920omcY+pkShLPMIIedVcsmsasSQn1jnWfIzAXTkpiVEwJL59bd6PFMNXhuEPUDcw33WjWs4gglvah5slO979flmFxhrwWTEC+aUPHdKsxFqefZJsHS5ur68Q746hKvoyeDCaYyW6eVypkxZnXJrbRpcO4BJY6533MAAYevFT+4g7AefjPM6An8Ub9Fr+LTcn+I1Kqg1ztRAbRSVUSil0aBrOE0\\\\\\\/5YWF+9sSRrS38SxsGN+BiFT0dS9stkeLm9nHTyc0iM9E5yB9D81a+K8wKYP90lWFIDItveDj\\\\\\\/RB92tZ3Yhr52fy9LRJeD2PB9C55fiBewUXDmr0Volh\\\\\\\/8KK5t\\\\\\\/FKO5aCUYLw+BmUQnA530RYxd6sTroFCfOnihiih7xtWK65EU219CpVuFu5K3wTgxKx+i6d7\\\\\\\/D1tNnn\\\\\\\/8glpx2f06haNg2vvbqWP7ywh7dJra3zkJEQ2JhKPO\\\\\\\/QxeLDOm85bL8Ne76VyRZ524c\\\\\\\/5d6+XL+hqNC+o9SbEbvwu+W6RDeuqg\\\\\\\/I7bQSuEicMcQ7Igf+W0UG0ScLZkeiOvA76MyI\\\\\\\/QTQPibM0n0A1u+ZJ6WkhzkIycfUJr+6UdaKPjmFeszGkjqMYDUps0AarU1bU3EWf6q3QS4BqGaY6jCOF3pmZIZgQ\\\\\\\/C6VjBQjqOF9GjAJ1FDG7e\\\\\\\/c2UfgigLyNXTYGPf\\\\\\\/MKJak3g1vHhNL5hyBbtQlH58M4a8u9Me66FtcU\\\\\\\/2avd4K7sI8Zh8s4y+undEqynlNkaE\\\\\\\/IIWSUuUtkJwCgjsojHTdDxXS2Q3gXctA2JEjiggKg6sj4kwQZT8lrBrzBoN9II26KwjkaRl1S55x0rql7yukAv6lhhLs62HjsvF5tzpHS3nMbDGE3MsDQIUWramOYc6\\\\\\\/5jMB5qAvjA\\\\\\\/mZAYrm728iTj\\\\\\\/J1R0qVF2cc58WOZnaF60uOpaef4Ucnx1hRh8kIaMyHVWxELJWRPv0szwaoKAiuYGb7qOF04+r4lzosLkuTG0Xh0GH85gAzFL3W8o0XdMqEy\\\\\\\/TOTYc2aJsKhHb9xfQKbUqMMhMI\\\\\\\/srSKr23QWOnruXBUTtiUzR7df7pYOp3ope3ugv29TaNkh1XAhEYEw97QQcLX+4kTOCmskEEvEUxuDlIOzFzgyhg9NV3PWrgGSUqlJIQ8fQjpspL9aY5tTiNodxFMcj0bu0nUJ3gVZZUDiu\\\\\\\/5XEsOp8281jEMLDW0PjEOjNpR97DZbqSwGqG+veRrpw5T0n66UnuuE9RRe40Dhbs38nKz2nRG\\\\\\\/ghz9abUm4\\\\\\\/5uUJ3b+Rsa9A\\\\\\\/5oM4wACYPKms11aEfoEbwJ6rQbk\\\\\\\/JakzEFbbcZA6iNHqxE2b2pXQXDQRUd54o1aGO4ZmRGOUXPW2\\\\\\\/Yjrh\\\\\\\/VB2aJ49WVYDa7KbxaLtYjFOJ5IJxWp0HE0pSpm1ZBYvVpOVLy6WbuOUP6YikwfU62\\\\\\\/ACyoSq+tadBPVKu1mEgkpfSOgDRAYx2pmWd\\\\\\\/+nvud+eKFl04Ct2V7s44ER4ix03zF9fmOc7MEsrgtaCA6ZfA2Jzl0J\\\\\\\/TH2fUJFvtxdh6H5ZCOAYIFpbsckS62H4ROV1c+nqP\\\\\\\/NIo6U5tfm5MLSZUlBX22\\\\\\\/xO8wti5qpJQObIBP8CC25fYXTKw1qULu1aoq0YgRPUU7Pxp4GzAdgHNrFoaqYrnCe0WLcPViKSQjMBPeagPEso1XYG4OSPkvR5RF\\\\\\\/V7zrUvBw7Z3wqa\\\\\\\/U0sgx2yl1YxpUrimPIbfmwe1u4T0bbLmtgbLxUmQ\\\\\\\/AZf8aHRE3sG19cvyvMdIV0BfDExjccNA1pMy\\\\\\\/XoRvWNi3VtGMsFtgbRRVIv6YonWG9yk2NdzLirvAf93XEEm7LRHOGteTjYE3VMFqUlGj\\\\\\\/LRaojJkGOX9L4iyoGN3vquq5W+S4QNE97rDorxdwDIEVrR4a64Fs8W6QvXIDNwmHGW2C0knuQN+smSySOiCMSJ1Y6jWAo62MRu13bUT0wcOy9SrywrMrLElT3oo3k2D1w3EwFlWtNI95OILgemAi1viFCL6Y+rB1xbFqfE5S11gX7XT\\\\\\\/R+R8HVRpth9T9fsSPZZsScuzeF4lVTqS0Jer4May89Msrn8tUbRV3bpxcRx9mVm8OyeYmYxNXzxOT\\\\\\\/6x27rsz0S2kE0q3zV4ltqhl6QCKknD1SXJoLKJGKhECO0+243w7jACGkgjFbr7Gj5jO8P8iGDovgpXhDUDHUszX35kmvLXVIZ7Zeqec3jPoM11cvUr9AfbdTBU3qDiH0AxS7lXNeqnpjIq7mhiLc2eiX8NCo78OtnXFSqmdpPQiavRXwKTAKDWInILYymw5lt17DxTgerv1NFjqpinQOgHPO87tpPkBOsFVTdNMTJOYyUKJT1b5oXRE4tmoBUu5QwUaem8T+jckeC54gBh5Nu0G7Yngsj5qlzQKWWHto3e3ayC2uSaWdxIMBXwS8s7zLpIYN13pC919Pe6ZKvjOPa6S9wTo\\\\\\\/18AIGNac3TrUk5nLtClBNLpJ6bkNC8iuqgKF\\\\\\\/K1ZeH20N8GTWL2HvlXgZB59NyD9yu0175yEqI4wvxE5mNVKQUgBbJn5tdWiktwWM+cR2h4tFvkg3+MyAO9CM2TVKm3ArrAwrwWSb4XuJEuD2uovSgbt6Z5LCShLLGEtaHfXIbYRDX709k250p0eM+5GQtcFBCA+hjVxy3In3pGrr1a5JPlrIuGI\\\\\\\/+f0nEFnXlxWP042ripunDc6FBUw5AHWWm1drePZpLR\\\\\\\/gM9XktZri17LFv0hOoXg0V6PtlQZcg2cJZRrxu01U91z9641+GLvCwKtXNUiXEyCLe8er9M5RmsS6dnwxASJ1tu33KCvWmRJlnASfdhcXVnwlI0BlUfroInrE+EWL\\\\\\\/03F9c\\\\\\\/IGrpqZr2Xq3L9YkMpmlPSRXObzOZxyoyG+GQZiGSur4Pe8a8H7\\\\\\\/gmdLQbnrWGfs13GrpdqpPgCTfQ7tusZCH4QrFM0D7VigdUYtzDLYbnqNguvLgHo4CVvQatryc4rskWTjQHnI3DKnD0U8fVl\\\\\\\/ss+7e+RBbsv1IAmehUbX80guMyk2KRj05WKODh9cxafhloOV35VLWTgOEeAH\\\\\\\/sErSUfDB4Zu7ICcdYG\\\\\\\/jGitXIOO+ZPDaGpoqRo\\\\\\\/fnesZpeeTl7jqPOrruLBI7dIaAqRoxLFwfrFo5U4PCPDSjuNzLUrKOvvu0cYjwLwgMFPpNls2\\\\\\\/gTuLvOAJzioz1rTwtK3bumcWnO+398opqaejp0iXa\\\\\\\/yrl3LuQCF07Y60dlkHscr0VzrFEQ6m9vOqjfghTL55FfNFrde4jj3d+\\\\\\\/m0Lc9tEhtl5bsZdJ9tAjP0Q+0UKRnsudcDGAgvgD8KZUAFoNWt2DFaURIo9ei6WbqS2MpKGDiyFjyuiGF1OnxbauWqoEBDwkwtFNtFnfAqBXU3hC6l98MmAuiK5ZuJYi69Ri61farM5OP98AFZp4iOG6kotwTk4wyxjqYvEgadYY07f1YN0PSIgTpmN9gM96YN\\\\\\\/DEHP2clBtOff2+Bo6uG3XFQXSd7wOd5HpNhVB\\\\\\\/9Txh7j07gZRnlZkK8jVOEjhLjWdGNOBynBBFS4mgvTbwDkfTq7eop9VomzWzZxSrm2QoQcLQRSMcQZaamvTLbsKuR\\\\\\\/RTO4nL+O7GUnKJaEUgIANjItKESJOQ6VYNZvo8LldDTOwbk3ARs6BH2r\\\\\\\/5BFpUUrj60e+CNUz68P9FKKEcjnO1r\\\\\\\/7cz2htICiC8utoPD8G7KFlaSuJYE6icRPYmcMfkjZiHGJGL48n1owkxiAIopY7yEYLB+MeGwc8+yhOYphTUrY3p8JS7cdxgOXrOklJbaPyJZ2hh9soUfRHuyYF+KsPoCws\\\\\\\/4KcY4dneHYttvV0H1a5ogONovTH4ET57sw5njD1CNP0Do33bDbNrPw1y1irlygSmKuTTUZDAmF+wA8YujQjxl8OqrQ3DCCNUOdNYtwNRJIPDxpYus5sSjRywWzav8h8lBmWHzHWsdxHYMOQJKQj5AVjIM0ifgCZLeM7Xlf287dgWFCrP82YRraXMPuHqfeWqjk4dSZLaY6nFuT6Wp4bXpyjsTptDlaK9cfhoWZXnNycGyYBbWRx4neTjfBi6O\\\\\\\/I+2snXl2DO43G5ga2DeZciy523OfxXTKS+TmS1KVz9UE3+ewx\\\\\\\/ZgCOrxz7txxWuPHG7hfDhqGARc7+9bSsY2WQrLlrJ6KYvV5r3VmfmErjX1vIUWimH\\\\\\\/jMUy2blDwEacKqGPfJkm83RmDx6ZClamwXvSaVArR8VgfPT8L\\\\\\\/z81K14F1jCMkFfHivhD4IIdWFhOan9v0w3beOR84GrTAVOkEh8JyWX6\\\\\\\/1Wi4EQHuQ2ArPuBiSuY3+JbK06eougShOA02\\\\\\\/8QcUQ5pf1n\\\\\\\/oSWrvSHfzBgm7EnvOi1UJkC\\\\\\\/KoLuCCXAZXwHZP+hk9YF0qQ82mVKKlOUbqho7VxCwo4n7GcoGQSsUlgOtNcmelsEFwb4JohRRtpdERW5Bb2QU3KvJOnr27p\\\\\\\/3FxNg0POhWqpEdQTYekskTexiyW8RnPGaqcgYjDOtVp6DGtHsMwJ2bt13qfQw51lN8jmy0xMXukgqJXCZLZgW8oCSy5EZNgUeray+QUg+0WquMDly8Wll3YzFdJNxf6hNDvnC7QOSVsgHWbwqdP68i0Wobwj0qKmgfAW0phyAgXuN14Ub6LIkB5PNz75rc8lqX18xWeimLKihyFJEEBIMQe57mLY3pMpiy2t9JrGXQe5clHzGxOfy6MUUJ6Cwq7d62iww+6UtzxdhQsaCqTpoPJOO938+zaWa7yIi76y\\\\\\\/BuaOxjQSXksFxirVQi5NqYAU\\\\\\\/N7uXYnxneuYb1KKL+xdR\\\\\\\/KNMpl4t6l2hHIavGY\\\\\\\/VBkBvEHh2q2tPAkfHnGmMWdinx\\\\\\\/AfhVOY2YV\\\\\\\/LleHsD\\\\\\\/muPPuT0br2o3I+3CqbDV9zVq+JjUMRIQmzDBDb4CoJnWSsPjQNo+4vE6docSVbd2Xw9eiuWfbsG7p7LZ7qE1S24kCOdKzVlGtVm9Cm2GI0CGtgYxHWtjwXTeXsBg4hx2GJGvQ0SiH+n3lVEaA3R+wmaKdolI6ZDoItQ7Amvs0Oap++auVVR2Xil23Aj3exv4odV4UQj8F8Zu7WqL26uUC9bHsXXllcmi5q5ahdkAA1svWrdJxByvW1yvMex5BPWel+\\\\\\\/iyUdhheo7kugZDAZJ9vW3JiPleW\\\\\\\/O\\\\\\\/b7gyR01t9gHo58qHkzhCxoYsYywl+C8qNEdi1zLlhyJIdkL7lY+7Wqz2Kfy6uSVQ3+5XHjgRNG76FIkhhZVudWqpb03+S+X26x44Sv7Tsrgc0x+0cZ1akPK+HYG8zY1SGH+Na7Nuq5kSENlXb6fRt03z8Zejzlk8ZHLevw883vMBYgowkAX9sBDq1vK3GeTiHcIIFISl8Tgxyl8Ei9OV5qhJWSHHWX52udhSg2MoKFNrzDtl8FtJ8a4iqmEpPfMf3vqtei+BwhoJygZZqaf0hhCeAhqG6tXL4rSCMHI0HYLOBOxxnJzlvEx0Yn2B76hvGjZDn1eBZeJFvHEC84r6Bw5N5Y7U6iIACes+WkGtVmfmy7b\\\\\\\/S3qE+SVFhz5lrU6x0Y6UAu6yccz1dnofdPNay7UCZUx5SsJmWrXOJbPQ9236vQAY9jAZfujHOikZ6GFbB+1Obs863lQSn0mjvORD8q5Y8gAVtgRJ3rzU27J8sV11rNcG9+m7H\\\\\\\/FMR3ho+WwWPf\\\\\\\/jzniYF\\\\\\\/MFswN99kiIsgzX6UCMlJVWttseFw6s6E+aciNd55laQ7PMKPs8DlNVBd0VCvPnv2YEOlRnwdkrhPOjcnb2xpboorB6JHncdNKeqn2bBTKPKt0Khz6mDr6s31SquO61wDSj\\\\\\\/FxsSu4jBGlqn6bj9Y96QePLbAzNw3mgvs7N0Hv3LNYnMW0NKlkAw8AQtmUr3RqpTjjUNG24+AmOVRlH5E2N+UEuctBuAtewWQ1uhmVMw7+lpbW9TCFMDnkoBuMnBRnJvXFsu51p+NzSuH4WM5nJBgNWRhrCY856AiKgD2JkiEvCAfjKvv5mMBsuu\\\\\\\/sfw9l3X1zQfMvF7txoTXyXCufzjo9wZLvH+2ZU1d8ga\\\\\\\/qoMtHuHtaIZsQwlvcALd6Ez1vbWyVXxyZdmxlF5aANzWdDCRnwms8YaZ5nGfilETE7NYxwFS290GAjqnoxtkIicsDaLlytlc5XUPyj5CuJSZ9nddw\\\\\\\/73hXd5hMnobu71zTOdQcTFptFljr1+AoTC6C9KzbIKkxzclOVeny99GCKWt5dqFc5NRTWjjZtKbPGtuEk7093T9zVG9mYUssTvR61ncUxpEnkW1IXoyH7fB5PB1YZQ\\\\\\\/7jcWX45JzJChgVmpPQoQie3DUF4QWKDm6D5hcE25ahyC\\\\\\\/0\\\\\\\/PypfuZ4NEv3y1ptIVoVtRLb7qHZl0loDOc5LDz5aigwlRuZ58UPcr4lxYl9FkEZRgoWR9nQlbKhDlieUhdAQl\\\\\\\/kb7ShkX9hFMJlK6ecI\\\\\\\/puZ+AdW4CUD6u3Xjl\\\\\\\/fEe156X2jRp+KYjn\\\\\\\/XdsTObtJpHnC0esR\\\\\\\/rZi\\\\\\\/rqSZ5mdXWQO2tfM1wrSNr47KaXQNO57GtNSZahhAgvR5+oE7u0FrgKp5ELAg4xKEqm3PpDhE2WhSDtoAbJNbD8SOfrXaWySpylMYfEVdNOlojxZcoi\\\\\\\/DdB6Ql\\\\\\\/QLRFcPPlZh2GGUMcw4o+6qoD0AoTGda3BF8TpxQj1rtQejgx5cp+77\\\\\\\/MyWBByqlIwkAQCJVQnkXgi6jX7imf4vosxcN\\\\\\\/SKRhqJG0BmbhicjyVnDasSbu8ivC04wPHIlmnUVFjfs1ya+cVLrHR+WOShsmeL5QpYnMb7Gn9Xo4mce+ZTpXPId88JaNK5NqVzD13Egrh8lb19GrjAJzzz97yDpC2BAzi6aVeqn4S1ocd4DUd57OlBXMGiZ5fMrxPBJp4K3URu+rs2bEeE+Ifs5VkCfIWhfn9fgY2qnQuYP8oNHHaET\\\\\\\/2aqigx\\\\\\\/WZVqkuv\\\\\\\/lY\\\\\\\/8GgJnrlHW3zH5gGNLLfVDCNLCxz5IpXQLcVX\\\\\\\/Iyi4x2xIPiWHdCJTScUVqppIEz26EZrsRNf1f05TSeaM6JnphLPIFfLKZAkHE7nDnlzTI7d4colbhdT6lcJuztq4SKONgL+Yc+6or3wpy\\\\\\\/dhA4vddc8uPb6ngJcqGQiKzVhmw7oubrQKtorAEL3\\\\\\\/BhjvcMOj0jLJ7BDiB5B4Kr1pY8dFd1+xmA3BG8RNETvtDxMp1QhQ+tyGMksB\\\\\\\/rua1k2+riCnSd\\\\\\\/yUV+ZQRhJyYc8tUXb1g0VX3040yFPKlAQ89pCtEMSG5osmL2EQsiHndHiPn2TNL\\\\\\\/A2IQRbEOZ4WFS83QkXSGCChrbjuT53BGeL28JLb28DBIbV9+ILulBJhetlNY8NqvuiEDuWoUMjDtj7wBeF7ZIzAAcBGTdWbaexqaQ2hpq90\\\\\\\/vf\\\\\\\/p5FhIv4ESGtHEG+0x57bvNI7dEkbIHZpnt\\\\\\\/cdhtb2W9MZpQNy29e3HEG5BrqkJciQ3+7y0NNvqJgHDJCOtEz240dNpU1gkiwXyFGk+JKtF2\\\\\\\/Z6kz4DBirjA1da1hiQAjb8ciLPjSo3Nixot+v9YBqQZYix5n\\\\\\\/yPYFEJBdX4nsxZh7cKph9jlCSB\\\\\\\/6EC5QdYqz5d2jPuB9haU5WG+R2H1CLFsZwH580y1g5DiOR6l40FkAa6hNZcxdTohy2N+eAF+WWElvgxilHJaBB20BR6agLX\\\\\\\/9V0NYj1SfNNbjOIHpHwG0x2W5GWJ+pb6JuQeOwqnc\\\\\\\/nezeKnfjeVRONghEBt5m74qDHAeTRMtG+VMqIzmY7rghlmrdZupSj4guWRxkB0hcNY2kM2wnZaK7sV3ojb+YL4nXa73\\\\\\\/HOhcOF7Ll3hqGj5Ml9eX0c2B6o1ZcngXIkOJtv7ssUsAaVgTgek\\\\\\\/suAY0IceA7gGkFTZHJUZJKFZgAccyzAU\\\\\\\/1XzSwxH936JkwY4I\\\\\\\/bdPzMP8KZDT4eoeZvWFTGhuxBxfiC8Fjt6ZLwaHUjWphAlqv\\\\\\\/GhJ3M8SAr73KxhEFe82hU7t2Ge4TdlVzX7dq2xHJ4PDlx\\\\\\\/0dauNjiTqrHzNfBj7WaDMVurWoIthrpcuufwIrM1ulMtvbG\\\\\\\/DhqRCGt+57Sr63K4fxp\\\\\\\/cDM5g81bPQLkG+B1EmeVZavjnyD+vIkLKK\\\\\\\/Mah35XDoU1yqzqU1zVUINxSlzrEM73EafpWIZGhz+NCAKQDgf\\\\\\\/cnz0D+czKpNyw7xgnqEAI2ZAAvVpAngNTfwgDjKEE\\\\\\\/S5ZCAASluonOELlyH9jwVai5APJBGTEBSeCGQAD0InrU+kPuOglPUbYSt61\\\\\\\/k+Y0Cz070U4NhdkKrXQ4eVOQprgqvrQdhiJEOOSP4dipr4PlqfUrVWNyiRQcHb9ndQyQyh5UXNtJ8CuiIygnmxxZGUJ9iAngpJMdHaRsw7Favzs+do42a33w7rNi\\\\\\\/ARPY9h20KFhGB1oCmjXkxAuArhCWflCBiG8jQGcylo9ra3IjXKH+HcqnXpr+92tbd+nDIe2SBAoDwjGqPNJVu5MUTRpRODrvUcDo+cwGfbHfyg5u\\\\\\\/RQIxM2DaXUL0jsZ7Hn35OXYyiIn1tNLQDR3berOn1I+CFGef1CgbcBBW+HOFlUvkNGKa7ksCZlZi3m0LnYdLw0epSQJlZFMHJqNDLyDz3NU5Qdh5Bo6ZaaOFl9czVBxy5\\\\\\\/oX4iAuTKKoE\\\\\\\/g0j7iQP+ORcNdee\\\\\\\/KZnywQRLHVKkXTXVe54FB+VY7er1RTDjdZgf86tAjMqhMFnf70fUKEKKp4Hw\\\\\\\/EreF5IViPPx0HaQPVvZY+b5dmbtDsVoIOqBYTeB95jIzExp5PEpkZMRzq\\\\\\\/8qjNGW\\\\\\\/KwimKuZRojAhz5E0E1YErXWuTQlAz4KlB\\\\\\\/ojXpB+Zoatn3a4t0oJjiUEEnoiJXpkr0Gdp9vNkElLumVmXdx3EBUnAVid1DoJkkhZ4aYknGN9N2vhpN0ED5xeA+68P3dxXk4o4+m0HYYMJRpLmiMAJ67Z3x7MDk7NJUC6Gpfn\\\\\\\/jsihxG5QMbiJqqug28I70iXS0v9rxGgsDpeU0XLcmOsTDw6VTUUXcwxWdDBNzmOWAEHvAgpRikJP340wPPxRUQuQ1H\\\\\\\/22J+Ip7tN8L7AWpUxV44pazewNmXSfVItlFQou\\\\\\\/5ws8du+l0j95CC4FqjU5+26GgJTwj0o2SSLcWNd54eZIHEN4FumF3SWwYYeJxJui7+AIdc61Nitildqo49pwS0MmwtOhvdFWd5avxGKPRw6W6fpmxBA68dwFZRtPCCDVqqO2Nt\\\\\\\/yT3eWdYdKxKv66kyY6h9FCiBZC7sJAojEQXP8r98HUgTHchkKZd87IZsEOVsEMZlnW1LEEzMwVHF2w2S1HanBMQbLv92zVstJAQr+vEkgVdHUlfo20MbPL2eKoiVgNjHmSGNEAFXaL2KD5rVIkOrvuTUGAwkiEEhyDPO8Dp29j9Phl\\\\\\\/23Q0fqZVvhn90Gc9prdGqbgePTuNytwbwHS2a9\\\\\\\/wIegblun+GMd8jHcPagG+e7QYWx4JKys2Te6OLa162+Z60y3KgWKM5W3lgDU1vzDZFO6EjiXsjx4s6sHMvQ7VcwndKidmGKm2EALKacX+p8++94Ag\\\\\\\/CdYaqsteZ\\\\\\\/o3ISKHb589CTbTBGb2efoVs+CEa7s\\\\\\\/w\\\\\\\/WEHS1NygCqAobrTzMFRp4ZEY9xnkKbD38PPm5fWkNna5n+i3dQqOcAdJI3gMrj03o7Wzo+mKOMJLARKaa8VF35OCP5eAxS4UPh2jMCntsmoND4Pwb2MJApuHGrcxQ\\\\\\\/xpkCoI64EbanJXc2WcHraD48lmBfiduXmwPDJnS2JKn61drsOkcvNhIWT12uBUUazqm0u5lpS0MLo5xKbl0uEUyvL++Mu+2KecNc0v43Mb60COVALK7XFMlKpvyOo13sLMn89Hb8Kkz\\\\\\\/cdjlSxJqng8ntLQ19TlLlm0VmEejm6fZRdmmAUrMxwQfS1tQUIvb3PrEB7DL0VBOpHCE8GFvZliKrzRBLkE2DRqM6DGbIvHasfiPz6w6wSOysAhEW3ZpK+7oaPJOIuvrvVuV0Xj6XjldBxjqdjrAj2eGGIIclPqMOv\\\\\\\/py\\\\\\\/qo9FNBc4JnRDWCwx0ChwcKdS\\\\\\\/TXlwlQLPHG8T47h6qxKaWi82UG3OB6ydZ\\\\\\\/3Yf903G2V6AuZ3PqOSZUHNFHFkH47Ab3br1VzN+drmkR4leBbFuM\\\\\\\/bZBK4483dCzcrDyacDoOatWHrvrlbkQXUnLQEPpiDD4EHF2nbyXgmR5qCeoSyaWXCYNPBGj3M2jkx7ONTrvszUcIYuefzHr5lTswzQ8SNFaoJH4X7CiZ5lLnK4Wxnbg820GbvJ4nXWfHW7mn4o4KULlKkg70dyY3qOROwDtK\\\\\\\/txT25w72LaXOCgFFADhA4Gid903Sw+vcdt7Ex2EIoCpvD\\\\\\\/snLox7zRgtRVWGMgGs7rkYJ8p\\\\\\\/iDDIf7STWyXGHkxxr1PK90UUCQp6A3Kxby+pvftqquhy6gtFWDla68pwwLZYRa\\\\\\\/2VxbTo5XtBeA3Nwm79K\\\\\\\/TRVSGyRAYr\\\\\\\/51KFHYY3LBzAQtWn+3AKN3jglpHCb2JnE5Jx1RyQwQQFEwthmJs12i\\\\\\\/N7R1ppIUKzjvt7CyT\\\\\\\/O6T+hxiaC6xbHkIMvWWIkbaiKFI1CHT7aSmgm0wdFOIRCAkXOQh4TsdXP86aBrxdjudXe1GQFbNsc1H8ZZ\\\\\\\/8K7oYbBtCKEayqN5NWWld4tqDu9GeaxYycUSd8KlbVla2EsyfmOj4b2w1zxrhI3d+LW11YosihzTZ4C1NJBrO1MJ+dfWmJ4Zeu\\\\\\\/BDAHLOl0ZifzoNTLDrTwvsjXrRAC30qpAsSIeiKIanUP+M1FyJtTIlhhJ5ahV1PWuxwLsP6jqWCOLndgRaF\\\\\\\/c2BM9fVPNwv3QFNt1NwPGzxY6VrfpHL\\\\\\\/DmHAzeAnxol2gP3EH7Wo3M3YIPlVyRBSkxaxqPu4yicGMy7FG9f57GZv3N1lTlhrBMuhlPFfgwd2xqSjjkVFrPyZbP6sn4Tzx\\\\\\\/8I9WOomFV4u7FDdcz7Cq2XB7x6E3wxSo069JvpcbZm50MT4AK+cmRaiMc+tIXTvoebO1m+mTMIEg7op5bpClnKlzFvKL9nQrLD\\\\\\\/f0hWCfo5pYVfVGjRZ3S3xiJ3tI4cnNJBS\\\\\\\/Kz116rMsLXE58ykGA33cv\\\\\\\/xdaNKnMCfehMuDPhUlAg0\\\\\\\/Kfe6MnKXyyLWiBqG\\\\\\\/WLCRv5hijF42LUARuWUXyenLEhyIk+SsIZML+Va9Yy2BQa0YLlMQJWQK14S289pYTrvd32F8GxYbCn86gr8Cxz7J3wr8t+7RhKztve677EbwrifLPfrmoTd3L7YRF4z73eYT9F6ALG\\\\\\\/59D9e+KnHy35MozUBJapJRXpTtCIWyn9oDhMVNH90KDmVigwFUZlGnUGdME\\\\\\\/JUnWFa8Ovzv56G75\\\\\\\/reAocbquczbiDhotEtCb9OlgMUck3RbLCwC\\\\\\\/Te\\\\\\\/KqJUx\\\\\\\/UPGGWs\\\\\\\/2RTR5uLESXciK4hbK2hOkPUmG1+A24GD92gweIE4EP7jxfXRS\\\\\\\/MGh8MLQOGDqY8EXwzVHDbE+oleIqgbgFlv6vpT8kIZgRJCfDyRxh7MWisLMRRCMf9FQomjEaezY\\\\\\\/NZ46dDB5ZJh8RM6pcxwxpMVLD35pYUmzsBMVpv41ZEuR6hPIQtR1c52wikVEksJA6Gkh1+7M83UI4Fdh8JuNwlUqGIwqXGM2ACfjxrjotrg9ju81YxkZ6Aqr9FuCrFtzzeyCqK9ReHbWiMpFyP+LM1bk\\\\\\\/pYYTzdMQNiO+2FJRvBlXHKV7ZPObZEFzK4lTZys9Pzj1OhPNJg4rFqdaQZzGHo2Fa9OixPvxLIVjGVT5LMXOWimR2FRhMFPL7KVFhl7mr0e3FkxyBidsgk9N1ZuZW\\\\\\\/TUe78rBMwS0TuKbB+NSnNh2usm4PforOxQ37V\\\\\\\/hFxkqYqDcnpn6JVsOIGx69doqO7kS3Quj5T6HbswYUlJLu888Smk9Bzs3b77Nm4\\\\\\\/ECiEV0Pz80KnSVPMt3yiz4gu8PQmLDPHrTvsmt59j\\\\\\\/BUOT9GT5ENcvcGbPbIUxJHcQKsxU7o2i0svZSXtxhSV5q2fIblcmu7D9pIlWcswJGmV35MYt+fpW9t1SJ8kqKCgfYZRJP9f1ZZkND8HnllFUBKA56NCNnvDL\\\\\\\/B9OQFF1Ybmy573Tg0\\\\\\\/rl0KgPK4iVGwTlwSZ+P4c\\\\\\\/3BNv\\\\\\\/4XHsSV72kIKFXETe7xdEC2iRYraYqK4zzFFxrA8p5b9X9iSbAUhRDZA0ccFfnLZs3X9lYGs4Q18GC6f9ja0\\\\\\\/VzHdO0bpKrXkZ4Daiu4Xm8lFvP9ISOL3Dq5+dLHu3QJCsSPn9wbT16fPz1LIfYdrBWkgxAPr6sf0G1MGZpVJP7KYPKnDgSomc\\\\\\\/Oymglf45RcLu130e3VIYB1rbKgjTyc01qFP\\\\\\\/IcI+KhXdWJYlfv3B5kjDeC8vXqJGak5Zb4PwIDVNmN5HdRlH3LxzpUso32y+xvTc\\\\\\\/PuFVQEDvHaOdqu5S4uA9liSv7oioYR0Y+9cZeoxfrcfVKGWz3G3XMXwf+Ugwn\\\\\\\/f9A371q091+6n+K6w15mXeZK5ulDUmaHrBdtDdDqISxDjuH5580nj6ejVP2opGRZqxcndG3UooJ+vhU2Q5\\\\\\\/0TW2JQyBvNrxoV3nawnSXtaPy4\\\\\\\/tuaS4JZTAne2v8AamSEvo2vorjfmaW9O50SYaCVfjBmWNSTrt0wJgIk2Z+btzSQ47gXWTma+ldhuQ2hWC54LBsY10skQI8e75niJjy9yJ3dwH31KG5XL4Me39vncpfiBapNiBg1rDQBquHp1sQrrUkmmwSTY8K1d1J85SR+Rdte25LRw9Mz+oejfW++hMtbAacvjZZQrXX85V\\\\\\\/iMiYHOzmB5Mxg7iIXD9plIkxmEjJgShvPLJPoML2+cs8M3Mgh4l1+vD7\\\\\\\/9O7yES3VZV9IUqYzv1YS3iFsi1Bc7MoQV8zVhIOWXRe6RQu1AaeZTX32COOcK16Oc4GGLwi9yT00E2aDtZP7StEdeZynYczgikpMKPQrY7xsNOxYMExbSv1LBWuIPkxFXUISph+X+oGnLUHn6xZWBvjD5NzvLjLGK4XdxoeyLJJD4ZnLKyfkA+cowrNeUE8yjUU3cNMuUNp1XNmbjwMDweB8f5H\\\\\\\/xXuxsg8NPWxKDlTt+s8NBDTo7Kz9dlahw7mtNbWJ\\\\\\\/llNVQyIr\\\\\\\/a9nxkLdA962I27AMg0SRa8SSdZqdRTUtmoHUAlR+kAIO4ynzE7b60D9UQTlcl7WznNykci3kM27hTuwan8b3F3DDjqTQnku\\\\\\\/xZF1LZyA+4hZay9tYhc9NGTbGiwKFBlLIx\\\\\\\/XJnbOCtXIpAAixh9DiSGSViPpbtYPFbjlGIY1irMc23\\\\\\\/41Ub2nNMwaP4Fkhv5PY2FyaaRkznHuoGeguqkNumg78KXxY+TrH6ghdFPr3Xn\\\\\\\/4yJk4V7yDSFZwbIKkJGYWhj5CvIoRzJsd1WoVHK8ewzLs2euJBbYIOG2w1AbCtY0J2IVEtY7XZLaPFcWF9Wi0nSuTe1bqD\\\\\\\/v1fG1pu4HNOo0YdKYeT4EgAWR8z3eBpOxnv9wYLFgJxYEzYEbwFembCu4+Sg\\\\\\\/yRWlYSulgvVVloIrN3djeZAdfYIHUbeM4tGM4tS+MwRYDmQAHk\\\\\\\/ekxIF\\\\\\\/r4L43uhL\\\\\\\/mjsdOu0\\\\\\\/YkbUybd0eGPSP2\\\\\\\/zXTLEeQJvx7YTThHbTCCo17u0ppCKCF9uIfWp2eq7WRXjNF3HgKiRVV3HKOK3x6DXaLa3pFOt+jybW6F0+UT664xc99TIQeH5qdNgmbIKxx8iGqpp+XQWgebtYOpqaWmgInwRyth5BjrR0zWR0e0fsJTBWHb3MvMLVf8OsF6Y8Kwcj+8gFhHg8RaaabaK1oUSNDKmml4ACPb89aXUbk7w\\\\\\\/TWRoSYCuKe2WSz0OwqSWFuVzbsiEmTXN9PVnWKlwbgqZAo4EkTWMPdk2TzHutHZt3I\\\\\\\/40vv6ERhqo80cT84RN0ISkrref6lrAJ7Azrzi60n9sjZnOnBI7vqZXk9K+QAcUkGe9nG+7JlgQt2EJCUo2P1S5ywNBIbK\\\\\\\/pq8jdmeYC3Jd4irJAnRfwrajwSgFBQEKQtfkvUKQQSleM+tW8DwctxHtpPHbPoFdMyWyaR+q812XRH9Qv3QnE\\\\\\\/7kvr3AlhfuOxcMnwX0llvRGq6aFEvG69pKdMJFIIUYhtbH+oEUjz9PCzhO04Ucgc9nshkf4diAi\\\\\\\/9ZkD7jXXcmAjwNcCWNC0YuQV4hVfuAQ2Y9waIadE07iajWltuPbLbzGjTgJhbrPkuRxZmNKo+AsfyNhjqoEfS2T7BVlKvGGKCs4FRqPBzf4IF3RWl1HFd+lzjOL5KUtbdc2FVHlEoTgtuMFmlEOkgWbE\\\\\\\/6vzGlx2YwTEO4cKdcX3ESa2KkbUfVzb7mIw6z0eay21VCF64BdgXjwUzgnk0IoIpXxJo\\\\\\\/oJfTuXvuILn5wyRcDMhsUioZjj+lhn+jvy7Raur33BbGskBxWld\\\\\\\/3mbEOXLbHiqqhXBWKMNWznilPzS8PCN9ZlcXQGEXOl2UfOBS8MQ1cYypiWsabQTo1po+FTqHgx3NdPqyGLXLe3H8SulPdI3tPbT4vdjLzorAySPnCRCUv\\\\\\\/y2RzE7pt77b8B51JB\\\\\\\/5W3Y2u9IH7RvvYkdS2zSo674pQbE0GYLEUQujIF7w7wnmmsFqrexUNgcOZEGFRiFLaI\\\\\\\/mGQeVbueQ\\\\\\\/jClUgo4J2jtVclCHqWYBOUqjuvSq+RMzWS0DtyMQbCWAx0KS0VEiv4k68BPAK9+Trx3H\\\\\\\/ThGhgCE3\\\\\\\/Q\\\\\\\/ADuP2ebyyxcOptFY1y\\\\\\\/bwNdoeCd23L5upS8wJavrGkuN8EazACq+8JUhcjs0BxiCyPBxY64l528+NJUTc7WVPOY7URFkdHQJoBPxZRj\\\\\\\/TD6vS7kQtd+fJNEbpb\\\\\\\/r9gpkk92X8LXsK6lrlOVWsHpxNvTeVVzZ\\\\\\\/Gq6GXNsPAGqJzGWr1mkMTBnCvy+zei9ZTEdN\\\\\\\/fsZBRoht7hSX5bm0TVnM8IounuOSGMofbnSHEzTbYwryWAgCxuyV2YHR5poGcIytewmaQ5EcFDtBqLI+CrFtor2z+a7y25q+hrv1k\\\\\\\/FyTHYyZneTqifixpsf7Ds46pb+eTI6cLm1uX\\\\\\\/w4gJF3TCu0t7RXFqhl43B\\\\\\\/WhfPTx\\\\\\\/46c9Itkf2TH9KD2ebfJftBqenxO43TmciOJ4haKQz+FHfSWKShuwqMv6C89RmdzFIpVsNYxkjNTS\\\\\\\/f8JyECMhCvmMNy5JCgLNf2wsiaNDG4AZPsobSMb5wJtkI7947Y\\\\\\\/FfbTUKz229kq+p8UCmj2m+fzCdUcQfZ0PebYP\\\\\\\/INU78U0KINuqUT5N4+tLTnyDnR+U98wVcOSXDjPnhPsarqamcS3vaoTlM6UL\\\\\\\/oGTlwTH+ydXAQNbl295Xpf0nzYFwNK1kIySrsm4qh3IQwGymMStT2jG17Lvo87Ixnyp8W9JFcQTuRxHXNcnfb1wU113YSPO2qhNNhJufNTmRzU18b\\\\\\\/ysTx+r\\\\\\\/k9\\\\\\\/PeOGaxOwoVRYFKeEL9HRFMaKXWbllApaqafK+z4DbBEVcnviTJHaNxRQV6p9sEM20GqHT7wxALXDmRcDxnt9\\\\\\\/5T4p1GFy509hKpKFpe+zpj8YDLowM1OV0LQA8aBea8jzmVqbtA\\\\\\\/xPBa549YUY5OP4EPtO25aOBDDr6l4E1Yvx8HCmGKrMxawo8fuD6zlm8yNtPioiRxVOD6RBM0S\\\\\\\/aGaYaMwQdae0WQaRxFXRHHa0Lff9vwSfGN0Kiu4mKfsQ+mOggV2oX2Yb8BoHLXXEEE1djlc3wQeT5eDyrwYYy3uvqeZgZ1YTzA1pPu22ZZjgtc6Bfkz+IGNPZpl4afhiq1kK2KcA00Ejqsj4\\\\\\\/mF0Ms2MCybEH\\\\\\\/CHbzGSUR7xfx9aIer\\\\\\\/ImWL7uvT8CytUz2IVmqLhgd\\\\\\\/CVZT2K\\\\\\\/5IfQJGw5bds\\\\\\\/LEPXaGBbNaU1Pv96SpUWsrDfWV35RQvusblD7zO7JVlZVmsNA9XvJJt3LG5C4UwYZGmusOdUBKnfBEcOltTrY6H0vw2ngZYpJyGN2jlHNjNSjdfVp8bMpkEj3tmUEE6rdR4P6MF7Tqhm\\\\\\\/hwMj7iUEs2qHF394ocG7eIxOIkVl0qFdp\\\\\\\/a96l0QbD44bBO3YMku\\\\\\\/kGuEj1xMKzjFIJ2OL7vD3u0ZflbZGwGhNdqoGOoHWFgjMplbGnz94W6atk8vYVOXH5LAH3erTf6DNCPSpS8cqIrnzmLm1V+zr\\\\\\\/WNGJ6ho2d3XSn41Zsgy3m279z\\\\\\\/mpozjQce0OUUqmJ83lL+w2Ae2vUcINZI3OtxID+V8D5Jk681Vw49J3t3r9l2JNltne6Z3GnjGoZhQ36ar1ZnozfXr1OlBe7YOkrzPpQbAc09JyOnzfzbP2dqxZUaZlg+9rwiAgUYcAHjFw5lqxCzK2FCAaRBdS9tkQgX\\\\\\\/Or+3G+jc99TkfMfi6VdjCG310gepvB2KmM2nqzamu0uFmJCheYKfBl23ZcQfZKE\\\\\\\/8eG2v58WXcsg93EQc3zjDrl9ucCRQ4NHo424zZd0TKDNVhf9SyXcjWe8Np67q1mRbqa2\\\\\\\/83Bf4kPw5p4gCgJ9Tx97T7LF1bAdHtS5t4LBaMs8JbBu955uVjqQTXyM22cp3PfiTSfevZcUY+VYd+7Fn1zY2q6fbNI9HzMEoalGdwqtHp42m7VPjCM5Jk+DwPNZuL3bwWmlAhdqSv+uia83BtW8r8RioMWi2AdEP9PzZ7W5fF6gvPnq6dIjrWiv4b199zjF2JFS2Vi7hUc4x8A0rg\\\\\\\/BlEw+ZMfO5O4TtmQSkusvPtFh47HQctwNy809B\\\\\\\/uyj+SkXp8iEJaCNNYlWaDWW5LNmWqVWTdxU8npNSN0ONKVx0UmxLdRJlJ88hJ5vEYmyYS4cj9axkTV4UcFyXsFWxzqyct0bEZfQBiyMKjUaOETI8Qc3ErHJkWtkWK\\\\\\\/PzwcWIDow2NYIhZx98CbtG8iwqj8ch8JjqRsNiVYEHM3w6MbshvCWjSUx2xgWZmDUhBKXob9QjJKN5x8\\\\\\\/1x\\\\\\\/jBmctkVs343ZhaVhvWgzMWUkKwlZRXcWZ46rKfFahpMCM79dmNaRWMvAO7WgjwSril\\\\\\\/Fj9EPIP73kER09\\\\\\\/3cRkR9J4qeEV9\\\\\\\/8K\\\\\\\/BZ7R8t4mYs\\\\\\\/c9p52dIHQBSBaHXeONb9j01nR\\\\\\\/OiPgZ0uhAbID1LpdmVBcu37u9DVt3K9wBWHLxseJbER8NLYFZzXi1BXx3lQB3hzcfptGgApRrlG7zykaMQzorXiEfsdOm3caHsgwURBmQbosP4kx5CHKzq0njervyZEZhndop9rOSvAE\\\\\\\/ddUT87PYEbNaFuBxMRfL0T3Lzo727JiGO9J1JlHdwNYco8U5kIUHrPYhG39Sjt8HKx7NT1hVissXdRcvo1zvH3tF3t0afmpK822zK9m17kjUAvLP3JuhLyEg8Ba0eB6iLdKgOducGG5IWxywQsdFZDDZWRPiQsnhgH4phanwNjkb\\\\\\\/qofH4g1+HwVK0RGtZBSsSLYjknwYpwhBYy0EcPCgel17aBOA0zt4s8ZBMBBYFk0gW\\\\\\\/c7YAHTEEDgHm4x7\\\\\\\/TjCsE944Iq2ey8p8OZd1vPBUvDIfYSPWvww4TgeFSVNk5VryHF0LUNGFzI6GoB+r1HK5PDyobNA2ofk7um+QIC5iw6dm4o5u77Zd1ODqiOBlAFAhOzHdQTjmKKEpK7ozBJCb81ElYyxC1fwqnmg\\\\\\\/8yCOh5Kqnoyr1lu2iXSAtSGyegnirQ99TfCD3rbFvWSLURbpxJiD1xkQVV6YYQOqkdcNT2s0P2wk+ATC2JfBwYQXoZaQ0JxjVAoV2sx2hBspbIfXsoQXmUtNL75hpHaDaYbOXC+LTw7Ewi3NqU4WWNlAHlUIPYM8a4QF73\\\\\\\/cPAq2IyQx9pHjB6VcR5\\\\\\\/qHJaophBH88r4jxzZF4Mmqi\\\\\\\/h6TLSBm1CNN3XO3bSuDin5FKtaQPhORfmtDa8Se7idHH8+nLZ043tLKKnCO2k3t+0WbkgyOLvid0dS9r1Hd1ePn\\\\\\\/u1boYiGvgeSoaetFtLY+O0WeTRDhFDKml0KsMRIEs3SFTxMZ7C5qlVWoWiCCldrDJ5NTEgVLwg0QpT49RKWSXSjhvkxTT\\\\\\\/jNSJelio0eCrd\\\\\\\/6WS05Wb59OXmj4aCSI7c6kfqasKUjzdZOSpDHUjce\\\\\\\/U6EyW1ENvdSW2s2a6jsnMt27Sn4WN3Nhul9xMwDzQpFP2\\\\\\\/695QXxAt8\\\\\\\/H1yVOjjicA8mnpR3juOcdMvReKS5jwcg6xJVpqwH5h+D+ysSDZpZwjaA3Qa52JKWtyfCoxOWnDyD7L9pAXtiFjTrZp2mDKVGXTpTsdcdfYVwZzEvdV5yG7v5PzGXlMnuQtoouU8Tmw+z7twPlBcRcl08fzgVlR1a101Lem+qpLcSmUMPjqazZ49t9ruTXMwKC+AgxmC3elFmG12etQ1ssBdmystyYlDt\\\\\\\/UMnxAw3kEA6x8nE6V+xnlsh32EDGcEJZJcFcBcO\\\\\\\/q8qBcXMeIOUFnOK8ep1eSYCyRvxLON+hhIIrb4VXha05KFq+aYgJUWmdJHwIpyHJ9nTwcNZLYjj6qVZ+L12hMm6vS0gRvIyf12XqC3TMl8G36P\\\\\\\/2b+w62oy\\\\\\\/+O1zYySiiFJ5sxofjHaequS9ner9iVbiYuFGCa+C3vBymdm7lEvUpdrNP1+9q7E\\\\\\\/taIXX2eFSXZENh7foe\\\\\\\/Koxv\\\\\\\/X8HmLK2Bmjruk8WgxSZiKNGngnpZ3XDtvONb4mX8XgiqOvmTopTOAuKQJHkJiCcUMjG1bPysyOFJqo+UrR6utLOwTFbTOy3wMiGj1nSlQKMC8YGLQ2GwMF\\\\\\\/0Zjaxx5F\\\\\\\/6EHpAxb676Ty5YJ+mkh7mrNI0p\\\\\\\/1ydlLzgTCv5Un09WL0iXIMefp6PvLHAsUB771AERiR9RgtRWuW6USTLLdvWfCZehgKFJ2RhT3HFDGX5vUOYsCGUxSTuydHtZIRv+6xk2M4UQvMzW4VDRxpDg1EfKfIu5Nnobd\\\\\\\/ejAa6sqHRthdhMkf0KYM6rjWzNd4XYOLoDQGFcbrbI0K47upebJG5m8zHTdQihXv4DtV5Asnk4IKll6xHat6vd4vCdx+RUgQFhvBorflWOmIK7UO+8+XDXZKE9seeOX6xc1ts+fMkn65XynFmFQN+IHoa6iIXHJuoazk98ZAggVlj4fWdjtkKm9+1Ahe\\\\\\\/rPPR9oE2OJl0sLTDQZyWfhT4tVfQFIten7r9Loojbu9sjcFLu4IHjeszbPkgfolXPY+DZ6zb2DJxcQL2\\\\\\\/zAZnjebnw27PnnJsqvL6+C2jsOO34etrzOwDkzmCJS0aV0fueVhU5rsPBKjAeeo\\\\\\\/vGkKXAaXJAEyyU073PieDLCzPig34872vB6f28FMKP\\\\\\\/u4bBqO5lPgAiJjzUJFpuyuosM8m8HEkra\\\\\\\/V\\\\\\\/MufND\\\\\\\/FYW7OXp4WBeEuSq2DeY8VKKtfPs0B98uv8yg62iAgwwj8AxXirp6xVSU3yG+ViBlUFc44FBSEychj36YyYqWHPKdBIOvpJI\\\\\\\/OLqtvuFGv9MUsmJqydqJ8d0z0fZukMOsOW4PgwZGwoBQxbSxLwwggwuOrNwouwdUtpQ0oLtVAn5Sv\\\\\\\/khGLpPV6+wnWDVAndHN1b6mk3A5FaDBzHipTb1wA\\\\\\\/n9cUSUEUlqFtn6cRvjPJ2eiOjP0UvFPs6aksFz0EzmXRF3VS+PRr0x0X5AlLMz4NNbwW1egs9DrH+kMjUdigWVpZsm2QbIlk0DbHW4H8VJf8ykuQZe1n4m3tgD3sWiJMYLVvPJ4QZVmonHcW1Le1icGBgQZhOmmb3rqYXuIUsTmy3ZbZbVgdy3cUJLczzH4UAYuMDSW1f1TVetEo8yhAz3g8j7SG33wgL6YzzOW69\\\\\\\/ccYq7DdGUV85og2Up8Vw8ttpdYmpYp2FYEf65QmYe6BtYJEZZd3QceHPLRY\\\\\\\/Xfi0bOw6JIOLJGSL9T9MBHvvl1eQzVCJfdPQaV9Glnk6KC\\\\\\\/4pt58\\\\\\\/dOvs9f8tWc56ILtQ\\\\\\\/i+KxHbhSvi+afH4YIxhtd3DCLDOXhemKC090RHRfcyK\\\\\\\/kJwMykwD37rR+IBQ2OA33KnVPUmcQyWAKURKv5ZlHUd9ZtT6pop36SEvPqtsaPJgJUSuJwHprrfmNvBx40xdgygiM0aBr6Xe+rOb9j5J0eV50uZ7n6T3\\\\\\\/GKKLjTArpEd7jIY+wEzCAOUOKSbWARWOcTogez5GPn9NowLYSYzNEGMD7udMtv92VwOdXrwC6SP5iRG2gWNND3pcs2v7PMq9B5Ic4mSTBB6DWe249IDVXV0KfhXrDUqR\\\\\\\/THI+cFQFYGfI6LQD5l7jEpfTZUJciBN9V8kKg42KRg0XI7EBQt8h32er4mW1bnAAMY0s1fN2wHdb4ZzzTyGRjmIYGhF9r7GkdTfIPmMgF9tk+V+6qh8AcICuKCU3UTcKMqSFjG10rZhfUIHD1pD9kdvi0346VbzZ4GtuNBMB8FzF+HQPIfErIKtWQAO+uNyjQXTtGvFNnRalFXDk2Zy3jEj+rus9WANzqpScjJ1OL2pEnXEA1kHL6GjSDeJts36eiEwVEQl7\\\\\\\/4YLBZZTRaspY70PDAIATxUNvrIEdD9abL8qOgGVYpS9C+4oAzS7l74IqMfJGahmbkJiAgO3hMw5pgme+LNsHR1u2danhGU\\\\\\\/3t88hpAzkoAd+Tp8bi4Ch8SjNcS56qsyKVaAXqHFisGzipZab7zh\\\\\\\/BrCJ+ocrlZ\\\\\\\/TVIgvAiOYL9LnAhI8aAmhh\\\\\\\/nP9U7q8nAWVJC1l3QCmeWfOsRNp1Yb37BNbOKBsU0h\\\\\\\/53F7sAufCxZOe9RcpjUdG1D34L4IsiMGPhPhSF02tw8RQDcwr+P3ZbzlpOE3yHcZLtnvHQQfIoRX\\\\\\\/OHVkwLwuYE6MCL4pdYM5yi27Vy0StB7t7QjKPt6JUZwNvzwVAvRHwwLgdMUU0Lp9ms2eV94JOEKLXZOCMrfmZcwo0EsXNInACCAyuIpypZ+TBIMnnJcf0ze1ie4+M5Ek0MhoFol2eu\\\\\\\/xwLVvMdGEjnUQ+lMbH1JCeR5u5MkQ81wQjzizd2wDWizkGWizbQzuGj6K8nts8JgFDbEEm6Omu0wjyMkQ\\\\\\\/0u5hCgIFRSSRcBmRFbPUEGY1mE6MnfITHKljUhBJ\\\\\\\/BMT7Oml81ajf0e4M83zCAPCpD6oiOwI9QQ\\\\\\\/w8MyTG2e5uesVudjZlifdMK+pa7EOP+YiO+BX+4xcS1vrbkHnDun+hv7MNQJe2\\\\\\\/eUP2Zs9tLTSWEZ4kT3PswW5GuX7eIHZ36LxD6eFvUyRF9AOioyybaWoz8J3VFCAzCvJG+0I+pyEqiy9UP8dG0QS2IwSxAMReKroUUSIqB2UPuzgI8yUKAPqMvpyq8RPTHhoDGh1hTgk4TTFVswukF7TAH6rwk5RyLU2gGptpSPTfHS+7Muue5uLQSOXaGrek2zAXLzZub0LhagKd76V48fF85n8e\\\\\\\/XFZ2PD\\\\\\\/\\\\\\\/srALQQ6b4k+f32iwGDsMhyc6IKaJrLojWDas+ooSyD13uQjg91K+1\\\\\\\/tywMIZwSFNFNp+HlVeJ9AZpmR2Y9GcqGJ+oubP5jDvCQOC28BT3IHqkEbRGyqW5srU0xcUaFGyGoKWqGJL0V47g9wa3kJCWWNIuE3wTNtUfYJP6FhAP3217vFiWfAK5+L4uQYcpdBRYC3SVM947i1ePsyzLMq08IKca70fRW9VNcg3auNRDlUrcuzWlUqL18EtojtUOVZSto6Q\\\\\\\/z5ayzGYLrnh65GfTdyVy8eBue8nhQMh8++JMrPbisy9p8uffzM5CsxLH6iJjyiFgsIr0dO0KWj+k5w6PcH4arfMZNWzWxB80xlw1fLEDpDZ\\\\\\\/7zC9ER07vo\\\\\\\/xzjCrqgb2bKZBBhw6PzWFnC6B+oypz7pTHlvWlq5lqtqGRtmkwsCI4L5R1mhMB0fUQ7byif4Y3xNh17O80Xw768q3172yzd38M9qIkzkGz853OD34bL1aHZzglJMEJ3QszMDEzsndqQZtKcduu1SeTntHi7mKhGvf1maisR0zxH1lwbw06rteAkLLymdxN6qJhSNvFAfMNJn1x6aWzyErl5OvOqSYJRJxm9JKx1K3YsAVACZOzfFF3YkW5GlPpnGrlolvstZhHWRJRJ9XpqutLgEqz0b9PJsVdG8rqiyjmPVmRb+qmn5\\\\\\\/v47LNeo3j9Jq4+Eu\\\\\\\/aDcDfhoGtpE5LHHxPQHVcuoIEp+nToZ\\\\\\\/ad3TdHLArvAq4gSSjPWPW4CGEFCRki9cJHmOlMhwzFRubXW0Ha9ASrOEjsmBJQPO3w2oHukCeWQN83DJOLzvYTftgUbamZkjkJAkX\\\\\\\/tn3LrbsfLHLXTl\\\\\\\/rSoSoCJUctlgNoqnedlkdlip3lyibj65Sz\\\\\\\/IdT4oGYnYTQ2nJvuMph7r9YOUsTuINU8cAJ4Do9kdC7EqyLyYj+nUIw0evKGTCcGq+7ex1fZ2HuEQXWdt2H7Kb18EZV3fwmwXeXCLnucjH5A3ErxLTDpzJRKuXi1etrugVJBWKYMr5WEtQjUnekmAxOh9k36ebejn8hw6ELyynQqI4+F6IuP1346qBMUoFluSLxwiEEif+qU0FdIlZCSUDvafXe6Yh5ejq\\\\\\\/\\\\\\\/HjQSjHBKD84QvNmVhLixy+lAf4BAPCsIW7WFCr+\\\\\\\/\\\\\\\/sEE57DnP8sBiiKHqXPqEipbTsuJ87vNbtjQNs5jFbubOR\\\\\\\/SFJR4\\\\\\\/voHFPOVjHuG0DUONAZqTq2No3llU3rbWDwscr1dOiDxQ\\\\\\\/ZMHj1SZvakXS0WG1oe7Y4haVkbdRlqVyG4peMVKXpbb+M4mAPxZu+YbUzF6kV9ZDcS8BY\\\\\\\/70uCvj7Uu518aTFWbdBH3pYODnG9sv5jxnX\\\\\\\/YFebaZnCDRRMUjPI1mEZytqvV3TOXn6am2L0gcQz\\\\\\\/wOW5f5bDvrfAWcsK0hqiJauvGypg5ucPrvRiEbhaYFnqF55WQ3VHD2nKXyxeS7INsT7NI5Z5KVv0v6hgrXIPLLL\\\\\\\/0u3t1BvBhBeZzhjVSIAdWN9BxMqzfG7dgahdyOqp+pi3ONWDVa7PYC637Tu7cM3zVb8VNptBTPncVFNiTZxTJRM2v2vLhezGthMCMPrZQ\\\\\\\/+rhFuxC\\\\\\\/pLS6uoXQ5TqSFU0RxXkI+kaxwjy19LW+Fc3IFlbAseKbnoE63d0s9OuI9gvQ2KslXXcBuWVt2aRibPxmmKB+Ud6iJ\\\\\\\/jCy3NJK70Quej3uKyns7RzKGmG6eZEB2H+gGp8D27xUuQjf8lHyOf2IhpuedJfXhmGkitWlfT1vy2P6hasAO6\\\\\\\/4rBtSek3uleyTbvtlUn2vo82H6V+DpswCaON0eoX6eY5Oo\\\\\\\/hf8XugsFixS8WXu8U4SZsARAWPizAbSQRJw9eo7WdU0FJQlTMhn8q6P5KfWkfAAB9d6+GgCIac9TtcnvU1Fjc+wBJyMsRksFvH6ZuXBm7QcgGe1K6w6AYuRQtVpERawP+ldxxPN3q1SVBTcwfKA8vPqg78nb6NJGfjMPffRZupnBaivyqOAR49FjEFCJ7XtOfUpiETyYJKn5AIZd5\\\\\\\/b1odwiwdZSy8EKjMoefYAb9xFspdgYxn7j4aGY38\\\\\\\/Gcl+X9ErMdkkOEvf4NkWIgrbLbTeuK4FdwDuFNAPsEMQ\\\\\\\/kz6dU2mkqohc+Wv3SZ\\\\\\\/AGtvJ2J4PWtzVfCPMsE5QyUjgIFBYid2MUvfedR0wHApj5UiR6pgCfM02sWCTw0zldjbfEcs3knkqIZqoGK37xn5az1OMUdu\\\\\\\/fZ7bIok2YDWkvIokG7mMuoP\\\\\\\/53Lq52ROGkqIzRST1qsGLKnyv+5dCEj3lffYZWGPHeuvp6hJXTDb82EwSFmeQdgVNg1HkXPEyo0+\\\\\\\/4RRvtjmu6EmV3YZMcQzNQAKLivnz+ZbJDEcu+ENF4i0wSM0BwCwbt071QfFS5\\\\\\\/0LDrd27V6W4TnQG3B5ZI8FnSRxFrJktNJj5pbMXc3muS9JA6cISU9feTwbEHjEBVVJ2PkUibpRwtbFg44X2ExNoBExqxeDBkOsOSgE19\\\\\\\/F8zZnLLor1m6nUDscs5+P2Z7O0YAe5qwR4N7Kw4IAKkein0q2YOLCLDlmTw\\\\\\\/ton\\\\\\\/TbMAsaNs\\\\\\\/yjq\\\\\\\/\\\\\\\/ysni1oUFmWc1NLmmlWY4EtpOY6z8di8MIOeU0PBhYll6DIIzQAWa5Jcr7US9698uqYl8xTJA80KwyYfr6Zc1ac4gCtjd20NbaMORfX3AVHvWxV1NCKpLaJH\\\\\\\/W07Dje2Xv6\\\\\\\/RvuOLgnHFqytOmYmUyYBCGtegarTTYKkFKppPrtc1Jx9XB7juYDgmMJ3sEb8sRSKlGhP2xQdg1ANVMOyUc8pHvwYEt+pR7NU2eA0ZPTisBFhE0vv5FJmeiQqxBQCQXibvQMTQpQ\\\\\\\/gYQMARWfRoWpLyChHrSuiWTg6Ap74NIL3F1KdKNSlMJXdnc5af4bLjG++y8cDMtQRidUglj9aD0GvXgzsNLPA8U4zsnN0uY28sX1e5oN6opi6cxrm1MhAMTD2WKzRLb\\\\\\\/AnSJj6180156EHfgZpLjf2GpsqWzBsRfZl5t1Mu+U0NHJ0qtjkGUCH20EmVu9ivccJydpEOlnBnDCmzgb4gMoUKpLITTqBCdex3iNjxU0Fa1k30zKPQoRKfN9Y4DLaEXRF\\\\\\\/E1MsPPSuHODwPGZZWUEoq5jfQ8LtPLrHJ7ECsA3c+kFF9IRBDo8Q6l9KGnZOST6CCVHPibjHK0JEMJdZMaOvwWRPaoPTG3bNhzk2RwR\\\\\\\/bn\\\\\\\/luwFS8fRXhvSCrhWxkT2Lx0WwosNBdIdlLBvWysulc3V8wUN6Y7MCrTWRXBLEIA+gXSxPUpinVu9r709Ws1Y9v1dDaetOk3VeSgGTVnQz8+GSzxoNEYOwhnWM+kDVBlFn50CsRhVVjVudRHcbUSnZRRaajbTgHjAYFieLMmtpHKzGpF6XVa4XSWdtNQLH72K6hfHRArTcPIAepO2f80hP7WNI0\\\\\\\/8xFdzZPtdl1vP\\\\\\\/xJCOEpU1TzMbwi7lDB0cyMA8umgDttakSgzyqnSkoSt0radDkYUg45e3NM6nAMQFi9WtMqxWJEt+yYzSBU3Ugm\\\\\\\/i226RdAkqwkgv1D7ROHRCXCFtIoV0lLv9644rQlm8gvOrOFZ7PDM5dt4XFHvxGqxGQMhMRPwIU0UJKDxUK1lFuT2e0xxJF9Z+qAOvnKYkLR4WzMuCDTyazqSbKUrBfO51Q\\\\\\\/9LsZFbKnXr7TgwkxtdAPBapTxw72FkMgcqg9r3KIk9EtYYOmi6v6E2OZuuNro4iZ1ntqkRt7fzQZE\\\\\\\/rznio1MK+C90vxbReA8aEHK6uGDoyhwwp5fp6FcWIxROZkZbSBiZpf0BBs2qnPvdg+2FLEZvCtM3317dP23NJL6UuBRRvAq4m2FLXDS3vTHP2rJFsUV5S0N+F8Uk7yjXuyu22d43ghKa1qj\\\\\\\/uqTfhQ6s3ZdNW9wZJDlCm\\\\\\\/p0CJg1ixe03tLvroTFOv5SkXI2hwNsH9Tb\\\\\\\/VCszpi6iTC2ouTdd80EOs22+qW6m2YPgCiRs\\\\\\\/mN+A8DJyx3U+FRYR7os3wkhoB+5XbPbMATI+CY2\\\\\\\/2hY5g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022onLpBhI4wcUvJwfWtU2UlQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ae84956de13353cc74877c689763c8196a4231204d030e5b08f8d63060dc2e23\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WOXT7oO8ZQL+ulXGvzNnajEVcy+dugtiQ94T7LMwmtPWWfmp4nlTg6V4cnrTeuf\\\\\\\/r7ehcFwdg7sNrpA+2udlufDJh63zPWf1bu4fIsldk47kQB64XJSCq2xJV3dAwzf59jnMQ32oKEAd7TFpHRTRTWSpscsJQ5oZ7sMdfjnBuWN9Fqn+TopZzSttRQbqbaTU3+WU7hFGtGWzzf9XX9zxIg9EkKY8HHyaTHT4x50CzejLuZJLfp9mmH4gYYvHw+Qymof635sp8f7LLM1Ul\\\\\\\/vVS4fIuu03FObyyQD1TLFSyt0Z4PpckcOUNBcoGpP1lVfvGKJCHPH36pLy58UqPSX9RSiSqGAQS4SKUwS4TLZwEhHrQD7gHJvB4rJW7LsYf32RDPe7Jeg78LnpIs64ajgpd+SeE1aL5gCbcf9tM8o1n89kbVc1eAIFgowesdOIoqJpLfMyYL5x8yDSdeUVOOwySCFvwknDe8ndRQlFt7Vzclm6fMPaX2SAk4U22dfllXTbUNwm1oX4ewFZg+f9IobHAQl0r3rwWA+TRWdng2Q3zfIbNkFH63l5PSPz32XSHjHZnCqZkPoVnWCitTQ2tbgPkddCGCn99lD13wB9C88Q2X876JqzUnKAPr8A41W6z4gLAQ67+1SS9gv8P4ZgpD\\\\\\\/1SVkSGGe3SYxUpr+1f\\\\\\\/zG7R7\\\\\\\/AiqV8D1kbreMopQyWBwX0uUtNJNde8fgKkVcRIGTP134OdpLE0rsw9w2OhZRw0mo4AGWrnaJKNeqXDwqnHuP56pQN1oGUF2vpj8+q40\\\\\\\/EQlMRrKcS+b+1qFiA91\\\\\\\/mE6yoMMqgi3ldvy+2sstC5T\\\\\\\/zBfnmiDxgkB1TLMwpj\\\\\\\/K7MCbTs7Oo52B6XeWMXIZEHy5H4Y0CKTLNya77CxzcgS6K19DS1A3t2SYCQnutoJBikf4p6uipVKUbt\\\\\\\/6o8oSZQq7MBNiOBwBxRMByXJz6OqeQP26HJeX2mH3\\\\\\\/uSr6wwWNbR0OpnR6SWB7fWmnSNTPC8MM7mdCxINiQJbJ\\\\\\\/8YH+E+Z0oD64pU9ZSEqDGB+QChJL6ZxJhwlz2KDSy9iX2X5thRcRnOyJcVB\\\\\\\/kHHzM+Elo71Q2KagFVQE\\\\\\\/UzEbNDHrwMu69iNfuS0VUZZ5DifTfOpyqhDptQS1nnyQ9gtinOUtPbQbKuE6\\\\\\\/sTT4W+vTC5333zDdi3XBBY0\\\\\\\/Wo0n9+soktF\\\\\\\/bRFA52RqOLyQK2GpYxkfEQ0Ri+zl141JQsY8dNswar9AHtYwO7LPSfklrfkJ6y0R210fwNJeE8oStdVSFUwoHaUKL58GcxZWMh35fVXUQsvuxFJ2A2ZI3S9ciMOfRmvGpDJksUg9coTt2uqK5bRgaQTyB\\\\\\\/wkENwcsdpqNB5qpcody6I9VFdXEnkBa9QrBjIQq8By0rd\\\\\\\/qNjcH9Bj8Zc6fXgV4YwLMgkNdK1WlpwQhOzteTDmcMRKwGHNUu0Vo2rt88gtXKj6p4PeKTBaXcP\\\\\\\/ODeWBRp+TlLQU6zpbQOF173OgUeqMSKUz+UmQlAQB5ZgrI08wTH2VLHJqZHr9sRqyZA\\\\\\\/TFpGQrfASRNn2OQcu6UzWyyMl8bi8Ckx8WxPJDM\\\\\\\/kxFqBaBf4hxe3Bc4FB2VWkTEEDvT1O7PbgP+wjS032x68YCjzJ1XFybRsJhxgJrWPC3P3TwibMtt+PX9D5JT3Cb\\\\\\\/Hj9yjegy+WxVkC4oJoOK5JwJctP53yM06A3OGDHk1bFK0g55KfXuq0EY\\\\\\\/rxKpdnDCWrzr9NA0vE8wkTaedVZYQC5\\\\\\\/Lk60aw1XYgyvGkDKKjKPsXSmrYSpRiokoNS2N4MCYsYR4e+8cWl1GLWnSKqOTqdiPJSqemot7VbT2S6TdE\\\\\\\/GhzgmxoTSYTAf7GZyxi0ZNkY4r6to2VWWtW0fBoCSk0SK7UNCfOEqp3wCoyqA26+xA1bFTXrgSlxewjEAVy+tmQhsDywgyVd5x76G1uhtoYKFlKniuFas9ub2FnMJWPJ\\\\\\\/\\\\\\\/JckILLFY82q6Ykj8SBFaEdhxn\\\\\\\/RJEV4O1XUA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022M+8Z5L6XcHX9mbT1NQmtSg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bf5c564e568dc7acbaa39eaeec0f854a5c7f0534a68a0d21531e49dd5fde6b9a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485843 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2859" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VHaLuyA3MR_kocLwqm9aa6z03uPWXaMxhrODvjUjSV4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2859" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lqKWAjvAPSDGQCBeErj6iI0m4LL67AnhQxRfcR4D0is" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2855" title="Gene2Function: An Integrated Online Resource for Gene Function Discovery" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2871" title="An RNA Recognition Motif-Containing Protein Functions in Meiotic Silencing by Unpaired DNA" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485843" id="node485843" data-pisa="ggg;7/8/2859" data-pisa-master="ggg;g3.117.044198" data-apath="/ggg/7/8/2859.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (<em>Cervus elaphus</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5623-8902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Susan E. Johnston</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9744-7196" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jisca Huisman</span>, <span class="highwire-citation-author" data-delta="2">Philip A. Ellis</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0075-1504" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Josephine M. Pemberton</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2859-2870;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.044198 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485843" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Susan E. Johnston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BE.%2BJohnston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnston%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BE.%2BJohnston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5623-8902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Susan E. Johnston</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Susan.Johnston@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">Susan.Johnston@ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jisca Huisman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jisca%2BHuisman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huisman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJisca%2BHuisman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9744-7196" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jisca Huisman</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Philip A. Ellis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philip%2BA.%2BEllis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ellis%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilip%2BA.%2BEllis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Josephine M. Pemberton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josephine%2BM.%2BPemberton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pemberton%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJosephine%2BM.%2BPemberton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0075-1504" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Josephine M. Pemberton</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/8/2859" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485843" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485843/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2859.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485843" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485843/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2859.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485843" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485843/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2859.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485843" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485843/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/8/2859.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485843" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485843/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">High-density linkage maps are an important tool to gain insight into the genetic architecture of traits of evolutionary and economic interest, and provide a resource to characterize variation in recombination landscapes. Here, we used information from the cattle genome and the 50 K Cervine Illumina BeadChip to inform and refine a high-density linkage map in a wild population of red deer (<em>Cervus elaphus</em>). We constructed a predicted linkage map of 38,038 SNPs and a skeleton map of 10,835 SNPs across 34 linkage groups. We identified several chromosomal rearrangements in the deer lineage relative to sheep and cattle, including six chromosome fissions, one fusion, and two large inversions. Otherwise, our findings showed strong concordance with map orders in the cattle genome. The sex-averaged linkage map length was 2739.7 cM and the genome-wide autosomal recombination rate was 1.04 cM/Mb. The female autosomal map length was 1.21 longer than that of males (2767.4 cM <em>vs.</em> 2280.8 cM, respectively). Sex differences in map length were driven by high female recombination rates in peri-centromeric regions, a pattern that is unusual relative to other mammal species. This effect was more pronounced in fission chromosomes that would have had to produce new centromeres. We propose two hypotheses to explain this effect: (1) that this mechanism may have evolved to counteract centromeric drive associated with meiotic asymmetry in oocyte production; and/or (2) that sequence and structural characteristics suppressing recombination in close proximity to the centromere may not have evolved at neo-centromeres. Our study provides insight into how recombination landscapes vary and evolve in mammals, and will provide a valuable resource for studies of evolution, genetic improvement, and population management in red deer and related species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/heterochiasmy" class="hw-term hw-article-keyword hw-article-keyword-heterochiasmy" rel="nofollow">heterochiasmy</a></li><li class="kwd"><a href="/keyword/linkage-map" class="hw-term hw-article-keyword hw-article-keyword-linkage-map" rel="nofollow">linkage map</a></li><li class="kwd"><a href="/keyword/meiotic-drive" class="hw-term hw-article-keyword hw-article-keyword-meiotic-drive" rel="nofollow">meiotic drive</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/red-deer" class="hw-term hw-article-keyword hw-article-keyword-red-deer" rel="nofollow">red deer</a></li></ul><p id="p-3">The advent of affordable next-generation sequencing and SNP-typing assays allows large numbers of polymorphic genetic markers to be characterized in almost any system. A common challenge is how to organize these genetic variants into a coherent order for downstream analyses, as many approaches rely on marker order information to gain insight into genetic architectures and evolutionary processes (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ellegren 2014</a>). Linkage maps are often an early step in this process, using information on recombination fractions between markers to group and order them on their respective chromosomes (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Sturtevant 1913</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lander and Schork 1994</a>). Ordered markers have numerous applications, including: trait mapping through quantitative trait locus mapping, genome-wide association studies, and regional heritability analysis (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bérénos <em>et al.</em> 2015</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fountain <em>et al.</em> 2016</a>); genome scans for signatures of selection and population divergence (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bradbury <em>et al.</em> 2013</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">McKinney <em>et al.</em> 2016</a>); quantification of genomic inbreeding through runs of homozygosity (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kardos <em>et al.</em> 2016</a>); and comparative genomics and genome evolution (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brieuc <em>et al.</em> 2014</a>; Leitwein <em>et al.</em> 2016). Linkage maps also provide an important resource in <em>de novo</em> genome assembly, as they provide information for anchoring sequence scaffolds and allow the prediction of gene locations relative to better annotated species (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fierst 2015</a>).</p><p id="p-4">One application of high-density linkage maps is the investigation of variation in contemporary recombination landscapes. Meiotic recombination is essential for proper disjunction in many species (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hassold and Hunt 2001</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fledel-Alon <em>et al.</em> 2009</a>); it also generates new allelic combinations upon which selection can act, and prevents the accumulation of deleterious mutations (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Muller 1964</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Felsenstein 1974</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Charlesworth and Barton 1996</a>). Linkage maps have shown that recombination rates can vary within and between chromosomes, populations, and species in a wide variety of taxa (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Stapley <em>et al.</em> 2008</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kawakami <em>et al.</em> 2014</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Smukowski and Noor 2011</a>). One striking observation is that sex is consistently one of the strongest correlates with recombination rate and landscape variation. The direction and degree of sex differences in recombination, known as “heterochiasmy,” can differ over relatively short evolutionary timescales, and while broad trends have been observed (<em>e.g.</em>, increased recombination in females), many exceptions remain (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lenormand and Dutheil 2005</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brandvain and Coop 2012</a>). Theoretical explanations for the evolution of heterochiasmy include haploid selection, sex-specific selection, and sperm competition (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lenormand and Dutheil 2005</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Trivers 1988</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Burt 2000</a>), but empirical support for each of these theories had been limited (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Mank 2009</a>). One emerging hypothesis is the role of meiotic drive, where asymmetry in cell division during oogenesis can be exploited by selfish genetic elements (<em>i.e.</em>, variants that enhance their own transmission relative to the rest of the genome) associated with centromere “strength” (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Brandvain and Coop 2012</a>). Strong centromeres have increased levels of kinetochore proteins, and will preferentially be drawn to one pole of the oocyte, which will become an egg or a polar body, resulting in biased transmission at the stronger/weaker centromere, respectively (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pardo-Manuel de Villena and Sapienza 2001</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chmátal <em>et al.</em> 2014</a>). Theoretical work has shown that higher female recombination at centromeric regions may counteract drive by increasing the uncertainty at which linked genomic regions segregate into the egg (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Haig and Grafen 1991</a>). As linkage map data for nonmodel species continues to proliferate, it is now increasingly possible to investigate the key hypotheses for recombination rate variation and heterochiasmy in a wider variety of taxa.</p><p id="p-5">Nevertheless, creating linkage maps of many thousands of genome-wide markers <em>de novo</em> is a computationally intensive process requiring pedigree information, sufficient marker densities over all chromosomes, and billions of locus comparisons. Furthermore, the ability to create a high resolution map is limited by the number of meioses in the dataset; as marker densities increase, more individuals are required to resolve genetic distances between closely linked loci (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Kawakami <em>et al.</em> 2014</a>). While <em>de novo</em> linkage map assembly with large numbers of SNPs is possible (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rastas <em>et al.</em> 2016</a>), one approach to ameliorate the computational cost and map resolution is to use genome sequence data from related species to inform initial marker orders. Larger and finer scale rearrangements can then be refined through further investigation of recombination fractions between markers.</p><p id="p-6">In this study, we use this approach to construct a high-density linkage map in a wild population of red deer (<em>Cervus elaphus</em>). The red deer is a large deer species that is widely distributed across the northern hemisphere, and is a model system for sexual selection and behavior (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Clutton-Brock <em>et al.</em> 1982</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kruuk <em>et al.</em> 2002</a>), hybridization (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Senn and Pemberton 2009</a>), inbreeding (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Huisman <em>et al.</em> 2016</a>), and population management (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Frantz <em>et al.</em> 2006</a>). They are also an increasingly important economic species farmed for venison, antler velvet products, and trophy hunting (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brauning <em>et al.</em> 2015</a>). A medium density map (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-1.gif"/></noscript></span></span>600 markers) is available for this species, constructed using microsatellite, RFLP, and allozyme markers in a red deer <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-2.gif"/></noscript></span></span> Père David’s deer (<em>Elaphurus davidianus</em>) F<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-3.gif"/></noscript></span></span> cross (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Slate <em>et al.</em> 2002</a>). However, these markers have been largely superseded by the development of a Cervine Illumina BeadChip, which characterizes ∼50,000 SNPs throughout the genome (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Brauning <em>et al.</em> 2015</a>). SNP positions were initially assigned relative to the cattle genome, but the precise order of SNPs in red deer remains unknown. Here, we integrate pedigree and SNP data from a long-term study of wild red deer on the island of Rum, Scotland to construct a predicted linkage map of <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-4.gif"/></noscript></span></span>38,000 SNP markers and a “skeleton” linkage map of <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-5.gif"/></noscript></span></span>11,000 SNP markers that had been separated by at least one meiotic crossover. As well as identifying strong concordance with the cattle genome and several chromosomal rearrangements, we also present evidence of strong female-biased recombination rates at peri-centromeric regions of the genome, which is more pronounced in fission chromosomes that would have had to produced new centromeres. We discuss the implications of our findings for other linkage mapping studies, and the potential drivers of recombination rate variation and sexual dimorphism of this trait within this system.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Study population and SNP dataset</h3><p id="p-7">The red deer population is located in the North Block of the Isle of Rum, Scotland (57<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-6.gif"/></noscript></span></span>02‘N, 6<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-7.gif"/></noscript></span></span>20‘W) and has been subject to an ongoing individual-based study since 1971 (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Clutton-Brock <em>et al.</em> 1982</a>). Research was conducted following approval of the University of Edinburgh’s Animal Welfare and Ethical Review Body and under appropriate UK Home Office licenses. DNA was extracted from neonatal ear punches, postmortem tissue, and cast antlers [see <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Huisman <em>et al.</em> (2016)</a> for full details]. DNA samples from 2880 individuals were genotyped at 50,541 SNP loci on the Cervine Illumina BeadChip (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Brauning <em>et al.</em> 2015</a>) using an Illumina genotyping platform (Illumina Inc., San Diego, CA). SNP genotypes were scored using Illumina GenomeStudio software, and quality control was carried out using the check.marker function in GenABEL v1.8-0 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aulchenko <em>et al.</em> 2007</a>) in R v3.3.2 with the following thresholds: SNP genotyping success &gt; 0.99, SNP minor allele frequency &gt; 0.01, and ID genotyping success &gt; 0.99. A total of 38,541 SNPs and 2631 IDs were retained. The function identified 126 pseudoautosomal SNPs on the X chromosome (<em>i.e.</em>, markers showing autosomal inheritance patterns). Any heterozygous genotypes at nonpseudoautosomal X-linked SNPs within males were scored as missing. A pedigree of 4515 individuals has been constructed using microsatellite and SNP data using the software Sequoia [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Huisman 2017</a>; see <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Huisman <em>et al.</em> (2016)</a> for information on deer pedigree construction].</p></div><div id="sec-3" class="subsection"><h3>Linkage map construction</h3><p id="p-8">A standardized subpedigree approach was used for linkage map construction (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Johnston <em>et al.</em> 2016</a>). The pedigree was split as follows: for each link between a focal individual (FID) and an offspring, a subpedigree was constructed that included the FID, its parents, the offspring, and the other parent of the offspring (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), and were retained where all five individuals were SNP genotyped. This pedigree structure characterizes crossovers occurring in the gamete transferred from the FID to that offspring. In cases where an individual had more than one offspring, an individual pedigree was constructed for each FID–offspring relationship. A total of 1355 subpedigrees were constructed, allowing characterization of crossovers in gametes transmitted to 488 offspring from 83 unique males and 867 offspring from 259 unique females. Linkage mapping was conducted using an iterative approach using the software CRI-MAP v2.504a (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Green <em>et al.</em> 1990</a>), with input and output processing carried out using the R package crimaptools v0.1 (S.E.J., available <a href="https://github.com/susjoh/crimaptools">https://github.com/susjoh/crimaptools</a>) implemented in R v3.3.2. In all cases, marker order was specified in CRI-MAP based on the criteria outlined in each section below. In order to ensure that sex differences in map lengths are not due to the overrepresentation of female meioses in the dataset, maps were reconstructed for 10 subsets of 483 male and 483 female FID–offspring pairs randomly sampled with replacement from the dataset.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Subpedigree structure used to construct linkage maps. Rectangle pairs next to each individual represent chromatids, with black and gray shading indicating chromosome or chromosome sections of focal individual (FID) paternal and FID maternal origin, respectively. White shading indicates chromatids for which the origin of SNPs cannot be determined. Crossovers in the gamete transferred from the FID to its offspring (indicated by the gray arrow) can be distinguished at the points where origin of alleles flips from FID paternal to FID maternal and vice versa. From Johnston et al. (2016)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190048829" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Subpedigree structure used to construct linkage maps. Rectangle pairs next to each individual represent chromatids, with black and gray shading indicating chromosome or chromosome sections of focal individual (FID) paternal and FID maternal origin, respectively. White shading indicates chromatids for which the origin of SNPs cannot be determined. Crossovers in the gamete transferred from the FID to its offspring (indicated by the gray arrow) can be distinguished at the points where origin of alleles flips from FID paternal to FID maternal and vice versa. From Johnston &lt;em&gt;et al.&lt;/em&gt; (2016).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2859/F1.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/8/2859/F1.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486142" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Subpedigree structure used to construct linkage maps. Rectangle pairs next to each individual represent chromatids, with black and gray shading indicating chromosome or chromosome sections of focal individual (FID) paternal and FID maternal origin, respectively. White shading indicates chromatids for which the origin of SNPs cannot be determined. Crossovers in the gamete transferred from the FID to its offspring (indicated by the gray arrow) can be distinguished at the points where origin of alleles flips from FID paternal to FID maternal and vice versa. From <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Johnston <em>et al.</em> (2016)</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-4" class="subsection"><h4>Build 1: order deer SNPs based on synteny with cattle genome:</h4><p id="p-10">Mendelian incompatibilities were identified using the CRI-MAP prepare function, and incompatible genotypes were removed from both parents and offspring. SNPs with Mendelian error rates of &gt; 0.01 were discarded (<em>N</em> = 0 SNPs). Subpedigrees with &gt; 50 Mendelian errors between an FID and its offspring were also discarded (<em>N</em> = 4). All SNPs were named based on direct synteny with the cattle genome (BTA vUMD 3.0; <em>N</em> = 30). Therefore, loci were ordered and assigned to linkage groups assuming the cattle order, and a sex-averaged map of each chromosome was constructed using the CRI-MAP chrompic function (<em>N</em> = 38,261 SNPs, Supplemental Material, Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-5" class="subsection"><h4>Build 2: rerun cattle order with wrongly positioned chunks removed:</h4><p id="p-11">All SNP loci from Build 1 were assigned to “chunks,” defined as a run of SNPs flanked by map distances of <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-8.gif"/></noscript></span></span>3 cM. Several short chunks were flanked by large map distances, indicating that they were wrongly positioned in Build 1 (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>); chunks containing &lt;20 SNPs were removed from the analysis for Build 2 (<em>N</em> = 327 SNPs). A sex-averaged map of each chromosome was reconstructed using the chrompic function (<em>N</em> = 37,934 SNPs, Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-6" class="subsection"><h4>Build 3: arrange chunks into deer linkage groups:</h4><p id="p-12">SNPs from Build 2 were arranged into 34 deer linkage groups (hereafter prefixed with CEL) based on a previous characterization of fissions and fusions from the red deer <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-9.gif"/></noscript></span></span> Père David’s deer linkage map (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Slate <em>et al.</em> 2002</a>) and visual inspection of linkage disequilibrium (LD, <em>R</em><span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-10.gif"/></noscript></span></span> calculated using the r2fast function in GenABEL; Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>). At this stage, the orientation of linkage groups was made to match that of <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Slate <em>et al</em>. (2002)</a>. There was strong conformity with fissions and fusions identified in the previous deer map (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>); intramarker distances of <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-11.gif"/></noscript></span></span>100 cM between long chunks indicated that they segregated as independent chromosomes. In Build 2, chunks flanked by gaps of <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-12.gif"/></noscript></span></span>100 cM but &gt;10 cM were observed on the maps associated with BTA13 (CEL23) and BTA28 (CEL15; Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>). Visual inspection of LD indicated that these chunks were incorrectly orientated segments of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-13.gif"/></noscript></span></span>10.5 and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-14.gif"/></noscript></span></span>24.9 cM in length, respectively (Figure S3, a and b in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Reversal of marker orders in these regions resulted in map length reductions of 19.4 and 20.9 cM, respectively. Visual inspection of LD also confirmed fission of CEL19 and CEL31 (syntenic to BTA1), with a 45.4 cM inversion on CEL19 (Figure S3c in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>). The X chromosome (BTA30, CEL34) in Build 2 was more fragmented, comprising nine large chunks (Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>). Visual inspection of LD in females indicated that chunks 3 and 7 occurred at the end of the chromosome, and that chunks 4, 5, and 6 were wrongly oriented (Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>). After rearrangement into new marker orders, a sex-averaged map of each deer linkage group was reconstructed using the chrompic function (<em>N</em> = 37,932 SNPs, Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486039/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486039/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Synteny between the cattle and deer genomes</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h4>Build 4: solve minor local rearrangements:</h4><p id="p-16">Runs of SNPs from Build 3 were reassigned to new chunks flanked by map distances of <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-20.gif"/></noscript></span></span>1 cM. Maps were reconstructed to test whether inverting chunks of &lt;50 SNPs in length and/or the deletion of chunks of &lt;10 SNPs in length led to decreases in map lengths by <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-21.gif"/></noscript></span></span>1 cM. One wrongly-orientated chunk of 25 SNPs was identified on CEL15 (homologous to part of the inversion site identified on BTA28 in Build 3), and the marker order was amended accordingly (reducing the map length from 101.4 to 98.1 cM). Three chunks on the X chromosome (CEL34) shortened the map by <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-22.gif"/></noscript></span></span>1 cM when inverted and were also amended accordingly, reducing the X-chromosome map by 10.8 cM relative to Build 3. The deletion of 35 individual SNPs on 14 linkage groups shortened their respective linkage maps by between 1 and 6.3 cM. A sex-averaged map of each deer linkage group was reconstructed using the chrompic function (<em>N</em> = 37,897 SNPs, Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-8" class="subsection"><h4>Build 5: determining the location of unmapped markers and resolving phasing errors:</h4><p id="p-17">In Builds 1–4, 372 SNPs in 89 chunks were removed from the analysis. To determine their likely location relative to the Build 5 map, LD was calculated between each unmapped SNP and all other SNPs in the genome to identify its most likely linkage group. The CRI-MAP chrompic function provides information on SNP phase (<em>i.e.</em>, where the grandparent of origin of the allele could be determined) on chromosomes transmitted from the FID to offspring. The correlation between allelic phase was calculated for each unmapped marker and all markers within a 120 SNP window around its most likely position. A total 186 SNPs in 18 chunks could be unambiguously mapped back to the genome; for all other markers, their most likely location was defined as the range in which the correlation of allelic phase with mapped markers was <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-23.gif"/></noscript></span></span>0.9 (adjusted <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-24.gif"/></noscript></span></span>). A provisional sex-averaged map of each deer linkage group was reconstructed using the chrompic function (<em>N</em> = 38,083 SNPs). Marker orders were reversed on the deer fission linkage groups 6, 8, 16, 22, and 31 to match the orientation of the cattle genome.</p><p id="p-18">Errors in determining the phase of alleles can lead to incorrect calling of double crossovers (<em>i.e.</em>, two or more crossovers occurring on the same chromosome) over short map distances, leading to errors in local marker order. To reduce the likelihood of calling false double crossover events, runs of grandparental origin consisting of a single SNP (resulting in a false double crossover across that SNP) were recoded as missing (Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>) and the chrompic function was rerun. Of the remaining double crossovers, those occurring over distances of <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-25.gif"/></noscript></span></span>10 cM (as measured by the distance between markers immediately flanking the double crossover) were also recoded as missing. Our justification is that the majority of crossovers captured should be subject to some degree of crossover interference (<em>i.e.</em>, Class I crossovers; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Phadnis <em>et al.</em> 2011</a>); for the purposes of creating a broad-scale map, we have removed any crossovers that may not have been subject to interference and inflate map distances in this dataset. Finally, sex-averaged and sex-specific maps of each deer linkage group were reconstructed using the chrompic and map functions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and Figure S9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sex-specific linkage maps for C. elaphus (CEL) linkage groups after Build 5. Map data are provided in Table 1, Table 2, and Table S1. CEL34 corresponds to the X chromosome; the short map segment in male deer indicates the pseudoautosomal region (PAR)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190048829" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sex-specific linkage maps for &lt;em&gt;C. elaphus&lt;/em&gt; (CEL) linkage groups after Build 5. Map data are provided in Table 1, Table 2, and Table S1. CEL34 corresponds to the X chromosome; the short map segment in male deer indicates the pseudoautosomal region (PAR).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2859/F2.medium.gif" width="318" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/8/2859/F2.medium.gif" width="318" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486104" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Sex-specific linkage maps for <em>C. elaphus</em> (CEL) linkage groups after Build 5. Map data are provided in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS1.txt">Table S1</a>. CEL34 corresponds to the X chromosome; the short map segment in male deer indicates the pseudoautosomal region (PAR).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h4>Build 6: building a skeleton map and testing fine-scale order variations:</h4><p id="p-20">In Build 5, 71.6% of intramarker distances were 0 cM; therefore, a “skeleton map” was created to examine local changes in marker orders. All runs of SNPs were reassigned to new chunks where all SNPs mapped to the same centimorgan position; of each chunk, the most phase-informative SNP was identified from the .loc output from the CRI-MAP prepare function (<em>N</em> = 10,835 SNPs). The skeleton map was split into windows of 100 SNPs with an overlap of 50 SNPs, and the CRI-MAP flips function was used to test the likelihood of marker order changes of two to five adjacent SNPs (flips2 to flips5). Rearrangements improving the map likelihood by more than two would have been investigated further; however, no marker rearrangement passed this threshold and so the Build 5 map was assumed to be the most likely map order (map provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS1.txt">Table S1</a>).</p></div></div><div id="sec-10" class="subsection"><h3>Determining the lineage of origin of chromosomerearrangements</h3><p id="p-21">Lineage of origin and/or verification of potential chromosomal rearrangements was attempted by aligning SNP flanking sequences [as obtained from <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Brauning <em>et al.</em> (2015)</a>] to related genome sequences using BLAST v2.4.0+ (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Camacho <em>et al.</em> 2009</a>). Cattle and sheep diverged from deer <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-26.gif"/></noscript></span></span>27.31 MYA, and diverged from each other <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-27.gif"/></noscript></span></span>24.6 MYA (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hedges <em>et al.</em> 2015</a>); therefore, rearrangements were assumed to have occurred in the lineage that differed from the other two. Alignments were made to cattle genome versions vUMD3.0 and Btau_4.6.1, and to the sheep genome Oar_v3.1 using default parameters in <em>blastn</em>, and the top hit was retained where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-28.gif"/></noscript></span></span>85% of bases matched over the informative length of the SNP flanking sequence.</p></div><div id="sec-11" class="subsection"><h3>Variation in recombination rate and landscape</h3><p id="p-22">Estimated genomic positions were calculated for each SNP based on the differences between the cattle base pair position of sequential markers. At the boundaries of rearrangements, the base pair difference between markers was estimated assuming that map distances of 1 cM were equivalent to 1 Mb. The first SNP on each linkage group was given the mean start position of all cattle chromosomes. Estimated genomic positions are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS1.txt">Table S1</a>. The relationship between linkage map and estimated chromosome lengths for each sex were estimated using linear regression in R v3.3.2.</p><p id="p-23">To investigate intrachromosomal variation in recombination rates, the probability of crossing over was determined within 1 Mb windows using the estimated genomic positions, starting with the first SNP on the chromosome. This was calculated as the sum of recombination fractions <em>r</em> within the window; the <em>r</em> between the first and last SNPs and each window boundary was calculated as <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-29.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-30.gif"/></noscript></span></span> is the number of bases to the window boundary and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-31.gif"/></noscript></span></span> is the number of bases to the adjacent window SNP. Windows with recombination rates in the top one percentile after accounting for chromosome size were removed, as very high recombination rates may indicate map misassembly and/or underestimation of physical distances. All deer chromosomes are acrocentric, with the exception of one unknown autosome (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gustavsson and Sundt 1968</a>). The Build 5 linkage groups maps were orientated in the same direction as the cattle genome, and so we assumed that centromere positions in deer were at the start of the chromosome, as in cattle (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Band <em>et al.</em> 2000</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ma <em>et al.</em> 2015</a>).</p><p id="p-24">In fission events, assuming no change in the original centromere position, one fission chromosome would have retained the centromere (in this case, CEL3, 17, 28, 29, 31, and 33), whereas the other would have had to have formed a new centromere (CEL22, 6, 26, 16, 19, and 8). As all acrocentric chromosomes showed a consistently high recombination rate around the female centromere (see <em>Results</em>), we assumed that neo-centromeres had positioned themselves at the beginning of these chromosomes. We defined chromosome histories as follows: those with fissions retaining the old centromere; fissions that would have formed a new centromere; and chromosomes with no fission or fusion relative to sheep/cattle lineages. Comparison of recombination landscapes between chromosomes of different histories was carried out using general additive models (GAM) from 0 Mb (centromere) to 40 Mb, specifying <em>k</em> = 10, using the R library mgcv v1.8-15 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wood 2011</a>) implemented in R v3.3.2. Recombination rates within each bin were adjusted for chromosome size by dividing the bin rates by the overall chromosome recombination rate (cM/Mb) for each sex. As these chromosome comparisons have a relatively small sample size (<em>n</em> = 32), the GAM analysis was repeated (a) excluding each chromosome and (b) excluding two chromosomes in turn, in order to determine whether the observed effect was driven by one or two chromosomes, respectively. As chromosome sizes are markedly different between fissions retaining a centromere and those forming a new centromere (see Figure S10 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>), comparisons were also made between new centromere chromosomes and unchanged chromosomes of similar size (in this case, CEL6, 8, 16, 22, and 26 <em>vs.</em> CEL2, 7, 10, 24, 27, and 32).</p></div><div id="sec-12" class="subsection"><h3>Transmission distortion</h3><p id="p-25">We conducted a preliminary analysis to identify regions of the genome associated with transmission distortion in the red deer pedigree. Specifically, we wished to determine if regions in close proximity to centromeres had biased transmission, which if occurring close to centromeric regions, may indicate differences in centromere strength in the contemporary pedigree. At a given locus, the specific allele transmitted from an FID to a given offspring can be identified in cases where the FID is heterozygous and its mate is homozygous. For each locus per FID sex, an exact binomial test was used to determine whether the transmission frequency of allele A relative to allele B was significantly different from that expected due to chance. The associated <em>P</em> values were transformed to follow an approximate normal distribution using the equation <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-32.gif"/></noscript></span></span> and each SNP locus was assigned to a 1 Mb bin. General linear models were run in males and females separately for the 10 Mb interval in closest proximity to the centromere on all acrocentric chromosomes, including an interaction term between chromosome history and bin identity.</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-26">The Supplemental Material contains information on additional analyses conducted and is referenced within the text. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS1.txt">Table S1</a> contains the full red deer linkage maps for both sexes, including estimated megabase positions and information on marker informativeness. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS2.txt">Table S2</a> contains comparisons of red deer linkage map positions with cattle and sheep genomes for the X chromosome. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS3.txt">Table S3</a> contains the approximate positions of unmapped loci. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS4.txt">Table S4</a> contains the probabilities of crossing over within 1 Mb windows in both sexes. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS5.txt">Table S5</a> contains BLAST results to determine lineage of origin of chromosome rearrangements. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS6.txt">Table S6</a> contains the per-locus results for the transmission distortion analysis. Raw data, supplementary tables, and sequence information are publicly archived at doi: 10.6084/m9.figshare.5002562. Code for the analysis is archived at <a href="https://github.com/susjoh/DeerMapv4">https://github.com/susjoh/DeerMapv4</a>.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Linkage map</h3><p id="p-27">The predicted sex-averaged red deer linkage map contained 38,083 SNP markers over 33 autosomes and the X chromosome (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>; full map provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS1.txt">Table S1</a>), and had a sex-averaged length of 2739.7 cM (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). A total of 71.6% of intramarker recombination fractions were zero, and so a skeleton map of 10,835 SNPs separated by at least one meiotic crossover was also characterized (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS1.txt">Table S1</a>). The female autosomal map was 1.21 times longer than in males (2767.4 and 2280.8 cM, respectively, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). In the autosomes, we observed six chromosomal fissions, one fusion, and two large and formerly uncharacterized inversions occurring in the deer lineage (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">FileS1</a> and <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Otherwise, the deer map order generally conformed to the cattle map order. The X chromosome had undergone the most differentiation from cattle, with evidence of three translocations, including two in the deer lineage and one in the cattle lineage, and one inversion in the cattle lineage (Figure S5 and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS2.txt">Table S2</a>), although we cannot rule out that this observation is a result of poor assembly of the cattle genome (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Zimin <em>et al.</em> 2009</a>). The estimated positions of 90 unmapped markers are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS4.txt">Table S4</a>. The BLAST results for determining lineage of origin are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS5.txt">Table S5</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486136/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486136/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Marker numbers and sex-averaged and sex-specific map lengths for each deer linkage group in Build 5</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Variation in recombination rate and landscape</h3><p id="p-29">There was a linear relationship between estimated chromosome length and sex-averaged linkage map lengths (adjusted <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-33.gif"/></noscript></span></span> = 0.961, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Smaller chromosomes had higher recombination rates (cM/Mb, adjusted <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-34.gif"/></noscript></span></span> = 0.387, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>), which is likely to be a result of obligate crossing over. Female linkage maps were consistently longer than male linkage maps across all autosomes (adjusted <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-35.gif"/></noscript></span></span> = 0.907, Figure S11 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>) and correlations between estimated map lengths and linkage map lengths were similar in males and females (adjusted <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-36.gif"/></noscript></span></span> = 0.910 and 0.954, respectively; Figure S12 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>). There was no significant difference between the true and sampled map lengths in males and females (Figure S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>), suggesting that the data structure did not introduce bias in estimating sex-specific map lengths.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Broad-scale variation in recombination rate, showing correlations between (A) sex-averaged linkage map length (centimorgan) and estimated chromosome length (megabase) and (B) estimated chromosome length (megabase) and chromosomal recombination rate (centimorgan per megabase). Points are chromosome numbers, and lines and the gray-shaded areas indicate the regression slopes and SEs, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190048829" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Broad-scale variation in recombination rate, showing correlations between (A) sex-averaged linkage map length (centimorgan) and estimated chromosome length (megabase) and (B) estimated chromosome length (megabase) and chromosomal recombination rate (centimorgan per megabase). Points are chromosome numbers, and lines and the gray-shaded areas indicate the regression slopes and SEs, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2859/F3.medium.gif" width="440" height="182"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/8/2859/F3.medium.gif" width="440" height="182"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486050" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Broad-scale variation in recombination rate, showing correlations between (A) sex-averaged linkage map length (centimorgan) and estimated chromosome length (megabase) and (B) estimated chromosome length (megabase) and chromosomal recombination rate (centimorgan per megabase). Points are chromosome numbers, and lines and the gray-shaded areas indicate the regression slopes and SEs, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Fine-scale variation in recombination rate across chromosomes was calculated in 1 Mb windows across the genome; recombination rate was considerably higher in females in the first <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-37.gif"/></noscript></span></span>20% of the chromosome, where the centromere is likely to be situated (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). This effect was consistent across nearly all autosomes (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Male and female recombination rates were not significantly different across the rest of the chromosome, although male recombination was marginally higher than females in subtelomeric regions (<em>i.e.</em>, where the centromere was absent; <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Both sexes showed reduced recombination in subtelomeric regions; this effect is likely to be genuine and not due to reduced ability to infer crossovers within these regions, as the number of phase-informative loci at these loci did not differ from the rest of the chromosome (Figure S14 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>). It should be noted that in some chromosomes, female recombination rates dropped sharply in the first window of the chromosome (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>), indicating that recombination rates are likely to be very low in close proximity to the centromere.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loess smoothed splines of recombination rates across 32 acrocentric autosomes for males and females with a span parameter of 0.15. The centromere is assumed to be at the beginning of the chromosome. Splines for individual chromosomes are shown in Figure 5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190048829" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Loess smoothed splines of recombination rates across 32 acrocentric autosomes for males and females with a span parameter of 0.15. The centromere is assumed to be at the beginning of the chromosome. Splines for individual chromosomes are shown in Figure 5.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2859/F4.medium.gif" width="440" height="294"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/8/2859/F4.medium.gif" width="440" height="294"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485900" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">Loess smoothed splines of recombination rates across 32 acrocentric autosomes for males and females with a span parameter of 0.15. The centromere is assumed to be at the beginning of the chromosome. Splines for individual chromosomes are shown in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loess smoothed splines of recombination rates in 1 Mb windows across 33 autosomes for males and females with a span parameter of 0.2. All chromosomes are acrocentric with the centromere at the beginning of the chromosome (Gustavsson and Sundt 1968), with the likely exception of CEL5. CEL34 is the X chromosome, with the pseudoautosomal region at the telomere end." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190048829" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Loess smoothed splines of recombination rates in 1 Mb windows across 33 autosomes for males and females with a span parameter of 0.2. All chromosomes are acrocentric with the centromere at the beginning of the chromosome (Gustavsson and Sundt 1968), with the likely exception of CEL5. CEL34 is the X chromosome, with the pseudoautosomal region at the telomere end.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2859/F5.medium.gif" width="317" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/8/2859/F5.medium.gif" width="317" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486158" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">Loess smoothed splines of recombination rates in 1 Mb windows across 33 autosomes for males and females with a span parameter of 0.2. All chromosomes are acrocentric with the centromere at the beginning of the chromosome (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Gustavsson and Sundt 1968</a>), with the likely exception of CEL5. CEL34 is the X chromosome, with the pseudoautosomal region at the telomere end.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">GAM of recombination rate variation in acrocentric chromosomes indicated that female recombination rates at the closest proximity to the centromere were higher in fission chromosomes that would have had to have formed a new centromere (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>); this result held when one or two chromosomes were removed (data not shown), and when considering small chromosomes only (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). There were no differences in recombination rates in males with differences in chromosome history (Figure S15 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>). There was no evidence of differences in transmission distortion with chromosome history in closest proximity to the centromeres in either sex, although there were subtle differences 5–6 Mb from the centromere (<em>P</em> &lt; 0.05, Figure S16 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>); full per-locus results are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/TableS6.txt">Table S6</a>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="General additive model curves of adjusted recombination rate in females (k = 10). (A) All acrocentric chromosomes, including fission chromosomes forming a new centromere (n = 6), fission chromosomes retaining the existing centromere (n = 6), and chromosomes with no fission or fusion (n = 20). (B) Small acrocentric chromosomes, including fission chromosomes forming a new centromere (n = 5) and chromosomes with no fission or fusion (n = 6). Dashed lines indicate the SEs. Recombination rates were adjusted for chromosome length (see main text)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190048829" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;General additive model curves of adjusted recombination rate in females (&lt;em&gt;k&lt;/em&gt; = 10). (A) All acrocentric chromosomes, including fission chromosomes forming a new centromere (&lt;em&gt;n&lt;/em&gt; = 6), fission chromosomes retaining the existing centromere (&lt;em&gt;n&lt;/em&gt; = 6), and chromosomes with no fission or fusion (&lt;em&gt;n&lt;/em&gt; = 20). (B) Small acrocentric chromosomes, including fission chromosomes forming a new centromere (&lt;em&gt;n&lt;/em&gt; = 5) and chromosomes with no fission or fusion (&lt;em&gt;n&lt;/em&gt; = 6). Dashed lines indicate the SEs. Recombination rates were adjusted for chromosome length (see main text).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2859/F6.medium.gif" width="440" height="202"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/8/2859/F6.medium.gif" width="440" height="202"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2859/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485924" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-35" class="first-child">General additive model curves of adjusted recombination rate in females (<em>k</em> = 10). (A) All acrocentric chromosomes, including fission chromosomes forming a new centromere (<em>n</em> = 6), fission chromosomes retaining the existing centromere (<em>n</em> = 6), and chromosomes with no fission or fusion (<em>n</em> = 20). (B) Small acrocentric chromosomes, including fission chromosomes forming a new centromere (<em>n</em> = 5) and chromosomes with no fission or fusion (<em>n</em> = 6). Dashed lines indicate the SEs. Recombination rates were adjusted for chromosome length (see main text).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-36">In this study, we constructed predicted and skeleton linkage maps for a wild population of red deer, containing 38,083 and 10,835 SNPs, respectively. Females had higher recombination rates than males, which were driven by significantly higher recombination rates in peri-centromeric regions. These rates were unusually high compared to other mammal species such as cattle, sheep, and humans (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Ma <em>et al.</em> 2015</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Johnston <em>et al.</em> 2016</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kong <em>et al.</em> 2010</a>), and the effect was more pronounced in fission chromosomes that have formed centromeres more recently in their history. Here, we discuss issues related to the map assembly and utility, before proposing two explanations to explain strong heterochiasmy in peri-centromeric regions: (1) that this mechanism may have evolved to counteract centromeric drive associated with meiotic asymmetry in oocyte production and/or (2) that sequence characteristics suppressing recombination in close proximity to the centromere may not have yet evolved at the neo-centromeres.</p><div id="sec-18" class="subsection"><h3>Utility of the red deer linkage map</h3><p id="p-37">The final predicted linkage map included 38,083 SNPs, accounting for 98.8% of polymorphic SNPs within this population. While several large-scale rearrangements were identified in the red deer lineage (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>), marker orders generally corresponded strongly to the cattle genome order. We are confident that the maps presented here are highly accurate for the purposes of genetic analyses outlined in the introduction; however, we also acknowledge that some errors are likely to be present. The limited number of meioses characterized means that we cannot guarantee a correct marker order on the predicted map at the same centimorgan map positions, meaning that some small rearrangements may be undetected within the dataset. Furthermore, the use of the cattle genome to inform initial marker order may also introduce errors in cases of local genome misassembly. Considering these issues, we recommend that the deer marker order is used to verify, rather than inform, any <em>de novo</em> sequence assembly in the red deer or related species.</p></div><div id="sec-19" class="subsection"><h3>Mapping of the X chromosome (CEL34)</h3><p id="p-38">The X chromosome (CEL34) showed the highest level of rearrangement, including two translocations in the deer lineage, one of which was a small region in the pseudoautosomal region (PAR) remapped to the distal end of the chromosome (Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>). However, some caution should be exerted in interpreting whether these rearrangements relative to other species are genuine, as it has been acknowledged that the X chromosome assembly in cattle is of poorer quality in comparison to the autosomes (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Zimin <em>et al.</em> 2009</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Ma <em>et al.</em> 2015</a>). The X chromosome showed a similar pattern to the autosomes in the relationship between estimated chromosome length (megabase) and linkage map length (centimorgan, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). This may seem counterintuitive, as recombination rates in the X should be lower due to it spending one-third of its time in males, where meiotic crossovers only occur on the PAR. However, female map lengths were generally longer, and 64% of the meioses used to inform sex-averaged maps occurred in females; furthermore, the female-specific map showed that the X conformed to the expected map length (Figure S12 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>). Therefore, the linkage map length of the X is as expected; however, we acknowledge that some errors or inflation may be present on the X given that fewer informative meioses occur in non-PAR regions.</p></div><div id="sec-20" class="subsection"><h3>Predicting centromere positioning on the deer linkage groups</h3><p id="p-39">Cytogenetic studies have shown that deer chromosomes are acrocentric (<em>i.e.</em>, the centromere is situated at one end of the chromosome), with the exception of one unknown metacentric autosome, which is one of the physically largest (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Gustavsson and Sundt 1968</a>). Our results suggest that the strongest candidate is CEL5, which has undergone a fusion event in the deer lineage (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). Unlike other autosomes, this linkage group shows strong concordance between male and female centimorgan maps (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), elevated male recombination rate at the chromosome ends, and reduced recombination in a <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-38.gif"/></noscript></span></span>8 Mb region that corresponds with the fusion site at the centromeric regions of BTA17 and BTA19 (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). On the acrocentric chromosomes, we have assumed that centromeres are at the beginning of each linkage group, based on synteny of centromere positions with the cattle genome (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Ma <em>et al.</em> 2015</a>). There is evidence that centromeres can change position on mammalian chromosomes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Carbone <em>et al.</em> 2006</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Graphodatsky <em>et al.</em> 2011</a>). However, the frequency of this is sufficiently low, and recombination patterns so consistent in our dataset (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>) that we believe our assumption is justified, particularly for chromosomes that have not undergone fission or fusion events in either lineage (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). Six of the fission chromosomes (CEL6, CEL8, CEL16, CEL19, CEL22, and CEL26) would have had to form new centromeres in the deer lineage. Direct orientation with the cattle genome shows similar patterns of recombination to other chromosomes (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>), indicating that telomeric regions have most likely not changed, and that centromeres have positioned themselves at the beginning of the chromosomes. Nevertheless, we acknowledge that confirmation of centromeric positions will require further investigation.</p></div><div id="sec-21" class="subsection"><h3>Sexual dimorphism in recombination landscape: a consequence of centromeric drive?</h3><p id="p-40">Females had considerably higher recombination rates in peri-centromeric regions, resulting in female-biased recombination rates overall; recombination rates along the remainder of the chromosome were similar in both sexes, and lowest at the closest proximity to the telomere (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Sampling identical numbers of males and females with replacement confirmed that this observation is unlikely to be a result of differences in sample sizes between the sexes (Figure S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1/FileS1.pdf">File S1</a>). Identifying female-biased heterochiasmy is not unusual, as recombination rates in placental mammals are generally higher in females, particularly toward the centromere (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Lenormand and Dutheil 2005</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Brandvain and Coop 2012</a>). Nevertheless, the patterns of recombination rate variation observed in this dataset are striking for several reasons. First, our findings are distinct from the other ruminants, namely cattle and sheep, which both exhibit male-biased heterochiasmy driven by elevated male recombination rates in subtelomeric regions, with males exhibiting higher recombination rates in peri-centromeric regions (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Ma <em>et al.</em> 2015</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Johnston <em>et al.</em> 2016</a>). Indeed, all other mammal studies to date show increased male subtelomeric recombination even if female recombination rates are higher overall (<em>e.g.</em>, in humans and mice; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Kong <em>et al.</em> 2010</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Liu <em>et al.</em> 2014</a>). Second, while female recombination rates tend to be relatively higher in peri-centromeric regions in many species, the degree of difference is relatively small compared to that observed in the deer, and is generally suppressed in very close proximity the centromere (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Brandvain and Coop 2012</a>); in this study, we observed female recombination rates of 2–5 times that of males within this region.</p><p id="p-41">There are several hypotheses proposed to explain increased recombination rates in female mammals (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Brandvain and Coop 2012</a>). The most prevalent has been the idea that increased crossover number in females protects against aneuploidy (<em>i.e.</em>, nondisjunction) after long periods of meiotic arrest during Prophase I (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Morelli and Cohen 2005</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Coop and Przeworski 2007</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Nagaoka <em>et al.</em> 2012</a>). While we cannot rule this out as a potential driver, this hypothesis dominates the human literature, where females have one of the longest meiotic arrests of any mammal (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Burt and Bell 1987</a>). Female deer on Rum reach sexual maturity at a relatively young age (<span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2859/embed/mml-math-39.gif"/></noscript></span></span>1.5–2.5 yr of age) compared to other mammals, such as monkeys and great apes, and similar to other ruminants such as sheep and cattle (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Burt and Bell 1987</a>). A more compelling hypothesis relates to the role of meiotic drive, where asymmetry in meiotic cell divisions in females can be exploited by selfish genetic elements associated with the centromere (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Brandvain and Coop 2012</a> and <em>Introduction</em>), where higher female recombination at the peri-centromeric regions may counteract centromeric drive by increasing the uncertainty associated with segregation into the egg (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Haig and Grafen 1991</a>). In addition to a global mechanism driving local increases in recombination, our observation that chromosomes with newer centromeres show increased recombination in close proximity to the centromere may support this idea (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). While there is still generally little consensus on the mechanisms related to the formation of new centromeres (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rocchi <em>et al.</em> 2011</a>), conflict between centromeric proteins and repetitive centromeric DNA may lead to rapid evolution of centromere strength in a new or recently formed centromere (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Rosin and Mellone 2017</a>). Increased recombination rates in close proximity to a newer centromere could provide a mechanism to counter stronger drive.</p><p id="p-42">However, there are alternative (but not exclusive) arguments to this in the current dataset. The first is that increased recombination on new centromere chromosomes may be because sequence characteristics suppressing peri-centromeric recombination have not yet evolved in proximity to more recent centromeres. This is supported by our findings that there is no evidence of transmission distortion at centromeric regions on any of the chromosomes, although it also can be argued that centromeres have stabilized in the contemporary population, and that our current study cannot investigate historical differences in centromere strength. Additionally, the observed effect may be partially driven by mapping errors at the chromosome ends, particularly if polymorphisms in close proximity to the centromere have not been characterized and/or mapped.</p></div><div id="sec-22" class="subsection"><h3>Conclusions</h3><p id="p-43">Our study has created a new linkage map resource for red deer and will facilitate genome-wide studies and genome assembly projects in red deer and related species. We have argued that increased recombination at peri-centromeric regions in females may be a mechanism to counteract meiotic drive; however, testing this hypothesis will require further investigation. Cytogenetic studies will allow confirmation of centromere positioning, and will give insight into how and why chromatin and cohesin structure does/does not suppress recombination in the pericentromeric region, and how their dynamics vary across the deer lineage (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Vincenten <em>et al.</em> 2015</a>). In addition, <em>de novo</em> genome assemblies have the potential to verify map orders where possible; sequencing multiple deer genomes will allow us to determine population-scale recombination rates and hotspots (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chan <em>et al.</em> 2012</a>), with the potential to investigate historical variation in rate based on signatures of biased gene conversion (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Capra <em>et al.</em> 2013</a>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">We thank T. Clutton-Brock, F. Guinness, S. Albon, A. Morris, S. Morris, M. Baker, and many others for collecting field data and DNA samples and their important contributions to the long-term Rum deer project. Discussions with C. Bérénos, J. Risse, and M. D. Edge aided the study; we also appreciate discussion with G. Coop, T. Lenormand, L. Ross, D. Charlesworth, L. Ma, and many Twitter users (Y. Brandvain, L. Holman, A. Kern, A. Mason, T. Price, and L. Theodosiou) on interpreting the pattern of high peri-centromeric recombination in females. We thank Scottish Natural Heritage for permission to work on the Isle of Rum National Nature Reserve and the Wellcome Trust Clinical Research Facility Genetics Core in Edinburgh for performing the genotyping. This work has made extensive use of the resources provided by the University of Edinburgh Compute and Data Facility (<a href="http://www.ecdf.ed.ac.uk/">http://www.ecdf.ed.ac.uk/</a>). The long-term Rum deer project is funded by the UK Natural Environment Research Council, and single nucleotide polymorphism genotyping was supported by a European Research Council Advanced Grant to J.M.P. S.E.J. is supported by a Royal Society University Research Fellowship.</p><p id="p-45">Author contributions: J.M.P. and J.H. organized the collection of samples. P.A.E. and J.H. conducted DNA sample extraction and genotyping. S.E.J. designed the study, analyzed the data, and wrote the paper. All authors contributed to revisions.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-46">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044198/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-47"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-04-05"><span class="received-label">Received </span>April 5, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-06-27"><span class="accepted-label">Accepted </span>June 27, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Johnston <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.1" data-doi="10.1093/bioinformatics/btm108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isaacs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Duijn</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">GenABEL: an R library for genome-wide association analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1294</span>–<span class="cit-lpage">1296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm108%26rft_id%253Dinfo%253Apmid%252F17384015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17384015&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247348300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.2" data-doi="10.1101/gr.145900"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Band</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rebeiz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heyen</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">An ordered comparative map of the cattle and human genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1359</span>–<span class="cit-lpage">1368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DBand%26rft.auinit1%253DM.%2BR.%26rft.volume%253D10%26rft.issue%253D9%26rft.spage%253D1359%26rft.epage%253D1368%26rft.atitle%253DAn%2BOrdered%2BComparative%2BMap%2Bof%2Bthe%2BCattle%2Band%2BHuman%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.145900%26rft_id%253Dinfo%253Apmid%252F10984454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEwLzkvMTM1OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yODU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.3" data-doi="10.1111/mec.13146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bérénos</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilkington</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gratten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Heterogeneity of genetic architecture of body size traits in a free-living population.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1810</span>–<span class="cit-lpage">1830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D1810%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.4" data-doi="10.1111/eva.12026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borza</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic islands of divergence and their consequences for the resolution of spatial structure in an exploited marine fish.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">450</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D6%26rft.spage%253D450%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12026%26rft_id%253Dinfo%253Apmid%252F23745137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23745137&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.5" data-doi="10.1534/genetics.111.136721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandvain</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Scrambling eggs: meiotic drive and the evolution of female recombination rates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.136721%26rft_id%253Dinfo%253Apmid%252F22143919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNzA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI4NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-web" id="cit-7.8.2859.6"><div class="cit-metadata unstructured">Brauning, R., P. J. Fisher, A. F. McCulloch, R. J. Smithies, J. F. Ward <em>et al.</em>, 2015 Utilization of high throughput genome sequencing technology for large scale single nucleotide polymorphism discovery in red deer and Canadian elk. bioRxiv Available at: <a href="http://www.biorxiv.org/content/early/2015/09/23/027318">http://www.biorxiv.org/content/early/2015/09/23/027318</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brieuc</span>  <span class="cit-name-given-names">M. S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A dense linkage map for chinook salmon (<em>Oncorhynchus tshawytscha</em>) reveals variable chromosomal divergence after an ancestral whole genome duplication event.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.8" data-doi="10.1554/0014-3820(2000)054[0337:PSRATE]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Sex, recombination, and the efficacy of selection—was Weismann right?</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBurt%26rft.auinit1%253DA.%26rft.volume%253D54%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D351%26rft.atitle%253DPerspective%253A%2Bsex%252C%2Brecombination%252C%2Band%2Bthe%2Befficacy%2Bof%2Bselection--was%2BWeismann%2Bright%253F%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282000%2529054%255B0337%253APSRATE%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F10937212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2000)054[0337:PSRATE]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10937212&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086886500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.9" data-doi="10.1038/326803a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Mammalian chiasma frequencies as a test of two theories of recombination.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBurt%26rft.auinit1%253DA.%26rft.volume%253D326%26rft.issue%253D6115%26rft.spage%253D803%26rft.epage%253D805%26rft.atitle%253DMammalian%2Bchiasma%2Bfrequencies%2Bas%2Ba%2Btest%2Bof%2Btwo%2Btheories%2Bof%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F326803a0%26rft_id%253Dinfo%253Apmid%252F3574451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/326803a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3574451&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G979300069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.10" data-doi="10.1186/1471-2105-10-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avagyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DMa%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DRegularized%2Bgene%2Bselection%2Bin%2Bcancer%2Bmicroarray%2Bmeta-analysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-1%26rft_id%253Dinfo%253Apmid%252F19118496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19118496&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.11" data-doi="10.1371/journal.pgen.1003529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capra</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kostka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollard</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siepel</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A model-based analysis of GC-biased gene conversion in the human and chimpanzee genomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.12" data-doi="10.1016/j.ygeno.2005.11.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carbone</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nergadze</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magnani</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misceo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardone</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evolutionary movement of centromeres in horse, donkey, and zebra.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">777</span>–<span class="cit-lpage">782</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DCarbone%26rft.auinit1%253DL.%26rft.volume%253D87%26rft.issue%253D6%26rft.spage%253D777%26rft.epage%253D782%26rft.atitle%253DEvolutionary%2Bmovement%2Bof%2Bcentromeres%2Bin%2Bhorse%252C%2Bdonkey%252C%2Band%2Bzebra.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2005.11.012%26rft_id%253Dinfo%253Apmid%252F16413164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2005.11.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16413164&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238168400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide fine-scale recombination rate variation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.14" data-doi="10.1017/S0016672300033450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Recombination load associated with selection for increased recombination.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D67%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D41%26rft.atitle%253DRecombination%2Bload%2Bassociated%2Bwith%2Bselection%2Bfor%2Bincreased%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300033450%26rft_id%253Dinfo%253Apmid%252F8919888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300033450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8919888&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UE29800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.15" data-doi="10.1016/j.cub.2014.08.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chmátal</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabriel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitsainas</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martínez-Vargas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ventura</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Centromere strength provides the cell biological basis for meiotic drive and karyotype evolution in mice.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2295</span>–<span class="cit-lpage">2300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D24%26rft.spage%253D2295%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2014.08.017%26rft_id%253Dinfo%253Apmid%252F25242031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2014.08.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25242031&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2859.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clutton-Brock</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guinness</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albon</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-source"><em>Red Deer. Behaviour and Ecology of Two Sexes</em></span>. <span class="cit-publ-name">University of Chicago Press</span>, <span class="cit-publ-loc">Chicago</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.17" data-doi="10.1038/nrg1947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">An evolutionary view of human recombination.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDennis%2BLo%26rft.auinit1%253DY.%2BM.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D34%26rft.atitle%253DAn%2Bevolutionary%2Bview%2Bof%2Bhuman%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1947%26rft_id%253Dinfo%253Apmid%252F17146469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1947&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17146469&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242934300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.18" data-doi="10.1016/j.tree.2013.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequencing and population genomics in non-model organisms.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D29%26rft.spage%253D51%26rft.atitle%253DGENOME%2BSEQUENCING%2BAND%2BPOPULATION%2BGENOMICS%2BIN%2BNON-MODEL%2BORGANISMS%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2013.09.008%26rft_id%253Dinfo%253Apmid%252F24139972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2013.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24139972&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330148000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The evolutionary advantage of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D737%26rft.epage%253D756%26rft.atitle%253DTHE%2BEVOLUTIONARY%2BADVANTAGE%2BOF%2BRECOMBINATION%26rft_id%253Dinfo%253Apmid%252F4448362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzgvMi83MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjg1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fierst</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Using linkage maps to correct and scaffold <em>de novo</em> genome assemblies: methods, challenges, and computational tools.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D220%26rft_id%253Dinfo%253Apmid%252F26150829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26150829&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.21" data-doi="10.1371/journal.pgen.1000658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Broad-scale recombination patterns underlying proper disjunction in humans.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFledel-Alon%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De1000658%26rft.epage%253De1000658%26rft.atitle%253DBroad-scale%2Brecombination%2Bpatterns%2Bunderlying%2Bproper%2Bdisjunction%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000658%26rft_id%253Dinfo%253Apmid%252F19763175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000658&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19763175&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fountain</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ravinet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naylor</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinhardt</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butlin</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A linkage map and QTL analysis for pyrethroid resistance in the bed bug <em>Cimex lectularius</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">4059</span>–<span class="cit-lpage">4066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D4059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.23" data-doi="10.1111/j.1365-294X.2006.03022.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frantz</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pourtois</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heuertz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schley</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flamand</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic structure and assignment tests demonstrate illegal translocation of red deer (<em>Cervus elaphus</em>) into a continuous population.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">3191</span>–<span class="cit-lpage">3203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DFrantz%26rft.auinit1%253DA.%2BC.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D3191%26rft.epage%253D3203%26rft.atitle%253DGenetic%2Bstructure%2Band%2Bassignment%2Btests%2Bdemonstrate%2Billegal%2Btranslocation%2Bof%2Bred%2Bdeer%2B%2528Cervus%2Belaphus%2529%2Binto%2Ba%2Bcontinuous%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2006.03022.x%26rft_id%253Dinfo%253Apmid%252F16968264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2006.03022.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16968264&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240440800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.24" data-doi="10.1186/1755-8166-4-22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graphodatsky</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trifonov</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanyon</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The genome diversity and karyotype evolution of mammals.</span> <abbr class="cit-jnl-abbrev">Mol. Cytogenet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BCytogenet%26rft.aulast%253DGraphodatsky%26rft.auinit1%253DA.%2BS.%26rft.volume%253D4%26rft.spage%253D22%26rft.epage%253D22%26rft.atitle%253DThe%2Bgenome%2Bdiversity%2Band%2Bkaryotype%2Bevolution%2Bof%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1755-8166-4-22%26rft_id%253Dinfo%253Apmid%252F21992653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1755-8166-4-22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21992653&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2859.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falls</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooks</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source"><em>Documentation for CRIMAP</em>, <em>version 2.4</em></span>. <span class="cit-publ-name">Washington University School of Medicine</span>, <span class="cit-publ-loc">St. Louis</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gustavsson</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundt</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Karyotypes in five species of deer (<em>Alces alces</em> l., <em>Capreolus capreolus</em> l., <em>Cervus elaphus</em> l., <em>Cervus nippon nippon</em> temm. and <em>Dama dama</em> l.).</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DGustavsson%26rft.auinit1%253DI.%26rft.volume%253D60%26rft.issue%253D1%26rft.spage%253D233%26rft.epage%253D248%26rft.atitle%253DKaryotypes%2Bin%2Bfive%2Bspecies%2Bof%2Bdeer%2B%2528Alces%2Balces%2BL.%252C%2BCapreolus%2Bcapreolus%2BL.%252C%2BCervus%2Belaphus%2BL.%252C%2BCervus%2Bnippon%2Bnippon%2BTemm.%2Band%2BDama%2Bdama%2BL.%2529.%26rft_id%253Dinfo%253Apmid%252F5721324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=5721324&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.27" data-doi="10.1016/S0022-5193(05)80155-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grafen</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Genetic scrambling as a defence against meiotic drive.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DHaig%26rft.auinit1%253DD.%26rft.volume%253D153%26rft.issue%253D4%26rft.spage%253D531%26rft.epage%253D558%26rft.atitle%253DGenetic%2Bscrambling%2Bas%2Ba%2Bdefence%2Bagainst%2Bmeiotic%2Bdrive.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-5193%252805%252980155-9%26rft_id%253Dinfo%253Apmid%252F1806752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-5193(05)80155-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1806752&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GY28500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.28" data-doi="10.1038/35066065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hassold</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">To err (meiotically) is human: the genesis of human aneuploidy.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHassold%26rft.auinit1%253DT.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D280%26rft.epage%253D291%26rft.atitle%253DTo%2Berr%2B%2528meiotically%2529%2Bis%2Bhuman%253A%2Bthe%2Bgenesis%2Bof%2Bhuman%2Baneuploidy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35066065%26rft_id%253Dinfo%253Apmid%252F11283700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35066065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11283700&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167837900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.29" data-doi="10.1093/molbev/msv037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedges</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suleski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paymer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Tree of life reveals clock-like speciation and diversification.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">835</span>–<span class="cit-lpage">845</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv037%26rft_id%253Dinfo%253Apmid%252F25739733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25739733&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.30" data-doi="10.1111/1755-0998.12665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huisman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Pedigree reconstruction for SNP data: parentage assignment, sibship clustering and beyond.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/1755-0998.12665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12665&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.31" data-doi="10.1073/pnas.1518046113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huisman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruuk</span>  <span class="cit-name-given-names">L. E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clutton-Brock</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Inbreeding depression across the lifespan in a wild mammal population.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">3585</span>–<span class="cit-lpage">3590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1518046113%26rft_id%253Dinfo%253Apmid%252F26979959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzEzLzM1ODUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjg1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.32" data-doi="10.1534/genetics.115.185553"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérénos</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Conserved genetic architecture underlying individual recombination rate variation in a wild population of soay sheep (<em>Ovis aries</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.185553%26rft_id%253Dinfo%253Apmid%252F27029733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzEvNTgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI4NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kardos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomics advances the study of inbreeding depression in the wild.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1205</span>–<span class="cit-lpage">1218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D9%26rft.spage%253D1205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.34" data-doi="10.1111/mec.12810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smeds</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Backström</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Husby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qvarnström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-density linkage map enables a second-generation collared flycatcher genome assembly and reveals the patterns of avian recombination rate variation and chromosomal evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4035</span>–<span class="cit-lpage">4058</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D4035%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12810%26rft_id%253Dinfo%253Apmid%252F24863701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12810&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24863701&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340419800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.35" data-doi="10.1038/nature09525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorleifsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigurdsson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fine-scale recombination rate differences between sexes, populations and individuals.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKong%26rft.auinit1%253DA.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1099%26rft.epage%253D1103%26rft.atitle%253DFine-scale%2Brecombination%2Brate%2Bdifferences%2Bbetween%2Bsexes%252C%2Bpopulations%2Band%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09525%26rft_id%253Dinfo%253Apmid%252F20981099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981099&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.36" data-doi="10.1554/0014-3820(2002)056[1683:ASIRDH]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruuk</span>  <span class="cit-name-given-names">L. E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brotherstone</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guinness</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Antler size in red deer: heritability and selection but no evolution.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1683</span>–<span class="cit-lpage">1695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DKruuk%26rft.auinit1%253DE.%2BB.%26rft.volume%253D56%26rft.issue%253D8%26rft.spage%253D1683%26rft.epage%253D1695%26rft.atitle%253DAntler%2Bsize%2Bin%2Bred%2Bdeer%253A%2Bheritability%2Band%2Bselection%2Bbut%2Bno%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282002%2529056%255B1683%253AASIRDH%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F12353761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2002)056[1683:ASIRDH]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12353761&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178154400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schork</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic dissection of complex traits.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">265</span>: <span class="cit-fpage">5181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D265%26rft.spage%253D5181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.8.2859.38" data-doi="10.1534/g3.116.038497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leitwein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guinand</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pouzadoux</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desmarais</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berrebi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A dense brown trout (<em>Salmo trutta</em>) linkage map reveals recent chromosomal rearrangements in the salmo genus and the impact of selection on linked neutral diversity.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1365</span>–<span class="cit-lpage">1376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.038497%26rft_id%253Dinfo%253Apmid%252F28235829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNC8xMzY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI4NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.39" data-doi="10.1371/journal.pbio.0030063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenormand</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutheil</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Recombination difference between sexes: a role for haploid selection.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DLenormand%26rft.auinit1%253DT.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253De63%26rft.epage%253De63%26rft.atitle%253DRecombination%2Bdifference%2Bbetween%2Bsexes%253A%2Ba%2Brole%2Bfor%2Bhaploid%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030063%26rft_id%253Dinfo%253Apmid%252F15736976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15736976&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.40" data-doi="10.1534/genetics.114.161653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-resolution sex-specific linkage maps of the mouse reveal polarized distribution of crossovers in male germline.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161653%26rft_id%253Dinfo%253Apmid%252F24578350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk3LzEvOTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjg1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.41" data-doi="10.1371/journal.pgen.1005387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padhi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Cattle sex-specific recombination and genetic control from a large pedigree analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005387%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005387%26rft_id%253Dinfo%253Apmid%252F26540184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26540184&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.42" data-doi="10.1017/S0016672309990255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mank</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The evolution of heterochiasmy: the role of sexual selection and sperm competition in determining sex-specific recombination rates in eutherian mammals.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DMank%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D91%26rft.issue%253D5%26rft.spage%253D355%26rft.epage%253D363%26rft.atitle%253DThe%2Bevolution%2Bof%2Bheterochiasmy%253A%2Bthe%2Brole%2Bof%2Bsexual%2Bselection%2Band%2Bsperm%2Bcompetition%2Bin%2Bdetermining%2Bsex-specific%2Brecombination%2Brates%2Bin%2Beutherian%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672309990255%26rft_id%253Dinfo%253Apmid%252F19922699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672309990255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19922699&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Uchida</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An integrated linkage map reveals candidate genes underlying adaptive variation in chinook salmon (<em>Oncorhynchus tshawytscha</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.44" data-doi="10.1530/rep.1.00865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morelli</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">P. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Not all germ cells are created equal: aspects of sexual dimorphism in mammalian meiosis.</span> <abbr class="cit-jnl-abbrev">Reproduction</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">781</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReproduction%26rft.stitle%253DReproduction%26rft.aulast%253DMorelli%26rft.auinit1%253DM.%2BA%26rft.volume%253D130%26rft.issue%253D6%26rft.spage%253D761%26rft.epage%253D781%26rft.atitle%253DNot%2Ball%2Bgerm%2Bcells%2Bare%2Bcreated%2Bequal%253A%2BAspects%2Bof%2Bsexual%2Bdimorphism%2Bin%2Bmammalian%2Bmeiosis%26rft_id%253Dinfo%253Adoi%252F10.1530%252Frep.1.00865%26rft_id%253Dinfo%253Apmid%252F16322537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoicmVwcm9kIjtzOjU6InJlc2lkIjtzOjk6IjEzMC82Lzc2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yODU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.45" data-doi="10.1016/0027-5107(64)90047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">The relation of recombination to mutational advance.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%26rft.volume%253D1%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252864%252990047-8%26rft_id%253Dinfo%253Apmid%252F14195748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(64)90047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14195748&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1964WT37300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.46" data-doi="10.1038/nrg3245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagaoka</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hassold</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Human aneuploidy: mechanisms and new insights into an age-old problem.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNagaoka%26rft.auinit1%253DS.%2BI.%26rft.volume%253D13%26rft.issue%253D7%26rft.spage%253D493%26rft.epage%253D504%26rft.atitle%253DHuman%2Baneuploidy%253A%2Bmechanisms%2Band%2Bnew%2Binsights%2Binto%2Ban%2Bage-old%2Bproblem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3245%26rft_id%253Dinfo%253Apmid%252F22705668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22705668&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sapienza</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Female meiosis drives karyotypic evolution in mammals.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1179</span>–<span class="cit-lpage">1189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPardo-Manuel%2Bde%2BVillena%26rft.auinit1%253DF.%26rft.volume%253D159%26rft.issue%253D3%26rft.spage%253D1179%26rft.epage%253D1189%26rft.atitle%253DFemale%2BMeiosis%2BDrives%2BKaryotypic%2BEvolution%2Bin%2BMammals%26rft_id%253Dinfo%253Apmid%252F11729161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OS8zLzExNzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjg1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.48" data-doi="10.1016/j.tig.2011.06.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phadnis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyppa</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">New and old ways to control meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DPhadnis%26rft.auinit1%253DN.%26rft.volume%253D27%26rft.issue%253D10%26rft.spage%253D411%26rft.epage%253D421%26rft.atitle%253DNew%2Band%2Bold%2Bways%2Bto%2Bcontrol%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2011.06.007%26rft_id%253Dinfo%253Apmid%252F21782271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2011.06.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21782271&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295489100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.49" data-doi="10.1093/gbe/evv250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calboli</span>  <span class="cit-name-given-names">F. C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shikano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Construction of ultradense linkage maps with Lep-MAP2: stickleback F2 recombinant crosses as an example.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv250%26rft_id%253Dinfo%253Apmid%252F26668116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26668116&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archidiacono</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schempp</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capozzi</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanyon</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Centromere repositioning in mammals.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D108%26rft.spage%253D59%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosin</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mellone</span>  <span class="cit-name-given-names">B. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Centromeres drive a hard bargain.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D33%26rft.spage%253D101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.52" data-doi="10.1111/j.1365-294X.2008.04051.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Senn</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Variable extent of hybridization between invasive sika (<em>Cervus nippon</em>) and native red deer (<em>C. elaphus</em>) in a small geographical area.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">862</span>–<span class="cit-lpage">876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSenn%26rft.auinit1%253DH.%2BV.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D862%26rft.epage%253D876%26rft.atitle%253DVariable%2Bextent%2Bof%2Bhybridization%2Bbetween%2Binvasive%2Bsika%2B%2528Cervus%2Bnippon%2529%2Band%2Bnative%2Bred%2Bdeer%2B%2528C.%2Belaphus%2529%2Bin%2Ba%2Bsmall%2Bgeographical%2Barea.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.04051.x%26rft_id%253Dinfo%253Apmid%252F19175500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.04051.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19175500&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263521800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Stijn</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEwan</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maqbool</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A deer (subfamily <em>Cervinae</em>) genetic linkage map and the evolution of ruminant genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1587</span>–<span class="cit-lpage">1597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSlate%26rft.auinit1%253DJ.%26rft.volume%253D160%26rft.issue%253D4%26rft.spage%253D1587%26rft.epage%253D1597%26rft.atitle%253DA%2BDeer%2B%2528Subfamily%2BCervinae%2529%2BGenetic%2BLinkage%2BMap%2Band%2Bthe%2BEvolution%2Bof%2BRuminant%2BGenomes%26rft_id%253Dinfo%253Apmid%252F11973312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MC80LzE1ODciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjg1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.54" data-doi="10.1038/hdy.2011.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smukowski</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recombination rate variation in closely related species.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">496</span>–<span class="cit-lpage">508</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DSmukowski%26rft.auinit1%253DC.%2BS.%26rft.volume%253D107%26rft.issue%253D6%26rft.spage%253D496%26rft.epage%253D508%26rft.atitle%253DRecombination%2Brate%2Bvariation%2Bin%2Bclosely%2Brelated%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.44%26rft_id%253Dinfo%253Apmid%252F21673743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.44&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21673743&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297214900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.55" data-doi="10.1534/genetics.107.086264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stapley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birkhead</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A linkage map of the zebra finch <em>Taeniopygia guttata</em> provides new insights into avian genome evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">651</span>–<span class="cit-lpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStapley%26rft.auinit1%253DJ.%26rft.volume%253D179%26rft.issue%253D1%26rft.spage%253D651%26rft.epage%253D667%26rft.atitle%253DA%2BLinkage%2BMap%2Bof%2Bthe%2BZebra%2BFinch%2BTaeniopygia%2Bguttata%2BProvides%2BNew%2BInsights%2BInto%2BAvian%2BGenome%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.086264%26rft_id%253Dinfo%253Apmid%252F18493078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc5LzEvNjUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI4NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.56" data-doi="10.1002/jez.1400140104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1913</span> <span class="cit-article-title">The linear arrangement of six sex-linked factors in <em>Drosophila</em>, as shown by their mode of association.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zool.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BZool.%26rft.volume%253D14%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1400140104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1400140104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000187376000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2859.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trivers</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Sex differences in rates of recombination and sexual selection</span>, pp. <span class="cit-fpage">270</span>–<span class="cit-lpage">286</span> in <span class="cit-source"><em>The Evolution of Sex</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Michod</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">B.</span></span>. <span class="cit-publ-name">Sinauer Press</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vincenten</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhl</span>  <span class="cit-name-given-names">L.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The kinetochore prevents centromere-proximal crossover recombination during meiosis.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e10850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D4%26rft.spage%253De10850%26rft_id%253Dinfo%253Apmid%252F26653857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26653857&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.59" data-doi="10.1111/j.1467-9868.2010.00749.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fast stable restricted maximum likelihood and marginal likelihood estimation of semiparametric generalized linear models.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D73%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2010.00749.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2010.00749.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2859.60" data-doi="10.1186/gb-2009-10-4-r42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Florea</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A whole-genome assembly of the domestic cow, <em>Bos taurus</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DZimin%26rft.auinit1%253DA.%2BV.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253DR42%26rft.epage%253DR42%26rft.atitle%253DA%2Bwhole-genome%2Bassembly%2Bof%2Bthe%2Bdomestic%2Bcow%252C%2BBos%2Btaurus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-4-r42%26rft_id%253Dinfo%253Apmid%252F19393038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-4-r42&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19393038&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2859.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/8/2859.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2855" title="Gene2Function: An Integrated Online Resource for Gene Function Discovery" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2871" title="An RNA Recognition Motif-Containing Protein Functions in Meiotic Silencing by Unpaired DNA" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/8"><span>Volume 7 Issue 8, August 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/8.cover-source.jpg?itok=apH_YDuu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2859" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-H7ziPMRuBi1kd9h_oV7RH02Lzhyr9khkR78GuGV7aK8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/8/2859" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/8/2859" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/8/2859" class="active" data-icon-position="" data-hide-link-title="0">A High-Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (Cervus elaphus)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485843" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_rupqP6juVc1jKhxVJtDo-yMQu5uUTsXymP120IvPlc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2859.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/8/2859" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-IAyUyDPTHA_iNz3Aj9fDgMKoigP2dpJ4e5R21Fn3vUU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/8/2859" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2859.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485843" id="node485843--2" data-pisa="ggg;7/8/2859" data-pisa-master="ggg;g3.117.044198" data-apath="/ggg/7/8/2859.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (<em>Cervus elaphus</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5623-8902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Susan E. Johnston</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9744-7196" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jisca Huisman</span>, <span class="highwire-citation-author" data-delta="2">Philip A. Ellis</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0075-1504" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Josephine M. Pemberton</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2859-2870;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.044198 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485843--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Susan E. Johnston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BE.%2BJohnston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnston%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BE.%2BJohnston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5623-8902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Susan E. Johnston</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Susan.Johnston@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">Susan.Johnston@ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jisca Huisman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jisca%2BHuisman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huisman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJisca%2BHuisman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9744-7196" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jisca Huisman</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Philip A. Ellis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philip%2BA.%2BEllis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ellis%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilip%2BA.%2BEllis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Josephine M. Pemberton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josephine%2BM.%2BPemberton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pemberton%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJosephine%2BM.%2BPemberton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0075-1504" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Josephine M. Pemberton</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/8/2859" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/8/2859&amp;title=A%20High-Density%20Linkage%20Map%20Reveals%20Sexual%20Dimorphism%20in%20Recombination%20Landscapes%20in%20Red%20Deer%20%28Cervus%20elaphus%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/8/2859&amp;title=A%20High-Density%20Linkage%20Map%20Reveals%20Sexual%20Dimorphism%20in%20Recombination%20Landscapes%20in%20Red%20Deer%20%28Cervus%20elaphus%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/8/2859&amp;title=A%20High-Density%20Linkage%20Map%20Reveals%20Sexual%20Dimorphism%20in%20Recombination%20Landscapes%20in%20Red%20Deer%20%28Cervus%20elaphus%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/8/2859&amp;text=A%20High-Density%20Linkage%20Map%20Reveals%20Sexual%20Dimorphism%20in%20Recombination%20Landscapes%20in%20Red%20Deer%20%28Cervus%20elaphus%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/8/2859&amp;title=A%20High-Density%20Linkage%20Map%20Reveals%20Sexual%20Dimorphism%20in%20Recombination%20Landscapes%20in%20Red%20Deer%20%28Cervus%20elaphus%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/8/2859&amp;t=A%20High-Density%20Linkage%20Map%20Reveals%20Sexual%20Dimorphism%20in%20Recombination%20Landscapes%20in%20Red%20Deer%20%28Cervus%20elaphus%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/8/2859&amp;title=A%20High-Density%20Linkage%20Map%20Reveals%20Sexual%20Dimorphism%20in%20Recombination%20Landscapes%20in%20Red%20Deer%20%28Cervus%20elaphus%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/8/2859&amp;title=A%20High-Density%20Linkage%20Map%20Reveals%20Sexual%20Dimorphism%20in%20Recombination%20Landscapes%20in%20Red%20Deer%20%28Cervus%20elaphus%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485843/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485843" id="node485843--3" data-pisa="ggg;7/8/2859" data-pisa-master="ggg;g3.117.044198" data-apath="/ggg/7/8/2859.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Linkage Map Reveals Sexual Dimorphism in Recombination Landscapes in Red Deer (<em>Cervus elaphus</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5623-8902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Susan E. Johnston</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9744-7196" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jisca Huisman</span>, <span class="highwire-citation-author" data-delta="2">Philip A. Ellis</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0075-1504" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Josephine M. Pemberton</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2859-2870;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.044198 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485843--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Susan E. Johnston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BE.%2BJohnston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnston%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BE.%2BJohnston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5623-8902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Susan E. Johnston</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Susan.Johnston@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">Susan.Johnston@ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jisca Huisman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jisca%2BHuisman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huisman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJisca%2BHuisman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9744-7196" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jisca Huisman</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Philip A. Ellis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philip%2BA.%2BEllis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ellis%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilip%2BA.%2BEllis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Josephine M. Pemberton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, University of Edinburgh, EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josephine%2BM.%2BPemberton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pemberton%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJosephine%2BM.%2BPemberton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0075-1504" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Josephine M. Pemberton</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2859&amp;rft.epage=2870&amp;rft.atitle=A%20High-Density%20Linkage%20Map%20Reveals%20Sexual%20Dimorphism%20in%20Recombination%20Landscapes%20in%20Red%20Deer%20%28Cervus%20elaphus%29&amp;rft.volume=7&amp;rft.issue=8&amp;rft.date=2017-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Johnston%2C+Susan+E.&amp;rft.au=Huisman%2C+Jisca&amp;rft.au=Ellis%2C+Philip+A.&amp;rft.au=Pemberton%2C+Josephine+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485843/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485843/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485843/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485843/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485843/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485843/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485843/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485843/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485843/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485843/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485843/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485843/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/8/2859" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/8/2859.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/8/2859.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/8/2859.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/8/2859.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
