<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/8/2461" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/8/2461.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/8/2461.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/8/2461.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/8/2461.atom" />
<meta name="HW.pisa" content="ggg;7/8/2461" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Patterns of Genetic Structure and Linkage Disequilibrium in a Large Collection of Pea Germplasm" />
<meta name="DC.Identifier" content="10.1534/g3.117.043471" />
<meta name="DC.Date" content="2017-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Siol et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Pea ( Pisum sativum , L.) is a major pulse crop used both for animal and human alimentation. Owing to its association with nitrogen-fixing bacteria, it is also a valuable component for low-input cropping systems. To evaluate the genetic diversity and the scale of linkage disequilibrium (LD) decay in pea, we genotyped a collection of 917 accessions, gathering elite cultivars, landraces, and wild relatives using an array of ∼13,000 single nucleotide polymorphisms (SNP). Genetic diversity is broadly distributed across three groups corresponding to wild/landraces peas, winter types, and spring types. At a finer subdivision level, genetic groups relate to local breeding programs and type usage. LD decreases steeply as genetic distance increases. When considering subsets of the data, LD values can be higher, even if the steep decay remains. We looked for genomic regions exhibiting high level of differentiation between wild/landraces, winter, and spring pea, respectively. Two regions on linkage groups 5 and 6 containing 33 SNPs exhibit stronger differentiation between winter and spring peas than would be expected under neutrality. Interestingly, QTL for resistance to cold acclimation and frost resistance have been identified previously in the same regions." />
<meta name="DC.Contributor" content="Mathieu Siol" />
<meta name="DC.Contributor" content="Françoise Jacquin" />
<meta name="DC.Contributor" content="Marianne Chabert-Martinello" />
<meta name="DC.Contributor" content="Petr Smýkal" />
<meta name="DC.Contributor" content="Marie-Christine Le Paslier" />
<meta name="DC.Contributor" content="Grégoire Aubert" />
<meta name="DC.Contributor" content="Judith Burstin" />
<meta name="article:published_time" content="2017-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Patterns of Genetic Structure and Linkage Disequilibrium in a Large Collection of Pea Germplasm" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Pea (&lt;i&gt;Pisum sativum&lt;/i&gt;, L.) is a major pulse crop used both for animal and human alimentation. Owing to its association with nitrogen-fixing bacteria, it is also a valuable component for low-input cropping systems. To evaluate the genetic diversity and the scale of linkage disequilibrium (LD) decay in pea, we genotyped a collection of 917 accessions, gathering elite cultivars, landraces, and wild relatives using an array of ∼13,000 single nucleotide polymorphisms (SNP). Genetic diversity is broadly distributed across three groups corresponding to wild/landraces peas, winter types, and spring types. At a finer subdivision level, genetic groups relate to local breeding programs and type usage. LD decreases steeply as genetic distance increases. When considering subsets of the data, LD values can be higher, even if the steep decay remains. We looked for genomic regions exhibiting high level of differentiation between wild/landraces, winter, and spring pea, respectively. Two regions on linkage groups&amp;nbsp;5 and 6 containing 33 SNPs exhibit stronger differentiation between winter and spring peas than would be expected under neutrality. Interestingly, QTL for resistance to cold acclimation and frost resistance have been identified previously in the same regions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/08/01" />
<meta name="citation_mjid" content="ggg;7/8/2461" />
<meta name="citation_id" content="7/8/2461" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/8/2461" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/8/2461.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/8/2461.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/8/2461.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.043471" />
<meta name="citation_pmid" content="28611254" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2461" />
<meta name="citation_lastpage" content="2471" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Mathieu Siol" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France" />
<meta name="citation_author_email" content="mathieu.siol@inra.fr" />
<meta name="citation_author" content="Françoise Jacquin" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France" />
<meta name="citation_author" content="Marianne Chabert-Martinello" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France" />
<meta name="citation_author" content="Petr Smýkal" />
<meta name="citation_author_institution" content="Palacky University, Faculty of Science, Department of Botany, Holice, 783 71 Olomouc, Czech Republic" />
<meta name="citation_author" content="Marie-Christine Le Paslier" />
<meta name="citation_author_institution" content="INRA, US 1279 Etude du Polymorphisme des Génomes Végétaux (EPGV), Centre de Recherche Ile-de-France-Versailles-Grignon, Commissariat à l’énergie atomique (CEA)-Institut de Génomique, Centre national de génotypage (CNG), Université Paris-Saclay, 91000 Evry, France" />
<meta name="citation_author" content="Grégoire Aubert" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France" />
<meta name="citation_author" content="Judith Burstin" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=S. Alves-Carvalho;citation_author=G. Aubert;citation_author=S. Carrère;citation_author=C. Cruaud;citation_author=A. L. Brochot;citation_title=Full-length de novo assembly of RNA-seq data in pea (Pisum sativum L.) provides a gene expression atlas and gives insights into root nodulation in this species.;citation_pages=1-19;citation_volume=84;citation_year=2015;citation_pmid=26296678;citation_doi=10.1111/tpj.12967" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Baranger;citation_author=G. Aubert;citation_author=G. Arnau;citation_author=A. L. Lainé;citation_author=G. Deniot;citation_title=Genetic diversity within Pisum sativum using protein- and PCR-based markers.;citation_pages=1309-1321;citation_volume=108;citation_year=2004;citation_issue=7;citation_pmid=14727027;citation_doi=10.1007/s00122-003-1540-5" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Burstin;citation_author=P. Salloignon;citation_author=M. Chabert-Martinello;citation_author=J. B. Magnin-Robert;citation_author=M. Siol;citation_title=Genetic diversity and trait genomic prediction in a pea diversity panel.;citation_pages=105-121;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. R. Cavanagh;citation_author=S. Chao;citation_author=S. Wang;citation_author=B. E. Huang;citation_author=S. Stephen;citation_title=Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.;citation_volume=110;citation_year=2013;citation_pmid=23630259;citation_doi=10.1073/pnas.1217133110" />
<meta name="citation_reference" content="citation_title=Elements of Evolutionary Genetics;citation_author=B. Charlesworth;citation_author=D. Charlesworth;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. M. Clark;citation_author=E. Linton;citation_author=J. Messing;citation_author=J. F. Doebley;citation_title=Pattern of diversity in the genomic region near the domestication gene tb1.;citation_volume=101;citation_year=2004;citation_pmid=14701910;citation_doi=10.1073/pnas.2237049100" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Comadran;citation_author=B. Kilian;citation_author=J. Russell;citation_author=L. Ramsay;citation_author=N. Stein;citation_title=Natural variation in a homolog of Antirrhinum CENTRORADIALIS contributed to spring growth habit and environmental adaptation in cultivated barley.;citation_pages=1388-1392;citation_volume=44;citation_year=2012;citation_issue=12;citation_pmid=23160098;citation_doi=10.1038/ng.2447" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_title= EggLib: processing, analysis and simulation tools for population genetics and genomics.;citation_author=S. De Mita;citation_author=M. Siol;citation_volume=13;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. Desgroux;citation_author=V. L. Anthoëne;citation_author=M. Roux-Duparque;citation_author=J. Rivière;citation_author=G. Aubert;citation_title=Genome-wide association mapping of partial resistance to Aphanomyces euteiches in pea.;citation_pages=124-144;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Cytometry. Part A :  the journal of the International Society for Analytical Cytology;citation_journal_abbrev=Cytometry A;citation_author=J. Dolezel;citation_author=J. Greilhuber;citation_title=Nuclear genome size: are we getting closer?;citation_pages=635-642;citation_volume=77;citation_year=2010;citation_issue=7;citation_pmid=20583277" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Foll;citation_author=O. Gaggiotti;citation_title=A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: a Bayesian perspective.;citation_pages=977-993;citation_volume=190;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Gao;citation_author=S. Williamson;citation_author=C. D. Bustamante;citation_title=A Markov chain Monte Carlo approach for joint inference of population structure and inbreeding rates from multilocus genotype data.;citation_volume=176;citation_year=2007;citation_pmid=17483417;citation_doi=10.1534/genetics.107.072371" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=D. Gianola;citation_author=S. Qanbari;citation_author=H. Simianer;citation_title=An evaluation of a novel estimator of linkage disequilibrium.;citation_pages=275-285;citation_volume=111;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=V. Hecht;citation_author=R. E. Laurie;citation_author=J. K. Vander Schoor;citation_author=S. Ridge;citation_author=C. L. Knowles;citation_title=The Pea GIGAS gene is a FLOWERING LOCUS T homolog necessary for graft-transmissible specification of flowering but not for responsiveness to photoperiod.;citation_volume=23;citation_year=2011;citation_pmid=21282524;citation_doi=10.1105/tpc.110.081042" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=E. L. Heffner;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program.;citation_pages=65-75;citation_volume=4;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Huang;citation_author=X. Wei;citation_author=T. Sang;citation_author=Q. Zhao;citation_author=Q. Feng;citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces.;citation_pages=961-967;citation_volume=42;citation_year=2010;citation_issue=11;citation_pmid=20972439;citation_doi=10.1038/ng.695" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Jakobsson;citation_author=N. A. Rosenberg;citation_title=CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.;citation_volume=23;citation_year=2007;citation_pmid=17485429;citation_doi=10.1093/bioinformatics/btm233" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Jing;citation_author=R. Johnson;citation_author=A. Seres;citation_author=G. Kiss;citation_author=M. J. Ambrose;citation_title=Gene-Based Sequence Diversity Analysis of Field Pea (Pisum);citation_pages=2263-2275;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073431;citation_doi=10.1534/genetics.107.081323" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=R. Jing;citation_author=A. Vershinin;citation_author=J. Grzebyta;citation_author=P. Shaw;citation_author=P. Smýkal;citation_title=The genetic diversity and evolution of field pea (Pisum) studied by high throughput retrotransposon based insertion polymorphism (RBIP) marker analysis.;citation_pages=44-44;citation_volume=10;citation_year=2010;citation_pmid=20156342;citation_doi=10.1186/1471-2148-10-44" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=T. Jombart;citation_author=S. Devillard;citation_author=F. Balloux;citation_title=Discriminant analysis of principal components: a new method for the analysis of genetically structured populations.;citation_pages=94-94;citation_volume=11;citation_year=2010;citation_pmid=20950446;citation_doi=10.1186/1471-2156-11-94" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=D. Jost;citation_title=G(ST) and its relatives do not measure differentiation.;citation_pages=4015-4026;citation_volume=17;citation_year=2008;citation_issue=18;citation_pmid=19238703;citation_doi=10.1111/j.1365-294X.2008.03887.x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Klein;citation_author=H. Houtin;citation_author=C. Rond;citation_author=P. Marget;citation_author=F. Jacquin;citation_title=QTL analysis of frost damage in pea suggests different mechanisms involved in frost tolerance.;citation_pages=1319-1330;citation_volume=127;citation_year=2014" />
<meta name="citation_reference" content="citation_title=The Search for Wild Relatives of Cool Season Legumes;citation_author=G. Ladizinsky;citation_author=S. Abbo;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_author=J. L. Jannink;citation_title=Potential and optimization of genomic selection for Fusarium head blight resistance in six-row barley.;citation_pages=1609-1621;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.09.0503" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=B. Mangin;citation_author=A. Siberchicot;citation_author=S. Nicolas;citation_author=A. Doligez;citation_author=P. This;citation_title=Novel measures of linkage disequilibrium that correct the bias due to population structure and relatedness.;citation_pages=285-291;citation_volume=108;citation_year=2012;citation_issue=3;citation_pmid=21878986;citation_doi=10.1038/hdy.2011.73" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. A. Mather;citation_author=A. L. Caicedo;citation_author=N. R. Polato;citation_author=K. M. Olsen;citation_author=S. McCouch;citation_title=The extent of linkage disequilibrium in rice (Oryza sativa L.).;citation_volume=177;citation_year=2007;citation_pmid=17947413;citation_doi=10.1534/genetics.107.079616" />
<meta name="citation_reference" content="citation_title=Plant Genetic Resources of Legumes in the Mediterranean;citation_author=N. Maxted;citation_author=N. Ambrose;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=M. N. Nelson;citation_author=M. Ksiazkiewicz;citation_author=S. Rychel;citation_author=N. Besharat;citation_author=C. M. Taylor;citation_title=The loss of vernalization requirement in narrow-leafed lupin is associated with a deletion in the promoter and de-repressed expression of a Flowering Locus T (FT) homologue.;citation_pages=220-232;citation_volume=213;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=R. K. Pasam;citation_author=R. Sharma;citation_author=M. Malosetti;citation_author=F. A. van Eeuwijk;citation_author=G. Haseneyer;citation_title=Genome-wide association studies for agronomical traits in a world wide spring barley collection.;citation_pages=16-16;citation_volume=12;citation_year=2012;citation_pmid=22284310;citation_doi=10.1186/1471-2229-12-16" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. C. Pennycooke;citation_author=H. Cheng;citation_author=E. J. Stockinger;citation_title=Comparative genomic sequence and expression analyses of Medicago truncatula and Alfalfa subspecies falcata COLD-ACCLIMATION-SPECIFIC genes.;citation_volume=146;citation_year=2008;citation_pmid=18218976;citation_doi=10.1104/pp.107.108779" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. J. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Raj;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=fastSTRUCTURE: variational inference of population structure in large SNP data sets.;citation_volume=197;citation_year=2014;citation_pmid=24700103;citation_doi=10.1534/genetics.114.164350" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Rincent;citation_author=D. Laloë;citation_author=S. Nicolas;citation_author=T. Altmann;citation_author=D. Brunel;citation_title=Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (Zea mays L.).;citation_volume=192;citation_year=2012;citation_pmid=22865733;citation_doi=10.1534/genetics.112.141473" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Ross-Ibarra;citation_author=P. L. Morrell;citation_author=B. S. Gaut;citation_title=Plant domestication, a unique opportunity to identify the genetic basis of adaptation.;citation_volume=104;citation_year=2007;citation_pmid=17494757;citation_doi=10.1073/pnas.0700643104" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Roullier;citation_author=L. Benoit;citation_author=D. B. McKey;citation_author=V. Lebot;citation_title=Historical collections reveal patterns of diffusion of sweet potato in Oceania obscured by modern plant movements and recombination.;citation_volume=110;citation_year=2013;citation_pmid=23341603;citation_doi=10.1073/pnas.1211049110" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Schmutz;citation_author=P. E. McClean;citation_author=S. Mamidi;citation_author=G. A. Wu;citation_author=S. B. Cannon;citation_title=A reference genome for common bean and genome-wide analysis of dual domestications.;citation_pages=707-713;citation_volume=46;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. Siol;citation_author=S. I. Wright;citation_author=S. C. H. Barrett;citation_title=The population genomics of plant adaptation.;citation_pages=313-332;citation_volume=188;citation_year=2010;citation_issue=2;citation_pmid=20696011;citation_doi=10.1111/j.1469-8137.2010.03401.x" />
<meta name="citation_reference" content="citation_journal_title=Agronomy;citation_author=P. Smýkal;citation_author=G. Aubert;citation_author=J. Burstin;citation_author=C. J. Coyne;citation_author=N. T. H. Ellis;citation_title=Pea (Pisum sativum L.) in the genomic era.;citation_pages=74-115;citation_volume=2;citation_year=2012;citation_doi=10.3390/agronomy2020074" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=N. Tayeh;citation_author=N. Barhman;citation_author=R. Devaux;citation_author=A. Bluteau;citation_author=J. M. Prosperi;citation_title=A high-density genetic map of the Medicago truncatula major freezing tolerance QTL on chromosome 6 reveals colinearity with a QTL related to freezing damage on Pisum sativum linkage group VI.;citation_pages=279-289;citation_volume=32;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=N. Tayeh;citation_author=C. Aluome;citation_author=M. Falque;citation_author=F. Jacquin;citation_author=A. Klein;citation_title=Development of two major resources for pea genomics: the GenoPea 13.2K SNP Array and a high density, high-resolution consensus genetic map.;citation_pages=1257-1273;citation_volume=84;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=N. Tayeh;citation_author=A. Klein;citation_author=M.C. Le Paslier;citation_author=F. Jacquin;citation_author=H. Houtin;citation_title=Genomic prediction in pea: effect of marker density and training population size and composition on accuracy of prediction.;citation_pages=941-951;citation_volume=6;citation_year=2015;citation_pmid=26635819" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. V. Vershinin;citation_author=T. R. Allnutt;citation_author=M. R. Knox;citation_author=M. J. Ambrose;citation_author=N.T.H. Ellis;citation_title=Transposable elements reveal the impact of introgression, rather than transposition, in Pisum diversity, evolution, and domestication.;citation_volume=20;citation_year=2003;citation_pmid=12949152;citation_doi=10.1093/molbev/msg220" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Vigouroux;citation_author=M. McMullen;citation_author=C. T. Hittinger;citation_author=K. Houchins;citation_author=L. Schulz;citation_title=Identifying genes of agronomic importance in maize by screening microsatellites for evidence of selection during domestication.;citation_volume=99;citation_year=2002;citation_pmid=12105270;citation_doi=10.1073/pnas.112324299" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot.;citation_author=N. F. Weeden;citation_title=Genetic changes accompanying the domestication of Pisum sativum: is there a common genetic basis to the ‘domestication syndrome’ for legumes?;citation_volume=100;citation_year=2007;citation_pmid=17660515;citation_doi=10.1093/aob/mcm122" />
<meta name="citation_reference" content="citation_title=Domestication of Plants in the Old World;citation_author=D. Zohary;citation_author=M. Hopf;citation_year=2000" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Patterns of Genetic Structure and Linkage Disequilibrium in a Large Collection of Pea Germplasm" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="twitter:description" content="Pea ( Pisum sativum , L.) is a major pulse crop used both for animal and human alimentation. Owing to its association with nitrogen-fixing bacteria, it is also a valuable component for low-input cropping systems. To evaluate the genetic diversity and the scale of linkage disequilibrium (LD) decay in pea, we genotyped a collection of 917 accessions, gathering elite cultivars, landraces, and wild relatives using an array of ∼13,000 single nucleotide polymorphisms (SNP). Genetic diversity is broadly distributed across three groups corresponding to wild/landraces peas, winter types, and spring types. At a finer subdivision level, genetic groups relate to local breeding programs and type usage. LD decreases steeply as genetic distance increases. When considering subsets of the data, LD values can be higher, even if the steep decay remains. We looked for genomic regions exhibiting high level of differentiation between wild/landraces, winter, and spring pea, respectively. Two regions on linkage groups 5 and 6 containing 33 SNPs exhibit stronger differentiation between winter and spring peas than would be expected under neutrality. Interestingly, QTL for resistance to cold acclimation and frost resistance have been identified previously in the same regions." />
<meta name="og-title" property="og:title" content="Patterns of Genetic Structure and Linkage Disequilibrium in a Large Collection of Pea Germplasm" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/8/2461" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Pea ( Pisum sativum , L.) is a major pulse crop used both for animal and human alimentation. Owing to its association with nitrogen-fixing bacteria, it is also a valuable component for low-input cropping systems. To evaluate the genetic diversity and the scale of linkage disequilibrium (LD) decay in pea, we genotyped a collection of 917 accessions, gathering elite cultivars, landraces, and wild relatives using an array of ∼13,000 single nucleotide polymorphisms (SNP). Genetic diversity is broadly distributed across three groups corresponding to wild/landraces peas, winter types, and spring types. At a finer subdivision level, genetic groups relate to local breeding programs and type usage. LD decreases steeply as genetic distance increases. When considering subsets of the data, LD values can be higher, even if the steep decay remains. We looked for genomic regions exhibiting high level of differentiation between wild/landraces, winter, and spring pea, respectively. Two regions on linkage groups 5 and 6 containing 33 SNPs exhibit stronger differentiation between winter and spring peas than would be expected under neutrality. Interestingly, QTL for resistance to cold acclimation and frost resistance have been identified previously in the same regions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
    <title>Patterns of Genetic Structure and Linkage Disequilibrium in a Large Collection of Pea Germplasm | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8nTLBrs-Wvmh8QpISOw2BceEw1LACBdh2bjOhLk2m2Y","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485857","apath":"\/ggg\/7\/8\/2461.atom","pisa":"ggg;7\/8\/2461","ac":{"\/ggg\/7\/8\/2461.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/8\/2461.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2461"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2461"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/8\/2461":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RHEyjmViR7FqLfTaXhzImnUcnhm8I62oeSQMiNoVJW9H7Rc1lgpKGbjvCzLnsmyQHxELG1ZLu54rsNjrJxrKTu\\\\\\\/+KxpYHg8VItXBMfgapfRe53nJ9mK35zKHa2pTElu0aMyfIZC6dK1dzd7kjQBR4IRzu+ceix9dWexPsO8Cl0Axmp+gZsqkx9cml8Rqp8ISWMF5UOwPBJMQPcTZjR0wgQc24tEviLZJrtDdNtz\\\\\\\/JjDUDj1r4ZBHmUOL+\\\\\\\/iY1Ypsbl90PfJJTm7hwOx2EXFKXjPf29fU8tg9H8q2n8udcuK\\\\\\\/iBPUFwXlPLDbxR4rOzniRRfGvstbdGlLveh942BJIBDgYD5GWJSucgE6HwDAKtKIQjHC7QnOMWtJtkne1xqpQIbJ0vMV+k5CAzP2JEGMvRLZJME4MymPALe5UIcaRMbET3T97\\\\\\\/UB0LzWEBFcj2BAtMusSfTIWjQDfVdj8UCL0+DnWT\\\\\\\/CLiRyhnyaXYffDlNJXHBO5QHMgDMIVBejd9VldcqUScv0GlQ3FlO1IQVVV596ec19bLUBJcP0WvCO2uj7AqpbTQ8gnJQcRDP+idHvdFP3c4n6cFXoCQG+U7MTRKWdEeHosZ+qtT+G7xwrSFPkYh1qXNxcEWIFPvMRmiz5TEEvKU4VOMj30iOw3SNK8ctbGbovAuIld6EpAApeO7UqE8143vbPlJxvv0fYxz2xI+SN0v+6JLsIEkShdEhJgQFoTuzPUajU\\\\\\\/uJZuwfhByNlYHdqUkYff0eEphm+oNYCay\\\\\\\/ghDrS+UvfYm\\\\\\\/R4SNvHpsGduiPyZgVJ3oTyLMFgJK6d4QTaGvOBaAqbKeMTtZfd4fCkEjXGQcAKsVT5kHbv8QSLafH5mq8Xj\\\\\\\/\\\\\\\/uIbRIdWvIF1fW9xhDmLZ1sKCtztWkKRunzvpN4wwR+OTR0tKW2KCA4s+NzjJngNHEd\\\\\\\/IjqLeZIsl9cSMVOwAZuC9oIYK12msmKf\\\\\\\/lZTobXuj4xb2Ya59S6SR3EKAPhPpIDRkAvzKu\\\\\\\/cbxaAoMrBGq8H7Y3dtZX9mTzoSi39DDnUyBY\\\\\\\/4pSEOmjuSPoxcYAU9iwlygk6dVBXTuohQbZ29qGTGTvDWCXv\\\\\\\/dao9Vgu26K5iC0rcaC5Wyt2+Xhh2PBeDYCYIcjjxYPyWgROAJ+aC8q3Fco7vGhm9pYC2ffBlKujm9S19b4UUs6qAEs\\\\\\\/DX0xdv8EvcWJ\\\\\\\/fcYMfgu5sllooVj+wZfD1Er6pJ28RT5K9Wp6oaFqO3d2pRy\\\\\\\/J5ReXSHkkqVMs2XpBCRKNpa+zzT44muIqUI4\\\u0022,\\\u0022iv\\\u0022:\\\u0022xOOvuPAPn2hPO+vpLncwYA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ca9949157b31d6cbb04478efc9ebe951098bc43574920b3b4616858dc9169920\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225yTVw08BIEDEwEURs8HRiE7pjUdFXLVKQxdLPyRBu9c8jBDFlDD3\\\\\\\/ujqcTAr6lO75Yn10MHyC5mKDZ0tDeDjj0Lrqg0xZXD2tlFKr7hWmS+vWSz4TDc5lh25wiSTUzEcB4zzvZ1kOwMBTW7GGRlIwy58zz7X\\\\\\\/fThyX97AaGUT9ek+QmYGpiofYPOIhUUc2nx\\\\\\\/qy\\\\\\\/j\\\\\\\/x4hDXo7L5eH0Oxw7WkssXV9lp5bDAM22P8z46Orl9qvCVwJFACqZXvGhvhpEXWOMaFiFZb9ivKXXhV0vZvzu0211DwzKnLt6u+XC\\\\\\\/HD+O2GVOjlV3iPYp+5o+HC0ccFxf548DzYOKS+ZN07Kjwngnf8Xdedcq0B42inCvn5Actkb28oh406o94WWNTNXzzyyEbmz0PbWqWt8N9b+E64pSA8o4b6q4sJPOcX2osEfY8rfu02ukK638jBKIsD5QrUCXHwiV6ibbB38tBTudwfvJ1ADW6EIfvDzVlMSPzwPCU0HLaccaTw+Zhy4ydRe3mKQVd8evdz8odUCu5gn4oFpa8WDWH54cYU05reRUSpfUHNLwP2Ya+UDD+PcHYWUCsY9nSKxzvwlvsVpWadsD5KVtyVGLTRFkgtV+3cDukdAj2TjgPDQ17VkXpf\\\\\\\/Cao+OSuuB3ka+cKV\\\\\\\/s246JGa2cjAtBXo6o8di8IMX\\\\\\\/BZ5Gsl1SF4AJvYvxvYKoplXUZYmcAeHPdRYbEKBJHwj+pAzOpKJHxhAfvsBWb19wdmbNDETrhw3zbYBcTqVDZEdkKeKNMCghcXLF5xahTxgzNxDUiOmYKxHKtVP+GKOtE0n90lYY8E8LaOuvXYjI7F8bjErDkspSGMmxp16wac8gd9gTzGQsiP88Y1+lcA18QGy0vY5wHjm\\\\\\\/f2dsjo0b4MqKRNaM70DZvK8GYG5F1lELW+PYZF2uZgbtoWi7tO3kI1FaLbBAN\\\\\\\/P1ozhOFEXsIk63FJCvSgqxjMUPekVQ7qPaPPStxRoInm1lwXigKJGu3C2f1meIPieVpmKAEhbk6bWnf\\\\\\\/DPkDobob+RZ2gdYqysL1OSzpqugJWv2uHUlKNwcWRvj3725nuolg2szPub6RcO4n8bkYGyc64UIXVyvivBeN9WQld4roW7PNtdR2ewNYmJhJDWDGSgL2MGdTCmSkWA+Ai11VetGoYo65wd8n1h4KbD6xnviGLtbtO\\\\\\\/+YqmvooVDXRoM9HLEFLMRe13Lpfj+w2AsW9SRXOBDJ6\\\\\\\/ZhpxNxUxO8fCqmMDAt4KJDSk+ivo1FCBRsSQWqi+\\\\\\\/9rky9hjnLlx9+fMmyjZOFfIanFeIZUNyL9ANu75t\\\\\\\/yI4MazylB2CVpMDdnyLr7w4N2vPNtU3TC\\\\\\\/imLNRSnzG5ki9dJ8AQv+tASrorw5ppBrzrCaKgeKhSqxWc\\\\\\\/aU8y5oIRA4\\\\\\\/zXl9Q6OXd+U8qX\\\\\\\/O9XejHGWzwcTRoHNuOHv0eFAKaenIZQ+6Vbnd449xSUOKeBYjsUCnVmU8eEHJSy2oYWcmlhASinXookQr9UGzaPZqRfAz6rzn6dqIVHR1NnfIP+AB41ofNyzbSprG8GGLujYrPJBTMBkr4eGKlQWht+sP4o3519m\\\\\\\/kwtMhR5uziyNUBy\\\\\\\/9Xv7NM7AUIZp+kBDrdV0QdAuk\\\\\\\/ZtZjPVSLE\\\\\\\/hLRSbRSMw3OpUP6X1qirPHHzVDg0ak8cJH0xS38t2XOOxFEYmBQXwp\\\\\\\/CihGmHWt+FCMCcBJrt3t9LcdlB9AI\\\\\\\/uQOvG4wn7GdahcgD5+\\\\\\\/AoKFNxtwhyUV\\\\\\\/Y7t0CEUvx+e\\\\\\\/TO3ovPfY1GoQo5naDRvJ2aYG59dPRM4hG2xdMfsbJV21zeEp5Oip2zuUP\\\\\\\/PfWikoy8s+X1vZkjS+QDj5QcZVv7YdC9TjqddhckyqM9T3h6xaTyoBcrhGRQ\\\\\\\/cicP9jibPzFdpks9fOqSIevWv0x2GngEPiLwlZPX5RhBX6T4sA2M5Jxd\\\\\\\/g3EZVn3DDI+ARWBFtXZGGCGWHWCcEBGOlIvXL0QcH1q6K5ZPr6F1lwsF\\\\\\\/guLCA8kgsbzpuSUpr4DWIvTx1\\\\\\\/NVcBLybQXmNS196+GhSn\\\\\\\/nUbwz0hX+Jh+ouhY8Hwp6OViZ0U\\\\\\\/Pmlqf9xVVnwebMsMbiIvJAw5xQVXsToNvTm3erdQNPDv+snMF6EPtA63w5WS4LQAIuAGH4gHDiPe\\\\\\\/5xcjIV1yhk1tdta23coGTOIX+FigZnCvgFctLbWb92yuZ9TxV2F4rYam34l4rQqU3Bqw0EKvr5epiXbFNdcgqjQsA6eRRQNoJau1UQtFMK2G7a\\\\\\\/h2jpw19RFD0YrjRRRORtIqlHbDzk9NMionie3DJPh0DOtnbHk\\\\\\\/dWUIdz5vr3ttT\\\\\\\/ElE1ndruXz+y\\\\\\\/09b46n9L1AqVWKLTySdscdPDRTEgpYVoC6\\\\\\\/YwD71U2pN3xjEJZU7ocn3U\\\\\\\/brXu0LxAS775sYfatwS2lBIwmYUi0OXT8T7\\\\\\\/OWoM4pCMFrSEGl4u4lPvdRihzOfxKRrgOwsErBvWu9bpUKN8EL4dPymHw2FkoNfHsIJQoBvXoqGLuPkpz\\\\\\\/xSLBmcX6kd4KniDUzsNI+pThfhC3ByCXCTjgCByxVt6pncvusCMN\\\\\\\/gXf9ZrpTN\\\\\\\/lY2AYMObrqjjyeuUSyf+rxu382r8HOHB\\\\\\\/qpeNyvyNTbYwcg1kvkbwAQjVLZJnJR11TzVrxXZU7PMsUc22YlJ0Lo0HpzPxQdXEaLiuxZjV3\\\\\\\/fdSACNjTaQ0z6j7oGPsyVdfjLAy7Maq2CAczWkkZ380uVhSHzTgpxaFBvFTL\\\\\\\/hlGwd+VAbS6tA5W2VF9sQJw16bcpFxDraN7kddW\\\\\\\/\\\\\\\/u40nQSPKhqtngg9vOx1bj7DIYgnmpVFopIKDdu0oH1DZ8R1\\\\\\\/9K39Q12r5ADEzLBRxZrktWgRChoJrRnxgHMunOQhetk6zdOIs9wx1SKscxL7uQyEw6YQpox1hpj9XGRxGrfFMLE5b0oYeZaQeXpizpnTDkc\\\\\\\/+1pwVOtdstcmQtt9qPqwCxgduRY7APodrB8xQJbSuj1N1Mbea0wlPE7XgqPlVBZgf2XHsrN87857y2kSUZCureSlPXJOk\\\\\\\/eN4jQKcPwqdjSfCGnjw5G5JhAsYOOHHsftY9T5jpdnWQhuimD9JYvNzwhKEu11g5vfVmwPmrECgwnXFkt87JO4u5MtzS1UWzJTPaaEM3nbK8ds0kDu3MYMNvpEVoXFTD6XDbWG2jKn4bX1fauaVk2F2omCAQYMTECTOxjxyfR1BGqAITn3wNx3VSdd8scp8\\\\\\\/8qztlhS\\\\\\\/p2DImNd3zCzAWnM60JW\\\\\\\/wf2wl0JkXkmNtzepQuna22NUSV4oBP9CXBRcw6LCVUs7pR1pi1O1enF37XocKIIHEwa2qREjBM0Wbg4JYN2hunknD5kxSowop6aEzlbO9VxMZd1NuAZW1MMmDeSAXMgKATRi\\\\\\\/YbZY8vzv6kFliEMMPqGMRE0RznjmnxL7rDsjeUcc9AMtWT2DXE3kmV\\\\\\\/A1bIlLHYQL3TDZ0ydysZo4cSCwIbHB9Oq\\\\\\\/st5BgsW4e0PghcKsQjMzvLt2opMYl9XJlwynAeXxopX355dFNRlOQV6rf745oLDXJpT+fLqm1PIZILiHFu1xPj0p4Fxfo+0jeFtjFndK7kHzQTAZGNOpFEIagX5\\\\\\\/4KHiX+dlJqMsSiDMPopI8qyf4teN6k6t\\\\\\\/VFWgTMX9knagzzJYRauVLd2j6fxS6\\\\\\\/k1HLdtMs4Y\\\\\\\/PNnUug2IGRORKIielMkVq1IdLAVfId1YZC5ociqOyKs7mbQ4eLTPfK+t5Xuvo7COU81zZJFAOrtrIk6g6NhZf5ZAHJuGWzG4aZuHQY73nzfUDJBHIQHPNMT6kF4LpK+PBXZeiXmFpyqf6pOxQr3v2pb6Qj37Y\\\\\\\/KjU7TqfeaC5bbSNE7lmN+9p\\\\\\\/9YbbzS5E96KmtsKFPYl0IJV76lb+BdFos5oqnFPyyf8Z8UYdDL648je6CTA5aQBm4stV97YH2v+TLL\\\\\\\/VpAHiR\\\\\\\/CDqlFdOG4WEJHvnDjve88WyMuwqO7qfNJup02Cf78OS\\\\\\\/fCtzvdsiE5WEmS9NxfqQ2RZFYRFyePAO9xSjS9Ryy6NZljSoQTzihji9R3tCxs3SymaS3S6SbzUVy+HzZ3zRWI1c5aobSpe6jSQJQ\\\\\\\/T3k7+P3DFX6vlzf6zd6T3JQtYEHHwBi4souXFvzGLg7sm9gufh8cBMt2o1cu6\\\\\\\/GbZ22KhBANhNFXRB+eENXhNPDvBP+hmlKA4t6EubIl6cXxXO9+n9bEZ0CMWEZdd1z4kzsWPM2\\\\\\\/I9XAxGSyukJznnPqC4Oo\\\\\\\/Fgq2O57cgqxCV5uIkSh2w4553y1BN+MYBadKy0x36y0Qdvi5e2qTidffkqD8l1bKgkT3StoMqVaBsTnFYmPXZ+D9N+EsH5\\\\\\\/MStEp0heaXXR11VG3olRhZgLr9twhwV6btQfb\\\\\\\/tTlfpX6Erje8Gi8uc\\\\\\\/uT2bKXZ78lq2khHk79AbU8JzUzmD5ZNxU2\\\\\\\/CYBWgu0E9SLod7UBuiIVodAOqNtMlVbj5+nOiuOPfQtBSkGDzt5DvJKysM5A+ZdBmuop3peHW7gMKJyegGxHjXEu21HQ5d7EGMU0EIQ3as8WpQU88alr1pcCiNODQswjLIQyskSUF6\\\\\\\/J8sq\\\\\\\/g5HNYbSIiyNWby3YMtNLf2bE5QQdMmaVc7oz\\\\\\\/0PH9HUd8rbn08gwTN70DjEtucVlVRu8FuRqEDibnbXj8CnJa69PGwKQS2k6eXWrZq0ICzpTeSDYOIgLlYvdknFMEy6e4LrN11g3mDf0jokjiyLg3cDv0cDvJRwIQuL1ADkVOKt39QmgB4pXrFFWzUwtNSOp0CEIDUaapv+5otfe11MxG3hZ7Afulm8kL072+hlJCqHMcqKPE0fnMPdZo5VU9lERQGCnuLm\\\\\\\/9CmutS3BAwPJrEfx+dKYHu3ksSneREDcbFXTxodNVvj1xYHq3eWn67i\\\\\\\/KSEzw41239HU3FlXjD3JXk+VC1vGXAL6v4+N3q8ONP5\\\\\\\/qWRrcrvdvepupFG+pjx9fCCL0vt4\\\\\\\/D5UsYrdG\\\\\\\/CwuxaQI7GZzfJgmKZXUL+Byv7qgW3JOwAaTp7qGHu33IJeB19irvEtr9fZbxJPLrsi0EfyGV6rOfmJGwjIQPROCbtGah+G\\\\\\\/fRmW3GOT0r6ldD\\\\\\\/B8lfoAkgAlYx174WCH\\\\\\\/NwY6AmGrZ3uOQ1knK+xTfc3h7rVgRA28WO+DKcxGXLCUmfFSd0it+r4ScM\\\\\\\/YM+x+0KzSJJ8hDtA4aZEcns7pwE50JkPIcOS6kJYx\\\\\\\/8jT5mSNwTt6OQa1EISuHSywPAUu0bJsIOnfWTx5aM1iHrtdha4Gr1D\\\\\\\/K9bPiGKfZZO1si0l779Lf4FUGsHqq4dK7MDRyYvzs0B\\\\\\\/Ph+HQdPevU4bBbb+CxqMn\\\\\\\/5lj\\\\\\\/+6Da7ZIqizr8onRYUgjtPuBr6y\\\\\\\/KCX4tNUcW8Fj9DL87LJaUYOghHYVrRSraN+9U+wawRBj8Tdrgdji6NtSegKTxirbPN9EC7ekZGgP1qtTWvvVrGgmOBdh7gbLfNrtdqZ\\\\\\\/ETBAeW5xSJiWlNWBErefMAPpeC1rTQqdL7iieSse6+IYaGlcatxdugzNWia8Ufl6ZHNm2ip+fC9MfTk7wesOsKd9jZi1etmSe4dkGKtmTrN077D2W47307bd8GpM0ZzWhJ5HMFhtgtIb+IqBdsr+t+t5KKSGnGpvjXnzytVmfrv8B8t+YxHd9SgInikG+dMxAj+8xFVd0WiW6977B9dqt8LV2nmRYtXzM51RnYrDWwwc\\\\\\\/\\\\\\\/+jpzrMfL\\\\\\\/LbbxGANShbwi7Hdegarz9ouKsCehl0jhF\\\\\\\/tpt+zn2a9ShAURM+tjPVBIKL26MLj3NNpzIUcKyHf+6Vgw34upZ3KKvn\\\\\\\/aDzpTKbhQa9K62m8zwvG4sY4aeAmTNGLpd9CaJLjMqiBOwzpGGZcftmDLBWX3N1XcVXCkj6TOX9JwMj2zglvuLmBtoEKCges6u7O1I6UnsVDEEwrKIuYnI2f+mQPRI1qQ\\\\\\\/PaWPsNuYVOMkSDVjYfPZnXi3b\\\\\\\/6ouC3Ha9g8QDzqSLrr4NAKN0avmY6vOGnCxEJVH1WrL\\\\\\\/hXRCczFu1XUzQfoChk4Y7L3Cx359QCffkpb4K3stpw6JdYSXZrz1F6HS7cbKzSt3umCGcc9wHgiT\\\\\\\/UJbe8obU6DwLNL6nSakgPhSw+wZ68wWg+Z+Msmyy68kZ\\\\\\\/6SxKs8OX9EF+YXyPXRz2iekbfw+eWPwIg1fuCC5i4Izg8hibF5TrQeSkJ9ZXeFZy\\\\\\\/u8XW4vhVgaRinTfpjpBalwAVHF4De4qpzIwLGCVMno2afKW9LFgyTjbEvdG8PisXTgDHscezZ3HjZgxV1Q6l1bL3BhbguZ4ZEHctr4VChH\\\\\\\/GrW\\\\\\\/hjEntHA7SsUWRbb3LcbmbWjHXXX\\\\\\\/rW5IB6xUwyHUHtUTmt0UGt+UvONZol990Gabzld140K0NTDAEk4iugbpQEM+0ljx1owgVv7G38jn6j++RqbQOwL8mPZ+pHgfBfbXVkQoVE0sAacV6PEZW4R37rt6nVTQGwWM7lpG7KWZT9BRLsYuvxhB5QfJKLCaQD3ayr7jR1sLhDEbhPfuuW3SZLey8u+YYhmZqM6UY3+LZKg4DVRdSFJKpC55rai3HwoEMriBXJoIWBzBndaBa9zJYtYbC1Ss+VjAlyrKDHPp3lKe2Z8FhHxwo2\\\\\\\/JoOkCfM6ctjkx\\\\\\\/wVh9\\\\\\\/wWuUlUE86CWRYfHUdNt1aw0\\\\\\\/OVxmc8ALCq2WJRgqRJdvLNfiEr2tKKlIKGIWeUAi4ZlaNN5Bv68Q5CfV5ztIROWV9\\\\\\\/UQlMKeOv6oJwikR8rZo9ImqSrsJKX4aJgy+Pk3ik1\\\\\\\/fv5ImILMvTDWye3JsiluTLkyvjAPE+aUI28PaksZtZJ+D90jUoketBA\\\\\\\/vNEonFbWXr3yG1s5WlgmL7U6fDdMqN1Syi+9kqa6Nt89HHSxtcMCn3hngqbwu5exvTyF5C8imPE9cHtt6FqwwqMe+VJ4C6TFUlGRqgB5dkfvJGDXxX9AO7QHmWx9KVsN6JVilwAyWzlpO31KKR\\\\\\\/7WlSrDvz3F1xxC3s3EDXH4+oRwlka6RzETpD6TTxTvyAwopIr\\\\\\\/6xLZIXGh6esrN57elDrKTCPjzBPXlRHN8MGJDcifqeKtRv2zVzkeDf7zIeTP7fxNhEddNcwHWtK0rbn648BZojtKgEqOFx0KAb\\\\\\\/ji78p15AGIPo2Hk3zbQaJvsxnDa0pf7rx\\\\\\\/\\\\\\\/1hqSsb8oLBeOkp2JH9NvCDoEWQqjbDthCo\\\\\\\/ua1UxhZ6NwsZ0iB3mq41xyInPK6ri54hdBq8GabIAzT7ApIJlfSAi78XVX9vTR6XclHenpt+Bp1wcRa9Attg00h\\\\\\\/LbmdbKRz+vUfOCqPL6urAhJ164NgH9juukrYvJ8gX7eoytG55U0ZWZj1SjmVKy20GWxbCM7\\\\\\\/DOi3ZTxpjQ5X8gBAamGCrNdJB06VJtgC0VsuqaDUEgyNxYdm4LHUWQtL9qOvyCFxCKhPByQSgHOJWhYN4k1OGgRNrd0jMB18OBA6U+8Df2HqD7Q1nbg5wyaJOKthhtpF0XpiSuK6ZfZt1MD1JbAHvOyqJSpEBjpRF6uoYvisSMSut1jKdIYZz3mZxdPZ4RIZIkYKUDTZz34gD63dTY\\\\\\\/fdTrTTuaKTk0aMRPok4rapveTg8JTV5Wt5X+Q3XLBGKuuW3ODo37JwSckrkrw8VViIBeHAQgHJBViFP44pavv7FAqK3XGBXC0MMUWjrBqH3DZ0nOE8i+qJrCK9H7VgvU7WDnAIXnt92U63KVNXhCa0wae0D4sPgkUeZWL0bk1LsXKomHmkRkVrXXMYSe+vsdKQ0HVJ2skYNyXroL1AOxnOPJynTlEi77YvaCScvp5yhsyXmYizIaUwj2hVkN2hjKsqeLE8IqIWSWDHu4Sx\\\\\\\/XcUT9C9pm8W6RAoBIjUgPOIuKpY4gvmNqoJYgXzSHwJqQIz5JD276QxYlLH+Jq7lzVOrXTIKxgfZAuZQC+RixFTP\\\\\\\/NAXB9Aj6G6MZ1ySAZKHkqfM+j6B6xgN1Y1jI28w+rOK5YZEKb1oYvWQbPMfLJcVlU1SsvdtDWq4LVr3L5V2cS1USveHmrkVr1FlIlffpHMFOGLVWWUAcsuKzrDiG\\\\\\\/IY+IufvJJVUTMXOryuF0xV4Tz8Kw06XGNcMfcIR9+KMJEMkYFIvrh7Sn1tMr7vhLQbsPyw\\\\\\\/X0R\\\\\\\/JeIAcvHjgMHcyUM\\\\\\\/2ZGjLlirlYtjKLv\\\\\\\/gd6QiMdo3VRQngyaXVXxnuvoBmEu0KXKuh7rp7jkOwFy\\\\\\\/5HWvr57iEj115BBNLlvr2vjIdak3SFBrwsh4NkgoQoWR9SpGJc1Inucy1eNt4vZtG2qRywX8BYfcakeGUkgAPLMFEinyEUeHYdDGcO640oh1P9jaGSa6Le39N4F8MPUwI4HHTI5vbJR6vtQG6KlbO3RjbG1CRW06iTI6qwhD4lTsLGGp+NEGfxKGo9YVDlBr14\\\\\\\/Moll4htDE\\\\\\\/uwZSLEvlaRdeMD4JIYPZFnlL504kk+05xrkBDvnHA+hJZUo3cSmBJIr4K9dT3t+w8xubfhCk+gE4+NhnN6zbhHMU0E4H74BXhsBIGnoYqCrOfb3OJUcKmY26Fi0KE20JC+aGiuHxFSSMuORBeWi0bDcyilENuonilLbj4dQCjBr5oQkUklM9sQCxjjUYpGQ4wuQ+bICY2fPZ900WRIWzlnZCBdKQjLZrvKKS8znUKVek1bjzJvxg4IdryQm0ZZOP7\\\\\\\/tAfsCk34bGsJsM8WpnGfcjx9g2q9qcry9kVjs+gJtcRBT5IIvpPyPPSDglzGboKLsPOPyfiOIQBDiPgMoivn8OlOphkyWxovMXcbu+q7En8Q8hrQWN\\\\\\\/0StIs0c7vghaKRQvOy3cugOHXGe1gdxytkzgEtbcxhOaI\\\\\\\/A0N8AzY4JNa8IDUSmkdNMxBzWGxuheovFE\\\\\\\/CqsxT5iwxzHMmf3zdUpikB9MJaNgpi\\\\\\\/ml1QBV3Oq7jUklbkqAkg8Uc00ZtcPN27BD8pJ3ZRpNspHeBCbyWl\\\\\\\/DFTxtZwPUOoZMFm53Ulytzkjaw\\\\\\\/J6s2hawBKDk\\\\\\\/JgeGSMz5LQBv6+UzlsnU2pNI8K+ym7W4cboh627S8D6SExP7vbawyymaDxxOPMi0qxZFpxrxOF+u4wrOHx\\\\\\\/5v\\\\\\\/SXaIsWmEgGfwZTY2ReTqYfzheeWRGwOvDZY9nFhdQiOFdOFllEShjjGiYM3hgBO1q5mGed\\\\\\\/l5RJXQOLwD4Ft7FVZSSLdFm8jtEB5K8h6U2ziyPlRSDE6WA3Di6yzLlC\\\\\\\/H\\\\\\\/\\\\\\\/OOMnHqXwQqX\\\\\\\/Tq2fCPrEbxKpSQZvBmksqKgJp8rPR7B5YpMgTQiBZf8\\\\\\\/TBbru+Vu7LAc4fmjS02W50r0mVDXuZse0YXNiwwWKGRFCkEdXqDAJ+22CSzP+4IoEXWyB\\\\\\\/dZ6r7VlDrCDJaAlD4KtlZIX4VbDRZMhzPGqLYHHzCelRv0e7wHL++w7QhP3HNvknZn7+mqReVsg1Hhsus38cVBOLKBjfJq94aIUz0cOpwTcE2ImsrDdlQSdOlqfqUGdWMu8avhmN\\\\\\\/P7wvNijLaNjvZsGVBmJqbA6BsloIotelIyXQI+AYFh06fbsVfZ6Fdc9n9LDEdtNndsrgipdkiEBXMTH3RjnD5ytjNjnXLWg+nwYqlBTP\\\\\\\/Gi4lfIwqr\\\\\\\/oxmTtn5X4qikEEgoHBmLW1i\\\\\\\/u11cc6LasudBMbNyskFWRk+SEVLBHoP0q3vaHsF5bEkIzY+VERQg8lgHK+SFAr2Fbq3mZJ0EqGGapy3apDsbBVzrxpv2pMZ0G3NgXbUXnkpI1Ip5dM0G6JRgN1bJO3WfW5xgR0jToi+vi8EtAs6xqHPr8v6WWEAyMpXQRRNRCUMtasfPw7LswW2RZRY1phmAI8wFp3dt9oFI+V82QpLwhb\\\\\\\/ID6h8YbuD1gmWHdaUb3hC7X4iXz2A2Es367Qcd0lgbCAo620xiOkaLOggfmBxyG4UZOjvu46TnYFj9uw9kE07I5f8YuRKkzlGTjMbxKxar7y9DW3+k0HhiZ7P\\\\\\\/iWzROLMPoEiWaRTyKcmVGCBUnPY7fu\\\\\\\/T+7VHS94wHUIyj2oxZM4iP817Lx5DfmtS1gvWxF5Uju+phpIZAM4JOpoMxOFfv+oafGnF9fkumisUFRtOgnkiah\\\\\\\/XcoCjyfkuuytZGTAOJghhFA3g3u4uBDCje9JCuDAo0CyI1w1taU6FM26XS000qccgG8\\\\\\\/8cQPTuIP2VSQVb50qlFEcD37wG\\\\\\\/yajlhZp7+j2GQZ159w\\\\\\\/n6mQN7NPwEoiLCNa2G3HpjdjnmT+s1nH5MTyNeRIC9aZqz58LLT+4pVDFmhTmHZJLaDFZCqcdEF6CUyaPyl3try0TxKuMYSEV4lTP9VxIAJJTAzD5ZHOcFZFLtIdnXzDYeiK5CTR7QA6iw2NKILHInlmRJCUIEvEMPCnEtRiCPTBdQJM6q4IgbZJcwz+nvvJkK6F6GZM\\\\\\\/9kzvFmmoQVQYUx0pj2LUab3Uml5k9Yc+0OfjUd7eEsgY2HWKpG38Qs+C6kv3dcoT\\\\\\\/JxYnw89Oy\\\\\\\/uk1zzmADm8kw+RITvCKzNqF4e\\\\\\\/nkGsdmaoYhsjlWWDCm1SDg3hU9ZHHpNhvd0LfESjsNdO5TjJeGqkypGqiyjGLoVs+U+QJQRYa5xBUSCk6ixdmNWNLQmBBPWaZyDfzcR5BSkbYIGfljsVcp4Srxy6nru+rTvQfDlD2IOVTm8zV9zaGZ8jAJgJL1YnG3URqJRjxRAnU6W7rVGZWPebaV6YUydJnRy1TxZf3m2giPU5x8ils5ZzccCWE4F3O7vZZdiTYWNKaGVrraREMyOEi0Pkj9W0ZxbXGc0mtvo3dQ7mVCHaGMMUNdnr7OY7UIv+swlGwvNDh0VZmAuvxFXNKEfwyr4GyOTrepwWR7CMu8nWMxnq8czaOUkd+nH\\\\\\\/1raGP+rUtND21T+WyB7C64NT2ncgDcVD6NSuCjpv1esoQB5JZSXK3JvCtQJxaY3HZPZMAd4XLRWvO\\\\\\\/4WvOphud8SOcmbr1KAb3wDc4luoRuOAKr9L7tAAkchrpRzuhWSg7shkvY6jCXbeHryOcqwOadDag2\\\\\\\/uGOfKq6U8pMME\\\\\\\/3tHA3zrgYPlZcb4vKWl9dQoDADtEUE3ovWuhnz4nd4Vpq6yOQW+B\\\\\\\/WJa0ZfA8LQozu78C8o4zuMnA3ZpijQuOqOxCWvcSEhHT3uZDRL+4ee2mdZR+lyOFWCD03BgZIqsfdriOGvGi1Z8El72LtdNgdUal0OoiH3MiHMwNKXiWlVpxUlhfPX+Gick9X6BWwbNimd7arxPrpoyZncQreQGp+S0Y8u3abVLp8PXOnkx66YPnT4mFNF5Vk1A97bGLbJvHk7bZF1VFnE0zOqvvnfxOKPELWAF+oR4Vt2kv3IesBQv6zqpTfK4e8gZt5RgZLeb6XvrCkJcuM+5VUA+ucyVvMbbtpcE6i3KGT6PqA2o4IInAwjbP0ANgLdKlR70rpzNJue22\\\\\\\/ULF9PHQtmMd9ZKB7l08uLvZ6qq8N+QAru7zOcbVWOi+bT41CLmC\\\\\\\/uWtp5\\\\\\\/BehrFvSmfh\\\\\\\/t74vWuo57kvtcgGfTA6yR9n84qNdq5G5ziKuftl6Wk4uawBfZ+8ZLr5JAoxdxpEgjCqlr6\\\\\\\/GCCFV3eGrHMFply17uwpj\\\\\\\/qdchPgfy9BzdJC82IEnWJ9fY9bwdCoMbVPt9Qwwpjd3iPilZFiD1kFDC7us8hm7Di4npHd17LFgav1uQR4W\\\\\\\/DCMSGKcFQGrdyO+TFUKSJ3zQ7Zw8ymHZb8qTOmq0wxqncXtthXJY\\\\\\\/DackioUOtJE42NPyAYasq6dECLf2Lk4+L0am1VcLokENWIwsx+YIdD4vHXQqW4zB20m4tQStS+e4Rjqg402SpunafIPQvgPtXxzL3NJdeTmE\\\\\\\/Ujq6dU+fiqrv7jZTfHzDC9IGHl+g8XnZJ0MTyNFSWpygY7Z0jBBjpF\\\\\\\/9DrnyT+yZMFmTL5arB1mzbxoPjsUIeBd0h4Xr4fXnSzUTGnC5noB4ttB5j8LV2AV9RmPmtXIiofG58Sz7o\\\\\\\/lnrn+UDD4iHnCGcCsFDzna2A3NtqkkcEn\\\\\\\/QtPH08ZGC2h8YcTiFr0hTYMAL1kz2LcKWtqEFMAvHo2iUnqxoF\\\\\\\/aLwbzcHPmgjaIKFLPkINIBvYHU8K1njdZ0qMZCqiV\\\\\\\/kelwe4ZNqXNQs7gr5xZUB4DuQDBObBFcbSRYUeStEysMeGfuTM3TT5YUdKrzv5wkSd5SpAARyXdgdy5OWARgF3sps9WoyXy3ouUGEv5dTIKhaDC3zIggMlP+aVsaPDyMP57Q8UAu1AY2bObyfQHEmVzrGe7y9ltoBmz\\\\\\\/A8IO5VS9v9Kp4n0uqgxrOU35jFDdi\\\\\\\/3KAc8gWD\\\\\\\/Yv0YPiWzXENZJgWNEmYTMQCBUqkCRcMkF1dgLad6yEY++7t2I5SjH8q\\\\\\\/usIdn270H4SDMGHBoVDrSPPpj4WRVJXMuzh\\\\\\\/Ygci+q890JrXm\\\\\\\/PrnR0iC\\\\\\\/4YgOx2FUsBg73LtBEf\\\\\\\/9bD3VarugZmXItQLMYRmUmIEDgjUZCbBWIKmAfXWDmJvzQ7oxD5MXcJXpgOnX1Mzjp6NvsMK1h6XeJ8bIca3JYyvcY1r6gx+QuUvfp0pLhTyzr8d\\\\\\\/NEvRyPHCgrVve8S6RG2IGQA+eH+Rd0IrVgUDOw2Kud0EiqfnsXW81JO5EAVOlLVU0C3orkXxU3yQ9CHtuC9+xOpY2EQE6hcL0yGilrR\\\\\\\/1oKhfcAQVE3BQR910FJ4J1cRw1Ki1i+XE6pK0hMNjcuJmaQbEN5k8HtznSjJLm1b8kg3wsqOnH+lID++qFDiaNeCFlPk+giXZvupOQh5ibVX822G9SDweUaSwnIjX+Q+jQzbT3opa2NAQpBxn+SdvD1mae7e0UEBsjYKj628f2wyQDTCffm6vdslDvBj166MDLCjnicbEbPiVJEAcdv2lk1bqo5NtyvRHIqyV2BX\\\\\\\/DQlx0mvUnLA004cAC7UIXgIcYnuKpMC6xnz5Pr1JN2cBRSR+1oij\\\\\\\/P0oQvSxeyogEAOz3cw3G3RNJLU6WDJySsiUzrZwX2xP104FLk4Og0qBmLIYO8RVst3DJoS9XIvToND1tq2hvI87KxwM4OLYItinNlk6mSH4tAYTZdVhxA9LwyRejIgk1pSXQ7axoOSXn97ULqdCCAiOXNXRuNZZgQHDG0R3v1619iDy41jr1i8dX7dM7E6BpLraNXANNoFpxu25l1ub3VKlE\\\\\\\/zGp\\\\\\\/HZ\\\\\\\/41VlrTy14s6LqWe+A\\\\\\\/nW+GD8mk07x1BN5KZdnNOA\\\\\\\/hgnWUO6p8vVBhsILi33G4Hkf13yIAz6RFi5JxhhREDAeM2ZkTRPiXJG0W2\\\\\\\/1ReCI9JD6SI3CP8L9EoOUSuxcizm2+CPHzaG721tFbGOwVhgRVBhR4VWF\\\\\\\/h\\\\\\\/rq7SxJpw6xvsX\\\\\\\/QyVpPwTJOp3ZKhhYFjnXoFCS+OEb8Peu4T\\\\\\\/mzJ1\\\\\\\/\\\\\\\/8ZN5\\\\\\\/jVLESXJ7EiTUnrGgVGKo5DGE3nddGCKF65vuEhX8dL8\\\\\\\/fD\\\\\\\/hgPSLrDUmW3bHwOG8CjygQvKFIcmzvyetK3rpqjOOCIDwjeEcX5OJM\\\\\\\/6IpnNkQ4Srr5KlCccrvy57ZTbshDMYqsRb80N1zL6hZaAFJ\\\\\\\/YBmlbswBaB0EwJPfJGeJ5XQLuPl0WxbXBW5BvCp3EItPn+AEqiR1+gbEVWvVvCwrFPrriTTJiZxTDabqmn\\\\\\\/LMFc7fqbFfLJAkMtZhpCg\\\\\\\/jqd3iY3Wf10Wr0xjJQO3Mq2iwVnmwdTToswUQVNkqYIkmb9r5KI+CkCannkNbvQm4QqKJXenLr391c2Ko4maUtP3Zz0VmdepxfNRSHO26y+p6ETOMKmtM5lXfGKtGUDcuEj5JRlHjXJrdChajAyCG\\\\\\\/rF+R9\\\\\\\/EondGntpaEDtolyx9kzv0wdHwCGNHckY3hocDTpwnil2zUWaGROUXES9\\\\\\\/08Xox1Gz\\\\\\\/r1V7NxuWPpbDB5kM0ZDjEFQVjZ8j71KMYFwoBh1Z\\\\\\\/mIlCXa003v1Fgrej9\\\\\\\/NSj3EFe7xaS4gdBSeb9c9ojUDPe37St3qRzMvq5zlEzgO3AviSS0jFZe0tzhpJunIz\\\\\\\/3eTChaBh8za9wp3IqbIttAk1apYUkTAa\\\\\\\/ysxXs0f8lW\\\\\\\/+CbQiXcYbzN2eSJyHDcucOlKMtRZ1alpjHFJOL45gBZQgd\\\\\\\/kZha+7W3y9g4w0i1qbKWlVN6aFrhkNkvf9MWntvqlFIN+FlSBcniAUvI3qCsrMTgnhb2MEbg\\\\\\\/mJu4sdPAd0aa+YeZE9fhiR54mamrk1NJ1ERI6szZ1oMztjTazO9wO0ixNcnffKU7GTYnMwaAmUXgIP5+ACO9PBX5EFdiquJNXnZ8Ll9mg\\\\\\\/QIvpr+OCTaiC3YUdIM\\\\\\\/0ZG1z5KufkZJxFWzsSFMtghdFiU0nlXsEkEzoXiytN4iYUy9dTuBgDqbFz2qMfLePku06BlluOI1zUhqLNqO+bHn+XbIPtUtKMRxAaVcsQkid4GtJgmwskwilm+jKBH1Wm+oD2bHwU3gg6TjaIglrBqMIy\\\\\\\/oMqGQ07OKri+GVCYOs6slJsKA1eRiJs2m1Bc6afmuGXy2v5YlJR34+9NmoKDryNEePUVpS9bYzOd4hW0dScGdZdJ9GkIb6IPNajUTTpXMGDUU9U3p2OLyqsXQ+N15l\\\\\\\/5WpOwNZEr0i83eDsppDH3PKBBOALC99IXQ15gf+3tT+C9F9Wu6Ytc9Q1v1i1gnpZkI5A9bpAbpuYEXUmtiOuXJ7SxW\\\\\\\/Sfyg0moMhVaRnGR9y+VtpK+iMNsjOU+VZ+2vCms8S6kfSOSjPc3jJjL\\\\\\\/Rs40ussuoq2JGDo9d+aeixHPxeqK2A8InedAic2X5Ai\\\\\\\/rGgYz4dSRLxZlzdlJ\\\\\\\/mPoRfwVZOgA1WKTI6KYY\\\\\\\/D0wLVFXut6GAKNDc3HhQcshsaqPA+a2uQs\\\\\\\/bYMip0EBLgKYi+cMhCibyDkIrMovO2dFM4LLzNzJspuklqJLIli4s2L0k5g0dbxmcmQz0Qwrhs4YIZL+H2zXPCrt6CYwpfl8t01XoSIc4ZHcp1bt1izRLqHtpxQAXb+\\\\\\\/og\\\\\\\/HrEcCl+CKuW2kYYrYWphC9fUD\\\\\\\/NRwjqprTAQnUufGCO5wb\\\\\\\/I\\\\\\\/2rCmR0le+jOTfGzwmIRChF180XgOOwOK9Quz1UGn6zCSGCd5NGh8K58Lp78x15wQfw+TD4gBUUKp7\\\\\\\/kxu+frtcL56vJlq6swLjVr\\\\\\\/q+4wxbfj\\\\\\\/yZ1GNZPsB6LQ7grgNBYmaLwdl4sQHTDvlMvsZLOnSxKOOaOQGYgDO3j6fYVFpvYY0QO0XXCGkbfDuLmrZoNWoXmbfYqzrPHoBBGKvSPUcma0XVIaa+PfGod+DqnitH4jZJB458xF6Yln8GWmQJj4pQ7i457qKUA0CHYfSEgQuY0x3STRwQH4x08dJ4rhzrJIotBBXZUwEUsbzRZmiwib+UTOAWog1Kj4arKo2O5zz5w9qnB58WSWQdUd4NnQzCfIc4QlSsGV4WhWprrV2MwCWobpSlYFUXBsGMbz3uGV1+u9bD2xeDpFNvz4sd5ie36r7MqmpQjrrbl1iz2jMfnXpugx0rW6VMgO18y27tZHjNpn4sCQs9MDS7XdrBoS\\\\\\\/q6ke+gdMuwZVTIe4Px48jQLQ7uRFlJqDKtrWM4UiZtUHX49gVwHPQui1g5MgfA0h86RyuXuN\\\\\\\/89BRwaz9J2fzeVSA7xaYqKLE0AxreyEzVubg9CweW5xqRSNZxkynl9oRcUdS6W2+s\\\\\\\/xcmhK9Pcj\\\\\\\/PsMGooNfMrVf7GSD\\\\\\\/ShHtoz+EUwqk0deDbwhmiKTRK2S+\\\\\\\/DdgqUblZcoW8rKbNFsx5UFy6WelAjPpJoxmpu5LX5YM45aB6p\\\\\\\/c7EjiJHBWnUy\\\\\\\/JPJVA9yApf3MP2RhNxLYObybhmo1tdhS6lh9e7fMnphOP3jmhpzJyzgAb8VUqoFNxtTwMyDnqLvEOdBVNn9MPQuxn4HLOYmtkpo\\\\\\\/zf55jaF\\\\\\\/bXIt4ZDrPZj0F+zU3aOOrCrD0BRhJYfdalB8ML3TfGxmYmCb2OlkizbRVcEieWcjtA7wzY647eXWejGt5x1ESe2qOZhDFTkujwPM43K\\\\\\\/wsESVIP\\\\\\\/eXUMu71cAhBluG5iFPphDyHAGEWmVIDEOoOShLeRhypg1QVBMRj6Um1hHBhtAQnlWV67vtSuWoj7j6cBQEdjQTUWYLc2vyD9kn4HPUducbBsp7X3CbbvbCRTHD\\\\\\\/cfWljSaoToe2F+cymT0oLcuMAi1WXX1bDTrDwwwO5D\\\\\\\/j\\\\\\\/eIBXrAdRpM+jZSyxVBdeJZ9lGXXvGQt5uTcPihWNTVfa0VN\\\\\\\/xOa\\\\\\\/VLtqWAFk7aTk6tQuuYfGO0mGJHgmAnx5ZnTH6YaPG7OWbCjqn13f6PJMDEN3zQ4Cq15R43id8DrtTxBXVzKlSyWe\\\\\\\/neMezikRyr2\\\\\\\/En41mSF1Z92xgOyfnADkONXKCoISDRu7Pm7I+PNgRpjIvVPbDrCH7fApJowJzspon\\\\\\\/Q3py8vUs9omQpI3hMx2rsjAy\\\\\\\/kzFdpqQuWtCOBC+c2cPe6quV2J6n4pmwZLvqCeiWQBbiPhNFHfTi3lO6KV133xLhMfHjZPvRqaxJnpuGl0Eic2regXsQI8WL0mqc7HDBvqX+TUwmjENSsIvfM2M8ymyQKSkwTF1q0iF3w7Lsv9ijdzOh4d0jJAy+2a7aKmjcQ0uvrwfzK\\\\\\\/KcsM2PM5nO8zcw6VkVRZNU8w5l9R4I\\\\\\\/m+8LwNKkk\\\\\\\/ckTssaVp4jTWgGAt1QajuI6xZkMxgJc8yqp5Kb\\\\\\\/tVGqb34zdRGICja40WIzB1L0HfQZEFU1kO6S+GQ2yfw8\\\\\\\/9xk4ZIi3O\\\\\\\/7Xcdw2FLjhAZ1+7zmIp6TneIWt8Z2ysbm\\\\\\\/\\\\\\\/QbFIvG4pXQPxn\\\\\\\/VAwZCJ6\\\\\\\/DOpgUlH+ao24YoX9mJfIcZbpIYOh4iOI9\\\\\\\/aZ5hkpbHmnnLvCFmlrSA2qnyRtEfEIrLEYoF34hJS90Ng8WfJA3kw4SKHczLe7vC\\\\\\\/AKxFgTQosEiGO8pV06yjHaRoNTq4cDnbKjZvcQHxH4zGP0gHNE3uKfn0G10d5IV\\\\\\\/fay4V6E0tS0bWWivMD0NJXIGcxSV0RzXa5I9lgug8G5aUWSyepgadZl7Yg8arant+AZ+fuCZGRqA1a2TDlXveQL4Hy9L1IY5Ht1u60zacZCY\\\\\\\/Hx334MSo\\\\\\\/AMavSRqB1yKjs1yqJJCRKPY5FC37x32TTVRTdSwYCfNCq4yHAbzDVeX8AqnS1kEOXqkQHcjV6gbEAELZVwCJLLDnX6Lwl4ca9yg4KS+eMyxqjvU373SSP0ZM\\\\\\\/eYcIOuElCSAe4kVLNDxPDowJB+PpE58SpONw3mDNaTHMB+E7Yxhvk0RRXmRoXOxpOZImCdI07mj0EfAH+JijgHEv5UpnQC\\\\\\\/8qojKUX9AO0q5b4kHgmDyj0uQ1QibZNAg0\\\\\\\/qy1tO1i0sAjmqgDfQwScti+EhGJQKnAFTSTXefyVv36rcWK\\\\\\\/1UGWveyIEsiUhp\\\\\\\/pjjsW8OHDrzxrWPmVF94BkYQhYvyPI1lxqbI4CBqvdIrMP38CqLPaAdz2\\\\\\\/EUa3AtECOccxdiisH3PsJiys5JaXlzqpDMiE5g1NPqdCKoFuG3GFFGfQuHeEFGN3r1LEXi\\\\\\\/vMspOmSu+6YHmlTocKOZELUofmoDc+umbLGDRM15bO0aC9EacWoXmCwF9F2pWw\\\\\\\/LHJBF1VDB+lEh4WTcSKKU\\\\\\\/JONGqEyFGi4Ip\\\\\\\/Eg1DLaAxZXseSahJFJIfi4B1o3Ei6NoyqCUs5pU5SA4bvinwirmu5RhKa4M3hoDtiC4uMF9BEbRNaU3gF0C5SF5oYfk7w5nVNBT2Nnv0Q5uJYqIo5WFR3Y7Cv2bDJgUqPAhnsn0gAyq\\\\\\\/kjcOhJ4eTONXB9JxjKrh+ykV1ssNOs8FN6+X8NNtcw4jwr15uHldGYkEIa0jivaCph9OsdxhT7PwoDX4u5wWanYYqa6S+bY\\\\\\\/kJTB3QrX\\\\\\\/6EE1vsJk\\\\\\\/N\\\\\\\/NJ9yJxzqodZ+jCxr2LsP1O4FKAv3D8Y8FSHVVN\\\\\\\/OSOK11RLY6ShBqKx2Ta80WxpM8FqVGLrird+WY1L1ckC0TkOSJLvemThnc6yOVkxGFE6TzfTf\\\\\\\/w1hnFtpJOE0Yso9atc+LOgU+cyp7suRxkeVe\\\\\\\/5qN6smqgrWpYvETKC4DI5l1IEIO+ZEDzcVUG8CnRv8a21wLlMF8fgbjAdh65LRrsGvppQM\\\\\\\/WlwvKxyV3DLrt8NYjoTAQ8zkT14Wf\\\\\\\/2xW8IZu2q2hCrnMyFdNmDDVPFVfBYm+VpSETUOIOY13aFMTGkbFTieEFM7Q7kREuIaShX\\\\\\\/AN5q4K9C1YtBU0SIJ3QEMurLoO5FvKBkxF4GUuEyaQpzhprVsO8ouXm4kQWMyt2JrCfEtOjD+zn4NMvOf\\\\\\\/1HbyNe91a1As2eDGBqwE49ER6hghPTxO3+P+oYPcat5+\\\\\\\/CqK1JgT+O7vSBE0VJboZBEptWtYL8L985x6LKJ0lrrXHEqgkJnnGP+cVB9Wz939M4KbdgYvjOcIDZCT0QQQMuriu7nOhmfGtW7JxUuuy2xvfuFegqTynlqdTaZz5Wg5AgrY+lXwEnwwDPEMIXJQLNIrwlpfznSspN81OOSPHniPSVEIdy62JBKCSRQIW7vqWgtT7kWbaoGOQ4riT+DlJnOJ1QAfQWobY49\\\\\\\/aL+IjLveEgoGk8OhI+x+e+IZE7IfvzszJEQ7U7EWbzf\\\\\\\/nrSTzZq85K3jm4z2tnk3dzHsHGeFvyNPnmNybx5CiKEMh+mYbT+NVOg13DPmfQISABAnHjJI4GeoijrjeEcfhBeR3W8tcBY1nFIIJz2RvD5U7tERuXS0CKsKL9+qLuXPaZPLK+cEdIomVumSHyP9gRONq5gyhiPtaFaPRJPoahYLhUw40KmHPZ3N4mpdRRHCFgKs3GcSUXIsJCaASFpU3wJrqJPOIjYiy2wy9M9LatFyuZjgHL3o+kkXViBGNWgWbCQmAqkt6vax\\\\\\\/+d+gVcSdpYPjw1uf6bA\\\\\\\/a++U94MFLJiqxWpDN9vOIjKx1z6IdNn4nvWT0ATzePw1ZbgsZJe6rk9IRIGpV2BhWGTRW9QRqeu5QGcFYPz6HaYHkyu110liCO\\\\\\\/1bllBvt8R+hSkx53BBDYLwwZOwlmYyPN99uRr1B8T8sVKXE9ckRAsnrmB0bQ8FF0YCZDY0fx6iEVX6t9elh1mBxpcfu0ywThi+t4SLGQIYA0T9oinQ6C\\\\\\\/XJcAJ7pfl5ELYlkIgtsjb2J+a9rMejavNY2N8tMvxZ+fnVooMe9H2S9+FSU8Q4LJobTwadmFn53GqSuxB8oJxVGgjXUtUiS2pz\\\\\\\/S6GELjSkehXd5h5ieKcfmE5C6MpOO1VnIJLc+P5oYKwzW2+Vla\\\\\\\/xWnmErHhGlePfLG++5ynkMId8BAwr0DJVGSt9N301tUAMrEKOCWM16seb\\\\\\\/UtZEuVmTiPIglfYu2FahaJpAWhGvupjZrvDPd8EQEK6S+cakjeLQ48KmSY1WsYiPCpXBXpzoD1jxz\\\\\\\/GLkSkVn8BEkx4FZRtC9WbWL3EazMFIHcufbBdV5CVyJv39j94J3UHNVpIAk0Xd5\\\\\\\/JUew7PC6VFx7zlkDs95xbA6qi963jTnvuOBgzAPRK+pCUzUUHZuSUpMyufxm0RuPw2qILSmISPEUgRoucQfxftJLra\\\\\\\/zZhdG1UxGs\\\\\\\/Q5gW7D+1lHBE1oNxmZAjE29gmnTa86RcTRIZ990O2\\\\\\\/DfDhPsC0HabX7DRGK6DEekNpXUPX7fSaGSDrTQnZB2j505W\\\\\\\/DEDO+A78OS6yB8GzD2Euh+la4vgxDAvk7fVnlORXJGtPa6\\\\\\\/m5Ijk3vsl317uTlO8yP\\\\\\\/rdoJarpfMhBpvi0O\\\\\\\/GSBmYL+9\\\\\\\/LidS4DLdM9iEoNIhYseADFDSsR2ot1Lh0s0YDi2aAmDigEUQ8AE2GN5sLeQ6XjGim9fMPbbCzve6eSvJURHg5EBNO0Z9EsgK\\\\\\\/n0H8KVGmoOQPNisVjpgZnH8333xU3nExzM1p8cJBoFJ6T6r0N6\\\\\\\/wW+Ml8udTde85XhZLFS9eV+NqWP9C2Y1NmsQwy+Q6LbVkf5RuA\\\\\\\/VIy\\\\\\\/b8k9diHbGT9Y4cTiMj+eWSEg7a70Qm0GQU3ZRl2huP9wE08NCoZjCp+LZx5l4lmuSuffRs4PPWvEECMJspW+\\\\\\\/RIl4c8KgxrAvRmRuPLJtttzL\\\\\\\/FKJ1Ab2Ehbvdkuw3zv1kZZmXR+LLPtpzaeA+GcAcT+lxcryquQi\\\\\\\/FbUgL7f6lETP0rEcToYTFuf0c6uTUWOwe3ptA9LqgRludK8yfUgPunVcl6cN5Bj1IXcYZsTylp43gRIc5zqqrkvXwXeeTvl2zIoP1lSRRmRJZfOohsWyu1OTJCU8pErKKP7pppapgJE7xsEw+yqO0mw6LRxg+xt\\\\\\\/0RK0VpVEuj13uAHmne7ZdiblkcHVFWSNNCAu9rBtIr2\\\\\\\/30t5mHDljVyje02o8GbOU5\\\\\\\/h7FRxj0S+g6bi1oZrenJERwXL+mMVqgNvpFIKSkbtB6eQ6P1Vi43goiox1vce1BoZtJKk\\\\\\\/8LIw3deHUQiuoWYv5\\\\\\\/jWhAWI\\\\\\\/DJpMuy54HboR7QhUADBxN8tlzuCAtWvl7UVkpMC1Y35zN7CR4CAOSyu+hpvmfC9piq4yP3JXntokl3vvPgHEWrkmPfy6HmqHS7lHP27JEQJsnc6z0Y099mCIl5HrOXrNLOINwr3nS8KsVlGj3ohPW1W88uDQJT8A3gZbGYTVOI+Lv+sgPCkc3baSrWNIGx5xD4rQPCGrdE8F3LFLD6BQLTNr4bzSSC\\\\\\\/S2WHd7nYyVMOXxqUQKFwTtt25JN6acrm272ZpNsZAsMGv9IYOSUKrCvLgZCIywdT9Xu7zAIXG0PujtXhv+ckczhs405pZeLM2bDKSxwCvg3K+PC2lYXjhty2V6Ms9Wk9ypjCiB1S3AqcAggQzV\\\\\\\/pDgutGPU8oq8chplVW3Slhs4wQPMd3gr\\\\\\\/FPQ0yAqpJdCeF+20JUBaSErXOH25Lh26jnJShO4mWs3arCU9iF7KVh8\\\\\\\/UUMIioXmwFQzUWopvBufbEY0YGbsAuBeN8N1z+IJVLPTf1grGFTintbYEt5PeyZv\\\\\\\/cr0bFztXEEDNbxoPJ64GdRzZNObjhoMcPkS4+b17fbuYkslUVq1205j6rW6S+4J9YjNtNzk4WGOnfSvxhTnuadsLEvC9YG1MWP5p+Fqtix+\\\\\\\/v61ChhPh6SJ++oVRl2qp\\\\\\\/oU1HizRDU9a4xOXJNj9ewWVfqx7RFEv6QQvsWFe5GAW6ErMupWllrOzeOJHIyrqiAnvyzXGupjBviX9bBmuBiN9vcbZHSiyvYr0tU90Z40PVp3KpjAE2gLLFZtb5cUJnI6PLhdlZLMmntPytWlGFoSaauygkkJrenItZpBf9U1eIA6\\\\\\\/qhRcTZhcVpi80LcG8rOuA7E41KfHiKalGAJD+Q8msCWfw\\\\\\\/RvP7CUVsjTfdtyd+2MX65GbE+uCfxjQ9Ted\\\\\\\/y033B49kw9wfX5CC04MYAxp4ds9wfs3ekL4fWsriNg7Yy2rjDA1FcNuy2Nzi3dmxZ+JSoCUE8ZXz1yN3NWJDZkfXaNgP4bLaAMmRPY1WOIxLyMhorpTLnxCpYWae5xVDoCOJd3mq96ZitiADYcvKiQhgcflPTprgqFIb8sEWFPMlBgPZMIQ\\\\\\\/t18E4I0Kz8WOqnXkGp3oLRh\\\\\\\/XdbUWQiaKFjG+BaDUGAmS\\\\\\\/g8j8fa+6dJ1GKjDXTSwzoxAaSxQCwguYpYZuGyUM2\\\\\\\/yqcTpelaIHqYiFmOsLgC9O4jdBB5OK3YxrobWZhJTskYv0fDZsMQaU0M0dsZznmr3GhOTlT+1aK\\\\\\\/fc81fzLoMdi7j8s8Y9x7jwzjAr3SB1JT3Di\\\\\\\/NwXRDzCbBQlmRrviSRi0dSCdZa2zuBmq03j7ADJXdqFcc3GRJm51ZkmNRvJZ6RpFpx1f4k087CBLZcnN41X61ULcr1hoi\\\\\\\/xORP0WG4FGULHW33LLE\\\\\\\/JXhPB0\\\\\\\/7qo5fnCVLXom3DNFL9h3LhF+h4\\\\\\\/E+TG1lgUf7fwtKRYC4JysZXp\\\\\\\/rnrZ4sO\\\\\\\/WrKB22aUmm+hEI1odLmbZD1caoh1LnNDDlKs9Wsqb2NiuZs8Gyk8eHgdPYLqOM8pASNHnPq6\\\\\\\/JO0LeI3VtbcINBvXEWtJqauCGPaNbvlZilndQQph0bQeGlAkImaxSaf0pwRaUz1f3QT1NSjcgDdg87o7LbxdALXY3ieij62csF5z+V5ynCEB632P1bRVthcAX6NoKX6pNMEuFNUW+u2oJL68i9xxlUVyFvaPTr3Yurt6+c6Qr9JkEZ9NVfueBtuPqWdFqpsRFkb+isV926RcFjDaAZU17LapJreEo2myd3m8vQ73jASSon21jwRiE0Pu9HMNEq25f6mssiXgMTjEeg09i0+fXYF7ULk8x+8Uya\\\\\\\/8lVwpvo6pMxL\\\\\\\/lXh91DFeHVF8gdccurqH3VeVEsEghjYXNLzfHdobWZ4EZqXhivqnhJE4mNS8aXfdOWmjTbp4KCIDUlwPquvKHhCHjTe1E8Pt8HY6\\\\\\\/xAzTfQQXnJwwe\\\\\\\/YuD+t1vVugIAK+IFduMM\\\\\\\/+j9xQhZMEEc5RT6cFP2n64BYG+43jDOEFbUTXOhF\\\\\\\/sUW+5x0XWqQQZ3AgiYjy+DudKDhsXjfqH3Ju1D22SS92go2CwFDNUK5VmHr83z6Do372W5Sof9upql9278C0DV+uRc9+kCsAH4tIcXIkAeUWQq1cp4aPfBlp6TyY8zdfxirqXxLOS7N0F9laeKZzXs\\\\\\\/9CnUxV+dQ8QKm4BNJxiBeeudOp9SHlW73HIDJoeaRsq8vRmleBo7+g3X4h3SJfmjm9CB0WJ9m6Nqozm2GDBmvcaZKn1nu1+2WuBDZJZs+jgdP8GIBF\\\\\\\/gYU1LsodNF4kkVvvFugUaezp0cJrHDlL6n7c60ffx8glbcYg5oGUsV8gVCvhhm9E4W4yfdcMbPq\\\\\\\/K9xrrZvsGcE4A1fkYnhz51mKil2Jv3P\\\\\\\/u5qud6qR6oIJy3ye65EJVy2jmlUyg7NWzt\\\\\\\/S12XKkPgAO4n\\\\\\\/a+x3GIPu2W2KaMMml2sGTlA1lYdwQiBjTl9QP2KNBBiWH+3mcjzOzfTb3y416v1YedqJAzSTsW77q8pipCHsjbkFQIE0M93At77HLi0bM\\\\\\\/hbo16THvnoWKrcIaGrmdxC1go98P09CvNc7Lb9oXvbeghyKfdbBNEVglqafhBQVet5DWAhKaRk0kXfmfRhNeB+EH\\\\\\\/GJ8XYdPEffPdWGxBecwOhGN3No5WYg\\\\\\\/nJVNUJI\\\\\\\/jgOXygySMSci6NlPEUOHE5GZ\\\\\\\/rFvmwr\\\\\\\/lw2DbHR55xryFp6A1XLfFKs0LXBiRwtTHx2h5fHzwP+bdLG4Ox4KDbTiyJylIFpCgPHblgLgZnazgQxGeQPIekKrAEirP3kBSYb\\\\\\\/bv4rB0ic6o3mINxQfFNw81gItmaRYpQEvLMJrWzg2l2LXX8YnpnGGFri5a0FfscPTVIiDJdm+ROAUP4edAwiHvRMLGv3JN8gpBlmDMEMPVd3LWQh9T5uBMqsVFDO6V2RzMOWNUXPaF+adzHP4BuRKaD18nbSvYsgtpKZyPYfpqAnnUe\\\\\\\/qLFKNWWm17CGYogN71LukPwQPnKTe2+PqP0\\\\\\\/fd57nzlfSDgfoqDm0vl6W4muuZutK75vmvzvPoFpO3i7+8f9pCuPYbIhDavxblsib15SRHQEX4TSXdiTthGdcNpvZ7Cb+y+\\\\\\\/\\\\\\\/nQ+4YwT2w2qB11H5Tpbsu4iTa\\\\\\\/UhmDuVkMURQBHW\\\\\\\/VTqxC\\\\\\\/A4IJlThtClZJKuYfQ0t\\\\\\\/z97kPmUtxqt1czbLBtGWVptWtzm11YvFFnX6ZyErO9eY\\\\\\\/RWFrirz8a1YtlotnrXc05laYcrSwuzZ0R7VN2qGF9CfjnozVBVvyEglL5gKU\\\\\\\/BiP9ro0mNNuVYzUQctFcMpTQ9KyTKSz9x5lzMVonGNZKRMmUMS8yx\\\\\\\/sP4ep5oob4tBn459k586UbzktWgS4IplqJ+J24\\\\\\\/3952OupOJD0kUppInmMcpKUWcIb4CMNyUqVtrRHS0rOwJ2bOSuoz7hDutlWmduQVAxkaJpExn+9hycyhePHb60bYQX0BUMUl2DWac4oHxRpUb4uS6cgIWi+VY4wYniYUj5tYW+KhEqWuO9wvJHlipdbTvmq1Tc7Sspph6Zu+XRu9ippCBwDSZHO\\\\\\\/hieyg2qp91zRlQR11bED28DbFmjpk4JIjd+W6\\\\\\\/PrPP6FLnDjq3zJeutSa+potXeNnCxpbdTC\\\\\\\/NiuKz4ttjr1ZmE6nNXBCIxaWMW5dWmhChSyRzIagemX7Rmirwrwr1k+JM0+1cchhJFSwSkEYb+4QbJTMrCoTma2+oWsObjLriOEk74gPvrlw+knvm\\\\\\\/iEatXekc92EJTxmFtcNiiEngqxtW5cJFJ9EWUoCrF+8Bf54ngjU3kHAXXtqxPtPZj3h1vyITmmzNxNSNv7OuxgueIHaq8qOsatez4RUkP\\\\\\\/JW3M3E06tUjyWPfWAIVTPq0l8D1eFuXEDrDReLc+XaOnwephlEIPbzU5PjGKAIgYm5P6zFkmY6qN3\\\\\\\/etBhlrsEVOo+uz9UdS6J7C2kfTcNVpQhTBDWdFB\\\\\\\/arAPFAhsaK2F\\\\\\\/+cjIMmjLCOdxK4ZRzqoeKJozuRA2PLhjPxDCAmGh0VEQhU1prw9gUss8lT9MH1wSc+neVExNIeAjf1BLoL2qY+61OAH9OfDJ1Tw4w2YN7LS5OpeYw3usKtihbo05NmEG4pkopG0s9Xh6++wZehl+51K\\\\\\\/DUj2BglYCyKLHHiamxX2s1fasKoPElm7i1eyAe3xybaZA8Zqy0nAVesNKfjMB2w8cad8AP1CJhSNLFc4VgZKXDSQvObSCkv7rMcjNLsGA3rjvs+N50+inay2QPiFCa+WY4UF5194pGFY\\\\\\\/wZCkfXzekw4CTcjLMbJC6QE86ZG2C1G\\\\\\\/frv4\\\\\\\/zF+nRQmKkOR2yOTXphwR7tQSo9MkHUwo1N+hlvqAPJche0\\\\\\\/q9ZEzhjGWTMIqu20S5Pyl9\\\\\\\/5q\\\\\\\/tmlMhhIAvd4qaJKQYQ3cyBN4Ci3w7P1PZ73IpfLjSeDYQLlucwddkMdyfjHJAKzl9zmXxJf91+7o5Qwlnhyml4Xe4Am\\\\\\\/sBfWhZMW7JQmsXcNlWDaWjm6BmY2+9aH4TZrSl5YKeBNoqPwnuSGRaoY2nmUCy9EroXtSpreW5a0Ezxg0YQCcj43GDIGswxRh8arlonpUflPH7MPj1NAUIMrv1h04cEcuLKh1Mj+3TCRBCcP9mKF7ADKtMXTUH8ECKZ6RBoDSrzs4+mlRrfwMjv\\\\\\\/Q1FVwg0EGko9xkOvrFM6ybsnh5uYVjy\\\\\\\/fI750tIbEDmKY8an6oApNnkhw+U7TDtI+mx5gSxt5FmB3x1pXp3vJhUBzyY+zeb3t0rwUZa39hKY9Ut3vzLYMO0imnA3KDQ7dQLRJ38Lh67zycAC\\\\\\\/u8G7HAc\\\\\\\/37UOHs9Rw2rAHdV2xW7TWgBnqokl+yO6VpaH1M0JAtTc\\\\\\\/2Wtzb5Mbd81np71RRqUJVYmGwdtcxkNjvxpzNXNna3kKtZssAhHW2\\\\\\\/68thZYDFiubXa\\\\\\\/mxeEIDzsRfRdo4PEua4Yg0cKdYjYpZy+htqFuzqksC\\\\\\\/OQUZ3amaZV3sSzx6jP0gUGPtmlHt0UdaBYqJralxB3dWvHhgLDaWTJ967Fr+TKOItxRT92\\\\\\\/G+r6LGesFg6+zRP91F7mEaqwygkzKjvSfeL3amHOPw4a3y25uN44+Wtw\\\\\\\/9GQpuYEj9ghgn1sYX5v6KDCi4Px2N4vkr1khguc2fHjUxpUmalKmSNye7BFkff6+u3akA+iLYuxzQED+8gKpUr1tUY4SR71zf1a8rHvrWHYoqttHoHz13PNjhNw3vpVJSiH7VUbAdDFnZlbNcT\\\\\\\/28kB+7A2CTRy4PDbN7W\\\\\\\/JaWM4bXkTkEeLyAtfCnM3vMfoJF+AATEnc6b8k0R3yX9Z8Zz6OzCKJtHSYtaW9j6mlWCILShOJcRpVP1FIzh2VQswEgvZNBkOFU4Su847JS18fiBiAJaIpM7cN4uRW\\\\\\\/Pb1npAevqm9Gb1jLdausS3Iig9E9ERY\\\\\\\/xKraZ\\\\\\\/MswijSkg4ZL4sQLcsp8rn4CtZiJhtA5xd1df52yeQcDJwfH9jO4P\\\\\\\/yIQZ8KD1VHcEeg8JQEcNznOZlamMShXpMA4d3und7NeX4GlsLkYMdQRzcC2BHz1aAkoka0+bC5bnaTTZoRTQ\\\\\\\/\\\\\\\/F1tDM9DU90Q93TXf0rihsy0pn4YvrjyQm9otFkGR8PbBftcOpeVKpJXoYdiBCII5FT5T0P2DI0DNik\\\\\\\/6Eq43mm5ZvJfgEVbUiqDIxCJzhltV9v3Kk6kv+eTWaQ\\\\\\\/KE4wdlTUWoJ+ekQ2S\\\\\\\/HDJXd7HXyB1\\\\\\\/SkIFQRqBIwefrLchgP\\\\\\\/hxTQMyySiUrU5xxMawhx427gnunDuPlUYPVIiVulq8xfE4sHedMizcftm7TgzLyM63XSksPvhXNJhI9BLgcZ15qsYhHxnXlN\\\\\\\/LrOBPKG27ok1s+ZOiJohcfpBfLvzSYJ9wVyuCRL8ymyj\\\\\\\/VyHhQjjaHehg65hMJMqWv1rDH6Tf8zMMoqDorNkk187MyJfZTXJqVPt+7swZ1p2bcqRvQR9rHIn+VWciNw6HEqfLiDp61n\\\\\\\/qFU0\\\\\\\/HQXBq8+nEtCY6vwwv4uK+\\\\\\\/8ThCsXE74m254eXRk9AJXY1OaO0TxEq9pXXA\\\\\\\/Q7\\\\\\\/syHtOpsiu9ehRcriVECrLhE8FWyBsFx7YNSY3OCoDu+qJa0aO3bN0Uoz5+homs+VhfKqRNPQ33Pd+lc3spc3zFPXqzouhEp8nazy5rjnAShSnGB1XOIeA3lj1abuIF7AqW4GGIVd4C6Nb9gkv9kPvD4DVV5X6sIjXcVfAP5YyWgaWlF3oiKVZ0qRvQc2AImKdtKS+gseFVDHiE7Jz+AMPjHtHb64DdBg2Ru3LmaXY4v9bUjzRWpcks0FpC1F3Nfm2SvErf9I8S1iAy\\\\\\\/uLjfefO7AKxMNy9DnNB\\\\\\\/rqkp2HNG5JAC2vVRgWav0JKi17xKT\\\\\\\/3EWHw9QLMwcUIhJn04ned8rL4saJ0nxsCyo8iqrBcHEQzlb1oJr2ZUNnfd2lm+C2kb53rP2zg4dedXaL3aDIBuDFh5SeQrUGgcSClR5QnigqBcI7Wm8HOGVo+G+zjyVRF+BYfUD75e2zLiaoqA+50KC6WOlxDpjK1fncKXJr7fO8lAckKwQt38YuR0UNZbGI\\\\\\\/X5J0iWetGNONWWqRNfSWO2MxOlL\\\\\\\/PCNo6LkgyeZI6mSFG8CR6cm1bqOAtkEdeQ98WpzxhTtXr0nb74mHh9wRoLEtdETYcW3Wmv4Ps1KkgJSX7h6JHoBFFCNCSE9Re6EpkRAD0iBEEm+sEbuPvql41Or2agY0cBnp884SEOIrX\\\\\\\/5NOEKv9XStyVjc9ntbAXo7wmeFeLSRhq3X\\\\\\\/99Z3rgUbxtWF2wNyjIX7pxj5yTWmG7Gu3s5DOdHOcZap9BDCleKdzCwFRu7RD2sb4oHFpt+2tFcQZAzzMCQx1yWCcjIRNcl3takB3vV8jTUAjmPj+ZfO+5K9rIjO7bScZy06gUMogVyjL\\\\\\\/nDGSuGwmPflpfs6jxzH\\\\\\\/nb8oLD5mI1DuJMkPyyse39Nv5ETvnsYhWFv5sHEvGtWR8gCUXv2QJ5RKD0ZDtBIoD7TzU9gpDUnfnlmgsMc+CnhFNpgn1qviYNJ08YYQnCTxwliwJ5dj3hcl1COW8sp6F+mVp5uJLl5sUhosEh72KbQnjwOoy5gGfvNYZ\\\\\\\/80tzO0zEtOb6Y3lR+t6BXS++ebe1qg5a9dNWnpz9NedrUN5+CnAS69LU809K3+EFz+OYv90MDqxfQ0rRNVDL3j\\\\\\\/ImIGla5JVeRvFEeZ5IcolNwc5el6udVy3OP26urEwr8YFVFv9+j8bFOhKLQhk1sv1R0tlhjSI3SJKCLITZOAtF\\\\\\\/mR0YYKahvzoNpf3MBCMn8hyLIcJfZLkOLVIvtOrKr8IqQCAkU9JSJv0lvTX7+\\\\\\\/9\\\\\\\/PlhEEn+BgDTD3Oy8eeeiP8NSFytDNhTeemlKSqt1UTbJZPBckqbseUB1gGXzif3HkLTFHwitXoy56Uoi20a4fhMbumI4eYSYaza1vvRRJOxjZpnvuw+bnVMCwdlhqVGWCI9kDcuXIkCTC9eRMUBdAASvDhMnPPgAi6E7ShdnKuhFLQQvtDqYicC+KrQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EYmjqKB7eRnlnCvnQUTiYg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dd0237344b52c78cb1048f7f6f42343f453851ddb551c6cd169d0d81f0a3a671\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RbXPUhlQyn\\\\\\\/5hJbPpTWuhQ29HY+canIU4PlEx5jzX9SCyU6MJwr8tuQrC\\\\\\\/bYmAGqJ8c1Qksf5nvpf4bdZlnRm4WSGiCoPIemAVqXcnNdhzhPqun4Ia8JShJE\\\\\\\/gusaf37Bzi7wqjUmDtqgHke6ESoRfbsKDV+908RqKVNrY68hevwRy4G\\\\\\\/vewIxFUytXWHnwZfg66E94OYV1tDJcoP7iYOIBajN8myvIpBt6nHOY3R2ZbNJhuZhWHdIzi5hSEZe6+jw3CfVq6bv1LGGwLj7nFG2JhK4cTICIstHdLro0CROiPoaVIuKfECIAHtdeHEglZUU16A4BCCK\\\\\\\/tVUc7DaqsjxXhjGYvAzMp3wIIpvzaPtwy\\\\\\\/PyKXpk3lnOeL1uzzhgnaU2sHYy1Y07Grgk7lzFR2Mr0gu+AAI0pPqA6BPvlm31QPGYHwAFisT7uWblheu8p8nJDKXzFNX4rjYWUBF9VgCIINonditdz7pfSZZfrvy443dl4UurTbwLk9\\\\\\\/6\\\\\\\/ElE8LuP+amShHBTszD6sgNgGJnXCmdX6eRLIDbXqRN0H1E0Ki\\\\\\\/1ZO65t849yr4Y4K6Y7L69kbfUiKxmNhMEYqxfGvo7h6FjxNy6mj4o9KYuCqU2y4LzH0XN3cGa6B76CGm724FO+JDrWb8fDcl5VpCi39qIzTd8S2AfLfT8MztijELIPgAUvqqAvDoFGRGHmX9H5UA4590sWepRDmjgjmgVF72WClGSRZ2m79A8s5YyEGcV4ERDw32LlqYblwXDE2h90ckSj2HjsvHZQ02ZQ2F1e+XIbn5LxSgJJrqWi9vR9PETJllUaUPj6HxGBmm1jzB3j+0ZEJrg5NbWIPm+k51NqlzHZtSmHMk78IO\\\\\\\/CpguQ21P4ejAGgUPNoMK2oJk\\\\\\\/u9eQ6Hq2zPLuAOnMA8Cx8U\\\\\\\/+6Mf8a1PwDeiYomSz1pC+KsqVF6aoVBFx94Nfe4bhWKSfx1UKaVjEk49uUAmSTw3YqNTCLi14dbd7Q0VYMPk81bGZ0vrBXuygP+qjR3RZWK+Oz86G4\\\\\\\/wLr9Qo79oQaHrftZaDdjG5pkyNm2dmff7JozmC+d1wbuljYeCo\\\\\\\/7c9iMibrcl\\\\\\\/hE6EBVKnn6hS\\\\\\\/jRJnYGeOFA16W4erpOBLRKaALxZhRjEKzW79S+bUyFkgH+QIDw\\\\\\\/iekEH2vti9jhOb2WqeW3YPN+op6qSUdAaJbWLxFR1GAdr7LJaKfEhBtytl+r1sbFUn15cpwrH5T1cSxHDqOkteMnH0zDJx0lkEPU0P9JK6Dlp4WuBSZSpGRBKhmVx3YeZOPJu8a6mYyyi5i1nQo3zo9kGGMHdl0gbCQ3RX+pV1i+GCgBe3a+8lii9ZEB833jBydHP5Swh24b3hDCxj4kvCFUNntCJ8t8f\\\\\\\/Su\\\\\\\/fdrQ5NztThEe3Dq2IVbPHonFdwhW7RyUOk2YacuwK+CU890Cj0CkrNjY\\\\\\\/j19zTWj22V6arlkXo1DX4dP\\\\\\\/KmdcfMYGN+aOEOMKtlmeG1dQrk0jci9vYMpSXEg8\\\\\\\/osrDeEPd3VXATd40QL06HYBTmMufeuXB3SV5lfn+nMdHmDrzEv6fGrfHNqTn\\\\\\\/0XyjGndzzEOSjkXKqQKklV2pQOkR2wCtUVb03RU4Ue\\\\\\\/HG5KJY+PDzBdxkfp6zTls2eIYPCJBodj5yOepU3yUrijhccQF3NGA+xEKHuxpM3a2\\\\\\\/7hvPXU0n6Qg\\\\\\\/xg9V161F\\\\\\\/kOKGbYEOLL65qoloxXHGZl9UX5md3dX1+yyHiJGqFTkg3Ryj\\\\\\\/w0PzkKCtG2ZdMFJ80BJk9yvdOVoybBs9JDt1QiQ6z15VquE1VefUnurUTN+uPfrb\\\\\\\/jvlUvJ7qxHXPyV89U7V0w0lMr5Vi04wkfa0q74IJ1T8X4tpuPnBC41Zu43Y6IWoV+RioGOeH+7\\\\\\\/HQHKavlpnBslr2J6Q2I4wGcdtn6lFVnDrwg1q3itqW9R6AmOZdQRXhksls6mMU4PhiMAL3t4mO1zRMMJakm+ay00V9\\\\\\\/D9Zr3ZZw2FlTgS81JEVmcKaMPrjA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022WgX6wjDMcLHUPpcDWlflfg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c1a8f164e2119603752ef664582af1d3fb15a7466b4ef2215509e80872ef183a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485857 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2461" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TLBaN3v14F0XquLBu2uLZF9-gjPMg8cctAjPDXsBbe0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2461" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-f1Abz5_BPxCnyx20vT18DmfjXIkBFtg5kzqYIGAoYzQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2427" title="A Genetic Screen Reveals an Unexpected Role for Yorkie Signaling in JAK/STAT-Dependent Hematopoietic Malignancies in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2473" title="A High-Density Genetic Linkage Map and QTL Fine Mapping for Body Weight in Crucian Carp (Carassius auratus) Using 2b-RAD Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485857" id="node485857" data-pisa="ggg;7/8/2461" data-pisa-master="ggg;g3.117.043471" data-apath="/ggg/7/8/2461.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Patterns of Genetic Structure and Linkage Disequilibrium in a Large Collection of Pea Germplasm</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mathieu Siol</span>, <span class="highwire-citation-author" data-delta="1">Françoise Jacquin</span>, <span class="highwire-citation-author" data-delta="2">Marianne Chabert-Martinello</span>, <span class="highwire-citation-author" data-delta="3">Petr Smýkal</span>, <span class="highwire-citation-author" data-delta="4">Marie-Christine Le Paslier</span>, <span class="highwire-citation-author" data-delta="5">Grégoire Aubert</span> and <span class="highwire-citation-author" data-delta="6">Judith Burstin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2461-2471;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043471 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485857" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mathieu Siol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BSiol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Siol%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathieu%2BSiol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mathieu.siol@inra.fr" class="" data-icon-position="" data-hide-link-title="0">mathieu.siol@inra.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Françoise Jacquin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fran%C3%A7oise%2BJacquin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jacquin%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFran%25C3%25A7oise%2BJacquin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marianne Chabert-Martinello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marianne%2BChabert-Martinello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chabert-Martinello%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarianne%2BChabert-Martinello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Petr Smýkal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Palacky University, Faculty of Science, Department of Botany, Holice, 783 71 Olomouc, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petr%2BSm%C3%BDkal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sm%C3%BDkal%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetr%2BSm%25C3%25BDkal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marie-Christine Le Paslier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>INRA, US 1279 Etude du Polymorphisme des Génomes Végétaux (EPGV), Centre de Recherche Ile-de-France-Versailles-Grignon, Commissariat à l’énergie atomique (CEA)-Institut de Génomique, Centre national de génotypage (CNG), Université Paris-Saclay, 91000 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Christine%2BLe%2BPaslier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%20Paslier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Christine%2BLe%2BPaslier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Grégoire Aubert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gr%C3%A9goire%2BAubert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aubert%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGr%25C3%25A9goire%2BAubert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Judith Burstin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judith%2BBurstin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burstin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJudith%2BBurstin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/8/2461" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485857" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2461.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485857" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2461.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485857" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2461.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485857" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/8/2461.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485857" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Pea (<em>Pisum sativum</em>, L.) is a major pulse crop used both for animal and human alimentation. Owing to its association with nitrogen-fixing bacteria, it is also a valuable component for low-input cropping systems. To evaluate the genetic diversity and the scale of linkage disequilibrium (LD) decay in pea, we genotyped a collection of 917 accessions, gathering elite cultivars, landraces, and wild relatives using an array of ∼13,000 single nucleotide polymorphisms (SNP). Genetic diversity is broadly distributed across three groups corresponding to wild/landraces peas, winter types, and spring types. At a finer subdivision level, genetic groups relate to local breeding programs and type usage. LD decreases steeply as genetic distance increases. When considering subsets of the data, LD values can be higher, even if the steep decay remains. We looked for genomic regions exhibiting high level of differentiation between wild/landraces, winter, and spring pea, respectively. Two regions on linkage groups 5 and 6 containing 33 SNPs exhibit stronger differentiation between winter and spring peas than would be expected under neutrality. Interestingly, QTL for resistance to cold acclimation and frost resistance have been identified previously in the same regions.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-diversity" class="hw-term hw-article-keyword hw-article-keyword-genetic-diversity" rel="nofollow">genetic diversity</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/fst" class="hw-term hw-article-keyword hw-article-keyword-fst" rel="nofollow">F<sub xmlns:default="http://www.w3.org/1999/xhtml">ST</sub></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/pisum-sativum" class="hw-term hw-article-keyword hw-article-keyword-pisum-sativum" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Pisum sativum</em></a></li></ul><p id="p-3">In crops, patterns of genetic diversity and the extent of linkage disequilibrium (LD) often result from a complex evolutionary history, including domestication bottlenecks, selection of favorable alleles, secondary admixture, or introgression of genetic material from wild relatives into cultivars. Studying these processes in crop species has proved of tremendous interest to evolutionary geneticists and breeders alike (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Vigouroux <em>et al.</em> 2002</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ross-Ibarra <em>et al.</em> 2007</a>). The domestication process of pea (<em>Pisum sativum</em>, L.), although as ancient as 10,000 yr (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zohary and Hopf 2000</a>) is still a matter of debate. A few studies, however, have investigated genetic diversity at the species level, and results tend to indicate a surprisingly high level of genetic diversity in the cultivated gene pool given its highly inbreeding reproductive system (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baranger <em>et al.</em> 2004</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jing <em>et al.</em> 2010</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burstin <em>et al.</em> 2015</a>). This could result from a weak bottleneck at domestication, important diversification after diffusion in Asia, Africa, and Europe, and/or gene flow between wild and cultivated material, none of these being mutually exclusive.</p><p id="p-4">Concerns related to rapid human-induced climatic changes and increasing food demand owing to population growth have rekindled an interest in better characterization of the extant genetic and phenotypic diversity in cultivated plants. There is a wide spectrum of phenotypic diversity in pea, relating to varied agricultural practices and characteristics of the cultivated material (sowing date, usage, etc.). The genomic regions underlying many of these key adaptations are of particular interest, yet only a few have been roughly identified, mainly using QTL mapping approaches. The advent of high-throughput genotyping technologies in pea enables one to look for the footprints of selection using a different approach, scanning the genome to identify unusual regions in terms of diversity, LD, allelic differentiation and other genomic characteristics. These so-called genome scans have been applied in other crops, and represent an interesting complement to more targeted QTL, or association mapping, approaches (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Schmutz <em>et al.</em> 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Siol <em>et al.</em> 2010</a>).</p><p id="p-5">Understanding the patterns of LD in germplasm and breeding collections is also of great relevance for applied genetic studies looking for genomic regions or QTL underlying traits of agronomic interest. In particular, the scale at which LD decays is one of the main factors to consider when evaluating the density of markers necessary to achieve sufficient power in association mapping or genomic selection approaches. This is particularly important in species such as pea, with a very large genome (∼4.45 Gb). While genetic diversity and genetic structure have been investigated on various collections in pea, examination of LD patterns have been conspicuously scarce to date (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jing <em>et al.</em> 2007</a>). With the recent increased availability of markers across the genome (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tayeh <em>et al.</em> 2015a</a>), it is now possible to assess how genetic diversity is distributed at the species level, as well as LD patterns with increased power.</p><p id="p-6">The present study aimed to (i) describe the diversity patterns in a collection of germplasm representing elite cultivars, landraces, and wild peas; (ii) assess LD decay in different panels, and try to correlate it with the genetic characteristics of the panels; and (iii) identify genomic regions that might have been under selection either during domestication or postdomestication.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-7">A total of 917 <em>Pisum</em> accessions was selected from various collections. Overall, the goal was to represent the full spectrum of genetic variability from wild peas (<em>Pisum fulvum</em> and <em>P. sativum elatius</em>) and old landraces to modern elite cultivars, and also spanning the usage spectrum (winter <em>vs.</em> spring sowing types, fodder, field, and garden peas). Other genetic stocks and reference lines were included, in particular accessions exhibiting some resistance to <em>Aphanomyces euteiches</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Desgroux <em>et al.</em> 2016</a>). A number of subsets have been defined in the current total sample: (i) the first [hereafter called INRA (Institut National de la Recherche Agronomique) reference collection] contains 342 genotypes from the 372-accession pea genetic resource collection described in <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Burstin <em>et al.</em> (2015)</a>, and is reasonably representative of the whole sample; (ii) the second consists of 189 genotypes representing the history of winter pea breeding programs (called Winter pea breeding) gathered by A. Baranger (UMR IGEPP Rennes); (iii) a set of 176 genotypes derived from crosses between garden pea from the United States, and pea showing good levels of resistance to <em>A. euteiches</em> (Aphanomyces breeding) provided by M.L. Pilet-Nayel (UMR IGEPP Rennes); (iv) a set (called Elite cultivars) of 97 elite cultivars from Europe, Canada. and the US; and (v) a set of 57 genotypes representing mostly wild material and old landraces from the Middle-East (called Wild/landrace) provided by P. Smýkal (University of Olomouć, Czech Republic). <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows the overlap between these subsets. Even though the global sample is heterogeneous in nature, we aimed primarily at identifying patterns of genetic similarity and redundancy within the sample. This was done using a high marker density and a method that is essentially indifferent regarding the particulars of the genetic model that gave rise to the data (see <em>Genetic structure and MAF distribution</em>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagram showing the number of accessions of each of the five panels (Elite cultivar, INRA reference collection, Winter Pea Breeding, Wild/landrace, and Aphanomyces Breeding). Figure obtained using http://bioinformatics.psb.ugent.be/webtools/Venn/." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513231932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Venn diagram showing the number of accessions of each of the five panels (Elite cultivar, INRA reference collection, Winter Pea Breeding, Wild/landrace, and Aphanomyces Breeding). Figure obtained using http://bioinformatics.psb.ugent.be/webtools/Venn/.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2461/F1.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/8/2461/F1.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486167" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Venn diagram showing the number of accessions of each of the five panels (Elite cultivar, INRA reference collection, Winter Pea Breeding, Wild/landrace, and Aphanomyces Breeding). Figure obtained using <a href="http://bioinformatics.psb.ugent.be/webtools/Venn/">http://bioinformatics.psb.ugent.be/webtools/Venn/</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Genotyping</h3><p id="p-9">The total sample was genotyped using the newly developed custom Infinium BeadChip (Illumina, San Diego, CA) Genopea 13.2K SNP Array as described in <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Tayeh <em>et al.</em> (2015a)</a>. This SNP array has been defined so as to include a maximum of mapped SNP. This generated a bias toward SNPs that were polymorphic among parents of <em>P. sativum</em> recombinant inbred line populations (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Tayeh <em>et al.</em> 2015a</a>). Data were analyzed using the Genotyping Module v1.9.4 of Illumina’s GenomeStudio software version 2011.1 (<a href="http://support.illumina.com/array/array_software/genomestudio.ilmn">http://support.illumina.com/array/array_software/genomestudio.ilmn</a>). When necessary, GenoPlots were edited manually so that three genotype clusters AA, AB, and BB could be obtained. Filters were applied to discard SNPs with missing rate ≥0.2, and/or heterozygosity ≥0.05, in further analyses.</p><p id="p-10">For genetic structure analyses, SNPs with minor allele frequencies &lt;0.02 were discarded, while, for LD analyses, the minor allele frequency (MAF) threshold was 0.05. Only the very best markers were kept (11,142), essentially those with a high-quality GenoPlots score and a high mapping quality on individual and consensus maps of biparental populations as described in <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Tayeh <em>et al.</em> (2015a)</a>.</p></div><div id="sec-4" class="subsection"><h3>Genetic structure and MAF distribution</h3><p id="p-11">To investigate the genetic structure of our material, we used two methods: the first, Discriminant Analysis on Principal components (DAPC), implemented in the Adegenet R package (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jombart <em>et al.</em> 2010</a>), is nonparametric, since it does not rely on any assumption regarding the genetic make-up of the data. By contrast, the second (fastStructure) uses an approach similar to the widely used program Structure, but tailored to deal efficiently with dataset containing several tens of thousands of markers through a variational Bayesian framework (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Raj <em>et al.</em> 2014</a>). It postulates a population genetic model, as do various other popular approaches, such as STRUCTURE or InStruct (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gao <em>et al.</em> 2007</a>). The comparison of the results obtained with these two methods allowed us to evaluate the impact of violating the assumptions made by fastStructure.</p><p id="p-12">DAPC was run without prior knowledge of groups, and the optimal number of clusters was thus assessed through sequential k-means and model selection using the Bayesian Information Criterion. The number of principal components was determined to be 13 through maximization of the α-score (measuring the difference between the proportion of successful reassignment of the analysis and values obtained using random groups), and the number of discriminant axes was set to six for a proportion of explained variance of 35.7%.</p><p id="p-13">FastStructure was run for a number of clusters (<em>K</em>) ranging from 1 to 20 with five replicates for each value of <em>K</em> and using the “simple prior” option (flat beta-prior over allele frequencies). To evaluate the repeatability across runs, and rule out for true multimodality (as opposed to cluster labels switching), we ran the program CLUMPP v.1.1.2 using the Greedy algorithm (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jakobsson and Rosenberg 2007</a>). The putative optimal number of clusters was assessed from the likelihood profile and by confronting the results with the DAPC analysis, and admixture plots were obtained using a custom python script.</p><p id="p-14">Using the same genotypic data, we also calculated a simple distance matrix between individuals by counting the number of different alleles over the total number of alleles, accounting for missing data. This distance matrix was used to create a hierarchical tree with the Ward clustering algorithm with the hclust function in <em>R</em>. The tree was edited and displayed using the Interactive Tree of Life website (<a href="http://itol.embl.de/">http://itol.embl.de/</a>). Finally, we also examined the MAF distribution overall, and in different sets identified through the structure analysis above (namely wild/landraces, spring peas, and winter peas), as well as the joint distribution of allele frequencies between pairs of such ensembles.</p></div><div id="sec-5" class="subsection"><h3>LD and kinship comparisons</h3><p id="p-15">For all LD analyses, SNPs were ordered according to the consensus genetic map produced by <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Tayeh <em>et al.</em> (2015a)</a>. LD <em>r</em><sup>2</sup> values were calculated between SNPs located on the same linkage groups and plotted against pairwise genetic distance using custom python scripts and functions from an expanded version of the EggLib package (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">De Mita and Siol 2012</a>). More specifically, the mean and quantiles of the distribution of <em>r</em><sup>2</sup> values were calculated and plotted, both for all the linkage groups pooled together and by linkage group. We also plotted the <em>r</em><sup>2</sup> values as a function of the physical distance between SNPs found on the same genomic scaffolds, and for which a physical distance could be calculated.</p><p id="p-16">To evaluate the magnitude of the effect of sample structure on the value of LD values, we computed the <em>r</em><sup>2</sup><em><sub>V</sub></em> value (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mangin <em>et al.</em> 2012</a>), which estimates the correlation in allele frequencies correcting for the effect of relatedness by using a kinship matrix with the R package LDcorSV. The kinship matrix was estimated as the cross-product of the genotype matrix (with genotypes centered and standardized) in <em>R</em>. A heatmap of raw and corrected <em>r</em><sup>2</sup> values was obtained using the LDheatmap package in <em>R</em>. To allow for comparison in the level of kinship between different subsets, submatrices were extracted from the global kinship matrix. Since the kinship calculated in this way is dependent on allele frequencies, we also computed the identity-by-state (IBS) as described in <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rincent <em>et al.</em> (2012)</a>, which measures the genetic similarity without regard to allele frequencies.</p></div><div id="sec-6" class="subsection"><h3>F<sub>ST</sub> scan</h3><p id="p-17">Following our analysis of genetic structure, we wanted to investigate which regions of the genome were the most divergent between wild, winter, and spring peas to pinpoint potentially interesting candidate genes. To do so, we computed both F<sub>ST</sub> and Jost’s D genome-wide on an SNP basis, and plotted the values obtained ordering them using the genetic map. F<sub>ST</sub> measures the proportion of the variance in allele frequencies attributable to variation between populations (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Charlesworth and Charlesworth 2010</a>), and has a long history of being used as a proxy for the level of differentiation between populations in population genetics. It has been noted, however, that its value is constrained by the level of heterozygosity of the marker used, and other measures have been proposed, such as <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jost (2008)</a>. This was implemented by custom Python scripts using the revised version of the EggLib Package (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">De Mita and Siol 2012</a>). The same was done with Nei’s heterozygosity. To test more formally for outliers in the F<sub>ST</sub> genome scan that could constitute good candidates for loci having undergone selection for local adaptation, we used the BayeScan software (version 2.1). The method use a Bayesian framework to estimate the posterior probability of a locus being under selection by contrasting two alternative model of divergence from a ancestral population, one with selection and one without (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Foll and Gaggiotti 2008</a>). Annotations for the candidate SNPs identified through this approach were obtained from the Pea RNA-Seq gene atlas portal (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alves-Carvalho <em>et al.</em> 2015</a>).</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-18">The genotyping data, the consensus map and the python and <em>R</em> custom scripts to analyze the data are all available in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043471/-/DC1/FileS1.zip">File S1</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Genetic structure</h3><p id="p-19">A collection of 917 <em>Pisum</em> accessions representing a wide diversity was genotyped using the Genopea 13.2K SNP Array. From the initial 13,204 SNPs on the 13.2K chip, we retained information from 11,142 SNPs for genetic structure analyses. Following the analysis with DAPC, and, using the BIC profile, we chose to retain 16 clusters (see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). These clusters tend to be consistent with information regarding the type of material and its use. <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> shows the membership probabilities of each individual. When 16 clusters were considered, the results obtained with fastStructure were very congruent with the results from DAPC (results not shown). Three large ensembles emerge from this analysis: wild peas and landraces from the “Fertile crescent” and Asia (mainly Afghanistan, Nepal, and India, clusters 6 and 15), winter peas (clusters 4, 9, 10, and 12), and spring peas (all remaining clusters).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic structure of the collection of 917 accessions of pea. (A) Scatterplot showing the first two principal components of the DAPC. Wild peas are grouped in clusters&#xA0;6 and 15, winter sown peas are found in clusters&#xA0;4, 9, 10, and 12, and the remaining clusters are spring sown peas. Note the position of the peculiar cluster&#xA0;7, containing very original east Asian peas cultivated in tropical regions. The first axis nicely segregate wild vs. cultivated material, while the second axis spring and winter peas. (B) Membership probability plot of individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513231932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic structure of the collection of 917 accessions of pea. (A) Scatterplot showing the first two principal components of the DAPC. Wild peas are grouped in clusters&#xA0;6 and 15, winter sown peas are found in clusters&#xA0;4, 9, 10, and 12, and the remaining clusters are spring sown peas. Note the position of the peculiar cluster&#xA0;7, containing very original east Asian peas cultivated in tropical regions. The first axis nicely segregate wild &lt;em&gt;vs.&lt;/em&gt; cultivated material, while the second axis spring and winter peas. (B) Membership probability plot of individuals.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2461/F2.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/8/2461/F2.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486133" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Genetic structure of the collection of 917 accessions of pea. (A) Scatterplot showing the first two principal components of the DAPC. Wild peas are grouped in clusters 6 and 15, winter sown peas are found in clusters 4, 9, 10, and 12, and the remaining clusters are spring sown peas. Note the position of the peculiar cluster 7, containing very original east Asian peas cultivated in tropical regions. The first axis nicely segregate wild <em>vs.</em> cultivated material, while the second axis spring and winter peas. (B) Membership probability plot of individuals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Reassuringly, there was an overall good correspondence between the groups as inferred from DAPC and the distance tree (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Groups 4 and 9 mainly represented the history of winter pea selection, whereas groups 10 and 12 were composed of winter peas used mostly as fodder. Groups 14, 8, 3 and 16 were mostly dry seed peas. Two groups (2 and 11) were composed exclusively of genotypes from the Aphanomyces breeding subset (although other genotypes from the subset are not included in these groups). The constitution of two such groups is intriguing, and could be indicative of a technical artifact. One potential reason for the grouping of these accessions together, however, is that many of them are highly related recombinant inbred lines and not typical accessions in the classical sense (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Desgroux <em>et al.</em> 2016</a>). We thus decided to exclude these two groups from subsequent analyses involving spring peas. Groups 1, 5, and 13 were mostly garden peas, both recent and ancient varieties. A few fodder spring peas are also observed in group 13. Finally, group 7 is a very small, yet particular, group with only Far Eastern genotypes from China (see intermediate position between other cultivated peas and wild peas on the first axis of the discriminant analysis in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Global pattern of genetic differentiation. Distance tree based on a custom distance matrix measuring the number of shared alleles over the total number of alleles between two individuals. The tree was constructed using the Ward clustering algorithm. The distances are not represented to scale on the tree so as to make the dendrogram more readable. Red, wild/landrace; Blue, winter; Green, spring." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513231932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Global pattern of genetic differentiation. Distance tree based on a custom distance matrix measuring the number of shared alleles over the total number of alleles between two individuals. The tree was constructed using the Ward clustering algorithm. The distances are not represented to scale on the tree so as to make the dendrogram more readable. Red, wild/landrace; Blue, winter; Green, spring.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2461/F3.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/8/2461/F3.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485967" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Global pattern of genetic differentiation. Distance tree based on a custom distance matrix measuring the number of shared alleles over the total number of alleles between two individuals. The tree was constructed using the Ward clustering algorithm. The distances are not represented to scale on the tree so as to make the dendrogram more readable. Red, wild/landrace; Blue, winter; Green, spring.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Overall, the proportion of admixed individual, and the level of admixture, is lower when using the DAPC method than with fastStructure, as seen in other studies (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Burstin <em>et al.</em> 2015</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Roullier <em>et al.</em> 2013</a>). Using the groups as defined by DAPC, the mean F<sub>ST</sub> value (omitting groups 2 and 11) is 0.298.</p><p id="p-24">As expected, the genotypes from the INRA reference subset are spread among the inferred groups, consistent with the fact that this collection has been built in order to best represent the diversity at the species level. There is, however, an imbalance in favor of group 1 (garden spring peas and fodder spring peas). The Aphanomyces breeding subset is strongly imbalanced, with most of its accessions found in either group 2, 11, or 16 (140 genotypes out of 176). As stressed before, groups 2 and 11 contain almost exclusively related genotypes from this subset. Accessions from the Winter pea breeding subset are scattered across different groups (with the exception of groups 2, 11, 15, and 16), but with a majority assigned to winter groups (4, 9, 10, and 12). Accessions from the Elite cultivar panel are predominantly dry seeds peas (group 8 and 14). Finally, and as expected, the accessions from the Wild/landrace panel are found almost exclusively in the two groups gathering the wild peas and old landraces of Middle-Eastern origin (groups 6 and 15). Overall, these groups are also found on the distance tree, although a few get mixed (4 and 9, 1 and 5, and 16, spread between different clusters of the tree).</p></div><div id="sec-10" class="subsection"><h3>MAF distributions and kinship</h3><p id="p-25"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the MAF distribution for the total dataset, spring peas, winter peas, and wild/landraces peas—the three groups being defined through the previous analysis. The shape of the distribution on the total dataset results from the interplay of the allele frequencies in each subset. A striking feature of this distribution is the paucity of SNPs exhibiting very low minor allele frequency (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). It is interesting to note the differences in MAF distribution in each subset; in particular, the wild peas MAF distribution has a very different shape than that of cultivated peas, with a lot more SNPs with rare alleles, and a relative paucity of intermediate frequency alleles.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histograms of MAF in the full dataset (total), spring, winter, and wild/landrace peas as identified by the DAPC analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513231932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Histograms of MAF in the full dataset (total), spring, winter, and wild/landrace peas as identified by the DAPC analysis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2461/F4.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/8/2461/F4.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486011" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">Histograms of MAF in the full dataset (total), spring, winter, and wild/landrace peas as identified by the DAPC analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows the joint allele distribution in the following pairs: winter–spring, winter–wild/landraces, and spring–wild/landraces. Allele frequencies are a lot more correlated between winter and spring peas than between either of them and wild/landraces peas; <em>i.e.</em>, alleles that are, say, at a low frequency in wild peas are less likely to be also at low frequency in cultivated peas. The distribution observed in wild peas is more akin to what is observed under a classic mutation/drift equilibrium model. There is probably a strong effect of the ascertainment scheme.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Joint allele frequency densities between wild and spring peas (A), wild/landrace and winter peas (B), and spring and winter peas (C). The color scale shows the minor allele frequency (MAF) counts (number of SNPs in that class of frequency, 10 equally spaced bins having been used to discretize the allele frequencies)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513231932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Joint allele frequency densities between wild and spring peas (A), wild/landrace and winter peas (B), and spring and winter peas (C). The color scale shows the minor allele frequency (MAF) counts (number of SNPs in that class of frequency, 10 equally spaced bins having been used to discretize the allele frequencies).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2461/F5.medium.gif" width="162" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/8/2461/F5.medium.gif" width="162" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486140" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-28" class="first-child">Joint allele frequency densities between wild and spring peas (A), wild/landrace and winter peas (B), and spring and winter peas (C). The color scale shows the minor allele frequency (MAF) counts (number of SNPs in that class of frequency, 10 equally spaced bins having been used to discretize the allele frequencies).</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">In our data, probably due to the ascertainment bias following the SNP selection for the Genopea 13.2K array (see <em>Materials and Methods</em>), wild peas appeared less polymorphic than cultivated peas with this set of markers. For example, the 17 accessions of <em>P. fulvum</em> are only polymorphic for 1764 SNPs (over the ∼11,000 used in this study). While many more markers were polymorphic in the accessions indicated as <em>P. s. elatius</em> (9172 markers), their relative allele frequencies differed drastically from those of cultivated pea.</p><p id="p-30"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows an IBS network for the accessions belonging to the wild/landrace peas. Only links exhibiting an IBS &gt;0.8 between any two accessions are shown. When using all SNPs, two very conspicuous groups of accessions showing high IBS are present. Interestingly, all <em>P. fulvum</em> accessions fall within the same group, while <em>P. s. elatius</em> are spread more evenly, indicating more genetic heterogeneity in the latter group. The second group gathers mainly <em>P. sativum</em> accessions from Afghanistan, India, and Nepal (roughly corresponding to cluster 6 in the DAPC analysis). Using the same accessions, but calculating IBS using only SNPs polymorphic in <em>P. fulvulm</em>, we observe that <em>P. fulvum</em> accessions are more spread out, while the group gathering Asian <em>sativum</em> is essentially unchanged.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="IBS networks of individuals belonging to the wild/landrace group. Only links showing an IBS &gt;0.8 are shown on the network. (A) Network obtained with all the SNPs. (B) Network obtained using only SNPs polymorphic in P. fulvum (n&#xA0;=&#xA0;1764). P. fulvum are indicated in green, P. s. elatius are indicated in blue on both networks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513231932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;IBS networks of individuals belonging to the wild/landrace group. Only links showing an IBS &gt;0.8 are shown on the network. (A) Network obtained with all the SNPs. (B) Network obtained using only SNPs polymorphic in &lt;em&gt;P. fulvum&lt;/em&gt; (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1764). &lt;em&gt;P. fulvum&lt;/em&gt; are indicated in green, &lt;em&gt;P. s. elatius&lt;/em&gt; are indicated in blue on both networks.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2461/F6.medium.gif" width="440" height="212"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/8/2461/F6.medium.gif" width="440" height="212"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486161" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-31" class="first-child">IBS networks of individuals belonging to the wild/landrace group. Only links showing an IBS &gt;0.8 are shown on the network. (A) Network obtained with all the SNPs. (B) Network obtained using only SNPs polymorphic in <em>P. fulvum</em> (<em>n</em> = 1764). <em>P. fulvum</em> are indicated in green, <em>P. s. elatius</em> are indicated in blue on both networks.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>LD patterns</h3><p id="p-32">We investigated the patterns of LD decay in the genome using the classic <em>r</em><sup>2</sup> estimator as a function of the genetic distance. Considering the whole dataset, there is a steep decay in the LD values as the genetic distance increase; for example, the median value does not exceed 0.05 at a distance of 5 cM. <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> shows that LD patterns are also strongly dependent on the sample considered; for example, while LD is low and decays steeply in spring peas, the trend is essentially similar in winter peas, but with qualitatively higher <em>r</em><sup>2</sup> values, and there is almost no LD in wild peas. This dependence upon the sample can also be observed by contrasting the panels available; for example, in the Elite cultivar subset, comprising mainly dry seeds pea from North America and Europe, the level of LD is substantially higher than in other subsets, although still decreasing rapidly (data not shown). This could be due to a history of higher levels of genetic drift, since the genetic basis in this sample is narrower.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Patterns of intrachromosomal LD decay as a function of genetic distance over all linkage groups in (A) the spring (B) winter, and (C) wild/landrace groups. Each quantile from 5 to 95% around the median are represented. The genetic distances come from Tayeh et al. (2015a)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513231932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Patterns of intrachromosomal LD decay as a function of genetic distance over all linkage groups in (A) the spring (B) winter, and (C) wild/landrace groups. Each quantile from 5 to 95% around the median are represented. The genetic distances come from Tayeh &lt;em&gt;et al.&lt;/em&gt; (2015a).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2461/F7.medium.gif" width="179" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/8/2461/F7.medium.gif" width="179" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485901" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-33" class="first-child">Patterns of intrachromosomal LD decay as a function of genetic distance over all linkage groups in (A) the spring (B) winter, and (C) wild/landrace groups. Each quantile from 5 to 95% around the median are represented. The genetic distances come from <a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Tayeh <em>et al.</em> (2015a)</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Population and/or kinship structure are known to generate long-range LD. To examine this in our dataset, we calculated the <em>r</em><sup>2</sup><em><sub>V</sub></em> estimator (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Mangin <em>et al.</em> 2012</a>), accounting for kinship. Results indicate that some of the LD observed in this dataset indeed come from the underlying genetic/kinship structure (see <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8, A and B</a> for linkage group 6 in the winter group for an example).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heatmap of LD values both before (A) and after (B) correcting for kinship structure. Shown are the values for the sixth linkage group in the winter group. Kinship structure clearly accounts for some LD, especially at long range." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513231932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Heatmap of LD values both before (A) and after (B) correcting for kinship structure. Shown are the values for the sixth linkage group in the winter group. Kinship structure clearly accounts for some LD, especially at long range.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2461/F8.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/8/2461/F8.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485929" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-35" class="first-child">Heatmap of LD values both before (A) and after (B) correcting for kinship structure. Shown are the values for the sixth linkage group in the winter group. Kinship structure clearly accounts for some LD, especially at long range.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Fst scans</h3><p id="p-36">A popular method to look for the signature of selection at the molecular level is to examine the level of differentiation along the genome. The underlying idea is that populations subjected to different environmental pressures might favor traits with different adaptive optima, and that these optima are attained by increasing the frequency of different alleles, respectively. As a result, regions of the genome exhibiting inordinately high levels of differentiation are good candidates for being involved in adaptation (to human needs in the case of a cultivated crop). A first simple way to visualize such regions is to plot the SNP-wise F<sub>ST</sub> values on a Manhattan plot to look for obvious F<sub>ST</sub> peaks. SNPs were ordered according to the consensus genetic map described in <a id="xref-ref-40-7" class="xref-bibr" href="#ref-40">Tayeh <em>et al.</em> (2015a)</a>. <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a> shows a Manhattan plot of F<sub>ST</sub> values between winter and spring peas, wild/landraces and winter peas, and wild/landraces and spring peas. Thereafter, we discuss only the results obtained for the winter/spring comparison, since no genomic regions could be identified that are particularly differentiated between wild and cultivated peas (either spring or winter types)—the F<sub>ST</sub> being high across the whole genome. For the spring/winter comparison, a clear peak of differentiation could be observed in the middle of linkage group 6, while most of linkage group 5 showed high F<sub>ST</sub> values. A more rigorous approach to detect significant outliers was undertaken, using the Bayesian framework described in <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Foll and Gaggiotti (2008)</a>. The method uses a model describing a number of subpopulations that evolved in isolation after splitting from an ancestral population. Each subpopulation may have experienced a varying degree of genetic drift, and the goal is then to determine the posterior probability that a particular locus has undergone selection. Using this method, with a threshold <em>q</em>-value set at 0.05 (meaning that, among the SNPs called significant, on average no more than 5% will be false positives), 33 SNPs were detected as exhibiting significantly higher levels of differentiation than expected under neutral divergence from an ancestral population. These SNPs (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) were located only on linkage groups 5 and 6. A one-way ANOVA indicated significant differences between the mean F<sub>ST</sub> values across linkage groups. While F<sub>ST</sub> is often used as a proxy for population differentiation, its value is constrained by the heterozygosity at the marker. We calculated Jost’s D, which is specifically designed to measure differentiation in allele frequencies. The values were generally close to the values of F<sub>ST</sub> as could be expected for SNPs with only two alleles per locus and a maximal value of heterozygosity of 0.5. Interestingly six kinases, along with three zinc-finger transcription factors, were found in these highly differentiated regions.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plot of FST values across the genome contrasting spring vs. winter peas. SNPs are ordered according to Tayeh et al. (2015a)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513231932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plot of F&lt;sub&gt;ST&lt;/sub&gt; values across the genome contrasting spring &lt;em&gt;vs.&lt;/em&gt; winter peas. SNPs are ordered according to Tayeh &lt;em&gt;et al.&lt;/em&gt; (2015a).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2461/F9.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/7/8/2461/F9.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485938" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-37" class="first-child">Manhattan plot of F<sub>ST</sub> values across the genome contrasting spring <em>vs.</em> winter peas. SNPs are ordered according to <a id="xref-ref-40-8" class="xref-bibr" href="#ref-40">Tayeh <em>et al.</em> (2015a)</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485993/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485993/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">List of SNPs detected as significant with BayeScan in a comparison between spring and winter peas</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">Finally, no obvious “dip” in genetic diversity (<em>H</em><sub>e</sub>, Nei’s heterozygosity) could be noted along the genome in either winter peas or spring peas. Such dips have frequently been observed in domesticated species, and can be the result of selective sweeps around loci that have been particularly instrumental in the domestication process (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Clark <em>et al.</em> 2004</a>). Furthermore, the mean <em>H</em><sub>e</sub> value is lower in our wild peas than in both winter and spring peas (0.264 <em>vs.</em> 0.353 and 0.337). In order to make sure that this difference was not a byproduct of the difference in sample sizes, we generated 50 datasets for both the spring and winter peas containing the same number of individuals than the wild peas, and calculated the mean <em>H</em><sub>e</sub> over these 50 datasets. The strong difference remained, showing it was not due to differences in sample size, but more likely to the SNP selection process (see <em>Materials and Methods</em>).</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-40">Genetic diversity as a whole seems quite high in <em>P. sativum</em>, as has already been noted in previous studies (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Baranger <em>et al.</em> 2004</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Jing <em>et al.</em> 2007</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">2010</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Burstin <em>et al.</em> 2015</a>). One of the primary goals of this study was to investigate the genetic diversity and structure patterns in a large collection comprising 917 accessions of field pea and a few relatives using a large array of SNP markers. Despite the heterogeneous nature of this sample, the overall picture provided by our study is very coherent with the biological characteristics of the material and its usage. At the highest level the dataset is divided in three groups: a group gathering all the wild peas/landraces, and two groups of cultivated peas, including all winter peas and most spring peas. These unequivocal groups identified through our study stem from actual genetic relationships between genotypes. The finer subdivision also makes good biological sense, especially when considering the cultivated material and its various usage type (food peas, feed peas, and fodder or garden peas), or the geographical origin, with, for example, clustering of some fodder winter peas from China (in cluster 7).</p><p id="p-41">This type of structure according to cultivated type (winter/spring), end-use (feed, food, or fodder) has already been described in <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Baranger <em>et al.</em> (2004)</a> and <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Burstin <em>et al.</em> (2015)</a>. Similarly, structure according to sowing types (winter/spring) and end-uses has been noted in wheat (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cavanagh <em>et al.</em> 2013</a>) and barley (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Comadran <em>et al.</em> 2012</a>). The overall congruence between the nonparametric (DAPC) and parametric (fastStructure) methods tend to show that those assumptions likely to be violated by the constitution of our heterogeneous sample were not critical for the characterization of genetic structure.</p><p id="p-42">In the present study, the panel used for SNP discovery was of modest size, comprising 16 accessions chosen to span the widest spectrum possible, so as to minimize ascertainment bias. However, the SNP selection criterion of being polymorphic in at least one <em>P. sativum</em> mapping population has introduced a bias in the survey of the diversity of wild accessions where the number of polymorphic markers was lower and the level of resolution achieved poorer. A resampling procedure ensured that this effect was not a side-effect of a lower sample size. Yet, the IBS networks of individuals belonging to the wild/landrace group showed interesting patterns of groupings among this group. Using SNPs that are polymorphic among <em>P. fulvum</em> accessions allowed better visualization of this group’s diversity. Regardless of the true level of genetic diversity in the wild material, the high genetic diversity in the cultivated material could indicate a scenario involving a weak bottleneck at domestication, possibly complemented by the diversity of uses, and the contrasting environmental conditions experienced in pea-producing areas worldwide.</p><p id="p-43">While detecting the genomic regions affected by selection during domestication using this dataset seems difficult, we found a clear signal of strong differentiation on linkage groups 5 and 6 between spring and winter peas, strongly suggesting an effect of postdomestication selection. Interestingly, a QTL involved in frost tolerance is known on linkage group 6, and has been mapped between 49 and 53 cM using a different mapping population and mainly SSR markers (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tayeh <em>et al.</em> 2013</a>). Our distinct F<sub>ST</sub> peak nicely coincides with the position of this QTL. In particular, one of the markers detected under selection is in common between our studies [NT6083 in <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Tayeh <em>et al.</em> (2013)</a>, corresponding to PsCam057485]. A very interesting candidate (PsCam03730) is located in a contig exhibiting great homology with the CBFs genes in <em>Medicago truncatula</em> and <em>M. falcata</em>. As in <em>Arabidopsis thaliana</em>, the CBF genes have been shown to produce key transcriptional activators for cold acclimation (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Pennycooke <em>et al.</em> 2008</a>). Another QTL for frost tolerance has been found on linkage group 5, and mapped between 67.7 and 82.8 cM (again using a different map, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Klein <em>et al.</em> 2014</a>). Looking at the functional annotations, one of the more interesting candidate SNP (PsCam040405) is located on a gene encoding a phosphatidyl ethanolamine-binding protein (PEBP), and is mapped at 59.1 cM on LG5 in the map used for our study. This gene has been shown to be homolog of the <em>A. thaliana FT</em> gene in pea, which promotes flowering under long days (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hecht <em>et al.</em> 2011</a>). In narrow-leafed lupin, another legume species, the loss of vernalization requirement has been shown to be associated with a deletion in the promoter, and a derepressed expression of a <em>FT</em> homolog (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Nelson <em>et al.</em> 2017</a>). Interestingly, the gene VRN-H3 found in barley is also a homolog of <em>FT</em>, and has been shown to be involved in the adaptation to spring growth in cultivated barley (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Comadran <em>et al.</em> 2012</a>).</p><p id="p-44">Our data revealed a steeply decaying LD as a function of genetic distance, with a median <em>r</em><sup>2</sup> value of &lt;0.05 at ∼3 cM on the total dataset, when all linkage groups are pooled together. The trend is the same regardless of the subsample considered, even though the values of LD can differ significantly. For example, LD is higher in the Elite cultivar and the Aphanomyces Breeding panels than in other panels. These differences are to be expected, since <em>r</em><sup>2</sup> strongly depends on allele frequency (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gianola <em>et al.</em> 2013</a>), and, depending on how we subsample the data, the allele frequencies might vary widely. There is, for example, almost no LD when considering wild peas. A deeper evolutionary history with more time for recombination to break down LD could explain this difference with the cultivated material, which has most probably experienced bottlenecks and selection. These two factors are known to influence patterns of LD, along with other forces such as mutation, population structure, and admixture. For comparison in barley, another predominantly selfing crop, LD (<em>r</em><sup>2</sup>) was observed to drop below what they called a basal level (0.2) at a distance of ca. 10–15 cM (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pasam <em>et al.</em> 2012</a>). Comparison between species using genetic distances is, however, fraught with difficulties.</p><p id="p-45">The extent of LD is an important factor to consider in light of the increasing interest in applying genome-wide association methods and genomic selection to pea. Indeed, the chromosomal extent of LD dictates the density of markers necessary to achieve sufficient power to detect associations, and the accuracy with which loci will be mapped. A recent study in one of our laboratories, however, has applied genomic prediction methods to a collection of 339 accessions from the INRA reference collection with satisfying results (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Tayeh <em>et al.</em> 2015b</a>). This study showed that, provided that the markers used were relatively evenly distributed across the linkage groups, even a relatively reduced set of markers (down to &lt;1000) was enough to obtain good prediction accuracy. The same trend has been noted in wheat and barley (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Heffner <em>et al.</em> 2011</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lorenz <em>et al.</em> 2012</a>).</p><p id="p-46">In the above, we plotted LD as a function of genetic distance, and it is therefore difficult to compare with other published reports of LD decay in crop species, where physical distances are available. The genome size of <em>P. sativum</em> is ∼4.45 Gb (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dolezel and Greilhuber 2010</a>). With the consensus map used in this study, this indicates that 1 cM corresponds, on average, to a 5.6 Mb DNA chunks, this of course varying across the genome according to recombination rate. Using the set of SNPs located on identical scaffolds, we draw a first picture of the level of LD as a function of physical distance (see <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a>). The plots obtained are necessarily sparser, since the number of SNP used was a lot smaller than for the previous analysis, but the trend is of a LD decaying on the scale of 200 kb in both winter and spring peas, and 100 kb in wild/landraces peas. Genome-wide LD decay rates have been estimated at ∼123 and ∼167 kb in 14 <em>indica</em> and <em>japonica</em> rice landraces (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Huang <em>et al.</em> 2010</a>), and the long range LD in cultivated rice is estimated at between 100 and 200 kb (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mather <em>et al.</em> 2007</a>). This is often considered as a quite long-ranging LD, and has been linked to the self-fertilizing nature of rice, coupled with a small effective population size. Therefore, considering the genetic distance and physical distance can leave a different impression with our data. Indeed even though LD decays steeply with genetic distance, a centimorgan still spans a big physical distance in pea, and many genes can still be in substantial LD. However, we should wait for the full pseudomolecules to be available in order to reassess the level of LD decay using physical distances with more confidence and draw firmer conclusions.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LD as a function of physical distance between SNPs. (A) spring peas, (B) winter peas, and (C) wild/landrace peas." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513231932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;LD as a function of physical distance between SNPs. (A) spring peas, (B) winter peas, and (C) wild/landrace peas.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2461/F10.medium.gif" width="177" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/7/8/2461/F10.medium.gif" width="177" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2461/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485990" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-47" class="first-child">LD as a function of physical distance between SNPs. (A) spring peas, (B) winter peas, and (C) wild/landrace peas.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">In stark contrast with other well studied crops such as rice, barley, wheat, or maize, the domestication scenario particulars (timing and bottleneck intensity) in pea are not well known, and the taxonomy within the <em>Pisum</em> genus is not fully resolved, with opinions varying with regard to the number of species and their relations (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Vershinin <em>et al.</em> 2003</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Smýkal <em>et al.</em> 2012</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Weeden 2007</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ladizinsky and Abbo 2016</a>). Currently, the preferred hypothesis recognizes three species: <em>P. sativum</em> (including <em>elatius</em> which is thought of as the wild ancestor of cultivated peas), <em>P. fulvum</em>, and <em>P. abyssinicum</em>, which might have been domesticated independently (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Maxted and Ambrose 2001</a>).</p><p id="p-49">A more in-depth investigation of the evolutionary history of the <em>Pisum</em> genus, its domestication, and subsequent breeding should probably use whole-genome resequencing data to alleviate the ascertainment bias issue. A more balanced sample with a larger diversity of wild and old landraces would also be useful. Once a workable domestication model identified, the systematic search for the footprints of the domestication at the molecular level should prove more fruitful.</p></div><div class="section" id="sec-14"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">We thank the Etude du Polymorphisme des Génomes Végétaux (EPGV) group, Dominique Brunel and Christelle Aluome, respectively, for their expertise and work in the construction of the Genopea 13.2K SNP Array, and Aurélie Bérard and Aurélie Chauveau for performing and analyzing the genotyping. We thank Anne Boland, Marie-Thérèse Bihoreau and their staff. We thank Marie-Laure Pilet-Nayel and Alain Baranger. We also thank Commissariat à l’énergie atomique-Institut de Génomique/Centre national de génotypage (CEA-IG/CNG), for supporting the Institut National de la Recherche Agronomique (INRA)-EPGV group for quality control of DNA, and for giving them access to their Illumina Infinium genotyping platform. The research of P.S. is funded by the Grant Agency of the Czech Republic, 16-21053S project.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-51">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043471/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043471/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-52"><em>Communicating editor: A. H. Paterson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-03-15"><span class="received-label">Received </span>March 15, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-05-22"><span class="accepted-label">Accepted </span>May 22, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Siol <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.1" data-doi="10.1111/tpj.12967"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alves-Carvalho</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aubert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrère</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruaud</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brochot</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Full-length <em>de novo</em> assembly of RNA-seq data in pea (<em>Pisum sativum</em> L.) provides a gene expression atlas and gives insights into root nodulation in this species.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D84%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12967%26rft_id%253Dinfo%253Apmid%252F26296678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12967&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26296678&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.2" data-doi="10.1007/s00122-003-1540-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baranger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aubert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnau</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lainé</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deniot</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic diversity within <em>Pisum sativum</em> using protein- and PCR-based markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1309</span>–<span class="cit-lpage">1321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBaranger%26rft.auinit1%253DA.%26rft.volume%253D108%26rft.issue%253D7%26rft.spage%253D1309%26rft.epage%253D1321%26rft.atitle%253DGenetic%2Bdiversity%2Bwithin%2BPisum%2Bsativum%2Busing%2Bprotein-%2Band%2BPCR-based%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1540-5%26rft_id%253Dinfo%253Apmid%252F14727027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1540-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14727027&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220988400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burstin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salloignon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chabert-Martinello</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magnin-Robert</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siol</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic diversity and trait genomic prediction in a pea diversity panel.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.4" data-doi="10.1073/pnas.1217133110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">8057</span>–<span class="cit-lpage">8062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1217133110%26rft_id%253Dinfo%253Apmid%252F23630259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIwLzgwNTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjQ2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2461.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source"><em>Elements of Evolutionary Genetics</em></span>. <span class="cit-publ-name">Roberts and Company publishers</span>, <span class="cit-publ-loc">Greenwood Village, CO</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.6" data-doi="10.1073/pnas.2237049100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linton</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messing</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Pattern of diversity in the genomic region near the domestication gene <em>tb1</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">700</span>–<span class="cit-lpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2237049100%26rft_id%253Dinfo%253Apmid%252F14701910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDEvMy83MDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjQ2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.7" data-doi="10.1038/ng.2447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Natural variation in a homolog of <em>Antirrhinum</em> CENTRORADIALIS contributed to spring growth habit and environmental adaptation in cultivated barley.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1388</span>–<span class="cit-lpage">1392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DComadran%26rft.auinit1%253DJ.%26rft.volume%253D44%26rft.issue%253D12%26rft.spage%253D1388%26rft.epage%253D1392%26rft.atitle%253DNatural%2Bvariation%2Bin%2Ba%2Bhomolog%2Bof%2BAntirrhinum%2BCENTRORADIALIS%2Bcontributed%2Bto%2Bspring%2Bgrowth%2Bhabit%2Band%2Benvironmental%2Badaptation%2Bin%2Bcultivated%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2447%26rft_id%253Dinfo%253Apmid%252F23160098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2447&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23160098&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Mita</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siol</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"> EggLib: processing, analysis and simulation tools for population genetics and genomics.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">13</span>: 27.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desgroux</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anthoëne</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roux-Duparque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivière</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aubert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide association mapping of partial resistance to <em>Aphanomyces euteiches</em> in pea.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">124</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dolezel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greilhuber</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nuclear genome size: are we getting closer?</span> <abbr class="cit-jnl-abbrev">Cytometry A</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytometry.%2BPart%2BA%2B%253A%2B%2Bthe%2Bjournal%2Bof%2Bthe%2BInternational%2BSociety%2Bfor%2BAnalytical%2BCytology%26rft.stitle%253DCytometry%2BA%26rft.aulast%253DDolezel%26rft.auinit1%253DJ.%26rft.volume%253D77%26rft.issue%253D7%26rft.spage%253D635%26rft.epage%253D642%26rft.atitle%253DNuclear%2Bgenome%2Bsize%253A%2Bare%2Bwe%2Bgetting%2Bcloser%253F%26rft_id%253Dinfo%253Apmid%252F20583277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20583277&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaggiotti</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: a Bayesian perspective.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">977</span>–<span class="cit-lpage">993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D190%26rft.spage%253D977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.12" data-doi="10.1534/genetics.107.072371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A Markov chain Monte Carlo approach for joint inference of population structure and inbreeding rates from multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1635</span>–<span class="cit-lpage">1651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.072371%26rft_id%253Dinfo%253Apmid%252F17483417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8zLzE2MzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjQ2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qanbari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An evaluation of a novel estimator of linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D111%26rft.spage%253D275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.14" data-doi="10.1105/tpc.110.081042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hecht</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vander Schoor</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ridge</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knowles</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Pea GIGAS gene is a FLOWERING LOCUS T homolog necessary for graft-transmissible specification of flowering but not for responsiveness to photoperiod.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.081042%26rft_id%253Dinfo%253Apmid%252F21282524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjIzLzEvMTQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI0NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D65%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.16" data-doi="10.1038/ng.695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association studies of 14 agronomic traits in rice landraces.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D42%26rft.issue%253D11%26rft.spage%253D961%26rft.epage%253D967%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bof%2B14%2Bagronomic%2Btraits%2Bin%2Brice%2Blandraces.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.695%26rft_id%253Dinfo%253Apmid%252F20972439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20972439&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283540500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.17" data-doi="10.1093/bioinformatics/btm233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm233%26rft_id%253Dinfo%253Apmid%252F17485429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17485429&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249248300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.18" data-doi="10.1534/genetics.107.081323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jing</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seres</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiss</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambrose</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gene-based sequence diversity analysis of field pea (<em>Pisum</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2263</span>–<span class="cit-lpage">2275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJing%26rft.auinit1%253DR.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2263%26rft.epage%253D2275%26rft.atitle%253DGene-Based%2BSequence%2BDiversity%2BAnalysis%2Bof%2BField%2BPea%2B%2528Pisum%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081323%26rft_id%253Dinfo%253Apmid%252F18073431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIyNjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjQ2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.19" data-doi="10.1186/1471-2148-10-44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jing</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vershinin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grzebyta</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smýkal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic diversity and evolution of field pea (<em>Pisum</em>) studied by high throughput retrotransposon based insertion polymorphism (RBIP) marker analysis.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DJing%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.spage%253D44%26rft.epage%253D44%26rft.atitle%253DThe%2Bgenetic%2Bdiversity%2Band%2Bevolution%2Bof%2Bfield%2Bpea%2B%2528Pisum%2529%2Bstudied%2Bby%2Bhigh%2Bthroughput%2Bretrotransposon%2Bbased%2Binsertion%2Bpolymorphism%2B%2528RBIP%2529%2Bmarker%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-10-44%26rft_id%253Dinfo%253Apmid%252F20156342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-10-44&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20156342&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.20" data-doi="10.1186/1471-2156-11-94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devillard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Discriminant analysis of principal components: a new method for the analysis of genetically structured populations.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DJombart%26rft.auinit1%253DT.%26rft.volume%253D11%26rft.spage%253D94%26rft.epage%253D94%26rft.atitle%253DDiscriminant%2Banalysis%2Bof%2Bprincipal%2Bcomponents%253A%2Ba%2Bnew%2Bmethod%2Bfor%2Bthe%2Banalysis%2Bof%2Bgenetically%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-11-94%26rft_id%253Dinfo%253Apmid%252F20950446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-11-94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20950446&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.21" data-doi="10.1111/j.1365-294X.2008.03887.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jost</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">G(ST) and its relatives do not measure differentiation.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4015</span>–<span class="cit-lpage">4026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DJost%26rft.auinit1%253DL.%26rft.volume%253D17%26rft.issue%253D18%26rft.spage%253D4015%26rft.epage%253D4026%26rft.atitle%253DG%2528ST%2529%2Band%2Bits%2Brelatives%2Bdo%2Bnot%2Bmeasure%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03887.x%26rft_id%253Dinfo%253Apmid%252F19238703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03887.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19238703&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259152900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houtin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rond</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marget</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">QTL analysis of frost damage in pea suggests different mechanisms involved in frost tolerance.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1319</span>–<span class="cit-lpage">1330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2461.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ladizinsky</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbo</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source"><em>The Search for Wild Relatives of Cool Season Legumes</em></span>. <span class="cit-publ-name">Springer International Publishing</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.24" data-doi="10.2135/cropsci2011.09.0503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Potential and optimization of genomic selection for <em>Fusarium</em> head blight resistance in six-row barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1609%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.25" data-doi="10.1038/hdy.2011.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mangin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siberchicot</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doligez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">This</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Novel measures of linkage disequilibrium that correct the bias due to population structure and relatedness.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMangin%26rft.auinit1%253DB.%26rft.volume%253D108%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D291%26rft.atitle%253DNovel%2Bmeasures%2Bof%2Blinkage%2Bdisequilibrium%2Bthat%2Bcorrect%2Bthe%2Bbias%2Bdue%2Bto%2Bpopulation%2Bstructure%2Band%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.73%26rft_id%253Dinfo%253Apmid%252F21878986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.73&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878986&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.26" data-doi="10.1534/genetics.107.079616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caicedo</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polato</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The extent of linkage disequilibrium in rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2223</span>–<span class="cit-lpage">2232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.079616%26rft_id%253Dinfo%253Apmid%252F17947413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIyMjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjQ2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2461.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maxted</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambrose</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-source"><em>Plant Genetic Resources of Legumes in the Mediterranean</em></span>, chap 10. <span class="cit-publ-name">Kluwer Academic Publishers</span>, <span class="cit-publ-loc">Dordrecht, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ksiazkiewicz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rychel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Besharat</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The loss of vernalization requirement in narrow-leafed lupin is associated with a deletion in the promoter and de-repressed expression of a <em>Flowering Locus T (FT)</em> homologue.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">213</span>: <span class="cit-fpage">220</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D213%26rft.spage%253D220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.29" data-doi="10.1186/1471-2229-12-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pasam</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haseneyer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association studies for agronomical traits in a world wide spring barley collection.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DPasam%26rft.auinit1%253DR.%2BK.%26rft.volume%253D12%26rft.spage%253D16%26rft.epage%253D16%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bfor%2Bagronomical%2Btraits%2Bin%2Ba%2Bworld%2Bwide%2Bspring%2Bbarley%2Bcollection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-12-16%26rft_id%253Dinfo%253Apmid%252F22284310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-12-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22284310&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.30" data-doi="10.1104/pp.107.108779"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pennycooke</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stockinger</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparative genomic sequence and expression analyses of <em>Medicago truncatula</em> and Alfalfa subspecies falcata COLD-ACCLIMATION-SPECIFIC genes.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">1242</span>–<span class="cit-lpage">1256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.107.108779%26rft_id%253Dinfo%253Apmid%252F18218976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQ2LzMvMTI0MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNDYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.31" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI0NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.32" data-doi="10.1534/genetics.114.164350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">fastSTRUCTURE: variational inference of population structure in large SNP data sets.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164350%26rft_id%253Dinfo%253Apmid%252F24700103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzIvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI0NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.33" data-doi="10.1534/genetics.112.141473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laloë</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (<em>Zea mays</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141473%26rft_id%253Dinfo%253Apmid%252F22865733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI0NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.34" data-doi="10.1073/pnas.0700643104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Plant domestication, a unique opportunity to identify the genetic basis of adaptation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>(<span class="cit-supplement">Suppl. 1</span>):<span class="cit-fpage">8641</span>–<span class="cit-lpage">8648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0700643104%26rft_id%253Dinfo%253Apmid%252F17494757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxNjoiMTA0L3N1cHBsXzEvODY0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNDYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.35" data-doi="10.1073/pnas.1211049110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roullier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benoit</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKey</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lebot</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Historical collections reveal patterns of diffusion of sweet potato in Oceania obscured by modern plant movements and recombination.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">2205</span>–<span class="cit-lpage">2210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1211049110%26rft_id%253Dinfo%253Apmid%252F23341603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEwLzYvMjIwNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNDYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClean</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mamidi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A reference genome for common bean and genome-wide analysis of dual domestications.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.37" data-doi="10.1111/j.1469-8137.2010.03401.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siol</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">S. C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The population genomics of plant adaptation.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSiol%26rft.auinit1%253DM.%26rft.volume%253D188%26rft.issue%253D2%26rft.spage%253D313%26rft.epage%253D332%26rft.atitle%253DThe%2Bpopulation%2Bgenomics%2Bof%2Bplant%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03401.x%26rft_id%253Dinfo%253Apmid%252F20696011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03401.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20696011&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282225000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.38" data-doi="10.3390/agronomy2020074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smýkal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aubert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burstin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">N. T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Pea (<em>Pisum sativum</em> L.) in the genomic era.</span> <abbr class="cit-jnl-abbrev">Agronomy</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">74</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%26rft.volume%253D2%26rft.spage%253D74%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fagronomy2020074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/agronomy2020074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tayeh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barhman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devaux</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bluteau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prosperi</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A high-density genetic map of the <em>Medicago truncatula</em> major freezing tolerance QTL on chromosome 6 reveals colinearity with a QTL related to freezing damage on <em>Pisum sativum</em> linkage group VI.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D32%26rft.spage%253D279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tayeh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aluome</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Development of two major resources for pea genomics: the GenoPea 13.2K SNP Array and a high density, high-resolution consensus genetic map.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">1257</span>–<span class="cit-lpage">1273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D84%26rft.spage%253D1257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tayeh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Paslier</span>  <span class="cit-name-given-names">M.C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houtin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic prediction in pea: effect of marker density and training population size and composition on accuracy of prediction.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">941</span>–<span class="cit-lpage">951</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D941%26rft_id%253Dinfo%253Apmid%252F26635819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26635819&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.42" data-doi="10.1093/molbev/msg220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vershinin</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allnutt</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knox</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambrose</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">N.T.H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Transposable elements reveal the impact of introgression, rather than transposition, in <em>Pisum</em> diversity, evolution, and domestication.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2067</span>–<span class="cit-lpage">2075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg220%26rft_id%253Dinfo%253Apmid%252F12949152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msg220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12949152&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187969400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.43" data-doi="10.1073/pnas.112324299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vigouroux</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hittinger</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houchins</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Identifying genes of agronomic importance in maize by screening microsatellites for evidence of selection during domestication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">9650</span>–<span class="cit-lpage">9655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.112324299%26rft_id%253Dinfo%253Apmid%252F12105270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTkvMTUvOTY1MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNDYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2461.44" data-doi="10.1093/aob/mcm122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weeden</span>  <span class="cit-name-given-names">N. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic changes accompanying the domestication of <em>Pisum sativum</em>: is there a common genetic basis to the ‘domestication syndrome’ for legumes?</span> <abbr class="cit-jnl-abbrev">Ann. Bot.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">1017</span>–<span class="cit-lpage">1025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcm122%26rft_id%253Dinfo%253Apmid%252F17660515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcm122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17660515&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2461.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zohary</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hopf</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source"><em>Domestication of Plants in the Old World</em></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford, UK</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/8/2461.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2427" title="A Genetic Screen Reveals an Unexpected Role for Yorkie Signaling in JAK/STAT-Dependent Hematopoietic Malignancies in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2473" title="A High-Density Genetic Linkage Map and QTL Fine Mapping for Body Weight in Crucian Carp (Carassius auratus) Using 2b-RAD Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/8"><span>Volume 7 Issue 8, August 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/8.cover-source.jpg?itok=apH_YDuu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2461" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4qGCQH8M0g6KyTe9zsjjV3z0f6OL3O2Zzaae-t-defg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/8/2461" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/8/2461" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/8/2461" class="active" data-icon-position="" data-hide-link-title="0">Patterns of Genetic Structure and Linkage Disequilibrium in a Large Collection of Pea Germplasm</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485857" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-FrvkN_KBM0F8rGx_4sjjH_iCFyAs6QPktPNYQC2uWR8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2461.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/8/2461" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-vOCRmeuGimDK73eACuYYdfjuu5-GPRtXYgrgH6MXMQQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/8/2461" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2461.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485857" id="node485857--2" data-pisa="ggg;7/8/2461" data-pisa-master="ggg;g3.117.043471" data-apath="/ggg/7/8/2461.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Patterns of Genetic Structure and Linkage Disequilibrium in a Large Collection of Pea Germplasm</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mathieu Siol</span>, <span class="highwire-citation-author" data-delta="1">Françoise Jacquin</span>, <span class="highwire-citation-author" data-delta="2">Marianne Chabert-Martinello</span>, <span class="highwire-citation-author" data-delta="3">Petr Smýkal</span>, <span class="highwire-citation-author" data-delta="4">Marie-Christine Le Paslier</span>, <span class="highwire-citation-author" data-delta="5">Grégoire Aubert</span> and <span class="highwire-citation-author" data-delta="6">Judith Burstin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2461-2471;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043471 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485857--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mathieu Siol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BSiol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Siol%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathieu%2BSiol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mathieu.siol@inra.fr" class="" data-icon-position="" data-hide-link-title="0">mathieu.siol@inra.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Françoise Jacquin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fran%C3%A7oise%2BJacquin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jacquin%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFran%25C3%25A7oise%2BJacquin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marianne Chabert-Martinello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marianne%2BChabert-Martinello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chabert-Martinello%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarianne%2BChabert-Martinello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Petr Smýkal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Palacky University, Faculty of Science, Department of Botany, Holice, 783 71 Olomouc, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petr%2BSm%C3%BDkal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sm%C3%BDkal%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetr%2BSm%25C3%25BDkal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marie-Christine Le Paslier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>INRA, US 1279 Etude du Polymorphisme des Génomes Végétaux (EPGV), Centre de Recherche Ile-de-France-Versailles-Grignon, Commissariat à l’énergie atomique (CEA)-Institut de Génomique, Centre national de génotypage (CNG), Université Paris-Saclay, 91000 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Christine%2BLe%2BPaslier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%20Paslier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Christine%2BLe%2BPaslier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Grégoire Aubert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gr%C3%A9goire%2BAubert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aubert%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGr%25C3%25A9goire%2BAubert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Judith Burstin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judith%2BBurstin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burstin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJudith%2BBurstin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/8/2461" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/8/2461&amp;title=Patterns%20of%20Genetic%20Structure%20and%20Linkage%20Disequilibrium%20in%20a%20Large%20Collection%20of%20Pea%20Germplasm" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/8/2461&amp;title=Patterns%20of%20Genetic%20Structure%20and%20Linkage%20Disequilibrium%20in%20a%20Large%20Collection%20of%20Pea%20Germplasm" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/8/2461&amp;title=Patterns%20of%20Genetic%20Structure%20and%20Linkage%20Disequilibrium%20in%20a%20Large%20Collection%20of%20Pea%20Germplasm" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/8/2461&amp;text=Patterns%20of%20Genetic%20Structure%20and%20Linkage%20Disequilibrium%20in%20a%20Large%20Collection%20of%20Pea%20Germplasm" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/8/2461&amp;title=Patterns%20of%20Genetic%20Structure%20and%20Linkage%20Disequilibrium%20in%20a%20Large%20Collection%20of%20Pea%20Germplasm" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/8/2461&amp;t=Patterns%20of%20Genetic%20Structure%20and%20Linkage%20Disequilibrium%20in%20a%20Large%20Collection%20of%20Pea%20Germplasm" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/8/2461&amp;title=Patterns%20of%20Genetic%20Structure%20and%20Linkage%20Disequilibrium%20in%20a%20Large%20Collection%20of%20Pea%20Germplasm" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/8/2461&amp;title=Patterns%20of%20Genetic%20Structure%20and%20Linkage%20Disequilibrium%20in%20a%20Large%20Collection%20of%20Pea%20Germplasm" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485857/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485857" id="node485857--3" data-pisa="ggg;7/8/2461" data-pisa-master="ggg;g3.117.043471" data-apath="/ggg/7/8/2461.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Patterns of Genetic Structure and Linkage Disequilibrium in a Large Collection of Pea Germplasm</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mathieu Siol</span>, <span class="highwire-citation-author" data-delta="1">Françoise Jacquin</span>, <span class="highwire-citation-author" data-delta="2">Marianne Chabert-Martinello</span>, <span class="highwire-citation-author" data-delta="3">Petr Smýkal</span>, <span class="highwire-citation-author" data-delta="4">Marie-Christine Le Paslier</span>, <span class="highwire-citation-author" data-delta="5">Grégoire Aubert</span> and <span class="highwire-citation-author" data-delta="6">Judith Burstin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2461-2471;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043471 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485857--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mathieu Siol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BSiol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Siol%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathieu%2BSiol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mathieu.siol@inra.fr" class="" data-icon-position="" data-hide-link-title="0">mathieu.siol@inra.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Françoise Jacquin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fran%C3%A7oise%2BJacquin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jacquin%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFran%25C3%25A7oise%2BJacquin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marianne Chabert-Martinello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marianne%2BChabert-Martinello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chabert-Martinello%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarianne%2BChabert-Martinello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Petr Smýkal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Palacky University, Faculty of Science, Department of Botany, Holice, 783 71 Olomouc, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petr%2BSm%C3%BDkal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sm%C3%BDkal%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetr%2BSm%25C3%25BDkal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marie-Christine Le Paslier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>INRA, US 1279 Etude du Polymorphisme des Génomes Végétaux (EPGV), Centre de Recherche Ile-de-France-Versailles-Grignon, Commissariat à l’énergie atomique (CEA)-Institut de Génomique, Centre national de génotypage (CNG), Université Paris-Saclay, 91000 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Christine%2BLe%2BPaslier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%20Paslier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Christine%2BLe%2BPaslier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Grégoire Aubert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gr%C3%A9goire%2BAubert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aubert%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGr%25C3%25A9goire%2BAubert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Judith Burstin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), Unité Mixte de Recherche (UMR) 1347, Agroécologie, 21065 Dijon, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judith%2BBurstin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burstin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJudith%2BBurstin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2461&amp;rft.epage=2471&amp;rft.atitle=Patterns%20of%20Genetic%20Structure%20and%20Linkage%20Disequilibrium%20in%20a%20Large%20Collection%20of%20Pea%20Germplasm&amp;rft.volume=7&amp;rft.issue=8&amp;rft.date=2017-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Siol%2C+Mathieu&amp;rft.au=Jacquin%2C+Fran%C3%A7oise&amp;rft.au=Chabert-Martinello%2C+Marianne&amp;rft.au=Sm%C3%BDkal%2C+Petr&amp;rft.au=Le+Paslier%2C+Marie-Christine&amp;rft.au=Aubert%2C+Gr%C3%A9goire&amp;rft.au=Burstin%2C+Judith"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485857/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485857/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485857/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485857/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485857/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485857/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485857/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485857/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485857/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485857/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485857/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485857/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/8/2461" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/8/2461.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/8/2461.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/8/2461.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/8/2461.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
