<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/8/2779" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/8/2779.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/8/2779.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/8/2779.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/8/2779/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/8/2779.atom" />
<meta name="HW.pisa" content="ggg;7/8/2779" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Bayesian Networks Illustrate Genomic and Residual Trait Connections in Maize (Zea mays L.)" />
<meta name="DC.Identifier" content="10.1534/g3.117.044263" />
<meta name="DC.Date" content="2017-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Töpner et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Relationships among traits were investigated on the genomic and residual levels using novel methodology. This included inference on these relationships via Bayesian networks and an assessment of the networks with structural equation models. The methodology employed three steps. First, a Bayesian multiple-trait Gaussian model was fitted to the data to decompose phenotypic values into their genomic and residual components. Second, genomic and residual network structures among traits were learned from estimates of these two components. Network learning was performed using six different algorithmic settings for comparison, of which two were score-based and four were constraint-based approaches. Third, structural equation model analyses ranked the networks in terms of goodness of fit and predictive ability, and compared them with the standard multiple-trait fully recursive network. The methodology was applied to experimental data representing the European heterotic maize pools Dent and Flint ( Zea mays L.). Inferences on genomic and residual trait connections were depicted separately as directed acyclic graphs. These graphs provide information beyond mere pairwise genetic or residual associations between traits, illustrating for example conditional independencies and hinting at potential causal links among traits. Network analysis suggested some genetic correlations as potentially spurious. Genomic and residual networks were compared between Dent and Flint." />
<meta name="DC.Contributor" content="Katrin Töpner" />
<meta name="DC.Contributor" content="Guilherme J. M. Rosa" />
<meta name="DC.Contributor" content="Daniel Gianola" />
<meta name="DC.Contributor" content="Chris-Carolin Schön" />
<meta name="article:published_time" content="2017-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Bayesian Networks Illustrate Genomic and Residual Trait Connections in Maize (Zea mays L.)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Relationships among traits were investigated on the genomic and residual levels using novel methodology. This included inference on these relationships via Bayesian networks and an assessment of the networks with structural equation models. The methodology employed three steps. First, a Bayesian multiple-trait Gaussian model was fitted to the data to decompose phenotypic values into their genomic and residual components. Second, genomic and residual network structures among traits were learned from estimates of these two components. Network learning was performed using six different algorithmic settings for comparison, of which two were score-based and four were constraint-based approaches. Third, structural equation model analyses ranked the networks in terms of goodness of fit and predictive ability, and compared them with the standard multiple-trait fully recursive network. The methodology was applied to experimental data representing the European heterotic maize pools Dent and Flint (&lt;i&gt;Zea mays&lt;/i&gt; L.). Inferences on genomic and residual trait connections were depicted separately as directed acyclic graphs. These graphs provide information beyond mere pairwise genetic or residual associations between traits, illustrating for example conditional independencies and hinting at potential causal links among traits. Network analysis suggested some genetic correlations as potentially spurious. Genomic and residual networks were compared between Dent and Flint.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/08/01" />
<meta name="citation_mjid" content="ggg;7/8/2779" />
<meta name="citation_id" content="7/8/2779" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/8/2779" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/8/2779.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/8/2779.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/8/2779.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.044263" />
<meta name="citation_pmid" content="28637811" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2779" />
<meta name="citation_lastpage" content="2789" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Katrin Töpner" />
<meta name="citation_author_institution" content="Plant Breeding, TUM School of Life Sciences Weihenstephan, Technical University of Munich, 85354 Freising, Germany" />
<meta name="citation_author_institution" content="Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany" />
<meta name="citation_author" content="Guilherme J. M. Rosa" />
<meta name="citation_author_institution" content="Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany" />
<meta name="citation_author_institution" content="Department of Animal Sciences, University of Wisconsin–Madison, Wisconsin 53706" />
<meta name="citation_author" content="Daniel Gianola" />
<meta name="citation_author_institution" content="Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany" />
<meta name="citation_author_institution" content="Department of Animal Sciences, University of Wisconsin–Madison, Wisconsin 53706" />
<meta name="citation_author" content="Chris-Carolin Schön" />
<meta name="citation_author_institution" content="Plant Breeding, TUM School of Life Sciences Weihenstephan, Technical University of Munich, 85354 Freising, Germany" />
<meta name="citation_author_institution" content="Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany" />
<meta name="citation_author_email" content="chris.schoen@tum.de" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=C. F. Aliferis;citation_author=A. Statnikov;citation_author=I. Tsamardinos;citation_author=S. Mani;citation_author=X. D. Koutsoukos;citation_title=Local causal and Markov blanket induction for causal discovery and feature selection for classification part I: algorithms and empirical evaluation.;citation_pages=171-234;citation_volume=11;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=C. F. Aliferis;citation_author=A. Statnikov;citation_author=I. Tsamardinos;citation_author=S. Mani;citation_author=X. D. Koutsoukos;citation_title=Local causal and Markov blanket induction for causal discovery and feature selection for classification part II: analysis and extensions.;citation_pages=235-284;citation_volume=11;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=J. E. Aten;citation_author=T. F. Fuller;citation_author=A. J. Lusis;citation_author=S. Horvath;citation_title=A simple principle concerning the robustness of protein complex activity to changes in gene expression.;citation_pages=1-1;citation_volume=2;citation_year=2008;citation_pmid=18171472;citation_doi=10.1186/1752-0509-2-1" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=E. Bauer;citation_author=M. Falque;citation_author=H. Walter;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Intraspecific variation of recombination rate in maize.;citation_pages=R103;citation_volume=14;citation_year=2013;citation_pmid=24050704;citation_doi=10.1186/gb-2013-14-9-r103" />
<meta name="citation_reference" content="citation_author=D. M. Chickering;citation_pages=87-98;citation_year=1995" />
<meta name="citation_reference" content="citation_author=R. Daly;citation_author=Q. Shen;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. L. de Maturana;citation_author=X.-L. Wu;citation_author=D. Gianola;citation_author=K. A. Weigel;citation_author=G. J. M. Rosa;citation_title=Exploring biological relationships between calving traits in primiparous cattle with a Bayesian recursive model.;citation_volume=181;citation_year=2009;citation_pmid=18984571;citation_doi=10.1534/genetics.108.094888" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=E. L. de Maturana;citation_author=G. de los Campos;citation_author=X.-L. Wu;citation_author=D. Gianola;citation_author=K. A. Weigel;citation_title=Modeling relationships between calving traits: a comparison between standard and recursive mixed models.;citation_pages=1-1;citation_volume=42;citation_year=2010;citation_pmid=20100345;citation_doi=10.1186/1297-9686-42-1" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=D. S. Falconer;citation_title=The problem of environment and selection.;citation_pages=293-298;citation_volume=86;citation_year=1952;citation_doi=10.1086/281736" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Poult. Sci.;citation_author=V. P. S. Felipe;citation_author=M. A. Silva;citation_author=B. D. Valente;citation_author=G. J. M. Rosa;citation_title=Using multiple regression, Bayesian networks and artificial neural networks for prediction of total egg production in European quails based on earlier expressed phenotypes.;citation_volume=94;citation_year=2015;citation_pmid=25713397;citation_doi=10.3382/ps/pev031" />
<meta name="citation_reference" content="citation_journal_title=Trans. R. Soc. Edinb.;citation_author=R. A. Fisher;citation_title=The correlation between relatives on the supposition of Mendelian inheritance.;citation_pages=399-433;citation_volume=52;citation_year=1918;citation_doi=10.1017/S0080456800012163" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. W. Ganal;citation_author=G. Durstewitz;citation_author=A. Polley;citation_author=A. Bérard;citation_author=E. S. Buckler;citation_title=A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.;citation_pages=e28334-e28334;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22174790;citation_doi=10.1371/journal.pone.0028334" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=D. Sorensen;citation_title=Quantitative Genetic Models for Describing Simultaneous and Recursive Relationships Between Phenotypes;citation_pages=1407-1424;citation_volume=167;citation_year=2004;citation_issue=3;citation_pmid=15280252;citation_doi=10.1534/genetics.103.025734" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=G. Guo;citation_author=F. Zhao;citation_author=Y. Wang;citation_author=Y. Zhang;citation_author=L. Du;citation_title=Comparison of single-trait and multiple-trait genomic prediction models.;citation_pages=1-7;citation_volume=15;citation_year=2014;citation_pmid=24387126" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. S. Hageman;citation_author=M. S. Leduc;citation_author=R. Korstanje;citation_author=B. Paigen;citation_author=G. A. Churchill;citation_title=A Bayesian framework for inference of the genotype–phenotype map for segregating populations.;citation_volume=187;citation_year=2011;citation_pmid=21242536;citation_doi=10.1534/genetics.110.123273" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. N. Hazel;citation_title=THE GENETIC BASIS FOR CONSTRUCTING SELECTION INDEXES;citation_pages=476-490;citation_volume=28;citation_year=1943;citation_issue=6;citation_pmid=17247099" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-trait genomic selection methods increase genetic value prediction accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Jiang;citation_author=Q. Zhang;citation_author=L. Ma;citation_author=J. Li;citation_author=Z. Wang;citation_title=Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.;citation_pages=29-36;citation_volume=115;citation_year=2015;citation_pmid=25873147;citation_doi=10.1038/hdy.2015.9" />
<meta name="citation_reference" content="citation_title=Information Theory and Statistics;citation_author=S. Kullback;citation_year=1959" />
<meta name="citation_reference" content="citation_journal_title=Comput. Intell.;citation_author=W. Lam;citation_author=F. Bacchus;citation_title=Learning Bayesian belief networks: an approach based on the MDL principle.;citation_pages=269-293;citation_volume=10;citation_year=1994;citation_doi=10.1111/j.1467-8640.1994.tb00166.x" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Comput. Simul.;citation_author=P. Legendre;citation_title=Comparison of permutation methods for the partial correlation and partial Mantel tests.;citation_pages=37-73;citation_volume=67;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Lehermeier;citation_author=N. Krämer;citation_author=E. Bauer;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Usefulness of Multiparental Populations of Maize (Zea mays L.) for Genome-Based Prediction;citation_pages=3-16;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236445;citation_doi=10.1534/genetics.114.161943" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Lehermeier;citation_author=C.-C. Schön;citation_author=G. de los Campos;citation_title=Assessment of genetic heterogeneity in structured plant populations using multivariate whole-genome regression models.;citation_volume=201;citation_year=2015;citation_pmid=26122758;citation_doi=10.1534/genetics.115.177394" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Li;citation_author=S.-W. Tsaih;citation_author=K. Shockley;citation_author=I. M. Stylianou;citation_author=J. Wergedal;citation_title=Structural model analysis of multiple quantitative traits.;citation_pages=e114-e114;citation_volume=2;citation_year=2006;citation_issue=7;citation_pmid=16848643;citation_doi=10.1371/journal.pgen.0020114" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=R. Maier;citation_author=G. Moser;citation_author=G.-B. Chen;citation_author=S. Ripke;citation_title=Joint analysis of psychiatric disorders increases accuracy of risk prediction for schizophrenia, bipolar disorder, and major depressive disorder.;citation_pages=283-294;citation_volume=96;citation_year=2015;citation_pmid=25640677;citation_doi=10.1016/j.ajhg.2014.12.006" />
<meta name="citation_reference" content="citation_journal_title=J Agric Soc Sci;citation_author=H. N. Malik;citation_author=S. I. Malik;citation_author=M. Hussain;citation_author=S. U. R. Chughtai;citation_author=H. I. Javed;citation_title=Genetic correlation among various quantitative characters in maize (Zea mays L.) hybrids.;citation_pages=262-265;citation_volume=1;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=G. Morota;citation_author=B. D. Valente;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=D. Gianola;citation_title=An assessment of linkage disequilibrium in Holstein cattle using a Bayesian network.;citation_pages=474-487;citation_volume=129;citation_year=2012;citation_issue=6;citation_pmid=23148973" />
<meta name="citation_reference" content="citation_title=Bayesian Networks in R with Applications in Systems Biology;citation_author=R. Nagarajan;citation_author=M. Scutari;citation_author=S. Lèbre;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=A. Nazarian;citation_author=S. A. Gezan;citation_title=GenoMatrix: a software package for pedigree-based and genomic prediction analyses on complex traits.;citation_volume=107;citation_year=2016;citation_pmid=27025440;citation_doi=10.1093/jhered/esw020" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. C. Neto;citation_author=C. T. Ferrara;citation_author=A. D. Attie;citation_author=B. S. Yandell;citation_title=Inferring causal phenotype networks from segregating populations.;citation_volume=179;citation_year=2008;citation_pmid=18505877;citation_doi=10.1534/genetics.107.085167" />
<meta name="citation_reference" content="citation_journal_abbrev=Ann Appl Stat;citation_author=E. C. Neto;citation_author=M. P. Keller;citation_author=A. D. Attie;citation_author=B. S. Yandell;citation_title=CAUSAL GRAPHICAL MODELS IN SYSTEMS GENETICS: A UNIFIED FRAMEWORK FOR JOINT INFERENCE OF CAUSAL NETWORK AND GENETIC ARCHITECTURE FOR CORRELATED PHENOTYPES.;citation_pages=320-339;citation_volume=4;citation_year=2010;citation_issue=1;citation_pmid=21218138;citation_doi=10.1214/09-AOAS288" />
<meta name="citation_reference" content="citation_title=Causality: Models, Reasoning, and Inference;citation_author=J. Pearl;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=BMC Syst. Biol.;citation_author=F. Peñagaricano;citation_author=B. D. Valente;citation_author=J. P. Steibel;citation_author=R. O. Bates;citation_author=C. W. Ernst;citation_title=Exploring causal networks underlying fat deposition and muscularity in pigs through the integration of phenotypic, genotypic and transcriptomic data.;citation_pages=1-9;citation_volume=9;citation_year=2015;citation_pmid=25582171;citation_doi=10.1186/s12918-014-0137-8" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=I. Porth;citation_author=J. Klápště;citation_author=O. Skyba;citation_author=M. C. Friedmann;citation_author=J. Hannemann;citation_title=Network analysis reveals the relationship among wood properties, gene expression levels and genotypes of natural Populus trichocarpa accessions.;citation_pages=727-742;citation_volume=200;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Animal;citation_author=M. Pszczola;citation_author=R. F. Veerkamp;citation_author=Y. de Haas;citation_author=E. Wall;citation_author=T. Strabel;citation_title=Effect of predictor traits on accuracy of genomic breeding values for feed intake based on a limited cow reference population.;citation_pages=1759-1768;citation_volume=7;citation_year=2013;citation_pmid=23915541;citation_doi=10.1017/S175173111300150X" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Automatica;citation_author=J. Rissanen;citation_title=Modeling by shortest data description.;citation_pages=465-471;citation_volume=14;citation_year=1978;citation_doi=10.1016/0005-1098(78)90005-5" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=A. Robertson;citation_title=The sampling variance of the genetic correlation coefficient.;citation_pages=469-485;citation_volume=15;citation_year=1959;citation_doi=10.2307/2527750" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. V. Rockman;citation_title=Reverse engineering the genotype-phenotype map with natural genetic variation.;citation_pages=738-744;citation_volume=456;citation_year=2008;citation_issue=7223;citation_pmid=19079051;citation_doi=10.1038/nature07633" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=D. A. Roff;citation_title=The estimation of genetic correlations from phenotypic correlations: a test of Cheverud’s conjecture.;citation_pages=481-490;citation_volume=74;citation_year=1995;citation_doi=10.1038/hdy.1995.68" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=G. J. M. Rosa;citation_author=B. D. Valente;citation_author=G. de los Campos;citation_author=X.-L. Wu;citation_author=D. Gianola;citation_title=Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.;citation_pages=1-1;citation_volume=43;citation_year=2011;citation_pmid=21208445;citation_doi=10.1186/1297-9686-43-1" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. E. Schadt;citation_author=J. Lamb;citation_author=X. Yang;citation_author=J. Zhu;citation_author=S. Edwards;citation_title=An integrative genomics approach to infer causal associations between gene expression and disease.;citation_pages=710-717;citation_volume=37;citation_year=2005;citation_issue=7;citation_pmid=15965475;citation_doi=10.1038/ng1589" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=M. Scutari;citation_title=PyMC: Bayesian Stochastic Modelling in Python.;citation_pages=1-81;citation_volume=35;citation_year=2010;citation_issue=4;citation_pmid=21603108;citation_doi=10.18637/jss.v035.i03" />
<meta name="citation_reference" content="citation_author=M. Scutari;citation_author=R. Nagarajan;citation_pages=15-27;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Stat. Appl. Genet. Mol. Biol.;citation_author=M. Scutari;citation_author=I. Mackay;citation_author=D. Balding;citation_title=Improving the efficiency of genomic selection.;citation_pages=517-527;citation_volume=12;citation_year=2013;citation_pmid=23934612" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Scutari;citation_author=P. Howell;citation_author=D. J. Balding;citation_author=I. Mackay;citation_title=Multiple Quantitative Trait Analysis Using Bayesian Networks;citation_pages=129-137;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236454;citation_doi=10.1534/genetics.114.165704" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=S. R. Searle;citation_title=Phenotypic, genetic and environmental correlations.;citation_pages=474-480;citation_volume=17;citation_year=1961;citation_doi=10.2307/2527838" />
<meta name="citation_reference" content="citation_title=Numerical Taxonomy. The Principles and Practice of Numerical Classification;citation_author=P. H. A. Sneath;citation_author=R. R. Sokal;citation_year=1973" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Ser. B. Stat. Methodol.;citation_author=D. J. Spiegelhalter;citation_author=N. G. Best;citation_author=B. P. Carlin;citation_author=A. Van Der Linde;citation_title=Bayesian measures of model complexity and fit.;citation_pages=583-639;citation_volume=64;citation_year=2002;citation_doi=10.1111/1467-9868.00353" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=I. Tsamardinos;citation_author=L. E. Brown;citation_author=C. F. Aliferis;citation_title=The max-min hill-climbing Bayesian network structure learning algorithm.;citation_pages=31-78;citation_volume=65;citation_year=2006;citation_doi=10.1007/s10994-006-6889-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. D. Valente;citation_author=G. J. M. Rosa;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=M. A. Silva;citation_title=Searching for recursive causal structures in multivariate quantitative genetics mixed models.;citation_volume=185;citation_year=2010;citation_pmid=20351220;citation_doi=10.1534/genetics.109.112979" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. D. Valente;citation_author=G. J. M. Rosa;citation_author=D. Gianola;citation_author=X.-L. Wu;citation_author=K. Weigel;citation_title=Is structural equation modeling advantageous for the genetic improvement of multiple traits?;citation_volume=194;citation_year=2013;citation_pmid=23608193;citation_doi=10.1534/genetics.113.151209" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. D. Valente;citation_author=G. Morota;citation_author=F. Peñagaricano;citation_author=D. Gianola;citation_author=K. Weigel;citation_title=The causal meaning of genomic predictors and how it affects construction and comparison of genome-enabled selection models.;citation_volume=200;citation_year=2015;citation_pmid=25908318;citation_doi=10.1534/genetics.114.169490" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=A. I. Vázquez;citation_author=D. M. Bates;citation_author=G. J. M. Rosa;citation_author=D. Gianola;citation_author=K. A. Weigel;citation_title=Technical note: an R package for fitting generalized linear mixed models in animal breeding.;citation_volume=88;citation_year=2010;citation_pmid=19820058;citation_doi=10.2527/jas.2009-1952" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Wang;citation_author=F. A. van Eeuwijk;citation_title=A new method to infer causal phenotype networks using QTL and phenotypic information.;citation_pages=e103997;citation_volume=9;citation_year=2014;citation_pmid=25144184;citation_doi=10.1371/journal.pone.0103997" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. J. Winrow;citation_author=D. L. Williams;citation_author=A. Kasarskis;citation_author=J. Millstein;citation_author=A. D. Laposky;citation_title=Uncovering the genetic landscape for multiple sleep-wake traits.;citation_pages=e5161-e5161;citation_volume=4;citation_year=2009;citation_issue=4;citation_pmid=19360106;citation_doi=10.1371/journal.pone.0005161" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Bayesian Networks Illustrate Genomic and Residual Trait Connections in Maize (Zea mays L.)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="twitter:description" content="Relationships among traits were investigated on the genomic and residual levels using novel methodology. This included inference on these relationships via Bayesian networks and an assessment of the networks with structural equation models. The methodology employed three steps. First, a Bayesian multiple-trait Gaussian model was fitted to the data to decompose phenotypic values into their genomic and residual components. Second, genomic and residual network structures among traits were learned from estimates of these two components. Network learning was performed using six different algorithmic settings for comparison, of which two were score-based and four were constraint-based approaches. Third, structural equation model analyses ranked the networks in terms of goodness of fit and predictive ability, and compared them with the standard multiple-trait fully recursive network. The methodology was applied to experimental data representing the European heterotic maize pools Dent and Flint ( Zea mays L.). Inferences on genomic and residual trait connections were depicted separately as directed acyclic graphs. These graphs provide information beyond mere pairwise genetic or residual associations between traits, illustrating for example conditional independencies and hinting at potential causal links among traits. Network analysis suggested some genetic correlations as potentially spurious. Genomic and residual networks were compared between Dent and Flint." />
<meta name="og-title" property="og:title" content="Bayesian Networks Illustrate Genomic and Residual Trait Connections in Maize (Zea mays L.)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/8/2779" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Relationships among traits were investigated on the genomic and residual levels using novel methodology. This included inference on these relationships via Bayesian networks and an assessment of the networks with structural equation models. The methodology employed three steps. First, a Bayesian multiple-trait Gaussian model was fitted to the data to decompose phenotypic values into their genomic and residual components. Second, genomic and residual network structures among traits were learned from estimates of these two components. Network learning was performed using six different algorithmic settings for comparison, of which two were score-based and four were constraint-based approaches. Third, structural equation model analyses ranked the networks in terms of goodness of fit and predictive ability, and compared them with the standard multiple-trait fully recursive network. The methodology was applied to experimental data representing the European heterotic maize pools Dent and Flint ( Zea mays L.). Inferences on genomic and residual trait connections were depicted separately as directed acyclic graphs. These graphs provide information beyond mere pairwise genetic or residual associations between traits, illustrating for example conditional independencies and hinting at potential causal links among traits. Network analysis suggested some genetic correlations as potentially spurious. Genomic and residual networks were compared between Dent and Flint." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
    <title>Bayesian Networks Illustrate Genomic and Residual Trait Connections in Maize (Zea mays L.) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"LbQG6fhuaNn6AhqWUnm9qXoBI-fokMker51kXpDeZ0c","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485855","apath":"\/ggg\/7\/8\/2779.atom","pisa":"ggg;7\/8\/2779","ac":{"\/ggg\/7\/8\/2779.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/8\/2779.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2779"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2779"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/8\/2779":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+MnQYLA2Ym+fu1+dbR2Eegl6y6q0lwQN0hGrlBMbKEpyzJ5x8QXMWGq0bVRnmh7I8vtXIPcQa7Rna8RrP5z+pVnYGU18XOA5RDIMCJc0crHOJcoQijxySiTUW0VZ4fS\\\\\\\/iNGs5RULAZKWgTiKXJSSw4Pn2LHf1YcMrKUV4Q0R2+vELrhWeIXgoQjrah0Stt\\\\\\\/qyrTXIfbYpjCAxOwnrSStjokX8ntY2GNyHoYqYAxrqPH\\\\\\\/b0F2W4EPnR+6kD6rtw6HC6WVn6bzhqNu3sYKRiGSxl\\\\\\\/rVU3y9A8lMkQAzNw5gCh7Zfc\\\\\\\/skL34pYyhId5DLpDDTXesqqxfQDS9tlUHXoySebi23vjjD2ithY9VzgrndfL45Jo47BkjZbiQTkFh2osIFe310sSdp9su+B0ZENCcAQWYcLpHL55ZnxW2Pk2AhVVwRJvLY9+Wketc5mTsGE1EdFqxytbtZ8d+1IX2wpruoQwga7o4hvnEgEjQkKcOTqHU2Uzt8cE7TdwBYHq0xnR5yOe8pZVGrrpkOGcyu0iAVAGUSOhE42I\\\\\\\/NKEX9DWcsKgE9r\\\\\\\/Ipn45YVeBw5Ldf\\\\\\\/sLn2vm\\\\\\\/KEnbG+a1Ucb0DlVrhmBiCrrILSzyjgwllsilBQInPCnzmjO\\\\\\\/\\\\\\\/axfgyOyJ2jeleldm\\\\\\\/jGCu3fXU7vHjrZz1G5Q2Rn0Yi57GkjpA4H\\\\\\\/RypbCJPPLUSByG3NvbLp37msxpSGv4ijG\\\\\\\/gXwot\\\\\\\/r6TK8KypP2JnmkgH8lFW7JcQIsh5Lc+DyvHDn0FOE0QKrmd3tZDCYqkYVO+nTASN6RIH5gzAiGRYtPU5vQev2iGhTw7dVxLVfJVpsjeRNH4BOkTTl3p3aPa61AfoUh5ChPlzKxjgSRh5G6+gyYqKSSTs26f17vthhIufYhaekBzHJhUbaRG2G6VEgaAJfXztStdKy\\\\\\\/oeBlAs+jnDg7fLEKKhtvPGy\\\\\\\/rJ7VFer0yrpQLHf9n2q024Zc7J7FNQcb4pcgtxJ0PQVTQqHbrB7SKQRtBYO4ISyH0G1e45Ghp+ki35mf6q9MlGMSFPbCBAsHR6xg953WzDEddZ1k2QFNGAg2bSOBto41EkWSVoYSZJZdUrm\\\\\\\/I93YqV2sJ9tw94mQMfaeh4zs9ZGln98J04T1CKI4fVS4jMsn96P2hemfnNLn+9w2TKOuqFD4fRfnMgO8RXwxMFuJyyE9noMPRfxYKJ\\\\\\\/MuQAeR3TfCgs1uUQ3Dez+mdgZDLBcW7Di3hLbbWJFQLwTfx9BD\\\\\\\/o\\\\\\\/K08ZvPRuQTnQfbr3\\\\\\\/7bhB5v6kLtizXMLhJX\\\u0022,\\\u0022iv\\\u0022:\\\u0022ud8Eyx7c+WeVsD4JZwVRDQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223d674899fed1b6e68e4bc4d416a600d0528be10a7da1ac1e88496e337c2c6a46\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+GXiRtk1tBQl5UNSBNtANmrXVerihceCUO3AIoIAxzUtZd6NM+Y3D18j0qd9WqRytR6LGNZuUwM4Z0iw0W+dwX1XcMeKHKTx6Bab4mcOW2ziLclFP4EStfuMGZQApjwBsZYcdIkCugosZvUf+HmNPwC4RtnQCo893KAWWgyn8LaljzGpemSqaOfCZO\\\\\\\/m4bNhFKeEMuPNO6nOnboOqLW0blBEmvOVkAtFRJes+OaQiYaD6SXfsx\\\\\\\/Az+dA2wY9jjuMeKgQIfs8YwdW7ZLrq9uZZ13UC6SOJvkxC9sr8iJ0RId2plANFGq6cLME8EXiPgEqKgoETbmi54fsVrPmxcROtCytYepn+zsXdw\\\\\\\/sx8EqQ0zwalRvKFsBIlkAGFd72wTOt40\\\\\\\/vDtYiaxUHI8dgrvFsxDV2lXBxI8o0F1ucNV\\\\\\\/JyU431VWT1xpRgwVOLYpSrt\\\\\\\/nL+f8LF3C4Gntfk585OXKnH+qPUOzksr3RnN8Jh9fGwdRMkjAg4sKfDlIDQspZBV9q14ZEDgxNAiHb+yVQKEv4c+4aJFTjVJiu6BJ9XxnSvgEqwHZ64C576Etr1caEZsq\\\\\\\/XOxlR2ktMlQheYnt+HgPz8DLYDVwSGDb75eOtzAw4JGR+6RzQdvq5HlBoXzintqAiN0LqqsA30UECFPbOf\\\\\\\/adctSh1HTCdM7rOwIiOrlc2LcourT2gJj\\\\\\\/84BHXG9C5YBi+Y1qZPXwx5QOqmtNZ\\\\\\\/LNLRbuhWrMiGNiRACF4qZNycv1Kt+UT9ScsHgmwpcPuaLnFQXWtsBBe6E7hZbmi6+pSG+hXc\\\\\\\/y7M3kFWu3S47CvD2yBomeFtz0Ro31jXRZOpW+L1AMbsi8RaDqH3c\\\\\\\/HhoRnlDaubT5avYlggjX\\\\\\\/xDUOV5TQ2fyFmW+hKioEqQ9msz6oYVzWkhoDBzIZH2BUBeh5Ln7vXuJ9Uq2sZSOhv8hSoYBTMHNktt\\\\\\\/Cawve17N08JGrNQislcNcw5XgLr1ZImWvsvucj\\\\\\\/vvI1VtBUe9ALBW9xc6B7YZDCQlFrdRZY\\\\\\\/XeSYjr\\\\\\\/j7tlzOojlqosqPLlxtC9uNQPqs\\\\\\\/VCtZtmUb0lQIOGX37L1XhiTSw0a6uho2QjRUGm141EYI\\\\\\\/yaZDb0MS8lgpvrIF+eb4Dl2sxxvJs2\\\\\\\/mzjK91dydtI4Fa30\\\\\\\/Gf8hrcBUED\\\\\\\/FneBWEHmX0EzidXIIw81pdPV8+FqZjd6phOBgGy8kZDOu+tKpSQB3mrpYwS0jR4LnspXFguo76iMI3W6+DTwwzL6aN\\\\\\\/I5RnS5kPKstr6zc0KtGS59p\\\\\\\/vmbudHGNUv7iDdpzqJXDKcaO5FU562OezI+3cr1CSxGY0omgbWxjBXAWNAEjhTezfUsOUQDcQ8a8XFQYd6b8P7dWM4OuXaA+tVZhb8uQVkqzwSrX80833ty4B4YSBNJ\\\\\\\/kypc44UuFwVvOHOKJOeyh0IfxOljvh0UXHc3MEmc9pIzkB5FNxN7ojenUlIk6lGTBcFxlFv4dpMB63pBqvcb1GlUh\\\\\\\/lyNc6CK8wCeCTuPPI4szHQVeZaWs1V4Mv+zNSLJTGpIDaPoKbSv17zUIRp2rRVQqwM2tspTc1Z+3OhxNY4t+eAUIHEOkS3EslYJuteQytje4mAd41J3T5bu24GMgtcRHIblEJ60i+nAGYbTGyawUEsUQ2+IXFqnTMMpMzo2uaAPJ9+wOAfGfUSLvqLHXcluGk2IiAIMjtuOipB7uJLNNISBt+1wubXQx+i9Vs7AOluMOSi7xZIr37qjm4vGnC2G9OGnk0PRGTUum3gFCc8eEQErAlXKSO5GOsqZXDv681fTvhevGy2joFNYN2csOGwg+IJZ9IWDoMHQMue6j911M6RovOPhR5D595w3BDRQ3kmXHo5iUPcBRIAH\\\\\\\/w1Zjr1tXaEvBNRot+sOGjYDxaAh3SydAILy\\\\\\\/bxbMrixNBbFYCxvSMyveL64J1LCJ+M4GPvQqnGK0w2wYWpJIyYgOQJgVYknNPqpUl52vXsWHJ6\\\\\\\/qLPoGzl1HbREBrMsifeX0WfKvG0claKyTVRA5\\\\\\\/rPJYyGckEe1BcuaJpJGwcxWDumNZ+kr7YnjwDogfMNll40sIViPMBOGgOjD2xjv6D4nHed6clIgQkZnfNZILBcAvQKq+erm8YP84S2c\\\\\\\/1YR1enQ8yEXX1S7HOHo0xgJhi2logjFZRd1\\\\\\\/pK1ni\\\\\\\/O7+u\\\\\\\/py8q8W6VZydITctNAl2HZ+PBIht+PsoX8BpKzzzf1RpuYOmaSPV0uT0t9EywEU8MnAGEJA4R10f9\\\\\\\/roEr0JkJTYlLdV1jPJA+ZTjiKvpOekQZx\\\\\\\/hge96U9WybGWmk1y\\\\\\\/TaNY5byevcGadJW7R9Y\\\\\\\/5l9TPKm5KScEbjB79DUMK30TI3K\\\\\\\/yMMnZmcVTCvTBR\\\\\\\/5kg2TMp8lR13ooDPwywRVxivWGcJwrGFbJkvRT\\\\\\\/DQMb+fdNDFuJYCgFCtyi4wVn4wa5\\\\\\\/FhWu7G1LH6ByuYf6C3YNzicoCahNn36vgF+9hob4Gddl5HLhZe8nPU3dlf55MIdwijU4YeaA0\\\\\\\/Ph5T5oAap9dFGBefYph2cHvrXMbauxi1FZh4UPBtsSE9xkzD+YFuMz4Hn3LwTZmU1g7nhql3JNwhLFU7iyDmHuYdI+Eg3jioWBgBlqRkYkF6pzFnANOLDVLrbLIV3dLI7zLQV81D9XqCFIuYNVyG\\\\\\\/pUPgidbUvukIO4HkMr3rxmbVSan5ePIrWdq2llsbIQnIebtDjGzvBOxdcJfHfT47TfiuYd8wfDKse17Yq+y2W2uH5ukREZQyugtt4kfQHM9Io94jHOsnWTqVDgnw6zRHNr2YW3DveNHfAB7NoSwSdfA4KfEzS+A50y8pGodE4WB5zcqBt9HpUWnwJ5L8x18EawWEuCTWKVtkH36oM2\\\\\\\/uAkJ66JrSxoF\\\\\\\/bqb1JBTa5Ms6Bkd9My1Rt4C1up25zojrg5M2fWMEFmnZEqmifPCYHZf9sI8AFamuSutALtX1mg0dpeUy8JRO0ckKQLhA2Qa0tgfhZ8jPp5oAsyliickBPw9h0cHzcsO7MLWjYjqw3k5zl\\\\\\\/\\\\\\\/GtP0OfYU68Cd2fs+oZSMc4u63pwT4qNkpSTfNF9POiJmaO\\\\\\\/xMQB4XJJ42PGF8tMkLweR6jKynHx8ONLTi8t\\\\\\\/1OqvRhyfPNtWARUrTJuV6iV+VaoGTYw+Ibuyk90E6DMVghBcNQeP108iQJH9P20WpF8nbBhpjZ1EODBoWhx4ziHiY2pxviA09mgxurcxsKMYcFdXqIl+nhSdohUlXW+NbzRxCg7Giuw98ouKTIoM3Jh35cWHm\\\\\\\/GyruuXMhFz03Uszu\\\\\\\/JevmRQkhn2Xx7bLK79NjKKMSwISptX3+d+TzmpfLE0xXQBXJ68jud+XEXJ\\\\\\\/L6GYOwD\\\\\\\/GwaX8Hwep4SBDdHnHsVSJ\\\\\\\/0JYwAhc3SqNh87GB3FsNoVkMjRNxpkQdP6tXP5kTSVSDadaHTd8\\\\\\\/WdTNtj49cHesZd\\\\\\\/Jyylw0jlUeswUJ8G6BdGG4OX5XLMJC8ZsCif1aAJKDLx92kJrE4m+oA208eWGyUvoSoo6EfnXtRr8DP4nDbCuHljbv6CWIMFzleIJzOe56YYOIaZA1FQ+oCn7vcTPvh\\\\\\\/bNqH\\\\\\\/iYhl1AA2ZHy2G3FzfL4etAFwWnXQNZ9TjaFzM5tsRAP165WHt3jtgafseha6gYwHicOwqwKvIHUuqXGPDg8XFAEdk+CQ08DsvmMlNTalsNFPc3WMgO85somX+quMs2xX7hcQciqE3sP2Ab7wrJo+C4MwPrIv5qBgwnZOkoTzy4CjhBCFvJl6H6mY19+21k801\\\\\\\/LxoZapbBk88kEoWWWEurGHPFlC0s+mzXqjjk00dW2ywiw4dJIRo1rXOrQ8RxCZOAJkXZ8R7ErOCoRvqwv47mOa7McYS8XB3NGblG3w9\\\\\\\/TzFowhJyJXas5olg0CfuNLg6QVTXef+ctA5o1hXTvqyWQY2Ijtbleg+8sfNs+Iy0e5ccBWxQhG4zZVqCnuLie5IdqVPIpxluUWj8qk6q31+7MkuRYyqfI4Ms\\\\\\\/b5zC\\\\\\\/ouxwQgxlleqMsp0IRDdeQUPdea4fa7CQjfW1LM1Ku1qBz9tytqmT7MOwq+i95Uuo6OLGvH91CPZcIy1XAH1\\\\\\\/ZnRlEFlCNMuElHFiW5t9OmRs6FUwPwY4VoUiiJz1k9GuT9KKz0yD64TckkibQCGmJIx90DQxC2hf5niiu2nZRCrBjCbuhpNg0NGu71zE5JqskaCUwIln6IjI8Lcm1iVDMYyi3JjqGGEdttcn3dqMS5sbXxUDJgyfb8TYUgEHZdG14YgXasCwBFJB5U8j+hvNmnZ1C0AuYkWh39hmcGuJuM7HCpnb6z3EC9JXDI+31MH4jJ5qmci6b6Svt+5U9X2deIhBVcZPrf9M4MBi1+ZAn0pfQXi1HlrUZQsLqRTW1ylABSelA\\\\\\\/Aj52jsuXflfvghCjiZ6HhQ7hhrI8p69d4NsibIfn5OvC9c2wl1jKrGob7f3vPf2VRajKPt6QmQTrIZ8CgJhntogyr+Ds2SCn9dQOSM9VS8ifw9ynj239rns8vj\\\\\\\/+h8QIrB\\\\\\\/Dw6vFic7NZa8GAPebkxu+b91cVE7Ceo\\\\\\\/+12YwvWgPwUKlJ2qgTcc8TfnzzkSRsTVC0O2O53wtSrnDfK56EwY9ktDXnG7+amCRZ9s1I8VsLN5acV2EIhFvGEF2D2Lfr7ll3Y2Cec\\\\\\\/h+Po+zYJH6lVYrWUBqyyACn1h8H\\\\\\\/X5gxzHMXOCpIQs9Bux+ei+IBO0l\\\\\\\/\\\\\\\/aoGcuag6mdd0Jx6xHOSeoBXbarkt82uDR+i110INsuMGjzaEfgYJTgviO2oIHHch7k+TQdJI8zEjOjeHe8Th7LdK5+gfHSuaCY21AIrP3YAsft5EOhykpj+iuY2y2DPw2sy7jHoDfm1Q\\\\\\\/CwxEpYkc0EL9UFq9jJvVEgyr1Fjfp4ejwh6Wli6VahNABFCDL1HrYxLnS5mRBtWLEO0VRaBTK+c5ENbx+dwjF7VgJzRe4JAjNv\\\\\\\/YO9x6vUIT7NkpwxBKnm20gMMBgduOxI1wb8y+YpSCdSen4pZJKUOLLkKlPHtON\\\\\\\/Fc329YBVhceWD4dpGNGmKil4rs2zlmuiiCWHHs\\\\\\\/Fyn4++aFSLXtiQnUupdE65KMugY6bRqY64819WNlTR4s1fbS1VrmXrXqub9ehgXiwc4VABmqsAnOwQOHH5gchb7H91Lx3wnAAvfERSxxv1kzO12brnoVIvChoAjzDnPml\\\\\\\/fD5KA4rpVoZCTMQQStS8OyEFK3BBMKLJBl\\\\\\\/UTPk+TW\\\\\\\/3ULWGsF3960lz6V7IQenLDgskm0GnZpzzrMUXa7DqjyoWSBcutK8xjaU6PZwNg4D4Kiej1qxVg78MUgP\\\\\\\/780jZ\\\\\\\/D5\\\\\\\/pjDGXxMD6cgDaBuiFZQGn9gg+oVJIaNezYKPGu8HjvWiQLC9AzJGyCHgK6Oe0snQ6+lVTHh6pVk9GpxqrscAlu3ctlSaANg3hVk8gfJFuwrbK3gGBMc0qbU+bPtnmvLLQGq8pjqIK1pCFFJXMjOVmpEJTUfe5Jmy4mQTKcyaQJa6YjWaW2glteqF+Essnez8PeUBbAeWYMWHVjlLscTE2PP34jAVwuSsh6EbjIxXdpzkw0X3a5hExZnTaLkIIXnKWdESaffiKTZPehsGw+tcnt\\\\\\\/preGblgRRb+LREo4c5d1qyDFv0VVTo8+oE3QWNh5UVyGKzXJAOs1oUmlemK3JOCr0KWGNbzSNjmuzvSr\\\\\\\/TECLM0NcyAFpAwVhM1vYLObuuisX1Y9SSf8JWNStL4LxgtXHeWQgQwOZ\\\\\\\/ZAK3NYkDXWXkA3vACdWj1x2rAqoLjsYVQbCvVof3aNCxng+oZKT9Cp6i4yEFYO2bRPw4huWtOonQv9OPKMiXlSnFUhkZQ\\\\\\\/pu3KbwFcNfQaRCQdIrVwkcdtqcqTcN1+JQUknZ573731vixNYfkH7IpX0up6aAguo1NTtjLx5ePAdg3DjLMe4Wm8D4QSMbnOcmhhmAZIXPGsZEgju9+GbfeWmJxlqGrVHc3Olik0FfZoHCcK1G5IKvhb9BmVdtbW+aOB1+LFhjbtojM9LjkL4fEB1DcZDcJRAPnVbBvP+ClERw2xHLTx1+5H4hcFG94iI8nx+FSDPpcvFXUCNpz46cs\\\\\\\/sb2W1Y9c9HX2q+\\\\\\\/oHd5Uq6zlVHdVFG6bB3HLNWrsXLmextQtenruwAuS9PqJARYxS502wX8K9tz01\\\\\\\/FrlLzer\\\\\\\/PqIxnQlKRE+bd\\\\\\\/e+83TSleRr2lsb8t9iD8aepQPZqzYu7HE6NFAN380+UPqN2ou5hNQs5\\\\\\\/U+gtocPz9GGzja3SKprV1MTPT2p0xZkns6\\\\\\\/EETikUJC1D6TOpDwUfYFdy3HA+6jSY\\\\\\\/qfOVFFpSNIQNhuYEoSpmSommdGvq8Obf861wB0EjT\\\\\\\/5QhutGfktjS+TMVucfSPGRZF8ud7IccuR0oP1xDZAFtRAh9R1SvlSQ3VvHlVhhVqRJ3\\\\\\\/lP4Jws3HfP\\\\\\\/+Rrx3WRolzy4Wcyaol\\\\\\\/Jxe38ZhrhOIMG9L1ALDrkjZajbLrrqcGtEJkd6MdNffpI4Lx\\\\\\\/qmBGNQ61dph8ePBd\\\\\\\/pprDIN6IWirLz05GSHFnLJKMkgQ0VYlYHirbSW1MVcarGmTEkVo2DDNRIUsWKx8UXuesU2ow\\\\\\\/ZoMHsq3Ua+NMbMY13NGaJa0MvyXpCjiwVAzt\\\\\\\/qcA3KVsWZ2fuaJ9E3MZQ\\\\\\\/FLI5taOS2gvDH9Lhkg29DS34whx3Q6VgB3v0RAktm7jqAix7GhyzaW\\\\\\\/47\\\\\\\/tCJxrtgGsQuo4eDJSWQYUnsHq9OAvOxOXwpyi0t4rksd7V2MAckAvJJVn6aiF18J5ieKc6zH+7mQAUyT1nroXPqLz0Vhl2664n5Oe5WaLl4KPyIOcLJ+CLMQR+jcPs7IATvEhhxNChxbQGXUwvdr+mtLPoBdML5r1fPUif48kU3ys1y3sF1izWtIoldBx8iDFp60pCNJwLY72qeOs2UF\\\\\\\/MrV4DQDP7hW6RvhRLELbBaAyzNQ5dbNg+DxuFfCqm5HStpXk3VaP97zPliwNLROwSDHqiOkblDfmfLzekS\\\\\\\/rpelnJQ1FSRoxngHmBHbjwpRk1QW04tib3P\\\\\\\/Pv5oJiadN4Y7jZd6ak89ahbVl4wupVIqNnYs5js6JIa1HeEkb99ln0il2UsFm++1ngwbcV9OD7MZWzLXInb4IGXmfCqlI\\\\\\\/+NMXoBM0FUaoPaW2uztUdd1MWQHvi5rYTb\\\\\\\/LjBGhZEfkPMExGkm8Dm2MbRJlon44gWxzZo6JjvsoIM8dwa4k\\\\\\\/G1Jw3wAnFlnYfSlgTe7MnXJhvQNzOi+UGJwjJP6xMLpNEDf9EauQaee2UiwcjxEym+qjiG9vqonDyk4L\\\\\\\/gARKwVhkoz3CZEUZ+6OAj2tkXLcObrRSaYWS7dQ\\\\\\\/irGU2J9QA4+FOpQrkApMpEnWP++h4MdWrlpaxw\\\\\\\/+3e8SSBaOMwCqb0+vvPFfkK0tkd7MzzJ+aoqS5vezbnGEbEMuw7c3K2PRomK+VBJragqO\\\\\\\/tMaRGW2Rn7AeQCI+Ii7QdZxRx0h2+KmT8yyVMpokamesSnlNh8KX491lrW+zbxr5C8mCHqXpzbkEQ2nv0mtQYEv3LM1axwQ1W3UtcGxec8zPYRwm4q5nXvvDB4Ua6hAYLjL3HlwhqAagA4u6RED24\\\\\\\/32UX9aWCBK\\\\\\\/S3BBkB6p0dN+8DQ\\\\\\\/ijBU+zJLEtj7MstvK48osfjGLfl1LG+cbYZ1nPPKY27oEA7O4\\\\\\\/wWjzDdbAO\\\\\\\/SVn\\\\\\\/\\\\\\\/FhuHZzUcBrJJtUmhL6yjiaU1GcZuPwFcWtdeWhbL7YJUycJF+wlrorrKrHokWRI\\\\\\\/yFjZ8l0uyk18epze8KokdJHzegz0eyCoZDY37mUIRwSBRFZsS9XP4u2K04KN+jZjSz7olk8orFcrkupt76kO4bzzdbne41H7SGgTwNdY1UfdOk4\\\\\\\/IiOzIW8Xn9tiTlKNliKRYcCK4yJRV4OkSkF1eR5X2npD37FjO9PbBy+ZvN9iD9raaabPJTuHWhcLAY3wfwLOCVM52Wox\\\\\\\/2xAJBT1FK6CUK6HJIF+T45tASL2ddZFeSX+7mpvNl8fSMZtvS9cyIsRLfna5VGOgGyDJIfk9NcJXKQs8Okzgr99GccdvKj9Q30IHvlJ3q082iez6s3e\\\\\\\/Gtu+9nGFHAuvisBsFxxU922X61OsmGmtF0C+Ns9xRhgr1YKUG8ti7360GuyGy5EdcWuG4wSP57irT3WkmS\\\\\\\/Gns2s863aJmolQxn+gNFDF7XMAS8fHcQW9fHeibjF8aXAil7icUr+l\\\\\\\/bLl2+lNLlbMdU5dqwBauYdl9bNob5JfQMD7Y3mdcHpPbh26cQbXhlQ37ucs8EBUPkFVsvAk6zShbD6p\\\\\\\/9R16cq7gWPVUCXYk1jWhqwrYnOtVIDFLRIRRSUfSeLGgYPgwU1RgyKraB0\\\\\\\/E24UYAJVTrd1UxHlscVloYbuCUzol7cxZuE9BhAP16Y+wAbHZNSiG+jwzcQ+PVVm2iOax2M8H55BBVRpRHk7SB2SkAvtkaEwpaqfVIh4u\\\\\\\/d8StlQeP01Wn77Oa0U3\\\\\\\/xQ2007k1c5tspEGo1QOafdLT4DHa855xB5zWEuhuT2wGN4Rn4PJSOSc8ZzKEr3VhdhOM4LiLYyAXQoSXmJnztoqdRRV6z\\\\\\\/JT41GmZQtBx08Wg20R2n5NgVn+un8RbNRTdStljEwksUIR9iJM2CRI1uD9p4vGU583RAHlwNafVI\\\\\\\/dHCc9rYqJoFABumDSaWWi1C2xW8XX9lBgvwrhKQtZF8Za2iFdgLbYZCssM42g7RsFvP7TW8famleza5dttfDHAb9akUmMr\\\\\\\/FQmFvxo9QbUePQ8HkWo4oxfCZmpb96akzuJNVFpDatc\\\\\\\/RP5ApukR9a1ev6lEgTRYJThOF9J0rcicd6V2WXGE7QOOILTjPWqhdJX+1hFtE1AROT6YzUaPqIvx9uAl1JFhsE0331+k16QUwvc4TlAhhvlcvhVZEHh1+flhk7bslj8zj\\\\\\\/iM4xGysGW40LsXLoHPNCpcUgAGjtoQsnNy9Rhxc57bvs2gVF6npM\\\\\\\/Z412pJxwmawf7rQZF8TyDw8MOYU7R\\\\\\\/drdoMaJbaaFVXU\\\\\\\/Fo7lGfMcbZxUC+TFn3Frve\\\\\\\/ekzh\\\\\\\/2zkp5VtBnwNPh0W8YsptMeSxIqcbGlcr8Bk+\\\\\\\/3c65\\\\\\\/1dft3x1brw8\\\\\\\/swdi+yiWXRAqIqrtQ4UFFUSVuArBCv7tF\\\\\\\/sRsnBaS6P\\\\\\\/jHCvrAgMYMRTM1wf7krMv5pSzuZSFAqWQAEckdkg1QJxZDbXW3zKsx75+wBWNUb1qn8XbZY8IgTB063CwWmnqK0WnthsigYUZDoyzNzEucUdnsph0DNfdIWnYktDX809u5ZPKb8XVzcbnIY1Fwxdqi+iB3JYjIApPdzrsWPJnvYKXa+cPkjHIHmw8jwLY1\\\\\\\/om+vddNLO8ZsiXIBV7aN6oR1KH\\\\\\\/BNGZIqOs+DbXtmOKGS4K5YlupWzIvAYhDbB76wvdbNszXQvwecCfXEmU3LQ6hj\\\\\\\/K1SOZo\\\\\\\/bxYsylZJmKe+t0bgM+kdIuzGBJV\\\\\\\/P029Tnwzh+tuTmccs9hBVksosmojYxru4sCGdLVXsAAazU4nvaMJomci4n3xzCjhZ\\\\\\\/LW8uSD0tJjfyV3bhT0WVim+93uq5SKtcB+fq1GPVhVW57AekysuRpHdsxnCyC+rZieW\\\\\\\/7ebzWqX8367YMK1SBoYVi5BtLQjAQo28tRyc3cl+CLysLooNZEf4HRIBBxr5PMylcSvopudzAGvFWr0fhFlb\\\\\\\/fIPkJTtHuqKVL6ZuQ1Y3GiP26g8hG9fvgvnkqIVQdunnYJp7FJva6DKt0aBv2yXyxzXOMjBURBlfeyJelElcjVx2VsnmNppuJU6qeof6KDtg+V2N8T\\\\\\\/CNwhYLRFPx+USHahofKm8J5WEhrPTOt3se4Z8pKD6sJz1NmSPdKkd8XSi9CkWYwQQIB6LBgGmFunuYdNvs82L++QY5O6ukmAb3AaRpsxLgLX\\\\\\\/CfESZg2L1t9ptOvxnpPsLcRcWXbCOrAqlNyw\\\\\\\/2cfXRcplzGaSn8FadwuFbuCWqqcH6RnCkJQK6X7lvTN1wEk9DibrTg46WUQBfN4FivTPvwSK0+Z1Kb2LI\\\\\\\/fdzUYdk9BGB0vmCQfBkhrt6INi3HRHlec1g2xZbZFc5YtaARjf9+P854Jha1OA1vbjTbgNX7R8VLjdPaStVbmiDEZwzuymTdirWilExImK9VT1cTfop\\\\\\\/72DAhGCEkoqUbCLjwd9OMj+sE\\\\\\\/XXMXYQm3f7GA\\\\\\\/2GUgBBor8NW+1gPRZuPp1msq9dSVjbE0GMzzrVcV\\\\\\\/n79L8Aq5HNiJHtV4UWG6evbvAH8wLkJ39p\\\\\\\/qIAk4DUstLWn1Tp9Eg6FKVP1fGG5m1m+8mR+TJ7uA\\\\\\\/eNw\\\\\\\/C\\\\\\\/Pc4wfbfsRNk5qOTTOfgL3COZaYsJjYuvkpAistYoOBgo1V+NklZnBbiCwCu2JA5yFVKNR\\\\\\\/uCQQMHf2\\\\\\\/KUMjF96sm5qjmMy9oU7Lfjx0Mb9UhgLfiJqC+SWuDfB1Sr6dbcpacFZUnDop6esKd8zZzs7\\\\\\\/BJAzAThGCXePFIQBTitvTIHkE51pIWO0IIdcofh1BS6KLe70r5crMeFOjDvmZlW2r8Nql5sYyrsc\\\\\\\/Mc4QlMUoOaR\\\\\\\/mMrQoY0Jvd906iSzvuL0\\\\\\\/VW5+kBW5vPQAaCqL1Ej9hk5kYRbh5gKb\\\\\\\/GRd6vmPk1JG4c6fgG1WDuOzIKNq8TebcxbVFGgbZHq9LmZcPNooeePj9xM9ZEV9Y5B3Axu3KHCp4GrR7MH+Z2bcHP\\\\\\\/w0NCHkewv5uHxPRvyBatDX+KVkODDgrvdYgOeOxyo9yrx7pihXs5H0udKvb6qfOFAvvBF6KGLSq9WTKAynHZTL16D38fV4Z9lC17XqD2PIfckGy\\\\\\\/owQx8XNDE2GR\\\\\\\/xtAN0yyAPjTfJDrAXF59U1Aw7kyFqOPZWbZrhcU3\\\\\\\/nHjT3OXf6z4kq47oALc+iKNm4DCOX08AbFr65Rvvsew5Ry+DfLxKfkgCHb6GQ7zmHTu63aLesdh9XeFanYULM8XfCcPYGoNAc71S1C4b+fqtsBtQctIdKjvbF9RkRn3u2Hjp3dttDhb5289dOYS8miO4WjDGb4bhuJuW3Xq2RT8eYiBaAxk78qFbMpOFlDdUx5V6AzvU5NSRuKzDjvecx9oKl+rKOO4w3grH6JkKxoSUAP5i1c9eW8kJKoQOPGO8FeqxnKKS+51qyBBHFHPu5VgZC526Uwa8ZN8et56Mv4ERaaYx9ZDAH6sjzzVSLiAySzN3Cxs4quRZXqbkAN2fgGkXipUgO9K8YI230ySHMf5jS82F965dn0MErwYAY7RYKAU37jD7j+SM4OnYJ2Lm4MYO016EuSW1w4a9+evOiKvrn8mDn\\\\\\\/EMSWOOZc2LDUEF6UNg6t+lsFkuIzpDFebjUv+myVvpuxppu\\\\\\\/mx8vDgJOd6iFjVNA3cWC62LzaklzIOCw4M7OYl\\\\\\\/8mxPAOKAkriW6RX+Exh0WaFC1kLXK6\\\\\\\/GfNnVqCed7PloJ66HSjPddo+nr865RLxJJqyz+Ab0yUcmr2jSU\\\\\\\/twxvGLWFwRhsM0aayHM+K83VlHKv8yxXOqLniJa\\\\\\\/tj1F4zSeIuCvniM3QcCsQMi2B\\\\\\\/Zv9wQXXhHwvywzk0JLx2UHSuPPNccGg7DCt4SebkJhSyC9z+MdyyXoGVB20txe6kBJzScdjTSri8yjtKCcZpisktUkdTzY11Km\\\\\\\/vl7qgJui7vcJgo9tKPTIS8oIIq6EG1YzGIJNmjGep3WnFpTCFlDVkCOYaCUC8dXtJ1ok6Z4zMha0K+Fi6aDc5WdPxdVKlK50+1L1zWa41LJGH3klHx4bVU\\\\\\\/27UR7NYhhonQR6SQEFg6YmBu0QFFw9v8DXVN9Vv5O9Tqrb4kUxswJbfZILv72XhjXoubAG7aSTWK17BoIjDry8K56pElELOC60RzE7plIX0S6XGS4Vqw1XKqvXD6sh+I9\\\\\\\/6xIesCnQ15HDyDSGaD5+asF0T\\\\\\\/xdNyls8uH5UqQntB96txmUk9MCvThSxcXRCHQNZPDb50kdfpdSFwXw4dceDFtRY+MbW129nrVJ1jEFaMzhJXUqqEa\\\\\\\/JEvG+4mdlWWW7k\\\\\\\/LZ2O9DkQAF7xGwPQ7B3pfCEvDD3JxXQYUy\\\\\\\/M6ldpTyw5PAJn08YknnF4emezimrnoeVY+woDH20W98Xo7zpqwvmtvfeenHHpXRUhfVxWM+rdgjzFVul5kRiXFPN0NFSypf7\\\\\\\/UexCB5yQ+TP7UOyHdZI8it1DXvALAbw7rJ2BRpxCHh65ILM\\\\\\\/7+3kmxoTTp3w6sG7YFVTZo9EmzxgRa\\\\\\\/nPaGyOKzUo0JcnWUF7h1wrBxcu\\\\\\\/qDNebaNp5Vv+ojzfG0Cde4fI6uY9wFlaSiHjzKkDUPtMeSaCBTqPL1UxqQhYisbeE8Qz\\\\\\\/9y21qTvSTu1T+wsyh9oK91HT\\\\\\\/btnesJtAxR0+fbtYYmzhw71fpbVpkRrViSBEMmrdDeZRJubQQFHIkD3KQJInn0qyKRy7dn8HqTLFTlJCfiyhHwRP13cT4giCYThD+njzsRX1rdCwyKejdPtvluOnVXl8zmyBg5DRq4O7pi3kdIwyPf9QjioCZKl31y0UGQlOcBSrP9+KwobivGhaT2wygJ1AXrUS0NTtlw6VQuwSviN6fd1AULkTkgw1rNSut0UqejFYezlwQy1afM+opzrWvzsPG7L6DlKVedJq17COd8P6wH4kbbh4z\\\\\\\/37ZwVdryKtHXafFBjLfG\\\\\\\/z\\\\\\\/iW6sElTxQBH7cbO3iAFR33Awy3xgrfTe3llYk+DUvx6UURFDFS+Ryk\\\\\\\/ZqRxxfDezEcH47OCkWf30gIDUQEU0wS+5N+4jJzvHZx22zR3E6FRsUG+TgJIRqY+OMvfaT9QEl1Ntal6wAR3yQWTp5KkBc8g5tmTHis9Z51qowhq325kpg5IQE3rKFEv28U2MSf3TVJ7GKrRmkK5hd+zQV0ZwBXqYXlM3dyLBLTgcnDfWFuqaRZ4vXmPJzvBZoWMOglpoCX10TTzEFr\\\\\\\/60zNLQ4ZqjAR+VZwwXKCIwigZ2hkMJBo\\\\\\\/zcDbYOKR81zWefxc0NBSzyXaGcTYochd6En0CwqFLJD6fVdexzmvHZGxigtLFoM\\\\\\\/I\\\\\\\/AmMxVKozYNdIEkep5pTxkQFibdc0EzYGlwggnGGxbXH\\\\\\\/Z4BV267Vd99W1aFK\\\\\\\/sRZOoV7Dvj3Oa3tbTZhqvodRrznEPm7bUcjOpS+traBCbde2hapojnCSH12vu5qZu0H+gqyioOwQ8eCgYufHiXCsnHFc896jN1jFLm8rQqbyDZ7MXDJ5GoUPDSousyt8Gskq86aqux1uZ+xXoHPQjZ7QLaxnq9gUcycS9q+APzoxsQ0JAiqJo7NG04rtAoO7FOkK8Y\\\\\\\/69tnRrgJUL\\\\\\\/iXCha3Cu0fknZVCSgXayjHbAbbXavdZI3OmyEOjhggJQMlD2nfav2CRmfMy1iV9XfN9XzjurfYFty8VXLcaYCQhuKROmYAX6il0L1VsN7WrcSQ7qlQWML+d3WqelA5osU8T3mOIgwwzRbUWdVTXfnRlX\\\\\\\/CBwNcm8n24jWufAMVt9+VP8\\\\\\\/9yawUOCon2MVfIIhcwMITPlH5G2NiLYKOdtWLnjWaB4cy\\\\\\\/u6t2ogcSaOGpKF7Bea4y0R3RwsK3ag6dcQLhQ73VP0Pn\\\\\\\/BHShXj1SzG+eWNvf4yK6Y6N\\\\\\\/t0SBzXPP2I4Au\\\\\\\/mEAnGEOkB1NHnqLKjBi\\\\\\\/j967VHSWY5JQZziB3SKHYwvCtaVIMYRS1MxXtuHH5ZEOdfMwLWfvHdmHDCjClMLbFGCrl9kkTKvSyh4xlfNXxTrZ9gI6NEjALUUKEnOX52TJ6iAG4rkqfrqg\\\\\\\/Q2E4dG3vpzSQ6z0pieNPcC\\\\\\\/QinzWkieJE60Ww9U+yQ7\\\\\\\/qcWgS4kox+ffO2QPL8PcQ\\\\\\\/US5v0BqbYLawivGvip1mOpRQUBv6Ej4FDv1HNOzkvjy5Q7KycFb2Fn5DdUQPDr+aNcjm3XJY3FssMxnyAI0wQ4lvL7CQAOzUPO6JMXfMt0fRRrjgp8DgVrY4GH2CRCml4MABAtaxGDzPyTH\\\\\\\/ycBDxePv2fCvzLpkDRjAIR2D8KPInkm2XyDn0+IvrR9zAJ0ngW4suzZjrjfTaWgSxjPFvYAtck4ZlbfzfInOnbUknud978inn5ZMxyRw3p9fAyzQQFlxG7OlCBBJVirga5g8hvkwQjfZb5Q\\\\\\\/UPPp\\\\\\\/SR9Y2jyWpNCPFeJS0Kd4U3m1qZxCGKQIWKlaqvK7yLrXTlPSfO7OmGb9dlovw\\\\\\\/qLgTcghSGUf\\\\\\\/\\\\\\\/qdD4+yEgren\\\\\\\/pi9IwgikaHaQ28Y6JNAZyOwkW0rh7Di+xnjQVzqLhhd8I3HOHyoAMuOxlppmOfpUrhikL9ryY7EiLiGFZfdbcNyXNQPLtb59n0BQ4bxmpWNGE9E9wuDWNRRonRqF5nTlR7VLjwz0wzVny87PMUT4bxQ2Td0nAz8WmGWP9ENt3OsCJxP876quIizooOq76\\\\\\\/W3d+cj2A8X9hUdMTMjy7G1POiaU1zmRVDqDrCYiTNdf7CU\\\\\\\/41tWtn4w2gbNZl9pFHVpKY\\\\\\\/A669hyBHuLRxSAbuGWTbRpUD1cyfPIGE+Cj0jHVwRiMUUdD4eQfqC4PQDWg8luBSa4yCRISkzeiCSCthF6U+CU0Z+9r\\\\\\\/AKRJ2yCAZ4llRdgKFYxU2ONBdjCumxDtbUJ4PWqmhgtzqOJf8FxslSdQIBmo37t\\\\\\\/DBFUD2vcRGE11MINhWQiOLMEUxGIkmiQF3x9lr9l0GUgkl569VQjEGDESDOMl5VekRx9uTZo36B+mYJLkFUvNe804usWOrY54yD7ta4yfs3mr5Z6MecBd7SURaHsMcA7MZkwNXPWFVgakwhutjAVathvVJjiwRCUs3nruEZ4E4kC43PopGi8Pprq9uubw4tA74vCS2Biw2p8xr+wdOkToHl7EOz3sxs8h+FZ0i\\\\\\\/n0VMnU8IGo58ZPopazwBg5740pkKBBGCsgmLnUioQck8YTM+8i4FAaqLXiMhpZU7KRdJWDrH2fZJFEFihF5+Z\\\\\\\/tRvMVVJ\\\\\\\/8h0XISrMiHqOnFrr6ZJiyMRdn6ddM9TNWU7uN1p9yUuC\\\\\\\/y0jBb1u4UWdeqyr41q43Z75hwrhQuS+cZcGYxGbWt2ujcDnkUgQ9laKu5HY2U4yfuPJ7fARfzclxMr1cyIImUU26YuoMJWrOOqb86v6d6+\\\\\\\/IvG7pM\\\\\\\/XDGHT4p7gC19QzQmZZdX1vIJmxdv1eJCXE2je3IebsDXhl5m9AetAgBOxwsGeiw\\\\\\\/W+93QIVF92eRutGdceJ+sO1ISs7mRLz27VKfEMt9bVJWPL+xBqPvqNfU7bRmsc+osWk8BohdV3SHUHDeadF0cg3UW9ncOwqF47j8tcq\\\\\\\/+tnRkgT52ujiBHQeBwoO62lCzY6WeyQ9J0J2YH7mBKcDF8YmgYLU0Xa9ecFduzepJmhNT+WsHi9f+F+hlpeRFKY5F8bbmoZZtl60QcFIW2sukcYDCjCBlIcouO6DlBRmAcfl69VisPV4zWXLX7Z860XlrVTNZiuzT+eqXZrI2CMuFoC2JVivBDGWuJGUGYCl0fGnZ49u8FgARp3YBoFtqVUhNixthWQMl2Ds5O2tPu5y3q0kGoGV2Pkx7d6+iLRZ6zViGb2bKHYWiXYmshVhhyM9xrsDACX0O8WUefXWfwcrk8qSGp7NASv\\\\\\\/uD9DhB\\\\\\\/ly+XK9UMXIHTQeijgFvOe1efy3r5iik70I1eZIUMg8xJ071Rs72uHn3pqOvUudH3C0XFETU8DjlVz++GdyWQkyzpDILrTeyH49jXdQvOwlTcKdyN1auxZdQwCSnE\\\\\\\/MXHNjVGfqhJJTpD\\\\\\\/Y9MKEwDEI1wDcq0EV9LjuAQNY5bzTjFFBfIh8SxTkOi1M8rSjJ0d4yMEpRaRqInTs9yNdWxvxftH7mLMOo+zI6ZVziuneO3lKFhLGBnViyMZ\\\\\\\/49ULYxOx28hQNsdCbxlEi\\\\\\\/SOo3V6rWJ+Nhik8svXSyqhEjFi89vUfSRZwPiM\\\\\\\/P4PXcBGBXKRhKrpO9esy9zA+V788FyAj6HOFGrl18hLaFPphnXt76Zz64CsW2bLZq0NK4y9IuJ8CNqDgbI3S+CTLfeEShorowN7+H0kuEr5G7nbWOqZKgv7KmlXfBatR06enhexHaIKNq\\\\\\\/nL37or+WF+NtoQh7TghpvNvJwIq1GYvbtX7Olu6jQf1BMsOS0doXH82dAlgaCXNV8ShgBjBNg+eA9jBijsD+vqSaT5DB+JMQk8IYoIZlC2B\\\\\\\/3COT4uW8j7ouo3cc97IYBAZ4vMagBERMeh2bJbv2dJFaVuLjY1Y9ZyKe3W5ShH2cJl5q5A\\\\\\\/CK7vVfhkwFT1Z9mqhPW\\\\\\\/qyTxASN8Cceqx+H89NYeNs0QfI80INH7A9S2k+d1N8rsJyaZ6q7Omfrofa0KGqn4ha5y4orokRqAXyjcEn7WHgtu2PFGM704Ue0FjwFWaTezUPY02FxOAjB8C7lW9MqJum7sgm+AQOOGlpaovsEJxBQD\\\\\\\/iUqwRtdTmQCCcHrceXsLIcIswSjKOPIaketpPtBJ+SWRj60sruQD6y2REJJgiTaRLlEFTBC7hfA4KijkWtUL+jMV7DFqXvudbi2Uh2+G0J4rNF3R8Brlaj4\\\\\\\/s+Q0GjUSoNBripLYMb3slVUTALl5004z8iqbfcH7+C+O+jM+JbcHQMZxAcRCYLRx9WerIfk5NygGC8+9s5oKxqc1PfZAil\\\\\\\/bdQDgrJpEfWyXx+l5xeVFQ+yqgdfY4bNNejVDzfJqe75VXlwWyYeOMocJxpV9SPJ486rjiGY8HsJ2MpeikabskJ5b6\\\\\\\/KrGAQw9ooi3JYVyfuxRVvJkgb5ZWpmbT93NeksEuSG1sN2nWzu6Y34WiaUvxItcVOsBJW+HtDaQC8QKTUZP1CFqxwRvcyHPYRonVuX9UCnKliQXJ\\\\\\\/979\\\\\\\/qMD9oP54Fvy6wEd6qd1G79lMgp3YmM2nqgsFwlGwuYjkYDYi8DRmUkSbY6zPFEONgmMOuYLunoL0FFG7MCALxKJ1tFdG6u+tuBMMFoHfr9ytOshyHpbRlKjVHTHP0delVpah0HEPrMFXwBTs+hQi2R3D7D\\\\\\\/FEEz\\\\\\\/8jdaC3LS\\\\\\\/Kb\\\\\\\/5SY0EzvQytiKro1YPEib5bnwO+iiFldgY0+R0\\\\\\\/C7Wzs\\\\\\\/GtjcmqrSIOK+LVZP0J\\\\\\\/mGaWMpt1ms+ol+D7+I1jY+sx+1IXr6\\\\\\\/aoyZILU5OuUBRlXinlLQWTSFe5TTRta6Lle9xyaRWkMBYWpeFtiaNl7HoP+Wt6C4XHa5rQEy6p8SRNswKbckrQi+u5ChWrhO1P57tzF5M\\\\\\\/ql9kmS80l6WEzvLk+U5Yr29mQK9hCEt89omm4F7JFIJMfF58Ynj287E6IPkY1F\\\\\\\/sas6ybKNpEoahFIQ5rCzR6fUT3odzUw1e6rH\\\\\\\/GapIdwWRofIOy\\\\\\\/p0aMevEdRC09ytoD\\\\\\\/D0yw642IETR99tB9oXn0FOtx6+MhSGF8nBlOjdjreKTCg9TmWyBR17PlTBLU0Z4k9a\\\\\\\/ZgZTORf12jF6brGZ8n8MDGPmWikzieGuV\\\\\\\/X7U\\\\\\\/QdbmETYnyCIu9u7sS5ts3ZW8NKiNYjP\\\\\\\/JrKtsk7Sg+PtHihoCIAF+evk8vttYMdfO6cpnCEPNwzks\\\\\\\/SoZMOAxomInRwtjAQuwjRxqq0yvzxBYMJ2aqvLaj85ywUi9xC2ReuHfAf95tsnvcpyMnhTknZ7hpdsXzfFA5KtIRswLlxS2YMh3c7Tp0GRlcfKmMROwxdi2gvkEusuh75YrYqylWkiXHjN6ApsIWsUbJhhe7PpfPtIMBX3PL5DR0yoreJv\\\\\\\/ni09fKMs277gUoTeb9DvlPYBkNtNX2\\\\\\\/psWVaXoqHaFdyejcSfqS6\\\\\\\/AJlzhdPmVo4hgrjNFNOYHUaDOZi3\\\\\\\/cz7vgLwts4knLc2nONwbF0QTbfM2m+AWp70NfS572SkD9sIZ5+O4P0wSVFKU\\\\\\\/7XAWZ3NOtVVaM2oveoGAKt9GbDFP5QhW2aZ\\\\\\\/g8AKXTTG+UMbNDht44EvswXkw7ZoKXMqal+5UnSXT\\\\\\\/ZuS+ZMO6\\\\\\\/FPgmRF7vjAleOk4jC56KlH3v1WnYYxyiORtfB4o2e66SxUOGFI4Jv1R3a41vbYY8aOJW+kgNPMglyg0GsJgzxDB+9D2lQM5iZ+Xh4H2A2\\\\\\\/RTah2hVGFJB\\\\\\\/VZTMFA490ET\\\\\\\/y4eSiDprlY2kv+bp8\\\\\\\/bjZXRgiKi97jBMG+f6htkVNE7fZk\\\\\\\/xhpfIl5tGZJtP4t7zWOo0MD3x2SppE1IbjhLw0aU8uhHRJSzgam1AhVY8t6yRgBeRCvJC2MyTT4KFq0vn8ykLunxlO+9YR1O8DI3t7vRWn9ns3g978f19+t5s3CuUFtmy7Yzko6wh0arWeYQj1NLybKZkoMTEz3RXtODMY9IeVovMBrw1svVyaOVFaycrMYfa5dfu\\\\\\\/SHDW3QYpQeN+VztxqBY3L94ea7sltWJAKTwIlRaejwCA8u2gxWy2rsqfqu\\\\\\\/GPQRrInco9cvwaIa033ygUarNmPY8uNmFmwEH7\\\\\\\/8ChB7hdW42tc4KEy+Kr2dnrMiIl4NHUs6E52JgBAhb2klAqRpldqBb53+d1KqsFAPdAKsLUux8FuFvKDGU+RbOP9ybqV4cNp\\\\\\\/oPh\\\\\\\/aCnB+EowwNuVtMcYX2U3symkcvTWCz8noKdCthI9tU6Q5CHmmOwzp89t3ScePDc+LwTGnKqmHF7NkOLShdCp67Hk7Dl9YGE86OVGDKdaz\\\\\\\/SKZEOY0XwVEr22z4s8QziSTe1Qb3KO2oNY0B6G4OIQEFptIUh6I8WCe647OV+l2\\\\\\\/5fomRtlP2ABjN8\\\\\\\/gjDu6z6QvWS5N\\\\\\\/sxuTLJekwilPC2GWQW7aylV2bHw249lXnCb+OYIJRhRGPsPqs5lF2qoEnqxTRxLIG6S2iVUJAaidIAcvWO8pexpoCBsrrOkkDEmkQuaIMz+mawS\\\\\\\/owW+PXdgIwjxJYXtSBNsTnybc19nhadJczvrzAa4rOP+BTj53OLNIROSUPiKNHYdAyOP\\\\\\\/\\\\\\\/bJ+SAVQYM3ehV0VqVH9afwXnnG\\\\\\\/AjGP0QOKQ8sCkY+oQdFClRPpU0vQ3fE2JRkZVbUZy93LiGIfGAw8uteFk7KGRMPyY\\\\\\\/Fy5Syk\\\\\\\/1LdFdldc+fYLC\\\\\\\/p8\\\\\\\/CrgrYi8ZdfXHXP\\\\\\\/Ya7u3ZqYL9v1UaetNP7v9aBUAyl+jZRbP0SBH\\\\\\\/zTAI5NgVGmwYtgntB6Q+t\\\\\\\/7a5+iPos6\\\\\\\/6P43GomCIbLGut9oRTvtXBrEjgVEO2g2bNrO1UhvFXwu1IyVV0zOM3gb9C2A7tnmkXGG55Kikp1mwfC4nBznN9CwULoUU\\\\\\\/Z3EZAysdp\\\\\\\/xMZLHMWPxCOj8K0kCzlfT6awYW9XkQbtZ3KZapMSKXU2BPrMQxEQIGF9LQXrkEqFhbjhkXc3hXnLKhGbiS9ORqFFEhRcDLy+8D+pF7JiC2RXqBaOgEyJFQ\\\\\\\/vT2BuL4zg75pw8LYyry6qDN2TZOrdYcZZkxoHMdqKKE\\\\\\\/6Ej2m29zVXyfMFtWgjMmANcr2FF99tBsvpWobPhNijf3R2IRYsWxzabdyLod6LYOgtv7+znZPx+bezFSDE80itupsgEmUL\\\\\\\/we3qOym6MCrroWgZAcsoPffUpwL7Iw9TZXQiMCA4\\\\\\\/ZNUsVr0V3RMu5XPAbB\\\\\\\/SCvSL40UKhuV2fQBRfWqqmI7GobtrqhcKfzAFBy0Y8g+jF6si+F0jby7\\\\\\\/H3tsJXvg4bmf4hNGE9qXTqY1rZKrvhqyo0V2RZOzoVZanq3y5eV1ZQEPd6m1zIXWDpvopiS8+cCpRqBRllEkVd+EP3i2yuGP3jsVQ3UE+3C2A0tJ2XjilfxXVLIbC0T\\\\\\\/27nuCHb6QuEUAoVwQFjWez8wlxrHR5mIHCIEmASHWkQgHFFbo\\\\\\\/bRE\\\\\\\/VY\\\\\\\/f4\\\\\\\/S7EoH3Y+OtDKjSYEz5dJkJZzHW1VW4fe6y2Mh789bltl+HWk2+r8FW4PoQHVTpdS\\\\\\\/0g6ZpQu4buVQxbR6AA5qbMmzc7mzfDeeTRGBkPDtI6ey4tRL5xPmoPqrRHBZUQLvu8WpyxouLbItO140Bx4qikrI1oXVgEjpFTPlTvUEPKDi0h\\\\\\\/5AukyNopQmczdmXYyYgO9retKviD1IKFw0I13pykm4cTqlKfhF0e9ET1yCBwg4muGLYNjJIAQdXaPEwPPBZyBTmyh8JMtTARv2aNesS7jDmDRQc0Lg3HwJF8F+mwk0ASwODolVr8E\\\\\\\/IJSCpXMjHFiDt4LfXRMI1dYxg833rPhkDZBpiactkUy80uMfd8IBc5ABUkfZMiGKtoE5jKOYZ6rXYafyBQIY58ngm6faklWvN1blhZ93PgESZG6TTEnRnb8r1sScREtyLJ9KSrYO\\\\\\\/g9Q+wvRLJbwYrYc3OTnZQu9kI26ROi\\\\\\\/XVtLUIODVfnxFRCWkAkxv942mddZ0JXkJPXWMl86p4neTVSoXEE+MuPBsmrDe\\\\\\\/P9UFsfn+dWEt27547vi6OpXykErzHl8wq8lMSqWQVnV0Ab4QpDwkE9kU4EVWkyKV4sD\\\\\\\/zvVWgOdBAEmIrzIwG4XuhuoHYoIRotLIRdMSqQ5eagXeQmalu9bbvXneUS0EpvQcwUj6\\\\\\\/fjkcUur0UmQ3A2akZuX8cZQwcCrjH6XcI\\\\\\\/4Tp2JTajueX99n5sqT7oM3yL4qkFW9LPES9l\\\\\\\/AjCr0eXzf1PxGIISOA7szAdDJVIC236fCYHuHx1FuiZLx7F3xCRgCFmbTm6Xh3MEy5OawzovgMkb3QOnK5NNUnKRlT6HDj\\\\\\\/sS5XC7hv6O7htZlL6GJgYE73q9t+ntJt2FA+8BCcHJGuIYEQDr7hmRPGUO+AvijzjN6cSt2jR+erGI1SOWCkjNvWsDXQ0BG5vKzPJWAKtgNvZEkJBVtzd5ujBXeEnkscarbgMq8Oj0rX8lRY2g+W0MYhi4rcHtMykAnZQvkqMz9qtTofPm\\\\\\\/eBffUGtF1VzYGFY\\\\\\\/A4lFvz6W2QMxRExJJdqG8FvoMifDH73Fw8P3JppvdVVdTsvi9auBua3Yd6AMR3P7zThejPBY9N+NoC\\\\\\\/HnQL46j5yJ59y48\\\\\\\/jk26dxjEExf7N\\\\\\\/9jMkIw+dUkOLuWu7h1s3K0aU9h4HZCbyUCsoC+6QCT0XpR7VgpEZtU+KCH\\\\\\\/6iMjp7vEzdw5G3lCXevXSSI4EAxAoWKBrteOwB7jqPf9d6Wg58hbP4BzYloDaPtJefzFp0Vl+bC7sUG7M4WYTcEjjhb6SlF07E2Um88miWXDxIlYjTJUTHYkIw\\\\\\\/4dV3dUMnPBZVPX\\\\\\\/L7Zd3s7mkLXPT8zt9b84WwRwKlcjclqgKFOjQtslHF+lS0K\\\\\\\/ZKwNMgdfZ0Y16Vp03UaRTppONZeUUjvF5oMrgn+pDkHMqslqEeruxOtEBqypbdKKZjsSAEZpxWFwh8v4kWabSDXoXkT+A47X2j1RGsbL016fl50LYsKmWzrmv\\\\\\\/ko2Gv3eJwpula+PgZgca7bS7EwQe9+xDuRpPJ\\\\\\\/uBmu37cPC8rW2A9hr\\\\\\\/DJwmLNDsFMgzsYD72IU3R4Ibic3Al4ss7bxJkXZBLKD5vpJdObMlwjALtWqZq47Nd0WfOu2iEkZ9Q2t2m7bJ3k5Uh0a2T+MMo+fypUL0hcvt8wOEYzw0SYtSM6drHMg76MbND9UkpliAJxPlrkaFKZv3yJ9irIxjV5RfCcx7zn\\\\\\\/1XZ6qeE2AIoIrzXHMKR5I+4Y79qo3PoRq1zXeihzlObeCogdS6VjbPaAGNU\\\\\\\/pFWDI1x3rKSFTPvR25cC1BlOTMFi\\\\\\\/YhcKy9KbWa6HYyDCw2WiWnoRgr4oZBJN6XHs3kqwfmYK9zYHT7QzpnXrB1syTvG1M9qjXPdUYZUPry3h4sW57oUsRelfCtHmaJ5u3JuAzgeoBRaKWk2ct9gp0uMlFkwum+MmzwI7vuh\\\\\\\/6J9bqb5btS5RJCFkDnHYclHaA3v1Pa\\\\\\\/xg5M1L7yrLFfXXN5YbnAOFzrZHGu+fibS0nlKZ3oc8dFyDQFFeIwZfKyxFSkZhmHvUnYZmZIiBtkfXHg0UUItIUNIk6PWVqt0EJU0K9lH1Hmjh4eAal\\\\\\\/Nb\\\\\\\/EawIw81sBJx0v52yFF2ffT2X5F8Z1gPyam3VTXIEW3oyQ8wWIGYKyxX1uTIpusfKKBe0qEPrGodb\\\\\\\/oahtDVpyAMteJV7CqTzMTcLI3h0syYgslbY8oDRRR1Ru3Iuk\\\\\\\/q3RsnC\\\\\\\/xQmbC7SSDQ++77eTIv5LM8IMlw7hJrOOxsgbJvARlqVQsuoP4cYqgBVEyFif4iVJTagUFnnMUohj64sJIomS9s+VbFPagjd3kN13qQjTGdOKaeTQ1x\\\\\\\/RLvnQ4qTLEKYKfFXPrQyHmvrG7bWQ\\\\\\\/jQcoUOoB4T0d0N4apmXkSlD4M1+YFm19hvcbMTAxZPDdoOBykxzufuLcmzgU9zKLHW\\\\\\\/EFGI99FZj6OZwdFHafFkGHI3B98PROUhIZQJ8LTPgaxED5Vib4kb1P5DdJujrVmuFJWroQS\\\\\\\/I8oGNOXWUyjyHhRHQjKavlyj\\\\\\\/BSL8PyI2DpBhddQmF2gY1p+zzz7v\\\\\\\/+RJdJryoSuVBRaDvOTFRPfj876yV15Ve2vey1nc8830CEkqi7RcrZHsjtmUDhl9KPzC+pC7KGdNI\\\\\\\/XscqpxhD+rLcgIzXPKOloFAYEIp3EbqDG3HPRvJCSQd07unXXp4qKyL38t+j7Trq77AUQoZXbLlD2h+BoPfxknVfbANp+pnlbouXNUsquIOlcqMAXPRbN6jV4NsXMf2mpdynxonc61Afq9uNV6gxyri2EQmZAIg89oYniHTgXvsys24Ww\\\\\\\/zM0N0h7XjtE1JeJFQ1FMcD4ZES3LkIwR5DY\\\\\\\/IwhkkVvtAlOPA5yz2ndSMbpW9ecicJJAK7LN844OQuN8BChPK+8rXcFRWiRlTcQiEo2gu3Na6A8SfC9nD9NynQwuOl1fmFoP0z8fmSnGidugOfyDuAyPrI1+EqdtAst6ziDeEtKjNN4AXJDLehcEG15GEv1kJUcOuFahS1udO9iTBYcJ8+4\\\\\\\/His+J6A56hr1VuWphBf4uTM6bAhpLvVWqppUInc+ZZNVo11\\\\\\\/1gTuEc0v98vb9MrwK5egt9G2rborPZbJphps\\\\\\\/V4fQ3fjEU0CSaL9oRbVPM6rAl4vtkutBt9DYPAfl2MM6whzaLXqLg5ikq2sZkIPSb6+jkgQleQC1U23txVYuw1xQd10NHpYQfG0QMOY2\\\\\\\/YN3acz\\\\\\\/Cpoq5snsilq2QgExtoYRj20SpkJhLCLHD4bokzfG50mX3MeXMmmjMR9Ybza\\\\\\\/NV5jgAcZddURC3ajMSUFcsnOxc1QrDhmkZbZb2fNeH2jssKoUkiii8cq5C1L1zBFFKZSYQNvfI8aIt5YLOYJqt3WfJebeZosa\\\\\\\/qz+2ZAcEpALEl0Pmrfq7tGygOBjwu6+aOro2gc\\\\\\\/W+b\\\\\\\/Cnzu+dFJwKBwO\\\\\\\/j8Vn9XzwenCxx2q3BcyN0xIv0+aCAi+AQ5TBagk1J\\\\\\\/3N8KnNm03tiZmDn\\\\\\\/eDBaO8A8eNTSKCG7\\\\\\\/P4OstKd+TT4iLX2CAMWb6ccsr\\\\\\\/Wn6yBYh9tTc7tDiga5hHHOFlCdPNhc6Zd8GQYtgE5J3RCRX2f\\\\\\\/Y7Fq75jwHzF4eSV2XhTtCZ\\\\\\\/ZDkSlgme+t8JCddTwsM2WbXRVQ\\\\\\\/371o5tKHjyg2x\\\\\\\/mz1oHw078Ixk2R1t4vh4H\\\\\\\/1ERTwwTKM1ppEGxtM30XE+82hnzl5r+4UkQ+XUAa3Ilszk3wNkN5yfr9ppNj5MCquktrlVJoyc5\\\\\\\/IXUlr2g1YaRO6rSV9NsaHVQKvSsccXpLuV3Mi4kMhICnj7Y43YG+7a1m+vB9Ka6Kim5+9t9\\\\\\\/HqVHiZD1Zhl1YgVHSPuSulJTtz2l4yJIuU1vYVerpgqPKwotZPMvmxs9QN\\\\\\\/WUoC53BpSOeYrEPr5KaerjbLspLdZXbOkvX0kviDRnzO8pn1R4fHd81vw2kEl+DpvXpTOq1yvYR676XjmwtzU37Ol4wWCnk0TztTfzg+sYlTk85uEfY\\\\\\\/8tWFpou2NU1iHBm6IhQuh\\\\\\\/+U6BSrEnKFdwLFU6K6YGx41pNfuki+fizt2nWdG+NDbLusSSkdV93SE+Q0CM38Xeqqw7qnTQs7x6J1L5mT9aVUS4qJ25ScxCWnmWqBG9uOis4Vyrj22uG\\\\\\\/hWZ4xvwAib+noOTnu+JjErKrdB6k1CYYEpWTEIiZ5fTZj9uZeE0dAQ149RaZdKtNswqRVCxjRGmcXdxHKTGL+cqKKBrBjVWZaOVXCpHv1y\\\\\\\/l5ZuB8fKIOgHXp5v2PQ1T1Dx9sooIP3VXDt8UkYZxqehazkTwcgUxqGml1vStAAsJtAaMeOsVreu+\\\\\\\/WpMozQmpcq6sYxwh\\\\\\\/ft16d7uEA8iaJSH4hf4Z49l9JyRt0b3Relzd37LRnllxCZH5QXsUYaSpJXWdalN\\\\\\\/xmrgmvMNrVdol7iRmPHIKHSnmjDfAq3yL7F+5Lm4WICbA5UO9H8R8wch+ZL9Okq83BMBCvrSQRcwCFRmXtTGF7QPvnW0H4SmarD45E94JAM1OBTCDpR+x01u9+A6rXaALadxU6hE7cccdkosQrgmpLs500AE8Q84Ncggb9oeb\\\\\\\/DIRgasPCXCpVZh\\\\\\\/pJYsNi5MK92DysrYssUZMvWDWD7n79EEpfX2smbsTHh4lgd+a3i3BvJPSneCgsm\\\\\\\/MCxgOl2lmK1CFRN7okn+aDqVnVH9WidwSIoqEaxQGi5VrjkQ338srJM9CswR8\\\\\\\/xR4PD4vC1nMm9bga69pSYhnEawxN4gBNd\\\\\\\/blgp8aixZIlZSfUiZFNynt8u2NggjF8Db6LArNKuOWh2gn280FrX\\\\\\\/e5Hu4sp1tKxeQozfIlqEWGPnJSdAr8m0EdIXD8XmYijnBAS5+KIwx\\\\\\\/\\\\\\\/FCPCuNMT07a6HQ5xuTJ1t0Ad4exQ4IjVUF8ql3pOJB+i9V6owvlsWDhRNNHE2UCPKNT6n7gIWm7FpFQv9gfGpFi6ukq5ZrK8agF7\\\\\\\/sQ4BCV15pRXT02gTOM6bpWyyfBOWeIVJ9YMAAps1HjEFB57ZB259C5iepNZKBqkThDwBzgOf9hRQQC76fYn4T8uwvwLpU4gc1ahay7eLF8prIYm5vKh8x984obITivVF7+82dUQCYpFzgnOEf24VJUiUh8fIzmQxek2uCm88I49lNQwO6G1ER1Wkow\\\\\\\/5AD2HCK\\\\\\\/uC4E5i8DZJ+sTjq6AxtgrhS9n+3wsMO0hqHzcgqQm9uqNXdFE4nRcBPWvf7+1GYkhpqGp0kka\\\\\\\/jhSH1nGj+6iwzvGmnY4I09r5W6J4cVTRed9QuEF+48p1hhRkEoQRuYU0ywyTEeFDiMxKIJxSTP10vAiuHoOhGmPzhwwMpQIXsS4ZTLLnqXi6QPpdPVE5yIL2\\\\\\\/kCLQZGHmEXSr5Ds5nyF2XymBOIw7ILfDqVP2YgLxD+Hnkc4ksfv8sMuHWWpPJFjW1e7drhGGQRa7L\\\\\\\/zKT4dKPCs+5Y3GgVJkfnGKkJPl5dcHT6rQnuZIl0FH9QQTws4J2PPOQIklIh40DwZQ1qzEaiCQDKQJvNRuhGStgTwSqdryZJB3qS6GyxgApppmTkOa04JDve9x+PGxanKNzyi5gnqWPifMRB5uDZf3bINd0NeR94vVwjgdWuWsvaqNtAXuEk7THPfO5otO4JIVwo1zvd7ctwMnzIpQmUFE+SV8zQmqwfXE7YsxT4p8Itjl91BJ0qxLjcXp8v4ejuAWj34Io0Hj+FNAhOraQ8SICfePtG+jK4nJn\\\\\\\/ptJjuZhrF2tG8sldAmUkEl3bhr1O5IBr+Py3aXLaplcIXJBgs7bXl6hz7bMZfWNILrwOHh+HcqfvXen\\\\\\\/KYHBuFImv2373bQsFugBw4SVyTm3PLKm4wIq85LXwyWXpG1MwhVL1cWACbM4QQg2V1zT1N4glF5hO0c0z5AtolZ464cproqK7d5\\\\\\\/h8p3N0VB\\\\\\\/ICJYMKf5b4BASB6fGlHVBsWWUxy2JeC9q5d2fm4T7dkSw6N9V5r8qAvuBgPumIYN1QBR9aJNL89f5q7sumSgpCrZ7jvC6CEot211OfvgsNCzBFCl6l2+THnfA9EjFB1JM+86wFF6Y5vYUgANcdlkpjvAikoTry9pTR5tva3HXoo6TkDIDf7Bi72vSwSPiz3EXWomHYUgDTyCIECafXl+EDgyoMDJL2ZaWZN\\\\\\\/9RTiF5OkT3ORsweElNXS2dkn+WkHREeQPwKaFWj+oXYwy91V5YMckkl1nBrjwLEQ6r5gxQReKjAhUJNzCRA6RNkiedeBQCwXSfPoXq9DYewVKk2VS2msPLkT2\\\\\\\/NzRYfCuwid1WYHkVhf\\\\\\\/3VHYte9w4PTK93uv+3GXqfwq\\\\\\\/by8VMFWW2UKwylkKf3W+M++CjSu8477vGuUlWe8mhd112+Al6xFhEpllymduQcFQ78in5L2joRH4K00ts+TYDLZrgkewk2C+CQZ4FkAzBheh0NA\\\\\\\/udpNW7Mo5TVkW65lkQO+w8DVk157UL4Mtq1J5+9IZ4OtyTLDUnj48deRyHjROWqtCGgcqe6G\\\\\\\/eiNor7BtOtgroqPQ+OpHN2dnmO3n6HShTGaYyK9p2PhHqa2kt5HrTf70NrN75fDw\\\\\\\/vxbVtexY2\\\\\\\/Tqum41pbWDz5f24ViEjhM4nH6YfuPVzhGCGEAta8yGTXdzx9zmXfPbPaE3z8MOui6WCtyFbA9gfXyWRqgUkgOZqp2BdqVFvt5OjhPbfIq0XyC0mV+dbYSyUP8dWGXAM0IeTjNW6Y6YVdXaJfkgUEddQeoiVVRgyHnq071YB0ZU\\\\\\\/L4xS5j3F4ZnkCBwTQJyztJWT5kt7GzPvIAi1\\\\\\\/s\\\\\\\/K5joWg8KTP0XwLA+R1zHzG1n3RbGzfwvy+DdcJJ04ROKMAPDh8eGuHEldDFJ7bAidUpP4lz63ejxBgKG\\\\\\\/vl7e2qPZCGCM6J+Km5n3V2jEnXd7RQkPOvLc++BQwT+fmzvq7iOIfOBePUeYihzIhQPtIxK\\\\\\\/al7mCQUjQPlifs4lhgWaczVyOzQankLxGN0y2eaF4PUZh6ZJvcop5fBe6EijKTl+qmTbmQwnatdhHnwqTLex+S2rFbOgQSWO+UPIsDt9wcw9ge3C4h0qopSYwiknpWi3aCkONSOGnQstyfiFd\\\\\\\/H4ZRRDNrRuBLQQWdsIDxgLGCYCnZ2yOkVaiT3rB1cIewd52rBoJiTvASjoErjjVz\\\\\\\/rb0\\\\\\\/nJhCrzwXBwxCZ9Ot313\\\\\\\/Zx9CM76ghkhS+bGnOJJIFfNIUapflxDJLy4ZkXx+3xeUfaVFQ5Vsq9Xf7rlYtHFdqyTq7s4ikcwqOzBNDTkyVRBpSxc2ajetemlwq99J+zzIUTSbKFC6UWio6WilQpG8JnIdqnaI02MsM1zdLcNnvGgQ2Jc8b3cPaz528RQpE1t8VATK7hztTdj0Yz8voKzJ\\\\\\\/8lse8m9LKvx\\\\\\\/zIiyV0iNzEBsnYQcitQHGn\\\\\\\/9\\\\\\\/G0WfJQRr2KXe7PiOtWdV3Y\\\\\\\/POVlRIPlgkIGAuV5ybVJfRwYywWcUMcw+tI+ax6ufejL0gG67mVdPMZch+gAqcv1lh35J9442UPXLLmELFsIIizKpxUOoXgK9xggGaGhykfZ4Go\\\\\\\/G2MZgTXXyOkzmF79IDKd3P9+lcPuPVCsyL3E2OjEa5dJORAM+LRWVkhYyjJnfjjp3vLWNYBn9jofpg8lDkKLaquXlXs0M+efC4wU8EO6sapcawZjQHDwTe04rB\\\\\\\/75hXSsn\\\\\\\/7GSj3v987hr9rzWr07vQM5IvjQ3obIAxzGCn\\\\\\\/xJH6e\\\\\\\/JkWSOKHLVefl7ZvYty2rqOK5SolU7tZGNRU\\\\\\\/9hJTXZyfYxd+zuzMZKxCI1UaFlos9G47140\\\\\\\/Rw9QKpoLiV8F7ju3Phmb6ayspiYJK8326jSiKqMekfVaaSuKPJBeoxmpPHWjqyXnZdJD5U5QhpF7S+OyaMmI2vIUMIzdda9rG9aAr\\\\\\\/aPrl8KSCuty0+bb7bweyqmutWzAwcyXoynBLHeF2dDXNfgkX5OahX88BIMlfcSD8Sv2+J610HY66iaqM7QR9RSxfcylHkJkcyzaq730mc8Nt1FrmY72NZ7hdN4ybP9o09vVXLYM+LCthVRj7UJR+0+Q9zPvHnRugf8GYWCiA1x9QCSWGHL2\\\\\\\/hzMCOfjF3bN56imNyvg8+2kVlUj8Y5aqNXrPf1E8yTHz+TFUEvkFh\\\\\\\/phMV8LzEImxnfP+Y8Ll4rjaT7gM16nwCrCg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022nHLx25WnMZ63JqxcPbkxxQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222b84112cfa7584a376c802fec390f459d20215246d99db03954ef3ac938ac788\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kHZCEcDx6+yOxll43syLqpXDHqyVUI8tF5gV0oeWwmnVaZN0bzOQyOV6ZbuG7xwqn7QBs0DPbWPCrpiY3oYf+Dhn3DSwZVTpZdFI33SQMqJT\\\\\\\/s6rxz21jUZw0frfo\\\\\\\/H\\\\\\\/zCx9ojuLBlh9icye14SRq5uhmB+Fu0yYd71XoVhuoBqGTEkThHRgI\\\\\\\/5wd5V5gy1I+doQeNYQ9KHLeEm674CbJv5tGOfGwD6jH3ZAAGhEP8wQPwGlgL0Xiv+fE+eGN4KWMNnF08BnCGoIKuupHY6VnoQrid0dJIQkh1JXqCdVBxGdvW4SwmhIezFba6MhWUIMdFv53XPJ2p9L3T2CqzOC9IpniQLUrYhPjdlR4zW+kt82VMrafgY2EPPhZc8It0dyrMSS10UWPLow2JfEOX+YRHLKSBq6SPLMWOceDQ7YygpAVTSyUVjBWKb1BMvqJPXoXcTu9jq1H8QZMYZiK5Sh8J6mKGJDZNyTPS9Mpl4dkVS7o0JU0meZkWHYgnhp21iGlbkUhud16Zjc3qJaC56hgh+AB3s9vqi+QU3SJKCMaHvqfgKP4DS5jW1Eqk7zxNRDsRmsJxaIwBmN3cX0HgFDh5WoEHOTuA1Sky5nBApLjBktw4jMLJfDKyvE81CEXeflv1tNxJQMdQLqtF\\\\\\\/HcGt+GLG1NamoppOo0BTnZI8\\\\\\\/+0i2by\\\\\\\/DpILiFdl+OAWr+sxEMejM0b4aN0F9xNno9L1rJzSSDtL6L2kqlWXe65o59pd7HvS7wVNgx\\\\\\\/SVlbZ800j8nJH2XdLqgdBlznAsucobqt0p2dmbn4lLwvIKN2sGAgjhODKh5qLmUyz+p4GEFicNAW17CIe0MWcktMhN0LETWlR3wWGOC4lUZIWqqLpg0ol06zgSj3kqe1B4oMnOjblvCQZSP63hCQfrij0xM1q3mJDbIrfmEkUwDNOXfeylVmUHmtBRFhLxBEDU\\\\\\\/3Qbur3zRJBFWaYmqnC0lEmTEf0xc8OV9hlziMonIxLDby2BPmn0rJPywp4CGQhTh07i1MZWFLlpbB1KoK1hrN\\\\\\\/vbLHqI7FfZy6KzoRzp0J1xIymdjSXQNNbSmA1XQZeUdDQ\\\\\\\/pSnd2hjt26t8nYhuQdPKOTgZ4Haj2BXWg5pDoaTM7PbccT1NAQIPyjOGQse19pOeERKVEeZeKqLTUeQl5ugqJ\\\\\\\/oOOqqkqVdLG\\\\\\\/n89siQorHu8B7QH8y7DhxDTEqFWDkooBeIlQWomrjkC9W+KEPrf\\\\\\\/gx8TUaHXCsJxdjhyP50opa\\\\\\\/ag6tAxXfyzl4eYy476T+REP4yRpZMkDWvKinMWt3vw3BRRC2xu51wrXqjdBK6+oM68RS0DbubCia1e1jUoc4dV7DVBe+bd8ZL8b4nn1c1eqytlqKWKpIWEycR4kzAAO+HbpzrgfqE4gM8+KD3LMfu6lqEReZqhV1gk6mzw8D8fvSjwWILpBiEuQefdWjXwIqeyWjRw\\\\\\\/FA2DWnII+4C\\\\\\\/ut3bqngLskyq6rYOmswB6XsUqnyxMt0Sxkvrm5odxikksnB7KaoF5g0bNgz\\\\\\\/UoSIWSAHcEzADaoFLoGvy6TA49JQrCzrS8\\\\\\\/pCLDLxaKsYOinkBwOIdr5Z9AqyyN7b5Jj1AI6+8DQ766iuWbZ1xBCQGmxzsuBaNsKJwCI76Ug0gGU5bvRjCtW6SsxjA6fk9Q401W9dguAbc2HD3wmuvJXVarNNqQwIMFDANTzlBlTeTRLiO+ULnjgx8lQMyLGD7rRC+cEMBBR8+5MOatHYkqlw8JgIc+yaxcSRUKeOKFfX3tFXVlSf3QYzPuF83OftfkJZgE4Nv\\\\\\\/J4Q00RiVO9ar+MdgyGp8niMDpcnspAtxjBm6nJ3OzOdR64TGXyDwgyls5WG440LbObAiRhCTlkAlM5kaK1b3pFCi1GTfcG8s4gLAAr3YuUnqDQhFIrIrwWXwC5wXRXYJV\\\\\\\/zZBtY\\\\\\\/ZEF\\\\\\\/pbP8ctgGGHQpJxJU8qLRlCGKXKWM3ybZrHKqvLr2ce1mkAC1g0YAi55SYgWOKCxjK4bRAw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022OzMJTfqzwCDTFvJQQlNmNQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224707cad2a1c47e8546683a9808d7eeffdf38a97434a665b71afc4539950d78a6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485855 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2779" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iHZ0jmgWTQ1-oSJWaRgdCUBZdFuJ6BqLIYaoswjIBj8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2779" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2qz_ojD0BFAy5BcdzYlVOsQst8jiQ3DAsFK46_5PNZA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2763" title="McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2791" title="FlyExpress 7: An Integrated Discovery Platform To Study Coexpressed Genes Using in Situ Hybridization Images in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485855" id="node485855" data-pisa="ggg;7/8/2779" data-pisa-master="ggg;g3.117.044263" data-apath="/ggg/7/8/2779.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bayesian Networks Illustrate Genomic and Residual Trait Connections in Maize (<em>Zea mays</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Katrin Töpner</span>, <span class="highwire-citation-author" data-delta="1">Guilherme J. M. Rosa</span>, <span class="highwire-citation-author" data-delta="2">Daniel Gianola</span> and <span class="highwire-citation-author" data-delta="3">Chris-Carolin Schön</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2779-2789;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.044263 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485855" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katrin Töpner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, TUM School of Life Sciences Weihenstephan, Technical University of Munich, 85354 Freising, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katrin%2BT%C3%B6pner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=T%C3%B6pner%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatrin%2BT%25C3%25B6pner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guilherme J. M. Rosa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal Sciences, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guilherme%2BJ.%2BM.%2BRosa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosa%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuilherme%2BJ.%2BM.%2BRosa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal Sciences, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chris-Carolin Schön </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, TUM School of Life Sciences Weihenstephan, Technical University of Munich, 85354 Freising, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris-Carolin%2BSch%C3%B6n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sch%C3%B6n%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris-Carolin%2BSch%25C3%25B6n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chris.schoen@tum.de" class="" data-icon-position="" data-hide-link-title="0">chris.schoen@tum.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/8/2779" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485855" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2779.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485855" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2779.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485855" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2779.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485855" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/8/2779.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485855" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Relationships among traits were investigated on the genomic and residual levels using novel methodology. This included inference on these relationships via Bayesian networks and an assessment of the networks with structural equation models. The methodology employed three steps. First, a Bayesian multiple-trait Gaussian model was fitted to the data to decompose phenotypic values into their genomic and residual components. Second, genomic and residual network structures among traits were learned from estimates of these two components. Network learning was performed using six different algorithmic settings for comparison, of which two were score-based and four were constraint-based approaches. Third, structural equation model analyses ranked the networks in terms of goodness of fit and predictive ability, and compared them with the standard multiple-trait fully recursive network. The methodology was applied to experimental data representing the European heterotic maize pools Dent and Flint (<em>Zea mays</em> L.). Inferences on genomic and residual trait connections were depicted separately as directed acyclic graphs. These graphs provide information beyond mere pairwise genetic or residual associations between traits, illustrating for example conditional independencies and hinting at potential causal links among traits. Network analysis suggested some genetic correlations as potentially spurious. Genomic and residual networks were compared between Dent and Flint.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bayesian-network" class="hw-term hw-article-keyword hw-article-keyword-bayesian-network" rel="nofollow">Bayesian network</a></li><li class="kwd"><a href="/keyword/structural-equation-model" class="hw-term hw-article-keyword hw-article-keyword-structural-equation-model" rel="nofollow">structural equation model</a></li><li class="kwd"><a href="/keyword/multivariate-mixed-model" class="hw-term hw-article-keyword hw-article-keyword-multivariate-mixed-model" rel="nofollow">multivariate mixed model</a></li><li class="kwd"><a href="/keyword/multiple-trait-genome-enabled-prediction" class="hw-term hw-article-keyword hw-article-keyword-multiple-trait-genome-enabled-prediction" rel="nofollow">multiple-trait genome-enabled prediction</a></li><li class="kwd"><a href="/keyword/indirect-selection" class="hw-term hw-article-keyword hw-article-keyword-indirect-selection" rel="nofollow">indirect selection</a></li></ul><p id="p-3">Selection exploits genetic properties of a population by passing favorable alleles onto subsequent generations. In breeding of animals and plants, it is crucial to distinguish between genetic (heritable) contributions and environmental factors influencing traits, and the decomposition of phenotypic variances and covariances between traits into their genetic and environmental components has been under investigation for decades (<em>e.g.</em>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fisher 1918</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hazel 1943</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Falconer 1952</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Robertson 1959</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Searle 1961</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Roff 1995</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lynch and Walsh 1998</a>). Identifying genetic relationships between traits is an important research focus; an example is a study of genetic correlations among 16 quantitative traits of maize (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Malik <em>et al.</em> 2005</a>). Insights into the connections between developmental or phenological traits with target traits such as stress resistance might assist breeding decisions. While such relationships may be antagonistic when two target traits affect each other negatively, they can be useful in genome-enabled prediction of traits when a complex trait is influenced by a second trait that is easier to measure, assess, or breed for. In addition, it has been found that selection for traits with low heritability can be enhanced by joint modeling with genetically correlated traits. Several advantages of multiple-trait prediction models have been corroborated by experimental and simulation studies (<em>e.g.</em>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jia and Jannink 2012</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Pszczola <em>et al.</em> 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2014</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jiang <em>et al.</em> 2015</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Maier <em>et al.</em> 2015</a>).</p><p id="p-4">Discovering and understanding phenotypic relations among traits is therefore important in this context. Statistical methods have been developed that specify connections among traits as directed influences, in contrast to standard multiple-trait models where connections among traits are represented by unstructured covariance matrices. One effective approach, structural equation models (SEM), can describe systems of phenotypes connected via feedback or recursive relationships (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gianola and Sorensen 2004</a>). SEM are regression models that allow for a structured dependence among variables, <em>e.g.</em>, by a structure matrix <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-1.gif"/></noscript></span></span> Elements of <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-2.gif"/></noscript></span></span> represent effects of dependence and can be either freely varying, or the structure can be constrained by setting some entries to zero <em>a priori</em>. For a given structure <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-3.gif"/></noscript></span></span> the effect of one variable on another can be estimated using likelihood-based or Bayesian approaches. SEM are well established and many studies in the animal sciences have investigated causal relationships among traits using various SEM approaches and data sets, <em>e.g.</em>, body composition and bone density in mouse intercross populations (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2006</a>), calving traits in cattle (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de Maturana <em>et al.</em> 2009</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2010</a>), or gene expression in mouse data by incorporating genetic markers (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Schadt <em>et al.</em> 2005</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aten <em>et al.</em> 2008</a>). Methodology, applications, and potential advantages of SEM in prediction of traits were reviewed by <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rosa <em>et al.</em> (2011)</a> and <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Valente <em>et al.</em> (2013)</a>. One of the limitations of SEM is that connections between traits must be assumed known to be able to estimate their magnitude.</p><p id="p-5">To investigate connections among traits, Bayesian-network (BN) learning methods can be used. BN are models representing the joint distribution of random variables (<em>e.g.</em>, traits) in terms of their conditional independencies. There are two main types of algorithms for learning a BN: constraint-based and score-based algorithms. The former uses a sequence of conditional independence tests to learn the network among variables, while the latter compares the fit of many (ideally all) possible networks to the empirical data using a score. BN have been used for many purposes in quantitative genetics, for example, to predict individual total egg production of European quails using earlier expressed phenotypic traits (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Felipe <em>et al.</em> 2015</a>) and to study linkage disequilibrium using single nucleotide polymorphism (SNP) markers (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Morota <em>et al.</em> 2012</a>). BN have also been employed in genome-assisted prediction of traits, with a performance that was at least as good as that of other methods such as genomic best linear unbiased prediction or elastic nets (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Scutari <em>et al.</em> 2014</a>). In addition, many studies have investigated connections among several traits via a BN analysis incorporating quantitative trait loci and phenotypic data (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Neto <em>et al.</em> 2008</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">2010</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Winrow <em>et al.</em> 2009</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hageman <em>et al.</em> 2011</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Wang and van Eeuwijk 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Peñagaricano <em>et al.</em> 2015</a>). BN can also search for connections among SNPs and traits for feature selection, or in genome-wide association studies by finding the Markov blanket for one or several traits (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Porth <em>et al.</em> 2013</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Scutari <em>et al.</em> 2013</a>). However, interpretation of BN connections as causal influences is a delicate issue. Methodology and issues in causal inference were reviewed by <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rockman (2008)</a> and go back to <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pearl (2000)</a>. Within the scope of the present study, BN are seen as a hypothesis-generating tool with respect to the causal nature of the connections found.</p><p id="p-6"><a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Valente <em>et al.</em> (2010)</a> investigated causal connections among phenotypic traits using an approach combining BN and SEM. They learned network structure from Markov chain Monte Carlo (MCMC) samples of the residual covariance matrix of a Bayesian multiple-trait model using the inductive causation (IC) algorithm. This was a preliminary step before fitting an SEM with the selected structure to estimate the magnitude of influences among traits. These authors illustrated the effectiveness of their approach with a simulation study.</p><p id="p-7">Here, we also combine BN and SEM analysis to investigate several traits simultaneously, but with respect to their genomic and residual relationships: arguably, genomic connections among traits must be different from residual relationships, the latter mainly due to environmental causes. First, we learn genomic and residual networks using different BN algorithms. The learned networks may provide a deeper insight into relationships among traits than pairwise-association measures (such as correlations or covariances). Instead of using the IC algorithm to analyze MCMC samples of covariance matrices, we use predicted (fitted) genomic values and residuals from a multiple-trait model as input variables for structure-learning algorithms. This approach allows using available software for learning BN, is computationally easy and fast, and use of bootstrapping to assess uncertainty regarding the edges is straightforward. Using this strategy, we study the behavior of various BN algorithms on experimental data. Afterward, we compare and assess the BN algorithms by fitting an SEM to the structures learned from them.</p><p id="p-8">The methodology is illustrated using a maize data set. Biological objectives of this case study are the investigation of connections among maize traits and the comparison of networks with respect to two underlying sources of information (genomic and residual) and two heterotic groups (Dent and Flint). Five well-characterized complex traits were studied: biomass yield, plant height, maturity, and male and female flowering time. With these, well-known connections can be reconstructed and new insights into potentially spurious trait connections are obtained. After illustrating trait connections with respect to their genomic and residual sources and identifying spurious connections, a clearer picture on which traits should be included in a multiple-trait prediction model or in an indirect selection process may be gained.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Material</h3><p id="p-9">The data contain a sample of phenotypes and genotypes representing two important European heterotic pools, Dent and Flint (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bauer <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lehermeier <em>et al.</em> 2014</a>). In the Dent (Flint) panel, 10 (11) founder lines were crossed to a common central Dent (Flint) line. Derived double haploid (DH) lines were evaluated as testcrosses with the central line from the opposite pool.</p><p id="p-10">In both panels, five phenotypic traits were recorded: biomass dry matter yield (DMY) (decitons per hectare), biomass dry matter content (DMC) (percentage), plant height (PH) (centimeters), days to tasseling (DtTAS) (days), and days to silking (DtSILK) (days). Phenotypic values are available as adjusted means across locations and replications. The adjusted means were mean-centered in each DH population, as we focused on allelic rather than on population effects. All traits were standardized to unit variance based on their sample variance.</p><p id="p-11">All DH lines of both panels were genotyped with the Illumina MaizeSNP50 BeadChip containing 56,110 SNP markers (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ganal <em>et al.</em> 2011</a>). After imputation and SNP quality control, 34,116 high-quality SNPs remained in the Dent and Flint panels. Only markers that were polymorphic within a panel were considered, which were 32,801 (30,801) SNPs in the Dent (Flint) panel. A total of 831 (805) Dent (Flint) DH lines were used for analyses.</p></div><div id="sec-3" class="subsection"><h3>Methods</h3><p id="p-12">The methodology includes a genetic and a residual structure that are expected to be different. The rationale behind this assumption is that genetic connections among traits do not necessarily resemble the residual ones, and vice versa; typically, genetic and residual factors are assumed to have independent distributions, reflecting control by disjoint factors.</p><p id="p-13">Exploring these connections among traits includes the following steps: (1) fitting a Bayesian multiple-trait Gaussian model (MTM) to all five traits to obtain posterior means of genomic values and of model residuals for each panel, and transforming the predicted genomic values to meet assumptions on sample independence required by a BN analysis; (2) applying the BN analysis to the residual and transformed genomic trait components; and (3) assessing the quality of the inferred structures by a structure-comparative SEM analysis.</p><div id="sec-4" class="subsection"><h4>MTM:</h4><p id="p-14">We fitted a Bayesian multivariate Gaussian model to the <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-4.gif"/></noscript></span></span> traits and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-5.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-6.gif"/></noscript></span></span> genotypes within the Dent (Flint) panel to separate genomic values from random residual contributions to the phenotypic values, and to estimate the genomic and residual correlations between traits. The MTM model can be expressed as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-7.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-8.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-10.gif"/></noscript></span></span> are (<em>nd</em> × 1)-dimensional column vectors of scaled adjusted means, genomic values, and residuals sorted by trait and genotype within trait, respectively; <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-11.gif"/></noscript></span></span> is a Kronecker product of a (<em>d</em> × 1)-dimensional intercept vector <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-12.gif"/></noscript></span></span> and an (<em>n</em> × 1)-dimensional vector of ones. Since phenotypes <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-13.gif"/></noscript></span></span> are mean centered, only trait-specific intercepts are included in the model as fixed effects represented by the <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-14.gif"/></noscript></span></span> coefficients in <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-15.gif"/></noscript></span></span></p><p id="p-15">The joint distribution of <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-17.gif"/></noscript></span></span> was assumed to be multivariate normal with the following specification:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-18.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-19.gif"/></noscript></span></span> is an (<em>n</em> × <em>n</em>)-dimensional realized-kinship matrix estimated using simple matching coefficients (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sneath and Sokal 1973</a>). <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-20.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-21.gif"/></noscript></span></span> are (<em>d</em> × <em>d</em>)-dimensional covariance matrices containing the genomic and residual variance and covariance components, and the (<em>n</em> × <em>n</em>)-dimensional identity matrix <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-22.gif"/></noscript></span></span> represents the independence of residuals among genotypes.</p><p id="p-16">Flat priors were assigned to the elements of the intercept vector <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-23.gif"/></noscript></span></span> The covariance matrices <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-24.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-25.gif"/></noscript></span></span> were assumed to follow <em>a priori</em> independent inverse-Wishart distributions with specific degrees of freedom <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-26.gif"/></noscript></span></span> and scale matrix <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-27.gif"/></noscript></span></span> regarded as hyper-parameters, <em>i.e.</em>,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-28.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>and<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-29.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>The hyper-parameters <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-30.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-31.gif"/></noscript></span></span> were chosen such that: (1) they implied relatively uninformative prior distributions of matrices <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-32.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-33.gif"/></noscript></span></span> (2) the prior mode of genomic and residual trait variances was 0.5 for each trait, and (3) the choice of hyper-parameters needed to ensure that there exists an analog choice of hyper-parameters for a scaled-inverse <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-34.gif"/></noscript></span></span> prior defined later in the SEM models for trait variance components. Considering (1), (2), and (3), we set <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-35.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-36.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-37.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-38.gif"/></noscript></span></span> in accordance with the parametrization of the respective distributions in the R package <em>BGLR</em> and the <em>MTM</em> function of its multiple-trait extension.</p><p id="p-17">An MCMC approach based on Gibbs sampling was used to explore posterior distributions. A burn-in of 30,000 MCMC samples was followed by an additional 300,000 MCMC samples. The MCMC samples were thinned with a factor of two, resulting in 150,000 MCMC samples for inference. Posterior means were then calculated for <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-39.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-40.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-41.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-42.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-43.gif"/></noscript></span></span> for the Dent and Flint panels separately. Genomic and residual trait correlations and their SDs were estimated from samples of the posterior distributions of <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-44.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-45.gif"/></noscript></span></span></p><p id="p-18">Subsequent BN analyses concentrated on <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-46.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-47.gif"/></noscript></span></span> separately, the posterior mean estimates of <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-48.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-49.gif"/></noscript></span></span> respectively. Traditionally in quantitative genetics and plant breeding, phenotypic variation is decomposed into genetic and environmental random contributions; genotype-by-environment interaction can confound both of these contributions. In experimental data from plant breeding, phenotypic data points are usually averaged across locations representing different environmental conditions, and every location contains replications of experiments. This approach reduces noise from environmental effects and also effectively decreases the influence of genotype-by-environment effects in the data (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Falconer and Mackay 1996</a>). As a consequence, residuals estimated with model (1) from such data include, <em>e.g.</em>, nonadditive genetic effects and nonaccounted-for (micro-)environmental effects, such as nongenetic physiological and morphological trait dependencies. Random genomic values fitted with model (1) represent, <em>e.g.</em>, genetic connections among traits, induced <em>inter alia</em> by pleiotropy and linkage disequilibrium. Here, we take the fitted genomic values and residuals and investigate them with BN.</p><p id="p-19">Predictive ability from a univariate Bayesian prediction model was derived for each trait with 10 replicates of a fivefold cross-validation to compare single-trait prediction with multiple-trait prediction. Prior assumptions were analogous to those of the MTM, that is, scaled-inverse <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-50.gif"/></noscript></span></span> distributions with four degrees of freedom and a scale parameter equal to three were employed for each of the genomic and the residual variances.</p></div><div id="sec-5" class="subsection"><h4>Transforming the genomic component to meet assumptions of the BN learning algorithms:</h4><p id="p-20">BN learning algorithms assume independent observations. In the MTM described above, independence of residuals between genotypes was assumed. In contrast, the genomic components in <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-51.gif"/></noscript></span></span> were correlated between genotypes due to kinship (represented by the matrix <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-52.gif"/></noscript></span></span> in addition to being dependent within genotypes due to genomic trait covariance (represented by the matrix <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-53.gif"/></noscript></span></span> Before learning BN from the genomic component, a transformation was therefore applied to vector <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-54.gif"/></noscript></span></span> such that the transformed vector <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-55.gif"/></noscript></span></span> was distributed as <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-56.gif"/></noscript></span></span> <em>i.e.</em>, elements of <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-57.gif"/></noscript></span></span> were independent between genotypes while still preserving genomic relationships among traits as encoded by <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-58.gif"/></noscript></span></span></p><p id="p-21">For this purpose, we decomposed the kinship matrix <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-59.gif"/></noscript></span></span> into its Cholesky factors, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-60.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-61.gif"/></noscript></span></span> is an (<em>n</em> × <em>n</em>)-dimensional lower triangular matrix, and define a (<em>dn</em> × <em>dn</em>)-dimensional matrix <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-62.gif"/></noscript></span></span> so that <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-63.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-64.gif"/></noscript></span></span></p><p id="p-22">For <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-65.gif"/></noscript></span></span> we have<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-66.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>because <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-67.gif"/></noscript></span></span> which follows directly from the definition of the Cholesky factor. This transformation and the resulting covariance structure was used by <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Vázquez <em>et al.</em> (2010)</a>, but for a single-trait model.</p><p id="p-23">For a standard Cholesky decomposition of <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-68.gif"/></noscript></span></span> a positive definite realized kinship matrix is needed. Realized kinship matrices are not guaranteed to be positive definite, in contrast to pedigree-based kinship matrices. However, adjustments to assure positive definiteness are available (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nazarian and Gezan 2016</a>).</p><p id="p-24">In subsequent BN analysis, <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-69.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-70.gif"/></noscript></span></span> denoted two (<em>n</em> × <em>d</em>)-dimensional matrices formed from the vectors <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-71.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-72.gif"/></noscript></span></span> by arranging genotypes and traits in rows and columns, respectively.</p></div><div id="sec-6" class="subsection"><h4>Learning genomic and residual BN:</h4><p id="p-25">A BN is a model that describes connections among random variables. It consists of two components: a statement about the joint distribution of these random variables, and its graphical representation as a directed acyclic graph (DAG) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nagarajan <em>et al.</em> 2013</a>). The statement about the joint distribution <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-73.gif"/></noscript></span></span> of the <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-74.gif"/></noscript></span></span> random variables <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-75.gif"/></noscript></span></span> is that <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-76.gif"/></noscript></span></span> can be decomposed into a product of conditional distributions:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-77.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>Above, <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-78.gif"/></noscript></span></span> the “parents” of <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-79.gif"/></noscript></span></span> denotes the subset of random variables in <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-80.gif"/></noscript></span></span> on which <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-81.gif"/></noscript></span></span> depends. Accordingly, <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-82.gif"/></noscript></span></span> is then called the “child” of all elements in <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-83.gif"/></noscript></span></span> In the respective DAG, all random variables <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-84.gif"/></noscript></span></span> are represented as nodes, and arcs point from parents to their children. The joint distribution is also referred to as a global distribution and the conditional distributions are called local distributions. If the global and local distributions are normal and the variables are linked by linear constraints, the BN is a Gaussian BN.</p><p id="p-26">Here, the random variables investigated are the genomic and residual components of phenotypic traits. We chose a Gaussian BN as a consequence of assuming that quantitative traits, and particularly genomic values and residuals, follow a normal distribution. Many tests and scores for learning BN assume independence among the observations of the random variables. In our study, the realized random variables were the genomic values and the residuals from the MTM, of which the former had to be transformed as described above. So we searched for the decomposition of the following global distributions:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-85.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>and<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-86.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where the index “<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-87.gif"/></noscript></span></span>” denotes the <em>k</em>th column of the respective matrix. There are two types of algorithms to learn the structure of networks: constraint-based and score-based algorithms (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Nagarajan <em>et al.</em> 2013</a>). The constraint-based algorithms employ a series of conditional and marginal independence tests to infer potential connections and directions between each pair of variables. First, an undirected structure linking variables directly related to each other is constructed. Next, trios of variables where one is the outcome of the other two (“v-structures”) are sought. Last, the remaining connections are oriented whenever possible, such that neither cycles nor new v-structures are created. For illustration, core structures with three nodes are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. In contrast, score-based approaches search through the space of possible networks (including direction of edges) and compare them by a goodness-of-fit score, returning the network with the highest score.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2779/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure learning: constraint-based algorithms search for trios of variables and test their relations with marginal and conditional independence tests. Thereby, they distinguish between the stochastic decomposition of the joint distribution of all variables as in (A) and the decomposition as in (B). These two decompositions can be represented in a DAG as shown. Whereas the directions are clear and unique in (B), the influences&#x2019; directions in (A) can form three different graphs. Which of these equivalent structures is most likely, one can only deduce from the context of other already tested decompositions in their neighborhood and from the rule that no cycles must be formed in the graph." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1036785186" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Structure learning: constraint-based algorithms search for trios of variables and test their relations with marginal and conditional independence tests. Thereby, they distinguish between the stochastic decomposition of the joint distribution of all variables as in (A) and the decomposition as in (B). These two decompositions can be represented in a DAG as shown. Whereas the directions are clear and unique in (B), the influences&#x2019; directions in (A) can form three different graphs. Which of these equivalent structures is most likely, one can only deduce from the context of other already tested decompositions in their neighborhood and from the rule that no cycles must be formed in the graph.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2779/F1.medium.gif" width="440" height="341"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/8/2779/F1.medium.gif" width="440" height="341"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2779/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2779/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486034" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">Structure learning: constraint-based algorithms search for trios of variables and test their relations with marginal and conditional independence tests. Thereby, they distinguish between the stochastic decomposition of the joint distribution of all variables as in (A) and the decomposition as in (B). These two decompositions can be represented in a DAG as shown. Whereas the directions are clear and unique in (B), the influences’ directions in (A) can form three different graphs. Which of these equivalent structures is most likely, one can only deduce from the context of other already tested decompositions in their neighborhood and from the rule that no cycles must be formed in the graph.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">In this study, we chose the Grow-Shrink algorithm (GS) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Margaritis 2003</a>) as an example of a constraint-based approach, and tabu-search (TABU) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Daly and Shen 2007</a>) as an example of a score-based algorithm. For comparing the constraint-based with the score-based approach, these two algorithms are ideal when dealing with small networks of up to 10–15 traits. For larger networks, runtime-optimized network learning algorithms might be preferred.</p><p id="p-29">The GS was combined with four different tests for marginal and conditional independence. We used the simple correlation coefficient with an exact Student’s <em>t</em>-test (GS 1) and with a Monte Carlo permutation test (GS 2) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Legendre 2000</a>). Further, GS was combined with mutual information, an information-theoretic distance measure. It was investigated with an asymptotic <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-88.gif"/></noscript></span></span> test (GS 3) and with a Monte Carlo permutation test (GS 4) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kullback 1959</a>). Multiple testing issues in the context of a constraint-based algorithm have been addressed by <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aliferis <em>et al.</em> (2010a</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">b</a>). According to these authors, for a sample size of ∼1000, a significance level of <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-89.gif"/></noscript></span></span> resulted in a worst-case false positive rate of <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-90.gif"/></noscript></span></span> for arc inclusion. Choosing <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-91.gif"/></noscript></span></span> in the more exhaustive GS, false positives should be avoided effectively with five variables (traits) and ∼800 genotypes.</p><p id="p-30">The TABU was combined with two different scores: the Bayesian information criterion (BIC) (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rissanen 1978</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lam and Bacchus 1994</a>) (TABU 1) and the Bayesian Gaussian equivalent score (BGE) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Geiger and Heckerman 1995</a>) (TABU 2). The BGE is a Bayesian scoring metric for Gaussian BN with many favorable properties. <em>Inter alia</em>, BGE assigns the same scores to equivalent network structures (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chickering 1995</a>). Equivalent network structures refer to the same decomposition of the global distribution with different edge directions (<em>e.g.</em>, part A of <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-31">In total, six (GS 1, GS 2, GS 3, GS 4, TABU 1, and TABU 2) different learning settings were applied to the genomic and residual components of model (1), respectively; all algorithms assuming a Gaussian BN. Regarding the constraint-based approaches, the number of permutations used for each of the permutation tests was 450 (slightly more than half of the genotypes in each of the two panels). Each of the six settings was run with 500 bootstrap samples and the significance level was set to 0.01 (as mentioned above). After structure learning from the bootstrap samples, we averaged over the 500 resulting networks as described in the R package <em>bnlearn</em> (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Scutari 2010</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Nagarajan <em>et al.</em> 2013</a>). Significance of edges in the averaging process was assessed by an empirical test on the arcs’ strength (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Scutari and Nagarajan 2011</a>).</p><p id="p-32">The structure learned by the BN needed to be translated into a structure matrix for SEM analysis. Thus, a (<em>d</em> × <em>d</em>)-dimensional matrix <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-92.gif"/></noscript></span></span> was formed from each learned structure. Each arrow of the DAG pointing from a parent trait to a child trait becomes a freely varying coefficient in the structure matrix in the column of the parent and the row of the child. All other entries were set to zero <em>a priori</em>. The resulting structure matrices are lower triangular matrices, since loops are not allowed in a BN structure. The lower triangular form might not be produced by an arbitrary order of traits but can always be formed by rearranging the order of the traits by columns and rows; see Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/FigureS1.pdf">Figure S1</a>, for an example of how a DAG is transformed into a structure matrix. If a learned structure was an element of an equivalent class of several structures, only one representative of this equivalent class was translated into a structure matrix <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-93.gif"/></noscript></span></span> and assessed in SEM analyses. Different SEM based on different <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-94.gif"/></noscript></span></span> from the same equivalent class may lead to different sets of coefficients. However, they lead to the same deviance information criterion (DIC) or likelihood of the respective SEM.</p><p id="p-33">Hereinafter, <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-95.gif"/></noscript></span></span> denotes the structure matrix of a network learned from <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-96.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-97.gif"/></noscript></span></span> denotes the structure matrix of a network learned from <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-98.gif"/></noscript></span></span></p></div><div id="sec-7" class="subsection"><h4>Assessing BN inference by SEM:</h4><p id="p-34">In the following paragraph, we present how the structures found in the BN analysis translate into SEM; doing so also clarifies how MTM and SEM are related. Genomic or residual trait dependence can be described by the recursive linear regression models<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-99.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>and<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-100.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>where <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-101.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-102.gif"/></noscript></span></span> denote the (<em>d</em> × <em>d</em>)-dimensional genomic and residual structure matrices and <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-103.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-104.gif"/></noscript></span></span> are <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-105.gif"/></noscript></span></span> independent and identically normal-distributed residuals. Independence among genotypes in <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-106.gif"/></noscript></span></span> follows from the transformation of <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-107.gif"/></noscript></span></span> into <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-108.gif"/></noscript></span></span> elements of <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-109.gif"/></noscript></span></span> are independent since <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-110.gif"/></noscript></span></span> are independent among genotypes by the model assumptions of MTM. If the genomic and residual structures in Equations 9 and 10 explain all genomic and residual trait covariances (which true structures ideally do), then <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-111.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-112.gif"/></noscript></span></span> are also independent between traits.</p><p id="p-35">For genotype <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-113.gif"/></noscript></span></span> the models are<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-114.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>and<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-115.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>Equations 11 and 12 can be rearranged into<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-116.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>and<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-117.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span>which imply that<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-118.gif"/></noscript></span><span class="disp-formula-label">(15)</span></span>and<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-119.gif"/></noscript></span><span class="disp-formula-label">(16)</span></span>where <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-120.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-121.gif"/></noscript></span></span> are unknown diagonal matrices. Let <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-122.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-123.gif"/></noscript></span></span> for all <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-124.gif"/></noscript></span></span> <em>i.e.</em>, the model is homoscedastic. Note that <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-125.gif"/></noscript></span></span> when considering individual genotypes. Assuming that both genomic and residual variances of traits are the same for all genotypes, the genomic and residual covariance matrices <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-126.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-127.gif"/></noscript></span></span> can be replaced by<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-128.gif"/></noscript></span><span class="disp-formula-label">(17)</span></span>for all <em>i</em> and<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-129.gif"/></noscript></span><span class="disp-formula-label">(18)</span></span>for all <em>i</em>. Since the true structure matrices <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-130.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-131.gif"/></noscript></span></span> are unknown, the inferred structures from the BN, <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-132.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-133.gif"/></noscript></span></span> are used instead.</p><p id="p-36">Then, the MTM can be reformulated as the following SEM in its reduced form (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gianola and Sorensen 2004</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Valente <em>et al.</em> 2010</a>):<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-134.gif"/></noscript></span><span class="disp-formula-label">(19)</span></span>where <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-135.gif"/></noscript></span></span></p><p id="p-37">The Bayesian Gibbs sampling implementation is as for a MTM, but, additionally, <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-136.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-137.gif"/></noscript></span></span> is sampled from its posterior distribution based on Equation 17 (Equation 18). The structures of <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-138.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-139.gif"/></noscript></span></span> affect whether or not the use of <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-140.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-141.gif"/></noscript></span></span> instead of <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-142.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-143.gif"/></noscript></span></span> induces a reduction in the number of parameters. For example, <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-144.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-145.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-146.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-147.gif"/></noscript></span></span> have the same number of nonnull parameters if the structure is fully recursive.</p><p id="p-38">In our Bayesian model, the diagonal elements of matrices <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-148.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-149.gif"/></noscript></span></span> were assigned independent scaled-inverse <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-150.gif"/></noscript></span></span> prior distributions with four degrees of freedom and scale of three. As mentioned above, this choice of hyper-parameters yields the same prior distribution as in the MTM with respect to the diagonal elements of the covariance matrices. Also, the trait variances’ prior mode is again 0.5 and the priors are relatively uninformative. The nonnull coefficients of <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-151.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-152.gif"/></noscript></span></span> were assigned independent normal distributions with null mean and variance of <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-153.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-154.gif"/></noscript></span></span> respectively.</p><p id="p-39">The prior distribution’s variance for the nonnull coefficients of <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-155.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-156.gif"/></noscript></span></span> was chosen to align with the estimated genomic (residual) covariance components in the MTM, which lay in the interval between <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-157.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-158.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-159.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-160.gif"/></noscript></span></span> When a normal prior distribution with null mean is assumed for the SEM structure matrices’ coefficients, then <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-161.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-162.gif"/></noscript></span></span> lies within two times its SD when its variance (a hyper-parameter) is chosen to be larger than <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-163.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-164.gif"/></noscript></span></span></p><p id="p-40">Predictive ability and goodness of fit of the MTM <em>vs.</em> the SEM were used to assess if the structures found in BN analysis were a better representation of trait connections than a fully recursive structure. We employed the DIC (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Spiegelhalter <em>et al.</em> 2002</a>), the marginal likelihood, and the predictive ability from 10 replicates of a fivefold cross-validation. The same 50 training- and test-set combinations were used for all models, including the single-trait model. Predictive ability was measured as the correlation between the adjusted means <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-165.gif"/></noscript></span></span> and their predicted values <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-166.gif"/></noscript></span></span> derived from the MTM or SEM. We fitted SEM with both or only one structured component, <em>i.e.</em>, SEM with <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-167.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-168.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-169.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-170.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-171.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-172.gif"/></noscript></span></span></p></div></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-41">Phenotypic data are available from file S1 of <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lehermeier <em>et al.</em> (2014)</a> at <a href="http://www.genetics.org/content/suppl/2014/09/17/198.1.3.DC1">http://www.genetics.org/content/suppl/2014/09/17/198.1.3.DC1</a>. Genotypic data were deposited in a previous project (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bauer <em>et al.</em> 2013</a>) at National Center for Biotechnology Information Gene Expression Omnibus as data set GSE50558 (<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE50558">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE50558</a>). Analyses were performed with the statistical programming tool R (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">R Core Team 2014</a>). Single-trait prediction was implemented with the <em>BLR</em> function (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">de los Campos and Pérez-Rodríguez 2012</a>). Structure learning, MTM, and SEM analyses were implemented using the R packages <em>bnlearn</em> (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Scutari 2010</a>; <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Nagarajan <em>et al.</em> 2013</a>) and an extension of the <em>BGLR</em> package (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de los Campos and Pérez-Rodríguez 2014</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lehermeier <em>et al.</em> 2015</a>) which is available on github (<a href="https://github.com/QuantGen/MTM">https://github.com/QuantGen/MTM</a>).</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Trait correlations</h3><p id="p-42">Phenotypic, genomic, and residual correlations between the five traits were estimated with the MTM for the Flint and Dent panels separately (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In both panels, there was a strong positive genomic correlation between flowering traits DtTAS and DtSILK, and between the yield-related traits DMY and PH. Genomic correlations were positive for trait combinations not including DMC, whereas DMC was negatively correlated with all other traits. The magnitude of genomic correlations differed between the two panels; for example, the genomic correlation between DMC and DMY was −0.29 in the Dent panel, and −0.64 in the Flint panel. Posterior SDs of correlation estimates varied between 0.01 and 0.10. Single-trait predictive abilities (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) were high and similar to those found in <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Lehermeier <em>et al.</em> (2014)</a>, where a detailed single-trait analysis of the phenotypic and genotypic data can be found.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485876/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485876/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Phenotypic, genomic, and residual correlations for five traits in Dent (lower triangular) and Flint (upper triangular) with posterior SDs in parentheses where appropriate as well as single-trait predictive abilities</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Transformation of the genomic component</h3><p id="p-45">As the genotypic values in the MTM were correlated between genotypes due to kinship, estimates were transformed to meet the model assumptions of the BN algorithms. The transformation modified the relationship between estimated genotypic values as expected (as an example, see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2779/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transformation of the genomic component: relationships between estimated genomic values of DMY  and DMC  from the Dent panel and their counterparts  and  after transformation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1036785186" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Transformation of the genomic component: relationships between estimated genomic values of DMY &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-173&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and DMC &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-174&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; from the Dent panel and their counterparts &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-175&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-176&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; after transformation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2779/F2.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/8/2779/F2.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2779/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2779/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486049" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-46" class="first-child">Transformation of the genomic component: relationships between estimated genomic values of DMY <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/F2/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/F2/embed/mml-math-173.gif"/></noscript></span></span> and DMC <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/F2/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/F2/embed/mml-math-174.gif"/></noscript></span></span> from the Dent panel and their counterparts <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/F2/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/F2/embed/mml-math-175.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/F2/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/F2/embed/mml-math-176.gif"/></noscript></span></span> after transformation.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>BN</h3><p id="p-47">BN inference was sensitive to the choice of algorithm and test (for detailed results see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/FigureS4.pdf">Figure S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/FigureS5.pdf">Figure S5</a>). The inferred networks varied in up to two edges and/or three directions. In general, score-based algorithms tended to choose more edges than constraint-based algorithms, and the networks of residual components were sparser than those of genomic components. Within the constraint-based or score-based approaches, inference was more consistent than between them.</p><p id="p-48">BN inference of residual components was very similar in the Dent and Flint panels, irrespective of the algorithmic settings (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/FigureS5.pdf">Figure S5</a>). Connections that appeared in both panels were those between the flowering traits, from DtSILK to PH, from DMY to DMC, and between DMY and PH. Specific to the Flint panel was the edge from DtTAS to DMC, while the Dent panel showed a corresponding connection from DtSILK to DMC. An additional edge in the Dent panel not detected in the Flint panel was between DMC and PH. One network in the Flint panel showed an edge from PH to DtTAS, but it was only supported by 56% of the bootstrap samples in the respective network algorithm.</p><p id="p-49">BN inference of genomic components was more variable than that of residual components within and across panels. In the Flint panel, algorithms recognized edges from DtSILK to DtTAS, from DtTAS to PH, from DtSILK to DMC; and between DMY and DMC, between DMY and DtTAS, between DMY and PH, and between DMC and DtTAS. In the Dent panel, only the connection between DMC and DMY was never inferred by any algorithm. A fully recursive network (<em>i.e.</em>, including all possible arcs) was never inferred, neither on the genomic nor on the residual components.</p></div><div id="sec-13" class="subsection"><h3>Network assessment by SEM analysis</h3><p id="p-50">For assessing the structures inferred by the different BN algorithms, we integrated them into SEM, and compared them with MTM using goodness-of-fit criteria and predictive ability (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/TableS1.pdf">Table S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/TableS2.pdf">Table S2</a>). When two or more BN algorithms inferred the same network for the same component (genomic or residual), SEM analysis was only performed once for this structure. In the SEM, genomic and residual networks were assessed separately <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-177.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-178.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-179.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-180.gif"/></noscript></span></span> (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/TableS1.pdf">Table S1</a>) and jointly <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-181.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-182.gif"/></noscript></span></span> (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/TableS2.pdf">Table S2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486032/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486032/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Single-structure evaluation: goodness of fit for the MTM and for SEM including a genomic <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/T2/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/T2/embed/mml-math-183.gif"/></noscript></span></span> or residual <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/T2/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/T2/embed/mml-math-184.gif"/></noscript></span></span> trait structure denoted by the BN algorithm it originates from</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486007/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486007/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Double-structure evaluation: goodness of fit for MTM and for SEM including both genomic <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/T3/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/T3/embed/mml-math-187.gif"/></noscript></span></span> and residual <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/T3/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/T3/embed/mml-math-188.gif"/></noscript></span></span> trait structure denoted by the BN algorithms they originate from</span><div class="sb-div caption-clear"></div></div></div><p id="p-65">For illustration of the variable reduction attained by application of the structures to the residual or genomic components, the number of connections in the networks and the resulting number of nonnull entries in <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-191.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2779/embed/mml-math-192.gif"/></noscript></span></span> were derived (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). For each SEM, the sum of the number of connections in the networks and the sum of nonnull entries in the covariance matrices were split into the genomic and the residual contributions. In general, residual networks were sparser than genomic networks, and SEM on residual structures were more parsimonious in the Flint panel than in the Dent panel.</p><p id="p-66">Goodness of fit was assessed with the DIC and with the logarithm of the Bayesian marginal likelihood (logL) (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). logL evaluates the fit of the model to the data and prior assumptions, and DIC takes into account and penalizes the number of parameters in the model. Models were ranked according to their DIC, which differed from their logL ranking. In Dent (Flint), eight (four) SEM with both a structured residual and structured genomic component had a lower DIC than the MTM (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). In Flint, no SEM with only a genomic structure had a lower DIC than the MTM (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). In general, DIC and logL varied more in Flint than in Dent (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>).</p></div><div id="sec-14" class="subsection"><h3>Predictive abilities of SEM</h3><p id="p-67">Predictive abilities and their SDs were derived from 10 replicates of a fivefold cross-validation with random sampling of training and test sets within each panel (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/TableS2.pdf">Table S2</a>). Averaged over traits and for each trait individually, predictive abilities were similar for all models (single-trait, MTM, and SEM) and differences between models were &lt;1 SD and not significant. The magnitude of predictive-ability estimates in the single-structure SEM was consistent with predictive-ability estimates in the double-structure SEM in both panels. Since all SEM with a genomic structure in the Flint panel had higher DIC (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>) and slightly lower predictive ability estimates (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1/TableS1.pdf">Table S1</a>) than the MTM, genomic structures might not have been identified correctly, which might also have affected the double-structure SEM.</p></div><div id="sec-15" class="subsection"><h3>Goodness of fit of SEM and choice of best networks</h3><p id="p-68">Considering all model performance criteria jointly, the best-performing networks in the SEM analysis are given in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. In the Dent panel, the SEM with the structure derived from the residuals with GS 3 and the SEM derived from the genomic component with TABU 1 performed best considering both the double-structure and single-structure settings. In the Flint panel, the SEM with the structure derived from the genomic component with TABU 1 and 2 had a better fit than the SEM with the structure derived from the genomic component with GS 1, 2, 3, and 4, although both had a higher DIC than the MTM. Regarding the residual structure in the Flint panel, we selected the structure derived with GS 1, 2, 3, and 4 because the SEM with the structure derived with TABU 2 had a higher DIC than the SEM with the structure derived with GS 1, 2, 3, and 4; and the SEM with the structure derived with TABU 1 had a considerably smaller logL than the SEM with the structure derived with GS 1, 2, 3, and 4. Consequently, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> shows the structures derived with TABU 1 for the genomic components and the structure derived with GS 3 for the residual components for both panels.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2779/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trait networks for genomic values and residuals in Dent and Flint: graphs of the genomic component from the tabu-search algorithm using BIC as score. Structure learning was performed with 500 bootstrap samples. Graphs of the residuals from the GS algorithm with mutual information as test statistic, also with 500 bootstrap samples and a significance level of &#x3B1; = 0.01. Labels of edges indicate the proportion of bootstrap samples supporting the edge and (in parentheses) the proportion having the direction shown. Edges that were not significant in the averaging process due to a network-internal empirical test on the arc&#x2019;s strength are not shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1036785186" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Trait networks for genomic values and residuals in Dent and Flint: graphs of the genomic component from the tabu-search algorithm using BIC as score. Structure learning was performed with 500 bootstrap samples. Graphs of the residuals from the GS algorithm with mutual information as test statistic, also with 500 bootstrap samples and a significance level of &lt;em&gt;&#x3B1;&lt;/em&gt; = 0.01. Labels of edges indicate the proportion of bootstrap samples supporting the edge and (in parentheses) the proportion having the direction shown. Edges that were not significant in the averaging process due to a network-internal empirical test on the arc&#x2019;s strength are not shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2779/F3.medium.gif" width="363" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/8/2779/F3.medium.gif" width="363" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2779/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2779/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486105" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-69" class="first-child">Trait networks for genomic values and residuals in Dent and Flint: graphs of the genomic component from the tabu-search algorithm using BIC as score. Structure learning was performed with 500 bootstrap samples. Graphs of the residuals from the GS algorithm with mutual information as test statistic, also with 500 bootstrap samples and a significance level of <em>α</em> = 0.01. Labels of edges indicate the proportion of bootstrap samples supporting the edge and (in parentheses) the proportion having the direction shown. Edges that were not significant in the averaging process due to a network-internal empirical test on the arc’s strength are not shown.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><div id="sec-17" class="subsection"><h3>Scope of methodology</h3><p id="p-70">Structures derived from BN algorithms conveyed more information than mere trait-association values from an MTM because many pairwise correlations are suggested to arise from indirect associations that are mediated by one or more variables. For example, the relatively strong genomic correlations between PH and DMC and between DtSILK and DMY in the Flint panel, and the relatively strong genomic correlation between DtSILK and DMC in the Dent panel, did not correspond to an edge in the genomic networks (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). As illustrated by <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Valente <em>et al.</em> (2015)</a>, a wrong interpretation or use of trait associations can lead to erroneous selection decisions or interventions. Therefore, knowledge on network structures among traits might be useful in addition to knowledge on trait correlations with respect to indirect selection efforts.</p><p id="p-71">Genomic correlations, networks, and model performance criteria varied between the Dent and Flint panels, and incorporating genomic networks into prediction models improved goodness of fit more in Dent than in Flint. As the DH lines in the Dent panel were genetically more diverse than in the Flint panel (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Lehermeier <em>et al.</em> 2014</a>), this could imply that formulating an SEM is more advantageous when dealing with a diverse set of genotypes. We also noted that the DIC in the single-structure SEM was more variable in Flint than in Dent. Variable selection through the network algorithms was more stringent in Flint and, therefore, the SEM differed more from each other and from the MTM than in Dent (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>). Based on the comparison of the two data sets, the impact of genetic structure on network inference could not be shown conclusively. In combination with an investigation of methods for control of variable selection intensity (<em>i.e.</em>, propensity of an algorithm, test, or score to a sparser network), this topic warrants further research.</p><p id="p-72">In both panels, the marginal logL was higher for many SEM than for MTM, even though the network structure on the covariance matrices implied variable selection. This shows that restrictions on the covariance structures among traits were generally supported by the data. Following the principle of parsimony, a fully recursive structure might not be the best representation of connections among traits.</p></div><div id="sec-18" class="subsection"><h3>Choice of method for network construction</h3><p id="p-73">Our results suggest that the choice of method for network learning is crucial, and that a thorough assessment of network structures is necessary when dealing with real-life data. Resulting networks might not explain or fit the data better than a full network as seen here (especially for the genomic component in Flint). Assessment of the different network structures was done by considering several model quality criteria (DIC, logL, number of parameters, and cross-validated predictive ability) together, and by evaluating if these criteria ranked networks consistently. Marginal likelihoods are used routinely to evaluate the plausibility of prior assumptions (<em>e.g.</em>, <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Spiegelhalter <em>et al.</em> 2002</a>) from the observed data. After we learned the BN, we translated them into prior assumptions for SEM, and then assessed the various SEM for their priors via marginal likelihoods. DIC conveys information on number of effective parameters in model comparisons. Different prior assumptions translate into different numbers of effective parameters in the SEM, <em>i.e.</em>, model complexity varies over networks. As DIC reflects model complexity, it is a natural companion to the marginal likelihood for SEM ranking. While goodness-of-fit criteria evaluate the quality of data description by a network, predictive ability reflects the generalization of structure estimation from one subset of the data (training set) to another subset (test set). However, true structures of traits remain unknown and validating inferred connections in experiments with a broad range of genetic material after hypothesis generation with BN and SEM is crucial.</p><p id="p-74">SEM analysis suggested score-based approaches are better for learning the structure of the genomic networks, and constraint-based approaches are better for the residual networks. This might have resulted from constraint-based algorithms having been more restrictive (given the chosen significance level) and from residual networks being sparser than genomic networks. Evidence for the significance of edges (<em>i.e.</em>, support of the edge in &gt;70% of the bootstrap samples) was generally higher in the constraint-based networks than in the score-based networks.</p><p id="p-75">Influence of the significance level in the constraint-based algorithms warrants further investigation. However, it might be expected that a larger <em>α</em> for learning BN from the residuals would result in more complex networks, and, at some point, that additional complexity would result in lower predictive ability and/or overfitting. Advanced network-construction algorithms, such as max-min hill-climbing (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Tsamardinos <em>et al.</em> 2006</a>), combine a constraint-based edge search with a score-based directing of the edges. Such combined approaches should be investigated in future research because they allow variation of the significance level in combination with a score-based network search (<em>i.e.</em>, directing arcs).</p></div><div id="sec-19" class="subsection"><h3>Interpretation of network structures</h3><p id="p-76">If variables A and B are directly associated with each other, and B and C are alike, then an association between A and C would be a logical consequence, even if there is no direct association between A and C (<em>cf</em>. part A of <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). When a BN is inferred, such associations that can be explained by a chain of other associations are not depicted as connections. This means that a connection in a network is more reliable than a significant association between two variables because BN eliminate connections that are unlikely to be direct.</p><p id="p-77">Therefore, BN connections provide information beyond mere pairwise associations between traits. If there were a causal connection between child and parent variable, then the child variable would change with a change of the parent variable. For example, if silking time in Dent material is changed, <em>e.g.</em>, by early seeding, it is likely that PH at harvest changes too, according to the residual network in the Dent panel. If the genotypic value of PH is changed in the Dent panel, <em>e.g.</em>, by selection, this change is likely to affect the genotypic value of DMY, according to the genomic network in the Dent panel. Nonetheless, the interpretation of BN connections as causal effects is delicate and needs further assumptions than those made here, especially the assumption of absence of additional variables influencing those already included in the study. Thus, we suggest interpretation of our networks as an overall association among more than two entities, <em>i.e.</em>, values of the child variable are associated with values of the parent variable.</p><p id="p-78">Residual networks followed some temporal order as traits measured at harvest (PH, DMY, and DMC) depended on traits measured during the vegetation period (DtTAS and DtSILK). Additionally, the inference of direction between DtTAS and DtSILK in the residual networks was relatively uncertain with 52 and 55% of bootstrap samples supporting the direction. This might be because both flowering traits are determined by the time of transition from vegetative to reproductive growth (TVR) and no direct dependence between them truly exists, as both traits depend on TVR. The connection between DtTAS and DtSILK would then be an example of an induced connection by an unobserved confounder in a network.</p><p id="p-79">A conclusion of our study is that by illustrating trait connections with respect to their genomic and residual nature separately and identifying spurious connections, a clearer picture on traits can be gained. This can be useful for multiple-trait prediction and indirect selection.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-80">We acknowledge the support of the Technische Universität München - Institute for Advanced Study, funded by the German Excellence Initiative. G.J.M.R. acknowledges the receipt of a fellowship from the OECD Co-operative Research Programme: Biological Resource Management for Sustainable Agricultural Systems in 2015.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-17"><p id="p-81">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044263/-/DC1</a>.</p></li><li class="fn" id="fn-18"><p id="p-82"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-03-17"><span class="received-label">Received </span>March 17, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-06-15"><span class="accepted-label">Accepted </span>June 15, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Töpner <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aliferis</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Statnikov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsamardinos</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koutsoukos</span>  <span class="cit-name-given-names">X. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">a</span> <span class="cit-article-title">Local causal and Markov blanket induction for causal discovery and feature selection for classification part I: algorithms and empirical evaluation.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D11%26rft.spage%253D171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000277186400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aliferis</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Statnikov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsamardinos</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koutsoukos</span>  <span class="cit-name-given-names">X. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">b</span> <span class="cit-article-title">Local causal and Markov blanket induction for causal discovery and feature selection for classification part II: analysis and extensions.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D11%26rft.spage%253D235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.3" data-doi="10.1186/1752-0509-2-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aten</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuller</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lusis</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Using genetic markers to orient the edges in quantitative trait networks: the NEO software.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DSemple%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D2%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DA%2Bsimple%2Bprinciple%2Bconcerning%2Bthe%2Brobustness%2Bof%2Bprotein%2Bcomplex%2Bactivity%2Bto%2Bchanges%2Bin%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-2-1%26rft_id%253Dinfo%253Apmid%252F18171472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-2-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18171472&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.4" data-doi="10.1186/gb-2013-14-9-r103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camisan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Intraspecific variation of recombination rate in maize.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR103%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-9-r103%26rft_id%253Dinfo%253Apmid%252F24050704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-9-r103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24050704&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.8.2779.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chickering</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> A transformational characterization of equivalent Bayesian network structures, pp. <span class="cit-fpage">87</span>–<span class="cit-lpage">98</span> in <em>Proceedings of the Eleventh Conference on Uncertainty in Artificial Intelligence</em>. <span class="cit-publ-name">Morgan Kaufmann Publishers Inc.</span>, <span class="cit-publ-loc">Montréal, Canada</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.8.2779.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> Methods to accelerate the learning of Bayesian network structures in <em>Proceedings of the 2007 UK Workshop on Computational Intelligence</em>. Imperial College, London, UK.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-7.8.2779.7"><div class="cit-metadata unstructured">de los Campos, G., and P. Pérez-Rodríguez, 2012 BLR: Bayesian Linear Regression. R package version 1.3. <a href="http://CRAN.R-project.org/package=BLR">http://CRAN.R-project.org/package=BLR</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-web" id="cit-7.8.2779.8"><div class="cit-metadata unstructured">de los Campos, G., and P. Pérez-Rodríguez, 2014 BGLR: Bayesian Generalized Linear Regression. R package version 1.0.3. <a href="http://CRAN.R-project.org/package=BGLR">http://CRAN.R-project.org/package=BGLR</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.9" data-doi="10.1534/genetics.108.094888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Maturana</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Exploring biological relationships between calving traits in primiparous cattle with a Bayesian recursive model.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.094888%26rft_id%253Dinfo%253Apmid%252F18984571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgxLzEvMjc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.10" data-doi="10.1186/1297-9686-42-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Maturana</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Modeling relationships between calving traits: a comparison between standard and recursive mixed models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DLopez%2Bde%2BMaturana%26rft.auinit1%253DE.%26rft.volume%253D42%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DModeling%2Brelationships%2Bbetween%2Bcalving%2Btraits%253A%2Ba%2Bcomparison%2Bbetween%2Bstandard%2Band%2Brecursive%2Bmixed%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-1%26rft_id%253Dinfo%253Apmid%252F20100345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20100345&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.11" data-doi="10.1086/281736"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1952</span> <span class="cit-article-title">The problem of environment and selection.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D86%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1086%252F281736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/281736&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1952UY00500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2779.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source"><em>Introduction to Quantitative Genetics</em></span>. <span class="cit-publ-name">Longmans Green</span>, <span class="cit-publ-loc">Harlow, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.13" data-doi="10.3382/ps/pev031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felipe</span>  <span class="cit-name-given-names">V. P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valente</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Using multiple regression, Bayesian networks and artificial neural networks for prediction of total egg production in European quails based on earlier expressed phenotypes.</span> <abbr class="cit-jnl-abbrev">Poult. Sci.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPoult.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.3382%252Fps%252Fpev031%26rft_id%253Dinfo%253Apmid%252F25713397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3382/ps/pev031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25713397&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.14" data-doi="10.1017/S0080456800012163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1918</span> <span class="cit-article-title">The correlation between relatives on the supposition of Mendelian inheritance.</span> <abbr class="cit-jnl-abbrev">Trans. R. Soc. Edinb.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BR.%2BSoc.%2BEdinb.%26rft.volume%253D52%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0080456800012163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0080456800012163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.15" data-doi="10.1371/journal.pone.0028334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A large maize (<em>Zea mays</em> L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28334%26rft.epage%253De28334%26rft.atitle%253DA%2Blarge%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%2BSNP%2Bgenotyping%2Barray%253A%2Bdevelopment%2Band%2Bgermplasm%2Bgenotyping%252C%2Band%2Bgenetic%2Bmapping%2Bto%2Bcompare%2Bwith%2Bthe%2BB73%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028334%26rft_id%253Dinfo%253Apmid%252F22174790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22174790&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.8.2779.16"><div class="cit-metadata unstructured">Geiger, D., and D. Heckerman, 1995 Learning Gaussian networks: a unification for discrete and Gaussian domains, pp. 274–284 in <em>Proceedings of the Eleventh Conference on Uncertainty in Artificial Intelligence</em>. Morgan Kaufmann Publishers, San Francisco, CA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.17" data-doi="10.1534/genetics.103.025734"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Quantitative genetic models for describing simultaneous and recursive relationships between phenotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">1407</span>–<span class="cit-lpage">1424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D167%26rft.issue%253D3%26rft.spage%253D1407%26rft.epage%253D1424%26rft.atitle%253DQuantitative%2BGenetic%2BModels%2Bfor%2BDescribing%2BSimultaneous%2Band%2BRecursive%2BRelationships%2BBetween%2BPhenotypes%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.025734%26rft_id%253Dinfo%253Apmid%252F15280252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ny8zLzE0MDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjc3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparison of single-trait and multiple-trait genomic prediction models.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D1%26rft_id%253Dinfo%253Apmid%252F24387126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24387126&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.19" data-doi="10.1534/genetics.110.123273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hageman</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leduc</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korstanje</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paigen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Bayesian framework for inference of the genotype–phenotype map for segregating populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">1163</span>–<span class="cit-lpage">1170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.123273%26rft_id%253Dinfo%253Apmid%252F21242536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ny80LzExNjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjc3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hazel</span>  <span class="cit-name-given-names">L. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">The genetic basis for constructing selection indexes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">476</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHazel%26rft.auinit1%253DL.%2BN.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D476%26rft.epage%253D490%26rft.atitle%253DTHE%2BGENETIC%2BBASIS%2BFOR%2BCONSTRUCTING%2BSELECTION%2BINDEXES%26rft_id%253Dinfo%253Apmid%252F17247099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIyOC82LzQ3NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.21" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-trait genomic selection methods increase genetic value prediction accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjc3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.22" data-doi="10.1038/hdy.2015.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.9%26rft_id%253Dinfo%253Apmid%252F25873147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25873147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2779.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kullback</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-source"><em>Information Theory and Statistics</em></span>. <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.24" data-doi="10.1111/j.1467-8640.1994.tb00166.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bacchus</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Learning Bayesian belief networks: an approach based on the MDL principle.</span> <abbr class="cit-jnl-abbrev">Comput. Intell.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BIntell.%26rft.volume%253D10%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-8640.1994.tb00166.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-8640.1994.tb00166.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legendre</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Comparison of permutation methods for the partial correlation and partial Mantel tests.</span> <abbr class="cit-jnl-abbrev">J. Stat. Comput. Simul.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BComput.%2BSimul.%26rft.volume%253D67%26rft.spage%253D37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.26" data-doi="10.1534/genetics.114.161943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krämer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camisan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Usefulness of multiparental populations of maize (<em>Zea mays</em> L.) for genome-based prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLehermeier%26rft.auinit1%253DC.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D16%26rft.atitle%253DUsefulness%2Bof%2BMultiparental%2BPopulations%2Bof%2BMaize%2B%2528Zea%2Bmays%2BL.%2529%2Bfor%2BGenome-Based%2BPrediction%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161943%26rft_id%253Dinfo%253Apmid%252F25236445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTk4LzEvMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.27" data-doi="10.1534/genetics.115.177394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assessment of genetic heterogeneity in structured plant populations using multivariate whole-genome regression models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177394%26rft_id%253Dinfo%253Apmid%252F26122758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzEvMzIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.28" data-doi="10.1371/journal.pgen.0020114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsaih</span>  <span class="cit-name-given-names">S.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shockley</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stylianou</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wergedal</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Structural model analysis of multiple quantitative traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DR.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253De114%26rft.epage%253De114%26rft.atitle%253DStructural%2Bmodel%2Banalysis%2Bof%2Bmultiple%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020114%26rft_id%253Dinfo%253Apmid%252F16848643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16848643&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2779.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source"><em>Genetics and Analysis of Quantitative Traits</em></span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.30" data-doi="10.1016/j.ajhg.2014.12.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripke</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint analysis of psychiatric disorders increases accuracy of risk prediction for schizophrenia, bipolar disorder, and major depressive disorder.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D96%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.12.006%26rft_id%253Dinfo%253Apmid%252F25640677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25640677&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malik</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malik</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hussain</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chughtai</span>  <span class="cit-name-given-names">S. U. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Javed</span>  <span class="cit-name-given-names">H. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic correlation among various quantitative characters in maize (<em>Zea mays</em> L.) hybrids.</span> <abbr class="cit-jnl-abbrev">J Agric Soc Sci</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">262</span>–<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BAgric%2BSoc%2BSci%26rft.volume%253D1%26rft.spage%253D262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-thesis" id="cit-7.8.2779.32"><div class="cit-metadata unstructured">Margaritis, D., 2003 Learning Bayesian network model structure from data. Ph.D. Thesis, School of Computer Science, Carnegie Mellon University, Pittsburgh, PA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valente</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An assessment of linkage disequilibrium in Holstein cattle using a Bayesian network.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">474</span>–<span class="cit-lpage">487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMorota%26rft.auinit1%253DG.%26rft.volume%253D129%26rft.issue%253D6%26rft.spage%253D474%26rft.epage%253D487%26rft.atitle%253DAn%2Bassessment%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2BHolstein%2Bcattle%2Busing%2Ba%2BBayesian%2Bnetwork.%26rft_id%253Dinfo%253Apmid%252F23148973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23148973&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2779.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagarajan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scutari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lèbre</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source"><em>Bayesian Networks in R with Applications in Systems Biology</em></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.35" data-doi="10.1093/jhered/esw020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nazarian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gezan</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">GenoMatrix: a software package for pedigree-based and genomic prediction analyses on complex traits.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">372</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesw020%26rft_id%253Dinfo%253Apmid%252F27025440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esw020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27025440&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.36" data-doi="10.1534/genetics.107.085167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neto</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrara</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Inferring causal phenotype networks from segregating populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1089</span>–<span class="cit-lpage">1100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085167%26rft_id%253Dinfo%253Apmid%252F18505877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8yLzEwODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjc3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.37" data-doi="10.1214/09-AOAS288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neto</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Causal graphical models in systems genetics: a unified framework for joint inference of causal network and genetic architecture for correlated phenotypes.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Stat.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnn%2BAppl%2BStat%26rft.aulast%253DNeto%26rft.auinit1%253DE.%2BC.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D320%26rft.epage%253D339%26rft.atitle%253DCAUSAL%2BGRAPHICAL%2BMODELS%2BIN%2BSYSTEMS%2BGENETICS%253A%2BA%2BUNIFIED%2BFRAMEWORK%2BFOR%2BJOINT%2BINFERENCE%2BOF%2BCAUSAL%2BNETWORK%2BAND%2BGENETIC%2BARCHITECTURE%2BFOR%2BCORRELATED%2BPHENOTYPES.%26rft_id%253Dinfo%253Adoi%252F10.1214%252F09-AOAS288%26rft_id%253Dinfo%253Apmid%252F21218138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/09-AOAS288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21218138&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2779.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearl</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source"><em>Causality: Models</em>, <em>Reasoning</em>, <em>and Inference</em></span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.39" data-doi="10.1186/s12918-014-0137-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peñagaricano</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valente</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steibel</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Exploring causal networks underlying fat deposition and muscularity in pigs through the integration of phenotypic, genotypic and transcriptomic data.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BSyst.%2BBiol.%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12918-014-0137-8%26rft_id%253Dinfo%253Apmid%252F25582171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12918-014-0137-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25582171&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porth</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skyba</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedmann</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hannemann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Network analysis reveals the relationship among wood properties, gene expression levels and genotypes of natural <em>Populus trichocarpa</em> accessions.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D200%26rft.spage%253D727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.41" data-doi="10.1017/S175173111300150X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pszczola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Haas</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strabel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Effect of predictor traits on accuracy of genomic breeding values for feed intake based on a limited cow reference population.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%26rft.volume%253D7%26rft.spage%253D1759%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS175173111300150X%26rft_id%253Dinfo%253Apmid%252F23915541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S175173111300150X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23915541&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2779.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em></span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.43" data-doi="10.1016/0005-1098(78)90005-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rissanen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Modeling by shortest data description.</span> <abbr class="cit-jnl-abbrev">Automatica</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">465</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAutomatica%26rft.volume%253D14%26rft.spage%253D465%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0005-1098%252878%252990005-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0005-1098(78)90005-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978FU79500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.44" data-doi="10.2307/2527750"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">The sampling variance of the genetic correlation coefficient.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D15%26rft.spage%253D469%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2527750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2527750&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1959WX52300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.45" data-doi="10.1038/nature07633"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reverse engineering the genotype–phenotype map with natural genetic variation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">738</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D456%26rft.issue%253D7223%26rft.spage%253D738%26rft.epage%253D744%26rft.atitle%253DReverse%2Bengineering%2Bthe%2Bgenotype-phenotype%2Bmap%2Bwith%2Bnatural%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07633%26rft_id%253Dinfo%253Apmid%252F19079051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07633&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19079051&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261559900038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.46" data-doi="10.1038/hdy.1995.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roff</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The estimation of genetic correlations from phenotypic correlations: a test of Cheverud’s conjecture.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D74%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1995.68%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1995.68&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995QX25900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.47" data-doi="10.1186/1297-9686-43-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valente</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Inferring causal phenotype networks using structural equation models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DForni%26rft.auinit1%253DS.%26rft.volume%253D43%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DDifferent%2Bgenomic%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Banalysis%2Busing%2Bphenotypic%252C%2Bpedigree%2Band%2Bgenomic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-1%26rft_id%253Dinfo%253Apmid%252F21208445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208445&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.48" data-doi="10.1038/ng1589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schadt</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">An integrative genomics approach to infer causal associations between gene expression and disease.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">710</span>–<span class="cit-lpage">717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSchadt%26rft.auinit1%253DE.%2BE.%26rft.volume%253D37%26rft.issue%253D7%26rft.spage%253D710%26rft.epage%253D717%26rft.atitle%253DAn%2Bintegrative%2Bgenomics%2Bapproach%2Bto%2Binfer%2Bcausal%2Bassociations%2Bbetween%2Bgene%2Bexpression%2Band%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1589%26rft_id%253Dinfo%253Apmid%252F15965475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15965475&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230196400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.49" data-doi="10.18637/jss.v035.i03"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scutari</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Learning Bayesian networks with the bnlearn R package.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DPatil%26rft.auinit1%253DA.%26rft.volume%253D35%26rft.issue%253D4%26rft.spage%253D1%26rft.epage%253D81%26rft.atitle%253DPyMC%253A%2BBayesian%2BStochastic%2BModelling%2Bin%2BPython.%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v035.i03%26rft_id%253Dinfo%253Apmid%252F21603108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v035.i03&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21603108&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.8.2779.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scutari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagarajan</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> On identifying significant edges in graphical models, pp. <span class="cit-fpage">15</span>–<span class="cit-lpage">27</span> in <em>Proceedings of Workshop on Probabilistic Problem Solving in Biomedicine</em>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Bled, Slovenia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scutari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Improving the efficiency of genomic selection.</span> <abbr class="cit-jnl-abbrev">Stat. Appl. Genet. Mol. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BAppl.%2BGenet.%2BMol.%2BBiol.%26rft.volume%253D12%26rft.spage%253D517%26rft_id%253Dinfo%253Apmid%252F23934612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23934612&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.52" data-doi="10.1534/genetics.114.165704"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scutari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multiple quantitative trait analysis using Bayesian networks.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DScutari%26rft.auinit1%253DM.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D129%26rft.epage%253D137%26rft.atitle%253DMultiple%2BQuantitative%2BTrait%2BAnalysis%2BUsing%2BBayesian%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165704%26rft_id%253Dinfo%253Apmid%252F25236454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.53" data-doi="10.2307/2527838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-article-title">Phenotypic, genetic and environmental correlations.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">474</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D17%26rft.spage%253D474%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2527838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2527838&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19617259A00032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2779.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sneath</span>  <span class="cit-name-given-names">P. H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokal</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-source"><em>Numerical Taxonomy. The Principles and Practice of Numerical Classification</em></span>. <span class="cit-publ-name">W.H. Freeman and Company</span>, <span class="cit-publ-loc">San Francisco, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.55" data-doi="10.1111/1467-9868.00353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spiegelhalter</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlin</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Linde</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Bayesian measures of model complexity and fit.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Ser. B. Stat. Methodol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSer.%2BB.%2BStat.%2BMethodol.%26rft.volume%253D64%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.56" data-doi="10.1007/s10994-006-6889-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsamardinos</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aliferis</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The max-min hill-climbing Bayesian network structure learning algorithm.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D65%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10994-006-6889-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10994-006-6889-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000240797500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.57" data-doi="10.1534/genetics.109.112979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valente</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Searching for recursive causal structures in multivariate quantitative genetics mixed models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.112979%26rft_id%253Dinfo%253Apmid%252F20351220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNjMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.58" data-doi="10.1534/genetics.113.151209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valente</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Is structural equation modeling advantageous for the genetic improvement of multiple traits?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151209%26rft_id%253Dinfo%253Apmid%252F23608193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.59" data-doi="10.1534/genetics.114.169490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valente</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peñagaricano</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The causal meaning of genomic predictors and how it affects construction and comparison of genome-enabled selection models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169490%26rft_id%253Dinfo%253Apmid%252F25908318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.60" data-doi="10.2527/jas.2009-1952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vázquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Technical note: an R package for fitting generalized linear mixed models in animal breeding.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2009-1952%26rft_id%253Dinfo%253Apmid%252F19820058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2009-1952&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19820058&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274040700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.61" data-doi="10.1371/journal.pone.0103997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A new method to infer causal phenotype networks using QTL and phenotypic information.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e103997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De103997%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0103997%26rft_id%253Dinfo%253Apmid%252F25144184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0103997&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25144184&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2779.62" data-doi="10.1371/journal.pone.0005161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winrow</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasarskis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laposky</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Uncovering the genetic landscape for multiple sleep-wake traits.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWinrow%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253De5161%26rft.epage%253De5161%26rft.atitle%253DUncovering%2Bthe%2Bgenetic%2Blandscape%2Bfor%2Bmultiple%2Bsleep-wake%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005161%26rft_id%253Dinfo%253Apmid%252F19360106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19360106&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2779.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/8/2779.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2763" title="McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2791" title="FlyExpress 7: An Integrated Discovery Platform To Study Coexpressed Genes Using in Situ Hybridization Images in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/8"><span>Volume 7 Issue 8, August 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/8.cover-source.jpg?itok=apH_YDuu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2779" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3mDw4qrzQgTrfa8OXFmBBTFR73OsIASEfm0GHzIpcYg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/8/2779" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/8/2779" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/8/2779" class="active" data-icon-position="" data-hide-link-title="0">Bayesian Networks Illustrate Genomic and Residual Trait Connections in Maize (Zea mays L.)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485855" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-EkGzXfYZz47agxudmmuPPdm14GR8OWkU9tfysLo4Duc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2779.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/8/2779" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-_mi-vyGWiKyZgCBvKu1SZGZc1fpiqSNhcRWYSztiw7k" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/8/2779" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2779.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485855" id="node485855--2" data-pisa="ggg;7/8/2779" data-pisa-master="ggg;g3.117.044263" data-apath="/ggg/7/8/2779.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bayesian Networks Illustrate Genomic and Residual Trait Connections in Maize (<em>Zea mays</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Katrin Töpner</span>, <span class="highwire-citation-author" data-delta="1">Guilherme J. M. Rosa</span>, <span class="highwire-citation-author" data-delta="2">Daniel Gianola</span> and <span class="highwire-citation-author" data-delta="3">Chris-Carolin Schön</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2779-2789;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.044263 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485855--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katrin Töpner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, TUM School of Life Sciences Weihenstephan, Technical University of Munich, 85354 Freising, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katrin%2BT%C3%B6pner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=T%C3%B6pner%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatrin%2BT%25C3%25B6pner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guilherme J. M. Rosa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal Sciences, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guilherme%2BJ.%2BM.%2BRosa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosa%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuilherme%2BJ.%2BM.%2BRosa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal Sciences, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chris-Carolin Schön </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, TUM School of Life Sciences Weihenstephan, Technical University of Munich, 85354 Freising, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris-Carolin%2BSch%C3%B6n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sch%C3%B6n%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris-Carolin%2BSch%25C3%25B6n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chris.schoen@tum.de" class="" data-icon-position="" data-hide-link-title="0">chris.schoen@tum.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/8/2779" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/8/2779&amp;title=Bayesian%20Networks%20Illustrate%20Genomic%20and%20Residual%20Trait%20Connections%20in%20Maize%20%28Zea%20mays%20L.%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/8/2779&amp;title=Bayesian%20Networks%20Illustrate%20Genomic%20and%20Residual%20Trait%20Connections%20in%20Maize%20%28Zea%20mays%20L.%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/8/2779&amp;title=Bayesian%20Networks%20Illustrate%20Genomic%20and%20Residual%20Trait%20Connections%20in%20Maize%20%28Zea%20mays%20L.%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/8/2779&amp;text=Bayesian%20Networks%20Illustrate%20Genomic%20and%20Residual%20Trait%20Connections%20in%20Maize%20%28Zea%20mays%20L.%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/8/2779&amp;title=Bayesian%20Networks%20Illustrate%20Genomic%20and%20Residual%20Trait%20Connections%20in%20Maize%20%28Zea%20mays%20L.%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/8/2779&amp;t=Bayesian%20Networks%20Illustrate%20Genomic%20and%20Residual%20Trait%20Connections%20in%20Maize%20%28Zea%20mays%20L.%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/8/2779&amp;title=Bayesian%20Networks%20Illustrate%20Genomic%20and%20Residual%20Trait%20Connections%20in%20Maize%20%28Zea%20mays%20L.%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/8/2779&amp;title=Bayesian%20Networks%20Illustrate%20Genomic%20and%20Residual%20Trait%20Connections%20in%20Maize%20%28Zea%20mays%20L.%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485855/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485855" id="node485855--3" data-pisa="ggg;7/8/2779" data-pisa-master="ggg;g3.117.044263" data-apath="/ggg/7/8/2779.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bayesian Networks Illustrate Genomic and Residual Trait Connections in Maize (<em>Zea mays</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Katrin Töpner</span>, <span class="highwire-citation-author" data-delta="1">Guilherme J. M. Rosa</span>, <span class="highwire-citation-author" data-delta="2">Daniel Gianola</span> and <span class="highwire-citation-author" data-delta="3">Chris-Carolin Schön</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2779-2789;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.044263 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485855--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katrin Töpner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, TUM School of Life Sciences Weihenstephan, Technical University of Munich, 85354 Freising, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katrin%2BT%C3%B6pner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=T%C3%B6pner%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatrin%2BT%25C3%25B6pner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guilherme J. M. Rosa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal Sciences, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guilherme%2BJ.%2BM.%2BRosa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosa%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuilherme%2BJ.%2BM.%2BRosa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Animal Sciences, University of Wisconsin–Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chris-Carolin Schön </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, TUM School of Life Sciences Weihenstephan, Technical University of Munich, 85354 Freising, Germany</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Advanced Study, Technical University of Munich, 85748 Garching, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris-Carolin%2BSch%C3%B6n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sch%C3%B6n%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris-Carolin%2BSch%25C3%25B6n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chris.schoen@tum.de" class="" data-icon-position="" data-hide-link-title="0">chris.schoen@tum.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2779&amp;rft.epage=2789&amp;rft.atitle=Bayesian%20Networks%20Illustrate%20Genomic%20and%20Residual%20Trait%20Connections%20in%20Maize%20%28Zea%20mays%20L.%29&amp;rft.volume=7&amp;rft.issue=8&amp;rft.date=2017-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=T%C3%B6pner%2C+Katrin&amp;rft.au=Rosa%2C+Guilherme+J.+M.&amp;rft.au=Gianola%2C+Daniel&amp;rft.au=Sch%C3%B6n%2C+Chris-Carolin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485855/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485855/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485855/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485855/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485855/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485855/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485855/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485855/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485855/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485855/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485855/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485855/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/8/2779" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/8/2779.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/8/2779.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/8/2779.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/8/2779.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
