<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/8/2519" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/8/2519.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/8/2519.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/8/2519.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/8/2519.atom" />
<meta name="HW.pisa" content="ggg;7/8/2519" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genetic Dissection of Morphometric Traits Reveals That Phytochrome B Affects Nucleus Size and Heterochromatin Organization in Arabidopsis thaliana" />
<meta name="DC.Identifier" content="10.1534/g3.117.043539" />
<meta name="DC.Date" content="2017-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Snoek et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Microscopically visible chromatin is partitioned into two major components in Arabidopsis thaliana nuclei. On one hand, chromocenters are conspicuous foci of highly condensed “heterochromatic” domains that contain mostly repeated sequences. On the other hand, less condensed and gene-rich “euchromatin” emanates from these chromocenters. This differentiation, together with the dynamic nature of chromatin compaction in response to developmental and environmental stimuli, makes Arabidopsis a powerful system for studying chromatin organization and dynamics. Heterochromatin dynamics can be monitored by measuring the Heterochromatin Index, i.e. , the proportion of nuclei displaying well-defined chromocenters, or the DNA fraction of chromocenters (relative heterochromatin fraction). Both measures are composite traits, thus their values represent the sum of effects of various underlying morphometric properties. We exploited genetic variation between natural occurring accessions to determine the genetic basis of individual nucleus and chromocenter morphometric parameters (area, perimeter, density, roundness, and heterogeneity) that together determine chromatin compaction. Our novel reductionist genetic approach revealed quantitative trait loci (QTL) for all measured traits. Genomic colocalization among QTL was limited, which suggests a complex genetic regulation of chromatin compaction. Yet genomic intervals of QTL for nucleus size (area and perimeter) both overlap with a known QTL for heterochromatin compaction that is explained by natural polymorphism in the red/far-red light and temperature receptor Phytochrome B. Mutant analyses and genetic complementation assays show that Phytochrome B is a negative regulator of nucleus size, revealing that perception of climatic conditions by a Phytochrome-mediated hub is a major determinant for coordinating nucleus size and heterochromatin compaction." />
<meta name="DC.Contributor" content="Basten L. Snoek" />
<meta name="DC.Contributor" content="Penka Pavlova" />
<meta name="DC.Contributor" content="Federico Tessadori" />
<meta name="DC.Contributor" content="Anton J. M. Peeters" />
<meta name="DC.Contributor" content="Clara Bourbousse" />
<meta name="DC.Contributor" content="Fredy Barneche" />
<meta name="DC.Contributor" content="Hans de Jong" />
<meta name="DC.Contributor" content="Paul F. Fransz" />
<meta name="DC.Contributor" content="Martijn van Zanten" />
<meta name="article:published_time" content="2017-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genetic Dissection of Morphometric Traits Reveals That Phytochrome B Affects Nucleus Size and Heterochromatin Organization in Arabidopsis thaliana" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Microscopically visible chromatin is partitioned into two major components in &lt;i&gt;Arabidopsis thaliana&lt;/i&gt; nuclei. On one hand, chromocenters are conspicuous foci of highly condensed “heterochromatic” domains that contain mostly repeated sequences. On the other hand, less condensed and gene-rich “euchromatin” emanates from these chromocenters. This differentiation, together with the dynamic nature of chromatin compaction in response to developmental and environmental stimuli, makes &lt;i&gt;Arabidopsis&lt;/i&gt; a powerful system for studying chromatin organization and dynamics. Heterochromatin dynamics can be monitored by measuring the Heterochromatin Index, &lt;i&gt;i.e.&lt;/i&gt;, the proportion of nuclei displaying well-defined chromocenters, or the DNA fraction of chromocenters (relative heterochromatin fraction). Both measures are composite traits, thus their values represent the sum of effects of various underlying morphometric properties. We exploited genetic variation between natural occurring accessions to determine the genetic basis of individual nucleus and chromocenter morphometric parameters (area, perimeter, density, roundness, and heterogeneity) that together determine chromatin compaction. Our novel reductionist genetic approach revealed quantitative trait loci (QTL) for all measured traits. Genomic colocalization among QTL was limited, which suggests a complex genetic regulation of chromatin compaction. Yet genomic intervals of QTL for nucleus size (area and perimeter) both overlap with a known QTL for heterochromatin compaction that is explained by natural polymorphism in the red/far-red light and temperature receptor Phytochrome B. Mutant analyses and genetic complementation assays show that Phytochrome B is a negative regulator of nucleus size, revealing that perception of climatic conditions by a Phytochrome-mediated hub is a major determinant for coordinating nucleus size and heterochromatin compaction.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/08/01" />
<meta name="citation_mjid" content="ggg;7/8/2519" />
<meta name="citation_id" content="7/8/2519" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/8/2519" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/8/2519.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/8/2519.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/8/2519.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.043539" />
<meta name="citation_pmid" content="28592555" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2519" />
<meta name="citation_lastpage" content="2531" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Basten L. Snoek" />
<meta name="citation_author_institution" content="Laboratory of Nematology, Wageningen University, 6708 PB, The Netherlands" />
<meta name="citation_author_institution" content="Theoretical Biology and Bioinformatics, Institute of Biodynamics and Biocomplexity" />
<meta name="citation_author" content="Penka Pavlova" />
<meta name="citation_author_institution" content="Laboratory of Genetics, Wageningen University, 6708 PB, The Netherlands" />
<meta name="citation_author_institution" content="Plant Development and (Epi)Genetics, Swammerdam Institute for Life Sciences, University of Amsterdam, 1098 SM, The Netherlands" />
<meta name="citation_author" content="Federico Tessadori" />
<meta name="citation_author_institution" content="Plant Development and (Epi)Genetics, Swammerdam Institute for Life Sciences, University of Amsterdam, 1098 SM, The Netherlands" />
<meta name="citation_author_institution" content="Hubrecht Institute, Royal Netherlands Academy of Arts and Sciences, University Medical Center Utrecht, 3584 CT, The Netherlands" />
<meta name="citation_author" content="Anton J. M. Peeters" />
<meta name="citation_author_institution" content="Department of Biology, Institute of Education, Utrecht University, 3584 CH, The Netherlands" />
<meta name="citation_author" content="Clara Bourbousse" />
<meta name="citation_author_institution" content="Institut de Biologie de l’Ecole Normale Supérieure, Centre National de la Recherche Scientifique, Unité Mixte de Recherche 8197, Institut National de la Santé et de la Recherche Médicale U1024, Ecole Normale Supérieure, Paris Sciences et Lettres Research University, F-75005, France" />
<meta name="citation_author" content="Fredy Barneche" />
<meta name="citation_author_institution" content="Institut de Biologie de l’Ecole Normale Supérieure, Centre National de la Recherche Scientifique, Unité Mixte de Recherche 8197, Institut National de la Santé et de la Recherche Médicale U1024, Ecole Normale Supérieure, Paris Sciences et Lettres Research University, F-75005, France" />
<meta name="citation_author" content="Hans de Jong" />
<meta name="citation_author_institution" content="Laboratory of Genetics, Wageningen University, 6708 PB, The Netherlands" />
<meta name="citation_author" content="Paul F. Fransz" />
<meta name="citation_author_institution" content="Plant Development and (Epi)Genetics, Swammerdam Institute for Life Sciences, University of Amsterdam, 1098 SM, The Netherlands" />
<meta name="citation_author" content="Martijn van Zanten" />
<meta name="citation_author_institution" content="Molecular Plant Physiology, Institute of Environmental Biology, Utrecht University, 3584 CH, The Netherlands" />
<meta name="citation_author_email" content="m.vanzanten@uu.nl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2810-7374" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=C. Alonso-Blanco;citation_author=A. J. M. Peeters;citation_author=M. Koornneef;citation_author=C. Lister;citation_author=C. Dean;citation_title=Development of an AFLP based linkage map of Ler, Col and Cvi Arabidopsis thaliana ecotypes and construction of a Ler/Cvi recombinant inbred line population.;citation_pages=259-271;citation_volume=14;citation_year=1998;citation_issue=2;citation_pmid=9628021;citation_doi=10.1046/j.1365-313X.1998.00115.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C. Alonso-Blanco;citation_author=M. G. M. Aarts;citation_author=L. Bentsink;citation_author=J. J. B. Keurentjes;citation_author=M. Reymond;citation_title=What has natural variation taught us about plant development, physiology, and adaptation?;citation_volume=21;citation_year=2009;citation_pmid=19574434;citation_doi=10.1105/tpc.109.068114" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=F. Barneche;citation_author=J. Malapeira;citation_author=P. Mas;citation_title=The impact of chromatin dynamics on plant light responses and circadian clock function.;citation_volume=65;citation_year=2014;citation_pmid=24520020;citation_doi=10.1093/jxb/eru011" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=J. O. Borevitz;citation_author=M. Nordborg;citation_title=The Impact of Genomics on the Study of Natural Variation in Arabidopsis;citation_pages=718-725;citation_volume=132;citation_year=2003;citation_issue=2;citation_pmid=12805600;citation_doi=10.1104/pp.103.023549" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Bourbousse;citation_author=I. Mestiri;citation_author=G. Zabulon;citation_author=M. Bourge;citation_author=F. Formiggini;citation_title=Light signaling controls nuclear architecture reorganization during seedling establishment.;citation_volume=112;citation_year=2015;citation_pmid=25964332;citation_doi=10.1073/pnas.1503512112" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. W. Brown;citation_title=Heterochromatin;citation_pages=417-425;citation_volume=151;citation_year=1966;citation_issue=3709;citation_pmid=5322971;citation_doi=10.1126/science.151.3709.417" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=C. Chandrasekhara;citation_author=G. Mohannath;citation_author=T. Blevins;citation_author=F. Pontvianne;citation_author=C. S. Pikaard;citation_title=Chromosome-specific NOR inactivation explains selective rRNA gene silencing and dosage control in Arabidopsis.;citation_volume=30;citation_year=2016;citation_pmid=26744421;citation_doi=10.1101/gad.273755.115" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. L. Filiault;citation_author=C. A. Wessinger;citation_author=J. R. Dinneny;citation_author=J. Lutes;citation_author=J. O. Borevitz;citation_title=Amino acid polymorphisms in Arabidopsis phytochrome B cause differential responses to light.;citation_volume=105;citation_year=2008;citation_pmid=18287016;citation_doi=10.1073/pnas.0712174105" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=P. Fransz;citation_author=H. de Jong;citation_title=From nucleosome to chromosome: a dynamic organization of genetic information.;citation_pages=4-17;citation_volume=66;citation_year=2011;citation_issue=1;citation_pmid=21443619;citation_doi=10.1111/j.1365-313X.2011.04526.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Fransz;citation_author=J. H. de Jong;citation_author=M. Lysak;citation_author=M. R. Castiglione;citation_author=I. Schubert;citation_title=Interphase chromosomes in Arabidopsis are organized as well defined chromocenters from which euchromatin loops emanate.;citation_volume=99;citation_year=2002;citation_pmid=12384572;citation_doi=10.1073/pnas.212325299" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=P. Fransz;citation_author=R. ten Hoopen;citation_author=F. Tessadori;citation_title=Composition and formation of heterochromatin in Arabidopsis thaliana.;citation_pages=71-82;citation_volume=14;citation_year=2006;citation_issue=1;citation_pmid=16506097;citation_doi=10.1007/s10577-005-1022-5" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. W. M. Green;citation_author=L. B. Snoek;citation_author=J. E. Kammenga;citation_author=S. C. Harvey;citation_title=Genetic mapping of variation in dauer larvae development in growing populations of Caenorhabditis elegans.;citation_pages=306-313;citation_volume=111;citation_year=2013;citation_pmid=23715016;citation_doi=10.1038/hdy.2013.50" />
<meta name="citation_reference" content="citation_journal_title=Jahrb Wiss Bot;citation_author=E. Heitz;citation_title=Das heterochromatin der moose.;citation_pages=762-818;citation_volume=69;citation_year=1928" />
<meta name="citation_reference" content="citation_journal_title=Ber Bot. Ges;citation_author=E. Heitz;citation_title=Heterochromatin, chromocentren, chromomeren.;citation_pages=274-284;citation_volume=47;citation_year=1929" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J.-H. Jung;citation_author=M. Domijan;citation_author=C. Klose;citation_author=S. Biswas;citation_author=D. Ezer;citation_title=Phytochromes function as thermosensors in Arabidopsis.;citation_volume=354;citation_year=2016;citation_pmid=27789797;citation_doi=10.1126/science.aaf6005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. J. B. Keurentjes;citation_author=L. Bentsink;citation_author=C. Alonso-Blanco;citation_author=C. J. Hanhart;citation_author=H. Blankestijn-De Vries;citation_title=Development of a near-isogenic line population of Arabidopsis thaliana and comparison of mapping power with a recombinant inbred line population.;citation_volume=175;citation_year=2007;citation_pmid=17179089;citation_doi=10.1534/genetics.106.066423" />
<meta name="citation_reference" content="citation_journal_title=Z. Pflanzenphysiol.;citation_author=M. Koornneef;citation_author=E. Rolff;citation_author=C. J. P. Spruit;citation_title=Genetic control of light-inhibited hypocotyl elongation in Arabidopsis thaliana (L.) Heynh.;citation_pages=147-160;citation_volume=100;citation_year=1980;citation_doi=10.1016/S0044-328X(80)80208-X" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Legris;citation_author=C. Klose;citation_author=E. S. Burgie;citation_author=C. Costigliolo;citation_author=M. Neme;citation_title=Phytochrome B integrates light and temperature signals in Arabidopsis.;citation_pages=897-900;citation_volume=18;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=C. Liu;citation_author=D. Weigel;citation_title=Chromatin in 3D: progress and prospects for plants.;citation_pages=170;citation_volume=16;citation_year=2015;citation_pmid=26294115;citation_doi=10.1186/s13059-015-0738-6" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=T. F. C. Mackay;citation_title=THE GENETIC ARCHITECTURE OF QUANTITATIVE TRAITS;citation_pages=303-339;citation_volume=35;citation_year=2001;citation_issue=1;citation_pmid=11700286;citation_doi=10.1146/annurev.genet.35.102401.090633" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=O. Mathieu;citation_author=Z. Jasencakova;citation_author=I. Vaillant;citation_author=A.-V. Gendrel;citation_author=V. Colot;citation_title=Changes in 5S rDNA chromatin organization and transcription during heterochromatin establishment in Arabidopsis.;citation_volume=15;citation_year=2003;citation_pmid=14630972;citation_doi=10.1105/tpc.017467" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=H. Nijveen;citation_author=W. Ligterink;citation_author=J. J. B. Keurentjes;citation_author=O. Loudet;citation_author=J. Long;citation_title=AraQTL - workbench and archive for systems genetics in Arabidopsis thaliana.;citation_pages=1225-1235;citation_volume=89;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=V. Pavet;citation_author=C. Quintero;citation_author=N. M. Cecchini;citation_author=A. L. Rosa;citation_author=M. E. Alvarez;citation_title=Arabidopsis displays centromeric DNA hypomethylation and cytological alterations of heterochromatin upon attack by pseudomonas syringae.;citation_pages=577-587;citation_volume=19;citation_year=2006;citation_issue=6;citation_pmid=16776291;citation_doi=10.1094/MPMI-19-0577" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=P. Pavlova;citation_author=F. Tessadori;citation_author=H. J. de Jong;citation_author=P. Fransz;citation_title=Immunocytological analysis of chromatin in isolated nuclei.;citation_pages=413-432;citation_volume=655;citation_year=2010;citation_pmid=20734277;citation_doi=10.1007/978-1-60761-765-5_28" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=A. Pecinka;citation_author=H. Q. Dinh;citation_author=T. Baubec;citation_author=M. Rosa;citation_author=N. Lettner;citation_title=Epigenetic regulation of repetitive elements is attenuated by prolonged heat stress in Arabidopsis.;citation_volume=22;citation_year=2010;citation_pmid=20876829;citation_doi=10.1105/tpc.110.078493" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=G. Perrella;citation_author=E. Kaiserli;citation_title=Light behind the curtain: photoregulation of nuclear architecture and chromatin dynamics in plants.;citation_pages=908-919;citation_volume=212;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Sci.;citation_author=A. Poulet;citation_author=C. Duc;citation_author=M. Voisin;citation_author=S. Desset;citation_author=S. Tutois;citation_title=The LINC complex contributes to heterochromatin organisation and transcriptional gene silencing in plants.;citation_volume=130;citation_year=2017;citation_pmid=28049722;citation_doi=10.1242/jcs.194712" />
<meta name="citation_reference" content="citation_journal_title=Cell Signal. Gene Regul.;citation_author=A. V. Probst;citation_author=O. Mittelsten Scheid;citation_title=Stress-induced structural changes in plant chromatin.;citation_pages=8-16;citation_volume=27;citation_year=2015" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=J. W. Reed;citation_author=P. Nagpal;citation_author=D. S. Poole;citation_author=M. Furuya;citation_author=J. Chory;citation_title=Mutations in the Gene for the Red/Far-Red Light Receptor Phytochrome B Alter Cell Elongation and Physiological Responses throughout Arabidopsis Development;citation_pages=147-157;citation_volume=5;citation_year=1993;citation_issue=2;citation_pmid=8453299;citation_doi=10.1105/tpc.5.2.147" />
<meta name="citation_reference" content="citation_journal_title=Methods in cell science :  an official journal of the Society for In Vitro Biology;citation_journal_abbrev=Methods Cell Sci;citation_author=I. Schubert;citation_author=P. F. Fransz;citation_author=J. Fuchs;citation_author=J. H. de Jong;citation_title=Chromosome painting in plants.;citation_pages=57-69;citation_volume=23;citation_year=2001;citation_issue=1-3;citation_pmid=11741144;citation_doi=10.1023/A:1013137415093" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. Shannon;citation_author=A. Markiel;citation_author=O. Ozier;citation_author=N. S. Baliga;citation_author=J. T. Wang;citation_title=Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks;citation_pages=2498-2504;citation_volume=13;citation_year=2003;citation_issue=11;citation_pmid=14597658;citation_doi=10.1101/gr.1239303" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=L. B. Snoek;citation_author=I. Terpstra;citation_author=R. Dekter;citation_author=G. Van den Ackerveken;citation_author=A. J. M. Peeters;citation_title=Genetical genomics reveals large scale genotype-by-environment interactions in Arabidopsis thaliana.;citation_pages=317;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=L. B. Snoek;citation_author=H. E. Orbidans;citation_author=J. J. Stastna;citation_author=A. Aartse;citation_author=M. Rodriguez;citation_title=Widespread genomic incompatibilities in Caenorhabditis elegans.;citation_pages=1813-1823;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=W. J. J. Soppe;citation_author=Z. Jasencakova;citation_author=A. Houben;citation_author=T. Kakutani;citation_author=A. Meister;citation_title=DNA methylation controls histone H3 lysine 9 methylation and heterochromatin assembly in Arabidopsis;citation_pages=6549-6559;citation_volume=21;citation_year=2002;citation_issue=23;citation_pmid=12456661;citation_doi=10.1093/emboj/cdf657" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. J. Stastna;citation_author=L. B. Snoek;citation_author=J. E. Kammenga;citation_author=S. C. Harvey;citation_title=Genotype-dependent lifespan effects in peptone deprived Caenorhabditis elegans.;citation_pages=16259;citation_volume=5;citation_year=2015;citation_pmid=26539794;citation_doi=10.1038/srep16259" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=F. Tessadori;citation_author=R. van Driel;citation_author=P. Fransz;citation_title=Cytogenetics as a tool to study gene regulation.;citation_pages=147-153;citation_volume=9;citation_year=2004;citation_issue=3;citation_pmid=15003238;citation_doi=10.1016/j.tplants.2004.01.008" />
<meta name="citation_reference" content="citation_journal_title=Journal of Cell Science;citation_journal_abbrev=J. Cell Sci.;citation_author=F. Tessadori;citation_author=M.-C. Chupeau;citation_author=Y. Chupeau;citation_author=M. Knip;citation_author=S. Germann;citation_title=Large-scale dissociation and sequential reassembly of pericentric heterochromatin in dedifferentiated Arabidopsis cells;citation_pages=1200-1208;citation_volume=120;citation_year=2007;citation_issue=7;citation_pmid=17376962;citation_doi=10.1242/jcs.000026" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=F. Tessadori;citation_author=R. K. Schulkes;citation_author=R. van Driel;citation_author=P. Fransz;citation_title=Light-regulated large-scale reorganization of chromatin during the floral transition in Arabidopsis.;citation_pages=848-857;citation_volume=50;citation_year=2007;citation_issue=5;citation_pmid=17470059;citation_doi=10.1111/j.1365-313X.2007.03093.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=F. Tessadori;citation_author=M. van Zanten;citation_author=P. Pavlova;citation_author=R. Clifton;citation_author=F. Pontvianne;citation_title=Phytochrome B and histone deacetylase 6 control light-induced chromatin compaction in Arabidopsis thaliana.;citation_pages=e1000638-e1000638;citation_volume=5;citation_year=2009;citation_issue=9;citation_pmid=19730687;citation_doi=10.1371/journal.pgen.1000638" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=M. van Zanten;citation_author=F. Tessadori;citation_author=F. McLoughlin;citation_author=R. Smith;citation_author=F. F. Millenaar;citation_title=Photoreceptors CRYTOCHROME2 and Phytochrome B control chromatin compaction in Arabidopsis.;citation_volume=154;citation_year=2010;citation_pmid=20935177;citation_doi=10.1104/pp.110.164616" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. van Zanten;citation_author=M. A. Koini;citation_author=R. Geyer;citation_author=Y. Liu;citation_author=V. Brambilla;citation_title=Seed maturation in Arabidopsis thaliana is characterized by nuclear size reduction and increased chromatin condensation.;citation_volume=108;citation_year=2011;citation_pmid=22123962;citation_doi=10.1073/pnas.1117726108" />
<meta name="citation_reference" content="citation_journal_title=Molecular Plant;citation_journal_abbrev=Mol Plant;citation_author=M. van Zanten;citation_author=F. Tessadori;citation_author=A. J. Peeters;citation_author=P. Fransz;citation_title=Shedding light on large-scale chromatin reorganization in Arabidopsis thaliana.;citation_pages=583-590;citation_volume=5;citation_year=2012;citation_issue=3;citation_pmid=22528207;citation_doi=10.1093/mp/sss030" />
<meta name="citation_reference" content="citation_title=Environment-induced chromatin reorganisation and plant acclimation;citation_author=M. van Zanten;citation_author=F. Tessadori;citation_author=A. J. M. Peeters;citation_author=P. Fransz;citation_pages=21-40;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Journal of cellular biochemistry;citation_journal_abbrev=J Cell Biochem;citation_author=A. D. Walters;citation_author=A. Bommakanti;citation_author=O. Cohen-Fix;citation_title=Shaping the nucleus: factors and forces.;citation_pages=2813-2821;citation_volume=113;citation_year=2012;citation_issue=9;citation_pmid=22566057;citation_doi=10.1002/jcb.24178" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=L.-C. Wang;citation_author=J.-R. Wu;citation_author=W.-L. Chang;citation_author=C.-H. Yeh;citation_author=Y.-T. Ke;citation_title=Arabidopsis HIT4 encodes a novel chromocentre-localized protein involved in the heat reactivation of transcriptionally silent loci and is essential for heat tolerance in plants.;citation_volume=64;citation_year=2013;citation_pmid=23408827;citation_doi=10.1093/jxb/ert030" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=D. Weigel;citation_title=Natural variation in Arabidopsis: from molecular genetics to ecological genomics.;citation_volume=158;citation_year=2012;citation_pmid=22147517;citation_doi=10.1104/pp.111.189845" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=R. Yelagandula;citation_author=H. Stroud;citation_author=S. Holec;citation_author=K. Zhou;citation_author=S. Feng;citation_title=The histone variant H2A.W defines heterochromatin and promotes chromatin condensation in Arabidopsis.;citation_pages=98-109;citation_volume=158;citation_year=2014;citation_pmid=24995981;citation_doi=10.1016/j.cell.2014.06.006" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genetic Dissection of Morphometric Traits Reveals That Phytochrome B Affects Nucleus Size and Heterochromatin Organization in Arabidopsis thaliana" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="twitter:description" content="Microscopically visible chromatin is partitioned into two major components in Arabidopsis thaliana nuclei. On one hand, chromocenters are conspicuous foci of highly condensed “heterochromatic” domains that contain mostly repeated sequences. On the other hand, less condensed and gene-rich “euchromatin” emanates from these chromocenters. This differentiation, together with the dynamic nature of chromatin compaction in response to developmental and environmental stimuli, makes Arabidopsis a powerful system for studying chromatin organization and dynamics. Heterochromatin dynamics can be monitored by measuring the Heterochromatin Index, i.e. , the proportion of nuclei displaying well-defined chromocenters, or the DNA fraction of chromocenters (relative heterochromatin fraction). Both measures are composite traits, thus their values represent the sum of effects of various underlying morphometric properties. We exploited genetic variation between natural occurring accessions to determine the genetic basis of individual nucleus and chromocenter morphometric parameters (area, perimeter, density, roundness, and heterogeneity) that together determine chromatin compaction. Our novel reductionist genetic approach revealed quantitative trait loci (QTL) for all measured traits. Genomic colocalization among QTL was limited, which suggests a complex genetic regulation of chromatin compaction. Yet genomic intervals of QTL for nucleus size (area and perimeter) both overlap with a known QTL for heterochromatin compaction that is explained by natural polymorphism in the red/far-red light and temperature receptor Phytochrome B. Mutant analyses and genetic complementation assays show that Phytochrome B is a negative regulator of nucleus size, revealing that perception of climatic conditions by a Phytochrome-mediated hub is a major determinant for coordinating nucleus size and heterochromatin compaction." />
<meta name="og-title" property="og:title" content="Genetic Dissection of Morphometric Traits Reveals That Phytochrome B Affects Nucleus Size and Heterochromatin Organization in Arabidopsis thaliana" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/8/2519" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Microscopically visible chromatin is partitioned into two major components in Arabidopsis thaliana nuclei. On one hand, chromocenters are conspicuous foci of highly condensed “heterochromatic” domains that contain mostly repeated sequences. On the other hand, less condensed and gene-rich “euchromatin” emanates from these chromocenters. This differentiation, together with the dynamic nature of chromatin compaction in response to developmental and environmental stimuli, makes Arabidopsis a powerful system for studying chromatin organization and dynamics. Heterochromatin dynamics can be monitored by measuring the Heterochromatin Index, i.e. , the proportion of nuclei displaying well-defined chromocenters, or the DNA fraction of chromocenters (relative heterochromatin fraction). Both measures are composite traits, thus their values represent the sum of effects of various underlying morphometric properties. We exploited genetic variation between natural occurring accessions to determine the genetic basis of individual nucleus and chromocenter morphometric parameters (area, perimeter, density, roundness, and heterogeneity) that together determine chromatin compaction. Our novel reductionist genetic approach revealed quantitative trait loci (QTL) for all measured traits. Genomic colocalization among QTL was limited, which suggests a complex genetic regulation of chromatin compaction. Yet genomic intervals of QTL for nucleus size (area and perimeter) both overlap with a known QTL for heterochromatin compaction that is explained by natural polymorphism in the red/far-red light and temperature receptor Phytochrome B. Mutant analyses and genetic complementation assays show that Phytochrome B is a negative regulator of nucleus size, revealing that perception of climatic conditions by a Phytochrome-mediated hub is a major determinant for coordinating nucleus size and heterochromatin compaction." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
    <title>Genetic Dissection of Morphometric Traits Reveals That Phytochrome B Affects Nucleus Size and Heterochromatin Organization in Arabidopsis thaliana | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TBLpvMJMPsoKbWeSBa3UfYGVnwDYi9c8wKnqgk92axU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485840","apath":"\/ggg\/7\/8\/2519.atom","pisa":"ggg;7\/8\/2519","ac":{"\/ggg\/7\/8\/2519.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/8\/2519.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2519"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2519"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/8\/2519":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HCou\\\\\\\/3obW1wUlJMhVyMJflNCCwKgRzbzexJyhxJUacREM3vmB8OnWo525kwJL+dinxSXhkcPjgqR1gw7Po6rdsbniv6p\\\\\\\/bOOq1PiS7V\\\\\\\/kTmr68drod3Eh\\\\\\\/XHBj1PCm7+AJYOGsKSaqsixRx7E\\\\\\\/OOt9U+iVHollhWlF91fNYq6rcGN5Bfbn8wgGLYWB\\\\\\\/Pg4KffCsF7wE4Z0lFh6lMM3YPFMKdUsG+JhY5bXPQjuLc1SfdiBCbSRoz1KBPxKpdipy9RjX05X9uwtBuu+FGVFw5maahps9SlLAIWmdseIyNevHDMb8jm8RENXAGkrigi7ODI4dNxm\\\\\\\/qIeXL+2uWXCiAThJxckP\\\\\\\/fSY8qx9P1KRm+IaXmWxYu2Yhy+d2FVeoXxdoVku+hEJnpRmp0clrT+Pyrwq6OWXOs2ljzH3XfamWGgutgS6R1WvJ997tyADVechYk2GY\\\\\\\/RyByrRLZnrbbX4Z0drto7YMCQ8F0cbLJSdlihqTwqwyzbSTc1+YP1l+VNZjd1Jqkw288wR+wYZ7JnCjAUApzD3SCQlM2kEhBdT3Fdb+\\\\\\\/ClAZ00d4jQxiK2GCegS3Zet1aQ\\\\\\\/CzbQWggBbCs7yp2ULakeO9kWT8MEK42qkaB7fmAElNDGXXPGi\\\\\\\/S7xX8IjcPBX5vhrn6qZ6foQLsnXsKhT\\\\\\\/H0HJf9Xfl5d3R5tIQboUZK7GaAlyMRsh6D9cacnRxsLc8bchu8XqZf+qgd7ul3sbWmsph7rOWJWJzJxEpE65rS5hWvbEWygaeDSjouvBEJP7npZJpdQhLeh7KLpTuTTXdIkQ\\\\\\\/TY52W\\\\\\\/FTN62anRZwommdSGbiWRHTb0\\\\\\\/sBPWNRHPRKZaHT85Jx57WpG0DoUQZu\\\\\\\/SgP1l4rBG4gu1mFxuPey3y+nsSAHkYn\\\\\\\/IDEePYPSUJHIMC5dG43jFnGUGw\\\\\\\/Jl8VbP+I0mCO2oZCsKfMQQ5JDFFHZ+ssduSvM16AUATOCI\\\\\\\/nSaJjy3YnYqk1OWhwrGu4dyIJIxV3etvveK5fUTlLwNBVxsiFPU03QVuNL7neA232DugUUee9hTVtdFjx1bEnVotZD8wcjVRM1LR8ehA9SnZVLN8AJQowhz92Iy1q+LdHgxDPOSt7Vm6iohcQhEysM9VeAy45SHYIL64JcTiW0MbfkOCRbJMsgbViNmr87nLO16B7QPEEdtivhSAOnXsWDQBJxatamuqL6UOfEA\\\\\\\/RF5MrjwuQVfKwuwiAQ5YYXChT\\\\\\\/janODJQ\\\\\\\/1mgkcdvaccUseEf0cGqwedj2eZz2xxF\\\\\\\/OaR6FGp1lQ3qI94\\\u0022,\\\u0022iv\\\u0022:\\\u0022LMDwl+\\\\\\\/lKP72ewY1Ox+\\\\\\\/sA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ba7f7ee1eb9c607bc718ff677fed01544ab505d738e24a99e009cbea31085818\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uAk2dp3vFxoohS2GYHLBCQikqN+QahEkcsOIDj1gunzXE4Lof1HCau9+LeeCEnhdCTEO4wsJsYqZBlCOfHQpZXNrvc\\\\\\\/QQPowk9a+pdrUxXeWYvZz0jh3Rt4JZt5kVoHQkeFQ9pBpJPZTF0Fk3YDxBi9cxC3XZaWGPTmO+8mN24VA9oCdKw0OvBVAx\\\\\\\/aQLfOSMm5SQexvSzCKhuHJopVhbV+NGP8hNzkM+4nctsBUMm+zlSbQV8CggYz6XBek6TJw0zLTH1zoBI3B12K2cJWOq7Ipj5SouHKtxPwLXMY2\\\\\\\/ruAM6hxp+KV5TrLmOhKlpHAZTtNiwJ2eXzizl+vscwD8HBQhWxjp+Qul9wI4Yy3KzIdmtbcAZ9Sbz3N4sdhTyrR7s4l49q78BCbVmwGcmVnea7JKIP+kvDYjvUHs6kAs0U2U7PnKMaUV4oiJqN0sDSw4rXEH3TKI2xoqAxB9be92hL5mpIwexUnTg7ZNygc7AMD10Q1X\\\\\\\/8UhE5\\\\\\\/5ynfmluMnIS2cxpbzpKKVTmukRNDmZ\\\\\\\/xUIrbneu38CeCRSBFJ1wlyPiEj7mNhNZhkG2XyrVp0mwqnoDrIFFmm\\\\\\\/fe91LD8obOiZijwVcQClGN350MyFRzu2l7lzsfmVh7RbztZU2groApWbGxyVmzdu0BZ5F\\\\\\\/1DqA8VTJhqNWnbXLFnYwI1VDCsdE172\\\\\\\/Ca7yi5dyQ80EiyVOT+alOdCOXjPbapa9DR5u3gp4XPBfJq4GbDyZJXaC9+oY059YXoiANDJEDlLpSmIQFwIYwUSdCe3CfS5MQtHqapybUWKrX68YW8UJmPukwtK4Ymo8K1QbVNw4BiKsu4qWRuQNcReoaWxSuacr6W7D9PYHxDqxbQMiQiJFEdS+B0shb2qje5uwEOhaHyHGrEUpxk20mQP6R7XjK7L3UrJEPgYe8Ka5TPV0+zkSW7fGsv25JPHtfCrK7nlaKMEJE4ncWByrLKloNXDH20dM6HVWN0FNmhGD4MBqsvbAW8Mwcgmde2zAFk7C1gJwlFJU9jAixcJOabhxMZDAuzUuUfNiCEiE3LxW36wncBf3vpfBYz+ANxRmTnlMxVC7O7PwEC9Y\\\\\\\/Ud4HmwEY\\\\\\\/XBoia\\\\\\\/cC+kcbGjz38pg+ZPHkzjy9cGL+XqzDnqCauvDVCpa4ClXQHKdxUGSHFhTDsz6p9uZLYjk+ORKntoawCBA9T4+wFFQ2Re+Ii6rr0RqlmR6D8+4spS3Z5\\\\\\\/TXL2V4EOKqv5aGd\\\\\\\/KYurlLhEoHI7nTsKh8EFN3RDx3nTVtbWtswpxWAn\\\\\\\/Plwge3qObSYvcqIYSsoqs9L85q6WMKzN20u\\\\\\\/LxvYj2XGL8IDG4vROBTZaRGkV3tgr0\\\\\\\/KbAvdlVlnBULMkAwjX\\\\\\\/Eje0E3fOjczkszojvRq4SQeUqrNYBlXYZ3bKPcSkq0M6uL0yKSmtZf6UQfj\\\\\\\/CZptqjcwYsSy3avPBfCFM4KTWJgc\\\\\\\/QeWuxPxwEKq0s1zijgssmpEwGcr3bvwyQ1FnEZpDecP5g\\\\\\\/MMFPKvGJ72J6kUYgWQgi\\\\\\\/2PZ6sZzjWBnGM1cxreP3lIsPwPZ\\\\\\\/QkFszV0KKjQgcckHekel9aWC8pidQC06kEl40Sz2m+P+KVHr9ENmhPxcWkdz413yumkDrTw0xZ2e9kLWLZAYhP9lYznaWcLy4Jjh0Y4gTMBKkeelzqm2GCJ+bbDt2GDiU1PNqB7k6yx+RS41Fp8hdCAYbAUCqGepcU\\\\\\\/3RxdYvB7FOpMIAJ5isfc3IsTzu4fTp058nC4SipopisXm+kjaDYC54XxAKpTyDGf3b+OgnnU6zPIL\\\\\\\/bR52HJXN4MZRqbPdslvy8C8m9oR5lcbs85GKByZDUIGKm9PqF\\\\\\\/sgUZuLxrVX0CLyGM\\\\\\\/jLujt7Ug2T2RbTfd8gzUrP82dfNvNPiDEANa628mx44dtrGfz5DEVjYnl4\\\\\\\/zDbnmNAM+IbZs1fyyof2RXsiS1AEZ8qJbLdCHC4pRbDIgXj7B1GrOX9N\\\\\\\/OPVwCFfqlEiL2mGfTWZqkywSOYHy1zkx2KiDwuOOVepkgeAko72khrGfUqEEZk3ErGNF8bEUmsMEJtIg3BYOA6iFMKog7WmywwXdJiBYXDRdAz0jeeNB6ATsHNjt5AVtdZi22VST8ZInf5INkxH\\\\\\\/kyxtWVqBwTyYhIXuHH+Xced9B+t9GjSMM2jmgQPSLp8DnR\\\\\\\/0wcmShHzF93Au+oFiD\\\\\\\/y5ueUxwkPQTqdiTko3vkCjp3EED\\\\\\\/\\\\\\\/xegwFh2JJfHuIB4fCi2\\\\\\\/wSPk9prT9vtcU0MRV9bkf\\\\\\\/FnKL0AGG5xjclq\\\\\\\/aASrMTTo2luABznUH8nOi03b1n5n0OWo3LQ2O5m01FObOk8zkaJYayECFU2gKVDb1kGHIGdOebEQypprT+KQqZ6vCiq\\\\\\\/ad39YXVuLrJVI3WEABZrlgmCsTS5CrsSlVciLYEuwTYC+6U5oQlrFdcZZ9NqWGmKd4EVkgjoJ\\\\\\\/rFSdTMh3vDtP4\\\\\\\/tKdyleuyDR79GfCxkL+QgR6zZLQLeOKHhBlpgFMECK0saVstMrn6CUNyJtpnf7PgonhAVLdTH9ZUi45tk+ShukrE7P2wlx2N8\\\\\\\/A40GIkZ5cGc1Iwojf254tL9opfefRFF54hz0X8sjmfQFU\\\\\\\/QSRKGO2fQFp+b8D0wtXaCqhacOd4PLauF8eVPun5DLvl55Jbsz\\\\\\\/KPqdwZWPRZE3eUQbIHaFJWYHWi03AajXx96R0EPK0lDfJ3KwQGrKH3Ri3G064DuzNCqtgN8D8iQh8fXQbO5\\\\\\\/E0QwxMEAQZV0imEs4CRhBsBgaqXAN0OLINds0k\\\\\\\/3EwRplqhnZFLf4vwztJAv15zx39Pvo6lrsCHR790ynCE\\\\\\\/nPnF6iUb0smpwahKkBsoCtSHOhyyM+WkoHTPQboFfwVvppcS2a91NSyja0qZTbEBJvuf\\\\\\\/Or1sOeFKue2qg7uy2MWxLfBjHS10JlEv5MWc9NhImIMasLmTkWvecT0Ala5y+9LpoBhOE6\\\\\\\/J1daoOxQ+RdgQXqnc+rFcSiRXrA5SI3Irpw0RB1dJCnjbuQzENwtY+iQIkgVrQC8MARc8RuA+9Pw+rYzGfuTRKhaP7edhNTZqV0VDfJaL7YvodZaidbMg4wkMfJ41v5qBO5g1eylceRpk5TXSK0QibQ2f0Txk6kIS6ur9CIj1cWcIENYAE47T1DzVegM2nxMnOYzvBCUZxERre0eGJe3G2egA+dwal1IpwwkQMg160KtfYJihfMCTPeNwHvSdHjjBb0GtRX2XoCaD6qbpTlYCw1YgYM+SScLzpfEJ1bYVzNKTNJeaz98FZOMUNgX2zahbTZ\\\\\\\/S+9nUd5CyRkr3MRJwwSNPHvl7Trsj2b1Df\\\\\\\/3M974o6b6UUoV4Y\\\\\\\/CrBIAIlo\\\\\\\/q1uQ4IlBxhAxnhkeyGF1PGUfLJJpvH5bUd5FWIxLY7jTkQTSzkoAw7q2u8KdNXW3540\\\\\\\/EFiz6usxR+NsKcT6ehpXjcR8O\\\\\\\/YHrI8uMuzKK1IGkxzEAuGfYKY7uY2L+eok6cnpIPPXzKk15cy7yOPgjfe3trE7nfDV7aqAhnlBCQClk6EDRMZJxoTuEKAMkFVIITx4a2aU7L0U3Sb8KQsBQfVtS\\\\\\\/DJzSwPnfMsAwYB6eZuGcEqBz0edQ++cW8y12bDa1vxOaKbnjatNEvilZDWuTzlTWrDlglaEzjNDpSUQf8f3w0Cs7EaHj31oYQlJEj2\\\\\\\/AHhET9hkjQBQfnjaRBIOt+Cdeyx0LX8doBHuUQ1PTc7LW1yh7Vr\\\\\\\/lr7uOf0xFf2Zz8jjfSw2gfs4022m2Ka4c+2h0a+ML0WA5EURCXMLK2xjYe6vFBgue+MxEQdf49vqzuHCeIc1ifz01aLhIp\\\\\\\/x7Pg1OEZH6HP4a8TJN5YKimwfT1aedvYzt8YutscngEDBfslzkWbaKZn3sq59GvdFzvJ\\\\\\\/0N6Csx+4as4qEn9pe01IyVyRRJsqVmM6ADF78DMCvCzgNmWNhZ09UiC48RF79PCU\\\\\\\/HNFYpERZ3CzwFxVIzP7y2kscKFAYvVSgHfV1V8ccpoZTuF9AH0eJ4CLV3U2m9PCQKrVpl\\\\\\\/ccO+dKsg8Dv6fpeUzlg8h\\\\\\\/gdfTHuAfwXS8QllhDpubvSEvWwgdhM\\\\\\\/tDdXwCliyoWD24D\\\\\\\/+Bcr0Xo26AR0DQZ0R9CyzdXWRL+Uu9i9DdA3Z7aWf\\\\\\\/3Xp571zQdA\\\\\\\/Kt4mGNLmWUa4xjSraGS1YjUPJbX+2aHGLViADJrQtvXg24LEhklNByWZ\\\\\\\/0AU56Xon9ptIQaPyCVfbe70U9wM7xGDs02052\\\\\\\/YJfI5bfKzIo+sPXZ\\\\\\\/W6qYVf5EZlu3ZLOfMb0YyBPQiwWIcsTfYXWL\\\\\\\/dLNIp2XtXPTw4M4E2uEIATQEA0w2ONLx8PWT37L1fpC\\\\\\\/YAV4hCf\\\\\\\/6S7fmkrM+Y5CAryzFPb9FxPGkgrfJdcFeCz5X22CxFw\\\\\\\/7wDow1RiQwL1W\\\\\\\/4fw2a\\\\\\\/JG7clT8FFwb2l8oOj2Do2Hkwq5eRqJUrT8UATnK0bgWXhKifi3O+PB9Sx09Nopk2cJHSkVQLcales8B4qMXyNZXzhpzlgm+dlCZyOPshDxaZpWK7jKGgS9enwVsJXaehhDSAyi+mMXLkgpZ\\\\\\\/c73eaGpo+oUbT2pN\\\\\\\/UKXkgNGyNKJSNJ4C\\\\\\\/5XJd2hgKjhljfMd1LRWXPIIjDTjz8hVNa78F1+Vq3oGd+XXnkHxqk3YF5dRLh5DZhc9S1z8nr238+uRFtLblXF8IKt1lOA164YUE1aaJkBQ9ZZyrkqAg1RrffuO8usy8y\\\\\\\/+epjptdPhaIcpF+oVHvVO1krgm2urFeqQkiTIhZekhszT1+\\\\\\\/cbyVMZCFJOkHDipoMtKb7ejtZrpEKtkoMfmBJ5JenDCO5Kucd3bDEomiTCm4FY6KGyxD2hLBZa3ej3KHnHYnMmAyXJ9\\\\\\\/\\\\\\\/mNpNv68ZX+YFAqoFRXPSvI5opARGQAc4hMq14mxHsIPhj59Btw+PBo1xm9oLGqIZcGLU08XKK6Zz2Te6lV6pmeyEPXZekyhrXyVLUBnTuM1OMp8WPjmJA4teU86361u+9SQNJhLc\\\\\\\/qnGw9WKgOfwDRebwJquEkD5ududXOuFYSfowcNVwci5bhtUUCZwaBr7BuK2vXj0s7iXEMs4xBXUK4k65lP+pIB19m4R5izkkBhnjR6KOH+QYCS1OhFqx+GHn8bk1ko9xmFzmmziAY8Lilk7bEqozZcczT10HX8XDyMJ0F9CsMIZbuniz9fL\\\\\\\/lPtZAgoSAOdxwooC4FP5p+Httc2LkdS3IXApUFsgkmBAY9wbjwO\\\\\\\/rfkjH\\\\\\\/Aq9b8NHcqcZdMIILfLvrJzO7QtE2K9sR63cq50kRMHmH8lmSLEV9Xkk8+NVYvjN3rJsyZaCyaFnDh0o0hjP+U8tBqnLC++4a5h+8ZWmTbYuAyMtNJapjT6rzh5yHEeK+X027y9Ljxssmi2z4\\\\\\\/hWrxvmVtNGoLs9FKNw5DmF7xGspUKnQnA4e25s647wUSwAuPU0W6ekWV2A9ybWzUdIfNoA4nl0bkL\\\\\\\/BnPpy5XeY\\\\\\\/EqhRsKhlcfZm2gCTFcfwkbxZXY6aH79RLZ38YZjjXqtqvMeMsxTwo2hbViX0xxb01ri2N1Xkvryc2Bb9KbKG7kUfkA8N8mTM\\\\\\\/VGE\\\\\\\/0lq5R3ME1yAWtQ03EBRyGs8L71GVUUQPjPBpZsGgNpTRzeZNi1FeuHzcNvs5jZJ8FsxtVGBiugRPNj4OSQzHHQl+RQeAuU0Tgwg4TZH78K5u6N45Wl1hkW+1Ld3IzjqG7saG4JJ2DiH63B1ERy\\\\\\\/v6a8LY2FA8ziwMSnFe0VzB7zFckOgoX9OBn7XkETbJV9SgOUsg8aT2FGMhMK7Hvk1i+eLFBSgdozHwxVDscVzzP+OKza22o2lu0mTZhRqemJqEdfi4VQjDUGL9Eb3+fDuwShd\\\\\\\/6hqArDMedHYLuomsrmxQvzDWtWrYk8sIB44CezveIZ4E4MI44vcWlOS9x\\\\\\\/0km82YKBZdh4TU435AIi6HXo6Kfw74Xq6zQd3sVZ1twPFDq\\\\\\\/7w9XJb1anGbetd0EmdtnSgBFqEav\\\\\\\/WtqtkACJlcSCexVhJ8Ga6XA8r9I+R8gcWtbfzgqVp65\\\\\\\/mpa3zFrHGqSlvT\\\\\\\/UTCC7c123ps9eh1ZcgZQvcmkoyVrcu1VuemfrY7yaV4UhR7kho\\\\\\\/PMS11jycaQOdGFsU0MelbsCW8rCesdoinyrpGFvrQ6LyMDmah5jqHy3egjGHgmHO3mZtWMaK\\\\\\\/NQY5FPS7eSJNP4tn8RSgEbj1ByW+pOv\\\\\\\/5wyp67DOigF9PZcDAHWLeRy5TdoaIWGSUe2ya3CXJfNWIYulRZJCt8H2kAboUQVRDjc36IrPJbWL3idP1sXWfmCEC6JNMHSM84BTOcci9z4OCwaOJMDYRexdAUGfbJz0oIRNDfXhvoWCw53x7ScK6mioFFHNlObfEEzjpRsgMVnXSi26nzcrMyxtLLRPZ9WFQ6mThq\\\\\\\/ECUVKGBs3lzcI4I\\\\\\\/5kTUEvOp8FgY7PQd8IsKzUDJVVZ7QqBNn4bPT\\\\\\\/ZmW9oqSim+opKULwkRLbRgUrD17TgaD1V3IecewBn\\\\\\\/p05jUfXkGuS4BHxF0GgRHhtwD+JOjwiTzoRHTAsplUISLrJq7BWeW2fNhZM0aFJ4ZKdCgbZcl5zXCXbv3+qNr6ORCEKuGv+P92\\\\\\\/Qke09GwYFtV3n3ECN82MYyDmw8RdpMowYxhaof97scu0yj+jhn1t4p3lfLeeWIBvLER8wJvUuxvS0uQ39qRAEEkZLrBmpdOXpUWcOnx87xpuJuP1QqS9Z5GSvnGORJL3qtX3MIpC38SAkGcoIwiIQkwYun5A2yLjwyWZ6EH0Q7SjxkZnHBuyMUb5kJHT0arI6qvaZB2ukYk9jtrSk2yo2Q\\\\\\\/qLVzt3aupZkc1eU1kriOvD77QP5j86TiQlOPPF0pPOQnJonspiuR1H\\\\\\\/dmZRg298jqv7O6C1Q7ulKOB44Fm3LCgPSW1CtwNlH58BYTL0Jf6DutrV4zEhdy31zKRBCxlRZomXGG7wRr3T1WHLfYpucchXG5lfVBXr0Ux50xyIHLSuV05aU61tMwibwbhoC2RAAKSzyZytxU9MJJxjDnfURLZ5VYAzrpbbuFIYyX3\\\\\\\/3D9UdHHUghyPkEXnQrWCJkOg7kOgLI65vIU8uqXnJVN4kKaOT5ZUjVzQt2b0bw9JBsQFFe6+AI6TnnNfzvL+S+pGxBbZm7Ws7HoA\\\\\\\/CgZYVl2ctcBTrqOLcgj0M5wJANOtitNCLqmQNCiIJI9HiMoxn850y49HxpvCmcgWgA4U3SgXJ93zShY4G+1J+NLwrIUYoUo4+3TAQXbzi83fa+9lTg8NwY8SfXnUef\\\\\\\/NHbTdvwAb\\\\\\\/yAANTLVjIJj0pYJ\\\\\\\/kdBq7WEhJLv+d1TEXcsLl1Iz4+\\\\\\\/X8sCsDLUZ9odox7oUniX00WgQhPryoAtDuuMfy8NYX0mb2yFuDL+SlXOqzUjB2gVxtluvHzEcnEqbAK25KnG7DOVzaDRJxBJj20nDm0BAT0JfoXuLUobUW6DQ+4+EgrwhQNu77Vbze28OTJ6JqF+GyiZv59X6fnk4b9Z2Oc\\\\\\\/LdgpwSpUF+4JcsqOCmhlz5VkcwxpSbe3l41onza0GSRjXEELiRWbXC2FdOP27rFgQVjdx9rz3oJcZpPUbuYiyheeNUdYLcjxzlvxMIBdm3o8RJy\\\\\\\/qREgoIuflbAWlxysXesMoVm9Ok0SfW1rp\\\\\\\/YrNvAZSKW5I+AGQNlMV1G8Y7MorsVuOD8OTT8oVH5FuuEpF4QDmeDhKA56QGuVjEO5PBCT240cF+4F1Vno5lE0+x1g\\\\\\\/8knzHU2nhd1ataGhPmESVPQ0GXnL3jj6FjiV40sGUWUzhaQ23314Dbi\\\\\\\/CAVH4q4cWjqV2BiW+iOeuAZdEsAtoAs9mA0OsH14QZCFGJL+OqG2AIBRep9\\\\\\\/wEOjD+IqNBaZHqN2koK2J5BqwjteyH560FSOpw1SAA4zwVdf6vfFS3+A6W34sidrS1lwmKTeFDUHiFD4Bna6aBEMXyMSldcpD7f1kOqfVphSfZ2g31TnPJ8EsEnFcd91KJ4McoimGEU4oOasVjUTXnMGPmCi3M+iWtnIIGwfmQ8EdyAIBTauO6HIp2ha8SdAHEeLzd6rShqOqJEH48D9nYSL2kaG8wDzuEcCzRHmFOJVJiCdt4QwcuBPp0\\\\\\\/Bnp1W3bjAmRCItF6a1PNnUIARZB4yP4\\\\\\\/SkvWHuerZjCN4knzYRUNwEUeJeyS3in1JeaGcvlFwWX+6f5Dde1ubFB63n+tucKWTLg3zAP7+OFN+PsJu5mEXEsM0y42nD7jAWs95hhdeRSTEmD5pcMPaaym5itHAOSDtFiXBGmLPEJfjSI\\\\\\\/MDXtJFhBtNHJteSenPUONHb+ROz5z\\\\\\\/Lcu5QpewnLLOJtFKNrO05LG5JqWjglITgU6xqx1dMfYLkQNrwXn5trkI1SF12EJNGDAGCka7Iory\\\\\\\/aO6rgAstgd2tH5bGXJwTeKsG4lO6FIfIcaCHnVOjycFLXvuY3jV19vzy0idW0xdGr+JztPjAqt\\\\\\\/T4aJ55L66A4f8xRe+e9QiQdVBRkmhPQkKCdvBiDPa4ObrTFJAMqgU7t3\\\\\\\/AcAZTUDLEE2GPpXH\\\\\\\/0MFEiJIpluW6K9cFQTqQhi9sL00g277YvsD0UkWl4rwq8BN4UacZDFFNqHTMtwmh1ZIiNtMG++5PbbIz0MO+gPYMYRpNtql\\\\\\\/6WBdaHDaaQHEpVbom0VzlGPAFwj4XDQ3DPUk4MbaYOhyQATKJnibWj2S+XvPzwyA08cdVefO5w3fAOJEcj2fVbO1ZGhnvkbL9zKaogn6GUyPJdGTxZfoiXL\\\\\\\/MY+FE+V3qCcClayPmlE5dW+lYxUcC5noXbYS\\\\\\\/446AdrkNBNElfAnYo2gQ5zO\\\\\\\/bhiDZLcK8SwUHjIm2RJVLrNEZzZHRG1aCVP1pmOaxUmaZsSaTlC2cMi4yvzrJPSdRUQowDvc96sGRYkNpQnIJWRHqsVyVJOgM8Ao6eik9W5GAKeIEe008V0W5eHYOulD9MELE483w2ZC7VBwdkdNch2wNw\\\\\\\/JiLPgEk9bFSkNpLEkKs9xLpySB0W9nSXTYtpsFeZm7eMt+KH2K6meAxzukK1kxNCkSOH3tCU8FkG\\\\\\\/4Ii3IT7Lv7mNbqEWNVn4EnS0VMNBWPFcy8uqfI5eeT23TKkQ4DMQkRTO6PcDHxyE46nkkdXNmXDREgab9uibGu3OIA0HIriO2HCqLVuSvJKFa3f5Yuc9J4CPe41LNGuDzqnoBS3mtQOT6+XsMyiNo7EnxdA03O2Qtji2CoLxVDJAtx6NWjiBFAuO+JTl84XT5i0YogD6GYV87mkObC7DrcuN9MucwcKEyMrfY8kpE6vd\\\\\\\/H\\\\\\\/UAqpspa44p8mF\\\\\\\/yALWEvZnnHTp32ZgwXI0PXlzgq5tlb9KGxABiopBzkVqPvM7q+5btJxj76xgr7xcluhGBebA\\\\\\\/xyEafo7DjnxQGxigq3MCwjHSgkX922PWlCv8kHo3fv3F0ovpMXu5UGsMwYkFlbPRn3IhMOlneahjTYbgSpSKHBkgmJgPIcKEGnqkEfSQHqf6ePO8da0odlplABM9LTYsGYKszVaL73gFAvPxS2i\\\\\\\/cbY1zWJm5x+JuacnMNPbRj6mM1umMCbAhHXfWDgcXDTTt\\\\\\\/Yu4moxfeFPBtMhpefBpZKHY7i9ziVsPGRrOOrNpsozYg8JFGwXyvDjCAMtTIiCAj\\\\\\\/nARpZwNofY9K\\\\\\\/2CUJmTtPwTOaIQoDjam5O84TSFXFThL4walTPORoBpKyIijx3OK0HihdU67XONC6+KkV9ywdoHbq\\\\\\\/d0rc11tvoj2KjiS7APVjhk2xTvfmnpDKBNzeLqaRamkX2J6ST+oYNYXR6c0jfeUk4LRFEsOPrphSmPaDPoSEtriQcTAKyu\\\\\\\/zlPVdNKPuiJ2cdZnF5uk0LLvNxd53DfG28WKbC4dkOL\\\\\\\/TMCBEGNG7QmWz6f878YHG6+1e+dw3lk7B7iarSSvlTUWVExp1ebJdMnelICJvYZ7Q20Bj5Y4j4DeW3Acgby\\\\\\\/FCNs1nXBmhamRK8MVqyYaAMKneKMtEddprsLyEABwpNrDQPP7QCP\\\\\\\/Ro9H02E73qFsZwyNMwmG1XQfyTc5gwEMmi\\\\\\\/nxiyEn9ae\\\\\\\/6MNShnKo5y+a9qFAJQlWPeDQFszus9WdXdmuGaQOtDI4izCkLN00Z\\\\\\\/h6HHuXwo5Qmt5vlZ2hM5HtFdCsBCVgG70JtlsCCsR4CtTHBC3DTc+eYeogmW0DRA4o1D7pWH0q4aqrk6IlHtsDeZgLgCiCWxl\\\\\\\/6WJnLTldBuzmUZG41V\\\\\\\/0EqEwzW2\\\\\\\/zIKZMJQrUFgL8wpWgaqbmjPeWVzOYc5sodZELJ5\\\\\\\/M\\\\\\\/Hu2OZ0FArrLPIfhN09YyLRUu0SVmbs434zX4nrX7bJn9XCIeVbqikzhjG1MVTl474KFwmnxNEGUMukKbnHyaA5LTqoq4MDKcKvgjZyvCaBr+16JtgVqN\\\\\\\/EXOgMqVecvTI5YsuqBVCpdJaBWuWaRY219c0wy7NtOpTexd9LBAJi\\\\\\\/qHsLok0IyRuQ606\\\\\\\/eq4PDooAo9gpUmkxiYC2KhhqZpmE+wUit\\\\\\\/j\\\\\\\/dUnEgzqnpas6fobbWxExixDyb7RH0F1E3VYPl1ehQrSLzESG2JwFh0YANd7VMhbROKHFn0pAVfnYHfc\\\\\\\/NXx52tmsk2mDct3V+htPz37aFtBIm0n2H0Vh44DO7CfqL7gWone6Z9nkKpngPgB7U2pcCF4HcGzim1+Gmp6UmLOVAdKQuUtP1nmTDUrJWTPORFTq6xDAYO6pLu889+7xyWIrBRo2eUwL2ySerTop4BDYSuObHolnxXSvz5XJdK0JCaEn1oznZXkdMn8iFtXZDdfmOEttPiXh29oupl\\\\\\\/JKA0JKXU4a2PlTu0rObkkZgqXWv2UCpe9ADlltjb\\\\\\\/bDpJnNrBWs3d7TjAJhKI\\\\\\\/72FbMreIc8e10p2oLhZtWZIfSE9F+i\\\\\\\/rNhIjeFwGY3LqflbuztI9AYQJ9u9SkpPc9ggQ9DG61V7J2YOFFMqepPyFobkIN9+16l3Js4i0y9KbbvDjlTl2Y0PYB0qNnI1UbtrxGdc5K2TaeigltBvpd9Bi2+zpqaCqlABoymxTZqePOlrJzVm+l9\\\\\\\/1fcKkZXqrHyShnhhGUyzTIoid6x6nI6TkyD0Xe3HIm5WqTSc1OWrCyiQYtKcrgp6U6Y4XbefflAFTI5zUg6kxEi8+ShW+BVanfFTcWuBeEix88TuL8EV5gev7w9DGnwhmW2Bp8GviMT\\\\\\\/lqd6UT6PtVsW89S4KPC0HmjaNsrjzsbij6+wihPyxvri7PFO2TfrLH\\\\\\\/f2I6Sj8WdbGgS4dB0z9omA8ifrdPoXYEuIwCbKSGpp3L7UqmOcgmg+8q+BlPwOxuGj8+W4Y+dhOV2aQyfqUXcfBDcHBQDGVJOUqIgOFu\\\\\\\/3B3360HgTfqLLIIP\\\\\\\/Wjgnun2HSFZDzupZ3Io9J716WW42br4CsUqa\\\\\\\/72v5GkVCdqatyTnRF++EXP894+0MmFsRLIru2N3oi3rDx4ofaD6eSRGHzPKFfuUML\\\\\\\/B8NJGJtngOV7rtwimFJsi7MAAFaIupm5Si0tTWcr30CLXToPtYtBPWzj6fAMoQkfS\\\\\\\/3LE\\\\\\\/RmO113r\\\\\\\/85QIIal4JuZX2XLpfjysk0MBu0z0E2wlMV3xVjyQYoEAA+e22KYlHQdTfZzAzo1vLFOjItu+TE0NSjSqD4V1+UD8o7kDrlf8jp9cb9TuuuHcFSqVqdD2BL1DfqkiC5PRRsWceU7kaiW2kZpbJ7rB+77yKPHYQB05ZLPYDm1HG+SZXLUUmP+cZZBsEPibbMgFdJZqvEX5Yhfn4cRj0geYB05d1a5srINInBxp+dseC70YXoQT2Kwxfb\\\\\\\/qIKHTsrJdy88FovR2k9RhzsMbzE9nuZqAnt4PcYHPmwugcrzX9yGIfyRwxP438oaa12LZITcjdqwQof2LnHJ5ZocBp9lCX0waGXVQ7Jtzy9PLuKR\\\\\\\/pdb9H7iIgTgNsuTh6TFhi8LDaR0sRjALwq6fNsuslwzg8olecXyXBrgXyKj86o6TzL9e\\\\\\\/Owzrlo\\\\\\\/2m9h7EiGdTbKFa4pxVzwEIqaI3SDGs\\\\\\\/6RfvEIi+HpegSdv9g+t2BA0Vuvs1oOa8j7N8LfhB\\\\\\\/yllzoYFwQ4PxuADoF9fiYbQHiIxnMcqGUEGYDZCZdGYyr1elrtPTnHAe03amti8TfnqanE0gL8wZLh73XsFOYfbpBbcdHkwjzrM\\\\\\\/APovMkPCQ0A93zT\\\\\\\/FdY38odarx9ZIWe3SNyto88K4DKQHcmw7inmRx7lBYQFj\\\\\\\/iOb1A0tpbbEHtM02N3UGWfXlzGOV6XBtDZk4BzwLVsWp8hZte8JzmSNri1BPvzfzObvg1a6XAg80ea5UjiXlgAVTReOcAjLuRHRp3WlZKSqp9AnKBYx7GCL5mFOaSRpE8apwIDktPobEc0TR5mzkHPZYE2b5\\\\\\\/\\\\\\\/ZJ1o\\\\\\\/9bOrLbyoaRtiARyOXeZtPdx8MO9zm0I6q\\\\\\\/S8mPiLtyvheV3fEnxNEhCGq+zlLqLhXsXfZzjsvQyneDPGH5OqC2CjSXq3jo9S50NvHoceprlbKnn4MCGli2giYHbhO8oLWy7MRYcx2u1vSnxA29hpj54PyAWitdEenyMilUBkpbF5zpy4VRn4LhIINJUIPnm7LIH77XmSdow3NOfLU7xc4Q32icrMEw8WpGjFle2LSjZxdDgnVIK4Ji1PnVl9y\\\\\\\/tVHU854VfrZG2dOsp+LiHM0qAX4eVChd\\\\\\\/0DVrSv6YToSZqtV1XSEPR9JrN86dCxSVEBO8y4ok6rOPcStsCGTF\\\\\\\/UArqy1s\\\\\\\/0YbWzCnEjobkZkZprFh44s+BTQ2Y2HZJ2w9hXHP1\\\\\\\/3MrOnglBSoQSIvOwaKAuSgZ\\\\\\\/kNyS1Ssed4OQnqnGFnjHQ7qnoEdeNX\\\\\\\/0Qnn4pSQWg0fQ9+euLT2J3mma0WBOWpfrzO9YRKNJUj\\\\\\\/GNP26KuXw70y1G0ve69M\\\\\\\/LcWmqasu4Xp8Nc6b8\\\\\\\/H8yEiYrRMC7iIl4o+KFcpL0D9Ue\\\\\\\/1xcLC6IHTcKGr\\\\\\\/7ZDQ7X+HmH3PgPpmyM5AxWn9n3kpY6VKBJXrkXksZXryrDNbNUlZUyno7g5B2yM\\\\\\\/cp7dkohslfqXB38JdnqvoTc+1PjfmjrlvZsjTmWPIYv5F2KVF9MUC2ZJrXGMZwL0E3ckt2FWc\\\\\\\/gZifrDd8y7mL74\\\\\\\/X2VFlVZvUtot4f39gzwf1AItV2p\\\\\\\/LxR8g0z4KRFU8GuBgYvCRL3FL3ZfUHh3q+AdUJwk5dyHm3v++JR+zy9H6L9Tm9Y04UPJKzAnc7x1nlLLKe2Ketj4EXJf+9S0AIwM1tMNDPRsO6LnbVB0Ylhu8K3JyV6ZghcjO3z+XFOnty+tv1lQ\\\\\\\/tpue4knz8ENU2y6Ex56Ia6hYtw7t4ce0VN20qoHXtPvdEi975nqAoxrl4ZDtKMFnRXnXJjlVG7ipELO2R3lHNDjYyiMGyJIVsL1LikcxIl2yA34DyyGvmBVgCXu6CPRl1Qz58OYKWpgtfSQCH1u+sT+P\\\\\\\/gbVkahN\\\\\\\/j+MPlcEHZTYXEddlQ+Or7pANXei5j9sgyPCfyShtx9ZFGOtIUc6agfdBID4IdRbinc8VhwHTz4C0lLk3opW7yRMFi1TwnencU3j6+jYybNWsYJclIUAowXrXydulrILMPYRVO5K3vcnkWbRg6WOmnmdz9QsGg0uVTt0nEyhjIfw3cYMkR9rEsfxpuJqsocZW2owUpfgtLlRNbO6hm0Idy7ga0iAg7GyUtZyW9zLoLi45Pb76NYSDvI3k2UdzAwMAOJiZjD825bqmlqMn1iv3DNIH0O3aITlG89jz\\\\\\\/Fdjho5gNL3Ag7yjaJj7eeE2m90y\\\\\\\/nv0+Za17ABnpGUHZF\\\\\\\/WJk9LSRO6IC5K3XyLN0zivuv0eVOu0Jr7KqMNLHwLINsNw3JCclOQs4Ya+P0lRWTnP5eZ\\\\\\\/ht7ufmDpxrdevmQiVxpKzPxqbg8jVRyqQKNqYrnciaNvGa9DdXitiBPMjUuC8Pe\\\\\\\/0L3tPYrMWHRGPpxNyXAFLVB17Rj860BXQfhgTSnJyiBhKiw2Qv4eMNFDCTY5PpLeaO+F3RtLyY0fF2pRUAM+xROyPlnRA+7hiqnlbhD0JXldneAvJrrR7CqMr\\\\\\\/7TGafBC\\\\\\\/saXWSj3Je47K2+hXhKtgaai+JFn6eq+LMzhJjI3KjxlPFTf3zW8SdLItrcZ7g8WmLVjtYNaUDX9D5mijZlGCFHgqqRkJXYRltTHW+gkcvUUFIWrXX1\\\\\\\/U6srZRVL67rixiPF1jg0X08DZ\\\\\\\/8RjUqodVZY3IfOHQhzrEwaipMY3eiiP\\\\\\\/HpRZxcrUEFqlHRAsTlfakWBqBgj\\\\\\\/zmUK+fvFNCeYu5FVKkOjVf37HDMTROwjdBMslLH8t2jszvmlSgu9zbNgIbWK9LVT6Vl7cZV0Zdi2FtafJ4H7geV2ylu7Ypz4k+OzjTunUa5y0abrsxP4eepIj+FeQR2NARP9mlOcVz6okCdtCADQcRSBzKwZN0x59tjAgl85p0p1BZrtuz52nXPspQX4iC\\\\\\\/zY41kUccvbuX7BpS2fhlikC0wzVdl6oFGbrksg0U9QzA\\\\\\\/tmyehLGrcLV\\\\\\\/tYtB4w6TxMPl+QDX1giRa+MFAbO2NWjsvUjfkJ1Yj4FqkuehBSPASfQu8bR0Q40BMlOLAI+jDHAOP5B8IORjoRN\\\\\\\/Z6V9gz3eB0+Nq2+rWp4gPeNpWMzpYEnFmUa51AypuKEs493AvmKuvJdOVi5Ahd4yNPZccrTmKnniMx\\\\\\\/hX7gGSirbNtqosXn+dPseHr47KZn3djiX7aVFZtaU+j3r8P4e9LqV\\\\\\\/nA0xqiEuxnudI3ipP9yXStIiN+wsMlTFnsj2rDPKWE3+l2mQU8qzq2DHVn8IBTMKYzlTBh2lFWJslUjLYr7m+3WAG82F+eeK5MpzcTMpUPn\\\\\\\/FH0N4AI\\\\\\\/naJi6Du+CsrWHAE3TtUwkgbXyk8RDx9mO7WGueh8sG6GcaUXOH16h13thR55dwKBdnJFETUX+J9t2IJ+HG0LXAsBX8cVyXLFQZXX8AQfPtJOa\\\\\\\/uQ\\\\\\\/f8FRCKVJkrDT1cHz27K3KqbsuGpLzQyMXL0mlPB\\\\\\\/LdtZM\\\\\\\/NdOWvVqvX1PTOVXNP+ozZkPT2IU4TKLGludsQ1VMOAGYF1i7Fmh7\\\\\\\/A3RNzoHl7XkCiYTFvQUxCyzRyjP9SnkTT0aVcKA2KClo6C3oKFGU\\\\\\\/lxosvF1AXlM28aMCYw6MBI5lMaEh+9AXEzXgTeofxXItP8a6Wof1mQrCZO5OY9BKLSLvGyyPIn5dY\\\\\\\/9Df4LYlP8CNm\\\\\\\/iIXG50kYdZw7FEJqzo84Xsl9dgJ0GcHK+oJcCxkh7PDvVr5A+SPBuh0NJuJKg2MNWejrjAnPDLFVT5wEA8ajIPKVS9CuAs1\\\\\\\/BOgv+kRdP+qhIHUyMagw6uAsdGrvXms+AzIKQxUryH+Z\\\\\\\/+1lW0Q3mMZgLThYI9Ia2UtSern93W4iSIQZuEr0JFIXQIiSufGnU+U1+1CspaAMBmB5rRXV3TrSUBIZXJN4CzIH8K+ehXY9W7zyydH9EAG3sOC39xdkrxvJiILEjOdwV4t9+1dnuzyqKy3e88fHlAVl9Mnec+b31pc6twVA67PPp2UDqrek\\\\\\\/KD1+F\\\\\\\/vpEMJOqSb8fAJhJ6ZLIAcTPNqobHOZQIDUxKpXdgYyxOLj\\\\\\\/tGV5vFOU3dc5Llm8O73l0lxR6468NSaiz0miA9Z\\\\\\\/bmApBYMyqV1CbEO+60hPNay2vhXGLSm6Zj9f1BOBBkTyuI6Em4PqRb+IZxbOYgYPtbGknw3qi55whEcZfx249\\\\\\\/boUPZmq+4WcfE\\\\\\\/A5FWeY4e3FNakkHemaWsesswQU5cs5ykTQfwriGea8RKaWSHh5p3Gbdzg5kK5hxmL\\\\\\\/47MG1tcev59co3wffOoU42xN0HJm30rVT+vQT0\\\\\\\/Fsj+w4NUrlVfnK8HeALpJrkqn+i6+HcLR\\\\\\\/BL\\\\\\\/peXVTbLZ1Qc3Rz+Ycw\\\\\\\/UwVFbUfFn5iPExDOSRQ5a6R8cYTifvGcd36yed8Cj0tol\\\\\\\/J0GD5eY4oUPY+vmTzgIC8X6Rhj78pmb42T2GzJMMgOQamDLUV30LsarH7fpTNYjlV3\\\\\\\/LJqekUFWmOXAS+27CyyIR8Io\\\\\\\/p46g0NdpeRJlk1cm1tM6dNZegoBpEKQtY1+6bu2qWn\\\\\\\/kUqKIiUdKLft6l7maEMEpeZS\\\\\\\/Hyqn2q3detcYhM11qPYpz\\\\\\\/G\\\\\\\/jm9aeguaU2rEGGw7OyVhdZR\\\\\\\/La5\\\\\\\/2eFRT8I1FFEA2FmNhgsUy\\\\\\\/8YFwOjENvQtFjd0K+7mnSrVKdN6ap1DLvGAF7A7blkQSmescZTAb3HjZFYUh7dLBYkxeOfwdeDzlTUzM8JFz5i8cl0R3\\\\\\\/C5EUiDZCxfN5XRIzbxQHJAjIMUfvqQYWraK2VooVIo0gyH9v+dcJfOCC\\\\\\\/TSfzUbU95GNDg36\\\\\\\/QeglupQJjZqZhycZBP4zhVPlfQJfPSmyIyXvhQU3fdMreNVb8A1Q3boakTRvSsiNYRUIzyj32YMNH0uw5QpqvfODSbjpwoogifP3VYbuReaMT3T6TFm9ofg2qZvK9OHlVYSh\\\\\\\/xevTAmeLrwxLQf2EEip8Leql9j5C6Vq6gEqqmQXBLALbyRXtYtH9qsG8QzXyJ3MgqI+HcQRsFb5UEcppyT5V2nTwv\\\\\\\/7RGgm8OuMBK1S5UBtdw9c+G1Ms84lE+RhwY+fjfGbuIN6qROwcd\\\\\\\/SWKT8cbirsjULNL8xXKDiXtw46A9uduzQGl2+E5gTDHpaXR+GMML409v4MrA3vCaS7UXiUhrr95TeKgSKVhSmnPDIbGwp3kKXpOYYdXmkIrfIInCtrhzulVHVGgk95NjuDn0TrxYLW2Qyy7Q+taQu6K4A0vbQZ4A0p67wLz5zLWgLDY4slUgkM6yN9Eh+RTc3DFEMA065S\\\\\\\/1bMRSyYoAneDSmZFx\\\\\\\/iTz0Q4Boa48hc\\\\\\\/8M0v9Gh+\\\\\\\/y8FoviSfp6XxQ4hOcMFKYcnUK\\\\\\\/BM9QF0dHAhpg3\\\\\\\/k2sZA\\\\\\\/z7FpT2NrZOBj\\\\\\\/I4LOFxe\\\\\\\/lvtXfNfpOz\\\\\\\/FEQL2SLaE\\\\\\\/DeIoGi2qfbzCjzX\\\\\\\/PXxwBFLVjWF\\\\\\\/L7VsFKh4v1DEMnaB+u6a75Fr5xWogDtcia+EnWJW7jAtxMw8ymYyfkIEWX8xGevomzCZDVPEih4E82hSzuslKl1AsKSPT3IAUbzQlYI2u7wnkT2gYgGAyWLeRyxk79vqZueFvtdvpGOfJhkX3ocH1SX5z3U09Aca\\\\\\\/HZrfDVSMzuzvZrPp01yRev\\\\\\\/DzJdQm+krjVUjGj4xk0lCMjwDFYYIBXzhfF6nD1Y0lj0s7rev8busvS8+b5Dmn3sVr3B4eVIHd3U0eqrTQ9tYDMYqjj7oEHyN3nPdgdeGyYKIF5619gYeyXtUBslrIOY5ecd8OeQcI+\\\\\\\/JCq74\\\\\\\/0iRP0IkAtKDzq2TceeDGxirROzPp\\\\\\\/jzG9yaFGXOT+tZZkaoNV+rUx5uun0Mv+QJ1EjslukPbV+eLkQ84iAs4vDnYGaM65ZlmqCIYE+cL9qJCa9732Rf4nkCv4YvLyyyMCUFstDIIQOamj13Vmidgryj\\\\\\\/q3yS4QPr60qrznmfzD8V2vR+bZdLJwDJS+DC6YoMck\\\\\\\/50m9lqu2\\\\\\\/TL91EI4XgAjtf71\\\\\\\/UgtG0HIbwYRx3NfkB2wN460i8eAdcPtbveUxYfOHmXZNHxQ8aTznmbGZgYPxA1j2thyBIwWn9Ay7e4ZGHjvNTAXYxnAlfLx9pdM0Mzx8uIORXDXJkJyHIUMihFU4tsBE759ba4byv0U\\\\\\\/sCy7trbJ8e6GNspU2YPsB4iWycWR0mRGnsBYIJRP4LdCVQqxm3zm4dtmfmzfIGQVne9hqANgaWj22NKJ\\\\\\\/g79BCjkI9iZ7g6ajVX1WhxuPFbFHnnfiSMlA5DOvYCoNUsQ0jvuc9QzGwhOg6qbXn\\\\\\\/+CZTpSYnpkOWEQ5N3vuDkG7R6sr0UjqfnBOTSosYu5SfrLv8eON4pJjghGwV6VHjHC74H9Iny3+cO50EXLCVtSpw2doSkxiSeQhTIpItTruDKnSHfP0NcQsIkAOd8rWMUrQPgNi2V5eT\\\\\\\/ovDka2HLa6osFvO1z2pnzUmVexM\\\\\\\/JoQukD7jVR5SXsVi9Lril4NyKXANqP0A4dzhMHBGZQxSTVge9u1I\\\\\\\/rXvgP5yxVXXF223eF6NXrEdvelAs4mBIkvvCOiGyf1vtUZTiOTVJcLx4dOHQzsrFXwQSe9iBdP1rUr2ZrRMgAQA\\\\\\\/YAEKP\\\\\\\/LDoUThkfgquIMFreSOVo5xEvNyd0qNDPaf7opRGZe7BVVO3s8\\\\\\\/2LR9XJ2DHhd7n9Mu4y3sUy2N5ykGWm\\\\\\\/Y09bxxgfvCnYqwwoqzWUqiGxoybg4UrKDYyv8BVkmpP+9euSPFIUXJPJdYVa7FA3ctVdjUoFwtTks+5B8lC\\\\\\\/CVPcsv6r+8TvMHIRYoxpCKoM0vA1fAqaZ3RKreNpNYUqAn66V9Lfj4K1BTlbdX0XpDI+jObSlgx202E2O3ZdMn2dXlwMZ\\\\\\\/101NghSKzQD\\\\\\\/5fgZgYlGm\\\\\\\/xeb33TRyx0VKeBMojrhhBxd8WA4H6RfZ+GJWQGkozcWgUOXJwp+JR2lEomfc95SNmTRnnqv6tL6h4bujCx2v8X4vfC\\\\\\\/msembzWdwewqeQMSihvzlDj6lvWJS3h1vo4f\\\\\\\/D3+tw4hvsywnLIdzoOMdtKy\\\\\\\/Js3WYO4r0vKnv6Xkh2FFQ2MZfcnb2\\\\\\\/H+PACMevm3FaGOLXTtuHhFgjjuWU04sw5MlbGsxWOmNyoNcnIc1YaVmG6b39fwpG4+XjtrRvJElxUZLVAsfWK6BKt060IhbY7tsjGvgWWnLy+tJCzAF3EEb3ffIdUTMqQ6SESeXaAvcCC78gXAqQpjueNQZtzQCHM8953uSm3JAlFRI3Q76RljjlKWGy\\\\\\\/b7q3IYeIJTx3Zn5+urowA\\\\\\\/7\\\\\\\/\\\\\\\/H4G30WW5OAaimpgXkvd8jWCqZYkX3aGAGQE6hPuSGizgPz+gPLjxJJZlSRLxzR1HsSEpctBLMEqPISJeP2LXwLFUFwGPgpxrKp3ds8ynlBv1bgbCITwn+rL6N34yixKQyzcAV8AZ2GqjQeUw9RUavcXjge0dt\\\\\\\/fnQWhBGuU3b4Ic9baIJOd+YSQFrHK87U7MgydxyU8h\\\\\\\/BnNUQSl+YWkpRlQanaeqmBbJGmddkHCdIrBMVJM1r1RWSDNOPpuFPOUKNLbDqf8AZc4eHn70Or\\\\\\\/Jo6MWdktRKeIeJs2hX+zFHVBLymLbYnlYXUh7IE9zoWzg0tReB3xusptgl\\\\\\\/\\\\\\\/8TuvzxK7+wmuJ1HOyJ9enxPDPyAnaVK+1ZqZOb6fxP8r\\\\\\\/GEhgPK\\\\\\\/dt86vbO+5TxBJeN+0NMea+OIdXPekthLxWYSVn\\\\\\\/Zc21ltlqMC1ubJ3Oac2Osn93pqYMXBoEU\\\\\\\/J7kOpFGD7U7rZcO2uTbK8WxG0bM0nydZrf17N36JCUtJp9JjptTSzXfyg7KbWzj8zmSSrq1C87mWNq1XEouiBGWTNZapdjFXr7tgbRzYcv+ce3T0QoT+jwm\\\\\\\/46qpXfC0MBCJcQ\\\\\\\/fqQ4hxC3a0ePkxV4Te77Hp1bXyevWVclWG\\\\\\\/3fe044BOI+vmmBc5V8qrtfuVjHS+HKomaH+VpusfZ2UNd4AYckyaKP2GFzns7fk7DJRfm4sLKrFb+R\\\\\\\/PLTbZ85n6qOR7trZkmVgUBRc\\\\\\\/kDCF7x70AYJKg2+byi0hla1r6Bgpl87f2CLELldDIOe4bH4LbHpgb61XeGX3p\\\\\\\/knUiq2T2t6CbLrw0Ci1RUekdlxYvvRn9W5OkBrG05anU1nAC0dYexxp9xN7HxeLh\\\\\\\/pesELEoqaZQFW1eQ3m50egbmKgm5u7soSKUoOMJK8KI7exGvCZJz26UQDX0vShsb9Tua8C5XoQZbY66X8ldyXNj2K+mJPmO+J2PKtqxe6H2MCms+RJKFxlN7pVtpOczIKtvV3YLL7cz6zWzkMmci1P25mJbbP18pc+o83XfuGOBmkg4UghAcEGeJCGAnU7MlJ\\\\\\\/UPCyECZAJEkL0e81CCNBHL2hm1qXuFJWMwPfYyVUg973RkNwZogVBJZYPEeXk1saCM39mZKeJI5GMY3uuAiWtq63pz2q09ona\\\\\\\/BcPyq0MNJdIiIYi7y1UCaqWFm7oR2+xMmpJpWqWz1GuQAdXbY5ixfGjSaKHRDYhFkwSMSV90W0l9alwKi4WLapdPitPYBwNpulFIYJHRKkLA5SRFhPG0hkuJuPodKza9CGLIJ2uRHZeUkXcudyJ59fDpgVANhGo8gd9tScJvJ\\\\\\\/OuRb2T2NaOs5o0EvEMM5iOdb37lT9+fe6Z8IHc4Maq6GRyxCbFDS9cI9rL5URNu3aJXKVN+e7gH9tokndIBptDw9roXSEthZMOfOTzR6z8HEmIqzvSKvdsrLUFV49GBMaugYtBoPGCuPepQOWtj3QvrDu8kAFLCacTQs\\\\\\\/HD93eDa7UwdFgyqnHbk4U75XbecuD3JIt3kOMWYLvtm68PP6deBLaERWlwm8QaQOlO5z2aFmp+vGXkVhfNRnZcxizOwkQIEJTYomDBLRs0wcG50vfyNFSFqVZCNExFcmfYLzcSzFbNVTN6GNucBwA5uJ5xLszFuxz8OdITP7S3CVyam67I8S0c\\\\\\\/JWsGJ3hIleKscBriS4f6nEA9bnd3OYHa2ARQhFoWYX5e5FxJJE\\\\\\\/6ovY2V2lz3jt0UbuTY9lsmrrgFMwP8LUJjTd6Mq+TineDyDwF62GT\\\\\\\/073hEvSxfg1k+KmyFG1ISCBM3LAKceyuFasqx3+swhxv52BftIGUW4BlD54uqDRVpIVof6ZJO9G1rMPF2z+DoyvZZDlqRNUpwaUSCesiQ0Fg5CWawJmqY2DEISIQzbUHrrGKzw\\\\\\\/JPqEqerWAozzrYjMIwpfn\\\\\\\/y3n884WFt+sF1\\\\\\\/pYfbVUrgaGpKTk3i4LUiBrZYHAGcXFeXxcIAe6pjAI+s1ixcljNA5UQcIWntu3cpz2SdqwX6JLZcagU\\\\\\\/BI6Br5e4P+QWG+OQr7jbbQ5YVn695sX4Qo8BcsIKox9lR0cr3zYkIAQo982wGT8flhrKHc4pVBJWJQHozSW9y4CmcNIqwLwYdGHyF1tkjvfuWHemzjjeRLDpIbgxBEyJUGLLUzInW5FLd\\\\\\\/rvfcf591QhyhFJ5Y\\\\\\\/\\\\\\\/ajkcDbdldm8CZ7+yDEdiNs1XKSboC+mCvy+Y9okZ9iOkQJZgBBtjW8oaOMfxu2Q5QK2OUAhsEiYnTo8JkleQj7+Q8cWMXUbo0gFBjF9PiRF+idwmMmduIeZGookH6LGRrTDlZvP8v854AU61VlGKZwPy4CcVuQjemHgCftOe673IXbP6p1B+s3bxFZ5\\\\\\\/e9QlW9HtNnH1\\\\\\\/NO6Ipkusftk8Prd4kQqQoSdjW9gy5Ft\\\\\\\/YH7ojyhGTag+UAaCfhr+Q+H1MGXiqPj6NUp352bNUxKcj9UWradM8i1hbbDRJUSNlKFmoifdDIdLpzxBg\\\\\\\/uhYtzurZjekb3bXsY6K29T5GfUdb0woFnH2VVc\\\\\\\/mLwz4ZLMEcoY77p+dHPMX9\\\\\\\/oTWWaAWCu\\\\\\\/JQ6p6yAxwePrNb9a3objURJAZH7JpMzAayz0Oi0ERja4\\\\\\\/mBi6RLDq7lqVzFmECYfrgUv2PbVbZ2v0QQGBE5NImCebKcu2hmFVnGjOmcwEyZlD6UJTiq8JwXa0gEZqvGG6XHMQ5LySI3ACRkL0\\\\\\\/J3fpujd\\\\\\\/PWhLmAeTTMn6YyNedGata4aHpWBONVfDf+XFZHTkV2f1zPO7f9VrMbaUyOfDKJ6nBfJfVNTv72T\\\\\\\/RPbEoeQ89zyrP39DVWVsY3kokiRwic6JHTIsfzsl2dZxmpOJEmiq2hsTO2ngMmmpk\\\\\\\/Pb6dZLpm46Zce4cRjhz2+2YlhOP93\\\\\\\/0lZSatc1kfjdc0wTBCTBxsUlPwJSLy6gGH4jfEC+7k\\\\\\\/8uhqYEefdS0ONOYKM111YjPDKkZGPHDnwA\\\\\\\/P7QEjkG6qFfDEKpvZfotZM4\\\\\\\/a1lweo9WoEPz6MAvLZeoxKBZuHssEbpSBVaQJrQaFxpBdN8L8M5n1juoPQwTI6ux7ENun2YGTUFN1K5tsc+mOpeHrE0VQunahoWRtn+5Llyt3i1pnEpUne8Msjx+KqTZPaFB4BQG+A\\\\\\\/mzrrh6Wkk7bVPLwDaSI3ejEkZZS9t\\\\\\\/043quhEZZJ75XEQg5tM0GXgogNEXeLsiFzeatmp8tMZRoy\\\\\\\/j7xqzxgBvUI4+vjCIJqFy7vdErKufjCgoJaa\\\\\\\/9eXs93AkySOr9j7yOfCq171Z14EAMTkuJBLFG4c0QlpaCxM4RcG83Zwr\\\\\\\/D+38IV+8SEi+9t9HO6oJpOfiQTjEtUQmZP12jJMUnDxhUV9IgKYMytkxqPXO1GvgiKIDQUktcggvJBz7arO0qA5gIWNABZcrmf+HcAthidm9YesdY3dqB6SbsM7Aj6IHmdqSoXfJ3mg8tbdwQZS1bmJic5M6Q5LX+1333tu9TZLbXycAAmg67Ar4JwTMtObYJqW\\\\\\\/utyYFF5IEtVtu7pgYl1Ho1MWbY8j0m6sIhjLuvQw0hzs84uXEKzEewgyaZJQ60q2VBEV8F2tOf8Cr8mxJG+wCczcCetPrj6oXqv\\\\\\\/n438hqiNB9AW7S03yaHXkDwfhLfHuko\\\\\\\/6KA0yJx57aIWWOnD\\\\\\\/Umxvl32bfwmmWk1xtp4kUj\\\\\\\/QUMzPUjBH6b4iuKxZi3aMuibnj+xGHOHoXKWV3XxwRoyhNbLTdkcjicmEESzltmCxMaO+Bot9C9AsxEOcPOGpHxpTK8lSN62wplPmlMKjMIIA50OMeWMQDLvkWHcAWbFrFqUwOw8QMzf4u9UvLiHHUa6Do1c7vovnAcihg0vr\\\\\\\/\\\\\\\/zXcDmsVk8vCJcgZGfjSqZ4PntE3STT3RVCsjK+8Ko2CYoY5Ttqv0BiM6Uc0b6FgdC8YJUv0ajB0dxDnJ8gbfB8UyOAdjD9oLPbA3py6ZhEa\\\\\\\/Dh557XDAwUy+GDdMI4gZUK\\\\\\\/7vhwDJ0JhpAuzi7+0uv3Q6C\\\\\\\/sDRsniLZ\\\\\\\/eCZVdacZgxCgTckoYPr26in62LR39HUQATBGiVzu8AuOQSeAubQK8caw3nxB8a3iN5068J\\\\\\\/+W+fgRL48SYteUlONAOjRbfoEDnaPmID2P5\\\\\\\/51ZOtSTyeoK9dNPx5OY+ldgwd6m8sn\\\\\\\/yS+dHfZb31SENhkpn7MMMHKfVJpJoGGp7bClDp17Z2IE+oQ\\\\\\\/XZSD\\\\\\\/GRzEA84O7l4eMb6dy06H8KC4Dl5S4ruefE85icG\\\\\\\/W99ybXxM2V7sANQJa3l6+5vMFMF5dq35Bo6hV\\\\\\\/BAMPYVMZiafgyTyjOzQEMtW6YnYPh42zAANg35rkic6R3lq0g8\\\\\\\/RXdtJVHj4ZXuTO3M\\\\\\\/0rZzxBxGhjyECR9lywhhEgFBpW1Jn3Bw76mv8VwLbfCm5RsQMbp5f7egKOGEzSb+oL3uIqtdAinQWOuXSAuEb+FTYPiA3xOBm+HjiHzhI5Ydxcv8uIake+yBEMbo5BVIfhRQUP5jbdUVD8iCZky5Y5\\\\\\\/OWlY2K4SuCR5iaf7tnQXmVxz2\\\\\\\/N3NnfOAV5MmBP73\\\\\\\/xJhucGnztdoTFSA70O0J9e81OiPwfk1seg5zhqt1BEEcWK+fW8Bdxf5zd3mBnlvwMFuE4azVZ2d2ZJ1coe+hsSYGgHBAdHJJ\\\\\\\/ZdhSSK27bs5HuhOatmRYPMa1NNPMxD4r93YvuaCRYr90U8LT4HSdCzHPDm4fQsZp84mgrZaDLfqZRztDk8uXmrRPfCOarcX8ayG4qeDq7xgpVHfLeOalz\\\\\\\/OUr\\\\\\\/wx8iqnIo\\\\\\\/YA2IKLk3W8C3DjHNY3gjJTRLfJHA+2Itzg+W+QCf1bKItvWKcqCCbIg\\\\\\\/b\\\\\\\/a2pKmeJl9o0SPUWhvKjP5bqL7CGh3A9IeOf8WEqWuR\\\\\\\/iizq80DrsEHLPzLx5oB7muJZKukxeizswkrix+VV75P3X0ezoQrLEWUCPHopC\\\\\\\/KknCcPm3NTHceUhtEyOq0TAPic7lNpWnHmkrPsx6PVkjFalujCSpvQbaeJkeUmmma1TFcV+hWx26Oc2u6S6Y1adsqPwsD03WMvvrPZXNNsrblJ46pY0Zk1slTKLW5x1PDy0Y\\\\\\\/UpvWr4iiXPmephOO\\\\\\\/d6aw2pSLlNBAZ+UQqtCCZPi6Vvc\\\\\\\/3WcCfdJBZP3HxKOMa6QVu+kQQ42gfN2SaYhJIBXB\\\\\\\/He\\\\\\\/dZRua9757HKzcvs3MUV1K9sfxZFw6TXoUtCdyOWR+he9xSN6tGiGCqC61TU1Syganfd0j97\\\\\\\/2KH1UA8qqm\\\\\\\/Zz+2VKeg5\\\\\\\/liXCAb8GFLZnq5JoWBlEs8CAeqIpw2zXCxEcs0MGDDQqkQshvlnJJ5acO8N+GEtECWk\\\\\\\/aMv1gmN4LPNKqRm8pelx5TCiZOvPlIGntYA8B5gnP5jIfDLWVuCBGsUdwaAyjxprYSJGp3EI+pdL7OmI0ZiYkwilwgtXAsOm9MUg8\\\\\\\/m9Q4k2seCwb5Pk+Lo5Olk063ooedDfbMdBmdNR+G+KCMM4oZs7I9ApVUjZL7Ve0tJWvRt\\\\\\\/KdoIE+KxzP4rECfAzF+gAil2\\\\\\\/0x\\\\\\\/2bbxabyFMUdDP98W4ghz2qSpN3aJgc4ZCo37JabkwzuKp2LymHxhyyH4xeMP9XHfpQndwGJYkdhft4Cf9UL67caW4CYhkdGM7oNfsyO1\\\\\\\/b\\\\\\\/ACn+E9dZhLtvuPkZSnexy29fNRrxscY6cIl6F15DTLohNbGPXYEndrK4OD\\\\\\\/A7LSpiaC8JxYVnMhxl6ACPaXKyuqPybm4ccqTf4OFPwDMUmIFO9opUINKhntFbF2ZkWy9mBiyUdTQlS5s9T\\\\\\\/iYhhtFhQQwEejlX3aRsHP2k3dZATC\\\\\\\/80eaD\\\\\\\/F46NHdKx8VcnPHInCk3G3EflnToG0jDJcLJyChSNjuwLoWGVKVVWYgGE+6zpdjlgfbkO7F2ypFgssc3RW9IaXayStzP6wEoVa21HP71D6TUfcJZYC7ICwNQaiT3zxOrWOfv169zuzqZtN3lfiBYtvsoDkztEyHG0\\\\\\\/GERMQRSfQinDoP1OgaCCpdB4D\\\\\\\/7ZzMnjUNP6wEQJXpkSZfdNbGzkiQr\\\\\\\/CeSiIDQDsJgO8Xo4tfg\\\\\\\/nfROLvcxRZHKrtVJ66Em2Ctn1yIFBKK6JndgmHepGh8lMk2sNqT9lZqcUf3F3+97jGk\\\\\\\/aqDLqcPg4jv4\\\\\\\/rYesdhLURIis4W3uHBaaQFGlkDkoIlmoPX3r9tdNfqozXKuiTkzqyr0Kdevvmmwkmynm7BPTBBw3JtCOpp8PrFUv\\\\\\\/L9duEfnu1sVgLYafxyQjZ7aLDK2L2a8aUmpgoi503bscfG9PII5zg1+9\\\\\\\/IgHFrN4TM5+s3HsZj3kYYJtyXXEQeZKvFc6J8Hfq\\\\\\\/C7rEq8OPu5ocYZOD+Sd1JplUDg1gjPs3nQFfVw7HW5mHmQ8qhcgxBhoHMEmO7YwiFRArIfZgw\\\\\\\/T0ET8EmV44Pkd0Qn7L8bkqy7c3Am6b70OqG0rK1HHu+65tdrOQuscnbRV393x7kOlUF9XwhqrdU83Hi\\\\\\\/UdUJSKVxFvba9066rmaToKfsLz5FxxuXgste3ZEI\\\\\\\/rD0Fgn+1YvVOIyfPu7R9yjPBX2ufF4RIPBifsG4OACNBaENl0yNeYeIwxyHkXLO38Qg4JHPHR\\\\\\\/7WIRNKv4Dsvg\\\\\\\/HFctlYHsfxWzhHsoDHRTgbQe9MtZHm3zhT9mq\\\\\\\/N7flIbBUzr00omEVlWhiUp3oOs+9SU+yItOLyAFk1bhmoHeQ+sdnzdOMiNrZvXOgy7p4Ybuzj7IOvaW70GGX+nB9O8n8h1HcENBqEdxrazqCuYABKG7dcdEslpcECXTCb+C9wgqDTGWH64df3+B8qg7u7fD14oOGXR186y4JyXVVndagbBJ2ThEPArgVCdR5a0Yx+U+iK1koMvFc++bS\\\\\\\/W2Zb8hJGNe\\\\\\\/K4yRV3L8I2nIGHsOdkJCY+QTWUzGiP3qWR1\\\\\\\/agvgutf9UhwD2SyN08Nic8FYOHO6wgT21\\\\\\\/oshc6X8uuB2PT80VXpjFqgVL3Y6ijLkKm23D2lEgDQsxZiS7z3oGV7nnX2g5h6ar9GPMucSfVmvo8LR0wqIC1YEhp2fRcDc1naqCt4LWQJS\\\\\\\/XrsaQfnQzHO\\\\\\\/s94ZYmjRjMZw60\\\\\\\/eG\\\\\\\/JN6Rcw0bwHjlDmJweRADVZtahRXugf\\\\\\\/RWcWdIiPbGicKJCgsPx\\\\\\\/8bqx82F529R0cRe010636H5Fj1BEue6nfIly9V70Z2pWfqxxFQ+v8KXy75izuC+LdPdrXML4xu1esLLYJL0tJddx13+53x7G91WM+TSUW7+eKExiAsk9nfDq8R3aZ1p2qR\\\\\\\/RgI\\\\\\\/+3EnEekWr\\\\\\\/c\\\\\\\/\\\\\\\/J0cBsLxfsX3H9SMFEzu1YJZfJp0ZdcYySDoI1LdTz6Bo1nqcfcGCr6GS8ezdqPQwaugVY0\\\\\\\/VI9ecife5avmgME9o+Fffgtq0U1LkR8yOtl\\\\\\\/5knTBTwPdkN6xvtFOIizFFir7LxOeTZ8fExhb9C4QzTvnZEM3\\\\\\\/DclN\\\\\\\/SGCo76Ra4vxvwqIPIi7KViyYO1Fo5NtTqYiZznS8l97kmHRIiU32ZVA4BrmxyvIfJgVf1bbIKvGjVQUoi4CPoR8lkzHiGKFpUJHihBMabsWpPqkRjnBMGn9WdGQkFUiOe8XA96y7CEkpiW8tgYlUFkk+0SPHcQMUqTAqaDIc6Pt94V2JRZS6ilKVOBSwsW4JDaZpoOVCXCf3xMka1ajk0PLeIZMHcxoRV+w\\\\\\\/ipFvLf5qLgaM6Hy1YeN3zu6Rgx54qHs2iH7M+3FNSQN2X7NzzjYvgDquZulCJStksodSimWZrasr8BaK9Km7sYquqc6\\\\\\\/GoMYxHbnywbHQbZYWD0unpSiSSk+aQbNypPWVmHu6MxRHI1b9QUDzz9LlHR7K2lpCKFkqpsgxVb3fLij0sX47KEfB1+XSfuNE2hwLzBqu+EPWeYi8l3PClxxJRUSySkmmP6ge2iNZccqpdsdBNPHAo9KNMeZfp51MZFlZ1hhDF0scAJ6AudVRL7\\\\\\\/ZRbyXhOZvtTTqNKSsiPp6Jnlc31WjNVOsvEIBiRKY22GXK8q9Pexn2beSZP4vN4UJLeT7EE6lUHm2kEon3YUv\\\\\\\/eSHzVbsQTNRCLv+pv0rYMRjI7n51M\\\\\\\/JE6HFzSiR3tnwi1CeD3mNIy6vVfbyfbs6f2oaaw0DEufN1T\\\\\\\/bq0jlsAuVADXfWVvB6QKotpyFjyh7FEc1WsPBCRslJXS6A7HJMfbLu6IU6Nvl7rN6om49OhN6PoiiYfPr1tbS+XHa+fYwFKlkrDRBXeE7HZCNS7p0cC\\\\\\\/Zgd4AOX2fedsl6K2\\\\\\\/hjcvOOAOGfou\\\\\\\/IoLyqI+Lelb9iL03ONOzv6DJwBogyczFp1OlfjbPDvYlT4Y12ttwmVwP4U8TToolsy0XLy4ilTY8MVGSun\\\\\\\/p7OPh4MIc+Bj5k5vPQd42k\\\\\\\/kwHEnELL\\\\\\\/5xP5\\\\\\\/VfcD0ervZ1QfacGsoe5EH6V8+LwcHViE\\\\\\\/hDwlqU0rD+VeZpptfDcCHIAKiD7k8t9gG4axbf1sKeTR4b\\\\\\\/4HRm2UhJ4loZakkHNAfQGrqNNylnHrDo6TrxcaCE3sxrJdfjoEDjxtLenUl9NJjDvwPf9BXfbA9pkbTsh55RtP3Z\\\\\\\/ecEq3LWSs1L\\\\\\\/6xT4bqVbFj+v4D6oQBs1dkuK\\\\\\\/VAYch3MsQNBqBoL1PnylXMOgOlSuSJy9fsdXxzOv\\\\\\\/JdrdYa415cQK69Gy+YRTm+LGHxh3QlmZJBZ4oMosIbnDK2vBn6JC4+7P6sy5PPmOP+QAMA6LmISp+Z0Umo9vWcQTSQu3XYyiduyvm8ZhQ1KT4QXfBnTpVdsmekGlnh5s7LfGD\\\\\\\/jm5o\\\\\\\/fC1Z3s3eUdHBbb2j2mMLmWAbGLi8W59FLU\\\\\\\/U4kYT2nXwS0FbFwCTQ0\\\\\\\/y+MDXkFPrKD8MbMP9wIu2tI\\\\\\\/DMiiIhimlL2k5rSxjnIvjtWRML7UewsSId8UTWqjpF1Lm0sHqjRFKxOCfbgTtQePh\\\\\\\/OtijoqK6OyTbS+q6ocmectxSUuDz+MJOqSo0mKA1WPKYIZEvxsuKh74zErbzBvi8yVJQuNoyecXR8xNL4CPCPhJwrceur3nkpXwJ3tOZoKgUP94MYDazfZXVoQLA8h2xRUtBiOPgsYP\\\\\\\/+kvRRbZRo2lG9YkWEi9UGNEi910PQE5ZMYFLin7AQ3pFJRpMzF1YGWM+X5XrH9FIKFqy+oDIj94dnhE6NFt0AFPUVJupbIceY1jbPk9Eu03\\\\\\\/0Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022g6xZDoaVwRdcCgueb7f42g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224d881a6fa40360f38da136a6e93fe9384b948c516b7e3de18c083601f984fd05\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022im4TTvpRa65DtyjfEQHNO3cNA4HC7a7Ad0OSm1AMfMSTEn9QtYLMyhghOoETqZSg2IP9rU284ptub1415oPDoAdb9V+zXFgHTa+eZsTNCamZN127XpkewXqOyAsuZXPY8KI\\\\\\\/NH5W3JASAWJFJ1yAVZgum\\\\\\\/gZdpelX2xQvgPqsZeuvRWL4ub7KP8eMaIf69x17Ie8G5AAT9R9NDnCnDpFlpPq9bmMmaK46PW6mSzs8tytx0f22mD5VwtSxeUCnlMjQVEcunw6fxXKIS9EgtTgKbqdHcxgX\\\\\\\/eqbsCcIEnrF87AZEVSgT1I9l50JZG4b5eoUij9g\\\\\\\/\\\\\\\/6sAvgmYhmSrENJ2mnfPu+Sle\\\\\\\/1tAij7qschsFvtVyKpQzKtoaH64RfaUmdE8KJp3cUws9xAeI4txEnzbKbHAVZ19a+iqH9eol7tkGzdD4a3R64IoU20lqqQ4Dvr3gVOLFNf7YV0kj3n0F60M7oLfDL663MA8Xfked5SB23b0n7ksGlwWXEfZSVzM7SVIOa\\\\\\\/xS2NkuZ9QpgzVR07mS+FOfQ0CEJ12\\\\\\\/\\\\\\\/88OP5n8kfb9V8vX\\\\\\\/WIef4VyLjPPtUEwP3FoM9yS2vUBuXQzpKXZ3JkQZgitFQZdHVZIewMuHxIDwBGHpHGcO5LOFbb6HwtnOjpqF2x43Ak40clz+UHG34OdSET6MgcmG4X1aQAE1HNInxJUQARYZN3KyWb8+kv3iwz+q0p54NFUd\\\\\\\/db2x4XXZJR6ifqMaOD2XP\\\\\\\/mySUXyEV+fH\\\\\\\/vzI8ZzfwmYc8FF0FybLsa6r6TAKxwqx93xHWqBrbn+MCWjmSLWRxashmjN9TrWvcXVnpM6msw2VWoIqibAWkY9Kl06LmTGgvmqXSM4hOu4RdyIRAsTHyiezGAs5uK0NPXhoQZwShakMy3hjAoarviRCdMn1E9cfNG1gCOFHU89MKZp8zfJQtniqumfS2DjGCp9sCMppW5LZe7uXA1BeTZUET8DseOoaDBdY6CWaTNJ+DCC6\\\\\\\/siiB98OQrGeQmQvtlnOb9h0zIdTCJ78IwF9qF\\\\\\\/PzziV6ypebecfiIlbdHehhdaQ6U2c4Ulel0Q93nHJqCSwmJLSoBQhzmn6+gjdP5HRIgTwsYfWf60wGuORWsOyQ+tmiH787XpNdDxMTt5En\\\\\\\/8To\\\\\\\/tPrtvBycyUybPAjBhpfHi6fnv8+TpufRlv\\\\\\\/uSPuvhNtJ\\\\\\\/b4UiR04hMM+C2Q4UcIhgOJHnohtgBOqTb2Tn9w7\\\\\\\/WfcXcH1KwRr6Yy5\\\\\\\/d+oXucuqK5nwHkwaM+N\\\\\\\/IB1jOqxEsS66SdT0lrs5NgOeX24tV\\\\\\\/3GGv69JQ\\\\\\\/zrgNyfa2tNHRm33OPMOzRgdcWDrmHfT3mKaso+AMRK10CngbeWz4JWNz6d4WeycLliz4w+P6omZxPO6QdbVbno+wRLcx8MECD9Qxq+VtmBIFzVzK+rn4dXChb0CfVYC0oPMYT7PSqpGTAVRFN6yZULzeUJcBojmDsH6hsiKS6bUIcZpyioMks9UbA35czTlJuSWV+L2G7lFCISBca\\\\\\\/Y8KYv4fyPa1DNvpqHu7c7kJzUVqUTu608raU149XCEftrcYBCPjuket6BRzDnbHYUUlLWO2S\\\\\\\/q\\\\\\\/Xhl1\\\\\\\/V4RD+TMhxHl7k3o5eeoJMtpWQO8P+hdkMOOZBdcxuJfhSf5rFuIxVIF2FUWItT2L73vCxvHDUAxCkO0Hc1RkZ9m7CCYDqySk21ULNufARtwIT0aLCH6jDs9eCF0Uazv29nOFmLbkBysl049ZjmyhiUU39oPW6YmGBRUE7D2KH4ng2d2D0RUnwQ7Z4X24LZa6jCJvB6J8vi14PMzNtb0igCKzT8MDSmFjAoJu6YZOJ453UB4pSYKYlQ8oHJx3Hh25n0ckxVeOHEhj2OeAwphsnDXZh1AxfmcBEvb+eGRtCEp\\\\\\\/X94FcEDhXbbSvcd63Z3GwRi6\\\\\\\/c0IzjrPUWjbR7QK2L\\\\\\\/XId2cQaeWezWN3AP6zoWd0fLWBGrUh8ekRvp5NOADTPngWGrh5hCu+tHzi9Anm9eNC1Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022clX7TJOrpE5+Rec5TkbwVg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220320ede51ca9df10b733a3cc944990babd5bffdf870558626403b5f7de5c2803\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485840 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2519" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EEIToQg-zy9kRy0VQJUNj-x7qCz1Z73zyf0Vn4wPyW8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2519" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8rTm4iinIR_68MjL5e0cvsvBO12rRnwYfnfqQVDQahA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2511" title="RNA Binding Protein Vigilin Collaborates with miRNAs To Regulate Gene Expression for Caenorhabditis elegans Larval Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2533" title="Constraints on eQTL Fine Mapping in the Presence of Multisite Local Regulation of Gene Expression" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485840" id="node485840" data-pisa="ggg;7/8/2519" data-pisa-master="ggg;g3.117.043539" data-apath="/ggg/7/8/2519.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Dissection of Morphometric Traits Reveals That Phytochrome B Affects Nucleus Size and Heterochromatin Organization in <em>Arabidopsis thaliana</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Basten L. Snoek</span>, <span class="highwire-citation-author" data-delta="1">Penka Pavlova</span>, <span class="highwire-citation-author" data-delta="2">Federico Tessadori</span>, <span class="highwire-citation-author" data-delta="3">Anton J. M. Peeters</span>, <span class="highwire-citation-author" data-delta="4">Clara Bourbousse</span>, <span class="highwire-citation-author" data-delta="5">Fredy Barneche</span>, <span class="highwire-citation-author" data-delta="6">Hans de Jong</span>, <span class="highwire-citation-author" data-delta="7">Paul F. Fransz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-2810-7374" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Martijn van Zanten</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2519-2531;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043539 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485840" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Basten L. Snoek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University, 6708 PB, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Theoretical Biology and Bioinformatics, Institute of Biodynamics and Biocomplexity</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Basten%2BL.%2BSnoek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snoek%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABasten%2BL.%2BSnoek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Penka Pavlova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Genetics, Wageningen University, 6708 PB, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant Development and (Epi)Genetics, Swammerdam Institute for Life Sciences, University of Amsterdam, 1098 SM, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Penka%2BPavlova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pavlova%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APenka%2BPavlova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Federico Tessadori </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant Development and (Epi)Genetics, Swammerdam Institute for Life Sciences, University of Amsterdam, 1098 SM, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Hubrecht Institute, Royal Netherlands Academy of Arts and Sciences, University Medical Center Utrecht, 3584 CT, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Federico%2BTessadori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tessadori%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFederico%2BTessadori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anton J. M. Peeters </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Biology, Institute of Education, Utrecht University, 3584 CH, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anton%2BJ.%2BM.%2BPeeters%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peeters%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnton%2BJ.%2BM.%2BPeeters%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clara Bourbousse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institut de Biologie de l’Ecole Normale Supérieure, Centre National de la Recherche Scientifique, Unité Mixte de Recherche 8197, Institut National de la Santé et de la Recherche Médicale U1024, Ecole Normale Supérieure, Paris Sciences et Lettres Research University, F-75005, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clara%2BBourbousse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourbousse%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClara%2BBourbousse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Fredy Barneche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institut de Biologie de l’Ecole Normale Supérieure, Centre National de la Recherche Scientifique, Unité Mixte de Recherche 8197, Institut National de la Santé et de la Recherche Médicale U1024, Ecole Normale Supérieure, Paris Sciences et Lettres Research University, F-75005, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fredy%2BBarneche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barneche%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFredy%2BBarneche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hans de Jong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Genetics, Wageningen University, 6708 PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2Bde%2BJong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Jong%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2Bde%2BJong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Paul F. Fransz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant Development and (Epi)Genetics, Swammerdam Institute for Life Sciences, University of Amsterdam, 1098 SM, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BF.%2BFransz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fransz%20PF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BF.%2BFransz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Martijn van Zanten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Molecular Plant Physiology, Institute of Environmental Biology, Utrecht University, 3584 CH, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martijn%2Bvan%2BZanten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Zanten%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartijn%2Bvan%2BZanten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2810-7374" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martijn van Zanten</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.vanzanten@uu.nl" class="" data-icon-position="" data-hide-link-title="0">m.vanzanten@uu.nl</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/8/2519" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485840" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485840/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2519.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485840" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485840/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2519.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485840" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485840/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2519.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485840" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485840/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/8/2519.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485840" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485840/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Microscopically visible chromatin is partitioned into two major components in <em>Arabidopsis thaliana</em> nuclei. On one hand, chromocenters are conspicuous foci of highly condensed “heterochromatic” domains that contain mostly repeated sequences. On the other hand, less condensed and gene-rich “euchromatin” emanates from these chromocenters. This differentiation, together with the dynamic nature of chromatin compaction in response to developmental and environmental stimuli, makes <em>Arabidopsis</em> a powerful system for studying chromatin organization and dynamics. Heterochromatin dynamics can be monitored by measuring the Heterochromatin Index, <em>i.e.</em>, the proportion of nuclei displaying well-defined chromocenters, or the DNA fraction of chromocenters (relative heterochromatin fraction). Both measures are composite traits, thus their values represent the sum of effects of various underlying morphometric properties. We exploited genetic variation between natural occurring accessions to determine the genetic basis of individual nucleus and chromocenter morphometric parameters (area, perimeter, density, roundness, and heterogeneity) that together determine chromatin compaction. Our novel reductionist genetic approach revealed quantitative trait loci (QTL) for all measured traits. Genomic colocalization among QTL was limited, which suggests a complex genetic regulation of chromatin compaction. Yet genomic intervals of QTL for nucleus size (area and perimeter) both overlap with a known QTL for heterochromatin compaction that is explained by natural polymorphism in the red/far-red light and temperature receptor Phytochrome B. Mutant analyses and genetic complementation assays show that Phytochrome B is a negative regulator of nucleus size, revealing that perception of climatic conditions by a Phytochrome-mediated hub is a major determinant for coordinating nucleus size and heterochromatin compaction.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/chromocenter" class="hw-term hw-article-keyword hw-article-keyword-chromocenter" rel="nofollow">chromocenter</a></li><li class="kwd"><a href="/keyword/phytochrome-b" class="hw-term hw-article-keyword hw-article-keyword-phytochrome-b" rel="nofollow">Phytochrome B</a></li><li class="kwd"><a href="/keyword/nucleus-size" class="hw-term hw-article-keyword hw-article-keyword-nucleus-size" rel="nofollow">nucleus size</a></li><li class="kwd"><a href="/keyword/qtl-analysis" class="hw-term hw-article-keyword hw-article-keyword-qtl-analysis" rel="nofollow">QTL analysis</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/arabidopsis-thaliana" class="hw-term hw-article-keyword hw-article-keyword-arabidopsis-thaliana" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Arabidopsis thaliana</em></a></li></ul><p id="p-3">The eukaryotic chromosome is composed of a long chromatin fiber consisting of DNA bound by histones and many different nonhistone proteins that control chromosome organization and gene activity. Several levels of chromatin folding are required to fit the DNA in the confined space of the nucleus and to allow differential access of the transcription machinery and transcription regulatory factors to the DNA (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fransz and de Jong 2011</a>). Based on classical microscopic staining methods, two types of chromatin states, euchromatin and heterochromatin, were distinguished (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Heitz 1928</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">1929</a>). DNA fluorescence <em>in situ</em> hybridization (FISH), protein immunolabeling, and epigenomic methodologies have revealed that euchromatin and heterochromatin are genetically and biochemically distinct. While euchromatin is gene-rich and heavily marked by acetylated histones and histone H3 lysine 4 methylation (H3K4me), the brightly stained heterochromatin is rich in repetitive sequences, heavily marked by H3K9 methylation and 5-methylcytosine, and is largely transcriptionally inactive (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fransz <em>et al.</em> 2006</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Fransz and de Jong 2011</a>).</p><p id="p-4">Despite improved chromatin staining methods, reliable quantification of condensation states remains difficult as in many species the boundaries between visible heterochromatin and euchromatin are blurred (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brown 1966</a>). The model plant species <em>Arabidopsis thaliana</em> is one exception, with a characteristic and discernible chromatin organization at the light microscopic level. In most <em>Arabidopsis</em> cell nuclei, including those of leaf mesophyll cells, typically 8–10 discrete and brightly stainable chromocenters can be recognized, in which most heterochromatin aggregates (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fransz <em>et al.</em> 2002</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tessadori <em>et al.</em> 2004</a>). DNA-FISH experiments demonstrated that chromocenters in <em>Arabidopsis</em> are formed around the 180-bp centromeric repeats and pericentromeric domains and contain the major repeat fraction of the genome, including silent 45S rDNA tandem repeats from the nucleolar organizing region (NOR) of chromosome 2 and 4 (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Fransz <em>et al.</em> 2002</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chandrasekhara <em>et al.</em> 2016</a>). Gene-rich euchromatic loops that are prone to transcriptional regulation emanate from these chromocenters. Chromocenters generally spatially localize near to the nuclear periphery (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Poulet <em>et al.</em> 2017</a>) and may consequently influence the whole genome topology (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Liu and Weigel 2015</a>).</p><p id="p-5">Based on 4′,6′-diamidino-2-phenylindole (DAPI) phenotypes of chromatin compaction patterns, three types of nuclei have been recognized in <em>Arabidopsis</em> (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">van Zanten <em>et al.</em> 2010</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bourbousse <em>et al.</em> 2015</a>): type 1 (condensed) nuclei contain 8–10 brightly stained, distinct round chromocenters; type 2 (intermediate) nuclei contain irregular shaped, diffuse chromocenters; and type 3 (decondensed) nuclei lack visual chromocenters. The two NOR-containing chromocenters flanking the nucleolus usually remain compacted under chromocenter-destabilizing contexts. The fraction of type 1 nuclei is used as a Heterochromatin Index (HX) to arbitrarily estimate chromatin compaction levels in a given population of nuclei (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">van Zanten <em>et al.</em> 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wang <em>et al.</em> 2013</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yelagandula <em>et al.</em> 2014</a>). A more quantitative measure of chromatin condensation levels can be obtained by computational image analysis of individual nuclei by defining the relative heterochromatin fraction (RHF). This trait represents the sum of DNA fluorescence intensity (density) of chromocenters relatively to the entire nucleus, and hence, represents the proportion of DNA found within chromocenters (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Soppe <em>et al.</em> 2002</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pavlova <em>et al.</em> 2010</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">van Zanten <em>et al.</em> 2010</a>).</p><p id="p-6">The use of the composite traits HX and/or the RHF has been the basis of several studies aimed at investigating <em>Arabidopsis</em> heterochromatin organization. For instance, analysis of RHF revealed strong aggregation of heterochromatin in the small-sized nuclei of embryonic cotyledons during seed maturation. Upon subsequent steps of seed germination, the nuclear size increases extensively. Heterochromatin is first decompacted during germination (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">van Zanten <em>et al.</em> 2011</a>) and subsequently recompacted into chromocenters when photomorphogenic seedlings establish and cotyledon cells differentiate (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Mathieu <em>et al.</em> 2003</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">van Zanten <em>et al.</em> 2011</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bourbousse <em>et al.</em> 2015</a>). Compaction progresses when leaves mature (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Tessadori <em>et al.</em> 2004</a>). Accordingly, artificial dedifferentiation of mesophyll cell nuclei during protoplast culturing leads to a severe decompaction of the chromocenters (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Tessadori <em>et al.</em> 2007a</a>).</p><p id="p-7">Fluctuations in heterochromatin compaction also occur during developmental phase transitions in the mature plant. Prior to bolting, a transient decline in the level of heterochromatin compaction mediated by the photoreceptor cryptochrome 2 is observed in the rosette leaves (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tessadori <em>et al.</em> 2007b</a>). In addition, chromatin compaction is strongly influenced by biotic (<em>e.g.</em>, pathogens; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Pavet <em>et al.</em> 2006</a>), and abiotic conditions (<em>e.g.</em>, heat; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Pecinka <em>et al.</em> 2010</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Wang <em>et al.</em> 2013</a>), and notably, by light (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barneche <em>et al.</em> 2014</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Probst and Mittelsten Scheid 2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Perrella and Kaiserli 2016</a>). Light perception by photoreceptors is essential for both nucleus growth and chromocenter formation during seedling cotyledon development, by releasing CONSTITUTIVE PHOTOMORPHOGENIC 1–dependent repression of heterochromatin condensation in darkness (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bourbousse <em>et al.</em> 2015</a>). At later developmental stages, chromocenters in leaf nuclei also severely disaggregate when plants are subjected to suboptimal light conditions, such as low light intensities, low blue light, or light with a low red/far-red ratio (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>; <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">van Zanten <em>et al.</em> 2010</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">2012</a>). Conversely, light intensity is limiting for chromocenter formation in the subtropical <em>Arabidopsis</em> accession Cvi-0, as compared to laboratory strains Col-0 and L<em>er</em> (<a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>).</p><p id="p-8">Despite extensive use of HX and RHF, full molecular understanding of chromocenter formation and (de)condensation events is hampered by the fact that these measures are the result of an interplay between several morphometric parameters, such as sizes and shapes of both nucleus and chromocenters. Here, we investigated the genetic basis of nuclear organization in <em>A. thaliana</em> in a novel reductionist manner, by using refined quantitative morphometric analyses of area, perimeter, density, heterogeneity, and roundness. We examined the genetic architecture of these five morphometric traits in recombinant inbred lines (RILs) of the L<em>er</em> × Cvi-0 population (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alonso-Blanco <em>et al.</em> 1998</a>). This revealed quantitative trait loci (QTL) for all individual morphometric traits with little overlap among genomic QTL positions, indicating that chromatin organization has a highly complex genetic basis. We conclude that multiple loci are involved in the genetic regulation of morphometry traits of nucleus and chromocenters that together contribute to chromatin (de)condensation. Confirmed morphometric QTL for nucleus area and perimeter colocated with the previously mapped negative-effect RHF QTL on chromosome 2 (RHF2) (<a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>). In this previous work, we demonstrated that RHF2 could be explained by natural genetic variation in the photo- and thermoreceptor Phytochrome B (PhyB) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Koornneef <em>et al.</em> 1980</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jung <em>et al.</em> 2016</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Legris <em>et al.</em> 2016</a>). Here we present mutant and genetic data indicating that PhyB is a negative regulator of nucleus size (area and perimeter) independent of its role in chromocenter formation and/or maintenance. Given the strong correlation between RHF on one hand and nucleus area and perimeter on the other, we propose that PhyB activity is a central component in the coregulation of nucleus size and heterochromatin condensation levels.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials and growth conditions</h3><p id="p-9">Plant materials and growth conditions were as described in (<a id="xref-ref-40-7" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>) unless stated otherwise. Seeds of Cvi-0 (N902), L<em>er</em> (NW20), <em>phyb-5</em> (N69) (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Koornneef <em>et al.</em> 1980</a>), and <em>phyb-9</em> (N6217) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Reed <em>et al.</em> 1993</a>) were obtained from Nottingham <em>Arabidopsis</em> Stock Centre. The Cvi-0 × L<em>er</em> RILs (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alonso-Blanco <em>et al.</em> 1998</a>) and near isogenic lines (NILs) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Keurentjes <em>et al.</em> 2007</a>) were kindly provided by M. Koornneef (Wageningen University, Wageningen, The Netherlands).</p><p id="p-10">For morphometry trait measurements, correlations and QTL analyses, 3–4-wk-old RILs of the L<em>er</em> × Cvi-0 core population (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Alonso-Blanco <em>et al.</em> 1998</a>), parental lines, and hybrid Cvi-0 × L<em>er</em> F1s, derived from crossing, were selected for fixation of the rosettes (see section Leaf nuclei spread preparations). Plants were grown in a greenhouse in 16 hr light/8 hr darkness on standard potting soil in a randomized design. These long day conditions induce the potential for flowering, a trait known to be segregating in the RIL set used (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Alonso-Blanco <em>et al.</em> 1998</a>), and was chosen to synchronize this trait, thereby circumventing interference of floral transition–related chromatin reorganization (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Tessadori <em>et al.</em> 2007b</a>).</p><p id="p-11">NILs for QTL confirmation, Cvi-0, L<em>er</em>, <em>phyb-5</em>, and hybrid F1, for mutant and genetic complementation analysis were grown in controlled, in-house, growth cabinets (20°, 70% v/v relative humidity during day and night) under long day conditions, with 150–200 μmol m<sup>-2</sup> s<sup>-1</sup> photosynthetic active radiation. Plants were harvested 3 wk after germination.</p><p id="p-12">(De)etiolation experiments were performed as previously described (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Bourbousse <em>et al.</em> 2015</a>). Sterilized seeds were sown on filter papers on top of MS medium supplemented with 0.9% agar. Germination was induced by exposing imbibed seeds to white light for 4 hr and subsequently shifting to darkness. After 4 d, half of the plates were put into white light (100 µmol m<sup>-2</sup> s<sup>-1</sup>) conditions 24 hr before harvesting, while the other half remained in darkness for the full 5 d.</p></div><div id="sec-3" class="subsection"><h3>Leaf nuclei spread preparations</h3><p id="p-13">Rosette leaf material was harvested and directly fixed in ice-cold Carnoy’s fixative (3:1 ethanol/acetic acid) and kept at –20° until use. Nuclei spread preparations for microscopic analysis were made essentially as described by <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Schubert <em>et al.</em> (2001</a>) and <a id="xref-ref-40-8" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> (2009</a>) using modified enzymatic cell wall–degrading mixture [cellulose Onozuka R10 (Yakult), 0.25% macerozyme R10 (Duchefa) in 10 mM citrate buffer, pH 4.5] to digest cell walls. The air-dried slides were mounted in Vectashield (Vector Laboratories) with DAPI (2 µg ml<sup>-1</sup>) before observation and capturing.</p><p id="p-14">For the (de)etiolation experiments, cotyledons of 5-d-old seedlings were fixed in 4% paraformaldehyde for 3 hr under white light condition or under a safe green light for the dark-grown seedlings, and treated with a solution containing 0.5% cellulose Onozuka R10 (Yakult), 0.25% macerozyme R10 (Duchefa), and 0.1% Triton X-100 for 1 hr. Cotyledons from at least three seedlings were isolated and squashed on a glass slide, flash frozen in liquid nitrogen, and incubated with PEMSB (50 mM Pipes, pH 7.3; 5 mM EGTA, pH 7.1; 5 mM MgSO<sub>4</sub>; 0.05% saponin; 5% wt/vol BSA) before being mounted with Vectashield (Vector Laboratories) supplemented with 2 μg/ml DAPI before observation and capturing.</p></div><div id="sec-4" class="subsection"><h3>Quantitative morphometric analysis of chromatin and nuclei</h3><p id="p-15">Between 30 and 100 nuclei per sample were captured with an Olympus BX6000 epifluorescence microscope (Olympus, Tokyo, Japan) coupled to a CCD camera (Coolsnap FX; Photometrics, Tucson, AZ). Images were captured underexposed (30–50% of the gray scale) for a high accuracy in measuring the intensity of the pixels. Subsequently, images were exported as 8 bit in TIFF format for quantitative analysis and analyzed using a (semi)automatic macro, developed in-house in Image ProPlus (Media Cybernetics, Bethesda, MD). A detailed description of this macro and macro script are available in <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Pavlova <em>et al.</em> (2010</a>). In short, outlines of the nucleus and the individual chromocenters (“finding edges”) were established by a threshold algorithm and next the following primary morphometric parameters were analyzed for both nuclei and chromocenters: area (area of the object, excluding holes in pixels), density (averaged optical density), perimeter (length of the object’s outline in pixels), roundness (relative deviation from a perfect circle), and heterogeneity (fraction of pixels that deviate &gt;10% from the averaged intensity). All data were exported to Microsoft Excel and further processed and analyzed in this software package. An overview of the primary and derived morphometric parameters used in this study is presented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486048/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486048/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Nucleus (<em>nu</em>) and chromocenter (<em>cc</em>) morphometric parameters described in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-16">For the skoto- and photomorphogenesis (dark + 24 hr light) experiments, <em>Z</em>-stack images were acquired using a confocal laser-scanning microscope (SP5; Leica) and processed using ImageJ (<a href="https://imagej.nih.gov/ij/">rsb.info.nih.gov/ij/</a>) to measure the area, chromocenter numbers, and RHF (<em>n</em> &gt; 100 nuclei for each sample).</p></div><div id="sec-5" class="subsection"><h3>QTL analysis and statistics</h3><p id="p-17">Morphometric trait values of nuclei and chromocenters (average value per nucleus) of at least 26 nuclei from at least two individual plants per RIL were used for QTL mapping. Outliers were removed from the dataset before QTL mapping was performed. This was done by excluding all values that extended beyond the 95% interval from the mean. The core marker map (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Alonso-Blanco <em>et al.</em> 1998</a>) was used for QTL analysis using the QTL-Cartographer algorithm, composite interval mapping (<a href="http://statgen.ncsu.edu/qtlcart/WQTLCart.htm">http://statgen.ncsu.edu/qtlcart/WQTLCart.htm</a>), employing the “forward and backward” search method (parameters used: 10 cM window; P<sub>in</sub>: 0.05 P<sub>out</sub>: 0.05). The threshold value was determined using QTL-Cartographer by a 1000 permutation test (95% confidence interval).</p><p id="p-18">Broad sense heritability (BSH) was calculated by the proportion of between-line trait variance divided by the total trait variance (using variation in a population of nuclei and chromocenters derived from a RIL as within variance and variation between the RILs as between variance). Variance was calculated by a generalized linear model with the SPSS statistical package by means of an ANOVA type 3.</p><p id="p-19">The statistical programming language R (<a href="https://www.r-project.org/">https://www.r-project.org/</a>) was used for statistical analyses and data visualizations (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">R Core Team 2015</a>). Pearson correlations and significance used for the correlation network were calculated in R. Cytoscape (<a href="http://www.cytoscape.org/">http://www.cytoscape.org/</a>) was used to visualize the correlation network of nucleus and chromocenters morphometric parameters (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Shannon <em>et al.</em> 2003</a>).</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-20">All (raw) data presented in this article, including nucleus and chromocenter morphometric trait values per measured nucleus and per line, all QTL profiles, additive effects, and explained variances are available in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS1.xlsx">File S1</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>L<em>er</em> and Cvi-0 mesophyll cells display contrasted nucleus and chromocenter morphometric traits</h3><p id="p-21">Under standard laboratory conditions, chromatin organization of leaf mesophyll interphase nuclei of the accession Cvi-0 greatly differs from the laboratory strain L<em>er</em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and B</a>). The average chromocenter number per nucleus was slightly different, being 8.4 in Cvi-0, 9.3 in L<em>er</em>, and 8.5 in F1 hybrids obtained from a cross between Cvi-0 and L<em>er</em>. However more strikingly, Cvi-0 chromocenters are smaller and have an irregularly shaped appearance (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, A and B</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Accordingly, Cvi-0 has a low RHF compared to L<em>er</em> (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>) (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Tessadori <em>et al.</em> 2007b</a>, <a id="xref-ref-40-9" class="xref-bibr" href="#ref-40">2009</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Typical images of DAPI-stained interphase leaf mesophyll cell nuclei of Arabidopsis thaliana. (A) The nuclei of Ler accession shows distinct and conspicuous chromocenters, while those of (B) Cvi-0 are more diffuse and irregularly shaped. Scale bar corresponds to 5 &#x3BC;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491360472" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Typical images of DAPI-stained interphase leaf mesophyll cell nuclei of &lt;em&gt;Arabidopsis thaliana&lt;/em&gt;. (A) The nuclei of L&lt;em&gt;er&lt;/em&gt; accession shows distinct and conspicuous chromocenters, while those of (B) Cvi-0 are more diffuse and irregularly shaped. Scale bar corresponds to 5 &#x3BC;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2519/F1.medium.gif" width="440" height="412"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/8/2519/F1.medium.gif" width="440" height="412"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486038" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Typical images of DAPI-stained interphase leaf mesophyll cell nuclei of <em>Arabidopsis thaliana</em>. (A) The nuclei of L<em>er</em> accession shows distinct and conspicuous chromocenters, while those of (B) Cvi-0 are more diffuse and irregularly shaped. Scale bar corresponds to 5 μm.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantification of nucleus and chromocenter morphometric parameters. (A) RHF (Tessadori et al. 2009), (B) morphometric parameters of nuclei, and (C) morphometric parameters of chromocenters (averaged per nucleus) of DAPI-stained interphase nuclei, of Ler (blue), Cvi-0 (red), and F1 hybrid (purple). Boxes indicate the boundaries of the second and third quartile of the data distribution. Black bars within the boxes indicate the median and the error bars (whiskers) indicate the values in Q1 and Q4 within 1.5 times the interquartile range. Observations outside 1.5 times the interquartile range are indicated as dots. Light colored areas behind the boxes indicate sample density. Significance levels are indicated as letters above the bars and represent a two-side t-test assuming unequal variances. Different letters indicate significant differences (P &lt; 0.01) per panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491360472" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Quantification of nucleus and chromocenter morphometric parameters. (A) RHF (Tessadori &lt;em&gt;et al.&lt;/em&gt; 2009), (B) morphometric parameters of nuclei, and (C) morphometric parameters of chromocenters (averaged per nucleus) of DAPI-stained interphase nuclei, of L&lt;em&gt;er&lt;/em&gt; (blue), Cvi-0 (red), and F1 hybrid (purple). Boxes indicate the boundaries of the second and third quartile of the data distribution. Black bars within the boxes indicate the median and the error bars (whiskers) indicate the values in Q1 and Q4 within 1.5 times the interquartile range. Observations outside 1.5 times the interquartile range are indicated as dots. Light colored areas behind the boxes indicate sample density. Significance levels are indicated as letters above the bars and represent a two-side &lt;em&gt;t&lt;/em&gt;-test assuming unequal variances. Different letters indicate significant differences (&lt;em&gt;P&lt;/em&gt; &lt; 0.01) per panel.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2519/F2.medium.gif" width="262" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/8/2519/F2.medium.gif" width="262" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485913" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Quantification of nucleus and chromocenter morphometric parameters. (A) RHF (<a id="xref-ref-40-10" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>), (B) morphometric parameters of nuclei, and (C) morphometric parameters of chromocenters (averaged per nucleus) of DAPI-stained interphase nuclei, of L<em>er</em> (blue), Cvi-0 (red), and F1 hybrid (purple). Boxes indicate the boundaries of the second and third quartile of the data distribution. Black bars within the boxes indicate the median and the error bars (whiskers) indicate the values in Q1 and Q4 within 1.5 times the interquartile range. Observations outside 1.5 times the interquartile range are indicated as dots. Light colored areas behind the boxes indicate sample density. Significance levels are indicated as letters above the bars and represent a two-side <em>t</em>-test assuming unequal variances. Different letters indicate significant differences (<em>P</em> &lt; 0.01) per panel.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">We quantified various morphometric traits of chromocenters and nuclei to assess which parameters underlying composite RHF contribute to low heterochromatin compaction in Cvi-0 leaf mesophyll nuclei. We measured five nuclear parameters (area, density, perimeter, roundness, and heterogeneity; see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Only heterogeneity, which refers to the uniformity of chromatin (DNA) compaction, differed significantly between Cvi-0 and L<em>er</em> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). Nevertheless, L<em>er</em> did differ significantly in all five measured nucleus traits from F1 hybrids (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). Cvi-0, on the other hand, only differed significantly from the F1 hybrids in nucleus perimeter and heterogeneity (fraction of pixels that deviate &gt;10% from the averaged intensity, hence low heterogeneity value represents high heterogeneity; <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). This indicates that Cvi-0 likely contains dominant-effect loci controlling nucleus area, density, and roundness, in agreement with the observed dominance of Cvi-0 in RHF (<a id="xref-ref-40-11" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>). Alternatively, several positive- and negative-effect loci controlling individual nucleus morphology traits may exist in both L<em>er</em> and Cvi-0, of which the combined effect results in the observed phenotypes. The observation that the heterogeneity value of the hybrid F1 significantly extends the value of both parental lines supports this hypothesis (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>).</p><p id="p-25">The absence of clear differences between Cvi-0 and L<em>er</em> in most nucleus morphometric parameters suggests that the low RHF of Cvi-0 is largely determined by differences in chromocenter morphometric traits. This is in line with the observation that next to a low RHF also the HX, a qualitative measure of chromatin compaction independent of nucleus morphology, is low in Cvi-0 compared L<em>er</em> (<a id="xref-ref-40-12" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>). Accordingly, Cvi-0 and L<em>er</em> differed significantly in all measured morphometric chromocenter parameters (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2C</a>). Moreover, L<em>er</em> differed significantly from all parameter values from the hybrid F1, except density. In addition, Cvi-0 and F1 chromocenter density, roundness, and heterogeneity values were significantly different from each other (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2C</a>).</p><p id="p-26">Especially large differences were observed in absolute values of chromocenter area and heterogeneity when comparing morphometric traits between Cvi-0 and L<em>er</em>. In other words, compared to L<em>er</em>, chromocenters of Cvi-0 are small (low area and perimeter), less round (roundness value deviating from one) and more heterogeneous (low heterogeneity value; <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2C</a>). These data are in line with the visual observations that L<em>er</em> chromocenters are more condensed and conspicuous than in Cvi-0 plants (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, A and B</a>) (<a id="xref-ref-40-13" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>), which further suggests that the low RHF of Cvi-0 nuclei is largely assigned to small and diffuse chromocenters of Cvi-0 compared to L<em>er</em> and not directly by differences in nucleus morphology.</p><p id="p-27">However, when we compared the distribution curves of nucleus area values of Cvi-0, L<em>er</em>, and F1 (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>), we observed that the fraction of very large nuclei in the measured population of Cvi-0 nuclei is significantly higher than in L<em>er</em> (<em>γ</em><sup>2</sup> test: comparing nuclei &lt;40,000 <em>vs.</em> &gt;40,000 pixels; <em>P</em> = 0.0184), whereas comparison of average nucleus area for the same trait and the same population of nuclei did not differ significantly (<em>t</em>-test: Cvi-0 × L<em>er</em>, <em>P</em> = 0.247; Cvi-0 × F1, <em>P</em> = 0.167; L<em>er</em> × F1, <em>P</em> = 0.0511) (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2B</a>). This shows that our mathematical approach based on averages does not fully rule out contribution of nucleus morphometric trait distributions to RHF within the population of nuclei.</p></div><div id="sec-9" class="subsection"><h3>Multiple alleles underlie natural variation in nucleus and morphometric chromocenter parameters</h3><p id="p-28">Genetic loci contributing to segregating quantitative trait variation can be detected by QTL analysis (<a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Alonso-Blanco <em>et al.</em> 1998</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2009</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Weigel 2012</a>). We measured morphometric parameters of nuclei and chromocenters (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) in a core population of 46 RILs derived from a cross between L<em>er</em> and Cvi-0 (<a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">Alonso-Blanco <em>et al.</em> 1998</a>) (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>), with the goal of detecting QTL for these traits. With the exception of roundness, the observed nucleus trait distributions reflected a Gaussian curve (normal distribution) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). Conversely, it was found that with the exception of density, all chromocenter traits deviated from a normal distribution. Transgression was observed for all traits, <em>i.e.</em>, the phenotypic values of RILs extend beyond those of the parental lines and the F1 hybrid (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and Figure S2, B and C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). This indicates that positive- and negative-effect loci segregate in the RIL population, and hence, each trait is influenced by interactions between multiple loci, resulting in the observed phenotype in a given RIL. The largest variation in nucleus traits was observed for heterogeneity (lowest value being 17.4% of that of the highest) and the smallest variation was observed for roundness (76.1%). The chromocenter traits show less transgression beyond the parental phenotypes, with heterogeneity (8.9%) having the largest variation and density (72.5%) the smallest variation.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Nucleus and chromocenter trait value distributions in RILs and parental lines. Frequency distribution of trait values (in absolute occurrences/per trait, indicated by the gray bars) of (A) nucleus and (B) chromocenters morphometric parameters obtained from mesophyll interphase nuclei of 46 RILs of the Cvi-0 &#xD7; Ler population (Alonso-Blanco et al. 1998). Trait averages of the parental lines are indicated by red (Cvi-0) and blue (Ler) arrows and hybrid F1 is indicated by purple arrows. Outliers beyond two times the SD from the mean per line, per trait, and tested plant genotype, were removed prior to classification. Note the occurrence of transgression for each morphometric trait." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491360472" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Nucleus and chromocenter trait value distributions in RILs and parental lines. Frequency distribution of trait values (in absolute occurrences/per trait, indicated by the gray bars) of (A) nucleus and (B) chromocenters morphometric parameters obtained from mesophyll interphase nuclei of 46 RILs of the Cvi-0 &#xD7; L&lt;em&gt;er&lt;/em&gt; population (Alonso-Blanco &lt;em&gt;et al.&lt;/em&gt; 1998). Trait averages of the parental lines are indicated by red (Cvi-0) and blue (L&lt;em&gt;er&lt;/em&gt;) arrows and hybrid F1 is indicated by purple arrows. Outliers beyond two times the SD from the mean per line, per trait, and tested plant genotype, were removed prior to classification. Note the occurrence of transgression for each morphometric trait.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2519/F3.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/8/2519/F3.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485984" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Nucleus and chromocenter trait value distributions in RILs and parental lines. Frequency distribution of trait values (in absolute occurrences/per trait, indicated by the gray bars) of (A) nucleus and (B) chromocenters morphometric parameters obtained from mesophyll interphase nuclei of 46 RILs of the Cvi-0 × L<em>er</em> population (<a id="xref-ref-1-8" class="xref-bibr" href="#ref-1">Alonso-Blanco <em>et al.</em> 1998</a>). Trait averages of the parental lines are indicated by red (Cvi-0) and blue (L<em>er</em>) arrows and hybrid F1 is indicated by purple arrows. Outliers beyond two times the SD from the mean per line, per trait, and tested plant genotype, were removed prior to classification. Note the occurrence of transgression for each morphometric trait.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Correlation analysis among individual traits is informative in revealing associations and/or mutual exclusion of traits. Various significant intraspecific correlations among nucleus or chromocenter morphometric values were detected (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). For instance, area and perimeter have a strong positive correlation in both the nucleus and chromocenter datasets (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). In other words, nuclei and chromocenters with a large area (volume) also tend to have a large perimeter, as expected. Nucleus area and perimeter also strongly correlates with roundness (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>), which suggests limited occurrence of invaginations of the nucleus envelope (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Walters <em>et al.</em> 2012</a>) in the measured populations of nuclei. We further observed no correlation between area/perimeter and roundness for chromocenters, which points at the dynamic shape of individual chromocenters.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trait correlation network of nucleus and chromocenter morphometric traits. Correlation network based on nucleus and chromocenters morphometric parameters of the tested RILs, parental lines Cvi-0 and Ler and hybrid F1 (Figure 2). Nucleus morphometric traits are shown as circles and chromocenter morphometric traits as diamonds. Composite RHF is shown as a white octagon. Significant correlations between traits are shown as edges (see Table 2 for an overview of Pearson R2 correlation values and Table S2 in File S2 for their significance). Positive correlations are shown in green and negative in red. Color depth and line width indicates the strength of the correlation. Correlations (Pearson; R2) are shown in black." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491360472" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Trait correlation network of nucleus and chromocenter morphometric traits. Correlation network based on nucleus and chromocenters morphometric parameters of the tested RILs, parental lines Cvi-0 and L&lt;em&gt;er&lt;/em&gt; and hybrid F1 (Figure 2). Nucleus morphometric traits are shown as circles and chromocenter morphometric traits as diamonds. Composite RHF is shown as a white octagon. Significant correlations between traits are shown as edges (see Table 2 for an overview of Pearson &lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; correlation values and Table S2 in File S2 for their significance). Positive correlations are shown in green and negative in red. Color depth and line width indicates the strength of the correlation. Correlations (Pearson; &lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;) are shown in black.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2519/F4.medium.gif" width="440" height="208"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/8/2519/F4.medium.gif" width="440" height="208"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485970" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Trait correlation network of nucleus and chromocenter morphometric traits. Correlation network based on nucleus and chromocenters morphometric parameters of the tested RILs, parental lines Cvi-0 and L<em>er</em> and hybrid F1 (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2</a>). Nucleus morphometric traits are shown as circles and chromocenter morphometric traits as diamonds. Composite RHF is shown as a white octagon. Significant correlations between traits are shown as edges (see <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a> for an overview of Pearson <em>R</em><sup>2</sup> correlation values and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a> for their significance). Positive correlations are shown in green and negative in red. Color depth and line width indicates the strength of the correlation. Correlations (Pearson; <em>R</em><sup>2</sup>) are shown in black.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486130/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486130/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Pearson correlations between averaged trait values obtained from the RILs, parental lines, and F1 hybrid</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">Only a limited set of interspecific correlations were detected, in accordance with previous observations that chromocenter and nucleus size dynamics are regulated independently in different developmental and genetic contexts (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">van Zanten <em>et al.</em> 2011</a>; <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Bourbousse <em>et al.</em> 2015</a>). Of note, however, is the strong correlation (0.87) between nucleus heterogeneity and chromocenter density (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>, and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). This is in line with the diffuse appearance of heterochromatin in the nucleoplasm <em>vs.</em> highly compacted heterochromatin in chromocenters (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Tessadori <em>et al.</em> 2004</a>, <a id="xref-ref-40-14" class="xref-bibr" href="#ref-40">2009</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Fransz and de Jong 2011</a>). The morphometric analyses further revealed that the parameters of nucleus area, density, and perimeter and chromocenter roundness are negatively correlated (<em>P</em> &lt; 0.01) with the RHF.</p></div><div id="sec-10" class="subsection"><h3>Morphometric QTL collocate with RHF QTL</h3><p id="p-34">Calculation of BSH values (<em>H</em><sup>2</sup>) indicated that between 14.1% (area) and 38.5% (heterogeneity) of the total variation observed in nucleus traits, and 10.8% (roundness) and 27.3% (heterogeneity) in chromocenter traits, can be explained by genetic variation segregating among the RILs (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). For each trait, one to five QTL (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>, and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) could be identified above the permutation-calculated likelihood-of-odds (LOD) threshold (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). Both positive- and negative-effect QTL were detected for all traits for which multiple QTL were mapped (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). This indicates that these traits are likely affected by multiple genes with contrasting allelic effects, in line with observed transgression among the RILs (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> and Figure S2B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heatmap summary of all QTL discussed in this study for nucleus and chromocenters morphometric parameters. Significance (&#x2212;10 log(p) (LOD) scores are indicated in colors. Genome-wide significance thresholds were determined by a 1000-permutation test for each trait (Table S4 in File S2). For clarity, the LOD threshold in this figure is set at the lowest value detected in our set (2.61; density CCs, Table S4 in File S2). The lower row indicates the three QTL previously mapped for RHF (RHF2, RHF4, and RHF5) (Tessadori et al. 2009). See Table 3 for a numeric description of all QTL and for QTL profiles, including additive effects and QTL names, see Figure S2 in File S2. The genomic location of PhyB (chromosome 2, 8.14 Mb, 34.5 cM) is indicated by a blue arrow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491360472" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heatmap summary of all QTL discussed in this study for nucleus and chromocenters morphometric parameters. Significance (&#x2212;10 log(&lt;em&gt;p&lt;/em&gt;) (LOD) scores are indicated in colors. Genome-wide significance thresholds were determined by a 1000-permutation test for each trait (Table S4 in File S2). For clarity, the LOD threshold in this figure is set at the lowest value detected in our set (2.61; density CCs, Table S4 in File S2). The lower row indicates the three QTL previously mapped for RHF (RHF2, RHF4, and RHF5) (Tessadori &lt;em&gt;et al.&lt;/em&gt; 2009). See Table 3 for a numeric description of all QTL and for QTL profiles, including additive effects and QTL names, see Figure S2 in File S2. The genomic location of &lt;em&gt;PhyB&lt;/em&gt; (chromosome 2, 8.14 Mb, 34.5 cM) is indicated by a blue arrow.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2519/F5.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/8/2519/F5.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485951" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Heatmap summary of all QTL discussed in this study for nucleus and chromocenters morphometric parameters. Significance (−10 log(<em>p</em>) (LOD) scores are indicated in colors. Genome-wide significance thresholds were determined by a 1000-permutation test for each trait (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). For clarity, the LOD threshold in this figure is set at the lowest value detected in our set (2.61; density CCs, Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). The lower row indicates the three QTL previously mapped for RHF (RHF2, RHF4, and RHF5) (<a id="xref-ref-40-15" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>). See <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a> for a numeric description of all QTL and for QTL profiles, including additive effects and QTL names, see Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>. The genomic location of <em>PhyB</em> (chromosome 2, 8.14 Mb, 34.5 cM) is indicated by a blue arrow.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486114/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486114/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Overview of nucleus and chromocenter morphometric trait QTL and their confirmation by NILs</span><div class="sb-div caption-clear"></div></div></div><p id="p-37">Colocation of QTL, <em>i.e.</em>, occurrence of QTL for different traits on the same genomic region, was not frequent (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>, and <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). This indicates that several allelic variants that contribute to individual morphometric traits segregate between L<em>er</em> and Cvi-0, and that individual morphometric traits have a unique molecular basis. This is in agreement with previous observations that chromocenter and nucleus size dynamics are controlled by largely independent molecular processes (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">van Zanten <em>et al.</em> 2011</a>; <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Bourbousse <em>et al.</em> 2015</a>). Interestingly, however, some major QTL of individual morphometric parameters do colocate with QTL for RHF that were reported before by <a id="xref-ref-40-17" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> (2009</a>). For instance, RHF2 (<a id="xref-ref-40-18" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>) colocates with the negative-effect QTL Area_Nuc_2 and Per_Nuc_2-1 reported here (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>, and <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). The positive-effect RHF QTL on chromosome 4 (RHF4) (<a id="xref-ref-40-19" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>) colocates with the positive-effect QTL Dens_Nuc_4-2 and Het_CC-4, and the positive-effect RHF QTL on chromosome 5 (RHF5) (<a id="xref-ref-40-20" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>) is in the vicinity of Het_CC_5-1 and Het_CC_5-2 (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>, Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>, and <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). Such colocalization could indicate that the same segregating causal allele(s) within the QTL interval is/are responsible for both the observed natural phenotypic variation in individual morphometric traits and for the composite RHF. Hence, this individual morphometric trait could directly influence the RHF value. This reasoning is supported by the observation that nucleus area, perimeter, and density variables in the RILs show a negative correlation with RHF, <em>i.e.</em> where the area, perimeter, or density of the nucleus increase, the RHF decreases (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). The same can be argued for chromocenter heterogeneity, which shows a significant positive correlation with RHF.</p></div><div id="sec-11" class="subsection"><h3>Confirmation of the detected QTL by NIL analysis</h3><p id="p-38">To confirm the QTL positions and their effects and to confine the QTL intervals, we measured nucleus and chromocenter morphometric parameters in a selection of near isogenic lines (NILs) that cover and flank the QTL positions (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a> and Figure S4 and Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). These NILs contain (small) introgressed regions of the Cvi-0 genome in an otherwise isogenic L<em>er</em> genomic background (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Keurentjes <em>et al.</em> 2007</a>) (Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). If a difference in trait values is found between a given NIL and L<em>er</em>, this difference can be attributed to a causative allele segregating between L<em>er</em> and Cvi-0 located within the introgressed region.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486083/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486083/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Confirmation of QTL by NILs</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">Based on the introgression positions (Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>) we anticipated that the set of L<em>er</em> × Cvi-0 NIL (LCN) lines used in this study together could explain nine nucleus and eight chromocenter QTL. We were able to ratify five nucleus morphometry QTL (Area_Nuc_2, Dens_Nuc_4-2, Per_Nuc_2-1, Round_Nuc_2-2, and Het_Nuc_2), while four of them could not be confirmed (Dens_Nuc_4-1, Round_Nuc_2-1, Round_Nuc_4, and Het_Nuc_5). Several QTL could not be challenged by NIL analysis, as these QTL mapped to regions that are not covered by the Cvi-0 introgressions in the used set of NILs. This was the case for nine nucleus and seven chromocenter QTL (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a> and Figure S4, Table S5, and Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>).</p><p id="p-41">Among the confirmed QTL were those that colocate with RHF2 (Area_Nuc_2 and Per_Nuc_2-1; <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>, and Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). These negative-effect QTL could be confirmed using LCN2-7 plants. In contrast, nuclei of LCN2-4, LCN2-5, and LCN2-11 plants did not differ significantly from L<em>er</em> in area and perimeter (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a> and Figure S4 and Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>), restricting the genomic interval explaining Area_Nuc_2 and Per_Nuc_2-1 between 162 and 173.2 cM (Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). Similarly, Round_Nuc_2-2 was confirmed by LCN2-7. In contrast, nucleus shapes from LCN2-4 and LCN2-5 plants were not significantly different from L<em>er</em> (Figure S4 and Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). However, like LCN2-7, LCN2-11 also confirmed the QTL. Therefore, the causal allele can be assigned to the region between 173.2 and 197.7 cM, and thus is probably different from the allele that explains Area_Nuc_2 and Per_Nuc_2-1.</p><p id="p-42">The positive-effect QTL Het_Nuc_2 was explained by LCN2-7 and LCN2-11, as in Round_Nuc_2-2. However, LCN2-4 and LCN2-5 also had a significantly lower heterogeneity (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a> and Figure S4 and Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>), whereas no heterogeneity QTL mapped to the genomic regions introgressed in LCN2-4 and LCN2-5 (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>, Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). This indicates that at least two alleles with positive effect (L<em>er</em>) on nucleus heterogeneity are present on chromosome 2. These could be the same alleles that explain Area_Nuc_2 and/or Per_Nuc_2-1 and Round_Nuc_2-2.</p><p id="p-43">The positive-effect QTL Dens_Nuc_4-1 is covered by introgressions of LCN3-17 and LCN4-2. Both introgressions have the same borders (Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>), but only LCN4-2 (and not LCN3-17) had a lower density than L<em>er</em> (Figure S4 and Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). Therefore, this QTL could only be partially confirmed. Nevertheless, the significantly lower density of LCN4-2 compared to L<em>er</em> did confirm Dens_Nuc_4-2.</p><p id="p-44">Of the chromocenter morphometric QTL (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-table-wrap-3-9" class="xref-table" href="#T3">Table 3</a>), Dens_CC_4-1 could be confirmed. Dens_CC_4-1 is a positive-effect QTL and, accordingly, both the LCN3-17 and LCN4-2 NILs indeed had a significantly lower density than L<em>er</em> (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a> and Figure S4 and Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>).</p><p id="p-45">Additional QTL that were not detected by the initial QTL mapping were detected for all morphometric parameters by directly comparing the NIL introgression lines to the wild-type L<em>er</em> background (<a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a> and Figure S4 and Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). This underlines the polygenic and highly complex genetic architecture of morphometric traits of both nucleus and chromocenters.</p></div><div id="sec-12" class="subsection"><h3>PhyB is a negative regulator of nucleus size</h3><p id="p-46">In our previous work (<a id="xref-ref-40-21" class="xref-bibr" href="#ref-40">Tessadori <em>et al.,</em> 2009</a>), we demonstrated that RHF2 can be explained by natural genetic variation in the photo/thermoreceptor <em>PhyB</em> (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Koornneef <em>et al.</em> 1980</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Jung <em>et al.</em> 2016</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Legris <em>et al.</em> 2016</a>). <em>PhyB</em> is located on position 160.5 cM on the L<em>er</em> genetic map (<a href="http://www.Arabidopsis.org">www.Arabidopsis.org</a>) and Cvi-0 and L<em>er</em> <em>PhyB</em> alleles differ at multiple nucleotides along its coding sequence (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Filiault <em>et al.</em> 2008</a>). The <em>PhyB</em> locus is covered by the QTL LOD confidence interval of Per_Nuc_2-1 (159–174 cM), but is just outside the confidence interval of Area_Nuc_2 (164–166 cM). However, only a small part of the latter QTL meets the significance threshold and the <em>PhyB</em> locus is part of the broader area QTL in this genomic region (see Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). The <em>PhyB</em> locus is covered by the introgression regions of LCN2-4, LCN2-5, and most likely, LCN2-7 (Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). Although Area_Nuc_2 and Per_Nuc_2-1 could not be confirmed in the LCN2-4 and LCN2-5 NILs, the nucleus phenotypes of LCN2-7 did confirm these QTL (<a id="xref-table-wrap-4-8" class="xref-table" href="#T4">Table 4</a> and Figure S4 and Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>).</p><p id="p-47">We directly tested whether <em>PhyB</em> affects nucleus size using mutant analyses and genetic complementation tests. In this experiment, Cvi-0 nuclei were significantly bigger (area and perimeter) than L<em>er</em> nuclei (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Interestingly, mesophyll nuclei isolated from <em>phyb-5</em> mutant plants (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Koornneef <em>et al.</em> 1980</a>) also displayed significantly higher area and perimeter values than the corresponding L<em>er</em> wild-type background, reaching statistically similar sizes as Cvi-0. These results indicate that <em>PhyB</em> is a negative regulator of nucleus size in L<em>er</em>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutant analyses and genetic complementation tests confirm the PhyB effect on nucleus area and perimeter. (A) nucleus area and (B) nucleus perimeter of parental lines Ler (blue) and Cvi-0 (red), mutant phyb-5 (yellow) and F1 hybrids derived from crosses: Cvi-0 &#xD7; Ler (purple) and Cvi-0 &#xD7; phyb-5 (orange). Boxes indicate the boundaries of the second and third quartile of the data distribution. Black horizontal bars within the boxes indicate the median and the error bars (whiskers) indicate the values in Q1 and Q4 within 1.5 times the interquartile range. Individual observations are shown as black dots. Light colored areas behind the boxes indicate sample density. Significance levels are indicated as letters above the bars and represent a two-side t-test assuming unequal variances. Different letters indicate significant differences (P &lt; 0.01) per panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491360472" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mutant analyses and genetic complementation tests confirm the PhyB effect on nucleus area and perimeter. (A) nucleus area and (B) nucleus perimeter of parental lines L&lt;em&gt;er&lt;/em&gt; (blue) and Cvi-0 (red), mutant &lt;em&gt;phyb-5&lt;/em&gt; (yellow) and F1 hybrids derived from crosses: Cvi-0 &#xD7; L&lt;em&gt;er&lt;/em&gt; (purple) and Cvi-0 &#xD7; &lt;em&gt;phyb-5&lt;/em&gt; (orange). Boxes indicate the boundaries of the second and third quartile of the data distribution. Black horizontal bars within the boxes indicate the median and the error bars (whiskers) indicate the values in Q1 and Q4 within 1.5 times the interquartile range. Individual observations are shown as black dots. Light colored areas behind the boxes indicate sample density. Significance levels are indicated as letters above the bars and represent a two-side &lt;em&gt;t&lt;/em&gt;-test assuming unequal variances. Different letters indicate significant differences (&lt;em&gt;P&lt;/em&gt; &lt; 0.01) per panel.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2519/F6.medium.gif" width="436" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/8/2519/F6.medium.gif" width="436" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486093" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-48" class="first-child">Mutant analyses and genetic complementation tests confirm the PhyB effect on nucleus area and perimeter. (A) nucleus area and (B) nucleus perimeter of parental lines L<em>er</em> (blue) and Cvi-0 (red), mutant <em>phyb-5</em> (yellow) and F1 hybrids derived from crosses: Cvi-0 × L<em>er</em> (purple) and Cvi-0 × <em>phyb-5</em> (orange). Boxes indicate the boundaries of the second and third quartile of the data distribution. Black horizontal bars within the boxes indicate the median and the error bars (whiskers) indicate the values in Q1 and Q4 within 1.5 times the interquartile range. Individual observations are shown as black dots. Light colored areas behind the boxes indicate sample density. Significance levels are indicated as letters above the bars and represent a two-side <em>t</em>-test assuming unequal variances. Different letters indicate significant differences (<em>P</em> &lt; 0.01) per panel.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">To directly test whether natural variation in <em>PhyB</em> could be responsible for the Area_Nuc_2 and Per_Nuc_2-1 QTL and possibly underlies nucleus size differences between L<em>er</em> and Cvi-0, we performed genetic complementation tests (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Mackay 2001</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Borevitz and Nordborg 2003</a>). Hybrid F1 progeny of a cross between L<em>er</em> and Cvi-0 exhibited a similar nucleus size (area and perimeter) than wild-type L<em>er</em>, <em>i.e.</em> significantly smaller than Cvi-0 (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). This indicates that L<em>er</em> carries a dominant locus that can complement the large-nucleus phenotype of Cvi-0. In contrast, nucleus size of the F1 progeny from the Cvi-0 × <em>phyb-5</em> cross was similar to Cvi-0. Accordingly, Cvi-0 × <em>phyb-5</em> F1 hybrids did differ significantly from L<em>er</em> (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). We therefore conclude that a functional L<em>er</em> <em>PhyB</em> allele is required for the small nucleus size in the L<em>er</em> × Cvi-0 cross. Taken together, these observations show that <em>PhyB</em> is a negative regulator of nucleus size and that natural variation in <em>PhyB</em> most likely underlies the Area_Nuc_2 and Per_Nuc_2-1 QTL.</p><p id="p-50">To independently confirm the negative effect of <em>PhyB</em> on nucleus size, we tested the influence of the <em>phyb-9</em> mutation in a Col-0 genetic background on the dynamic increase of both nucleus size and chromocenter condensation during cotyledon de-etiolation (<a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">Bourbousse <em>et al.</em> 2015</a>). In dark-grown seedlings, no difference was found in the number of chromocenters in cotyledon nuclei of <em>phyb-9</em> and wild-type Col-0. Exposing the etiolated seedling to white light for 24-hr induced chromocenter formation in the wild type as expected (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a> and Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>), but the increase in the number of chromocenters per nucleus was significantly lower in the <em>phyb-9</em> mutant (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>). This indicates that PhyB is a positive regulator of chromocenter condensation dynamics under white light conditions. This observation is in accordance with previous data showing that the HX of both <em>phyb-9</em> (in Col-0) and <em>phyb-5</em> (in L<em>er</em>) is lower than their corresponding wild types and is consistent with the observation that <em>phyb-9</em> plants display a reduced RHF compared to wild type when grown under standard light conditions (<a id="xref-ref-40-22" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>) (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a>). Interestingly, nuclei of dark-grown <em>phyb-9</em> cotyledons were significantly larger than those of Col-0 (area; <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7C</a>). These data confirm that <em>PhyB</em> is a negative regulator of nucleus size (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a> and <a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7C</a>) independently of its effect on chromatin compaction, as no differences in chromocenters are visible at this stage between the wild-type and <em>phyb-9</em> mutant nuclei. Exposure to 24-hr light, however, led to a similar increase in nucleus size in both Col-0 and <em>phyb-9</em> (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7C</a>). This suggests that <em>PhyB</em> effects on nucleus size in dark-grown seedlings is overruled by the 24-hr light exposure (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7C</a>) or that expression of the <em>phyb</em> mutant effect on nucleus size requires &gt;24 hr of light exposure.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PhyB control of nucleus size and chromatin compaction dynamics during de-etiolation. (A) Number of chromocenters per nucleus, (B) RHF, and (C) nucleus area of Col-0 wild type (green) and phyb-9 mutants (red) in cotyledon nuclei of seedlings grown in darkness (Dark) or shifted to light for 24 hr (Light). Boxes indicate the boundaries of the second and third quartile of the data distributions. Black horizontal bars within the boxes indicate the median and the error bars (whiskers) indicate the values in Q1 and Q4 within 1.5 times the interquartile range. Individual observations are shown as dots. Light colored areas behind the boxes indicate sample density. Significance levels are indicated as letters above the bars and represent a two-side t-test assuming unequal variances. Different letters indicate significant differences (P &lt; 0.05) per panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491360472" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;PhyB control of nucleus size and chromatin compaction dynamics during de-etiolation. (A) Number of chromocenters per nucleus, (B) RHF, and (C) nucleus area of Col-0 wild type (green) and &lt;em&gt;phyb-9&lt;/em&gt; mutants (red) in cotyledon nuclei of seedlings grown in darkness (Dark) or shifted to light for 24 hr (Light). Boxes indicate the boundaries of the second and third quartile of the data distributions. Black horizontal bars within the boxes indicate the median and the error bars (whiskers) indicate the values in Q1 and Q4 within 1.5 times the interquartile range. Individual observations are shown as dots. Light colored areas behind the boxes indicate sample density. Significance levels are indicated as letters above the bars and represent a two-side &lt;em&gt;t&lt;/em&gt;-test assuming unequal variances. Different letters indicate significant differences (&lt;em&gt;P&lt;/em&gt; &lt; 0.05) per panel.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2519/F7.medium.gif" width="440" height="200"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/8/2519/F7.medium.gif" width="440" height="200"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2519/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486053" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-51" class="first-child">PhyB control of nucleus size and chromatin compaction dynamics during de-etiolation. (A) Number of chromocenters per nucleus, (B) RHF, and (C) nucleus area of Col-0 wild type (green) and <em>phyb-9</em> mutants (red) in cotyledon nuclei of seedlings grown in darkness (Dark) or shifted to light for 24 hr (Light). Boxes indicate the boundaries of the second and third quartile of the data distributions. Black horizontal bars within the boxes indicate the median and the error bars (whiskers) indicate the values in Q1 and Q4 within 1.5 times the interquartile range. Individual observations are shown as dots. Light colored areas behind the boxes indicate sample density. Significance levels are indicated as letters above the bars and represent a two-side <em>t</em>-test assuming unequal variances. Different letters indicate significant differences (<em>P</em> &lt; 0.05) per panel.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>Dissection of composite RHF and HX in distinct morphometric parameters provides comprehensive information on chromatin dynamics</h3><p id="p-52">The contribution of chromatin organization to nuclear structure is a topic of increasing interest (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Liu and Weigel 2015</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Poulet <em>et al.</em> 2017</a>). The clear separation between microscopic appearance of euchromatin and heterochromatin makes <em>Arabidopsis</em> a system particularly well suited to study chromatin organization and dynamics (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">van Zanten <em>et al.</em> 2012</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">2013</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Probst and Mittelsten Scheid 2015</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Perrella and Kaiserli 2016</a>).</p><p id="p-53">Quantification of HX and/or the RHF has been extensively used in studies on dynamics in chromatin compaction in relation to development (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Mathieu <em>et al.</em> 2003</a>; <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Tessadori <em>et al.</em> 2004</a>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">2007a</a>,<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">b</a>; <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">van Zanten <em>et al.</em> 2011</a>; <a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">Bourbousse <em>et al.</em> 2015</a>) and response to biotic (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Pavet <em>et al.</em> 2006</a>) and abiotic stimuli (<a id="xref-ref-40-23" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Pecinka <em>et al.</em> 2010</a>; <a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">van Zanten <em>et al.</em> 2010</a>, <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">2012</a>, <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">2013</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Wang <em>et al.</em> 2013</a>; <a id="xref-ref-5-9" class="xref-bibr" href="#ref-5">Bourbousse <em>et al.</em> 2015</a>). Despite the extensive use of RHF and HX, the exact molecular determinants and the functional significance of heterochromatin condensation changes in <em>A. thaliana</em> remain largely unclear. A reason for this is that the RHF and HX values provide only partial information about the mechanisms underlying chromocenter condensation, because various (and possibly unrelated) primary parameters, such as chromocenter number, nucleus size, and chromocenter size, contribute to the composite RHF. Complementary analyses that uncover hidden information provided by other morphometric parameters were required to better understand chromatin compaction controls. For example, an increase in RHF could either be the result of chromocenter enlargement or densification with maintenance of nucleus size or reciprocally result from a decrease in nucleus size or density with maintenance of chromocenter size. Hence, similar changes in composite RHF could be caused by distinct molecular mechanisms.</p><p id="p-54">Gaining full understanding of the biological function of chromatin (de)condensation and the contributing molecular factors, requires an experimental genetic system in which morphometric parameters underlying chromatin organization can be independently quantified. In this work, we exploited the natural existence of genetic variation in nucleus and chromocenter morphometric parameters to provide such new fundamental insights in the molecular genetic basis of chromocenter formation and/or maintenance. Our morphometric analyses unveiled that nuclear area, density, and perimeter, as well as chromocenter roundness and heterogeneity correlate with the RHF. This is not the case for chromocenter density and the correlation between RHF and chromocenter area is low. This suggests that condensation of heterochromatin regions and overall chromocenter organization might not be directly linked.</p><p id="p-55">Of note is the apparent distinction of phenotypes in nuclear heterogeneity, which seems to fall apart into three distinguishable classes (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). This illustrates how individual morphometric traits could be informative to understand the dynamics of chromocenter compaction. Although the distribution of phenotypes of this trait still has a Gaussian distribution (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>), we speculate that the distinguishable classes may reflect the three chromatin compaction types observed in a population of mesophyll nuclei; from type 1 containing conspicuous chromocenters to type 3 containing relaxed heterochromatin (<a id="xref-ref-40-24" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>).</p></div><div id="sec-15" class="subsection"><h3>Detecting QTL for nucleus and chromocenter morphometry</h3><p id="p-56">In general, the heritability of morphometric traits of individual chromocenters was much lower than when averaged per nucleus (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>). This is indicative for a general control mechanism that simultaneously determines the chromocenter state in a particular nucleus. This observation, however, does not rule out that morphometry of particular chromocenters is individually regulated.</p><p id="p-57">For each individually assessed morphometric trait one or more QTL could be mapped (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>), highlighting the highly complex genetic basis of chromatin organization of <em>Arabidopsis</em> nuclei. This indicates that several molecular regulators contribute to heterochromatin condensation levels. Several QTL covered by the introgression borders of the NILs were not confirmed. Moreover, additional QTL were identified by NILs that were not detected in QTL analysis. The existence of more QTL in NILs than in RILs has been repeatedly reported (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Keurentjes <em>et al.</em> 2007</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Green <em>et al.</em> 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Snoek <em>et al.</em> 2013</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Stastna <em>et al.</em> 2015</a>). This can be caused by closely linked QTL, which are hard to detect in the RILs, or by complex interactions between QTL, for which the model “to-test” is difficult to predict. It should also be noted that our QTL detection is based on a moderately small core set of 46 RILs. A consequence of a small population size is that statistical power of QTL detection is relatively limited, <em>i.e.</em>, only strong QTL are detected. It is therefore possible that these population sizes preclude the detection of small-effect QTL, which appear in the NILs. Regardless of the reason, the presence of such extra QTL further underlines that the genetic architecture of both nucleus and chromocenter morphometric traits is polygenic and involves highly complex genetic interactions. The underlying molecular components and effects on gene expression could be further studied using colocating expression QTL (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Nijveen <em>et al.</em> 2016</a>). Taken together, our study unveils the existence of multiple loci regulating specific nucleus and chromocenter morphometric traits that together contribute to heterochromatin organization. Therefore, this work provides an experimental toolbox to identify novel genetic factors and molecular mechanisms underlying changes in chromatin compaction induced during development or in response to the environment.</p></div><div id="sec-16" class="subsection"><h3>PhyB is a negative regulator of nucleus size</h3><p id="p-58">Much to our surprise, only limited genomic overlap (colocation) was detected among the identified QTL intervals. Though, the morphometry QTL Dens_Nuc_4-2 and Het_CC_4 colocate with the previously identified RHF4 (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>, Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>, and <a id="xref-table-wrap-3-10" class="xref-table" href="#T3">Table 3</a>). This RHF4 could be explained by natural variation in HISTONE DEACETYLASE 6 (HDA6) (<a id="xref-ref-40-25" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>). This suggests that also nucleus density and heterogeneity may be under the control of HDA6.</p><p id="p-59">Another notable exception was the overlap between QTL for nucleus area and perimeter (Area_Nuc_2 and Per_Nuc_2-1; <a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5</a>, Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1/FileS2.docx">File S2</a>) with RHF2 (<a id="xref-ref-40-26" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>), which results from natural variation in the PhyB nucleotide sequence (<a id="xref-ref-40-27" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>). Using mutant analyses and genetic complementation tests, we were able to show that, in addition to its influence on heterochromatin organization, PhyB is also a negative regulator of nucleus size of mesophyll leaf cells (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>) and in young Col-0 seedlings (<a id="xref-fig-7-8" class="xref-fig" href="#F7">Figure 7</a>).</p><p id="p-60">A likely cause of the limited overlap in QTL positions is that dynamic regulation of individual morphometric phenotypes not necessarily cooccur. This is in accordance with our previous work showing that changes in chromatin compaction and changes in nucleus size are developmentally and genetically uncoupled (<a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">van Zanten <em>et al.</em> 2011</a>; <a id="xref-ref-5-10" class="xref-bibr" href="#ref-5">Bourbousse <em>et al.</em> 2015</a>). Yet they may share common signaling components or molecular determinants to cooperatively regulate nucleus reorganization events. Indeed, our genetic study of natural variation unveiled a clear correlation between nucleus size (area and perimeter) and RHF values (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>). Hence, distinct processes such as endoreduplication and activity of chromatin remodeling proteins, at different developmental stages or in response to environmental cues, might converge to allow coordination of nucleus size and heterochromatin organization. From our observations presented here and previous work, we propose that PhyB-mediated control of nucleus size is at least in part responsible for the natural variation in chromatin condensation levels among <em>Arabidopsis</em> accessions (<a id="xref-ref-40-28" class="xref-bibr" href="#ref-40">Tessadori <em>et al.</em> 2009</a>).</p><p id="p-61">We found that PhyB has a negative effect on nucleus size of rosette leaves grown in growth cabinets (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6</a>) or in a greenhouse (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). This is also the case in cotyledon nuclei of 5-d-old etiolated seedlings, but not upon exposing them to light for 24 hr (<a id="xref-fig-7-9" class="xref-fig" href="#F7">Figure 7C</a>). This might directly relate to <em>phyB</em> mutant plants having defective capacity for rapid or efficient heterochromatin compaction. Light-mediated changes in chromatin compaction occur notoriously slow, however. For instance, maximum chromocenter decondensation is only reached 4 d after shifting plants to low light intensity treatment (<a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">van Zanten <em>et al.</em> 2010</a>) and, vice versa, chromocenter formation is not completed upon 24 hr of de-etiolation (<a id="xref-ref-5-11" class="xref-bibr" href="#ref-5">Bourbousse <em>et al.</em> 2015</a>). The lack of a <em>phyB</em>-dependent effect on nucleus size after a relatively short exposure to light therefore could result from the early time point (24 hr), at which the phenotypic effects linked to PhyB photoperception are not yet saturated. Accordingly, chromocenter number in <em>phyb-9</em> mutants after 24 hr of light lies in-between those of dark- and light-grown Col-0 seedlings (<a id="xref-fig-7-10" class="xref-fig" href="#F7">Figure 7A</a>). Future studies should address the precise light-dependent and temporal components of PhyB-mediated nucleus size control and dynamic chromatin compaction.</p><p id="p-62">Interestingly, high temperature also induces severe chromocenter decondensation (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Pecinka <em>et al.</em> 2010</a>; <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Wang <em>et al.</em> 2013</a>), a second chromatin decompaction process in which Phytochromes, acting both as a photo- and thermoreceptor, could have a role. Obtaining dynamic data under different environments at various developmental stages could be used to develop integrated computational models parameterized by experimentally determined values of individual nucleus and chromocenter morphometric traits and their correlations. Such modeling would be helpful to further elucidate the molecular mechanisms behind chromatin compaction dynamics.</p></div></div><div class="section" id="sec-17"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-63">The authors thank Maarten Koornneef and Joost Keurentjes (Wageningen University, The Netherlands) for kindly sharing the Cvi-0 × L<em>er</em> RIL population and NILs. Work by F.B. was supported by grant ANR-11-JSV2-003-01, the Investissements d’Avenir Labex Memory in Living Systems (MEMOLIFE) grant ANR-10-LABX-54, as well as a grant from the Paris Sciences et Lettres Research University (to F.B. and C.B.). P.F.F. was supported by the Netherlands Organization for Scientific Research (NWO) genomics program, grant 050-10-020, and M.v.Z. was supported by NWO VENI grant 863.11.008. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-64">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043539/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-65"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-03-14"><span class="received-label">Received </span>March 14, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-05-31"><span class="accepted-label">Accepted </span>May 31, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Snoek <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.1" data-doi="10.1046/j.1365-313X.1998.00115.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Blanco</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peeters</span>  <span class="cit-name-given-names">A. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koornneef</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Development of an AFLP based linkage map of L<em>er</em>, Col and Cvi <em>Arabidopsis</em> <em>thaliana</em> ecotypes and construction of a L<em>er</em>/Cvi recombinant inbred line population.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DAlonso-Blanco%26rft.auinit1%253DC.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D259%26rft.epage%253D271%26rft.atitle%253DDevelopment%2Bof%2Ban%2BAFLP%2Bbased%2Blinkage%2Bmap%2Bof%2BLer%252C%2BCol%2Band%2BCvi%2BArabidopsis%2Bthaliana%2Becotypes%2Band%2Bconstruction%2Bof%2Ba%2BLer%252FCvi%2Brecombinant%2Binbred%2Bline%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313X.1998.00115.x%26rft_id%253Dinfo%253Apmid%252F9628021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313X.1998.00115.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9628021&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073794300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.2" data-doi="10.1105/tpc.109.068114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Blanco</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aarts</span>  <span class="cit-name-given-names">M. G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentsink</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keurentjes</span>  <span class="cit-name-given-names">J. J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reymond</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">What has natural variation taught us about plant development, physiology, and adaptation?</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1877</span>–<span class="cit-lpage">1896</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.068114%26rft_id%253Dinfo%253Apmid%252F19574434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIxLzcvMTg3NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.3" data-doi="10.1093/jxb/eru011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barneche</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malapeira</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mas</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The impact of chromatin dynamics on plant light responses and circadian clock function.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">2895</span>–<span class="cit-lpage">2913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Feru011%26rft_id%253Dinfo%253Apmid%252F24520020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/eru011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24520020&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.4" data-doi="10.1104/pp.103.023549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The impact of genomics on the study of natural variation in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">718</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DBorevitz%26rft.auinit1%253DJ.%2BO.%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D718%26rft.epage%253D725%26rft.atitle%253DThe%2BImpact%2Bof%2BGenomics%2Bon%2Bthe%2BStudy%2Bof%2BNatural%2BVariation%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.103.023549%26rft_id%253Dinfo%253Apmid%252F12805600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzIvMi83MTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.5" data-doi="10.1073/pnas.1503512112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourbousse</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mestiri</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zabulon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourge</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Formiggini</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Light signaling controls nuclear architecture reorganization during seedling establishment.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">E2836</span>–<span class="cit-lpage">E2844</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1503512112%26rft_id%253Dinfo%253Apmid%252F25964332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzIxL0UyODM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.6" data-doi="10.1126/science.151.3709.417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Heterochromatin.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBrown%26rft.auinit1%253DS.%2BW.%26rft.volume%253D151%26rft.issue%253D3709%26rft.spage%253D417%26rft.epage%253D425%26rft.atitle%253DHeterochromatin%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.151.3709.417%26rft_id%253Dinfo%253Apmid%252F5322971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTI6IjE1MS8zNzA5LzQxNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.7" data-doi="10.1101/gad.273755.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandrasekhara</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohannath</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blevins</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pontvianne</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pikaard</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Chromosome-specific NOR inactivation explains selective rRNA gene silencing and dosage control in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.273755.115%26rft_id%253Dinfo%253Apmid%252F26744421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMzAvMi8xNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.8" data-doi="10.1073/pnas.0712174105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Filiault</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wessinger</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinneny</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lutes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Amino acid polymorphisms in <em>Arabidopsis</em> phytochrome B cause differential responses to light.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">3157</span>–<span class="cit-lpage">3162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0712174105%26rft_id%253Dinfo%253Apmid%252F18287016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA1LzgvMzE1NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.9" data-doi="10.1111/j.1365-313X.2011.04526.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">From nucleosome to chromosome: a dynamic organization of genetic information.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFransz%26rft.auinit1%253DP.%26rft.volume%253D66%26rft.issue%253D1%26rft.spage%253D4%26rft.epage%253D17%26rft.atitle%253DFrom%2Bnucleosome%2Bto%2Bchromosome%253A%2Ba%2Bdynamic%2Borganization%2Bof%2Bgenetic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2011.04526.x%26rft_id%253Dinfo%253Apmid%252F21443619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2011.04526.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21443619&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288862500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.10" data-doi="10.1073/pnas.212325299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lysak</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castiglione</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Interphase chromosomes in <em>Arabidopsis</em> are organized as well defined chromocenters from which euchromatin loops emanate.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">14584</span>–<span class="cit-lpage">14589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.212325299%26rft_id%253Dinfo%253Apmid%252F12384572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjIvMTQ1ODQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.11" data-doi="10.1007/s10577-005-1022-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">ten Hoopen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tessadori</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Composition and formation of heterochromatin in <em>Arabidopsis</em> <em>thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DFransz%26rft.auinit1%253DP.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D82%26rft.atitle%253DComposition%2Band%2Bformation%2Bof%2Bheterochromatin%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-005-1022-5%26rft_id%253Dinfo%253Apmid%252F16506097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-005-1022-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16506097&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235703700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.12" data-doi="10.1038/hdy.2013.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">J. W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harvey</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic mapping of variation in dauer larvae development in growing populations of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D111%26rft.spage%253D306%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.50%26rft_id%253Dinfo%253Apmid%252F23715016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.50&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23715016&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324580600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heitz</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1928</span> <span class="cit-article-title">Das heterochromatin der moose.</span> <abbr class="cit-jnl-abbrev">Jahrb Wiss Bot</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJahrb%2BWiss%2BBot%26rft.volume%253D69%26rft.spage%253D762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heitz</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1929</span> <span class="cit-article-title">Heterochromatin, chromocentren, chromomeren.</span> <abbr class="cit-jnl-abbrev">Ber Bot. Ges</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">274</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBer%2BBot.%2BGes%26rft.volume%253D47%26rft.spage%253D274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.15" data-doi="10.1126/science.aaf6005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">J.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domijan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klose</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biswas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ezer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Phytochromes function as thermosensors in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">354</span>: <span class="cit-fpage">886</span>–<span class="cit-lpage">889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf6005%26rft_id%253Dinfo%253Apmid%252F27789797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTQvNjMxNC84ODYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.16" data-doi="10.1534/genetics.106.066423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keurentjes</span>  <span class="cit-name-given-names">J. J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentsink</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Blanco</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanhart</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blankestijn-De Vries</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Development of a near-isogenic line population of <em>Arabidopsis</em> <em>thaliana</em> and comparison of mapping power with a recombinant inbred line population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.066423%26rft_id%253Dinfo%253Apmid%252F17179089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzIvODkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.17" data-doi="10.1016/S0044-328X(80)80208-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koornneef</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rolff</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spruit</span>  <span class="cit-name-given-names">C. J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Genetic control of light-inhibited hypocotyl elongation in <em>Arabidopsis</em> <em>thaliana</em> (L.) Heynh.</span> <abbr class="cit-jnl-abbrev">Z. Pflanzenphysiol.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZ.%2BPflanzenphysiol.%26rft.volume%253D100%26rft.spage%253D147%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0044-328X%252880%252980208-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0044-328X(80)80208-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legris</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klose</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgie</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costigliolo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neme</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Phytochrome B integrates light and temperature signals in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">897</span>–<span class="cit-lpage">900</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D18%26rft.spage%253D897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.19" data-doi="10.1186/s13059-015-0738-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Chromatin in 3D: progress and prospects for plants.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D170%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0738-6%26rft_id%253Dinfo%253Apmid%252F26294115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0738-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26294115&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.20" data-doi="10.1146/annurev.genet.35.102401.090633"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The genetic architecture of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%2BC.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D303%26rft.epage%253D339%26rft.atitle%253DTHE%2BGENETIC%2BARCHITECTURE%2BOF%2BQUANTITATIVE%2BTRAITS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.35.102401.090633%26rft_id%253Dinfo%253Apmid%252F11700286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.35.102401.090633&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11700286&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172991500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.21" data-doi="10.1105/tpc.017467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathieu</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jasencakova</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaillant</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gendrel</span>  <span class="cit-name-given-names">A.-V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colot</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Changes in 5S rDNA chromatin organization and transcription during heterochromatin establishment in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">2929</span>–<span class="cit-lpage">2939</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.017467%26rft_id%253Dinfo%253Apmid%252F14630972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMi8yOTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nijveen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ligterink</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keurentjes</span>  <span class="cit-name-given-names">J. J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loudet</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">AraQTL - workbench and archive for systems genetics in <em>Arabidopsis</em> <em>thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">1225</span>–<span class="cit-lpage">1235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D89%26rft.spage%253D1225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.23" data-doi="10.1094/MPMI-19-0577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavet</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quintero</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cecchini</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title"><em>Arabidopsis</em> displays centromeric DNAhypomethylation and cytological alterations of heterochromatin upon attack by <em>Pseudomonas syringae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">577</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DPavet%26rft.auinit1%253DV.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D577%26rft.epage%253D587%26rft.atitle%253DArabidopsis%2Bdisplays%2Bcentromeric%2BDNA%2Bhypomethylation%2Band%2Bcytological%2Balterations%2Bof%2Bheterochromatin%2Bupon%2Battack%2Bby%2Bpseudomonas%2Bsyringae.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-19-0577%26rft_id%253Dinfo%253Apmid%252F16776291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-19-0577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16776291&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237757200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.24" data-doi="10.1007/978-1-60761-765-5_28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlova</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tessadori</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Immunocytological analysis of chromatin in isolated nuclei.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">655</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DPavlova%26rft.auinit1%253DP.%26rft.volume%253D655%26rft.spage%253D413%26rft.epage%253D432%26rft.atitle%253DImmunocytological%2Banalysis%2Bof%2Bchromatin%2Bin%2Bisolated%2Bnuclei.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60761-765-5_28%26rft_id%253Dinfo%253Apmid%252F20734277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60761-765-5_28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20734277&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.25" data-doi="10.1105/tpc.110.078493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pecinka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinh</span>  <span class="cit-name-given-names">H. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baubec</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lettner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Epigenetic regulation of repetitive elements is attenuated by prolonged heat stress in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3118</span>–<span class="cit-lpage">3129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.078493%26rft_id%253Dinfo%253Apmid%252F20876829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzkvMzExOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perrella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaiserli</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Light behind the curtain: photoregulation of nuclear architecture and chromatin dynamics in plants.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">908</span>–<span class="cit-lpage">919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D212%26rft.spage%253D908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.27" data-doi="10.1242/jcs.194712"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poulet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duc</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voisin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desset</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tutois</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The LINC complex contributes to heterochromatin organisation and transcriptional gene silencing in plants.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">590</span>–<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.194712%26rft_id%253Dinfo%253Apmid%252F28049722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6OToiMTMwLzMvNTkwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Probst</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mittelsten Scheid</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Stress-induced structural changes in plant chromatin.</span> <abbr class="cit-jnl-abbrev">Cell Signal. Gene Regul.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">8</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSignal.%2BGene%2BRegul.%26rft.volume%253D27%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2519.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <em>R: A Language and Environment for Statistical Computing</em>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.30" data-doi="10.1105/tpc.5.2.147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagpal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furuya</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chory</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Mutations in the gene for the red/far-red light receptor phytochrome B alter cell elongation and physiological responses throughout <em>Arabidopsis</em> development.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DReed%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D147%26rft.epage%253D157%26rft.atitle%253DMutations%2Bin%2Bthe%2BGene%2Bfor%2Bthe%2BRed%252FFar-Red%2BLight%2BReceptor%2BPhytochrome%2BB%2BAlter%2BCell%2BElongation%2Band%2BPhysiological%2BResponses%2Bthroughout%2BArabidopsis%2BDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.5.2.147%26rft_id%253Dinfo%253Apmid%252F8453299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjUvMi8xNDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.31" data-doi="10.1023/A:1013137415093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromosome painting in plants.</span> <abbr class="cit-jnl-abbrev">Methods Cell Sci.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bcell%2Bscience%2B%253A%2B%2Ban%2Bofficial%2Bjournal%2Bof%2Bthe%2BSociety%2Bfor%2BIn%2BVitro%2BBiology%26rft.stitle%253DMethods%2BCell%2BSci%26rft.aulast%253DSchubert%26rft.auinit1%253DI.%26rft.volume%253D23%26rft.issue%253D1-3%26rft.spage%253D57%26rft.epage%253D69%26rft.atitle%253DChromosome%2Bpainting%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1013137415093%26rft_id%253Dinfo%253Apmid%252F11741144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1013137415093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11741144&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.32" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markiel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baliga</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terpstra</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekter</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van den Ackerveken</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peeters</span>  <span class="cit-name-given-names">A. J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetical genomics reveals large scale genotype-by-environment interactions in <em>Arabidopsis</em> <em>thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D3%26rft.spage%253D317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orbidans</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stastna</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aartse</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Widespread genomic incompatibilities in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1813</span>–<span class="cit-lpage">1823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D4%26rft.spage%253D1813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.35" data-doi="10.1093/emboj/cdf657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soppe</span>  <span class="cit-name-given-names">W. J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jasencakova</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houben</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kakutani</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meister</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">DNA methylation controls histone H3 lysine 9 methylation and heterochromatin assembly in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">6549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DSoppe%26rft.auinit1%253DW.%2BJ.%2BJ.%26rft.volume%253D21%26rft.issue%253D23%26rft.spage%253D6549%26rft.epage%253D6559%26rft.atitle%253DDNA%2Bmethylation%2Bcontrols%2Bhistone%2BH3%2Blysine%2B9%2Bmethylation%2Band%2Bheterochromatin%2Bassembly%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdf657%26rft_id%253Dinfo%253Apmid%252F12456661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjEvMjMvNjU0OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.36" data-doi="10.1038/srep16259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stastna</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harvey</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genotype-dependent lifespan effects in peptone deprived <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">16259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D16259%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep16259%26rft_id%253Dinfo%253Apmid%252F26539794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep16259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26539794&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.37" data-doi="10.1016/j.tplants.2004.01.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tessadori</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Driel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Cytogenetics as a tool to study gene regulation.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DTessadori%26rft.auinit1%253DF.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D147%26rft.epage%253D153%26rft.atitle%253DCytogenetics%2Bas%2Ba%2Btool%2Bto%2Bstudy%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2004.01.008%26rft_id%253Dinfo%253Apmid%252F15003238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2004.01.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15003238&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220461100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.38" data-doi="10.1242/jcs.000026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tessadori</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chupeau</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chupeau</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knip</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Germann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">a</span> <span class="cit-article-title">Large-scale dissociation and sequential reassembly of pericentric heterochromatin in dedifferentiated <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">1200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCell%2BScience%26rft.stitle%253DJ.%2BCell%2BSci.%26rft.aulast%253DTessadori%26rft.auinit1%253DF.%26rft.volume%253D120%26rft.issue%253D7%26rft.spage%253D1200%26rft.epage%253D1208%26rft.atitle%253DLarge-scale%2Bdissociation%2Band%2Bsequential%2Breassembly%2Bof%2Bpericentric%2Bheterochromatin%2Bin%2Bdedifferentiated%2BArabidopsis%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.000026%26rft_id%253Dinfo%253Apmid%252F17376962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTA6IjEyMC83LzEyMDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.39" data-doi="10.1111/j.1365-313X.2007.03093.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tessadori</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulkes</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Driel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span> <span class="cit-article-title">Light-regulated large-scale reorganization of chromatin during the floral transition in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">848</span>–<span class="cit-lpage">857</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DTessadori%26rft.auinit1%253DF.%26rft.volume%253D50%26rft.issue%253D5%26rft.spage%253D848%26rft.epage%253D857%26rft.atitle%253DLight-regulated%2Blarge-scale%2Breorganization%2Bof%2Bchromatin%2Bduring%2Bthe%2Bfloral%2Btransition%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2007.03093.x%26rft_id%253Dinfo%253Apmid%252F17470059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2007.03093.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17470059&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247175200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.40" data-doi="10.1371/journal.pgen.1000638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tessadori</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Zanten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlova</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clifton</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pontvianne</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>PHYTOCHROME B</em> and <em>HISTONE DEACETYLASE 6</em> control light-induced chromatin compaction in <em>Arabidopsis</em> <em>thaliana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DTessadori%26rft.auinit1%253DF.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De1000638%26rft.epage%253De1000638%26rft.atitle%253DPhytochrome%2BB%2Band%2Bhistone%2Bdeacetylase%2B6%2Bcontrol%2Blight-induced%2Bchromatin%2Bcompaction%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000638%26rft_id%253Dinfo%253Apmid%252F19730687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000638&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19730687&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.41" data-doi="10.1104/pp.110.164616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Zanten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tessadori</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLoughlin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millenaar</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Photoreceptors CRYTOCHROME2 and Phytochrome B control chromatin compaction in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1686</span>–<span class="cit-lpage">1696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.110.164616%26rft_id%253Dinfo%253Apmid%252F20935177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU0LzQvMTY4NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.42" data-doi="10.1073/pnas.1117726108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Zanten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koini</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geyer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brambilla</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Seed maturation in <em>Arabidopsis</em> <em>thaliana</em> is characterized by nuclear size reduction and increased chromatin condensation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">20219</span>–<span class="cit-lpage">20224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1117726108%26rft_id%253Dinfo%253Apmid%252F22123962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzUwLzIwMjE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.43" data-doi="10.1093/mp/sss030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Zanten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tessadori</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peeters</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Shedding light on large-scale chromatin reorganization in <em>Arabidopsis</em> <em>thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BPlant%26rft.stitle%253DMol%2BPlant%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253D583%26rft.epage%253D590%26rft.atitle%253DShedding%2Blight%2Bon%2Blarge-scale%2Bchromatin%2Breorganization%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fsss030%26rft_id%253Dinfo%253Apmid%252F22528207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/sss030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22528207&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304890600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2519.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Zanten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tessadori</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peeters</span>  <span class="cit-name-given-names">A. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Environment-induced chromatin reorganisation and plant acclimation</span>, pp. <span class="cit-fpage">21</span>–<span class="cit-lpage">40</span> in <em>Epigenetic Memory and Control in Plants: Signaling and Communication in Plants</em>, edited by <span class="cit-ed"><span class="cit-name-surname">Grafi</span>  <span class="cit-name-given-names">G.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Ohad</span>  <span class="cit-name-given-names">N.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.45" data-doi="10.1002/jcb.24178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walters</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bommakanti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen-Fix</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Shaping the nucleus: factors and forces.</span> <abbr class="cit-jnl-abbrev">J. Cell. Biochem.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">2813</span>–<span class="cit-lpage">2821</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcellular%2Bbiochemistry%26rft.stitle%253DJ%2BCell%2BBiochem%26rft.aulast%253DWalters%26rft.auinit1%253DA.%2BD.%26rft.volume%253D113%26rft.issue%253D9%26rft.spage%253D2813%26rft.epage%253D2821%26rft.atitle%253DShaping%2Bthe%2Bnucleus%253A%2Bfactors%2Band%2Bforces.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcb.24178%26rft_id%253Dinfo%253Apmid%252F22566057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcb.24178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22566057&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.46" data-doi="10.1093/jxb/ert030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">W.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ke</span>  <span class="cit-name-given-names">Y.-T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Arabidopsis</em> HIT4 encodes a novel chromocentre-localized protein involved in the heat reactivation of transcriptionally silent loci and is essential for heat tolerance in plants.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">1689</span>–<span class="cit-lpage">1701</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fert030%26rft_id%253Dinfo%253Apmid%252F23408827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/ert030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23408827&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.47" data-doi="10.1104/pp.111.189845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Natural variation in <em>Arabidopsis</em>: from molecular genetics to ecological genomics.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.189845%26rft_id%253Dinfo%253Apmid%252F22147517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo3OiIxNTgvMS8yIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2519.48" data-doi="10.1016/j.cell.2014.06.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yelagandula</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stroud</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holec</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The histone variant H2A.W defines heterochromatin and promotes chromatin condensation in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D158%26rft.spage%253D98%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.06.006%26rft_id%253Dinfo%253Apmid%252F24995981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.06.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24995981&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/8/2519.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2511" title="RNA Binding Protein Vigilin Collaborates with miRNAs To Regulate Gene Expression for Caenorhabditis elegans Larval Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2533" title="Constraints on eQTL Fine Mapping in the Presence of Multisite Local Regulation of Gene Expression" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/8"><span>Volume 7 Issue 8, August 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/8.cover-source.jpg?itok=apH_YDuu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2519" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-stnEKuC3io0JhqQBKZtoBDkpfbK3xMhqmgf_8Kfh-3U" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/8/2519" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/8/2519" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/8/2519" class="active" data-icon-position="" data-hide-link-title="0">Genetic Dissection of Morphometric Traits Reveals That Phytochrome B Affects Nucleus Size and Heterochromatin Organization in Arabidopsis thaliana</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485840" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-R7WPhoYV_r_Gh-SZT06JRFEfQPC0qGDJjIUsVpS1HPs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2519.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/8/2519" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-JW1YdY-XzH8fNtAf3giBCG1OecwpZJwl43YzlOb8_TQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/8/2519" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2519.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485840" id="node485840--2" data-pisa="ggg;7/8/2519" data-pisa-master="ggg;g3.117.043539" data-apath="/ggg/7/8/2519.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Dissection of Morphometric Traits Reveals That Phytochrome B Affects Nucleus Size and Heterochromatin Organization in <em>Arabidopsis thaliana</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Basten L. Snoek</span>, <span class="highwire-citation-author" data-delta="1">Penka Pavlova</span>, <span class="highwire-citation-author" data-delta="2">Federico Tessadori</span>, <span class="highwire-citation-author" data-delta="3">Anton J. M. Peeters</span>, <span class="highwire-citation-author" data-delta="4">Clara Bourbousse</span>, <span class="highwire-citation-author" data-delta="5">Fredy Barneche</span>, <span class="highwire-citation-author" data-delta="6">Hans de Jong</span>, <span class="highwire-citation-author" data-delta="7">Paul F. Fransz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-2810-7374" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Martijn van Zanten</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2519-2531;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043539 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485840--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Basten L. Snoek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University, 6708 PB, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Theoretical Biology and Bioinformatics, Institute of Biodynamics and Biocomplexity</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Basten%2BL.%2BSnoek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snoek%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABasten%2BL.%2BSnoek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Penka Pavlova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Genetics, Wageningen University, 6708 PB, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant Development and (Epi)Genetics, Swammerdam Institute for Life Sciences, University of Amsterdam, 1098 SM, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Penka%2BPavlova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pavlova%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APenka%2BPavlova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Federico Tessadori </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant Development and (Epi)Genetics, Swammerdam Institute for Life Sciences, University of Amsterdam, 1098 SM, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Hubrecht Institute, Royal Netherlands Academy of Arts and Sciences, University Medical Center Utrecht, 3584 CT, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Federico%2BTessadori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tessadori%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFederico%2BTessadori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anton J. M. Peeters </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Biology, Institute of Education, Utrecht University, 3584 CH, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anton%2BJ.%2BM.%2BPeeters%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peeters%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnton%2BJ.%2BM.%2BPeeters%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clara Bourbousse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institut de Biologie de l’Ecole Normale Supérieure, Centre National de la Recherche Scientifique, Unité Mixte de Recherche 8197, Institut National de la Santé et de la Recherche Médicale U1024, Ecole Normale Supérieure, Paris Sciences et Lettres Research University, F-75005, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clara%2BBourbousse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourbousse%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClara%2BBourbousse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Fredy Barneche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institut de Biologie de l’Ecole Normale Supérieure, Centre National de la Recherche Scientifique, Unité Mixte de Recherche 8197, Institut National de la Santé et de la Recherche Médicale U1024, Ecole Normale Supérieure, Paris Sciences et Lettres Research University, F-75005, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fredy%2BBarneche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barneche%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFredy%2BBarneche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hans de Jong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Genetics, Wageningen University, 6708 PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2Bde%2BJong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Jong%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2Bde%2BJong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Paul F. Fransz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant Development and (Epi)Genetics, Swammerdam Institute for Life Sciences, University of Amsterdam, 1098 SM, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BF.%2BFransz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fransz%20PF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BF.%2BFransz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Martijn van Zanten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Molecular Plant Physiology, Institute of Environmental Biology, Utrecht University, 3584 CH, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martijn%2Bvan%2BZanten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Zanten%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartijn%2Bvan%2BZanten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2810-7374" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martijn van Zanten</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.vanzanten@uu.nl" class="" data-icon-position="" data-hide-link-title="0">m.vanzanten@uu.nl</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/8/2519" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/8/2519&amp;title=Genetic%20Dissection%20of%20Morphometric%20Traits%20Reveals%20That%20Phytochrome%20B%20Affects%20Nucleus%20Size%20and%20Heterochromatin%20Organization%20in%20Arabidopsis%20thaliana" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/8/2519&amp;title=Genetic%20Dissection%20of%20Morphometric%20Traits%20Reveals%20That%20Phytochrome%20B%20Affects%20Nucleus%20Size%20and%20Heterochromatin%20Organization%20in%20Arabidopsis%20thaliana" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/8/2519&amp;title=Genetic%20Dissection%20of%20Morphometric%20Traits%20Reveals%20That%20Phytochrome%20B%20Affects%20Nucleus%20Size%20and%20Heterochromatin%20Organization%20in%20Arabidopsis%20thaliana" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/8/2519&amp;text=Genetic%20Dissection%20of%20Morphometric%20Traits%20Reveals%20That%20Phytochrome%20B%20Affects%20Nucleus%20Size%20and%20Heterochromatin%20Organization%20in%20Arabidopsis%20thaliana" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/8/2519&amp;title=Genetic%20Dissection%20of%20Morphometric%20Traits%20Reveals%20That%20Phytochrome%20B%20Affects%20Nucleus%20Size%20and%20Heterochromatin%20Organization%20in%20Arabidopsis%20thaliana" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/8/2519&amp;t=Genetic%20Dissection%20of%20Morphometric%20Traits%20Reveals%20That%20Phytochrome%20B%20Affects%20Nucleus%20Size%20and%20Heterochromatin%20Organization%20in%20Arabidopsis%20thaliana" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/8/2519&amp;title=Genetic%20Dissection%20of%20Morphometric%20Traits%20Reveals%20That%20Phytochrome%20B%20Affects%20Nucleus%20Size%20and%20Heterochromatin%20Organization%20in%20Arabidopsis%20thaliana" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/8/2519&amp;title=Genetic%20Dissection%20of%20Morphometric%20Traits%20Reveals%20That%20Phytochrome%20B%20Affects%20Nucleus%20Size%20and%20Heterochromatin%20Organization%20in%20Arabidopsis%20thaliana" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485840/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485840" id="node485840--3" data-pisa="ggg;7/8/2519" data-pisa-master="ggg;g3.117.043539" data-apath="/ggg/7/8/2519.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Dissection of Morphometric Traits Reveals That Phytochrome B Affects Nucleus Size and Heterochromatin Organization in <em>Arabidopsis thaliana</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Basten L. Snoek</span>, <span class="highwire-citation-author" data-delta="1">Penka Pavlova</span>, <span class="highwire-citation-author" data-delta="2">Federico Tessadori</span>, <span class="highwire-citation-author" data-delta="3">Anton J. M. Peeters</span>, <span class="highwire-citation-author" data-delta="4">Clara Bourbousse</span>, <span class="highwire-citation-author" data-delta="5">Fredy Barneche</span>, <span class="highwire-citation-author" data-delta="6">Hans de Jong</span>, <span class="highwire-citation-author" data-delta="7">Paul F. Fransz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-2810-7374" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Martijn van Zanten</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2519-2531;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043539 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485840--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Basten L. Snoek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University, 6708 PB, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Theoretical Biology and Bioinformatics, Institute of Biodynamics and Biocomplexity</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Basten%2BL.%2BSnoek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snoek%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABasten%2BL.%2BSnoek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Penka Pavlova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Genetics, Wageningen University, 6708 PB, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant Development and (Epi)Genetics, Swammerdam Institute for Life Sciences, University of Amsterdam, 1098 SM, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Penka%2BPavlova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pavlova%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APenka%2BPavlova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Federico Tessadori </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant Development and (Epi)Genetics, Swammerdam Institute for Life Sciences, University of Amsterdam, 1098 SM, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Hubrecht Institute, Royal Netherlands Academy of Arts and Sciences, University Medical Center Utrecht, 3584 CT, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Federico%2BTessadori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tessadori%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFederico%2BTessadori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anton J. M. Peeters </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Biology, Institute of Education, Utrecht University, 3584 CH, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anton%2BJ.%2BM.%2BPeeters%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peeters%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnton%2BJ.%2BM.%2BPeeters%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clara Bourbousse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institut de Biologie de l’Ecole Normale Supérieure, Centre National de la Recherche Scientifique, Unité Mixte de Recherche 8197, Institut National de la Santé et de la Recherche Médicale U1024, Ecole Normale Supérieure, Paris Sciences et Lettres Research University, F-75005, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clara%2BBourbousse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourbousse%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClara%2BBourbousse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Fredy Barneche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institut de Biologie de l’Ecole Normale Supérieure, Centre National de la Recherche Scientifique, Unité Mixte de Recherche 8197, Institut National de la Santé et de la Recherche Médicale U1024, Ecole Normale Supérieure, Paris Sciences et Lettres Research University, F-75005, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fredy%2BBarneche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barneche%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFredy%2BBarneche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hans de Jong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Genetics, Wageningen University, 6708 PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2Bde%2BJong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Jong%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2Bde%2BJong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Paul F. Fransz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Plant Development and (Epi)Genetics, Swammerdam Institute for Life Sciences, University of Amsterdam, 1098 SM, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BF.%2BFransz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fransz%20PF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BF.%2BFransz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Martijn van Zanten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Molecular Plant Physiology, Institute of Environmental Biology, Utrecht University, 3584 CH, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martijn%2Bvan%2BZanten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Zanten%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartijn%2Bvan%2BZanten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2810-7374" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martijn van Zanten</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.vanzanten@uu.nl" class="" data-icon-position="" data-hide-link-title="0">m.vanzanten@uu.nl</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2519&amp;rft.epage=2531&amp;rft.atitle=Genetic%20Dissection%20of%20Morphometric%20Traits%20Reveals%20That%20Phytochrome%20B%20Affects%20Nucleus%20Size%20and%20Heterochromatin%20Organization%20in%20Arabidopsis%20thaliana&amp;rft.volume=7&amp;rft.issue=8&amp;rft.date=2017-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Snoek%2C+Basten+L.&amp;rft.au=Pavlova%2C+Penka&amp;rft.au=Tessadori%2C+Federico&amp;rft.au=Peeters%2C+Anton+J.+M.&amp;rft.au=Bourbousse%2C+Clara&amp;rft.au=Barneche%2C+Fredy&amp;rft.au=de+Jong%2C+Hans&amp;rft.au=Fransz%2C+Paul+F.&amp;rft.au=van+Zanten%2C+Martijn"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485840/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485840/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485840/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485840/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485840/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485840/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485840/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485840/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485840/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485840/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485840/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485840/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/8/2519" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/8/2519.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/8/2519.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/8/2519.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/8/2519.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
