<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/8/2533" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/8/2533.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/8/2533.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/8/2533.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/8/2533/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/8/2533.atom" />
<meta name="HW.pisa" content="ggg;7/8/2533" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Constraints on eQTL Fine Mapping in the Presence of Multisite Local Regulation of Gene Expression" />
<meta name="DC.Identifier" content="10.1534/g3.117.043752" />
<meta name="DC.Date" content="2017-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Zeng et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Expression quantitative trait locus (eQTL) detection has emerged as an important tool for unraveling of the relationship between genetic risk factors and disease or clinical phenotypes. Most studies use single marker linear regression to discover primary signals, followed by sequential conditional modeling to detect secondary genetic variants affecting gene expression. However, this approach assumes that functional variants are sparsely distributed and that close linkage between them has little impact on estimation of their precise location and the magnitude of effects. We describe a series of simulation studies designed to evaluate the impact of linkage disequilibrium (LD) on the fine mapping of causal variants with typical eQTL effect sizes. In the presence of multisite regulation, even though between 80 and 90% of modeled eSNPs associate with normally distributed traits, up to 10% of all secondary signals could be statistical artifacts, and at least 5% but up to one-quarter of credible intervals of SNPs within r 2 &gt; 0.8 of the peak may not even include a causal site. The Bayesian methods eCAVIAR and DAP (Deterministic Approximation of Posteriors) provide only modest improvement in resolution. Given the strong empirical evidence that gene expression is commonly regulated by more than one variant, we conclude that the fine mapping of causal variants needs to be adjusted for multisite influences, as conditional estimates can be highly biased by interference among linked sites, but ultimately experimental verification of individual effects is needed. Presumably similar conclusions apply not just to eQTL mapping, but to multisite influences on fine mapping of most types of quantitative trait." />
<meta name="DC.Contributor" content="Biao Zeng" />
<meta name="DC.Contributor" content="Luke R. Lloyd-Jones" />
<meta name="DC.Contributor" content="Alexander Holloway" />
<meta name="DC.Contributor" content="Urko M. Marigorta" />
<meta name="DC.Contributor" content="Andres Metspalu" />
<meta name="DC.Contributor" content="Grant W. Montgomery" />
<meta name="DC.Contributor" content="Tonu Esko" />
<meta name="DC.Contributor" content="Kenneth L. Brigham" />
<meta name="DC.Contributor" content="Arshed A. Quyyumi" />
<meta name="DC.Contributor" content="Youssef Idaghdour" />
<meta name="DC.Contributor" content="Jian Yang" />
<meta name="DC.Contributor" content="Peter M. Visscher" />
<meta name="DC.Contributor" content="Joseph E. Powell" />
<meta name="DC.Contributor" content="Greg Gibson" />
<meta name="article:published_time" content="2017-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Constraints on eQTL Fine Mapping in the Presence of Multisite Local Regulation of Gene Expression" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Expression quantitative trait locus (eQTL) detection has emerged as an important tool for unraveling of the relationship between genetic risk factors and disease or clinical phenotypes. Most studies use single marker linear regression to discover primary signals, followed by sequential conditional modeling to detect secondary genetic variants affecting gene expression. However, this approach assumes that functional variants are sparsely distributed and that close linkage between them has little impact on estimation of their precise location and the magnitude of effects. We describe a series of simulation studies designed to evaluate the impact of linkage disequilibrium (LD) on the fine mapping of causal variants with typical eQTL effect sizes. In the presence of multisite regulation, even though between 80 and 90% of modeled eSNPs associate with normally distributed traits, up to 10% of all secondary signals could be statistical artifacts, and at least 5% but up to one-quarter of credible intervals of SNPs within &lt;i&gt;r&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt; &amp;gt; 0.8 of the peak may not even include a causal site. The Bayesian methods eCAVIAR and DAP (Deterministic Approximation of Posteriors) provide only modest improvement in resolution. Given the strong empirical evidence that gene expression is commonly regulated by more than one variant, we conclude that the fine mapping of causal variants needs to be adjusted for multisite influences, as conditional estimates can be highly biased by interference among linked sites, but ultimately experimental verification of individual effects is needed. Presumably similar conclusions apply not just to eQTL mapping, but to multisite influences on fine mapping of most types of quantitative trait.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/08/01" />
<meta name="citation_mjid" content="ggg;7/8/2533" />
<meta name="citation_id" content="7/8/2533" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/8/2533" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/8/2533.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/8/2533.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/8/2533.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.043752" />
<meta name="citation_pmid" content="28600440" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2533" />
<meta name="citation_lastpage" content="2544" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Biao Zeng" />
<meta name="citation_author_institution" content="School of Biological Sciences and Center for Integrative Genomics, Georgia Institute of Technology, Atlanta, Georgia 30332" />
<meta name="citation_author" content="Luke R. Lloyd-Jones" />
<meta name="citation_author_institution" content="Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia" />
<meta name="citation_author" content="Alexander Holloway" />
<meta name="citation_author_institution" content="Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia" />
<meta name="citation_author" content="Urko M. Marigorta" />
<meta name="citation_author_institution" content="School of Biological Sciences and Center for Integrative Genomics, Georgia Institute of Technology, Atlanta, Georgia 30332" />
<meta name="citation_author" content="Andres Metspalu" />
<meta name="citation_author_institution" content="Estonian Genome Center, University of Tartu, 50090, Estonia" />
<meta name="citation_author" content="Grant W. Montgomery" />
<meta name="citation_author_institution" content="Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia" />
<meta name="citation_author" content="Tonu Esko" />
<meta name="citation_author_institution" content="Estonian Genome Center, University of Tartu, 50090, Estonia" />
<meta name="citation_author" content="Kenneth L. Brigham" />
<meta name="citation_author_institution" content="Department of Medicine, Emory University, Atlanta, Georgia 30329" />
<meta name="citation_author" content="Arshed A. Quyyumi" />
<meta name="citation_author_institution" content="Department of Medicine, Emory University, Atlanta, Georgia 30329" />
<meta name="citation_author" content="Youssef Idaghdour" />
<meta name="citation_author_institution" content="Division of Biology, New York University Abu Dhabi, Saadiyat Island, United Arab Emirates" />
<meta name="citation_author" content="Jian Yang" />
<meta name="citation_author_institution" content="Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia" />
<meta name="citation_author" content="Peter M. Visscher" />
<meta name="citation_author_institution" content="Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia" />
<meta name="citation_author" content="Joseph E. Powell" />
<meta name="citation_author_institution" content="Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia" />
<meta name="citation_author" content="Greg Gibson" />
<meta name="citation_author_institution" content="School of Biological Sciences and Center for Integrative Genomics, Georgia Institute of Technology, Atlanta, Georgia 30332" />
<meta name="citation_author_email" content="greg.gibson@biology.gatech.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5352-5877" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=F. W. Albert;citation_author=L. Kruglyak;citation_title=The role of regulatory variation in complex traits and disease.;citation_pages=197-212;citation_volume=16;citation_year=2015;citation_pmid=25707927;citation_doi=10.1038/nrg3891" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Benner;citation_author=C. C. Spencer;citation_author=A. S. Havulinna;citation_author=V. Salomaa;citation_author=S. Ripatti;citation_title=FINEMAP: efficient variable selection using summary data from genome-wide association studies.;citation_volume=32;citation_year=2016;citation_pmid=26773131;citation_doi=10.1093/bioinformatics/btw018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Chen;citation_author=B. R. Larrabee;citation_author=I. G. Ovsyannikova;citation_author=R. B. Kennedy;citation_author=I. H. Haralambieva;citation_title=Fine mapping causal variants with an approximate Bayesian method using marginal test statistics.;citation_volume=200;citation_year=2015;citation_pmid=25948564;citation_doi=10.1534/genetics.115.176107" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. Chun;citation_author=A. Casparino;citation_author=N. A. Patsopoulos;citation_author=D. Croteau-Chonka;citation_author=B. A. Raby;citation_title=Limited statistical evidence for shared genetic effects of eQTLs and autoimmune disease-associated loci in three major immune-cell types.;citation_pages=600-605;citation_volume=49;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. Chung;citation_author=C. Yang;citation_author=C. Li;citation_author=J. Gelernter;citation_author=H. Zhao;citation_title=GPA: a statistical approach to prioritizing GWAS results by integrating pleiotropy and annotation.;citation_pages=e1004787;citation_volume=10;citation_year=2014;citation_pmid=25393678;citation_doi=10.1371/journal.pgen.1004787" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=F. A. Cubillos;citation_author=V. Coustham;citation_author=O. Loudet;citation_title=Lessons from eQTL mapping studies: non-coding regions and their role behind natural phenotypic variation in plants.;citation_pages=192-198;citation_volume=15;citation_year=2012;citation_issue=2;citation_pmid=22265229;citation_doi=10.1016/j.pbi.2012.01.005" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. P. Dickson;citation_author=K. Wang;citation_author=I. Krantz;citation_author=H. Hakonarson;citation_author=D. B. Goldstein;citation_title=Rare variants create synthetic genome-wide associations.;citation_pages=e1000294-e1000294;citation_volume=8;citation_year=2010;citation_issue=1;citation_pmid=20126254;citation_doi=10.1371/journal.pbio.1000294" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=An integrated encyclopedia of DNA elements in the human genome.;citation_pages=57-74;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955616;citation_doi=10.1038/nature11247" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=K. K. Farh;citation_author=A. Marson;citation_author=J. Zhu;citation_author=M. Kleineweitfeld;citation_author=W. J. Housley;citation_title=Genetic and epigenetic fine mapping of causal autoimmune disease variants.;citation_pages=337-343;citation_volume=518;citation_year=2015;citation_pmid=25363779;citation_doi=10.1038/nature13835" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. K. Finucane;citation_author=M. Bulik-Sullivan;citation_author=A. Gusev;citation_author=G. Trynka;citation_author=Y. Reshef;citation_title=Partitioning heritability by functional annotation using genome-wide association summary statistics.;citation_pages=1228-1235;citation_volume=47;citation_year=2015;citation_pmid=26414678;citation_doi=10.1038/ng.3404" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. D. Fortune;citation_author=H. Guo;citation_author=O. Burren;citation_author=E. Schofield;citation_author=N. M. Walker;citation_title=Statistical colocalization of genetic risk variants for related autoimmune diseases in the context of common controls.;citation_pages=839-846;citation_volume=47;citation_year=2015;citation_pmid=26053495;citation_doi=10.1038/ng.3330" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=K. J. T. Gaulton;citation_author=Y. Ferreira;citation_author=A. Lee;citation_author=R. Raimono;citation_author=R. Mägi;citation_title=Genetic fine mapping and genomic annotation defines causal mechanisms at type 2 diabetes susceptibility loci.;citation_pages=1415-1425;citation_volume=47;citation_year=2015;citation_pmid=26551672;citation_doi=10.1038/ng.3437" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. Giambartolomei;citation_author=D. Vukcevic;citation_author=E. E. Schadt;citation_author=L. Franke;citation_author=A. D. Hingorani;citation_title=Bayesian test for colocalisation between pairs of genetic association studies using summary statistics.;citation_pages=e1004383;citation_volume=10;citation_year=2014;citation_pmid=24830394;citation_doi=10.1371/journal.pgen.1004383" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=A. Gusev;citation_author=S. H. Lee;citation_author=G. Trynka;citation_author=H. Finucane;citation_author=B. J. Vilhjálmsson;citation_title=Partitioning heritability of regulatory and cell-type-specific variants across 11 common diseases.;citation_pages=535-552;citation_volume=95;citation_year=2014;citation_pmid=25439723;citation_doi=10.1016/j.ajhg.2014.10.004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Hormozdiari;citation_author=E. Kostem;citation_author=E. Y. Kang;citation_author=B. Pasaniuc;citation_author=E. Eskin;citation_title=Identifying causal variants at loci with multiple signals of association.;citation_volume=198;citation_year=2014;citation_pmid=25104515;citation_doi=10.1534/genetics.114.167908" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=F. Hormozdiari;citation_author=M. van de Bunt;citation_author=A. V. Segre;citation_author=X. Li;citation_author=J. W. Joo;citation_title=Colocalization of GWAS and eQTL signals detects target genes.;citation_pages=1245-1260;citation_volume=99;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Howie;citation_author=C. Fuchsberger;citation_author=M. Stephens;citation_author=J. Marchini;citation_author=G. R. Abecasis;citation_title=Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.;citation_pages=955-959;citation_volume=44;citation_year=2012;citation_issue=8;citation_pmid=22820512;citation_doi=10.1038/ng.2354" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=Y.-F. Huang;citation_author=B. Gulko;citation_author=A. Siepel;citation_title=Fast, scalable prediction of deleterious non-coding variants from functional and population genomic data.;citation_pages=618-624;citation_volume=49;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Y. Idaghdour;citation_author=W. Czika;citation_author=K. V. Shianna;citation_author=S. H. Lee;citation_author=P. M. Visscher;citation_title=Geographical genomics of human leukocyte gene expression variation in southern Morocco.;citation_pages=62-67;citation_volume=42;citation_year=2010;citation_issue=1;citation_pmid=19966804;citation_doi=10.1038/ng.495" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=G. Kichaev;citation_author=B. Pasaniuc;citation_title=Leveraging functional-annotation data in trans-ethnic fine-mapping studies.;citation_pages=260-267;citation_volume=97;citation_year=2015;citation_pmid=26189819;citation_doi=10.1016/j.ajhg.2015.06.007" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. Kichaev;citation_author=W. Y. Yang;citation_author=S. Lindstrom;citation_author=F. Hormozdiari;citation_author=E. Eskin;citation_title=Integrating functional data to prioritize causal variants in statistical fine-mapping studies.;citation_pages=e1004722;citation_volume=10;citation_year=2014;citation_pmid=25357204;citation_doi=10.1371/journal.pgen.1004722" />
<meta name="citation_reference" content="citation_journal_title=Genome Med.;citation_author=J. Kim;citation_author=N. Ghasemzadeh;citation_author=D. J. Eapen;citation_author=N. C. Chung;citation_author=J. D. Storey;citation_title=Gene expression profiles associated with acute myocardial infarction and risk of cardiovascular death.;citation_pages=40;citation_volume=6;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Kircher;citation_author=D. M. Witten;citation_author=P. Jain;citation_author=B. J. O’Roak;citation_author=G. M. Cooper;citation_title=A general framework for estimating the relative pathogenicity of human genetic variants.;citation_pages=310-315;citation_volume=46;citation_year=2014;citation_pmid=24487276;citation_doi=10.1038/ng.2892" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=T. Lappalainen;citation_author=M. Sammeth;citation_author=M. R. Friedländer;citation_author=P. A. ’t Hoen;citation_author=J. Monlong;citation_title=Transcriptome and genome sequencing uncovers functional variation in humans.;citation_pages=506-511;citation_volume=501;citation_year=2013;citation_pmid=24037378;citation_doi=10.1038/nature12531" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Liang;citation_author=N. Morar;citation_author=A. L. Dixon;citation_author=G. M. Lathrop;citation_author=G. R. Abecasis;citation_title=A cross-platform analysis of 14,177 expression quantitative trait loci derived from lymphoblastoid cell lines.;citation_volume=23;citation_year=2013;citation_pmid=23345460;citation_doi=10.1101/gr.142521.112" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=C. T. Liu;citation_author=S. Raghavan;citation_author=N. Maruthur;citation_author=E. K. Kabagambe;citation_author=J. Hong;citation_title=Trans-ethnic meta-analysis and functional annotation illuminates the genetic architecture of fasting glucose and insulin.;citation_pages=56-75;citation_volume=99;citation_year=2016;citation_pmid=27321945;citation_doi=10.1016/j.ajhg.2016.05.006" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=L. R. Lloyd-Jones;citation_author=A. Holloway;citation_author=A. Bakshi;citation_author=A. McRae;citation_author=J. Yang;citation_title=The genetic architecture of gene expression in peripheral blood.;citation_pages=228-237;citation_volume=100;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. T. Maurano;citation_author=R. Humbert;citation_author=E. Rynes;citation_author=R. E. Thurman;citation_author=E. Haugen;citation_title=Systematic localization of common disease-associated variation in regulatory DNA.;citation_volume=337;citation_year=2012;citation_pmid=22955828;citation_doi=10.1126/science.1222794" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. T. Maurano;citation_author=E. Haugen;citation_author=R. Sandstrom;citation_author=J. Vierstra;citation_author=A. Shafer;citation_title=Large-scale identification of sequence variants influencing human transcription factor occupancy in vivo.;citation_pages=1393-1401;citation_volume=47;citation_year=2015;citation_pmid=26502339;citation_doi=10.1038/ng.3432" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Musunuru;citation_author=A. Strong;citation_author=M. Frank-Kamenetsky;citation_author=N. E. Lee;citation_author=T. Ahfeldt;citation_title=From noncoding variant to phenotype via SORT1 at the 1p13 cholesterol locus.;citation_pages=714-719;citation_volume=466;citation_year=2010;citation_issue=7307;citation_pmid=20686566;citation_doi=10.1038/nature09266" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. C. Nica;citation_author=S. B. Montgomery;citation_author=A. S. Dimas;citation_author=B. E. Stranger;citation_author=C. Beazlev;citation_title=Candidate causal regulatory effects by integration of expression QTLs with complex trait genetic associations.;citation_pages=e1000895-e1000895;citation_volume=6;citation_year=2010;citation_issue=4;citation_pmid=20369022;citation_doi=10.1371/journal.pgen.1000895" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. L. Nicolae;citation_author=E. Gamazon;citation_author=W. Zhang;citation_author=S. Duan;citation_author=M. E. Dolan;citation_title=Trait-associated SNPs are more likely to be eQTLs: annotation to enhance discovery from GWAS.;citation_pages=e1000888-e1000888;citation_volume=6;citation_year=2010;citation_issue=4;citation_pmid=20369019;citation_doi=10.1371/journal.pgen.1000888" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=J. K. Pickrell;citation_title=Joint analysis of functional genomic data and genome-wide association studies of 18 human traits.;citation_pages=559-573;citation_volume=94;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. E. Powell;citation_author=A. K. Henders;citation_author=A. F. McRae;citation_author=A. Caracella;citation_author=S. Smith;citation_title=The Brisbane Systems Genetics Study: genetical genomics meets complex trait genetics.;citation_pages=e35430-e35430;citation_volume=7;citation_year=2012;citation_issue=4;citation_pmid=22563384;citation_doi=10.1371/journal.pone.0035430" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. E. Powell;citation_author=A. K. Henders;citation_author=A. F. McRae;citation_author=J. Kim;citation_author=G. Hemani;citation_title=Congruence of additive and non-additive effects on gene expression estimated from pedigree and SNP data.;citation_pages=e1003502;citation_volume=9;citation_year=2013;citation_pmid=23696747;citation_doi=10.1371/journal.pgen.1003502" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_title=Integrative analysis of 111 reference human epigenomes.;citation_pages=317-330;citation_volume=518;citation_year=2015;citation_pmid=25693563;citation_doi=10.1038/nature14248" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. Schramm;citation_author=C. Marzi;citation_author=C. Schurmann;citation_author=M. Carstensen;citation_author=E. Reinmaa;citation_title=Mapping the genetic architecture of gene regulation in whole blood.;citation_pages=e93844;citation_volume=9;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. Servin;citation_author=M. Stephens;citation_title=Imputation-based analysis of association studies: candidate regions and quantitative traits.;citation_pages=e114-e114;citation_volume=3;citation_year=2007;citation_issue=7;citation_pmid=17676998;citation_doi=10.1371/journal.pgen.0030114" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=R. Tewhey;citation_author=D. Kotliar;citation_author=D. S. Park;citation_author=B. Liu;citation_author=S. Winnicki;citation_title=Direct identification of hundreds of expression-modulating variants using a multiplexed reporter assay.;citation_pages=1519-1529;citation_volume=165;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=G. Trynka;citation_author=C. Sandor;citation_author=B. Han;citation_author=H. Xu;citation_author=B. E. Stranger;citation_title=Chromatin marks identify critical cell types for fine mapping complex trait variants.;citation_pages=124-130;citation_volume=45;citation_year=2013;citation_pmid=23263488;citation_doi=10.1038/ng.2504" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=M. S. Udler;citation_author=J. Tyrer;citation_author=D. F. Easton;citation_title=Evaluating the power to discriminate between highly correlated SNPs in genetic association studies.;citation_pages=463-468;citation_volume=34;citation_year=2010;citation_issue=5;citation_pmid=20583289;citation_doi=10.1002/gepi.20504" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=X. Wen;citation_author=F. Luca;citation_author=R. Pique-Regi;citation_title=Cross-population joint analysis of eQTLs: fine mapping and functional annotation.;citation_pages=e1005176;citation_volume=11;citation_year=2015;citation_pmid=25906321;citation_doi=10.1371/journal.pgen.1005176" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=X. Wen;citation_author=Y. Lee;citation_author=F. Luca;citation_author=R. Pique-Regi;citation_title=Efficient integrative multi-SNP association analysis via deterministic approximation of posteriors.;citation_pages=1114-1129;citation_volume=98;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. J. Westra;citation_author=M. J. Peters;citation_author=T. Esko;citation_author=H. Yaghootkar;citation_author=C. Schurmann;citation_title=Systematic identification of trans eQTLs as putative drivers of known disease associations.;citation_pages=1238-1243;citation_volume=45;citation_year=2013;citation_pmid=24013639;citation_doi=10.1038/ng.2756" />
<meta name="citation_reference" content="citation_journal_title=Brain Behav. Immun.;citation_author=A. P. Wingo;citation_author=G. Gibson;citation_title=Blood gene expression profiles suggest altered immune function associated with symptoms of generalized anxiety disorder.;citation_pages=184-191;citation_volume=43;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. R. Wood;citation_author=T. Esko;citation_author=J. Yang;citation_author=S. Vedantam;citation_author=T. H. Pers;citation_title=Defining the role of common variation in the genomic and biological architecture of adult human height.;citation_pages=1173-1186;citation_volume=46;citation_year=2014;citation_pmid=25282103;citation_doi=10.1038/ng.3097" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=F. A. Wright;citation_author=P. F. Sullivan;citation_author=A. I. Brooks;citation_author=F. Zou;citation_author=W. Sun;citation_title=Heritability and genomics of gene expression in peripheral blood.;citation_pages=430-437;citation_volume=46;citation_year=2014;citation_pmid=24728292;citation_doi=10.1038/ng.2951" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Yang;citation_author=T. Ferreira;citation_author=A. P. Morris;citation_author=S. E. Medland;citation_title=Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits.;citation_pages=369-375;citation_volume=44;citation_year=2012;citation_pmid=22426310;citation_doi=10.1038/ng.2213" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Yang;citation_author=A. Bakshi;citation_author=Z. Zhu;citation_author=G. Hemani;citation_author=A. A. Vinkhuyzen;citation_title=Genetic variance estimation with imputed variants finds negligible missing heritability for human height and body mass index.;citation_pages=1114-1120;citation_volume=47;citation_year=2015;citation_pmid=26323059;citation_doi=10.1038/ng.3390" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=J. Zhao;citation_author=I. Akinsanmi;citation_author=D. Arafat;citation_author=T. J. Cradick;citation_author=C. M. Lee;citation_title=A burden of rare variants associated with extremes of gene expression in human peripheral blood.;citation_pages=299-309;citation_volume=98;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=X. Zhou;citation_author=P. Carbonetto;citation_author=M. Stephens;citation_title=Polygenic modeling with Bayesian sparse linear mixed models.;citation_pages=e1003264;citation_volume=9;citation_year=2013;citation_pmid=23408905;citation_doi=10.1371/journal.pgen.1003264" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=Q. Zhu;citation_author=D. Ge;citation_author=E. L. Heinzen;citation_author=S. P. Dickson;citation_author=T. J. Urban;citation_title=Prioritizing genetic variants for causality on the basis of preferential linkage disequilibrium.;citation_pages=422-434;citation_volume=91;citation_year=2012;citation_issue=3;citation_pmid=22939045;citation_doi=10.1016/j.ajhg.2012.07.010" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Constraints on eQTL Fine Mapping in the Presence of Multisite Local Regulation of Gene Expression" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="twitter:description" content="Expression quantitative trait locus (eQTL) detection has emerged as an important tool for unraveling of the relationship between genetic risk factors and disease or clinical phenotypes. Most studies use single marker linear regression to discover primary signals, followed by sequential conditional modeling to detect secondary genetic variants affecting gene expression. However, this approach assumes that functional variants are sparsely distributed and that close linkage between them has little impact on estimation of their precise location and the magnitude of effects. We describe a series of simulation studies designed to evaluate the impact of linkage disequilibrium (LD) on the fine mapping of causal variants with typical eQTL effect sizes. In the presence of multisite regulation, even though between 80 and 90% of modeled eSNPs associate with normally distributed traits, up to 10% of all secondary signals could be statistical artifacts, and at least 5% but up to one-quarter of credible intervals of SNPs within r 2 &gt; 0.8 of the peak may not even include a causal site. The Bayesian methods eCAVIAR and DAP (Deterministic Approximation of Posteriors) provide only modest improvement in resolution. Given the strong empirical evidence that gene expression is commonly regulated by more than one variant, we conclude that the fine mapping of causal variants needs to be adjusted for multisite influences, as conditional estimates can be highly biased by interference among linked sites, but ultimately experimental verification of individual effects is needed. Presumably similar conclusions apply not just to eQTL mapping, but to multisite influences on fine mapping of most types of quantitative trait." />
<meta name="og-title" property="og:title" content="Constraints on eQTL Fine Mapping in the Presence of Multisite Local Regulation of Gene Expression" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/8/2533" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Expression quantitative trait locus (eQTL) detection has emerged as an important tool for unraveling of the relationship between genetic risk factors and disease or clinical phenotypes. Most studies use single marker linear regression to discover primary signals, followed by sequential conditional modeling to detect secondary genetic variants affecting gene expression. However, this approach assumes that functional variants are sparsely distributed and that close linkage between them has little impact on estimation of their precise location and the magnitude of effects. We describe a series of simulation studies designed to evaluate the impact of linkage disequilibrium (LD) on the fine mapping of causal variants with typical eQTL effect sizes. In the presence of multisite regulation, even though between 80 and 90% of modeled eSNPs associate with normally distributed traits, up to 10% of all secondary signals could be statistical artifacts, and at least 5% but up to one-quarter of credible intervals of SNPs within r 2 &gt; 0.8 of the peak may not even include a causal site. The Bayesian methods eCAVIAR and DAP (Deterministic Approximation of Posteriors) provide only modest improvement in resolution. Given the strong empirical evidence that gene expression is commonly regulated by more than one variant, we conclude that the fine mapping of causal variants needs to be adjusted for multisite influences, as conditional estimates can be highly biased by interference among linked sites, but ultimately experimental verification of individual effects is needed. Presumably similar conclusions apply not just to eQTL mapping, but to multisite influences on fine mapping of most types of quantitative trait." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
    <title>Constraints on eQTL Fine Mapping in the Presence of Multisite Local Regulation of Gene Expression | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"dJen1N-_ZQbwjozWJgugy5zqlgFp6XF4IRJIbE_2HtY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485850","apath":"\/ggg\/7\/8\/2533.atom","pisa":"ggg;7\/8\/2533","ac":{"\/ggg\/7\/8\/2533.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/8\/2533.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2533"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2533"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/8\/2533":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tPFCkjkrM0K5Wo\\\\\\\/KxarA5q+vmc9BV\\\\\\\/XhTgfWyJXTWZrZW4Ww3d7m5ynaUfz285LL3NvxMlT+cusc5lSrvw9l3GqsHdsfLU7sb4tbWUShO1sMKLqi\\\\\\\/HC6KvBlG\\\\\\\/kcPIjOTcaSA40rqRg2V7tbTllGB\\\\\\\/4eUBX1EwqiCdloxxqhYv9tRWHMdFW4VTVyTfFJLUwHBlz+QqswwbfavTi+kADIX+9\\\\\\\/Lzo+\\\\\\\/iGyula34cASnRUYYVfxRz\\\\\\\/DtaZVsNjdUzexAaiaRGTsY8gn2oKMOBBMgUD4hPsqQ10Lzp1LqM0rD9Y3Qw0e6ogjusmFwRq0wyDMdiK79gXHpJmMxF1C6F9p4B6FpTdTDUPkvMWIZHasUnNgB6hqTDMwEZtUEpB6env7fPRd5RComd3DCfRE+oY\\\\\\\/+VVZraWzmhVTasmb0c+8rcaBv8DmkZkaryP7s2igYH3HRtY1T2SUcGggR9pitGmRnfQYF82DcO4ligachrKkKkR+RhZDqxOkXXfGkcskrcVCaky\\\\\\\/x4huyUUSpZeYansrABuyRiEEPurpnc55TZVeL1ogS2h4yq24KvGeJMNUdgnM2jfi40ooh0r\\\\\\\/xjoYWUkA7mK\\\\\\\/OI0xxzPA\\\\\\\/mToPr9qN9+1tozBjxS2kqfXzAENnfQpr2XadjBqta1b0Lr4IZgPFQMKNyC49PXMLEiH6Q1s7h+Ene+VrnYCNLgy3XgcCALiLqL0nPzcHxYIr6z\\\\\\\/Z0BGlv289E1TOEHAesr9brH1IVDwI04Mej\\\\\\\/OAO0FOXlflO6WamMKeY0\\\\\\\/hSVSHJtG7vDBIrdMShEpQ6xtEaZpN6VEzsIx5FkiFW+z3Pztg68EjMLe+qil00nN3Ln46Ex+uhgKk+gLwZ\\\\\\\/DnwI92f+mKp1WDWobroHT196wepHXp3QORdbI7iWOJrD0AgZd8vaUfD\\\\\\\/e+WMM\\\\\\\/IUDPBGAEJL+K8aDlv7b\\\\\\\/oUulhV+T6ApyuAJfBIuBmE2ZlMUg7c0VcvETDs9LgYzq7oj8QzGGwnYjM7uhhT7wOlIezfWdp6Zac1oAksoc\\\\\\\/cNXfolIgEC4O8Rh39fFPrj6uU32r2no+q\\\\\\\/bV5uWgio+NoaYWczhwdGgAYazOZB5RcIcTe5wcF6fhdy8P04RmKGcKWz6uk+xwu2HGmeujM0CXwvcQHzhyXiOrjURpz7XdrA9iZS\\\\\\\/0ZCJdwkkRpWZuobv6rpaDAqlyiZAsMeKIwBd7F3mgSTTx5hFsJPYOLwCP0Y78u4ZZJBAVRwz6t19E2UPWmHVawASfRjFm2+k\\\\\\\/ysTqrMoQpUNQI1\\\\\\\/7R\\\\\\\/CLW8\\\u0022,\\\u0022iv\\\u0022:\\\u0022eKP+JSV7\\\\\\\/hWDwCzLKkG0qg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229db500984fb2ec54e9493282d708f73b3097d6654688b2233b7ae9390a746fb1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HXw7DSOYu1cZSX2di3Xv6FSEmMSCXCjLX43n7tDc3mO76SaoR6j7bBYPRTstbWIUpO7\\\\\\\/AJbSp7Z8H8xte9Yun4h2zOiK4VSbzwxSsKxzZ\\\\\\\/leOGniIUWEVwlNNtc+Om6ON\\\\\\\/PNLh\\\\\\\/FK+8wcjNcXGU8\\\\\\\/iakm58804gVF43otT+IOKMCPI3zhkntpQVv2cNNIs08IDarm5NdQ2HdYbIHprnYeQgg2VJS05ZurAku8\\\\\\\/DhfcovToR4pidiqfoJ9ZSed+lbndGhuX5cvCXNPQvwfzhLYzv2mZxBju6vkwhw3n\\\\\\\/LyHPVJvhJv5Zk+aHdbT9hAid8MprUGWp4HIoF3behDKEPE6qird99hW2zVRblM96zc+khUYvqZHMuszmL4Sjnz0RUY\\\\\\\/Gey6x2ipOYvApU5CZXoeF3\\\\\\\/AFxIwKvxkaD2QWRVsjFtj9HYWq2dlc7mPl6Z2ErNIP9zgHbIKUlfj6uyHzjqPPI6bu6ZCmDRQtLRljLJDNkWn+C13MVGVmnWlKarC4NgV3G4DTwL7gkjuIL\\\\\\\/KRPQF8gdJ+hwvKeZAR7Akr67aOVyRIkAVlgqaCN8lWspm5oqi5zqybQkYTEcV79PlHXr29x7HnM+YimKw3YIyXzvwEduw9oxVKdMUXfON7uhNfdotXAeWsPUH3EfflMTM0XUXmY6Ie7vjXbSAFJsZwAmKswiF7XUjIBP0J23knTV6bfUPKKQPbgM4Gl2xXBJ2AZizJUwVDeAN\\\\\\\/Hi7J2cruBjm8XJbGhs2KPvV3oVUIZb+Abchm6LPpHVzhNz+jZHbl6RxtIzR3ciHh\\\\\\\/1kV6IdQg4HtwypQyGqRZyynvHdG\\\\\\\/2FpTm7s\\\\\\\/T6sjId0p9OE3Jef5CYz6zyv45j8tntJpMPu58KRXf5p21\\\\\\\/pxo+F66bvMW05Yp5DO7plMiKXjEQaYbKtFmUQmGLm4sKzJjsJjsjSZf66TFsZRY+taFKRP\\\\\\\/V80F9VlZd0gJ7AprWs8EN+J\\\\\\\/9HGzKb\\\\\\\/iMSafWU3ubSGPlV04I5vSUldi5VYOmt7FsHhPjruLwIiCExShCOynm25ZnOInbavUiMTN1NvTnToS6\\\\\\\/OEEuplt+PkAlOZR2LZJnOnng\\\\\\\/7+Kb4Gf+w10NUVK+oCP6rkc4RSsMlTIMIVaMd5AJlQ0SeFVrkJZgoU+qmHpXzhYzOMiDNri0rR0oW05ixm+tlYEQl8vm52hndPrspkPQH3eOll+md1Foxu2OA4SKNR7x43j6ioOn9jCH+KUojO28UHNt+2uT9hcKiamxOqa\\\\\\\/fOUblnep69yaAGwjirxCr2dL+kDv3qkw66J\\\\\\\/G1j9pexKX2hxxVWB8UWuVGvAd3YjNgviMdjwuuQOGBUUCcIn3wwFOTZ2IvwZ47lymvjntahEAF4OgY1JqFLkMBV62BcuMe3XVQ5MqV5ZKeogZJggHhc+fCuGUGNJy3+6SLGxzjVYq+uV4Gp5zV7nMLvoZpNzsj5o6mNEKtua+wrn3pmu732co60omly9rff7Or1isOxAaqRksKmI89QPbWR2QAdDgqQhqVqREO5atwVt82XpPKNnAoxWsrFJ0MtwBqaKD7sdLwG3cifE0E4nn6Drc8fZS3BFAtPVW\\\\\\\/hznmQ4a+S7JYjRjzZ6PF7UNJf+4pFR37IDGDQzaHCSknIlmZY2Mpw6Ju46jr7srvjlwBPvRSOZIrmY+c8\\\\\\\/6SD68EFGk93jtcjAxTXILAiMVdydOMsuRpOBsI3Y210SsS4AkLbb6RwMAjBgabMSXvXj+2UTD1J\\\\\\\/I2I87A2eVsKRaRJuTadxVjcT5uxmmxX8usr\\\\\\\/numa8\\\\\\\/GDolHM0knK9PXRy9ckFpT8hUq6kYPm4K9qZ17ak1a0v039hJP6I+PyfbMgJ1xe0iJH1CR28tPmX8axDjBA9bwXSdZxiRGsbBG+hagWYfa0gcYEhOg0jkR0CI\\\\\\\/fq48ej3ftFpDneXpgNx6fyMEESm9FcQov753vO0FCR5atHl7gecUXHTb9h60n5Kly6YP6IQhpuIPjiQJsZDWavgySZ5zcS\\\\\\\/wLKPKXUSCj9GNdpT\\\\\\\/QXFsIFGwVlCcf64lQe+dKeDed719rIkqn4VsWssEnuPao36l3JZYhlbyMdYPBiaaZGilgnxEf\\\\\\\/qt0PROaI7zlYvbbu5CLGIUhrmVudojvGHrNKKIxLDknLHElom1uQ5VBGxQds58EmKb+Rmf6w6IxEWgggC8mCZQ2rapf6ypcyVOXd3cpZBBDNdcM5EcAW237e9Px+hRakwPvJhIlrBxKa98Dz8fqyYhoAUTvBy3DHu602YvOgfG48cquASn0+5urvehiLxEkxbd3\\\\\\\/B4PF4gGgehSfzSLWTvHqyx8P3srL0J0LJHyXOG17noO8trIl1WTUE1BsQubRU7hJYwYKQLeqvFNX7GR2tTF+B6GxWu7FKRkJ0H52cNwdSVI2a5gKIRNJbXnH2SQbou2qRkdUC+Auox\\\\\\\/Ub3rAueNXIubXs6nPn9USjEY3lgEtYM43LjEAeWtiOO5hugTJRPBvZ41XVNZ7HBFOkYNL91+z7+sf\\\\\\\/6tpvQqJwGBEt9qXsS3IOTv0mCngSxAGkcSfWCHwI7ZkGjltybfLFqHdtvkqEdka0oelhrMfURWulDLGBqSpAk57R8ZXjmxT6Pi6PyJotdqVEiJyIF1ARe1YzCRbxfLmxjRDxChWAqS16+2MORRUmmWCPXx0qyMIpQo3UeWWhzDyRAbZPdX\\\\\\\/8C+TeQbNizIQiTPi68RQFDXHYQLLBIqmkLTvNvQARNPTBuEvoA+XjOdnraj8qHi7zFFheGGWv\\\\\\\/6096Pz8kuJn81E2u9UYaX0i\\\\\\\/glcYaCXWVeD1o40\\\\\\\/ywZ980MF9dTYoJa1ZXffgtW+OspcZcvW68UCv\\\\\\\/NQFOC57FYVp\\\\\\\/DTtE4xafXuxYTycocNCF0G17NhNVChDzOCEAMT1ETz8ZTeZKNyq3QHiPkxXeg6pXHNZC5dQDBE8Q+vc3oXPSeXandixWRAnbOurz7pT3HtYn571ZaKMXUaMMbE3JMDilCpg7w2Zd3kulNJeRpZbZDHjTCjLU8C\\\\\\\/rxjWOgKTV2xG2dPSIPor29d598whB+Zn6toUZN4531jsTM5xjYJy7FG6QQXIwA+vLxhq\\\\\\\/PUAIy6him2q6AmCdZwEwGaPIeR7blKhHdH9I9sbKJe5YhY92txl9E1jzktwhtCVDBoX4Pjs0aTH1zSRNE0hp2H+g4m3l6Q6Ef0ca52cvZZ8FVKfPzv4Mv8UN7FmMPkZuBeiM5m3x1O0\\\\\\\/xRrM4pgF2oKjGnAUEutaA50zN5Np0C2GfRCXB+3v2n3+7nVPGylywyoROVx1v33peK3WnV9Nj7gJXDxnP9jXxQAWI8rw8jgu4qZCKks93qcrqwQIBh+ZFSyRvvIc7ihVaTMCrd2rME0oM3EnO9THxFd3\\\\\\\/l874UmaUkH8DjmvELZI9RVWRVxx8nXI7XZBaTEKRfMy\\\\\\\/lS5LfL8qKPeg9+MCJRlHORQ6ixk1oynP7v5nqI6Ji2cHXjWpn30uHJzvAitmwMZmuZ5rvbPh48kuhDZAZz7U7hjCSc3SLGzqEGwhRPl25ON5Idn94k0UkKajq+rEjOvcFnZZpvUSk\\\\\\\/j+G9OuS0BNbNU9slH97wDrqnHuW7+uDwHsyWdr+w0TsL\\\\\\\/cJA8+rX6APzW+vLowKXy\\\\\\\/gLAodbtMJR4FYVoCSa1Px6yf5b5eSbZEt1iZv7giPGD22+QQYcQw70TZQZC\\\\\\\/p+HxjoEg0FYiRh2MjJBmssUZF6daE1VcxsibzIgiU0Kjk3exZIWhYzvSBYThQ8ShddVERMH49KLQ2\\\\\\\/u9sXeddGAWtBGpJGQp44\\\\\\\/6BKQex8hk8zd0urJSJoYqZtQL0pcSnFAgG7fHatsDDjnshQ9Hd5odaHSJBJrhKCm5j4qRMQWXIsoa9CuK\\\\\\\/3+bnpIyG215SLpmukvMvIRB8TouhCJmDi2A+yPPYEIzOs6LMu8YvzQhod0zNApcwOB\\\\\\\/BiAjQOKvbzAlmDpkYkKIjBo8WjRjpo6OAYjnWEoK9vyeA4AXizxSF1JKCnWR5wps6oLO1gyLiUUEtsz2DoFWJQthBuXR5VxDl3t9RTEFjLLg9lvOXyqVGJnWBlh4mFWzCM874tpsVOJbF8n\\\\\\\/XQb3y9pialZxA9NF0HOZkFMafaATUEEdil2bZb+b2AQ8yA56vFnwUsOcA6PgHMBzJWCQDWuR8d\\\\\\\/cOOdz0vwQwWX6wj+Bnoy6+phtYH\\\\\\\/Zr5jhfqAtF0mwUeMISC9n3eIHiyM4BGZtWlL9e\\\\\\\/nbwmwlYfK4Alkp9unrbc1Tj0drT+K8edogdCHtg3ztLOBfJj+hSxEmToB+zFKYDW6jt5\\\\\\\/3W5rkXfn3bk8toC6vaOTm0\\\\\\\/A7ByBHrlrCXgSB5Af\\\\\\\/I25KQwQmLFjCZ8VX6Y6PUSdZ\\\\\\\/lMTa3Lp9W02j9bSwSXI7RBG3e3pJvWuepDvdErXXEEUixOMJZAaqeJGEOwX3Odf5QCFynZeuVYPY0FH4vnv5WUOVkCJf8\\\\\\\/n5QRa0T6hwZF1yUVzoBmvQbw6JUuez4nW3LMpdbBiyw7ZgX7DdNp1adu\\\\\\\/4ipDkmuQ4u2KDA\\\\\\\/BeFZchq1VVjIAU25OtO5lqAhZZGpT6LDk8HOATZx0lGd+FIr\\\\\\\/9\\\\\\\/DKV0ZS2tXhjbOOj4unB3nzNM3ZEu9ix6oUm4WqpxQlr+uvYuIvu3hJqd7Bc04J22ox\\\\\\\/\\\\\\\/9e7kzHDpD6VORcZ\\\\\\\/QQ3v6+fRjmdpmico\\\\\\\/0AYmpwdgoRgjLOOIZXXFd1RkmFZeyMQXb+gJ2fMxIFXhX8OJfKpIf3m\\\\\\\/hIDADrMTYyhcv50tfJ6mhAG4\\\\\\\/Wr5krRz0YvlEN+Bbfhh998aJ+oHXlb7ChVqnjj+sQbKtH1mKma\\\\\\\/YW79OMQ7hbq7TPV5oNxvjsWIQvXLptxRBQtUJJAs\\\\\\\/bp9a8axU4CAqzeWfp3UN4FwwOot82zc4CUMyWSwfAUODUXgWH\\\\\\\/iVQbtV3Tu+IQfaQ++q0PLqvE1NdcnUoh9IhzEm6+DmcXPurJX4uuWjvoq0UuKfjO3z9JId7vinuRvduGFLpYsOBIbkII6uCTdaP5erABX8bhPcIL1O+2xEXW3+veJhOj1DqPHXkURWhWlBqoUBwQ6EIC7Gkdt+MsT8soZdb3xAp\\\\\\\/VpnjQh2RiV16JdmYaDAARQ8Zbf0bkfPaDbCue8TY2eGZB9FxZg84D9izev0Ws5mm1Esem1dgsG38lW5eAIEYmZgsgQIBjcVB3Abo0TEvHflmH4Ev00oyzZCkmlugWpBiQor09u0cX0zT8eZAUCMOGLC5n7wBHppv0CQxTWFDYnMPBVP6Xv1j\\\\\\\/Ake80cqR4gE2wYv6AsQ6pOz6bLp\\\\\\\/ZMiXOayRumh65PX7DuUT4Zrz6EKi\\\\\\\/bD1vcxai0Q0dpdkmfk4v46uiwTAf5Qi9ZpPFvt1\\\\\\\/hkiae07CahTgbvCbuEBmqJE10+dbkb7VSWaJDlGYaTIY0BdGdZBVNq1Dhe+cRKD2yAan7FEvSgn3FTuA\\\\\\\/w3gQz2R9IlvNL0kufuroeq+9AyVfmJf2j3228jmGfFMMNo07zpnz9O\\\\\\\/9Afagi7o+9CT8\\\\\\\/eiSqEXzwjctLuhQlSlFILzsgWHvJvAONf0h5CL\\\\\\\/v0qs3iAGO\\\\\\\/fmrNa0CPt3qRMjn2\\\\\\\/2JRRW8AcXKkVV08PYWkJfuJkzWVlx6rz5GQrxKdXXHrYjPMZN\\\\\\\/OwlMD6CTAuY\\\\\\\/sgrFrDWmPAGv\\\\\\\/ISkJq3Ujs0b22Zjtl70HbPviKLYabUKU20lkUgdeinNxRJEi+iV1CtDH1rBdx6\\\\\\\/SZySMcHiMjlmI9b+jw4KyTTPo7LUyd92sJPPqbR9zlqJIniw2iIimacF+ZbC1o8QRGuG6sj7CWhjXZ\\\\\\\/fQ2LynRbAlUAQlAWjYVHrFN4ACl8LuXmLRkiby7EeSCg8cA22pl4tYTYWg5\\\\\\\/IdAZonixkqhd49LBk836tjc9ZL87zLtCl87s7XEw49tqqOfutZAM2MUEUQZ1apV9K7rkHe0bLtHSQBkt7yCM2rCPglLZJCJXPrF+gk43EBZEps4uvIZX+NyvNyvTL2VZNAoc4F+JznKZmvMo7wFeeCo8j+V7wUvm4wqFc1KJleJQLOdOA+AtiIVxP98aYrW25K72ts+GMXoA9aFE2WQ7iDnxXYsmyIa0eL4YNFuCp9Rtu0+CHKjElPem6\\\\\\\/Bbu+95+4AH1nOtnd32\\\\\\\/ZCIylXmn29\\\\\\\/A4HJKc2d2fzrMzut958674aAEGIGO05pT4Z6ib2wYS8uHu2d9NmJXbnNR8Gs6YG\\\\\\\/t4FBYniy6Nfm7WumTPGFCEcHSatx7mxucJQusiVElWHqC4SJoNxWCzKituE\\\\\\\/y+A2OPGITAXkz1giQE5qvOK\\\\\\\/UIEAEJ0K9gjpbO4QDP9Y9I4SYzIVoVelVMZlolLaa+8Qq2Ioo\\\\\\\/SJL+2QwSjlFs0Yq9DSuHeksRLPSeBX09o\\\\\\\/Z2Z8mU0QhUoj7kNkQ3jIxwPQLxouJtuKFP12JBjzOyWUB2RBaFBYIk4EmwTGWi3eAZp8dzwkhI0xRIMl0dPkf1nLTk9eZjPbBh6fITb8mqcvC094eCYhHcIyNnFAeeyGrf5EYsqtX3WzF2EVVBf8TQ8ll4DDsF6vf0WjgPsTEGfgdy6vrnv1R8lDVIw+ZD8SgffNpk195MhUJDdJF8k+Ian7ToJYxBE6lwWomyL4ct+7tE60FnOpUYnhFZGwESpfJWZR8SO2ufT082TOGXExPcwmBaHDxTj9Ij2W7bitIrLim8FHA01+04xKFTlHIXiTUgX\\\\\\\/isYyIhpuWsDx3RIRsWdV3C0iP\\\\\\\/Beq6K44Zemu+vIzzbCOeawEmAD7PAF2HnlyhgC4KPFvs+KMVk2URflakHlbTSAZj1\\\\\\\/AD8kmh0DYp46fAaRtjVSbvbN956zPSWRgQcl9WVLtVfvSUcgnSWy+prz9P3BOU9z6yDqvBGSVudP7mZAjJYRswQXlvbF1PF\\\\\\\/IhJmB\\\\\\\/+Zq057u9+ITE8kKT8\\\\\\\/CGyCPS1e6kxXhYqILK61jc1rJV2jYwNxGTq\\\\\\\/Yr4lSQTvmgX0tVqUr0Ef8pAQEwz6YrfV7PBWfdiaXHHYN0IPhzfOrdkfv4yEMCMNRla2QIFYSah9MvoWkuoAC9WF2oIS9LtbYTcdMHm\\\\\\\/lXKx3mz05uLjSDMtdDlpwNE5Ug5nrOOw+hTYtDBCJhPea9AEx37S\\\\\\\/NzKoxCypcZ9zHIJnAwtG\\\\\\\/Q1gRyxrGA\\\\\\\/W6SZ5FEwP4kyrR07inZVi7sc9ZdeZYAALe7f6XiOc04wpPzIoblmGDEUJbWrPEArh4vcMHxfrKobDZFqx\\\\\\\/tlRlrpNkWv9wgECvKM1+mURHPRAdVkix8J3cIR8Io\\\\\\\/Gqvxb73KduYWuYR5TFw3Mu\\\\\\\/YyX2PMw4DieYvA+JTS6P8q5Cc+ApRqJ0oVa32TLwdoemCBhHY2kfMXsh0FSaGOcfK9Uufjj5fXUWbiGudc4IyFLNSD+ubdVmNonZ1R5U7mJSk1pO0Aozur1itpikQabmAWuJFNhmQVbkL1SSNxpqC1GPDjvvxrl3uCbIknOKYVYdK7xBHUAfyxGQ+o4KyM7U7YpYYM6NegYkTbNiLLOWdMKh0kd+MGe9aALeG0d1qJEhMDfVaFlBUIM6EaqkItkmeqj0YIGRESmMVD0L2\\\\\\\/YGMhWcU0417ytSj6he\\\\\\\/SoIZWYzE+mUHoxyam6SkEWOvce5diZVwb\\\\\\\/p1NM\\\\\\\/rTiX5yVd5GcM\\\\\\\/0Z+y+PrPXDjPbvigbJy2ZixSorxVuQR2KjqlLfFWSxo2CRR\\\\\\\/dmKw1qVwJB9OP4WyUbsYvQ2hrlFLwWJsDwmmxIQOaJT2loVz+7S0hVEJTrfmEbZZrfVV9k+fHL+7Iu9JvHpbjREmlXleB+TtBjhuOz7sPMQhGZ+4ng1M2TrE5kVMqrMhvVl7u0fEjUvnR\\\\\\\/RWg3iQp+gs6ES0qT4P8HMNCFp4qkC0rLHncRAX1LDJrybnaHGCzwKUFdWdo58F+KvxnTNBGn6m4CDBh\\\\\\\/lgrsvkmm418RhNTtJC7rzfFTcN+RCv3KjhgjzcE8RPczSQKg6S8Q3PfVtvWcew6FPxQqbDqrxNsc7N3eaLhd0obOJ7MtQqQ2ZBrOU1gD7sDDbQQMfKNhTGJyCnOVc3VzpAmS4XXvhaq\\\\\\\/zI4HHSAgRhEJlrzWgCyhTuSsbQbBPzl\\\\\\\/jLsQeOQkCxJ7KVbbwEL\\\\\\\/WC\\\\\\\/MzDxKSMW0JNVyR8EXWFk5\\\\\\\/DuhtREGK+NxVNSj15G9qE6hCNSkWAzeYCx\\\\\\\/fIGjAD0cL03NBVA1WA9I5v7HV79tN5tK4oflxMT1+xOH6OFXQ42wumqUhd5huINy59yYf7Me8wyA6Si6DmO9nf0cP5POLAH+lbg2jpFNpyfhqsAVa9fzRToa\\\\\\\/ZU7sI31su5veH0K3fK8NaRQ2bg0l\\\\\\\/0hWHNJMXkU\\\\\\\/dmks+4pfgvZOBPSfdFTFHh2k\\\\\\\/9IS8EteL1s03P5QBypH9nDETCupbL4bnOfnyqkLreAu\\\\\\\/LO91erOhBktj0F9QEB80nvyItjSRdFolci3fkdLBXpRFVarcsOG2OqNH6ZuKbRQzlmAKVxGCDZ0evtU7PfNpa3tZbjBAFn5KJeo5abAB+oTIsxgMY51FJVWXwcIUPwlFKVWF8YPfkOViMEdNrjgpeSfbU7\\\\\\\/qP7TdC56NaP7WWasNXca54O9KgtHaQXfaAoWUBdqANnCta\\\\\\\/NqIwfMo+LlNmLOdJ2On5gSjhDv3i7\\\\\\\/ZRDXojLCBjM2Ot\\\\\\\/no632w6ud09Pl7YMZfTLHD8EKf8H1BskOUXdJyXrya3b1ACMbjWpTmffYpwdJCn0Rue2RyimdKhzsVbZ\\\\\\\/XEr3mfNCjrEMMEqssCQ6M6rCBsGFMPNh70f8OqYGm1A0KxofisSB7DzkITjC0GH69zMjOBIb9ua9gkrxUTQr2s47TSdbPdBKBu8p6TcWdyvTjSHhPH1H\\\\\\\/4aeiPKb1FJpo3VzIlrN7kyuAFdlO6ZcL8\\\\\\\/FS+NPiWaK7XJCzBdhXxpCnhhElSiK5dfG1M6qH\\\\\\\/1KtoFXbk3yqk\\\\\\\/IxUL6VLqi5Mj5OGFFfwTUhZASi\\\\\\\/L15hAHvY29uGGiFsZjLCpvHiosxt3ILozeC9ztKb3t4xKBhMUUM2S14BVInz5IrRMzuHCAgUeYpCZwOALyZYMkozBt2uFBi6ByGrqDxAiB8oXP\\\\\\\/KamWi2Y+xgATA9bPeZ53Z12E5s5SJVxChqo6vw3O0olNhZWDDWOh+qFXzoslQP+jBjNEuuWdhLth9fgMMzGoHInXAjigwXP0VwjMCG1JwPTi+34BZM9SFpQJNjxuvobwyavm0ooARDk29EkbjtccqfJnAU3Usi2gy5VMBFbLbUeXvFbn5bQtY3Wxq09eMVNqaPdBV1bBivZCIa1WYGfacmO+j5cWMtESDDleKb+sHZnh19mRQeyScfzqRdmrIDxqkLEckhNcOdQppDwShX2+9mKPpHRwFWVMDp4Pxm3dukRMCJfEO3m6742HWskF7jFqv5YBSRqrshlWIA5f9SnvCciWdrXWw6GLn+QNHoq1pjXDFQxoBo9WHkm9bJg76+9yeTt0eG\\\\\\\/AMrwjNW0\\\\\\\/nLfUEvUYd3MYmHqOaR9Vj5rFPgs2dv1JcAC0noSRXSdwhcdCfkBoZb6mPdNZKdlAUkPyCsaOA5zV4Rq+pPwYP9iTu05\\\\\\\/oedh2haqRV2URqABM3wSWIxXlAUl4gHNGzLchHhPdZugfykatzKDw+kxM5Hw8m70TWdf7YoSvQSZnPJfUodHvhP1JVlYKAOtkKosU2HpB3kSdUNXY0ymalXgsI5Fel\\\\\\\/QFg8sj4N\\\\\\\/z8HGH+axJ2h1+dtDyeL1NjYLghJ0twrQjb125D0uQSxLOQTJwHH0M5S\\\\\\\/Qdl6ugOvrRf+nRQhV9OdjKVMm8iSwg4hzDF+5MJDq133NgWVrKFCHN\\\\\\\/danXD6XP6or5CDOcVbMbTB71pPZ\\\\\\\/4c6s5I9VYdo5vGopMiNB3v+ulA20pICjNtx1mLP2I5ySwqHdR2kTsVi4SiT3nbsdYjZmMb2sgW7O8p7\\\\\\\/qzAw7aKNWAoA35V4yDj\\\\\\\/pr4qDiEEu01\\\\\\\/iiaMgcC3RT+kODnw2FvB9wr2XJh6Cc\\\\\\\/qMRKevZTo\\\\\\\/RHbkSbmuSMxTzVHdO0dM7kDilbM9tqLfonng5TE+VKGJkEq8sKkgDD6KqdsGRUS0gnetUzNCRL3Pf3QALCtULt8nW3l6tHvkhbbC0YcfmRurUebXGbz4kMr1x+3sXLZ3w\\\\\\\/ia07XxXkh5D6sNgewbEdkyNSxRKgDBq5ZiSne0mMUXMY\\\\\\\/bcSV9K\\\\\\\/LFtK1viyKWLGsMBuRGa7z3XQj5H84XbUO+9B6S3rm\\\\\\\/G9FtCO6GIQxWtB4sOEzqyN2e2PcoUFBxB5vnGSVPDO2\\\\\\\/6IZLXcHXCL7xFZdOwWF9edV8kK626Fox+fKs7ndYGb4TdWsIrtX0OH4X89brQ2mf1KKKB9DFg\\\\\\\/lp9PJGx7xjvqNnKFtI5HqZB5hl565qtIoGkuKzEdoFMXktha2+uaQFUx6r\\\\\\\/8EdT1u1\\\\\\\/6q5iqn+h+naQda5\\\\\\\/PrruRyGrkl4lfJYu2mSB2yJllQ4U\\\\\\\/1SuIAIU1ZbrGAIKSBLZwJm1LwOunDNZDrMY\\\\\\\/IZelgzVmDZflYenzyOa1o96jR7gNv+Tdx2p3qokzEgCcoIPojLRcA0V9f59fAjLCjHyAry2SxCljoC6ejgSJCV9csXnejS+yyjBjxDCtNLgLcEFMQjehEBpeD8mPxz2Gs+N2xdHyevxpPkdvJStIn0Jxr20ssU94CyQITQzsWsfmHQVkysW\\\\\\\/S011QP50kdv10vACLU5J9Je2iqTTQ2qkcDOO3FqRPeQ6+UVPPFEp2mH4tH+FLpE7UkqQm7DpI\\\\\\\/\\\\\\\/Z+VamIYFjbvnmx1erhxXYajb3g6RIC3jBUDzY\\\\\\\/W+QRyv\\\\\\\/hhSVJZ\\\\\\\/Aelc+1WjuxlN\\\\\\\/tXdgf2rPfsv0t30PJM5SzU7egA4VPbNoU7bOxsbRS++\\\\\\\/oeO3H7JkvGZti\\\\\\\/q0YYFHuZzmKtPmh6CIeliZGrPoHuU9BLS9x7h8JqeLesXakForUHwY3JSz0csn3EFV4X4k6RYcp0Vy8XkK4dLLTffelKNvOnWz+eiz+Gs\\\\\\\/F8vSfFg5OM4J4k\\\\\\\/i9wrwNiHUTuDs\\\\\\\/s75DW\\\\\\\/SKSITj8MZ7qu0eViF8IivwqEhf9ld6v7VfWxcyN3yiwN3aB5CQEE4OI4jK0XF+48pwPr7lkBx\\\\\\\/oxD0UHz7BJkanu\\\\\\\/y\\\\\\\/0+wQfhUNr+rL95PBrw8D3lTUPlyf6A+ZlzEFw23lyFcTFlK8imlJw1WFPh4MJRBGfmplp0goKr81u\\\\\\\/I+orpDWjQLo9pKytfviLzxTQavXlNO\\\\\\\/225ZnsFlu1gfUhXptspRi2iseTHTIdmxK0Fgg5q6ZRcFFx465iOX6bvN1eEHSjEGX6q8BI2GDnvk126H9YShYscH8+zwVKA7EbT4dm8pGRH2XaPeVOmUnxq7fcExFdWgLCUyWgoBzff52PvkhFOnhVtUYSsBtkAKk0elV7N5ymJ2GsHF20zvFCeVu7EiCT27Stl97ulngYUJAQMMsuBLxlEYJ8j4aBApEOai2Eot5LGkfPg02wU42uKSJl37OPMml7m7BpLq9vO4hQHxhI4xV1VEX60B5qR9RdppdxbDg8QKgU6VNsRiTAyE\\\\\\\/Ui\\\\\\\/CyzkNWCJm2BzTj21gqeaK1vfm\\\\\\\/zdGEPUXtDJLVaKxujpfJcbvW+Rw\\\\\\\/LBqRMUtocTYBjm+WuV9P7f1avNu4g9milnhWV+rI9LO5UcIxBvVWkMNM0aoXgeEM1Xk3xN7EbpAq857EX0EOKptP7fBpUL2FpKnCOeTPD4hpK2eJKb\\\\\\\/a+MoipvgbPQHIe8D6C1hGyoWbWjcxv7+DgAVVGp6o84phsEMl77xp+d+xDlsoJcfLz2pTG4so2C\\\\\\\/z6hfMagsb8UNgAXZcI0UtmCfOsBb3uPC1JM3WU8T9qsM5WCJ2Hh0zmdRgSbu5fYueVpIKTAH03NWiR6t1Vk\\\\\\\/SmfUFPC\\\\\\\/3tKirG2rUahtC+fbUsRB3T1ojpjSLNMk2mYFn7hmMr2+rpRk2N7LmFIhy1HHgdDP2c8Da8skde9mEQc6LDQfmXfdDbE7lep+R0jWwqZNtFdsPCljpIA723xkPqjxLzAinw+a60ZWICv9XaEHTcaiEN64keSZC3XLU7MIaJRCFOL3+1Wez4+nVPsbLkS1cISjNay+DI+R8MaX7TqITQn7KSQ4KqDzyJ8yyZjqEJZ9BFSg1ne3QycIlfaXCmCxe9lGfOwX6dlnb\\\\\\\/t9kKVaiujessnxVIazqMsfCg8TJs0NN5CnI46I\\\\\\\/hTA4mpbKj\\\\\\\/B0SaaKRY3PVz0fl1DCwFP5Uol5E81eJ5pM92fsb\\\\\\\/7Hn1ZNwvNq9TZicVXhA3ArI27f2qKd3sIVV2339d\\\\\\\/RJdPSwCMElQ9lcndMdI0Kq1DP8N8JZXFFgb7Zs2AwJeA6ieFVLMw5jPsvbd\\\\\\\/UEUoz\\\\\\\/TfqA6MWeHdMoNPyF4WKoCFr6TpWejXfMp8hy1E2EpEqpFyPoe4f5MRQg3bYlsMFAgMp2N9cHIMDfHaL92s7r3qUpKynhRwQcEuYsBZZNHsjFwsiHM98GlmbKfSsDy9\\\\\\\/crjqn7qnrkhs+mDwxSPGjWObCozZu19QC2ftMBtgFwGF8d6ZUevJNnuJJ\\\\\\\/\\\\\\\/OG3jEfeDmHOwEFCIbmOFXG91QLATqx5FuQQX9vSvbjQhcsOinmk5BLbeL0JdiksVZvKQotGn4wW+DGYiekKIHd+4q+j3H15eop8WXRxLLXGBBTtr8toT7RYmwt\\\\\\\/yRG12yGIO4syxm4R0dBcFdqKyMI2Xst2cR+Y4n5boANxqppqLVVVBlik2R6dNcK2G+218fg9zPKbVBE+0QLLxjwV1r79HR1HEpMPHJkpil+xy1GUt54GZVc+YLm0d0sdDdbt7vdZIGmwrzvv45gMiVN64mxKUCeyn1SznVKqsjc1PPitcyZEUNTzz+9gZ7+1uVDED2eQjhXBuPEkHOTizKhWT4jk780F571PBn+r7X4OWUWh5RbgrXmuDFJwQBRoP7BJU\\\\\\\/+zxHGaKf+p3AC24j1lw+lDI2UgYyI3ptjpMCuJBNOHNqYdUJNv8M46y6igKQzjbsWx+zgBFjzRtiNSjJist+bQHJt1PRNLUis+7fFW\\\\\\\/rhtnmfayeMVYNVqYzymHSe0zH9KnUhlPYt1Hj2YSgRHT6TmAqDEMB4YNPVfreF6B8UN0w0B2ChH97gwNAfbs\\\\\\\/tDgbHQj1ZyYT+AaMftroRTkYHXBhToFXsPMPLyJQsSGv66\\\\\\\/DLs0\\\\\\\/C5QqY5Z7yKa08N5fkFcscm7eHT+qxgLzHANPlub57zQEHog6TUDxQKiag4wmJT1kfQP9DFndiDfRjEJNpnpBiWuyxrYPwF6bL\\\\\\\/uYTlwshiuBIH0IsmOIL3K8ZkP+CcdJ1VpVs1yROoUCzBMK3wFx4CUHBshyTvJYVYSkhtrOmzrK4Yx6tAittMkeCT+4r1XLUJgeH6fpxsX0HbUnUM8pVUAZnI8O6WY3sAhuHji3iciKkOM5AXTI\\\\\\\/KhCPUFdt5WxTyUi3xrfPh2kTWm7v0UjAeHvTsbIGefGwxl2HnNcgWeC5QPa6FReLj3n8oiW2zRE1KIp6SMKEr41jR1\\\\\\\/aCMIC8vro0ui1SzXAWOT+ivv7kaInfi1+rZsjVb1UBkQHX61Fb\\\\\\\/uTjHJv9xkSKG3l5YlOTEmIZuC5mz9CfnGOwEp2QKEUdNYhHF+piIocLyB67e1HrETbNg8VeEXVGmK2VFtjgSMpg75\\\\\\\/twTAm+HSPTMqPIHXl2QgIvxWLz4Tcz351y11ew\\\\\\\/paqOGcU5GZkW1cWjp7IZu1tNUQmrEJKodgRQXwaS9zQq2zerd3X6lJXfMRVBCSOKvLOgnRukBS0serUo2qysl7vsAwR3HStakO+Znj3A8Js10RMgIiIpBzbuyiWBs42t3nN\\\\\\\/6GJGS4uRgGjJxz0dPjesvQ75RyuWNCCNTXJ7VOzoC\\\\\\\/p3PhlNw2sgIMuTDIl1lKgdIpNLJkJP91rpF9XeNsAATPZLuIsFFZ4bZOssA\\\\\\\/Y2zwdvHwXgG5eLRwjNcEFzBxD1FrxhQo6EQS4K67Djn36foMtawi1Wl4GFHQa\\\\\\\/u4balPa7xnZp+aXYdQrTM4G8ax+HheHu+todcaN8Q4nJT\\\\\\\/Oni2vmM\\\\\\\/sylF\\\\\\\/Ie\\\\\\\/gJ5xU31U7oWZxQoE8Bpd5mDBehGbt8pL1p5weTd6tZGRWtFLF6EhfcU0QrKh1y7KX85KIJQTxFYdEEBdI5gMC8OHVFsFOiZIRhr5jPWFR3KvyMS\\\\\\\/RFSLF13R8XMUjpM6RtaaBEL73NiieN78G0DFcyLHLDhbHP9w0Gkl9S83U4NAVSCpkZt4fEnhNvkJ1e5f8o\\\\\\\/hg8Cou633Ctp\\\\\\\/DWgibBfdGKF3MPTLhYPykPek4ZhDNKfyYbd95sTyujvUgsUDkYf3l9sQWpV1ZFG2bjlnMkwwOBh2avZZ9m1GvoayXFmRFQ2hPqMEWVahFMtZaUw5G3qITDemuhAqoVYwyCHClahLHYorZEcxkOYCPzIIPJIxgked00ha9nFq5et2lF5\\\\\\\/lXcGQdaftCUEtfQTovMu33D72w5uBOsHJ9+8eM3r6myiHSORC9HrF4PfXkWILCiPoAiu9cwMPT6dcFzY9dxNDEjb3W30xpw7JDMRgkJx8fZino+wpu4CHR9iWBgZkOuPZDlDRJOEzbGji8Qf0NNIKDZu\\\\\\\/QYAcUaSIHWcPWQ+8U7f7mTD4EVnht8nj8U271Q1LePhRSJuEkFDQ00NRbPy22RTVKd4mi9TYHTk1SH3ewWACk\\\\\\\/PGAyXpZGELKTC8TT4cDZH\\\\\\\/Z27y\\\\\\\/4sm9gyue8HgqTpo8rJd7Ed1TJKM7yT6Ba75QmGF6ZH9d5mcLer\\\\\\\/9QX79J26zfFtpMNXvtgLZI961tC\\\\\\\/iHlV5QjDnlo5apc+zd+p5BVkb1IiPSwp\\\\\\\/MHN0G6qWx61b49yHbGKplsxmJM1SvC0cYMvRtef4W+Q13rXPzzozn7j2TRw11ct7soAVaHtKQyhHt\\\\\\\/pP2tNiykBS1sXUoBpGKHtPSn2zyhTSWYkZM19NEpQ\\\\\\\/tvFxAEmM3CbItEo\\\\\\\/mIj+Y\\\\\\\/gmwNMMFNTEM6DXN1+iAJbCaeofOW156K3xe6GTQ\\\\\\\/5s707BNICc8HM3f+7eZGeIN9Dg5JqiPBMHe0fm7jG0qDHHss6dEi\\\\\\\/xOEqnCkQeO4Tffvvpfc1SXaC3em5APYfa0E3zDvJpjfEhTuCbzuZ1Vgj5om6kb\\\\\\\/xrhH+SyPbeQ9RM2jJIt\\\\\\\/TZsFJyfum2h9Vp6pzbniTxsFxJLlse7HOHeW8i8EGHLZNU4ZXgfpqQVZoeA0+lZV4jhmoInAx+Dv8SuMpKjgyT0HDN\\\\\\\/LWzw8pgkyLEgRzNAZ\\\\\\\/AXPEHK7SUI4lYzpDSzWXzSvfmlPsu08Ot9Njuot3\\\\\\\/CM8T2c2CCRuH2\\\\\\\/9Jz4HZDJepv3P6K5THpqFcvPDj9K\\\\\\\/f1m0r3eahWiGC3h5WfQ2e0yvDHuAwxH+ZGNgKBoUf67pmOFGd2BRfGpg7bUEbLWUmQyiOW5yFb0SD8scVC1DNrR1n8EoApV1880gFnRoKeXCb3z9FuVuWO8lzL0k9tXbvA6fPb204rvID8gLRr3LwPZDTsEivFKDfAGJrhTn\\\\\\\/n9\\\\\\\/K6KDe6GhK9ZMqCsH9ESa0+tnZ19jMWCbI4lr93m4gmZgpfEesSoH1hDmplth6FvA0RIRB8sVnxkbpgF\\\\\\\/Vdz5KQggpQw732LcIvR2OWdXh\\\\\\\/KnB90T86HpyasJo2i+FufKqZ3Ojg+DHWPZ2JzwPk+eU\\\\\\\/kp33aVbC0a16IUgyxClQRJeU5I+t7au3odYCcNgR9K7KVEP0x0QfVdI80BvP8LGOauL8Qh+Xr+VQAOxb6oUrkB1ufzgwPMr0GAqR6rOMW5QV+J9Mahb6iAh0XifP1WxiPCEPg0Ygyz+zSVekPu+yF6rjNEgdnig87kOGeLwHHgPB845TfuTxdXYCGH8tZarCTnCN85QX0e\\\\\\\/bSlXwiOLWMiQV4Lj+Eoa8y4gmCFu\\\\\\\/prdX2RZmMm+jXOwbC46dezyDhnx3ugiGovSfMKedOFJUnYSN8ocj0hgcePodR9qEYW1Fd0jmDtzJxqhCy7Dn7WM3EmU4pyOOwfxkIHzF71hKGvGR+UGEV7umz6MOtN2pa5r1gP9uBA0VCakeftILbIXBWUUh\\\\\\\/01f1nb386SrT3YQ++aiDMCCBoWYA0JTI0B9Rzt0Srmmzh85MxhdLCbjG567UCCnmt5mgxoehl9a0+tes4AbikZ6cEY27Vy9++QN9TwRkCBWhiMluC7T7NHrAzHpI3dNAZEQxTem\\\\\\\/HU2CsBF7G8IfBjsQn3cXhQvlXyBNM32Gkh\\\\\\\/rt0YmXveYzcmxR1FE1kYcNUXJOpJcEfwAo934zgkheFqsW\\\\\\\/9XzXX6quE2l0Mv0UJF+o4RFluwgo9rK4UrGIgeyZ5en7S9s83JjJ8Wsn9lAqEMUwtSzfvaTb0++XGxAutx+p7Ae6uPpKN8WMUVGxuX7pSz5w2Oxp59qGndCjMLYIghv1yHkn2yuqpNSslh1WC6nRuWqb1ghHAwkF\\\\\\\/7zEndOUZPC5tICEUncRJcc9UMtWRdI1iVfzT6O8+rERitsksSLAb\\\\\\\/vHj0jVy8RpUoR+RFjO5\\\\\\\/wPnwBXiX\\\\\\\/MxYSL3M\\\\\\\/XqKIcEBse\\\\\\\/mFSVlaaXU1HTQWy5x6oOVKWH+DCK3AzjjUw80Un6oRTFw434qaCY\\\\\\\/kAANinOvpb+bWslMwq5Mfpk\\\\\\\/0X8TAKnMpzG\\\\\\\/sqCy13YKegtwKEgWRuVw2V35fOG2HyuJytKxydCbkv7wB36D8Sp6u4\\\\\\\/Zwbal4Tqrf4sQKh3r9gZKgZgnngbyZ9e9dIZzghAEgAhELLAZ8JK7cyxt06XHCEZAXzyBAUxKNNELijeeQ1o+ChWyQDKsmfP5rzXLBElJUVnXvOQhnqqLj6POZ39czXKn9rWjF+nYcn1RmErhdiG3HNRu5ZNlDq034\\\\\\\/EVJ1zR3oEDy9Y\\\\\\\/mGXr\\\\\\\/y6\\\\\\\/LQAh1kQF24zuuix8JTNPWg76sNcPTf9mv8fl+M0Uh6oSXvNCcu\\\\\\\/lw3ZT+3JI+ZDwp2+I9I0OBj193+eQOe\\\\\\\/Ma9gnGn0jcD1uwsZ1cjugdgwvdPTS2tb01fq966TqUNU4gV7cpmHXipPQ31IdjllJ3Tsmzm85QDZeSMltR5j2MNUkVplASbwJuxC+lKSoCfUsDU9OMwWbjNSMyt8IYqwAQiKITnOjuRg2BiTwhblPU\\\\\\\/moukj888bUYy6aFeqCfrtbW8sdQ\\\\\\\/Ic9Nc\\\\\\\/YZnA8\\\\\\\/oWvs9eNNXD4TyouUj1gx1dsVs0rbs16EcPYCJLujkRg81On247u3FSTJCawvexPyFfLoVZLPgzvXtoAXCZY7IwpsLe+m8VCQrA7tFfwpyyZ6dkTGnASTWmOUzSANkYjrFs6c8ILbsJqeippKjc60c\\\\\\\/xYKstmQ31H7Hn6nOmBXHAQDkD\\\\\\\/xsQVJSVrA0hJNKn2E+UuvbEft8k9C3buelQM4q0HfLxYyzNUPuUskvyJviZmzs335zMgKGeuGSZDC9Uy3X1ASoIv3ZMPv\\\\\\\/OUOFiEa2wSP1IfAYYtfLKFMYtDqxv3AACmW6ZJUrgdpPN1fNJRiyrg74\\\\\\\/vHW4vBUP7QlSSco1IN3PcGXoG9G51scwKReOMc67wECLz3GM0153tKgNnFCEVqXiSYvLQlhtSrFRzqi8nzLGRfIlx4PloCV\\\\\\\/c+r8w\\\\\\\/cjdj\\\\\\\/DMcNzsLnBBYIBZkxr4FIESruy4kXGor3brJ49VN0pTjtvAy4rFcXENdGnNqoCHyrdUzFnoCEeY+2vwVq0XOII1maBf2lrq9M5QXT1abogI5xJyCDlOzJKHVO33EWgV9e3xt3KMGGx5sm2hF58cU9qfHG++CTvHjnXep3vkIknRTdjr0RiHYjeb7+wHGrGnY4OxlozRX5IyzvnxT60u3hST51pwAddkD6Mn6hJkdTSiOkfXDvgMDzEkPm8yetthL6Q6+VJsus7wD4OR0KkCy8k76k3TMZdU0WMaU\\\\\\\/9CXwrFHVKbI\\\\\\\/sGTbptqOov0zUL0uoQs+A7fAlR5V3zoS+A5W6Xh5Lz7Z3nTdhsAeu81VaKL5UWgB2qQ14WeORJRuqdezeFaq+\\\\\\\/Sh8Nya117QbTzlfjJZhRYFndHQLRy+Cs0n8RJNgCRUitFcZHwaKd0n4fWdnvoJaW0imnOSlt6RLF953WPQ1gQn3cnkKbUZadxnlPFZodYJgdsWcs40UFPSk7FJr2znIdsZdb5CiBUI+I9pW5E0LUrcmADz8u\\\\\\\/OiglU0\\\\\\\/zkoIDkNBkQxHRTaqMWEq43sghUGXmRjPpb7ELhgJcls5poW62Sxclejp0JAdt0s+3KZcqRKVCD3tA9LY6hatQ\\\\\\\/IfTwnobE0boZJIkCOYFIExWKWr7zvPbHkrrq21D2U1MgKnileTalYMPpSfQOpK\\\\\\\/1+++YJ52n8dUZVLuUQJbYomT5za2zZHuqNS1i0CujvPGzaqmr+EnAVd5hdjNAUUE7GvbuBhIvAmugrIx1oDG2Bad6ewLItbflgEbkOhhqdqeKj3Zb46RUgbKZkMrA68kKJcOBwC4a6jDT3LtHgA\\\\\\\/ot3knJRdwS\\\\\\\/+wgFKUhrJt0UZkb31gBd57LNZzDNkE05nLpP2bcog8kio253+cEnQr4JC9MtcH+n8YrtQkigaRUXxhXHyd553Zw9hTqxfdpPo\\\\\\\/BkaEpLMnO0pWtzwDnOODdaYchscI8Ul52kqusEyzK+x92nyaU4cd99B25ahrOTkNTgDYsnofVnmk+FGEe0Fxme0qiNvYkprRz4qTmX6osHrSi8eh8ZhkrTAtA9jNIum8d72NvP6LIIWQExf6TxRmQ2tQX\\\\\\\/OIWWks1GGbV1+vqajBA6t4u+Ei5gzNkio3LdOB3y6QkDICVHDtr9m7yfHv0QTBRZapCUZlynhsf6uypyryl18tEknfXB86sfAyUIB\\\\\\\/iFV36uIZfBSB\\\\\\\/Cfy2EibLuHQwwNXBAQSSHA\\\\\\\/YIF0VjDTv0lf8BszpTIGcci5S5NFBShcNNux5KkfONF7sGSYGjpW3aCs6g+Jji+vQS7NCzpOgeZzTKgW2FdhlP4wsZ+kZQNapnAqKvHOJWTR4H9oAucefO7JjM3hue0nFA8nci4GKNRmUAiuox4IA63P6Vzqyk9yFGhUALmZpOw0xKtoLcVcROxScIv+FGEW+DFBDf4UCuEF0AyYvhk2QAFiMhqfpeTaqkI9qIfcBZrvZbd2vAaQ7nPU0yOX+FSAFoD0rqj2u3Pz780WFqvh4Adk\\\\\\\/1Fz8N+jwABkLV7Wt8s7aIMDtG6WQxmmoqsh+gWeZLagSBIkI7fQ0HRhN2VaSGtTdVhDfctz8Un39Lxrz+W1O\\\\\\\/gVmc3M6WulVYl7N6E8ROBV3OhQq5dm704YtWnWmdbzdhip\\\\\\\/oBNR6dBVaPjZsi912Vkw2Yh\\\\\\\/dBLziIOuzXzSZQ6\\\\\\\/WR+F3otMbOkjtbtCrFK2KWiegzTAIIS+5wAR3AGADlnqnnQ9lfHmgI6NKJhtch0+Qv8HcspgV3rXEb49zBceNIKCYWvvH6tCMmbuxCpfm7nvJembIptklS7QDRBaVIqnYeB8xZbse9x9SuTntlxVJG4rPXPm3p9PZMftTKGhdgRwzeOgFytVEs+i7qG1TbIRW4pRs5WwLMu2he8Pu4b9WKKS12SaK9Z1pRd3VYlKS\\\\\\\/TaYxAZWGQWF4qMSbohZKUYStBp3Up0QZXJWWmIFEqUoun7RSrjVrxle\\\\\\\/+fYFVoDzOYr1EFrV2aXve3bmdfO6Oo99Kh38113GH+6bfwt1nnnYDygh0L3JK\\\\\\\/nJAa1Jfsn8wdD+2aFCCwxHm6CEIMxaMjI65jymgBpfrNplWEVH8MR+4KbkOmAuXDc+4G4FWRGKZfHQhNcZxGVeW7JyeamjmzJ7p6N2iVwMjiwIX0wpeROvv0+mYXOhsKtUjSFBnzwvapsJt2s3ykMcdo8qbZhl13cEar0gP\\\\\\\/whYatqRX4+3cql0stj7lh6s41SuUXrmOsxj2hNEDH6xnL6UIhR2obIW79SopqXZpFoYjZ29mL2L12V5wWFeyPRek7mQdR7NirrKFsdNdeT4X\\\\\\\/ZXXGEsD4cq2HJj96AChoo7VY2B0xKV4kGOrVEbh47+iCjliqpmJPRu3Zfm48Tp3AJD9KAvc\\\\\\\/7QUpWzucuAEUJIT0wkaYKMFEn4ealMhSpedPR32EigE7FGaeQ6wxciY2QzQs1zpz6SWzjBmnovfwH\\\\\\\/eopQbfzhLEiyCgAq1fWjk8PbhxQu4+8apXUMEej6YLzwPRSAAP+yqb4VFUUw+iz6JrUIeia7VFkCAFxYB3+5CwYt3UBqaN0fkLkkKhgI481n0JQTOZteCe6A6JmSHlHRmpm0ZILxQCm9VEUZj3hgVUtq925ypH4G+GqYpyBiyLQdqCL9k3Sqn\\\\\\\/yidyiuV0K5NuRBLMoYOGA3RvRiNDrvmUFwlzOs3xzB22n5Ar+hDekCVvia5T3EOn9759ThsSgt1yV19AcWaU7mJDG4WIolJPiWfYdQIS9rkuBz2ViFo\\\\\\\/Wh3+yl5Z93ksxbYkajQ3P2QpkGFxTicYLImr6wsfMFu2Mt0IrHYIbyu07toJbUyDFowOfANuyHzmN1X8jx46q7Cme4NVpPtuE8rm2lmNClxEnGVjjsvMW\\\\\\\/eWedhdulviitDH8wX8Fmm79B+NuwzMxAzTM75c3APZj2rs+KSpk+Ig6ANbfwCQxfRXT6GGD3U3FxN5EzZ17Yqw8yExVAbqYktohJ7ir\\\\\\\/phpKQ7loRXGMGtgf2nL5Y+H524p\\\\\\\/ajCkuVABHyevxCzCrwxqwB3819r3CJEUTKuugKdKes9\\\\\\\/52VrII8MRHC4obANVQZobFZnJkD\\\\\\\/uhedsLcPqggCEdigBH6YkhFbY0huRRLevjk0jHG4SNe3KRTT3OBfs2WjiiKmIYj6Q9S9UUwzrliV7\\\\\\\/3C9R\\\\\\\/L5BOAjGfJZiimxm5OMTk\\\\\\\/EjSyuRJObnGmXqNWnRdO2gfJAKSCdi\\\\\\\/A1CJ6V4zPwtdBUDUEVXbk3pP3GFj5xnGzAnTXNBmqfmVmxPEA7B6Xvimy52n5ZqQMAYYPeipaP3VEfxts3qWzNOPwFAUpjGFOLv8nnxMsOxqG6gG8xqJg8xckYwDZbAE3PDjL1ca3mfeHqDY1vc6TQFds7l33coMBZQr8wX9O1cPvvNbWio9muoKEc7LOvEIIoA9I8wpHHnxiFYinO0L6Q424hP4UN\\\\\\\/6Jky6UUC3sZ0OPJbx5AbY93UKvt+mRP2T+Ic0TW4fChWTMtpHeYg9nZIIqxBbqkZoSupS4UV5C9Xz+WPvVAuTEtEPVAAibXiUV1Vyr8wZRV8c8CQT3ubwl3Lce7b4dHFG\\\\\\\/VNRp0+znqNoDR1pcVaNyag6qTsn3mAh8GlJxlKCmLUNKo\\\\\\\/1jWI54wYsa1YNzvjdxFjhv8IsJ4dtgz+oEtSTxFJDjQzFPkrl\\\\\\\/FzKzAI44D7IVZiaFeLm1jp9gU99ocf3\\\\\\\/affWjb+6xFUPtsJQgg2F273Wj8XHLbFDj8RPePWJ\\\\\\\/CHlh5cKHVpduxJ6Xj8St2QVw8+GgtP+7dOvsuNrp7ocLfI4\\\\\\\/UD9N+br16XJ1cg53bby9D53cUaH8fxFVizb44aFs733ZhXqfswvnWl+eBg+of4LGf5nGA4UcG81YND4ivVyBspKltH6MqcAg92UrY6VEiqsENAS5IHLmAg5wsGL2YhQqRPXQy1IM\\\\\\\/g+Ef7U2AQhp2LuAJloXbipUqyiYnUayI4Bm5POYnY33SFSptmKG\\\\\\\/eLfSL9vcjOyFZVdRLkkT5sWe7Fnq5W6wTbsANy7b3nnjfu3pXUoRtj0p5gvd6eBxOJe5DIdVhL19BO\\\\\\\/hIZww6BBUkoAMS2D1T1U8N2ELHuTUbSmSmQb+FL90jAVhVx6kL96XkdCYPW8DjNUq4Rbf1IXPQnmk54CO+61TqJF8QXnvpfsmoHQLPs4IifqHTELjpmpBvp+h74PGnH3m5mc1zBD+FSptNHFhVCXbVD9EttH95FHxx4EUfi62KpxV4Hzcs+U5jWFgkrRhM9wVyMpApCqgvxVW+U28CQO8xQAwK9+YxGXgkvNCb6m\\\\\\\/Z\\\\\\\/pV2585tYPOYIbuyofeVFmg\\\\\\\/9boGkPgwAcLhvsHDL7uwjshCcplbix0jJB2UtB1eTq8O0C3kQQbFaXsAOjV6LbMvLI8CcxPD9e37jJCxL\\\\\\\/5L8WK2lvVjW8rSn7IBVwiPSeZiu7wY5\\\\\\\/V0Q8bvTYnSIeCwNBv03Vb11X16Im12HkUbOmVYC45SYvLVrIRNHyr0Eh0C9vGqG4YgcjQht13eFV5qfB8Ty\\\\\\\/nFlDEqaaiLpfmGpqZMNERkaQmSKyRFFTkAvx8WmzdJ95Q6vwtNvepKVsnpus0oakOHe\\\\\\\/TCzG29EEVDEvTYmOMygA2FAqT8II9fKwpzcyrskuCyk8EJC1m17XyDAmGStYJ37vGk80N0sTQjUfe92apKE9r\\\\\\\/iNov8TKyxrPPMJwSh5pDFsqxCeI2WHMCQdJ3qt3\\\\\\\/yg\\\\\\\/OH1zINT6GM5MYtBTIYwSw6F3M2HVWrzHbn2hypfEcREOL\\\\\\\/iy1jWy6sQq2XJDyEt9Wv2cD27dVtz5M1r4ISGgY29Qij6J4YFv2SM3RUePaw5QAH3gKvnNqLRRk65J1f9aKtkXINRmx6+SjCy0kl\\\\\\\/4HJazvwCZiJK4PLuI7IXG+W4hgWT8Hvh5JVT0nXhwx\\\\\\\/9TG5MkP43Nmc0fWIYV6xRXoez678XiOS+eUXsoJa2\\\\\\\/DIKl4RPRsvRX+m90hlUkU92CYSAQxKtz6GnYDMy7US25ytPs5qoEZQWWH3+pvvNjPx4pjygsQolz5iZ8WCRBQa86VhTQtwgGpltOmpOQs5\\\\\\\/SyAFuu8EmrwbHDHWVk+GPYHrS4LzAoYNAXn9oYjIBUgV8WEcUTwV+KZMiPpq+GDy3U4r2pt6vKIf0TtrLplsnsMl3u52VK0qkWQR7J5QF+H3Ggc7hhLC1QrpeJAv9Id3clViEEPD+iiqIMMJvztFHJ5w02tIwBvA8G1TTv8mOelIi0ZiCpbBPZKbRHQnSMg1oVF5wCfisi22bAY4xwsYJ+PnjtnYliiYxa7V0YAs24wwE6hBS+qdZ\\\\\\\/\\\\\\\/e2Qixyr14lhlmRuGrhgWZdWxD+rOh0Ig4DJmJZOhUbPUCr+0PDkeQKMtn8jUd9aMCjtqgtzw3ZGcjQNwbWN+5UYp+XjiFozsgqRiIsV1Rj9AjJFahCOwmiBR6qJh+k51nkim\\\\\\\/B6El9YMYg2v2YmEt7iRPgcYwurR1zJIjElKuq2DwNIYJu47qJpEuOn5mdt1fQLCMlNms2lqymsL+l7oPf90BaBW5HINUNfSTnkgBQnEFMYjagJfKTWx9itY\\\\\\\/vYRGzEav+DUXAJXTHbp5Eq+9wyVQS5OataGRyWdLkIxHXahocjNj4EP82Ar0zyQjWXWXhWE8KzWBoQSc\\\\\\\/i44hH7wplPqOWLLzS0SnNxI2PI7QOTRJkaOJPTQxrxMCLVKwFrZlXzMJ0kQZhTWX08gN1zQKr2ig515Jskx8twWEi+J59d5IKzp0Q59rlUrwKIx01r\\\\\\\/0jz5CNf+gShGaGdr\\\\\\\/3jOEy0qYIFbC1UFm\\\\\\\/zn2x97zGQ7f2dzfbSSE4I0\\\\\\\/IJrxmOn3DNypK66422\\\\\\\/3aDGG1L+odEqRRgOyBggEcyh\\\\\\\/SlAUBIt2EEeSxGYEbuppn5J9ik2jb6IN4uUL7oyl3t0FST7r8BiX0wsY7AemyQQtCKh3NruO\\\\\\\/+wGLqedc3kZ7h8B66Z9K9+ekq1ugS5GSpU1Yt7fC+M5oLq1C7NsQYvqIc3RrW+6IdC18GGh7eiqY4fTup+mtFJR37mb5C4\\\\\\\/cSF5dJrYljykLuO\\\\\\\/FKYNA\\\\\\\/Lx43oeC\\\\\\\/WLNJY+ezms7MYhDFUz5tbm3R\\\\\\\/T4dgd2P0VAXZ1V8vRtj7hRo\\\\\\\/cQ+zvIP6yjzjMyMo2+s8DHFgO95nhk9cIReiP7MZFD00buhXQFiMJs0Ucgk9IlZB2kAellus3r6LJqWYradRablQ2XBWxjsj822W5WyZsRaEECwGKS5zjGKBuAGBWmOLFFSN\\\\\\\/hjm+r40Bu5ga\\\\\\\/OaowIy0Xp98KvwfvfmJ6DGmLg0o7X0xd1Ankc1PpEOUgomJIoYUOsyvi1fqjOftOxnO6J2F4bdwPTTwYjWMP2cRcTCO2wSfO4XNZRiLyEBx0zuabP3gVXD1qPdDNapmFK1hXtZJ8KmvBJqulfbwHNW6Q1q6+uaFj+pcSduoqC++JihzqDjLgylF5w63qVJsizuyZ44f7K0FXnXGZrWpQSw5uA7\\\\\\\/o52hJ1kGOzPR9qT5S+c+tWVsd\\\\\\\/jR8mkM\\\\\\\/youxW2xIxF7s206jHcaz6cXlZgOLDILPtARLh0czMW\\\\\\\/eqln7QX1N62icUZRDRH+gIRGzcIejysgXjzgDP\\\\\\\/Qlq4AEOtYB9a0z8COURGzV6mrBqhuZu3H5+sC5er+bZldU\\\\\\\/LCUFDBf160AM30ABTD6oFNYADfddUEmjxvSx0GoXo7XqAqYIi+E7i\\\\\\\/o7cghArpUFpkCu5S6+7P\\\\\\\/T6Qw86cixU0D0qxOBrh1SCGmJSnOyp0aoQXaWpJPR85anN3j0TTIgimJKrhDbR70YIUq+o4hUoK9\\\\\\\/iCnaCQFrEzqs1fNyiGVFY69V\\\\\\\/LdmnnwCGm2FiSJOpNc3\\\\\\\/cDYFyvngnI4pqDXGAgoXyYjDrZE1RABhgf+SrRjmhN2X8r6M8ylnVfaOf\\\\\\\/eo0Nf+O9JZfCJqF3X2qVUd8ZbsynsyT68dqFb\\\\\\\/CE1tTTFiNUa44+nbR3zmvMz3O6XBM1WhmoAZWWYnehA9ZicauMG9MuxdObvVFaxsXXoNVJEjIONrADrvDma3FDIqg\\\\\\\/jovefTzwjIsMH4Xv8t5CZX3YeY6ylglRF5acAFYXJodXTmvtIPPrNxhv3sqix71+m4If5hHWU6I9tjRNEOt30YC7+S0hPIW2COmI9VZ49cMK4gGL9eE0+huECyTYRny9ytzeFX3qERKFo1EW+LNosrmmsotO0BFWSDoDhBOQJgfF5\\\\\\\/++Lhjh3OIH4LDdeJ71\\\\\\\/pzm6g8fsIEMkmDogOrjNDyPFbBjxKFZfrZnVVVhjQonTDDPsOxba5oFUZzDSDP87hOaKUd+Q016wSZ\\\\\\\/xlbwWFZzCEAYyBvN3Uu8kVPRfF7PmjLQnzQ8Awb+jjr4GtT84+88J7mddMvC38Isu+mgKCzkOMHutKbG\\\\\\\/SFauxxLzye\\\\\\\/tqp56OBDWsZ9RyInn3bTvgyemeFWJinzvWj\\\\\\\/UfpZ7s8MKW3DP8RXuS0WpLlERYsMwGJK9AJrxYCHRZKJkekiMPidZ2HDnikvEatZk2ZO8\\\\\\\/rgF+fLM8eICn0yC1qDS66Af8N5j5wCaL+ds8GawEDxmCQFkJ5RKtg52SA\\\\\\\/HjXOhZeXhE03dBEi5MPehIbihHlv0qa\\\\\\\/nN4ep8Lt+ok1meFNzBgb5kphJOmORLFavAsIdalchriw8KGLe9GpmTIbBnlnR+XsGN7iRAuHxMeZlrF0I9tAe213J5NwdWStHscLp33G9Dfuw1wO5ZlZO8UDMEv8uC6QEr8Ag5mQ+7DJjtalLBxJBXnGvVhTQNlGvDsNoASPgQam1gFYshAgtEMuh0jEgdYzQvkfMF9eGdEgSqrkJEwUDGCDmJmUqGsF3C5mis2ln8PGHtmKZddJa7yxqKHF4jXBS2nDui8YUPq5BDLXQcRCbjsSf1Los9HNT\\\\\\\/RVF9flr5xke3znJhqsA1wBajxsxWQk3rGj09XfweE2LCNfSJYT+pX6LoU18SAh12jPSvqqrJiCHFUcdZezIBTfltdP7iABqBMO1ZwIvZrCO3EgMI87cmRhyC10Df9rZfrRJ3c6SW1p0ZKP2tcTuAOd6EZ\\\\\\\/2Cqfw2oGNA6rmZe37oI\\\\\\\/5+ur7AuBccshCj4AsGy4s+RwcuPbAy0KOzxbVWEaUD2QUsnAd8gKQNUjcXN45lK77QhFRdHTuYoZH\\\\\\\/wunmHzGGdy\\\\\\\/Ikl1Q29q5sNXFt2UJCsEPVv3iKsguVwwHatKdLY9BErzRA5mfK4er5DdnfiJ15rVBXvJhDT2YcG+AeoNkoaGBpPoOi9PZcE2TdYfpwqtgAqupVNF1pfbuNw6Xvn03hoedfnMEybYBl8pcPCpK8ngSdLI+s3Sj9t6vVCLsfK8snzHGoPy4G31Q0+2pNsMFyYven6Lk059DxkQyAK44QF31JYjZAhr7\\\\\\\/EC+X9gmiNTwVTin3kIkDWWt0\\\\\\\/zS+caruCuMGiFt\\\\\\\/5abFkEVvd\\\\\\\/5eKuGpOW5rQC6NjlqnZ9hj7Qgffev4+Nrj0Q1HTExPC+66AMLTO0X0GA+hpWB6q7wASp72nbXLzrurQLFV3w572CE5OUtMabXlmyC\\\\\\\/EKoohe\\\\\\\/LYNuocJqymlhSIh28vjsME2Ql2tewgfZE+4L2MVDf3AyZF+04txAIH9xGbxMgQntuF\\\\\\\/7z3auXUd5JRE+XTcjShrEf+tC3eZNgvcBZX7Xxnk5yqYTse5CZkb7Mdouw7Owzl0HtibBhRZI5o4p\\\\\\\/4zh2FiYfkAz4faaxrQNNlx+9iyfObcxXMk5UMrV+LNR05qF+W8JCUUc\\\\\\\/pEumqDas2e6lJ6ErfItqlLtUAq0awg5Q42nIKoHZkdYKx+zZW\\\\\\\/y70eRu\\\\\\\/Z0vkkqCmCeQJZ06JRQoE450RO1RamcZBVSawjvyUTZdr3UisGtBezoL1MyAJFJZE1wFr4xkwxn9olexPNczjpg8MS+xpx4w6HJsOUeza2wn3A13sqk8iqepx46Swmg77vmwdrN4D\\\\\\\/h5Y\\\\\\\/vQKmkiRgu42lnRrr3yGHwPDQIzM76EvmACsO+1JD2WgtuYmLOt3s0EWVoxYlSFPAhjbuUZoGghiSwzdzsyCij319PjSu4FiI354I1Un4q02ydu33ti+TpMtemouo8TAi3mlsM+1YgI3rbKJzfTFimVgZ8IXgrkA8l3lhWKmKyG80eusx4YRuWr+4gQ7aah810XURSIy0qAathODKewlA2UWWQe85hxCLAP5q+JHzm1lh9q6waRoca+k+IoX7SeRcYIAyOHpa42XXF4t\\\\\\\/XbdeqjkA0fli9GBq4pJO7yFZp1h+jt1ATiPZdtrTv0uUiEc1kFAsv6pvxiMqZi+QjgNqOskefLruSAg2AZgeahcFU2e\\\\\\\/9i3Bdpm0ctRrYmbcifyTL8YtN8+csSHlVvmEUg28gR68VeeTfnaWvHoA+q1TkMhY8LjILHstCKkjE+OuKqmxptiP594toOQG+aio5HIz9hysS059KdYbAIvTIStcRkhoE4WpUNsv50rnY4jPccWNwg+0X+1JG7OIKjkPo2PcN+vnbvJrQGTzITeOGH1x3Kpno7ZlggbqLnjWNFTocA5KmlWveAa6z3EPnhRegM+NGbHiFPATemHbl8VkIvvw0HjP4tCYGgPIiJnWcL8aiLtELryjIrTvL1kbvDZai\\\\\\\/klEsabyEGXz0970N\\\\\\\/b\\\\\\\/y9R2x6a0vbGMMQqkxZAP\\\\\\\/QrDi66eV71Nuzly0m\\\\\\\/tPz0PZN+G06Kj2OEVy8snUwQ2o60hQ+CNfkj94rTZVXHOy4DRrrlPB903zESGhtFLWwAHcpgLRTVUmbpXbxGdSSz8z3cHCiqDAvoBXYmLbtefMLartFoLWrk5E04ytb\\\\\\\/L+MkFT2tlNzcSAHRm+LPmSWJmt\\\\\\\/f+fGN0jA8GeAHPQuADczT6YgABVEkIKSYUmRmG2XzmZhIkIY7jcEgSZRF75qWb3+src8\\\\\\\/QKSLSj5R9bgJHLCMjo0s+Y0CfJZzLDt8UmeyuhtWoYU2BlmiJ2iW5Fvog5Zng9SpVdNVYvhQRwwP6Hvd1F+IP9O42B1f\\\\\\\/qawvhI9iEZrvIK6PJb\\\\\\\/KErv3lmKKcNCUbki+6suR+HtWlNPXfpFQcfsi6Oncsrix3gx4zJesiWykdD24U00qLjDIUiNxGN1encCAM9Ka7fuOXBrheDK2+4yptHsPTglXegWz5CeIHGUea+CBYjGWItnk3Tele8CsGz9OopTSFsDaKc\\\\\\\/H63mmFeLvCzbywf\\\\\\\/Ftz2\\\\\\\/BHQHYDs6WeLdkWNEmEIFsTHMHbrwrYG1csfAysLHBbqzEjSRhPuli\\\\\\\/GeKznXgmv9jmb7FdJzvkF5Uz99BxjuD19tHZFnyuoYKQoe6zwFYkqjHZFm15uhjJ3brJ8+uYYfggVKii4j9ckESJu\\\\\\\/X4TzvE35iciwsVpdUwrxs51lwC+GdbfAj+5EZy\\\\\\\/AimziM9eKZGgZ7Pfuo6YOEDI85poQ3b98U9OeF7JPjdAuq5Vpu172l0F9hfGJb3ksizslFkL86JUX7hsYFSXy7c3g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ApnDss2ooni58s8QOBVOgQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a064b721e1501311782a4776350fb61e0a684728346f11c1990a1bb1c0d6eb96\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RnKiKtMfYhokS2yXARaRi7EjlTHaJQb99nmOl+7EyW7n4pmHG1PbVJQf4R4u6p9Kcm2So2NiAItCf19J0CChUU\\\\\\\/PCIvQ\\\\\\\/2LMKEMFOJUxYWfN6xdxQzq9KepwMvGjTzJTeEItDCaC72gXOQVmhuZ94rP6NUKNxg0cNRLCfEdrHdGOUyuVBDbKOgGmKRmNKjaA5pfDLDwDclhoR6j3w1au6I52dy3nlnELPQHvJRYVE86vAMV\\\\\\\/KURjwHZCQ2P6yrG1aLQlE\\\\\\\/P+2cZoQ5jX2eqORUNVRBX+ik8giOGphVzr5zbgjveOs2Jgv3dHq7fMv+EudXGz9AUD47ceoNYFkJfD7G0m63ll2bMQ9qpkrLUg+y+0jiuoxm\\\\\\\/NXmJLsSOBnhaVBRbxTLPBLi+465ZDignFJhNxp3SMIFKo69bK46rP29HY2B7QHq+pW2BhsdrFeItQMmmlsJ7r8gq+xeaXBeQjG3sD4FOQvbsh1H7upg0L8PgC4rHSjZAEEMDPjhYuXw6TfnGE4vbj37hpJZApgHgyCLoY\\\\\\\/xuHvHT2O9JMbYtMFIC1AhayjXUrG5VjSg8IkXMMkOzGQPCt5dMDAa0F6CqTLaNOevc9CGx01hl+2VSGl6wplRU7Dx32AaUky2J29xwIFw8IDxwW7iL2hnn9NFaOC5JPLcgKs+erZ5oNM9lj\\\\\\\/FDwAKNyvqctI2I2fSWdMA6i58R8TieBsS1ZXIMjoCg07pqyTKeQuajHdDWGWuknJ8uhbJ3x08KwZjAwz9y46TUAsP8Sk9JetjnQhkehdyEW0QWM7Z3JsEeBUO2U+BqCdLMjk1TRqHLVOmrlPjqJtYPUnSynbbbwEWikToXSTYnqY5Ol3bI5fyF5+aSnOmiMCRkXA89OjUcVdjxWgD+tlKZvQHleN0GXWVZH6S+9agj3FPQtNkgeoPCbVAqQLMFQPyp4FZDsCfE4QL+chpOK5P0pt58b6RC06Adjxb7gfUtsExDpN84mtzc+Nb42WKvTI8l25G3eAsfxairaC3ZnYglF1COyfxz75UDnilhwin060gLj\\\\\\\/0+iSNol8V5e5VfcQXYMLwBagXiWiG2R\\\\\\\/WJUSzKKnaZGVTfh3I0M2JQ0YnbGTX0Zg81NqSYxNIK3FGLxJWeXCGZOmSrgyqFpc7PckLcGJzLyHnWeR\\\\\\\/UAtDTuZQMQ0MDbs083i9uc0L0VSkZop7XPBuMKiXhcUkXxiaHGXmWyHeDa0jO67QX1j2V+Q\\\\\\\/b8lWOmxeW\\\\\\\/v2uy8xN892q9OqRg6yRGm\\\\\\\/XJ+Pqu2S7RscMdW8uDfu2yU3kDhSfVHEIGMuc8f0jgTnXVtn6krvJpMcAO+7je9kDAsLDVj0PO9O1P6BpV\\\\\\\/cbw3NvmxjB3KS8OYF1j0mfEG1M2xrgdlFeDXKoGLWtiMcsxuEDji6s2QaPWMppG+eQ9Sy8cOTcU2q+qpWaUB7dqXHNK1HKeZxxrgg3wKAQlcYlJRKTc\\\\\\\/tyyRLuonG9h5HEXYH9pI+vKwHWBhAHmI92nfI19pz+78da9vr5ZfiHVYpOYVYJ0KbljYR93SZAaV8X6lXI5H+lfZFkmwbwmYuaqjDPhXXrm+OC6LtHBYO37Lgbpk8vkikqCaVKt1q1ow81YPjJfCBdPlEWMHEOak61s\\\\\\\/3XOqiVbx9a57u+kBpUWQTPri3+6T2lwvwsPTSUNQQFSKdAC7b+bHoAkSHIQp9osCPt7ZoZlipIlXZ0U3F5KuiA17+IxOAabXCpdqSKKwHG5P1h7\\\\\\\/BAsfIyRe04hRl1nbJIBfQ\\\\\\\/BUZ4ee03N\\\\\\\/s32KJ\\\\\\\/PE4fzXpfH1v71wO1oI7xyqWgvtirkisKUhb\\\\\\\/II9Yyq2LRGIs5sOKlGRevi\\\\\\\/Q87spXpMeSXqODJfUZ1CxndfPenIA1UsnUj8s4KWTfK8fK9Kxbs1NTfklqhyhL\\\\\\\/X3WXD5VcIEiZC0EMQMxdF\\\\\\\/i1jh\\\\\\\/PSEBaxEvE3K9TB30R4lB9tGVlIHIApGpdKgvUCwyBJl9jMEezkalz4sO3A7n61pqvrHa7whcRvkj1g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022TSW5fXht5gxfVADimOEaXg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022706e5daa99d74c60522def558fc012eefc3ffeb6f8a118778fd267d6e798d3e9\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485850 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2533" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rquk6thrcF1gvYqZrrxjaoPW4vU1OcuKo-cHbgtI_i4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2533" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-izbkQwprYTwNti68sS1kUu2y7tn8xp47hp_a2UfcL_M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2519" title="Genetic Dissection of Morphometric Traits Reveals That Phytochrome B Affects Nucleus Size and Heterochromatin Organization in Arabidopsis thaliana" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2545" title="Multidimensional Genetic Analysis of Repeated Seizures in the Hybrid Mouse Diversity Panel Reveals a Novel Epileptogenesis Susceptibility Locus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485850" id="node485850" data-pisa="ggg;7/8/2533" data-pisa-master="ggg;g3.117.043752" data-apath="/ggg/7/8/2533.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Constraints on eQTL Fine Mapping in the Presence of Multisite Local Regulation of Gene Expression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Biao Zeng</span>, <span class="highwire-citation-author" data-delta="1">Luke R. Lloyd-Jones</span>, <span class="highwire-citation-author" data-delta="2">Alexander Holloway</span>, <span class="highwire-citation-author" data-delta="3">Urko M. Marigorta</span>, <span class="highwire-citation-author" data-delta="4">Andres Metspalu</span>, <span class="highwire-citation-author" data-delta="5">Grant W. Montgomery</span>, <span class="highwire-citation-author" data-delta="6">Tonu Esko</span>, <span class="highwire-citation-author" data-delta="7">Kenneth L. Brigham</span>, <span class="highwire-citation-author" data-delta="8">Arshed A. Quyyumi</span>, <span class="highwire-citation-author" data-delta="9">Youssef Idaghdour</span>, <span class="highwire-citation-author" data-delta="10">Jian Yang</span>, <span class="highwire-citation-author" data-delta="11">Peter M. Visscher</span>, <span class="highwire-citation-author" data-delta="12">Joseph E. Powell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-5352-5877" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Greg Gibson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2533-2544;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043752 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485850" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Biao Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences and Center for Integrative Genomics, Georgia Institute of Technology, Atlanta, Georgia 30332</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Biao%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABiao%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Luke R. Lloyd-Jones </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luke%2BR.%2BLloyd-Jones%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lloyd-Jones%20LR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuke%2BR.%2BLloyd-Jones%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexander Holloway </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BHolloway%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holloway%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BHolloway%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Urko M. Marigorta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences and Center for Integrative Genomics, Georgia Institute of Technology, Atlanta, Georgia 30332</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Urko%2BM.%2BMarigorta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marigorta%20UM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUrko%2BM.%2BMarigorta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andres Metspalu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Estonian Genome Center, University of Tartu, 50090, Estonia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andres%2BMetspalu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Metspalu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndres%2BMetspalu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Grant W. Montgomery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grant%2BW.%2BMontgomery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montgomery%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGrant%2BW.%2BMontgomery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Tonu Esko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Estonian Genome Center, University of Tartu, 50090, Estonia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tonu%2BEsko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Esko%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATonu%2BEsko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kenneth L. Brigham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Medicine, Emory University, Atlanta, Georgia 30329</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BL.%2BBrigham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brigham%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenneth%2BL.%2BBrigham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Arshed A. Quyyumi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Medicine, Emory University, Atlanta, Georgia 30329</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arshed%2BA.%2BQuyyumi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Quyyumi%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArshed%2BA.%2BQuyyumi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Youssef Idaghdour </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Biology, New York University Abu Dhabi, Saadiyat Island, United Arab Emirates</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Youssef%2BIdaghdour%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Idaghdour%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoussef%2BIdaghdour%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jian Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJian%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Peter M. Visscher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BM.%2BVisscher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Visscher%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BM.%2BVisscher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Joseph E. Powell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BE.%2BPowell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Powell%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BE.%2BPowell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Greg Gibson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences and Center for Integrative Genomics, Georgia Institute of Technology, Atlanta, Georgia 30332</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Greg%2BGibson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gibson%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGreg%2BGibson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5352-5877" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Greg Gibson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:greg.gibson@biology.gatech.edu" class="" data-icon-position="" data-hide-link-title="0">greg.gibson@biology.gatech.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/8/2533" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485850" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485850/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2533.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485850" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485850/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2533.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485850" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485850/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2533.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485850" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485850/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/8/2533.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485850" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485850/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Expression quantitative trait locus (eQTL) detection has emerged as an important tool for unraveling of the relationship between genetic risk factors and disease or clinical phenotypes. Most studies use single marker linear regression to discover primary signals, followed by sequential conditional modeling to detect secondary genetic variants affecting gene expression. However, this approach assumes that functional variants are sparsely distributed and that close linkage between them has little impact on estimation of their precise location and the magnitude of effects. We describe a series of simulation studies designed to evaluate the impact of linkage disequilibrium (LD) on the fine mapping of causal variants with typical eQTL effect sizes. In the presence of multisite regulation, even though between 80 and 90% of modeled eSNPs associate with normally distributed traits, up to 10% of all secondary signals could be statistical artifacts, and at least 5% but up to one-quarter of credible intervals of SNPs within <em>r</em><sup>2</sup> &gt; 0.8 of the peak may not even include a causal site. The Bayesian methods eCAVIAR and DAP (Deterministic Approximation of Posteriors) provide only modest improvement in resolution. Given the strong empirical evidence that gene expression is commonly regulated by more than one variant, we conclude that the fine mapping of causal variants needs to be adjusted for multisite influences, as conditional estimates can be highly biased by interference among linked sites, but ultimately experimental verification of individual effects is needed. Presumably similar conclusions apply not just to eQTL mapping, but to multisite influences on fine mapping of most types of quantitative trait.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/fine-mapping-0" class="hw-term hw-article-keyword hw-article-keyword-fine-mapping" rel="nofollow">fine mapping</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/multivariable-regression" class="hw-term hw-article-keyword hw-article-keyword-multivariable-regression" rel="nofollow">multivariable regression</a></li><li class="kwd"><a href="/keyword/colocalization" class="hw-term hw-article-keyword hw-article-keyword-colocalization" rel="nofollow">colocalization</a></li><li class="kwd"><a href="/keyword/gene-regulation" class="hw-term hw-article-keyword hw-article-keyword-gene-regulation" rel="nofollow">gene regulation</a></li></ul><p id="p-3">Since it is now recognized that many SNP–trait associations identified by genome-wide association studies (GWAS) can be attributed to effects on gene expression, precise estimation of the location and effect sizes of regulatory polymorphisms has become important for understanding the relationship between genetic and phenotypic variation (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Maurano <em>et al.</em> 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Farh <em>et al.</em> 2015</a>). eQTL analysis and related functional genomic strategies are, thus, now a standard component of genetic fine mapping (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Nicolae <em>et al.</em> 2010</a>). The minimal expectation is that they can identify the gene within a locus that accounts for a GWAS signal, although even this is a far from trivial undertaking (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chung <em>et al.</em> 2014</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pickrell 2014</a>). Many investigators make the stronger assumption that colocalization of eSNP and GWAS signals to a tight LD interval implies the ability to define if not the causal variant, then at least a credible set of SNPs that include the causal site (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Trynka <em>et al.</em> 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gaulton <em>et al.</em> 2015</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kichaev and Pasaniuc 2015</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 2016</a>). Studies across a wide range of organisms including yeast, mice, and several plant species, reviewed by <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albert and Kruglyak (2015)</a> and <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cubillos <em>et al.</em> (2012)</a>, show that individual regulatory substitutions can be experimentally defined and linked to visible phenotypes. Similarly, the <em>SORT1</em> example in humans (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Musunuru <em>et al.</em> 2010</a>) showed how dissection of the path from regulatory variant to tissue-specific expression can define causal influences on (heart) disease. However, this is painstaking work that relies on strong prior statistical or functional prediction of likely credible intervals. The enrichment of chromatin marks such as DNAse hypersensitive sites in the vicinity of eQTL validates the assumption that many credible intervals encompass regulatory SNPs (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">ENCODE Project Consortium 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Roadmap Epigenomics Consortium 2015</a>), but conversely raises the question of why there are so many instances of discordant fine localization (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Huang <em>et al.</em> 2015</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chun <em>et al.</em> 2017</a>); does this reflect biochemistry (regulatory sites do not always map to ENCODE elements), or simply limits to the statistical resolution of association signals (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Udler <em>et al.</em> 2010</a>)?</p><p id="p-4">The general and parsimonious assumption is that functional variants are sparsely distributed, and hence that their precise localization or estimation of effect sizes is not affected by interference due to confounding of statistical signals. However, as GWAS have increased in size, it has become clear that multisite effects are not uncommon. For example, the latest meta-analysis of height suggests that over one-third of the &gt;400 identified loci have multiple independent signals (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wood <em>et al.</em> 2014</a>), and that the expression of a large proportion of genes in lymphocyte cell lines is regulated by two or more locally acting variants (<em>cis</em>-eQTL) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liang <em>et al.</em> 2013</a>). Since LD within a locus can be extensive, the potential for misestimation of eQTL effects due to interference between signals from tightly linked polymorphisms is high. Here, we address this concern through a combination of simulation studies.</p><p id="p-5">Heritability analyses in recent years have shown that, on average, up to half of the variance of phenotypic traits, or of transcript abundance, can be explained by genetic factors, mostly acting in an additive manner (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Powell <em>et al.</em> 2013</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wright <em>et al.</em> 2014</a>). An important difference with visible phenotypic traits is that one or a few SNPs are often found to explain a large proportion of the genetic variance. These typically lie within 1 Mb of the transcript and are regarded as <em>cis</em>-acting regulatory polymorphisms. As shown by <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lloyd-Jones <em>et al.</em> (2017)</a>, 35% of all expressed genes in peripheral blood have narrow sense heritability &gt;0.1, with a median of 0.3. The sentinel <em>cis</em>-eQTL, namely the SNP with the strongest association signal at a locus, typically explains 85% of the locally acting variance, which is two-thirds of that attributed to all detected eQTL, but the majority of the genetic variance is generally actually due to <em>trans</em>-acting polymorphisms of small effect.</p><p id="p-6">The largest blood eQTL study reported to date, assembled from meta-analysis of over 5000 individual Illumina microarray samples (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Westra <em>et al.</em> 2013</a>), reports single site local associations that are genome-wide significant for 6418 genes (44% of those tested) with a 5% false discovery rate. However, the blood eQTL browser only provides single site (unconditional) estimates for all local SNPs at each locus. A more powerful cross-population Bayesian method (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gusev <em>et al.</em> 2014</a>), applied to just 420 lymphocyte cell lines in the Geuvadis dataset (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lappalainen <em>et al.</em> 2013</a>), found a very similar number of genes with evidence for regulation by a local eQTL (eGenes), 14% of which had strong evidence for secondary association signals in a multisite analysis (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wen <em>et al.</em> 2015</a>).</p><p id="p-7">While increased sample size is certain to reveal specific instances of multisite regulation, it also provides the opportunity to more accurately define effect sizes in the presence of multiple sites that have varying degrees of LD. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> illustrates the reasoning for a hypothetical case. Five true eSNP effects are indicated by red lines with increasing effects above the horizontal and decreasing below it, while single site unconditional estimates at common variants across the locus are indicated as black lines. Where two sites in high LD have effects in the opposite direction (green circle), they will either cancel each other out or substantially bias the effect size estimates. Where two sites act in the same direction (blue circle), their effects will tend to be added together, and hence the strongest association will overestimate the effect while the secondary site will be underestimated or not detected. Weaker associations (brown circle) may also go undetected if they are influenced by even low levels of LD. In theory, if the location of the functional sites is known <em>a priori</em>, these difficulties can be resolved by multisite linear regression simultaneously fitting all of the identified SNPs. In practice, the identities of the functional sites are unknown, and exhaustive multisite modeling is impractical, so sequential conditional analyses are used to find secondary, tertiary, and so forth associations that are independent of the primary signal. In the presence of strong LD, this approach is expected to miss independent associations, which will remain confounded with the primary signal.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of multisite regulation of gene expression. Black bars indicate univariate estimates of allelic effects of minor alleles increasing (above the horizontal) or decreasing (below the horizontal) gene expression without conditioning on other sites. Red bars show the actual effects at five single nucleotide polymorphisms in this locus, which has a linkage disequilibrium profile with two large and one small block of elevated linkage disequilibrium (pink squares). Dotted horizontal lines indicate a statistical significance threshold, which is only exceeded in the univariate modeling by the two left-hand sites (blue circle). Since these two sites act in the same direction, they reinforce one another, leading to overestimation of their effect sizes, whereas the two at the right (green circle) interfere with one another antagonistically, leading to underestimation of their effects. The effect at the fifth site (brown circle) may only be identified following conditional analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1647342842" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic of multisite regulation of gene expression. Black bars indicate univariate estimates of allelic effects of minor alleles increasing (above the horizontal) or decreasing (below the horizontal) gene expression without conditioning on other sites. Red bars show the actual effects at five single nucleotide polymorphisms in this locus, which has a linkage disequilibrium profile with two large and one small block of elevated linkage disequilibrium (pink squares). Dotted horizontal lines indicate a statistical significance threshold, which is only exceeded in the univariate modeling by the two left-hand sites (blue circle). Since these two sites act in the same direction, they reinforce one another, leading to overestimation of their effect sizes, whereas the two at the right (green circle) interfere with one another antagonistically, leading to underestimation of their effects. The effect at the fifth site (brown circle) may only be identified following conditional analysis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2533/F1.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/8/2533/F1.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485980" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Schematic of multisite regulation of gene expression. Black bars indicate univariate estimates of allelic effects of minor alleles increasing (above the horizontal) or decreasing (below the horizontal) gene expression without conditioning on other sites. Red bars show the actual effects at five single nucleotide polymorphisms in this locus, which has a linkage disequilibrium profile with two large and one small block of elevated linkage disequilibrium (pink squares). Dotted horizontal lines indicate a statistical significance threshold, which is only exceeded in the univariate modeling by the two left-hand sites (blue circle). Since these two sites act in the same direction, they reinforce one another, leading to overestimation of their effect sizes, whereas the two at the right (green circle) interfere with one another antagonistically, leading to underestimation of their effects. The effect at the fifth site (brown circle) may only be identified following conditional analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Several Bayesian methods have recently been introduced that should improve localization of linked causal variants, although they do not in general estimate effect sizes. CAVIAR (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hormozdiari <em>et al.</em> 2014</a>) enumerates all possible causal states for one or more sites in a short interval of ≤100 SNPs, but to control the computational burden, the maximum number of causal variants is typically set to two. It is claimed to improve identification of causal variants by 20–50% over existing methods such as BIMBAM (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Servin and Stephens 2007</a>). The eCAVIAR extension for combined eQTL and GWAS analysis (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hormozdiari <em>et al.</em> 2016</a>) uses a greedy method to find a subset of SNPs with a specific confidence (95% by default) that causal variants are identified as candidates. PAINTOR (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kichaev <em>et al.</em> 2014</a>) uses a similar algorithm, whereas FM-QTL (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Wen <em>et al.</em> 2015</a>) applies an MCMC algorithm to explore the causal status space, utilizing a posterior inclusion probability to choose the causal variant credible interval DAP software was then developed (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wen <em>et al.</em> 2016</a>) to explore high probability causal intervals with reasonable runtime. FINEMAP (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Benner <em>et al.</em> 2016</a>) uses logical schema that is similar to that of CAVIAR, but adopts a Shotgun Stochastic Search method to restrict the search space and focus on combinations of high probability intervals.</p><p id="p-10">This study explores the sources of error in estimating eQTL effects. We start by using simulations to ask how multisite regulation influences (i) the number of independent peaks detected by stepwise conditional analysis, (ii) the accuracy of localization of true causal variants, and (iii) the effect size estimation of discovered causal variants. We also address the question of what proportion of discovered peaks may be driven by undocumented variants in LD with the genotyped sites, and conclude with a comparison of the performance of two recently developed Bayesian joint localization methods, eCAVIAR and DAP, finding minor improvements in detection of linked causal variants but little impact on fine mapping, particularly in regions of high LD or if sample sizes are small.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Consortium for the Architecture of Gene Expression (CAGE) dataset</h3><p id="p-11">Our simulations utilize genotypes obtained from the CAGE dataset, which consists of Illumina HT12 microarray-based gene expression profiles, as well as whole-genome genotype information from five research studies: the Brisbane Systems Genetics Study (BSGS, <em>N</em> = 926) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Powell <em>et al.</em> 2012</a>), the Atlanta-based Centre for Health Discovery and Well-Being (<em>N</em> = 439) (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wingo and Gibson 2015</a>) and Emory Cardiology Genebank (<em>N</em> = 147, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kim <em>et al.</em> 2014</a>), the Estonian Genome Centre, University of Tartu study (<em>N</em> = 1065, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schramm <em>et al.</em> 2014</a>), and the Morocco Lifestyle study (<em>N</em> = 188, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Idaghdour <em>et al.</em> 2010</a>), for a total of 2765 individuals. Since the BSGS sample includes twins, it was removed to avoid complications of relatedness, leaving a set of 1839 European-ancestry unrelated individuals. IRB approval was obtained for the combination of data into a mega-analysis, both by the University of Queensland and for each participating site.</p><p id="p-12">Genotype imputation for the CAGE cohort was performed jointly on the five contributing studies to ensure uniformity of assignment of strand identities of SNPs, and is described in detail in <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lloyd-Jones <em>et al.</em> (2017)</a> and at <a href="https://github.com/CNSGenomics/impute-pipe">https://github.com/CNSGenomics/impute-pipe</a>. Briefly, the pipeline was as follows: (1) preimputation quality control and data consistency checks; (2) imputation to the 1000G reference panel with Impute2 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Howie <em>et al.</em> 2012</a>); (3) postimputation quality control (filtering on various data features); and (4) merging datasets on common SNPs.</p></div><div id="sec-3" class="subsection"><h3>Simulation studies</h3><p id="p-13">Four different simulation studies were conducted. In all cases, we use the terminology uni-site (univariable) to refer to models where a single causal variant is modeled as a fixed effect, and multisite (multivariable) where two or more variants are modeled, also as fixed effects. The term multivariate modeling is used for situations where there are two or more dependent variables, whereas in these models we are assessing the joint effects of two or more causal variables. Some models also incorporate random effects of covariates such as a genetic relationship matrix.</p><p id="p-14">The first set of simulations assessed the power and accuracy of two site regressions assuming that the identities of the two causal variants are already known. We modeled the influences of effect size, minor allele frequency (maf), LD, and sample size. Environmental variance was randomly generated as a z-score (mean 0 and SD 1) and genotype effects (β) were added in SD units (sdu) multiplied by ± 0, 1, or 2 according to genotype so as to account for from 2 to 30% of the phenotypic variance, computed as 2<em>p</em>(1−<em>p</em>)β<sup>2</sup>. Thus, an allele with β = 0.8 is expected to explain 20% of the variance if maf <em>P</em> = 0.2, or 32% of the variance if <em>P</em> = 0.5. The influence of LD was assessed at <em>r</em><sup>2</sup> = 0.1, 0.5, or 0.9, noting that as LD increases, high <em>r</em><sup>2</sup> values are not obtained for combinations of a rare and a common allele. For each combination of parameter values, we generated 1000 randomizations of the environmental variance, and assessed (i) the univariate estimate at each genotype, (ii) the mean conditional estimate of the second SNP, and (iii) the joint effect estimates with both SNPs. From these values, we computed the mean absolute value of the deviation between the observed estimate and the true effect size from the univariate, conditional, and joint (two site) models. The univariate estimates agree extremely well with expectations from the analytical solution described in the <em>Results</em>.</p><p id="p-15">The second simulation study asked whether unimputed variants influence the localization of eSNP signals. Since nonimputed SNPs are not present in the CAGE data, we approximated their identities by randomly sampling from a set of CAGE-imputed SNPs weighted to have the same frequency distribution shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5C</a> and assigning effect sizes from 2 to 10% of the variance explained for normally distributed pseudogene expression traits using the CAGE (minus BSGS) genotypes. We then removed the SNP and all other SNPs with <em>r</em><sup>2</sup> &gt; 0.8, and performed stepwise conditional regression, documenting instances of primary and secondary signals at <em>P</em> &lt; 10<sup>−5</sup>, as plotted in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5E</a>. The cumulative proportion of spurious secondary signals was computed by summing the detection rate by the size of the maf bin of the unimputed SNPs.</p><p id="p-16">The third set of simulations were performed to evaluate the difference in effect size estimates using the multisite linear regression method for parameter estimation from data representative of the LD structure in the CAGE dataset. For each of 500,000 iterations, four sites were chosen at random from a window extending from 200 kb upstream of the transcription start site to 200 kb downstream of the transcription termination site of a randomly picked gene in the CAGE cohort (excluding the BSGS data, since it includes twins), and assigned an effect size from a uniform distribution of variance explained (VE) relative to environmental noise ranging between 0.02 and 0.1. The effect size β for an allele with maf <em>p</em> is computed as √[VE/2<em>p</em>(1−<em>p</em>)]. Subsequently, each phenotype was simulated as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2533/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2533/embed/mml-math-1.gif"/></noscript></span></span> + <em>N</em>(0,1), where β<em><sub>i</sub></em> is the simulated allelic effect size for a SNP <em>i</em>, and <em>geno<sub>i</sub></em> is the dosage of minor allele at the simulated SNP for a given sample. The significance threshold for sequential conditional detection of the variants in a sample of 1839 CAGE individuals was set at <em>P</em> &lt; 10<sup>−5</sup>, since simulations indicated a &lt;1% false discovery rate for null variants at this level. We evaluated (i) how many of the four SNPs were significant in sequential conditional modeling, (ii) the mean LD between each SNP and the other three SNPs in the model, (iii) the effect size estimates from the conditional and joint multisite models, (iv) the difference between these two estimates as a function of the mean LD, and (v) the rank of the discovered SNP for each peak eSNP and the modeled sites, which were assumed to be the causal variants for some trait.</p><p id="p-17">The fourth set of simulations was performed to evaluate the influence of two Bayesian methods for fine mapping that is sensitive to the LD structure at a locus. First, eCAVIAR was used to also assess the accuracy of colocalization of eQTL and GWAS signals. Summary statistics were generated for normally distributed traits where either one, two, or three sites chosen at random from contiguous intervals of 100 SNPs in the full sample of 1835 CAGE genotypes were assigned to explain between 2 and 10% of the variance. Effects were assigned in the same direction for each minor allele. Marginal single site estimates were generated by univariable regression, and then eCAVIAR was used to combine the Posterior Probabilities, which were multiplied together to yield the Combined Likelihood Posterior Probabilities (CLPP) with a significance threshold of 0.001 as recommended (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hormozdiari <em>et al.</em> 2016</a>). Owing to the high computational burden, only 4000 simulations were performed. GWAS variants are in general unlikely to explain this amount of variance, but the statistical evidence is approximately equivalent to that expected for typical trait associations where a SNP explains ≤0.1% of the variance in a sample of 20,000 individuals. The effect of sample size was evaluated by fitting a single eQTL effect to just 200 individuals in each simulation. Second, the DAP simulations were performed using the adaptive algorithm, which estimates the number of causal variants from the data and also generates a list of possible sites that could explain the effect(s). Again, owing to the high computational burden, only 130 simulations were performed, using the same parameters as for the sequential conditional analyses with four assigned causal variants. A final set of simulations designed only to fine map three causal sites in a single moderate to high LD block extracted contiguous sets of 100 SNPs, and randomly assigned effects only on the condition that three sites selected from the set each had <em>r</em><sup>2</sup> &gt; 0.3 with one another.</p></div><div id="sec-4" class="subsection"><h3>Data availability</h3><p id="p-18">All scripts are available at <a href="https://github.com/jxzb1988/Script-for-constraint-paper">https://github.com/jxzb1988/Script-for-constraint-paper</a>.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Underestimation of allelic effects by sequential conditional analysis</h3><p id="p-19">Our basic simulation framework utilizes the current standard mapping approach of sequential conditional analysis, in which the residuals from discovery of each SNP are taken forward as the dependent variable in a new scan for an independent SNP (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Yang <em>et al.</em> 2012</a>). To explore the performance of this strategy in the context of four causal regulatory variants in the vicinity of a typical gene, 500,000 simulations were carried out by randomly picking four SNPs within 200 kb up- or downstream of the 5′ and 3′ ends of a randomly chosen gene, from the imputed whole-genome genotypes of 1839 unrelated European-ancestry individuals. We assigned each SNP an allelic effect size so as to explain between 2 and 10% of the variance of a trait otherwise uniformly distributed with a mean of 0 and SD of 1. Power to detect individual univariate effects of this magnitude is close to 100% at the significance level <em>P</em> &lt; 10<sup>−5</sup>. The sampling was performed across all genes so as to sample from the typical LD structure in the European-ancestry human genome. Furthermore, effects were randomly assigned under three scenarios, with either four positive (4:0), three positive and one negative (3:1), or two positive and two negative (2:2) effects of the minor allele on the trait. For the eQTL detection, once the sequential conditional detection was completed, we determined which of the four causal variants was in high LD (<em>r</em><sup>2</sup> &gt; 0.8) with one of the discovered sites. If a peak was in high LD with more than one causal variant, it was assumed that it tagged the highest effect site.</p><p id="p-20"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> summarizes the “tagging efficiency,” namely the percent of simulations in which the indicated number of significant independent sites was detected, as well as the proportion of the discovered variants that are in high LD (<em>r</em><sup>2</sup> &gt; 0.8) with one of the simulated causal variants. Across all three scenarios, at least three independent peaks are detected in ∼90% of the simulations, and at least four independent peaks in two-thirds of the simulations. Notably, in the scenarios where all four minor alleles influence expression in the same direction, 10% of the simulations detected five or more independent peaks, at least one of which must be a spurious association, despite a &lt;1% false discovery rate for univariate associations of the same magnitude. The proportion of credible intervals (<em>r</em><sup>2</sup> &gt; 0.8 regions around each discovered variant) that contain the actually simulated site ranges between 85 and 90% in each scenario, again indicating relatively poor localization of the causal variant.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486115/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486115/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Tagging efficiency of detection of causal variants with <em>r</em><sup>2</sup> cutoff 0.8</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">Similar results are reported in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> for the reciprocal measure of what fraction of simulated variants is captured by discovered variants. It includes results for simulations with just two or three causal variants, and reports the percentage of cases where the causal variant was within the <em>r</em><sup>2</sup> &gt; 0.8 credible interval for a discovered peak. Across the sets of 500,000 simulations, at least two variants are detected &gt;85% of the time, but the power to detect all of the multiple eSNPs is a function of the number of sites operating in the same direction. It is highest for the case where the minor alleles for all four variants have effects in the same direction and least where two are in one direction and two in the opposite direction. In the 4:0 scenario, three or more of the four eSNPs are detected three-quarters of the time, whereas this proportion drops toward two-thirds with the simulations for 2:2. No variants are detected in just over 1% of the simulations, and just one variant in ∼6% of them, while 80% of the variants are detected overall. This proportion rises to 90% for the two-variant simulations, illustrating how multisite interactions reduce the discovery of independent eQTL peaks.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485909/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485909/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Detected true causal variants in simulations with four, three, or two causal variants</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">Another way to consider the power of multisite detection is to ask how much of the variance explained by the four SNPs is captured by the discovered variants. Box and whisker plots in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> show that, under all three scenarios, on average 85–90% of the variance is captured, namely in these simulations ∼15–20% of the transcript abundance. Although effect sizes of all SNPs were drawn from the same distribution, the first discovered SNP (rightmost box in each panel) typically explains between one-third and one-half of the expected variance, suggesting that it often tags some of the effect of another site. Since the primary SNP captures on average more than two-thirds of the heritability at each locus in actual peripheral blood data (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Lloyd-Jones <em>et al.</em> 2017</a>), it is likely that secondary and tertiary SNP effects are, in reality, smaller than primary SNP effects. As expected, summation of the independent contributions from the sequential conditional models, or fitting all of the discovered variants simultaneously in a multivariable model, explains very similar proportions of the variance overall. Similar results are seen with two or three simulated causal variants (Supplemental Material, Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of variance explained by detected eSNPs in simulations. Box and whiskers show median, interquartile range, and 95% C.I. for the proportion of variance explained under three scenarios for 500,000 simulations of four sites affecting gene expression. From left to right in each simulation, Simu is the variance explained by the known sites, Multi is the result fitting discovered eSNPs jointly, Uni is the result of summing the effects from sequential conditional modeling, and Single is the effect of the peak detected eSNP. The y-axis shows the proportion of variance explained. Scenarios are 4:0, all four minor alleles with effects in same direction; 3:1, one minor allele effect in the opposite direction; 2:2, two minor alleles on one direction and the other two in the opposite direction. eSNP, expression single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1647342842" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proportion of variance explained by detected eSNPs in simulations. Box and whiskers show median, interquartile range, and 95% C.I. for the proportion of variance explained under three scenarios for 500,000 simulations of four sites affecting gene expression. From left to right in each simulation, Simu is the variance explained by the known sites, Multi is the result fitting discovered eSNPs jointly, Uni is the result of summing the effects from sequential conditional modeling, and Single is the effect of the peak detected eSNP. The &lt;em&gt;y&lt;/em&gt;-axis shows the proportion of variance explained. Scenarios are 4:0, all four minor alleles with effects in same direction; 3:1, one minor allele effect in the opposite direction; 2:2, two minor alleles on one direction and the other two in the opposite direction. eSNP, expression single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2533/F2.medium.gif" width="409" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/8/2533/F2.medium.gif" width="409" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485902" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Proportion of variance explained by detected eSNPs in simulations. Box and whiskers show median, interquartile range, and 95% C.I. for the proportion of variance explained under three scenarios for 500,000 simulations of four sites affecting gene expression. From left to right in each simulation, Simu is the variance explained by the known sites, Multi is the result fitting discovered eSNPs jointly, Uni is the result of summing the effects from sequential conditional modeling, and Single is the effect of the peak detected eSNP. The <em>y</em>-axis shows the proportion of variance explained. Scenarios are 4:0, all four minor alleles with effects in same direction; 3:1, one minor allele effect in the opposite direction; 2:2, two minor alleles on one direction and the other two in the opposite direction. eSNP, expression single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Estimation of the proportion of secondary associations that are false positives</h3><p id="p-25">The detection of more association peaks than the number of simulated sites implies that some fraction of peaks are false positives that arise due to sampling artifacts in the presence of high LD, whereby an imperfectly tagged site is split into two or more spurious signals. An example is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B</a>, contrasting the local Manhattan profiles for a single causal variant that splits into two associations when the peak SNP, rs9806753, is excluded from analysis. To explore the frequency with which this occurs, we conducted simulations assuming a single causal variant based on the genotypes measured in 1839 European-ancestry samples from the CAGE cohorts (see <em>Methods</em>). Randomly assigning causal effects resulted in the appearance of a secondary signal at <em>P</em> &lt; 10<sup>−5</sup>, conditioned on the causal site, at 0.3% of the loci. This is approximately as expected given 8.3 million imputed SNPs at 22,000 loci, and is also the same as the false discovery rate of primary signals in the absence of any simulated causal locus; that is to say, our random expectation is for 0.3% of transcripts to have a false eQTL discovery at <em>P</em> &lt; 10<sup>−5</sup> in the CAGE dataset.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of false multiple eQTL detection due to unimputed variants. (A and B) Example showing how poor tagging can split one causal into two separate signals at the SHC4 locus. (A) SNP rs9806753 (European-ancestry maf = 0.23) was simulated in CAGE to generated an eSNP effect, but removal of this variant and all SNPs within r2 &gt; 0.5 from the analysis (B) results in the effect being captured by both a primary (rs62010876, maf = 0.10) and secondary (rs1974961, maf = 0.13) signals. (C) Empirical maf density distribution of 1.4 M unimputed 1000G (red) and 8.3 M imputed 1000G variants in CAGE (blue), demonstrating shift to lower frequencies for variants not tagged. (D) Tagging efficiency as a function of maf based in mean r2 for the strongest correlated SNP for 10,000 randomly selected variants across the frequency spectrum in the CAGE genotype dataset. (E) Corresponding signal detection rate at P &lt; 10&#x2212;5 for randomly assigned effect sizes, explaining between 2 and 8% of a simulated gene expression trait for primary (red) and secondary (blue) signals when the simulated variant is excluded from the analysis. (F) Cumulative proportion of sites expected to generate a false multiple eQTL detection, calculated as the sum of the false secondary signal detection rate weighted by the maf frequency in the indicated bins. Multiplication of this proportion by the number of unimputed SNPs in genic regions (14% of 1.4 M) and the actual proportion of SNPs that have effects (unlikely to be &gt;1%) yields up to 400 possible false positive secondary associations. CAGE, Consortium for the Architecture of Gene Expression; eQTL, expression quantitative trait locus; eSNP, expression SNP; maf, minor allele frequency; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1647342842" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proportion of false multiple eQTL detection due to unimputed variants. (A and B) Example showing how poor tagging can split one causal into two separate signals at the &lt;em&gt;SHC4&lt;/em&gt; locus. (A) SNP rs9806753 (European-ancestry maf = 0.23) was simulated in CAGE to generated an eSNP effect, but removal of this variant and all SNPs within &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; &gt; 0.5 from the analysis (B) results in the effect being captured by both a primary (rs62010876, maf = 0.10) and secondary (rs1974961, maf = 0.13) signals. (C) Empirical maf density distribution of 1.4 M unimputed 1000G (red) and 8.3 M imputed 1000G variants in CAGE (blue), demonstrating shift to lower frequencies for variants not tagged. (D) Tagging efficiency as a function of maf based in mean &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; for the strongest correlated SNP for 10,000 randomly selected variants across the frequency spectrum in the CAGE genotype dataset. (E) Corresponding signal detection rate at &lt;em&gt;P&lt;/em&gt; &lt; 10&lt;sup&gt;&#x2212;5&lt;/sup&gt; for randomly assigned effect sizes, explaining between 2 and 8% of a simulated gene expression trait for primary (red) and secondary (blue) signals when the simulated variant is excluded from the analysis. (F) Cumulative proportion of sites expected to generate a false multiple eQTL detection, calculated as the sum of the false secondary signal detection rate weighted by the maf frequency in the indicated bins. Multiplication of this proportion by the number of unimputed SNPs in genic regions (14% of 1.4 M) and the actual proportion of SNPs that have effects (unlikely to be &gt;1%) yields up to 400 possible false positive secondary associations. CAGE, Consortium for the Architecture of Gene Expression; eQTL, expression quantitative trait locus; eSNP, expression SNP; maf, minor allele frequency; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2533/F3.medium.gif" width="439" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/8/2533/F3.medium.gif" width="439" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485991" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Proportion of false multiple eQTL detection due to unimputed variants. (A and B) Example showing how poor tagging can split one causal into two separate signals at the <em>SHC4</em> locus. (A) SNP rs9806753 (European-ancestry maf = 0.23) was simulated in CAGE to generated an eSNP effect, but removal of this variant and all SNPs within <em>r</em><sup>2</sup> &gt; 0.5 from the analysis (B) results in the effect being captured by both a primary (rs62010876, maf = 0.10) and secondary (rs1974961, maf = 0.13) signals. (C) Empirical maf density distribution of 1.4 M unimputed 1000G (red) and 8.3 M imputed 1000G variants in CAGE (blue), demonstrating shift to lower frequencies for variants not tagged. (D) Tagging efficiency as a function of maf based in mean <em>r</em><sup>2</sup> for the strongest correlated SNP for 10,000 randomly selected variants across the frequency spectrum in the CAGE genotype dataset. (E) Corresponding signal detection rate at <em>P</em> &lt; 10<sup>−5</sup> for randomly assigned effect sizes, explaining between 2 and 8% of a simulated gene expression trait for primary (red) and secondary (blue) signals when the simulated variant is excluded from the analysis. (F) Cumulative proportion of sites expected to generate a false multiple eQTL detection, calculated as the sum of the false secondary signal detection rate weighted by the maf frequency in the indicated bins. Multiplication of this proportion by the number of unimputed SNPs in genic regions (14% of 1.4 M) and the actual proportion of SNPs that have effects (unlikely to be &gt;1%) yields up to 400 possible false positive secondary associations. CAGE, Consortium for the Architecture of Gene Expression; eQTL, expression quantitative trait locus; eSNP, expression SNP; maf, minor allele frequency; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">However, this ignores the possibility that the causal variants are not present in the imputed genotypes. Of the 9.7 million SNPs, indels, and CNV with maf &gt; 0.01 in the European populations in the 1000G database, 1.9 million are not imputed in our CAGE samples. <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3C</a> shows that the maf distribution for these variants is strongly shifted toward rarer alleles relative to the imputed SNPs, and is centered at a maf ∼0.02. Consistent with <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Yang <em>et al.</em> (2015)</a>, the average tagging efficiency (<em>r</em><sup>2</sup> value) of these SNPs is a function of maf, being &gt;0.7 for maf &gt; 0.05, but dropping to &lt;0.5 for maf = 0.01, as seen in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3D</a>.</p><p id="p-28">Since we cannot simulate effects at nonimputed SNPs, we approximated such alleles by randomly simulating a causal variant from the CAGE SNPs with the same frequency distribution, but excluding it from the analysis along with all variants that would tag it at the typical level observed for the nonimputed SNPs of the same maf. We then asked how often the effect is captured by multisite signals, as a function of residual tagging efficiency. We allowed for increased effect sizes with lower maf by simulating effects in the constant range of 2–10% of the variance explained. The proportion of such pseudounimputed SNPs that generate primary signals is reduced with lower maf, due in part to the smaller proportion of variance explained by the less common variants that partially tag them. For common variants, there is almost always a second site in high enough LD to capture most of the causal signal in the absence of genotypes at the causal variant, but rare variants are insufficiently tagged to generate a signal at all, 90% of the time.</p><p id="p-29">In the presence of tagging SNPs with <em>r</em><sup>2</sup> &gt; 0.5 to the “unobserved” causal variant, false secondary associations are observed ≤40% of the time. At the other end of the spectrum, rare variants (maf ∼0.01) that produce a primary signal at a tagging SNP with 0.1 &lt; <em>r</em><sup>2</sup> &lt; 0.3 also produce a secondary signal but less frequently. The blue curve in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3E</a> indicates the inferred fraction of unimputed variants that could induce secondary signals as a function of maf, and <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3F</a> shows that the cumulative proportion of such spurious eQTL weighted by observed maf proportions approaches 20%. Approximately 14% of the 1.9 M unimputed variants are located within 200 kb of a gene, and assuming that 0.1% of these actually have an eQTL effect, this suggests the potential for ∼250 such effects.</p><p id="p-30">These computations argue that up to 10% of the observed &gt;2300 multisite associations reported by <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Lloyd-Jones <em>et al.</em> (2017)</a> have the potential to be false signals driven by inefficient tagging of unimputed variants in CAGE. The proportion could be greater if the fraction of functional SNPs is higher, as suggested for example by <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tewhey <em>et al.</em> (2016)</a>, who used a very sensitive MPR assay to implicate 3% of regulatory sites in 3642 eQTL regions (842/32,373 tests) as capable of modulating transcript abundance. However, the proportion of sites with detectable signals capable of explaining &gt;2% of the variance is certainly lower, and 1 in 1000 (0.1%) is a reasonable estimate given that there are of the order of 1300 documented variants in the vicinity of each gene and no more than 30% of expressed genes have a secondary eQTL signal. Synthetic associations due to even rarer variants may be expected to generate split associations as well (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dickson <em>et al.</em> 2010</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zhu <em>et al.</em> 2012</a>). <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Yang <em>et al.</em> (2015)</a> found that ∼20% of the variance for height can be explained by SNPs with maf &lt; 0.1, in part due to larger effect sizes of prevalent very rare SNPs, many of which are likely secondary associations. We also found that there is an excess of rare variants (maf &lt; 0.01) influencing extremes of gene expression, also with a slightly larger distribution of effect sizes than common variants (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2016</a>). Too many unknown parameters need to be evaluated to give a good estimate of the number of false positive secondary associations due to synthetic effects of very rare alleles, but it may be another few percent.</p></div><div id="sec-8" class="subsection"><h3>Effect of multisite modeling on accuracy of localization of associations</h3><p id="p-31">A possibly more important measure of estimation bias is the location of the peak SNP relative to the causal site. The most straightforward measure of colocalization is the Regulatory Trait Concordance (RTC) score (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Nica <em>et al.</em> 2010</a>), which is intuitive and easily implemented on the scale of our simulations. It is essentially a ranking of the significance of the detected eQTL <em>P</em>-value relative to the casual site, RTC = (<em>N<sub>SNPs</sub></em> − <em>Rank</em><sub>causal SNP</sub>)/<em>N<sub>SNPs</sub></em>, where a value of one indicates identity, and zero that the two sites are in the same locus but highly unlikely to be capturing the same signal. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> plots the cumulative frequency distribution for RTC scores for the primary eQTL signals relative to the largest effect causal variant in each simulation, contrasting the 4:0, 3:1, and 2:2 scenarios. For comparison, under a single variant model, RTC is always close to one as expected [note that, since we do not simulate the GWAS signal as well, these values are inflated relative to data where the identity of the actual causal variant is unknown (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Nica <em>et al.</em> 2010</a>)]. For 10% of simulations in the presence of multiple regulatory variants, the RTC score of the primary SNP drops below 0.9, again with greater tendency toward misestimation of the eQTL location in models with opposing effect directions of the minor alleles. This analysis confirms the results in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, indicating that up to 15% of all detected SNPs are not in high LD (<em>r</em><sup>2</sup> &gt; 0.8) with a simulated variant in the imputed panel of SNPs.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The single causal variant assumption biases fine mapping of causal variant locations. Each curve represents the cumulative probability distribution for RTC scores for the primary causal variants under a model with a single causal variant (purple), or with four causal variants under scenarios 4:0, 3:1, and 2:2 (blue, green, red curves, respectively). RTC scores close to one imply equivalence of the significance values of the eQTL and causal variant. eQTL, expression quantitative trait locus; RTC, Regulatory Trait Concordance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1647342842" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The single causal variant assumption biases fine mapping of causal variant locations. Each curve represents the cumulative probability distribution for RTC scores for the primary causal variants under a model with a single causal variant (purple), or with four causal variants under scenarios 4:0, 3:1, and 2:2 (blue, green, red curves, respectively). RTC scores close to one imply equivalence of the significance values of the eQTL and causal variant. eQTL, expression quantitative trait locus; RTC, Regulatory Trait Concordance.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2533/F4.medium.gif" width="440" height="277"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/8/2533/F4.medium.gif" width="440" height="277"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486171" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">The single causal variant assumption biases fine mapping of causal variant locations. Each curve represents the cumulative probability distribution for RTC scores for the primary causal variants under a model with a single causal variant (purple), or with four causal variants under scenarios 4:0, 3:1, and 2:2 (blue, green, red curves, respectively). RTC scores close to one imply equivalence of the significance values of the eQTL and causal variant. eQTL, expression quantitative trait locus; RTC, Regulatory Trait Concordance.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Localization of tertiary and quaternary signals is affected more strongly, but intriguingly, considering just associations within <em>r</em><sup>2</sup> &gt; 0.8 of a simulated SNP, the secondary signal is slightly more likely to be the first or second ranked SNP for one of the causal variants than is the primary signal. This is true under all three scenarios, which have very similar profiles to that shown for the 3:1 scenario in Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.pdf">File S1</a> (since there is wide variance in the number of SNPs in each region, we simplified the analysis by reporting just the SNP ranks in this figure, rather than RTC). It should be noted that there is not strong concordance between the relative proportion of variance explained by the causal variants and whether they are the primary through quaternary association, since LD has a strong influence on detection power. Although the vast majority of discovered sites are within three or four SNPs of at least one of the four causal variants when they are in high LD with one of them, it cannot be concluded that the order of discovery corresponds to the true order of effect sizes.</p></div><div id="sec-9" class="subsection"><h3>Joint fitting pairs of known causal variants accurately estimates effect sizes</h3><p id="p-34">Before addressing the accuracy of effect size estimation following stepwise conditional analysis, it is worth noting that, in the case where the identities of two causal variants are known <em>a priori</em>, joint fitting of the two SNPs in a single regression on transcript abundance always results in more accurate effect size estimates, given a large sample size. The bias in estimation due to LD between pairs of SNPs is a function of the two effect sizes (β<sub>1</sub> and β<sub>2</sub>), the correlation between the SNPs (<em>r</em>), and the ratio of the square root of the product of their allele frequencies: Ê(β<sub>1</sub>)−β<sub>1</sub> = <em>r</em>β<sub>2</sub> √(<em>p</em><sub>1</sub>(1−<em>p</em><sub>1</sub>)/<em>p</em><sub>2</sub>(1−<em>p</em><sub>2</sub>)) (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Yang <em>et al.</em> 2012</a>). This is maximized for pairs of SNPs at the same frequency, increases with high LD, and can be either positive or negative depending on whether the signs of the minor allele effects are coupled or not. Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.pdf">File S1</a> provides a visual summary of the biases, compared with the effect of jointly fitting the two SNPs with a sample size of 2000, which uniformly improves the effect size estimates.</p><p id="p-35">Three results deserve highlighting. First, for each combination of allele frequencies, increasing the allelic effect size results in more severe biases, in the most severe cases over- or underestimating the effects by as much as 50% of the variance explained. Second, the first picked SNP (with the larger effect size) has the greater deviation between the estimated and true effect size. This makes intuitive sense as the larger effect will generally be the first detected one and absorbs much of the effect of the other SNP, which will typically be underestimated in the conditional analysis, but to a lesser extent. If the deviation is computed simply on the uni-site unconditional values, the opposite result is obtained: the deviation is greatest for the smaller effect site. Third, the misestimation is greatest for lower allele frequencies, which is particularly noteworthy since most eQTL have maf in the range of 0.1–0.3.</p><p id="p-36">We also considered the power to detect joint effects in the presence of LD. As the <em>P</em> value cutoff for detection becomes more stringent, the bias in estimation becomes more severe, since the first picked SNP absorbs the effect of both alleles into the same estimate, leaving the statistical power of the other allele, conditioned upon the first one, close to zero. With a sample size of 2000 and intermediate LD, when both alleles are modeled jointly, power to detect both effects remains high across the plausible parameter space once the effect size exceeds 5% variance explained, and the estimation for the β value is still accurate. Downsampling suggests that in order to estimate effects within 0.1 sdu, for pairs of variants with LD <em>r</em><sup>2</sup> ∼ 0.9, each explaining 10% of the variance (namely having effect sizes of at least 0.5 sdu), a sample size of at least 900 is required.</p><p id="p-37">Consequently, most small sample eQTL studies will fail to resolve linked sites into two effects. These results indicate how the typical assumption that an eQTL effect is due to a single variant in a set of credible SNPs in high LD is potentially highly biased. Similar conclusions apply to the situation where two SNPs operate in opposite directions, with the additional dilemma that they will not be detected at all and consequently strongly underestimate the regulatory variance at a locus.</p></div><div id="sec-10" class="subsection"><h3>Misestimation of allelic effects sizes by sequential conditional analysis</h3><p id="p-38">Even though the sequential conditional and multisite models capture essentially equivalent proportions of the variance tallied across sites, biases in estimation of individual site effects ought to be reduced by the multisite modeling. To quantify this difference, we computed the deviation between the observed and true simulated effect sizes (β in sdu) for each discovered peak located within <em>r</em><sup>2</sup> &gt; 0.8 of an independent causal variant, and evaluated the absolute value of these deviations as a function of the mean LD between the causal variant SNP and the other three causal variants in the model. <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5, A and B</a> show the average absolute value of the deviation for SNPs with the indicated true effect size and LD, for the sequential conditional estimates, and for multivariable models fitting all discovered variants jointly, respectively. The figure shows results for the 4:0 scenario where all minor alleles operate in the same direction. The scale from dark blue to yellow indicates misestimation of effect sizes ranging from less than 0.5 sdu to more than three units, where each pixel is averaged over the number of simulations with the indicated allele effect sizes and average LD in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Biases in effect size estimation from conditional and joint analysis. All panels refer to 500,000 simulated data points where effect sizes were sampled from a uniform distribution to explain from 2% to 10% of the expression at a locus for each of 4 SNPs picked at random from 400kb intervals of the CAGE genotype data. Panel (A) compares estimates from joint and conditional modeling, as a heatmap of the average difference in panels C and D, where yellow indicates that joint modeling produces a larger estimated effect size, and blue a lower estimate with three bands of negative values indicating greater bias in the conditional estimates. Panel B shows the density distribution on the log2 scale of the number of simulations with alleles for each pixel with the indicated &#x3B2; (in standard deviation units, sdu) on the x-axis, and average LD with the other 3 sites at the locus on the y-axis. Panels C and D show the average absolute value of the deviation between the observed and known effect size for sites under the multi-site model where all discovered sites are fit jointly (C) or from single site estimates after each step of sequential conditional analysis (D), for the 4:0 scenario where all minor alleles have effects in the same direction. See Figure S4 for the equivalent panels for the 3:1 and 2:2 scenarios." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1647342842" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Biases in effect size estimation from conditional and joint analysis. All panels refer to 500,000 simulated data points where effect sizes were sampled from a uniform distribution to explain from 2% to 10% of the expression at a locus for each of 4 SNPs picked at random from 400kb intervals of the CAGE genotype data. Panel (A) compares estimates from joint and conditional modeling, as a heatmap of the average difference in panels C and D, where yellow indicates that joint modeling produces a larger estimated effect size, and blue a lower estimate with three bands of negative values indicating greater bias in the conditional estimates. Panel B shows the density distribution on the log2 scale of the number of simulations with alleles for each pixel with the indicated &#x3B2; (in standard deviation units, sdu) on the x-axis, and average LD with the other 3 sites at the locus on the y-axis. Panels C and D show the average absolute value of the deviation between the observed and known effect size for sites under the multi-site model where all discovered sites are fit jointly (C) or from single site estimates after each step of sequential conditional analysis (D), for the 4:0 scenario where all minor alleles have effects in the same direction. See Figure S4 for the equivalent panels for the 3:1 and 2:2 scenarios.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2533/F5.medium.gif" width="440" height="160"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/8/2533/F5.medium.gif" width="440" height="160"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486009" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">Biases in effect size estimation from conditional and joint analysis. All panels refer to 500,000 simulated data points where effect sizes were sampled from a uniform distribution to explain from 2% to 10% of the expression at a locus for each of 4 SNPs picked at random from 400kb intervals of the CAGE genotype data. Panel (A) compares estimates from joint and conditional modeling, as a heatmap of the average difference in panels C and D, where yellow indicates that joint modeling produces a larger estimated effect size, and blue a lower estimate with three bands of negative values indicating greater bias in the conditional estimates. Panel B shows the density distribution on the log2 scale of the number of simulations with alleles for each pixel with the indicated β (in standard deviation units, sdu) on the x-axis, and average LD with the other 3 sites at the locus on the y-axis. Panels C and D show the average absolute value of the deviation between the observed and known effect size for sites under the multi-site model where all discovered sites are fit jointly (C) or from single site estimates after each step of sequential conditional analysis (D), for the 4:0 scenario where all minor alleles have effects in the same direction. See Figure S4 for the equivalent panels for the 3:1 and 2:2 scenarios.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Several results are noteworthy. First, for causal variants in low LD, as expected, neither model results in appreciable estimate bias, but once the average LD rises above 0.5, effects can be misestimated by more than the effect size. For example, for β = 0.5, the absolute value of the difference between the observed and true effect is typically between 1 and 2 sdu, which depending on the allele frequency may correspond to at least 2% of the total gene expression variance. Second, for large effect alleles, the misestimation is appreciable even at intermediate levels of LD, and it is not unusual for estimates to be off by as much as 4 sdu under either model.</p><p id="p-41">Third, overall, the multisite modeling corrects some of the sequential conditional analysis bias. The difference in performance of the two estimation procedures is shown in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5D</a>, a plot of the average multisite estimate minus the average sequential conditional estimate. Most values are pale green, indicating close similarity of the estimates, but bluish-tinged bands imply that the multisite model gives a better approximation to the true effect size for LD centered ∼0.1, 0.4, and 0.8. Misestimation without multivariate estimation can be twice as severe for very large effect alleles, although these only account for a very small fraction of all simulated alleles.</p><p id="p-42">Similar trends are seen for the 3:1 scenario, where one of the minor alleles operated in the opposite direction to the other three, as well as in the 2:2 scenario (Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.pdf">File S1</a>), and simulations with just two or three causal variants yield similar conclusions (Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.pdf">File S1</a>). The advantage of joint modeling is reduced in the presence of opposing allelic effects, but still prevalent in the region with high allelic effect size and low LD; and, as noted above, a large proportion of causal sites are not discovered in the 2:2 scenario, so are not included in the estimation.</p><p id="p-43">In summary, stepwise conditional eQTL discovery is expected to discover between 70 and 80% of eQTL within realistic effect size ranges typical of those reported in the literature. Once discovered, multi-locus estimation of effect sizes provides slightly more accurate estimates than the estimates from sequential conditional models, but for large effect alleles in high LD the corrections can be substantial.</p></div><div id="sec-11" class="subsection"><h3>Bayesian modeling only slightly improves mapping of multisite associations</h3><p id="p-44">Recently, a number of Bayesian approaches have been introduced that are designed to improve fine mapping of eQTL effects (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Giambartolomei <em>et al.</em> 2014</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Zhou <em>et al</em>. 2013</a>). One of these is eCAVIAR (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Hormozdiari <em>et al.</em> 2016</a>), which reports a CLPP based on the combined likelihood that a variant influences both the abundance of a transcript and a phenotype given the LD structure at a locus. The authors proposed a CLPP cutoff of 0.001 (for example, a posterior probability of 0.1 for the eQTL and 0.01 for a disease association), which corresponds in our simulations (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) to discovery of 80.7% of single variants sampled at random from contiguous blocks of 100% SNPs in the CAGE European-ancestry cohort genotype data. The computational burden of evaluating all possible four site combinations is too large for this model to be applied in genome-wide scans. Instead, we performed 4000 simulations of 1835 individuals in the presence of two or three regulatory variants, as well as a normally distributed phenotype, and evaluated the CLPP distributions. In the case of two causal variants, just 94.7% generated CLPP &gt; 0.001, and for three causal variants, 84.9%. <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the cumulative distribution functions of the CLPP scores as a function of the number of causal variants, clearly documenting the trend for reduced confidence in joint localization as the degree of multisite regulation increases. Similar trends were seen with a more conservative CLPP cutoff of 0.01, confirming that interference among tightly linked sites reduces the power to detect independent causal variants. The upper red curve also indicates that the power to detect colocalization is greatly reduced with sample sizes of just 200 for the eQTL sampling: in fact, just 60% of simulations with a single causal site yielded a CLPP &gt; 0.001.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486006/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486006/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Effect of multisite regulation on colocalization of eQTL with eCAVIAR</span><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Colocalization with eCAVIAR in the presence of multiple regulatory sites. Cumulative Distribution Functions summarize CLPP scores for 4000 simulations each with one, two, or three assigned causal sites within a contiguous block of 100 SNPs. Green, purple, and teal curves show progressive degradation of evidence as the number of modeled causal variants increases, for simulations with 1835 subjects. The red curve shows that more than half of the simulations with just 200 subjects for the eQTL component have CLPP &lt; 0.01, and just one-quarter &gt;0.1, compared with one quarter &gt;0.98 with 1835 subjects. CLPP, Combined Likelihood Posterior Probabilities; eQTL, expression quantitative trait locus; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1647342842" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Colocalization with eCAVIAR in the presence of multiple regulatory sites. Cumulative Distribution Functions summarize CLPP scores for 4000 simulations each with one, two, or three assigned causal sites within a contiguous block of 100 SNPs. Green, purple, and teal curves show progressive degradation of evidence as the number of modeled causal variants increases, for simulations with 1835 subjects. The red curve shows that more than half of the simulations with just 200 subjects for the eQTL component have CLPP &lt; 0.01, and just one-quarter &gt;0.1, compared with one quarter &gt;0.98 with 1835 subjects. CLPP, Combined Likelihood Posterior Probabilities; eQTL, expression quantitative trait locus; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2533/F6.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/8/2533/F6.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485999" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-46" class="first-child">Colocalization with eCAVIAR in the presence of multiple regulatory sites. Cumulative Distribution Functions summarize CLPP scores for 4000 simulations each with one, two, or three assigned causal sites within a contiguous block of 100 SNPs. Green, purple, and teal curves show progressive degradation of evidence as the number of modeled causal variants increases, for simulations with 1835 subjects. The red curve shows that more than half of the simulations with just 200 subjects for the eQTL component have CLPP &lt; 0.01, and just one-quarter &gt;0.1, compared with one quarter &gt;0.98 with 1835 subjects. CLPP, Combined Likelihood Posterior Probabilities; eQTL, expression quantitative trait locus; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">Since eCAVIAR is not designed to cover intervals encompassing all of the regulatory regions of a typical gene and hence is not directly comparable with the stepwise conditional regression, we also evaluated the DAP algorithm (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Wen <em>et al.</em> 2016</a>). DAP is designed to identify independent credible intervals and report candidate SNPs across a locus, incorporating priors that weight likely functional or evolutionary evidence. Using the adaptive DAP procedure, which does not make any prior assumptions about the number of causal variants at a locus, we performed 130 simulations of four variants with effect sizes drawn as before to explain between 2 and 10% of the variance, all operating in the same direction. The average number of detected variants was 3.76, which included 3.52 of the four simulated sites (88%) in the candidate list. Due to different statistical thresholds, it is difficult to compare this result with the stepwise conditional model, but it appears to be an improvement on the 80% reported in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>.</p><p id="p-48">As expected, DAP fails to detect true causal variants in the presence of high LD. <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a> shows the dependency of the number of discovered variants on the mean LD between the four simulated sites in each simulation, while <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a> shows how many of the true causal variants are detected. Notably, if all four variants are in a single block of high LD, no sites are detected since the posterior probabilities are dispersed across all of the variants. Across the full extent of up to 500 kb at most loci there are usually multiple LD blocks, so DAP, like stepwise conditional modeling, is quite efficient at detecting independent credible intervals. However, it consistently overreports the number of candidate variants and <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7C</a> shows that this number also increases with LD. To confirm that DAP is still able to resolve multiple causal variants in the presence of high LD, we also ran 400 simulations with the constraint that three sites must be within <em>r</em><sup>2</sup> &gt; 0.3, using the DAP-k algorithm with <em>k</em> = 3 (assuming three sites), showing the results in <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7, D–F</a>. In this case, the number of detected independent associations dropped to 1.95, namely 65% of the simulated number. Although 79% of the simulated variants were among the candidate lists, these can become very large with a ratio of ten-to-one candidates for each true causal variant. Stepwise conditional analyses on the same simulations with a cutoff of <em>P</em> &lt; 10<sup>−5</sup> discovered on average 1.56 (52%) of the three simulated effects and included 2.29 (76%) of the sites within the credible interval. Consequently, DAP does appear to improve performance, at the cost of a considerably higher computational burden.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fine mapping with DAP in the presence of multiple regulatory sites. (A&#x2013;C) Results for simulations with four causal sites drawn at random from 200 kb upstream and downstream of each gene. (D&#x2013;E) Results for simulations with three causal sites drawn from 100 continuous SNPs, each in LD with r2 &gt; 0.3. (A and D) show the estimated number of sites as a function of the mean LD between the sites, showing that as LD increases, detection of independent intervals decreases. (B and E) show the number of modeled (true) causal sites in the candidate lists, as a function of mean LD, which in this case increases for high LD. (C and F) show that the number of candidate sites increases with high LD, sometimes with 20 or more candidates defining a credible set for each true site. DAP, Deterministic Approximation of Posteriors; LD, linkage disequilibrium." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1647342842" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fine mapping with DAP in the presence of multiple regulatory sites. (A&#x2013;C) Results for simulations with four causal sites drawn at random from 200 kb upstream and downstream of each gene. (D&#x2013;E) Results for simulations with three causal sites drawn from 100 continuous SNPs, each in LD with &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; &gt; 0.3. (A and D) show the estimated number of sites as a function of the mean LD between the sites, showing that as LD increases, detection of independent intervals decreases. (B and E) show the number of modeled (true) causal sites in the candidate lists, as a function of mean LD, which in this case increases for high LD. (C and F) show that the number of candidate sites increases with high LD, sometimes with 20 or more candidates defining a credible set for each true site. DAP, Deterministic Approximation of Posteriors; LD, linkage disequilibrium.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2533/F7.medium.gif" width="440" height="293"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/8/2533/F7.medium.gif" width="440" height="293"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2533/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486108" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-49" class="first-child">Fine mapping with DAP in the presence of multiple regulatory sites. (A–C) Results for simulations with four causal sites drawn at random from 200 kb upstream and downstream of each gene. (D–E) Results for simulations with three causal sites drawn from 100 continuous SNPs, each in LD with <em>r</em><sup>2</sup> &gt; 0.3. (A and D) show the estimated number of sites as a function of the mean LD between the sites, showing that as LD increases, detection of independent intervals decreases. (B and E) show the number of modeled (true) causal sites in the candidate lists, as a function of mean LD, which in this case increases for high LD. (C and F) show that the number of candidate sites increases with high LD, sometimes with 20 or more candidates defining a credible set for each true site. DAP, Deterministic Approximation of Posteriors; LD, linkage disequilibrium.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-50">Studies of the genetic regulation of gene expression are making a meaningful contribution to the interpretation of GWAS results, as they provide functional insight into the nature of the causal genes. However, efforts to fine map causal variants are complicated by the limits of statistical resolution as it is not uncommon for tens, if not hundreds, of polymorphisms in a credible set to have similar statistical support (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gaulton <em>et al.</em> 2015</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kichaev and Pasaniuc 2015</a>). Inclusion of experimental evidence from epigenetic marks or signatures of evolutionary conservation into scores such as CADD (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kircher <em>et al.</em> 2014</a>), CATO (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Maurano <em>et al.</em> 2015</a>), and LINSIGHT (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2017</a>) promises to improve resolution, as do methods such as RTC (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Nica <em>et al.</em> 2010</a>) and PICS (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Farh <em>et al.</em> 2015</a>), which prioritize variants based on the structure of LD at a locus. In general, these approaches assume parsimony, namely that there is a single variant that is responsible for the major GWAS or eQTL signature. Although it has become increasingly clear that many loci harbor multiple independent regulatory variants, we argue here that if the parsimony assumption is relaxed and it is assumed that multiple sites in strong LD commonly account for a signature that is compounded into a single significant association, then the estimates from sequential conditional analysis can be highly biased. To summarize, we find that &gt;5% of primary sites and more than one-fifth of all causal sites are unlikely to be tagged at all; that in the presence of multisite regulation at least 15% of all mapped sites are not in strong LD with any of the multiple imputed causal variants at a locus; and that another 10% of the associations are plausibly due to splitting of the signal due to an unimputed site. Taken together with increasing evidence that up to a third of all eGenes have two or more independent eSNPS (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gusev <em>et al.</em> 2014</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Wen <em>et al.</em> 2015</a>; <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Lloyd-Jones <em>et al.</em> 2017</a>), these results suggest that at least 5% and perhaps as many as a one-quarter of mapped credible intervals may not include the actual causal variant.</p><p id="p-51">Theory and simulation both indicate that if two linked sites both influence a trait, including gene expression, then multisite models will uniformly outperform sequential uni-site ones with regard to estimation of the true effect size. When the identities of the variants are known, sample sizes of several thousand individuals are sufficient to jointly estimate their effects with high accuracy even in the presence of high levels of LD with <em>r</em><sup>2</sup> up to, or even exceeding, 0.9. The problem is that the identities of the variants are generally not known, and there are no established methods for comprehensive screening transcriptome-wide for localization of multi-locus local eQTL effects. Two exhaustive search algorithms, PAINTOR (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kichaev <em>et al.</em> 2014</a>) and CAVIARBF (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> 2015</a>), hold promise for detailed dissection of multisite models at individual loci, and a Bayesian shotgun stochastic search algorithm, FINEMAP (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Benner <em>et al.</em> 2016</a>), has recently been proposed for rapid maximum likelihood estimation of multi-SNP contributions. Here, we show by simulation that DAP (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Wen <em>et al.</em> 2016</a>) does indeed improve on sequential conditional analysis for localization of multiple linked causal variants, but that in regions of high LD its performance remains constrained. Given the computational burden, it may be more efficient to use stepwise modeling, perhaps supplemented with a lasso regression method, to map independent sites in low LD, and then concentrate on each credible interval with these Bayesian fine mapping tools. It should also be recognized that single site effects may sometimes be artificially split into two or more linked contributions under each of these strategies.</p><p id="p-52">We also estimated the bias in the estimates from conditional analysis, by fitting multi-locus linear models to all of the discovered eSNPs at each locus. This revealed only modest improvements in accuracy for most of the discovered sites, but the modesty is in part an artifact of the discovery bias introduced by the sequential conditional process. Our simulations assuming two to four effective sites per locus across a wide and representative range of LD, show that in a sample size of 2000, in general no more than 85% of the simulated causal sites are tagged by discovered associations that explain typically observed magnitudes of effect and would almost always be detected if a single site explained the variance. Similarly, <a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Lloyd Jones <em>et al.</em> (2017)</a> estimate that, in the CAGE dataset, on average between 50 and 75% of the heritability due to locally acting regulatory polymorphism can be attributed to discovered variants. Multisite modeling readjusts the remaining estimates typically by between 0.1 and 0.5 sdu, which, depending on the allele frequency, accounts for between 2 and 5% of the variance explained, and only rarely &gt;10%.</p><p id="p-53">However, any variants with effect sizes &gt;1 sdu, and whose average <em>r</em><sup>2</sup> with the other three SNPs is &gt;0.9, will be misestimated in both the single site and joint models, typically by 1.5 sdu or more. The misestimation is on average the greatest where all of the effects are in the same direction, but is consistently observed also in the presence of associations with alternate signs. Unavoidably, the sequential conditional estimates of eQTL effects are actually highly biased for a considerable proportion of variants. Although this does not impact the total amount of variance explained by the discovered variants, it is likely to greatly impact fine mapping efforts, particularly where two or more effects are collapsed into one site in a credible interval.</p><p id="p-54">While large datasets have very good power for detection of complex regulatory contributions for individual genes, there are a host of technical and statistical reasons why fine mapping of causal variants remains a challenge. There are two immediate strong implications of these results. One is that even though the majority of identified eSNPs are expected to map to credible intervals that include the causal variant (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Gusev <em>et al.</em> 2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Finucane <em>et al.</em> 2015</a>), there will also be many instances where incongruence between the statistical interval and chromatin or other functional evidence (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Huang <em>et al.</em> 2015</a>) is to be expected. The causal variant may simply be poorly mapped due to interference among linked functional sites. This effect may also influence the fine mapping of pleiotropic associations (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fortune <em>et al.</em> 2015</a>). The second implication of the high frequency of multisite regulation is to emphasize caution in using univariate statistical support for an eSNP effect as sufficient evidence that an association between a SNP and a trait is evidence for causation. At a minimum, it is imperative that the full spectrum of eSNP effects across the locus be evaluated to confirm that the site is not simply in LD with higher likelihood eSNPs that are not themselves associated with the trait. Experimental validation of individual sites seems warranted in situations where establishment of the identity of the causal variant(s) is desired.</p></div><div class="section" id="sec-13"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-55">Assembly of the CAGE cohort was supported by Australian NH&amp;MRC Grant 1046880 to P.V. G.G. is partially supported by US NIH Grants P01-GM099568 and R01-HG008146, and B.Z. was the recipient of a Chinese Scholarship Council pre-doctoral fellowship.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-56">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043752/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043752/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-57"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-04-11"><span class="received-label">Received </span>April 11, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-05-31"><span class="accepted-label">Accepted </span>May 31, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Zeng <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.1" data-doi="10.1038/nrg3891"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The role of regulatory variation in complex traits and disease.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3891%26rft_id%253Dinfo%253Apmid%252F25707927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3891&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25707927&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.2" data-doi="10.1093/bioinformatics/btw018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Havulinna</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salomaa</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripatti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">FINEMAP: efficient variable selection using summary data from genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1493</span>–<span class="cit-lpage">1501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw018%26rft_id%253Dinfo%253Apmid%252F26773131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26773131&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.3" data-doi="10.1534/genetics.115.176107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larrabee</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ovsyannikova</span>  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haralambieva</span>  <span class="cit-name-given-names">I. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fine mapping causal variants with an approximate Bayesian method using marginal test statistics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.176107%26rft_id%253Dinfo%253Apmid%252F25948564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzMvNzE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casparino</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patsopoulos</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croteau-Chonka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raby</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Limited statistical evidence for shared genetic effects of eQTLs and autoimmune disease-associated loci in three major immune-cell types.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">600</span>–<span class="cit-lpage">605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.5" data-doi="10.1371/journal.pgen.1004787"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelernter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">GPA: a statistical approach to prioritizing GWAS results by integrating pleiotropy and annotation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004787%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004787%26rft_id%253Dinfo%253Apmid%252F25393678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004787&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25393678&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.6" data-doi="10.1016/j.pbi.2012.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coustham</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loudet</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Lessons from eQTL mapping studies: non-coding regions and their role behind natural phenotypic variation in plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DCubillos%26rft.auinit1%253DF.%2BA.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D192%26rft.epage%253D198%26rft.atitle%253DLessons%2Bfrom%2BeQTL%2Bmapping%2Bstudies%253A%2Bnon-coding%2Bregions%2Band%2Btheir%2Brole%2Bbehind%2Bnatural%2Bphenotypic%2Bvariation%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2012.01.005%26rft_id%253Dinfo%253Apmid%252F22265229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2012.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22265229&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.7" data-doi="10.1371/journal.pbio.1000294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickson</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krantz</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rare variants create synthetic genome-wide associations.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDickson%26rft.auinit1%253DS.%2BP.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253De1000294%26rft.epage%253De1000294%26rft.atitle%253DRare%2Bvariants%2Bcreate%2Bsynthetic%2Bgenome-wide%2Bassociations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000294%26rft_id%253Dinfo%253Apmid%252F20126254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20126254&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.8" data-doi="10.1038/nature11247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">ENCODE Project Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated encyclopedia of DNA elements in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBernstein%26rft.auinit1%253DB.%2BE.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D57%26rft.epage%253D74%26rft.atitle%253DAn%2Bintegrated%2Bencyclopedia%2Bof%2BDNA%2Belements%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11247%26rft_id%253Dinfo%253Apmid%252F22955616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955616&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.9" data-doi="10.1038/nature13835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farh</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleineweitfeld</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Housley</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic and epigenetic fine mapping of causal autoimmune disease variants.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">518</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D337%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13835%26rft_id%253Dinfo%253Apmid%252F25363779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25363779&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.10" data-doi="10.1038/ng.3404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finucane</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bulik-Sullivan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trynka</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reshef</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Partitioning heritability by functional annotation using genome-wide association summary statistics.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1228</span>–<span class="cit-lpage">1235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D1228%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3404%26rft_id%253Dinfo%253Apmid%252F26414678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26414678&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.11" data-doi="10.1038/ng.3330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fortune</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burren</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schofield</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Statistical colocalization of genetic risk variants for related autoimmune diseases in the context of common controls.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">839</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D839%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3330%26rft_id%253Dinfo%253Apmid%252F26053495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26053495&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.12" data-doi="10.1038/ng.3437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaulton</span>  <span class="cit-name-given-names">K. J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raimono</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mägi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic fine mapping and genomic annotation defines causal mechanisms at type 2 diabetes susceptibility loci.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1415</span>–<span class="cit-lpage">1425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D1415%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3437%26rft_id%253Dinfo%253Apmid%252F26551672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26551672&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.13" data-doi="10.1371/journal.pgen.1004383"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giambartolomei</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vukcevic</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schadt</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franke</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hingorani</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Bayesian test for colocalisation between pairs of genetic association studies using summary statistics.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004383%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004383%26rft_id%253Dinfo%253Apmid%252F24830394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004383&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24830394&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.14" data-doi="10.1016/j.ajhg.2014.10.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trynka</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finucane</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjálmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Partitioning heritability of regulatory and cell-type-specific variants across 11 common diseases.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D95%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.10.004%26rft_id%253Dinfo%253Apmid%252F25439723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25439723&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.15" data-doi="10.1534/genetics.114.167908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hormozdiari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kostem</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identifying causal variants at loci with multiple signals of association.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">508</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.167908%26rft_id%253Dinfo%253Apmid%252F25104515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hormozdiari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Bunt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segre</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joo</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Colocalization of GWAS and eQTL signals detects target genes.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1245</span>–<span class="cit-lpage">1260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D99%26rft.spage%253D1245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.17" data-doi="10.1038/ng.2354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchsberger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%26rft.volume%253D44%26rft.issue%253D8%26rft.spage%253D955%26rft.epage%253D959%26rft.atitle%253DFast%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bin%2Bgenome-wide%2Bassociation%2Bstudies%2Bthrough%2Bpre-phasing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2354%26rft_id%253Dinfo%253Apmid%252F22820512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820512&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-web" id="cit-7.8.2533.18"><div class="cit-metadata unstructured">Huang, H., M. Fang, L. Jostins, M.U. Mirkov, G. Boucher <em>et al.</em>, 2015 Association mapping of inflammatory bowel disease loci to single variant resolution. bioRxiv. Available at: <a href="http://biorxiv.org/content/early/2015/10/20/028688">http://biorxiv.org/content/early/2015/10/20/028688</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gulko</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siepel</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Fast, scalable prediction of deleterious non-coding variants from functional and population genomic data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">618</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.20" data-doi="10.1038/ng.495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Idaghdour</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czika</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shianna</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Geographical genomics of human leukocyte gene expression variation in southern Morocco.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DIdaghdour%26rft.auinit1%253DY.%26rft.volume%253D42%26rft.issue%253D1%26rft.spage%253D62%26rft.epage%253D67%26rft.atitle%253DGeographical%2Bgenomics%2Bof%2Bhuman%2Bleukocyte%2Bgene%2Bexpression%2Bvariation%2Bin%2Bsouthern%2BMorocco.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.495%26rft_id%253Dinfo%253Apmid%252F19966804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19966804&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273055100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.21" data-doi="10.1016/j.ajhg.2015.06.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kichaev</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Leveraging functional-annotation data in trans-ethnic fine-mapping studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">260</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D97%26rft.spage%253D260%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.06.007%26rft_id%253Dinfo%253Apmid%252F26189819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.06.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26189819&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.22" data-doi="10.1371/journal.pgen.1004722"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kichaev</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindstrom</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hormozdiari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integrating functional data to prioritize causal variants in statistical fine-mapping studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004722%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004722%26rft_id%253Dinfo%253Apmid%252F25357204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004722&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25357204&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghasemzadeh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eapen</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Gene expression profiles associated with acute myocardial infarction and risk of cardiovascular death.</span> <abbr class="cit-jnl-abbrev">Genome Med.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMed.%26rft.volume%253D6%26rft.spage%253D40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.24" data-doi="10.1038/ng.2892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kircher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witten</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Roak</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A general framework for estimating the relative pathogenicity of human genetic variants.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">310</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D310%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2892%26rft_id%253Dinfo%253Apmid%252F24487276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24487276&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.25" data-doi="10.1038/nature12531"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lappalainen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sammeth</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedländer</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">’t Hoen</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monlong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transcriptome and genome sequencing uncovers functional variation in humans.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">501</span>: <span class="cit-fpage">506</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D501%26rft.spage%253D506%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12531%26rft_id%253Dinfo%253Apmid%252F24037378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12531&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24037378&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324826300049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.26" data-doi="10.1101/gr.142521.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morar</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lathrop</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A cross-platform analysis of 14,177 expression quantitative trait loci derived from lymphoblastoid cell lines.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">716</span>–<span class="cit-lpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.142521.112%26rft_id%253Dinfo%253Apmid%252F23345460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzQvNzE2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.27" data-doi="10.1016/j.ajhg.2016.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maruthur</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kabagambe</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Trans-ethnic meta-analysis and functional annotation illuminates the genetic architecture of fasting glucose and insulin.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D99%26rft.spage%253D56%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2016.05.006%26rft_id%253Dinfo%253Apmid%252F27321945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2016.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27321945&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lloyd-Jones</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holloway</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakshi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McRae</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The genetic architecture of gene expression in peripheral blood.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">228</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D100%26rft.spage%253D228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.29" data-doi="10.1126/science.1222794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maurano</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humbert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rynes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurman</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haugen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Systematic localization of common disease-associated variation in regulatory DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">1190</span>–<span class="cit-lpage">1195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1222794%26rft_id%253Dinfo%253Apmid%252F22955828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzcvNjA5OS8xMTkwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI1MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.30" data-doi="10.1038/ng.3432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maurano</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haugen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandstrom</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vierstra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Large-scale identification of sequence variants influencing human transcription factor occupancy <em>in vivo</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1393</span>–<span class="cit-lpage">1401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D1393%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3432%26rft_id%253Dinfo%253Apmid%252F26502339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26502339&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.31" data-doi="10.1038/nature09266"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Musunuru</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frank-Kamenetsky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahfeldt</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">From noncoding variant to phenotype via <em>SORT1</em> at the 1p13 cholesterol locus.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">466</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMusunuru%26rft.auinit1%253DK.%26rft.volume%253D466%26rft.issue%253D7307%26rft.spage%253D714%26rft.epage%253D719%26rft.atitle%253DFrom%2Bnoncoding%2Bvariant%2Bto%2Bphenotype%2Bvia%2BSORT1%2Bat%2Bthe%2B1p13%2Bcholesterol%2Blocus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09266%26rft_id%253Dinfo%253Apmid%252F20686566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09266&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20686566&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280562500030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.32" data-doi="10.1371/journal.pgen.1000895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nica</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dimas</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stranger</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beazlev</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Candidate causal regulatory effects by integration of expression QTLs with complex trait genetic associations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000895</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DNica%26rft.auinit1%253DA.%2BC.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De1000895%26rft.epage%253De1000895%26rft.atitle%253DCandidate%2Bcausal%2Bregulatory%2Beffects%2Bby%2Bintegration%2Bof%2Bexpression%2BQTLs%2Bwith%2Bcomplex%2Btrait%2Bgenetic%2Bassociations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000895%26rft_id%253Dinfo%253Apmid%252F20369022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000895&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20369022&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.33" data-doi="10.1371/journal.pgen.1000888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicolae</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gamazon</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolan</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Trait-associated SNPs are more likely to be eQTLs: annotation to enhance discovery from GWAS.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DNicolae%26rft.auinit1%253DD.%2BL.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De1000888%26rft.epage%253De1000888%26rft.atitle%253DTrait-associated%2BSNPs%2Bare%2Bmore%2Blikely%2Bto%2Bbe%2BeQTLs%253A%2Bannotation%2Bto%2Benhance%2Bdiscovery%2Bfrom%2BGWAS.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000888%26rft_id%253Dinfo%253Apmid%252F20369019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000888&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20369019&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Joint analysis of functional genomic data and genome-wide association studies of 18 human traits.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">573</span> (erratum: Am. J. Hum. Genet. <span class="cit-vol">95</span>: 126).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D95%26rft.spage%253D559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.35" data-doi="10.1371/journal.pone.0035430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McRae</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caracella</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Brisbane Systems Genetics Study: genetical genomics meets complex trait genetics.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e35430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPowell%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De35430%26rft.epage%253De35430%26rft.atitle%253DThe%2BBrisbane%2BSystems%2BGenetics%2BStudy%253A%2Bgenetical%2Bgenomics%2Bmeets%2Bcomplex%2Btrait%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0035430%26rft_id%253Dinfo%253Apmid%252F22563384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0035430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22563384&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.36" data-doi="10.1371/journal.pgen.1003502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McRae</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hemani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Congruence of additive and non-additive effects on gene expression estimated from pedigree and SNP data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003502%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003502%26rft_id%253Dinfo%253Apmid%252F23696747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23696747&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.37" data-doi="10.1038/nature14248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Roadmap Epigenomics Consortium</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Integrative analysis of 111 reference human epigenomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">518</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D317%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14248%26rft_id%253Dinfo%253Apmid%252F25693563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25693563&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schramm</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marzi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schurmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carstensen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinmaa</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mapping the genetic architecture of gene regulation in whole blood.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e93844</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De93844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.39" data-doi="10.1371/journal.pgen.0030114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Servin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Imputation-based analysis of association studies: candidate regions and quantitative traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DServin%26rft.auinit1%253DB.%26rft.volume%253D3%26rft.issue%253D7%26rft.spage%253De114%26rft.epage%253De114%26rft.atitle%253DImputation-based%2Banalysis%2Bof%2Bassociation%2Bstudies%253A%2Bcandidate%2Bregions%2Band%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030114%26rft_id%253Dinfo%253Apmid%252F17676998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17676998&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tewhey</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kotliar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winnicki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Direct identification of hundreds of expression-modulating variants using a multiplexed reporter assay.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">1519</span>–<span class="cit-lpage">1529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D165%26rft.spage%253D1519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.41" data-doi="10.1038/ng.2504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trynka</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandor</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stranger</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Chromatin marks identify critical cell types for fine mapping complex trait variants.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">124</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D124%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2504%26rft_id%253Dinfo%253Apmid%252F23263488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2504&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23263488&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.42" data-doi="10.1002/gepi.20504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Udler</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyrer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Easton</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evaluating the power to discriminate between highly correlated SNPs in genetic association studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DUdler%26rft.auinit1%253DM.%2BS.%26rft.volume%253D34%26rft.issue%253D5%26rft.spage%253D463%26rft.epage%253D468%26rft.atitle%253DEvaluating%2Bthe%2Bpower%2Bto%2Bdiscriminate%2Bbetween%2Bhighly%2Bcorrelated%2BSNPs%2Bin%2Bgenetic%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20504%26rft_id%253Dinfo%253Apmid%252F20583289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20504&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20583289&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.43" data-doi="10.1371/journal.pgen.1005176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luca</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pique-Regi</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Cross-population joint analysis of eQTLs: fine mapping and functional annotation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005176%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005176%26rft_id%253Dinfo%253Apmid%252F25906321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25906321&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luca</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pique-Regi</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Efficient integrative multi-SNP association analysis via deterministic approximation of posteriors.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">1114</span>–<span class="cit-lpage">1129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D1114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.45" data-doi="10.1038/ng.2756"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westra</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yaghootkar</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schurmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Systematic identification of trans eQTLs as putative drivers of known disease associations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1238</span>–<span class="cit-lpage">1243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D1238%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2756%26rft_id%253Dinfo%253Apmid%252F24013639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2756&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24013639&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wingo</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Blood gene expression profiles suggest altered immune function associated with symptoms of generalized anxiety disorder.</span> <abbr class="cit-jnl-abbrev">Brain Behav. Immun.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrain%2BBehav.%2BImmun.%26rft.volume%253D43%26rft.spage%253D184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.47" data-doi="10.1038/ng.3097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vedantam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pers</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Defining the role of common variation in the genomic and biological architecture of adult human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D1173%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3097%26rft_id%253Dinfo%253Apmid%252F25282103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25282103&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.48" data-doi="10.1038/ng.2951"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Heritability and genomics of gene expression in peripheral blood.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">430</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D430%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2951%26rft_id%253Dinfo%253Apmid%252F24728292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2951&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24728292&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.49" data-doi="10.1038/ng.2213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medland</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth cit-collab">GIANT Consortium</span>, </li><li><span class="cit-auth cit-collab">DIAGRAM Consortium</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D44%26rft.spage%253D369%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2213%26rft_id%253Dinfo%253Apmid%252F22426310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22426310&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.50" data-doi="10.1038/ng.3390"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakshi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hemani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinkhuyzen</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic variance estimation with imputed variants finds negligible missing heritability for human height and body mass index.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1114</span>–<span class="cit-lpage">1120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D1114%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3390%26rft_id%253Dinfo%253Apmid%252F26323059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3390&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26323059&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akinsanmi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arafat</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cradick</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A burden of rare variants associated with extremes of gene expression in human peripheral blood.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.52" data-doi="10.1371/journal.pgen.1003264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbonetto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Polygenic modeling with Bayesian sparse linear mixed models.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003264%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003264%26rft_id%253Dinfo%253Apmid%252F23408905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003264&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23408905&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2533.53" data-doi="10.1016/j.ajhg.2012.07.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heinzen</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickson</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Prioritizing genetic variants for causality on the basis of preferential linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DZhu%26rft.auinit1%253DQ.%26rft.volume%253D91%26rft.issue%253D3%26rft.spage%253D422%26rft.epage%253D434%26rft.atitle%253DPrioritizing%2Bgenetic%2Bvariants%2Bfor%2Bcausality%2Bon%2Bthe%2Bbasis%2Bof%2Bpreferential%2Blinkage%2Bdisequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2012.07.010%26rft_id%253Dinfo%253Apmid%252F22939045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2012.07.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22939045&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2533.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/8/2533.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2519" title="Genetic Dissection of Morphometric Traits Reveals That Phytochrome B Affects Nucleus Size and Heterochromatin Organization in Arabidopsis thaliana" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2545" title="Multidimensional Genetic Analysis of Repeated Seizures in the Hybrid Mouse Diversity Panel Reveals a Novel Epileptogenesis Susceptibility Locus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/8"><span>Volume 7 Issue 8, August 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/8.cover-source.jpg?itok=apH_YDuu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2533" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-I8XOkmxLsts-bJvf1Da1wfmOvViZ4gsrJRqAkPW4D6Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/8/2533" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/8/2533" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/8/2533" class="active" data-icon-position="" data-hide-link-title="0">Constraints on eQTL Fine Mapping in the Presence of Multisite Local Regulation of Gene Expression</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485850" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-gX6Ffr-PMMj8Ekw6TB3q8_qNJDatQT4_MXLVNYe5ZGs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2533.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/8/2533" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-lZzmZc67tlRy9U5WhWzpFM6yK8pGdeCvY27OMHA6sas" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/8/2533" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2533.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485850" id="node485850--2" data-pisa="ggg;7/8/2533" data-pisa-master="ggg;g3.117.043752" data-apath="/ggg/7/8/2533.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Constraints on eQTL Fine Mapping in the Presence of Multisite Local Regulation of Gene Expression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Biao Zeng</span>, <span class="highwire-citation-author" data-delta="1">Luke R. Lloyd-Jones</span>, <span class="highwire-citation-author" data-delta="2">Alexander Holloway</span>, <span class="highwire-citation-author" data-delta="3">Urko M. Marigorta</span>, <span class="highwire-citation-author" data-delta="4">Andres Metspalu</span>, <span class="highwire-citation-author" data-delta="5">Grant W. Montgomery</span>, <span class="highwire-citation-author" data-delta="6">Tonu Esko</span>, <span class="highwire-citation-author" data-delta="7">Kenneth L. Brigham</span>, <span class="highwire-citation-author" data-delta="8">Arshed A. Quyyumi</span>, <span class="highwire-citation-author" data-delta="9">Youssef Idaghdour</span>, <span class="highwire-citation-author" data-delta="10">Jian Yang</span>, <span class="highwire-citation-author" data-delta="11">Peter M. Visscher</span>, <span class="highwire-citation-author" data-delta="12">Joseph E. Powell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-5352-5877" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Greg Gibson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2533-2544;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043752 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485850--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Biao Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences and Center for Integrative Genomics, Georgia Institute of Technology, Atlanta, Georgia 30332</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Biao%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABiao%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Luke R. Lloyd-Jones </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luke%2BR.%2BLloyd-Jones%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lloyd-Jones%20LR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuke%2BR.%2BLloyd-Jones%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexander Holloway </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BHolloway%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holloway%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BHolloway%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Urko M. Marigorta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences and Center for Integrative Genomics, Georgia Institute of Technology, Atlanta, Georgia 30332</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Urko%2BM.%2BMarigorta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marigorta%20UM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUrko%2BM.%2BMarigorta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andres Metspalu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Estonian Genome Center, University of Tartu, 50090, Estonia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andres%2BMetspalu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Metspalu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndres%2BMetspalu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Grant W. Montgomery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grant%2BW.%2BMontgomery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montgomery%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGrant%2BW.%2BMontgomery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Tonu Esko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Estonian Genome Center, University of Tartu, 50090, Estonia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tonu%2BEsko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Esko%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATonu%2BEsko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kenneth L. Brigham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Medicine, Emory University, Atlanta, Georgia 30329</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BL.%2BBrigham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brigham%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenneth%2BL.%2BBrigham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Arshed A. Quyyumi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Medicine, Emory University, Atlanta, Georgia 30329</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arshed%2BA.%2BQuyyumi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Quyyumi%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArshed%2BA.%2BQuyyumi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Youssef Idaghdour </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Biology, New York University Abu Dhabi, Saadiyat Island, United Arab Emirates</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Youssef%2BIdaghdour%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Idaghdour%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoussef%2BIdaghdour%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jian Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJian%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Peter M. Visscher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BM.%2BVisscher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Visscher%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BM.%2BVisscher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Joseph E. Powell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BE.%2BPowell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Powell%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BE.%2BPowell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Greg Gibson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences and Center for Integrative Genomics, Georgia Institute of Technology, Atlanta, Georgia 30332</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Greg%2BGibson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gibson%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGreg%2BGibson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5352-5877" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Greg Gibson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:greg.gibson@biology.gatech.edu" class="" data-icon-position="" data-hide-link-title="0">greg.gibson@biology.gatech.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/8/2533" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/8/2533&amp;title=Constraints%20on%20eQTL%20Fine%20Mapping%20in%20the%20Presence%20of%20Multisite%20Local%20Regulation%20of%20Gene%20Expression" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/8/2533&amp;title=Constraints%20on%20eQTL%20Fine%20Mapping%20in%20the%20Presence%20of%20Multisite%20Local%20Regulation%20of%20Gene%20Expression" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/8/2533&amp;title=Constraints%20on%20eQTL%20Fine%20Mapping%20in%20the%20Presence%20of%20Multisite%20Local%20Regulation%20of%20Gene%20Expression" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/8/2533&amp;text=Constraints%20on%20eQTL%20Fine%20Mapping%20in%20the%20Presence%20of%20Multisite%20Local%20Regulation%20of%20Gene%20Expression" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/8/2533&amp;title=Constraints%20on%20eQTL%20Fine%20Mapping%20in%20the%20Presence%20of%20Multisite%20Local%20Regulation%20of%20Gene%20Expression" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/8/2533&amp;t=Constraints%20on%20eQTL%20Fine%20Mapping%20in%20the%20Presence%20of%20Multisite%20Local%20Regulation%20of%20Gene%20Expression" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/8/2533&amp;title=Constraints%20on%20eQTL%20Fine%20Mapping%20in%20the%20Presence%20of%20Multisite%20Local%20Regulation%20of%20Gene%20Expression" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/8/2533&amp;title=Constraints%20on%20eQTL%20Fine%20Mapping%20in%20the%20Presence%20of%20Multisite%20Local%20Regulation%20of%20Gene%20Expression" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485850/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485850" id="node485850--3" data-pisa="ggg;7/8/2533" data-pisa-master="ggg;g3.117.043752" data-apath="/ggg/7/8/2533.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Constraints on eQTL Fine Mapping in the Presence of Multisite Local Regulation of Gene Expression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Biao Zeng</span>, <span class="highwire-citation-author" data-delta="1">Luke R. Lloyd-Jones</span>, <span class="highwire-citation-author" data-delta="2">Alexander Holloway</span>, <span class="highwire-citation-author" data-delta="3">Urko M. Marigorta</span>, <span class="highwire-citation-author" data-delta="4">Andres Metspalu</span>, <span class="highwire-citation-author" data-delta="5">Grant W. Montgomery</span>, <span class="highwire-citation-author" data-delta="6">Tonu Esko</span>, <span class="highwire-citation-author" data-delta="7">Kenneth L. Brigham</span>, <span class="highwire-citation-author" data-delta="8">Arshed A. Quyyumi</span>, <span class="highwire-citation-author" data-delta="9">Youssef Idaghdour</span>, <span class="highwire-citation-author" data-delta="10">Jian Yang</span>, <span class="highwire-citation-author" data-delta="11">Peter M. Visscher</span>, <span class="highwire-citation-author" data-delta="12">Joseph E. Powell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-5352-5877" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Greg Gibson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2533-2544;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043752 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485850--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Biao Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences and Center for Integrative Genomics, Georgia Institute of Technology, Atlanta, Georgia 30332</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Biao%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABiao%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Luke R. Lloyd-Jones </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luke%2BR.%2BLloyd-Jones%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lloyd-Jones%20LR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuke%2BR.%2BLloyd-Jones%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexander Holloway </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BHolloway%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holloway%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BHolloway%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Urko M. Marigorta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences and Center for Integrative Genomics, Georgia Institute of Technology, Atlanta, Georgia 30332</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Urko%2BM.%2BMarigorta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marigorta%20UM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUrko%2BM.%2BMarigorta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andres Metspalu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Estonian Genome Center, University of Tartu, 50090, Estonia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andres%2BMetspalu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Metspalu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndres%2BMetspalu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Grant W. Montgomery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grant%2BW.%2BMontgomery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montgomery%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGrant%2BW.%2BMontgomery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Tonu Esko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Estonian Genome Center, University of Tartu, 50090, Estonia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tonu%2BEsko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Esko%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATonu%2BEsko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kenneth L. Brigham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Medicine, Emory University, Atlanta, Georgia 30329</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BL.%2BBrigham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brigham%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenneth%2BL.%2BBrigham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Arshed A. Quyyumi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Medicine, Emory University, Atlanta, Georgia 30329</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arshed%2BA.%2BQuyyumi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Quyyumi%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArshed%2BA.%2BQuyyumi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Youssef Idaghdour </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Biology, New York University Abu Dhabi, Saadiyat Island, United Arab Emirates</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Youssef%2BIdaghdour%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Idaghdour%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoussef%2BIdaghdour%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jian Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJian%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Peter M. Visscher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BM.%2BVisscher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Visscher%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BM.%2BVisscher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Joseph E. Powell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Molecular Biosciences, University of Queensland, Brisbane, Queensland 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BE.%2BPowell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Powell%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BE.%2BPowell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Greg Gibson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences and Center for Integrative Genomics, Georgia Institute of Technology, Atlanta, Georgia 30332</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Greg%2BGibson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gibson%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGreg%2BGibson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5352-5877" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Greg Gibson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:greg.gibson@biology.gatech.edu" class="" data-icon-position="" data-hide-link-title="0">greg.gibson@biology.gatech.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2533&amp;rft.epage=2544&amp;rft.atitle=Constraints%20on%20eQTL%20Fine%20Mapping%20in%20the%20Presence%20of%20Multisite%20Local%20Regulation%20of%20Gene%20Expression&amp;rft.volume=7&amp;rft.issue=8&amp;rft.date=2017-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zeng%2C+Biao&amp;rft.au=Lloyd-Jones%2C+Luke+R.&amp;rft.au=Holloway%2C+Alexander&amp;rft.au=Marigorta%2C+Urko+M.&amp;rft.au=Metspalu%2C+Andres&amp;rft.au=Montgomery%2C+Grant+W.&amp;rft.au=Esko%2C+Tonu&amp;rft.au=Brigham%2C+Kenneth+L.&amp;rft.au=Quyyumi%2C+Arshed+A.&amp;rft.au=Idaghdour%2C+Youssef&amp;rft.au=Yang%2C+Jian&amp;rft.au=Visscher%2C+Peter+M.&amp;rft.au=Powell%2C+Joseph+E.&amp;rft.au=Gibson%2C+Greg"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485850/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485850/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485850/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485850/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485850/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485850/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485850/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485850/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485850/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485850/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485850/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485850/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/8/2533" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgements</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/8/2533.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/8/2533.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/8/2533.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/8/2533.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
