<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/8/2685.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/8/2685.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/8/2685/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/8/2685.atom" />
<meta name="HW.pisa" content="ggg;7/8/2685" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Accuracy and Bias of Single-Step Genomic Prediction for Populations Under Selection" />
<meta name="DC.Identifier" content="10.1534/g3.117.043596" />
<meta name="DC.Date" content="2017-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Hsu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In single-step analyses, missing genotypes are explicitly or implicitly imputed, and this requires centering the observed genotypes using the means of the unselected founders. If genotypes are only available for selected individuals, centering on the unselected founder mean is not straightforward. Here, computer simulation is used to study an alternative analysis that does not require centering genotypes but fits the mean ![Formula][1]&lt;/img&gt; of unselected individuals as a fixed effect. Starting with observed diplotypes from 721 cattle, a five-generation population was simulated with sire selection to produce 40,000 individuals with phenotypes, of which the 1000 sires had genotypes. The next generation of 8000 genotyped individuals was used for validation. Evaluations were undertaken with (J) or without (N) ![Formula][2]&lt;/img&gt; when marker covariates were not centered; and with (JC) or without (C) ![Formula][3]&lt;/img&gt; when all observed and imputed marker covariates were centered. Centering did not influence accuracy of genomic prediction, but fitting ![Formula][4]&lt;/img&gt; did. Accuracies were improved when the panel comprised only quantitative trait loci (QTL); models JC and J had accuracies of 99.4%, whereas models C and N had accuracies of 90.2%. When only markers were in the panel, the 4 models had accuracies of 80.4%. In panels that included QTL, fitting ![Formula][5]&lt;/img&gt; in the model improved accuracy, but had little impact when the panel contained only markers. In populations undergoing selection, fitting ![Formula][6]&lt;/img&gt; in the model is recommended to avoid bias and reduction in prediction accuracy due to selection.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif
 [6]: /embed/mml-math-6.gif" />
<meta name="DC.Contributor" content="Wan-Ling Hsu" />
<meta name="DC.Contributor" content="Dorian J. Garrick" />
<meta name="DC.Contributor" content="Rohan L. Fernando" />
<meta name="article:published_time" content="2017-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Accuracy and Bias of Single-Step Genomic Prediction for Populations Under Selection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In single-step analyses, missing genotypes are explicitly or implicitly imputed, and this requires centering the observed genotypes using the means of the unselected founders. If genotypes are only available for selected individuals, centering on the unselected founder mean is not straightforward. Here, computer simulation is used to study an alternative analysis that does not require centering genotypes but fits the mean &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me1&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;μ&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt; of unselected individuals as a fixed effect. Starting with observed diplotypes from 721 cattle, a five-generation population was simulated with sire selection to produce 40,000 individuals with phenotypes, of which the 1000 sires had genotypes. The next generation of 8000 genotyped individuals was used for validation. Evaluations were undertaken with (J) or without (N) &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me2&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;μ&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt; when marker covariates were not centered; and with (JC) or without (C) &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me3&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;μ&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt; when all observed and imputed marker covariates were centered. Centering did not influence accuracy of genomic prediction, but fitting &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me4&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;μ&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt; did. Accuracies were improved when the panel comprised only quantitative trait loci (QTL); models JC and J had accuracies of 99.4%, whereas models C and N had accuracies of 90.2%. When only markers were in the panel, the 4 models had accuracies of 80.4%. In panels that included QTL, fitting &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me5&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;μ&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt; in the model improved accuracy, but had little impact when the panel contained only markers. In populations undergoing selection, fitting &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me6&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;μ&lt;/mi&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt; in the model is recommended to avoid bias and reduction in prediction accuracy due to selection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/08/01" />
<meta name="citation_mjid" content="ggg;7/8/2685" />
<meta name="citation_id" content="7/8/2685" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/8/2685" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/8/2685.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/8/2685.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/8/2685.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.043596" />
<meta name="citation_pmid" content="28642364" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2685" />
<meta name="citation_lastpage" content="2694" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Wan-Ling Hsu" />
<meta name="citation_author_institution" content="Department of Animal Science, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author" content="Dorian J. Garrick" />
<meta name="citation_author_institution" content="Department of Animal Science, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author_institution" content="Institute of Veterinary, Animal and Biomedical Sciences, Massey University, Palmerston North 4442, New Zealand" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8640-5372" />
<meta name="citation_author" content="Rohan L. Fernando" />
<meta name="citation_author_institution" content="Department of Animal Science, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author_email" content="rohan@iastate.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5821-099X" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=D. L. Johnson;citation_author=A. Legarra;citation_author=S. Tsuruta;citation_title=Hot topic: a unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.;citation_pages=743-752;citation_volume=93;citation_year=2010;citation_issue=2;citation_pmid=20105546;citation_doi=10.3168/jds.2009-2730" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=H. Cheng;citation_author=L. Qu;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_title=A fast and efficient Gibbs sampler for BayesB in whole-genome analyses.;citation_pages=80;citation_volume=47;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. F. Christensen;citation_author=M. S. Lund;citation_title=Genomic prediction when some animals are not genotyped.;citation_pages=2-2;citation_volume=42;citation_year=2010;citation_pmid=20105297;citation_doi=10.1186/1297-9686-42-2" />
<meta name="citation_reference" content="citation_journal_abbrev=Animal;citation_author=O. F. Christensen;citation_author=P. Madsen;citation_author=B. Nielsen;citation_author=T. Ostersen;citation_author=G. Su;citation_title=Single-step methods for genomic evaluation in pigs.;citation_pages=1565-1571;citation_volume=6;citation_year=2012;citation_issue=10;citation_pmid=22717310;citation_doi=10.1017/S1751731112000742" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=R. L. Fernando;citation_author=J. C. Dekkers;citation_author=D. J. Garrick;citation_title=A class of Bayesian methods to combine large numbers of genotyped and non-genotyped animals for whole-genome analyses.;citation_pages=50;citation_volume=46;citation_year=2014;citation_pmid=25253441;citation_doi=10.1186/1297-9686-46-50" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_title=Bayesian Methods in Animal Breeding Theory;citation_pages=217-244;citation_volume=63;citation_year=1986;citation_issue=1;citation_doi=10.2527/jas1986.631217x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=B. Goffinet;citation_title=Selection on selected records.;citation_pages=91-98;citation_volume=15;citation_year=1983;citation_doi=10.1051/gse:19830107" />
<meta name="citation_reference" content="citation_author=C. R. Henderson;citation_pages=10-41;citation_year=1973" />
<meta name="citation_reference" content="citation_title=Applications of Linear Models in Animal Breeding;citation_author=C. R. Henderson;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. Im;citation_author=R. L. Fernando;citation_author=D. Gianola;citation_title=Likelihood inferences in animal breeding under selection: a missing-data theory view point.;citation_pages=399-414;citation_volume=21;citation_year=1989;citation_doi=10.1051/gse:19890401" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. Legarra;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=A relationship matrix including full pedigree and genomic information;citation_pages=4656-4663;citation_volume=92;citation_year=2009;citation_issue=9;citation_pmid=19700729;citation_doi=10.3168/jds.2009-2061" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=D. Sorensen;citation_author=R. L. Fernando;citation_author=D. Gianola;citation_title=Inferring the trajectory of genetic variance in the course of artificial selection.;citation_pages=83-94;citation_volume=77;citation_year=2001;citation_issue=1;citation_pmid=11279833;citation_doi=10.1017/S0016672300004845" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=I. Strandén;citation_author=O. F. Christensen;citation_title=Allele coding in genomic evaluation.;citation_pages=25-25;citation_volume=43;citation_year=2011;citation_pmid=21703021;citation_doi=10.1186/1297-9686-43-25" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=Z. G. Vitezica;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=A. Legarra;citation_title=Bias in genomic predictions for populations under selection.;citation_pages=357-366;citation_volume=93;citation_year=2011;citation_issue=5;citation_pmid=21767459;citation_doi=10.1017/S001667231100022X" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Accuracy and Bias of Single-Step Genomic Prediction for Populations Under Selection" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="twitter:description" content="In single-step analyses, missing genotypes are explicitly or implicitly imputed, and this requires centering the observed genotypes using the means of the unselected founders. If genotypes are only available for selected individuals, centering on the unselected founder mean is not straightforward. Here, computer simulation is used to study an alternative analysis that does not require centering genotypes but fits the mean ![Formula][1]&lt;/img&gt; of unselected individuals as a fixed effect. Starting with observed diplotypes from 721 cattle, a five-generation population was simulated with sire selection to produce 40,000 individuals with phenotypes, of which the 1000 sires had genotypes. The next generation of 8000 genotyped individuals was used for validation. Evaluations were undertaken with (J) or without (N) ![Formula][2]&lt;/img&gt; when marker covariates were not centered; and with (JC) or without (C) ![Formula][3]&lt;/img&gt; when all observed and imputed marker covariates were centered. Centering did not influence accuracy of genomic prediction, but fitting ![Formula][4]&lt;/img&gt; did. Accuracies were improved when the panel comprised only quantitative trait loci (QTL); models JC and J had accuracies of 99.4%, whereas models C and N had accuracies of 90.2%. When only markers were in the panel, the 4 models had accuracies of 80.4%. In panels that included QTL, fitting ![Formula][5]&lt;/img&gt; in the model improved accuracy, but had little impact when the panel contained only markers. In populations undergoing selection, fitting ![Formula][6]&lt;/img&gt; in the model is recommended to avoid bias and reduction in prediction accuracy due to selection.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif
 [6]: /embed/mml-math-6.gif" />
<meta name="og-title" property="og:title" content="The Accuracy and Bias of Single-Step Genomic Prediction for Populations Under Selection" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/8/2685" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In single-step analyses, missing genotypes are explicitly or implicitly imputed, and this requires centering the observed genotypes using the means of the unselected founders. If genotypes are only available for selected individuals, centering on the unselected founder mean is not straightforward. Here, computer simulation is used to study an alternative analysis that does not require centering genotypes but fits the mean ![Formula][1]&lt;/img&gt; of unselected individuals as a fixed effect. Starting with observed diplotypes from 721 cattle, a five-generation population was simulated with sire selection to produce 40,000 individuals with phenotypes, of which the 1000 sires had genotypes. The next generation of 8000 genotyped individuals was used for validation. Evaluations were undertaken with (J) or without (N) ![Formula][2]&lt;/img&gt; when marker covariates were not centered; and with (JC) or without (C) ![Formula][3]&lt;/img&gt; when all observed and imputed marker covariates were centered. Centering did not influence accuracy of genomic prediction, but fitting ![Formula][4]&lt;/img&gt; did. Accuracies were improved when the panel comprised only quantitative trait loci (QTL); models JC and J had accuracies of 99.4%, whereas models C and N had accuracies of 90.2%. When only markers were in the panel, the 4 models had accuracies of 80.4%. In panels that included QTL, fitting ![Formula][5]&lt;/img&gt; in the model improved accuracy, but had little impact when the panel contained only markers. In populations undergoing selection, fitting ![Formula][6]&lt;/img&gt; in the model is recommended to avoid bias and reduction in prediction accuracy due to selection.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif
 [4]: /embed/mml-math-4.gif
 [5]: /embed/mml-math-5.gif
 [6]: /embed/mml-math-6.gif" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/7/8/2685" />
    <title>The Accuracy and Bias of Single-Step Genomic Prediction for Populations Under Selection | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"vnXxlAxiAM179du_8WTrkCFejX7e2Zjx6vEAQBxzloc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485831","apath":"\/ggg\/7\/8\/2685.atom","pisa":"ggg;7\/8\/2685","ac":{"\/ggg\/7\/8\/2685.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/8\/2685.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2685"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2685"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/8\/2685.full":true,"\/content\/7\/8\/2685":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iJ5BkC3t58+ClcTXmCoYIx5ztfuYwggb+y4gJJVzB3rfGQE2B4d7fhsieN\\\\\\\/+qPKKmD8+INeAMsZLQX78RLdJI4unJw9R49Da0PGyu6\\\\\\\/h20h9PIt6V5PfoLrkhpdcDeryqnj260BXDUPlpZe52GWikc4dli5tLWuz4gIqtxctkkooJ0OdIu4Fw95gJddx1U+KFRULzC9FKUo3mFbkG+lVhCRRntESs2C2KzvRlImLVqjVYHuD\\\\\\\/Jtbd5QUT6HCvrgv2DrztEwkDsISA9\\\\\\\/0KAHod6Sux1sods2MZyXsXoUaEY7Rtei5EzqnErcUk9\\\\\\\/CTsniiw6m8HoAKAy4rl80f2bLuUgfqAMhUnbjua6f1gmMhzfmA5ssjdT3Bfoj+oOpoK4dCvFDYFXQTbn\\\\\\\/pKMpwkim2skp2AcEuC44VpcDp7jkiovWWECZ8vbUKbjkcTK2eQhCTSKH3Q9JaNPio\\\\\\\/88QmrEHPrvLe9yVrH8WLdr7WG3kUNQy9heYbCB1HTk7sys5GcgN1zAEo3EHTKjmQrpdIVJujjBg2caCX2OQ9XI1MlyD9E8kKaDFvKnMLrOJ9LOkJywKcHpz95c8UBW+gNuuOTysO\\\\\\\/fcz47jwg4tARzVIGIH3J2e6gGwuEkIXk5d3uwrwV8WOdjVkJrP59nnO0Jq+blWvn7SFSoV1g9VT4IK13T7QlC\\\\\\\/jM8TFTLOIDEP9VqWX7Q\\\\\\\/9hjp9Ny\\\\\\\/VYrMTPKfgXqkqlpL1CakazH7NysTEqTe8mLirrTuF7atwYgFuRZLZ8JKabYk4ib5ycGQ9EgTEzYB8xdg586IyoXShP\\\\\\\/Ep9MN9BfwHbiNxUyIgpQTlbX+2vdrJkErXFunSrcCcx0Rfxt476oHqLKGDLHF9MgXtDDlj\\\\\\\/5cPCmU4KCUzfCDTj52IlyepjM2S50z34qkFq+XMv+5xwzPueDff9nPulbdAcJqUJ+BnRdSprNyVc8L0m3W0bRbytbU39Froj4ujZ8\\\\\\\/VeaZuaoo3V201KWq4J7+vkepr+dqPfUslb\\\\\\\/3QqquQ3DijpPGd7y+lO2GlDgo7jldBnpNaZXG+8u9+2CI9aXvDiJRRWumPj5czao8V07dRK3lLqXU9RyKi2rfFVyDG4UHElTfDKrIbHTRNzEW51NhFL+KhL1TcGc\\\\\\\/KyxBS50TE+ggcyosXUWOdpPh5I1ZeKTcOWjUYVWZpA6x38xsD9XkqafNssFslVJWdHwq9ZWtTGnJ6iEPLMn8neT5slZoVAf9sdqrjYnDMmFrjcFk2GtYkvwVTuWi\\\\\\\/XzINN1RTRtEWOQTC6h\\\u0022,\\\u0022iv\\\u0022:\\\u0022IWvnD6L9oZ+cwvz7rMNs7g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022471622bd2959ba1bfed0916ad960372d6786a1d49cc82f6ca2c59fb2f7762c65\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022arbpqOXfoZejvCWQ0ul4ak6QmDNqwkJ0Phnbhwnl0Y8xkzJ7qhvuhi8TQG0EWn+YEWi\\\\\\\/f5DseApTIXFOaYTjYVlRqt7iN\\\\\\\/d3YKZnC+NsY0NrbkvARe3OjANi8VvfyB\\\\\\\/XEX82sDuK0pYbeZym+Ow\\\\\\\/fntFjPZytHrj\\\\\\\/6fdOUgkpYWBAoF0P+np6PdfxoJaZygRlbS9TxlykP5ZwYhSKhsU+\\\\\\\/1JfyzyMm+zrQs8vz96P2zKmocUEy+yQyA8lYmcvWmE9LjZ8vVNeiEh2ID12XydyUZCAe+dduymEDRNnvknb+AG2NimGxCJlnltCWV9tKeTebCs1U73YOu+1XPqzpK+oef6Y57dqV469\\\\\\\/aX42nWJEwq1LuBUTGGhe9boApfbF5lwjITih87FYOW1WXJUUrtWphfa505p+ezP3rhEO7dNflyGfTo259AV1cDteLyMD\\\\\\\/SGOnw7Yfz8BHJHXRTORiHVfJuPLxTd7f+PjHTzDiNssz7UxLi7\\\\\\\/lTZPWUwMH9k54LNtTK7R\\\\\\\/IwK4xCGY6HqTqrOqms5EZ\\\\\\\/fScKlRm8PuQdgiDcPAfcTW+ZyX+0QlNjuog3gY4ByXk2aggM\\\\\\\/mONN2xb5RiTj4cdTM84WeqbfJxYeTY9CdKaxwrwJ6LQjOQoFgo6eFm+sQSPUKCoj6HqOX+l3qRMLW7LHTqc4bTIv3iGXaam+nOwKIMum8mkSREAfRAYyoarkxk\\\\\\\/0D8lBHqozpVupkQQCRVenXxtbdNfeBbzRJcUoIDTCHV0WAAz6ZPMJ3tJG7fKGoiCxYp7gXaQ2GYMhiZiOtOQmtM56XzDCfR695CALCBPqcF4icfseFhk0ToR2jCljuNVzIx0GZdv3fCTjQoVOMxLbscKSpf5KF8V+1PZNzuPJ3XFRNl5a2PbRlhEFf7vy7Nabm+S6Hfn8wSdJYAk10pcxaWDFzWtHgvX4i3I+Z4psKXy3s1evbAy66zB2F28X1YYr5bZ3IUEX8TvSx1iLElawRE6BaE2Xj6TrpeUxebsJ7uD10WUCqSQaaFxWfri1M7vkQomauBW3IJyxr2El5xg9TgPod\\\\\\\/qKJmNDH5t3rpWedlpbf6ZAxnMkC4pL4q+85yNKwB6SUq3lkdAx0ePBIbghFyj3YFwDQAug2AkWuX\\\\\\\/\\\\\\\/gedNLNqPrPYyB\\\\\\\/n0bqE27DDN8+c\\\\\\\/O4gP3b5t3HkdQLHUaQvrurISX2RadLDXhpZEADk10kYPEX7qeRAPTrjkr6xI6JbYZIjWURekncRpddWyVoDbofKW3KCrJ0Uf+P\\\\\\\/T54KMuqCJenP4fhzewicIaIX98G69ezdsLuYamIV\\\\\\\/5psNpPf+59cckqOVvQqeY11pLmUVzynx4aRivo7BRhnsi\\\\\\\/rz6D7m42S2Ti\\\\\\\/emkJwSPHVZH9JuxVER+IW0OqQLPIlglfE1GsfaPmYYCqw\\\\\\\/HLtGu476NQu6LLmhVnwE0MO8sfTFhL\\\\\\\/PFqW2\\\\\\\/bzRCDKjmZAZfGHdrz33UvMReRJxvaSA0D9tGtPNEYy6K5cCgsN\\\\\\\/uSWHT83ZGuYefqNHCAUSA6JHOm+\\\\\\\/GNzTEGTIbWx3uif84z100TGHktYg9zDKNW0rx4aYwFn2wEd2H\\\\\\\/EM8dOs1EPOz1ofEhr5F4\\\\\\\/d51Co6Lg1R5LaG5eirUJPXn+XJ4MmhDzEvAOFwZ7582miveTeriVZqOZ2EP9zXdIhGRmuJ9Ii0EITs4p82MSed5u2TeSSS22Vh3RdNRC1v8sAnB8dyBwhgaZBr9ftQdpYtifs5bEY1xvk06P5NoPZgAjnDevYcmsiNbdBCjuGABh\\\\\\\/d1s7ZGtRE4Zu9Ls9E+sd+xAUVx7NdTUIVKoUwYmdDzsmaVLxNrgiCWivoftyzrb3XhpAH9ICnPTcZRug8Ooc9wSeeOViaae2GTkU1FwuGzS4dcUfJ0Ua7EfyySjgrPdHpRYddDR9m8JRZ3kVFouwyBTKpOvIofcNBUbrnZzpcXKO0xRo5Oi+NSGDHIQS3BvQ\\\\\\\/vpEklvC1ZB6Flhwc+FcHXO\\\\\\\/6e94kt3Ttxpv\\\\\\\/GudxHo4YXOyGX2PkKWBFbZ71gaahIhAcBs30B7PnM8\\\\\\\/MHllLBS0eFppjtv0+4JU+R7uAf1ehW5LQ\\\\\\\/RTNBFn5FhVvsLkNy4hITQqg9+RDfp6nuRuQbBwm86eJqpikniZo6qXMeUuRoPxVZxxDZBZvpzxyRvpS+0a\\\\\\\/aXM0MqGM\\\\\\\/Pa2I65Ta5tKydLbQ1RsnKoSLOMN6D\\\\\\\/Dh4k9ANN7pNjjOWsAH6qq+lSj8APfmDMeBWsGgIfQpUCghEbUvIoOEzQmWCAqrNzqdFaVGsXe3nhOIEci9uzFb+LDTcXQyRW9Pvev50mRESd8Y87DcYhiNJ6yDy84+HVp9ZdDviCzoa0cGl7qVDdyPva0NAO1bge3ogH2nQQQEa4xUcMWaIGvY2XEc4ZSj3mG+WMS8Oh0VpQc9KRaCXIGJrkrcG3ex0yU8KL+v7tPfDGNfsh3STALJ5LXonjuZ7XsskIyuNw74OnepxCknjIvAK5FiscJ4ckttYDda3VbL8ZZmAYmDmghqlQ1aucTpt9RP05qDJcA93AGnuJAvyO4obQR1hVWlLtLb9OJ8wdM0a6+MtesA0rDONDYS82rcdnzJ2JBCK13lwF81\\\\\\\/\\\\\\\/EyEHUyWCwOpjad13w4zZtGF\\\\\\\/Z3SaEP3B5hipzByyC42BmxLjo2BG623fEQZzumn3StB5vyiBRfyimEVnjgHXHWuSvKGciTBR8xpZTiRcLEAdIuo7uUhpzJm31Xtq82e9MkQ6\\\\\\\/P6+L1TkjOl82sC9MWK2\\\\\\\/\\\\\\\/V1MwVjLl9kmavC1vro+35XHPRim+EqMId7HP3ANvipN5RMaKkcQ1SMIuPHibym1e98wTD+nbt5rXI9AJt808U4MKxa9fL1UsPb4ODOX1fDaO7gSnWAPVYJ1gH\\\\\\\/3Xw8kvXbBL2sLa7\\\\\\\/gf9sv3bQTFdVMZwDm2FyYKqn+QQqwRBXF76DtBYGWxGKmK16jOhuJcYBoMuTWWYBnB81aL18j9sRWBVl74oLLUVa6N7yZGYmApKcYWIxAwA0Cg\\\\\\\/yL9dtkexKhWUQ9MzEqzVNmvU+CG7WsA\\\\\\\/2r8BEazi0lY9szJa213+DIg80jYAKbBuIdVOenFxjLY\\\\\\\/T8zauHQ3riPehwmyrjV9+eLzyGQtEzwAzF8stLeqwQrX8KULUfIkoqXRLrsF5qUG2fFXcq7Y4CzoaKWaz6yxIKWm2ClKYJ98MdQgjZvlcmhotUE5IsmtiP6tmq3QO8lEmpaZTmFCykfc86CTfZQIiwubB5K+x1GguuBPiiNYbVX5efEZ03r0AH5s9Ex7dc\\\\\\\/+AD7i\\\\\\\/Z2xc6z5rxt2\\\\\\\/rZCTEIgIcPQqQiorC1Xny2Y+pJeLsE6osGIjxlKE\\\\\\\/xEe6yfohWPVG5CRDIvCP+qddZhXC+1tcnEcQDYjfrmEx48zB7Pkdw8FWV+P4NJi3KB87l5\\\\\\\/xxVKrgk54xMVdfAdjjNn9oA7Qq61PmAUAxN1dg+YxWJdcnTe\\\\\\\/lhk5Q0XS+HVAVTHsXuBAmSgsh+f8NYJPVSLhxQT6aFvi4vU4eFyaGL1JxZFbq9+iNDnnsS57LfxiFRrZdDRmEL0U\\\\\\\/OKjMPekKL8AB7a3\\\\\\\/foVWglzfuUYWUofbCr4zae9xTP+2KbNw38F80FtM2f07KADBjJ8+lOd3K5LWJbP8A8Abl+HPIcgSoK1OcKid2WtMF6iLhA18zS522CURDKBh8cu48kI4vUZHE5XetfENujCQij1UJVlQdSt0s0nB16azq34WPb2zKGY0Fpm4nAZHhW0dGrqo+m4ioz6zUgXjufsY9y4833FEpnOINGA\\\\\\\/UMvX8TIy6vFsmvKGxFgadsSnIyk\\\\\\\/CjQpQyF2XlMCkVjGNQn35lWJjbtEYQCTUtxX6JJXhkcaIZ1pkd+0K\\\\\\\/T6sZ9TKC7HYcq1oqnG3vUtyDFCrF9tVFgT56OzfHLqslb86\\\\\\\/z3K6WqqTdWJWwaw2P56hxjhcbEA1\\\\\\\/5OCskiIUXCdlU9U4U+kuexRcxrrKaBtNAQxowpmYTv8seB5NB3RAzAa7xML\\\\\\\/x00uAddaIu4RfmOoXldZDk5yNyQXEyuXkrpFAvefLa4ir3otUXt1gww6IoaqRxhwjhSGBs2COwJfcYaNkyeze8S9tKMvb9lU1eDLJ1Da4t+nu+6WGDonL2Dul22GM6Z9HWn1r7iM8u+8FoCfnAnjvCntXRWtLumQkVSzWN0Uhr7I0S0gw\\\\\\\/SasI1UJvN7\\\\\\\/HkARii6aODYp3A3EFKVS5LDrDE+PLzYGt1DteFG7teDBA5LAwfjqHTNu0KLuJqwji30xq\\\\\\\/nlugRrP7nXxY45RwgAjGujlTCdRkfs4\\\\\\\/JzXYdtiEnTcnOg8MRkU8fr96\\\\\\\/RVO1fvGDhrb\\\\\\\/I7+0k9KW2+ZG2eQIUxCqyW1Waz4GvtDk7PtWZh\\\\\\\/inUcf3Hx7d3eKYFgWuFfyoycxjIhIDaXjYqxuQGar8XhrRASB6ypHLq2aDuTHOAnZWx0bxikfAMkGm9fKRNeFWJZqCLTfjLvLTPT1UKpG6nj4pWNNQ2pLAn3tYDiut7n09LusDBDkCvlnNb9CFt4YBJ9\\\\\\\/XURsOuezQgTQz\\\\\\\/dNseEk02ZZycc\\\\\\\/xiW4n+ijtaTWf3QhUIeur\\\\\\\/jBArV\\\\\\\/yQY+a7o6IbQmWiYlA4NnHelVHZYxu0MgwXShwfvjpTzKyuMkRkV54Bb4BWKiXLXNkE1gaCKjlSSrIyxUEneHbmgnytcikJvdXw5PfzqfKIB7ZaGJaVcYUl+O8L92fb1yaAoffaEGg9b2dHAjTsE8c4z+Nuk\\\\\\\/q+bLEEG6BCMNwqzmOfcn57Pb7MRqeTZH\\\\\\\/LSbEXh4qlIq1MKL3jeHWifzI6G3tb1K4hZxal4fVhLo\\\\\\\/gLYIfUtQxBKutvCdohsz4WC+zIxLk\\\\\\\/G4IUiHJsAd6Ro5kgfSusm6xhbS\\\\\\\/xq3y8KY2lQMqdV+XhgTkwpoO2SaXDnwUSevrLaqkJEp4CDPSlizjM+1S9VnQV1GQEcn+Cb9bzgDlXVc0KiaiFgROugXcpI\\\\\\\/PHTTnbhO4cO3bvUNQ+gH\\\\\\\/IjOyM9AoBz1IrBBE5BYtfu6pqXAceJyZ5XivsBD5x7uDpd+ZkgjDJyHG3UTQDIdcsX0G8C0DBOYGJt0R6rJJNJrJJvmJ6HHJ1ApGtuxQKj\\\\\\\/TULbzaiH2SVHMNwk1O9fekB1StuGCBnxlqgCDUuyBbM2T2W8CpIln95E25JNr6YD1PDMvd2cKH8MOlUVUlvgzGYoFzrF0p\\\\\\\/wJtmjzq+HG4WonwVJnmOuybZPx3uOKI++tNA1ZbWvZI219FQcrgHBfnOwvCJD6Q2ka+VrRHnXcwU5auxP3Jd5owDW5hwjwf+K7EoON1VXhMOKvLJrdYqiTni2RKdxQJqssRsghfo33UJKTiOMxuRdCgTXeqaFiD71evsrWKtJAU3g3phMVTmQHBxYSdGLZ3qviau+xpZChPN8T42jJeFBcZodZGOjWtmiwiMw6jKNThADeFkqGQTATgBq87x06IItAd7MLt8LmrxRmc9pogc\\\\\\\/3EI9gE1E+uPK3qQSqoAhmooEyXcwLIWYaUWKx\\\\\\\/fnsMKnzkbJS3jQz8ZCu0h6KaW1vMMWUG1wpyPc2tRLFKVmkLQI2mZ6GXItE3C4Y8XPCz4ldOq2GldKp4HwiysFFXJo\\\\\\\/BjVGDY6xGWt4XGQz+uJlohAjmcKYRBeiH7sQmGgzaQ9NpsdU4KKEzUyE5qyxmuNKsy4yxVwWt9Yj6oocJz6SoH1g7AnwfsGkV9a8pPoUkHtbZpIco+yXK8f\\\\\\\/7ts\\\\\\\/fpW+5BzKIleEMumvN8PuHhLig0zSCePdB+Ss8rcAhGJxT7qL5YXFqJTbuvNYzcFgonFp5OcFz+CIFIjlBVDhYFO98ToltwzRI00YplK\\\\\\\/HuEAGU2IAZhu1b1vg\\\\\\\/zFSKXOscvHrV+lbMLV+MbMavHrMySwK+Wqqrfb1PdVo1ExyDN2IS6thtwQHV8dBA9S6EPqzvZmoM3I1u7KYYhZIwL6kEkQRC5ewkBzjXtRoI4DOup8g2dPqsWNx77HKW4pL9t7ndV9DAMA+Z\\\\\\\/xJLmCKeqkkUrTi86SpydV3h3ZkFwYZ\\\\\\\/FU9Q\\\\\\\/J2Ufmhgxg881VKXWa90Pcpu\\\\\\\/8rCxK6UzoIBiLSsGVZe7C2D0f6dt4s6+iN8JyHju+uCfqEHRHUifzFo5OPUIkvO5JEH3A6iKef1U7v2r\\\\\\\/iy85qVlGQur9whWaLq8epjBvWrSSczpsY3Hnq7BBuAqPIWlEFJNv+PS17ApgD82UR3TYqaFcalJpVYDbrXICis82JZhXiJNsBYr1FKx8aK\\\\\\\/XGrm8Df5uks\\\\\\\/hKhpiE1fS2uubby7u1hP9pp63vVfECCrl4oMqA9wgpUSD8l1GZmcdn7kJr9gzUD5KApYad2UAUdDEsx38vL+LwGZyW\\\\\\\/9UBt4kF9PcwPTtY74kGhZyd8nbaauouvCCHYHBewPjKb9AVpFcCbQ8SjpyY3Anzjj7AMGJozKIixh3\\\\\\\/vNBf2w2Go52gM38JSwKwO2aCtPj9lorisZkNR24N84k62F9Yv0vZ4jbJq89WRN011GMUUjzVPA9P923Q9P7lTsNw5aCrTetJ8u7EX5se9uwKiXNKVnX+Q4EHJ1pPGJIwPjJf7z2TUwIenYwuOMjczehRNmakU9NeCcqWOF+Z0CU83u36TzC8AZMSUo2S6yU8U6GEAbMxZngJmjmSqT34NFjmrR0zfEd3E2\\\\\\\/Wp6XOw90vMqnGhPHOduridmtWXiMAXXWznwdo2QZXXd\\\\\\\/IeQ5ExJ+laRfim2w4sh\\\\\\\/gkD4elk5uCbbxQae3np9pHRWHOmwF+qpLCYQsiUcoYURvjFmlkCjvfaRYnt4jV+vb7tyxUsf3Ta4TaaxQSACD6Gim23q9t8pfomGnIbm9O8\\\\\\\/nauRUYZKBh75p6bL12KcyaeDbdZqnd2V+Qm9E6ia+EPJuiRC8g+ayHpsF1EhFxsjbaiqpKHR6Yxgz2H1lJD+X5Hq20HcSRNa615V9t9npXnhy\\\\\\\/F5RjznU9ZMU9XDcjbwfOWwslp8W0\\\\\\\/H9y\\\\\\\/T7LoRMnYZZLYhAaERxkYVrMrRuSv6GhOLUpNrcbsZCEsd5xoMlIxAMY\\\\\\\/sA2I3mFuwpQf0y8x7tFkw2Oz7pEaG4esDVWvPq\\\\\\\/beGqA+9N6k\\\\\\\/CgoT83mhNmfrdM\\\\\\\/Vxzwxn30B2NYGjQtOf9oOwAiVdL+333IcdffUp17GXriPEXqqNG2iycLc+Ec34rGm2UIOcXUW\\\\\\\/knPjrzigjrnxwRR9KXgA2q3zbUAWR\\\\\\\/Db3VVFpqM2VXaLCPZRZa8gfBuLIRIjHyTAqFD88mhF97nzUQlIJEwSxEeCw9xrcVOINdOisD+8YQtfe8yMWaFEgmfOTy2Z37ovKvh\\\\\\\/oOxXTcdIwR21jEKyuuzIcs0k4xN1J0pDqCI87BI\\\\\\\/jZsJJz51MzgCF4PnmJNP2xCH82KAQITzFq63YSAkENqT91p+2R09U3+OKia0lK31TRruF9d63yRspOkwcW5t3U5hdVHyG3gvnyEivAE5XFsFZyBsoaNWQNHQAx9J+6NCDXT6pPvkAYVXb2I64Cp4ZIpOTozTdQBBb1osYPMByQAEKAJ6Ry5T9aYruqG8wJCp+RZqw0vwRVdvQPNQ8MtDDG0GkxInW6FpdTOwkNTOSTh2xulLfke7oLOO7JYh0gYz29dlxbsQLlgGkRIoCEbNQ00k2XxDTJ+h6ysq7RQAV7xejIh3HxavolKRj6dhiTcahUpjfVuUlnNE+nkpqRlaYzpDTdyNteW9UPMBI\\\\\\\/i2nTvN5Twy65VrIP5fN8V3jJPkZ10ldIqjfIJTii\\\\\\\/rXtA0R9jvZXpZTaCkUa27yv1Tz\\\\\\\/l5eZ9CA56q7F6OHJ0tM5pCMX5m47EwfBiEhmwq\\\\\\\/gJCJEyRpcuMlWfv7VEfgQf1eoimDBF0U3uZmfl3A4e3+4FJCneAZWwW7UeweuarztbmO7bO46genFKU67BY6CAc5RYNIEbYqVyaWzVUYaygnpJuQUMI0B\\\\\\\/Fe7cJvpl9KHeP6mXckUl1SrlDrT3RL6S6gnxj89imlqUlupJ8YcaldYqXil\\\\\\\/NIP3tvj2MTD4bV5T5scqPHwLNG42hIVozkTyCbv+t+AFZOKLV8YgUiB7MCCReN8SBNUu7imcyUiZZ0ORwtXo0FHZFQ97H3TSN1CdMmKE6Q9RCYv4MwpwqlI35g0TcFRnLTijb1sNmdKhCC\\\\\\\/tXgOu2\\\\\\\/JOEs+LH2Rd4ITz7nXktKAXvRtVk0w\\\\\\\/PVO3k7Ysbmu9j4FHrkVbYvp4cGNZ5cs8Il3n58twqYs6MCFDl6DNDccFcOhCjGu+YcxE3\\\\\\\/6mIDgEBtrt9jejR4RuJXx1C+Bk262Dv78lO2jE47vumEWcN7RlC48Tu5TJ6DnPvEaCugasf0ps3wIidwJDtqhCTA5yT53RI3ZpHS0i4xl0u\\\\\\\/CvTCYuQakzbVK0p6b\\\\\\\/X3oHorOwYgGO0xa\\\\\\\/e60d9jePKdx0FERmGKrtSKDzqc0ANOiFoR2+lUfTBefxnMOE2trF10qd4rxRnCXPYMt0QOItJu7Nez2mZXR\\\\\\\/K\\\\\\\/+hDbWjWJXThDunImr14uVpiAHmEy1pqwTPXJUTu7kOsWJfvCPKELYr8UrkaN3sxmUNepBAi5dJrIb9nq6thAsheY8\\\\\\\/Ewp8yj90FUonBFzVyu37kHgP5f7cVloqL5SDYQaMN3AseK0A+yKdrtPDNjfvu3TkzsaYA48Hucj6PI8xrfcai3eHFXGoSn3nAxuXOifAaJRsua8\\\\\\\/DcInVi2i+CBHjfPimBNcidPEKx2rntuharCDdX1J2JIMMvFQCh1arrj8zn89dq+S+wdUslqm2BAakthuw4e3b0rr4O\\\\\\\/Xn\\\\\\\/BsMEw5Pw\\\\\\\/rNwBxBCul9I38uLA4m3p4JY\\\\\\\/X0rzo4pW1FXvSiafEH9CJV+gzzaq8\\\\\\\/JHPozhBymfVX9B2eq4Q6v1g1dlT17rHut8zu+v1bwE179BT\\\\\\\/F25Hl5gs3+hJPM4U1j3zZ7CFBARYwHDv7HrnFuzfhXtPRJpm9T9aVfVc8Mja8coFvQbtke9MUQN\\\\\\\/xGZSyCUGzkeTUlEqt62+BoU\\\\\\\/2QQI9xPPyqabgDNEArLETjKe1j2wlH0lWk9JduqxISUlSxLf8YYoXLe\\\\\\\/mVVhXqQz3ohhRyp3w80AGxQ8Wqkioh7LMeep4G8obhtxiVYf6n1mIKsWm37\\\\\\\/vB6LtRYNFBmUesllHbqYFoDO38GmSamZ11nY\\\\\\\/IX5ft5NbFJYZEPT3g+pCFIeDaBm1XDrXbP3zExvBDJRL\\\\\\\/MggIFi1nnVKJbyC5Uew5SABAbIlkKs4CyzKt7S26rPkuW\\\\\\\/Tx3yldgsioR9yyO\\\\\\\/08bKNVRAigp\\\\\\\/p4V29kJaGAHWhqRr5roTV\\\\\\\/Oq16bxY0ShjYvuaUh27dvJbPaQhgyeHXuvSEZ3l5xGkR\\\\\\\/sFg9xxdRsuDBVEXGPy+8m8Oy1X5VWmfykxeS7r9NMSgvZ+0p7HIBgbBgKT\\\\\\\/kBbd6C4JJ8D2wSAkXgDRd+BV\\\\\\\/kxlsaVFtu3LfPkRhXMRC2ot\\\\\\\/oJUTgCtWoy6\\\\\\\/ZalZbP6L9LZ4ricCvB0FUr+pgblO1c9XtLhJXWHD9obCMfrpt0VI15XNMulrM0ofNMQGHQR4rn7pAY6QK1oR\\\\\\\/fovotQhZkHAAWeOuh3ulS7cBF2xZLgN7kHAWuD+RECPXV\\\\\\\/LEVQt3MUjvXr1yIucCvRMM5JGEgTsixYW2Dryt+NnHoZ+YZRDakYJMHzyWo7BKoR2\\\\\\\/sdzxK16k6DSRQFwnH+FY6pnfbAt8cQGlQvJNOIvkwvuczjuQr1j+xkHqpKhve5QCq1U8xHtNcae5jkAGco8\\\\\\\/J5o7oe6\\\\\\\/Xg5VW2xLoLDCzqsltiPI86HFXEwht+if5DOzlIx2n1jgPRYXgK7hLuW9+xakQ85i7pVwhbywcExTEJscSsIFsatSL1Fot0tDTfIK7LZO1PBB77tCEHnWPDHskEbliZVDNptzhj4Y85HxCFkBx+JbAfWfocfXADutD7Q2aOyon2wKV3c8nwLQy\\\\\\\/uEtj\\\\\\\/uGbwS10z+hxSsnjFvipn6Mszyz27Zmq1h7igBnNpyR10Hf8Gywut25uizOM45toFhkuKQXmtWqg2hpIzxoEGI0GbGoXw2QuIVLeRskNQxQYKANNUtG9YBhh12y8BLkVsBmu3KXBFJz4NjZl3ts3HP10+Afsyvev8FyOiAeNYMcEmXR0LrPIqSb14AfmI8S94r9vlxf92PKdwcVYTxh8vsfMEE0jo+0oKKCdqgCqWEjFMNLfiZdeq3Qy5s\\\\\\\/uI1DyQ7BLZRCqAWnWW4tZhrE8oEHoToOL66159gICyC2Bopomj4m6Wv\\\\\\\/FoDG\\\\\\\/U55QhcIZ2YNk78X07gVZszGPP2oygH8xxWNasIqRQBCYlaW5ktnj\\\\\\\/WY+J9c7jekgY3r6RsGkOaDZcWNKtnCAMGTPHtahLHyr+IXnw\\\\\\\/zWTndYPtXWhuW7AA+femFl2XY6E0Zc95Eu3ZJwFYtR0OO5ngwwJm0vvR+OHHbBa5cDOBGsvBX642DiDYoLWCbfaYAxZcUe2TeOmGA7k0F5Xe110oCUwfjd9TMyIjanOPmDBPUhCMHvxN3oTfxgtePehGvsPv\\\\\\\/5KgZ5dqxoVc5LtgHwTe8eozUG5h3YGqLW8Ul6wkGWxBu3q8VcAhaISpUmBD9gmJ+r3I1dm\\\\\\\/2tiw5sG5Ch9D0fL7L47TPN5n2+KuKupi3SwNSJdyF78+7hAbpSGeh3pvWGd3Hk3aj1EMhzZYueM4QZg\\\\\\\/0WF4d4lPs0q0SSD+bYLiQHHSxQTnRmJa9Sj7SsVVGKTnUPnhAw2fFI\\\\\\\/hdNN5g3vxcTfUu72vhiguYiDNmk7aVvd7V3Jrzm762CUeDUsaldjZ8LcOovkrTtFA6BIdm\\\\\\\/kULdvRM\\\\\\\/Zl86F8IuCiz3z941T041LeHMgiFL6Y6SJII68yYCKT51pQSK1xw5kmU38l+VL10yhBVuoCTqNyUv8IHYQwldpMLLZ81qrNbKDyT0Wqo0ZfWedJZazkxhYrafv5TRTxljbnD1Eww8xHdW7Bw9QdYh2KAjBVjOnDR\\\\\\\/R37JF\\\\\\\/mAOgjK+DnD5IfxXzTe7OHBTC29N6OtWocY8JCtD4csiszg6Hr901D8jPWn900yEXuZ1L0rsYcFwkj79Z51JYew2I7wrK3RWW7XCYOMk3hs1sdVzCIBY5DVrybl15PQWZ\\\\\\\/RndaExe3ISdzo6rgX9llKCCry2i3MQsoNHE2XJ3vSOTT+QFv7jw6o0koZ0Of1XtW5EtvyrnfN65WO1bm052eGyuKgIwngw4TqMtD4UgLVZQ1fy1kFeZjc1tTbJOVjP2awK0HnJMHAM4RDedITzsQAmoqL2U3UArvhG\\\\\\\/8hccn2DC+8lNMFR3QbVeqqr2whUeMPRlabU+aZMhEMdCI6JpqtrZKZZyTIsc9O06TBx8669dceAOQxMOF01DE4VMpklzvaMW\\\\\\\/gRySJfczQxhA2RAPRvMPjJWwRow1szUW4MM0jAzgikoyL1xtNZwflKtaFf\\\\\\\/cUddk8yZGdHccO1Z1cHcWJt9jQ2RC3Tv5FfpTxS7XtP4baHeQp8DEwWgNIHmKfXReanQ8i2MgT8i0mqitA+cVw4cZyJHigYniQjeoIcs8LR6FrrLrvCkO\\\\\\\/Pj2yugONdvYGojoYf\\\\\\\/aGaOTQa+2wWYJRfCtet3GEySEz2LZqlG8pDiU+tYyzv8jIOdF4qF4l49kpuJSa1Ka+xPYK8Wq7uDHV+87HGJdpa+EHvJipQ\\\\\\\/48ccjZSLT\\\\\\\/KXx4d4kwBk3OpZEqaaeF2nbZz8ZjMJZ0sq10xWX8aWLIl76C8i+SmbzwaoxFpMDH2QUEjHFp+zYVU32lBrQzdJ2yVzzGeq6CVhHvMh+p4bJ0vDb4bVmvy9NL6FMaYlh0buaFfsB76McbEzEps5HNuARAGwW6E5QhSmH8su+92e+CCyId2wqH8zFALyfFaAWVL426wxi1TF+fVG4U7OTHBwjFySXBun9Xrn7sbehDCbVlhTHpLARNNoA7wofjYv8kXIRY+OQKJEMUsVWl1ArwL1mZ+mzi\\\\\\\/+yRLtzTvJ8c1rBOixB4QxngMJZnGXDm1RaO3bjHE7LrWictfm6Qd3Hb7JC03OAh41Li5XJQhCzxRXwKCrxWV72Ken1l\\\\\\\/WORzzuunRvCQT0at0vEC4ROlmGpFe+CvjSZ7Hafbw5IgGG9yp7+obYMF8p05Ti8AWemdr4q8D9e8qXo+qfgPYpYrjxxdAEG+XAwq3QWgOeAYMnmydvq0lAuf2PWo6PJ8ohRyormDeGxM6RfPkfor0d6bOqC2VVSnkKrc7x3SpxFW9m21HfQ8V4TuYPze6aNNVsrvIvjjZ+efUr1zESM\\\\\\\/1rwOKlrG8hhLG6rpyuBmkK1mhycVnOudq4mJEsMrTylaabnyuO3YPV7fK3zn9jeKlfCnFhRhj3SNNlax+TtaEVhQdYfzadhARP7fSjlLer5vlf2M5LyuYB1DAFgb3GsD6at3kQmYkbwmWdsyLhtPn+jb7XyBGJcgqaz8GPnYRh0wAU\\\\\\\/DTEDn4QERH+rT0S7xcAwfQY42PxxvEi5r18O7IhUi0pFhtcDC+rSxmgj7eE\\\\\\\/XTA06HR6eG2hFA+CmE4YP6lvvs1SeiDuvHgXgR9fSbz6tWxkCBiXNHbdY6HZgPDnsSEBuaQz1pyh6XmlccmlhGeprUXKCMzX8tirmDHVcZ9xgkY3qsDsXgq2MPlWqjrT8Cw5f0kwfiQEASvFeg69nYsQCgbePjbIHgbXsDBFBk9N16AYhJUxSZMLjvLL3OG0Ck\\\\\\\/rSAnVuz0\\\\\\\/o0Cz4tSWzltWv3rNJKI7UbM9Zk10zx\\\\\\\/eV1aQJ+xch\\\\\\\/DtBWJd9UGQvrZfmq1PsK1SPWJU2t0CxTIlWS\\\\\\\/NTnglblm0cTeu99vX+GzmFTARZZ+nBhWyY7cz6jLIYvlW0pokEGma5uL1AFIW\\\\\\\/+v3gl39jCebbVgWQX0m9FD212tV6vih4CRcZEK3Ra5pCA1HBGOX+KGBXnnMFPu2mVftoD94gF3Ns6lohuGhgcMm0h6L\\\\\\\/tOgRNQ+mqdtS8IBfNG7BCrqa855SyCy2LrnzZdfqqWm8csWBV2\\\\\\\/d5eU2SH2yRqyNkg3FolvKCxAjXaEmGiqiUlNHNh5KEHQt9XqNw9rqH03K2pb5RVFH1o8ZaquGZHvstkKyJn4Re8NwvLx82bQyiTxLaHpFOUNWgZ3LRDt1pfjfVebbH2axw99IPBxpbkBys7\\\\\\\/6Dr\\\\\\\/KHnF3da0ToHsjSrN890T0kTX1tlTV3iZMUgC5M17kZb6ih6FfURh\\\\\\\/h+btiAwYeeqprkpn3AOHDmMFwtPkKYFupwkAtUpwTlwR2rjUjztJigR4nlI6zx4rhePC1rOj56jzfjI66LullFKkKDwgc2YHMtaBd0sCVXrWa\\\\\\\/VsTCrKv60MxsfxA1KRPCgLV1QzIOXmxhpScAB7ejRO4p8HOMEEQ0sbORSwYelzsfjSS22Ebqni8Hq1jTT0a9GydGAkHPSNqIIQYMeSHT7YGWnPRN\\\\\\\/laSP3kKWvkZftHWGljecugqMWwZlciv2Z61oRy46IW0Jgfn3oPYWGhX78zyLkqBUq+vcOQgC8uxS+cuaL3Vw71PaS4\\\\\\\/LYcla34MSO9E1BIFYuwjzhZlZakB2DMri5lBbIhnX1lH3eCv00s7rcUAynGqmsaHlB8aaT7p1ZizljOcKyFTQSwgAWCU615ulpb81rWT1CIKRZAUs+P5dD3UYs0EYgSuAX97HWRDjMYhAO2fGveYm5CwfdHs8uwREnRW8wddVF0Vc0r516mJ8OvmWqEEDALUZo3IwiQMbD4aOuWwqUx9mt3v6NPIwZ9HOf2FARvzQ9nbkcMoOjVYeepCFFEVmTioSPO6hAdQtzd5vdHo\\\\\\\/43YAK+REa2EK51xVfu\\\\\\\/nF7XNBNdSTLSXM2axGaPSLcO6ZCmTXluDKFX9oSV3yPa\\\\\\\/Wl7n39iu9jliyvHjpLuqDOB4AZeaq9ohPJaLqXVVibft66zL3DxudU6Zk3OycHvMJVbuIi4UztbNlO4mnbgcbiLaraHpDmomMa6edMqdMouWrwGpQTCn51Jxf0Gh7VexTlWOnBozt3g+cBvHG+FXT2k2JhFHxbyQzWwMY8thewSkeQ+gTpzysUjU47w+2VgcS5LXYxVjxtloAi714NNlpLn96oz5poc48sRi1GvCj42zHXpS6n2W14bdowEeHwDqb9h4Ch354zicbWDiNvaR2jti0BWYkncNxRdO1bnT1SoGUoSOIk0kYMJk5yDXRypVMDxIT5hJ57fTCcu40\\\\\\\/8nIayK7p8QLNGUcS4+Sllt7ywMOZ7pzgCutpyEJS1q61OGJf6CuoZ1IIlXeWOAGhuny5+COo28FCo6uhXZ\\\\\\\/IB4GMA0SR8cMuvNwQAMU8NhjK96fPmDlPz6GAj9E1j68TbGwzYKwXBa0ZKCzpAivGO3F7\\\\\\\/G9RtRaTKrxYZj2zXy7AYwoKsQoBP0I4oF0HNeyQtCIVRNGzALIHh4INWvurdcGODm2uO3WdrdL+pVSUSM4Ypq3B3fjZ\\\\\\\/+6Anz9gg2YQ4oBkC8xmY6CShlMYXj20SlxNr91rE5tSV\\\\\\\/gTAoDfC6d4xfF6ED+jTN7N0j\\\\\\\/OlLTLziS5ucldpa2e4S+irc38mTuxs0wRsla7ZeLlVNB0qsWk9dgdPmNINzxnFJUEUQFN7IY20bH6QE6kP4JF8ZtMFkOYNfFSoK30PHyqGXeYier0MG9z89\\\\\\\/NaXBAsimDIx4RBpiVBFpVQTvKvpe1jWlZvJp7cBd4bSC5TvM3YMX7BrDSGotGBsyNj4cWFzNPQPInIF7d3ETsPrQyySqEmWyydQRFUK+WyCEIZd0mLqA4WkqU1\\\\\\\/ya+3Noi9hYA1FRceo1CTDaU8OOuPN5pkwP09+9OG9zN8adlM1zhbmD35TEq5XWbWySvr+OwKUTpqeSpkp+PPkbh41Pnb9IhDGYOqKJ7nfCXbBnlEvVAof90i4ycNjUvS1CV7KNXZuXFjtEgsXAQS95uUdGVfAxYtLT1gMsDXxrZhKT97buDW0alAeVfgZ7da6obMjBlPPRZ07wJSs26t3K4dgmoLcFh+v9G6Ye374xUPoiCfzgy9ZyZimQDT1RY91j7l2rcRdYun9mb3HpQIH2YaC+\\\\\\\/44guTvUqBmFi43kqYhooppI\\\\\\\/TMYav8kR19LPNinMLy03gqSh11oCQamZt6DPZSEwD+4kuR2yfSOTz21a90bZhjxATItyk5ha8h9kPiUePqt++Qn2S3rA8hVPF023+ZzNVjy42HaWqMG8MC9RAGaIAMTG2qvZ3A3586E0B6g4fD68VdFD0aE+ikhTHistALsDU2ZBEOAhKycVkuk0kIKwC3mhhW9M3XpP3FgaRombPM3QQvEeKBitXymSdYtKJ5uvHxapwgLguhPGpyqOsnYuzOd8jtB\\\\\\\/p9yNiGmkwU1Iiy8b9nD\\\\\\\/J6HBnURA2vpdmMP0Hm3xQfndsLw2GAhxiQNnmu2axJrCXM6AScCj30aC7PjvETU5Gy5swQ42XI2h3TyO0h\\\\\\\/Ati4JDaQ6PytdCaAzcjdXnwjoY9cGe1qIXeGpOONIyGht\\\\\\\/phqteDPxOIAw96o1b69u70DCL6G8tzM4dDOouGu52Kn6cClsVIlUi8LCpg04Ox1fWiarCgxnDggS9CuUSszzp\\\\\\\/d\\\\\\\/vNZKWAqF5VmfaN7CYdbLyRnvvwnY\\\\\\\/hXlaS1KtgtLpIgoeAEmxz3UXZQ7zzv6v9Ps7ZPPV2wLHSMfTv7d\\\\\\\/j6N1OGdHOc3tP5xGSS1bB9rwAiIQikxEJC7O5QJKkXCNoCYdV0sJap\\\\\\\/OO75z1HbVBRzfGjCFrmsG\\\\\\\/3b0pNEhQImmeLpu8DHf\\\\\\\/HHPinUoNSkURDxtxJlh4iVTXv922djho\\\\\\\/tn4eiNWG\\\\\\\/dOeOqZRGYRds3i9KewI+R7FYoy7cTTxcyL78vsLcCK+ihcBwemNkEB8Ut\\\\\\\/Pii8yMltFiTee4uq7h0nl27gVWKzbYw0OazR2P0GdIzwUmtqcg0m+\\\\\\\/a9qExG4uRAlRE15tu84OzvvHeHcV5lpsjd44Paj2+a6mdlk8ubQ1ERcizHsMUULVX4IHS5cDvYNGhLUObIBGoJxv3D+mU3EXGSWcTUYVmusyetPmO2E3rGCiWF3bjorUl31Ij59rgYBxiEv+E729a2+FTX3Bfsw0IPwcGCkUVWsydTKoXQ2MvTcuNgkC1RHA8ruedp9nPTAnOQxuUO\\\\\\\/AgtQUuDeH2LDqPNf7dkhUilteMFLnvqLYUuWlCHetE4ztgcuyrY9p29Gx+Wk7l4Czwto5E3M58BndX+v07+MfDDNq8UuKcHbymNGHgmwrBBxmyxQRuJjB89GICyo5U67vXfmfov63EreUmHhRXpdg9vmUURw941+MDHThjK1Smj7uUF2pDsB1ugcvkAbpOJZD9EfvqhPlGcFx2wuJJ55MyCLcisqPSp+TIyH+gMVIBaVWjfAgld+aW1KF\\\\\\\/ARJpu85lHgP6Ov+HvkNe+LytdkLIwtAgRsDgZautbJMzNoCyRtE9X28oCTg9fweRnf9hglFKA8fC\\\\\\\/9Pq+D8X+23bHnvRgAMTpEBIFE1qyx5sNeXLL7BtkF\\\\\\\/EuwGGkfKPhSWKa29OaAKsj2x\\\\\\\/GBERQZ\\\\\\\/GU5czwq4EmLOmeA2kjk3CkJoyeaqz0u4455dDFU6J853Ofhtevk1\\\\\\\/m36+WAq8YCchKG8k9wXWpX0t\\\\\\\/ngjtZUSOY2UPKt2BNKh0j7iOLZ+OUg1tiXpt8nv7HtJ7U7tUtivI45ryh6vCrZmyOX5ypWXR3c1rvyOaMVoOTQq9NOJmEM\\\\\\\/B6iNoWqL2jiHznvDDC8pa6d3dZ0OQAqg5D8MlOQMXFMkNc4ZV\\\\\\\/QWDcRWVapxpe3LDZFbOOwZxGeIwS1k26mZ+XP\\\\\\\/ZmN+2nYplFuDx9lEQJXIl6U468o81EXHnCR41HEk6L1WoO+kLoG53p3u6moAw70GiqLuc9D1eRxN\\\\\\\/G0jwI8w87YaEXBnuxbcfTImCFX2VAHAj9zhqmZOjvl5MP+QQJ5F2R+PCRFR5szYzhHuAUq1FOPzhhtsz0X92g9lE9RUtZRpjTlKH9lCwxi0HXnYip9JC1cisdYCdJx6Pop2vWxvqyg31ocNNtOkOvPCNh5tq65+sFcsE\\\\\\\/oDMjwSlIGkJIEDQm1t1awTkdVwm8RQM5WKewh9jHhINNez82zWx9rZBFdk0J8SDU7+W2Pyp65ztmhux\\\\\\\/w9vxKeP7uq4aG3UXJw5c7OcWHAlYCMmMRMmG8EAVxXGggDKD2Y8W\\\\\\\/zCZ+uh\\\\\\\/QmRvzV+EaDLLwqieB0zD6kUIeSY7KD3NVKtrazIEZ9tklascf3siFL0PH3oAIQcn4p4qoKjsSutTldfkzRga\\\\\\\/OpGoI4e977\\\\\\\/ApSteFkULRTMAQmY2n6GTwzSz9askbLPuf9yx\\\\\\\/hKZOKr52fZ1+iO7xzlGXG7341r650c4BoQOAhYQPR9mQLsWyfo6UhGAhIRh1+H09rg1+cDAxe0heZDZCV6Oci3Unbm6FDG5J1\\\\\\\/tILCyp6t7HHls71Y49G\\\\\\\/gfzOCV0gU8yS45MB9w8XPMi+H979xGcIU89gDs63TIb6hBiIy6bt3lQPO+MI0gvcL9IJqHGPcbjjwYeF2T757YhweHT6DC0h2z\\\\\\\/kPp7MGW7kVlKlew5qxt0FA+Kb5nZ+vHxSPeR4hUgZw7G6GgEKQsg3+VQ2OFKyInxahDxy9KzdqstPc7GZiNHGFAs7mZPUNs8XsBoYXi3861iIu7tYpGqsl65G+90TPLAs793JaVvUjp\\\\\\\/1yzRW6Y7+zHiPyAauVY\\\\\\\/Qb6zwxmn+Z2XtHzWLgLkoqZItxJEOnJFxX+ggEi0Ssa7gYr\\\\\\\/VCeqRfgGTz0nv3xeqZqMFAZjN0jKOwbxU5+AqVA3xBpD+KnMNIb8tWkCwceDLvSAvOGFKxRApseHH\\\\\\\/aNNZldNws9uoETI5qvayPy8LY1NuFjA5U4TvHb\\\\\\\/oxgaTJrTlGZY94Tyyag27wPdoetZUiwQyd72p22ZBCWbK\\\\\\\/Qg\\\\\\\/J\\\\\\\/oWZFjb5dxOu9Mfh3FteopS3zoDaJkTCNPV9zKWs0jTKI5QEaoW7RJpjYPgRSvG4SMLdD+ApW\\\\\\\/4VPoIe7IA+p3dAqxsFjipRIs8pDqUdrhGKBf\\\\\\\/lqdjtzVWSTcEa4br1+hYvNXFzFLnfsFj05AKwd5FXvIzBc9SHsHkqm98Ebe8I27yEy3EKeZ5WBDWikpNX8WYkHAud6qmSmT\\\\\\\/8yq0o0Oerh+4MTQCHVwuf6Mk1S8CKuFW6ojG7sCRsae+5EgR3JSDr+Krgf7QvFcagoiptdM2PxhLeR2N2DmqUbL9cnHwSXFIPLK7MhAZ96AEzgnqg6xQDS59j\\\\\\\/6Uy+srO7Ck2UdFaQCq4BroIs6x1TM62eGy+LNC8hHibLyAt5lBvYLHdz9\\\\\\\/Ja5ThsmdBzpv0C2RJ5JjU22O0C3s\\\\\\\/VDrY4Q8+tQKVULlLmzT0hW8dMlhW1tveFUtWkEC86Fwe54rxlS84JNsqkyRAP3zcwQ8XTYQ5VrVrSD029+PZemVrPM9XdFNOm5h3xPNSFg7JbRVfcIqkHJEgNHpCIDEk09rwlb\\\\\\\/LIFYk0qjDX9xepzwnBNM4ifgDjbjOGw4XNnvWXeqd4hHWJ2bfVgfP+zmB3xodv7kU4+GP9Ngs4GN+ADmdQu6TSha9FUBnFOcrC8hI1WIUaxskFgMtVzRVfPGQkE7fzrt3zKuKlTVCTQGMgbgVn2YlYYQqUoepKkYu6vEpxHfnoLjxF4nbICe91wLnU8rreBSA8Amnx9mlzCwb+bxwAk0IjuhC4HF4pdzuysg+MRZUuTEfrEOZtuQFGDj4DnzAGVQSQcfe0rUAZXsrDpKOQC47+0NGieK46Jtb0lA2JovO8RGJ3f0nbF+9NA+FMOMq\\\\\\\/tSCrmMcVnVJ\\\\\\\/qlpQltYk0q6NeKZZQiGJ0WYELpRZfbqhUnvkB4Dcsb2SXF8XAOuc6AwvtY7kLqoj+FCHoAFe6prHeCp7jJELAI4QS65wIkQrXhdIwD7OxrAM43KhI00PfcBVR32QkYsca4ObOlW1LpWCr8+RfU7IkRprb2Po0s0iOS2yIwnGlrsqAPswfziaZ28c2+mm6Hc5axrdpmnxONgrv1qPVWWeDDcB839AHeJh5BcHBdHLDMPVHZMRs7mFzrZbhkFKyF1hkuxv4RI+wYivyjVYBeeHctXFDqCQZPwDThtAFXW+r1COy9+emgWzA82nvpeLVf9S8VVBuTfQSumf7C+v4mcVvoqc5QK1b5pNNHFAHXQUyEh4aE6fjGP50bVvEedZQm47u\\\\\\\/gDCsgWaS7yeQBcFiYSsS3ggx94JS4XA6BLy9GpNhXt9rvc180AElJfVLpJ+rtO8qcvk21XT8UOPYw4fPlblT53RJZvDZD36V5a08yEC6vYMVZIY5U\\\\\\\/0HSAPVEUagymmPKBW\\\\\\\/Yms0QvZ+AAmS2KclwfPnqz1mNBrqXqAqjpg7bco+gHFijTn31tJSVGe2qgEj4O1JzUFgG6uIs9BSQjdXObes3lvcEHaeOp1\\\\\\\/0y6+22rHdSIIt5gVbNTkVTgrweKBfH\\\\\\\/x2Pm2AHdLAfwyQYsUrq90w7a8tQmI1TWFG9dINNdoqAnRXO1CMhYENYQ6sPECzy0EGeTO5AAPZxtAoySPoMNTEQxSAQJP+1Mgbx7P788Jzzhl7Lpmi1INLXlIj7wN8s\\\\\\\/8a0vwC7bwk2yxPKujZElDQFirmpa94cd2w+buw3d0KinamTRI4hDkAryGuBrpQFvkK1PEvxWEl3WyAYx9NZAgtVKIgzqGOS9ZdiWx5qUxG5FozOoHl7BR\\\\\\\/Oq6lh3I+0xzHqsEqu8EeYTd4sbaCWkAV5unJ5nR1YdlDPy0h088lnB2j14eTrMDRqYI963j6K06\\\\\\\/Eriplb4iRhUeT5KdbgnSxDvmjiLjiSxEKwWWiSFmjsrLZBlVnVkpSNpmnn7MkDenii7z3D4+c3QmzWwKszEocmfaaFYBcF7b9OKs9j1B4PgbHGKp3xLHY12u\\\\\\\/NvWPh+xANV8NUswlqVxA7ursMOT4\\\\\\\/xMRW9emARVhK7Tx4fDbNmOXwYomzc8qVYcUyMUK\\\\\\\/8\\\\\\\/Zd56wPkTcpYutTCGI4vw913Ly31jzK+NBC7LmL\\\\\\\/1xLHHgYkhGBLs5Fmfc9d6JZfuNl\\\\\\\/swWigK3\\\\\\\/q+uPD9YtcdqA9FdFbyup3BxOslGLa6X2uZhOFgI1j3LJyukxYB1nPJ49PABVtiiPQIuI4ymwrtVEMormn\\\\\\\/a3b5jdVmTTIMeSPIzGOkRL5kOEMMNqULxGsJYis33r557s7NZnOkPT+HfWwkfLQ80rwea0AY301sNGGRWernS6mfeHnW2NKmksPcwHiYNkc9Lcnt5favtxh1f31QuGmsoORzqfmr8XJYUMFj7YniA58kelzO+ZGANU3WM8gfSLB8aOW1G+mNkwuoZXJKBsFa\\\\\\\/3V1Utwe4uxaIDgny3E26cBi0UojCa4qIYmFJzeNK5+nIfffppV4l\\\\\\\/8G+x7JWOTtuU80SybX0FtBOdXMsXiwyBaGiEthPnzunBV6etCz8zoazKSyIMqsMOBpM9mWctPJRwIplTbZYyI\\\\\\\/E5eGTZEMixh51OnAq17Gj4JcnGvIwfSVOQQOsfOY5w1FnXD\\\\\\\/vdV0N3K\\\\\\\/Z65AY4wWrcC4tedlGlLSRtsh5XW24WG8smbZgHtfyXEuDMNfEnSNGXhMkHsIjFI5pi0UxKouDtsiVBCzjgxCfnmIMZjXLXo+hcZT4KbyB6uBLwqyG+ftfxGOai+Xdev5kx\\\\\\\/HfrjdXFahqjKH+qFcZT5S6p1vsYCD+eMkHdWCT2XOgJgB+xRR9xbIBbJOpiCG+9g+GovaFcWgrQt8dRg5UJp4qhDvyTemZXXpS54QY9EZ6U2xRWtVF9y84bLz5kSggXRYu4z88iw2sd8Iqr78WEG+yM6wmWaZ1qSFstVLiPeiBdLridDYBkBT2ruTIGiIXhBhCY8w7SBQi6lAiC5t5jRj38Lox6fjdm7QTMCkkrx2WT0PhKgCdmjIq+3pNnMD6qfct4MQZOBlOoxkGWNV2uCnDLvpMKGVXIntY6P3BgpGSbSsmXGU16QgWXfUxVW2CoUvd4ESe\\\\\\\/XU4WBaB5AwLa4SKI1Uut9ls0sY8XEueUB4yExgxuKhDCLmOufVJhZpbDd+l7EFvxSibE\\\\\\\/u56AulzbPebaAnV5cPskTCWqZ4VKnFdWGAX4ILJMQSyRqvGo8UhFxjoxpC2MTVVqy5qQ9IozeaEnSXjSgk4GMLccFUPAqhEPQ+XOE2DZY\\\\\\\/sTJklk2AWDH0n7i1ELlZNOttlLt+MVbLYsVczsme5+t5Dnkdz7It06LYCyETEe9KqTCa8Z5+sS2ZQDpsRHbpNQeLapIHRmJNzmkex4l5ltzDRq9Zo\\\\\\\/fpI2WVTDb6W2sbQ7dtj5lY8QhBaTWUWAtNgpCT4eXy9vBDZ8Fdvs8LsEhyah2buH3yDBGm5UZiP8LHK36VpyjA\\\\\\\/yj5TQ51nA9z\\\\\\\/+8eUXdmeyzidknmyrwFHdQP2At9XjFWtGg4mvvukD2n4CBGKumiJi2L+fmWb9q4n4BmErEfBuVVP0gX9nyNXsNd4sMMMwy\\\\\\\/aurhS\\\\\\\/RLUfT3WXDKQiqjPf2+UYA1VDEAnx+IzSA\\\\\\\/9wlnCSSoHcv20YUylZQmUmoy4+wAUIjWB9tEyzfZVca8+3AlG\\\\\\\/FpkS7em4lcp3jkfIN7pUo4rrCmu0piyxmoSZ0sFzMtJapPKDnasjQ2ulooA2+fBwaVoyeePLMm9pbVAtQVGN1Gga6XGVXDdsdNcsBp02tPmxCvO0IJDi9PLTRvxjmKW35LiLlOiB\\\\\\\/qmMX7QUoQlxMGj8C4twm+wJr\\\\\\\/Cz2JNhGQKwlxmlRaJHxvoJKUM93PX6amZD9va7oDfMubTc3HCBwYG8oUVh\\\\\\\/ZVY10sonWwWFHEmpbgpsLh+m0i+Bco14YccYeDfzR+UfjznwFFfiMtDfuyx9Q\\\\\\\/J47DGZffdx0XbuH1CDKoPkBIJbQS8WlR6KpydpSMBU38\\\\\\\/fsC6XqjlELyLLhmPjyLfai8Mco1G7ZYPfKsiuLtOHm0LTzZyzAwpqbR3fwFIqCortWID5rd4PJ6vfFdJ6EJwejyTKLuYkLoq0EIsv3f09+gnp09ll9ZrMKwA\\\\\\\/iVezv2Cwk3l4Q9UuRTmVhxk2kWWdZugZtmY2toLdMcHat\\\\\\\/nNCbGSDjpVaGVKgOlACj3vj+u57CttWhK\\\\\\\/rJ5zAoniAk3NqhCUHRAAgUx+2CIv4W76Pw2ArmlKeQLI4x1c50orwJXkSLeKA6kIneOKe4SnnmTg0OrrswrBR+kCzUySxACmW0RHkMSX4BwTipCF3Dq6DElWUfI4HrB9ABtJtdvzyFfv4H8Vx5hyFetmSFZ8G6atL0K9Yt++n02j0H9Rulld1gmhBwWR4lOpSGKZr0TWO0RdRh714hXiehyKhAIMFqxJVcy4lYQcTR1uRP0t1hmu4FtfB6SkE91hkw\\\\\\\/To9GGlI8LLF0XN6KYaw16tKE7JS5KcgHaw9pFOy+myqbCfzghhAApBJWNbB8NqrEyYm32E9ELjk7effo5gblVTR0DMLv4+QnZatlXqWwWpaNOv+TpW00RB8DoItzO27ydst35g9SeWrXk9g1UHBzk1Bvsh4uJYmurAFxkGkxw3WSkjG+8fj8QOo21bd1d7uhoOEAw0NPZsedf5DPZab5zYH1ZSRPPs2rhi5zZ0rcZj2ojfHWNfq+Su1V9gTawpCrun4ab81yg3FG1ERsSG+4EMvnopz2BGliDy+E1uVkE0JkV7xb5ZwjueFA4UvFnYgHNYM1vNk7mKft+SOQD0al0\\\\\\\/MR5ImlIe4E9\\\\\\\/mPB2GFKFaoA953v3JtAiI6JWnSu\\\\\\\/BU0wZyCUYSX5PIEwIn7BTjgEq1MxY2B+CM7dd1knGc4CElv8iqFtDabCMxJdK0VHqstw97VF8veJhB9Zez5rei\\\\\\\/uKI38N8z2J3ajAgGKe6a2Tcccz5VakhM+Yi2sYxuuEMfpX+r3UTLWwKiUvKqgoe+4tOxK0QISCEYTY3P2O6F6pOZjqSauJOITq07dcqYtcukhdziZ64s1D5tPaN48\\\\\\\/QRfADKR+mrOARB\\\\\\\/ESn4IOw5UNw0zdabzBvBJAExZcGsx8SxgU7EgAPb0h6CQoZNttZYYrEhaHwEuHY6X2SWpt8NmiyjNnsbs0xxRJ+k+IvPeFverDeBsFADRZ+WhkXqy6tXRgUyYHAs6y55g2iJM9F66qrx0zq\\\\\\\/MXEf9nv0kaE2pGHReGEsYlINvXw6JtzDs\\\\\\\/bG4HctcYoxgC5QawYsUqtDf67cfeSseJq3MLv2kJ+T4N\\\\\\\/SGZHA\\\\\\\/ER0Jy5LOeAbEvAx9Dl1xAAMspNVrUNNllc4LNk0aTexTJq99cWztuAtgdZu4ffraRFmBY4pjRkSXou\\\\\\\/vQ16LYbu0Y+hnfz4saYFw4k3Jr0OtvbwDTl2SxnGm39JIEkesXlSv5pMqSkIx7Ym8tZPO9\\\\\\\/UKA\\\\\\\/t7v8xfMo4p4nu0v5Oz1ceURm\\\\\\\/UE0ew4GHrWhULONRt1me1tS\\\\\\\/4DKI0H9FlUFr+zWojTNwRJl3LWAsTuqnH6VY9H0ULHvAmGRPhs6iKBzQnyXHiMGe8PUy1oI4YGaL1gOQqr3vizUP9fsTidCzZzb7ftyfT94\\\\\\\/b4PpPbbTVM9+zDlxKJbbq8D8bGVTbmpR1ggr2NqLNvtbS9ed4ezqZaAlxyYQjLn8\\\\\\\/zSU26fQuSTW6UoXizUH90g9h1iOhAajiBQS+52RZPnB6Xoeq6AYIKcNhVtd96zNIti2txDNX6hUYINWoW0iyMMiaKFClWEu6NLTuaFiDVLQ1rfkjuXIvkky2F2sTaena6146WMovZ86\\\\\\\/ZxC7DsjGUU5qKWhuxU8r9xneGoy0gB1o5qTy8sMS+5pdGX199\\\\\\\/d0GLlexJQNt3P4onMqJePL204pamfrc3\\\\\\\/i4lAMAyV2WV4tAtrxeRZ5khAjAswzkkgd3vYH9ni7KZ+kZjaEirvWEIehVVT1EhWsi6NmRornzwNY4ACwCHD01LTDLrLVpjSB\\\\\\\/BjcCsR3Fq+no8YNDrOg2pUYHz9BU5yETFw8YIhTR3FJD3giLOanDwwovA4PbzY\\\\\\\/obcVxQLrFoPevqZDPRahlaiN+ueLB1s+JQk6JhCcEwOo+lyeZOZAEbEqe4lmps\\\\\\\/9q0TPc98fFpHTGUhBvftgvuIGm9doCXDNNt4vEJjSKSLdacvwSzYPujiXuhXfSBFx11qxmwq8rydk73hG8tZZni6XCrGp0TgHCfbDZNijqvTpl5y4Uwi5bDQ3GW9aDQHOr\\\\\\\/MEDDCXvwkl8aLoOHr\\\\\\\/6MOsqtEp22KE8R0f5Z0xmphwm\\\\\\\/Ijcs9fdPnqqx0U1A7TaVGSL6Ue1r\\\\\\\/Mv0zhXAin5YZL8Z0ocIdmJM3+xkJNrKZCb4YLTzVxIsf0zAcyPwvw36DHEkkrNKak980tOIdeusopSExXoWWhPjBSeP3mt3mrkOeQKTvUm90dPYa2AAjrKwrjWg4R6LU\\\\\\\/LTtWIiw12AROysYnlVx7EMWJymEkL\\\\\\\/Jk72e0y8RwOJyoTGr4ivUlF+nI58w9LLAkdsQ0Xub+1tEKddXCAptxs87Bn5VuU1l\\\\\\\/frOWhxcZbfIWuCfeng5dD07GgccgseqBAKr\\\\\\\/O6Y01qRGrlsqFce\\\\\\\/4j2U6bhaPGidE7CDBNfevgR5TRkQp970oFWkIeGIkuRW+7eoJbqgye+Ixx9+EOyrAIVWiAsyJuLPjgZFOgcsP2Gs2cSixnzvI6Dtk1KFkq7tZDdma5PmEC3geA8iDjLGJg3oCdhWc6T+hAzecjWAybnxDRIAVtRTg3FDTu16zXurb8EpvuUdfg49X9VMINSY\\\\\\\/Q6nulTkHWd9nIpXkCdxV5PThO6JVZsURw\\\\\\\/r8T32b7jRqmV347096wSl\\\\\\\/JBDiytgqfP9vwJGVYLpdVJjkvgfDDN0+6NVIkf8xSWC1IVrNVKKoEkJbhp1qeLUu1FE8OBfsVdx+l7SSP\\\\\\\/jZtL776i5n9WITC1nkiQVnKzDQVLAtouSLw486HpkzB7Z7sLJ+T9kVgPcUr3ilmYjxXDu8LQ7Xw1NnZcA\\\\\\\/XxpECJCST5hLgNW3a\\\\\\\/9QT2rkfZ6o9alHYvucHZWMUxIREP+531RFH\\\\\\\/3aRU6FeyEmWxt8kAOyzCaVP9dNe7iaJ9Q24q8pjvwER7bD0Ipc7BCaBf1SONKhBk8KHWwFUnvtL2kiBdTlhRV6GeJMi5ij82T88a+WWVUJfgtpOzLPOad2guAVpqiS270GHvYZDYerkkDofj+8fIpiRWbbyq\\\\\\\/e6SHUmBOOOHTL24FLzky\\\\\\\/uPMR\\\\\\\/jgFksIXt5OSnDe25o4DFH3vU9A90EfU9MxiFkn9xFaUJZ1ewT5AHm+Ng2\\\\\\\/yX8hrG13mpc6zhQ7U9bwzVKHJwTESGMpxQbFuHrpbfL4EybeeP\\\\\\\/q3S0hnKaCPRXEcF2Y4+g3mDIJy\\\\\\\/4nY7ngFKJSQeifg+Q3Cdi\\\\\\\/6\\\\\\\/XoPcC7DmcybIkZbNu49tA8W75meREA8qHjLm\\\\\\\/g4bNAXkAvWTnI7tjEcjEihyg4\\\\\\\/o0bJopFhIGn3PsApXwiordJ+GI\\\\\\\/YHbpiih3mbY7VuCppH+q1UhTL81s3RxndoQHTwA18tc0JIV2cWXOHmrOs84hy3z4kebMUoI9BWdAsqzfNrqzpMS03YJ8zFEa+FXoR4qA4XNCvsRvLTs1NQ11smwae7tGvtjwCR26nvSui4gH6wtb0\\\\\\\/mO1779vbOWnPwM8RC+ZRlzRGL5dga9gO4hJo+dU\\\\\\\/d6zGp+mgCom89qwhDyRHtns0UL6RipiZP2lIXiFBkQzIrhVqb8fCx4dHnQfZDNIySafhpUwEvcJVcLeuv3MrHk9y9p6sCs8fv65vFPm0ez16N+W0onRvqRkCf9QwlzKthwibhdYAH9HSPWVU7eZ2KQOWtx4JoHVOzbQw8GwftUXGHKcNgbdAIswRxXcCDB28RNMsY3A4B6x560Y3JucEcCpFAp5Zw9wxClg5OvWqmN6tEci6nlIFkOHn6TgSDCNMb+MI\\\\\\\/6baN\\\\\\\/qB76aqY3yzJrdM8APa9fsYOVvGaI3xTb7kjR\\\\\\\/1j5gEAr8bM+L6evK27TAtQ50umf0Xn81k90o414Iy1yPu\\\\\\\/N9EXZLIta6e8dn8Nm\\\\\\\/1ukOwfXoP6JxCr98wey+Xo\\\\\\\/pOmeP+tSZSG2UwO0tH6+1deZ\\\\\\\/oVTrnYrZcXE\\\\\\\/Voun272NMkyNZk1N3KkoufmhJ+Z8lB2qaMc9BJhKk56L\\\\\\\/fyHkzvf5aeCUX0vSM01Z31VECq\\\\\\\/McRMtq3qeZzZ87LRGY5yasyZcnJpFxbGeZrOLOKpF1vyER3VmTg9RZGXvn+gZcSdYu9zhL2DaGmxqe7n+ghf1EbTOTiCrD\\\\\\\/Nu\\\\\\\/ooJXkoyPecPl5RYfppBPsxgxaYCv1nDmU3WOpjGV7tQqJ61cVV3vHWp2y74dahqhBMfwTLH7EpAxG9xZjy76cdj+lw5wJB358PAMF0gxEVnzFPZ0QEGlAxama86ey9\\\\\\\/oVhqtHFA7nOoKurjiQSt+moPKlWxY+8LzidpG1pNjniA7V6BK7Dv0vBMUmuBJHKrRNMsSwVnJG35HqCHc8WqlOQfBqAF+a4Bi6n99bWTbDR9J2lU94mYXIEnPqYQMjjGg9exFRnUi+qIsJdWRVaIvIlp3uVOiLaDhaNH5sI89J2XxfUXJnhLFOxcZOSU0yoz209UQSD6GiiW\\\\\\\/OHDHkOwcnADyKZ\\\\\\\/rfv7ie2zPIEa\\\\\\\/wvOXnR2NJjhor1nUXUEJZChulmLRt1yLU9UCjW49UetGGkUxUTGaJanCm0YHiKItMOLztpY3MVMNj5iqSOsDZy3TfC+R2Xd3ZMR0EHz0IzzP2UhDAwa+OAydEUzMZky\\\\\\\/cytKffj89BWIrro54VZs3ngjE7icZHV+FeyP+wfi0Ato3Ejng7+pPC+BA8xhEr9ccIZw8Vf3HdIetPotr+djoBgOFC+MWWKHK+Q40r64jKWZU6fiSv6BkP4GzIhzTcEJ+sdXaMREOkiqPh35rykCfqVtJ8ihpQ7QhxUnz\\\\\\\/HsFkn+vxI8AzuAJpUuwkU5N6Qa7H+TB4sd9uyMQ2\\\\\\\/3zpWfWv\\\\\\\/rYbrno2v4+SfHUuKHfRqosGhBQzxrsct3Ub77bvf9Um1AwsDmF+AZBEd\\\\\\\/HXIraKgarGkHsuVeKNwTP5BaDi4P3TBknfKQjt0i3+bMoXaRps9XutjYcnrQiWM3qxMGZKKWan0VzXARJtNQ80Yh45CucZLl8WZAznjT85dZ6DmZflXNOb39Ew4LYWHKCV5gFwe03V1Te89Fox5OeHCyTkw6fQyQlsVLx\\\\\\\/hBMZCyH09GuGdj81ZRpZh5p6P7TFTX\\\\\\\/lc+ynXsy0cpU2lNVRsWTH2bpdhPyiq7Ji3f4wQNgUJYApZhwoXGw4g8JFiW1uGEBsV9juEEmX1MZvJ9BjJ\\\\\\\/uNttZwngHL1O43lSegmhBhGL+2p87wntr1RHTVkWxEQXglJlemNhFWn6T+qu7poCWdbifd11AZTM8FSZJvoxYdzOxLj5jJY9v2YKWOPUzhTI7u1VZ\\\\\\\/NsVRpl+1\\\\\\\/CTcfmS12v7AhqhP7omWe8YBMI8eUezT6a+AJBYCHQXAs7en+cQQ2q3AhtneveUuTrqsSlxD1+05FlR23zx+fGhUrTNigPb\\\\\\\/hCMHz0oSXpfy5Rgmxx2jj5VSLD60S+g6tz7+WJ42oRXsoWTOtAfvYqDFR1Jy6JefQGN7cVHL6JnQanO9RoCdG9iME\\\\\\\/kIC1iLpMjlzrO2PKNNiP0Teo6zYvS+KcQBDlFQ4+070DnErZPlWswyJ9WV40MZeFSc+XLFp6rCPyvCf+fxtxAUSm0U\\\\\\\/hkHIMyjkt69oEonYPEEunysa0v\\\\\\\/oWzbXeOKIzRYdKqGelO6mTYQc4ljsynVpEHhsxRiMmb80YttDftqv3fDJbwHXX\\\\\\\/zqWYnsfSvVSNj4R10R7GS7VpAk9sD3kyirVsihHV9U2AsJqDXkrmq1CbEKOymHf3VnGlCwns\\\\\\\/goiyJFhpZh92MsmO0pVk9nYxYEjBYYwwaMhds47128MKkoaRtM061rVFpk\\\\\\\/4P22zT5afkemw2Up1Wt3KN\\\\\\\/CViCQpWqIvC45P5bd+knp3liNODVG+z6CLiGHsCLVadBJoIuNJ\\\\\\\/oa7mAeCZYKH2yMi6VZdwDP\\\\\\\/jVZ2e2xT43hTIWCMC8Rq\\\\\\\/CHaDuqYU3CJr370euAqBpOd+uQ\\\\\\\/J5LlEgFHo9GJ02xzGHVLV7\\\\\\\/vwfUWbRSthqcv0zVKPJi30wZ36LMhJd2DoYOjNHbvs7w3IwKVstz1CdMTQiRKdpVxSV9vLR5UxacqBOB1CTey\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022S7Izjgmx7dfdWd6+Bw\\\\\\\/ebw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022be45d8c6a16d52e83ee33a77c6b9e9856825c706481383065dcebb37af3737c8\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MHYoZHNLH4GnU7FLvd7a7Ub6fnmFlqP5MPz8mqcUsUccnDPbJEt6QPtAfmr2fnh1+tC5W7eykEjxdK73oFrifN6O8CNQV4CwrRdS+OxJMOYVnflG9C9zf1\\\\\\\/+VSd6vrZ4toi1aQounSFZkMu9Cv2KsmlaImAEav10XxFQeU24E7hQu\\\\\\\/upvCin0hsfqx8IwJOlO8XqmUDsYrMMCJMl28v+aBzscVluIlX4lVX4y4zuv7t2HQ5ZVYIoD6RO6bQ0tdQ5T5ScIiUY7xGcMIOuW0E2Q4MtJMcFXYCyiD3kl6k3Rvv9J2N9nTWZxgW8l+p7OOdFF46fqf6bE\\\\\\\/jgsmBzQDhgnZ94vuMbKOcJAfVpH1FGa+ABXZ6rw6cTRQp6meeHkgwnoU2bwBU7jo11AiQUtzesRqiPPUDYsUs\\\\\\\/WYsBaqsb8U5dV+o0O4VTjRe3xKeQup+sTYOOji2AvKeqi+crVmOsqmu88hbhH4paWk+QODEi+woJulvebHma\\\\\\\/K+4YfKpQpM2Fb1qte17HrQvo\\\\\\\/0Mcx854bHVQ5UpOl0ajychGcqWnhHOOQcv\\\\\\\/mL1B0G1F4CAvpgtGG\\\\\\\/WOMdk2lujGZXCYijdKm1Tg1rC4rNMjHrzX4P3MuL2exFism++qilMk4OgEL\\\\\\\/D327unwKFmCYIKouSk8VHNZuExdf7Mbn\\\\\\\/mxealsBIg4TQf75Cr5Ix\\\\\\\/Lx4HUW5OqXeRxSDLnapb2YXM7M0cgQF9jJnw9QTigEX1rERUkcUtlan4aguoOdc2kXD9yTakRYPkG7R+3vQiku3nhFKaT2baIaE1xPronvMqr7D+xSP9fg2YSPvo3oyenoZT+4C7lJ4COizlN9azRTjV+Vt\\\\\\\/lalHIfmGrc4leDhYzhCiLlNAy2V93u1OYM6lVd\\\\\\\/qU6oW3w6sJukS2rk3+T7QEpmS\\\\\\\/TY0cFIFNxYF0kr71yaDAHYGHa8ZrWXek4HpFToZhsZLotGLTJR+K3xD\\\\\\\/\\\\\\\/4f+pyTdeeUJ20mbGGeAxm5GyzwkhgtCvbFD9uOom7v2Nuw3eFrVvXMBGvPhgMO2gtxCKZ0AfpuFK8b3B7IK\\\\\\\/4ijrY3oB\\\\\\\/1WWqPfAgFO1Jx+VHxm+FF1eiHMiUPoV60EtAvwuAbgaAKs9SysWHDSKdu4eZ\\\\\\\/sNpJ79H4DK7qDtMUCG27of4sYvMt6qxXQk6SmNvnQ5k6Xy8xNbyUkOFlQoAg+YYYlS55PZFnjXIrQI0U9F4io0Ijk9rC6uORQr42+t1dRxFdC01ZZ4KJ+HyQFS+gUPAApX6eL2n38f+hJ+DFplIFGtxZLbHF\\\\\\\/o4QnH7toIWg6CwwdHgLViDh+nrLlWNOoWVo4Pv1Jc4qy9UqeEG995oy1zC2cbycYoXQjkHkYv8ceNi2xPKWeN1BPujmi36+Waqu+2rBtBU1HM7SKZNSeovDi1z4Q4wF7surl+aNm\\\\\\\/9A4Srd8jk+5eM8RRy4y5sUL1nu5QcXKPL3NUGIdPx3gUBSJaFMUbSfshB64Qlw4eV1ASNzST2pAD9iVbvrcbZ+FCgRTSBi+9UhJVrcfXT4pTBlHGO+hGqSr83o5lYCDoG9a7hKerjZd6rTqZlfj1PK1aIp0RhnD9g9S6f6RHa5H1fnLrI0B0hkdY8VvE2khScV4YiIxEO1ONLNbXFdJMHVzUjuTbUVrrbIB+CG\\\\\\\/ngMImGevcPpMu+x5rw+4Yr0wMqDNT8onSvOwbVTUzr1hOPsKlzUcE\\\\\\\/tkKieXscw47MMbMJgxQzFI1nvnO3XsnRAm8Mn5bpRBBzlnNYD75Sg9DQj3+KzSq\\\\\\\/Crszq5z7jKAv+wHZ2IG+7GQ5H2Jdi7TCeU9aRnPERykPRFnzR1EbLCMg+\\\\\\\/CZbnek7Atu1Hvtzpdnas59UKEv\\\\\\\/2hytz0Q7uMP2bG+aP7QuTw1tOxmjTvyVwsb\\\\\\\/8j1frU+tJJNasnTnliRtw\\\\\\\/A11qVr5CDFi6cip60QjAN8W5K85LNSn59e5nD2W+HNbt1FFYW1ZwrBvBSYFnu\\\\\\\/AH1DxqqQecCU\\\\\\\/fFwOx\\\\\\\/mKbVGXOuDGKlYSziTU9gKaeTVtF3hDxCqE8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022yFhyHsGWZAdPPtT89Yzm6g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223445b795a4f30571648aaf85177f05521acbfee0dbef05e816a687346f2fb0c1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485831 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2685" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QrLUvkDnH2be8UTrYhMjXAWd13s9gD62YutryYqDS74" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2685" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cNeIGqLqpazhWXIqtvaLLbISGWWiep22dJC7rNiiUCo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2677" title="Evidence for Adaptive Introgression of Disease Resistance Genes Among Closely Related Arabidopsis Species" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2695" title="rRNA Pseudogenes in Filamentous Ascomycetes as Revealed by Genome Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485831" id="node485831" data-pisa="ggg;7/8/2685" data-pisa-master="ggg;g3.117.043596" data-apath="/ggg/7/8/2685.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Accuracy and Bias of Single-Step Genomic Prediction for Populations Under Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Wan-Ling Hsu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8640-5372" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dorian J. Garrick</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5821-099X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rohan L. Fernando</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2685-2694;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043596 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485831" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Wan-Ling Hsu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wan-Ling%2BHsu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hsu%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWan-Ling%2BHsu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dorian J. Garrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Iowa State University, Ames, Iowa 50011</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Veterinary, Animal and Biomedical Sciences, Massey University, Palmerston North 4442, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorian%2BJ.%2BGarrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garrick%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADorian%2BJ.%2BGarrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8640-5372" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dorian J. Garrick</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rohan L. Fernando </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rohan%2BL.%2BFernando%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernando%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARohan%2BL.%2BFernando%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5821-099X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rohan L. Fernando</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rohan@iastate.edu" class="" data-icon-position="" data-hide-link-title="0">rohan@iastate.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/8/2685" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485831" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485831/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2685.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485831" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485831/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2685.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485831" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485831/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/8/2685.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485831" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485831/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In single-step analyses, missing genotypes are explicitly or implicitly imputed, and this requires centering the observed genotypes using the means of the unselected founders. If genotypes are only available for selected individuals, centering on the unselected founder mean is not straightforward. Here, computer simulation is used to study an alternative analysis that does not require centering genotypes but fits the mean <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-1.gif"/></noscript></span></span> of unselected individuals as a fixed effect. Starting with observed diplotypes from 721 cattle, a five-generation population was simulated with sire selection to produce 40,000 individuals with phenotypes, of which the 1000 sires had genotypes. The next generation of 8000 genotyped individuals was used for validation. Evaluations were undertaken with (J) or without (N) <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-2.gif"/></noscript></span></span> when marker covariates were not centered; and with (JC) or without (C) <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-3.gif"/></noscript></span></span> when all observed and imputed marker covariates were centered. Centering did not influence accuracy of genomic prediction, but fitting <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-4.gif"/></noscript></span></span> did. Accuracies were improved when the panel comprised only quantitative trait loci (QTL); models JC and J had accuracies of 99.4%, whereas models C and N had accuracies of 90.2%. When only markers were in the panel, the 4 models had accuracies of 80.4%. In panels that included QTL, fitting <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-5.gif"/></noscript></span></span> in the model improved accuracy, but had little impact when the panel contained only markers. In populations undergoing selection, fitting <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-6.gif"/></noscript></span></span> in the model is recommended to avoid bias and reduction in prediction accuracy due to selection.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/centering-genotype-covariates" class="hw-term hw-article-keyword hw-article-keyword-centering-genotype-covariates" rel="nofollow">centering genotype covariates</a></li><li class="kwd"><a href="/keyword/estimated-breeding-value" class="hw-term hw-article-keyword hw-article-keyword-estimated-breeding-value" rel="nofollow">estimated breeding value</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/selection" class="hw-term hw-article-keyword hw-article-keyword-selection" rel="nofollow">selection</a></li><li class="kwd"><a href="/keyword/single-step" class="hw-term hw-article-keyword hw-article-keyword-single-step" rel="nofollow">single-step</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-3">In pedigree-based analyses, the expected value of breeding values is zero. In order to achieve similar properties in whole-genome analyses, marker genotype covariates are often transformed. When all individuals are genotyped, it has been shown that inference on genotype effects does not depend on how the covariates are transformed (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Strandén and Christensen 2011</a>). However, when data includes genotyped and nongenotyped individuals, inference on marker effects from single-step analyses may depend on how the covariates are transformed (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Fernando <em>et al.</em> 2014</a>). In single-step analyses using marker effects models, the breeding values of nongenotyped individuals are partitioned into components representing the prediction of nongenotyped individuals conditional on their genotyped relatives and an independent deviation (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Fernando <em>et al.</em> 2014</a>). The prediction of nongenotyped individuals conditional on their genotyped relatives is done based on best linear prediction, which requires the first moments to be known without error. This is straightforward if the mean of the genomic breeding value is zero in the absence of selection. Centering the observed genotype covariates using what their means would be in the absence of selection would result in genomic breeding values with null means. However, such genotype covariate means are typically unavailable. <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Fernando <em>et al.</em> (2014)</a> proposed a solution for the marker effects model that involves fitting an additional fixed covariate that estimates the mean <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-7.gif"/></noscript></span></span> of the linear component of the genotypic value, which is denoted by <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-8.gif"/></noscript></span></span> in Equation 1 below, in a population where selection is absent. Using this approach, even when there is selection, the selection process can be ignored, because the analysis is conditional on the data used for selection (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Goffinet 1983</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Gianola and Fernando 1986</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Im <em>et al.</em> 1989</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Sorensen <em>et al.</em> 2001</a>). In Markov chain Monte Carlo (MCMC) analyses, centering results in better mixing (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Strandén and Christensen 2011</a>), reducing the number of iterations required to obtain converged genomic predictions. In practice, centering the entire matrix of genotype covariates, including the observed and imputed genotypes, using their sample means can be done in addition to the <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Fernando <em>et al.</em> (2014)</a> approach of fitting <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-9.gif"/></noscript></span></span> This type of centering of the entire genotype matrix does not affect inference on marker effects.</p><p id="p-4">The same issue with centering of the observed genotype covariates that we described above for the marker effects model is also implicit for the single-step breeding value model (single-step GBLUP), and a similar solution was proposed by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Vitezica <em>et al.</em> (2011)</a>. In their proposed solution, the observed genotype covariates are centered using their means; in addition, the genomic covariance matrix is corrected for the change in the mean breeding value of the genotyped individuals (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Vitezica <em>et al.</em> 2011</a>). It was shown in that paper that this is equivalent to fitting the change in breeding value due to selection as a random effect.</p><p id="p-5">Here, we use simulated data to compare the accuracy and bias in genomic prediction applied to populations under selection with and without centering the entire matrix of genotype covariates, and with and without fitting <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-10.gif"/></noscript></span></span> as a fixed effect. Further, we show that when the observed genotype covariates are centered using means calculated from selected individuals rather than means from all individuals, the meaning of <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-11.gif"/></noscript></span></span> changes from the mean of unselected individuals to become the mean breeding value in selected individuals, as claimed by <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Vitezica <em>et al.</em> (2011)</a>.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Theory</h3><p id="p-6">To simplify the presentation of the genetic model without loss of generality, we will assume that the unconditional expectation of the phenotypic value for all individuals is the same. Let <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-12.gif"/></noscript></span></span> denote the row vector of genotypes for individual <em>i</em>, which is often coded as −1,0,1. Then, under additive gene action, the genotypic value, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-13.gif"/></noscript></span></span> which is the expected phenotypic value of an individual with genotypes <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-14.gif"/></noscript></span></span>, can be written as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-15.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>β</em> is the value of <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-16.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-17.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-18.gif"/></noscript></span></span> is the vector of substitution effects. The scalar <em>β</em> and the vector <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-19.gif"/></noscript></span></span> are constants, but <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-20.gif"/></noscript></span></span> will be a random variable because of randomness in <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-21.gif"/></noscript></span></span> owing to the randomness in the genotypes for a randomly sampled individual. Note that the expected value of the linear component <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-22.gif"/></noscript></span></span> of the genotypic value in Equation 1 is <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-23.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-24.gif"/></noscript></span></span> which may not be equal to zero. Thus, it is customary to write the model for the genotypic value, as can be derived from Equation 1, as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-26.gif"/></noscript></span></span> is a constant, representing <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-27.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-28.gif"/></noscript></span></span> is a random variable that has null expectation, which is the breeding value predicted in a pedigree-based BLUP (PBLUP) evaluation. When genotypes are observed and used in a genomic analysis, they may be transformed or coded by subtracting their expectations, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-29.gif"/></noscript></span></span> from the observed values, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-30.gif"/></noscript></span></span> In both Equations 1 and 2, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-31.gif"/></noscript></span></span> is the same substitution effect for locus <em>j</em>. The intercepts in these models, however, are different. In Equation 1, the intercept is <em>β</em>, and it is the value of <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-32.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-33.gif"/></noscript></span></span> In Equation 2, on the other hand, the intercept is <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-34.gif"/></noscript></span></span> and it is the value of <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-35.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-36.gif"/></noscript></span></span> More generally, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-37.gif"/></noscript></span></span> is not known, so genotypes are coded by subtracting a different vector <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-38.gif"/></noscript></span></span> from the observed genotypes as <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-39.gif"/></noscript></span></span> Still, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-40.gif"/></noscript></span></span> is the substitution effect for locus <em>j</em>, but the intercept will change to become <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-41.gif"/></noscript></span></span> which is the value of <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-42.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-43.gif"/></noscript></span></span> Thus, as more rigorously shown in (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Strandén and Christensen 2011</a>), inference about <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-44.gif"/></noscript></span></span> does not depend on how the genotypes are coded. A simpler but rigorous proof is given in the Appendix of this paper.</p><p id="p-7">In single-step analyses, where some individuals are not genotyped, the missing genotypes are imputed either implicitly (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Legarra <em>et al.</em> 2009</a>) or explicitly (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Fernando <em>et al.</em> 2014</a>) using best linear prediction. Let <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-45.gif"/></noscript></span></span> denote the matrix of genotypes for individuals that were genotyped. Then, the genotypes of the individuals with missing genotypes are imputed as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-46.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-47.gif"/></noscript></span></span> is the matrix of pedigree-based additive relationships between the nongenotyped and genotyped individual, and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-48.gif"/></noscript></span></span> is the matrix of additive relationships among genotyped individuals. Now, the model for the genotypic values, when genotypes are coded as in Equation 2, becomes<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-49.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-50.gif"/></noscript></span></span> is that part of <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-51.gif"/></noscript></span></span> that cannot be imputed from knowledge of the breeding values of genotyped relatives. In practice, the true value of <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-52.gif"/></noscript></span></span> is not known, and data for its estimation may not be available. Rearranging these equations in terms of the uncentered <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-53.gif"/></noscript></span></span> rather than the centered matrix of genotype covariates <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-54.gif"/></noscript></span></span> results in<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-55.gif"/></noscript></span> </span>and substituting <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-56.gif"/></noscript></span></span> as previously defined, and letting <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-57.gif"/></noscript></span></span> results in<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-58.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>which suggests that <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-59.gif"/></noscript></span></span> could be treated as an unknown constant and estimated as a fixed effect from the data (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Fernando <em>et al.</em> 2014</a>). The covariate vector for <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-60.gif"/></noscript></span></span> is denoted by <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-61.gif"/></noscript></span></span> for nongenotyped individuals, and by <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-62.gif"/></noscript></span></span> for genotyped individuals. So, Equation 3 becomes<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-63.gif"/></noscript></span> </span>which can be combined as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-64.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-65.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-66.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-67.gif"/></noscript></span></span> and the <strong>0</strong> matrix in the last term of Equation 4 is required because <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-68.gif"/></noscript></span></span> does not appear in the model for genotyped individuals.</p><p id="p-8">When the vector <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-69.gif"/></noscript></span></span> represents the substitution effects of a large number of loci containing positive and negative effects, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-70.gif"/></noscript></span></span> will tend to have a value close to zero. Accordingly, we have simulated some scenarios with positive <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-71.gif"/></noscript></span></span> so that the entire <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-72.gif"/></noscript></span></span> vector is positive to exacerbate the impact of <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-73.gif"/></noscript></span></span> Nevertheless, when marker rather than causal alleles are fitted in the model, the sign of the substitution effects depends on the phase relationship between marker and causal allele, which is equally likely to be positive or negative.</p><p id="p-9">Even if <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-74.gif"/></noscript></span></span> in a population undergoing selection, it is expected that <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-75.gif"/></noscript></span></span> in nonfounders. Suppose <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-76.gif"/></noscript></span></span> is the mean of the observed genotype covariates in such a population undergoing selection, and these means are used to center the matrix <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-77.gif"/></noscript></span></span> of observed genotypes. Then, the model for the genotypic values can be written in terms of the matrix <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-78.gif"/></noscript></span></span> of centered covariates as<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-79.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>and using <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-80.gif"/></noscript></span></span> for the covariate corresponding to <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-81.gif"/></noscript></span></span> Equation 5 can be written as<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-82.gif"/></noscript></span></span>which can be combined as<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-83.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>Note that the regression coefficients for <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-84.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-85.gif"/></noscript></span></span> in Equation 4 and <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-86.gif"/></noscript></span></span> in Equation 6, must be equal. This implies that <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-87.gif"/></noscript></span></span> Similarly, the intercepts of these two models must be equal too, and this implies <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-88.gif"/></noscript></span></span></p></div><div id="sec-3" class="subsection"><h3>Simulations</h3><p id="p-10">Phenotypic and genotypic data were simulated using XSim (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Cheng <em>et al.</em> 2015</a>), based on haplotypes from 10 genomic regions of 721 US Hereford beef cattle that were genotyped with the Illumina 770K BovineHD BeadChip, and reported in terms of the number of copies of the A allele at each locus. The selected regions came from choosing every 10th single nucleotide polymorphism (SNP), starting at SNP 5001, to get a low-density panel of 200 SNPs from each of chromosomes 1–10 (BTA1–BTA10), after eliminating SNPs with MAF &lt; 0.01. These 2000 SNPs represent 10 0.1 M chromosomes. Average linkage disequilibrium (LD) between adjacent SNPs was 0.300. 50 SNPs from each chromosome were randomly chosen to represent QTL. The QTL effects were sampled from a normal distribution with mean <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-89.gif"/></noscript></span></span>= 0.2 and multiplied by the number of copies of the A allele to produce the true breeding value (TBV). The TBVs were added to a normally distributed residual term scaled by the sample variance of the TBV to simulate a trait with a heritability of 0.5. The first 20 SNPs from each of the 10 chromosomes were also used to simulate a smaller panel, with 5 QTL and 15 markers per chromosome. The average LD between adjacent SNPs was 0.289. TBV were simulated in the same way as for the 2000 SNP scenario, then scaled to simulate traits with heritabilities (<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-90.gif"/></noscript></span></span>) of 0.1, 0.3, or 0.5. An additional scenario with <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-91.gif"/></noscript></span></span>= 0 was used to simulate TBV for a trait with heritability 0.5.</p><p id="p-11">Half the observed diplotypes from US Hereford cattle were assigned to represent males and the remainder to represent females. Those 360 males and 361 females were sampled in pairs, with replacement, to produce 4000 male and 4000 female offspring representing generation G4. There were no mutations. Four more nonoverlapping generations of random mating were carried out, with one male and one female offspring per dam mated to randomly chosen sires to produce the founder population (G0).</p><p id="p-12">The G1 generation was produced by mass phenotypic selection of the top 200 G0 males, and this was repeated for five more generations. Each female was randomly mated twice to selected males to produce one offspring of each sex each generation. Across nonoverlapping generations G0–G5, a total of 48,000 individuals with phenotypes, genotypes, and TBVs were simulated for each scenario.</p><p id="p-13">The training data included phenotypes from all individuals in G0–G4 (<em>n</em> = 40,000), and genotypes from all 1000 sires and all 8000 G5 animals. Fixed loci, if any, were filtered from the panel before genomic prediction analyses. The genetic and residual variances used in genomic prediction were the sample variance of the TBV in G0 and the corresponding residual variance used to define the desired heritability in the founder population.</p></div><div id="sec-4" class="subsection"><h3>Models</h3><p id="p-14">Five statistical models were compared for differences in accuracy and bias of prediction. These included models with or without <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-92.gif"/></noscript></span></span>, and with or without centering of observed and imputed marker covariates, and a model that used pedigree relationships but not marker covariates.</p><div id="sec-5" class="subsection"><h4>Mixed linear model:</h4><p id="p-15">Accuracy of PBLUP was quantified using the correlation of TBV and estimated breeding values (EBV), where TBV was as simulated and EBV were obtained by fitting the mixed linear model (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Henderson 1973</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">1984</a>):<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-93.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-94.gif"/></noscript></span></span> is a vector of phenotypic observations, <strong>1</strong> is a vector of 1s, <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-95.gif"/></noscript></span></span> is a general mean, <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-96.gif"/></noscript></span></span> is a vector of random additive genetic effects, <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-97.gif"/></noscript></span></span> is a vector of random residual effects, and <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-98.gif"/></noscript></span></span> is a known incidence matrix relating observations to <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-99.gif"/></noscript></span></span> In this model, <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-100.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-101.gif"/></noscript></span></span> so that <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-102.gif"/></noscript></span></span> Further, <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-103.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-104.gif"/></noscript></span></span> being the numerator relationship matrix, and <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-105.gif"/></noscript></span></span> so that <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-106.gif"/></noscript></span></span></p></div><div id="sec-6" class="subsection"><h4>Single-step Bayesian regression model:</h4><p id="p-16">All genomic EBVs (GEBV) were obtained by single-step Bayesian regression (<a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">Fernando <em>et al.</em> 2014</a>), using BayesC priors for marker effects with <em>π</em> = 0, which gives predictions that are identical to those from single-step GBLUP. The model was implemented in Julia (<a href="http://julialang.org">http://julialang.org</a>) based on the SSBR package (<a href="http://QTL.rocks">http://QTL.rocks</a>) to construct an MCMC chain of 50,000 samples. Individuals were separated into two groups designated with subscripts <em>g</em> or <em>n</em> according to whether or not simulated genotypes were assumed to be observed or missing. The single-step Bayesian regression model including a covariate <strong>J</strong> for <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-107.gif"/></noscript></span></span> (model J) was:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-108.gif"/></noscript></span> </span>where <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-109.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-110.gif"/></noscript></span></span> are vectors of phenotypes for nongenotyped and genotyped individuals, <strong>1</strong> is a vector of 1s, <em>μ</em> is a general mean, <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-111.gif"/></noscript></span></span> is the expected value of the linear component <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-112.gif"/></noscript></span></span> of the genotypic value if selection was absent, <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-113.gif"/></noscript></span></span> is a vector of random substitution effects of markers, <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-114.gif"/></noscript></span></span> is a vector of imputation residuals, <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-115.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-116.gif"/></noscript></span></span> are incidence matrices relating the breeding values of nongenotyped and genotyped individuals to their phenotypes, <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-117.gif"/></noscript></span></span> which is defined for genotyped individuals, is a vector of −1s, <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-118.gif"/></noscript></span></span> which is defined for nongenotyped individuals, is a vector computed as <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-119.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-120.gif"/></noscript></span></span> is the matrix of imputed marker covariates, <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-121.gif"/></noscript></span></span> is the matrix of observed marker covariates, and <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-122.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-123.gif"/></noscript></span></span> are vectors of random residual effects for nongenotyped and genotyped individuals. This model can be represented as<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-124.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-125.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-126.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-127.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-128.gif"/></noscript></span></span> <strong>U</strong> = <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-129.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-130.gif"/></noscript></span></span> is a vector of random residual effects.</p><p id="p-17">This model was used for four variants of the single-step Bayesian regression analysis, depending on whether or not the covariate <strong>J</strong> corresponding to the mean <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-131.gif"/></noscript></span></span> was in the model, and whether or not the columns in the marker covariate matrix <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-132.gif"/></noscript></span></span> were centered using means of the observed and imputed genotype covariates. The analyses with <strong>J</strong> or without <strong>J</strong> are denoted as J or N when covariates were not centered, and as JC or C when the entire matrix of imputed and observed genotype covariates were centered, respectively (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486124/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486124/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Four combinations of the single-step Bayesian regression analyses</span><div class="sb-div caption-clear"></div></div></div><p id="p-20">Accuracy of genomic prediction was quantified using the correlation of TBV and GEBV (<span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-134.gif"/></noscript></span></span>), where GEBVs were obtained from each of the four analyses described above. Bias of genomic prediction was quantified using the deviation from unity of the coefficient of regression of TBV on GEBV (<span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-135.gif"/></noscript></span></span>). In models JC and J, GEBVs are obtained using Equation 24 in (<a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">Fernando <em>et al.</em> 2014</a>):<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-136.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-137.gif"/></noscript></span></span> is the GEBV, <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-138.gif"/></noscript></span></span> is the best linear unbiased estimate of the mean of breeding values, <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-139.gif"/></noscript></span></span> is the BLUP of the vector of random substitution effects of all markers, and <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-140.gif"/></noscript></span></span> is the BLUP of the imputation residual.</p><p id="p-21">In model J, the matrix <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-141.gif"/></noscript></span></span> contains the uncentered number of copies of the A allele at each locus, and the uncentered version is used to impute <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-142.gif"/></noscript></span></span> In model JC, the entire matrix of imputed, <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-143.gif"/></noscript></span></span> and observed, <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-144.gif"/></noscript></span></span> genotype covariates is centered. In models C and N, the GEBVs are computed in a corresponding manner, except that the covariate <strong><em>J</em></strong> and its coefficient <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-145.gif"/></noscript></span></span> are not included in the model.</p><p id="p-22">The four analyses J, N, JC, and C were all applied to three different genotype panels, comprising the causal QTL plus markers, just the causal QTL, or just the markers. All 12 combinations of four analyses and three genotype panels were applied to data simulated with 200 loci comprising 50 QTL whose effects were sampled from a normal distribution with <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-146.gif"/></noscript></span></span> to construct phenotypes with <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-147.gif"/></noscript></span></span> The four analyses JC, J, C, and N were repeated using only genotype panels comprising QTL plus markers for three other scenarios: 200 loci, <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-148.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-149.gif"/></noscript></span></span> 200 loci, <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-150.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-151.gif"/></noscript></span></span> and 200 loci, <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-152.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-153.gif"/></noscript></span></span></p><p id="p-23"><a id="xref-ref-5-9" class="xref-bibr" href="#ref-5">Fernando <em>et al.</em> (2014)</a> had observed that including <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-154.gif"/></noscript></span></span> in the model was necessary only when <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-155.gif"/></noscript></span></span> and the number of observations exceeds the number of markers. So, to examine the impact of <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-156.gif"/></noscript></span></span> the scenario with 200 loci and <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-157.gif"/></noscript></span></span> was repeated with <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-158.gif"/></noscript></span></span> and to examine the impact of the number of loci, the scenario with <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-159.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-160.gif"/></noscript></span></span> was repeated with 2000 loci.</p><p id="p-24">Every scenario was replicated 10 times with each replicate having been constructed starting from the sampling of G5 which represented simulated offspring from the haplotypes of real animals. Every phenotypic dataset was also fitted using the PBLUP model. All reported correlations and regression coefficients are the means of 10 replicates. These are presented along with the SEs of those means.</p><p id="p-25">In single-step GBLUP (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Legarra <em>et al.</em> 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Christensen and Lund 2010</a>), the missing genotypes are not explicitly imputed, and only the observed genotype covariates are centered using their means. We argued earlier that when the number of loci is large, <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-161.gif"/></noscript></span></span> will be close to zero, especially when the panel consists of only marker loci. However, in a population undergoing selection, <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-162.gif"/></noscript></span></span> which is the mean breeding value of genotyped individuals, is not expected to be zero even when the panel only includes marker loci and <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-163.gif"/></noscript></span></span> So, in addition to the above, single-step Bayesian regression analyses, using the model given in Equation 7 with and without <strong>J</strong> (models JC* and C*), were applied to a marker panel with 200 loci, <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-164.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-165.gif"/></noscript></span></span> when the matrix of observed genotype covariates were centered using their means as: <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-166.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-167.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-168.gif"/></noscript></span></span> is the number of individuals with genotypes. Recall that even when the means of the observed genotypes are used for centering, the model for the genotypic values can be written in terms of the matrix <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-169.gif"/></noscript></span></span> of uncentered covariates as shown by Equation 6. We will compare the estimates of <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-170.gif"/></noscript></span></span> in this model with those of <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-171.gif"/></noscript></span></span> from Equation 4 where the means of observed genotype covariates are not used for centering.</p></div></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-26">The genotypes representing G0 from each replicate and scenario are available at: <a href="https://figshare.com/s/d7798b811a9a6a4172fc">https://figshare.com/s/d7798b811a9a6a4172fc</a>. These genotypes and the methodology described previously are sufficient to reproduce the simulations used in this study.</p></div></div><div class="section results-discussion" id="sec-8"><h2 class="">Results and Discussion</h2><div id="sec-9" class="subsection"><h3>Effect of fitting a genotypic mean and centering of observed and imputed marker covariates</h3><div id="sec-10" class="subsection"><h4>Accuracy:</h4><p id="p-27">The accuracies of genomic prediction as assessed by validation in G5 after training using G0–G4 for a trait with 50 QTL whose effects were sampled from a normal distribution with <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-172.gif"/></noscript></span></span>= 0.2 and <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-173.gif"/></noscript></span></span>= 0.5 are in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The accuracy of PBLUP in predicting breeding values for individuals without phenotypes was 41.5%, accounting for &lt;20% of genetic variance. All analyses using genotypes resulted in more accurate predictions than using pedigree alone. Centering the entire matrix of imputed and observed genotype covariates had no effect on the accuracy of the genomic analyses regardless of the nature of the marker panel. This is because, as shown in the Appendix, this type of centering is equivalent to adding and subtracting <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-174.gif"/></noscript></span></span> from the model equation, and this has no effect on the mixed model solutions for <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-175.gif"/></noscript></span></span></p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486075/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486075/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Correlations (%, <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/T2/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/T2/embed/mml-math-176.gif"/></noscript></span></span>) between TBV and (G)EBV for alternative analyses</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">In this study, selection resulted in the successive advances in mean TBV from G0 to G5 being 10.14, 10.63, 11.18, 11.68, 12.15, and 12.57. The mean genotypic value was not zero in G0 because QTL genotypes were not centered and the mean QTL effect was <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-191.gif"/></noscript></span></span> = 0.2. Since the QTL effects do not change with selection, the advance in TBV reflects changes in the frequencies of the favorable alleles of the 50 QTL. So centering using the allele frequency means of the genotyped sires in G1–G4 and all individuals in G5 does not closely approximate the centering that would have occurred if the allele frequency means had been obtained from the unselected population. By contrast, fitting <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-192.gif"/></noscript></span></span> in the model estimates the relevant mean from the data.</p><p id="p-32">In panels that included causal variants (QTL), fitting <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-193.gif"/></noscript></span></span>in the model substantially improved the accuracy to being near perfect. This is not surprising, given that there were only 50 QTL, the heritability was 0.5, and there were 40,000 phenotyped ancestors, including 200 genotyped sires per generation in the training. However, in the panel that contained only markers with no causal variants, fitting <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-194.gif"/></noscript></span></span>in the model had little impact. This is because in the population that was simulated here the founders were not selected, and thus <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-195.gif"/></noscript></span></span>was close to zero for the panel with only markers. This would not be the case in a population where even the founders are descendants of selected parents. Thus, for traits that have been subject to selection, fitting <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-196.gif"/></noscript></span></span> in the model is expected to improve accuracy.</p><p id="p-33">Using one replicate as an example, for the panel including both QTL and markers, the estimate of <em>μ</em> was ∼9.90 for the analyses using J and N. The estimate of <em>μ<sub>g</sub></em> was 4.98 for the analysis using J. For the genotyped individuals, the covariate values in <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-197.gif"/></noscript></span></span> are all −1, so <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-198.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-199.gif"/></noscript></span></span> is a vector of values equal to 9.90−4.98 = 4.92. For nongenotyped individuals, the covariate values in <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-200.gif"/></noscript></span></span> can vary widely, but many are close to −1 while others are close to 0. This means that <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-201.gif"/></noscript></span></span> will include values that range from 9.90 to 4.92, accounting for variation in accuracy of imputation. When <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-202.gif"/></noscript></span></span> is not included in the model, these effects are ignored, which can reduce the accuracy of predicting nongenotyped individuals. Failing to account for these effects will propagate errors in <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-203.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-204.gif"/></noscript></span></span> the latter impacting the accuracy of predicting genotyped individuals. Collectively, these errors reduced accuracy from 97 to 96% for the panel including QTL and markers, and from 99 to 90% for the panel including only QTL. However, when the panel comprised only markers, the estimates <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-205.gif"/></noscript></span></span> will include both positive and negative values because the phase of markers and QTL are equally likely to take either sign, in which case <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-206.gif"/></noscript></span></span> will be close to zero as confirmed in the above mentioned replicate where the estimate was −1.18.</p><p id="p-34">Here, results from the analysis with only QTL on the panel are used to show that <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-207.gif"/></noscript></span></span> is the mean of <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-208.gif"/></noscript></span></span> in the founder population and not the mean of the breeding value <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-209.gif"/></noscript></span></span> of the genotyped individuals. Recall that the QTL model was used with an intercept of <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-210.gif"/></noscript></span></span> to simulate the data. Thus, when only QTL are on the panel, the true value of <em>β</em> is zero. In analysis J because <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-211.gif"/></noscript></span></span> both <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-212.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-213.gif"/></noscript></span></span> are estimates of <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-214.gif"/></noscript></span></span> and could be pooled, which for the replicate above would be <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-215.gif"/></noscript></span></span> In that replicate, the actual mean of <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-216.gif"/></noscript></span></span> in G0 was 10.16, which was estimated in the analysis to be 9.05. On the other hand, the mean of the breeding value <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-217.gif"/></noscript></span></span> in the 9000 genotyped individuals was 2.4, which is clearly not near the pooled estimate of 9.05 for <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-218.gif"/></noscript></span></span> These genotyped individuals included 1000 selected sires, of which 200 were genotyped in each generation from G0 to G4, and 8000 offspring from G5. The mean values of <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-219.gif"/></noscript></span></span> for the selected sires were 1.00, 1.59, 2.08, 2.50, and 2.88, respectively, for G0 through G4, and 2.45 for the offspring in G5. It is apparent that the <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-220.gif"/></noscript></span></span> parameter corresponding to the covariate <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-221.gif"/></noscript></span></span> is the mean of <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-222.gif"/></noscript></span></span> in the founder population and not the mean breeding value <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-223.gif"/></noscript></span></span> in the selected individuals. In analysis JC with the covariates centered, the intercept <em>β</em> is the value of <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-224.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-225.gif"/></noscript></span></span> which is the case when <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-226.gif"/></noscript></span></span> The estimate <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-227.gif"/></noscript></span></span> was 16.11 in this analysis, but <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-228.gif"/></noscript></span></span> remained about the same value, namely 7.80. This shows that <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-229.gif"/></noscript></span></span> has the same interpretation whether the entire matrix of observed and imputed genotypes is centered or not. In neither case does it represent the mean breeding value of selected individuals.</p><p id="p-35">Accuracy of PBLUP increased with heritability, as expected (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Further, genomic predictions using panels that include causal mutations were near perfect when <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-230.gif"/></noscript></span></span>was included in the model. These high accuracies are a reflection of these phenotypes being influenced by only 50 QTL and there being a large training dataset. Accuracy was reduced when <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-231.gif"/></noscript></span></span>was not fitted in the model. There was no advantage in terms of accuracy to centering the covariates, but MCMC mixing may have been improved, although this was not investigated.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486134/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486134/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Correlations (%, <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/T3/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/T3/embed/mml-math-232.gif"/></noscript></span></span>) between TBV and (G)EBV for alternative analyses for different heritabilities</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485976/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485976/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Regression coefficients (<span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/T4/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/T4/embed/mml-math-252.gif"/></noscript></span></span>) of TBV on (G)EBV</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h4>Bias:</h4><p id="p-41"><a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows the regression coefficients of TBV on (G)EBV for <span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-266.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-252"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-267.gif"/></noscript></span></span> = 0.2, the same scenarios represented in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. The regression coefficients of TBV on GEBV for each scenario were close to 1 with very low SE, which indicates that the genomic predictions exhibited almost no bias. The differences in regression coefficients between analyses were very small, but the marker panel comprising only markers was biased upward, whereas the marker panels that included causal mutations were biased slightly downward.</p></div></div><div id="sec-12" class="subsection"><h3>Effect of mean QTL effect (<span class="inline-formula" id="inline-formula-253"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-268.gif"/></noscript></span></span><em>vs.</em><span class="inline-formula" id="inline-formula-254"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-269.gif"/></noscript></span></span>)</h3><p id="p-42">We had hypothesized that the impact of omitting <span class="inline-formula" id="inline-formula-255"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-270.gif"/></noscript></span></span> from the model would be greatest when <span class="inline-formula" id="inline-formula-256"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-271.gif"/></noscript></span></span> departs significantly from 0, which is more likely to occur when <span class="inline-formula" id="inline-formula-257"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-272.gif"/></noscript></span></span>departs from 0. For that reason, our base simulation used <span class="inline-formula" id="inline-formula-258"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-273.gif"/></noscript></span></span>= 0.2. Results are shown in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> for the panel including QTL and markers with <span class="inline-formula" id="inline-formula-259"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-274.gif"/></noscript></span></span> = 0.5 for <span class="inline-formula" id="inline-formula-260"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-275.gif"/></noscript></span></span> = 0.2 compared with <span class="inline-formula" id="inline-formula-261"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-276.gif"/></noscript></span></span> = 0. These results confirmed that the benefit of fitting <span class="inline-formula" id="inline-formula-262"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-277.gif"/></noscript></span></span>was greatest when <span class="inline-formula" id="inline-formula-263"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-278.gif"/></noscript></span></span> = 0.2, but there was still an advantage to fitting <span class="inline-formula" id="inline-formula-264"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-279.gif"/></noscript></span></span>when <span class="inline-formula" id="inline-formula-265"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-280.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-280.gif"/></noscript></span></span>= 0. That advantage is likely to erode as the number of QTL increases. Changing the mean QTL effect had no impact on bias, except for a slight influence on PBLUP.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486069/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486069/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Accuracy and bias of genomic prediction (<span class="inline-formula" id="inline-formula-266"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/T5/embed/mml-math-281.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/T5/embed/mml-math-281.gif"/></noscript></span></span>) for alternative QTL distributions and analyses</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Effect of more QTL and markers (200 SNP <em>vs.</em> 2000 SNP)</h3><p id="p-47">We had hypothesized that the improvement of accuracy from adding an extra covariate for <span class="inline-formula" id="inline-formula-294"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-309.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-309.gif"/></noscript></span></span> would reduce as the number of QTL increases, because <span class="inline-formula" id="inline-formula-295"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-310.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-310.gif"/></noscript></span></span>is likely to be closer to zero for a trait that is more polygenic. <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> shows that PBLUP was largely unaffected by changes to genetic architecture, but the accuracy of genomic prediction declined slightly as the number of QTL increased. This reflects the fact that the precision of estimating QTL effects is greater when the effects are large, and polygenic traits with more QTL must have on average smaller effects when compared at the same genetic variance. The benefit of fitting <span class="inline-formula" id="inline-formula-296"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-311.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-311.gif"/></noscript></span></span>in the model was virtually eliminated when the number of substitution effects to estimate increased from 150 to 1500. In contrast to the results for accuracy, there was no impact of QTL number on bias. Centering had no impact on accuracy or bias.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486103/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486103/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Accuracy and bias of genomic prediction (<span class="inline-formula" id="inline-formula-297"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/T6/embed/mml-math-312.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/T6/embed/mml-math-312.gif"/></noscript></span></span>) for different numbers of QTL and alternative analyses</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Centering using the entire matrix of genotype covariates or only the observed genotype covariates</h3><p id="p-52"><a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a> shows the accuracies and regression coefficients of TBV on (G)EBV for the genotype panel with 150 markers, <span class="inline-formula" id="inline-formula-318"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-333.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-333.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-319"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-334.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-334.gif"/></noscript></span></span> = 0. The analyses were performed after centering: the entire matrix of imputed and observed genotype covariates (<span class="inline-formula" id="inline-formula-320"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-335.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-335.gif"/></noscript></span></span>); only observed genotype covariates (<span class="inline-formula" id="inline-formula-321"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-336.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-336.gif"/></noscript></span></span>), which is the type of centering done in single-step GBLUP; or not centering the covariates (<span class="inline-formula" id="inline-formula-322"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-337.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-337.gif"/></noscript></span></span>). The accuracy of the genomic analysis with covariates centered as <span class="inline-formula" id="inline-formula-323"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-338.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-338.gif"/></noscript></span></span> but without <strong>J</strong> (model C*) was ∼17% lower than the other genomic analyses. However, when <strong>J</strong> was included in the model with covariates centered as <span class="inline-formula" id="inline-formula-324"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-339.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-339.gif"/></noscript></span></span> the accuracy of prediction was markedly improved.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485939/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485939/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Accuracy and bias of genomic prediction (<span class="inline-formula" id="inline-formula-325"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/T7/embed/mml-math-340.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/T7/embed/mml-math-340.gif"/></noscript></span></span>) when centering for all genotypes or observed genotypes</span><div class="sb-div caption-clear"></div></div></div><p id="p-56">As explained previously, <span class="inline-formula" id="inline-formula-340"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-355.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-355.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-341"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-356.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-356.gif"/></noscript></span></span> is the expected value of the covariates in the founders, will tend to zero for the marker panel that does not include QTL, even with <span class="inline-formula" id="inline-formula-342"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-357.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-357.gif"/></noscript></span></span> However, even if <span class="inline-formula" id="inline-formula-343"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-358.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-358.gif"/></noscript></span></span> in a population undergoing selection when selected individuals are genotyped, <span class="inline-formula" id="inline-formula-344"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-359.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-359.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-345"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-360.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-360.gif"/></noscript></span></span> is the expected value of the observed genotype covariates. In this study, selection was used to increase the mean of the trait. Thus, <span class="inline-formula" id="inline-formula-346"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-361.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-361.gif"/></noscript></span></span> is expected to be negative because most of the genotyped individuals were from G5, whereas <span class="inline-formula" id="inline-formula-347"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-362.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-362.gif"/></noscript></span></span> is expected to be zero. The negative estimate of <span class="inline-formula" id="inline-formula-348"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-363.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-363.gif"/></noscript></span></span> from 10 replicates of the JC* analysis, −2.69, confirms that <span class="inline-formula" id="inline-formula-349"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-364.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-364.gif"/></noscript></span></span> On the other hand, the mean of <span class="inline-formula" id="inline-formula-350"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-365.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-365.gif"/></noscript></span></span> from 10 replicates of the JC analysis was −0.75. This explains why fitting <strong>J</strong> in the model improved the accuracy of genomic prediction when covariates were centered as in single-step GBLUP.</p><p id="p-57"><a id="xref-ref-5-10" class="xref-bibr" href="#ref-5">Fernando <em>et al.</em> (2014)</a> found that centering using <span class="inline-formula" id="inline-formula-351"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-366.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-366.gif"/></noscript></span></span> improved the accuracy without <strong>J</strong> in the model when the population was not under selection and the genotyped individuals were unselected. In that study, mating was random with no selection, so the allele frequency means of the genotyped individuals were a reasonable approximation of the allele frequency means in the founder population. By contrast, our simulation here shows that centering using <span class="inline-formula" id="inline-formula-352"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-367.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-367.gif"/></noscript></span></span> can reduce the accuracy when the population is under selection, unless <strong>J</strong> is fitted in the model.</p><p id="p-58">In single-step GBLUP, the observed genotypes are commonly centered by subtracting their mean and used to construct a genomic relationship matrix, for example, by using the first method proposed by <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">VanRaden (2008)</a>. Using that genomic relationship matrix in the single-step GBLUP formula in <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> (2010)</a> does not account for <strong>J</strong>. This was recognized by <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Vitezica <em>et al.</em> (2011)</a>, who proposed a modification for populations under selection that involved adding a constant to all elements of the genomic relationship matrix that they derived by equating the sum of the elements of the genomic relationship matrix to the sum of the elements of the numerator relationship matrix. In the appendix of that paper they showed that this modification is equivalent to fitting a covariate <strong>Q</strong> = −<strong>J</strong> and treating −<span class="inline-formula" id="inline-formula-353"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-368.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-368.gif"/></noscript></span></span> as a random effect. In addition to this modification, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Christensen <em>et al.</em> (2012)</a> proposed a multiplicative scaling to the genomic relationship matrix such that its diagonals have the same mean as the diagonals of the numerator relationship matrix. <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Vitezica <em>et al.</em> (2011)</a> claimed that −<span class="inline-formula" id="inline-formula-354"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-369.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-369.gif"/></noscript></span></span> represents the mean breeding value of selected individuals, and we have confirmed here that this is true provided the observed genotype covariates are centered by their mean.</p><p id="p-59">Most populations are under natural or artificial selection. In many cases, genotypes are only available on selected individuals. In single-step genomic analysis that combine genotyped and nongenotyped individuals in a joint analysis, the mean of observed genotypes are available for centering. If the observed genotypes include QTL, the accuracy of genomic prediction can be severely compromised, unless the <strong>J</strong> covariate is fitted in the model. If the observed genotypes are only markers, the accuracy of genomic prediction may not necessarily be improved by fitting <strong>J</strong> in the model, but it doesn’t do any harm. However, if centering is applied only to the observed genotypes, which is the type of centering used in single-step GBLUP, accuracy could be severely compromised, unless the <strong>J</strong> covariate is fitted in the model or an equivalent approach is adopted.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-63">The authors are grateful to Bruce L. Golden and Hao Cheng for assistance in the implementation of SSBR models, to Jack C. M. Dekkers for his constructive comments on design of the simulation, and to the very helpful anonymous reviewers. R.L.F. acknowledges useful discussions with Andres Legarra and Zulma Vitezica. This work was supported by the US Department of Agriculture’s Agriculture and Food Research Initiative National Institute of Food and Agriculture competitive grant 2015-67015-22947.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><p id="p-60">Here we show that inference about <span class="inline-formula" id="inline-formula-355"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-370.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-370.gif"/></noscript></span></span> does not depend on how the genotypes are coded. The marker effects model can be described by the following general model:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-371.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-371.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <strong>y</strong> is a vector of observed phenotypes, <strong>1</strong> is a vector of 1s, <em>μ</em> is a general mean, <strong>M</strong> is a matrix of marker covariates, coded 0, 1, 2, which might represent the number of copies of the A allele, <span class="inline-formula" id="inline-formula-356"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-372.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-372.gif"/></noscript></span></span> is a vector of random substitution effects of markers, and <strong>e</strong> is a vector of residuals. Henderson’s mixed model equations (MME) that correspond to Equation 8 are:<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-373.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-373.gif"/></noscript></span> </span>where <span class="inline-formula" id="inline-formula-357"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-374.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-374.gif"/></noscript></span></span> is the best linear unbiased estimate of the mean, and <span class="inline-formula" id="inline-formula-358"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-375.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-375.gif"/></noscript></span></span> is the best linear unbiased predictor of the vector of random substitution effects of all markers. Now we can eliminate <span class="inline-formula" id="inline-formula-359"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-376.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-376.gif"/></noscript></span></span> from the equations for <span class="inline-formula" id="inline-formula-360"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-377.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-377.gif"/></noscript></span></span> by subtracting from those equations the equation for <span class="inline-formula" id="inline-formula-361"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-378.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-378.gif"/></noscript></span></span> premultiplied by <span class="inline-formula" id="inline-formula-362"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-379.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-379.gif"/></noscript></span></span> Then, the MME are transformed to<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-380.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-380.gif"/></noscript></span> </span>and substituting <span class="inline-formula" id="inline-formula-363"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-381.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-381.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-364"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-382.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-382.gif"/></noscript></span></span> and its transpose, the transformed MME become<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-383.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-383.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where <span class="inline-formula" id="inline-formula-365"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-384.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-384.gif"/></noscript></span></span> is the row vector of column means of <span class="inline-formula" id="inline-formula-366"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-385.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-385.gif"/></noscript></span></span> as in <span class="inline-formula" id="inline-formula-367"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-386.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-386.gif"/></noscript></span></span></p><p id="p-61">Now, consider the coding obtained by centering the marker genotypes as <span class="inline-formula" id="inline-formula-368"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-387.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-387.gif"/></noscript></span></span> Then the model can be written as<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-388.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-388.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>where <span class="inline-formula" id="inline-formula-369"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-389.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-389.gif"/></noscript></span></span> The MME that correspond to Equation 10 are<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-390.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-390.gif"/></noscript></span> </span>but <span class="inline-formula" id="inline-formula-370"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-391.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-391.gif"/></noscript></span></span> and, similarly, its transpose is <span class="inline-formula" id="inline-formula-371"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-392.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-392.gif"/></noscript></span></span> Then the MME become<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-393.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-393.gif"/></noscript></span></span>Expanding <span class="inline-formula" id="inline-formula-372"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-394.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-394.gif"/></noscript></span></span> gives <span class="inline-formula" id="inline-formula-373"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-395.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-395.gif"/></noscript></span></span> but because <span class="inline-formula" id="inline-formula-374"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-396.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-396.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-375"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-397.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-397.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-376"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-398.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-398.gif"/></noscript></span></span> as previously shown, the second term, <span class="inline-formula" id="inline-formula-377"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-399.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-399.gif"/></noscript></span></span> which is equal to the last term in the expansion. Thus, the MME become<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-400.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-400.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>The equations for <span class="inline-formula" id="inline-formula-378"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-401.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-401.gif"/></noscript></span></span> in (9) and (11) are identical, and this proves that centering with <span class="inline-formula" id="inline-formula-379"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-402.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-402.gif"/></noscript></span></span> doesn’t affect inference about <span class="inline-formula" id="inline-formula-380"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-403.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-403.gif"/></noscript></span></span></p><p id="p-62">Now suppose an arbitrary vector <span class="inline-formula" id="inline-formula-381"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-404.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-404.gif"/></noscript></span></span> is used to transform the genotypes as <span class="inline-formula" id="inline-formula-382"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-405.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-405.gif"/></noscript></span></span> Then the model becomes<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-406.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-406.gif"/></noscript></span> </span>Adding and subtracting <span class="inline-formula" id="inline-formula-383"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-407.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-407.gif"/></noscript></span></span> the above equation can be written a:<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-408.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-408.gif"/></noscript></span> </span>which is identical to Equation 10, proving that inference about <span class="inline-formula" id="inline-formula-384"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-409.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2685/embed/mml-math-409.gif"/></noscript></span></span> does not depend on how the genotypes are coded.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-22"><p id="p-64"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-12-14"><span class="received-label">Received </span>December 14, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-06-09"><span class="accepted-label">Accepted </span>June 9, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Hsu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.1" data-doi="10.3168/jds.2009-2730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>Hot topic:</em> a unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DAguilar%26rft.auinit1%253DI.%26rft.volume%253D93%26rft.issue%253D2%26rft.spage%253D743%26rft.epage%253D752%26rft.atitle%253DHot%2Btopic%253A%2Ba%2Bunified%2Bapproach%2Bto%2Butilize%2Bphenotypic%252C%2Bfull%2Bpedigree%252C%2Band%2Bgenomic%2Binformation%2Bfor%2Bgenetic%2Bevaluation%2Bof%2BHolstein%2Bfinal%2Bscore.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2730%26rft_id%253Dinfo%253Apmid%252F20105546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105546&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274102500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A fast and efficient Gibbs sampler for BayesB in whole-genome analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D80%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.3" data-doi="10.1186/1297-9686-42-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic prediction when some animals are not genotyped.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DChristensen%26rft.auinit1%253DO.%2BF.%26rft.volume%253D42%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DGenomic%2Bprediction%2Bwhen%2Bsome%2Banimals%2Bare%2Bnot%2Bgenotyped.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-2%26rft_id%253Dinfo%253Apmid%252F20105297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105297&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.4" data-doi="10.1017/S1751731112000742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Single-step methods for genomic evaluation in pigs.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1565</span>–<span class="cit-lpage">1571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnimal%26rft.aulast%253DChristensen%26rft.auinit1%253DO.%2BF.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253D1565%26rft.epage%253D1571%26rft.atitle%253DSingle-step%2Bmethods%2Bfor%2Bgenomic%2Bevaluation%2Bin%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1751731112000742%26rft_id%253Dinfo%253Apmid%252F22717310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1751731112000742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22717310&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308833700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.5" data-doi="10.1186/1297-9686-46-50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A class of Bayesian methods to combine large numbers of genotyped and non-genotyped animals for whole-genome analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D46%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-46-50%26rft_id%253Dinfo%253Apmid%252F25253441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-46-50&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25253441&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.6" data-doi="10.2527/jas1986.631217x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Bayesian methods in animal breeding.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D63%26rft.issue%253D1%26rft.spage%253D217%26rft.epage%253D244%26rft.atitle%253DBayesian%2BMethods%2Bin%2BAnimal%2BBreeding%2BTheory%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1986.631217x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1986.631217x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986D043100027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.7" data-doi="10.1051/gse:19830107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Selection on selected records.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D15%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19830107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19830107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983RN18100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.8.2685.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> Sire evaluation and genetic trends, pp. <span class="cit-fpage">10</span>–<span class="cit-lpage">41</span> in <em>Anim. Breed. Genet. Symp. in Honor of Dr. J. L. Lush</em>. <span class="cit-publ-name">Amer. Soc. Anim. Sci. and Amer. Dairy Sci. Assoc.</span>, Champaign, IL.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2685.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source"><em>Applications of Linear Models in Animal Breeding</em></span>. <span class="cit-publ-name">University of Guelph</span>, <span class="cit-publ-loc">Guelph, ON, Canada</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.10" data-doi="10.1051/gse:19890401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Im</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Likelihood inferences in animal breeding under selection: a missing-data theory view point.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D21%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19890401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19890401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989CK31500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.11" data-doi="10.3168/jds.2009-2061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A relationship matrix including full pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">4656</span>–<span class="cit-lpage">4663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4656%26rft.epage%253D4663%26rft.atitle%253DA%2Brelationship%2Bmatrix%2Bincluding%2Bfull%2Bpedigree%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2061%26rft_id%253Dinfo%253Apmid%252F19700729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700729&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.12" data-doi="10.1017/S0016672300004845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Inferring the trajectory of genetic variance in the course of artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSorensen%26rft.auinit1%253DD.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D83%26rft.epage%253D94%26rft.atitle%253DInferring%2Bthe%2Btrajectory%2Bof%2Bgenetic%2Bvariance%2Bin%2Bthe%2Bcourse%2Bof%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300004845%26rft_id%253Dinfo%253Apmid%252F11279833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300004845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11279833&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.13" data-doi="10.1186/1297-9686-43-25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strandén</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Allele coding in genomic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D43%26rft.spage%253D25%26rft.epage%253D25%26rft.atitle%253DAllele%2Bcoding%2Bin%2Bgenomic%2Bevaluation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-25%26rft_id%253Dinfo%253Apmid%252F21703021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21703021&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.14" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2685.15" data-doi="10.1017/S001667231100022X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bias in genomic predictions for populations under selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DVitezica%26rft.auinit1%253DZ.%2BG.%26rft.volume%253D93%26rft.issue%253D5%26rft.spage%253D357%26rft.epage%253D366%26rft.atitle%253DBias%2Bin%2Bgenomic%2Bpredictions%2Bfor%2Bpopulations%2Bunder%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS001667231100022X%26rft_id%253Dinfo%253Apmid%252F21767459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S001667231100022X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21767459&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/8/2685.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2677" title="Evidence for Adaptive Introgression of Disease Resistance Genes Among Closely Related Arabidopsis Species" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2695" title="rRNA Pseudogenes in Filamentous Ascomycetes as Revealed by Genome Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/8"><span>Volume 7 Issue 8, August 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/8.cover-source.jpg?itok=apH_YDuu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2685.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3lMnQbjnFTj8epJ5B0PQPQP159zq-s3GCk_7qn2hTag" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/8/2685" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/8/2685" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/8/2685" class="active" data-icon-position="" data-hide-link-title="0">The Accuracy and Bias of Single-Step Genomic Prediction for Populations Under Selection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485831" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ONF7oSq86Gg0mXS5hWPy8vcEAalOnyhIQCqIpS4tFcs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2685.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/8/2685" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-q3h0abYGxIQj5S5CUS41GD--a2UNItvEBtiUXRLAKKA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/8/2685" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2685.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485831" id="node485831--2" data-pisa="ggg;7/8/2685" data-pisa-master="ggg;g3.117.043596" data-apath="/ggg/7/8/2685.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Accuracy and Bias of Single-Step Genomic Prediction for Populations Under Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Wan-Ling Hsu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8640-5372" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dorian J. Garrick</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5821-099X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rohan L. Fernando</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2685-2694;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043596 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485831--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Wan-Ling Hsu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wan-Ling%2BHsu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hsu%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWan-Ling%2BHsu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dorian J. Garrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Iowa State University, Ames, Iowa 50011</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Veterinary, Animal and Biomedical Sciences, Massey University, Palmerston North 4442, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorian%2BJ.%2BGarrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garrick%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADorian%2BJ.%2BGarrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8640-5372" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dorian J. Garrick</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rohan L. Fernando </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rohan%2BL.%2BFernando%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernando%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARohan%2BL.%2BFernando%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5821-099X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rohan L. Fernando</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rohan@iastate.edu" class="" data-icon-position="" data-hide-link-title="0">rohan@iastate.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/8/2685" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/8/2685&amp;title=The%20Accuracy%20and%20Bias%20of%20Single-Step%20Genomic%20Prediction%20for%20Populations%20Under%20Selection" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/8/2685&amp;title=The%20Accuracy%20and%20Bias%20of%20Single-Step%20Genomic%20Prediction%20for%20Populations%20Under%20Selection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/8/2685&amp;title=The%20Accuracy%20and%20Bias%20of%20Single-Step%20Genomic%20Prediction%20for%20Populations%20Under%20Selection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/8/2685&amp;text=The%20Accuracy%20and%20Bias%20of%20Single-Step%20Genomic%20Prediction%20for%20Populations%20Under%20Selection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/8/2685&amp;title=The%20Accuracy%20and%20Bias%20of%20Single-Step%20Genomic%20Prediction%20for%20Populations%20Under%20Selection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/8/2685&amp;t=The%20Accuracy%20and%20Bias%20of%20Single-Step%20Genomic%20Prediction%20for%20Populations%20Under%20Selection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/8/2685&amp;title=The%20Accuracy%20and%20Bias%20of%20Single-Step%20Genomic%20Prediction%20for%20Populations%20Under%20Selection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/8/2685&amp;title=The%20Accuracy%20and%20Bias%20of%20Single-Step%20Genomic%20Prediction%20for%20Populations%20Under%20Selection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485831/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485831" id="node485831--3" data-pisa="ggg;7/8/2685" data-pisa-master="ggg;g3.117.043596" data-apath="/ggg/7/8/2685.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Accuracy and Bias of Single-Step Genomic Prediction for Populations Under Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Wan-Ling Hsu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8640-5372" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dorian J. Garrick</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5821-099X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rohan L. Fernando</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2685-2694;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043596 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485831--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Wan-Ling Hsu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wan-Ling%2BHsu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hsu%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWan-Ling%2BHsu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dorian J. Garrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Iowa State University, Ames, Iowa 50011</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Veterinary, Animal and Biomedical Sciences, Massey University, Palmerston North 4442, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorian%2BJ.%2BGarrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garrick%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADorian%2BJ.%2BGarrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8640-5372" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dorian J. Garrick</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rohan L. Fernando </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rohan%2BL.%2BFernando%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernando%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARohan%2BL.%2BFernando%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5821-099X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rohan L. Fernando</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rohan@iastate.edu" class="" data-icon-position="" data-hide-link-title="0">rohan@iastate.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2685&amp;rft.epage=2694&amp;rft.atitle=The%20Accuracy%20and%20Bias%20of%20Single-Step%20Genomic%20Prediction%20for%20Populations%20Under%20Selection&amp;rft.volume=7&amp;rft.issue=8&amp;rft.date=2017-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hsu%2C+Wan-Ling&amp;rft.au=Garrick%2C+Dorian+J.&amp;rft.au=Fernando%2C+Rohan+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485831/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485831/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485831/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485831/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485831/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485831/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485831/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485831/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485831/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485831/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485831/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485831/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/8/2685" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/8/2685.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/8/2685.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/7/8/2685.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__CK8qNg7zHnCPfipcSU0qjCyIiw2pe-WKGx2V0TgizGw__wprDSLLH9CuacxJkxX2DuWt0EaauZA-f_LGjovfY3aw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
