<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/8/2413" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/8/2413.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/8/2413.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/8/2413.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/8/2413.atom" />
<meta name="HW.pisa" content="ggg;7/8/2413" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparative Genomics of Two Sequential Candida glabrata Clinical Isolates" />
<meta name="DC.Identifier" content="10.1534/g3.117.042887" />
<meta name="DC.Date" content="2017-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Vale-Silva et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Candida glabrata is an important fungal pathogen which develops rapid antifungal resistance in treated patients. It is known that azole treatments lead to antifungal resistance in this fungal species and that multidrug efflux transporters are involved in this process. Specific mutations in the transcriptional regulator PDR1 result in upregulation of the transporters. In addition, we showed that the PDR1 mutations can contribute to enhance virulence in animal models. In this study, we were interested to compare genomes of two specific C. glabrata -related isolates, one of which was azole susceptible (DSY562) while the other was azole resistant (DSY565). DSY565 contained a PDR1 mutation (L280F) and was isolated after a time-lapse of 50 d of azole therapy. We expected that genome comparisons between both isolates could reveal additional mutations reflecting host adaptation or even additional resistance mechanisms. The PacBio technology used here yielded 14 major contigs (sizes 0.18–1.6 Mb) and mitochondrial genomes from both DSY562 and DSY565 isolates that were highly similar to each other. Comparisons of the clinical genomes with the published CBS138 genome indicated important genome rearrangements, but not between the clinical strains. Among the unique features, several retrotransposons were identified in the genomes of the investigated clinical isolates. DSY562 and DSY565 each contained a large set of adhesin-like genes (101 and 107, respectively), which exceed by far the number of reported adhesins (63) in the CBS138 genome. Comparison between DSY562 and DSY565 yielded 17 nonsynonymous SNPs (among which the was the expected PDR1 mutation) as well as small size indels in coding regions (11) but mainly in adhesin-like genes. The genomes contained a DNA mismatch repair allele of MSH2 known to be involved in the so-called hyper-mutator phenotype of this yeast species and the number of accumulated mutations between both clinical isolates is consistent with the presence of a MSH2 defect. In conclusion, this study is the first to compare genomes of C. glabrata sequential clinical isolates using the PacBio technology as an approach. The genomes of these isolates taken in the same patient at two different time points exhibited limited variations, even if submitted to the host pressure." />
<meta name="DC.Contributor" content="Luis Vale-Silva" />
<meta name="DC.Contributor" content="Emmanuel Beaudoing" />
<meta name="DC.Contributor" content="Van Du T. Tran" />
<meta name="DC.Contributor" content="Dominique Sanglard" />
<meta name="article:published_time" content="2017-08-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Comparative Genomics of Two Sequential Candida glabrata Clinical Isolates" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Candida glabrata&lt;/i&gt; is an important fungal pathogen which develops rapid antifungal resistance in treated patients. It is known that azole treatments lead to antifungal resistance in this fungal species and that multidrug efflux transporters are involved in this process. Specific mutations in the transcriptional regulator &lt;i&gt;PDR1&lt;/i&gt; result in upregulation of the transporters. In addition, we showed that the &lt;i&gt;PDR1&lt;/i&gt; mutations can contribute to enhance virulence in animal models. In this study, we were interested to compare genomes of two specific &lt;i&gt;C. glabrata&lt;/i&gt;-related isolates, one of which was azole susceptible (DSY562) while the other was azole resistant (DSY565). DSY565 contained a &lt;i&gt;PDR1&lt;/i&gt; mutation (L280F) and was isolated after a time-lapse of 50 d of azole therapy. We expected that genome comparisons between both isolates could reveal additional mutations reflecting host adaptation or even additional resistance mechanisms. The PacBio technology used here yielded 14 major contigs (sizes 0.18–1.6 Mb) and mitochondrial genomes from both DSY562 and DSY565 isolates that were highly similar to each other. Comparisons of the clinical genomes with the published CBS138 genome indicated important genome rearrangements, but not between the clinical strains. Among the unique features, several retrotransposons were identified in the genomes of the investigated clinical isolates. DSY562 and DSY565 each contained a large set of adhesin-like genes (101 and 107, respectively), which exceed by far the number of reported adhesins (63) in the CBS138 genome. Comparison between DSY562 and DSY565 yielded 17 nonsynonymous SNPs (among which the was the expected &lt;i&gt;PDR1&lt;/i&gt; mutation) as well as small size indels in coding regions (11) but mainly in adhesin-like genes. The genomes contained a DNA mismatch repair allele of &lt;i&gt;MSH2&lt;/i&gt; known to be involved in the so-called hyper-mutator phenotype of this yeast species and the number of accumulated mutations between both clinical isolates is consistent with the presence of a &lt;i&gt;MSH2&lt;/i&gt; defect. In conclusion, this study is the first to compare genomes of &lt;i&gt;C. glabrata&lt;/i&gt; sequential clinical isolates using the PacBio technology as an approach. The genomes of these isolates taken in the same patient at two different time points exhibited limited variations, even if submitted to the host pressure.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/08/01" />
<meta name="citation_mjid" content="ggg;7/8/2413" />
<meta name="citation_id" content="7/8/2413" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/8/2413" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/8/2413.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/8/2413.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/8/2413.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.042887" />
<meta name="citation_pmid" content="28663342" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="2413" />
<meta name="citation_lastpage" content="2426" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Luis Vale-Silva" />
<meta name="citation_author_institution" content="Institute of Microbiology, University of Lausanne, CH-1011, Switzerland" />
<meta name="citation_author_institution" content="Lausanne University Hospital, CH-1011, Switzerland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9658-5335" />
<meta name="citation_author" content="Emmanuel Beaudoing" />
<meta name="citation_author_institution" content="Center for Integrative Genomics, Lausanne Genomic Technologies Facility, CH-1015, Switzerland" />
<meta name="citation_author" content="Van Du T. Tran" />
<meta name="citation_author_institution" content="Vital-IT Group, SIB Swiss Institute of Bioinformatics, CH-1015 Lausanne, Switzerland" />
<meta name="citation_author" content="Dominique Sanglard" />
<meta name="citation_author_institution" content="Institute of Microbiology, University of Lausanne, CH-1011, Switzerland" />
<meta name="citation_author_institution" content="Lausanne University Hospital, CH-1011, Switzerland" />
<meta name="citation_author_email" content="Dominique.Sanglard@chuv.ch" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5244-4178" />
<meta name="citation_reference" content="citation_journal_title=Clin. Infect. Dis.;citation_author=B. D. Alexander;citation_author=M. D. Johnson;citation_author=C. D. Pfeiffer;citation_author=C. Jimenez-Ortigosa;citation_author=J. Catania;citation_title=Increasing Echinocandin resistance in Candida glabrata: clinical failure correlates with presence of FKS mutations and elevated minimum inhibitory concentrations.;citation_volume=56;citation_year=2013;citation_pmid=23487382;citation_doi=10.1093/cid/cit136" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. An;citation_author=J. Lai;citation_author=A. Sajjanhar;citation_author=J. Batra;citation_author=C. Wang;citation_title=J-Circos: an interactive Circos plotter.;citation_volume=31;citation_year=2015;citation_pmid=25540184;citation_doi=10.1093/bioinformatics/btu842" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Infect. Dis.;citation_author=M. C. Arendrup;citation_author=D. S. Perlin;citation_title=Echinocandin resistance: an emerging clinical problem?;citation_pages=484-492;citation_volume=27;citation_year=2014;citation_pmid=25304391;citation_doi=10.1097/QCO.0000000000000111" />
<meta name="citation_reference" content="citation_title=PLoS ONE;citation_author=O. Bader;citation_author=A. Schwarz;citation_author=E. A. Kraneveld;citation_author=M. Tangwattanchuleeporn;citation_author=P. Schmidt;citation_title=Gross karyotypic and phenotypic alterations among different progenies of the Candida glabrata CBS138/ATCC2001 reference strain.;citation_pages=e52218;citation_volume=7;citation_year=2012;citation_pmid=23284942;citation_doi=10.1371/journal.pone.0052218" />
<meta name="citation_reference" content="citation_journal_title=Science Translational Medicine;citation_journal_abbrev=Sci Transl Med;citation_author=G. D. Brown;citation_author=D. W. Denning;citation_author=N. A. R. Gow;citation_author=S. M. Levitz;citation_author=M. G. Netea;citation_title=Hidden Killers: Human Fungal Infections;citation_pages=165rv13-165rv13;citation_volume=4;citation_year=2012;citation_issue=165;citation_pmid=23253612;citation_doi=10.1126/scitranslmed.3004404" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=D. Calabrese;citation_author=J. Bille;citation_author=D. Sanglard;citation_title=A novel multidrug efflux transporter gene of the major facilitator superfamily from Candida albicans (FLU1) conferring resistance to fluconazole;citation_pages=2743-2754;citation_volume=146;citation_year=2000;citation_issue=11;citation_pmid=11065353;citation_doi=10.1099/00221287-146-11-2743" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=Patterns of genomic variation in the opportunistic pathogen Candida glabrata suggest the existence of mating and a secondary association to the human host.;citation_author=L. Carreté;citation_author=E. Ksiezopolska;citation_author=C. Pequeroles;citation_author=E. Gómez-Molero;citation_author=E. Saus;citation_pages=105791;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. Chaudhuri;citation_author=F. A. Ansari;citation_author=M. V. Raghunandanan;citation_author=S. Ramachandran;citation_title=FungalRV: adhesin prediction and immunoinformatics portal for human fungal pathogens.;citation_pages=192-192;citation_volume=12;citation_year=2011;citation_pmid=21496229;citation_doi=10.1186/1471-2164-12-192" />
<meta name="citation_reference" content="citation_journal_title=Clin. Infect. Dis.;citation_author=A. A. Cleveland;citation_author=M. M. Farley;citation_author=L. H. Harrison;citation_author=B. Stein;citation_author=R. Hollick;citation_title=Changes in incidence and antifungal drug resistance in candidemia: results from population-based laboratory surveillance in Atlanta and Baltimore, 2008–2011.;citation_volume=55;citation_year=2012;citation_pmid=22893576;citation_doi=10.1093/cid/cis697" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. C. E. Darling;citation_author=B. Mau;citation_author=F. R. Blattner;citation_author=N. T. Perna;citation_title=Mauve: Multiple Alignment of Conserved Genomic Sequence With Rearrangements;citation_pages=1394-1403;citation_volume=14;citation_year=2004;citation_issue=7;citation_pmid=15231754;citation_doi=10.1101/gr.2289704" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=P. W. J. de Groot;citation_author=E. A. Kraneveld;citation_author=Q. Y. Yin;citation_author=H. L. Dekker;citation_author=U. Gross;citation_title=The cell wall of the human pathogen Candida glabrata: differential incorporation of novel adhesin-like wall proteins.;citation_volume=7;citation_year=2008;citation_pmid=18806209;citation_doi=10.1128/EC.00284-08" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=P. W. J. de Groot;citation_author=O. Bader;citation_author=A. D. de Boer;citation_author=M. Weig;citation_author=N. Chauhan;citation_title=Adhesins in human fungal pathogens: glue with plenty of stick.;citation_volume=12;citation_year=2013;citation_pmid=23397570;citation_doi=10.1128/EC.00364-12" />
<meta name="citation_reference" content="citation_journal_title=Front. Microbiol.;citation_author=S. Dellière;citation_author=K. Healey;citation_author=M. Gits-Muselli;citation_author=B. Carrara;citation_author=A. Barbaro;citation_title=Fluconazole and Echinocandin resistance of Candida glabrata correlates better with antifungal drug exposure rather than with MSH2 mutator genotype in a French Cohort of patients harboring low rates of resistance.;citation_pages=2038;citation_volume=7;citation_year=2016;citation_doi=10.3389/fmicb.2016.02038" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=R. Diderrich;citation_author=M. Kock;citation_author=M. Maestre-Reyna;citation_author=P. Keller;citation_author=H. Steuber;citation_title=Structural Hot spots determine functional diversity of the Candida glabrata Epithelial adhesin family.;citation_volume=290;citation_year=2015;citation_pmid=26105055;citation_doi=10.1074/jbc.M115.655654" />
<meta name="citation_reference" content="citation_journal_title=Journal of Clinical Microbiology;citation_journal_abbrev=J. Clin. Microbiol.;citation_author=A. R. Dodgson;citation_author=C. Pujol;citation_author=D. W. Denning;citation_author=D. R. Soll;citation_author=A. J. Fox;citation_title=Multilocus Sequence Typing of Candida glabrata Reveals Geographically Enriched Clades;citation_pages=5709-5717;citation_volume=41;citation_year=2003;citation_issue=12;citation_pmid=14662965;citation_doi=10.1128/JCM.41.12.5709-5717.2003" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=A. R. Dodgson;citation_author=C. Pujol;citation_author=M. A. Pfaller;citation_author=D. W. Denning;citation_author=D. R. Soll;citation_title=Evidence for recombination in Candida glabrata.;citation_pages=233-243;citation_volume=42;citation_year=2005;citation_issue=3;citation_pmid=15707844;citation_doi=10.1016/j.fgb.2004.11.010" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. Dujon;citation_author=D. Sherman;citation_author=G. Fischer;citation_author=P. Durrens;citation_author=S. Casaregola;citation_title=Genome evolution in yeasts.;citation_pages=35-44;citation_volume=430;citation_year=2004;citation_issue=6995;citation_pmid=15229592;citation_doi=10.1038/nature02579" />
<meta name="citation_reference" content="citation_journal_title=Emerg. Infect. Dis.;citation_author=D. Farmakiotis;citation_author=J. J. Tarrand;citation_author=D. P. Kontoyiannis;citation_title=Drug-resistant Candida glabrata infection in cancer patients.;citation_pages=1833-1840;citation_volume=20;citation_year=2014;citation_pmid=25340258;citation_doi=10.3201/eid2011.140685" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=S. Ferrari;citation_author=F. Ischer;citation_author=D. Calabrese;citation_author=B. Posteraro;citation_author=M. Sanguinetti;citation_title=Gain of function mutations in CgPDR1 of Candida glabrata not only mediate antifungal resistance but also enhance virulence.;citation_pages=e1000268-e1000268;citation_volume=5;citation_year=2009;citation_issue=1;citation_pmid=19148266;citation_doi=10.1371/journal.ppat.1000268" />
<meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.;citation_author=S. Ferrari;citation_author=M. Sanguinetti;citation_author=F. de Bernardis;citation_author=R. Torelli;citation_author=B. Posteraro;citation_title=Loss of mitochondrial functions associated with azole resistance in Candida glabrata results in enhanced virulence in mice.;citation_volume=55;citation_year=2011;citation_pmid=21321146;citation_doi=10.1128/AAC.01271-10" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Forche;citation_author=P. T. Magee;citation_author=A. Selmecki;citation_author=J. Berman;citation_author=G. May;citation_title=Evolution in Candida albicans populations during a single passage through a mouse host.;citation_volume=182;citation_year=2009;citation_pmid=19414562;citation_doi=10.1534/genetics.109.103325" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=T. Gabaldón;citation_author=T. Martin;citation_author=M. Marcet-Houben;citation_author=P. Durrens;citation_author=M. Bolotin-Fukuhara;citation_title=Comparative genomics of emerging pathogens in the Candida glabrata clade.;citation_pages=623;citation_volume=14;citation_year=2013;citation_pmid=24034898;citation_doi=10.1186/1471-2164-14-623" />
<meta name="citation_reference" content="citation_journal_title=Genome Announcements;citation_journal_abbrev=Genome Announc;citation_author=O. E. Håvelsrud;citation_author=P. Gaustad;citation_title=Draft Genome Sequences of Candida glabrata Isolates 1A, 1B, 2A, 2B, 3A, and 3B;citation_pages=e00328-16-e00328-16;citation_volume=5;citation_year=2017;citation_issue=10;citation_pmid=28280017;citation_doi=10.1128/genomeA.00328-16" />
<meta name="citation_reference" content="citation_journal_title=Front. Microbiol.;citation_author=K. R. Healey;citation_author=C. Jimenez-Ortigosa;citation_author=E. Shor;citation_author=D. S. Perlin;citation_title=Genetic drivers of multidrug resistance in Candida glabrata.;citation_pages=1724-1729;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=K. R. Healey;citation_author=Y. Zhao;citation_author=W. B. Perez;citation_author=S. R. Lockhart;citation_author=J. D. Sobel;citation_title=Prevalent mutator genotype identified in fungal pathogen Candida glabrata promotes multi-drug resistance.;citation_pages=1-10;citation_volume=7;citation_year=2016;citation_pmid=27009409;citation_doi=10.1038/ncomms11388" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=G. I. Lang;citation_author=L. Parsons;citation_author=A. E. Gammie;citation_title=Mutation rates, spectra, and genome-wide distribution of spontaneous mutations in mismatch repair deficient yeast.;citation_pages=1453-1465;citation_volume=3;citation_year=2013;citation_doi=10.1534/g3.113.006429" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Linde;citation_author=S. Duggan;citation_author=M. Weber;citation_author=F. Horn;citation_author=P. Sieber;citation_title=Defining the transcriptomic landscape of Candida glabrata by RNA-Seq.;citation_volume=43;citation_year=2015;citation_pmid=25586221;citation_doi=10.1093/nar/gku1357" />
<meta name="citation_reference" content="citation_journal_abbrev=Cell Rep;citation_author=X. Ma;citation_author=M. V. Rogacheva;citation_author=K. T. Nishant;citation_author=S. Zanders;citation_author=C. D. Bustamante;citation_title=Mutation hot spots in yeast caused by long-range clustering of homopolymeric sequences.;citation_pages=36-42;citation_volume=1;citation_year=2012;citation_issue=1;citation_pmid=22832106;citation_doi=10.1016/j.celrep.2011.10.003" />
<meta name="citation_reference" content="citation_journal_title=Clin. Infect. Dis.;citation_author=O. Marchetti;citation_author=J. Bille;citation_author=U. Fluckiger;citation_author=P. Eggimann;citation_author=C. Ruef;citation_title=Epidemiology of candidemia in Swiss tertiary care hospitals: secular trends, 1991–2000.;citation_volume=38;citation_year=2004;citation_pmid=14727199;citation_doi=10.1086/380637" />
<meta name="citation_reference" content="citation_journal_title=Clin. Infect. Dis.;citation_author=D. S. Perlin;citation_title=Echinocandin resistance in Candida.;citation_volume=61;citation_year=2015;citation_pmid=26567278;citation_doi=10.1093/cid/civ791" />
<meta name="citation_reference" content="citation_journal_title=Curr Clin Micro Rpt;citation_author=D. S. Perlin;citation_author=E. Shor;citation_author=Y. Zhao;citation_title=Update on antifungal drug resistance.;citation_pages=84-95;citation_volume=2;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Journal of Clinical Microbiology;citation_journal_abbrev=J. Clin. Microbiol.;citation_author=M. A. Pfaller;citation_author=S. A. Messer;citation_author=L. Boyken;citation_author=S. Tendolkar;citation_author=R. J. Hollis;citation_title=Variation in Susceptibility of Bloodstream Isolates of Candida glabrata to Fluconazole According to Patient Age and Geographic Location;citation_pages=2176-2179;citation_volume=41;citation_year=2003;citation_issue=5;citation_pmid=12734273;citation_doi=10.1128/JCM.41.5.2176-2179.2003" />
<meta name="citation_reference" content="citation_journal_title=J. Clin. Microbiol.;citation_author=M. A. Pfaller;citation_author=S. A. Messer;citation_author=R. J. Hollis;citation_author=L. Boyken;citation_author=S. Tendolkar;citation_title=Variation in susceptibility of bloodstream isolates of Candida glabrata to fluconazole according to patient age and geographic location in the United States in 2001 to 2007.;citation_volume=47;citation_year=2009;citation_pmid=19656983;citation_doi=10.1128/JCM.00946-09" />
<meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.;citation_author=C. D. Pham;citation_author=N. Iqbal;citation_author=C. B. Bolden;citation_author=R. J. Kuykendall;citation_author=L. H. Harrison;citation_title=Role of FKS mutations in Candida glabrata: MIC values, echinocandin resistance, and multidrug resistance.;citation_volume=58;citation_year=2014;citation_pmid=24890592;citation_doi=10.1128/AAC.03255-14" />
<meta name="citation_reference" content="citation_journal_title=Clinical microbiology and infection :  the official publication of the European Society of Clinical Microbiology and Infectious Diseases;citation_journal_abbrev=Clin Microbiol Infect;citation_author=M. Richardson;citation_author=C. Lass-Flörl;citation_title=Limiting the attributable mortality of nosocomial infection and multidrug resistance in intensive care units.;citation_pages=5-13;citation_volume=14;citation_year=2008;citation_issue=1;citation_pmid=17944969;citation_doi=10.1111/j.1469-0691.2008.01978.x" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=A. Roetzer;citation_author=T. Gabaldón;citation_author=C. Schüller;citation_title=From Saccharomyces cerevisiae to Candida glabrata in a few easy steps: important adaptations for an opportunistic pathogen.;citation_volume=314;citation_year=2011;citation_pmid=20846362;citation_doi=10.1111/j.1574-6968.2010.02102.x" />
<meta name="citation_reference" content="citation_journal_title=Clin. Microbiol. Infect.;citation_author=M. Ruhnke;citation_title=ANTIFUNGAL STEWARDSHIP IN INVASIVE CANDIDA INFECTIONS;citation_pages=11-18;citation_volume=20;citation_year=2014;citation_issue=Suppl 6;citation_pmid=24661820;citation_doi=10.1111/1469-0691.12622" />
<meta name="citation_reference" content="citation_journal_title=Front Med (Lausanne);citation_author=D. Sanglard;citation_title=Emerging threats in antifungal-resistant fungal pathogens.;citation_pages=11;citation_volume=3;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Antimicrobial Agents and Chemotherapy;citation_journal_abbrev=Antimicrob. Agents Chemother.;citation_author=D. Sanglard;citation_author=F. Ischer;citation_author=D. Calabrese;citation_author=P. A. Majcherczyk;citation_author=J. Bille;citation_title=The ATP Binding Cassette Transporter Gene CgCDR1 from Candida glabrata Is Involved in the Resistance of Clinical Isolates to Azole Antifungal Agents;citation_pages=2753-2765;citation_volume=43;citation_year=1999;citation_issue=11;citation_pmid=10543759" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. R. Scannell;citation_author=A. C. Frank;citation_author=G. C. Conant;citation_author=K. P. Byrne;citation_author=M. Woolfit;citation_title=Independent sorting-out of thousands of duplicated gene pairs in two yeast species descended from a whole-genome duplication.;citation_volume=104;citation_year=2007;citation_pmid=17494770;citation_doi=10.1073/pnas.0608218104" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=S. D. Singh-Babak;citation_author=T. Babak;citation_author=S. Diezmann;citation_author=J. A. Hill;citation_author=J. L. Xie;citation_title=Global analysis of the evolution and mechanism of echinocandin resistance in Candida glabrata.;citation_pages=e1002718-e1002718;citation_volume=8;citation_year=2012;citation_issue=5;citation_pmid=22615574;citation_doi=10.1371/journal.ppat.1002718" />
<meta name="citation_reference" content="citation_title=PLoS Pathog.;citation_author=S. Tati;citation_author=P. Davidow;citation_author=A. McCall;citation_author=E. Hwang-Wong;citation_author=I. G. Rojas;citation_title=Candida glabrata binding to Candida albicans Hyphae enables its development in Oropharyngeal Candidiasis.;citation_pages=e1005522;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.ppat.1005522" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Trapnell;citation_author=L. Pachter;citation_author=S. L. Salzberg;citation_title=TopHat: discovering splice junctions with RNA-Seq.;citation_volume=25;citation_year=2009;citation_pmid=19289445;citation_doi=10.1093/bioinformatics/btp120" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=L. A. Vale-Silva;citation_author=D. Sanglard;citation_title=Tipping the balance both ways: drug resistance and virulence in Candida glabrata.;citation_volume=15;citation_year=2015;citation_pmid=25979690;citation_doi=10.1093/femsyr/fov025" />
<meta name="citation_reference" content="citation_journal_title=Infect. Immun.;citation_author=L. Vale-Silva;citation_author=F. Ischer;citation_author=S. Leibundgut-Landmann;citation_author=D. Sanglard;citation_title=Gain-of-function mutations in PDR1, a regulator of antifungal drug resistance in Candida glabrata, control adherence to host cells.;citation_volume=81;citation_year=2013;citation_pmid=23460523;citation_doi=10.1128/IAI.00074-13" />
<meta name="citation_reference" content="citation_title=mSphere;citation_journal_abbrev=mSphere;citation_author=L. A. Vale-Silva;citation_author=B. Moeckli;citation_author=R. Torelli;citation_author=B. Posteraro;citation_author=M. Sanguinetti;citation_title=Upregulation of the Adhesin Gene EPA1 Mediated by PDR1 in Candida glabrata Leads to Enhanced Host Colonization;citation_pages=e00065-15-e00065-15;citation_volume=1;citation_year=2016;citation_issue=2;citation_pmid=27303714;citation_doi=10.1128/mSphere.00065-15" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=M. Weig;citation_author=L. Jänsch;citation_author=U. Gross;citation_author=C. G. de Koster;citation_author=F. M. Klis;citation_title=Systematic identification in silico of covalently bound cell wall proteins and analysis of protein-polysaccharide linkages of the human pathogen Candida glabrata;citation_pages=3129-3144;citation_volume=150;citation_year=2004;citation_issue=10;citation_pmid=15470094;citation_doi=10.1099/mic.0.27256-0" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=N. Xu;citation_author=C. Ye;citation_author=X. Chen;citation_author=J. Liu;citation_author=L. Liu;citation_title=Genome sequencing of the Pyruvate-producing strain Candida glabrata CCTCC M202019 and genomic comparison with strain CBS138.;citation_pages=34893;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=M. L. Zupancic;citation_author=M. Frieman;citation_author=D. Smith;citation_author=R. A. Alvarez;citation_author=R. D. Cummings;citation_title=Glycan microarray analysis of Candida glabrata adhesin ligand specificity.;citation_pages=547-559;citation_volume=68;citation_year=2008;citation_issue=3;citation_pmid=18394144;citation_doi=10.1111/j.1365-2958.2008.06184.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparative Genomics of Two Sequential Candida glabrata Clinical Isolates" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="twitter:description" content="Candida glabrata is an important fungal pathogen which develops rapid antifungal resistance in treated patients. It is known that azole treatments lead to antifungal resistance in this fungal species and that multidrug efflux transporters are involved in this process. Specific mutations in the transcriptional regulator PDR1 result in upregulation of the transporters. In addition, we showed that the PDR1 mutations can contribute to enhance virulence in animal models. In this study, we were interested to compare genomes of two specific C. glabrata -related isolates, one of which was azole susceptible (DSY562) while the other was azole resistant (DSY565). DSY565 contained a PDR1 mutation (L280F) and was isolated after a time-lapse of 50 d of azole therapy. We expected that genome comparisons between both isolates could reveal additional mutations reflecting host adaptation or even additional resistance mechanisms. The PacBio technology used here yielded 14 major contigs (sizes 0.18–1.6 Mb) and mitochondrial genomes from both DSY562 and DSY565 isolates that were highly similar to each other. Comparisons of the clinical genomes with the published CBS138 genome indicated important genome rearrangements, but not between the clinical strains. Among the unique features, several retrotransposons were identified in the genomes of the investigated clinical isolates. DSY562 and DSY565 each contained a large set of adhesin-like genes (101 and 107, respectively), which exceed by far the number of reported adhesins (63) in the CBS138 genome. Comparison between DSY562 and DSY565 yielded 17 nonsynonymous SNPs (among which the was the expected PDR1 mutation) as well as small size indels in coding regions (11) but mainly in adhesin-like genes. The genomes contained a DNA mismatch repair allele of MSH2 known to be involved in the so-called hyper-mutator phenotype of this yeast species and the number of accumulated mutations between both clinical isolates is consistent with the presence of a MSH2 defect. In conclusion, this study is the first to compare genomes of C. glabrata sequential clinical isolates using the PacBio technology as an approach. The genomes of these isolates taken in the same patient at two different time points exhibited limited variations, even if submitted to the host pressure." />
<meta name="og-title" property="og:title" content="Comparative Genomics of Two Sequential Candida glabrata Clinical Isolates" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/8/2413" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Candida glabrata is an important fungal pathogen which develops rapid antifungal resistance in treated patients. It is known that azole treatments lead to antifungal resistance in this fungal species and that multidrug efflux transporters are involved in this process. Specific mutations in the transcriptional regulator PDR1 result in upregulation of the transporters. In addition, we showed that the PDR1 mutations can contribute to enhance virulence in animal models. In this study, we were interested to compare genomes of two specific C. glabrata -related isolates, one of which was azole susceptible (DSY562) while the other was azole resistant (DSY565). DSY565 contained a PDR1 mutation (L280F) and was isolated after a time-lapse of 50 d of azole therapy. We expected that genome comparisons between both isolates could reveal additional mutations reflecting host adaptation or even additional resistance mechanisms. The PacBio technology used here yielded 14 major contigs (sizes 0.18–1.6 Mb) and mitochondrial genomes from both DSY562 and DSY565 isolates that were highly similar to each other. Comparisons of the clinical genomes with the published CBS138 genome indicated important genome rearrangements, but not between the clinical strains. Among the unique features, several retrotransposons were identified in the genomes of the investigated clinical isolates. DSY562 and DSY565 each contained a large set of adhesin-like genes (101 and 107, respectively), which exceed by far the number of reported adhesins (63) in the CBS138 genome. Comparison between DSY562 and DSY565 yielded 17 nonsynonymous SNPs (among which the was the expected PDR1 mutation) as well as small size indels in coding regions (11) but mainly in adhesin-like genes. The genomes contained a DNA mismatch repair allele of MSH2 known to be involved in the so-called hyper-mutator phenotype of this yeast species and the number of accumulated mutations between both clinical isolates is consistent with the presence of a MSH2 defect. In conclusion, this study is the first to compare genomes of C. glabrata sequential clinical isolates using the PacBio technology as an approach. The genomes of these isolates taken in the same patient at two different time points exhibited limited variations, even if submitted to the host pressure." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
    <title>Comparative Genomics of Two Sequential Candida glabrata Clinical Isolates | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8yT9KJwz5zpl2ECKU7bRVgd6KZdc074yKgjVJ8HgQHA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485847","apath":"\/ggg\/7\/8\/2413.atom","pisa":"ggg;7\/8\/2413","ac":{"\/ggg\/7\/8\/2413.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/8\/2413.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2413"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2413"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/8\/2413":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022s7cuyZD1Av5F31NtkfDLdKPc80mdbrIqggSnETdapeElKv+\\\\\\\/7qG7BbHmsp5WFnRIQuasj6EB6\\\\\\\/KaSFIZZfXT3vScmC1aISV\\\\\\\/mV0YsatpyI15AzmanzHrdZr2TDLi9lpuFvPzgeaxGYJ9OlnNLKV7qc3GK6I8CGR58iHS8zhoAGjoNNkgnyDK2KycTlHHwWrwQvlRaKr2wiVdj2\\\\\\\/eBI4J35xFvMCe9PA6M8sbmXfW86uhoM5tGF0elm\\\\\\\/GPr\\\\\\\/g24ltTRMgT8y0UOxX15B1d91vhQOM7Qvfp8gUKTO7OHJzIB0OYefcXepWl5f\\\\\\\/BF8x71E8fbGQFDsmFSxIlsXRxdGmmf3T+xcQ\\\\\\\/4dy12c8QTl5Mps7BX95kGzkpM0Pk4dovtBvT5R2UnOtXunV5q7cRCNa5jFPg06Lx6XSy8aws5gBw\\\\\\\/KbVfPzpfQfHB73KqzcJnqVhRF5Qhr4ftFdORef35k0dcqEZFH7\\\\\\\/v7Y1EO7UQbooHuYPlFG6ajJ5gzI7MaLL6zerzE5Pgso+3yCUBH\\\\\\\/kIOPjb8+1aCeAwF\\\\\\\/fXDJ+GUl\\\\\\\/P95sR3M79ogMDaKQ\\\\\\\/4KdAdQPYd6\\\\\\\/Bc5i4E\\\\\\\/9VhNSKL1f\\\\\\\/z3OEWS6AfDikeqtj\\\\\\\/4RU593o9vjZV7xDBZz\\\\\\\/Znk02VabJZZ5TJf5x7+wW+WMVBO4jUNAQQCKv4bRh\\\\\\\/9pA+Ggy9TEt7rVY09ZfIy0wJNxGwWhBdtuyzuk27EkSBBRyG6rsHQL+NoyTMmr2aJ++j2aSktRTkB+HW0KTwdpPIhFDq2QiJyqi7LYX\\\\\\\/5VvsTgNo\\\\\\\/vsugWc5bKvgfivDhjBi3uSoerz1yNInuqrE9h128aQrU\\\\\\\/ZBwQAJsky87zrmJ0WEJVwx0WM7FSLBmxVVnIMpBcKjLD0LY+E8u3qr0e7ib5zUuQIblz\\\\\\\/pW4xfTEkBFUEL97sJnQdmLF9cB3BVqDsU+cTdbItDrSeyiAfdqyJymDAZkryZqObrt2mh0p7dFl0Wr3a2wHKiKD06lwF1A0RLAEUh1sb6v4mntdbP6w6buoYNrVNOnq\\\\\\\/nynryBUp\\\\\\\/EJ3lWw9\\\\\\\/+UvsUcmNFOo3DHkSHKW5SYqw9T5OXcHVRPaPEpRwDQmPIaB2\\\\\\\/0hkkmED+5wkMrPV4XFw4kgsNoQToXtMCUyW511ABTq8K+5OGeojZQkR+5SgyI8L1EZ5b25WW8+Zn\\\\\\\/qq1DSFGeNK+I99d4K81ItNqZQIFQcubAz03QqR8IYrP7CXQ+ln0km1TxTrASPYQk5fZZinDgWZgiTIv7fMSyyvzw8saeEi1eQld26X94bD\\\u0022,\\\u0022iv\\\u0022:\\\u0022mUvW9c7dAyWdHhwIKU\\\\\\\/9Ng==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222bfd993b3da4d8c747431fc3b620c7cdf0d423073bd242dc2e3fbaf577d17417\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224+AFJaeytcUxAFhWM83VnUKBJ63ihA8Ll4zneZMzqXm1b0GHuOPgFsUb1v9THXMk4wGvepyYmBXxXAVc46ZYcrRpJPuaFW9UXRkUnYacvKBcsm9roWtIH5DW0BDA5LvlYM1hYyRfXHKKc0liTaGz28PJU3jxZwa1HeOv1CWeoQeKIIEjHuI3aTiI3PAyPj74LWU5Kppjm7MuOS5NwWImy0fc4uKzVjYCPXp6vQtaPGq\\\\\\\/9xUwfCSHRdPtmUTFa2tl+kK24P9x4Txr64cfemuHJ2RkA4prt9YJK8Ihzl6Z4KB4jTa5Z9rSfh7DWZ7nAWmGF2C\\\\\\\/fBjQxP7JEQVDmAIq2UraN7gVjs+sqhOmbtRBzJK\\\\\\\/tAAbUmyxoy5Mf38QXi2r4+\\\\\\\/CrcMy55quiEdheWfqGBpecxnRooKfMyQ6\\\\\\\/N8sVy5RR1K95PV8flNYXUJviMcQxCHGizkk2w7giR35znVs8ufAfeunEKV+Vtb0F4ASkkoSMEGuyD3wM1j\\\\\\\/woyPaByfYeKnIfcXhFGPncL\\\\\\\/SFDTwRFHpjKKB7fIFwef9PkkIAfBl0sdLc4k+1qzsYYXIdjVKW3NwMiDqHbtLoc6y8G0x77iklcyKaCB1Cnq2Sk\\\\\\\/kW6oQU9VE07FjEn\\\\\\\/fG2\\\\\\\/cBO1BeQX493gCxwPZC8JJgXhjqp6JD78OkPx+qQTqRsS8hUtxysW9KdqGHxkmXxe+wlJNnMnXsTCGXlNWTAV+ZiZQ3XFLJvIQ\\\\\\\/3M5cJx4iLmAG1UHfOMEfZAGZuE5JWLDe9keIT4e6+t0NsKQvG3hjuO+NlFQGkCL0fTKddEZQOo2sceCCaKFVzMTmA0QyNMqqUVSHoMldvQztB+BQg7lKlu3dwbtMaFefPvTo5ljgXEcYW\\\\\\\/57e72xVSrWux26xElyDJ9X6YsvJbGTv0Qyg9oCFhbI1atGdDecPW45ItSiyNf3DFR+8HMV3sOFYBp9yG305Fv+R28ypBWeCgW\\\\\\\/fCtZ0nqXW\\\\\\\/KGTGuyrPq0HPpQupJep3z86LGBxjcVz2ewO81a3RAa\\\\\\\/gKlGQT5g7WOAI7KoMGkO8zMwO1SgnobKiabssa4g0xNIukCCqHskjXuGRAm9N6smKpQOKlWRSrpnufP0dAJS2rmFGci+5+2dMVHkBIxG6yxOYoBYtX8DZ7tWRtJLricd3huaA6\\\\\\\/utHxdGtmAF83snLzinwjxiGAyysf3AKyOZ+nnUbG\\\\\\\/75cthlSpl5u\\\\\\\/cB\\\\\\\/db6XFj6zqua7uirnR\\\\\\\/U942LTLRYr9bfk9GPW\\\\\\\/x8ofPH+pRqjBNKfDXGcDhiZYaKDqBEdlFpdBLXqJPByBRLTpFcw2679EI3N\\\\\\\/kl3FKrr4f1uZQyAU75bo3FU6ldLL2udDTQUwk45Ap1rok4hVI\\\\\\\/PI8WnUjLtEMM4FDoFaekIGHF\\\\\\\/R2+TQD2gqT0zbhmcjCxIg1OHRb6f+P0O6IxopIC5AxbC8OGqPdpn9avF2shHHpKuTQD+U4vK0Djcd4xtMQgRzE74dKYitZwnTl1uFfOy0qCOEMQuECy8RbN7BBOPUyd\\\\\\\/eSVwJA5X1XUZct9sI4EyspB0hLJYBnEphjLKeQQMmMpN4WwENRwNeSQb5MtRwyQYLjTbI057pEffJsJyrSzqD+9AfmJlOs3t084XxwrpRYk8JnKrWIIXF8IYcNKjID9T4Bplv7YRrjfXk73NogfH\\\\\\\/J6lfjkUnfly4AoFxiF+T6ZlnAFcgg3PF0i7CrVuHcfToHXdW95bsRIkkcVLFbWqMkBObhggyFMkUkWSuqNekZJwe\\\\\\\/KmVC0a3JN5\\\\\\\/1J6LdB5tkKbQmWusDx\\\\\\\/Hfk9ga53VkGbHR7qYNhMoXDElw8SXtS8JQEDDfn0hoTys96Rh350c1nCZEuFvWun97azEpTMIjiXb3aFKq03IVsgJbsximhGjA6nSGbVMyyqmDDkvqC1ehgl9o450UNUQtGy73krr6\\\\\\\/L5a6WdXW+MoDX41NQfmtRY6PEZ+wq1MTxQpU8kDH7dh9c8lGNZ7WQq2iHgR7Jrp+X10wW5bjfIbLrkgfR9+pqtasbF+G9L1ErI+Y1cg4iCfpfRgit2wAci50UMnAyWjJSIBAxJD34B7TMNQ4U0gygPHq8QDZCZ5WvYr6K+KyikaJ+VHb\\\\\\\/WlT4vsmgE5i7w2ocxgRBmK34uJXhjL\\\\\\\/IhOEcxcIeQdqXpq5M+Jl1e6sljEOQRG0vD8UOyHWlczbeZ8nqaAVxDbOoukEC6B158YPbg6XIvEBRsmlz+ef1NA0vCqytkr2O1owMSvIiO91bPZX5nvYMmofCjMGSn+JqAziVGn1nBey7V7nqrys4nHpHcEbma6dYatAObdgJm+SE4ujmGXDnWZQIjp+TAseM4leNWZi4CSdWeLHUQAxeaUvq6PJeR5HaroJu9l6K1YWYK0F3halecooCF+Ya\\\\\\\/EOxnBqQ60wVQx85Hqw1NxjtY2I2cl2Rs1n+jxqOuK99yXlZy4Wd3cwzrehQi1k0P1X\\\\\\\/y+YtLuULai260qmruSt9CPQ\\\\\\\/\\\\\\\/VNlmJ+0pxiWtMrtKMJqgfcqpvKNQGKgbJe6+L2GtNKt0K6pTtujipYbwislo64jfZ9F+vcqKWHzcj9h\\\\\\\/7TA1b8wW8gNp58q1g6nDXBC8XwfneRJNxNFTB3ZYtq21c9MT031bOUEEw0Z00Yl4pZpl2fZwMUlYAcKW5Ru8mpfP3YSH4U+dpzqmmfyOChROc6XmvHMNEKmh73dBqvl1IlM8Q9Thfgbw9OtPqm+AIBUrxJqSTbkwNA03L6wClgQ2Hxd1ZZ32IfaJivxqMB2S3yxcDavcjTgY1OzcDT1fUOoDJQtVPzXsq7UVbQhNpFiFb7A5GMZ5WQBl8\\\\\\\/SsGf5QomkjNA8gcPblJ\\\\\\\/rPugxeC+uhyRiSH9DpO3B6eMyHRG2C1FPr+ykZ\\\\\\\/UpOnc2IegcSE26xGKlGPAoXoAy6RbLeFpQcQULTC8O7hnEGKjIzFCyh6qdhcY6cQtb13BY2k7GbsIdL3CSouLLB1Di9DRBvK7am7kNmcfuDB4QBWekomgey83AJuzJQcaAMpWaI9uQ+8HgBvD\\\\\\\/oRrxh5ttZXzrIwxFKPXm0XsSXYO8X3O3\\\\\\\/egtgcmgSskhVrnK5doupPvsyCZvqhLJ\\\\\\\/XmjNoe95FQhka1O0OTKbdMlCk0qBQ1cdwzF\\\\\\\/NkJBxEVhKsmFQvU9bd5PzJjRAttVuGeDN3ol3ibPsn848coszcPfNrjIF+Wi4HXK9MNQpYVfNVNu6JqP3a76QP4\\\\\\\/y0jK1O8SCxCEJrsBDeuU+Snxcz6PviLvKnozEUzCepM2D0cJBk7t3j3gTTxd7VfqNgPfI96VTfMHqxpjvl7AaTlWeDzhHAHdLH1uIWBIQQRe90y2EWD2aOSlz0VgpYTj80H3qwgdv\\\\\\\/LX8z+AKGxCKVj+u10jZXRYDM5agwgQTH4uJjLAzxuQmJ1+tXa3twOQtnWpqE+Hb8vHNTB74Dc3pVNFC3yqdDBguhtYdapbDDSvHaJE1dfno75AnCPpGpO8akNFLdfwxs4tAXRsHLg5OCbTlAwswdZDmlfzdPObhw1OBOjawkdsXZYWCJp69UvgLVwrZNMfVIc0c+TefG1tZMvUl897PApDwm8I7KmXxo6aHugAi4ZDbPizPGMvAqw\\\\\\\/gbbEPr+XcTxJCglpweonlG0oW1i4zQgYDu\\\\\\\/zRTLnWfaHHW4Uz\\\\\\\/cxiUTpAqMfu3cHAJjer3bLM65HnmXYzm42VKnaiXuOlLKKQz4OyRFeq5\\\\\\\/656rBJYlHND65Vrbp5Yr2cSjqZd53EwqsPnXo7FGujvAITQAOgaQLlTY71zZM911CJOXXWS\\\\\\\/FCuQZjtuHYGB9eKYDHa9pEuIO5hm+dHt3uL638GKa6jCeuGll7V9UcP2i9o1jn39QAIbdI23DxO0okzXFM6Fs1X4RMBptKMtv3f\\\\\\\/3Be7tJw2MLXdmMdwTuPbLoRdNOhGNOK5gMGrGnZ7SOmumQug5Dz0wKE8FgaSUiHd3XOYs2iL2Wdtl8CN5Z3XkSkfU058+\\\\\\\/u3UG\\\\\\\/AAr7c44L4+COVbfqHs4LSRo8MfQ5n4yWxbgnCzOKzcoK54vwbABsHGJc1I1iRUvh+q8yfpTdrF8NzE1OwawZaUpH2cETc4IDAwdiATAA0mHwWiGzidjQb6jQ6tYAGBF+1DfiT0NTPGQg34K3zRr6rlktJLoF+fdvF2n2fkbEVn9l8wYpFbXRKbTxqNr2ZowAF+Hwx0iKnMhAg8\\\\\\\/yCwpfiKoLiL+hYsxDJckeFlXWRMW5RdSNBHACiaKbY0pJAUTbryMLEJ79+vlrJPesdWDn+bMCwEd+PrVyD0Ks\\\\\\\/66xtA6obHQM5CKVEKVna9swJBU6fqSG88y0HwnRkgXkPz2h90GTEVy+iMnWES1LZTO3NHWByMNBUxv2fLaoDLWVPE44h777qQj+aUwmmOOwyHLaWrF1FcoNFD7+KxUdKfYD7GRvcVbqsPn+o6CKnCWOdVuRBxkeCVu+cuY828Za5m7zmqoJaIDFx0bIF6xCUbZxZD6RKz5aB2LYGZaPpya7t6Wlp5sV\\\\\\\/779zFkUPiSMPmX3S+7O6XIhMwjvbChtHlRBs1kvLmkWM0eRhBAuM6stD8vTQ9xfffs6MOMafRQ5D8KCmV0X0vyQz1zQ9HG8xogmG3CM2Vl0hCe89LL99romMwvNOmxsyN4mVFsTAw70geM69uI2osRye5GbpQO6B\\\\\\\/UmK+JovrLQzFGuFl6unWarglbOmgs8QdBjQJj+EYvyrv3DvbJkceQfZJlZE41rbSL26LcsEzA8lN65XC73qBqlQ0GnJzWl2L6NTrnVFnKUHKErBAoGuhNOaiBrgEe6uRIBlpnd0jlC2XPVagJ9L2fDJZjzR3Yh2jL1mMp5mn+7UbBsZGmr6uB6qxVt\\\\\\\/44qYBpkqyHMssgNi2dOKepH\\\\\\\/9AHUQLjpAVpYduaPZxghLTekxWToWlisfJaHkm02q7K6DhEigffI860HXU+floXOhO4\\\\\\\/FxyU9TBdEytkszcKK\\\\\\\/PD4YkrvgrG+Qh3cf\\\\\\\/MEYrRy8f2htmw8Hud6qZ\\\\\\\/zvZvKT0HT1YvrI8sogjU5b6f03eKwnM1Uqn5cEKdvxNULbWtWNVCJtapJDd2QCWWy\\\\\\\/r6F6BKOxaxH2iM0YfcX\\\\\\\/2TvMKd2vQHznZDo7GX\\\\\\\/9pE+u2QrDfzBPOkjgxa1BWeEJ8MlWNjVRIGw\\\\\\\/VzZVIgYyWTd\\\\\\\/rgcaPAAwzT0wRBbrMILSGPZmU1nlnSJSF9WSaFaIWSohVuwQWwh9v85\\\\\\\/iS8Bzn8QHN8\\\\\\\/Pf1PQ8IqtwQ7cEz5HdMYWziAsp18SrnNQD1OYDIQpGYFabQAdJiDRA8L32rtEgbkL2sYmsRD8DtLHBSwePc9vLoS5rZWXNApYxfDeicUM7KZN6B4TJ0kWp187nNjWmprMJSxtN2rmiBlP6JWQgIzuYHz1sNlOIzoqvOHACV6986sQegWhJOp9W83lxvL9a+O9JBnosPAg4YI15Fzyu0BtR7pgQRt+unBNCClRHN1HHLjL0TCLQMPDPcNPAZ4CcNqz+4rtWrSUbzNuJA3\\\\\\\/APbVFMB0ExlOuneqguEHF3fNn5tmoyTVSxqeVeW8f7RkqNFS6qQlWumW6UVvSPcYZUMNcsm8ZK5UE3EOt0\\\\\\\/KqjXz6xgP+beZ43z1gl7dPV\\\\\\\/N+r2rJU9Lmn1XTPjLwk6MvRXi4L1HrV6udxCIG3yiYgoWhbqZ2mVrXcwCgDJ0fRZazY4xST6kLKkdiVdbjIBToS6M36iu7ydtY63OV3vluzzGmfqeQixNX1yS\\\\\\\/S6OwKbIarY4+XHIvbC4kJnwV9vWmMY33VriJ477pxQF6RUDF8d\\\\\\\/WxC0uQq+vlrON\\\\\\\/MOeAJCYr89nXnls4UOROFNimPg611GvoetmoVOHuqqcIP8ZhPcydQKkSIr+rMsuG4gQ8jmUzHag7\\\\\\\/aBZ8AFAP29IsKleikg5aZ0Km9c3TxUKenFSRNiOHc7TFmzXWKi3wOWaTLGOl3sfdWMKVMMDgBujAOfW42al3Qck8L\\\\\\\/e33DguhTxR69C6PHCN8w9CdGqrNRg1Oz9ZZ8nTDVLvaUF7wp7qwYAoETYfnWKQloipATIsPgKTD0Kmv4UhhKXcrWswtp45+0rvoFCFhf7M1JoHM0GwwQbgDIlP3V0QcAjRKT5VCxF4ocFXScP8LZxug0ILwVjfCMzsQ\\\\\\\/YKP4BtYvVaFc9FlRvF00YHIOOaaiUfQfogicaza7i\\\\\\\/\\\\\\\/YfmYOxEnFUPmoOVn8h9K6YOZOPowJfmUa52TQ9ugy7f0d9Ku3KDbYvht+ZjlylUll+z48Y1tgSaY8479vKhn9ZbBPwYeQh1uxlxRbR4AI7M2DSeVYuvguCROCEV56d8sZJ+z1j0XpdntolM22X2Za5kLBhbIK2AfBHFY2VeMlRMROIQfOa0FSV018npPiJO9VxNkrrrSCKtq00wdAGYLCkeE+b98CqQL67pl6h74Ic9tyoPwrEKsiteHXk3KYYz6CoVrP8aOQlRvjWQo80cfvduf7V8F6GaUfcUe02QJrORawC9vibj\\\\\\\/C8W+m7Pqr8WvXy4VIWpF+ibeI3dytAKH9b1MQdSO3TWV0WBlMzBI2zyuJCqzkN6r4jq\\\\\\\/Mjuxf0AJmclp82hMrkmts8oZvlcOw9kHIG+FXo802+4pFr2WAMYPUDFDdE1ExC3ZvO+ihBBpnsJxJYX3L2mfmFZ7\\\\\\\/VqPYucla3y5eo+pCs67iCGCevso0E8umGEvMNdm1NgwcD1eG4WPk3SBy8RA9KKlVSYUSzcrdNCZeYc0Zpbaj9INTJUHuNfSCsToTEo+naKqAMaGgnZY\\\\\\\/2pA2ogfraKGqsVhj8YxZTJ8ac9E6PlKYA7B0ABg53Td7QSZdma3+Ls1hWvauq9EkZk3+7YbU1cSTA6taNmZiyPcKK9WGLZONDQT+Ldn0gnVzvCibF9laSJ6gq4BqfndCUWYxjhZBdlmOvPg\\\\\\\/yg29QZ6j68a7VmNLnMsiuTeiWiRBIEL0P3AIgSZORfii75TOkEyhG3\\\\\\\/UGorNlAm1\\\\\\\/wWBUckQnV4qNCrfEGiNWsAFt9CKSmUvwzkpDYdfIyy4aX7D5jP+DsbYZtia\\\\\\\/YcnbwvZEsW9DhPITp\\\\\\\/t\\\\\\\/jP\\\\\\\/EIye7fKvriHHdFkpeQ7DRTziN3nWQDS85ODyQ9PW8xn8GWiL4DzN\\\\\\\/JPFT\\\\\\\/hWWrwAu1snfvEQul0xZ1A2ya14uHD3q5+rkhjAS\\\\\\\/Gdar\\\\\\\/PJkWdWgdCMWozZBhipz\\\\\\\/CTruUYJDjbF0bM95LSCxKYrG07FaZswNugMArNFh5cMfzMuJwoH8WsBJN0nK9JoG9PwLQ6JWVPRfesjyHaFixm+\\\\\\\/+CnQ3p1v549QwFJbOUSISu\\\\\\\/K5vo2X8uNiq48i5amamroKFXYQ4xOeHYfz6EDLN29dL5dW2XK787URTHBWA9arxSCw9aFYl8sQChABN44ZZmZxw0xaK4ku+fOFqb+tc9U9NIHTKrf0RragRMTmYfDtlt8OaQjYnXO\\\\\\\/6gjcsp9mHf0hBglLbsTApdr3JuzT8JHYMGOppA\\\\\\\/S1wvUBtc53k\\\\\\\/EygX7QcCDfK1jITtjKW4JUTKqdQIuvWZbVcDv64jQpyF+0eAEJBUtZ8JWnkVHB1FmsdEkfdPoJKa9svZ9HBir9oIpZeqLmH3OeGOrzUdNe6Rr1MCOCWyrGHiXEBl0bFqj3NES+Ddk\\\\\\\/dJ4wLgoYKyrxqEoeVw7yBwnCUq9wBZ4k33K4vaMH8ng9BU7FF07zk9DdNslDTUhF\\\\\\\/7s24Spg3JAvGzPb8TrD0HP1Zq9MPHi+jsR21qFWNLuKgczn8g3P56+sAY\\\\\\\/h2d3hnEpLCvkU2VqJcho74iTnVkSuLFu8sB6n3D6VxjsQSppYJ0pfKrcFD4qogzCnmMkNqAvLNgkv3Phg6Fr8B7k76R3vUmAGNAHfemtP5BFWk\\\\\\\/U8Lis3qjHtlSBYFWYnLbbcWC2VBJcxyIhDV9U\\\\\\\/O5ZUNUqSe5fBxXGJdSpkAPOKeR+UMrXaaNzSBLJwYxWQxvFfNdTZsEFCb7v7PQgEva3cqLFZrfAI+rat7ska3hwrmMklDhi0j+HmVJGrmCweiZjnl7QzZ6x1yYVh+12VCOIZ9OE4SigdqtjrVzZ\\\\\\\/YLlmIHeblJBYg9doFLcS748NA3hYlH8E16lJjH1aSwAnDsaq8t\\\\\\\/kOn\\\\\\\/bMrdCVHgi52cCv9AJR52QDA\\\\\\\/BhKV4wiYujQlvyBLBgPkBt7lobnlqymBHeAHtPWvEI8BEaV5CB5rwycvjPJtjrk2qhX2u+sqEVZMxSEAwxAhSYbrTBCJH8Xysv3YhvZqCnKRnmaf62jySJcUGbjMXyczjhXMlhCVVPWiOGDl2QQJLuaDLrgpBEUdhoDUjuY1bD\\\\\\\/lw5AzwvkTmPeeJpN9ExylQo3cIABQvtuWZI\\\\\\\/N3a3XSBV1nq5Vy2Kb+fnGvN9ubr\\\\\\\/UtUhLeLu+Ed4ri6+OZw\\\\\\\/gJaK3AYkSUsbUAW8TCErAFP58WD2zJNkFgFEXRJTQUUwYGdpqAZ4pgRcLHDT9NbaBcOviYsFXHwxPyi3Aw1ata7sw2stkJW+WgpE2Qt1zNB8an975K+7hBB\\\\\\\/i3xfzysZyHvAnSLHipGc16ShCp0a2C4Khyg0PolKaOHEEghDwjA2Qk\\\\\\\/Gslu20xTvYYrvhiICWUSXGIhEit6sDwfRdtQ0c564EkQ2z31FRXTZZqsRgVX9aDvtbiGyt9qk22cqua+sX3IUvF+hA\\\\\\\/cWHplRtl1PxYw3u9y3ob67xr7hjVxFdiWWsdNJd0Qc\\\\\\\/yg2Lpn2pb46J5AlZ6pEpt5QLRgU4XHAw3VwMHaAj+T\\\\\\\/wKO\\\\\\\/WJoVroEbO8J10LtZ43GkTI5aA1ErOlfYrKKw8TkXRVjpTcmfw7aRrq9+rg15MAq0YG08LZnW3ClCDwPznUBhls\\\\\\\/d+g8ZB9Y5DllAb1KQvcjko\\\\\\\/XUzcI+dj6LTd9RINtkRtIvubqYQ8XAObjmheYxYo9dhPxCj4ck5DsXBH4lQ1ScJaQ7CT9W3SD0dw1GFhg1d9VXPDh3xAaj3+kRQBMNMPnHBI1hXYJg93P9aeUYXtLeB2XDO5q1IXbg2qPAwjX3tPhE2wVte6kU7Zm+5VM6dzv6yCqJMEig0MNIfTWzFcWY81ckTLx8qp9mAT7M8WMDls\\\\\\\/z5w2wmYxTwkH1iULX51C8yH4PDQrw8AtIkyB29rRT0LWb7B\\\\\\\/V2HHxguTAMZpZsE77Lvjtpugb509rz3yFshL6Yz6mAZgFsiuVOA0DOnvUxRIBzCxw7Kuu3BojMnGCl8cLCZ6u7SG080b8+qqnURcOz3VSl7apD1PURYRH4n7axL3kxH+04LNrhqNqRBkKmnU33Y1JIEVR99UhJJsLQUQHqsg1pE\\\\\\\/3q+FtqxRoupWXUzqOcz5hUcbjH3YfF1P8cmlLnoD6t38nt41qhIH9zWAaWSgybJi+0n4I9kZXlkilt3zAqdEjlkfouotZOq73L38upli41z7yOiukSyB10yB+OS\\\\\\\/XytUE0euup0\\\\\\\/3HLOQ8OZOkz66MHToAfDwVTKo24mjQ2kYuuFOfBKFqk18UKi7\\\\\\\/JtI55sORMid9O7CvzhYpJV43gD83oAlFNbPM7BfwSJD+BqMsVevDYhjZmBQtcv\\\\\\\/q+dxpYdq2w7dkzg+Xn9ur6e5eOgEHv18nCgYSP8xJbejkSS1YK1V4t9H0SWkPW788EJAeX4TchAtUBCLl\\\\\\\/GRZYOrYl3UZVZFRrcl15YD3mAnhKVY6trRhMKkwUMhzFRqfV3anvXnh3yoN9JZTiMGxAg3cWQue\\\\\\\/0h2PDUfodXX8uzRPIT4yxKJ2iZ83JV7AvkNppIWJ0IZ183y6g6Buq+Hxns9tYpXXlsCwPg3OiL1evBNuaegOQReuD50ou8A5FLaXuNowRVyFtgUGwLW08WzwvwGAAwekOfmnIOvaQoT\\\\\\\/90Wxtg+dg1anr0utwNi2Bsv7NmbyNszi0WtsDYr8tJoaW6B\\\\\\\/KlS0evwzo8IaNZ3l3KPlE9PLMyDiN2lh4wr+Yy0Bf5+ErtMFpDwShh4LNtim8WNLgNjQNSiyqYNk90KNP0OpqoK\\\\\\\/qBHUkzV\\\\\\\/7ALLCmltsM1j4x90MyUHZj7X9S3MXTeq\\\\\\\/M6XBuBPorAhuyhR\\\\\\\/zQuN28WZNvvxCtuzuSu9N179iyrgMjcFJjoc3RGdrAXaq6T9yT7XPiNC70ywZNpZrpPp3naK76\\\\\\\/Y2RJxsNT17UM6N5F9cm8QBTYjaV+CGD7+Snr1EfJFpjaH9MisJfxVD6++FjWUMcxfw\\\\\\\/Qko7NfKLJxl5F2hoAp7SMsnSY\\\\\\\/PaROrkfAI0bgzIMluSsEA8bLWfPNl9O\\\\\\\/gA6l3+WwcxAe1GJYRBmac4wAnlfapokP2X\\\\\\\/VvI11v64eF2jAuOA\\\\\\\/LiV6fdGqlui8+w\\\\\\\/iu3vQfMerJ2EPGplcVWKB94ONQY6bwgb9Or5ZXy19zjAxxvtZqzteJ\\\\\\\/9JbmGQ7QprYef\\\\\\\/zwv8inboxe9HotxQ7y7nPdXAMxE44ajM3XKXLpIRrn38y8lNRl+CkMdaZO9UayfL8Wyzh6DoWQ8bGvSRYKtehV7j1aeeXH\\\\\\\/eV9t9+jOk5CUPg9qkugTed5vVM60zFpb\\\\\\\/atfG5FMWYPbJX5maSD3UtJ7R+40ixMFk69bD1GNiSkVVInWY3S823HJETliuZE+LuUqm9Ws9DR8eCryEzg6oGUEpBZu6xlH4QEBJFyDsejdU+yVHep3RxkCxkaYKS8FMx7HPXzcFRroN47U\\\\\\\/9L3zdLhLhJfbeeLTDkTcBryAjLy5\\\\\\\/3OVwwJmiIyHb\\\\\\\/XmFyIIF6unduek0CTY+Hkg9W4+HbUtqBlvUamZy6rjDGi+wlwt0lR5qM8afTXbOUtHyFEP6aub8erjQE00zjT6WSzBTpbKSUPSgJcTb1owFYHY3U5528U0k75e81KeYQ82BPPoWEaOq4qhQL\\\\\\\/fRDWQjycKs8UBTVbBQT0x2KpHoS600tTWlOuepZO8cPCsBjpSlA9zOvFrke6Mmi\\\\\\\/KmJSlEY7gsCYlOqLO7k\\\\\\\/D7zoqwj4K47SnWRKIBa2as\\\\\\\/xxK55cx6txM5pIekkcV70JGhg08BitmovCO9kkOevi7zpSQry0\\\\\\\/uUBq12sEGPR4PRljaoSn17Lw3t9k1l5nRwR+34HfcqbOS+xZqs77KPCGVFBvGOrlbT4q7X8ggoyzpqUEunjDy2VUfg9jDjfooJmgU3jyJ+pccknPad5tjvU6ti5fGlcVLHMuDnsMZsJ3zw0Cw42RjoItUwutlaFE4PMd2DqEGi110jg22XCUyk4B+vZwlYxK\\\\\\\/Ih+KXmhCwr+pWxy6MWR4fkWecDwXIMnz4s2Op98PLERZlb7cNYDyRYBwUcMBaa1sepl5EjWMp228\\\\\\\/AVyC4Afv+KpIn+jwi2quqjYU2xF41KK7v+0BKILzVPTrEc3pxf9W7fliVSc1VfA1sRsgGcuvjY0xudT3s6P9hpdKgv0fLZxtLBIdyQFoEdUH4AKH7YcGOfiIlG4XOYgk+NKMec+iSOsE\\\\\\\/pRzbI7czcvHLJERYYgPV4qy3lc4duhf9UJdrbGspUE8SeZRq\\\\\\\/6LO8zPrUm5gUVwBY+uNwikks3baYY5CXcAX3KruBZ0P9zj0m9\\\\\\\/6j9RggwxWWZMzUFh3JGPrNjIk4QT8JkVDPCDhfLcsP5iBOn1Ip7o8JeB+yNCZrDkItiXMJ9ai99+Evj4l33Jer+KttKZqmx7FkckYdMnhpgFPytVIY4OUrcmFZeveSUSG7THY3Kcascw9rLPSpRlKivxXnm9W45ftc4pGtHnSI0oSwMde4f7xnjAwxTZTgCDNHbJEC8nNnLFViXT\\\\\\\/fgubc8DZn7vV8RQBkzlZUE6CE6+Ezd9nysUtW3GU+YRh+XLi+l4C\\\\\\\/Lb5HOeECRCPNCTEOi9BkLtFlXWf62u5uhcGTZnllIb4mBwrpJ1ogMOq7YVqILN5m+twgqBn1wg5eky596oyafJ4d1XrpRjR\\\\\\\/g0UkiIe6Ols5m5h2upSvVLBBuEiCyI6iP+eD3dQ1d1dsVlhvti14Z\\\\\\\/DnM9hlpC6h1MER6IZQkjHQAaWPWfp3OeEh3wzKFrPB\\\\\\\/U9s6B2eqPRMIDoFcWftxrXOEI7EolPcMhlvXLCvRf3A1FzktR+uaIrHEpe9WWbo02DNS7gSTGpRBh87aVxphI7jHXGhW2lWq0RLe8jImSw2uLSR+yJs0hnU6kXm7XsKJUew3R5HWinBruHvTtTVejMPRynuvhXcKuOsyGjDvKFejTqx49VffBTmiBs3PCYJT6ByKPQAt0jx\\\\\\\/4eqFgxwaA5FAX\\\\\\\/mMBF63A6byT8RXY4biEn0+Y0ZTJSeqEueU0B7O\\\\\\\/dPP6tvJaRJnbLly0nJ2c+I5usZmiJOqm+M31VdflLZffsIFO5AoCVo0cG0AOeQVUgtaS0biel0eDHXtuT+zEHQvQH2VZd3uCU06nMnz\\\\\\\/cG+pg5czKmDOTvvNNRBALshJXF5g7fV2z78POkBiAUbGTtlU2HOVRAfz0T1g9vBVLJOQreEjl1cbmZz6P+6K0Lf7W7iKN7JDYFkbsqcjAxdkU\\\\\\\/lmEccZHpDcJQw3rgzTYA3xwnY74dyKPfezv2tNLn79IIXcwaqS4dkg+vqS23h1+u9G+9Ri4F8+QxiwCQB3ZIgfxWwGr1m7bd72hHyBSuaWhKnl97S2N3ONYd1PY7L99sgUh39M9s0hDofTg0h7S5uG2YbAO1h36Uy3IYQjeYn5bxMcSVidBYLb7o5cgYr9+RjvRmM0eob+5b5FmxsJoW1Hn2PwtAQEMIebJV5leFKPfR77\\\\\\\/iDoYh2JJ5qYyuuZoyDln+AhHIo8+NqLy8W8L+AQj\\\\\\\/BElsZFmaUW9MHwPqjKwXyk5Qgow0dcefeFeN4XO2pmar\\\\\\\/Pz9G1MB+twjGE\\\\\\\/F0AGzOO6x4cvszQ1btCSO9+dWCyzN7Xcd09qPNqtYWud\\\\\\\/qNvBu79OHNZpyaXUTDC8etmNVx21zCLIVjbh3ZuhLWFI2kOrDTVDXhnK3XP\\\\\\\/vh+cmjACd6f7M4BSXp2pP49PW1YQ99Rb5PfOrSYmQ23aceDZGMg7Q5vAHsJUh9kWGa5xOyOxpwc9lx\\\\\\\/peOLdghQPzTHgQ6BqcMnnZ4Vy35xHVlAjtcpu3j5bDjndtRapjLS+AZKJNVd94N9hiVv\\\\\\\/\\\\\\\/Gv9h\\\\\\\/OECbRrcO3CkT1WU7ealgrZilFTUlcTvijstpJQTPaOsK1pN3AkHOChA2Dpem1rwD2Ekgx1dATNIm84wv6fiZUAvu8iTeV4KO4WGMHE7JSpa9\\\\\\\/WtSmNp6tLlYzgCpNTjm17b7ynf9mdB07XniSksl9QZboTR6l5eyn82t9lh3TA6AfcngG7QqypAWAIIQnI+zSzFH81Nxry6EUeS0RkSwJMZLFDGcQBcsDjdayr4S6mCxLxHViVPRpy24FyCTgUqBAOjgZZiNhdEuXSpc82Pd9\\\\\\\/3cXqkFUtL0tVaAXOW9UBpiJiHpA60kvMraNYZVsJCCglsOyfoXCxM65LAEYsASeQYPq+JftViQd42KEJQJWclDz6ZJfgVDK8jsIjS81BCjtFUt6T2f8y7ccs5JLsuGEBoGciSkDxyUHLXuxAn6oG1GVgNYitT0bRw\\\\\\\/ZEOzJFwZtwuivFLoH6Kr82Q1Gr6DRQ2k5pHxJXfznuWeSbWNi3TCCBT4xwDnYVXNkUSA7v263hUbAYHLybU89OmQ2P6uri13Br27cArFPdetply8cdSYA90TYeAUiY4iRh5ZCJe6rs1C5tzXcWB73zgZLXp3nRexMTj2Nu54SE1PuS5yLeFkkOyl4HiNrSIrsXzoJzlTKKyQo\\\\\\\/wQehldawzqNGN2d8vDT\\\\\\\/D6tUYSsQCoLAiyyaJkckB68Hw9X6XkGe\\\\\\\/5zZNKr9kBLBgqEfC30eepmgB8u5JWnSgABUmvmDembs1CDUjKNKiX1wlUBfIL+7CwQmjWdXwVb9Z03KbM9COcq82WqEOKWewwCkH1t04I6csiUvPED+GdvOKXmf8TSANHGJOEYNBd7JKL1ei++E3RcDOvoJAH6R0ndmGuw99s7C\\\\\\\/h4gAqPhT0MR8nXx9QER8frg64ol5v7Fwyn\\\\\\\/GVYFl5awRfTH6hYK7m2ikj2YY83LqzEnjnt2sbZC6LReBsKdFqq6GyqOaC9IM+tG2Lhu2T3f9z3s5RIxzdL4u2tdGh\\\\\\\/pgsuAfdpuNdzctRMbMXp3tEkpMmQNWg7zaakGNq2bpfZeyOslTAwbIURTwr6PdyD0Z+QQjAokqDocSfcF7BYwku\\\\\\\/MDiFXTKfdJdAqebsnLWVeG2UW7ay3tD\\\\\\\/iFc7IcUHFu8R3VBo9n7NPUcFLPUO9qykjyykWtU3dkr\\\\\\\/VhzyImXKttXF2Jdh7OPB7m4P1jMwtHTfukxwI+gbH9RkEwQ3LggafGiUPNs1cN9b6MMdtEamgQygyoCyvdPEmCKdv8AJsMk\\\\\\\/cJt\\\\\\\/8dimJ5Dtv18AY9HcLKkCTj2\\\\\\\/Wv1ctFcQLTETECXTK579OcHGzvmJWU5qZ1EvAOxrC5eNC0uo8Atc6W97FDDzF6W7hjquNGGvUS\\\\\\\/IULn4I8LN04YsQFLZRXMQWr6gyX5tvKacWq47smMbxeRJH8DXXsacD1YEJXks1\\\\\\\/yOTVrIDzc6xAAUbVeEE0a4xrhNxE0FBkAyBa9KYagNfUCeBXm9u2goZWBrb+DWNpHKt7R+45OJ4vB6a6iUlLJNnqEWLUb4lsAowaI7Y+K4BgSgOAe\\\\\\\/6pln6cqHTZARbnQWfr5tv3D4VFncng\\\\\\\/Fxsgv\\\\\\\/20oXBSbgf8gIk5QJ763FqdsdIkTS4viUM1Wu0qQrStWH9UkDdJWzRmzmrRqBR8aVTC5SQ02vAzQnoj61524Da1I8csDin7oE2qBVTsdnrlqTCpjNIXUC2qrU2aEEPvAVXWzH4+Paxns5rUK7bI5SYhXxrkYHxWSGqZuoRD4pQdqTFJVwJNALIGodjnJO7sx\\\\\\\/VR2yO3E7pkNYwtY4AXjQn76EKFL8iHCN5GPD7D8KL3cOHsRcfuxHTTLz\\\\\\\/cE8UkujCDk3XN9un9gCwDFcC9gfSxNGdQ10VFFAG+r+jj4AGcnLbdd124Xro9wTWjCjHPnCR0q4joSeqrEnCpoEKmF7NYhXoyE6EpQcVY2pVTXs\\\\\\\/aFmw\\\\\\\/wa5BWfhubS\\\\\\\/x+DpZtWNKA49wxMmhUTtRKYbYHMCKLgS5E1p0RwvwL8t8ce2w4q0VJs43WznEPDKNZEKQmdjUCPeWVmGBWrGJqkBvGrcTyjqQJbLFgAB5TlMvuWM4QGt9bOxue9FUd5WXU0OWGHP4R0NBae58OPdldXYjuK1JKYcFdLwogNw3naAy4AKIXs\\\\\\\/VpDoCge7gq9RoRJcCBP4ZjZC69su63g9R4X9bBC1KZpUKqz\\\\\\\/guXlsxMkAijqUqNRYhH7ujAavhyBHXkv+vKut3oiCKoppw6arB88XrdWwVoPxJ\\\\\\\/Srwx4GpXu4g3Aho8QW44cKqbAsiWLT16QTBjSDr50F\\\\\\\/NbMB9FZcwrnG5XAXYuNlFOiQOKb7NYwJz93MBMl106xvRFKSSgPVSRKOwoxylayqIrnNS7vrJC2O\\\\\\\/mZuiH+vDPi19cBBG3wM0hLKpDRQGGCIRR0m8KElq9lHGMdnc6UPqPIv\\\\\\\/MH3PAJprs4kYcB\\\\\\\/8rg8\\\\\\\/IyjUQefU5ULozoaWn9jMh9uZKT60wGGGj5Xc3DG1fH7NlZXy5UpFBakQ2q\\\\\\\/owqNur6SJIMzrmK7sQixHWUXGnXxXBfXEXVrTS3gg7UAhwCM79yCK4LWnWYJHegJdUXtR9Evf8o0+CbOTE4mBFhI9OICaXo2E7Fz6z7jzSSW0gGjSEIrNZZ+h4VoSLAP0XRYtxqHAq4FbUJdQWrKSDt+EMENIQ\\\\\\\/Sjh2iaZ6eDjbNYHzvIiRMSsAGI1DKtdTXXvBUMydl4O02kOs+YXQo5bs0eHXu6lzKhfzw8GtCgRiZokPjlvDq37vG2aXwIKzp\\\\\\\/m4dxMzTeByv\\\\\\\/Y4iDLxmY5JGIx5vzZvQgkVKKMgDPTpnEBiGZYTNcasNZ6D4+NZhzBRDZAk0wgGPYS0KJqKVyYklqD3Iv0RzgD8+W2b40+rajOWZr1jiHzJbnN73dra3n87nbdYMHZOiktEh\\\\\\\/we6JhD726ZzPxdeo2kiwgDM\\\\\\\/A7KMi1RwmaONW4uJHMOH3NZflcFLyqfKAeb9L\\\\\\\/lGKnhouf+p3pYEAUd7GHCkNDj85\\\\\\\/8mHrBkLZ84jAaXq4d6sVNjkaYjk8xHAFLsZz5xd0bgOIKwFlNiogYE73L048o3JW+DI4FX1qQrSxG9A7oMVYsujftpfqpafHmLMGiFiKLk+ZI6l1QrQoWKX2sMvuuzNw0ak+Oc97lnRUdfMyTnZUCVndxWduge0QBMrhsjRmTL3ecupMHQ1RjRLIv38zvmXelhgwGrChLaySjfQLrkntC3Ci7BCM3fxi4qeWLbGUSE8D5bMS3yWCpzD24y50Pa+didE+5+oy85d9jihU+r\\\\\\\/3x5+ot12WbDEz7nyjD4VqiSbZSffnOTGo31llQI9DlU8Pjj80vmdGJubanarXhL9QpQ4OCON5f5sqhe00G7KIBK56HTH7Keh+PEcuD+jEIj\\\\\\\/\\\\\\\/LAovQAbuu+aBygAdsLv6DllsSDt45l\\\\\\\/XdI191M53y\\\\\\\/68Pv0QphDtWvTCKFZ8k7qcEWGcDigbxM+deZb5W4JF2FSlOGlLfBoBK79wgygu7o08hHA5VXrwCrgRuBcuZmUubx0Biq9a7SOXdF4zmcOQWXXYiIKZGf5S5A\\\\\\\/k\\\\\\\/Ekm3TD0AzAx\\\\\\\/FNQLQGWJzyYc+nwnGzwFCsKsN1xH21Nqe\\\\\\\/D1zOGT+2GKNW3z3HArmdUZm2jqW8TqUp5XS7nqbp\\\\\\\/9pKFR\\\\\\\/TwcnXG5v9L6uCf609sg55cUJt6ELWhinr1rNzxRQAz8HyxkAp5VfKxLiZ7My0rxMw1BqxVupID2pAz83224qU1l9unBuGfnduvP00PCW1P+HpYhxnJ7sXCmc7x92cLsSyz2pO36V2zkcgIq0UbG7W8uvFIlH\\\\\\\/dV3zyvApXbucokhJuTfDxVo\\\\\\\/g8+HsdU1DCgQyj+02pRI5tOLZaOgsfeEnoHtVBGqo6pTaMQlXuf81cPd90JdBn+0DcyC6eW3BcmTq7JdLUncjl2m3DPHpJUCFVG3efSbDqg8sFfIjyvUq8WemjrkQDNLXrTS70lED\\\\\\\/0jyeouveSVsK1oZDsBWmMgvGi5+BsVBh389cmYLMwdUBLFs1Cr38ABzfdTcye9wSuBnIV16i31UWT+7UO6jeRERjJ7OKCRRUdoQGWEW\\\\\\\/doDv6TwxonkoI\\\\\\\/I4BGUQDo8gDojMqwQZ90TP4\\\\\\\/iUHmnUsV8Ooja0XHxrsztyMEg0jV7K83r1LfjtmXUcgqcfWncVmaB\\\\\\\/Tlrojht0JPkUpfl4x2OKSYxOgAW0E5BqnDOhr8MM9hKObdgMTarMQiFxtSEdaC99lE\\\\\\\/7+FSe0o9OJFkTIMa378n8XfwkSijMAe8Yb9P2jVtU1lhn3slW2fQuekkQVcxKfOR9wUlqXmeYj\\\\\\\/koBgCtqnsvV9h8WzWkQbinZkY8uJ7Vq1ypcGzdlvDPZTj2RElcaw+I8RXndSCxljXR2gsEMuQoHS66daHYPt+MOHCnNTysC5jZPTzJCTSszCRXMF67hqRAW4QRwCE9sfJGG+iWGC+yAjGPdIxMc7Pk+gCSad9G7jWaFUD\\\\\\\/GULKu8GUm96viPvXOPfNzCZ0rdWPm1k\\\\\\\/JQh9GAPZHgGIpf1V57i+B6LMEZE2CvxJsN1hPGwG\\\\\\\/JanDsvCO6+nCGeDTVC5MdgZSPJDfO1\\\\\\\/J30KngLCkChgEcwJDaFDf9upT+TTGwABe2WgrfWq+vBC6MaAcuXxrqiPGnVNhG9JNlvEASYzW5eC8NMejv95h\\\\\\\/7vch2eImTZsxs2zpPH+wGTocoIo25\\\\\\\/VkxLOaHfk8vWFUlj6PnSeIIWfG2DqzbTC155\\\\\\\/LBj16qvd\\\\\\\/kQ9+vQhBp8InNQrUoW0uz0Dxp6ztD084+3CsJ\\\\\\\/y+qlHwFiaf0mXEKjjffVR1uLrCXfats3I7Nk8u\\\\\\\/f7YUSNVjz6W58ob+KsmIQu75jBa3FxuDiF8fMEBwUb3WnIjI1Co5efGTwPimlbHT7cGIjrctLv\\\\\\\/friLf0Fwa0u465DpxZ8YNZDuPwM96\\\\\\\/++g8u1+q\\\\\\\/V+TaV9lFQulB80xn7As0V4L6lJEWN7pSoJ8KzfUMk0LlJK5lVXb7joYQZqzJngANeF6o5GW9xRC65rAQrJxT44ohZNF0HP7\\\\\\\/it9p8KyfjGD+GcJrTG8ZZsuhY52lk3vmu6Dvb8AHaYtJwtGsSyz0lrK+6jtKzEVzWG8MKPIrt78QQC2deHCoxFLaGoI49Nx8lk7\\\\\\\/39NU+GaYbBr7Np9wKQlzYDIBUGAyKCkCZ00sHV5GkMOXtBH61hhWWQdgX7E16G\\\\\\\/ogxnS06TJeTcnWuAwaXmMM\\\\\\\/roNhhAnzlS6BRqgR0q2MURjhZ8ZWJYQ70U\\\\\\\/ziWw0zCeQS963hkydMA4uyif+C5x5D\\\\\\\/uSDyAp\\\\\\\/NKAhvQN8+VzoGrRCamS+6P\\\\\\\/eAp4tlmSMtGKUa5RWSoRHZ452S9EftxP5K2lQ+uUgz8nsRo3rpFKE6SeICJcBm0adQepRT\\\\\\\/rGnNhR6EmlUDMDjPla+Rst5byoEIh61yTMxB1UBEHw7BRNalrX4utm6fbL0LuEyOmr0gUK24xzP4pfI9HwpMrdnbG07QodAyHuFOP5xBVdJpMKItn7ACEC6g7bjOXq6+Xo37D34b+\\\\\\\/TRbb7QOsFSFXtyf5rNwt0FVZyijy9MVfgpoUa04IT7oJy0rfErBWZSFp88Oa7lW7xYFattlvXk3yYWp3yIDoDlHnBpPToXgz4XFbNOpxuglpW96VdvxHK4IwFfrN+MYSlaEfdFI40Y5JEp\\\\\\\/RtWBk2Ec8Lcyo45B8jOuwA4eiz65COjDAjCGGKU7KBxUpxTf159+wNbtYFPrQjfYd4zHcm\\\\\\\/8E4451CvdRgKeYR254HUVn1W22hG12Jzdz2D5kmMO3U4Gf6iG8OhPbddXwrXjo8i3mUAjajTP5an5s4jLaQtnd\\\\\\\/LfPZv74eclDXo6gt9FI9uXX10dyN9szRd1UufS2TkhM+sQ8i6zNlDlxCabxBz2Y\\\\\\\/PMIAdH2FZCq4ox3CExcQW6hbJOnh+WaCJoqKgeKNQmU4ho8omqRX+caKaxVwrZUAMfCWOh+gM3DlJMfxPME4a++i6tdQpCMzrBfNCby56COateL0elUTFCHWUNk4W1G98TrhHFpXrEgvav7CVpE3u00BJNLneCjxnfz5oSs1E93jS\\\\\\\/YPfuBXOfvookW63DQHRUI3XII0gnMcBFEw8xiLzeIRU6fexS0pL1svs\\\\\\\/7lmoLpb2HQhgXJw4IEI4AaciGr9lAg+W\\\\\\\/v96f23t1pTJbcq12zUJpOlvoH\\\\\\\/gvk9pcRNpgu0orX6lQAbU3q4eVkFSZdOvjR8\\\\\\\/i5zk5n1IozdagWtUO\\\\\\\/LRG4JzwGzPYq0fCJVl\\\\\\\/ylblekNjmdcnt\\\\\\\/XcA\\\\\\\/JfKBbhfEL1cktHoRQIwyGyrQBaMdoBotNwMbIUFTEdHcbz4MdmHzwdknFfQBt0xn42SjqFk2cx3D+7r5OOzYWsgsLh5ZLS33FQUwLpyim4oHo7tZz2\\\\\\\/gYB1Gkt6g4DWyetqZuAQxjZnqP8NU5Q9RF5WJV2jlornUvWHd5v9ydsEwGXHnmyWKNZuQHb9GJYopo7P6GYKlcOhKBEKpACkC6v2s08tCyWLB2t\\\\\\\/UKGhTHHGS7Yo+Yr7FYdxfdhUzmElyWYZ2iErGfyBdwVFY2nd9eHOHmNn268az+XfdgjS7wQIS6LXfjNMI26MktVOLO5D7iNfc0pBU2cRxfBexAMvL4oloNAY4U4gsQPMBmYiUglRl+hCSwNrvXFz5IrvKg95ToA95tDX0B0EvBm7yNiqyhuwsPaPU9jvYl01cSCTnIVUzCCCLafnXLQ\\\\\\\/YdzoWpgjEscVQzE+7YfZkH8xYXpZvJNFd2CbDafskOvJnEvr4HBRLZ20lRCyLRjgH4ijuHCYZYT4u7+rrtCUHbtpPW0T\\\\\\\/QUQ6GhWmeaqN58ZlmT8KkGW5v+J5Ts9tBCXCUiYGHVmqJomKpJZkm59Sg5mEay0E+t5IdhzuxCzPz9\\\\\\\/KH8BDzSJfWHV6ftFMERNZIhDuWxFvDO1zSvfj2V6SCYRhWgkJnG3xTG2iQRXDaoGI3iuwOsw7kKLg6JXLY+wMkYkhbW4aKWVr\\\\\\\/YgCZr3Tnpjc0\\\\\\\/9YjXy2uHCIBEumCkdjGfKdFqJcWTuWr0u6BgMoPJElRCHYse1AIoPt6DbyIFVbrMfBzL3w8IOYY0x2F1nnr6rIkt\\\\\\\/VnTkd36MuoZIiGgqtUM2C\\\\\\\/6bgmc2lTto3RTNP6ofsv4fuMITsRyZQyGVhLNBmSJ6gep8hW3QcI5gHWY2mi9tLd3vxZCJOa1By0PoW9UrT5d\\\\\\\/oFpWuxq+tJScUi0RA7cLROUj41SsuCnsglMigxVqi2dM6fGOeWm2RSE5f9ybESqxqlF8KZhFTN7QHrPQHz6d+6mn9\\\\\\\/S4eOiCFvg0iYPSVFITPll6XJiBnvVetMVOhhCmgQ\\\\\\\/nhzaQ6vSCWJWlwKMpISUFDuCNQR0EmfJxwWvrTP1lf7hcfr6ocOfOiyv9MYzehcCTB2d4bicbPPF3uxyYGZA17MS4uc8Tc+KZ2B0t5zfqNEOTJoTTCGJnzXXEpR5pE3wm6NUrKEna5sHidEoU66EHFXKYUT3VbKW2mo\\\\\\\/J\\\\\\\/r2FHPTzpJJYkez8AtD5RDh346GOBmkhIETUdEyvWiyzhw6cwn8nbR9Y\\\\\\\/j7HupDu3FlBQPdKrW4VHt8LO0TA9UAz53FR2nxQCImySmJyRpjXDpNS79uRJ5exeEnc0KP1ZUjxEWY1GsFDFEey5GRBakJRgcZstRitjzdjcj40uBOqNJh5wwkUPAVjtCeq+9t5TJXqJ8iwjqsb6YQy5c92DcUqC3YPRi8JqnzPjF1\\\\\\\/97Gmi8RTW+xrsah12qLPbr+V8KUlYWyvkre3dQOY3rRLTdO6nI9CfiAFO0bamgU41NvJSJBkSq9Ye433KId9Nz9DJ6NYqZM2eFsV5NHyZyiXn3ltrSMCLcKj3+zFZm0qmcljIdi8UuXk2aYz4QZPByEXVwh5SxniYt0tBH3ZXEgm0CholDxOGsfeQmygYIZwe+SNosg5OFlotRYr4aJoDdZFwnmK58ytZm0gkTgWBPsoOa5\\\\\\\/9zg4OEvUfbvwp9uWKFHH7tMSNkGSkmyNmgGoeymeLGYMPyHdjrkOu7K6PW7i53TBqYTLaNsdYptP4\\\\\\\/6BxfUB7hS1WRRSoGWBeRN+Jv8Mfu7o8tmwzveaSq7RVkSh\\\\\\\/jQ8E2VXp5jIiNzcN3N7VYJDaq30Ntv45ILrATpQm7tULZJBzakhw6fTNEaB7jN7sPlHzzKr76FiVbRAuTCPXVfCzSP+sVjLhbCXjbToeJJ\\\\\\\/KTxV3YRnlrhLxhBfa9VmLpEy3V4GohDztyf1cSELRxMwwZO6uvMPFQoxtWkZAQ6osW\\\\\\\/QKyIfUhEM4sk5Yc2bg52GJEbWRy9xlViuPhC\\\\\\\/PCHwNEBM19T+PWz97Zu12Ya9Nu9YcbX+FDvpdXHWpUzugf0JmrkIvKP29EoOn2GCwMsgqbfIhHe6rFtos6Fg07RQU7K42h72shi+qysh0xewKJx3iA8K5ontYDMKWeDfLnx2ucIT327bnK4bUpch3znJLccwhEZNVDiF3FS9hIM\\\\\\\/BbSXwhyAec7L+mJHaQhoLawV3HX+DUyCY\\\\\\\/C8dxpZndwMStZgmIN34aIXmMq8ACcUIz2jvLTa9ZeL3DivZq2SJ\\\\\\\/qdfq49nxjQ9Ccqy2mKmrJbZcXJPrM4AfjqbX+k8spGy5iqPU3WVXJvcusdzpIXR87QwZPp0usmqc11viP9xVrmLT8lGRpFP0rOhvq9eUPBchjzcl7+4FGGetsKrXNG9G3MMLLPK\\\\\\\/L7JuE51WpShdZWgE4U3GPtvTZKDDfR5imQAseDa8Ky1hJM+ipMpuc3dFxB8x8nMwNvqYCnTh3YdSTC5LfSn+Cz2SQm9a5uze1M2SkAD7rUBrMtGcWYtuwZNt+vW1uTHIxVruJknzYvchqhZU5tAstjw20VLtB2ZTL0HqZywjeOJrJVoLmENdOadAExgExtkTYe2HnGlaHA0lbzYCpSZ2YjZkLVpmByx3NOqRldKWeWUAUHtkSQSzNE13gpdIpss5zBDmzn\\\\\\\/ZI8hS6v0Pkd+9+OT5pEfYewIXkIyXIiNrs0MwsLhI26BOjyETi1dF\\\\\\\/WeQQbkCxO\\\\\\\/EE54QyqRgrlw+jid5XbriFY1zyb5nzS7CZHIPbX66+2P3Hfx+UkOwc1BSBBxMtHdYf4yfkX1L\\\\\\\/HMGnRxrLFVzkG0iNyEjJsf4HHdFO0rneVvDDe4PxwXI1H\\\\\\\/IsS6tKwjeTfoTiPXyG9ckMVeV5p9OPStQmu+1R333dNPmuJX1HmiPvteM+d7LjPL6TNiM1VPtSmBdie+Oy\\\\\\\/SvWttmn1vQ\\\\\\\/vKD+85mczzarPbVfddVIjLGy6ctCwS9GeTOs8WnT3CJ8NgSsB46CldxVFFdQQujrDtuIidLwW8OIfzY+nTt6o9nLweZQs9lvu2YV4i6xX46Tyk7mpTazH9WHQCvtuqr1Dmore2FA8DCqvaZ3rt2DRMopIzu9XGIg0d\\\\\\\/VSzIxT5E\\\\\\\/E7bw0uzC4QPbHtcORgA8y4BDxomTyjqg5gAWBwWF4SQ8NE+v0nEFJvwEEFMXKNpNvBeOueMkTmYV5ffa6X8qOqdgTB9iwcd7pvHKgRY1sMCVdEeo8pwh2ss+Th6aQBjidEUkhkfSHFxlil\\\\\\\/bpV5GZEg6Cz514VrwK1V3TrxJCGNLtxRP7RgVHtz\\\\\\\/7qTm7PVfDMq+7kGIpwWPuIFdHD0Q11\\\\\\\/hCr79DlR9n0qACgj\\\\\\\/yotvK2PM8LWLgKDzpuXvl4b5lL9RT+qLGSIDPiX4w36KS5LU1ktZf96cSMWEWMRrtbcW7VLua4e3apDdFZUkb9McWqx5Jt10H9xenxYLUr8enZWE2zkKFfxpWWZOx9rmoBJQrRs9rmZoW3PN9iNh1Sfgh575tTnUeClG1KsROZ2VVezDq\\\\\\\/mKgXFKgKoPLBtPz5AJo4OunqJnxDjd6vxUT1cp7IQNZtcxijCsYsdA2nmN6lnxBqx9LHEFbaqB79Qwlxc9oCCURkEf3daxtBvCyqGuM3eioLkti8ZPMYmMQT71mrTUDH9dRNtjIVCmWdtObFSU7tgd7bisa5xPNIbA9DbSIbBWAKt+KYQ9QPmC95iFr\\\\\\\/uAWnbXydxxIKgyae1E\\\\\\\/qKDDLl41ETvcyH0P4O3YypJD93asUSPaq8rifPPWL3\\\\\\\/GYV2fp7KuneZQatO6pObYySMk0NflodznvDCbYksxC\\\\\\\/j3tSqLrCf+4VHnx+Wwhyo38XEG0e+Hrp0i2cXCSdjNdqYHwnm8GkV8aBmon6Udm\\\\\\\/XfkjzYajMvLXuTB0QzxL0s89IJPIRRZqbuIyuWGGheWBmiITuC+Xr0PnhuoOyHV3bMyA7Zq+GFPI+9RQXmMYCMtBr1MpGX8vOdFKmLvGkGv+HYfDrv4F8o9CJ0UfXxihkorGYovfz6heTvKbAPIK\\\\\\\/9OiN\\\\\\\/X0FyGUMTBmtO5BwvV6IdHTrZyxK14kwfUvAkGHZq3\\\\\\\/HYcUuPsFXi5Tre5VOAbbhfsuKP6z6GfWZS5CI74aEohJo3ZA0dc17rQYF9zpDLOSyUp5gaZa1v7DX0nqUKpwV0vbESPU6N6mjK+R24jMRsckXL5sGweBMQXEDshUonwtY4S5p6MVU4ylEtCtPPbQbo2AU9TSz3pGGJsz\\\\\\\/GpzngMcxA3HPqzmNy0GJxAR8K3b47HlW69q\\\\\\\/6tJLtA\\\\\\\/bHiQTu0jiboZTROFUjAoU2tpRP9SK0mB3k03hkoBBsN401T1jMRN61op4pQBMwRnvJYf7+pKRTmhKF11XnFk4jfsMc2D+IRRGG2waV0UlcRAtxbe0OLiCJGsFdrCJ7\\\\\\\/i6pFSaL3Fdw5\\\\\\\/sdRw5MBUkmPmbDRZOoV5sMugWfaDu5\\\\\\\/4MtqoKj7h69Ljtqpe96YPrUTcY9h7RNjI41Mr4mxQHrY4BKHvOrQDZGMfukAQxva5fF8XOt7h6qCGyCjIUC0dLFmOmuHK1vyqcPf127x1fQe31wY5QaDPwqYNjVFiZEFNlaVRHIjNEViwSax0qrYjQRKNJUNyHJpmULPUIY6ewRM0Ky9l1qqrzq4Wy3ZTbzhdqDUTeR96zSz1ezbYPQRzCnefjjRF0M0PPtx0EGMKM\\\\\\\/OfgcaDwW7F\\\\\\\/cxaR+ABwbbpy6lcfV7BMeUow4BT6QnSIKAx6HrRq1yEAx6kcKA6j4K2NqQ2mGdxd2e9h6OXZplGupSEsKboPjCEKXtoHfEBgQTW5YMkCrSyl54dkyZNsRL\\\\\\\/uf+ztNyVOuXchrR\\\\\\\/6FTximltgz1KmJi3cU5VzGKqNx2xXh5A2nxdy+c7l1Cmc53jp8trFYpbygfxfF+sgEFQM\\\\\\\/sfmGAWW\\\\\\\/xhLgnIF0luO8p7pDDCKLwIbejytQZE1ELA3jgk8gFRmHTDaKKz5h7FEPQ6n8oh+LH7DqeT68zotNro13sMpcWQZxbELiyOopapD7KE19UJgFkipSQkRb\\\\\\\/EKSHorl9qh0tOaOWzp6iX0H\\\\\\\/rVzGlej1O9y51FUo55T3h\\\\\\\/RFgXKOrfLv8gWSWxCl9Ykxbwk1CzPmrn0Lu\\\\\\\/GGmGafphJbplmlfD7b7l4kTF2D6y\\\\\\\/ePThjn2CB68LW\\\\\\\/+7NixgjXaRwVkL94HAvp\\\\\\\/MHFqUF4BUXzvKK63+r9PhZVSlY9hqkTSHP1FJIz7ZxLHhX4d0TRdMPjwNORgOOupGcN3p9p5GVCYZPZtA0kk17vy7fh39xfbwi3PFqdf6Tx5TtyUAMbVEUsjnpYn9YUWCGeYVo7T+q+8gN\\\\\\\/Bs6ZWMBDYN9d3s12YIVaDzPmxJ7QxFvN+xoIEHRR8RBKgLDPyyfcp4NGO\\\\\\\/vaYFtlNkhWwpBCCZF6OxxyTLRoKw1E2FPZVLjovORfnP49\\\\\\\/QY0gIS3YRFQpUKUCzb84qqBJ\\\\\\\/7sZ5+OC5cBq5m+3QSkYs87bYBqEy84fAN84qw60ElWXfjuGaTynMCuqdcUEANoeDnCOdc7m+FebH3lurU\\\\\\\/W6X4nLJ+\\\\\\\/i5Pd4F4OxCZo2wJrBllG6UNpf5SeTU\\\\\\\/exXLZEab+kEmqUiSjBwv7shlrFNuuyjIoRLTw3RP64RCtkKWribHiRoZNzO3+bqifcdwN3osFF18nFj+j2wwNwY04UyKKwVwlVVzh+r8FHWuqpWydemPUWLbz7W6cX+yFbnXV2XXeOk2wJc\\\\\\\/Q5170AFswvx6bxAYifiXOALzCl3V+tFyN9SUyfiGGHJwbHTQqyaVXj0wfXOIdKROa9KHPT3r56JfmrQfhnea4JwYM9XgnC62FwEGKh6LlEiaBGW7ufWzQsx9xxJ1z9+nCk1dkiJZ5BfXcpgQodz3tetymrgm0EavAyp4rT3AqHpdWH6vQenTmXr\\\\\\\/zu1pQfBzTMrr86pQVcY1JaBGXureEK\\\\\\\/qz2o3EES2XvDSPQIXny7GEpr0VLgOWyZ36tPWlZ1Qe7umQeozk3iuGRMAVrKqy+\\\\\\\/3skC5VQmk\\\\\\\/POIFU6YOpVmqHS3JRrHygHwHGPJ0HYf6EvMrIZB9iWW2N0+HmXKUKxOxFGtgjabLLruJgQv8bfplGv+Ix1127AS9NAT1cfydZxY1YksRzY+zkjKD3lnev7w0cpkEDcEPwou2r+oGaNwOLAW5Suzp0YTMuRDD5gq5zoB910TQQeNxSswvUJi8cYHLky9bMa7UfQTVTNpGH3NF62vZK49KCeFQVQec+v+tmhS2bqCXV0a1fJO+JQwXVPbpvsDpAOgc0IlhVHqGjfgcXyQpHUqTruES\\\\\\\/mAQgaNqLXXsaN6GIZm5WmDpNFN3ztD5uPgaqJZPdk3QEYDSwFuRaD+cbTPeWRavWoBGzh98+OuAWl45WMQ7Tlg1S+0uiwDTDeJNynwzmmpvALDkz2jHf0RLgLkbLGs0iMEygghK9g\\\\\\\/Axcl0ymAoR8M0Mf7lFTlET55B0W+GGoe+9IR+EtkX0FB\\\\\\\/pX5dAjOqGKzLqtlrw6EyttS8INbvz1JZl26+0+U6BFaNB2TbmLeWczq98yY1se3ff1co7acf0cIishhAjdVhe3HhP67W\\\\\\\/BZgoljOUaVfv2mU8IroHxr63orkEMuf88bQW85MieY3mGjm+AGM0qrbLPe0EwjqJrwsDbEQ9AvyWpc+LkvCrh0sby2nScMhYqOm2q7XIYIgzL+RGbIYgoqoZpmHcuXWtzqtL8RYT6+OQIGZobuJTgXwFxhDbjSCv8TwzIm9fTc8DIRK8yvoh3zVCp5fAdnlB6PhpsKIDaLSKyZ2reRUJ2tuTJHC6iSjnQ3044MVuShj6MaUctmMITc8QYFuU5zD2Tb9csePD9yYQ93l2+1Ew\\\\\\\/5WGZsUjN8eHqf1p0zm3dUUIww3a1yIcTyjKaWRVvPIbEsW7POeZ4amYGT1MVcp85blBErxA\\\\\\\/Lm\\\\\\\/hWMaRZ9YAVmpH6LiTi2bbwTTqCVmUtGpvMxXErmUCbyayaKfEaDh4pZ1Qsb1o38ld\\\\\\\/Z7wzw\\\\\\\/GwTe7lIs1SergdywN+tGlNNBGCB4h\\\\\\\/vN84mjXGYvrbct\\\\\\\/h\\\\\\\/9Qhsa7mOKNE4ciAw9yOKxYnrzI+wGa6ydU0Fu3khLo\\\\\\\/7YRsPpnXcISluMD0wqBc08S6iBv4gbwQQvQD\\\\\\\/G\\\\\\\/UtV82G9tPVpQ4SXzM\\\\\\\/t7rTFubnw+eiWrxEspXJ+A+2XUyJ5jSGN0ZdYS0A9al7d+OwWJ8Wxry596mWtPresCZBVm5p3vRKL215Jf8wTCPsUGI0dR9j\\\\\\\/MwGs6Sqyb7yLELTQJdvFHdD2nt98NVC5F7KCyIh9dYIadrD1fFZyNp0XymwoWScGuQGlx5JrzbCYfHTHrUqVROe66kSyXs2ehIi7Hb+7nyfoVjshBi8IBwADkG5690uqvtYeurk\\\\\\\/B2DQkdEEM1vwv0xGYABHxjlUbOEsIFdFq3ovJoQycU9dA\\\\\\\/XY\\\\\\\/72pleKeRmEfxQmk8vhn0Fk8jIhzBasgG0Sz9OceZE9hHkyBlSd98xwXgn3fIYw17FTJhKIdxtzvNNpEillVyu7P7RY6mFej9UCqO9t2ssQ7qyU0sFgAs\\\\\\\/2Ac7l7khe1JwQkoHAZ3yXYla2x3Pj6NNONwMDjXYCrRyLzrHBqlSsKol3xmTL3PZOAXI0E+H6rxZAP3W3RvZhN4xwdzQAYdjeqTKxcThZzjPZA6Umon2wPflRLqtIRhLL18VM9rkZoChFO7\\\\\\\/pXZGVtT\\\\\\\/OgWL8F9FxREPBNNrOJNRlQcI96NvdWaDdO9PIlWuYKOZ\\\\\\\/AP3g6qS2i6plg3z7DtewLe4JiDkSukncTRjzOQfET1XO0ii\\\\\\\/yAVB3ziAX89G6+4Hy34nJvNXL0I\\\\\\\/3RRA16gaizIzTOMJ2bZcM2FXgV7HqBCEq5s7C8ipushblUlDbPYHArmdVUsS3hGrTVzl8z0lRS2SxXCPgJmrjYeeZiOykNsJI\\\\\\\/BfUpZwrH9DrAbiH3c+WPGVkWZfPy67I5ck7KaFjee69rwY6vYOypopN+tdNVK6prts\\\\\\\/13MmDeFRaeSbNC3Ha2WUme7wbeSnoZq6hGWoynHuYLH4I1pI\\\\\\\/+yJ0flSTpCqrd3CYtSWXmUzbR\\\\\\\/8kvz+6vZHruWkDnRP572q7UCYCAa22QUdQshVvmAwrOISdwOvy33xZMRFFuXdA0jfWdlEt92iqjID2C9I2boUDKEJbhoRDSzDvXgdp6KFfyNARTeOOSEvg9aS1zu+6oBKKmqOumVloWteXiRkK5AY\\\\\\\/lOk8ou2gRD0S1GjUgsxjnukGYNOdUMvgHmxXNZQ4mV8xv0mdBhZ+jC\\\\\\\/5ZcUeTPSNjtN8U1xNCJLxstFIpozOgqAXEXn8dHlaVGkfujhN280HhAqBsoeKWOgEHPqEoPggm33eB8BL27qLNnJS2ZXbAX56IKJMJiNLqEIqBCJLCV4DpIU34prQE9AoywJxoOcNMBqrFyU\\\\\\\/qVSgSiOY4jbfs2Ooka7mvHLMZh3Do5akS0VWJmLHKVIq1AxycW8HNtFmhagABBGZrip9zBhP3h3ko9\\\\\\\/M7vK4tgUyIUz8PCQdEilkYRLcFBslzi\\\\\\\/p4EZ1jlmnGfXOwAl6Twrv\\\\\\\/M1Ud1pm\\\\\\\/iZG2ptCF\\\\\\\/+a\\\\\\\/3BPT1fuBnJD9aUK8mnk7BzQr09LNPum7mgyHmNzEuayHSQmMm7HBn1EUpH5KpacPPsf+h1h\\\\\\\/bkkUkFACWpPOd8vBxocj0qCfA\\\\\\\/Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EYixwAeNIs9DOQEIO4PqPQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221d5d6e254ffc346510596db43089e2d4b6d3454229bdcebb7f05dc2d053d3326\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022org+ygM3Bax0Ow\\\\\\\/7uSGKcdrVZIA5wPx3X5MEEmse99P1GLVVpvLT+tR7diQcqpTY71zzhvbfCjG1C8EBVxmkHw0cuXIl0PLDt711FLaNx\\\\\\\/ULsh7cME4Jx\\\\\\\/iSIeNOR9FEpB9ewifQabYIgzlwcd79mB0DwsWSCVEn5on3KLaOaMHo0TR\\\\\\\/YyDB4y6lG+QkEnPN+Ut64JZ\\\\\\\/0OXkVe9DvG1hNi8+cctnBYAE6ybUNbsuWmCOG7jFeVPHNZC1SxBJF0NgZZc+j+gtWG7x68vLoBwpWsfs2DdX2ucqq8Dun4S33E8PE23dh9dXy\\\\\\\/+HVKMjg0tslM4v\\\\\\\/Be9+BXnQVL5a1tJo4\\\\\\\/mvIJcODSBPsqYuLF9MR9NNXmni2xhE4FqWWcI48kD20\\\\\\\/FLAa9Qz9Xs1OK74k0LL25G6sQXLY\\\\\\\/zjZ2fA3flwVv6iVVx6DeOHZWL8vvb1uHOgtpuJSoTakYVW+Sq9CHJ9\\\\\\\/l5BhTxaHs25kSEYVKePpMW9LsSMTlOD7SG6wO5kxuxq1umy6Tzdi6+RJAy8xttmEBLi2AHaXAT96nBHMzsm8D\\\\\\\/uGE8Nus3Sp6uWjtMIZWoeH+PEQ\\\\\\\/dRNvq563bR4fGePU+Hn47qjtF46YPIuSV\\\\\\\/7w9Is2\\\\\\\/qsdisRtuWs0LXdotcsoavGb0TjjLqDXOAL6UCeYKqYCKk1euB5vshZ3O+BIHB00rzZcIAkcNsD7e9vbb9RtqifpERVuoybcm6lCwoK1WervsSP4xrx2rXhFT6N27BRIu\\\\\\\/BWC9Ry+pk1RYah1Emt6OtN2ytO\\\\\\\/Vx21nD19Wf26kG7ZKXZzoj3Wnu\\\\\\\/bddeZEdjmcK7YYqVmGcqTj5YEapdfYls\\\\\\\/hYRGbEvMofGzhkLt9fZw0rmlDa42E+V\\\\\\\/mZP4yQEJU3dCjndWlxc5Arvn1mjQlC1vhpqZQrSu7poYAouwwvjiMqxLXSPCQkF1LGBJtWNF7+KFQpSTMWXGMNdEM8E3wqYLOq7GXyaof5vlBtQEN5F5MzLRz4dildzY6ixBJF0wG0SoKQ4v3wpNshzJ5vo+fjkszEk9n6qvqAQjbiGxLub5j4yV+ZCYQkM32MenwUlumorck3sk9\\\\\\\/GdwoVYTRkH3QlFOB2abjC8g+NHEcY3+rE+fmGZYwECWqpBgaFLbgYNMQaCJNzEaGT0SnIKjRXePqQM9\\\\\\\/g5aShZcxvgHmKrm9q1WKQBpjFW8TXmtxpEOBQgLfhKSH2OLZfOZcMRFpUJivKYTL5bYcNm5\\\\\\\/P\\\\\\\/oNlUH+SKKpE7XsCxGeibLqcXvR7BnboHIcQvNJKB9egIVmVMlV9dkWTpnZ8Trxp8NprNxhfYZ4ACd8+hTyMo2zD93bc\\\\\\\/qVyceoNApHpE7Owp8C3TPUkuc0+O+T0jykp+madqzTafrchzE6pLA7pTnhbOhl+oS3AZGMsMFSM+nQJZ0wMOqd4cqCMT+uJRCxaXHhqlPN5SvbcVLRqSDeC7B\\\\\\\/umyn7k\\\\\\\/iahCPQmGP2ahEb7FMFkpE8n0OTZQQ6cQsWRymE8EC0Wiq7NLW\\\\\\\/Z8e8qVwIZ\\\\\\\/3JsEJxEfhXTTiaZ1VClQn1R1X6Nto0ymZmf+7+Agq+4TrPSjeWYem7vR1VodRV5v\\\\\\\/yYcC9BhL1b2vOCP6w5gZc2EgV\\\\\\\/YOU\\\\\\\/TgOu8qIskPKbuzbC+SiGOorIOeJ3IxueW8F3ODtYpmYoWL\\\\\\\/O\\\\\\\/DiEpWwGuFH1fXN15zp2b5cFRCG7cT+22iMpt43BGjGijuqN1KbSb1yLJaMwb5R6pmIU4l3p44UKgZauUb8s0RsHpCTIDPiddTY4XZ4iGI3VT2MrPCQNe87NpiEl05H8BdqGAymqgsfkNcIo+g57SncWGzfRzbXWDvnB4cov+mEbHNTHySi1PepNAT5EjTJ\\\\\\\/QL7P83RVUROGaSVG9haHLXssfdXvaHdMJ+\\\\\\\/AiZlo\\\\\\\/G0F6zozWTXq1ppmN1FY5DkZAtx+S4BZgaNfhsnjxd6Um50PaVjHHdyr7ZfeAik5zlRjOEppmqJh9NgH2i12p+ITmNN\\\\\\\/dOrHYN4nfAqt5GXhbM9JQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022NSzFxzhCrZy8+jSc0g54yA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022728246127f45c1b7a6f0b9c739e4ce9fc277785bb3201894aa2e976bc363bd26\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485847 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2413" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WHsgAEpUC4sohXxFXRcQGoSwtW2EpFcG7BRNrYknwiw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2413" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aN2zBQvMdhAjXfl4zhZPgaO-p-HECg3x89LBVBjvacY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2407" title="Whole Genome Sequencing of the Braconid Parasitoid Wasp Fopius arisanus, an Important Biocontrol Agent of Pest Tepritid Fruit Flies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2427" title="A Genetic Screen Reveals an Unexpected Role for Yorkie Signaling in JAK/STAT-Dependent Hematopoietic Malignancies in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485847" id="node485847" data-pisa="ggg;7/8/2413" data-pisa-master="ggg;g3.117.042887" data-apath="/ggg/7/8/2413.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genomics of Two Sequential <em>Candida glabrata</em> Clinical Isolates</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9658-5335" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luis Vale-Silva</span>, <span class="highwire-citation-author" data-delta="1">Emmanuel Beaudoing</span>, <span class="highwire-citation-author" data-delta="2">Van Du T. Tran</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5244-4178" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dominique Sanglard</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2413-2426;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042887 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485847" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Luis Vale-Silva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Microbiology, University of Lausanne, CH-1011, Switzerland </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lausanne University Hospital, CH-1011, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BVale-Silva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vale-Silva%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuis%2BVale-Silva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9658-5335" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luis Vale-Silva</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Emmanuel Beaudoing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Integrative Genomics, Lausanne Genomic Technologies Facility, CH-1015, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emmanuel%2BBeaudoing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beaudoing%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmmanuel%2BBeaudoing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Van Du T. Tran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Vital-IT Group, SIB Swiss Institute of Bioinformatics, CH-1015 Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Van%2BDu%2BT.%2BTran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tran%20VD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVan%2BDu%2BT.%2BTran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dominique Sanglard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Microbiology, University of Lausanne, CH-1011, Switzerland </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lausanne University Hospital, CH-1011, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominique%2BSanglard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sanglard%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADominique%2BSanglard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5244-4178" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dominique Sanglard</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Dominique.Sanglard@chuv.ch" class="" data-icon-position="" data-hide-link-title="0">Dominique.Sanglard@chuv.ch</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/8/2413" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485847" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485847/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2413.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485847" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485847/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2413.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485847" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485847/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2413.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485847" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485847/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/8/2413.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485847" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485847/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3"><em>Candida glabrata</em> is an important fungal pathogen which develops rapid antifungal resistance in treated patients. It is known that azole treatments lead to antifungal resistance in this fungal species and that multidrug efflux transporters are involved in this process. Specific mutations in the transcriptional regulator <em>PDR1</em> result in upregulation of the transporters. In addition, we showed that the <em>PDR1</em> mutations can contribute to enhance virulence in animal models. In this study, we were interested to compare genomes of two specific <em>C. glabrata</em>-related isolates, one of which was azole susceptible (DSY562) while the other was azole resistant (DSY565). DSY565 contained a <em>PDR1</em> mutation (L280F) and was isolated after a time-lapse of 50 d of azole therapy. We expected that genome comparisons between both isolates could reveal additional mutations reflecting host adaptation or even additional resistance mechanisms. The PacBio technology used here yielded 14 major contigs (sizes 0.18–1.6 Mb) and mitochondrial genomes from both DSY562 and DSY565 isolates that were highly similar to each other. Comparisons of the clinical genomes with the published CBS138 genome indicated important genome rearrangements, but not between the clinical strains. Among the unique features, several retrotransposons were identified in the genomes of the investigated clinical isolates. DSY562 and DSY565 each contained a large set of adhesin-like genes (101 and 107, respectively), which exceed by far the number of reported adhesins (63) in the CBS138 genome. Comparison between DSY562 and DSY565 yielded 17 nonsynonymous SNPs (among which the was the expected <em>PDR1</em> mutation) as well as small size indels in coding regions (11) but mainly in adhesin-like genes. The genomes contained a DNA mismatch repair allele of <em>MSH2</em> known to be involved in the so-called hyper-mutator phenotype of this yeast species and the number of accumulated mutations between both clinical isolates is consistent with the presence of a <em>MSH2</em> defect. In conclusion, this study is the first to compare genomes of <em>C. glabrata</em> sequential clinical isolates using the PacBio technology as an approach. The genomes of these isolates taken in the same patient at two different time points exhibited limited variations, even if submitted to the host pressure.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/fungal-pathogens" class="hw-term hw-article-keyword hw-article-keyword-fungal-pathogens" rel="nofollow">fungal pathogens</a></li><li class="kwd"><a href="/keyword/genome-comparisons" class="hw-term hw-article-keyword hw-article-keyword-genome-comparisons" rel="nofollow">genome comparisons</a></li><li class="kwd"><a href="/keyword/drug-resistance" class="hw-term hw-article-keyword hw-article-keyword-drug-resistance" rel="nofollow">drug resistance</a></li><li class="kwd"><a href="/keyword/adhesins" class="hw-term hw-article-keyword hw-article-keyword-adhesins" rel="nofollow">adhesins</a></li><li class="kwd"><a href="/keyword/genome-report" class="hw-term hw-article-keyword hw-article-keyword-genome-report" rel="nofollow">Genome Report</a></li></ul><p id="p-4">Infectious diseases caused by fungal pathogens represent a major threat to human health. It is estimated that ∼2 billion people suffer from several forms of fungal diseases worldwide, from which ∼1.5 million deaths will occur every year (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brown <em>et al.</em> 2012</a>). Invasive fungal infections cause on average 50% mortality, with total number of deaths comparable to tuberculosis and malaria (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Brown <em>et al.</em> 2012</a>). The most problematic fungal pathogens include <em>Candida</em>, <em>Aspergillus</em>, and <em>Cryptococcus</em> species (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Richardson and Lass-Flörl 2008</a>). Within <em>Candida</em> species, <em>Candida albicans</em> and <em>C. glabrata</em> are known as the most-recovered species from infected patients (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Marchetti <em>et al.</em> 2004</a>). The current reports on the epidemiology of fungal diseases and their associated impact on human health reflect that the available therapeutic options, which include antifungal drugs, have limited efficacy. Only four major classes of antifungal drug are used to treat patients including azoles, polyenes, echinocandins, and pyrimidine analogs. Each of these agents have specific cellular targets and are used in several formulations with different derivatives (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ruhnke 2014</a>). In general, most <em>Candida</em> species are susceptible to these drugs <em>in vitro</em>.</p><p id="p-5">However, the use of antifungal agents has resulted in the inevitable occurrence of antifungal resistance. Antifungal resistance in a specific fungal isolate can be defined as a lack of activity of a given drug at concentrations that are higher than in a wild-type isolate. Antifungal resistance has been reported for virtually all existing antifungal agents and major fungal pathogens (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Perlin <em>et al.</em> 2015</a>). Frequencies at which antifungal resistance occur in hospitalized patients are variable based on data collected by population-based surveillance programs and available for major fungal pathogens including <em>C. albicans</em> and <em>C. glabrata</em>. In general, antifungal resistance rates for <em>C. albicans</em> are low. In a study performed between 2008 and 2011, resistance to fluconazole (cut-off: ≥64 µg/ml) or echinocandins (cut-off: ≥4 µg/ml) ranged from between 1 and 2% in bloodstream isolates (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cleveland <em>et al.</em> 2012</a>). Resistance rates in <em>C. glabrata</em> are much higher: <em>C. glabrata</em> infections as a cause of invasive candidiasis increased from 18% of all BSI isolates in 1992–2001 to 25% in 2001–2007. Fluconazole resistance rates in <em>C. glabrata</em> increased from 9 to 14% in the same time period (cut-off: ≥64 µg/ml) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pfaller <em>et al.</em> 2003</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2009</a>). Resistance of <em>C. glabrata</em> to the class of echinocandins is also increasing: within a 10-yr period (2001–2010) in a US hospital, the echinocandin resistance rate increased from 4.9 to 12.3% (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2013</a>). Similar trends are documented in Europe, however resistance rates are reported to be between 1 and 4% (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arendrup and Perlin 2014</a>).</p><p id="p-6">Recent studies have also documented the occurrence of multidrug resistance not only in <em>C. albicans</em> but also in <em>C. glabrata</em> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sanglard 2016</a>). <em>C. glabrata</em> cross-resistance between azoles and candins has captured more attention recently (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Perlin 2015</a>). For example, among echinocandin-resistant isolates sampled between 2008 and 2013 in two US surveillance hospital sites, 36% were also resistant to azoles (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pham <em>et al.</em> 2014</a>). Between 2005 and 2013 in another US site, 10.3% <em>C. glabrata</em> isolates from cancer patients were resistant to caspofungin, from which ∼60% were cross-resistant to azoles (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Farmakiotis <em>et al.</em> 2014</a>). Intriguingly, the later study reported that caspofungin exposure alone could induce multidrug resistance without azole exposure. While <em>C. glabrata</em> azole resistance is mediated principally by efflux-dependent mechanisms, candin resistance is the consequence of mutations in the genes (<em>FKS1</em> and <em>FKS2</em>) encoding the candin target enzyme (β-glucan synthase) (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Perlin 2015</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Vale-Silva and Sanglard 2015</a>). It has been shown recently that <em>C. glabrata</em> can exhibit high mutation rates in the presence of antifungal agents. This property is based on the decreased activity of the DNA repair machinery that exists in specific clinical isolates (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Healey <em>et al.</em> 2016b</a>).</p><p id="p-7">We have shown in the past years that <em>C. glabrata</em> azole resistance is coupled with selective fitness advantages in animal models of infection (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ferrari <em>et al.</em> 2009</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Vale-Silva <em>et al.</em> 2013</a>). Specific isolates exhibit upregulation of the adhesin <em>EPA1</em> when the transcriptional activator responsible for azole resistance <em>CgPDR1</em> is mutated. When <em>EPA1</em> is upregulated, the consequence is that <em>C. glabrata</em> exhibits increased adherence to host cells, which is one of the factors that contributes to enhance virulence of this yeast species which develops azole resistance (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Vale-Silva <em>et al.</em> 2016</a>). Our recent data also highlighted that <em>C. glabrata</em> can exhibit different degrees of adhesion to host cells. Given that adhesins are implicated in this process, it becomes relevant to address the repertoire of adhesins in specific <em>C. glabrata</em> lineages. Genome data are critical to achieve this goal, however the only available <em>C. glabrata</em> genome is up to now from the isolate CBS138 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dujon <em>et al.</em> 2004</a>). It is estimated that <em>C. glabrata</em> contains 63 ORFs with adhesin properties (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de Groot <em>et al.</em> 2013</a>). In a recent study, we compared CBS138 to other clinical isolates and found that a specific isolate (DSY562) exhibited the much higher adherence capacities to epithelial cells as compared with CBS138. This result was not dependent on <em>EPA1</em> expression since DSY562 expressed <em>EPA1</em> at much lower levels than CBS138.</p><p id="p-8">These results stimulated our interest into the acquisition of the DSY562 genome. At the same time, it became also pertinent to compare the genome of this isolate with one (DSY565) that acquired azole resistance within a time-lapse of 50 d from DSY562 in a patient with oropharyngeal candidiasis (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sanglard <em>et al.</em> 1999</a>). In this way, evolutionary pressure on the derived isolate could be estimated at the level of genome differences.</p><p id="p-9">Here we report genome sequences from both related isolates. We used several approaches to resolve the genome with high accuracy. We analyzed the occurrence of polymorphisms between the two strains and also addressed the repertoire of genes encoding adhesins.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and media</h3><p id="p-10">Strains DSY562 and DSY565 were reported earlier (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Sanglard <em>et al.</em> 1999</a>) and were grown in YEPD at 30° under constant agitation.</p></div><div id="sec-3" class="subsection"><h3>DNA isolation</h3><p id="p-11">Genomic DNA from DSY5652 and DSY565 was obtained by the spheroplasting method as previously described (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Calabrese <em>et al.</em> 2000</a>).</p></div><div id="sec-4" class="subsection"><h3>Multilocus sequence typing</h3><p id="p-12">Multilocus sequence typing (MLST) was performed according to the study of <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dodgson <em>et al.</em> (2003)</a>. Sequencing data of individual loci were analyzed using an online web tool (<a href="http://cglabrata.mlst.net">http://cglabrata.mlst.net</a>) to determine the sequence type (ST).</p></div><div id="sec-5" class="subsection"><h3>Genome sequencing</h3><p id="p-13">To produce high-quality genomic DNA from <em>C. glabrata</em> isolates, overnight cultures (5 ml) were first grown in YNB minimal medium [0.67% yeast nitrogen base (Dicfo) with 2% glucose] at 30° under constant shaking to obtain 2 × 10<sup>8</sup> cells for each strain. The DNA isolation protocol followed instructions of the Gentra Puregene Yeast/Bact kit (Qiagen) with slight modifications. First, yeast cell lysis was performed with Zymolyase 100T (3 µg/µl end concentration) for 30 min at 37°. In addition, after RNAse A treatment of precipitated nucleic acids, phenol/chloroform extractions were carried out according to recommendations issued by Pacific Bioscience (PacBio SampleNet—Shared Protocol) for the use of phenol/chloroform/isoamyl alcohol. After final precipitation with NH<sub>4</sub>OAc and several washes with 80% alcohol, the genomic DNA was dissolved carefully in a small volume of elution buffer (10 mM Tris-HCl, pH 8.5). Aliquots of 5 μg of extracted, high-quality genomic DNA was diluted to 150 μl using elution buffer at 30 μg/μl. Long insert SMRTbell template libraries were prepared (20-kbp insert size) according to PacBio protocols. In total, nine SMRT cells per strain were sequenced using P4 and P5 polymerase binding and C2 and C3 sequencing kits with 120 min (five first SMRT cells) and 180 min acquisition on PacBio RSII. <em>De novo</em> genome assemblies were produced using PacBio’s SMRT Portal (v2.0.0) and the hierarchical genome assembly process (HGAP version 3.0), with default settings and a seed read cut-off length of 6000 bp.</p><p id="p-14">Illumina-based genome sequencing was performed by FASTERIS SA (Plan-les-Ouates, Switzerland). Briefly, 4 µg of each DSY562 and DSY565 genomic DNA was fragmented by nebulization and the genomic libraries were prepared following the procedure described in the Genomic DNA Sample Prep Kit (Illumina, FC-102-1003). To obtain inserts of 300 bp, a size selection was performed on agarose gel before a 10-cycle PCR amplification. The resulting libraries were sequenced in a 2 × 38-bp run on a Genome Analyzer IIx using the Sequencing Kit v4 (Illumina).</p></div><div id="sec-6" class="subsection"><h3>Genome comparisons</h3><p id="p-15">The final PacBio assemblies of DSY562 and DSY565 were analyzed using Geneious (9.1.8). Genome comparisons were performed on the basis of each chromosome’s homolog using the Mauve aligner plug in (progressive Mauve algorithm) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Darling <em>et al.</em> 2004</a>). The alignments were next inspected with a SNP discovery tool integrated in the software.</p><p id="p-16">For comparisons between CBS138 and the Illumina-based genome sequencing of DSY562 and DSY565, reads were imported in the CLC genomic workbench software (v.9.5.2) (Qiagen). Variant detection was performed from the mapped reads using both probabilistic and quality-based variant detection tools available in CLC. A pipeline of the analysis of genome data are included in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FigureS4.eps">Figure S4</a>.</p></div><div id="sec-7" class="subsection"><h3>RNAseq</h3><p id="p-17">RNA extractions from the <em>C. glabrata</em> strains DSY562 and DSY565 were carried out as previously published (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ferrari <em>et al.</em> 2011</a>). The strains were grown to logarithmic phase in 5 ml YEPD medium [1% Bacto peptone (Difco), 0.5% yeast extract (Difco), 2% glucose (Fluka)] at 30° under constant shaking. RNA libraries for RNAseq were prepared with a TruSeq Stranded Total RNA Library Prep Kit (Illumina). The resulting libraries were processed using the Illumina TruSeq PE cluster kit v3 reagents and sequenced on the Illumina HiSequation 2500 system using TruSeq SBS Kit v3 reagents. Sequencing data were processed using Illumina Pipeline software version 1.82 to obtain fastq files.</p></div><div id="sec-8" class="subsection"><h3>Sanger sequencing</h3><p id="p-18">Sequencing reactions were carried with a BigDye Terminator DNA Sequencing Kit (Thermo Fisher Scientific, Waltham, MA) and analyzed with a 3130XL genetic analyzer (Applied Biosystems). Primers used are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/TableS1.docx">Table S1</a>. Forward and reverse tag primer tails (5′-CGACGCCCGCTGATA-3′ and 5′-GTCCGGGAGCCATG-3′) were added to each primer pair (labeled by suffixes “-R” and “-F”). PCR were performed on DSY562 and DSY565 DNA with the forward and reverse primers which was followed by sequencing with the tag primers.</p></div><div id="sec-9" class="subsection"><h3>Adhesins predictions</h3><p id="p-19">DSY562 ORFs were analyzed with the help of the software FungalRV (<a href="http://fungalrv.igib.res.in/index.html">http://fungalrv.igib.res.in/index.html</a>), which helps to predict the occurrence of adhesin-like proteins in fungal genomes. To select for potential adhesins, a value of 0.5 was set as a threshold, as recommended (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chaudhuri <em>et al.</em> 2011</a>). The list of potential adhesins was further subjected to several additional tests including a search for PFAM domains (<a href="http://pfam.xfam.org/search#tabview=tab1">http://pfam.xfam.org/search#tabview=tab1</a>) and the presence of specific signatures relevant to the detection of adhesin-like proteins. These signature included (i) the presence of signal peptides and absence of internal transmembrane domains using the online programs SignalP and TMHMM (<a href="http://www.cbs.dtu.dk/services/">http://www.cbs.dtu.dk/services/</a>); (ii) the presence of consensus sequence for GPI anchor ([NSGDAC] – [GASVIETKDLF] – [GASV] – X(4,19) – [FILMVAGPSTCYWNQ](10)&gt;), as recommended by <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Weig <em>et al.</em> (2004)</a>; (iii) the presence of AWP motifs (VSHITT) (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">de Groot <em>et al.</em> 2013</a>); (iv) the presence of EPA motifs (EPA CC: CX(27)CX(40)CX(44)DDX(13)CC) and EPA: GCSX(8,9)GL) following <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Diderrich <em>et al.</em> (2015)</a> and <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gabaldón <em>et al.</em> (2013)</a>; and (v) the presence of PIR motifs (Q[IV]XDGQ[IVP]Q) following (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de Groot <em>et al.</em> 2008</a>).</p></div><div id="sec-10" class="subsection"><h3>Protein alignments</h3><p id="p-20">Protein alignments were performed with a Web tool available at <a href="http://mafft.cbrc.jp/alignment/server">http://mafft.cbrc.jp/alignment/server</a> (MAFFT version 7) and default settings. A phylogenetic tree was constructed from the alignment produced from the tool. The produced alignments were next visualized after average linkage (UPGMA) and a bootstrapping with 1000 resamplings.</p></div><div id="sec-11" class="subsection"><h3>Detection of splicing junctions</h3><p id="p-21">RNAseq data obtained from DSY562 and DSY565 were imported as fastq files into Geneious (9.1.8). Splice junctions were detected using the TopHat RNAseq aligner (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Trapnell <em>et al.</em> 2009</a>).</p></div><div id="sec-12" class="subsection"><h3>Counterclamped homogeneous electric field (CHEF) analysis</h3><p id="p-22">Two or three colonies of each yeast strain were inoculated in YEPD liquid medium and incubated overnight at 30° with agitation. A culture volume corresponding to 10<sup>9</sup> cells was pelleted at 3000 × <em>g</em> for 5 min, and cells were washed in 5 ml 50 mM EDTA (pH 9). The pellet was resuspended in 330 µl 50 mM EDTA. Next, 110 µl of SCE [1 M sorbitol, 10 mM EDTA, 100 mM sodium citrate (pH 5.8), adjusted with citric acid] was added and then completed with 5% β-mercaptoethanol and Zymolyase 100T (100 units/ml) (solution I). The solution was gently mixed with 560 µl of GTG agarose and rapidly distributed into molds of 10 by 5 by 2 mm. The solidified plugs were immersed in solution II [450 mM EDTA, 10 mM Tris-HCl (pH 8), 7.5% β-mercaptoethanol] and incubated overnight at 37° without agitation. Solution II was replaced by solution III [450 mM EDTA, 10 mM Tris-HCl (pH 8), 1% <em>N</em>-lauryl- Sarkosyl, and 150 mg/ml proteinase K]. Plugs were incubated for 6 hr at 65° without agitation and kept for 10 min on ice. Finally, plugs were transferred to a 0.5 M EDTA (pH 9) solution and stored at 4° for several months. One-third of the plug was loaded into wells of 0.6% GTG agarose gel in 0.5% Tris-borate-EDTA. The gel was then placed into the electrophoresis chamber of a CHEF DR II (Bio-Rad, Zürich, Switzerland) apparatus. Migration was performed in 0.5% Tris-borate-EDTA at 14° with the following steps: block 1, 60- to 120-min switch, 6 V/cm, and 120° over 24 hr; block 2, 120- to 300-min switch, 4.5 V/cm, and 120° over 22 hr. After gel electrophoresis, gels were stained with ethidium bromide to reveal DNA under ultraviolet light.</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-23">Strains described here are available upon request. PacBio assemblies are available under Bioproject <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA374542">PRJNA374542</a> (SRR5298594–SRR5298602 for DSY565, SRR5298595 and SRR5298512–SRR5298519 for DSY562). RNAseq data are available under Bioproject <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA374632">PRJNA374632</a> (SRR5296197–SRR5296202). Paired-end reads from Illumina-based genome sequencing are available under Bioproject PRJNA374542 (SRR5459192–SRR5459193). The final assembled genomes for DSY562 and DSY565 and their annotations are available under accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/MVOE00000000">MVOE00000000</a> and <a href="https://www.ncbi.nlm.nih.gov/nuccore/MVOF00000000">MVOF00000000</a>, respectively.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Genome data acquisition</h3><p id="p-24"><em>C. glabrata</em> DSY562 and DSY565 are paired clinical isolates that were isolated from an HIV-positive patient presenting oropharyngeal candidiasis. DSY565 was isolated after 50 d of fluconazole therapy and exhibited azole resistance (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Sanglard <em>et al.</em> 1999</a>) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The relationship between the two isolates was assessed at this time by RFLP which concluded a high relationship between the isolates. Before undertaking genome sequencing, both isolates were subjected to MLST typing according to <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dodgson <em>et al.</em> (2005)</a> (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), and thus the high relationship between both isolates was confirmed.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485992/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485992/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Characteristics of DSY562 and DSY565</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">Genome sequencing was first approached by Illumina HiSequation 38-bp, paired-end sequencing. Briefly, trimmed reads of both DSY5652 and DSY565 were aligned to the reference genome of CBS138 using the software CLC Genomics Workbench. In both cases, ∼95% of reads aligned to the CBS genomes and coverage was 72- to 64-fold for DSY562 and DSY565 data, respectively. Details on comparisons can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FileS1.xlsx">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FileS2.xlsx">File S2</a>. Both comparisons yielded a high number of changes (85,711–85,986 including SNPs and indels) and resulted in ∼10,390 nonsynonymous changes as compared to CBS138. Given that the genomes were aligned and compared directly to CBS138, they did not reflect actual genome rearrangements of strains DSY562 and DSY565. We therefore undertook an alternative genome-sequencing approach using PacBio technologies enabling <em>de novo</em> assembly of large reads. Since this technology is prone to random intrinsic high error rates, we aimed to reach genome sequencing coverage between 200- and 300-fold, which therefore required the use of several sequencing SMRT cells. The PacBio data are summarized in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. As summarized in this table, the number of major assembled contigs (size &gt;20 kb) were 14 and 15 for DSY562 and DSY565, respectively. N50s were 1.06 and 1.17 Mb for both isolates, thus highlighting the high quality of the assemblies. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> summarizes the sums of nucleotides for each assembled contig, which were assigned to chromosomes according to the current CBS138 genome data (see below). DSY562 and DSY565 accounted for totals of 12,729,188 and 12,728,164 nucleotides. This represents a difference of 1024 nucleotides between the two genomes. The difference in nucleotide counts for major contigs (including mitochondrial DNA) is larger (58,400 nucleotides), however this is compensated for by nucleotide counts of the small contigs which could not be consistently assembled to larger contigs.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485931/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485931/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">PacBio sequencing analysis of DSY562 and DSY565</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486004/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486004/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Details of PacBio assembly of DSY562 and DSY565</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Genome comparisons between clinical strains and CBS138</h3><p id="p-27">Assembled contigs almost reconstituted the entire set of chromosomes that is known from the CBS138 genome. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows the chromosome maps of DSY562 as compared to CBS138 as well as a circular map to highlight major translocations between the reference and the DSY562 genomes. For practical reasons, the naming of chromosomes (A to M) was kept as proposed initially for CBS138. The data highlight several chromosomal rearrangements between the two isolates. While Chr A, B, E, G, H, and K kept similar structures between both strains, other chromosomes underwent several modifications. Chr C, D, J, and M in DSY562 underwent exchanges from CBS138 chromosomes essentially at chromosomal extremities. Parts of Chr L and I were added to Chr F (which was renamed as Chr F-L-I) in DSY562, thus yielding the largest chromosome in this species (1.59 Mb). Chr D and L were joined to the shortened Chr I (which was renamed as Chr I-L-D). The left arms of Chr L and C were inverted as compared to CBS138. Remarkably, the separate DSY565 assembly yielded the same chromosome arrangement as DSY562, with the exception that Chr J was not completely assembled in DSY565. This chromosome was still split in two parts (Chr J1 and J2; see <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FigureS1.eps">Figure S1</a>) in DSY565. A CHEF was performed with strains CBS138 and DSY562/DSY565 to compare these assemblies with physical chromosome separations. Remarkably, size predictions from the assemblies corresponded well to physical separations (see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Red arrows shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> correspond to size conservation of Chr A, B, G, H, and J between the two strains, which is consistent with predictions. Chr C, D, E, and F-L-I were shifted to higher sizes (yellow color) by CHEF in DSY562, which is also consistent with assembly predictions. Chr M and I-L-D exhibited decreased sizes (blue color) as compared to CBS138, which is also predicted from assemblies. Chr K from DSY562 showed a discrepancy from the expected patterns from CBS138, however this result may be due to intrinsic chromosome size variations observed by others for this type of strain (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bader <em>et al.</em> 2012</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosome structures of DSY562 as compared to CBS138. A circular map representing the major translocations between the CBS138 and DSY562 genomes was produced using J-Circos v2 (An et al. 2015)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1297383144" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromosome structures of DSY562 as compared to CBS138. A circular map representing the major translocations between the CBS138 and DSY562 genomes was produced using J-Circos v2 (An &lt;em&gt;et al.&lt;/em&gt; 2015).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2413/F1.medium.gif" width="440" height="162"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/8/2413/F1.medium.gif" width="440" height="162"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486029" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-28" class="first-child">Chromosome structures of DSY562 as compared to CBS138. A circular map representing the major translocations between the CBS138 and DSY562 genomes was produced using J-Circos v2 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">An <em>et al.</em> 2015</a>).</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485911/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485911/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">DSY562 and DSY565 annotation features</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CHEF chromosome separations of isolates CBS138, DSY562, and DSY565. Chromosome sizes are indicated for CBS138 and were extracted from public databases. Colored &#x2192;&#x2019;s show the size changes between the CBS and DSY562/DSY565 genomes. Red, yellow, and blue colors signify size conservation, increased sizes, and decreased sizes between CBS138 and the two different genomes, respectively. The attached table lists the chromosome sizes deduced from genome data of CBS138 and DSY562/DSY565. Plotting the relative migration distance (Rf values) of known CBS138 chromosome sizes with the obtained sizes of DSY562 and DSY565 chromosomes yielded an R2 value of 0.981." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1297383144" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;CHEF chromosome separations of isolates CBS138, DSY562, and DSY565. Chromosome sizes are indicated for CBS138 and were extracted from public databases. Colored &#x2192;&#x2019;s show the size changes between the CBS and DSY562/DSY565 genomes. Red, yellow, and blue colors signify size conservation, increased sizes, and decreased sizes between CBS138 and the two different genomes, respectively. The attached table lists the chromosome sizes deduced from genome data of CBS138 and DSY562/DSY565. Plotting the relative migration distance (Rf values) of known CBS138 chromosome sizes with the obtained sizes of DSY562 and DSY565 chromosomes yielded an &lt;em&gt;R&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; value of 0.981.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2413/F2.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/8/2413/F2.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486057" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">CHEF chromosome separations of isolates CBS138, DSY562, and DSY565. Chromosome sizes are indicated for CBS138 and were extracted from public databases. Colored →’s show the size changes between the CBS and DSY562/DSY565 genomes. Red, yellow, and blue colors signify size conservation, increased sizes, and decreased sizes between CBS138 and the two different genomes, respectively. The attached table lists the chromosome sizes deduced from genome data of CBS138 and DSY562/DSY565. Plotting the relative migration distance (Rf values) of known CBS138 chromosome sizes with the obtained sizes of DSY562 and DSY565 chromosomes yielded an <em>R</em><sup>2</sup> value of 0.981.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">With these data, we first compared the DSY562 PacBio-assembled data (DSY562-HGAP3) with the CBS138 published genome (CGD, version_s02-m04-r02). The DSY562-HGAP3 genome was first annotated with the tools available in the software Geneious (version 9.1.8, “live-annotation” tool) with a threshold level of 95% between comparisons. DSY562-HGAP3 was also further annotated on the basis of a recently published annotation using CBS138 and RNAseq data (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Linde <em>et al.</em> 2015</a>). As observed from <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, the number of deduced CDS (locus tag prefix B1J91) in DSY562 was 5294, which was near to the 5293 predicted CDS from CBS138. The DSY565 genome was next annotated on the basis of DSY562 annotations with the same Geneious tool and yielded 5298 CDS (locus tag prefix B1J92). <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a> indicates that the number of CDS in mitochondrial DNA and numbers of tRNA were identical between strains. rDNAs were identified in DSY562 and DSY565, and were located at the telomeric ends of Chr L and M-J as in CBS138.</p><p id="p-31">The similarity between the total number of CDS in the three strains (5293–5298 CDS) obscures, however, the following divergence: when comparing the CDS of each isolate, only a total of 5225 CDS were shared between them. 65 CDS from CBS138 were not detected in DSY562 and DSY565 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FileS3.xlsx">File S3</a>). 78 CDS from DSY562 and DSY565 were not found in CBS138. From these 78 CDS, 61 were common to DSY562 and DSY565; thus highlighting unique CDS for each of the strains. These above-mentioned results suggest that several selection events resulted in the loss and gain of several genes between these strains. When inspecting the identity of these 61 genes, we found that most of the genes gained in DSY562 and DSY565 as compared to CBS138 belonged to adhesin-like genes (34). Nine genes originating from DSY562 and DSY565 (named here Tkp5-1 to -9) were most similar to the Tkp5 protein from <em>Vanderwaltozyma polyspora</em>. This type of protein encodes elements of Ty-like retrotransposons (see <em>Discussion</em>). Other common genes between DSY strains underwent duplication events, including B1J91/B1J92_D02794g (endoplasmic reticulum membrane protein), B1J91/B1J92_M05115g (a putative retrotransposon protein), B1J91/B1J92_H04257g, and B1J91/B1J92_H04279g (encoding copper metallothionein MT-II and MT-IIB in DSY562 and DSY565 respectively). Interestingly, B1J91/B1J92_H04257g and B1J91/B1J92_H04279g were found in four tandem repeats on Chr H as opposed to CBS138, in which only one repeat is found. The remaining 10 genes out of the 61 common to DSY562 and DSY565 were ranked as pseudogenes in the current CGD (some of which are also described as adhesin-like genes), thus explaining their absence from the CBS genome used in our analysis. Genes that were CBS138 specific (65) included, among others, adhesin-like genes (6), CAGL0B01243g (MATα1), and CAGL0G07175g (RNA binding protein with RNA-directed DNA polymerase activity). Notably, MATα1 is still present in DSY562 and DSY565 but as another CDS (B1J91/B1J92_B00242g). Most of the remaining CBS138-specific genes were of unknown function (48) (see details on <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FileS3.xlsx">File S3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagram representation of genes shared between DSY562, DSY565, and CBS138. The diagram was generated with a Web tool available at http://bioinformatics.psb.ugent.be/webtools/Venn/." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1297383144" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Venn diagram representation of genes shared between DSY562, DSY565, and CBS138. The diagram was generated with a Web tool available at http://bioinformatics.psb.ugent.be/webtools/Venn/.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2413/F3.medium.gif" width="436" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/8/2413/F3.medium.gif" width="436" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485962" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Venn diagram representation of genes shared between DSY562, DSY565, and CBS138. The diagram was generated with a Web tool available at <a href="http://bioinformatics.psb.ugent.be/webtools/Venn/">http://bioinformatics.psb.ugent.be/webtools/Venn/</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">At the level of single nucleotides, large differences were observed between DSY562 and CBS138 (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Nuclear and mitochondrial genomes diverged by 0.7% between the two strains. These data indicate extensive divergence between the two genomes, however they are in general agreement with the analysis deduced from other <em>C. glabrata</em> genomes (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Singh-Babak <em>et al.</em> 2012</a>). The high number of nonsynonymous SNPs (11,014) underscores a large number of amino acid substitutions. A high number of SNPs occurred in genes encoding adhesins, and the following section will discuss the resolution of this important gene category. A few genes underwent frameshifts resulting in protein truncations. For example, B1J91/B1J92_D00154g (<em>AQY1</em>, role in ion transport) was truncated in DSY562 and DSY565 (CDS length of 459 bp) as compared to CBS138 (CDS length of 873 bp). Interestingly, another homolog is present in DSY562 and DSY565 (B1J91/B1J92_A01221g), thus likely compensating for the probable loss of function of B1J91/B1J92_D00154g. B1J91/B1J92_C02343g (<em>ARB1</em>: a protein with ATPase activity and a role in cellular response to drugs, a ribosomal small subunit exported from the nucleus and cytosol), which contains an intron in CBS138, lacks a reading frame in DSY562 and DSY565. Removal of the intron in this gene restores a truncated CDS as compared to CBS138. B1J91/ B1J92_E01309g (<em>EKI1</em>: choline kinase activity, a role in the phosphatidylcholine biosynthetic process) exhibited an N-terminal deletion due to the use of other intron splicing as compared to the CBS138 homolog. B1J91/B1J92_M05687g (<em>NPY1</em>: has NAD+ diphosphatase activity and a role in the NADH metabolic process), B1J91/B1J92_G03487g (<em>TMN3</em>: has a role in cellular copper ion homeostasis), B1J91/B1J92_I10725g (YGR122W: has a role in negative regulation of transcription from the RNA polymerase II promoter), B1J91/B1J92_E01155g (<em>RPA14</em>: RNA polymerase I activity), and B1J91/B1J92_H06017g (<em>FLR1</em>: a multidrug transporter of the major facilitator superfamily) exhibit N-terminal truncated proteins as compared to CBS138. Whether the identified protein truncations may result in deficient functioning remains to be established. We believe that these changes do not reflect PacBio errors since (i) the DSY562 and DSY565 assemblies were produced independently to produce the same changes as compared to CBS138, and (ii) Sanger sequencing verifications confirmed the absence of PacBio errors [see example of B1J91/B1J92_H06017g (<em>FLR1</em>) sequencing in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FigureS2.eps">Figure S2</a>]. We also noticed that some previously annotated tandem CDS in CBS138 were now forming a single CDS in DSY562. For example, B1J91/B1J92_I02816g and B1J91/B1J92_I02838g (<em>AZF1</em>: asparagine-rich zinc finger protein) were now generating a single CDS (B1J91/B1J92_I02838g) in DSY562. B1J91/B1J92_E03421g and B1J91/B1J92_E03432g (close homologs of <em>Saccharomyces cerevisiae CDA1</em> and <em>CDA2</em> genes; chitin deacetylases) were now forming a single CDS (B1J91/B1J92_E03421g). It is interesting to observe that there are ∼370000 nt differences between the CBS138 and DSY562 genomes (the latter having the largest genome), thus highlighting that additional/novel genes may be present in the DSY strains (see below). As a last step in these comparisons, we analyzed the presence of introns in strains DSY562 and DSY565 as compared to CBS138. This was performed first by the transfer of annotations from the recently updated CBS138 genome. We next used RNAseq data from both DSY562 and DSY565 transcriptomes to detect spliced RNAs. <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a> shows that DSY562 and DSY565 genomes possessed at least 144 intron-containing genes, which is close to the number identified in CBS130 (150). RNAseq data from DSY562 and DSY565 showed that 114 and 115 splice sites, respectively, coincided with the predicted splice sites (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FileS4.xlsx">File S4</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485885/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485885/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Whole genome differences between DSY562 and CBS138</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>The cluster of adhesins in DSY562</h3><p id="p-34"><em>C. glabrata</em> is known for its extensive repertoire of adhesins. The genes encoding adhesins often contain repeated motifs and are mostly localized at chromosome extremities. These two characteristics make it challenging to establish a detailed and complete listing of adhesins from <em>C. glabrata</em> genomes. Here we have used PacBio genome assembly approaches to address these challenges and we used a systematic analysis (described in <em>Materials and Methods</em>) to identify adhesin-like genes in DSY562 and DS565. Briefly, all DSY562 CDS were screened with the help of the software FungalRV, which helps to predict the occurrence of adhesin-like proteins in fungal genomes (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chaudhuri <em>et al.</em> 2011</a>). We next analyzed the list of potential adhesins for the occurrence of PFAM domains and the presence of specific signatures, including (i) the presence of signal peptides and absence of internal transmembrane domains, (ii) the presence of a consensus sequence for GPI anchor, (iii) the presence of AWP motifs (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">de Groot <em>et al.</em> 2013</a>), (iv) the presence of motifs typical for the class of EPA proteins following <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Diderrich <em>et al.</em> (2015)</a> and <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Gabaldón <em>et al.</em> (2013)</a>, (v) the presence of PIR motifs, and lastly (vi) the occurrence of hyphally regulated cell wall proteins (Hyphal_reg_CWP, IPR021031). For practical reasons and for comparison purposes with CBS138, we used the same CAGL0 gene naming that is currently accepted for CBS138. When several highly similar proteins were identified in DSY562, we add a numerical suffix to them (<em>i.e.</em>, 1, 2, <em>etc</em>.). According to our analysis, we proposed a list of 101 adhesin-like proteins in DSY562 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FileS5.xlsx">File S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FileS6.xlsx">File S6</a>), which well exceeds the number of deduced adhesins in CBS138 (66) according to <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">de Groot <em>et al.</em> (2013)</a>. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the genomic localization of adhesins detected in DSY562 as compared to CBS138. Notably, the repertoire of adhesins in DSY565 is very similar to DSY562 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FigureS1.eps">Figure S1</a>, see below). ∼50% of the adhesin genes were located in DSY562, near to the telomeric ends (region of 50 kb down to chromosome ends) of the chromosomes. One example of the expansion of adhesins in DSY562 is given by the end of Chr C (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The size difference between Chr C between DSY562 and CBS138 may be explained by the tandem arrangement of a group of four adhesins. The DSY562 potential adhesins were aligned and a phylogenetic tree was assembled (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This tree was merged with conserved domain features and with CBS138 protein homologs to, on the one hand, help distinguish between these proteins and, on the other hand, to better delineate extensions of protein families in DSY562. Several protein clusters (highlighted by colors in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) were observed. Members of the EPA family (19) were well discerned in this tree (blue color of <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>) and harbor typical signatures of EPA adhesins, including a PA14 domain as well as EPA-specific domains (see details in <em>Material and Methods</em>), with the exception of B1J91_K00170g4. Duplication of specific EPAs occurred in DSY562 for <em>EPA2</em>, <em>EPA6</em>, <em>EPA8</em>, <em>EPA12</em>, and <em>EPA22</em>. The largest cluster of proteins contained VSHITT-motif repeats and PA14 motifs, however, they lacked EPA signatures (green color of <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). CBS138 PWP homologs (PWP1, -2, -3, and -5) were found in this cluster. VSHITT-motif repeats were also found in another cluster (red color of <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>) that essentially lacked PA14 motifs. The largest extension occurred for adhesin-like proteins with VSHITT repeats that include members of the PWP and AWP family (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>). Another well-distinguished cluster lacked both VSHITT and PA14 motifs and contained CBS138 AWP homologs (AWP6 and -7). Only one gene (B1J91_M11726g) contained a PIR motif and was extensively unrelated to other genes of the phylogenetic tree. The remaining proteins could not be assembled in clusters with common features. Taken together, the data suggest that 61 novel adhesin-like proteins were assigned to DSY562, while 40 remained in common with CBS138 (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>, only 50 adhesin-like proteins without frameshifts were taken into account in these calculations). The majority of the novel proteins exhibited VSHITT domains.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Arrangement of adhesin-like genes in C. glabrata CBS138 and DSY562. The position of adhesins is shown by red and blue &#x2192;&#x2019;s in CBS138 and DSY562, respectively. Chr C ends of DSY562 and CBS138 were enlarged to highlight the positioning of adhesins in the two genomes. Black &#x2192;&#x2019;s indicate the position of adhesins in CBS138 which were inversely located in the Chr C arm of DSY562. A group of three tandemly arranged adhesins in CBS138 (orange &#x2192;) is increased by one member in DSY562. This group of four adhesins is tandemly arranged with a second group of four adhesins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1297383144" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Arrangement of adhesin-like genes in &lt;em&gt;C. glabrata&lt;/em&gt; CBS138 and DSY562. The position of adhesins is shown by red and blue &#x2192;&#x2019;s in CBS138 and DSY562, respectively. Chr C ends of DSY562 and CBS138 were enlarged to highlight the positioning of adhesins in the two genomes. Black &#x2192;&#x2019;s indicate the position of adhesins in CBS138 which were inversely located in the Chr C arm of DSY562. A group of three tandemly arranged adhesins in CBS138 (orange &#x2192;) is increased by one member in DSY562. This group of four adhesins is tandemly arranged with a second group of four adhesins.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2413/F4.medium.gif" width="407" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/8/2413/F4.medium.gif" width="407" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486165" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Arrangement of adhesin-like genes in <em>C. glabrata</em> CBS138 and DSY562. The position of adhesins is shown by red and blue →’s in CBS138 and DSY562, respectively. Chr C ends of DSY562 and CBS138 were enlarged to highlight the positioning of adhesins in the two genomes. Black →’s indicate the position of adhesins in CBS138 which were inversely located in the Chr C arm of DSY562. A group of three tandemly arranged adhesins in CBS138 (orange →) is increased by one member in DSY562. This group of four adhesins is tandemly arranged with a second group of four adhesins.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Adhesins in DSY562. (A) Phylogenetic tree of DSY562 adhesin-like genes. Each gene is labeled with conserved PFAM functional domains (see Materials and Methods). Adhesin clusters (I&#x2013;VII) were those reported in de Groot et al. (2008). (B) Difference between DSY562 and CBS138 adhesins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1297383144" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Adhesins in DSY562. (A) Phylogenetic tree of DSY562 adhesin-like genes. Each gene is labeled with conserved PFAM functional domains (see &lt;em&gt;Materials and Methods&lt;/em&gt;). Adhesin clusters (I&#x2013;VII) were those reported in de Groot &lt;em&gt;et al.&lt;/em&gt; (2008). (B) Difference between DSY562 and CBS138 adhesins.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2413/F5.medium.gif" width="440" height="247"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/8/2413/F5.medium.gif" width="440" height="247"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486122" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Adhesins in DSY562. (A) Phylogenetic tree of DSY562 adhesin-like genes. Each gene is labeled with conserved PFAM functional domains (see <em>Materials and Methods</em>). Adhesin clusters (I–VII) were those reported in <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">de Groot <em>et al.</em> (2008)</a>. (B) Difference between DSY562 and CBS138 adhesins.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Genome comparisons between DSY562 and DSY565</h3><p id="p-37">As we mentioned earlier, DSY5652 and DSY565 are two highly related isolates that were obtained from a patient treated with fluconazole for oropharyngeal candidiasis. DSY565 was isolated ∼50 d after initiation of fluconazole treatment and was azole resistant. We reported that azole resistance was due to a mutation in the transcription factor <em>CgPDR1</em>. This mutation was contributing to an increase of virulence as compared to a parental strain and was measured in different animal models (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Ferrari <em>et al.</em> 2009</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Vale-Silva <em>et al.</em> 2016</a>). Given that virulence traits may be multifactorial, we reasoned that other mutations may contribute to this phenotype. Direct genome comparisons with PacBio-generated data may provide some cues into this hypothesis. We first annotated the DSY562 genome to a high resolution as described above and used this basis as a tool to annotate the DSY565 genome. The use of CBS138 as an annotation basis provided similar results (data not shown). <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, and <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a> compare several features between DSY562 and DSY565. At the genomic level, a few changes occurred. While 101 adhesin-like genes were identified in DSY562, 107 adhesin-like genes were found in DSY565 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FigureS1.eps">Figure S1</a>). 95 adhesins were shared between both isolates (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>) and the discrepancies between the two genomes were principally due to the presence of adhesin-like genes in small contigs, which could not be mapped to the chromosomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FigureS1.eps">Figure S1</a>). 12 and three adhesin-like and unique genes were identified in the small contigs of DSY565 and DSY562, respectively. In addition, DSY562 possesses one gene (B1J91_E00110g1) located at the extremity of Chr J that is not present in DSY565 (Chr J1). This may be due to incomplete assembly of Chr J1 in this isolate. One other specific change between DSY562 and DSY565 occurred at the left arm of Chr C. A size difference of ∼29 kb exists at the left arm of Chr C between DSY562 and DSY565 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FigureS1.eps">Figure S1</a>). While a tandem repeat of B1J91_C01067g is present in DSY562 at the Chr C extremity, only one copy remained in DSY565. However, this gene is now situated between the adhesins B1J92_C00968g1 and B1J91_C01133g1 of DSY565 (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). One remaining copy of this specific adhesin (B1J92_C01067g1) is situated in the small contig unitig16_17kb, which suggests that this small contig may be assigned to the missing nucleotides of the DSY565 Chr C left arm.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Adhesin distribution between CBS138 and DSY strains. (A) Divergent position of adhesins between DSY565 and DSY562. The map was created with the Mauve plug in available in the Geneious software. (B) Venn diagram representation of adhesins shared between CBS138, DSY562, and DSY565 isolates. Venn diagram data can be found in File S6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1297383144" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Adhesin distribution between CBS138 and DSY strains. (A) Divergent position of adhesins between DSY565 and DSY562. The map was created with the Mauve plug in available in the Geneious software. (B) Venn diagram representation of adhesins shared between CBS138, DSY562, and DSY565 isolates. Venn diagram data can be found in File S6.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2413/F6.medium.gif" width="440" height="239"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/8/2413/F6.medium.gif" width="440" height="239"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2413/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486071" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-38" class="first-child">Adhesin distribution between CBS138 and DSY strains. (A) Divergent position of adhesins between DSY565 and DSY562. The map was created with the Mauve plug in available in the Geneious software. (B) Venn diagram representation of adhesins shared between CBS138, DSY562, and DSY565 isolates. Venn diagram data can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FileS6.xlsx">File S6</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Another major difference between DSY562 and DSY565 is that DSY565 lacks the retrotransposon Tkp5-7 that is situated at the right arm of Chr M in DSY562. Interestingly, DSY565 has retained a single long terminal repeat (LTR) at this locus as a trace of the presence of the retrotransposon. A PCR-based approach confirmed the absence of Tkp5-7 in DSY565 while leaving the LTR signature (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FigureS3.eps">Figure S3</a>).</p><p id="p-40">At the single nucleotide level, differences between DSY562 and DSY565 include synonymous and nonsynonymous SNP variations as well as indels. <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> gives an overview of the observed differences. First we noticed that most of the indels in these comparisons were in homopolymeric sequences, which are long stretches (10–30) of identical nucleotides (108 in total). Since it is known that PacBio can yield these differences, we undertook systematic verifications of these sequences by separate Sanger sequencing of small PCR fragments comprising these putative indels. Only eight of these indels were confirmed by this approach (95 out of 108 were corrected; see <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FileS7.xlsx">File S7</a>). Other indels in nonhomopolymeric sequences (14 indels − 8 indels in homopolymeric sequences = 6) were identified by genome comparisons between DSY562 and DSY565. Their presence was confirmed by separate genome alignment with Illumina reads obtained in the early stage of the project (data not shown). We also identified a number of indels within coding regions that were also nonhomopolymeric. These variations between the two strains ranged from three to a larger number of nucleotides, however only a few of them created frameshifts (see <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485883/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485883/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Nucleotide variations between DSY562 and DSY565</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">A total of 17 nonsynonymous SNPs were identified between both genomes (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). As expected, the transcription factor <em>CgPDR1</em> harbored the L280F substitution in DSY565. This mutation is known to be sufficient to mediate azole resistance in <em>C. glabrata</em> (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Ferrari <em>et al.</em> 2009</a>). The other nonsynonymous SNPs were located in genes not known to be associated with drug resistance. Related with transcription was the identification of a nonsynonymous SNP in B1J91_L0L01001g (<em>NOT3</em>: subunit of CCR4-NOT, a global transcriptional regulator). Three other nonsynonymous SNPs were located in genes involved in rRNA metabolism and ribosome biogenesis: B1J91_H02057g (<em>GAR1</em>: nucleolar rRNA processing protein), B1J91_M00946g (YLR435w: a protein with a potential role in pre-rRNA processing), and B1J91_M08954g (YOR019w: a protein of unknown function; may interact with ribosomes). We noticed several genes containing nonsynonymous SNPs that annotated with transporter functions, including B1J91_A01804g (HXT3/HXT1: hexose transporter), B1J91_D02442g (<em>LEM3</em>: translocation of phospholipids and alkylphosphocholine drugs), B1J91_K04565g (YGR065c: vitamin H transporter), and B1J91_M13233g (<em>MNR2</em>: vacuolar membrane protein; putative magnesium transporter). Two other nonsynonymous SNPs were situated in adhesin genes (B1J91_C00968g3 and B1J91_C00209g2). The question of whether or not these mutations result from specific selection pressure from the host or from drug exposure cannot be answered yet; but either is possible. Besides these nonsynonymous SNPs, larger deletions an insertions were identified in coding sequences (<a id="xref-table-wrap-8-2" class="xref-table" href="#T8">Table 8</a>). The majority of these deletions/insertions (7 out of 11) were observed in adhesin-like genes. We cannot presently exclude that these changes are the results of PacBio read mis-assemblies. Some of the affected CDS are located near to chromosome ends, which are known to be recalcitrant to correct assembly and include repetitive sequences. It is likely that recombination events in these regions may generate variations which could result in a beneficial phenotype for the fungus.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486119/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486119/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Nonsynonymous SNPs between DSY562 and DSY565</span><div class="sb-div caption-clear"></div></div></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486027/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486027/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">Deletions/insertions in CDS between DSY562 and DSY565</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-48">We described here the genomes of two <em>C. glabrata</em> sequence isolates, DSY562 and DSY565, originating from an HIV-positive patient with oropharyngeal candidiasis. The close relationship between these two isolates reported earlier (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Sanglard <em>et al.</em> 1999</a>) is confirmed here by the high level of similarity between the sequenced genomes. We showed earlier that DSY565 contained a GOF mutation in <em>PDR1</em> (L280F), contributing to azole resistance but also enhancing virulence in animal models. <em>PDR1</em> GOF can modulate the expression of the adhesin <em>EPA1</em> and this effect results in increased adherence to host tissues. Enhanced virulence due to <em>PDR1</em> in DSY565 may also be due to other factors. On the other hand, we observed that DSY562, the parent of DSY565, was intrinsically highly adherent to host cells, even with the lowest <em>EPA1</em> expression levels as compared to other tested strains. This intrinsically high adherence capacity was observed with a <em>PDR1</em> wild-type allele (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Vale-Silva <em>et al.</em> 2016</a>). Interestingly, a separate study demonstrated that DSY562 was a <em>C. glabrata</em> isolate with the highest capacity among those tested to bind <em>C. albicans</em> hyphae (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tati <em>et al.</em> 2016</a>). Binding of <em>C. glabrata</em> involved the <em>C. albicans</em> Als3 adhesin. All these data suggested the potential presence of other adhesins in the genetic background of DSY562 and its related strain DSY565. Given these reasons, we aimed here to obtain the two <em>C. glabrata</em> genomes with a high resolution to identify all possible differences between the two genomes and to determine all possible adhesin and adhesin-like genes in the two genomes.</p><p id="p-49">At the genomic level, DSY562 and DSY565 underwent several major chromosomal rearrangements. No data are currently available on PacBio-assembled <em>C. glabrata</em> genomes of sequential isolates, and it is therefore difficult to compare our results with other studies. One recent short report described the genomes of three pairs of <em>C. glabrata</em> isolates from patients with candidemia (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Håvelsrud and Gaustad 2017</a>). The genomes were assembled <em>de novo</em> using Illumina reads, thus resulting in different genome outputs (119–186 scaffolds). Comparison between the DSY genomes and the three published pairs using the same chromosomal fragments highlighted divergences as high as those between DSY isolates and CBS138. This underlines the high genetic diversity existing between <em>C. glabrata</em> isolates of different origins which has been observed by others using different approaches. One other recent study also applied Illumina-based <em>de novo</em> assembly of 33 genomes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Carreté <em>et al.</em> 2017</a>) which identified 17 different chromosome patterns. Some patterns are similar to those observed in DSY562 and DSY565. While Chr D-L and Chr L inversions, as well as Chr I-L-D are among the common patterns, Chr F-L-I and Chr C inversions seem unique to DSY562 and DSY565. Another separate study established the genome of a <em>C. glabrata</em> isolate (CCTCC M202019) which is an industrial yeast strain widely used to produce α-oxocarboxylic acid (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Xu <em>et al.</em> 2016</a>). Genome data were established on the basis of a 150-bp, pair-end library sequenced by Illumina approaches and 111 contigs and 74 scaffolds were generated. Interestingly, this <em>C. glabrata</em> isolate was remarkably closely related to CBS138 (205 SNP differences) (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Xu <em>et al.</em> 2016</a>).</p><p id="p-50">Taken together, these studies underline the high genetic diversity existing between <em>C. glabrata</em> isolates of different origins. It is likely that future studies will reveal additional patterns. In proportion to the size of their genomes, the number of changes between the DSY strains is rather small. We noticed only 17 nonsynonymous changes between these isolates (<a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a>). Besides the known L280F change in <em>PDR1</em>, several other proteins with diverse functions were affected. The role played by these alterations is not clear at this time. One might speculate that they could contribute to the fitness of DSY565 or that they might be accidental. Other larger changes were detected between both strains (<a id="xref-table-wrap-8-3" class="xref-table" href="#T8">Table 8</a>) and they occurred mainly in adhesin-like genes. This could reflect some adaptation due to host selection pressure.</p><p id="p-51">It has been reported earlier that the mismatch-repair gene <em>MSH2</em> can modulate drug resistance when its activity is compromised by specific SNPs, giving rise to a hyper-mutator phenotype (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Healey <em>et al.</em> 2016b</a>). Interestingly, DSY562 and DSY565 carry a V239L nonsynonymous mutation in <em>MSH2</em>, which is known to favor this hyper-mutator phenotype (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Healey <em>et al.</em> 2016b</a>). One might expect that this phenotype could generate genetic diversity within the context of host conditions. DSY565 was isolated with a time-lapse of 50 d in the oral cavity of a patient as compared to DSY562. Within this elapsed time, the <em>MSH2</em> defect may lead to mutation accumulations. In a study probing the accumulation of mutations due to <em>MSH2</em> defects in <em>S. cerevisiae</em> over 170 generations <em>in vitro</em>, mutation rates (mutations per bp per generation) were in the range of 5–9 × 10<sup>−8</sup>, which corresponded to 8–16 single base pair substitutions and 110–180 deletions/insertions in the haploid <em>S. cerevisiae</em> genome (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lang <em>et al.</em> 2013</a>). The number of generations that occurred within the time-lapse of DSY562 and DSY565 isolation may correspond to at least 250–500 generations, when taking into account 5–10 generations/d according to published data (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Forche <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Roetzer <em>et al.</em> 2011</a>). Taking into account the data of <a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7</a> and <a id="xref-table-wrap-8-4" class="xref-table" href="#T8">Table 8</a>, the number of SNPs and indels between DSY562 and DSY565 are in agreement with numbers reported in <em>S. cerevisiae</em>. One may expect that in <em>C. glabrata</em> the <em>MSH2</em> defect may generate random mutations with a negative effect on <em>in vivo</em> fitness. The host conditions may serve as a selection pressure to eliminate progeny with fitness costs. We only sampled successful progenitors from the clinic, which of course strongly reduces the diversity of isolates for further analysis. This underpins that genetic diversity by the <em>MSH2</em> defect may be much higher than observed from the collected clinical strains. The current literature on the effect of DNA mismatch-repair defects also highlighted that the occurrence of mutations is not random: DNA regions with a greater density of repeats are more mutable in mismatch-repair defective cells (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ma <em>et al.</em> 2012</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lang <em>et al.</em> 2013</a>). In the context of adhesins, which are enriched in repetitive elements, <em>MSH2</em> defects may perhaps contribute to increase the diversity of this gene family over generations. It has been observed that 44–50% of investigated isolates (<em>n</em> = 625) were carrying <em>MSH2</em> defects which thus indicates that the <em>MSH2</em> defect may confer some selective advantage in <em>C. glabrata</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dellière <em>et al.</em> 2016</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Healey <em>et al.</em> 2016a</a>,<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">b</a>). In agreement with our hypothesis, it is interesting to mention that CBS138 and CCTCC M202019 do not exhibit <em>MSH2</em> defects and they contain a limited repertoire of adhesins as compared to DSY562/DSY565.</p><p id="p-52">Besides the above-mentioned SNPs, the DSY562 and DSY565 genomes harbored transposon-like genes not reported previously in <em>C. glabrata</em>. These Ty-like retrotransposons genes (named here Tkp5-1 to -9) were most similar to the Tkp5 protein from <em>V. polyspora</em>, a yeast representing the postwhole-genome duplication lineage most divergent from <em>S. cerevisiae</em> (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Scannell <em>et al.</em> 2007</a>). We observed that, while Tpk5-2 and Tpk5-5 to -9 encoded the same proteins, Tpk5-3 and -4 were different (66% identity) but still similar to the Tkp5 protein from <em>V. polyspora</em>. Tpk5-1 shared the C-terminal ends of Tpk5-3 and -4. As expected from retrotransposons, Tpk5-2 and Tpk5-5 to -9 were flanked by the same LTRs, while Tpk5-3 and -4 were flanked by other distinct LTRs. Curiously, Tpk5-1 was located next to B1J91/B1J92_M05115g on Chr G, which is described as similar to Ty5-6, a <em>S. paradoxus</em> retrotransposon. Both genes are flanked by the same LTR. Taken together, these features suggest that DSY562 and DSY565 have acquired retrotransposons from different sources and that these strains have undergone several transposition events. Notably, DSY565 did not retain Tkp5-7, which is otherwise situated at the right arm of Chr M in DSY562. Given that a single LTR remained at this location in DSY565, this suggests an active transposition system. This retrotransposon was not identified in another location in DSY565.</p><p id="p-53">We determined the presence of &gt;100 adhesin-like genes in both DSY strains, which was not yet anticipated from other genome-wide studies. This number exceeds by far the numbers published for CBS138 (50–66, depending on the criteria used for selection) and therefore suggests that an expansion of this gene family occurred in our isolates. A genome report on a separate <em>C. glabrata</em> isolate closely related to CBS138 identified 49 adhesin-like genes (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Xu <em>et al.</em> 2016</a>). Since no equivalent <em>C. glabrata</em> genome assembly has been yet published using a PacBio approach, we still cannot confirm whether or not the investigated isolates constitute a unique case or that a high repertoire of adhesin genes (&gt;100 genes) is shared in several <em>C. glabrata</em> clades. The number of identified adhesins between both isolates differs substantially (101 in DSY562 <em>vs.</em> 107 in DSY565). It is unlikely that this discrepancy could reflect an expansion of adhesins in DSY565 as compared to DSY562 in such a short time-lapse between sampling of isolates. We rather attribute this difference to technical issues during the assembly of PacBio reads. These reads produced small contigs, as illustrated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1/FigureS1.eps">Figure S1</a>, which are enriched in adhesin genes. These genes probably form part of chromosomal ends, but remain presently unassembled to major contigs.</p><p id="p-54">To help distinguish the adhesins in DSY562 and DSY565, we used a combination of hierarchical clustering and mapping of PFAM and functional domains. With this approach, we could distinguish four major groups including the EPA-like genes, two VSHIT-containing groups, and another group with genes lacking EPA and VSHIT signatures. This grouping is less complex than proposed earlier using CBS138 data (seven clusters) (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">de Groot <em>et al.</em> 2008</a>). The overlay between both grouping showed consistently that cluster I contained the EPA-like genes and that clusters V and VII and clusters II and III were grouped in two categories that include VSHIT-containing adhesins. The other clusters (IV and VI) are located in distinct branches, the latter being grouped in genes lacking EPA and VSHIT signatures. It is presently difficult to validate one or the other categorization, since no systematic functional analysis of all adhesins has been yet carried out with the exception of EPA-like genes (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zupancic <em>et al.</em> 2008</a>). Now that a more extensive repertoire of adhesins is available from our studies, such analysis may be undertaken in the future.</p></div><div class="section" id="sec-20"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">The authors are thankful to F. Ischer for technical assistance and the LGFT (University of Lausanne) for technical support. This work was supported by the Swiss National Research Foundation grant no. 31003A_146936. The purchase of the Pacific Biosciences RSII instrument at the University of Lausanne was financed in part by the “Loterie Romande” through the “Fondation pour la Recherche en Médecine Génétique.”</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-9"><p id="p-56">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042887/-/DC1</a>.</p></li><li class="fn" id="fn-10"><p id="p-57"><em>Communicating editor: J. Berman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-04-30"><span class="received-label">Received </span>April 30, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-06-26"><span class="accepted-label">Accepted </span>June 26, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Vale-Silva <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.1" data-doi="10.1093/cid/cit136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeiffer</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jimenez-Ortigosa</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catania</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Increasing Echinocandin resistance in <em>Candida glabrata</em>: clinical failure correlates with presence of <em>FKS</em> mutations and elevated minimum inhibitory concentrations.</span> <abbr class="cit-jnl-abbrev">Clin. Infect. Dis.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1724</span>–<span class="cit-lpage">1732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fcit136%26rft_id%253Dinfo%253Apmid%252F23487382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/cit136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23487382&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.2" data-doi="10.1093/bioinformatics/btu842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sajjanhar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">J-Circos: an interactive Circos plotter.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1463</span>–<span class="cit-lpage">1465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu842%26rft_id%253Dinfo%253Apmid%252F25540184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25540184&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.3" data-doi="10.1097/QCO.0000000000000111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arendrup</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perlin</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Echinocandin resistance: an emerging clinical problem?</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Infect. Dis.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">484</span>–<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BInfect.%2BDis.%26rft.volume%253D27%26rft.spage%253D484%26rft_id%253Dinfo%253Adoi%252F10.1097%252FQCO.0000000000000111%26rft_id%253Dinfo%253Apmid%252F25304391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/QCO.0000000000000111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25304391&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2413.4" data-doi="10.1371/journal.pone.0052218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraneveld</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tangwattanchuleeporn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gross karyotypic and phenotypic alterations among different progenies of the Candida glabrata CBS138/ATCC2001 reference strain.</span> <span class="cit-source">PLoS ONE</span> <span class="cit-vol">7</span>: <span class="cit-fpage">e52218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D7%26rft.spage%253De52218%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0052218%26rft_id%253Dinfo%253Apmid%252F23284942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0052218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284942&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.5" data-doi="10.1126/scitranslmed.3004404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denning</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gow</span>  <span class="cit-name-given-names">N. A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levitz</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Netea</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hidden killers: human fungal infections.</span> <abbr class="cit-jnl-abbrev">Sci. Transl. Med.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">165rv13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BTranslational%2BMedicine%26rft.stitle%253DSci%2BTransl%2BMed%26rft.aulast%253DBrown%26rft.auinit1%253DG.%2BD.%26rft.volume%253D4%26rft.issue%253D165%26rft.spage%253D165rv13%26rft.epage%253D165rv13%26rft.atitle%253DHidden%2BKillers%253A%2BHuman%2BFungal%2BInfections%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscitranslmed.3004404%26rft_id%253Dinfo%253Apmid%252F23253612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6InNjaXRyYW5zbWVkIjtzOjU6InJlc2lkIjtzOjEzOiI0LzE2NS8xNjVydjEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI0MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.6" data-doi="10.1099/00221287-146-11-2743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calabrese</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bille</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanglard</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A novel multidrug efflux transporter gene of the major facilitator superfamily from <em>Candida albicans</em> (<em>FLU1</em>) conferring resistance to fluconazole.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">2743</span>–<span class="cit-lpage">2754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DCalabrese%26rft.auinit1%253DD.%26rft.volume%253D146%26rft.issue%253D11%26rft.spage%253D2743%26rft.epage%253D2754%26rft.atitle%253DA%2Bnovel%2Bmultidrug%2Befflux%2Btransporter%2Bgene%2Bof%2Bthe%2Bmajor%2Bfacilitator%2Bsuperfamily%2Bfrom%2BCandida%2Balbicans%2B%2528FLU1%2529%2Bconferring%2Bresistance%2Bto%2Bfluconazole%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-146-11-2743%26rft_id%253Dinfo%253Apmid%252F11065353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-146-11-2743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11065353&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165344100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carreté</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ksiezopolska</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pequeroles</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gómez-Molero</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saus</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Patterns of genomic variation in the opportunistic pathogen Candida glabrata suggest the existence of mating and a secondary association to the human host.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">105791</span> DOI: https://doi.org/10.1101/105791.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.8" data-doi="10.1186/1471-2164-12-192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaudhuri</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ansari</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghunandanan</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramachandran</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FungalRV: adhesin prediction and immunoinformatics portal for human fungal pathogens.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DChaudhuri%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.spage%253D192%26rft.epage%253D192%26rft.atitle%253DFungalRV%253A%2Badhesin%2Bprediction%2Band%2Bimmunoinformatics%2Bportal%2Bfor%2Bhuman%2Bfungal%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-192%26rft_id%253Dinfo%253Apmid%252F21496229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21496229&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.9" data-doi="10.1093/cid/cis697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farley</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollick</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Changes in incidence and antifungal drug resistance in candidemia: results from population-based laboratory surveillance in Atlanta and Baltimore, 2008–2011.</span> <abbr class="cit-jnl-abbrev">Clin. Infect. Dis.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1352</span>–<span class="cit-lpage">1361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fcis697%26rft_id%253Dinfo%253Apmid%252F22893576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/cis697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22893576&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.10" data-doi="10.1101/gr.2289704"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A. C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blattner</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perna</span>  <span class="cit-name-given-names">N. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mauve: multiple alignment of conserved genomic sequence with rearrangements.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1394</span>–<span class="cit-lpage">1403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DDarling%26rft.auinit1%253DA.%2BC.E.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D1394%26rft.epage%253D1403%26rft.atitle%253DMauve%253A%2BMultiple%2BAlignment%2Bof%2BConserved%2BGenomic%2BSequence%2BWith%2BRearrangements%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2289704%26rft_id%253Dinfo%253Apmid%252F15231754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzcvMTM5NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNDEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.11" data-doi="10.1128/EC.00284-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">P. W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraneveld</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Q. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The cell wall of the human pathogen Candida glabrata: differential incorporation of novel adhesin-like wall proteins.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1951</span>–<span class="cit-lpage">1964</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00284-08%26rft_id%253Dinfo%253Apmid%252F18806209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI3LzExLzE5NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.12" data-doi="10.1128/EC.00364-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">P. W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Boer</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weig</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauhan</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Adhesins in human fungal pathogens: glue with plenty of stick.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr>. <span class="cit-vol">12</span>: <span class="cit-fpage">470</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00364-12%26rft_id%253Dinfo%253Apmid%252F23397570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxMi80LzQ3MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNDEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.13" data-doi="10.3389/fmicb.2016.02038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dellière</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Healey</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gits-Muselli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrara</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Fluconazole and Echinocandin resistance of Candida glabrata correlates better with antifungal drug exposure rather than with MSH2 mutator genotype in a French Cohort of patients harboring low rates of resistance.</span> <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D7%26rft.spage%253D2038%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2016.02038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2016.02038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.14" data-doi="10.1074/jbc.M115.655654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diderrich</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kock</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maestre-Reyna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steuber</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Structural Hot spots determine functional diversity of the Candida glabrata Epithelial adhesin family.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">19597</span>–<span class="cit-lpage">19613</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M115.655654%26rft_id%253Dinfo%253Apmid%252F26105055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTAvMzIvMTk1OTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.15" data-doi="10.1128/JCM.41.12.5709-5717.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodgson</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pujol</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denning</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soll</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Multilocus sequence typing of Candida glabrata reveals geographically enriched clades.</span> <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">5709</span>–<span class="cit-lpage">5717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DDodgson%26rft.auinit1%253DA.%2BR.%26rft.volume%253D41%26rft.issue%253D12%26rft.spage%253D5709%26rft.epage%253D5717%26rft.atitle%253DMultilocus%2BSequence%2BTyping%2Bof%2BCandida%2Bglabrata%2BReveals%2BGeographically%2BEnriched%2BClades%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.41.12.5709-5717.2003%26rft_id%253Dinfo%253Apmid%252F14662965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjEwOiI0MS8xMi81NzA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI0MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.16" data-doi="10.1016/j.fgb.2004.11.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodgson</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pujol</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfaller</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denning</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soll</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evidence for recombination in Candida glabrata.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DDodgson%26rft.auinit1%253DA.%2BR.%26rft.volume%253D42%26rft.issue%253D3%26rft.spage%253D233%26rft.epage%253D243%26rft.atitle%253DEvidence%2Bfor%2Brecombination%2Bin%2BCandida%2Bglabrata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2004.11.010%26rft_id%253Dinfo%253Apmid%252F15707844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2004.11.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15707844&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227169500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.17" data-doi="10.1038/nature02579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrens</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casaregola</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome evolution in yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDujon%26rft.auinit1%253DB.%26rft.volume%253D430%26rft.issue%253D6995%26rft.spage%253D35%26rft.epage%253D44%26rft.atitle%253DGenome%2Bevolution%2Bin%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02579%26rft_id%253Dinfo%253Apmid%252F15229592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15229592&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222356800036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.18" data-doi="10.3201/eid2011.140685"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farmakiotis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarrand</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kontoyiannis</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Drug-resistant <em>Candida glabrata</em> infection in cancer patients.</span> <abbr class="cit-jnl-abbrev">Emerg. Infect. Dis.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1833</span>–<span class="cit-lpage">1840</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg.%2BInfect.%2BDis.%26rft.volume%253D20%26rft.spage%253D1833%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2011.140685%26rft_id%253Dinfo%253Apmid%252F25340258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2011.140685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25340258&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2413.19" data-doi="10.1371/journal.ppat.1000268"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferrari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ischer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calabrese</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posteraro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanguinetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gain of function mutations in CgPDR1 of Candida glabrata not only mediate antifungal resistance but also enhance virulence.</span> <span class="cit-source">PLoS Pathog.</span> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DFerrari%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De1000268%26rft.epage%253De1000268%26rft.atitle%253DGain%2Bof%2Bfunction%2Bmutations%2Bin%2BCgPDR1%2Bof%2BCandida%2Bglabrata%2Bnot%2Bonly%2Bmediate%2Bantifungal%2Bresistance%2Bbut%2Balso%2Benhance%2Bvirulence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1000268%26rft_id%253Dinfo%253Apmid%252F19148266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1000268&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19148266&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.20" data-doi="10.1128/AAC.01271-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferrari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanguinetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Bernardis</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torelli</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posteraro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Loss of mitochondrial functions associated with azole resistance in <em>Candida glabrata r</em>esults in enhanced virulence in mice.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1852</span>–<span class="cit-lpage">1860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob.%2BAgents%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.01271-10%26rft_id%253Dinfo%253Apmid%252F21321146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjk6IjU1LzUvMTg1MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNDEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.21" data-doi="10.1534/genetics.109.103325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magee</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution in Candida albicans populations during a single passage through a mouse host.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103325%26rft_id%253Dinfo%253Apmid%252F19414562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzMvNzk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI0MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.22" data-doi="10.1186/1471-2164-14-623"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gabaldón</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcet-Houben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrens</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolotin-Fukuhara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative genomics of emerging pathogens in the Candida glabrata clade.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-623%26rft_id%253Dinfo%253Apmid%252F24034898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-623&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24034898&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.23" data-doi="10.1128/genomeA.00328-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Håvelsrud</span>  <span class="cit-name-given-names">O. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaustad</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Draft genome sequences of Candida glabrata isolates 1A, 1B, 2A, 2B, 3A, and 3B.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e00328-16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnouncements%26rft.stitle%253DGenome%2BAnnounc%26rft.aulast%253DHavelsrud%26rft.auinit1%253DO.%2BE.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De00328-16%26rft.epage%253De00328-16%26rft.atitle%253DDraft%2BGenome%2BSequences%2Bof%2BCandida%2Bglabrata%2BIsolates%2B1A%252C%2B1B%252C%2B2A%252C%2B2B%252C%2B3A%252C%2Band%2B3B%26rft_id%253Dinfo%253Adoi%252F10.1128%252FgenomeA.00328-16%26rft_id%253Dinfo%253Apmid%252F28280017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiZ2EiO3M6NToicmVzaWQiO3M6MTQ6IjUvMTAvZTAwMzI4LTE2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI0MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Healey</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jimenez-Ortigosa</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shor</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perlin</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Genetic drivers of multidrug resistance in Candida glabrata.</span> <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1724</span>–<span class="cit-lpage">1729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D7%26rft.spage%253D1724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.25" data-doi="10.1038/ncomms11388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Healey</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lockhart</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobel</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Prevalent mutator genotype identified in fungal pathogen <em>Candida glabrata</em> promotes multi-drug resistance.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms11388%26rft_id%253Dinfo%253Apmid%252F27009409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms11388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27009409&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.26" data-doi="10.1534/g3.113.006429"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gammie</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mutation rates, spectra, and genome-wide distribution of spontaneous mutations in mismatch repair deficient yeast.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1453</span>–<span class="cit-lpage">1465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D1453%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.006429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.113.006429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.27" data-doi="10.1093/nar/gku1357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linde</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duggan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horn</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sieber</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Defining the transcriptomic landscape of Candida glabrata by RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1392</span>–<span class="cit-lpage">1406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1357%26rft_id%253Dinfo%253Apmid%252F25586221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25586221&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.28" data-doi="10.1016/j.celrep.2011.10.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogacheva</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishant</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zanders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mutation hot spots in yeast caused by long-range clustering of homopolymeric sequences.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BRep%26rft.aulast%253DMa%26rft.auinit1%253DX.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D42%26rft.atitle%253DMutation%2Bhot%2Bspots%2Bin%2Byeast%2Bcaused%2Bby%2Blong-range%2Bclustering%2Bof%2Bhomopolymeric%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2011.10.003%26rft_id%253Dinfo%253Apmid%252F22832106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2011.10.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22832106&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309709500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.29" data-doi="10.1086/380637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchetti</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bille</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fluckiger</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eggimann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruef</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Epidemiology of candidemia in Swiss tertiary care hospitals: secular trends, 1991–2000.</span> <abbr class="cit-jnl-abbrev">Clin. Infect. Dis.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F380637%26rft_id%253Dinfo%253Apmid%252F14727199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/380637&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14727199&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188207600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.30" data-doi="10.1093/cid/civ791"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perlin</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Echinocandin resistance in <em>Candida</em>.</span> <abbr class="cit-jnl-abbrev">Clin. Infect. Dis.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">S612</span>–<span class="cit-lpage">S617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fciv791%26rft_id%253Dinfo%253Apmid%252F26567278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/civ791&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26567278&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perlin</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shor</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Update on antifungal drug resistance.</span> <abbr class="cit-jnl-abbrev">Curr Clin Micro Rpt</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">84</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BClin%2BMicro%2BRpt%26rft.volume%253D2%26rft.spage%253D84%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.32" data-doi="10.1128/JCM.41.5.2176-2179.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pfaller</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyken</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tendolkar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollis</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Variation in susceptibility of bloodstream isolates of <em>Candida glabrata</em> to fluconazole according to patient age and geographic location.</span> <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">2176</span>–<span class="cit-lpage">2179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DPfaller%26rft.auinit1%253DM.%2BA.%26rft.volume%253D41%26rft.issue%253D5%26rft.spage%253D2176%26rft.epage%253D2179%26rft.atitle%253DVariation%2Bin%2BSusceptibility%2Bof%2BBloodstream%2BIsolates%2Bof%2BCandida%2Bglabrata%2Bto%2BFluconazole%2BAccording%2Bto%2BPatient%2BAge%2Band%2BGeographic%2BLocation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.41.5.2176-2179.2003%26rft_id%253Dinfo%253Apmid%252F12734273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjQxLzUvMjE3NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNDEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.33" data-doi="10.1128/JCM.00946-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pfaller</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollis</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyken</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tendolkar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Variation in susceptibility of bloodstream isolates of <em>Candida glabrata</em> to fluconazole according to patient age and geographic location in the United States in 2001 to 2007.</span> <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">3185</span>–<span class="cit-lpage">3190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClin.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.00946-09%26rft_id%253Dinfo%253Apmid%252F19656983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjEwOiI0Ny8xMC8zMTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI0MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.34" data-doi="10.1128/AAC.03255-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iqbal</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolden</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuykendall</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Role of <em>FKS</em> mutations in <em>Candida glabrata</em>: MIC values, echinocandin resistance, and multidrug resistance.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">4690</span>–<span class="cit-lpage">4696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob.%2BAgents%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.03255-14%26rft_id%253Dinfo%253Apmid%252F24890592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjk6IjU4LzgvNDY5MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNDEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.35" data-doi="10.1111/j.1469-0691.2008.01978.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lass-Flörl</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Changing epidemiology of systemic fungal infections.</span> <abbr class="cit-jnl-abbrev">Clin. Microbiol. Infect.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Bmicrobiology%2Band%2Binfection%2B%253A%2B%2Bthe%2Bofficial%2Bpublication%2Bof%2Bthe%2BEuropean%2BSociety%2Bof%2BClinical%2BMicrobiology%2Band%2BInfectious%2BDiseases%26rft.stitle%253DClin%2BMicrobiol%2BInfect%26rft.aulast%253DBlot%26rft.auinit1%253DS.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D13%26rft.atitle%253DLimiting%2Bthe%2Battributable%2Bmortality%2Bof%2Bnosocomial%2Binfection%2Band%2Bmultidrug%2Bresistance%2Bin%2Bintensive%2Bcare%2Bunits.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-0691.2008.01978.x%26rft_id%253Dinfo%253Apmid%252F17944969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-0691.2008.01978.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17944969&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251562700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.36" data-doi="10.1111/j.1574-6968.2010.02102.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roetzer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldón</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schüller</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">From Saccharomyces cerevisiae to Candida glabrata in a few easy steps: important adaptations for an opportunistic pathogen.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">314</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2010.02102.x%26rft_id%253Dinfo%253Apmid%252F20846362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.2010.02102.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20846362&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.37" data-doi="10.1111/1469-0691.12622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruhnke</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Antifungal stewardship in invasive Candida infections.</span> <abbr class="cit-jnl-abbrev">Clin. Microbiol. Infect.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BMicrobiol.%2BInfect.%26rft.volume%253D20%26rft.issue%253DSuppl%2B6%26rft.spage%253D11%26rft.atitle%253DANTIFUNGAL%2BSTEWARDSHIP%2BIN%2BINVASIVE%2BCANDIDA%2BINFECTIONS%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1469-0691.12622%26rft_id%253Dinfo%253Apmid%252F24661820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1469-0691.12622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24661820&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanglard</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Emerging threats in antifungal-resistant fungal pathogens.</span> <abbr class="cit-jnl-abbrev">Front Med (Lausanne)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BMed%2B%2528Lausanne%2529%26rft.volume%253D3%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanglard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ischer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calabrese</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Majcherczyk</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bille</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The ATP binding cassette transporter gene CgCDR1 from Candida glabrata is involved in the resistance of clinical isolates to azole antifungal agents.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">2753</span>–<span class="cit-lpage">2765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrobial%2BAgents%2Band%2BChemotherapy%26rft.stitle%253DAntimicrob.%2BAgents%2BChemother.%26rft.aulast%253DSanglard%26rft.auinit1%253DD.%26rft.volume%253D43%26rft.issue%253D11%26rft.spage%253D2753%26rft.epage%253D2765%26rft.atitle%253DThe%2BATP%2BBinding%2BCassette%2BTransporter%2BGene%2BCgCDR1%2Bfrom%2BCandida%2Bglabrata%2BIs%2BInvolved%2Bin%2Bthe%2BResistance%2Bof%2BClinical%2BIsolates%2Bto%2BAzole%2BAntifungal%2BAgents%26rft_id%253Dinfo%253Apmid%252F10543759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjEwOiI0My8xMS8yNzUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI0MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.40" data-doi="10.1073/pnas.0608218104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frank</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolfit</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Independent sorting-out of thousands of duplicated gene pairs in two yeast species descended from a whole-genome duplication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">8397</span>–<span class="cit-lpage">8402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0608218104%26rft_id%253Dinfo%253Apmid%252F17494770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzIwLzgzOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.41" data-doi="10.1371/journal.ppat.1002718"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh-Babak</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babak</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diezmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Global analysis of the evolution and mechanism of echinocandin resistance in Candida glabrata.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DSingh-Babak%26rft.auinit1%253DS.%2BD.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002718%26rft.epage%253De1002718%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bthe%2Bevolution%2Band%2Bmechanism%2Bof%2Bechinocandin%2Bresistance%2Bin%2BCandida%2Bglabrata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002718%26rft_id%253Dinfo%253Apmid%252F22615574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002718&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22615574&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2413.42" data-doi="10.1371/journal.ppat.1005522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tati</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidow</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCall</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwang-Wong</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rojas</span>  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Candida glabrata binding to Candida albicans Hyphae enables its development in Oropharyngeal Candidiasis.</span> <span class="cit-source">PLoS Pathog.</span> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D12%26rft.spage%253De1005522%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005522&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.43" data-doi="10.1093/bioinformatics/btp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TopHat: discovering splice junctions with RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19289445&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265523300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.44" data-doi="10.1093/femsyr/fov025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vale-Silva</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanglard</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Tipping the balance both ways: drug resistance and virulence in Candida glabrata.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">fov025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffov025%26rft_id%253Dinfo%253Apmid%252F25979690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/femsyr/fov025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25979690&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.45" data-doi="10.1128/IAI.00074-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vale-Silva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ischer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leibundgut-Landmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanglard</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gain-of-function mutations in <em>PDR1</em>, a regulator of antifungal drug resistance in <em>Candida glabrata</em>, control adherence to host cells.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1720</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BImmun.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.00074-13%26rft_id%253Dinfo%253Apmid%252F23460523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzUvMTcwOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNDEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2413.46" data-doi="10.1128/mSphere.00065-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vale-Silva</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moeckli</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torelli</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posteraro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanguinetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Upregulation of the adhesin gene EPA1 mediated by PDR1 in Candida glabrata leads to enhanced host colonization.</span> <span class="cit-source">mSphere</span> <span class="cit-vol">1</span>: <span class="cit-fpage">e00065-15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmSphere%26rft.stitle%253DmSphere%26rft.aulast%253DVale-Silva%26rft.auinit1%253DL.%2BA.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253De00065-15%26rft.epage%253De00065-15%26rft.atitle%253DUpregulation%2Bof%2Bthe%2BAdhesin%2BGene%2BEPA1%2BMediated%2Bby%2BPDR1%2Bin%2BCandida%2Bglabrata%2BLeads%2Bto%2BEnhanced%2BHost%2BColonization%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmSphere.00065-15%26rft_id%253Dinfo%253Apmid%252F27303714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibXNwaCI7czo1OiJyZXNpZCI7czoxMzoiMS8yL2UwMDA2NS0xNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNDEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.47" data-doi="10.1099/mic.0.27256-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weig</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jänsch</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koster</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klis</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Systematic identification in silico of covalently bound cell wall proteins and analysis of protein-polysaccharide linkages of the human pathogen Candida glabrata.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">3129</span>–<span class="cit-lpage">3144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DWeig%26rft.auinit1%253DM.%26rft.volume%253D150%26rft.issue%253D10%26rft.spage%253D3129%26rft.epage%253D3144%26rft.atitle%253DSystematic%2Bidentification%2Bin%2Bsilico%2Bof%2Bcovalently%2Bbound%2Bcell%2Bwall%2Bproteins%2Band%2Banalysis%2Bof%2Bprotein-polysaccharide%2Blinkages%2Bof%2Bthe%2Bhuman%2Bpathogen%2BCandida%2Bglabrata%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.27256-0%26rft_id%253Dinfo%253Apmid%252F15470094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.27256-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15470094&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224695800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome sequencing of the Pyruvate-producing strain Candida glabrata CCTCC M202019 and genomic comparison with strain CBS138.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">34893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D34893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2413.49" data-doi="10.1111/j.1365-2958.2008.06184.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zupancic</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frieman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cummings</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Glycan microarray analysis of Candida glabrata adhesin ligand specificity.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DZupancic%26rft.auinit1%253DM.%2BL.%26rft.volume%253D68%26rft.issue%253D3%26rft.spage%253D547%26rft.epage%253D559%26rft.atitle%253DGlycan%2Bmicroarray%2Banalysis%2Bof%2BCandida%2Bglabrata%2Badhesin%2Bligand%2Bspecificity.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2008.06184.x%26rft_id%253Dinfo%253Apmid%252F18394144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2008.06184.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18394144&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254641600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/8/2413.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2407" title="Whole Genome Sequencing of the Braconid Parasitoid Wasp Fopius arisanus, an Important Biocontrol Agent of Pest Tepritid Fruit Flies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2427" title="A Genetic Screen Reveals an Unexpected Role for Yorkie Signaling in JAK/STAT-Dependent Hematopoietic Malignancies in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/8"><span>Volume 7 Issue 8, August 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/8.cover-source.jpg?itok=apH_YDuu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2413" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xsc_ZoPk_TXJfOMzw1oKHNTYVeCwuT4HBZI2URG-Jhw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/8/2413" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/8/2413" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/8/2413" class="active" data-icon-position="" data-hide-link-title="0">Comparative Genomics of Two Sequential Candida glabrata Clinical Isolates</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485847" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-d-ERSH1YDKKprOfs98r3TEZ2LGlfnuRicDaPno43AjU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2413.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/8/2413" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-rcRPOKngbocC-phjfPziYws9Sk1Mw2_E0eTUXgi_1ok" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/8/2413" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2413.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485847" id="node485847--2" data-pisa="ggg;7/8/2413" data-pisa-master="ggg;g3.117.042887" data-apath="/ggg/7/8/2413.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genomics of Two Sequential <em>Candida glabrata</em> Clinical Isolates</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9658-5335" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luis Vale-Silva</span>, <span class="highwire-citation-author" data-delta="1">Emmanuel Beaudoing</span>, <span class="highwire-citation-author" data-delta="2">Van Du T. Tran</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5244-4178" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dominique Sanglard</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2413-2426;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042887 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485847--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Luis Vale-Silva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Microbiology, University of Lausanne, CH-1011, Switzerland </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lausanne University Hospital, CH-1011, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BVale-Silva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vale-Silva%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuis%2BVale-Silva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9658-5335" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luis Vale-Silva</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Emmanuel Beaudoing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Integrative Genomics, Lausanne Genomic Technologies Facility, CH-1015, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emmanuel%2BBeaudoing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beaudoing%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmmanuel%2BBeaudoing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Van Du T. Tran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Vital-IT Group, SIB Swiss Institute of Bioinformatics, CH-1015 Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Van%2BDu%2BT.%2BTran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tran%20VD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVan%2BDu%2BT.%2BTran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dominique Sanglard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Microbiology, University of Lausanne, CH-1011, Switzerland </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lausanne University Hospital, CH-1011, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominique%2BSanglard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sanglard%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADominique%2BSanglard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5244-4178" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dominique Sanglard</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Dominique.Sanglard@chuv.ch" class="" data-icon-position="" data-hide-link-title="0">Dominique.Sanglard@chuv.ch</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/8/2413" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/8/2413&amp;title=Comparative%20Genomics%20of%20Two%20Sequential%20Candida%20glabrata%20Clinical%20Isolates" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/8/2413&amp;title=Comparative%20Genomics%20of%20Two%20Sequential%20Candida%20glabrata%20Clinical%20Isolates" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/8/2413&amp;title=Comparative%20Genomics%20of%20Two%20Sequential%20Candida%20glabrata%20Clinical%20Isolates" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/8/2413&amp;text=Comparative%20Genomics%20of%20Two%20Sequential%20Candida%20glabrata%20Clinical%20Isolates" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/8/2413&amp;title=Comparative%20Genomics%20of%20Two%20Sequential%20Candida%20glabrata%20Clinical%20Isolates" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/8/2413&amp;t=Comparative%20Genomics%20of%20Two%20Sequential%20Candida%20glabrata%20Clinical%20Isolates" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/8/2413&amp;title=Comparative%20Genomics%20of%20Two%20Sequential%20Candida%20glabrata%20Clinical%20Isolates" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/8/2413&amp;title=Comparative%20Genomics%20of%20Two%20Sequential%20Candida%20glabrata%20Clinical%20Isolates" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485847/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485847" id="node485847--3" data-pisa="ggg;7/8/2413" data-pisa-master="ggg;g3.117.042887" data-apath="/ggg/7/8/2413.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genomics of Two Sequential <em>Candida glabrata</em> Clinical Isolates</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9658-5335" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luis Vale-Silva</span>, <span class="highwire-citation-author" data-delta="1">Emmanuel Beaudoing</span>, <span class="highwire-citation-author" data-delta="2">Van Du T. Tran</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5244-4178" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dominique Sanglard</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2413-2426;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042887 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485847--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Luis Vale-Silva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Microbiology, University of Lausanne, CH-1011, Switzerland </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lausanne University Hospital, CH-1011, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BVale-Silva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vale-Silva%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuis%2BVale-Silva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9658-5335" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luis Vale-Silva</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Emmanuel Beaudoing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Integrative Genomics, Lausanne Genomic Technologies Facility, CH-1015, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emmanuel%2BBeaudoing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beaudoing%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmmanuel%2BBeaudoing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Van Du T. Tran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Vital-IT Group, SIB Swiss Institute of Bioinformatics, CH-1015 Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Van%2BDu%2BT.%2BTran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tran%20VD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVan%2BDu%2BT.%2BTran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dominique Sanglard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Microbiology, University of Lausanne, CH-1011, Switzerland </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Lausanne University Hospital, CH-1011, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominique%2BSanglard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sanglard%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADominique%2BSanglard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5244-4178" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dominique Sanglard</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Dominique.Sanglard@chuv.ch" class="" data-icon-position="" data-hide-link-title="0">Dominique.Sanglard@chuv.ch</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2413&amp;rft.epage=2426&amp;rft.atitle=Comparative%20Genomics%20of%20Two%20Sequential%20Candida%20glabrata%20Clinical%20Isolates&amp;rft.volume=7&amp;rft.issue=8&amp;rft.date=2017-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Vale-Silva%2C+Luis&amp;rft.au=Beaudoing%2C+Emmanuel&amp;rft.au=Tran%2C+Van+Du+T.&amp;rft.au=Sanglard%2C+Dominique"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485847/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485847/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485847/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485847/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485847/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485847/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485847/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485847/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485847/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485847/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485847/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485847/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/8/2413" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/8/2413.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/8/2413.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/8/2413.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/8/2413.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
