<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/8/2763" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/8/2763.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/8/2763.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/8/2763.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/8/2763/embed/mml-math-2.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/8/2763.atom" />
<meta name="HW.pisa" content="ggg;7/8/2763" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data" />
<meta name="DC.Identifier" content="10.1534/g3.117.043893" />
<meta name="DC.Date" content="2017-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Nelson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Transposable element (TE) insertions are among the most challenging types of variants to detect in genomic data because of their repetitive nature and complex mechanisms of replication . Nevertheless, the recent availability of large resequencing data sets has spurred the development of many new methods to detect TE insertions in whole-genome shotgun sequences. Here we report an integrated bioinformatics pipeline for the detection of TE insertions in whole-genome shotgun data, called McClintock (&lt;https://github.com/bergmanlab/mcclintock&gt;), which automatically runs and standardizes output for multiple TE detection methods. We demonstrate the utility of McClintock by evaluating six TE detection methods using simulated and real genome data from the model microbial eukaryote, Saccharomyces cerevisiae . We find substantial variation among McClintock component methods in their ability to detect nonreference TEs in the yeast genome, but show that nonreference TEs at nearly all biologically realistic locations can be detected in simulated data by combining multiple methods that use split-read and read-pair evidence. In general, our results reveal that split-read methods detect fewer nonreference TE insertions than read-pair methods, but generally have much higher positional accuracy. Analysis of a large sample of real yeast genomes reveals that most McClintock component methods can recover known aspects of TE biology in yeast such as the transpositional activity status of families, target preferences, and target site duplication structure, albeit with varying levels of accuracy. Our work provides a general framework for integrating and analyzing results from multiple TE detection methods, as well as useful guidance for researchers studying TEs in yeast resequencing data." />
<meta name="DC.Contributor" content="Michael G. Nelson" />
<meta name="DC.Contributor" content="Raquel S. Linheiro" />
<meta name="DC.Contributor" content="Casey M. Bergman" />
<meta name="article:published_time" content="2017-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Transposable element (TE) insertions are among the most challenging types of variants to detect in genomic data because of their repetitive nature and complex mechanisms of replication . Nevertheless, the recent availability of large resequencing data sets has spurred the development of many new methods to detect TE insertions in whole-genome shotgun sequences. Here we report an integrated bioinformatics pipeline for the detection of TE insertions in whole-genome shotgun data, called McClintock (https://github.com/bergmanlab/mcclintock), which automatically runs and standardizes output for multiple TE detection methods. We demonstrate the utility of McClintock by evaluating six TE detection methods using simulated and real genome data from the model microbial eukaryote, &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;. We find substantial variation among McClintock component methods in their ability to detect nonreference TEs in the yeast genome, but show that nonreference TEs at nearly all biologically realistic locations can be detected in simulated data by combining multiple methods that use split-read and read-pair evidence. In general, our results reveal that split-read methods detect fewer nonreference TE insertions than read-pair methods, but generally have much higher positional accuracy. Analysis of a large sample of real yeast genomes reveals that most McClintock component methods can recover known aspects of TE biology in yeast such as the transpositional activity status of families, target preferences, and target site duplication structure, albeit with varying levels of accuracy. Our work provides a general framework for integrating and analyzing results from multiple TE detection methods, as well as useful guidance for researchers studying TEs in yeast resequencing data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/08/01" />
<meta name="citation_mjid" content="ggg;7/8/2763" />
<meta name="citation_id" content="7/8/2763" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/8/2763" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/8/2763.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/8/2763.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/8/2763.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.043893" />
<meta name="citation_pmid" content="28637810" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2763" />
<meta name="citation_lastpage" content="2778" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Michael G. Nelson" />
<meta name="citation_author_institution" content="Faculty of Life Sciences, University of Manchester, M13 9PL, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2567-2634" />
<meta name="citation_author" content="Raquel S. Linheiro" />
<meta name="citation_author_institution" content="Faculty of Life Sciences, University of Manchester, M13 9PL, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2659-0910" />
<meta name="citation_author" content="Casey M. Bergman" />
<meta name="citation_author_institution" content="Faculty of Life Sciences, University of Manchester, M13 9PL, United Kingdom" />
<meta name="citation_author_email" content="cbergman@uga.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5462-9854" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P. Almeida;citation_author=R. Barbosa;citation_author=P. Zalar;citation_author=Y. Imanishi;citation_author=K. Shimizu;citation_title=A population genomics insight into the Mediterranean origins of wine yeast domestication.;citation_pages=5412-5427;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13341" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. A. Baller;citation_author=J. Gao;citation_author=D. F. Voytas;citation_title=Access to DNA establishes a secondary target site bias for the yeast retrotransposon Ty5.;citation_volume=108;citation_year=2011;citation_pmid=21788500;citation_doi=10.1073/pnas.1103665108" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. A. Baller;citation_author=J. Gao;citation_author=R. Stamenova;citation_author=M. J. Curcio;citation_author=D. F. Voytas;citation_title=A nucleosomal surface defines an integration hotspot for the Saccharomyces cerevisiae Ty1 retrotransposon.;citation_volume=22;citation_year=2012;citation_pmid=22219511;citation_doi=10.1101/gr.129585.111" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=P. Bardou;citation_author=J. Mariette;citation_author=F. Escudié;citation_author=C. Djemiel;citation_author=C. Klopp;citation_title=Jvenn: an interactive Venn diagram viewer.;citation_pages=293;citation_volume=15;citation_year=2014;citation_pmid=25176396;citation_doi=10.1186/1471-2105-15-293" />
<meta name="citation_reference" content="citation_journal_abbrev=Mob Genet Elements;citation_author=C. M. Bergman;citation_title=A proposal for the reference-based annotation of de novo transposable element insertions.;citation_pages=51-54;citation_volume=2;citation_year=2012;citation_issue=1;citation_pmid=22754753" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=C. M. Bergman;citation_author=H. Quesneville;citation_title=Discovering and detecting transposable elements in genome sequences.;citation_volume=8;citation_year=2007;citation_pmid=17932080;citation_doi=10.1093/bib/bbm048" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Carr;citation_author=D. Bensasson;citation_author=C. M. Bergman;citation_title=Evolutionary genomics of transposable elements in Saccharomyces cerevisiae.;citation_pages=e50978;citation_volume=7;citation_year=2012;citation_pmid=23226439;citation_doi=10.1371/journal.pone.0050978" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. L. Chalker;citation_author=S. B. Sandmeyer;citation_title=Transfer RNA genes are genomic targets for de Novo transposition of the yeast retrotransposon Ty3.;citation_pages=837-850;citation_volume=126;citation_year=1990;citation_issue=4;citation_pmid=1963869" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=D. L. Chalker;citation_author=S. B. Sandmeyer;citation_title=Ty3 integrates within the region of RNA polymerase III transcription initiation.;citation_pages=117-128;citation_volume=6;citation_year=1992;citation_issue=1;citation_pmid=1309715;citation_doi=10.1101/gad.6.1.117" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Chen;citation_author=H. Yao;citation_author=E. J. Thompson;citation_author=N. M. Tannir;citation_author=J. N. Weinstein;citation_title=VirusSeq: software to identify viruses and their integration sites using next-generation sequencing of human cancer tissue.;citation_volume=29;citation_year=2013;citation_pmid=23162058;citation_doi=10.1093/bioinformatics/bts665" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. M. Cridland;citation_author=S. J. Macdonald;citation_author=A. D. Long;citation_author=K. R. Thornton;citation_title=Abundance and distribution of transposable elements in two Drosophila QTL mapping resources.;citation_volume=30;citation_year=2013;citation_pmid=23883524;citation_doi=10.1093/molbev/mst129" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. E. Devine;citation_author=J. D. Boeke;citation_title=Integration of the yeast retrotransposon Ty1 is targeted to regions upstream of genes transcribed by RNA polymerase III.;citation_pages=620-633;citation_volume=10;citation_year=1996;citation_issue=5;citation_pmid=8598291;citation_doi=10.1101/gad.10.5.620" />
<meta name="citation_reference" content="citation_journal_title=Mob. DNA;citation_author=A. D. Ewing;citation_title=Transposable element detection from whole genome sequence data.;citation_pages=24;citation_volume=6;citation_year=2015;citation_pmid=26719777;citation_doi=10.1186/s13100-015-0055-3" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. D. Ewing;citation_author=H. H. Kazazian;citation_title=High-throughput sequencing reveals extensive variation in human-specific L1 content in individual human genomes.;citation_volume=20;citation_year=2010;citation_pmid=20488934;citation_doi=10.1101/gr.106419.110" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. D. Ewing;citation_author=H. H. Kazazian;citation_title=Whole-genome resequencing allows detection of many rare LINE-1 insertion alleles in humans.;citation_volume=21;citation_year=2011;citation_pmid=20980553;citation_doi=10.1101/gr.114777.110" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=E. G. Fingerman;citation_author=P. G. Dombrowski;citation_author=C. A. Francis;citation_author=P. D. Sniegowski;citation_title=Distribution and sequence analysis of a novel Ty3-like element in natural Saccharomyces paradoxus isolates.;citation_pages=761-770;citation_volume=20;citation_year=2003;citation_issue=9;citation_pmid=12845602;citation_doi=10.1002/yea.1005" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A.-S. Fiston-Lavier;citation_author=M. Carrigan;citation_author=D. A. Petrov;citation_author=J. González;citation_title=T-lex: a program for fast and accurate assessment of transposable element presence using next-generation sequencing data.;citation_pages=e36-e36;citation_volume=39;citation_year=2011;citation_issue=6;citation_pmid=21177644" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A.-S. Fiston-Lavier;citation_author=M. G. Barrón;citation_author=D. A. Petrov;citation_author=J. González;citation_title=T-lex2: genotyping, frequency estimation and re-annotation of transposable elements using single or pooled next-generation sequencing data.;citation_volume=43;citation_year=2015;citation_pmid=25510498;citation_doi=10.1093/nar/gku1250" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Gafner;citation_author=P. Philippsen;citation_title=The yeast transposon Ty1 generates duplications of target DNA on insertion.;citation_pages=414-418;citation_volume=286;citation_year=1980;citation_issue=5771;citation_pmid=6250065;citation_doi=10.1038/286414a0" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=A. Gilly;citation_author=M. Etcheverry;citation_author=M.-A. Madoui;citation_author=J. Guy;citation_author=L. Quadrana;citation_title=TE-Tracker: systematic identification of transposition events through whole-genome resequencing.;citation_pages=377;citation_volume=15;citation_year=2014;citation_pmid=25408240;citation_doi=10.1186/s12859-014-0377-z" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. Goffeau;citation_author=B. G. Barrell;citation_author=H. Bussey;citation_author=R. W. Davis;citation_author=B. Dujon;citation_title=Life with 6000 genes.;citation_pages=563-567;citation_volume=274;citation_year=1996;citation_doi=10.1126/science.274.5287.546" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Hawkey;citation_author=M. Hamidian;citation_author=R. R. Wick;citation_author=D. J. Edwards;citation_author=H. Billman-Jacobe;citation_title=ISMapper: identifying transposase insertion sites in bacterial genomes from short read sequence data.;citation_pages=667;citation_volume=16;citation_year=2015;citation_pmid=26336060;citation_doi=10.1186/s12864-015-1860-2" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=E. Hénaff;citation_author=L. Zapata;citation_author=J. M. Casacuberta;citation_author=S. Ossowski;citation_title=Jitterbug: somatic and germline transposon insertion detection at single-nucleotide resolution.;citation_pages=768;citation_volume=16;citation_year=2015;citation_pmid=26459856;citation_doi=10.1186/s12864-015-1975-5" />
<meta name="citation_reference" content="citation_journal_title=Mob. DNA;citation_author=D. R. Hoen;citation_author=G. Hickey;citation_author=G. Bourque;citation_author=J. Casacuberta;citation_author=R. Cordaux;citation_title=A call for benchmarking transposable element annotation methods.;citation_pages=13;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. Hormozdiari;citation_author=I. Hajirasouliha;citation_author=P. Dao;citation_author=F. Hach;citation_author=D. Yorukoglu;citation_title=Next-generation VariationHunter: combinatorial algorithms for transposon insertion discovery.;citation_volume=26;citation_year=2010;citation_pmid=20529927;citation_doi=10.1093/bioinformatics/btq216" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=H. Ji;citation_author=D. P. Moore;citation_author=M. A. Blomberg;citation_author=L. T. Braiterman;citation_author=D. F. Voytas;citation_title=Hotspots for unselected Ty1 transposition events on yeast chromosome III are near tRNA genes and LTR sequences.;citation_pages=1007-1018;citation_volume=73;citation_year=1993;citation_issue=5;citation_pmid=8388781;citation_doi=10.1016/0092-8674(93)90278-X" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=C. Jiang;citation_author=C. Chen;citation_author=Z. Huang;citation_author=R. Liu;citation_author=J. Verdier;citation_title=ITIS, a bioinformatics tool for accurate identification of transposon insertion sites using next-generation sequencing data.;citation_pages=72;citation_volume=16;citation_year=2015;citation_pmid=25887332;citation_doi=10.1186/s12859-015-0507-2" />
<meta name="citation_reference" content="citation_title=Genome Biol.;citation_author=J. S. Kaminker;citation_author=C. M. Bergman;citation_author=B. Kronmiller;citation_author=J. Carlson;citation_author=R. Svirskas;citation_title=The transposable elements of the Drosophila melanogaster euchromatin: a genomics perspective.;citation_pages=research0084;citation_volume=3;citation_year=2002;citation_pmid=12537573" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. M. Keane;citation_author=K. Wong;citation_author=D. J. Adams;citation_title=RetroSeq: transposable element discovery from next-generation sequencing data.;citation_volume=29;citation_year=2013;citation_pmid=23233656;citation_doi=10.1093/bioinformatics/bts697" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=BLAT—the BLAST-like alignment tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. J. Kent;citation_author=A. S. Zweig;citation_author=G. Barber;citation_author=A. S. Hinrichs;citation_author=D. Karolchik;citation_title=BigWig and BigBed: enabling browsing of large distributed datasets.;citation_volume=26;citation_year=2010;citation_pmid=20639541;citation_doi=10.1093/bioinformatics/btq351" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. M. Kim;citation_author=S. Vanguri;citation_author=J. D. Boeke;citation_author=A. Gabriel;citation_author=D. F. Voytas;citation_title=Transposable Elements and Genome Organization: A Comprehensive Survey of Retrotransposons Revealed by the Complete Saccharomyces cerevisiae Genome Sequence;citation_pages=464-478;citation_volume=8;citation_year=1998;citation_issue=5;citation_pmid=9582191" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Kofler;citation_author=A. J. Betancourt;citation_author=C. Schlötterer;citation_title=Sequencing of pooled DNA samples (Pool-Seq) uncovers complex dynamics of transposable element insertions in Drosophila melanogaster.;citation_pages=e1002487-e1002487;citation_volume=8;citation_year=2012;citation_issue=1;citation_pmid=22291611;citation_doi=10.1371/journal.pgen.1002487" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. Kofler;citation_author=D. Gómez-Sánchez;citation_author=C. Schlötterer;citation_title=PoPoolationTE2: comparative population genomics of transposable elements using pool-seq.;citation_volume=33;citation_year=2016;citation_pmid=27486221;citation_doi=10.1093/molbev/msw137" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=R. M. Kuhn;citation_author=D. Haussler;citation_author=W. J. Kent;citation_title=The UCSC genome browser and associated tools.;citation_volume=14;citation_year=2013;citation_pmid=22908213;citation_doi=10.1093/bib/bbs038" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=E. Lee;citation_author=R. Iskow;citation_author=L. Yang;citation_author=O. Gokcumen;citation_author=P. Haseley;citation_title=Landscape of somatic retrotransposition in human cancers.;citation_volume=337;citation_year=2012;citation_pmid=22745252;citation_doi=10.1126/science.1222077" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=E. Lerat;citation_title=Identifying repeats and transposable elements in sequenced genomes: how to find your way through the dense forest of programs.;citation_pages=520-533;citation_volume=104;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. S. Linheiro;citation_author=C. M. Bergman;citation_title=Whole genome resequencing reveals natural target site preferences of transposable elements in Drosophila melanogaster.;citation_pages=e30008-e30008;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22347367;citation_doi=10.1371/journal.pone.0030008" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Mularoni;citation_author=Y. Zhou;citation_author=T. Bowen;citation_author=S. Gangadharan;citation_author=S. J. Wheelan;citation_title=Retrotransposon Ty1 integration targets specifically positioned asymmetric nucleosomal DNA segments in tRNA hotspots.;citation_volume=22;citation_year=2012;citation_pmid=22219510;citation_doi=10.1101/gr.129460.111" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=M. Nakagome;citation_author=E. Solovieva;citation_author=A. Takahashi;citation_author=H. Yasue;citation_author=H. Hirochika;citation_title=Transposon Insertion Finder (TIF): a novel program for detection of de novo transpositions of transposable elements.;citation_pages=71;citation_volume=15;citation_year=2014;citation_pmid=24629057;citation_doi=10.1186/1471-2105-15-71" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=C. Nellaker;citation_author=T. M. Keane;citation_author=B. Yalcin;citation_author=K. Wong;citation_author=A. Agam;citation_title=The genomic landscape shaped by selection on transposable elements across 18 mouse strains.;citation_pages=R45-R45;citation_volume=13;citation_year=2012;citation_issue=6;citation_pmid=22703977;citation_doi=10.1186/gb-2012-13-6-r45" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=R. Norel;citation_author=J. J. Rice;citation_author=G. Stolovitzky;citation_title=The self-assessment trap: can we all be better than average?;citation_pages=537-537;citation_volume=7;citation_year=2011;citation_issue=1;citation_pmid=21988833;citation_doi=10.1038/msb.2011.70" />
<meta name="citation_reference" content="citation_journal_title=Biology (Basel);citation_author=A. Platzer;citation_author=V. Nizhynska;citation_author=Q. Long;citation_title=TE-Locate: a tool to locate and group transposable element occurrences using paired-end next-generation sequencing data.;citation_pages=395-410;citation_volume=1;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Qi;citation_author=K. Daily;citation_author=K. Nguyen;citation_author=H. Wang;citation_author=D. Mayhew;citation_title=Retrotransposon profiling of RNA polymerase III initiation sites.;citation_volume=22;citation_year=2012;citation_pmid=22287102;citation_doi=10.1101/gr.131219.111" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_author=L. Quadrana;citation_author=A. B. Silveira;citation_author=G. F. Mayhew;citation_author=C. LeBlanc;citation_author=R. A. Martienssen;citation_title=The Arabidopsis thaliana mobilome and its impact at the species level.;citation_pages=e15716;citation_volume=5;citation_year=2016;citation_pmid=27258693" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. R. Quinlan;citation_author=R. A. Clark;citation_author=S. Sokolova;citation_author=M. L. Leibowitz;citation_author=Y. Zhang;citation_title=Genome-wide mapping and assembly of structural variant breakpoints in the mouse genome.;citation_volume=20;citation_year=2010;citation_pmid=20308636;citation_doi=10.1101/gr.102970.109" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Rahman;citation_author=G. W. Chirn;citation_author=A. Kanodia;citation_author=Y. A. Sytnikova;citation_author=B. Brembs;citation_title=Unique transposon landscapes are pervasive across Drosophila melanogaster genomes.;citation_volume=43;citation_year=2015;citation_pmid=26578579;citation_doi=10.1093/nar/gkv1193" />
<meta name="citation_reference" content="citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. A. Rinckel;citation_author=D. J. Garfinkel;citation_title=Influences of Histone Stoichiometry on the Target Site Preference of Retrotransposons Ty1 and Ty2 in Saccharomyces cerevisiae;citation_pages=761-776;citation_volume=142;citation_year=1996;citation_issue=3;citation_pmid=8849886" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_title=Benchmarking computational tools for polymorphic transposable element detection.;citation_author=L. Rishishwar;citation_author=L. Mariño-Ramírez;citation_author=I. K. Jordan;citation_pages=bbw072;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. M. C. Robb;citation_author=L. Lu;citation_author=E. Valencia;citation_author=J. M. Burnette;citation_author=Y. Okumoto;citation_title=The use of RelocaTE and unassembled short reads to produce high-resolution snapshots of transposable element generated diversity in rice.;citation_pages=949-957;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genome Biology and Evolution;citation_journal_abbrev=Genome Biol Evol;citation_author=T. B. Sackton;citation_author=R. J. Kulathinal;citation_author=C. M. Bergman;citation_author=A. R. Quinlan;citation_author=E. B. Dopman;citation_title=Population genomic inferences from sparse high-throughput sequencing of two populations of Drosophila melanogaster.;citation_pages=449-465;citation_volume=1;citation_year=2009;citation_pmid=20333214" />
<meta name="citation_reference" content="citation_journal_title=Trop. Plant Biol.;citation_author=S. Saha;citation_author=S. Bridges;citation_author=Z. V. Magbanua;citation_author=D. G. Peterson;citation_title=Computational approaches and tools used in identification of dispersed repetitive DNA sequences.;citation_pages=85-96;citation_volume=1;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=G. S. C. Slater;citation_author=E. Birney;citation_title=Automated generation of heuristics for biological sequence comparison.;citation_pages=31-31;citation_volume=6;citation_year=2005;citation_pmid=15713233;citation_doi=10.1186/1471-2105-6-31" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. E. Stajich;citation_author=D. Block;citation_author=K. Boulez;citation_author=S. E. Brenner;citation_author=S. A. Chervitz;citation_title=The Bioperl Toolkit: Perl Modules for the Life Sciences;citation_pages=1611-1618;citation_volume=12;citation_year=2002;citation_issue=10;citation_pmid=12368254;citation_doi=10.1101/gr.361602" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. K. Strope;citation_author=D. A. Skelly;citation_author=S. G. Kozmin;citation_author=G. Mahadevan;citation_author=E. A. Stone;citation_title=The 100-genomes strains, an S. cerevisiae resource that illuminates its natural phenotypic and genotypic variation and emergence as an opportunistic pathogen.;citation_volume=25;citation_year=2015;citation_pmid=25840857;citation_doi=10.1101/gr.185538.114" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_author=T. Stuart;citation_author=S. R. Eichten;citation_author=J. Cahn;citation_author=Y. V. Karpievitch;citation_author=J. O. Borevitz;citation_title=Population scale mapping of transposable element diversity reveals links to gene regulation and epigenomic variation.;citation_pages=e20777;citation_volume=5;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=P. H. Sudmant;citation_author=T. Rausch;citation_author=E. J. Gardner;citation_author=R. E. Handsaker;citation_author=A. Abyzov;citation_title=An integrated map of structural variation in 2,504 human genomes.;citation_pages=75-81;citation_volume=526;citation_year=2015;citation_pmid=26432246;citation_doi=10.1038/nature15394" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. T. Thung;citation_author=J. de Ligt;citation_author=L. E. Vissers;citation_author=M. Steehouwer;citation_author=M. Kroon;citation_title=Mobster: accurate detection of mobile element insertions in next generation sequencing data.;citation_pages=488;citation_volume=15;citation_year=2014;citation_pmid=25348035;citation_doi=10.1186/s13059-014-0488-x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Wu;citation_author=W.-P. Lee;citation_author=A. Ward;citation_author=J. A. Walker;citation_author=M. K. Konkel;citation_title=Tangram: a comprehensive toolbox for mobile element insertion detection.;citation_pages=795;citation_volume=15;citation_year=2014;citation_pmid=25228379;citation_doi=10.1186/1471-2164-15-795" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Zhuang;citation_author=J. Wang;citation_author=W. Theurkauf;citation_author=Z. Weng;citation_title=TEMP: a computational method for analyzing transposable element polymorphism in populations.;citation_volume=42;citation_year=2014;citation_pmid=24753423;citation_doi=10.1093/nar/gku323" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Zou;citation_author=D. A. Wright;citation_author=D. F. Voytas;citation_title=The Saccharomyces Ty5 retrotransposon family is associated with origins of DNA replication at the telomeres and the silent mating locus HMR.;citation_pages=920-924;citation_volume=92;citation_year=1995;citation_issue=3;citation_pmid=7846079;citation_doi=10.1073/pnas.92.3.920" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. Zou;citation_author=N. Ke;citation_author=J. M. Kim;citation_author=D. F. Voytas;citation_title=The Saccharomyces retrotransposon Ty5 integrates preferentially into regions of silent chromatin at the telomeres and mating loci.;citation_pages=634-645;citation_volume=10;citation_year=1996;citation_issue=5;citation_pmid=8598292;citation_doi=10.1101/gad.10.5.634" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
<meta name="twitter:description" content="Transposable element (TE) insertions are among the most challenging types of variants to detect in genomic data because of their repetitive nature and complex mechanisms of replication . Nevertheless, the recent availability of large resequencing data sets has spurred the development of many new methods to detect TE insertions in whole-genome shotgun sequences. Here we report an integrated bioinformatics pipeline for the detection of TE insertions in whole-genome shotgun data, called McClintock (&lt;https://github.com/bergmanlab/mcclintock&gt;), which automatically runs and standardizes output for multiple TE detection methods. We demonstrate the utility of McClintock by evaluating six TE detection methods using simulated and real genome data from the model microbial eukaryote, Saccharomyces cerevisiae . We find substantial variation among McClintock component methods in their ability to detect nonreference TEs in the yeast genome, but show that nonreference TEs at nearly all biologically realistic locations can be detected in simulated data by combining multiple methods that use split-read and read-pair evidence. In general, our results reveal that split-read methods detect fewer nonreference TE insertions than read-pair methods, but generally have much higher positional accuracy. Analysis of a large sample of real yeast genomes reveals that most McClintock component methods can recover known aspects of TE biology in yeast such as the transpositional activity status of families, target preferences, and target site duplication structure, albeit with varying levels of accuracy. Our work provides a general framework for integrating and analyzing results from multiple TE detection methods, as well as useful guidance for researchers studying TEs in yeast resequencing data." />
<meta name="og-title" property="og:title" content="McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/8/2763" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Transposable element (TE) insertions are among the most challenging types of variants to detect in genomic data because of their repetitive nature and complex mechanisms of replication . Nevertheless, the recent availability of large resequencing data sets has spurred the development of many new methods to detect TE insertions in whole-genome shotgun sequences. Here we report an integrated bioinformatics pipeline for the detection of TE insertions in whole-genome shotgun data, called McClintock (&lt;https://github.com/bergmanlab/mcclintock&gt;), which automatically runs and standardizes output for multiple TE detection methods. We demonstrate the utility of McClintock by evaluating six TE detection methods using simulated and real genome data from the model microbial eukaryote, Saccharomyces cerevisiae . We find substantial variation among McClintock component methods in their ability to detect nonreference TEs in the yeast genome, but show that nonreference TEs at nearly all biologically realistic locations can be detected in simulated data by combining multiple methods that use split-read and read-pair evidence. In general, our results reveal that split-read methods detect fewer nonreference TE insertions than read-pair methods, but generally have much higher positional accuracy. Analysis of a large sample of real yeast genomes reveals that most McClintock component methods can recover known aspects of TE biology in yeast such as the transpositional activity status of families, target preferences, and target site duplication structure, albeit with varying levels of accuracy. Our work provides a general framework for integrating and analyzing results from multiple TE detection methods, as well as useful guidance for researchers studying TEs in yeast resequencing data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8.cover-source.jpg" />
    <title>McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"erXQaCHS7JxmdmBEotI35uvtSlCRHvydJklUVfJL9XI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"485854","apath":"\/ggg\/7\/8\/2763.atom","pisa":"ggg;7\/8\/2763","ac":{"\/ggg\/7\/8\/2763.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/8\/2763.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2763"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/8\/2763"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/8\/2763.supplemental":true,"\/content\/7\/8\/2763":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022emRmc0j5KjouV0dngDjiBW\\\\\\\/9plwCo\\\\\\\/igcDv1PEVrK8SGM+QIMD1q5ON3MNRf7k\\\\\\\/cPjrJj92hV9V8A1+0586COKZy6kLdQBFO4qihLDXnkW+CtKUEriEjQLkn1AvOsRHUfHL\\\\\\\/DJ8St9kBG2vpDaY41XuCAYGq\\\\\\\/tf8mHFXdISGZ1RlrEHbMDYwVU3rIwQ\\\\\\\/jS3UZm7UiSpkeKjkUFlqaprQDczbl4HCRweGbZUgYH4O1zjWIMYBHvT1t2rhoeNE7qnSoAZU2+sOi94gevisaaKJvtimVZV25yeDnOvIi7cg\\\\\\\/+dqTrvvnmZUdfMeQiSTFiIWLClg16YCsWg53KL04zRt3Zr3EJHZ6G0uWublqhUNPEwb6ztwYEGFpqxAL207HDls2YoSm6pdVsR89ThOMtwIgZeKwlAmuLDdHRMoZ6Jue9Fs\\\\\\\/wLyGWFCiUMahJUCKZL3GFBkdPG3ohIBg8WKOHOWK5XQRCDrFLjMpoGJ9oqoDIGnNFcD2Rgdz8jfytYvUbY4kApZjqpQZTnTvnEH9rJ+MnQtZQDzb711KRlqUlmcxX8nWgRXCYHU3EUCGmE806o\\\\\\\/mp0Om9EuvHFi7scxaiCyKTI5sWkJ6JQ1t3WM60CShSNd+OZREdZ1X7whY5RzNio\\\\\\\/Pb566nNBRxn3176eyXOQJFnPDr3T03iwXvPQ2XzX+oOykuGTFI26RRAdb5WOOmjgnDGLnkz98+tW0KciYy21KotF+ntYdf0bpRut7+i702mhOM4bWcmbxWg87ygU\\\\\\\/6nRr2Fxk9\\\\\\\/2SjR0TOcpIkPbxY6QynPr8Pl5ieNxvtOzTYRsZS9fFXkS1f9\\\\\\\/gdf2KKeL+DiUnmv++QJYAi\\\\\\\/hSBwNrWoXa1aph1t8lz\\\\\\\/2DsHpB8KS1IYLSJst9YgIgHg2MHPxOAkGm9\\\\\\\/DbAzYrvDQxrPZ6ljhk0JfapfZ0WWyalvmG5aocXPSCSgyQrDsK4CIwcQC5IAlJInvJ6L5WnySdjZ1ZZsbpZZQnYFjx8An\\\\\\\/BTDWhH7gXqVYh9Lr62+eNT6smrVoyqqX3JOZr4TQQakBXzdAjz3g6ZsTQZeoAeUW5Ckj7js9h+qDeRpOHGG3nMRLxussvHJPyXrIsaKTNPmf59WCoCvnGSY7\\\\\\\/RvWJEG1m0f6ne89Kz\\\\\\\/kfRpOND2xZ6lLW\\\\\\\/LBFlAu\\\\\\\/kjlFc75awBLkq\\\\\\\/nTLViGDCJySz3aYFj33r1F0rbSVFNbsM6XiFMohwgtM4xcYOuXrT6oK05IMw27GDHCGj6yPSrZiJeWFjvW7YUL3yTrrJ2e4e3GYwP15HonYx\\\u0022,\\\u0022iv\\\u0022:\\\u00227+PdAE0iMC9hYuzntYzEaQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022563b85edcc6bf19e40a3de512b488c0be1372c2b728b7769a291a69023c06b75\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WXo5UD9Z32XCg0cGyUvPujJCgL7sg815xRZTEYcnHt3uQcu13EyZi+aL366Un9bL6nyP\\\\\\\/sZYHTN9VKlDNDn326HQoo3BVBePP0naq\\\\\\\/pDuvUvJl7pT37khHZYLXhMD2CZp47knPrSYPrVXlyPZlv5PUEnj\\\\\\\/JlfGD4EcTB\\\\\\\/N0PGXD1zGvsObbjnWa9rcootbDz6MMlqqLZ+2cGE55YmGhybMZVbAsp1inMKOiJR+xoM71bdVOB24vAPl\\\\\\\/EGdgJ60brC4Ra5TXE+4Yz4mdqN+1iYFSrKqtkd9zVT0IL+9a+aIVhih6pOQvNqo+ErFJYDAxmF3LIKbMnJevZ+a7+lpeJknaOBHMBmo\\\\\\\/rdX92GcdvriwWFBWhqi+WO24E+DTVL+Zp\\\\\\\/WfWNwjDmzZBF8h8U6Qx2qsRIIwUSZRjTqV3jTu1Ymb81t02C63+uJnxSa6GHJ7jWz9o8VHcVyPG+bivxVR5u6oX6HYaeXw6W4SufcQ8cofidgCiOZOOb52qxXbmWf0Tsz++jBGC9nQi2fdUMX+lP\\\\\\\/ADku5al2oBeNksWciRvHQWikzYJzOXvAlNBbVK3tDam77UWlmDnTRS5Nzc50CCVYPUzKANjZRt3iBxrYrAhzQCdTIEbwskFpTxOC\\\\\\\/nxwAMgBU1kv0F\\\\\\\/PpFSXBEz7n1T4DYNWJOcCfNA06UjZhcpQ9fQ8JCsRIr2QS0296QlKT5yquFW2RTGX7ihcjDUs89U9s0NOgwZx5VmVOyUs+PCJpUmTQfxNWj+ueJVPAKiqFpw4PonnvxcG0CrVxQJrRASSiGkgn\\\\\\\/DPNPuAey5A2nu3xs3Lsly4jbUdguO9pnSxSOOsuEnKDf7pbtxf539wzeCrPBHxH8KQp57OgG6fiaHOrf85S7IFNr79y00B9Jq8tS36CB6pf98oM0e9KW5RD6NETK4YK3CLyLdZaDNC7pqfDUp4m6Ja1qdWbZVE3BoGu4nsCDF5EKslsnGd5qaCO+GVzP\\\\\\\/Yb7FpB5b6RKjKaEJTG6q\\\\\\\/HlOoducpOudDal5IgwK7Ues8LwSDbDnhKAzF\\\\\\\/rQkYVNwHUE1QbSNu2BzzwOHLniqrBpBcCrTZl+97pJ45Al5ZED0ZuAixs9wt3BuLl67mxzkIIfdKtI7yJ3fS1xe4vHYqqyJtxLe1Cd1PzK5Ge3ubgYWgCTe6zqlIC3HbjGpqijVGy05g\\\\\\\/TK48X5gCz4MznO+BoVCEbLNWDgo5DE9SZF\\\\\\\/HlqJaI5vn0kVcto9XVCnc6L4UVg74zDZ6mRYzpleR+agukPNUlfx4wPM6Nb581yso86K4HXQ58TDXKrwqLnFH0G5HYZ1AZEfkzTFWH0Tfl\\\\\\\/CTx21nWVg8Ox\\\\\\\/BYNI7ashhfG8HdhCwCjFUqscArY8H44l6OchFgP4lZx2P0ZYcqpfWWcdXDFXa7aDiC10KKhW3LH4PeKk1aLunxS4xEDCI2psxAilQqoDesngw\\\\\\\/mUAQTnD6s9vJOYBhfOp2GSAzna1IGTic3Ehj6ApenntM8cQVI+PJG2gZfoCXIEik4TOJEkpe7k\\\\\\\/zCngD8Gtx5BUhl1AFQGXwbDxj18Zp34PdO\\\\\\\/mO+OU5\\\\\\\/XE5ahqRDRguNo6+WEJAs0+713iNzVKg6vsdFNwuXQFbVtwd3K8qdsUYJWrhNoTDbFZn\\\\\\\/VechlRfFwe0wWZLYIYN4nYIbseu+pUy8zXYFbHqi4cuIwXhxV\\\\\\\/UupbCONj\\\\\\\/jSFIZvo8w8OnQmBxY4ti6NTACEUgVjCWC9qCdXJCX0n2riRUGEtB0we\\\\\\\/hOUkAWAvjra2zlTphTri+b1MG4IYnCARnNArAY9mjyEeWAYj45StvLVJyKJuZezjXMBisr5U03ALMnUFXyDd5m3NzHkOLQUtQvg5eiiREwte\\\\\\\/cZ82k4koJZ8DxKkcHschh\\\\\\\/cc\\\\\\\/KDG6CCAw+ax\\\\\\\/5zBi6tfOlg3TA2\\\\\\\/UEvzDb4cbIloUP0vkMoPsrCv4\\\\\\\/lSqLlUxMUTDx\\\\\\\/5pdu2OvGiLS1OT3TzD2W4KCeuW4woM3ezyTC8SUyS57o2uQQynylwZPKeh8lrA5h33vNEStmVvPt\\\\\\\/QSDqFbVA9yC76si+S8QzRGfv6ivKPM\\\\\\\/aK4sXR+VcW07gAkp1L+kHHTBBy8snnF8iXeVP7YTnmqCjqARNBBr\\\\\\\/a48dnaNWNfoWShGwcsbPcufOtcWa2SOaXU9lRr4rYVc53nFVlruJb0fesTuNZSvP0g0yPAK8qcUWWg1fI87KBGFmoBYM2liGL6qWNohuk3X7BYVhjkVmkOKDBTSpIev6H9TbMv+VV4yWLKfn1qZp+10uDUsdKi6txH4w5trFqx1AMbl+RY4pEIxRHuCwhu9pVAu6qYQ19QwvzF3kXD9mkeZJ1hjVFWQXvd6MF00XvgIzUO9vS7s+Hdiq547Un94L5GfS4unLKU2rzh9AkosE0NJ3uCPnc4OQRKASl70TEzrxTDgzeN6x+fZkOHVm3hKAHbdla3PyGaFYlB+SDeSgSlGmk9FSUIr+paNGrHIG8OiToNOtHdTPCyb1dNIyHLrjdY0V\\\\\\\/B646oP8cNtCNhh2u+rVYuHRiwthu8cHFSP9qdmdCO2T1oiKuZeEkIdnJ1To6m7eyFRGz32dU4kx2zWGR06M7grkMp8L+LrOX86CK1OzGvHGFDyOLidGw61Yv\\\\\\\/sV+b2wSp4QNZLISbpdWNd\\\\\\\/4ycSo8AmzGcO8F6dsqgYznyX5Nzkkxz0j7kmrORr5+O37siuINNk5G9IJSQOl1Yv0HY+BRql2mOWj+ppr7rD2TsD4Y9XnFiz4Fm04ajp+LgxsDiR9xb3K1mRVYrNaVIHwgyjBihPS7cAqhL5beydmaB9qxjwULjHudbOctbBDZtyrJ77ksQh0FDrTu+Nj5PvV\\\\\\\/GxJ1ye1H6NRIZkRwFeiVRkGUWXZaB\\\\\\\/zmU7rF0gQhPUep9q\\\\\\\/AnKDBFjjAsOxXOY3OhgOMpkWL1TJAzWh285qdt9FrO\\\\\\\/5yuZ7zK6Oie73l65xZSg3lWCVfN7Maz61A18m5QL5IOq1KUzNHIyU\\\\\\\/serhYSKPLFwr7t67UIcWPGb5xlF4jCHj3nDGoxrzFIV2CFJZBeHfXtrY+8Dg9z2oIVtXqpQf0NDpxIm+6Ex0ykgMHNfKD5mvZeNMkaNokg89emnTJ4r50nPDjbmqiQFIcKQNaEyDY9nNQf8aQWJAmjWO5Ec7Qh8IqzdyI\\\\\\\/JaMMzGHlcDnhxAUOqToMfVs71oKW7FnjftWfLu5orPejc1X1B+EtmRJGHuOHjuSesbPnvs7G3jBcRhLxSO+RpSLGBZhijbUQ2hMxcgfjqoHUHJrCsr6BbPkzRUIFYGiVgYVeMc+oyqRUrQ+W5qzA5581xbLMAHd\\\\\\\/U\\\\\\\/Cwp\\\\\\\/XICYuCuhAVkHXrxrol2anj6qBH39jx60Wqa9igE75MiHgfXCIHXmhWomkKnXfFniwHgTqElpASw5ELYx+h5lDkpK8ZisXChl+unRYrwuHDbvtSm0iRPrNVllirnOnor1beT59hCCMzK1KNfamN09Q+kqNDOF0hBYrwVlVhGi4hp3HWJ9mKD1PmkWVU\\\\\\\/ZmRCxJEaknxnC1ys94NSF5N2prcXU1jSMrjktt2GiHqF4AZ\\\\\\\/Jh3YHgwexHgGEIW+a4ThspjM7BU\\\\\\\/bAtBjpfzbT5yYZxL\\\\\\\/quDup+twEGGl8iWoV1yVYQ3bDywd3mP\\\\\\\/5ktrefe6zBgfiGPakuy9ykgoDi49lYYqehy5Fkb0Yfr0tTYmb2uBhbmg6XnB23sfSf3MSgJRJkXSRzZxVpgC7LdBCCSKHWTcphZTHD3AM88h9JuDKagJmOpWa5glAxdJ+EB0UczagXPv6n0mwl1fOUog7mZQ7OA4Q9Cj3Ng7bVojmuAQd3igZvpSkf\\\\\\\/vbzTIgKKpXRUM2JMF7NuM29JfOGS7EjQKshu3+EF+0HTQROpy3Qj\\\\\\\/hH2g7zzbZY2IGwj00y0mVHWU76Girv1nnwma1v+tpSh\\\\\\\/591JVffeIIuuaYkXajiH2CdOiX4mnAasH3EddLcFkKJJfzPaFJI2eMoE3\\\\\\\/Zs+QTngQVtyQhXuCayjn6FclSS712tYMFyr0MqZ0AlyaA7FaR9BSQh3PMWUV90L1OOmEoDxp+GQIkIwIj1zpXI6yl42yEl\\\\\\\/lwPadSgZY+1AGn4FQvqiOARh8lqz5SGj498DjRGAXxawvhP8GUjR\\\\\\\/xTdm\\\\\\\/evu9kzMW4NuaUi8rngZCnh9uQmG4PoDMUFapMbFZGG1q\\\\\\\/wshxg9lQQLOUdF0H+vBRR0tHVNFPHN7FBbmw5+pXwcSHmxDj4gywQaB\\\\\\\/xpfiiZy4zvQYPRKR6bHbTvdUTPhrNCwNII04dqj9vxhKcPsenJiYN7cnItEkvr0DYvXvDJUtJfCnYA77CNHJ1qAHESZ4KdoalpD6qwmAvjuStcJdsGCoPh3I6L7NLVFR7ncQq09K2NBfNQ4pMzZ2ymo4UYiVrpZ618zwvxH5m0cIyFmJ4yw115PpC8rwNHmFbAGRel1Xyez8oPnApo3\\\\\\\/LAhz33+X1+njDIgeZGJujhrD8g2KZYfc6aZyqp8X7Jt6WiIgc2FpUVWAmoucplEBowe8uqrk\\\\\\\/cyJnbEnJX5hDkt+6QeMF7aFpBtRg2Ajdl+12V4t3TqjYllNgRGzJQiPuPNpQGS+nWzZKaUA3Kozu1OoQFkF9\\\\\\\/rfwrSzbHujpnNSMCWUySbXkN5TP5eENgJFstdgu9Goki4cC\\\\\\\/MjL9E5aOMaFh29WE9NNtuMsCt5MGkC\\\\\\\/xk9OPvpJHGLV8rmiKwN+bcCPO3Jmm1aYcjelc0+LcIXRtlWSranWrwWZP0az6UvGMPp6m+Yh44FB\\\\\\\/XK3RF\\\\\\\/HM17\\\\\\\/uoRgZkApj\\\\\\\/K\\\\\\\/19faSUqd5HxBJQly9ZR+tRB5SOQ6A3jcv6HHmNO3ZfoG1Ud+j3jw2mLVz90CozzLlIJ\\\\\\\/wiflD2Ur9ypHGwi503qd7lzaCePeIkdyHRjJEkd2zylS9O7rskOlvGreZbEfvzqo\\\\\\\/cZKOtMOwGS7GUtyddQGK+nAQlNjsP2Ku23URDY1O8HTZBVs+LWJZOZ0\\\\\\\/JbIha1YinuN5+IagI7MPyQMhq0sidI+P++pKbv906KdhdJPr18+h\\\\\\\/ZQcIdqnaxfaWrDaikN2Ufz\\\\\\\/vXoaXGgIMSvkqiciJ64taYst3LMHVRZ7+XfjfkRDuNKFb51v8L\\\\\\\/G6sjvJ9bAz4EGARo1xexHdf+cw0Faf+N2ws+1hflkYTgjfpd2QavIvIItSLcw4tVP3KYqZ7ryJeAZ45d50SuFCxbJxZkA6GH+x3iYkZR1wHZIWVFoD+ndqQVfnmRNsC7rQEPB6dILw2fx7\\\\\\\/\\\\\\\/9REd0zfMJpbpgI7z1MnANtcZm+AsMztw2UvPm44d5qNc3Zj6599K+0er\\\\\\\/weXqQny5Rycx0lCFoNTufeZnaggq6G59xvsZctN+8K3MVnsqJZmGASkbwTtOCL6AdkPTM96mJeQOH7d5TKQrV4FIvMjsRWp+oNnHeBKZVyIhZDttYCCOh\\\\\\\/PnEcQDODMU7KXH4WhnYQxPPMv3nWl4zdLxf1dqc1sp02ASU5SlhKw0+OjAzDIAmHVpSlZPSGN0\\\\\\\/b0zGu\\\\\\\/WOpNhDw2Y3qgsy3B6NUIp\\\\\\\/VsEUmfQ\\\\\\\/hBWijFmymc20tMwTYg2zJg8U8xnJqInIWKPQ5eAkxCxB3CQ7e019oYErE41MrVB5pJHK9rxP4MvF6TPsO5\\\\\\\/rfMkEJ1FakXMSRDmYYs9yFAX9ig9mSOGtMBdY6EnwEdJVbQQNIhYwQVqVGNCkzVF1WLWTE+bi47UgnpoGeBx0hp\\\\\\\/uLywpTX5\\\\\\\/qizyPSCoj6DlO1Ng75AL6px2BKITY5Q\\\\\\\/WKA5dC9n3xwcBbmQHYWHZYzuovhyxFyYHGKHNqkIxCLcrQ67Br\\\\\\\/6fD5N633\\\\\\\/V45z2qyhwWi1beKLik5+PTDuD1atMKCln2jZUKPUCWsIa1a\\\\\\\/vWXkSa124zInvHdwxtSzFUonkXWnOhoaJ89qrrlWF7\\\\\\\/H2whDk69Cz4gBLwfAAf8ftpUSKd1Y6\\\\\\\/T\\\\\\\/+K8Kd62jD3\\\\\\\/E5UBZr1LmOLInkcS4iLkrZ4dYOKeIdAl9qKo2QwyrRX5p0yigelnP22+4XSuv9aGaDwu4\\\\\\\/YqSdRtlHoRRkFTp5A7HVJLnxx3C7pCVsJZttrm1AFyphs3tj3FamLg+ACfDWyqQerXEBoGEBW2ABNygmoio+I9NbASAwvwDVdb6MSJTl3rDjGMc0OH+KMyDr+3G11eVMiutlxjuw8a64cttf5VuE9SuMKPc1MSZmN9Wx+iRMjgYUvFkpPHU9ikxFckmHgl0hq2YYcidn5539VMNYObDYK03cEqHHLV9lN25n2Za+PmUeySivFTp4QeJPA8uQd81jhJrgdnAsVfq2AfUmADb57rcm7L5nlL9kGux35LlgYQClJcdPOk4QwM17wrUEFVZhYi5ScNBb4shdWwNlDe9PlXkbqsWXO30jfaCVRHbB0ZeY4szH0OY6\\\\\\\/ykVZcG896gHT0hRX8wkGqExdEnbJ67CSLH3qsLV046zldR+HQlhDIHDGYQev24TEc+1T817yB6Rk0S6X8yf3klk6OAEeBajsXgyBdh67YMjDPK9f3eZTA4ZYM1ekcvqysyfdZJram4Od1HPb1B1gjEqFdA\\\\\\\/40vb2XdvKGnEMSkjYNp9DO9Z83wXarQssIP6XBfpUAJrtdT61cM0X2c3a5eg1LptGQtjS+D3jWj97x1VbRo9\\\\\\\/cK1KeySUcObKYcJ92nfNGO7lUFzl5n7NvEMdKi0z1aU29ZSyjttoD2FSq\\\\\\\/vPkwyAJ0yaDR5ip7EEx4q4AJIzyKwzOAjBP15y1eHyNZOzQohW+V38DecX6u\\\\\\\/1ld7ZEC\\\\\\\/qii1emRkp1P0h3KY1e5UXRrhS5ETyJDUGl4g2JQZjSc6RnuV8MgktdpJZi8Bu+9N8SiZ6snOzIjfoNQRjAkKOeFYs+QKeLz07\\\\\\\/+TDfpHN\\\\\\\/jfjp8PtVs03tz8qIvqbzz\\\\\\\/cJVOIz163CzQjIgNoKG5MXST22Da47XqAS6Iaa7Yaqob3h7OHn7bXN5jLfgG6zRmFTkHVL9mAI0WI7Gv5\\\\\\\/nHwLl20qJO44NJek+MgooMCx32NQAo1Vi+xrFsZax+OLw0BoKhs+sCFCqfeO2iL8J1jCxTQ\\\\\\\/bdp5bMq9K+G4YOmSwdMiJXBqbyzDlaQAOZuYosD+bzCKwlhPv2985J+ZYpneb+Sf5\\\\\\\/RcGwWhpf8xeZ3xvQPZVZZX9FbeDiTfzXylNgeSL1omIPPffQpQYQ9BvPyftzM7ZtbBK\\\\\\\/PzUXPX\\\\\\\/47jpY+KIHAMSZykH5iCoxQQ0uPAWJJeeb6zV4+tI9HkNypw+M1Z\\\\\\\/UDEaEYGdqpYhzdYrsdllXzsldO8oAPkH0o+ie7s0tUbPVe7YtqUiPdvn5L9X99fkfIIeRtyZ0wiRjp+G0zx8pcqTIT7DNF4fkYJNIoBv9LfoXVd+jKIPgTqpeZfKqlqBcFm4MNsXopMOOm9jXbVZUSdlFPq\\\\\\\/uyAUlVHGSNJcGvJo\\\\\\\/Ehlr3zOCBMXBv4ttWvz+BxzngkHcT+hJ9okbQR6AmsWUMLCpy9Acri7T5+ENN1jSlHbnC6z22uwjyUZQDryYzE3gboXSdZcu5G9FUaKxdHlYURiNPlF8MoN33Q1a4wCCP4C4q9pfjJUf0Hi+htJlbTCThBSA8bLiXlEVJoGtBOlj+AbgFF\\\\\\\/JqtkgZCIoIm\\\\\\\/4cQHtcBaw6XjJKgYNf5kZViJW2H4i55IHLXJUxD5TiXieqBj\\\\\\\/Ke6103qAiwb8\\\\\\\/opppr3mflfBz\\\\\\\/lHDjCfB+e5AdZnwbvPWaABLYY+RKtuFBuErEKrSnIrWuAj1VwqAMHIJVDGgDdBEGiSs0caTbzmPeTUwJ75DhGqMCswZEBeqzAdzoTKcElgh9tIaAisx\\\\\\\/iiwtNcawM5l8fIdIlH+jejF4U5yRz63qspr0lhv7KRhISgf6F6C2vV\\\\\\\/0uLqqhRLCTLjxUiZANpRngyXL\\\\\\\/R5+5rixecx0ev5LoMCRKDbvnlG\\\\\\\/2d82JiRodZnsm1hmgSsZI08XKbEFoM\\\\\\\/bgDzs7PTAymVRRNIKzNnrwiow5lyWIuQCzc73CdInE4ZPgK02ZtWvdkASEzc8G5AwM5sZtZ9GRCfkVRQyYoKnvkEW\\\\\\\/LuSsduvIRjDpQcmaBJqCUOEqNh56f3UZ1RZVE6tIGnnncQW7QeYOcQ6FNihMEM\\\\\\\/zlgnLG4zMd57FRwKr6bNSjKlpTl0kuyTmFPuCsWwmOpr3l3Vg1uZtvTbibORlfJElri3LF0aieeiE+JkhDfF3Cbe3FP\\\\\\\/9OzGuzE2SCQH1HFGtirZZlipzHJr77XwW8Jd6WgoyynLzchztSr8Ilf1jrihrFB4bTEFlilFDck+V4uahdADlBT8pJSXnVO5DirgnpxQzqft5PiRVfEu06mssMY\\\\\\\/rtoe65Wlixpq8n+pMjIC2gY+6QPtx+WnpU5DUBc+FS4a9x9lotbnWn\\\\\\\/cjF7aM3TxTYvBIY90j6jUhy10yoCbIKOlFcYOrgUYZz2EAnWt6ZU8AFSXorypukTxaIaMOI0cgh2U3J0oyB0HvdsNK50triuc8QRTW5Srpg5XwCyHI7+clK1h8tbo85BefnDXBE3O9dbPRIBOnslID49oIzYU482E2UzZ1Ton6aPGxaBV3+MOeGLp0wwAI\\\\\\\/ZjyLzvZ7ZEi1LGlgj5e5VCC8T\\\\\\\/T5YzclgYcBvgiviEpTc2o18y2EsSb+s3HNdA0IYXI32t8HaVqyH\\\\\\\/SH+hLNqup30xuB\\\\\\\/Yb9J8gy8ghtTvg\\\\\\\/crQStuXtQIcyJXHmbj0QosgXkqN+R1JdIvi9Z7rnvK94453Tmfp46nqcAd7b1VrxWJ4J4Df1UeTuuQgEQc3aLxrDNntLxYuYTq57PSlCL8g9kAFcdWC43zDeHjP6L+cBn+\\\\\\\/4P17f\\\\\\\/OB0+XL1h1hswAfTs0cwHqgptouPVfnT1edEA\\\\\\\/UIUKC03xUBZdul30PT+Dq4dUrQR9cLacNH6pUhz6t53tv+0Qnnt9fQx4K55S3Kfb4CQWvSmbVEDRt0G9ES7FGwkGmA6iV5XDEYa0+wdApwsIzQ4YFVq\\\\\\\/LjBZu8P4bt9VSIx09BkHb6+DZZLmfr38gEaOWgAW+Ze2c+kbIO+8RXpfAAkteRNPkJE0N+Ro2z1wrsnLMYyJUUeY4zjyC0+Pg1XF31IbD3T2N6MUmvtzOuzb540sgrGAjlEsTlnat5H5WOo244M42hasjsQw\\\\\\\/2LRNyK8nYUJCWF3GZ6Z97Xhyjo4UpnmBAG37H7zKS2qyzLjZYjjcaLOKZzxVR3gkwag0H9eZPWazYYl5DKRuKcocZJGMKK7QXlV5XulZKXg5U2HNHLSuUkJDnt2ZdtR9QuTTS6xfogsuvv+66x8CtmQW\\\\\\\/vBFkiC0lERfXlh0iuPC0ljNherPbOLVHLlUW25XdR2XLYNnnxmoVudpVRGXF2vRWt9\\\\\\\/0K78sOi6PTdBFYTPupyXlHQseQELIFluWkRgBPEYFr0kXRA3uiMeDTholbQu4mcmpaIJt\\\\\\\/5y7bLi6hdRzKG3CF4Hh1l7I9PRDkQH\\\\\\\/LC7pXq6QzY6pVS5WNaGE\\\\\\\/AopY4rPE7Jxi7TAnI40QH2wcoQNYdXs1QbeNX+xgSkQkGak6GuXDtXNDNeAHx4qKcG4kyzj5Y4OOhXseBHPkf+Se2QDriQCAl+1Uhm9pZEE4NVTYPQNAxWi22R\\\\\\\/yPWcs4GnmT9Qk0F93TWwjWlWdjUGA+KYrmxkRMIdR2vmsvxakFoybNKhlpiZv1cNVWDJd1oZiY0jXY1NhiMJq5kxsei3NUYoG0QTUNxotzeWLwTJ4vaFhwgxDqeBk3NJ5+TmGYmqrYAvoTEDquUXrSQ4hroLVnHjMZlFOLgwzPsPlLJJlK6XJjQaz\\\\\\\/Yx4nDNnl62tLHIA9DaNzeTbbGPBoZrMklQZQle9mdSNNbFNS2csTm\\\\\\\/6EQerN26q5bRGONpgA4IoORIY1lpxoj6BoT6m10Ahsoo0NYICPL07AaIjjIHvVlKDCfp48wcd9bTOXjLU4ERaqzMTkcWahPXdLTRyVVT+mG9iiN5uEgMR8Zi2xCOQ\\\\\\\/0ET6qw1vzA\\\\\\\/w8jlal5vSK\\\\\\\/X4r7nFfgw993a5g1dp8G6LOaIQCEWq2T9q1nhJdxN5WGi21pxBBDrViYcro6BZxVT0\\\\\\\/d9MQQR3kiwaywHQtww352aMb3+phm68vF8b8XmzT6+BiPY\\\\\\\/SOIlMv4HSvfBDake8rcvfd6cq6xkOZmP1h6BMyPxKbGYM8G3a+bSg80c8ykRVRpzmf2MJUI\\\\\\\/vf+PlHI7NBT9JuEliQt9sDLP1UNn7G52pA3MWICzXRtGjYaAZenoD4isRQhD2QVPkNVi4OeKUASukchD82YvYX7mcJlyw\\\\\\\/zaoA\\\\\\\/kTMOrDk7iDn4q3uRKukL5FFX9NC1i2niJki1f1lQkfkyPWPjJxUxv8vCjzBT9GKSfPoF+PSQssmjzEKzp2Z\\\\\\\/tf+l46mktCWmOhYkIEmTSi\\\\\\\/mn503n255a59dpfCMvTT5LQPepNgzg8Xp46TN4s21+BVX+wNItB7WbGxlZgfjz+X0j7PvsWdYvOcvUlKORQ63ha1UVkquU6HRiArYFetcBsUNw+IDyaYHGDNq8EOj9Jn3Yq0TnggygfWj\\\\\\\/9I8GchmgqHV7zQqd7O+L5hQvh2e5dn+v4wF+ahYB0FkE+TdcVgP746C\\\\\\\/ZyYjdpw++up8H1URdaJk\\\\\\\/NQvEqaf9MDrSMAc6W+eWmT2IqndLd59q1pCCXPZyCMcET5EySEE+X3KYU41cS3MCC54FxcgLiDz7+isX7c4QKz0a7gSVTmsm075kn4v7HXQsSIeZDskHquKcfV2116bQva44+DUYAGmsNaTsvUDX77V7TSf88kxOBLKEAwUlb53YW\\\\\\\/jLNoDNXQLir5uMUVlNhqRZK+gGArM7rc50VkBkD6sHH3UrkQZdUnUEcGCajlEfVgj9i1P30UFYJFkr1g3eINffVDZG1uXTEiLjRUvrI+xx1jN5\\\\\\\/JdA\\\\\\\/BlU6DKbka2w6DFlU99qMlQJ3mr5e\\\\\\\/ZC6fWigmGhIHF7i5JDGeYfLcZmXETP\\\\\\\/LOgfL8e1tCTDtpXpBx9SCb56TQ7gQ0FsPEfEV4jiq65chbeHapM8azIxkCtyEvGz3XMq+XPKRA0sYkmS32DypNY3Xf0\\\\\\\/mmgXBFH4dvkYVQNUZ6rUk+gdh1F485H1sWdP1IZ0TqyDWQahsP5hnm+\\\\\\\/YExe3ZcyzmU5fs3BRFfTFtq9uAq7lU34Ohnh43mQ6R9MEyUlcRlh4lMPX12DBeG1zKS6PRub2OWF4OHXjqqEpG\\\\\\\/EW7234sNQ1Ybfk8qv49vWUyKuB6CcPpJB0VJarkSwbgVlBCmeKcAVFr71P1awf63IGArGR3mQLJTIphhe3TzXnb\\\\\\\/mHaXFkImPyi0cBPZuOImHYu5tkph7VTdD9Q34+o14taitAoBsk+3jBW0OIO3FrxQT3YH+XirmUymWyH4ekqn1zbg4q4mZZx9h0z1Ot8mLuEuwwXuzn67n6iQDeLBrJWgKG+jj3no4NgGlW6KoiS4+sgtXS2zJmJK+ZgxbjoyRR6QUCpJSKv1RlTClH1Sl1CJv1NpwoWTR0for0Un1B2iCU6yEUS8+rSTNl758FPsb1DIJg93vZ1DgUQc5p0KvUWHOSwKzG9cU8hBfLbqBDmKzcf7v2AxucE90zRvmKZ6qNv2sOlCvyEgyqN324+8vcjod2VKONZ00MBCeT84CXUAncJBp8U88VI7TDqoxeozvI2GPAndKD5XMgEUP6axChgPs1tFJoVJxRFqIfVEo1rxy8x78+3+qR+406W+MWXsVwduzezsaoCEmuPTRl2wI1W\\\\\\\/IZBPpoGlgkAvmEqSuxjK75aP0DhWFUQTH\\\\\\\/Ir+kXl+sNG9KSgQPdrly9r+y4RHCkDTz92x9uWKHYTkrdDH3rD3UkRVHbQMDsnQfSXIgFAOwe5s\\\\\\\/oRGEvnmEN8220hrBcNG1NNRimAI0rJRraEWzGNCzBYRckfnhAetxkQeT5EfhznNejg+DUVYby98p8zYUEx53sEM2CcDPkVx+kP1ayouIRaXAAg95Rk26yabhzKkkGxn466OrwsUMxjusf98mqK9pq\\\\\\\/amxDYr+fOrHKCxy9EnJSrPC0Tk4JGOKVHlXZXv95MJw61GqO64LS58haHYTpG0izbxuPCPlUXzjZHm9Z2PBg08jog76lnI57fspP4RjtgeTVQgO\\\\\\\/w642MzpwzqiXEI6GEjbit6qECUrfYZ8cy9W3jrcG6OHL9cxq8sM2jROS5uNAbe78uYzzgeaJaIWV+BUF104pdudrtEREWzzVJ5HfpnYUhFF9QI+MLnleojVDqqByY95wKAFE1ROwb\\\\\\\/XRfekXOOE6yqUAH6z6N9xQPG\\\\\\\/nA8bj1+DnT+G01TU6fSLOmJy+zBMMzbhuStfbk2JuZzAhfM2lFhwYduQ8LtjjKPVjRq2JBd48ojrrW\\\\\\\/ZQnWeEOK1utPy0rpzY5Z3y49UC5Bgfu4SgHt0UgpBxV4KyToZdzo17\\\\\\\/Un22dD9HOlo3MLFWl+uX0vUdID0Y+NKdOx\\\\\\\/XydrPW\\\\\\\/PCwuttb9\\\\\\\/6DO6A+re7XsS61JrCW1FVFHcXwXaebdYWzC+03+S1+E5O7OMI\\\\\\\/HZyjEvKEppackLCfUEtIR\\\\\\\/B2dqI5j6ITHbACjfXSbJ1L\\\\\\\/QBPsoD6kbcBusYqIUObdmbEinMrN04NuhBb5MhDgvNeSxBXUY1vcTh3WdYoFoGGIMBrUE3SmDyZGIlqxCGyToyrxf5gPV9EzYnKYJ7jITwD7wQR2ruKnw9zDi3ihe3lDv\\\\\\\/RF+zy0cYl7zqlaLZbuNbqcyHwv0xDgIMVzjfhgiDNjYTIKoUI3kJ6LV8B3RXD6t94zOjNfYYSCs8cl4ufAaHpPrMmTn2c9fIqlCjEdYpnJxJxHHYIh3\\\\\\\/bnwWSOauq4cvvS8ktKQjvWZ+dOBBlGg4AJVOEMyZudjGJAAjDZBQ9uZhG7R042mfpGJXcnDWu9nQK8jMq9AXB7WXPqcNXJyrTVv2gXzZkJDzDegAO+\\\\\\\/Vd43\\\\\\\/\\\\\\\/FGup2cJzoXRQu+52sPKvPOYqclHQYwv56AB18jtDl1ZY90QCmnmpf13GYeHB23wagjUSHFbWU2A4fpdEIG7JEtPEAJnPV\\\\\\\/jSz2i8u\\\\\\\/E\\\\\\\/1cL5gQwyfvUKaxkRqu0yWmQMZ\\\\\\\/4oPqAfpNlObor2bi2eFY1vl8vGoi3yXuRfHl7PpMoxp1m\\\\\\\/vDgNgLxz3TE1HhZJGlqFLw0rZxqK+mufSWM0aLTpNviIhs9JWN3jQjTZt0IKEtyyp1q9rTMMmG2M2GSF21PQMNg+t2+rOOQ9UzZ6um5C8R10A6nuBuV7AVMKJNVD4DyoxqyNCpfhfGZ4UaVuxOp7iTWBmnSrcC3K9MSIfHsG7C77wjQMsY2SaOdB16sU3Trd6\\\\\\\/CqBptiq70gDqnBtsavGmQN2UufAm0ZmIVqhQ5MVwsmfITD6o9fxlqBPEZJB8\\\\\\\/4Vqp1jI0VZ9AXasYv5Gt+v7\\\\\\\/\\\\\\\/CFjy8GaB03oQi89ISNdAldL79zaL02E7w5ELhfZcvHWRLphHGVdnODK4fA178f2E1cTL9Kz52eFmyBznTHhc8tiVp76s5SZDUhS\\\\\\\/GwUgT6ym6ziYzP5+RzPnYD23O\\\\\\\/HZ8MMMHpYs+Qcp\\\\\\\/NTqM6HsAyrytZE0NmargoGpADG7B2z\\\\\\\/UJtKOUhofshSN1Q+kcKSYUur8Z\\\\\\\/LEPbsV6QNB8AximuMKkKrOGhJt9flqrLLxMvPRlqmOqeHA3vK\\\\\\\/ujj0RgaQrRJaqTOYKdOz\\\\\\\/IFwc\\\\\\\/8IWi+9b5tBrI5FrLQlc7DJAOD6R+CeDtUGpT9e\\\\\\\/8+oIjS1w4oEPePLCFBvHClcFLbcKECRcYwNUIKYDcclNqQZExDcEDat0sXlDFcdmYmH8TfOAbXj3tTEtj9nOs5m\\\\\\\/MYhX7cqirW59hULGYq+3gyXa17yZZJTbDdw6xRwtzkpEIZMAjbsQ9mQaCPHo+sG0tF3+ksEZYsV6qn4rkJO19zXg1uiScuu8asuzn6pN3X4z\\\\\\\/S3A\\\\\\\/Rmvx9etl9cuYzvYBeKpHf1A\\\\\\\/fm51U5Gcln\\\\\\\/mVB0\\\\\\\/kjkf5ozG4HFtT\\\\\\\/19SRaP\\\\\\\/RNKfopdivS1uJMx\\\\\\\/FyIkXG5qqk\\\\\\\/BMpHGB+\\\\\\\/59iCToC2F4M6mUTyED\\\\\\\/oR1eLA7sGFNizTKddHriRukAe00OMx1Da4cASfW7Sv2pry45YyIntp13L9vksD9nCA5ELW7RxepFazoCMLBQgRo9pqEkdXtDCp1X8Bv63zYN6OgH0WJkRwHBHyrOESKlAHuOGyjVrFwEbcu70Gf2WhAx9REOh\\\\\\\/M2obtj9f6GVJHYrSzgj8zY81nbmcyn5uWCj1+UCB4kLxEnlDSR4SFxT9r1mRN6Ng9z+WY93Mj2Y\\\\\\\/tlCfGITjFnOn47X\\\\\\\/mH0Ph5xpJsbM76T5h5bj1CwtRS9LrjuqmN1ZAJ3vJj2GTMVNG0WjvVzLyhpldmUhCzncGPblvqG3kqzs1eT0KYUCqvpDg2s5xL0HzeARPVzRgFYno94Wy71\\\\\\\/Q8EdnQE0KF4TVtSQvxr0DEQJTbhnNCBommgaPb2rsT6iJQaT1dSc7a1NVz0lfRgZw4yEf9ABJf+hrrvhw\\\\\\\/gcM2WDjkREt2GFVqSbJMlYLjRzHWVNWecmnAR6IE2Zp3RAR4PPb8DKHKs3HWhAnlIQ+IWZV9euTCbD789oEd\\\\\\\/mws0QneF73pWL5ygdJcFrd0AB9o6fzidTzB+dW2k5pdouiifWK4QtJNPlvrHXE8euI+fCPGosmJ9on7pyXf0NV1eQfCSBrtLSXzfkOEXF3r9ETaGWGMDqC8vk8L7cXFclNwdTNe0sHNDhjdebHj3IO2XIXqpuo26PLJPJjBVDtB5LzDNKPpnbvwgYvGFzo4EcAvMBOJPdFJS7vmc5cMKd148lROIYhZCVAddgPiIoT63t8b0pDCyK9JQl3dMeIkexUK02Knc+LLx18wNERXcK\\\\\\\/8ABcXUipKfTXDIj4RUfhvmcfziXpG\\\\\\\/P4w\\\\\\\/sTROxpCZ2azQxavJS8suOl+qwG4yHUI5Joa3TW8Xgh840g\\\\\\\/qRSfJ5iuggrytlDZS13cc62PgQEYJ27oTk1Rl2JkXZmA16fJalGDSHw+\\\\\\\/qsEElOEPWdie2Z6a7XR0lcB6UsBjWVNxp7ZpbiFRJx6EK6CE7qzICtqAr7Rg8nyGawd7pPxxNhfWiv7vlCabjbetRWnNknll7LNSj33\\\\\\\/seL1tza7IPDtpB6pSZoBe8klBOXr5tMlbIxbGtS0f6vbf23GXylcK0t0c5vSNPZMNPpI4iecJ82ecWY+e1GbOHkgJfWCWXg1myzW8cvHS2t0UnPmckjOs1urC946D+719Q8ab15QHezj9PFLntwH33a42z+mxV\\\\\\\/pNgXrYDv7BWIsTGLmv0u38c+mh3tzyTbb2nVrs7T1GmIxJdp7ObM4B0FD2YEaIT3xQLYBdsFtQmroCu8A5oU\\\\\\\/AfmywiK72ERInuhgK7ANh2W\\\\\\\/RJ0IG9cA2qQSVmRmbb5YVgPhJr7IRuKG2SggEg\\\\\\\/pBsX3twuIlFZChrQq3S7qRlsCtbWakrCdV1zjJRmY6wlsc\\\\\\\/idRG4DTlhcVoNNZLzKDkY\\\\\\\/2QLd2yO0CEUTlWHBWKzHe5DwmvpwoViz4Gua9uALAYSeciB\\\\\\\/3wk3O7dU+XFvf9qH9ZmqSksu9wgvB13UAhzZ3sjZ9VDvPbnYTShn9z3e8wha5n+\\\\\\\/Shf6GxpOmSZKNHeOCk6ceWeBawCayPrnm+e2tjMJQlLRU4oib21M7OwHDv85H34aELNLL53FTkoprsQ3bYj6Gsr5O4P4W\\\\\\\/r6Fdeh2E09d6CWtjU5m6TOmMY0nipXdY9MBAnMdsYwC8brnZ1\\\\\\\/x3KDlOax6itWK2mmfi4vgBhC2VIUwyXin9MPvaO3M93atc9G8SGeriEzvPoEvP9RmGpmHlP+b6zQRYRZqFp\\\\\\\/6+f1hhaenC7N5+EiRCBxwOyt6qpqoO8Wi4lSHWSbkkR0bWHzW4sV9cVRufZPEVTl3QXPtF1jT0sVbdkGppkhfnpX6d208IOz6jKamTmyGUbav7iph81ZDjSOiRZnmm5PBY6V5s3B3+zgcl1NYq9zTT4Sk7zkJ+BnEx6vthLfBKuq1izy2YFnrEOEShUFP3ei7diCfiGPCzi\\\\\\\/s8\\\\\\\/GXeOGgA3el9SnjTT45239gvcryaPf3gTfuAgx2EisXQTkqMZGtuYHMiioWtGGstFTWYu6obZ8uoPdnaEJMEaMI4C8TZ1YvKrDRgxvHo6kVua7ZPId9sODsDlaiBggaDuwb4vXfErKeC06Rgidxy04Px9fuHyj+hLBGB2oO2l0WyNmphL\\\\\\\/5P3V+uToNz0aGwEzrLZbD829g1Zl8Kw4gXUNpa8i8tVbE9qcsmoQ8jr6F1D29NT2r3nTXgIj575bsHUa+sbaIg7Kc09NnTlR+ndN\\\\\\\/8XLpxNNgoAGY4NRi+aDAnrGv8Md8JyA61EQwTxUciQL47SrDhHlAU5N7\\\\\\\/YRmtQ3PB3m6kB6Bh7gw5xQM3A35dbzPS\\\\\\\/uyMMsB44w22TYbz1JiYkrLZ6WzlBazMsZv+ktSHfMaM7Z+eqSabCZV87uJvkX7mPkMMLxWSONKfAq2v2bqrUrYMB+IlryXUiLHyD+DdiFxaH6xZCxExxVmgmt+w8yAcL5mvHcZYVwemDc0eg8AciwNrv0SIJxAD3buqGJY7tQNHcmD\\\\\\\/fgXyE1rjLHVGfA25PZHaUDSuJ5xv0ZD9rNDOpDJ008q6LqJliIbN8arWCfDo7UEVAvJFMCAfl4yjxQLryj13loUgw3XC9q9juNQQwFN4Ky\\\\\\\/Q1fcOSS9mjbD7kWE7HxtuCLUSssNhW1Nq+da6anGuhx\\\\\\\/NnVNqWVNlDOWBE5sgQYQCjEpuCTKndRrynqa1o7IyR5mBLJs7vJTsOnASjE8LuBhuZqemhNsj24wVavSo7HDswVBVkVjDgNpJctpu\\\\\\\/tngMKpkWjsUulgul5MboHI6lvnQPVNsz7N4QKotNTaqWvAVo6HVGZZn+nuQq62o+7e9NlOFFsZSuMXqDLbA\\\\\\\/JEQUrO06A7exyjU1ahgrjGPh1MiC7D0JaqICBqjkXVo5iTwDGzoSXRux9rvv5f4yftG3V4CcyQoZhOrptN4Wx2V9l\\\\\\\/d5lRdtgSg1BOIekuEvJ3ZaE4AsWhweWyRSR4dyVn3PKnHmaENfHMAEsgso7VWbJynAk23\\\\\\\/rIqxrc1BOdzLSGXDDfbTnnUNWs5krhg3rsXtyyYtPQmFn3HUhhAOTpiHbf3Gpd4aY3XQAo3f1QuYmEg8ldoimLfX+5JImsx0VsIOriKlXJnWdWulNW1bzYaiU7+FifMdRoUoU8\\\\\\\/\\\\\\\/HHRxyDwl\\\\\\\/LUZbeVCk80V\\\\\\\/2\\\\\\\/pzTaFz8D3t3EOoGxj5d25vGxM\\\\\\\/qGat+wHfCZMgO8B2YJ7B9T1wsyAJNef9ltKD0QZZ4ruRf71FmiaHfkWbDL9sWgz4FtvC26W9Gf2E4ax7PY\\\\\\\/gzODaVqhxuG2LmZi7K3GK6J+1ZOP6Pzxr522WVxruGLZnMFKnsA\\\\\\\/XzJbkLp25YKaVkrBNl5d6pc7nxtNxl8cNjZB9sevsiBbh8VzdS15MsMOJ6rPHk0gsOctqlAEuca9H8dv7dHhLZiKGKbkuSrqvHRB9UrVzalRpZsFJJzcVemLrXE+yhaQW\\\\\\\/uyA9T1nQhKvmxlg3exzP5vMvRzxa18YLGehMgt87AjxeG7VGTGjFJTbxOtLKs8NbF\\\\\\\/C2a7XdKaGJ6qIuZw2k+NFfv6f0cl8vNfg1SpO20rFiclrfgEEv3i0sbu4\\\\\\\/Vw9hGQVREpfwrgcZ42shRSXVU1krJVGdoH4wA4NvWFeTO0DJw\\\\\\\/BMVb0UXvuurT1+nRFhYDXRq+fftq2DaF71i\\\\\\\/pfvedJ8h+rWly0M\\\\\\\/OfEH\\\\\\\/wPvMQ5x7o+DipYGGt9q+LwWu0CYRR46uFd\\\\\\\/oB\\\\\\\/HuV2QT26fUjCWZ5Ofc93C9XPzjhJvL5t1fYnV8cGCFVwLtQXExM0fsf88FNdFCVTsn9CeP5GxspUOBAU7cW81+P942DVbpacnbGWX1J\\\\\\\/A5MeE1BBTb+5iJZF5EzSN2+RSIned0nyguimNF4CDDZp2C+QVat4JDsshEzZgQetAGQuuEbZQgPz8S6LbF035dhHM73C10IZE8DPakFfOWUTaLZs\\\\\\\/s\\\\\\\/lTt3t6r39n+J14rsQh5O\\\\\\\/KGjhVAo8qgLlWe1QYc57Tn+WmNYaRoUo63FoxCaD5dbG\\\\\\\/dC+Oj\\\\\\\/wL7qR9VcWyW2RrMml\\\\\\\/SrO3oAHoMqhmB5yWJ4cNJhXeHD2xfWI7SuG5BtHY1HIVc1uDHwecuRn1INiE0jjhjyXP6DeuNGTIzmhZMpwjmnuYNlGtC+99g7kWJJl3MJkavjO18v2O0maPIqXCFkXGXlmwY95gTXls9\\\\\\\/qya\\\\\\\/OwKM6TzXB6Dap2VCnRcagvYz9FSpCY1iqT7ntv40KlN90K66yoc6nZQsjpKk9zR8h1O2mGdrG4vKOvhHgHWwbNa1Cy7gg0Fx5LBtoFuRd6g5sqOlK6zYehcDhecY0UR1pbEpl4OMNt8uSoFDUPFbyjVkAudsbqaVTyO88buHXLp1BwobSngGdk36RC+X7YqSx+plsJhXWteMa4P+HTypfne\\\\\\\/pi+upyH2zqQaAqh\\\\\\\/KjrEOAYsVF+Voxq+fnRRIGnjMNWt4+AvmLYXBmErVGEINaBI9Vyv21e6hfpyg1t6neURXR9AbjXbA7NM0COwEoVlM8E24qwDCwuWlK3Cbs4GiRfUYy3ZWiB6l+qTx2vER8gsTfigpYt\\\\\\\/Cfxuoa05WxGcQ5b0P9SKs7Qf3maqUiLSB81outD0ydyJSHgCpOlxsmHtOuwIJlBn1FLo5a1NnZyIa7B8wPQQwL5M+Wb\\\\\\\/5CRxZez6CjeThGmZE6LYWjcvVncWvPSS14CeN6gmO1ZsoWRx764y92pNN7DgKGaPTM6ZkC9n458yiZg4O1\\\\\\\/QYe4fNmHoYqlgYy\\\\\\\/2At1iBd0kzWOWqgDVpIXr8aG1P8DOklhRc+i2UtcirNBzD6kq6upLh5IZk95Dg5INMp89iN\\\\\\\/3PkiKOkhqQbd5krygT71Jgb4Tv91dbnxxqRWP9A\\\\\\\/BJWYpiS7KdNPqE5thv7zdj2KD5QgGLwuuuMFUTr57aFC9LrDxKoUrhI3EJi8UmVydpofboeHZIzjQgEhkVvGbVn2q4JFlYcppsiS7pUWj\\\\\\\/QInbbyJBg4oWvkIIM61JuxSJ7K37MwOCJrTazTgtojD8hTlJO0SQlB6hqjhl0ss+Qg8iFjAD\\\\\\\/ZQ0E40rPtSAgI\\\\\\\/y1CnPqJoE7sFuxFCeau1S6FbGdf9q\\\\\\\/L7lbZ4C1hWbMNzKMbj0UfZVzASmGwXlezsceOHZsgkbfP\\\\\\\/Ym8Da0GGHoFUufzwJaXuD43X4aQEcZKpI0s5GF5f20BzbEsii83UafC2MyoqxtK6+KMSzzCN4B71cqukNo9031+STeBbskI82Z9vpwIqI6jEobqqLXZ8N2iBgIh7sKfb2ls\\\\\\\/kZ0bpQG8H60R5dRpHZmo20DZZ1hpfEeBPUtomRB4uELWsOz9AyB\\\\\\\/2Sg6Rgwz\\\\\\\/k6UVF5i3CId8\\\\\\\/wdOvQqlES02gn5VHEPF7YhjGfBTl88BR4K6bJu626l9L2O3y1NLSBwcR4nK6nZT7KOs2kxVeNQtKTUC0ODkxg0ITUnNqkJH+2o79+f4y8tp0ZTYPZ64Lw9KmFe4t\\\\\\\/j\\\\\\\/7YElc8OSSXgshfYe2klSWnB3o51JnpW8sWW\\\\\\\/iqSV\\\\\\\/Q9xZmbQhR+SE+BW9YKC0Czd0AiJwxZP0NwVT9quMPCWvStm72E5pRzphe\\\\\\\/ZYIAgNjaV2DkWxPblBOjcj5UL7PUqK9psS5JdZUrrcwEChycaavz3hN1l0oKZDCrxDvDaraFhFzb02YW\\\\\\\/+BRhelQ+zYVzCBXdElv1Vvy3b9Big37cOd5jWqprXVXUlad037JHNsj8qEyxOFaFoMZsHOzhAreuRH2WKBBEPzLukW2rq\\\\\\\/+ahriEGfja5tQNlkD327Ogr0dNfGs0jA9jKw192MvcvLx\\\\\\\/Rl5d6bcCSN1ys2hpGJDwHulKSWs0c9ingD54d1W7PGTufgJxTjEyy6VG8EGb\\\\\\\/gtk6aSc6iQXJ\\\\\\\/8PmjMOkVMcS93yZcMrMMB726ZSxLZBaYy0ciKTt9ZLWewaLPKYlBj0WXnMk\\\\\\\/E7eu+QHKmpyJfHWdCc\\\\\\\/l35tZPqNm6i\\\\\\\/ttnNS+dfv3Wvb3MfFfReFaqmozi2y0aeENG\\\\\\\/1QbJ59BB3XQ1sJXZ\\\\\\\/KzR4VqDTK6aGhRHr6Xz4Vyj5u3COir5f4En03XPM7q3DHkm1XII+NtKoOHZIbFe55hXnlqehoHXNk07GRZrN\\\\\\\/U\\\\\\\/e1YrSyTJSe7NNs6qMYNVhMCQQPM2mMIrL4V6YTV3AtqFi7ucwCXfz1gddFqkFjaQCwwQiuaVlNoRE+uprtcze9lyl\\\\\\\/b10dnCAfMnZsd2HnhccnkflJKnr66OeybQzqNGjozRLkzrR3D\\\\\\\/dU+Ik1YdEqEcm4V2hf07IemkmdHWDk3zrEBtEBU2c3\\\\\\\/ZTN4KKDYJSjYj6ztN9Z0tB1mNVNde1G3Bpd2vVL6Ow4NuMYtI+YGyfR3oz5laHV8182CTwcu9IYHD9bSu0xbcIHAsrxi0v7qMD9FKrzafcJYSmZrHwlc8YUhq4xPxM56h1mR3q6QjVYNAHNR2Jtowq5apJRVXu7UpaNd8f8d+ZGbpd92fZB7Y+qZUI2CZAZOOyZHWUlQcXytxcpbI\\\\\\\/SaQYeGxXoFbSB9ptA5Mb2b6pE6RmtvJpDp8bvOPjwjLLLTxyt\\\\\\\/zeH89GWdw+FWGE5q0Wr\\\\\\\/ET3IK\\\\\\\/WRkU7kFblEDGEjN\\\\\\\/jCsRXhD0oVAXd1h5U5t3Is2PeBkPOZdNxDxPQ4jPXVzMxVNx1pazcMaa3J1tx6HxA3QqvKoSbE3RXCYZxUzOhxHTlVhdK\\\\\\\/S7MT+a1usU7MhvX8Ym4ZdmUjl68Zwjf8bH57K+lcp29Utu1bGT56q6kmx29l7A6SuEwARoSLh8gFtV4+M4vunlrxosJTRKVTlHE4S+V6+P6GaVwzjz0nOi6JPr5FDG3XaZLcEypPl9wGnbBAgePHDcilJS+CtoJMYoqc3jbZfXTpGucZ8uYf0H33ssn6f39ZyLCAKz0WC7Lr18Fs3ukS3zQujmHOACccJggaYneZG3a2UzcT551L3wagHuAIZChIZj8U2GXgc5tMOwGmqCalL6CKUPwy0P3tjjsSmStf2S3k5njIcx9FP\\\\\\\/gNdqOP8yBO+41ecefz9EimDMZF0ge4+KQgJ4OmIt\\\\\\\/EAjLv6n\\\\\\\/1FgNJ6i5XdBw2LDBlkKNBx8dAVRt01kvYF3wymlhmSOpvi62V7S2+OWOwVa\\\\\\\/uQGjA\\\\\\\/rDscg5vL4sYotZ\\\\\\\/3\\\\\\\/DX4VZX7NyGn00ROj5Sm48KT8LdtE0DlvlMMkPW7tFe0Suu3bULkLc7pUOYaaNEuuoTLyZEzJCUu790CeBo9m91V4kkbOg6Zwmg2ld+n1LZ8ux4VZMkL6WJAoIFJg36Z2NthabOgYT0FzxsWHjLanhHL3d8p71nnPErAnWq8pBJDhOlPOINvsjy967so6F7WQ\\\\\\\/p4uIeNIgIKzYnEKACC9+l76P8h7cb4vspyQbkouQyG0PXYE1GlO+MlhvLFiopY0DlbWhc1WdXpAFCETtUNazLPciQFUzoDHvN+H0+gwc9Eo2d1o062h9D1k3RgYLGW6M8BMvEKRg92QAkSLWV5W8U98zkd2JYI5j5eclzY4vnU\\\\\\\/+ccOwGLLAO50rdkEGADF3g573FOzrY8OrLr7ZsIzCxgeXeo76W5pc2z4PNAvaEXFBR6j2OTvO9g\\\\\\\/6YJbgc7T5uy8URAWRx9jqjqfxSMdIBlMwehWlQzYanGEaTL2iA9O19VtPmP+58hittcSslREC6+ElCDs9OC+a0SV1jDoMTrxeuFyt9GYoS12\\\\\\\/pZeXKQ9rxE83vcaiuT1LVOJj1kWbUY1IahYlce1FfY00VAbjJclgQytpv+tlh0FcxORHNIT\\\\\\\/3aE2M0DL30Z\\\\\\\/9yPdLEOw4CznTIkvj01fozBSyH4PWOr7u4J0vHHZlyYgAxu2OYsMknc8sz2ja2xPUcv3+kir83sjCc1Y\\\\\\\/BYj5bFf8LMZ69+FPgRfbisGgydPkXJLZnhFUaqdgNUWIa2nZg\\\\\\\/rpwSzlvHcKIgd1cQFIU49AKOxUGXEoyjdpKQZeLQ084Z5vG3IPVTkaT6VPdeaBgwwGDhFhBeZ8ZjGFiVUAm26e9wqb7cRLziWD1NyC69LrmmOooLNPHqhevD7CMmtjypfSY4VSb2MWGVGQt521fXLhBZoo6dEYa31vev\\\\\\\/4KubgLT3iSEIeTfTZkFxL47skU1hQ3vO7V\\\\\\\/wwnX9\\\\\\\/YAb40mXQShpmBkoQvUa5FubgFJr02odicx5H9QKA2xEs\\\\\\\/Tn1fEhzOh+2qJeVHfwA7obKWPGPxzBD8ovKF4wew2UdaeddL6mB6462VZ7yYpDyw4GqBlVTsaAdB\\\\\\\/8GWEXKKGjlCiweP4+VE1p8n440oMMWtGumDhsZX7BqAU6dGFp+iF17CuqWSkE+VwxOR3JF4CerdT3hWRtewo1reFH+KAIawXdvCnlfAf+UqCFkl6qPJLH40XUrFjbg4MfTsaVawXV7ISZ40efVO\\\\\\\/EaFTKBXEd+BW+tj\\\\\\\/WlML2yVmLxi3Or0F4oOBWCBdMo\\\\\\\/q3WXQGTACMorKgYXJ66rOJd9yLKApZht7c8cuUNZcef7NK+HeO0stch\\\\\\\/esLsm\\\\\\\/pQJKYC3HOUwdtbJQSoVRfXTG0RQMiNF7MZMcGPcpfd+h+UhZ0+fy8hbLbpxDsa95qT3WSpVFwEuUN64ImjPEnML1woMRutFDA128B1wOWznrXJw77oMa1396Y8h\\\\\\\/dQZSesoQ4LOte1JLD4FUtFbsDKoU4L1y9dzbK31tycHOKKtcJL+\\\\\\\/A\\\\\\\/6zUr7Q323V49PBfGUdjB7jztXGlQ3BtKqIgk9eRSEpifpuJmXetXzywbhg9t\\\\\\\/zBduZtyox041KvtSMhrC\\\\\\\/c8mib3jBKfOedHqP9R6wWt4bR01hJOsFJM953TpdoyD+4bKmlrxm0TA52lrLr6AWfimWST35HZS5Tk699KbiGuIp2My304hnw0PUIuPyVqLAmXmd2CSI1nToYsUpkDi+g2I4HBVARqDwKBWcta9nms5ys1OPG++xMAn\\\\\\\/6oZAMgOdVz9anm6\\\\\\\/3we\\\\\\\/8sAICwRC2pMg6VI5C62FbtkcwZvpyAbcUH6oswyIK8NZem4FdR9iiv8bi9w02I1zbZXs9ZOLzn2fG1cp3sRmNgNiBzFiRAN6c2q4TdORiXS8cPrOFcisZDBjV031vQ1SX2c6s0kboYXHv03PO5kzOeXEjLjCj3WDr+5Skpxg8szgzx0iddDSrt+6beKDIch8ZIdaEgKqJ\\\\\\\/yimwLA0mL0nRLT35rzhkLRX\\\\\\\/RtLuYXyGJYBc3HyMcREkxhgtTsHMpkIkdDgJAVyDFPam9WV7HqVW0VMDf\\\\\\\/vWfRKVWs4b1bYfQBKTMWCaRFaQMXhOnorcqzyHPEXe+Qu4vpf7mnznm8BuIwkpuZMuUz0LZs284hkF0PWdukStYzjRiw9sQu3K4fTSRx74fbpqi\\\\\\\/NAwmed9Z9JWn4Op\\\\\\\/VNj1oOXCAL6hiTcXlTvf0ElYdESL0GYusKOKw0HmKrb7ZY5va9cF9vnvxF8nvcstZar2OpFiFpXsT19vKBLxfcvBkPPTd8ozmD+bbGTf\\\\\\\/h79vn8l8Mo+QoRkzHdbIkaG\\\\\\\/Ig\\\\\\\/zFWnnUKIACX6gSvcmm3aQn+14B7BEmFxrRKxDVSfifEkl3Z+hfWbFOTmqUrJF4QwtceoUnQEIdIxg3ONU+AzPoM3evvSMF1yCUCgeWMSQ0uL50M2uljstkqexMhaVi6G3D3KG3WTWDwpOlC\\\\\\\/+YD+pxkmLokdFv8cmWuHs1QUmhIbfWpqIVV7sFYYHx7xjv\\\\\\\/7eaHYyhFThZtXUToD4u20uXh98i3EzDOKhG0bbpW5HIWHXo7m0UFVDgOkD+NruZsE+RV5L4iIWFlOnLcetDq+h9azni\\\\\\\/TGdX7pVbPQD\\\\\\\/+KuAs0vRClA463K+O6cQFdBahVjiWfvWEVFoTKhvnSsCxoRhihoCuXLXxHSw5ayXgFC3\\\\\\\/MwVBGsVE+2BpdY98pzvnk5uIM0EeMri+9RdT4sprSrrzCIOuRpsnQr4DKswa65MC46FcKKNnwh7Bn4RhvzjhpRLv8b8ZafmU34BkdocCH4u5u3dqybIgFVd2KX0LuUleaG7V\\\\\\\/7JrCxNZ1b3OTDzZ3T6K9jSgI2px5I\\\\\\\/VxLKhV53uAstI7k6AqXTeGssjKgRHMY4RGNJnS900PJNBIiF+8+4hIzy5p4KxP\\\\\\\/PMmqIVcAw2Jsm2VKJD+CX87vMjvmLVQABAGZ2MmFaCao2MpPj8TjmiRYNRjLL+ZvVTPwj0cPpaGzA3J2prgmZXLEBRMFOq0R2loSajFvDueEeh8alOQddpt88I3+IdnK7TLzDnLbtfoeOgyzDg4wmiazdl7HVWtHMgLHVcQlN\\\\\\\/c3o4ALVgNXeL+HJUcpsS6PWnGjjZd0lRV5EMB+3GW7abrFrspYC4xMAR3TLj5mxDCV76+1pew52UtZ9YMD+187Ep4X5zOk547JN6S\\\\\\\/O0sf8ufTb4sRWwQmtsRmJ8y7Fc6NVgD4qbaJoom7zQIOWbaAhy3Gi\\\\\\\/FlMB4\\\\\\\/vb8TeGhFgWHEKKSORn1+laNPWQ9hVjB+idEsOBoDf2V919MWL\\\\\\\/JCv1zmBirHZCr1MN2cR1qPKGQNIDBwrYDhnopsKgBSTLK7ff0cZRmZL3RXq5oQN4P3GZ6emtcvHaDahX42n1l37IlAZJU7H77VRsYnuTUC1r0EIV4UB4vk0cNVnQlLnq+bEQcwUXCWj8k0cRzXb90bz9Ng9dHG02G0bobahln6qaq6zRrikuJF+T+\\\\\\\/+4G54cNNiZgQSnRzKvjkM7IHM2ktDnhdEp4A5sZ\\\\\\\/bW5U8inkhLjgHTzTqeuoUbpB0dyN2nVnU8Zs+B0pV354NYlS8nQmsv3Aa7Q+2BRvOLEdy0N97uRfAMWvEbsuU1o31kZwEyiVcyEsyLg1qgyhfN\\\\\\\/YbqU9Hj+MBj8pUWbRl4ACkMZdDDjqm8H1JdB1DAMwAQpdIE\\\\\\\/0ArqrDe32clIRxR44lggJG\\\\\\\/JUJ33BxBbYp65mz1n1tmifp+rUm0+omn\\\\\\\/BUpSFrg16PM+\\\\\\\/nKgfSl7fUij5PoENfcBevhlakVDBQMRbItNUxQoUylpOUMf6mAbzK4ZGwJfDM5cObxmYZXdqPvJ\\\\\\\/7xqN\\\\\\\/ZNtS2n6tFObCfre+gAz0slRiFTxyhOC6ioBJuIgoVWFFhqbgcexgisM4qa5AmGAH0zzZSj9N6UxkqvDr4DaiK\\\\\\\/xXNr0Q1H2am7FHdSAVRZol\\\\\\\/fjR\\\\\\\/OK8YbTKIPHMvOFLfUZTEKKLJWeH3mRgXFAO6VNHolGUlz29K4vhy7jE2whEWoG7qg2mtkIf431zOLHUO2X8p7kD\\\\\\\/qVF9HU+AFQM9bs3kCf6iUdTiYfV9qNc6Q3VNFaO1BF5k7eUFojvPEVNZHQCinmIXP87ja52rhsdzS5XllP5vE9I0S3j+vjQAo0aAYcn\\\\\\\/fZofk1OiLsfbCJjDPM5L+vNJtFQqclPE31AtFXhggoVZYtnPqZHqFksTVcoJAP4WQBiVDllFWa64MgCgDsqIjcBdd985OVH1abUeuilzWWwCF6WNGjqqYnrN19S31PJvqhXRRo5y+SlWktcGSd2x6xX25xjsgOHs3I19afaoeMEvyrol2oNe0m80xG2cNihwMpgGWis9KrPywbIrk+aRXf182Cbj7xmyznEiQgQacP2lb2bEEOUVJSH9ZkX4Eud43C7tnIUMPA1z\\\\\\\/fvN7C+6SA3lVuXYUB5NzXCc64Z5hdIl97P7+KsYuyrVlkUIY2pvd3mIgFGrqcsxCAAJD0aV+XF3H1hSZPmaRLSrhzcDNEIWh55zvDyU+RDlW+mxnah9WYpknUscPqr2ng3P6H+RDG\\\\\\\/COcFeHFUAjx+Egon6o2xe4p4+azKfVgjETWaC9Zkf7aeqjqnFkH7yDtMZH7ffFX7SkeCqErOS6LPh\\\\\\\/iY8ClIK8JDEDa\\\\\\\/m9dOFYpilxgnvlA9AxCbNDnd10gfTWEhcbVWuzRLiuKMLGLx2w66fjlKIo7x\\\\\\\/EAo0g07JjIT+dCsUGAM8jzVi9WtGyVwbul2AlntVtk+1svzf+Td2cOFbJ\\\\\\\/yLrKnQEZBT5izkujRyfh8W41p+9ROY1G945Hnr6O6iIWy17aARWrU+jID7RXYCc\\\\\\\/n1bk0kYjB9Q1kTbSexZIShugZzfSLpZwwPLFl5zvSPT9rZmbHmzozu5FVbDdWvezjDfjl\\\\\\\/4IBJY2Wc17qSUyYcXhXhZH18J9uFN11+kmb7ib0O9UteeV+Bh8oXN0YTy8E2XBs0G2BVCItmGZLUsiJsyAseVWLwOneKsSqZEZhV3s059dxGwj9UdLvgtlqcWeF7Uhr2aOTdAg9HQ3+eAxL3VhEy91+hv6wB8jzz31xW1EUSEQ4lRwE3hghZu0nJ\\\\\\\/8T2QVM1FYVRTQAGcqEp1a0tg3C4zJ0tiKiDqIAD2dbZwTb04NubWafDav8ARnTMdbRc7njsOatpxZrQjls7GC3QHkc6fvnW8itW7WA7JGRWU+Lsdl1PMQAPG1h4qnlQXAe0WpXYe7LPwU2h7HRyTgVDb2ODIg09Q5IsAuSr4ofHySYiR98LoDVmCDAjamq\\\\\\\/l0c6lkQprkGkVl5aKNivlbp\\\\\\\/DuXzZfT4wpyQ0cR3zwD82ihIB8cyYqDzUGugmBBkpD+oedZHwwk\\\\\\\/ZoI1pTFQ5mlO0jqqUXl67KQUKdV\\\\\\\/sJ0idu4UIpl0CNYjtE6Y9rfyUa+bBJAooRZoi2MwUwVF\\\\\\\/5H7FO\\\\\\\/jFh35id5m+IH8ScKm1dbktAycee95KxPM4\\\\\\\/S9gAYdYjVtlwwmdWZ268MvJjonq\\\\\\\/+qNb2MJu5et0kJAp39\\\\\\\/9hc2aPaaw+Wz5fQXp9eRRAmKgFLUDyb6kuvadKalAZuQwhBlYinWoz8zG3sZKq5pg0M+OW3PAKdhLsOPXs\\\\\\\/ibU3ep2Gmp\\\\\\\/IvH\\\\\\\/xsg3LaL7sWV9YfDT1F\\\\\\\/SqDa8iP\\\\\\\/nXQ\\\\\\\/OmuC9y9zf30r9hZxR\\\\\\\/ItN5F\\\\\\\/o1b4iK+9wCeHVTc3BnkCUyKWtyLqhGjQZVyC8Vl\\\\\\\/UCf4IA0mXej3XnTAPvI\\\\\\\/8UiopqgqOf0Q+xvcLxGCuyQgyR\\\\\\\/zwD+tk8HEzKBYsy\\\\\\\/LX5s+GYNj10Ybi07M4T7jVc27VhF7vqRPhmTvnzjUZr0cNPpjfaHmfTL8ALNVUvQsB1VrwQItDFN+RVrVx7jk2mmvwh9dYFzXKq\\\\\\\/7q4rWvP9R1IGghuC1Th6R6szfFhYJQQ3SV0OAGq\\\\\\\/2TD1zRBnIWVYbAtxGlifEGLoTT7KoVZs8Yy8IH0eTyYKamNl5k+YkxRhzZoziPUraiBE6tEOKViSTgDY9+Vf8vCqeL7SznIP7ZQ0jHN4JMMfbRjMloAw+0PqmSfxlqIkwVcBITBUcZzhmRw6RCWfEr6YLYiU6Sk8jf+W2DvH5WhRBK\\\\\\\/xZwazXhy5fPqRFGnNFUQZ6gasjUuzuqa+JQzMxDl+JAoznNwRbW5RdV0VcRetBbde\\\\\\\/7ciiep27LbN1ITJ5n7N6b9Bchdj0vTmSYkHSDstd9768nmFgPrYVVdZ02rYIPWOVtUC0CPwKog2wEN5Y6fbp9\\\\\\\/lqOzO2Ik8I6fK+UVUbkxmGS2TfhUdBJLv40NCoKCAVSy8MrH7vVrVKt041IU2wGxoentSvtu7jmGzkFlLxQt9chNTlLKoBwR00G0xtPB5jxLVW97w6\\\\\\\/aqAHMZCgHc0J5vmmpRfYvIenLMaerKTYCC9K36d5CYAB9RANzmv4I9rqK3Cl9+xWNAUNrn\\\\\\\/I3\\\\\\\/XXCsTUlhhchukbsqg\\\\\\\/Wt+0r88jaOG9jtVghRIHiUgEKCrhKSk9kXEzq6gXi7Tvq4etooEBXQjJ4Z3sRqy\\\\\\\/YWDB3rOaKBjYb9BxLNGmgj+r4OVLeft+JwyFtyxTr\\\\\\\/vC282ddmTrl8NLWxsL4leP4rOo89gcJjWWswL8srIjlma4XG4\\\\\\\/YvuP4zxGL7Elxma2hMXUCx\\\\\\\/JGR\\\\\\\/X2EBJ3HBtG0Ha1tQ6gxqRhTLY51kHgsUSv0VXUqaGnOCf3V3w6R+qSLPbATgUnDSZ0Vx\\\\\\\/L1Mb8VI57F5OFUs1LvpheO25Rl9zTwZyrrJ779MHk61Cn4FF6RkE5TXuBrFGxeKiAUQ7rRHoJu\\\\\\\/aRD60bRNtVHcmw5R9mw+mfOc7x+a8tbUGhhYS3YVpF1CUP7tFL+DmKV7JFn9XzX8BzcbZZg45Ml2\\\\\\\/i5K4F5OhQGKQtewfi8rBK7nsN2X2bJg8RPdJc1lgtVFbTonHQVSToCUNiyPEQtzYjKdnratpbq584EgTRnSgwsu4wrHDpXa5BwN9Z8Ly6mpaIP+Mczlv0f+xSB+88vw=\\\u0022,\\\u0022iv\\\u0022:\\\u0022RfWe7xNIb4RnS1nYDBdgTw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002253946bbad1838f227d9dcf973e7b10874275af4057ac3dab35a5f9ceef07ca96\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BTH8da6Ob+mWsXtjIlwXOAJk\\\\\\\/in2oAs9rUpFLPxgz82CA4GFEwrP5s6zRfoGGA3\\\\\\\/oJWF5UrJoZWrnqPtNl3qWuTt7rjWHKaHlGK3Mf2dY4lXd1BAxfjWUSSV+hCSLFUJXT1zRYHD81NOL+cIARXO\\\\\\\/hl5LfyVFwElnkl2OtZA8\\\\\\\/EZzzbZtjnzLmGGox9UFW8rKiKBEWrbirgQ3KGupAxuq9TmXISrDXfh8HVJEM45v7pwGi+a0fq5hWxv86MTzw2yNz0EejDFw+xPdbLhnaUnDyFmaAp6STuBqv2RGB7\\\\\\\/wEHV847Z+bhlCK4uCOFHHIxXKGYfpBy1AXwmBOC7nnXWWMeAYvFDQD2gF035Q7BACyUxKoOLBI87FwpcmWPwrPXTiD0xUPdUwQb8F98Z1Vvf35W8uUu57vHAcw\\\\\\\/wfs\\\\\\\/EukS3xdWBpdYBJTqe3HyJ9BcRg1CyMCgaxryuBn5wadRxe\\\\\\\/GurAumAdHxMyY2TO\\\\\\\/g6f1Kyugk1qvcXtHLRG2e37fCVRXWaIgRNyqb7KXvaP7c59ZvguuwFSRg4tISeo4cLYwOFuWP+8v+hUlmBmHoMBtsEKvH20wEW1ZqIflS\\\\\\\/vkWU+4zXwXvP8nKONzDmx4S05PbGW2phGRGfdPh\\\\\\\/4emxCHQooQq3++Wv3HafkIvvPjaI5d\\\\\\\/NeVFf7hv7nPWJeEDrbV8o9cfAT65Wz41knhwEJcXBJn2wOW7V1jZL6SVeARiuAS9IR1znIRkSJZAOWZcytq\\\\\\\/ACb0PqJQFF6HD5k1pheLuNagjb8OkkjBx4Rdm5azsp1Idsw78WtYZ5N\\\\\\\/oszkgZniXTwIiqtd99if7g4aZCC+lRZ6\\\\\\\/o4KvA54CpSIZrFh0iFnYAtR4QAKP4A6KkPJ8zuMIeNUQ\\\\\\\/fWMXJ5U2sMU8fKhuYcNU14095vQNy519XfbnuGuqXvePSBpNy+em43pdNIsfgE9Q2yJyq9Et+DbfUPGud5ESMVRdom1Vb07zWou1wTKZ4XmNt6W+RUOqdxtOsCxMBkfwAgSwz6PoJrqQHesIE9r8k9JqrsNMeDNvazjxyi8aXtCEjy4WUQ0SbA1lrF5HfVJVxe1ZReenaXuMfhd9EBpSsV\\\\\\\/4vTwLCLy+6GoNtWTsbYVfti+RP0r\\\\\\\/ULoaZxnT22DuLqjx2Cx72LStXNz8RZ4UivgR0NVhyo2QIO6Slp9yLak\\\\\\\/7tdEFMUZKIC0KMmRyS8+q8c8kPni2yjjaVwVIBSZw5uujBM55FZHFuWcHTpup\\\\\\\/Ygp\\\\\\\/kPh87swIWqe8u8cj+Ujt+dqc0Gp8byI26GjgacOLRp6vNadigkDlkbnOvCMTt\\\\\\\/wvXFwY+kH0ufuPivKhNsx7p6cUZEqxu8WpiesKMlnDaFKp3GmD+nUpGf+k\\\\\\\/07ARfZ2+5b3mljd75MSEqcdwPgcBN07WP3zOx02Mk8hSNl9+1WHjEHjqPTWRm\\\\\\\/mIw3sYJ0USLX55VpBiN1cBm6nPazsoin\\\\\\\/dylj0ZYVajlW9cFvHwTtu9UZD0DvCl0gx76V5qKPNBzuUY11E9RY4qTDuZbJCptUZWYy3tuenW4LGeqyGuDitrMJbcr4lNDGfzcHnZd5bMO4+qfBNpsmu8Nyh+CZJV5hZCc+PMXIAfRhDlSyhnHSJU5iHXSBmbrmXXVMUo66CV8bN69XyR3pvYTZPpyNnh+7rxWu5Kx24RfA\\\\\\\/kVQ1PJYaPFT2PjfrN0BUgsHn1nuTGSfwcW8Q11tulnHq5rlNGxmdDTA\\\\\\\/lWiVWUpZgyM37Zum1dKUt0hdYmQytb945vHYkP7Gr8FACEtgij\\\\\\\/coxV3jALJ5vHVbLjoCE09BYP\\\\\\\/hMevO8Y3WPjc8O6UI1TMDkp5z1T3u67OuF2c92nbaq5TQEVmL+d5OI4LT5URFm1my9VH5oO6REJltPFQpBLZegfWT9r3zpXBAU91Lg9qkczVmqdfyrgjCjdqW4Hw461aeIYrqk+O0e6Fz3dmPeG2HOKg03DbbJquaVufM32YzGxjLYmfnN7JXHh4PtrRGbqjlFum3BxT8LHFyZFuQnKtA1ngdoFamKhB5S\\\\\\\/x3mv\\\\\\\/kIKeFcO35tVEZzYVJ3Bgw4k6HjcuxOVowlfT++SywQ9Eky2zaR3SW+YGgqBaVE35qXd5k0UJCW5XpUIGBmEY1sdgmUmQ9Xm8BtgPg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022spVXDb9SUhAAVMPG9tgfjw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220d37cce208f431e97b9b6acbcf1ed6bce19a896857612a1142a27284c6c3e8ea\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-485854 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2763" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kzNvh3TIhIGXnhmF8CXGJrkJlBUT6lY6Bf6_5n4kXlo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2763" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-R0YEBdBY_yaeyFoaliq5MgeyBRA4nbYy30_4MmuWbvw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2749" title="Sex Chromosome Evolution, Heterochiasmy, and Physiological QTL in the Salmonid Brook Charr Salvelinus fontinalis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2779" title="Bayesian Networks Illustrate Genomic and Residual Trait Connections in Maize (Zea mays L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485854" id="node485854" data-pisa="ggg;7/8/2763" data-pisa-master="ggg;g3.117.043893" data-apath="/ggg/7/8/2763.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2567-2634" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael G. Nelson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2659-0910" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Raquel S. Linheiro</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5462-9854" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Casey M. Bergman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2763-2778;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043893 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485854" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael G. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, M13 9PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BG.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BG.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2567-2634" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael G. Nelson</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Raquel S. Linheiro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, M13 9PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raquel%2BS.%2BLinheiro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Linheiro%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARaquel%2BS.%2BLinheiro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2659-0910" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Raquel S. Linheiro</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Casey M. Bergman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, M13 9PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Casey%2BM.%2BBergman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bergman%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACasey%2BM.%2BBergman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5462-9854" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Casey M. Bergman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cbergman@uga.edu" class="" data-icon-position="" data-hide-link-title="0">cbergman@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/8/2763" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:485854" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:485854/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2763.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:485854" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:485854/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2763.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:485854" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:485854/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/8/2763.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:485854" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:485854/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/8/2763.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:485854" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:485854/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Transposable element (TE) insertions are among the most challenging types of variants to detect in genomic data because of their repetitive nature and complex mechanisms of replication . Nevertheless, the recent availability of large resequencing data sets has spurred the development of many new methods to detect TE insertions in whole-genome shotgun sequences. Here we report an integrated bioinformatics pipeline for the detection of TE insertions in whole-genome shotgun data, called McClintock (<a href="https://github.com/bergmanlab/mcclintock">https://github.com/bergmanlab/mcclintock</a>), which automatically runs and standardizes output for multiple TE detection methods. We demonstrate the utility of McClintock by evaluating six TE detection methods using simulated and real genome data from the model microbial eukaryote, <em>Saccharomyces cerevisiae</em>. We find substantial variation among McClintock component methods in their ability to detect nonreference TEs in the yeast genome, but show that nonreference TEs at nearly all biologically realistic locations can be detected in simulated data by combining multiple methods that use split-read and read-pair evidence. In general, our results reveal that split-read methods detect fewer nonreference TE insertions than read-pair methods, but generally have much higher positional accuracy. Analysis of a large sample of real yeast genomes reveals that most McClintock component methods can recover known aspects of TE biology in yeast such as the transpositional activity status of families, target preferences, and target site duplication structure, albeit with varying levels of accuracy. Our work provides a general framework for integrating and analyzing results from multiple TE detection methods, as well as useful guidance for researchers studying TEs in yeast resequencing data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/transposable-elements" class="hw-term hw-article-keyword hw-article-keyword-transposable-elements" rel="nofollow">transposable elements</a></li><li class="kwd"><a href="/keyword/bioinformatics" class="hw-term hw-article-keyword hw-article-keyword-bioinformatics" rel="nofollow">bioinformatics</a></li><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">genomics</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li></ul><p id="p-4">The widespread availability of genomic data over the last two decades has provided unparalleled opportunities to learn about the abundance, diversity, and functional consequences of transposable elements (TEs) in modern genomes. However, the computational analysis of TE sequences in both reference and resequenced genomes remains a challenging area of bioinformatics research because of the repetitive nature of these sequences. Development of bioinformatics tools for the detection and annotation of TEs in reference genomes is now a relatively mature field (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bergman and Quesneville 2007</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Saha <em>et al.</em> 2008</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lerat 2010</a>), although many open questions remain about choosing the best tools for specific biological applications (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hoen <em>et al.</em> 2015</a>). In contrast, detection of reference and nonreference TE insertions in whole-genome shotgun (WGS) resequencing data are an active research area (reviewed in <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ewing 2015</a>), with a large number of methods published in recent years (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sackton <em>et al.</em> 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ewing and Kazazian 2010</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">2011</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hormozdiari <em>et al.</em> 2010</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Quinlan <em>et al.</em> 2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fiston-Lavier <em>et al.</em> 2011</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2015</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kofler <em>et al.</em> 2012</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">2016</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lee <em>et al.</em> 2012</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Linheiro and Bergman 2012</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Nellaker <em>et al.</em> 2012</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Platzer <em>et al.</em> 2012</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2013</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2017</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cridland <em>et al.</em> 2013</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Robb <em>et al.</em> 2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gilly <em>et al.</em> 2014</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Nakagome <em>et al.</em> 2014</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Thung <em>et al.</em> 2014</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wu <em>et al.</em> 2014</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zhuang <em>et al.</em> 2014</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hawkey <em>et al.</em> 2015</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hénaff <em>et al.</em> 2015</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jiang <em>et al.</em> 2015</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Rahman <em>et al.</em> 2015</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Quadrana <em>et al.</em> 2016</a>).</p><p id="p-5">Because of the wide array of available methods, it remains unclear which method for detecting TEs in resequenced genomes is best suited for particular genomic problems, leading to substantial investigator effort in terms of installation and testing, or the application of suboptimal bioinformatic approaches. Most papers reporting new methods to detect reference or nonreference TEs in WGS data provide some measure of their own performance relative to using simulations, benchmark genomic data, or PCR-based validation. However, only a handful of papers have reported new methods that include performance evaluation relative to other methods (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Gilly <em>et al.</em> 2014</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Ewing 2015</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hawkey <em>et al.</em> 2015</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hénaff <em>et al.</em> 2015</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Jiang <em>et al.</em> 2015</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Rahman <em>et al.</em> 2015</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2017</a>), and these are often limited in scope to only a single organism or TE family. In addition to being incomplete, comparative analysis of bioinformatic systems in papers that report new methods can fall victim to the “self-assessment trap” (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Norel <em>et al.</em> 2011</a>). Moreover, there is no common format for the annotation of nonreference TE insertions (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bergman 2012</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rishishwar <em>et al.</em> 2016</a>), making direct comparison of predictions from different methods more challenging. Recently, <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Rishishwar <em>et al.</em> (2016)</a> performed an independent comparative evaluation of seven WGS-based TE detection methods using human genomic data, which revealed many method-specific predictions and recommended combining the results of multiple systems followed by manual curation (see also <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Ewing 2015</a>). <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Rishishwar <em>et al.</em> (2016)</a> also highlighted the challenges users face when installing and running multiple TE detection methods, and provide helpful advice for users and developers.</p><p id="p-6">As a step toward a fully automated framework for running and evaluating multiple methods to detect TEs in WGS resequencing data, we have developed an integrated pipeline called McClintock (<a href="https://github.com/bergmanlab/mcclintock">https://github.com/bergmanlab/mcclintock</a>) that generates standardized output for multiple WGS-based TE detection methods. The primary goal of the McClintock pipeline is to lower the barrier to installation, use, and evaluation of multiple WGS-based TE detection methods. Several key features of the McClintock pipeline are that it automates formatting of key input files and standardizes output of multiple TE detection methods to allow easy comparisons of results from different systems, as recommended by <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Rishishwar <em>et al.</em> (2016)</a>. In the initial version of McClintock, we incorporate six complementary TE detection methods that make predictions based on split-read- or read-pair-based evidence in Illumina WGS data. Here we describe the McClintock system and its component methods, and perform a comparative evaluation using simulated and real yeast genome data. Our analysis supports previous conclusions that no single TE detection method provides a comprehensive detection of nonreference TEs (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Ewing 2015</a>; <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Rishishwar <em>et al.</em> 2016</a>), but provides a framework for further testing, development, and integration to achieve this ultimate aim, as well as useful guidance for yeast researchers to select appropriate TE detection tools.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Analysis of simulated WGS data sets with single artificial TE insertions</h3><p id="p-7">To investigate the performance of McClintock component methods on data containing known, nonreference TE insertions, we created simulated <em>Saccharomyces cerevisiae</em> genomes, each containing a single synthetic nonreference TE insertion from one of the four active TE families in an otherwise unmodified <em>S. cerevisiae</em> reference genome. Since active <em>S. cerevisiae</em> TEs (<em>Ty1</em>, <em>Ty2</em>, <em>Ty3</em>, and <em>Ty4</em>) are known to target tRNA genes (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ji <em>et al.</em> 1993</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chalker and Sandmeyer 1990</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">1992</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Devine and Boeke 1996</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baller <em>et al.</em> 2012</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mularoni <em>et al.</em> 2012</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Qi <em>et al.</em> 2012</a>), each synthetic insertion was placed upstream of a different annotated tRNA gene in the reference genome, taking the orientation of the tRNA gene into consideration. The annotation for 299 tRNAs was extracted from the SGD genome annotation for sacCer2 (SGD version R61.1.1). <em>Ty1</em>, <em>Ty2</em>, and <em>Ty4</em> have been shown to insert predominantly within the first 200 bp upstream of tRNA genes, and <em>Ty3</em> appears to target more specifically the region of RNA polymerase III transcription initiation, 16 or 17 nucleotides from the 5′ ends of tRNA genes (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Ji <em>et al.</em> 1993</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chalker and Sandmeyer 1990</a>, <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">1992</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Devine and Boeke 1996</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Baller <em>et al.</em> 2012</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Mularoni <em>et al.</em> 2012</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Qi <em>et al.</em> 2012</a>). All active <em>S. cerevisiae</em> TEs produce 5-bp target site duplications (TSDs) on insertion (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gafner and Philippsen 1980</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Rinckel and Garfinkel 1996</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Chalker and Sandmeyer 1990</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zou <em>et al.</em> 1996</a>). To mimic these insertion preferences in our simulations, <em>Ty1</em>, <em>Ty2</em>, <em>Ty3</em>, and <em>Ty4</em> were alternately selected for insertion; a 5-bp TSD was created (either 200–195 bp upstream of a tRNA gene for <em>Ty1</em>, <em>Ty2</em>, and <em>Ty4</em>; or 17–12 bp upstream of tRNA genes for <em>Ty3</em>); and the corresponding full-length <em>Ty</em> canonical sequence was inserted in the reference genome. 299 insertions were produced with the TE sequence inserted on the positive strand of the genome, and 299 were produced with the TE sequence reverse complemented to test the effects of TE orientation on method performance.</p><p id="p-8">We simulated resequencing of single-insertion synthetic genomes using Wgsim (<a href="https://github.com/lh3/wgsim">https://github.com/lh3/wgsim</a>) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2009</a>) with a 1% base error rate (−e 0.01). Read lengths were chosen to be 101 bases each with an insert size of 300 bp (42-bp SE) and 100× coverage to mimic the properties of a large sample of WGS data sets collected by <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> (2015)</a>, which we use in our analysis of real yeast genomes (see below). To generate an average read depth of 100× across the length of sacCer2 reference genomes with additional single TE insertions, <em>in silico</em> WGS samples were created with 6,024,220 read pairs for <em>Ty1</em> insertions, 6,024,237 read pairs for <em>Ty2</em> insertions, 6,023,936 read pairs for <em>Ty3</em> insertions, and 6,024,369 read pairs for a <em>Ty4</em> insertion.</p><p id="p-9">McClintock (version e945d20da22dc1186b97960b44b86bc21c96ac27) was run on each of these simulated data sets using reference TE annotations and canonical TE sequences from <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Carr <em>et al.</em> (2012)</a>, plus a manually produced hierarchy file based on the reference TE annotation in <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Carr <em>et al.</em> (2012)</a>. We used the standard, unmodified reference genome sequence option of McClintock for these single synthetic insertion simulations. The mean of the number of nonreference and reference TEs predicted per sample was calculated across all 299 simulated samples for each strand. The proportion of correct predictions of nonreference TEs was calculated at four thresholds of accuracy: (i) requiring the exact TSD to be annotated correctly, (ii) requiring a prediction to be within a 100-bp window either side of the TSD, (iii) within a 300-bp window either side of the TSD (the insert size of the simulated sequencing data set), or (iv) within a 500-bp window either side of the TSD. BEDtools window (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Quinlan and Hall 2010</a>) was used to calculate correct predictions within the given windows. A prediction was classified as exactly correct only if the same TE family was predicted to occur at the exact coordinates of the TSD of the synthetic TE insertion location. For nonexact overlaps, BEDtools window allows a permissive definition of a true positive, where a correct TE prediction is counted when any part of a predicted insertion falls within the given threshold distance if the correct TE family is predicted. The orientation of a predicted insertion was not taken into account for determining a correct prediction because some methods do not predict orientation.</p><p id="p-10">To visualize the accuracy of nonreference TE predictions, the results files for the 299-positive strand and 299-negative strand single-insertion samples were converted into two bigWig files (one for each strand) using BEDtools and wigToBigWig (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kent <em>et al.</em> 2010</a>). This was performed for each TE family and each component method of McClintock. SeqPlots (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Stempor 2014</a>) was then used to produce plots of the genome coverage of predictions for each TE family, centered around the simulated insertion locations for that family. Visualization of predicted insertions negative strand simulations were reverse complemented and depicted on the same plot as positive strand simulations in different colors. Plots were centered on the 5-bp TSD and extended ±10 bp for split-read methods, and ±500 bp for read-pair methods, respectively. Results for TEMP were partitioned based on whether or not split-read support was available for a prediction. Prior to visualization, we attempted to filter out any obvious false-positive predictions using the fact that each synthetic insertion location should only be predicted in one simulated sample. Thus, any locations where a predicted nonreference insertion was observed across multiple simulated samples indicated a potential false positive. This filtering was necessary to prevent a nonreference insertion that was predicted by RelocaTE in the same location in 149 single synthetic insertion samples from dominating the visualization for this component method. False-positive filtering prior to visualization only affected five other potential insertions for PoPoolationTE, and thus this filtering procedure does not substantially alter positional accuracy results. To further investigate the accuracy of TSDs predicted by split-read methods, the length of the predicted TSD was plotted for each active yeast TE family. To be consistent with analysis of real yeast genomes (see below) and to mitigate effects of false-positive predictions found at the same site in multiple samples, TSD lengths predicted in simulated data were only plotted for unique insertion sites rather than all insertions.</p><p id="p-11">To investigate the concordance of nonreference TE predictions made by different McClintock component methods, we first determined whether or not each method had made a “correct” prediction in each of the simulated samples with a synthetic TE insertion. Predictions for ngs_te_mapper, RelocaTE, TEMP (both split read and read pair), and PoPoolationTE were classified as correct if they overlapped with the true location of the TSD. Predictions for RetroSeq and TE-locate were classified as correct if they occurred within a 100- or 500-bp window of the correct location of the TSD, respectively. The orientation of a prediction was not taken into account when classifying a prediction as “correct” or not, because not all methods predict orientation. The overlap of these correct predictions was then plotted as Venn diagrams using jvenn (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bardou <em>et al.</em> 2014</a>), comparing split-read methods, read-pair methods, and finally the total set of correct predictions from all split-read <em>vs.</em> all read-pair methods.</p></div><div id="sec-3" class="subsection"><h3>Analysis of real WGS data sets</h3><p id="p-12">To assess the relative performance characteristics of the component methods on real data, McClintock was run on a large sample of <em>S. cerevisiae</em> data sets from <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> (2015)</a> that includes 93 <em>S. cerevisiae</em> strains from different geographical locations and clinical origins. The <a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> (2015)</a> samples were sequenced on an Illumina HiSeq 2000 with paired-end reads of 101 bases each, an average insert size of 300 bases, and a median coverage of &gt;117×. We used these general library characteristics in our single synthetic insertion simulations (see above) to allow more direct comparison with analysis of these real yeast genomes. The raw fastq files for the 93 sequenced strains were obtained from the EBI Sequence Read Archive (SRA072302).</p><p id="p-13">McClintock (version 354acec977e37c354f6f05046940b0dabf09b331) was run on each of these samples using reference TE annotations and canonical TE sequences from <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Carr <em>et al.</em> (2012)</a>, and a manually produced hierarchy file based on the annotation in <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Carr <em>et al.</em> (2012)</a>. The McClintock version used for the analysis of real yeast data differs slightly from that used for simulated data in terms of three small improvements that were required to handle variation in sample names (for ngs_te_mapper) and differences in read lengths of paired-end fragments (for PoPoolationTE) which were encountered when analyzing real yeast genome data. We used the standard, unmodified reference genome sequence option of McClintock for these analyses. The average number of nonreference and reference TEs predicted per strain was plotted as box plots for each method. In addition, the total numbers of nonreference and reference TE insertions per TE family were summarized across all strains for each McClintock component method, both genome wide and in tRNA gene regions.</p><p id="p-14">To biologically validate results of different component methods of McClintock, we took advantage of the fact that <em>Ty</em> elements are known to insert in close proximity to tRNA genes in <em>S. cerevisiae</em> (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Ji <em>et al.</em> 1993</a>; <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Chalker and Sandmeyer 1990</a>, <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">1992</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Devine and Boeke 1996</a>; <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Baller <em>et al.</em> 2012</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Mularoni <em>et al.</em> 2012</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Qi <em>et al.</em> 2012</a>). A prediction was counted as within a tRNA gene region if any part of the annotation was with 1000 bp upstream or 500 bp downstream of the transcription start site of one of the 299 annotated tRNA genes, taking tRNA gene orientation into account. To visualize the patterns of nonreference TE predictions around tRNA genes, all results for all 93 samples were converted to a single genome coverage bigWig file for each TE and each component method. SeqPlots (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Stempor 2014</a>) was used to produce plots of the genome coverage averaged across the 299 tRNA genes. Plots were centered on the start of the tRNA gene and extended 1000 bp upstream and 500 bp downstream, taking into account the orientation of each tRNA gene. Results for TEMP were subset into two groups based on whether split-read support for a prediction was available or not.</p><p id="p-15">The lengths of TSDs for nonreference TE insertions predicted by the split-read methods were plotted by TE family. To prevent any nonreference TE insertions present at the same location in multiple samples from biasing the results, only unique insertion sites were plotted. If a method called an insertion at nearly the same location but with a longer or shorter TSD in different samples, these were classed as unique sites.</p></div><div id="sec-4" class="subsection"><h3>Data availability</h3><p id="p-16">The McClintock pipeline is available under the FreeBSD license at <a href="https://github.com/bergmanlab/mcclintock">https://github.com/bergmanlab/mcclintock</a>. Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a> contains a combined supplement text including: detailed descriptions of McClintock components; an overview of the McClintock execution process; details of postprocessing of component method output; methods, results, and discussion for analysis of McClintock applied to simulated resequencing data created for unmodified <em>S. cerevisiae</em> reference genomes; and Figures S1–S4 and Tables S1 and S2. Supporting data sets of McClintock predictions for real yeast genomes in SRA072302 are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS2.zip">File S2</a>. Code used to generate simulated yeast genomes and apply McClintock to simulated and real yeast genome data are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS3.zip">File S3</a>.</p></div></div><div class="section results-discussion" id="sec-5"><h2 class="">Results and Discussion</h2><div id="sec-6" class="subsection"><h3>McClintock component methods and their dependencies</h3><p id="p-17">We initiated our design of McClintock with a literature search for candidate bioinformatic systems that can detect TE insertions from NGS data in 2014, which yielded 33 potential systems. Our main project objective was to develop a system that automatically detects nonreference TE insertions in raw WGS data for any species. Thus, we excluded systems that required any wet-laboratory enrichment from further consideration. Systems that did not make their code available were also rejected. This left a list of 12 candidate software systems. After preliminary testing of these 12 methods, six were rejected from further testing because of difficulties during installation [Tangram (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Wu <em>et al.</em> 2014</a>) and VariationHunter (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Hormozdiari <em>et al.</em> 2010</a>)], reliance on data for a specific organism [TEA (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Lee <em>et al.</em> 2012</a>) and VirusSeq (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2013</a>)], inability to detect nonreference insertions [T-lex (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Fiston-Lavier <em>et al.</em> 2011</a>)], or the inability to distinguish general structural variations from TE insertions [HYDRA (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Quinlan <em>et al.</em> 2010</a>)]. Six remaining methods [ngs_te_mapper (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Linheiro and Bergman 2012</a>), TE-locate (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Platzer <em>et al.</em> 2012</a>), PoPoolationTE (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Kofler <em>et al.</em> 2012</a>), RetroSeq (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Keane <em>et al.</em> 2013</a>), RelocaTE (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Robb <em>et al.</em> 2013</a>), and TEMP (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Zhuang <em>et al.</em> 2014</a>)] had publicly available code that could be installed reproducibly and met project objectives were selected for incorporation into the initial McClintock pipeline. Since the original selection of methods for inclusion in McClintock, a number of additional methods that meet the initial project requirements [“pecnv teclust” (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cridland <em>et al.</em> 2013</a>), TIF (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Nakagome <em>et al.</em> 2014</a>), TE-Tracker (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Gilly <em>et al.</em> 2014</a>), Mobster (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Thung <em>et al.</em> 2014</a>), ITIS (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Jiang <em>et al.</em> 2015</a>), Jitterbug (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hénaff <em>et al.</em> 2015</a>), TIDAL (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Rahman <em>et al.</em> 2015</a>), ISmapper (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Hawkey <em>et al.</em> 2015</a>), MELT (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Sudmant <em>et al.</em> 2015</a>), SPLITREADER (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Quadrana <em>et al.</em> 2016</a>), and TEPID (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Stuart <em>et al.</em> 2016</a>)] and new versions of some methods [PoPoolationTE2 (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Kofler <em>et al.</em> 2016</a>) and RelocaTE2 (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2017</a>)] have been released. These methods have not yet been incorporated into McClintock, but the flexible architecture of our system permits their inclusion in the future.</p><p id="p-18">A summary of the main features of the six component methods included in McClintock is shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. A more detailed overview of the component methods, their original use case, software/data dependencies, and limitations is provided in the “Description of McClintock Component Methods” section of <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>. While none of the McClintock component methods were originally designed for detecting TEs in yeast, using the yeast system as a test bed does not favor any particular component method and realistically models application of component systems to a new species. The six component systems each have many dependencies on other pieces of software, which must all be correctly installed before the component system will function correctly. These software dependencies are listed in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. Several of these component dependencies require end-user licenses, and thus it was not possible to fully automate installation of all component methods. McClintock therefore assumes component dependencies are installed system wide, but automates installation of the component methods themselves. A passive check is performed during installation of McClintock that reports whether component dependencies are available, though installation is not halted if they are missing. Because of the large number of component dependencies and subsequent development of components themselves, we developed McClintock to use specific versions of components and their dependencies. <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> also lists the version of each dependency that was used with McClintock to obtain the results presented here.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485926/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/485926/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">An overview of the features of the component TE detection methods in the McClintock pipeline</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486098/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486098/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Software dependencies required to install and run each component TE detection method in the McClintock pipeline</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">McClintock component methods also have a variety of data dependencies that are required as inputs, which are listed in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. The component methods incorporated into McClintock together require a total of 13 different data dependencies to run. However, since many of these data dependencies can be automatically generated or are format alterations that can be automatically achieved with simple preprocessing steps, the number of data dependencies can be reduced to three required inputs for McClintock: a fasta file of the reference genome, a fasta file of the canonical TE sequences, and fastq files of NGS reads (paired or single ended).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486024/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486024/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Data dependencies required to successfully run each component of the McClintock pipeline</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>The McClintock pipeline</h3><p id="p-31">An overview of the data flow and processing steps performed by the McClintock pipeline is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. A detailed description of how the McClintock pipeline is executed can be found in the “Overview of the McClintock process” section of <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>. In the following sections, we describe the options for running the McClintock pipeline, then describe how component methods are parsed in the context of the McClintock pipeline to create standardized output for downstream analysis.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the McClintock pipeline. In the flowchart, important processes are shown as boxes, decision points as diamonds, and data at important steps as parallelograms. Note that the last three steps of the pipeline are applied independently to each method. Final results from each component method are output independently by McClintock, allowing the user to easily merge output or assess for overlap among methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1548353408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overview of the McClintock pipeline. In the flowchart, important processes are shown as boxes, decision points as diamonds, and data at important steps as parallelograms. Note that the last three steps of the pipeline are applied independently to each method. Final results from each component method are output independently by McClintock, allowing the user to easily merge output or assess for overlap among methods.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2763/F1.medium.gif" width="400" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/8/2763/F1.medium.gif" width="400" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485955" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-32" class="first-child">Overview of the McClintock pipeline. In the flowchart, important processes are shown as boxes, decision points as diamonds, and data at important steps as parallelograms. Note that the last three steps of the pipeline are applied independently to each method. Final results from each component method are output independently by McClintock, allowing the user to easily merge output or assess for overlap among methods.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-8" class="subsection"><h4>Reference TE annotation options:</h4><p id="p-33">Several McClintock component systems rely on information about TEs in the reference genome as part of their workflow, which can be either supplied by the user or automatically generated by McClintock. If a preexisting annotation of the TE sequences in the reference genome is available, a one-based GFF file of this data can be used as input for the McClintock pipeline. If such a reference TE annotation is provided, then the user must also create and supply a TE “hierarchy” file as another input. The hierarchy file contains two tab-delimited columns, the first listing the name of each instance in the reference TE annotation and the second listing the canonical TE family to which that instance belongs. If no reference TE annotation is provided, then a reference TE annotation and hierarchy file is created automatically by running RepeatMasker and postprocessing RepeatMasker output files.</p></div><div id="sec-9" class="subsection"><h4>Reference genome sequence options:</h4><p id="p-34">McClintock provides options to automatically create various different modified reference genomes. These options were implemented because some component methods (RetroSeq and TE-locate) require an instance of a TE to exist in the reference genome for nonreference instances of that family to be detected in a resequenced sample. This is important because, in some cases, like the <em>Drosophila melanogaster P</em>-element (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kaminker <em>et al.</em> 2002</a>), the reference genome does not include any copies of a TE family that occurs in natural populations. This situation may also occur when a TE family has been introduced experimentally into a strain lacking that TE to study its transposition. To allow for these cases, McClintock has an option to generate modified reference genomes that include additional “chromosomes” comprised of canonical TE sequences or TE sequences extracted from the reference genome. An annotation of TEs in the additional “chromosomes” is then appended to the reference TE annotation file. PoPoolationTE requires a modified reference genome with canonical TE sequences and reference TE sequences added as additional “chromosomes.” Thus these reference genome modifications are always made specifically for PoPoolationTE, regardless of whether user-supplied options to modify the reference genome are provided globally for other component methods.</p></div><div id="sec-10" class="subsection"><h4>Run options:</h4><p id="p-35">McClintock offers additional options to customize the way the pipeline is run. It is possible to specify which component methods are executed, allowing tailored output and shorter run times. McClintock and its component methods produce short-read alignment files and other intermediate files that can be very large, and thus an option is provided to remove unwanted intermediate files. BAM files output by McClintock may be useful for other purposes, so an option is provided to eliminate all intermediate files other than BAM files. The location of all output files can be changed to any absolute path that the user requests. Within the specified location, all output files will be produced in a directory named after the reference genome sequence with results for each sample stored in subdirectories named after the fastq files for that sample, allowing multiple samples for the same reference genome to reuse common index files.</p></div><div id="sec-11" class="subsection"><h4>Postprocessing and standardization of output format:</h4><p id="p-36">The component methods within McClintock produce their output in different file formats and annotation frameworks (see <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bergman 2012</a> for discussion). Therefore, McClintock performs a number of postprocessing steps to standardize outputs from different methods into a common annotation framework. Details of the native annotation framework for component methods and the postprocessing steps made by McClintock can be found in the “Postprocessing and Standardization of Component Method Output” section of <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>. Before performing these steps, the original (unedited) results for each method are saved in the output directory for that sample. If TE predictions are made by any component method in the additional “chromosomes” added in modified reference genomes (see above), these results are removed from the standard results files and retained in a subdirectory within the results directory called “non-ref_chromosome_results.”</p><p id="p-37">The output file format chosen to standardize results for all component methods is a zero-based BED6 format because it allows easy integration with the BEDTools and UCSC genome browser. The BED format provides a fourth column to contain a name for the annotated feature. All records in these BED files contain the name of the TE family predicted at that location and whether the prediction is of a nonreference or reference TE. The name column also reports the sample ID from the fastq input file and the name of the component method that made the prediction. The type of evidence used for the prediction is also listed, either “sr” representing a prediction made from split-read evidence, “rp” representing a prediction made from read-pair evidence, or “nonab” for TEMP reference TE predictions that rely on no evidence for the absence of the TE in the sample. In addition, filtering and redundancy removal was performed within the result file for each component method. No redundancy filtering is performed by McClintock across component methods, allowing users to more directly compare output from different methods. To facilitate viewing of results on the UCSC genome browser, a header is included in each BED file. This header is read by the UCSC browser and lists the sample name and McClintock component system that produced the results as the track name and description, allowing multiple result files for the same sample to be merged and visualized simultaneously.</p></div></div><div id="sec-12" class="subsection"><h3>Application of McClintock to simulated S. cerevisiae genomes with single synthetic TE insertions</h3><p id="p-38">To test McClintock and its component methods, we used simulated WGS data sets based on the genome of the model eukaryote, <em>S. cerevisiae</em>. We chose <em>S. cerevisiae</em> for testing McClintock because its reference genome is relatively small and has been completely determined (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Goffeau <em>et al.</em> 1996</a>), it has large samples of publicly available resequenced genomes (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Almeida <em>et al.</em> 2015</a>; <a id="xref-ref-64-4" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> 2015</a>), and the genome biology of its TEs is relatively simple and well characterized (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Carr <em>et al.</em> 2012</a>). Briefly, the 12-Mb <em>S. cerevisiae</em> reference genome contains 483 annotated TEs from six long terminal repeat (LTR) retrotransposon families (<em>Ty1</em>, <em>Ty2</em>, <em>Ty3</em>, <em>Ty3_1p</em>, <em>Ty4</em>, and <em>Ty5</em>) (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Carr <em>et al.</em> 2012</a>), a type of TE that can be processed effectively by all six McClintock component methods. <em>Ty1</em> and <em>Ty2</em> share a nearly identical LTR sequence but differ in their internal regions (<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>), while <em>Ty3</em> and <em>Ty3_1p</em> have 82% nucleotide identity over their entire length (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fingerman <em>et al.</em> 2003</a>). Most TEs in <em>S. cerevisiae</em> are solo LTRs or otherwise truncated copies, with only ∼50 full-length elements from four active families in the reference genome (<em>Ty1</em>, <em>Ty2</em>, <em>Ty3</em>, and <em>Ty4</em>) (<a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Carr <em>et al.</em> 2012</a>). <em>Ty1</em> and <em>Ty2</em> have the most full-length copies in the <em>S. cerevisiae</em> reference genome, with very few full-length copies being observed for <em>Ty3</em> and <em>Ty4</em> (<a id="xref-ref-33-8" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">Carr <em>et al.</em> 2012</a>). The active TE families in <em>S. cerevisiae</em> are known to target tRNA genes (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Ji <em>et al.</em> 1993</a>; <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Chalker and Sandmeyer 1990</a>, <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">1992</a>; <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Devine and Boeke 1996</a>; <a id="xref-ref-33-9" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Baller <em>et al.</em> 2012</a>; <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Mularoni <em>et al.</em> 2012</a>; <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Qi <em>et al.</em> 2012</a>) and create a 5-bp TSD on insertion (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gafner and Philippsen 1980</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Rinckel and Garfinkel 1996</a>; <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Chalker and Sandmeyer 1990</a>; <a id="xref-ref-33-10" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Zou <em>et al.</em> 1996</a>).</p><p id="p-39">We first performed control analyses by simulating WGS resequencing of unmodified <em>S. cerevisiae</em> reference genome samples and applying McClintock to these data sets (see “Simulating Resequencing of the <em>S. cerevisiae</em> Reference Genome” in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>). While not the major focus of this study, these reference genome simulations allowed us to evaluate how often McClintock component methods detected reference TEs and, more importantly, how often component methods detected false-positive nonreference TEs (in the absence of any true, nonreference TE insertions). An example of reference TE predictions for all six component methods is shown in Figure S1A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>. In general, analysis of unmodified simulated reference genomes showed that McClintock component methods cannot detect all reference TEs (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>), but also typically have low false-positive rates for predicting nonreference TE insertions when they are truly absent (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>). Additionally, these simulations showed that McClintock had better performance at 100× <em>vs.</em> 10× coverage, and that neither the choice of reference TE annotation nor reference genome options substantially affected the detection of reference or nonreference TEs for most McClintock component methods.</p><p id="p-40">Next, we simulated WGS samples for reference genomes that include a single synthetic TE insertion from one of the four active TE families (placed at biologically realistic locations upstream of tRNA genes) to evaluate the ability of McClintock component methods to detect true positive nonreference TE insertions. To do this, WGS reads were simulated for 598 samples, each with a different synthetic TE insertion placed upstream of one of the 299 tRNA genes in the yeast genome. 299 samples were created for single synthetic insertions in the positive orientation upstream of tRNA genes, and 299 samples for single synthetic insertions in the negative orientation. Genomes with synthetic insertions were created by alternately selecting one of the four active TE families and creating a 5-bp sequence 12–17 bp upstream of a tRNA start site for <em>Ty3</em> or 195–200 bp upstream of a tRNA start site for <em>Ty1</em>, <em>Ty2</em>, and <em>Ty4</em>. This 5-bp sequence formed the basis of a synthetic TSD and became the location into which a full-length <em>Ty</em> canonical sequence was inserted in the sacCer2 reference genome. All single-insertion samples were simulated at 100× coverage since the ability of component methods to detect reference TEs improved with increasing coverage and to better match properties of the real yeast genomes analyzed below. An illustration of nonreference TE predictions for all six component methods in a genomic segment containing a synthetic TE insertion is shown in Figure S1B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>. In the following sections, we detail the analysis of these single synthetic insertion simulated samples in terms of overall numbers of reference and nonreference TE predictions and positional accuracy of nonreference TE predictions.</p><div id="sec-13" class="subsection"><h4>Numbers of reference and nonreference TE predictions:</h4><p id="p-41"><a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows the mean number of reference and nonreference TE insertions predicted across all 299 simulated single-insertion samples on the positive and negative strands, respectively. The proportion of correct predictions of nonreference TEs was calculated at four thresholds of accuracy: (i) requiring the exact TSD to be annotated correctly, (ii) requiring a prediction to be within a 100-bp window either side of the TSD, (iii) within a 300-bp window either side of the TSD (the insert size of the simulated sequencing libraries), or (iv) within a 500-bp window either side of the TSD. If all single TE insertion samples were predicted correctly for a method, it would lead to an average value of exactly one nonreference TE predicted per sample. Comparing row one of <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a> (single-insertion simulation) with row nine of Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a> (unmodified reference simulation), we can infer that the inclusion of single synthetic insertions into the yeast genome does not substantially alter the ability of any McClintock component method to predict reference TEs. As expected, comparing row two of <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a> (single-insertion simulation) with row nine of Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a> (unmodified reference simulation), we see gains in the numbers of nonreference TE insertions predicted for all methods; demonstrating that McClintock components can detect true positives above false-positive baselines in our simulation framework.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486074/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486074/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Average numbers of predictions and correct predictions, by method, for simulated yeast WGS samples with a single synthetic TE insertion upstream of tRNA genes</span><div class="sb-div caption-clear"></div></div></div><p id="p-43">For ngs_te_mapper, the average number of nonreference predictions shows this method systematically underpredicts nonreference TE insertions. However, the average number of predictions made overall per sample is only slightly higher than the average number of exact predictions. Consistent with unmodified reference genome simulations (see row nine of Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>), this result indicates that only a small number of nonreference predictions made by ngs_te_mapper are false positives. Moreover, whenever ngs_te_mapper makes a prediction of a nonreference TE (that is within 500 bp of the true insertion site), the prediction was always at the exact TSD, suggesting high accuracy in terms of position and TSD structure for this method (see below). We also observed that ngs_te_mapper detected fewer insertions when the synthetic insertion is on the negative strand relative to the tRNA gene, suggesting there can be strand bias in the detection of nonreference TEs. This bias could be due to yeast genome organization, our simulation framework, the ngs_te_mapper algorithm, or a combination of these factors.</p><p id="p-44">RelocaTE produced, on average, slightly more than one nonreference TE prediction per sample. At face value, this result suggests that RelocaTE may detect essentially every synthetic insertion, but also makes occasional false-positive predictions. In fact, the average excess number of predictions made by RelocaTE in single-insertion simulated genomes is very close to the false-positive rates observed in simulations of unmodified reference genomes (see row nine of Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>). However, only ∼50% of the total RelocaTE predictions are made within 500 bp of the true insertion. Thus, it appears that the inclusion of single synthetic insertions increases the rate of false-positive nonreference TE predictions by RelocaTE relative to unmodified reference genomes. Nevertheless, RelocaTE produces more correct predictions within 100 bp of the true insertion site than ngs_te_mapper, the other purely split-read method, despite producing fewer exact predictions than ngs_te_mapper. Thus many of the nonexact RelocaTE predictions within 100 bp of the true location are likely to be accurately positioned, but simply not have the correct TSD structure (see below). Like ngs_te_mapper, RelocaTE also appears to have a slightly higher true-positive rate for positive strand insertions, with the difference in the number of correct predictions on the positive strand being greater in the exact prediction category.</p><p id="p-45">The average total number of nonreference TE predictions for TEMP is nearly one (0.90), confirming results from unmodified reference genome simulations (see row nine in Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>) that TEMP makes very few false-positive nonreference predictions. Moreover, the total number of nonreference TE predictions for TEMP is the same as the average number that are accurate within 100 bp of the true insertion site. These results suggest TEMP is correctly predicting most simulated insertions, but not to base pair accuracy (see below). Some positional inaccuracy is expected for TEMP since not all predictions for this method are supported by split-read evidence. For TEMP, there appears to be no difference in detection ability for TE insertions on the positive or negative strand.</p><p id="p-46">RetroSeq predicted nearly as high an average number of nonreference TE predictions per sample as TEMP, but the proportion predicted correctly was lower than TEMP for all length thresholds. The fact that not all RetroSeq predictions are within 500 bp of the true insertion suggests that RetroSeq can produce some false-positive predictions of nonreference TE insertions when the sample is not identical to the reference genome, unlike what was observed for simulations of unmodified reference genomes (see row nine of Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>). Because RetroSeq does not use split-read information, no predictions from this method were exact, however most predictions were generally within 100 bp of the true location. For RetroSeq there is a only slight reduction in ability to detect nonreference TE insertions on the negative strand compared with the positive strand at all length thresholds.</p><p id="p-47">PoPoolationTE produces an average of slightly more than one nonreference TE prediction per sample, but this method shows the lowest proportion of true-positive predictions at the most permissive length thresholds, suggesting most predictions are false positives. This result supports those obtained from unmodified reference genomes that PoPoolationTE makes approximately one false-positive prediction per genome in the absence of any synthetic nonreference TE insertions (see row nine of Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>). Because PoPoolationTE does not use split-read information and the span predicted by this method is often large (see Figure S1B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>), no predictions made by PoPoolationTE were exact. For PoPoolationTE there appears to be no difference in ability to detect nonreference TE insertions correctly on the positive or negative strand.</p><p id="p-48">TE-locate produced an average of nearly one nonreference TE prediction per sample. However, these include some false-positive predictions or at least predictions that are &gt;500 bp from the actual insertion location. The proportion of correct nonreference TE insertions predicted by TE-locate drops steadily from 500 to 100 bp, with TE-locate predicting the lowest number of correct insertions for any method at the 100-bp scale. As with the other read-pair methods, no predictions could be considered exact because TE-locate does not predict a TSD. These numbers indicate that, though the ability of TE-locate to detect the presence of a TE in the general vicinity of its true location is good, the annotation will not be as positionally accurate as other read-pair methods like TEMP or RetroSeq. For TE-locate there appears to be a reduction in detection ability at all thresholds for TE insertions on the negative strand compared with the positive strand.</p></div><div id="sec-14" class="subsection"><h4>Positional accuracy of nonreference TE predictions:</h4><p id="p-49">To visualize more clearly the positional accuracy of McClintock component methods, predicted nonreference insertions were plotted around the known location of synthetic insertions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Plots were produced for each TE family and method to determine if the family of the synthetic TE insertion affected results for a particular method. <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a> showed that for split-read methods, there was no increase in the accuracy at thresholds of ∼100 bp and many predictions were exactly correct. For read-pair methods, it appeared predictions could be several hundred base pairs from the correct location. As such, split-read (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) and read-pair (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) results were plotted on different spatial scales. Since TEMP could use both split-read and read-pair evidence, results for this method were partitioned into two categories for visualization. For a small number of cases, RelocaTE (one location) and PoPoolationTE (five locations) predicted nonreference TE insertions at the same genomic location in multiple samples. These predictions must include false positives based on the fact that each synthetic genome had only a single insertions at different genomic locations. Inclusion of these high-frequency, false-positive predictions dominated the visualization of results for these two methods, and thus predictions for these six cases were filtered prior generating <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> (see <em>Materials and Methods</em> for details).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Positional accuracy of nonreference TE insertions made by methods using split-read evidence on single-insertion synthetic genomes. Data for TEMP are for predictions that do have split-read evidence and may or may not have read-pair evidence. The location of the synthetic TSD is from position 0&#x2013;5 bp on each plot. The darker line for each method indicates predictions averaged across simulated genomes with insertions on the positive strand; the lighter line indicates predictions averaged across simulated genomes with insertions on the negative strand. A value of one would indicate a perfect prediction in all samples since there is one synthetic insertion per genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1548353408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Positional accuracy of nonreference TE insertions made by methods using split-read evidence on single-insertion synthetic genomes. Data for TEMP are for predictions that do have split-read evidence and may or may not have read-pair evidence. The location of the synthetic TSD is from position 0&#x2013;5 bp on each plot. The darker line for each method indicates predictions averaged across simulated genomes with insertions on the positive strand; the lighter line indicates predictions averaged across simulated genomes with insertions on the negative strand. A value of one would indicate a perfect prediction in all samples since there is one synthetic insertion per genome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2763/F2.medium.gif" width="435" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/8/2763/F2.medium.gif" width="435" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486045" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-50" class="first-child">Positional accuracy of nonreference TE insertions made by methods using split-read evidence on single-insertion synthetic genomes. Data for TEMP are for predictions that do have split-read evidence and may or may not have read-pair evidence. The location of the synthetic TSD is from position 0–5 bp on each plot. The darker line for each method indicates predictions averaged across simulated genomes with insertions on the positive strand; the lighter line indicates predictions averaged across simulated genomes with insertions on the negative strand. A value of one would indicate a perfect prediction in all samples since there is one synthetic insertion per genome.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Positional accuracy of nonreference TE insertions made by methods using read-pair evidence on single-insertion synthetic genomes. Data for TEMP are for predictions that do not have split-read evidence but do have read-pair evidence. Note that the y-axes of plots are scaled differently for each method. The location of the synthetic TSD is from position 0&#x2013;5 bp on each plot. The darker line for each method indicates predictions averaged across simulated genomes with insertions on the positive strand; the lighter line indicates predictions averaged across simulated genomes with insertions on the negative strand. A value of one would indicate a perfect prediction in all samples since there is one synthetic insertion per genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1548353408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Positional accuracy of nonreference TE insertions made by methods using read-pair evidence on single-insertion synthetic genomes. Data for TEMP are for predictions that do not have split-read evidence but do have read-pair evidence. Note that the &lt;em&gt;y&lt;/em&gt;-axes of plots are scaled differently for each method. The location of the synthetic TSD is from position 0&#x2013;5 bp on each plot. The darker line for each method indicates predictions averaged across simulated genomes with insertions on the positive strand; the lighter line indicates predictions averaged across simulated genomes with insertions on the negative strand. A value of one would indicate a perfect prediction in all samples since there is one synthetic insertion per genome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2763/F3.medium.gif" width="440" height="272"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/8/2763/F3.medium.gif" width="440" height="272"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485966" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-51" class="first-child">Positional accuracy of nonreference TE insertions made by methods using read-pair evidence on single-insertion synthetic genomes. Data for TEMP are for predictions that do not have split-read evidence but do have read-pair evidence. Note that the <em>y</em>-axes of plots are scaled differently for each method. The location of the synthetic TSD is from position 0–5 bp on each plot. The darker line for each method indicates predictions averaged across simulated genomes with insertions on the positive strand; the lighter line indicates predictions averaged across simulated genomes with insertions on the negative strand. A value of one would indicate a perfect prediction in all samples since there is one synthetic insertion per genome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52"><a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> shows that when ngs_te_mapper makes a prediction, it produces the TSD at the correct location, apparently with no TSDs called too long or too short. Direct analysis of TSD length distributions supports this conclusion: for simulated data, ngs_te_mapper always predicts the correct TSD length for nonreference insertions (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>). For <em>Ty1</em>, <em>Ty2</em>, and <em>Ty4</em>, ngs_te_mapper detected insertions on the positive or negative strand with similar accuracy. Thus, the main difference in detection rates on the positive and negative strands for ngs_te_mapper observed in <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a> appears to be for <em>Ty3</em> insertions, where many fewer insertions were detected correctly on the negative strand. For RelocaTE, the predicted TSDs of nonreference insertions are in approximately the correct locations but with coordinate ranges that are frequently too short (see also Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>). As with ngs_te_mapper, RelocaTE shows the biggest difference in ability to detect <em>Ty3</em> insertions on the negative strand relative to the positive strand. TEMP split-read predictions for <em>Ty1</em>, <em>Ty3</em>, and <em>Ty4</em> are often predicted correctly but with the TSD often annotated to be longer than its true length (see also Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>). Surprisingly, TEMP made no predictions for nonreference <em>Ty2</em> insertions using split-read evidence, perhaps because of the ambiguous signal arising from the similarity of <em>Ty1</em> and <em>Ty2</em> LTR sequences. For TEMP, there is no difference in detection ability for insertions on the positive or negative strand for any family.</p><p id="p-53">Results of the positional accuracy for read-pair methods are shown in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>. For <em>Ty1</em>, <em>Ty3</em>, and <em>Ty4</em> there were very few insertions (only three per family) that TEMP did not have split-read supporting evidence for, and thus few insertions for these families are plotted in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>. In contrast, all <em>Ty2</em> predictions made by TEMP in the single-insertion simulations had read-pair evidence. For all families, when only read-pair evidence is used, TEMP generally predicts an insertion at the correct site, but with some slight inaccuracy on either side. The majority of RetroSeq predictions appear to be clustered close to the true insertion locations, but there appears to be a slight bias for RetroSeq to predict insertions 3′ of where the true TE is located on reference genome coordinates. This bias is potentially introduced by the breakpoint determination step of RetroSeq, which always scans in the 5′ to 3′ direction (see section “Description of McClintock Component Methods” in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>). PoPoolationTE produced the highest number false-positive predictions (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>). When these false-positive nonreference predictions are filtered from the results, all predictions for <em>Ty1</em> and <em>Ty2</em> in the windows around simulated insertions are eliminated. The effect of removing false positives is probably most pronounced for <em>Ty1</em> because it is the most common TE family in <em>S. cerevisiae</em>, and thus would be the most likely family to have a reference insertion with sequence similarity to the synthetic insertion in the vicinity of tRNA genes. PoPoolationTE makes no predictions for <em>Ty2</em>, even including false positives. For <em>Ty3</em> and <em>Ty4</em>, PoPoolationTE has the capability of producing relatively accurate predictions, albeit with low resolution (nearly 100 bp around the true insertion site). For TE-locate, many predictions are made within 500 bp of the true insertion, but they are clearly spread further from the true insertion location than other methods. TE-locate also appears to have a slight bias to predict insertions 5′ of the true insertion location on reference genome coordinates.</p></div><div id="sec-15" class="subsection"><h4>Overlap between methods:</h4><p id="p-54">To understand the concordance of predictions made by the McClintock components, we investigated the overlap among methods for predictions that were made correctly at the sites of synthetic insertions. As shown in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>, different methods have different positional accuracy, and thus we used different windows to classify if a method made a “correct” prediction for a known insertion or not. Predictions for ngs_te_mapper, RelocaTE, TEMP (both split read and read pair), and PoPoolationTE were classified as correct if they had any overlap with the true location of the TSD; while predictions for RetroSeq and TE-locate were classified as correct if they occurred within a 100- or 500-bp window, respectively, of the correct location of the TSD. Neither the orientation nor the TE family was taken into account when classifying a prediction as correct or not. The overlap of correctly detected insertions are shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4, A and B</a>, for split-read and read-pair insertions, respectively. The overlap of correct predictions made by all split-read methods <em>vs.</em> all read-pair methods is shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4C</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Concordance of correctly predicted nonreference insertions among McClintock component methods. (A) The concordance of nonreference predictions by methods that use split-read evidence that overlap with the true location of a synthetic insertion. (B) The concordance of nonreference predictions made by methods that use read-pair evidence that either overlap (TEMP, PoPoolationTE), or are within 100 bp (RetroSeq), or 500 bp (TE-locate) of the true location of a synthetic insertion. (C) The concordance of correctly predicted synthetic nonreference TEs with split-read or read-pair evidence. Predictions for TEMP were partitioned based on whether they had split-read evidence (split-read) or not (read-pair). Counts in all diagrams total 598, the number of simulated samples with single synthetic insertions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1548353408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Concordance of correctly predicted nonreference insertions among McClintock component methods. (A) The concordance of nonreference predictions by methods that use split-read evidence that overlap with the true location of a synthetic insertion. (B) The concordance of nonreference predictions made by methods that use read-pair evidence that either overlap (TEMP, PoPoolationTE), or are within 100 bp (RetroSeq), or 500 bp (TE-locate) of the true location of a synthetic insertion. (C) The concordance of correctly predicted synthetic nonreference TEs with split-read or read-pair evidence. Predictions for TEMP were partitioned based on whether they had split-read evidence (split-read) or not (read-pair). Counts in all diagrams total 598, the number of simulated samples with single synthetic insertions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2763/F4.medium.gif" width="220" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/8/2763/F4.medium.gif" width="220" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485904" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-55" class="first-child">Concordance of correctly predicted nonreference insertions among McClintock component methods. (A) The concordance of nonreference predictions by methods that use split-read evidence that overlap with the true location of a synthetic insertion. (B) The concordance of nonreference predictions made by methods that use read-pair evidence that either overlap (TEMP, PoPoolationTE), or are within 100 bp (RetroSeq), or 500 bp (TE-locate) of the true location of a synthetic insertion. (C) The concordance of correctly predicted synthetic nonreference TEs with split-read or read-pair evidence. Predictions for TEMP were partitioned based on whether they had split-read evidence (split-read) or not (read-pair). Counts in all diagrams total 598, the number of simulated samples with single synthetic insertions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56"><a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a> shows that the majority of split-read predictions are supported by at least two methods (<em>n</em> = 340, 57%) but that each method made many correct TE predictions that were not made by any other method. RelocaTE and TEMP made a greater number of correct overlapping predictions with each other than either of these method did with ngs_te_mapper. <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a> also shows that 16% (<em>n</em> = 94) of synthetic insertions were not predicted by any split-read method at the threshold of positional accuracy used here. <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a> shows that the vast majority of synthetic TE insertions (<em>n</em> = 428, 72%) are predicted by at least two of the read-pair methods, but that only 24% (<em>n</em> = 143) of insertions are supported by three or more methods. RetroSeq and TE-locate make the highest number of unique correct predictions. ∼10% (<em>n</em> = 58) of synthetic insertion samples were not predicted by any read-pair method at the threshold of positional accuracy used here. Finally, <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4C</a> shows that, while the overwhelming majority of insertions are predicted by at least one split-read and one read-pair method (<em>n</em> = 470, 79%), there are many insertions that are only predicted using one type of evidence or the other (<em>n</em> = 104, 17%) given the thresholds of positional accuracy used here. Nevertheless, use of all six methods recovers nearly 96% of synthetic insertions, demonstrating the utility of integrating multiple TE-identification methods enabled by McClintock.</p></div></div><div id="sec-16" class="subsection"><h3>Application of McClintock to 93 yeast genomes</h3><p id="p-57">The previous sections presented results on the accuracy of McClintock component methods on simulated resequencing data. Simulations are useful for testing methods under controlled settings, but do not capture all aspects of how methods perform when applied to real data. Since much is known about the expected insertion preferences of TEs in <em>S. cerevisiae</em> (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Gafner and Philippsen 1980</a>; <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Ji <em>et al.</em> 1993</a>; <a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Chalker and Sandmeyer 1990</a>, <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">1992</a>; <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Devine and Boeke 1996</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Rinckel and Garfinkel 1996</a>; <a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Zou <em>et al.</em> 1996</a>; <a id="xref-ref-33-11" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Baller <em>et al.</em> 2012</a>; <a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Mularoni <em>et al.</em> 2012</a>; <a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Qi <em>et al.</em> 2012</a>), analysis of real WGS data sets can be used as an alternative approach to evaluate if McClintock component methods can recapitulate the known genome biology of yeast TEs. To do this, we analyzed 93 high-coverage <em>S. cerevisiae</em> WGS data sets from <a id="xref-ref-64-5" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> (2015)</a> using McClintock to generate TE predictions for all six component methods. <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a> and Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a> show how many of the nonreference and reference TEs per strain, respectively, are detected by the different McClintock component methods across all 93 samples. In general, split-read methods predict between 5 and 20 nonreference TE insertions per strain, whereas read-pair methods predict ∼40–100 nonreference TE insertions per strain (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>). Numbers of reference TEs predicted per strain in real data (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) are generally lower than in simulated genomes (<a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a> and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>). The exceptions to this pattern are TEMP and PoPoolationTE, which show similar or higher numbers of reference TE predictions per strain in real data relative to simulations. We note that for a few strains in the <a id="xref-ref-64-6" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> (2015)</a> data set, TE-locate predicted several hundred nonreference insertions; these strains did not appear to be outliers in terms of their nonreference TE content based on other methods (results not shown).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Numbers of nonreference TE insertions per strain predicted by McClintock component methods in real yeast genomes. Predictions for TEMP were partitioned based on whether they had split-read evidence (split-read) or not (read-pair). Data are from 93 yeast strains taken from Strope et al. (2015). Methods are classified based on whether they use split-read or read-pair evidence to make a nonreference TE prediction. The box plot is shown on a  scale. The thick line indicates the median, the colored box is the interquartile range, the whiskers mark the most extreme data point which is no more than 1.5 times the interquartile range from the box, and the &#x25CB;&#x2019;s are outliers. Note that for TE-locate, several outlier samples generated hundreds of predicted nonreference TE insertions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1548353408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Numbers of nonreference TE insertions per strain predicted by McClintock component methods in real yeast genomes. Predictions for TEMP were partitioned based on whether they had split-read evidence (split-read) or not (read-pair). Data are from 93 yeast strains taken from Strope &lt;em&gt;et al.&lt;/em&gt; (2015). Methods are classified based on whether they use split-read or read-pair evidence to make a nonreference TE prediction. The box plot is shown on a &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-1&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; scale. The thick line indicates the median, the colored box is the interquartile range, the whiskers mark the most extreme data point which is no more than 1.5 times the interquartile range from the box, and the &#x25CB;&#x2019;s are outliers. Note that for TE-locate, several outlier samples generated hundreds of predicted nonreference TE insertions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2763/F5.medium.gif" width="436" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/8/2763/F5.medium.gif" width="436" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485937" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-58" class="first-child">Numbers of nonreference TE insertions per strain predicted by McClintock component methods in real yeast genomes. Predictions for TEMP were partitioned based on whether they had split-read evidence (split-read) or not (read-pair). Data are from 93 yeast strains taken from <a id="xref-ref-64-7" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> (2015)</a>. Methods are classified based on whether they use split-read or read-pair evidence to make a nonreference TE prediction. The box plot is shown on a <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2763/F5/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2763/F5/embed/mml-math-1.gif"/></noscript></span></span> scale. The thick line indicates the median, the colored box is the interquartile range, the whiskers mark the most extreme data point which is no more than 1.5 times the interquartile range from the box, and the ○’s are outliers. Note that for TE-locate, several outlier samples generated hundreds of predicted nonreference TE insertions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">We evaluated the quality of nonreference TE predictions made by McClintock component methods on the <a id="xref-ref-64-8" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> (2015)</a> data set using three aspects of the known biology of TEs in <em>S. cerevisiae</em>: (i) activity of families, (ii) tRNA targeting, and (iii) TSD length. Our expectations based on prior knowledge of yeast TE biology are that methods that make high quality nonreference TE predictions should (i) show few nonreference predictions for inactive TE families (<em>Ty3_1p</em> and <em>Ty5</em>), (ii) show a high proportion of nonreference predictions in the vicinity of tRNA genes, and (iii) show characteristic 5-bp TSDs for nonreference predictions made by split-read methods.</p><div id="sec-17" class="subsection"><h4>Prediction of active and inactive families:</h4><p id="p-60"><a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> shows numbers of nonreference TE predictions made by McClintock component methods across all strains in the <a id="xref-ref-64-9" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> (2015)</a> data set. As expected, all methods predicted multiple nonreference insertions for TE families that are known to be active in this species. Additionally, ngs_te_mapper and TEMP make no nonreference TE predictions for both inactive families in <em>S. cerevisiae</em>, supporting simulation results above that show these methods have low false-positive rates. RelocaTE makes nonreference TE predictions for <em>Ty3_1p</em> but not <em>Ty5</em>, PoPoolationTE makes nonreference TE predictions for <em>Ty5</em> but not <em>Ty3_1p</em>, and both RetroSeq and TE-locate predict nonreference insertions for <em>Ty3_1p</em> and <em>Ty5</em>. RelocaTE is the only split-read method that predicts nonreference insertions for an inactive family, suggesting that split-read methods generally have a higher ability to discriminate active from inactive TE families. Compared to the total numbers predicted for other active TE families, the three pure read-pair methods predicted fewer nonreference insertions for both <em>Ty3_1p</em> and <em>Ty5</em>, suggesting false-positive rates for these methods are not so high as to overwhelm true signal. The one exception is for TE-locate, which predicted relatively high numbers of <em>Ty5</em> insertions, which is likely related to the outlier samples noted above where TE-locate predicts hundreds of presumably false-positive nonreference insertions.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486143/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486143/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Number and location of nonreference TEs predicted by McClintock component methods in 93 yeast genomes</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h4>Predicted insertions in tRNA regions:</h4><p id="p-62">Active TE families in <em>S. cerevisiae</em> are known to target tRNA genes (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Ji <em>et al.</em> 1993</a>; <a id="xref-ref-8-8" class="xref-bibr" href="#ref-8">Chalker and Sandmeyer 1990</a>, <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">1992</a>; <a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Devine and Boeke 1996</a>; <a id="xref-ref-33-12" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Baller <em>et al.</em> 2012</a>; <a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Mularoni <em>et al.</em> 2012</a>; <a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Qi <em>et al.</em> 2012</a>). The highest density of <em>Ty1</em> and <em>Ty2</em> insertions are in the 200 bp upstream of the tRNA transcription start site (<a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Ji <em>et al.</em> 1993</a>; <a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Devine and Boeke 1996</a>; <a id="xref-ref-33-13" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Baller <em>et al.</em> 2012</a>; <a id="xref-ref-44-7" class="xref-bibr" href="#ref-44">Mularoni <em>et al.</em> 2012</a>). <em>Ty3</em> targets a specific location just upstream of tRNA gene transcription start sites (<a id="xref-ref-8-9" class="xref-bibr" href="#ref-8">Chalker and Sandmeyer 1990</a>, <a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">1992</a>; <a id="xref-ref-33-14" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">Qi <em>et al.</em> 2012</a>). Patterns of <em>Ty4</em> insertions have not been experimentally determined, although the locations of insertions in the reference genome suggest a similar pattern to that of <em>Ty1</em> and <em>Ty2</em> (<a id="xref-ref-33-15" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>).</p><p id="p-63">To evaluate if nonreference TE insertions predicted by McClintock component methods show expected hallmarks of tRNA targeting, we plotted locations of nonreference TE insertions identified in the <a id="xref-ref-64-10" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> (2015)</a> strains using split-read evidence and read-pair evidence in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> and <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, respectively. The expected profiles of insertion into tRNA gene regions is observed for all <em>Ty</em> families for ngs_te_mapper, RelocaTE, TEMP, and RetroSeq, albeit with the different levels of resolution that are characteristic of each method. Consistent with simulation data (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>), TEMP appears to have difficulty predicting <em>Ty2</em> using split-read data in real yeast genomes, and this effect also appears to impact prediction of <em>Ty1</em> insertions using split-read data in real data (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). PoPoolationTE can predict meaningful profiles of insertion for <em>Ty3</em> and <em>Ty4</em> (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>), as expected based on simulation data (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>). However, in contrast to simulation data where only putative false positives are predicted (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>), PoPoolationTE also predicts nonreference insertions for <em>Ty1</em> and <em>Ty2</em> in real data (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>). Since PoPoolationTE predicts reference and nonreference insertions in the same way, and since many <em>Ty1</em> and <em>Ty2</em> insertions exist in the reference genome upstream regions of tRNA genes, it is possible that these <em>Ty1</em> and <em>Ty2</em> insertions predicted in the <a id="xref-ref-64-11" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> (2015)</a> data set are actually reference insertions that are mislabeled by PoPoolationTE as nonreference insertions. Finally, nonreference insertions predicted by TE-locate are only weakly enriched in tRNA regions for all families, and the positional profiles produced by TE-locate are shifted relative to expectations and predictions made by other methods.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Locations of nonreference TE predictions relative to tRNA genes made by methods using split-read evidence on real yeast genomes. Data for TEMP are for predictions that do have split-read evidence and may or may not have read-pair evidence. The transcription start site of each tRNA gene is aligned at position zero, taking into account tRNA gene orientation, for a window extending 1 kb upstream and 500 bp downstream. The frequency of a prediction at each base is counted across all 93 strains in the Strope et al. (2015) data set, then averaged across the 299 tRNA genes, and plotted as a line for each method and TE family. These plots show all predictions and therefore include allelic predictions present in more than one strain. Also, any given strain may have more than one insertion at the same relative location in a tRNA gene, and thus the scale for these plots can go above one." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1548353408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Locations of nonreference TE predictions relative to tRNA genes made by methods using split-read evidence on real yeast genomes. Data for TEMP are for predictions that do have split-read evidence and may or may not have read-pair evidence. The transcription start site of each tRNA gene is aligned at position zero, taking into account tRNA gene orientation, for a window extending 1 kb upstream and 500 bp downstream. The frequency of a prediction at each base is counted across all 93 strains in the Strope &lt;em&gt;et al.&lt;/em&gt; (2015) data set, then averaged across the 299 tRNA genes, and plotted as a line for each method and TE family. These plots show all predictions and therefore include allelic predictions present in more than one strain. Also, any given strain may have more than one insertion at the same relative location in a tRNA gene, and thus the scale for these plots can go above one.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2763/F6.medium.gif" width="440" height="389"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/8/2763/F6.medium.gif" width="440" height="389"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485920" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-64" class="first-child">Locations of nonreference TE predictions relative to tRNA genes made by methods using split-read evidence on real yeast genomes. Data for TEMP are for predictions that do have split-read evidence and may or may not have read-pair evidence. The transcription start site of each tRNA gene is aligned at position zero, taking into account tRNA gene orientation, for a window extending 1 kb upstream and 500 bp downstream. The frequency of a prediction at each base is counted across all 93 strains in the <a id="xref-ref-64-12" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> (2015)</a> data set, then averaged across the 299 tRNA genes, and plotted as a line for each method and TE family. These plots show all predictions and therefore include allelic predictions present in more than one strain. Also, any given strain may have more than one insertion at the same relative location in a tRNA gene, and thus the scale for these plots can go above one.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Locations of nonreference TE predictions relative to tRNA genes made by methods using read-pair evidence on real yeast genomes. Data for TEMP are for predictions that do not have split-read evidence but do have read-pair evidence. The transcription start site of each tRNA gene is aligned at zero on the plots, taking into account orientation, for a window extending 1 kb upstream and 500 bp downstream. The frequency of a prediction at each base is counted across all 93 strains in the Strope et al. (2015) data set, then averaged across the 299 tRNA genes, and plotted as a line for each method and TE family. These plots show all predictions and therefore include allelic predictions present in more than one strain. Also, any given strain may have more than one insertion at the same relative location in a tRNA gene, and thus the scale for these plots can go above one." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1548353408" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Locations of nonreference TE predictions relative to tRNA genes made by methods using read-pair evidence on real yeast genomes. Data for TEMP are for predictions that do not have split-read evidence but do have read-pair evidence. The transcription start site of each tRNA gene is aligned at zero on the plots, taking into account orientation, for a window extending 1 kb upstream and 500 bp downstream. The frequency of a prediction at each base is counted across all 93 strains in the Strope &lt;em&gt;et al.&lt;/em&gt; (2015) data set, then averaged across the 299 tRNA genes, and plotted as a line for each method and TE family. These plots show all predictions and therefore include allelic predictions present in more than one strain. Also, any given strain may have more than one insertion at the same relative location in a tRNA gene, and thus the scale for these plots can go above one.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/8/2763/F7.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/8/2763/F7.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/8/2763/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485879" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-65" class="first-child">Locations of nonreference TE predictions relative to tRNA genes made by methods using read-pair evidence on real yeast genomes. Data for TEMP are for predictions that do not have split-read evidence but do have read-pair evidence. The transcription start site of each tRNA gene is aligned at zero on the plots, taking into account orientation, for a window extending 1 kb upstream and 500 bp downstream. The frequency of a prediction at each base is counted across all 93 strains in the <a id="xref-ref-64-13" class="xref-bibr" href="#ref-64">Strope <em>et al.</em> (2015)</a> data set, then averaged across the 299 tRNA genes, and plotted as a line for each method and TE family. These plots show all predictions and therefore include allelic predictions present in more than one strain. Also, any given strain may have more than one insertion at the same relative location in a tRNA gene, and thus the scale for these plots can go above one.</p><div class="sb-div caption-clear"></div></div></div><p id="p-66">To quantify the proportion of nonreference TEs that were predicted in tRNA regions, we counted predictions 1000 bp upstream and 500 bp downstream of a tRNA gene, taking into account the orientation of the tRNA gene but not the orientation of the TE insertion. The expected percentage of TEs located in these regions if they were inserted randomly in the genome would be 0.037% [(299 tRNA genes <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2763/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/8/2763/embed/mml-math-2.gif"/></noscript></span></span> 1500 bp window)/12,162,995 bp genome]. Previous analyses of tRNA targeting of TEs in the <em>S. cerevisiae</em> reference genome (<a id="xref-ref-33-16" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>) assessed whether TEs were within 750 bp of a tRNA gene or other RNA polymerase III gene (excluding other intervening TE sequences). Here we use extended regions for tRNA targeting based on the inaccuracy in nonreference predictions observed for some methods in the simulations above. For comparison with previous results, we first applied our definition of tRNA targeting to the reference TE annotation from <a id="xref-ref-7-11" class="xref-bibr" href="#ref-7">Carr <em>et al.</em> (2012)</a> (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). Estimated proportions of <em>Ty</em> elements in tRNA regions for the <a id="xref-ref-7-12" class="xref-bibr" href="#ref-7">Carr <em>et al.</em> (2012)</a> reference annotation are lower than those reported by <a id="xref-ref-33-17" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> (1998)</a>, however, they still show highly biased targeting toward tRNA regions.</p><p id="p-67">Nonreference TE predictions of all four active <em>Ty</em> elements show the expected enrichment in tRNA regions for each McClintock component method (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). For all methods, <em>Ty3</em> is the active TE family most strongly associated with tRNA regions, consistent with experimental data and observations based on the reference genome (<a id="xref-ref-8-10" class="xref-bibr" href="#ref-8">Chalker and Sandmeyer 1990</a>, <a id="xref-ref-9-8" class="xref-bibr" href="#ref-9">1992</a>; <a id="xref-ref-33-18" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-49-8" class="xref-bibr" href="#ref-49">Qi <em>et al.</em> 2012</a>). Split-read methods predict a higher proportion of nonreference TEs in tRNA regions relative to expectations based on TEs in the reference genome. For read-pair methods, at least one TE family showed a lower proportion of nonreference TEs in tRNA regions relative to reference TEs. We interpret this observation to be due to the lower positional accuracy of read-pair methods. TE-locate consistently predicted the lowest number of TEs in tRNA regions for active <em>Ty</em> families, though predicted insertions for this method still showed an enrichment in tRNA regions relative to random expectation. We interpret the low tRNA enrichment for TE-locate to be a consequence of the low positional accuracy of read-pair methods combined with the presence of outlier samples for this method which have very high numbers of nonreference predictions.</p><p id="p-68">As discussed above, nonreference predictions were made by RelocaTE, RetroSeq, and TE-locate for the inactive <em>Ty3_1p</em> family. Despite most likely being false positives, these predictions were predominantly in tRNA regions, suggesting they could either be nonreference <em>Ty3</em> insertions that are miscalled as nonreference <em>Ty3_1p</em>, or reference <em>Ty3_1p</em> insertions called as nonreference <em>Ty3_1p</em> insertions. Nonreference predictions were also made by RetroSeq, PoPoolationTE, and TE-locate for the inactive <em>Ty5</em> family. The majority of these predictions are made outside of the tRNA regions, as is expected based on the known location of <em>Ty5</em> insertions in the reference genome prior knowledge about <em>Ty5</em> target preferences (<a id="xref-ref-71-4" class="xref-bibr" href="#ref-71">Zou <em>et al.</em> 1996</a>; <a id="xref-ref-33-19" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baller <em>et al.</em> 2011</a>). These nonreference TE predictions may be false positives (possibly caused by mapping inconsistencies in heterochromatic regions where <em>Ty5</em> elements typically insert) or real nonreference “insertions” that arose by recombination events rather than transposition events (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zou <em>et al.</em> 1995</a>).</p></div><div id="sec-19" class="subsection"><h4>Prediction of TSDs by split-read methods:</h4><p id="p-69">Finally, we evaluated the performance of split-read methods to predict the known TSD lengths of active yeast <em>Ty</em> families in real WGS data. All available experimental and genomic data indicates that active yeast <em>Ty</em> families create 5 bp TSDs on insertion (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Gafner and Philippsen 1980</a>; <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Rinckel and Garfinkel 1996</a>; <a id="xref-ref-8-11" class="xref-bibr" href="#ref-8">Chalker and Sandmeyer 1990</a>; <a id="xref-ref-33-20" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 1998</a>; <a id="xref-ref-71-5" class="xref-bibr" href="#ref-71">Zou <em>et al.</em> 1996</a>). TSD length distributions of unique insertion sites for McClintock predictions in real yeast genomes are shown for <em>Ty1</em>, <em>Ty2</em>, <em>Ty3</em>, and <em>Ty4</em> in Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>. As observed in simulated data (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a> and Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1/FileS1.pdf">File S1</a>), ngs_te_mapper predictions had the highest proportion of correct TSD lengths predicted per family. However, in contrast to simulated data, ngs_te_mapper can infrequently make incorrect TSD-length predictions in real data. Confirming simulation results, RelocaTE generally underpredicts the length of TSDs, and TEMP consistently overpredicts the lengths of TSDs for all families in real data. For all split-read methods, the modal value of the TSD-length distribution reflects the true TSD length for all families. Thus, the modal TSD length provided by each of the split-read methods yields biologically meaningful inferences about TSD structure.</p></div></div></div><div class="section conclusions" id="sec-20"><h2 class="">Conclusions and Future Directions</h2><p id="p-70">Here we describe McClintock, an integrated pipeline for detecting TE insertions in WGS resequencing data. McClintock offers many advantages relative to running multiple TE detection methods in isolation. Specific versions of compatible software dependencies required to run each method are fully documented, allowing users to easily set up their environment. The number of input files required to run all methods is reduced and complex processing of input files to create the correct custom formats and file relationships is automated. In addition, the pipeline is structured to allow parallel computations for multiple samples, so population data sets can be analyzed more quickly. Finally, results from individual methods are standardized to facilitate comparisons across methods and easy visualization in the UCSC genome browser. Overall, McClintock greatly lowers the barriers to running multiple TE detection methods, allowing users to gain more insight into how various methods work for their samples. McClintock does not currently include all published TE detection methods, although additional methods can be easily incorporated into the pipeline due to the flexible architecture and open-source nature of the project.</p><p id="p-71">In addition, we have applied McClintock to simulated and real yeast WGS samples to evaluate the performance of McClintock component methods. Simulations on the unmodified <em>S. cerevisiae</em> reference genomes reveal that sequencing coverage influences detection of reference TEs, but that recovery of reference TE insertions and false-positive rates for nonreference TE insertions are generally low even at high sequencing coverage. Simulations on <em>S. cerevisiae</em> reference genomes including a single nonreference insertion showed that pure split-read methods may detect fewer TE insertions than read-pair methods, but they have much higher positional accuracy. Single-insertion simulations also revealed that the TE family affects the ability of methods to detect nonreference TE insertions. We find substantial difference in the ability of McClintock component methods to detect subsets of nonreference insertions in the yeast genome, but that by combining multiple methods that use split-read and read-pair data, nonreference TEs at nearly all biologically realistic locations can be detected in simulated data. Finally, application of McClintock to a large sample of real yeast genomes reveals that most but not all McClintock component methods can recover known aspects of TE biology in yeast such as family activity status, tRNA gene targeting, and TSD structure. Together, our results suggest that even in the context of a simplified model eukaryotic genome like <em>S. cerevisiae</em>, current TE detection methods using short-read data do not provide comprehensive recovery of all TE insertions in WGS resequencing samples. Further performance studies in other genomic contexts, including newer methods not currently included in McClintock, are needed to generalize the results presented here, and to provide a road map for developing more advanced systems for the detection of TEs in unassembled short-read genomic data.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-72">We thank Matthew Ronshaugen, Douda Bensasson, and members of the Bergman Laboratory for helpful comments throughout the project; Robert Kofler, Thomas Keane, Alexander Platzer, Sofia Robb, and Jiali Zhuang for code fixes and insights into McClintock component methods; and Nick Gresham for high-performance cluster computing assistance. This work was supported by Wellcome Trust Award 096602/B/11/Z (M.G.N.) and Human Frontier Science Program grant RGY0093/2012 (C.M.B.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-15"><p id="p-73">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.043893/-/DC1</a>.</p></li><li class="fn" id="fn-16"><p id="p-74"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-02-14"><span class="received-label">Received </span>February 14, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-06-15"><span class="accepted-label">Accepted </span>June 15, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Nelson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.1" data-doi="10.1111/mec.13341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zalar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imanishi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A population genomics insight into the Mediterranean origins of wine yeast domestication.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">5412</span>–<span class="cit-lpage">5427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D5412%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.2" data-doi="10.1073/pnas.1103665108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baller</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Access to DNA establishes a secondary target site bias for the yeast retrotransposon Ty5.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">20351</span>–<span class="cit-lpage">20356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1103665108%26rft_id%253Dinfo%253Apmid%252F21788500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzUxLzIwMzUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.3" data-doi="10.1101/gr.129585.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baller</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamenova</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A nucleosomal surface defines an integration hotspot for the Saccharomyces cerevisiae Ty1 retrotransposon.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">704</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129585.111%26rft_id%253Dinfo%253Apmid%252F22219511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzQvNzA0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.4" data-doi="10.1186/1471-2105-15-293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bardou</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mariette</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Escudié</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Djemiel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klopp</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Jvenn: an interactive Venn diagram viewer.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-293%26rft_id%253Dinfo%253Apmid%252F25176396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25176396&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A proposal for the reference-based annotation of de novo transposable element insertions.</span> <abbr class="cit-jnl-abbrev">Mob. Genet. Elements</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMob%2BGenet%2BElements%26rft.aulast%253DBergman%26rft.auinit1%253DC.%2BM.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D54%26rft.atitle%253DA%2Bproposal%2Bfor%2Bthe%2Breference-based%2Bannotation%2Bof%2Bde%2Bnovo%2Btransposable%2Belement%2Binsertions.%26rft_id%253Dinfo%253Apmid%252F22754753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22754753&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.6" data-doi="10.1093/bib/bbm048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quesneville</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Discovering and detecting transposable elements in genome sequences.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">382</span>–<span class="cit-lpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbm048%26rft_id%253Dinfo%253Apmid%252F17932080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbm048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17932080&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251197600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.7" data-doi="10.1371/journal.pone.0050978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bensasson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolutionary genomics of transposable elements in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e50978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D7%26rft.spage%253De50978%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0050978%26rft_id%253Dinfo%253Apmid%252F23226439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0050978&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23226439&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chalker</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandmeyer</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Transfer RNA genes are genomic targets for de Novo transposition of the yeast retrotransposon Ty3.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChalker%26rft.auinit1%253DD.%2BL.%26rft.volume%253D126%26rft.issue%253D4%26rft.spage%253D837%26rft.epage%253D850%26rft.atitle%253DTransfer%2BRNA%2Bgenes%2Bare%2Bgenomic%2Btargets%2Bfor%2Bde%2BNovo%2Btransposition%2Bof%2Bthe%2Byeast%2Bretrotransposon%2BTy3.%26rft_id%253Dinfo%253Apmid%252F1963869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI2LzQvODM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.9" data-doi="10.1101/gad.6.1.117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chalker</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandmeyer</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Ty3 integrates within the region of RNA polymerase III transcription initiation.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DChalker%26rft.auinit1%253DD%2BL%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D117%26rft.epage%253D128%26rft.atitle%253DTy3%2Bintegrates%2Bwithin%2Bthe%2Bregion%2Bof%2BRNA%2Bpolymerase%2BIII%2Btranscription%2Binitiation.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.6.1.117%26rft_id%253Dinfo%253Apmid%252F1309715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiNi8xLzExNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-7.8.2763.10"><div class="cit-metadata unstructured">Chen, J., T. Wrightsman, S. R. Wessler, and J. E. Stajich, 2017 RelocaTE2: A High Resolution Transposable Element Polymorphism Mapping Tool for Population Resequencing. PeerJ. 5: e2942.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.11" data-doi="10.1093/bioinformatics/bts665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tannir</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinstein</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">VirusSeq: software to identify viruses and their integration sites using next-generation sequencing of human cancer tissue.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">266</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts665%26rft_id%253Dinfo%253Apmid%252F23162058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts665&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23162058&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.12" data-doi="10.1093/molbev/mst129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cridland</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Abundance and distribution of transposable elements in two Drosophila QTL mapping resources.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2311</span>–<span class="cit-lpage">2327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst129%26rft_id%253Dinfo%253Apmid%252F23883524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23883524&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326744700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.13" data-doi="10.1101/gad.10.5.620"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devine</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Integration of the yeast retrotransposon Ty1 is targeted to regions upstream of genes transcribed by RNA polymerase III.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">620</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DDevine%26rft.auinit1%253DS%2BE%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D620%26rft.epage%253D633%26rft.atitle%253DIntegration%2Bof%2Bthe%2Byeast%2Bretrotransposon%2BTy1%2Bis%2Btargeted%2Bto%2Bregions%2Bupstream%2Bof%2Bgenes%2Btranscribed%2Bby%2BRNA%2Bpolymerase%2BIII.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.5.620%26rft_id%253Dinfo%253Apmid%252F8598291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTAvNS82MjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.14" data-doi="10.1186/s13100-015-0055-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transposable element detection from whole genome sequence data.</span> <abbr class="cit-jnl-abbrev">Mob. DNA</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMob.%2BDNA%26rft.volume%253D6%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13100-015-0055-3%26rft_id%253Dinfo%253Apmid%252F26719777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13100-015-0055-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26719777&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.15" data-doi="10.1101/gr.106419.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kazazian</span>  <span class="cit-name-given-names">H. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">High-throughput sequencing reveals extensive variation in human-specific L1 content in individual human genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1262</span>–<span class="cit-lpage">1270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.106419.110%26rft_id%253Dinfo%253Apmid%252F20488934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.16" data-doi="10.1101/gr.114777.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kazazian</span>  <span class="cit-name-given-names">H. H.</span>  <span class="cit-name-suffix">Jr.</span>.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome resequencing allows detection of many rare LINE-1 insertion alleles in humans.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">985</span>–<span class="cit-lpage">990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.114777.110%26rft_id%253Dinfo%253Apmid%252F20980553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.17" data-doi="10.1002/yea.1005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fingerman</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dombrowski</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sniegowski</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Distribution and sequence analysis of a novel Ty3-like element in natural Saccharomyces paradoxus isolates.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DFingerman%26rft.auinit1%253DE.%2BG.%26rft.volume%253D20%26rft.issue%253D9%26rft.spage%253D761%26rft.epage%253D770%26rft.atitle%253DDistribution%2Band%2Bsequence%2Banalysis%2Bof%2Ba%2Bnovel%2BTy3-like%2Belement%2Bin%2Bnatural%2BSaccharomyces%2Bparadoxus%2Bisolates.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1005%26rft_id%253Dinfo%253Apmid%252F12845602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12845602&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fiston-Lavier</span>  <span class="cit-name-given-names">A.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrigan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">T-lex: a program for fast and accurate assessment of transposable element presence using next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DFiston-Lavier%26rft.auinit1%253DA.%2BS.%26rft.volume%253D39%26rft.issue%253D6%26rft.spage%253De36%26rft.epage%253De36%26rft.atitle%253DT-lex%253A%2Ba%2Bprogram%2Bfor%2Bfast%2Band%2Baccurate%2Bassessment%2Bof%2Btransposable%2Belement%2Bpresence%2Busing%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Apmid%252F21177644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21177644&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.19" data-doi="10.1093/nar/gku1250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fiston-Lavier</span>  <span class="cit-name-given-names">A.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrón</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">T-lex2: genotyping, frequency estimation and re-annotation of transposable elements using single or pooled next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">e22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1250%26rft_id%253Dinfo%253Apmid%252F25510498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25510498&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.20" data-doi="10.1038/286414a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gafner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippsen</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">The yeast transposon Ty1 generates duplications of target DNA on insertion.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">414</span>–<span class="cit-lpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGafner%26rft.auinit1%253DJ.%26rft.volume%253D286%26rft.issue%253D5771%26rft.spage%253D414%26rft.epage%253D418%26rft.atitle%253DThe%2Byeast%2Btransposon%2BTy1%2Bgenerates%2Bduplications%2Bof%2Btarget%2BDNA%2Bon%2Binsertion.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F286414a0%26rft_id%253Dinfo%253Apmid%252F6250065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/286414a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6250065&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.21" data-doi="10.1186/s12859-014-0377-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilly</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etcheverry</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madoui</span>  <span class="cit-name-given-names">M.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quadrana</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TE-Tracker: systematic identification of transposition events through whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-014-0377-z%26rft_id%253Dinfo%253Apmid%252F25408240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-014-0377-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25408240&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.22" data-doi="10.1126/science.274.5287.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffeau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Life with 6000 genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D274%26rft.spage%253D563%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.274.5287.546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.274.5287.546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.23" data-doi="10.1186/s12864-015-1860-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawkey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamidian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wick</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Billman-Jacobe</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">ISMapper: identifying transposase insertion sites in bacterial genomes from short read sequence data.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D667%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1860-2%26rft_id%253Dinfo%253Apmid%252F26336060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1860-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26336060&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.24" data-doi="10.1186/s12864-015-1975-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hénaff</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zapata</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casacuberta</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Jitterbug: somatic and germline transposon insertion detection at single-nucleotide resolution.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D768%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1975-5%26rft_id%253Dinfo%253Apmid%252F26459856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1975-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26459856&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoen</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourque</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casacuberta</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordaux</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A call for benchmarking transposable element annotation methods.</span> <abbr class="cit-jnl-abbrev">Mob. DNA</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMob.%2BDNA%26rft.volume%253D6%26rft.spage%253D13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.26" data-doi="10.1093/bioinformatics/btq216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hormozdiari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hajirasouliha</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dao</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hach</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yorukoglu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Next-generation VariationHunter: combinatorial algorithms for transposon insertion discovery.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">i350</span>–<span class="cit-lpage">i357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq216%26rft_id%253Dinfo%253Apmid%252F20529927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20529927&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278689000043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.27" data-doi="10.1016/0092-8674(93)90278-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braiterman</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Hotspots for unselected Ty1 transposition events on yeast chromosome III are near tRNA genes and LTR sequences.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">1007</span>–<span class="cit-lpage">1018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DJi%26rft.auinit1%253DH.%26rft.volume%253D73%26rft.issue%253D5%26rft.spage%253D1007%26rft.epage%253D1018%26rft.atitle%253DHotspots%2Bfor%2Bunselected%2BTy1%2Btransposition%2Bevents%2Bon%2Byeast%2Bchromosome%2BIII%2Bare%2Bnear%2BtRNA%2Bgenes%2Band%2BLTR%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252893%252990278-X%26rft_id%253Dinfo%253Apmid%252F8388781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(93)90278-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8388781&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LF06100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.28" data-doi="10.1186/s12859-015-0507-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdier</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">ITIS, a bioinformatics tool for accurate identification of transposon insertion sites using next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-015-0507-2%26rft_id%253Dinfo%253Apmid%252F25887332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-015-0507-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25887332&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-other" id="cit-7.8.2763.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaminker</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kronmiller</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svirskas</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The transposable elements of the Drosophila melanogaster euchromatin: a genomics perspective.</span> <span class="cit-source">Genome Biol.</span> <span class="cit-vol">3</span>: <span class="cit-fpage">research0084</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D3%26rft.spage%253Dresearch0084%26rft_id%253Dinfo%253Apmid%252F12537573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12537573&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.30" data-doi="10.1093/bioinformatics/bts697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RetroSeq: transposable element discovery from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts697%26rft_id%253Dinfo%253Apmid%252F23233656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23233656&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314892000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.8.2763.31" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT—the BLAST-like alignment tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.32" data-doi="10.1093/bioinformatics/btq351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zweig</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinrichs</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karolchik</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BigWig and BigBed: enabling browsing of large distributed datasets.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2204</span>–<span class="cit-lpage">2207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq351%26rft_id%253Dinfo%253Apmid%252F20639541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20639541&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281738900023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanguri</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabriel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Transposable elements and genome organization: a comprehensive survey of retrotransposons revealed by the complete Saccharomyces cerevisiae genome sequence.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">464</span>–<span class="cit-lpage">478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DKim%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D464%26rft.epage%253D478%26rft.atitle%253DTransposable%2BElements%2Band%2BGenome%2BOrganization%253A%2BA%2BComprehensive%2BSurvey%2Bof%2BRetrotransposons%2BRevealed%2Bby%2Bthe%2BComplete%2BSaccharomyces%2Bcerevisiae%2BGenome%25A0Sequence%26rft_id%253Dinfo%253Apmid%252F9582191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjgvNS80NjQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.34" data-doi="10.1371/journal.pgen.1002487"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betancourt</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sequencing of pooled DNA samples (pool-seq) uncovers complex dynamics of transposable element insertions in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKofler%26rft.auinit1%253DR.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253De1002487%26rft.epage%253De1002487%26rft.atitle%253DSequencing%2Bof%2Bpooled%2BDNA%2Bsamples%2B%2528Pool-Seq%2529%2Buncovers%2Bcomplex%2Bdynamics%2Bof%2Btransposable%2Belement%2Binsertions%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002487%26rft_id%253Dinfo%253Apmid%252F22291611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002487&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22291611&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.35" data-doi="10.1093/molbev/msw137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gómez-Sánchez</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">PoPoolationTE2: comparative population genomics of transposable elements using pool-seq.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2759</span>–<span class="cit-lpage">2764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw137%26rft_id%253Dinfo%253Apmid%252F27486221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27486221&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.8.2763.36" data-doi="10.1093/bib/bbs038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haussler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The UCSC genome browser and associated tools.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">144</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs038%26rft_id%253Dinfo%253Apmid%252F22908213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbs038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22908213&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.8.2763.37" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.38" data-doi="10.1126/science.1222077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iskow</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gokcumen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haseley</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Landscape of somatic retrotransposition in human cancers.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1222077%26rft_id%253Dinfo%253Apmid%252F22745252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5Ny85NjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lerat</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identifying repeats and transposable elements in sequenced genomes: how to find your way through the dense forest of programs.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D104%26rft.spage%253D520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-7.8.2763.40"><div class="cit-metadata unstructured">Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv: 1303.3997.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.41" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.42" data-doi="10.1371/journal.pone.0030008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linheiro</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Whole genome resequencing reveals natural target site preferences of transposable elements in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e30008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLinheiro%26rft.auinit1%253DR.%2BS.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De30008%26rft.epage%253De30008%26rft.atitle%253DWhole%2Bgenome%2Bresequencing%2Breveals%2Bnatural%2Btarget%2Bsite%2Bpreferences%2Bof%2Btransposable%2Belements%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0030008%26rft_id%253Dinfo%253Apmid%252F22347367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0030008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22347367&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.43" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.44" data-doi="10.1101/gr.129460.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mularoni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gangadharan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheelan</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Retrotransposon Ty1 integration targets specifically positioned asymmetric nucleosomal DNA segments in tRNA hotspots.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129460.111%26rft_id%253Dinfo%253Apmid%252F22219510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzQvNjkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.45" data-doi="10.1186/1471-2105-15-71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakagome</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solovieva</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasue</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirochika</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transposon Insertion Finder (TIF): a novel program for detection of de novo transpositions of transposable elements.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-71%26rft_id%253Dinfo%253Apmid%252F24629057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-71&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24629057&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.46" data-doi="10.1186/gb-2012-13-6-r45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nellaker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomic landscape shaped by selection on transposable elements across 18 mouse strains.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DNellaker%26rft.auinit1%253DC.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253DR45%26rft.epage%253DR45%26rft.atitle%253DThe%2Bgenomic%2Blandscape%2Bshaped%2Bby%2Bselection%2Bon%2Btransposable%2Belements%2Bacross%2B18%2Bmouse%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-6-r45%26rft_id%253Dinfo%253Apmid%252F22703977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-6-r45&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22703977&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.47" data-doi="10.1038/msb.2011.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stolovitzky</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The self-assessment trap: can we all be better than average?</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DNorel%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D537%26rft.epage%253D537%26rft.atitle%253DThe%2Bself-assessment%2Btrap%253A%2Bcan%2Bwe%2Ball%2Bbe%2Bbetter%2Bthan%2Baverage%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.70%26rft_id%253Dinfo%253Apmid%252F21988833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMS81MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Platzer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nizhynska</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">TE-Locate: a tool to locate and group transposable element occurrences using paired-end next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Biology (Basel)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%2B%2528Basel%2529%26rft.volume%253D1%26rft.spage%253D395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.49" data-doi="10.1101/gr.131219.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daily</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayhew</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Retrotransposon profiling of RNA polymerase III initiation sites.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.131219.111%26rft_id%253Dinfo%253Apmid%252F22287102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzQvNjgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quadrana</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silveira</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayhew</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LeBlanc</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martienssen</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Arabidopsis thaliana mobilome and its impact at the species level.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e15716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D5%26rft.spage%253De15716%26rft_id%253Dinfo%253Apmid%252F27258693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=27258693&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.51" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.52" data-doi="10.1101/gr.102970.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokolova</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leibowitz</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide mapping and assembly of structural variant breakpoints in the mouse genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.102970.109%26rft_id%253Dinfo%253Apmid%252F20308636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzUvNjIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.53" data-doi="10.1093/nar/gkv1193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rahman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chirn</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanodia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sytnikova</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brembs</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Unique transposon landscapes are pervasive across Drosophila melanogaster genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">10655</span>–<span class="cit-lpage">10672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1193%26rft_id%253Dinfo%253Apmid%252F26578579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26578579&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-7.8.2763.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <em>R: A Language and Environment for Statistical Computing</em>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rinckel</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garfinkel</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Influences of histone stoichiometry on the target site preference of retrotransposons Ty1 and Ty2 in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRinckel%26rft.auinit1%253DL.%2BA.%26rft.volume%253D142%26rft.issue%253D3%26rft.spage%253D761%26rft.epage%253D776%26rft.atitle%253DInfluences%2Bof%2BHistone%2BStoichiometry%2Bon%2Bthe%2BTarget%2BSite%2BPreference%2Bof%2BRetrotransposons%2BTy1%2Band%2BTy2%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F8849886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQyLzMvNzYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rishishwar</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mariño-Ramírez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">I. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Benchmarking computational tools for polymorphic transposable element detection.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-fpage">bbw072</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robb</span>  <span class="cit-name-given-names">S. M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valencia</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnette</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okumoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The use of RelocaTE and unassembled short reads to produce high-resolution snapshots of transposable element generated diversity in rice.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">949</span>–<span class="cit-lpage">957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sackton</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulathinal</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dopman</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomic inferences from sparse high-throughput sequencing of two populations of Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%2Band%2BEvolution%26rft.stitle%253DGenome%2BBiol%2BEvol%26rft.aulast%253DSackton%26rft.auinit1%253DT.%2BB.%26rft.volume%253D1%26rft.spage%253D449%26rft.epage%253D465%26rft.atitle%253DPopulation%2Bgenomic%2Binferences%2Bfrom%2Bsparse%2Bhigh-throughput%2Bsequencing%2Bof%2Btwo%2Bpopulations%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Apmid%252F20333214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20333214&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bridges</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magbanua</span>  <span class="cit-name-given-names">Z. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Computational approaches and tools used in identification of dispersed repetitive DNA sequences.</span> <abbr class="cit-jnl-abbrev">Trop. Plant Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrop.%2BPlant%2BBiol.%26rft.volume%253D1%26rft.spage%253D85%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.8.2763.60" data-doi="10.1186/1471-2105-6-31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">G. S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Automated generation of heuristics for biological sequence comparison.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSlater%26rft.auinit1%253DG.%2BS.%26rft.volume%253D6%26rft.spage%253D31%26rft.epage%253D31%26rft.atitle%253DAutomated%2Bgeneration%2Bof%2Bheuristics%2Bfor%2Bbiological%2Bsequence%2Bcomparison.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-31%26rft_id%253Dinfo%253Apmid%252F15713233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15713233&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-7.8.2763.61"><div class="cit-metadata unstructured">Smit, A. F. A., R. Hubley, and P. Green, 2013 RepeatMasker. Available at: http://www.repeatmasker.org.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.8.2763.62" data-doi="10.1101/gr.361602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Block</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulez</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chervitz</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Bioperl toolkit: perl modules for the life sciences.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1611</span>–<span class="cit-lpage">1618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DStajich%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D1611%26rft.epage%253D1618%26rft.atitle%253DThe%2BBioperl%2BToolkit%253A%2BPerl%2BModules%2Bfor%2Bthe%2BLife%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.361602%26rft_id%253Dinfo%253Apmid%252F12368254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMC8xNjExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-web" id="cit-7.8.2763.63"><div class="cit-metadata unstructured">Stempor, P., 2014 seqplots: an interactive tool for visualizing NGS signals and sequence motif densities along genomic features using average plots and heatmaps. R package version 1.6.0. Available at: <a href="http://github.com/przemol/seqplots">http://github.com/przemol/seqplots</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.64" data-doi="10.1101/gr.185538.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strope</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skelly</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozmin</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahadevan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The 100-genomes strains, an S. cerevisiae resource that illuminates its natural phenotypic and genotypic variation and emergence as an opportunistic pathogen.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.185538.114%26rft_id%253Dinfo%253Apmid%252F25840857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzUvNzYyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy84LzI3NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cahn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karpievitch</span>  <span class="cit-name-given-names">Y. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Population scale mapping of transposable element diversity reveals links to gene regulation and epigenomic variation.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e20777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D5%26rft.spage%253De20777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.66" data-doi="10.1038/nature15394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sudmant</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rausch</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abyzov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">An integrated map of structural variation in 2,504 human genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">526</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D526%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15394%26rft_id%253Dinfo%253Apmid%252F26432246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature15394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26432246&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.67" data-doi="10.1186/s13059-014-0488-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thung</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Ligt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vissers</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steehouwer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mobster: accurate detection of mobile element insertions in next generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D488%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0488-x%26rft_id%253Dinfo%253Apmid%252F25348035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0488-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25348035&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.68" data-doi="10.1186/1471-2164-15-795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konkel</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Tangram: a comprehensive toolbox for mobile element insertion detection.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D795%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-795%26rft_id%253Dinfo%253Apmid%252F25228379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25228379&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.69" data-doi="10.1093/nar/gku323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhuang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theurkauf</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TEMP: a computational method for analyzing transposable element polymorphism in populations.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">6826</span>–<span class="cit-lpage">6838</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku323%26rft_id%253Dinfo%253Apmid%252F24753423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24753423&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F8%2F2763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338769400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.70" data-doi="10.1073/pnas.92.3.920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The Saccharomyces Ty5 retrotransposon family is associated with origins of DNA replication at the telomeres and the silent mating locus HMR.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">920</span>–<span class="cit-lpage">924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DZou%26rft.auinit1%253DS.%26rft.volume%253D92%26rft.issue%253D3%26rft.spage%253D920%26rft.epage%253D924%26rft.atitle%253DThe%2BSaccharomyces%2BTy5%2Bretrotransposon%2Bfamily%2Bis%2Bassociated%2Bwith%2Borigins%2Bof%2BDNA%2Breplication%2Bat%2Bthe%2Btelomeres%2Band%2Bthe%2Bsilent%2Bmating%2Blocus%2BHMR.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.3.920%26rft_id%253Dinfo%253Apmid%252F7846079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5Mi8zLzkyMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOC8yNzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.8.2763.71" data-doi="10.1101/gad.10.5.634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ke</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The Saccharomyces retrotransposon Ty5 integrates preferentially into regions of silent chromatin at the telomeres and mating loci.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">634</span>–<span class="cit-lpage">645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DZou%26rft.auinit1%253DS%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D634%26rft.epage%253D645%26rft.atitle%253DThe%2BSaccharomyces%2Bretrotransposon%2BTy5%2Bintegrates%2Bpreferentially%2Binto%2Bregions%2Bof%2Bsilent%2Bchromatin%2Bat%2Bthe%2Btelomeres%2Band%2Bmating%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.5.634%26rft_id%253Dinfo%253Apmid%252F8598292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTAvNS82MzQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzgvMjc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/8/2763.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/8/2749" title="Sex Chromosome Evolution, Heterochiasmy, and Physiological QTL in the Salmonid Brook Charr Salvelinus fontinalis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/8/2779" title="Bayesian Networks Illustrate Genomic and Residual Trait Connections in Maize (Zea mays L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/8"><span>Volume 7 Issue 8, August 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/8.cover-source.jpg?itok=apH_YDuu" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/8/2763.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--gfhQ0RZToLU3-TmbWn35R9IYg-LqjNBiZ25CWn6hzM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/8/2763" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/8/2763" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/8/2763" class="active" data-icon-position="" data-hide-link-title="0">McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/485854" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-B-DkjDcigZxqqJED8aCl5e9Q6c7pUbQCQOhsQv0TleE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2763.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/8/2763" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-C0chF-imReuW0VPSN_jrs1RcE9eCDkGFE7JCB2zcWZE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/8/2763" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/8/2763.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485854" id="node485854--2" data-pisa="ggg;7/8/2763" data-pisa-master="ggg;g3.117.043893" data-apath="/ggg/7/8/2763.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2567-2634" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael G. Nelson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2659-0910" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Raquel S. Linheiro</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5462-9854" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Casey M. Bergman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2763-2778;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043893 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485854--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael G. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, M13 9PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BG.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BG.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2567-2634" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael G. Nelson</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Raquel S. Linheiro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, M13 9PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raquel%2BS.%2BLinheiro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Linheiro%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARaquel%2BS.%2BLinheiro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2659-0910" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Raquel S. Linheiro</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Casey M. Bergman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, M13 9PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Casey%2BM.%2BBergman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bergman%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACasey%2BM.%2BBergman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5462-9854" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Casey M. Bergman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cbergman@uga.edu" class="" data-icon-position="" data-hide-link-title="0">cbergman@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/8/2763" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/8/2763&amp;title=McClintock%3A%20An%20Integrated%20Pipeline%20for%20Detecting%20Transposable%20Element%20Insertions%20in%20Whole-Genome%20Shotgun%20Sequencing%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/8/2763&amp;title=McClintock%3A%20An%20Integrated%20Pipeline%20for%20Detecting%20Transposable%20Element%20Insertions%20in%20Whole-Genome%20Shotgun%20Sequencing%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/8/2763&amp;title=McClintock%3A%20An%20Integrated%20Pipeline%20for%20Detecting%20Transposable%20Element%20Insertions%20in%20Whole-Genome%20Shotgun%20Sequencing%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/8/2763&amp;text=McClintock%3A%20An%20Integrated%20Pipeline%20for%20Detecting%20Transposable%20Element%20Insertions%20in%20Whole-Genome%20Shotgun%20Sequencing%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/8/2763&amp;title=McClintock%3A%20An%20Integrated%20Pipeline%20for%20Detecting%20Transposable%20Element%20Insertions%20in%20Whole-Genome%20Shotgun%20Sequencing%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/8/2763&amp;t=McClintock%3A%20An%20Integrated%20Pipeline%20for%20Detecting%20Transposable%20Element%20Insertions%20in%20Whole-Genome%20Shotgun%20Sequencing%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/8/2763&amp;title=McClintock%3A%20An%20Integrated%20Pipeline%20for%20Detecting%20Transposable%20Element%20Insertions%20in%20Whole-Genome%20Shotgun%20Sequencing%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/8/2763&amp;title=McClintock%3A%20An%20Integrated%20Pipeline%20for%20Detecting%20Transposable%20Element%20Insertions%20in%20Whole-Genome%20Shotgun%20Sequencing%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/485854/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="485854" id="node485854--3" data-pisa="ggg;7/8/2763" data-pisa-master="ggg;g3.117.043893" data-apath="/ggg/7/8/2763.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2567-2634" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael G. Nelson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2659-0910" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Raquel S. Linheiro</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5462-9854" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Casey M. Bergman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2763-2778;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043893 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node485854--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael G. Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, M13 9PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BG.%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BG.%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2567-2634" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael G. Nelson</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Raquel S. Linheiro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, M13 9PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raquel%2BS.%2BLinheiro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Linheiro%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARaquel%2BS.%2BLinheiro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2659-0910" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Raquel S. Linheiro</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Casey M. Bergman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Life Sciences, University of Manchester, M13 9PL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Casey%2BM.%2BBergman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bergman%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACasey%2BM.%2BBergman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5462-9854" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Casey M. Bergman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cbergman@uga.edu" class="" data-icon-position="" data-hide-link-title="0">cbergman@uga.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2763&amp;rft.epage=2778&amp;rft.atitle=McClintock%3A%20An%20Integrated%20Pipeline%20for%20Detecting%20Transposable%20Element%20Insertions%20in%20Whole-Genome%20Shotgun%20Sequencing%20Data&amp;rft.volume=7&amp;rft.issue=8&amp;rft.date=2017-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Nelson%2C+Michael+G.&amp;rft.au=Linheiro%2C+Raquel+S.&amp;rft.au=Bergman%2C+Casey+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/485854/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/485854/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/485854/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/485854/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/485854/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/485854/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/485854/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/485854/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/485854/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/485854/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/485854/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/485854/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/8/2763" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions and Future Directions</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/8/2763.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/8/2763.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/8/2763.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/8/2763.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
