<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/12/2365" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/12/2365.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/12/2365.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/12/2365.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/12/2365/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/12/2365.atom" />
<meta name="HW.pisa" content="ggg;4/12/2365" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Functional Multi-Locus QTL Mapping of Temporal Trends in Scots Pine Wood Traits" />
<meta name="DC.Identifier" content="10.1534/g3.114.014068" />
<meta name="DC.Date" content="2014-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Li et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Quantitative trait loci (QTL) mapping of wood properties in conifer species has focused on single time point measurements or on trait means based on heterogeneous wood samples ( e.g. , increment cores), thus ignoring systematic within-tree trends. In this study, functional QTL mapping was performed for a set of important wood properties in increment cores from a 17-yr-old Scots pine ( Pinus sylvestris L.) full-sib family with the aim of detecting wood trait QTL for general intercepts (means) and for linear slopes by increasing cambial age. Two multi-locus functional QTL analysis approaches were proposed and their performances were compared on trait datasets comprising 2 to 9 time points, 91 to 455 individual tree measurements and genotype datasets of amplified length polymorphisms (AFLP), and single nucleotide polymorphism (SNP) markers. The first method was a multilevel LASSO analysis whereby trend parameter estimation and QTL mapping were conducted consecutively; the second method was our Bayesian linear mixed model whereby trends and underlying genetic effects were estimated simultaneously. We also compared several different hypothesis testing methods under either the LASSO or the Bayesian framework to perform QTL inference. In total, five and four significant QTL were observed for the intercepts and slopes, respectively, across wood traits such as earlywood percentage, wood density, radial fiberwidth, and spiral grain angle. Four of these QTL were represented by candidate gene SNPs, thus providing promising targets for future research in QTL mapping and molecular function. Bayesian and LASSO methods both detected similar sets of QTL given datasets that comprised large numbers of individuals." />
<meta name="DC.Contributor" content="Zitong Li" />
<meta name="DC.Contributor" content="Henrik R. Hallingbäck" />
<meta name="DC.Contributor" content="Sara Abrahamsson" />
<meta name="DC.Contributor" content="Anders Fries" />
<meta name="DC.Contributor" content="Bengt Andersson Gull" />
<meta name="DC.Contributor" content="Mikko J. Sillanpää" />
<meta name="DC.Contributor" content="M. Rosario García-Gil" />
<meta name="article:published_time" content="2014-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Functional Multi-Locus QTL Mapping of Temporal Trends in Scots Pine Wood Traits" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Quantitative trait loci (QTL) mapping of wood properties in conifer species has focused on single time point measurements or on trait means based on heterogeneous wood samples (&lt;i&gt;e.g.&lt;/i&gt;, increment cores), thus ignoring systematic within-tree trends. In this study, functional QTL mapping was performed for a set of important wood properties in increment cores from a 17-yr-old Scots pine (&lt;i&gt;Pinus sylvestris&lt;/i&gt; L.) full-sib family with the aim of detecting wood trait QTL for general intercepts (means) and for linear slopes by increasing cambial age. Two multi-locus functional QTL analysis approaches were proposed and their performances were compared on trait datasets comprising 2 to 9 time points, 91 to 455 individual tree measurements and genotype datasets of amplified length polymorphisms (AFLP), and single nucleotide polymorphism (SNP) markers. The first method was a multilevel LASSO analysis whereby trend parameter estimation and QTL mapping were conducted consecutively; the second method was our Bayesian linear mixed model whereby trends and underlying genetic effects were estimated simultaneously. We also compared several different hypothesis testing methods under either the LASSO or the Bayesian framework to perform QTL inference. In total, five and four significant QTL were observed for the intercepts and slopes, respectively, across wood traits such as earlywood percentage, wood density, radial fiberwidth, and spiral grain angle. Four of these QTL were represented by candidate gene SNPs, thus providing promising targets for future research in QTL mapping and molecular function. Bayesian and LASSO methods both detected similar sets of QTL given datasets that comprised large numbers of individuals.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/12/01" />
<meta name="citation_mjid" content="ggg;4/12/2365" />
<meta name="citation_id" content="4/12/2365" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/12/2365" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/12/2365.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/12/2365.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/12/2365.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.014068" />
<meta name="citation_pmid" content="25305041" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2365" />
<meta name="citation_lastpage" content="2379" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zitong Li" />
<meta name="citation_author_institution" content="Department of Mathematics and Statistics, University of Helsinki, Helsinki, Finland" />
<meta name="citation_author" content="Henrik R. Hallingbäck" />
<meta name="citation_author_institution" content="Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden" />
<meta name="citation_author" content="Sara Abrahamsson" />
<meta name="citation_author_institution" content="Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden" />
<meta name="citation_author_institution" content="Skogforsk, Sävar, Sweden" />
<meta name="citation_author" content="Anders Fries" />
<meta name="citation_author_institution" content="Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden" />
<meta name="citation_author" content="Bengt Andersson Gull" />
<meta name="citation_author_institution" content="Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden" />
<meta name="citation_author_institution" content="Skogforsk, Sävar, Sweden" />
<meta name="citation_author" content="Mikko J. Sillanpää" />
<meta name="citation_author_institution" content="Department of Mathematical Sciences and Department of Biology, University of Oulu, Oulu, Finland" />
<meta name="citation_author_institution" content="Biocenter Oulu, Oulu, Finland" />
<meta name="citation_author" content="M. Rosario García-Gil" />
<meta name="citation_author_institution" content="Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden" />
<meta name="citation_author_email" content="m.rosario.garcia@slu.se" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Beaulieu;citation_author=T. Doerksen;citation_author=B. Boyle;citation_author=S. Clément;citation_author=M. Deslauriers;citation_title=Association genetics of wood physical traits in the conifer white spruce and relationships with gene expression.;citation_volume=188;citation_year=2011;citation_pmid=21385726;citation_doi=10.1534/genetics.110.125781" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=L. Bouffier;citation_author=A. Raffin;citation_author=P. Rozenberg;citation_author=C. Meredieu;citation_author=A. Kremer;citation_title=What are the consequences of growth selection on wood density in the French maritime pine breeding programme?;citation_pages=11-25;citation_volume=5;citation_year=2009;citation_doi=10.1007/s11295-008-0165-x" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Environ.;citation_author=O. Brendel;citation_author=D. Pot;citation_author=C. Plomion;citation_author=P. Rozenberg;citation_author=J. M. Guehl;citation_title=Genetic parameters and QTL analysis of δ13C and ring width in maritime pine.;citation_pages=945-953;citation_volume=25;citation_year=2002;citation_doi=10.1046/j.1365-3040.2002.00872.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. R. Brown;citation_author=D. L. Bassoni;citation_author=G. P. Gill;citation_author=J. R. Fontana;citation_author=N. C. Wheeler;citation_title=Identification of Quantitative Trait Loci Influencing Wood Property Traits in Loblolly Pine (Pinus taeda L.). III. QTL Verification and Candidate Gene Mapping;citation_pages=1537-1546;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930758" />
<meta name="citation_reference" content="citation_journal_title=For. Sci.;citation_author=R. B. Burdon;citation_author=R. P. Kibblewhite;citation_author=J. C. F. Walker;citation_author=R. A. Megraw;citation_author=R. Evans;citation_title=Juvenile vs. mature wood: A new concept, orthogonal to corewood vs. outerwood, with special reference to Pinus radiata and P. taeda.;citation_pages=399-415;citation_volume=50;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Stat. Appl.;citation_author=P. Bühlmann;citation_author=M. Kalisch;citation_author=L. Meier;citation_title=High-dimensional statistics with a view towards applications in biology.;citation_pages=255-278;citation_volume=1;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=A. Chatterjee;citation_author=S. N. Lahiri;citation_title=Bootstrapping LASSO estimators.;citation_pages=608-625;citation_volume=106;citation_year=2011;citation_doi=10.1198/jasa.2011.tm10159" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. K. Dillon;citation_author=M. Nolan;citation_author=W. Li;citation_author=C. Bell;citation_author=H. X. Wu;citation_title=Allelic variation in cell wall candidate genes affecting solid wood properties in natural populations and land races of Pinus radiata.;citation_volume=185;citation_year=2010;citation_pmid=20498299;citation_doi=10.1534/genetics.110.116582" />
<meta name="citation_reference" content="citation_title=Timber: Its Nature and Behaviour;citation_author=J. M. Dinwoodie;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Focus;citation_author=J. Doyle;citation_author=J. Doyle;citation_title=Isolation of plant DNA from fresh tissue.;citation_pages=13-15;citation_volume=12;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=B. Efron;citation_author=R. Tibshirani;citation_author=J. D. Storey;citation_author=V. Tusher;citation_title=Empirical Bayes analysis of a microarray experiment.;citation_pages=1151-1160;citation_volume=96;citation_year=2001;citation_doi=10.1198/016214501753382129" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=L. C. Emebiri;citation_author=M. E. Devey;citation_author=A. C. Matheson;citation_author=M. U. Slee;citation_title=Linkage of RAPD markers to NESTUR, a stem growth index in radiata pine seedings.;citation_pages=119-124;citation_volume=95;citation_year=1997;citation_doi=10.1007/s001220050539" />
<meta name="citation_reference" content="citation_journal_title=Holzforschung;citation_author=R. Evans;citation_title=Rapid measurement of the transverse dimensions of tracheids in radial wood sections from Pinus radiata.;citation_pages=168-172;citation_volume=48;citation_year=1994;citation_doi=10.1515/hfsg.1994.48.2.168" />
<meta name="citation_reference" content="citation_title=Characterization of the Cellulosic Cell Wall;citation_title=Wood stiffness by X-ray diffractometry;citation_author=R. Evans;citation_pages=138-146;citation_year=2006" />
<meta name="citation_reference" content="citation_title=Bayesian Smoothing and Regression for Longitudinal, Spatial and Event History Data;citation_author=L. Fahrmeir;citation_author=T. Kneib;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Ann. For. Sci.;citation_author=C. Feng;citation_author=Y. Yuan;citation_author=W. Wei;citation_author=Z. Fan;citation_author=T. Zhang;citation_title=Climatic response of ring width and maximum latewood density of Larix sibirica in the Altay Mountains, reveals recent warming trends.;citation_pages=723-733;citation_volume=69;citation_year=2012;citation_doi=10.1007/s13595-012-0187-2" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=J. Friedman;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_title=Regularization Paths for Generalized Linear Models via Coordinate Descent.;citation_pages=1-22;citation_volume=33;citation_year=2010;citation_issue=1;citation_pmid=20808728;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_reference" content="citation_journal_title=Ann. For. Sci.;citation_author=A. Fries;citation_title=Genetic parameters, genetic gain and correlated responses in growth, fibre dimensions and wood density in a Scots pine breeding population.;citation_pages=783-794;citation_volume=69;citation_year=2012;citation_doi=10.1007/s13595-012-0202-7" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=N. A. Furlotte;citation_author=E. Eskin;citation_author=S. Eyheramendy;citation_title=Genome-wide association mapping with longitudinal data.;citation_pages=463-471;citation_volume=36;citation_year=2012;citation_issue=5;citation_pmid=22581622;citation_doi=10.1002/gepi.21640" />
<meta name="citation_reference" content="citation_journal_title=Don. Ann. For. Sci.;citation_author=W. J. Gapare;citation_author=H. X. Wu;citation_author=A. Abarquez;citation_title=Genetic control of the time of transition from juvenile to mature wood in Pinus radiata D.;citation_pages=871-878;citation_volume=63;citation_year=2006;citation_doi=10.1051/forest:2006070" />
<meta name="citation_reference" content="citation_journal_title=Don. Ann. For. Sci.;citation_author=W. J. Gapare;citation_author=B. S. Baltunis;citation_author=M. Ivkovíc;citation_author=H. X. Wu;citation_title=Genetic correlations among juvenile wood quality and growth traits and implications for selection strategy in Pinus radiata D.;citation_pages=606;citation_volume=66;citation_year=2009;citation_doi=10.1051/forest/2009044" />
<meta name="citation_reference" content="citation_journal_title=Ann. For. Sci.;citation_author=M. J. Gaspar;citation_author=J. L. Louzada;citation_author=A. Aguiar;citation_author=A. M. Almeida;citation_title=Genetic correlations between wood quality traits of Pinus pinaster Ait.;citation_pages=703;citation_volume=65;citation_year=2008;citation_doi=10.1051/forest:2008054" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=C. Gee;citation_author=J. L. Morrison;citation_author=D. C. Thomas;citation_author=W. J. Gauderman;citation_title=Segregation and linkage analysis for longitudinal measurements of a quantitative trait.;citation_pages=S21;citation_volume=4;citation_year=2003;citation_pmid=14975089;citation_doi=10.1186/1471-2156-4-S1-S21" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. C. González-Martínez;citation_author=N. C. Wheeler;citation_author=E. Ersoz;citation_author=C. D. Nelson;citation_author=D. B. Neale;citation_title=Association genetics in Pinus taeda L. I. Wood property traits.;citation_volume=175;citation_year=2007;citation_pmid=17110498;citation_doi=10.1534/genetics.106.061127" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Grattapaglia;citation_author=R. Sederoff;citation_title=Genetic linkage maps of Eucalyptus grandis and Eucalyptus urophylla using a pseudo-testcross: mapping strategy and RAPD markers.;citation_pages=1121-1137;citation_volume=137;citation_year=1994;citation_issue=4;citation_pmid=7982566" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Grattapaglia;citation_author=F. L. Bertolucci;citation_author=R. R. Sederoff;citation_title=Genetic mapping of QTLs controlling vegetative propagation in Eucalyptus grandis and E. urophylla using a pseudo-testcross strategy and RAPD markers.;citation_pages=933-947;citation_volume=90;citation_year=1995;citation_pmid=24173047" />
<meta name="citation_reference" content="citation_journal_title=Tappi.;citation_author=H. V. Green;citation_author=J. Worral;citation_title=Wood quality studies I. A scanning microphometer for automatically measuring and recording certain wood characteristics.;citation_pages=419-427;citation_volume=47;citation_year=1964" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Stat.;citation_author=Y. Guan;citation_author=M. Stephens;citation_title=Bayesian variable selection regression for genome-wide association studies, and other large-scale problems.;citation_pages=1780-1815;citation_volume=5;citation_year=2011;citation_doi=10.1214/11-AOAS455" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Silva Fenn.;citation_author=H. R. Hallingbäck;citation_author=G. Jansson;citation_author=B. Hannrup;citation_author=A. Fries;citation_title=Which annual rings to assess grain angles in breeding of Scots pine for improved shape stability of sawn timber?;citation_pages=275-288;citation_volume=44;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Scand. J. For. Res.;citation_author=B. Hannrup;citation_author=I. Ekberg;citation_author=A. Persson;citation_title=Genetic correlations among wood, growth capacity and stem traits in Pinus sylvestris.;citation_pages=161-170;citation_volume=15;citation_year=2000;citation_doi=10.1080/028275800750014966" />
<meta name="citation_reference" content="citation_journal_title=Silvae Genet.;citation_author=B. Hannrup;citation_author=H. Säll;citation_author=G. Jansson;citation_title=Genetic parameters for spiral grain in Scots pine and Norway spruce.;citation_pages=215-220;citation_volume=52;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Ann. For. Sci.;citation_author=Z. Hong;citation_author=A. Fries;citation_author=H. X. Wu;citation_title=High negative genetic correlations between growth traits and wood properties suggest incorporating multiple traits selection including economic weights for the future Scots pine breeding programs.;citation_pages=463-472;citation_volume=71;citation_year=2014;citation_doi=10.1007/s13595-014-0359-3" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=P. X. Hurtado;citation_author=S. K. Schnabel;citation_author=A. Zaban;citation_author=M. Veteläinen;citation_author=E. Virtanen;citation_title=Dynamics of senescence-related QTLs in potato.;citation_pages=289-302;citation_volume=183;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C.-H. Kao;citation_author=Z.-B. Zeng;citation_author=D. Teasdale;citation_title=Multiple Interval Mapping for Quantitative Trait Loci;citation_pages=1203-1216;citation_volume=152;citation_year=1999;citation_issue=3;citation_pmid=10388834" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Z. Kaya;citation_author=M. M. Sewell;citation_author=D. B. Neale;citation_title=Identification of quantitative trait loci influencing annual height- and diameter-increment growth in loblolly pine (Pinus taeda L.).;citation_pages=586-592;citation_volume=98;citation_year=1999;citation_doi=10.1007/s001220051108" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. A. Knott;citation_author=D. B. Neale;citation_author=M. M. Sewell;citation_author=C. S. Haley;citation_title=Multiple marker mapping of quantitative trait loci in an outbred pedigree of loblolly pine.;citation_pages=810-820;citation_volume=94;citation_year=1997;citation_doi=10.1007/s001220050482" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=S. T. Kujala;citation_author=O. Savolainen;citation_title=Sequence variation patterns along a latitudinal cline in Scots pine (Pinus sylvestris): signs of clinical adaptation?;citation_pages=1455-1467;citation_volume=8;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. Kumar;citation_author=R. J. Spelman;citation_author=D. J. Garrick;citation_author=T. E. Richardson;citation_author=M. Lausberg;citation_title=Multiple-marker mapping of wood density loci in an outbred pedigree of radiata pine.;citation_pages=926-933;citation_volume=100;citation_year=2000;citation_doi=10.1007/s001220051372" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. Kumar;citation_author=P. Saranpää;citation_author=J. R. Barnett;citation_author=M. J. Wilkinson;citation_title=Juvenile-mature wood transition in pine: correlation between wood properties and candidate gene expression profiles.;citation_pages=341-355;citation_volume=166;citation_year=2009;citation_doi=10.1007/s10681-008-9815-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=E. Lerceteau;citation_author=A. E. Szmidt;citation_title=AFLP mapping and detection of quantitative trait loci (QTLs) for economically important traits in Pinus sylvestris: a preliminary study.;citation_pages=451-458;citation_volume=6;citation_year=2000;citation_doi=10.1023/A:1026548716320" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=E. Lerceteau;citation_author=A. E. Szmidt;citation_author=B. Andersson;citation_title=Detection of quantitative trait loci in Pinus sylvestris L. across years.;citation_pages=117-122;citation_volume=121;citation_year=2001;citation_doi=10.1023/A:1012076825293" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Z. Li;citation_author=M. J. Sillanpää;citation_title=Overview of LASSO-related penalized regression methods for quantitative trait mapping and genomic selection.;citation_pages=419-435;citation_volume=125;citation_year=2012;citation_issue=3;citation_pmid=22622521;citation_doi=10.1007/s00122-012-1892-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Li;citation_author=M. J. Sillanpää;citation_title=A Bayesian nonparametric approach for mapping dynamic quantitative traits.;citation_volume=194;citation_year=2013;citation_pmid=23770698;citation_doi=10.1534/genetics.113.152736" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. Lin;citation_author=R. Wu;citation_title=A joint model for nonparametric functional mapping of longitudinal trajectory and time-to-event.;citation_pages=138-138;citation_volume=7;citation_year=2006;citation_pmid=16539724;citation_doi=10.1186/1471-2105-7-138" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=R. Lockhart;citation_author=J. Taylor;citation_author=R. J. Tibshirani;citation_author=R. Tibshirani;citation_title=A significance test for the LASSO.;citation_pages=413-468;citation_volume=42;citation_year=2014;citation_pmid=25574062;citation_doi=10.1214/13-AOS1175" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Ma;citation_author=G. Casella;citation_author=R. Wu;citation_title=Functional Mapping of Quantitative Trait Loci Underlying the Character Process: A Theoretical Framework;citation_pages=1751-1762;citation_volume=161;citation_year=2002;citation_issue=4;citation_pmid=12196415" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. Ma;citation_author=M. Lin;citation_author=R. C. Littell;citation_author=T. Yin;citation_author=R. Wu;citation_title=A likelihood approach for mapping growth trajectories using dominant markers in a phase-unknown full-sib family.;citation_pages=699-705;citation_volume=108;citation_year=2004;citation_issue=4;citation_pmid=14586505;citation_doi=10.1007/s00122-003-1484-9" />
<meta name="citation_reference" content="citation_journal_title=Silvae Genet.;citation_author=T. Markussen;citation_author=M. Fladung;citation_author=V. Achere;citation_author=J. M. Favre;citation_author=P. Faivre-Rampant;citation_title=Identification of QTLs controlling growth, chemical and physical wood property traits in Pinus pinaster (Ait.).;citation_pages=8-15;citation_volume=52;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=N. Meinshausen;citation_author=L. Meier;citation_author=P. Bühlmann;citation_title=P-Values for high-dimensional regression.;citation_pages=1671-1681;citation_volume=104;citation_year=2009;citation_doi=10.1198/jasa.2009.tm08647" />
<meta name="citation_reference" content="citation_journal_title=Stat. Soc. Series B.;citation_author=N. Meinshausen;citation_author=P. Bühlmann;citation_title=Stability Selection. J. R.;citation_pages=417-473;citation_volume=72;citation_year=2010;citation_doi=10.1111/j.1467-9868.2010.00740.x" />
<meta name="citation_reference" content="citation_journal_title=Der Papier-Fabrikant.;citation_author=E. Mork;citation_title=Die qualität des Fichtenholzes unter befonderer Rückfichtnahme auf Schleif—und Papierholz.;citation_pages=741-747;citation_volume=26;citation_year=1928" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. E. Palmé;citation_author=M. Wright;citation_author=O. Savolainen;citation_title=Patterns of divergence among Conifer ESTs and polymorphism in Pinus sylvestris identify putative selective sweeps.;citation_volume=25;citation_year=2008;citation_pmid=18775901;citation_doi=10.1093/molbev/msn194" />
<meta name="citation_reference" content="citation_journal_abbrev=Biom J;citation_author=H. P. Piepho;citation_author=J. Moehring;citation_author=T. Schulz-Streeck;citation_author=J. O. Ogutu;citation_title=A stage-wise approach for the analysis of multi-environment trials.;citation_pages=844-860;citation_volume=54;citation_year=2012;citation_issue=6;citation_pmid=23007738;citation_doi=10.1002/bimj.201100219" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Plomion;citation_author=C.-E. Durel;citation_author=D. O’Malley;citation_title=Genetic dissection of height in maritime pine seedlings raised under accelerated growth condition.;citation_pages=849-858;citation_volume=93;citation_year=1996;citation_pmid=24162417;citation_doi=10.1007/BF00224085" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=D. Pot;citation_author=J. C. Rodrigues;citation_author=P. Rozenberg;citation_author=G. Chantre;citation_author=J. Tibbits;citation_title=QTLs and candidate genes for wood properties in maritime pine (Pinus pinaster Ait.).;citation_pages=10-24;citation_volume=2;citation_year=2006;citation_doi=10.1007/s11295-005-0026-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Pyhäjärvi;citation_author=M. R. García-Gil;citation_author=T. Knürr;citation_author=M. Mikkonen;citation_author=W. Wachowiak;citation_title=Demographic History Has Influenced Nucleotide Diversity in European Pinus sylvestris Populations;citation_pages=1713-1724;citation_volume=177;citation_year=2007;citation_issue=3;citation_pmid=18039881;citation_doi=10.1534/genetics.107.077099" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=T. Pyhäjärvi;citation_author=S. T. Kujala;citation_author=O. Savolainen;citation_title=Revisiting protein heterozygosity in plants-nucleotide diversity in allozyme coding genes of conifer Pinus sylvestris.;citation_pages=385-397;citation_volume=7;citation_year=2011;citation_doi=10.1007/s11295-010-0340-8" />
<meta name="citation_reference" content="citation_title=Review of the Swedish Tree Breeding Programme;citation_author=O. Rosvall;citation_year=2011" />
<meta name="citation_reference" content="citation_title=Handbook of Wood Chemistry and Wood Composites;citation_author=R. M. Rowell;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Wood Sci. Technol.;citation_author=K. Schulgasser;citation_author=A. Witztum;citation_title=The mechanism of spiral grain formation in trees.;citation_pages=133-156;citation_volume=41;citation_year=2007;citation_doi=10.1007/s00226-006-0100-y" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=J. G. Scott;citation_author=J. O. Berger;citation_title=Bayes and empirical-Bayes multiplicity adjustment in the variable-selection problem.;citation_pages=2587-2619;citation_volume=38;citation_year=2010;citation_doi=10.1214/10-AOS792" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=V. Segura;citation_author=B. J. Vilhjálmsson;citation_author=A. Platt;citation_author=A. Korte;citation_author=Ü. Seren;citation_title=An efficient multi-locus mixed-model approach for genome-wide association studies in structured populations.;citation_pages=825-830;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22706313;citation_doi=10.1038/ng.2314" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. M. Sewell;citation_author=D. L. Bassoni;citation_author=R. A. Megraw;citation_author=N. C. Wheeler;citation_author=D. B. Neale;citation_title=Identification of QTLs influencing wood property traits in loblolly pine (Pinus taeda L.). I. Physical wood properties.;citation_pages=1273-1281;citation_volume=101;citation_year=2000;citation_doi=10.1007/s001220051607" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. M. Sewell;citation_author=M. F. Davis;citation_author=G. A. Tuskan;citation_author=N. C. Wheeler;citation_author=C. C. Elam;citation_title=Identification of QTLs influencing wood property traits in loblolly pine ( Pinus taeda L.). II. Chemical wood properties.;citation_pages=214-222;citation_volume=104;citation_year=2002;citation_issue=2-3;citation_pmid=12582689;citation_doi=10.1007/s001220100697" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=M. J. Sillanpää;citation_author=P. Pikkuhookana;citation_author=S. Abrahamsson;citation_author=T. Knürr;citation_author=A. Fries;citation_title=Simultaneous estimation of multiple quantitative trait loci and growth curve parameters through hierarchical Bayesian modeling.;citation_pages=134-146;citation_volume=108;citation_year=2012;citation_issue=2;citation_pmid=21792229;citation_doi=10.1038/hdy.2011.56" />
<meta name="citation_reference" content="citation_journal_title=Stat. Med.;citation_author=K. Sikorska;citation_author=F. Rivadeneira;citation_author=P. J. F. Groenen;citation_author=A. Hofman;citation_author=A. G. Uitterlinden;citation_title=Fast linear mixed model computations for genome-wide association studies with longitudinal data.;citation_pages=165-180;citation_volume=32;citation_year=2012;citation_pmid=22911890" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=J. Slate;citation_title=From Beavis to beak color: a simulated study to examine how much QTL mapping can reveal about the genetic architecture of quantitative traits.;citation_pages=1251-1262;citation_volume=67;citation_year=2013;citation_pmid=23617906" />
<meta name="citation_reference" content="citation_title=Statistical Methods;citation_author=G. W. Snedecor;citation_author=W. G. Cochran;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=R. Tibshirani;citation_title=Regression shrinkage and selection via the lasso.;citation_pages=267-288;citation_volume=58;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=N. K. Ukrainetz;citation_author=K. Ritland;citation_author=S. D. Mansfield;citation_title=Identification of quantitative trait loci for wood quality and growth across eight full-sib coastal Douglas-fir families.;citation_pages=159-170;citation_volume=4;citation_year=2008;citation_doi=10.1007/s11295-007-0097-x" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Signal Behav;citation_author=C. van der Schoot;citation_author=L. K. Paul;citation_author=S. B. Paul;citation_author=P. L. H. Rinne;citation_title=Plant lipid bodies and cell-cell signaling: a new role for an old organelle?;citation_pages=1732-1738;citation_volume=6;citation_year=2011;citation_issue=11;citation_pmid=22057325;citation_doi=10.4161/psb.6.11.17639" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_author=M. Ventrucci;citation_author=E. M. Scott;citation_author=D. Cocchi;citation_title=Multiple testing on standardized mortality ratios: a Bayesian hierarchical model for FDR estimation.;citation_volume=12;citation_year=2011;citation_pmid=20577014;citation_doi=10.1093/biostatistics/kxq040" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. Vos;citation_author=R. Hogers;citation_author=M. Bleeker;citation_author=M. Reijans;citation_author=T. van de Lee;citation_title=AFLP: a new technique for DNA fingerprinting;citation_pages=4407-4414;citation_volume=23;citation_year=1995;citation_issue=21;citation_pmid=7501463;citation_doi=10.1093/nar/23.21.4407" />
<meta name="citation_reference" content="citation_journal_title=Scand. J. For. Res.;citation_author=M. Warensjö;citation_author=G. Rune;citation_title=Effects of compression wood and grain angle on deformations of studs from 22-year old Scots pine trees.;citation_pages=48-54;citation_volume=19;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=C. Wasternack;citation_author=B. Hause;citation_title=Jasmonates: biosynthesis, perception, signal transduction and action in plant stress response, growth and development. An update to the 2007 review.;citation_volume=111;citation_year=2013;citation_pmid=23558912;citation_doi=10.1093/aob/mct067" />
<meta name="citation_reference" content="citation_title=Modeling Longitudinal Data;citation_author=R. E. Weiss;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=R. L. Wu;citation_author=C. X. Ma;citation_author=M. C. K. Yang;citation_author=M. Chang;citation_author=U. Santra;citation_title=Quantitative trait loci for growth trajectories in Populus.;citation_pages=51-64;citation_volume=81;citation_year=2003;citation_issue=1;citation_pmid=12693683;citation_doi=10.1017/S0016672302005980" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Xu;citation_title=Estimating Polygenic Effects Using Markers of the Entire Genome;citation_pages=789-801;citation_volume=163;citation_year=2003;citation_issue=2;citation_pmid=12618414" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=S. Xu;citation_title=An empirical Bayes method for estimating epistatic effects of quantitative trait loci.;citation_pages=513-521;citation_volume=63;citation_year=2007;citation_issue=2;citation_pmid=17688503;citation_doi=10.1111/j.1541-0420.2006.00711.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Yang;citation_author=S. Xu;citation_title=Bayesian shrinkage analysis of quantitative trait loci for dynamic traits.;citation_volume=176;citation_year=2007;citation_pmid=17435239;citation_doi=10.1534/genetics.106.064279" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z.-B. Zeng;citation_title=Precision mapping of quantitative trait loci.;citation_pages=1457-1468;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=8013918" />
<meta name="citation_reference" content="citation_title=Juvenile Wood in Forest Trees;citation_author=B. J. Zobel;citation_author=J. R. Sprague;citation_year=1998" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Functional Multi-Locus QTL Mapping of Temporal Trends in Scots Pine Wood Traits" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
<meta name="twitter:description" content="Quantitative trait loci (QTL) mapping of wood properties in conifer species has focused on single time point measurements or on trait means based on heterogeneous wood samples ( e.g. , increment cores), thus ignoring systematic within-tree trends. In this study, functional QTL mapping was performed for a set of important wood properties in increment cores from a 17-yr-old Scots pine ( Pinus sylvestris L.) full-sib family with the aim of detecting wood trait QTL for general intercepts (means) and for linear slopes by increasing cambial age. Two multi-locus functional QTL analysis approaches were proposed and their performances were compared on trait datasets comprising 2 to 9 time points, 91 to 455 individual tree measurements and genotype datasets of amplified length polymorphisms (AFLP), and single nucleotide polymorphism (SNP) markers. The first method was a multilevel LASSO analysis whereby trend parameter estimation and QTL mapping were conducted consecutively; the second method was our Bayesian linear mixed model whereby trends and underlying genetic effects were estimated simultaneously. We also compared several different hypothesis testing methods under either the LASSO or the Bayesian framework to perform QTL inference. In total, five and four significant QTL were observed for the intercepts and slopes, respectively, across wood traits such as earlywood percentage, wood density, radial fiberwidth, and spiral grain angle. Four of these QTL were represented by candidate gene SNPs, thus providing promising targets for future research in QTL mapping and molecular function. Bayesian and LASSO methods both detected similar sets of QTL given datasets that comprised large numbers of individuals." />
<meta name="og-title" property="og:title" content="Functional Multi-Locus QTL Mapping of Temporal Trends in Scots Pine Wood Traits" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/12/2365" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Quantitative trait loci (QTL) mapping of wood properties in conifer species has focused on single time point measurements or on trait means based on heterogeneous wood samples ( e.g. , increment cores), thus ignoring systematic within-tree trends. In this study, functional QTL mapping was performed for a set of important wood properties in increment cores from a 17-yr-old Scots pine ( Pinus sylvestris L.) full-sib family with the aim of detecting wood trait QTL for general intercepts (means) and for linear slopes by increasing cambial age. Two multi-locus functional QTL analysis approaches were proposed and their performances were compared on trait datasets comprising 2 to 9 time points, 91 to 455 individual tree measurements and genotype datasets of amplified length polymorphisms (AFLP), and single nucleotide polymorphism (SNP) markers. The first method was a multilevel LASSO analysis whereby trend parameter estimation and QTL mapping were conducted consecutively; the second method was our Bayesian linear mixed model whereby trends and underlying genetic effects were estimated simultaneously. We also compared several different hypothesis testing methods under either the LASSO or the Bayesian framework to perform QTL inference. In total, five and four significant QTL were observed for the intercepts and slopes, respectively, across wood traits such as earlywood percentage, wood density, radial fiberwidth, and spiral grain angle. Four of these QTL were represented by candidate gene SNPs, thus providing promising targets for future research in QTL mapping and molecular function. Bayesian and LASSO methods both detected similar sets of QTL given datasets that comprised large numbers of individuals." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
    <title>Functional Multi-Locus QTL Mapping of Temporal Trends in Scots Pine Wood Traits | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"i6HSqjeFUO9gLaKADVLSrF7g0izOBdzaL2HQCDIeiu0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459746","apath":"\/ggg\/4\/12\/2365.atom","pisa":"ggg;4\/12\/2365","ac":{"\/ggg\/4\/12\/2365.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/12\/2365.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/12\/2365"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/12\/2365"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/12\/2365":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lcH4ydn4asQx48xG9GYDy\\\\\\\/DpLWL7tPgqEFQR0LUmz+znoE12M4FtWfnycP64nE\\\\\\\/aW\\\\\\\/WBhvwL\\\\\\\/tCmbWfAkag\\\\\\\/AZx42laBwwolJLp5N8RkCDnpFDaaUILAiK1hy\\\\\\\/9RcR\\\\\\\/ksXLbvkD2ZnhV5wudAJ3eIBuUnzoU2TEDtyVpvJbcH3GSY+H\\\\\\\/1063Qd09nBM1ijPl2OVrqMsQGf6WT2hFIYaRg6zy5UPom8d8VPzVimDjgVhUxldcKwKAYMZMhOeIAXalHFYXmZuuzyOvIsZDWhDQSuBO0rzwmAfiWIyxN9wNfvFSH1TayS\\\\\\\/xNdphnzJccekTfbnxDa4T6\\\\\\\/mFztg6K1FfHB+avFI6YzlZgnfY1buY9PFS2Es7Yg58+igNw7AHLvWUtGAIUblttg+4TtMPIezA3dyE61FX0\\\\\\\/\\\\\\\/g4ixZRwhJYYtnNkCZoabYFV910xbwiyb+79BIR0juQNioX68kE3QOt9suiCP4FBaUeYIjanbW1zYO7TiQQkroeNv60FeUifuOvAA2CYEIwbMTwTObqyLqT3wFbYS9fcu38Tk4pfBm32aPEQy4fGtVm2MyDno0hLGdaTl6wsUnWrvVLAf1JVslhruPeyLChaPYJywtmPLHH4+onwIVEpTK6n05rzRHDVAGOZ8RXpQRZR6NbI5cDzSpnsJZk\\\\\\\/myhJRsspx2Tt+B\\\\\\\/MB08BfkUv\\\\\\\/r1a12hjaXPDGi+Qw9+j\\\\\\\/E1DNl0rjBbaHqlrUznVo7FhfDVPx5RZ+sXUQSZQVgZ6IS2oRgMKrYzw4ieslVl3p\\\\\\\/oYwudE\\\\\\\/WUJki\\\\\\\/hXEQHhGLunXJvmKK3dKRtgVbvYgrksdObcC+eU9m4UE+jgO9QEK72vQhjLsu7lrH\\\\\\\/wO7NlxQKpLqG8r2azK2pKeh4Q4G1PtIw9iXcoi5KUhEHqQebOVztC5D7Y405HFD8gmcojw3brbAK7OdA3QBUOUfvHlb89fgsRuFDhIXknvFXKaOOvCMQIjXhXpO+MB7np5\\\\\\\/4udGjHv8PCVDuhv1Rprn7cgATJXYmABBV3\\\\\\\/JEC8S+thezyHa7fBKV91pLplnVFC92lPuVn+6zMsVsGy6HDuC7kSOE2qjud0NrKrRp+hJSMWR7f\\\\\\\/JT2unVPv8mcaFZuIwCCDczjwNwHw5tyWn98u6BQwdjpkPVpq2sEpY9a5omgjc8\\\\\\\/sQYUJsEPhaaUqqrOZQDPB8j7NwN6SpgKef5M88F7a6Rl1HBCVExtK9mbroR1fq0bO5qwbb1NQuzSx06Vl1GHDv+hiPOHtwpHF8ihbONPYXN3ZEzKPUVw9khZs\\\u0022,\\\u0022iv\\\u0022:\\\u0022wswXIqfGN88yT0S3x2JVhw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222c2ab65a4a0e1ab7caf1ea51e304497081da85566dd5a9b2a6302b431f39d3aa\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002292PBEUhwaGtbkE9c3qoHSN+Hl66cF+IFNO76SlZiq\\\\\\\/YbpPjeSmWWh5hXOxkeP2Qoevc\\\\\\\/70seEsGTGxdyUvv1sYU+zV7pPSiFX4k+fsF8N0mC81BkBMteNmkFGoHwj7M8hBdUYxHa4gie1oMrHF29y+zxH4diojO8K1aOKNee8TcqM1gK+IvJzV\\\\\\\/goBzF4R7hh1b4CkCzOFh9c\\\\\\\/8QXr0AcCI3zizj37oyyphOfac4D50xzoYcsGWBw+j++V+I1OwXfN33a6uwVhyUxfo9S5faKzvjxdpvM4kO\\\\\\\/ToKYEOgCe3XDWTH5w6IyWqF5lpTvO1qUi0xi0s2MErWPWKLBNCWWFhL9zcvQNIFU3R+uMKRFxUn1egL3NA189A46TsR8Ra5hj3arnLhLbogGmPf6U2M9HdwMNSqnTU0kbppzd+PMVzRLA6xUNfg5mdP0U5rwskiucDcWnKTLpyRs7z8WWIsDEV7LCBiftYpWVXpeEE3cH\\\\\\\/PVwVByX892taEt5wu55UrN\\\\\\\/3SaSkOhqmcaV2ZFatzXa\\\\\\\/+0TVjQaU31Ys\\\\\\\/TDx5ex3DC05877s7bXVhbI0c42qSdF75XTiIXArXpsXbClI4ddbByjALG9IdYGmhYx9VoGIOx+BhtMKnPVYKJV0kYzvanZWZk0hKVo\\\\\\\/+09zz66jWzfgyn+B3hUBZdDdVV0t4VhY7SknKINTlGNjhg5aTzLHBERDqdjCXedDTpvK8GHkT4FZrA9j3BbJYt16IGE79XOh3Jo0UOJQAtJvjFxBVKWS+gl1yrnPulFgDWat3AjtBT5jSq7yQWqXXL\\\\\\\/lWIzqkxUR\\\\\\\/qLCs0ohAQgYn+16PRNFIjDAL6\\\\\\\/chUDQiaE5m5qYEB5ylK\\\\\\\/2cPUnzXy7vZXzZ2diNyw3wYXu9rhXjPXEotOyIciSAmAd09LEfDG9qIFl87xj7xzlH+dtVdDZJBhBlTR5z5V+2zQbZOmPXRyRl9JcWr2+ojrTIRIw6OAG7uYbdaSezqdWw3tOpYF7HmqitmV34SNiu2xIPHubDZ\\\\\\\/mZOzXdna0k6fRJeJtyrIMIyYa6JzgDWM+Iq2enCByd\\\\\\\/thRbWLS1tYmsNdI0JFbfoMxrNFSwRq0dK8Cct6plc7mr992VOJj4\\\\\\\/Tvfc4921feLiklU2HVNU8wDiSVYiRX\\\\\\\/g+Hcy7QsYooWHQatfJ\\\\\\\/CTT3ckbQx7ulscMp5iosw4dt5Hbp4RugzY5LcgmKunGuCF8uVj\\\\\\\/Vi1FtAPbkDAAsA7d1ksf5gLuIQt7gATjj9Xp2iPXfxl+OiGk6cQbkei5RaOE\\\\\\\/7mWjoYuU1O4QA2l7u1A8E7MLkdmDGyOb7lFaIirFjZT8msJ6vENCLpRoj+SG4LhUR+o9R32S3X\\\\\\\/3auJgd5LGy7goU1FUGD9UedzfMhxcthw11z5wq5CuKKqD1B2nKRUmRvUVUOPwVrFcjm0XKaLNDbnrR\\\\\\\/MQpBZsJyvyOBoD8WrW3QKMDa7o58A8XL0Nxq0LJUDAJjPdz+aDbqGdEdIb\\\\\\\/zu4GNEMr\\\\\\\/sWHNz1LVpP3XQIwP6Nuwj7t\\\\\\\/0MfHERemukyJ3P9B1co+hfijIW7rKNLDMBA5M0O58jxveByBYgLHjFjfJ5yoJboI96UcjNfiMiVFcljGmI8VZuM25oNT1iEDDxumZ3CCyqZS5WO8TPkQ7hYrDg3OInuver2mDfyWdiQyYlFcxuRBeiAG4FN9ctAtYsrr3rNyOZ4NizA\\\\\\\/7Tl3KZWxPwnK5tqw+9bfKR\\\\\\\/GP4uIgD1x+6TR0l+ys99vujBpoxJ5qgz7PyoZfelBaQSt5aX9RbZ5c\\\\\\\/KAKm7UsOOEDr4LiP\\\\\\\/9htDq+OB6\\\\\\\/aWU6pj\\\\\\\/utJeGfzvLui00F87A6vyFq1DBXeBR+XAJ9YcTzvCOOt2cdn4QYJ+hEVdVd3\\\\\\\/TSphII\\\\\\\/vm3SQj6lai7ECHUQUiHTixgzW22\\\\\\\/l2GpwHVarEy9DON5FjGpo3SK+YMz7B9zT6f2KzAnmz54xVg1jejkxGGxtoPFSLoWrsG8Vp470zl5e2LFw8ABuEoBoscJbx3EmR5rj7spjl3GhfdwiUhut\\\\\\\/lzRsOQpaMAfYuMc+kwH\\\\\\\/IRaUEKZjzlBBvgbZarGP8QnpYtYMVBamMzXprQ7MVGof8EKoPOuW+YmAO3rqxG1ZHtihvNnmJ9F02dStKSaA\\\\\\\/vq1oCMmxlZyBS7gRqP+AzLnKhddKxs7iWmRlVooQtUtsspAuaHbAgHsGRzBb\\\\\\\/M0amSTFoiL9GCaI3FTQnHodTjwzeu+XPOJjG5AXly6oysOpMVJ11XW8cMwO0WNPB5GRZZWTQ+W90dQ7z\\\\\\\/6A\\\\\\\/YeuvNzUI5asuXqq9wPh8aAjHzwsfr5NV\\\\\\\/0l1q5kQN2Hj5Hz+H80+Sgv1aTFUTUtauMbM9bhE\\\\\\\/35LwFL6KV7UEeS9uI4hqb6gXDLxpsgp1YdPuVXrYNdPZs7vzM6lSK\\\\\\\/rZ2cZaXPhHcgDq2iYqIgtRSXoFjCuoCvFviLMY4O2bd84s6VU9EUD7FfszdVlhof2bFBua\\\\\\\/SboOuy8lOfrV2VE41E\\\\\\\/OIJKDCinHSMxfcxp4evuWCkhKDaw6ZPbofhz8TMZUDN67+uAYC5AtAsPoHUfa6kSEFvo6dBjwyZjaP0N2VhLJENdc+fcKGWtV3sch6YuVrFcbZr82IQn9mMVh4c8jjY9QVR+CGSws9uLrZ4ZfZNCG4OleuHsahW6zyzxT5g4HstTnPP7XJRyeIP4f4kgR4lQeaBuTyy1KhLq3EbmJmr8UfVBfEIFBBgEIQkuRYz4Jw4BKzMunsGcM0xHnBbwQqrxvKibFaKj\\\\\\\/EpIfTrRwTEI\\\\\\\/PrlYiCHqFpMAjU1geWeAHTaUilVm6b6IwzxlbZ6Hwiue2ErBWP9lOmHXJCV6ctPTCQ1YZGMYqNQBn7gzjR5b1wgT3exMRjp9u9cZBj47zmWswzTGo4u0sXcaWKCVdS1\\\\\\\/pteT4z9Z2NeLG5O4swaFqS7M3wn3\\\\\\\/\\\\\\\/Mi\\\\\\\/iO6yRk4Q3th6WXxDNfxZh13cavbOo3hZpUvJoXkFZroxbe7S68WTjcnsDbLi\\\\\\\/cplTPOicf01halSwFrdelVGrrFHf4Icnrnu6Pb5D+EKve+Icx6CQ5nrOWNB3sLsVFzQDB\\\\\\\/iaQENLAHgJ8DSm4Srz58CWM+ea0mOMl4JrlrAjdVfbaVu1gMqOzPALbyH\\\\\\\/Vb\\\\\\\/1E1auOtsTwnj\\\\\\\/Ntl5C+ChWkzRWK+BC+6MxUVYdU0b6QPKhY1nJfaohre5Km9Cr4IO+ze8HuOHFjZIFTwVdSdDjXOc+NNEEv\\\\\\\/LM5yLszfKR4PNrivpRmQFh7jFJ+EmzLmkSwNxNjnYQekBE1NgPzvbEEn9euVj4BL4Ph1bNIPQRqnd+128ES\\\\\\\/gpSoTdZrT7oRHwsa62KPx20bRmmVTUeGj\\\\\\\/v6uUf42HF6omHfbkxcrqdGQXYXvLw\\\\\\\/In1wEOMyOoNK7hBAnS2mEJo8\\\\\\\/r9nzR8k0yD0njWgKVxqzdoaxXaPS9s4K\\\\\\\/vJG\\\\\\\/nBwu6dPqwabsdk5IyJi2\\\\\\\/uJZ+k7muG4P4A7bJ\\\\\\\/4SOQrayqiYiZcvP3TYYfNMrp0Mj6cq366gqqOif8Pw4cTsmnFF5ufQbEO7ewz2rzROWDS9VZFBTFD3aaSHw9IOVDfJUMHE0KGc8JYyjuJc38Yue3u4WBI9tFWjlmKBaJOMRgU7rQDdobcdVzGbxy6Td0YH7zTgnF6GcnwaGzQbqOdUmRHtjTA2xnsx\\\\\\\/R2atjTWgNQj6VQ\\\\\\\/Vbb+fEq4zgs88Sg30PhtkPGDfvrknRiTxfK8gOyCnwgUFlXRqi485ZGGlJpLzmeOY7bpnUJ1Iyp\\\\\\\/tCMdzfjOgDKQYfqP9q2Umqi5H\\\\\\\/xCS85Y\\\\\\\/gzuBeHQB6V+poBTchsP0jHS0hsFiGm760pom68aDLz8mrRV64S5qHB6Q6TzgGRJjOtnVBPYAfZxHwqm5guwDit9h4K+KTKwaTemKxssp+47tolfTcxs5qAxqcj0a5Uygg2kXHsrmxs1qX2edo+K9oxZBSWJ3wVHc64Rixr13qoKvjlhop8e4Ur+SwWKb+aMiLyp6EFyTv2IQtIXQyiYuSOI0kawQ86rsbYeR+tQVdPjqMB0kjCvFtDGL7K7AEuOZ3Lm1ak4VsR3+7hYKhXms9X30sYLKqRY0dt2eIqAFqDSNC770mPJe6MSM2iOae30mucUHUHLJhfRtRXmPKjAj6APgIxt6XWQYgFguKZpYf7AZcY07V5VM+Ft4JnHJg\\\\\\\/DNJ1TRFfFkU+9mxdamFBbmfma0fWZo+VPRFP8PDa0oRWbwtmg5VKiFvmXQbuOeL86J4mYDAtYPeAGjyGE0L8slA+MIWBFGQMb2zRBHI3+paq5xYValSVXqF6pFh099u6\\\\\\\/Vn6UAcIcHF+sV18PJiQnP8SktHf1D9x89pQxN2O8OZ5sqsGnE11ViaojGpWtWHm+wD9PeETeNQhBtf6WHmIWCAIShnirA7EhiOdwbSZ08IqHELLKsyP+uDNF1YnQ0U1FlCLMfptvNrauGdPhr5XcZajHIb4IGkMjwRTqdeY1\\\\\\\/xne\\\\\\\/rCOBrr8VsVHpasXQYb2t3YXEgoU1lcd75ZYxOxbrUkaA0HMr0CzF\\\\\\\/\\\\\\\/NVpQJonI+jRdAovlqXXEjb1UU4CZRyFFmHfC3G6gKgbSOjpTZHIZkNZMbd5CLdMtYIq9Y1205lyrr02y8ILi5oRPJDA8NKsoflkqUKULQfHQscEYWyP9d9GcPB8HCKHFimZRuy3DTapHuuC+dtsFBBKcozx7Fh6Z3kUKp1KojEygcqC5BbpiexY0zU0cuFwNhsEdcJC6TcgEi3yW3qFadJfZHkCc4jmNNMnD4ysK46e7NsEmX+6fHQCDH3NoOnuy+ejVTja2jLH5u\\\\\\\/XE+Lf1KJ4vgATzow4Qgz+3gV7ka03iH2CGnenQ4z\\\\\\\/LxC\\\\\\\/NN0ZXjQydHOcQEaDvM8s5Qg9syBmTxmSFQkOnKQKr0Vc4I8DoN2KZD71QdZKqnn9SoFsAHtUyphmNeKQ9eET7YylJxuSBT1sXiNIpUStexMwjQJApY11vyEXKn5hxSzvfKE+YDu8ij9+jIS5RXy5dYlULtMwPwcUQfQ388sFZedMYT1m2hb2EgHsmJMCj4oZQs6H6Swv5TBErfdCAVUVcOGxRGEZH\\\\\\\/R\\\\\\\/OH3qr00SOJObAThcy+UcawR0UZEf\\\\\\\/QPgpdeAcTK2qHcY2QhVXpFesaPWo3hAeo0c96yOpP2TZd5h4iez99jA7+6e2db3vLrwV6OT9US+GI\\\\\\\/09pnHV4TQc7OAuzhf07WFBGKCCOVkZ4HrDbXTvhMRYjL9scTA0BtgXWVPc+cuBav3hXR5VjgQhBDVS0FkRWPDR5jRUoLm7PZx9exqqp+npJBS2W1nT0z3kC0b0EBQtR+vLEM9F1OneMX4Ac37aSW1\\\\\\\/mG0lLu3No9FMe\\\\\\\/+1dZMIBtV3yLymmJpWkABCgiAQiF6rVeYM4rcnsu+kv3XzdHK59iNlO85wmtZ4ZuwdhP2e0bVHQ2kAX62ruHptWjLtDUoZ2iYA4LzdS3P9pOBzndKxB73uUQ2ycj+pc3OrEl+czOIkX0ZHwNSVfQx9lVEywLF\\\\\\\/upfWDXAV+Cixci47Sj5GEGsd21h\\\\\\\/Sqh7RyfKBT4y\\\\\\\/N2Lrpah\\\\\\\/aTuyuoR3aD+vXkPbIWCAQWBm1Y6t\\\\\\\/4eEKRH0vIE4qjryrkJ4aV1MHXRCsSDXoVcCSlX0O4Hjo4hAL\\\\\\\/D8Q7W5QAx\\\\\\\/8JrpkAuDcAzhYPexhCq5gfN\\\\\\\/x\\\\\\\/tCaeDGXVJdYMWDc3K9FFHy39jBD8izKpzMKAp9tveA0fZjztMWqTvTdlwH2vYD0MGmFqWJQiYF7coMDLurcOf8644gpHRMssYP6x\\\\\\\/Vg5OhJWX+TOOYIDHerYOC4ncmWk47fa2vXKQoqIKFQNJALpCieyHt2g8\\\\\\\/RttmDY+kBdL5WBSzWW7pY4wcMcBTONGYZnnaktzW+zKA92553rP07itNI1wigxVXA6KXGWkwdbMrA8ma8a5nfjBx6zccz1LIGlDJVcf8dZSuJKpUrJpdswdSVmuJmx9kgWjHFXCDvrbexybMCJroL49tUg3HGUIZLk\\\\\\\/QjUU0wXYNx\\\\\\\/8eDQbu3Fu9cUMgbLbk\\\\\\\/TKAe1GSqOolUfvifSDjV5OEpX1YkiAgyWvLZJNA2rD3hRroXQBFtyXDoOOLAaKKDECPNp3iLYjsWRWJAD1O\\\\\\\/mvVy3Wo0k3bJPvk9uAx3KHrVDSE1aCWEUB4X8NaP0RCXS7sYHQApvFMf8S8HkNlQxpGx1vp1I5eaNOLCoDzKw0ew0VPPcPkx41qsLU5eTEdPyCBsbixkyABP6\\\\\\\/3RWzmE9ThNPeEfHZ1Gm5Y7dSJZCP6PL\\\\\\\/NjTog4GK2HmjiLaVJfj\\\\\\\/YHR24hk4vcKMUix0bCLOG4kU1dG83XB0Hmx6\\\\\\\/qI+\\\\\\\/lCkXT\\\\\\\/k7Kprw\\\\\\\/J35TZ3V8qjwgqAbWDogeH7gY2Vf11FTKvOpm0eQtDu3WQFm2x+m0Onc0X1v1cV1Fu7ik4bER1l2lzw1XSCEWNMT6PcOZqOveZXTmTPow72w9uIs7M+8aMFk3jqot54PzZgj2eV+GnbfhDwtTtFvdjNO0CMc6HyzKU46eV1blNMdWTHPs2b9yBdsN40s0N\\\\\\\/ne\\\\\\\/LCskfGRzGYplou0ZacUanbIBfvYCSXK1Lpc3P4KpJzT+OgeqdL6duSnrQMoCtZZd0AxJsLYEKu+F485uJ3rIWMTE9nF896WT59C853hA9HZs6HCkkGKI1LY+IhABwq71XUSRGQ3NvopLgt\\\\\\\/r\\\\\\\/Z8jCrP8zFSWp6rsI6eN9pXQVO1h6dzJG6foGtA5vnu7\\\\\\\/OImKSB3wyOzNkwnSyzmF6LVrMqbfET0uHjra4txhWWZTfu1er\\\\\\\/3HCSQgEGhDDAZiwGkgzeta3b5kMLO4XG37GdJnTGjjocH3TOsSHFFdX03oqBVqFE1XlGN5aox3jPu+Cxf7snz9twyWrDrpYLYnqTVdc43+AnkuyXVH511k+d+28kVIVaEQm3RwdLtZ36iW2muWvrszUHXOweWHPa07\\\\\\\/JdMUY1BI+ZEh3MzV+lc2X6WF1cBnvK94Zn8t3fQnCtUyMAFIRAOBCaZVKtLbHKv5i3ac4Z5pUT0ySo\\\\\\\/cJRHuCTCWKcKeT6VGQXW8He9gS3qHhhwRJjy76sj71a1cw8jHBcCoYgLVig4sWX6fH9FYaS9m8\\\\\\\/hmwoz+v2cMeyaPMTb9iTF6sSZH0kaVZFxFlTfhTGZjgfNwetS8k\\\\\\\/2HW6K\\\\\\\/avcEP46mWKYKJXTOtr\\\\\\\/iIVAJ49\\\\\\\/VNe4eC2VhUnD6AlMJnqIBR4Loj7VlqDTTUpJO+xdHY9ShAT6fY7lJNEuXMQOK3+ThubN8K6+c0UXbP\\\\\\\/1rfFVgZF2IDXR2bpEHWuEjV85GB+oxHVaGaRgv1G3qisHzq1fxVDEZtYvnsHdQDR6cjH4DdHkFP76dyT98esJsZw3uoQq+4HXdoZ\\\\\\\/sZnl9A+4aEgyi3GCZZu0gEq+KmXzpse5QEa4LAosUmtf+0Tl+zqQ3ZvFUEYTUtlTizbcQn8RKw5WwocTr62HBl4w144gS4rHvMoXoOQRxXU8GrjmbSI5R\\\\\\\/1oPRWAvLnh0SpvQ1aO0rlkz5NRIbdE77aVvK4OVHVJICE637WXBJ61+iIHqAUI6d4wIxQbJnohr8VUhptRlDbHV3HHzCKsS\\\\\\\/1utRbmY30m4TmfaxodhW4dDCzti+7pOgroT8tJcuiSMXAJ07sHhO8QgQkwbhm++tocNWKLWsqIFWmLxNJwfQwA7ah1JeYxYdAeFGmp1Z5nFJ7Yoc84KU8p2tSf5JS\\\\\\\/X4\\\\\\\/AegmoZDUMdinptQU0z\\\\\\\/L+H\\\\\\\/04hMD2zI1JJET7J2POWsdMxkQhd36GuCgKO+dDEEulmWEwWRpmVdJaD7n8VkpzPYM2GVA7FSTogeW7UafzoB8e2SDbkgs+ic9qOrPje9IzCo2gneEP+2Knrjw50LKpqalhbLScvrLEp5btgvbVnUwYDV872yD4rhHQtaaW\\\\\\\/PCBj4H7KFIhKx2eCMlRlh8abEDlqCj4BJC4AteU7UYVjheFMeYs6V56SbFvpNn\\\\\\\/j+eeNiMm2nhmln+pnIneC8hrCG8U8aIe+HJrwttou67PMRqeCpBf9Dqeh8GYbg1wJF4G9XfRjm4qaEG9MaHfQPRuG\\\\\\\/eQjrMrP4p5A0C\\\\\\\/x5WaqeQU48ikh9SsmfSOqx71I0mnp6Q05qS1wJYcNFHUaU2eWirMyb+lIkitd5wr6xxvcDZbHuVlvuB+dPC53k4BKTIVpxVjQ6dznWsSNJ\\\\\\\/FVaHk+SZ6x+sLsOmGwpACgVufl9iTq3vounM+zKVYhYG2mYVZlXSSde7VF6654s3tQLWeuOgzv4O\\\\\\\/k5vNbkEY3iMGtRSWtQlQ1nFdAZ\\\\\\\/hRFq\\\\\\\/oJuG81w9r+0StAV7VJu8o+E3wAz9BN7vzJ0B5Im5WD5RfAPId7mmTuQMbCntdJ+DF6CuZ\\\\\\\/k629lIVg95GfCFxtLuhq48kccE1PkjUbQMk9p1QqoA5PUChkxE2aUp2zCfy8Uxg\\\\\\\/\\\\\\\/aY2E\\\\\\\/8CI9ZaCo5j6TpiNCd+whkb7wwD6dq9FJulHGGAapa4LYkpd4mKcMVC+sjcNyEQn4rIGjwj50tdfUYS0QxPDi42CI5JM4AAj3eY5jkRx6psXvkhEJW01PwR6AWsUdS8TUw4ix7C8g23buwQTuxjapKRCol74rdZ5Aet9cxz5dQHdEu9bg\\\\\\\/i2ikumCDd1e\\\\\\\/ef8hqxMY7H1Jii8qW8JMw+bRLJe\\\\\\\/3vK\\\\\\\/DehXt6z7B1tYU6z+DtPaTyRNMwOemfgdPuBsIsSDa0HGMLQ4gi5p1Qj8oLXHDLNUMxILPV5vWduG5jn4nz\\\\\\\/kyTfHnHdEJC4XFS+ljJtlqwkqVOz7zG+g8DMv2ADnjTdAivp+HNyvXUoNKz1dJ0kL9IB5q5em3F\\\\\\\/AZZIe6lUS556wmbWB3Yx6Y6uVqt1B1rymRrYu43XbFzL46t8YdiMOXac73YdjzLBq3AYA\\\\\\\/hxTcfbdUeKDKSg9FmiHL4RYZWUf2q32+g4SmQ4oAztnGq65HoEjejQLxMTxxqeuUevtU5RI3XjUY18ApgR8u\\\\\\\/fUKiHXtZ1n3u91LGcfV+E631UN\\\\\\\/vNH\\\\\\\/cTBBRm60nvrizVnt0lqoCQO1qtbw3Sa+9\\\\\\\/mf8rHJzPbSA4W5CLjWqpc5qUWSl+f7sSZSkK0cNeH\\\\\\\/o2CT356t9lsTClb2Zjkc\\\\\\\/6vP4t8PmH\\\\\\\/E6c7VkU6fo2hRCJhxEvmzOW4sMKYWaHeijCouUv8sMvU71EvqDmrRCAFoA+njkP63MPC2O023vAJCxVU\\\\\\\/NLy3AqtGVEcuIKpQL129ZwOLEVQshRZzanmruQOneZ6dY6d8WHePu9pNoIv9TCupR2fr9AjuW3UWNhWUpayo0x6jUs2z14ft4QD2tAn+SVlE+usQeR5QianrkcyzJ24Yrf\\\\\\\/HAqCTT9zjCUsoQv\\\\\\\/2lBrTVWAdgNATqWhP5k9P8eAjUzRrqmu6Pm8J+9GJVkENRS5G33SpZPM7PZiY2bhPFQraxd2GCRZQzhjalHaoQpz1PpbsI70RLtN8MYwkAHiEL3Mja\\\\\\\/s8JXGPpqcBvmbr+rze\\\\\\\/00h4K34QSBt++I0CiWHCwA8DTiLe4O0C\\\\\\\/GoWiY9aD449VZkCTZ1JPTZEZc1nZHOQ6msgagRelk\\\\\\\/zeVT2N+G\\\\\\\/\\\\\\\/CwNihjTfdTyqG16HQnlAqJPK3WCSGjeBfXp5bJKJSJIUSy6b\\\\\\\/TDHekbV+GqBJKdD1+Lii1Nuey9jC86XT8cb8BLEVbsNFubsnwb9OBmKQebdansdB6gMEHbzbUkxHZ+dwJNoipedyHm0NY2VQZ5BolsQImczaX87uGRSvna1ZmKZx22A+Jg6lPoM1cYAk5KT+MfZYAPScjBIhGJhqqrcbMpkTqXWtRhv7DIaY6bs6\\\\\\\/06IVmdY+iSWS8\\\\\\\/VXjR3213vpDaKcO1fB0lUMlWZ7dXo\\\\\\\/YIsGUgBcmU\\\\\\\/QLZ4huJQpQKEublpPCylnkjCNkDLCUQcKJUT4\\\\\\\/lc+WbtfNVqFHu9MwU657d8cVK3zqMztD4zXWT3lwBkynMVlQXwuIfPbBhQM4PDIAFj7bIUokqLryehorZBSIkz2+zPNDvxDKE3pRLPdT1+t1F8TG9Na4iPUBImoUMehCiN99S2eAOCnd0elGAlQxCUjCxlc14t8S6sAySca8P4FJd48XfnakGBZI1S8eu3C4aOLCUrqNCgE0KDcQJNC9KaR2zofJmWT2ksl1dTy3kiLjbNpwDjjBoX3mJOGdCzwT5dJSZ7mtTrryglcHs7x453wTyNEugTtSWxUJpVEeWcoGWvmx1nQh9iydGrK0UX4S5\\\\\\\/gZUTmvxRw6HhX22I+MV6mP7YF\\\\\\\/YI2jeFr8MKCTz3hgZKp0WIUWJR0ULmeXqStRJmYUJWA2cguYTbdLpAMiKiaCdhr4pKHpVc\\\\\\\/J6PcDMM\\\\\\\/4+DpwjOyNHP6FQC2AYimj587x7XWIInFLIV5TLPkv3gI94Yr0xrG6txRB4trHlhHWOYHXJRggiLbCGBujv95sdbuWrCs7zo\\\\\\\/KXiorUgk3wpr\\\\\\\/kz6S7da3uuC6cR1lNgIhNkhLFsl800c6yLcWkCG8x+O5oBdEqNIAGwd6dU2ffD8+9d0TiJ74x6olwf2mtjo8mrbaMrAsX1PcY4vvgkwk8R4n0uq1CPFOLVWfIra1I5HsEaBscjPOkv6HM4I2Mi7MOP\\\\\\\/Syq9RDRupUW2fAl\\\\\\\/+XjQjCegKtXrOZMmp5FGX3ulXP1BXOv7vh5bCc\\\\\\\/y8\\\\\\\/FRwxrFV\\\\\\\/xW06RUQRQUEglxSxEUi\\\\\\\/1yzZGJz8SbbQnU\\\\\\\/bjLaBC3UvSL2L6+OEF1iXFLlwOFC8QQG31OyAkix7uA\\\\\\\/JpoRssoPCDamFZplz1aGb8EtOkBsUPJC\\\\\\\/HXfoXa\\\\\\\/d7CI9RmYoteHGEXAgjgcCEGG6oBYauog8W99Ua+5A\\\\\\\/tlkqYDnKK1Z0QtWVQTUD0i6f+y8sV27VnUVXNSEwOmORiPFoAVr8KIOiu1XNAaQxWCUJfBsTEbo0WKMNGx68MFBqZiIp\\\\\\\/L9oSCVvRummGdlNqIC7UWJr7Wl7\\\\\\\/zW1+uT8OHx65pUFfhQksJKtZInUAi\\\\\\\/aGfmcxpE0C7j3nVmDXgGPeXBkykw8bn7h7fmLR\\\\\\\/8+ZWYrIWiQUZke4uHKzyKy46OpUJZ+URNaqq8IMeBv7gM8HGZ7KK\\\\\\\/AErVPF03G56z\\\\\\\/tkvqYGgsY2oh2p96gVr9QEZzRoSxEZzepFUziCsX4gMBq+JoIqkOkihfvvj5nUCpiSzYOrZ8jXR3u3UlZ6l0XPPPiNPc713RELmd8FcHS0kpNUSGioaM4tgoITbEFu8uZIviIJ+opT23yeONARCjEcGZ9cWNGw6H52UpYy9Ydh37TmdvKGl5xpCuS0xz5jrcYmizm8+lv66VCSFcQJ1tOOQ9dhM\\\\\\\/xeDxDgpgb83nmajCGrGAtHI2A3plNxOUd0gqs123G6BKfJBSjRH\\\\\\\/dU+6s67eevtco4KBiqh8QswV1PduN\\\\\\\/HXkVWnRFgUR5NMuwvJ2YhP3Oms57YL79QqWT1R9vodq8PeS+9Y+0+w5mvYo71+kwDbttf6KNF5It+WZ7EtLA5L+L9KyVDXKd3gh9GjXIAFahe3NOAcxJb3S4oXGr9IuyIKOxYXsvqZZ91ElXhYMTwMo1sQcOhqRKubmSOtT20xn24EhyMa88CJEKjyWVIpf2IcpnwROzpmTcwHZMTGjp\\\\\\\/IcuSdXnwmzVM2+LthPr+hthWlyc4sPdqXSkWt+Ume8nRBKtdXiFudw7TVJSYAYZd23+SDeaPkwsslUdnfgL6N\\\\\\\/EYXpO3q88GQ39ig5sx\\\\\\\/ijPHcE7KbZ1d5diTJqICj0pcL4ikmX5Yfk8EPt8HD3uKr30o798KU5aSmGeuPG8sKEnmY01+XrzZ8osKKsDehEPELH\\\\\\\/k1Y0ITYasbFk8MvPmkrA+ZgGwQDVHk8s1XDqSObrf+FXjBig7a+AVT7w5a4+JkH1kU6T3NeTTro9Kzge70T3Fi3+GL+1TV5NQwot5NkbkrHRRFqG5QOvZLYhlywePGUJqCLVwSgCBJRzXzKRvrSoOoBDXTKEBrPjCr3XeFgigJ\\\\\\\/nkIz1KJYv7XGeP5vHXdk+BXmaT2UcGRvUmsMQhmZjb6AP8ZoQnulxNiAHrs+Z53Nl+jt+hlei6uQdr793D4MCcDijPPj9JjvfRTOof7x9fowy2U1635udx1orDzh5e9QO7P4uH2\\\\\\\/5V487nIXj+DXzawTSrarmEnU+T4AUOSWrxdz7Qc8N7NLoFn7nQcpiMvDy+Qj9R7+Xpg6BGqNwpOUdwJwi5vn5\\\\\\\/hLEDajRaTtCfC8BC8fq3PPKtEyn4XUAoIfcCPBIz3FMFKRUHAxucX5PNCSsVPYLvTWkrMaU9XfJ\\\\\\\/Y6\\\\\\\/t6Tenn\\\\\\\/JeYN9aI5dmgscRozXlqnfXUkKe4FUkn15zBeAQy\\\\\\\/guGnkIRLozBvK3bcIpnOO+AAWoeR3D1Wska4u34onxjI8r0qm3MX8zP1b19Emz0YMnbp2KnmTesQk4V4AoYU2G7stYxa5lI27r8nwXbnSXCANxxmLifkDb7ckegzChTiAKBOXHbGE3uLtlukrlmCYScimlRe\\\\\\\/qtVKSQZPpuFwFTipgAPnQbfg5K7LlyGRF4iaAIv3sSiA5Y6wPzwaGfMVIvR6VEu1DjLPejkKwnsLvbaFNWEdxelkNPZUgGmz3HnGUZCwQL4v2SpRSUBGUmCwtTCksk3lWQaxZvDpa6Kb8KDQ1Vfy9YMmSWmVYdtN6xT3EmUv1Vqr65q2qQE+7NacL01aV9D3KBKTpgcRjZP817UakZIf05h1Tr\\\\\\\/sjtOCWxDpgdmQ7mqUU+fqoJ8kugJFBm8waVl3NJrDcoQVraBev6uGMkgEK2pfNa1MmnC+Cm7Ej\\\\\\\/AE6uRlTQEIUEfOEy3GPjQFwNrE3IkUGOYNo6pmZ6m3QyyayObx69D8ObdmUcMbGKVdnMlfm8etGwqqqlQtIKVy9FzIbk34V+2IA\\\\\\\/HJ1ncJ7P2hQe5Qo8bdJqqb3ekT8R\\\\\\\/pV3mIDiaCUCzNTHFo+ZEPUoZJmv1n+Zb1K4QBwdlNULjg6ufibkqWLwJ1NkLdH9jJE3X1q8VSZJpa0r5CiNPAK55s3omo4eDWWt7vMU9jWyU0CIDU8MSq76D0BB7sVY4EJzx9CMgwDrnnDFU1XybwuKcdUN8yvtp06b9CRGtJwNiOJtdQRafi40cDUvoKwhRVF+p3BTXzP4VXVmIJyijqwD8Y0fUROKIQ6L2e14k07BRHHOgqbJ\\\\\\\/Jo6B6DxlBSm416C9s8UUbcA9QLdRMZ7+y1WbWeKrkwkbjteSgm78xWZk8rw8ZxUsf7zGqLDyYXkwlOBfRZKC69neLV2O55d7bSImfI7VY0pKpO\\\\\\\/YrHhy4P2fN+4JhsG1mQ7bjQ0IL4DuCUFNdAgJXC4Y9KFNKOCh9f7fwA4C0YIRT4qIYm5PyB9Zu+qMR8DYXfLHVWgKI5PHy\\\\\\\/brSs4xqjQxL5\\\\\\\/ZwKEKzOuOLCxyfizlPFE+9noR40lMWyh\\\\\\\/8Ae\\\\\\\/S2Did3HO\\\\\\\/ca6g28UTDXTc80qsQiZpuYOWQb\\\\\\\/dw6qGGv0ChN+fe9rcRCOc75XEj9h7RSyaZc0QiWl\\\\\\\/Qf9q77tykcAjpRuyHXwAbT1hruk8bvwtXXRGBeGSkN8D3gy2Lic5Ie8S3+EiA71qKs3CQgR5gUHw4VSW3nbVhaIaCvmq2Wdw1hLHRLIOmmLlcgwDDRAChQzf2lqKUODEzQ33kvPVljrl339RKEU8eKBr2aazfDvTUSQ3pM8Snc6DC7mg0oWas55EEGpz2gzv8G1ZQnfHIB3C455Dk705kh8Bob0uU8Pol6HnFzxu6CnrhnvdNmgyK4lb7M+oQx7uIL5QJk6+nxmkCCULxmsyBWT4c5j8ydy\\\\\\\/PEYOHUyxm7glyi\\\\\\\/7gAQYalEu2La3z2TzGU3EIzsbgLAvt+PhYmvo45O1DJvb62sYpSvXTJ4tjOzL2Zmh\\\\\\\/qMkKd5DEMZqbKrg\\\\\\\/khKVwcYMtDHyp7tyTm+rLSUDMk\\\\\\\/q9+vMbox0XHopqxw4zfaJ\\\\\\\/VCFNC\\\\\\\/D1HTeebhV61h1aiUelZ\\\\\\\/VQwlwfVvfaiPmHJmh4H4QJoqwuQGhsricPdnAAVqLG7R\\\\\\\/00jFGLsoaA0FvSNbzVm8+HnsqNSWhZicfG\\\\\\\/oaD5fRBuym95omwRDk88EQ\\\\\\\/EfOGwUcLGrnWdN0uqC9U7SKysnUcoiF+mnWCnSkb0Ik+hED6tdlY7c0GNBujKhGp8rZrxqkSzkQyt3vStmVwTW+CpnOYuqF7Rs\\\\\\\/yggqMygUHh+r8ppmHGp1wEs5nmYN3\\\\\\\/TQ29W5OS2+96lGYGmZewfXkq3+vgiH2EYV+nP2XwOW0x2Z4WIx4wdyOormt6Ajd4FyuhgHCxlugyQnPQ9SL5IjvukQ6bErNHYAjiTFU0mTRvTXAj\\\\\\\/adsF2ZxQC0+5twQCx2WibXqmNs\\\\\\\/QCx2g9PdbTSWc\\\\\\\/2SAYN4lluP6bZSMP9ujSw\\\\\\\/lulcovUscGL2e3WxgC0AYWWFkVv5\\\\\\\/H5Wp5zWiDTZ1OWKpFMybhygBy\\\\\\\/Vt0APFwHb8Px4hAqdn2uQBI4UZhOulfjp8qiQExMcxpFSMdt1s1+efVlA+NShQ5YIoQ1MxSwB9iCJcA8xRcUpASoY39c0QCEvEmrZ2bGn8qXVCsENtcZKYaFsxKGqhW6dBTM5L8DPdriFnV8MjHZ+r\\\\\\\/9gEoO8ye+x6BpmJ2EUUOOduyOcItVq6vKBTGsnFBYDI6Wq5M9FxehsiyqdSIp+NiQVMTCYll4zIOu1WgBY88VvMAdxGQIVm8gI7YGq2amGXYUbgHTtIdqAVTSw4JnE89l3uPYSycakHVOqR1tDkP4AWpkS04pDdhO8imFvk\\\\\\\/Ypk3KEysnK183bNLvm6VB5mdggBW7R43SSLHqcosfOu7Rej63sxaHlPl4EarScNopv8q8Q1eIX0J0u+aeo\\\\\\\/IMK7EdrEV\\\\\\\/D6WfrYWLvM2wpJM3aX1i7g7Yu+O+pWyxwKyfG28HkFDL\\\\\\\/k3YKTdfV3vYXmoGynl9m32C7g9LkNnbdIXf2vDZXdyrrONUUjHbFxjJWbPeTI2RSU+FXqSMuYPKMK1nomvFE13ChDN3rQ+Xq8ii7jKvwwHcvjg9ZsaVlPK+LNZNXc4rlTn5FNweUZTvjLO86+fuUL3yF7jKbZAalDndceb3Xnd\\\\\\\/fGdvjKkZWAWmOye6uqZQnY98\\\\\\\/ttfTwqA0TkC5ZiPpsIGATmo4GC9QrfZlxw2KS5evbL+nBZyeFCGfsdbA0zmukhZFoN80uwhAmG+OCTH3B3xv37ImirmhWQN02xo+xXbrJUgI75Uejd0XX5Netap4edf1\\\\\\\/kB5kabAUXaZ2WTEz7BH2K2xrSr94eZrQtJIY2u2uieKCmPqij5w\\\\\\\/VnEA1nE0+aXMawBX8jbnKE6uBKc6RAwStfN6SuuixsTJZjXxc9KOSyJzrfv2aYV94NZekTJDJ\\\\\\\/ZHbkw2LJPSVLE6NPixQ6dSX+xm2w1byQUmPPKTvpX8xlpc2oN0pnTbclZfXuTFjEmvbiSEUWcnyBmLkjP2YeQjgLlQNyHd5T01\\\\\\\/rxbd9TKKqpvTJwFsaOfQC8sRMQUFYySzDxLLlELuA\\\\\\\/kxAqnrxTs02DmIs3KraM0yYd8GLUYiIWsfmiDEZBx8U9WqnjCFfyZU7XNjxb+kILz4fX8XIJfQFDKM3J98Mthw8PUajdN+d1D4sqiokfuU5ZGXFTl6wu9DbsUPQSJqqxVNHcCAacNRjn4Z\\\\\\\/eRbWMED7htt9hcx8goND72b4Wdpvqszaj\\\\\\\/m3a51UcxzDDY2HkyL0Jy8ak4XJ1cwM7bXZYJbm8Kls2MM4J\\\\\\\/sozgtajsepPYSyr7nt+vBTtkKlQ8lw2578wSHvn2EdpTEO+Wla+kupmuDpmRsD4ALHE2C2WKucvv++VGYjzl62ryUSFbjEVAjK0LSu\\\\\\\/HCzNpWbBoGuCEtC0R9PnEVnpF3E0JR7\\\\\\\/idFTDliK3d8B\\\\\\\/EaurqRVEddG\\\\\\\/t8oJU5J\\\\\\\/jjw6t9WjDjTBs6g7+easo4j9uDQyrIYu\\\\\\\/MH\\\\\\\/CRBeLyRsopDWdgQn\\\\\\\/HGz9XjvlddKwOs8tFenqVjbRugNdIcrpd1b224oBZ7LkkpgMjM8RSGaRNaZzAcsxnztU7WrsrQbBVmjP\\\\\\\/YZF9LHxgGSzs\\\\\\\/dFG8kG7cJrCevlKQvOdFFtoFd7tU1w\\\\\\\/ehfcMYKmNhKF5R0rioEjOIPz7Hgv9\\\\\\\/fJ897lu\\\\\\\/tvysqkkmVGXAFdo\\\\\\\/WqEVplIGppN+V4QXoBsAcrv49W83BIb9Cg4kKujWIqVbEg1PLktaiARaSSvQRvW5ivkyISivG1r3zDkZdN\\\\\\\/ekv7IuIK1VwY6g1v2nrbRpkFWUHzDoQjCHNoy1Ksqj70rIUo5ihw1nFcnLKCxnZe9tHmK+6bXHCSC\\\\\\\/UJYAI9USvMQ7VFq4fJK3JC05BW3rwkO+mz4yD8QiCd7gO63TDT81bQHTVUobeFIMqeoAd2grTw0mChiAfQ0bxm2dkrWtpyAkvQc6PnnWpoxBqWbusAe\\\\\\\/XUBz6YN7VLSYN3aR\\\\\\\/rptp7krZoWNjdbTSCkgg2ygU1vBYtiJzWKlIg7uY0DI26P5mwCip6ij6kiyf6sJth\\\\\\\/N8T97yaa0x\\\\\\\/dzMTlNK1MzVwEhbz6LjYNz+ZH08+yiL509vxE6VZ+BJfv+kGJkPzEQJ4BmYlG6aTPfIYy0y09mJ8lqopNXB9eW23BNLiT7XLti1nwDZoyaIdN6O4HICuDoxAC5Qbc3YiWoSHvVDN1es7CBqrH3abcatUQ7wxPSVyjOSLcepwIOfFvnSLUapKmDQm7Fr7BfK298j7D2O4vyzfEUzoxTnTTb6WhAGkpBXzOndQ0V2x1F+I6groXiyc9I3kbn8pMKtOsTrBobfbClRXh6g4pObFEaUc2LnLPKsO3Kz6dbXbckCFWyd7lhn9X8W7qrTV0VAZtJ3gJk8\\\\\\\/BwSmrwqf1FMi2BMZkESPlL8BP\\\\\\\/C0HOV4cnnQ5O0JjI7hSp\\\\\\\/LzehDR3Bf+Chekn9stsokwzQ9l8uw9vVxH+eh73\\\\\\\/mxZc3eH5YFffWcl\\\\\\\/rsP+qxkNSRwev\\\\\\\/cGbp0A\\\\\\\/VYTLTlrMwdDshjCQy4foWtwiipJpmuo+zkirdv5KEEddKVt2NNyvTaxzX14F+VgJqBpsRPXxL1r0xNRW6AU6UkuEgoRmzfQ6oZ7DjtfSiS4F6RZg1qNg+H5MduI5AHv5w\\\\\\\/a4l2T6FmkQR6CJ815PR7HCLs7HyCFh5kMntcP\\\\\\\/fJGX05vRmOf2sU7rkHhEPIycMkPcVC5YA83EGIWL\\\\\\\/+hWV1vpm0O4O6C\\\\\\\/WVOfl5+pSDSCj2Z6g0VF0SDXgoVvhtID0BpUgvVoXUUunGf6scddXPL+HgL6EZ51Ycarloa4PZmj06kDEjVMg6VHDf0vIrM8Afcd\\\\\\\/eMSeoE2peAdnx9GSRTkWWeBsyRBKDjivPHbYvVtdsb7LRLIGy6D\\\\\\\/efPjhBX44Fb+ylaXULanRJQtbH5mtahoJxgtgT11JmZxP9uXBPW\\\\\\\/TniRv4bcXYTYhc5\\\\\\\/lyPvkAvvqvZzFCVQILQZPy9vEtFr0EfU\\\\\\\/1LCAmDeMM8DLGahqsQvg+JmYT9Wx+VbSIAAQaGq8BIyxahlNrP2OdJWdZYm0FSltMXKaGEZM+dt8vTt1QMJ5FneHmgM02JMIicW+PH\\\\\\\/XmBYKShnhOphEmBHc+UPJh3MWtIuLNyHqekpA17kpiX6Wr7sG2Ba9psEGIibTUa3AGiUCj9FKlED4sYAbQMDWTtZbEsDmew0hFvmLOlpwrUHSaaaGT0ZGYGMOSsWTiEsLUnXXGIv8TvPBYlrRdDdWkZAEs8we5NVU+nJtiYRDobS4Y1+yIUUznwFgVgS74cR9Hz9aDqfrfKmako4ZoidKgkulSi5J6lfYEAToKVqdU3r78j82jOeo9B4CJkIz1t2VQW8HMof1w19vRzGVeT8bU9FuEknbb4VbGy+Ch7QKJ\\\\\\\/MEY0VdkzzbjgXe14w1GYZYKhfZk0NoHgYqOZZJogX1T5Qdyn9G2Joyt+3JGNy+Fh7J6auRRshPqK9nefu5NcaprAjW9TuJ4MWks6OfCR8lidSTcoD29iHfoK4rywmQhdFQnRMcAR6T+vcuuFcTJ3nxO+S0TuSu5qki8TbtEpGA05KNV8AuXM+xbg9xuWFCqIx\\\\\\\/gKRov75LzCo7w+7pKmbV8WpQxlNPzJLUIR0UG0tUCq4c4tNOLZsKEiBGn5rccISkqtTjdzg5FohUgHVnBXWG9W4Ejt2a+C+++HB7m3A7kIwlgf+\\\\\\\/qpYBV6LfHnbDs+cIzGznsA2ErEpj3Dm1hxmR5VgIVnJ0LMTi5UBcYoU26+63f4VWDeXp2t+cuQXm1LUUPw0daSwglC2qBty7TlebgviUywE0jTR0fj4KRktegKPoC8XlRKY+tW1q3CZi20g\\\\\\\/U0vO0X+QJfSEXz4yg5EWlntkBMqVFBpjQEneDfqT\\\\\\\/ZxDrnjuVwWk1Fk5QQJi9mQ7IGv1bZYNPuKz4kr\\\\\\\/8QTmY985h4HOFUS8S4m0gUtvijPD\\\\\\\/NnHolN09u+sD5xjBtL6uYipWgVhevobkueJ+aMjtYofWHlog+vl9iEd5L3VfddB6FmabAk83L7ih0RoEyUcPIQ7mWwBAKZCGqXWkA5ql73niiqhCxouESXizZ9gzlqp9gLW\\\\\\\/SVbwrMShw01TkrJWW\\\\\\\/egjzctwpYkh9KXei2mL6uGPWHKoqwC0YSgmiD8EIRa0THoLPmQceD9t0qVG4g50mng+2i3HHoACtEcQRieaOzR+hIMcOx5aDYEq1epojYtXTx7uxz1nhhWNbx8Uvb9hZ4yCCWtMUxAEWLF2FvP91DVHMdaj\\\\\\\/MfjxZujA472R+soGLij1phMSe8y+MKE\\\\\\\/Hxi1q9P29PT6GdWdB9Y9Sq\\\\\\\/CfeXPeXUbNQngHYi2HXwx\\\\\\\/aOLCaAVjFzC18h8ZcgTSwXIYdVwtq7ypFpjhJbn8omaohCf7MFhOpYRx3Q2QDYz+ATWeegdF1WJKNT3LoOZAwQusXIdoR\\\\\\\/QcgNXZUAnDqdvd8AKfds\\\\\\\/hy6ihIzRQB4guIhPpTrCh51OHX0tMQQp8+5G1KmpXQeWhA5zYcsFXZ34ThD7mGXehy3ldo+ejUIL+rrADAZhdKqjyYl0Uh9eBlpYeXcvBd+KgByq6pX7LXcJVQyil7x6IsFbisUX95nKTszPgzIfMrlGtdhrT9BPj67oWA7Pe8\\\\\\\/\\\\\\\/o\\\\\\\/Q3ee5\\\\\\\/7vesR5gdP1naV7fsVNBEY6X8BtlS0nyZXT8O29IL7QsDX77TMdPUOTSFHUPuqyCJWl7r6zDm8+dcn\\\\\\\/JEg9lHaqSAWZPyHtmEIxIdT2wBy7QvaoKRFUkUJKnU7sJM0DSn7XlVxwnUsTuxOV63xr4NW6VMCW1JzGhGS9c5+HJQYtrg5qt\\\\\\\/ulGnXtYtSS+izdY9yLlj14\\\\\\\/Nh46uWwOkHPMidlL25hyp239OTIbu1DKJ3idbRL\\\\\\\/XPvAOHdP5fFys0DnJjM17sY43D9ISIh+bQjHO7BK6mGeN8jXLFsKYNfqXqsMwrj3xiXU\\\\\\\/Q7RbqjsSoC6w7fI1hDsIoGLzO6f2s6y8gcemZ6\\\\\\\/ztNYdPK4BLTTbaJ4S\\\\\\\/8u6KqRWvYgFTJr0\\\\\\\/TzLYeHoiWP76lt95\\\\\\\/7vD6r3Z0sGGZWTbdKA8VpCHZuDd+cwVeRA+IybIAJKaJ\\\\\\\/y2y4jc1Q6o7QPIihj6ocrZURsuNUMAyTeH8+Fw\\\\\\\/m\\\\\\\/8\\\\\\\/duTkmluCtwyHbzKdG0xCZ+6+MjTUzhQMbJm20jnYPgd7v+\\\\\\\/3RkSCdfG+90p0yROYNZmneD4cYhQKWstUsEerP8IKHRgDOmYE6sUTtNbsW8HC\\\\\\\/5xOzSvOHkz0BlMF52RzKsEIY4Sg+Xem22X2fbc08HvGA2JifYdKvqf6GC69Sk6NuouL9bxi0hvhmqq7c2KElz8c6KOxFC2LkKezoU+T5L435iZSxT1hsAqAzGEzhHbb9JqKdSJrUEO3tYAQneqezzebqZTiEtRjjptQqe\\\\\\\/5xYV\\\\\\\/l+dLiquUbcJXPhU0LiXUiapCLywIHm17VOw2qiaZQ7u5LAp\\\\\\\/NcpRfLD2nl+bCLtXW6dRYxixol2QqCe+nbR1hZGT0Wiqr8\\\\\\\/oYKsXmL1jwEDTJO94ONZZW\\\\\\\/ZkwSvHtHrQs3B9CxpU9L8UvElh2M9KrFb+K6epWZw6inO5Elef55P8eH\\\\\\\/7Ebq6XlDx9M+8\\\\\\\/jNRJoMUYL7A+hJU44yxrS\\\\\\\/ID\\\\\\\/J8b6+xmoZK2p127nTeScVn9k+s5QBa9DGTXtc\\\\\\\/R4BxkSGLlfiW3ybz\\\\\\\/DPhA0+XNaRAfZTQhx\\\\\\\/CHb+BJwbGmsXDrcULOJmb0g\\\\\\\/DQx8REdozNZMqwLabMUa\\\\\\\/aof2KHASPtvCZP1Zjt0TelE8XS1NOPSf0pIe2lrto46gvS5Mw7qizL0fCH3RnYywNp5uuwg6nDEQPrE8tvzpQUJ+I2bCw8KQUpJiwKdmMN4yHYiKW8Iz5tm7FXfI8OXdHt44fnMTyb12zUsxC5E2eqN1KWPFBuzJ3dc04rgmHKKeNQePEG54xCxHWMisakOIiJHk2oLcVNTXsmgi9KcS8D72Whq5OPRTyRMWAobqoGnG7Mz+flsMBevapDWZx3qBcPfFdalDqmnAxGRtaJF9f+NT5tV75FgtenKVfnO+N+r8UjINgsFvyXUIVUsVn\\\\\\\/XxYc+UkSv7eVFjb+u6rE+3SctXl3CAQzRlt7pLpt8VD6H7zQB38gOuGDro2R6oydRLI+LMi2HasmtCUCbSIv\\\\\\\/GCerxuJZ238cr66VNs\\\\\\\/8i17j8WG+\\\\\\\/rc9akUdbTxNnFMQ1Wdf9LcleQqP8Ij8cBg55K861\\\\\\\/6GCOfTc4bVvY\\\\\\\/4FajSB0+4ohamXGAAyqVEM7JMsByJJJfFkqlj7mynOYoqew5ZfWMrMoN6Ga8nY+apGCIvMRAfufpZ3OoXJ0uoiABYsH\\\\\\\/zsD3WHWeFfK7Fj6HvsxNosiHcGuIiMNdJrNtzadDQg61H8UJuQKWqsHY8lTHOVjXpCC6Wpqah185RTCbEaMBGk05aVgNnB3e6HdTDLhEHRH8ZKjI7x\\\\\\\/1bDtaTRqDimTK+LONcUXWZtxCVbsywlvZK1oSIXVNZuJXF2EVsg3KVNuhxXOPpwUfGM67qC6Vgse8wbbEflorE9spVVkH2SCfgXLW+P8QwxP8tS9Infl4ly\\\\\\\/pqHR8VvnDGR0p7rdt0WDR4A14QFlhaUwDrONlskAi6Mh2iVK0MXhjQtWDiu\\\\\\\/8dh2prygjfa+W8rFLjINqcabeII+sgpYfwODSV6ZfHHt7VP7FjMfQm45d6I00JMfIsv3h3LLsF+xJXGYLo8motD08iXqXzghUG3t8PBNTaQv8ScmHsRmO3X3pGmH2RqcTJNXgR\\\\\\\/krCKYAGeaQn5bdXUWK5BsZJ8YiH3dMD7ebNF0FL83HTnJecqAQ3r5N34kyfKQVXN7fkC\\\\\\\/6ElgOCazcY1uFkxwFkLTUcy+75gwEcALqkLLK6xw4Qr4SKQiRP\\\\\\\/Uh1mv1tfy3yRW7O47e76xohIy1F0BDWyCfYza8FP65npo5\\\\\\\/gtn8m3a9JbLxxspIEB60K+yiNA48ZIRjLaSNu30gXDwHXcd06laMtB2rwwhzzBSOjxRetdwNXJmRq4emdClSdtc2VEQBs8cp9n+heafP5r9doyX28x6zeKpde7TDPGt\\\\\\\/G3SMxuAgsIsSjJiu1QTKDL0isLtJMqm4X1Ig91ek0zs8wEhdT7CDziLsvd+bNWs09tx3GInAjVtGhhBwr3+L8MBspz4CgR00ZyyjUNrryygmsTm6D\\\\\\\/VpualW\\\\\\\/ZY2cm7Jack+xFcqdpuKld7anUaTmEHvAUqVO\\\\\\\/xKt3GbP7A68qcWAav5EbdvNuWIm4ucLTq4WI6FyC93MQI\\\\\\\/3bsGN0VRP\\\\\\\/1\\\\\\\/efzf6IO242mC\\\\\\\/DoQmYTaVOMw6MOnMD3A0njDlbYRg2WPs3Crxw1m\\\\\\\/0+xEBazOkmUHncP0VqbaRC48C1vqanMIxgwH3yA8mNRdpSE+fMYb8W836Agj00fFBLXGX2QSfbyS9lgUbI+t6d8lMKv6FvKt3mcGaKjiZdhLoG4V3NbXiqfs5IvcwnXZ34y2zQ0FyZcxs+W5lo9PwBBxYJmlq1cWsYEURnBSZ7c\\\\\\\/iBfzAHYlQ6HN4xgPAGjdUrssz7q82GZZ8EdErK0wr4oGirkASTmwo4mZjmWSO\\\\\\\/mmeAnASwnNQwhUcUdNmT+zNxsacjTvvI3tzYnVMTZ8++nQc6VXyW06xhiDjv1mRaedZ8q8R85Edik1SFfIUvQjQHAAanqpt64dR0bQjDcWyDwJgxWRD4qXCJBaTj2qa\\\\\\\/jnU9s3HWVdF7RZD5awghC5xA9lJPA5SqHkCQYvhBYYKLRxqq+CZL8U3nf5YFfR12xeb7KP5WLQFTkJz6+Asgd8fDhpLSvcEEO529ye8pb2c230pHpSffdOBcijRcpJDaSX2Wedu45TvNkXLmZ4Bg2FXZthNRIXHTKsYWd+QnEGLbWLpBn0+PYG8J\\\\\\\/slZFwIoOQTrNYK4WmIvRmmTTKOoiKMkXY80+nMeahDlZSzUPUQjxRVfWHB3FV7KPqr6O784q245VInVm0ZxtXj4zTCHVQgTP\\\\\\\/OAO5V9rag+7P1RO3LLC\\\\\\\/CFeMKWMOcSH0tZgiUB0unsMyMXxG\\\\\\\/m4Y6+GYyC+RaOGXvtuH6fjvsmEPXjFopJTlsOetGbJ5pa4lf36y0NrNuz395lFQz6IJ1VwQAzwMm0FSFsFRUnbGXXf5gWGrTba1TNkVUQv+z86hPeICIzircAZ5h48FAOvkyrkwLZa295SrCMRd2B4EDiqzlNF8MhtATLY7YXmt4rUFdPpsJUzDqc6TzpFulHiN6mbw\\\\\\\/X0rxpO5w\\\\\\\/KtsUWxv9A1Ovc3\\\\\\\/IjiobtlkSK2c1oBmJTc6iJOSV6pCffT4lzAIVp\\\\\\\/6KQoLULXer42HcMbTR9x8wtVJxphDjVvDUjZ5dB1xOE26srZJqDnw6RDe2AYOcFY1RVW5Wpsyd8+\\\\\\\/BjcQz\\\\\\\/uMw8G62wia2waVD1M2E\\\\\\\/LE+ozZZUWPhCY9thq03wGA52m6uxynmfFxRbwb6jXFlL7\\\\\\\/gLR5y9zmVkk8MQpCd7+XTodk15NruoLZ\\\\\\\/W76VXFaxaCxM0n6RMfO7W5VPP4JGUHJohW8FL+PQ2Pgipw1o3Y+8qC\\\\\\\/WpqNC20gHZLxla\\\\\\\/vwzWw7x5SaVNpjDrrx4trR1wFInKgjMmQo2V8wlzK6lU0VOxyx2P3F5ACYFU6D4Y4JNm+Gcf29RXy9YTgNg29hlJOJ\\\\\\\/6ta7vLAtlyWdH10jyM5qXPh5WG557uhz9qoKN32Fv9MCmUDeDFIU1+R8fHxNoRJ3p33idO\\\\\\\/gqB4z7fGVf8M7Ci\\\\\\\/IDgInLV1QqZfJigFs7iWWt1BXepQJhRbXm06L6l\\\\\\\/gMpn8NwS9hoReZCmRTOQWxBfFidtw6CuOf61WJ+GtuPBn00gadmxwIkmid270NRZPammkDwgTPFm71zLJJdBEbwAdDT5\\\\\\\/F+cdQCSstVtFQuHhqSYVjMTiFyPZmopmYyzN3yYzT4vDEO0pPMSkB6K2Fc7GZynPwsKK2GMfWmRNxnt2pp5CMXMQQ7bGA0PqJ2TVWAhL2BwWrpIKPOvokXR7+5CQxVVYNXOthzV8wNWzPkkhrRjbfXSqDt0mmWRtugLGysL0yfzmXTDfROgdfOX8HzsBNXVo5fS3Cs\\\\\\\/nXJ5SsWbn4JrwQvzos+T8j5s\\\\\\\/8mgRf6rqoa0Lpu\\\\\\\/UkboFTiAHTjX5HWgclB3Y0UbmnMU1P\\\\\\\/pYTB9v0a+oUQFe8wz7kO2ZZP0C7Ker7XMFKg1SpTbpzxaFKtjViAOGcq5eiJrvgMiUP+YODoI+COliztjkRQMlpIs8qAt6KD076DtaCuB4usbyS5s1twXXgCYOWtkzwTyvpjzDBra7eBPr4uslj2HFT\\\\\\\/zlqzDir6dSxozh+vBWiAoPS9uMfn4N9YHYrq+CirXZcp3Grh\\\\\\\/u0SimxS5Nra6XbA63AWu2d85zIjA5PDjaWyYfRlW0RjqosAgplX0zRuJTf0aqXizbRd7deh11A8WVQAEGLgeHcY5zKRQUxiDLij1hb1ae2Bp0ZTgSSwHNILNNAOTeM\\\\\\\/Bvyfu9L0dlkeivBOKJCGj1nmwJSaAvgboPBbhy1x67pJ1DeezROQuKlrduZaeabE7ZnkVWvyRiyxNLGCDqZ9a\\\\\\\/Aey7tftDmyuxeoqyqZ8KB0qIVg4w5PH98gp+MwJ0f6HohvjW36w7NCwnB0+2tAa5Dm41Vf2iV81DVNXQjL5u2iFTU3C+61ABWVNg05sNXbsFmNgZVf9sPPAN6fSBSBwFXzsU1kfJ8waVA5Plq6UxLgJEsU9rPtD\\\\\\\/Z+o4hAapksZCBsRPqhkP9nXf6V2fx5LqNsXqPm+qoQ\\\\\\\/dkA3KNA6MjtEUPEEjQoq1HdoopWWkzMU49Uz3ON\\\\\\\/i8yWO7BDGwlrGDPc4UtWeELlfx\\\\\\\/MW\\\\\\\/p38Qe6QEFiIaoIUD9\\\\\\\/rL+ahEWfNvI8OZvu3aCDzLCgs2SoP68EKTHSax4XS5Rzp9\\\\\\\/46ZPSqL+teNIg74ikXX1qv4THzdTpVYnmvMqiEfc31b8h2PPXkCjsBLMcRYZF7WoxvJz2pgt4VethXXqS2PqPOctT3TX15cChFWlDEy9hM94V3pUCFepBCMP522ZJ+Km7SUx6VBVWQcWH6DEYy6ASaKIGJXYsZejSHoIwDKGBfUO3ppHP3\\\\\\\/YoG6Q2Ri4YVrcD4a\\\\\\\/wRPx19kx9yfdw94IiyRol7NwqMjbYB2MQ1KJbtWD4ymZpE74lvdCBZe5DnWBUNBNGoqtKHvX3Tw7FgKUyhxxGNy7ahiuIeRuXwOa7Z055m34PAZTkeTKPlN+Zl\\\\\\\/ceOLmvy0i3WfTtI08in6UeehYt2kJHK+FoU\\\\\\\/0yqKJArIO73ot+bgSJXWB91eCSNc+epVKJe\\\\\\\/ZInCr6Oj5+ITeUK+d8zJGCVaE92LgU1iQqQ7jX4fpVhWVSmwIFMPfmdCCpytVrBW0Cn1T7YfCi3ebjCIH7901FdLpZyssEUXhz4rVfDlenRdRqaQUGi5BSK5NOK9z7kj8U+eH0UqmxcVryq14Jn+uu8VUNMCNymtzq3kImynAbS6i\\\\\\\/NS2bL\\\\\\\/Rbab4wE0dlwV+b\\\\\\\/y9AmebrP\\\\\\\/hfbSO\\\\\\\/cCo7vqniXC7OwiGuTRtC88qCQ4uKR4kU\\\\\\\/KktsUuKWq6rZrxMn5zXj944Glb4VJJaIwrUV2xGVtKCYZ1hBebse8Cns0\\\\\\\/loIXEzZjJFAhoq84g7DlZLBiKN1sjojI2fn6e5EXa1mKEqOWWDJeX2ZQ2Q\\\\\\\/ECrEhiGAlZqcxMklhVvcGaKFrO4GPEna6\\\\\\\/4vmnPymugXsMtaQYPguXV5rTPGy6ecxbh8a1SlLYGp1dPO9+7efiQzeiyYrYM+8jboe0N8elyKzCGKt\\\\\\\/TU1fD86plLrHhS\\\\\\\/J3TkNYOwNz1AqzDnvTea5KGr6pSgsed4saw4d+zW6Oaq5j+gxynmL5uZB+H1uTn27Ml+0nezzkb+re9Oyi1xS0PqwCQm7kgVd7ybPUBQVFkxzM3D94Z8tiSY+SxG0OuhqodGxRwWMFR\\\\\\\/KO4NHrU9CvGvvSJaWHNfZ+\\\\\\\/3EN5R2hp1WETAb5rfLcaMNTbZwfq0PgbeiNEJ8bgfDbffkH6ss5gn5+YFRnljfHNfIEVioXwO14bVp2Kfa08v4CRdzMA0KVfkZfs+JBPt6Tm4hmIt6zaGCML+RbUylyPLSam4FoUClEOsEnDuik94TmnimimyVxWbNTv1iTJBGTWLuj83msEg\\\\\\\/P4jKRzhoLtaqBQtw0ubhyVq3Gw0eMi0MfPX2Y1DBmLmxIUpvJ3Lb8TnqDTFtiCGBe1iOyb0Gpio8\\\\\\\/M8ia5yWgyufUQi8t8QIALp18EQM6b+8fULyLxElqqJT4YryjvXtCf2yl\\\\\\\/PKwnWItqgeX0bfbvGulYpW5X3bGsYGMMpklxCzEqVZIqMdf1pNeR11Avl59U4lBPr2GNcnlODZtEdzBQfTDf4CwEJSnNpgueL3ogGOEV0Gv7hdYiZOnIvEth48VQMke57kYmvg8VVCLpgD3nfZNt4Gsci3kE+socs+r9UHN7lMh+AnsxWBK33t8Nr3X7t+reGGSAYNubck0YoN67XierJiM2tayqn0GIEM3KVhPdzNsQOIyRvtKXS+JiasHvPq+D0sJfmfPx\\\\\\\/gNWcW2NIZsErspK19e2hB4uepgR8AamgbhL1LHN1m7b0fXM7xrAnJzr449XCQ\\\\\\\/e\\\\\\\/K3TZS9hb5HzlnrKLdhQZE5aAAZMwl6wPz85T+dMXoNGRINmf+EQ7GGS0mTf8Lk9RZx30TiYkcGmzbR4G4ixMBDII\\\\\\\/C7P6HpXER8WVMK09u+MElIOeYPtqc3i41sBMyh1VSi3mkhSWjPtJmvIXy\\\\\\\/9CKRsZWs7Z9ztavmv0IoJ8yUN73xXmNZVSCZgmszT7uLg4FXhZrze3zG5U6C8JL2qatE\\\\\\\/Fi2msKxcJaYwzILLXEgjW9\\\\\\\/GOymVSy3f3TMe005doAgKjv3jyL7sKvZMZny2K+tzOKo5owCk2QSYgcM5m9LxPFsUkvWUg9DHLaxNeJp3UfQPME50qzJCphoV6Onf4+ch3A+OjRTpeI\\\\\\\/V6uiZVvbG1qmL5gCYcdsbC\\\\\\\/VtJMEL0o8iyXs8iED6u\\\\\\\/AWATlxOJLH9QnOCe637FObK4qVkUPpsNN9xkLxwhMueZoAGIuN18\\\\\\\/9tF5P0dw1h5nvFcYgVgxYmQ4RZMyRA5sfEhIlc9gjKFhDgwo2zfgCfVxpPiRc256EMO+MOsR3JDHj\\\\\\\/Z3PsuokOCcbpGw1O9fy+m4W\\\\\\\/ooz+V+2qRy+ZOUGSpNwPlt\\\\\\\/6Lhnj\\\\\\\/uaSINtvkBvumozU\\\\\\\/HjRtg0bIoTYHoxgWiuFCrIAEeBtW9KdlC9S96JXPM\\\\\\\/aYt14QikEeeGQAVOy8kgMX2Oh7jmcuIgWj+BXRoHQXyPM4LCA2Hyyi1zHTrZsIjO5XFgo9ExS+eLKnF+PpS4sRueuI9eqUOyWNCl7NQLEz2n7nAx4x1M2t216dJnXcHoarj4fxa3\\\\\\\/xHX8pYRL2GQx\\\\\\\/zjcCEbXrMu+Yygdjy8zavSFJDYZND85GNZ8ls0dMcj569dfBeARLJ9RAr8Pgh\\\\\\\/AMG9D3xxJwdGGRxbQf8k5keNIviMmOA27ppFDtPipYlW90yAQ8LGNQ8VBPlACyOq70OGPe6hKx3rVOSBvztGW3jXrR7CazGaaN5b7NHiFrD9bWmZ1vd4Nk2zklEZoYXiO1l+wI9JEK5R9qMMgbz9WX8s4uXcE5rz+7aGTi7v8qDTYyLSWw5WFXWXh6kb70S33j\\\\\\\/RKXQixDtbOEcVzNB0fw6hgmBUjGf90zgcKn8d3S2yCYi6LiJwGKacla265aPO3aG2zWeLkkRxNPioyWYaFs0+KFzSTaW+n273oZ7GzKKP\\\\\\\/5nJkD0QIiRdDO59srh5WUI20vbX\\\\\\\/qGfGrSDAMFKBWdumJZKa3TNtjdNcFZ7NvicACYfntOtYUuUS5hlEFNrGddnJ2WTriFE4WzyCQ8B6JOpV153QLzUvN+QD2hTR9ihFUxnSajW2kvzFmeDsO+bJzhV2YJkTOJtzvldxSBMTby0XAVHw5V882C2QXRyKXOFY84CIVR77TAMXvFQTf+iCE+mg0nfEQ7MJgPgG\\\\\\\/RVOZidw9NNexx0fwP77ACHEw1pLxzKKzP\\\\\\\/p8GzTTItO3ZvuHEoQoQp0nMMkFKuRDdHXDHu4KXz+Bei2d32OiN+Az41fzOkiceJDsmmItfTsL7Tax+VWcIyQL\\\\\\\/ba4EMJLesVLBwefrKmkCF3qBL40YbTBqHnLRupKx9GqVWoh9pC2eESR3YY3Yd+54WdQnivI9h0bm1eAj7443hDkVtcu7sbsM\\\\\\\/h8GqWm1PdAbLWt\\\\\\\/xVt44n3D2MpWvek8ArBVr+8bH3tUjVH370uGXlqVW5soQ6EgvlnrQ5lG+LmKaleTr21VcKtpzcgEp1K\\\\\\\/1rYPA435b7M3EVuxf394IOB20euu7pjtKacCacetiT1aiSCSw1ITZI\\\\\\\/2dqvQXZkN\\\\\\\/BBjIO0C4w8wLVsrNa3R\\\\\\\/oX7xX6o5FZQIzqSrEP2uzC95EKUdSIXw27ZjPGvfNSDPufltkIJqh+XUzbAb5fNlwG7bf69zAxxkVUAIRCmvAS9Gt2y\\\\\\\/d1B0oM4UH3M3BstR1qrTFqhciR+05B9r5+jiAhmyd8ctVihPBzcJsON7WofQPsq\\\\\\\/nkBHN9COnrNXsGqMUPXpMZkrhH+CpL6PRj+KC+TRl4TvlbzYlkFxdaCooegHL\\\\\\\/u1gS3HVANTaq8l7RifgLIJ96F2i7poOsjb7Qg9iMLYRoTDw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ucbZSCEEngVFJzeEO7um5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227439aa56c36b18b9877c6ef3fb84a8894ad75c8f445d645dedc145677d75334b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bM0UBP91IAxP1dWTu2JeO+Ll37vpBTsKJNDwLoVp1Pu8sRIn2ZFvSvgtuyqQro\\\\\\\/+\\\\\\\/ruyxqHMXcWeo73DOvH1kFprQNViNxUEWsDh93ca\\\\\\\/HTK+5nFvTtSEFI1CMzNxtVW9gYcd+trMRSLdviTj1T5odvAQxb0FoU91PQxSUoqBMMMiryqIBDIRFq6dIMOoq+w+SmhWiwjfI\\\\\\\/bAjzdsI9tzbWZyWbzqHsPcFYQyT\\\\\\\/yKimThlB+64aCBNDnaOiRRdsKjhqZNLHJzLTB9m87gN8Cece2ce4TVoLHCOCh7UMH5LlH\\\\\\\/sr3vCXI\\\\\\\/Yd8B9O8ZXLLBqqFt0uu2y6QULVIUNzUQoeUtaTUCgy9nLbWbQ0LdW02FiGa6mENPqSy8\\\\\\\/MYOpN\\\\\\\/A3FnS7853cEPlgvd+CLfMfgYFEzINRj+KyOM5Q\\\\\\\/uSCpaWvxjpC5W2HZxCD\\\\\\\/uYTJBd981am5pHaSYeeDp7PoN8\\\\\\\/jcyDXx28udNLuPyDemELPzU2jPJ01q4nLt6i660LzYrfj8M8EvV1K04lF8zqrtYEcAZMNg\\\\\\\/armGkQKeWzFb\\\\\\\/zf0cOKtMQNRzNCqeYCGBire6sYOshSi1iqd4pIM3WoWvmMcfo5Fz4UpXFVwMp0ptVS0Nw6cuFPMGBCBJ\\\\\\\/XY6wv76\\\\\\\/QU7OtjS70CUMG3Q4M\\\\\\\/r\\\\\\\/FTxd2GCnzDIrJBIJyGuwfbIWKHJHJbbZyAhEX\\\\\\\/OCU36PPSmvgL0ksqFJZOEJzZMoz7H86askou+RFTWUAqIzvcKbzHIlnzog+B0zPWO2s0Hlz2oZvJgIFwzyY4xTdq6\\\\\\\/aSUi3kTfEJzMpzj7dEaY8kuj2IYDvk\\\\\\\/ehURWy0kox7eYtFp6X45fhTf5w9FcaE2BNsza9iEZpNLPYNJzFTE8Hv1TppWxswHNI2lGcPhTi7GTZFwKtwH2kYry0yLzHolkfWUcqKPtaRxt2GOf8FVtDZDAsmeHynLbsASGdN27ZGGJja4ZLvrxF7pbAxpDUSX1KlC\\\\\\\/EhO2pItuaatnNtEt8XyTdaoRGes\\\\\\\/sFR7P9QHyqUoaN4Vffe7mX5rzG7ltYNROWMsM3G3jwMGkHRB202x51gXkeosURdYUfbTArfwks6ulCx45QxVufeuMhqiZFlOYnTjUw4e3e2Lzis\\\\\\\/6eMP0xKGW8555TE0MJIE0S3t5f9uYPn1aX0YMwNtjxiOOISb3\\\\\\\/IFN5TtvoXqqT3AFQVDYKVvDuEqOPXCZtDg25F9KDbpd5xuHRhaGHGsS1TJYRKuhrFKUKvbIQdUa\\\\\\\/7VEeiljZzQnSow2TTS\\\\\\\/VH7wkP1EZs7HDMnZ2BKFgD5tLDiwKcDRyDQWdapHZE1+fZ2ttC\\\\\\\/FExo4n1VLJM8chQ9\\\\\\\/yoHrX0ktCyRFv9AKit6jRElI0dJNNvciitfNkZo9hNgSp4gxholhWAg9R+04WYTtfJJNuTHQDJx9fHj8xH2sZm1lp+5\\\\\\\/fXe4SoViRFLqWCzd+Q+goBCGXEBCn1HLCwOX3K+oWbcNK6oCflsFY\\\\\\\/U+w7ACAyXFw\\\\\\\/d3BnuLwhgUqLJgzC46QqQvO+Pny1D1h0lwsa0g5U4XRITJTk+82XNBWQuZXj0UtRSwKr+WcJm2TuNTIrPDrBVQc\\\\\\\/codC5PW80uG+NbZ8tNOQaVdA1xl\\\\\\\/RQxG0d6GFmb4h7Eo5i5elgUp\\\\\\\/j\\\\\\\/+wJK9KzevQ7Gntdmc7gRAGAHjYmZje34aV\\\\\\\/9Sl\\\\\\\/ZfRHtBOaINqdTGIR8VGs5WSlGEpMlPyOzfONOV94kanhzniKY6IMvJLmRjxT1befldlxU9DJ0IdngJEPvxBLlZqlWWgnXmu+NRsJdRnAkNefd2OPEyXxfszwFxGhSuClDMFxgYY7UTardxgVX56d264FdoS+HGHuhqXfbwfV7K5UIEfnBb8pcswbinhcQ\\\\\\\/ntsAAPmi03UTVqhjfB+7SZEyDjECH7Y0kfSF74rq2pZvJo8iwEyAdXcOcQjWCuI0USdovrEWZwdVhhZxPkiiOTI9nMGLRIXCQ\\\\\\\/gvyaL\\\\\\\/7\\\\\\\/x\\\\\\\/fafrKvhR8iJNop+bQmxID7hlFKQg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022t487ZZpYVFDJTsfEmmnN7g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002253b80ef1105e0f1f2cc5c25be08dfd554e0ad9c4739b7e54f6d98da3172e60f6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459746 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2365" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GmXGoF3FDGEyvGaD4x4xg9J08SpShIJ8KO7B7Rl-2vE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2365" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ay1SoSlQULKUb9LdLy8lHDM-1552vz4A1iWSDrSKwDQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/12/2353" title="High-Resolution Genetic Mapping in the Diversity Outbred Mouse Population Identifies Apobec1 as a Candidate Gene for Atherosclerosis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/12/2381" title="Fast and Efficient Drosophila melanogaster Gene Knock-Ins Using MiMIC Transposons" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459746" id="node459746" data-pisa="ggg;4/12/2365" data-pisa-master="ggg;g3.114.014068" data-apath="/ggg/4/12/2365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Functional Multi-Locus QTL Mapping of Temporal Trends in Scots Pine Wood Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zitong Li</span>, <span class="highwire-citation-author" data-delta="1">Henrik R. Hallingbäck</span>, <span class="highwire-citation-author" data-delta="2">Sara Abrahamsson</span>, <span class="highwire-citation-author" data-delta="3">Anders Fries</span>, <span class="highwire-citation-author" data-delta="4">Bengt Andersson Gull</span>, <span class="highwire-citation-author" data-delta="5">Mikko J. Sillanpää</span> and <span class="highwire-citation-author" data-delta="6">M. Rosario García-Gil</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2365-2379;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014068 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459746" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zitong Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Mathematics and Statistics, University of Helsinki, Helsinki, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zitong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZitong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Henrik R. Hallingbäck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henrik%2BR.%2BHallingb%C3%A4ck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hallingb%C3%A4ck%20HR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenrik%2BR.%2BHallingb%25C3%25A4ck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sara Abrahamsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Skogforsk, Sävar, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BAbrahamsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abrahamsson%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BAbrahamsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anders Fries </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anders%2BFries%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fries%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnders%2BFries%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bengt Andersson Gull </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Skogforsk, Sävar, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bengt%2BAndersson%2BGull%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gull%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABengt%2BAndersson%2BGull%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mikko J. Sillanpää </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Mathematical Sciences and Department of Biology, University of Oulu, Oulu, Finland</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Biocenter Oulu, Oulu, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mikko%2BJ.%2BSillanp%C3%A4%C3%A4%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sillanp%C3%A4%C3%A4%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMikko%2BJ.%2BSillanp%25C3%25A4%25C3%25A4%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">M. Rosario García-Gil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BRosario%2BGarc%C3%ADa-Gil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garc%C3%ADa-Gil%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BRosario%2BGarc%25C3%25ADa-Gil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.rosario.garcia@slu.se" class="" data-icon-position="" data-hide-link-title="0">m.rosario.garcia@slu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/12/2365" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459746" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459746/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2365.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459746" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459746/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2365.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459746" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459746/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2365.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459746" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459746/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/12/2365.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459746" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459746/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Quantitative trait loci (QTL) mapping of wood properties in conifer species has focused on single time point measurements or on trait means based on heterogeneous wood samples (<em>e.g.</em>, increment cores), thus ignoring systematic within-tree trends. In this study, functional QTL mapping was performed for a set of important wood properties in increment cores from a 17-yr-old Scots pine (<em>Pinus sylvestris</em> L.) full-sib family with the aim of detecting wood trait QTL for general intercepts (means) and for linear slopes by increasing cambial age. Two multi-locus functional QTL analysis approaches were proposed and their performances were compared on trait datasets comprising 2 to 9 time points, 91 to 455 individual tree measurements and genotype datasets of amplified length polymorphisms (AFLP), and single nucleotide polymorphism (SNP) markers. The first method was a multilevel LASSO analysis whereby trend parameter estimation and QTL mapping were conducted consecutively; the second method was our Bayesian linear mixed model whereby trends and underlying genetic effects were estimated simultaneously. We also compared several different hypothesis testing methods under either the LASSO or the Bayesian framework to perform QTL inference. In total, five and four significant QTL were observed for the intercepts and slopes, respectively, across wood traits such as earlywood percentage, wood density, radial fiberwidth, and spiral grain angle. Four of these QTL were represented by candidate gene SNPs, thus providing promising targets for future research in QTL mapping and molecular function. Bayesian and LASSO methods both detected similar sets of QTL given datasets that comprised large numbers of individuals.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/functional-qtl-mapping" class="hw-term hw-article-keyword hw-article-keyword-functional-qtl-mapping" rel="nofollow">functional QTL mapping</a></li><li class="kwd"><a href="/keyword/wood-quality-traits" class="hw-term hw-article-keyword hw-article-keyword-wood-quality-traits" rel="nofollow">wood quality traits</a></li><li class="kwd"><a href="/keyword/scots-pine" class="hw-term hw-article-keyword hw-article-keyword-scots-pine" rel="nofollow">Scots pine</a></li><li class="kwd"><a href="/keyword/multi-locus-model" class="hw-term hw-article-keyword hw-article-keyword-multi-locus-model" rel="nofollow">multi-locus model</a></li><li class="kwd"><a href="/keyword/shrinkage-estimation" class="hw-term hw-article-keyword hw-article-keyword-shrinkage-estimation" rel="nofollow">shrinkage estimation</a></li></ul><p id="p-5">Wood is one of the most abundant and versatile organic materials on earth. It is mainly composed of secondary cell wall consisting primarily of cellulose, lignin, and glycoprotein. Its great importance for the world economy and society has motivated the establishment of a large number of conifer breeding programs worldwide with the primary aim of increasing wood production and improving its quality (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Mullin <em>et al.</em> 2011</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Rosvall 2011</a>). With respect to the genetic improvement of growth and form properties, conifer breeding has achieved notable successes. However, the costs and time required for evaluating wood properties in a large number of individuals have prevented wood quality to be incorporated into most operational breeding programs despite evidence that wood properties such as density, microfibril angle, fiber dimensions, and spiral grain harbor considerable genetic variation and moderate-to-high heritability (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gaspar <em>et al.</em> 2008</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bouffier <em>et al.</em> 2009</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gapare <em>et al.</em> 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fries 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hong <em>et al.</em> 2014</a>). The assessment of wood properties is further complicated by substantial and systematic variation within trees such as early-latewood and juvenile-mature wood transitions (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Zobel and Sprague 1998</a>). Developing affordable molecular tools that could be used in, for example, early screening of tree seedlings is therefore considered a potential solution to assist wood quality improvement operationally. Moreover, access to new instruments for efficient wood characterization offers new opportunities to analyze large numbers of trees for more traits and with higher detail.</p><p id="p-6">Quantitative trait loci (QTL) analysis is a technique applied to elucidate the genetic basis of quantitative traits by placing the position of putative genes underlying a complex trait onto a genetic map (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lander and Botstein 1989</a>). QTL identification can be used to develop molecular strategies for early tree selection of economically important tree properties. QTL analyses based on anonymous DNA markers, such as amplified length polymorphisms (AFLP) or random amplified polymorphic DNA (RAPDs), identified major QTL, each explaining from 5% to 27% of the variance for pine wood traits such as density, ring width, microfibril angle, and lignin content in loblolly pine (<em>Pinus taeda</em> L.) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Knott <em>et al.</em> 1997</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kaya <em>et al.</em> 1999</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Sewell <em>et al.</em> 2000</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">2002</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brown <em>et al.</em> 2003</a>), maritime pine [<em>Pinus pinaster</em> (Ait.)] (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Plomion <em>et al.</em> 1996</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brendel <em>et al.</em> 2002</a>), radiata pine (<em>Pinus radiata</em> D. Don.) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Emebiri <em>et al.</em> 1997</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kumar <em>et al.</em> 2000</a>), and Scots pine (<em>Pinus sylvestris</em> L.) <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lerceteau and Szmidt 2000</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lerceteau <em>et al.</em> 2001</a>). More recently, high-throughput technologies have contributed to the development of a third generation of genic markers such as expressed sequence tag polymorphisms (ESTs) and single nucleotide polymorphisms (SNPs) that may increase the resolution and informativity of QTL mapping considerably. Nonetheless, only a few QTL studies have been published that have utilized ESTs and SNP in pines (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Markussen <em>et al.</em> 2003</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Brown <em>et al.</em> 2003</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Pot <em>et al.</em> 2006</a>). Although these studies incorporated a very limited number of gene-based markers, they have nonetheless become a useful source of candidate genes for wood related traits.</p><p id="p-7">Previous QTL studies of wood properties have been limited to measurements at single time points or to trait averages of a heterogeneous wood sample (<em>e.g.</em>, stem sections, increment cores). Thus, the dynamic and systematic within-tree variation due to age and seasons was not taken into account, limiting the extent to which results could be generalized. For instance, given wood trait data from increment cores, a cumbersome process of repeated analyses on separate annual rings has been required to assess the stability of a QTL over time (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Sewell <em>et al.</em> 2000</a>, <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Brown <em>et al.</em> 2003</a>, <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Ukrainetz <em>et al.</em> 2008</a>). However, multiple observations of wood properties obtained by dendrochronological studies could instead be fitted to mathematical functions or curves dependent on the year of wood formation, on cambial age, or on distance from pith, a procedure called <em>functional QTL mapping</em> (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ma <em>et al.</em> 2002</a>). The QTL analysis could then be performed on these functions, thus accounting for the systematic within-tree variability and increasing the relevance of detected QTL in comparison with analyses performed on samples specific for a certain tree age or annual ring. Functional QTL analyses have been conducted in Scots pine (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Sillanpää <em>et al.</em> 2012</a>) and <em>Populus</em> (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Wu <em>et al.</em> 2003</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Ma <em>et al.</em> 2002</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">2004</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Lin and Wu 2006</a>) but were focused on the dynamic nature of growth rather than that of wood properties.</p><p id="p-8">The functional/longitudinal QTL analysis needs to take both between- and within-individual variation into consideration. This can be implemented by a two-step multilevel approach (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gee <em>et al.</em> 2003</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Heuven and Janss 2010</a>). The phenotypic temporal trend of each individual is in the first step estimated as a curve. The curve parameters are then considered as latent traits for which marker effects are evaluated in a second independent analysis by some common QTL mapping tools. A possible alternative to such a two-step model is the linear mixed effects model (LMM) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Furlotte <em>et al.</em> 2012</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Sikorska <em>et al.</em> 2012</a>), which fits the temporal trend and estimates marker effects simultaneously in one single, albeit complex, procedure. Both two-step multilevel and LMM models are well-suited to the simultaneous effect estimation of several loci (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Zeng 1994</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kao <em>et al.</em> 1999</a>). Single locus methods, such as classical interval mapping (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Lander and Botstein 1989</a>), are simple and computationally efficient, but in cases where the sample size is small, investigated markers are numerous, and significance testing is conservative, single locus effects may suffer from severe overestimation (denoted the “Beavis effect”) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beavis 1994</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Slate 2013</a>). Multiple locus approaches such as stepwise regression (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Segura <em>et al.</em> 2012</a>), LASSO (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Tibshirani 1996</a>), and Bayesian methods (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Xu 2003</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Guan and Stephens 2011</a>) can improve in this matter. For these methods, variable selection and shrinkage estimation play key roles in avoiding oversaturated models and locus effect overestimation. In this study, we considered the LASSO variable selection for the marker effect estimation step of the multilevel approach and a Bayesian spike and slab prior approach for the linear mixed model (henceforth abbreviated as BLMM). To our best knowledge, these advanced multi-locus methods, and especially the extended functional mapping methods, have not been used in any previous QTL study for mapping wood properties. In QTL analyses, a typical procedure is to estimate the effects of the loci first, and then to quantify multi-locus association detection uncertainty or significance by applying some sort of decision rules (<em>e.g.</em>, hypothesis testing). Tests designed for LASSO and Bayesian variable selection often require some adjustments with respect to multiplicity (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Scott and Berger 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bühlmann <em>et al.</em> 2014</a>). For the multilevel LASSO (mLASSO) model, one may consider a multiple-split test method (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Meinshausen <em>et al.</em> 2009</a>), a covariance test method (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Lockhart <em>et al.</em> 2014</a>), or stability selection method (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Meinshausen and Bühlmann 2010</a>). For the BLMM, which uses indicator variables, a false discovery rate control approach (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Ventrucci <em>et al.</em> 2011</a>) can be pursued.</p><p id="p-9">The primary aim of this study was to determine the extent to which QTL with major effects could be detected for a set of important conifer wood properties considered as dynamic functional traits dependent on time (<em>i.e.</em>, cambial age). A secondary aim was to examine and compare two different multi-locus functional QTL mapping methods, mLASSO and BLMM on a real wood property data set, and to characterize and compare several decision rules that can be applied within the framework of mLASSO and BLMM. To achieve these two aims, functional QTL mapping was performed on a Scots pine (<em>Pinus sylvestris</em> L.) full-sib family established in a single large plot in northern Sweden using a set of AFLP and SNP molecular markers paired with phenotypic assessment of the wood of multiple annual rings in increment cores or by repeated assessments on standing trees.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Field test</h3><p id="p-10">The studied field test, named Flurkmark (S23F881485), is located 25 km north of Umeå in northern Sweden (latitude 64°02′N, longitude 20°13′E, altitude 115 m above sea level). The site is a mesic dwarf-shrub type with site index T21 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Hägglund and Lundmark 1982</a>) and the plant material consists of 1000 offspring (F<sub>1</sub>) of one full-sib family. The parental (P) trees AC3065 and Y3088 were plus-trees originating from north Sweden (latitudes 65°08′N and 64°09′N, respectively) selected for the Swedish Scots pine breeding program based on their vitality, rapid growth, small branch diameters, and wide branch angles. One-yr-old seedlings were planted in 1988 using an approximate spacing of 2 × 2 m. QTL studies have previously been conducted on parts of this material with respect to other traits (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Lerceteau and Szmidt 2000</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Lerceteau <em>et al.</em> 2001</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Sillanpää <em>et al.</em> 2012</a>).</p></div><div id="sec-3" class="subsection"><h3>Wood sampling and measurements</h3><p id="p-11">A subset of 286 trees was sampled for wood cores. To limit the impact of environmental variation, trees selected for sampling were all situated together in a part of the trial that exhibited the highest survival (98% in 2007) and growth. Wood samples were obtained during autumn 2004 using an electric drill combined with a 10-mm-diameter increment borer (Haglöf Sweden, AB). Samples were taken from a height of 0.8 m above the ground. They were subsequently sent to Innventia, where their properties were analyzed; the first part of the evaluations was performed and the data were made ready for the genetic studies to follow. The samples were air-dried, sawed, and polished into vertical 2 × 7 mm. From these strips, radial data profiles of wood traits (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, Supporting Information, <a href="http://www.g3journal.org/content/suppl/2014/10/09/g3.114.014068.DC1/FileS1.xlsx">File S1</a>) such as wood density (WD), radial fiberwidth (FWr), tangential fiberwidth (FWt), fiberwall thickness (FTh), microfibril angle (MFA), and dynamic modulus of elasticity (MOE) were acquired with the Silviscan instrument (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Evans 1994</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">2006</a>). In addition to the traits assessed on increment cores, the spiral grain angle (GA) was measured at breast height (1.3 m above the ground) under bark in an extended subset of trees (492) using a nondestructive weight-and-wedge device (Chalmers University of Technology, Sweden) (<a href="http://www.g3journal.org/content/suppl/2014/10/09/g3.114.014068.DC1/FileS2.xlsx">File S2</a>). Measurements were made in accordance with <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hannrup <em>et al.</em> (2003)</a> in 2006 and 2007 in the north, south, east, and west directions using the means of each year as measurement traits.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465446/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465446/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">List of the wood traits, their abbreviations, measurement unit, and the number of values per tree used in analysis</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Initial processing of wood property data</h3><p id="p-14">The first step in the evaluation of the radial profile wood trait data was to identify the interfaces between all annual rings and their parts of earlywood and latewood using methods developed by Innventia based on analysis of radial variations in wood density. Then followed calculations of the widths of each annual ring (RW) and its parts of earlywood and latewood, as well as the proportions of earlywood (EP) and latewood (LP) of each ring. Further, wood trait averages were calculated for each annual ring, for most traits averages were also calculated for their earlywood and latewood (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Then, the ring locations were manually cross-checked and the data were corrected for some errors. Finally, the annual ring data were also sorted according to the calendar year of their formation. The cambial age variation in annual rings of the same calendar year (reflecting that the trees reached the sampling height at different ages) was small, with a variation range of maximum 4 yr. The effect of cambial age was therefore assumed to be synchronous with the effect of calendar year. The nine annual rings formed during the calendar years 1995–2003 (approximately corresponding to 5–13 rings from pith) were chosen for further study because observations from those rings were reasonably complete (&gt;95%). For MFA and MOE, the measurement resolution was low (1 observation/mm) and only allowed the study of averages of three adjacent annual rings (1995–1997, 1998–2000 and 2001–2003, respectively). In contrast, the resolution for WD, FWr, and FTh (20 observations/mm) was high enough to enable the separation of observations into earlywood and latewood components within each annual ring (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) and to calculate wood trait averages for the components. There are many definitions for designating wood as earlywood and latewood (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Mork 1928</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Green and Worral 1964</a>). The transition between pronounced earlywood and latewood is, however, gradual and easily influenced by site, weather, and other factors. If the annual ring is divided into two components only, then the averages for both will be influenced by all these factors, which is negative for many types of studies. To deal with this, Innventia introduced designation criteria including a third <em>transitionwood</em> component in between earlywood and latewood (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Olsson 2000</a>). Each location <em>i</em> within an annual ring was designated as earlywood or latewood depending on whether the wood density measured at that location (<em>WD<sub>i</sub></em>) exceeded thresholds related to the span between the minimum wood density (<em>WD<sub>min</sub></em>) and the maximum wood density (<em>WD<sub>max</sub></em>) within each individual annual ring:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-1.gif"/></noscript></span></span>In the current study, the transition wood component was not studied <em>per se</em> due to its unspecific character. The data for all traits were checked with respect to normal distribution. Earlywood and latewood ratio (EP and LP) were found to deviate considerably from normality and were therefore arcsine square root–transformed (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Snedecor and Cochran 1980</a>) prior to further analysis.</p></div><div id="sec-5" class="subsection"><h3>DNA extraction, marker development, and scoring</h3><p id="p-15">Total DNA was extracted from vegetative buds of the studied full-sib individuals. The buds were pealed and freeze-dried before being grinded and DNA was extracted using the CTAB method (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Doyle and Doyle 1990</a>).</p><p id="p-16">An array of 768 single nucleotide polymorphism (SNP) markers was developed for Illumina Golden Gate assay at the University of Oulu, Department of Biology (S. T. Kujala, T. Knürr, K. Kärkkäinen, D. B. Neale, M. J. Sillanpää, and O. Savolainen, unpublished data), and it is publicly available at the Evoltree website (<a href="http://www.evoltree.eu/">http://www.evoltree.eu/</a>). During the development process, 237 SNPs were extracted from 56 gene fragments that had been sequenced in individuals of different populations by <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Palmé <em>et al.</em> (2008)</a>, <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Pyhäjärvi <em>et al.</em> (2007</a>, <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">2011</a>), and <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kujala and Savolainen (2012)</a>. These gene fragments were selected mainly for their value as candidates for timing of bud set and cold tolerance. In addition, 531 SNPs from 341 other gene fragments were obtained from the <em>Comparative Resequencing in Pinaceae Project</em> (CRSP) in the laboratory of D. B. Neale, University of California at Davis (<a href="http://dendrome.ucdavis.edu/NealeLab/crsp/">http://dendrome.ucdavis.edu/NealeLab/crsp/</a>). The SNP genotyping was performed using GoldenGate Illumina technology at Centre National de Genotypage (CNG) in Evry, France. The automatic allele calling for each locus was accomplished with the GenCall software (Illumina, San Diego, CA).</p><p id="p-17">In addition to the SNPs, several amplified length polymorphism markers (AFLP) were developed. The AFLPs were produced according to <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Vos <em>et al.</em> (1995)</a>. The following 15-primer enzyme combinations were used: E-act/M-cctg; E-act/M-cccg; E-act/M-ccgc; E-act/M-ccgg; E-act/M-ccag; E-acg/M-cctg; E-acg/M-cccg; E-acg/M-ccgc; E-acg/M-ccgg; E-acg/M-ccag,; E-aca/M-cctg; E-aca/M-cccg; E-aca/M-ccgc; E-aca/M-ccgg; and E-aca/M-ccag. The amplified fragments were sent to the DNA facility at Iowa State University and run on ABI3100 Genetic Analyzer. The mapping data were analyzed with GeneMarker v1.6 (SoftGenetics, State College, Pennsylvania).</p><p id="p-18">A large set of 492 progeny individuals were thus genotyped using 508 AFLP markers and a small subset (91) of these individuals were also genotyped using the previously developed 768-SNP array (<a href="http://www.g3journal.org/content/suppl/2014/10/09/g3.114.014068.DC1/FileS3.xlsx">File S3</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/09/g3.114.014068.DC1/FileS4.xlsx">File S4</a>). The individuals of the smaller subset were all sampled for wood traits by Silviscan. Furthermore, both parents (AC3065 and Y3088) of the field trial progeny were genotyped using both SNP and AFLP markers.</p></div><div id="sec-6" class="subsection"><h3>Sorting and mapping of marker data</h3><p id="p-19">Marker sorting and mapping were performed with all the available genotype data simultaneously, but with the aim of constructing two sorted genotype datasets, one pure AFLP dataset for the larger subset of individuals (henceforth abbreviated as the <em>A-set</em>) and one mixed SNP+AFLP dataset intended for the smaller subset (abbreviated as the <em>S+A set</em>). Because the studied full-sib family was generated by two noninbred and highly heterozygotic parents, a two-way pseudo-testcross mapping strategy was used (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Grattapaglia and Sederoff 1994</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Grattapaglia <em>et al.</em> 1995</a>). Markers for which genotyping scoring success was inadequate (&lt;80%) and poorly genotyped individuals (&lt;70%) were excluded from further study. Even though the inclusion of 1:2:1-segregating SNPs made the construction of a consensus map possible, it was nonetheless reasonable to assume that different sets of QTL segregated within each of the unrelated and heterozygotic parents. Therefore, the marker linkage mapping was performed on maternal and paternal sections separately (see <a href="http://www.g3journal.org/content/suppl/2014/10/09/g3.114.014068.DC1/FileS5.pdf">File S5</a> for more detailed information about the linkage mapping).</p><p id="p-20">In summary, 153 AFLP markers genotyped on 455 individuals (the A dataset) and 153 AFLP and 166 SNP markers genotyped on 91 individuals (the S+A dataset) were retained in the analysis after filtering and sorting. Two-hundred fifty-one markers were distributed on 26 maternal and 24 paternal linkage groups (LGs), whereas 68 markers could not be assigned to any linkage group (unmappable).</p></div><div id="sec-7" class="subsection"><h3>Multilevel functional QTL detection by LASSO variable selection</h3><p id="p-21">Following the <em>multilevel</em> model approaches of <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Gee <em>et al.</em> (2003)</a>, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Heuven and Janss (2010)</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hurtado <em>et al.</em> (2011)</a>, and <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Sillanpää <em>et al.</em> (2012)</a>, we fitted our multiple phenotype measurements within each individual to a simple linear curve dependent on time <em>t<sub>ik</sub></em>:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-2.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>for individuals <em>i</em> = 1,…,<em>n</em>, and repeated measurements <em>k</em> = 1,…,<em>m<sub>i</sub></em> (<em>m<sub>i</sub></em> = 9 for an individual with a complete set of measurements). The least square estimates of the intercept <em>μ<sub>i0</sub></em> and the slope <em>μ<sub>i1</sub></em> were considered to be new <em>latent</em> traits aimed at evaluating the overall trait mean across annual rings and the trait rate of change, respectively. According to <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Heuven and Janss (2010)</a>, the intercepts and slopes are less correlated compared with the repeated measurements in the original scale and should have a constant variance, which would reduce the necessity to account for residual dependencies in the model. To make <em>μ<sub>i0</sub></em> correspond to the biologically meaningful mean across annual rings, the time variable <em>t<sub>ik</sub></em> was recoded and centralized to range from −4 to 4 instead of using the original calendar years (1995–2003). Individuals with more than five missing measurements were excluded from the analysis. Examples of the wood property development by time for a subset of traits are presented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2365/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trajectories of four wood traits by time including (A) wood density, (B) earlywood percentage, (C) radial fiberwidth, and (D) fiberwall thickness. For each trait, individual trajectories are shown in light blue lines, and the mean trajectory is shown in a black line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-670383820" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Trajectories of four wood traits by time including (A) wood density, (B) earlywood percentage, (C) radial fiberwidth, and (D) fiberwall thickness. For each trait, individual trajectories are shown in light blue lines, and the mean trajectory is shown in a black line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2365/F1.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/12/2365/F1.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2365/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2365/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465448" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Trajectories of four wood traits by time including (A) wood density, (B) earlywood percentage, (C) radial fiberwidth, and (D) fiberwall thickness. For each trait, individual trajectories are shown in light blue lines, and the mean trajectory is shown in a black line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Next, to describe the association between the latent traits and the molecular markers, the following multiple linear regression models were used to map intercept (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-3.gif"/></noscript></span></span>) and slope (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-4.gif"/></noscript></span></span>) traits separately as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-5.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <em>x<sub>ij</sub></em> is the genotype value of individual <em>i</em> and marker <em>j</em> (<em>j</em> = 1,…,<em>p</em>) coded as 0 and 1 for two possible genotypes, respectively, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-7.gif"/></noscript></span></span> are the intercept terms, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-8.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-9.gif"/></noscript></span></span> are the effects of marker <em>j</em>, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-11.gif"/></noscript></span></span> are the residuals for intercept and slope traits, respectively. For the three wood traits that only have two or three phenotype assessments per tree (GA, MFA, and MOE), we performed separate LASSO analyses for each assessment. Because the ordinary least squares (OLS) method usually cannot provide accurate estimates for a number of markers near to or larger than the number of individuals assessed (the <em>p</em> &gt; <em>n</em>-problem) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hastie <em>et al.</em> 2009</a>), we instead applied a popular penalized regression approach named LASSO as a marker (variable) selection method (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Tibshirani 1996</a>):<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-12.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where the penalty terms <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-14.gif"/></noscript></span></span>, (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-15.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-16.gif"/></noscript></span></span>&gt;0) reduce the number of markers included in the model by shrinking unimportant effects to zero. The tuning parameters <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-17.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-18.gif"/></noscript></span></span>, determine the degree of shrinkage and model sparsity. It was chosen explicitly by the 10-fold cross-validation (CV) model selection procedure (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Li and Sillanpää 2012</a>). The LASSO computation was performed by the MATLAB/R package “glmnet” (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Friedman <em>et al.</em> 2010</a>).</p></div><div id="sec-8" class="subsection"><h3>Single-step functional QTL detection by Bayesian linear mixed effect model</h3><p id="p-24">Instead of relying on a two-step multilevel model, we may alternatively estimate the temporal trend and effects of markers simultaneously in one linear mixed effect model (BLMM). By substituting equations (2) back to equation (1), we obtain<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-19.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>which is equivalent to the longitudinal random intercept and random slope model (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fahrmeir and Kneib 2011</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Furlotte <em>et al.</em> 2012</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Sikorska <em>et al.</em> 2012</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Wang 2012</a>). The fixed intercept and slope parameters <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-20.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-21.gif"/></noscript></span></span> describe the population time trends, and the random effect parameters <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-22.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-23.gif"/></noscript></span></span> describe the individual specific time trends. The individual specific random intercept and slope terms are important because they: describe the deviation of individual trends over the population trend; construct a certain temporal covariance structure among phenotypes; and take the possible heterogeneity in the data caused by missing (environmental) covariates into account (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Fahrmeir and Kneib 2011</a>). We formulated a full Bayesian procedure for model inference. The fixed intercept and slope parameters were assigned with noninformative priors: <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-24.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-25.gif"/></noscript></span></span>. The residual variance <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-26.gif"/></noscript></span></span> was assigned with a Jeffreys’ noninformative prior: <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-27.gif"/></noscript></span></span>. The random parameters were assigned with multivariate normal priors with common variance component <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-28.gif"/></noscript></span></span> over all the individuals: <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-29.gif"/></noscript></span></span>. An inverse-Wishart prior was specified for the covariance matrix: <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-30.gif"/></noscript></span></span> with fixed hyperparameters <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-31.gif"/></noscript></span></span> (identity matrix) and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-32.gif"/></noscript></span></span>. Under such prior settings of random intercept and slope parameters, the marginal variance of <em>y<sub>ik</sub></em> is <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-33.gif"/></noscript></span></span>, and covariance between <em>y<sub>ik</sub></em> and <em>y<sub>ih</sub></em> (<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-34.gif"/></noscript></span></span>) is <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-35.gif"/></noscript></span></span>, implying that the random intercept and slope terms describe serial correlation and heterogeneity of variance for the repeated phenotype measurements in the mixed model (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Weiss 2005</a>, p. 255).</p><p id="p-25">For marker effect parameters, we assigned mixture priors (or so-called spike and slab priors):<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-36.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>Such spike and slab priors play a similar role as the <em>l</em><sub>1</sub> penalty in LASSO to perform variable selection (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Habier <em>et al.</em> 2011</a>). The binary indicator variable <em>r<sub>j</sub></em> (<em>r<sub>j</sub></em> = 0, 1) determines whether marker <em>j</em> should be included in the model. A Bernoulli prior was assigned to <em>r<sub>j</sub></em>: <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-37.gif"/></noscript></span></span>. We fixed <em>w</em> to be 0.5, so that the estimates of <em>r<sub>j</sub></em> were mainly determined by the data. Furthermore, the variance parameter <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-38.gif"/></noscript></span></span> was assigned with inverse-gamma prior: <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-39.gif"/></noscript></span></span>, with fixed hyperparameters <em>a</em> = 0.1, and <em>b</em> = 0.1.</p><p id="p-26">For traits including MFA, MOE, and GA with two or three phenotype assessments per tree, we simplified equation (4) by including only tree-specific intercept terms but not tree-specific slope terms:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-40.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>In practice, a Markov Chain Monte Carlo (MCMC)-based method was used for estimating all the parameters defined in our Bayesian model. The detailed information of the MCMC sampling methods is available in the <a href="http://www.g3journal.org/content/suppl/2014/10/09/g3.114.014068.DC1/FileS5.pdf">File S5</a>.</p></div><div id="sec-9" class="subsection"><h3>Quantification of uncertainty and hypothesis testing</h3><p id="p-27">In this study, we also treated the question of how to best quantify QTL uncertainty and significance by hypothesis testing based on LASSO and BLMM, respectively. Empirical studies (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Xu 2007</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Li and Sillanpää 2012</a>) have shown that LASSO-CV tends to select many small effect markers in addition to the major QTL. The small marker effects may contribute to phenotype prediction but may not represent the foremost QTL candidates. Therefore, some post variable selection QTL inference is needed to judge which markers selected by LASSO can be declared as putative QTL with any confidence.</p><p id="p-28">Constructing a test statistic for a marker under the LASSO scheme is, however, not a straightforward task, because LASSO estimates, unlike the least squares method, do not asymptotically follow any standard parameter distribution (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chatterjee and Lahiri 2011</a>). The simplest method attempted was to first select a subset of markers by using LASSO on the whole data set, and then re-estimate the effects of those markers one-by-one with a simple regression model by the OLS method and perform the standard <em>t</em>-test (including the standard Bonferroni correction for multiple testing). This method was used as a baseline for comparisons (<em>p</em>-value is abbreviated as Single-p) and is similar to the tests of many single locus methods (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Furlotte <em>et al.</em> 2012</a>; <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Slate 2013</a>).</p><p id="p-29">Recently, several theoretically more solid approaches have been developed for indirectly quantifying the uncertainties of the markers selected by LASSO. We investigated the following three methods. The first is the <em>multiple-split test</em> (<em>p</em>-value is abbreviated as MST-p) of <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Meinshausen <em>et al.</em> (2009)</a>, which involves dividing the sample at random into two parts with equivalent number of individuals. The first half of the data are used to determine the optimal tuning parameter and to choose a subset of markers by LASSO, whereas the second half of the data are used to <em>t</em>-test those markers. For this study, the sample split procedure was repeated 100 times and the <em>p</em>-values (adjusted by Bonferroni correction) were combined to minimize the effect of random sampling of the data. The second is the <em>covariance test</em> (<em>p</em>-value is abbreviated as COV-p) proposed by <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Lockhart <em>et al.</em> (2014)</a>, which tests the significance of a marker entering the LASSO model and differs from the classical <em>t</em>-test of whether the marker effect equals zero. The COV-p test statistic is constructed directly based on the LASSO solution path, which is a compromise between shrinkage estimation and variable selection. Therefore, the COV-p for those selected markers should be more accurate than the <em>p</em>-values calculated by the <em>t</em>-tests. The third is the <em>stability selection</em> proposed by <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Meinshausen and Bühlmann (2010)</a>, involves drawing a subsample of half the number of individuals available and applying the LASSO on it to select a set of markers repeatedly (1000 times). Then, the stability selection probability (SSP) of each marker being selected was calculated and used to judge the support of QTL. <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Meinshausen and Bühlmann (2010)</a> suggested a decision rule based on SSPs to control the expected number of false positives. In practice, we calculated MST-p and SSP by our own Matlab codes [please see <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bühlmann <em>et al.</em> (2014)</a> for more information about these two methods]. The COV-p was calculated by the R package “covTest” (<a href="http://cran.r-project.org/web/packages/covTest/index.html">http://cran.r-project.org/web/packages/covTest/index.html</a>).</p><p id="p-30">For the BLMM model (4), we took the advantage of the sampling of indicator variables <em>r<sub>j</sub></em> defined in the prior (5). The empirical marginal posterior distribution of <em>r<sub>j</sub></em>: <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-41.gif"/></noscript></span></span>is often viewed as a posterior inclusion probability (PIP) (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Guan and Stephens 2011</a>) and has a similar interpretation as the previously mentioned SSP (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Meinshausen and Bühlmann 2010</a>). A main difference, however, is that the PIP is calculated from the MCMC samples based on the whole data without the need of repeated sub-sampling. However, the probability <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-42.gif"/></noscript></span></span> can be interpreted as an approximation of a local false discovery rate (LFDR) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Efron <em>et al.</em> 2001</a>; <a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Ventrucci <em>et al.</em> 2011</a>) for each individual marker <em>j</em> = 1,…,<em>p</em>. We can calculate a global level posterior false discovery rate (BFDR) (<a id="xref-ref-81-3" class="xref-bibr" href="#ref-81">Ventrucci <em>et al.</em> 2011</a>; <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Heuven and Janss 2010</a>) by combining LFDRs for a group of markers. A BFDR-based decision rule can be derived to judge significant markers to control BFDR under a certain threshold <em>α</em> (<em>α</em>=0.05) so that the multiplicity adjustments are achieved. Specifically, the LFDRs for each marker are sorted in ascending order. The average value of the LFDRs for the first <em>T</em> markers (<em>T</em> = 1,…,<em>p</em>) is defined as a BFDR for these markers. We find the highest possible value of BFDR (the average of the <em>T</em> smallest LFDRs), which is still smaller than the given threshold, and the corresponding markers are defined to be significant.</p><p id="p-31">The QTL uncertainty evaluation of the results of our particular analysis was performed at two levels. If a marker attained a value less than 0.2 for <em>any</em> of the test statistics Single-p, MST-p, COV-p, or BFDR, then it was declared as a <em>suggestive</em> QTL. For the suggestive level, an SSP inclusion ratio of at least 0.66 and 0.58 was required for AFLP and SNP+AFLP datasets, respectively, which, according to the formula of <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Bühlmann <em>et al.</em> (2014)</a>, guaranteed the expected number of false selected markers to be less than 2. However, for a marker to be declared a <em>significant</em> QTL, requirements were stricter. For the LASSO analyses, <em>at least two</em> of the test statistics were required to show <em>p</em>-values ≤ 0.05 (Single-p, MST-p or COV-p) or an SSP inclusion ratio of at least 0.83 and 0.66 for AFLP and SNP+AFLP datasets, respectively, which guaranteed the expected number of false selected markers to be less than 1. For the Bayesian analysis, a BFDR value less than 0.05 was required. Declarations of suggestive or significant QTL were made separately for each dataset and analysis method.</p><p id="p-32">We calculated the percentage of phenotypic variation explained by both suggestive and significant QTL for intercept and slope traits, respectively, as:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-43.gif"/></noscript></span>,</span>respectively. Note that for the BLMM methods, we have <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-44.gif"/></noscript></span></span> and</p><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-45.gif"/></noscript></span>.</span></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><p id="p-33">From what could be observed from general trends of the studied wood material, the percentage ratio of earlywood (EP) decreased by 3.0% per year and latewood increased by 2.1% per year (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). This trend in combination with the mean latewood density (LWD, 755 kg/m<sup>3</sup>) being much higher than that of earlywood density (EWD, 328 kg/m<sup>3</sup>) likely caused the overall ring wood density (WD) to increase by age by approximately 14.5 kg/m<sup>3</sup>·yr, although EWD and LWD did not show such pronounced trends <em>per se</em>. In a fashion similar to wood density, the fiber cell wall thickness (FTh) also increased (0.09 µm/yr) by increasing ring number from pith. The mean grain angle under bark (GA) at the tree age of 17–18 yr was 0.82°, implying a mild left-handed spiral grain.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465329/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465329/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Time-adjusted population means, average trends by increasing tree age, and individual (phenotypic) standard deviations for means and trends of each trait</span><div class="sb-div caption-clear"></div></div></div><div id="sec-11" class="subsection"><h3>QTL detection for intercepts (means)</h3><p id="p-36">In total, 24 suggestive (minimum one <em>p</em>-value &lt; 0.2) and five significant (minimum two <em>p</em>-values <em>&lt;</em> 0.05) QTL were detected across different intercept traits (including single time point traits), datasets, and QTL mapping methods (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The fractions of phenotypic intercept trait variation that was explained by QTL (<em>H<sup>2</sup><sub>QTL</sub></em>) ranged from zero to 0.15 (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Notably, several appreciably strong QTL were observed for the intercept of WD and EWD (three suggestive and one significant QTL) and for FWr and EFWr (three suggestive and two significant QTL) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). These QTL explained 0.07–0.15 fractions of the phenotypic variance for the method/dataset combinations where they were observed. For EWD, both BLMM and mLASSO suggested AFLP GGG191 (unmapped) to be significant (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>, part A) and suggestive, respectively, in the A-set. Likewise, the SNP 0_11919_01-122 at the maternal LG 14 was indicated to have a significant association with FWr and to be suggestive for EFWr according to the mLASSO analysis performed on the S+A-set (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). The AFLP AGG142 (unmapped) was, however, only significantly associated with EFWr. The same QTL markers (suggestive or significant) were shared between EWD and WD in two instances and between FWr and EFWr in one instance. Because the QTL effect signs of whole ring and earlywood components were the same, these QTL should contribute to a positive genetic correlation between the whole ring and earlywood components for WD and FWr.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2365/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trait intercept marker effects (&#x3B2;) for whole ring, earlywood and latewood density, whole ring radial, earlywood radial, latewood radial, whole ring tangential fiberwidths, whole ring, earlywood and latewood fiberwall thickness (WD, EWD, LWD, FWr, EFWr, LFWr, FWt, FTh, EFTh, LFTh) selected by the multilevel LASSO model in A datasets (black) and S+A datasets (gray) and of markers showing LFDR " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-670383820" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Trait intercept marker effects (&lt;em&gt;&#x3B2;&lt;/em&gt;) for whole ring, earlywood and latewood density, whole ring radial, earlywood radial, latewood radial, whole ring tangential fiberwidths, whole ring, earlywood and latewood fiberwall thickness (WD, EWD, LWD, FWr, EFWr, LFWr, FWt, FTh, EFTh, LFTh) selected by the multilevel LASSO model in A datasets (black) and S+A datasets (gray) and of markers showing LFDR &lt;0.5 for the Bayesian linear mixed effect model in the A datasets (red) and S+A datasets (magenta) are plotted against their estimated locations on maternal (&lt;em&gt;m&lt;/em&gt;) and paternal (&lt;em&gt;p&lt;/em&gt;) linkage groups (LG); 1 Morgan is approximately the length of LG 1m. Markers in section &lt;em&gt;u&lt;/em&gt; were not mappable to any LG. Significant and suggestive QTL are shown as diamonds and squares, respectively, whereas all other selected markers are shown as circles. All markers in considerable linkage (recombination frequency &lt;0.3) with a significant or suggestive QTL are highlighted within a rectangle.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2365/F2.medium.gif" width="330" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/12/2365/F2.medium.gif" width="330" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2365/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2365/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465361" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-37" class="first-child">Trait intercept marker effects (<em>β</em>) for whole ring, earlywood and latewood density, whole ring radial, earlywood radial, latewood radial, whole ring tangential fiberwidths, whole ring, earlywood and latewood fiberwall thickness (WD, EWD, LWD, FWr, EFWr, LFWr, FWt, FTh, EFTh, LFTh) selected by the multilevel LASSO model in A datasets (black) and S+A datasets (gray) and of markers showing LFDR &lt;0.5 for the Bayesian linear mixed effect model in the A datasets (red) and S+A datasets (magenta) are plotted against their estimated locations on maternal (<em>m</em>) and paternal (<em>p</em>) linkage groups (LG); 1 Morgan is approximately the length of LG 1m. Markers in section <em>u</em> were not mappable to any LG. Significant and suggestive QTL are shown as diamonds and squares, respectively, whereas all other selected markers are shown as circles. All markers in considerable linkage (recombination frequency &lt;0.3) with a significant or suggestive QTL are highlighted within a rectangle.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2365/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker effects (&#x3B2;) for wood grain angle (GA), microfibril angle (MFA), and dynamic modulus of elasticity (MOE), selected by the multilevel LASSO model in A datasets (black) and S+A datasets (gray) and of markers showing LFDR " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-670383820" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Marker effects (&lt;em&gt;&#x3B2;&lt;/em&gt;) for wood grain angle (GA), microfibril angle (MFA), and dynamic modulus of elasticity (MOE), selected by the multilevel LASSO model in A datasets (black) and S+A datasets (gray) and of markers showing LFDR &lt;0.5 for the Bayesian linear mixed effect model in the A datasets (red) and S+A datasets (magenta) are plotted against their estimated locations on maternal (&lt;em&gt;m&lt;/em&gt;) and paternal (&lt;em&gt;p&lt;/em&gt;) linkage groups (LG). 1 Morgan is approximately the length of LG 1m. Markers in section &lt;em&gt;u&lt;/em&gt; were not mappable to any LG. Significant and suggestive QTL are shown as diamonds and squares, respectively, whereas all other selected markers are shown as circles. All markers in considerable linkage (recombination frequency &lt;0.3) with a significant or suggestive QTL are framed in a rectangle. The mLASSO results are illustrated for one time point only (year 2006 for GA and the period 1998&#x2013;2000 for MFA and MOE).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2365/F3.medium.gif" width="440" height="297"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/12/2365/F3.medium.gif" width="440" height="297"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2365/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2365/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465348" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Marker effects (<em>β</em>) for wood grain angle (GA), microfibril angle (MFA), and dynamic modulus of elasticity (MOE), selected by the multilevel LASSO model in A datasets (black) and S+A datasets (gray) and of markers showing LFDR &lt;0.5 for the Bayesian linear mixed effect model in the A datasets (red) and S+A datasets (magenta) are plotted against their estimated locations on maternal (<em>m</em>) and paternal (<em>p</em>) linkage groups (LG). 1 Morgan is approximately the length of LG 1m. Markers in section <em>u</em> were not mappable to any LG. Significant and suggestive QTL are shown as diamonds and squares, respectively, whereas all other selected markers are shown as circles. All markers in considerable linkage (recombination frequency &lt;0.3) with a significant or suggestive QTL are framed in a rectangle. The mLASSO results are illustrated for one time point only (year 2006 for GA and the period 1998–2000 for MFA and MOE).</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465319/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465319/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">The ratios of phenotypic variance</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465373/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465373/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Description of significant QTL</span><div class="sb-div caption-clear"></div></div></div><p id="p-50">For all other traits assessed with functional mapping (RW, EP, LP, LWD, LFWr, FWt, FTh, EFTh, and LFTh) QTL were fewer (in total eight suggestive) (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>) and were influencing their respective traits to a relatively limited degree (<em>H<sup>2</sup><sub>QTL</sub></em> in the ranges 0.00–0.02) (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). However, all three suggestive QTL markers for fiberwall thickness traits were also suggestive of or significant for QTL (<em>e.g.</em>, GGG191, unmapped) for the corresponding wood density traits (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). The respective QTL effects for FTh and WD had the same sign, thus suggesting a positive genetic relationship. No QTL were observed for the intercepts of annual ring width (RW), earlywood (EP), and latewood percentage (LP) (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-51">For the traits subjected to single-time-point LASSO analyses (GA, MFA, and MOE), the results of only one time point is shown (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) because the QTL patterns observed for the other time points were similar. Among these traits, grain angle (GA) exhibited the highest number of QTL (three suggestive and two significant), explaining up to 0.08 fraction of the phenotypic variation (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). The SNP Axs_47_502 (LG 3m) and the AFLP TCG51 (unmapped) were observed to be significant QTL for GA in both the assessed years. The latter QTL exhibited the strongest overall significance values (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, part A) being the only QTL significant with respect to the extremely conservative multiple-split testing method (MST-p = 0.0014 in the year 2006). This is also the reason why MST-p values were not included in <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>. In contrast, microfibril angle (MFA) and modulus of elasticity (MOE) only exhibited two suggestive QTL each (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), which were not consistently observed across time points (not shown) and explained, at most, a 0.02 fraction of the phenotypic variation.</p></div><div id="sec-12" class="subsection"><h3>QTL detection for slopes</h3><p id="p-52">For trait slopes, 11 suggestive and four significant QTL were detected (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Interestingly, a substantial portion of the phenotypic variation for earlywood percentage (up to 0.22) (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>) was explained by the single QTL-AFLP GCG64 (unmapped) (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>, part B) significant in the mLASSO analysis and exhibiting an effect size of 13% in comparison with the trait mean (transformed scale). For the EWD slope, one significant (AFLP TGG57, unmapped) and four suggestive QTL jointly accounted for up to 0.51 fraction of the phenotypic variation.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2365/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trait slope marker effects (&#x3B3;) for earlywood, latewood percentage ratio, whole ring, earlywood and latewood density, whole ring radial, earlywood radial, whole ring tangential fiberwidths, whole ring, and earlywood fiberwall thickness (EP, LP, WD, EWD, LWD, FWr, EFWr, FWt, FTh, EFTh) selected by the multilevel LASSO model in A datasets (black) and S+A datasets (gray) and of markers showing LFDR " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-670383820" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Trait slope marker effects (&lt;em&gt;&#x3B3;&lt;/em&gt;) for earlywood, latewood percentage ratio, whole ring, earlywood and latewood density, whole ring radial, earlywood radial, whole ring tangential fiberwidths, whole ring, and earlywood fiberwall thickness (EP, LP, WD, EWD, LWD, FWr, EFWr, FWt, FTh, EFTh) selected by the multilevel LASSO model in A datasets (black) and S+A datasets (gray) and of markers showing LFDR &lt;0.5 for the Bayesian linear mixed effect model in the A datasets (red) and S+A datasets (magenta) are plotted against their estimated locations on maternal (&lt;em&gt;m&lt;/em&gt;) and paternal (&lt;em&gt;p&lt;/em&gt;) linkage groups (LG). 1 Morgan is approximately the length of LG 1m. Markers in section &lt;em&gt;u&lt;/em&gt; were not mappable to any LG. Significant and suggestive QTL are shown as diamonds and squares, respectively, whereas all other selected markers are shown as circles. All markers in considerable linkage (recombination frequency &lt;0.3) with a significant or suggestive QTL are framed in a rectangle.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2365/F4.medium.gif" width="314" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/12/2365/F4.medium.gif" width="314" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2365/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2365/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465383" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-53" class="first-child">Trait slope marker effects (<em>γ</em>) for earlywood, latewood percentage ratio, whole ring, earlywood and latewood density, whole ring radial, earlywood radial, whole ring tangential fiberwidths, whole ring, and earlywood fiberwall thickness (EP, LP, WD, EWD, LWD, FWr, EFWr, FWt, FTh, EFTh) selected by the multilevel LASSO model in A datasets (black) and S+A datasets (gray) and of markers showing LFDR &lt;0.5 for the Bayesian linear mixed effect model in the A datasets (red) and S+A datasets (magenta) are plotted against their estimated locations on maternal (<em>m</em>) and paternal (<em>p</em>) linkage groups (LG). 1 Morgan is approximately the length of LG 1m. Markers in section <em>u</em> were not mappable to any LG. Significant and suggestive QTL are shown as diamonds and squares, respectively, whereas all other selected markers are shown as circles. All markers in considerable linkage (recombination frequency &lt;0.3) with a significant or suggestive QTL are framed in a rectangle.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">For the slopes of LWD and FWr, one significant and one significant plus one suggestive QTL, respectively, were detected (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), but the <em>H<sup>2</sup><sub>QTL</sub></em> estimates for these associations were 0.02 at most (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). Among those, the SNP 2_10306_01-354 (LG 1p) significantly associated with LWD nonetheless exhibited a large effect of 3.2 kg/m<sup>3</sup>·yr (49% in comparison to the mean), likely as a result of the considerable phenotypic variation for the LWD slope [see <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2365/embed/mml-math-52.gif"/></noscript></span></span>in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>]. For all the other traits (LP, WD, EFWr, LFWr, FTh, EFTh, and LFTh), QTL findings for slopes were scarce (six suggestive QTL in total) (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>) and weak (<em>H<sup>2</sup><sub>QTL</sub></em> ≤ 0.03) (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>).</p></div><div id="sec-13" class="subsection"><h3>Comparing QTL mapping methods and datasets</h3><p id="p-55">For the statistically high-powered AFLP dataset, BLMM and mLASSO methods both appeared to detect approximately equal numbers of suggestive or significant QTL per trait (in the range of 0–5) (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). The same set of A-set markers were also frequently detected as significant QTL according to both modeling methods or were at least observed as significant for one method and suggestive for the other (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>). However, given that one or more QTL were detected in the A-set, the estimated proportions of explained phenotypic variances of the BLMM method (<a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>) were usually higher (0.01–0.15 and 0.22-0.51 for intercept and slope traits, respectively) than the corresponding mLASSO <em>H<sup>2</sup><sub>QTL</sub></em> estimates (≤0.03). Moreover, the BLMM effect estimates of the A-set were often higher than the corresponding mLASSO estimates <em>per se</em> (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). Such inconsistencies can be attributed to the different shrinkage penalties or priors involved in the two methods. LASSO assumes the shrinkage factor <em>λ</em> to be equal across all markers, whereas the Bayesian approach allows each marker to have its own individual indicator variable. Therefore, the BLMM can estimate marker effects in a more adaptive manner, <em>i.e.</em>, by shrinking less if the unaltered effect of a marker is already large.</p><p id="p-56">In contrast to the observations made on the A-set, BLMM did not even detect a single suggestive QTL in the statistically low-powered SNP+AFLP dataset, whereas the mLASSO method detected similar numbers of suggestive or significant QTL in both the S+A-sets and A-sets (<a id="xref-table-wrap-3-9" class="xref-table" href="#T3">Table 3</a>). To further dissect potential causes of this discrepancy, we conducted additional analyses combining the Bayesian spike and slab approach and BFDR testing with the two-step method. Interestingly, this two-step Bayesian spike and slab model was, similar to mLASSO, able to detect a number of QTL even in the S+A-set (not shown), implying that the previously observed discrepancies reflect differences in the modeling procedure (one-step <em>vs.</em> two-step) rather than statistical approach (LASSO <em>vs.</em> Bayesian). The Bayesian spike and slab model and LASSO should thus have similar power for detecting QTL given that the same model procedure is pursued.</p></div><div id="sec-14" class="subsection"><h3>Comparison between decision-making/hypothesis testing methods</h3><p id="p-57">To have an approximate comparison between the testing methods, we simply screened all the traits and counted the total number of significant or suggestive QTL for the large A- and small S+A datasets, respectively (for LASSO analyses of single time-point traits, only the counts for GA measured in 2006 and MFA and MOE observed in 1998–2000 were included). For the A-set, the ranking of testing methods based on the greatest total number of detected significant and suggestive QTL fell in the order BFDR &gt; Single-p &gt; SSP &gt; COV-p &gt; MST-p (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). For the S+A-set with fewer numbers of individuals, the ranking order was COV-p &gt; Single-p &gt; SSP &gt; MST-p &gt; BFDR. The results for the BFDR of the extra two-step Bayesian analysis were quite similar to that of the Single-p and SSP tests of the mLASSO analysis (not shown).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465347/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465347/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Counts of significant or suggestive QTL summed over all the traits for AFLP data and SNP+AFLP data</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-58">To the knowledge of the authors, this study presents the first functional/longitudinal QTL analysis of a conifer wood properties dataset with repeated phenotype measurements made possible by use of efficient measurement methods. Such analyses, so far, have only been performed for growth trajectories (<a id="xref-ref-74-4" class="xref-bibr" href="#ref-74">Sillanpää <em>et al.</em> 2012</a>). Compared with earlier QTL studies, functional mapping analyses utilize all the longitudinal data for a trait simultaneously and may better account for temporal trends and correlation structures across years. It can thus detect QTL that are stable over time (<em>i.e.</em>, the QTL associated with intercept traits) with greater statistical evidence, but it can also identify QTL that interact with time (<em>e.g.</em>, the QTL associated with slope traits) (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Wu and Lin 2006</a>). Summary statistics on the wood properties of the studied Scots pine full-sib family (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) showed levels and trends well in line with expectations of young Scots pines (compare with <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Fries 2012</a>) in the cambial age range of approximately 5–13 yr. The tree age of this material (17 yr) is also considered to be appropriate for assessment and selection in breeding (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Rosvall 2011</a>), making the results of this study also relevant from a breeding point of view. Given the cambial age range of the increment cores, our wood trait slope parameters should illustrate the rate of the biological transition from juvenile to mature wood by increasing cambial age. Previous research on wood properties has indicated wood trait radial trends of mature trees to be best-fitted by exponential or logistic models (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Burdon <em>et al.</em> 2004</a>). However, the nine investigated annual rings only partly cover the juvenile-to-mature wood transition phase, which in Scots pine may extend over a time period as long as 20 yr (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hannrup <em>et al.</em> 2000</a>). Moreover, wood properties are also known to be strongly influenced by annual environmental fluctuations such as climate (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Feng <em>et al.</em> 2012</a>), which could have caused the deviations from linearity observed in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> as readily as increasing cambial age. Therefore, a simple linear function was chosen for the functional mapping of this study despite the minor nonlinearities observed for the trait trajectories.</p><p id="p-59">The functional mapping of this study moreover investigated two advanced but closely related multi-locus model approaches: a multilevel LASSO model (mLASSO) and a newly developed Bayesian linear mixed model (BLMM). The use of multi-locus models with shrinkage of QTL marker effects (LASSO and BLMM) could act as a safeguard against the systematic effect overestimation associated with single-locus models paired with conservative multiple significance testing (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Beavis 1994</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Slate 2013</a>). Compared with the approach of <a id="xref-ref-74-5" class="xref-bibr" href="#ref-74">Sillanpää <em>et al.</em> (2012)</a>, our new BLMM approach can be more efficient because of the prior conjugacy and its fast implementation in Matlab. The BLMM is also closely connected to many other Bayesian functional mapping approaches such as those of <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Yang and Xu (2007)</a> and <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Li and Sillanpää (2013)</a>. However, those studies modeled the marker effects as nonlinear curves over time, whereas the marker effects of our study simply constitutes parameters that describe a linear trend <em>per se</em>. Another difference is that they assumed the effects of the same marker on slope and intercept to be correlated in the priors, whereas we assume them to be independent. The principle of the two-step mLASSO of this study is similar to QTL mapping approaches introduced by <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Gee <em>et al.</em> (2003)</a>, <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Heuven and Janss (2010)</a>, and <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Hurtado <em>et al.</em> (2011)</a>, even though they used effect estimation methods other than LASSO.</p><div id="sec-16" class="subsection"><h3>QTL observed for the intercept of important wood traits</h3><p id="p-60">Results from the wood intercept trait analyses, aimed at detecting QTL stable over a specified period of time, largely indicated zero to five QTL per trait explaining a small fraction of the phenotypic variation and with limited effects on the traits. This observation agrees with many published conifer QTL and association mapping studies (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Brown <em>et al.</em> 2003</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Pot <em>et al.</em> 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">González-Martínez <em>et al.</em> 2007</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Ukrainetz <em>et al.</em> 2008</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dillon <em>et al.</em> 2010</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beaulieu <em>et al.</em> 2011</a>) and, given that only major QTL usually are detected, it suggests wood intercept traits to be largely polygenic. In our study, EWD and FWr exhibited a greater number of appreciably strong QTL stable over time in comparison with the latewood proportion of those traits or to other wood traits (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-table-wrap-3-10" class="xref-table" href="#T3">Table 3</a>). However, the three QTL markers significant for EWD, FWr, and EFWr exhibited individual effects in a range of 0.8%–2.3% in relation to the overall population mean of the respective traits. Consequently, selection assisted by markers developed from these QTL would likely not achieve any dramatic improvements if used in isolation. The observations of numerous and stable EWD QTL agree with previous results where separate annual ring QTL analyses were performed on the wood of a full-sib family (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Brown <em>et al.</em> 2003</a>) and a three-generation pedigree (<a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Sewell <em>et al.</em> 2000</a>) of loblolly pine. Also, <a id="xref-ref-79-3" class="xref-bibr" href="#ref-79">Ukrainetz <em>et al.</em> (2008)</a> detected several QTL for both EWD and LWD in several separate annual ring QTL analyses on eight full-sib families of Douglas fir [<em>Pseudotsuga menziesii</em> (Mirb) Franco var <em>menziesii</em>].</p><p id="p-61">The observed tendency of WD and FWr to exhibit sets of QTL markers similar to those of EWD and EFWr, respectively, indicates genetic regulation of the earlywood component of the annual ring to be highly influential on properties of the annual ring as a whole. This is not very surprising because earlywood constituted the major component in most annual rings studied (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Similarly, the observation of three QTL markers suggestively or significantly associated with both fiberwall thickness and wood density traits is consistent with the strong phenotypic and genetic correlations repeatedly observed between wood density and the thickness of the tracheid cell walls (for examples of Scots pine, see <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Fries 2012</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Hong <em>et al.</em> 2014</a>). The apparent universal applicability of these correlations therefore suggests that the QTL colocalizations we observed for these trait pairs (WD/EWD, FWr/EFWr, and WD/FTh) more likely stem from pleiotropy than from close linkage.</p><p id="p-62">Also, for spiral grain angle, which is a trait closely connected to the twist propensity of sawed timber during drying (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dinwoodie 2000</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Warensjö and Rune 2004</a>), a number of influential QTL (two significant and three suggestive) were observed (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). To the knowledge of the authors, no QTL for grain angle have been published for any conifer species. In contrast to the estimated marker effects of the intercept traits, effects of the significant GA QTL (0.30°–0.52°) (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>, part A) indicate that an appreciable reduction of GA could be achieved by marker-assisted selection in the material of this study and potentially improve the shape stability of small sawed timber (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hallingbäck <em>et al.</em> 2010</a>). However, it must still be cautioned that the GA measurements were taken directly beneath the bark at breast height during two consecutive growth seasons (cambial age range of 12–14 yr) and the detected QTL thus may not be stable over a wider range of cambial ages. For other traits observed at few time points such as MFA and MOE, only a limited set of suggestive QTL were observed.</p></div><div id="sec-17" class="subsection"><h3>QTL observed for the slopes of important wood traits</h3><p id="p-63">The slope of wood traits over cambial ages or the rate of juvenile-to-mature wood transition has, to our knowledge, never been included in any published QTL or association mapping analyses, even though one association mapping study of radiata pine (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dillon <em>et al.</em> 2010</a>) considered the transition in a different manner. They regarded the juvenile-to-mature wood transition as a discrete phase change whose timing could be predicted using latewood density as an indicator trait (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gapare <em>et al.</em> 2006</a>), and we have rather dissected the dynamics of the transition process itself. In the results of this study, it was notable that large portions of the phenotypic variation for the slopes of earlywood percentage and earlywood density were explained by a limited number of suggestive and significant QTL (<a id="xref-table-wrap-3-11" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>) indicating the existence of major effect loci. Also, the effect of the QTL marker significantly associated with the slope of LWD (SNP 2_10306_01-354) (<a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a>, part B) was notably large despite the low corresponding <em>H<sup>2</sup><sub>QTL</sub></em> estimates. The results for EP and LWD are particularly intriguing because these traits are well-known to exhibit decreasing and increasing trends, respectively (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>, <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>) as a result of the wood maturation process (<a id="xref-ref-93-2" class="xref-bibr" href="#ref-93">Zobel and Sprague 1998</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Burdon <em>et al.</em> 2004</a>). Taking the significant EP QTL (GCG64) (<a id="xref-table-wrap-4-8" class="xref-table" href="#T4">Table 4</a>, part B) as an example and assuming conditions similar to those of this study, trees with the <em>band present</em> genotype (heterozygote) are predicted to decrease their EP from 56.3% at cambial age 9 to 42.2% at age 14 (5 yr later), whereas trees with the <em>band absent</em> genotype (homozygote) would decrease their EP even faster (from 56.3% to 40.3%) during the same time period. Admittedly, this calculation example may be oversimplistic and one should remember that QTL observed in a single full-sib family are usually not generalizable to other families, populations, or environments. However, the results of this study nonetheless illustrate potential to regulate the speed of certain aspects of wood maturation by marker-assisted selection.</p></div><div id="sec-18" class="subsection"><h3>Comparison between QTL mapping methods and decision rules</h3><p id="p-64">As shown in <em>Materials and Methods</em>, the mLASSO model and BLMM model are in principle closely connected approaches. The mLASSO model is a two-step approach because it estimates the temporal trends and marker effects on those trends separately, whereas the BLMM is a one-step approach estimating the temporal trends and marker coefficients simultaneously. Furthermore, in mLASSO, the intercept and slope latent parameters are treated independently. However, in BLMM, intercepts and slopes are jointly analyzed in a bivariate model, so the dependency between them may be more properly taken care of (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Piepho <em>et al.</em> 2012</a>). In general, it is believed that BLMM should be a more precise approach for analyzing this type of longitudinal data, and that the mLASSO model can sometimes be used as an approximation to BLMM (<a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Sikorska <em>et al.</em> 2012</a>). In our work, mLASSO and BLMM detected similar sets of QTL under conditions of higher statistical power (the A-set with data for at least 250 individuals), whereas under conditions of low statistical power (S+A-set with data for less than 100 individuals regardless of the trait) only the mLASSO was able to detect any QTL (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-4-9" class="xref-table" href="#T4">Table 4</a>). Supported by results of additional two-step Bayesian analyses, it is possible that the one-step BLMM, due to the simultaneous estimation of a higher number of parameters than the mLASSO, provided estimates with larger variances and lost power to detect QTL when sample sizes were insufficient. However, it also cannot be excluded that the small sample size of the S+A-set combined with the approximative behavior of the two-step methods (like mLASSO) could have caused a greater-than-expected number of falsely detected QTL. In any case, the inconsistency between BLMM and mLASSO at low statistical power suggests that QTL observed in the S+A dataset should be interpreted cautiously.</p><p id="p-65">Another methodological objective of this study was to evaluate the performance of several hypothesis-testing or uncertainty assessment methods for QTL identification. Regarding the mLASSO framework, the Single-p is very similar to many single locus hypothesis testing methods commonly used, whereas MST-p, COV-p, and SSP are more specifically designed for multi-locus LASSO methods. One common problem regarding these tests is their inconsistency with penalized estimates of marker effects obtained at a specific value of tuning parameter <em>λ</em> in LASSO. The development of more reliable LASSO test statistics is still an ongoing research topic (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Lockhart <em>et al.</em> 2014</a>). Under the BLMM framework, it is possible to derive a BFDR-based decision rule by relying on a single MCMC simulation without any extra effort such as re-sampling. Given the results of this study, one can conclude that MST-p was much more conservative than all other methods. This was expected because MST-p relies on a stricter assumption than the others (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Bühlmann <em>et al.</em> 2014</a>). In future research, simulation studies are needed to perform more intensive comparisons between the BLMM and the mLASSO approaches and between their associated significance testing methods.</p></div><div id="sec-19" class="subsection"><h3>Potential links between QTL and candidate genes</h3><p id="p-66">Among the nine significant QTL observed in this study, four were linked to SNP markers (<a id="xref-table-wrap-4-10" class="xref-table" href="#T4">Table 4</a>), which would enable speculations about the potential function of the candidate genes from which the SNPs were developed. One should, however, note that none of these QTL SNPs were mapped in the proximity to any of the AFLP markers, which could have verified the QTL in the statistically high-powered A-set. Moreover, because conifer genomes are very large, the genetic linkage within our full-sib family likely extends over numerous genes that, due to the limited statistical power of the S+A-set, all could be considered as alternative loci for the QTL. Consequently, the interpretations made here need to be taken with caution.</p><p id="p-67">The significant SNP-QTL for the intercept of FWr (0_11919_01-122) is situated in a gene coding for a F-box protein, an enzyme link to the pathway of lignin precursor biosynthesis (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersson Gunnerås 2005</a>). This suggests that a putative F-box association to fiber width may be mediated through the control of lignin biosynthesis, an integral part of the secondary cell walls of plants. The SNP-QTL (Axs_47_502) significantly associated with GA is an UDP-D apiose/UDP-D-xylose synthase (AXS) that catalyzes the conversion of UDP-D-glucuronicacid to UDP-D-apiose and UDP-D-xylose. D-xylose is the second most common component of hemicellulose abundant in the cell wall, particularly in the S2 layer, which heavily influences the properties of the fiber cells (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Rowell 2005</a>). By theoretically investigating mechanical aspects of fiber cell division and maturation, <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Schulgasser and Witztum (2007)</a> suggested the S2 microfibril angle (<em>i.e.</em>, MFA) to be involved in the development of grain angle. An association between GA and AXS observed in this study is thus conceivable.</p><p id="p-68">Among the significant QTL for trait slopes, the SNP-QTL 2_10306_01-354 and 0_18350_01-393 were associated with LWD and FWr, respectively. The SNP associated with LWD is located in a gene coding the NINJA protein, which acts as a transcriptional repressor and whose activity is mediated by a functional TPL-binding EAR repression motif. Both NINJA and TPL proteins function as negative regulators of jasmonate responses and are therefore involved in the regulation of gene expression of proteins related to stress and growth (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Wasternack and Hause 2013</a>). The significant association of NINJA protein with WD in this study could be understood considering that wood density is a consequence of cell wall growth. The SNP associated with FWr is located in a gene coding a lipase class 3 family protein. Lipases are involved in the accumulation and storage of lipid triglycerides and steryl esters into an organelle in the cell called the lipid body (LB). Among other functions, LBs are involved in providing building blocks for the enlargement of cell membranes (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">van der Schoot <em>et al.</em> 2011</a>), such as the plasma membrane of the cell walls that form the wood. These speculations on the nature of the significant SNPs control of certain traits will certainly require further research advances in the molecular biology of wood development to be confirmed. However, we do not doubt the value of our findings as candidate targets for such studies.</p><p id="p-69">Previous studies of QTL and association analysis for wood properties resulted in the identification of multiple candidate genes, some of which have been confirmed across several studies in conifers (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Brown <em>et al.</em> 2003</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">González-Martínez <em>et al.</em> 2007</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Dillon <em>et al.</em> 2010</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Beaulieu <em>et al.</em> 2011</a>). The majority of these previously assayed candidate genes were included in our array, but we found no significant trait associations for them. The only exception is the finding made by <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kumar <em>et al.</em> (2009)</a> that a gene coding for a lipid transfer protein was differentially expressed between Scots pine juvenile and mature wood. This is consistent with the SNP located in the lipid transfer protein coding gene and associated with the slope of FWr in this study. The authors argued about a possible gene role on cell wall deposition but admitted that this interpretation was vague and required further research for its validation.</p></div><div id="sec-20" class="subsection"><h3>Conclusions and future research</h3><p id="p-70">With respect to the genetic dissection of wood properties, the functional QTL mapping approach appeared promising and produced interesting results. Significant QTL were observed for the wood trait intercepts of EWD, FWr, and EFWr that appeared fairly stable within the studied cambial age range (approximately 5–13 yr). Two highly significant QTL were also detected for GA, which has never been included in wood property QTL studies. Furthermore, significant QTL were detected for the wood trait slopes of EP, EWD, LWD, and FWr, indicating marker associations with the rate of wood maturation. Four of the significant QTL were observed using SNP markers developed from candidate genes, thus making them interesting targets for further linkage or association mapping efforts or for molecular biology studies of the wood development in conifers.</p><p id="p-71">In addition to the genetic implications of this study, two functional mapping methods and several methods for quantifying certainty or significance of QTL were compared. The one-step BLMM and two-step mLASSO methods detected similar sets of QTL given that a large number of individuals were studied and the adaptive shrinkage method of BLMM appeared to allow reasonably large effect estimates in case QTL were significant. When the number of individual studies was small, however, the BLMM method did not detect any QTL indicating greater requirements than mLASSO in terms of the number of assessed individuals. Future research in this regard could use simulated data to further elucidate the differences between the two functional mapping approaches.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-72">We are grateful to Ross Whetten for guidance in the development and genotyping of the AFLP markers, to Outi Savolainen for the SNP data, and also to Skogforsk for arranging the establishment of the full-sib trial used in the study. Furthermore, we acknowledge Sven-Olof Lundqvist, Åke Hansson, and Lars Olsson at Innventia for advice on coring, sample preparation, and measurements with SilviScan, and for the evaluations of measurement data to provide trait data for annual rings, earlywood, and latewood ready for the genetic investigations. This study was supported by the Research School in Forest Genetics and Breeding at the Swedish University of Agricultural Sciences in Umeå, the Finnish Population Genetics Doctoral Program, the Doctoral Program in Mathematics and Statistics in University of Helsinki, and the EVOLTREE EU project.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-18"><p id="p-73">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.014068/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.014068/-/DC1</a></p></li><li class="fn" id="fn-19"><p id="p-74"><em>Communicating editor: R. W. Doerge</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-08-21"><span class="received-label">Received </span>August 21, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-10-03"><span class="accepted-label">Accepted </span>October 3, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Li <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-thesis" id="cit-4.12.2365.1"><div class="cit-metadata unstructured">Andersson Gunnerås, S., 2005 <em>Wood formation and transcript analysis with focus on tension wood and ethylene biology</em>. PhD dissertation, 2005:15, Swedish University of Agricultural Sciences, Umeå, Sweden.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.2" data-doi="10.1534/genetics.110.125781"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerksen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clément</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deslauriers</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association genetics of wood physical traits in the conifer white spruce and relationships with gene expression.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.125781%26rft_id%253Dinfo%253Apmid%252F21385726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzEvMTk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.12.2365.3"><div class="cit-metadata unstructured">Beavis, W. D., 1994 The power and deceit of QTL experiments: lessons from comparative QTL-studies, pp 250–266 in <em>Proceedings of the 49<sup>th</sup> Annual Corn and Sorghum Industry Research Conference</em>. American Seed Trade Association, Washington DC..</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.4" data-doi="10.1007/s11295-008-0165-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raffin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozenberg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meredieu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">What are the consequences of growth selection on wood density in the French maritime pine breeding programme?</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D5%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-008-0165-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-008-0165-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000261265100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.5" data-doi="10.1046/j.1365-3040.2002.00872.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brendel</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pot</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozenberg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guehl</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic parameters and QTL analysis of δ<sup>13</sup>C and ring width in maritime pine.</span> <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BEnviron.%26rft.volume%253D25%26rft.spage%253D945%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-3040.2002.00872.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-3040.2002.00872.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000176924000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassoni</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fontana</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Identification of quantitative trait loci influencing wood property traits in loblolly pine (<em>Pinus taeda</em> L.). III. QTL verification and candidate gene mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1537</span>–<span class="cit-lpage">1546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBrown%26rft.auinit1%253DG.%2BR.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1537%26rft.epage%253D1546%26rft.atitle%253DIdentification%2Bof%2BQuantitative%2BTrait%2BLoci%2BInfluencing%2BWood%2BProperty%2BTraits%2Bin%2BLoblolly%2BPine%2B%2528Pinus%2Btaeda%2BL.%2529.%2BIII.%2BQTL%2BVerification%2Band%2BCandidate%2BGene%2BMapping%26rft_id%253Dinfo%253Apmid%252F12930758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1MzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burdon</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kibblewhite</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">J. C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megraw</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Juvenile <em>vs.</em> mature wood: A new concept, orthogonal to corewood <em>vs.</em> outerwood, with special reference to <em>Pinus radiata</em> and <em>P. taeda</em>.</span> <abbr class="cit-jnl-abbrev">For. Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFor.%2BSci.%26rft.volume%253D50%26rft.spage%253D399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000224197800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bühlmann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalisch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meier</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-dimensional statistics with a view towards applications in biology.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Stat. Appl.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BStat.%2BAppl.%26rft.volume%253D1%26rft.spage%253D255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.9" data-doi="10.1198/jasa.2011.tm10159"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chatterjee</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lahiri</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bootstrapping LASSO estimators.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">608</span>–<span class="cit-lpage">625</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D106%26rft.spage%253D608%26rft_id%253Dinfo%253Adoi%252F10.1198%252Fjasa.2011.tm10159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/jasa.2011.tm10159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.10" data-doi="10.1534/genetics.110.116582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dillon</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H. X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Allelic variation in cell wall candidate genes affecting solid wood properties in natural populations and land races of <em>Pinus radiata</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1477</span>–<span class="cit-lpage">1487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116582%26rft_id%253Dinfo%253Apmid%252F20498299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS80LzE0NzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2365.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dinwoodie</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">Timber: Its Nature and Behaviour</span>, <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">E &amp; FN Spon</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Isolation of plant DNA from fresh tissue.</span> <abbr class="cit-jnl-abbrev">Focus</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFocus%26rft.volume%253D12%26rft.spage%253D13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.13" data-doi="10.1198/016214501753382129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Efron</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tusher</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Empirical Bayes analysis of a microarray experiment.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D96%26rft.spage%253D1151%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214501753382129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214501753382129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000172728000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.14" data-doi="10.1007/s001220050539"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emebiri</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devey</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matheson</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slee</span>  <span class="cit-name-given-names">M. U.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Linkage of RAPD markers to NESTUR, a stem growth index in radiata pine seedings.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D95%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050539&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.15" data-doi="10.1515/hfsg.1994.48.2.168"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Rapid measurement of the transverse dimensions of tracheids in radial wood sections from <em>Pinus radiata</em>.</span> <abbr class="cit-jnl-abbrev">Holzforschung</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">168</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHolzforschung%26rft.volume%253D48%26rft.spage%253D168%26rft_id%253Dinfo%253Adoi%252F10.1515%252Fhfsg.1994.48.2.168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1515/hfsg.1994.48.2.168&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994NF57700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2365.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Wood stiffness by X-ray diffractometry</span>, pp. <span class="cit-fpage">138</span>–<span class="cit-lpage">146</span> in <span class="cit-source">Characterization of the Cellulosic Cell Wall</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Stokke</span>  <span class="cit-name-given-names">D. D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Groom</span>  <span class="cit-name-given-names">H. L.</span></span>. <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2365.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fahrmeir</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kneib</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">Bayesian Smoothing and Regression for Longitudinal, Spatial and Event History Data</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.18" data-doi="10.1007/s13595-012-0187-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Climatic response of ring width and maximum latewood density of <em>Larix sibirica</em> in the Altay Mountains, reveals recent warming trends.</span> <abbr class="cit-jnl-abbrev">Ann. For. Sci.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BFor.%2BSci.%26rft.volume%253D69%26rft.spage%253D723%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13595-012-0187-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13595-012-0187-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.19" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regularization paths for generalized linear models via coordinate descent.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DFriedman%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DRegularization%2BPaths%2Bfor%2BGeneralized%2BLinear%2BModels%2Bvia%2BCoordinate%2BDescent.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft_id%253Dinfo%253Apmid%252F20808728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808728&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275203200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.20" data-doi="10.1007/s13595-012-0202-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fries</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic parameters, genetic gain and correlated responses in growth, fibre dimensions and wood density in a Scots pine breeding population.</span> <abbr class="cit-jnl-abbrev">Ann. For. Sci.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BFor.%2BSci.%26rft.volume%253D69%26rft.spage%253D783%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13595-012-0202-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13595-012-0202-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.21" data-doi="10.1002/gepi.21640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Furlotte</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyheramendy</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association mapping with longitudinal data.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DFurlotte%26rft.auinit1%253DN.%2BA.%26rft.volume%253D36%26rft.issue%253D5%26rft.spage%253D463%26rft.epage%253D471%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Bwith%2Blongitudinal%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21640%26rft_id%253Dinfo%253Apmid%252F22581622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21640&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22581622&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.22" data-doi="10.1051/forest:2006070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gapare</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abarquez</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic control of the time of transition from juvenile to mature wood in <em>Pinus radiata</em> D.</span> <abbr class="cit-jnl-abbrev">Don. Ann. For. Sci.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">871</span>–<span class="cit-lpage">878</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDon.%2BAnn.%2BFor.%2BSci.%26rft.volume%253D63%26rft.spage%253D871%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fforest%253A2006070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/forest:2006070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.23" data-doi="10.1051/forest/2009044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gapare</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baltunis</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivkovíc</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H. X.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic correlations among juvenile wood quality and growth traits and implications for selection strategy in <em>Pinus radiata</em> D.</span> <abbr class="cit-jnl-abbrev">Don. Ann. For. Sci.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDon.%2BAnn.%2BFor.%2BSci.%26rft.volume%253D66%26rft.spage%253D606%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fforest%252F2009044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/forest/2009044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.24" data-doi="10.1051/forest:2008054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaspar</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louzada</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguiar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic correlations between wood quality traits of <em>Pinus pinaster</em> Ait.</span> <abbr class="cit-jnl-abbrev">Ann. For. Sci.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BFor.%2BSci.%26rft.volume%253D65%26rft.spage%253D703%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fforest%253A2008054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/forest:2008054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.25" data-doi="10.1186/1471-2156-4-S1-S21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gee</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauderman</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Segregation and linkage analysis for longitudinal measurements of a quantitative trait.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">S21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D4%26rft.spage%253DS21%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-4-S1-S21%26rft_id%253Dinfo%253Apmid%252F14975089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-4-S1-S21&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14975089&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.26" data-doi="10.1534/genetics.106.061127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Martínez</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Association genetics in <em>Pinus taeda</em> L. I. Wood property traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061127%26rft_id%253Dinfo%253Apmid%252F17110498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzEvMzk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sederoff</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic linkage maps of <em>Eucalyptus grandis</em> and <em>Eucalyptus urophylla</em> using a pseudo-testcross: mapping strategy and RAPD markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1121</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGrattapaglia%26rft.auinit1%253DD.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1121%26rft.epage%253D1137%26rft.atitle%253DGenetic%2Blinkage%2Bmaps%2Bof%2BEucalyptus%2Bgrandis%2Band%2BEucalyptus%2Burophylla%2Busing%2Ba%2Bpseudo-testcross%253A%2Bmapping%2Bstrategy%2Band%2BRAPD%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F7982566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNy80LzExMjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertolucci</span>  <span class="cit-name-given-names">F. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sederoff</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genetic mapping of QTLs controlling vegetative propagation in <em>Eucalyptus grandis</em> and <em>E. urophylla</em> using a pseudo-testcross strategy and RAPD markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">933</span>–<span class="cit-lpage">947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D90%26rft.spage%253D933%26rft_id%253Dinfo%253Apmid%252F24173047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24173047&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Worral</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Wood quality studies I. A scanning microphometer for automatically measuring and recording certain wood characteristics.</span> <abbr class="cit-jnl-abbrev">Tappi.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTappi.%26rft.volume%253D47%26rft.spage%253D419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.30" data-doi="10.1214/11-AOAS455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bayesian variable selection regression for genome-wide association studies, and other large-scale problems.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Stat.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1780</span>–<span class="cit-lpage">1815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BStat.%26rft.volume%253D5%26rft.spage%253D1780%26rft_id%253Dinfo%253Adoi%252F10.1214%252F11-AOAS455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/11-AOAS455&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.31" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the Bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallingbäck</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hannrup</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fries</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Which annual rings to assess grain angles in breeding of Scots pine for improved shape stability of sawn timber?</span> <abbr class="cit-jnl-abbrev">Silva Fenn.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilva%2BFenn.%26rft.volume%253D44%26rft.spage%253D275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.33" data-doi="10.1080/028275800750014966"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hannrup</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ekberg</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Persson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetic correlations among wood, growth capacity and stem traits in <em>Pinus sylvestris</em>.</span> <abbr class="cit-jnl-abbrev">Scand. J. For. Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScand.%2BJ.%2BFor.%2BRes.%26rft.volume%253D15%26rft.spage%253D161%26rft_id%253Dinfo%253Adoi%252F10.1080%252F028275800750014966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/028275800750014966&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hannrup</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Säll</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genetic parameters for spiral grain in Scots pine and Norway spruce.</span> <abbr class="cit-jnl-abbrev">Silvae Genet.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilvae%2BGenet.%26rft.volume%253D52%26rft.spage%253D215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2365.35"><div class="cit-metadata unstructured">Hastie, T., R. Tibshirani, and J. Friedman, 2009 <em>Elements of Statistical Learning</em>, Ed. 2. Springer: New York..</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.12.2365.36"><div class="cit-metadata unstructured">Heuven, H. C. M., and L. L. G. Janss, 2010 Bayesian multi-QTL mapping for growth curve parameters. BMC Proceedings 4: S12.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.37" data-doi="10.1007/s13595-014-0359-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fries</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H. X.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High negative genetic correlations between growth traits and wood properties suggest incorporating multiple traits selection including economic weights for the future Scots pine breeding programs.</span> <abbr class="cit-jnl-abbrev">Ann. For. Sci.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BFor.%2BSci.%26rft.volume%253D71%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13595-014-0359-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13595-014-0359-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hurtado</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaban</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veteläinen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Virtanen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Dynamics of senescence-related QTLs in potato.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D183%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2365.39"><div class="cit-metadata unstructured">Hägglund, B., and J.-E. Lundmark, 1982 <em>Handledning i bonitering med skogshögskolans boniteringssystem</em>. National board of forestry, Jönköping, Sweden. ISBN 9185748641.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teasdale</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Multiple interval mapping for quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1203</span>–<span class="cit-lpage">1216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKao%26rft.auinit1%253DC.-H.%26rft.volume%253D152%26rft.issue%253D3%26rft.spage%253D1203%26rft.epage%253D1216%26rft.atitle%253DMultiple%2BInterval%2BMapping%2Bfor%2BQuantitative%2BTrait%2BLoci%26rft_id%253Dinfo%253Apmid%252F10388834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi8zLzEyMDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.41" data-doi="10.1007/s001220051108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaya</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sewell</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Identification of quantitative trait loci influencing annual height- and diameter-increment growth in loblolly pine (<em>Pinus taeda L</em>.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">586</span>–<span class="cit-lpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D98%26rft.spage%253D586%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.42" data-doi="10.1007/s001220050482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sewell</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Multiple marker mapping of quantitative trait loci in an outbred pedigree of loblolly pine.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">810</span>–<span class="cit-lpage">820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D94%26rft.spage%253D810%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XE63000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kujala</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sequence variation patterns along a latitudinal cline in Scots pine (<em>Pinus sylvestris</em>): signs of clinical adaptation?</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1455</span>–<span class="cit-lpage">1467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D8%26rft.spage%253D1455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.44" data-doi="10.1007/s001220051372"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spelman</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lausberg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Multiple-marker mapping of wood density loci in an outbred pedigree of radiata pine.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">926</span>–<span class="cit-lpage">933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D100%26rft.spage%253D926%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051372&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.45" data-doi="10.1007/s10681-008-9815-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saranpää</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnett</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Juvenile-mature wood transition in pine: correlation between wood properties and candidate gene expression profiles.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D166%26rft.spage%253D341%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-008-9815-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-008-9815-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263788600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.47" data-doi="10.1023/A:1026548716320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lerceteau</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szmidt</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">AFLP mapping and detection of quantitative trait loci (QTLs) for economically important traits in <em>Pinus sylvestris</em>: a preliminary study.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D6%26rft.spage%253D451%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1026548716320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1026548716320&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.48" data-doi="10.1023/A:1012076825293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lerceteau</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szmidt</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Detection of quantitative trait loci in <em>Pinus sylvestris</em> L. across years.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D121%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1012076825293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1012076825293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.49" data-doi="10.1007/s00122-012-1892-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Overview of LASSO-related penalized regression methods for quantitative trait mapping and genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DZ.%26rft.volume%253D125%26rft.issue%253D3%26rft.spage%253D419%26rft.epage%253D435%26rft.atitle%253DOverview%2Bof%2BLASSO-related%2Bpenalized%2Bregression%2Bmethods%2Bfor%2Bquantitative%2Btrait%2Bmapping%2Band%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1892-9%26rft_id%253Dinfo%253Apmid%252F22622521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1892-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22622521&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.50" data-doi="10.1534/genetics.113.152736"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A Bayesian nonparametric approach for mapping dynamic quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152736%26rft_id%253Dinfo%253Apmid%252F23770698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvOTk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.51" data-doi="10.1186/1471-2105-7-138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A joint model for nonparametric functional mapping of longitudinal trajectory and time-to-event.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLin%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.spage%253D138%26rft.epage%253D138%26rft.atitle%253DA%2Bjoint%2Bmodel%2Bfor%2Bnonparametric%2Bfunctional%2Bmapping%2Bof%2Blongitudinal%2Btrajectory%2Band%2Btime-to-event.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-138%26rft_id%253Dinfo%253Apmid%252F16539724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-138&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16539724&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.52" data-doi="10.1214/13-AOS1175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lockhart</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A significance test for the LASSO.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D42%26rft.spage%253D413%26rft_id%253Dinfo%253Adoi%252F10.1214%252F13-AOS1175%26rft_id%253Dinfo%253Apmid%252F25574062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/13-AOS1175&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25574062&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional mapping of quantitative trait loci underlying the character process: a theoretical framework.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">1751</span>–<span class="cit-lpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMa%26rft.auinit1%253DC.-X.%26rft.volume%253D161%26rft.issue%253D4%26rft.spage%253D1751%26rft.epage%253D1762%26rft.atitle%253DFunctional%2BMapping%2Bof%2BQuantitative%2BTrait%2BLoci%2BUnderlying%2Bthe%2BCharacter%2BProcess%253A%2BA%2BTheoretical%2BFramework%26rft_id%253Dinfo%253Apmid%252F12196415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MS80LzE3NTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.54" data-doi="10.1007/s00122-003-1484-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Littell</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A likelihood approach for mapping growth trajectories using dominant markers in a phase-unknown full-sib family.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">699</span>–<span class="cit-lpage">705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMa%26rft.auinit1%253DC.%2BX.%26rft.volume%253D108%26rft.issue%253D4%26rft.spage%253D699%26rft.epage%253D705%26rft.atitle%253DA%2Blikelihood%2Bapproach%2Bfor%2Bmapping%2Bgrowth%2Btrajectories%2Busing%2Bdominant%2Bmarkers%2Bin%2Ba%2Bphase-unknown%2Bfull-sib%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1484-9%26rft_id%253Dinfo%253Apmid%252F14586505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1484-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14586505&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Markussen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fladung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Achere</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Favre</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faivre-Rampant</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Identification of QTLs controlling growth, chemical and physical wood property traits in <em>Pinus pinaster</em> (Ait.).</span> <abbr class="cit-jnl-abbrev">Silvae Genet.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">8</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilvae%2BGenet.%26rft.volume%253D52%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.56" data-doi="10.1198/jasa.2009.tm08647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meinshausen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bühlmann</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">P-Values for high-dimensional regression.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">1671</span>–<span class="cit-lpage">1681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D104%26rft.spage%253D1671%26rft_id%253Dinfo%253Adoi%252F10.1198%252Fjasa.2009.tm08647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/jasa.2009.tm08647&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000273995500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.57" data-doi="10.1111/j.1467-9868.2010.00740.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meinshausen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bühlmann</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Stability Selection. J. R.</span> <abbr class="cit-jnl-abbrev">Stat. Soc. Series B.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSoc.%2BSeries%2BB.%26rft.volume%253D72%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2010.00740.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2010.00740.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mork</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1928</span> <span class="cit-article-title">Die qualität des Fichtenholzes unter befonderer Rückfichtnahme auf Schleif—und Papierholz.</span> <abbr class="cit-jnl-abbrev">Der Papier-Fabrikant.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDer%2BPapier-Fabrikant.%26rft.volume%253D26%26rft.spage%253D741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2365.59"><div class="cit-metadata unstructured">Mullin, T., B. Andersson, J.-C. Bastien, J. Beaulieu, R. D. Burdon <em>et al.</em>, 2011 Chapter 2: Economic Importance, Breeding Objectives and Achievements, pp. 40–127 in <em>Genetics</em>, <em>Genomics and Breeding of Conifers</em>, edited by C. Plomion, J. Bousquet, and C. Kole. Science Puslishers, Inc., Enfield, NH, Edenbridge Ltd, UK.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.12.2365.60"><div class="cit-metadata unstructured">Olsson, L., 2000 Measurement of Latewood—Comparison of different methods and definitions, <em>Third Workshop of COST Action E10 Wood Properties for Industrial Use</em>. Helsinki, Finland, June 19–21,2000.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.61" data-doi="10.1093/molbev/msn194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palmé</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Patterns of divergence among Conifer ESTs and polymorphism in <em>Pinus sylvestris</em> identify putative selective sweeps.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2567</span>–<span class="cit-lpage">2577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn194%26rft_id%253Dinfo%253Apmid%252F18775901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMi8yNTY3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.62" data-doi="10.1002/bimj.201100219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moehring</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A stage-wise approach for analysis of multi-environment trials.</span> <abbr class="cit-jnl-abbrev">Biom. J.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">844</span>–<span class="cit-lpage">860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiom%2BJ%26rft.aulast%253DPiepho%26rft.auinit1%253DH.%2BP.%26rft.volume%253D54%26rft.issue%253D6%26rft.spage%253D844%26rft.epage%253D860%26rft.atitle%253DA%2Bstage-wise%2Bapproach%2Bfor%2Bthe%2Banalysis%2Bof%2Bmulti-environment%2Btrials.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbimj.201100219%26rft_id%253Dinfo%253Apmid%252F23007738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bimj.201100219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23007738&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.63" data-doi="10.1007/BF00224085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durel</span>  <span class="cit-name-given-names">C.-E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Malley</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Genetic dissection of height in maritime pine seedlings raised under accelerated growth condition.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D93%26rft.spage%253D849%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00224085%26rft_id%253Dinfo%253Apmid%252F24162417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00224085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24162417&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.64" data-doi="10.1007/s11295-005-0026-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pot</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodrigues</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozenberg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chantre</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibbits</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">QTLs and candidate genes for wood properties in maritime pine (<em>Pinus pinaster</em> Ait.).</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D2%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-005-0026-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-005-0026-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000244896400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.65" data-doi="10.1534/genetics.107.077099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Gil</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knürr</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mikkonen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wachowiak</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Demographic history has influenced nucleotide diversity in European <em>Pinus sylvestris</em> populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1713</span>–<span class="cit-lpage">1724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPyhajarvi%26rft.auinit1%253DT.%26rft.volume%253D177%26rft.issue%253D3%26rft.spage%253D1713%26rft.epage%253D1724%26rft.atitle%253DDemographic%2BHistory%2BHas%2BInfluenced%2BNucleotide%2BDiversity%2Bin%2BEuropean%2BPinus%2Bsylvestris%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.077099%26rft_id%253Dinfo%253Apmid%252F18039881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzE3MTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.66" data-doi="10.1007/s11295-010-0340-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kujala</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Revisiting protein heterozygosity in plants-nucleotide diversity in allozyme coding genes of conifer <em>Pinus sylvestris</em>.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D7%26rft.spage%253D385%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-010-0340-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-010-0340-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2365.67"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed"><span class="cit-name-surname">Rosvall</span>  <span class="cit-name-given-names">O</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">Review of the Swedish Tree Breeding Programme</span>, <span class="cit-publ-name">Stiftelsen Skogsbrukets Forskninsinstitut</span>, <span class="cit-publ-loc">Uppsala, Sweden</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2365.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rowell</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Handbook of Wood Chemistry and Wood Composites</span>, <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton, Florida</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.69" data-doi="10.1007/s00226-006-0100-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulgasser</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witztum</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The mechanism of spiral grain formation in trees.</span> <abbr class="cit-jnl-abbrev">Wood Sci. Technol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWood%2BSci.%2BTechnol.%26rft.volume%253D41%26rft.spage%253D133%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00226-006-0100-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00226-006-0100-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.70" data-doi="10.1214/10-AOS792"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bayes and empirical-Bayes multiplicity adjustment in the variable-selection problem.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">2587</span>–<span class="cit-lpage">2619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D38%26rft.spage%253D2587%26rft_id%253Dinfo%253Adoi%252F10.1214%252F10-AOS792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/10-AOS792&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.71" data-doi="10.1038/ng.2314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjálmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seren</span>  <span class="cit-name-given-names">Ü.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An efficient multi-locus mixed-model approach for genome-wide association studies in structured populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">825</span>–<span class="cit-lpage">830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSegura%26rft.auinit1%253DV.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D825%26rft.epage%253D830%26rft.atitle%253DAn%2Befficient%2Bmulti-locus%2Bmixed-model%2Bapproach%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2314%26rft_id%253Dinfo%253Apmid%252F22706313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706313&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.72" data-doi="10.1007/s001220051607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sewell</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassoni</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megraw</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Identification of QTLs influencing wood property traits in loblolly pine (<em>Pinus taeda</em> L.). I. Physical wood properties.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">1273</span>–<span class="cit-lpage">1281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D101%26rft.spage%253D1273%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051607&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.73" data-doi="10.1007/s001220100697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sewell</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuskan</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elam</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Identification of QTLs influencing wood property traits in loblolly pine (<em>Pinus taeda</em> L.). II. Chemical wood properties.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">214</span>–<span class="cit-lpage">222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSewell%26rft.auinit1%253DM.%2BM.%26rft.volume%253D104%26rft.issue%253D2-3%26rft.spage%253D214%26rft.epage%253D222%26rft.atitle%253DIdentification%2Bof%2BQTLs%2Binfluencing%2Bwood%2Bproperty%2Btraits%2Bin%2Bloblolly%2Bpine%2B%2528%2BPinus%2Btaeda%2BL.%2529.%2BII.%2BChemical%2Bwood%2Bproperties.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220100697%26rft_id%253Dinfo%253Apmid%252F12582689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220100697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582689&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.74" data-doi="10.1038/hdy.2011.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pikkuhookana</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrahamsson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knürr</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fries</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Simultaneous estimation of multiple quantitative trait loci and growth curve parameters through hierarchical Bayesian modeling.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">134</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DSillanpaa%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D108%26rft.issue%253D2%26rft.spage%253D134%26rft.epage%253D146%26rft.atitle%253DSimultaneous%2Bestimation%2Bof%2Bmultiple%2Bquantitative%2Btrait%2Bloci%2Band%2Bgrowth%2Bcurve%2Bparameters%2Bthrough%2Bhierarchical%2BBayesian%2Bmodeling.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.56%26rft_id%253Dinfo%253Apmid%252F21792229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21792229&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sikorska</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivadeneira</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">P. J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uitterlinden</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast linear mixed model computations for genome-wide association studies with longitudinal data.</span> <abbr class="cit-jnl-abbrev">Stat. Med.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BMed.%26rft.volume%253D32%26rft.spage%253D165%26rft_id%253Dinfo%253Apmid%252F22911890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22911890&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">From Beavis to beak color: a simulated study to examine how much QTL mapping can reveal about the genetic architecture of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D1251%26rft_id%253Dinfo%253Apmid%252F23617906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23617906&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318234900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2365.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snedecor</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cochran</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-source">Statistical Methods</span>, <span class="cit-edition">Ed. 7</span>. <span class="cit-publ-name">The Iowa State University Press</span>, <span class="cit-publ-loc">Ames, Iowa</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Regression shrinkage and selection via the lasso.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D58%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.79" data-doi="10.1007/s11295-007-0097-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ukrainetz</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mansfield</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification of quantitative trait loci for wood quality and growth across eight full-sib coastal Douglas-fir families.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D4%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-007-0097-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-007-0097-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.80" data-doi="10.4161/psb.6.11.17639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van der Schoot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinne</span>  <span class="cit-name-given-names">P. L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Plant lipid bodies and cell-cell signaling: A new role for an old organelle?</span> <abbr class="cit-jnl-abbrev">Plant Signal. Behav.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1732</span>–<span class="cit-lpage">1738</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BSignal%2BBehav%26rft.aulast%253Dvan%2Bder%2BSchoot%26rft.auinit1%253DC.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D1732%26rft.epage%253D1738%26rft.atitle%253DPlant%2Blipid%2Bbodies%2Band%2Bcell-cell%2Bsignaling%253A%2Ba%2Bnew%2Brole%2Bfor%2Ban%2Bold%2Borganelle%253F%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fpsb.6.11.17639%26rft_id%253Dinfo%253Apmid%252F22057325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/psb.6.11.17639&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22057325&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.81" data-doi="10.1093/biostatistics/kxq040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ventrucci</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cocchi</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multiple testing on standardized mortality ratios: a Bayesian hierarchical model for FDR estimation.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxq040%26rft_id%253Dinfo%253Apmid%252F20577014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvc3RzIjtzOjU6InJlc2lkIjtzOjc6IjEyLzEvNTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.82" data-doi="10.1093/nar/23.21.4407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hogers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleeker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reijans</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Lee</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">AFLP - a new technique for DNA-fingerprinting.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4407</span>–<span class="cit-lpage">4414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DVos%26rft.auinit1%253DP.%26rft.volume%253D23%26rft.issue%253D21%26rft.spage%253D4407%26rft.epage%253D4414%26rft.atitle%253DAFLP%253A%2Ba%2Bnew%2Btechnique%2Bfor%2BDNA%2Bfingerprinting%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F23.21.4407%26rft_id%253Dinfo%253Apmid%252F7501463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8yMS80NDA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-thesis" id="cit-4.12.2365.83"><div class="cit-metadata unstructured">Wang, T., 2012 Linear mixed effects model for a longitudinal genome wide association study of lipid measures in type 1 diabetes. PhD thesis, McMaster University, Hamilton, Canada.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warensjö</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rune</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Effects of compression wood and grain angle on deformations of studs from 22-year old Scots pine trees.</span> <abbr class="cit-jnl-abbrev">Scand. J. For. Res.</abbr> <span class="cit-vol">19</span>(<span class="cit-supplement">Suppl 5</span>): <span class="cit-fpage">48</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScand.%2BJ.%2BFor.%2BRes.%26rft.volume%253D19%26rft.spage%253D48%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.85" data-doi="10.1093/aob/mct067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wasternack</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hause</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Jasmonates: biosynthesis, perception, signal transduction and action in plant stress response, growth and development. An update to the 2007 review.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1058</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmct067%26rft_id%253Dinfo%253Apmid%252F23558912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjEwOiIxMTEvNi8xMDIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2365.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Modeling Longitudinal Data</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.87" data-doi="10.1017/S0016672302005980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">M. C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santra</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Quantitative trait loci for growth in <em>Populus</em>.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWu%26rft.auinit1%253DR.%26rft.volume%253D81%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D64%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bfor%2Bgrowth%2Btrajectories%2Bin%2BPopulus.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672302005980%26rft_id%253Dinfo%253Apmid%252F12693683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672302005980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12693683&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182269400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-other" id="cit-4.12.2365.88"><div class="cit-metadata unstructured">Wu, R. L., and M. Lin, 2006 Functional mapping—A new tool to study the genetic architecture of dynamic complex traits. Nat. Rev. Genet. <strong>7:</strong> 229–237.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimating polygenic effects using markers of the entire genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D163%26rft.issue%253D2%26rft.spage%253D789%26rft.epage%253D801%26rft.atitle%253DEstimating%2BPolygenic%2BEffects%2BUsing%2BMarkers%2Bof%2Bthe%2BEntire%2BGenome%26rft_id%253Dinfo%253Apmid%252F12618414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzIvNzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.90" data-doi="10.1111/j.1541-0420.2006.00711.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">An empirical Bayes method for estimating epistatic effects of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D513%26rft.epage%253D521%26rft.atitle%253DAn%2Bempirical%2BBayes%2Bmethod%2Bfor%2Bestimating%2Bepistatic%2Beffects%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1541-0420.2006.00711.x%26rft_id%253Dinfo%253Apmid%252F17688503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1541-0420.2006.00711.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17688503&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246843900023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.91" data-doi="10.1534/genetics.106.064279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Bayesian shrinkage analysis of quantitative trait loci for dynamic traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1169</span>–<span class="cit-lpage">1185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.064279%26rft_id%253Dinfo%253Apmid%252F17435239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8yLzExNjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2365.92"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Precision mapping of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZeng%26rft.auinit1%253DZ.%2BB.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1457%26rft.epage%253D1468%26rft.atitle%253DPrecision%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F8013918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzE0NTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2365.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zobel</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sprague</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Juvenile Wood in Forest Trees</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/12/2365.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/12/2353" title="High-Resolution Genetic Mapping in the Diversity Outbred Mouse Population Identifies Apobec1 as a Candidate Gene for Atherosclerosis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/12/2381" title="Fast and Efficient Drosophila melanogaster Gene Knock-Ins Using MiMIC Transposons" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/12"><span>Volume 4 Issue 12, December 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/12.cover-source.jpg?itok=ezho2p8P" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2365" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fSmEGkAI0o2xRWT4AGwJNuE_052pvk5HFmujxKZtFr4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/12/2365" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/12/2365" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/12/2365" class="active" data-icon-position="" data-hide-link-title="0">Functional Multi-Locus QTL Mapping of Temporal Trends in Scots Pine Wood Traits</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459746" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-KGCT2_EfzyPC0zl66YCiWTzQKgVNpOaf7EMyiQTvbQY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/12/2365.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/12/2365" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-MWGEnJwbHtvWJSG6_C8l3ji7AxcYvdJIVkwdL6EFAag" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/12/2365" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/12/2365.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459746" id="node459746--2" data-pisa="ggg;4/12/2365" data-pisa-master="ggg;g3.114.014068" data-apath="/ggg/4/12/2365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Functional Multi-Locus QTL Mapping of Temporal Trends in Scots Pine Wood Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zitong Li</span>, <span class="highwire-citation-author" data-delta="1">Henrik R. Hallingbäck</span>, <span class="highwire-citation-author" data-delta="2">Sara Abrahamsson</span>, <span class="highwire-citation-author" data-delta="3">Anders Fries</span>, <span class="highwire-citation-author" data-delta="4">Bengt Andersson Gull</span>, <span class="highwire-citation-author" data-delta="5">Mikko J. Sillanpää</span> and <span class="highwire-citation-author" data-delta="6">M. Rosario García-Gil</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2365-2379;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014068 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459746--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zitong Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Mathematics and Statistics, University of Helsinki, Helsinki, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zitong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZitong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Henrik R. Hallingbäck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henrik%2BR.%2BHallingb%C3%A4ck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hallingb%C3%A4ck%20HR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenrik%2BR.%2BHallingb%25C3%25A4ck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sara Abrahamsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Skogforsk, Sävar, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BAbrahamsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abrahamsson%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BAbrahamsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anders Fries </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anders%2BFries%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fries%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnders%2BFries%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bengt Andersson Gull </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Skogforsk, Sävar, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bengt%2BAndersson%2BGull%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gull%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABengt%2BAndersson%2BGull%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mikko J. Sillanpää </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Mathematical Sciences and Department of Biology, University of Oulu, Oulu, Finland</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Biocenter Oulu, Oulu, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mikko%2BJ.%2BSillanp%C3%A4%C3%A4%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sillanp%C3%A4%C3%A4%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMikko%2BJ.%2BSillanp%25C3%25A4%25C3%25A4%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">M. Rosario García-Gil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BRosario%2BGarc%C3%ADa-Gil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garc%C3%ADa-Gil%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BRosario%2BGarc%25C3%25ADa-Gil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.rosario.garcia@slu.se" class="" data-icon-position="" data-hide-link-title="0">m.rosario.garcia@slu.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/12/2365" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/12/2365&amp;title=Functional%20Multi-Locus%20QTL%20Mapping%20of%20Temporal%20Trends%20in%20Scots%20Pine%20Wood%20Traits" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/12/2365&amp;title=Functional%20Multi-Locus%20QTL%20Mapping%20of%20Temporal%20Trends%20in%20Scots%20Pine%20Wood%20Traits" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/12/2365&amp;title=Functional%20Multi-Locus%20QTL%20Mapping%20of%20Temporal%20Trends%20in%20Scots%20Pine%20Wood%20Traits" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/12/2365&amp;text=Functional%20Multi-Locus%20QTL%20Mapping%20of%20Temporal%20Trends%20in%20Scots%20Pine%20Wood%20Traits" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/12/2365&amp;title=Functional%20Multi-Locus%20QTL%20Mapping%20of%20Temporal%20Trends%20in%20Scots%20Pine%20Wood%20Traits" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/12/2365&amp;t=Functional%20Multi-Locus%20QTL%20Mapping%20of%20Temporal%20Trends%20in%20Scots%20Pine%20Wood%20Traits" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/12/2365&amp;title=Functional%20Multi-Locus%20QTL%20Mapping%20of%20Temporal%20Trends%20in%20Scots%20Pine%20Wood%20Traits" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/12/2365&amp;title=Functional%20Multi-Locus%20QTL%20Mapping%20of%20Temporal%20Trends%20in%20Scots%20Pine%20Wood%20Traits" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459746/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459746" id="node459746--3" data-pisa="ggg;4/12/2365" data-pisa-master="ggg;g3.114.014068" data-apath="/ggg/4/12/2365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Functional Multi-Locus QTL Mapping of Temporal Trends in Scots Pine Wood Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zitong Li</span>, <span class="highwire-citation-author" data-delta="1">Henrik R. Hallingbäck</span>, <span class="highwire-citation-author" data-delta="2">Sara Abrahamsson</span>, <span class="highwire-citation-author" data-delta="3">Anders Fries</span>, <span class="highwire-citation-author" data-delta="4">Bengt Andersson Gull</span>, <span class="highwire-citation-author" data-delta="5">Mikko J. Sillanpää</span> and <span class="highwire-citation-author" data-delta="6">M. Rosario García-Gil</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2365-2379;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014068 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459746--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zitong Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Mathematics and Statistics, University of Helsinki, Helsinki, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zitong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZitong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Henrik R. Hallingbäck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henrik%2BR.%2BHallingb%C3%A4ck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hallingb%C3%A4ck%20HR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenrik%2BR.%2BHallingb%25C3%25A4ck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sara Abrahamsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Skogforsk, Sävar, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BAbrahamsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abrahamsson%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BAbrahamsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anders Fries </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anders%2BFries%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fries%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnders%2BFries%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bengt Andersson Gull </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Skogforsk, Sävar, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bengt%2BAndersson%2BGull%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gull%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABengt%2BAndersson%2BGull%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mikko J. Sillanpää </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Mathematical Sciences and Department of Biology, University of Oulu, Oulu, Finland</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Biocenter Oulu, Oulu, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mikko%2BJ.%2BSillanp%C3%A4%C3%A4%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sillanp%C3%A4%C3%A4%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMikko%2BJ.%2BSillanp%25C3%25A4%25C3%25A4%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">M. Rosario García-Gil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BRosario%2BGarc%C3%ADa-Gil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garc%C3%ADa-Gil%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BRosario%2BGarc%25C3%25ADa-Gil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.rosario.garcia@slu.se" class="" data-icon-position="" data-hide-link-title="0">m.rosario.garcia@slu.se</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2365&amp;rft.epage=2379&amp;rft.atitle=Functional%20Multi-Locus%20QTL%20Mapping%20of%20Temporal%20Trends%20in%20Scots%20Pine%20Wood%20Traits&amp;rft.volume=4&amp;rft.issue=12&amp;rft.date=2014-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Li%2C+Zitong&amp;rft.au=Hallingb%C3%A4ck%2C+Henrik+R.&amp;rft.au=Abrahamsson%2C+Sara&amp;rft.au=Fries%2C+Anders&amp;rft.au=Gull%2C+Bengt+Andersson&amp;rft.au=Sillanp%C3%A4%C3%A4%2C+Mikko+J.&amp;rft.au=Garc%C3%ADa-Gil%2C+M.+Rosario"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459746/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459746/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459746/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459746/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459746/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459746/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459746/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459746/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459746/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459746/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459746/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459746/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/12/2365" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/12/2365.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/12/2365.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/12/2365.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/12/2365.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
