<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/12/2545" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/12/2545.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/12/2545.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/12/2545.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/12/2545.atom" />
<meta name="HW.pisa" content="ggg;4/12/2545" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Correcting for Sequencing Error in Maximum Likelihood Phylogeny Inference" />
<meta name="DC.Identifier" content="10.1534/g3.114.014365" />
<meta name="DC.Date" content="2014-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2014 Kuhner and McGill. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Accurate phylogenies are critical to taxonomy as well as studies of speciation processes and other evolutionary patterns. Accurate branch lengths in phylogenies are critical for dating and rate measurements. Such accuracy may be jeopardized by unacknowledged sequencing error. We use simulated data to test a correction for DNA sequencing error in maximum likelihood phylogeny inference. Over a wide range of data polymorphism and true error rate, we found that correcting for sequencing error improves recovery of the branch lengths, even if the assumed error rate is up to twice the true error rate. Low error rates have little effect on recovery of the topology. When error is high, correction improves topological inference; however, when error is extremely high, using an assumed error rate greater than the true error rate leads to poor recovery of both topology and branch lengths. The error correction approach tested here was proposed in 2004 but has not been widely used, perhaps because researchers do not want to commit to an estimate of the error rate. This study shows that correction with an approximate error rate is generally preferable to ignoring the issue." />
<meta name="DC.Contributor" content="Mary K. Kuhner" />
<meta name="DC.Contributor" content="James McGill" />
<meta name="article:published_time" content="2014-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Correcting for Sequencing Error in Maximum Likelihood Phylogeny Inference" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Accurate phylogenies are critical to taxonomy as well as studies of speciation processes and other evolutionary patterns. Accurate branch lengths in phylogenies are critical for dating and rate measurements. Such accuracy may be jeopardized by unacknowledged sequencing error. We use simulated data to test a correction for DNA sequencing error in maximum likelihood phylogeny inference. Over a wide range of data polymorphism and true error rate, we found that correcting for sequencing error improves recovery of the branch lengths, even if the assumed error rate is up to twice the true error rate. Low error rates have little effect on recovery of the topology. When error is high, correction improves topological inference; however, when error is extremely high, using an assumed error rate greater than the true error rate leads to poor recovery of both topology and branch lengths. The error correction approach tested here was proposed in 2004 but has not been widely used, perhaps because researchers do not want to commit to an estimate of the error rate. This study shows that correction with an approximate error rate is generally preferable to ignoring the issue.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/12/01" />
<meta name="citation_mjid" content="ggg;4/12/2545" />
<meta name="citation_id" content="4/12/2545" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/12/2545" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/12/2545.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/12/2545.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/12/2545.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.014365" />
<meta name="citation_pmid" content="25378476" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2545" />
<meta name="citation_lastpage" content="2552" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Mary K. Kuhner" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065" />
<meta name="citation_author_email" content="mkkuhner@uw.com" />
<meta name="citation_author" content="James McGill" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Cornish-Bowden;citation_title=Nomenclature for incompletely specified bases in nucleic acid sequences: rcommendations 1984;citation_pages=3021-3030;citation_volume=13;citation_year=1985;citation_issue=9;citation_pmid=2582368;citation_doi=10.1093/nar/13.9.3021" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=K. de Queiroz;citation_author=J. Gauthier;citation_title=Phylogeny as a central principle in taxonomy: phylogenetic definitions of taxon names.;citation_volume=39;citation_year=1990;citation_doi=10.2307/2992353;citation_georef=1991009099" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=A. J. Drummond;citation_author=S. Y. W. Ho;citation_author=M. J. Phillips;citation_author=A. Rambaut;citation_title=Relaxed phylogenetics and dating with confidence.;citation_pages=e88-e88;citation_volume=4;citation_year=2006;citation_issue=5;citation_pmid=16683862;citation_doi=10.1371/journal.pbio.0040088" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Ewing;citation_author=P. Green;citation_title=Base-calling of automated sequencer traces using Phred. II. Error probabilities.;citation_volume=8;citation_year=1998;citation_pmid=9521922;citation_doi=10.1101/gr.8.3.186" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=J. Felsenstein;citation_title=Evolutionary trees from DNA sequences: a maximum likelihood approach.;citation_pages=368-376;citation_volume=17;citation_year=1981;citation_issue=6;citation_pmid=7288891;citation_doi=10.1007/BF01734359" />
<meta name="citation_reference" content="citation_title=Inferring Phylogenies;citation_author=J. Felsenstein;citation_year=2004" />
<meta name="citation_reference" content="citation_title=PHYLIP (Phylogeny Inference Package) version 3.6. Distributed by the author;citation_author=J. Felsenstein;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=N. Haiminen;citation_author=D. N. Kuhn;citation_author=L. Parida;citation_author=I. Rigoutsos;citation_title=Evaluation of methods for de novo genome assembly from high-throughput sequencing reads reveals dependencies that affect the quality of the results.;citation_pages=e24182;citation_volume=9;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=M. K. Kuhner;citation_title=Coalescent genealogy samplers: windows into population history.;citation_pages=86-93;citation_volume=24;citation_year=2009;citation_issue=2;citation_pmid=19101058;citation_doi=10.1016/j.tree.2008.09.007" />
<meta name="citation_reference" content="citation_journal_title=Sys Biol;citation_title=Practical performance of tree comparison metrics;citation_author=M. K. Kuhner;citation_author=J. Yamato;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. A. Machado;citation_author=N. Robbins;citation_author=M. T. P. Gilbert;citation_author=E. A. Herre;citation_title=Critical review of host specificity and its coevolutionary implications in the fig/fig wasp mutualism.;citation_volume=102;citation_year=2005;citation_pmid=15851680;citation_doi=10.1073/pnas.0501840102" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. R. McGill;citation_author=E. A. Walkup;citation_author=M. K. Kuhner;citation_title=Correcting coalescent analyses for panel-based SNP ascertainment.;citation_volume=193;citation_year=2013;citation_pmid=23335334;citation_doi=10.1534/genetics.112.146738" />
<meta name="citation_reference" content="citation_journal_title=Lecture Notes Mathematics;citation_author=D. F. Robinson;citation_author=L. R. Foulds;citation_title=Comparison of weighted labeled trees.;citation_pages=119-126;citation_volume=748;citation_year=1979;citation_doi=10.1007/BFb0102690" />
<meta name="citation_reference" content="citation_journal_title=Math. Biosci.;citation_author=D. F. Robinson;citation_author=L. R. Foulds;citation_title=Comparison of phylogenetic trees.;citation_pages=131-147;citation_volume=53;citation_year=1981;citation_doi=10.1016/0025-5564(81)90043-2" />
<meta name="citation_reference" content="citation_journal_title=Divers. Distrib.;citation_author=F. Rutschmann;citation_title=Molecular dating of phylogenetic trees: a brief review of current methods that estimate divergence times.;citation_pages=35-48;citation_volume=12;citation_year=2006;citation_doi=10.1111/j.1366-9516.2006.00210.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Correcting for Sequencing Error in Maximum Likelihood Phylogeny Inference" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
<meta name="twitter:description" content="Accurate phylogenies are critical to taxonomy as well as studies of speciation processes and other evolutionary patterns. Accurate branch lengths in phylogenies are critical for dating and rate measurements. Such accuracy may be jeopardized by unacknowledged sequencing error. We use simulated data to test a correction for DNA sequencing error in maximum likelihood phylogeny inference. Over a wide range of data polymorphism and true error rate, we found that correcting for sequencing error improves recovery of the branch lengths, even if the assumed error rate is up to twice the true error rate. Low error rates have little effect on recovery of the topology. When error is high, correction improves topological inference; however, when error is extremely high, using an assumed error rate greater than the true error rate leads to poor recovery of both topology and branch lengths. The error correction approach tested here was proposed in 2004 but has not been widely used, perhaps because researchers do not want to commit to an estimate of the error rate. This study shows that correction with an approximate error rate is generally preferable to ignoring the issue." />
<meta name="og-title" property="og:title" content="Correcting for Sequencing Error in Maximum Likelihood Phylogeny Inference" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/12/2545" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Accurate phylogenies are critical to taxonomy as well as studies of speciation processes and other evolutionary patterns. Accurate branch lengths in phylogenies are critical for dating and rate measurements. Such accuracy may be jeopardized by unacknowledged sequencing error. We use simulated data to test a correction for DNA sequencing error in maximum likelihood phylogeny inference. Over a wide range of data polymorphism and true error rate, we found that correcting for sequencing error improves recovery of the branch lengths, even if the assumed error rate is up to twice the true error rate. Low error rates have little effect on recovery of the topology. When error is high, correction improves topological inference; however, when error is extremely high, using an assumed error rate greater than the true error rate leads to poor recovery of both topology and branch lengths. The error correction approach tested here was proposed in 2004 but has not been widely used, perhaps because researchers do not want to commit to an estimate of the error rate. This study shows that correction with an approximate error rate is generally preferable to ignoring the issue." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
    <title>Correcting for Sequencing Error in Maximum Likelihood Phylogeny Inference | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"YQnNn4cGEUROCXwKkl7JPmhM7II0lkYbzdEyIK1rylo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459732","apath":"\/ggg\/4\/12\/2545.atom","pisa":"ggg;4\/12\/2545","ac":{"\/ggg\/4\/12\/2545.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/12\/2545.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/12\/2545"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/12\/2545"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/12\/2545":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wjPVcGh7\\\\\\\/LZnCMeoUID9O0Ctg3UGAIELua649Yvj5wT0THruxdELrK4gB49PtGu1QIiY4QEdsCcfQCQicDDZu5pATXRCtVuH6uMqPK4RxyNLCzPlyTl7WUKYlQNmLfGJrJAusDt64oKXREgnwKqNo9gdcOiotAY0Fpksg\\\\\\\/x50DmGvMvd19Bs7Sj0GALG8HgBl\\\\\\\/e2NTgZpWaKKF8bq5DEeYlsLC2WwUZC5xiNE0Q7a9bv\\\\\\\/59KP+xBDwCU8+sa\\\\\\\/hP\\\\\\\/bYy6C2BV6sS8\\\\\\\/1iZrTYxS0t0W+XN1nmgx0etQfrXPoAG2Fofm0DMP8CX8EIZMd37ECSOSo2howuzlEHe9NGFQzrPCTTZ\\\\\\\/9k8aG1MXhbrbeegLGyy2LHN2oO9\\\\\\\/gZWBGt2LahxuK9gUzdxOVwRlI1DGbU+XQ3mU3weid1y9DmTM1hiHSWsA9YXjMJBeF3q6g0zks0pJbw7JXSeDvpk0VgAJf49\\\\\\\/n2mH7\\\\\\\/Ww3Q6L1PY6dQB7O36R5lyOfZDESSMTqfa4t0q0Q6KUy1WazggEDMpwje8aI7oGDi0xDi8l173OJvuQj+ESU8lD9KaRIztSM56nrVaOjXObR60Fphb5cDzn5YTUbMOGhHjI9efpYjmxYXpf7WB0lbvBN9rfMHCUuYQ0qeh2uqoSqYywu+QMAVPKtEvcFCf9wvAH5hwL0c0hOqAjSvD78Ji4OHw\\\\\\\/God27h1y8n9izrDSj58jnA0YEd5PfFY3OXNKvp5YkxwlEO1RsKIWcpiWTDQshYEBN6phgLXOtIYpmhTFM9JrCqkuP2g+nSkJ6pRWRpNgw5KGA3EGq4QpQa8w\\\\\\\/ZyvkAjr+d+jV0CRXa76FFBsoQDwKO7RtgGL53Wk9X1SLr\\\\\\\/Oh7qtsiNAtLqiGBQQhrYmVPvF+pn7hWKGlSA09xL4vMpygFmMUSyjXOHftIwVBk6XEFkmi6dhCiFsimxVBGoLMP\\\\\\\/rXIHlOJ9\\\\\\\/Af6X3NGhqkMqSWlELDu1GFLgbb+O4yII3mTiKPCbFOxDRpX1Uzpp8v4z8Pe9GbJKpWSOPWE8rLzU\\\\\\\/rCh84UrvxBk6LPP7QaryWRekxe2VK16roNPRiTS8e8+X4dKYg1w7H1\\\\\\\/6BO0NEwrw11aGRTVT6Zx7qD+RKiRQXtF1\\\\\\\/WhxuEVpYJ\\\\\\\/Vuov1NZXJ\\\\\\\/EMClWz54eVjqYeK1EWQim6HAKjlOGbrH+e22XsxD4uo\\\\\\\/fp3uTYpHMmTmdweCbWtDVKXMoQHdgN2pZ8K3KrF3YeKTCjexuNuKQWDeX34GmmEJt9KByzE3bbb9P8FKzAEv2BhJq\\\u0022,\\\u0022iv\\\u0022:\\\u0022gStfiVAZpKp\\\\\\\/WhHloc2NSg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002253c4fcc2f79e9c5df8e1a2be55b8570bea7f03382a608acdf6e45097caca63a3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XKvm2eElCtY3CTsiw7wRg7k4ahq5aEwB2MWNf8ZApu5rJxomX9u9\\\\\\\/SVeWeH3k7Snjr4O8OGoMp3EMi4KK8txsTMQiGszLUJPf5AkkqrkPdkjeIbLZrmXUEmOIAUSuJFpIv+iGg\\\\\\\/RaTUIRfYhiTxr0Q5uNZyUKM0HrXYk5kg06tAihHVbXSbEVMZ7nRTFRI03G1txrldgBflal9FPjFKmLfk9lkpv47MBjPfB+\\\\\\\/IToKrl6iqmiD+6euuxbruioiYLaY1nLVWGRL2am7AHnVnbkRIeI2PXjNBZX97hWlhwd0hgmCt9VnF1RL3WWDiSVIfQL9VGifKnjUkJD9q2tc1ZqlTbDHjwZ+\\\\\\\/cND4o7EwRYjHgcArfCjgc3kkTZGKWnfgHbpG9LR7TjG98xGvYEnZzd7M1vofziwFgLp+UggHsCcq+NTDEn55nW0A4PsoNbwKRv5hC1xWwllLOw2SBw6vSEgPZuQLHl7WPTHVQRT91l0L6ztOVDtXhFDbr32aNssj+HSgLpDj6YxBjIi9wntiy8yd+9AwyDKdXDVTJxl17M7Nklqz62dmkbThVkNSt+ZMSRn6KCaqp\\\\\\\/YZgLMd56sVNAbk4ADbz2kEduNJr9GCLF5zSaFdJe7TVNEQEAr31L\\\\\\\/EmLlyYMGDoNTT5mSkbzpAx9RQeK6A1SbOaw6G38Jw+\\\\\\\/\\\\\\\/goeCQCpekCTZotxNrYtrZxJPruZoMBQoQcBnpfzcxFBzG3zlURmPzGTJXWpVbNIRuRTzy++MnQBS+QkopwvpNlM3VPoJjmgRQUscaDCO\\\\\\\/EK6y2tKe1jvWLcDEKPW0sdLIbWG73j3czGnDeZCSt2PaseNIwROdcJ4HqLeYV0AeLY2RlqjKBOCLFYYPIg6bW6TfVtPdl1mWQXeLIFTpk02DUET+sTD9mnanOfM5e3OUjAJNcphvXYIT90uXI7b1xjCw1BcVFLah3CdSW0LReg8n+wC5ZknfsGX27yWM2QygbRRhHTtoD6OBgQdRoQZ79Ei9JMKjjgJXPiAQzC8yRx4NmDw17F0bXSWTDE\\\\\\\/Y9cJ0IzmU\\\\\\\/OIfv5e3GvBwKFjLO5NHvTvCcZ03gP93ZKrvlSodO8q5gYccMRdvcDobUEGJpcLu79QzxBi1WL5tu26VWO+e5c1AnRGCC8Qf6Ct075rGr7F+1owfpQdV6QkHvYsDjssmtYqHpkcXEUafRsKxcnvUCqmE3z7xK\\\\\\\/ZVwlov+er1IFNB4sE\\\\\\\/yvfLqhk5nIaXxHL6L65WNqoLxNak5eJFuNHQ6AOck3l1YlvjbCLBcbr8zqPVMCuluUcVGzuK\\\\\\\/9U1iW8MGT\\\\\\\/5tGic4nzrdkxPdB6m0iSmdmIWAo2gP4T4MGZBp+msAZKfvuLHJ544dIf7x+Dmiyq92reT55Q\\\\\\\/PyolPPtNiWFWc0rW6VPYRyH5k7c96\\\\\\\/vW09It9YqfmvfFwSni5QbCt5LnS3Qbm7eWB38nPDBYnidIInh5LJ79TgmnYofhH23yjryHITvyySeYWOn4ji9JU7tNCVGfSXDp0YNl70E60PhK9RIbTHb4FzdvAvQSMG6St9XytyvsZHoap57wmhhTYiRnBcyTvUCzNs0UW69PkUPtfgG589x\\\\\\\/uN1rQh+AsoiC+3cOY1AwbiuAgBKX35FhvMp\\\\\\\/dW+gwjhB5j8ojX9L1mnWkFKDxnOgGlYUnTdDqe8gi2nTmXQeyXQ5VmkWH47QWX8lAleSOlZhn2Rcost9zj9Tp0VSqy7HmCbS1dN23YWLhCFGPbrze83p9xBmruck3mAz\\\\\\\/uV5xytaRZvJFmYCDBmPgRqvnRHlxLPdDhj\\\\\\\/3KYSRHRvrQusb\\\\\\\/o2gNBWf3rnSEvzU9ge2KF7xJ8ebg7IrbYpBtmMBo\\\\\\\/RVgXQTbjeAzPlH19V7HONfbd5imlHYGrL3VD+OKMxGXXv++TDECnZWwTBnAlozLBLpe0bhYHRYEL179byYT7ZyfZhA4iP1vKpUQM18QWCM1i9xhxisM3Tl0\\\\\\\/Kfn3OZnmcr083dC29ATGGmR1gUAtc2p6wPsk1EsCxXFMZmXggSx\\\\\\\/It2GN26HoCBRuQyA+RnJ96cSddKdmROWXLsxAIi9CDCIVMEaThCTUzYtEufippAGDzTWXrLiltZ0Gcx4bwL7bHPHCpCo0dJEcSxozJm4iS6nQUj3IVhLRCcvbkyFXkFDjRxWbgXbpdGS\\\\\\\/8X25N2dFg34iWlLBZgti0S2iwJGIRWOJwpOBRZrvwKjkNd64Uacbk5F7nXTTJejoe7prsAX\\\\\\\/5b4FD11EF67gzDwH4aD\\\\\\\/YsK4XhRxFWW4FDadkRV2nVXBvhoEI51qSh6+0tgYRGXRnNQw0sR68Cc\\\\\\\/dbweuUn+CQKdo7zkZTRg7el+WICAsgbspiZ6dm1yRVFze1FTiX+IacQMaaPOsjNsx6Hkl439ONIznjcT3J2gjO35199WJAVTYEsaNvU7sAYznGTTsLYvlRu1y\\\\\\\/qlEgvwqL8Syr2ed4aSy1Gq3BJIGd8tX6Z9j+Rf4c8UUGM+eIdEmjCYToJou6OcKNF1\\\\\\\/JCLTx4KPHa61k4A\\\\\\\/y+xG0SkKqWmEyal\\\\\\\/2spw3BZ5jzXiyqzi08k60Nm6bd2FP+hFV+M2COzy\\\\\\\/Gf8RQdWdyp7MMkhvz0dpHZynO7WHfXXXP2c3jV8NtzO+TPjWaqRLDI6+Umcvibff3czuKdaqq6UNoexhwGpVNucW1NLTifvvIvQtub\\\\\\\/KrZQRQUNZYs7stW4vGKo2FZrktiBgYF+\\\\\\\/XtjHK83Yc4rPOk7zKrqCSQ1\\\\\\\/gQO\\\\\\\/1ZhDx0IvotJ+3zJ7FsD+35Lc0YRtED7b4nfbFYFKkd8FVqYZaR72MOeHKYGtVtXArpqSnt5Vyt0xV9Ej0wxFl83FEPR8o7Z4+IZft2HQbII5RUJM+q2YMc0ijMXhDt+R4gAiVVDJlZwj3iG8gfqIZOzXwsr5EoSrGP98nExtJMvyJ+l7b50ck4bpT1hb40\\\\\\\/OjWwMlW4pa6rx2J4SsKHtHSmhs0BMGeQ\\\\\\\/oY2wsExvvsMJcLhhw1rRspfjjJB7nzt339sq4Ef3PO4FSx9lOIdhe1sK4d2m5g9bor4TvKOQPXu1l\\\\\\\/vtPq1Hacjdnn3fhAXisXagxj54PODoZwKvb++rkCTnjaJplos\\\\\\\/2WwBXjr4CkEdGMn\\\\\\\/gjuEPyZTG5Gb+Te1ARk4JZj8jfDhYrWVjaEDlsKZvaCH0uLrvzTTskdBt+7IC0PeV67SjtyqZg26kTCppfu09\\\\\\\/dCrOjrtycuOPY+X6X9XSCtOoiAz0Ob1CqCpBodbrtqwJVgFIuU9DW1clYjXRtIVvKEoS1vxHteBf0j7O1lxkpV818Z1CVyDmi9HGjOLVgNhdQ\\\\\\\/DF22vG00hAyiGTWDlndbi7S9dgM9Kyj4L76qdso3H0x5xbM3Cc6\\\\\\\/u2IrsiFI3btY9Z6VMnqQ1FONAIWtKsF\\\\\\\/NRW8KuZS8g68i3WYs3W0WHRUQTFgP8R7GWtZDRX7MjFsvXw6dXi17FvNNMQpOOOFG+bd3UTzXJH1NjaY\\\\\\\/ymUCRAhvjwhQXm9A4SwtrtOoIV+hLEv+2ihLWHpTWt\\\\\\\/Y2q5zSawE1nqZQmPEYGiYvbnjQAA4gB8wK9ao4Ane8IYTH+7FSdJXZN5DWj3oWmWr6+cis9B+1vzgTj\\\\\\\/F+VwwzbciD+pN0B\\\\\\\/\\\\\\\/ibK4IVCpwziKyxD5YEK3kMz1pmnTOMT+D821KHhPHmW8HuvT+HtYtU+Xdc+sXJRtCCymDwdHhnybJaqFKDKNI\\\\\\\/oX\\\\\\\/gGnmAQxGsQ6XdtLLq\\\\\\\/lMFrjTPxDz8NfmvkvAT6xFNlDHwih955l9TxW\\\\\\\/ZUSN4MgYc9+U3TCMGZeXNrBswX3KPfHhFOu0VqyZMAZggrfkJpdocD067Ygu9m4F9GJG5W\\\\\\\/KotlnZm+GHlvnjxLG1r4Ecpj3uXelUTD19bO43N6vRSihUNIWyYc3AkVZnQDMin\\\\\\\/tBq\\\\\\\/jARXAFdQKgK+dEauvxlU23hbu4ZBYOF8sBV7v6zNEmzj\\\\\\\/07smvzYyF\\\\\\\/JYJ0C2poZpnDXg00kRcITYDnDAubAL2D5JgGaC3ZLIKc4tDy3j7b7NdkButOEFvLOQH0hbrNQD4PK+wPHd0aaP+Up4F1oTVfJJzk82nwf9VrJZmduZSXr4u9GSZuFnrklL9KH0Qxh\\\\\\\/AGV8Sms7lBo7gVf5LpFjtCC+33M\\\\\\\/hDTZp\\\\\\\/mFK\\\\\\\/v7FjqentlZQRJa7Ea\\\\\\\/izDJl5NMFXTILvITf0\\\\\\\/wgQFhGEWIMQ2mzKZUvZqQOLIoAzwySfRpvY8YraxfUjxN31L6aC50QFoxFuXHTtZ75CdzH65rfkD7aOhPH4Tm1\\\\\\\/qepCRlDfheckivd3iXMZjxhD8BR7sGHMYZW03Anj6yAT5WUo0PpQnvFxV1yA4ZyM5y5hdrvi3m4KaRFA+Q737TtTMGSqu0EvstexepGhy\\\\\\\/jlJz4AX3phypmd9mDn51QD4BD7WODZMiKSbxPovdZXDm8tLyOWZ7s\\\\\\\/Q4aI0t7CQhURduUxOcDTGdMJcG5D8Rh7kdse3sNEoDEaxd\\\\\\\/ifobvMQlG5IDwKirLUIEJ0ODkaq7nSxuKidTg7vRjK4YebYA7tKmpliXTPICJMPeZIjcNuUZx3UhWwQ5YsvsM1rjCTJo3mqO6x\\\\\\\/qBgKub+Ll1poBCyDxhreUvkMe35h+9lGImWprdkAC9NQwumKEt1bKfu6SquZcfH6UZYv0E15uFAOBRAC3tIrpKNRzldaB7ZYvHSOcKOG6ilXQCmL36oDrc4hxfiRAg5w4e5fAAfO0ozra3cNVaMLKZofQaa4TO+wuBm9oK2PY\\\\\\\/4WiEI7yvb1YFFGVghWw0qzoeSJS2yZe4biYf0RPK2zKj5+t37BMQvCpzCs7XaXLLWpu+UOBV09+dO0Nf1vMSjJduSzS8Tz9hOGn0+d4puHZOHhyJ2HMW8XGMpXwLfOS7TAkG4RPjPORisIYgn03HhTSgU87prXnBaIBkD1yZeCVmqUZvpm0+uRSoFVyyyAaKBBQ4MNie2LulrsLifSZRiu1pCFwU2O6qugmDtvLj6hvehLy514LTuIueak\\\\\\\/Mxs0mF31CjZyLkA\\\\\\\/8MOoQRhDU0cZ0GESC3FNHm2v+mwzPJjD9y\\\\\\\/NRlagcdPy3m9idhgDSXdmh+Ec+s4ZTCD8P5mDm+Sj6cQSWmCa2FgNMx1a4CGoMt\\\\\\\/hpFBs8wvVlb40vExcGtf7laWuoqWAYXRTSRQKq4+cr6v6weGuaPZDFlafibVwS6SAlqzZL6F2QeyrP5gZ1b3EV\\\\\\\/IAdgaEoshZAW7Qj3uZeu08vcoISBcR9D0YWtYWc6jAoFS0Rsxp7NQmH0aH0O3F2HgQ2ENFR0I1SbfbZce529z901zfJ5e6TxPiLxqR+5yAFzqUOSpHkcyBYFdfHrU+ghwk++U7OBwpLDuck0whxi\\\\\\\/BC6coTseX3tmRML\\\\\\\/2dl6UoxE9Uxqufx2RHrYDYTysS18vEaJ1OtgLK1wzsgFsy82LyylrvUlr\\\\\\\/s+rpYSRv48\\\\\\\/4IJg61eKkJv0OjWIvoa9oGuY5uchopKg2tHkbS6FJTD1lSm7wVAWS0zuZKehccic8EZY93uUWBuYk2HXE4oCsRnk+n90WHZGtOL1chXP6d6wpiuPE2PKK57reTqpkTT0kYA74GNRu5ge1s9IBChzsCvTam1tM1Ck76MandD1ToZo2wJ5vMjazqEHS9RR172m3cYyAXOIUi962LM3Xp5rDmaWD+Yxy85F1QDxG5GSpSjA7zhyvvFe8hhVEV34IfkvThYE89aVTykzXv0DB3gTNxQabFidddXDV1PG5oU43Nx8+2GIuGc4bd2PqoWOSv3H3pf7E9QPxE3LWrjW0SBLsfA25HNkpUzaR6lurZnnYWFL5ngd\\\\\\\/AUQ6kJBtVwFzOBMQC20K4KjuDfDnWfVKR+QrTl+uqU24\\\\\\\/oiyIbYOejrvfKV1tHx8CQDt8w5Nt5ta9JLHF2Zq26WeSit\\\\\\\/FQRIgdQ6iCKugUhgtzM+N4kyZ4pW5LQnG5vWq7B0\\\\\\\/\\\\\\\/yCau3hFgBRz1M8QcLG1hnZ1NwXRJGZ9kkoNZv9sqnK\\\\\\\/ApeobyExU04Xj13EdN6eMh3Lo57aUo7Nvdb1GThfc2igg1dFV03mkB8x3e1rFZBIke947QK5\\\\\\\/fIUL75I8fxyH0VXs8BhqxDvqeL4KYnp63zo5AkgqSJcjIc28VwF5RTP9a\\\\\\\/UMJ9Kw68pm7HBTtNpI3wawcJWyrpUVMoY\\\\\\\/lBOgXjadEIgwtgTDGCh3v5rU7DmiY\\\\\\\/PS9r06xdzuB2uK26q05A+5vFVAulB0wo0bcMnBmEnM8GDOEUlAiEAjctCBThY7NkDfsv1vgPuhJ6AjGhJrAdeQbvjNMhwXztb7j97Ecnm2g1Vs6PSCVbSBCJLQtxOSYHh9lm4GX6gKhAUfZLyTGTtMTpTrA3emFVV0bkGoJe6nRNEfsBOdiCIaEluNKtXfUMJekpnXqTkTZ+o+MlrSmlWNyg3L+MN5P+4jjgK9AWIc\\\\\\\/WdTXy1E26xplMoPo8bs\\\\\\\/puZv\\\\\\\/oAgeFADUcclegizX\\\\\\\/x1GcEWgciVFkVPYSUogVWzFcVXnNRVcoqJs57JW6CLN3zMpRY4gAwPyM1uW2vLoH07LKym5JM2SswdJ4OKu4tkzK5PPJtywO\\\\\\\/4S8mDMej8s4iRauXvBGSR1DZd3q5GX6tvsjw+gazjkLuSATYQPQEMRI9shjAcXWMCne4N7J\\\\\\\/T0cZmkYfp4bacVE2iylcXc8uhUdMXGfhj5ZLzcmDlfOJOgkS5M8uALH\\\\\\\/9eJtpvgRLAhCLVVNnbdwSiuVJXRSsA0mWARCIWH\\\\\\\/WbRNiwgjyoMdiGcTy8iuTrWeI7t6KPfP6MLrkjE5\\\\\\\/kyxGizAN7VA5roeCK0m7YJu617J8iTNComLjy0j9zdR\\\\\\\/v+4q\\\\\\\/rqKq6lwEn2JZQC7C467WnLWD9CvZ9iCz1J64MyxCUdggrtNtzKFwez3gHW4VxVlqDvaouJtoSlAoXGK025lyAi0WMbMACWXOEnzuIJbQAOEfpWOy9ECJntmYzjTgzlJvQUF5wkzf6fIEjXyyBA9nPn0kG3CZ6dq44YzSO+82t28T7Wn8JwSgQ5yIErpRjl\\\\\\\/AgvZ\\\\\\\/CD+fV9KvKjlBMhR33nDjhYRT6aAvu9B6+3S4UdDkmELK9lmvaL4B\\\\\\\/K3E9s6rSekaZd6ktaYJupTz4niA4hFO\\\\\\\/WpowWl92mbOO7eFrLGdSBYt6kzCi\\\\\\\/DoUQNKaNfOMOC9ioaUPdTo4HWnY083jVJ8sZo5g98FLtDDeH+cjJ6x3k8feCI5rUg2sptGa40eeB\\\\\\\/ij6TIpASFnhep3iWzkmlpXE4uH+\\\\\\\/R4P5l3FHvUdpTZU4+uzKlFqi\\\\\\\/mZf+og+5N9ZChSRuIfru3pC84rlI5GF1Yi40if34U2sm7MRgUULqlCMA2It1B9FmSSGAIv9M7deHWjN66VqtEwbdbvm+DlN047mG6ANhpiSK0nRLXgxmiPv2VEDhDuLjfDc\\\\\\\/tFSl0ODXH3D7l0SNqd2N23KpEp12BBXrF3wg1kAn2mlainE0OfwOsiD8Jiq+XJXowZUXavwUDOe7lc3Edtlt0XL+9cX9T6SUqcxCg1P3xCxDrCIuZpoDJhemsimiCtM1\\\\\\\/bKkKBVpE8GVQfaRxtY41yXqFYDEURlJBUP0vdNj8zR4asUJLWtf7KPP5I13P\\\\\\\/2Q9I1vzA6QWJklRoNFn9fiJeeOG7aZ3E4GcvDLVdBwyz\\\\\\\/jhqkA\\\\\\\/ZD8cAdY0jCgY5WIwtU4hCq7nsSHiPB4NsQFjRl5t3GPxi3q4GuFfxtl13bdhNtkdJqgeJQqAdRSM63VjVozS9O9mKjWMAwS7cUCKOGN6Qpkj8UhVvDGKBBenGsWVqz6mx\\\\\\\/3u0wUwvdGWh0ciTXELFaSKubxZz+aKKzVQEl1e68soHJV485iiAlE+PwAZXjI3dMqlA7vEBx7h7qe9Bbc2t\\\\\\\/J99HiT08mOUGd\\\\\\\/vBSJI45jUbar1O0SD0FVbUMJaW2+m0xBdjt7ealFeLgyFMnYymSzAqy9hehROSpVow9YTfsO\\\\\\\/uktuRAUr6esSnpZyWVOnGy8vEwrRP0QqKc2S2o0mBK9jkZ9QKnEiZlaYpiGf2PVmD17iq8V5EPrOOIjO\\\\\\\/30jsXleXd+g8zaHvuDO9+AYo6Zn05XPUt+0F0UfzLVCJb26YLs1N0vxLHf+vW2wQElTh2VCT4T9jJUcXStlErfqZHvED\\\\\\\/jGnJ304MhYZfoRlrCvQMqGrNp\\\\\\\/J822h7JH9vIlbOuVmrAE03mZMZU\\\\\\\/HP5MVDC9Qe\\\\\\\/xKGID7PWhNn1u3jzUQU9f4ahqkSV8Qy91+52mSQZ58c9WPI\\\\\\\/vHgtBM56n3K0Z7eCXqmEHaqhhnFKn3\\\\\\\/IkLWxhSbNLMxyF87eAu3yx1YZfv+PN5Pln\\\\\\\/qNFVvL2m6veiTRYPV3AOkrclOcWzPCXPl7L0TjVOFahq3EeBrqp1eE6BptO9x+WFsm+NG2shEgy7c0U+Is+CaUisUb1EOHP9Pmvd\\\\\\\/BTiAhBHfbr+jnDbF+Ev0BhposV7ct0qg0uKxAgvHi+RnZL+uxHMUTPw9kD5Fo2UzUaWuS+oygbJim5eKZ5XOzsBRtcD3FtboqFAp6+EVi4IHeV\\\\\\\/J5Z14JiUEe1FQWV9D1eR9UJkxgs+frCt3KQm6yRg5lCs4kMA1uquJuliGwEaq\\\\\\\/1QoSf\\\\\\\/y2VaVoiX6pMPrTPhcoYOp4JjwX6PkmpzPlaL+Y4QErqC8I4Fr2CzZS5gO62Uf94QjtJhdA\\\\\\\/yuy5nPS2gi4Gs2zdXZuI\\\\\\\/nsWBb+6BfVXnfrXPffKJHVO9Pq0H1NsgKapn5AJYGzhNKXgxG04zFJGa9AqfuGq4\\\\\\\/UGnrTzQcBnH0O1H007KIF+1L\\\\\\\/UaICODZk45aYfiQy9NIfqpCzj6qaxSZx1R7uooTAfBTfUBVPbn\\\\\\\/Fo4pxF6dIqLAirRbOLN5u1LT055Fn80bknLiqkSHPbtL\\\\\\\/djsGcq4vajRWhQhoFOJ9onTOoOTdE2Rkyq\\\\\\\/DZSl0lX6L3rnoKKqSiD4o3d7oKW+veUFTN+P4eMnataSPPZKxcDPQot7fpwhIMMr2MuEDcLZbyjyvTHngseoY+yiWbVv0UyBw9PhMRay5CJFPLI1C\\\\\\\/mo6YfPL8DTCskTSCJ4mhzWhvtuNoLEQBH5ckdO+3jGIkmowduGDpwEFR17l30a\\\\\\\/IlQQ9BEv1T8g5pzUSylddQI2w6keifAPcL4e6wD4PzTs7LVa8GPbRpsLGewbH+xr414Y4ZneZSCsurJG8tf4yApZVk9idNB38nX8zhZkstT70rhJ7fN54Hn4w+PlkeBXzxv9LJ5iVdjQaKezshuRb22k7qwaOitz8UG0haHAFtUJeAUehVlQxyNpaRaviUng8RAdABM34XDbFvNC7h8NFElIC1qHwT7oF\\\\\\\/kuBlYnB5B9sOfVSt9+\\\\\\\/x7ZNiIwpZoKg5PbUAkaxafIkOBqca+n0hS7fJ6ms9fWfTARmDbjjYhIC9ZFcRnVmVFbAONeE7bgn5QZCqg2BEey9of7hFgXsJv3dpQMbqnHG3PotGGPKTbIOAhtaP\\\\\\\/gfxRVTyzH80yzq2PbiCrucgZ+e9LMPpCV08IrsgzFJcukKuzIfwZ9CNgQ59Ueh1ubFlroFw1q\\\\\\\/C\\\\\\\/BJ40P3AJbw8o1wjerKbpVXHQPC+nPpLZlb\\\\\\\/uICvrbOh+j48zewoOCIzF1fS4HYV1hbWtSvPXaIhVbIiiPRvVlx\\\\\\\/+vVkXQnfgZedo8VVQUXn\\\\\\\/vxPwcreyaif5oKF4c7+N5eNq1W8HQgIMrM1IlhiUTexYITU0m\\\\\\\/mveuNCO+7skY9rwS19ue2\\\\\\\/gP56GyjumnsJt7vFlFJQDME1vJRMitX7UJbhX3nJYuuvAGfLbLtPLnBOwuieS0VIKMEH23b58Li7vI5GJW4rcp\\\\\\\/RGeygVXEPC0HuwRTntJZb7hgOZCkrINGA4ZImzJRW+Rctd2H7bjv7S9t8ZuchoNKn4QzO4V7ZowQuGoL+ZOpzr+Qb1lzgtC2C\\\\\\\/tVbdaRmW5blRYCm+jCfqALEc8PYsMcUmVLslGwxzk6\\\\\\\/S4NkPZQ1LC5onQjuSLvCVDVs1E\\\\\\\/QB88KCtYV9r\\\\\\\/naalElNxoCLlwlDl5lADlVQ5fJvGGiF4+QaFc0lYoh9Za3y65V9ds93w+U\\\\\\\/8HV4PnpbkYxbPSqV3oqNXFz2qEvrFTvxStGWkTSrYB5a0TUDGTIbSbh9CRITxT5AMKHOf8Bej3Gss8OOyvvaGvxLV7qPBrgojim6wkzS6JKfq5E75P1PvYmBaUvUmfLwBR7Ys0Z5V1vG1FTytRLwXl8vhsYift29Z9Ubto1jUq\\\\\\\/oxcujAeLcEbEgXkm4BjBmVr+fxgDJZikEp5MIZu3MgZIF1gmkX4jkA3v4fpokJOozunnGBXb+hDVsiu00MsZUvU8hmUaYJzIKU+u9wpQcDhKoG\\\\\\\/JzRBn9iB4OMoymMKuq06lpWHNCycTwehrgIHG3hXkItVX3EzFUnuxz0ZaNvHyaF92aR+7vk43d932cjPRwQR9TOVmcLKIGP8kEyFavqJ\\\\\\\/LNT3CQapu0HwHDLfl8j+VsPKhDWR7YEJMBvLI6r0EBYMcWk0Omqwm5YR1UfeTYGyTwClEKCeIUxVEtSuz2i\\\\\\\/20pv4kVghJg+5dJN26mH9qJovFbUpJDtBS9dOZHKlgoKPHGCnsNbG\\\\\\\/CjbY+0DYAhMOgZ+jWp\\\\\\\/rh0n01XJGpVl1ayiHq1F9jzB354fV8jFbiMAVbIz15U9biIYAbsWf8sJFnokMLaAMS18ZJBiBA5GKEjRvbmiVu3OtaF\\\\\\\/B5gU6oYv9W1Jz+ztBJKSDKTKCSV1+drl8M812DwkfBLwoq1pQG7Pe9RYeGI1OOdH2Fx3Fj478LRYXzR6DRtgkEUdXxFgCoN1xJnXsaeDY37YOJkmlvfAJsKgu0zIagTxdDxvAnP04LqBB1g\\\\\\\/+d7oGMkBX6JnHzePqirwuEYLW9+UZur3qNexDWNd7nvm4TEYqC11hKbVm0PdNxsmUP+P6q+d3Lh7bzPp0bHQSczvDIC\\\\\\\/c8HWSYBdsKJXzTZmALUPD4H9+g0O82rLxdFcnp1tXxsyvHmL0V4yKJdLM0x9DFFfDs9bvRkGtlahTi5Aa1tqmMN6REj64\\\\\\\/9MiH1PfMGr080UeEo8ZU\\\\\\\/\\\\\\\/G7ldClnUjE1uSxAzX6QM3\\\\\\\/mpFdJP3wYvRB4QvgMsMK9hcayWP6XVWhPAptB+3EAQ4GRjI4njyTcwW22W5p9whKMUqkwGiRrVGD4SaVUjXOIHCIqqUpyBKOOnp0b6Zy\\\\\\\/9XAIyleIkf3rw7k5IEQzUkEZGgI23+c1ke22K\\\\\\\/Nbh0ljJ8CmGdqFvU1UCax1Zj2vBVOzxt8GUa3ZUb7P7zYCzNRE60g6Td7NPz7D93L1TIAuwwibyP1lrFTbo4FFQkfiGwM72Y+CTtFmECvLXP+BxyQfNmW3RG4RZttqvxJsX+lbuuEPUuDcL5OFpyW1FC2\\\\\\\/8n+rQxYwrQSKnGhRE3Exw6AKhgJFZVt72tqy1WLf+\\\\\\\/MAwgCDz1Ys+z6LXYGudlTTjCX2uwhGuDzB6HktyE1rbHeEtIv6YUWFDdAuXHeBZp+5WptEnjl5CV5JKDidDxiwSgjkBSwzNCJ9d0G74+zwkjy8SPJtHAObeapHx6kxo6GGTP1RovsM91rGKcZ93UXVNJ9cJ6U5GvA174X3TFEOy+fKTqm6ZINLyb8fTCzXWaHJbJ\\\\\\\/uoX4Z6dG9lnKYMfgiiXe53kb3zIlTYltta8RiN34XVdodtFD9zMObYJtu120mxOwIaJ7XvYMyDU9lZFvFhD79YXOjzV9SUaaMAbX8Y4LpIjZtX6\\\\\\\/7voVrQB1X6ALm3KHaxM3hR\\\\\\\/GVKaLTPvlRZ4J9r0HCufrIbf54SUuA63PobogXivRLU00bu9GvsuonkURby9tUlg5opaHbTDIu4w+IWv0xz8mxHbHiVn\\\\\\\/QewN9QP4KybUHIhv5YO1oYTB\\\\\\\/vyr\\\\\\\/KgoHGcIum6+IGx1CcT39IWf6c\\\\\\\/FmVj5Y+5t8L6USfhbjaPhijm5NK6afYuCjr8JJWQlCB7Qa8l+636VoqfEx17qnLS9MLINO4aB9evgNDgrqc6azNtpWmoHFqnufePMWPcMX7jhw5GUDUo12Dl7v5mjuAmQFGjm+U1FSNfMweE3mXR\\\\\\\/rc2nG1J+2uCpnIR9T6B\\\\\\\/Ma9iiQEvL7ZmLDAa5opbE39RXtfSzUh\\\\\\\/svFwCaiToZJFcwGk32lcRTHO6ZBWszOGzh6oGRWyBEQLvHByp4NVacXwH09MJznHhO8zeN\\\\\\\/uBStrAjkiGH0WXyNUZMlaAr2QbGyed7sLgKAyxcyFeBGCjJHBRQSHKv2qZjEnd1WGenuzHZmrmBUMlbk6b\\\\\\\/dwVKBq1nXz0z9gkB6HSaqifQUb1VaLuUT2kBm7PHROUYBSRmlWIlU7xAZJLs5hKCVmhE1C7jIklo1pPnzCSvlvwxuz7rK+VUL58kPi2Gpg\\\\\\\/Pd2ylXtD+VM7Tz\\\\\\\/oJiEUrfvt3bqBG7FJbgTUIvVPsOLUZMX+RgH5RgQzk+woZJN+8J1OZb8Xns5m0iDrX7uFGMnMMQEYx4kZDsYmSIQbYhZGytuaX1uCa29xViFXMLA+Jzh+caCo3g+1oKJIiMTaTUOlTR9r+QNzHCAxrq5KgmAS9M9w6EzVv5YhDmdBjTfkLSugC1ll+es7CBBeSlZ9Fa7ghHEUAB2rqDGDbAtvsEZt+VyJJIIaaPM2F8DNWK67GQJ4UlIb0xchKCBvPJX310FC1tPelU4ig82BvVnSztiQb5c6ou9Wi8BpgphQeEfOUalhG8O6wHRnZCuqxGrv5RR1OvMTufVV4cSTizCFqsJg0vFQIV5el2NuKSZaKXV8MD9jiAcEur3lucQCfV0bOfaYqNM2Fq2KUxpBP5oTm82C\\\\\\\/6uSWS+NcIqKqvEKP1kWd4QREBCjgg3l4k+GQxAOfCY\\\\\\\/nN\\\\\\\/bpODRJtp0ir0zgwYHqzotH3lL4wZgboBU6pYcR0wIf6n5Y8nrdbJLudfJJ14H3AdvxD6HPHSPKJjBQ3HlOXxzyf12Y3sKtoRepGkUmgcYNx1gC60Bl+PoQBjX\\\\\\\/jxEtoS2f3glq\\\\\\\/Qu98Rx5EHNRkpg+CEEvdAHel5493qU\\\\\\\/JGnXqaZW\\\\\\\/+5AXwDFfcS2xK+uNGE0lopjeD6UCSORinM4jKlG03cQYrWo3aeEG4OfYv6CLhShTFJBpN02GEoLsI+jvgXXzfRrdtHPJnPSZYrATfbcjI5\\\\\\\/JOpFKmdzoZ8UVR0eHb+2ImkQLzQUYmCmB1kvRlSaM6EdfxKQbvAN1wzRMix0d9A8O1SD8MA5ZN+gOsY44TtVqn+7+GNdeGywmoGsv00ifn4t\\\\\\\/zdwm2\\\\\\\/YtiC9AC9yczLFKzSQF6PCm\\\\\\\/Ses1qHJXWPZyt8BifEhPKPbDQYcrYXorIPpWAIFIdPcYQj5yLRsELqd47YBTOPdN5+maxU7zcm84D9nr01lAOJwsZwqJG0qEvumP3IBvaZsy95riguMehq0oiMw0hAT67tVFClYSk+SZcs\\\\\\\/CnGgC741qaS98rrt0D3wRxc+IoWiOc6fZFHgd+PYDFS4AZTaxRIV4wlKRi7V\\\\\\\/+1A4ekzvr2WyRzK71zRSq+IozYBAkGZ9LLqEBWoIbJ02rflgkPNduGLWjBqBKmacArknsjLTTnpvSjuPcNr4PDvMpFzt7oZjZ0WWx05hAofHsrSsOwoZox5Tj2pUf0+nKnot4YMZeb9uNvou9rAf4QihTa44hsyRQclmc35bhIlxXvTIqDhp1QGJvMAa2mdYnQAGpAY\\\\\\\/hm+heWXwGYIb6G5YNDuzk5Q0C85M6IevW0pCJPWhkqHuxUR7w24xcoMFsM7XNY\\\\\\\/HOtlpbXGu5QUwXD6zZS+j7R2AYsumSFh6SPBmgVOqeU5aWkHUJs3k6i5PT27eWM5TFKafPMTJwjHsaJj4te9RC7NMIcRCCuHEhxXz1GtuzJ4FMQ1VgvUx+URarCLeuKM+aw1weEJ0zu578tIEX8QfpeezNT3gzgF+0JDIBDur6Ece9HiMuDjnuJqzbWNWz1q0yj2M2slTFjB6gYvXCG+dnmJFozk0zyB0260q24pMlfwBaNAwVCo14\\\\\\\/n6Ye1og83hH\\\\\\\/ObtHdDuuIUMj5CG1TuX46Jdd+VCt0kIof2eQxzcx3jg\\\\\\\/58MYuPbN+PJ5mft\\\\\\\/2L+6FTTiikXFFnfXn\\\\\\\/Uj1ANzOl1Z1hZ3zVL8CUQyNCrImxvtlgWbidaCHBiwA8eGroHR42w9dyw3w7tv1mGeseeXtlbXgtHNOaioZK84Fj74flxtmzwwaAXGUnIbTNQ97xBwM5pe0+coE88\\\\\\\/GjPe9U1HBsHPjnKhzX6aPum2RSyPBnSyw0S8Ja6TMOf7mYR1dkqiz8dtO6UWwiSQWkb4JIbXAA8k\\\\\\\/rc+A3rqKHA4DbGh9c7PciiiIwi58f9nu3FTy66H+hRr\\\\\\\/sLL9+b\\\\\\\/4d5Tf5XNWkgK0UBdFZe+thXn01+oePNSqOjGu5lSUatLKyiO\\\\\\\/tXR151XwdBDbWLqV\\\\\\\/AOYB\\\\\\\/7YFDIgvBPYhBLGHblmfp8IdHhb26cc3mpTR99Tq4hqqpSbhl0L7u2mAfsx0xldnM9NSqJyhxzdAN3CwGzEfTBzPddAlNn9muNpbdDgrIlC52RNra9BqMcyU1efHhPy+0X5zxuNi3W4CRVEKGtjteVXWfYw3PNXk6c1VHS4jsbuoLm+x+WzHgsLFSGhBJP+WIBmSw8vtUQsleWNcFLhR8PTlWDMFwVwJGXxxK5EBiNX7nCR6GzFIVad195NXxcUVZBx6G5Oqy+OwUpZxQ1grRPMbyVGJLzmhhmj8X0vsIDQL\\\\\\\/wTJZSRdbkA6vSHUydP9H43tV3Db\\\\\\\/RT9LSbXRbOLCPjNPeFA+sxsykGL69aeFbW3UfDOF\\\\\\\/H10NQ4tihqZia5C6h\\\\\\\/fcR+vrrU4I09zJ0OpdZ8y+PTJS9BNkxc1oSPTNsyVjjaHKq9D8YFFogXZCmaUKFNVZpVLOlONSkt2\\\\\\\/UtwtmkyFuwEBXQVoJhUFNnvNqnxVrX5k+EUQVbDjXPk+Mc55zOE3BSSb4QYGMtYww648rbHcWGWyAIHHXgp0XKShsYE4XT\\\\\\\/dsyTkUw2QbuHAsGdFjMqj+6lLBf6Nt+K1EQtUOwILprDTiAWjf6NcStqhTv4uNIyozcnaVklthwcqDtiR42vWAjxgg3X11rgQKN4akuo8n9tGvpyyAbfkKY9NZkohWiQsOqGZIjgVy5hR0m8TBrkVZDg\\\\\\\/s0ES9WtYMSiF46+fOJs+QJ3kgKEoWgHjD5eOfpT\\\\\\\/64esghUR2JlCqs+iAJoXfVFYZHO7pH017qUhj5yRHtB\\\\\\\/tRvIPLdrf06ohWOoF0aHgkx2arMsqXG+TDjfjA8EPbnMKTeTSEM8uiQ2Yx\\\\\\\/OUd2rgYRN+8GxhzHKZBHE8c\\\\\\\/l0G4s9S\\\\\\\/+QO7+NFweIULakyBxZdUeEtpXJc0nPHKkbgU1V2cnUFCZIrB+LgaJSGlVW84HMdVIdDnBiRcvXiZIFeKXaBrNC+rZI3tzIfNFcwVzu4xp2KfMIu5vL+4HJOSDfBFbhE+tdtimzljcu1jvgiP7KosStyiwBb6xXr9nH4dw\\\\\\\/XlYHahMO1tfuyFyoMJtcyky0tfCeTN40p5RekPvfrLuFzctX80EISLtDeW+wXqCo+WdyXv4586FeGNtPRbyUXt0G6rQZ8pOh6C4MblhRPVaXnWjs\\\\\\\/EjQiCDoD85jPxEN1ddn52gE4JblItMFPV4MYac1puRThUOoKN5vAeGBLqjieDR8153GWudekIZUrUafNyBokCEwU+e\\\\\\\/YYQ1FafEcXwZxz6M7+KCUMIqRcUXS7UkK4\\\\\\\/0SdqPi4vmlLX7v3VQLso3\\\\\\\/3LRWeSTgDFDOQ6Y6QLlxryDGOLp3OANW1hXdZs9VXeiAHrftp0DWiNbgu1mYI3Dt+Bfg1TkTZg73ziDZbg5Fwm2TfjnAwKaHLI+zy6MomJlweEr4LgB4vJy52bGablH8gWtjZeVBRdDC9Z\\\\\\\/TzupttxFho8IiL1opf5qEMShCWW7Dnhx5ZDGTGWfCS3PtvmOtwLGqi4H\\\\\\\/Dh187\\\\\\\/quE3K9+LRfUIh8wFtCBZZa9mnqt22BSgw2xxNHSkqXX7wukETtg4Mr2IsDLh61t7V06jfg1N8i2NOT+KRJ4064JsJyXfKrveiY3u71c6Dh6\\\\\\\/AESpUoLH6oim7CPjkphZUfMr\\\\\\\/0ye+faN9bIFymwIgWPIwgBxTmu+9SV8Lc8IaFu+K3j7GPzrIvr4m+gFQkusLPAoY2UnopUXZqQYZFNudVLaAvpepZZMUIuJaGYhgADVwm46YRHjopV29cfYa+qtkqjvEOgd8j9UNrftvKIff5CIlqQUP+\\\\\\\/7tihLEVYGf3nu2Eui\\\\\\\/58rpGx5+liRv3UsK5h2f9r\\\\\\\/OJbHN7h70H0Pk1NwWpEVBYnMsiDWuBPcTZxDj\\\\\\\/g2l2z54T4dNn5p35YmxGD7KdNfNf+GqnQrjdd+DnVbrQU0rZZs5fL3WxadINEJ3oo6IgH4BzXqNcUsu54\\\\\\\/aSDXpgV\\\\\\\/vPXvTFQ4aC1tx25iTiVNnvFg0EThmf1xw58K7TVA7CavvbanNiISggc0sJXOTHeZjQxEyX2fuqlZ5q2CS9bFEzWdtLLlbJPMnQzkVyApGyvIVqYTwnKnE7lZXOFljM7agH9PYXhXQGlcCJZl7kHMZmWlOA9Rzz8pw2so2Fl7dYFTm7d6CZ76yqQRyNSCIQ86xe12N9gqfNE\\\\\\\/4lI+\\\\\\\/sPQZIKgjCedLhzLFoHVAn0\\\\\\\/59Ayhi2Q0wMYL5ohM9tRJyWNL3m7CWByahEccAWXKOv9r5Ewr68t2bTyOqH1sSNoo89h98z753TmQL786JubDhqW\\\\\\\/TvLQzJ8BjeYpFZkH4gkbbfV9EFxSWu\\\\\\\/69r9+dTsFPciYbwK1rQiCeTYur4KfqTpd56\\\\\\\/lEy3MU4WLn1cZZprT66tOa9qr\\\\\\\/3KOuBxp9EfbwkQYpKBuin0W+xUPh8DWWRKYx7+hNDy3gYXDld+V+PWEuxYdgMsuD+aqRulBDxC3GxGhL+oQ9KwRdcqWgaZtBkwKecYebdu0Xh\\\\\\\/4bewO0JRV3Xr80gnAbFuXUszXwfgfpdC2L\\\\\\\/AIvsbXeXCpBn0uiI0J6eyVAlTx4Qa3T8ZiE4NFoHwnQgftV2s+7s3aKbNmZt8dPNm2zKRDrGc7nC7hkrCzJb3\\\\\\\/PI5jy3yTtEOjbOUOfn\\\\\\\/rwFZdlhDystHxMfotqcr513vseW8f+RKrYCB0dFeNggTlzzqAHjUsnR6VXFspDVjWTrirqqU5fw3HRBF\\\\\\\/fbg8BXILIBZZlbZz726YZcgXX16\\\\\\\/NdLkRS9k1WFexxs2IOtTS8M665Qtn0Cc6iq5GMckZPKEpyZEYBK3QPzH\\\\\\\/7CQjSsQCyMJSkKmmA8liGRQ3zNrBg6ekIiLbWJ7HGAC8H0cvw\\\\\\\/McW1MfSM8Ts+QqM5HtoK77\\\\\\\/c0rw4hwZb5PVM+TRPWadCrjZSRtwLZJ+aoWf43Av4H4+3uADBcui0KFDWSoZQlaINd8cUTOsbvh0z\\\\\\\/0+Sem3QxuAw4Sm9Ih8qU\\\\\\\/10gDdyHKUDOkz0FP5heBkjyeZgDx4t06lQeezOe8rvIr3meyrO\\\\\\\/4bHYMYrKCOPnN6nm7rcdw9Uk91Rt61eFIAUZ8WNSUKkpamrWm+9hEFx1OWiFITThzJqcee9LpbvCcVyvodgqPjlz5gTvPjLakSbffe7E14b31GZg30IFnKcETDO5LX7XmAoZpLQ\\\\\\\/dvmaEywcu3Wh9eZq1ZZ8DQkFuM\\\\\\\/tgcUnpuUfo038WGyRzzMY2uWVLC4bRUH7AvyrSCVlCR0rn8qQfLwYr3YNpYr2ScXAcT5dPrUdad2yBulmADqK\\\\\\\/l4hbASJhq4V904g2phqqP6S2WuQsf2uWVkzNV3mXVveVZ1DRvuUTP9RTtVSoMoKDMeq37ILsEAChrRyGdgnfCJxOIAgjeI\\\\\\\/UOORltt\\\\\\\/JG3H8O+Fi1wOi7sesxeZNuKyw6GNdcqdfOaWqWFCLbfZK27C3LJhbxbR\\\\\\\/FqW2pV6+hKYYuHhMJpwAcTmkc9RJeUhzycPilFvLmrvv9iSLwSLg1\\\\\\\/amjH5\\\\\\\/4PFknrodMXV+jiB8rlQT1FaaY+3muY9y3wLajssK1kR2BeAcHiVvkIKwRYw54DKWYZxXJP4es6zcBltxtxnXdnR0ubM+3N8vXmYrhB9ZaMvzrWpl\\\\\\\/2AM\\\\\\\/EPlh8EoTzWmOWb0v5+RJfZ3cifrB3Q0qKKF\\\\\\\/quSadQ2us444D+QQ0NF2WOezy7N\\\\\\\/4Wwxueo8XX1j9QTKBdsOy4TwtIzGNPz+kraaijbIgMNeNMSLVFTLXqkVzDr5TrwMf3kS9xl5zwbs7+tuO1K7M+YhRPUDGev6i3cqN56qmRwKRh9ZIbNhZ10wTYD9fyDWAuCMekewLZ8o1JP5lmCPYyZb4Hf6o7ecszXvgTVRHsLZ8Fub\\\\\\\/yFTlpq+4UaSBsN9CgR7aafVlkaINKjFY5JBRtQRL4iTZJqnGqaJO38T8aGuo60YGK6u0Xj+bKYlDpQp4F9ThB+Yb2i2qaNjiXcmuRf5vBXJNFaDwY53iSzNxPqxQQZTwr5Cg0t+SSTGwpBPUqRyN8xFdr9lJzAFYR405hKsMVfneysWGB1npIScfOeQKWIZEHHBBPo5Xeu4lG3J0dKU3o7CmOt0pFtymO2K2cBUfgSyLiKd4kBpXytTjn8A3PZNRyQ8QEInRbZ0wOq8SgBP9QYxy4Icie1wus2PeU6m+9RdOyajrMm+6F\\\\\\\/h6t246QnWVg8U2KlFBO2qLZ1qteEvIEAxkfCF5yl\\\\\\\/uZvx8yAFKy22n55Rb6BmnBvRHInxRRphAkCL070Dv3+TFb7PQJSGm\\\\\\\/WjXUO3Z3EDzt6tDgrFHTsEps8rlWdKybmw57VTqYlgYbKYpRCD33yf5BGh1XAf3ljJXQRHZ9yk8UES4LZ2Kn54FltnoE0AXpiO2aGLND69qCrixtBGWdGKSyC3kO3snFCJnGVEnSIm8ksibqIXP8RpV5Nwwf\\\\\\\/QYJ41WEYtZTWzIUyxolLpg43JdYJ\\\\\\\/J4nOLX4SdtszYhNB3vxz34ovlBse8HBsX3pUdcSdZLXJsBHAth4VFL2lVy3dP+76pitE7JKf5n0+o1MYnZv5V9vFxAwwufDeQMTvp37pDY2eCHtIz32YWHnIgWMPYBYbTpXQS3ZCBMgBsCsBEsz8E07cf+lkuNnlx8uU0SPSp+kNjXaCbodbRZ\\\\\\\/CphsZoYeP2kxXgALtdYcIGcrG3oq8\\\\\\\/jHLXB3NT65U1jA9wPXdaNB4rK1QSyiw63rdkzTvJCXqunZxrK1ApJHiUWLlqu47ryTvVumw+bKPFPXnCtQzhEq7NhKeSZ9j2Lqu80ytb8M8WItWFcCx2IJdeOd6248\\\\\\\/lhp991i+RWbSw1qV39ZpFpTCj5D8rr9jFMx0r4JiqOI1AVnRQxu+qU6WEJm91XPosYIOp0B5Y9BuigQFGcO\\\\\\\/P6EmZvVDD6PrFMHvMnoFbZGoZKOhC4HK8PxvrS0ETm\\\\\\\/FVKJW+OJn7y36LkmCbjAgVhASWEdzmtdUT0a9bAG+PTBQ83toOCXjVVZVQCtXXUtxwdIunmPlDmEwAdHCYYHsArlQ93GO4fjpT7oweYVggE14szD8iY6mDdwi2WF0S9vJdkFNNGb0JmTL59PlW\\\\\\\/JRJq2Mvd9u4M1nCAlQ1yDEIkJKWkee6k8+UWGnY22kljzYSmvDDJthMpP66bclSudhAM\\\\\\\/LJfDy3cUtK9TCwFnTaGXUL9H9hdu\\\\\\\/59RLQXVoKu88CAqPtl4Vb0HCvL+0ZbfEWAg6NshIvADu6jVfd3MxIQNsdsOwdfZxNObOL6\\\\\\\/775TgoLqFs+4jAZx\\\\\\\/GSCXXaKz33mQWB+9nkVkVuM7OGM8XL1RYH7QHi79tA+JHNiPPm0bli2i8UoAlFne9EV4LPQU0n8Cjd1vRhCWPRJb3Cc+mgU6dnCLDBe2O6rFBLKCPzH\\\\\\\/wyxOt2m0i\\\\\\\/kNTMoyIpl5+HnUMRA6WxfG1OAyAtm8pYwnTO3kvLHaAySS+3byYd+MD7OGSSg8Jy5PaEPqD8PigptbEUyPjSPujx2\\\\\\\/lFl3NyB2z6mV7nvumqW6fET4uqHNzd8ZRukQk6e\\\\\\\/TDMcKq9gK\\\\\\\/qWk1rQeTuO7aw+VuR9t9B4BMSqP+ubwj4etKpNq99BDagBblH+uV9HWvfQGw02U73Q7xtaXPmFPH2XFaqZDuxNuTkqT\\\\\\\/fvW9o3R0b2u6OxncO15TLUJe7OHUq7FFAf51tY1Gztua6cB9pywK979vUAopI1zfHOAgJ1ALnz7+P1vN01dHx0VvMFpshYWPGxHQthH6apBlpIOu7qOW6chxOZb2wk1beey30bvZz3s0nn4EEr7syqr326P9gm1DENMPfm0MHg1bPfxR0VF+FhyDofc713n298lM6JK7zd3Zw6BVJ6cUnEXLxcDZt6YE18b+nVBPSzrGIIp8UqwBX7yortuEQXLzcrnWlU6ZQlhLyXZWq6kRtTtF0HvmRrE641J5zfDa2vi6DtzC2dZ6ZIEOCsQjs\\\\\\\/zZAFdguFbbxyt4ZJ1zuWqEIhJmh\\\\\\\/KBa13D0\\\\\\\/\\\\\\\/25g+g6nkJNiBhqGgvrwnJT3ZBHGQENd2V9mqY5nJCu31GFrqnK8JS6SBVpiHciAsSg1Iqb2ER1z33ATor0yxInjjPRf+md4E8dl\\\\\\\/cH\\\\\\\/aEf80FMEc5SZRJFpbchYjViCtdzT24RviPCEchaseoLUbQ3KJwMJbrwpuXuDLxy+WEwziqr6SLCqpXetZFKwi142J2WvD3aXm\\\\\\\/3xTwRhIohfBtBgU8iKIdqyrk9klojwRBZJHOXQefR0Q86toSITHTRHOzaIxLLvVpRk79e9X8d6W0hMionxQcVFQ3WS7DycFrYpEOWhArq7wzIDqFq32tX2\\\\\\\/4lpe+J+1VbQKbu8GYXst5olBV5UJV3QCIDwEcGuZhFsCXGYZujh\\\\\\\/+dDsT5ihiTtfLBY6sGz7pvlUsWMWJluwP8FIGf0JizHeMMwIS+qwyDVmBS9Pga4ETAppOZQTUSR3S9xT4EYiJ9WNQLmcFlgYiGIOYb6YKK9Mg2elozVW1nIb2\\\\\\\/R8iFBBxgzKjiIFcu9Qi6JuSa4k2\\\\\\\/9DX9B7OeyZQfhFjfn6NZhDZi8nVilLZV\\\\\\\/ywis0MCX\\\\\\\/uD8rw2lgSFGT51miyg6Ts54jOM6Rb0ImhlqVrQotaiqQUfzqrtBuaSichnFPhz6AKiIlirx8V\\\\\\\/qQPyEE6FMN4LunWOdTKHjWK+SiNwW8WPbQgul1bqUEi++tw1AlvDQEt8tGz2lJwGhNNESweVWjCl079AnOc1MW5TtHx8Zns\\\\\\\/+ErY5m9z3Ybtkj47jggjpDiXetiehal3+EUN5PgL21S02aH7tQeBrt0S2zDzzn0MjXQcXAtPhX7NFagBm2iOSG84YEjGZFkXZg0eTTIOwx0qQWDl0Fx\\\\\\\/sqUknmNeXJOcqquBFPpmAtpy\\\\\\\/CVn3QzsTqKB5E0yJs3Ml6fgUr\\\\\\\/+Of8WdKCuNfp0Sg6IcfMDeuFhswCaPa2za4C6e0utT5tGkT+L5ilieQki7kFoqJXtHuqi9mj5ig0FxFtvkL9dyYxwqjyVS9HPcOFn07LOA1wDvSSO6QRxJ8WXQsM\\\\\\\/FmXPkUCnU1lVbrXs3OrkxdivSDcKQiHt3DMfDjYbOYAQbh0tIO7sH0E\\\\\\\/C52OU0LhmbJoJ48mGCGfkunv13CgwMwBINdrFiMCvhZfVVP5LhFwRpfevFCGbolHj851lYLPHo8c1MuNP9E5uXN9CRWxG28H8rwgeaB7Su0cKZ2QISKLnITESuYXFOPOS5yTAhno42haFebgF8y13POUtE9pDPDSwNZLKY0qiI611EPANngjupYb4EkLm3Ce80xDo1J9FXAt4ilCGkh4Xos6N\\\\\\\/UF+8r68\\\\\\\/3mzlLcn0aBbXP6CbRjD0k9GILK5eNCPcZea17rerMVVN9vuXIMeBhAC75dJC5YmX3oz7D\\\\\\\/nrQUSooLSIRpEDvlhOL559d1iIVJX\\\\\\\/cxkLY+ri35vTMULc9FX6EzBWIXwKJ\\\\\\\/08OBt+sKp7MG6XE7izEzgxkMsYkQKgd0OCWbaO8wFfrC0fTSumE4RiXJvfwpOQIFlAyRuavjfAYmsg2gDBTbRTDnht4glPlkMMHK5zV1wXlNln+aUX1TMQ5ggBhgXaw7i3NtA8OF8PrW1Uq6Fj7iDEB6bmlyciW+qa+FZ8finWk21Q15qwpuSWb1usm3PWaDBbd+1eELNpXhyHkweTAj2FiIvW+ds6MB0F+tXFEdYXzMMTHsMDs4SovDjtboTO2mOD5Jst0XEGkjfXHTTnwL5Uh5NuJQyltTS4E8Pz7VP9rETHi32ygOSIwX48QM5IqmSVvrcVpPhPVQPoBYXjWLwWZizIClp6stVEkSkhfC6uCXqZCLL5E\\\\\\\/A8MS4Gu8S2BYo1ru54Q2B3dAPvhhwgwrYaX8OPqJwH668a+jhzwqoM+OpVNvruM+88s89bQ5VXockXDDb6PT4GJ+ldrF+9pWcLn2hLO6kb2RiB+gzQFVn6xlaIjTJJc5n4iIy8Ya5dP8pi5ZCPWMddqzFM1cEr9lvdDoYvcMjoih0QlA9Ukt+SuTAut2YRX9XNwqRmE0qIhMDKIrqCkJHR292nwBWhykurvOsZZJiV7ma9iCwcWHmw8KXgYt7jLQaSae+mz0Au5Q8QwXPOOJHyb5RxHVMk9Q475vGknJ0tmEiS1DV6QK8SUZT11ueS0\\\\\\\/\\\\\\\/BmzZGkNNMQwrn99AWAcypIyYuEEWmjzsGteL\\\\\\\/ej4i0ub1j\\\\\\\/K4Jo9xaOF4MQlh+2fSjjKx5obIAsesCCReJamX1C477GggZxjEIkMIQs3iEivrMPI4Z6YuM9dFvhJAyexCJybGrZYkPu2HXXuZogspymojz1DDvzNDi897nd82O7FLu56+FBkUER2TEBkxfD6RfK4hBMhRClECi6oU3uoXG2gO4xZXkfdMhZIDD2v8rCLnM7eKrktAtRj0n6c56yl\\\\\\\/l7G69OiC3WhzY9N6UvlpGagVZDk2SlJu4zAcQk\\\\\\\/XHA2EyavYwRKKufuV7klqFO49uQR9Y2nvAjFSno9VREjjaOOI+4MUjBooGAy75XB5OYgcNpq6qJ3MKLvKQjIq+PTa6EnLGsZ6RtaOICy28C+d70UnTFDZzxk4vybLLc8FvjBajrfbkDkBz1vS\\\\\\\/i6SBIub2iPdKnLgEhAwR\\\\\\\/Hng0Y3+T+emUooQaPdvJ52xAaA+SMMttlbYzIxb4Bum037Kr92gpGQNdp0J7qUVJk3XXKrukIUD2+DrcuQvfN+nI95jE1ic9Ean149D9Q\\\\\\\/zIKfgkDmivzf6EoNs3pX2d8w5aumAI595jeSQ\\\\\\\/6XJQSOmvxIXr5aIAZNAVoUBiLN0bG33cWJR9gV8vrY+w7EEg3lA2c6GAvvT+NY4JP3sshUvsjsG6K6EHM8sHpeVI4oGpnx9yeX8riKfbPDawXij2yk8sykRkYoVXfw6zRwrkAjs3LcDDsUZZ7ySsloDqoy8T8pRX9VCF7\\\\\\\/Ga9N4BlMjE4gmcvUMKAYCvNtjW6ob5ZLVobLHPn5glsWLfFYYTwpS+wiP+MULuWmZJw1XbbprU6Sue+DKWD3EKFXbg3mBnlLe3s6KMuOF9Y7s8QEXUe17Quql5zRN2ALna8GiCcTTmG+Xqir\\\\\\\/N7lkdPRlQlvcMnPTgUrTKi04vsg5I8n61jov2qkVDk6LIvD78+ZDGrZkelPYmKtCrgGkbP478ZjVDrcS6aSKKhJupTzWELV7K1c1aCVGqpTMEH8cNGcA6WrIIGDNeqm\\\\\\\/n6JyZpkqfmUxnvNTRzw+54fleUOPF0LqudXvNgpQSQkRIK6iDd3+st3UExr2zLOYFvrg8h2v7OfPld242SS62rIqv8GZmANHurs398nPsYX0zjXkNmdS1\\\\\\\/zGBvYAHu0r2KT7U9Fa9NP3NPFueHIWi5JXq362vypdkKK6u0o7r0q\\\\\\\/9197\\\\\\\/BsW7DqwMooar+YvTpULmGbk\\\\\\\/tYXMPlos\\\\\\\/8poNprzwBQpSRqhBnPxQaSgaSf2kRNDDtY54Qf7NgY4tDBLLQw3xcdb2pM6Xbp2K99G1S98cS5F+mvECxdl9Fz9xD\\\\\\\/ejLsx5RykD6ZRgPorzfifFQgv+i+WXMHz4HPeoC+YaSvtYNUeVV5NfxyAsa9tUKX5T+tpaEfseJDs2z0At6MAj6VdOOa8h2zVwjDL2Oj3IFIxtja7K4tQ0E58eSh7eEyuNUOuLe6vvF95JukbNjVa0qFWX9hny1VXyJZ9+qFdcE4ZJrXSyLvLJl2S06ESGbunRxbJhLbktvG7tFmER9FLgfqLmL8mNtSe0ZBT+686S5YXAv6PQNzekA57JQw83XsiSUV6gJotMq\\\\\\\/EoRXScuuNVmZowbErAHQCYlMnkO\\\\\\\/RZOQRZ2jlVxNg1xmGGquL4dL1UPhbtUJfDkE3mrp0wQD72ZXhronxYzrUJcmSfLv0lwynKjAtmEB\\\\\\\/U4c5hIzDK5cHskD6CXnawzlvBYsGrDuFebziJKdZBe9dTVXP\\\\\\\/\\\\\\\/3cwxUkaXjDwzJq\\\\\\\/+3ljb137ZiK4stNag\\\\\\\/XVyO\\\\\\\/lhLQUwreT5fAgrVqYghKT7KVZQHTIrSgB\\\\\\\/AV3MwhpZbE1dWfDWPjhScmGTQ0mB+XD6tHyox2UrrzZkPVpCZbVD\\\\\\\/TQTPHFZjUzKYgXYhUNhHM6pZI4zkZitv9vMxAfV6Qdz6m1VTgcua0ErugxR5ZQnAt8Tp7TwwOjfgi6J62\\\\\\\/1uD1WNP+Zh4kUsDKg\\\\\\\/VamJjqyxZhXHLZOw1mrv1fFMRMH2pf2GGfqg3ZYYjGzWrcUMRzPWgvCMzjQd\\\\\\\/8zd7ddshmH8e3aqQDC6pt3anIbmWnbXoEGcHHW0YYfUQILh+S0oK184NA6Mff7A6U1USLZN2ZJ2a2l94GV5hYIwssWSLhoR3ETMdGDXavxu5fLmqQNH485IjjGnHbqAomSMSsBOnkciXYtMymdBqqoIvyM4zUQP3V0vLfHviI+pMawRAF1FSWF6m55eCHk184Ta88b6KK9zAGjL7\\\\\\\/z1RvVRcaJIp5B\\\\\\\/6BzdBaJkPjMWcTGd\\\\\\\/OAqhTzgZcU6xE4dcUyypRM+cb\\\\\\\/7Oug7pJ9ilKDa5M4ldsgZU+RbbJyEkdC4gfJl0pNn34F0caUG\\\\\\\/7q3FkzOQR1rtf6p9PdokpHlCgrOqNTq6sw9k4pHgIrnzyYyruacb7z32AQM06LaPdUgFZ3kKfgVsmJ0EbLX+CTaxnvGq34cJ2s5ZTa+OfqJkXFQH7hmKNhOeX5AmWJVoxrNA3XBGD+Q08GJi\\\\\\\/7cHPrbRJ8U7xsT6MuaHBsoVBdW3cSFEJzI25gcJuUOt97vWp1on+zVU94iBEfi9mQp5hE8VQLKA\\\\\\\/G1Q8tjcB3HfAAkCsBhr5IAFL1fMLDOhw0DL9k98rZp\\\\\\\/KIBT6BNYDj82ZLGOmS6Tfob77f4sXl4Uwn9KvpNa6vsoc75Xx+rx8NaUqlsH5FUTV5nGyeXVZUm2WTeSmFga0o12Ho5LfaDdq4iNKUxPtJ+EiUgZwwOgB1LV29GRRL0kJfEuP60JshtOEOfROmxDaj2MBGzYAr\\\\\\\/1Z7nwSoZZpuoVlHDiFhmGm5OtdxOfC9HZLIa97vlpfFrJ5ECxBew5JO5h+xbQP2DMLSkJrG1GhJEA2VfJU\\\\\\\/me0eb46l+siAnXicmoP7Wy8n32fZ+dAgBra53TLbqv+kug3VBsNKNiSdK4hWR967jpBavLDjlwjI43lqlFO9U9bEJ4Bl8Bb+janGcVzJ4PBJ6RHGZ4YTCELcJVPUjtBtbLRyb20TBZRKrAdU09tezIzmDlFSKTRgTxJIMYZDnbxzbtF7\\\\\\\/s\\\\\\\/gCTHK2B7nFKZkIMU2lyRP3c9oIQNgvOA9zTNwBgm\\\\\\\/N6QYz3pzkGEXwqptVmbu6HJuZxV16YMQjRm8rvSB5esx6aoJu+KQbU4BP4\\\\\\\/FzgEEmx9Qpgr2\\\\\\\/0iTyuDrImX2WTKbLg3pFEDSu5V0wp8K5XkLoniBmuKXvDyhurGSJ9KctwD\\\\\\\/9\\\\\\\/EVDpvSfVsjrA+vp3acv3PdoeSJW8cEu9UQ7QI7g7M3E3NlVwO5iINK0kUmPSfk9bDqHdp7eftzuwEHRRrWeRQvAqxGrbuUm6trliaT6sqEj3qVi29ghBsILayzlrWAIF+3ImDdW53SUESDh8GeM6peqqNacLZC1RHWtJ8gWJpj4FxV8+21l8vVH9XCr20xyaHdzOY0VggL6p0P41IR70A9eJVD\\\\\\\/VY3k3A6PN9jv+AOVZuAi7ASjX\\\\\\\/wzpcb2Ri46rkXnnoZaGxaIfeutCQ4yX2QLpNa02D27GQee1qB6pe9GQEkQ67REMNmNm1Lw2HKKABIXgQISm+t7rXOR+7D765lSwDsNbDbyZmITsMUqltYeQQZqzhtLT0yi\\\\\\\/ysp+L867M0NulqzpGrBDtcoB7lHaf8nji3Ezaudfi\\\\\\\/9grD4rwDiJ+GZxjnwOURIPmcoyOlUobFSRb0PnxMGoWOWhJvc7FCFqBsPgp8H7deTwdzTa7eMoA8W2bcpRawjD5\\\\\\\/1FtU1X5xviRo21le2OB93lg1A1NDPueJ5t1+enOZ8Zj+cY8xpKrt7Up8AB+pjh6RUykr3NBlPuzob\\\\\\\/ILgMQ9qCHyz+TS7C1uJOCs89ijw1do\\\\\\\/XkPo\\\\\\\/fiteWFVY8RdljSwb\\\\\\\/7I1MNvZTeGpSrK7bpJn0+wD7nSlp4etpxqPiJNPhLMSUBUrWrSsNKX2uk0\\\\\\\/j241ymryGEL6GFnV8d7aeEmJ\\\\\\\/KaD08lLS6T8q1q6wp7DkLLCe06De8h\\\\\\\/ULFF7pUIPPsa9GkXPf\\\\\\\/X8Y6k6Ov9LL\\\\\\\/PpUYb2mTaPQ+vZQ2weAmCKg5aGohQ9WtPHddGWE6OL8C6MIbikX0bjCdWTOToy3HiaCvOarNMiOFWhq65bdiMaeFrvKZx0rVnMk3DIvvESvVNDqmA2sJUcI\\\\\\\/Yhdd9pgfVq2rQR\\\\\\\/X3LrUoQd4yWjCdUgwlzW4AjeB46I55H7R4wUmybqEg0j37gnaQeB4KX7jZTOKA8KkhTekLu9WYH7t5ZmuDUv9fF2pI0JeCowRrMF7A2ZLmuW63GyvqlUKvvcTeVXE8BGpJKOXjkBYAIKBzeg8sQvj9Ce7mhhURobYOXjJnT8\\\\\\\/Q22fdgZo7kPoBB2hxj\\\\\\\/m1Hguzoi5p2exXjr14GgSp2x6TQpXmZN5LZdaEa58RNPtgQPXY8ruOmiRQzOcjZiIXn2E+AwBPWA22KIersMt\\\\\\\/UlqzH97gm3JFhJbwV4U2tBtdzC6nDun8N6ISPjpUIjWkxvaP2UFqAH54JBWoYQsjagy4yetk7JZSx\\\\\\\/2+H27MQ0tHLJCjy27fIJxF9pPnGt0ptl9P0LcJ+RAdZHrskeKQp3Tb1XVgfLFJVT6tWkf5SgZI2cMwXljTmpIEtInPq+HX3G8sSPMqI6BySOjsjARpejZnf2ER1CLQLfBQ2h+jzjH6broT9UUnA1hME3NbnUnaBaHdEEpEwf1Jtgz+dVGd3PLUvy1v8PmYQpt33mSMuInNXy0NUc+j4goSVNPKL1S7yyyO5tW03C0z+UXxJQCJi+Ic\\\\\\\/ZtakwNZ12zamy0m1ZBvq6A49kQngP3pI1uqrw\\\\\\\/n2hmN7nO9X9tLZ4trTUJDfCEBuRWKjImZmJzcCkKOWB0tkN9vNSTAR\\\\\\\/9XztLGOsQuQvKEK8gt5L5lMpgyPmhcXt\\\\\\\/\\\\\\\/\\\\\\\/1f9WARjMeJoxSFEtwXZUyazfLjzE8VctdqLQBOYoY9keV3tKTw\\\\\\\/K4OE74ecSs4OtgRLXEEX3KyOcu6cXq9lCG2FrlyMeTU9CfKiPJufJoIZw+Dsc37mqj+Bk4Jgw7z1wQMrc8EAIJtzxRoVc81EQQCfzMUlzykJagIYKDKs6Aj4CAzaNB21mg1Aa5P8lo7OmK2k2LcxgcsufMct1G0Hbr5EoVHxSTKrlYNpXcOmKvyNjKgdvVft0h8F237N5PUyGiisdqOIGIX5F3LF3asBBgDGODZFUix2MdE2kl02pMcNkcIQUY1bpE\\\\\\\/KqSA8maAKUBAMss118FQJx8mey07qe52znljmZXGIIfh0zkAp4\\\\\\\/nzurReAaghak1dCe\\\\\\\/S7ZNvT8J5fWxw6vDwSIdcWMV9nIaDe8j6t\\\\\\\/Mq59xKzOqTYiJgL1d87wUvisERrpmryy3PLx+eVzE5OrAWNcKISL6Hkvo5t8+ddeQp0amZZ3J\\\\\\\/gOI2CjgtaPGhUmqmS815SKdSk7fGy4IhDVb0sSNf+IngsDoxWs6tlVt2ZRhSi1Y2h\\\\\\\/ZDseUYTETD2iD5riKdIqh\\\\\\\/7YLWsd\\\\\\\/8rqOLAavcMywZUg3CdGtLOH8wzwvffqrfZlUbQkNQ4lEufFFWDSiO++ayUPK05jMwfP4p3mEEN6QyqCojY+MjyRgy\\\\\\\/YDmBoYxMXMXnvmxjILjqwK\\\\\\\/F0BiHRWDw7xB22OQS+EKwxweobeHqRYAGy7rTTBRZ1uHUxuMmbxfTsQosuXKfn9Dks=\\\u0022,\\\u0022iv\\\u0022:\\\u0022xt65aE8Tae4IKEWMtbuXhw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b7f90fe9fe43d2db98a212d65c7a204b47b4d7d9e2a1ca076af3aee46c948393\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HMne8QP4GntoEqF\\\\\\\/\\\\\\\/gpsv5iAxBA+don\\\\\\\/Sq\\\\\\\/ZnYQnx9usbTMDEvbyCJkZgAI8tvWcpUrtQuX4woXIBBJKWmXj+6DgN9jl4w5dTn986LTZYQW\\\\\\\/K45eXYFRahfKaqFbkyiUvsBlhlA9rYeYoAhXcvV6CR56IUGrBm7ouanyX2W5mE8BbD7ZiDQwZ7tnpqU6dd3ox0nw7g7v\\\\\\\/b+v2QxRed9xKTHwHt2Ad5C0sI9\\\\\\\/Wx5dwi483HBB5QMIL709KZekPTp7yZ9yeU0pfcEWKw0SiDOGEWRNNea8af7K+EnAmQvEfUN9x5YIC9nhyeXE\\\\\\\/VD1xB7fO0IlzQloUPQ\\\\\\\/GljJEqjRVky2f4T02rPH3u2cmZ4q+M0Y2ynP2UfEWNrws7Q4K5U8AXwws3tCzDQyz4K36SwZuBO\\\\\\\/6Zd9VBh1LCNVoyN57Pli7ZSrXOLLdodLxyt3Q5HtTa0XdJVC8nH8WM3SXsaKJ3Ca5IqqmiDsFNmSR01U1jDsLe0bxNx9sqmKanZV7XL1oxlNSn6oHx2FgFspnwEkwBOQC0d4406JOI7ZubnMawbPlFH0kp1\\\\\\\/0jnSpZaF\\\\\\\/Fxq6mGltXhICmku6vd72+BZ\\\\\\\/rJhPO0DAx8oni4KBD6r1nrh6SAAHjtBNpXEqHtspvn0defInmKCY2QWuA6gEX227PUmX59\\\\\\\/R7RGOYLfHEdPjyMqCfr6jCareqm88cFzfAZtF5ctLpnW1G5SDnLpK0OZXaHr3lrBi2lxZ1Eug1+Nu6otwAaOLbso40fFMF9ZqfRvT+9+WhQrrtnciuwVYaX18P\\\\\\\/Mc3SO+X0B7X8s4nz6yTloqdj7i\\\\\\\/jnIdgt4xwuXQ8uVj6A4FYqg8E\\\\\\\/zA80GQzBIHCN0h5nGxlMvTF52HIajtlVnT6+edEGakHuuXjJdJIi1sbxpBm6cYBNvC7TBaHaF5U7gNucZk9y69bLER+H9vuo283wfTzEhA0\\\\\\\/fVPyGNh6L7dRKfOkIXduLxq7p3i5vO85SLtXV5RCTn6T7QlEtSK6VHxP0qskVlaT8z5oFZqjUE2AsBUKBF0jMU+SpR4CqId64q\\\\\\\/gOyh47+hhJERh4eLPjAi2VOBSKq0lMV2\\\\\\\/74a3pR23xVUbR3nd2hRDB0EaXzNHBkAQ\\\\\\\/fLK2Fo1xXnC72cmxd08d2hSvcIhHFJyeX58pjlds\\\\\\\/rkZC\\\\\\\/N1OFteaZHB4NvOfkh+oewfPt9A39SbWT\\\\\\\/Xo8qIeYjMQb5+SEgSEmcj5QnmtlMA6Zv1oBJmVxszqGYmgBEZBabHWYEeHIL\\\\\\\/kbSMv4mc\\\\\\\/7K1IthSNfQopegP14THCqc0YXZ+pB9ULdRFnS9NZx\\\\\\\/GoJ7De9dwLuitmeq1ADzr6NG+siEjQ+0bEogowoHPSs3a56+0HLxqO5GzawUJq0+GvatOBbQTOs6AZaQpMmM2FxKk8+1Bz71O6urBH\\\\\\\/68IZRYIuM\\\\\\\/Wj6F1z7bcXuEknef9VpEZJDYm6mynD4xFiIYDrp8EeAPWKak39RMkWc4+8yxkvX\\\\\\\/PKHH347nRQSMPHF4gSSQR5urPy+0y7Qkd8knszdI3k1Bw6SgtOUdoyUAWEFWlgpZoJNreMThxak2mcXUSKkYdHMJ1tBpVbFqtqal+lq\\\\\\\/\\\\\\\/7WgWZxDj1vWP76N87YMHq1IuKgI6Wi8UxQtl0wdsuIPFF0oxKovLLNp4LOVIhxhflQz8P9JN49T2ZmlfBXcpeSxVabeja8v75Li1EciKmNyqoGzlFwnUnglOaYuHcbJAZmHGrFhfhI9UPWvVZ7UGdOZD3\\\\\\\/USvB9k49gpEmCH9YQlFdyhqjQuULCtMNq34UzI4hniXMWTGG1I8WrfH+Apsc\\\\\\\/T6OURolz4a3P+QJlwJ\\\\\\\/d0KFIxtIrEFOjo84JHLkma8vN6mTEmPsoRrhACMrR8UKUTcm1\\\\\\\/JZCFBKAIRFzcufvq8+abBZ5JTG\\\\\\\/I2LwYpawUvxQ0l8cpabacAl+u849con8kXKoxk1FRyrhoAnC\\\\\\\/TsVRgA1f7Gt9VgXwDf7odXCWwlTKuTVxyczr3VHKcLOUhOSrzbWbQNkL1QULMcQCNwXyfskh3ZElXGiVagzX8t7HlTgd2jryOVeiFHPqpDJtJ38rrId90Wd13GepqmLd6JoxZYC5bB4bokKu9W2X3AG858HCgyBDsiCuatsCeU6mDirsyF46cp7Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022HlNOscti+N7HGuthEvOtDQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a5944547aca90058234a07bdf98480977561623b2c5b3ce1d88049a354c96bca\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459732 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2545" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Bk9TiD6sksAVd4E3i6vajzdPdoueag1rIQ8ySUqiMRA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2545" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rsais84JCQbZ0zntRUxFPehPz2a5UGm-UDqHJEct77U" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/12/2535" title="Regulation of Synaptic Transmission at the Caenorhabditis elegans M4 Neuromuscular Junction by an Antagonistic Relationship Between Two Calcium Channels" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/12/2553" title="Corrigendum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459732" id="node459732" data-pisa="ggg;4/12/2545" data-pisa-master="ggg;g3.114.014365" data-apath="/ggg/4/12/2545.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Correcting for Sequencing Error in Maximum Likelihood Phylogeny Inference</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mary K. Kuhner</span> and <span class="highwire-citation-author" data-delta="1">James McGill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2545-2552;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014365 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459732" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mary K. Kuhner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BK.%2BKuhner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuhner%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMary%2BK.%2BKuhner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mkkuhner@uw.com" class="" data-icon-position="" data-hide-link-title="0">mkkuhner@uw.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James McGill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BMcGill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGill%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BMcGill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/12/2545" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459732" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459732/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2545.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459732" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459732/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2545.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459732" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459732/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2545.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459732" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459732/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/12/2545.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459732" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459732/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Accurate phylogenies are critical to taxonomy as well as studies of speciation processes and other evolutionary patterns. Accurate branch lengths in phylogenies are critical for dating and rate measurements. Such accuracy may be jeopardized by unacknowledged sequencing error. We use simulated data to test a correction for DNA sequencing error in maximum likelihood phylogeny inference. Over a wide range of data polymorphism and true error rate, we found that correcting for sequencing error improves recovery of the branch lengths, even if the assumed error rate is up to twice the true error rate. Low error rates have little effect on recovery of the topology. When error is high, correction improves topological inference; however, when error is extremely high, using an assumed error rate greater than the true error rate leads to poor recovery of both topology and branch lengths. The error correction approach tested here was proposed in 2004 but has not been widely used, perhaps because researchers do not want to commit to an estimate of the error rate. This study shows that correction with an approximate error rate is generally preferable to ignoring the issue.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/sequencing-error" class="hw-term hw-article-keyword hw-article-keyword-sequencing-error" rel="nofollow">sequencing error</a></li><li class="kwd"><a href="/keyword/phylogeny-inference" class="hw-term hw-article-keyword hw-article-keyword-phylogeny-inference" rel="nofollow">phylogeny inference</a></li><li class="kwd"><a href="/keyword/maximum-likelihood-0" class="hw-term hw-article-keyword hw-article-keyword-maximum-likelihood" rel="nofollow">maximum likelihood</a></li></ul><p id="p-3">As originally developed, maximum likelihood (ML) phylogeny inference assumes that the data are known without error (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Felsenstein 1981</a>). A straightforward extension to incorporate a known sequencing error rate was proposed by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Felsenstein (2004)</a>. As far as we know it has never been implemented despite its obvious potential usefulness. We speculate that the correction has been neglected because it requires committing to an estimate of the error rate, and researchers fear that an inaccurate correction would be worse than no correction at all.</p><p id="p-4">We have implemented sequencing error correction in the PHYLIP (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Felsenstein 2005</a>) programs Dnaml and Dnamlk, which infer phylogenies via DNA-based ML with a molecular clock (Dnamlk) or without one (Dnaml). Using simulated data, we test both the usefulness of error correction in obtaining a correct phylogeny and its vulnerability to misstatement of the sequencing error rate.</p><p id="p-5">Topological accuracy of phylogenies is important for purposes such as taxonomic classification (see, for example, the discussion in <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">De Queiroz and Gauthier 1990</a>) and detection of cospeciation patterns (for example, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Machado <em>et al.</em> 2005</a>). Accuracy of branch lengths is critical in any use of phylogenies in the context of time, such as dating of key events (reviewed in <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Rutschmann 2006</a>), inference of mutation rates (for example, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Drummond <em>et al.</em> 2006</a>), and parameter inference via coalescent theory (for example, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Kuhner 2009</a>). Thus, sequencing error correction is broadly relevant to the use of ML phylogenetics. We also discuss its potential use in Bayesian phylogenetics.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sequencing error correction</h3><p id="p-6">In the standard DNA- or RNA-based ML algorithm, values stored at the tips of the tree indicate the probability of the observed data given the underlying true base. For example, in a no-error case when base A is observed, the four values stored will be (1,0,0,0), corresponding to a probability of 1 for (A observed|true base A) and a probability of 0 for (A observed|true base C, G, or T/U). These values form the basis for the peeling algorithm (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Felsenstein 1981</a>), which calculates probabilities working from the tips back to the root. Note that these numbers represent the probability of the observation, not the probability of the underlying base, and need not sum to 1; for example, a completely uninformative observation (missing data) corresponds to values of (1,1,1,1).</p><p id="p-7">Following <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Felsenstein (2004)</a>, we use a simple model of sequencing error in which a base is misread as a random different base with probability <em>Îµ</em> and this probability is the same across sequences and sites. Under this model, the probability of observing A given that the underlying base is A becomes 1 â <em>Îµ</em>, and the probability of observing each of C, G, or T, given that the underlying base is A, becomes <em>Îµ</em>/3. More complex models, including sequence-specific or site-specific error rates, can readily be derived by the same approach. (This discussion assumes DNA, but RNA is handled identically, substituting U for T.)</p><p id="p-8">Nucleotide ML algorithms often allow for the International Union of Pure and Applied Chemistry nucleotide ambiguity codes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Cornish-Bowden 1985</a>). To handle these, we must decide what an observation of, say, M (meaning that the base is ambiguous between A and C) implies. We assume that sequencing error generates incorrect calls of specific bases, which are then converted to ambiguity codes. (This relieves us from needing a separate model for the probability of observing M if the underlying base is A.) Under this assumption, if the true base is A, an observation of M could result either from a correct call of A (probability 1 â <em>Îµ</em>) or an erroneous call of C (probability <em>Îµ</em>/3) and therefore has probability 1 â 2<em>Îµ</em>/3, and similarly if the true base is C. If the true base is G, the chance of observing M is the chance of an erroneous call of G as either A or C, or 2<em>Îµ</em>/3, and similarly for a true base T. Similar reasoning gives tip values for the other IUPAC codes. A representative sample of these probabilities is given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465342/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465342/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Example probabilities for resolved nucleotides and IUPAC ambiguity codes under an error model</span><div class="sb-div caption-clear"></div></div></div><p id="p-10">An analogous approach could be used for amino acid ML algorithms. Application to codon-based algorithms would be more complex but is possible in principle.</p></div><div id="sec-3" class="subsection"><h3>Simulation design</h3><p id="p-11">We used the program <em>rantree.c</em> (J. Felsenstein, unpublished data) to create random clocklike branching-process trees of 20 tips for a given value of the tree-size scaling parameter <em>t</em>. The <em>t</em> parameter establishes the scaling of the tree: for example, the interval between the rootward and next-rootward splits has an expected mean length of <em>t</em>/2. We then used the program <em>rectreedna.c</em> (J. Felsenstein and M. Kuhner, unpublished data) to simulate 20,000 bp of DNA data per taxon on these trees using the Kimura 2-parameter model with transition/transversion rate 2.0. We inferred trees from these DNA data with versions of Dnaml and Dnamlk from PHYLIP 3.69 (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Felsenstein 2005</a>), which was augmented with the error correction described previously. For each value of <em>t</em>, 100 trees were simulated, and a no-error data set was made for each tree; with-error data sets were derived by independent addition of varying degrees of error to these 100 data sets.</p><p id="p-12">We considered a range of true values of <em>Îµ</em> and <em>t</em>, as shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, which also gives mean single-nucleotide polymorphisms (SNPs) per kilobase. The proportion of SNPs due to error rather than mutation can be roughly inferred by comparing the zero-error case to the others. We then inferred trees using no error correction and a range of error corrections bracketing the true value, or in cases where the true error was zero, from 0 to 10<sup>â2</sup>. We did not simulate the most unbalanced cases: for these sequence lengths, when <em>e</em> = <em>t</em> â 1000 inference is expected to be nearly impossible, and when <em>e</em> = <em>t</em>/1000 error will have almost no effect.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465458/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465458/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean SNPs per kilobase in data sets for each condition</span><div class="sb-div caption-clear"></div></div></div><p id="p-14">To compare the inferred trees to the true trees, we used the topology-only metric of Robinson and Foulds (âRF,â <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Robinson and Foulds 1981</a>) to assess topological correctness, and the branch-length metric of Robinson and Foulds (âRFL,â <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Robinson and Foulds 1979</a>) to assess correct recovery of branch lengths. Our simulation conditions were chosen to represent between-species phylogenies (hence the use of branching-process rather than coalescent trees) and to explore both fairly easy and more difficult phylogeny reconstructions. We used both RF and RFL because of the finding (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">M. K. Kuhner and J. Yamato 2014</a>) that branch-length tree comparison metrics are more informative for closely related trees (corresponding in this case to highly successful inferences) and topology-only metrics are more informative for discordant trees (relatively poor inferences).</p></div></div><div class="section results" id="sec-4"><h2 class="">Results</h2><p id="p-15">We show our main results as a series of figures relating actual and declared error to phylogenetic inference accuracy. In the upper panels, RF measures accuracy of the inferred topology only; an increase of 2 units indicates an average of 1 additional error per inferred tree. In the lower panels, RFL measures inference of topology and branch length; its values should be considered relative to the tree scaling factor <em>t</em>. (We did not rescale proportionate to <em>t</em> as in some plots this obscures readability.) Throughout, we observe that trees with <em>t</em> = 10<sup>â4</sup> (red lines) had poor topological accuracy due to a scarcity of SNPs, whereas trees with greater values of <em>t</em> were well inferred topologically (and had relatively more accurate branch lengths) unless error intervened.</p><p id="p-16">For comparison purposes, Sanger sequencing can achieve error rates down to 10<sup>â4</sup> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ewing and Green 1998</a>), while estimated error rates for next-generation sequencing methods vary between 10<sup>â3</sup> and 4 Ã 10<sup>â2</sup> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Glenn 2014</a>). Error rates can be further reduced from these levels by taking a consensus of multiple reads. Even in sequences with high read depth, errors can be introduced in assembly: the frequency of such errors depends on genome, algorithm, and the error rate of the underlying sequencing method (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Haiminen <em>et al.</em> 2011</a>).</p><p id="p-17"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows results when an error correction is erroneously imposed on data generated without error. Dnamlâs topology inference is robust across the range of declared error tested here, whereas Dnamlk sees some worsening of topological inference with high declared error values. Both algorithms see worsening of branch-length inference with increased declared error; this was least pronounced with the highly informative <em>t</em> = 10<sup>â2</sup> trees.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inference accuracy with true error 0. (A, C) Dnaml and (B, D) Dnamlk. (A, B) RF (topology only) and (C, D) RFL (topology and branch length). RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2043418801" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Inference accuracy with true error 0. (A, C) Dnaml and (B, D) Dnamlk. (A, B) RF (topology only) and (C, D) RFL (topology and branch length). RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds. &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2545/F1.medium.gif" width="440" height="333"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/12/2545/F1.medium.gif" width="440" height="333"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465371" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Inference accuracy with true error 0. (A, C) Dnaml and (B, D) Dnamlk. (A, B) RF (topology only) and (C, D) RFL (topology and branch length). RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds. </p><div class="sb-div caption-clear"></div></div></div><p id="p-19"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows results with a true error of 10<sup>â4</sup> miscalls per base. Topological accuracy was unaffected by error correction within the bounds tested here. Branch length inference was improved by error correction, particularly for the lower value of <em>t</em>. The apparent optimal value of the correction was somewhat higher than the actual error rate, and there was only modest worsening with overcorrection up to 2x the actual error rate. Both algorithms recovered branch lengths very close to the no-error case (dashed lines) using their optimal correction values. Dnamlk recovered branch lengths more successfully than Dnaml, as expected for data which match Dnamlkâs clock assumption.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inference accuracy with true error 10&#x2212;4. (A, C) Dnaml and (B, D) Dnamlk. (A, B) show RF (topology only) and (C, D) show RFL (topology and branch length). Dashed lines indicate performance with actual error 0 and declared error 0 for comparison. RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2043418801" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Inference accuracy with true error 10&lt;sup&gt;&#x2212;4&lt;/sup&gt;. (A, C) Dnaml and (B, D) Dnamlk. (A, B) show RF (topology only) and (C, D) show RFL (topology and branch length). Dashed lines indicate performance with actual error 0 and declared error 0 for comparison. RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2545/F2.medium.gif" width="440" height="323"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/12/2545/F2.medium.gif" width="440" height="323"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465475" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Inference accuracy with true error 10<sup>â4</sup>. (A, C) Dnaml and (B, D) Dnamlk. (A, B) show RF (topology only) and (C, D) show RFL (topology and branch length). Dashed lines indicate performance with actual error 0 and declared error 0 for comparison. RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows results with a true error of 10<sup>â3</sup> miscalls per base. Again, topological accuracy was unaffected, but branch length inference improved markedly with correction, especially for <em>t</em> = 10<sup>â4</sup>. Although branch lengths worsened with overcorrection in most cases, this was generally less severe than noncorrection.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inference accuracy with true error 10&#x2212;3. (A, C) Dnaml and (B, D) Dnamlk. (A, B) RF (topology only) and (C, D) RFL (topology and branch length). Dashed lines indicate performance with actual error 0 and declared error 0 for comparison. RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2043418801" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Inference accuracy with true error 10&lt;sup&gt;&#x2212;3&lt;/sup&gt;. (A, C) Dnaml and (B, D) Dnamlk. (A, B) RF (topology only) and (C, D) RFL (topology and branch length). Dashed lines indicate performance with actual error 0 and declared error 0 for comparison. RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2545/F3.medium.gif" width="440" height="329"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/12/2545/F3.medium.gif" width="440" height="329"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465305" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Inference accuracy with true error 10<sup>â3</sup>. (A, C) Dnaml and (B, D) Dnamlk. (A, B) RF (topology only) and (C, D) RFL (topology and branch length). Dashed lines indicate performance with actual error 0 and declared error 0 for comparison. RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">At a still higher error rate of 10<sup>â2</sup> miscalls per base, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows topological inference beginning to react negatively to overcorrection, especially in Dnamlk and with lower values of <em>t</em>. At this error rate, the effect of the correction on branch length inference is dramatic, with undercorrection worse than overcorrection in all cases. A novel pattern appears for <em>t</em> = 10<sup>â4</sup> (red line) in which branch length inference apparently continues to improve beyond the point where declared error equals true error.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inference accuracy with true error 10&#x2212;2. (A, C) Dnaml and (B, D) Dnamlk. (A, B) RF (topology only) and (C, D) RFL (topology and branch length). Dashed lines indicate performance with actual error 0 and declared error 0 for comparison. RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2043418801" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Inference accuracy with true error 10&lt;sup&gt;&#x2212;2&lt;/sup&gt;. (A, C) Dnaml and (B, D) Dnamlk. (A, B) RF (topology only) and (C, D) RFL (topology and branch length). Dashed lines indicate performance with actual error 0 and declared error 0 for comparison. RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2545/F4.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/12/2545/F4.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465350" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">Inference accuracy with true error 10<sup>â2</sup>. (A, C) Dnaml and (B, D) Dnamlk. (A, B) RF (topology only) and (C, D) RFL (topology and branch length). Dashed lines indicate performance with actual error 0 and declared error 0 for comparison. RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Finally, at the enormous error rate of 10<sup>â1</sup> miscalls per base, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows this tendency much more strongly. Topological inference worsens with both undercorrection and, especially, overcorrection. Branch length inference is very poor with undercorrection, decreases to a level close to the no-error case with correction, and then increases only slightly with overcorrection; but this apparently correct branch length inference is associated with increasingly poor topology inference.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inference accuracy with true error 10&#x2212;1. (A, C) Dnaml and (B, D) Dnamlk. (A, B) RF (topology only) and (C, D) RFL (topology and branch length). Dashed lines indicate performance with actual error 0 and declared error 0 for comparison. RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2043418801" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Inference accuracy with true error 10&lt;sup&gt;&#x2212;1&lt;/sup&gt;. (A, C) Dnaml and (B, D) Dnamlk. (A, B) RF (topology only) and (C, D) RFL (topology and branch length). Dashed lines indicate performance with actual error 0 and declared error 0 for comparison. RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2545/F5.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/12/2545/F5.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465440" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-26" class="first-child">Inference accuracy with true error 10<sup>â1</sup>. (A, C) Dnaml and (B, D) Dnamlk. (A, B) RF (topology only) and (C, D) RFL (topology and branch length). Dashed lines indicate performance with actual error 0 and declared error 0 for comparison. RF, topology-only metric of Robinson and Foulds; RFL, branch-length metric of Robinson and Foulds.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">We hypothesized that this pattern of apparently good branch length inference combined with increasingly poor topology inference represents inferred trees with extremely short branches and semi-randomized topologies due to interpretation of essentially all of the data as sequencing error. To test this, we plotted the mean length of the Dnaml inferred trees divided by the mean length of the trees on which the data were generated. These plots are shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, with the tree length ratio plotted on a natural-log scale. <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> makes it clear that branch lengths are fairly robust to error correction except when the true error is greater than <em>t</em>. For extreme error cases, however, mean tree length starts out much too high in the inferred trees, becomes correct around the point where declared error is equal to true error, and then becomes much too low with further correction. This confirms our understanding of the anomalous results in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figures 4</a> and <a id="xref-fig-5-2" class="xref-fig" href="#F5">5</a>: branch lengths appear good only because they are extremely short, and topology cannot be inferred because the ordering of these short branches is randomized.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ratio of inferred tree length to true tree length as a function of declared error. Dashed line indicates equality of inferred and true length. (A) True error 0.0. (B) True error 10&#x2212;4. (C) True error 10&#x2212;3. (D) True error 10&#x2212;2. (E) True error 10&#x2212;1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2043418801" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ratio of inferred tree length to true tree length as a function of declared error. Dashed line indicates equality of inferred and true length. (A) True error 0.0. (B) True error 10&lt;sup&gt;&#x2212;4&lt;/sup&gt;. (C) True error 10&lt;sup&gt;&#x2212;3&lt;/sup&gt;. (D) True error 10&lt;sup&gt;&#x2212;2&lt;/sup&gt;. (E) True error 10&lt;sup&gt;&#x2212;1&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2545/F6.medium.gif" width="374" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/12/2545/F6.medium.gif" width="374" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2545/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465388" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-28" class="first-child">Ratio of inferred tree length to true tree length as a function of declared error. Dashed line indicates equality of inferred and true length. (A) True error 0.0. (B) True error 10<sup>â4</sup>. (C) True error 10<sup>â3</sup>. (D) True error 10<sup>â2</sup>. (E) True error 10<sup>â1</sup>.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-5"><h2 class="">Discussion</h2><div id="sec-6" class="subsection"><h3>Error correction generally improves inference</h3><p id="p-29">In almost all of the cases examined, declaration of the true error rate produced the most accurate branch lengths; in a few cases with very low error, slight overcorrection was actually superior. Topology inference was less sensitive to error, but in cases in which it varied with error rate, declaration of the true error rate again produced the best results.</p><p id="p-30">Some improvement was seen even when the error rate was quite low (see <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), showing that the correction is not deadweight even for highly accurate sequences. However, the greater the error rate the greater the need for the correction. Somewhat to our surprise, even with an error rate of 0.01 errors per base, recovery of branch lengths with a correct declared error was only a little worse than the associated no-error case.</p></div><div id="sec-7" class="subsection"><h3>Aggressive overcorrection can lead to pathologies</h3><p id="p-31">A declared error of 0.2 errors/base, applied to data with an actual error of 0.1 errors/base and a low tree scaling (so that nonerror SNPs were rare), caused essentially all of the variable sites to be interpreted as error, resulting in extremely short branch lengths and randomized topologies. In general, overcorrection appeared problematic when <em>Îµ</em> exceeded <em>t</em>, but we did not test intermediate values of <em>Îµ</em> so this cannot be treated as an exact rule.</p></div><div id="sec-8" class="subsection"><h3>Type of inference matters</h3><p id="p-32">When error was low, the clock-assuming program Dnamlk produced more accurate branch lengths than Dnaml, presumably because it has fewer degrees of freedom in branch length inference. However, as error increased Dnamlk showed increasing difficulties. The error correction implementation is identical in both programs, but in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> Dnamlk performed substantially worse than Dnaml. This is particularly striking in that these data were simulated with a molecular clock. The clock constraint means that sufficient distortion of branch lengths can lead to an incorrect topology. For phylogenetic inference in the context of very dirty data, non-clocklike methods are more robust and should be preferred unless the clock assumption is essential (for example, in phylogenetic dating).</p><p id="p-33">The results presented here involve ML inference. Bayesian inference with a reasonable prior on <em>Îµ</em> might be able to tolerate higher declared error rates than ML for two reasons: the prior could help to direct attention away from zero-length trees, and even if near-zero trees were produced they would be accompanied by longer trees and the credibility intervals would therefore reflect topological signal. Presenting the single ML tree, in contrast, foregrounds the âall SNPs are errorsâ solution even though more resolved trees would have only slightly lower likelihood. This error correction should be tested in Bayesian inference programs with <em>Îµ</em> as a parameter. It would also be interesting to see whether the relaxed clock approach of <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Drummond <em>et al.</em> (2006)</a> could be combined with sequencing error correction to allow successful inference of clocklike trees in the presence of extreme sequencing error.</p></div><div id="sec-9" class="subsection"><h3>Sequencing error may not be constant</h3><p id="p-34">We have assumed that all sequences in a data set have the same error rate. This will not always be true in practice due to variation in sequencing methods, read depth, sample condition, and other factors. If the error rates of various sequences are known, it is trivial to set a separate <em>Îµ</em> for each one: we plan to add this capability to upcoming versions of our software. When the rates are unknown, any constant value of <em>Îµ</em> will be wrong for some sequences. Our results suggest that tree distortion is likely if the unknown values vary by more than a factor of two. Novel analytic tools will be needed to correct the inference in such cases. In the meantime, it is likely that even use of an average correction which is wrong for some sequences will be better than use of no correction, which is wrong for all sequences.</p><p id="p-35">We have also assumed that errors occur independently across the length of the sequence. This is not necessarily true: for example, assembly-based errors may tend to occur in clusters where the wrong fragment has been chosen. For conventional phylogenetic applications like the ones studied here, the main effect of this clustering is to introduce additional stochastic variability into the error rate per sequence, and the comments above apply. Clustered errors are a greater concern for models in which site locations are significant: codon likelihood models, models of secondary structure, and ancestral recombination graphs. Further work will be needed to appropriately handle clustered sequencing error in such models.</p><p id="p-36">We find that the sequencing error correction for ML analysis proposed by <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Felsenstein (2004)</a> improves recovery of branch lengths across a wide range of parameter values. In general, overcorrection does less harm than undercorrection. For any but the most extreme sequencing error rates, there is little effect of error correction on topological inference. When the error rate is extremely high, however, values of the declared error different from the true error are poorly tolerated in both branch length inference and topology inference, with undercorrection particularly detrimental to branch length inference and overcorrection to topological inference.</p><p id="p-37">We strongly recommend incorporation of sequencing error correction into ML phylogeny algorithms. In human SNP typing, it has been common to rely on allele frequency cutoffs as a correction for sequencing error (since independent errors generally manifest as singleton SNPs), but <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">McGill <em>et al.</em> (2013)</a> have shown that the sequencing error correction used here is more statistically powerful than omission of rare SNPs, which also removes considerable signal from the data. While this study did not test the usefulness of specifying sequence-specific error rates, it is very likely that a best-practices implementation should include this capability.</p><p id="p-38">It will also be important for the creators of sequencing and assembly pipelines to develop and publish accurate estimates of their error rates, in order to facilitate correct analysis of the resulting data. Although inference of the sequencing error rate from phylogenetic data might be possible, direct measurement will be more powerful and should be prioritized. When a study contains sequences of very different quality (ancient <em>vs.</em> modern DNA, model <em>vs.</em> nonmodel organisms, shallow <em>vs.</em> deep read depth, different sequencing platform) it will be important to develop and use sequence-specific error rates.</p></div></div><div class="section other1" id="sec-10"><h2 class="">Software Availability</h2><p id="p-39">The sequencing error correction described here has been implemented in the LAMARC program since version 2.1.5. It will also be implemented in the next release of PHYLIP. The unpublished programs <em>rantree.c</em> and <em>rectreedna.c</em> used to simulate and analyze data are archived in Supplementary Materials <a href="http://www.g3journal.org/content/suppl/2014/11/04/g3.114.014365.DC1/FileS1.tar">File S1</a>, along with example parameter files.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">This paper was inspired by simulations performed by Elizabeth Walkup using the LAMARC program, and her advice was extremely helpful in its preparation. We thank Jon Yamato for programming and editing assistance and Joe Felsenstein for conceptual advice and comments on the manuscript. This work was funded by National Science Foundation grant DEB-1256731 (to M.K.K.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-41">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.014365/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.014365/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-42"><em>Communicating editor: A. Rokas</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-09-09"><span class="received-label">Received </span>September 9, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-11-03"><span class="accepted-label">Accepted </span>November 3, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2014 Kuhner and McGill</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.1" data-doi="10.1093/nar/13.9.3021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cornish-Bowden</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span>â<span class="cit-article-title">Nomenclature for incompletely specified bases in nucleic acid sequences: recommendations 1984.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">3021</span>â<span class="cit-lpage">3030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DCornish-Bowden%26rft.auinit1%253DA.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D3021%26rft.epage%253D3030%26rft.atitle%253DNomenclature%2Bfor%2Bincompletely%2Bspecified%2Bbases%2Bin%2Bnucleic%2Bacid%2Bsequences%253A%2Brcommendations%2B1984%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F13.9.3021%26rft_id%253Dinfo%253Apmid%252F2582368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJuYXIiO3M6NToicmVzaWQiO3M6OToiMTMvOS8zMDIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.2" data-doi="10.2307/2992353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Queiroz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauthier</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">Phylogeny as a central principle in taxonomy: phylogenetic definitions of taxon names.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">307</span>â<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2992353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjM5LzQvMzA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTQ1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.3" data-doi="10.1371/journal.pbio.0040088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">S. Y. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Relaxed phylogenetics and dating with confidence.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDrummond%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De88%26rft.epage%253De88%26rft.atitle%253DRelaxed%2Bphylogenetics%2Band%2Bdating%2Bwith%2Bconfidence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040088%26rft_id%253Dinfo%253Apmid%252F16683862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16683862&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.4" data-doi="10.1101/gr.8.3.186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Base-calling of automated sequencer traces using Phred. II. Error probabilities.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">186</span>â<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.186%26rft_id%253Dinfo%253Apmid%252F9521922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/gr.8.3.186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9521922&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.5" data-doi="10.1007/BF01734359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span>â<span class="cit-article-title">Evolutionary trees from DNA sequences: a maximum likelihood approach.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">368</span>â<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D368%26rft.epage%253D376%26rft.atitle%253DEvolutionary%2Btrees%2Bfrom%2BDNA%2Bsequences%253A%2Ba%2Bmaximum%2Blikelihood%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01734359%26rft_id%253Dinfo%253Apmid%252F7288891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01734359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7288891&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MG91100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-book" id="cit-4.12.2545.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-source">Inferring Phylogenies</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-book" id="cit-4.12.2545.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-source">PHYLIP (Phylogeny Inference Package) version 3.6. Distributed by the author</span>. <span class="cit-publ-name">Department of Genome Sciences, University of Washington</span>, <span class="cit-publ-loc">Seattle</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-web" id="cit-4.12.2545.8"><div class="cit-metadata unstructured">Glenn, T. C., 2014. <em>2014 NGS Field Guide</em>. Available at: <a href="http://www.molecularecologist.com/next-gen-table-3c-2014/">www.molecularecologist.com/next-gen-table-3c-2014/</a>. Accessed: October 27, 2014.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haiminen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parida</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rigoutsos</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Evaluation of methods for <em>d</em>e novo genome assembly from high-throughput sequencing reads reveals dependencies that affect the quality of the results.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e24182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De24182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.10" data-doi="10.1016/j.tree.2008.09.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhner</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Coalescent genealogy samplers: windows into population history.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">86</span>â<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DKuhner%26rft.auinit1%253DM.%2BK.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D86%26rft.epage%253D93%26rft.atitle%253DCoalescent%2Bgenealogy%2Bsamplers%253A%2Bwindows%2Binto%2Bpopulation%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2008.09.007%26rft_id%253Dinfo%253Apmid%252F19101058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2008.09.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19101058&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2545.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263396900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhner</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamato</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Practical performance of tree comparison metrics</span>. <abbr class="cit-jnl-abbrev">Sys Biol</abbr> <span class="cit-comment">doi: 10.1093/sysbio/syu085</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.12" data-doi="10.1073/pnas.0501840102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Machado</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robbins</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">M. T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herre</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Critical review of host specificity and its coevolutionary implications in the fig/fig wasp mutualism.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>(<span class="cit-supplement">Suppl 1</span>): <span class="cit-fpage">6558</span>â<span class="cit-lpage">6565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0501840102%26rft_id%253Dinfo%253Apmid%252F15851680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxNjoiMTAyL3N1cHBsXzEvNjU1OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjU0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.13" data-doi="10.1534/genetics.112.146738"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGill</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walkup</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhner</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Correcting coalescent analyses for panel-based SNP ascertainment.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1185</span>â<span class="cit-lpage">1196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146738%26rft_id%253Dinfo%253Apmid%252F23335334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzExODUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI1NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.14" data-doi="10.1007/BFb0102690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span>â<span class="cit-article-title">Comparison of weighted labeled trees.</span> <abbr class="cit-jnl-abbrev">Lecture Notes Mathematics</abbr> <span class="cit-vol">748</span>: <span class="cit-fpage">119</span>â<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLecture%2BNotes%2BMathematics%26rft.volume%253D748%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBFb0102690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BFb0102690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.15" data-doi="10.1016/0025-5564(81)90043-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span>â<span class="cit-article-title">Comparison of phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">131</span>â<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMath.%2BBiosci.%26rft.volume%253D53%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0025-5564%252881%252990043-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0025-5564(81)90043-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1981LB66300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.12.2545.16" data-doi="10.1111/j.1366-9516.2006.00210.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutschmann</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Molecular dating of phylogenetic trees: a brief review of current methods that estimate divergence times.</span> <abbr class="cit-jnl-abbrev">Divers. Distrib.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">35</span>â<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDivers.%2BDistrib.%26rft.volume%253D12%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1366-9516.2006.00210.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1366-9516.2006.00210.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000234026600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/12/2545.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/12/2535" title="Regulation of Synaptic Transmission at the Caenorhabditis elegans M4 Neuromuscular Junction by an Antagonistic Relationship Between Two Calcium Channels" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/12/2553" title="Corrigendum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/12"><span>Volume 4 Issue 12, December 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/12.cover-source.jpg?itok=ezho2p8P" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2545" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-33crBF3BP_ldAKGb3DSc8R2xLAVwEa-V5E-e-AnHO8o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/12/2545" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/12/2545" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/12/2545" class="active" data-icon-position="" data-hide-link-title="0">Correcting for Sequencing Error in Maximum Likelihood Phylogeny Inference</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459732" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-EKa70AAwRLQDsnIFn_7rBfbmJbRkpRZoJve1yXpejZ4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/12/2545.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/12/2545" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-tJqW-eqk7bvWLYo8ckDmHK-VY-JHdDEC9cj3xAQxKKs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/12/2545" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/12/2545.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459732" id="node459732--2" data-pisa="ggg;4/12/2545" data-pisa-master="ggg;g3.114.014365" data-apath="/ggg/4/12/2545.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Correcting for Sequencing Error in Maximum Likelihood Phylogeny Inference</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mary K. Kuhner</span> and <span class="highwire-citation-author" data-delta="1">James McGill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2545-2552;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014365 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459732--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mary K. Kuhner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BK.%2BKuhner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuhner%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMary%2BK.%2BKuhner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mkkuhner@uw.com" class="" data-icon-position="" data-hide-link-title="0">mkkuhner@uw.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James McGill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BMcGill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGill%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BMcGill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/12/2545" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/12/2545&amp;title=Correcting%20for%20Sequencing%20Error%20in%20Maximum%20Likelihood%20Phylogeny%20Inference" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/12/2545&amp;title=Correcting%20for%20Sequencing%20Error%20in%20Maximum%20Likelihood%20Phylogeny%20Inference" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/12/2545&amp;title=Correcting%20for%20Sequencing%20Error%20in%20Maximum%20Likelihood%20Phylogeny%20Inference" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/12/2545&amp;text=Correcting%20for%20Sequencing%20Error%20in%20Maximum%20Likelihood%20Phylogeny%20Inference" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/12/2545&amp;title=Correcting%20for%20Sequencing%20Error%20in%20Maximum%20Likelihood%20Phylogeny%20Inference" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/12/2545&amp;t=Correcting%20for%20Sequencing%20Error%20in%20Maximum%20Likelihood%20Phylogeny%20Inference" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/12/2545&amp;title=Correcting%20for%20Sequencing%20Error%20in%20Maximum%20Likelihood%20Phylogeny%20Inference" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/12/2545&amp;title=Correcting%20for%20Sequencing%20Error%20in%20Maximum%20Likelihood%20Phylogeny%20Inference" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459732/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459732" id="node459732--3" data-pisa="ggg;4/12/2545" data-pisa-master="ggg;g3.114.014365" data-apath="/ggg/4/12/2545.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Correcting for Sequencing Error in Maximum Likelihood Phylogeny Inference</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mary K. Kuhner</span> and <span class="highwire-citation-author" data-delta="1">James McGill</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2545-2552;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014365 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459732--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mary K. Kuhner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BK.%2BKuhner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuhner%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMary%2BK.%2BKuhner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mkkuhner@uw.com" class="" data-icon-position="" data-hide-link-title="0">mkkuhner@uw.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James McGill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195-5065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BMcGill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGill%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BMcGill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2545&amp;rft.epage=2552&amp;rft.atitle=Correcting%20for%20Sequencing%20Error%20in%20Maximum%20Likelihood%20Phylogeny%20Inference&amp;rft.volume=4&amp;rft.issue=12&amp;rft.date=2014-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kuhner%2C+Mary+K.&amp;rft.au=McGill%2C+James"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459732/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459732/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459732/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459732/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459732/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459732/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459732/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459732/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459732/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459732/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459732/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459732/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/12/2545" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Software Availability</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/12/2545.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/12/2545.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/12/2545.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/12/2545.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
