<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/12/2433" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/12/2433.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/12/2433.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/12/2433.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/12/2433/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/12/2433.atom" />
<meta name="HW.pisa" content="ggg;4/12/2433" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Evidence for Stabilizing Selection on Codon Usage in Chromosomal Rearrangements of Drosophila pseudoobscura" />
<meta name="DC.Identifier" content="10.1534/g3.114.014860" />
<meta name="DC.Date" content="2014-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Fuller et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="There has been a renewed interest in investigating the role of stabilizing selection acting on genome-wide traits such as codon usage bias. Codon bias, when synonymous codons are used at unequal frequencies, occurs in a wide variety of taxa. Standard evolutionary models explain the maintenance of codon bias through a balance of genetic drift, mutation and weak purifying selection. The efficacy of selection is expected to be reduced in regions of suppressed recombination. Contrary to observations in Drosophila melanogaster , some recent studies have failed to detect a relationship between the recombination rate, intensity of selection acting at synonymous sites, and the magnitude of codon bias as predicted under these standard models. Here, we examined codon bias in 2798 protein coding loci on the third chromosome of D. pseudoobscura using whole-genome sequences of 47 individuals, representing five common third chromosome gene arrangements. Fine-scale recombination maps were constructed using more than 1 million segregating sites. As expected, recombination was demonstrated to be significantly suppressed between chromosome arrangements, allowing for a direct examination of the relationship between recombination, selection, and codon bias. As with other Drosophila species, we observe a strong mutational bias away from the most frequently used codons. We find the rate of synonymous and nonsynonymous polymorphism is variable between different amino acids. However, we do not observe a reduction in codon bias or the strength of selection in regions of suppressed recombination as expected. Instead, we find that the interaction between weak stabilizing selection and mutational bias likely plays a role in shaping the composition of synonymous codons across the third chromosome in D. pseudoobscura ." />
<meta name="DC.Contributor" content="Zachary L. Fuller" />
<meta name="DC.Contributor" content="Gwilym D. Haynes" />
<meta name="DC.Contributor" content="Dianhui Zhu" />
<meta name="DC.Contributor" content="Matthew Batterton" />
<meta name="DC.Contributor" content="Hsu Chao" />
<meta name="DC.Contributor" content="Shannon Dugan" />
<meta name="DC.Contributor" content="Mehwish Javaid" />
<meta name="DC.Contributor" content="Joy C. Jayaseelan" />
<meta name="DC.Contributor" content="Sandra Lee" />
<meta name="DC.Contributor" content="Mingmei Li" />
<meta name="DC.Contributor" content="Fiona Ongeri" />
<meta name="DC.Contributor" content="Sulan Qi" />
<meta name="DC.Contributor" content="Yi Han" />
<meta name="DC.Contributor" content="Harshavardhan Doddapaneni" />
<meta name="DC.Contributor" content="Stephen Richards" />
<meta name="DC.Contributor" content="Stephen W. Schaeffer" />
<meta name="article:published_time" content="2014-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Evidence for Stabilizing Selection on Codon Usage in Chromosomal Rearrangements of Drosophila pseudoobscura" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;There has been a renewed interest in investigating the role of stabilizing selection acting on genome-wide traits such as codon usage bias. Codon bias, when synonymous codons are used at unequal frequencies, occurs in a wide variety of taxa. Standard evolutionary models explain the maintenance of codon bias through a balance of genetic drift, mutation and weak purifying selection. The efficacy of selection is expected to be reduced in regions of suppressed recombination. Contrary to observations in &lt;i&gt;Drosophila melanogaster&lt;/i&gt;, some recent studies have failed to detect a relationship between the recombination rate, intensity of selection acting at synonymous sites, and the magnitude of codon bias as predicted under these standard models. Here, we examined codon bias in 2798 protein coding loci on the third chromosome of &lt;i&gt;D. pseudoobscura&lt;/i&gt; using whole-genome sequences of 47 individuals, representing five common third chromosome gene arrangements. Fine-scale recombination maps were constructed using more than 1 million segregating sites. As expected, recombination was demonstrated to be significantly suppressed between chromosome arrangements, allowing for a direct examination of the relationship between recombination, selection, and codon bias. As with other &lt;i&gt;Drosophila&lt;/i&gt; species, we observe a strong mutational bias away from the most frequently used codons. We find the rate of synonymous and nonsynonymous polymorphism is variable between different amino acids. However, we do not observe a reduction in codon bias or the strength of selection in regions of suppressed recombination as expected. Instead, we find that the interaction between weak stabilizing selection and mutational bias likely plays a role in shaping the composition of synonymous codons across the third chromosome in &lt;i&gt;D. pseudoobscura&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/12/01" />
<meta name="citation_mjid" content="ggg;4/12/2433" />
<meta name="citation_id" content="4/12/2433" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/12/2433" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/12/2433.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/12/2433.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/12/2433.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.014860" />
<meta name="citation_pmid" content="25326424" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2433" />
<meta name="citation_lastpage" content="2449" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zachary L. Fuller" />
<meta name="citation_author_institution" content="Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301" />
<meta name="citation_author_email" content="zlf105@psu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4765-9227" />
<meta name="citation_author" content="Gwilym D. Haynes" />
<meta name="citation_author_institution" content="Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301" />
<meta name="citation_author" content="Dianhui Zhu" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Matthew Batterton" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Hsu Chao" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Shannon Dugan" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Mehwish Javaid" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Joy C. Jayaseelan" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Sandra Lee" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Mingmei Li" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Fiona Ongeri" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Sulan Qi" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Yi Han" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Harshavardhan Doddapaneni" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Stephen Richards" />
<meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author" content="Stephen W. Schaeffer" />
<meta name="citation_author_institution" content="Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_title=Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy.;citation_pages=927-935;citation_volume=136;citation_year=1994;citation_issue=3;citation_pmid=8005445" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_title=Inferring weak selection from patterns of polymorphism and divergence at &quot;silent&quot; sites in Drosophila DNA.;citation_pages=1067-1076;citation_volume=139;citation_year=1995;citation_issue=2;citation_pmid=7713409" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_author=S. W. Schaeffer;citation_title=Natural Selection and the Frequency Distributions of ``Silent'' DNA Polymorphism in Drosophila;citation_pages=295-307;citation_volume=146;citation_year=1997;citation_issue=1;citation_pmid=9136019" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=P. Andolfatto;citation_author=F. Depaulis;citation_author=A. Navarro;citation_title=Inversion polymorphisms and nucleotide variability in Drosophila.;citation_pages=1-8;citation_volume=77;citation_year=2001;citation_issue=1;citation_pmid=11279826;citation_doi=10.1017/S0016672301004955" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=P. Andolfatto;citation_author=K. M. Wong;citation_author=D. Bachtrog;citation_title=Effective population size and the efficacy of selection on the x chromosomes of two closely related Drosophila species.;citation_volume=3;citation_year=2011;citation_pmid=21173424;citation_doi=10.1093/gbe/evq086" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. R. Arguello;citation_author=Y. Zhang;citation_author=T. Kado;citation_author=C. Fan;citation_author=R. Zhao;citation_title=Recombination yet inefficient selection along the Drosophila melanogaster subgroup’s fourth chromosome.;citation_volume=27;citation_year=2010;citation_pmid=20008457;citation_doi=10.1093/molbev/msp291" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=D. Ayala;citation_author=R. F. Guerrero;citation_author=M. Kirkpatrick;citation_title=Reproductive isolation and local adaptation quantified for a chromosome inversion in a malaria mosquito.;citation_pages=946-958;citation_volume=67;citation_year=2013;citation_pmid=23550747;citation_doi=10.1111/j.1558-5646.2012.01836.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Bachtrog;citation_title=Protein Evolution and Codon Usage Bias on the Neo-Sex Chromosomes of Drosophila miranda;citation_pages=1221-1232;citation_volume=165;citation_year=2003;citation_issue=3;citation_pmid=14668377" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=D. Bachtrog;citation_title=Reduced selection for codon usage bias in Drosophila miranda.;citation_pages=586-590;citation_volume=64;citation_year=2007;citation_issue=5;citation_pmid=17457633;citation_doi=10.1007/s00239-006-0257-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N.-K. S. Basatena;citation_author=C. J. Hoggart;citation_author=L. J. Coin;citation_author=P. F. O’Reilly;citation_title=The effect of genomic inversions on estimation of population genetic parameters from sNP data.;citation_volume=193;citation_year=2013;citation_pmid=23150602;citation_doi=10.1534/genetics.112.145599" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. J. Begun;citation_author=C. F. Aquadro;citation_title=Levels of naturally occurring DNA polymorphism correlate with recombination rates in D. melanogaster.;citation_pages=519-520;citation_volume=356;citation_year=1992;citation_issue=6369;citation_pmid=1560824;citation_doi=10.1038/356519a0" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=A. J. Betancourt;citation_author=J. J. Welch;citation_author=B. Charlesworth;citation_title=Reduced effectiveness of selection caused by a lack of recombination.;citation_pages=655-660;citation_volume=19;citation_year=2009;citation_issue=8;citation_pmid=19285399;citation_doi=10.1016/j.cub.2009.02.039" />
<meta name="citation_reference" content="citation_title=Bioinformatics for Geneticists;citation_title=Amino acid properties and consequences of substitutions;citation_author=M. J. Betts;citation_author=R. B. Russell;citation_pages=289-316;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Bulmer;citation_title=The selection-mutation-drift theory of synonymous codon usage.;citation_pages=897-907;citation_volume=129;citation_year=1991;citation_issue=3;citation_pmid=1752426" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. Campos;citation_author=K. Zeng;citation_author=D. Parker;citation_author=B. Charlesworth;citation_author=P. Haddrill;citation_title=Codon usage bias and effective population sizes on the X chromosome vs. the autosomes in Drosophila melanogaster.;citation_volume=30;citation_year=2013;citation_pmid=23204387;citation_doi=10.1093/molbev/mss222" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. A. Capra;citation_author=M. J. Hubisz;citation_author=D. Kostka;citation_author=K. S. Pollard;citation_author=A. Siepel;citation_title=A model-based analysis of GC-biased gene conversion in the human and chimpanzee genomes.;citation_pages=e1003684;citation_volume=9;citation_year=2013;citation_pmid=23966869;citation_doi=10.1371/journal.pgen.1003684" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. B. Carlini;citation_author=W. Stephan;citation_title=In Vivo Introduction of Unpreferred Synonymous Codons Into the Drosophila Adh Gene Results in Reduced Levels of ADH Protein;citation_pages=239-243;citation_volume=163;citation_year=2003;citation_issue=1;citation_pmid=12586711" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. H. Chan;citation_author=P. A. Jenkins;citation_author=Y. S. Song;citation_title=Genome-wide fine-scale recombination rate variation in Drosophila melanogaster.;citation_pages=e1003090;citation_volume=8;citation_year=2012;citation_pmid=23284288;citation_doi=10.1371/journal.pgen.1003090" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. Charlesworth;citation_title=Fundamental concepts in genetics: effective population size and patterns of molecular evolution and variation.;citation_pages=195-205;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19204717;citation_doi=10.1038/nrg2526" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Charlesworth;citation_title=Stabilizing selection, purifying selection, and mutational bias in finite populations.;citation_volume=194;citation_year=2013;citation_pmid=23709636;citation_doi=10.1534/genetics.113.151555" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=B. Charlesworth;citation_author=D. Charlesworth;citation_title=Selection of new inversions in multi-locus genetic systems.;citation_pages=167-183;citation_volume=21;citation_year=1973;citation_doi=10.1017/S0016672300013343" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=N. De la Chaux;citation_author=P. W. Messer;citation_author=P. F. Arndt;citation_title=DNA indels in coding regions reveal selective constraints on protein evolution in the human lineage.;citation_pages=191-191;citation_volume=7;citation_year=2007;citation_pmid=17935613;citation_doi=10.1186/1471-2148-7-191" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=E. T. Cirulli;citation_author=R. M. Kliman;citation_author=M. A. F. Noor;citation_title=Fine-scale crossover rate heterogeneity in Drosophila pseudoobscura.;citation_pages=129-135;citation_volume=64;citation_year=2007;citation_issue=1;citation_pmid=17160365;citation_doi=10.1007/s00239-006-0142-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Comeron;citation_author=M. Kreitman;citation_title=Population, Evolutionary and Genomic Consequences of Interference Selection;citation_pages=389-410;citation_volume=161;citation_year=2002;citation_issue=1;citation_pmid=12019253" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Comeron;citation_author=M. Kreitman;citation_author=M. Aguadé;citation_title=Natural Selection on Synonymous Sites Is Correlated With Gene Length and Recombination in Drosophila;citation_pages=239-249;citation_volume=151;citation_year=1999;citation_issue=1;citation_pmid=9872963" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. B. Corbett-Detig;citation_author=D. L. Hartl;citation_title=Population genomics of inversion polymorphisms in Drosophila melanogaster.;citation_pages=e1003056;citation_volume=8;citation_year=2012;citation_pmid=23284285;citation_doi=10.1371/journal.pgen.1003056" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol. CB;citation_author=A. D. Cutter;citation_author=B. Charlesworth;citation_title=Selection intensity on preferred codons correlates with overall codon usage bias in Caenorhabditis remanei.;citation_pages=2053-2057;citation_volume=16;citation_year=2006;citation_doi=10.1016/j.cub.2006.08.067" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Dobzhansky;citation_title=GENETICS OF NATURAL POPULATIONS. XIX. ORIGIN OF HETEROSIS THROUGH NATURAL SELECTION IN POPULATIONS OF DROSOPHILA PSEUDOOBSCURA;citation_pages=288-302;citation_volume=35;citation_year=1950;citation_issue=3;citation_pmid=15414931" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Dobzhansky;citation_author=M. L. Queal;citation_title=GENETICS OF NATURAL POPULATIONS. II. GENIC VARIATION IN POPULATIONS OF DROSOPHILA PSEUDOOBSCURA INHABITING ISOLATED MOUNTAIN RANGES;citation_pages=463-484;citation_volume=23;citation_year=1938;citation_issue=5;citation_pmid=17246895" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Dobzhansky;citation_author=A. H. Sturtevant;citation_title=INVERSIONS IN THE CHROMOSOMES OF DROSOPHILA PSEUDOOBSCURA;citation_pages=28-64;citation_volume=23;citation_year=1938;citation_issue=1;citation_pmid=17246876" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=T. Dobzhansky;citation_author=C. Epling;citation_title=The Suppression of Crossing Over in Inversion Heterozygotes of Drosophila Pseudoobscura;citation_pages=137-141;citation_volume=34;citation_year=1948;citation_issue=4;citation_pmid=16588794;citation_doi=10.1073/pnas.34.4.137" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. A. Drummond;citation_author=C. O. Wilke;citation_title=Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution.;citation_pages=341-352;citation_volume=134;citation_year=2008;citation_issue=2;citation_pmid=18662548;citation_doi=10.1016/j.cell.2008.05.042" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=L. Duret;citation_author=N. Galtier;citation_title=Biased gene conversion and the evolution of mammalian genomic landscapes.;citation_pages=285-311;citation_volume=10;citation_year=2009;citation_pmid=19630562;citation_doi=10.1146/annurev-genom-082908-150001" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Eyre-Walker;citation_author=L. D. Hurst;citation_title=The evolution of isochores.;citation_pages=549-555;citation_volume=2;citation_year=2001;citation_issue=7;citation_pmid=11433361;citation_doi=10.1038/35080577" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=K. J. Fryxell;citation_author=E. Zuckerkandl;citation_title=Cytosine Deamination Plays a Primary Role in the Evolution of Mammalian Isochores;citation_pages=1371-1383;citation_volume=17;citation_year=2000;citation_issue=9;citation_pmid=10958853;citation_doi=10.1093/oxfordjournals.molbev.a026420" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. I. Gossmann;citation_author=M. Woolfit;citation_author=A. Eyre-Walker;citation_title=Quantifying the variation in the effective population size within a genome.;citation_volume=189;citation_year=2011;citation_pmid=21954163;citation_doi=10.1534/genetics.111.132654" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=P. R. Haddrill;citation_author=D. L. Halligan;citation_author=D. Tomaras;citation_author=B. Charlesworth;citation_title=Reduced efficacy of selection in regions of the Drosophila genome that lack crossing over.;citation_pages=R18-R18;citation_volume=8;citation_year=2007;citation_issue=2;citation_pmid=17284312;citation_doi=10.1186/gb-2007-8-2-r18" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. R. Haddrill;citation_author=K. Zeng;citation_author=B. Charlesworth;citation_title=Determinants of synonymous and nonsynonymous variability in three species of Drosophila.;citation_volume=28;citation_year=2011;citation_pmid=21191087;citation_doi=10.1093/molbev/msq354" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. Han;citation_author=J. S. Sinsheimer;citation_author=J. Novembre;citation_title=Characterizing bias in population genetic inferences from low coverage sequencing data.;citation_pages=723-735;citation_volume=31;citation_year=2013;citation_pmid=24288159" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Hense;citation_author=N. Anderson;citation_author=S. Hutter;citation_author=W. Stephan;citation_author=J. Parsch;citation_title=Experimentally increased codon bias in the Drosophila Adh gene leads to an increase in larval, but not adult, alcohol dehydrogenase activity.;citation_volume=184;citation_year=2010;citation_pmid=19966063;citation_doi=10.1534/genetics.109.111294" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=Selection on codon bias.;citation_pages=287-299;citation_volume=42;citation_year=2008;citation_pmid=18983258;citation_doi=10.1146/annurev.genet.42.110807.091442" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=Evidence that mutation is universally biased towards AT in bacteria.;citation_pages=e1001115;citation_volume=6;citation_year=2010;citation_pmid=20838599;citation_doi=10.1371/journal.pgen.1001115" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=The effect of linkage on limits to artificial selection.;citation_pages=269-294;citation_volume=8;citation_year=1966;citation_issue=3;citation_pmid=5980116;citation_doi=10.1017/S0016672300010156" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=T. Ikemura;citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system.;citation_pages=389-409;citation_volume=151;citation_year=1981;citation_issue=3;citation_pmid=6175758;citation_doi=10.1016/0022-2836(81)90003-6" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=T. Ikemura;citation_title=Codon usage and tRNA content in unicellular and multicellular organisms;citation_pages=13-34;citation_volume=2;citation_year=1985;citation_issue=1;citation_pmid=3916708" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=K. Ishii;citation_author=B. Charlesworth;citation_title=Associations between allozyme loci and gene arrangements due to hitch-hiking effects of new inversions.;citation_pages=93-106;citation_volume=30;citation_year=1977;citation_doi=10.1017/S0016672300017511" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. D. Keightley;citation_author=U. Trivedi;citation_author=M. Thomson;citation_author=F. Oliver;citation_author=S. Kumar;citation_title=Analysis of the genome sequences of three Drosophila melanogaster spontaneous mutation accumulation lines.;citation_volume=19;citation_year=2009;citation_pmid=19439516;citation_doi=10.1101/gr.091231.109" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=W. Kennington;citation_author=A. A. Hoffmann;citation_title=Patterns of genetic variation across inversions: geographic variation in the In(2L)t inversion in populations of Drosophila melanogaster from eastern Australia.;citation_pages=100;citation_volume=13;citation_year=2013;citation_pmid=23688159;citation_doi=10.1186/1471-2148-13-100" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Kimura;citation_title=Estimation of evolutionary distances between homologous nucleotide sequences.;citation_pages=454-458;citation_volume=78;citation_year=1981;citation_issue=1;citation_pmid=6165991;citation_doi=10.1073/pnas.78.1.454" />
<meta name="citation_reference" content="citation_title=The Neutral Theory of Evolution;citation_author=M. Kimura;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Kirkpatrick;citation_author=N. Barton;citation_title=Chromosome inversions, local adaptation and speciation.;citation_volume=173;citation_year=2006;citation_pmid=16204214;citation_doi=10.1534/genetics.105.047985" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=R. M. Kliman;citation_title=Evidence that natural selection on codon usage in Drosophila pseudoobscura varies across codons.;citation_volume=4;citation_year=2014;citation_pmid=24531731;citation_doi=10.1534/g3.114.010488" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=R. M. Kliman;citation_author=J. Hey;citation_title=Reduced natural selection associated with low recombination in Drosophila melanogaster;citation_pages=1239-1258;citation_volume=10;citation_year=1993;citation_issue=6;citation_pmid=8277853" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=F. A. Kondrashov;citation_author=A. Y. Ogurtsov;citation_author=A. S. Kondrashov;citation_title=Selection in favor of nucleotides G and C diversifies evolution rates and levels of polymorphism at mammalian synonymous sites.;citation_pages=616-626;citation_volume=240;citation_year=2006;citation_issue=4;citation_pmid=16343547;citation_doi=10.1016/j.jtbi.2005.10.020" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Kovacevic;citation_author=S. W. Schaeffer;citation_title=Molecular Population Genetics of X-Linked Genes in Drosophila pseudoobscura;citation_pages=155-172;citation_volume=156;citation_year=2000;citation_issue=1;citation_pmid=10978282" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. J. Kulathinal;citation_author=L. S. Stevison;citation_author=M. A. F. Noor;citation_title=The genomics of speciation in Drosophila: diversity, divergence, and introgression estimated using low-coverage genome sequencing.;citation_pages=e1000550-e1000550;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19578407;citation_doi=10.1371/journal.pgen.1000550" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. H. Langley;citation_author=K. Stevens;citation_author=C. Cardeno;citation_author=Y. C. G. Lee;citation_author=D. R. Schrider;citation_title=Genomic variation in natural populations of Drosophila melanogaster.;citation_volume=192;citation_year=2012;citation_pmid=22673804;citation_doi=10.1534/genetics.112.142018" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=W. H. Li;citation_title=Models of nearly neutral mutations with particular implications for nonrandom usage of synonymous codons.;citation_pages=337-345;citation_volume=24;citation_year=1987;citation_issue=4;citation_pmid=3110426;citation_doi=10.1007/BF02134132" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=D. J. Lynn;citation_author=G. A. C. Singer;citation_author=D. A. Hickey;citation_title=Synonymous codon usage is subject to selection in thermophilic bacteria;citation_pages=4272-4277;citation_volume=30;citation_year=2002;citation_issue=19;citation_pmid=12364606;citation_doi=10.1093/nar/gkf546" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. A. Machado;citation_author=T. S. Haselkorn;citation_author=M. A. F. Noor;citation_title=Evaluation of the genomic extent of effects of fixed inversion differences on intraspecific variation and interspecific gene flow in Drosophila pseudoobscura and D. persimilis.;citation_volume=175;citation_year=2007;citation_pmid=17179068;citation_doi=10.1534/genetics.106.064758" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. Marais;citation_author=D. Mouchiroud;citation_author=L. Duret;citation_title=Does recombination improve selection on codon usage? Lessons from nematode and fly complete genomes.;citation_volume=98;citation_year=2001;citation_pmid=11320215;citation_doi=10.1073/pnas.091427698" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol. CB;citation_author=X. Maside;citation_author=A. W. Lee;citation_author=B. Charlesworth;citation_title=Selection on codon usage in Drosophila americana.;citation_pages=150-154;citation_volume=14;citation_year=2004;citation_doi=10.1016/j.cub.2003.12.055" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. E. McGaugh;citation_author=C. S. S. Heil;citation_author=B. Manzano-Winkler;citation_author=L. Loewe;citation_author=S. Goldstein;citation_title=Recombination modulates how selection affects linked sites in Drosophila.;citation_pages=e1001422-e1001422;citation_volume=10;citation_year=2012;citation_issue=11;citation_pmid=23152720;citation_doi=10.1371/journal.pbio.1001422" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=G. A. T. McVean;citation_author=B. Charlesworth;citation_title=A population genetic model for the evolution of synonymous codon usage: patterns and predictions.;citation_pages=145-158;citation_volume=74;citation_year=1999;citation_doi=10.1017/S0016672399003912" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. T. McVean;citation_author=B. Charlesworth;citation_title=The Effects of Hill-Robertson Interference Between Weakly Selected Mutations on Patterns of Molecular Evolution and Variation;citation_pages=929-944;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835411" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. A. T. McVean;citation_author=S. R. Myers;citation_author=S. Hunt;citation_author=P. Deloukas;citation_author=D. R. Bentley;citation_title=The Fine-Scale Structure of Recombination Rate Variation in the Human Genome;citation_pages=581-584;citation_volume=304;citation_year=2004;citation_issue=5670;citation_pmid=15105499;citation_doi=10.1126/science.1092500" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=F. Mestres;citation_author=J. Sanz;citation_author=L. Serra;citation_title=Chromosomal structure and recombination between inversions in Drosophila subobscura.;citation_pages=105-113;citation_volume=128;citation_year=1998;citation_issue=2;citation_pmid=9652229" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Muyle;citation_author=L. Serres-Giardi;citation_author=A. Ressayre;citation_author=J. Escobar;citation_author=S. Glémin;citation_title=GC-biased gene conversion and selection affect GC content in the Oryza Genus (rice).;citation_volume=28;citation_year=2011;citation_pmid=21504892;citation_doi=10.1093/molbev/msr104" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Navarro;citation_author=E. Betran;citation_author=A. Barbadilla;citation_author=A. Ruiz;citation_title=Recombination and Gene Flux Caused by Gene Conversion and Crossing Over in Inversion Heterokaryotypes;citation_pages=695-709;citation_volume=146;citation_year=1997;citation_issue=2;citation_pmid=9178017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Navarro;citation_author=A. Barbadilla;citation_author=A. Ruiz;citation_title=Effect of Inversion Polymorphism on the Neutral Nucleotide Variability of Linked Chromosomal Regions in Drosophila;citation_pages=685-698;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835391" />
<meta name="citation_reference" content="citation_title=Molecular Evolutionary Genetics. Reprint edition;citation_author=M. Nei;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=M. Nei;citation_author=T. Gojobori;citation_title=Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions;citation_pages=418-426;citation_volume=3;citation_year=1986;citation_issue=5;citation_pmid=3444411" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Nei;citation_author=K.-I. Kojima;citation_author=H. E. Schaffer;citation_title=FREQUENCY CHANGES OF NEW INVERSIONS IN POPULATIONS UNDER MUTATION-SELECTION EQUILIBRIA;citation_pages=741-750;citation_volume=57;citation_year=1967;citation_issue=4;citation_pmid=6082619" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. Nielsen;citation_author=V. L. Bauer DuMont;citation_author=M. J. Hubisz;citation_author=C. F. Aquadro;citation_title=Maximum likelihood estimation of ancestral codon usage bias parameters in Drosophila.;citation_volume=24;citation_year=2007;citation_pmid=17041152;citation_doi=10.1093/molbev/msl146" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=I. S. Novella;citation_author=S. Zárate;citation_author=D. Metzgar;citation_author=B. E. Ebendick-Corpus;citation_title=Positive selection of synonymous mutations in vesicular stomatitis virus.;citation_pages=1415-1421;citation_volume=342;citation_year=2004;citation_issue=5;citation_pmid=15364570;citation_doi=10.1016/j.jmb.2004.08.003" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=T. Ohta;citation_author=K. I. Kojima;citation_title=Survival probabilities f new inversions in large populations.;citation_pages=501-516;citation_volume=24;citation_year=1968;citation_issue=3;citation_pmid=5686303;citation_doi=10.2307/2528314" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. A. Palidwor;citation_author=T. J. Perkins;citation_author=X. Xia;citation_title=A general model of codon bias due to GC mutational bias.;citation_pages=e13431-e13431;citation_volume=5;citation_year=2010;citation_issue=10;citation_pmid=21048949;citation_doi=10.1371/journal.pone.0013431" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. L. Parmley;citation_author=J. V. Chamary;citation_author=L. D. Hurst;citation_title=Evidence for purifying selection against synonymous mutations in mammalian exonic splicing enhancers.;citation_volume=23;citation_year=2006;citation_pmid=16221894;citation_doi=10.1093/molbev/msj035" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. E. Pool;citation_author=R. B. Corbett-Detig;citation_author=R. P. Sugino;citation_author=K. A. Stevens;citation_author=C. M. Cardeno;citation_title=Population genomics of sub-Saharan Drosophila melanogaster: African diversity and non-African admixture.;citation_pages=e1003080;citation_volume=8;citation_year=2012;citation_pmid=23284287;citation_doi=10.1371/journal.pgen.1003080" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. Porceddu;citation_author=S. Zenoni;citation_author=S. Camiolo;citation_title=The signatures of selection for translational accuracy in plant genes.;citation_volume=5;citation_year=2013;citation_pmid=23695187;citation_doi=10.1093/gbe/evt078" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=J. Precup;citation_author=J. Parker;citation_title=Missense misreading of asparagine codons as a function of codon identity and context;citation_pages=11351-11355;citation_volume=262;citation_year=1987;citation_issue=23;citation_pmid=3112158" />
<meta name="citation_reference" content="citation_title=Numerical Recipes in C: the Art of Scientific Computing;citation_author=W. Press;citation_author=B. Flannery;citation_author=S. Teukolsky;citation_author=W. Vetterling;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genetics;citation_author=W. Qian;citation_author=J. R. Yang;citation_author=N. M. Pearson;citation_author=C. Maclean;citation_author=J. Zhang;citation_title=Balanced codon usage optimizes translational efficiency;citation_pages=1002603e;citation_volume=8;citation_year=2012;citation_doi=10.1371/journal.pgen.1002603" />
<meta name="citation_reference" content="citation_journal_title=DNA Res. Int. J. Rapid Publ. Rep. Genes Genomes;citation_author=Y. Rao;citation_author=G. Wu;citation_author=Z. Wang;citation_author=X. Chai;citation_author=Q. Nie;citation_title=Mutation bias is the driving force of codon usage in the Gallus gallus genome.;citation_pages=499-512;citation_volume=18;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=S. Richards;citation_author=Y. Liu;citation_author=B. R. Bettencourt;citation_author=P. Hradecky;citation_author=S. Letovsky;citation_title=Comparative genome sequencing of Drosophila pseudoobscura: Chromosomal, gene, and cis-element evolution;citation_pages=1-18;citation_volume=15;citation_year=2005;citation_issue=1;citation_pmid=15632085;citation_doi=10.1101/gr.3059305" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. A. Riley;citation_author=M. E. Hallas;citation_author=R. C. Lewontin;citation_title=Distinguishing the forces controlling genetic variation at the Xdh locus in Drosophila pseudoobscura.;citation_pages=359-369;citation_volume=123;citation_year=1989;citation_issue=2;citation_pmid=2583480" />
<meta name="citation_reference" content="citation_title=The Genetics and Biology of Drosophila. Vol. 1a;citation_title=The Genetics of Chromosomal Aberration;citation_author=P. A. Roberts;citation_pages=671-684;citation_year=1976" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=S. W. Schaeffer;citation_title=Molecular population genetics of sequence length diversity in the Adh region of Drosophila pseudoobscura.;citation_pages=163-175;citation_volume=80;citation_year=2002;citation_issue=3;citation_pmid=12688655;citation_doi=10.1017/S0016672302005955" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. W. Schaeffer;citation_title=Selection in heterogeneous environments maintains the gene arrangement polymorphism of Drosophila pseudoobscura.;citation_pages=3082-3099;citation_volume=62;citation_year=2008;citation_issue=12;citation_pmid=18764919;citation_doi=10.1111/j.1558-5646.2008.00504.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. W. Schaeffer;citation_author=E. L. Miller;citation_title=Estimates of gene flow in Drosophila pseudoobscura determined from nucleotide sequence analysis of the alcohol dehydrogenase region.;citation_pages=471-480;citation_volume=132;citation_year=1992;citation_issue=2;citation_pmid=1427038" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. W. Schaeffer;citation_author=W. W. Anderson;citation_title=Mechanisms of genetic exchange within the chromosomal inversions of Drosophila pseudoobscura.;citation_volume=171;citation_year=2005;citation_pmid=16143625;citation_doi=10.1534/genetics.105.041947" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. W. Schaeffer;citation_author=M. P. Goetting-Minesky;citation_author=M. Kovacevic;citation_author=J. R. Peoples;citation_author=J. L. Graybill;citation_title=Evolutionary genomics of inversions in Drosophila pseudoobscura: Evidence for epistasis.;citation_volume=100;citation_year=2003;citation_pmid=12824467;citation_doi=10.1073/pnas.1432900100" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Schrider;citation_author=D. Houle;citation_author=M. Lynch;citation_author=M. W. Hahn;citation_title=Rates and genomic consequences of spontaneous mutational events in Drosophila melanogaster.;citation_volume=194;citation_year=2013;citation_pmid=23733788;citation_doi=10.1534/genetics.113.151670" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;citation_journal_abbrev=Cold Spring Harb Symp Quant Biol;citation_author=J. Schultz;citation_author=H. Redfield;citation_title=INTERCHROMOSOMAL EFFECTS ON CROSSING OVER IN DROSOPHILA;citation_pages=175-197;citation_volume=16;citation_year=1951;citation_issue=0;citation_pmid=14942738;citation_doi=10.1101/SQB.1951.016.01.015" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications;citation_pages=1281-1295;citation_volume=15;citation_year=1987;citation_issue=3;citation_pmid=3547335;citation_doi=10.1093/nar/15.3.1281" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=P. M. Sharp;citation_author=L. R. Emery;citation_author=K. Zeng;citation_title=Forces that influence the evolution of codon bias;citation_pages=1203-1212;citation_volume=365;citation_year=2010;citation_issue=1544;citation_pmid=20308095;citation_doi=10.1098/rstb.2009.0305" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. R. Singleton;citation_title=A MECHANISM INTRINSIC TO HETEROZYGOUS INVERSIONS AFFECTING OBSERVED RECOMBINATION FREQUENCIES IN ADJACENT REGIONS;citation_pages=541-560;citation_volume=49;citation_year=1964;citation_issue=4;citation_pmid=17248203" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=L. S. Stevison;citation_author=K. B. Hoehn;citation_author=M. A. F. Noor;citation_title=Effects of inversions on within- and between-species recombination and divergence.;citation_volume=3;citation_year=2011;citation_pmid=21828374;citation_doi=10.1093/gbe/evr081" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. H. Sturtevant;citation_author=G. W. Beadle;citation_title=THE RELATIONS OF INVERSIONS IN THE X CHROMOSOME OF DROSOPHILA MELANOGASTER TO CROSSING OVER AND DISJUNCTION;citation_pages=554-604;citation_volume=21;citation_year=1936;citation_issue=5;citation_pmid=17246812" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=S. Vicario;citation_author=E. N. Moriyama;citation_author=J. R. Powell;citation_title=Codon usage in twelve species of Drosophila.;citation_pages=226-226;citation_volume=7;citation_year=2007;citation_pmid=18005411;citation_doi=10.1186/1471-2148-7-226" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. G. Wallace;citation_author=D. Detweiler;citation_author=S. W. Schaeffer;citation_title=Evolutionary history of the third chromosome gene arrangements of Drosophila pseudoobscura inferred from inversion breakpoints.;citation_volume=28;citation_year=2011;citation_pmid=21339510;citation_doi=10.1093/molbev/msr039" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=A. G. Wallace;citation_author=D. Detweiler;citation_author=S. W. Schaeffer;citation_title=Molecular population genetics of inversion breakpoint regions in Drosophila pseudoobscura.;citation_volume=3;citation_year=2013;citation_pmid=23665879;citation_doi=10.1534/g3.113.006122" />
<meta name="citation_reference" content="citation_title=Topics in Population Genetics;citation_author=B. Wallace;citation_year=1968" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=F. Wright;citation_title=The 'effective number of codons' used in a gene.;citation_pages=23-29;citation_volume=87;citation_year=1990;citation_issue=1;citation_pmid=2110097;citation_doi=10.1016/0378-1119(90)90491-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Zeng;citation_author=B. Charlesworth;citation_title=Estimating selection intensity on synonymous codon usage in a nonequilibrium population.;citation_volume=183;citation_year=2009;citation_pmid=19620398;citation_doi=10.1534/genetics.109.101782" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Evidence for Stabilizing Selection on Codon Usage in Chromosomal Rearrangements of Drosophila pseudoobscura" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
<meta name="twitter:description" content="There has been a renewed interest in investigating the role of stabilizing selection acting on genome-wide traits such as codon usage bias. Codon bias, when synonymous codons are used at unequal frequencies, occurs in a wide variety of taxa. Standard evolutionary models explain the maintenance of codon bias through a balance of genetic drift, mutation and weak purifying selection. The efficacy of selection is expected to be reduced in regions of suppressed recombination. Contrary to observations in Drosophila melanogaster , some recent studies have failed to detect a relationship between the recombination rate, intensity of selection acting at synonymous sites, and the magnitude of codon bias as predicted under these standard models. Here, we examined codon bias in 2798 protein coding loci on the third chromosome of D. pseudoobscura using whole-genome sequences of 47 individuals, representing five common third chromosome gene arrangements. Fine-scale recombination maps were constructed using more than 1 million segregating sites. As expected, recombination was demonstrated to be significantly suppressed between chromosome arrangements, allowing for a direct examination of the relationship between recombination, selection, and codon bias. As with other Drosophila species, we observe a strong mutational bias away from the most frequently used codons. We find the rate of synonymous and nonsynonymous polymorphism is variable between different amino acids. However, we do not observe a reduction in codon bias or the strength of selection in regions of suppressed recombination as expected. Instead, we find that the interaction between weak stabilizing selection and mutational bias likely plays a role in shaping the composition of synonymous codons across the third chromosome in D. pseudoobscura ." />
<meta name="og-title" property="og:title" content="Evidence for Stabilizing Selection on Codon Usage in Chromosomal Rearrangements of Drosophila pseudoobscura" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/12/2433" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="There has been a renewed interest in investigating the role of stabilizing selection acting on genome-wide traits such as codon usage bias. Codon bias, when synonymous codons are used at unequal frequencies, occurs in a wide variety of taxa. Standard evolutionary models explain the maintenance of codon bias through a balance of genetic drift, mutation and weak purifying selection. The efficacy of selection is expected to be reduced in regions of suppressed recombination. Contrary to observations in Drosophila melanogaster , some recent studies have failed to detect a relationship between the recombination rate, intensity of selection acting at synonymous sites, and the magnitude of codon bias as predicted under these standard models. Here, we examined codon bias in 2798 protein coding loci on the third chromosome of D. pseudoobscura using whole-genome sequences of 47 individuals, representing five common third chromosome gene arrangements. Fine-scale recombination maps were constructed using more than 1 million segregating sites. As expected, recombination was demonstrated to be significantly suppressed between chromosome arrangements, allowing for a direct examination of the relationship between recombination, selection, and codon bias. As with other Drosophila species, we observe a strong mutational bias away from the most frequently used codons. We find the rate of synonymous and nonsynonymous polymorphism is variable between different amino acids. However, we do not observe a reduction in codon bias or the strength of selection in regions of suppressed recombination as expected. Instead, we find that the interaction between weak stabilizing selection and mutational bias likely plays a role in shaping the composition of synonymous codons across the third chromosome in D. pseudoobscura ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
    <title>Evidence for Stabilizing Selection on Codon Usage in Chromosomal Rearrangements of Drosophila pseudoobscura | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__34Arr8QetTzezDWW4Etyv7gbgH-x-ZTNo3yO15Q-gHU__Mk-lyyG5WoK4SRy56eUq5eomxN7OGXjpDv-Ob1a1Q1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"lSEugpQQHWirdqaptNihAQ3sEJxmk4__hYnHlA29eRo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459744","apath":"\/ggg\/4\/12\/2433.atom","pisa":"ggg;4\/12\/2433","ac":{"\/ggg\/4\/12\/2433.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/12\/2433.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/12\/2433"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/12\/2433"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/12\/2433":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zM5SOVnonv8vCVo8ct+Yygj2xtK8\\\\\\\/rvyv5+oS1BLc44eJ3mCQ3fe89RjCfp6g6kGnjlqlxrPLnbfC\\\\\\\/7gvj\\\\\\\/N8ogL8pCnQ0gNxvRwSuwwElvCMvd8OD2PXf95GwubTfsrvrVBxlg2yWspcJx28laom79RKvHbBq9sEv8UqdTK4RbzlPr72XR+fqlkwZ2eiKFSXr6D7JbpjY\\\\\\\/5sAVPzm0TqrRzbb\\\\\\\/ki31aDAFHEXgGXotolqVdxa2enZiIkPMtBc8gy+5YA0B7DBttr+z0uwJ30DBWzJXE7Ii1Gg9favZw0QmRO5db8Uku8gSwxN1LwM54d6IYRRu7xODJJezvG\\\\\\\/Bdesi7n03vuPqWSep96LFvVVv3brd8fukGJpLZ0EfJro710yGjwKac5ZMAorJq4wgBVG21iZeM+recJpYE+cBCNRJ+7OGF99712K9VgEiGstY0J52jyRELT179lalTMEgl3YP15WHY0hPFyZO2+mj0+wAeIL9RRGK0BBpPbNRjWipXgcOa7akEkS25RRk8y0UyNsjJ6bXGxpAgVtfKWFQuLfw52rtJuwHA+6PlhThHP3UIowTEFI+heC5wn8PRWQ2R+X+4ttTD4Q6d8ShrFFFyTOrrz24ksYz1i+mFUX8dRIC8cQidCTQNuSrPxB4VPU79HudnlfOTQpwxB0V8\\\\\\\/i4BDjUTQdx93+wQV486nRi2iqAv4IjuApI4OOssve03SKCky+yyjwvhpVbi9I2khEb0JbiPuQzvCueO37+0QuxjHSgMKPR403uqqWd8KiKJAgIIcDzVxcPy4BJ2X4cYAkcF5IPKuo8TjV1el\\\\\\\/wYGj2s0+1z4no72ZqcDa\\\\\\\/GBdo7sNsJpd9AbgPxoisiFd2fkoIJ4Ax5WVsGEPL1H89zTbRYOEwwV2vG013fdVc4ezlbfZCmr5qvtl630EK1lIBbz1PDnhzxdlBZpkQOiQDLOwXyL8lV\\\\\\\/u2ny3rvBm897sujMYTVp3rMlUwSfQ9E7KQ6lvA1YZUi0Lzb3AIPfKmK24SYZEpYo+wR6c6aq8uQJhiW10xkgIDIr7XM9YJ2qA8+dR+qyREYHCQLbDPsBThkz7MY3A17xjGbo7wxe\\\\\\\/RC8mziDt8SNzTf4p9\\\\\\\/zTubZXg2tEY\\\\\\\/csTfZmTuMg9t7y2070XPHZIetcRj53KWcobEFIA9uwySducQihPOnhByGwI+aWHK+ETyuYw5FUz6sMebos7yz2jSNn3l4XXED+sZbeCOO27lmYo+Tya82JUB3ZhC8gTOpQQGqwYSawikm46ri+eOEcBt\\\u0022,\\\u0022iv\\\u0022:\\\u0022da42KQlHe77yy+AQdlzn+A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022826578271ec557f215656af35995443f7b8ca0ae326ec9a4ee84fcaaaa47c076\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YTgOJKJrujf91jTgqHv6Tw0OIg\\\\\\\/pimCKSTzohRKcsk4Ahfs8zbl7Sw+GHc\\\\\\\/J9q8Bh4ltTvnCJ18jw5XomwkPGG2oL70KN5EOPAeE92rTXMFch5b6bj4Qv3jGpdT585k3k0pk1qLnIPszATYFrDtbOAUU\\\\\\\/ejNQxeUF9OyQbRNmQ\\\\\\\/2ltNvsei3q8SfVOqgvy8Qp0oaRvH13QKGCr9OyHFvQhAmr1jebveTdp13LhP4aJBiYCZ9joSGddijY4Ww42pODrAc\\\\\\\/vHSKKsJCwjiW3r9Re2jBPGZIazOhV5+GBLTBEukbHF\\\\\\\/OvBH2UjQ35Nxg271URmTK+YSq5KyAOzivb8iVQi9sTs+8dWwT3jdX+8DVebfVuJif0eiSvDaBvhsF1N5xe5NxAfT1V1m41hsxnOtMX8gV6VTHiCnjDjtCLlQlzYq9H3bWcWMDBh9e6A43JRQSKWud8JT+k5pESx9r\\\\\\\/mBfrzOIPf1pCfPBhoqtZXjKu3aft8RycILAopx+7dBhVQC2mZHWS\\\\\\\/meNZDWc7RlmMqzKOieP0mP1buS3gCIRnTn5bNDgrGJpbLv87RCL0D7JGovhqmG0PNI6DG4o2SqxcO3BJxWFhIp+qhWPUJGIvIhr4q5FrPgmEHN80YpDb0UQF4SeTYWu2FawEqf6DmKZyYaoZPzDDAZEeubt+WwNtdK+\\\\\\\/\\\\\\\/z3RmA+ThE8qHiS0i\\\\\\\/Q30DhanAsQfoLYMPmfO0ZIueXTvg1lJAmmoTRESnjzqLtMcXYuULDdeSRYh9bGEhGbPIpelZAQNBc+SiH1P9qNNviJ5VYCnVlc27\\\\\\\/bqrBnApjR\\\\\\\/Q0mUFj2+LK0z6VliBR2uDP\\\\\\\/IJPoG6+6xzpMsocrNgrXnLICyOUc367qnQYQ19g3B6kPhn3T\\\\\\\/3VfeHrXnlsEwHSbEABMJtqguNYGev9Ox5SQ1ftDQKkk7nq7BMUi0YFYiWgjIa9Kc5vEkyJ9bCFhALqnE+hJGeB5EayP33lQmu63G7xnQ9nPkwSMNMYy2V68NnKAAOCntOgJG7n8iDAxeJGbyw8akFhgZlEFSlKG0tHzcuL1wtX55fVYHRJw0ckyNuD9haLuXQHPBx5mC\\\\\\\/NZqAspGTt7rriahxG1nSN8sMdcK6AHlMppUuUayiWngA78q0h08Hp76w912cxR2dFpntkHEdmrX+gP5J\\\\\\\/dbQngYo8BYXpa4BJR7POcu+BDkZDUW70FsLCUjsaRYjUem8PqHLle7SI6S1AIWf0XvphXDoNMcAlSgOKStxb79D56c0WUKINA7\\\\\\\/i0ej0k4YiBN\\\\\\\/sna34ieHofZh8d5JsVFaUUUDpOVYjmEVP5kOvfWGBmjjqIJfL0WnHSuxb2K7FhrTrkPjgMfLNIatQLjM3dRxkkjgPJY\\\\\\\/0DH7dOSQ1xxfPsPAiZhYY+PoMelPEMEXesoCY5yjlgyrWBmGg3BMKj4MclMld4xKC2Q2YhlKlPqGHcO5WzWKC109qK7PTTzJQkS5txx0bjKXWyPSf0rSYIlZE0SU0DazCAZfiYBNy0ahCeX9RR0QvjAWnEPXPdOJwtdhPg90PXLzCk6ND8dVaqaLXAEaKvaJLpBU9rpNkdoqXpjTlu7G7Gw0XIty7eAieO04MyvRaM4+jx1eQHtfameBfQjCfgwLewe0uLWzDWHHw58Go231xvjISEM7ZU0qu7Rl1HEzCVqYS+UAAV0UPgx6UkCjA6A2M\\\\\\\/axoVFgQVaTRCd17JyK33NCdwqzBEsH+rI47qRqeot0Y5DhJlYlVG0qvLo8zjhjk\\\\\\\/yVvSHW1cipQo9yPl\\\\\\\/wN4rzKQPzq+oPJk7H+I1b0LXldbY4TsULV4NpaJjeUezjOOdOCx3Vi\\\\\\\/2EFj8O7gLSA6UOXehn2lYoIN3K9ksEG7zgDTYUb2GSsr5rGY7NPW8LTDwo5Lw\\\\\\\/G8LqPe6PPuS++7L42cC1Sr+pW\\\\\\\/tdmZiUh9TdFtgLqztBr30yp4nkzoDFEPkrhFrbfGxTRXM8Hp6o8IZYQhzNUuDr+q49mi1ongIEHOOZjVkTKQnjtidh5DJkKHBvah2Df+8j1WhaPwD2xK3VyftwEmEqFCiqz1QuQuJzc1Loe6afv1t608BjICRvFtny1tz\\\\\\\/\\\\\\\/Lg4I0gWfL0IE2NNuUqR4ft2\\\\\\\/LKqGcIq4QQtDU3v+9ftPpUI5D8nbwZo4HDQrUzTWg67P2MYAIu+g4HAYG2CmY\\\\\\\/bP\\\\\\\/9uN+DtclAYY0ebZQW8++wFkJ7qKg0x6Ubmvy2wPJoGjtWusDFmYeglNqDgshIDXDHt98B15jOT9V2jkIKa08z5BH3\\\\\\\/gysqflFZGjQTvCTcarEqELyf5ART+4Pbuzqal\\\\\\\/ObYCA7aQ4ct5xtAJRqSAbu8fF\\\\\\\/ZqiDG6oU5\\\\\\\/OEPGsynz1ajHscyPZxT97wD5+1eRQCxjEjoyFUnfDt7PE8Zn8KIP7xvw7zUiV\\\\\\\/gZPI5MADqonwekPTUcW\\\\\\\/XnBJagBguX2UtU+5n4wM5qrkSJGx+rH0rbu7DbspVqLZZelxhVmMNHjVLtGR+IhqDLHtJ5WZVVE1cQ8Meh3AfKbE4+3NGM0WLIPfUFHBVlOWh4WaxomTpq+uBbXjv2NPQL5e3MIewdxP1cviQfWatSo8pTYxccho\\\\\\\/U697tDt5\\\\\\\/unGiiakFT8fWDbO5gjRyX\\\\\\\/\\\\\\\/fwdv3fYLZSef+rA2rux7hRa2ai3uVzQupTVihApRTunv3MbHhyCAma3rIlQW5Gs4P5MC1vpGI4fZgF8Sr\\\\\\\/UQN1TnOdgEmq165PoK3d5tytt4jhZjvZP6T1d6Jw\\\\\\\/j6XgFi0N592gU7CeWSVbp1kjzmz24NyNQ8ufdBXddqb8nA1LzfG+RlPDZgvzS5tyuFmwqaDNT547as2LDrL+\\\\\\\/a1EjTaD2Rtdm5SLXoXiwQj1vjxrpH2ZP8of22WclNj+SvlQn5\\\\\\\/8SZErybBqZPVvsXMTP0MoW8KTAkXbokCGP2\\\\\\\/GUHI9c74VQY2lx66A4hkX4faZwMPBHXw9lNfH2RJ2md4c3SZ7UrTNPK2gX\\\\\\\/FD1YDeeDAafr1e+COt2lAzS2NxDGx1GNlwHYUxTHDvEX8iLrLLgsnJsVsU\\\\\\\/VIZhzMcq4SqhSyVDKv8KVx8IDjd\\\\\\\/lUMKDdiajFy34IAJ+oZlE1RHU69vsDzRtj1TmwTaaKy6\\\\\\\/sALa3UrrAAROxuBuEVPFmVdR8hABXdF4gpQnqloPSLFaek5TWsHgo6E1fnI06VqMT4r1rLvZZ1t\\\\\\\/RBPYcf7upPQKS+aETO2RcFuDw9iRQ\\\\\\\/3uDOJQzECDg0iDx2G2NJQyaCNb+u2vuy+LnBUnCkUVzenyB1wInxkuNxccYJjntWZCgMOjx4jQVpM9tt09k3pdu+4t\\\\\\\/6tPv0R84hXnIXiBO7wtvU9q2\\\\\\\/DpZhYFd3xuJbADELDFIq4Wa0Fy0XSNRpr9K6lOi3ncP1qu8P8A1H0FxEQ+F90oRdJelGXC2zq68yLm3GsZ1hdonoQO6QFxO+WyQ2h8mJBOnOrWbZveLcTf7fJDJZJCNC83LgAbV8fzTrTFKsXVutnThF5Q8wZs0hgPj1XP0aYV70xHeHTp1vruWgd4rdBOuZMRw\\\\\\\/RbJvp1BjhqERgJ0pV1niA80SA0\\\\\\\/N8eOF17l39ri\\\\\\\/qgniAewwHKP6J5S4vbkqWbjNJMLrGvGnO5v32xS49tFz6SA7LEL3UneIonxYSlT0zm4+QpWVM5mDO0PYTjetTCKT8JxTialFUb30hwL8sUboYhsYCpNQgAbTCr1caHUEdkpJcjzhn6MLp9UYKSJQPjWuC1DXsDyBgl9kpZNONpAdsiLmuQJHCuaxxlGofy497LYY5kwxrl07mpI0c5X\\\\\\\/ZUrKkfu1jMLaSrStA1IJg8gv6gSmHqESo7m0TtbnVtYrUe+FaMsTNNCNq\\\\\\\/9csanvLq11RZSKvlpmmw5bLkJuRnv+IlOgHmrfyFm62ufFcU\\\\\\\/xozcOuoV7ylJtCtipf3VBNtNOpQGa1l\\\\\\\/D24qdgtG3SHiTvs0VoY1gZdGBaH6xbihQX7tySm9dYtCWqw69HFiPMDSaXZTAwEuunp77Eg9Yf+8knvpJUkAuS2nhmAEcopeD6UA8WZX0CTuU2N4QibZDkJLlUI+lUD1wdm5\\\\\\\/1rF07Mt\\\\\\\/CLLDrRlPXiZl+uYgoJ2NbxdCv2p4YDCcQspecflwVVMu6maAzq7KvnyWGTydxtMVW9ktbGEnpKoU4zQUBsEPz8LzL1SAF4LIEG8yHoVsVVwpdeGyXaWG1zdtDf8kw99SMO\\\\\\\/EIRUixb8w+H\\\\\\\/yDHA6dXTu0gC5jKxOSe2hv7++hsVnJRzCkXaZh2AI9MvakJjdw+VJsreWP7LW22f+R+RFAXij21a6ar\\\\\\\/K3QEXSZ+ERXKH84fenby\\\\\\\/ynijTX\\\\\\\/b3VZVXil03HpR+9HIG0NEnNOf4n44PKPcZ0Fg7taJnWbe8NBoG+1en+zHgrloDadLNaSXL7lzkarklNnP8bwDrejcrQssxeAsmeQb9qgC99wHkJdA\\\\\\\/q4+u3ci46ej\\\\\\\/e3rNn2Nxbp4vfzgitZUNNpZTIUF2410q8MZINumgJtyeYF+MhbH3VGDVdeZ2uf5dUkwQu4uBzDYPep\\\\\\\/ah5HeGa0APE5mvmoHpLHC2HWwZfgycQ4IvoyIQ1T+WLfPgpeZ66+acQHU7P0oRhDX4XYAU2LHr4GQ5ZKfcGTUk8dA8TUVIOgmEneH\\\\\\\/5Jcy1oPt4E7We9etwzBqLz9LDz+IaE0DuGDPxuTZNxE\\\\\\\/lWbtFvsNWgJUpGQvF9hxF\\\\\\\/uSwD0EVpQpzQXmZWQ30APEPES4rJCorFbrdDSxHePFaQNEx\\\\\\\/BIqx5Dy85n+Jo5yo+ME18oEKqtq8ilWDUb38NQ8YI9HqEvoh7QkoZCSXSk3wXotgJXY9ZYyeMpzISN8kkwX3ZLzw6J3jTtJBpoocW8Bp7IGzJSdbpnhfhMjScxRXBV6dpGER7yTd9UsQNDlC0\\\\\\\/blO6\\\\\\\/D40N64k4HUFP6VG\\\\\\\/MaYafZwplC7MuaIs+euKi2cMKjN2vwL9WTKMh2CV+LwHeBUWryaGi3HLNDxey+zTa4smF75Uly4AkCd3mgqUlQ6K1nOH908NWtzti11ptPfzedxyX+AWbAAbZBgXDIw0An9Y8LHhsIrvUEYij4ABLiUgNJP+X4hpAFMZ5AIkewiJY\\\\\\\/m0VSRBOoXARIXY5FjvR0vgD4HM34It+KSL7sYvNW05cByHUtGDOd2MyY6G5MlFyekF+SnMh+xCawnf3Nj1C5wbpA2cQhnED89jawOp2MBZwnF2ZGw+RCJwLOED8AcC2NkUtXBmj7ss3UB0iyYc9PSEiuaeTAbqdgFnCaNf0p9bg+v\\\\\\\/BaqHoOBQlTcmGKRq3KS2K6BSR4TRLcBRpuNvT0btb4GhCaV2gvEAw6HH5nPuBFPFHPs96OPccTk33QZkxjnwAz+awBz4s5TO4VV73yPNCluE05Y1YUnoL86ARehgtj6Yldgp\\\\\\\/JxNm7d\\\\\\\/7qlwDqlXGHxq6uvOO7\\\\\\\/41I5\\\\\\\/oihKBzxrmTIltI7nd+5A3op3Ra1QALame+xlwcH+HA0BwWoY4AYeaw4UwGjkne1ZnSqwqw+OVywAu1tF8iZcp5A2bOgf9pMCufeOq2\\\\\\\/g8mhjtyV9ik2zAlK48iLVQN5YRT05ZqXC+\\\\\\\/PrUPiymnV8KyOpEQ8r2xb1SV9kEXdQfv0PVWaONaOyoHZNoRRS77HKy7UFaHkj6jr8YwaAFyB4FdqRo1twewThzpmwpo4wQmxXQA7vtWHBURrgJvYxKjStri+TIzVERL08wHilwVjFLCdmi+nrB1xLBNufAIehhO7OeqjyTgpNcMt++bfIvcdRWpNww\\\\\\\/+IdHx+\\\\\\\/q\\\\\\\/KAN6Fqi\\\\\\\/9TvHlekKP4iNu76B3BwtsLJ8C4r9QsDoCoYNoXC4WJNjSTdqmzHdfIIOM\\\\\\\/LGBDxSByClxDm9ic0MHluFbULWhSwCtaqemIedgKWP8VFD51ILvx6\\\\\\\/iyEoQQeTDH4tfzE9F78nDclzOlsp2FBPzc9t\\\\\\\/2f6OXETLe\\\\\\\/kzX3zzwz9KSi49e8PyAaj2JnbLdrwNdkLpJPUKcUVId\\\\\\\/F5GZrdWbYAc1nXJGjNMOMLbEIZNFkKNYh7X52+NnlZasUdwPOBVg5RUahuiSHI1zNwGnz4ggxGR\\\\\\\/MdLwa2pr6xsMWt5TznwoEKf4EwlatuH6ihNZR8ApVj5ZyGZvbkYSWXfE+lk0h8LIKhQOg+V6i1i+PnL3h6xj8XHgwqTJp0kMDj54OZYlYdVU1kSX6O0+YD6ZriTIBXIzTYZZ+LWvRbEXnxKEWs6i9AfkQ+fbz7SFef1YUdxfLJYylfDdNTf8eN3kCbOF9d5qat6JI+M\\\\\\\/t+EuJYM0zCD1\\\\\\\/1nzRkqXoCbj3Ia9CRxgeTRpq+GMSkygfv6dWplwARe\\\\\\\/IaUiyCB0P1hAtPN9V8Hi+bKVmUam5MYcRrPRsfUiRGq6VfqTvxgyu3gLGWN+YrIforx3BlgT9jbQnQWysNEcddhOTiOSq9s6\\\\\\\/4ttao4NFPZacio2alp2EhWRJeOQn0heIzq5P+sVfN282\\\\\\\/UlfQIRC6Gae0bnaSr+qN\\\\\\\/mxtfFt1SfCGSYzsK1uDZyIsCGcVK4urM5KvH1V\\\\\\\/Bkhns8gewyHu5qhho2\\\\\\\/23OGt5s+4WEw5twBi+PdxYVhqRg\\\\\\\/1hiRXFrt9YCuOFN\\\\\\\/RufGQ1UBTBUcmMzDqlW5BYKMNxnG0sGiM0EogC0esax7xMviRdLxyW2salCRV8hGNnmbxacbZkcFM2zTGUyeTBs7la5EL+IG8GpHc0FMeniD5yIu24h3VQYU0zfmtsggi\\\\\\\/rJJ27YHAFzLdaQnTjvM4B3ZG8L9DQfKooatt5D1ZnZiLgG6zVGAeeK9VPCOpm0rg1f94+5Sac8o379D+2BMJgQPAf9A+7A8nk7bC4WHUtK9cNHlB0xOCy0bpPjm3mZ4ET0iWgNtSwXsaiyDRRaXJGthkTuw7vbJfsIvYrJKo83rJABLxszwbiGprP\\\\\\\/Nn36DX+d8Nqvqc\\\\\\\/GkjGUEI8AD4akp58Em47quRvelp6CI7kxCS0WJSXKpNSwK9URA1s6EuZOfygXStOojHu+nwWH4bbVRtDGFAK400DPHJHKFheN8JLV3ydZU0hqcf7NBrQnPJdFEm\\\\\\\/Zno3RWy+osK9ODsIFkkcCRVzEs5F6NHhtcnVKUessuJh\\\\\\\/5QEKMlmn+WEaRo04i4eKxYJTEjYwVGNtapG2yb3g+iNb1Qi1LSff0MBzC8jshSVZLvvpkZeZhbbpMrwPhl+jJghhFNLsjUVfLkBnbIug0sX1gmIPrg+pVd8cRWuF9KchIs68Frhdiky1BftTRocFdS7kbcUge521+MYATFBtA0l19GDWX2\\\\\\\/P4hJChCnnhs16H623krZH0wSRoAFYYML0LvCyo77hDXABx0nYzi0fqJ8QPK8+Lz9UEiku+1CHrd4HStpZFrvVgM1JMgo9qcT0QsKIKFbqgM8nwXSnhprxPcJX++Z9DbiajjMTRFAgHpd4e9GtQ8BmrxaUueanq\\\\\\\/j7+Nt+Rw5L\\\\\\\/rKeqZam8r7ZrX06\\\\\\\/cKK4aJ+Uv1gEmXHwCB9nc1Kp2TepV\\\\\\\/aC6GNZ2IfEVILvWlBoljQeRBrg2bXEtVphDEYyS34C29Tr2HUuVWs+T3nJdKgUCo+UapQ04pvJkowSR83bVkk66IzeLL7KM54ytIOIXd3wKaRpuZxNN1+h9yGOcGrik1viXFecfYqpTpFl7JuAB7N+VamGjS2Jjq\\\\\\\/ewnR2YDAw+p+nTgiaoEeVTVDJ44jywCE7+QhBuhfJ8LPvFt8y3oSMfEkpZqkPmifeszrNygtwkUiPhwEOoZk3kpA3TqgbBnK6w2PdXYD8CRqnLytu4zCK3s0h4mmG3kg\\\\\\\/lV+PPC+LmCC1r6XdcZy0l\\\\\\\/FJsP6U1kwswQg4YE6qxQMwTRQOTNJuOJnCg3xRajGIpfq2biCI98XvDN9Rjd3WgzIrOc1r3GUQlktMJi3JQJWvCKp1NMxlmvv7BtQ1gpf1o8CFpiO4tuiIh+zLqdN1i1OAUX6W4rIGPwi4FnqJJA1lHoJ6EZQI\\\\\\\/vBXJhb2Ga7qTrjw8P1DwCX9dIIGnvRJt+sWTAdWsUgEcDYJiL1BXmaxdn1w+5evvgaPdiYgenw6N9CIBCW7BNRhja9iiS3tPsE7jrftJurMTlaNE1iUStS3QamCEtSC5s\\\\\\\/aUvZW5ZuQP7tqYtO89fm2Pk\\\\\\\/CII9QWTb4pBUSRRDrxalIfy3egq0PZH4lF7Y5coGL3t6lp9OvMqIpPMLT12hwMCKZukYs7RwxW0KkozyeT2iIsQPcn0uRmYw8Lk7Vlja1Fb\\\\\\\/vILCDTsPZTJ+KrUAME7ijvQMsciW8X3SNJQugAaQcPInJmmbMdso\\\\\\\/5PEtxcbxL7+iNB5tMDkQisw7Ym2i4D2XMX+LzLgPMQvUGhrBwC9QrYxKqPzPQ\\\\\\\/tyoglhOHdvrhcYfEBMdKi2Dak3pF8nBrOdGbCIiBGnctc6ixCmqIPfBctA0Djy1hWOTpNqeyJr6Y6idSv\\\\\\\/aes3hx8iu5TEnq9kSQvhgZQhoX08WhNQaVoM9Vc2KbjoSbEe9TXzzqUzDqiOzpNDNPbgyOjOHHY\\\\\\\/M92cdgK3wHOlGmZBWkK0XA2kk\\\\\\\/Bj8otGPz+e+NozFgTdd1I6jP8ZQt2iQkK3Fny17L6VV0p+rvtxfg88eXPl8hRzXrnPid\\\\\\\/MpHejF\\\\\\\/C7NUMXPN8IYi6cLnAKFEALeS4E6uqGi7VQeKlJOvryej\\\\\\\/8O8J8z7yixd\\\\\\\/L0zvwWpnAX0Ssfkb7sLHiYZOwWr3mUSmDuTnCl9iBV\\\\\\\/nOLXfbn95IjIBfXmRzY0EzLFHyr7SOmQgX4UBg2FXdLpSmt4E8GKYmvI799hS\\\\\\\/9XnEbvOK\\\\\\\/2RSLaTHRS+CX1Q5sxZYeSHj7\\\\\\\/02vNIbYIeeVdk8OjaGZTWMRVz3w12k3G5SNxUJR5nSoKJ2kTiyZvb7kfcNmyboCS616dZcmX+CV2KuCKg17U6AyiwhnmciCzOhHk8zckEKiK8YaSSVyVaRIhuagb3JMiiV0azG7LyaS7Z\\\\\\\/c3+YxOOt73LBz33XozmVe5clKs\\\\\\\/ebofNU07wT1N0V3WMrkTXsVAX9JOn3auiEyon8Y+IXWrRmYgRdE1\\\\\\\/eA9Bgi\\\\\\\/VVtVNn92FXbXkgt8UrkYIW8eHExLHWami69QYnPf21uvygJsWzIfnmpBmkg4pv+SbltmT9EqdoR6DimJxQRFqwXMmpzsGvBlX6AT2gFW8n596dv22tvWmxbQ2o4U2CWjqcWT0sSic5dGX7u\\\\\\\/tTyuJgllCmzZe\\\\\\\/XDKO3kaT6vHffp6+AKzx8JG+ivWKStg2I6v\\\\\\\/4YF7tVLGSDEKYVAt4rv1nMS3zYDLNbk4WYQ99TjohQMp4lvdez2QyrerIhivBfeh\\\\\\\/yYJat8q0dmPaXloBcSbSB0BanC5Hk3XPCwihVtTVsMDXb2ZlMnlTqcCwJxv0m+rih2O6gAEE7x1fnYVt0iR0zU4Oh3bodPLTJo1TCCcqMZRUJRhCbh7doaOE9V0tdyQswhfFg0yGP8PpeiyX1MJgncueko+TY6MsOX+2P+gZmkICsvQQmL14v4w0QpZK7aNN3V1e5pNRWVP4ojz5r97AjrYFGB+89Dxovbng6lcGMI88412JGORrJ4NTksPLrqpQBNcYVeuC9KT\\\\\\\/bYwC\\\\\\\/SWG+8euNZZG\\\\\\\/AxVvPUUtAwF5R45lGAZpG9goPonyWyu9YM3VYunu+CNstM6wEVDXy3WN8Pli2rny6D3361yrYeWwc\\\\\\\/uZQtOj1LELJIQRUc9in3Jc8jjPOt3BG3PekZ4n2FeN4bZrCunugw3tZTOnKnhzVSl3YfyMdqUQH\\\\\\\/arJI77yH4uakG7YsdY\\\\\\\/4hsh+XaGQb95gsPJWPn4E6joyWI24EbIfjAKBrXdrhxI2PXk5vOSLJjZ4rLUhS9PbPssk1+E+qduvbWwzNZCqSKCcbeQKRRxSHOnN8hCb6fuEFhXnKM5uvgFWQ78AuSdgFi0aOXtVmCQNEC+gZEO1c46yTT7Ea5X2o2linCsG8LO\\\\\\\/hspunJ67aHcVKung54\\\\\\\/tyxj6sopGb7pucoZFYLB162wEqR89FqdXGOKf9G0qX\\\\\\\/R2ykbvIC6urfoOY9XUc5MlqPcg3vQ2eOOD9J7oOWH1H4zLBJ6r\\\\\\\/ecQG0orixilpHixVJQ96d+Yd0IVLxZBSqd+Em986TDysp997R2oLQMmRLpbemZXErMRBLlCrnBkCpIwd6vnxB8N3aTj9AW3WZVEG+MbElz\\\\\\\/25uftfF\\\\\\\/AWIph42jMVj+bhAEb5F107ecbI2Plo2sPLixnsf1l9\\\\\\\/WL9\\\\\\\/V6E8XjaU7u9Jv8PsjMl8Wk00F9lBsDcaz+cMmWs8GdsZ\\\\\\\/0uEVWwa39sSCZTPZ\\\\\\\/igGbZ+ZNEBJFWg280qweI+aJErDc2udF5SVHRLvwjXkWBNt9F3RPIochJe5NkrNHjMoErU1jEbvBKhqJ89zDsToHKU+W0l9mo2g8f8jJZrlufsJX1yrHgyI+yYJGG7ghjKcd22xyjzzIh45VWw6oip7GQ1b+GLWq08xHUSMJWgOr0Rj8VDvSRGGW1KiEC9cHLXGRyAu4jxSWCm1v\\\\\\\/fjmfFqLlHPyzpLA5D\\\\\\\/7qPxdmXUpZ9xQU0Y7yslbwxkPoD985mEObq29a+dv\\\\\\\/KbdY7L+camghCKYFTAnq+H03+8cH7hdBcpdWqet0XvVR0mEO5tkzdtyNkbmxKO5GJaQdV\\\\\\\/BAeq3JXaRv5ELDqYBllVZ3zB4tI0DchKla8IvkCkpRL+QDdP3uMdjB0553n\\\\\\\/uIPgztZWA3\\\\\\\/6PUcNqWkwb81jl7HBd9xmCE0q01fCHeHzckS5220+cz4Mi+58ZPxcr9Rn2nV0iq6ONAPAkibw4KNXofWEuj+xTK0vcKFA0+ngMaLnhE2KrgtsiK+Vob6Xtycw5IlLHB61F7If5yt0lvwNXkqBBhPlBV\\\\\\\/d6tKCRKei3xj60xUZq5lyoTF5h3ldffbkQGxabj5bipJlluN6Y4qeQdPw9YlySNfw0h0Hk\\\\\\\/gcpXWeagNyrOJ8AbnW++H4dv5LBNHxWu5wSG9qlHm8cP+2YPpD1MMPOJuZgdrlvW7+ekApJ8xiUgmVsqOcpcz1yt+SlmvAca2rtJrHgC3MVK65hPna6B17X7QnZXa5vNUuxWRVN1cCMThGbxqAkt1bC+qB6QN9aWiX6MhU2Arb9+0sDxi+s0yJth1fywf7E4ruCpUButoql6WsnCVXt30FbnzksCAN0tSxEaq0kUGEYyJ95OOUZMXg9HRZU\\\\\\\/w\\\\\\\/WZ9mUiJZjDirHUM9LE6YxxMs0F6FzzALVVCho9G6eHyCxWMXSH9MnOfV+dsF3RNlMQ42QVz8OPOesU7tjJ2U1Tbbide5pnmHt5GPV5jHAV3Tz+zXga89J3qcwO8Y4Sx\\\\\\\/1QrwyIFGXlNYJonRQASkyv0nm2VepJv3fy8aKvZGt3MFG0WS9+ZcBswRHjF+Udf8H6ItoQotiTX4dLRZPqPLaRm46Y9IBDmRqmozSgRGns1EqfbQKSes+L572eF2lNgwILvbB8eFoPsJwFq7Oe792BSSV7EEwnFTVrljNUt1pltvB8kyqx8s9jT6TBv82VuUi3pyiTKTq+ZqaPiAM173459nrauKNX+3hkgF6SAZiBB9iKvb59WzIr5uFuqqCBtl5KjBeYM7CaCE1JxHtDr3RWD+sCJfdyphhLKXFgjooiWFEl2dtD\\\\\\\/17YpJDtB0XHP2cS1Y0MmMlSX7kKsfFbLsJd+E1gpHdSRcaFMxgQ5KGEelawNTYF5fXMzgR+FSqq3hnPnfTHYF9O6twhNBIqI3CaNNTPq5sfLlV0AfErjt1JpAvielaauBLkuyPzSCCkpHVjdOgNzhjzbJbAs8gNn5mIOSndmamgQ13JtS0RP+N9t52IXoSgIR6Y5XMCCypbgIduZO6\\\\\\\/QGhz3F7zBnCnWbKm6gPQr+dv4cRXmKhrXFvENVhMI\\\\\\\/weZ61qXZgizLYtY18\\\\\\\/2AdSF\\\\\\\/lR1AbgQQdlyVgYcCn9wdHYtoBS4Ob\\\\\\\/EFED9uH57tfz36sH0z6EmmMaRby1wf4ZPkxnT+2klPv9nfQNOrR4AIC9RKDXsL7ieq1k\\\\\\\/s1SLeneuxedXgtNK9POXfcQj\\\\\\\/C5tlrBBpD16unYwhxuGT6v3mmyuZToEO2oGcITP232ogW0oY0uB8CRAzfVHkby4yyBJQ3sBFW3R2gO4CWoaG1HnQf6uQ1ur2d0GQbVifg3cefqfJESq5wpTjq7Y34T9WHn\\\\\\\/geutFjjcYlELm+4hEIcf27KTec5mvr3SreAOjzwwyWGompG9tR3ZDFvsWUxUI8Z2q0lQNK5Ke44eAtClqDLRA+VUSn\\\\\\\/fyCZLc+jkav9tk1\\\\\\\/vCz6iDozQIxtENjAbuPxQzOxGAzdL\\\\\\\/2uRMeBQSv8W3SN7uhC7YqQgxgExHltMHtNl\\\\\\\/H5W5X4eJ3en4a\\\\\\\/\\\\\\\/+aR8VjwWVARxm+442bbyltEddxUmWp0ylIplxd3S8HnNTRgLaouhPw8dQjdIpF2LhTfvrlqHR8nuW\\\\\\\/ee1f4RKd3\\\\\\\/kJRi1IimpuTPUP+dVU5Omerg7PbIJouivr+eMqvu5AT3FttDjFV6nBtqknOTSyMdY8kUB+2D9Rpj+Ys83QFe9UDJVzkMbzKcRChOmsYtNjm2dtRsaKhdLG+tI9\\\\\\\/jLxteiWFwEasr8\\\\\\\/Is0XRlfRUC0iKQvXl4dnNXz1O\\\\\\\/nPfhLKJZIaG4YvHo6bR+YvXTU+rRdy0oluZSmhF8GwJD3\\\\\\\/5UCfTjkPxRAIZGwOLJ1YeowoXnfn1m8y\\\\\\\/IwSPGsGHWe029I4LPI\\\\\\\/cltkp4pAInjkVEgltveQBSL\\\\\\\/SG2ukFk7XcWnhkpIZzdq+OgorSbpVUUSqo3AG7VWekQ1\\\\\\\/saC\\\\\\\/fwrZbyj8Er3BlRoESyCHCPcmbNQZ6jo2rujsLpZIQzvVizNS0GatzEH+L+iUcx4L3D9hON6qChRkex65DrB8kKJ6JbfVxx4HfBIFBtGxFQFml4yIuZxFonl2n6A+ZdGE6oNKlBCZ+s+MsN01wcp84zmaFlKypVjzFyjqfYIPuiY46fubhlCCA8IVdKFYLj7l2r0l8Qe49IGa\\\\\\\/ERShfzfAJ\\\\\\\/+iOaXq7bXFYkDUatxsJpLNfsGCqJLSniNMryJZHpHBOB5TZ88CNIYKjDbFgRkABJW4anAL7W9qRqZK4XF1bWnennR4IxlUXRzWZHk2nCPnRPKoQYeq5d6XyRrf8VUriXv\\\\\\\/HZX3h0m0DAA3QaflEkbvxMj\\\\\\\/k8MEKx8zHT\\\\\\\/NjbVk\\\\\\\/RAB9Z0oKeng2frZS2+XP7mFDUy70t6kzYmsl96bWUbWsmF7TRBVwI939urbDKowoNEexHwP7wcZpYNR2gBg8y7Yz\\\\\\\/qvG43\\\\\\\/U+rqmCcG5XC6pL4SJuiigKCfJ0MTaGaGMTS0oRpztLrvlieNMYMIVAObYmLwFtYtN9vn7ZJAUti+VsmnFg2hpU+rviZEurI19sQODvzQnjz5zpirD8JjrdQ32nIbUCVA4auJ\\\\\\\/G9OpYj+gtIwPtG1TxQ5oev7RWuEKyiTddqMZHEaBdNei8q\\\\\\\/Pn3D4VuRqU\\\\\\\/mDP8Q8yBP8QPxosRTm0f8dYY\\\\\\\/hsb0olckIioLossZSob0QLvbtIuu8idVVn9SprZQawRTPMrLabNKBEqdbZze3Ec5m2c1eGM7Mfl8qnAhTC601LHCQsFcMIH42FePdf2pQdVAOeXtMxydROn5itg\\\\\\\/qx5oR5BUuJzE68G9KHIThL7+Ehdyy2RaBCQlPNrPBzRZ3n7LoKoCHtfjj9ELGyxNMV0QJPnBv7o2qROSmpPFRfXn2mNG4KGoQe7boXdXUWsSmdmpx9vHu2DN0I51wJC+97hXRBeuG0YOqKbKVnPducPKz7xoZ1108eMyzzHpY5sfqP9VToEYL1yhx4knqUdmCN2IiZT2BeYyDnWXwUVWULTgq0s698VuChaY1VXvVtb6Sa0kk5EL2Ae9zVixYqNoy9bLGZj+IKY21YD4jYBhsiEufwv27JutfSyQK3tlnkz8JhNyoLre9SMbJ1PS7nmnWn\\\\\\\/bBPD1XL2RieuCQ26F1LgcGeTAP9y4L6eweORwFuq0A6bqcfTvQ3qctca3GQmbmH5iCYblT5gikAm1Wixn8gOsywUKu9soO+kn8h0ItVmgjgrdL9zQozJfFW75PE3+me7X4WSGTey1U+mEKvvRMA7CKl1DSUw+iylt6cB4zyc4pfBER5YhJbeBW6gDqkjwNFoVChn4wGoctXp62XVsfakZulCuMIhk+58flZ61+93ni14PDFt0CDFzzmUABJ\\\\\\\/ZXvE6evTz1MGvhYw5nhnKJ+KI\\\\\\\/qMXHlczg+iJUfXL0GQZr\\\\\\\/RiiXfOtg0Rweh8oP80i7iuH2WJqoAZ2qtof4g2rJFjBTrlrirGKgycnYx2epxi+mSE6DD8VPPaAeAlog1mdJYYGcPWXU6h3OhqoHT6btiQvf+IvugXlqS7Xiqt\\\\\\\/Xv550oxam529uF8FrP\\\\\\\/mM7w9zclkD10YM1+ekfu6qVmxRbBZ2bxdurRGMIOdjgkei4aGpnzbDYbZU1bNN5aIb7p1EuN3OecGAlrdOXx8Q0uPFfT0YLSL9BnarktjfgK35cuk2Z4Nfcf\\\\\\\/UpmWSarxY4eB+xu+AURrgzcxhSTWEJ\\\\\\\/NnCmgl393RZZcVQP8VdBQgeUAkPgBkKbkFMIH8EQSjC5m8oDYO3ewX1B\\\\\\\/FN6Dtd3zyOYZQJppojdVvayQAACz4Z+5XILSwwxu5qJ3+52H8bTtJZX789GPdxBoAe2rB6S6uCgtfDa\\\\\\\/nHkGP79CYZph8qVbLnsE0qN5ZJqEPBQkbQGL2ZHMBQGPq0cD7g3CmVca6WpbGTjU3mqvsJs7nGTa44oqqNSWD9g+wQS0PbEwWuMejcosDIjJaYNOaQYHRhCS+23Us8I4aG7yy3k8mNdpq9qBm\\\\\\\/dB0zTDZOMH3XOGimFWqFIrQiSk9\\\\\\\/uqVrhRqiqKojoLn6i7FIQefgXzkT+rQKxdomtXdkAsbLXs0cFwPB9qVcIUUJLL7ZH1+yLgpH1h93Z7WW+VIseYVzNNcjrI7GcjVqpykEtMRbUqOsg+p8bstHs3OEghSo\\\\\\\/MV\\\\\\\/UigoKiqOkLelTqG4LL4J0ycnYSelQHNd4qnf3ZchRkjW0zky8XxUB7vz2J2JZG5Pjf0KaTwsAad7HIzVkA7QyV5Rg8b4D8XtQOHjdLn2cNU6Tp2ON8nCpUSuSjH3FsrdcUHAEsl01rBZE3QtDOHV9awZWk7k6DnFuGCNvajn0f4QWFC1yKmCeI1fUoAacVChLJwgSxLqmI1IYL6\\\\\\\/s1db5EVoS3gZJvM4A7P+\\\\\\\/Haig1eAuedUmBRHA8YfqZGTbG6y\\\\\\\/zWgcrQYUe1dpOb\\\\\\\/gPxhugQ1BZbYOefRhBbwu2+eci39255DB7xI2xAlN6kikC1HJuRYi7GzKD\\\\\\\/pyZ5EV6jL67qEmgsfQDKMoEPkhOyOLvbXcV91gMNl8ikaAVPDnUHl1NY+J5UBY31dx1aMxB4pdBaQywZpNtaN48IdwfK150dOFm9HaOrHtI6k+hfW5oWHsPaeYgSc7RflPVA+s6IyT5mqemIVafut8jVM9m2lknsOk\\\\\\\/N7b1dffYbcOUtjotLThXBDc8Mmfypinf1WnmROCq1nJuduvQe0QkrGUr\\\\\\\/6\\\\\\\/TicGZoo29pYoSw3pzUab4qbvh\\\\\\\/Tzf8Q5WufKiw\\\\\\\/J5evOgZsX6tO2DFZKehFQqSWAHtc+MO95eJS48xdX4ypAHlGO0dMA\\\\\\\/RE3Tb1cHYdotylByLjuH6U6iWmwZKVc8+pfAWwt7A4cA4WO58cQpwjY\\\\\\\/R38bdbkbOQ87RL2WGWz9tFm4iIIO47KxDP5muLsi6wxzVPD\\\\\\\/05CEaNyzTK26ZYIrMBGSJ6P5BfqgQKfuDW4qynuEnLvJHWHrL9aO5mou6wQjZP21ixBtMkAkKxSCGZZTJVgkTzCHv5KS8m3l3Komt7dkF8YoxpDP75Oklk4wQysZ8r2QRuHSNj8H4dSIGighd7\\\\\\\/QWNMP16kWckVMM6ptPrpqpST6urV8OYeOCmm7GoDv7063Fusmm3l0rpmquc9p6aja9Lx731RS6YWOawTSTyxm08iQfNKaCchkBJtmCnvvLWOl6H\\\\\\\/ZeCO3OzAQLTVBBKvcnzPklsFng4+hhG\\\\\\\/8JkeQO06Fr0QhEOWxayke+bg\\\\\\\/4y8owhV5pcUGCGxE0M02CYeAHfmlIuoCbYruQaNC112nn6nSoKaQcUlaUjxL\\\\\\\/UXsS71d8nNFud48QShfi8rZp2\\\\\\\/kjfdp4Ne5Bc\\\\\\\/kFuPbcIFz5EBZoZly\\\\\\\/aptVrcjjU7P\\\\\\\/AkSyx+bXwgiwQgJsw5fhbC1xhpVq7fTklmXNAqFp2\\\\\\\/1ptK6YmSiBuSZdOtvtbmfmomWf\\\\\\\/aKM67fuxMNfi\\\\\\\/gLF1DJkufc5NAJCvg9svU\\\\\\\/gvbofv1KhjhoR2p0syQkr\\\\\\\/n2xQ8vMunAHduYuYhS\\\\\\\/LOLn\\\\\\\/FZ\\\\\\\/BEZ77a1f0jCqyA4BGcBCR1BPtuTG2VWnv21x8Ffo1MvRqHV51ixwWKs3WpaPfjHD1oYjKd86yRdEHlAcnGnT57zSJbmSZxxzwDs55cTTbppcRyNNpCPC2\\\\\\\/DHMTayGNaRmCStEAOEncSMtKg0197S3opnu5Aq0\\\\\\\/w6kXv8l0LJG7JE1tLIe4VOz+DCsCt1OXqI8mr4wbyGJxsIr1YG5zsjlcu53rSmrtEVEzpniJ82J66AKkQQkwdqrGD\\\\\\\/gp2UOyaHrtLhtavrA4yERs3NJkUKLcX9Wv4Jn3gbo4\\\\\\\/cqGYeQ4d8Pg3oqRgJ5Kb58Jz7fvfQS2h\\\\\\\/I43S\\\\\\\/7PUQWlIA0P3+kKLuJGJ0iObWue01nWLgO1I+AOaumllfl3G23zDTMxdAkNIY2BlRk0JdvvLTckg8ApKnRxb6LOiy\\\\\\\/Krj3IfBUfrHL+pQC\\\\\\\/HZHge+F18y3EJurfown8rsvTVdpLe\\\\\\\/mWntgYdaCUVFyoe24q5p8FNByCFNctkolNK6FvYLELvzgeZQ5KquNVmgodQaOABZeygTK83Y20pIeznn8zG1vXkHFtlGqvudjzHAeEYvqtmRrgfWydY1UBx3rwt5Biu5YODVAunNMwG+FcR4R0vH\\\\\\\/v4MhmJfGN0VLdVPeyE\\\\\\\/Zn3TehxoTDD\\\\\\\/k1KG1wSklYQAs4Zg+5Jw\\\\\\\/+O5AUZ8iObO\\\\\\\/svEwpYQ9Lzk8CErLF06Qxb1T51z5jVa2+k4fX4ev7EO1D8UOdPoL2lqQ27gb8e\\\\\\\/j5mgcJuefZRWAZS4+X9DCu69XrmNmOWhQpVUE3okjILO\\\\\\\/deUAPnVDhUiaLSqVMUG+CC9Fww4PAqtbC2p9MKU7KyEwlZWUscwCbva1zOOmFYZhY4jMr6aueILwCvyM0qReWtAZtf68QnoFo+nSnCg8jNFAdViyO9yuWAZdA+6B67BRLDggZWgZyUkDRXEA58l6AA7FLJ2SZ0zL5QQDTxCIwMoVCnYsuNdnoem3P7FKKCvsjKLqTGFGE8GFSe7CIsIvpOs\\\\\\\/46an7Z+n6xWUwefkDgj8wD7p9Y2ejIbsKQXnlPhCtqH0hXX6LNJqmqCoEhwLTLxL7TfDgi6bI3AlB47VMzJGgG4nhiU7ZXlPaHt7Iw+UriMpzEQIW8FRMy45lGVo0KhaPW7ptQzAwHXjeSz0x4i4Nip7nbwdjoH8uVxbwyaAEORp45jkCKKkrBsPitndn\\\\\\\/\\\\\\\/6htW7AtG3jb\\\\\\\/pVoqZQsgyXFBR+xnRUdITM9WTIMZA+vtDhIGdDBycr\\\\\\\/6idT6xc7IFX0nLFH4Wl8ihlCjwNLXLgt2BlFiNHX4rOWGC4lu2AbKLi6GdHmtokDaw6wG7QNtz6iT+lyFwMIfGEV01cysAOtgGispTfmsHDI8dLMah2654OA8agjRiCj3bxFGZQ9W6FELik7YFVJn25iGAtjB8YCd5RDZXRv0i\\\\\\\/S7QGyyQCIt1gvqIhdKLHaFw+z9Q7f+jBJbClwbIrFF4ChvN7xHIp1CvNogptdWr3M\\\\\\\/MJD+7q+7BzfQZ3OsaonK0MS+llyly8ISQaIFscf+7x8qeQPjdQZ7Dyz43h2EvYRZw6nPix41ogrAHoVePteFBTvg92SmHTwkQEbhy6BFLjllP5yDey9D7Ouz5jdMHXnn55u\\\\\\\/OaPP+PUcLKjM5RBcV92eULDHJzs4gfyEqSqJxQXJmyI4o\\\\\\\/+Ufc2n9RSA2WXoZqOGT2L2Oi6qt\\\\\\\/Iuxvz+Ixff55yt1aBf0v+GtvVdDnyHvL5UqslnAfLYS4\\\\\\\/8+6DyFWUo6q1vokAGM0uCqbm+XNbfY3ovRDvDX29NZRKO4ueJaeLtNb+fttLGR\\\\\\\/nM\\\\\\\/zs0ttNS2szudgt2GVRmdZOO5urZUvHlX0Uwd9h6VtxrWGXYt9fFHnaRbg7KzgxEwLUILKIHWMtlAQjg1HBnhalCW+m0IjHzx0DTNmppBxz60x6AAy7vo77np2B8Xbj+3avVr5057UNcEXsWLvPeWwcHdTFVK9QntbDSyg+WPDzY\\\\\\\/F0ydQM5WRY91GQfmDa+cc05zlhbcEaGBJmx6IwIRao4R1OZOcZLjRbMnkNa3UGkdTnEbBUSPnFVrCRwlXPYmgPJz4YTqIQaFMMWZ9sGdA3WfTO6ehgnMCeGpayg3xDKcA\\\\\\\/vgNcghxjUuXRepMGlxFxOej55Tieg1HmI7m27AC4peVhH1PQKVAYiVBBEumuj8g\\\\\\\/83TxcXvv2la9JIjOP3IGdInSi8eshg65z6TXIrwQExMgEDpvZYCQ7Bw8recQGhNMtF\\\\\\\/zK+Kb6NOgTkxs4JmNXn\\\\\\\/4L5KfguPbwyuO5\\\\\\\/dsCkh5iyhPl+Vb5MXfGy5SS4py5+FKphii8xbvXWAC+gUbiqDcmHh59X81KQ90d1DBOGpv+DW9A9FHFJv13q\\\\\\\/JGYP3En9V8H5uB1b7noXUmRVF0BJyCxwsie83pH1rZoXdyfPHbuLf1BnWlhqLovTeXq91XvnWWRtMKgkMR3pw770OFv7MZ3mDhzf2CAqVNE7JNGmUtYiYol11Ei21+7jTyk0eGv9jkKqvqzMDtzAll2DByG8IRsfkFWpflyqPXvXhFFTaEvYEOPeBajZHqpmfiISne2XMJYo9xI6ZL4HA+70OnUmVwSnT35XkqpS3fFt8sDWqEho9fDusDiFG2LMfzB9KvSTJY45jaXQ3skUH0PKeVX7eIurRT\\\\\\\/jbjvfMwbjw4fye07sHYr5UOZu5byG7qU6NAUjqXYL57rnKi7aY1l05pkY\\\\\\\/ksC1usEkZjCGSmSj4PI3DPotrqsgGClgvF+MgN9MNNMGI2\\\\\\\/8BRkWUoiLhhhzm3eLCFOif\\\\\\\/5wdYBZ2SkYkkHC4xhJFsTF86BB5MXcSWwGsiDFMf3b3i3FzslUu\\\\\\\/kPWskYqLfQvRgjEF3hv+zojQ\\\\\\\/TrzjLNqy1XePUXT+BZjtXVnAniqwDLp60eIzg+ro6RwLM1YoVqvSazB0lW2JLTlQOaAP6bVMhWpUSHK+QdZiAqmAMHo+IGyMYsaSpk4TWroQuYEcGUYsR9azI5Ca8d8rL2PnzPRT5xRXzEdMT02IOiV\\\\\\\/9Qw0zRsL4mK1cSBn0l+1vMWpa7VO7EhZ3cBi1g6WjbnxWnOIdFckOdpGDHF8to5YSt0XSREZyksLMezkaMw6K7sjw9opwleHyZql5Nqm9VAD9oXZG2AiGv9gAEyDIWRvE\\\\\\\/zPTfyJ9KPp3vtAyoB1B8rs9znZLHf9m3PQX+Q3EEV6CvXsRqsZGXZVs3xDAL0k3cdZWL8NR21SItrLGx9cf5JXUPTh1BbrLhbl+u2QSZHQ97pNn2DrArEkS\\\\\\\/CqQrMFOuvD6f6H2KdOlQjSaZ8V8ItwxfN+iXsQl5+Ihp\\\\\\\/ygMq5lex0wF7Do+jxeCslor79HYLBjLbiQjHwgC9EKwVvzO44COn0SClvSTAtYTsMf2udT5inJfaFLB9IojNrlN4bj+MYK9iYCSNsK3ZgSN+DIp7GGvM3RnlY3xbtJ4o0W4HK1nGRDzVzNvKi6H0RcPYKRkGgiUpqWcA2e2ZzyERSRb7NEzy\\\\\\\/cKaCo0jW3oK337fxsVI8BXoZ8l4\\\\\\\/q\\\\\\\/sIH562Vp\\\\\\\/BKpdKdvyhQfucXZHfJr6gkYfNOChbogDynJkHMkc3XiepCyXxkHFI9mjS7anl82qNQsCzLDruvCVLHNSES8f9bPwdnjlpYo0DcAtAEAGfcisFWgWebWgZlj9Z\\\\\\\/KesOn9Gw0Q+YRmtz9GWgMPezeuhfjEaRKiAbIOQMb0QKgbo0\\\\\\\/MVKga4\\\\\\\/\\\\\\\/f5YJYcmaXx2PoqlL3HNo9ZGGinZgb9z0TWJPj37HgLRqD\\\\\\\/SbC5GxM6pL3QQNz0C\\\\\\\/KSc0xxoJW6EZLjxEAplgHPl1Q+muhpMeckLh5Bx\\\\\\\/SB0JKVWLp4F3SUp0486mzCxs2hzyB8xhfyvDr+9ilPwGE236lhyxi3w1lmDEI9KwCGlW6puNMepdoTK9j8I63+mPvotwkFRE9yYVpZbrDBOM5Wt7WTrmPMBt22JTiRgpWZfsKKyhYxeMaPGleYP8OD6qIORjOhikXEYzkseRq3ntI6mJL8sy\\\\\\\/g0CE18PFtXkkiZ1CATQWZnIjiqbZg27oLbIAD7wixaGmHbVW4oX9hrAj75onEvKJpdeSu70Z8\\\\\\\/+9vENNL5xpmQOau9dG1gwmqh3FK6+j0yFr37RS2Kg9ezPyBnM\\\\\\\/iA3CjqNPazX6eFFKNwq1sKcxGL3FCuJdkY3bLwTgOv6E8XuZcY7hcteWZGcZjoi6VNi0lIQ+Lc3U\\\\\\\/5jwND6XtYaVmC9Zwv+apcMLXooiNwtdvQuGJh9vYbfNewKXcA0VAPsplU3O0P+68DXN57uE1UfZz5p+7pPEm9iCFD9wGwS5tCEoJYoOlTyq0GsDxPsZ\\\\\\\/RiS3WgLU37wDZzFJNxDTmJUTdqXyZv3vzElIWauSHbCRCP\\\\\\\/P2p74KNA0qj\\\\\\\/QcIIYi2F9Ad7l9vGrF20DgzqNpOxKqKVHdBF\\\\\\\/zQML5sccZSIjpoVZs+Gc9vZ65gGN3KHGYP40fHFRiwfK4BvGk7KKoh6DAkazdFRTqdAG9BCTUlDcbDvXBGsyA3XFHp58HRhJgdxsksM97YcoXmZVl9qsf30OGVbCYS3N2rRo7Uwqbo1o1d\\\\\\\/dCb5aO8R1OTMgzZQ8VQ+jO6RcJgx5Qwn0Gy0GWRT9Wu9ZarFxnc7VYywFoGmSaFH4lr8Wkuj+mITbvbrQ6lGUFCW1wuVzsxd0cixHzDVBZ3tf9RrNmYnjnhfzyOB3y1G8dKmSY0yQh8YDLnL22J+Cr5HH79qcRt2c2YxPLv30fxBscHAEWhEACBg6gL0usmCOdAU67W7WzTrLOKVm89zKpTGxQxoYUXfS1nQsU8M1PjZKmcrvhtLBTRApG69ZJX7+b6e+qzT0Ho8LqgWGWoB\\\\\\\/llcipJNuxfZvbRjlgTfBBWzryeXOta3ab4UfC2GhiV9sPJakKlR4DaWrlY8pGKmKfisf8KzCbTHqdC5sY1wl5cOBwUZwVeIiZAJDW02EDy9rirUAXEomt2tNrSkqv7u8qFgjaXa0ovhebn4nBWO1VD\\\\\\\/tyc44T6OUDN5RNWkuEM\\\\\\\/ywJAgLUA4ZbVsIujNpI3FNe7sxurUQm03wNhCeMEgZTufBHcgvhlUQkbpvxXu35tmZk9xz5w45YV1lv1pM5JT6uPb4NlOgLOPY2pnsosOebZzDoJG7bi5ePl0729kwA\\\\\\\/biCJk+NNIrpr2n5ZrQX2BcDvqa0HJrc4NIc8++96EVa37bL5QtGpA0oymtAjI0+SVOwW2\\\\\\\/ACaPk7sZyxL9milLj\\\\\\\/Jzjr+mNlK5+rS7xaDjNZVQS7owZ1\\\\\\\/zM1HzOeIzRKUlaGXJ5wtFG2dAEEKiYk364jIZ63C\\\\\\\/+5Io6wZ9QrVlvgyobE8UQTq\\\\\\\/qlSLt4x4qpboDkatBHdRQHicsdAxCSKnB5Oi9BHEn89RSudpKM0wkWtRNGr966VVBezR3avSImJu6pAoEymqJWOoHiPi88mxg6ApF4X4gsJLJIMVpu4BRpWlEWNia7z8KtlHBJvmtjQ7aau\\\\\\\/uPSAillEl6JCisSnY0ObQXPQd\\\\\\\/3eALqMJrXuYPq3I0RoDvPi7Q1M\\\\\\\/L8AHbivq74TI55gU2IgoQmDZgKPVWMa+aVDEqkMAy8ClnoE\\\\\\\/cmSvVP7NHZ1HHA08GO0th06WDplL+M9YOji78pomBBZXZuBvzh+yG85Qydv+vXccJKN8A\\\\\\\/L18WtS\\\\\\\/hg6MFxK5ltuhgiifmGhECZ22qn0ZzsmVvqA9p4GQnYgVqfllAlFwEpeAxHj4FQsRKjhsBIerve66CwVXk6QAO2AzzoULiQeplTFG2XMF8ryVwg+BwbNBTwMQ5VFzXDMidXLqBscH0GsQbpYu5sGwCQiuew9om9XXiDDruSpC4MANcxL8gTM8QSM37Z6yy3jaA\\\\\\\/wlhjJWsHwDsGYQeDZm8fTYjC97qx3p29mITgAoQPuifLx\\\\\\\/Xd2MJ+Rct7iTbkhSGtQJWsMFYpNOa\\\\\\\/Thu3UkxklJtuMo8JyHXfcqKMLHbJD+x382LrN4uX\\\\\\\/tqtTprI0XEI5NGzjg0u7vTDMGlCXJTJowFy9fSONvu7FKw1ngkS1VoVEiBWOkhXTm6S5Udp88QCvPGPf1pKGv4BfnRHMWcLp1I2WZBzG0HCUp\\\\\\\/UkBUbIa9drFoQzYLlYawa7Eq8y\\\\\\\/uhD0xa1EqtzK6MTY0q1iFcP4wspcY1wrwD7deEMSknpXum61GZ7WyId8uuuwVil5JlnxbfiexfaWSqAFbq0jZFbDFBMCu0VHodimIBXNwJNVnErUq38GRaFYfM5KDNykv2kqn10UKnbB51SBvz56TPh\\\\\\\/eG3fw6RXU5+rCPgUO7XJ8Wmq+axWkxf178DWdqBMufmrof8yiqAypu8MvxxMyeFdC7qGEoqpZZMdIKlYcoZ1OeXPkwpk0owrmL81b2n274r4VDqD2kq5QrtCGrBsSFrLCT5ga5XUJGl2a0xqI2Up+ObwM\\\\\\\/eTa4JadhJGeLAM88NwuqVlv4K10gn\\\\\\\/D0S\\\\\\\/rsWUqsvhvF7sP2qad8Sm07ikOfmQ5esa10X2KLWWniqLugkilrWpXYrpMkbkNvKrQuNeBC7d54txcI2pYT8g44WbNmlexSk8XJMpkeAGEV5Qpn6eivV1SpsQanNttyYsRg+Jzyz2FaHCEngqgg3jhC7jSnwyElLWQI6F5HVXgtNRL3nZCRVhyQH0\\\\\\\/kx85Qis4bonzDqEs2ACCW2TLtm\\\\\\\/IHMx3nLnbkwEecLU+VlX9n40f2fWGKAkxdXuuChpKRPdIEypiZa0fKEd\\\\\\\/pzSKD1qSjHt\\\\\\\/buU+qIceU3WRjBpcoMLxRFC4tGrFBv+Kld4Jg2xedpyEDUwC7IAvWGIsFdLArWT\\\\\\\/1C7MhPUVxX\\\\\\\/BIeKR8v1SbtTYw5pV+WbQD3aQ4C0adwiqGRVlShBvXj9y7tRP3uGnWXZ8zT3DIWFwMOFJfJSiJvTaa92gEvV9O57AB3TrbSxLs8ra8U8jHWby6JPU5D+pX+P+OpdiNzoRRyaqg1Pxqhbomp4lC04BhaCkH7bEyWM\\\\\\\/OMjLeIhgitBrvcgZfLvt0QdxTpZK+D+lOSjbLQO2ilNKZvHKh2oLEcU6fY+qdCqj3Tc+Tfw4A+7p3giZbECs2x41mWfzvFyb3EXzGoOxF2SupPBl9qnxhQloR5kTVWCKgSr5lhmoflIIg\\\\\\\/C96GGvzyPjRWQI7ipvh+h\\\\\\\/kro1QBE\\\\\\\/hhvdFEk8EeLQfCnbUe2AO93MtqGeFComKoIaFKV++2YTr1jxURu+gqxPlflv\\\\\\\/Z6zxHgu6itg3vjUqnnYb\\\\\\\/mTzBhvHDxrJ6kcKrJ8lH1hzE7AKYHCUKeWwkGhriWdZl1RCTETlq6+sMHby8mCNcq1FiW9hMMIOfwmfAaLHoOUQ0hm2eX2MKqK4Gn8K3P2++1IWbXiGgKL+J7RfQekyY6A+fmjEDgAX\\\\\\\/H+5I9vO0F3PV7gSpaoCP+917b85CvSrQhPLiEFdV9h1vQYyKdH74VDce8XI5lSBDtJxwfxhP9WrqYihMPepafB\\\\\\\/DseJQGnGBfqY4rVUa8g9cKtj9KBHgIGjFP3Mok3KDPXtKIE6UXIJlBL7bWr7fPqweuUqS4gWokL3EWCKJzy2VYK5hSsutEKDg7GT8u2VIB+Xdjf864goPC3GI646smhcewe+hrrGg+zBFWlrypLjuXZ7sE6mXEnn2Viq8d9i0PPKfLn2W6HBFUcn+UxqvAzyuSBHRRnYWjqJIFvRcCVfuowp7\\\\\\\/ywLWsw0aNHN59jJ93PPD06X\\\\\\\/Uyj3w2D7mKlsv2tgYmKuufN\\\\\\\/OKyMsY+NbaLSuPzuUdoPxYjIHQ9UNc\\\\\\\/tfBRUhNfM31bvBfXHM06esbtGnDV2sLvbQz++HVx73T6++WFsq0lu22mcvjKJCeIbuIu8eQvLaXprQTrQsKVKm3JWiYmRQGwIFQPUGn\\\\\\\/z7wpWFKhYtfRnd7v3LGPnpm+uSikz+zHqGy+5jTkPmkjcxP27Ms7NEjK6RCMkugP2DF5rq\\\\\\\/L0etd1P8ergQudpquzXZH2SutB7r\\\\\\\/k9Bpy7PegYP\\\\\\\/KtfPQMeO3TEc74yQfOVh1omnf+iUtyH0fXheGppigrxln5e7xI306tM8JQRbdZQpHkFrm6mrpZulw7rnBasx4GhgxUSPm0lJvXZ+yGyW0t3wZOKdaOD4PHUYjd4Lp7LxPMF9edhIYG0YsU\\\\\\\/VtChY8S1EgQRjeXTIliuzVziWIUCczG09gkK2\\\\\\\/2hZnCzW+DEYBVyrwlT3LcNx+S8NvGwH6tbvYhKJj0Sw5+vdw3IyEAU8YtSuN8oWZDNbxtkOZierPbFk+DTorR5bEPmuWJ5I54z5t95s+boPkgGw0Djt1vGclLF9pYxqf5PyiVI+RW8xlf213ckOyWLmC1vrnIuQ+rekefKaOTnmtuMQzTFXb7yoEau+jTg2HJ5TQOteaIj3GmRz1s5iC0W9hgwjRzAWJFgDBbzTFFdDnAFITd4HrQuCgACjw4OeD3Dmm5zg04CvqL0RWxTZguKjezrKzc45XV3cjZQr+SGPMvrrOwaJqDDJcx\\\\\\\/p56OeNqVy4r2WjkJ7ma7cMEt85sAzDxjLXA7210ZacR0QFCRYHNPHLAVDisc+K9k+yYcG8RTvQGuFAjx6\\\\\\\/+r\\\\\\\/xtzeOH97Bfpga0ipM7FDNG2XPUISAMwQyMEWKR4tOy9pbkOW7MtS6Y6Zt88hsYa6tJvbvGLY+5EZsi8Kw+m9WrzUlch38r8Xctt74BG+oecJuPsB7LxhJxKADyy0ZbaHMtmA+IljlvV822cLvvo7iMhmzyFIWBmvrl7zZp6INejp1Wynrsp6sajqY6l2LQqd8Z2plIrIR+m4F7C9OyCx623PSfKC0+E+prrmhmZwdke3MrYL8BNTlpJHKydGQeMyohCtnkI5Yvz477\\\\\\\/ZYA+4AnqN3Su0QlOoMVKvc3JORsK1XaAG3vnNFB3XnrMSNHrjvtHKEIIjFDwnB747Wg9BrBeaegGPM7d1Oukl1WfcS8dqorUuusaM\\\\\\\/faXU6t60Yw1+Dytd+fTLaJNPVdBzw148lFllso2g2yY4qKlEPSBMeZFGXr8culWjXH1R\\\\\\\/Szm8vlyWUYSk32BLlI7EIdjixIKb+72gJPcBqWiZOLumviID8kB\\\\\\\/rs3pQfwgWCl2RVC4BALM2PdBwQnkQ9aIEsFjeUWDdYwkEGc9YvmH8B2pjT5xrMXRPE2FdQD9s+2cXEWH5wxe38RDDOqlrXFy9nHlKyOD8Ae4oVu\\\\\\\/PKEK0b3CWdEEYqqJjXOg+pV+K884Nyq9fOSpFKIhy3Ix3uo7Z+rbCm+JUfvRigi7bVC6w+f4qBUQpPDLLPWL+6a7iJe\\\\\\\/IykyenEb89gRPwng5YeZP0JiQ4TPBCh8z+LYNZqXR+vE0EPFDtditmspRr5asEVhmx8s5GQUUavmMlC8aoesScAUq\\\\\\\/8S44aR1tTYZYxq997297xuDlvodmscPMdTdlVDsByYxy7Pke9pHfsdLYKUBloyGzuIwafZIG1Bfbp6M+d9Km9UPhuOCX4YovBM5qZlUV7qQbeyjNNUfhyPNYbrSmWkMg4sokQsOE4kB2HOKyvnxpJR34g\\\\\\\/x4tAtr3IpwfT75kkOIRgrKU0IGXh3oh34TJFD7pTW8SR6XS\\\\\\\/1nZxbFo3x7nQCyXBEvkjVZ91fSx7OpY3uG2L\\\\\\\/17+cIoJrR4G0aUh3rZjwrYSCZsdBcbMEeiwTBiRIxXfYoQw\\\\\\\/iYbL\\\\\\\/9eOtDeCq71ssaN5CBlTkbHfSEozp9Kiq0PkAWzH3lspbaMA4VilCQHB\\\\\\\/PvlOHKitlpqnWZsFBFB6bLbZIb3CWNjh7dU2ZGJ8Aa4fLnkr9O0zKKeogEWiNfODkM9RsDzXQY8o5rd8RkIsCgwW842ICyxKO\\\\\\\/OfUAE5oKK7MqtvLv7sWUESWKqamzOtEwUWBiHAOTzfqW2ddghcB7wi8qvW88pOqXH\\\\\\\/JSdQV2Jyvz6YB5nCJjMqdNOmEEwx03vZdLckrL7G2uxZe6NYjKo063Ec9AwVuNZ4S4GRsw8xYScofqFXWz3rgyjySf6nXkEKU0NiJZ3FkUgsc44JYKVV0DVfPYsep0fI+GolpviHCoza12314g97TMr5V91KSp0ABAIsVsffXNBroYbow\\\\\\\/ccE9fiTGP4oyktW45TT0S1yVU++4IJEBgxQYwFw+B+ItrJQIKQFkqv8L0yuskurKnDqlplqInltDNVnuEcCYPL2gkqAGA20Wll03dbiEy4RCR767lZGS6kGt6GAHc9+dVOcaz8\\\\\\\/ruf45MnxQVp5GThC\\\\\\\/Ccv543gR3k2H4doH76DNg7hhJ9GSOGjRHi+cJz09KQvAExFcu7d8GcOqcorgxuoR4P0sS4L9tcaIJbbldTzE3raRflubCVsSTYc\\\\\\\/tcWVW2SUB6QRDvtFdrFLmr0P9uqIobC2oc2jEn\\\\\\\/1Dr+1sFfAU4VeSaPpGn4vxWXsCNQsOQdAuoXXAkjAb0+CC2DCnMoplfx8F379Xps1wBxQfMifCmdB2C5b\\\\\\\/qGrI5OFWVCHEcgBiQwN\\\\\\\/dixsGkAW2UcNmacsybJCNTAEgbAER0Pni9tVh527TSzmFxdt1b+MrPoJx95xdXlnjR4jaKXSM5odo158BYvXGexfqmz2Cd3GW4ycgIUI\\\\\\\/\\\\\\\/2lxZF+r5c3kqMBf9+OfQfK9Zdvdfalitr20temLFKTZV73CiksgVdWl49n6xaqtjrPBUlRfdmw58DFykfkvf4AzUidsj7B6LbJ03VHuA+Dk4LCaoxZcw7hCweSCbzkXHch3xN12zK8b+I8ZIYg5+cLAgAvNwpD+cRQ1hx7TAGf4eb4KVkX7Wg45loXOMPyynD4VGqBKJrjlhu9BKfCi\\\\\\\/\\\\\\\/Q3ST4b2w6nLDAXCJR6Qv9LogRRvsufdC+vAjOQuj1sGRQBSztsi0foOYR6Jcc+YmX\\\\\\\/nmuT9HqTok6Cm4r1YxtYZ5JAhNIpLPI\\\\\\\/nukyAKfU0s7l2rmtyLlxStSU3+\\\\\\\/nph9y4DX27KZDQ5GP3U2E\\\\\\\/fLRAWRLUA7fa5HvcwUlk27gPSKx8zfES0nqX7nky3II83Pu1swPhr1kMxa52m0+ECeaUChCxopYrYxHqtpEIoZo46jJnat52jsWsYzyjbog+3wfg9vCKn1+pUIjpAvfEet0EHBPAa3yqJoOE4SYqi1CBw0o1pnPqE0yn\\\\\\\/bmwKiPEmd1lpCC6+rV8V0wH5s848L78FlgjZCsGMRv2NXQK9b3J6rTS9Bp8JkUbIdkhPTB\\\\\\\/sNdSEWtVrKctA7G8uKMmTZRFsm2O\\\\\\\/eHziSlhAu66MjOwHdHse28Am8abaiD8MAG0mPnqNA5fxbk\\\\\\\/38rx9DHISppZeW+X64VWl7zlBBGKyl\\\\\\\/2\\\\\\\/85RAnZrnovEx44unBTh\\\\\\\/8cy7imn8ZkBCIRTwlRxdQYVbYzVeayz25H1vFPnuiqQHPsKrBLNwULe7TqMBPHqFc97ioqrmrpwusEQHoPOp2\\\\\\\/24dlocLk+BI87YWvE3tjH+0HRKln1g+HZMrLGnA4gSzgOTb6kBl9YQepfr3hlPNo8fmo0AkuGPWP1kOuSne5zZmKU\\\\\\\/IwnsCim+DCciYNZ\\\\\\\/HYnf7YgTk2s6Qyc8xdo7w+3Xl5C6dButZWERAD25lYo1o0FoJg7MrkbFAUsE8b339A45DxIRcJYxp+d8m2k80qaINvrhQuT30b97Tapt42Rqev6JQScJ\\\\\\\/9f8aKvFXBTK5WfNdSROwvIQy2dTTrz2WvAZEl57fnhCGXWL67mgKDX3fyqfCscYKMX\\\\\\\/nFTbzGMjfdET\\\\\\\/PyeiqrwqL4CtOtOdkFPdaxpYmTBjHIrOgeVfXW2oPcgRlXzpCvHyAHcm8Nng0S\\\\\\\/bUuhIWfKGa1emy+WXmp3gDtw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022CRhsBgw0wYdMRdUU1UdIOw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222872cd7adb028751bd4915402e5ac7ddbfd7f4eea8329a87f1e8595212fc342e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NnhTn7k2KPyhApW+8UkfLeYuPxY88oBKp2RUoI9RhJxTK5TquVV3H4hZVLUsz2DgvW9fxRrRUadlSe6FMK60EfFecUuC9PGUHI0FOEdT8e5lqFLR0OT7ikXwKh+NVImSHC4TIkQkS+S+6MDeQS7iWKT8ItOiVSovyzg7KDi2zLyotmqZ8gA42zy1yEQZlC2jyu3H3heCqCNWy7MoygNNIx8OY7a9GdOwZb7NLEwq8rujLsE+iWw0puqMogoWq5Bo6PLHcQC\\\\\\\/3G1bWTAHmK0Qh+91XIxh7JsT7exadBgktJ6Q9nqGCASPJsCdrO2eyIb3DbIXnbDsMtKhV7yB8C2CIJffbjh5abYXN1I2CQIsTlmZ\\\\\\\/SV1tg4QBp+gbNVzODD5dQRlVSHLoSC3bQVjS8nRJqU2bKYXmMRWtNNsSiwL3ttv862snwjEw6jmpNZ0DaPsz6cVnYGReOeVnuVzuMHgGLS+oyRS1c7G+S+sEHPhJ9pzlbYxssRE+tlrj7NiRGRZAR\\\\\\\/vxsiS46TIz+GOyPKSFlZzFTR7minNzD47RSxd7yELTud8eXP6HVLTlcSTBr3sMjOeHfdUfUN4rJBYx4qGu7D1Lfoll6ruAAOUHxDg1nG0HEbCiLMv8Ci+9xhUdjCXZazrrqYDFJZM9hY7Xp1sbkXZJ\\\\\\\/FEt7lqobhPx\\\\\\\/G9YmcQR1mOEwIftpF928QW1cQsZxEdfG6l6BGxN+iMG4rzmi2Lwf5SkdasxzAJciRaoNWewKIbWANuUgJ0oKU1KPcjFhHVUjxNYF33vFByEkui8w0n6+Cai5c5ENciQgyitS7CkuUmOCdesFZ8d8R2M5xk3BOKp5VGPvAyA8YHMA5yFPptd4kVkOt15E+sp\\\\\\\/kFpESV0JntGiW8FTwB9W8O68K2SC4ZxuM4CEYTT+4a1W3AzuQQ4on3Hw1ccm0HX8SAw9ESdqZAU444HUnjIF11Z5h3veFH70PodLG51g4y+XwsxfyMud3xgMEJzv5S91LZkleUzTpuxMEg65mWA0PAVSFLSA7Gm3tTmssF22xQt40fHtYABfhfjIAuln04DljCKzbdEiD0+3ofd4PsmPLfQvdVa5BOYxF4VVDyhMzCoxEPdFdFOh9dMKHReB5tvFBUdCYBEXdwI0sav+M8LsINObggxwHZMyWAHt\\\\\\\/TUY5AeK\\\\\\\/cRWedJHyR4ZX6Bua24z6eEvPTEc4w8XiC+TWhPwvIRfRGvWiHExYDUsbLsPfxJ0HaXPs5Kb5IyAnld\\\\\\\/QYQNnCR3xZsq7DEYyCTo\\\\\\\/na4Pvux6frOnxc93slofWXXNWhuXDcJqYU0894DyyjKvm+cDu3JRvqI5MJpfaCxcp+rASLnCLWefgYve6xPDtmkChOy6WVAV+fmJQeobGCe6sp0i6FRUaEVJxztfkeR54a75PVQH6fX9uMCzst9r\\\\\\\/XfseTcV3eAkLuuQPGRqSmoBv1hjmNAdJr5U83EEXee0YYN28Y+qXbK2UO6NkW+GD0ec4u0yLjeWfymS5ewed8UzfrsuHAUM2\\\\\\\/n0coHKaSQ+h6Pjy4xcPEVVb6ghFDETqevsoGOpdtxaosCFJ\\\\\\\/5co7Hhlwjie58mcJ6CPcWqmZa2tnzwFGXpt\\\\\\\/XrErXm41gEkP43ASs0zm2i0NLwigOxmUwU8lpjFG3dKrZwXrJrPYDcVHWcdiseCjwdYZ1G5TYcZ\\\\\\\/LntPjnPQ8cb7ijuAvxkztXS8zfWaruI6SkpBbHzX+JTLLjZFUAXdVUyqTzko2PKpcGHgNOUpHPcVib5Epi1iZSOnozpPMji\\\\\\\/netreXxKCt94vxdHaPl1HlrgbLlDgeW5ephBbzqHkwCsk5\\\\\\\/YbIjjqy\\\\\\\/kiTMpTW+KkjEIbsA9VgOAgEZ18OzWpCUZBwFpThSxFihDWuWkoYI3pAAqUyc8EYFabKkxrcWLUpv8ryl7lW+SQxYktYB62hocBje8NqxyjRPo7ayGWukh94tIw7iZooxnOnp2wSz2COa6JY8tdBkQdsq7mIOpVvU+j1DYJLw6Lfo3Gq5+w==\\\u0022,\\\u0022iv\\\u0022:\\\u00227TDCbiKSjCAgcrbNxiAT5w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226cceb5a075229327b06896d38aaa7980460378b3c68a2450dc31f9687441ce77\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459744 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2433" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-s6AEGq5QB-6EaqCxjV716_AsX5jq3dEij9vHuFGPAhQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2433" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3K9lYT2bHW9Fs0PP1L4nk3EoblZX9ThQ1LWOrhBaLR8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/12/2425" title="Statistical Inference of Selection and Divergence of the Rice Blast Resistance Gene Pi-ta" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/12/2451" title="A Screen for F1 Hybrid Male Rescue Reveals No Major-Effect Hybrid Lethality Loci in the Drosophila melanogaster Autosomal Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459744" id="node459744" data-pisa="ggg;4/12/2433" data-pisa-master="ggg;g3.114.014860" data-apath="/ggg/4/12/2433.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evidence for Stabilizing Selection on Codon Usage in Chromosomal Rearrangements of <em>Drosophila pseudoobscura</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4765-9227" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zachary L. Fuller</span>, <span class="highwire-citation-author" data-delta="1">Gwilym D. Haynes</span>, <span class="highwire-citation-author" data-delta="2">Dianhui Zhu</span>, <span class="highwire-citation-author" data-delta="3">Matthew Batterton</span>, <span class="highwire-citation-author" data-delta="4">Hsu Chao</span>, <span class="highwire-citation-author" data-delta="5">Shannon Dugan</span>, <span class="highwire-citation-author" data-delta="6">Mehwish Javaid</span>, <span class="highwire-citation-author" data-delta="7">Joy C. Jayaseelan</span>, <span class="highwire-citation-author" data-delta="8">Sandra Lee</span>, <span class="highwire-citation-author" data-delta="9">Mingmei Li</span>, <span class="highwire-citation-author" data-delta="10">Fiona Ongeri</span>, <span class="highwire-citation-author" data-delta="11">Sulan Qi</span>, <span class="highwire-citation-author" data-delta="12">Yi Han</span>, <span class="highwire-citation-author" data-delta="13">Harshavardhan Doddapaneni</span>, <span class="highwire-citation-author" data-delta="14">Stephen Richards</span> and <span class="highwire-citation-author" data-delta="15">Stephen W. Schaeffer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2433-2449;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014860 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459744" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zachary L. Fuller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BL.%2BFuller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuller%20ZL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZachary%2BL.%2BFuller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4765-9227" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zachary L. Fuller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zlf105@psu.edu" class="" data-icon-position="" data-hide-link-title="0">zlf105@psu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gwilym D. Haynes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gwilym%2BD.%2BHaynes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haynes%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGwilym%2BD.%2BHaynes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dianhui Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dianhui%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADianhui%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew Batterton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BBatterton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Batterton%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BBatterton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hsu Chao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hsu%2BChao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHsu%2BChao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Shannon Dugan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BDugan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dugan%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShannon%2BDugan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mehwish Javaid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mehwish%2BJavaid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Javaid%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMehwish%2BJavaid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Joy C. Jayaseelan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joy%2BC.%2BJayaseelan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jayaseelan%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoy%2BC.%2BJayaseelan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sandra Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Mingmei Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mingmei%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMingmei%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Fiona Ongeri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fiona%2BOngeri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ongeri%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFiona%2BOngeri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Sulan Qi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sulan%2BQi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qi%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASulan%2BQi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Yi Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yi%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYi%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Harshavardhan Doddapaneni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harshavardhan%2BDoddapaneni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doddapaneni%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarshavardhan%2BDoddapaneni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Stephen Richards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BRichards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Richards%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BRichards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Stephen W. Schaeffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BW.%2BSchaeffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schaeffer%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BW.%2BSchaeffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/12/2433" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459744" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459744/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2433.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459744" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459744/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2433.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459744" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459744/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2433.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459744" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459744/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/12/2433.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459744" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459744/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">There has been a renewed interest in investigating the role of stabilizing selection acting on genome-wide traits such as codon usage bias. Codon bias, when synonymous codons are used at unequal frequencies, occurs in a wide variety of taxa. Standard evolutionary models explain the maintenance of codon bias through a balance of genetic drift, mutation and weak purifying selection. The efficacy of selection is expected to be reduced in regions of suppressed recombination. Contrary to observations in <em>Drosophila melanogaster</em>, some recent studies have failed to detect a relationship between the recombination rate, intensity of selection acting at synonymous sites, and the magnitude of codon bias as predicted under these standard models. Here, we examined codon bias in 2798 protein coding loci on the third chromosome of <em>D. pseudoobscura</em> using whole-genome sequences of 47 individuals, representing five common third chromosome gene arrangements. Fine-scale recombination maps were constructed using more than 1 million segregating sites. As expected, recombination was demonstrated to be significantly suppressed between chromosome arrangements, allowing for a direct examination of the relationship between recombination, selection, and codon bias. As with other <em>Drosophila</em> species, we observe a strong mutational bias away from the most frequently used codons. We find the rate of synonymous and nonsynonymous polymorphism is variable between different amino acids. However, we do not observe a reduction in codon bias or the strength of selection in regions of suppressed recombination as expected. Instead, we find that the interaction between weak stabilizing selection and mutational bias likely plays a role in shaping the composition of synonymous codons across the third chromosome in <em>D. pseudoobscura</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/codon-bias" class="hw-term hw-article-keyword hw-article-keyword-codon-bias" rel="nofollow">codon bias</a></li><li class="kwd"><a href="/keyword/stabilizing-selection" class="hw-term hw-article-keyword hw-article-keyword-stabilizing-selection" rel="nofollow">stabilizing selection</a></li><li class="kwd"><a href="/keyword/chromosomal-inversions" class="hw-term hw-article-keyword hw-article-keyword-chromosomal-inversions" rel="nofollow">chromosomal inversions</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li></ul><p id="p-4">The majority of amino acids can be encoded by multiple or synonymous codons because the genetic code is degenerate. In the absence of mutation, drift, or selection, synonymous codons are expected to occur at equal frequencies for each amino acid (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Hershberg and Petrov 2008</a>). In a wide variety of species, ranging from viruses to mammals, synonymous codons are used at different frequencies, a phenomenon known as codon bias (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ikemura 1981</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akashi 1994</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Lynn <em>et al.</em> 2002</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Novella <em>et al.</em> 2004</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Parmley <em>et al.</em> 2006</a>). Explanations for codon bias have traditionally fallen into two classes. The neutral explanation hypothesizes that bias exists because of nonrandom mutation patterns and genetic drift. Alternatively, weak directional selection has been hypothesized to generate codon bias through differences in translational accuracy and/or efficiency of an mRNA in the production of their encoded proteins (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akashi 1994</a>). Standard evolutionary models often incorporate weak purifying selection, together with the effects of mutation and drift, to explain the phenomenon of codon usage bias (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Li 1987</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Bulmer 1991</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">McVean and Charlesworth 1999</a>). Recently, however, there has been a renewed interest in the possible role of stabilizing selection on quantitative traits such as codon usage bias that extend from the neutral theory models of <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Kimura (1981</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">1983</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Charlesworth 2013</a>).</p><p id="p-5">The most frequently used synonymous codon is known as the “major” codon, whereas those used less frequently are known as “minor” codons. Mutations changing a minor codon to a major codon, hence increasing the frequency of major codons within a gene, are known as “preferred” changes. Alternatively, “unpreferred” mutations decrease the frequency of major codons by changing a major codon to a minor one. The magnitude of codon bias varies considerably between genes within a species and is found to be positively correlated with the relative level of gene expression (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Sharp and Li 1987</a>). It has been shown that artificially changing the numbers of major codons in a gene can alter the level of expression (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Carlini and Stephan 2003</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Hense <em>et al.</em> 2010</a>).</p><p id="p-6">The effective population size, <em>N<sub>e</sub></em>, is thought to be a major factor contributing to the efficacy of natural selection (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Charlesworth 2009</a>). For many traits, selection can act more strongly and more efficiently in larger populations, as has been demonstrated in several species of <em>Drosophila</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Andolfatto <em>et al.</em> 2011</a>). However, recent theoretical models of stabilizing selection predict that if sufficient mutational bias exists to alter the population mean of a quantitative trait away from its optimum value, the intensity of selection is nearly independent of <em>N<sub>e</sub></em> (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Charlesworth 2013</a>). Standard models of selection on codon usage predict that the intensity of selection scales with <em>N<sub>e</sub></em> (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Li 1987</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Bulmer 1991</a>).Within a species, <em>N<sub>e</sub></em> can vary within the genome of a species due to differences in recombination rates across chromosomes and the effects of background selection. Regions of the genome where recombination is suppressed have a lower <em>N<sub>e</sub></em> than genomic regions where recombination is unimpeded (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Comeron <em>et al.</em> 1999</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Comeron and Kreitman 2002</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Gossmann <em>et al.</em> 2011</a>). If codon usage is sensitive to <em>N<sub>e</sub></em> or the rate of recombination, one would expect to observe differences in the frequency of major codons in regions of suppressed recombination, where <em>N<sub>e</sub></em> is reduced, relative to regions where recombination is not inhibited.</p><p id="p-7">Suppressed recombination is expected to reduce the efficacy of natural selection due to linkage disequilibrium among loci under selection, an effect known as Hill-Robertson interference (HRI; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Hill and Robertson 1966</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">McVean and Charlesworth 2000</a>). Consistent with this expectation, several studies in <em>D. melanogaster</em> and various other <em>Drosophila</em> species have demonstrated that the level of codon bias is drastically reduced in regions of limited recombination that generate up to an approximate ten-fold reduction in <em>N<sub>e</sub></em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bachtrog 2003</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">2007</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Haddrill <em>et al.</em> 2007</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Betancourt <em>et al.</em> 2009</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Arguello <em>et al.</em> 2010</a>). However, recent work has highlighted cases in which the observed patterns of codon usage and base composition contradict predictions based on Hill-Robertson interference. <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Campos <em>et al.</em> (2013)</a> estimated the effective population size of the X chromosome to be approximately three-quarters of the autosomes in a Rwandan population of <em>D</em><em>. melanogaster</em> yet still observed significantly greater levels of codon bias on the X. This pattern of codon usage cannot be explained by mutational bias or biased gene conversion either, and is instead suggestive of either stronger selection on the X or a departure from traditional models of directional selection on synonymous sites. Additionally, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Kliman (2014)</a> did not find a significant association between recombination and the strength of selection on codon usage on the second chromosome of <em>D. pseudoobscura</em>. These recent results emphasize the complex interaction of mutation, recombination and natural selection in determining the pattern of base composition and codon usage in a genome, as well as suggest a possible role of stabilizing selection.</p><p id="p-8">The third chromosome of <em>D. pseudoobscura</em> is highly polymorphic for different gene arrangements that were generated through a series of &gt;30 overlapping paracentric inversion mutations. This gene arrangement polymorphism has been stable in natural populations for more than 70 years (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Dobzhansky and Sturtevant 1938</a>). Frequencies of particular chromosome arrangements form a classical cline across the species range in the American southwest despite the predicted effects of extensive migration, indicating the potential adaptive nature of the inversion mutations (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>; <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Schaeffer <em>et al.</em> 2003</a>; <a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Wallace <em>et al.</em> 2011</a>, <a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">2013</a>). Several molecular studies have estimated the population scaled migration rate, <em>N<sub>e</sub>m</em>, to be greater than 1, which is sufficient to homogenize gene arrangement frequencies (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Riley <em>et al.</em> 1989</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Schaeffer and Miller 1992</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Kovacevic and Schaeffer 2000</a>; <a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">Schaeffer <em>et al.</em> 2003</a>). An unrooted network representing the phylogenetic relationship (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>) between the most common gene arrangements has been soundly established through a long history of cytological and molecular studies dating back to the pioneering work of <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Dobzhansky and Sturtevant (1938)</a>. One arrangement was arbitrarily chosen as the Standard (ST) chromosome, and each new arrangement discovered was named after the location where it was first collected. These include the Arrowhead (AR), Pikes Peak (PP), Tree Line (TL), and Chiricahua (CH) arrangements. The inferred ancestral arrangement at the center of the phylogeny, Hypothetical (HY), has never been observed in nature. Distances vary between inversion breakpoints for each arrangement (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), ranging from the ST→PP inversion, which spans more than 12 Mb, to the HY→ST inversion, which spans just less than 3 Mb. The closest breakpoint to either the centromere or telomere is within 2.5 Mb (S. W. Schaeffer, unpublished data). Recombination is not hypothesized to be suppressed between heterokaryotypes in these homosequential regions.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The third chromosome inversion polymorphism of D. pseudoobscura. (A) The frequency and spatial distribution of D. pseudoobscura third chromosome gene arrangements (after Schaeffer 2008). Arrowhead (blue) has the largest distribution and is observed in all defined niches across the southwestern United States. (B) The relative frequency of arrangements in niches moving west to east. (C) The inversion phylogeny of D. pseudoobscura third chromosome arrangements. The Hypothetical (HY) arrangement is ancestral; however, it has never been observed in nature. Tree Line (TL) and Chiricahua (CH) are derived from the Santa Cruz (SC) arrangement, whereas Pikes Peak (PP) and Arrowhead (AR) are derived from the Standard (ST) arrangement. Each colored arrow represents an inversion event." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1380927261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The third chromosome inversion polymorphism of &lt;em&gt;D. pseudoobscura&lt;/em&gt;. (A) The frequency and spatial distribution of &lt;em&gt;D. pseudoobscura&lt;/em&gt; third chromosome gene arrangements (after Schaeffer 2008). Arrowhead (blue) has the largest distribution and is observed in all defined niches across the southwestern United States. (B) The relative frequency of arrangements in niches moving west to east. (C) The inversion phylogeny of &lt;em&gt;D. pseudoobscura&lt;/em&gt; third chromosome arrangements. The Hypothetical (HY) arrangement is ancestral; however, it has never been observed in nature. Tree Line (TL) and Chiricahua (CH) are derived from the Santa Cruz (SC) arrangement, whereas Pikes Peak (PP) and Arrowhead (AR) are derived from the Standard (ST) arrangement. Each colored arrow represents an inversion event.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2433/F1.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/12/2433/F1.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465293" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">The third chromosome inversion polymorphism of <em>D. pseudoobscura</em>. (A) The frequency and spatial distribution of <em>D. pseudoobscura</em> third chromosome gene arrangements (after <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Schaeffer 2008</a>). Arrowhead (blue) has the largest distribution and is observed in all defined niches across the southwestern United States. (B) The relative frequency of arrangements in niches moving west to east. (C) The inversion phylogeny of <em>D. pseudoobscura</em> third chromosome arrangements. The Hypothetical (HY) arrangement is ancestral; however, it has never been observed in nature. Tree Line (TL) and Chiricahua (CH) are derived from the Santa Cruz (SC) arrangement, whereas Pikes Peak (PP) and Arrowhead (AR) are derived from the Standard (ST) arrangement. Each colored arrow represents an inversion event.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465449/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465449/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Approximate location of inversion breakpoints on the third chromosome</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">Chromosomal inversions are known to suppress recombination between arrangement heterokaryotopes (<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Sturtevant and Beadle 1936</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Andolfatto <em>et al.</em> 2001</a>). Recombination is a fundamental evolutionary process having a strong impact on the genome-wide pattern of nucleotide variation and allelic combinations. Reduced recombination may inhibit the accumulation of deleterious recessive mutations (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Nei <em>et al.</em> 1967</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Ohta and Kojima 1968</a>), trap combinations of positively interacting alleles (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Dobzhansky 1950</a>; <a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">Wallace 1968</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Charlesworth and Charlesworth 1973</a>) or capture a set of genes conferring greater fitness to a particular local habitat while preventing exchange with migrant alleles (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Kirkpatrick and Barton 2006</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ayala <em>et al.</em> 2013</a>). Suppressed recombination between chromosome arrangements has been documented extensively (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Dobzhansky and Epling 1948</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Roberts 1976</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Ishii and Charlesworth 1977</a>), yet evidence suggests that this is nonuniform across an inversion, as genetic flux among arrangements has been observed within the boundaries of inversion breakpoints resulting from gene conversion or of double crossovers (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Kovacevic and Schaeffer 2000</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Andolfatto <em>et al.</em> 2001</a>; <a id="xref-ref-97-3" class="xref-bibr" href="#ref-97">Schaeffer <em>et al.</em> 2003</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Kennington and Hoffmann 2013</a>). Thus, the effect of inversion polymorphisms on the fine-scale rate of recombination at a chromosome-wide level is not well understood.</p><p id="p-12">The polymorphic third chromosome arrangement system of <em>D. pseudoobscura</em> presents a unique opportunity to investigate predictions from recent theory of stabilizing selection on codon usage because of the altered recombinatorial landscape mediated by inversions. Here, we produce fine-scale estimates of the recombination rate between gene arrangements, examine the pattern of synonymous and nonsynonymous polymorphism, and test for the presence of selection acting on codon usage by using full third chromosome sequences of 47 <em>D. pseudoobscura</em> individuals representative of five different gene arrangement backgrounds. Previous studies mapping recombination rates between chromosome inversions in <em>Drosophila</em> have involved complex crossing schemes of inversion heterokaryotypes or relied on estimating frequencies of double recombination events among progeny with genotypic markers separated by several hundred kilobases (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Mestres <em>et al.</em> 1998</a>; <a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Stevison <em>et al.</em> 2011</a>). In this study, we simply analyze a dense set of polymorphic sites across the chromosome, allowing us to construct fine-scale recombination maps inferred from the pattern of linkage between pairs of single-nucleotide polymorphisms (SNPs). By taking a high-throughput sequencing approach, we are able to identify more than 1 million SNPs among the 47 individuals across the third chromosome, permitting us to consider a large site frequency spectrum (SFS). This rich data set allows us to test whether selection acts on codon usage on the third chromosome and directly investigate how variation in recombination rates affects the intensity of selection.</p><p id="p-13">We find that recombination is significantly suppressed within the inverted regions and the level of synonymous site diversity is elevated between gene arrangements in the overlapping inverted regions. In contrast, both the rate of recombination and the level of nucleotide diversity at synonymous sites are consistent between inversion breakpoints within individual gene arrangements. We find that rates of synonymous and nonsynonymous polymorphism differ between amino acids. The pattern of codon usage bias and strength of selection acting on synonymous sites appears similar between gene arrangements despite varying levels of recombination, effective population sizes, demographic histories, and the relative frequencies of the gene arrangements in natural populations. In contrast to several studies in <em>D. melanogaster</em>, but in agreement with recent work, we do not find a significant reduction in either the magnitude of codon bias or the efficacy of natural selection acting on codon usage in regions of suppressed recombination in <em>D. pseudoobscura</em>. We conclude that codon usage is likely under stabilizing selection and provide empirical evidence that supports recent theoretical models (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Charlesworth 2013</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Fly stocks</h3><p id="p-14">The genomes of 43 <em>D. pseudoobscura</em> strains were sequenced for this study. These strains were derived from individuals collected from Mount St. Helena, CA (six strains collected by Wyatt W. Anderson, University of Georgia), James Reserve, CA (12 strains collected by Wyatt W. Anderson, University of Georgia), Santa Cruz Island, CA (one strain collected by Luciano Matzkin, University of Alabama, Huntsville, AL), Kaibab National Forest, AZ (eight strains collect by S.W.S.), Davis Mountains, TX (11 strains collected by S.W.S.), and San Pablo Etla, Mexico (five strains collected by Therese A. Markow, University of California San Diego). These 43 strains were made isochromosomal for the third chromosome using either the Blade or Lobe balancer strains (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Dobzhansky and Queal 1938</a>). The genome sequences on Muller Elements A, A/D, B, and E are expected to be a mixture of wild and balancer strain chromosomes and were not considered further in this study. The genome sequences of an additional four <em>D. pseudoobscura</em> strains also were included in this study and were collected from Mather, CA (one inbred strain from the Drosophila Species Stock Center, next-generation sequence 75 bp reads, Mohamed Noor, Duke University), Mesa Verde, CO (one inbred strain, Reference Genome MV2−25, collected by Wyatt W. Anderson, whole-genome shotgun sequence; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Richards <em>et al.</em> 2005</a>), and Bosque del Apache, NM (two strains collected by Bryant McAllister, University of Iowa, next generation sequence 75 bp reads, Mohamed Noor, Duke University). For an outgroup, the genome of one <em>D. miranda</em> strain also was included (strain SP138 collected from Spray, OR kindly provided by Mohamed Noor, Duke University, Illumina 75 bp reads; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">McGaugh <em>et al.</em> 2012</a>). The <em>D. pseudoobscura</em> strains carry one of five widespread third chromosome gene arrangements: 15 AR, 8 ST, 9 PP, 8 TL, and 7 CH arrangements.</p></div><div id="sec-3" class="subsection"><h3>Illumina library construction and sequencing</h3><p id="p-15">Genomic DNA samples from a single male fly of each strain were purified using QIAGEN DNAeasy Blood and Tissue Kit following the manufacturer recommended instructions. The DNA isolation included an RNAse digestion step.</p><p id="p-16">High molecular weight double-strand genomic DNA samples were constructed into Illumina paired end libraries according to the manufacturer’s protocol (Illumina Inc.) with modifications as described here. In brief, 100−400 ng of genomic DNA in 50 µL of volume of low TE buffer was sheared into fragments of approximately 300 base pairs with the Covaris S2 or E210 system (Covaris, Inc. Woburn, MA). The setting was 10% Duty cycle, Intensity of 4200 Cycles per Burst, for 80 sec.</p><p id="p-17">Fragments were processed through DNA End-Repair in 100 μL containing sheared DNA, 10 μL of 10X buffer, 5 μL of EndRepair Enzyme Mix, and H<sub>2</sub>O (NEB<em>Ne</em>xt End-Repair Module; Cat. No. E6050L) at room temperature for 30 min; A-tailing was performed in 50 μL containing End-Repaired DNA, 5 μL of 10X buffer, 3 μL of Klenow Fragment (NEB<em>Ne</em>xt dA-Tailing Module; cat. no. E6053L) at 37° for 30 min, with each step followed by purification using QIAquick PCR purification kit (cat. no. 28106). Resulting fragments were ligated to Illumina multiplexing PE adaptors using the NEB<em>Ne</em>xt Quick Ligation Module (cat. no. E6056L). Ligation was incubated at room temperature for 30 min in 50 μL of reaction containing 10 μL of NEB<em>Ne</em>xt Quick Ligase Module (5X buffer), 2 μL of ligation of the Illumina multiplexing PE adaptors (15 µM), and 3 μL of NEB<em>Ne</em>xt Quick Ligase. After adapter ligation, the library was purified with QIAquick PCR purification kit (cat. no. 28106) and eluted into 20 μL of H<sub>2</sub>O. The library was then amplified in 50 μL of polymerase chain reaction (PCR) containing 20 μL of library DNA, 1 μL of Illumina PE 1.0, and 1 μL of Illumina IBC1-12 index primers (15 µM), and 25 μL of 2x Custom Platinum PCR Supermix HiFi (Life Technologies, cat. no. A12125) and 3 μL of H<sub>2</sub>O. The standard thermocycling for PCR was 30 sec at 95° for the initial denaturation followed by 8 cycles of 15 sec at 95°, 15 sec at 60°, and 30 sec at 70° and a final extension of 5 min at 70°. A 1.8× volume of Agencourt XP Beads (Beckman Coulter Genomics, Inc.; cat. no. A63882) was used to purify the PCR products. Library size distribution and quantification were analyzed using the Agilent Bioanalyzer 2100 DNA Chip 7500 (Agilent Technologies; cat. no. 5067−1506). Libraries were diluted to 10 nM using 10 mM Tris buffer with 0.1% Tween 20. Five libraries were pooled equally and passed for Illumina sequencing. Shotgun DNA libraries were sequenced on Illumina’s HiSequation 2000 system according to the manufacturer’s specifications. In brief, sequencing libraries were quantified with an Agilent 2100 Bioanalyzer. Cluster generations were performed on an Illumina cluster station. A total of 101 cycles of sequencing were carried out with five barcoded libraries per flow cell lane. Sequencing analysis was first done with the Illumina analysis pipeline. Sequencing image files were processed to generate base calls and phred-like base quality scores and to remove low-quality reads. The reads for the 43 strains have been deposited in the short read archive at the National Center for Biotechnology Information (accession numbers SRX204748-SRX204792, excluding SRX204786 and SRX204788).</p></div><div id="sec-4" class="subsection"><h3>Analysis of high-throughput sequencing reads and final dataset</h3><p id="p-18">The 101-bp next-generation sequencing reads were aligned to the <em>D. pseudoobscura</em> reference strain FlyBase version 2.27 using BWA (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Li and Durbin 2009</a>) with default parameters. GATK (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">McKenna <em>et al.</em> 2010</a>) software was used to remove duplicate sequence reads, recalibrate base quality scores, and locally realign regions around indels for BWA alignments (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Depristo <em>et al.</em> 2011</a>).The reference strain carries the AR arrangement (<a id="xref-ref-90-2" class="xref-bibr" href="#ref-90">Richards <em>et al.</em> 2005</a>). SNPs relative to the reference strain were called with GATK (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">McKenna <em>et al.</em> 2010</a>). SNPs with a quality score less than 30 were filtered out of the data set. Nucleotide sites with coverage less than two were filtered from the aligned read data set. On average, 96.066% of nucleotide sites on the third chromosome met our filtering criteria for each individual <em>D. pseudoobscura</em> strain (see <a href="http://www.g3journal.org/content/suppl/2014/10/16/g3.114.014860.DC1/014860SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/16/g3.114.014860.DC1/TableS1.pdf">Table S1</a>). Because our crossing scheme produced individuals isogenic for the third chromosome, we discarded sites called as heterozygous as they are likely misaligned reads located in highly repetitive regions. Of all polymorphic sites, 7.7–10.9% were called as heterozygous and we found no evidence for clusters of such sites that might suggest particular regions resisted becoming isogenic (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Langley <em>et al.</em> 2012</a>; see <a href="http://www.g3journal.org/content/suppl/2014/10/16/g3.114.014860.DC1/TableS2.pdf">Table S2</a>).The set of SNPs were used to construct an assembled sequence that incorporated nucleotide differences from the reference sequence.</p><p id="p-19">Using the <em>D. pseudoobscura</em> reference annotation version 2.27 available from FlyBase, we extracted the coding sequences of each gene. We used SAMtools to generate pileup files from the BAM files (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Li <em>et al.</em> 2009</a>). The pileup files were used to determine the coverage distributions for each of the strains. Genes that had coverage levels above the 99% confidence interval (see <a href="http://www.g3journal.org/content/suppl/2014/10/16/g3.114.014860.DC1/TableS3.pdf">Table S3</a>) were excluded from codon bias analysis because this could indicate the presence of undescribed paralogous genes or the misalignment of nearly identical reads (see <a href="http://www.g3journal.org/content/suppl/2014/10/16/g3.114.014860.DC1/TableS4.pdf">Table S4</a> for list of genes). After filtering, our final dataset included 2798 genes.</p></div><div id="sec-5" class="subsection"><h3>Construction of fine-scale recombination maps</h3><p id="p-20">Because recombination only occurs in females, the population scaled recombination rate is defined as <em>ρ</em> = <em>2N<sub>e</sub>r</em>. We used the program package LDhelmet (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Chan <em>et al.</em> 2012</a>) to compute fine-scale estimates of <em>ρ</em> for all 47 third chromosomes together, as well as independently for each arrangement. LDhelmet follows the same reverse-jump Markov Chain-Monte Carlo approach of other statistical methods, such as LDHat (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">McVean <em>et al.</em> 2004</a>), with several key modifications that improve the accuracy and robustness of recombination rate estimates. Missing data can be handled at no additional computational cost, because the two-locus likelihoods are computed by solving a system of recursion relations. Thus, missing data were not filtered from the input. For each dataset, we generated haplotype configurations with the recommended window size of 50 SNPs (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Chan <em>et al.</em> 2012</a>). Likelihood lookup tables were computed for the total dataset and each arrangement separately, using independent estimates of the mutation rate parameter, <em>θ</em>, obtained from all noncoding sites as a prior (<em>θ</em><sub>Total</sub> = 0.0134, <em>θ</em><sub>AR</sub> = 0.0064, <em>θ</em><sub>ST</sub> = 0.0051, <em>θ</em><sub>PP</sub> = 0.0056, <em>θ</em><sub>TL</sub> = 0.0071, <em>θ</em><sub>CH</sub> = 0.0065). The sampling step of LDhelmet also incorporates a general quadra-allelic mutation model that can handle different rates of mutation for different nucleotides. Inferring ancestral states from <em>D. miranda</em>, we followed the procedure of <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Chan <em>et al.</em> (2012)</a> to incorporate the mutational bias observed on the third chromosome. We also computed Pad<em>é</em> coefficient tables to improve the accuracy of the sampling step.</p><p id="p-21">We divided each dataset into blocks of 2000 SNPs that overlapped by 200 SNPs with adjacent blocks. For each block, following the procedure of <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Chan <em>et al.</em> (2012)</a>, we ran LDhelmet for 3,000,000 iterations after a burn-in of 300,000 iterations. As suggested for <em>D. melanogaster</em>, we used a block penalty of 50 (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Chan <em>et al.</em> 2012</a>). After the reverse-jump Markov Chain-Monte Carlo was completed, the map of the third chromosome for each arrangement was constructed by concatenating adjacent blocks after removing 200 SNPs from the end.</p></div><div id="sec-6" class="subsection"><h3>Estimates of codon bias, diversity, and polymorphism</h3><p id="p-22">The effective number of codons (<em>ENC</em>) codon adaptation index (<em>CAI</em>), and frequency of preferred codons (<em>Fop</em>) were estimated for each gene in each arrangement using the software package CodonW (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Peden 1999</a>). Custom Python scripts were used to estimate the numbers of nonsynonymous and synonymous sites, diversity (<em>π</em><sub>S</sub>) and polymorphism for each gene within each arrangement (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Nei 1987</a>). We also counted the number of synonymous and nonsynonymous segregating sites independently for each amino acid to compute the <a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">Watterson (1975)</a> estimator of <em>θ</em> using a custom Python script. Finally, a custom Python script was used to estimate Tajima’s <em>D</em> statistic (<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Tajima 1989</a>) using every site on the third chromosome for each arrangement.</p></div><div id="sec-7" class="subsection"><h3>Measuring the strength of association between conserved sites and major codons</h3><p id="p-23">The strength of association between conserved sites and major codons for an amino acid can be computed as an odds ratio, denoted here as ψ (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Drummond and Wilke 2008</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Porceddu <em>et al.</em> 2013</a>). As in <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Akashi (1994)</a>, 2 × 2 contingency tables are first constructed for each amino acid from the counts of major and minor codons observed at evolutionary conserved or variable sites. For an amino acid type <em>i</em>, ψ is computed as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2433/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2433/embed/mml-math-1.gif"/></noscript></span></span>where <em>a</em> is the number of major codons that encode <em>i</em> at a conserved site, <em>b</em> is the number of major codons that encode <em>i</em> at a variable site, <em>c</em> is the number of minor codons that encode <em>i</em> at a conserved site and <em>d</em> is the number of minor codons that encode <em>i</em> at a variable site. In a gene, counts for all amino acids can be combined using a Mantel-Haesenzl procedure (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Akashi 1994</a>). ψ was computed for all genes using a custom Python script.</p></div><div id="sec-8" class="subsection"><h3>Estimating the strength of selection at synonymous sites</h3><p id="p-24">The method of <a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">Zeng and Charlesworth (2009)</a> was used to estimate the strength of selection on codon bias, as well as mutational and demographic parameters. All analyses were conducted with a program kindly provided by Kai Zeng. The parameters are estimated from the SFS, and the model can account for departures from population equilibrium and the demographic effects of a recent population expansion. The full model, denoted as L<sub>1</sub>, includes the following parameters: <em>θ</em><sub>01</sub>, <em>θ</em><sub>10</sub>, <em>γ</em>, <em>g</em>, <em>τ</em><sub>i</sub>. At equilibrium <em>θ</em><sub>01</sub> = 4<em>N<sub>e</sub></em><em>µ</em><sub>01</sub>, where <em>µ</em><sub>01 i</sub>is the mutation rate between allele <em>a<sub>0</sub></em> and allele <em>a<sub>1</sub></em>, and <em>θ</em><sub>10</sub> = 4<em>N<sub>e</sub></em><em>µ</em> <sub>10</sub>, where <em>µ</em><sub>10 i</sub>is the mutation rate between allele <em>a<sub>1</sub></em> and allele <em>a<sub>0</sub></em><sub>.</sub> The bias between these two mutation rates is represented by the parameter <em>κ</em>. <em>γ</em> = 4<em>N<sub>e</sub>s</em>, where <em>s</em> is the selection coefficient. When population expansion is included, <em>g</em> = <em>N<sub>e</sub></em><sub>i</sub>/<em>N<sub>e</sub></em>, where <em>N<sub>e</sub></em><sub>i</sub> represents the population size after the <em>i</em>-th change in population size and <em>N<sub>e</sub></em> is the effective population size prior to any change. Furthermore, <em>τ</em><sub>i</sub> = T/(2<em>N<sub>e</sub></em><sub>i</sub>), where T is the number of generations after the <em>i</em>-th change in population size. In the case of a one-step population increase, <em>i</em> = 1. In the case of constant population size, <em>g</em> = 1 and <em>τ</em><sub>i</sub> approaches ∞, and thus the model reduces to a simpler form, denoted as L<sub>0.</sub> Additionally, one can reduce the model by fixing parameters such as setting <em>γ</em> = 0, in the case of no selection. The log-likelihood of the data under any of the models can then be compared using the χ<sup>2</sup> test, as in <a id="xref-ref-112-2" class="xref-bibr" href="#ref-112">Zeng and Charlesworth (2009)</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Haddrill <em>et al.</em> (2011)</a>, and <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Campos <em>et al.</em> (2013)</a>. The parameter space was searched using the simplex algorithm of Press <em>et al.</em> (1992). To initialize the algorithms, multiple random start points were used and were iterated until they converged. The ML estimates are the result of 10000 independent runs of the program.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Genomic sequence data</h3><p id="p-25">The mean coverage of the Illumina HiSeq2000 reads for the 47 <em>D. pseudoobscura</em> strains on the third chromosome varied from a low of 31× to a high of 57×. Sites on the third chromosome homozygous for the alternative base relative to the reference sequence (AR strain MV2-25) occur at frequency of 0.0066 in AR, 0.0093 in ST, 0.0138 in PP, 0.0136 in CH, and 0.0147 in TL. Across the third chromosome, we identified 1,116,491 SNPs. The numbers of SNPs identified within each arrangement are as follows: 508,720 (AR), 327,079 (ST), 391,194 (PP), 464,651 (TL), and 396,485 (CH). An overestimation of rare variants may be produced by call-based genotype inference; however, the mean coverage in our data set is greater than the greatest mean coverage that produced negligible bias in a recent simulation study (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Han <em>et al.</em> 2013</a>). Consistent with previous studies analyzing the SFS in <em>D. pseudoobscura</em> (<a id="xref-ref-97-4" class="xref-bibr" href="#ref-97">Schaeffer <em>et al.</em> 2003</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Machado <em>et al.</em> 2007</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Haddrill <em>et al.</em> 2011</a>), we observe a slightly negative Tajima’s <em>D</em> across the chromosome (<a href="http://www.g3journal.org/content/suppl/2014/10/16/g3.114.014860.DC1/TableS5.pdf">Table S5</a>), indicating a skew toward low-frequency variants.</p></div><div id="sec-11" class="subsection"><h3>Recombination between chromosome arrangements is suppressed</h3><p id="p-26">From the sets of inferred SNPs, we used the software package LDhelmet (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Chan <em>et al.</em> 2012</a>) to construct fine-scale maps of the population scaled recombination rate, <em>ρ</em> = <em>2N<sub>e</sub>r</em>, for all individuals (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) and separately for each of the five gene arrangements (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). <em>r</em> is the recombination rate between base pairs per generation and <em>N<sub>e</sub></em> is the effective population size. Because recombination does not occur in <em>Drosophila</em> males, both <em>r</em> and <em>N<sub>e</sub></em> are multiplied by 2 instead of 4.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The population scaled recombination rate (&#x3C1;) of the third chromosome estimated across five gene arrangements of D. pseudoobscura. The line is fitted using a cubic smoothing spline with a smoothing parameter of 0.005 and is the mean estimate of &#x3C1; after 3,000,000 iterations of LDhelmet. The order of sites is according to the Arrowhead reference strain and vertical dashed lines represent the location of inversion breakpoints." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1380927261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The population scaled recombination rate (&lt;em&gt;&#x3C1;&lt;/em&gt;) of the third chromosome estimated across five gene arrangements of &lt;em&gt;D. pseudoobscura&lt;/em&gt;. The line is fitted using a cubic smoothing spline with a smoothing parameter of 0.005 and is the mean estimate of &lt;em&gt;&#x3C1;&lt;/em&gt; after 3,000,000 iterations of LDhelmet. The order of sites is according to the Arrowhead reference strain and vertical dashed lines represent the location of inversion breakpoints.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2433/F2.medium.gif" width="440" height="170"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/12/2433/F2.medium.gif" width="440" height="170"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465396" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">The population scaled recombination rate (<em>ρ</em>) of the third chromosome estimated across five gene arrangements of <em>D. pseudoobscura</em>. The line is fitted using a cubic smoothing spline with a smoothing parameter of 0.005 and is the mean estimate of <em>ρ</em> after 3,000,000 iterations of LDhelmet. The order of sites is according to the Arrowhead reference strain and vertical dashed lines represent the location of inversion breakpoints.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The population scaled recombination rate (&#x3C1;) for each individual third chromosome arrangement. The line is fitted using a cubic smoothing spline and is the mean estimate of &#x3C1; after 3,000,000 iterations of LDhelmet. The order of sites is according to the gene order of each arrangement. Vertical dashed lines represent the location of breakpoints of the inversion leading to the arrangement. The scale of the y-axis is the same for all arrangements. AR, Arrowhead; ST, Standard; PP, Pikes Peak; CH, Chiriachua; and TL, Tree Line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1380927261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The population scaled recombination rate (&lt;em&gt;&#x3C1;&lt;/em&gt;) for each individual third chromosome arrangement. The line is fitted using a cubic smoothing spline and is the mean estimate of &lt;em&gt;&#x3C1;&lt;/em&gt; after 3,000,000 iterations of LDhelmet. The order of sites is according to the gene order of each arrangement. Vertical dashed lines represent the location of breakpoints of the inversion leading to the arrangement. The scale of the y-axis is the same for all arrangements. AR, Arrowhead; ST, Standard; PP, Pikes Peak; CH, Chiriachua; and TL, Tree Line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2433/F3.medium.gif" width="266" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/12/2433/F3.medium.gif" width="266" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465393" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">The population scaled recombination rate (<em>ρ</em>) for each individual third chromosome arrangement. The line is fitted using a cubic smoothing spline and is the mean estimate of <em>ρ</em> after 3,000,000 iterations of LDhelmet. The order of sites is according to the gene order of each arrangement. Vertical dashed lines represent the location of breakpoints of the inversion leading to the arrangement. The scale of the y-axis is the same for all arrangements. AR, Arrowhead; ST, Standard; PP, Pikes Peak; CH, Chiriachua; and TL, Tree Line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Between all gene arrangements, <em>ρ</em> across the entire chromosome is estimated to be 0.052/bp (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), which is comparable with whole-chromosome estimates of <em>ρ</em> reported in an African <em>D. melanogaster</em> population (Rwanda), but greater than estimates in a North American <em>D. melanogaster</em> population (Raleigh) obtained using the same method (<a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Chan <em>et al.</em> 2012</a>). Considerable heterogeneity in the recombination rate is observed across the chromosome, with <em>ρ</em> values up to fivefold greater than the chromosomal average detected in several localized hotspots and greater than 100-fold reductions in <em>ρ</em> observed in other regions. Between inversion breakpoints, the average rate of recombination is significantly suppressed compared to the chromosome average (<em>P</em> &lt; 0.05, one-sided Wilcoxon rank-sum test, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).On the other hand, the average recombination rate is elevated in the centromeric and telomeric chromosome regions flanking the inversion breakpoints, with <em>ρ</em> values of 0.058/bp and 0.056/bp respectively.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465296/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465296/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Estimates of the average population scaled recombination rate (<em>ρ</em>) for different regions of third chromosome arrangements</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">In the fine-scale recombination mapping performed separately for each gene arrangement (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), no significant reduction in the rate of recombination is detected between inversion breakpoints. In fact, the average value of <em>ρ</em> is greater within inverted regions compared with regions located outside the breakpoints (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). The greatest chromosome-wide average recombination rate is detected in the AR arrangement (<em>ρ</em><em><sub>Avg</sub>=</em>0.061/bp), whereas the lowest is observed in the CH arrangement (<em>ρ</em><em><sub>Avg</sub></em> = 0.018/bp). The greatest increase of <em>ρ</em><em><sub>Avg</sub></em> from noninverted regions to regions within inversion breakpoints is observed within the PP arrangement, 0.015/bp to 0.030/bp.</p></div><div id="sec-12" class="subsection"><h3>Identification of major and minor codons</h3><p id="p-35">Major codons were identified by estimating the relative frequency of each synonymous codon encoding a particular amino acid across our set of third chromosome genes. Preferential use of a major codon was observed for all amino acids (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The relative frequencies of major and minor codons are similar to that observed by <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Akashi and Schaeffer (1997)</a>, where only 22 genes in <em>D. pseudoobscura</em> were analyzed (<em>P≈</em>1<em>,r=</em>.992). A strong GC bias is observed in the third nucleotide position. All C-ending codons, and all but three G-ending codons are identified as major, a finding identical to a recent study using the second chromosome of <em>D. pseudoobscura</em> (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Kliman 2014</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465392/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465392/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Major codons identified on the third chromosome of <em>D</em>. <em>pseudoobscura</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Estimates of codon bias</h3><p id="p-37">Several methods exist to estimate the level of codon bias within a gene. These include nondirectional measures such as the “Effective Number of Codons” (<em>ENC</em>; <a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">Wright 1990</a>) and directional measures such as the “Frequency of Optimal Codons” (<em>Fop</em>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Ikemura 1985</a>) and the “Codon Adaptation Index” (<em>CAI</em>; <a id="xref-ref-100-2" class="xref-bibr" href="#ref-100">Sharp and Li 1987</a>). For each gene and for each arrangement, we estimated the average <em>ENC</em>, <em>Fop</em>, and <em>CAI</em>.</p><p id="p-38">First, we sought to determine whether amino acid composition in genes of the third chromosome contributed to the level of codon usage bias. We adopted a sequence randomization method to randomize the possible synonymous codons at each amino acid site in our set of 2798 genes. After generating 1000 replicates for each gene, we then evaluated <em>ENC</em> for both the randomized and observed data. Here, we used <em>ENC</em> as a measure of codon bias, because it is nondirectional and the effects of amino acid composition could bias codon usage toward both AT and GC ending codons. We considered amino acid composition as strongly contributing to the level of codon bias in a gene if a significant number (&gt;0.05) of randomized sequences were as or more biased than the observed level. For 153 genes, there was a significant <em>(P &lt;</em> 0.05) effect of amino acid composition on the observed level of codon bias and these were removed from subsequent analyses.</p><p id="p-39"><em>ENC</em>, <em>Fop</em>, and <em>CAI</em> estimates are highly correlated (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>), and no significant differences are detected in the chromosome-wide mean estimates of all three measures between arrangements (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Furthermore, a high amount of correlation exists between average <em>Fop</em> values for each gene between arrangements (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). We also performed pairwise Kolmogorov-Smirnov tests between each arrangement and did not detect any significant differences in the distribution of average <em>Fop</em> values for any comparison. The pattern and extent of codon usage bias appears to be similar between the five third chromosome arrangements.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Measures of codon bias on the third chromosome. (A) The correlation and relationship between three measures of codon usage bias (CAI, Codon Adaptation Index; ENC, Effective Number of Codons; Fop, Frequency of Preferred Codon) across all individuals. The matrix in the top left shows Spearman&#x2019;s correlation coefficient (rs) between the three measures. (B) The correlation between average Fop values for each gene between arrangements. Off-diagonal plots show pairwise comparisons and plots on the diagonal depict the density distribution of average Fop values for genes within each arrangement." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1380927261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Measures of codon bias on the third chromosome. (A) The correlation and relationship between three measures of codon usage bias (&lt;em&gt;CAI&lt;/em&gt;, Codon Adaptation Index; &lt;em&gt;ENC&lt;/em&gt;, Effective Number of Codons; &lt;em&gt;Fop&lt;/em&gt;, Frequency of Preferred Codon) across all individuals. The matrix in the top left shows Spearman&#x2019;s correlation coefficient (&lt;em&gt;r&lt;sub&gt;s&lt;/sub&gt;&lt;/em&gt;) between the three measures. (B) The correlation between average &lt;em&gt;Fop&lt;/em&gt; values for each gene between arrangements. Off-diagonal plots show pairwise comparisons and plots on the diagonal depict the density distribution of average &lt;em&gt;Fop&lt;/em&gt; values for genes within each arrangement.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2433/F4.medium.gif" width="440" height="194"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/12/2433/F4.medium.gif" width="440" height="194"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465372" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-40" class="first-child">Measures of codon bias on the third chromosome. (A) The correlation and relationship between three measures of codon usage bias (<em>CAI</em>, Codon Adaptation Index; <em>ENC</em>, Effective Number of Codons; <em>Fop</em>, Frequency of Preferred Codon) across all individuals. The matrix in the top left shows Spearman’s correlation coefficient (<em>r<sub>s</sub></em>) between the three measures. (B) The correlation between average <em>Fop</em> values for each gene between arrangements. Off-diagonal plots show pairwise comparisons and plots on the diagonal depict the density distribution of average <em>Fop</em> values for genes within each arrangement.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465327/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465327/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Estimates of three different measures of codon bias</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>The pattern of synonymous polymorphism on the third chromosome</h3><p id="p-42">A balanced inversion system is predicted to generate a reduction in the overall level of diversity as a short term effect (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Andolfatto <em>et al.</em> 2001</a>). As the age of the inversion increases, total nucleotide variability is expected to be similar to that of a panmictic neutral population over much of the inversion due to the effects of mutation, gene conversion, recombination between individuals homozygous for the inversion, and rare double-crossover events between inversion heterokaryotypes (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Navarro <em>et al.</em> 1997</a>, <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">2000</a>). Under this model, even with low rates of genetic flux, differentiation is expected to be high near breakpoints, yet arrangements should be nearly indistinguishable toward the center of the inverted region. Inversions have been noted to drastically decrease nucleotide variation near breakpoints (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Corbett-Detig and Hartl 2012</a>). In recent studies of <em>D. melanogaster</em>, however, the level of polymorphism is found to correlate with the number of inversions present and nucleotide diversity is significantly increased over large stretches of different chromosome arms across inversion breakpoints (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Corbett-Detig and Hartl 2012</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Pool <em>et al.</em> 2012</a>). Furthermore, <a id="xref-ref-108-2" class="xref-bibr" href="#ref-108">Wallace <em>et al.</em> (2013)</a> observed that both genetic differentiation and nucleotide diversity were significantly increased for a subset of genes within inverted regions of the <em>D. pseudoobscura</em> third chromosome. Here, we estimated the nucleotide diversity at synonymous sites (<em>π</em><sub>S</sub>) in each gene for each third chromosome arrangement, as well as for all arrangements together (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Nucleotide diversity (&#x3C0;s, Nei 1987) estimated at synonymous sites for the total dataset and each arrangement. The colored line for each arrangement depicts the LOWESS smoothed line between mean estimates of &#x3C0;s for each gene on the third chromosome. The order of genes is according to the AR reference strain and vertical dashed lines represent the location of inversion breakpoints. AR, Arrowhead; ST, Standard; PP, Pikes Peak; CH, Chiriachua; and TL, Tree Line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1380927261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Nucleotide diversity (&#x3C0;&lt;sub&gt;s&lt;/sub&gt;, Nei 1987) estimated at synonymous sites for the total dataset and each arrangement. The colored line for each arrangement depicts the LOWESS smoothed line between mean estimates of &#x3C0;&lt;sub&gt;s&lt;/sub&gt; for each gene on the third chromosome. The order of genes is according to the AR reference strain and vertical dashed lines represent the location of inversion breakpoints. AR, Arrowhead; ST, Standard; PP, Pikes Peak; CH, Chiriachua; and TL, Tree Line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2433/F5.medium.gif" width="440" height="164"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/12/2433/F5.medium.gif" width="440" height="164"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465465" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-43" class="first-child">Nucleotide diversity (π<sub>s</sub>, <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Nei 1987</a>) estimated at synonymous sites for the total dataset and each arrangement. The colored line for each arrangement depicts the LOWESS smoothed line between mean estimates of π<sub>s</sub> for each gene on the third chromosome. The order of genes is according to the AR reference strain and vertical dashed lines represent the location of inversion breakpoints. AR, Arrowhead; ST, Standard; PP, Pikes Peak; CH, Chiriachua; and TL, Tree Line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">Within the inverted chromosome regions, <em>π</em><sub>S</sub> for all individuals (Total; TO) is significantly greater than <em>π</em><sub>S</sub> of each of the five individual gene arrangements (Wilcoxon rank-sum test: <em>W</em> = 162097, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>). For genes located within the inverted regions, the average <em>π</em><sub>S</sub> value is 0.029 for all individuals, whereas the average <em>π</em><sub>S</sub> value for each arrangement is as follows: 0.014 (AR), 0.012 (ST), 0.012 (PP), 0.016 (TL), and 0.016 (CH). <a id="xref-ref-108-3" class="xref-bibr" href="#ref-108">Wallace <em>et al.</em> (2013)</a> similarly reported comparable levels of diversity between different gene arrangements along the medial region of the third <em>D. pseudoobscura</em> chromosome but significantly elevated diversity when all arrangements are analyzed together. Outside the inverted chromosome regions, <em>π</em><sub>S</sub> estimates from the total dataset and for each individual gene arrangement are similar (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Between each gene arrangement, the 95% confidence intervals of the mean estimate of <em>π</em><sub>S</sub> overlap in both the proximal and distal regions outside of the medial inverted regions for each case (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). However, <em>π</em><sub>S</sub> is significantly different between particular arrangements in the medial region of the chromosome, suggesting that the effective population size differs among gene arrangements within inverted segments. For example, <em>π</em><sub>S</sub> of ST (0.012) is significantly lower than <em>π</em><sub>S</sub> of AR (0.014) in the medial inverted overlap region.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465341/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465341/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Estimates of recombination (<em>ρ</em>) and synonymous site diversity (<em>π</em><sub>S</sub>) for different third chromosome regions</span><div class="sb-div caption-clear"></div></div></div><p id="p-46">To gain insight into the relationship between synonymous polymorphism and recombination separately from <em>N<sub>e</sub></em>, we also estimated the mean ratio of <em>ρ</em>/<em>π</em><sub>S</sub> for the proximal, inverted, and distal regions of the chromosome for each arrangement and for all individuals together (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). For the total dataset, this ratio is significantly elevated in the proximal and distal regions outside of the overlapping inversions (<em>ρ</em>/<em>π</em><sub>S</sub> = 7.232 and 6.175 respectively) compared with the inverted region (<em>ρ</em>/<em>π</em><sub>S</sub> = 1.893). On the other hand, this ratio is either increased or not significantly reduced in inverted segments within each third chromosome arrangement.</p></div><div id="sec-15" class="subsection"><h3>Mutational bias and variability in rates of synonymous and nonsynonymous polymorphism</h3><p id="p-47">Following <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Akashi and Schaeffer (1997)</a>, we performed a <em>G</em>-test for heterogeneity of synonymous codons within each amino acid group (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Although major codons exist for every amino acid, the relative frequencies of major and minor codons vary widely between amino acids. For instance, the major codon for glutamic acid (GAG) is observed at a frequency of 73.1% (<em>G</em> = 22.1), whereas the major codon for aspartic acid (GAC) is only observed at a frequency of 53.8% (<em>G</em> = 0.56). We therefore investigated whether the rates of synonymous and nonsynonymous polymorphisms vary between amino acids as well.</p><p id="p-48">Synonymous and nonsynonymous sites were counted using the approach of <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Nei and Gojobori (1986)</a>, which takes into account the degeneracy of each amino acid. We estimated the rate of synonymous polymorphism per synonymous site and the rate of nonsynonymous polymorphism per nonsynonymous site using the <a id="xref-ref-110-2" class="xref-bibr" href="#ref-110">Watterson (1975)</a> estimator of <em>θ</em> separately for each amino acid. Nonsynonymous polymorphism rates vary between amino acids, with non-polar alanine estimated to have the greatest (<em>θ</em><sub>NSyn</sub> = 0.006) and aromatic phenylalanine estimated to have the lowest (<em>θ</em><sub>NSyn</sub> = 0.001). Interestingly, every twofold degenerate amino acid has an increased rate of synonymous polymorphism compared with the three-, four-, and sixfold degenerate amino acids (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). We hypothesized that this effect results from an unequal transition to transversion mutation ratio. If transversions occur less frequently than transitions, the average estimates of <em>θ</em><sub>Syn</sub> will be lower at sites where synonymous transversions are possible, such as third codon positions of fourfold degenerate amino acids, compared with sites were only synonymous transitions are possible, such as third codon positions of twofold degenerate amino acids. Indeed, if only third codon position transitional mutations are considered, this general trend disappears (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>), although the twofold degenerate amino acids phenylalanine, cysteine, and glutamic acid still have the greatest three rates of average polymorphism. After inferring ancestral states from <em>D. miranda</em>, it is also apparent from analyzing only transitional mutations that there is a greater rate of G/C→A/T mutations than A/T→G/C mutations (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). Across all amino acids on the chromosome, the average rate of third codon position G/C→A/T mutations is nearly four times greater than A/T→G/C mutations (<em>θ</em><sub>G/C→A/T</sub> / <em>θ</em><sub>A/T→G/C</sub> = 3.949). This bias of mutations toward A/T has been observed in a variety of other organisms, including spontaneous mutation accumulation <em>D. melanogaster</em> lines and several divergent taxa of eubacteria (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Keightley <em>et al.</em> 2009</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hershberg and Petrov 2010</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Schrider <em>et al.</em> 2013</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Synonymous and nonsynonymous polymorphism rates for amino acids. (A) The rate of synonymous and nonsynonymous polymorphism (&#x3B8;, Watterson 1975) for each amino acid. The degeneracy class is shown below each amino acid. Stop codons and the onefold degenerate amino acids (Met and Trp) were not considered. (B) The average rate of different transitional mutations for each amino acid. A greater rate of mutation toward AT is observed for all amino acids." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1380927261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Synonymous and nonsynonymous polymorphism rates for amino acids. (A) The rate of synonymous and nonsynonymous polymorphism (&lt;em&gt;&#x3B8;&lt;/em&gt;, Watterson 1975) for each amino acid. The degeneracy class is shown below each amino acid. Stop codons and the onefold degenerate amino acids (Met and Trp) were not considered. (B) The average rate of different transitional mutations for each amino acid. A greater rate of mutation toward AT is observed for all amino acids.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2433/F6.medium.gif" width="440" height="178"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/12/2433/F6.medium.gif" width="440" height="178"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465326" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-49" class="first-child">Synonymous and nonsynonymous polymorphism rates for amino acids. (A) The rate of synonymous and nonsynonymous polymorphism (<em>θ</em>, <a id="xref-ref-110-3" class="xref-bibr" href="#ref-110">Watterson 1975</a>) for each amino acid. The degeneracy class is shown below each amino acid. Stop codons and the onefold degenerate amino acids (Met and Trp) were not considered. (B) The average rate of different transitional mutations for each amino acid. A greater rate of mutation toward AT is observed for all amino acids.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">Mutational bias, especially in the presence of very weak selection, has been demonstrated to have a major effect on codon usage across the genomes of several species (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Palidwor <em>et al.</em> 2010</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Rao <em>et al.</em> 2011</a>). Consistent with several previous studies (<a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Vicario <em>et al.</em> 2007</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Haddrill <em>et al.</em> 2011</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Campos <em>et al.</em> 2013</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Kliman 2014</a>), there appears to be a chromosome-wide mutational bias toward A/T, inferred from the nucleotide composition in noncoding regions (%GC = 43.87) where selective constraints are presumably reduced. This is in contrast to the GC-biased nucleotide composition of coding regions (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>) and the observation of all major codons ending in G or C. We estimate that the proportion of third codon positions being a G or C (GC<sub>3</sub>) is 0.654, similar to the estimates of <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Campos <em>et al.</em> (2013)</a> using several autosomal <em>D. melanogaster</em> genes.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465351/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465351/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Nucleotide composition in coding and noncoding regions</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Codon usage at evolutionary conserved sites</h3><p id="p-51">If codon bias results from an effort to increase translational accuracy, the use of major codons is hypothesized to be favored at important and evolutionary conserved amino acid sites (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Akashi 1994</a>). For instance, <em>Escherichia coli</em> cells starved of asparagine tend to misincorporate lysine when minor asparagine codons are used instead of major codons (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Precup and Parker 1987</a>). On the other hand, minor codons may be tolerated at sites in the translated mRNA where errors are less likely to affect the function or folding of the protein product, such as at evolutionary unconserved amino acid positions. We examined the strength of association between major codons and conserved amino acid positions to test for the presence of selection for translational accuracy.</p><p id="p-52"><a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Akashi (1994)</a> devised a statistical test to detect selection on the translational accuracy of a protein coding sequence. The test is built around counting the numbers of major and minor codons at evolutionary conserved and variable sites for each amino acid in a given sequence and determines how likely an association is to have occurred by chance. From these counts, an odds ratio test, denoted as ψ, can be computed that gives the magnitude of the association between major codons and evolutionary conserved amino acid positions. Using <em>D. miranda</em> as an outgroup, we defined any site that was monomorphic in all <em>D. pseudoobscura</em> individuals and shared with <em>D. miranda</em> as conserved. We defined any site that was polymorphic in <em>D. pseudoobscura</em> as variable. For each gene, we then computed ψ.</p><p id="p-53">A total of 1887 genes had ψ &gt; 1, meaning there was a greater likelihood of observing an association between a major codon and conserved site than observing an association of a minor codon at a variable site. We find a highly significant linear relationship between ψ and <em>Fop</em> (<em>F</em> = 5325, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>) as well as a strong positive correlation, measured as Spearman’s rank correlation coefficient (0.825). In highly biased genes, major codons are more likely to be used at evolutionary conserved sites, while minor codons are more likely to be used at evolutionary variable sites.</p></div><div id="sec-17" class="subsection"><h3>Estimating the strength of selection at unpreferred and preferred synonymous sites</h3><p id="p-54">Several methods have been developed that take advantage of the SFS to estimate the population-scaled selection coefficient, <em>γ</em> = <em>4Ns</em>, at synonymous sites. (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Akashi and Schaeffer 1997</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Maside <em>et al.</em> 2004</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Cutter and Charlesworth 2006</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Nielsen <em>et al.</em> 2007</a>). Here, we used the method of <a id="xref-ref-112-3" class="xref-bibr" href="#ref-112">Zeng and Charlesworth (2009)</a>, which allows for demographic changes and does not require the polarization of mutations with an outgroup (see the section <em>Materials and Methods</em>). The model assumes the possibility of two alleles, <em>a<sub>o</sub></em> and <em>a<sub>1</sub></em>, at a given nucleotide site and uses the full SFS. If <em>γ</em> is positive, then allele <em>a<sub>0</sub></em> is selected for. We constructed allele-frequency spectra with respects to either the counts of major or minor codons to examine the strength of selection acting on preferred and unpreferred mutations. There is increasing evidence that <em>D. pseudoobscura</em> has undergone a recent population expansion, which may be more pronounced in the AR arrangement due to recent selection (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Schaeffer 2002</a>; <a id="xref-ref-97-5" class="xref-bibr" href="#ref-97">Schaeffer <em>et al.</em> 2003</a>; <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Haddrill <em>et al.</em> 2011</a>; <a id="xref-ref-107-2" class="xref-bibr" href="#ref-107">Wallace <em>et al.</em> 2011</a>). The full model of <a id="xref-ref-112-4" class="xref-bibr" href="#ref-112">Zeng and Charlesworth (2009)</a>, denoted as L<sub>1</sub>, allows for a one-step population expansion whereas a reduced model, L<sub>0</sub>, assumes a constant population size. We first tested the hypothesis of a recent population expansion by comparing the SFS under L<sub>0</sub> and L<sub>1</sub> in a maximum likelihood (ML) analysis, following the approach of <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Campos <em>et al.</em> (2013)</a>. An L<sub>1</sub> model with recent population expansion fits the data significantly better than the reduced L<sub>0</sub> model with constant population size for every third chromosome arrangement using the SFS constructed with regards to major codons (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7A</a>). Similar results were obtained using the SFS constructed with regards to minor codons (<a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7B</a>). Hence, we use a model of recent population expansion for subsequent analyses.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465450/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465450/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Likelihoods and estimates of mutation and demographic parameters from the site frequency spectrum of preferred (A) and unpreferred (B) mutations</span><div class="sb-div caption-clear"></div></div></div><p id="p-56">If natural selection favors translationally superior codons, the strength of selection should be positive for preferred mutations and negative for unpreferred variants. To test for the presence of positive selection for preferred mutations, we estimated the log-likelihood of the data using a reduced model with <em>γ</em> = 0, denoted as NS (<em>No Selection)</em>. We then compared the log-likelihood of this null hypothesis to the log-likelihood of the data under the full L<sub>1</sub> for the SFS constructed with respect to preferred changes. For each arrangement and the total data set, L<sub>1</sub> fits the data significantly better and <em>γ</em> is predicted as &gt;0, indicating the presence of a positive population-scaled selection coefficient for preferred synonymous mutations (<a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7A</a>). Similar results using the SFS constructed with regard to minor codons support γ &lt; 0 for unpreferred changes across all arrangements and the total data set (<a id="xref-table-wrap-7-4" class="xref-table" href="#T7">Table 7B</a>).</p><p id="p-57">For each arrangement, the ML estimate of the mutational bias parameter, <em>κ</em>, under the L<sub>1</sub> model is &lt; 1 for the SFS constructed based on counts of minor codons and &gt;1 for the SFS constructed based on counts of major codons, further supporting the presence of a mutational bias toward unpreferred changes. Furthermore, the absolute values of ML estimates of <em>γ</em> are more than triple for unpreferred mutations for each case. These results suggest either a greater intensity of selection chromosome wide against more frequent unpreferred mutations than for preferred changes or a departure from assumptions of the model. ML estimates of the population expansion parameter <em>g</em> not only vary widely between the SFS of unpreferred and preferred mutations, but also for arrangements within each SFS (2.37 &lt; <em>g<sub>Pref</sub></em> &lt; 6.01; 0.58 &lt; <em>g<sub>Unpref</sub></em> &lt; 24.38). Additionally, ML estimates of the expansion time parameter τ vary between arrangements (0.026 &lt; <em>τ</em><em><sub>Pref</sub></em> &lt; 1.00; 0.003 &lt; <em>τ</em><em><sub>Unpref</sub></em> &lt; 1.00). Together, these results indicate different demographic histories of different third chromosome gene arrangements.</p><p id="p-58">It is argued that the optimum level of gene expression, and therefore the need for increased translational accuracy and/or efficiency, may vary widely from gene to gene (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Hershberg and Petrov 2008</a>). Hence, the strength of selection acting on preferred and unpreferred sites may vary from gene to gene as well. To test this hypothesis, we estimated <em>γ</em> independently for genes in different classes of codon bias. We grouped genes into four different equally sized bins of codon bias by dividing the distribution of <em>Fop</em> scores into quartiles for each arrangement (see <a href="http://www.g3journal.org/content/suppl/2014/10/16/g3.114.014860.DC1/TableS6.pdf">Table S6</a> for cutoff values). When all arrangements are analyzed together, there is a significant linear relationship between the class of codon bias and <em>γ</em>, with stronger selection acting on more highly biased genes <em>(F</em> = 257.2, <em>P</em> = 0.00386<em>)</em>. Estimates of <em>γ</em> between <em>Fop</em> bins are not statistically different within individual arrangements as the 95% confidence intervals overlap in every case, but still exhibit the same trend observed in the total data set, where the average <em>γ</em> value decreases with the strength of codon bias (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). The magnitude of <em>γ</em> is greater against unpreferred changes than for preferred changes in all classes of codon bias. Furthermore, the estimates of <em>γ</em> for unpreferred changes are not statistically different between classes of codon bias among any of the datasets.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The intensity of selection acting on preferred and unpreferred changes. (A) The population scaled selection coefficient (&#x3B3;) for preferred and unpreferred mutations estimated in genes that vary in levels of codon bias across arrangements. Each bin represents a quartile of the distribution of Fop values within each arrangement and is ordered in decreasing levels of codon bias. For example, Bin 1 contains genes with the greatest levels of codon bias and Fop values in the top 25% (see Table S6 for the cutoff values for each arrangement). Black bars represent the 95% confidence intervals of the mean ML estimate after 10,000 iterations. (B) The population scaled selection coefficient (&#x3B3;) for preferred and unpreferred mutations estimated in genes with different levels of recombination across arrangements. Each bin represents a quartile of the distribution of average values of &#x3C1; for genes within each arrangement and is ordered in increasing levels of recombination. For example, Bin 1 contains genes with the lowest level of recombination and average &#x3C1; values in the lowest 25% (see Table S7 for the cutoff values for each arrangement). Black bars represent the 95% confidence intervals of the mean ML estimate after 10,000 iterations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1380927261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The intensity of selection acting on preferred and unpreferred changes. (A) The population scaled selection coefficient (&lt;em&gt;&#x3B3;&lt;/em&gt;) for preferred and unpreferred mutations estimated in genes that vary in levels of codon bias across arrangements. Each bin represents a quartile of the distribution of &lt;em&gt;Fop&lt;/em&gt; values within each arrangement and is ordered in decreasing levels of codon bias. For example, Bin 1 contains genes with the greatest levels of codon bias and &lt;em&gt;Fop&lt;/em&gt; values in the top 25% (see Table S6 for the cutoff values for each arrangement). Black bars represent the 95% confidence intervals of the mean ML estimate after 10,000 iterations. (B) The population scaled selection coefficient (&lt;em&gt;&#x3B3;&lt;/em&gt;) for preferred and unpreferred mutations estimated in genes with different levels of recombination across arrangements. Each bin represents a quartile of the distribution of average values of &lt;em&gt;&#x3C1;&lt;/em&gt; for genes within each arrangement and is ordered in increasing levels of recombination. For example, Bin 1 contains genes with the lowest level of recombination and average &lt;em&gt;&#x3C1;&lt;/em&gt; values in the lowest 25% (see Table S7 for the cutoff values for each arrangement). Black bars represent the 95% confidence intervals of the mean ML estimate after 10,000 iterations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2433/F7.medium.gif" width="440" height="136"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/12/2433/F7.medium.gif" width="440" height="136"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465464" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-59" class="first-child">The intensity of selection acting on preferred and unpreferred changes. (A) The population scaled selection coefficient (<em>γ</em>) for preferred and unpreferred mutations estimated in genes that vary in levels of codon bias across arrangements. Each bin represents a quartile of the distribution of <em>Fop</em> values within each arrangement and is ordered in decreasing levels of codon bias. For example, Bin 1 contains genes with the greatest levels of codon bias and <em>Fop</em> values in the top 25% (see <a href="http://www.g3journal.org/content/suppl/2014/10/16/g3.114.014860.DC1/TableS6.pdf">Table S6</a> for the cutoff values for each arrangement). Black bars represent the 95% confidence intervals of the mean ML estimate after 10,000 iterations. (B) The population scaled selection coefficient (<em>γ</em>) for preferred and unpreferred mutations estimated in genes with different levels of recombination across arrangements. Each bin represents a quartile of the distribution of average values of <em>ρ</em> for genes within each arrangement and is ordered in increasing levels of recombination. For example, Bin 1 contains genes with the lowest level of recombination and average <em>ρ</em> values in the lowest 25% (see <a href="http://www.g3journal.org/content/suppl/2014/10/16/g3.114.014860.DC1/TableS7.pdf">Table S7</a> for the cutoff values for each arrangement). Black bars represent the 95% confidence intervals of the mean ML estimate after 10,000 iterations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>The relationship between selection, recombination, and codon bias</h3><p id="p-60">Using our fine-scale maps, we took the average recombination rate between the beginning and end coordinates of each gene to estimate a mean rate for genes in each independent arrangement and in the total sample. We then grouped genes, independently for each arrangement, into four equally sized bins of recombination rates by dividing the distribution of mean <em>ρ</em> values into quartiles (see <a href="http://www.g3journal.org/content/suppl/2014/10/16/g3.114.014860.DC1/TableS7.pdf">Table S7</a> for cutoff values). Again under the full L<sub>1</sub> model, we then estimated <em>γ</em> for the SFS constructed with respects to major or minor codons in genes within each class of recombination for each arrangement as well as for the total sample.</p><p id="p-61">Similar to the results obtained from grouping genes by <em>Fop</em>, the magnitude of selection is greater against unpreferred changes than for preferred changes in each bin (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). Unlike the previous result, however, <em>γ</em> does not significantly change between different classes of recombination. Instead, selection both for preferred changes and against unpreferred changes appears fairly consistent between bins, as the 95% confidence intervals of <em>γ</em> overlap across all recombination rate bins for each arrangement. Within individual arrangements, there is not a significant association between the strength of selection either for preferred changes or against unpreferred changes and the rate of recombination. In addition, across the total sample where recombination is genuinely suppressed between arrangements, the strength of selection is not significantly impacted across different rates of recombination.</p><p id="p-62">Finally, we analyzed the relationship between the rate of recombination and the magnitude of codon usage bias along the third chromosome. Across all individuals, there is a weak (<em>R<sup>2</sup> =</em> 0.012), yet statistically significant (<em>F</em> = 36.61, <em>P</em> &lt; 1.63 × 10<sup>−8</sup>) positive linear relationship between <em>Fop</em> and the average recombination rate for each gene. Spearman’s rank correlation coefficient (<em>r<sub>s</sub></em>) is 0.097. However, this is likely a result of the small number of genes located in regions of relatively high recombination (<em>ρ</em> &gt; 0.2 for 104 genes) and the non-normal distribution of average <em>ρ</em> values which violate the assumptions of a simple linear relationship. Instead, if a power transformation (λ = 0.30, determined from a Box-Cox procedure; Supplementary Material) is used on average values of <em>ρ</em>, the significance of this relationship disappears <em>(P &lt;</em> 0.362). Furthermore, if genes are grouped by percentiles of <em>ρ</em> spaced in 5% increments (see <a href="http://www.g3journal.org/content/suppl/2014/10/16/g3.114.014860.DC1/TableS8.pdf">Table S8</a> for cutoff values), no significant differences are detected between mean values of <em>Fop</em> (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>), although a slight correlation does exist (<em>r<sub>s</sub></em> = 0.475). In agreement with the recent findings of <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Kliman (2014)</a>, using sites on the second chromosome of <em>D. pseudoobscura</em>, we conclude that there is not a significant association between the rate of recombination and either the strength of selection acting on preferred/unpreferred sites or the magnitude of codon bias on the third chromosome.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Codon bias (measured as Fop) estimated in genes grouped by percentiles of recombination rate (&#x3C1;) spaced in 5% intervals. The violin plots depict the distribution of Fop values within each class of &#x3C1;. Each class of 5% intervals of &#x3C1; values is ordered in increasing levels of recombination for the total dataset. For instance, Class 20 contains genes with the greatest level of recombination and average &#x3C1; values in the greatest 5% (see Table S8 for the cutoff values of each class). The black bars within each violin plot represent the 95% confidence intervals of the mean Fop value within each class of &#x3C1;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1380927261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Codon bias (measured as &lt;em&gt;Fop&lt;/em&gt;) estimated in genes grouped by percentiles of recombination rate (&lt;em&gt;&#x3C1;&lt;/em&gt;) spaced in 5% intervals. The violin plots depict the distribution of &lt;em&gt;Fop&lt;/em&gt; values within each class of &lt;em&gt;&#x3C1;&lt;/em&gt;. Each class of 5% intervals of &lt;em&gt;&#x3C1;&lt;/em&gt; values is ordered in increasing levels of recombination for the total dataset. For instance, Class 20 contains genes with the greatest level of recombination and average &lt;em&gt;&#x3C1;&lt;/em&gt; values in the greatest 5% (see Table S8 for the cutoff values of each class). The black bars within each violin plot represent the 95% confidence intervals of the mean &lt;em&gt;Fop&lt;/em&gt; value within each class of &lt;em&gt;&#x3C1;&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2433/F8.medium.gif" width="440" height="208"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/4/12/2433/F8.medium.gif" width="440" height="208"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2433/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465379" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-63" class="first-child">Codon bias (measured as <em>Fop</em>) estimated in genes grouped by percentiles of recombination rate (<em>ρ</em>) spaced in 5% intervals. The violin plots depict the distribution of <em>Fop</em> values within each class of <em>ρ</em>. Each class of 5% intervals of <em>ρ</em> values is ordered in increasing levels of recombination for the total dataset. For instance, Class 20 contains genes with the greatest level of recombination and average <em>ρ</em> values in the greatest 5% (see <a href="http://www.g3journal.org/content/suppl/2014/10/16/g3.114.014860.DC1/TableS8.pdf">Table S8</a> for the cutoff values of each class). The black bars within each violin plot represent the 95% confidence intervals of the mean <em>Fop</em> value within each class of <em>ρ</em>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><div id="sec-20" class="subsection"><h3>Recombination is significantly reduced between third chromosome gene arrangements</h3><p id="p-64">Here, as in other recent studies involving <em>D. pseudoobscura</em> and <em>Drosophila</em> species, we demonstrate that the fine-scale rate of recombination is heterogeneous (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Cirulli <em>et al.</em> 2007</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Kulathinal <em>et al.</em> 2009</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">McGaugh <em>et al.</em> 2012</a>). Moreover, we show that the landscape of recombination is drastically modified by chromosomal inversions and that it is significantly suppressed in regions where inversions overlap between heterokaryotypes. Although inversions do not appear to impact local recombination rates within individual gene arrangements, considerable heterogeneity is observed in the average recombination rates when comparing different arrangements.</p><p id="p-65">The chromosome-wide average of <em>ρ</em> within AR (<em>ρ<sub>Avg</sub></em> = 0.061/bp) is nearly double that of the arrangement with the next greatest chromosome-wide average (TL, <em>ρ<sub>Avg</sub></em> = 0.038/bp) and over triple compared with the arrangement with the lowest (CH, <em>ρ<sub>Avg</sub></em> = 0.018/bp). This elevated rate of recombination in AR is likely because of the AR arrangement’s demographic history and frequency distribution. Of all the arrangements considered in this study, AR and TL have the largest distribution across the species range as well as the greatest frequency in nature, and thus have the greatest opportunity to form homokaryotypes and freely recombine across the length of the chromosome. AR is nearly fixed in Niche 4 located in the four corners region of the southwestern United States (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>) where homozygotes would be expected to be found at appreciable frequencies. Furthermore, AR is estimated to be the youngest arrangement, diverging from ST 0.58 million years ago (Mya), but yet despite its age has reached high frequency in a relatively short time (95% confidence interval 0.51−0.65 Mya; <a id="xref-ref-107-3" class="xref-bibr" href="#ref-107">Wallace <em>et al.</em> 2011</a>). TL is capable of forming homozygotes at appreciable frequencies in Mexican populations, but not as high as AR in the United States.</p><p id="p-66">Although recombination is significantly suppressed in inverted regions of the chromosome where inversions overlap, several hotspots exist outside of the inversion breakpoints near the telomere and centromere that contain peaks of <em>ρ</em> that are greater than any observation within inverted segments. Additionally, for the total dataset, the average estimates of <em>ρ</em> in noninverted regions are significantly greater than the overlapping inverted regions <em>(P &lt;</em> 0.05, Wilcoxon rank-sum test). This finding suggests that genetic exchange and recombination still occurs freely outside of inversions between arrangements. This result is surprising, given that recombination is expected to increase near the telomere yet decrease near the centromere in the presence of an inversion (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Navarro <em>et al.</em> 1997</a>). Recombination also has been observed to be elevated in regions outside of overlapping inversions in female heterokaryotypes relative to homokaryotypes in several investigations of cross-over rates in <em>D. melanogaster</em>, suggesting the interchromosomal effects of inversions may be another possible mechanism increasing recombination outside of breakpoints (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Schultz and Redfield 1951</a>; <a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Singleton 1964</a>).</p><p id="p-67">It should be noted that in computer simulations of human sequences, a statistical method similar to LDhelmet, LDhat (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">McVean <em>et al.</em> 2004</a>), has been demonstrated to produce a slight downward bias in the recombination rate estimates when inversions are present (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Basatena <em>et al.</em> 2013</a>). However, these simulations were performed using just one gene arrangement and detectable differences between the predicted and actual recombination rate only occurred when the inverted chromosome was present at high frequencies (&gt;80%). Furthermore, recombination was modeled according to human genetic variation and inversions were assumed to rise in frequency due to neutral drift, both of which are unlikely scenarios in <em>D. pseudoobscura</em>. Because of the large number of third chromosome gene arrangements in <em>D. pseudoobscura</em>, inversions not included in this study may potentially impact the estimation of the population-scaled recombination rate. Although very little population structure is observed on the other chromosomes, the overlapping inversions have effectively created a varying number of demes for different genomic regions on the third chromosome. It is unclear how this complex genetic structure affects the estimation of recombination rates from polymorphism data using LDhelmet. Although the relative pattern of suppressed recombination is consistent with the expectation in a polymorphic inversion system, we caution against interpretations of the absolute values obtained from our estimates.</p></div><div id="sec-21" class="subsection"><h3>Nonrandom patterns of synonymous and nonsynonymous polymorphism</h3><p id="p-68">Theoretically, each arrangement arose as a single inversion mutation and phylogenies using the SNP data are consistent with this prediction (S. W. Schaeffer, unpublished data). Thus, a sweep-like effect would be expected as the chromosome becomes established in a population, lowering variability within breakpoint regions (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Navarro <em>et al.</em> 2000</a>). However, we observe consistent levels of synonymous diversity across inversion breakpoints within each arrangement and that diversity is increased across inverted segments when all individuals are analyzed together (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). These results are in contrast to the neutral theoretical models of <a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Navarro (1997</a>, <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">2000</a>), which predict low levels of genetic differentiation between arrangements in the central regions of large inversions. In addition, levels of variation are expected to be lower in the youngest arrangements and this is not true in the case of the AR arrangement. Our results also further highlight the importance of considering inversions in population genetics studies, as they can have a dramatic impact on nucleotide diversity estimates across large genomic regions.</p><p id="p-69">Besides variability between regions of high and low recombination, we also observed that average rates of synonymous and nonsynonymous polymorphism differ between amino acids. Our results demonstrate the nonrandom nature of synonymous and nonsynonymous polymorphism and perhaps indicate differential selective constraints for each amino acid. We observe the lowest three rates of nonsynonymous polymorphism in two of the aromatic amino acids (phenylalanine and tyrosine) and cysteine, which is essential for the formation of disulfide bonds and a critical component of protein secondary and tertiary structures. It is hypothesized that purifying selection acts more strongly on such structurally important amino acids and studies in humans have found significant absences of deleterious indel polymorphism compared to genome-wide averages at aromatic amino acid sites (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">De La Chaux <em>et al.</em> 2007</a>). Consistent with this hypothesis, we also observe the greatest rate of nonsynonymous polymorphism in alanine, which is the second smallest amino acid, extremely nonreactive, and rarely involved in protein function (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Betts and Russell 2003</a>). These results further demonstrate the nonrandom nature of synonymous and nonsynonymous polymorphism and are consistent with differential selective constraints for each amino acid playing a role in this observation.</p><p id="p-70">A strong bias toward the use of GC nucleotides in codons was observed, with coding regions being GC-rich and all major codons ending with a C or G nucleotide. However, the rate of mutations at synonymous codon positions and nucleotide composition in noncoding regions (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a> and <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>) indicates a strong bias for G/C→A/T mutations over A/T→ G/C mutations. This mutational bias in favor of AT has been noted numerous times in <em>D. pseudoobscura</em> and other <em>Drosophila</em> species and occurs in the opposite direction expected from the high GC content of coding regions (<a id="xref-ref-106-2" class="xref-bibr" href="#ref-106">Vicario <em>et al.</em> 2007</a>). Because this mutational bias is observed across the chromosome in both coding and noncoding regions, it is unlikely that any proposed heterogeneous mutational mechanism, such as positive GC feedback (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Fryxell and Zuckerkandl 2000</a>) or misincorporation of G and C bases in early-replicating regions (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Eyre-Walker and Hurst 2001</a>) contribute to the observed pattern of codon usage bias in <em>D. pseudoobscura</em>.</p></div><div id="sec-22" class="subsection"><h3>Codon usage in regions of suppressed recombination</h3><p id="p-71">Although we observe differences in synonymous site diversity between regions of limited recombination, we do not detect a significant association of synonymous codon usage with the recombination rate. Rather, the pattern of biased codon usage is statistically indistinguishable in regions of greatly suppressed recombination from regions of elevated recombination. Contradicting the expectations of Hill-Robertson interference and numerous analyses in <em>D. melanogaster</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Begun and Aquadro 1992</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Kliman and Hey 1993</a>), this result agrees with the recent observations of <a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">Kliman (2014)</a>, who did not find a significant relationship between recombination and codon bias on the second chromosome of <em>D. pseudoobscura</em>.</p><p id="p-72">Significant differences are detected in the mean estimates of <em>ρ</em> and <em>π</em><sub>S</sub> between the medial overlapping inverted regions of the chromosome among gene arrangements. Because both <em>ρ</em> and <em>π</em><sub>S</sub> are scaled by <em>N<sub>e</sub></em>, our results indicate that the relative effective population size differs among gene arrangements as a consequence of suppressed recombination resulting from the presence of overlapping inversions. Contrary to the observations of <em>ρ</em> and <em>π</em><sub>S</sub>, we do not observe any significant differences in codon usage between gene arrangements in the medial regions of the chromosome and the mean estimates of <em>Fop</em> are similar in every case.</p><p id="p-73">It is possible the recombination rate has recently changed in <em>D. pseudoobscura</em> such that the <em>N<sub>e</sub></em> has remained adequate for effective selection or the divergence of the different arrangements is young enough such that the level of biased codon usage has yet to reach equilibrium. When selective pressures change, it can take long evolutionary time periods for codon bias to decay (<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Sharp <em>et al.</em> 2010</a>). However, high levels of linkage disequilibrium are observed between distant markers on the third chromosome (<a id="xref-ref-97-6" class="xref-bibr" href="#ref-97">Schaeffer <em>et al.</em> 2003</a>), suggesting that selection on codon usage is overcoming the effects of HRI among linked loci and recombination has been suppressed since the inversions arose. Furthermore, the origin of third chromosome arrangements considered in this study varies from young (0.58 Mya, ST→AR) to relatively close to the estimated divergence of <em>D. pseudoobscura</em> from <em>D. miranda</em> 2 Mya (1.38 Mya, HY→ST; <a id="xref-ref-107-4" class="xref-bibr" href="#ref-107">Wallace <em>et al.</em> 2011</a>). We do not find a significant difference in codon usage between arrangements, suggesting the age of inversions and HRI as a result of suppressed recombination are unlikely to impact base composition and codon usage on the third chromosome of <em>D. pseudoobscura</em>.</p></div><div id="sec-23" class="subsection"><h3>Selection at synonymous sites</h3><p id="p-74">Another process hypothesized to increase GC content and affect synonymous codon usage is GC-biased gene conversion (gBGC), where double-strand break repairs are favored toward GC alleles during meiosis in heterozygotes (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Marais <em>et al.</em> 2001</a>). Although gBGC itself is neutral with respect to fitness, it can mirror the effects of selection by causing GC alleles to have a high probability of fixation (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Capra <em>et al.</em> 2013</a>). The intensity of gBGC is predicted to be greater in large outcrossing populations and in regions of high recombination (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Duret and Galtier 2009</a>). gBGC can be distinguished from natural selection because it affects both coding and noncoding regions, whereas selection acts directly on coding regions and particularly third codon sites (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Muyle <em>et al.</em> 2011</a>). We observe greater AT content in noncoding regions across the chromosome and the greatest levels of GC in third codon positions. It is therefore implausible that gBGC explains the chromosome-wide patterns of synonymous codon usage in arrangements of <em>D. pseudoobscura</em>. Additionally, as discussed earlier we do not observe a significant association between codon usage and recombination that would be expected if biased gene conversion were responsible for base composition and codon bias. The independence of codon usage from the effects of HRI and gBGC has recently been suggested on the X chromosome of <em>D. melanogaster</em> (<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Campos <em>et al.</em> 2013</a>). Thus, selection on synonymous sites arises as the likely explanation for the patterns of codon usage in <em>D. pseudoobscura</em>.</p><p id="p-75">Using the model of <a id="xref-ref-112-5" class="xref-bibr" href="#ref-112">Zeng and Charlesworth (2009)</a>, we find that a model including selection fits the SFS of preferred and unpreferred polymorphisms significantly better than a model only including mutational bias, while also taking into account the demographic histories of each arrangement independently. We also find that that the strength of selection acting on synonymous sites does not change as a function of the recombination rate, further supporting our conclusion that selection overcomes the <em>N<sub>e</sub></em> reducing effects of HRI in <em>D. pseudoobscura</em>. The intensity of selection does not significantly change between arrangements across classes of codon bias and recombination rates, despite evidence of considerably different effective population sizes, demographic histories and distribution in nature. Selection on codon usage thus appears largely insensitive to the effects of demography, <em>N<sub>e</sub></em>, recombination, biased gene conversion and mutational bias in <em>D. pseudoobscura</em>.</p><p id="p-76">Alternative explanations are possible for the apparent lack of association between recombination and codon usage. As suggested by <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Haddrill <em>et al.</em> (2007)</a>, even very low frequencies of cross over events may be enough to maintain the efficacy of selection. Although significantly reduced in medial regions of the chromosome, our results show that recombination still takes place between arrangements and double crossover as well as gene conversion events have been observed to occur (<a id="xref-ref-97-7" class="xref-bibr" href="#ref-97">Schaeffer <em>et al.</em> 2003</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Schaeffer and Anderson 2005</a>).</p></div><div id="sec-24" class="subsection"><h3>Stronger selection against unpreferred change than for preferred changes</h3><p id="p-77">Because the full SFS is used in the model of <a id="xref-ref-112-6" class="xref-bibr" href="#ref-112">Zeng and Charlesworth (2009)</a>, <em>γ</em> estimated from the SFS constructed with regards to major codons would be expected to equal –<em>γ</em> estimated from the SFS constructed with regards to minor codons. Furthermore, <em>κ</em> estimated from the SFS constructed with regards to major codons would be expected to equal 1/<em>κ</em> estimated from the SFS constructed with regards to minor codons. As expected, we observe a negative selection coefficient and a mutational bias parameter &lt; 1 for estimates using the SFS constructed with regards to minor codons. However, for the entire chromosome and for every arrangement, we find that the magnitude of <em>γ</em> is greater for the SFS based on minor codons and unpreferred changes. This may be the result of a true increase in selection against unpreferred changes or a departure from the assumptions of the model. Any algorithm that searches for ML estimates can become trapped in local maximums within the parameter space. However, we used multiple random start points for each parameter and obtained ML estimates after running the algorithm for 10000 independent iterations of each parameter combination.</p><p id="p-78">Significantly stronger selection against unpreferred changes is also found across classes of codon bias and recombination rates (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>), which suggests that there is consistent selection against unpreferred mutations regardless of the level of codon bias in a gene. On the other hand, we find a significant relationship between the strength of selection acting on preferred mutations and increasing levels of codon bias. Selection acts stronger to increase the frequency of major codons for genes with higher levels of codon bias. Furthermore, we observe that the association between major codons and evolutionary conserved sites is strongest in the most highly biased genes. Thus, our data suggests the equilibrium level of optimal codon bias varies between genes.</p><p id="p-79"><a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Hershberg and Petrov (2008)</a> theorized that the function describing the relationship between fitness and codon bias, and therefore the strength of selection in favor of preferred synonymous changes, may differ between genes. For a gene with a low optimal level of codon bias, there may be no fitness benefit to further increase the frequency of major codons in the gene. Hence selection will not act to increase any additional preferred mutations. However, fitness may decrease if unpreferred mutations move the level of codon bias away from the optimum equilibrium value. Here, selection will still act against mutations increasing the number of minor codons in the gene. For a gene with a high optimal level of codon bias, stronger selective pressure will be generated to increase the frequency of major codons and the gene will have to travel through a greater space of suboptimal levels of codon bias until it reaches the optimum. Our results empirically support such a theoretical model of differential selection strength on codon bias. We observe that <em>γ</em> is indistinguishable or near-indistinguishable from 0 for preferred mutations in genes with the lowest <em>Fop</em> estimates, yet the intensity of <em>γ</em> against unpreferred mutations is equally strong across all genes regardless of <em>Fop</em>. Genes with the lowest estimates of <em>Fop</em> may have low equilibrium optimal levels of codon bias, and therefore additional preferred mutations are effectively neutral.</p></div><div id="sec-25" class="subsection"><h3>Evidence for stabilizing selection acting on synonymous codon usage</h3><p id="p-80">Results from this study follow in a long line of literature documenting a preference for G and C ending codons in a background of mutational bias toward A and T across <em>Drosophila</em> species (<a id="xref-ref-106-3" class="xref-bibr" href="#ref-106">Vicario <em>et al.</em> 2007</a>). Despite extensive evidence for selection maintaining high codon bias and GC content in coding regions, the mechanism for selection on this preference in the opposite direction of more frequent AT mutations remains less clear. Furthermore, standard models of selection on codon usage (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Li 1987</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Bulmer 1991</a>), where the strength of selection scales with <em>N<sub>e</sub></em>, cannot explain the results of this study and the general lack of relationship between codon usage for a species and inferred <em>N<sub>e</sub></em> for a species. Instead, our work provides empirical observations that are consistent with recent models of stabilizing selection causing codon usage bias and support a mechanism for preference of GC-ending codons in an AT-rich mutational background.</p><p id="p-81">Previous work has attributed the apparent lack of association between the effective population size of a species and its level of codon usage bias to synergistic fitness effects at synonymous sites (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akashi 1995</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Kondrashov <em>et al.</em> 2006</a>). Under this framework, it is predicted that the intensity of selection against unpreferred mutations increases as the proportion of sites with minor codons increases. In contrast to this expectation, we find that the intensity of selection against unpreferred mutations remains constant regardless of the level of codon bias and across chromosome arrangements of the same species that differ in <em>N<sub>e</sub></em>. Additionally, recent simulations demonstrate that the scaled intensity of purifying selection with synergistic fitness effects is in fact proportional to <em>N<sub>e</sub></em> (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Charlesworth 2013</a>), which contradicts these previous suggestions and the results presented here. Recently, <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Charlesworth (2013)</a> developed analytical approximations for parameters of interest in a model of interacting stabilizing selection, mutational bias and drift. A novel conclusion of this model is that the intensity of selection for individual sites under stabilizing selection is effectively independent of <em>N<sub>e</sub></em> if mutational bias is large enough to perturb the population mean of a quantitative trait away from the optimal value by even small amounts.</p><p id="p-82">By definition, selection occurs in the opposite direction of mutational bias under this model. Our results demonstrate that nearly a fourfold mutational bias toward AT exists across the chromosome, which is clearly sufficient to alter the base composition of noncoding regions. As discussed previously, we also find support for the hypothesis that the mean optimal level of codon bias varies among genes and provide evidence that preferred mutations are effectively neutral in genes that may have a low optimal level of codon bias. Numerical results of <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Charlesworth (2013)</a> demonstrate that a significant pressure of selection is created even when the difference between the mean value of a trait and its optimum is small under a model of stabilizing selection. These analytical and simulation results of a trait evolving under stabilizing selection use biologically plausible parameter values with a level of mutational bias similar to our observations (<em>κ</em> = 2−4). For a quantitative trait under stabilizing selection, the intensity of selection is expected to scale with the amount of mutational bias, allowing <em>γ</em> to stay constant over a wide range of parameter space. Our data support this prediction, as each chromosome arrangement differs in local recombination rate, demographic history and effective population size, yet the pattern of codon usage and intensity of selection acting on synonymous sites remain consistent between them.</p><p id="p-83">Importantly, the hypothesis of stabilizing selection does not contradict the well-established relationship between gene expression and codon bias. <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Qian <em>et al.</em> (2012)</a> developed evolutionary models from <em>in vivo</em> estimates of translation speeds of codons in <em>Saccharomyces cervisiae</em> that consider the fitness benefits of translational accuracy and translational efficiency as two separate and sometimes conflicting processes. Genes that are highly expressed have a greater overall need for cellular translational efficiency than genes that are expressed at low levels. Codons that increase the translational accuracy of a gene in some instances decrease the efficiency of translation, and thus different selective pressures for accuracy and efficiency jointly determine the optimal level of codon usage. Here, there may be negative fitness effects to alter the level of codon usage away from the optimal value in either direction. If <em>D. pseudoobscura</em> has evolved increased translational efficiency for GC-ending codons, greater GC levels are expected in genes with fitness benefits of increased expression.</p><p id="p-84">For the third chromosome of <em>D. pseudoobscura</em>, stabilizing selection can explain the pattern of codon usage and lack of association between selection intensity on synonymous sites and suppressed recombination between arrangements. Stabilizing selection also explains the preference of GC codons in the presence of a high mutational bias toward AT. Because AT mutations are more frequent, this model also explains the consistent and greater intensity of selection acting against unpreferred changes as they have a greater probability of moving the level of codon bias in a gene away from its optimal level. Our results suggest that a role for stabilizing selection acting on codon usage bias should be considered more carefully, especially with the increasing number of population resequencing studies.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-85">We thank Mohamed Noor and Doris Bachtrog for providing additional sequence data, Kai Zeng for providing computer code, as well as three anonymous reviewers for their helpful comments on the analyses presented in this manuscript. In addition, we thank two anonymous reviewers for their constructive comments on an early version of this manuscript submitted to <em>Molecular Biology &amp; Evolution</em>. This work was supported by National Institutes of Health grant R01 GM098478 to S.W.S. and The Penn State/ National Institutes of Health CBIOS Training Grant to Z.L.F.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-11"><p id="p-86">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.014860/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.014860/-/DC1</a></p></li><li class="fn" id="fn-12"><p id="p-87"><em>Communicating editor: J. M. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-10-07"><span class="received-label">Received </span>October 7, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-10-16"><span class="accepted-label">Accepted </span>October 16, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Fuller <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Synonymous codon usage in <em>Drosophila melanogaster</em>: natural selection and translational accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D136%26rft.issue%253D3%26rft.spage%253D927%26rft.epage%253D935%26rft.atitle%253DSynonymous%2Bcodon%2Busage%2Bin%2BDrosophila%2Bmelanogaster%253A%2Bnatural%2Bselection%2Band%2Btranslational%2Baccuracy.%26rft_id%253Dinfo%253Apmid%252F8005445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM2LzMvOTI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Inferring weak selection from patterns of polymorphism and divergence at “silent” sites in <em>Drosophila</em> DNA.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">1067</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D139%26rft.issue%253D2%26rft.spage%253D1067%26rft.epage%253D1076%26rft.atitle%253DInferring%2Bweak%2Bselection%2Bfrom%2Bpatterns%2Bof%2Bpolymorphism%2Band%2Bdivergence%2Bat%2B%2522silent%2522%2Bsites%2Bin%2BDrosophila%2BDNA.%26rft_id%253Dinfo%253Apmid%252F7713409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzOS8yLzEwNjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI0MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Natural selection and the frequency distributions of “silent” DNA polymorphism in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D146%26rft.issue%253D1%26rft.spage%253D295%26rft.epage%253D307%26rft.atitle%253DNatural%2BSelection%2Band%2Bthe%2BFrequency%2BDistributions%2Bof%2B%2560%2560Silent%2527%2527%2BDNA%2BPolymorphism%2Bin%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F9136019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzEvMjk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.4" data-doi="10.1017/S0016672301004955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Depaulis</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Inversion polymorphisms and nucleotide variability in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DAndolfatto%26rft.auinit1%253DP.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D8%26rft.atitle%253DInversion%2Bpolymorphisms%2Band%2Bnucleotide%2Bvariability%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672301004955%26rft_id%253Dinfo%253Apmid%252F11279826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672301004955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11279826&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168340200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.5" data-doi="10.1093/gbe/evq086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachtrog</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Effective population size and the efficacy of selection on the x chromosomes of two closely related <em>Drosophila</em> species.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevq086%26rft_id%253Dinfo%253Apmid%252F21173424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjMvMC8xMTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI0MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.6" data-doi="10.1093/molbev/msp291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arguello</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kado</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Recombination yet inefficient selection along the <em>Drosophila melanogaster</em> subgroup’s fourth chromosome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">848</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp291%26rft_id%253Dinfo%253Apmid%252F20008457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI3LzQvODQ4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.7" data-doi="10.1111/j.1558-5646.2012.01836.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayala</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guerrero</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reproductive isolation and local adaptation quantified for a chromosome inversion in a malaria mosquito.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">946</span>–<span class="cit-lpage">958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D946%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2012.01836.x%26rft_id%253Dinfo%253Apmid%252F23550747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2012.01836.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23550747&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bachtrog</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Protein evolution and codon usage bias on the neo-sex chromosomes of <em>Drosophila miranda</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">1221</span>–<span class="cit-lpage">1232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBachtrog%26rft.auinit1%253DD.%26rft.volume%253D165%26rft.issue%253D3%26rft.spage%253D1221%26rft.epage%253D1232%26rft.atitle%253DProtein%2BEvolution%2Band%2BCodon%2BUsage%2BBias%2Bon%2Bthe%2BNeo-Sex%2BChromosomes%2Bof%2BDrosophila%2Bmiranda%26rft_id%253Dinfo%253Apmid%252F14668377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS8zLzEyMjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI0MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.9" data-doi="10.1007/s00239-006-0257-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bachtrog</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Reduced selection for codon usage bias in <em>Drosophila miranda</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">586</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DBachtrog%26rft.auinit1%253DD.%26rft.volume%253D64%26rft.issue%253D5%26rft.spage%253D586%26rft.epage%253D590%26rft.atitle%253DReduced%2Bselection%2Bfor%2Bcodon%2Busage%2Bbias%2Bin%2BDrosophila%2Bmiranda.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-006-0257-x%26rft_id%253Dinfo%253Apmid%252F17457633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-006-0257-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17457633&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246311200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.10" data-doi="10.1534/genetics.112.145599"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basatena</span>  <span class="cit-name-given-names">N.-K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoggart</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coin</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Reilly</span>  <span class="cit-name-given-names">P. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The effect of genomic inversions on estimation of population genetic parameters from sNP data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145599%26rft_id%253Dinfo%253Apmid%252F23150602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzEvMjQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.11" data-doi="10.1038/356519a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Levels of naturally occurring DNA polymorphism correlate with recombination rates in <em>D. melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">356</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBegun%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D356%26rft.issue%253D6369%26rft.spage%253D519%26rft.epage%253D520%26rft.atitle%253DLevels%2Bof%2Bnaturally%2Boccurring%2BDNA%2Bpolymorphism%2Bcorrelate%2Bwith%2Brecombination%2Brates%2Bin%2BD.%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F356519a0%26rft_id%253Dinfo%253Apmid%252F1560824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/356519a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1560824&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HN23100057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.12" data-doi="10.1016/j.cub.2009.02.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Betancourt</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reduced effectiveness of selection caused by a lack of recombination.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">655</span>–<span class="cit-lpage">660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBetancourt%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D19%26rft.issue%253D8%26rft.spage%253D655%26rft.epage%253D660%26rft.atitle%253DReduced%2Beffectiveness%2Bof%2Bselection%2Bcaused%2Bby%2Ba%2Black%2Bof%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.02.039%26rft_id%253Dinfo%253Apmid%252F19285399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.02.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19285399&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265718300027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2433.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Betts</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Amino acid properties and consequences of substitutions</span>, pp. <span class="cit-fpage">289</span>–<span class="cit-lpage">316</span> in <span class="cit-source">Bioinformatics for Geneticists</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">M. R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">I. C.</span></span>. <span class="cit-publ-name">John Wiley &amp; Sons, Ltd.</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The selection-mutation-drift theory of synonymous codon usage.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">897</span>–<span class="cit-lpage">907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBulmer%26rft.auinit1%253DM.%26rft.volume%253D129%26rft.issue%253D3%26rft.spage%253D897%26rft.epage%253D907%26rft.atitle%253DThe%2Bselection-mutation-drift%2Btheory%2Bof%2Bsynonymous%2Bcodon%2Busage.%26rft_id%253Dinfo%253Apmid%252F1752426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI5LzMvODk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.15" data-doi="10.1093/molbev/mss222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haddrill</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Codon usage bias and effective population sizes on the X chromosome <em>vs.</em> the autosomes in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">811</span>–<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss222%26rft_id%253Dinfo%253Apmid%252F23204387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzQvODExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.16" data-doi="10.1371/journal.pgen.1003684"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capra</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kostka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollard</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siepel</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A model-based analysis of GC-biased gene conversion in the human and chimpanzee genomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003684%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003684%26rft_id%253Dinfo%253Apmid%252F23966869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003684&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23966869&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlini</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">In vivo introduction of unpreferred synonymous codons into the <em>Drosophila</em> Adh gene results in reduced levels of ASH protein.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCarlini%26rft.auinit1%253DD.%2BB.%26rft.volume%253D163%26rft.issue%253D1%26rft.spage%253D239%26rft.epage%253D243%26rft.atitle%253DIn%2BVivo%2BIntroduction%2Bof%2BUnpreferred%2BSynonymous%2BCodons%2BInto%2Bthe%2BDrosophila%2BAdh%2BGene%2BResults%2Bin%2BReduced%2BLevels%2Bof%2BADH%2BProtein%26rft_id%253Dinfo%253Apmid%252F12586711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzEvMjM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.18" data-doi="10.1371/journal.pgen.1003090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide fine-scale recombination rate variation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003090%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003090%26rft_id%253Dinfo%253Apmid%252F23284288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284288&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.19" data-doi="10.1038/nrg2526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Effective population size and patterns of molecular evolution and variation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D195%26rft.epage%253D205%26rft.atitle%253DFundamental%2Bconcepts%2Bin%2Bgenetics%253A%2Beffective%2Bpopulation%2Bsize%2Band%2Bpatterns%2Bof%2Bmolecular%2Bevolution%2Band%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2526%26rft_id%253Dinfo%253Apmid%252F19204717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19204717&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263457000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.20" data-doi="10.1534/genetics.113.151555"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stabilizing selection, purifying selection, and mutational bias in finite populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151555%26rft_id%253Dinfo%253Apmid%252F23709636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvOTU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.21" data-doi="10.1017/S0016672300013343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Selection of new inversions in multi-locus genetic systems.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D21%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300013343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300013343&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1973P624400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.22" data-doi="10.1186/1471-2148-7-191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De la Chaux</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arndt</span>  <span class="cit-name-given-names">P. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">DNA indels in coding regions reveal selective constraints on protein evolution in the human lineage.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253Dde%2Bla%2BChaux%26rft.auinit1%253DN.%26rft.volume%253D7%26rft.spage%253D191%26rft.epage%253D191%26rft.atitle%253DDNA%2Bindels%2Bin%2Bcoding%2Bregions%2Breveal%2Bselective%2Bconstraints%2Bon%2Bprotein%2Bevolution%2Bin%2Bthe%2Bhuman%2Blineage.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-191%26rft_id%253Dinfo%253Apmid%252F17935613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17935613&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.23" data-doi="10.1007/s00239-006-0142-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cirulli</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Fine-scale crossover rate heterogeneity in <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DCirulli%26rft.auinit1%253DE.%2BT.%26rft.volume%253D64%26rft.issue%253D1%26rft.spage%253D129%26rft.epage%253D135%26rft.atitle%253DFine-scale%2Bcrossover%2Brate%2Bheterogeneity%2Bin%2BDrosophila%2Bpseudoobscura.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-006-0142-7%26rft_id%253Dinfo%253Apmid%252F17160365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-006-0142-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17160365&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243047800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Population, evolutionary and genomic consequences of interference selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D161%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D410%26rft.atitle%253DPopulation%252C%2BEvolutionary%2Band%2BGenomic%2BConsequences%2Bof%2BInterference%2BSelection%26rft_id%253Dinfo%253Apmid%252F12019253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYxLzEvMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguadé</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Natural selection on synonymous sites is correlated with gene length and recombination in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D151%26rft.issue%253D1%26rft.spage%253D239%26rft.epage%253D249%26rft.atitle%253DNatural%2BSelection%2Bon%2BSynonymous%2BSites%2BIs%2BCorrelated%2BWith%2BGene%2BLength%2Band%2BRecombination%2Bin%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F9872963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUxLzEvMjM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.26" data-doi="10.1371/journal.pgen.1003056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of inversion polymorphisms in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003056%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003056%26rft_id%253Dinfo%253Apmid%252F23284285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284285&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.27" data-doi="10.1016/j.cub.2006.08.067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Selection intensity on preferred codons correlates with overall codon usage bias in <em>Caenorhabditis remanei</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol. CB</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">2053</span>–<span class="cit-lpage">2057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%2BCB%26rft.volume%253D16%26rft.spage%253D2053%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2006.08.067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2006.08.067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.28" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-article-title">Genetics of natural populations. XIX. Origin of heterosis through natural selection in populations of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDobzhansky%26rft.auinit1%253DT.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D288%26rft.epage%253D302%26rft.atitle%253DGENETICS%2BOF%2BNATURAL%2BPOPULATIONS.%2BXIX.%2BORIGIN%2BOF%2BHETEROSIS%2BTHROUGH%2BNATURAL%2BSELECTION%2BIN%2BPOPULATIONS%2BOF%2BDROSOPHILA%2BPSEUDOOBSCURA%26rft_id%253Dinfo%253Apmid%252F15414931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzNS8zLzI4OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Queal</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-article-title">Genetics of natural populations. II. Genic variation in populations of <em>Drosophila pseudoobscura</em> inhabiting isolated mountain ranges.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDobzhansky%26rft.auinit1%253DTh.%26rft.volume%253D23%26rft.issue%253D5%26rft.spage%253D463%26rft.epage%253D484%26rft.atitle%253DGENETICS%2BOF%2BNATURAL%2BPOPULATIONS.%2BII.%2BGENIC%2BVARIATION%2BIN%2BPOPULATIONS%2BOF%2BDROSOPHILA%2BPSEUDOOBSCURA%2BINHABITING%2BISOLATED%2BMOUNTAIN%2BRANGES%26rft_id%253Dinfo%253Apmid%252F17246895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIyMy81LzQ2MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-article-title">Inversions in the chromosomes of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDobzhansky%26rft.auinit1%253DTh.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D28%26rft.epage%253D64%26rft.atitle%253DINVERSIONS%2BIN%2BTHE%2BCHROMOSOMES%2BOF%2BDROSOPHILA%2BPSEUDOOBSCURA%26rft_id%253Dinfo%253Apmid%252F17246876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiIyMy8xLzI4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.32" data-doi="10.1073/pnas.34.4.137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Epling</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1948</span> <span class="cit-article-title">The suppression of crossing over in inversion heterozygotes of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDobzhansky%26rft.auinit1%253DT.%26rft.volume%253D34%26rft.issue%253D4%26rft.spage%253D137%26rft.epage%253D141%26rft.atitle%253DThe%2BSuppression%2Bof%2BCrossing%2BOver%2Bin%2BInversion%2BHeterozygotes%2Bof%2BDrosophila%2BPseudoobscura%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.34.4.137%26rft_id%253Dinfo%253Apmid%252F16588794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiIzNC80LzEzNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.33" data-doi="10.1016/j.cell.2008.05.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDrummond%26rft.auinit1%253DD.%2BA.%26rft.volume%253D134%26rft.issue%253D2%26rft.spage%253D341%26rft.epage%253D352%26rft.atitle%253DMistranslation-induced%2Bprotein%2Bmisfolding%2Bas%2Ba%2Bdominant%2Bconstraint%2Bon%2Bcoding-sequence%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.05.042%26rft_id%253Dinfo%253Apmid%252F18662548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.05.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18662548&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257891700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.34" data-doi="10.1146/annurev-genom-082908-150001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Biased gene conversion and the evolution of mammalian genomic landscapes.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D10%26rft.spage%253D285%26rft.epage%253D311%26rft.atitle%253DBiased%2Bgene%2Bconversion%2Band%2Bthe%2Bevolution%2Bof%2Bmammalian%2Bgenomic%2Blandscapes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-082908-150001%26rft_id%253Dinfo%253Apmid%252F19630562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-082908-150001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19630562&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270313200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.35" data-doi="10.1038/35080577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The evolution of isochores.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253D549%26rft.epage%253D555%26rft.atitle%253DThe%2Bevolution%2Bof%2Bisochores.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35080577%26rft_id%253Dinfo%253Apmid%252F11433361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35080577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11433361&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169681600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.36" data-doi="10.1093/oxfordjournals.molbev.a026420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fryxell</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuckerkandl</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Cytosine deamination plays a primary role in the evolution of mammalian isochores.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1371</span>–<span class="cit-lpage">1383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DFryxell%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D17%26rft.issue%253D9%26rft.spage%253D1371%26rft.epage%253D1383%26rft.atitle%253DCytosine%2BDeamination%2BPlays%2Ba%2BPrimary%2BRole%2Bin%2Bthe%2BEvolution%2Bof%2BMammalian%2BIsochores%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026420%26rft_id%253Dinfo%253Apmid%252F10958853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjE3LzkvMTM3MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.37" data-doi="10.1534/genetics.111.132654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gossmann</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolfit</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantifying the variation in the effective population size within a genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">1389</span>–<span class="cit-lpage">1402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132654%26rft_id%253Dinfo%253Apmid%252F21954163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4OS80LzEzODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI0MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.38" data-doi="10.1186/gb-2007-8-2-r18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haddrill</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomaras</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Reduced efficacy of selection in regions of the <em>Drosophila</em> genome that lack crossing over.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHaddrill%26rft.auinit1%253DP.%2BR.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253DR18%26rft.epage%253DR18%26rft.atitle%253DReduced%2Befficacy%2Bof%2Bselection%2Bin%2Bregions%2Bof%2Bthe%2BDrosophila%2Bgenome%2Bthat%2Black%2Bcrossing%2Bover.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-2-r18%26rft_id%253Dinfo%253Apmid%252F17284312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-2-r18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17284312&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.39" data-doi="10.1093/molbev/msq354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haddrill</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Determinants of synonymous and nonsynonymous variability in three species of <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1731</span>–<span class="cit-lpage">1743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq354%26rft_id%253Dinfo%253Apmid%252F21191087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzUvMTczMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinsheimer</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Characterizing bias in population genetic inferences from low coverage sequencing data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D31%26rft.spage%253D723%26rft_id%253Dinfo%253Apmid%252F24288159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24288159&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.41" data-doi="10.1534/genetics.109.111294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hense</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsch</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Experimentally increased codon bias in the <em>Drosophila</em> Adh gene leads to an increase in larval, but not adult, alcohol dehydrogenase activity.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.111294%26rft_id%253Dinfo%253Apmid%252F19966063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg0LzIvNTQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.42" data-doi="10.1146/annurev.genet.42.110807.091442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection on codon bias.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DHershberg%26rft.auinit1%253DR.%26rft.volume%253D42%26rft.spage%253D287%26rft.epage%253D299%26rft.atitle%253DSelection%2Bon%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091442%26rft_id%253Dinfo%253Apmid%252F18983258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983258&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.43" data-doi="10.1371/journal.pgen.1001115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evidence that mutation is universally biased towards AT in bacteria.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001115%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001115%26rft_id%253Dinfo%253Apmid%252F20838599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838599&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.44" data-doi="10.1017/S0016672300010156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The effect of linkage on limits to artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D294%26rft.atitle%253DThe%2Beffect%2Bof%2Blinkage%2Bon%2Blimits%2Bto%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010156%26rft_id%253Dinfo%253Apmid%252F5980116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5980116&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668750300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.45" data-doi="10.1016/0022-2836(81)90003-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the <em>E. coli</em> translational system.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DIkemura%26rft.auinit1%253DT.%26rft.volume%253D151%26rft.issue%253D3%26rft.spage%253D389%26rft.epage%253D409%26rft.atitle%253DCorrelation%2Bbetween%2Bthe%2Babundance%2Bof%2BEscherichia%2Bcoli%2Btransfer%2BRNAs%2Band%2Bthe%2Boccurrence%2Bof%2Bthe%2Brespective%2Bcodons%2Bin%2Bits%2Bprotein%2Bgenes%253A%2Ba%2Bproposal%2Bfor%2Ba%2Bsynonymous%2Bcodon%2Bchoice%2Bthat%2Bis%2Boptimal%2Bfor%2Bthe%2BE.%2Bcoli%2Btranslational%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252881%252990003-6%26rft_id%253Dinfo%253Apmid%252F6175758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(81)90003-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6175758&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MJ92600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Codon usage and tRNA content in unicellular and multicellular organisms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DIkemura%26rft.auinit1%253DT%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D34%26rft.atitle%253DCodon%2Busage%2Band%2BtRNA%2Bcontent%2Bin%2Bunicellular%2Band%2Bmulticellular%2Borganisms%26rft_id%253Dinfo%253Apmid%252F3916708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjY6IjIvMS8xMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.47" data-doi="10.1017/S0016672300017511"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishii</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Associations between allozyme loci and gene arrangements due to hitch-hiking effects of new inversions.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D30%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300017511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300017511&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.48" data-doi="10.1101/gr.091231.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trivedi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Analysis of the genome sequences of three <em>Drosophila melanogaster</em> spontaneous mutation accumulation lines.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1195</span>–<span class="cit-lpage">1201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.091231.109%26rft_id%253Dinfo%253Apmid%252F19439516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzcvMTE5NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.49" data-doi="10.1186/1471-2148-13-100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kennington</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Patterns of genetic variation across inversions: geographic variation in the In(2L)t inversion in populations of <em>Drosophila melanogaster</em> from eastern Australia.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D13%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-13-100%26rft_id%253Dinfo%253Apmid%252F23688159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-13-100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23688159&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.50" data-doi="10.1073/pnas.78.1.454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Possibility of extensive neutral evolution under stabilizing selection with special reference to non-random usage of synonymous codons.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">454</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D78%26rft.issue%253D1%26rft.spage%253D454%26rft.epage%253D458%26rft.atitle%253DEstimation%2Bof%2Bevolutionary%2Bdistances%2Bbetween%2Bhomologous%2Bnucleotide%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.78.1.454%26rft_id%253Dinfo%253Apmid%252F6165991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI3OC8xLzQ1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-other" id="cit-4.12.2433.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-source">The Neutral Theory of Evolution</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.52" data-doi="10.1534/genetics.105.047985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Chromosome inversions, local adaptation and speciation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047985%26rft_id%253Dinfo%253Apmid%252F16204214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzEvNDE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-other" id="cit-4.12.2433.53" data-doi="10.1534/g3.114.010488"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evidence that natural selection on codon usage in Drosophila pseudoobscura varies across codons.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">4</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.010488%26rft_id%253Dinfo%253Apmid%252F24531731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvNC82ODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI0MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Reduced natural selection associated with low recombination in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1239</span>–<span class="cit-lpage">1258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DKliman%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D1239%26rft.epage%253D1258%26rft.atitle%253DReduced%2Bnatural%2Bselection%2Bassociated%2Bwith%2Blow%2Brecombination%2Bin%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Apmid%252F8277853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjEwLzYvMTIzOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.55" data-doi="10.1016/j.jtbi.2005.10.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogurtsov</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">A. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Selection in favor of nucleotides G and C diversifies evolution rates and levels of polymorphism at mammalian synonymous sites.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">240</span>: <span class="cit-fpage">616</span>–<span class="cit-lpage">626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DKondrashov%26rft.auinit1%253DF.%2BA.%26rft.volume%253D240%26rft.issue%253D4%26rft.spage%253D616%26rft.epage%253D626%26rft.atitle%253DSelection%2Bin%2Bfavor%2Bof%2Bnucleotides%2BG%2Band%2BC%2Bdiversifies%2Bevolution%2Brates%2Band%2Blevels%2Bof%2Bpolymorphism%2Bat%2Bmammalian%2Bsynonymous%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2005.10.020%26rft_id%253Dinfo%253Apmid%252F16343547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2005.10.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16343547&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238458000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kovacevic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Molecular population genetics of X-linked genes in <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKovacevic%26rft.auinit1%253DM.%26rft.volume%253D156%26rft.issue%253D1%26rft.spage%253D155%26rft.epage%253D172%26rft.atitle%253DMolecular%2BPopulation%2BGenetics%2Bof%2BX-Linked%2BGenes%2Bin%2BDrosophila%2Bpseudoobscura%26rft_id%253Dinfo%253Apmid%252F10978282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU2LzEvMTU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.57" data-doi="10.1371/journal.pgen.1000550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulathinal</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevison</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genomics of speciation in <em>Drosophila</em>: diversity, divergence, and introgression estimated using low-coverage genome sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKulathinal%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000550%26rft.epage%253De1000550%26rft.atitle%253DThe%2Bgenomics%2Bof%2Bspeciation%2Bin%2BDrosophila%253A%2Bdiversity%252C%2Bdivergence%252C%2Band%2Bintrogression%2Bestimated%2Busing%2Blow-coverage%2Bgenome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000550%26rft_id%253Dinfo%253Apmid%252F19578407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000550&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19578407&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.58" data-doi="10.1534/genetics.112.142018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y. C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic variation in natural populations of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142018%26rft_id%253Dinfo%253Apmid%252F22673804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNTMzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.59" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.60" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.61" data-doi="10.1007/BF02134132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Models of nearly neutral mutations with particular implications for nonrandom usage of synonymous codons.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DLi%26rft.auinit1%253DW.%2BH.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D337%26rft.epage%253D345%26rft.atitle%253DModels%2Bof%2Bnearly%2Bneutral%2Bmutations%2Bwith%2Bparticular%2Bimplications%2Bfor%2Bnonrandom%2Busage%2Bof%2Bsynonymous%2Bcodons.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02134132%26rft_id%253Dinfo%253Apmid%252F3110426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02134132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3110426&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G990300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.62" data-doi="10.1093/nar/gkf546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynn</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">G. A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Synonymous codon usage is subject to selection in thermophilic bacteria.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">4272</span>–<span class="cit-lpage">4277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLynn%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D30%26rft.issue%253D19%26rft.spage%253D4272%26rft.epage%253D4277%26rft.atitle%253DSynonymous%2Bcodon%2Busage%2Bis%2Bsubject%2Bto%2Bselection%2Bin%2Bthermophilic%2Bbacteria%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkf546%26rft_id%253Dinfo%253Apmid%252F12364606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xOS80MjcyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.63" data-doi="10.1534/genetics.106.064758"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Machado</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haselkorn</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evaluation of the genomic extent of effects of fixed inversion differences on intraspecific variation and interspecific gene flow in <em>Drosophila pseudoobscura</em> and <em>D. persimilis</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1289</span>–<span class="cit-lpage">1306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.064758%26rft_id%253Dinfo%253Apmid%252F17179068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS8zLzEyODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI0MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.64" data-doi="10.1073/pnas.091427698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marais</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mouchiroud</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Does recombination improve selection on codon usage? Lessons from nematode and fly complete genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">5688</span>–<span class="cit-lpage">5692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.091427698%26rft_id%253Dinfo%253Apmid%252F11320215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTgvMTAvNTY4OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.65" data-doi="10.1016/j.cub.2003.12.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maside</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Selection on codon usage in <em>Drosophila americana</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol. CB</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%2BCB%26rft.volume%253D14%26rft.spage%253D150%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2003.12.055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2003.12.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.66" data-doi="10.1371/journal.pbio.1001422"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGaugh</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heil</span>  <span class="cit-name-given-names">C. S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manzano-Winkler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loewe</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recombination modulates how selection affects linked sites in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1001422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DMcGaugh%26rft.auinit1%253DS.%2BE.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253De1001422%26rft.epage%253De1001422%26rft.atitle%253DRecombination%2Bmodulates%2Bhow%2Bselection%2Baffects%2Blinked%2Bsites%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001422%26rft_id%253Dinfo%253Apmid%252F23152720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001422&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23152720&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.67" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.68" data-doi="10.1017/S0016672399003912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A population genetic model for the evolution of synonymous codon usage: patterns and predictions.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D74%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399003912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399003912&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000083662500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The effects of Hill-Robertson interference between weakly selected mutations on patterns of molecular evolution and variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">929</span>–<span class="cit-lpage">944</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%2BA.%2BT.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D929%26rft.epage%253D944%26rft.atitle%253DThe%2BEffects%2Bof%2BHill-Robertson%2BInterference%2BBetween%2BWeakly%2BSelected%2BMutations%2Bon%2BPatterns%2Bof%2BMolecular%2BEvolution%2Band%2BVariation%26rft_id%253Dinfo%253Apmid%252F10835411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTI5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.70" data-doi="10.1126/science.1092500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deloukas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The fine-scale structure of recombination rate variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%2BA.%2BT.%26rft.volume%253D304%26rft.issue%253D5670%26rft.spage%253D581%26rft.epage%253D584%26rft.atitle%253DThe%2BFine-Scale%2BStructure%2Bof%2BRecombination%2BRate%2BVariation%2Bin%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1092500%26rft_id%253Dinfo%253Apmid%252F15105499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDQvNTY3MC81ODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI0MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mestres</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serra</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Chromosomal structure and recombination between inversions in <em>Drosophila subobscura</em>.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DMestres%26rft.auinit1%253DF.%26rft.volume%253D128%26rft.issue%253D2%26rft.spage%253D105%26rft.epage%253D113%26rft.atitle%253DChromosomal%2Bstructure%2Band%2Brecombination%2Bbetween%2Binversions%2Bin%2BDrosophila%2Bsubobscura.%26rft_id%253Dinfo%253Apmid%252F9652229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9652229&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074318500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.72" data-doi="10.1093/molbev/msr104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muyle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serres-Giardi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ressayre</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Escobar</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glémin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GC-biased gene conversion and selection affect GC content in the Oryza Genus (rice).</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2695</span>–<span class="cit-lpage">2706</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr104%26rft_id%253Dinfo%253Apmid%252F21504892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzkvMjY5NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betran</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Recombination and gene flux caused by gene conversion and crossing over in inversion heterokaryotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNavarro%26rft.auinit1%253DA.%26rft.volume%253D146%26rft.issue%253D2%26rft.spage%253D695%26rft.epage%253D709%26rft.atitle%253DRecombination%2Band%2BGene%2BFlux%2BCaused%2Bby%2BGene%2BConversion%2Band%2BCrossing%2BOver%2Bin%2BInversion%2BHeterokaryotypes%26rft_id%253Dinfo%253Apmid%252F9178017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzIvNjk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Effect of inversion polymorphism on the neutral nucleotide variability of linked chromosomal regions in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">685</span>–<span class="cit-lpage">698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNavarro%26rft.auinit1%253DA.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D685%26rft.epage%253D698%26rft.atitle%253DEffect%2Bof%2BInversion%2BPolymorphism%2Bon%2Bthe%2BNeutral%2BNucleotide%2BVariability%2Bof%2BLinked%2BChromosomal%2BRegions%2Bin%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F10835391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvNjg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2433.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-source">Molecular Evolutionary Genetics. Reprint edition</span>. <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gojobori</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">418</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DNei%26rft.auinit1%253DM%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253D418%26rft.epage%253D426%26rft.atitle%253DSimple%2Bmethods%2Bfor%2Bestimating%2Bthe%2Bnumbers%2Bof%2Bsynonymous%2Band%2Bnonsynonymous%2Bnucleotide%2Bsubstitutions%26rft_id%253Dinfo%253Apmid%252F3444411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjMvNS80MTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI0MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kojima</span>  <span class="cit-name-given-names">K.-I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">Frequency changes of new inversions in populations under mutation-selection equilibria.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">750</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D741%26rft.epage%253D750%26rft.atitle%253DFREQUENCY%2BCHANGES%2BOF%2BNEW%2BINVERSIONS%2BIN%2BPOPULATIONS%2BUNDER%2BMUTATION-SELECTION%2BEQUILIBRIA%26rft_id%253Dinfo%253Apmid%252F6082619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI1Ny80Lzc0MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.78" data-doi="10.1093/molbev/msl146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer DuMont</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Maximum likelihood estimation of ancestral codon usage bias parameters in Drosophila.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">228</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl146%26rft_id%253Dinfo%253Apmid%252F17041152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI0LzEvMjI4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.79" data-doi="10.1016/j.jmb.2004.08.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novella</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zárate</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzgar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebendick-Corpus</span>  <span class="cit-name-given-names">B. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Positive selection of synonymous mutations in vesicular stomatitis virus.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">1415</span>–<span class="cit-lpage">1421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DNovella%26rft.auinit1%253DI.%2BS.%26rft.volume%253D342%26rft.issue%253D5%26rft.spage%253D1415%26rft.epage%253D1421%26rft.atitle%253DPositive%2Bselection%2Bof%2Bsynonymous%2Bmutations%2Bin%2Bvesicular%2Bstomatitis%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2004.08.003%26rft_id%253Dinfo%253Apmid%252F15364570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2004.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15364570&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224135500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.80" data-doi="10.2307/2528314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kojima</span>  <span class="cit-name-given-names">K. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Survival probabilities f new inversions in large populations.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DOhta%26rft.auinit1%253DT.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D501%26rft.epage%253D516%26rft.atitle%253DSurvival%2Bprobabilities%2Bf%2Bnew%2Binversions%2Bin%2Blarge%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2528314%26rft_id%253Dinfo%253Apmid%252F5686303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2528314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5686303&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.81" data-doi="10.1371/journal.pone.0013431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palidwor</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A general model of codon bias due to GC mutational bias.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e13431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPalidwor%26rft.auinit1%253DG.%2BA.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De13431%26rft.epage%253De13431%26rft.atitle%253DA%2Bgeneral%2Bmodel%2Bof%2Bcodon%2Bbias%2Bdue%2Bto%2BGC%2Bmutational%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0013431%26rft_id%253Dinfo%253Apmid%252F21048949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0013431&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21048949&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.82" data-doi="10.1093/molbev/msj035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parmley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamary</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evidence for purifying selection against synonymous mutations in mammalian exonic splicing enhancers.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsj035%26rft_id%253Dinfo%253Apmid%252F16221894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIzLzIvMzAxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-thesis" id="cit-4.12.2433.83"><div class="cit-metadata unstructured">Peden, J., 1999 Analysis of codon usage. Ph.D. Thesis, University of Nottingham.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.84" data-doi="10.1371/journal.pgen.1003080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of sub-Saharan <em>Drosophila melanogaster</em>: African diversity and non-African admixture.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003080%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003080%26rft_id%253Dinfo%253Apmid%252F23284287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284287&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.85" data-doi="10.1093/gbe/evt078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porceddu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenoni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camiolo</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The signatures of selection for translational accuracy in plant genes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1117</span>–<span class="cit-lpage">1126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt078%26rft_id%253Dinfo%253Apmid%252F23695187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjUvNi8xMTE3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Precup</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Missense misreading of asparagine codons as a function of codon identity and context.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">262</span>: <span class="cit-fpage">11351</span>–<span class="cit-lpage">11355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DPrecup%26rft.auinit1%253DJ.%26rft.volume%253D262%26rft.issue%253D23%26rft.spage%253D11351%26rft.epage%253D11355%26rft.atitle%253DMissense%2Bmisreading%2Bof%2Basparagine%2Bcodons%2Bas%2Ba%2Bfunction%2Bof%2Bcodon%2Bidentity%2Band%2Bcontext%26rft_id%253Dinfo%253Apmid%252F3112158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNjIvMjMvMTEzNTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI0MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-4.12.2433.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Press</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flannery</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teukolsky</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vetterling</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source">Numerical Recipes in C: the Art of Scientific Computing</span>. <span class="cit-edition">2nd ed</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.88" data-doi="10.1371/journal.pgen.1002603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maclean</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Balanced codon usage optimizes translational efficiency</span>. <abbr class="cit-jnl-abbrev">PLoS Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenetics%26rft.volume%253D8%26rft.spage%253D1002603e%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutation bias is the driving force of codon usage in the <em>Gallus gallus</em> genome.</span> <abbr class="cit-jnl-abbrev">DNA Res. Int. J. Rapid Publ. Rep. Genes Genomes</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%2BInt.%2BJ.%2BRapid%2BPubl.%2BRep.%2BGenes%2BGenomes%26rft.volume%253D18%26rft.spage%253D499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.90" data-doi="10.1101/gr.3059305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bettencourt</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hradecky</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Letovsky</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparative genome sequencing of <em>Drosophila pseudoobscura</em>: chromosomal, gene, and cis-element evolution.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DRichards%26rft.auinit1%253DS.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D18%26rft.atitle%253DComparative%2Bgenome%2Bsequencing%2Bof%2BDrosophila%2Bpseudoobscura%253A%2BChromosomal%252C%2Bgene%252C%2Band%2Bcis-element%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3059305%26rft_id%253Dinfo%253Apmid%252F15632085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjY6IjE1LzEvMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.91"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallas</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Distinguishing the forces controlling genetic variation at the Xdh locus in <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRiley%26rft.auinit1%253DM.%2BA.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D359%26rft.epage%253D369%26rft.atitle%253DDistinguishing%2Bthe%2Bforces%2Bcontrolling%2Bgenetic%2Bvariation%2Bat%2Bthe%2BXdh%2Blocus%2Bin%2BDrosophila%2Bpseudoobscura.%26rft_id%253Dinfo%253Apmid%252F2583480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzIvMzU5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2433.92"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">The Genetics of Chromosomal Aberration</span>, pp. <span class="cit-fpage">671</span>–<span class="cit-lpage">684</span> in <span class="cit-source">The Genetics and Biology of Drosophila. Vol. 1a</span>. (<span class="cit-ed"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Novitski</span>  <span class="cit-name-given-names">E.</span></span>, eds). <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.93" data-doi="10.1017/S0016672302005955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Molecular population genetics of sequence length diversity in the Adh region of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSchaeffer%26rft.auinit1%253DS.%2BW.%26rft.volume%253D80%26rft.issue%253D3%26rft.spage%253D163%26rft.epage%253D175%26rft.atitle%253DMolecular%2Bpopulation%2Bgenetics%2Bof%2Bsequence%2Blength%2Bdiversity%2Bin%2Bthe%2BAdh%2Bregion%2Bof%2BDrosophila%2Bpseudoobscura.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672302005955%26rft_id%253Dinfo%253Apmid%252F12688655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672302005955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12688655&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181272900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.94" data-doi="10.1111/j.1558-5646.2008.00504.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection in heterogeneous environments maintains the gene arrangement polymorphism of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">3082</span>–<span class="cit-lpage">3099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DSchaeffer%26rft.auinit1%253DS.%2BW.%26rft.volume%253D62%26rft.issue%253D12%26rft.spage%253D3082%26rft.epage%253D3099%26rft.atitle%253DSelection%2Bin%2Bheterogeneous%2Benvironments%2Bmaintains%2Bthe%2Bgene%2Barrangement%2Bpolymorphism%2Bof%2BDrosophila%2Bpseudoobscura.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2008.00504.x%26rft_id%253Dinfo%253Apmid%252F18764919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2008.00504.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18764919&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimates of gene flow in <em>Drosophila pseudoobscura</em> determined from nucleotide sequence analysis of the alcohol dehydrogenase region.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchaeffer%26rft.auinit1%253DS.%2BW.%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D471%26rft.epage%253D480%26rft.atitle%253DEstimates%2Bof%2Bgene%2Bflow%2Bin%2BDrosophila%2Bpseudoobscura%2Bdetermined%2Bfrom%2Bnucleotide%2Bsequence%2Banalysis%2Bof%2Bthe%2Balcohol%2Bdehydrogenase%2Bregion.%26rft_id%253Dinfo%253Apmid%252F1427038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMyLzIvNDcxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.96" data-doi="10.1534/genetics.105.041947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mechanisms of genetic exchange within the chromosomal inversions of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">1729</span>–<span class="cit-lpage">1739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.041947%26rft_id%253Dinfo%253Apmid%252F16143625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS80LzE3MjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI0MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.97" data-doi="10.1073/pnas.1432900100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goetting-Minesky</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovacevic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peoples</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graybill</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Evolutionary genomics of inversions in <em>Drosophila pseudoobscura</em>: Evidence for epistasis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">8319</span>–<span class="cit-lpage">8324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1432900100%26rft_id%253Dinfo%253Apmid%252F12824467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE0LzgzMTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI0MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.98" data-doi="10.1534/genetics.113.151670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rates and genomic consequences of spontaneous mutational events in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">937</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151670%26rft_id%253Dinfo%253Apmid%252F23733788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvOTM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.99" data-doi="10.1101/SQB.1951.016.01.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redfield</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">Interchromosomal effects on crossing over in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BSymposia%2Bon%2BQuantitative%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb%2BSymp%2BQuant%2BBiol%26rft.aulast%253DSchultz%26rft.auinit1%253DJ.%26rft.volume%253D16%26rft.issue%253D0%26rft.spage%253D175%26rft.epage%253D197%26rft.atitle%253DINTERCHROMOSOMAL%2BEFFECTS%2BON%2BCROSSING%2BOVER%2BIN%2BDROSOPHILA%26rft_id%253Dinfo%253Adoi%252F10.1101%252FSQB.1951.016.01.015%26rft_id%253Dinfo%253Apmid%252F14942738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjg6IjE2LzAvMTc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.100" data-doi="10.1093/nar/15.3.1281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The codon Adaptation Index—a measure of directional synonymous codon usage bias, and its potential applications.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1281</span>–<span class="cit-lpage">1295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D1281%26rft.epage%253D1295%26rft.atitle%253DThe%2Bcodon%2Badaptation%2Bindex-a%2Bmeasure%2Bof%2Bdirectional%2Bsynonymous%2Bcodon%2Busage%2Bbias%252C%2Band%2Bits%2Bpotential%2Bapplications%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.3.1281%26rft_id%253Dinfo%253Apmid%252F3547335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjE1LzMvMTI4MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.101" data-doi="10.1098/rstb.2009.0305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emery</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Forces that influence the evolution of codon bias.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. B Biol. Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1203</span>–<span class="cit-lpage">1212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D365%26rft.issue%253D1544%26rft.spage%253D1203%26rft.epage%253D1212%26rft.atitle%253DForces%2Bthat%2Binfluence%2Bthe%2Bevolution%2Bof%2Bcodon%2Bbias%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0305%26rft_id%253Dinfo%253Apmid%252F20308095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjUvMTU0NC8xMjAzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singleton</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">A mechanism intrinsic to heterozygous inversions affecting observed recombination frequencies in adjacent regions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSingleton%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D541%26rft.epage%253D560%26rft.atitle%253DA%2BMECHANISM%2BINTRINSIC%2BTO%2BHETEROZYGOUS%2BINVERSIONS%2BAFFECTING%2BOBSERVED%2BRECOMBINATION%2BFREQUENCIES%2BIN%2BADJACENT%2BREGIONS%26rft_id%253Dinfo%253Apmid%252F17248203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI0OS80LzU0MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.103" data-doi="10.1093/gbe/evr081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevison</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoehn</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Effects of inversions on within- and between-species recombination and divergence.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">830</span>–<span class="cit-lpage">841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr081%26rft_id%253Dinfo%253Apmid%252F21828374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjMvMC84MzAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI0MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beadle</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1936</span> <span class="cit-article-title">The relations of inversions in the X chromosome of <em>Drosophila melanogaster</em> to crossing over and disjunction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">554</span>–<span class="cit-lpage">604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSturtevant%26rft.auinit1%253DA.%2BH.%26rft.volume%253D21%26rft.issue%253D5%26rft.spage%253D554%26rft.epage%253D604%26rft.atitle%253DTHE%2BRELATIONS%2BOF%2BINVERSIONS%2BIN%2BTHE%2BX%2BCHROMOSOME%2BOF%2BDROSOPHILA%2BMELANOGASTER%2BTO%2BCROSSING%2BOVER%2BAND%2BDISJUNCTION%26rft_id%253Dinfo%253Apmid%252F17246812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIyMS81LzU1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.106" data-doi="10.1186/1471-2148-7-226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vicario</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriyama</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Codon usage in twelve species of <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DVicario%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.spage%253D226%26rft.epage%253D226%26rft.atitle%253DCodon%2Busage%2Bin%2Btwelve%2Bspecies%2Bof%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-226%26rft_id%253Dinfo%253Apmid%252F18005411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18005411&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.107" data-doi="10.1093/molbev/msr039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Detweiler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolutionary history of the third chromosome gene arrangements of <em>Drosophila pseudoobscura</em> inferred from inversion breakpoints.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2219</span>–<span class="cit-lpage">2229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr039%26rft_id%253Dinfo%253Apmid%252F21339510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzgvMjIxOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference  in text" id="ref-108">↵</a><div class="cit ref-cit ref-other" id="cit-4.12.2433.108" data-doi="10.1534/g3.113.006122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Detweiler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Molecular population genetics of inversion breakpoint regions in <em>Drosophila pseudoobscura</em>.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.006122%26rft_id%253Dinfo%253Apmid%252F23665879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjMvNy8xMTUxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference  in text" id="ref-109">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2433.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-source">Topics in Population Genetics</span>. <span class="cit-publ-name">W. W. Norton</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference  in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.110" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference  in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.111" data-doi="10.1016/0378-1119(90)90491-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The “effective number of codons” used in a gene.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DWright%26rft.auinit1%253DF.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D29%26rft.atitle%253DThe%2B%2527effective%2Bnumber%2Bof%2Bcodons%2527%2Bused%2Bin%2Ba%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252890%252990491-9%26rft_id%253Dinfo%253Apmid%252F2110097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(90)90491-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2110097&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DA22500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference  in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2433.112" data-doi="10.1534/genetics.109.101782"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimating selection intensity on synonymous codon usage in a nonequilibrium population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">651</span>–<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101782%26rft_id%253Dinfo%253Apmid%252F19620398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzIvNjUxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/12/2433.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/12/2425" title="Statistical Inference of Selection and Divergence of the Rice Blast Resistance Gene Pi-ta" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/12/2451" title="A Screen for F1 Hybrid Male Rescue Reveals No Major-Effect Hybrid Lethality Loci in the Drosophila melanogaster Autosomal Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/12"><span>Volume 4 Issue 12, December 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/12.cover-source.jpg?itok=ezho2p8P" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2433" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4dwo8-xLm8kO8SgN5xVR-R6sv0oMkF-VT14RlBQrFKs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/12/2433" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/12/2433" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/12/2433" class="active" data-icon-position="" data-hide-link-title="0">Evidence for Stabilizing Selection on Codon Usage in Chromosomal Rearrangements of Drosophila pseudoobscura</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459744" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-JpBzRRDvNGZ3FP1p23DvTGMnqVG2z_PCx8m6QcU4xi0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/12/2433.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/12/2433" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-7Z0Mxj4-2xsU5iWcapLPmhF-IpjHeqag3NTbbIln4pA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/12/2433" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/12/2433.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459744" id="node459744--2" data-pisa="ggg;4/12/2433" data-pisa-master="ggg;g3.114.014860" data-apath="/ggg/4/12/2433.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evidence for Stabilizing Selection on Codon Usage in Chromosomal Rearrangements of <em>Drosophila pseudoobscura</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4765-9227" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zachary L. Fuller</span>, <span class="highwire-citation-author" data-delta="1">Gwilym D. Haynes</span>, <span class="highwire-citation-author" data-delta="2">Dianhui Zhu</span>, <span class="highwire-citation-author" data-delta="3">Matthew Batterton</span>, <span class="highwire-citation-author" data-delta="4">Hsu Chao</span>, <span class="highwire-citation-author" data-delta="5">Shannon Dugan</span>, <span class="highwire-citation-author" data-delta="6">Mehwish Javaid</span>, <span class="highwire-citation-author" data-delta="7">Joy C. Jayaseelan</span>, <span class="highwire-citation-author" data-delta="8">Sandra Lee</span>, <span class="highwire-citation-author" data-delta="9">Mingmei Li</span>, <span class="highwire-citation-author" data-delta="10">Fiona Ongeri</span>, <span class="highwire-citation-author" data-delta="11">Sulan Qi</span>, <span class="highwire-citation-author" data-delta="12">Yi Han</span>, <span class="highwire-citation-author" data-delta="13">Harshavardhan Doddapaneni</span>, <span class="highwire-citation-author" data-delta="14">Stephen Richards</span> and <span class="highwire-citation-author" data-delta="15">Stephen W. Schaeffer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2433-2449;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014860 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459744--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zachary L. Fuller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BL.%2BFuller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuller%20ZL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZachary%2BL.%2BFuller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4765-9227" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zachary L. Fuller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zlf105@psu.edu" class="" data-icon-position="" data-hide-link-title="0">zlf105@psu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gwilym D. Haynes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gwilym%2BD.%2BHaynes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haynes%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGwilym%2BD.%2BHaynes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dianhui Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dianhui%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADianhui%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew Batterton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BBatterton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Batterton%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BBatterton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hsu Chao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hsu%2BChao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHsu%2BChao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Shannon Dugan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BDugan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dugan%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShannon%2BDugan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mehwish Javaid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mehwish%2BJavaid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Javaid%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMehwish%2BJavaid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Joy C. Jayaseelan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joy%2BC.%2BJayaseelan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jayaseelan%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoy%2BC.%2BJayaseelan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sandra Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Mingmei Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mingmei%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMingmei%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Fiona Ongeri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fiona%2BOngeri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ongeri%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFiona%2BOngeri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Sulan Qi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sulan%2BQi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qi%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASulan%2BQi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Yi Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yi%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYi%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Harshavardhan Doddapaneni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harshavardhan%2BDoddapaneni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doddapaneni%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarshavardhan%2BDoddapaneni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Stephen Richards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BRichards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Richards%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BRichards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Stephen W. Schaeffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BW.%2BSchaeffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schaeffer%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BW.%2BSchaeffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/12/2433" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/12/2433&amp;title=Evidence%20for%20Stabilizing%20Selection%20on%20Codon%20Usage%20in%20Chromosomal%20Rearrangements%20of%20Drosophila%20pseudoobscura" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/12/2433&amp;title=Evidence%20for%20Stabilizing%20Selection%20on%20Codon%20Usage%20in%20Chromosomal%20Rearrangements%20of%20Drosophila%20pseudoobscura" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/12/2433&amp;title=Evidence%20for%20Stabilizing%20Selection%20on%20Codon%20Usage%20in%20Chromosomal%20Rearrangements%20of%20Drosophila%20pseudoobscura" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/12/2433&amp;text=Evidence%20for%20Stabilizing%20Selection%20on%20Codon%20Usage%20in%20Chromosomal%20Rearrangements%20of%20Drosophila%20pseudoobscura" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/12/2433&amp;title=Evidence%20for%20Stabilizing%20Selection%20on%20Codon%20Usage%20in%20Chromosomal%20Rearrangements%20of%20Drosophila%20pseudoobscura" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/12/2433&amp;t=Evidence%20for%20Stabilizing%20Selection%20on%20Codon%20Usage%20in%20Chromosomal%20Rearrangements%20of%20Drosophila%20pseudoobscura" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/12/2433&amp;title=Evidence%20for%20Stabilizing%20Selection%20on%20Codon%20Usage%20in%20Chromosomal%20Rearrangements%20of%20Drosophila%20pseudoobscura" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/12/2433&amp;title=Evidence%20for%20Stabilizing%20Selection%20on%20Codon%20Usage%20in%20Chromosomal%20Rearrangements%20of%20Drosophila%20pseudoobscura" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459744/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459744" id="node459744--3" data-pisa="ggg;4/12/2433" data-pisa-master="ggg;g3.114.014860" data-apath="/ggg/4/12/2433.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evidence for Stabilizing Selection on Codon Usage in Chromosomal Rearrangements of <em>Drosophila pseudoobscura</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4765-9227" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zachary L. Fuller</span>, <span class="highwire-citation-author" data-delta="1">Gwilym D. Haynes</span>, <span class="highwire-citation-author" data-delta="2">Dianhui Zhu</span>, <span class="highwire-citation-author" data-delta="3">Matthew Batterton</span>, <span class="highwire-citation-author" data-delta="4">Hsu Chao</span>, <span class="highwire-citation-author" data-delta="5">Shannon Dugan</span>, <span class="highwire-citation-author" data-delta="6">Mehwish Javaid</span>, <span class="highwire-citation-author" data-delta="7">Joy C. Jayaseelan</span>, <span class="highwire-citation-author" data-delta="8">Sandra Lee</span>, <span class="highwire-citation-author" data-delta="9">Mingmei Li</span>, <span class="highwire-citation-author" data-delta="10">Fiona Ongeri</span>, <span class="highwire-citation-author" data-delta="11">Sulan Qi</span>, <span class="highwire-citation-author" data-delta="12">Yi Han</span>, <span class="highwire-citation-author" data-delta="13">Harshavardhan Doddapaneni</span>, <span class="highwire-citation-author" data-delta="14">Stephen Richards</span> and <span class="highwire-citation-author" data-delta="15">Stephen W. Schaeffer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2433-2449;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014860 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459744--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zachary L. Fuller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BL.%2BFuller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuller%20ZL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZachary%2BL.%2BFuller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4765-9227" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zachary L. Fuller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zlf105@psu.edu" class="" data-icon-position="" data-hide-link-title="0">zlf105@psu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gwilym D. Haynes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gwilym%2BD.%2BHaynes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haynes%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGwilym%2BD.%2BHaynes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dianhui Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dianhui%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADianhui%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew Batterton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BBatterton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Batterton%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BBatterton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hsu Chao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hsu%2BChao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHsu%2BChao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Shannon Dugan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BDugan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dugan%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShannon%2BDugan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mehwish Javaid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mehwish%2BJavaid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Javaid%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMehwish%2BJavaid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Joy C. Jayaseelan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joy%2BC.%2BJayaseelan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jayaseelan%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoy%2BC.%2BJayaseelan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sandra Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Mingmei Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mingmei%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMingmei%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Fiona Ongeri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fiona%2BOngeri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ongeri%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFiona%2BOngeri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Sulan Qi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sulan%2BQi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qi%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASulan%2BQi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Yi Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yi%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYi%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Harshavardhan Doddapaneni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harshavardhan%2BDoddapaneni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doddapaneni%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarshavardhan%2BDoddapaneni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Stephen Richards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BRichards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Richards%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BRichards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Stephen W. Schaeffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, The Pennsylvania State University, University Park, Pennsylvania 16802-5301</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BW.%2BSchaeffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schaeffer%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BW.%2BSchaeffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2433&amp;rft.epage=2449&amp;rft.atitle=Evidence%20for%20Stabilizing%20Selection%20on%20Codon%20Usage%20in%20Chromosomal%20Rearrangements%20of%20Drosophila%20pseudoobscura&amp;rft.volume=4&amp;rft.issue=12&amp;rft.date=2014-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Fuller%2C+Zachary+L.&amp;rft.au=Haynes%2C+Gwilym+D.&amp;rft.au=Zhu%2C+Dianhui&amp;rft.au=Batterton%2C+Matthew&amp;rft.au=Chao%2C+Hsu&amp;rft.au=Dugan%2C+Shannon&amp;rft.au=Javaid%2C+Mehwish&amp;rft.au=Jayaseelan%2C+Joy+C.&amp;rft.au=Lee%2C+Sandra&amp;rft.au=Li%2C+Mingmei&amp;rft.au=Ongeri%2C+Fiona&amp;rft.au=Qi%2C+Sulan&amp;rft.au=Han%2C+Yi&amp;rft.au=Doddapaneni%2C+Harshavardhan&amp;rft.au=Richards%2C+Stephen&amp;rft.au=Schaeffer%2C+Stephen+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459744/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459744/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459744/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459744/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459744/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459744/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459744/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459744/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459744/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459744/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459744/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459744/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/12/2433" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/12/2433.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/12/2433.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/12/2433.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/12/2433.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
