<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/12/2519" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/12/2519.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/12/2519.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/12/2519.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/12/2519.atom" />
<meta name="HW.pisa" content="ggg;4/12/2519" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Quantitative Trait Locus Mapping of Melanization in the Plant Pathogenic Fungus Zymoseptoria tritici" />
<meta name="DC.Identifier" content="10.1534/g3.114.015289" />
<meta name="DC.Date" content="2014-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Lendenmann et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Melanin plays an important role in virulence and antimicrobial resistance in several fungal pathogens. The wheat pathogen Zymoseptoria tritici is important worldwide, but little is known about the genetic architecture of pathogenicity, including the production of melanin. Because melanin production can exhibit complex inheritance, we used quantitative trait locus (QTL) mapping in two crosses to identify the underlying genes. Restriction site−associated DNA sequencing was used to genotype 263 (cross 1) and 261 (cross 2) progeny at ~8500 single-nucleotide polymorphisms and construct two dense linkage maps. We measured gray values, representing degrees of melanization, for single-spore colonies growing on Petri dishes by using a novel image-processing approach that enabled high-throughput phenotyping. Because melanin production can be affected by stress, each offspring was grown in two stressful environments and one control environment. We detected six significant QTL in cross 1 and nine in cross 2, with three QTL shared between the crosses. Different QTL were identified in different environments and at different colony ages. By obtaining complete genome sequences for the four parents and analyzing sequence variation in the QTL confidence intervals, we identified 16 candidate genes likely to affect melanization. One of these candidates was PKS1 , a polyketide synthase gene known to play a role in the synthesis of dihydroxynaphthalene melanin. Three candidate quantitative trait nucleotides were identified in PKS1 . Many of the other candidate genes were not previously associated with melanization." />
<meta name="DC.Contributor" content="Mark H. Lendenmann" />
<meta name="DC.Contributor" content="Daniel Croll" />
<meta name="DC.Contributor" content="Ethan L. Stewart" />
<meta name="DC.Contributor" content="Bruce A. McDonald" />
<meta name="article:published_time" content="2014-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Quantitative Trait Locus Mapping of Melanization in the Plant Pathogenic Fungus Zymoseptoria tritici" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Melanin plays an important role in virulence and antimicrobial resistance in several fungal pathogens. The wheat pathogen &lt;i&gt;Zymoseptoria tritici&lt;/i&gt; is important worldwide, but little is known about the genetic architecture of pathogenicity, including the production of melanin. Because melanin production can exhibit complex inheritance, we used quantitative trait locus (QTL) mapping in two crosses to identify the underlying genes. Restriction site−associated DNA sequencing was used to genotype 263 (cross 1) and 261 (cross 2) progeny at ~8500 single-nucleotide polymorphisms and construct two dense linkage maps. We measured gray values, representing degrees of melanization, for single-spore colonies growing on Petri dishes by using a novel image-processing approach that enabled high-throughput phenotyping. Because melanin production can be affected by stress, each offspring was grown in two stressful environments and one control environment. We detected six significant QTL in cross 1 and nine in cross 2, with three QTL shared between the crosses. Different QTL were identified in different environments and at different colony ages. By obtaining complete genome sequences for the four parents and analyzing sequence variation in the QTL confidence intervals, we identified 16 candidate genes likely to affect melanization. One of these candidates was &lt;i&gt;PKS1&lt;/i&gt;, a polyketide synthase gene known to play a role in the synthesis of dihydroxynaphthalene melanin. Three candidate quantitative trait nucleotides were identified in &lt;i&gt;PKS1&lt;/i&gt;. Many of the other candidate genes were not previously associated with melanization.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/12/01" />
<meta name="citation_mjid" content="ggg;4/12/2519" />
<meta name="citation_id" content="4/12/2519" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/12/2519" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/12/2519.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/12/2519.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/12/2519.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.015289" />
<meta name="citation_pmid" content="25360032" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2519" />
<meta name="citation_lastpage" content="2533" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Mark H. Lendenmann" />
<meta name="citation_author_institution" content="Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland" />
<meta name="citation_author_email" content="mark.lendenmann@usys.ethz.ch" />
<meta name="citation_author" content="Daniel Croll" />
<meta name="citation_author_institution" content="Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland" />
<meta name="citation_author" content="Ethan L. Stewart" />
<meta name="citation_author_institution" content="Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland" />
<meta name="citation_author" content="Bruce A. McDonald" />
<meta name="citation_author_institution" content="Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Arends;citation_author=P. Prins;citation_author=R. C. Jansen;citation_author=K. W. Broman;citation_title=R/qtl: high-throughput multiple QTL mapping.;citation_volume=26;citation_year=2010;citation_pmid=20966004;citation_doi=10.1093/bioinformatics/btq565" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. C. Brunner;citation_author=S. F. F. Torriani;citation_author=D. Croll;citation_author=E. H. Stukenbrock;citation_author=B. A. McDonald;citation_title=Coevolution and life cycle specialization of plant cell wall degrading enzymes in a hemibiotrophic pathogen.;citation_volume=30;citation_year=2013;citation_pmid=23515261;citation_doi=10.1093/molbev/mst041" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=G. W. Burton;citation_author=E. H. Devane;citation_title=Estimating Heritability in Tall Fescue (Festuca Arundinacea) from Replicated Clonal Material;citation_pages=478-481;citation_volume=45;citation_year=1953;citation_issue=10;citation_doi=10.2134/agronj1953.00021962004500100005x" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Microbiol.;citation_author=M. J. Butler;citation_author=A. W. Day;citation_title=Fungal melanins: a review.;citation_pages=1115-1136;citation_volume=44;citation_year=1998;citation_doi=10.1139/cjm-44-12-1115" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=Y.-E. Choi;citation_author=S. B. Goodwin;citation_title=Gene encoding a c-type cyclin in Mycosphaerella graminicola is involved in aerial mycelium formation, filamentous growth, hyphal swelling, melanin biosynthesis, stress response, and pathogenicity.;citation_pages=469-477;citation_volume=24;citation_year=2011;citation_issue=4;citation_pmid=21171890;citation_doi=10.1094/MPMI-04-10-0090" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=Y.-E. Choi;citation_author=S. B. Goodwin;citation_title=MVE1, encoding the velvet gene product homolog in Mycosphaerella graminicola, is associated with aerial mycelium formation, melanin biosynthesis, hyphal swelling and light signaling.;citation_volume=77;citation_year=2011;citation_pmid=21115702;citation_doi=10.1128/AEM.01830-10" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. K. Christians;citation_author=M. S. Cheema;citation_author=I. A. Vergara;citation_author=C. A. Watt;citation_author=L. J. Pinto;citation_title=Quantitative trait locus (QTL) mapping reveals a role for unstudied genes in Aspergillus virulence.;citation_pages=e19325-e19325;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21559404;citation_doi=10.1371/journal.pone.0019325" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. L. Wang;citation_author=M. Coon;citation_author=N. Tung;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w(1118); iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=A. Cousin;citation_author=R. Mehrabi;citation_author=M. Guilleroux;citation_author=M. Dufresne;citation_author=T. Van der Lee;citation_title=The MAP kinase-encoding gene MgFus3 of the non-appressorium phytopathogen Mycosphaerella graminicola is required for penetration and in vitro pycnidia formation.;citation_pages=269-278;citation_volume=7;citation_year=2006;citation_issue=4;citation_pmid=20507446;citation_doi=10.1111/j.1364-3703.2006.00337.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. Croll;citation_author=M. Zala;citation_author=B. A. McDonald;citation_title=Breakage-fusion-bridge cycles and large insertions contribute to the rapid evolution of accessory chromosomes in a fungal pathogen.;citation_pages=e1003567;citation_volume=9;citation_year=2013;citation_pmid=23785303;citation_doi=10.1371/journal.pgen.1003567" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=N. Eliahu;citation_author=A. Igbaria;citation_author=M. S. Rose;citation_author=B. A. Horwitz;citation_author=S. Lev;citation_title=Melanin biosynthesis in the maize pathogen Cochliobolus heterostrophus depends on two mitogen-activated protein kinases, Chk1 and Mps1, and the transcription factor Cmr1.;citation_volume=6;citation_year=2007;citation_pmid=17237364;citation_doi=10.1128/EC.00264-06" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=P. D. Etter;citation_author=S. Bassham;citation_author=P. A. Hohenlohe;citation_author=E. A. Johnson;citation_author=W. A. Cresko;citation_title=SNP discovery and genotyping for evolutionary genetics using RAD sequencing.;citation_pages=157-178;citation_volume=772;citation_year=2011;citation_pmid=22065437;citation_doi=10.1007/978-1-61779-228-1_9" />
<meta name="citation_reference" content="citation_title=The Septoria Diseases of Wheat: Concepts and Methods of Disease Management;citation_author=Z. Eyal;citation_author=A. L. Scharen;citation_author=J. M. Prescott;citation_author=M. van Ginkel;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. D. Finn;citation_author=A. Bateman;citation_author=J. Clements;citation_author=P. Coggill;citation_author=R. Y. Eberhardt;citation_title=Pfam: the protein families database.;citation_volume=42;citation_year=2014;citation_pmid=24288371;citation_doi=10.1093/nar/gkt1223" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. Flint;citation_author=T. F. C. Mackay;citation_title=Genetic architecture of quantitative traits in mice, flies, and humans;citation_pages=723-733;citation_volume=19;citation_year=2009;citation_issue=5;citation_pmid=19411597;citation_doi=10.1101/gr.086660.108" />
<meta name="citation_reference" content="citation_journal_title=Applied Microbiology and Biotechnology;citation_journal_abbrev=Applied Microbiology and Biotechnology;citation_author=M. Foulongne-Oriol;citation_title=Genetic linkage mapping in fungi: current state, applications, and future trends.;citation_pages=891-904;citation_volume=95;citation_year=2012;citation_issue=4;citation_pmid=22743715;citation_doi=10.1007/s00253-012-4228-4" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. Fridman;citation_author=F. Carrari;citation_author=Y. S. Liu;citation_author=A. R. Fernie;citation_author=D. Zamir;citation_title=Zooming In on a Quantitative Trait for Tomato Yield Using Interspecific Introgressions;citation_pages=1786-1789;citation_volume=305;citation_year=2004;citation_issue=5691;citation_pmid=15375271;citation_doi=10.1126/science.1101666" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=N. Fujihara;citation_author=A. Sakaguchi;citation_author=S. Tanaka;citation_author=S. Fujii;citation_author=G. Tsuji;citation_title=Peroxisome biogenesis factor PEX13 is required for appressorium-mediated plant infection by the anthracnose fungus Colletotrichum orbiculare.;citation_pages=436-445;citation_volume=23;citation_year=2010;citation_issue=4;citation_pmid=20192831;citation_doi=10.1094/MPMI-23-4-0436" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Mapping genes for complex traits in domestic animals and their use in breeding programmes.;citation_pages=381-391;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19448663;citation_doi=10.1038/nrg2575" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. B. Goodwin;citation_author=S. Ben M’Barek;citation_author=B. Dhillon;citation_author=A. H. J. Wittenberg;citation_author=C. F. Crane;citation_title=Finished genome of the fungal wheat pathogen Mycosphaerella graminicola reveals dispensome structure, chromosome plasticity, and stealth pathogenesis.;citation_pages=e1002070-e1002070;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21695235;citation_doi=10.1371/journal.pgen.1002070" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=N. V. Hardwick;citation_author=D. R. Jones;citation_author=J. E. Slough;citation_title=Factors affecting diseases of winter wheat in England and Wales, 1989–98.;citation_pages=453-462;citation_volume=50;citation_year=2001;citation_doi=10.1046/j.1365-3059.2001.00596.x" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=J. M. Henson;citation_author=M. J. Butler;citation_author=A. W. Day;citation_title=THE DARK SIDE OF THE MYCELIUM: Melanins of Phytopathogenic Fungi;citation_pages=447-471;citation_volume=37;citation_year=1999;citation_issue=1;citation_pmid=11701831;citation_doi=10.1146/annurev.phyto.37.1.447" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=J. B. Holland;citation_title=Genetic architecture of complex traits in plants.;citation_pages=156-161;citation_volume=10;citation_year=2007;citation_issue=2;citation_pmid=17291822;citation_doi=10.1016/j.pbi.2007.01.003" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. J. Howard;citation_author=M. A. Ferrari;citation_author=D. H. Roach;citation_author=N. P. Money;citation_title=Penetration of hard substrates by a fungus employing enormous turgor pressures.;citation_pages=11281-11284;citation_volume=88;citation_year=1991;citation_issue=24;citation_pmid=1837147;citation_doi=10.1073/pnas.88.24.11281" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and immunology;citation_journal_abbrev=Microbiol Immunol;citation_author=R. Ikeda;citation_author=T. Sugita;citation_author=E. S. Jacobson;citation_author=T. Shinoda;citation_title=Effects of melanin upon susceptibility of Cryptococcus to antifungals.;citation_pages=271-277;citation_volume=47;citation_year=2003;citation_issue=4;citation_pmid=12801064;citation_doi=10.1111/j.1348-0421.2003.tb03395.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=S. V. S. Ipcho;citation_author=J. K. Hane;citation_author=E. A. Antoni;citation_author=D. Ahren;citation_author=B. Henrissat;citation_title=Transcriptome analysis of Stagonospora nodorum: gene models, effectors, metabolism and pantothenate dispensability.;citation_pages=531-545;citation_volume=13;citation_year=2012;citation_issue=6;citation_pmid=22145589;citation_doi=10.1111/j.1364-3703.2011.00770.x" />
<meta name="citation_reference" content="citation_journal_title=Clinical Microbiology Reviews;citation_journal_abbrev=Clin. Microbiol. Rev.;citation_author=E. S. Jacobson;citation_title=Pathogenic Roles for Fungal Melanins;citation_pages=708-717;citation_volume=13;citation_year=2000;citation_issue=4;citation_pmid=11023965;citation_doi=10.1128/CMR.13.4.708-717.2000" />
<meta name="citation_reference" content="citation_journal_title=Acta biochimica Polonica;citation_journal_abbrev=Acta Biochim Pol;citation_author=J. Karkowska-Kuleta;citation_author=M. Rapala-Kozik;citation_author=A. Kozik;citation_title=Fungi pathogenic to humans: molecular bases of virulence of Candida albicans, Cryptococcus neoformans and Aspergillus fumigatus.;citation_pages=211-224;citation_volume=56;citation_year=2009;citation_issue=2;citation_pmid=19543556" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=G. H. J. Kema;citation_author=E. C. P. Verstappen;citation_author=M. Todorova;citation_author=C. Waalwijk;citation_title=Successful crosses and molecular tetrad and progeny analyses demonstrate heterothallism in Mycosphaerella graminicola.;citation_pages=251-258;citation_volume=30;citation_year=1996;citation_issue=3;citation_pmid=8753655;citation_doi=10.1007/s002940050129" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. H. J. Kema;citation_author=S. B. Goodwin;citation_author=S. Hamza;citation_author=E. C. P. Verstappen;citation_author=J. R. Cavaletto;citation_title=A Combined Amplified Fragment Length Polymorphism and Randomly Amplified Polymorphism DNA Genetic Linkage Map of Mycosphaerella graminicola, the Septoria Tritici Leaf Blotch Pathogen of Wheat;citation_pages=1497-1505;citation_volume=161;citation_year=2002;citation_issue=4;citation_pmid=12196395" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=A. Kimura;citation_author=Y. Takano;citation_author=I. Furusawa;citation_author=T. Okuno;citation_title=Peroxisomal Metabolic Function Is Required for Appressorium-Mediated Plant Infection by Colletotrichum lagenarium;citation_pages=1945-1957;citation_volume=13;citation_year=2001;citation_issue=8;citation_pmid=11487704;citation_doi=10.1105/tpc.13.8.1945" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=J. Kyte;citation_author=R. F. Doolittle;citation_title=A simple method for displaying the hydropathic character of a protein.;citation_pages=105-132;citation_volume=157;citation_year=1982;citation_issue=1;citation_pmid=7108955;citation_doi=10.1016/0022-2836(82)90515-0" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Biochemical pharmacology;citation_journal_abbrev=Biochem Pharmacol;citation_author=B. Larsson;citation_author=H. Tjalve;citation_title=Studies on the mechanism of drug-binding to melanin.;citation_pages=1181-1187;citation_volume=28;citation_year=1979;citation_issue=7;citation_pmid=36092;citation_doi=10.1016/0006-2952(79)90326-5" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=International journal of antimicrobial agents;citation_journal_abbrev=Int J Antimicrob Agents;citation_author=S. J. Liaw;citation_author=Y. L. Lee;citation_author=P. R. Hsueh;citation_title=Multidrug resistance in clinical isolates of Stenotrophomonas maltophilia: roles of integrons, efflux pumps, phosphoglucomutase (SpgM), and melanin and biofilm formation.;citation_pages=126-130;citation_volume=35;citation_year=2010;citation_issue=2;citation_pmid=19926255;citation_doi=10.1016/j.ijantimicag.2009.09.015" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=S. Y. Lin;citation_author=S. Okuda;citation_author=K. Ikeda;citation_author=T. Okuno;citation_author=Y. Takano;citation_title=LAC2 Encoding a Secreted Laccase Is Involved in Appressorial Melanization and Conidial Pigmentation in Colletotrichum orbiculare.;citation_pages=1552-1561;citation_volume=25;citation_year=2012;citation_issue=12;citation_pmid=22934563;citation_doi=10.1094/MPMI-05-12-0131-R" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=X. R. Lin;citation_author=J. C. Huang;citation_author=T. G. Mitchell;citation_author=J. Heitman;citation_title=Virulence attributes and hyphal growth of C. neoformans are quantitative traits and the MAT alpha allele enhances filamentation.;citation_pages=1801-1814;citation_volume=2;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Lohse;citation_author=A. M. Bolger;citation_author=A. Nagel;citation_author=A. R. Fernie;citation_author=J. E. Lunn;citation_title=RobiNA: a user-friendly, integrated software solution for RNA-Seq-based transcriptomics.;citation_volume=40;citation_year=2012;citation_pmid=22684630;citation_doi=10.1093/nar/gks540" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=T. F. C. Mackay;citation_title=THE GENETIC ARCHITECTURE OF QUANTITATIVE TRAITS;citation_pages=303-339;citation_volume=35;citation_year=2001;citation_issue=1;citation_pmid=11700286;citation_doi=10.1146/annurev.genet.35.102401.090633" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Manichaikul;citation_author=J. Dupuis;citation_author=S. Sen;citation_author=K. W. Broman;citation_title=Poor performance of bootstrap confidence intervals for the location of a quantitative trait locus.;citation_volume=174;citation_year=2006;citation_pmid=16783000;citation_doi=10.1534/genetics.106.061549" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=A. J. Mednick;citation_author=J. D. Nosanchuk;citation_author=A. Casadevall;citation_title=Melanization of Cryptococcus neoformans Affects Lung Inflammatory Responses during Cryptococcal Infection;citation_pages=2012-2019;citation_volume=73;citation_year=2005;citation_issue=4;citation_pmid=15784542;citation_doi=10.1128/IAI.73.4.2012-2019.2005" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=R. Mehrabi;citation_author=T. van der Lee;citation_author=C. Waalwijk;citation_author=G. H. J. Kema;citation_title=MgSlt2, a cellular integrity MAP kinase gene of the fungal wheat pathogen Mycosphaerella graminicola, is dispensable for penetration but essential for invasive growth.;citation_pages=389-398;citation_volume=19;citation_year=2006;citation_issue=4;citation_pmid=16610742;citation_doi=10.1094/MPMI-19-0389" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=R. Mehrabi;citation_author=L.-H. Zwiers;citation_author=M. A. de Waard;citation_author=G. H. J. Kema;citation_title=MgHog1 regulates dimorphism and pathogenicity in the fungal wheat pathogen Mycosphaerella graminicola.;citation_pages=1262-1269;citation_volume=19;citation_year=2006;citation_issue=11;citation_pmid=17073308;citation_doi=10.1094/MPMI-19-1262" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=R. Mehrabi;citation_author=S. Ben M’Barek;citation_author=T. A. J. van der Lee;citation_author=C. Waalwijk;citation_author=P. J. G. M. de Wit;citation_title=G alpha and G beta proteins regulate the cyclic AMP pathway that is required for development and pathogenicity of the phytopathogen Mycosphaerella graminicola.;citation_volume=8;citation_year=2009;citation_pmid=19411619;citation_doi=10.1128/EC.00258-08" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=R. Morris-Jones;citation_author=S. Youngchim;citation_author=B. L. Gomez;citation_author=P. Aisen;citation_author=R. J. Hay;citation_title=Synthesis of Melanin-Like Pigments by Sporothrix schenckii In Vitro and during Mammalian Infection;citation_pages=4026-4033;citation_volume=71;citation_year=2003;citation_issue=7;citation_pmid=12819091;citation_doi=10.1128/IAI.71.7.4026-4033.2003" />
<meta name="citation_reference" content="citation_journal_title=Australas. Mycologist;citation_author=P. Ngamskulrungroj;citation_author=W. Meyer;citation_title=Melanin production at 37 degrees C is linked to the high virulent Cryptococcus gattii Vancouver Island outbreak genotype VGIIa.;citation_pages=9-14;citation_volume=28;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Antimicrobial Agents and Chemotherapy;citation_journal_abbrev=Antimicrob. Agents Chemother.;citation_author=J. D. Nosanchuk;citation_author=A. Casadevall;citation_title=Impact of Melanin on Microbial Virulence and Clinical Resistance to Antimicrobial Compounds;citation_pages=3519-3528;citation_volume=50;citation_year=2006;citation_issue=11;citation_pmid=17065617;citation_doi=10.1128/AAC.00545-06" />
<meta name="citation_reference" content="citation_journal_title=Lancet;citation_journal_abbrev=Lancet;citation_author=J. D. Nosanchuk;citation_author=A. L. Rosas;citation_author=S. C. Lee;citation_author=A. Casadevall;citation_title=Melanisation of Cryptococcus neoformans in human brain tissue.;citation_pages=2049-2050;citation_volume=355;citation_year=2000;citation_issue=9220;citation_pmid=10885360;citation_doi=10.1016/S0140-6736(00)02356-4" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=A. O’Driscoll;citation_author=S. Kildea;citation_author=F. Doohan;citation_author=J. Spink;citation_author=E. Mullins;citation_title=The wheat–Septoria conflict: a new front opening up?;citation_pages=602-610;citation_volume=19;citation_year=2014;citation_pmid=24957882;citation_doi=10.1016/j.tplants.2014.04.011" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=D. Parisot;citation_author=M. Dufresne;citation_author=C. Veneault;citation_author=R. Lauge;citation_author=T. Langin;citation_title=clap1, a gene encoding a copper-transporting ATPase involved in the process of infection by the phytopathogenic fungus Colletotrichum lindemuthianum.;citation_pages=139-151;citation_volume=268;citation_year=2002;citation_issue=2;citation_pmid=12395188;citation_doi=10.1007/s00438-002-0744-8" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. A. Schneider;citation_author=W. S. Rasband;citation_author=K. W. Eliceiri;citation_title=NIH Image to ImageJ: 25 years of image analysis.;citation_pages=671-675;citation_volume=9;citation_year=2012;citation_issue=7;citation_pmid=22930834;citation_doi=10.1038/nmeth.2089" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=P. S. Solomon;citation_author=K. C. Tan;citation_author=P. Sanchez;citation_author=R. M. Cooper;citation_author=R. P. Oliver;citation_title=The disruption of a Galpha subunit sheds new light on the pathogenicity of Stagonospora nodorum on wheat.;citation_pages=456-466;citation_volume=17;citation_year=2004;citation_issue=5;citation_pmid=15141949;citation_doi=10.1094/MPMI.2004.17.5.456" />
<meta name="citation_reference" content="citation_journal_title=Mycopathologia;citation_journal_abbrev=Mycopathologia;citation_author=C. P. Taborda;citation_author=M. B. da Silva;citation_author=J. D. Nosanchuk;citation_author=L. R. Travassos;citation_title=Melanin as a virulence factor of Paracoccidioides brasiliensis and other dimorphic pathogenic fungi: a minireview.;citation_pages=331-339;citation_volume=165;citation_year=2008;citation_issue=4-5;citation_pmid=18777637;citation_doi=10.1007/s11046-007-9061-4" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=Y. Takano;citation_author=Y. Kubo;citation_author=K. Shimizu;citation_author=K. Mise;citation_author=T. Okuno;citation_title=Structural analysis of PKS1, a polyketide synthase gene involved in melanin biosynthesis in Colletotrichum lagenarium.;citation_pages=162-167;citation_volume=249;citation_year=1995;citation_issue=2;citation_pmid=7500937;citation_doi=10.1007/BF00290362" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=D. Peterson;citation_author=N. Peterson;citation_author=G. Stecher;citation_author=M. Nei;citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods.;citation_volume=28;citation_year=2011;citation_pmid=21546353;citation_doi=10.1093/molbev/msr121" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=W. R. Taylor;citation_title=The classification of amino acid conservation.;citation_pages=205-218;citation_volume=119;citation_year=1986;citation_issue=2;citation_pmid=3461222;citation_doi=10.1016/S0022-5193(86)80075-3" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=G. Tsuji;citation_author=Y. Kenmochi;citation_author=Y. Takano;citation_author=J. Sweigard;citation_author=L. Farrall;citation_title=Novel fungal transcriptional activators, Cmr1p of Colletotrichum lagenarium and Pig1p of Magnaporthe grisea, contain Cys2His2 zinc finger and Zn(II)2Cys6 binuclear cluster DNA-binding motifs and regulate transcription of melanin biosynthesis genes in a developmentally specific manner.;citation_pages=940-954;citation_volume=38;citation_year=2000;citation_issue=5;citation_pmid=11123670;citation_doi=10.1046/j.1365-2958.2000.02181.x" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=S. Upadhyay;citation_author=G. Torres;citation_author=X. Lin;citation_title=Laccases Involved in 1,8-dihydroxynaphthalene melanin biosynthesis in Aspergillus fumigatus are regulated by developmental factors and copper homeostasis.;citation_volume=12;citation_year=2013;citation_pmid=24123270;citation_doi=10.1128/EC.00217-13" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=C. Waalwijk;citation_author=O. Mendes;citation_author=E. C. P. Verstappen;citation_author=M. A. de Waard;citation_author=G. H. J. Kema;citation_title=Isolation and characterization of the mating-type idiomorphs from the wheat septoria leaf blotch fungus Mycosphaerella graminicola.;citation_pages=277-286;citation_volume=35;citation_year=2002;citation_issue=3;citation_pmid=11929216;citation_doi=10.1006/fgbi.2001.1322" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. H. J. Wittenberg;citation_author=T. A. J. van der Lee;citation_author=S. Ben M’Barek;citation_author=S. B. Ware;citation_author=S. B. Goodwin;citation_title=Meiosis drives extraordinary genome plasticity in the haploid fungal plant pathogen Mycosphaerella graminicola.;citation_pages=e5863-e5863;citation_volume=4;citation_year=2009;citation_issue=6;citation_pmid=19516898;citation_doi=10.1371/journal.pone.0005863" />
<meta name="citation_reference" content="citation_journal_title=Journal of Medical Microbiology;citation_journal_abbrev=J Med Microbiol;citation_author=S. Youngchim;citation_author=R. Morris-Jones;citation_author=R. J. Hay;citation_author=A. J. Hamilton;citation_title=Production of melanin by Aspergillus fumigatus;citation_pages=175-181;citation_volume=53;citation_year=2004;citation_issue=3;citation_pmid=14970241;citation_doi=10.1099/jmm.0.05421-0" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=J. Zhan;citation_author=G. H. J. Kema;citation_author=C. Waalwijk;citation_author=B. A. McDonald;citation_title=Distribution of mating type alleles in the wheat pathogen Mycosphaerella graminicola over spatial scales from lesions to continents.;citation_pages=128-136;citation_volume=36;citation_year=2002;citation_issue=2;citation_pmid=12081466;citation_doi=10.1016/S1087-1845(02)00013-0" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Quantitative Trait Locus Mapping of Melanization in the Plant Pathogenic Fungus Zymoseptoria tritici" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
<meta name="twitter:description" content="Melanin plays an important role in virulence and antimicrobial resistance in several fungal pathogens. The wheat pathogen Zymoseptoria tritici is important worldwide, but little is known about the genetic architecture of pathogenicity, including the production of melanin. Because melanin production can exhibit complex inheritance, we used quantitative trait locus (QTL) mapping in two crosses to identify the underlying genes. Restriction site−associated DNA sequencing was used to genotype 263 (cross 1) and 261 (cross 2) progeny at ~8500 single-nucleotide polymorphisms and construct two dense linkage maps. We measured gray values, representing degrees of melanization, for single-spore colonies growing on Petri dishes by using a novel image-processing approach that enabled high-throughput phenotyping. Because melanin production can be affected by stress, each offspring was grown in two stressful environments and one control environment. We detected six significant QTL in cross 1 and nine in cross 2, with three QTL shared between the crosses. Different QTL were identified in different environments and at different colony ages. By obtaining complete genome sequences for the four parents and analyzing sequence variation in the QTL confidence intervals, we identified 16 candidate genes likely to affect melanization. One of these candidates was PKS1 , a polyketide synthase gene known to play a role in the synthesis of dihydroxynaphthalene melanin. Three candidate quantitative trait nucleotides were identified in PKS1 . Many of the other candidate genes were not previously associated with melanization." />
<meta name="og-title" property="og:title" content="Quantitative Trait Locus Mapping of Melanization in the Plant Pathogenic Fungus Zymoseptoria tritici" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/12/2519" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Melanin plays an important role in virulence and antimicrobial resistance in several fungal pathogens. The wheat pathogen Zymoseptoria tritici is important worldwide, but little is known about the genetic architecture of pathogenicity, including the production of melanin. Because melanin production can exhibit complex inheritance, we used quantitative trait locus (QTL) mapping in two crosses to identify the underlying genes. Restriction site−associated DNA sequencing was used to genotype 263 (cross 1) and 261 (cross 2) progeny at ~8500 single-nucleotide polymorphisms and construct two dense linkage maps. We measured gray values, representing degrees of melanization, for single-spore colonies growing on Petri dishes by using a novel image-processing approach that enabled high-throughput phenotyping. Because melanin production can be affected by stress, each offspring was grown in two stressful environments and one control environment. We detected six significant QTL in cross 1 and nine in cross 2, with three QTL shared between the crosses. Different QTL were identified in different environments and at different colony ages. By obtaining complete genome sequences for the four parents and analyzing sequence variation in the QTL confidence intervals, we identified 16 candidate genes likely to affect melanization. One of these candidates was PKS1 , a polyketide synthase gene known to play a role in the synthesis of dihydroxynaphthalene melanin. Three candidate quantitative trait nucleotides were identified in PKS1 . Many of the other candidate genes were not previously associated with melanization." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
    <title>Quantitative Trait Locus Mapping of Melanization in the Plant Pathogenic Fungus Zymoseptoria tritici | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"tO_OIiPREpOnaMJt1v0OKb4JuNHgrSSsPxE4wiREkbo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459743","apath":"\/ggg\/4\/12\/2519.atom","pisa":"ggg;4\/12\/2519","ac":{"\/ggg\/4\/12\/2519.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/12\/2519.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/12\/2519"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/12\/2519"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/12\/2519":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022k3bApHKCWI9pM5\\\\\\\/ve7kC0P4IkxAv8fcyP81x0xfrQjfmucT3d5aenVN+dMm5exZfJgOFWNZ\\\\\\\/zkjRDFlCIeB8ZYC85QUmillFzQL6Qbu\\\\\\\/iBl\\\\\\\/1lDCuPdZ\\\\\\\/Vj0\\\\\\\/65IvgK9KgN3Ec9\\\\\\\/RitYG9aqW5dX+ZsBM+KCdVQKbgl9wMf+HiMPOJJof+sH0gqT+DIvKeOVYEodKGa4FOGGpwyrelvXtKDjzzR4A2ilR6FJiRkTieyMwgryEGpBVkZ3LmHj0GpzgqHHDPxtS6wV47kfUZdMwm7MSyGDKAKbaMip2cbC4\\\\\\\/7I+z8vDDruMj6DjfAL16FnK+NL9SS3TL3rCrsrsRb9iPLPExUZdbJiJ4ai5XS1mt5e+D0\\\\\\\/M0Uk6g9t1k3Hetel8IjBLVgqi8AX3Ak4MilcB7JQqAhI+V8XmN0qZFW\\\\\\\/GFPZL\\\\\\\/ZYnZzV6FPAhuvihYUHl8dYKkn+AluI8JEzJpK8CJ9l9MsOl\\\\\\\/\\\\\\\/qGd+\\\\\\\/UjeuF68XAUUhCNYJpBtoGanNFvLc0bdrnC+LPoHNiB2sicnJCmC2wy8avARo1w53ibvgBAPi6x20ebtGuyeX0dCwCzxAS3NM1tu1Zd8VNQY3hgu7L95WPi8vEW542MENhhtaV1qBs0FeCA+rceAwUTqu2sHShywxjSOdVFoO+Gtc6FzFMlz0qlrqwf2wV9UAkqChQ2sJn0Q9NJwaIf4rgDOP53gLhcufqAHeLdSDOEWbGdpAgWJig8LABe5bM3Pcb5RUzXMdlhh3ws1cl7InHK4li72TLDQzRGrDv\\\\\\\/kztELEHViqAcZ7hzb8tsh8R3xAYSp\\\\\\\/U3HHwOv86SoGnnRswtUsCUozN8RTOYYt2Foie+9Ch2vKiO\\\\\\\/oQHuTHKdgzaDTEhlOY7S8VDisvbqqDNhSmMUotgg9reZ4drR1KBa7UCJSWI0JfUAyU\\\\\\\/+lUr+d0cOdKqYRYcojgxTbNC1ZSG1ATWQqG+ao89L7vd9EndNXo4B2pqfwQhZQGmsU5LOc4uFykH5rdb7ceiP3JlI9v2vaKAJG6ol2v7WOFMH247w1vD8m\\\\\\\/wuaVP8Elno5pTxTa3Nt2UL7X7WQdDPMQ4ve+wyhBGLZ8Oap\\\\\\\/HkUyRfi2EtldZwBUTCSrt+FddFmdH0QX7uhRPx2QuG9p\\\\\\\/xM1e9UlELxGE3mdrOnn6JZ797BAxkjBf\\\\\\\/vdHGRsbrI0zdsQVT0AFRjx4+NajAE5+0dllW4kgyhfZ9\\\\\\\/GLG7rnVNoKrICIqvxaR7X\\\\\\\/DN6B\\\\\\\/NINS8o2ZVCoszYmOt71mGpQEetYX46aN9DgMtKmn+\\\u0022,\\\u0022iv\\\u0022:\\\u0022doG4tfjOfta28fJrNoakfw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022aa9f26014394861f2910b19a30fed6bf410913be5213dd15777902cdc8842195\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xC7w0Nyct4LVN6diCal8C9Ljccj3N4N0zUAtzePBkr0xi2y6QtOb0gLpeAUD3o6uirVTcA8n\\\\\\\/zihspA0rP3Pt4odaVfi\\\\\\\/MMpzqmYehPGEon6ZQkHsFkFdKBN8FXN83Nz0dIqmqh4k2MIoU01O4y2F0G+yKP6yWqb9z9Iq0125s5ufBDBCLeCq\\\\\\\/OOwwH6kU9mNmrtAmKIutG0a7kNZB0HPNIRTVgqyulBJAvr3oL2Wtxsite+\\\\\\\/sBN5lbDqy+5a3LzohEDiTu0r7clrM1MG4bU2uYEUPKH5WPTm4AH4+xpWJJJLV6YotklaoSSjrrTA02d2XQiuQ0bCPj1sJmVCBYDArKUiV3fDdUabsl6P+KXr0ag64iqY4JSAFRbrNVUOyu5OxT+HbNpYMplhz1Qs\\\\\\\/a4c1iZ1gU4vjQ9n7vvELgIxRucUAOKnyKXFfp2ep+DSfjqbpOKzunGxoBTEN2FF\\\\\\\/bLkkUK0+9od\\\\\\\/Ib8ygJkf1hBqAGsT\\\\\\\/dnAdFC7UfdQJSnxLe8wpwj6c1frDjbNn0EjN4T3NxRvG7XgrAxlgM3DKhL1EBjW+vcCiHcq55ePIUEndk8UzVeVZ8CsVs9t7ZB1qate6mw4VppUyfFGgocLNQV69U8ftdSi6r25r7mg3q6Ui2Za2lGSttjMvp1OTk9mauu3TOK8MKTp2tYHZyHY3i9YBR2Z8HyJRFI6NdH0dfFLtQEa\\\\\\\/qHcp5XQSE7\\\\\\\/lVVqFUWL90CGgsI80YNvza+Hkv9\\\\\\\/Xx+g\\\\\\\/91+lJ6h77DtxbRc\\\\\\\/wUoTbroBJmZSPcqidpfHOQpRVfHDsHvRVvzhaF5+BQlFaQIwVQCxx\\\\\\\/a1WxhHSRHp2az9O6azuQzpeHZDgCHI8xIduqV28epbdY0yDXbvCqk\\\\\\\/Cvk1ShoQJKCyv9ekb4I2e86KFEKkwynLcNONbQShxwZ29C2GNgB8cKXLqhmUKk+YQvSnPrJDPEdcqwCAlhlw3gsknyZMDOcnfROwFUbUqGYPnm6lhw4Z9KWT8nP7wI4WYFyZWpvCxPDo7i8zfI+kuF3gw16lGCIWxnFndu8yBoDgVKldMfs2IBVSSpX31IEjWYv7hpPYQBtNNIEJsPQoA5+1XAZPhchXHltN87KSo3TIotdxeHLPnm4F5huwp6ExYAoy4uGnLGZIqAC5+lxnkDL2JgxBC5Fk+r9IRlQhTNAP3QtGxegb5XawbNpkgVPL0o9WH5PZIcLt04pCD0YSE7e0GEvALFD80DiUQtHjbV7osJ9cueQkveW+gxGGb+vL1G2gZ7rc5AXuJfe2CnMhTGuyy2DeMwrq3pUnBzOo\\\\\\\/86se4ID1ApGZCXboAx8tUFMvTpyLrgTlt+YSsJgwaylxz9ytQYnSwK00RtBFvh2lOZgCS\\\\\\\/9eKUZYIEbW2rMcc12lxl\\\\\\\/+2RPAsHkWAlkKhe6jxXpvArbVreplxW6dT1LZvm3E64iHbxkgI1tFi2Yy6IRv0N0upz7C6QYsPw1TIRyAWcus2dSVUvd3yIUKm454ROahtWv92hSQbJ1pvM5SqrqtSx5\\\\\\\/e2SPeOUnVgz5mK2Uf5xO93kLR7EN4qIRKFV2aX6n6yl\\\\\\\/NqLQzoaIXwy54C7sbXOLHZuGtFBEUlbhEDvB\\\\\\\/k0t1+YtB64JypuVPgDVQmj2mXfxRRPffjEBKymA8yfe0JZ8dkcSAdHSUoYR3MwpOL5KlW4pomz\\\\\\\/CpnVkqgLY\\\\\\\/KVTm0qS7c\\\\\\\/zbm7pQSWoLmZnXtbtd1W\\\\\\\/65RcKXlsUh9glj8RO1gbrrn8vIWHYhk5X61qL3Hyi2T4+KSnL1457W3yKJ8I+eYgmsgEtR8x8S2ZTnnEY1n7ubiPM9syCPGR9t7nLu+HlIj3XTynNXF+liW+dus80wqCqfuN4MEbED1jT5szAKsKNNOh2OJ9hl6ZRhA+4hzwcPM0a1fWMLVZiqB66mkZiatCMd2LRfF3JBBcMv6iGpB4BHaX3amz7RLh1vl4SCM8gIuZcd59vnHHxu5hWPPmoQ\\\\\\\/kSwpvA33CNHkMVDclDNCQIJxginfrnUpcOYenDPjsl9PP5HpHO0fdeE4vqhDis+HRrocXwxDgPwEKYRlxAmteEscjSQLv2jEzw4m0SCte5T8BhcsWHQbENtGGRp8NO0GbWOXKMHgvQ5hjHV9w2Cpu2xKGxe4Sxp6poR9e7GbLnvAIx6bZGaVyT8KYepaVdtKKGD7gSMzsYRsX95QMs2WZ88V2UxCtDCtCAnrXdVIbUACP18XxPe24Mn8FkptjRDinhwWQWjzf405IqAzRoJS+qnAl3TPQKRYitEJESLiJO64k+ULl2j3aONeZyiepBoJHmx+2ysX6WWBAXl2Eoqzr5TE7\\\\\\\/IDg7NTszMaamt3EvHZeunoQHT8BUQ+jf6dccTpnq03idiPorjCAYJXIZSvldmtN3rI8n4rJdTBtan\\\\\\\/URrFqrAI3ljgtnpe+MAKSwa84DASV26XxNYqaAkeYgqcMbeiB5vAy321ayoV9dEum5NFJRqhEQTpyv02347LT0a4TjziRwJsJrEcPNdlefDfAuOkNtX5khSkHaHtQrjeWUBopZ5X4\\\\\\\/YL4rsRE\\\\\\\/Be1xEeyu4Uouxb1uUje\\\\\\\/6mtnPUYLt7hmZSN6lEYhTBFw9kZHjA6UTC5lcfUnHN299oXfqFB59iH2doO9HhxGWzVlF3U0c1jtRx8YSb44IQIgFoJ4fo0hc8QS2yZyQxNYr0rstvJiE\\\\\\\/MhgarNJ1pRWDmKFi7\\\\\\\/lEtSqQQcV2sftCdByf5P2Y2xPFPBMo0lZTy+Jolw4F1dmb6MQBEkdzUbkPcV51MZI8AVf5pNNMnH+RZPI6zXXb8+iFR+mj+e8RDVRNlrEff7Cs6VQnJ8Pfl90FdLuGyIU7+RmUS6HLgIdNlaBNZWUARdZAJorlCsL1T2872mHJTIA5W6fURguRTHiTBAjeHqipFlaE5sdnzTjnjFm9yM2QUnOC7xZFw8LcGMT80addXdTEiYDWEFrh3VhTPikAdH9pGXkf1rruTKvTRp4G2BDrfIW0SWzS27DrFu4H6CbX8XEXvBT\\\\\\\/K0mis\\\\\\\/6S6WXeaUEpBm6eO7iRZgymYZDJXefBaUdF7YLR3jnZsXGejms6FbKVF+Q8Moo6XFyIMG2zrpd7EkNReqvOR0c5wTw5OjGFuv0VQBM8nZBbwSukYPFNk2gNr1vaNBdSr4nWynRajbEGkrj7507Rs6oWGLjeCaMDIYdCsA\\\\\\\/+C4otMQFaeQEUIrErf3bz2\\\\\\\/HyzC10eCgrTd4pz5GOI0lOuOC98lBlS6hBpSM+nV83oHmE3evIB1IBnfdWbVcjMVL\\\\\\\/V7hyvIxz2eklpnFQk6lb7zC2D7nyzsKQNDcZed\\\\\\\/7ssxOwrqchUcUDAHw7i5uqEhoriIGjB2UDHXDJmPvftrpwnJJll0NXBS88+GpPWaphFrz4zhHqybQMmAMREB+M7L51yJ8cN8NwFRrazUQlK3\\\\\\\/T6blqsDTjuIsjSfUCxHhdha871+jjtAvu4gP0QnDsA3j0SlsMZfNMtdUcUlQ21wNlDwjdX\\\\\\\/hAmem1oS+gBDuBgTbun8SUCAsZabJk9wvW7WqRrXzZ3keRMGi3+tCg2jHSJz+FfcWhbIczNPXeFQJ\\\\\\\/\\\\\\\/ZoHmE+u60dYnouB9XhnXj92PGG8RtqATOMYgWVsobuFXa5KsRX4rd02lm6iv8um3wJL2Dyhu2mqxhVji\\\\\\\/WncXCZX+pmlSiJaHnCqFnwjo+G9BGn2KYpWNsIGQM1dCOZusqzgCAMRJFnWFx\\\\\\\/oWxJj+ng1zy2hA5QPuVnsssbXLUcJxx16h17D05A1FDuzl+G+GvVI22YWNCy\\\\\\\/OcVmKvsuM8E+n8\\\\\\\/yZjDN0AtoA0LbTiGhQjfYtmjrY6MCj9Nl2fZnAqUhTtvzl5NhkQQqvq3FRvWQvLYJyXxMKnDtX1WieJG9ADq5R8+CjgmIbRBqHCFvglzPX4aLcvRg9asePRaQA8BrU0jgI1pq8fSc6ulR2httOkrTlOCW4FwUXZ4XCyNmy1WmEsvZ8GYIcYKtDRyZtmMz\\\\\\\/2CRTHJeizu1T\\\\\\\/XPNkFBjV6TA8zqST10yxDPN\\\\\\\/JRsi6XhXf7nMuMLIjumCZnPXhHIo8qMIHQGrpkOduGGvzGvRqbldFV\\\\\\\/fs1oTOjOqU15Ehw0sAx9WakAmqq2FmBKAKdThbE2NkWXp3RdAdgjyDazn6zrM5tP6JQbq0iBDWWsFB9rHWIjn35t1VwJSz8TJCXnxpbSt\\\\\\\/sjSMngFfI3WM4vMFqtLRAkUxux3v+2leIjopmwiNqOZu3RRNLR\\\\\\\/oD4K7nXSITjz5YXc+UL8IHjkQTJkoiKrV8xn\\\\\\\/b6cxiU4o62AEINWuXtTYh8rcCWqkAijBvGjNEb\\\\\\\/wyvR2gPcIw+5I6zp\\\\\\\/8UZGzaGelSufzXerAdwPb2cnM4UbM8UdMyqlmzZw5Kc6NApj5WCUb4\\\\\\\/lqQ8cSBJfqEARocZmtW+yLzATcuc1ZkYp\\\\\\\/Y3WpywF2iiLogx1WvXsNcTKW0d2XeyLTWkOzdyl6M7ZSTXxN53EG6GiCEIrT4L6xlAO3qWyo+3Km6UnyCPvGCSklMmZULbyBLQajfjT8DlqGZM1x5DagFQC7scAoLU7Pk4GIYpi9jV9cSxhtmUPj\\\\\\\/OW\\\\\\\/S6KIi1zFmRV6IvTcJdcHMagTw4zUtsLFMrpA\\\\\\\/Wig0+SVizELBqpvDXEWorA4Bfx6lu0oBFYVbHjFgHnt\\\\\\\/SBBvayzQQG0uzxjbTUI9FpM\\\\\\\/S0gqyjfJKu4tKueTdeRY9kfyG8TR4\\\\\\\/BTc+ji09sScAdaDy1iBDcUZ0o0pmkmjN7Oc29RuAeLB1TJx2Vvo31Ss0NPT7OzJz+sgcYSF3qDIKAxBazMha6jvfoeecGq9lH2YuDS9xzwhgra5owf67py\\\\\\\/tAp3zWzHubTkRo4nZZgz0q9t4VFNx\\\\\\\/ciIgzjlQbzxjHOk\\\\\\\/nmSz5BmFlpXjNwr\\\\\\\/1BCB7CtXAJtWsByEgsnR\\\\\\\/M01pxYmyGfDeraueBZ4QKIwuBygwciu52UDCpR3ZbRKlBtezxTs9QOImgh\\\\\\\/MVQt1jIPz4PrAp7ryZP9GFxFl\\\\\\\/GKAzQmosWa24TN4LSA4woH3CjNgmxJGmSo\\\\\\\/Tado98EjNFdpQG9u50zAsgbxF14C\\\\\\\/r+QnX4MMc2+jxm+mjPxCZqwOkMKrGslRmFA3e9Fdql53BJph+uhfDCblwCF9TKBM\\\\\\\/B7irwDDvkdegqHk5\\\\\\\/iSjdU4ZIuLakz4z60onSKJd67CX9Ziw9YD4lDCqd7TPCKxKCmS8FpmqPTwf4bVMd+mDLXj1DX9nxDXSn4ZKAEPfFm+zIfxlCiy7rToB4\\\\\\\/ZE5t5Ek2nqhpEPxGJ5TbMUD8hAAZvGiNxYIIOXB5uqqVshZ9p6eO5QMy8nL9ij4i+LWAdXrmzridMaJaxA7iIWtrF11f2Sab+9wZhIL\\\\\\\/HTAZu5KKfAwn+xghzWLPVHDBHD8X21Ha9sJBXhc5kCyX6JmTbQFid6JPBbsv6wClSpbzd4oPBjocjnNGfDwwP21LEjBQ2V3yMTAQUyuv8pMqgxhc+3HqPkmFLHAl7JSgi2Vq8vOHtDx4Lgk3WQXT6UK2OqvLzGw549vEuXno4bA3U4azipj54h12fD025vYmrdl5zRv190H\\\\\\\/KDw1WT5PygYR3tFVma4c85vujybIJfSs12pyZEuLXp2G4BI\\\\\\\/fmALwy1dDvaU4UJyhQFGWvqBUOjl5hgLNJdhZp1Oup2uckdkV4r+tH3H1+JbtcmZs3UrF1kH\\\\\\\/Nfy95944LNt5pUUB3+k0Pl2xmzWpZEQYvJogFEDkOqLwt\\\\\\\/YlX7IxCQZTAZTSQ3XMW3+16hbA6JLKgUM4bIBpJytRrEuLMTPeHw5wGZIjOSKa3xfTsS1eUUqO0HJbOpHdtZ7sWAf8mMT57Les3zw0qZmxtAH70PpSyPAflYe7L72qRTtN0PFC\\\\\\\/YRxTqyX4toCnatleUvnCFpq0Tg9PbaG9ewGo7bMutHSVq\\\\\\\/N+SRzb6rXZKlM+4REYaXz50jo0oUYKS9tVisF4LT2TtC6pqrV63iFHhZ4NQyJccmGyO1Kn1zmhxvzEgLT88pAeRZFowtPokNxQXUq1nC214J\\\\\\\/b5Bk\\\\\\\/Db38oSAWZ7c7p0VvnkSA+Z3ocQYM6S7ZEC90RIkIQBeG1\\\\\\\/5TcOLA8pjUNCqguxlyxHk3V6T0GKPpJwi0Y12VH544cMq\\\\\\\/vPsmJuWALzRoBA+xsg33QrKy3anQiWdPytTRnV\\\\\\\/gBVSSHCylID+YQhyhlubMknHnNQzbdpvBXVV73e28OSeDXQzKsvtOAa2f8rd3ruiqWA4jfelS8265BUAc0MkTW5GU2kIfavgPg9Uo0g0vqsYQyzeqnt2NNMPNdWCuAA\\\\\\\/1xq4gLvAW5q6psKD\\\\\\\/LEFV1hToEf3BU\\\\\\\/HgxExmk6EeoNwmpZOpbM5YpoztntOSnooSArAwHG4wyewRakfT8JzUndkgCDY2dNiefxy4U+KhtgaIX\\\\\\\/w24V5kmWjC4w4NQ73vPof0mRYWLmvkYKcRZz6sJ3Eix8ZJZEBWW5GWd4eMtwxnN3HKax9co1pwNqvHCLuQZvTjz+m1swCx5Jcp733jVzUwuuAV0jaCeJT3jrVVLrVf2bp5bFObDpvJ5U5T66IQgL48hA\\\\\\\/YQAdR7qBjt+5d7fIH4o08ZOtgS8xCMfO7\\\\\\\/cxRK\\\\\\\/iofWp\\\\\\\/KyRK+fHtn2uL2rwAXeu74r63uKNiMNfJmxQE5U3Wa3XHMbKoAfKcJvX1y3xkLMhTQ2BLuCr3gVzOYOPR09rLO54\\\\\\\/M\\\\\\\/KVAPEb9jetTFdpTEtr3T6km\\\\\\\/T1RKLswNWnHxt3Gp7X+jVByKU4Pw+apMJs\\\\\\\/H4W\\\\\\\/pYdhJU\\\\\\\/QulHxDO2bFgdh2N4ER1SEQnC6vsmwTzssG+RPnKRlnQxK8aWN6KTy0OQ5EwCx2yKABD7g5CmdlWMki+pwkIL1U90P2lCK\\\\\\\/f1wSffAmq3yY\\\\\\\/Vus2K55j62C0d2WwvA6zLdESjimz0D\\\\\\\/550Gz1RtPNVDNmY8t7AR+RixO9k2s30qOhaKnqMxcZmO4VBS13WuB9rZdZFtr0A5KyEnkFx3CyyymBHIOrz5wXIdonByT4YXGuxSt5hYzzDqx3mjN5nCH2gFM+fSAfMHWRjs9npJUT25myguVhlLEouRZYBiifXrfMIemX2D0XSAP1Qds5fb7+\\\\\\\/EVklvL5Iy5f3OFGHTH4jhiM6M5AqOLKcG5SD6Nt1nNaI\\\\\\\/UokKWYvP92viRgMQX2Ku4kXXPz7t9G53lMVJXu\\\\\\\/vXRJcB6O7BgGAH7AbZzo\\\\\\\/RMOIDQPVBEhjHBImAL4MHrX0u9tAaAfzyAbzR9TFse2QlPonFPwNLJZCAiZso+CusnITvx3TyOKOTpksUGiz3inWEGoLMZ85WDEO+NS+W6CG3fTZRGjtIWbur18ccbfa\\\\\\\/Xp3XZYUMjlGjekN0VBbLwTOeHh6FA\\\\\\\/RpWTuZR3KI2R21tZCLXab43fO+D\\\\\\\/zXFtd+wgQcVfXJK\\\\\\\/Tfw8UMAOSMgBfe2MJzPCUX36kunWiydTNz4HyjSvNhhCcGTIVGXLIGJaZuZlOzDOxPrGKdwXN4r5+tjqXQcs01q6Q6y+scU00\\\\\\\/u7y1qHNeL2DAwJh8MrBggzeAKMQeF8ANNVAoIUbiOHGIiCkD\\\\\\\/gC+AUixaUd\\\\\\\/h6IDbTrXxE5QHUISNipu2HB50cMghwjt5yXcD3lX51UuZIZva1YPbmgDT047lp+Z9ruv+pMfVYDlufDg1YSm49jM4Q9yd5H2XKuFahMp9S52TDx6yVCPpRjR1davabhQVzykFoMKcOZjj7ikaX5L+N8aReuOKT0C2rR0zT8Kt58O07+mMXnagvgtPop89\\\\\\\/wucriqZI3rF3tzp6DYlafglEMHReHQ4zzS3PHS1k\\\\\\\/wj7Ht+NTdMVp8d3d4LVYrr0zhz4ToBR+khXuoAIzLiMEbb8TyhTZBYJEPnEuYCS\\\\\\\/Y1tbvhfjmIMjnQu\\\\\\\/JAJdx8jlhPqSd1v6hNrJw0I3hkeOTugkF8z7LpEjxofSVwjg76\\\\\\\/k2wo0HMNYL8VsoFmSasFDi8LJCcPiaVbWDBJrb662W8JKk1ildIwb\\\\\\\/+ssovKJOY1Xfc3mHLlTtvUA5o94EpxKH5s2nzoJMrr1vnhxECinQ\\\\\\\/kcOR7q3kki\\\\\\\/41ATvehUzJwE4aRacvSLMXfLL9Bue0C6BUJ39ExN4Rl9Hf\\\\\\\/3EqcjYCckIjhjypyT5WtIsWj19eaebqT9dPlHNSlJ4941iy3r9x9D4YgV0KG+ZR35k6hoKVzDVUu2eAXVCB3SX5VqqkTKbNiKzyzORhmOzbaI3zJ2gc16gP9hFEOqP+wCq5XLB0\\\\\\\/v4RlAZ06CsPUfBUFH0PuOWhbg3UT5Ba0prR10tRK8kpIrxHVIZm5vJqyz3JqkGFGnOL187ddr0t8jVBAfMO41ZUKIYstTH5R6WSAVnCUo\\\\\\\/j5bixtdERFKnQi79RzS46UkfcuiHpspAy5jWV3nRx+6Pxg\\\\\\\/gXqvdeyk85r1T7xVKkSMzT\\\\\\\/Q+D7H3ZRaEoeXD3Xp4IsFSoj4NPz5fJQZGJxZmWXhmZbFKDhXkF0cxZzlOMGbXvvAVU1tj0fi7v7FQOD\\\\\\\/3ocrLbZTf78lqr4ncpkal01YvIh9Xe39uC0DVgHc2L0yXSZg6L6ds8l+pLX484dBxQgtnS4TJ4z2Vl9oIxSX+FsBArCcS6\\\\\\\/O6hqdh3TCF\\\\\\\/N0lSGaaUnXc82Xb3\\\\\\\/ZgQHLrXUV\\\\\\\/xP3j5k7ejsvJ9A4pd5wL\\\\\\\/z+ULmq7syUaRaVEL5On00PpDpcE3w6SYk1HpAuGwAceFuKKqg39za0wlsVOQgmzvrre7vz6QMwIYTMJFpuf4AKY6dTnAlsp1ORcBZHEeKiXtw\\\\\\\/ZXTuDixMcy7M31WucR8HqoEab8U\\\\\\\/Ou+Bt6AEvAwHyJQNugLKhSxoiUb5uial8cdEQ0NlA15y6dMSu8RjUTvqCOGQUgS0JlKTeQBbWmjfxS8YCRCQ6PsOMOvOsSAP2BMZrmrn21nn89ctU4qEjiWu2b6zzCs0ojiQAI88c7poK5b+vgNbK4oZSdwgTGBeSJs\\\\\\\/TXEnoft7xX3Gm40l8RpAgWJMCnMOBbAaO9BjooxDGnGPGjnF8xkEeJSNMRFAltbvTGFcRzCgv93lUpoqHNqXZMaBbhnwy4pAUAL1X13KstIJbhj\\\\\\\/1vdpRasbSL6fNlt76VJq559OXtvINoLgxY5lOfAtq+5cqSH77Za7h3Xg1OAWbx4Osapejtac5OpboZdcVKf+2y6hO9ISLkvjyjw3mHXkCzMIJ90q6eyRwmF9e4p+f3XZNKObiljgcDPSoJwNmTl5\\\\\\\/RUOtxweEH2SKrde8r2FOxYUBflE6FpruSKczfoVmGGy5GIGR6xI+\\\\\\\/+8fL1caO\\\\\\\/Yh3AH7b0DctC+Fn9DTmPTbtEuw0hNCNKzEKWKkQfD5Rsb+2jaFJ7kxPLeWqq7KBvhIt9udah\\\\\\\/8cuQwX4MWLUr\\\\\\\/9QZLfmIvf1wtcUd1mjCQbCvPLhsgDOwpa0oplEiI7lgMDrd0\\\\\\\/LLubaXtbdaTFND9Od3lrMfAEVjNx\\\\\\\/WAYgNKujzT5QRUXGbANOBSlF2QsSOFgm4cot9pRmbIOGbthiUGjIeTEmEqhKvmwriUZpLypFZ3IMrH4L4jjTvzcKzfyiWpRiIYqQ\\\\\\\/KJ3Lb97ayIBKADvkzfM6mcYuwdaSGfaDnOs0qsIUjhnefCcG1X2AKHA+7imXx4Q6NBc3H+F9KyCPw2uxBezxiRZMD3Y8bEoJS5vbBxWALPll4HaI7q9ynFSXBfIkiB7KFX7D5PghqnLLAHOhz6h0xerZx0Qqz+GVtyXimyZePlYV+DaghU\\\\\\\/Mgl0QReRP622dRWTCmUsTjmuQxTIhi5s3XM6uC0fBIe0eMUZ\\\\\\\/keOnVED7Lw+Nr+GU8ZN5H5NgDRUtWFvxHIZUIQ0Se+Mdv\\\\\\\/pnPqgSFkV9bNf0GRaW\\\\\\\/Q23grt2KGclXGwqmxqfMeYW3m\\\\\\\/jWVDKbt3Hyfexz8NpMovgR3ieG5caDyS6WeKnTTXgaGgwUQqMgzbq+cBq1ju0oRPDpNEMEHRYNpPk2kuVHw6wbr8EetnJQGRaDyFAtpFyjrmIrg\\\\\\\/6qQ8lrPQ3UpQnT9HAXZ8JTgbGhlYgWFk8zVKwc4uEsULyVIIc\\\\\\\/Xu33EmK8\\\\\\\/yY1WqMkMeXCxMsbnMVLXhnGVpSpgbo7fwZEYfW\\\\\\\/VMDQffBL9B\\\\\\\/IHd5drQzr960JDNujs61RoIQA0ESFeqQbZl13FdSy2kHrNlcJduds7Rhd9VqswvzVA1p7BXJItFRrYQHa3aV04DsvBbCmqtUG1JtPne2A7GKAgGILJluc8KjWSUesoqa5N\\\\\\\/WKXMq3ogz0EovgdmW\\\\\\\/XfqMtF5TC+7AdO+fd4\\\\\\\/zbdrfAsKSuQW79+gt0Jvp09hGFoZB78HWI+onCBXrfstYTJHnhExXtzpKs1sM9iukE8Bdl3JYyAulA6KmqzV844BCLXc4SbxhpyBZDCLknNYAg6ytt3uNiv0lL\\\\\\\/P4iLkryYd\\\\\\\/i67mOwUWN2s\\\\\\\/70cAuTAYQP3p9eH7rSg5rUCuAVsYImqCem2LracGNnp1GMNefbfWgq18fpyg0WO1vomPlEwAXntGLe+agUb1DK+lqZOhpRt2UDrvsk+dH86fmsoXYwmAYLmeKCNQL0S0Sjrhr9YoCp2iAfMbL\\\\\\\/uwnzSVsSHsXY\\\\\\\/\\\\\\\/zCYs6n\\\\\\\/HirkhzAte2XRZV+r\\\\\\\/Mj58pMxztPcYK6hCZWz5DMFz3DDDec9JqclZX3YgZ5YdQdSpwxe7Ns+8HnKnnk9Jqp6d+XbMJK8L\\\\\\\/E09SLVoeMcVNlcq7T4s\\\\\\\/zhO8TVEwhi\\\\\\\/b6IHslPToLZYbcbErVrvSmSxDR5ILgbSdkAUo9pgvrQS\\\\\\\/+MKfOfs82CdTdfrr318QwvR5N1IBK3MIHLiMAyzCXIV3i7ghg6P6FDpwWjkgo+d5ftEA4v+K\\\\\\\/AFhL8618plxmCqXCmrb078EPrGpEiJB24kohUCjMPNRokjfaA9M6UvvZBJy7gA5SLP\\\\\\\/FzfMwO+cWLeYNzpWjmDOUij+S6CdhdUCefIhrO2GcREGNfDzjvAUiVHboAQbpeFf8lSjRzuVLdUhqczs5cBwtME8gifCP5RNodPS1ouAopvNalN\\\\\\\/Rmzk\\\\\\\/6huVcAM9SZba7omupGmWvZWZ8zWKe3Xlye2GoW\\\\\\\/JjmFcltVc1HV2EtrN\\\\\\\/Gk8FI9v2AJ4IBj2Kv4wfjUjT6wdCKQXISjzkOljaHW80nH\\\\\\\/fAy+bdJLRToQX4+Q6U49LzZwJZU24iCMkKKAre6bUu9hZ3woVLK7vJCqNHV4q7BU4dAaAG0qZVHJAy3zA2ESAs+Pak7pASFYHPAXMVILmNg7cF72vRc56f5xPPp9qju\\\\\\\/FTg7diNtvZ5eJeeKGBfAH5MWp+Y1EtdVnk49vz77Hl1Mq0nfZcb3GNMFpD+NJbWouO3Wq2zfPcw+K9+ObECjvUTpiXRd6g3DXk\\\\\\\/RQXzKS2RzFPMnuMvBts57RELiJea+rgI0wfv5IXY2Td3vHjS\\\\\\\/hR28Wd5NG1M46N+GmgbecEuuEVq1x4xnCXghVmlbsUGL\\\\\\\/tFJ3SfndeRtDCQCo9AV8YuOU0wwyYHsf6uUBT4jJOCyUGju5RRLRGpZst5IjpY3FI+Mh4F+OQ3jEk\\\\\\\/66J\\\\\\\/n5Gk3qCv7Mn7Rt190RrvLd6AQRRb31aVihPYsg5xCcdVhD4F1OxRwlMsXYQ4lnaDA4aDW0cMkYkLyJEiIKIfcXxKvhnTKJDh\\\\\\\/\\\\\\\/XXO4HXRPQYLVgxcy9LGkTgnz5LZeoZB8pkG\\\\\\\/UPyJdHqyhtge6BLRB6tLeyHIaQdsFOKotnZfiuJFVqozahuaWaYA8f7vHl8azWUHWzb\\\\\\\/wpv\\\\\\\/bDENXuFBU9LA3nw+pPqDFkq18yDxeHab8UnxDs4zu6GAOnEQSmSKVKKIiG6hTGO7AFT1ntFkssLrFJ+qWbkJI\\\\\\\/TSjAa24qthKsqBmGeIuD7pvZ0tkffRAQc75RTH+k+kPZTfT84qmSCzMNZjqkYUDppPkugz5mi+ErcP7zkVW\\\\\\\/kgtmbPZto4dLDtMAo0nyWJjj+LB5PZbvns3s\\\\\\\/qdybRV5OfVlH7gQJoUTTwQ5C+13XEQw\\\\\\\/90PrhZZmOBko5lqkXk2jnk+GbYN38Jqz8P0n6GeErbtORTVOuZmaLBStZtMQv5xGsGoKaGmnBDAlkUXgyHCJ8K6JmUSHWZKi9cUmd1Y9uA0a34fpY6Qcw2iTMur8M9posZh90Z887dhcWWRnjHxAqXsbQ9wvVgltnySfEJBAcw8wMKzt8b583rDPR9qYzEwZoC9xknqe2Dk9w5DqR2I39xT3kPKuFtDpAZbuU05TszpzRXyEik\\\\\\\/0tOA4bRWU\\\\\\\/z3QgrlsjNbaru0x6XvJUsALWJGo8DOt\\\\\\\/BWTKtao0RpDDRWMS0Oyqo6V0jSzM70eJ1AtLmeGUhcNVmhnrXGUEPCYeDkSWx0Q8gxJflvKTbXLfsVuuUEWQknpzZi1dtiCWumLSYRT35HBIzHsLHYV41kDZUXuWXWNj9OrsvPWXUwHSaWqdggWUyuAdI91Cv7+1QlGf7Y98AQvdp1UhO9BOGg86VI4GxL89uVqL69kTC+YKnULnhKkMu+DT2wRaeGI5KHZ+m32cCjNUmkHiEvgZP2fxmCPMPNxZHhoGp8MDVzFcPsPlY1MR7NxSAoRVAkYa6U7jq9KoE48nAi1pQvnFv2ate58dbE1GG3UoUtJ387gD0oJKaiOPML50UbzULF9b5tci9fxuovCZUJKqo\\\\\\\/mgwuEGvOFvbKa8yR9JKyd\\\\\\\/WOcHVXJYV6QQiojkEgjyYXp8mHkxCcdJoSCScNM8mRQUOeOJsofA7d+QK9dR\\\\\\\/e65Goa91y2Cy3UoeQWUnU7bYNgKf5Eb9cvo1nLcFrlJHQu+ibHXYLxr7M2IHgBtuhmKLG4bA8Vp0XBak9hbZs\\\\\\\/0r6ODNwssA42DD6tMkVgzT6NJAwoZaSfqVgTTaU2GzMKYeEg8BKZhgxVzoxxh6FnooW1dli5FTdh7SfqMJDwTkaugoByivIdBZrbJoN5Rqq2i2lOMhu2+W54VWwri2fvMVm4dWI5xsKcpOtY93mySO\\\\\\\/Z9jKyYUy3DUSTkHVpjYT5MBz74BMIDMveo36zDxE\\\\\\\/YDPRiVTtq\\\\\\\/pPPSAApw3byFIyIU6wuXr5OGstJMMGTLLG6KBwKsz4GSVj8N7J2fVVtI\\\\\\\/WuwUKnY3yqpTSNemIKGkdTAXxD\\\\\\\/dAB4wDO2PijXKlAgOr8o5ZFZl+DB9TAsM58m9ByOUEm2G21tKS1a0WYN+eeh6F9Svsob1Mltbg8N6PZSYa6DP9qoxQ7P4QMF8zTWRC4YaJhYN\\\\\\\/oRDtCnCW3ypRi5xbF9kUQOaWuiSbYN0qyrwQ5aMXfHq80AIQA\\\\\\\/VpcCXbfJ7cTRt8rFySGfRAK698BqLzXyt2gtYp4OoycRjO6c0r8PmlERPfywflJ4Sd9VhZ3wtylHVMdk4dkJs6pFvg6dnuWWdC+pWp+cy5sto\\\\\\\/oJM044v1oC4JN3nu1lC8THl948xGESnp80En0bH6pqki3KM5TrRMrDultb+kYDqq\\\\\\\/vvq7hD4slxVYwj0xgKF3NENmmkVh3nSvMd5ZrT9TjZ069SkirTkWWpf2Rwex7o\\\\\\\/TW5aSOJBbKNn+AbuY9v1BnV1LeZSrm0WNVbn1mvan02TsjSa06wc3CZ\\\\\\\/dYdfbcMk7rMfHlWGRezwFo9tkxXs8jZELV+oz+59WZXnz3OmOQtT2kjW7tvby\\\\\\\/oTewk5C5gvBBjrc+YHF71taURw0cGikFwUqUREC9xXbgIBdxQAfRJOfW4X18LgDSmn3uydO5xF3NJaszIeCj7zAaa0aScivK1XSwBdADMPLGrE3fsOevubUzLBlF4wSJydg02BxMHuxFneidUM3l5STNeKN1V4OL4Nm7UW5RWMuqIJdgMfhuM3mBBg4uk\\\\\\\/9RqJGUFOQmdrIM\\\\\\\/dCvAe6YoIA3WV6C\\\\\\\/pHzF+eTwKJk5RVCfKW3O9tx4mNI5+Le+LwXLfnFQBJ4CiKzVztzuYm5StGIFFMoB4W8N2d9Js3MIEZrpqPlY22UBj6AR0c7Hs03\\\\\\\/hDvDJ3hCtF9Xcovugq0IS+Gh6yU11mBTazdStdUjriu5ZikcMsBYgQyoA0uUazH41eHMXj1nGzh2xNYUkkXfHM0w\\\\\\\/iMo2g7+Uci2CWzA3IwjLnvLQ30XhPEaDfoluIkAq+28oXq9pCEJ\\\\\\\/A\\\\\\\/ZNVQIlU94pkF86xDyp\\\\\\\/mZrTgIzFhOj5OCcGclsv\\\\\\\/BH09NcqpDQY35n7gYAKN\\\\\\\/GVJti\\\\\\\/GLN\\\\\\\/jso\\\\\\\/FfECy7IxS6fYt\\\\\\\/aYDOzerJEJ6oJjnV9x8GQURHdnKGyGBeiD98yx9h1\\\\\\\/eE2gYa\\\\\\\/iB3yzWi+5nD8IwXWESy9CHp4\\\\\\\/mt0Tp63vdCaQcc3altOSwjXEKmv4n7LKTmC6y7yYh38aKzVT2crD7G+QLjanGNDXyL7yQGzeoqFCf6NEsd8sfbO9awZRy\\\\\\\/1cjTGAG1K0ON0XMxVqYk+o4XRoR4rFk25yDeOzM4mdriL16PC3CTPvT4GlW1xK05c+eTtaZwkYLDxjvJSO397+rCR7a3Akyp04CbDuLnoAEjrGUEphRleWg3BJIM3sm9xBwd9b1QPCYvWQ58LvQH9\\\\\\\/3YG7ua4HzAOBS70jAyferfcUv\\\\\\\/+Kb7daGvDdcDYZMcnu8rHhtmEYGcIRRMNd08rQHUEtmH4UR28mRLcUDzmFixWpbhCe1fLDNda\\\\\\\/21RQt\\\\\\\/yeV1pEtFlpgy+yi323RocbfqzcUXUfv0gHfrEVVY73WEiDxuRG3XTSl4ntOFnc6NsjPzyE9yLU2Oj6zAltOa2TsT0CekbLZi3aBEJ5TyL1l9BYCIfW+3TZ2VqresLrmtUfr1ESUPZf913XZ+8ESeeSdlN9DjNN321fPXf5ZghYdIZlMu7+VRUQ7g70enQMrXtwWYbi73zMwaaudy1j5+mjVwoYmOlRGIpWJ4YjCKaBTgKaa0K4\\\\\\\/aezEsaz8fKY2L3lKTffHpZAo7HXMM9ndncbcSxRYn+diXJ+d+K3pxIUKilNIRYD2qCq7Lm6itwldJeK4o9OYE5FYfXUuZq6\\\\\\\/Gox1IGL25qrxEWet2Vqyjj57X1xAt8yol\\\\\\\/JbLenlKBdFTjtRgJVsoXcrKFX82jVvrXg3\\\\\\\/Xwy8Wvy0EKw7vSAGcaeWU0D7Po6n8h5BBhQGhDUZZ3\\\\\\\/UEfjbhXzdjKSzH2778\\\\\\\/k4CRpEItBEZ\\\\\\\/dL32iogV1aTnUblj9C7k93QX5C91\\\\\\\/dvMpFtQObEBa6G1CU\\\\\\\/MXfQLtdxkmcTdvnIXvkvuSKQBUKsGH03aJieqanGqjAdUWYS4nGdmNVLP9bDoE5\\\\\\\/kl2wJLIGk65ncAtTBKcT7BFLUZJimOotgqcT4JAIb1n9lObEUNJJBlJdtvq\\\\\\\/KB4j6UlXbYC7eg+RDADS6e9pvUdqiwZZjewyLVlkqifSrvnJGZ+GEY85x1tR8JWc8c1SSzyp3TC3txT4er2MjNvMhjoz9jqjN18TEi7TewAjRqATQ+ezY9urYgwBlD2eAUmXX7P37Aau2BDRlgrV9e5HsfF7M+S51Pqvjg1QO67KlAFj4JwV91c6lfKt4f3QfptNA+Q0JGKuQB0+PoDIEZVFw6K25TDyunMT8Tte3Yf9hbNlnjQI4WerWkGvsxmIyEu+KAVUaejF\\\\\\\/ZrRxl66m3QrjvAgbf1m6I79TRtykGUbi+9KGkeq9jWk1JNQjtBGyxbNRQaM+XxNwG6I6rEskg+qrJ6+Hu1tu86s+T\\\\\\\/yck3siYjPtH0xJGuqANwQT4b3ddWhH8sgGSXKNKZcjNaTG6USNpXQdLLuKr4sjmWZTSxj0rZp9StS5ZZAl1JBV7t4TWKkyUYw2x38tdJR76xSxrye\\\\\\\/Qza0dD8ntsAkxcpsfhmeGwtobqbJHkMBMsCa2e7DEpbS+RcB2ZCa1cYVQ4AxNacT9Qw2Vdw1kRyufLoORWsXNbsbxqvj5Hu8APBpbAz\\\\\\\/jaWvWfHw09J2vzB81PXz9NmUc2baPpOqcDpituWZlYOKVRHJvMzAJDBikCo0Qb5xeGzbfX2sD2dw9IXj10BBAZxU8mE5La+HIlTVb3SuvdtleQuxEpWWvZ0lSew5irCclAYTCgzs89c+tCVHhMSbjq3oLXdwfztSWQoXE53khp3mZZtJZNI3A4QFIGAVGoMpwDUiUTdpX7BtbwrTnoZt5EmaGZjjeXZsjNwRGqyQ7fZahmdz4c3eveVln9kDC1YjjSfKkMtv6KPhWanGK3ML1DgZxSz8xAE4laOEoO4irqhRAA8Trl8BM+EXFvxpOypl1pPs0XiB3MlMF+CFUF3V5AHzv0yY9EQTfHv0SVQgu81ZkfiDbR3o7o82g6zpDAJ35o7FwWPmbGOaMqfJX1FjpVN5s16QvtLoegm0iPswUi0BBrdG8EAXHBEjpOBuZUtK0TXXdy10KoA6ACcuXTmyHC7aX3Ng1LcW8WX4jT4KglDn30unsZNKXstB36hjwjupfWTr+OSRcblLzpZariOiUQEa+g+TLad4m3Zx9hY9pJY6Tp\\\\\\\/nAEFuuk+BmylJzvvCF+SlmpA08nb0rzGrXzY7dHHDRQiv+SFGp8o0ykjoIgfXPQxBsoUpVZwY+HzVG9dYRyidtaG1xA+ZQrpF+Tv7qUf+rtoCf2fH48ECp\\\\\\\/m3\\\\\\\/MKHUVG9znjk7F99ttWyfLvVeV8EDWZqFF6jFaGgSNZssJ22S5oHmPYnI0QqnbsU6CR6785o6kWcV83fPbBBlClWO\\\\\\\/tFbFpN4QOG5dOgly9ibkffjL5RCddNyanfS76fhkUCbPnxjnDX3ZE\\\\\\\/zkW5OEV2qETC6EwAhp8cIQ5LRFCsGfJiqLpzOAkPA2Fo4\\\\\\\/MdKHRqdMbspdZxLKlXLL7D9ptwPqRZzpMUHkCZLfMZAvGGZJ11FuOYPUuqH\\\\\\\/T5soQE0VdKQVGdiFSloVFQDJj+oQO1ml9klMwcm5KFQ7T995TdDjaFxDji2UMbHw7ur3udpU7RarUMssZ+gp2BMlWK2JCKJ95GW6Jinm5N5SsNGdnp\\\\\\\/bdmskTaktXdJL9lzRVAQ0Gvwkw8NNSgZ04s+fwWDNKXjSmLYYmyykVVOIAURlPxZw3J60fmJ7fae\\\\\\\/54A6uNlRIRj5wzVMwmAfUCObAY4jwnPGAce39xieNvwQSyzCJDVTIdgL+KspEUf8XTeVyLLu1RITOlbaT8kZEVT+M7NSZWelENsHCANcZWmu0UDwGznOLnt4VbcwgpmkffNlmwbHuWBKWIwkloLloI3\\\\\\\/GBIBQ0qyoTo8UHYzgfcDnLZvBnLg45QgXhdIgNbFJLidVpQsyO2xbGUtjGTI17b0aiCGg8+I9Dx329zkg36OaN1NiqoGwkZuGIeAilTUFYupggUE8CAnfFE5Xzr3G2d0A2ogeHzBW4m24lmEMFx9n5YA0urY28BFVXwbTWYJCO9vVxwGlME4KiEOK5asza1VX+KhBhWys7LuwXc90my5FBChvl3NYsfvBtFFPOQxs0nVYNluU232HmwizkVR+V2vRYxAs43A0DVMZ1WqptHtuBDyA4R497xhjp8seQ1ioiANFh+7fZ9inRnNTKLnuDBKC5FAgEzUfy201SYSzuBe080rJ2mVxXNmGOMes8ggNNEHdZCA7yQE9zwKJaQPIaZYjyQMOdCbT9LMe5YMXEYffzD3uVIX59BlONw1GTOma4Kas4AlBBXeQTCqfZIPux5eK9FHalfH\\\\\\\/nFsIwy9BRPRn4PAYqU9Zl1iai\\\\\\\/3yElBDtrzaJ29Sg4bkzi\\\\\\\/gRYPtnj51AXkEA4B\\\\\\\/xmf7FwYoo\\\\\\\/rR7zvczAt4PRUfVVkaP07ghmffVELSWCVqdLJLe4QXZfv2KLQgtcdkbYuAnqDnf7uqLYc7sc1gCn34ZbAV4+jUf46qVNouI2kdbuK4\\\\\\\/nHR8dqNz2mVM194sZzTH\\\\\\\/k5XirL2ayzl5NZXk6vSAMjozQAcB0AYznf+mdvvRyPxBWpINyP6kmUHtovGnSj+yJmnUiDuNf4ObPQ0jTaw28Tn6C62ZvL4VDBi2HN2e+5XVITrGe7VhnEuSXFMvvSc9GbyOchP8+tZAuTpqpiShq0N3BcnBoVyNl0+XeydlRtzse7uziRMKGZCOTp8YLSh58p1dw48Nk+fB+KuG2a6XaH9B7\\\\\\\/52BRh\\\\\\\/sHfM74\\\\\\\/R4IotcEa7ss2sRvKl9XW4gl5zI9wFPxZJDaXM3mM3nUCDoGee8eKDSS1\\\\\\\/f6q7K6r+DvpPhzehapyQP7UGz8BDzAC7QV\\\\\\\/AN9gBYg8McBSIP4PcXR3+AEGZ1r1250n+IVmxoAgZftOBhmOekzPe\\\\\\\/k16S0OdmwMQFMtw3pjDb0dPJnJVYgIehwywDifDX0bdCfoKM0JSpeoL5AopHd\\\\\\\/j\\\\\\\/JCHzJbP8P0oA9jZfK4bZYTwKrqvr0xbZXov4DovkGIkZSQCWJkv6qnRMbc6kCNbOFv09cw2tfnIp04bzZYDEPDeCWmUVbEPjdnLx1QmTIQKhKifOa9tnMKXqylKnYtYEhvXlFJq7Qbcl6i\\\\\\\/iBT53u1k2kZEmmg\\\\\\\/Dyw0UppsNlmHcS0mi\\\\\\\/4QMnujELb85xOv5YG5xojYAt3Fwxxv3X26hJQScw6k6bmyS2O9XLCLKytH9XWAG3+O\\\\\\\/ZQta9W+p2SQ2PacOjIgzZiwYNVVXiiFoyLuGwKS3k1uDgIL7zlA\\\\\\\/cY4Tkc4143v6\\\\\\\/2aR80ZYBpo9iJvBZSS+eOqpYwIeqmZ7tMshR3yTpofDVfGV5HB2dBtNUbEVOeaq6LJYWy3Q0\\\\\\\/lDuXaDn1NmMCpQH+FjZ0\\\\\\\/YQSZxu0p5k060RqbDwihxUg5oyy\\\\\\\/I\\\\\\\/g88akLVDXk69vJn69MC15BxwMZTaEBv8qKGHlU0gHBRZF26K9CXpymjRSfZP1EwwgeldGOyomaAECB9ASNSVbqE10uU1FTtFIAxqxb4\\\\\\\/M62yvvd0ntdfyk+VjkGdvCvzS6n75zcJwHSkCGR\\\\\\\/OHheG8brlFJwsx2YWxzfb1XMq1koNaIstlxWksKL4uNj+M6XLC97Kj9jRMKjIDRBe1kuHCnRGq0G48ap1bCki0muCBz73+x83oMzHiJt6xnFgdkd3AK\\\\\\\/mHIUD2R7fvuLQb7+vZ53b4wqdt56AZMWIOvZZ1cmwRgm3uCt4pKMjwyBC4XIcCdAoxJ4YUaIri12NpW8lbRlHMCJIiLcKZzzIy8mBossfYKGEnvMPyhilPmqWYQDCFuSGnBxbVzBYoVGvguL5DLDxXl0hLULbC0hLbX1oNStUa6U7HLZ+qVDJSHHuGrobGJb1GBE6iZTRpYjeEnlCCaLbHbfVM\\\\\\\/nUzcNoq2HLX1rmB84JDsWFIqVOj1e563On3cdNRGt029DoJsr3Mkk109sR5\\\\\\\/FYa8fJjrps23kwrw+KJ7uvGcuwaVc\\\\\\\/irZaj+yI0Nc58yG01FMT7iQQK6ZTwi8w7NBDRxK66cAy3Z9Q5nVVKb0QHzJyHxJI7EvxcewDNRKTGISUyZ3Dtk+rAOHw4RRHRqKp\\\\\\\/YLM+oHd\\\\\\\/SUboRbj3d\\\\\\\/Oc4f2CVnUbzsg4VjrAeMLOCGs4iojLSmHfW5+Fc3UAKUWpXALEGazuTZXR7pTi+3EbYGmKBqU0bqh+Qhavl4oIsZSd2awsVf8lpR4Fe5GdGmCDPzHXuqVbbNTttzU0L8AAk\\\\\\\/647mdHQ+QxqfLhVjwJFSFKp9I0PrLLE4bMRWJiU7H8fVWaMTwAxgFKm6iHV3WBPTGCti7eAw6jxkkjsJ40LiVriQzzDbq4qrrCrfIxCAlwg5Gini9oyL+sTEXYAUohPal5rXGVtUFMVtAdehRK\\\\\\\/r478VDib8\\\\\\\/5LB99cjNce\\\\\\\/wSb3ZuB+sOhEqbG5wtZCYXWimYV2Uq7\\\\\\\/SAad3pXZO9rVVcJsZgH2W4cyuVcEyHqsaJhjIP5AsFGAmzrWkoRuw9RKB2EfAt4yklBmZlKSzzmS8JIDoetpr8ilStXTVCNPciRH+RG55HE6mElb3qA2RmSya7IbSj4yKYM7uXSUOxnZd6eeHiK6BnDsIIIxTej2Vy9xBje16QcO+1RZrALGAroCW8\\\\\\\/ueFJgwJcmkrx8M+k+1xnAv\\\\\\\/VS6uU34I5n+9erzk71mzney8auYRfiV94uErQRL1qmctWpcGe5HC1p6jUxxXM+JOA5rIhHES2guF34AMoLBsB3Td\\\\\\\/zBuAJLBSGAOuGLJ5zhZTa5BYs74BKhPxJ\\\\\\\/uvMDCjdF\\\\\\\/lQ0XD\\\\\\\/ri6FMiPBeH5dcPklKCYvPdkF8vqukxx8Riq3OqVdEgJOIzOQ\\\\\\\/E+X7Jq0EHTvglQD9eJ75OMIhUOE5ZPC7rKd\\\\\\\/BrT5tqEHSbaodjVFWjIyNzIG1BPVVBT1DrVx3xiJuEotKwEKG3cTp2fSm1+JzUKhgYUySpFjDYO\\\\\\\/ni0GHW8tLK8b3brLfGw\\\\\\\/+W7xOVx2N8oBYfyXhaAH1RTQShaNxVKzud3XHlghKmQcgNsDDpIIXOL\\\\\\\/0p5XP\\\\\\\/7Rw9glaf2IKlFzhA7bzOh6FdBaih6XYg6mA4FEM+IWFfwIirUYitZAuo5d9CqcajakRr9b34RQJLzTLymrvfI4ftj5Rj57eAkmXW74m2lWzq3vEKcx+chFRQ8gq7yANlZ\\\\\\\/bcvA+90jZAgmGsmMzgK8NAFb5sfQxVXUd1T6Jq4k4jj420xOmQ4HFKIjhQx4ZJPtslNKolFTw7N3Bn4VeNmKJfPPrTd\\\\\\\/J0qWAt5NZE8UeTvE2nlewu0pkImgYehfR0qt6MhT6KeUJcgQCYlBh6RuHfB+3z+6jS3DEVlWJgrrG8uK891gpAfYBZyWEMCicg\\\\\\\/qEPCGgBwOn0y1sShAXiK4U8R3rDIt2m8QUCJtzp2ePciw4VpVgW2ACpucN2ZWSMk0NZGQK+psegk8Rrfcek0AnvZLLPwvz07CCSEVhmYOfR1pt3QLvnB1jMQj6EdlY8BF\\\\\\\/0G+A51DXtyYU0zZxM2BT24YabkK4vLmeSkn7p1QilhGGbk9fj6jYp6NRR+aVtjnsjMggVKU4gSCkeBJs81UYwFpzf2hoNpKsMReM+SiWsGnjpCCWLoN7BZThayIHvykqM0OkVD6fYwM4VgRrGpLINwlhfeju\\\\\\\/bSFOIz\\\\\\\/luweS12c2KizE4VynQEmBm1ueUlJjB27O\\\\\\\/zu9Mzg5DtGD\\\\\\\/RrDtvnXsGA\\\\\\\/TYgXNHUAQobfgVwtiotS7bHX14pWlm\\\\\\\/4Zg8daQ5xzqYoyFmS1cumNvixyKI3IcHZGtaCK1yI9807zCted5KMnj9WcX2QE\\\\\\\/aVckOemPxae7KY5wCPTZJQAqaAVk7okTcjtY5p16rcYf9As4qrmsTt8zRNv1cXHlykrdxm+FsIj41A1Dn\\\\\\\/zKJTJbXxkQkvQItuO7y09Pqv2p9tThnTlUPx3+tRHItbr6GQagR9NjE\\\\\\\/UK0QMINW\\\\\\\/MsnYKCA4NIxbfXJMwY6E356IgxglPXv0NsX498URS2bMEhoG+aOuG\\\\\\\/AP1mQkL9Cdblpb1U1VpR5Em4zBgi7VDYDXD8H4JI7B7HEHyty\\\\\\\/y+62Gn+qpLAFg\\\\\\\/Zz5aqzLGqLm8VLgD63Sgu6kSfu\\\\\\\/3EU46hc12V3hBcuWrasHeEOFNAhI1R0czB9NcYS+ZbxYlvewTVGxtutTmduBsZOPaHQDgMSlE07HDjd6sn9lq\\\\\\\/Ouh5ejgwIJsx8msZJgb0dMY4rv0s3NuXZawSOJydKZG3Rf0H4R6vxSxk6m+l7+EkiJwvu1iOJExlRq9xgae3EO8DVKN20AV9PDvbcd\\\\\\\/5upOqkW4rbW7IbKTc\\\\\\\/\\\\\\\/ZnVOIYsDce\\\\\\\/Um5OMkXev\\\\\\\/P4LOv731Hw3StTnxam\\\\\\\/IryCi36CUmZaGBUQ8IopB7TnF1Dk7sfW8lwf7IswsjN2SRyqGPt14+aopSkYSvGql+L8evEp7zQtDBn40TS7U3jSy5n0infsQ7XWvI1Xry7UlcCtS41h9O0mNjmsSiWicBr8O1uQxnGqsv\\\\\\\/ygmXSVv5F8UYlKfNieeVJt3\\\\\\\/qNs9yM0LGWf3zY9feOaRknUDNRp8JeAET8kzvBcZ\\\\\\\/9ekZ\\\\\\\/EDHGpFmlQDklp2qTfutX9SDnXC\\\\\\\/7yROOW5\\\\\\\/7aM267jah2DT2nIVBaqPZPoSljqfZJtWSAa1XACvqPK3Hr3gqUF5llPRsv6O+Xbx8aEVv0TecCapy40nDzrAtf7jqGvdhHotUVT+Wb3+\\\\\\\/vr\\\\\\\/qF\\\\\\\/o4TVJ6Iisdpwpv6lIHDjvsPOiNb8JgWCcBNIaWhAWsK3wmHt3AE5pFRo3zR8fLZ0DDtbJkrEJ4M2LnCRBHn3aQ\\\\\\\/BCwIyp7zA2DPeEZW3qh0kLCdG68WSOgH8ovSCHC5\\\\\\\/JZcBu7KUj9CpSBc\\\\\\\/0TVztoxpvyVYe6CSmp4ZLT7YR6JaKe7RbJhqIHn\\\\\\\/jzNQD9vTwyuG3prXhPQ00PhyqniwzwIEkreqh00nsWYH9GgM\\\\\\\/z1TVxu1BtZhgwLClvcCziF9KrBMnYzaSPvZiBi4degSiMg+0IXop91zVLVoNmC0n0NkGYytoYWE\\\\\\\/OIO1IdQaM0AR4YJz\\\\\\\/1L3PJ6VUutJPFL7by21s6jFmt6lvBQyemnvWUEui74yeuvMDF\\\\\\\/1kMEuQKIrA8Xy4QR9axdBxrjZgL9Lo+4n5i6tu8uKgmcvOqT2\\\\\\\/DKKHQrN5oR\\\\\\\/Cd2C6VVTtO7MBIJpLvM9TnU7PnFrASRtWvGl\\\\\\\/+Skci+GF4HImZFFJlDOHrOMXmecDk03Xkr\\\\\\\/lHn4PA4pakNLeQaCt1+UH0YIzTDQrFq9jXNfuuqY7B7AGJMY22kw67Zp3G+EjbeZGH93kDgX02l6CPI1hGbtgy8OZa\\\\\\\/4AiNLX56vJeHy40I3Mc8HL5Cr1QMo09O8UIsaLyFZqk0fHBTSkNhtwH+41qquf5Kake3Jw7+nZsGUbdyRbgGnPm+HrE\\\\\\\/Ve+W4KVENS1zl9Jq4be4bDXFrItX7F4XJwiSZYxSwr24TFwVmwxzZcNc3lr7e+OouOYqB+bJUVO9n+UVCElbYdzMLAXDtNQzz915whPTvX9BU9z8SvATiUTT55E5gnIe377dS+bmC+DV1qAZrhJBtWX405CS30Tf3a+qFj7soXEy7w+enapnH+Nzvo3+JWVXaWfgAgd4ANCwZfJmNB0ecuEcjcTjmNgkmCNBLwoHQNnABNmvULw2ouWQMMvbjBQsQSOSQKuIHzIZlVprKGhd02JrQVjG74zJWIQ6weHSzA7+BnkfyHcRukyM8o45Xim9QoiNxIfAfSMHkCvRrk8Lh1H+FbDl9pdWPuqVmnG09mw32WlSal97m5vKU9XnvWI4HByujfll9c70ASBWNO550gODnnTel9+JR+LFVB7o8LrQJFLG3zNtHm1tP9FXmSY9RsyRZ\\\\\\\/aSs2d5Ab3qPUO8L0sdg06BnpEM9pr6BFYXSl9jZ1Z8nYEPHH8fa8yUrR9x2VhIHC+sCF++hNoHumH7s84tU+4KfDcOKip6qv6i1ah3f+EGukbEpmjDLHuU8zKoR9Bnituo3MYEt8pjMqs460Qj1XkW\\\\\\\/2\\\\\\\/k3wDH8\\\\\\\/QPniRrrPJPFcRD2k3dJUTriB1p5kcSmszmk0MO4GZkxYz9HDYxViVdgs3XW\\\\\\\/kn+QDwR4u1yw7fgdXeLw6SyKVoa+GJCz8PDH8yABvsVjMf6TYGakSyWqC4xUnB8Qpa3sAGh+le7NTJqVCBwJudFe8t2+scPm4vKhGv0O0fxKysOfL78TxD\\\\\\\/epVAMQFUcWFxScb1+XevOp1cFi6xcYRV5BfgX3pKhA7o\\\\\\\/F5xcO7ONNgYjiojb1Lpl4oh6whL+nOdldao\\\\\\\/zwPoYnX1qouZv6q5YtDnAGSKHW6voY0Vc1A1dcuDzfK9NKbb6c3F\\\\\\\/ewnWIOniME478RtrOMqzVSnDcHKhGF6KVoRGGd\\\\\\\/wrVF1IVLSrzxeGh\\\\\\\/6Mb1TRWgpYTedznEr4bcesQOfn\\\\\\\/ziJz2rmjJKM2egxEGj9gYrrj09MdfSjVtQSHZxF9eQouFFt6aZvz1UylEzpzZ36vk6LqnKX+2s\\\\\\\/H6h7xuXiu4oXbuiFyJgTl2CnFqh6QKNYV+w1bnAJk48VNr9Y\\\\\\\/EZH0tI5qH4VO2ysYbWpe+byg3xpinN6VpaOOcLFTIeAhDJQrwc7OdoOyCgwflqXYlNy7TXX49M7oI4inwJxY3t5WGHgQajdcbiSO7JBweVxdpToEc0RkQp9tPhfTNfV5I9OdcrOJYsG3+r0FWJ07x7G3s4LeQv+ecfm4gHgnK6mmY1UyVOuI67fnH1XOoOiBb3EUdNIsjmGCYgHI3YC69dbXErD\\\\\\\/spVkNo\\\\\\\/RqkPygm9NhqEXr8k2peTjCTXTxPdHZj58Xxqgr2ye09gJcf17HKGMjnTKn8SFY7o2jbFWhYQe6znG7AvTyxwqwqlSaU+obBw8A28c5KNGr7wwje5Be2Mqey5b4Iq\\\\\\\/eER3AWebICyd\\\\\\\/TXnSUkXt+QEX1WA87JRtJeBAs0fqCwfWEvdH1Bk\\\\\\\/VQ5Z0Fa\\\\\\\/w5A7aWxKMqNIpMv4tQghDecTqP8myLh4heOsthRsGRo2VpQa6F6io87vdNrCFgZmsMY7Gpf9x\\\\\\\/9rv4Wx9ZkUU+bCSaILbEuhGZN8SATo3fO7AattqFHtvG1jLSXqKzKZF+prAmcsINMLvbnfvIio2B7rjNLLgOREcg3DQclGYfxpjkTn\\\\\\\/xGut7lGRW3FTlugrCmM2qYir6J4g+SSrohTha46RyudyhMu7t44mrzGZk+eTrUrM\\\\\\\/c4fv9m3cEaKR7OMNGc4dVVBdkB1Xelae\\\\\\\/rXapc93rzkrcxgHjYCGGNUUk9FHPDBQmTQhLVB9SDCif6T9bcJq6MCvZAegM2x5WVCDVG1l7swG6rGJUMpQ1a0mZuQN0m573718JfWUf2B3rgEPUuUdLBIYcSzyeA82tG+oFoeasQbp+YSfM\\\\\\\/4k\\\\\\\/o124rV3IrV1pxDOA0XZljX+Y5Xl5f1Rc3HJlbbj8ZNbZ0C2+Kc5\\\\\\\/BER\\\\\\\/3WuuCl6zTW\\\\\\\/hoNH77zHwZDYJO+vp9Gp3HboAp9rHUefhbLki\\\\\\\/8BMX65MTnf\\\\\\\/6qyViMJRzCTfSgdiyBor89mRxn6iDC29\\\\\\\/Leyv4Qs4IVux2DfPuAnWqoTWtV485By7EPruUeaoueluJynMb1\\\\\\\/ir80YYHKj4axBLTyskc2FxsIOylcXxznpWWTNSnh4dtle3U6WLpqoLKCP6nOtMf8huOPq9zNZCegNktxYRLubX+qLysD2RB2Qmf0KNbMBRph\\\\\\\/CksFam2acNcHWlk1vt5flZSUeZdwCiFEGlpo47yIhP7EKjepmTIjedQRqVfMuYyTU5ouzKcxVK8LaVfCdLTf1ckDCJh\\\\\\\/tSCJMnPx88\\\\\\\/LKkDkG0RS+Yr4Km\\\\\\\/dNavBOxuJ41V6tp8t03rFa9eXTV3z4gtUq4SmKLWC7ehm0GyEZiBEAOAd9wAVuVld+bh+VJoUo\\\\\\\/\\\\\\\/IiRAC5307IPwBmuWBG87dploXBMrRU4yUWN9CUfnBKorMrC0IsDZ4aHF3UTa2vgM4ELFc4HKv2JlVonG1ddQErCXzhrCork+XesHy+ovXVyGNT0yFSZAVLDZyRGrUba\\\\\\\/OLu+vz7NrqVasajHqF0DMaaSwHioZShFcYS\\\\\\\/U\\\\\\\/+LItlOhgy78KrH1X12IZiUo1tjJWawZg2jYxZJiemCVRujhuB4RWNw9G4+ifxo15zgPIU242lHPWFl17VF5KRUQzJ19IhCr0UGWS6s8CeiYicMMeXx3NZdHlVhYMJJvdsJgzoQaUpvxfLXsvgDe8+IEg6HTCp043KNMqz6jM3zptWEgrAGNVrXWKTFGngbsBRNX+4hLDi7yx+SSz+nUY3BywxjDkNIOZ0nrs\\\\\\\/yd+bltjVAz1SS9QVDLhuqUJpdPWXuvcMhlY+tzsIok4wSNSsN5xTqKLEW+58+10JbuP8guCPkmV8J7NOo92+BAO5QEtY\\\\\\\/aanqxsESWD5fq90pS\\\\\\\/L6mElwIJ48ZvV3aqhOODCm\\\\\\\/C+7WzoEZjdufD1btV7PmWKuwOxIDX\\\\\\\/KA8\\\\\\\/7m4vqkTcJm81vPkoGQ\\\\\\\/PpwE8IG4r6vFAKjYnJY9EF44LVEEiDssNX7Qn4O5nGnukPIcCeG8+iL1cCstvtvUWwn2UsgkrdV03ROcXqBvl7qboiZd42EK2cWBGQ51ryGV5gNUcXezF+MjWIcn1oj5lT1PL9lVG0tLL3nU2p4SVm7gAMlLpXYMSUH9P9CvieVNaswMuZnBjO8kd\\\\\\\/rf202pEh21Xx39rbPEzUTPiuaTW16SkGob0wUu0yjwoKdRiBVli\\\\\\\/jd8S+nxKVkwrowNnHEBebzizMDGR9ANiOIxXhodWeRHf0mZ2jlwGrUa6vHb4x45y\\\\\\\/KBiAh7qTM47MRYVoqMf8XSa9fxXQ5hnIarwCGU9sZKAOhSyD68uhtF\\\\\\\/LMhrrpyFw\\\\\\\/MPFgq7CCUNMbBOZ\\\\\\\/RcNieGR\\\\\\\/LxkL646CAtZ0H8EybwlQiIDQXiU1ihzZoaDzDYJQefylMybcpxtpNp41HW6zusCmwcsyn2woGHSdVpXAHcKLALiyq3SmeKBnI43hG4x+DNtIA8WcN\\\\\\\/\\\\\\\/TpDNy8Xbk2QJsj+tvvtMLckpdWGadWErUPa8gXF\\\\\\\/kiPIpv8FxEYtuysSIBHStuxAwJ61QBNymdnGKFA9ORupYA4lBLWWllK3lRIulHmiRZDTlT2PhncWeqd0ZUe0J3WK47nya+9JWT7krzrvAUcmac4z8gxKLUlWP27SIYpNuGIZ9ri1oVTj8JTWPSH7H7qyXxSKKgwuZAiiWex+WwLh7dg\\\\\\\/k6evbQL5q7bjkTECJsdEZK+kCmxBDANOMYty9HXIbM8uMJ0Hpq+m8pBm36rj4r30FaMZe22Qn2QWoaIHHbKebvdCzlBiBZN7MkgJ4PlyEqjDYSGXLbe2PWrpYe6llTyvN7GDQWTr6EqTR4FggMpWphWjGVFUwXv6egux4+hN0yrG0ga6I2u9CpfVyotcXwiey9tPSFOFYXWEPR4aM4EAEoI4OBndX3Lz1+\\\\\\\/C7EbwKGlewfk1AoN+bwcMOOr0tT96TsVOWmkU844FNl\\\\\\\/WagpJaVa3siLi6ROeZ+xHdCZZirjghGUrkhkrjS6vj20uB11d7zamSj4IvjUy40VcHSXIVL8T8+yiA6WLMjwlyKHIlT5tJjvd3X+TLeFGrUfKwlKgzQm6nppeZ3Y89LhsWxW79VaaSObabd7ZeiN9f2gtygsMSUekf8QZ\\\\\\\/gEUiCbNAnSq1vUFMcDlc6n3iFMeExjv73oDftFtSFYtYwntUofB1PqBpHLk4Yg+o0VsnGU1TmM8KZCBMq7IQZEXAxFotUaUethWYq+FODFJRvj6d7wqWkS7ucO1AgxJs1ipbnF+MevFVdzPcXn4fSWe87xNX\\\\\\\/ZsAMHHX3A7Ofulv4lVy9jXttMXcTAjQHvVgyqorHiZgm5WAFPPiZ+ysDWi+gCItI1k9yRzOcQUENEfne8fnKnCNBqs4QHRIxdKmVuEsvb5J57lqcoj+3OCwAt1lqxMFAXK3Etnqxnlv3hE\\\\\\\/FGvp1Le9lj5wETd0Z5O5GxEYq634nrCWHxZ1xI\\\\\\\/dobHMZPRNK9mHRR3TWx5SPeG6dreuxvxcc9KioJt6bqv+W2zx8DX\\\\\\\/nj5OlLmaaYhMqZ4374+JKgkKTAvlxrMuYx7q1PTk4MNZw9FHPQIi8ywZz6tt5hgEoCF2tRidiPYcFZbFLyzyl1jl0l6iRDyaEVsP5JTPKlcPJeEd2N8LmK1hKp6y3lSpaeobbm9WoGLtxVoO0b0xFUJeQQjekNhVRGjO3t11dLd0hYrDBzL9H3juPt6Q6uQbORorFh8e\\\\\\\/r2FKXKITunSQiDrYaJEvBzgLXv2sQfIfMsefH5+SDLlHXjAv+M+ja6D7hYFkXIb\\\\\\\/XTtx6LgwUKqBm9x6kdCJLh\\\\\\\/LMD1feH7D2WbgWCFzwAoKXMc3P\\\\\\\/4hTSFbEwFFh9jM2gNa0qSrbZyUrOKtJhv4F411BknzL2TLJgoHLa9GdphvX4T6Yha8fxs5DkHG\\\\\\\/U5OtPcXrcdCib2l7OJeL3sXoo94wtizjhlnD3mO7q16k4SpNWoqr3bE+UZJaHGxncrq\\\\\\\/MuyAS28X28NTXd9V3+Vu8srB4WonT5wUutPKzIDGUPr1NMYNitBZlYQkMm+7Le+2V7pPVG3ZLBTGSB4dDqPrcZVFbCSSZ\\\\\\\/pre+vqSHiOnBOMZ6cSaU\\\\\\\/mQyOBFMw71sNiGkL+7lXHTpnYcdMmRehWUwG++PuYDlY9zJZjFJYhOcrvoyGt\\\\\\\/IgNanQa2mVaHaJRXKNZi+woNl\\\\\\\/sHhnt8Gbp7c00ie0TVIgiLF06oJ8Ahdy+Z2EUoynkR6bsYYq0ReYKzIibIzXkpmvLSPvDWW0Cd+4Rt+x0i0ISns99HSG2GbZrW7vWpjrME0m7Oq4D5A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022p9vtAqttAEccON9pNM1K0w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224b058a717045a093076879e1686e03c74dba364a87f760ea810e9a19f7073eac\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lP4Q1TC+VV1CgeVjF4Aof0v0zHgIYYvfEPkbPcxYaU45k+6pyK1grsdu0Xb6qUBv9XLtpIbNVQEEFlTOaazroMi1pUxvY07bt63\\\\\\\/GdO8v4WUZuHH1fF6XQb5bindHBGFRyBzA5yfvxLW9nCodD6BFYE9N7zZw1AbIamDWA0sK1H\\\\\\\/KHlp2jTXDqumnboDV9cNSXx80T96IDQW+R30VMKtBv0Jrp67NpIy20RcrIeXAJv2QBDzDNUJHaDv8Kfl5moBUfLfqVV56qqstS5OMA0hLLqW1kqDjcw9qrUQy\\\\\\\/hsal4spYIMh7JNmVP\\\\\\\/39qgFqljsnqqdyLJpEK5NUyBgssBNYaiXewQIOIfaII4xF1dgXa9eW\\\\\\\/XPatLfrpEBiAMrMlJ4Y2\\\\\\\/NAEi8CTzM0yqbG63u2oX+geHwADNe1\\\\\\\/pMrj9xkEVdOI30ZNCi781UIPjv6pZ6hfUKHfFiJV8H4+lk2aYCVOIvfa3duYqZQTjczH6qyF5sTJyR2Fz+YI\\\\\\\/L6pEnjRQRFQJM9KLiU2C9y08LWFkX9Q7kNURIgnqriqVf9tBH6GqGPFvpxG\\\\\\\/4fsGjxao2XgBWxAnoVN+kTDjfKK3O5UfWYy\\\\\\\/lNI\\\\\\\/GdIZxobbV8RcX7forwj\\\\\\\/u8448XG2U3HtLYPM\\\\\\\/+YM7zcFyZmc8rXDNwU\\\\\\\/h7ecJwOczxb+sgj\\\\\\\/aOzbejidlXrF37HS5qkK9skigQv6r+CDUlzVlUbafqwEJTx\\\\\\\/T3eR7XL+qaRxExc\\\\\\\/quCpscWWXPOVOulpvMfCg\\\\\\\/H7zC0dH0YNfd5Xf+PMy6lx4nlXN84ONODr8C2x6IIXCR11OKI2CPN69\\\\\\\/4bOchZjiPpQVXmOIRsPhrCCho8c0GljAA31PRyPAyomzHjgxbt1bX5zmLWqKxbN1ZMXnS0ixvcjPaDfE7yJ9sHAljk0Be5HxE6eB1OY8iZNFqTtXhc1FkBieYX8rGwepV05EI+qUQEpnMzGo\\\\\\\/5MdYJTNdJEWtR6ZGlHgrT4xxYATmrYdth0fhTuwECMU91YDPGvRK4rd+CibWcRxjNi9p5SKOAAOwW6a+4HFq3E2G8oymqI0i220VuEbsDbygUti+5s6ui5A8Ld7rAVPyQxQmJlj1Ae47i79xYFM66spQES58juS0TZd3vXSIMm8gT9Sr5YVRL\\\\\\\/GD8g3nhjS7S22knybRhmPbluIW+MgYhOl\\\\\\\/bxlv\\\\\\\/ooJbzSw1c5WzfqTnOLB73hy4J9iW4Sbg5KhZ8i\\\\\\\/qgCSH3RG2xDCn\\\\\\\/seT7Os+uORqNQNymlVhvfWHx5+qsbK\\\\\\\/wLiVlC73dFjSk742mLQB7WStD6WBpCS1xlmqLigcDrKfeDFIEJ84wVjF1yjbKRsWogMSa\\\\\\\/bJAMIt8x\\\\\\\/33BWCt82ff43jNZrBFrmdOa6EhtZ5qNs8wyI3LkT3+YXGK5BQiJHGbjQBKUOw35fSyrxP+mZzC6qoWfEGV65ctNJeLlnVxtIn3XM3MmXbqmljFIz0goM2NM4yHDJY2SavpFVqbMcF7ABjY8BnxtgZ5nCwqB1JqlGGsBLQeEZ\\\\\\\/5xQOvRW778u0PFdlsLcNORYQmJi7yQAtmXArn5sqZ2ZQ+QNIyjdJ3Mcy7FNXyxXRZb9LYqNLfQU2ANY6lzxFE8tEGRCGi87YLnPSXbcYcUUy6kK7tPuTue780mbq3wu4cX3Qj1Jy2F5DDJuiaEBlYItPrPNkFIQG277mJgY14ExK7HTixDEU0QJE3NuVnnlzjDJJXYsfiJSaXIdhnHwKtnS1Nun2LHbNEvmqMeDhvzrEoxoEGlG7uPNfeJRCRuURUp2zhmHpto8er6ZEbdFtYl\\\\\\\/K5uNgGzzcMGNMKIEljpElakRxndu1Bnu1VvpNo9bJ6M9iIalZWutiXuW8BWJxJitMWEkazbjHskP2TIakaYIcoTvkvW\\\\\\\/OrI4wH1rlXM0yghAFbgcdnnm\\\\\\\/bUcz4o52dEznWsno2dk\\\\\\\/Zxm0Cu5d4p7SEVW4CwfWKIWUGljWGakel8K33cJ9KsaB\\\\\\\/uo1V42Hk43LArgoQZv+xTq6LGKDTA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022SM1Y\\\\\\\/oQv00YZX2ujwG0mgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222208ec4dd054e6a3ec7d1c3ba91591d427a693785aeec137abcd50fa77b18213\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459743 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2519" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--NCtraMqERat9kXv-ozKRGOyqdZ3fs9PoMAaoVrTJ34" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2519" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6qb7l-mlW2WeJYTOZo0tSUo3db4FuZz6IaBXRE_Rj8w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/12/2505" title="Spatial Localization of Recent Ancestors for Admixed Individuals" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/12/2535" title="Regulation of Synaptic Transmission at the Caenorhabditis elegans M4 Neuromuscular Junction by an Antagonistic Relationship Between Two Calcium Channels" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459743" id="node459743" data-pisa="ggg;4/12/2519" data-pisa-master="ggg;g3.114.015289" data-apath="/ggg/4/12/2519.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Trait Locus Mapping of Melanization in the Plant Pathogenic Fungus <em>Zymoseptoria tritici</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mark H. Lendenmann</span>, <span class="highwire-citation-author" data-delta="1">Daniel Croll</span>, <span class="highwire-citation-author" data-delta="2">Ethan L. Stewart</span> and <span class="highwire-citation-author" data-delta="3">Bruce A. McDonald</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2519-2533;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015289 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459743" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mark H. Lendenmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BH.%2BLendenmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lendenmann%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BH.%2BLendenmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mark.lendenmann@usys.ethz.ch" class="" data-icon-position="" data-hide-link-title="0">mark.lendenmann@usys.ethz.ch</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Croll </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BCroll%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Croll%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BCroll%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ethan L. Stewart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ethan%2BL.%2BStewart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stewart%20EL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEthan%2BL.%2BStewart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bruce A. McDonald </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruce%2BA.%2BMcDonald%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McDonald%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruce%2BA.%2BMcDonald%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/12/2519" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459743" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459743/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2519.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459743" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459743/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2519.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459743" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459743/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2519.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459743" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459743/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/12/2519.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459743" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459743/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Melanin plays an important role in virulence and antimicrobial resistance in several fungal pathogens. The wheat pathogen <em>Zymoseptoria tritici</em> is important worldwide, but little is known about the genetic architecture of pathogenicity, including the production of melanin. Because melanin production can exhibit complex inheritance, we used quantitative trait locus (QTL) mapping in two crosses to identify the underlying genes. Restriction site−associated DNA sequencing was used to genotype 263 (cross 1) and 261 (cross 2) progeny at ~8500 single-nucleotide polymorphisms and construct two dense linkage maps. We measured gray values, representing degrees of melanization, for single-spore colonies growing on Petri dishes by using a novel image-processing approach that enabled high-throughput phenotyping. Because melanin production can be affected by stress, each offspring was grown in two stressful environments and one control environment. We detected six significant QTL in cross 1 and nine in cross 2, with three QTL shared between the crosses. Different QTL were identified in different environments and at different colony ages. By obtaining complete genome sequences for the four parents and analyzing sequence variation in the QTL confidence intervals, we identified 16 candidate genes likely to affect melanization. One of these candidates was <em>PKS1</em>, a polyketide synthase gene known to play a role in the synthesis of dihydroxynaphthalene melanin. Three candidate quantitative trait nucleotides were identified in <em>PKS1</em>. Many of the other candidate genes were not previously associated with melanization.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/mycosphaerella-graminicola" class="hw-term hw-article-keyword hw-article-keyword-mycosphaerella-graminicola" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Mycosphaerella graminicola</em></a></li><li class="kwd"><a href="/keyword/genetic-linkage-map" class="hw-term hw-article-keyword hw-article-keyword-genetic-linkage-map" rel="nofollow">genetic linkage map</a></li><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/restriction-site%E2%80%93associated-dna-sequencing-radseq" class="hw-term hw-article-keyword hw-article-keyword-restriction-site&#x2013;associated-dna-sequencing-radseq" rel="nofollow">restriction site–associated DNA sequencing (RADseq)</a></li><li class="kwd"><a href="/keyword/automated-digital-image-analysis" class="hw-term hw-article-keyword hw-article-keyword-automated-digital-image-analysis" rel="nofollow">automated digital image analysis</a></li></ul><p id="p-3">Most fungi produce melanin pigments that are located mainly in their cell walls. Melanins are dark, often black, biological macromolecules composed of various types of phenolic or indolic monomers that often form complexes with proteins and carbohydrates. Proposed functions of fungal melanins include protection against irradiation, enzymatic lysis, and extreme temperatures (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Butler and Day 1998</a>). Melanin also plays an important role in virulence (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nosanchuk <em>et al.</em> 2000</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Morris-Jones <em>et al.</em> 2003</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Youngchim <em>et al.</em> 2004</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mednick <em>et al.</em> 2005</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ngamskulrungroj and Meyer 2009</a>) and resistance to antimicrobial compounds (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Larsson and Tjalve 1979</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ikeda <em>et al.</em> 2003</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nosanchuk and Casadevall 2006</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Taborda <em>et al.</em> 2008</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Liaw <em>et al.</em> 2010</a>). Melanin is needed in appressoria to contain the high turgor pressure formed during penetration of the plant epidermis by fungal pathogens (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Howard <em>et al.</em> 1991</a>). Thus, melanization plays an important role in fungal biology and in host−parasite interactions.</p><p id="p-4">At least four melanins have been identified in fungi, and two of these, dihydroxynaphthalene (DHN) and dihydroxyphenylalanine melanin, have been subjected to intensive study. DHN melanin is considered to be the main fungal melanin. It is produced by a wide range of plant pathogenic fungi and is the best characterized fungal melanin, with a known biosynthetic pathway. The genetic basis of melanization can differ among fungi and the complexity in known melanin production pathways suggested that a quantitative trait locus (QTL) mapping approach would be useful to identify candidate genes (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Butler and Day 1998</a>).</p><p id="p-5"><em>Zymoseptoria tritici</em> (syn <em>Mycosphaerella graminicola</em>) is a heterothallic, hemibiotrophic filamentous ascomycete that causes the foliar disease Septoria tritici blotch on wheat worldwide. Under favorable conditions, yield losses can reach 30–50% (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Eyal <em>et al.</em> 1987</a>), especially in regions with humid and temperate climates such as Northwestern Europe. <em>Z. tritici</em> is currently one of the most important wheat pathogens in Europe (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hardwick <em>et al.</em> 2001</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">O’Driscoll <em>et al.</em> 2014</a>); however, only eight genes involved in melanization have been investigated in <em>Z. tritici</em>. These eight genes encoded G proteins (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mehrabi <em>et al.</em> 2009</a>), mitogen-activated protein kinases (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cousin <em>et al.</em> 2006</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mehrabi <em>et al.</em> 2006a</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Mehrabi <em>et al.</em> 2006b</a>), a velvet protein (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Choi and Goodwin 2011b</a>), and a c-type cyclin (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Choi and Goodwin 2011a</a>). The G proteins and c-type cyclin had been associated with melanization in other filamentous fungi, but none of the eight genes were associated with a specific melanin biosynthetic pathway (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Butler and Day 1998</a>). The effect of each gene on melanin production was validated by the use of knockout mutants. Seven of the knockout mutants showed reduced virulence in addition to reduced melanization (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cousin <em>et al.</em> 2006</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Mehrabi <em>et al.</em> 2006a</a>,<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">b</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">2009</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Choi and Goodwin 2011b</a>). The eighth knockout mutant exhibited increased melanization and reduced virulence (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Choi and Goodwin 2011a</a>). Lower melanization was correlated with greater sensitivity to azole fungicides in one of the mutants (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Mehrabi <em>et al.</em> 2006a</a>), but in another study a knockout of a different gene (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Mehrabi <em>et al.</em> 2006b</a>) showed lower melanization as well as lower sensitivity to phenylpyrrole and dicarboximide fungicides. Hence, the typical pattern of greater melanization correlating with greater virulence and lesser fungicide sensitivity is not clearly established in <em>Z. tritici</em>.</p><p id="p-6">Melanization is likely a quantitative trait in <em>Z. tritici</em> because several genes were associated with the trait in other pathogenic fungi (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Takano <em>et al.</em> 1995</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Eliahu <em>et al.</em> 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Karkowska-Kuleta <em>et al.</em> 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fujihara <em>et al.</em> 2010</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ipcho <em>et al.</em> 2012</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lin <em>et al.</em> 2012</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Upadhyay <em>et al.</em> 2013</a>) and some gene knockouts caused only reduced melanin production (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kimura <em>et al.</em> 2001</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Parisot <em>et al.</em> 2002</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Solomon <em>et al.</em> 2004</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lin <em>et al.</em> 2006</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Mehrabi <em>et al.</em> 2009</a>). We tested this hypothesis by using QTL mapping to determine the genetic architecture of melanization in <em>Z. tritici</em>. QTL mapping enables the identification and characterization of the chromosomal segments and corresponding genes that encode quantitative traits (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Mackay 2001</a>). QTL mapping has been used extensively in animals (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Goddard and Hayes 2009</a>), including humans (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Flint and Mackay 2009</a>) and also plants (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Holland 2007</a>), but very few QTL mapping studies have been reported for filamentous fungi (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Foulongne-Oriol 2012</a>). Although fungal melanin biosynthetic pathways were characterized in earlier studies, additional melanin genes were identified in recent studies (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Fujihara <em>et al.</em> 2010</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Choi and Goodwin 2011a</a>,<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">b</a>), indicating that melanin biosynthetic pathways may be more complex than previously thought. QTL mapping was applied to fungal melanization in an earlier study with <em>Cryptococcus neoformans</em> (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Lin <em>et al.</em> 2006</a>), but this is the first QTL analysis of melanization in a fungal plant pathogen.</p><p id="p-7">We used two mapping populations derived from four unique wild-type strains to identify QTL involved in melanization. Because melanin production can be affected by stress (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Butler and Day 1998</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Henson <em>et al.</em> 1999</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jacobson 2000</a>), we exposed each offspring to two stressful environments, cold temperature and sublethal fungicide exposure, that could be compared with a control environment to determine whether temperature or fungicide stress would result in environment-dependent QTL. We chose a next-generation sequencing genotyping method called restriction site−associated DNA sequencing (RADseq) to construct two highly dense genetic maps using single-nucleotide polymorphism (SNP) markers covering most of the reference genome. Phenotyping was based on high-throughput digital image processing to score the melanization of single-spore colonies grown on solid media.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Generation of mapping populations</h3><p id="p-8">Two crosses were made between four <em>Z. tritici</em> isolates. Isolate ST99CH3D1 (3D1: SRS383146) was crossed to ST99CH3D7 (3D7: SRS383147), and ST99CH1A5 (1A5: SRS383142) was crossed to ST99CH1E4 (1E4: SRS383143). All four parents were collected from naturally infected wheat fields in 1999 in Switzerland. The isolates were previously characterized phenotypically (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Zhan <em>et al.</em> 2002</a>) and genetically (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Zhan <em>et al.</em> 2002</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Croll <em>et al.</em> 2013</a>) and found to differ for virulence and several life history traits, including melanization. Crosses were performed by coinfecting wheat leaves via an established protocol (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kema <em>et al.</em> 1996</a>). Ascospores were collected, grown <em>in vitro</em>, and stored on anhydrous silica gel and glycerol at −80°. Cross 3D1 × 3D7 produced 359 progeny and cross 1A5 × 1E4 produced 341 progeny.</p></div><div id="sec-3" class="subsection"><h3>Genotyping</h3><p id="p-9">We used RADseq (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baird <em>et al.</em> 2008</a>) to identify SNP markers segregating in the progeny populations. RADseq SNPs in the parental strains were confirmed using complete parental genome sequences obtained previously (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Croll <em>et al.</em> 2013</a>). National Center for Biotechnology Information (NCBI) BioSample accession numbers of parental strains were as follows: SRS383146 (ST99CH3D1), SRS383147 (ST99CH3D7), SRS383142 (ST99CH1A5), and SRS383143 (ST99CH1E4). To construct libraries, spores were lyophilized, and DNA was extracted using the DNeasy plant mini kit (QIAGEN Inc., Basel, Switzerland). DNA was then quantified and standardized for all samples. A total of 1.5 μg of genomic DNA per progeny was digested with the restriction enzyme <em>Pst</em>1. Libraries and pools were constructed following a modified RADseq protocol (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Etter <em>et al.</em> 2011</a>). The main modification was to use Illumina TrueSeq compatible P2 adapters. Pools containing an average of 132 progeny were generated, with each pool consisting of six different P2 adapters. 22 P1 adapters with distinct inline barcodes were used to distinguish progeny DNA with an identical P2 adapter. Pools were sequenced on an Illumina HiSeq2000 in 100 bp paired-end mode.</p><p id="p-10">Raw sequence reads were quality checked using the tool FastQC (Babraham Bioinformatics; Cambridge, UK). Reads were then quality trimmed with Trimmomatic v. 0.30 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lohse <em>et al.</em> 2012</a>) by using the following settings upon phred + 33 quality scores: trailing = 3, slidingwindow = 20:5 and minlen = 50. Progeny reads were separated according to the P1 adapter using the FASTX-Toolkit v. 0.13 (Hannon Lab: <a href="http://hannonlab.cshl.edu/fastx_toolkit/links.html">http://hannonlab.cshl.edu/fastx_toolkit/links.html</a>). Progeny reads were aligned individually against the IPO323 reference genome (assembly version MG2, Sept 2008) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Goodwin <em>et al.</em> 2011</a>) using the short-read aligner Bowtie 2 version 2.0.2 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Langmead and Salzberg 2012</a>) to create progeny sequence alignment/map (sam) files. Default settings for a sensitive end-to-end alignment were used (-D 15; -R 2; -L 22; -I S, 1,1.15). All four parental genome sequences (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Croll <em>et al.</em> 2013</a>) also were aligned to the reference genome using identical trimming and assembly parameters. Parental and progeny sam files were converted into binary alignment/map files (<em>i.e.</em>, bam) using SAMtools version 0.1.18 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2009</a>). RADseq progeny-aligned read data are available from the NCBI Short Read Archive under the BioProject accession numbers PRJNA256988 (cross: ST99CH3D1 × ST99CH3D7) and PRJNA256991 (cross: ST99CH1A5 × ST99CH1E4).</p><p id="p-11">SNPs were identified using the Genome Analysis Toolkit (GATK), version 2.6-4-g3e5ff60 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Depristo <em>et al.</em> 2011</a>) and VCFtools version 0.1.10 (<a href="http://vcftools.sourceforge.net">http://vcftools.sourceforge.net</a>). Initial SNP calling was performed in comparison with the reference genome using the GATK UnifiedGenotyper with a maximum alternative allele setting of 1. The sample-level was set to ploidy of 1 (haploid) using the sequence alignment/map files of progeny of one of the crosses combined with their respective parents. The genotype likelihood model was set to the SNP general ploidy model. Marker genotype SNP filtering was performed using GATK VariantFiltration, by setting the following filters for SNPs to pass: quality by depth (QD ≥ 5), Fisher’s exact test for strand bias (FS ≤ 60), haplotype score (HaplotypeScore ≤ 10), overall quality score (QUAL ≥ 1000), lower and upper allele frequency (AFlower ≥ 0.2 and AFupper ≤ 0.8), and total number of alleles within each marker genotype (AN ≥ 60). This filter excluded SNPs only called among the parental genomes. Progeny and parent genotypes were then filtered further using a phred-scaled genotype quality setting of at least 30.</p></div><div id="sec-4" class="subsection"><h3>Genetic map construction and quality assessment</h3><p id="p-12">The construction of the linkage map was performed using R/qtl version 1.27-10 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arends <em>et al.</em> 2010</a>), a package of the open-source software R (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">R_Core_Team 2012</a>). A genotype matrix was constructed containing only progeny with a minimum of 45% of all SNPs genotyped. We further omitted any markers if less than 70% of the progeny were genotyped. Potential clones in the progeny populations were excluded by randomly selecting one progeny from a group of potential clones. Clones were defined as having at least 90% of SNP alleles in common. Adjacent nonrecombining markers were reduced to retain only a single marker for each cluster of nonrecombining markers. We excluded any markers showing significant evidence of non-Mendelian segregation (χ<sup>2</sup> test: <em>P</em> &lt; 0.1), but no progeny genotypes matched the omission criteria for segregation distortion. Progeny genotypes were further investigated for evidence of genotyping errors (error.prob &lt; 0.01). Markers with significant evidence for genotyping error in at least one progeny were excluded from all progeny. NCBI Short Read Archive accession numbers for the retained progeny within each cross can be found in <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/015289SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS1.pdf">Table S1</a>.</p><p id="p-13">The genetic map was constructed by estimating the genetic distance among pairs of neighboring markers and translating recombination frequency into map distance (centiMorgans, cM). We checked for appropriate linkage group assignment, order and binary status among the retained markers by inspecting each genetic map for unusual inflation patterns. We constructed a plot of pairwise marker recombination fractions and logarithm of odds (LOD) scores for tests of r = 1/2 (heat map). A plot with a diagonal red line indicates a good assignment with consistent marker order and an absence of switched alleles. We compared the chromosome coverage of our linkage groups relative to the reference genome IPO323.</p></div><div id="sec-5" class="subsection"><h3>Phenotyping</h3><p id="p-14">Melanization was measured using a Petri dish assay combined with digital image analysis. Progeny of each cross were retrieved from long-term storage and grown on yeast malt sucrose agar (4 g/L yeast extract, 4 g/L malt extract, 4 g/L sucrose, and 50 mg/L kanamycin). Petri dishes were stored for approximately 4 d at 18°. A total of 600 μL of sterile water was added to each plate, and blastospores were gently scraped into the solution using a sterile glass slide. A total of 450 μL of the highly concentrated spore suspension were then transferred into sterile 500-μL Eppendorf tubes and stored for no longer then 3 mo at −20°. Spores of each offspring were taken from these tubes and diluted in sterile water to a concentration of 200 spores per milliliter using a hemocytometer. A total of 500 μL of the spore suspension was spread across Petri plates containing potato dextrose agar (PDA; 4 g/L potato starch, 20 g/L dextrose, 15 g/L agar) using a sterile glass rod. Only single-spore colonies were scored. An average of 20 colonies formed on each plate, but only nine individual colonies were scored on average because ~50% of all colonies had fused with a neighboring colony. Each treatment produced the same average number of colonies.</p><p id="p-15">Each progeny was exposed to two stress environments (cold temperature and sublethal fungicide concentration) and one control environment, with five technical repeats for each environment. The data point from a technical repeat was the average gray value from an average of nine colonies scored on one Petri dish. The control environment consisted of PDA plates growing at 22°. The cold-stress environment was PDA plates growing at 15°, whereas the fungicide-stress environment was PDA plates containing 0.75 ppm propiconazole (Syngenta, Basel, Switzerland) growing at 22°.</p><p id="p-16">After inoculation, Petri plates were dried for 30 min in a laminar flow cabinet and sealed using Parafilm. Plates were then randomized in a growth chamber set to a constant temperature with 70% humidity and no light. Plates were photographed at 8, 11, and 14 days postinoculation (dpi) for image analysis. These time points were chosen because they represented different colony ages and provided the largest phenotypic variance in preliminary experiments. This experimental design resulted in nine unique Environment-Colony-Age-Melanization (ECAM) phenotypes for each isolate, which we will refer to as ECAM phenotypes.</p><p id="p-17">Digital images were captured through the Petri dish lid using standardized camera settings and lighting environments (see <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS2.pdf">Table S2</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS3.pdf">Table S3</a>, and <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/FigureS1.pdf">Figure S1</a>). After images were acquired, Petri dishes were rerandomized and returned to the growth chamber for further colony growth and later image acquisition. Images were processed using a batch macro developed in the open-source software ImageJ (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Schneider <em>et al.</em> 2012</a>), enabling automated image analysis. The macro identified and scored individual colonies in the images (see <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/FileS1.pdf">File S1</a>). Melanization was measured using average gray values of individual colonies. Gray values range from 0 to 255, with 0 representing black and 255 representing white (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We used untransformed gray values as phenotypic values representing different degrees of colony melanization. We calculated broad-sense heritability (H<sup>2</sup>) values (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burton and Devane 1953</a>) over means of the five technical repeats for each environment and colony age using a one-way analysis of variance model in R (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">R_Core_Team 2012</a>). The melanization phenotypes for the retained progeny within each cross can be found in <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS4.pdf">Table S4</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Colony melanization was measured by digital image analyses of grayscale values ranging from 0 (black) to 255 (white). Images from the control environment at 14 dpi for representative progeny from the cross 1A5 &#xD7; 1E4 are positioned onto the scale to illustrate the different degrees of melanization observed in the crosses. Each image is labeled with the corresponding progeny name (top) and the corresponding gray value (bottom) as measured in the images. The gray values shown represent the full phenotypic variance found under the control environment at 14 dpi in the progeny from the cross 1A5 &#xD7; 1E4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1427346362" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Colony melanization was measured by digital image analyses of grayscale values ranging from 0 (black) to 255 (white). Images from the control environment at 14 dpi for representative progeny from the cross 1A5 &#xD7; 1E4 are positioned onto the scale to illustrate the different degrees of melanization observed in the crosses. Each image is labeled with the corresponding progeny name (top) and the corresponding gray value (bottom) as measured in the images. The gray values shown represent the full phenotypic variance found under the control environment at 14 dpi in the progeny from the cross 1A5 &#xD7; 1E4.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2519/F1.medium.gif" width="440" height="135"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/12/2519/F1.medium.gif" width="440" height="135"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465420" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Colony melanization was measured by digital image analyses of grayscale values ranging from 0 (black) to 255 (white). Images from the control environment at 14 dpi for representative progeny from the cross 1A5 × 1E4 are positioned onto the scale to illustrate the different degrees of melanization observed in the crosses. Each image is labeled with the corresponding progeny name (top) and the corresponding gray value (bottom) as measured in the images. The gray values shown represent the full phenotypic variance found under the control environment at 14 dpi in the progeny from the cross 1A5 × 1E4.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>QTL mapping</h3><p id="p-19">QTL mapping was performed in R/qtl version 1.27-10 (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Arends <em>et al.</em> 2010</a>) using single marker analysis combined with interval mapping, resulting in simple interval mapping (SIM) analysis. The analysis was based on progeny mean values, calculated over the five technical repeats for each environment and colony age, so that nine different ECAM phenotypes were generated for each cross (three environments × three colony ages). Interval mapping improves on the marker regression method by estimating markers (pseudomarkers) in between true markers. For SIM analysis we used the EM algorithm. Significance thresholds were determined by permutating the marker data. We applied genome-wide permutations, with 1000 permutations for each ECAM phenotype. We considered only QTL peaks with LOD scores that provided <em>P</em>-values lower then 0.05. We used the reference IPO323 genome to convert cM positions of markers into base pair (bp) positions on the reference genome. For 95% confidence interval calculations, we used Bayesian credible intervals from SIM combined with the EM algorithm (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Manichaikul <em>et al.</em> 2006</a>). Marker mean differences and the amount of variance explained by significant markers were calculated using true markers and not pseudomarkers. We assumed that our marker alleles were additive because we did not detect any significant epistasis.</p></div><div id="sec-7" class="subsection"><h3>Identification of candidate genes within QTL confidence intervals</h3><p id="p-20">To identify candidate genes within QTL confidence intervals, we first identified sequence variants among the parents in each cross by comparing their genome sequences, which had been previously aligned against the IPO323 reference genome. SNPs associated with synonymous and nonsynonymous mutations were called using the GATK UnifiedGenotyper with a maximum alternative allele setting of 2. Filter settings in the GATK VariantFiltration were as follows: QD ≥ 5, FS ≤ 60, HaplotypeScore ≤ 10.0, QUAL ≥ 100, AFlower ≥ 0.2, AFupper ≤ 0.8, and AN ≥ 2. For all other sequence variants, we used default settings of the GATK tools (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Depristo <em>et al.</em> 2011</a>). SNP and other sequence variation annotation results among the parental strains within each cross were obtained using the open-source tools SnpEff and SnpSift version 3.3h (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cingolani <em>et al.</em> 2012</a>). Synonymous SNPs were not considered in further analyses. Genes without Gene Ontology annotations were described as having unknown functions. Genes were considered as candidate genes within a confidence interval if they contained at least one sequence variant within the boundaries of the confidence interval, excluding genes with no sequence variation or with only synonymous SNPs.</p><p id="p-21">A total of 29 genes involved in fungal melanin biosynthesis have been described in the literature. Orthologs (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS5.pdf">Table S5</a>) for each of these genes were identified in <em>Z. tritici</em> by performing a BlastP search in the National Center for Biotechnology Information nonredundant protein database (<a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>) and conducting phylogenetic analyses of the amino acid sequences using the maximum likelihood algorithm with default parameters in MEGA5 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Tamura <em>et al.</em> 2011</a>). All QTL confidence intervals were searched for presence of the identified orthologs, which were considered as candidate melanization genes.</p><p id="p-22">RNA-Seq data were obtained for parent 3D7 as described previously (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brunner <em>et al.</em> 2013</a>). In summary, RNA-Seq data were obtained during an in planta infection time course that included all major phases of the pathogen life cycle (<em>i.e.</em>, biotrophic, necrotrophic, and saprotrophic phases), including three biological repeats at 7, 13, and 56 dpi. Reads per kilobase per million per gene values were calculated by normalizing the RNA reads and mapping them onto the reference genome. Significant differences in transcript abundances were calculated as described previously (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Brunner <em>et al.</em> 2013</a>). Transcription profiles were obtained for the orthologs to genes involved in melanin biosynthesis as well as candidate genes identified within QTL confidence intervals.</p></div><div id="sec-8" class="subsection"><h3>Amino acid differences in <em>PKS1</em> among parents 3D1 and 3D7</h3><p id="p-23">We identified synonymous and nonsynonymous substitutions in the gene <em>PKS1</em> (ProteinID 96592) encoding a polyketide synthase that catalyzes the first step of the DHN melanin biosynthetic pathway. Sequence polymorphism was scored in a Swiss field population of 25 <em>Z. tritici</em> isolates. Nine of the 25 isolates, including the four parental strains, had their genomes sequenced in a previous study (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Croll <em>et al.</em> 2013</a>), whereas complete genome sequences of 16 additional isolates (3A2, 3A4, 3A5, 3A6, 3A8, 3A10, 3B2, 3B4, 3C7, 3D3, 3D5, 3F1, 3F3, 3F4, 3G3, and 3H1) were obtained using the same methods for this study. SNPs among the isolates were called and filtered using the GATK tools (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Depristo <em>et al.</em> 2011</a>) with similar settings as applied for the RADseq dataset. Introns as well as synonymous and nonsynonymous SNPs were manually curated.</p><p id="p-24">A BlastP search in the National Center for Biotechnology Information non-redundant protein database (<a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>) was performed to investigate sequence conservation among ascomycetes. The <em>PKS1</em> amino acid sequence of the <em>Z. tritici</em> reference genome isolate IPO323 was aligned against 10 ascomycetes, including <em>Pseudocercospora fijiensis</em>, <em>Cladosporium phlei</em>, <em>Elsinoe fawcettii</em>, <em>Leptosphaeria maculans</em>, <em>Alternaria alternata</em>, <em>Phaeosphaeria nodorum</em>, <em>Pyrenophora tritici-repentis</em>, <em>Magnaporthe oryzae</em>, <em>Neurospora crassa</em>, and <em>Aspergillus nidulans</em>. The Pfam database was used to identify functional <em>PKS1</em> protein domains (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Finn <em>et al.</em> 2014</a>).</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Genetic maps</h3><p id="p-25">We retained 263 unique progeny and 9745 SNP markers in the 3D1 × 3D7 cross. A total of 96 progeny were excluded because of missing genotypes (18) or clonality (78). The quality-filtered dataset in the 1A5 × 1E4 cross included 261 progeny and 7333 SNP markers; 80 progeny were excluded because of missing genotypes (14) or clonality (66). <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> summarize the results for each genetic map.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465447/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465447/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genetic map summary for the cross between <em>Zymoseptoria tritici</em> isolates 3D1 and 3D7</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465413/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465413/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genetic map summary for the cross between <em>Zymoseptoria tritici</em> isolates 1A5 and 1E4</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">Marker density was high with an average genetic spacing of 0.44 cM, representing an average physical spacing of ~3600 bp between SNP markers in cross 3D1 × 3D7 and average spacing of 0.71 cM (~4700 bp) in cross 1A5 × 1E4. On average, our genetic maps covered around 90% of the reference genome over both crosses. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the genetic maps as well as plots of pairwise marker recombination fractions and LOD scores for tests of r = 0.5, which indicate a distorted diagonal red line for chromosome 13 in both crosses, indicating marker order inconsistencies. Chromosome 13 likely exhibits map inflation in both crosses, indicated by an unusual genetic length (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) and larger gaps on the genetic map (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A, B) Genetic maps and (C, D) pairwise linkage comparisons of markers for each cross (Top: 3D1 &#xD7; 3D7; Bottom: 1A5 &#xD7; 1E4). (C, D) Within the pairwise linkage comparison plot marker-pairwise recombination fractions are shown in the upper left triangle and LOD scores for tests of r = 1/2 are shown in the lower right triangle. Red corresponds to a large LOD or a small recombination fraction, while blue is the reverse. Thus red indicates linkage, while blue indicates no linkage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1427346362" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A, B) Genetic maps and (C, D) pairwise linkage comparisons of markers for each cross (Top: 3D1 &#xD7; 3D7; Bottom: 1A5 &#xD7; 1E4). (C, D) Within the pairwise linkage comparison plot marker-pairwise recombination fractions are shown in the upper left triangle and LOD scores for tests of r = 1/2 are shown in the lower right triangle. Red corresponds to a large LOD or a small recombination fraction, while blue is the reverse. Thus red indicates linkage, while blue indicates no linkage.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2519/F2.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/12/2519/F2.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465443" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">(A, B) Genetic maps and (C, D) pairwise linkage comparisons of markers for each cross (Top: 3D1 × 3D7; Bottom: 1A5 × 1E4). (C, D) Within the pairwise linkage comparison plot marker-pairwise recombination fractions are shown in the upper left triangle and LOD scores for tests of r = 1/2 are shown in the lower right triangle. Red corresponds to a large LOD or a small recombination fraction, while blue is the reverse. Thus red indicates linkage, while blue indicates no linkage.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The eight smallest chromosomes (14 to 21) in the <em>Z. tritici</em> reference genome are accessory chromosomes (ACs) (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Goodwin <em>et al.</em> 2011</a>) that often exhibit presence/absence polymorphisms (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Croll <em>et al.</em> 2013</a>). In the 3D1 × 3D7 cross, the parental genome sequences revealed that ACs 14, 15, 18, and 21 were missing in one of the parents, hence only ACs 16, 17, 19, and 20 were mapped. In the 1A5 × 1E4 cross, AC 17 was missing in one of the parents and therefore only ACs 14, 15, 16, 18, 19, 20, and 21 were mapped.</p></div><div id="sec-11" class="subsection"><h3>Melanization is a quantitative trait that shows transgressive segregation</h3><p id="p-29">For both crosses over all three environments and all three colony ages, we found that melanization showed a continuous distribution consistent with a quantitative character (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/FigureS2.pdf">Figure S2</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/FigureS3.pdf">Figure S3</a>). Many progeny had more extreme phenotypes than the parents, indicating transgressive segregation. The degree of melanization increased over time for most isolates in most environments, indicating that melanin concentration increases as colonies age.</p><p id="p-30">Reaction norms over the three colony ages differed among the progeny in each cross, providing evidence for differential interactions between genotypes and environments over time. To illustrate this, we compared the two progeny showing the most extreme phenotypes at 14 dpi in one environment with their corresponding phenotypes in the other two environments (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/FigureS4.pdf">Figure S4</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/FigureS5.pdf">Figure S5</a>). In both crosses we found strong evidence for phenotype-by-environment interaction over all colony ages. Hence, we analyzed each colony age separately in the QTL analyses.</p></div><div id="sec-12" class="subsection"><h3>Comparison of mapped QTL reveals unique and shared QTL between the two crosses</h3><p id="p-31">We found six significant QTL in cross 3D1 × 3D7 and nine significant QTL in cross 1A5 × 1E4 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). QTL were considered to be different if the confidence intervals did not overlap. This approach was applied across environments and colony ages as well as within crosses and among crosses. Three QTL were shared between the two crosses. Shared QTL were further characterized based on the 3D1 × 3D7 cross. A detailed description of all significant QTL found in both crosses is given in <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS6.pdf">Table S6</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS7.pdf">Table S7</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS8.pdf">Table S8</a>, and <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS9.pdf">Table S9</a>. No QTL were found on ACs.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL peaks (red dots) and associated confidence intervals for all significant QTL detected in the experiment. Cross 3D1 &#xD7; 3D7 is shown in the upper half and cross 1A5 &#xD7; 1E4 is shown in the lower half of the figure. Environments are color-coded: green = fungicide stress environment; orange = control environment; purple = cold stress environment. Colony age increases within each environment from the bottom up. The different sizes of red dots represent the relative size of the associated LOD scores. The 12 cross-specific QTL are marked with a green triangle. Orthologs involved in melanin biosynthesis found within 12 cross-specific QTL confidence intervals as candidate genes are indicated with names in blue positioned above the corresponding chromosome. Chromosome sizes are presented in mega base pairs (Mb). QTL, quantitative trait loci; LOD, logarithm of odds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1427346362" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;QTL peaks (red dots) and associated confidence intervals for all significant QTL detected in the experiment. Cross 3D1 &#xD7; 3D7 is shown in the upper half and cross 1A5 &#xD7; 1E4 is shown in the lower half of the figure. Environments are color-coded: green = fungicide stress environment; orange = control environment; purple = cold stress environment. Colony age increases within each environment from the bottom up. The different sizes of red dots represent the relative size of the associated LOD scores. The 12 cross-specific QTL are marked with a green triangle. Orthologs involved in melanin biosynthesis found within 12 cross-specific QTL confidence intervals as candidate genes are indicated with names in blue positioned above the corresponding chromosome. Chromosome sizes are presented in mega base pairs (Mb). QTL, quantitative trait loci; LOD, logarithm of odds.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2519/F3.medium.gif" width="440" height="408"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/12/2519/F3.medium.gif" width="440" height="408"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465487" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">QTL peaks (red dots) and associated confidence intervals for all significant QTL detected in the experiment. Cross 3D1 × 3D7 is shown in the upper half and cross 1A5 × 1E4 is shown in the lower half of the figure. Environments are color-coded: green = fungicide stress environment; orange = control environment; purple = cold stress environment. Colony age increases within each environment from the bottom up. The different sizes of red dots represent the relative size of the associated LOD scores. The 12 cross-specific QTL are marked with a green triangle. Orthologs involved in melanin biosynthesis found within 12 cross-specific QTL confidence intervals as candidate genes are indicated with names in blue positioned above the corresponding chromosome. Chromosome sizes are presented in mega base pairs (Mb). QTL, quantitative trait loci; LOD, logarithm of odds.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465398/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465398/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Positions and effects of the six cross-specific QTL identified in cross 3D1 × 3D7</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465308/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465308/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Positions and effects of the six cross-specific QTL identified in cross 1A5 × 1E4</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Environment and colony age affect QTL mapping results</h3><p id="p-41">In both crosses we found environment-specific QTL as well as QTL that were shared across all environments. In cross 3D1 × 3D7, five QTL were environment-specific and a QTL on chromosome 11 was found in all environments. In cross 1A5 × 1E4, six QTL were environment-specific and three QTL were found in all environments (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-42">Colony age also affects QTL detection for some environments in both crosses. For example, in cross 3D1 × 3D7 the chromosome 5 QTL was found in colonies that were 11 and 14 d old, whereas the chromosome 8 QTL was found only in colonies that were 11 d old. In the 1A5 × 1E4 cross, the QTL on chromosomes 5 and 8 were significant in colonies that were 11 and 14 d old, but the QTL on chromosomes 6, 7 and 11 were found only in colonies that were 14 d old (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>).</p></div><div id="sec-14" class="subsection"><h3>Genetic architecture of melanization in the two crosses</h3><p id="p-43">We found differences in the genetic architecture of melanization among the two crosses. On average, cross 3D1 × 3D7 had fewer (1.8) melanization QTL for each colony age and environment than cross 1A5 × 1E4 (3.6); however, the average total variance explained by the QTL for each ECAM phenotype was similar (~35%) in both crosses. In cross 3D1 × 3D7 nearly half of the total phenotypic variance was explained by one QTL on chromosome 11, whereas in cross 1A5 × 1E4 a similar amount of variance was explained by six QTL distributed across several chromosomes, suggesting a more complex genetic basis of melanin production in cross 1A5 × 1E4 compared with cross 3D1 × 3D7 (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465311/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465311/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Phenotypic contributions and distribution of all significant QTL</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Identification of candidate genes and orthologous genes related to melanization in QTL regions</h3><p id="p-47">A summary of the candidate genes found within each confidence interval for cross 3D1 × 3D7 and 1A5 × 1E4 is given in <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS8.pdf">Table S8</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS9.pdf">Table S9</a>, respectively. On average we found ~190 candidate genes per confidence interval, with a minimum of one candidate gene (14 dpi, fungicide environment, 1A5 × 1E4) and a maximum of 1245 genes (8 dpi, control environment, 1A5 × 1E4). Approximately 32% of the candidate genes found in a confidence interval had no known function and ~42% showed significant changes in transcript abundance across biotrophic, necrotrophic, and saprotrophic stages in the life cycle. Eleven orthologs of the 29 genes described to be involved in melanin biosynthesis (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS5.pdf">Table S5</a>) were identified as candidate genes in four of the 12 cross-specific QTL confidence intervals (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Nine of the 29 orthologs were found in three of the QTL in cross 3D1 × 3D7, and two were found in one of the QTL in cross 1A5 × 1E4 (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). These 11 orthologs encode two (<em>MgSlt2</em> and <em>MgFus3</em>) mitogen-activated protein kinases, one (<em>CLAP1</em>) copper-transporting ATPase, two (<em>LAC4</em> and <em>LAC8</em>) laccases, one (<em>PRF1</em>) prefolding chaperone, one (<em>PKS1</em>) polyketide synthase, one (<em>MgGpa1</em>) G alpha protein, as well as two (<em>PEX6</em> and <em>PEX13</em>) genes involved in peroxisome biosynthesis and one (<em>AYG2</em>) of an unknown function (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS5.pdf">Table S5</a>). Three (<em>MgSlt2</em>, <em>MgFus3</em> and <em>MgGpa1</em>) of these 11 genes were associated with melanization in <em>Z. tritici</em> in earlier studies (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Cousin <em>et al.</em> 2006</a>; <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Mehrabi <em>et al.</em> 2006a</a>, <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">2009</a>), but the other eight genes have not previously been associated with melanization in <em>Z. tritici</em>.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465357/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465357/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Genes associated with melanization that were identified as candidate genes within the six cross-specific QTL in each cross</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">To narrow the search for candidate genes affecting melanization, we focused on major QTL (LOD &gt; 7.5) containing ≤30 candidate genes in their confidence intervals. We identified two confidence intervals meeting these criteria in each cross (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). For cross 3D1 × 3D7, the QTL on chromosome 10 for the 11 dpi cold environment had 12 candidate genes and the QTL on chromosome 11 for the 8 dpi control environment had four candidate genes (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS10.pdf">Table S10</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). For cross 1A5 × 1E4, the QTL on chromosome 4 for the 14 dpi fungicide environment had one candidate gene and the QTL on chromosome 8 for the 11 dpi cold environment had 22 candidate genes (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS11.pdf">Table S11</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Taking into account all of the available information (gene ontology, the predicted impact of observed sequence variation, the position of the QTL peak, gene expression) as shown in <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS10.pdf">Table S10</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS11.pdf">Table S11</a>, we further narrowed the list down to eight high-priority candidate genes in the 3D1 × 3D7 cross and eight high-priority candidate genes in the 1A5 × 1E4 cross, resulting in a total of 16 high-priority candidate genes over both crosses.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465407/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465407/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Large-effect QTL intervals containing ≤30 candidate genes</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LOD plots from the single marker interval mapping analysis over all chromosomes for the phenotypes that provided the peaks with confidence interval containing the smallest number (&#x2264; 30) of candidate genes. The horizontal dashed red line in the LOD plots represents the significance threshold (P = 0.05) obtained by 1000 genome-wide permutations. Below each LOD plot is the confidence interval (gray line) for the peak with the fewest candidate genes (gene bars), which are positioned within the confidence interval. The genes are color coded according to the predicted impact by SnpEff of the observed sequence variation [white = modifier, light gray = low (no case within the confidence interval shown), gray = moderate, black = high] and labeled with their protein ID just above the gene bars. Below the gene bars are vertical lines, representing the position of each sequence variant within a gene. The asterisk symbol represents the position of the peaking marker within the confidence interval. Orthologs to genes involved in melanin biosynthesis found as candidate genes within the confidence interval are represented by their name in blue, just above the corresponding protein ID. High-priority candidate genes are colored in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1427346362" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;LOD plots from the single marker interval mapping analysis over all chromosomes for the phenotypes that provided the peaks with confidence interval containing the smallest number (&#x2264; 30) of candidate genes. The horizontal dashed red line in the LOD plots represents the significance threshold (&lt;em&gt;P&lt;/em&gt; = 0.05) obtained by 1000 genome-wide permutations. Below each LOD plot is the confidence interval (gray line) for the peak with the fewest candidate genes (gene bars), which are positioned within the confidence interval. The genes are color coded according to the predicted impact by SnpEff of the observed sequence variation [white = modifier, light gray = low (no case within the confidence interval shown), gray = moderate, black = high] and labeled with their protein ID just above the gene bars. Below the gene bars are vertical lines, representing the position of each sequence variant within a gene. The asterisk symbol represents the position of the peaking marker within the confidence interval. Orthologs to genes involved in melanin biosynthesis found as candidate genes within the confidence interval are represented by their name in blue, just above the corresponding protein ID. High-priority candidate genes are colored in red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2519/F4.medium.gif" width="420" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/12/2519/F4.medium.gif" width="420" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465455" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-55" class="first-child">LOD plots from the single marker interval mapping analysis over all chromosomes for the phenotypes that provided the peaks with confidence interval containing the smallest number (≤ 30) of candidate genes. The horizontal dashed red line in the LOD plots represents the significance threshold (<em>P</em> = 0.05) obtained by 1000 genome-wide permutations. Below each LOD plot is the confidence interval (gray line) for the peak with the fewest candidate genes (gene bars), which are positioned within the confidence interval. The genes are color coded according to the predicted impact by SnpEff of the observed sequence variation [white = modifier, light gray = low (no case within the confidence interval shown), gray = moderate, black = high] and labeled with their protein ID just above the gene bars. Below the gene bars are vertical lines, representing the position of each sequence variant within a gene. The asterisk symbol represents the position of the peaking marker within the confidence interval. Orthologs to genes involved in melanin biosynthesis found as candidate genes within the confidence interval are represented by their name in blue, just above the corresponding protein ID. High-priority candidate genes are colored in red.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><div id="sec-17" class="subsection"><h3>Genetic maps</h3><p id="p-56">The two genetic maps have very high marker density compared with other reported genetic maps in filamentous fungi (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Foulongne-Oriol 2012</a>). The high marker density reflects the high degree of recombination in the mapping populations as well as the large number of markers provided by our genotyping method. Earlier genetic maps based on the genome reference isolate IPO323 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kema <em>et al.</em> 2002</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Wittenberg <em>et al.</em> 2009</a>) were ~72% smaller than our genetic maps. We hypothesize that our maps are larger due to greater recombination resulting from larger numbers of offspring and more extensive chromosome coverage resulting from larger numbers of genetic markers. Our maps covered ~90% of the reference genome on average. Incomplete coverage can be explained by missing restriction sites or low quality mapping reads toward one or both telomeres of each chromosome. Other explanations include a lack of recombining polymorphic markers or lower polymorphism toward the telomeres. The ratio of physical distance to genetic distance was ~7.5 kb/cM for both crosses, comparable with the ratio of 8.1 kb/cM described in <em>Aspergillus nidulans</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Christians <em>et al.</em> 2011</a>).</p><p id="p-57">The unusual genetic length (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) and larger gaps on the genetic map of chromosome 13 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) are consistent with map inflation resulting from incorrect marker order, most likely due to inappropriate alignment of our illumina reads onto the reference genome. We hypothesize that this could be due to the presence of the mating type idiomorphs (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Waalwijk <em>et al.</em> 2002</a>) on this chromosome, the occurrence of transposable elements or because of incorrect reference genome assembly. As chromosome 13 coverage was good (88% in cross 3D1 × 3D7 and 90% for cross 1A5 × 1E4) and no QTL were found on this chromosome for either cross, this issue was not investigated further.</p></div><div id="sec-18" class="subsection"><h3>Melanization is a quantitative trait that is affected by environment and colony age</h3><p id="p-58">The finding that all ECAM phenotypes showed a continuous distribution (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/FigureS2.pdf">Figure S2</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/FigureS3.pdf">Figure S3</a>) and high broad-sense heritability (74–95%; <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/FigureS4.pdf">Figure S4</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/FigureS5.pdf">Figure S5</a>) in both crosses indicates that several genes are likely to contribute to the melanization phenotype in <em>Z. tritici</em>. We found 15 significant QTL in total, with more than one significant QTL identified for 13 out of 18 ECAM phenotypes (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). On average, a QTL contributed ~15% of the phenotypic variance observed in each cross. The pattern of transgressive segregation is also consistent with the hypothesis that melanization is a quantitative trait. Our findings stand in contrast to an earlier melanization mapping study (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Lin <em>et al.</em> 2006</a>) that identified only one significant QTL associated with several melanin phenotypes in a mapping population.</p><p id="p-59">Our experimental design allowed us to investigate the effects of both colony age and environment on the melanization phenotype. We were able to demonstrate both colony age and environmental effects, as reflected by unique QTL, within both mapping populations. These findings are consistent with the hypothesis that genes affecting melanization are differentially regulated in different environments and at different stages of colony development.</p></div><div id="sec-19" class="subsection"><h3>Unique QTL and shared QTL were identified by comparing QTL of the two crosses</h3><p id="p-60">We identified three shared QTL and nine QTL that were unique to one of the crosses. For the three shared QTL (found on chromosomes 1, 5, and 11), we compared the candidate genes found in the corresponding confidence intervals and found that more than 40% of the candidate genes in these confidence intervals were shared among the two crosses, suggesting that the shared QTL are likely to be due to same source of genetic variance in each cross. Among the 15 identified QTL, we postulate that the six QTL in <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a> are the most likely to be replicated in other QTL mapping studies, based on having a combination of high LOD scores (≥ 7) and being found at least twice over different colony ages and/or environments.</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465382/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465382/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">Most reproducible QTL in the two crosses</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Identification of orthologs to genes involved in melanin biosynthesis within QTL intervals</h3><p id="p-62">All the previously identified orthologs of genes involved in melanin biosynthesis (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS5.pdf">Table S5</a>) were expressed (≥ 2 reads per kilobase per million), hence all of the orthologs could affect melanization in <em>Z. tritici</em>. Eleven of the 29 orthologs were found within four of the 12 cross-specific QTL confidence intervals (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a> and <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>) identified in our analyses. We postulate that several of these known genes contributed to the melanization phenotypes in our crosses. Among the 11 orthologous candidates, three [<em>MgSlt2</em> (<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Mehrabi <em>et al.</em> 2006a</a>), <em>MgGpa1</em> (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Solomon <em>et al.</em> 2004</a>; <a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Mehrabi <em>et al.</em> 2009</a>) and <em>MgFus3</em> (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Cousin <em>et al.</em> 2006</a>)] were functionally validated in <em>Z. tritici</em>, but none of these have been investigated as contributing to a quantitative character. The other eight candidates have not yet been analyzed in <em>Z. tritici</em>, but our QTL analyses indicate that these genes may also affect melanization in <em>Z. tritici</em>. After gene disruption, eight of these 11 orthologous candidates exhibited a measurable change in mycelial melanization <em>in vitro</em>, and three (<em>PEX6</em>, <em>LAC4</em>, and <em>LAC8</em>) affected melanization of appressoria but not mycelia in Colletotrichum spp (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Kimura <em>et al.</em> 2001</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lin <em>et al.</em> 2012</a>). None of the 18 remaining orthologs was found as candidate genes in the 12 cross-specific QTL confidence intervals.</p></div><div id="sec-21" class="subsection"><h3>QTL confidence intervals contain both novel and known candidate genes affecting melanization</h3><p id="p-63">Three of the 16 high-priority candidate genes carried no Gene Ontology annotation and thus have no known function. These candidates could represent novel transcription factors, enzymes, or structural proteins that affect the melanization phenotype. The remaining 13 genes had Gene Ontology annotations. Seven of the 16 high-priority candidate genes showed significant changes in transcript abundance across biotrophic, necrotrophic, and saprotrophic stages of the life cycle, consistent with expected changes in melanin production across the pathogen life cycle. Melanin production is expected to increase during the development of the black pycnidia that form early in the saprotrophic stage of the life cycle (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Eyal <em>et al.</em> 1987</a>).</p><p id="p-64">Within three of the four major-effect QTL confidence intervals, none of the candidate genes is an ortholog of genes known to be involved in melanin biosynthesis; thus, these represent novel candidates for genes affecting fungal melanization. Gene 92291 encodes a transcription factor and was the only candidate gene within the QTL confidence interval on chromosome 4. We hypothesize that this transcription factor regulates genes under fungicide stress because this QTL was found only in the fungicide environment.</p><p id="p-65">The major-effect QTL on chromosome 11 contains three high-priority candidate genes and had the greatest LOD score (32.2) among the 12 cross specific QTL over both crosses (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Two of the three genes were orthologous to genes known to be involved in melanin biosynthesis (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS5.pdf">Table S5</a>), namely <em>PRF1</em> (Protein ID: 96591) and <em>PKS1</em> (Protein ID: 96592). The third gene (96588) is involved in zinc binding and also was considered a high-priority candidate because the melanin biosynthesis ortholog <em>CMR1</em> (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Eliahu <em>et al.</em> 2007</a>) (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS5.pdf">Table S5</a>) encodes a transcription factor with a zinc finger (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Tsuji <em>et al.</em> 2000</a>) and is located 28 kb upstream from 96588. We hypothesize that gene 96588 is involved in zinc homeostasis of <em>CMR1</em> and ultimately affects melanin biosynthesis; however, our analyses of sequence diversity led us to conclude that <em>PKS1</em> is the major contributor to the phenotypic variance explained by the large effect QTL on chromosome 11.</p></div><div id="sec-22" class="subsection"><h3>Identification of a candidate quantitative trait nucleotide in <em>PKS1</em></h3><p id="p-66"><em>PKS1</em> is the polyketide synthase enzyme catalyzing the first step of DHN melanin synthesis through head-to-tail joining and cyclization of acetate molecules (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Takano <em>et al.</em> 1995</a>; <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Butler and Day 1998</a>) in the DHN melanin biosynthetic pathway. <em>PKS1</em> carries three nonsynonymous mutations among parents 3D1 and 3D7 while <em>PRF1</em> and 96588 contain 1 and 0 nonsynonymous mutations, respectively (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS10.pdf">Table S10</a>). The region on chromosome 11 containing <em>PKS1</em> comprises the main DHN melanin biosynthetic gene cluster. We interpret this finding as a validation of our QTL mapping approach to identify genes affecting melanization. DHN melanin, which is the best characterized melanin in fungi (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Butler and Day 1998</a>), has so far not been studied in <em>Z. tritici</em> (<a href="http://www.g3journal.org/content/suppl/2014/10/29/g3.114.015289.DC1/TableS5.pdf">Table S5</a>). The identification of <em>PKS1</em> as a highly probable candidate gene within the QTL having the largest LOD score among the 12 cross-specific QTL suggests that DHN melanin plays a major role in melanization in <em>Z. tritici</em>.</p><p id="p-67">Three nonsynonymous substitutions at amino acid positions 155, 884, and 1783 were found in <em>PKS1</em> among parents 3D1 and 3D7, but positions 884 and 1783 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) were considered as more likely candidates to explain differences in melanization because position 155 was not in a functional domain and the alternative amino acids did not differ for polarity, acidity (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Taylor 1986</a>) or hydropathy index (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kyte and Doolittle 1982</a>). Fifteen of the 25 field isolates had an alanine at position 884 while 10 isolates carried a valine at that position. At position 1783, only the 3D7 parent had a threonine residue while the other 24 isolates had a proline (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Position 1783 was more conserved among ascoymcetes than site 884, but neither site was located within an annotated functional domain of <em>PKS1</em> (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). The amino acid property changes were most striking for site 1783, because proline is nonpolar while threonine is polar. In addition, proline is unique among the 20 proteinogenic amino acids because its side group links to the amino group, often resulting in strong effects on protein secondary structure.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PKS1 amino acid alterations between parent 3D1 and 3D7 and a schematic diagram of the entire PKS1 gene (2175 amino acids). The two investigated amino acid alterations at position 884 and 1783 are indicated on the gene, showing their positions relative to the predicted functional domain sites. The lower panel shows polymorphisms present among 11 ascomycetes as well as 25 genetically distinct Z. tritici field isolates from Switzerland." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1427346362" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;PKS1&lt;/em&gt; amino acid alterations between parent 3D1 and 3D7 and a schematic diagram of the entire &lt;em&gt;PKS1&lt;/em&gt; gene (2175 amino acids). The two investigated amino acid alterations at position 884 and 1783 are indicated on the gene, showing their positions relative to the predicted functional domain sites. The lower panel shows polymorphisms present among 11 ascomycetes as well as 25 genetically distinct &lt;em&gt;Z. tritici&lt;/em&gt; field isolates from Switzerland.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2519/F5.medium.gif" width="431" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/12/2519/F5.medium.gif" width="431" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2519/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465412" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-68" class="first-child"><em>PKS1</em> amino acid alterations between parent 3D1 and 3D7 and a schematic diagram of the entire <em>PKS1</em> gene (2175 amino acids). The two investigated amino acid alterations at position 884 and 1783 are indicated on the gene, showing their positions relative to the predicted functional domain sites. The lower panel shows polymorphisms present among 11 ascomycetes as well as 25 genetically distinct <em>Z. tritici</em> field isolates from Switzerland.</p><div class="sb-div caption-clear"></div></div></div><p id="p-69">Based on these analyses of <em>PKS1</em> sequence polymorphism, we postulate that the nonsynonymous mutation found at amino acid 1783 explains most of the phenotypic variance associated with the large-effect QTL on chromosome 11. If this is confirmed, then the associated SNP represents a quantitative trait nucleotide (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fridman <em>et al.</em> 2004</a>) that explains the majority of the phenotypic variance in this cross. Functional validation will be needed to confirm this hypothesis.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-70">We thank Marc-Henri Lebrun (UR BIOGER, INRA, France) for information on the annotation of melanin biosynthesis genes. RADseq libraries were constructed at the Genetic Diversity Centre (GDC) and sequenced in the Quantitative Genomics Facility at the Department of Biosystems Science and Engineering (D-BSSE) at the scientific central facilities of ETH Zurich. The research was supported by a grant from the Swiss National Science Foundation (31003A_134755). Technical help was provided by Michael Mielewczik and Tryggvi S. Stefansson. The RNA-Seq data were kindly provided by Stefano F. F. Torriani.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-19"><p id="p-71">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015289/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015289/-/DC1</a></p></li><li class="fn" id="fn-20"><p id="p-72"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-09-02"><span class="received-label">Received </span>September 2, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-10-28"><span class="accepted-label">Accepted </span>October 28, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Lendenmann <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.1" data-doi="10.1093/bioinformatics/btq565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arends</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prins</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">R/qtl: high-throughput multiple QTL mapping.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2990</span>–<span class="cit-lpage">2992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq565%26rft_id%253Dinfo%253Apmid%252F20966004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjMvMjk5MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.2" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced rad markers.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.3" data-doi="10.1093/molbev/mst041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brunner</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torriani</span>  <span class="cit-name-given-names">S. F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croll</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Coevolution and life cycle specialization of plant cell wall degrading enzymes in a hemibiotrophic pathogen.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1337</span>–<span class="cit-lpage">1347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst041%26rft_id%253Dinfo%253Apmid%252F23515261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMwLzYvMTMzNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.4" data-doi="10.2134/agronj1953.00021962004500100005x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devane</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1953</span> <span class="cit-article-title">Estimating heritability in tall fescue (<em>Festuca-arundinacea</em>) from replicated clonal material.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">478</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DBurton%26rft.auinit1%253DG.%2BW.%26rft.volume%253D45%26rft.issue%253D10%26rft.spage%253D478%26rft.epage%253D481%26rft.atitle%253DEstimating%2BHeritability%2Bin%2BTall%2BFescue%2B%2528Festuca%2BArundinacea%2529%2Bfrom%2BReplicated%2BClonal%2BMaterial%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1953.00021962004500100005x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1953.00021962004500100005x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.5" data-doi="10.1139/cjm-44-12-1115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Fungal melanins: a review.</span> <abbr class="cit-jnl-abbrev">Can. J. Microbiol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BMicrobiol.%26rft.volume%253D44%26rft.spage%253D1115%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fcjm-44-12-1115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/cjm-44-12-1115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000079370800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.6" data-doi="10.1094/MPMI-04-10-0090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">Y.-E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Gene encoding a C-type cyclin in <em>Mycosphaerella graminicola</em> is involved in aerial mycelium formation, filamentous growth, hyphal swelling, melanin biosynthesis, stress response, and pathogenicity.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DChoi%26rft.auinit1%253DY.%2BE.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D469%26rft.epage%253D477%26rft.atitle%253DGene%2Bencoding%2Ba%2Bc-type%2Bcyclin%2Bin%2BMycosphaerella%2Bgraminicola%2Bis%2Binvolved%2Bin%2Baerial%2Bmycelium%2Bformation%252C%2Bfilamentous%2Bgrowth%252C%2Bhyphal%2Bswelling%252C%2Bmelanin%2Bbiosynthesis%252C%2Bstress%2Bresponse%252C%2Band%2Bpathogenicity.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-04-10-0090%26rft_id%253Dinfo%253Apmid%252F21171890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-04-10-0090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21171890&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.7" data-doi="10.1128/AEM.01830-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">Y.-E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title"><em>MVE1</em>, encoding the velvet gene product homolog in <em>Mycosphaerella graminicola</em>, is associated with aerial mycelium formation, melanin biosynthesis, hyphal swelling and light signaling.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">942</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01830-10%26rft_id%253Dinfo%253Apmid%252F21115702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjg6Ijc3LzMvOTQyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.8" data-doi="10.1371/journal.pone.0019325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christians</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheema</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vergara</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watt</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative trait locus (QTL) mapping reveals a role for unstudied genes in <em>Aspergillus</em> virulence.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChristians%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De19325%26rft.epage%253De19325%26rft.atitle%253DQuantitative%2Btrait%2Blocus%2B%2528QTL%2529%2Bmapping%2Breveals%2Ba%2Brole%2Bfor%2Bunstudied%2Bgenes%2Bin%2BAspergillus%2Bvirulence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019325%26rft_id%253Dinfo%253Apmid%252F21559404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21559404&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.9" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain w(1118); iso-2; iso-3.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.10" data-doi="10.1111/j.1364-3703.2006.00337.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cousin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehrabi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guilleroux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dufresne</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van der Lee</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The MAP kinase-encoding gene <em>MgFus3</em> of the non-appressorium phytopathogen <em>Mycosphaerella graminicola</em> is required for penetration and in vitro pycnidia formation.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DCousin%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D269%26rft.epage%253D278%26rft.atitle%253DThe%2BMAP%2Bkinase-encoding%2Bgene%2BMgFus3%2Bof%2Bthe%2Bnon-appressorium%2Bphytopathogen%2BMycosphaerella%2Bgraminicola%2Bis%2Brequired%2Bfor%2Bpenetration%2Band%2Bin%2Bvitro%2Bpycnidia%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2006.00337.x%26rft_id%253Dinfo%253Apmid%252F20507446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2006.00337.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20507446&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238394100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.11" data-doi="10.1371/journal.pgen.1003567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Croll</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zala</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Breakage-fusion-bridge cycles and large insertions contribute to the rapid evolution of accessory chromosomes in a fungal pathogen.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003567%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003567%26rft_id%253Dinfo%253Apmid%252F23785303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003567&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23785303&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.12" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.13" data-doi="10.1128/EC.00264-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eliahu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Igbaria</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horwitz</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lev</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Melanin biosynthesis in the maize pathogen <em>Cochliobolus heterostrophus</em> depends on two mitogen-activated protein kinases, <em>Chk1</em> and <em>Mps1</em>, and the transcription factor <em>Cmr1</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00264-06%26rft_id%253Dinfo%253Apmid%252F17237364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI2LzMvNDIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.14" data-doi="10.1007/978-1-61779-228-1_9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SNP discovery and genotyping for evolutionary genetcs using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">772</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DEtter%26rft.auinit1%253DP.%2BD.%26rft.volume%253D772%26rft.spage%253D157%26rft.epage%253D178%26rft.atitle%253DSNP%2Bdiscovery%2Band%2Bgenotyping%2Bfor%2Bevolutionary%2Bgenetics%2Busing%2BRAD%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-228-1_9%26rft_id%253Dinfo%253Apmid%252F22065437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-228-1_9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22065437&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321876000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2519.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyal</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scharen</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prescott</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Ginkel</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-source">The Septoria Diseases of Wheat: Concepts and Methods of Disease Management</span>. <span class="cit-publ-name">CIMMYT</span>, <span class="cit-publ-loc">Mexico, DF</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.16" data-doi="10.1093/nar/gkt1223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bateman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coggill</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pfam: the protein families database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D222</span>–<span class="cit-lpage">D230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1223%26rft_id%253Dinfo%253Apmid%252F24288371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi9EMS9EMjIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.17" data-doi="10.1101/gr.086660.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic architecture of quantitative traits in mice, flies, and humans.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DFlint%26rft.auinit1%253DJ.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D723%26rft.epage%253D733%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bquantitative%2Btraits%2Bin%2Bmice%252C%2Bflies%252C%2Band%2Bhumans%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.086660.108%26rft_id%253Dinfo%253Apmid%252F19411597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzUvNzIzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.18" data-doi="10.1007/s00253-012-4228-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foulongne-Oriol</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic linkage mapping in fungi: current state, applications, and future trends.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">904</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DFoulongne-Oriol%26rft.auinit1%253DM.%26rft.volume%253D95%26rft.issue%253D4%26rft.spage%253D891%26rft.epage%253D904%26rft.atitle%253DGenetic%2Blinkage%2Bmapping%2Bin%2Bfungi%253A%2Bcurrent%2Bstate%252C%2Bapplications%252C%2Band%2Bfuture%2Btrends.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-012-4228-4%26rft_id%253Dinfo%253Apmid%252F22743715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-012-4228-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22743715&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.19" data-doi="10.1126/science.1101666"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fridman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernie</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Zooming in on a quantitative trait for tomato yield using interspecific introgressions.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">1786</span>–<span class="cit-lpage">1789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFridman%26rft.auinit1%253DE.%26rft.volume%253D305%26rft.issue%253D5691%26rft.spage%253D1786%26rft.epage%253D1789%26rft.atitle%253DZooming%2BIn%2Bon%2Ba%2BQuantitative%2BTrait%2Bfor%2BTomato%2BYield%2BUsing%2BInterspecific%2BIntrogressions%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1101666%26rft_id%253Dinfo%253Apmid%252F15375271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDUvNTY5MS8xNzg2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.20" data-doi="10.1094/MPMI-23-4-0436"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fujihara</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakaguchi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujii</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuji</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Peroxisome biogenesis factor <em>PEX13</em> is required for appressorium-mediated plant infection by the anthracnose fungus <em>Colletotrichum orbiculare</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DFujihara%26rft.auinit1%253DN.%26rft.volume%253D23%26rft.issue%253D4%26rft.spage%253D436%26rft.epage%253D445%26rft.atitle%253DPeroxisome%2Bbiogenesis%2Bfactor%2BPEX13%2Bis%2Brequired%2Bfor%2Bappressorium-mediated%2Bplant%2Binfection%2Bby%2Bthe%2Banthracnose%2Bfungus%2BColletotrichum%2Borbiculare.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-23-4-0436%26rft_id%253Dinfo%253Apmid%252F20192831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-23-4-0436&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20192831&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.21" data-doi="10.1038/nrg2575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping genes for complex traits in domestic animals and their use in breeding programmes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D381%26rft.epage%253D391%26rft.atitle%253DMapping%2Bgenes%2Bfor%2Bcomplex%2Btraits%2Bin%2Bdomestic%2Banimals%2Band%2Btheir%2Buse%2Bin%2Bbreeding%2Bprogrammes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2575%26rft_id%253Dinfo%253Apmid%252F19448663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448663&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.22" data-doi="10.1371/journal.pgen.1002070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben M’Barek</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittenberg</span>  <span class="cit-name-given-names">A. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crane</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Finished genome of the fungal wheat pathogen <em>Mycosphaerella graminicola</em> reveals dispensome structure, chromosome plasticity, and stealth pathogenesis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGoodwin%26rft.auinit1%253DS.%2BB.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002070%26rft.epage%253De1002070%26rft.atitle%253DFinished%2Bgenome%2Bof%2Bthe%2Bfungal%2Bwheat%2Bpathogen%2BMycosphaerella%2Bgraminicola%2Breveals%2Bdispensome%2Bstructure%252C%2Bchromosome%2Bplasticity%252C%2Band%2Bstealth%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002070%26rft_id%253Dinfo%253Apmid%252F21695235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21695235&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.23" data-doi="10.1046/j.1365-3059.2001.00596.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hardwick</span>  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slough</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Factors affecting diseases of winter wheat in England and Wales, 1989–98.</span> <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">453</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D50%26rft.spage%253D453%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-3059.2001.00596.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-3059.2001.00596.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.24" data-doi="10.1146/annurev.phyto.37.1.447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The dark side of the mycelium: Melanins of phytopathogenic fungi.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DHenson%26rft.auinit1%253DJ.%2BM%26rft.volume%253D37%26rft.issue%253D1%26rft.spage%253D447%26rft.epage%253D471%26rft.atitle%253DTHE%2BDARK%2BSIDE%2BOF%2BTHE%2BMYCELIUM%253A%2BMelanins%2Bof%2BPhytopathogenic%2BFungi%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.37.1.447%26rft_id%253Dinfo%253Apmid%252F11701831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.37.1.447&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11701831&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083446400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.25" data-doi="10.1016/j.pbi.2007.01.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic architecture of complex traits in plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DHolland%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D156%26rft.epage%253D161%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2007.01.003%26rft_id%253Dinfo%253Apmid%252F17291822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2007.01.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17291822&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245573100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.26" data-doi="10.1073/pnas.88.24.11281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrari</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roach</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Money</span>  <span class="cit-name-given-names">N. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Penetration of hard substrates by a fungus employing enormous turgor pressures.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">11281</span>–<span class="cit-lpage">11284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHoward%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D88%26rft.issue%253D24%26rft.spage%253D11281%26rft.epage%253D11284%26rft.atitle%253DPenetration%2Bof%2Bhard%2Bsubstrates%2Bby%2Ba%2Bfungus%2Bemploying%2Benormous%2Bturgor%2Bpressures.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.24.11281%26rft_id%253Dinfo%253Apmid%252F1837147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiODgvMjQvMTEyODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI1MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.27" data-doi="10.1111/j.1348-0421.2003.tb03395.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikeda</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugita</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinoda</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of melanin upon susceptibility of <em>Cryptococcus</em> to antifungals.</span> <abbr class="cit-jnl-abbrev">Microbiol. Immunol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2Bimmunology%26rft.stitle%253DMicrobiol%2BImmunol%26rft.aulast%253DIkeda%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D4%26rft.spage%253D271%26rft.epage%253D277%26rft.atitle%253DEffects%2Bof%2Bmelanin%2Bupon%2Bsusceptibility%2Bof%2BCryptococcus%2Bto%2Bantifungals.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1348-0421.2003.tb03395.x%26rft_id%253Dinfo%253Apmid%252F12801064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1348-0421.2003.tb03395.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12801064&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182506500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.28" data-doi="10.1111/j.1364-3703.2011.00770.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ipcho</span>  <span class="cit-name-given-names">S. V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antoni</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahren</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henrissat</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transcriptome analysis of <em>Stagonospora nodorum</em>: gene models, effectors, metabolism and pantothenate dispensability.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DIpcho%26rft.auinit1%253DS.%2BV.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D531%26rft.epage%253D545%26rft.atitle%253DTranscriptome%2Banalysis%2Bof%2BStagonospora%2Bnodorum%253A%2Bgene%2Bmodels%252C%2Beffectors%252C%2Bmetabolism%2Band%2Bpantothenate%2Bdispensability.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2011.00770.x%26rft_id%253Dinfo%253Apmid%252F22145589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2011.00770.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22145589&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306003000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.29" data-doi="10.1128/CMR.13.4.708-717.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Pathogenic roles for fungal melanins.</span> <abbr class="cit-jnl-abbrev">Clin. Microbiol. Rev.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">708</span>–<span class="cit-lpage">717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DJacobson%26rft.auinit1%253DE.%2BS.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D708%26rft.epage%253D717%26rft.atitle%253DPathogenic%2BRoles%2Bfor%2BFungal%2BMelanins%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.13.4.708-717.2000%26rft_id%253Dinfo%253Apmid%252F11023965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjg6IjEzLzQvNzA4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karkowska-Kuleta</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rapala-Kozik</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fungi pathogenic to humans: molecular bases of virulence of <em>Candida albicans</em>, <em>Cryptococcus neoformans</em> and <em>Aspergillus fumigatus</em>.</span> <abbr class="cit-jnl-abbrev">Acta Biochim. Pol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2Bbiochimica%2BPolonica%26rft.stitle%253DActa%2BBiochim%2BPol%26rft.aulast%253DKarkowska-Kuleta%26rft.auinit1%253DJ.%26rft.volume%253D56%26rft.issue%253D2%26rft.spage%253D211%26rft.epage%253D224%26rft.atitle%253DFungi%2Bpathogenic%2Bto%2Bhumans%253A%2Bmolecular%2Bbases%2Bof%2Bvirulence%2Bof%2BCandida%2Balbicans%252C%2BCryptococcus%2Bneoformans%2Band%2BAspergillus%2Bfumigatus.%26rft_id%253Dinfo%253Apmid%252F19543556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19543556&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267607200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.31" data-doi="10.1007/s002940050129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kema</span>  <span class="cit-name-given-names">G. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstappen</span>  <span class="cit-name-given-names">E. C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todorova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waalwijk</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Successful crosses and molecular tetrad and progeny analyses demonstrate heterothallism in <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DKema%26rft.auinit1%253DG.%2BH.%26rft.volume%253D30%26rft.issue%253D3%26rft.spage%253D251%26rft.epage%253D258%26rft.atitle%253DSuccessful%2Bcrosses%2Band%2Bmolecular%2Btetrad%2Band%2Bprogeny%2Banalyses%2Bdemonstrate%2Bheterothallism%2Bin%2BMycosphaerella%2Bgraminicola.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002940050129%26rft_id%253Dinfo%253Apmid%252F8753655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002940050129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8753655&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VD79200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kema</span>  <span class="cit-name-given-names">G. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamza</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstappen</span>  <span class="cit-name-given-names">E. C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavaletto</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A combined amplified fragment length polymorphism and randomly amplified polymorphism DNA genetic linkage map of <em>Mycosphaerella graminicola</em>, the septoria tritici leaf blotch pathogen of wheat.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">1497</span>–<span class="cit-lpage">1505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKema%26rft.auinit1%253DG.%2BH.%2BJ.%26rft.volume%253D161%26rft.issue%253D4%26rft.spage%253D1497%26rft.epage%253D1505%26rft.atitle%253DA%2BCombined%2BAmplified%2BFragment%2BLength%2BPolymorphism%2Band%2BRandomly%2BAmplified%2BPolymorphism%2BDNA%2BGenetic%2BLinkage%2BMap%2Bof%2BMycosphaerella%2Bgraminicola%252C%2Bthe%2BSeptoria%2BTritici%2BLeaf%2BBlotch%2BPathogen%2Bof%2BWheat%26rft_id%253Dinfo%253Apmid%252F12196395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MS80LzE0OTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI1MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.33" data-doi="10.1105/tpc.13.8.1945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takano</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furusawa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okuno</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Peroxisomal metabolic function is required for appressorium-mediated plant infection by <em>Colletotrichum lagenarium</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1945</span>–<span class="cit-lpage">1957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DKimura%26rft.auinit1%253DA.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D1945%26rft.epage%253D1957%26rft.atitle%253DPeroxisomal%2BMetabolic%2BFunction%2BIs%2BRequired%2Bfor%2BAppressorium-Mediated%2BPlant%2BInfection%2Bby%2BColletotrichum%2Blagenarium%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.13.8.1945%26rft_id%253Dinfo%253Apmid%252F11487704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEzLzgvMTk0NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.34" data-doi="10.1016/0022-2836(82)90515-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kyte</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doolittle</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">A simple method for displaying the hydropathic character of a protein.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKyte%26rft.auinit1%253DJ.%26rft.volume%253D157%26rft.issue%253D1%26rft.spage%253D105%26rft.epage%253D132%26rft.atitle%253DA%2Bsimple%2Bmethod%2Bfor%2Bdisplaying%2Bthe%2Bhydropathic%2Bcharacter%2Bof%2Ba%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252882%252990515-0%26rft_id%253Dinfo%253Apmid%252F7108955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(82)90515-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7108955&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982NQ28000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.35" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.36" data-doi="10.1016/0006-2952(79)90326-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tjalve</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Studies on the mechanism of drug-binding to melanin.</span> <abbr class="cit-jnl-abbrev">Biochem. Pharmacol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Bpharmacology%26rft.stitle%253DBiochem%2BPharmacol%26rft.aulast%253DLarsson%26rft.auinit1%253DB.%26rft.volume%253D28%26rft.issue%253D7%26rft.spage%253D1181%26rft.epage%253D1187%26rft.atitle%253DStudies%2Bon%2Bthe%2Bmechanism%2Bof%2Bdrug-binding%2Bto%2Bmelanin.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0006-2952%252879%252990326-5%26rft_id%253Dinfo%253Apmid%252F36092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0006-2952(79)90326-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=36092&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979GX07400041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.37" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.38" data-doi="10.1016/j.ijantimicag.2009.09.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liaw</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsueh</span>  <span class="cit-name-given-names">P. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Multidrug resistance in clinical isolates of <em>Stenotrophomonas maltophilia</em>: roles of integrons, efflux pumps, phosphoglucomutase (SpgM), and melanin and biofilm formation.</span> <abbr class="cit-jnl-abbrev">Int. J. Antimicrob. Agents</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bantimicrobial%2Bagents%26rft.stitle%253DInt%2BJ%2BAntimicrob%2BAgents%26rft.aulast%253DLiaw%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D126%26rft.epage%253D130%26rft.atitle%253DMultidrug%2Bresistance%2Bin%2Bclinical%2Bisolates%2Bof%2BStenotrophomonas%2Bmaltophilia%253A%2Broles%2Bof%2Bintegrons%252C%2Befflux%2Bpumps%252C%2Bphosphoglucomutase%2B%2528SpgM%2529%252C%2Band%2Bmelanin%2Band%2Bbiofilm%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijantimicag.2009.09.015%26rft_id%253Dinfo%253Apmid%252F19926255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijantimicag.2009.09.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19926255&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.39" data-doi="10.1094/MPMI-05-12-0131-R"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okuda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikeda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okuno</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takano</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>LAC2</em> encoding a secreted laccase is involved in appressorial melanization and conidial pigmentation in <em>Colletotrichum orbiculare</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1552</span>–<span class="cit-lpage">1561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DLin%26rft.auinit1%253DS.%2BY.%26rft.volume%253D25%26rft.issue%253D12%26rft.spage%253D1552%26rft.epage%253D1561%26rft.atitle%253DLAC2%2BEncoding%2Ba%2BSecreted%2BLaccase%2BIs%2BInvolved%2Bin%2BAppressorial%2BMelanization%2Band%2BConidial%2BPigmentation%2Bin%2BColletotrichum%2Borbiculare.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-05-12-0131-R%26rft_id%253Dinfo%253Apmid%252F22934563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-05-12-0131-R&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22934563&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311186600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">X. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heitman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Virulence attributes and hyphal growth of <em>C. neoformans</em> are quantitative traits and the MAT alpha allele enhances filamentation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D2%26rft.spage%253D1801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.41" data-doi="10.1093/nar/gks540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernie</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lunn</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">RobiNA: a user-friendly, integrated software solution for RNA-Seq-based transcriptomics.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">W622</span>–<span class="cit-lpage">W627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks540%26rft_id%253Dinfo%253Apmid%252F22684630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9XMS9XNjIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.42" data-doi="10.1146/annurev.genet.35.102401.090633"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The genetic architecture of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%2BC.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D303%26rft.epage%253D339%26rft.atitle%253DTHE%2BGENETIC%2BARCHITECTURE%2BOF%2BQUANTITATIVE%2BTRAITS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.35.102401.090633%26rft_id%253Dinfo%253Apmid%252F11700286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.35.102401.090633&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11700286&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172991500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.43" data-doi="10.1534/genetics.106.061549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manichaikul</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupuis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Poor performance of bootstrap confidence intervals for the location of a quantitative trait locus.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061549%26rft_id%253Dinfo%253Apmid%252F16783000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc0LzEvNDgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.44" data-doi="10.1128/IAI.73.4.2012-2019.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mednick</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nosanchuk</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casadevall</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Melanization of <em>Cryptococcus neoformans</em> affects lung inflammatory responses during cryptococcal infection.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">2012</span>–<span class="cit-lpage">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DMednick%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D73%26rft.issue%253D4%26rft.spage%253D2012%26rft.epage%253D2019%26rft.atitle%253DMelanization%2Bof%2BCryptococcus%2Bneoformans%2BAffects%2BLung%2BInflammatory%2BResponses%2Bduring%2BCryptococcal%2BInfection%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.73.4.2012-2019.2005%26rft_id%253Dinfo%253Apmid%252F15784542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6IjczLzQvMjAxMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.45" data-doi="10.1094/MPMI-19-0389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mehrabi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Lee</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waalwijk</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kema</span>  <span class="cit-name-given-names">G. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title"><em>MgSlt2</em>, a cellular integrity MAP kinase gene of the fungal wheat pathogen <em>Mycosphaerella graminicola</em>, is dispensable for penetration but essential for invasive growth.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DMehrabi%26rft.auinit1%253DR.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D389%26rft.epage%253D398%26rft.atitle%253DMgSlt2%252C%2Ba%2Bcellular%2Bintegrity%2BMAP%2Bkinase%2Bgene%2Bof%2Bthe%2Bfungal%2Bwheat%2Bpathogen%2BMycosphaerella%2Bgraminicola%252C%2Bis%2Bdispensable%2Bfor%2Bpenetration%2Bbut%2Bessential%2Bfor%2Binvasive%2Bgrowth.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-19-0389%26rft_id%253Dinfo%253Apmid%252F16610742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-19-0389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16610742&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236263600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.46" data-doi="10.1094/MPMI-19-1262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mehrabi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zwiers</span>  <span class="cit-name-given-names">L.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Waard</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kema</span>  <span class="cit-name-given-names">G. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title"><em>MgHog1</em> regulates dimorphism and pathogenicity in the fungal wheat pathogen <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1262</span>–<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DMehrabi%26rft.auinit1%253DR.%26rft.volume%253D19%26rft.issue%253D11%26rft.spage%253D1262%26rft.epage%253D1269%26rft.atitle%253DMgHog1%2Bregulates%2Bdimorphism%2Band%2Bpathogenicity%2Bin%2Bthe%2Bfungal%2Bwheat%2Bpathogen%2BMycosphaerella%2Bgraminicola.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-19-1262%26rft_id%253Dinfo%253Apmid%252F17073308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-19-1262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17073308&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241424300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.47" data-doi="10.1128/EC.00258-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mehrabi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben M’Barek</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Lee</span>  <span class="cit-name-given-names">T. A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waalwijk</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Wit</span>  <span class="cit-name-given-names">P. J. G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">G alpha and G beta proteins regulate the cyclic AMP pathway that is required for development and pathogenicity of the phytopathogen <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1001</span>–<span class="cit-lpage">1013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00258-08%26rft_id%253Dinfo%253Apmid%252F19411619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI4LzcvMTAwMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.48" data-doi="10.1128/IAI.71.7.4026-4033.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morris-Jones</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Youngchim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aisen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hay</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Synthesis of melanin-like pigments by <em>Sporothrix schenckii</em> in vitro and mammalian infection.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">4026</span>–<span class="cit-lpage">4033</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DMorris-Jones%26rft.auinit1%253DR.%26rft.volume%253D71%26rft.issue%253D7%26rft.spage%253D4026%26rft.epage%253D4033%26rft.atitle%253DSynthesis%2Bof%2BMelanin-Like%2BPigments%2Bby%2BSporothrix%2Bschenckii%2BIn%2BVitro%2Band%2Bduring%2BMammalian%2BInfection%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.71.7.4026-4033.2003%26rft_id%253Dinfo%253Apmid%252F12819091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6IjcxLzcvNDAyNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ngamskulrungroj</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Melanin production at 37 degrees C is linked to the high virulent <em>Cryptococcus gattii</em> Vancouver Island outbreak genotype VGIIa.</span> <abbr class="cit-jnl-abbrev">Australas. Mycologist</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAustralas.%2BMycologist%26rft.volume%253D28%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.50" data-doi="10.1128/AAC.00545-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nosanchuk</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casadevall</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Impact of melanin on microbial virulence and clinical resistance to antimicrobial compounds.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">3519</span>–<span class="cit-lpage">3528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrobial%2BAgents%2Band%2BChemotherapy%26rft.stitle%253DAntimicrob.%2BAgents%2BChemother.%26rft.aulast%253DNosanchuk%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D50%26rft.issue%253D11%26rft.spage%253D3519%26rft.epage%253D3528%26rft.atitle%253DImpact%2Bof%2BMelanin%2Bon%2BMicrobial%2BVirulence%2Band%2BClinical%2BResistance%2Bto%2BAntimicrobial%2BCompounds%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.00545-06%26rft_id%253Dinfo%253Apmid%252F17065617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjEwOiI1MC8xMS8zNTE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.51" data-doi="10.1016/S0140-6736(00)02356-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nosanchuk</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosas</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casadevall</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Melanisation of <em>Cryptococcus neoformans</em> in human brain tissue.</span> <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">355</span>: <span class="cit-fpage">2049</span>–<span class="cit-lpage">2050</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253DNosanchuk%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D355%26rft.issue%253D9220%26rft.spage%253D2049%26rft.epage%253D2050%26rft.atitle%253DMelanisation%2Bof%2BCryptococcus%2Bneoformans%2Bin%2Bhuman%2Bbrain%2Btissue.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252800%252902356-4%26rft_id%253Dinfo%253Apmid%252F10885360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(00)02356-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10885360&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087519400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.52" data-doi="10.1016/j.tplants.2014.04.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Driscoll</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kildea</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doohan</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullins</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The wheat–Septoria conflict: a new front opening up?</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">602</span>–<span class="cit-lpage">610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D602%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.04.011%26rft_id%253Dinfo%253Apmid%252F24957882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.04.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24957882&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.53" data-doi="10.1007/s00438-002-0744-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parisot</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dufresne</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veneault</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lauge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langin</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title"><em>Clap1</em>, a gene encoding a copper-transporting ATPase involved in the process of infection by the phytopathogenic fungus <em>Colletotrichum lindemuthianum</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">268</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DParisot%26rft.auinit1%253DD.%26rft.volume%253D268%26rft.issue%253D2%26rft.spage%253D139%26rft.epage%253D151%26rft.atitle%253Dclap1%252C%2Ba%2Bgene%2Bencoding%2Ba%2Bcopper-transporting%2BATPase%2Binvolved%2Bin%2Bthe%2Bprocess%2Bof%2Binfection%2Bby%2Bthe%2Bphytopathogenic%2Bfungus%2BColletotrichum%2Blindemuthianum.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-002-0744-8%26rft_id%253Dinfo%253Apmid%252F12395188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-002-0744-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12395188&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-web" id="cit-4.12.2519.54"><div class="cit-metadata unstructured">R_Core_Team, 2012 R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna, Austria. Available at: <a href="http://www.R-project.org/">http://www.R-project.org/</a>. Accessed: November 5, 2014.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.55" data-doi="10.1038/nmeth.2089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasband</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eliceiri</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">NIH Image to ImageJ: 25 years of image analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchneider%26rft.auinit1%253DC.%2BA.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D671%26rft.epage%253D675%26rft.atitle%253DNIH%2BImage%2Bto%2BImageJ%253A%2B25%2Byears%2Bof%2Bimage%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2089%26rft_id%253Dinfo%253Apmid%252F22930834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22930834&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305942200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.56" data-doi="10.1094/MPMI.2004.17.5.456"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The disruption of a G alpha subunit sheds new light on the pathogenicity of <em>Stagonospora nodorum</em> on wheat.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">456</span>–<span class="cit-lpage">466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DSolomon%26rft.auinit1%253DP.%2BS.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D456%26rft.epage%253D466%26rft.atitle%253DThe%2Bdisruption%2Bof%2Ba%2BGalpha%2Bsubunit%2Bsheds%2Bnew%2Blight%2Bon%2Bthe%2Bpathogenicity%2Bof%2BStagonospora%2Bnodorum%2Bon%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI.2004.17.5.456%26rft_id%253Dinfo%253Apmid%252F15141949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI.2004.17.5.456&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15141949&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221016700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.57" data-doi="10.1007/s11046-007-9061-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taborda</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">da Silva</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nosanchuk</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travassos</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Melanin as a virulence factor of <em>Paracoccidioides brasiliensis</em> and other dimorphic pathogenic fungi: a minireview.</span> <abbr class="cit-jnl-abbrev">Mycopathologia</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycopathologia%26rft.stitle%253DMycopathologia%26rft.aulast%253DTaborda%26rft.auinit1%253DC.%2BP.%26rft.volume%253D165%26rft.issue%253D4-5%26rft.spage%253D331%26rft.epage%253D339%26rft.atitle%253DMelanin%2Bas%2Ba%2Bvirulence%2Bfactor%2Bof%2BParacoccidioides%2Bbrasiliensis%2Band%2Bother%2Bdimorphic%2Bpathogenic%2Bfungi%253A%2Ba%2Bminireview.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11046-007-9061-4%26rft_id%253Dinfo%253Apmid%252F18777637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11046-007-9061-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18777637&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255256100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.58" data-doi="10.1007/BF00290362"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takano</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mise</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okuno</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Structural-analysis of <em>PKS1</em>, a polyketide synthase gene involved in melanin biosynthesis in <em>Colletotrichum lagenarium</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">249</span>: <span class="cit-fpage">162</span>–<span class="cit-lpage">167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DTakano%26rft.auinit1%253DY.%26rft.volume%253D249%26rft.issue%253D2%26rft.spage%253D162%26rft.epage%253D167%26rft.atitle%253DStructural%2Banalysis%2Bof%2BPKS1%252C%2Ba%2Bpolyketide%2Bsynthase%2Bgene%2Binvolved%2Bin%2Bmelanin%2Bbiosynthesis%2Bin%2BColletotrichum%2Blagenarium.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00290362%26rft_id%253Dinfo%253Apmid%252F7500937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00290362&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7500937&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.59" data-doi="10.1093/molbev/msr121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2731</span>–<span class="cit-lpage">2739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr121%26rft_id%253Dinfo%253Apmid%252F21546353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xMC8yNzMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.60" data-doi="10.1016/S0022-5193(86)80075-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">The classification of amino-acid conservation.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DTaylor%26rft.auinit1%253DW.%2BR.%26rft.volume%253D119%26rft.issue%253D2%26rft.spage%253D205%26rft.epage%253D218%26rft.atitle%253DThe%2Bclassification%2Bof%2Bamino%2Bacid%2Bconservation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-5193%252886%252980075-3%26rft_id%253Dinfo%253Apmid%252F3461222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-5193(86)80075-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3461222&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986A790500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.61" data-doi="10.1046/j.1365-2958.2000.02181.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsuji</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenmochi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takano</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweigard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farrall</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Novel fungal transcriptional activators, Cmr1p of <em>Colletotrichum lagenarium</em> and Pig1p of <em>Magnaporthe grisea</em>, contain Cys2His2 zinc finger and Zn(II)2Cys6 binuclear cluster DNA-binding motifs and regulate transcription of melanin biosynthesis genes in a developmentally specific manner.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">940</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DTsuji%26rft.auinit1%253DG.%26rft.volume%253D38%26rft.issue%253D5%26rft.spage%253D940%26rft.epage%253D954%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2000.02181.x%26rft_id%253Dinfo%253Apmid%252F11123670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2000.02181.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11123670&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166576500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.62" data-doi="10.1128/EC.00217-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Upadhyay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Laccases Involved in 1,8-dihydroxynaphthalene melanin biosynthesis in <em>Aspergillus fumigatus</em> are regulated by developmental factors and copper homeostasis.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1641</span>–<span class="cit-lpage">1652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00217-13%26rft_id%253Dinfo%253Apmid%252F24123270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czoxMDoiMTIvMTIvMTY0MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjUxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.63" data-doi="10.1006/fgbi.2001.1322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waalwijk</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendes</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstappen</span>  <span class="cit-name-given-names">E. C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Waard</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kema</span>  <span class="cit-name-given-names">G. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Isolation and characterization of the mating-type idiomorphs from the wheat septoria leaf blotch fungus <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DWaalwijk%26rft.auinit1%253DC.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D277%26rft.epage%253D286%26rft.atitle%253DIsolation%2Band%2Bcharacterization%2Bof%2Bthe%2Bmating-type%2Bidiomorphs%2Bfrom%2Bthe%2Bwheat%2Bseptoria%2Bleaf%2Bblotch%2Bfungus%2BMycosphaerella%2Bgraminicola.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ffgbi.2001.1322%26rft_id%253Dinfo%253Apmid%252F11929216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/fgbi.2001.1322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11929216&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.64" data-doi="10.1371/journal.pone.0005863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wittenberg</span>  <span class="cit-name-given-names">A. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Lee</span>  <span class="cit-name-given-names">T. A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben M’Barek</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ware</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Meiosis drives extraordinary genome plasticity in the haploid fungal plant pathogen <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5863</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWittenberg%26rft.auinit1%253DA.%2BH.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De5863%26rft.epage%253De5863%26rft.atitle%253DMeiosis%2Bdrives%2Bextraordinary%2Bgenome%2Bplasticity%2Bin%2Bthe%2Bhaploid%2Bfungal%2Bplant%2Bpathogen%2BMycosphaerella%2Bgraminicola.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005863%26rft_id%253Dinfo%253Apmid%252F19516898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005863&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19516898&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.65" data-doi="10.1099/jmm.0.05421-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Youngchim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris-Jones</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hay</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Production of melanin by <em>Aspergillus fumigatus</em>.</span> <abbr class="cit-jnl-abbrev">J. Med. Microbiol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMedical%2BMicrobiology%26rft.stitle%253DJ%2BMed%2BMicrobiol%26rft.aulast%253DYoungchim%26rft.auinit1%253DS.%26rft.volume%253D53%26rft.issue%253D3%26rft.spage%253D175%26rft.epage%253D181%26rft.atitle%253DProduction%2Bof%2Bmelanin%2Bby%2BAspergillus%2Bfumigatus%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fjmm.0.05421-0%26rft_id%253Dinfo%253Apmid%252F14970241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/jmm.0.05421-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14970241&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220667900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2519.66" data-doi="10.1016/S1087-1845(02)00013-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kema</span>  <span class="cit-name-given-names">G. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waalwijk</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Distribution of mating type alleles in the wheat pathogen <em>Mycosphaerella graminicola</em> over spatial scales from lesions to continents.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">128</span>–<span class="cit-lpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DZhan%26rft.auinit1%253DJ.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D128%26rft.epage%253D136%26rft.atitle%253DDistribution%2Bof%2Bmating%2Btype%2Balleles%2Bin%2Bthe%2Bwheat%2Bpathogen%2BMycosphaerella%2Bgraminicola%2Bover%2Bspatial%2Bscales%2Bfrom%2Blesions%2Bto%2Bcontinents.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1087-1845%252802%252900013-0%26rft_id%253Dinfo%253Apmid%252F12081466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1087-1845(02)00013-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12081466&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2519.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/12/2519.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/12/2505" title="Spatial Localization of Recent Ancestors for Admixed Individuals" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/12/2535" title="Regulation of Synaptic Transmission at the Caenorhabditis elegans M4 Neuromuscular Junction by an Antagonistic Relationship Between Two Calcium Channels" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/12"><span>Volume 4 Issue 12, December 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/12.cover-source.jpg?itok=ezho2p8P" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2519" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Zn0Wq72impAPBaN6vKhOaMMn5Ci-2IxOsmjK6LSfuLA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/12/2519" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/12/2519" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/12/2519" class="active" data-icon-position="" data-hide-link-title="0">Quantitative Trait Locus Mapping of Melanization in the Plant Pathogenic Fungus Zymoseptoria tritici</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459743" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Fm6qFEu_CI02_Cd4ANxt-Ebf6Z7ENMqFO--jYsYnRK4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/12/2519.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/12/2519" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-FYUNWKYG3YTSNKS__g1C6fn9sBfnr3eBp2rkQLyocBo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/12/2519" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/12/2519.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459743" id="node459743--2" data-pisa="ggg;4/12/2519" data-pisa-master="ggg;g3.114.015289" data-apath="/ggg/4/12/2519.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Trait Locus Mapping of Melanization in the Plant Pathogenic Fungus <em>Zymoseptoria tritici</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mark H. Lendenmann</span>, <span class="highwire-citation-author" data-delta="1">Daniel Croll</span>, <span class="highwire-citation-author" data-delta="2">Ethan L. Stewart</span> and <span class="highwire-citation-author" data-delta="3">Bruce A. McDonald</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2519-2533;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015289 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459743--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mark H. Lendenmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BH.%2BLendenmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lendenmann%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BH.%2BLendenmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mark.lendenmann@usys.ethz.ch" class="" data-icon-position="" data-hide-link-title="0">mark.lendenmann@usys.ethz.ch</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Croll </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BCroll%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Croll%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BCroll%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ethan L. Stewart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ethan%2BL.%2BStewart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stewart%20EL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEthan%2BL.%2BStewart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bruce A. McDonald </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruce%2BA.%2BMcDonald%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McDonald%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruce%2BA.%2BMcDonald%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/12/2519" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/12/2519&amp;title=Quantitative%20Trait%20Locus%20Mapping%20of%20Melanization%20in%20the%20Plant%20Pathogenic%20Fungus%20Zymoseptoria%20tritici" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/12/2519&amp;title=Quantitative%20Trait%20Locus%20Mapping%20of%20Melanization%20in%20the%20Plant%20Pathogenic%20Fungus%20Zymoseptoria%20tritici" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/12/2519&amp;title=Quantitative%20Trait%20Locus%20Mapping%20of%20Melanization%20in%20the%20Plant%20Pathogenic%20Fungus%20Zymoseptoria%20tritici" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/12/2519&amp;text=Quantitative%20Trait%20Locus%20Mapping%20of%20Melanization%20in%20the%20Plant%20Pathogenic%20Fungus%20Zymoseptoria%20tritici" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/12/2519&amp;title=Quantitative%20Trait%20Locus%20Mapping%20of%20Melanization%20in%20the%20Plant%20Pathogenic%20Fungus%20Zymoseptoria%20tritici" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/12/2519&amp;t=Quantitative%20Trait%20Locus%20Mapping%20of%20Melanization%20in%20the%20Plant%20Pathogenic%20Fungus%20Zymoseptoria%20tritici" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/12/2519&amp;title=Quantitative%20Trait%20Locus%20Mapping%20of%20Melanization%20in%20the%20Plant%20Pathogenic%20Fungus%20Zymoseptoria%20tritici" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/12/2519&amp;title=Quantitative%20Trait%20Locus%20Mapping%20of%20Melanization%20in%20the%20Plant%20Pathogenic%20Fungus%20Zymoseptoria%20tritici" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459743/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459743" id="node459743--3" data-pisa="ggg;4/12/2519" data-pisa-master="ggg;g3.114.015289" data-apath="/ggg/4/12/2519.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Quantitative Trait Locus Mapping of Melanization in the Plant Pathogenic Fungus <em>Zymoseptoria tritici</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mark H. Lendenmann</span>, <span class="highwire-citation-author" data-delta="1">Daniel Croll</span>, <span class="highwire-citation-author" data-delta="2">Ethan L. Stewart</span> and <span class="highwire-citation-author" data-delta="3">Bruce A. McDonald</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2519-2533;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015289 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459743--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mark H. Lendenmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BH.%2BLendenmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lendenmann%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BH.%2BLendenmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mark.lendenmann@usys.ethz.ch" class="" data-icon-position="" data-hide-link-title="0">mark.lendenmann@usys.ethz.ch</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Croll </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BCroll%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Croll%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BCroll%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ethan L. Stewart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ethan%2BL.%2BStewart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stewart%20EL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEthan%2BL.%2BStewart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bruce A. McDonald </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Integrative Biology, Plant Pathology, ETH Zürich, 8092 Zürich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruce%2BA.%2BMcDonald%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McDonald%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruce%2BA.%2BMcDonald%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2519&amp;rft.epage=2533&amp;rft.atitle=Quantitative%20Trait%20Locus%20Mapping%20of%20Melanization%20in%20the%20Plant%20Pathogenic%20Fungus%20Zymoseptoria%20tritici&amp;rft.volume=4&amp;rft.issue=12&amp;rft.date=2014-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lendenmann%2C+Mark+H.&amp;rft.au=Croll%2C+Daniel&amp;rft.au=Stewart%2C+Ethan+L.&amp;rft.au=McDonald%2C+Bruce+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459743/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459743/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459743/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459743/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459743/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459743/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459743/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459743/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459743/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459743/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459743/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459743/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/12/2519" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgements</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/12/2519.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/12/2519.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/12/2519.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/12/2519.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
