<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/12/2317" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/12/2317.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/12/2317.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/12/2317.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/12/2317/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/12/2317.atom" />
<meta name="HW.pisa" content="ggg;4/12/2317" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Influence of Outliers on Accuracy Estimation in Genomic Prediction in Plant Breeding" />
<meta name="DC.Identifier" content="10.1534/g3.114.011957" />
<meta name="DC.Date" content="2014-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Ould Estaghvirou et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Outliers often pose problems in analyses of data in plant breeding, but their influence on the performance of methods for estimating predictive accuracy in genomic prediction studies has not yet been evaluated. Here, we evaluate the influence of outliers on the performance of methods for accuracy estimation in genomic prediction studies using simulation. We simulated 1000 datasets for each of 10 scenarios to evaluate the influence of outliers on the performance of seven methods for estimating accuracy. These scenarios are defined by the number of genotypes, marker effect variance, and magnitude of outliers. To mimic outliers, we added to one observation in each simulated dataset, in turn, 5-, 8-, and 10-times the error SD used to simulate small and large phenotypic datasets. The effect of outliers on accuracy estimation was evaluated by comparing deviations in the estimated and true accuracies for datasets with and without outliers. Outliers adversely influenced accuracy estimation, more so at small values of genetic variance or number of genotypes. A method for estimating heritability and predictive accuracy in plant breeding and another used to estimate accuracy in animal breeding were the most accurate and resistant to outliers across all scenarios and are therefore preferable for accuracy estimation in genomic prediction studies. The performances of the other five methods that use cross-validation were less consistent and varied widely across scenarios. The computing time for the methods increased as the size of outliers and sample size increased and the genetic variance decreased." />
<meta name="DC.Contributor" content="Sidi Boubacar Ould Estaghvirou" />
<meta name="DC.Contributor" content="Joseph O. Ogutu" />
<meta name="DC.Contributor" content="Hans-Peter Piepho" />
<meta name="article:published_time" content="2014-12-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Influence of Outliers on Accuracy Estimation in Genomic Prediction in Plant Breeding" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Outliers often pose problems in analyses of data in plant breeding, but their influence on the performance of methods for estimating predictive accuracy in genomic prediction studies has not yet been evaluated. Here, we evaluate the influence of outliers on the performance of methods for accuracy estimation in genomic prediction studies using simulation. We simulated 1000 datasets for each of 10 scenarios to evaluate the influence of outliers on the performance of seven methods for estimating accuracy. These scenarios are defined by the number of genotypes, marker effect variance, and magnitude of outliers. To mimic outliers, we added to one observation in each simulated dataset, in turn, 5-, 8-, and 10-times the error SD used to simulate small and large phenotypic datasets. The effect of outliers on accuracy estimation was evaluated by comparing deviations in the estimated and true accuracies for datasets with and without outliers. Outliers adversely influenced accuracy estimation, more so at small values of genetic variance or number of genotypes. A method for estimating heritability and predictive accuracy in plant breeding and another used to estimate accuracy in animal breeding were the most accurate and resistant to outliers across all scenarios and are therefore preferable for accuracy estimation in genomic prediction studies. The performances of the other five methods that use cross-validation were less consistent and varied widely across scenarios. The computing time for the methods increased as the size of outliers and sample size increased and the genetic variance decreased.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/12/01" />
<meta name="citation_mjid" content="ggg;4/12/2317" />
<meta name="citation_id" content="4/12/2317" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/12/2317" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/12/2317.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/12/2317.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/12/2317.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.011957" />
<meta name="citation_pmid" content="25273862" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2317" />
<meta name="citation_lastpage" content="2328" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sidi Boubacar Ould Estaghvirou" />
<meta name="citation_author_institution" content="Biostatistics Unit, Institute of Crop Science, University of Hohenheim, 70599 Stuttgart, Germany" />
<meta name="citation_author" content="Joseph O. Ogutu" />
<meta name="citation_author_institution" content="Biostatistics Unit, Institute of Crop Science, University of Hohenheim, 70599 Stuttgart, Germany" />
<meta name="citation_author_email" content="jogutu2007@gmail.com" />
<meta name="citation_author" content="Hans-Peter Piepho" />
<meta name="citation_author_institution" content="Biostatistics Unit, Institute of Crop Science, University of Hohenheim, 70599 Stuttgart, Germany" />
<meta name="citation_reference" content="citation_title=Plots, Transformations, and Regression: An Introduction to Graphical Methods of Diagnostic Regression Analysis;citation_author=A. C. Atkinson;citation_year=1985" />
<meta name="citation_reference" content="citation_title=Regression Diagnostics: Identifying Influential Data and Sources of Collinearity;citation_author=D. E. Belsley;citation_author=R. E. Welsh;citation_author=E. Kuh;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=A. Cerioli;citation_title=Multivariate outlier detection with high break-down estimators.;citation_pages=147-156;citation_volume=105;citation_year=2010;citation_doi=10.1198/jasa.2009.tm09147" />
<meta name="citation_reference" content="citation_journal_title=Comput. Stat. Data Anal.;citation_author=A. Cerioli;citation_author=A. Farcomeni;citation_title=Error rates for multivariate outlier detection.;citation_pages=544-553;citation_volume=55;citation_year=2011;citation_doi=10.1016/j.csda.2010.05.021" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=R. D. Cook;citation_title=Detection of influential observations in linear regression.;citation_pages=15-18;citation_volume=19;citation_year=1977;citation_doi=10.2307/1268249" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_title=On the design of early generation variety trials with correlated data.;citation_pages=381-393;citation_volume=11;citation_year=2006;citation_doi=10.1198/108571106X154443" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. B. O. Estaghvirou;citation_author=J. O. Ogutu;citation_author=T. Schulz-Streeck;citation_author=C. Knaak;citation_author=M. Ouzunova;citation_title=Evaluation of approaches for estimating the accuracy of genomic prediction in plant breeding.;citation_pages=860;citation_volume=14;citation_year=2013;citation_pmid=24314298;citation_doi=10.1186/1471-2164-14-860" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Stat.;citation_author=G. Filzmoser;citation_title=Identification of multivariate outliers: A performance study.;citation_pages=127-138;citation_volume=34;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Genomic selection.;citation_pages=323-330;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076469;citation_doi=10.1111/j.1439-0388.2007.00702.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=C. R. Henderson;citation_title=Comparison of Alternative Sire Evaluation Methods;citation_pages=760-770;citation_volume=41;citation_year=1975;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Using genomic prediction to characterize environments and optimize prediction accuracy in applied breeding data.;citation_pages=921-933;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.07.0420" />
<meta name="citation_reference" content="citation_journal_title=Comput. Stat. Data Anal.;citation_author=V. M. Lourenço;citation_author=A. M. Pires;citation_title=M-regression, false discovery rates and outlier detection with application to genetic association studies.;citation_pages=33-42;citation_volume=78;citation_year=2014;citation_doi=10.1016/j.csda.2014.03.019" />
<meta name="citation_reference" content="citation_title=Linear Models for the Prediction of Animal Breeding Values;citation_author=R. A. Mrode;citation_author=R. Thompson;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_title=Agricultural Field Experiments/Design and Analysis;citation_author=R. G. Petersen;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_abbrev=Biom J;citation_author=H.-P. Piepho;citation_author=J. Möhring;citation_author=T. Schulz\x{2010}Streeck;citation_author=J.O. Ogutu;citation_title=A stage-wise approach for the analysis of multi-environment trials.;citation_pages=844-860;citation_volume=54;citation_year=2012a;citation_issue=6;citation_pmid=23007738;citation_doi=10.1002/bimj.201100219" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=H. P. Piepho;citation_author=J. O. Ogutu;citation_author=T. Schulz-Streeck;citation_author=S. B. O. Estaghvirou;citation_author=A. Gordillo;citation_title=Efficient computation of ridge-regression best linear unbiased prediction in genomic selection in plant breeding.;citation_pages=1093-1104;citation_volume=52;citation_year=2012b;citation_doi=10.2135/cropsci2011.11.0592" />
<meta name="citation_reference" content="citation_title=SAS System for Windows, Version 9.4;citation_year=2014" />
<meta name="citation_reference" content="citation_title=Comparisons of single-stage and two-stage approaches to genomic selection;citation_author=T. Schulz-Streeck;citation_author=J. O. Ogutu;citation_author=H.P. Piepho;citation_pages=69-82;citation_volume=126;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. Van Raden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. Via;citation_author=G. Conte;citation_author=C. Mason-Foley;citation_author=K. Mills;citation_title=Localizing F(ST)  outliers on a QTL map reveals evidence for large genomic regions of reduced gene exchange during speciation-with-gene-flow.;citation_pages=5546-5560;citation_volume=21;citation_year=2012;citation_issue=22;citation_pmid=23057835;citation_doi=10.1111/mec.12021" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. C. Whittaker;citation_author=R. Thomson;citation_author=M. C. Denham;citation_title=Marker-assisted selection using ridge regression.;citation_pages=249-252;citation_volume=75;citation_year=2000;citation_issue=2;citation_pmid=10816982;citation_doi=10.1017/S0016672399004462" />
<meta name="citation_reference" content="citation_journal_title=J. Data Sci.;citation_author=T. Zewotir;citation_author=J. S. Galpin;citation_title=Influence diagnostics for linear mixed models.;citation_pages=153-177;citation_volume=3;citation_year=2005" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Influence of Outliers on Accuracy Estimation in Genomic Prediction in Plant Breeding" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
<meta name="twitter:description" content="Outliers often pose problems in analyses of data in plant breeding, but their influence on the performance of methods for estimating predictive accuracy in genomic prediction studies has not yet been evaluated. Here, we evaluate the influence of outliers on the performance of methods for accuracy estimation in genomic prediction studies using simulation. We simulated 1000 datasets for each of 10 scenarios to evaluate the influence of outliers on the performance of seven methods for estimating accuracy. These scenarios are defined by the number of genotypes, marker effect variance, and magnitude of outliers. To mimic outliers, we added to one observation in each simulated dataset, in turn, 5-, 8-, and 10-times the error SD used to simulate small and large phenotypic datasets. The effect of outliers on accuracy estimation was evaluated by comparing deviations in the estimated and true accuracies for datasets with and without outliers. Outliers adversely influenced accuracy estimation, more so at small values of genetic variance or number of genotypes. A method for estimating heritability and predictive accuracy in plant breeding and another used to estimate accuracy in animal breeding were the most accurate and resistant to outliers across all scenarios and are therefore preferable for accuracy estimation in genomic prediction studies. The performances of the other five methods that use cross-validation were less consistent and varied widely across scenarios. The computing time for the methods increased as the size of outliers and sample size increased and the genetic variance decreased." />
<meta name="og-title" property="og:title" content="Influence of Outliers on Accuracy Estimation in Genomic Prediction in Plant Breeding" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/12/2317" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Outliers often pose problems in analyses of data in plant breeding, but their influence on the performance of methods for estimating predictive accuracy in genomic prediction studies has not yet been evaluated. Here, we evaluate the influence of outliers on the performance of methods for accuracy estimation in genomic prediction studies using simulation. We simulated 1000 datasets for each of 10 scenarios to evaluate the influence of outliers on the performance of seven methods for estimating accuracy. These scenarios are defined by the number of genotypes, marker effect variance, and magnitude of outliers. To mimic outliers, we added to one observation in each simulated dataset, in turn, 5-, 8-, and 10-times the error SD used to simulate small and large phenotypic datasets. The effect of outliers on accuracy estimation was evaluated by comparing deviations in the estimated and true accuracies for datasets with and without outliers. Outliers adversely influenced accuracy estimation, more so at small values of genetic variance or number of genotypes. A method for estimating heritability and predictive accuracy in plant breeding and another used to estimate accuracy in animal breeding were the most accurate and resistant to outliers across all scenarios and are therefore preferable for accuracy estimation in genomic prediction studies. The performances of the other five methods that use cross-validation were less consistent and varied widely across scenarios. The computing time for the methods increased as the size of outliers and sample size increased and the genetic variance decreased." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
    <title>Influence of Outliers on Accuracy Estimation in Genomic Prediction in Plant Breeding | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"OSNdSJJf7Y6iDwbZ7u5NSBETT7iAOYNA3eBJuO5-vj8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459749","apath":"\/ggg\/4\/12\/2317.atom","pisa":"ggg;4\/12\/2317","ac":{"\/ggg\/4\/12\/2317.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/12\/2317.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/12\/2317"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/12\/2317"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/12\/2317":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VrCiVlIBwWvYB7xtMUCg+VHVj9HRnmmd6LSyksiRULZxrEgx2ijKSxDtgAD8nITkt8giIECe+\\\\\\\/YKPcQPwBGmAo+0If4N5W54CyPjjm3Gnb7JBk\\\\\\\/HCvDlfk71IakELEHTSL7ArETPkR\\\\\\\/GBdfchHBbXtqyhJCfEEBQGH37u56qA9zOb1Bx39cc8uanrdlpDBwkMvX3A4cY4NX7GdXMKnhLgIuPhOn6CLBoTm7e2iRLlPfPVFOotQvmHWrjLh0fXWWpctYAzB8WnwkR9c8d3BVqxTzhFkyncRgc3X0IIqIfo9oyPm5oaH\\\\\\\/e0opVwTODpKU42Tn9d1RpIF3ni5KJ6LZDNhDGukmvjbH9+kWB01QIYV1yp0hP0aMn1XGWF0oTcbKeUhuUoXkDyTAxfKEubVhJpJ1alryjkJ7zQaXsguXnk2EpzS\\\\\\\/o2bOOkFCiIk1IdjVUjQBrM8eZ7DjZsD7y+aK520xdRHfMWmaTWnyvuatNbsb4A4jE6FKkapuzAiVOlnoTMwYL6MPyKsc+Kn0Bnse3e9J4m4spnsmDNe5alkY9L+KLUPwMy6FepxKbXYHARi0hz+d5noJqc7\\\\\\\/IaqJNGDWcJRkbIQCVawbj+vM6hJBsalc7DAO+qm7XF35dFF3t+cwH\\\\\\\/OrQpzDW\\\\\\\/CyGwOLe8JmCp+3eC2sIO9NAWfMRaYhCefOUv169ODvUbZ5gj4XL\\\\\\\/tbIY3j\\\\\\\/0C4M4DwJDApcsaWmSmykn5qvt550DxB69C2oyFLUnMjeHVBXeSYTTPOFcN6Tj+H97\\\\\\\/sOd8+huY+13mTP+iRwYHiNwhHZRtIxaoclVogkVnn0Rr9XCPm0GPfthjMOtYj7Md1lUWsRYjdGVKenb8QsJZ5oy5bOWEuXD4uCSntH5itRxdzneRs\\\\\\\/Vzs3FtBAZP5WSbVOPESts+VDLdaydTCKQpiTZ96fwBC1DsfvqmosG\\\\\\\/1fLxBUuX\\\\\\\/vyh9u6ArA8UcOHk1j7ydDpPXan3fo7ggUVcheRNrr0RGZhP\\\\\\\/s0\\\\\\\/NazQs7pinxJQOj5J7eQbB11mO5GhJfz34XyRTp21nWh0grgLhHGvUizR9cSe\\\\\\\/bJ45fN\\\\\\\/YOyZC\\\\\\\/mY6gK2PO2kkyyrleMaGkSBA7Gl3KV+CXotyIeuIxL2LsDcDpW4lSTkLC7r8nhniunFjDIdWnk1CayMD3T2O\\\\\\\/o51IYWkMyHISIqSCLKXtBWX0snnwQMbZGBOg3vQohb6ysgo8uC\\\\\\\/jv9j05WJjhdaZp50LoHr4gHGRlX4CA2wnsusMrEkBWgpdvYuW7vdKonnfMK++L8yItpA0\\\u0022,\\\u0022iv\\\u0022:\\\u0022j5uQu5lwBtrrOgb1veajAw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002228cf444bae75ad4b3e4f215e3b42a7ceee9ae197e92d0caea2dbda1e3fab00cd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VmfcGSTXTN7OQi3ahOmp2KU1L4AWM3EwVHeZYmH2yVG0hH+tlI5yFqCxLicMw83+u9+2GzPt8QJky6BBSMT77BTRnMw137IjvU2Ouczvx3m6ObGqYCZvPC\\\\\\\/zv2MnDaHX5VzDaGjfcc\\\\\\\/Pte7jjts5bw+heuOcYnmFSIloOk95vDDHplFB1pmX3NCGnhKI18GzBWXmQArcz6atjltI42h6IsgbYMGkzCpZA1dfHmKegK6u2mWkrQZ+HYYC0QdGFKdJ\\\\\\\/Qa12db5mGztON4YeNmUY2V4im+3DYD2YvNgy2myEfkLC7JIK8J2TkOvZEGgKurzW\\\\\\\/PvgmXXkoYBYseR5zmWyM4+DQrFE30pIhKtP4HKo8j8iv7P1T9NvqgQcOIefRQU+NVoIBWyDX6venRCynLK7LgCMpWgiVwSGw2+OFbvZA4BbTCUMeJiAplJlqofVOPde5KPELpTT\\\\\\\/PYSZUlpSuq\\\\\\\/AklnZNJb2zRdQU2z028dr\\\\\\\/+lQfa6OHYrE9D7uUadhfqfMm\\\\\\\/oxw8JW1qSG+aPXoIEQJHkGIR92z\\\\\\\/+l951+CisKd4pKctBuMSY0veHJFN3TIsHqUIQ9hZ6p5pOIMXehve8+7QltpAROPUEMlMJROkPcFmMrxiLqI2b+w+qU0OFt1gYW9hfp0x2X0tWnAT7sNWImFxI6GPNQuK4a4zhiTZudtqCrDWyz5M2N+8q\\\\\\\/EHe8nCe4TUULLUOYpnTx7Fcke2GlssnnkWtbJ6qDBoJZvOBCsOgIERUYkXoRVEkZsbYBzTiOgFYcYhDQwBFaSJtHmZEtsj6pA73x\\\\\\\/yvpH\\\\\\\/roH1PlSedivalJILh5blGVKpMBmUWGZdVGst4RA0nkGzo2NYZnkivnkAY+21\\\\\\\/e6YtxqM9XY2s532Si7sXg2CtM3n0DVz38Nyogdk42NDwI7vg+Su\\\\\\\/UZG6ZWQxnCsjN1eM+4a87b4Bajm9nm8PLcU2mBZm2Tc5BuYyFfTcWiOg+tzWTrQapSZzQBF9224IvIs7mJ7k6UQPw\\\\\\\/7XVpOllfdkdIPQrBszmdEz2r9bOq+1v3RTmZsUFpskvnz+D0l4VbuXLVtjL1mzb2YQlv1Uh+HfS2JC9tOH1nLAve+TO3weiPC3z+TkxYKiZFupCEW+IshN5DKuyM2ZhSZHBqtQjt8MIetKsC8HR2kt+I\\\\\\\/JWf0Y84Kykxdwy9HHKoY\\\\\\\/KA6FKwo2\\\\\\\/ZfK0lP96uh2z1By3WtVitb7bnGJtLutX3Ak0bwyoNhOFzEWGcsmgUtri+bwK1hYDUBqOpwsEkNmWQTMgz37ASjaCszMgNrgsDEzLJYf+yLfO8VkQQKcEPb46Nx5Lh10EWUwm6GnjM7CmQ8dfhmwaP7aNpdkYtU6nZu8s+7ol3oREtBE15kC4lPRtt+VBVKjHeHYyiCHuhOll\\\\\\\/x5stMvuSY\\\\\\\/n4cQWenjtaPIBSbFbPDzU4UXhrtAl6tXjffkugcHR9qArqoOX8+6f9WL8474HxVVB\\\\\\\/Y\\\\\\\/85Im9yxrztJ2DD74aR2Yyyjt7iwQlUznrJPFZTMBljE4Z+xHPhcHfF+tztjmqsgj1tLaNuCHntvML7WtGXCTU+YX2wNyY45KVWnML3NvEXvgI2BQBx\\\\\\\/e9olcuFgqb\\\\\\\/g1thtWLyiLSPLon1uFJDcbKhJ0DuR11fcPKuPo9T9AwE6z6zrBjjuJQGEC9prW8VQFnrTLhDdz2YZ59yt+Y6BEcfmP0iduUc4KhAPoa57\\\\\\\/zSp0sJLoQ9LsaLc1MuZYSJSalYhuQ65MoIcEuVHrFuI3W6Pn4XGs3EnpEahh0DO\\\\\\\/XJBEtoD6hS7fSYauqxPs4Xa1diuPpDkV+QOkHaweJhlqr0aCrkCMDc4xXdf4Bw15yLyhO5xr7nJ21J2lb3loxHF3s\\\\\\\/lfih3HePvXcrGsta8qh2lITNiDqhHiSdPMnnVz1cwmbFMiCCPjWI80jFDG2rRBH\\\\\\\/dWmXDDBlcj\\\\\\\/1hh2QaHhgVQCeJLHdO\\\\\\\/gSYPMx2TVDBhZ44dTKSP0JAAx2\\\\\\\/Z5uvRhjnG9LHhlEnduRfbLXtuXkaRdRTqIrKlgMukjAvbCtqQNF+ChmTeNRNpffzfwRANNrtcRj77IUld5f\\\\\\\/AXJpDmqx3nUxaA5KUZFwxvldFJSRU88e\\\\\\\/qtdhqR3FnT6+GI8JeTzMHlQ9UDTo\\\\\\\/rt+qZgvayMxaN0w51WGsgTBuFDdMptdnaGognJj\\\\\\\/URosTr8\\\\\\\/VwnVZ5gQZ7QIG6f9yny3WbIGNicwQrXCcG4ZYXK9vxe1gj6joqXWR1TxS3\\\\\\\/nIsU0DRqVbbvQwhwtATtGWtA5vZ2zh+TaJ2jx7aR8LDuHExJm0Zpe721jVb2VM8LFR8kt+vez3Upkxg8jfB6C66UNz0PTPKrMcEWO2BayMzlnLSbGbbf\\\\\\\/aTpWq1\\\\\\\/3SS32m1DyeNj+2iCYUDxYso8JeoU4rSzBmDwDjDS4qXkNOn8pp6qrqPY91LDXnSogLE8MQIrXttywauID2i7hkROYEurk1Q\\\\\\\/hYus5SwK17CTcSPkIkoVnPfDH7JlcvqZahLebhit2BiJCOSBkq74dIddsZ9llPjBGxxhrFf0CoJHTua2O0pAGg0noLFhWQa30CJ5K\\\\\\\/wI5Hsw+Aah8+iXza7wdIZHqB\\\\\\\/epeBbPlZqvG8eavp4eAO7lUawYAwoc7fLhf+yBtd6W1gWO1WC+FpEaunhwWA3A\\\\\\\/nhTgOhRYp3tIM9THiLmZ2wWPk\\\\\\\/mu4QIf0EcdHTgWxL9nZP2BZbyJGKwOhQkDOh5Ef\\\\\\\/wWzg+PEvOZSyvwXVZdsmO0qZ9e3mw26zQdyTaTKoRWofCLq4jQ5QTAeuABTEcx2pKZQ9LmObc0DlND0YEphUtbtMzx0WzTjJ24ESNEcIfserIPAf0TFT2bLvYDkDV62HEuTrSBIFrnxDVTVM8vXPq4VEuNBWpyqz1CAayfWnBP49wu7kWBh+dBox9pXodXbgMwdhAx21nSvs\\\\\\\/\\\\\\\/\\\\\\\/3eigo24Ksa+UHKaPs+GUNUaLJ1Z2VcNLcdT29lPDDFDn30MKGeydJgpIZ\\\\\\\/JkL68RlYu0+DOVf2egNcGjWS3yC90nDsCiSoc8wqMfG6s2G84QJyYs4iRyrkbyCYFCHnsEebB2NjBKFOTYiYDqPC8nj9FyghYfVL8yVIgLjdyqigcYcW6Uo+3pRCxen5OKA3fblbMWQ99m88r+K15Og8EGw1tg939Yh+B25RishxhwyYjyMPpXJndtDIK+dqM61WJ\\\\\\\/7q\\\\\\\/z59yUz8wI4bdm8ACCmEN0JnnSlPKF6IeBOPrxJ1gEIxL89tdam9YP37XQoYwF\\\\\\\/5tuu0i5hAxUiovzE9uY7v+lz6ohR+Bg8wygdExivHTm9jfF7zsgqMAXbAaw5Wjs2MpkBe2r8+DRFohTVA8WmavqY7sS2QPQ9wPF65CqWxH41mDVjIG0moq6s9QgF63nIgFm0Y5jJuPTdWo8XVEtUXAcMU03p18QIipG2ZAHKL9bAFbuX42XOuMJumxx1OqUy6+KJaplILGr5KqWccfVErgD3VRUzFB7if+8cpTsYFDzPBqoCuHGSMk\\\\\\\/\\\\\\\/I7W4rpb2IhbvW8H0QM1fHrTs7TYbS1PzXsdKFheMcPSbeh2H8xDGf15j8ga5dHVTVhYetmeJK7Ozok3v7VM81ZRZNa7B0TAJeGvXlKBDR+q\\\\\\\/8yN\\\\\\\/mP+6Shx18nONqa8Wy9rQ9RtU2UNV+hKNNe4qCa79ZeijXa4vPNvLEytwvCj1kOKmLOGkAzfFV46gU+mYXuoEwVQ5JeImPtFZNVUNxQ5VpLdnyrBnetdXxBxrNTfNgRp+mrS+qd3U1W+w66D9vqz4TgH3tdcHRUWfTka6VldOa646RfrwAxJ2IJWOvhvTrTV4moj9v3CYb646LzcmjrEXxuIJ\\\\\\\/C\\\\\\\/UfJfpiwwvLk2SSuc87hyYITcwCGI0Hw1\\\\\\\/wMw13fgri3IVcmqkf\\\\\\\/eRgya5bi3hYBMVn75wSLfXxvUctlH0j8jTvswrjk4RJc+6LvbzY7kQgHVnJtylPhVpw+HzZei2IiFnkLuUncajcXsRmOXWv\\\\\\\/Qu\\\\\\\/yFWI3fGCB6+32ckb7cstnwrO7N\\\\\\\/51yiHbULEPAKjdMcUgnx5fE4iV3W8IR4pdwz7rnjhEbN7otHmFP1BCrVf+bLCU9RHbCekjQUMoUNWIJ\\\\\\\/zIr7EOrBXqKf1JADPGvLo0vNgEgCP4lKih++GK5wbo7J+ohx3r+42BhDkbVr9x1\\\\\\\/CPD9ududZXxe425izvmMnJndQL48YNyDCY0+sJCYfNIQ9WsMMLTuOVkxHULdVe8z+9exDq\\\\\\\/XoprZc6O4nJSi056hBe2+b8I\\\\\\\/uQLvgnx5ine4u7wp3FIwwk1w4NTAHQ+3FQfGidUl8Bym4606KOhUj9DtODxnkFHGW3iI1u0JippM9V5tHnatDKk7EwrdHSVUDLmtfn+hpcp+KzR9mWOtk+q2OZA8JIPRY1goGacKr3ga1Y8JFu0mPhjpbwP7pOTdxGuvplOKD5Dgv8LnVDhc9YNEnLTnUuVPDYuX208Z\\\\\\\/rXl0KRk9kTnUOQ383bhtNsqnRJyaMQm6Sf2DWBjMHY9NCjb8ac8NY04lPUeekuzSl3BPV1+PbUg8N8ILP0jZw8TTOVStiDV3H11VyOrPEcD7bKdsQmy+ninQsTm6OaFRgZ528hQr3roEtRgTExRd\\\\\\\/cGWowICFhEPlx5dh8IBno722By0O\\\\\\\/FpD71hRGRdJEOyC659TpN\\\\\\\/NgY2d5LfalFx6OHA7tWBMeS9NvlPfQSYwrXzXJNZFW0taOQfUWf7cuTpD+++Sx1pn4PcpkXFpl6ro8TB2CwzgppXilNUDcrEqfQKKLsbezZK9+qdsWkdQyPj+D1sqzYraz0suNW1NRwL7VY1m+gvYhrKbEnTMIYJN9CG5eQ\\\\\\\/b2yrRwq\\\\\\\/By76uf\\\\\\\/gptGgldH4cmGr4XingWGTQ9MmsM8ymYwEyjtq0eIZSVoFB8dnwrtB4YyUcayRsuvyk8QSeRStMnBsSsibt4sBIqYMHdPhdRwxdRu7BBs3u88u2wiCuCGAGbhOtGwTg7bxj3Jfr74tvNxC3OmtfrIGdz\\\\\\\/qiCGaFW3ybBlqCUA0zXavdY1RQ4szIym\\\\\\\/es5eRSxyZnJxHlk1glyupCpbAeVN031y49aodC5MP+Wgfl+xEces7ln09xeHoxPJAN+4iLuOjiYVgxHNoKqafsYhxxxIB1aaLTnBIBFTXx7s25qi53Sqq6RoP7nVKZkAnDwfQtgT4QbAQRfGJ6MW1BCOhypXmWXofBJ\\\\\\\/8SEhxkRHU+X2zMxPVKFMy0AkdKIHfd0W3xP9El5O4NHBaEBP49iHT6QnMtDYSBhDhwrQzmWweB7lEuQZHKZn7i2GvjyO+F89JN7cN8p5xrADkpUT9bf3KxaT1JoKItln6cQMWmdoLOJw7p73DHizcpJzLrYAVHp9CeGBrg4BDujXpdM19Fr7UmTd6CXAFLyWQPn280ITYNXcoGf1HbaV+qNDswjBJcLtnJgBNCs0NaPDCFzaJNkhVGBpF7jofK6F4qCdgURaO0m2FPqBLQtgWbG5EolIvhAK\\\\\\\/RWmPJlJLa2+SHFitXTMusc3uTFFUl5\\\\\\\/qERG4W96+S\\\\\\\/bpogLIkrqrPix3AkD4pmB43Lpw3Sj+75aOc5qbgHk3BjzVp1KJrHcB2PGe8ypZ5857If+QqNgMvZD0LhKaEi270TOwlC6JYwO5EHA7yky9fAjpd5qxXZVRUFkwy+aJrwTzvLiAuqhY8q9fZVY5AbeAwbJ5pWeoevq3J3SkdmLFurjv5+eSP8+F7XKb1prYWmWotFl64V5fF+rDYuGqxw4Xr83WvgxL59onQoP2IYwEggJtoD7VcCjVlB8rxg1TUzWuWzkZO7Rjzp2VMIeBOME0rlmWkEOwMFGFO2FtwOPsnOaxMqM2LfAwtWzjtZgvP0WvapBXbfT2Ft5x3BPt2UVQIHIFdRpNAQHu7ewatleD5C2X5WB3px\\\\\\\/XGcdUOQCIKrcf5iNSgqm7yE3ts8W3LRh9KVLep4PQTmBIlyxuP5a6Ul0X0mvYn19GCSiqHtwjK7WW1pVXo9kH8AL7Vd1CJLZqMMlCATxxONAcAsqbWFdDi7o+quWbbjXX8CJ5nAy4TAZ6bdY7tHOF1lj8adh3OddOll7LKUCyvF5bWS+skgaWSOkCvpNwB1Hs13XdmDO1V4VhtY9xF5+qleWbnzRlPRpk+BDbs8bYSNrkxe7htVIwlZQTbWPPA17B+9tqnLBaS2k6zpO241NHtG+FaPrXEbFFJQlDarQuvdH9uzrubRTbGx1F4ez3iw1inQCFGCybmHtmKwx9zvINvb3wNHRJJqqzUOyJj5lvgjpqhg319kCyynSkWf6zISy2ldlzDmROgS89iq8MvHSgtYdqjPVoRf+liU6+Mcn1JMaHCRwnmP3dbrt5gDq7VBLlDkY3\\\\\\\/RKJM5etl0w9HXBFxin64zJfykA+JGn1YKDNpz+SNguegIbsRAE4xEvQA2KGntnekdbTodDpK4Amam+FA1NFzj9ZlESiVMLcPpxGbRUhAi3uc6iEm0K8cNnx0O6y\\\\\\\/bObzL2zxtcESCCgPjcU3G5l8j3MJCupxSbSyrOqYNDDz+6mSxED0nRY1VZLrg0HSRNsyxXUR3AVgnyN9dvPkYgvcmz4qeZZJhXdzfNR4F+CetE9pO6tWCHdIoEwJhZe39bTp++UV7zXy4RXOEDR5B3R4K7EccSUUF9zGHiM6rUSXs20MlZExvHCLEn98lts1x6SUWSVHin3gk6Cew4eY6gfWatsOR5gFbeSATrzDEfmOw+O+FKReKU4uPa8gxHZkITAUCU05AvorLxA3lT53w3qwZMK\\\\\\\/hn9W14F3v8EstJRmN\\\\\\\/KYDbGGIXOA2u4E3UESTZ0zCYmoiAhJMKXUg32WRfl6i2tws5Plhfj81+nQJwBK\\\\\\\/W6aGl8NXqPGOTUfN1H884ZhekNRjJg+mHFoBenzoIpCFCgdSRY5rjGSs31bk70WU9iIMBp31pdsYmPAki4EyoJJFqb6stbXxg29MySqjUaU\\\\\\\/fAR25r5onXacPE9oPhNSzp+k\\\\\\\/l9KvI\\\\\\\/DaawZITAJ6dBjNPkbc+7JhZ88QnHUqVtpPVKW\\\\\\\/KOaZ8ebbzXNZB672fxtQBaQ96yCCLMUG1nOfCnivsULf+SRXCvrVLvCfuuGXAsySf4Z9Ndsd4XUlvX+vo\\\\\\\/8JwRSpW2stQFyXUYUGB4Th7rReaZk27wo2fRQ2UrbZYbSLYY5vr4xwyyWW3MttJZNZJ9JcNudc6qaLV9Pf\\\\\\\/IYDdljK7KoM\\\\\\\/W671t\\\\\\\/NNn3kJidlPsJTGdtGFNhQA4mRaLF94yT0zlzTxMWADIQPdmIhhhytGIapHPMZlE32b9Z0W7zDa9R1r+IwFSAq4JKMEq8zULYfzHqHFuVPyRSReZrrjXhJguNf70RaNgH+fIEUbhx1X0UvFR0lR7WDcYMOhyU3ruLV+THidmFBwIZK8BSU81c37gEoULq8bjoMhxnQyVn2p3UnKGWAhel9yplxAbFw3n0nb+hdXpZ5uMg8bTH9qukhVdw2j5lR8Jgq90rbEIO8CtCVPE2YrG\\\\\\\/b\\\\\\\/EFy1937l7xGCxpOy\\\\\\\/dKhH2LPVhiCnl8FdkGGfPzWBIWk\\\\\\\/GXy6CoAjGBKKRJ6UJr+lMfORHBRda9SkI0Ync0LRNVbcuFFtVL6Sfj7c8WbQE63ZWtdCR9QHB967k0SL6hxRLDbvCzPgQUwl+wu5GlENMCsjz1pvwHVChYtYF+HRMY3iyG4pZNmzh++isj2NKX6HqkBKTrACnbxaVunbpfwUfP46XlfSgMm9HhR2AtVdEc4bKvO9O\\\\\\\/MfaxjKN1dRirKUf+qKQr5cAvLgCBxDLoB0S2A00CA2SpIL7mZrG56sQY3n\\\\\\\/z\\\\\\\/LgjYRJY\\\\\\\/xGDZtddrHd317dU7hw7nf5PPwxMCdiCAFY2z\\\\\\\/OrIC144Tevs65sR2DCkTR7l\\\\\\\/eNftRadQXZ0fMRqFC45wp0j9vUyhGdesfFpIG9GsdNt6wmWOwyx7DJjG9mD3KLeLEZ3oXIbgVUxaYlBfm\\\\\\\/gEDlv1H58Nzz4b7U72C9tyujy1zwMZWZ5J0Xh5mSa8a9+H29miTuAztrDTOEbZVvuWrDht3LIcCz9nGV3FY8ZiZg2U+kgsTsyAEUYIHy9xQXJA5Ebc6Wh+wxq6ByOui1ykq++8N4UF930vGN4TT3xvPwlgR\\\\\\\/Vy4YblARJSsjto1eNA3T05+n18XjZ62AYJJglpaYnoNyZbUXgqf11iz1EjVBOGdxnXGAKIs3zsNK5tzHjTPgaoWCYiM42BKdwGFzjqyIrAmyYTR+ea8xkH1CNHJy4QEgvzYXuNg08xo+JIhkc\\\\\\\/3WQXEDyRmMFnCebQV9EmX9SniyctqFOG10tpAHSG2oG4J0nGg6HPtbCLCMo\\\\\\\/GGZJyZW5FuAQSwILYiQm+QwkE8\\\\\\\/vp7AIq28+aRMndet5ZGrP4beG2BwIWbosFHlBghaKE493y4wbDDqc6eiRX+5CE1Lvt9fCbveuXl\\\\\\\/pwRY59W5Ly5YvwQ8MUZeDoaMo5rUkYWdUxJgpuXrqkEWpRw6KuUho\\\\\\\/IS5TcBJX9HQSVmVo+Ao0En6AB2E8FPZgDoWane\\\\\\\/fJ5dlwvIIta+ADsX4dX0nbOHjZxw0zoe8r1wNdch+9UNE3Sa2nbd4KYBqoPfrF9b5vb4QlgBvfuT8vC9FUQc3Hp7u+Nj7nhSog3y80twk7kOQlwxGzml3ZbNyxr\\\\\\\/1Y\\\\\\\/JCglWjdjeeE9FFqGdE0Dvb7H535NHWJb9u+dypenAHRR8mRuGY4JLzz95fuyhn6xxZfRkuVKSasSzbkQd8Id6n3XAIjmd\\\\\\\/n6K3bBugC0VVb31po1qjQW8Bm\\\\\\\/nEQiWw26npvHbCkoBwyKC+veAzR1qvGQaqivejt1Fth4QpHg+IGj8RUoJpbJS9JEWjouVSM9caEK6SkIhMNQ2ml\\\\\\\/TZ\\\\\\\/RGMuSYHcUg2RTu9xAjPcYl7ho9ojI+7Hcwphi3hh2EqRnoPK1GuJscGyoZ5rykVH0N73UkdMeP6VWFoLov6JVo+KIc5UzWVlth3oSj7trKufCzVunEuqcwZyQA1n6ldb1TROt54vsADz2wQVL4\\\\\\\/U6ToCgHPOQt2I0e8XoRqOf7+BTQRQ8nlI7E4afCWj15\\\\\\\/SYLj06Ilm6c7AF+cjlSL7m6Q4mI8pOV03QOrsuHopJ+Rz2TqG5tQ8fOaiXw56Xbw1rdbiAQV3y8hFQx5P+YG8kBtjaYOtFa7RQDNUHzLQnB4\\\\\\\/QcYz5IFgFZwQBTs17ZtpbyGMrVEVUdOR14uNdPf4N+9es\\\\\\\/6Qq8HHufnv0vWoewqoh3JHzYYJXUCge8cDUD6QXVCliU5mxzbc3MSwoKszE50Tef5TE3FKt\\\\\\\/F4u9h6D\\\\\\\/P3dcer5Ttl864FwaO+wnQT90EdOBPQjk7pIEVZe5dfM3MrBfhylG5BxaCN5BvDINgNhAcfCILmQEFM3CHt5jEvs77zBMoJ1AK+01bF4G0TumonvF8pCYTdbs+rC+mXqRW\\\\\\\/pJQVvwlCTxprtiAJH1l8AO2g0iWHpEK0hXP7Zza0AzFTAv2O9QRgzIyUmKsqfThm9greH\\\\\\\/PY6EHU3ALEDDkgw9uJX8ymNZhC522RzzrCnW+0+0slpkXEZFkWOxBGxyLaJtPf26LMB\\\\\\\/t4gYxGK6kbTcptqz+2AU6MYHkaiYOBGzBDvIp5gATjjXOVRNUU3XY0dNqc5IqXJ\\\\\\\/fXQWLRG1lhksvH3aH+5YikZHUrkhM5IGQUktmOkaLf5wfZ9MNmpHblB+kL1\\\\\\\/tIcEaHEZsDhyTaiCrvuJ2q7YAsFfFOAwZuuvMR94wIMEE275b+Auhz5RxjKuEZ1VGxpZXzzvX+7d3HLklBT2Ubzl\\\\\\\/H9QDOrPXtNlkaQxR+N871CCU24uTmfXA0K8M\\\\\\\/Ei+P5mDG0vXAD9R1vYubRIousY0SZOXBiCX9QliK7is0kyyQXMQSCgBHFK3JqI7SKMuS1+xcJ24gsPG0OmyHj5nfo3AC0I7KNsmGgy5xY9yirBe7b5ZzyR+WOLeOBu81cTfwJkL7RJcpMEaRchb9CNGvmhAzb+F5j1+UlDv9fPdbFcJq9vWNiQKOPk954exs08xkXIKRAMcYNfgxlU\\\\\\\/Z4VdWjiDyE7Rhl5dp\\\\\\\/1N7n7p1ZzkFk2\\\\\\\/YQvFFVmnpuV+m6rG3uiJDZKuCwtFg8i5Zk5SbcLVTlds4PCfmM5JmyMSC2iUn8Jt4v0fbBM0kEQDsM4sKM3vq+ATbpJTPtniSzuYE5nT07lBCU3dqzYMuWaLMLjXUM52Uyj0qAqGYs3sBb69u2RALRLCSRLrX6vdBNYN+AAXI0+FHvg6tq0kmMImhpJQZVA0WELISL2XBVk6xGFctDxqSTpSgSXCRcTut6dG9yvNFkvwAEiq3ChrMLUx+lPM\\\\\\\/6wNq2Jaiie1cZOxiw6pLXXf7KBuRAAr8z0qzKKIK7f9HRuVu0Soa5Nm9YByQdwkz+VtxwopOyQn5WcPcyHk9v69AZwv0Aem7352J6s+5UAYDBWmfn\\\\\\\/rvySpnjQZ\\\\\\\/LeCCF\\\\\\\/JX3TLzzDSlemx4TfR7etR9yAaluTwMQ+4OKHKuIPPvcGqhjGRv9sa6BRkZr7XU1sMVRc\\\\\\\/KHZ0DbX5CGEOCVWQxAK9GxJ1dSsor7ryMIdSIyrIqVUydHUQN1AV4iS9aQT3rmJO6lYaW2qb5GSj7CXlHjndom+9u+qZwJXLqii7H\\\\\\\/bjdhSKnJFdUkSWPRUQim\\\\\\\/6z7WiXjM2egV53huLvCEJxke1Yha+gcgeHKCKE6FbSGfWK7zF9NEVOSsacAoZdW5ZiqvAi3DdM\\\\\\\/MDAPxwbWBAlxUDQQA67E+VPIMaoZAOWZBGDkOSYiTsqPD0FtzbG+77I0ufM9+VOKZCpViDM+2vVTbMWNJ7Z8L7GNgjOBjGR1WI4US3hIN0AiG9+wPA\\\\\\\/B3yzLT41nJrrt1Lz\\\\\\\/HwFlwwKFmr9RDUtTmA0SsD5kqm9XhaBUooZ94JA0iacYqQMtKxO3Qz2aacshrWHxTBKIW3okZCpjOLb61YvH49HcqLRrFqTyp9Csv1pzfTL3Etw29GkY2Ji2\\\\\\\/\\\\\\\/N1BPkN9oj+fG0z0xtOqWIcGPVhBBri1HHIViYAU7QRJD5syw0cSEhmPsNtSCs3U1d35b1uq6yR\\\\\\\/EBW+bDz4h1jP1y2oH6iJAsGoMJnhvKeSoz\\\\\\\/FGRRpg2SAXnaYGSAKORzfDiukKbVPhJpsrW7H3a0s6h7RdTNWXfaeroDA+TOk222kJESZDZOiQKQXYWzw9ije9YhBeZv8zwowEKxSujW37MGdCBSzOYTVICQRCJqEpEDLGuG7sVY4bEYI3cQl7KIWzj4pJHutNGhWVToIksV5vQWDLkdDfmTQ91CtMlkKGFutuR\\\\\\\/auzw0ZNQdSRUlgiqVtmRIM8TMb7T+Tc1IjJ4BEr\\\\\\\/1N8kKSsdTqoceTDNEMRSrHthN55oRDfWCRXfgG5y4DNyNAjyYqphNNXgqEgGsuNih6YkWcTIPbN3MU9hLy7CvjH+0T6OcpQScNsNK9\\\\\\\/sKH1\\\\\\\/\\\\\\\/fibHyQoy5xiMriYRKA3oSDunkNwMwpB8reY\\\\\\\/KE2F68GW8Yn7wvg25AaptrDUtesHgn+Jbjvax8fkqib+4cCCUUgdNGum2YKtwBjl2rYHSTtashJms+7Gk94OUWhL6xkBK4nnnD4gJaQncRhkTwfNRdyHd5iSNbJ++IJFSSNyKAYLEzzfkO4dBpBjQPsG2uUel\\\\\\\/AA1idwurjeA5Uq7X+DT+kdL+tpNFwrwjviPzmOhRgkcfeiEvc\\\\\\\/aIfMAR4nKc1MKUa1P9RQqlqNzYTa01ULpFj40dYBjzLoo7ALhDtXRPZqyvX7BuuQueFSSjCq2AQ1NYYBvVVnSejMuQitoe8CRrynnyllTcNWhXHnMMcKo03dwBayfWX\\\\\\\/zTBBwSn3MCjIitrY6wapPWfWd5ehR8PVAlGNNXm\\\\\\\/Iqd5Z\\\\\\\/pcyhvi0oxjqRo35e457TtruT9Jbj1LqGFnq1d9DJijTtnnYlBOUscVTHuC9d9+NYk5kxMCgOyS9CRYhtlPaFD59ONMQfEu8YvMkqBj39i17\\\\\\\/07fIMPsoD9Q2pyJqNU7EwNzPdHjnV09WfZaErzaaLXDaFEVw0g4cH3fl6C6pk14l\\\\\\\/4BfpoK2CiyDGGLNbP2tCPVM6hZ0pkH3i4UEQYg7c48MGJu+x2LyRhoNviyB68wcrK5REpluCbs5B1kTM7J9IAxciClzMzkyBUKSa8Luy2flheBViqwTwB1etgskS9D0C1x3AjaMSqZwxy7Jua9ENC4bNCHSGTemJ+lwvMS6GHtFn1ADno8hJTXPpFmWIUAdikN470\\\\\\\/yA3j61kQiB2gdd\\\\\\\/lu\\\\\\\/cyIbf591Ypxoc5NQNwp0MopYTButeFxNeB9gt9mhjplxiHnKff4mYMFmMLPHdXON2R318nSNpDHjWQO8OwnAZ7UodQ6vgGFg8uKRMchuMALPt65+f09Bc3rgTpnAWemHa95hUxig0HH+m4n6MHyeTFMs3YS\\\\\\\/q7isBUQa8KMt8Eg7ibQASG7wu10ND91omekmC2TRUkRXEC+inZHhTvGj4lLTHgexwXpHeBX\\\\\\\/UefZA\\\\\\\/cpnelbIvgLCPedSM65LrJ7zkK9\\\\\\\/AxF18v5EbPsUAat50yCppykjzQPCe6BAZ+KzKbzTllHMSxO1gfDlP5jPK5EzPD2yiJxGXbfBTQMzPN8aJ4sNpbxViAG9z2YES28JzKZDidUnHo\\\\\\\/uLaeTPt3\\\\\\\/zHf5U79RCeuaLcTB6yDg8Fa1CQ7OK8CdIlcZ+cPhmwDTmXFMXtbBG+B7\\\\\\\/xuTnuM\\\\\\\/9dq0p\\\\\\\/algqZuGoEgoXYfdMf03hNyAVqRsDJtmws3V8JJMCYKqow7tn6Jo2xsrl6k3s5baWfUU27Uv\\\\\\\/\\\\\\\/HGZBS5aFoJnkqp5X78\\\\\\\/6dqpC9LNNylp19V\\\\\\\/Z\\\\\\\/0QRXdgtbnp37t7sYGkrvYagX4TQTgMjBognD+ffDoiVPdytsWEdFh44EFaOBsY5QJyZfoKGsVxs5vIQFnZdy6i3ZOUyJdnnrCWUVTvyg36qNyGO\\\\\\\/SoUHPWkbS11VUNUrE13LUhlJG\\\\\\\/OcR9x5ZCJICYAlNARobjlvLZoGVwBBbE7DVWEofFtgKaZ\\\\\\\/6DMnUC\\\\\\\/mJSdpg19YTkzzehlggwIpnt1NIpyWki0RYo2PLUX\\\\\\\/GwlVYEfOHSyoU5fLWYCNAwF6CDWZ6c7A\\\\\\\/XVVlnruolFvDM0tlVAnPY1CXkWh7\\\\\\\/cLQoE\\\\\\\/oIM25NjpIRK61One5feg03\\\\\\\/EzpQVNjY7BmxqaQfIUrr7Rx2SoyuCKtG+zrNI+yvzCoeGq4K6gdlF9DSLUQORJ4WY1Nh18a4gTxAkuYMfAXZ\\\\\\\/JrvaaDqHCvy3Sr+9Sx05G4WqJzUvWQ8m0cNYiO8ENcsTdAYXgVDx7TZeiyWovq77FqSrM4QM8EnMiK8bxBseSKoKcatzK2eH0sot+D+juqQ2T27PWsr1ZWNGlU+vXjVTAlMGRYSg88REtKk72v63yUoVEdzLzZ4YjP455Vc6UrhQFedfumTSr79cN3fgkSwJCGIBhWFs68uF2bhVtvTZDKiNcgg038GOvZ5nsR46eTLo88llm2mU+YTYpKWsS4+9rf0RGZEr0\\\\\\\/\\\\\\\/ni+j\\\\\\\/J1n2oMz4Z\\\\\\\/\\\\\\\/T2g4DK7c7BK3jxQPG03uFMYORzWWoJvXRD1Tb4pRvGrV3KTQvnRaXhqWJX5\\\\\\\/v4xEhxH3wZggCxvIEA5xcv2lDUWx01djoxV4104mjMYIOqSBbxdVWKKZhETxz5+Kzk2lhmm2nyN9zbqI8FSVk5LOQGLrAHyMZvDBq7Kxe\\\\\\\/ZkcYR8mj1uemlJw71mzGX3lwom\\\\\\\/k9JyWSlXepWIo5HSWkPYPPsgZRtZsvzgFbFpFxypfTTXVwYvTQOoXp260wjuIivh8jmwxei4LbVCO9yAy3TJS14WmTWyhvDOtCYVcJTQxica6ipmN0Hr6a\\\\\\\/PMr0mgRqakBH6d6lLBd7\\\\\\\/Ol8uzuHYGxcc7MnI5xNyWF29FY0AQk2SqY6opQiAaMtvR8UTT16NH0V7Rgrfq4a0AaEiTomNgaqtneKjdS81M8bhnrttgfXRuZg+HxZtXD0skKUqF8WrkCUQG07rovyuKDKr7VI5NUt9\\\\\\\/UCtHNAnhQQeGOee\\\\\\\/78hNSNXNRqLGlVykyrD9flW67stgjHrqOTDxYcEzA8Lg8N9K6VlwynAmHOzUtaoR2XiQ6xDVyEUTYk854\\\\\\\/JvlqjchIne\\\\\\\/7yRlBbYks23wJ2dSozNoW0y5V2MovF\\\\\\\/H+6Q96CrdIyJe5KL8KSrJkrcY738ZUBZMlFvbc4ILI6Pyx3beOm2o\\\\\\\/YyQEQYwkeUxShbEA7E5gvjBOhct+JPD5SOzc4I2SaAk7wMZKTSV0IViBAmm1XAORVzgXibAvzBI1\\\\\\\/A1oupGLTlXTTG+n2tezyJoVqDU\\\\\\\/DIPuesItKn4J+FSH5oshBdkxqkcOfOeWqiRNTkSLpdHNN1zJe5aQwLx1lMce\\\\\\\/k9wBLRtNt0ViDDKOTysEeTrbpBOjIBB1zZR51JbB78MHbymdkZCRe8yyaPVDKq8XG\\\\\\\/efFNIoNhn\\\\\\\/7clNTa4afumriDngUXsV270C+MZXnpLBGB6OShLVkNOHnHhTs9ky+VAgyrcLS7ur91QCqNx59RUB3arvc9yqcPh6ktpAp\\\\\\\/0k7jr4e1Kl7bbwooP0YAENMbTznEpBbPZlpmnCswXLsbMgLeW9SEALy4BOh68D392mgyKeBKL2NaKZg6aXIEf00YSUbaFRVZOcvML6aK0vbf4gvvNi7A4GzqsgEU9M6X3j0niFP4XTi10ti3G0kBS3IrzQZXxZu6y06xmBDJ3DofxsaHmr3QeGVpzLBojTYutrAFZ\\\\\\\/M1P\\\\\\\/HQF5SbWsj756OyUgK01o0FmpE7pODfKTtso\\\\\\\/ppPo4QlySzEdMx1dl8ym4r7KlNHJR+l3l1CJQ4rl+xiY8CSOUg9FvwHhn4WpMd8vuqFCbwvkHI6uHRysT45ZfSWGlpVEWN\\\\\\\/8tEPRtPFyBjYGQfDnYpoPWxYuYGhuUoDiyBbGaTKQI3AQ3EnquFTRKvlnp2v5RN7Z+kgYvWRSH41uS1YLn5wXmptoQVV3aCdw8ZqDigu6KRn95T3ghtIhEzGkhPjnKdaTPrUx9RGDnRFB\\\\\\\/Sd120EyjbtZrj9iuPhMxBg4WoYyE4g+lCUYcZ3ofK6pzCapHm1ApSMz9rG1SXacNi7lx6IOUj7oZT3+qExwhBpQkspGrrtieWytnxLWGjocBZvProkcUrLcGHQdGTRkFW3qIcdSODNN9VbJ3xmTMng4wCn6zwqFdRDQdo\\\\\\\/aTvp1nvsas8O4vlpzBB102o1fW9cDsrWzAqWoVoaT6vXFlPsNj4p5fTIL8G0Sgpc9wmQkMwZ06n7aVd0mxh38cP2yjmX6yqYrkcaNsxN4Xbw7+4VbCPvVX1cdiooyAdgza1N8ghz+\\\\\\\/oZxnnqp+1VbY5jLLxHAZa739u0jmJpdPLZpfUnq165CxnjAL2WXzBuWyx19InukBBLJy6t3JCnKl7LDDodu+Bcsbm+TeexP3EFbbMwiV\\\\\\\/B7kWJTJA7+DJBLN09VH33l6R5V4M4Qmf2fpNOpwQW9fXBM8\\\\\\\/b8Dp1GOCOo1RgANwI3VWl4ISQYYlh2w4JP8CyBCOqJT98SKwLWMpve3UfoGOYyhfjf2NLBaL8il+AVxnG2H4VegVLyjdLWGNqMkyjbsPHfpUdsBsyg40HPuQ2aNKPhywsDlgQ7Hm7ogZd35YgghqUymguku8N\\\\\\\/RBLX4Sa2A2auJ1iujZsuRG1+hm5OEL74GEwkumNdJnHsIkkTBv765U6hsDzjB882uEaKTHw7TXkAaBU+Cci8rOSitIGCKTT8PA6XOPyJ6iI7is6kiQtFfEEXgwP7obcRsOVkowhrviujU2R9jlLYXDhp5W31YNyfnAsn6lx+Zn\\\\\\\/eE3xqT5LS8UFmsiqllrRK0fQlc+Zh2DbURrNEDkn76c17L1NBESkUgVhR1J4KbN4EPWCAeyPdMcytN9KcrQRe9e4wKayVVhr3PVNGYZPQrh\\\\\\\/78mkHKR9InZmobdUcTETx7r5YJ3DfLRRoFN+EZkWELD55ozrAO\\\\\\\/CSHiPrqIq9k5V2MbZAjZcWvvCR1Pj1EEdQ6VhPMu2Jsi09vFQnKlanpLmxFEwpf6TCAQlogSTaWs\\\\\\\/bd0JWryeCFcUIGfL+DH0YrN1p\\\\\\\/p9m0Cd7cs4iU8qpWmIklt816+0ipJDV6C2PHLGMido\\\\\\\/ODK6CbH+9cPMI\\\\\\\/t29MjGqDc9ETV9KLvyhl5JiklyLYaO8SxT5MtnWehjUczn9dAv3+nmqArO3ekZGyQF2Oob8Fi78Ju3fUGHAf8RM0d5Cw\\\\\\\/EnS+kvgHpbypeYw7BoGyrnpIz4vC7cL9FOPd3zrF4fXi2g5pFdO2kaNeO7lX+i0ntXEzMWcjmyxO2FqlV+vzaRSGBEE4G+dfcmS4DLaSXM7wvbxbn2kR1iVsXYrmeT+hbon2cOLtpgk7kYxwtF5OXs3P7nxijA2tNBDrhtqJVAWZIS\\\\\\\/RoJK\\\\\\\/Wh7+DotOLUsiIZpo4nrWfnySbQi5DJr+KKjyUAIkEe4m4iYA2\\\\\\\/LPTCIBzrysnWe4MhBOJPzvjLvLn2FKYW5fO5eSnwrkOOQiKvqhzcJ7aPCr+X4+olOLt3qyj7tIgQhNL26gJD6oJ2Wh\\\\\\\/NYGTo18NMvE7ZtqBAzxMr6juXxQxZubA304m8\\\\\\\/0GjJk2158YURZho8AOwzsminxfWDzqWAZsXl70EMAhwJgKBtjwk+82QKbIrkTAjpmQU2Wb\\\\\\\/IlVsMn9cHDDD6YpcJZzbx9O2KZppVZpcTgrTF+dTYYNVmCbLmz7E5B711XM5bFHZasijYxyJF+accQp4DMfEylryaWVC3Ta8GXIxpUW2k\\\\\\\/2VaShEX8psDlcfjvLR7Qg1+jP0bfintmjTIgMLz8VPFZtz4+6BoY3tmW4wCsy2Xl28tY+qAyy0QE1oNobYkT4Q5dN2VJtBq4U\\\\\\\/SG5JITHqEBaZ7tijML\\\\\\\/SvMsH4hcPyCMbD97eLF6E9K3Svg\\\\\\\/y6qeqz+sS8gaDWIgsEucDvHF24SBZxntPjNK0rB6gO+mGSdkpvwX6XSgOFJnhleN3U2iGyGfJyVThPcxHqMWvWqX6p99oZr8w6xuCJSu14VtbrXvkv0ZqaHcwE+d7f6OhXqTG\\\\\\\/A6EpnmtQIXYtCTS1xPc7CWy2HvNRbu2mkMB+iZyel3tO3GHzcR9gyDbSj2Yp8FLtdwko9CR678a6pyUD2ZoEIYYC9MlR4nAVJvbJjC4rUOFOs1if9v5DmRcTm2myRkWSjxrnAPU4BUr8tQoiS33uVmJf93ipr\\\\\\\/k\\\\\\\/qPrAlqWUaRymdInOam\\\\\\\/bmKcZuxEuDjik1YLfgUwd4HUWH32nuX0i3b5jIMJC0VZNk88yHivjmsL\\\\\\\/9NEYICDVbapOukWPM1ACePBlYTYSyGQVwIipKxppnkYgsMGhG11nDrrD+csIF4GkJDvoUNKMhgSyEdRHEKHpCCD8ELhxVilkF6ik9h0tKeg+lyV2qy\\\\\\\/SVNQDVFprSmrCkFTzNeu3\\\\\\\/26qVuG1lq23Yhz6WciZpKHs+KitXJVlWyXsbqVe4vua9TDb91gIJmDwuLI\\\\\\\/FciREpgyLACUvJkPht\\\\\\\/XjUqQUQpVyMPnNCaHTvFx\\\\\\\/OOMKT9ajUGHkucUAWHjNKD32wKw5SP8sRF01jOnhqqnryMLawxb9FTepAqQqwAeglhWvaOEwhk9zvnMe9Slfdnndiqd4k3L8IB\\\\\\\/Z1tG9HhBWtD+a46CE6wMu9JybT66AsJyY9yAlUwoL7e+8uGRzXcoW5fQXm1LVMsAdjG+Aoh8GgX7nRIPZRzWEMJXAtqBAHd2D3NjPmCWg7aaaL3JwlXYumwoCjeUGF+ZW5RWiKTILY6y+wLmpraDmQ5jOTzhU5DQl652GcR9uH1ND8SDzjfFijKSajx8CimxWedLYY9HQO9RZ0pYzsXx8Gw7MWs57SdAFu1WuXhzy+RzfqRfsWDChcsfdncvrF9DfUDJmeHwxZA1988+kAmwbR7\\\\\\\/SgPmp2NLYuxflokK5kDuypiavFiFT6oWd6cfAhsKwxXsxOjrH\\\\\\\/q+HJF9igpnsM253k3mchudfZjzz8zmSmFNCw\\\\\\\/fE+FOXhFdrCKyTeNoMB8z\\\\\\\/rCAlhtsp8IRl\\\\\\\/x3PcRI40vN8SVWFck9V1KGc964iyHO\\\\\\\/ciFM7rUdBSbwftAIyYe+LxWf6UiXXTdS9r6yt+5juLcikyjxYr4nSb9NJH7ckVhGensGdxRVlOfA70PlzaqmYVsDlKkb5Rll6hbw8iuqrREkMH4q9HhuSNHEnnwJDFrsZVgOuzeN\\\\\\\/Nv8tGyvacuwCyUu7UYmCFhL\\\\\\\/iZL+oETiopc2cDKcoKFnk8NKK0zvH0Zzv4tndb8lq\\\\\\\/SvyU2Ay9DlDU9Yk8OVENdnv9cEDdmD775zUw4iJQRiOqF+onqBQBY2njJ64ZydKzAVF55CvaM0BqcGnqk8y2dJWyO0b0R5UM\\\\\\\/xTSlYagOLWKwAzS+MKhXY1K0bUUm16IK5H34rqINMOHZ6aWHd1rmz+mTkuoagHF5FgwICJX7HZrX0f4jUDnBXN2IlaCqNbfqzYesiJkDFpyEPD4n4GoWr1eK6waUpn38CZeG9P7g16oMWxeA6WgSk1tc2DVmPKONhfck2\\\\\\\/ElAM+tGpMROHy++SRrFw3\\\\\\\/zs\\\\\\\/cQAzAMZGfQ7ehKZ1qwJJzXBQQIH81zI4ZGkKyyb+S\\\\\\\/zdmvvWBykXZu+hLwant61uxvZMD+5D4xYOUr7nUeytx2YY8bCHF\\\\\\\/cD+aE1U12nhrr0daYUQJmu24M5Sjo618vlvJYW+SclepAyeo9mLfrrSQAB6jXXkoASFr+3sCZLKGi5KuIBuohNG5K2kiX2aKkmkGhVWBykUl90VzdgxagE+SwaA51XKjnS9alJqvIhLUi3J3r9xOE8ZE3vzDEcfCXDb2+\\\\\\\/KNmHpy0hBSoSlNO5Up52gmLPZCp\\\\\\\/bn60bT46byozdB\\\\\\\/rVKFXSYaGzEW5WU9eOWHleQh5bLJ3gOA1vRubgXxNFLPsVRivJAnyxnvaaqhJb52TZlzBtnVnkHPdk4tQ1nUTLw1lgMhnyDT3kZYLleZGGqxIwNeCrFL0CE9Hlr21lsfKqawrFPieDQyYGJPcJUZ633NszAve+CoqS2Ly4QkuqwdXKdhRhAgNhNDmi2Wcg0DrqAr0zNV2RZfX0Fv0e6TKEFSBWHBSqSRl2OuMvnYXHgZp2wt+4Tjj6xjrbwGxJ24OP5vx7x+R8BVT5X7lArfBf7YTf2K5DtcOmbpNGHk5jp9gxcnWz8gcnxW9tvvvk9vv+NoPV+VXD2RciWdy\\\\\\\/kXwO9jIYbiWDOSe1P23jD9uo8wNU4AiAtWlJ+TDw\\\\\\\/keKgZVnuerae\\\\\\\/siXyboGX8KpH875A9B53uOPUMNmYZFsvQDI4EjjQ8A1hbf9rcEZanKMfrWuor0MPftVxR1FNVLeAIdJqHY1W0MNX+qqzQAKGPxVRLQvOL2jAx92hKDN\\\\\\\/uG2taJJLbgWVwCEO9UQWTcqJprDmr8pLyXnxryCCUlcjd5rgVhhZagjec3MBZ94ewKcGBH4lc8JaQ5+sPR6TPNGcgMdNstW2Hgs5WHnYtTKYKFOdIkqkukvFE4i4K46a+un9NWISadU1fR1RMaufkC3aj8yEJ7RW5FKrIcJ4p2eUU8TiCvO1HEoVL9MDQa+VGBmS61iFSAuakzhU+n7fJZHvp6qwTK6M0p9lfxLUSQ1zyDCUVAjybTvFs0NbFiEOAxvryYE3tqbkEDbDz5ZchfwVVE6J7ESwUK95jQzv6HMCh06ksnRJOogTzY8+TdSY2EpnQ3j9oouI+qitPWq5BnB1cO9Kz7UjauWgkYxoohCOGFHBxXbut0Xp4bSIGfaV5P3Xuhb\\\\\\\/x\\\\\\\/FzO+CqxUTBk9cCmWudRWMLje6dK7GKm+tMvQF7oSwhMY1Q7L2rUmi96PCHw10HdkNN37kSpc4qdPcKEGFN4T3uCku8mqmCRWiB7TVdb6JPNwymlQWufWxzekn7L3\\\\\\\/5W10Bt9BA1asZ\\\\\\\/aj0cDQVnFcPRIZrPNNZ9Cu++m+KmrimVBN4J3OtVn++XMjik2XRhXvtshmLWSjMF0VbQNHpMGvqnswKnaV16\\\\\\\/WMyeHbn\\\\\\\/obL2SMRyPKvMS6xPziEMrVdtSAKUWF0QrgY6o3eWg7nLu+tiRpEzoaK1UmsRMc0mM47isQwPgPjek46hzvd72xYb0kYjAc00ECU21fT70tXLIweCxnU4k38UECeDr2zyQ6E2m6AMir6YMmtmzASFUPVBqBX3QxVon5A\\\\\\\/G\\\\\\\/WlA84+CiVoD0VWQf8Csj0Q2rW3b6b+BY+6OynbkB\\\\\\\/bE3bhspeW\\\\\\\/3kHhn6TFZhxQMRsNsLbBREfYwuGgGYuYvS5a6sT05i3HpfRiDRhHP93YD2yGjmzcE\\\\\\\/avrAb9YQ8sJUsnO2jlF7VwC6kG5bNLVpLiNBtXBu\\\\\\\/+Z1Hsvdvv\\\\\\\/YBSEhL4OvDHpyr2YZjDoMLpOuu3Dd9B+Z24ojfdLJTpd+LDqa6AF5yUUsUu0zFvQsgmRTwmnAixrN5BeyWnNowgWrGLwWiB1OK\\\\\\\/kqe8xjEC970i\\\\\\\/CYYyhxrhB6xa2hoNYa8tdkgr5r4oEuav1Cb+yfEiAuT1qSZ1pGmVO9soYXHtPlTXvYy1WcoiXYYa2lak9TbIPvbWc8LLJMrDHmzn16ESZELtvhVSq8d4HBD9MjiWrydaNVfEAzgIbra\\\\\\\/Xkyr4Bx52T61t0\\\\\\\/HzE4zfTgYbAUPtrrTCjQMZ1FJcZjy0GFDb1idVZKkaOhTcaIBpjd+e9I7AdQqVah7RBuKCRtt9rWLkkBepPFojSdi1z\\\\\\\/+1Yv6PiDib+wz45paEhxFJSPyUhitQyHe6Hn8xbI\\\\\\\/w+gkmIB1w3PimtkvA\\\\\\\/c9e23JIhRPbt3B2HQtjGdT6dm7c7yxMIw+86BArR8I0URwkk1YaG6MiKaRDPa+LDtcTj4d4fIynWdfYBElVh77K3sc\\\\\\\/JcpsTNSNQhgnbi4xsOb+MnfqO7uJ\\\\\\\/kmhNy1F+dzNmeDrypUbClgcxEDGV+JmJpFolgFgeaXTv0SK1J0X2QwxinTWTKXgHolfvIAt6YoHmJ6qer1vyFhoF2Yf3wgP\\\\\\\/WHYYc9VU\\\\\\\/NbHkPWx0JwzzUagHjukoXmwywCi+wua4zHpPMDoLmiZtMI4Yf7KUTbIuzED\\\\\\\/ccuD21CwFEApqgiRVP2LjwSOTAw5isL7+LCIfHJnuuYEpdE7eI9Cutn360sJ33tzHt+7XDT5nMEj+IssaOl3b+RtlOvAYy8+zJ803CX712ozlYTM3znbJz38aU+\\\\\\\/IlgHg3sqoLO2myb0dqyeotjruuZT+9hYFoUDeV+FW8FrTpMmBLmdAihEh9OHVIUmWu8LFyVqotdLJDyqkPnQitgQFNTCNUTpeqmX1i80MKMIMuOboIi1cJF79AWXpKYcx0epnImf0xLuH0\\\\\\\/TM9H+1DoWtBGGrdMbGN7ERHuMOCxy89sbytPYkGvFf\\\\\\\/Dicx6VaDhbTk45KHNpLTcZ+V8IqCiwgAeTNhGraVxaanPj4js7wSbkmgZE2HXQP2ABJCSXr2rPicnvj+yXM0f+RM2Bt7q9pX9Ot3aaZvnpoIYtatvw1HY3f6nSruEXPh06POzwaKwCLp8mnz41LJwZNnwvIAuojX0z5zIwkUQRkqy41KXm7irMUo7muypUMsfWno9OsLKrmGkLdGKp+aQQAnufBMVIET378M3sMtUBIda0plPQa28dOkKU6nbEPglT97Veg2aSW8UdXdHMnu+e9Ur9gaxjhvoC8dUrhDdaGvuC7svhCVcLWKTgeHq6NPmQIYGmBA97twYxjcJqyE3tE4zICiHYJiXQ3h64lqAP+COuoh6a4a5zg9uD+XTFDhCdqrF0PvuszgZ1jEdQuyCByoNLQ3yzoAJWw4N78+Y1qr9+VClpt9bArlMe5KMbCvBBZ4BMbIcuB6sTJrjmEe4nXL6fCx2keiyxTIQRzLCXZvlYvqKgEBsBDzBqvDEX7iAr034M2IMDMETYBiZMTSbPtJHo6BA8RWtrKcePcR2uwqmA8fsbdK+5Q74KlRyvZrMgqKOifBa4RKEc8I6OzaxJHSYeBi12aa4JU4D\\\\\\\/06QIAngCG0F6cpYw2mqN6c1d2XZVR6l6PRtgsihlawCv\\\\\\\/fpJbGyYEw77B9ucLaR4WuRqKuO6dT0WytTqDXqQORLWsHWzSLxONUNP8ts32NZseAfzULDccxwqucuP0QO9fBDaHes\\\\\\\/OFbTcObw9JyJbr10qC0JdUAyRFHC40DygNOw+T4EYsCV0s4A\\\\\\\/XmOZ1EBzBIUKZTcfSSoCT9d6JQHa\\\\\\\/v\\\\\\\/QMpC37\\\\\\\/be7tgNYhE8SS4JznA1VE9Tud58vlBS1fZizX7+sbYrYYJa8LiBn0ujQ8HzgVbNNTyXwE9DMD2PK6pwfTSWufyGL\\\\\\\/bO46OxxgBX\\\\\\\/R2XSbBxL5NExRMSqy7W+iOvrfPR4GlMp3IRdtaNLTp2QH+U5KW2gHj13GtdYdIz2ITCrl\\\\\\\/V4BrI28IzJsmdgfeL\\\\\\\/ozLLbxySOMmzy2qRXEVQbUnn93K11vlyNcEy5QzXZ6f+7OCBg1gdc0ZUg4UcFv188fOlinHU+AB1MjjSLNGA8+EPYhotdD1WADbZUYclbJH7lxXO5JFj5DD3jpK0ukde80nHoqm7xUdINlL5y\\\\\\\/uQXqzEFjC6imKCZUw8E6ljytKrk76Cx4j+gH+RSg3CTnsItCQ4jir0zeWnB+4Yxjz0h1a8aAOvAE5SgboExIW2bkqAudPcLsPEmtyzfJvWdoQHCW+fSCo+zml9Xj3rI1VL5HLnS2qBeB2Xk\\\\\\\/0I52RRrDNT9sfDXAs6RV3tIDu1OzXV0S0eiejfmONP9QbFNUA8tEOVNAb8ZpxPoXCADIbVlj1BjOvqV\\\\\\\/AP4PND+tAFgTTkELv1AccS\\\\\\\/A31Qh18Qfp8vLZXyZW\\\\\\\/nuT3gTx2EU6EZMbY4MjashmdFrIqv1e13znJXJ4DzgEuVUqWltvnEvCx0JlN15522WkKLdmcSPC9pfe2QSBjqAGNGLgjntqhCAmmp9gSzdXnRY8LbU8NhxYBvLRGsBvxUPtYA29wMr4b5yiTBIYBsCRhXtojPDqK2GZlI2nZglCAfNw5KpysPmZEJeICCajL5EJK8Y0A3D5B3EMC7MLtDHjNKiqZwr3BxzO4RRcvv9xKYHtRBlHvrYQOhl24c\\\\\\\/sR5Mw\\\\\\\/uGP5\\\\\\\/cmeKa9kKEONtL3AjDya3a+OtQlGuIX+d+QZqppO7oAyyQGSO0rqMkf3ZQK4n5ogpt4PlXNdtgqP093m7gKBBHifyFNg4Lm9FuZTFV3aZHun+yrDb0oO+8HENdnV+jkjj1PeHEG\\\\\\\/Hjhnv6xC4+nKZem9bpU+Hgvc6DzGSkKMa1gkVPpErpqzWN5xDX2ubFcKeYdQjNlbrQ18QnPQpySBGpFBY3ZcQ+h1yoXIM9jCZsg5GmSYYq2bvqPEC1m0Yfad4XA4CwP\\\\\\\/7NsEYxGiLCqGUgGX70Tsp7TwHMEoFsepwSiOB4NKb9GNIUBe5xoy2z\\\\\\\/88hdfVzjJFuIuv3OguPLXg0osqh9so3am69t3ykXL10XwCWm9cy1IDNiGRbXREdBnd4hk2z+Br8cHc1EWkeYuAJH+bs+k06VqO\\\\\\\/vKUyIAmPyTu1yK78TYz4hmA\\\\\\\/SMTCRqVKUYp0OI8FBM6WyhlKZDs223KCaaQzuIXY9T1sWgJDv4aemnz58nvenUjZyMBLps3blVNI\\\\\\\/FXNXEyUcLoEucB5SSfSp4hNIqZlLJR+4tA7NC0CkuX0kIW5rjMR9WpZEURCNX0QWD4YEbkc7wYp1hM7AWwN5lfnQYkHXvmn7xhTKatfc\\\\\\\/EDFMKg2G92WFBl0GfyDJwgbIEIOeI3goU2yJAiEIovUCWbXcTvRL71J80OjGPNNf1IlllZNDIKyQCW39\\\\\\\/xfmaMsBsUQrYGygYZmvqG1P\\\\\\\/tDeHKam2VMNwwHguFtSAp0eCvgL7tojqAfRONuVGe7PCQMTmPYSFzanF9N7A\\\\\\\/sfPpCxs71GD2oWJSxWw7mKVjMWkO32nzTZ8BCkJjq+E81n3DdecpBOlHLRq8CFxLaPESBy4LJ8ZcxjcNw249T+Jrf19YtPf8fEjQjq0O8PWzkQ7+g8teMTGN2bmA5QRMD63TXQXQWtdbDBuvrC6BOf6yqpEJFJ0AAD\\\\\\\/p1BTpRfEc+lZZp8oD\\\\\\\/vHceWAegfjQqL+rDlTAwQTr4mS\\\\\\\/dYMbw24AgMq4xokiixyuQZ5\\\\\\\/C3cYDtHKK+b3pB4LhKL1A4i3XdWLQzbPAlKjhOT6AgTlY5hJbVWsQVNMK9R6UAT7EGVyGZJGeYVuKZD\\\\\\\/vV42K1oT49qejVFYGLSm0cyRARW92jiYxBXKgrAc2nJm0jmmDappgLuHD7Y+97k2xSfr78H4Bue4SDLZyXigLPNmHUhr\\\\\\\/0uD6Ni+MXAVC6wmz7zFOGvPEFvwjr0Yi0pd83brXf+ipXK+pDSjCVnXNvzzSurkkwb7RQn7+nrp+yamto5zhkHXpjVtWZDcGNAWZaMVwDpK0HWN4DKZT5rdX0AEC4u+jC4PHu0gZOd+4S9XnsLTM\\\\\\\/iCoHitUWItAyR9OFIzMeo9x8yD+h2E4Ap2gp89fHvD5fMAAH6jmZE0ylJGnmOyJgfefh9Cvxz2cY77Qwu0b2X\\\\\\\/d\\\\\\\/a6N0tC6KmbW4930XVuasB17Xsg4XnNyWY8XWCX5Qvh58jhg+8eu1GaPD0T\\\\\\\/zt3mdHZ8R0iEBNlh0KtKjqiEWW0QbtiumsXRbxXQ9xD8OoXAzDyWGDx9w9ehG3jQfOrA\\\\\\\/CR4bZvmLYnnuvhOxurKRLvDIu0iLANi+YTOHQx2EU\\\\\\\/LZKmWauxljWYflng5se3fUpJCuEAe9z9bKL8Cdxg1a6VGe5UCLZgmr7s65dyD1tPrfdvgU3m7OBy\\\\\\\/BH8OeS4+e5R3Llf2VCgjO29Nc2K2+kA+B2caeFeq9p0HQS5BMKozG0druponNX2QlIXVIXsQ5MSEKA33sapkDtyF5VsmHnLbg3r+eg4JliCxe81bVmUTCaeQ2O5jzXtInk0ResE8D\\\\\\\/\\\\\\\/e3gzIetHjzTUmtOddAgJ1T+Td99\\\\\\\/ouEPY0IgTy63W3pH2BpJdNaB6MsGPsqaZldLnBmH7UJpOctvYVKFlwjED4uk7Ao8+\\\\\\\/uj0NbUouCKW3Mjfik4\\\\\\\/Va3\\\\\\\/l5TtvkEfuMiHm55Ie6vMFKjnRveV4dUDkkZzF5mAVmU7kS8W\\\\\\\/TVlFWJWZv0qJkrC6jbB+DlFxND3dvYFsy5\\\\\\\/aWFiLI\\\\\\\/L1mB9LlEFJeXHu\\\\\\\/VRUEy8X+GsSQFwdK6NZRa8v\\\\\\\/wfotJ\\\\\\\/6Gaq4laY+msRYZwYEJUPBP+TAebrnC7I0o9fBMahqWXChIh4+fjls6N+M0cmzlBF0Z0AIure8wEB7Sa\\\\\\\/i7PvH3gck8YduYcs4V84wDcf9SGS9tln0ZFCzYaNhy+CfYxcRL9zuinQlS\\\\\\\/evPyCtquLfqA\\\\\\\/Gw4WwcnaSxd+KTC7X+o2ca7v2HmfZPn9h9cdxZasRP1+SK4gqxfg0fK+qtbGpTpXOrWpEJELMPFxNDXT\\\\\\\/mWh2RQD8eDNygO1Ltn3aAxa8HYXNmipIppWNhAA2KbMkPFr1Lun692RgCcluz2iF8f2xgYyBpPPcPXYVO+YeIu0+0kIJJ1phZMpGL1RZxw3jhc9di8t7GYvoJn6kaodTBBTlH6JRZ9xzFiPxptDzRS\\\\\\\/BlWkxdIk25\\\\\\\/v3ko5HzeTWi5\\\\\\\/NDz8a48VLLFDiQWN+VfwPF8DwgaCK0DRR3Cyd\\\\\\\/I3CBe2IyTwOuCIsGlMBHvKjpY\\\\\\\/oPGAVisO5iBuWAHGU9DqaHo+Kd6mAAKnZJqdAg4rjxF0VW35qDCyHEsndzvQF4eQB7GgON81gXffBfgooeAyj34D1jfStQ2ibNZNnf1ZI6CzzfcBkSTVU92v54ewOT4KVprEJ1zweS9VSnncF0lm87m+04e3GoiiroOFUi2oeoGn5Ki1FUETNn\\\\\\\/ok4Xm\\\\\\\/K3IKhmrvfhSzN1CmgXiNjdGjC245edjoCX8kS0YQn9p7oWzQdAVv4b70IHgydV0tr4qm0rT99U7G0ejzxUZNaFKulk3tW9Z0PCeQ51aRsh7DIpV7LeEYNizehNz5B7B2EioB1euqrHHDxTjq5n8u3Y4sa9ppBATrI93EBQG9c0ah0gDzyo1eW+a12q5cWYiCOb1\\\\\\\/6IuXzg2HAmgbqM\\\\\\\/O5gG+TLZWFCNjIh1FV6VnGPf74riO1uwuqK46FcJzQhklPpf4ziPHOSmFNDsyBPXnZzCbomUXV+v7D7FWV6E3N8IYJ3q6Ry2MLGjeV2jQMf7LNLezzrEd\\\\\\\/TTzPC+x\\\\\\\/lMGSHs68wzm\\\\\\\/5WuelooZyBoW\\\\\\\/FZXePIHx\\\\\\\/BVNip7d3aD+bgBQVNqrtDRF7kBIUs9rep1dLtTLvl0PIMpxcxAFLzCZnOZ7XMB\\\\\\\/M5M5u0dhHsEKGtfE4\\\\\\\/yOOKBxffMuiRhh1zdL37bSjJogVsNV\\\\\\\/2GF8\\\\\\\/5ZmdPX64jPJZcZPaj0iMtiQk4gUwoexBPY341Ag0TN93f4JPLucMEDZgTp+s4JreqVUlNf7yREEU+LSnlDdfGT\\\\\\\/+BMCiJZeeCGxPSLoIv\\\\\\\/WE44LUK4htYE\\\\\\\/puEHYTGeOTY\\\\\\\/LeLugUU4BjtGXj5pkI9EZsHUKIeFD7juqNGHXZ1ZXgWhr91cjMF8uMOsRLoGAPcBxcO8Th4qpk9rH3zP9NQtQ4MGJv2EXPyytpRjpjCE88zrOa+DIQ7mLXzBW+6YQOQ7iTxXX2En8bw1UyRldrmpywMMhlwgCL3b3+KGqBW+VVP+yIC8pNDXcSPscpAYibq40Bi5gxPAC1T5TtESz5+i7WtO5dwc2dowLpkWQTnUuhxKll6izj0Kyt9pwZslN4vD\\\\\\\/9\\\\\\\/8qa5SrNHEbEmMphbUI5+mTKfRALAuRBb29gQDjiFZQhywO6YvHDrUUW6Abyr32sWrcgGUU9yvHkQdL1hN0jG31R0JAAI6\\\\\\\/PZxHD9kME2wPjNmgX02iOKZ1FTMdE6qvtxF1jdOPaponItBRrrpw7Rv0EES4TkHbGxTqIF21pPPyrCaCZ0BEiec\\\\\\\/PLQ8Zt0MRwlhqRK1KqZrTJO0jPc3h7J0xTneDbD6DAXOdzgHPfap5eCNeLfDB4OkISxPUJIOhGa2CdPW7UqP+oT+ZYmburV0C9kyfdOsorisYoxkoueZcJZok8okdvt1+woDPgT41UohiszkwlWjjJbdsMhLzskNja26tDcaJr1xJJ2tYdNuuddlQa+uZJGKpFxnDaOZdnIqkj3ACvky\\\\\\\/Thmohi1z692pu\\\\\\\/ZE8hnUt1MJDwy7\\\\\\\/nZagdytcUoCHnQxNTcsM+ddFhxVePynAyrxhb6E4Sl+XQzapFOx8BgyvFUQuAYWFZ5JyfADajP7oR2l3OROqX2vfETiOe309OBridGPLFBFLQYXmDa2GmcY\\\\\\\/oDtVWVBNCA5mlfyCJXUE2qf54oUK5POTt314iVP54RkoJbPz6lmyfAIH+jVSKzwEYG9q4jnZoVnFnuuTPHxn1HqQIuFX\\\\\\\/IzQRJrSdYrWoV1s79jMPcrT4f613tHiLnMM0W0HUgoa\\\\\\\/e5npUe2jrfjQhFUpco7D+bt\\\\\\\/yOXM5x9m27eYqX\\\\\\\/cTn\\\\\\\/UDgQ82llYjSN63ly\\\\\\\/FmZYZo3Bh9xdhBHf6wdLCXkSSj0lXpfdPPr3ql6XClQ4peupMmksznXrFDXoKtQRuf7Fkh0VUAV85tSrC25vwZnjsw44nNrk+8Ev2pEaSfyO\\\\\\\/Yi\\\\\\\/A0W25aw3vdkpaeCOOOrunc0cYv60B8IkdY4PBqCB\\\\\\\/3F9itjowQ+F4ngZ2+6kQEh88xcZ8GckPi527S\\\\\\\/X+GRNVN6uHQ43VSme7Y7hN9g0uc8XV9xDix48kN2etZoWoxXsnpr4jzw3fTQLV7T5APiQbfBMv9kApqHVidFZO\\\\\\\/\\\\\\\/s9oTOii8DXO+0+OxgN83XfrfYPU+Rgb\\\\\\\/aMM1Qtdbpe7Qod5u\\\\\\\/ugJ+vsUWhYJ02hVAWqEEFjFc68LbTb2ZkwbdES0fHt2EYe9Q9+aKipOGPk4Dj\\\\\\\/yc2f3i6uUid9hJViY6GN3PHgloGiSfUEujmOVZzskYPY5sW\\\\\\\/pQQ1PxzsNizq7Mo\\\\\\\/NCpKKgfUsinxmSlVn3d\\\\\\\/APldxzgJk3VIK+smNOodXm4SWcNyBqF\\\\\\\/DJu7hSLcb6o3\\\\\\\/0j9FCTQcK69Ydc0uCcs+8Bzw3\\\\\\\/FHnBwDGzp9pprOq7\\\\\\\/mEFD0XhIOblFWq34MmTGISCCq8wMIy7Ykf\\\\\\\/KlcliKZq5FeRohUqODF\\\\\\\/wN+2Z\\\\\\\/K\\\\\\\/ZYyqclDQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022N8n1EeFNg0VSH1nqgwHdhg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226035cad0511df16709d1cfd8ff99e17d821f74f459b9135f787946055a04e511\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GzFbbGnP1Q1IJuXNaXHxvIhmWAkJNmXQ8kEXTiKmp9bhb44MCW9y\\\\\\\/Xyzx0yVAi3EJziYlElcWbJr2Sbbc4u6NSgJKid7umNb0gf3+sZcu64FJ+y2TyNXOaW8vM88\\\\\\\/vw2GKpT2TJAgUsajcXsJjnXrv12ez2v7robgZnzAnl+t0eaZegds+8WPw4Evjz4DU6Z3ljbykPqCCj5MQ5To5Ipc73f16B8wpT3nQU\\\\\\\/RzztMNkPyC6FdVdig+8x\\\\\\\/2zL6ctpdHsbb32fw5KSVqAxGeX26NbaVLGx+w6XTGCz6jW6vCZFPxfeuJoctTY5h5uSX1iqzGwsMQ1Xo2CPJG88HC5lm+vQ51+gfnKJyomPw7pB1n6TdMserv5JXjnLHGz4xL++ecXfYqpkxjih8jw0lr5YD1opeWqGi+\\\\\\\/c5LPxvpwrVh7uBvHCleMkNcjxZNM7NkBj622fSwfvxNb0B6lDifVdyeTf5c2axO2QYk41mKipmjn4gwOilK519N6g+PEasQ57yrU+TgJdl1msPOkjKiZuHstVYCrzMkzxOBLOcBbHcf01wPgYS1kmCZnBALUm8imQwn6khKl+FHMlhPf6hNd6YOAhkXczJOhcOLBks4Azph3ixttUjY3HLbZJxSOGhcj2erC1joP4tihTw1GP057r9ByxdC60925K6GGul5TxaAszDrIGJo+G3NK1rLG7626w5DI+GKsljKJpUUmK9xH4jBnNE0P6HaFLaGtTDaoMEKlGS3b06kEnUxTJERNeqqrm9giQmN6rw1uXh\\\\\\\/w8qRl7KqB\\\\\\\/WwmtHZxgD85QvNlAfV9Jirnv2Jsaky6Gc0D9WmYdQFqIxqCkcV6lc21LbOqlIAPTXiWHMHEc3Wr85BSJDvP6DLCJ7pfFwGVrb\\\\\\\/H2EjqUvw7MYnq+p+HeP+vxQd9+F87GRoZJgxW6EQWmJR6VL3nZiiiliUq+uOAn96td8OOAImu5byfIT+\\\\\\\/JHMejJItsSUUHwIOCrI2EqiR8PE1hRITmFEZeAbwcv2ettzW6D99KSgdPtDgiNXV9N9T\\\\\\\/oS\\\\\\\/Upl08ZEwlCPG3hUI3FbCKisiPP3zoYS54KhWMHmXpIDYqtoFOV4pXlCG4FMPwKFBKUbLEWk7IW6eM2lUQv3dggiAvrBGLNvQ3Mo68sRUb0cF8eC1S0Cvi9k8fhHy2rpQ5zEOKJTwhPAbe1MjILnb3Hypot1sp0+xmmYKgfv3YK8WFjeOopUEq\\\\\\\/XogMxa7SOr6znjA40bZ9oeFNYopjoVARpADMHwbseQgmsbXqwKZXZoTtc0A8ZveCGVEFGI8+2b+ogViNUCv9xweoLPHlsTqVpAkmzHwVjyOSRV3WGy4EfNNddMrD+h0+8jwDjo\\\\\\\/uG1JTDrlTPO3Rf8\\\\\\\/70FD+CDrV+KcDRToZSgREybCz9DMHmotpIzrVR+JyIG2RLK3m6OcF11r1X5wBjdDRWriaqI1I\\\\\\\/2CFYQNyJB8m+LZWQxv1paYWp1vK3VqTe3dTX+oF3ETmMFcxzIzlvfb8fbOd5Hvs9ZodOiHTZvpVkRB7aGJmwnl8CcoFUJUhQmCdwBxO94A8s1S2LzCWNAO9mfdZji4uyP\\\\\\\/j9gvBPIu3plLbLxZuxoy0yB0hiAwPRePJNLeu6Nc1xJ3eTWSEV5Th2R9WHcnkXOj\\\\\\\/Bmg4zlF3TNwJzOv+bMDnPykNhxjDG1w0qzqIHJ6CVXiGZg6fYu4v9Hcry6+dwpgr5Q+qVMvf0Pw\\\\\\\/tA+xF\\\\\\\/WBT7s\\\\\\\/sTjT4zFsG4onu55A8g8LpIBmZ2c78QABGrfW4gvMCLyraxqvX87fFY3WNK9ZkwTHVWhsBzB3QUslkvpowBSCvafebZdNre7QHi5OIrPVHW6fIzOWrVLKdhUMj9HhYGCd+7MaZsRvsFW9PTZ3+fC69km6rShM+5VoyY2gUUMKLUQg60nUXIeXHp+dph83FsJ0xt86eNUSF5tjdw5qi\\\\\\\/SxqXao2YDd51oZb166nYDNwUj++fTxARf6XN\\\\\\\/fh+woj2CltwYlKT3U6GRKJqzuA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022nAOfQ5lQeNCjwh2iKuXePg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a73357c28bb0cdcbd28de9f2ed2d71b530533c5d421789584565fefd18552bde\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459749 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2317" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1LpeucAYu72bB_hQpcw3BsoAComZSD4s7kdOUWsvSb4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2317" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-H938oMrUPY1x13K-PCM_OwebIa6w33rVkdy0giSTIx4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/12/2307" title="Population- and Sex-Biased Gene Expression in the Excretion Organs of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/12/2329" title="An RNAi-Based Suppressor Screen Identifies Interactors of the Myt1 Ortholog of Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459749" id="node459749" data-pisa="ggg;4/12/2317" data-pisa-master="ggg;g3.114.011957" data-apath="/ggg/4/12/2317.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Influence of Outliers on Accuracy Estimation in Genomic Prediction in Plant Breeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sidi Boubacar Ould Estaghvirou</span>, <span class="highwire-citation-author" data-delta="1">Joseph O. Ogutu</span> and <span class="highwire-citation-author" data-delta="2">Hans-Peter Piepho</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2317-2328;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011957 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459749" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sidi Boubacar Ould Estaghvirou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biostatistics Unit, Institute of Crop Science, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sidi%2BBoubacar%2BOuld%2BEstaghvirou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Estaghvirou%20SB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASidi%2BBoubacar%2BOuld%2BEstaghvirou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joseph O. Ogutu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biostatistics Unit, Institute of Crop Science, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BO.%2BOgutu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ogutu%20JO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BO.%2BOgutu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jogutu2007@gmail.com" class="" data-icon-position="" data-hide-link-title="0">jogutu2007@gmail.com</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hans-Peter Piepho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biostatistics Unit, Institute of Crop Science, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans-Peter%2BPiepho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Piepho%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans-Peter%2BPiepho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/12/2317" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459749" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459749/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2317.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459749" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459749/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2317.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459749" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459749/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2317.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459749" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459749/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/12/2317.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459749" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459749/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Outliers often pose problems in analyses of data in plant breeding, but their influence on the performance of methods for estimating predictive accuracy in genomic prediction studies has not yet been evaluated. Here, we evaluate the influence of outliers on the performance of methods for accuracy estimation in genomic prediction studies using simulation. We simulated 1000 datasets for each of 10 scenarios to evaluate the influence of outliers on the performance of seven methods for estimating accuracy. These scenarios are defined by the number of genotypes, marker effect variance, and magnitude of outliers. To mimic outliers, we added to one observation in each simulated dataset, in turn, 5-, 8-, and 10-times the error SD used to simulate small and large phenotypic datasets. The effect of outliers on accuracy estimation was evaluated by comparing deviations in the estimated and true accuracies for datasets with and without outliers. Outliers adversely influenced accuracy estimation, more so at small values of genetic variance or number of genotypes. A method for estimating heritability and predictive accuracy in plant breeding and another used to estimate accuracy in animal breeding were the most accurate and resistant to outliers across all scenarios and are therefore preferable for accuracy estimation in genomic prediction studies. The performances of the other five methods that use cross-validation were less consistent and varied widely across scenarios. The computing time for the methods increased as the size of outliers and sample size increased and the genetic variance decreased.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/accuracy-estimation" class="hw-term hw-article-keyword hw-article-keyword-accuracy-estimation" rel="nofollow">accuracy estimation</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/outliers" class="hw-term hw-article-keyword hw-article-keyword-outliers" rel="nofollow">outliers</a></li><li class="kwd"><a href="/keyword/predictive-accuracy" class="hw-term hw-article-keyword hw-article-keyword-predictive-accuracy" rel="nofollow">predictive accuracy</a></li><li class="kwd"><a href="/keyword/heritability" class="hw-term hw-article-keyword hw-article-keyword-heritability" rel="nofollow">heritability</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-3">Genomic selection, the prediction of genomic breeding values using molecular markers spanning the entire genome (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Piepho 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Whittaker <em>et al.</em> 2000</a>), is becoming a widespread practice in plant and animal breeding because of recent advances in high-throughput marker technologies and decreasing costs of genotyping (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo and Yu 2007</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Goddard and Hayes 2007</a>).</p><p id="p-4">Accordingly, it is important to assess the performance of procedures for estimating predictive accuracy in genomic prediction studies under a variety of realistic scenarios likely to be encountered in plant breeding research and practice. In particular, the performance of methods for estimating predictive accuracy in genomic prediction studies may be expected <em>a priori</em> to be adversely affected by outliers. As a result, outliers are potentially a highly relevant problem in accuracy estimation in genomic prediction (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Via <em>et al.</em> 2012</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Heslot <em>et al.</em> 2013</a>). It is therefore crucial to evaluate and rank contending methods for accuracy estimation in genomic prediction in terms of their estimated predictive accuracies and degree of robustness to outliers in phenotypic datasets. Accurately conducting such evaluations presupposes knowledge of the underlying true predictive accuracy for use in benchmarking the relative performance of the competing methods with and without outliers. However, for real datasets, the true underlying predictive accuracy is rarely, if ever, known.</p><p id="p-5">Outliers can potentially have devastating effects on estimated quantities of interest (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Atkinson 1985</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Belsley <em>et al.</em> 2005</a>), but their effects on accuracy estimation in genomic prediction studies in plant breeding thus far have not been evaluated. In plant breeding, outliers can arise from an outlying observation, environment, or year in a series of trials. Although many statistical procedures have long been developed for identifying outliers, successful identification and removal of true outliers from data remain a very challenging task to date, especially in high-dimensional data typically used in genomic prediction, and may even have undesirable consequences (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cook 1977</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Filzmoser 2005</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cerioli 2010</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cerioli and Farcomeni 2011</a>). Key complications in outlier detection and removal include masking or swamping of true outliers rendering them inconspicuous, difficulties associated with differentiating mild outliers from regular observations, and reduction in sample size and potentially adverse impact on the assumed distribution theory and, hence, estimated variance of removing outliers (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lourenço and Pires 2014</a>). Because data cleaning procedures are imperfect and do not always succeed in detecting and eliminating all true outliers, it is important to assess the performance and robustness of methods for accuracy estimation in the presence of outliers in genomic prediction studies.</p><p id="p-6">Here, we examine the influence of outliers on the performance and robustness of seven methods for estimating the accuracy of heritability and prediction accuracy in genomic prediction studies in plant breeding. To mimic outliers, we added to only one randomly chosen observation in each of the 1000 phenotypic datasets simulated for each scenario, in turn, each of the three multiples of the SDs of the two residual error variances used to simulate the small and the large datasets. The influence of the outliers on the performance of each of the seven methods was then evaluated by using each method to estimate predictive accuracy based on matched datasets with and without outliers. The deviation of the predictive accuracy estimated using the dataset without outliers from the simulated true predictive accuracy was used as the gold standard against which to evaluate the corresponding deviation in the predictive accuracy estimated using the same dataset with outliers.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-7">This article expands on the article by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Estaghvirou <em>et al.</em> (2013)</a> by contaminating the phenotypic datasets simulated for that article with outliers to assess their influence on the methods for accuracy estimation in genomic prediction. Otherwise, both articles use the exact same seven genomic prediction methods, simulation models, real maize, and SNP marker datasets to parameterize the simulation models and, hence, the same simulated phenotypic datasets. The new aspects of the methods concerning the outliers are thus highlighted here, whereas aspects of the methods that are similar between the two articles and that have already been described more fully in the earlier articles are only briefly summarized. This is particularly the case for the derivations of the equations used to compute heritability by method 4 and to estimate prediction accuracy by methods 5 and 7. Key details of the SAS code (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">SAS Institute 2014</a>) for implementing all seven methods are described comprehensively in the earlier article and therefore are not reproduced here. The statistics, parameters, and symbols used to denote them in the text are summarized in <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/011957SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS1.pdf">Table S1</a>.</p><p id="p-8">More precisely, we simulated 1000 datasets for each of 10 scenarios to evaluate the magnitude and direction of the influence of outliers on estimates of predictive accuracy for seven contending methods for accuracy estimation in genomic prediction. The 10 scenarios were defined by combinations of the number of genotypes (small = 177, large =698), marker effect variances [(small = (0.2019, 0.2019/10), large = (0.005892, 0.005892/10)], and multiples (5, 8, 10 for small datasets; 5, 10 for large datasets) of the SDs of the errors (small = 48.6728, large = 53.8715) used to simulate the phenotypic datasets. Although we showed the performances of methods 5 and 7 to be the best overall of the seven methods in a previous study (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Estaghvirou <em>et al.</em> 2013</a>), we nevertheless consider all seven methods again here because it is not possible to deduce their likely performances in the presence of outliers based on findings of that study.</p><div id="sec-2" class="subsection"><h3>Prediction accuracy</h3><p id="p-9">The correlation between the true <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-1.gif"/></noscript></span></span> and predicted (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-2.gif"/></noscript></span></span>) breeding values, called the simulated true accuracy or predictive accuracy, is defined as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-3.gif"/></noscript></span>, <span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-4.gif"/></noscript></span></span> is the covariance between the true and predicted breeding values, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-5.gif"/></noscript></span></span> is the sample variance of the true genetic breeding values, and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-6.gif"/></noscript></span></span> is the sample variance of the predicted breeding values. Our use of (1) is premised on the assumption that the correlation between the true and predicted breeding values (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-7.gif"/></noscript></span></span>) is the quantity of primary interest to breeders or geneticists.</p></div><div id="sec-3" class="subsection"><h3>Two-stage approach for predicting breeding values</h3><p id="p-10">We used a two-stage approach to predict the breeding values (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Schulz-Streeck <em>et al.</em> 2013</a>). We considered only one trial conducted in one location. In the first stage, we used the model for the observed plot data to estimate the adjusted means <em>p</em> for the testcross genotypes<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-8.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <em>p</em> is the vector of the adjusted means of the observed phenotypic values, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-9.gif"/></noscript></span></span> is the vector of the genotypic means from the second stage, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-10.gif"/></noscript></span></span> is the design matrix, and <em>f</em> is a matrix containing all the fixed and random (design and error) effects (replicates, blocks, etc.). A more elaborate specification of the two-stage-model, including a more precise description of the terms in model (2), is provided in <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Piepho <em>et al.</em> (2012a)</a>. In our case, the residual vector <em>f</em> for the simulated data comprises effects for complete replicates, incomplete blocks, and plot error.</p><p id="p-11">In the second stage, the true breeding values <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-11.gif"/></noscript></span></span> were predicted using the adjusted means for the testcross genotypes (but excluding any standard varieties) computed in (2) as the response and the model<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-12.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-13.gif"/></noscript></span></span> is the adjusted mean of the <em>i</em>-th genotype, <em>φ</em> is the general mean, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-14.gif"/></noscript></span></span> is the random effect of the <em>i</em>-th genotype, and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-15.gif"/></noscript></span></span> is the residual error assumed to be <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-16.gif"/></noscript></span></span>. The random vector <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-17.gif"/></noscript></span></span> is modeled through a linear regression on the random marker (SNP) effects <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-18.gif"/></noscript></span></span> by<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-19.gif"/></noscript></span>, <span class="disp-formula-label">(4)</span></span>where <em>Z</em> is the matrix of the marker covariates and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-20.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-21.gif"/></noscript></span></span> is the <em>p</em>-dimensional identity matrix, and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-22.gif"/></noscript></span></span> is the variance of marker effects. The marker information stored in the matrix <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-23.gif"/></noscript></span></span> consisted of 275 (AgReliant dataset) or 11646 (KWS Synbreed dataset) SNP markers derived from genotyping all the testcross genotypes. The marker covariate <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-24.gif"/></noscript></span></span> for the <em>i-th</em> genotype <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-25.gif"/></noscript></span></span> and the <em>k-th</em> marker (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-26.gif"/></noscript></span></span>) for biallelic SNP markers with alleles <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-27.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-28.gif"/></noscript></span></span> was recoded as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-29.gif"/></noscript></span></span>The variance of the genotypic effect <em>g</em> is thus modeled as<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-30.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-31.gif"/></noscript></span></span> is the transpose of <em>Z</em> (SAS code for simulating the matrix G is provided in Supplementary Materials SAS code S1). When the model is parameterized directly in terms of genotypic effects <em>g</em> rather than marker effects <em>u</em>, the method is often referred to as GBLUP rather than RR-BLUP. Here, we use the GBLUP parameterization. A modified version of model (3) assuming that genotypic effects are independent so that <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-32.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-33.gif"/></noscript></span></span> is the genetic variance and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-34.gif"/></noscript></span></span> is the <em>n</em>-dimensional identity matrix, was used to compute some measures of heritability.</p></div><div id="sec-4" class="subsection"><h3>Real maize datasets and estimation of variance components</h3><p id="p-12">AgReliant Genetics provided a maize dataset with 177 genotypes and 275 markers, whereas KWS-Synbreed Project availed a larger maize dataset with 698 genotypes and 11646 markers. Marker, block, and error variance components and marker information derived from the two real maize datasets were used to simulate true breeding values and phenotypic data assuming that the genotypes are correlated (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS2.pdf">Table S2</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS3.pdf">Table S3</a>). To obtain smaller estimates of heritability for scenarios 4 to 6, the marker effect variances for scenarios 1 to 3 were each divided by 10. Similarly, the marker effect variances for scenarios 7 and 8 were each divided by 10 to obtain smaller estimates of heritability for scenarios 9 and 10 (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS2.pdf">Table S2</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS3.pdf">Table S3</a>). The variance components were estimated for the two real maize datasets and for the 1000 datasets simulated for each of the 10 scenarios using two mixed models that assume that the genotyped lines are correlated according to the linear variance–covariance structure used in the RR-BLUP model.</p><p id="p-13">The AgReliant dataset contained 177 doubled haploid maize lines generated from biparental crosses. An unreplicated augmented trial design with incomplete blocks and the same common tester were used to assess the hybrid performance of testcross genotypes in terms of kernel dry weight per plot. The testcross genotypes were tested in six locations in 1 yr but not all genotypes were tested in all locations. Data from only one of the six locations were used with the RR-BLUP model to estimate the variance components used to simulate the random marker, block, and plot effects for scenarios 1 to 6 (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS2.pdf">Table S2</a>). The selected location contained only one unreplicated trial, five blocks, two checks, and 177 lines. All the testcross genotypes and checks were genotyped and were treated alike in the RR-BLUP model. Between three and five incomplete blocks each with one row of plots were used per location. The standard varieties used to connect the different blocks enabled estimation of the inter-block variance and partitioning of the variance into block and error components. Although used to facilitate the analysis of the testcross genotypes, the standard varieties did not contribute to the prediction of <em>g</em>. All markers with more than 20% missing values, more than 5% heterozygous genotypes, or minor allele frequencies less than 2.5% were excluded from the analysis.</p><p id="p-14">The KWS-Synbreed Project dataset was also taken from one of several locations contained in a larger dataset. This dataset contained 900 doubled haploid maize lines, 698 of which were genotyped but the other 202 were not. It also had six hybrid checks and five line checks. The genotypes were crossed with four testers and tested using nine trials, each of which was laid out as a 10×10 lattice square design with incomplete blocks and had two replicates. The sample size was 1800, and 38 observations lacked yield measurements. A more complete description of this dataset and how the RR-BLUP model was fitted to it has already been provided by <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Estaghvirou <em>et al.</em> (2013)</a>.</p><p id="p-15">Because this article expands on the work of <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Estaghvirou <em>et al.</em> (2013)</a>, we present below a synopsis of the key statistical models and methods and refer the reader to the earlier paper for further technical details on derivations of the models.</p></div><div id="sec-5" class="subsection"><h3>Simulation of datasets</h3><div id="sec-6" class="subsection"><h4>Assumed field design, model, and outliers:</h4><p id="p-16">We used variance components estimated from the real maize (<em>Zea mays</em>) dataset provided by AgReliant to simulate block and plot effects (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS2.pdf">Table S2</a>). We generated a dataset with 177 genotypes and used an α-design with two replicates to distribute these genotypes over 10 incomplete blocks per replicate, each of which had 18 plots. For this design, a suitable model must have one effect for the complete replicates and another effect for the incomplete blocks, nested within replicates. Accordingly, we simulated the field trial data using an α-design (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Petersen 1994</a>) and the model<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-35.gif"/></noscript></span>, <span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-36.gif"/></noscript></span></span> is the yield of the <em>i</em>-th genotype in the <em>j</em>-th block nested within the <em>k</em>-th complete replicate, µ is the general effect or mean, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-37.gif"/></noscript></span></span> is the fixed effect of the <em>k</em>-th complete replicate, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-38.gif"/></noscript></span></span> is the random effect of the <em>j</em>-th block nested within the <em>k</em>-th complete replicate, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-39.gif"/></noscript></span></span> is the random effect of the <em>i</em>-th genotype, and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-40.gif"/></noscript></span></span> is the residual plot error associated with <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-41.gif"/></noscript></span></span>. We likewise used variance components estimated from the real maize dataset with 698 genotypes provided by KWS to simulate the block and plot effects based on an α-design with two replicates and model (6) (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS3.pdf">Table S3</a>).</p></div></div><div id="sec-7" class="subsection"><h3>Simulation of outliers</h3><p id="p-17">Outliers were mimicked by adding 5-, 8-, or 10-times the SD of the residual error variance, computed from each of the two real maize datasets and used for the simulations, to a single randomly chosen observation in each simulated dataset. All the simulated datasets based on variance components estimated from the real maize datasets used the three multiples of the exact same SD: 6.977 for the small and 7.340 for the large datasets. The correlation between the predicted and the true breeding values, predictive accuracy, was estimated by each of four different indirect (methods 1 to 4) and three direct (methods 5 to 7) methods for each of the simulated datasets with and without outliers. A total of 10 simulation scenarios were defined by varying the configuration of the number of genotypes, size of marker effect variance, and multiples of the two SDs used in simulating the error variances (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The three outliers were carefully chosen to represent the three scenarios likely to be encountered in practice. The smallest outlier was chosen so that it would be difficult to detect using standard outlier diagnostic tools such as plots of Studentized residuals against the predicted mean and Q-Q plots (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS2.pdf">Figure S2</a>). By contrast, the largest outlier was easily detectable using these diagnostic tools (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS4.pdf">Figure S4</a>), whereas the difficulty with which the middling outlier would be detected was intermediate between the other two (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS5.pdf">Figure S5</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465399/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465399/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The 10 simulated scenarios defined by configurations of the number of genotypes, the number of markers, the size of the marker effect variance, and the size of outliers</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Estimating predictive accuracy</h3><p id="p-19">Predictive accuracy <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-42.gif"/></noscript></span></span> was estimated either indirectly (methods 1 to 4) or directly (methods 5 to 7). Indirect estimation of predictive accuracy involved dividing predictive ability, the correlation between the estimated breeding values, and the observed phenotypic values <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-43.gif"/></noscript></span></span>, by the square root of heritability <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-44.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-45.gif"/></noscript></span></span> following the work of <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Legarra <em>et al.</em> (2008)</a>. By contrast, direct estimation of predictive accuracy does not involve the use of heritability. Because methods 1 to 4 use cross-validation, predictive accuracy was calculated for each of 15 three-fold cross-validation replicates derived for each simulated dataset. A common estimate of predictive ability used with methods 1 to 4 is given by:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-46.gif"/></noscript></span>, <span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-47.gif"/></noscript></span></span>, is the covariance between the predicted breeding values and the observed phenotypic values, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-48.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-49.gif"/></noscript></span></span> are defined as in equation (1).</p><p id="p-20">Two important assumptions are implicit in equation (7). The first is that <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-50.gif"/></noscript></span></span> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dekkers 2007</a>), which is equivalent to assuming that the unobserved genotypic effect <em>g</em> is not correlated with the environmental components in <em>p</em>. Accordingly, on substituting <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-51.gif"/></noscript></span></span> in equation (1) with <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-52.gif"/></noscript></span></span> (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dekkers 2007</a>), we obtain<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-53.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span>where all the terms are defined as in equations (1) and (7). The second crucial assumption implicit in equation (7) is that <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-54.gif"/></noscript></span></span>, from which we deduce that<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-55.gif"/></noscript></span>. <span class="disp-formula-label">(9)</span></span>From equation (9), all that remains to compute <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-56.gif"/></noscript></span></span> is to specify how to compute heritability <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-57.gif"/></noscript></span></span>. To do this, we recall that the definition of <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-58.gif"/></noscript></span></span> used in <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-59.gif"/></noscript></span></span> requires a marker-based model for <em>g</em>. It is thus logical to use the same model to define heritability <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-60.gif"/></noscript></span></span>. However, working with the new definition is complicated by the fact that the true model is not known and must be approximated by suitable methods for genomic prediction. One way to overcome this complication is to estimate predictive accuracy using the same model used to define predictive accuracy, provided the model is close to the model for some genomic prediction method. Such a model should be chosen judiciously, however, because some methods for genomic prediction, for instance, in the machine learning field, lack explicit underlying models.</p><p id="p-21">A precise definition of heritability <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-61.gif"/></noscript></span></span> is thus impeded by the difficulty inherent in choosing a suitable model for heritability. Because of this intrinsic difficulty, any estimate of predictive accuracy should be regarded as furnishing only an approximation of precision. Because the model underlying ridge regression BLUP is the most widely used method for genomic prediction, possesses several attractive properties, uses a specific mixed model, and allows an estimate of <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-62.gif"/></noscript></span></span>to be obtained in several different ways (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Piepho and Möhring 2007</a>), we use it to define heritability <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-63.gif"/></noscript></span></span>.</p></div><div id="sec-9" class="subsection"><h3>Methods for estimating heritability and predictive accuracy</h3><p id="p-22">We used five approaches (methods 1 to 5) to estimate heritability for the datasets with and without outliers. The first method is commonly used by plant breeders (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>) to estimate heritability. The second and the third methods are modifications of the <em>ad hoc</em> measure using BLUE and BLUP (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Piepho and Möhring 2007</a>). The fourth method used to estimate heritability was proposed by <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Estaghvirou <em>et al.</em> (2013)</a>. This method uses the ratio of the expected value of the genetic variance to the expected value of the phenotypic variance. The fifth method was the second method proposed by <a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Estaghvirou <em>et al.</em> (2013)</a> to estimate heritability without cross-validation based on ideas similar to those used in computing the <em>ad hoc</em> measures of heritability <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-64.gif"/></noscript></span></span>. Methods 1 to 3 assume the genotypes are independent, whereas methods 4 and 5 assume that the genotypes are correlated according to the linear variance–covariance structure assumed to underlie the RR-BLUP model. The quantity <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-65.gif"/></noscript></span></span> was used to estimate predictive accuracy, where <em>H</em> is estimated using each of the first four approaches (methods 1 to 4) only and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-66.gif"/></noscript></span></span> is the Pearson product-moment correlation between the estimated breeding values and the simulated phenotypes. This is because, despite estimating heritability, method 5 computes predictive accuracy directly, as do methods 6 and 7. Methods 5 to 7 were thus used to estimate predictive accuracy for the datasets with and without outliers directly without first dividing predictive ability by the square root of heritability. The three direct approaches assume that the effects of genotypes are correlated according to the model underlying the RR-BLUP.</p><p id="p-23">We now present the essential details of each method in what follows.</p><div id="sec-10" class="subsection"><h4>Method 1:</h4><p id="p-24">This method first computes heritability using the relation (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Legarra <em>et al.</em> 2008</a>)<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-67.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span>where <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-68.gif"/></noscript></span></span> is the genetic variance, <em>r</em> is the number of replicates, and <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-69.gif"/></noscript></span></span> is the variance of plot error. Plant breeders routinely use this estimator with randomized complete block designs as an <em>ad hoc</em> approximation with incomplete block designs. However, the method is not appropriate for spatially correlated data (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gonçalves <em>et al.</em> 2013</a>).</p></div><div id="sec-11" class="subsection"><h4>Method 2:</h4><p id="p-25">This method uses an estimator based on the BLUEs and first computes heritability using the relation (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Piepho and Möhring 2007</a>)<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-70.gif"/></noscript></span> <span class="disp-formula-label">(11)</span></span>where <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-71.gif"/></noscript></span></span> is the mean variance of a difference of two adjusted genotypic means (BLUE) and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-72.gif"/></noscript></span></span> is the genetic variance estimated using equation (6), assuming uncorrelated genotypic effects.</p></div><div id="sec-12" class="subsection"><h4>Method 3:</h4><p id="p-26">This method computes an <em>ad hoc</em> measure of heritability based on BLUP and assumes that genotypic effects are not correlated (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cullis <em>et al.</em> 2006</a>)<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-73.gif"/></noscript></span> <span class="disp-formula-label">(12)</span></span>where <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-74.gif"/></noscript></span></span> is the mean variance of a difference of the BLUP of two genotypic effects <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-75.gif"/></noscript></span></span>. The BLUP of <em>i</em> was used as the phenotypic data in the mixed model for method 3.</p></div><div id="sec-13" class="subsection"><h4>Method 4:</h4><p id="p-27">This method defines heritability as the expected genetic sample variance <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-76.gif"/></noscript></span></span> divided by the expected phenotypic sample variance <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-77.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-78.gif"/></noscript></span>, <span class="disp-formula-label">(13)</span></span>where<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-79.gif"/></noscript></span> <span class="disp-formula-label">(14)</span></span>with <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-80.gif"/></noscript></span></span> and <em>R</em> being the variance–covariance matrices of genotypes and errors, respectively, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-81.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-82.gif"/></noscript></span></span> denoting an <em>n</em>-dimensional identity matrix and <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-83.gif"/></noscript></span></span> is a square matrix of ones. Heritability in equation (13) is estimated by plugging in estimates of <em>G</em> and <em>R</em>.</p></div><div id="sec-14" class="subsection"><h4>Method 5:</h4><p id="p-28">This method directly defines both the predictive accuracy and heritability as<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-84.gif"/></noscript></span>, <span class="disp-formula-label">(15)</span></span>where <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-85.gif"/></noscript></span></span> and <em>R</em> are the variance–covariance matrices of genotypes and errors. <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-86.gif"/></noscript></span></span> is the variance–covariance matrix of the phenotypes, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-87.gif"/></noscript></span></span> is defined as above, and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-88.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-89.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-90.gif"/></noscript></span></span> is the inverse of the variance–covariance matrix of the phenotypes and 1 is a vector of ones.</p></div><div id="sec-15" class="subsection"><h4>Method 6:</h4><p id="p-29">This method directly estimates predictive accuracy using the relation<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-91.gif"/></noscript></span>, <span class="disp-formula-label">(16)</span></span>where <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-92.gif"/></noscript></span></span> is the sample variance of the predicted breeding values <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-93.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-94.gif"/></noscript></span></span> is the sample covariance between the predicted breeding values <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-95.gif"/></noscript></span></span>, and the phenotypic values <em>p</em> and <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-96.gif"/></noscript></span></span> is estimated using equation 14.</p></div><div id="sec-16" class="subsection"><h4>Method 7:</h4><p id="p-30">This method is commonly used in animal breeding to directly compute predictive accuracy from the mixed model equations based on reliability <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-97.gif"/></noscript></span></span> as (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Mrode and Thompson 2005</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Henderson 1975</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Van Raden 2008</a>):<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-98.gif"/></noscript></span> <span class="disp-formula-label">(17)</span></span>where <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-99.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-100.gif"/></noscript></span></span> are the variances of the true and predicted breeding values and <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-101.gif"/></noscript></span></span> is the covariance between the true and predicted breeding values. The reliability of all the genotypes in each dataset is then estimated by:<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-102.gif"/></noscript></span> <span class="disp-formula-label">(18)</span></span>from which the estimated predictive accuracy is computed as the square root of reliability <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-103.gif"/></noscript></span></span>, where <em>n</em> is the total number of genotypes in the dataset. The practical details of how the statistics involving the true breeding values are actually estimated from the mixed model, despite the fact that the true breeding values are unknown, are omitted here because they are provided by <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Estaghvirou <em>et al.</em> (2013)</a>.</p></div></div><div id="sec-17" class="subsection"><h3>Cross-validation</h3><p id="p-31">The correlation between the predicted breeding values and the observed phenotypic values<span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-104.gif"/></noscript></span></span>, called predictive ability, was estimated for methods 1 to 4 and 6 using a threefold cross-validation (CV) and was used to estimate predictive accuracy. The choice of the threefold cross-validation procedure was dictated by the small number of genotypes (177) in the AgReliant dataset. The dataset with the adjusted means for the testcross genotypes was split into three random subsamples, one of which was held out in turn as a validation set. The other two subsamples were amalgamated into a single training set. The cross-validation procedure was replicated five times to yield 15 replicate datasets. A ridge regression model (6 and 7) was fitted to each of the 15 replicate validation and training sets. Predictive ability was then computed for all the genotypes. This procedure was repeated for each of the 1000 datasets simulated for each of the 10 scenarios. Predictive accuracy, the correlation between the predicted and the true breeding values <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-105.gif"/></noscript></span></span>, was estimated by the ratio of predictive ability <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-106.gif"/></noscript></span></span> to the square root of heritability for each of the indirect methods using datasets with and without outliers. Moreover, we directly estimated predictive accuracy using methods 5, 6, and 7 for the datasets with and without outliers as detailed below. The estimated predictive accuracy for each method for the datasets with outliers was compared with the estimates for the corresponding datasets without outliers. Furthermore, we compared all seven methods among themselves using the datasets with outliers.</p></div><div id="sec-18" class="subsection"><h3>Evaluation of the simulated data</h3><p id="p-32">The two-stage analysis and the preceding methods were used to estimate the predictive accuracy for the datasets with and without outliers. We also estimated heritability using each of the methods 1 to 5. The adjusted means <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-107.gif"/></noscript></span></span> of the genotypes computed from the simulated data at the first stage were used as the phenotypic data in the second stage of the cross-validatory analyses. Equation 11 was used to compute an <em>ad hoc</em> measure of heritability from the variance–covariance matrix of the adjusted means.</p></div><div id="sec-19" class="subsection"><h3>Comparing heritabilities</h3><p id="p-33">We estimated heritability for the simulated datasets with and without outliers for each of the 10 scenarios using each of the four different indirect methods (methods 1 to 4) and method 5. We then used the mean deviation (MD) and its SE to quantify the deviations in the estimated heritability computed for the <em>j</em>-th dataset with outliers <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-108.gif"/></noscript></span></span> from the corresponding simulated true heritability, regarded as the benchmark, <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-109.gif"/></noscript></span></span>:<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-110.gif"/></noscript></span> <span class="disp-formula-label">(19)</span></span>Similarly, we used MD and its SE to quantify the deviation of the estimated heritability for the <em>j</em>-th dataset without outliers <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-111.gif"/></noscript></span></span> from the corresponding simulated true heritability:<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-112.gif"/></noscript></span> <span class="disp-formula-label">(20)</span></span>Further, we used MD to compare the estimated heritability between pairs of all five methods.</p><p id="p-34">Finally, we computed descriptive statistics for the deviations in the estimated heritability for the datasets with outliers from the simulated true heritability as the benchmark and compared them with the corresponding deviation of the heritability estimates for the datasets without outliers from the simulated true heritability.</p></div><div id="sec-20" class="subsection"><h3>Comparing predictive accuracies</h3><p id="p-35">The simulated true predictive accuracy for the <em>j</em>-th dataset <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-113.gif"/></noscript></span></span> was computed as the correlation between the simulated true and the predicted breeding values for each of the 1000 datasets simulated for each of the 10 configurations of number of genotypes, marker effect variances, and outliers (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS4.pdf">Table S4</a>). Each of the seven methods was then used to estimate predictive accuracy as the correlation between the simulated true and the predicted breeding values for the <em>j</em>-th dataset simulated with <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-114.gif"/></noscript></span></span> and without <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-115.gif"/></noscript></span></span> outliers. The MD of the estimated predictive accuracy for the <em>j</em>-th dataset with outliers from the corresponding simulated predictive accuracy was then computed as:<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-116.gif"/></noscript></span> <span class="disp-formula-label">(21)</span></span>In the same vein, the MD for the estimated predictive accuracy for the <em>j</em>-th dataset without outliers from that for the corresponding simulated predictive accuracy was calculated as:<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/embed/mml-math-117.gif"/></noscript></span> <span class="disp-formula-label">(22)</span></span>Moreover, for each method and scenario combination, the deviation of the estimated predictive accuracy for the datasets with outliers from the simulated true predictive accuracy was compared with the deviation of the estimated predictive accuracy for the datasets without outliers from the simulated predictive accuracy as the benchmark. We also compared estimates of MD between pairs of the seven methods both within and between scenarios.</p><p id="p-36">Also, we calculated and compared descriptive summaries for the deviations of the estimated predictive accuracies for the datasets with outliers from the benchmark accuracies with the summaries for the deviations for the datasets without outliers from the benchmark accuracies. Finally, we used frequency histograms, box whisker plots, and bar charts to compare the deviation and mean deviation estimates between methods within and between scenarios (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS7.pdf">Figure S7</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bar plots for the mean deviation (MD) in the estimated heritability for the datasets with  (empty bars) and without  (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true heritability  for each of the five methods for estimating heritability in scenarios 1 to 6. Scenarios 1 to 3 are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019 and scenarios 4 to 6 are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019/10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-618729200" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bar plots for the mean deviation (MD) in the estimated heritability for the datasets with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-118&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (empty bars) and without &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-119&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true heritability &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-120&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for each of the five methods for estimating heritability in scenarios 1 to 6. Scenarios 1 to 3 are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019 and scenarios 4 to 6 are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019/10.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2317/F1.medium.gif" width="440" height="352"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/12/2317/F1.medium.gif" width="440" height="352"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465339" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-37" class="first-child">Bar plots for the mean deviation (MD) in the estimated heritability for the datasets with <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F1/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F1/embed/mml-math-118.gif"/></noscript></span></span> (empty bars) and without <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F1/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F1/embed/mml-math-119.gif"/></noscript></span></span> (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true heritability <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F1/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F1/embed/mml-math-120.gif"/></noscript></span></span> for each of the five methods for estimating heritability in scenarios 1 to 6. Scenarios 1 to 3 are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019 and scenarios 4 to 6 are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019/10.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bar plots for the mean deviation (MD) in the estimated heritability for the datasets with  (empty bars) and without  (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true heritability  for each of the five methods for estimating heritability in scenarios 7 to 10. Scenarios 7 and 8 are based on the same 1000 datasets simulated assuming 698 genotypes and a marker effect variance of 0.005892, whereas scenarios 9 and 10 are based on the same 1000 datasets simulated assuming 698 genotypes and a marker effect variance of 0.005892/10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-618729200" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bar plots for the mean deviation (MD) in the estimated heritability for the datasets with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-121&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (empty bars) and without &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-122&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true heritability &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-123&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for each of the five methods for estimating heritability in scenarios 7 to 10. Scenarios 7 and 8 are based on the same 1000 datasets simulated assuming 698 genotypes and a marker effect variance of 0.005892, whereas scenarios 9 and 10 are based on the same 1000 datasets simulated assuming 698 genotypes and a marker effect variance of 0.005892/10.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2317/F2.medium.gif" width="440" height="349"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/12/2317/F2.medium.gif" width="440" height="349"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465456" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-38" class="first-child">Bar plots for the mean deviation (MD) in the estimated heritability for the datasets with <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F2/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F2/embed/mml-math-121.gif"/></noscript></span></span> (empty bars) and without <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F2/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F2/embed/mml-math-122.gif"/></noscript></span></span> (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true heritability <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F2/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F2/embed/mml-math-123.gif"/></noscript></span></span> for each of the five methods for estimating heritability in scenarios 7 to 10. Scenarios 7 and 8 are based on the same 1000 datasets simulated assuming 698 genotypes and a marker effect variance of 0.005892, whereas scenarios 9 and 10 are based on the same 1000 datasets simulated assuming 698 genotypes and a marker effect variance of 0.005892/10.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">For each of the 10 scenarios we compared the difference in the deviations of the estimated predictive accuracies for the datasets with and without outliers from the simulated predictive accuracy between all pairs of the seven methods using Pearson correlations and <em>t</em>-tests (α=5%) adjusted for multiplicity using simulation adjustment. The <em>t</em>-tests were based on a mixed model with fixed effects for method and scenario and their interaction and a random effect for simulation replicates nested within scenarios (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Piepho <em>et al.</em> 2012b</a>).</p></div></div><div class="section results" id="sec-21"><h2 class="">Results</h2><div id="sec-22" class="subsection"><h3>Heritability</h3><p id="p-40">Only five (methods 1 to 5) of the seven methods estimate heritability. For the small datasets (<em>n</em> = 177), the accuracy of the estimated heritability decreased as the magnitude of the outliers increased or as the genetic variance decreased for all the five methods based on the deviation of the simulated true heritability from the heritability estimated using datasets with or without outliers (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Across all the scenarios based on the same datasets, outliers evidently reduced the accuracy with which heritability was estimated (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>, <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Moreover, reducing the genetic variance by a factor of 10 in the presence of outliers reduced the accuracy of the estimated heritability even more in scenarios 4 to 6 relative to scenarios 1 to 3, whereas increasing the size of the outliers markedly degraded accuracy in scenarios 9 to 10 relative to scenarios 7 and 8. For the large datasets, this pattern was less clear-cut. Of the seven methods, outliers had the least influence on method 5 in all the 10 scenarios. Consequently, method 5 was the most resistant to increasing the magnitude of outliers or additionally decreasing the genetic variance, followed by methods 2, 3, 4, and 1, in decreasing order (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>).</p></div><div id="sec-23" class="subsection"><h3>Predictive accuracy</h3><p id="p-41">Overall, outliers adversely affected the estimated predictive accuracy for all seven methods but the severity of this effect varied across methods and scenarios. Nevertheless, some methods (<em>e.g.</em>, 2 and 3 or 5 and 7) responded similarly to outliers across all 10 scenarios. Outliers had the least influence on the estimated predictive accuracies for methods 5 and 7 across all 10 scenarios (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, and <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS6.pdf">Figure S6</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS7.pdf">Figure S7</a>, and <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS8.pdf">Figure S8</a>). Methods 5 and 7 were therefore the most robust: to outliers and also gave the best estimates for predictive accuracy. For the five methods more adversely affected by outliers, the specific types of adverse effects of outliers on the estimated predictive accuracy were manifold and varied by both sample size (number of genotypes) and the genetic variance. More precisely, outliers degraded estimation accuracy the most for the small datasets with reduced genetic variance. Outliers were associated with a greater tendency to underestimate predictive accuracy for the small datasets in scenarios 1 to 6 (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, and <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS7.pdf">Figure S7</a>), to overestimate predictive accuracy for the large datasets in scenarios 7 to 10 (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>, <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS8.pdf">Figure S8</a>), and a greater likelihood of the mixed models to fail to converge. Overestimation was more serious for methods 1 to 4 than 5 and 7 in scenarios 7 to 10. The degree of underestimation or overestimation increased with increasing magnitude of the outliers.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box whisker plots of the deviations in the estimated predictive accuracy for the datasets with (empty box and whiskers capped with brackets) and without  (green boxes) outliers, regarded as the benchmark, from the simulated true predictive accuracy  for each of the seven methods in scenarios 1 to 3. All the scenarios are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019. a/b/c denotes the difference in the number of datasets with outliers for each scenario, out of a possible total of 1000, that showed undershooting, overshooting or did not converge relative to the numbers for the corresponding datasets without outliers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-618729200" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Box whisker plots of the deviations in the estimated predictive accuracy for the datasets with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-124&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;(empty box and whiskers capped with brackets) and without &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-125&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (green boxes) outliers, regarded as the benchmark, from the simulated true predictive accuracy &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-126&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for each of the seven methods in scenarios 1 to 3. All the scenarios are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019. a/b/c denotes the difference in the number of datasets with outliers for each scenario, out of a possible total of 1000, that showed undershooting, overshooting or did not converge relative to the numbers for the corresponding datasets without outliers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2317/F3.medium.gif" width="440" height="341"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/12/2317/F3.medium.gif" width="440" height="341"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465437" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-42" class="first-child">Box whisker plots of the deviations in the estimated predictive accuracy for the datasets with <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F3/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F3/embed/mml-math-124.gif"/></noscript></span></span>(empty box and whiskers capped with brackets) and without <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F3/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F3/embed/mml-math-125.gif"/></noscript></span></span> (green boxes) outliers, regarded as the benchmark, from the simulated true predictive accuracy <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F3/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F3/embed/mml-math-126.gif"/></noscript></span></span> for each of the seven methods in scenarios 1 to 3. All the scenarios are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019. a/b/c denotes the difference in the number of datasets with outliers for each scenario, out of a possible total of 1000, that showed undershooting, overshooting or did not converge relative to the numbers for the corresponding datasets without outliers.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box whisker plots of the deviations in the estimated predictive accuracy for the datasets with  (empty box and whiskers capped with brackets) and without  (green boxes) outliers, regarded as the benchmark, from the simulated true predictive accuracy  for each of the seven methods in Scenarios 4-6. All the scenarios are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019/10. Values presented as &#x201C;number/number/number&#x201D; denote the difference in the number of datasets with outliers for each scenario, out of a possible total of 1000, that showed undershooting, overshooting, or did not converge relative to the numbers for the corresponding datasets without outliers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-618729200" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Box whisker plots of the deviations in the estimated predictive accuracy for the datasets with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-127&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (empty box and whiskers capped with brackets) and without &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-128&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (green boxes) outliers, regarded as the benchmark, from the simulated true predictive accuracy &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-129&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for each of the seven methods in Scenarios 4-6. All the scenarios are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019/10. Values presented as &#x201C;number/number/number&#x201D; denote the difference in the number of datasets with outliers for each scenario, out of a possible total of 1000, that showed undershooting, overshooting, or did not converge relative to the numbers for the corresponding datasets without outliers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2317/F4.medium.gif" width="440" height="340"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/12/2317/F4.medium.gif" width="440" height="340"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465376" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-43" class="first-child">Box whisker plots of the deviations in the estimated predictive accuracy for the datasets with <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F4/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F4/embed/mml-math-127.gif"/></noscript></span></span> (empty box and whiskers capped with brackets) and without <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F4/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F4/embed/mml-math-128.gif"/></noscript></span></span> (green boxes) outliers, regarded as the benchmark, from the simulated true predictive accuracy <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F4/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F4/embed/mml-math-129.gif"/></noscript></span></span> for each of the seven methods in Scenarios 4-6. All the scenarios are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019/10. Values presented as “number/number/number” denote the difference in the number of datasets with outliers for each scenario, out of a possible total of 1000, that showed undershooting, overshooting, or did not converge relative to the numbers for the corresponding datasets without outliers.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box whisker plots of the deviations in the estimated predictive accuracy for the datasets with  (empty box and whiskers capped with brackets) and without  (green boxes) outliers, regarded as the benchmark, from the simulated true predictive accuracy  for each of the seven methods in scenarios 7 to 10. All the scenarios are based on the same 1000 datasets simulated assuming 698 genotypes and a marker variance of 0.005892 for scenarios 7 and 8 and 0.005892/10 for scenarios 9 and 10. Values presented as &#x201C;number/number/number&#x201D; denote the difference in the number of datasets with outliers for each scenario, out of a possible total of 1000, that showed undershooting, overshooting, or did not converge relative to the numbers for the corresponding datasets without outliers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-618729200" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Box whisker plots of the deviations in the estimated predictive accuracy for the datasets with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-130&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (empty box and whiskers capped with brackets) and without &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-131&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (green boxes) outliers, regarded as the benchmark, from the simulated true predictive accuracy &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-132&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for each of the seven methods in scenarios 7 to 10. All the scenarios are based on the same 1000 datasets simulated assuming 698 genotypes and a marker variance of 0.005892 for scenarios 7 and 8 and 0.005892/10 for scenarios 9 and 10. Values presented as &#x201C;number/number/number&#x201D; denote the difference in the number of datasets with outliers for each scenario, out of a possible total of 1000, that showed undershooting, overshooting, or did not converge relative to the numbers for the corresponding datasets without outliers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2317/F5.medium.gif" width="440" height="341"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/12/2317/F5.medium.gif" width="440" height="341"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465436" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child">Box whisker plots of the deviations in the estimated predictive accuracy for the datasets with <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F5/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F5/embed/mml-math-130.gif"/></noscript></span></span> (empty box and whiskers capped with brackets) and without <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F5/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F5/embed/mml-math-131.gif"/></noscript></span></span> (green boxes) outliers, regarded as the benchmark, from the simulated true predictive accuracy <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F5/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F5/embed/mml-math-132.gif"/></noscript></span></span> for each of the seven methods in scenarios 7 to 10. All the scenarios are based on the same 1000 datasets simulated assuming 698 genotypes and a marker variance of 0.005892 for scenarios 7 and 8 and 0.005892/10 for scenarios 9 and 10. Values presented as “number/number/number” denote the difference in the number of datasets with outliers for each scenario, out of a possible total of 1000, that showed undershooting, overshooting, or did not converge relative to the numbers for the corresponding datasets without outliers.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bar plots for the mean deviation (MD) of the estimated accuracy for the datasets with  (empty bars) and without  (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true predictive accuracy  for each of the seven methods in scenarios 1 to 3. All the scenarios are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-618729200" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bar plots for the mean deviation (MD) of the estimated accuracy for the datasets with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-133&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (empty bars) and without &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-134&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true predictive accuracy &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-135&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for each of the seven methods in scenarios 1 to 3. All the scenarios are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2317/F6.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/12/2317/F6.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465473" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-45" class="first-child">Bar plots for the mean deviation (MD) of the estimated accuracy for the datasets with <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F6/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F6/embed/mml-math-133.gif"/></noscript></span></span> (empty bars) and without <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F6/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F6/embed/mml-math-134.gif"/></noscript></span></span> (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true predictive accuracy <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F6/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F6/embed/mml-math-135.gif"/></noscript></span></span> for each of the seven methods in scenarios 1 to 3. All the scenarios are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bar plots for the mean deviation (MD) of the accuracy for the datasets with  (empty bars) and without  (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true predictive accuracy  for each of the seven methods in scenarios 4 to 6. All the scenarios are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019/10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-618729200" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bar plots for the mean deviation (MD) of the accuracy for the datasets with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-136&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (empty bars) and without &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-137&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true predictive accuracy &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-138&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for each of the seven methods in scenarios 4 to 6. All the scenarios are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019/10.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2317/F7.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/12/2317/F7.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465480" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-46" class="first-child">Bar plots for the mean deviation (MD) of the accuracy for the datasets with <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F7/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F7/embed/mml-math-136.gif"/></noscript></span></span> (empty bars) and without <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F7/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F7/embed/mml-math-137.gif"/></noscript></span></span> (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true predictive accuracy <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F7/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F7/embed/mml-math-138.gif"/></noscript></span></span> for each of the seven methods in scenarios 4 to 6. All the scenarios are based on the same 1000 datasets simulated assuming 177 genotypes and a marker effect variance of 0.2019/10.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bar plots for the mean deviation (MD) of the accuracy for the datasets with  (empty bars) and without  (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true predictive accuracy  for each of the seven methods in scenarios 7 to 10. All the scenarios are based on the same 1000 datasets simulated assuming 698 genotypes and a marker effect variance of 0.005892 for scenarios 7 and 8 and 0.005892/10 for scenarios 9 and 10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-618729200" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bar plots for the mean deviation (MD) of the accuracy for the datasets with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-139&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (empty bars) and without &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-140&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true predictive accuracy &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-141&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for each of the seven methods in scenarios 7 to 10. All the scenarios are based on the same 1000 datasets simulated assuming 698 genotypes and a marker effect variance of 0.005892 for scenarios 7 and 8 and 0.005892/10 for scenarios 9 and 10.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2317/F8.medium.gif" width="440" height="349"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/4/12/2317/F8.medium.gif" width="440" height="349"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2317/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465461" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-47" class="first-child">Bar plots for the mean deviation (MD) of the accuracy for the datasets with <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F8/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F8/embed/mml-math-139.gif"/></noscript></span></span> (empty bars) and without <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F8/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F8/embed/mml-math-140.gif"/></noscript></span></span> (filled and hatched bars) outliers, regarded as the benchmark, from the simulated true predictive accuracy <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F8/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2317/F8/embed/mml-math-141.gif"/></noscript></span></span> for each of the seven methods in scenarios 7 to 10. All the scenarios are based on the same 1000 datasets simulated assuming 698 genotypes and a marker effect variance of 0.005892 for scenarios 7 and 8 and 0.005892/10 for scenarios 9 and 10.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">We now elaborate on the specifics of the adverse effects of outliers on accuracy estimation by method and scenario. Even though all seven methods exhibited varying degrees of robustness to outliers, clear differences were often discernible in their performances. In general, the modification of the influence of outliers on accuracy estimation of the seven methods by the genetic variance became more severe when the genetic variance was reduced. For example, dividing the genetic variance by 10 but leaving all the other variance components intact was associated with a marked increase in the frequency of undershooting (estimates of predictive accuracy less than 0), a decrease in the frequency of overshooting (estimates of predictive accuracy more than 1), and increase in the frequency of datasets for which the mixed models failed to iterate to convergence (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>, and <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS7.pdf">Figure S7</a> and <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS8.pdf">Figure S8</a>). The degree to which genetic variance modified the influence of outliers varied, in turn, with sample size (number of genotypes), such that outliers had the greatest adverse effect on the estimated predictive accuracy for the small datasets (<em>n</em> = 177 genotypes) with reduced genetic variance.</p><p id="p-49">The deviations in the estimated predictive accuracy from the simulated true accuracy with and without (benchmark) outliers showed that outliers had the smallest, albeit similar, effects on methods 5 and 7 but were associated with overestimation of accuracy for all methods except 5 and 7 (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>, and <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS7.pdf">Figure S7</a>). Furthermore, outliers were also associated with underestimation of the variance of predictive accuracy across replicate datasets for all the methods in scenarios 1 to 6 (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>). Even so, outliers did not alter the relative performances of the seven methods established in the absence of outliers.</p><p id="p-50">Across all the scenarios, methods 5 and 7 produced the most accurate predictions in scenarios 1 to 6. But a comparison of scenarios 1 to 3 with 4 to 6, all of which are based on datasets with the same sample size but different genetic variances, makes it apparent that reducing the genetic variance markedly reduces predictive accuracy, even in the absence of outliers. However, reducing the genetic variance does not alter the relative performances of the seven methods or the general tendency of the estimated accuracy to decrease with increasing magnitude of outliers in scenarios 1 to 3 compared with scenarios 4 to 6. Instead, it greatly increases convergence problems, most critically for methods 1 to 3 (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>), and the variance of the benchmark predictive accuracy for all seven methods, particularly for methods 1 to 4 and 6 (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>, <a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>, and <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). Methods 5 and 7 still gave the most precise accuracy estimates even when the genetic variance was reduced.</p><p id="p-51">For all seven methods, outliers did not cause any noteworthy change in the frequency of undershooting in the estimated predictive accuracy for either the small (scenarios 1 to 3) or the large (scenarios 7 and 8) simulated datasets with large variances (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>). But when the genetic variances were reduced by a factor of 10, the cases of undershooting in the estimated predictive accuracy became clearly more pronounced for the small simulated datasets with outliers for all the methods except 5 and 7 in scenarios 4 to 6 (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5</a>). The extent of the increase in cases of undershooting for methods 1 to 4 and 6 in scenarios 4 to 6 also tended to increase with increasing size of the outliers (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5</a>). The same 10-fold reduction in genetic variance similarly increased convergence problems and the frequency of overshooting in the estimated predictive accuracy for the simulated large datasets with outliers for all the seven methods, especially for methods 1 to 4 in scenarios 9 and 10 (<a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5</a>).</p><p id="p-52">Outliers did not materially change the number of datasets for which the estimated predictive accuracy exceeded 1 (<em>i.e.</em>, overshooting) for methods 2, 3, 5, 6, and 7, but was associated with either increased (method 1) or decreased (method 4) frequency of overshooting in scenarios 1 to 3 (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). The magnitude of change in the frequency of overshooting for methods 1 and 4 increased with increasing size of the outliers. The frequency of overshooting decreased for the small datasets with outliers compared to the same size datasets without outliers for methods 1 to 4 and 6, but it did not change for methods 5 and 7 in scenarios 4 to 6 (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4</a>). Also, the degree of the decrease in the frequency of overshooting increased with increasing size of the outliers. By contrast, for the large datasets the frequency of overshooting increased only for method 4 in scenarios 7 and 8 and for methods 1 to 4 in scenarios 9 and 10 (<a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5</a>). The increase in the frequency of overshooting tended to increase with increasing size of the outliers (<a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-13" class="xref-fig" href="#F5">Figure 5</a>).</p><p id="p-53">For all seven methods except 5 and 7, both of which were the most resistant to outliers, outliers were also associated with a greater tendency for the mixed models to fail to iterate to convergence, zero, or very small estimated heritability for both the small and large datasets with small genetic variances. Thus, for scenarios 4 to 6 and 9 to 10 based on the datasets with reduced genetic variances, there was a clear increase in the number of datasets for which the mixed models failed to converge as the magnitude of the outliers increased (<a id="xref-fig-4-13" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-14" class="xref-fig" href="#F5">Figure 5</a>). This increase was more pronounced for the small than for the large datasets (scenarios 4 to 6) (<a id="xref-fig-4-14" class="xref-fig" href="#F4">Figure 4</a>).</p><p id="p-54">However, all the methods that use cross-validation (methods 1 to 4 in scenarios 7 and 8 in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> and methods 1 to 4 and 6 in scenarios 7 to 10 in <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8</a>) show a somewhat anomalous and counterintuitive pattern that deviates from the expected pattern and seems to suggest, quite intriguingly, that outliers enhance prediction. This unexpected behavior can be explained by two different biases of opposing sign. Thus, for example, if a method shows a large negative bias in the estimation of predictive accuracy, whereas the additional bias caused by outliers is positive in sign, the two biases may partly annihilate each other, leading to the somewhat unexpected behavior we observed. Nevertheless, we emphasize that this behavior was not found for the two best performing methods 5 and 7.</p><p id="p-55">The pairwise comparisons using <em>t</em>-tests reinforced the preceding patterns, including the observation that methods 5 and 7 gave similar and the most accurate estimates of predictive accuracy (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>). These tests demonstrated clear variation among methods in terms of their performances. The performances of methods 5 and 7 were similar and outstanding in several important respects. First, all their estimated predictive accuracies ranged between 0 and 1 in all 10 scenarios (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-15" class="xref-fig" href="#F4">Figure 4</a>, and <a id="xref-fig-5-15" class="xref-fig" href="#F5">Figure 5</a>). Second, the mixed models for methods 5 and 7 converged for all 1000 datasets simulated for each of the 10 scenarios (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-16" class="xref-fig" href="#F4">Figure 4</a>, and <a id="xref-fig-5-16" class="xref-fig" href="#F5">Figure 5</a>). In contrast to methods 5 and 7, the remaining methods were much less consistent in their sensitivity to outliers across scenarios. Methods 2 and 3 also responded similarly to outliers as indicated by the very high similarities of their estimated predictive accuracies across all the 10 scenarios (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>). The inconsistency in response to outliers was apparent in an increased likelihood of overestimating accuracy when outliers are present than when they are not in scenarios 1 to 6 (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-17" class="xref-fig" href="#F4">Figure 4</a>). It was also manifested in a higher frequency of failure of the mixed models to converge in scenarios 4 to 6 than in scenarios 1 to 3. We recall that both sets of scenarios are based on datasets with the exact same sample size and outliers. The only important difference between them is that the datasets in scenarios 4 to 6 have 10-times smaller genetic variances than those in scenarios 1 to 3 (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). The relatively close match between the estimated and the benchmark accuracy for all seven methods in scenarios 1 to 3 (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS6.pdf">Figure S6</a>) but striking overestimation of the benchmark accuracy by all methods but methods 5 and 7 in scenarios 4 to 6 (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>, <a id="xref-fig-4-18" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>, <a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/FigureS7.pdf">Figure S7</a>) can thus be attributed to the reduced genetic variance. Underestimation of the benchmark predictive accuracy was another consequence of outliers that was common for all seven methods but methods 5 and 7 in scenarios 7 to 10 (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>, <a id="xref-fig-5-17" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8</a>). Three further shortcomings were apparent for all the methods but methods 5 and 7 in scenarios 4 to 6 and 9 to 10. The first was that predictive accuracy could not be computed because the estimated heritability was zero, in particular for methods 1 to 3 (<em>n</em> = 193 datasets) in scenario 6. The other deficiency was that the estimated genetic variance was sometimes zero [<em>e.g.</em>, for methods 1 to 3 (<em>n</em> = 30) in scenario 6], thereby precluding the computation of predictive ability. Finally, the mixed models sometimes failed to converge (<em>n</em> = 219), so that the results required by methods 1 to 4 and 6 to compute predictive ability in scenario 6 were unavailable in these cases.</p><p id="p-56">The <em>t</em>-tests showed that the seven methods clustered into two fairly distinct groups, with each consisting of methods with comparable performances in terms of their responses to outliers in each of the 10 scenarios. The first cluster consisted of methods 1 to 4 and 6 and the other consisted of methods 5 and 7 (<a href="http://www.g3journal.org/content/suppl/2014/10/01/g3.114.011957.DC1/TableS5.pdf">Table S5</a>, <a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-19" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-18" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>, <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7</a>, and <a id="xref-fig-8-6" class="xref-fig" href="#F8">Figure 8</a>). The estimated predictive accuracies for each of the methods were often high and positive, but some methods (mainly 5 and 7) tended to produce high predictive accuracies when others (<em>e.g.</em>, methods 1 and 4) produce low accuracies for the same datasets and scenarios (results not shown).</p></div></div><div class="section discussion" id="sec-24"><h2 class="">Discussion</h2><p id="p-57">Outliers have long been known to present serious complications to regression problems (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cook and Weisberg 1982</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Belsley <em>et al.</em> 2005</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Schabenberger 2004</a>); therefore, they can be expected <em>a priori</em> to similarly do so to genomic prediction problems. However, to our knowledge, the influence of outliers on the performance of methods for estimating accuracy in genomic prediction studies has barely been investigated to date. <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Heslot <em>et al.</em> (2013)</a> examined the effect of entire outlying environments on genomic prediction, so their findings are difficult to compare with ours. As a result, there is a general dearth of comparable studies against which our findings can be reliably evaluated. Detection of outliers using standard diagnostic tools is well-known to be often far from trivial and less likely to be reliable because outliers can be present yet have inconspicuous residuals (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Zewotir and Galpin 2005</a>). Because the process of identifying and eliminating outliers prior to performing accuracy estimation in genomic prediction is only approximate, and because outliers can arise from multiple sources, including outlying individual observations, whole environments, or years, and therefore can be subtle and elusive to detect and eliminate, it is crucial to minimize potential adverse effects of any undetected outliers on the accuracy estimation in genomic prediction and on the reliability of selection decisions. To assess the impact of outliers on the performance of seven methods for estimating accuracy in genomic prediction based on RR-BLUP, we simulated 1000 replicate datasets according to an α-design for each of 10 scenarios distinguished by four configurations of genetic variances and sample sizes derived from two real maize datasets. The estimated predictive accuracy of each method on each dataset was evaluated against the simulated predictive accuracy and the accuracy estimated for the same dataset but contaminated, in turn, with one of three (scenarios 1 to 6) or two (scenarios 7 to 10) different outliers of increasing magnitudes.</p><div id="sec-25" class="subsection"><h3>Heritability</h3><p id="p-58">Outliers strongly affected the accuracy with which heritability was estimated by all five methods used to estimate heritability regardless of scenario. The adverse influence of outliers on the estimated predictive accuracy of heritability increased with increasing magnitude of outliers regardless of sample size or genetic variance. However, the effect of outliers was not uniform and varied in both its character and severity across methods such that method 5, followed by methods 2 and 3, were the most robust to the influence of outliers in all scenarios. The adverse influence of outliers on the accuracy of heritability was also greater for the small than for the large datasets and became larger when the genetic variance was reduced irrespective of sample size. These results have three important practical implications for the design and choice of both the sample size and method to ensure reliable estimation of heritability in plant breeding studies. Specifically, studies designed to capture most of the phenotypic variance and that use datasets with large sample sizes and method 5 to estimate heritability are likely to yield the most accurate estimates of heritability.</p></div><div id="sec-26" class="subsection"><h3>Predictive accuracy</h3><p id="p-59">Our comparative evaluation of the performance, stability, and robustness of the seven methods for estimating the accuracy of genomic prediction when phenotypic data are contaminated with outliers compared with when they are not (the benchmark) yielded many interesting and useful insights. Because the key assumptions, theoretical properties, computational efficiencies, and relative empirical performances of the seven methods under the same four configurations of genetic variances and sample sizes used to construct the 10 scenarios studied here were explored in depth by <a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Estaghvirou <em>et al.</em> (2013)</a>, our prime focus here is specifically on how outliers modify the performance of these methods across the 10 simulated scenarios.</p><p id="p-60"><a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Estaghvirou <em>et al.</em> (2013)</a> found estimates of predictive accuracy produced by method 5, which they proposed for estimating both heritability and predictive accuracy, and by method 7, which is widely used in animal breeding studies (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Mrode and Thompson 2005</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Henderson 1975</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Van Raden 2008</a>), to be the most accurate. In the present study, we found estimates of predictive accuracy for methods 5 and 7 to be the most stable and robust to the influence of outliers across all 10 scenarios we examined. Furthermore, methods 5 and 7 also gave broadly comparable estimates of predictive accuracy across all 10 scenarios. The other five methods were much more influenced by outliers across scenarios, portraying strong modification of their performances not only by the influences of outliers but also by genetic variance (and hence heritability) and sample size (number of genotypes).</p><p id="p-61">Outliers had several evident detrimental effects on the performances of methods 1 to 4 and 6. These effects included a greater propensity to underestimate predictive accuracy and its variance when the sample size is small, overestimate predictive accuracy when the sample size is large, or for the mixed models to fail to converge to optimal parameter values, thus precluding computation of predictive accuracy. Increasing the size of outliers by a factor of 8/5 and 10/5 exacerbated these effects. The combination of large outliers and reduced genetic variance clearly aggravated the adverse influence of outliers on the estimated predictive accuracy and was associated with a far greater increase in the prevalence of undershooting, reduced incidence of overshooting, failure of the mixed models to iterate to convergence, and zero or extremely small estimated heritabilities. The composite effects on the estimated predictive accuracy of simultaneously increasing the magnitude of outliers and reducing the genetic variance were substantial regardless of sample size but were more debilitating when the sample size was small than when it was large.</p><p id="p-62">For a given sample size, outliers substantially increased the time required by each of the seven methods to compute an estimate of predictive accuracy by prolonging the time needed to evaluate the log likelihood of the mixed models. This increase was especially marked for methods 1 to 4 and 6 that require cross-validation and therefore are more computationally expensive to implement than methods 5 and 7 that do not. Increasing the size of outliers (from 5 to 10) at a fixed sample size was associated with a further increase in computing time.</p><p id="p-63">We have focused exclusively on genotypes for which phenotypic data are available. An extension to unphenotyped genotypes is thus highly desirable for applications in breeding programs. Such an extension is readily available for method 7 using standard procedures (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Henderson 1984</a>). A similar extension is currently lacking for method 5, but we are working on developing one to be presented in a future paper.</p><p id="p-64">In conclusion, outliers had significant influences on the seven methods for estimating accuracy of genomic prediction, and these influences became more extreme when sample size and genetic variance were small. To ensure reliable estimation of accuracy in genomic prediction, it is therefore advisable to carefully check for and eliminate outliers whenever possible to design genomic selection studies to capture as much phenotypic variance as possible and yield large sample sizes. Because methods 5 and 7 are the simplest, computationally least expensive to implement, the most accurate and robust to outliers, small sample size, and genetic variance, using them to assess estimation accuracy in genomic prediction studies is decidedly preferable to using the five other competitor methods. Nevertheless, it is worth reiterating that we have considered only methods based on the RR-BLUP model, two sample sizes for genotypes, two sample sizes for markers, phenotypic data generated according to an α-design, and a single outlying observation in the phenotypic dataset. Further simulation studies that allow for wider variation in each of these five aspects would help establish the generality of our conclusions. We are currently exploring the consequences for accuracy estimation in genomic prediction of allowing for a greater range of variation in the number of genotypes and the number of markers and generating the phenotypic data according to different designs in follow-up studies.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-65">We thank the Associate Editor and an anonymous reviewer for suggestions and comments that led to improvements in the clarity of the presentation. We thank AgReliant and KWS for providing the datasets used in this paper. This research was funded by KWS SAAT AG, AgReliant Genetics and the German Federal Ministry of Education and Research (Bonn, Germany) within the AgroClusterEr “Synbreed-Synergistic plant and animal breeding” (Grant ID: 0315526). The authors declare that they have no competing interests. S.B.O.E. and J.O.O. participated in the design of the study, conducted the simulations, interpreted the results, and drafted and edited the manuscript. H.P.P. participated in conceiving the study and its design and contributed to editing the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-66">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011957/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011957/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-67"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-05-03"><span class="received-label">Received </span>May 3, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-09-25"><span class="accepted-label">Accepted </span>September 25, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Ould Estaghvirou <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-other" id="cit-4.12.2317.1"><div class="cit-metadata unstructured">Albrecht T., V. Wimmer, H.J. Auinger, M. Erbe, C. Knaak, M. Ouzunova, H. Simianer, C. C. Schön, 2011 Genomic-based prediction of testcross values in maize. Theor. Appl. Genet. 123: 339–350.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2317.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atkinson</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-source"><em>Plots</em>, <em>Transformations</em>, <em>and Regression: An Introduction to Graphical Methods of Diagnostic Regression Analysis</em></span>, <span class="cit-publ-name">Clarendon Press</span>, <span class="cit-publ-loc">Oxford, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2317.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belsley</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuh</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Regression Diagnostics: Identifying Influential Data and Sources of Collinearity</span>, <span class="cit-publ-name">John Wiley and Sons</span>, <span class="cit-publ-loc">New Jersey</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-other" id="cit-4.12.2317.4"><div class="cit-metadata unstructured">Bernardo R, J. Yu, 2007 Prospects for genomewide selection for quantitative traits in maize. Crop Sci. 47: 1082–1090.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.5" data-doi="10.1198/jasa.2009.tm09147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cerioli</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Multivariate outlier detection with high break-down estimators.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D105%26rft.spage%253D147%26rft_id%253Dinfo%253Adoi%252F10.1198%252Fjasa.2009.tm09147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/jasa.2009.tm09147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.6" data-doi="10.1016/j.csda.2010.05.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cerioli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farcomeni</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Error rates for multivariate outlier detection.</span> <abbr class="cit-jnl-abbrev">Comput. Stat. Data Anal.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">544</span>–<span class="cit-lpage">553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BStat.%2BData%2BAnal.%26rft.volume%253D55%26rft.spage%253D544%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.csda.2010.05.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.csda.2010.05.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.7" data-doi="10.2307/1268249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Detection of influential observations in linear regression.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D19%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1268249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1268249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1977CV78500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2317.8"><div class="cit-metadata unstructured">Cook, R. D., and S. Weisberg, 1982 <em>Residuals and Influence in Regression</em>. Chapman and Hall, New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.9" data-doi="10.1198/108571106X154443"><div class="cit-metadata"><cite>Cullis B. R., A. Smith, N. Coombes, <span class="cit-pub-date">2006</span> <span class="cit-article-title">On the design of early generation variety trials with correlated data.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D11%26rft.spage%253D381%26rft_id%253Dinfo%253Adoi%252F10.1198%252F108571106X154443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/108571106X154443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000242477500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-other" id="cit-4.12.2317.10"><div class="cit-metadata unstructured">Dekkers J. C. M., 2007 Prediction of response to marker-assisted and genomic selection using selection index theory. J. Anim. Breed. Genet. 124: 331–341.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.11" data-doi="10.1186/1471-2164-14-860"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Estaghvirou</span>  <span class="cit-name-given-names">S. B. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouzunova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evaluation of approaches for estimating the accuracy of genomic prediction in plant breeding.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D860%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-860%26rft_id%253Dinfo%253Apmid%252F24314298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-860&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24314298&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Filzmoser</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Identification of multivariate outliers: A performance study.</span> <abbr class="cit-jnl-abbrev">Aust. J. Stat.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BStat.%26rft.volume%253D34%26rft.spage%253D127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.13" data-doi="10.1111/j.1439-0388.2007.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D323%26rft.epage%253D330%26rft.atitle%253DGenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00702.x%26rft_id%253Dinfo%253Apmid%252F18076469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076469&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-other" id="cit-4.12.2317.14"><div class="cit-metadata unstructured">Gonçalves E, I. Carrasquinho, A. St Aubyn, A. Martins, 2013 Broad-sense heritability in mixed models for grapevine initial selection trials. Euphytica 189: 379–391.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Comparison of alternative sire evaluation methods.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">760</span>–<span class="cit-lpage">770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D41%26rft.issue%253D3%26rft.spage%253D760%26rft.epage%253D770%26rft.atitle%253DComparison%2Bof%2BAlternative%2BSire%2BEvaluation%2BMethods%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-other" id="cit-4.12.2317.16"><div class="cit-metadata unstructured">Henderson C.R., 1984 Applications of Linear Models in Animal Breeding, University of Guelph, Guelph, Ontario. ISBN 0-88955-030-1.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.17" data-doi="10.2135/cropsci2012.07.0420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Using genomic prediction to characterize environments and optimize prediction accuracy in applied breeding data.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D921%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.07.0420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.07.0420&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-other" id="cit-4.12.2317.18"><div class="cit-metadata unstructured">Legarra A., C. Robert-Granie, E. Manfredi, J. Elsen, 2008 Performance of genomic selection in mice. Genetics 180: 611–618.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.19" data-doi="10.1016/j.csda.2014.03.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lourenço</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">M-regression, false discovery rates and outlier detection with application to genetic association studies.</span> <abbr class="cit-jnl-abbrev">Comput. Stat. Data Anal.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BStat.%2BData%2BAnal.%26rft.volume%253D78%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.csda.2014.03.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.csda.2014.03.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2317.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mrode</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Linear Models for the Prediction of Animal Breeding Values</span>, <span class="cit-edition">Ed. 2</span>. CAB International, <span class="cit-publ-loc">Wallingford, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker map.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzIzMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2317.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-source">Agricultural Field Experiments/Design and Analysis</span>, <span class="cit-publ-name">Marcel Dekker</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2317.23" data-doi="10.1002/bimj.201100219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Möhring</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz\x{2010}Streeck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J.O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012a</span> <span class="cit-article-title">A stage\x{2010}wise approach for the analysis of multi\x{2010}environment trials</span>. <span class="cit-source">Biometrical Journal</span>, <span class="cit-vol">54</span>:<span class="cit-fpage">844</span>–<span class="cit-lpage">860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiom%2BJ%26rft.aulast%253DPiepho%26rft.auinit1%253DH.%2BP.%26rft.volume%253D54%26rft.issue%253D6%26rft.spage%253D844%26rft.epage%253D860%26rft.atitle%253DA%2Bstage-wise%2Bapproach%2Bfor%2Bthe%2Banalysis%2Bof%2Bmulti-environment%2Btrials.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbimj.201100219%26rft_id%253Dinfo%253Apmid%252F23007738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bimj.201100219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23007738&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.24" data-doi="10.2135/cropsci2011.11.0592"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estaghvirou</span>  <span class="cit-name-given-names">S. B. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordillo</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012b</span> <span class="cit-article-title">Efficient computation of ridge-regression best linear unbiased prediction in genomic selection in plant breeding.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1093</span>–<span class="cit-lpage">1104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1093%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.11.0592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.11.0592&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-other" id="cit-4.12.2317.25"><div class="cit-metadata unstructured">Piepho H. P., 2009 Ridge regression and extensions for genomewide selection in maize. Crop Sci. 49: 1165–1176.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-other" id="cit-4.12.2317.26"><div class="cit-metadata unstructured">Piepho H.P., J. Möhring, 2007 Computing heritability and selection response from unbalanced plant breeding trials. Genetics 177: 1881–1888.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2317.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">SAS Institute</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source"><em>SAS System for Windows</em>, <em>Version 9.4</em></span>, <span class="cit-publ-name">SAS Institute Inc.</span>, <span class="cit-publ-loc">Cary, NC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.12.2317.28"><div class="cit-metadata unstructured">Schabenberger, O., 2004 Mixed model influence diagnostics. Paper 189–29. Proceedings of the Twenty-Ninth Annual SAS Users Group International (SUGI) Conference. SAS Institute Inc., Cary, NC.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-4.12.2317.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparisons of single-stage and two-stage approaches to genomic selection</span>. Theoretical and Applied Genetics, <span class="cit-vol">126</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.30" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Raden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.31" data-doi="10.1111/mec.12021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conte</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason-Foley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Localizing FSToutliers on a QTL map reveals evidence for large genomic regions of reduced gene exchange during speciation-with-gene-flow.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">5546</span>–<span class="cit-lpage">5560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DVia%26rft.auinit1%253DS.%26rft.volume%253D21%26rft.issue%253D22%26rft.spage%253D5546%26rft.epage%253D5560%26rft.atitle%253DLocalizing%2BF%2528ST%2529%2B%2Boutliers%2Bon%2Ba%2BQTL%2Bmap%2Breveals%2Bevidence%2Bfor%2Blarge%2Bgenomic%2Bregions%2Bof%2Breduced%2Bgene%2Bexchange%2Bduring%2Bspeciation-with-gene-flow.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12021%26rft_id%253Dinfo%253Apmid%252F23057835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23057835&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310583800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.32" data-doi="10.1017/S0016672399004462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denham</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Marker-assisted selection using ridge regression.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWhittaker%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D75%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D252%26rft.atitle%253DMarker-assisted%2Bselection%2Busing%2Bridge%2Bregression.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004462%26rft_id%253Dinfo%253Apmid%252F10816982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10816982&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2317.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086569100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2317.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zewotir</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galpin</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Influence diagnostics for linear mixed models.</span> <abbr class="cit-jnl-abbrev">J. Data Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BData%2BSci.%26rft.volume%253D3%26rft.spage%253D153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/12/2317.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/12/2307" title="Population- and Sex-Biased Gene Expression in the Excretion Organs of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/12/2329" title="An RNAi-Based Suppressor Screen Identifies Interactors of the Myt1 Ortholog of Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/12"><span>Volume 4 Issue 12, December 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/12.cover-source.jpg?itok=ezho2p8P" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2317" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-W2wPYMGfvDSI6b6pI_-ncnKloguZHXGt97bw2HqmLzI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/12/2317" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/12/2317" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/12/2317" class="active" data-icon-position="" data-hide-link-title="0">Influence of Outliers on Accuracy Estimation in Genomic Prediction in Plant Breeding</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459749" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Zm-B9VwCaeDH7Itepg5L4eblsHp9p5Ra9CdmT_OzFls" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/12/2317.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/12/2317" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-GTxIkOIj_r5Cr79VXJHUv0y9hCUr1uYIGz11ro17yvo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/12/2317" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/12/2317.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459749" id="node459749--2" data-pisa="ggg;4/12/2317" data-pisa-master="ggg;g3.114.011957" data-apath="/ggg/4/12/2317.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Influence of Outliers on Accuracy Estimation in Genomic Prediction in Plant Breeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sidi Boubacar Ould Estaghvirou</span>, <span class="highwire-citation-author" data-delta="1">Joseph O. Ogutu</span> and <span class="highwire-citation-author" data-delta="2">Hans-Peter Piepho</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2317-2328;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011957 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459749--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sidi Boubacar Ould Estaghvirou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biostatistics Unit, Institute of Crop Science, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sidi%2BBoubacar%2BOuld%2BEstaghvirou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Estaghvirou%20SB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASidi%2BBoubacar%2BOuld%2BEstaghvirou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joseph O. Ogutu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biostatistics Unit, Institute of Crop Science, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BO.%2BOgutu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ogutu%20JO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BO.%2BOgutu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jogutu2007@gmail.com" class="" data-icon-position="" data-hide-link-title="0">jogutu2007@gmail.com</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hans-Peter Piepho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biostatistics Unit, Institute of Crop Science, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans-Peter%2BPiepho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Piepho%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans-Peter%2BPiepho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/12/2317" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/12/2317&amp;title=Influence%20of%20Outliers%20on%20Accuracy%20Estimation%20in%20Genomic%20Prediction%20in%20Plant%20Breeding" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/12/2317&amp;title=Influence%20of%20Outliers%20on%20Accuracy%20Estimation%20in%20Genomic%20Prediction%20in%20Plant%20Breeding" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/12/2317&amp;title=Influence%20of%20Outliers%20on%20Accuracy%20Estimation%20in%20Genomic%20Prediction%20in%20Plant%20Breeding" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/12/2317&amp;text=Influence%20of%20Outliers%20on%20Accuracy%20Estimation%20in%20Genomic%20Prediction%20in%20Plant%20Breeding" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/12/2317&amp;title=Influence%20of%20Outliers%20on%20Accuracy%20Estimation%20in%20Genomic%20Prediction%20in%20Plant%20Breeding" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/12/2317&amp;t=Influence%20of%20Outliers%20on%20Accuracy%20Estimation%20in%20Genomic%20Prediction%20in%20Plant%20Breeding" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/12/2317&amp;title=Influence%20of%20Outliers%20on%20Accuracy%20Estimation%20in%20Genomic%20Prediction%20in%20Plant%20Breeding" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/12/2317&amp;title=Influence%20of%20Outliers%20on%20Accuracy%20Estimation%20in%20Genomic%20Prediction%20in%20Plant%20Breeding" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459749/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459749" id="node459749--3" data-pisa="ggg;4/12/2317" data-pisa-master="ggg;g3.114.011957" data-apath="/ggg/4/12/2317.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Influence of Outliers on Accuracy Estimation in Genomic Prediction in Plant Breeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sidi Boubacar Ould Estaghvirou</span>, <span class="highwire-citation-author" data-delta="1">Joseph O. Ogutu</span> and <span class="highwire-citation-author" data-delta="2">Hans-Peter Piepho</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2317-2328;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011957 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459749--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sidi Boubacar Ould Estaghvirou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biostatistics Unit, Institute of Crop Science, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sidi%2BBoubacar%2BOuld%2BEstaghvirou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Estaghvirou%20SB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASidi%2BBoubacar%2BOuld%2BEstaghvirou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joseph O. Ogutu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biostatistics Unit, Institute of Crop Science, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BO.%2BOgutu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ogutu%20JO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BO.%2BOgutu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jogutu2007@gmail.com" class="" data-icon-position="" data-hide-link-title="0">jogutu2007@gmail.com</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hans-Peter Piepho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biostatistics Unit, Institute of Crop Science, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans-Peter%2BPiepho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Piepho%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans-Peter%2BPiepho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2317&amp;rft.epage=2328&amp;rft.atitle=Influence%20of%20Outliers%20on%20Accuracy%20Estimation%20in%20Genomic%20Prediction%20in%20Plant%20Breeding&amp;rft.volume=4&amp;rft.issue=12&amp;rft.date=2014-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Estaghvirou%2C+Sidi+Boubacar+Ould&amp;rft.au=Ogutu%2C+Joseph+O.&amp;rft.au=Piepho%2C+Hans-Peter"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459749/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459749/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459749/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459749/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459749/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459749/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459749/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459749/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459749/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459749/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459749/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459749/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/12/2317" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/12/2317.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/12/2317.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/12/2317.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/12/2317.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
