<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/12/2505" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/12/2505.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/12/2505.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/12/2505.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/12/2505/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/12/2505.atom" />
<meta name="HW.pisa" content="ggg;4/12/2505" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Spatial Localization of Recent Ancestors for Admixed Individuals" />
<meta name="DC.Identifier" content="10.1534/g3.114.014274" />
<meta name="DC.Date" content="2014-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Yang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Ancestry analysis from genetic data plays a critical role in studies of human disease and evolution. Recent work has introduced explicit models for the geographic distribution of genetic variation and has shown that such explicit models yield superior accuracy in ancestry inference over nonmodel-based methods. Here we extend such work to introduce a method that models admixture between ancestors from multiple sources across a geographic continuum. We devise efficient algorithms based on hidden Markov models to localize on a map the recent ancestors ( e.g. , grandparents) of admixed individuals, joint with assigning ancestry at each locus in the genome. We validate our methods by using empirical data from individuals with mixed European ancestry from the Population Reference Sample study and show that our approach is able to localize their recent ancestors within an average of 470 km of the reported locations of their grandparents. Furthermore, simulations from real Population Reference Sample genotype data show that our method attains high accuracy in localizing recent ancestors of admixed individuals in Europe (an average of 550 km from their true location for localization of two ancestries in Europe, four generations ago). We explore the limits of ancestry localization under our approach and find that performance decreases as the number of distinct ancestries and generations since admixture increases. Finally, we build a map of expected localization accuracy across admixed individuals according to the location of origin within Europe of their ancestors." />
<meta name="DC.Contributor" content="Wen-Yun Yang" />
<meta name="DC.Contributor" content="Alexander Platt" />
<meta name="DC.Contributor" content="Charleston Wen-Kai Chiang" />
<meta name="DC.Contributor" content="Eleazar Eskin" />
<meta name="DC.Contributor" content="John Novembre" />
<meta name="DC.Contributor" content="Bogdan Pasaniuc" />
<meta name="article:published_time" content="2014-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Spatial Localization of Recent Ancestors for Admixed Individuals" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Ancestry analysis from genetic data plays a critical role in studies of human disease and evolution. Recent work has introduced explicit models for the geographic distribution of genetic variation and has shown that such explicit models yield superior accuracy in ancestry inference over nonmodel-based methods. Here we extend such work to introduce a method that models admixture between ancestors from multiple sources across a geographic continuum. We devise efficient algorithms based on hidden Markov models to localize on a map the recent ancestors (&lt;i&gt;e.g.&lt;/i&gt;, grandparents) of admixed individuals, joint with assigning ancestry at each locus in the genome. We validate our methods by using empirical data from individuals with mixed European ancestry from the Population Reference Sample study and show that our approach is able to localize their recent ancestors within an average of 470 km of the reported locations of their grandparents. Furthermore, simulations from real Population Reference Sample genotype data show that our method attains high accuracy in localizing recent ancestors of admixed individuals in Europe (an average of 550 km from their true location for localization of two ancestries in Europe, four generations ago). We explore the limits of ancestry localization under our approach and find that performance decreases as the number of distinct ancestries and generations since admixture increases. Finally, we build a map of expected localization accuracy across admixed individuals according to the location of origin within Europe of their ancestors.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/12/01" />
<meta name="citation_mjid" content="ggg;4/12/2505" />
<meta name="citation_id" content="4/12/2505" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/12/2505" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/12/2505.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/12/2505.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/12/2505.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.014274" />
<meta name="citation_pmid" content="25371484" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2505" />
<meta name="citation_lastpage" content="2518" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Wen-Yun Yang" />
<meta name="citation_author_institution" content="Department of Computer Science, University of California, Los Angeles, California 90095" />
<meta name="citation_author" content="Alexander Platt" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095" />
<meta name="citation_author" content="Charleston Wen-Kai Chiang" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095" />
<meta name="citation_author" content="Eleazar Eskin" />
<meta name="citation_author_institution" content="Department of Computer Science, University of California, Los Angeles, California 90095" />
<meta name="citation_author_institution" content="Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095" />
<meta name="citation_author_institution" content="Department of Human Genetics, University of California, Los Angeles, California 90095" />
<meta name="citation_author" content="John Novembre" />
<meta name="citation_author_institution" content="Department of Human Genetics, University of Chicago, Chicago, Illinois 60637" />
<meta name="citation_author" content="Bogdan Pasaniuc" />
<meta name="citation_author_institution" content="Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095" />
<meta name="citation_author_institution" content="Department of Human Genetics, University of California, Los Angeles, California 90095" />
<meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Geffen School of Medicine at University of California, Los Angeles, California 90095" />
<meta name="citation_author_email" content="bpasaniuc@mednet.ucla.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Baran;citation_author=B. Pasaniuc;citation_author=S. Sankararaman;citation_author=D. Torgerson;citation_author=C. Gignoux;citation_title=Fast and accurate inference of local ancestry in Latino populations.;citation_volume=28;citation_year=2012;citation_pmid=22495753;citation_doi=10.1093/bioinformatics/bts144" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=Y. Baran;citation_author=I. Quintela;citation_author=A. Carracedo;citation_author=B. Pasaniuc;citation_author=E. Halperin;citation_title=Enhanced localization of genetic samples through linkage-disequilibrium correction.;citation_pages=882-894;citation_volume=92;citation_year=2013;citation_pmid=23726367;citation_doi=10.1016/j.ajhg.2013.04.023" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=A. Basu;citation_author=H. Tang;citation_author=X. Zhu;citation_author=C. C. Gu;citation_author=C. Hanis;citation_title=Genome-wide distribution of ancestry in Mexican Americans.;citation_pages=207-214;citation_volume=124;citation_year=2008;citation_issue=3;citation_pmid=18752003;citation_doi=10.1007/s00439-008-0541-5" />
<meta name="citation_reference" content="citation_journal_title=Psychometrika;citation_author=H. Bozdogan;citation_title=Model selection and Akaike’s information criterion (AIC): The general theory and its analytical extensions.;citation_pages=345-370;citation_volume=52;citation_year=1987;citation_doi=10.1007/BF02294361" />
<meta name="citation_reference" content="citation_journal_title=Hum. Biol.;citation_author=A. Brisbin;citation_author=K. Bryc;citation_author=J. Byrnes;citation_author=F. Zakharia;citation_author=L. Omberg;citation_title=PCAdmix: principal components-based assignment of ancestry along each chromosome in individuals with admixed ancestry from two or more populations.;citation_pages=343-364;citation_volume=84;citation_year=2012;citation_pmid=23249312;citation_doi=10.3378/027.084.0401" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Bryc;citation_author=C. Velez;citation_author=T. Karafet;citation_author=A. Moreno-Estrada;citation_author=A. Reynolds;citation_title=Genome-wide patterns of population structure and admixture among Hispanic/Latino populations.;citation_volume=107;citation_year=2010;citation_pmid=20445096;citation_doi=10.1073/pnas.0914618107" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=A. P. Dempster;citation_author=N. M. Laird;citation_author=D. B. Rubin;citation_title=Maximum likelihood from incomplete data via the EM algorithm.;citation_pages=1-38;citation_volume=39;citation_year=1977" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=P. Drineas;citation_author=J. Lewis;citation_author=P. Paschou;citation_title=Inferring geographic coordinates of origin for Europeans using small panels of ancestry informative markers.;citation_pages=e11892-e11892;citation_volume=5;citation_year=2010;citation_issue=8;citation_pmid=20805874;citation_doi=10.1371/journal.pone.0011892" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=E. Elhaik;citation_author=T. Tatarinova;citation_author=D. Chebotarev;citation_author=I. S. Piras;citation_author=C. M. Calò;citation_title=Geographic population structure analysis of worldwide human populations infers their biogeographical origins.;citation_pages=3513;citation_volume=5;citation_year=2014;citation_pmid=24781250" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of Population Structure Using Multilocus Genotype Data: Linked Loci and Correlated Allele Frequencies;citation_pages=1567-1587;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930761" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Gravel;citation_title=Population genetics models of local ancestry.;citation_volume=191;citation_year=2012;citation_pmid=22491189;citation_doi=10.1534/genetics.112.139808" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Guillot;citation_author=R. Leblois;citation_author=A. Coulon;citation_author=A. C. Frantz;citation_title=Statistical methods in spatial genetics.;citation_pages=4734-4756;citation_volume=18;citation_year=2009;citation_issue=23;citation_pmid=19878454;citation_doi=10.1111/j.1365-294X.2009.04410.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Hinch;citation_author=A. Tandon;citation_author=N. Patterson;citation_author=Y. Song;citation_author=N. Rohland;citation_title=The landscape of recombination in African Americans.;citation_pages=170-175;citation_volume=476;citation_year=2011;citation_issue=7359;citation_pmid=21775986;citation_doi=10.1038/nature10336" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Jarvis;citation_author=L. Scheinfeldt;citation_author=S. Soi;citation_author=C. Lambert;citation_author=L. Omberg;citation_title=Patterns of ancestry, signatures of natural selection, and genetic association with stature in Western African pygmies.;citation_pages=e1002641-e1002641;citation_volume=8;citation_year=2012;citation_issue=4;citation_pmid=22570615;citation_doi=10.1371/journal.pgen.1002641" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. Johnson;citation_author=M. Coram;citation_author=M. Shriver;citation_author=I. Romieu;citation_author=G. Barsh;citation_title=Ancestral components of admixed genomes in a Mexican cohort.;citation_pages=e1002410-e1002410;citation_volume=7;citation_year=2011;citation_issue=12;citation_pmid=22194699;citation_doi=10.1371/journal.pgen.1002410" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. K. Maples;citation_author=S. Gravel;citation_author=E. E. Kenny;citation_author=C. D. Bustamante;citation_title=RFMix: a discriminative modeling approach for rapid and robust local-ancestry inference.;citation_pages=278-288;citation_volume=93;citation_year=2013;citation_pmid=23910464;citation_doi=10.1016/j.ajhg.2013.06.020" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. McVean;citation_title=A genealogical interpretation of principal components analysis.;citation_pages=e1000686-e1000686;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19834557;citation_doi=10.1371/journal.pgen.1000686" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Moreno-Estrada;citation_author=S. Gravel;citation_author=F. Zakharia;citation_author=J. L. McCauley;citation_author=J. K. Byrnes;citation_title=Reconstructing the population genetic history of the Caribbean.;citation_pages=e1003925;citation_volume=9;citation_year=2013;citation_pmid=24244192;citation_doi=10.1371/journal.pgen.1003925" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. N’Diaye;citation_author=G. K. Chen;citation_author=C. D. Palmer;citation_author=B. Ge;citation_author=B. Tayo;citation_title=Identification, replication, and fine-mapping of Loci associated with adult height in individuals of african ancestry.;citation_pages=e1002298-e1002298;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=21998595;citation_doi=10.1371/journal.pgen.1002298" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. R. Nelson;citation_author=K. Bryc;citation_author=K. S. King;citation_author=A. Indap;citation_author=A. R. Boyko;citation_title=The Population Reference Sample, POPRES: a resource for population, disease, and pharmacological genetics research.;citation_pages=347-358;citation_volume=83;citation_year=2008;citation_issue=3;citation_pmid=18760391;citation_doi=10.1016/j.ajhg.2008.08.005" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Novembre;citation_author=T. Johnson;citation_author=K. Bryc;citation_author=Z. Kutalik;citation_author=A. Boyko;citation_title=Genes mirror geography within Europe.;citation_pages=98-101;citation_volume=456;citation_year=2008;citation_issue=7218;citation_pmid=18758442;citation_doi=10.1038/nature07331" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Paşaniuc;citation_author=S. Sankararaman;citation_author=G. Kimmel;citation_author=E. Halperin;citation_title=Inference of locus-specific ancestry in closely related populations.;citation_volume=25;citation_year=2009;citation_pmid=19477991;citation_doi=10.1093/bioinformatics/btp197" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Pasaniuc;citation_author=S. Sankararaman;citation_author=D. G. Torgerson;citation_author=C. Gignoux;citation_author=N. Zaitlen;citation_title=Analysis of Latino populations from GALA and MEC studies reveals genomic loci with biased local ancestry estimation.;citation_volume=29;citation_year=2013;citation_pmid=23572411;citation_doi=10.1093/bioinformatics/btt166" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=P. Paschou;citation_author=E. Ziv;citation_author=E. G. Burchard;citation_author=S. Choudhry;citation_author=W. Rodriguez-Cintron;citation_title=PCA-correlated SNPs for structure identification in worldwide human populations.;citation_pages=e160;citation_volume=3;citation_year=2007;citation_doi=10.1371/journal.pgen.0030160" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. Paschou;citation_author=P. Drineas;citation_author=J. Lewis;citation_author=C. M. Nievergelt;citation_author=D. A. Nickerson;citation_title=Tracing sub-structure in the European American population with PCA-informative markers.;citation_pages=e1000114-e1000114;citation_volume=4;citation_year=2008;citation_issue=7;citation_pmid=18797516;citation_doi=10.1371/journal.pgen.1000114" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. Patterson;citation_author=N. Hattangadi;citation_author=B. Lane;citation_author=K. E. Lohmueller;citation_author=D. A. Hafler;citation_title=Methods for high-density admixture mapping of disease genes.;citation_pages=979-1000;citation_volume=74;citation_year=2004;citation_issue=5;citation_pmid=15088269;citation_doi=10.1086/420871" />
<meta name="citation_reference" content="citation_journal_title=Lancet;citation_author=M. A. Perera;citation_author=L. H. Cavallari;citation_author=N. A. Limdi;citation_author=E. R. Gamazon;citation_author=A. Konkashbaev;citation_title=Genetic variants associated with warfarin dose in African-American individuals: a genome-wide association study.;citation_pages=790-796;citation_volume=382;citation_year=2013;citation_pmid=23755828;citation_doi=10.1016/S0140-6736(13)60681-9" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Price;citation_author=N. Patterson;citation_author=R. Plenge;citation_author=M. Weinblatt;citation_author=N. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Price;citation_author=A. Tandon;citation_author=N. Patterson;citation_author=K. Barnes;citation_author=N. Rafaels;citation_title=Sensitive detection of chromosomal segments of distinct ancestry in admixed populations.;citation_pages=e1000519-e1000519;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543370;citation_doi=10.1371/journal.pgen.1000519" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Price;citation_author=N. Zaitlen;citation_author=D. Reich;citation_author=N. Patterson;citation_title=New approaches to population stratification in genome-wide association studies.;citation_pages=459-463;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20548291;citation_doi=10.1038/nrg2813" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. A. Rosenberg;citation_author=L. M. Li;citation_author=R. Ward;citation_author=J. K. Pritchard;citation_title=Informativeness of genetic markers for inference of ancestry.;citation_pages=1402-1422;citation_volume=73;citation_year=2003;citation_issue=6;citation_pmid=14631557;citation_doi=10.1086/380416" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=N. A. Rosenberg;citation_author=L. Huang;citation_author=E. M. Jewett;citation_author=Z. A. Szpiech;citation_author=I. Jankovic;citation_title=Genome-wide association studies in diverse populations.;citation_pages=356-366;citation_volume=11;citation_year=2010;citation_issue=5;citation_pmid=20395969;citation_doi=10.1038/nrg2760" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. F. Seldin;citation_author=R. Shigeta;citation_author=P. Villoslada;citation_author=C. Selmi;citation_author=J. Tuomilehto;citation_title=European population substructure: clustering of northern and southern populations.;citation_pages=e143-e143;citation_volume=2;citation_year=2006;citation_issue=9;citation_pmid=17044734;citation_doi=10.1371/journal.pgen.0020143" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. Seldin;citation_author=B. Pasaniuc;citation_author=A. Price;citation_title=New approaches to disease mapping in admixed populations.;citation_pages=523-528;citation_volume=12;citation_year=2011;citation_issue=8;citation_pmid=21709689;citation_doi=10.1038/nrg3002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Shringarpure;citation_author=E. P. Xing;citation_title=mStruct: inference of population structure in light of both genetic admixing and allele mutations.;citation_volume=182;citation_year=2009;citation_pmid=19363128;citation_doi=10.1534/genetics.108.100222" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. Storfer;citation_author=M. A. Murphy;citation_author=S. F. Spear;citation_author=R. Holderegger;citation_author=L. P. Waits;citation_title=Landscape genetics: where are we now?;citation_pages=3496-3514;citation_volume=19;citation_year=2010;citation_issue=17;citation_pmid=20723061;citation_doi=10.1111/j.1365-294X.2010.04691.x" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Inf. Theory;citation_author=A. Viterbi;citation_title=Error bounds for convolutional codes and an asymptotically optimum decoding algorithm.;citation_pages=260-269;citation_volume=13;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. K. Wasser;citation_author=A. M. Shedlock;citation_author=K. Comstock;citation_author=E. A. Ostrander;citation_author=B. Mutayoba;citation_title=Assigning African elephant DNA to geographic region of origin: applications to the ivory trade.;citation_volume=101;citation_year=2004;citation_pmid=15459317;citation_doi=10.1073/pnas.0403170101" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. K. Wasser;citation_author=C. Mailand;citation_author=R. Booth;citation_author=B. Mutayoba;citation_author=E. Kisamo;citation_title=Using DNA to track the origin of the largest ivory seizure since the 1989 trade ban.;citation_volume=104;citation_year=2007;citation_pmid=17360505;citation_doi=10.1073/pnas.0609714104" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Wegmann;citation_author=D. Kessner;citation_author=K. Veeramah;citation_author=R. Mathias;citation_author=D. Nicolae;citation_title=Recombination rates in admixed individuals identified by ancestry-based inference.;citation_pages=847-853;citation_volume=43;citation_year=2011;citation_issue=9;citation_pmid=21775992;citation_doi=10.1038/ng.894" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Yang;citation_author=J. Novembre;citation_author=E. Eskin;citation_author=E. Halperin;citation_title=A model-based approach for analysis of spatial structure in genetic data.;citation_pages=725-731;citation_volume=44;citation_year=2012;citation_issue=6;citation_pmid=22610118;citation_doi=10.1038/ng.2285" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=F. Zakharia;citation_author=A. Basu;citation_author=D. Absher;citation_author=T. L. Assimes;citation_author=A. S. Go;citation_title=Characterizing the admixed African ancestry of African Americans.;citation_pages=R141-R141;citation_volume=10;citation_year=2009;citation_issue=12;citation_pmid=20025784;citation_doi=10.1186/gb-2009-10-12-r141" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Spatial Localization of Recent Ancestors for Admixed Individuals" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
<meta name="twitter:description" content="Ancestry analysis from genetic data plays a critical role in studies of human disease and evolution. Recent work has introduced explicit models for the geographic distribution of genetic variation and has shown that such explicit models yield superior accuracy in ancestry inference over nonmodel-based methods. Here we extend such work to introduce a method that models admixture between ancestors from multiple sources across a geographic continuum. We devise efficient algorithms based on hidden Markov models to localize on a map the recent ancestors ( e.g. , grandparents) of admixed individuals, joint with assigning ancestry at each locus in the genome. We validate our methods by using empirical data from individuals with mixed European ancestry from the Population Reference Sample study and show that our approach is able to localize their recent ancestors within an average of 470 km of the reported locations of their grandparents. Furthermore, simulations from real Population Reference Sample genotype data show that our method attains high accuracy in localizing recent ancestors of admixed individuals in Europe (an average of 550 km from their true location for localization of two ancestries in Europe, four generations ago). We explore the limits of ancestry localization under our approach and find that performance decreases as the number of distinct ancestries and generations since admixture increases. Finally, we build a map of expected localization accuracy across admixed individuals according to the location of origin within Europe of their ancestors." />
<meta name="og-title" property="og:title" content="Spatial Localization of Recent Ancestors for Admixed Individuals" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/12/2505" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Ancestry analysis from genetic data plays a critical role in studies of human disease and evolution. Recent work has introduced explicit models for the geographic distribution of genetic variation and has shown that such explicit models yield superior accuracy in ancestry inference over nonmodel-based methods. Here we extend such work to introduce a method that models admixture between ancestors from multiple sources across a geographic continuum. We devise efficient algorithms based on hidden Markov models to localize on a map the recent ancestors ( e.g. , grandparents) of admixed individuals, joint with assigning ancestry at each locus in the genome. We validate our methods by using empirical data from individuals with mixed European ancestry from the Population Reference Sample study and show that our approach is able to localize their recent ancestors within an average of 470 km of the reported locations of their grandparents. Furthermore, simulations from real Population Reference Sample genotype data show that our method attains high accuracy in localizing recent ancestors of admixed individuals in Europe (an average of 550 km from their true location for localization of two ancestries in Europe, four generations ago). We explore the limits of ancestry localization under our approach and find that performance decreases as the number of distinct ancestries and generations since admixture increases. Finally, we build a map of expected localization accuracy across admixed individuals according to the location of origin within Europe of their ancestors." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12.cover-source.jpg" />
    <title>Spatial Localization of Recent Ancestors for Admixed Individuals | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"9oVf_qbkl0xl_r9atGYuXraQ4XVDbyx0-tUkE7sIWOU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459747","apath":"\/ggg\/4\/12\/2505.atom","pisa":"ggg;4\/12\/2505","ac":{"\/ggg\/4\/12\/2505.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/12\/2505.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/12\/2505"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/12\/2505"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/12\/2505":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022H12oSjjos9iU9TK4vUSfMM2kC+NFrd3cCl7Lfxg25QWg9U\\\\\\\/k0T\\\\\\\/ofiCFTCAHnrFsO8yJDsdnJAZo7hcZ\\\\\\\/zb4A732NABXZJtAPxwO2B\\\\\\\/s47ysZd3PS+i0O\\\\\\\/\\\\\\\/Za8Sgrqpj2UBh3G+9c\\\\\\\/PEaT+S9pjTDjRe20ZZkwimrKHHYAZ8Dh9yi2QTcvh45MolRfUPImevXZuDOy8RemoUbTcsdHaucPtlnxlKVqIfasPr+awNBnpgGZZmuK3nsM8r+rcvUFNntCE7cl+6AQTKAPvwvWpxnbCTc8XZC2GofBbYG+hLbR8h9Fx9T6ITjQrJZ2xzNtDw+VNTIIT+xXAzeBf\\\\\\\/TVeydEKi9Tslb\\\\\\\/ogWYe+RjuAshf3t7lVaHn4T7f6Fjl6SYfDoshDTG+zhQxS7U4LLB336YhTapwF5nC4OdMJS11JgZrLT2U26s+BB+dC27v+52oUN2vJ12GWvPGyOh6eAPOq9Jt+ZA9BPgnUuwK117ry\\\\\\\/G8vrOkzKVWlBYUrB8yyY9Htn2Z3fuwBPKSOYobsRxgCl+QPG5utuBykRdjsqVrdKigR4H37P5uKJr130KWI3URi3b8MOQpi9BDoPJM1vGmBWv+AIgEmye0bca8Dxk2OWwaxPQIM8\\\\\\\/5wTPrURhLAcnFZj+u0Olv2if8C32gn3MncskQX2jarK7nft+G6K0lyRLBVWfmLOV4BibZ87pTyxR3JnVL8dt834raUNh7uAp9PFQBCwW0XzjqvzsBfSr8OzElfddeeGwM1AlJAkQOHVrBcEQxN+qD12hBut54vDBKVlgUlQ52p3gGGwLmti7OtaEq6UsJuFi22E\\\\\\\/U\\\\\\\/hAZJFai4DJ9ytWZmyGAFfJNh65yLwf\\\\\\\/S2iYv6kN1P6JIhHcWTqcYQql9d1fh8CtCtBL\\\\\\\/JSi8IDQ2IVFDO8zLgGRzesdFvbxWz6IZ9TPIFy9LbckMZUjYQvRin6DqGG\\\\\\\/X16SWDhO9ubCX2wm+j0GJGQw7x11bKDiYde4jJM1\\\\\\\/355PyuMrB9b+rSnEKa204YssRE4z\\\\\\\/nC+jCvE37o\\\\\\\/8QPter+Mvc018GIKgk7i1lb2GXIlYj\\\\\\\/GyAofIZwdSAkIG1i6APCl4QTZj07DYrJhNgQVY2mEThWhSvyC8SHV8fNlACyCnlYQgFMnHXP\\\\\\\/hagb7ybYuqaU\\\\\\\/kAWRZl8jvPTDFoyqtA2shJGPVguLKuOWDvUBmopKcm8hRetlb4u5gepP+FxdYE7MyLDCpdMPHQuGJBEwMhpjvXQmiUnutjFHsNXkJAlMgr82T9AhvHdpf9xxSY0EBgZT7i3\\\u0022,\\\u0022iv\\\u0022:\\\u0022j+8JK4\\\\\\\/fbml30aBFaq1MrA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bd92451cac4e90ae648c5fcb05799bcb216b122b883def1333335ddd3761f34a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022y408GbZ7ECSkHr33MAUnzvFhgLi5WAT1GsTttzYklLXiHt7FjQg2aYzStPhAflQLgc9ITdOLx+HJ1fSp3RFtlgPLNU07amMDw5\\\\\\\/omgpKYlb7HHtnDwy0Y7OfNnAaa5s\\\\\\\/d89R8YPgDnVnzHhTdoqz\\\\\\\/QYMxGesELMmkwmjjGW84A9H6qA8ZvfnrgLeZSGAHE1AxZfoJ6shZIf0PIa3dPyfBZYjSDY3YWKBufIddR5vyb+A4k+IKii+A2NREk\\\\\\\/GaL7t3X8suPltCMw5UTh1adDEeNnblnLP4E0U+NN09Ni9vO+yG6lbYgz\\\\\\\/k3CTqgXMjj+hNtHbi\\\\\\\/rEeuR1EEBvGlu12j7bhEpLvcJUS\\\\\\\/rUSdh0ND1hQzUGEThnu8oky+VEo3oplIkPjnSCFr+1SzSzepvt6PEt5baLs+wLi7Uag+3IujLsZFfI1bY5igjbV5F8+efTWvsKVo\\\\\\\/vNrmZaJ6BRtbu0umycDrftrbWAUlepjH3rFMclkpUJtN\\\\\\\/p7H658RCrQBxzjWzGnlZiFrSyGq2QHet8NCbFoHcMWngFbrxEkF9H\\\\\\\/aokC0a3vcXSrW+wveiQ1gIbWUv56S9++LNSVWp8+pOfE9CUT760JFQjXdX2z53SN5J2HV3TWcl6BwIHRpjvqNgtOfzML\\\\\\\/bImdbwGIsWTBhCh5Yh0GSFaHwZ+AEDU6l\\\\\\\/l5Kf8mxQ\\\\\\\/7B2OuicQ0ZaZrSTd2NEpKvim9a46UrapxyeXZnf3cxn4VdiD3EuLGjEvdXh7YKzdgezJMc8\\\\\\\/8hU6RqRJ0FPeE3D5zRRUt97bAr6RvL2i+IaF51AMzMUcsdkQT1iLSZF4h2Hqm0uqvQ\\\\\\\/E2mBX+dYmJLaQNbRojfsWs0eznxlQQmnygoiA1\\\\\\\/+fq4k0dDsoTP\\\\\\\/2CQlZVz5Eqk4HJWqwdfYhOuUV7Mk1teMQj5mHmtqGfetAAFvS0Tk\\\\\\\/ZcF6mEDoEoXTyz0KbAQMO7luWb2qRHupcu1A\\\\\\\/UGHQ+MvMVXPEmyMy9cmqC6KSrML3PaKM+ZlfU+npuEiETz6EO1bg8TAU4GrjWEZlcoGdpukza7Py+pJyNAZ4no6vCoypLGQ7WHvaR+qGu0e16CKYsbvCsxTV7D5qJ7MxLk2WcaEr\\\\\\\/Bla1AJmC1MLSj9XinSXxTC1fInnR0nVDWyCHpVBdmNcqTU6PlBFO8xMJf9iJkHgrhX0hBFOfmTY0qDkRiahyJipjPJnmXQ1+GvlNL9YF997JDqIOfjOevo+w9RXKRMxt5mwcNJn\\\\\\\/J3yoZ8tD689I+obdcWdwct8UpzPupK9YZ8BJtvC2zmEQZZVhhEoG\\\\\\\/yDRY65BhA2XUlgxirrSk7Cr+s8yeW8IrPC0q9RhvQS8ah7BL+tmtAyoeK4QgdNffbyROAyz45hpufV+m2dTvAHCnpElQgGs5rUWO2HTJFrGMn3WqF1AfZmbzUOHxqVpB6Cw09M2oxok6B9jfuyFJEfDCcAzKm0H\\\\\\\/0Jww3QryCi8M6G2KUQiaOEd1eBV5llFi1EfX7dPBeDMwNrhWppdLGOATrCQQ5TM\\\\\\\/3n4ScT3k0namYkrewikEQQUjDn21B44im+IBWMjRcZ8LbootqAOUJzoCCqNmDWHY3TuI0noYNxYBt71s\\\\\\\/ZCTPv7xYlGf\\\\\\\/Hfs9mnF6Vyo\\\\\\\/5ijTFFXpRCx8jEQMFGazypdJ78wWZy3H+jaM42JagrAPDldwJOGrhZWdpQbezSQpkRuvoSCW94mE3EM9e4qf2n0S0cYQWIKqhgaz4XtCxA8F0vrHMZpMPd2xdoECIY8K3E\\\\\\\/PyHFAC\\\\\\\/IzCgALw0dRVEvWXMo0NZ9pMlxGpTHskc\\\\\\\/qb0WeTpUtzY4MU0WLGvmL7uWv4GIJ5z+qY0CSA1rh9TCyJ8EFZLymeVAa0hZIoNhek1HYy5QNGWnKxIS92ARQG5RA4WVE\\\\\\\/a16M4wObgkHr\\\\\\\/zO601e9O5tHiZOuGYt8cEmiKpXpMDLyT2fYgPH5Rwy9l4nMOYwtSEjxI2jwh0UEbYnRTSazqGbDoMKMXNthdBtVw9SprT8gVNfCpcCxXLDA3erBnnB1cpyzi653BEE292IMUcqaPTlWbgRbLak0H2cNXGp6TAIcXrj35Ga4FOuHWZJjkKa\\\\\\\/ueQR2yKt0mb67iAHJMYs5qeaLFMMHekyX+ZW6ijT6Yaf+1Ib8JPEVuk55G9KeTSc8fV7QsDtraqrvy6dDOuqAszIWzA9SnwHwwWO18MDgVdJGcMsuIXCXah+eJPTY\\\\\\\/GyzTaaUcICPgmJEHFlwwPyemLiWxrHG\\\\\\\/KV7K4qOf7aXxSJFxlC2mUcE\\\\\\\/7nfTq7c00b68ESew9tM\\\\\\\/yQt\\\\\\\/2fj31HzRxyxdRjK4bEA5dZzrYcNvYcC1Ayeal3foibjOuTTkwwsLPgmwWMNXVb6+r9m1QHdTb6P\\\\\\\/5CwLhm7HiCuGQhyibU8K2nggqg62NyWGd5HV0ZQQgUQ5VtAp\\\\\\\/SzdBIj2yiOP7pWEd0kj7UPlKKEUFdi9xBBBaRw2W9Dp2t5zDXbPdxKQ1Q54W5tvQQ+GE1TUkmtznzFbKbxRIxNq\\\\\\\/krCRpCo\\\\\\\/cRfpDl\\\\\\\/AO8nN1heUgoFroCyySJSNqm6u5lCt8bx+0xjNtNnIBGjcOjxAbZ+Bm88MfCdtvu7Y4dMCW7ctc1KSGSoHPluCjgVCyTlN6YUJMI8uCq2vlArLJBDKstE4R353hGSccKLUUxqrGZ0F\\\\\\\/c0Cs\\\\\\\/AYUOsAQyb70gEtNxoH+5mFleyeLBZiTeOi8McR6EB2Ubw11vRDmYX9UwlLH5ZAXhwe5UwXGOxnxMxmaiGSXI6n3PBTqDjCynMp9mfAJjRp7BiKBcZ46aJiNYqD\\\\\\\/cf1LMBTz8yr2b9BKSisnHpdVz1kAtjY7p4OX4ZnYIXBHBENGJ4zJlgj\\\\\\\/JpmklBzsw4V8r5QedJr81WCMtx97JsNyWQneSudWeTrrThVjYvBQKjAJZsciE5Blg6TEbQjFVhSUUMJOQbetDyWFOG+IiVDQRMf9ifg7\\\\\\\/3EYg9nwU3xexuHtRs9Jvv4599rdgVFGyjVSUOGnvNBz68RUpbOyPuuOTQlZI+uJobKPOHgLvIav0sjsXFrKrZqJ8uamcO\\\\\\\/n3pjKx7wP\\\\\\\/2GO+G4ZJvqAvzdJDUmfSRCsovGk1Vi4gju+ysxG7LtYLPvv+8iXtEM9dUnVwDFMGhrq1UNbEhVMjPPVHj1N0rDsA3nm3C1CtS40q49aYwYGWTCNkZvROYMU1DJETLAgRTD6IJZm8BOWALe3+hkKU2woeiFJj2rLeAhT7j7kznCYt5rI3YAI\\\\\\\/vX0Z8GUTTKzq6vVJ00SqOWpRX5dxZAuVVR\\\\\\\/opK7yem8+Tdow6gpWPMl1717YPmiT7kXNtee2U4S8t\\\\\\\/y3eRHTeF+gFieEMJUzGUYH66cLYqcai3ozIeOlDPX3Cb9xl\\\\\\\/O1\\\\\\\/sjZd26bnQ1mGRUPaSIa6sHoKXBIbDYJITDnZU+gdU1Nrb\\\\\\\/kRBA21duQCQzbYoo4\\\\\\\/nd4PgWPahy8V999fFuwnLj+EJeNU5jnRv\\\\\\\/gmFL0QoxV1NBXMDBxDuP8En35tTc+43zhODEnz0aqLnXb9TztyGdYrgIMntXdb73FZSUxr0uR0k3qvXgXg+zNIHMWTv9W\\\\\\\/sd2NFfGWBxdsT4joxiHXzgwhONJ7rpkNqN1PlX7IWhK8LZlUwfOGgTQ+nweCnr1tYSahOJPKE3hjfdfgaD0LjTpSZtdrnCDlxHSQlSuS918b2k4Gp8ZUSIYh15EiuefYEoM3fNnnVBG3L4Yihtgfkm\\\\\\\/5H+doM3xNbU+HD\\\\\\\/YHSvHNuUegNk4g69W0UA3yXYCFvPZTzV+oVX3HxAQWgad8EnUl1WnQPorq1mnZpInJq7bNEaA88QjWU1NHTvEYkCxXjT+x3KTW3IXe7Xn3j3\\\\\\\/cvXEsGDigppez31UXPYf\\\\\\\/61qQFpaWElWQMVczbn6hT7qtQ5xHk68TNOmqU+MOZKRmNEvZHe6nEDT0+8mrO7kcBvhtzz6sQyXzrlmaRZI1SImngUzNhNYMk75GcJ6zsHxnpMkZ2CWlQPZ60HC0CjEmHBKiSPGmUzsrQ9WNDAkafMqoTcvSj6OP7XGr3zfAUV823XsfPqJyImDghwy2bw1AnrZsf12CUcMjiQlPLiLB9eeT95G2rLlT3eZYdyj6o7PqBnQFYSgIm0pZNYK4rSlifv9NMM\\\\\\\/XYjtlp4T30fTHkRXuic6u866Bt+5BzGTAvQpqU4oc5oq+qkBbwpjLRc43f\\\\\\\/3XqUF3ZYn0E\\\\\\\/0uMF\\\\\\\/SoA1TEOR\\\\\\\/5\\\\\\\/wiMJf0aly9Loo6cupBuDMEpb87BHj1YUItTkUe6QnxG4iwwJ62Il+NXMBIbBpIKnlYEPVVLAee0\\\\\\\/jnD33C6eU5FZb+oZ3dybB5C59bYpOUiduuwvAh01RCuVon5oGI5wiNMtQaL8r69k2MFlHtPu5CzeBqwLgh8JTD1NTS3uDufrc3bvSzz6Bym3s8bco5\\\\\\\/8GpzZdf0qCkilv3ltBwPwb53hwWEB3teJvPS2cTtzhRcnWhzZlLKcoy2AUO4NNbFX3NmDGmBjs1b7s\\\\\\\/g\\\\\\\/+AMnHs9PyuEET3Dbo9qcm9wGjOCGLJwqQROuKLao3bxgObY0eP3ia0dheBcUqtXxaHXEBILVICU0mh2DdwPgTnC45CJP7OINS\\\\\\\/KMQXoUclZWQcoU13q+DFcxFDBTd+ANbS3ckvDqDtnMf9MxbLZifW+RFey+1Il\\\\\\\/9+Yvk9TJRTYfpUNfrOubx3+Z9\\\\\\\/0ypxQv2bmP8oMU+iG+JRExP8F\\\\\\\/y8bQorrORyuFLoxae3ICXKGYE8hDU1aml7q3OEjroVYmbmrxfdcTFWsYSLW3ddULzP16ZzcRdqHHiVwH91lzwUu2fFLAABB86KuaAvXu2gDvRjYnnXtUrmRSg7WmiUUDBms6xWKiR3\\\\\\\/NxgePADa+pVgdheIYIjfCJpwF2ZaO4dg5wjHtjA3nM5l9rmVoLcutrJDG2ehkTlkNZ67dvlQHxCKwBx1vi04JR9u5T9dwFmbYl3jz5rtr+elpAMUsQKhOXw89qKDa0I68HBOBNOCwQTT6YkU+I1Mlnp2fTV2hkwUahlJlawFZr1dg62nGX1kZw5Cd6WPx4v3RLojN3mhIJEtKzGLkD71vtnRSaqcjHUV4HHxCW6KTKKeOFAvCj+cpTbsHYZRIRl62PufXmFK+BFNOGOMuCsvclQFEaju1l7mWDPBOU9DgN957mamIKFefEh4AANR3p9Pc+xOMvD0m74tDZQrOTowyihZSqHGpmi0drhjTfTc\\\\\\\/2Z1obs4Wgw7H01zLUIhIHN+EOwfBaCYmZMMjZNNmjPsWqu5N0nGm7HdhF7fu64hq21459yucmPA92tGwqj72L2rl7SdPVhfcPy\\\\\\\/mmD5GcugdfrBllrqDt6tZE7olk9gkfYWGvrxz8sLqz6mkYe84cbl3D4tNRGbz3oZIMeENBorzZnfkCfZdpV40UOcR\\\\\\\/lUO0C8rmmzm+X4IKGs5Va4sAqtdvtyYAKnbhYLzod9J13hwz+7CcCOFAP0qiRczneVi4kQ+c8+\\\\\\\/pDLqYW+6il3MG9QepoFkQ0v1TKdtkgdJ8nqxCl14vhaKNvFC0jZHFIN\\\\\\\/5nnuMSnEhy3xto2ilkid6Ox3YhTFotClnvFWvqfabszV9x\\\\\\\/RgxFfYWTcdpZtEgGzNMhit8XhYvgiAAEgQQgY69UsJjhkWoaeWKJVu2ou8h7Pb9RQlOs3lYJePlF6aOtfcyJJhmwbOtspCm1FBbtgcCVUu1pJ7GWbIJEOPPt2xbr+BPcHqTc24Go0D9d\\\\\\\/YBojDfuoviD\\\\\\\/Z28cAOuxX3+VtFjtyePZDQiS43BaIIf0sFdlylSZPd9Yar3RfN+vBTk04ducbc1NZUrekPckHsww4Y3EGmU8f6\\\\\\\/tvRCGJf6GlXQz3RI52L+1GUi4Y8rS4vtFXNs5MhsGlj\\\\\\\/R9jvpI4vSAKJmhyX9G38qYLFw47fGA47suAYfkz6i3sD\\\\\\\/N5nZr6yKpOhWa80\\\\\\\/Kt8EmHfa29gAGml4+EEkIt+WCR5aFtXFBJxOwFZqkTpjMjaOGPZL4TCDI\\\\\\\/MiPfFHqUGsPYkx4nFya+Mcdjn7qPnQkpCLdohaOSYgiZGVXBJeRB5VWm2QYSWRIJjXiyuUR2PWenvavZ1FLAwoullNO+giuSGgMe6Ao7Yr36dWF8wuHdqs4N2bjyynpC1u+ak5DCgbZSxEzE8yVhfWlCp9BWQ\\\\\\\/Jrw6PyfIGW6g0xdTByppzP4xGQBk71E4OFGrN+Favf9JE7LMT2p1PaZt2U7O4wdPrZGdrx28TFRkuAMImEbK1\\\\\\\/E5a\\\\\\\/xmX5nM5Bw1M\\\\\\\/ZFP7mrXMg7s4++Nl9A4k+7\\\\\\\/+jK35sHeyyHfLzyXzzfbeX+Nb6Z0ja5bJVuI0Y4w8x1xDy9IGaqPMH2yap49BYMYKguM80LVVeE\\\\\\\/Tth6AERpmd36eAMjaMHuUqcAKHsgNpky4qMcbWYLmilI+7GnMeMUSy7FHp3XggUlKIGL+b0UlDt60IVcBIiLc\\\\\\\/yFjX8l4tYyHN\\\\\\\/+DuPIXQZnERFmM\\\\\\\/T\\\\\\\/W9fHXjuDgEUzJLfL+JAulXI8fWR5iaTYWIeami5kvgQVbFgmv7DOAsWipn+bUVwDJUzGwmoKaPtKoqDyeCpc3DEc08otB06iw6GakLFCsjO79Rpyhkk44ghoTkRj2KM+iAQAdlLOa680wxUtjh4wc7zZFD5cLQscQiwhuBKSn1wHLTgDpyi6p9MG5cpWTeS0SUnb4aET440awXEzyZ1V+wjBK+JVPIHoPuU9tOSX0m1EE2yNcXGhg68UFC38q0N74HK7\\\\\\\/F6ygoq84BbJyZxptBYUZTdQ\\\\\\\/meb0R2zCTJFAl71tSTgt4ibjc1Wfzy6FEeAM1lCP0ncrx0apMUeKbRYMe\\\\\\\/2gqJ8RKn\\\\\\\/ef3xnzgfUum1klI2VxmsEcRV1K1yNzFS0mCUj5HGuI1\\\\\\\/Llr6fof3seC8XFi7P\\\\\\\/zfpXWacoloGNZKRGSQ7I2k9Jwc1\\\\\\\/\\\\\\\/MIjodTnrpSgiGUQqDXXp7ZG+9NTGtsBJaYzDItwUzcbTqO8z5cEcEfCxb\\\\\\\/xfvsqHszWfuhdgeFlL\\\\\\\/Q6tvbiSHoxPiZWSkmOFGuXBuFchYQEbOHL279DgqF3ACTnO2el\\\\\\\/gAsyzZ7YomV\\\\\\\/5hl8irFtxOROip1PfYPKPqze65DxFYOGjyysNHbU7MkIAWdK4rFab3I2QCp\\\\\\\/i3A0gW12jbrIT+C5KsKLtFtlJ0ZmNE9hiweL1vrzjmzyHPC09KfbGv0eDH238m5HgJkEhx28uIBcfpzYGZyiRQg55xO5RoV5GCNWDcSVtV5bAXaMkwkdemfagijFV5ziY85gy\\\\\\\/FwFgCLYj81KmaPGUWdAodo04t1L4o4CEIAM\\\\\\\/5jesrPJ5Ie50N4P\\\\\\\/ZmJfj71X8KkhPrEUUEVfMYW83wcBX3D6yMPBfm3Pv7CBwdnrV2uXEMiDWu25DnpIPH7JlyaDxSI6P7zPTjaUgu5GfGax9b2n1f5sdnWtC3qSaxwWCFYS3en8CA\\\\\\\/OB2hTAZorkq+B1x9x61UCLd2Vxi9Z\\\\\\\/fTsjSXaDinTiPwFxSwoezGx2pVzhA8108Svc6k6X2q52oFG4uAz8HoSoquI7LtCNpLvP7GrX9PEkFLEv\\\\\\\/wvAf\\\\\\\/uUVxk0gcYqE9t4rzKLmDvPNyemaC0WchaZpMeZk+6jdyRGOqHdx1ff+dxTUS6e+v3V4Ea0cSuka6RVFmcqOM3I+0ShNATFSAmU3JiiZIwny2EROM8uLUPtz+I\\\\\\\/g+A5N\\\\\\\/wdqu2KMOaVRc5\\\\\\\/4lKFP7TrtLrPTHAOX5HdQQzHEH8v7JJq6u2DgiweKRfYAIXXcTS3bg1HCb5SgAfStqL0O9lJ8WjM3nj3koj8UPHvG0mZMSktu2z03u0aGkZyBtzYClwMaGkxlxKxCMOJsP\\\\\\\/jx6gKGA48GnA8I1oqcwjCch2DFqbASZqn\\\\\\\/VcshfBZvszluP4a0PZyxhTwsahYP3K7ALZbsxosneo18n6+MO0RyOxJ91kd162juUv+8gV8ebWFAWwfWIso9NiyE\\\\\\\/FLLaF3i1YA8H4iEyVZtr7\\\\\\\/piADPWHxpbZc3zJXXjtPAogfmJfmRl2NXaTEAgUj8665zXy5El4VcTtwtO6TOGKqFfzzlkMUtSfxBHJ9+Rep9cBV5YEYRCzQCV6uXCWbvRVT7EPhIoHaDyadAB\\\\\\\/310dNWCWmE33H77AN3GVm5sv4VEQkDhnTsByKJKC06YlSR7HC9NC\\\\\\\/bJN7Lgrr\\\\\\\/FE0Vf41hHsSCbpPyK+V4mnc0kVRqYcW6Ts5\\\\\\\/lMx8TflZsW7h2daKcg8zhB9glZ8Kg2fvzxND6jKkPfGzX+0neQRENBUib5o\\\\\\\/+DBKUFjFkXJOLcadPFm3YC5mcv5XWC5\\\\\\\/7Wei0NN+\\\\\\\/o\\\\\\\/AC2S6jLZLoQx9D16MpieOCLT1MqVvSO3YoRKrQMnlBJjqnGvYdtbaKoiXu1u2AMJ0ZE+5Z7b4waNXJKNotDmqpVuntQe5h5rDDFrpP\\\\\\\/\\\\\\\/PrALLQuVSMIHtR0Xz8HV5BmznT8sqY7\\\\\\\/gq0ODQy8+poS4ZiOS2tLisolwv24kVs\\\\\\\/BP6pWpAD6XzID3zquIWVqIKNLOOKWlD8kzZJdn8Qozr3nS1OxUt8uPZfDmDQAdNc5OqVlkUwj4drB\\\\\\\/HvSQ9FxoUPVAJCM7CowUR18wY+RakX1kuz1gpVKTqYrzuAQQa5J\\\\\\\/mgqZdv9Xksh9SxDzeGTzSzmblS\\\\\\\/VV\\\\\\\/PUZjNAOz92l5Oej1gMQT4sMthOiQXneM74iTrykCXyR9JK8R3z+8VSMSmBiZZcILPlRQrw9M6\\\\\\\/ZTp3EuOdSTkmxsn2cqcgO+vBFaXs20KGwiXUmqFYcCpBiIIDa45s+tx67GSo6L2PYO\\\\\\\/L9qhxP4qGzaW36+gBT4Et8Xv9R3L\\\\\\\/npm7lfqUt\\\\\\\/IZiRyyJ7b\\\\\\\/xJk9Gg8KTlXH2VowPMezWG6kIKu4fuTpI6A0DwAIns59bq4RC8kbgh4nM1myWtReWiU15LFsd\\\\\\\/iMgE4N+ftuw00mux2H0DImvtQl\\\\\\\/X6rd+dvLpIT99vBLhgFT8VWi4xEKusLKm4bRhTsOV6ov7KcU2YdUpxCZx5+3GnDyBhKFVckvGtmE35J9nrsmx4l2KErWvD2n5YcA+BzPaRHSi4hdzB8vxvUEetQ1PGn81kkjSbMmoQ06EGh81Ib+jbv0A9Rr8NOCFYAGwLGpVYDjL5qoPEkH7QiZ3WgCWpNt2aVJDS2kHeSXYOnjFdSctAhEEbjj5KBcDLXlxD3PxbIybYfQUbU+Q0PF5qudR5LsCG76+4T75RRmqgCXRKc4Afx1l7wo5c2Zo00vx0OHO4SJ9cM3pOLVb+cBuaZkIKsmjv+Yw70YprGuXfLb9Y7kFa1D7paRrH0H6saugFCTEkfEkYfLYr\\\\\\\/zdarr2tseIng4A7TkfCd6i3oV1LIjQbwhZB2JhFQRAYvzVaUayGa29a\\\\\\\/I34birpsNRhQ28MLGxqUQZ7mrvrwp0Lg0wOVsfrWUSOgrzBCYS6PWrcfjWidorQOWidhTe6YfCSEzWxt70zpq5qs+qE1eB5TrCPDUEEAhvH4gVW5Dc06oVeX4NJaP267k1Bk1HakOymd9EbMRErrZqASHnRgaeY9OUCxYP78RnNIexdCjdohVoUO5tzR87QLZzdN4xD2m27+TnIHcSkeg6wZ7JMlutZjMx4tItoR9R7qnu9s1DB2Jh+go7aMj\\\\\\\/8H7UjUj9TJfMdzryaE0MxCTPaz0riMhEemobsTWckE3zREGyJbpNLcExjt2nBAK3A8fo\\\\\\\/mfkFE1srY+zubktEzSzgIT9RPLVQhObxwUFB5ZzjUPvvuuc5vAQdsrgcowLTR1\\\\\\\/7sKNFTwrMsI4YS\\\\\\\/L+qZPQ4NDTp+E0f3aDQOMhSbxDI8yrqSeokVib+B8I8oTLZl4Iz4QVGVhjFP38+bhGeN4z5CkR5yp\\\\\\\/2qYCFZxVgqq0KXCXJZMw1JaWH1Q8qpfgoIBZZ6HjhtQoJShz91hiRQS\\\\\\\/lmvyrxoCdfdSQC9QX+fq3LNnnWcz37paax15XX9GnOrXAsVpUfZdw\\\\\\\/8F0mNFYbcQsEUVlSoWo1EwxMm1amHKQkG4QH7mF7V6ZJ7Wco7vCPTK9fsIwd2aZGeq7\\\\\\\/8O+GQLrFbZXpipa6t3gZu43\\\\\\\/cRxqUOM4QVhoAeJJD\\\\\\\/5Um1g+TZGUMqVb7T8rPKBUT2U3QaIfFeZSN1XJ+J55hQrtE5FbUY0mKU3QlLUWSVytB7omezeRsSuXDkI1jdHmPfhKd8IO3QLX\\\\\\\/jXy0vPzmOwp8Tyi5OZ65kNAplm+UtdRm1NsjJxuJzcB3YdyOmcagzBq9D+kCUSWKZZueci\\\\\\\/G+FlCeAIS\\\\\\\/EIuU9yl8z0of2eWzD433XBkYoLTJjPSUs2gPAoBaw686iD0GRgb99mp3uG6WiRB7zhqbEriF69nqTxc+LI\\\\\\\/xvjw4++Xoa6PfW6OYqDyHP7aIYkPg3tDXCLtZxon\\\\\\\/Ij93L3bbd9gaJWtQ6jVUvdpps9Hsoz+Dr6brlVj0cZYkNJSYtvzi66vJJoozVyffwM2oaVt5Q2fBJ8zSfuNdBKVxBNMOOTsJqErqvOJiSC6Qr5S78IOleIw1dPRjYAsraihQP2lT1djcvZKRAAm7IElUAb1netqTmSbxU\\\\\\\/xjqcD4PuFFwrtPYbNZ3zketlzKBkuePzpd0m+NtmexaJqPi81+l5v9GEMoK3zBbTTl5Ejks00fpQOSYah4N+t0sPTFGoIfC0Em6tKZ8nQpksAFmsPiolLmHV6ymQNi4bkVEuy95XofFUD13j3y+qwgIgLSanjxsGjZZzU9k6pBxnb0HLSowgYgfHRzNrKx\\\\\\\/fM1L6Qc1AWxRvEvKBGn3W38O5YKF5B3RQm5m2raxJVx2rdVHjx59yZ3u0an7ZopruG4mdRPHXcB0ihMJMn\\\\\\\/na5PvXDXnt1\\\\\\\/MemO53Co3KVy03mziE\\\\\\\/sFj152s+9QtVRGcevgqgF80J4Y1nCoNB4ZY406343L\\\\\\\/038KTUdorhpKjHD4ZebftxCISQK39vbdmzsXl5O5uA9HxrVLRw51RBfVVwKo8ZamYeklHvf\\\\\\\/4\\\\\\\/cJEyuMvlSqlry2Mel8EaFH2KQgIb0ms3t0XOFFTNp3DVmessThFXZOW+zE7QxEQS5kZORmPFECEfkgQWjOghRiJ6Qq5JPyRXmWovjMciG91iWXNy7wt4PkE6xQD67r8oPddF4uRiJzo\\\\\\\/O47bY7vr2PfFfU5abXbGfAclpRbP6iaLOKhLu6bxzeC39luOjKk+ErumLf2dWyMtR6MyaH8u623GuskWbAR7bgd554k\\\\\\\/N2gL4XfSngOzW6lkd3w16rWDdqjdHJKsl+RyEDORrUa7zYk7bBX9vmiOCvBTvXP78Ko0PGAprsQ6iAvgQh9KNcLCJBp3bRAbEwSxIKTEmo5k+\\\\\\\/\\\\\\\/dI7lFSWThDCyB8bHLce1BQYhTVwB16ib8FMWqVWO7MowMsd5DVvMJqjk+HGxJTEG9b4nyHd0tlgf4L493c2Mmwb4xC0VnhmDj802wE8z\\\\\\\/ANw7DHFsIdSCN4w6jM0ZdQ9NjO2LSQi5SwuTxN9pjXCN1MfHxda4N6TPI2+N1mPJmsL+2Fs87N\\\\\\\/bFA1DfdQoWuWki05ky0vyx43dPyYVlT0nkjGQTZIDA2fD9QxLht82HAnePBXEQa5XtkYjC4WwJ0nBWsTDdqpJPCa2irYlzAUCKp9MPOoz\\\\\\\/GL+Lzx5cHyiYaDzvvxRROS81lYmubuKzYMdyzLGQeIKmSutEkGy9WFMLJG25rwJDOn829dCFoZ+TnVrDpg1zD+LbfBUgGsyzduodoq3Qa6T3b2xSwEkCE9Mx2OTv8IIgr\\\\\\\/Tc8DbqT\\\\\\\/dtMZRFe96AqACQUnB1xNCRcDxipbgzezxa86MGYhzC5e0ow2u44Oc2lXEg6bx3Cnj\\\\\\\/mRY\\\\\\\/kYczBQJ0DP4dUF\\\\\\\/rhUfOStsdIVLkMCOG67sHTh6vP7RLJDmWk66wyDTPCt2QCavivmoISimdoQ3dxc8BpcD0KmLkdnQuJ20Q4t1n\\\\\\\/PZ47mJD9\\\\\\\/CVRjXCY+voReOY3PJj\\\\\\\/mfM7rx3Ppf1zovVKxxKy80\\\\\\\/j3TrNhgThPJ9PzmPRH7qnL6T4xRoUF7hhUWB0de8X5KD0rEsigrnTq\\\\\\\/zGwmB2e02d+eRrmtj3AJPN8TLNLng1JEIhuVX+nYYQRoMo6i2hzldplueilownnLKCe9Mis3Be1iRX\\\\\\\/NPqqYYcWCZuQuf+c8GYvsxAmcQv7C7eG2jXEiPn0mSLDCZVirsfDjYRxHGlUuRYaKhfOM+TWuMVJ1pyLCn7hWcDKtwLyl3uMRMauhGG7RCn29WG3eIPeVjVInxW7pWqCs+7tjM9wYYY2u9UC+ffRdvrHH9FoUz67ch9eFpTpu46Uv3kQ\\\\\\\/HutWmO8MtNB21tkL2Ewo2R80BzNQNK19LvuseqEhuw+EGGdv62TUEX4oxSlw6LKw6zMgToSsr7hrnAAbqBlH93PYz59X44PJdW6xQsMvXlhPsodqfVHMvEVV9RmJiw5a6With34CAInA9K8htwn8bEqJEd8FljD\\\\\\\/6YbCc+RrnQh9Q2\\\\\\\/qKAHUeJ46GfqXZXW6k1hSzJAMx8RlYOsPv6fqvM3itYgQzvVwvEK4tuZW9W+2qAKmBvKGQ8vv2+OulJu5cwOpb0VFMbNhSrlwjeNlZ29m9Sk\\\\\\\/LvyQ6AoUQg6Dih\\\\\\\/WlXPoXwYJZCgJjn8PGh8qDQ2BTSWgtXZNeZ56U7gS8VoEuSWNSHTIE9FsPyv9OVJoNx\\\\\\\/x8FDRBH904UriizXOViUEVCEmR7K\\\\\\\/QCcdX2dSM6zBc\\\\\\\/eBYYvku43gUP7nyTQjtGkQxXL80H2rIPSHkBfSzOo9Tt10HRSCQ8WKfVhgaWQ1QEgNcgvCQr6Jom4gWFx1Z\\\\\\\/scQI\\\\\\\/Kbg+bQ1hXeEndEtuGq6eILsBSVwVXVZv7txnff94TsrLcSQ2QR2aU3TpPLKPCwGvqPzxVybTm\\\\\\\/yoJIIOEzNyvXjUjbqP+XYEzNL3EbuySKeRg69h9EoYowmqhxC5Dw3Kf2mwBDNLm0kmGYzCcJ0eVwAj8tQgaaOoiRGqgcaWQyCml+Fw+YB1QgYtlA7eAJqJNAM+mI+tNnmr89N8lIoQ77ZMvXfFxHFp7wN12tCmV626Cj3N5jAe9Gc91S32YepslcsKwvNst+Bc+i9\\\\\\\/rCFP+VSj5yOTo0fzjcrbbGlGHn8bD04iKdEvShzSje2\\\\\\\/zQO9\\\\\\\/cw4Y6HxdXqWEv+NmrnUQMEYkhr6ALfrbPb9bZgGETGRwsYP6bYA7G\\\\\\\/BRgvEgzjVcfgxO8q\\\\\\\/842aIMWfjeItJBl1yoTkkMj7y7dftlBWp+8RkLky+ow2ogE9pM5ZGLLXujfXg\\\\\\\/EO\\\\\\\/x6MzXvWPAF1dxGLI1eQB6q4xSkuzdaT7sYcYr+M4k305hoiOtYzlDzAp7\\\\\\\/SJ0OUdD5A359w8KTxBfdNOhrbWpbzxAhPSq2qferuh+Jeo+LcOOZhz5F82aSZdVkxpRS0\\\\\\\/yKyHxO7eMW4pokD7w9uMSNJjJofYYiUvUmb8X+9k05LYZanL\\\\\\\/xkkOSUGIxHqB\\\\\\\/RbR\\\\\\\/0BAuyMLUzxbaPsMi9Jts6I0Cd\\\\\\\/rgDY0bp6Nuid9ov1qmpBdvqIoqsOgANYxkmU+M1qe\\\\\\\/FkjA9xCrWw7nYGPgEVdNRvBfSXY6dWDzkie6FRa5xH8lU5RZJjTUiH6tGCFfwJF\\\\\\\/J67Qx42yiMfU+QiJyLR+oPVXZF8Lzgcz9gqHLClK66me2+qd\\\\\\\/4g4h1HbVWd44rSM7vOL79dfor515zbi8NkAHs6w1sNBW4qemYdqMrBidmATwbYB+X1Hzpzx8e+0rBi1N9mwLinfaFIF9UdHxVwNy5GnU5bmJgzlg1nHB+6pbruLnapCbsLgch6jKRj0ykRcJ3OEv1RuiZJravCWfkOV93y9Z8UppwWohMvJQvtdKCFDqe+LusBdisWmoJnB7oqjz68cF6SJKoqTUTARo6wrZso5W2VR4s4pVKAWDRuB\\\\\\\/OOJQ0brrb2H\\\\\\\/qEjz8C7nrpf1vKaY0k6W9X265uIpzy8PHNW5JtsTzvGhaZtHUekVi+Ta7bBVnhkFrP1lHahSq+JekBqP\\\\\\\/kq2wdudeBMQE1I6tRBqqPUFt4I0r3gerDeJ7wYb50lxuGzFbdE+28lm6mf7RFCnpZF9KOKpXwEuhhkgRlKvL8uh7egzP11g5A05njrv1tPQFXU+a+Dil3Xu6hFt1pNz9W2YMScWWvu28M4yoz92avC54tNkvzeqdluO56J3g9lwjPRNbPQUSCI+8Jyp9BJoEz2mekRcXNiZNGmyJ\\\\\\\/Kc3sz5p1U3cTcA\\\\\\\/iyRVDMzoYNWJTdqJsWG4Trv4Hi2ANdWc6iinJYlaVYUaqegWxGM+NprAEWuMOkrZ0RKqumez+akP8DvmFh2PyLynIchJSKlLFGKlAPkDTgy\\\\\\\/EmGcNHEkDn4MH4HqE4YOSNPtJPU0KdUuM66yqLibKpk43gnCxUvIWrJDeZ7GAFxVarr5L31qtaOcP93OZow+u8p2kgIp8zmBrvB63eZhARgM\\\\\\\/iJ12kWG+OEGm0fVB5OM+V\\\\\\\/97CydgIrJKZNHsd0D5gvUF3MI+tzONgIMYVF5RehpIFMsQ6aGUUq5Bsv\\\\\\\/nfheMmO7ABPVrTa06G7klTiBeLh3JFfiPVquLC\\\\\\\/o1INmCNXB3TGa5VePNOf7Oc\\\\\\\/gdtvzZbijxTL5mKWApyuYVSWnZ38PDJLuCkf9dqUf9MSPKLwUh\\\\\\\/8\\\\\\\/UGcHW3lsYw9muTuuixrjyNfjfHfmarNqFtZRA5LJ9qbwl3nPftkKOhpkg7NNgK5hdTGKM8dbCB3zxjllKdFclXhgM\\\\\\\/15u9l6K+tVXGPGv6qugklxc4Wyrp2htX3kODtFYuYmkSlGdNxF4UUD4cxidaCsoDUqZkGMBx1zsNUfu3CQkRKZWmfSbbNn5Kuy+nId0bsmbkkxph6KqpHdTiga9zF7iMepMALxfAWbY+\\\\\\\/izIiZX7XWhNeZh5hEgOuB2N5CJBsQ5z7ztMNzprS6+N+x5dwA9elba\\\\\\\/GUfiZu9UZIBltsb2AbMtukKccgcbjROqL+Xvkp\\\\\\\/gb35\\\\\\\/cwco9kyn7bTgCipp4K4P1YnWk+a\\\\\\\/PIgSZEanLqNx57m74\\\\\\\/uIjkYqy7VrXOZN9HJ0\\\\\\\/GWxjKmlLDL0pMlcDW4LM9la80ZdTFiJTeujWAA\\\\\\\/eyVneI5FlTLMxUixurDTVG3nBSBUGi6vBIFtYGUKIMdOscLtaEVzSgvD2RpcQQQEMMBdSpvqYa\\\\\\\/TloOdX5qNkolFylv3DVZ5WI+x7imEu7IYjgSfTlni0jT5DqBDzWInnOJepxpHlxdSjCGfQyBJdDFxrURYeJRYhljXrIjtkq6YlIAOTlBIXAb8Em1WO9o2oaMBE37trrdPsMMQqqeQuqpcAkc8bKP1PcXErQvTU0hwwbDm8jDuWEZomustkXFE1ctDiVn7VuP0PXnLIiTEXi\\\\\\\/\\\\\\\/ZyKFPEChtZWvXKmcKOJT0lfkIZa7OEDjjGRYi6TlCfFr8nQ6lRIAoE2QSmRyhIwebqRpa8yjyrB9ubblnp9N8+8pvBa8NDAQCjnzVeu\\\\\\\/o7ZySN7OOHo3+QoTxVpXvhpP5Y6t6PTJz\\\\\\\/hdnNIJUSYSN0BSTeK27\\\\\\\/+IkUpgNv5ci0+MLM8NAS6BiFAQ8+svwt7+8enhc13Eb0qVBen7pmcnqu0avmqT7\\\\\\\/HaHlD\\\\\\\/rl\\\\\\\/EI1Y+722MbiGuDgvqd8OZds8eMV159buy6i4OBmnuLoIeIC4+CGBoqn6t7Qoh+XxeSOSEk3+SpmSTw3aX7roqM2z5AQ\\\\\\\/EsKELhywB4zzohlzBGaj2k\\\\\\\/i3\\\\\\\/CEE1oE8SobgoMHVncb0EyHPZMD+xX+FVTGh1ru0NzFwHTU2ZaQj5FTri\\\\\\\/GYddD0wNquo4plfUtpjdPB9GmKZPfXoF+PPi6kjYD+tVibwmabHxe+H8oJaYEaO1Oj27i4An4s1FYO3\\\\\\\/QGYF0DtsjmFvxfVeiRxnH4x\\\\\\\/myhTxi83QNssfKXxnXP0EomDE649\\\\\\\/R7W0Fwj3npoAQRJpm75YQJnJxkhNFIckXnq2veCdxyuJ5rTChNGt6bn654u26NzCYzZAGzxC84LlUX+UMAhl0n++aDmkhLtoc3+o5njJKnOe8ajWuf+V9JlJ6MK5Xm9RC4rHOEQuqztCj7rPHp9A1Lnb+KAIcZVuQ66pbGv3kMj9MeeVHF62fDsmPcmib0pqySnH3yjpIID0RG26V3YQuR7Y0XdFv2ioO2aBKo905da6X8rwPrgr3RrEIYkzEw88ZS83Mvaz7mKfEuf7R1j+mlmIORFgEWULkBxGbiifYJjRnWWRf9k\\\\\\\/VczuinAEPrDfRmGvThXsRiWSvrdjM84w6UEad28TZMjYJlYbmBVqjUOjJPYFm72s1nrYNWsASq2zMS+dh1kTdGe6j5EQSdB\\\\\\\/DB16YviM85ZAk2Vlv8pZkVbRKJpQX\\\\\\\/la0Hczk\\\\\\\/kTStyoYKSjLfapUuk4F+Wrj7Y9x+yS1jvYf0VJR9ImwTsIUPwWobDbEA5aPwEdrjYm8HctsrJiWetC\\\\\\\/GTIwjJ2r\\\\\\\/Ah6uGn9+EqEh203kyoCXB60U6\\\\\\\/hmAMjfrY4rrUMhadt9bnYHjSRNe9\\\\\\\/aOhVve\\\\\\\/xwfx0LYPlZpx5ssguTYD34saCnURH6k8S7DUgi+NwbNnBzNXphaPgRTPyCKvgj\\\\\\\/y2gBqlYfA\\\\\\\/3Ajq1TjqkCH+hbCbtQloSyIsWaYvTwMP4KLqGr7B9Mb5Q0GV4rVmyy4VcjNoFips\\\\\\\/V0difg8XwbbJnD6GC7NgmhTi8fdd+jSeUENmjdICqEFahtY7YICbsNkD13XxQaJNJLT0Ol7NKba\\\\\\\/GfbMgJt3FZSpcEYDBrpCPtf99IWyuoGn8ePMoyRDXU+fm7x2JeCmTHHVzNFtt8nB+j+i4VrzQIaAdKuY46xwgLvB5gJU8Yijkk+7WajlFYZBmBzFAbahRZdfqL6vad6RvcsjMtMnxZ8BgtHiIo3Z\\\\\\\/KC2Gb05nBYsgJGEDdW\\\\\\\/2CW+xHh4g45K6A0mhEi6JFycwebWlULrFOdAGl48g3qNAbroyUgJOdQ8oaxG0PPNx6VK6rle8NavUXNBPFi+SkRxXyMlOdd71L5wMsJSYfvip\\\\\\\/ipSDPYX4k3fgmtgevWDf2TEKDnwZ8UwksSZg3dpOvghYnES580LoeDH6RMQJUz0P9Rg+C+Zg1eH6GdYqlCwSes\\\\\\\/\\\\\\\/7iLCiTMWj3pFWT9abZFG4ZugbPNbsgMFEqewxQuk9vWYPpLItosENufP89EKlAa\\\\\\\/4p9sLAsspM2sNiWNYaVPbwIx6lyBQMyQnmYk7Ifk\\\\\\\/2+sjoDGGUEmuSmTpm\\\\\\\/pUbFiG+LXWbahlHiQtMEolvjgRHpgcngccawjWCdsTPm6esDYxgoMkA1IFgV3DcZRUmScD1vCIjn3WhOX35ghPBzUsfSjMza7yH5dote3TELW2NPPyuGSnpBuVXJMU8MZj9SKxmDgWvCV2wWl45Md4zdZsHV1UFtzgKQTsD0o89fi0HskwTW16+SuQcW118LyNGbMGimYuWNmTREVCqmAOQQdEJvhHtgZZhIXX820dFunbGBEt2Hh6ox1wqPYV42O6spKyuLagZ8LAeLd5rx29hKyfoyb\\\\\\\/nP108jGi54eTuklBKXq\\\\\\\/kyQ0hvNRU82lyoZDXqq\\\\\\\/L0\\\\\\\/peJ5Np7joattaO6zMvleD3F+5wQjsX6y\\\\\\\/ElRO7uVxDS1NHYv5nwksgnhFNZ1vC6CkW6RIll0TJ0BHMkdWRyVbEfi30X2VrjeocCEi7l9woasP\\\\\\\/nfs2v51+goEk+yzz3zMQapIHuXCfgKbeiToSLvNT8FxH5cV5BJuP\\\\\\\/5ghnVOiqADMwQLeKNDoTAoAkG1B8z7KBdUgQxvwHQScKLxInCPqN2E5YcnGN5wp5sNC4VFbUvYTNrWLFJc71wdtI5czzCoYZu9wDGUDPKBdIopB5+gQ\\\\\\\/W1mCzWAOUPa2fxHPA+vTsBmoYx7p+9mnx6QJ1+vI7VR2H4v5GZ7wR54OLyFDX0E1UzFIFcofAjgBuyOWw7Svg0LDJ+8tNzxfUkgkYPhhShjMd68AljJuev9KPu22GnlVoUdrvxMeqVz4UN03b9uKyedzEcRzYAHstTUc+pg09SrqRRhk0YC1leO10sFQp+6nYPDfXm3T8JUsHBsxY7TqyJHnI3F8kir63kIfb1iIYMY6ndWyuP7ZYPg+WUQwqqR+z0iX2GjWlpO79Qgdruh9X3r90+vo2alobySUMfU7r8xMykhxkuCMuhJ5kBh4BMIy\\\\\\\/iqDz+9pM2GP2q\\\\\\\/l682m3EhAus\\\\\\\/3UcyX\\\\\\\/mKL6k5rM6345Nm\\\\\\\/0nO8qEjA74CCq2tRPuNQr38iNQ7hONb7k6ak23xKmckRvoxVlYNoFpzUo7IhPtApMtPEQPJnUy9JDxNhFtAkzf12W+BQjD0\\\\\\\/WTWbTX\\\\\\\/EKxYRvhVL+x9gPsKyGmgHIQ0EjI2T9c1QPdxSg4qJKxz\\\\\\\/U+G+mD2\\\\\\\/Auux3lwxu3HiC10q3tycrojAMotpXMPu0qNznJTQ0cJidYHd4NWBHPrF\\\\\\\/Hhi3JJxMn9nMJLTT4E2VLbQnZFDCeLFsHfAApK0C0gcO5Ocd7PRLvO1p+dKwustgxNtNnvZ7gE4UmKRYczH32LXpsLnglI8qUoZ\\\\\\\/tzAvQsu37TPjwZaa8tlRvDk2z1kf8UDltBQtR4IzkCC1Fy\\\\\\\/gzrLt2KC0u\\\\\\\/brjdim9XA71I3DR6dcJcNLGCGnmmQPw1IIcgU0OXDs+7csDSpRWEk2PeTT5H8lu\\\\\\\/j1Ch6lDDm\\\\\\\/tJTIm1KtiNn2w1Cz\\\\\\\/RS6AZxhJccuN7muFsMAdwKo64z65r9ex+SIYeNZ2Ze4O17NgVTshDRDzncIgn7aN9NmjGFzix\\\\\\\/SfTlY9900JpzCWUlq78\\\\\\\/vAMC5ODWaXfTvT7FM+h7BVZXfGvitpSkS4\\\\\\\/pqvsQ+pZ9V+I6YxANEDDdKp1+YhUoNdS5MAy2f1toENIBPks41E7NnYOq9qHf1z5re2k2V9gP\\\\\\\/MNozfg4RxToqN3h9P5SLsySfhVN28Pic4ssmivm2W1x190qEVm86Im5r6rGFbuo3zdCJJHo6Qu\\\\\\\/2h8AtijWk\\\\\\\/n3XP8vyOz5KwUfkeW3Bg5Hsg4+949\\\\\\\/2rGmH5Wvz+T5tZaFXbJjVSGcapydxMV0wlyQdS1hmRe109GsZQS4FIvViFDrdse2QsxVSdjQtbJ5Qgv1GfCly6j7NBhKjaiBiIICsMBTD5A1Fs7wPrUER+\\\\\\\/EIh7Gh8F\\\\\\\/eBSyPRyzrz5VB1eNtqAYZHWyvT1cRF+6ajpqIkwg5L6OZJlmoUPNbI0zNzMtFTc84qrG1fC6b4PzkVAfSScuv9SSO5LTHgk3llIMmUrOYuUWehHpzJ2zFy4Zt2F3ce2gqA9tO1i8FSkOiTXzJBypkatWN\\\\\\\/ksOps4TyoClOxhVsardrZ8VmBBtsdJoGzwR3GOrCa8ePOwrSrr597OXcbgXSuxVtLapoG2BWLE1UipTnIpnn\\\\\\\/PC3dUrNKw\\\\\\\/1sbxzppqnUZACb8aMc2X\\\\\\\/3ZFbPCP++LxvlQjNe3tQTeLlMnp5L9OG+Z4w6NXOkQIU85FwYnsmjwSaPWMUxRc+e7qryw+SmNhvNifLrUiYD4QAYhnD37aKxZX+45fNkN\\\\\\\/WkxsAH\\\\\\\/wSgQW1vtlvX0r4IS4SFp4Vbx74dRtFyfS6OSeUmF3Ehlo9n9YNJWr8tbSAjGrk47R2Af+nhUmDRxiDcTwiB1MZyiN4AflzqjKSofHrP8Tgfwi4U06y\\\\\\\/GVWJs+Byjus3juXj4n5iMciAMYOsgHPhOe16DBdRs49eqrFXV7sURejeyACz6t7HIxI1M6csbaK2mSVQrRK1BDrgZYvTFX7mo9HDMqOzyxJifON6CjEwkxickSbGQQMVV0nQh7u5aexaxVzhvxlrowZLQ9sCogZygzHdh0GS62ikK3ODNagUsTpVhPleC1k0sTpFidIwN1zyy9lS+\\\\\\\/GW64vLwmtc+AoTK64a57SN3knglVyGMGIKmlw2fDMeP5ak3bHO4l5dnxdKKraQCo+MuaczjK1UwL45X6v71b6lbHAk+25t2i1BkLwNaf7tgBwUogefwVPWgZI1y+fu4uPO9\\\\\\\/Pccsn3UdyMpzL\\\\\\\/cKulGLGu\\\\\\\/+lewr8irrRbRdE58ZLIok8gAs59ws9ke31uLSceYCKRttk4ruZp\\\\\\\/RZk1ExiiFaSGaH7YyFqDeMX6PWODb4\\\\\\\/UmN21nRL+mUnZUqKilh07KD7XigaRLxKUHOeDp4q01VTLNnSiQ8b0kKXd3aiZKyW\\\\\\\/ykcVD0TgCN\\\\\\\/i5fRAR7jaAu+T5CifXVi4CKgS305uoPaHhdUwcla+LARxe6ILBettqplce\\\\\\\/LQuh2I047AwaJFBrVXACY8su7p02ZIVaD\\\\\\\/T7xmhf42P09wP0Ou0COxPOmbxGGHwViQEJAkVVO0ykRTsTbw8hZl\\\\\\\/cZcwUfYU\\\\\\\/VifkvzPnro9VzJqHjYvz4fZmQ+kxPNvCN13IDHOPrwfjdpwq6mP9iGAJ+Tm+hNi6IYlEe6Vm3hR+ZgVxZqZwIXSvJFZOiKR765sML125+KlzlsgbgAdf\\\\\\\/3g87qPPiUQzVNeNEeY2xu3szDA1s1g8P6nULjT4ZlKbl+6m8sGWrDaIWJXFL2qXQRGHH9CJNesONEP7QitonEI2AiAtv6Rk+7vtBAlI8CZMpux4hY2nyZmmhR6GE8ZqihYdiUmMfP4fbG76Y5WrmRmPsueaAYnB2546wckFHpFE2erA37rNRFfnxFcKVNZsimdMRq0xrb3MJR4AauIb9T3VdFa\\\\\\\/3UzrC4TfeAVXWruAmmvNdxX5Ff9oGwXDrijOe4C73ktHHi8IZDZb5B+3\\\\\\\/SuElGfVbttZ\\\\\\\/6liaP5YpfFN\\\\\\\/gUX1Ens3QrA2ELMLrWP6NcCpWLIzKHpLSVQGE1NCGs5ILmf21pPV84iAFsbzHDmWqD8\\\\\\\/1wlGouTCITotqFEoZnxrZ3ZgDLCd+lQWAV\\\\\\\/wRpLSWpNgk74DVScIEb5paYq+EVTKh8NFdWK8dYwWtESJ7dI5GfSGCeIZVgF9UwHoZtzkW2E0K01KiwKW1ZAKc6QeKq1xOpRldSn5LBsnkhA+ksQWgYu6jrNgzLoCk0HXawVjVcfXJBJVpVZXzhcfM8r60s4hKyBXO3P6HDwgWpQmG\\\\\\\/1LWPD+cKjIssYp00OpzkxQvthcN1WeonejKT2pjDG5PRivzVZw0IuQLyJr3S\\\\\\\/TctLil1g9dlKeHF54DP5H6a6mQLHPh732oaVycTYD\\\\\\\/EEs3\\\\\\\/c6B18qusNNhVp80T9cSaU+vTpDG2ISTM4xV3T7hsCbjIK7TRuRt3XxPVfHylgJz6LZS7aNhpt0giFJOfbgn0O4qyubcS3ndjT+rE3XRDnyzoCzNHSKoEJosX3nL76KjztomeKHX5QmULrsaIv2fSOeGmdMWpNbqTAbQsDQ2xZECWAVciP8oKHrCl0Z7LvDv6sp5x4fv9Fxr8j7kVh+2KblEd5dxIw3zZZ6xmnIqtME3YsLuWYIKexaM0kQgHLUG1ERiRiRjBWb3\\\\\\\/hzQAuhKmkwfjeUW9xoHy+sQx0xkygW83gt2hAvYJG2+uTWWeVfxYjesg0SOcUASxOTU1xZX4CULRtHgVEDM1t4KRybqTTH6gBzSLo97z+rzn10IAdm8dP6tICRyp3KqDk2gzAdpexrhJkQXmyPBEjPZZ\\\\\\\/JCfSXTO0Q+GQlJ5AcLV+yy9kF9XZt4LTIFFrKLVWLbRFUbtTf+0edAsbpNHGozPnGpusrDmh9mTR+DDEGPKpoS+4h\\\\\\\/tej6h5doqL75F\\\\\\\/mculu8A85Zmh+XU60ImCgJMSQaP1bXtaTJ7JeEE7g3lwhbrXqZYf8fDiWUib2cM8RUcnGPVZFsjpx0WNIL3Xb\\\\\\\/2SS24UMKAAi2\\\\\\\/NGu\\\\\\\/qoXFz9ioAoe+oPfPam4MDgk67Sinho1Yma67HXvSbW5WOjsoNw+hHXsvtqnsEETYwYigBEI3unw2LJeDgyVL1FonXoUPAbqmE+Z9mFph6ezBFVSm4MXh6oF7p5UQyD876uvFEGTqldQPcUA4A4NZADHOY9TDJwk8w32A9igCul+zpU1eR7BtacM9M8AUPZMMVoOd2ryaiSu2A6gYJCLmY37vrTKOzi\\\\\\\/Dh9E65Rmb\\\\\\\/aYWk3bdSLlt0N3L2EUqKyn8ayiWF4Cv9i4NBQn2rdYhF4XgdQKo1hQSPPmJIqV8bHK1MJRhmv8MCdgmoDTmGNkr2b5MeGcx4\\\\\\\/qyjg+u2+b9mg1RuGEz70DUutfh1KPwcIZXngWuxX+sk\\\\\\\/37GNwtc\\\\\\\/DaXom3eB2fvbX2QMhe4NaMgJQYjb+Ds\\\\\\\/NuDtrAfK6Hw3A\\\\\\\/SAqovaBLVKViODSbmlV3UT4NKEE5G0FxNyIeZYzDyaiSdKGQzHvoVBkOTGVBtrxR\\\\\\\/o7GNHieyhtJz1KHv22jNB6HNHitAv1yLhhulFWvv+8aOxtnzy2n+R680dmtAPpZjSiw6yAXv5+yXAqwTwtMdxps5adjnG28VXhWSeQm5jPsvwV5bLGjnH9kyJrFIwXJAT97S2yV8F1ZpYUPEmcP\\\\\\\/oYukrX\\\\\\\/kqXmbqg+5fWp0Dlv8pgUc3xy1xDKLSVbrtXkvayoW93M0xj+Ahx44JqGyIOT7Kl+e9+npPk2eJI4vVtZovg7fLDObgaYqfkVYadwx55HbievgshUcQbPHT62x9\\\\\\\/NKWbelY5ewK\\\\\\\/FcKWse4drA3BNchDnhX+jgdy85kWPaHG8g+6Lx+AqMw1ZPVNqzeeR+bV\\\\\\\/d4dOg4rhediDA9wn+uljB1RAcxpfhDR8Rl9VkLvefEqx4jouybzCG6cRgVYHUuWINzDNP4bV3C3B7Ziei\\\\\\\/lCh+h+Lf0WPImufGxFGLCs3oeHmsdt5nLKNQbdVIHIXwVx4PkDwz1ryrcoRceH3oPAi7qOckeTzNQ0\\\\\\\/b1gDcmAzan21mkcuacD51TGNwZIAPPuHTADlqwCzLJUyLKK+x8Cz6SXU9CuYdE67ZzGYH36LnFwNlt1pqTvYbfzVM4ZHQGFOKidG7GjF0sNiJHDUZLEHt7eJFdwfucm1CDKIqVJafgFZcxuUiTLJ0ybABMbuRmSFws8uhItrTX9+DSJRjsFPdNTcOXax1EPUxQNeKSoIrF7IlsT2fop8tHGRNkHO7Zbum8JP+ROV7RhY\\\\\\\/xN3PkiX9A9\\\\\\\/hzW1pZVOlkZ4rn01CHka2ZGbIWz\\\\\\\/VQ43V15fparmbob81k2aYkS5\\\\\\\/ggM4LAmeTMRxz2BD7IokbcmVG371rikc6PQmC2XGAIvLMeV1X76N+z0ZKPa7R63ujRWtrFGSgl3BNHIzfEqfISKgcEWIzFPSm8Ooa0lqFosizcae6KnfEPjtQte\\\\\\\/s3BX8LCZj9cpv9k6e8jx3aIwQ5JGQbUJ1G8WBswEx6Xf7UaSNFu5Cysd4jyr77HsIenobvgzE3DrRe5sKTTWqm+H1vg+ud0sTSKleeFnakiIxv\\\\\\\/+JiOcpohdT1Gkb9\\\\\\\/y5i0VWeJiL4afPMuu3Eez4jwSuGHW7FtL5Gmviaq+cVAT8H\\\\\\\/eZZqG+w\\\\\\\/MOkzsXlh3SliS\\\\\\\/voaCNSEOnl4Oq1VJBp9L9FyQEBKfCU8oGJU7qmW6lbChpvJ7WiOuBB6ok4gMXY1rAh4Hpmaq\\\\\\\/6YiVt43q8KP1JMWhamV+xoObbJeylFW3bwB2fL99VRH4pOGvwP4pJzfcXbpOIdSjjr8+L1adL5CuQV9KWUnCprZ10Ih3ohoxasBY0d85BCNUqhpkv9mF7br+7e\\\\\\\/Am5te9Y7quJFHXCW0wUcHnzrsEtrM\\\\\\\/0eEfS2SvOLfd4EkTBqHd7307wb6Ff\\\\\\\/qWaivmlKT\\\\\\\/\\\\\\\/1+uXuZ24CszxFJiYA0bxq\\\\\\\/XXbUpVO1zO6sFpPEUj3c8PhkIfr6GohBQeeo8eocu3isIcwKb3hc1dvUjyzB5dtzG59wLzTeFGM2GW6vkv\\\\\\\/u2DkJ\\\\\\\/HJsDrcYTbVLwfbAq+YnXDJNQPL+JTbHWTppcU4pL\\\\\\\/S82CPYudDEjHk7i6AsU3DR8mTxTG8cemPO06GSL94rKpbZy5czuVxADU\\\\\\\/+31rQ8soF3FfvV2mQyEeVEguvqjH9zZAAKuRblLZajlf8XNyeA7P5H0RgHRBYMouVa3b0jRnvwHNOO5QlkFL2wG1QKYFIHfupO3Gsfrh3AE99GOcXkd5Ns7V7uC4q\\\\\\\/zoDkcMb12tx\\\\\\\/yUjXX8Ig40x6oEiB4EeuA\\\\\\\/XZpS4RtL0rWyGEV031rW+BwVhC5YmVWyx1j3SjIObWUJ48tEuttnF0v56ygpgYeFGWT\\\\\\\/zpsXbi+uacFqqaHNepprbl5\\\\\\\/vuaw6eXwvHdzLt7tqOK5IjsgghIAqJ+WVHXdsPuD3sp+E03aXJss5AVi3WVPGo8NUH27crHBK7O+jU3SjJJZcuHDCwAHsSfVvazy845p\\\\\\\/zA4KgwFqh4tjMSQUpSIccaUzeymnuGgA84Aexo+iwMFyM9XbX\\\\\\\/2dge5dIeBl+ni51aPz8IezBQXosMerLWMoP6vTwncURy97PTUMGxKtsLxdrUoVMM6ivqxk5rN7wWFoFxSsjPlbpDkw7LeMn9q10ZqAXPmVjebxPxoh4VFZHrxO5+f2znN+bnVrDKmo3cVV1sOq2DqldTyBHRJXZLtBjj34FrVwib9gljMH47nmQeRdfzHvC5Dsv1wvmRx8yCuFs05riMJx9QC\\\\\\\/Z8vesRUbgmfyGbBHWFtBr5sGa2TxIkT2BGR6aIqFH\\\\\\\/1mMu51VqfM4vMPqTFZlzEzaOX6b9ljq4bjN7PyoiWD\\\\\\\/RwZStrDwxVmkv5UoTajel5RgbwkQ67rpUg+5n4X26YqP2BpdWxBBwFkVeZi8T5kmoKpZN8L3pW2qjkN7NTOGVMFKviavT7KiNSYhGw0OyA2OGBOaxvCKG1urZuolEsm0PGk0zvSW484bklmUTip6XKWYD6azJ6\\\\\\\/jYqQFx2JEStw4EZRSdxW3iulyrf+jzlho2jaHJXcSgOurQkZpIgB5GU2xNuOvRtfAd\\\\\\\/xLr5Auli8MOFV+We+Mc6CXyHLAWZUqy0DId9idNtpxLSX6Ev2YXlr+\\\\\\\/si+98UaPsgwukQVQ7oyv1ytwBwNcrBWjCgmwrtmoazFjFTkUi9xE5jwTJSCqRZV2HbzUblFbHVRs87CeEd8do9ZQdHekaItfHstw0I9qyC18njCMQr\\\\\\\/olS5tm+bGBm\\\\\\\/sLLUFT5QcNLKJCwFoBs9fKxJTGiZOBZIhvr\\\\\\\/qJhjOtrBfziHqExEbbBFYxVTc71XQod1mKI1ZpztTHuX5AWTx26u04vNCL3kf3VrYf0qAoEXRqXTok7t0OAZnRFnLXQgG7QaUtGc3Dx5pfgiAcJkYPK3Sryiyob51zVMumxAwy84\\\\\\\/TwXGukx2SqKCDPTt5YNewMVQKYXNF2kN+tSS4DF\\\\\\\/ZitFcACQk\\\\\\\/KwitaouKMRAa3HG6V9wekz+L6cmnQO8cPmg6MzaArpGQPvQzxmZ8ZDN0Sfsihmi9r8GWboJjJFfZ2J9UdqjTfze+H5nrCJSenKAtzVssNEgVWTWVb7+Mvi\\\\\\\/tVMKEusRCzXHm\\\\\\\/B6kInLRRTaSK4ixiiJ8cXudE9s3MhmzpMLSbBmTjB8EZbY6XKo5SQ6+b86OLjgOSCsoB7GfSw0D8zaMYkcLgX4Mg7MRY5lxTom0ZqMk+KCgsLuqze8JUMDeKnPEDCunkmqDOn7kirm9RMvJPxTMCGJWiraSk+W512g6ve4NYfTYQ+HJYKJIu66Xd+PDZwSKXKdVRI1wKZVOjS4c2uawKEP0fxjW7seau6PYICOqdz0UVdp7Sw6JNIgoz189oL2ZwILXMPxS878KppNVK8c3CvCBaTRLV\\\\\\\/w6vljCbt29L\\\\\\\/gcIv5ICJ7f9y8XmSoBowg5\\\\\\\/r+nlPrkQ3cKtpHM\\\\\\\/W8pYD\\\\\\\/Vb7vYYBNP3IMakeO+JDeTWKMWnEhsEublhbLHZhqyUP58g7gPixicpgOuUHWa9U\\\\\\\/yyD60RkKA+soQHh9khxrjWJbA70E6uL5f8ZS\\\\\\\/y4ugpdryh4FhWSuH503LUjXapp1CNT2PBjbCqRh\\\\\\\/7n\\\\\\\/dMxey2g08LZ\\\\\\\/3uJ5oqat1cMMrruWlGGHcAz\\\\\\\/8MoW0EW9lxWWSSu3hfm9pREMtRIJs3Rce8oXMUbCg3VWAZW63sHhdOnRisnKLnHG0+YsMeOCykckF29HYyA50PBsRi6cHEjSBfUCzkfpnduCeR1LUGZ+dSZUBNH+d1WWghQ5lVNRaMtYwflUPZthRFjHWZC+NrJFcCKtce24BopAlEbLVeKPSXziZ3Vcn6naAh47T9IC7+pARxufqlc9o8ZX7LI4HV\\\\\\\/K5qfkhJ8NsWg3tc4WD4LK6y4dt7mB9CowbwCCyDOTi8by52IKv\\\\\\\/8saDdbv1oTDsWV7M6jMvl+8uwjxGTAmUqxm5eLlLdWq\\\\\\\/XMEhtn2QL4N33wEtz46FnBp9+IaREBfHoK3ZFyJpC7cVcI7\\\\\\\/Kaj5wK3f\\\\\\\/pWHbtVvbRIm61qwpAjUM6n8phdLmGDhJ4McNpFjbroyxwjooz+Tq0NkIZ84rLXlgaoNLrdIa2xoNBJflyJCEV\\\\\\\/qVR3NYGsNF4fWbLKPOFTv6nLsBMYxgNwtUg+kSunoSlSFA9Zw8ylkB3cCOf5h7qOORjSAuqvHuzfLu+4ttBkRWq5F\\\\\\\/g+n3+pUeIjiHWqWYRash87joyxA36Wgc9jLKKen0m5qulFESPCQco15H\\\\\\\/DWx3Tw9DMCceoUSwSC433VaWeboSE86HGSOmHETUPr3enJWDod+2ZXDX47MG9jVt+AhR+nFOkHAbICZpqACnjzeJlphjsQwyjuzVgt0WQfQTR1PBdbyp8eAcO215cgANqNWtvOUiR5lU5t76xM9d9a9toRgEijA4CYdIvwf3vsBJS+7FvMYJCwl1EALJxCihBlVRBStXzEpeeAqSbrxgJVoXLUwXaVEjxMgPl+hItxaAmuGz+M\\\\\\\/eFmyND9UNxDN6k0L2Tk0OPpLH\\\\\\\/J9mky2Aufn1S6fXbdGdU0PZh3Hp1kBKMhaJ85hFg\\\\\\\/f9B\\\\\\\/VhB44opc3pFoqDlbkD+SRi6KqgFvy5zi4\\\\\\\/s6XGBgb9o9OO1XuXYQZ7irq\\\\\\\/BiC6YTcoplDgbxEgB7f0\\\\\\\/u3z1Ra0H95thZNpcbbM7rFrzryI2H2GK1AP574bCAllwQV5BTiX9KauhotH8kDqMVp7P34ck6KzoRCxjAFZQTOeANtpU+p3\\\\\\\/lOhr+zxPpj1oM5O792oVOlvgwvs3kgFYqAS0UkDjQtsRM5\\\\\\\/iECXIwCvJPtRPb9LuVSKSZ1Q7ZXlTE40era0INVuEDWd0btikPAEfkINHx1cupB\\\\\\\/YODLd\\\\\\\/G9DPYMDEbLAQcY0JkUxppdLdNFH0Ck8ZsDRX8Zn9qtxXhUUKTGKnJ1JpGGw349OnNeNQ23viTYWarUP5OMR4XqtpZ6rpvEEb\\\\\\\/ewnV11YCzFAcNmAMM0YfUTh+yn34+pyjohQZok1euibQrSYUV8vMIloiNiboPuSrgDsn+01l8eU5U7ckapgSZa8IaLr7NrrTKgCkguw7hEKYnEKgncrMNhVLzJg1R80pEhb8wuX3ILRNx2Ok0po8Bo+f\\\\\\\/wsWOybS2MxMPbKIP5+GMCf79oxFhId3JRvTUtxcheSMP0rEaq7fCKtFPr8fZixnvw2oUEdvRxhdYxJP5fLCGAg0ZmOZxGG5cl8+8uxK3K4iT2wI\\\\\\\/AMebJCXvoW4a8TowPu1XTEOL6UKh1KobOB0zeaEL0EmGoeXCWu++Pg3J2rctYbj2gkrdEPTRDtSJfNp7jMOf\\\\\\\/wUpuTUZ0FtA9a8X+8mQzUQKUxC90cm7r1UrYW9\\\\\\\/812MKqcgjXILEePuDaR9NoHwo3EeztWqcETPXGkpfQnltNsVw8ZPWBa8Q3XRHNx+ofQmNjO9CndIzPTXsJNkRSHBlD3D7HPTBALS1AQhS9uyF6mUPYiIt2CBTDRHXTHygrQeWohCMcpNO4Nxg3EoBN02apNWJ3nerBHq1vTtKz8bYp3IVHvL6ya2pLUWCyX+nWSPoVLmerp4pJxh0mlLN9gx58tBmnlNv4AXA1\\\\\\\/M3LnQdM0GgKrErGNHjb4apcn8TfoXiw11v+5RNdMbe6I5AnKGRKwlTsqk9vKMirXsHcsaHBuQ6Fu7UnYeCCvvvBXJDozXOszUnpZJYcwufWkqTBWQFU3i8JBUzoZQ\\\\\\\/oeyiaGbKxBILkwH8PwRlxXI1BK4Cyc2u\\\\\\\/OxmYpf7tCsLTUVzK0TM9U5CbptmU38qDTHBuDYuT4taxSqCDy6MCwwxFndAXrdfloquBQFNZDK5Q1CidE+zpyhZJv8hlpjrrlB0hLNVdMSi+acXe444HSdG6nN33HoRYCvx1bhCN1OFmBXuUAU\\\\\\\/N4BnDuPt1g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022naW1H9w+dB4AginnRS3Leg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002247074a594efa7b1d102e70b2f6a196276d4b080860e98c17ee54ffc82dc0b2d7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eCYei6oLSnAy5rsTJPXoHOv6tHh7clkWGJkMNkffPFzx5jHOhceJvi35d9722+aM8yjHyQoXB+z+vNN94fwd9cr0hl+aaDoPHVY+JuKNdP77IRcUQQR\\\\\\\/lZeTSesNt+PFl36JeEOR4aBxLiyaYcl94wNQ53\\\\\\\/v+twbVrGXmgqqff\\\\\\\/GR6r9ObiEF9tV9oPMg+LHw8RuIAIRzRjpc0v68SUHDYVL5ngr2pXPBuVIR2Cr7kU7hgLw4wOKpC9q2P5f44eVCleS5+l4+QUrWNXeiCO4QjjByWumLm5W4T\\\\\\\/Qxe9a7rRxazwBYfghEOarRq2fuYisathmxyfQztRHbR7k+ChKslHD1GwinRfTkjGhS4rKNt1JPk4Cwb9SAwmXoIV48dC5GCxJjVzJ8b7KO8q4N\\\\\\\/ZVNvlo7dogNTnnLzk9Lu9HqzOI0Nc4WBwuHrVmUU+oJRmsI4jmoe2GCu5bKGEFWae3dWDcvTaL1OCrplCCU7YWkRunNfKxDO0WAVuLAw0Yx2LlgnwEx\\\\\\\/jPMlK0SIdT0\\\\\\\/Y6pcRiBCrn\\\\\\\/FTvGFpEKJ7e2VAXzHxQ07QwKB1Va7iy73rNf+gVfDh\\\\\\\/2FwaILeE7mGcadaeyrg6kPTPX3f2m56bVD82wBS5xOmxSAMqJEPHPM+IZ\\\\\\\/ypPnypjN5VHj2NcPcO9rs00\\\\\\\/+PnGkdM+mwRyMHRFh0OhqtpVFCDyQZ9w5mTx6H+rwRvHrBRzpBlrcDLeNDqGGjZEmLis0I\\\\\\\/\\\\\\\/TfxtpVSS3wPlYSArzgMN1Up2xqzlCtTh8QUnwquXZziU3IsNJ\\\\\\\/HaPm5N+tdYao9OEsK1p+QFFKNtrQbMqYR6G4xt3gC77eGnl0sd0trf0FOEAoNSOyAGBTrv2ixt29VuuuQoBsifYP9Ly9+cAQS7MAmtnCEJ3M2ClOF38HfbucZo8SAKvJpArVizWvdTaBv425PjndSPxYofJNkqHlwgXvQDqpxZqKNPTn\\\\\\\/csLDIiErLG5ENx+eit8xdGKv2+zsY1d+WeLtTJmRC0Yg8ujd\\\\\\\/RGW4ehtWe8yF9lSKelLEapEo1sT8S6IwEVeF+Cdl6sJD\\\\\\\/zS2K+3XWyBphVWIeAXf6MSyLXQ6id4Q4w8Q4F9qVauKjmg9TFrjFoQP4fdiA83VHD68N9scI5AeK5N7oRs6FRmOf5vWW9xesn7I4YiDtzN+iS+3ZOeQmIesgCWLu8aBMLE6ryjfposcUPxyE6NOgQnQGL909Jpc+OCC2iDYa5CcVFkYDP89Eneneizhh12c\\\\\\\/xA8K0eAX4LNvwPZmDRiYUskS2GhoscF5riPG9aEzc\\\\\\\/VrtLBvH54nrGdWmof8l7uHaSquLq+ms79e\\\\\\\/alO1BjC9gHoxgkKrNzzRdfYxvrBDpYymiw94c+XtI5uv9IzautrNn5PlgC\\\\\\\/QBaD89jTlzpBYv738NnrJDkk\\\\\\\/QMMHBlm2qf0wJhF5wemglmsUXCNFCaLnMrSQodRAGVixQFwmwUyQ07FGZC0tdW\\\\\\\/KKf4Jpp6ejYftDJQBcaCTRv2mzn8f\\\\\\\/25WsrlD9mfOfgkgg8qoXGIWuf034t8Em576qmp\\\\\\\/jlINGFi6hKEXA8GsN2OVP\\\\\\\/IaSoigd1d1h4W09Mnw3nFFuuKaoe40rqrlK7irIo8jbiStC7VOuMB\\\\\\\/0o3xpta4k2qErj3yQML9RKbJ8hpABuofA1AoR6EvSSM2q3babfEi\\\\\\\/DSJxwgde0bWQ7NIYEfDbutzPWloHe1P0\\\\\\\/D+8VHskfRHotEaqRNnQ5SuwQvjBabrkWPlTdvwf5cA4\\\\\\\/qvHWWJPxP4GwoP1nfZT9YxXg9A591uIgf1a3VCLNRTzqjjMHnE3I4M5h\\\\\\\/aQkEkMvqInWgCVzQ+EioVgdpoGykHnFeifzouv0ua4cMqO7FSlRbOvSRqZVz7hnEIRvdiQgX7ahaQjMCCc4hfhTcZEy8kpM5KtX6tt6H9HFZqtqRe9CmOdvTAAOZP6u4skB3FZBcjZl7reY3JYXAOhbFRRVwdXNq1eg6eB6hoBFJs69X3fbXgTnrdUilzFBVwX8DjyQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wJm\\\\\\\/LAbb0PbXjDiPSKzV9w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002238da208c38dc21ef484f245af8dc2d27b54a13c3d3c66009496a9e45b2a76453\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459747 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2505" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nWXcCkNXL9aqcf545RH2Bm3PRJzQ0rfqjDQnFuIPDOE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2505" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ieWiXyHCD8Hs4nABIS2bAe_71OM8DZsOzgUEw9Pj08I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/12/2493" title="A Shigella flexneri Virulence Plasmid Encoded Factor Controls Production of Outer Membrane Vesicles" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/12/2519" title="Quantitative Trait Locus Mapping of Melanization in the Plant Pathogenic Fungus Zymoseptoria tritici" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459747" id="node459747" data-pisa="ggg;4/12/2505" data-pisa-master="ggg;g3.114.014274" data-apath="/ggg/4/12/2505.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Spatial Localization of Recent Ancestors for Admixed Individuals</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Wen-Yun Yang</span>, <span class="highwire-citation-author" data-delta="1">Alexander Platt</span>, <span class="highwire-citation-author" data-delta="2">Charleston Wen-Kai Chiang</span>, <span class="highwire-citation-author" data-delta="3">Eleazar Eskin</span>, <span class="highwire-citation-author" data-delta="4">John Novembre</span> and <span class="highwire-citation-author" data-delta="5">Bogdan Pasaniuc</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2505-2518;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014274 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459747" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Wen-Yun Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wen-Yun%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWen-Yun%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander Platt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BPlatt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platt%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BPlatt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charleston Wen-Kai Chiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charleston%2BWen-Kai%2BChiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiang%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharleston%2BWen-Kai%2BChiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Eleazar Eskin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Human Genetics, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleazar%2BEskin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskin%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEleazar%2BEskin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John Novembre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Human Genetics, University of Chicago, Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BNovembre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Novembre%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BNovembre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Bogdan Pasaniuc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Human Genetics, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Pathology and Laboratory Medicine, Geffen School of Medicine at University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bogdan%2BPasaniuc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pasaniuc%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABogdan%2BPasaniuc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bpasaniuc@mednet.ucla.edu" class="" data-icon-position="" data-hide-link-title="0">bpasaniuc@mednet.ucla.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/12/2505" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459747" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459747/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2505.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459747" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459747/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2505.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459747" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459747/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/12/2505.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459747" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459747/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/12/2505.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459747" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459747/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Ancestry analysis from genetic data plays a critical role in studies of human disease and evolution. Recent work has introduced explicit models for the geographic distribution of genetic variation and has shown that such explicit models yield superior accuracy in ancestry inference over nonmodel-based methods. Here we extend such work to introduce a method that models admixture between ancestors from multiple sources across a geographic continuum. We devise efficient algorithms based on hidden Markov models to localize on a map the recent ancestors (<em>e.g.</em>, grandparents) of admixed individuals, joint with assigning ancestry at each locus in the genome. We validate our methods by using empirical data from individuals with mixed European ancestry from the Population Reference Sample study and show that our approach is able to localize their recent ancestors within an average of 470 km of the reported locations of their grandparents. Furthermore, simulations from real Population Reference Sample genotype data show that our method attains high accuracy in localizing recent ancestors of admixed individuals in Europe (an average of 550 km from their true location for localization of two ancestries in Europe, four generations ago). We explore the limits of ancestry localization under our approach and find that performance decreases as the number of distinct ancestries and generations since admixture increases. Finally, we build a map of expected localization accuracy across admixed individuals according to the location of origin within Europe of their ancestors.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-variation" class="hw-term hw-article-keyword hw-article-keyword-genetic-variation" rel="nofollow">genetic variation</a></li><li class="kwd"><a href="/keyword/genetic-continuum" class="hw-term hw-article-keyword hw-article-keyword-genetic-continuum" rel="nofollow">genetic continuum</a></li><li class="kwd"><a href="/keyword/admixture" class="hw-term hw-article-keyword hw-article-keyword-admixture" rel="nofollow">admixture</a></li><li class="kwd"><a href="/keyword/localization" class="hw-term hw-article-keyword hw-article-keyword-localization" rel="nofollow">localization</a></li><li class="kwd"><a href="/keyword/ancestry-inference" class="hw-term hw-article-keyword hw-article-keyword-ancestry-inference" rel="nofollow">ancestry inference</a></li></ul><p id="p-3">Inference of ancestry from genetic data is a critical aspect of genetic studies, with applications ranging from controlling stratification in disease mapping to the inference of population history (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rosenberg <em>et al.</em> 2003</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Novembre <em>et al.</em> 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Drineas <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Price <em>et al.</em> 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rosenberg <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Seldin <em>et al.</em> 2011</a>). Although many initial large-scale genetic association studies have focused primarily on homogeneous populations, increasingly studies are addressing samples in which individuals have more complex backgrounds, including admixed ancestry (<em>i.e.</em>, emerging from the mixing of genetically diverged ancestors; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bryc <em>et al.</em> 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hinch <em>et al.</em> 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">N’diaye <em>et al.</em> 2011</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wegmann <em>et al.</em> 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jarvis <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Perera <em>et al.</em> 2013</a>). Such studies depend crucially on accurate and unbiased ancestry inference both at a genome-wide level as well as at each locus in the genome (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Seldin <em>et al.</em> 2011</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pasaniuc <em>et al.</em> 2013</a>).</p><p id="p-4">Traditional ancestry inference from genetic data has been focused on modeling populations as discrete units. As a result, traditional genome-wide ancestry inference estimates the proportion of sites in the genome coming from a set of source populations (continental or subcontinental), and locus-specific inference aims to assign each allele in the genome to one of the considered populations [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Falush <em>et al.</em> 2003</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Price <em>et al.</em> 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Shringarpure and Xing, 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baran <em>et al.</em> 2012</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Pasaniuc <em>et al.</em> 2013</a>). More recently, alternative approaches model population structure in a geographic continuum, capitalizing on the correlation of genetics and geography expected in isolation by distance models (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Price <em>et al.</em> 2006</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wasser <em>et al.</em> 2007</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2012</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baran <em>et al.</em>, 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Elhaik <em>et al.</em> 2014</a>) and observed in many organisms (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Guillot <em>et al.</em> 2009</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Storfer <em>et al.</em> 2010</a>). Spatial assignment offers three (related) advantages beyond simple population assignment. First, it appropriately acknowledges that nature rarely provides neat boundaries between distinct populations of exchangeable individuals. Second, it allows for model-based inference to exploit the geographic structure of allele frequencies for increased power, and third allows for the accurate assignment of ancestors in otherwise un-sampled or under-sampled regions.</p><p id="p-5">Spatial analysis of genetic data often are performed through principal components analysis (PCA) (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Price <em>et al.</em> 2006</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Seldin <em>et al.</em> 2006</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Paschou <em>et al.</em> 2007</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Wasser <em>et al.</em> 2007</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Novembre <em>et al.</em> 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Paschou <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">McVean 2009</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zakharia <em>et al.</em> 2009</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brisbin <em>et al.</em> 2012</a>), a general procedure for reducing the dimensionality of the data, whereas alternative approaches focus on explicit modeling of the relationship between patterns of genetic variation and geography (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wasser <em>et al.</em> 2004</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2012</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Baran <em>et al.</em> 2013</a>). These approaches typically assume that an individual’s genotype is drawn from the genetic variation present at a single geographic location, or (as in <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Brisbin <em>et al.</em> 2012</a>) assume that ancestral locations are specified <em>a priori</em> and then assign individual loci accordingly. These assumptions are clearly violated when individuals have ancestors from multiple unknown geographic regions, as occurs with recently admixed populations in the Americas (such as African-Americans) and more generally, individuals who have ancestry from multiple regions within the same continent (<em>e.g.</em>, individuals with recent ancestors from multiple regions of Europe). Recent works have circumvented this issue by first inferring segments of different continental ancestry (<em>i.e.</em>, locus-specific) followed by independent application of PCA only on segments of specific continental ancestry (<em>e.g.</em>, European segments; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Johnson <em>et al.</em> 2011</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Moreno-Estrada <em>et al.</em> 2013</a>). A critical component of such an approach is the performance of locus-specific ancestry inference, which has been shown to attain high accuracy for continental ancestries but to be less accurate in inferring subcontinental ancestry (<em>e.g.</em>, country of origin; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Price <em>et al.</em> 2009</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Baran <em>et al.</em> 2012</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Maples <em>et al.</em> 2013</a>). Other approaches to address admixed individuals have only considered the limited case of a first-generation admixed individual (<em>i.e.</em>, one parent from each of two locations), in which case local ancestry need not be inferred as individuals are heterozygous at all loci for both ancestries (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Wasser <em>et al.</em> 2004</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2012</a>).</p><p id="p-6">In this work, we introduce models of admixture across varying number of generations and ancestries in a geographic continuum. We model admixed genomes as having recent ancestors from several locations on a genetic-geographical map. We perform ancestry inference by simultaneously localizing on the map the recent ancestors of an admixed individual and partitioning the admixed genome into segments inherited from the same ancestor (<em>i.e.</em>, locus-specific ancestry). Assigning a small number of ancestors helps maintain computationally tractability and is biologically meaningful for various important use cases. Where admixture in the recent past is suspected, there will be a particular genomic signature of repeated observations of the same ancestral locations across many unlinked portions of the genome. Modeled this way, we both uncover the nature of the admixture events as well as combine the signal from each of these loci for increased accuracy of the locations of each contributing lineage. We also take advantage of the simple observation that if one allele is inherited from a specific ancestor, then most likely the neighboring alleles are also inherited from the same ancestor. Specifically, we use a model-based framework for genetic variation in the geographical continuum (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2012</a>) and use hidden Markov modeling (HMM) of the admixture process (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Patterson <em>et al.</em> 2004</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gravel, 2012</a>). We develop efficient optimization algorithms that allow us to accurately predict the geographic location of the recent ancestors of an admixed individual in conjunction with locus-specific ancestry inference. The results allow the localization on a geographical map of each allele in recently admixed individuals.</p><p id="p-7">We use empirical genotype data from the Population Reference Sample (POPRES) project (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Nelson <em>et al.</em> 2008</a>) to validate our approach. The POPRES project has genotyped more than 3000 individuals with ancestry distributed throughout Europe and has recorded the self-reported ancestry (typically at the level of country) for both individuals and their parents/grandparents. We use 1385 POPRES individuals with homogeneous ancestry (<em>i.e.</em>, all reported grandparents having the same ancestry) to infer patterns of variation across geography in Europe (<a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2012</a>) and use our method to localize the recent ancestors of individuals with self-reported admixed ancestry (<em>i.e.</em>, grandparents with multiple ancestries in Europe). Our method is able to localize the grandparents of the admixed individuals in POPRES data within an average of 470 km of their reported ancestry. The accuracy is dependent on the specific ancestries and ranges from 305 km for individuals with Swiss and French ancestry to 701 km for those with Spanish and Portuguese ancestry. We use simulations from the POPRES genotype data to show that the localization accuracy within Europe decreases with increased number of ancestors and with the number of generations since the admixture. We also show that inference accuracy (at the genome-wide and locus-specific level) increases as distance among ancestors increases. Finally, we provide an analysis of ancestry localization error across all pairs of countries in Europe as resource for the community interested in subcontinental ancestry in Europe.</p><p id="p-8">With the growing availability of high-quality reference panels for nonhuman and nonmodel systems, we expect our proposed methods to be broadly applicable to other organisms as well. Particularly, it could be used to identify the contributing strains to novel hybridized plants, to identify origins of recombinant human pathogens, to identify colonization origins of invasive species, and to investigate environmental refugia/dispersal sources in general source-sink dispersal systems. A software package implementing our methods is freely available at <a href="http://bogdan.bioinformatics.ucla.edu/software/">http://bogdan.bioinformatics.ucla.edu/software/</a>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Overview of spatial localization for admixed individuals</h3><p id="p-9">In this work, we consider models of ancestry for admixed individuals in a geographical continuum. We view the mixed ancestry genome as being generated from several geographical locations on a map, corresponding to the locations of their recent ancestors (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). For example, consider the case of an individual with one maternal grandparent from Italy and the other one from Great Britain (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). The maternal copy of its genome will be composed of segments originating from the two locations in Europe (see <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Each position in the genome has its own function that describes the population allele frequencies at that site as a function of geography. The approach we take follows spatial ancestry analysis (SPA) (<a id="xref-ref-43-6" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2012</a>) and assumes these functions take on logistic gradient shapes. Some variants may have steep gradients (<em>i.e.</em>, frequencies that change drastically with location) whereas other variants may not vary at all with geography (see <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). Although these types of functions clearly do not explain all correlation between genetics and geography, it has been previously shown that when there are basic isolation-by-distance patterns, such simple functions carry sufficient information to be very informative of ancestry status across individuals (<a id="xref-ref-43-7" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2012</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Baran <em>et al.</em> 2013</a>) and lead to likelihood functions that are simple to optimize. Other types of functions such as linear functions (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Baran <em>et al.</em> 2013</a>) or more complex functions could also be employed in our framework.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SPAMIX model for admixed individuals. (A) Example of haploid individual with two ancestry locations in Europe (circles denote the true ancestry locations). (B) The admixture process induces segments of different ancestry backgrounds. (C) SPAMIX uses logistic gradients to describe allele frequencies as a function of geographic map to instantiate an admixture hidden Markov modeling for each pair of locations on a map. Each location on the map is associated to a particular allele frequency at all sites in the genome. (D) SPAMIX finds the location of ancestors on a map (denoted by squares in A) and the locus-specific ancestry at each site in the genome by maximizing the likelihood of genotype data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1706163742" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;SPAMIX model for admixed individuals. (A) Example of haploid individual with two ancestry locations in Europe (circles denote the true ancestry locations). (B) The admixture process induces segments of different ancestry backgrounds. (C) SPAMIX uses logistic gradients to describe allele frequencies as a function of geographic map to instantiate an admixture hidden Markov modeling for each pair of locations on a map. Each location on the map is associated to a particular allele frequency at all sites in the genome. (D) SPAMIX finds the location of ancestors on a map (denoted by squares in A) and the locus-specific ancestry at each site in the genome by maximizing the likelihood of genotype data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2505/F1.medium.gif" width="440" height="384"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/12/2505/F1.medium.gif" width="440" height="384"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465401" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">SPAMIX model for admixed individuals. (A) Example of haploid individual with two ancestry locations in Europe (circles denote the true ancestry locations). (B) The admixture process induces segments of different ancestry backgrounds. (C) SPAMIX uses logistic gradients to describe allele frequencies as a function of geographic map to instantiate an admixture hidden Markov modeling for each pair of locations on a map. Each location on the map is associated to a particular allele frequency at all sites in the genome. (D) SPAMIX finds the location of ancestors on a map (denoted by squares in A) and the locus-specific ancestry at each site in the genome by maximizing the likelihood of genotype data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Having estimated gradient functions at each site in the genome, we extend standard HMMs for admixture to incorporate variation at each position on the map by allowing the emission probabilities to vary according to these gradients (see <em>Spatial modeling of allele frequency</em>). We perform inference in this model to find the ancestor locations on the map that maximize the likelihood of the observed genome (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>). After finding the location of the recent ancestors, we assign each allele in the mixed genome to one of the ancestor locations. This provides a locus-specific ancestry call across the genome. <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a> shows an output of our method (SPAMIX) with locations in the admixed genome being labeled according to the inferred ancestral location on the map.</p><p id="p-12">Although we presented a simple example of our framework with two ancestors on the map, the model flexibly handles diploid data with arbitrary number of generations and ancestors localized on the map (<em>e.g.</em>, diploid genome with four ancestors localized on the map two generations ago, diploid genome with eight ancestor locations three generations ago) (see <em>Diploid data with admixed ancestry</em>). As the number of generations since admixture increases, the total number of ancestors to localize increases dramatically, making the inference problem very challenging. To account for this effect we limit the number of different locations for the recent ancestors for the maternal (paternal) haplotypes to a fixed constant (<em>e.g.</em>, <em>M</em>(<em>N</em>), see below) with varying amount of contributions to ancestry (see below). We devise efficient algorithms to jointly optimize the locations of the ancestors as well as the proportion they contribute to the genome of the admixed individual (see below). For example, in the case of three generations ago, one ancestry location may contribute 1/8th to the admixed genome if only one ancestor comes from that location and may contribute 1/2 to the admixture process if half the ancestors come from that specific location. Finally, we note that the diploid model is symmetric making <em>M</em> and <em>N</em> interchangeable.</p></div><div id="sec-3" class="subsection"><h3>Spatial modeling of allele frequency</h3><p id="p-13">Although our base method for explicit modeling of genetic variation as function of geography has been described elsewhere (<a id="xref-ref-43-8" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2012</a>; <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Baran et al. 2013</a>), we briefly present here the generative model. We view each of the alleles of an individual as an independent Bernoulli draw from an allele frequency that changes across the map and we parametrize the allele frequency function through a logistic gradient as a function of position <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-1.gif"/></noscript></span></span> in the map. Formally, the probability of observing the reference allele in single-nucleotide polymorphism (SNP) <em>j</em> at position <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-2.gif"/></noscript></span></span> on the map <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-3.gif"/></noscript></span></span>, is defined as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-4.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-5.gif"/></noscript></span></span> and <em>b<sub>j</sub></em> are parameters specific to SNP <em>j</em>. We estimate <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-6.gif"/></noscript></span></span> and <em>b<sub>j</sub></em> from data containing individuals with known homogeneous locations (<a id="xref-ref-43-9" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2012</a>) and then use these coefficients in the inference of ancestries of mixed individuals.</p><p id="p-14">Although easy to manipulate mathematically, the logistic functions we use here clearly do not capture all genetic variation (for example, variants that have multiple modes or peaks in the allele frequency surface as may be typical of rare variants). However, these functions have been shown to capture general trends in common-variant frequencies sufficiently well enough in isolation-by-distance samples to produce highly accurate spatial assignment in individuals with nonmixed ancestry (<a id="xref-ref-43-10" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2012</a>). We hypothesize that such simple-to-manipulate functions are sufficient for accurate localization of recent ancestors in individuals with mixed subcontinental ancestries.</p></div><div id="sec-4" class="subsection"><h3>Haploid data with admixed ancestry</h3><div id="sec-5" class="subsection"><h4>Spatial model for admixed haploid data:</h4><p id="p-15">For simplicity, we start by introducing the model for haploid data and extend it to genotype data in the next section. Denote by <em>h</em> = (<em>h</em><sub>1</sub>, …, <em>h<sub>L</sub></em>) the multisite haplotype of an admixed haplotype, where <em>L</em> is the number of SNPs typed across the genome and <em>h<sub>i</sub></em> ∈ {0, 1} encodes the number of reference alleles at SNP <em>i</em>. Due to the admixture process, the haplotype can be viewed as a mosaic of segments coming from ancestors from multiple locations on the map. We define variables <em>Z</em> = (<em>z</em><sub>1</sub>, …, <em>z<sub>L</sub></em>) as indicators for an allele coming from ancestry location <em>j</em> (<em>z<sub>i</sub></em> = <em>j</em> if allele at locus <em>i</em> is from <em>j</em>-th ancestry location) and write the likelihood of the haplotype data as function of ancestry locations <em>X</em>. The likelihood for a given admixed haplotype data having <em>M</em> ancestry locations <em>X</em> = (<em>x</em><sub>1</sub>, …, <em>x<sub>M</sub></em>) where each ancestry contributes proportionally with Π = (<em>π</em><sub>1</sub>, …, <em>π<sub>M</sub></em>) is defined as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-7.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>The hidden variable <em>Z</em> encodes the mosaic structure of the admixed haplotype (<em>i.e.</em>, inheritance within the past generations for recent admixture, admixture-linkage disequilibrium [LD]) and can be modeled using a Markov chain as follows:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-8.gif"/></noscript></span></span>where the parameters <em>τ</em> = {<em>τ</em><sub>1</sub>, …, <em>τ<sub>L</sub></em><sub>−1</sub>} stand for the recombination probability (within the past <em>g</em> generations) between each two neighbor loci. The alleles present at a site <em>i</em> on a haplotype is modeled as a Bernoulli variable with a success probability given by the allele frequency <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-9.gif"/></noscript></span></span> as follows:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-10.gif"/></noscript></span></span>where the parameters <em>a</em> and <em>b</em> are estimated beforehand. Thus, they should be regarded as fixed parameters in this article. An illustration of the model is given in <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>. We note that our model makes the assumptions of independence of alleles conditional on local ancestry (no modeling of background LD or deviations from Hardy-Weinberg proportions).</p></div><div id="sec-6" class="subsection"><h4>Spatial ancestry inference for haploid data:</h4><p id="p-16">Under the generative model shown previously, spatial ancestry inference is reduced to inferring the <em>M</em> ancestral locations given data for an admixed haplotype, followed by posterior decoding in the HMM to obtain locus-specific predictions. This can be achieved by maximizing the likelihood function (2) with respect to <em>X</em>. By treating <em>X</em> as parameters and <em>Z</em>, Π as hidden variables, this maximization falls within the procedure of the standard expectation-maximization (EM) algorithm (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dempster <em>et al.</em> 1977</a>):</p><div id="sec-7" class="subsection"><h5>E step:</h5><p id="p-17">The expectation step is similar to the forward-backward algorithm for HMM, which calculates the posterior probability of hidden variables <em>Z</em> given current estimation of ancestral locations <em>X</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup> and ancestral proportion Π<sup>(</sup><em><sup>t</sup></em><sup>)</sup>:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-11.gif"/></noscript></span></span>where <em>α</em>/<em>β</em> are standard forward/backward HMM functions and can be efficiently calculated (see Supplementary Note).</p></div><div id="sec-8" class="subsection"><h5>M step:</h5><p id="p-18">The maximization step alternatively optimizes the Q functions in <em>X</em> and in Π (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Dempster <em>et al.</em> 1977</a>). In detail, it first optimizes the Q function in <em>X</em> by fixing Π. Second, it optimizes the Q function in Π by fixing <em>X</em>. These two steps are performed alternatively until the maximization converges. The Q function in <em>X</em> in the first step can be derived as<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-12.gif"/></noscript></span></span>where <em>C<sub>ij</sub></em> denotes the posterior <em>P</em>(<em>z<sub>i</sub></em> = <em>j</em>|<em>h</em>, <em>X</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup>, Π<sup>(</sup><em><sup>t</sup></em><sup>)</sup>) from the <em>E step</em>, and the shorthand <em>q<sub>i</sub></em>(<em>x<sub>j</sub></em>) is defined as:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-13.gif"/></noscript></span></span>The Q function in Π in the second step can be derived as follows<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-14.gif"/></noscript></span></span>where <em>D<sub>ij</sub></em> and <em>E<sub>ij</sub></em> denote constants calculated from the posterior <em>P</em>(<em>z<sub>i</sub></em> = <em>j</em>|<em>h</em>, <em>X</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup>, Π<sup>(</sup><em><sup>t</sup></em><sup>)</sup>) in the <em>E step</em>.</p><p id="p-19">We perform the maximization by taking advantage of the convex properties of the equation and using analytical forms for the Hessian of the function. The complete derivations are given in <a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/014274SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/FileS1.pdf">File S1</a>.</p><p id="p-20">To give an overview of the whole EM algorithm, an illustration of the aforementioned EM algorithm is given in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An illustration of the expectation-maximization (EM) algorithm for spatial ancestry inference for haploid data. The E-step and M-step are performed alternatively until the EM algorithm converges. The last M ancestral locations are used as the output of EM algorithm. SNPs, single-nucleotide polymorphisms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1706163742" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;An illustration of the expectation-maximization (EM) algorithm for spatial ancestry inference for haploid data. The E-step and M-step are performed alternatively until the EM algorithm converges. The last &lt;em&gt;M&lt;/em&gt; ancestral locations are used as the output of EM algorithm. SNPs, single-nucleotide polymorphisms.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2505/F2.medium.gif" width="440" height="339"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/12/2505/F2.medium.gif" width="440" height="339"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465410" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">An illustration of the expectation-maximization (EM) algorithm for spatial ancestry inference for haploid data. The E-step and M-step are performed alternatively until the EM algorithm converges. The last <em>M</em> ancestral locations are used as the output of EM algorithm. SNPs, single-nucleotide polymorphisms.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-9" class="subsection"><h4>Locus-specific spatial ancestral inference for haploid data:</h4><p id="p-22">Having obtained the maximum likelihood geographical locations <em>X</em><sup>∗</sup>, we compute the posterior probability for <em>Z</em>, which leads to a locus specific assignment of ancestry at each allele in the genome. The most probable local ancestral locations are found by maximizing<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-15.gif"/></noscript></span></span>which can be efficiently solved by the Viterbi algorithm (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Viterbi 2006</a>). To compute a posterior probability of each locus-specific ancestry, we use the forward-backward algorithm (see <a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/FileS1.pdf">File S1</a>).</p></div></div><div id="sec-10" class="subsection"><h3>Diploid data with admixed ancestry</h3><div id="sec-11" class="subsection"><h4>Spatial model for admixed diploid data:</h4><p id="p-23">We next extend the haploid model to genotypes by considering <em>M</em> paternal ancestry locations <em>X</em> = (<em>x</em><sub>1</sub>, …, <em>x<sub>M</sub></em>) with proportions Π = (<em>π</em><sub>1</sub>, …, <em>π<sub>M</sub></em>) and <em>N</em> maternal ancestry locations <em>Y</em> = (<em>y</em><sub>1</sub>, …, <em>y<sub>N</sub></em>) with proportions Ω = (<em>ω</em><sub>1</sub>, …, <em>ω<sub>N</sub></em>). Denote by <em>g</em> = (<em>g</em><sub>1</sub>, …, <em>g<sub>L</sub></em>) the multisite genotype of an admixed genotype, where <em>L</em> is the number of SNPs typed across the genome and <em>g<sub>i</sub></em> ∈ {0, 1, 2} encodes the number of reference alleles at SNP <em>i</em>. Then the likelihood becomes:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-16.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>The variables <em>Z<sup>p</sup></em> and <em>Z<sup>m</sup></em> now encode the ancestry status of the paternal (maternal) alleles (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-17.gif"/></noscript></span></span> denotes that the paternal allele at locus <em>i</em> is from <em>j</em>-th paternal ancestry) and can be modeled through the same Markovian process as:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-18.gif"/></noscript></span></span>Given the origin of alleles, the likelihood of the admixed individual genotype is modeled as two Bernoulli draws:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-19.gif"/></noscript></span></span>The function <em>f<sub>i</sub></em> is the allele frequency function in logistic form (1). The probability <em>P</em>(<em>Z</em>) models the recombination events in paternal and maternal ancestries, and the probability <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-20.gif"/></noscript></span></span> models the probability of generating the genotype from two ancestral geographical locations.</p></div><div id="sec-12" class="subsection"><h4>Spatial ancestry inference for diploid data:</h4><p id="p-24">We would like to infer <em>M</em> + <em>N</em> ancestral locations for a given mixed individual genotype. This can be achieved by maximizing the likelihood function (3) with respect to <em>X</em> and <em>Y</em>, which, analogous to the haploid case, can be performed using the EM algorithm (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Dempster <em>et al.</em> 1977</a>).</p><div id="sec-13" class="subsection"><h5>E step:</h5><p id="p-25">In short, the expectation step is similar with forward-backward algorithm in HMM, which calculates the posterior probability of hidden variables <em>Z</em> given current estimation of ancestral locations <em>X</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup> and <em>Y</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup>.<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-21.gif"/></noscript></span></span>where <em>α</em> and <em>β</em> can be calculated recursively using a procedure similar to the forward-backward algorithm for HMMs.</p></div><div id="sec-14" class="subsection"><h5>M step:</h5><p id="p-26">The maximization step alternatively optimizes the Q functions in <em>X</em>, <em>Y</em> Π and Ω. In detail, it first optimizes the Q function in <em>X</em> and <em>Y</em> by fixing Π and Ω. Second, it optimizes the Q function in Π by fixing <em>X</em> and <em>Y</em> and Ω. Third, it optimizes the Q function in Ω by fixing <em>X</em>, <em>Y</em> and Π. These two steps are performed alternatively until the maximization converges. The Q function in <em>X</em> and <em>Y</em> in the first step can be derived as follows<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-22.gif"/></noscript></span></span>where <em>C<sub>ijk</sub></em> denotes the posterior <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-23.gif"/></noscript></span></span> computed from <em>E step</em>, and the shorthand <em>q<sub>i</sub></em>(<em>x<sub>j</sub></em>, <em>y<sub>k</sub></em>) is defined as:<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-24.gif"/></noscript></span></span>The Q function in Π in the second step can be derived as<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-25.gif"/></noscript></span></span>where <em>D<sub>ij</sub></em> and <em>E<sub>ij</sub></em> denote constants calculated from the posterior <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-26.gif"/></noscript></span></span> in the <em>E step</em>. We omit the Q function in Ω in the third step here as it is very similar with the aforementioned function.</p><p id="p-27">As in the haploid case, we leverage the convexity of the function and analytical forms for the Hessian to efficiently optimize the Q function. The complete derivations and optimization details are given in <a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/FileS1.pdf">File S1</a>. As noted in <a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/FileS1.pdf">File S1</a>, the Q function <em>Q</em>(<em>X</em>, <em>Y</em>; <em>X</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup>, <em>Y</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup>, Π<sup>(</sup><em><sup>t</sup></em><sup>)</sup>, Ω<sup>(</sup><em><sup>t</sup></em><sup>)</sup>) in the first step is not concave in general. However, we can still use convex optimization techniques to get a local optimal solution. In practice, we observe that the function is concave almost all the time.</p></div></div><div id="sec-15" class="subsection"><h4>Locus-specific spatial ancestral inference for diploid data:</h4><p id="p-28">Having obtained the maximum likelihood geographical locations <em>X</em><sup>∗</sup> and <em>Y</em><sup>∗</sup> for each ancestry, we can compute the posterior probability for <em>Z<sup>p</sup></em> and <em>Z<sup>m</sup></em>, which leads to the spatial local ancestry inference. The most probable local ancestral states are obtained by maximizing<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/12/2505/embed/mml-math-27.gif"/></noscript></span></span>which can be efficiently solved by the Viterbi algorithm (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Viterbi, 2006</a>). The posterior of local ancestries for each allele can be obtained using a forward-backward algorithm following the <em>E step</em> in the algorithm (see Supplementary Note).</p></div><div id="sec-16" class="subsection"><h4>Homogeneous paternal and maternal ancestries:</h4><p id="p-29">In the aforementioned notations we derived the general solution that allows for paternal and maternal ancestries to be different from each other, which is suitable for applications of inference of parental locations or grandparent locations. A simplifying case is when maternal and paternal ancestries are homogeneous, <em>i.e.</em>, the paternal haplotype and maternal haplotype are from the same set of ancestral populations. We allow for this case by setting <em>M</em> = <em>N</em> and enforce a constraint <em>x<sub>j</sub></em> = <em>y<sub>j</sub></em> in the M step.</p></div></div><div id="sec-17" class="subsection"><h3>POPRES data set</h3><p id="p-30">We applied our methods to a data set collected from European populations, which was assembled and genotyped as part of the larger POPRES project (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Nelson <em>et al.</em> 2008</a>) and accessed via dbGAP accession number phs000145.v4.p2. A total of 3192 European individuals were genotyped at 500,568 loci using the Affymetrix 500K SNP chip. The same stringency criteria as in <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Novembre <em>et al.</em> (2008)</a> were applied to create the training data. We removed SNPs with low confidence scores and low call-rate (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Nelson <em>et al.</em> 2008</a>; <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Novembre <em>et al.</em> 2008</a>). We filtered individuals to avoid sampling individuals from outside of Europe, to create more even sample sizes across Europe, and to remove individuals whose self-reported data have grandparents with different origins. We note that this is the same sample set used in <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Novembre <em>et al.</em> (2008)</a> and <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Baran <em>et al.</em> (2013)</a>. For the remaining individuals who have reported grandparental data, we use that origin for the individual. Otherwise, we use the individual-level self-reported country of birth. From these data, we infer logistic gradients starting from genotype data from 447,245 autosomal loci in 1385 individuals from 36 populations. A total of 77.4% of SNPs are common SNPs (allele frequency &gt; 0.05), and the remaining 22.6% have low frequencies (&lt; 0.05). As in <a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Novembre <em>et al.</em> (2008)</a>, we use country geographical center as the geographical locations for all the individuals from that country population. For testing, we identified an additional 470 individuals from the POPRES data that have self-reported grandparental ancestry from two or more countries in Europe. A summary of homogeneous ancestry individuals used in estimating logistic gradients (1385) and with subcontinental European admixed ancestry (470) are given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465358/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465358/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Self-reported grandparental ancestry (location of origin) of the POPRES data individuals (1906 in total)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Background LD</h3><p id="p-32">Although our approach models admixture LD, it assumes that markers are independent conditional on local ancestry (no background LD). Preliminary results (not shown) that used transition rates based on the assumed number of generations and recombination rate (<em>i.e.</em>, similar to simulations, (<em>g</em> − 1)<em>φ</em>(<em>d<sub>i</sub></em><sub>+1</sub> − <em>d<sub>i</sub></em>) where <em>d<sub>i</sub></em>′s are the locations of each SNPs, <em>g</em> denotes the number of generations, and <em>φ</em> is the probability of one recombination per generation per base-pair (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Paşaniuc <em>et al.</em> 2009</a>) resulted in an increased number of inferred short ancestry windows which led to decreased accuracy. This effect is likely due to lack of modeling of background LD in the model. To remove short ancestry windows (likely spurious, induced by residual LD) we first performed LD pruning at a level of <em>r</em><sup>2</sup> &lt; 0.2 (72,418 SNPs retained) followed by adjustment of the transition rates in our model by a factor of 10<sup>−2</sup>. This adjustment factor is used to regularize the recombination rate, such that the number of inferred short ancestry windows is appropriate for the number of generations and distinct ancestries and will vary with these parameters. Results at different LD pruning levels and the corresponding adjustment factors are reported in <a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/TableS2.pdf">Table S2</a>.</p></div><div id="sec-19" class="subsection"><h3>Simulation setup</h3><p id="p-33">We use BEAGLE to phase the POPRES data then simulate offspring admixed individuals by modeling recombinations within the last couple of generations. The recombination probability between each SNPs is approximated as (<em>g</em> − 1)<em>φ</em>(<em>d<sub>i</sub></em><sub>+1</sub> − <em>d<sub>i</sub></em>) where <em>d<sub>i</sub></em>′s are the locations of each SNP in base pair, <em>g</em> denotes the number of generations, and <em>φ</em> is the probability of one recombination per generation per base pair (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Paşaniuc <em>et al.</em> 2009</a>). For the recombination map, we assumed a flat recombination rate of <em>φ</em> = 10<sup>−8</sup> per base pair. For given number of <em>M</em> paternal ancestries and <em>N</em> maternal ancestries, we randomly select from the POPRES data a set of <em>M</em> + <em>N</em> individuals, each of which has four grandparents from the same location and randomly select one haplotype from each individual. We simulate the admixed haplotypes independently for the maternal and paternal haplotypes using the standard Poission process of admixture block distribution (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Price <em>et al.</em> 2009</a>). If specified as homogeneous paternal and maternal ancestries, we pick <em>M</em> instead of <em>M</em> + <em>N</em> ancestries and use the same <em>M</em> ancestries for both paternal and maternal haplotype simulation.</p><p id="p-34">For the SPAMIX haploid model, the simulated haplotypes are used as input directly. Also, we always use the correct number of ancestries <em>M</em> or <em>N</em> as input. For the SPAMIX diploid model, the combined genotype from two simulated paternal and maternal haplotypes are used as input. To avoid testing bias, we estimate the allele frequency logistic gradients each time using the POPRES individuals with the <em>M</em> + <em>N</em> simulation ancestors excluded from the training set. We do not optimize over the ancestry proportions but provide them as input to SPAMIX. The ancestry proportions are fixed as uniformly distributed among all ancestors.</p><p id="p-35">We use several metrics to assess performance of SPAMIX in simulations and real data. For the ancestral location prediction, we evaluate the results by computing the average geographical distance between predicted locations and true locations in simulations (<em>prediction error</em>). To account for the distance among ancestries, we also compute the <em>relative prediction error</em>, defined as the ancestral location prediction error divided by the distance between the true ancestry locations used in simulations. Note that we use as the “true” ancestral locations for the admixed individual the set of country centers from the <em>M</em> + <em>N</em> ancestries.</p><p id="p-36">For locus-specific inference, we propose two different metrics. The first one is the <em>local ancestry prediction error</em>, which is the average distance between predicted location and true location at each locus. The second metric we use is the <em>local ancestry prediction accuracy</em>, defined as the percentage of loci across the genome with correct assignment of ancestry. To account for the ambiguity in matching the true to inferred ancestries, we permute the inferred ancestries to find the closest match in terms of inferred location to true location.</p></div></div><div class="section results" id="sec-20"><h2 class="">Results</h2><div id="sec-21" class="subsection"><h3>Performance of continuous ancestry inference in simulations</h3><p id="p-37">We investigated the performance of our model through simulations from the POPRES data (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Nelson <em>et al.</em> 2008</a>). The POPRES data measures genome-wide genetic variation in a large number of individuals with ancestries across Europe (with a larger proportion of individuals with ancestry from the Central and Western Europe). For each individual, the self-reported ancestry (typically at the level of country) of parents and grandparents was tabulated. To produce a large number of admixed individuals on which to test the data, we first randomly selected individuals with homogenous ancestry (<em>i.e.</em>, all four grandparents from the same country of origin) from various areas in Europe to serve as “ancestor” genomes. We used them to simulate an admixed individual and attempted to recover the original ancestral locations using the simulated genome and a set of logistic gradients inferred from the remaining unused individuals (see the section <em>Materials and Methods</em>). SPAMIX attains an ancestry localization accuracy (<em>i.e.</em>, average distance between true and inferred locations of the recent ancestors) for individuals with two recent ancestors in Europe of 550 km (see <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). There is a large variance (334 km) across different sets of ancestral pairs, showing the high variability in performance across subjects. Contributing to this effect is the variable sampling density of the POPRES data (which is denser toward the center of Europe) and variability of goodness-of-fit of the logistic gradients across the map (<em>e.g.</em>, poorer fit at range boundaries).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465314/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465314/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Average distance between inferred and true ancestry locations in simulated admixed individuals from POPRES data</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">To test how much is gained by explicit modeling of correlations among SNPs induced by segments of recent shared ancestry (admixture LD), we also inferred the recent ancestry location using a naïve model that assumes all SNPs to be independent (as in <a id="xref-ref-43-11" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2012</a>). It can also be understood as SPAMIX with completely random transition probability, which is equivalent to independent SNP assumption. We observe a significant increase in the average distance between true and inferred locations in this naïve model (880 km) <em>vs.</em> the 550 km for SPAMIX, thus showing that modeling admixture LD significantly increases performance. We also quantified the effect of correlations among markers conditional on local ancestry (background LD) in our approach. Eliminating loci found in strong disequilibrium with each other (LD pruning) was observed to increase accuracy even though the model had less data to use (see <a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/TableS2.pdf">Table S2</a>); therefore, all results in the main text are obtained after LD pruning (<em>r</em><sup>2</sup> &lt; 0.2, see the section <em>Materials and Methods</em>).</p><p id="p-40">It is increasingly often the case that access to pedigree data allows haplotypes to be determined with high accuracy. Therefore, we quantified the gain in ancestry localization accuracy arising from having access to phased haplotype data (<em>i.e.</em>, haploid data) compared with unphased diploid data. <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> shows that accurate phasing significantly increases localization accuracy. For example, having access to perfect phasing allows for the inference of the four ancestral locations (two ancestors for each haplotype) within 557 km of the simulated location where the diploid model for four ancestral locations attains an average of 639 km of its simulated locations.</p><p id="p-41">An important parameter of our model is the number of generations since admixture; with more generations, more recombination events have the opportunity to shuffle ancestry across the genome thus reducing the average length of the ancestry segments. We observe a slight decrease in performance from two to eight generations (548 to 562 km), which we expect to continue as the number of generations increases (in the limit of extremely large number of generations, our model is equivalent to the naïve model that does not model admixture-LD; see <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>), since the recombination rate will be large enough to produce random switches.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465343/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465343/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Average distance between inferred and true ancestry locations in simulated admixed individuals from POPRES data as function of number of generations in the mixture process</span><div class="sb-div caption-clear"></div></div></div><p id="p-43">Our framework models genetic variation as function of geography by assuming a logistic gradient for the spatial distribution of genetic variations (see the section <em>Materials and Methods</em>). That is, the frequency of a given variant is allowed to change in a given direction on a map only according to a parametrized logistic function. Although this approach has been shown to provide a good approximation of common variation that leads to accurate ancestry inference, we hypothesize that the error in fitting logistic gradients to real data limits the method’s accuracy. To assess this scenario, instead of using real individual’s haplotype data, we simulated admixed haplotypes directly from the logistic gradients we inferred from POPRES data (see the section <em>Materials and Methods</em>). We observe a large increase in accuracy in this idealized scenario as compared to simulations from real haplotype data (<em>e.g.</em>, 236 <em>vs.</em> 550 km for two ancestries four generations ago, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), thus indicating that logistic gradients do not account for all the correlation between geography and genetic variation. This suggests that further work on functions linking geography to genetics within our framework may yield additional improvements (see the section <em>Discussion</em>).</p><p id="p-44">We investigated the performance of our approach as we increase the number of ancestral locations (<em>M</em>/<em>N</em>, see the section <em>Materials and Methods</em>) to estimate for a given admixed individual. For a fixed number of generations (four), we varied the number of ancestry locations to estimate. The parental inference is different from two ancestry inference, as the parental inference assumes that one haplotype is from paternal ancestry and one from maternal ancestry. However, the two-ancestry inference assumes that both of the haplotypes are mosaic of two ancestries (<em>M</em> = <em>N</em> = 2). As expected, we observe decreases in performance as the number of ancestral locations increases. For example, the average prediction error increases from 550 for two ancestries to 639 km for four ancestral locations (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>).</p></div><div id="sec-22" class="subsection"><h3>Increased distance between ancestral locations improves performance</h3><p id="p-45">It is well known that accuracy of ancestry inference correlates with genetic distance between ancestral populations. Discrete local ancestry can be inferred with very high degree of accuracy in mixtures of highly diverged populations (<em>e.g.</em>, African Americans) compared with closely related ones (<em>e.g.</em>, subcontinental mixtures) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Basu <em>et al.</em> 2008</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Paşaniuc <em>et al.</em> 2009</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Price <em>et al.</em> 2009</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Maples <em>et al.</em> 2013</a>). Because geography correlates with genetic distance, we hypothesized that the accuracy of continuous ancestry inference in recently admixed individuals also correlates with distance among ancestries on the map. Indeed, we observe that the relative prediction error (<em>i.e.</em>, the difference between predicted and true locations normalized by the distance between the true ancestry locations, see the section <em>Materials and Methods</em>) decreases with the distance between ancestries in Europe (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). For example, if the ancestries are 500 km apart, we observe a relative prediction error of 0.75 compared with 0.50 when the ancestries are located 2000 km apart. Interestingly, when not normalizing for the distance between ancestries (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>), we observe that prediction error increases with increased distance. This shows that although the task of separating the ancestry locations becomes simpler, the localization accuracy becomes poorer (<em>e.g.</em>, two ancestors located 500 km apart are localized within 450 km of their true locations, whereas two ancestors located 3000 km apart are localized within 1000 km of their true locations). This effect is presumably due to assignment errors in the local ancestry that have a much bigger impact if the ancestral locations are further apart. Although fewer local ancestry errors are being made with increased distance (see <em>Locus-specific inference</em>), these errors have a stronger impact on the ancestral localization due to their higher distance to the true location.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ancestral location prediction error as a function of distance between ancestral locations in simulations over Population Reference Sample data. Left, the prediction error normalized by the distance between the ancestral locations used in simulations; right, plot of the prediction error. Simulations use the haploid model with two generations in the mixture." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1706163742" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Ancestral location prediction error as a function of distance between ancestral locations in simulations over Population Reference Sample data. Left, the prediction error normalized by the distance between the ancestral locations used in simulations; right, plot of the prediction error. Simulations use the haploid model with two generations in the mixture.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2505/F3.medium.gif" width="440" height="169"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/12/2505/F3.medium.gif" width="440" height="169"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465366" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-46" class="first-child">Ancestral location prediction error as a function of distance between ancestral locations in simulations over Population Reference Sample data. Left, the prediction error normalized by the distance between the ancestral locations used in simulations; right, plot of the prediction error. Simulations use the haploid model with two generations in the mixture.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>Inference of number of ancestors</h3><p id="p-47">In the aforementioned simulations, we have assumed that the true number of different ancestry locations is known. We investigated whether our approach can also be used to predict the number of distinct ancestries on the map for a given genome. We used the standard Akaike information criterion (AIC; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bozdogan 1987</a>) that balances the goodness of fit with the number of parameters in the model (more ancestries to infer increases the number of parameters in our method). Starting from POPRES individuals with homogeneous ancestry we simulated admixed individuals with up to 4 ancestry locations 4 generations ago under the constraint that the ancestries are at least 600 kilometers apart. For each simulated admixed individual, we ran our method SPAMIX using <em>N</em> = 1, 2, 3, 4 ancestry locations and used the AIC to infer the number of ancestors. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows that this procedure will on average estimate the number of ancestries correctly, but the error rate is expected to be high for any single case of inference.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inference of number of distinct ancestries using the Akaike information criterion (AIC). We simulated 1000 admixed individuals with up to four distinct ancestry sources in Europe and used the AIC within the SPAMIX model to infer the number of ancestries. (A&#x2212;D) Proportion of inferred number of ancestries (y-axis) as function of number of simulated ancestries (x-axis). Although we observed a large variance in the number of predicted ancestries, we note that the histogram is centered on the correct simulated number of ancestries, thus suggesting that AIC could be used to infer the number of distinct ancestors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1706163742" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Inference of number of distinct ancestries using the Akaike information criterion (AIC). We simulated 1000 admixed individuals with up to four distinct ancestry sources in Europe and used the AIC within the SPAMIX model to infer the number of ancestries. (A&#x2212;D) Proportion of inferred number of ancestries (y-axis) as function of number of simulated ancestries (x-axis). Although we observed a large variance in the number of predicted ancestries, we note that the histogram is centered on the correct simulated number of ancestries, thus suggesting that AIC could be used to infer the number of distinct ancestors.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2505/F4.medium.gif" width="440" height="396"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/12/2505/F4.medium.gif" width="440" height="396"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465432" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-48" class="first-child">Inference of number of distinct ancestries using the Akaike information criterion (AIC). We simulated 1000 admixed individuals with up to four distinct ancestry sources in Europe and used the AIC within the SPAMIX model to infer the number of ancestries. (A−D) Proportion of inferred number of ancestries (y-axis) as function of number of simulated ancestries (x-axis). Although we observed a large variance in the number of predicted ancestries, we note that the histogram is centered on the correct simulated number of ancestries, thus suggesting that AIC could be used to infer the number of distinct ancestors.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h3>Locus-specific inference</h3><p id="p-49">An advantage of our framework is that in addition to identifying the most likely locations of the recent ancestry of admixed individuals, it can also provide an assignment of each allele in the genome to each ancestry location. We observe that local ancestry prediction accuracy (<em>i.e.</em>, the proportion of alleles assigned to the correct ancestry, see the section <em>Materials and Methods</em>) increases with the distance between ancestral locations (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) from 55% of loci assigned accurately for very closely related ancestries (less than 500 km apart) to more than 70% for ancestries 2500 km apart (<a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/FigureS1.pdf">Figure S1</a>). Similar to the ancestor localization, we observe that although the total number of assignment errors is reduced with increased distance, these errors have a bigger impact when averaging across all sites to compute the average allele localization error. Therefore, we observe that the average local ancestry prediction error is increased as distance between ancestral locations is increased.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SPAMIX locus-specific ancestry prediction accuracy as function of distance between ancestral locations. Left, local ancestry prediction accuracy, defined as the percentage of all loci with correct assignment of ancestry. Right, average distance to true locations for each allele in the genome (local ancestry prediction error). Simulations use the haploid model with two generations in the mixture." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1706163742" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;SPAMIX locus-specific ancestry prediction accuracy as function of distance between ancestral locations. Left, local ancestry prediction accuracy, defined as the percentage of all loci with correct assignment of ancestry. Right, average distance to true locations for each allele in the genome (local ancestry prediction error). Simulations use the haploid model with two generations in the mixture.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2505/F5.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/12/2505/F5.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465354" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-50" class="first-child">SPAMIX locus-specific ancestry prediction accuracy as function of distance between ancestral locations. Left, local ancestry prediction accuracy, defined as the percentage of all loci with correct assignment of ancestry. Right, average distance to true locations for each allele in the genome (local ancestry prediction error). Simulations use the haploid model with two generations in the mixture.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h3>Map of accuracy across Europe</h3><p id="p-51">We also investigated the variance in performance according to the ancestor’s labeled origin (<em>i.e.</em>, typically to level of country). <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the prediction error for admixed individuals with ancestry from pairs of origins in Europe. In general, we observe decreased performance for populations at the boundary of the European map (<em>e.g.</em>, Portugal, Spain, Italy), and increased performance for subcontinental admixtures from populations located geographically in the center of Europe (<em>e.g.</em>, France, Switzerland) (<a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/FigureS2.pdf">Figure S2</a> and <a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/FigureS3.pdf">Figure S3</a>). This can be an effect of biased sampling in the POPRES data, that sampled more individuals from European center, but also can be an effect of having more information to localize individuals in SPAMIX. In general, we observe a prediction accuracy ranging from 411 km for admixtures from Spain and Italy to 641 km for individuals with recent ancestors from Spain and the United Kingdom.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ancestral location prediction error in simulations of European individuals with ancestry from two locations in Europe, stratified by the country of origin of each location (the country of origin is displayed in different colors). The assumed true locations are displayed by shaded circles. Results in parenthesis denote the average ancestral location prediction error across all simulations. In each simulation the reference data (used to estimate logistic gradients) is disjoint from data used to simulate admixed genomes (see the section Materials and Methods). The admixed genome is simulated as four generations ago, and SPAMIX diploid model is used for the inference. The number of simulated pairs can be found in Figure S3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1706163742" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ancestral location prediction error in simulations of European individuals with ancestry from two locations in Europe, stratified by the country of origin of each location (the country of origin is displayed in different colors). The assumed true locations are displayed by shaded circles. Results in parenthesis denote the average ancestral location prediction error across all simulations. In each simulation the reference data (used to estimate logistic gradients) is disjoint from data used to simulate admixed genomes (see the section &lt;em&gt;Materials and Methods&lt;/em&gt;). The admixed genome is simulated as four generations ago, and SPAMIX diploid model is used for the inference. The number of simulated pairs can be found in Figure S3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2505/F6.medium.gif" width="373" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/12/2505/F6.medium.gif" width="373" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465414" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-52" class="first-child">Ancestral location prediction error in simulations of European individuals with ancestry from two locations in Europe, stratified by the country of origin of each location (the country of origin is displayed in different colors). The assumed true locations are displayed by shaded circles. Results in parenthesis denote the average ancestral location prediction error across all simulations. In each simulation the reference data (used to estimate logistic gradients) is disjoint from data used to simulate admixed genomes (see the section <em>Materials and Methods</em>). The admixed genome is simulated as four generations ago, and SPAMIX diploid model is used for the inference. The number of simulated pairs can be found in <a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/FigureS3.pdf">Figure S3</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>Analysis of real admixed individuals from POPRES data</h3><p id="p-53">Finally, we investigated whether high accuracies observed in simulations also can be attained in real data. Using SPAMIX, we localized the recent ancestry of all admixed European individuals from POPRES (see the section <em>Materials and Methods</em>). A total of 470 admixed individuals were analyzed using SPAMIX (see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). As “ground truth” ancestral locations, we used the center of the self-reported grandparent country of origin. Therefore, we assume the mixed individuals from POPRES have two to four ancestry locations to infer. Across all 470 individuals, we observe an average prediction error distance of 470 km which decreases to 426 if outlier individuals defined as those with prediction errors greater than 1000 km are removed (all such outlier individuals are reported in <a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/TableS3.pdf">Table S3</a>). The error distance is lower than simulated experiments likely due to the large proportion of the admixed individuals of French and Swiss ancestries, which can be accurately localized (average of 305 km). As discussed previously, we note that SPAMIX ancestor localization performance varies greatly across Europe with ancestors from pairs of countries localized at the boundary of European map being harder to localize (<em>e.g.</em>, an average of 701 km for ancestor localization for Spanish/Italian mixed individuals).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ancestral location prediction error in real POPRES admixed individuals, stratified by the country of origin of each location. Letters are the inferred locations, and the shaded circles are the assumed true locations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1706163742" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Ancestral location prediction error in real POPRES admixed individuals, stratified by the country of origin of each location. Letters are the inferred locations, and the shaded circles are the assumed true locations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/12/2505/F7.medium.gif" width="440" height="381"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/12/2505/F7.medium.gif" width="440" height="381"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/12/2505/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465400" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-54" class="first-child">Ancestral location prediction error in real POPRES admixed individuals, stratified by the country of origin of each location. Letters are the inferred locations, and the shaded circles are the assumed true locations.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-27"><h2 class="">Discussion</h2><p id="p-55">We have introduced new models for predicting the geographical origins of multiple recent ancestors for individuals with recent mixed ancestry. Existing methods for local ancestry inference in admixed populations either focus on discrete ancestry assignment or use locus-specific ancestry inference followed by PCA on subsets of the data. We introduce models that leverage the spatial structure of genetic variation using HMMs for the admixture process to achieve high accuracy in localizing the recent ancestry of a given individual on a geographical map. Our proposed model can be viewed as a generalization of the parental localization model proposed in <a id="xref-ref-43-12" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> (2012)</a> to account for admixture-LD while allowing for multiple generations and ancestries. Our algorithm is very efficient. For the scale of 100,000 SNPs and four ancestral locations, computation is typically 10 min and uses less than 100 Mb of memory.</p><p id="p-56">Although in our framework we use standard logistic gradient functions that were previously used to link geography and genetic variation, it is worth mentioning that such functions do not capture the whole variability observed in empirical data. To that extent, introducing more flexibility in these functions within the framework for admixture we described here is more likely to provide considerable improvements in accuracy with a tradeoff of computational time. We view this as a promising direction for future study. This is especially important for handling sequencing data, as rare variants rarely are fit well by the gradient functions (results not shown). For example, we are working on using a logistic quadratic function to model the spatial genetic structure, which is a generalization of the original SPA model. It will further enhance the admixed ancestor inference.</p><p id="p-57">Another area for further developments is extending the framework to model background LD (correlations among variants on the same ancestral backgrounds). We found it necessary to modify the transition rates used in our inference by a multiplicative factor based on the level of LD pruning applied to the SNP list (<a href="http://www.g3journal.org/content/suppl/2014/11/03/g3.114.014274.DC1/TableS2.pdf">Table S2</a>). Such LD adjustments have proved fruitful in improving localization accuracy for unadmixed individuals (<a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Baran <em>et al.</em> 2013</a>) and are likely to improve inference for admixed individuals as well. Although we leave this for future work, one potential approach would be to perform inference within short windows (to account for the local structure of LD) and merge the information within each window into the overall likelihood.</p><p id="p-58">We also note that we used a simplified model of ancestry switching along chromosomes that approximates the pedigree structure. In effect, our approach assumes a fixed effective time-scale of admixture, and ignores the structured transition matrices that are expected due to a fixed pedigree. Future work that explicitly considers the pedigree structure could allow one to address questions regarding the specific timing and configuration of admixed ancestries. For example, for a mixed individual with one Italian and three British grandparents, we could incorporate the specific inheritance pattern in the HMM transition rates. The question of whether the ancestors themselves were admixed could be investigated by assigning local ancestry followed by analyzing the length distribution of the ancestry blocks, and we leave this as future work.</p><p id="p-59">For most of the results presented, we assumed that the number of ancestral locations is known. In practice, such information will often not be available. To address this we developed a procedure for inferring the number of ancestries using AIC. In <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> we showed that although on average the correct number of ancestries will be inferred, in a high proportion of cases the inferred number of ancestries will be mistaken. This type of model selection problem is akin to estimating K in the admixture model of STRUCTURE/admixture (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Falush <em>et al.</em> 2003</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>) and is typically challenging. In future work, pedigree-based models should lead to constraints on the possible observations that make the number of ancestors more straightforward to infer.</p><p id="p-60">In this work, we focus on the prediction of ancestral locations using an EM algorithm, which is a deterministic method to produce point estimates of the parameters of interest (geographic origins of ancestors) and missing data (the local ancestry of each allele copy). Alternative inference approaches can be taken, for example, the likelihoods we define in Equations (2) and (3) could be used in a Bayesian Markov Chain Monte Carlo approach method to sample from the posterior distribution of the spatial prediction (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Wasser <em>et al.</em> 2007</a>). In such an approach, an efficient starting point would be from the point estimate obtained via the EM algorithm, which could significantly expedite the convergence of the Markov Chain Monte Carlo approach.</p><p id="p-61">Our work provides a framework of predicting ancestral locations for admixed individuals which can be further improved. For example, due to the continuous nature of the approach, ancestral locations can be predicted to be outside of standard geographic boundaries (<em>e.g.</em>, ocean). Future work could improve on our framework by providing constraints to the optimization procedure or by a postoptimization adjustment (selecting the closest location that fits geographical boundaries). In addition, the underlying SPA approach works best in the simple isolation-by-distance settings and is not expected to work well in for complex scenarios with barriers or irregular geometries. We leave a full investigation of such modifications of our framework as ongoing work and recommend the use of cross-validation approaches with individuals of known ancestry to assess the suitability of the model prior to application.</p><p id="p-62">A direct benefit of the proposed model is that it leads to efficient optimization procedures for tackling inference problems such as localization of ancestors in the genetic-geographic map. In this work we have presented such an algorithm based on the well-known EM procedure, which leverages the HMM of the admixture process joint with the gradient representation of genetic variation as function of geography.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-63">This work is supported in part by the National Institutes of Health (R03-CA162200, R01-GM053275). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-64">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.014274/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.014274/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-65"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-07-07"><span class="received-label">Received </span>July 7, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-10-27"><span class="accepted-label">Accepted </span>October 27, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Yang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.1" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.2" data-doi="10.1093/bioinformatics/bts144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baran</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torgerson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast and accurate inference of local ancestry in Latino populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1359</span>–<span class="cit-lpage">1367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts144%26rft_id%253Dinfo%253Apmid%252F22495753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTAvMTM1OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.3" data-doi="10.1016/j.ajhg.2013.04.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baran</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quintela</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carracedo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halperin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Enhanced localization of genetic samples through linkage-disequilibrium correction.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">882</span>–<span class="cit-lpage">894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D92%26rft.spage%253D882%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.04.023%26rft_id%253Dinfo%253Apmid%252F23726367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.04.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23726367&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.4" data-doi="10.1007/s00439-008-0541-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide distribution of ancestry in Mexican Americans.</span> <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DBasu%26rft.auinit1%253DA.%26rft.volume%253D124%26rft.issue%253D3%26rft.spage%253D207%26rft.epage%253D214%26rft.atitle%253DGenome-wide%2Bdistribution%2Bof%2Bancestry%2Bin%2BMexican%2BAmericans.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-008-0541-5%26rft_id%253Dinfo%253Apmid%252F18752003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-008-0541-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18752003&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259909500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.5" data-doi="10.1007/BF02294361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bozdogan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Model selection and Akaike’s information criterion (AIC): The general theory and its analytical extensions.</span> <abbr class="cit-jnl-abbrev">Psychometrika</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPsychometrika%26rft.volume%253D52%26rft.spage%253D345%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02294361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02294361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1987K344100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.6" data-doi="10.3378/027.084.0401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brisbin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrnes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakharia</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Omberg</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">PCAdmix: principal components-based assignment of ancestry along each chromosome in individuals with admixed ancestry from two or more populations.</span> <abbr class="cit-jnl-abbrev">Hum. Biol.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BBiol.%26rft.volume%253D84%26rft.spage%253D343%26rft_id%253Dinfo%253Adoi%252F10.3378%252F027.084.0401%26rft_id%253Dinfo%253Apmid%252F23249312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3378/027.084.0401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23249312&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313648400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.7" data-doi="10.1073/pnas.0914618107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karafet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Estrada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide patterns of population structure and admixture among Hispanic/Latino populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">8954</span>–<span class="cit-lpage">8961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0914618107%26rft_id%253Dinfo%253Apmid%252F20445096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoyMToiMTA3L1N1cHBsZW1lbnRfMi84OTU0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dempster</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laird</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Maximum likelihood from incomplete data via the EM algorithm.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D39%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.9" data-doi="10.1371/journal.pone.0011892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drineas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paschou</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Inferring geographic coordinates of origin for Europeans using small panels of ancestry informative markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e11892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDrineas%26rft.auinit1%253DP.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253De11892%26rft.epage%253De11892%26rft.atitle%253DInferring%2Bgeographic%2Bcoordinates%2Bof%2Borigin%2Bfor%2BEuropeans%2Busing%2Bsmall%2Bpanels%2Bof%2Bancestry%2Binformative%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0011892%26rft_id%253Dinfo%253Apmid%252F20805874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0011892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20805874&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elhaik</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatarinova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chebotarev</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piras</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calò</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Geographic population structure analysis of worldwide human populations infers their biogeographical origins.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3513%26rft_id%253Dinfo%253Apmid%252F24781250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24781250&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1567</span>–<span class="cit-lpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1567%26rft.epage%253D1587%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%253A%2BLinked%2BLoci%2Band%2BCorrelated%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1NjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI1MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.12" data-doi="10.1534/genetics.112.139808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genetics models of local ancestry.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">607</span>–<span class="cit-lpage">619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139808%26rft_id%253Dinfo%253Apmid%252F22491189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzIvNjA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.13" data-doi="10.1111/j.1365-294X.2009.04410.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guillot</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leblois</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frantz</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Statistical methods in spatial genetics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">4734</span>–<span class="cit-lpage">4756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DGuillot%26rft.auinit1%253DG.%26rft.volume%253D18%26rft.issue%253D23%26rft.spage%253D4734%26rft.epage%253D4756%26rft.atitle%253DStatistical%2Bmethods%2Bin%2Bspatial%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2009.04410.x%26rft_id%253Dinfo%253Apmid%252F19878454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2009.04410.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19878454&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271904400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.14" data-doi="10.1038/nature10336"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tandon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohland</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The landscape of recombination in African Americans.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">476</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHinch%26rft.auinit1%253DA.%2BG.%26rft.volume%253D476%26rft.issue%253D7359%26rft.spage%253D170%26rft.epage%253D175%26rft.atitle%253DThe%2Blandscape%2Bof%2Brecombination%2Bin%2BAfrican%2BAmericans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10336%26rft_id%253Dinfo%253Apmid%252F21775986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10336&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21775986&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293731900028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.15" data-doi="10.1371/journal.pgen.1002641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheinfeldt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Omberg</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Patterns of ancestry, signatures of natural selection, and genetic association with stature in Western African pygmies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DJarvis%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253De1002641%26rft.epage%253De1002641%26rft.atitle%253DPatterns%2Bof%2Bancestry%252C%2Bsignatures%2Bof%2Bnatural%2Bselection%252C%2Band%2Bgenetic%2Bassociation%2Bwith%2Bstature%2Bin%2BWestern%2BAfrican%2Bpygmies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002641%26rft_id%253Dinfo%253Apmid%252F22570615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22570615&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.16" data-doi="10.1371/journal.pgen.1002410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coram</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shriver</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romieu</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barsh</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ancestral components of admixed genomes in a Mexican cohort.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DJohnson%26rft.auinit1%253DN.%2BA.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253De1002410%26rft.epage%253De1002410%26rft.atitle%253DAncestral%2Bcomponents%2Bof%2Badmixed%2Bgenomes%2Bin%2Ba%2BMexican%2Bcohort.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002410%26rft_id%253Dinfo%253Apmid%252F22194699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22194699&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.17" data-doi="10.1016/j.ajhg.2013.06.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maples</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RFMix: a discriminative modeling approach for rapid and robust local-ancestry inference.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">278</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D93%26rft.spage%253D278%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.06.020%26rft_id%253Dinfo%253Apmid%252F23910464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.06.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23910464&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.18" data-doi="10.1371/journal.pgen.1000686"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A genealogical interpretation of principal components analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000686%26rft.epage%253De1000686%26rft.atitle%253DA%2Bgenealogical%2Binterpretation%2Bof%2Bprincipal%2Bcomponents%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000686%26rft_id%253Dinfo%253Apmid%252F19834557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000686&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19834557&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.19" data-doi="10.1371/journal.pgen.1003925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Estrada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakharia</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCauley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrnes</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reconstructing the population genetic history of the Caribbean.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003925%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003925%26rft_id%253Dinfo%253Apmid%252F24244192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24244192&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.20" data-doi="10.1371/journal.pgen.1002298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">N’Diaye</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tayo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification, replication, and fine-mapping of loci associated with adult height in individuals of African ancestry.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DN%2527diaye%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002298%26rft.epage%253De1002298%26rft.atitle%253DIdentification%252C%2Breplication%252C%2Band%2Bfine-mapping%2Bof%2BLoci%2Bassociated%2Bwith%2Badult%2Bheight%2Bin%2Bindividuals%2Bof%2Bafrican%2Bancestry.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002298%26rft_id%253Dinfo%253Apmid%252F21998595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002298&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21998595&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.21" data-doi="10.1016/j.ajhg.2008.08.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The population reference sample, POPRES: a resource for population, disease, and pharmacological genetics research.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DNelson%26rft.auinit1%253DM.%2BR.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D347%26rft.epage%253D358%26rft.atitle%253DThe%2BPopulation%2BReference%2BSample%252C%2BPOPRES%253A%2Ba%2Bresource%2Bfor%2Bpopulation%252C%2Bdisease%252C%2Band%2Bpharmacological%2Bgenetics%2Bresearch.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2008.08.005%26rft_id%253Dinfo%253Apmid%252F18760391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2008.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18760391&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259307200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.22" data-doi="10.1038/nature07331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kutalik</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genes mirror geography within Europe.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNovembre%26rft.auinit1%253DJ.%26rft.volume%253D456%26rft.issue%253D7218%26rft.spage%253D98%26rft.epage%253D101%26rft.atitle%253DGenes%2Bmirror%2Bgeography%2Bwithin%2BEurope.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07331%26rft_id%253Dinfo%253Apmid%252F18758442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18758442&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260674000048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.23" data-doi="10.1093/bioinformatics/btp197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paşaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimmel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halperin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inference of locus-specific ancestry in closely related populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">i213</span>–<span class="cit-lpage">i221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp197%26rft_id%253Dinfo%253Apmid%252F19477991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTIvaTIxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.24" data-doi="10.1093/bioinformatics/btt166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torgerson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Analysis of Latino populations from GALA and MEC studies reveals genomic loci with biased local ancestry estimation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1407</span>–<span class="cit-lpage">1415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt166%26rft_id%253Dinfo%253Apmid%252F23572411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjkvMTEvMTQwNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTIvMjUwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.25" data-doi="10.1371/journal.pgen.0030160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paschou</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziv</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burchard</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choudhry</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez-Cintron</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PCA-correlated SNPs for structure identification in worldwide human populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D3%26rft.spage%253De160%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.26" data-doi="10.1371/journal.pgen.1000114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paschou</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drineas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nievergelt</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nickerson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Tracing sub-structure in the European American population with PCA-informative markers.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPaschou%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253De1000114%26rft.epage%253De1000114%26rft.atitle%253DTracing%2Bsub-structure%2Bin%2Bthe%2BEuropean%2BAmerican%2Bpopulation%2Bwith%2BPCA-informative%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000114%26rft_id%253Dinfo%253Apmid%252F18797516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18797516&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.27" data-doi="10.1086/420871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattangadi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lane</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hafler</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Methods for high-density admixture mapping of disease genes.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">979</span>–<span class="cit-lpage">1000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPatterson%26rft.auinit1%253DN.%26rft.volume%253D74%26rft.issue%253D5%26rft.spage%253D979%26rft.epage%253D1000%26rft.atitle%253DMethods%2Bfor%2Bhigh-density%2Badmixture%2Bmapping%2Bof%2Bdisease%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F420871%26rft_id%253Dinfo%253Apmid%252F15088269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/420871&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15088269&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220926100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.28" data-doi="10.1016/S0140-6736(13)60681-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perera</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavallari</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limdi</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gamazon</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konkashbaev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic variants associated with warfarin dose in African-American individuals: a genome-wide association study.</span> <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">382</span>: <span class="cit-fpage">790</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D382%26rft.spage%253D790%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252813%252960681-9%26rft_id%253Dinfo%253Apmid%252F23755828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(13)60681-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23755828&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324239200029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.29" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinblatt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadick</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.30" data-doi="10.1371/journal.pgen.1000519"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tandon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rafaels</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sensitive detection of chromosomal segments of distinct ancestry in admixed populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPrice%26rft.auinit1%253DA.%2BL.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000519%26rft.epage%253De1000519%26rft.atitle%253DSensitive%2Bdetection%2Bof%2Bchromosomal%2Bsegments%2Bof%2Bdistinct%2Bancestry%2Bin%2Badmixed%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000519%26rft_id%253Dinfo%253Apmid%252F19543370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543370&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.31" data-doi="10.1038/nrg2813"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">New approaches to population stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPrice%26rft.auinit1%253DA.%2BL.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D459%26rft.epage%253D463%26rft.atitle%253DNew%2Bapproaches%2Bto%2Bpopulation%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2813%26rft_id%253Dinfo%253Apmid%252F20548291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2813&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20548291&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.32" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.33" data-doi="10.1086/380416"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Informativeness of genetic markers for inference of ancestry.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">1402</span>–<span class="cit-lpage">1422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D73%26rft.issue%253D6%26rft.spage%253D1402%26rft.epage%253D1422%26rft.atitle%253DInformativeness%2Bof%2Bgenetic%2Bmarkers%2Bfor%2Binference%2Bof%2Bancestry.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F380416%26rft_id%253Dinfo%253Apmid%252F14631557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/380416&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14631557&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187491100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.34" data-doi="10.1038/nrg2760"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jewett</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szpiech</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jankovic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association studies in diverse populations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D356%26rft.epage%253D366%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bdiverse%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2760%26rft_id%253Dinfo%253Apmid%252F20395969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2760&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20395969&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276771400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.35" data-doi="10.1371/journal.pgen.0020143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seldin</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shigeta</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villoslada</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selmi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuomilehto</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">European population substructure: clustering of northern and southern populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSeldin%26rft.auinit1%253DM.%2BF.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253De143%26rft.epage%253De143%26rft.atitle%253DEuropean%2Bpopulation%2Bsubstructure%253A%2Bclustering%2Bof%2Bnorthern%2Band%2Bsouthern%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020143%26rft_id%253Dinfo%253Apmid%252F17044734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17044734&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.36" data-doi="10.1038/nrg3002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seldin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">New approaches to disease mapping in admixed populations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DSeldin%26rft.auinit1%253DM.%2BF.%26rft.volume%253D12%26rft.issue%253D8%26rft.spage%253D523%26rft.epage%253D528%26rft.atitle%253DNew%2Bapproaches%2Bto%2Bdisease%2Bmapping%2Bin%2Badmixed%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3002%26rft_id%253Dinfo%253Apmid%252F21709689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21709689&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.37" data-doi="10.1534/genetics.108.100222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shringarpure</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">E. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">mStruct: inference of population structure in light of both genetic admixing and allele mutations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">575</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.100222%26rft_id%253Dinfo%253Apmid%252F19363128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzIvNTc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.38" data-doi="10.1111/j.1365-294X.2010.04691.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storfer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spear</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holderegger</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waits</span>  <span class="cit-name-given-names">L. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Landscape genetics: where are we now?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3496</span>–<span class="cit-lpage">3514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DStorfer%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D3496%26rft.epage%253D3514%26rft.atitle%253DLandscape%2Bgenetics%253A%2Bwhere%2Bare%2Bwe%2Bnow%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04691.x%26rft_id%253Dinfo%253Apmid%252F20723061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04691.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20723061&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281285200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Viterbi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Error bounds for convolutional codes and an asymptotically optimum decoding algorithm.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Inf. Theory</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">260</span>–<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BInf.%2BTheory%26rft.volume%253D13%26rft.spage%253D260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.40" data-doi="10.1073/pnas.0403170101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wasser</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shedlock</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comstock</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostrander</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mutayoba</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Assigning African elephant DNA to geographic region of origin: applications to the ivory trade.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">14847</span>–<span class="cit-lpage">14852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0403170101%26rft_id%253Dinfo%253Apmid%252F15459317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQxLzE0ODQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMi8yNTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.41" data-doi="10.1073/pnas.0609714104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wasser</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mailand</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Booth</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mutayoba</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kisamo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Using DNA to track the origin of the largest ivory seizure since the 1989 trade ban.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">4228</span>–<span class="cit-lpage">4233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0609714104%26rft_id%253Dinfo%253Apmid%252F17360505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzEwLzQyMjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEyLzI1MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.42" data-doi="10.1038/ng.894"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wegmann</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veeramah</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathias</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolae</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recombination rates in admixed individuals identified by ancestry-based inference.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">847</span>–<span class="cit-lpage">853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWegmann%26rft.auinit1%253DD.%26rft.volume%253D43%26rft.issue%253D9%26rft.spage%253D847%26rft.epage%253D853%26rft.atitle%253DRecombination%2Brates%2Bin%2Badmixed%2Bindividuals%2Bidentified%2Bby%2Bancestry-based%2Binference.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.894%26rft_id%253Dinfo%253Apmid%252F21775992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.894&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21775992&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.43" data-doi="10.1038/ng.2285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halperin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A model-based approach for analysis of spatial structure in genetic data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DW.%2BY.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D725%26rft.epage%253D731%26rft.atitle%253DA%2Bmodel-based%2Bapproach%2Bfor%2Banalysis%2Bof%2Bspatial%2Bstructure%2Bin%2Bgenetic%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2285%26rft_id%253Dinfo%253Apmid%252F22610118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22610118&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.12.2505.44" data-doi="10.1186/gb-2009-10-12-r141"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zakharia</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Absher</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Assimes</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Go</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Characterizing the admixed African ancestry of African Americans.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DZakharia%26rft.auinit1%253DF.%26rft.volume%253D10%26rft.issue%253D12%26rft.spage%253DR141%26rft.epage%253DR141%26rft.atitle%253DCharacterizing%2Bthe%2Badmixed%2BAfrican%2Bancestry%2Bof%2BAfrican%2BAmericans.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-12-r141%26rft_id%253Dinfo%253Apmid%252F20025784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-12-r141&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20025784&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F12%2F2505.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/12/2505.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/12/2493" title="A Shigella flexneri Virulence Plasmid Encoded Factor Controls Production of Outer Membrane Vesicles" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/12/2519" title="Quantitative Trait Locus Mapping of Melanization in the Plant Pathogenic Fungus Zymoseptoria tritici" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/12"><span>Volume 4 Issue 12, December 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/12.cover-source.jpg?itok=ezho2p8P" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/12/2505" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fTQHnFMG4SXXDI1i39mIHnyIMJkVCG8Xe2OkFmJ_3M0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/12/2505" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/12/2505" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/12/2505" class="active" data-icon-position="" data-hide-link-title="0">Spatial Localization of Recent Ancestors for Admixed Individuals</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459747" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2fcA1dI-o8zHKTWMVLgT85_8Y38icbS0WbgRvURTHtE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/12/2505.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/12/2505" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-RImrjZLDryhZHqS98mNoGZuzNSaGvBV5I9dpR4KmzR4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/12/2505" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/12/2505.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459747" id="node459747--2" data-pisa="ggg;4/12/2505" data-pisa-master="ggg;g3.114.014274" data-apath="/ggg/4/12/2505.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Spatial Localization of Recent Ancestors for Admixed Individuals</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Wen-Yun Yang</span>, <span class="highwire-citation-author" data-delta="1">Alexander Platt</span>, <span class="highwire-citation-author" data-delta="2">Charleston Wen-Kai Chiang</span>, <span class="highwire-citation-author" data-delta="3">Eleazar Eskin</span>, <span class="highwire-citation-author" data-delta="4">John Novembre</span> and <span class="highwire-citation-author" data-delta="5">Bogdan Pasaniuc</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2505-2518;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014274 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459747--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Wen-Yun Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wen-Yun%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWen-Yun%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander Platt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BPlatt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platt%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BPlatt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charleston Wen-Kai Chiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charleston%2BWen-Kai%2BChiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiang%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharleston%2BWen-Kai%2BChiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Eleazar Eskin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Human Genetics, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleazar%2BEskin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskin%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEleazar%2BEskin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John Novembre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Human Genetics, University of Chicago, Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BNovembre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Novembre%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BNovembre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Bogdan Pasaniuc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Human Genetics, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Pathology and Laboratory Medicine, Geffen School of Medicine at University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bogdan%2BPasaniuc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pasaniuc%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABogdan%2BPasaniuc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bpasaniuc@mednet.ucla.edu" class="" data-icon-position="" data-hide-link-title="0">bpasaniuc@mednet.ucla.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/12/2505" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/12/2505&amp;title=Spatial%20Localization%20of%20Recent%20Ancestors%20for%20Admixed%20Individuals" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/12/2505&amp;title=Spatial%20Localization%20of%20Recent%20Ancestors%20for%20Admixed%20Individuals" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/12/2505&amp;title=Spatial%20Localization%20of%20Recent%20Ancestors%20for%20Admixed%20Individuals" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/12/2505&amp;text=Spatial%20Localization%20of%20Recent%20Ancestors%20for%20Admixed%20Individuals" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/12/2505&amp;title=Spatial%20Localization%20of%20Recent%20Ancestors%20for%20Admixed%20Individuals" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/12/2505&amp;t=Spatial%20Localization%20of%20Recent%20Ancestors%20for%20Admixed%20Individuals" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/12/2505&amp;title=Spatial%20Localization%20of%20Recent%20Ancestors%20for%20Admixed%20Individuals" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/12/2505&amp;title=Spatial%20Localization%20of%20Recent%20Ancestors%20for%20Admixed%20Individuals" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459747/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459747" id="node459747--3" data-pisa="ggg;4/12/2505" data-pisa-master="ggg;g3.114.014274" data-apath="/ggg/4/12/2505.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Spatial Localization of Recent Ancestors for Admixed Individuals</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Wen-Yun Yang</span>, <span class="highwire-citation-author" data-delta="1">Alexander Platt</span>, <span class="highwire-citation-author" data-delta="2">Charleston Wen-Kai Chiang</span>, <span class="highwire-citation-author" data-delta="3">Eleazar Eskin</span>, <span class="highwire-citation-author" data-delta="4">John Novembre</span> and <span class="highwire-citation-author" data-delta="5">Bogdan Pasaniuc</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2505-2518;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014274 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459747--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Wen-Yun Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wen-Yun%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWen-Yun%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander Platt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BPlatt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platt%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BPlatt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charleston Wen-Kai Chiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charleston%2BWen-Kai%2BChiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiang%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharleston%2BWen-Kai%2BChiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Eleazar Eskin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Human Genetics, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleazar%2BEskin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskin%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEleazar%2BEskin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John Novembre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Human Genetics, University of Chicago, Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BNovembre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Novembre%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BNovembre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Bogdan Pasaniuc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Interdepartmental Program in Bioinformatics, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Human Genetics, University of California, Los Angeles, California 90095</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Pathology and Laboratory Medicine, Geffen School of Medicine at University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bogdan%2BPasaniuc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pasaniuc%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABogdan%2BPasaniuc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bpasaniuc@mednet.ucla.edu" class="" data-icon-position="" data-hide-link-title="0">bpasaniuc@mednet.ucla.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2505&amp;rft.epage=2518&amp;rft.atitle=Spatial%20Localization%20of%20Recent%20Ancestors%20for%20Admixed%20Individuals&amp;rft.volume=4&amp;rft.issue=12&amp;rft.date=2014-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Yang%2C+Wen-Yun&amp;rft.au=Platt%2C+Alexander&amp;rft.au=Chiang%2C+Charleston+Wen-Kai&amp;rft.au=Eskin%2C+Eleazar&amp;rft.au=Novembre%2C+John&amp;rft.au=Pasaniuc%2C+Bogdan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459747/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459747/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459747/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459747/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459747/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459747/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459747/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459747/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459747/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459747/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459747/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459747/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/12/2505" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-27" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/12/2505.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/12/2505.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/12/2505.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/12/2505.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
