<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/10/1825" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/10/1825.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/10/1825.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/10/1825.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/10/1825.atom" />
<meta name="HW.pisa" content="ggg;4/10/1825" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Sensory Rewiring in an Echolocator: Genome-Wide Modification of Retinogenic and Auditory Genes in the Bat Myotis davidii" />
<meta name="DC.Identifier" content="10.1534/g3.114.011262" />
<meta name="DC.Date" content="2014-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Hudson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Bats comprise 20% of all mammalian species and display a number of characteristics, including true flight, echolocation, and a heightened ability to resist viral load that uniquely position this group for comparative genomic studies. Here we searched for evidence of genomic variation consistent with sensory rewiring through bat evolution. We focused on two species with divergent sensory preferences. Myotis davidii is a bat species that echolocates and possesses dim- but not daylight-adapted vision whereas the black flying fox ( Pteropus alecto) has highly developed day vision but does not echolocate. Using the naked mole rat as a reference, we found five functional genes ( CYP1A2 , RBP3 , GUCY2F , CRYBB1 , and GRK7 ) encoding visual proteins that have degenerated into pseudogenes in M. davidii but not P. alecto . In a second approach genome-wide codon usage bias (CUB) was compared between the two bat species. This CUB ranking systematically enriched for vision-related ( CLN8 , RD3 , IKZF1 , LAMC3 , CRX , SOX8 , VAX2 , HPS1 , RHO , PRPH2 , and SOX9 ) and hearing-related ( TPRN , TMIE , SLC52A3 , OTOF , WFS1 , SOD1 , TBX18 , MAP1A , OTOS , GPX1 , and USH1G ) machinery in M. davidii but not P. alecto . All vision and hearing genes selectively enriched in M. davidii for which orthologs could be identified also were more biased in the echolocating M. lucifugus than the nonecholocating P. vampyrus . We suggest that the existence of codon bias in vision- and hearing-related genes in a species that has evolved echolocation implies CUB is part of evolution’s toolkit to rewire sensory systems. We propose that the two genetic changes (pseudogene formation and CUB) collectively paint a picture of that incorporates a combination of destruction and gain-of-function. Together, they help explain how natural selection has reduced physiological costs associated with the development of a smaller eye poorly adapted to day vision but that also contribute to enhanced dim light vision and the hearing adaptations consonant with echolocation." />
<meta name="DC.Contributor" content="Nicholas J. Hudson" />
<meta name="DC.Contributor" content="Michelle L. Baker" />
<meta name="DC.Contributor" content="Nathan S. Hart" />
<meta name="DC.Contributor" content="James W. Wynne" />
<meta name="DC.Contributor" content="Quan Gu" />
<meta name="DC.Contributor" content="Zhiyong Huang" />
<meta name="DC.Contributor" content="Guojie Zhang" />
<meta name="DC.Contributor" content="Aaron B. Ingham" />
<meta name="DC.Contributor" content="Linfa Wang" />
<meta name="DC.Contributor" content="Antonio Reverter" />
<meta name="article:published_time" content="2014-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Sensory Rewiring in an Echolocator: Genome-Wide Modification of Retinogenic and Auditory Genes in the Bat Myotis davidii" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Bats comprise 20% of all mammalian species and display a number of characteristics, including true flight, echolocation, and a heightened ability to resist viral load that uniquely position this group for comparative genomic studies. Here we searched for evidence of genomic variation consistent with sensory rewiring through bat evolution. We focused on two species with divergent sensory preferences. &lt;i&gt;Myotis davidii&lt;/i&gt; is a bat species that echolocates and possesses dim- but not daylight-adapted vision whereas the black flying fox (&lt;i&gt;Pteropus alecto)&lt;/i&gt; has highly developed day vision but does not echolocate. Using the naked mole rat as a reference, we found five functional genes (&lt;i&gt;CYP1A2&lt;/i&gt;, &lt;i&gt;RBP3&lt;/i&gt;, &lt;i&gt;GUCY2F&lt;/i&gt;, &lt;i&gt;CRYBB1&lt;/i&gt;, and &lt;i&gt;GRK7&lt;/i&gt;) encoding visual proteins that have degenerated into pseudogenes in &lt;i&gt;M. davidii&lt;/i&gt; but not &lt;i&gt;P. alecto&lt;/i&gt;. In a second approach genome-wide codon usage bias (CUB) was compared between the two bat species. This CUB ranking systematically enriched for vision-related (&lt;i&gt;CLN8&lt;/i&gt;, &lt;i&gt;RD3&lt;/i&gt;, &lt;i&gt;IKZF1&lt;/i&gt;, &lt;i&gt;LAMC3&lt;/i&gt;, &lt;i&gt;CRX&lt;/i&gt;, &lt;i&gt;SOX8&lt;/i&gt;, &lt;i&gt;VAX2&lt;/i&gt;, &lt;i&gt;HPS1&lt;/i&gt;, &lt;i&gt;RHO&lt;/i&gt;, &lt;i&gt;PRPH2&lt;/i&gt;, and &lt;i&gt;SOX9&lt;/i&gt;) and hearing-related (&lt;i&gt;TPRN&lt;/i&gt;, &lt;i&gt;TMIE&lt;/i&gt;, &lt;i&gt;SLC52A3&lt;/i&gt;, &lt;i&gt;OTOF&lt;/i&gt;, &lt;i&gt;WFS1&lt;/i&gt;, &lt;i&gt;SOD1&lt;/i&gt;, &lt;i&gt;TBX18&lt;/i&gt;, &lt;i&gt;MAP1A&lt;/i&gt;, &lt;i&gt;OTOS&lt;/i&gt;, &lt;i&gt;GPX1&lt;/i&gt;, and &lt;i&gt;USH1G&lt;/i&gt;) machinery in &lt;i&gt;M. davidii&lt;/i&gt; but not &lt;i&gt;P. alecto&lt;/i&gt;. All vision and hearing genes selectively enriched in &lt;i&gt;M. davidii&lt;/i&gt; for which orthologs could be identified also were more biased in the echolocating &lt;i&gt;M. lucifugus&lt;/i&gt; than the nonecholocating &lt;i&gt;P. vampyrus&lt;/i&gt;. We suggest that the existence of codon bias in vision- and hearing-related genes in a species that has evolved echolocation implies CUB is part of evolution’s toolkit to rewire sensory systems. We propose that the two genetic changes (pseudogene formation and CUB) collectively paint a picture of that incorporates a combination of destruction and gain-of-function. Together, they help explain how natural selection has reduced physiological costs associated with the development of a smaller eye poorly adapted to day vision but that also contribute to enhanced dim light vision and the hearing adaptations consonant with echolocation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/10/01" />
<meta name="citation_mjid" content="ggg;4/10/1825" />
<meta name="citation_id" content="4/10/1825" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/10/1825" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/10/1825.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/10/1825.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/10/1825.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.011262" />
<meta name="citation_pmid" content="25096539" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1825" />
<meta name="citation_lastpage" content="1835" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nicholas J. Hudson" />
<meta name="citation_author_institution" content="Computational and Systems Biology, CSIRO Agriculture Flagship, Queensland Bioscience Precinct, Brisbane, Queensland, Australia" />
<meta name="citation_author_email" content="nick.hudson@csiro.au" />
<meta name="citation_author" content="Michelle L. Baker" />
<meta name="citation_author_institution" content="Australian Animal Health Laboratory, CSIRO Biosecurity Flagship, Geelong, Victoria, Australia" />
<meta name="citation_author" content="Nathan S. Hart" />
<meta name="citation_author_institution" content="School of Animal Biology and the Oceans Institute, University of Western Australia, Crawley, WA 6009, Western Australia, Australia" />
<meta name="citation_author" content="James W. Wynne" />
<meta name="citation_author_institution" content="Australian Animal Health Laboratory, CSIRO Biosecurity Flagship, Geelong, Victoria, Australia" />
<meta name="citation_author" content="Quan Gu" />
<meta name="citation_author_institution" content="College of Information Sciences and Technology, Donghua University, Shanghai 201620, China" />
<meta name="citation_author_institution" content="Faculty of Medicine, Imperial College London, South Kensington, London SW7 2AZ, England" />
<meta name="citation_author" content="Zhiyong Huang" />
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, 518083, China" />
<meta name="citation_author" content="Guojie Zhang" />
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, 518083, China" />
<meta name="citation_author" content="Aaron B. Ingham" />
<meta name="citation_author_institution" content="Computational and Systems Biology, CSIRO Agriculture Flagship, Queensland Bioscience Precinct, Brisbane, Queensland, Australia" />
<meta name="citation_author" content="Linfa Wang" />
<meta name="citation_author_institution" content="Australian Animal Health Laboratory, CSIRO Biosecurity Flagship, Geelong, Victoria, Australia" />
<meta name="citation_author_institution" content="Program in Emerging Infectious Diseases, Duke-NUS Graduate Medical School, Singapore" />
<meta name="citation_author" content="Antonio Reverter" />
<meta name="citation_author_institution" content="Computational and Systems Biology, CSIRO Agriculture Flagship, Queensland Bioscience Precinct, Brisbane, Queensland, Australia" />
<meta name="citation_reference" content="citation_title=Bat Ecology;citation_title=Sensory ecology and communication in the Chiroptera;citation_author=J. D. Altringham;citation_author=M. B. Fenton;citation_pages=90-127;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Biol. Rev. Camb. Philos. Soc.;citation_author=S. K. Behura;citation_author=D. W. Severson;citation_title=Codon usage bias: causative factors, quantification methods and genome-wide patterns: with emphasis on insect genomes.;citation_pages=49-61;citation_volume=88;citation_year=2013;citation_doi=10.1111/j.1469-185X.2012.00242.x" />
<meta name="citation_reference" content="citation_journal_title=Experientia;citation_journal_abbrev=Experientia;citation_author=K. P. Bhatnagar;citation_title=Olfaction in Artibeus jamaicensis and Myotis lucifugus in the context of vision and echolocation.;citation_pages=856-856;citation_volume=31;citation_year=1975;citation_issue=7;citation_pmid=1140332;citation_doi=10.1007/BF01938504" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=E. Birney;citation_author=M. Clamp;citation_author=R. Durbin;citation_title=GeneWise and Genomewise;citation_pages=988-995;citation_volume=14;citation_year=2004;citation_issue=5;citation_pmid=15123596;citation_doi=10.1101/gr.1865504" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. V. Chamary;citation_author=J. L. Parmley;citation_author=L. D. Hurst;citation_title=Hearing silence: non-neutral evolution at synonymous sites in mammals.;citation_pages=98-108;citation_volume=7;citation_year=2006;citation_issue=2;citation_pmid=16418745;citation_doi=10.1038/nrg1770" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Doherty;citation_author=J. O. McInerney;citation_title=Translational selection frequently overcomes genetic drift in shaping synonymous codon usage patterns in vertebrates.;citation_volume=30;citation_year=2013;citation_pmid=23883522;citation_doi=10.1093/molbev/mst128" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=L. Duret;citation_title=Evolution of synonymous codon usage in metazoans.;citation_pages=640-649;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12433576;citation_doi=10.1016/S0959-437X(02)00353-2" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=E. Eden;citation_author=R. Navon;citation_author=I. Steinfeld;citation_author=D. Lipson;citation_author=Z. Yakhini;citation_title=GOrilla: a tool for discovery and visualization of enriched GO terms in ranked gene lists.;citation_pages=48-48;citation_volume=10;citation_year=2009;citation_pmid=19192299;citation_doi=10.1186/1471-2105-10-48" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_journal_abbrev=Neuron;citation_author=J. Elliott;citation_author=C. Jolicoeur;citation_author=V. Ramamurthy;citation_author=M. Cayouette;citation_title=Ikaros confers early temporal competence to mouse retinal progenitor cells.;citation_pages=26-39;citation_volume=60;citation_year=2008;citation_issue=1;citation_pmid=18940586;citation_doi=10.1016/j.neuron.2008.08.008" />
<meta name="citation_reference" content="citation_journal_title=Journal of comparative physiology. A, Neuroethology, sensory, neural, and behavioral physiology;citation_journal_abbrev=J Comp Physiol A Neuroethol Sens Neural Behav Physiol;citation_author=Z. M. Fuzessery;citation_author=P. Buttenhoff;citation_author=B. Andrews;citation_author=J. M. Kennedy;citation_title=Passive sound localization of prey by the pallid bat (Antrozous p. pallidus).;citation_pages=767-777;citation_volume=171;citation_year=1993;citation_issue=6;citation_pmid=8441123;citation_doi=10.1007/BF00213073" />
<meta name="citation_reference" content="citation_journal_title=Trends in biotechnology;citation_journal_abbrev=Trends Biotechnol;citation_author=C. Gustafsson;citation_author=S. Govindarajan;citation_author=J. Minshull;citation_title=Codon bias and heterologous protein expression.;citation_pages=346-353;citation_volume=22;citation_year=2004;citation_issue=7;citation_pmid=15245907;citation_doi=10.1016/j.tibtech.2004.04.006" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=B. Gwynn;citation_author=R. S. Smith;citation_author=L. B. Rowe;citation_author=B. A. Taylor;citation_author=L. L. Peters;citation_title=A mouse TRAPP-related protein is involved in pigmentation.;citation_pages=196-203;citation_volume=88;citation_year=2006;citation_issue=2;citation_pmid=16697553;citation_doi=10.1016/j.ygeno.2006.04.002" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. J. Hudson;citation_author=Q. Gu;citation_author=S. H. Nagaraj;citation_author=Y. S. Ding;citation_author=B. P. Dalrymple;citation_title=Eukaryotic evolutionary transitions are associated with extreme codon bias in functionally-related proteins.;citation_pages=e25457-e25457;citation_volume=6;citation_year=2011;citation_issue=9;citation_pmid=21966531;citation_doi=10.1371/journal.pone.0025457" />
<meta name="citation_reference" content="citation_journal_title=Front Physiol;citation_author=G. Jones;citation_author=E. C. Teeling;citation_author=S. J. Rossiter;citation_title=From the ultrasonic to the infrared: molecular evolution and the sensory biology of bats.;citation_pages=117;citation_volume=4;citation_year=2013;citation_pmid=23755015" />
<meta name="citation_reference" content="citation_journal_title=J. Biogeogr.;citation_author=E. K. V. Kalko;citation_author=E. A. Herre;citation_author=C. O. Handley;citation_title=Relation of fig fruit characteristics to fruit-eating bats in the New and Old World tropics.;citation_pages=565-576;citation_volume=23;citation_year=1996;citation_doi=10.1111/j.1365-2699.1996.tb00018.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular and cellular neurosciences;citation_journal_abbrev=Mol Cell Neurosci;citation_author=V. S. Kerov;citation_author=M. Natochin;citation_author=N. O. Artemyev;citation_title=Interaction of transducin-alpha with LGN, a G-protein modulator expressed in photoreceptor cells.;citation_pages=485-495;citation_volume=28;citation_year=2005;citation_issue=3;citation_pmid=15737739;citation_doi=10.1016/j.mcn.2004.10.010" />
<meta name="citation_reference" content="citation_journal_title=The Laryngoscope;citation_journal_abbrev=Laryngoscope;citation_author=J. E. Kerschner;citation_title=Mucin gene expression in human middle ear epithelium.;citation_pages=1666-1676;citation_volume=117;citation_year=2007;citation_issue=9;citation_pmid=17667140;citation_doi=10.1097/MLG.0b013e31806db531" />
<meta name="citation_reference" content="citation_journal_title=Glycoconjugate journal;citation_journal_abbrev=Glycoconj J;citation_author=J. E. Kerschner;citation_author=P. Khampang;citation_author=C. B. Erbe;citation_author=A. Kolker;citation_author=J. A. Cioffi;citation_title=Mucin gene 19 (MUC19) expression and response to inflammatory cytokines in middle ear epithelium.;citation_pages=1275-1284;citation_volume=26;citation_year=2009;citation_issue=9;citation_pmid=19533339;citation_doi=10.1007/s10719-009-9245-x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. B. Kim;citation_author=X. Fang;citation_author=A. A. Fushan;citation_author=Z. Huang;citation_author=A. V. Lobanov;citation_title=Genome sequencing reveals insights into physiology and longevity of the naked mole rat.;citation_pages=223-227;citation_volume=479;citation_year=2011;citation_issue=7372;citation_pmid=21993625;citation_doi=10.1038/nature10533" />
<meta name="citation_reference" content="citation_journal_title=Neuroscience;citation_author=F. Liu;citation_author=G. Z. Xu;citation_author=L. Wang;citation_author=S. X. Jiang;citation_author=X. L. Yang;citation_title=Gene expression and protein distribution of orexins and orexin receptors in rat retina.;citation_pages=146-155;citation_volume=189;citation_year=2011;citation_pmid=21621592;citation_doi=10.1016/j.neuroscience.2011.04.011" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=X. Liu;citation_author=X. Yu;citation_author=D. J. Zack;citation_author=H. Zhu;citation_author=J. Qian;citation_title=TiGER: a database for tissue-specific gene expression and regulation.;citation_pages=271-271;citation_volume=9;citation_year=2008;citation_pmid=18541026;citation_doi=10.1186/1471-2105-9-271" />
<meta name="citation_reference" content="citation_journal_title=BMC molecular biology [electronic resource];citation_journal_abbrev=BMC Mol Biol;citation_author=R. S. Mali;citation_author=G. H. Peng;citation_author=X. Zhang;citation_author=L. Dang;citation_author=S. Chen;citation_title=FIZ1 is part of the regulatory protein complex on active photoreceptor-specific gene promoters in vivo.;citation_pages=87-87;citation_volume=9;citation_year=2008;citation_pmid=18854042;citation_doi=10.1186/1471-2199-9-87" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. W. Meredith;citation_author=J. Gatesy;citation_author=C. A. Emerling;citation_author=V. M. York;citation_author=M. S. Springer;citation_title=Rod monochromacy and the coevolution of cetacean retinal opsins.;citation_pages=e1003432;citation_volume=9;citation_year=2013;citation_pmid=23637615;citation_doi=10.1371/journal.pgen.1003432" />
<meta name="citation_reference" content="citation_journal_title=Visual neuroscience;citation_journal_abbrev=Vis Neurosci;citation_author=L. A. Newman;citation_author=P. R. Robinson;citation_title=Cone visual pigments of aquatic mammals.;citation_pages=873-879;citation_volume=22;citation_year=2005;citation_issue=6;citation_pmid=16469194" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Parker;citation_author=G. Tsagkogeorga;citation_author=J. A. Cotton;citation_author=Y. Liu;citation_author=P. Provero;citation_title=Genome-wide signatures of convergent evolution in echolocating mammals.;citation_pages=228-231;citation_volume=502;citation_year=2013;citation_pmid=24005325;citation_doi=10.1038/nature12511" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. D. Pettigrew;citation_title=Flying primates? Megabats have the advanced pathway from eye to midbrain;citation_pages=1304-1306;citation_volume=231;citation_year=1986;citation_issue=4743;citation_pmid=3945827;citation_doi=10.1126/science.3945827" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=I. Seim;citation_author=X. Fang;citation_author=Z. Xiong;citation_author=A. V. Lobanov;citation_author=Z. Huang;citation_title=Genome analysis reveals insights into physiology and longevity of the Brandt’s bat Myotis brandtii.;citation_pages=2212;citation_volume=4;citation_year=2013;citation_pmid=23962925;citation_doi=10.1038/ncomms3212" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=E. C. Teeling;citation_title=Hear, hear: the convergent evolution of echolocation in bats?;citation_pages=351-354;citation_volume=24;citation_year=2009;citation_issue=7;citation_pmid=19482373;citation_doi=10.1016/j.tree.2009.02.012" />
<meta name="citation_reference" content="citation_journal_title=Investigative Ophthalmology &amp; Visual Science;citation_journal_abbrev=IOVS;citation_author=L. Tong;citation_author=R. M. Corrales;citation_author=Z. Chen;citation_author=A. L. Villarreal;citation_author=C. S. De Paiva;citation_title=Expression and Regulation of Cornified Envelope Proteins in Human Corneal Epithelium;citation_pages=1938-1946;citation_volume=47;citation_year=2006;citation_issue=5;citation_pmid=16639001;citation_doi=10.1167/iovs.05-1129" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Y. Waldman;citation_author=T. Tuller;citation_author=T. Shlomi;citation_author=R. Sharan;citation_author=E. Ruppin;citation_title=Translation efficiency in humans: tissue specificity, global optimization and differences between developmental stages.;citation_volume=38;citation_year=2010;citation_pmid=20097653;citation_doi=10.1093/nar/gkq009" />
<meta name="citation_reference" content="citation_journal_title=Journal of neuroscience research;citation_journal_abbrev=J Neurosci Res;citation_author=J. Wang;citation_author=C. K. Chan;citation_author=J. S. Taylor;citation_author=S. O. Chan;citation_title=Localization of Nogo and its receptor in the optic pathway of mouse embryos.;citation_pages=1721-1733;citation_volume=86;citation_year=2008;citation_issue=8;citation_pmid=18214994;citation_doi=10.1002/jnr.21626" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. R. Weibel;citation_author=C. R. Taylor;citation_author=H. Hoppeler;citation_title=The concept of symmorphosis: a testable hypothesis of structure-function relationship.;citation_pages=10357-10361;citation_volume=88;citation_year=1991;citation_issue=22;citation_pmid=1946456;citation_doi=10.1073/pnas.88.22.10357" />
<meta name="citation_reference" content="citation_journal_title=Molecular vision [electronic resource];citation_journal_abbrev=Mol Vis;citation_author=G. Wistow;citation_author=S. L. Bernstein;citation_author=M. K. Wyatt;citation_author=R. N. Fariss;citation_author=A. Behal;citation_title=Expressed sequence tag analysis of human RPE/choroid for the NEIBank Project: over 6000 non-redundant transcripts, novel genes and splice variants.;citation_pages=205-220;citation_volume=8;citation_year=2002;citation_pmid=12107410" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=F. Wright;citation_title=The 'effective number of codons' used in a gene.;citation_pages=23-29;citation_volume=87;citation_year=1990;citation_issue=1;citation_pmid=2110097;citation_doi=10.1016/0378-1119(90)90491-9" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=C. Wu;citation_author=C. Orozco;citation_author=J. Boyer;citation_author=M. Leglise;citation_author=J. Goodale;citation_title=BioGPS: an extensible and customizable portal for querying and organizing gene annotation resources.;citation_pages=R130-R130;citation_volume=10;citation_year=2009;citation_issue=11;citation_pmid=19919682;citation_doi=10.1186/gb-2009-10-11-r130" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=G. Zhang;citation_author=C. Cowled;citation_author=Z. Shi;citation_author=Z. Huang;citation_author=K. A. Bishop-Lilly;citation_title=Comparative analysis of bat genomes provides insight into the evolution of flight and immunity.;citation_volume=339;citation_year=2013;citation_pmid=23258410;citation_doi=10.1126/science.1230835" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Sensory Rewiring in an Echolocator: Genome-Wide Modification of Retinogenic and Auditory Genes in the Bat Myotis davidii" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10.cover-source.jpg" />
<meta name="twitter:description" content="Bats comprise 20% of all mammalian species and display a number of characteristics, including true flight, echolocation, and a heightened ability to resist viral load that uniquely position this group for comparative genomic studies. Here we searched for evidence of genomic variation consistent with sensory rewiring through bat evolution. We focused on two species with divergent sensory preferences. Myotis davidii is a bat species that echolocates and possesses dim- but not daylight-adapted vision whereas the black flying fox ( Pteropus alecto) has highly developed day vision but does not echolocate. Using the naked mole rat as a reference, we found five functional genes ( CYP1A2 , RBP3 , GUCY2F , CRYBB1 , and GRK7 ) encoding visual proteins that have degenerated into pseudogenes in M. davidii but not P. alecto . In a second approach genome-wide codon usage bias (CUB) was compared between the two bat species. This CUB ranking systematically enriched for vision-related ( CLN8 , RD3 , IKZF1 , LAMC3 , CRX , SOX8 , VAX2 , HPS1 , RHO , PRPH2 , and SOX9 ) and hearing-related ( TPRN , TMIE , SLC52A3 , OTOF , WFS1 , SOD1 , TBX18 , MAP1A , OTOS , GPX1 , and USH1G ) machinery in M. davidii but not P. alecto . All vision and hearing genes selectively enriched in M. davidii for which orthologs could be identified also were more biased in the echolocating M. lucifugus than the nonecholocating P. vampyrus . We suggest that the existence of codon bias in vision- and hearing-related genes in a species that has evolved echolocation implies CUB is part of evolution’s toolkit to rewire sensory systems. We propose that the two genetic changes (pseudogene formation and CUB) collectively paint a picture of that incorporates a combination of destruction and gain-of-function. Together, they help explain how natural selection has reduced physiological costs associated with the development of a smaller eye poorly adapted to day vision but that also contribute to enhanced dim light vision and the hearing adaptations consonant with echolocation." />
<meta name="og-title" property="og:title" content="Sensory Rewiring in an Echolocator: Genome-Wide Modification of Retinogenic and Auditory Genes in the Bat Myotis davidii" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/10/1825" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Bats comprise 20% of all mammalian species and display a number of characteristics, including true flight, echolocation, and a heightened ability to resist viral load that uniquely position this group for comparative genomic studies. Here we searched for evidence of genomic variation consistent with sensory rewiring through bat evolution. We focused on two species with divergent sensory preferences. Myotis davidii is a bat species that echolocates and possesses dim- but not daylight-adapted vision whereas the black flying fox ( Pteropus alecto) has highly developed day vision but does not echolocate. Using the naked mole rat as a reference, we found five functional genes ( CYP1A2 , RBP3 , GUCY2F , CRYBB1 , and GRK7 ) encoding visual proteins that have degenerated into pseudogenes in M. davidii but not P. alecto . In a second approach genome-wide codon usage bias (CUB) was compared between the two bat species. This CUB ranking systematically enriched for vision-related ( CLN8 , RD3 , IKZF1 , LAMC3 , CRX , SOX8 , VAX2 , HPS1 , RHO , PRPH2 , and SOX9 ) and hearing-related ( TPRN , TMIE , SLC52A3 , OTOF , WFS1 , SOD1 , TBX18 , MAP1A , OTOS , GPX1 , and USH1G ) machinery in M. davidii but not P. alecto . All vision and hearing genes selectively enriched in M. davidii for which orthologs could be identified also were more biased in the echolocating M. lucifugus than the nonecholocating P. vampyrus . We suggest that the existence of codon bias in vision- and hearing-related genes in a species that has evolved echolocation implies CUB is part of evolution’s toolkit to rewire sensory systems. We propose that the two genetic changes (pseudogene formation and CUB) collectively paint a picture of that incorporates a combination of destruction and gain-of-function. Together, they help explain how natural selection has reduced physiological costs associated with the development of a smaller eye poorly adapted to day vision but that also contribute to enhanced dim light vision and the hearing adaptations consonant with echolocation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10.cover-source.jpg" />
    <title>Sensory Rewiring in an Echolocator: Genome-Wide Modification of Retinogenic and Auditory Genes in the Bat Myotis davidii | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"C1-4RTHEHok_9hllNfEGjq3lc77WsRHMUuFgr3WsjjM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459690","apath":"\/ggg\/4\/10\/1825.atom","pisa":"ggg;4\/10\/1825","ac":{"\/ggg\/4\/10\/1825.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/10\/1825.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/10\/1825"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/10\/1825"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/10\/1825":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+ZA3HPPWJOQmFGqFQjQ2akMbqDRykynYE5jUzOuNd\\\\\\\/zEcrapy195qQROhOsTusWrNkbeT4btQmIruyDsRoGDk0NHUDBjkBYaWi4a3jAhD7vSh7L9NqNokPcd\\\\\\\/2CrS71aMJeoHkPQ4xXewAzEzpTGby1HqJxTZ9um8RlQxwMRwzLqwts2dK3Z5zO0uSgZGNx4vZ2owFWxAEy+pmATsH\\\\\\\/kPkynKIUG3yztbAJAvYKhne8fCe8FLjF5IUhVUxW0byF8NLPryFoIMK67aI3ViW4t15YKAvNzi6jf00iO3AIs\\\\\\\/zCc5d2wV6yU0THwmpkVXlhLBnhsyQFmPrKsfIK6FfCVXIm5oK7X3b6+EXUKT0TEaKQU29\\\\\\\/D6qh7ys8eQ3gDghuXCIiq+5w\\\\\\\/LnfE6pk8aSNRqtEp+peFiv5J+8LKj8qg8E229x2jIw0sTdifHqYIldUjf2hXxQHmWMP3gzacQlcb9tF4EPmTfn0c5\\\\\\\/85fk+Mcfa2S\\\\\\\/QeoOCNiOmbEx+xe9rlGgtZ\\\\\\\/tjc+9TivshcJbpBsNWLIC6yTd3OD3ckPWMJzjw1ETMMbPd4uyH7uFxpPIoCUeWZUQj18Dccwgdpvxz8vPJARiJ9isjytRbBru0Ohw5\\\\\\\/WXyVRy6x2KSqvgA46tpOPXAIOSQP7iHfY7Sd+Ow0NifcIcUh0XhRinc\\\\\\\/+elN54JvMKNqSXudd9rjBmS2MHhqcvxqkRhgcknIEn6TwSZN3UMnmQFmtQtKxpd332GPtkokM8xRdiwKH\\\\\\\/Ppqx1CIpov4WjladnKpDzoLxgAq9sRtS4JS6uvc5kL2o4rsK04FovQD8b6As\\\\\\\/hkuJYbLy8lhuDc+wlgoa31K07C\\\\\\\/og14tYKWITIUOltyAtn7x3gMSB6+q3N+DZ3HQhBcq81HkzEDfQT+EwVdPRzbWXCQKK9eg9asPiy2LnuX1No3vZnx0oXvKUQgZh0hEvJLTq8UDLBafpxsrcRgKwNeJ7G6dMf0udFuL4E6EFpvPUVFcvSXA1LkQvJDSOQTwREkCUuwHd99pa4Jccer8A2H2Zr9zJx0PvrYnMlYTUJ5dgAvL45YWuScwsW6QM9tiRtNNNMoaKXwnguSV\\\\\\\/2oAB+uM6SPuXr\\\\\\\/AtjWFmVonRUJ0k2Wclf3EseRorp+OVPsOXO3qwwkX+28NnmFX0a8KQ5iYSXOj\\\\\\\/p1MxZNy9EnX\\\\\\\/Qc4IioP+fUvGV9\\\\\\\/J5gx984loNzOcsYG2MQzCAJtR4eI3Wg6AibJi\\\\\\\/2h\\\\\\\/pAv1xDWmEdhJQilx3LmZWrQ25sZ3v66mHl7yjyAAZ8Bi\\\u0022,\\\u0022iv\\\u0022:\\\u00227pKUjekxrn8whv03\\\\\\\/9KvYQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e3edb11194eca087eeb12761f9a468110639be67cec730564d2fbf24dc99c9fc\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qYvi3\\\\\\\/XrgwMGI+ak2uSm9oqZCLR5wtrvsLpcTmnKkQc5OZjP+Vjt5tELkwkVgaU8XgBmxEkg6J\\\\\\\/+abhd3V4\\\\\\\/dQ+hUdMI2xrcXiFW6I88DS5p408ve9l\\\\\\\/zkDJi2E9dfF\\\\\\\/SoQ1yP2tTEi04lcNhWBNBJLALzDlnB9r3d79hY21dDtzNtQO1iA6wcu9CrQheM404\\\\\\\/sSclSA7FhxephbWUbGDvwBp7vABtlbBWRmjtmunFIMmEGF4xizNWjknDgsRZVlod47Q3gGotBCBQnLhMDEY7ohky7fOUmEywShHIAEDNrIu\\\\\\\/8G4A3xu2DMJS7LslGwfptNtjAMeHCXWhctDEKfVTG6NkPRKoa0K7mQ20TAfqmPAjOJMLvgoAyTUMg3gJkJfEFPNZhYXub9OOFec3EO\\\\\\\/5MqVIMANfyAZJeRTBFPofBaV\\\\\\\/Hv7VvUWtqC1J3lcFv7PrWaw8Ri4a9daJiHJJYEaeJ\\\\\\\/JWe2MaKCNoSb72YmRdcvnry+6v0JCLwcv3G4XNk8fm1Rbc0mkRYqs9pI\\\\\\\/XKXohAayOWsHrS8kTvNavgL6GfNKeKp3z0UYhFp1KBFDhfoobZMdm1l4iyvylyPHpUQiWgQnNZIpKUREFmDrUDbl68oREuJvg3gnlhTGJXRZ1C9\\\\\\\/ID1GM0eG9PPNWaN7KV9+HiyomOkoPvTUWSMt4RlAqOOHcTjV3rhDbzqIHdNbTLkstldxzZhLHvHG0995sIpdMhjMqk3amMfN8jcecm\\\\\\\/52ohuJajGqd7lv\\\\\\\/pnTTk8BZz4uIza0UC9s94aMcqztdDsTIjYNEWKo\\\\\\\/QlXsis+3s8tWhbYz0Hfi4uY3kvs9Ll\\\\\\\/hNi1ALQL9AmFyQ1EAgDaEFnQzTwSyJf7UZgcxfTPWs8r86fUjwor76UxoiTnRYhKvq7Cdr60vnYPjNZs7Z86Jo6bQ\\\\\\\/+ARhZesSderCMUMi+FyOf05kUxjjUc9S9qm02E3XRjy3wrJlaDadtDkYJn72GrUfOho9TriXEmqP38vwLL0W4\\\\\\\/TLLNTat1B6284IVldhzsDu4R8KSJ6+bTEEtQC31mb3qjqQStM10qkE3puPcyNK\\\\\\\/S+VwVSv6kowzGeTA+M3y5e2w9otL1Bv9K47U50HU1oilf+WsTTZd6GZIFTTkAeTXDDdGuo6tqSkAr9hQQuy5puvCOuk1574Rz0Eywi61QBEgWqaukJsBMDGRfQVP0V2Rf9uhPG75DbaaUhuZZZGs1PEg3HYBqJy9fMSAyNxp5mkUDBIYqXSNWY7D1JN4tGbbEOeMCqBvdZyjO5kq17tD1w9FLUKTBv2Tghm6A8KvwCzfUxuh\\\\\\\/Eu21iVKyOocOmIrC99vznZshylyb0aDSo9F1LqHL1zztiLSBtJ2lVR74Da25VVkRdfJB6i7\\\\\\\/KaaSP0yvm8NM38c5CGWtJDEJv9SHNzEsa1\\\\\\\/74mn3aydyOf5ms\\\\\\\/HzfXTEgDHZLzZNMmH1sQCTex31JeZpaDT1DxMbbNWyTBWQOad+v\\\\\\\/rJm3o3uCSnJmtESeGmhlEv2llNjfORHVbVT44VNlYdl\\\\\\\/aWzbDGXLfX1B9oyQX5mL8es91lrHWrpiQ8q70Mfk0O7GIJo4+dEHjVHgMxCPkGbxpTxOrMxwGDoBIIzizWEjbMgS5AUFNSAAwxekrZgsu0Ka\\\\\\\/Ef1x+UvnBZqK95g1MafJG5RRQrC\\\\\\\/l5D\\\\\\\/Sye9Q8nFUVDYiv+HTQWom40zjzD2Xa2prrKB35vVolnvx9tS11kIJpflS93x0tU3HoaNFSy4Mjl3loWOgNd8YoF1iHDWCASVJ+RR8mZZZp3XWHAgN2aIPG02a2ATl2KcUQMhxiybiRMBEtq3Xv6hKydGsZU5UTfPnk2eJIe7poWvnNqAQhsPWJyjtkqueqWM7vpKk88g5wQ9rbUkYJ2hvTSppP3hpIiqSf13UC0W2EwXduhS2O9D31pvR12Vk18ZtUtxG16itoo2UFlmHR4jwdkA98dsRcPJ1eqna1gz+1WwYzOoVeaDg\\\\\\\/w3DBkohPice2UPMvwBcJRSFKvL4zjYldCG\\\\\\\/1AVBLzIra7j64lajJPQPP+2j+vIs\\\\\\\/46RQaKt1aLUFooT99nKy0BDf13fZg5V\\\\\\\/GvJrjIa7lto9uHNJ76wfSKi2Yjq8iAOt9Ky1GpxJt\\\\\\\/0A+pPYusWcQmHQYhz7yOJ7xlrzwWWRe5W3CZJykqcYTxfh6ae7\\\\\\\/hj3OppTewJBJiAVrHgS8gAdlDkBSaYZdYjWYU4ucLVhXPm\\\\\\\/dmVcUmGUOzHPvhJHJ6jIwY9kNTLgkwp4x8WqdKtKc5TuL5JTz\\\\\\\/z1cvhJigzb652pVXbWOCpY4O9m9L0S5voVFAaKdKIQunjzbFjUA8hPzxGVVbrwCe2S4lw+zU5QKzu0DRpHpLJyUVT3aURM+QxmvGxCrgU+a6Xi8T3QbDRQBclJll\\\\\\\/33SbdWmF76c4XMFXrn8HSdmLr2TOCwvlwgG5bMFEnBAdR8jJWekGV7Y1Nm7HxmKdsNRN1snXLr06bOF6MsBnjG5IxQODF7IoBpppP4kVp68RIUP4o5CBENkr1E6VupbgsPOuuB9cYFGrXdpJojo0RbgJX3w9Gxd8ndHMPyAi4npVktkBKldxHvt+OoBSULGrPcts5EwwHcwxkJxDE0s+4BWKRZEbzfyVi733YnY0wCM5w0ci0uibv6iWNVs2Cq4JGOJxdGPx92S24Fg1R1TnvAw2LuiddPL5ACV4HhQqtlCjc6fvbb7\\\\\\\/YQ2S3KcyEhVXI\\\\\\\/dLlh0juNdRqS3eR\\\\\\\/HHM7kpPTooM3xQF0t4zx2EHC4Q3CBFsr7v+tW84kFdwhMyQOGYVYHIGsiVEpqWOcdboZ98DcHb4PGJ8yXlkipaKIljLhjDK4wsigVx8OcjJnhPk7ZVkjtUnBkV3Hj+uROFTLohkXmwMDPYYOupMPhG9VHZPZaJXPeoq9Znq84W2zoV+g+pyKspRcxV2T8MnsQ+KH5Z8PNshH4GGiwW90huGVTqM6VxJDO4TQh14nv7nGlnNIjgSvwKq5DvCWCf4GVs3\\\\\\\/JdQrvIEBl\\\\\\\/CYkBgkbgemjz3UfwU+zacWDRoEX2Hz+4JrQF7YqUMOXbmlmj0qI27IjXCXTzYIS2FiMaYRH9x9WnoN1ze1VzwUIUKsmTaYK1KrIU2lv4xPlG6M6eHsyxcp44WE54zNZVJ1Tjhwqta11U+kiKtOldi7L3TSr3cgKcZvhv0ukSxUIqjuNMlqlnx+mtaMNDP7gpaPhFXwfhZ6keLyJ9OCBABtPOpRg1ho1vJreRGYLXD5HJ1cBfBiArowQ9AMdh34fWO4PJCNZ3OgW1WwG3RUb\\\\\\\/Oj\\\\\\\/1dQs+3mknGzBo5Qr9UetNEHFDPlzgdGTKO+ks1UqoemXDMcfNldfwrZAoCiPPXob7HVXelzyKc0f0+sRZ+bnRmbrHJqEzEb0HIP+\\\\\\\/FKzrI5siG7kfxCklDZVSZl8mcuku3wYv1KxfkK6mhge5dex5kyTc8pFp5eIE2jXUVUSScpDhruOORJ+rd+YSx0uNXSSfGHLI8k0z\\\\\\\/uqN0YP8D+sRMpJqZT5GWVfSkIob7UnCCUo2heXkDvT2RmrMvaq1ax4zC6LabyEWRGUr2mtBAP+DJgQ5YNDAD3h4ZBA17qZTWs9BYqr3vu4xJd4an3O0dgVDXiI3RJR+gYOV0MjCi1ZqJFCCPCen6SNYPqO9AZDPS7cbsxBcCm22wInQ8DjYW1XkyaZt7VDGRNUxpxjJ1sjxeBATG7muvWUgjQEw7UqR7hfmCYSX6m2eftZ2faqfrc\\\\\\\/BxTRbNVH0tSk8DvEItcYBq+fARIV8sKogOYnO73GOgG4iTb3gWhtttM4Qykimxox7MLLJoIR4rDqWJqv2NBFwgQBbU\\\\\\\/LxQxK77IgKSb94WfAeKuLFXlvsIPWieP\\\\\\\/PYMjOEL2uJ\\\\\\\/0Br1ZT5Z08gsPDGa2vblAobgR6wrimM3lrN+6T3TG+pMHIKiDonjva3uH+bQw\\\\\\\/slMQL2GetWFWOgxFPZPWZBIgkNfILvP9M93o4EoDJM1wdcd1E0LUfvjdnKsp6Eo1GGb3iDVxZikJ9IUlLWgyr5iUfW2w1+6cRX30swso8WU7kekPrjiDheMMQYWZPiSjFghQRu+B3S\\\\\\\/ypgZ+pGQ71UkN2wg1GXfgpZdkmnbX2xnDpgqwDA+ruoZ2wRFP0eqG0H\\\\\\\/yBh4MjBdf\\\\\\\/6L6IC9dqJYaFhKYGC1815PsmcPBclMOPLJD1FXvxoUZWjAJOx8XXHfWPToeuO5jSNy83RscoJOTw0DGbnoXqtpQfOTYJd4fSMy\\\\\\\/WMm4kxx0biVEDpLTUG7T1xEQa7UmnlpLFLTJ1T\\\\\\\/jBwHNKkwhTOGmJ5qa3OIJlGgF6DrlZAe1hja3TflCs35q\\\\\\\/yfAk\\\\\\\/1ObGHR+sZSSCswkUucwondsz+iCYuU21d9MIP\\\\\\\/VLnRBhajjhKD0LVXCFKt3nbopJoZe+NYCKuJBySLjhmjO7CHUe8ChN0KtbefUaCUZNjD9aaGa0KLntW4Ck9ohUFRgHh5\\\\\\\/WjV15YsLQ4GLUzKXT+brSnXAWY2jbpuLvjXxZDJXd0iGRZXTOYOutW1o2XRGuDhhgzv0x3o0RtWkagjqeT1lp+0yVZ8iSAMLwdMPAVxsk+oJIfcqgGDtJujGURchbSnvjVnItEAQbnwG\\\\\\\/eEdGbSBzwR1PaLyVoswKnW+P5\\\\\\\/q3Lx7S00SMIHDmwD0h\\\\\\\/KEGRXexYGbon3jyHPKWi1sWVaWV2ypGdQS9quSdBdvrT04csHZzbtqjs2Ug4OE4MTJYJ+XQMFVNSg1yxm+Y7E6Gods4VKrpUj\\\\\\\/+QFaca\\\\\\\/f\\\\\\\/Zo4jvJpBcxAaZDN1A75jrdL6dQAZkR40hsizilnQMuMV\\\\\\\/4LLIjJny8LWseD\\\\\\\/uRKMUMf13uXhA\\\\\\\/qcQdhKGjHnGEgHUvTydvqiUfcjpoQOqFEUPqyQnmsT4R9ovKS8Ov4NxpmLLewBAvq4KkkmiArRDP7n7j1Q9r3yzcwGziHDiIC05t0KoHopMfdXQ69PBnw4KXREri4aeCXuOL3NXGt7uls9smeacfK4eZvasSk95dbb0VL0Gf54S9qTyDoDr4Jrjl8vUVB3tYGIOCa\\\\\\\/B7l1NQV+D06HHRol4BV78v3SnR+VbhwEJNOCZff7yWLXIn5R2PMt1IfXoSvq1QYTss9pt9HU6exFZsizbRiYE15B6Xb0FVARWxE6iQgpeVGO\\\\\\\/Qt7a9A16quUwxY3bQNmfMwLmCPOeXhjuyxAuCQzGwR9i8nfdf4PaZ4O3lpFGsFb6ssPR31RfASBLn\\\\\\\/InjUnl7F13q+Q5HiMmexx\\\\\\\/KTgSy4k3yvMP6jiggbqhakW4eVrrIM+qt80OtM3UjtVlru1e\\\\\\\/i+SKFd\\\\\\\/e906bCPUW4nW7fzyP8fJiUrNKB\\\\\\\/XNHvE8GGa838JnFeyrm0TH7NL6miNghbQViFwlACahYMEuI2sAQ66ghl0\\\\\\\/NYsC+qmemyDuVHDZmwfxlalDAxQhYFHg01V4bE3zhwBaaD4liKQOMh7Fru+FH106UUKYCUQCsgT+qSY9Ucs6xVgsEMRoXgcZcfzJyy0emy\\\\\\\/CKYFLVzj1g04JZj9CoITWM10X3s3ALjLIp7anmMD+Ge4d0xLqAfA\\\\\\\/3a0UV3bLeZfaZWCLcC0gN77hwdFnq0QLlTIE5bwV31iXJuftUoKQ6NSMAgfMJvL3bYIqODot2D8BMyN3WetDJbv4+MHBEXKq1CCH5FsMVJiQ0cCait9eKydzJwjBSrvdI5jTMgr56zhSvi0aqFC9YSSxK3HYcPKn4+7VjBLrP6pLNnrvaH4sDB\\\\\\\/Pqx5nnDNCXBNTkz2vsdFJZv1zbKjDiVtgMMtBlbEzWrbbUKUMVGyc26kPmzw7aNSo\\\\\\\/EH9bRLnUY7uFPOF8RrCGpG1VnQxBinF5Jyx6EzAmBqKL1RHrf\\\\\\\/kasv\\\\\\\/y8H9AIWJBDFaQELbb6tC6QDSSk3CsGU6jsNnLA5ppJcktt2gwTIMLjs7sVKDY4ImjP3y12Wml8sseNpEL138XBBLwSAWx6Bgn4UgKH0P+f1tErZLbP33WaEQzlPaTWvoQnPkxSMcVnc6UaoweyZt2gfD1WOPzEa3ZP\\\\\\\/LnR4dr4D0+XkImhFHcVoDu5jSt49dqZDB3JNHSDFgoErg4zM7EcwaalhYB\\\\\\\/iiOQ7ez7GJQn\\\\\\\/4nnqW3ZvnuEu\\\\\\\/8ey5JH75tVNPLlFbnbmzH\\\\\\\/4zjVQLyIDoymkxHXAqsAyl2OzqFjL0vBxTycp2zOyBKkDIdX\\\\\\\/dOyAyY8bbqwCEFEDPRxcJw8YPe9Cu4j2d5ENAT52dOg8JM8f3ToGA7Bfh7O1Tqvx0OE4zxzlLtcMYVtu816NSD8tA4V6jVQ2mppZofi3YC2ke0aFfRxnv9toVvwChXF3DEvamYCHIfgo9\\\\\\\/A5yABKG1RUsEsF0ahbEAND+5i3A+j526MecCoZP7gVnmhF3ZAYSSDaXF5Wapi3OCwkT63mYhoSYoPQPg3wwDMwv2wgGvkK0bM4tziRYO9ZD2BdCBUSr3XHC+jJN8rmsrgUwMsirPp7gmRXYIRCk6RSML4C7UMdbM8loDMi0+x8ucA4o4Rq8jfmILiwTsM9fMtGGgLyGoN+Fb4Jnytnd3ctdd8pzW2RTtLTOkhZ\\\\\\\/gEUhww0p1w\\\\\\\/w0dsfz\\\\\\\/99ksJpAZ3NG7gIpT+5ua2fdfs03+zSZSWTi8qWSrvfQp+HZfF4uz9Ccc\\\\\\\/vf4MX1rFo3c5e05h3NmB328rM3U54PyFShjD\\\\\\\/Sj47+EHr\\\\\\\/lD7glEaQM5wGXb4NWFSDNuQjhJwWErZr\\\\\\\/OtMrsrV0\\\\\\\/tMS36M1wp7zX44jEDZVhFNtOPbYFZBB0B+J8sAVBarXPreKzPVFtLnqu9IbMsPtykY\\\\\\\/Ss7k2Li7v+pt9O8B4CiKJdArJziQxgabNTJ9NwLu\\\\\\\/mwnZZ0AK4v1xx02yJVQ7zQrRQMzER9ZP92rdGmx9\\\\\\\/TIfZmer7P9kbomkv4p81qYGh888skrtaDSTiXBYYM0Fl4p\\\\\\\/0W564R47HJOV3v86h3c600KGeDHqEKVZ0sJRHZD86dPsOTJgigd4CWPYwZ12Iw7nKwn8hGT\\\\\\\/2lMCSQu2dE9ur7SWCpS+O7\\\\\\\/es6sLDU6ztQr0qLkVtKjasr2BaI\\\\\\\/SYflNSOQFfRYTvaS6ng+ZGX5qwpXu70glbU3ZqawqM99HHrfnjw429qkhuNWcIQCrYxiK3lZy9f8D7+2AVdbizWhjtgs4w8ErXvhEyo3ODnDPbSCwZf+4b04MDd8gy6zxZ8fUYj9Qo6ZuMLG2DPqAdikFpDYh+BrFAs77hF5smPI+NsAOGwureKEelOiGRpGNeVTJeAxe\\\\\\\/PqEIXDb9N0DlkHndoNE2Bllg7xv9xz1wghc5YlupYhEoCc6ujI27ihKELxVFPA3RwSHPXPFJaZMKi9yUXP\\\\\\\/rzKbf30+djuA0foV2eNBztV+yvVizljopoPYk3RSr+992Nh0X7A\\\\\\\/m+\\\\\\\/qRoJPatIxvTAP4a3gUy21Ei3GVrhYm37Yi1dlqEO+dyu0EGkNgILoZIOQhOFyf1wqZRMWixBj30Prq+etgiJ+FrMsPM3sYC1s4lwfweNMO8XF7EV1E\\\\\\\/1h5qvv09QwA6c9tcyJZMU47yfei7gIeHrcxBkp8sPbFz6toM4tUg706K+3lRPa2pU+hgLTSHY93BEE\\\\\\\/8toTsLSk8BQ4+0DQxowOuMeZEZUGPXU3W9tWizkNQYf0BirwP\\\\\\\/6ncJ370je65QDssaIVTgmV5pMOq2K1zEcT\\\\\\\/E9u1QM9X+\\\\\\\/wi9ox2anN9lcXRzPnCiBZNBDOcLaAL+BKE52lK9kp\\\\\\\/1vHFW0p4gR\\\\\\\/9Feolp9Kl5+80W7rFlo8DYTO9g9J5ziFxlwk3xWEx6eWtB76m0JtuvOaAZF1nQrm0J9TqeWAowhgEALbysXYGxlIt6ISH3y9dZIn\\\\\\\/oFJSLhjZDXiYZ534VOMl+BwxZR247NRkDPncmZs8dRR8eEcLqTiBMTvU+pIoX6U2PHH7D6LEPMAwq+IwLYbWzzb3azvtPZasHhBSyLKSI1P94X+U0aoGPgE+t1Nn1yCXHQmGgpBc7FLuGczW8gWs48JRkEhjAy6ukjx1NkFIfppkKpyJYK4QxfIFLM7T6bFMI9xvfGRA1v4nVwHthORygTW8L4qfVsan97EOyG5cEv+1CfGfb\\\\\\\/LghKpE1NmDLf+2SOJ787uM1N8HDyWxjDm4XxEDzdogPDTJmwyCs\\\\\\\/PdfgPRpUD8n2Ls4EYfKRDiPFYFmY6l6VbZ7c0lTB92AKKkIUWO+6MYBl6Bgkrmx8AM9MNYJx8erSRL0H\\\\\\\/xMbFNttwVmuV4Exw98iEr8S15JrsVi1GgOV1ITYk3bUX26sh1G6nU6jjQyB2c91M\\\\\\\/6rEMsTGaHdtU1YOoI77W0CmV5BTpF5xK3likIATk5pjJxBWXjqlsIQ0DWvx3SMSVAYeb1XOWqobI984J4lIvZn4fvGsK1uHAHGOZn+I\\\\\\\/bjDAOPR78+gLFdO93ipIT+Vgn69PCNQqiDYdZe5c4bJKaR47IkVNzC0WfrRnsbFrIme2Ifze6tAE3pnu5AhdkLZwfbxm5H8CkjUt2HZZnAsf3ff4T7Y2edQ2uhVMBM7FK8dqapcW61blkXgUaaAYiNJRoPzWwWDwvtpWRtvx8sLd\\\\\\\/WHvLsu0vJxSt6sXX7K\\\\\\\/pby2goYw7MYMAiBMyteEG\\\\\\\/HFdAi7vduNkd\\\\\\\/YaY41N4KS0GRl0btsdYJEYC26paTaEBJRjPnWs1tEawie7EvbZUefvuPxwni91\\\\\\\/TEBRbVPvqnt3WeZr5qFKio\\\\\\\/fCTwsn5QcjipvTkAaAeUxjn+sPuiJpZSE\\\\\\\/2N3Oa9lvqQtalvVpFWsaflaRkyKyYdpMAZKVR1ROyEUrPQlWVKNFpHKs7JXqiRj1b7by\\\\\\\/x7r+L8bKDSX\\\\\\\/hAFy8GVkvK0fGN+fxVW\\\\\\\/sC3KKgxR13Fn0QN9BNFTs+k8UDSA5ckhlTq4UktuSDPiTPz7ecF3MtrBjxAt5ZcZxov+v0vcF0s3s5cmPJTdbfibAtutWYmVrTWM82O5I76S5ATcxd9YBFaDEMdttTfTImnHWlnyw9LtBkGquUf\\\\\\\/cpnlWSYTnJIOUAHQM1vMk5TZFw4DqGI3aPeCTeAat644vcsjXqrceu+45ALlb13FNyaS+aRM19U\\\\\\\/xK\\\\\\\/CLgw8zLD2IV30lkNhLdCriXo1UlLZpdJSHujq7TjWT2DGxmz\\\\\\\/AhBvnaYySt9S0U0uv+vD\\\\\\\/1dvW1QvXAtIueJBINV1FmhtEGqlnFTyIQARgIjKHUOsYyEE\\\\\\\/+TAJcAxn9sP2wB5kFadWlbMrns8\\\\\\\/lP8S9JT5tWgqdtsZYf9EWwj7xAMyjFt8dCLiaNPiGzCbTn42Kp\\\\\\\/k2Sk6TZoZuAg28muyrhZfM2zk8uwaTFPQJiW5ih5rd5YpnQWbvt+62uAgoNG0FISeDhE+NYcQ3AYm8c6iIop1DzYAMXzm4K02o3D00uRIr920VXiOy1UwL6qU2qIJn9aObAnmMDuY\\\\\\\/BZPnSwzTBh2RFoJOE+trEYgDzXgPnC6U1xjCgwWSvG6yaJxBq7Cw+wokFUHIw9ZVTZu2JOBS4qizQWrhaAteLk3R6rz+wag7t3riNattiWr3SP1Z2sbWluyggwx67hTgehSbaxp9zjqNCidQRbJ2e3c3sqR\\\\\\\/cTnhgi8Q2ryKPy4tjwoHVCFJViXQxfIZxl2miR0fiiieDL9BHERzkE9OAg1QYjS0R4vCAwMF2KP2oHgGNrizSrQy62qUD02TfLLXg6et479mzLWofWs0nfJIqieADHm7XL24tR9TOpU3Cc+fcK+rO3erW\\\\\\\/QrLz5zjt3tV\\\\\\\/Rzccaohwl01X4p3wYFz3cfYbdWD0Uqg24qo+HDdFQpLuGGG\\\\\\\/mp+T7SeI2UclL0qTeb3gMFC3uN8hBVk7nCJYx8yXdzqIrgzmDbhS\\\\\\\/K67uxexWdj4NhvQ8LzJ+8n+6TjDiKBru6Mfd9zopL+AvRr2agdIpwk\\\\\\\/jB\\\\\\\/NGxWwsv5yw4O5I6wvZn2Py571A5BhGdnuG3Mp4Gp0L6SdrQCAVCAzGxfArn+318ulrdhDHMHb5ZUTbCILl9\\\\\\\/ULJwIvcA5RQeWf2ZIqRRU\\\\\\\/2emyqj2gtPgbTeTi+7SDxNFzLeFNfAEjwofdOy\\\\\\\/QRWwS6Sm7Y66z1QWrYy\\\\\\\/rza+lYY9mahjfFfgs7TPqgjhigtMxPHTe32+3FfvO8MLW\\\\\\\/5sf3VvEq3KGlV8MU7Az8RDLtkz+5DBTuMu2XFS0jCVH6Hev9c7dQTqIoHz7tal9HzI\\\\\\\/VZXR3QO24LMmLbHoGro\\\\\\\/OeF\\\\\\\/PRdT5y1nu43ZU+eGFh3ZV1tKc1ZFYjHkWxbo8M25PcqS+h5o9Ez7NwnljuyyYgPBX4GFMw02EtFGz\\\\\\\/36JWo+n\\\\\\\/u1b58EeResq2BRyC\\\\\\\/+xFh+zdbnnFC8tMB7\\\\\\\/zI+SboJDur+DeU36vU\\\\\\\/pJ1a\\\\\\\/UMqlXDCz2ajNgxPAVAnxdnsQ1fdESywhxRjs9AxU+LsLMG5HLuNUeFMp71Gjj0TQNT\\\\\\\/oL9jXc6y2haLW6wsrLXHcUGjF4ujC97AoJaRxWL71QOUTo4agYrhappvUcZqbFvaO9xIzvrPEmwBrmflBKJ8l89bF2Fz66QLxss7YblWSFpK6TEDO0bpgD7gPAlM\\\\\\\/J7EKxPGbdw+jnoAIdtd7WEQVE4GYlWHysSs+HH\\\\\\\/1jqpmaOxUCTB9fNiLENCnsolWo\\\\\\\/01nJDvmGlDVrJLUqtAAZQ\\\\\\\/pxUi6ON12uG0y1Y+NBWvKqpB7U8NsAULhoNN+AGUF\\\\\\\/TCy7WSzGGYUEhaCYdlt4o+KQz7FR2svRfTxS\\\\\\\/PoRMmQYxn67XIVEbXEBjSfTCuQmWde3ayXSupndZsDVKnOd90eE0L9QyYCaddULy1AOqswRQMioog8WjuY\\\\\\\/HKbu7yYAvFD9ZAEmSkYKGJtNguc38PvMO9ZSGFDLnyi2njBS\\\\\\\/xUEhlC6ueVgKcSJsmEV4vkLdirB8d2p4Z0KiaL4q5XkASf\\\\\\\/U22GDTbKLT84hWUVCzaBq+xH+w3HVYHc4lo9ydiM7Dcj0KWgVlEnDS++O0i1ftRBfD7jRcC4c5bMZi6LJfXOQyDM3ilD\\\\\\\/8u0BIZUMfraWBKmdA2pYR+YRQzodNswiQJQs9Wi0POxMryrg0JgLDR3n7TIQiekAjB9X6sLiTWHwbCA0bNAhhyLUCWzDng80oHpwdCom97wPgkpD1w+bJRnysEl6\\\\\\\/+RBaaczo1Z4X8hjOXGIMdf81ka1weJuHeX3AIOPwLQW+iRNQmJ1fv5dOrABXkr4o4e5bAss9wE3hfjk566cM0KZ8OXHZ+EYgdtWvg47Ln11\\\\\\\/epnrqMgCLaXkd9nPk6ib7v5jng7B4BSKl5Y\\\\\\\/4SNScysi\\\\\\\/bz4WkPLBtWvWmZ4ZxmI82NhmX30hBvhwlJiOpwI4mx8QSWSrN1+WL0sAMI\\\\\\\/KB\\\\\\\/\\\\\\\/DU\\\\\\\/2AQ8dyHKKV+cTEs6oZ\\\\\\\/jHQ8sOGPJecnqZMbsqsy\\\\\\\/CP\\\\\\\/51iXUDhLy22fZwpMlhvcRz3\\\\\\\/KkGRWg5twoZQ2aJRh8dCpi70MglkLY9xaPFLwXzZo2i1FLRE8uxn+Wx5lNxX6pXskL8\\\\\\\/\\\\\\\/wdLt46m1vxmkV\\\\\\\/a4IT23woRUQHyYIMkiJc99w5fpSUkm\\\\\\\/bYvnCYS94WAyCcrf9NrwCQIwsMBtWh9zLuzV9GQMBA097+q7Kfo8oEfJ9XnyMcvzkCbuBPmXHkUhBUkj+KSzkZAgIGZ1m3jEOIynRcOGVkHl\\\\\\\/MNqUaI1rpF\\\\\\\/ZNLbuRhBhpJDvU00YGzJO0RSUzsGiKj2WqPKexP+tJ4moubTzUYWP089W9PoGdDqAPveLr1RazJdP5AGa9rTqIsmmXpGTeC57yDUkmTABdGqCAeA2bTLZ5yUKAbOziGyB4XdaDy2AGfJLxmaoA1xO5kFzzH4BLnsR93CtT72YMRceXhNbjxqG7rhoqNx49QnB96KA\\\\\\\/g9okrJzt2bWQ1eKx\\\\\\\/ph90cCzRRSlm4wdoCBpQQGcFTUJWUSR7riXBkYhZjeGmY+qCP3NhxtGMiU3UfoIlS4RnpDsogDjVR6Shj\\\\\\\/YrwDPoGQzEBjY\\\\\\\/8wjsyvX4fMsDeVXGOY0aa1c2JADTpWnACjfF9+hTpnuYsG\\\\\\\/JzStwVEb9MHFvj6S1ABHDJ1s8C0rx7oMcO\\\\\\\/FAy90Gc44fUJyCmlm9zZrz7BdLHfXprMqdn6QHKp7vNIIMUF\\\\\\\/IxZTPeL7IT7pUc0XIsKmClDVQEC3jpP1Vx71pPT160Wrg8munvO0CqbFjCI77zHfdY7gZEDLKUJfZ34rrysTPUPktP3lmGS+lten8JX0bQsFDPgOuznV6H3wXfiSS637Wy4Y7rjCfqYmWkL4HuuoVGlFGrPptIsFx5snT8gUcbCQ9\\\\\\\/vpIsfUjzO6LjGacTSxToHqMkkAoHasY\\\\\\\/DHw\\\\\\\/r18Q3mUivEqPBWXDc5C0qf+CuOef8oplz\\\\\\\/SohDjIFrgDt7+LQTmlW15AEqTfK8tuuHmUyf\\\\\\\/lLmYqE2Q5UjQ+EHRr+AAB3bY8ORMCDtkw9esK53dsmMU0e\\\\\\\/2Dz5Cg4CxA\\\\\\\/hK1UARsGOJfULFexzdnHL3wx\\\\\\\/akMEZ\\\\\\\/CZulbzif0gZSOH2uu+B+voLke1wqC6k5lE4cysRoIUfnCBLDUHhDPz\\\\\\\/hVdGafBtO8Fnn18bFNTTHTuGdiKrAddxClhwrulvbCAIdon8gewJR6cWWjjPn5Uy8eND\\\\\\\/kUoI0DKCYMaV3djB6NiNLEWtKSvXv3kBDNzCEiEcgCFbEXrL4J7sIfB7j7LHj3RH6CvOwkRGGSekrllN7rYACGJF7qc1hrU+r9LLh86wDjCQswODF404LOqzZ4aUN21dA2Vv3ztsDL3L7VPMxe8GbsigO1DwXkpHXDVDVD\\\\\\\/tRdGr\\\\\\\/l6\\\\\\\/UfydMftcEiIbFD4Rgb\\\\\\\/qtdSgNt4v7ohZASQkTgJyNCi\\\\\\\/ff8GO5x6dWr1wvyz4sfN6\\\\\\\/v\\\\\\\/IdU38jMa\\\\\\\/NNohLomukA5+16JnxBRhsjQ302z2IjsbIf8K+gCUiRGGrcuJp53AVTEeqxy69LVAtnVxLquLOYIKrzYZGoLy8lbkRB7s+J12SB5BbxZ2Dxe1937vDWOgEI8T3v7Ta3w9EPyO4WUESMB9XDB6b1\\\\\\\/VzXsXgRvbLOyiQFKT5ivLxjt3R1t7pZlZfkbxPdr7Mk\\\\\\\/4Ov+E4icOuzXS1iMPg61ON7Es3yDqe8ZkiAdpTyOrArPQH\\\\\\\/lm264Or14G3aFUIe67jTwRSu2JcfqG6dFf9A6za3Sl3AlUVA0HLsTO2NFA0qvReclRfYQKDmr+6N1V9bMZGZ\\\\\\\/szVtIYjq2fbUqEmSt0EusGLFg7h9KQ2pgBDXwtKslFYZDat64oZw2NX6KHb4WmUYmgNhT8FhhlgC4RnHStmVKVVn4c\\\\\\\/R6DQYS2LbJfM2vV9VJlanQ5+XnX9b24EiOjLWo6bcj3Oa4SASUE30hetPf4wSdVifo8SOrI3xK4pVJ88OscZyMCKpf+Oa3nli95SOcgj3jCfctdfzyMSWtllvE7u4aCz4wAVRju7SOJ7HZ2m0XNElyu+R2ONS1gBQUGvHIi8iNkSrUMdMvySghVUAHRDbVibpRmndjKWjVHMFL\\\\\\\/OSepmplwd62gE+ZoS7u6oW0jx8X20+n9GRvWRdcsICXEhEXQid2SNBw5kZmBtZTyq7XH19y1rWaXLiypfbALRiYsAnDE2\\\\\\\/ot4JQ69OASzeszF8cga3MuxQVGvmduMOid1qcg3EHm+Q27\\\\\\\/AzE2xACu5VcPVgnFzFNEexq2yhKAjVHjKfiCxw5qibZbE8qF48xEk5dk34OVPHSeMn6lqlJDNKmWoQoV1Y0QaiyX1HLHrEyRnTv9ubJRrMul31XG0OcDRRyM193mAhHY0TKxz9zQ+0asDiNQvchs4jk1UHno2qlRxBKvMUWSocIHBYCTWecjhFzaDCk7yMhn7FYDVlKUujJ72108o086lVW+jSEF7JjnKp+TUz3qk1YgFolYNxkC5LGsvUsPfKbbkWDWaZeirk2GQ3WG\\\\\\\/5Yg+EXDYb4ft9eEmAHEDwfcMNN4YiDmAZ\\\\\\\/QRpc5+FgT3XKydoHOMguIv8F4s5dc\\\\\\\/WVdb741sf0GtMcagpY8an7qp\\\\\\\/9bnsp9uHbyH9\\\\\\\/VTfdlUIlvtz1u0bbcEvJONyuSYzRB4j0YXOw+eAF8BMG\\\\\\\/UBVxiSjRvNn71ztTg0ObmMIyaYRRsUz8hZ9cLjov0s+8opXq2c9biX6cPzqt0ZjA5yOSpxyABXjY+g5h+555waohDEkSwZJyDGkH8SDyz+vQg5nc60x8hDVEtyEXlSyMrXwU6KuvYKvLQ9cm8DgWeqSTxirgTNt4Ce9YwXLbC1lud0Xjie8VrzFVzM4ZrmhFRFCIRjJjsGF9awNbjwd2f2Ihf8TOBfNvwqV1VDvFq2sycWAcx8gEy8nwSeVx8LEgGASOy\\\\\\\/TURY9mXrGxBPa1ltHVT4v1TCmFuq2By+Yn+OaO62lwpgLChEvKPT3U\\\\\\\/cC0f1HhOsGrBZHCCxHZ+ZAZXJu7vXw9kGoCeQB83Gw\\\\\\\/PjqF3LbUe1qPek2Cfikcu4O15sYpfARc8mDUzeKfc9rA\\\\\\\/Uoo\\\\\\\/e0i5TbVIocyHWYCjZI3LU8G8q5SQv89gKjsePKFrMJ7pTBrLclpTPt\\\\\\\/jOYujcTaOZ3EtrWhLOwMzfeTCW4b51hHGS0q5z9\\\\\\\/hU\\\\\\\/NRiYkiyqdcrP7\\\\\\\/XSoCtzyQNgcNespe9vRJ0eKrDFFAjHTO40gDMmQIZv1OndxcgsgT4Ip8OfkiOgqhQvW7jNEXG9OEreX873yVxLYzZdg68I+S78Lov5j\\\\\\\/tgPtdhw4r707mhfkyikFj1lYS1jGn5X+5x+e+J+L8zfswC\\\\\\\/mGujyket\\\\\\\/0zLol0AicEyYMmX0NgzdA7OItvY58WFE4Ei\\\\\\\/ZlhEQB1I1jZHqypNy9NJfQZWwdpjCezOBGyIERTPRD+2RaSC3wR\\\\\\\/g5NJWsnaqe4NfZmdOIxjCyGEMwC6KIRd1X1XyFegbeQBsFeRIekD5THrp7PVyEFxdWuSS1TVBljU3DZiw2lSpwNT2w\\\\\\\/bVGeKa54vux\\\\\\\/PoH3gYuynagXDKWLdQLI0iYdLzavOBPQoNkqGS3uDHeb9NBdL9DRGk4zX81WBDH\\\\\\\/dGdr3\\\\\\\/zSsUIF2l4xkpiiq\\\\\\\/4rV2RtozNy+Iy91CY17nMQqAUquOQuLLj4PPPnK4PzknDQGrGXZQcnSIEq6f2dZMdhUxx9UPpXggHFYHky1AFDRO+Ze4QWyyXbzi+lePpZHaDxK7b7O2mxkD8NSPrHXjpAqDZcIU3sb64FQou0vpOx6FvrosFU0RoL9+lKFb11xe6C++H+FdAcE9e8f6dC+DbXesEluSLDt7wmGJU\\\\\\\/KgrK69R07wwb5DHlpHrVjOv+DUfLxj6GQ1YgIwh+k1gWYkP+5umq7Zci6qRISzzmzrZkacXZk1i9HGan37LI7mpA\\\\\\\/xdyp7xgzLw92+Uo0J2uKhieyzvmE2mU9cx2EwRezWvKkGNA39fLuy7Ven8rCM\\\\\\\/rvLGc5yTCBSP89uYuGr5eG+1uHht2SYXEWoL9ry3PK168ly6wRY3F0mkJ8XY2LZbYAAgY9AJavaLzsmorpMrkO1\\\\\\\/eivA7LVRE9LuyWTAl5xH1JYbCJnrQU3pXDrGPf4KXMHwBp7vabjdqpEvVC82wvFKWzkuJadvRJxJ2fjOo9HQAMuHd6OrAbBghaK4dHYgu\\\\\\\/G9hZtDHhevZLkRY6ujfzlWlDiGshnznwN1TkQpjcWT+ZMfEN5nsTgFdLTlSxPqAb5aVHWozE31e1EnvK0Jaf0zMNhznQGeOhHAs+u1YsnKheLm4kRQna935GWd0K1fvJw+CvXHqNfjGacIneE6Ljo9WsMhbzzidJOT7Yl0TqUFkpQ\\\\\\\/097gh6W1DRdQRtDbFWvKwj5XFJnA4Wbi2ve2lSJ0y+wbKyIJnNgiaz0MIvSbOPPDC6cyw8Lo2U5mIRsL00r4XR5emWz1cbACE\\\\\\\/vn0dARnNegN7RR0jhmZPA6Max9Dz6MqXmzBdRI4\\\\\\\/ZnzrUZyrGCabKka2+DmweOm9fHGPFKBzzhnce2ooCPiFVsCE001EIMpCNldJ6a+RO++8WOBdmiUaEICsFslaRCqQrpTIcV+FDiu0tUClbn\\\\\\\/b3CNOMGRON4+UtowtNWqWrNUAjGbUnpBev+zA92AulKMvVcHv6lfa2XlLXJKFAAMSztqAWBzhy0e2xYX9jeNX7wEKxCYHQR6c5tPSlAQwn0P6WGnK+DXtg208lHQpgrsvm0Mc9PSZBE8W5wODskGdBchssvW1xw12wac13AP34u7wUeqaV4uDlCt3\\\\\\\/HLYbAoXFoFfGScSDdkyfPOyPKQq9eTmBD2Pr5ofVlFpAaFdpdNjiJI8WH1IYtbqP03+wy8t6gCjCqbdwm+XFrBYBqhuMchFxJlawbdW5nqC5aZiLmxaZFBRbKIPezpZpq4xRGIOrF+uG9AR\\\\\\\/vpYkB5nzSY++SVtoPr8TEkZv+roj808+95rhQjCQ4FKoElfMW1wTaJpahDoouLUupbZeZTfFM6P9bv3FkyBgMLfryox1su4QsAZT\\\\\\\/PgBf\\\\\\\/v8SrfL26qdaylj0wxgKptvMaxFAt3J7H14Dx0Iqb8B\\\\\\\/kDdN+qUNChBQwuDbu2XLs2bU5xzgo1xvyUoFx9c20+LEGaA+xKSdi6Kks\\\\\\\/n+Ps0pk+4JrSI931hqpyvkL7fQxkT74iaJ6sAnZxBJNB1dH3rXv5pGyUiI0DbniI6RHfyFtoAcpU6es6vudLECHEmDLTU3ARNa6yw6s9dB1NFUb6oe1Tu04gG+SLSsOxA8y2Rb+o1FT3WtScq9NpdjsGNfKQbZehbIVGr4dIuP\\\\\\\/ADNBtvoyKcRxrxYOJ3KKLAZriOkJYKODkIXGnSso8bhUAXdBKAw871kR\\\\\\\/BGuCPLD2iUrFpZqFcib1CM57\\\\\\\/CnBJ0FuKRNAVmLI9I1bhhpj0zsCnILPl+BQMKmz7ya9+\\\\\\\/DdEOJEba81vluMf\\\\\\\/gU9smxKsHnjPKfXB9rkm5pfM73dhWLbc+j8kbnFfBwzxf1agVVmmBpTS7N6J3gJ8KVyO3cYIpaHJnLu3pbk7xtEx4f9hMXG\\\\\\\/14BmcR9wSGy97vPFvMeANzCVsD3WNawWqUl4sI8xVvXbI6IXM9fcSr72zvLWzuhRHiiDMGFLjf5uHpOVhjAetJR6bE0TuzUFKROd57WsUhd28fGa\\\\\\\/ZZ\\\\\\\/d3BxQrBsLsqFfjNrhuhbGtLFRm02+LDaGJRPMKRniwT7uFbHXjkHeb5HA77LU0ZHteZVKuPqg72qdxre+gV1y7iHqNkV\\\\\\\/ESLKfl2gsGgDr\\\\\\\/Cs0Zapp1zqlYDt6EkslUC1b7CWOXrMCALnNuVORSdBtiNb3lsgS1T991ZWE4syMs0bWXa0UcFHIF6\\\\\\\/Y7GMwWSLd\\\\\\\/uBrSkUHSVV0NzY5BKQaCcplPaqIQxEtgzfCMAATbCdOrFOB6K+nPfHNcACv7d0e+9hOfvVjh12UB1Id0PdY9XVUrlQK0IBFzbsgUwJnLGifjBDNA1g3MS63aaYY+jYkfjiecgcNikSj6E\\\\\\\/WDmWfPF2mydDKw1qN29B0hGPRGHXfML2C6GOFpc0K3pk+kNo5QoWYu1BsT\\\\\\\/cLkFiafYklRH41AnPJDH4PdR+NXDxCoWENi9ons\\\\\\\/H\\\\\\\/CWH9AyJYQVJok+CoqDW8Rt9a95fHllQP6gNnGnxNfOx7FiFLnks7bHY4cdX\\\\\\\/0Vb7AQGA0IAztefuoQJIuxGO8unQszKDvooz56O48rcVY5mLwNcU15csDea5vVQMpKwrO1642+3bk8JRkN82dEw\\\\\\\/pdWmct4987bTxnrkRxaomwLbelQztD3fIB1FgyMHia4shqjnSEVqVnxBdWesKu1\\\\\\\/MdwakxYfGWjwF+JZ4CUhE5Ibkue0b4gdub3\\\\\\\/sx5t8IRsrr75ULLhLXiwYUpLWWCw7d8bC8UeKOM\\\\\\\/c8knMeksuUdb90tIdGZHWeJvq2NccRg3B5BWiwXNHwHdjIa6E7ddfMpd8uCA5Rg7vWFnyMpMP5kuSw6Kojz4dQdjT1i5TlQGcg9LE\\\\\\\/QISp+3kbWA2K\\\\\\\/UEo3zWLV5JLx7TrtXCVVjLtid\\\\\\\/\\\\\\\/hJhyYUTf9CRHMaRB1gnw6Ybp9tVC0OaOcnhO9s0xK5pstVIALOvvbi0d+ovYgSMDpvZqa24k7IHFlm5pw\\\\\\\/eIQprA+rvGI3jNmLkSy2O1CNhE9VGE54lRJYmNETZ95Um+mhBdjW1we4nBWtQhYabdETYCYrkmBRI5mgcJPFSdHKekcQZUKoQrl34O8YWBtYO+w3joN1VNPETHCctZYxx\\\\\\\/k8UKXIpIM0Vf7+jfudryHFKxfOglM3zti64wOpa1caAxXzJJjqEmHOo0t2NelwWstKy8h2HM6PVciUvIRMws5fJCuQrzdAZGyibNPMQk0syi\\\\\\\/PkXZGXyDP4l9OIwfnUExPqIAoMJrvnRoyvLthwW4HdIx5TxBkVkpg4VepSwcDNxTlPFy8Lq2pWwjyQFrG3aNPJ6yu7Xo0drpCZLnnp3QnZMvTb7E7OVMyEX+eXRqfNFihQDqwFv9lxxNJPUMxSz3m\\\\\\\/UUzKJLHN0fr5NeCHwzeAG\\\\\\\/2qpUP1uoa71ux6TYKy5uEPXYwy2uNeoW2HfeLiwJjJ0sNCNkYkObtY6bnAx9upW2er9XAmWMTOjMRBQUl6PEh3DfQrNTeQX8NyY2zZagzYfE83XV8JY1MFOu0vh8If9TvlBRBxZ1K03JTQFM5q3p3ZLSB8T5rJQ0PAECDKreOj4aItXqS7ZNcZugANyysfbS0HLRMm0AtFA6MNVzZRN7fxOv8ERu+BWIkuXtQDFAMs06KDD2OuQcVAv48Ip6GNy8fT4BKS\\\\\\\/ObECq6eacFiUUp\\\\\\\/4AZJCBGiBhg74xbDxb5tbJtExP2CmobXZObnhYdNZb0g+k6phqBYHQeNO3VCe7bAeBBvEm2ZtGLOai3Un616Jol0feTI9WkIJwjCAIvy7\\\\\\\/egB3YUXNizVfNJwHU40ewxVodoKE09pRcPnghVS7fLj03G8B798MhLZcVKPNc1OSMfD+41cNLlhFZOHqU3FHxgDrd1jGLD3cAvDuZ4\\\\\\\/kGhghlLUfwQsiJvbCrZFTUmybv\\\\\\\/kPJkBJuGLVOqvwdxo1EpcYivswzaMEtZWPkJ9WwjZNZe5nlfCaDwacxKAzdp6Wj10dNZ1uEncnSruK62aylv0h2E2ys\\\\\\\/KQvbXtn7OJSuY2M9MRpJvKgYX8fTwoI5T+vYQq+QemNIhx2f5h9DZrrLPGbeYSmg48fk1xkyx9xXbVbJF947bzNhwI\\\\\\\/yCCdau53TINPmxtf81QMtl+\\\\\\\/wkjlx+\\\\\\\/+Zn3i0ih\\\\\\\/vprcHOW+liareUESnEjHRp2aRVuZWZ54NkMjqvlLMG9VXrR562X3\\\\\\\/YKiwzil1VcDppFC81yCwdN39M0fabT8\\\\\\\/BOkSdPrZz7g8fcbTKgxt5XLy+MBtKMTRgRHVB2rPWmfsPkBcUrsfqkJ+lsuNV8g8uUZsiWGlvXI2\\\\\\\/TwWivdfV5r9J9pnpGVcN2OKIuJuWXkC8uzASDHOEIP\\\\\\\/VuQHDWTe6OHAnvVpkWND6qwpkHds3FjqCJ9Oph6sWQrf7B4ld8Mkb0peOasKlgfpcPT2FO\\\\\\\/GcXyn8leAjDn3LqiYa1bqXqa+8M\\\\\\\/8aanSLBT0lIXLW6M2\\\\\\\/E79ml3cUWMJYI7Y3WIV6sB1PkW3T\\\\\\\/SySsuyhcAFKHseNbTSzE3CoTkyunwkcjmn4yeeC7\\\\\\\/qdW5f7kqjHxOtaV1ZN\\\\\\\/SnILrxPdGO+5iJXK5X4VxMsfYn8ZabSZTqDOt6ZiC2cCxn6DGUahtRtcrM\\\\\\\/nDhq1XlddUFwanDyGp1fqFgbwOgj9mVF+dICCirc\\\\\\\/Vigl1dN4fcsUDQr44luZu54Ujw7d3XoVO4aX6GmfRYKVjzQ1luBxwPMfIXMRPfCOWDIWD\\\\\\\/kzbA4kQ7nPhYvEkg48a0ih099svE3j2uIqgUtJTLG0IIvAxCNa8aPPwaTDw9WINOmj2cDoEy5jU8hCjTmdCpaZ2lbm0F4dSM2Ua2PJLa1RjNxoE2BMCCkRswkpVxM8maGPcWn+jJ2gTMy2pmiSpW6hwmDPcV2tXf9RlDh1wJmGN\\\\\\\/FhBAdeGIg1AnlLdrDIRCT+0yWuQaYpxJ6U5WxmyoLdIvA9UlFPsV0r8KonO1\\\\\\\/IgXVRJIcPQxOFhYqFnibGLQ2hXhZqKy29tF0ERL9VAzGFz+dxzAv9ZVtZaSFg\\\\\\\/uR\\\\\\\/gq1vP3I\\\\\\\/wOn7AqC10WIhmoCNuryT4PmSZMXSiJBOn8MuVyvFs+JQM6gU2gLrnYfr2snm2dgBzVcBV2CUUOX2D9ngsHfSa5aYmoq4GK9A7cLk\\\\\\\/8Xm7BcBxSnFwBmYBQ3P7ytXUr9yr31EC56i8aBkJpihb41DNUlMOxT3JYzakxJCqfP5+OzuD4Vkuh6uKRXOCUaOqz8S5eSVjA4TBzxEJsVm33c8c\\\\\\\/hLXUm6fwnKtjSn5Cs9jAVZVPKrGmE16y8ylhDNuCKwyt99pg7rh3GF5r6I0OHeA9+PXJ75Z5WFFe1od5q5KrARjKoQLKwVkl3YoIaUmpwrbkKTsW8Q\\\\\\\/eVw9ESqO3hNFfuSSOMoQCpPggv37EAPt0N26DhDJ7Mfv4QrmvdIFd480D++82hjgmSaNkymykHB2UqcUbiKBaNM8SPtQ2fqNrS3RhOMTpd6VqzmyQ5FNsSOHTzHNG36PZdcw93vHDi7FwPp6jmpL9zNThul6y3CT6PQe9PbZWyqEeqbfliXsUZJ32FI+lwf30mk0uJwBI0pBdSvrpObrCMVBCsHsmccNPPhfDrlakFwKimt7tZfpDitql5Yik174hPaxKp\\\\\\\/u754lhvk5CuYCYHs2jQWGM5da2nEGSena3qY+qfBlVoYOESLq6WKvWd0SBKyG9Yx\\\\\\\/qqHcj5gHN1v0VYMuuJhXuwZLOtMF1igD82WsJSHEsa64suU9CwGTqkMU\\\\\\\/UG5dVyDdaR5SKtZsHIGqp2LTheU0kBzKCpWbL3kAciqk7zYfKem4+eYOT8vdMXei8hAAfSvfMS7s04OQ9Qwb3EVDE+AUMYBjnHrruSY3VRQzDIj\\\\\\\/D5OUJkFmwLxchKBYUuFZuYC0p5+dVoZxqU8P0HjXxSnL0OMwD0qvCA2S8ofUyMncjjXlWhL0zO0aWukTqX5Wgeq7pgTQjHRakOMOMiw\\\\\\\/0PQtF\\\\\\\/gt5iD4zVCy6qTlq6uCWBUrU1fI1VPHC\\\\\\\/Gy21IW4euEPQeZRAllC46Y8gZnwBgzID+I6P6NGfFovudz2im1yX4+jcdoRLpxTyFAW3sx9irDbZFGZ5X\\\\\\\/N2p8zswapwnvIMERSR8gS4hRhfmQOkxOmun19ElG7QPsB5xLh1xiRMk8rygsbvZ9FujT\\\\\\\/\\\\\\\/hdWTC33hsanWBlrukeB4nPa5DjDQwQhUBDefK\\\\\\\/F3EKLP9yb8KCpdfCmUxdFayiWLXDC3vlZGzgL4tyzFe6mhuheUxYbabZPVxNXU8LGzSSX0i8hTC5v2ub6IDhaZeef\\\\\\\/TDsxtNSE\\\\\\\/QK2fpxDe2BtI4msnzBw4UcaghBzGj8OjfxAOmwqLWqG0tfzIkkChxNsMPkfQdvaWp6yfA4ccuDWxL3nmnTGO2NeEuE9rfUkJSQHCICR3uR1OGurBXUARL356p8TCDWNMzrB0l+wDg6OFeQoHyXIFCB86iqFdyKQ+fbpFFVbLGvURAI9FZD2Zh7D47CqpJMoh5NhMUt09t\\\\\\\/QPVaZDtv7wR401YefbbmNIE2a+1n19vSAFu25PChIhkrXeqFtF8v4IRYv+FtAATnVuK28Vr5zWzqbp6QHXgSphlywXDxHiA1TcHL21VUR11+Lz1gviqfty21H36YAsg0Yo8wOBf0KrgYcuoE43KudGKV8x5pjFzBobDUGmHSebN8phoXw1n4Li5HBQvWaNlbOcSqopzhHZMa98YrPTNytMRZHSNEsDN5PSOxdiWBxSB6Cv27K9KpJM9ViCOcum2JbPPpcasuzO3FPzVmu1u2adAnwSL8resKldNYuAe32EOVAysxqaCsjNAeySvvog1jvP9axk6Q4OqS7Y9Z9kjViTDA2Nebl\\\\\\\/P5XZEbdbC8WuN\\\\\\\/4TDkG1i4fx\\\\\\\/k0P+D9SbODWSCNoAfB91BIDp5\\\\\\\/9dDB6nlTmA8qBKVdqEY\\\\\\\/ZQtLp09X\\\\\\\/o5A7Qa1M3v21wAL8CRXtdMBrGX8V6Iiccfe1cKk8bQ7g8RweT4gHmrXovK1Lb4LLy0ywr1X6dpURX7ZsJ5jFAlbygibC7KO5\\\\\\\/0cMdOxbE9+\\\\\\\/VGoRhpuU3Qipym+N1UP3NXbTTiBqTcbH7M6K9WGBjehr8q4e82SX8Dp3oGi904oX1bD8fAJJKjwchT2rb0NTBf2IezHap0h8xkYG\\\\\\\/TWt5ckskBxmLj0XyUoHaj+FQDEVwHlPNAn8og4POZb3f2kDj9JBZw3+PdBOu4iPiipEEoFxuHX\\\\\\\/HTRKLEvEV4hrKIQrqSW4RnpD+4UFhcw85vIMpFJ\\\\\\\/7OqqNozaRYwyPF+hqzXEgATXrmtX3QpGRs8gPe0bejfZSkhdqrIRFcr2V4ADbLZTNWJV1tFbAKAB6iarUXAuUEy3MrXW1c6fXXSbYMekvhFA5AzVL6VGZOU43a9HSuvzclFuVCf7SuIQi6KE\\\\\\\/6YEboJWgVX9qKfLvzG3WTpphT6Ri8NygI99v4HkbrsBSpeIjnqDWUOGkweCgw6y4SwVDhGwU9DZzPA7c5pR8D66Wkv4R0UYfd87Zy5fq0wcfm5W5UbW6xadqTC4g2HnSAkO6Nf4bwx2Ow2DC0xBFUWWXKlWTxwFa9c3B2j+hhyf\\\\\\\/SnJ20AjUQNjmM25xydu0cjZ8qqDHkiCC83fIXQ1GQyky9hup6+hI9xFfS+4bX\\\\\\\/HPXExnYzbbXnxJrIf4Kj5kDdu5H6lNE+UtlSD0W5rHTrPiWk9DTauiR1ENhAzEu2ddaW4+pFockUzU1BXIbA8n+ndzzAm9t+0\\\\\\\/2B8119z5ri8zyGfPuxhb7Sz2UyQnky2xoCXJcXHU7vRDEq+gtV\\\\\\\/a1jLA4gjqyGYxoRPfJT2nWMBTxanT\\\\\\\/bpM\\\\\\\/5s\\\\\\\/ZXsYmLmHa92A82aTgtbAc\\\\\\\/PDUePH5CkONNQ0o4b\\\\\\\/DWexWj0wKsvN+crXZahyKEPn5Y5u\\\\\\\/\\\\\\\/nU29gcDtbhL4KSxOKP0gc17Xs63ophd5\\\\\\\/Y785WGc8Fzk12h5LEJjWSECDATnBhtzFZFhCi8pNXORQU58zN5RUxzN15PWwXQYCJq9NWQvPCxFNX5dV5xnvWnXgQDNkMPf53vr9Bgl85ozue3vuKXSUjUY9zgjouNcilc9XIhOJ16b1kPZyKRrDpqNl5X8sVSYfw7suPitNn6WrBcA4TWoGVOrjv2Hc2EBG3NV2HODXoOBDirIWJKi5+d9azatNUyk2b3E0sr5JAkNFx0wBHE88ad6VUEMc7+ZJ6wr0GqDT2rVdrS\\\\\\\/ud98+z0Ffznmov3pqzAuBlWrIzF9rCiELWpMp+HNaVutbDrFwdhhzygBShWc9z\\\\\\\/YdUDhELEhYtQXvwO77JGQVT5doM4Sg0mDhYiUUiLj\\\\\\\/B6bt3z9yGheIb8r7cd4tWE9y0hFapEAjlIDRrQ5aijhgw3q8zUHai5EDFYFiPuU6lrSEuZL1dDqSVfSe8Myyy2K\\\\\\\/smfTwJvsAX8Ud49HDjsOcSmfytWROVR9yqEtvld0sDBfs\\\\\\\/lmHdjQ3h04FXQr\\\\\\\/GrW1W+xXWiLvWh+GnomePjRVLTBQ5IBTqz14vMetqYGL7+23Y8KUVHhQtUcsb0VU\\\\\\\/texjcjdyQOhYMlBDu\\\\\\\/Um5CA16iEEI367YDzO1tQDAF2RSQieKO7RaeJ7jRmmuBIZODUlFAQvRPMwF9StlrDnM5iVKkxf8WiEro0LKMpkR1UyccMIBM6jVFsO9B+M2i6G+gSQN6nnp1nQt0VJUBCtWvoP8S5LjObad3w9fNKsdRNmJNllmoP2N8L0K3JHAwsCIbFqu5DPcqT57bCdaL4lqELtk6WOAxd+6CNzuIAkdU+Q9ojDdt+YzFnSfSg1psrxEbsO8WvIUTt1VwQBrb26uXhTxVCHcfIZr3pULTmB3fENb39RODID6MU0neq4zgiPiKwUtiiGzfzl28xlelItwn\\\\\\\/KbZmLuq19N\\\\\\\/HSF4rBis70gcRN7eCe7X6AA\\\\\\\/fAgc0G+xJRjnQe6ov1g0zICcyewvfVIW2rgP5WBl\\\\\\\/ZVbekljNVThxqrdX+6hN9gu5hERuU+Dvdf7Rlhj+xgL8i+YZTPtjCOf7hg5HMPM+xVWy\\\\\\\/k66PCfqGK+G4mhvlQh1Eb9CKf2QGEr7O3xUgm4WwwWxwYqD5mW85PIlv0Ykm\\\\\\\/WCz6rnTiOdEIZqeLoRP5hUqI8VwGtuyvYs0OVzIwtutozsfjN8lBOdj27XikklFaWxtbVENtxEgv+p7G4NlNy91KeX\\\\\\\/I4+lL5J6lfLVsVDRxNvHXOcSzqrSvYb4p3GsZZHfIrXt1n\\\\\\\/pzXz4brCNG9Aq3DKREm0iQ+JZKUbmBlKkPpb8eXIE6TrYxmm8\\\\\\\/jfCTrWBimdwJtSSjUAUsqf62TJuT5DMi1ZSISfoGlJ2TuPGUAtXEzJGN3h4V1fWQgwof\\\\\\\/ABgvaP234KsVIPpAYMGN9beeBxAISJdtIUaHTqHwheAM03P26of2DsDar47kfa0e23VqbFM48IERqZ3trkyYlrcDdSVpN8u3Kd35ShY4aFKiv0rPtKgjHhZtKlWFFCFz7P0WsZAkTuB06J\\\\\\\/j4qtYBeKV2h34rJvfYyRPSqvf924GKg7eDj\\\\\\\/nHHwRR\\\\\\\/Ie\\\\\\\/RRrQWB8ECdURFJgq6o2oVAr2l0l5knWF2VYShaIxCkYIjucwF0BCtnIhs6JLfFWCE8bjZqDANpb8y83ss98qkS253O6NoxLU3Lm88uNH6zW2fRVSuUPvPF0bjV2hLN0SqYaw+4gptJEo5YuS+Hp4MJo8ZgqVvgFCX\\\\\\\/hELhGzj6ZYXuduzcQLVJEJuT3VLuEzYgXXNJae5rUGsR1Ksjg4tZD\\\\\\\/7MfP7R1opmvoogfmN\\\\\\\/qugor6kfW49eDshpkVQNbw\\\\\\\/\\\\\\\/HyF3iPEs4SQ1P35tLnm+vBomna7ezHu+4cPBv8cHuaX2uoRKxsUI2c7ak5j8aWD6bVSm642Xd+hCZj1yA2A1EQlb4DOyl7n\\\\\\\/Glmvo0+7a7Y0UOAcp7MLwvvK8q9xZVssBYKl3NJTVQ9bSKqLlbwCFHQ1zZVV6HOv4yrqbS81uQ+hUdhtcEtx\\\\\\\/n4SeoDpoJlrgwnt\\\\\\\/fiwhD3B4Gnr3bpUqpxBWiZlZroNKL1IbUZGI\\\\\\\/Cn3cEkU6YjpKdmsnN8nfSkERr6OrKZMXBmm9oNp1vSeP9TaujM7bPI0LdY8K+FQNXbQtwA\\\\\\\/a2rMBq4mhiC5JNu1XwwM34sReV15xwRe3qwFFqw\\\\\\\/vo48935GzJPt4d4pzrWeIM+Fm7gNj8GkNmZdhG3jyycq2AB0gdis09q6iH4Cur0PR7LjYfbe80TimBjtg\\\\\\\/esyZUdoI1+xSKRNWFoAEfOAfq7VAgHwdyzWBkXGM34RwxpBkFUutNZMOn\\\\\\\/zByxEf1q38dZxuAOydl9y96b9PjaxfiW3wOmvenDfoyb1OUXQLLM6lltArJPud\\\\\\\/Q9tgXYfNRELVcDeIEwRWyxIcgJ3fLTYa6I0dlu8WqZk4IjT9WRJsT+Sm0NsZju9N6ilPZfuekfE6R3KFupDmlrijtIm1UNjKf9vKgejmmnQg4xfUijq9MT4Jkir7R+A8f\\\\\\\/GLPSHFLqy3OWLzBceKxWATJzw35Jd+5PMMy5rUGy0qguypmlqn7ojZp+WtY2Vqlu4BogVJD7gsymK\\\\\\\/lErf+0wxSyVF4Quuy5coriZfZEsRV3OcNCFFM96uxoXyTfafQG479\\\\\\\/W6b4wR4r\\\\\\\/q6ExA36HhLCzYVZe7yPazjM9GYtyt6rGeQYwSwy4dN33VxYonwBBPb8aj8stagjTak6DhuOVi+wJ0ZFHYVG3aeIeg2mnOx4AsVX7qxNKxkJ7nL70aBTrKpjsoZNYl27aRsPSLGH1NilJKU8Gwfsxv0oOxxiFKwswTwpOfZqDcdxVpROamj5\\\\\\\/dOFPQrte+nyDm2AnLFBLpevsaxp66oXoF1V83ZRAuHumBtSUFCAD9yxc5jSA0sxD6Hh7tOJD5gUkYcNfMaC6bdzeNRATrfoPM3l8OdPeLwaGmVpPMLIIyWWf8dUJYqLYv05vKJLakBHHEEd7RUtVr7vREBzKKUlkrEYh5hU\\\\\\\/znxGBLVe64Jf1DYRDB+8sFW6WX7L0C7UCdFZWOORZoAPj7yxcIaPv4\\\\\\\/uZQJXy2SVxkrrVjN4UlAH\\\\\\\/69SP6zasUNs\\\\\\\/VWcqPlTFTPJ42AWzXfjPsdeM3RlTwm\\\\\\\/5ch2yMdb6VVOczxyBn+PqZV9bBjZE86BxE\\\\\\\/5DvEYicqJOhT\\\\\\\/T4bYcQkNgcCs\\\\\\\/g3n56JFWr6hziF8mKjoOZ5FeXWN2lCuOjL+2z9tVj4wBjT6sG6WsqSj9XKXtk8gCLd1bBf0udEC6c8pRgvtDU9\\\\\\\/fwarzGNvwRnGP8VXGdS0zvtJr2iSpT7fg0UlpOhXY+dvjpNFKR+PH+meeyk6uK9t6nyQN8SpEljMwt4D7PIY91YmQwTXIiQSZITZu4xQfitmHihKifF6mlrAIL5O5M2xfEo1oDQfVejhkE8YC87raBV89AQBpyG9R3HrcwZvLeWtlctB+Dyr5GbLborJZNKBopbW2+\\\\\\\/v+XdSZR3UiSHth96hxl9\\\\\\\/Ef61e+7rD4SLh5oOmIzchZBIc7zE20ZyTBmT\\\\\\\/NIu4soAOyslagYyuedFmWEUJNprWuV6EueWH7\\\\\\\/qpJ70gLC4tZlE0ABtXkY5UQ9scaNpDIbOrorMyxGr0t31\\\\\\\/4M5SDQ7F70jnQIiaKqJqv+KOfxIuj0tKkEBamemCVvBVJrR+7Ser+Sm\\\\\\\/TYsiIxpumhYTMSgELeBGyd6\\\\\\\/RtPOzHIgbK93XPHT39D0uII6kAUWDN0\\\\\\\/Q3weri1I7nFibbf59L\\\\\\\/Kz8wFoqyJOkhTMcMOCfPQUdQvHGwsuJr8wmrWKwQ9F7L5SWLeb2R4vBYp36lLGY9pI+uOqp+m+urmLn1rjKTcL6SCvrbJG\\\\\\\/yFireTW6OMTRHTpOosI01i+uAa2\\\\\\\/FZb9B7FmXE2X3i0Pl+sgl1Z\\\\\\\/M3qxkA1LNw8SwOpAhbaf3S45CjoEjIAsYyWjvbgMQ9KM7GGaJmR\\\\\\\/+9Uw4E6UQXgIdUeHBnFLmrfi2+9aEkYLfYa+avAUUpexc40nxOnITPwSN4l\\\\\\\/ZiXH59iQY1OlOv68OoMuESVL53oz8avXU3T6HRzkBLW66cq8\\\\\\\/NmgJLALq55gbVyb85V6ibRDZoRCbUaw0VCa07aXfw8PA89aVayehP9B8ck5B5JWKNGDrHe0cbA3GFeQtpVDZ0zQ1ZfDXDvA93FNICGAiwpKGfVZXZQ+mRSDTgVksrzFuOSFPX0kHAurcjayWUeZINI3jvJ+W\\\\\\\/hhAtW91Qf2TMOVOnpltHTLgo7s+yfdVc6J854kyv34D6RsP\\\\\\\/Dxsfnvgy\\\\\\\/4S1QsQWsoqyWth\\\\\\\/EKTcD9h2jMQRH5q\\\\\\\/d72urGQ1kpV1aSYt+cEJdX2K2OBEqbpia9Z73pFsHssIRwGydsiMxxn83KxeiR9DUZGb2f1mf7l2SBgW+ZCJqf\\\\\\\/rnL\\\\\\\/DFnmFAwzNDqfd5oSUEyodJKOEyaFC9JlAUn6gjqWZ+JCuSETgS76PlNAvOuT6fENuWntdECqpFclpJqs7a5VWMcbuDhPE5fSwwaYtKK+m\\\\\\\/w+u8Vffpla76QTv1bd139MVetpr\\\\\\\/\\\\\\\/\\\\\\\/3S1lAunw4XMwT5a+oeW4XxAVhIqFPCiG+xmhRm3rBcWgbwI+qIEPExqz9\\\\\\\/KNoOGW41gocQSrXSyJa6QDbDpZ7OnFrpiPMoaJiRX68ssIpbcurgicmlNQIzZKcIdBRtYaL5S9ER4Zum4k8StgGh9SuTLqdYMFwYWtVMjQjrAjh3h\\\\\\\/lGOEDCriEjSncAHWip3LoanEBbQS1JTGVV1aqcQfpJhcE5Gca7LmU7VljL2qI3yVmCZJCkrIQCKqgMlCef4zB+ewt3NVZmFTWtoO3YXNNfip3Ftb8euFXZ+Bja93DUAel+OFa3Cyx+aqwqYYQDPitX16BI5TbW1WjMUZHxN1N5jZ2Bn1EQo7ar8kW5R2gijO\\\\\\\/fm\\\\\\\/OnXrxL5fW8MD4qoO6OpdQgI7wOcYzuzw51I50liWL3HuW0r40fvWXsf4ZUZtTgI\\\\\\\/JA7Pe4CmRbhb30eSWhgakXcXcP\\\\\\\/CGzHjaMuKvSzi6WXelDdZzHEpJVfSzb6pbJENj1lQiP4Gq++HD1ACY8sxSlug9lhJnNXAHmmUQVIc2GA\\\\\\\/zMg7xLHwnRmRPXXQt2W+UWc9tX3931tWl40olf5g2oRgfqyqbz2k9f7HEGTVpkMUQ\\\\\\\/3Bp5Muqytyt84C26EgnuEvpQcYWvqsYbEeGgcVO+PAsnsRavmS5b\\\\\\\/EgqjYrQSiOFl+j1Nrr3oCgX7PeCfnAYColhmFb2uCZY1kR7hE=\\\u0022,\\\u0022iv\\\u0022:\\\u0022eckPI5kxmmBOuMWL6I0AvA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022183df9045288d83fb04611a8f4d4d627594cd6451f110b2e8249e8c3ab860049\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TNekNC68tMXGVmxFeGjlVlvwUUtz1JhnUsXDSLODyiFsW\\\\\\\/rjODTlBmjpW3+M7lparj8bzOP+O1OUdwq+xr9yMc+ihDGhCKopE4isOREZ0Xq01lv9Ha45I42ZX395BbDddFBUH0qwTmwp1Yx5PlUvX66lXqu2cRjCBTIY58Ff\\\\\\\/0iU9TPwiek6R\\\\\\\/YkmFDQeITvgsVaNWOAxbqEKqpZf\\\\\\\/xL8ESIM6lMx9UQlLLaHR2iaREPUN+XBWCjSdJAdlqF0FWLhqs0ocLEyh5dI9AQy8ZWiTbQwfobU7cO88fhbuEfyS70GkwmOoncXsydzgPbDLv+OxRDxR6veAjW04E4D947ZnQViVYigeMojuKd9l9pLDxOoNP14h6w7jU6SFkZkhb38+\\\\\\\/87E7OT2+gCH\\\\\\\/nslTDDa2Sxcm2WEuaix0udiN6cd3VG9gBYvCzqmTITUUXDzCoJFUEkrGE80teskxaCRy+XPO3lIE3JhJjC3JUWd04WlKLloNR1\\\\\\\/Qp1zehLjYCLEGIZ0YRRskhMYjaq968MZy9gAazf\\\\\\\/+hYaUWXrVN9lZE6A5+gIaVrKyBHzvJK9JjADgFK6uOVmM3MIWQjqwjPY\\\\\\\/4zksIglMuvLHmTdUjQIPAd3OOcHHXn5\\\\\\\/qt5pk2fFzGU+W+X\\\\\\\/K25Vy6cMuisJ0RTtIf9jF7sVXX\\\\\\\/ZV2IaotwDeGNA9Y6HBXSTDqaF7XL7Fe2z14fkDXuqAn93dMlrYJe7f5QHXdX3UWUKzX9i+QyKtvwF3cdFBDFhFVbnYrG76cxQA9ZjtzUU1fFjE\\\\\\\/kWwQGjcrW+ulDXM6ugtboohPG\\\\\\\/vz2au+6Hb3kPP+RALMsDTuYiGllUmBRhYt8RKPMy4MvQrtI\\\\\\\/8JpeDJANYflDx8BjKzxKo5mMvHQAvRnFy9H0ZLJsR6TXsygoahIZVz5PT5bzihuWKLZRArc6H5RqjOl+q04As8TNM4+xrul33+vEPjCPvA4NsljCU1v5SiJiGAXm9IBvTBa9AfRYqzXWyGFbT9FuW4HUp0Xw\\\\\\\/800rqnBuXHjKwbALXhtMO7k\\\\\\\/euSBUM6broF5shnxbg0VbgrOTN8AUbQ6dHs45p0I3xp8ppmmqb0M51P2+9Roc7XSEqjSEenDkW6dP9JjfIg7k+xnXXSi4Nemyr+KqwaLESfqhuySlxHCnCq4I8R8MJmHA6NjCtZ92ho4N\\\\\\\/IsTrHcY4gimEYYb0m3RiFy17XCAx5RBppCcPYAFwh57KBnf5qLCPC6OyhTqXYr1VGyBFXKNkQ5ErCNiQp6S+ZwLhYA5M38JmQUVgEZm7H3G05wzUgM8CFMCd08tJu\\\\\\\/d0DoePPKC2LxG1\\\\\\\/jIA9T6FECffuLYKdEeIAVICWqe3ccpt1SoHGBx6yiXBPjXXZmArMjC6LqJBjaWoSX56b1nTESwop5Wy2DMxVqUaJCYOxLwDUT+VOuCkkigcTDiQGeyqvwdPKSgDA9lv0v5k8Ay02NVdt\\\\\\\/0gakfKWNPjPSt6oIFILSt0yhMQ+ofgWua6hSPQ8XeQiqH\\\\\\\/65dVXMltL9rb2x0W8otBPfkzXdxeVaxfhVAGpalYxHl0nIXaaJwVsS6iqcApExTf0gNJ30V7JannID1CmrK8UP6v1BSs4ejASiM7o273+EY20hxzYbANH+I1K7O61ojIdMotQOX99YLBsyNPC0fkr5Y7sHki+Qfxd8pcCxGnSdFpQKZCWhXK6yE5gtJFHHPMSAsqAAt045jJva5zKx3VTNP6+dDfmAH8ShzvjsbD+DOhQl3tPHM7ExXQXdcUtFKgtNhWkD6i3yRXoAcMsoMzKfLlMFKgI6QjQEg8Ojw4bZBUmjXkoSByLg7NqO\\\\\\\/7oJqMxfjNGnl2xUEz6r8vNUb8S8iZeOHPhI6qxXutqRs8rqZ+a32f6ic+onlZpdoIC0+uUU\\\\\\\/5yzuDOPN\\\\\\\/cvMh8IV3vRX0cUrstKJ7VzSm1OjEwMfq5AHhnxPJ6OrlxyJlu3PX67iXHkRtONlZY23MYupO\\\\\\\/C4BWuNMfCji8H7EyuHNsgur0c1lvAbADc8\\\\\\\/LM3aqzwgxEU9fSurwHvjVrg6pyhq7HeCxGHG7EQouSuz4DE5rrOOWyKUIwKbZSBUISkzLlL5B3yeku4GjmFVbW4VNndHjqzhD\\\\\\\/QdCoGb8VOqgymU+kh2SEOqBrJw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022eqbrGpfL\\\\\\\/QPcJp52LuvxBQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022145e9b3e33c8be5fbb3561d786b318e332110b257b1cd827e9bb037bdf4637fe\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459690 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/10/1825" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PAfXXDOgWr158ZGyGzGo99yZie1j9zJDo7d59uuFup4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/10/1825" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-scYyYRg7gQuxpix3hkuG2A9wonMt6XA2iLN4HSQO1fg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/10/1813" title="Widespread Genomic Incompatibilities in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/10/1837" title="An X-Linked Sex Ratio Distorter in Drosophila simulans That Kills or Incapacitates Both Noncarrier Sperm and Sons" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459690" id="node459690" data-pisa="ggg;4/10/1825" data-pisa-master="ggg;g3.114.011262" data-apath="/ggg/4/10/1825.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sensory Rewiring in an Echolocator: Genome-Wide Modification of Retinogenic and Auditory Genes in the Bat <em>Myotis davidii</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nicholas J. Hudson</span>, <span class="highwire-citation-author" data-delta="1">Michelle L. Baker</span>, <span class="highwire-citation-author" data-delta="2">Nathan S. Hart</span>, <span class="highwire-citation-author" data-delta="3">James W. Wynne</span>, <span class="highwire-citation-author" data-delta="4">Quan Gu</span>, <span class="highwire-citation-author" data-delta="5">Zhiyong Huang</span>, <span class="highwire-citation-author" data-delta="6">Guojie Zhang</span>, <span class="highwire-citation-author" data-delta="7">Aaron B. Ingham</span>, <span class="highwire-citation-author" data-delta="8">Linfa Wang</span> and <span class="highwire-citation-author" data-delta="9">Antonio Reverter</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1825-1835;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011262 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459690" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas J. Hudson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational and Systems Biology, CSIRO Agriculture Flagship, Queensland Bioscience Precinct, Brisbane, Queensland, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BJ.%2BHudson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hudson%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BJ.%2BHudson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nick.hudson@csiro.au" class="" data-icon-position="" data-hide-link-title="0">nick.hudson@csiro.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michelle L. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Australian Animal Health Laboratory, CSIRO Biosecurity Flagship, Geelong, Victoria, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michelle%2BL.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichelle%2BL.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nathan S. Hart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Animal Biology and the Oceans Institute, University of Western Australia, Crawley, WA 6009, Western Australia, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathan%2BS.%2BHart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hart%20NS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANathan%2BS.%2BHart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James W. Wynne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Australian Animal Health Laboratory, CSIRO Biosecurity Flagship, Geelong, Victoria, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BW.%2BWynne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wynne%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BW.%2BWynne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Quan Gu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>College of Information Sciences and Technology, Donghua University, Shanghai 201620, China</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Faculty of Medicine, Imperial College London, South Kensington, London SW7 2AZ, England</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Quan%2BGu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQuan%2BGu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zhiyong Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>BGI-Shenzhen, Shenzhen, 518083, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiyong%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhiyong%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Guojie Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>BGI-Shenzhen, Shenzhen, 518083, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guojie%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuojie%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aaron B. Ingham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational and Systems Biology, CSIRO Agriculture Flagship, Queensland Bioscience Precinct, Brisbane, Queensland, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BB.%2BIngham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ingham%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BB.%2BIngham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Linfa Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Australian Animal Health Laboratory, CSIRO Biosecurity Flagship, Geelong, Victoria, Australia</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Program in Emerging Infectious Diseases, Duke-NUS Graduate Medical School, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linfa%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinfa%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Antonio Reverter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational and Systems Biology, CSIRO Agriculture Flagship, Queensland Bioscience Precinct, Brisbane, Queensland, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BReverter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reverter%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonio%2BReverter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/10/1825" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459690" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459690/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/10/1825.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459690" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459690/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/10/1825.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459690" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459690/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/10/1825.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459690" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459690/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/10/1825.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459690" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459690/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Bats comprise 20% of all mammalian species and display a number of characteristics, including true flight, echolocation, and a heightened ability to resist viral load that uniquely position this group for comparative genomic studies. Here we searched for evidence of genomic variation consistent with sensory rewiring through bat evolution. We focused on two species with divergent sensory preferences. <em>Myotis davidii</em> is a bat species that echolocates and possesses dim- but not daylight-adapted vision whereas the black flying fox (<em>Pteropus alecto)</em> has highly developed day vision but does not echolocate. Using the naked mole rat as a reference, we found five functional genes (<em>CYP1A2</em>, <em>RBP3</em>, <em>GUCY2F</em>, <em>CRYBB1</em>, and <em>GRK7</em>) encoding visual proteins that have degenerated into pseudogenes in <em>M. davidii</em> but not <em>P. alecto</em>. In a second approach genome-wide codon usage bias (CUB) was compared between the two bat species. This CUB ranking systematically enriched for vision-related (<em>CLN8</em>, <em>RD3</em>, <em>IKZF1</em>, <em>LAMC3</em>, <em>CRX</em>, <em>SOX8</em>, <em>VAX2</em>, <em>HPS1</em>, <em>RHO</em>, <em>PRPH2</em>, and <em>SOX9</em>) and hearing-related (<em>TPRN</em>, <em>TMIE</em>, <em>SLC52A3</em>, <em>OTOF</em>, <em>WFS1</em>, <em>SOD1</em>, <em>TBX18</em>, <em>MAP1A</em>, <em>OTOS</em>, <em>GPX1</em>, and <em>USH1G</em>) machinery in <em>M. davidii</em> but not <em>P. alecto</em>. All vision and hearing genes selectively enriched in <em>M. davidii</em> for which orthologs could be identified also were more biased in the echolocating <em>M. lucifugus</em> than the nonecholocating <em>P. vampyrus</em>. We suggest that the existence of codon bias in vision- and hearing-related genes in a species that has evolved echolocation implies CUB is part of evolution’s toolkit to rewire sensory systems. We propose that the two genetic changes (pseudogene formation and CUB) collectively paint a picture of that incorporates a combination of destruction and gain-of-function. Together, they help explain how natural selection has reduced physiological costs associated with the development of a smaller eye poorly adapted to day vision but that also contribute to enhanced dim light vision and the hearing adaptations consonant with echolocation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/evolution" class="hw-term hw-article-keyword hw-article-keyword-evolution" rel="nofollow">evolution</a></li><li class="kwd"><a href="/keyword/vision" class="hw-term hw-article-keyword hw-article-keyword-vision" rel="nofollow">vision</a></li><li class="kwd"><a href="/keyword/codon-usage-bias" class="hw-term hw-article-keyword hw-article-keyword-codon-usage-bias" rel="nofollow">codon usage bias</a></li><li class="kwd"><a href="/keyword/pseudogene" class="hw-term hw-article-keyword hw-article-keyword-pseudogene" rel="nofollow">pseudogene</a></li><li class="kwd"><a href="/keyword/bat" class="hw-term hw-article-keyword hw-article-keyword-bat" rel="nofollow">bat</a></li></ul><p id="p-4">As pointed out previously (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jones <em>et al.</em> 2013</a>), bats perceive the world using a range of sensory mechanisms, some of which have undergone specialization (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altringham and Fenton 2003</a>). <em>Myotis davidii</em> and <em>Pteropus alecto</em> are Chiropterans that use different habitats, prefer different diets, and occupy different sensory niches (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Zhang <em>et al.</em> 2013</a>). Previous work has compared and contrasted their genomes to shed light on the genetic basis on a host of their phenotypic similarities and differences. Recently, a new connection was drawn between bat immunity and bioenergetic demands based on gene family expansions and contractions and other lines of genetic evidence (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Zhang <em>et al.</em> 2013</a>). In taking a sensory perspective of the two species, <em>P. alecto</em> forages through vision and smell but cannot echolocate. In contrast, <em>M. davidii</em> is an echolocator thought to have relatively poor day vision. It exhibits the loss of function often observed when biological systems avoid paying the cost of unnecessary capacity (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Weibel <em>et al.</em> 1991</a>).</p><p id="p-5">Basic morphological data in echolocating bats—such as differences in the lens, cornea, vitreous body shape and proportion of rods and cones—are consistent with reduced visual acuity (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bhatnagar 1975</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Pettigrew 1986</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fuzessery <em>et al.</em> 1993</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kalko <em>et al.</em> 1996</a>). A complication to the sensory ecology of myotid bats is that although they possess a smaller eye, this does not exclusively imply degradation. Recent work in <em>M. brandtii</em> suggests this echolocator may be adapted to dim light conditions such as found at dawn and dusk and also ultraviolet light (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Seim <em>et al.</em> 2013</a>). These visual adaptations are related to photopigment opsins, rhodopsin, and downstream signaling components (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Seim <em>et al.</em> 2013</a>). The molecular basis of visual adaptation in <em>M. davidii per se</em> is unknown.</p><p id="p-6">Taking a broader sensory perspective, a very recently published analysis based on the ratio of nonsynonymous to synonymous substitutions common to both echolocating dolphins and echolocating bats identified genes encoding sensory proteins, particularly vision and hearing, as striking signatures of convergent evolution (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Parker <em>et al.</em> 2013</a>). The hearing adaptation relates to the echolocation facility, a multimodal capability comprising both specialized vocalization on the one hand and signal reception on the other. A recent review of the genetic basis of the sensory ecology of echolocating bats can be found here in <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Jones <em>et al.</em> (2013)</a>. Genes thought to be of particular interest for echolocation <em>per se</em> include those encoding proteins related to vocalization (<em>FOXP2</em>) and enhanced hearing (<em>SLC26A5</em>).</p><p id="p-7">There is some debate over whether bat echolocation is (1) a derived characteristic not present in the common ancestor that has subsequently evolved in multiple echolocating lineages by convergent evolution, or (2) was present in the ancestral species then selectively lost (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Teeling 2009</a>). Either way, our aim is to contribute to the elucidation of the genetic basis of sensory rewiring, with a focus on these two species. We compare the output to two other bat species for whom genome-wide data already exist, the echolocator <em>M. lucifugus</em> and the nonecholocator <em>P. vampyrus</em>. Pteropodids diverged from other bats 55 mya, whereas the myotid bats are considered by some authorities to be a more recent lineage that split 15 mya. Assembled genomes for these four bats provide great scope for comparative genomics. Indeed, it has been pointed out that the future for understanding the molecular basis of sensory biology is promising, with comparative genomics having great potential (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Jones <em>et al.</em> 2013</a>). Here, we find a combination of hypothesis-driven and hypothesis-free approaches to be mutually reinforcing, shedding molecular insight into the apparent rewiring of <em>M. davidii</em>’s visual modality, and also complementary changes in the hearing modality that we suggest likely relate to echolocation.</p><p id="p-8">Our motivation for exploring genome-wide patterns of codon usage bias (CUB) in the context of comparative mammalian evolution was our recent finding that pathway-level alterations in translation efficiency appear to have an underappreciated role in driving evolutionary change through gain of function, even in complex eukaryotes such as mammals and birds (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hudson <em>et al.</em> 2011</a>). We previously observed systematic CUB in genes fundamental to important characteristics of each lineage, such as cell wall and chloroplast physiology in plants, mitochondrial function in birds, and hair formation in mammals (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Hudson <em>et al.</em> 2011</a>). Other recent work also has reinforced a likely role for CUB in human evolution. For example, a large scale tissue-specific analysis in humans previously found that the tRNA adapatation index significantly correlated with expression levels both globally and in a range of tissues taken in isolation (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Waldman <em>et al.</em> 2010</a>). These lines of evidence support a role for CUB in mammalian evolution, but genome-wide CUB analysis has never previously been applied to any bat species.</p><p id="p-9">Here, we apply a very sensitive CUB detection tool that, when applied genome-wide, finds a coordinated pattern of enrichment for genes encoding vision and hearing-related machinery in the echolocating <em>M. davidii</em> but not the nonechlocating fruitbat <em>P. alecto</em>. Orthologs of those vision and hearing genes also were more biased in <em>M. lucifugus</em> than <em>P. vampyrus</em>, although the effect was not so strong. The outlier visual proteins tend to be highly expressed in human eye tissue but this is not always the case, implying no simple relationship to abundance. Also based on comparative human data, some of these genes are expected to have highly restricted tissue-specific expression patterns, whereas others are widely expressed. Given observations we have made applying the same method in a number of other genomes, we submit that the most likely explanation to be selection on translation efficiency that manifests at the pathway level. We propose that the coordinate impact of these changes (pseudogenes and CUB) is sensory rewiring, presumably helping to manipulate the <em>Myotis</em> retinogenic pathway toward the development of a smaller, low light-adapted eye with additional changes relating to the enhanced hearing that contributes to the echolocation facility.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Hypothesis-driven pseudogene analysis</h3><p id="p-10">A list of vision-related genes obtained from the gene ontology database (GO:0007601: visual perception) and visual system from the QIAGEN pathway library (<a href="http://www.qiagen.com/Products/Genes%20and%20Pathways/">http://www.qiagen.com/Products/Genes%20and%20Pathways/</a>) defined presence, copy number, and nature of corresponding genes in the <em>M. davidii</em> and <em>P. alecto</em> genomes. Proteins from human vision-related genes and their neighboring genes were downloaded from Ensembl release 64. The longest transcript was chosen to represent each gene in cases when alternative splicing variants have been shown to exist. We then subjected all proteins to tBlastn analysis against the bat genomes with the similarity cutoff threshold of e-value = 1e<sup>−5</sup>. With the human protein set as a reference, we found the best hit for each human protein in the two bat genomes by using the criteria that more than 30% of the aligned sequence showed an identity above 30%. We used GeneWise algorithm (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Birney <em>et al.</em> 2004</a>) (with parameters -genesf -for -quiet) to define the detailed exon-intron structure of each bat gene and to identify potential pseudogenes among the bat visual genes.</p><p id="p-11">Initially, we targeted the examination of the bat orthologs for vision-related pseudogenes identified in the naked mole rat <em>Heterocephalus glaber</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2011</a>), a species with poor vision adapted to life in the dark. Genes containing frame shifts or premature stop codons were considered candidates. We filtered as follows: (1) To avoid reported frame shifts or premature stop codons that were due to a flaw in the GeneWise algorithm, we aligned all human proteins to their corresponding loci in the human genome, and genes with frame shifts or premature stop codons in human-to-human alignments were filtered; (2) Using the results of the human-to-human alignment from GeneWise, candidate pseudogenes with obvious splicing errors near their frame shifts or premature stop codons were filtered; (3) Candidate pseudogenes with a low number of reads covering their frame shift or premature stop codon sites were considered assembly errors. Genes with a considerable number of reads due to genotype variation at these sites were treated as heterozygous and filtered.</p></div><div id="sec-3" class="subsection"><h3>Hypothesis-free genome-wide CUB analysis</h3><p id="p-12">A number of CUB statistics have been proposed, and there is little consensus on the optimal approach. A very recent review outlining the pros and cons of some commonly applied CUB quantification approaches can be found here in <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Behura and Severson (2013)</a>. Our CUB statistic uses the concept of entropy, which estimates data regularities based on a combination of order and proportion. Its strength lies in its broad applicability and sensitivity. Because it is not systematically influenced by gene length and accounts for amino acid composition, it can be fairly applied within and between genomes. It does not account, however, for background GC content, nor does it determine whether bias is toward preferred or nonpreferred codons. For this reason, background GC content and a more detailed analysis of codon usage must be retrospectively assessed on a case-by-case basis (<em>i.e.</em>, in those genes whose entropic properties have been determined as extreme and therefore worthy of deeper investigation).</p><p id="p-13">To summarize, we determined the extent of CUB for every coding sequence using an information theory-based statistic as previously described (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Hudson <em>et al.</em> 2011</a>). For each observed coding sequence, we computed the entropy of the nucleic acid and compared it against the average entropy of 20 random sequences. The random sequences were generated in a way that they would encode the same amino acid sequence, but where the codons were selected at random. Because entropy provides a measure of data regularity, the differential entropy between the observed sequence and the random corresponds to the extent of regularity attributable to CUB.</p><p id="p-14">We first ranked all coding sequences on extent of CUB on a within-genome basis. To ascertain the extent of functional enrichment analysis the ranked lists for <em>Myotis</em> and <em>Pteropus</em> were subjected to the GOrilla webtool (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Eden <em>et al.</em> 2009</a>). GOrilla uses hypergeometric statistics to determine whether particular biological processes are enriched within any ranked input set of genes. We used the single ranked list option based on human functional annotation, and imported SWISSPROT IDs, which is one of the designated preferred input identifiers for GOrilla analysis. We also compared relative codon bias in the orthologs in common between the two species (of which we identified 6748). This secondary analysis helps identify genes that may not be extreme on a within genome basis but still possess markedly different properties between the two species.</p><p id="p-15">Because the GOrilla enrichment relies on incomplete annotation of functional processes and can overlook functional connections even in well-annotated genes, we used the top enrichment as a focus for further manual curation, based on mining the PubMed literature database. A subset of outlier CUB genes were retrospectively assessed for regional GC content. This was computed from the gene itself, plus 10 kb upstream from the start codon and 10 kb downstream from the stop codon.</p><div id="sec-4" class="subsection"><h4>Independent validation of divergent CUB:</h4><p id="p-16">The differential entropy statistic is one of many ways of measuring CUB. In an effort to provide an independent line of evidence for differential CUB in the two bat species and subsequent differential functional enrichment, we also ran the previously published effective numbers of codons (ENC) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wright 1990</a>) through the CodonW website: <a href="http://codonw.sourceforge.net/">http://codonw.sourceforge.net/</a>. According to <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Behura and Severson (2013)</a>, ENC is used to measure how far the codon usage of a gene departs from equal usage of synonymous codons. EC values range from 20 (extreme codon bias when one codon is used exclusively for each amino acid) to 61 (in cases of no codon bias where synonymous codons are equally likely to code the amino acid).</p></div><div id="sec-5" class="subsection"><h4>CUB outlier genes quality checking:</h4><p id="p-17">Because it is possible that gaps in the genome assembly data can result in artifacts with the cds sequences, the cds sequence data for all <em>M. davidii</em> CUB outlier genes were confirmed by comparison with cds sequences from other species. We used translation BLAST to identify potential assembly problems, including gaps in our cds sequences. The bats <em>M. brandtii</em> and <em>M. lucifugus</em> plus other mammalian species available in Genbank were used for sequence alignment. In the event of potential missing exons we cross-referenced back to the <em>M. davidii</em> genomic assembly to confirm the quality of genomic sequence data.</p></div><div id="sec-6" class="subsection"><h4>CUB analysis in two additional bat species: M. lucifugus and P. vampyrus:</h4><p id="p-18">To detect comparative evidence for CUB in the sensory apparatus in echolocating bats, we computed CUB in an additional echolocator (<em>M. lucifugus</em>) and nonecholocator (<em>P. vampyrus</em>). The current genome assemblies from the Ensemble database for <em>P. vampyrus</em> (pteVam1) and <em>M. lucifugus</em> (Myoluc2.0) were used for this analysis. For these two comparison species, we obtained coding sequence data from Biomart, yielding 14,141 annotated cds for <em>M. lucifugus</em> and 8, 845 for <em>P. vampyrus</em>. CUB was estimated using the entropy statistic as previously described.</p></div></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Hypothesis-driven pseudogene analysis</h3><p id="p-19">Of the 19 visual perception genes identified as inactivated or missing in the naked mole rat, we identified five that have also become pseudogenes in <em>M. davidii</em> but not <em>P. alecto (CYP1A2</em>, <em>RBP3</em>, <em>GUCY2F</em>, <em>CRYBB1</em>, and <em>GRK7</em>; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Taxonomically the naked mole rat <em>H. glaber</em> is placed within rodents. Despite some exceptional traits the overall properties of the <em>H. glaber</em> genome are similar to other mammals (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2011</a>) making the comparison to the two bat species reasonable.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene structure, partial amino acid ,and nucleotide alignments for five M. davidii pseudogenes. Red triangles illustrate the location of stop codons compared with P. alecto. Amino acid alignment surrounding the stop codon region in P. alecto and M. davidii. Asterisks represent stop codons. Nucleotide alignment between P. alecto and M. davidii corresponding to the same region in amino acid alignment. Gray nucleotides represent the region after a frame shift in M. davidii that leads to a premature stop codon (red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-572324817" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gene structure, partial amino acid ,and nucleotide alignments for five &lt;em&gt;M. davidii&lt;/em&gt; pseudogenes. Red triangles illustrate the location of stop codons compared with &lt;em&gt;P. alecto&lt;/em&gt;. Amino acid alignment surrounding the stop codon region in &lt;em&gt;P. alecto&lt;/em&gt; and &lt;em&gt;M. davidii&lt;/em&gt;. Asterisks represent stop codons. Nucleotide alignment between &lt;em&gt;P. alecto&lt;/em&gt; and &lt;em&gt;M. davidii&lt;/em&gt; corresponding to the same region in amino acid alignment. Gray nucleotides represent the region after a frame shift in &lt;em&gt;M. davidii&lt;/em&gt; that leads to a premature stop codon (&lt;em&gt;red&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1825/F1.medium.gif" width="440" height="341"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/10/1825/F1.medium.gif" width="440" height="341"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465017" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Gene structure, partial amino acid ,and nucleotide alignments for five <em>M. davidii</em> pseudogenes. Red triangles illustrate the location of stop codons compared with <em>P. alecto</em>. Amino acid alignment surrounding the stop codon region in <em>P. alecto</em> and <em>M. davidii</em>. Asterisks represent stop codons. Nucleotide alignment between <em>P. alecto</em> and <em>M. davidii</em> corresponding to the same region in amino acid alignment. Gray nucleotides represent the region after a frame shift in <em>M. davidii</em> that leads to a premature stop codon (<em>red</em>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">For <em>CYP1A2</em>, <em>CRYBB1</em>, and <em>RBP3</em> single or multiple nucleotide deletions were responsible for frame-shift mutations leading to premature stop codons in <em>M. davidii</em>. For <em>CRYBB1</em>, the deletion was 31 nucleotides in the first exon compared with <em>P. alecto</em>. In contrast, the <em>GRK7</em> and <em>GUCY2F</em> pseudogenes were caused by single-point mutations that created single stop codons. In the case of <em>GUCY2F</em> two point mutations had created two stop codons in close vicinity to each other. The similarity between the intact <em>P. alecto</em> genes compared to their corresponding regions in the <em>M. davidii</em> genome, ranged from 85 to 90%.</p></div><div id="sec-9" class="subsection"><h3>Hypothesis-free genome-wide CUB analysis</h3><p id="p-22">We calculated codon usage preferences for the two species genome-wide (Table <a id="xref-fig-2-1" class="xref-fig" href="#F2">2</a> and Table <a id="xref-fig-3-1" class="xref-fig" href="#F3">3</a>). These genome-wide figures can be used to help establish preferred and nonpreferred codons for that particular species in concert with the available tRNA pool. In turn this helps shed light on the translational implications of CUB. Namely, is CUB likely to increase or decrease the energetic cost and efficiency of translation for the gene in question?</p><p id="p-23">We next used a hypothesis-free screen, based on CUB, to test for pathway functional enrichment. Before functional enrichment, GOrilla screens the imported gene list, removes duplicates (keeping the only the most highly ranked candidate), then assigns GO characteristics where those exist. For <em>M. davidii</em> we imported 13,654 genes that could be assigned SWISS PROT identifiers, of which 37% were recognized. A total of 6951 were recognized by Gene Symbol, 1731 duplicates were removed, and of the remaining 5220, 5056 had a GO term assigned. For <em>P. alecto</em> we imported 12,188 genes that could be assigned to SWISS PROT identifiers of which 40% were recognized. A total of 6183 were recognized by Gene Symbol, 1056 duplicates were removed, and of the remaining 5127, 4956 had a GO term assigned.</p><p id="p-24"><em>NEFH</em> (alias <em>YHU2</em>, encoding the neurofilament heavy polypeptide) possessed extreme CUB in both species. Consistent with previous research in other organisms, the translation initiation factor <em>IF2</em> (<em>PRP2</em>) was among the most biased genes in both species.</p><p id="p-25">Of the biased genes within <em>M. davidii</em> we found the top ‘Process’ enrichment to be “<em>retina development of camera type eye</em>” (hypergeometric test <em>P</em>-value = 0.0000775; False Discovery Rate q-value = 0.766) based on the prominent bias of the following genes: <em>CLN8</em>, <em>RD3</em>, <em>IKZF1</em>, <em>LAMC3</em>, <em>CRX</em>, <em>SOX8</em>, <em>VAX2</em>, <em>HPS1</em>, <em>RHO</em>, <em>PRPH2</em>, <em>SOX9</em>; <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011262/-/DC1/011262SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011262/-/DC1/FileS1.xlsx">File S1</a>). The remaining functional enrichments reported in descending order by GOrilla for <em>M. davidii</em> were “<em>O-glycan processing</em>” (<em>P</em>-value = 0.000127), “<em>protein O-linked glycosylation</em>” (<em>P</em>-value = 0.000393), “<em>adenylate cyclase-inhibiting G-protein coupled receptor signalling pathway</em>” (<em>P</em>-value = 0.000508), “<em>renal absorption</em>” (<em>P</em>-value = 0.00051), “<em>renal water transport</em>” (<em>P</em>-value = 0.00051), “<em>peripheral nervous system neuron axonogenesis</em>” (<em>P</em>-value = 0.000593), “<em>positive regulation of transcription from RNA polymerase II promoter in response to stress</em>” (<em>P</em>-value = 0.000791), “<em>sensory perception of light stimulus</em>” (<em>P</em>-value = 0.000853), “<em>sensory perception of sound</em>” (<em>P</em>-value = 0.000932), and “<em>negative regulation of neutrophil differentiation</em>” (<em>P</em>-value = 0.000989).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genes showing extreme codon bias (negative differential entropy) on a within-genome basis in (A) M. davidii and (B) P. alecto, with those encoding vision and hearing related proteins highlighted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-572324817" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genes showing extreme codon bias (negative differential entropy) on a within-genome basis in (A) &lt;em&gt;M. davidii&lt;/em&gt; and (B) &lt;em&gt;P. alecto&lt;/em&gt;, with those encoding vision and hearing related proteins highlighted.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1825/F2.medium.gif" width="336" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/10/1825/F2.medium.gif" width="336" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464933" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Genes showing extreme codon bias (negative differential entropy) on a within-genome basis in (A) <em>M. davidii</em> and (B) <em>P. alecto</em>, with those encoding vision and hearing related proteins highlighted.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">The ‘<em>sensory perception of light stimulus</em>’ contained the following additional genes not recognized in the ‘<em>retina development</em>’ function, but which play a related physiological role: <em>OPN4</em>, <em>NR2E3</em>, <em>CNGB1</em>, <em>RDH8</em>, <em>ROM1</em>, <em>CLN6</em>, <em>WFS1</em>, <em>KIFC3</em>, <em>RGR</em>, <em>RP1L1</em>, <em>RDH5</em>, <em>CABP4</em>, <em>AIPL1</em>, <em>RAX2</em>, <em>NRL and USH1G</em>. The sensory perception of sound comprised the following genes (<em>TPRN</em>, <em>TMIE</em>, <em>SLC52A3</em>, <em>OTOF</em>, <em>WFS1</em>, <em>SOD1</em>, <em>TBX18</em>, <em>MAP1A</em>, <em>OTOS</em>, <em>GPX1</em> and <em>USH1G</em>). Overall, we considered the collective presence of 4 functions in <em>M. davidii</em> (retina development, peripheral axonogenesis, light perception and sound perception) clearly associated with sensory rewiring to be noteworthy.</p><p id="p-28">There was no formal vision related enrichment detected for <em>P. alecto</em> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>) or indeed of any other sensory rewiring such as sound perception. The top ‘Biological Process’ enrichments in descending order were “<em>cellular developmental process</em>” (<em>P</em>-value = 0.0000178), “<em>regulation of intrinsic apoptotic signalling pathway</em>” (<em>P</em>-value = 0.000559), “<em>G-protein coupled receptor signalling pathway</em>” (<em>P</em>-value = 0.000634), “<em>developmental process</em>” (<em>P</em>-value = 0.000192), “<em>regulation of interferon-gamma biosynthetic process</em><strong>”</strong> (<em>P</em>-value = 0.000267), “<em>positive regulation of MAPK cascade</em>” (<em>P</em>-value = 0.000354), “<em>cell differentiation</em>” (<em>P</em>-value = 0.000381), “<em>cell surface receptor signalling pathway</em>” (<em>P</em>-value = 0.00056),“<em>regulation of MAPK cascade</em>” (<em>P</em>-value = 0.000652), “<em>regulation of interferon-gamma production</em>” (<em>P</em>-value = 0.000703) and “<em>blood vessel development</em>” (<em>P</em>-value = 0.000813). In both species, there was some indication of enrichment relating to immune function.</p><p id="p-29">When we compared these orthologs of the two genomes directly on a between-genome basis (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), <em>IKZF1</em> (also identified in the “retina development of the camera eye”) displayed a prominent outlier value in <em>M. davidii</em>. <em>LAMC3</em>, <em>RHO</em>, and <em>SOX9</em> were genes annotated as vision related that were absent in the <em>P. alecto</em> annotation and so orthologs could not be directly compared. A number of other orthologs were found to possess more bias in Myotis than in Pteropus, some of which were manually explored for their possible connection to the visual system. Although not identified as relating to hearing by GOrilla enrichment, the outlier position of <em>MUC19</em> is of interest, given (1) the general role of mucin family members in signal transduction during hearing (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kerschner 2007</a>) and (2) that <em>MUC19</em> specifically encodes a major gel-forming mucin expressed in the middle ear (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kerschner <em>et al.</em> 2009</a>). However, the mucins are known to bind pathogens, so an immune function or some other role cannot be discounted.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Orthologs showing differential codon bias (distance from diagonal) between M. davidii and P. alecto on a between-genome basis. Genes encoding vision and hearing elated proteins are highlighted. Of the sensory genes IKZF1 is particularly prominent. The highly codon biased LAMC3, RHO, and SOX9 identified by GOrilla as vision-related in M. davidii appear to be absent in P. alecto. MUC19, highly biased in M. davidii, is a Mucin family member expressed in the inner ear. NEFH is equally highly biased in both bat species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-572324817" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Orthologs showing differential codon bias (distance from diagonal) between &lt;em&gt;M. davidii&lt;/em&gt; and &lt;em&gt;P. alecto&lt;/em&gt; on a between-genome basis. Genes encoding vision and hearing elated proteins are highlighted. Of the sensory genes &lt;em&gt;IKZF1&lt;/em&gt; is particularly prominent. The highly codon biased &lt;em&gt;LAMC3&lt;/em&gt;, &lt;em&gt;RHO&lt;/em&gt;, and &lt;em&gt;SOX9&lt;/em&gt; identified by GOrilla as vision-related in &lt;em&gt;M. davidii&lt;/em&gt; appear to be absent in &lt;em&gt;P. alecto&lt;/em&gt;. &lt;em&gt;MUC19&lt;/em&gt;, highly biased in &lt;em&gt;M. davidii&lt;/em&gt;, is a Mucin family member expressed in the inner ear. &lt;em&gt;NEFH&lt;/em&gt; is equally highly biased in both bat species.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1825/F3.medium.gif" width="440" height="265"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/10/1825/F3.medium.gif" width="440" height="265"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465081" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Orthologs showing differential codon bias (distance from diagonal) between <em>M. davidii</em> and <em>P. alecto</em> on a between-genome basis. Genes encoding vision and hearing elated proteins are highlighted. Of the sensory genes <em>IKZF1</em> is particularly prominent. The highly codon biased <em>LAMC3</em>, <em>RHO</em>, and <em>SOX9</em> identified by GOrilla as vision-related in <em>M. davidii</em> appear to be absent in <em>P. alecto</em>. <em>MUC19</em>, highly biased in <em>M. davidii</em>, is a Mucin family member expressed in the inner ear. <em>NEFH</em> is equally highly biased in both bat species.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">The overall distribution of these data was not symmetrical. There were more relatively biased orthologs in <em>M. davidii</em> than in <em>P. alecto</em>. This may reflect the fact that the Myotis bats are a much more recent lineage that, according to some sources, diverged approximately 15 mya from other bat species. Alternatively, the skewed data may reflect the fact that <em>M. davidii</em> possesses more derived phenotypic characteristics than <em>P. alecto</em>, resulting from the evolution of enhanced echolocation in this species. The genome-wide GC content is similar in the two bat species, with <em>P. alecto</em> (<em>39.7%</em>) slightly more biased than <em>M. davidii</em> (42.7%). This finding implies genome-wide GC content does not account for the slight skew in the between species distribution.</p><p id="p-32">It is clear from scrutinizing the plots that the CUB statistic has the appealing feature of being independent of gene length because the mass of the data in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> follows a horizontal line in both species. It is also immediately apparent that, sitting close to the horizontal line, the vast majority of genes in both bat species do not exhibit extreme CUB. Nevertheless, in both cases there are a small subset of genes, many of which are short and some of which are long, that possess properties of relatively extreme CUB. Moreover, this subset enriches for a particular biological function, even though the physical location, gene length, and other molecular details vary. This observation can be generalized. We detected the exact same broad patterns previously identified in six other eukaryotic genomes: Arabidopsis, <em>S. cerevisiae</em>, <em>C. elegans</em>, chicken, chimpanzee, and human (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Hudson <em>et al.</em> 2011</a>) <em>i.e.</em>, whereas most genes are unbiased, a small subset of variable length are biased and these enrich for particular functions that appear relevant to that lineage.</p><p id="p-33">We calculated the GC content up and downstream of the differentially biased <em>IKZF1</em>, <em>GPSM1</em>, and <em>SPRR1</em> (cornifin) and found them to be either almost identical, or more out of equilibrium in the Pteropodid genome. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> summarizes these findings. Therefore, regional background GC content is unable to account for the greater CUB in these Myotid coding sequences. Gene length changes between species is also unable to account for the observation of differential CUB, as although it varies, the variation is not systematic. <em>M. davidii IKZF1</em> is longer than in <em>P. alecto</em>, <em>GPSM1</em> is shorter, and <em>SPRR1</em> is almost identical, whereas CUB is substantially more extreme in <em>M. davidii</em> in all three cases.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465067/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465067/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The genomic properties of three vision-related genes showing more CUB in <em>M. davidii</em> than <em>P. alecto</em></span><div class="sb-div caption-clear"></div></div></div><div id="sec-10" class="subsection"><h4>Independent validation of divergent CUB:</h4><p id="p-35">The ENC measure of genome-wide CUB reinforced the change in the visual system of <em>M. davidii</em> and partially reinforced the change in its auditory system compared with <em>P. alecto</em>. However, the sensory enrichments observed for <em>M. davidii</em> were no longer ranked among the top ten on a within-genome basis.</p><p id="p-36">The specific functional enrichments “retina development in camera-type eye” (based on <em>CLN8</em>, <em>BAX</em>, <em>PRPH2</em>, <em>LAMC3</em>, <em>IKZF1</em>, <em>SOX9</em>, <em>NPHP4</em>, <em>PFDN5</em>, <em>PVRL1</em>, <em>SOX8</em>, <em>RD3</em>) and “sensory perception of sound” (based on <em>ASIC2</em>, <em>TPRN</em>, <em>MYO15A</em>, <em>WFS1</em>, <em>SOBP</em>, <em>ESPN</em>, <em>GPX1</em>, <em>TIMM13</em>, <em>GRM7</em>, <em>TMIE</em>, <em>SLC52A3</em>, <em>OTOF</em>, <em>SOD1</em>, <em>MAP1A</em>, <em>OTOS</em>, <em>SIX1</em>, <em>USH1G</em>, <em>MYO7A</em>) were both significant in <em>M. davidii</em> (<em>P</em> = 0.000294 and <em>P</em> = 0.00066, respectively) but not <em>P. alecto</em>, broadly reinforcing the patterns identified by the original analysis performed using the entropy-based statistic. Interestingly, the Myotid bat also had the functional enrichment (<em>P</em> = 0.000232) “sensory perception of mechanical stimulus” (based on <em>ASIC2</em>, <em>TPRN</em>, <em>MYO15A</em>, <em>TSHZ3</em>, <em>WFS1</em>, <em>SOBP</em>, <em>GPX1</em>, <em>TIMM13</em>, <em>GRM7</em>, <em>TMIE</em>, <em>SLC52A3</em>, <em>OTOF</em>, <em>SOD1</em>, <em>MAP1A</em>, <em>OTOS</em>, <em>SIX1</em>, <em>USH1G</em>, <em>MYO7A</em>). The overlap in content with “sensory perception of sound” given genes such as <em>TMIE</em> (<em>trans</em>-membrane of inner ear), and components of the otolith (<em>OTOF</em> and <em>OTOS</em>) suggests this function is also essentially related to inner ear development.</p><p id="p-37">Vision enrichments for <em>P. alecto</em> are present but to a lesser extent and do not convincingly highlight sensory rewiring. Although <em>P. alecto</em> does have vision-related enrichments given the presence of “closure of optic fissure” (<em>P</em> = 0.00026) “positive regulation of lens epithelial cell proliferation” (<em>P</em> = 0.00026), and “regulation of lens epithelial cell proliferation” (<em>P</em> = 0.00026), these were all based on the ranked position of a single gene, <em>SOX11</em>. Finally, “Inner ear development” (based on <em>PDGFA</em>, <em>CCM2</em>, <em>CYTL1</em>, <em>HPCA</em>, <em>LHX3</em>, <em>GATA3</em>, <em>SHH</em>, <em>CEBPA</em>, <em>SOX2</em>, and <em>SIX1</em>) was enriched in Pteropus (<em>P</em> = 0.000816). The genes comprising the hearing enrichment in <em>P. alecto</em> are different to those highlighted in <em>M. davidii</em>.</p><p id="p-38">The genome-wide relationship between the two different CUB statistics (entropy <em>vs.</em> ENC) was plotted (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). We discovered a positive relationship (correlations of 0.619 and 0.672 in <em>M. davidii</em> and <em>P. alecto</em>, respectively). The entropy CUB statistic appears to discriminate the highly biased genes more sensitively, as evidenced by the skew at the bottom left of both species. This enhanced discrimination of entropy as one assesses CUB in the more biased genes appears a possible driver of any observed discrepancies in functional enrichment.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The relationship between ENC and entropy based CUB estimation for (A) M. davidii and (B) P. alecto." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-572324817" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The relationship between ENC and entropy based CUB estimation for (A) &lt;em&gt;M. davidii&lt;/em&gt; and (B) &lt;em&gt;P. alecto&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1825/F4.medium.gif" width="295" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/10/1825/F4.medium.gif" width="295" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465080" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-39" class="first-child">The relationship between ENC and entropy based CUB estimation for (A) <em>M. davidii</em> and (B) <em>P. alecto</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h4>CUB outlier genes quality checking:</h4><p id="p-40"><em>M. davidii LAMC3</em>, <em>CLN8</em>, and <em>SOX9</em> contained apparent gaps in the cds after translation BLAST and comparison with other bats and mammals. <em>IKZF1</em>, <em>WFS1</em>, and <em>SOX8</em> were in the expected reading frame but not contiguous. Collectively, these genes possess low-quality genomic sequence (string of Ns) in predicted exonic regions that might explain any missing cds sequence compared with other mammals. In contrast, alignment of <em>M. lucifugus</em> sequence for <em>IKZF1</em>, <em>LAMC3</em>, <em>SOX9</em>, and <em>SOX8</em> to other mammals is more conserved. This finding implies these genes may contain genomic assembly errors peculiar to <em>M. davidii</em> rather than being indicative of true evolutionary change. <em>SOD1</em> appears to be partial sequence only.</p></div><div id="sec-12" class="subsection"><h4>CUB enrichment across two additional bat species: M. lucifugus and P. vampyrus:</h4><p id="p-41">In an effort to find independent comparative evidence for CUB in genes encoding proteins related to the sensory apparatus of echolocating bats, we computed the entropy based CUB in an additional echolocator (<em>M. lucifugus</em>) and a nonecholocator (<em>P. vampyrus</em>) species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011262/-/DC1/FileS2.xlsx">File S2</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Sensory perception was not enriched in either species according to GOrilla analysis. The top three functional enrichments for <em>M. lucifugus</em> were G-protein−coupled receptor signaling pathway (<em>P</em> = 0.000000137), G-protein−coupled receptor signaling pathway coupled to cyclic nucleotide messenger (<em>P</em> = 0.00000667), and negative regulation of blood pressure (<em>P</em> = 0.0000295). For <em>P. vampyrus</em> they were G-protein−coupled receptor signaling pathway (<em>P</em> = 0.0000000196), adenylate cyclize-activating G-protein−coupled receptor signaling pathway (<em>P</em> = 0.0000296), and feeding behavior (<em>P</em> = 0.0000315).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genes showing extreme codon bias (negative differential entropy) on a within-genome basis in (A) M. lucifugus and (B) P. vampyrus, with those encoding vision and hearing related proteins highlighted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-572324817" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genes showing extreme codon bias (negative differential entropy) on a within-genome basis in (A) &lt;em&gt;M. lucifugus&lt;/em&gt; and (B) &lt;em&gt;P. vampyrus&lt;/em&gt;, with those encoding vision and hearing related proteins highlighted.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1825/F5.medium.gif" width="372" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/10/1825/F5.medium.gif" width="372" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465144" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-42" class="first-child">Genes showing extreme codon bias (negative differential entropy) on a within-genome basis in (A) <em>M. lucifugus</em> and (B) <em>P. vampyrus</em>, with those encoding vision and hearing related proteins highlighted.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Next, we identified the orthologs of those vision and hearing sensory genes previously enriched in <em>M. davidii</em>. In the three vision (<em>SOX8</em>, <em>HPS1</em>, <em>IKZF1</em>) and six hearing (<em>TMIE</em>, <em>SLC52A3</em>, <em>TBX18</em>, <em>OTOS</em>, <em>GPX1</em> and <em>USH1G</em>) instances in which both <em>M. lucifugus and P. vampyrus</em> orthologs could be identified, they were always more biased in <em>M. lucifugus</em> than <em>P. vampyrus</em> (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Orthologs showing differential codon bias (distance from diagonal) between M. lucifugus and P. vampyrus on a between-genome basis. Genes encoding vision and hearing elated proteins are highlighted. Of the sensory genes, TSPAN10, which has a highly restricted expression pattern in retina, is particularly prominent. All the sensory genes prioritized by CUB in M. davidii are more biased in M. lucifugus than P. alecto." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-572324817" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Orthologs showing differential codon bias (distance from diagonal) between &lt;em&gt;M. lucifugus&lt;/em&gt; and &lt;em&gt;P. vampyrus&lt;/em&gt; on a between-genome basis. Genes encoding vision and hearing elated proteins are highlighted. Of the sensory genes, &lt;em&gt;TSPAN10&lt;/em&gt;, which has a highly restricted expression pattern in retina, is particularly prominent. All the sensory genes prioritized by CUB in &lt;em&gt;M. davidii&lt;/em&gt; are more biased in &lt;em&gt;M. lucifugus&lt;/em&gt; than &lt;em&gt;P. alecto&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1825/F6.medium.gif" width="440" height="250"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/10/1825/F6.medium.gif" width="440" height="250"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1825/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465049" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-44" class="first-child">Orthologs showing differential codon bias (distance from diagonal) between <em>M. lucifugus</em> and <em>P. vampyrus</em> on a between-genome basis. Genes encoding vision and hearing elated proteins are highlighted. Of the sensory genes, <em>TSPAN10</em>, which has a highly restricted expression pattern in retina, is particularly prominent. All the sensory genes prioritized by CUB in <em>M. davidii</em> are more biased in <em>M. lucifugus</em> than <em>P. alecto</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">Finally, we manually explored the <em>M. lucifugus</em> and <em>P. vampyrus</em> CUB outliers by using the GO information in the Gene function of the National Center for Biotechnology Information database and searching the literature in PubMed. In <em>M. lucifugus</em> on a within-genome basis the second (<em>TSPAN10</em>, alias <em>OCSP</em>) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wistow <em>et al.</em> 2002</a>), 15th (<em>HCRT</em>) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Liu <em>et al.</em> 2011</a>), 20th (<em>FIZ1</em>) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mali <em>et al.</em> 2008</a>), 25th (<em>RTN4R</em>) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Wang <em>et al.</em> 2008</a>), and 34th (<em>TRAPPC6A</em>) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gwynn <em>et al.</em> 2006</a>) were all vision-related (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). <em>TSPAN10</em> has a very strong tissue-restricted expression in retina based on the BioGPS normalized tissue expression library (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wu <em>et al.</em> 2009</a>). In fact, retina is the only tissue of the 84 assayed in which <em>TSPAN10</em> has an expression level greater than threefold of the mean (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011262/-/DC1/FileS3.pdf">File S3</a>). <em>FIZ1</em> is a transcriptional co-factor that interacts with <em>NRL</em> and <em>CRX</em> to drive expression of rod photoreceptor-specific genes (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Mali <em>et al.</em> 2008</a>). Intriguingly, one of these partners (<em>CRX</em>) was actually prioritized by CUB in <em>M. davidii</em>. <em>TRAPPC6A</em> contributes to melanosome formation, and when mutated in mouse yields a loss of pigmentation phenotype in the retinal epithelial layer (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gwynn <em>et al.</em> 2006</a>).</p><p id="p-46">Although there was no formal enrichment for sensory perception in <em>M. lucifugus</em>, the CUB data obtained for the two myotid bats is broadly comparable at a pathway level (vision and hearing). Although some details differ, this is highly suggestive of a role for CUB in echolocating bat sensory evolution.</p></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-47">In this study we performed a comparative genomic assessment of Myotid and Pteropodid bats by using multiple approaches, and the results provide information on the genetic origins of sensory rewiring in an echolocating bat. We began by identifying pseudogenes using the naked mole rat as a comparative species with poor vision. Pseudogenes are relatives of genes that have subsequently lost their protein coding function, usually by disabling mutations such as premature stop codons and frameshifts. The observed correspondence between the naked mole rat and the echolocating bat (<em>CYP1A2</em>, <em>RBP3</em>, <em>GUCY2F</em>, <em>CRYBB1</em>, and <em>GRK7</em>) points to the general importance of these particular genes in controlling mammalian vision and also to pseudogenes in particular as a genetic signature after loss of physiological function (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Newman and Robinson 2005</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Meredith <em>et al.</em> 2013</a>).</p><p id="p-48">The next analysis was a hypothesis-free screen based on the genome-wide extent of CUB. In simple organisms, CUB is known to alter translation efficiency and subsequent protein abundance, but the potential role in governing translation efficiency during mammalian evolution is less clear (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Duret 2002</a>). Supporting the concept it is known that protein expressed from a foreign gene in human cells can be increased by two orders of magnitude if the sequence is engineered to contain the most commonly used human codons (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gustafsson <em>et al.</em> 2004</a>). However, there is an apparent lack of correlation between codon bias and protein abundance, and it is possible for codon-usage studies to be systematically influenced by amino acid content, gene length, and other potential artifacts (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chamary <em>et al.</em> 2006</a>).</p><p id="p-49">Despite these variable findings, all these studies have been impacted by an inability to exhaustively interrogate an entire and well-annotated genome (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Behura and Severson 2013</a>). For this reason, recent work that focuses on complete genome screens as well as other factors such as accounting for tissue and developmental stage, along with this study, may provide the first complete evidence for correlation between CUB and protein abundance in humans and other mammals (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Waldman <em>et al.</em> 2010</a>). A recent study concluded that although variation in mutational bias is the dominant force influencing CUB, translational selection acts as a weak additional factor and that this is widely true across the vertebrates (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Doherty and McInerney 2013</a>).</p><p id="p-50">On a within-genome basis, we were struck with the finding that the top functional enrichment for the echolocating <em>M. davidii</em> was retina development, a process not enriched at all on a within-genome basis in <em>P. alecto</em>. The additional <em>M. davidii</em> enrichments for ‘<em>axonogenesis</em>’, ‘<em>light perception</em>,’ and ‘<em>sound perception</em>’ that were absent in <em>P. alecto</em> reinforced evidence for sensory rewiring in the echolocating species. Equivalent analyses run on the additional echolocator <em>M. lucifugus</em> and the additional nonecholocator <em>P. vampyrus</em> did not receive formal enrichment scores for vision and hearing. However, manual exploration of the outliers found intriguing support for vision, particularly through <em>TSPAN10</em>, which has a very prominent tissue specific expression in retina, and the transcriptional co-activator <em>FIZ1</em>, which interacts with <em>CRX</em> to drive the expression of rod photoreceptor-specific genes.</p><p id="p-51">To complement our CUB within-genome analysis (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), we also compared the orthologs of <em>M. davidii</em> and <em>P. alecto</em> directly (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Orthologs identified as substantially more biased in <em>M. davidii</em> (but not previously identified by the within-genome GOrilla analysis) were explored manually for possible connections to the visual and hearing systems through targeted PubMed literature searches. Although these genes are bound together by their shared role in the biological processes of vision and hearing, they are somewhat diverse in terms of other more detailed molecular and functional properties. In regard to the visual proteins, some are transcription factors (<em>e.g.</em>, <em>CRX</em>), whereas others are visual pigment proteins (<em>e.g.</em>, <em>RHO</em>). Based on the TIGER database (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Liu <em>et al.</em> 2008</a>), which provides normalized tissue mRNA comparisons in humans, many of the vision genes are abundantly expressed in eye, and some are not. Furthermore, some are expressed highly across a wide range of tissues whereas others have an extremely restricted expression profile.</p><p id="p-52">Clearly, this range of tissue-level expression profiles renders any simple correlation of CUB to protein abundance unlikely. However, this does not mean one should discount CUB as a driver of eukaryotic evolution, as previously argued (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Waldman <em>et al.</em> 2010</a>). <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Waldman <em>et al.</em> (2010)</a> found significantly greater correlations observed in adult tissues than the fetus, pointing to the importance of developmental stage in these kind of assessments (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Waldman <em>et al.</em> 2010</a>). In that work, related sets of functional gene groups were found to be translated efficiently in each tissue, pointing to the importance of viewing groups of genes as components in a broader biological process or pathway (<a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Waldman <em>et al.</em> 2010</a>). In the future, comparison of biased genes to actual expression level in components of the bats sensory system would be of interest, but unfortunately these data are not presently available.</p><p id="p-53">To allow the CUB outliers to be placed in context, we calculated the genome-wide preferred codons for <em>M. davidii</em> and <em>P. alecto</em> based on all the available coding sequences available from the genome sequencing efforts (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). These data can be exploited to make predictions of bias toward or against preferred codons. In the context of explaining the low light−adapted vision and small eyes of <em>M. davidii</em>, we would predict enhanced translation efficiency (bias toward preferred codons) in negative regulators of eye size and visual pigments adapted for low light, and the converse (nonpreferred codons) for positive regulators of eye size and pigments adapted for day vision. However, the developmental pathways controlling retinogenesis are incomplete, and the genetic changes in the highlighted genes need to be assessed on a case-by-case basis and await future functional validation. Of the vision related genes highly biased in <em>M. davidii IKZF1</em> encodes a transcription factor called Ikaros family zinc finger 1, an ortholog of which confers competence to mouse retinal progenitor cells (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Elliott <em>et al.</em> 2008</a>). Manual exploration identified additional highly differentially biased genes encoding proteins expressed in eye components—<em>GPSM1</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kerov <em>et al.</em> 2005</a>) and <em>SPRR1</em> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Tong <em>et al.</em> 2006</a>). Enrichments for light and sound perception include numerous additional sensory genes not included in the specific retinogenic enrichment. The observation of extreme CUB in CRX in <em>M. davidii</em> and its binding co-factor FIZ1 in <em>M. lucifugus</em> supports this molecular pathway as a likely hotspot in vision-related adaptations that have evolved in nocturnal echolocating bats.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465124/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465124/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Patterns of genome-wide codon usage in <em>Myotis davidii</em></span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464921/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464921/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Patterns of genome-wide codon usage in <em>Pteropus alecto</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-56">Overall, our findings across two echolocating (<em>M. davidii</em> and <em>M. lucifugus</em>) and two nonecholocating (<em>P. alecto</em> and <em>P. vampyrus</em>) bats strongly support a recent observation made across echolocating mammals (bats and bottlenose dolphins) using a different comparative genomics approach. In this independent study, it was found that patterns of convergent protein evolution (based on nonsynonymous <em>vs.</em> synonymous amino acid substitutions) identify pathways contributing to hearing and vision as among the strongest signals (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Parker <em>et al.</em> 2013</a>). The exact overlap between our differential CUB analysis <em>vs.</em> the protein evolution study of (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Parker <em>et al.</em> 2013</a>) included the genes <em>RHO</em> and <em>OTOF</em>, which are involved in light and sound perception, respectively. The fact that not all genes exist in both analyses suggests that multiple regulatory mechanisms are likely to contribute to change of function.</p><p id="p-57"><em>NEFH</em> (alias <em>YHU2</em>; encoding the neurofilament heavy polypeptide) possessed extreme CUB in both <em>M. davidii</em> and <em>P. alecto</em>, implying it was either present in the ancestor or has subsequently converged due to a fundamental role in both bat lineages. A broader comparative analysis is needed to shed light on this gene, but unfortunately an annotated ortholog has not been identified in either the <em>M. lucifugus</em> or <em>P. vampyrus</em> sequences. One could sequence <em>NEFH</em> in a range of bats and one or more Laurasiatherian outgroups. <em>NEFH</em> is most highly expressed in the peripheral nervous system (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Liu <em>et al.</em> 2008</a>), so a generic sensory explanation seems plausible <em>i.e.</em>, one not specific to echolocation. Finally, a number of unannotated highly biased genes in both <em>M. davidii</em> and <em>P. alecto</em> were detected. These may represent targets for future functional studies.</p><p id="p-58">Overall, our findings provide insight into two genetic mechanisms contributing to evolution’s toolkit for adapting phenotypes. The deterioration of a number of core visual genes in <em>M. davidii</em> is consistent with genomic evolution in the naked mole rat (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2011</a>) and the poor day vision and small eyes of this species. Pseudogene formation is clearly a loss-of-function decay process. Disabling mutations are allowed to persist because of relaxation of selection pressure on that particular protein. Because mutation is random, if unchecked it will cause DNA sequence to tend toward (random) equilibrium. This finding contrasts with extreme CUB, which enforces structure and regularity on DNA sequence. This enforced regularity is strongly indicative of an applied evolutionary force, and tends to be associated with gain-of-function.</p><p id="p-59">The CUB evidence we have gathered for manipulation of translation efficiency at the biological pathway level has not previously been reported in the context of bat sensory ecology and warrants further investigation. The CUB data collectively imply a suite of adaptations to both the visual system (perhaps positively impacting on adaptation to dim light) and the enhancement of hearing (which presumably relates to echolocation). Overall, the general prioritization of sensory rewiring in the echolocator is highly resonant with the recent findings of <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Parker <em>et al.</em> (2013)</a>, despite a different line of genetic evidence and indeed different choice of echolocating species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">We wish to acknowledge fruitful discussions with Dr. James Kijas and Dr. Chris Elvin. This research was facilitated by the CSIRO Animal, Food, and Health Sciences post-genomics workshop, held at Coffs Harbor during April 2013 and was supported by CSIRO. Q.G. was supported by in part by the Shanghai Postdoctoral Sustention fund (no. 12R21410200). M.L.B. was supported by an Australian Research Council Future Fellowship (FT110100234).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011262/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011262/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-61"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-03-25"><span class="received-label">Received </span>March 25, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-07-23"><span class="accepted-label">Accepted </span>July 23, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Hudson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1825.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altringham</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fenton</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Sensory ecology and communication in the Chiroptera</span>, pp. <span class="cit-fpage">90</span>–<span class="cit-lpage">127</span> in <span class="cit-source">Bat Ecology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kunz</span>  <span class="cit-name-given-names">T. H.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Fenton</span>  <span class="cit-name-given-names">M. B.</span></span>. <span class="cit-publ-name">University of Chicago Press</span>, <span class="cit-publ-loc">Chicago</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.2" data-doi="10.1111/j.1469-185X.2012.00242.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behura</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Severson</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Codon usage bias: causative factors, quantification methods and genome-wide patterns: with emphasis on insect genomes.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BRev.%2BCamb.%2BPhilos.%2BSoc.%26rft.volume%253D88%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-185X.2012.00242.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-185X.2012.00242.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.3" data-doi="10.1007/BF01938504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhatnagar</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Olfactory in <em>Artibeus jamaicensis</em> and <em>Myotis lucifugus</em> in the context of vision and echolocation.</span> <abbr class="cit-jnl-abbrev">Experientia</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">856</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperientia%26rft.stitle%253DExperientia%26rft.aulast%253DBhatnagar%26rft.auinit1%253DK.%2BP.%26rft.volume%253D31%26rft.issue%253D7%26rft.spage%253D856%26rft.epage%253D856%26rft.atitle%253DOlfaction%2Bin%2BArtibeus%2Bjamaicensis%2Band%2BMyotis%2Blucifugus%2Bin%2Bthe%2Bcontext%2Bof%2Bvision%2Band%2Becholocation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01938504%26rft_id%253Dinfo%253Apmid%252F1140332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01938504&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1140332&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AK01900064&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.4" data-doi="10.1101/gr.1865504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clamp</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">GeneWise and Genomewise.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">988</span>–<span class="cit-lpage">995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DBirney%26rft.auinit1%253DE.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D988%26rft.epage%253D995%26rft.atitle%253DGeneWise%2Band%2BGenomewise%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1865504%26rft_id%253Dinfo%253Apmid%252F15123596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE0LzUvOTg4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xODI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.5" data-doi="10.1038/nrg1770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chamary</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Hearing silence: non-neutral evolution at synonymous sites in mammals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DChamary%26rft.auinit1%253DJ.%2BV.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D98%26rft.epage%253D108%26rft.atitle%253DHearing%2Bsilence%253A%2Bnon-neutral%2Bevolution%2Bat%2Bsynonymous%2Bsites%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1770%26rft_id%253Dinfo%253Apmid%252F16418745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16418745&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234714000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.6" data-doi="10.1093/molbev/mst128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doherty</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McInerney</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Translational selection frequently overcomes genetic drift in shaping synonymous codon usage patterns in vertebrates.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2263</span>–<span class="cit-lpage">2267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst128%26rft_id%253Dinfo%253Apmid%252F23883522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xMC8yMjYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xODI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.7" data-doi="10.1016/S0959-437X(02)00353-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evolution of synonymous codon usage in metazoans.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">640</span>–<span class="cit-lpage">649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D640%26rft.epage%253D649%26rft.atitle%253DEvolution%2Bof%2Bsynonymous%2Bcodon%2Busage%2Bin%2Bmetazoans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252802%252900353-2%26rft_id%253Dinfo%253Apmid%252F12433576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(02)00353-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12433576&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179003900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.8" data-doi="10.1186/1471-2105-10-48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eden</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinfeld</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yakhini</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">GOrilla: a tool for discovery and visualization of enriched GO terms in ranked gene lists.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEden%26rft.auinit1%253DE.%26rft.volume%253D10%26rft.spage%253D48%26rft.epage%253D48%26rft.atitle%253DGOrilla%253A%2Ba%2Btool%2Bfor%2Bdiscovery%2Band%2Bvisualization%2Bof%2Benriched%2BGO%2Bterms%2Bin%2Branked%2Bgene%2Blists.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-48%26rft_id%253Dinfo%253Apmid%252F19192299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19192299&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.9" data-doi="10.1016/j.neuron.2008.08.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jolicoeur</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramamurthy</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cayouette</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Ikaros confers early temporal competence to mouse retinal progenitor cells.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">26</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.stitle%253DNeuron%26rft.aulast%253DElliott%26rft.auinit1%253DJ.%26rft.volume%253D60%26rft.issue%253D1%26rft.spage%253D26%26rft.epage%253D39%26rft.atitle%253DIkaros%2Bconfers%2Bearly%2Btemporal%2Bcompetence%2Bto%2Bmouse%2Bretinal%2Bprogenitor%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2008.08.008%26rft_id%253Dinfo%253Apmid%252F18940586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2008.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18940586&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260237300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.10" data-doi="10.1007/BF00213073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuzessery</span>  <span class="cit-name-given-names">Z. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buttenhoff</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Passive sound localization of prey by the pallid bat (Antrozous p. pallidus).</span> <abbr class="cit-jnl-abbrev">J. Comp. Physiol. A Neuroethol. Sens. Neural Behav. Physiol.</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomparative%2Bphysiology.%2BA%252C%2BNeuroethology%252C%2Bsensory%252C%2Bneural%252C%2Band%2Bbehavioral%2Bphysiology%26rft.stitle%253DJ%2BComp%2BPhysiol%2BA%2BNeuroethol%2BSens%2BNeural%2BBehav%2BPhysiol%26rft.aulast%253DFuzessery%26rft.auinit1%253DZ.%2BM.%26rft.volume%253D171%26rft.issue%253D6%26rft.spage%253D767%26rft.epage%253D777%26rft.atitle%253DPassive%2Bsound%2Blocalization%2Bof%2Bprey%2Bby%2Bthe%2Bpallid%2Bbat%2B%2528Antrozous%2Bp.%2Bpallidus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00213073%26rft_id%253Dinfo%253Apmid%252F8441123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00213073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8441123&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.11" data-doi="10.1016/j.tibtech.2004.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gustafsson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Govindarajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minshull</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Codon bias and heterologous protein expression.</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiotechnology%26rft.stitle%253DTrends%2BBiotechnol%26rft.aulast%253DGustafsson%26rft.auinit1%253DC.%26rft.volume%253D22%26rft.issue%253D7%26rft.spage%253D346%26rft.epage%253D353%26rft.atitle%253DCodon%2Bbias%2Band%2Bheterologous%2Bprotein%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2004.04.006%26rft_id%253Dinfo%253Apmid%252F15245907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2004.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15245907&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222712000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.12" data-doi="10.1016/j.ygeno.2006.04.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gwynn</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowe</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A mouse TRAPP-related protein is involved in pigmentation.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">196</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DGwynn%26rft.auinit1%253DB.%26rft.volume%253D88%26rft.issue%253D2%26rft.spage%253D196%26rft.epage%253D203%26rft.atitle%253DA%2Bmouse%2BTRAPP-related%2Bprotein%2Bis%2Binvolved%2Bin%2Bpigmentation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2006.04.002%26rft_id%253Dinfo%253Apmid%252F16697553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2006.04.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16697553&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239399900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.13" data-doi="10.1371/journal.pone.0025457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagaraj</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalrymple</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Eukaryotic evolutionary transitions are associated with extreme codon bias in functionally-related proteins.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e25457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHudson%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De25457%26rft.epage%253De25457%26rft.atitle%253DEukaryotic%2Bevolutionary%2Btransitions%2Bare%2Bassociated%2Bwith%2Bextreme%2Bcodon%2Bbias%2Bin%2Bfunctionally-related%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0025457%26rft_id%253Dinfo%253Apmid%252F21966531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0025457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21966531&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teeling</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossiter</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">From the ultrasonic to the infrared: molecular evolution and the sensory biology of bats.</span> <abbr class="cit-jnl-abbrev">Front Physiol</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BPhysiol%26rft.volume%253D4%26rft.spage%253D117%26rft_id%253Dinfo%253Apmid%252F23755015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23755015&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.15" data-doi="10.1111/j.1365-2699.1996.tb00018.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalko</span>  <span class="cit-name-given-names">E. K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herre</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handley</span>  <span class="cit-name-given-names">C. O.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">1996</span> <span class="cit-article-title">Relation of fig fruit characteristics to fruit-eating bats in the New and Old World tropics.</span> <abbr class="cit-jnl-abbrev">J. Biogeogr.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiogeogr.%26rft.volume%253D23%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2699.1996.tb00018.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2699.1996.tb00018.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VZ75000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.16" data-doi="10.1016/j.mcn.2004.10.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerov</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natochin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artemyev</span>  <span class="cit-name-given-names">N. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Interaction of transducin-alpha with LGN, a G-protein modulator expressed in photoreceptor cells.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Neurosci.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2Bcellular%2Bneurosciences%26rft.stitle%253DMol%2BCell%2BNeurosci%26rft.aulast%253DKerov%26rft.auinit1%253DV.%2BS.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D485%26rft.epage%253D495%26rft.atitle%253DInteraction%2Bof%2Btransducin-alpha%2Bwith%2BLGN%252C%2Ba%2BG-protein%2Bmodulator%2Bexpressed%2Bin%2Bphotoreceptor%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mcn.2004.10.010%26rft_id%253Dinfo%253Apmid%252F15737739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mcn.2004.10.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15737739&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227620500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.17" data-doi="10.1097/MLG.0b013e31806db531"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerschner</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mucin gene expression in human middle ear epithelium.</span> <abbr class="cit-jnl-abbrev">Laryngoscope</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">1666</span>–<span class="cit-lpage">1676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLaryngoscope%26rft.stitle%253DLaryngoscope%26rft.aulast%253DKerschner%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D117%26rft.issue%253D9%26rft.spage%253D1666%26rft.epage%253D1676%26rft.atitle%253DMucin%2Bgene%2Bexpression%2Bin%2Bhuman%2Bmiddle%2Bear%2Bepithelium.%26rft_id%253Dinfo%253Adoi%252F10.1097%252FMLG.0b013e31806db531%26rft_id%253Dinfo%253Apmid%252F17667140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/MLG.0b013e31806db531&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17667140&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249265400025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.18" data-doi="10.1007/s10719-009-9245-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerschner</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khampang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cioffi</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mucin gene 19 (MUC19) expression and response to inflammatory cytokines in middle ear epithelium.</span> <abbr class="cit-jnl-abbrev">Glycoconj. J.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1275</span>–<span class="cit-lpage">1284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlycoconjugate%2Bjournal%26rft.stitle%253DGlycoconj%2BJ%26rft.aulast%253DKerschner%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D26%26rft.issue%253D9%26rft.spage%253D1275%26rft.epage%253D1284%26rft.atitle%253DMucin%2Bgene%2B19%2B%2528MUC19%2529%2Bexpression%2Band%2Bresponse%2Bto%2Binflammatory%2Bcytokines%2Bin%2Bmiddle%2Bear%2Bepithelium.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10719-009-9245-x%26rft_id%253Dinfo%253Apmid%252F19533339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10719-009-9245-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19533339&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.19" data-doi="10.1038/nature10533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fushan</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobanov</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome sequencing reveals insights into physiology and longevity of the naked mole rat.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">479</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKim%26rft.auinit1%253DE.%2BB.%26rft.volume%253D479%26rft.issue%253D7372%26rft.spage%253D223%26rft.epage%253D227%26rft.atitle%253DGenome%2Bsequencing%2Breveals%2Binsights%2Binto%2Bphysiology%2Band%2Blongevity%2Bof%2Bthe%2Bnaked%2Bmole%2Brat.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10533%26rft_id%253Dinfo%253Apmid%252F21993625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21993625&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298030800041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.20" data-doi="10.1016/j.neuroscience.2011.04.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">G. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">S. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Gene expression and protein distribution of orexins and orexin receptors in rat retina.</span> <abbr class="cit-jnl-abbrev">Neuroscience</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuroscience%26rft.volume%253D189%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuroscience.2011.04.011%26rft_id%253Dinfo%253Apmid%252F21621592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuroscience.2011.04.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21621592&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.21" data-doi="10.1186/1471-2105-9-271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zack</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">TiGER: a database for tissue-specific gene expression and regulation.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLiu%26rft.auinit1%253DX.%26rft.volume%253D9%26rft.spage%253D271%26rft.epage%253D271%26rft.atitle%253DTiGER%253A%2Ba%2Bdatabase%2Bfor%2Btissue-specific%2Bgene%2Bexpression%2Band%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-271%26rft_id%253Dinfo%253Apmid%252F18541026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-271&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18541026&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.22" data-doi="10.1186/1471-2199-9-87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">FIZ1 is part of the regulatory protein complex on active photoreceptor-specific gene promoters in vivo.</span> <abbr class="cit-jnl-abbrev">BMC Mol. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bmolecular%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BMol%2BBiol%26rft.aulast%253DMali%26rft.auinit1%253DR.%2BS.%26rft.volume%253D9%26rft.spage%253D87%26rft.epage%253D87%26rft.atitle%253DFIZ1%2Bis%2Bpart%2Bof%2Bthe%2Bregulatory%2Bprotein%2Bcomplex%2Bon%2Bactive%2Bphotoreceptor-specific%2Bgene%2Bpromoters%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2199-9-87%26rft_id%253Dinfo%253Apmid%252F18854042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2199-9-87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18854042&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.23" data-doi="10.1371/journal.pgen.1003432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meredith</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatesy</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emerling</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">York</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rod monochromacy and the coevolution of cetacean retinal opsins.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003432%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003432%26rft_id%253Dinfo%253Apmid%252F23637615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23637615&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">P. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Cone visual pigments of aquatic mammals.</span> <abbr class="cit-jnl-abbrev">Vis. Neurosci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVisual%2Bneuroscience%26rft.stitle%253DVis%2BNeurosci%26rft.aulast%253DNewman%26rft.auinit1%253DL.%2BA.%26rft.volume%253D22%26rft.issue%253D6%26rft.spage%253D873%26rft.epage%253D879%26rft.atitle%253DCone%2Bvisual%2Bpigments%2Bof%2Baquatic%2Bmammals.%26rft_id%253Dinfo%253Apmid%252F16469194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16469194&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235396400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.25" data-doi="10.1038/nature12511"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsagkogeorga</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotton</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Provero</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide signatures of convergent evolution in echolocating mammals.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">502</span>: <span class="cit-fpage">228</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D502%26rft.spage%253D228%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12511%26rft_id%253Dinfo%253Apmid%252F24005325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12511&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24005325&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325436100049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.26" data-doi="10.1126/science.3945827"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pettigrew</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Flying primates? Megabats have the advanced pathway from eye to midbrain.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">231</span>: <span class="cit-fpage">1304</span>–<span class="cit-lpage">1306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPettigrew%26rft.auinit1%253DJ.%26rft.volume%253D231%26rft.issue%253D4743%26rft.spage%253D1304%26rft.epage%253D1306%26rft.atitle%253DFlying%2Bprimates%253F%2BMegabats%2Bhave%2Bthe%2Badvanced%2Bpathway%2Bfrom%2Beye%2Bto%2Bmidbrain%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.3945827%26rft_id%253Dinfo%253Apmid%252F3945827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyMzEvNDc0My8xMzA0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xODI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.27" data-doi="10.1038/ncomms3212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seim</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobanov</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome analysis reveals insights into physiology and longevity of the Brandt’s bat <em>Myotis brandtii</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D2212%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms3212%26rft_id%253Dinfo%253Apmid%252F23962925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms3212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23962925&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.28" data-doi="10.1016/j.tree.2009.02.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teeling</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Hear, hear: the convergent evolution of echolocation in bats?</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DTeeling%26rft.auinit1%253DE.%2BC.%26rft.volume%253D24%26rft.issue%253D7%26rft.spage%253D351%26rft.epage%253D354%26rft.atitle%253DHear%252C%2Bhear%253A%2Bthe%2Bconvergent%2Bevolution%2Bof%2Becholocation%2Bin%2Bbats%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2009.02.012%26rft_id%253Dinfo%253Apmid%252F19482373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2009.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19482373&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267903100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.29" data-doi="10.1167/iovs.05-1129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corrales</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villarreal</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Paiva</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Expression and regulation of cornified envelope proteins in human corneal epithelium.</span> <abbr class="cit-jnl-abbrev">Invest. Ophthalmol. Vis. Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1938</span>–<span class="cit-lpage">1946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInvestigative%2BOphthalmology%2B%2526%2BVisual%2BScience%26rft.stitle%253DIOVS%26rft.aulast%253DTong%26rft.auinit1%253DL.%26rft.volume%253D47%26rft.issue%253D5%26rft.spage%253D1938%26rft.epage%253D1946%26rft.atitle%253DExpression%2Band%2BRegulation%2Bof%2BCornified%2BEnvelope%2BProteins%2Bin%2BHuman%2BCorneal%2BEpithelium%26rft_id%253Dinfo%253Adoi%252F10.1167%252Fiovs.05-1129%26rft_id%253Dinfo%253Apmid%252F16639001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiaW92cyI7czo1OiJyZXNpZCI7czo5OiI0Ny81LzE5MzgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE4MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.30" data-doi="10.1093/nar/gkq009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waldman</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shlomi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruppin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Translation efficiency in humans: tissue specificity, global optimization and differences between developmental stages.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">2964</span>–<span class="cit-lpage">2974</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq009%26rft_id%253Dinfo%253Apmid%252F20097653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM4LzkvMjk2NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTAvMTgyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.31" data-doi="10.1002/jnr.21626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">S. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Localization of Nogo and its receptor in the optic pathway of mouse embryos.</span> <abbr class="cit-jnl-abbrev">J. Neurosci. Res.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">1721</span>–<span class="cit-lpage">1733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneuroscience%2Bresearch%26rft.stitle%253DJ%2BNeurosci%2BRes%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D86%26rft.issue%253D8%26rft.spage%253D1721%26rft.epage%253D1733%26rft.atitle%253DLocalization%2Bof%2BNogo%2Band%2Bits%2Breceptor%2Bin%2Bthe%2Boptic%2Bpathway%2Bof%2Bmouse%2Bembryos.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjnr.21626%26rft_id%253Dinfo%253Apmid%252F18214994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jnr.21626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18214994&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.32" data-doi="10.1073/pnas.88.22.10357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weibel</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoppeler</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The concept of symmorphosis: a testable hypothesis of structure-function relationship.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">10357</span>–<span class="cit-lpage">10361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWeibel%26rft.auinit1%253DE.%2BR.%26rft.volume%253D88%26rft.issue%253D22%26rft.spage%253D10357%26rft.epage%253D10361%26rft.atitle%253DThe%2Bconcept%2Bof%2Bsymmorphosis%253A%2Ba%2Btestable%2Bhypothesis%2Bof%2Bstructure-function%2Brelationship.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.22.10357%26rft_id%253Dinfo%253Apmid%252F1946456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiODgvMjIvMTAzNTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE4MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wistow</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernstein</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyatt</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fariss</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Behal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Expressed sequence tag analysis of human RPE/choroid for the NEIBank Project: over 6000 non-redundant transcripts, novel genes and splice variants.</span> <abbr class="cit-jnl-abbrev">Mol. Vis.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bvision%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DMol%2BVis%26rft.aulast%253DWistow%26rft.auinit1%253DG.%26rft.volume%253D8%26rft.spage%253D205%26rft.epage%253D220%26rft.atitle%253DExpressed%2Bsequence%2Btag%2Banalysis%2Bof%2Bhuman%2BRPE%252Fchoroid%2Bfor%2Bthe%2BNEIBank%2BProject%253A%2Bover%2B6000%2Bnon-redundant%2Btranscripts%252C%2Bnovel%2Bgenes%2Band%2Bsplice%2Bvariants.%26rft_id%253Dinfo%253Apmid%252F12107410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12107410&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176701300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.34" data-doi="10.1016/0378-1119(90)90491-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The ’effective number of codons’ used in a gene.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DWright%26rft.auinit1%253DF.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D29%26rft.atitle%253DThe%2B%2527effective%2Bnumber%2Bof%2Bcodons%2527%2Bused%2Bin%2Ba%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252890%252990491-9%26rft_id%253Dinfo%253Apmid%252F2110097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(90)90491-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2110097&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DA22500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.35" data-doi="10.1186/gb-2009-10-11-r130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leglise</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodale</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BioGPS: an extensible and customizable portal for querying and organizing gene annotation resources.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DWu%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253DR130%26rft.epage%253DR130%26rft.atitle%253DBioGPS%253A%2Ban%2Bextensible%2Band%2Bcustomizable%2Bportal%2Bfor%2Bquerying%2Band%2Borganizing%2Bgene%2Bannotation%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-11-r130%26rft_id%253Dinfo%253Apmid%252F19919682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-11-r130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19919682&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1825.36" data-doi="10.1126/science.1230835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowled</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop-Lilly</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative analysis of bat genomes provides insight into the evolution of flight and immunity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">456</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1230835%26rft_id%253Dinfo%253Apmid%252F23258410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjExOC80NTYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE4MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/10/1825.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/10/1813" title="Widespread Genomic Incompatibilities in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/10/1837" title="An X-Linked Sex Ratio Distorter in Drosophila simulans That Kills or Incapacitates Both Noncarrier Sperm and Sons" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/10"><span>Volume 4 Issue 10, October 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/10.cover-source.jpg?itok=f3Vstm56" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/10/1825" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-o038Mfi2gHkcvX2eG7x3U6aoiozcsGFBXNR6CHuhlHg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/10/1825" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/10/1825" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/10/1825" class="active" data-icon-position="" data-hide-link-title="0">Sensory Rewiring in an Echolocator: Genome-Wide Modification of Retinogenic and Auditory Genes in the Bat Myotis davidii</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459690" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-8XGn13sNktz6kQuLbhV3u18PztjkFNwSsCcJ1FaD20E" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/10/1825.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/10/1825" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-eWbFyQa2EXmvvx6w0QHXVNZj-V7eK003IzXwZMZvbYQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/10/1825" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/10/1825.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459690" id="node459690--2" data-pisa="ggg;4/10/1825" data-pisa-master="ggg;g3.114.011262" data-apath="/ggg/4/10/1825.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sensory Rewiring in an Echolocator: Genome-Wide Modification of Retinogenic and Auditory Genes in the Bat <em>Myotis davidii</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nicholas J. Hudson</span>, <span class="highwire-citation-author" data-delta="1">Michelle L. Baker</span>, <span class="highwire-citation-author" data-delta="2">Nathan S. Hart</span>, <span class="highwire-citation-author" data-delta="3">James W. Wynne</span>, <span class="highwire-citation-author" data-delta="4">Quan Gu</span>, <span class="highwire-citation-author" data-delta="5">Zhiyong Huang</span>, <span class="highwire-citation-author" data-delta="6">Guojie Zhang</span>, <span class="highwire-citation-author" data-delta="7">Aaron B. Ingham</span>, <span class="highwire-citation-author" data-delta="8">Linfa Wang</span> and <span class="highwire-citation-author" data-delta="9">Antonio Reverter</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1825-1835;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011262 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459690--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas J. Hudson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational and Systems Biology, CSIRO Agriculture Flagship, Queensland Bioscience Precinct, Brisbane, Queensland, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BJ.%2BHudson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hudson%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BJ.%2BHudson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nick.hudson@csiro.au" class="" data-icon-position="" data-hide-link-title="0">nick.hudson@csiro.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michelle L. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Australian Animal Health Laboratory, CSIRO Biosecurity Flagship, Geelong, Victoria, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michelle%2BL.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichelle%2BL.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nathan S. Hart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Animal Biology and the Oceans Institute, University of Western Australia, Crawley, WA 6009, Western Australia, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathan%2BS.%2BHart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hart%20NS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANathan%2BS.%2BHart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James W. Wynne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Australian Animal Health Laboratory, CSIRO Biosecurity Flagship, Geelong, Victoria, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BW.%2BWynne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wynne%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BW.%2BWynne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Quan Gu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>College of Information Sciences and Technology, Donghua University, Shanghai 201620, China</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Faculty of Medicine, Imperial College London, South Kensington, London SW7 2AZ, England</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Quan%2BGu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQuan%2BGu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zhiyong Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>BGI-Shenzhen, Shenzhen, 518083, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiyong%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhiyong%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Guojie Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>BGI-Shenzhen, Shenzhen, 518083, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guojie%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuojie%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aaron B. Ingham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational and Systems Biology, CSIRO Agriculture Flagship, Queensland Bioscience Precinct, Brisbane, Queensland, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BB.%2BIngham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ingham%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BB.%2BIngham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Linfa Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Australian Animal Health Laboratory, CSIRO Biosecurity Flagship, Geelong, Victoria, Australia</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Program in Emerging Infectious Diseases, Duke-NUS Graduate Medical School, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linfa%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinfa%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Antonio Reverter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational and Systems Biology, CSIRO Agriculture Flagship, Queensland Bioscience Precinct, Brisbane, Queensland, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BReverter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reverter%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonio%2BReverter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/10/1825" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/10/1825&amp;title=Sensory%20Rewiring%20in%20an%20Echolocator%3A%20Genome-Wide%20Modification%20of%20Retinogenic%20and%20Auditory%20Genes%20in%20the%20Bat%20Myotis%20davidii" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/10/1825&amp;title=Sensory%20Rewiring%20in%20an%20Echolocator%3A%20Genome-Wide%20Modification%20of%20Retinogenic%20and%20Auditory%20Genes%20in%20the%20Bat%20Myotis%20davidii" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/10/1825&amp;title=Sensory%20Rewiring%20in%20an%20Echolocator%3A%20Genome-Wide%20Modification%20of%20Retinogenic%20and%20Auditory%20Genes%20in%20the%20Bat%20Myotis%20davidii" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/10/1825&amp;text=Sensory%20Rewiring%20in%20an%20Echolocator%3A%20Genome-Wide%20Modification%20of%20Retinogenic%20and%20Auditory%20Genes%20in%20the%20Bat%20Myotis%20davidii" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/10/1825&amp;title=Sensory%20Rewiring%20in%20an%20Echolocator%3A%20Genome-Wide%20Modification%20of%20Retinogenic%20and%20Auditory%20Genes%20in%20the%20Bat%20Myotis%20davidii" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/10/1825&amp;t=Sensory%20Rewiring%20in%20an%20Echolocator%3A%20Genome-Wide%20Modification%20of%20Retinogenic%20and%20Auditory%20Genes%20in%20the%20Bat%20Myotis%20davidii" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/10/1825&amp;title=Sensory%20Rewiring%20in%20an%20Echolocator%3A%20Genome-Wide%20Modification%20of%20Retinogenic%20and%20Auditory%20Genes%20in%20the%20Bat%20Myotis%20davidii" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/10/1825&amp;title=Sensory%20Rewiring%20in%20an%20Echolocator%3A%20Genome-Wide%20Modification%20of%20Retinogenic%20and%20Auditory%20Genes%20in%20the%20Bat%20Myotis%20davidii" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459690/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459690" id="node459690--3" data-pisa="ggg;4/10/1825" data-pisa-master="ggg;g3.114.011262" data-apath="/ggg/4/10/1825.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sensory Rewiring in an Echolocator: Genome-Wide Modification of Retinogenic and Auditory Genes in the Bat <em>Myotis davidii</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nicholas J. Hudson</span>, <span class="highwire-citation-author" data-delta="1">Michelle L. Baker</span>, <span class="highwire-citation-author" data-delta="2">Nathan S. Hart</span>, <span class="highwire-citation-author" data-delta="3">James W. Wynne</span>, <span class="highwire-citation-author" data-delta="4">Quan Gu</span>, <span class="highwire-citation-author" data-delta="5">Zhiyong Huang</span>, <span class="highwire-citation-author" data-delta="6">Guojie Zhang</span>, <span class="highwire-citation-author" data-delta="7">Aaron B. Ingham</span>, <span class="highwire-citation-author" data-delta="8">Linfa Wang</span> and <span class="highwire-citation-author" data-delta="9">Antonio Reverter</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1825-1835;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011262 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459690--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas J. Hudson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational and Systems Biology, CSIRO Agriculture Flagship, Queensland Bioscience Precinct, Brisbane, Queensland, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BJ.%2BHudson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hudson%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BJ.%2BHudson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nick.hudson@csiro.au" class="" data-icon-position="" data-hide-link-title="0">nick.hudson@csiro.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michelle L. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Australian Animal Health Laboratory, CSIRO Biosecurity Flagship, Geelong, Victoria, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michelle%2BL.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichelle%2BL.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nathan S. Hart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Animal Biology and the Oceans Institute, University of Western Australia, Crawley, WA 6009, Western Australia, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathan%2BS.%2BHart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hart%20NS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANathan%2BS.%2BHart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James W. Wynne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Australian Animal Health Laboratory, CSIRO Biosecurity Flagship, Geelong, Victoria, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BW.%2BWynne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wynne%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BW.%2BWynne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Quan Gu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>College of Information Sciences and Technology, Donghua University, Shanghai 201620, China</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Faculty of Medicine, Imperial College London, South Kensington, London SW7 2AZ, England</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Quan%2BGu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQuan%2BGu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zhiyong Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>BGI-Shenzhen, Shenzhen, 518083, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiyong%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhiyong%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Guojie Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>BGI-Shenzhen, Shenzhen, 518083, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guojie%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuojie%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aaron B. Ingham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational and Systems Biology, CSIRO Agriculture Flagship, Queensland Bioscience Precinct, Brisbane, Queensland, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BB.%2BIngham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ingham%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BB.%2BIngham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Linfa Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Australian Animal Health Laboratory, CSIRO Biosecurity Flagship, Geelong, Victoria, Australia</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Program in Emerging Infectious Diseases, Duke-NUS Graduate Medical School, Singapore</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linfa%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinfa%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Antonio Reverter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational and Systems Biology, CSIRO Agriculture Flagship, Queensland Bioscience Precinct, Brisbane, Queensland, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BReverter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reverter%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonio%2BReverter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1825&amp;rft.epage=1835&amp;rft.atitle=Sensory%20Rewiring%20in%20an%20Echolocator%3A%20Genome-Wide%20Modification%20of%20Retinogenic%20and%20Auditory%20Genes%20in%20the%20Bat%20Myotis%20davidii&amp;rft.volume=4&amp;rft.issue=10&amp;rft.date=2014-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hudson%2C+Nicholas+J.&amp;rft.au=Baker%2C+Michelle+L.&amp;rft.au=Hart%2C+Nathan+S.&amp;rft.au=Wynne%2C+James+W.&amp;rft.au=Gu%2C+Quan&amp;rft.au=Huang%2C+Zhiyong&amp;rft.au=Zhang%2C+Guojie&amp;rft.au=Ingham%2C+Aaron+B.&amp;rft.au=Wang%2C+Linfa&amp;rft.au=Reverter%2C+Antonio"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459690/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459690/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459690/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459690/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459690/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459690/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459690/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459690/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459690/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459690/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459690/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459690/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/10/1825" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/10/1825.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/10/1825.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/10/1825.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/10/1825.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
