<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/10/1971" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/10/1971.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/10/1971.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/10/1971.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/10/1971/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/10/1971.atom" />
<meta name="HW.pisa" content="ggg;4/10/1971" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Saturated Genetic Linkage Map of Autotetraploid Alfalfa (Medicago sativa L.) Developed Using Genotyping-by-Sequencing Is Highly Syntenous with the Medicago truncatula Genome" />
<meta name="DC.Identifier" content="10.1534/g3.114.012245" />
<meta name="DC.Date" content="2014-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Li et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="A genetic linkage map is a valuable tool for quantitative trait locus mapping, map-based gene cloning, comparative mapping, and whole-genome assembly. Alfalfa, one of the most important forage crops in the world, is autotetraploid, allogamous, and highly heterozygous, characteristics that have impeded the construction of a high-density linkage map using traditional genetic marker systems. Using genotyping-by-sequencing (GBS), we constructed low-cost, reasonably high-density linkage maps for both maternal and paternal parental genomes of an autotetraploid alfalfa F1 population. The resulting maps contain 3591 single-nucleotide polymorphism markers on 64 linkage groups across both parents, with an average density of one marker per 1.5 and 1.0 cM for the maternal and paternal haplotype maps, respectively. Chromosome assignments were made based on homology of markers to the M. truncatula genome. Four linkage groups representing the four haplotypes of each alfalfa chromosome were assigned to each of the eight Medicago chromosomes in both the maternal and paternal parents. The alfalfa linkage groups were highly syntenous with M. truncatula , and clearly identified the known translocation between Chromosomes 4 and 8. In addition, a small inversion on Chromosome 1 was identified between M. truncatula and M. sativa . GBS enabled us to develop a saturated linkage map for alfalfa that greatly improved genome coverage relative to previous maps and that will facilitate investigation of genome structure. GBS could be used in breeding populations to accelerate molecular breeding in alfalfa." />
<meta name="DC.Contributor" content="Xuehui Li" />
<meta name="DC.Contributor" content="Yanling Wei" />
<meta name="DC.Contributor" content="Ananta Acharya" />
<meta name="DC.Contributor" content="Qingzhen Jiang" />
<meta name="DC.Contributor" content="Junmei Kang" />
<meta name="DC.Contributor" content="E. Charles Brummer" />
<meta name="article:published_time" content="2014-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Saturated Genetic Linkage Map of Autotetraploid Alfalfa (Medicago sativa L.) Developed Using Genotyping-by-Sequencing Is Highly Syntenous with the Medicago truncatula Genome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A genetic linkage map is a valuable tool for quantitative trait locus mapping, map-based gene cloning, comparative mapping, and whole-genome assembly. Alfalfa, one of the most important forage crops in the world, is autotetraploid, allogamous, and highly heterozygous, characteristics that have impeded the construction of a high-density linkage map using traditional genetic marker systems. Using genotyping-by-sequencing (GBS), we constructed low-cost, reasonably high-density linkage maps for both maternal and paternal parental genomes of an autotetraploid alfalfa F&lt;sub&gt;1&lt;/sub&gt; population. The resulting maps contain 3591 single-nucleotide polymorphism markers on 64 linkage groups across both parents, with an average density of one marker per 1.5 and 1.0 cM for the maternal and paternal haplotype maps, respectively. Chromosome assignments were made based on homology of markers to the &lt;i&gt;M. truncatula&lt;/i&gt; genome. Four linkage groups representing the four haplotypes of each alfalfa chromosome were assigned to each of the eight &lt;i&gt;Medicago&lt;/i&gt; chromosomes in both the maternal and paternal parents. The alfalfa linkage groups were highly syntenous with &lt;i&gt;M. truncatula&lt;/i&gt;, and clearly identified the known translocation between Chromosomes 4 and 8. In addition, a small inversion on Chromosome 1 was identified between &lt;i&gt;M. truncatula&lt;/i&gt; and &lt;i&gt;M. sativa&lt;/i&gt;. GBS enabled us to develop a saturated linkage map for alfalfa that greatly improved genome coverage relative to previous maps and that will facilitate investigation of genome structure. GBS could be used in breeding populations to accelerate molecular breeding in alfalfa.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/10/01" />
<meta name="citation_mjid" content="ggg;4/10/1971" />
<meta name="citation_id" content="4/10/1971" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/10/1971" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/10/1971.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/10/1971.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/10/1971.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.012245" />
<meta name="citation_pmid" content="25147192" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1971" />
<meta name="citation_lastpage" content="1979" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Xuehui Li" />
<meta name="citation_author_institution" content="Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401" />
<meta name="citation_author" content="Yanling Wei" />
<meta name="citation_author_institution" content="Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401" />
<meta name="citation_author" content="Ananta Acharya" />
<meta name="citation_author_institution" content="Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401" />
<meta name="citation_author" content="Qingzhen Jiang" />
<meta name="citation_author_institution" content="Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401" />
<meta name="citation_author" content="Junmei Kang" />
<meta name="citation_author_institution" content="The Institute of Animal Science, Chinese Academy of Agricultural Science, Beijing, China 100193" />
<meta name="citation_author" content="E. Charles Brummer" />
<meta name="citation_author_institution" content="Plant Breeding Center and Department of Plant Sciences, The University of California, Davis, California 95616" />
<meta name="citation_author_email" content="ecbrummer@ucdavis.edu" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. Antanaviciute;citation_author=F. Fernandez-Fernandez;citation_author=J. Jansen;citation_author=E. Banchi;citation_author=K. M. Evans;citation_title=Development of a dense SNP-based linkage map of an apple rootstock progeny using the Malus Infinium whole genome genotyping array.;citation_pages=203-203;citation_volume=13;citation_year=2012;citation_pmid=22631220;citation_doi=10.1186/1471-2164-13-203" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Arends;citation_author=P. Prins;citation_author=R. C. Jansen;citation_author=K. W. Broman;citation_title=R/qtl: high-throughput multiple QTL mapping.;citation_volume=26;citation_year=2010;citation_pmid=20966004;citation_doi=10.1093/bioinformatics/btq565" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=F. Blondon;citation_author=D. Marie;citation_author=S. Brown;citation_author=A. Kondorosi;citation_title=Genome size and base composition in Medicago sativa and M. truncatula species.;citation_pages=264-270;citation_volume=37;citation_year=1994;citation_issue=2;citation_pmid=18470076" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. E. Bowers;citation_author=E. Bachlava;citation_author=R. L. Brunick;citation_author=L. H. Rieseberg;citation_author=S. J. Knapp;citation_title=Development of a 10,000 locus genetic map of the sunflower genome based on multiple crosses. G3: Genes, Genomes;citation_pages=721-729;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. J. Brouwer;citation_author=T. C. Osborn;citation_title=A molecular marker linkage map of tetraploid alfalfa (Medicago sativa L.).;citation_pages=1194-1200;citation_volume=99;citation_year=1999;citation_doi=10.1007/s001220051324" />
<meta name="citation_reference" content="citation_journal_title=BMC Biol.;citation_author=E. Chancerel;citation_author=J. B. Lamy;citation_author=I. Lesur;citation_author=C. Noirot;citation_author=C. Klopp;citation_title=High-density linkage mapping in a pine tree reveals a genomic region associated with inbreeding depression and provides clues to the extent and distribution of meiotic recombination.;citation_pages=50;citation_volume=11;citation_year=2013;citation_pmid=23597128;citation_doi=10.1186/1741-7007-11-50" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. K. Choi;citation_author=D. Kim;citation_author=T. Uhm;citation_author=E. Limpens;citation_author=H. Lim;citation_title=A Sequence-Based Genetic Map of Medicago truncatula and Comparison of Marker Colinearity with M. sativa;citation_pages=1463-1502;citation_volume=166;citation_year=2004;citation_issue=3;citation_pmid=15082563;citation_doi=10.1534/genetics.166.3.1463" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. Delourme;citation_author=C. Falentin;citation_author=B. F. Fomeju;citation_author=M. Boillot;citation_author=G. Lassalle;citation_title=High-density SNP-based genetic map development and linkage disequilibrium assessment in Brassica napus L.;citation_pages=120-120;citation_volume=14;citation_year=2013;citation_issue=1;citation_pmid=23432809;citation_doi=10.1186/1471-2164-14-120" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=K. J. Felcher;citation_author=J. J. Coombs;citation_author=A. N. Massa;citation_author=C. N. Hansey;citation_author=J. P. Hamilton;citation_title=Integration of two diploid potato linkage maps with the potato genome sequence.;citation_pages=e36347-e36347;citation_volume=7;citation_year=2012;citation_issue=4;citation_pmid=22558443;citation_doi=10.1371/journal.pone.0036347" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Grattapaglia;citation_author=R. Sederoff;citation_title=Genetic linkage maps of Eucalyptus grandis and Eucalyptus urophylla using a pseudo-testcross: mapping strategy and RAPD markers.;citation_pages=1121-1137;citation_volume=137;citation_year=1994;citation_issue=4;citation_pmid=7982566" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. G. Griffiths;citation_author=B. A. Barrett;citation_author=D. Simon;citation_author=A. K. Khan;citation_author=P. Bickerstaff;citation_title=An integrated genetic linkage map for white clover (Trifolium repens L.) with alignment to Medicago.;citation_pages=388;citation_volume=14;citation_year=2013;citation_pmid=23758831;citation_doi=10.1186/1471-2164-14-388" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. A. Hackett;citation_author=L. B. Broadfoot;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=C. A. Hackett;citation_author=I. Milne;citation_author=J. E. Bradshaw;citation_author=Z. Luo;citation_title=TetraploidMap for Windows: linkage map construction and QTL mapping in autotetraploid species.;citation_volume=98;citation_year=2007;citation_pmid=17965198;citation_doi=10.1093/jhered/esm086" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Han;citation_author=Y. Kang;citation_author=I. Torres-Jerez;citation_author=F. Cheung;citation_author=C. D. Town;citation_title=Genome-wide SNP discovery in tetraploid alfalfa using 454 sequencing and high resolution melting analysis.;citation_pages=350;citation_volume=12;citation_year=2011;citation_doi=10.1186/1471-2164-12-350" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=B. Julier;citation_author=S. Flajoulot;citation_author=P. Barre;citation_author=G. Cardinet;citation_author=S. Santoni;citation_title=Construction of two genetic linkage maps in cultivated tetraploid alfalfa (Medicago sativa) using microsatellite and AFLP markers.;citation_pages=9-9;citation_volume=3;citation_year=2003;citation_pmid=14683527;citation_doi=10.1186/1471-2229-3-9" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=L. G. Kamphuis;citation_author=A. H. Williams;citation_author=N. K. D’Souza;citation_author=T. Pfaff;citation_author=S. R. Ellwood;citation_title=The Medicago truncatula reference accession A17 has an aberrant chromosomal configuration.;citation_pages=299-303;citation_volume=174;citation_year=2007;citation_issue=2;citation_pmid=17388892;citation_doi=10.1111/j.1469-8137.2007.02039.x" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=D. M. Khu;citation_author=R. Reyno;citation_author=Y. Han;citation_author=P. X. Zhao;citation_author=J. H. Bouton;citation_title=Identification of aluminum tolerance quantitative trait loci in tetraploid alfalfa.;citation_pages=148-163;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.03.0181" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Agronomy;citation_author=X. Li;citation_author=E. C. Brummer;citation_title=Applied genetics and genomics in alfalfa breeding.;citation_pages=40-61;citation_volume=2;citation_year=2012;citation_doi=10.3390/agronomy2010040" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=X. Li;citation_author=X. Wang;citation_author=Y. Wei;citation_author=E. C. Brummer;citation_title=Prevalence of segregation distortion in diploid alfalfa and its implications for genetics and breeding applications.;citation_pages=667-679;citation_volume=123;citation_year=2011;citation_issue=4;citation_pmid=21625992;citation_doi=10.1007/s00122-011-1617-5" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=X. Li;citation_author=A. Acharya;citation_author=A. D. Farmer;citation_author=J. A. Crow;citation_author=A. K. Bharti;citation_title=Prevalence of single nucleotide polymorphism among 27 diverse alfalfa genotypes as assessed by transcriptome sequencing.;citation_pages=568-568;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23107476;citation_doi=10.1186/1471-2164-13-568" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=X. Li;citation_author=Y. Han;citation_author=Y. Wei;citation_author=A. Acharya;citation_author=A. D. Farmer;citation_title=Development of an alfalfa SNP array and its use to evaluate patterns of population structure and linkage disequilibrium.;citation_pages=e84329;citation_volume=9;citation_year=2014;citation_pmid=24416217;citation_doi=10.1371/journal.pone.0084329" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_title=Alfalfa and Alfalfa Improvement;citation_title=World distribution and historical development;citation_author=R. Michaud;citation_author=W. F. Lehman;citation_author=M. D. Rumbaugh;citation_pages=31-32;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. Musial;citation_author=J. M. Mackie;citation_author=D. J. Armour;citation_author=H. T. Phan;citation_author=S. E. Ellwood;citation_title=Identification of QTL for resistance and susceptibility to Stagonospora meliloti in autotetraploid lucerne.;citation_pages=1427-1435;citation_volume=114;citation_year=2007;citation_issue=8;citation_pmid=17356865;citation_doi=10.1007/s00122-007-0528-y" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. H. Paterson;citation_author=J. E. Bowers;citation_author=R. Bruggmann;citation_author=I. Dubchak;citation_author=J. Grimwood;citation_title=The Sorghum bicolor genome and the diversification of grasses.;citation_pages=551-556;citation_volume=457;citation_year=2009;citation_issue=7229;citation_pmid=19189423;citation_doi=10.1038/nature07723" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Y. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. G. Robins;citation_author=D. Luth;citation_author=I. A. Campbell;citation_author=G. R. Bauchan;citation_author=C. L. He;citation_title=Genetic Mapping of Biomass Production in Tetraploid Alfalfa;citation_pages=1-10;citation_volume=47;citation_year=2007;citation_issue=1;citation_doi=10.2135/cropsci2005.11.0401" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. Schuelke;citation_title=An economic method for the fluorescent labeling of PCR fragments.;citation_pages=233-234;citation_volume=18;citation_year=2000;citation_issue=2;citation_pmid=10657137;citation_doi=10.1038/72708" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. C. Sim;citation_author=G. Durstewitz;citation_author=J. Plieske;citation_author=R. Wieseke;citation_author=M. W. Ganal;citation_title=Development of a large SNP genotyping array and generation of high-density genetic maps in tomato.;citation_pages=e40563-e40563;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22802968;citation_doi=10.1371/journal.pone.0040563" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. K. Sledge;citation_author=I. M. Ray;citation_author=G. Jiang;citation_title=An expressed sequence tag SSR map of tetraploid alfalfa (Medicago sativa L.).;citation_pages=980-992;citation_volume=111;citation_year=2005;citation_issue=5;citation_pmid=16075206;citation_doi=10.1007/s00122-005-0038-8" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Sonah;citation_author=M. Bastien;citation_author=E. Iquira;citation_author=A. Tardivel;citation_author=G. Legare;citation_title=An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.;citation_pages=e54603;citation_volume=8;citation_year=2013;citation_pmid=23372741;citation_doi=10.1371/journal.pone.0054603" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Spindel;citation_author=M. Wright;citation_author=C. Chen;citation_author=J. Cobb;citation_author=J. Gage;citation_title=Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations.;citation_pages=2699-2716;citation_volume=126;citation_year=2013;citation_pmid=23918062;citation_doi=10.1007/s00122-013-2166-x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=S. Tavoletti;citation_author=F. Veronesi;citation_author=T. C. Osborn;citation_title=RFLP Linkage Map of an Alfalfa Meiotic Mutant Based on an F1 Population;citation_pages=167-170;citation_volume=87;citation_year=1996;citation_issue=2;citation_doi=10.1093/oxfordjournals.jhered.a022978" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. G. Uitdewilligen;citation_author=A. M. Wolters;citation_author=B. D’Hoop;citation_author=T. J. Borm;citation_author=R. G. Visser;citation_title=A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato.;citation_pages=e62355;citation_volume=8;citation_year=2013;citation_pmid=23667470;citation_doi=10.1371/journal.pone.0062355" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. W. Van Ooijen;citation_title=Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species.;citation_pages=343-349;citation_volume=93;citation_year=2011;citation_issue=5;citation_pmid=21878144;citation_doi=10.1017/S0016672311000279" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_title=ggplot2: Elegant Graphics for Data Analysis;citation_author=H. Wickham;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. S. Yang;citation_author=Z. J. Tu;citation_author=F. Cheung;citation_author=W. W. Xu;citation_author=J. F. Lamb;citation_title=Using RNA-Seq for gene identification, polymorphism detection and transcript profiling in two alfalfa genotypes with divergent cell wall composition in stems.;citation_pages=199-199;citation_volume=12;citation_year=2011;citation_pmid=21504589;citation_doi=10.1186/1471-2164-12-199" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=L. Zhang;citation_author=S. Wang;citation_author=H. Li;citation_author=Q. Deng;citation_author=A. Zheng;citation_title=Effects of missing marker and segregation distortion on QTL mapping in F2 populations.;citation_pages=1071-1082;citation_volume=121;citation_year=2010;citation_issue=6;citation_pmid=20535442;citation_doi=10.1007/s00122-010-1372-z" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Saturated Genetic Linkage Map of Autotetraploid Alfalfa (Medicago sativa L.) Developed Using Genotyping-by-Sequencing Is Highly Syntenous with the Medicago truncatula Genome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10.cover-source.jpg" />
<meta name="twitter:description" content="A genetic linkage map is a valuable tool for quantitative trait locus mapping, map-based gene cloning, comparative mapping, and whole-genome assembly. Alfalfa, one of the most important forage crops in the world, is autotetraploid, allogamous, and highly heterozygous, characteristics that have impeded the construction of a high-density linkage map using traditional genetic marker systems. Using genotyping-by-sequencing (GBS), we constructed low-cost, reasonably high-density linkage maps for both maternal and paternal parental genomes of an autotetraploid alfalfa F1 population. The resulting maps contain 3591 single-nucleotide polymorphism markers on 64 linkage groups across both parents, with an average density of one marker per 1.5 and 1.0 cM for the maternal and paternal haplotype maps, respectively. Chromosome assignments were made based on homology of markers to the M. truncatula genome. Four linkage groups representing the four haplotypes of each alfalfa chromosome were assigned to each of the eight Medicago chromosomes in both the maternal and paternal parents. The alfalfa linkage groups were highly syntenous with M. truncatula , and clearly identified the known translocation between Chromosomes 4 and 8. In addition, a small inversion on Chromosome 1 was identified between M. truncatula and M. sativa . GBS enabled us to develop a saturated linkage map for alfalfa that greatly improved genome coverage relative to previous maps and that will facilitate investigation of genome structure. GBS could be used in breeding populations to accelerate molecular breeding in alfalfa." />
<meta name="og-title" property="og:title" content="A Saturated Genetic Linkage Map of Autotetraploid Alfalfa (Medicago sativa L.) Developed Using Genotyping-by-Sequencing Is Highly Syntenous with the Medicago truncatula Genome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/10/1971" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="A genetic linkage map is a valuable tool for quantitative trait locus mapping, map-based gene cloning, comparative mapping, and whole-genome assembly. Alfalfa, one of the most important forage crops in the world, is autotetraploid, allogamous, and highly heterozygous, characteristics that have impeded the construction of a high-density linkage map using traditional genetic marker systems. Using genotyping-by-sequencing (GBS), we constructed low-cost, reasonably high-density linkage maps for both maternal and paternal parental genomes of an autotetraploid alfalfa F1 population. The resulting maps contain 3591 single-nucleotide polymorphism markers on 64 linkage groups across both parents, with an average density of one marker per 1.5 and 1.0 cM for the maternal and paternal haplotype maps, respectively. Chromosome assignments were made based on homology of markers to the M. truncatula genome. Four linkage groups representing the four haplotypes of each alfalfa chromosome were assigned to each of the eight Medicago chromosomes in both the maternal and paternal parents. The alfalfa linkage groups were highly syntenous with M. truncatula , and clearly identified the known translocation between Chromosomes 4 and 8. In addition, a small inversion on Chromosome 1 was identified between M. truncatula and M. sativa . GBS enabled us to develop a saturated linkage map for alfalfa that greatly improved genome coverage relative to previous maps and that will facilitate investigation of genome structure. GBS could be used in breeding populations to accelerate molecular breeding in alfalfa." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10.cover-source.jpg" />
    <title>A Saturated Genetic Linkage Map of Autotetraploid Alfalfa (Medicago sativa L.) Developed Using Genotyping-by-Sequencing Is Highly Syntenous with the Medicago truncatula Genome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"0JuSZiuPkn5vs7CPvi3--y0_y8HUn4tFOzL4Zvt7Zqs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459696","apath":"\/ggg\/4\/10\/1971.atom","pisa":"ggg;4\/10\/1971","ac":{"\/ggg\/4\/10\/1971.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/10\/1971.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/10\/1971"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/10\/1971"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/10\/1971.figures-only":true,"\/content\/4\/10\/1971":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022h2QCsKJU5aasdGvVy9mQV7OoR7YAGH8X\\\\\\\/BeuYwmGw+4Qt7X9SMRWFYk5Nl8DqBWYDXoCIUlR8RLSkHLMZG9HZwlyPeWZG95r7fHMO3OCY6iBvqvFfwYLmqsath3aWrLPaXpdn4wVo4OxLGCOJReDGOWk6BpVnzAxlOGFE7SV4Loaf5gw\\\\\\\/LV5GpmiRCTQ\\\\\\\/+5\\\\\\\/YsDuHuH3lOFnq2n+XTSIHbermIb\\\\\\\/WPR1wH2JUDREkOWdXyqg8ixS23GuFLPChyql26b9tYW20ZSg\\\\\\\/zcRx6yB47E67C2bWlYJcF3VeNAY2sIxHYT15kf28JOaoYymbPsgaa3AetxQqaThOxZdemDV9ajLYOBdfNJxzY8\\\\\\\/Dj3z3hxFpzxypFeUys8QLrO9iMlUfqje5spLtKkXBzO\\\\\\\/ALBSBsO6sINPbT6VwDbhHmOktMzEO1aIY1wn6M1O6p6C5vrpBbw6iLhxHy\\\\\\\/TYOmKPme8OtzRZvYgMSe9K4ACEcZ3Y5R4pJplIypIGOmyVWCfmFAu954xprON+PlE8m6KIabbjbt0C1UmoydYKFlqcZWaHU9xOvSEKdTCF2jM+YL1XkdHQsjgxp3iz7hQtUZUetpaTFQQKek2eXvTXdxkq6ZuXrrhZPzWMjp+kTUBMes59DOH12eXhUZaMaC+3JIOfEHo+BnexRRsPb2vJ1CdUFdfBE1dEMVEn2+VDV4vtX8SAhp5mALeergBAgaE5xl1B+Lv5TnFQUIekPspxd00KQMmJrOJMhFSagnTN9Tp2QHd2tofX98IEuQa2bF++7wMovNOYN\\\\\\\/DsmSoXS4H\\\\\\\/OvBkwpetf+dvlyA5\\\\\\\/P4VlWcG6VyMJYCSAtQ+YpecnWsLPwSg2i695nC9bnJrr8Ll6uiLOmiWHjamS6wv+2MRrtsQFLAQ2u5g+H1KTjVE\\\\\\\/lDdLdnx5MuzRls5okxKxV2fCbqP1f4eJoXSJqAq9knGQ2eDM+O0eY8+Arh19DwOJBhbWfuwinwgLbtG3WxSVrEK\\\\\\\/CuQCtQyNZ3HWiLgsHtIwnDf2Ow0mWZD41yfNGYtQodr4HjNvNFAavOW04GA\\\\\\\/b+c68FYbmdPe6T0xog7Q4BsFhYVWkkGSrgyecpNSoXbe2XBURR3orqJraIKG0CFe5GDVwp6+PES1FjE9RnrLWEIVfwG+2RgMXRcQ08y5OQ4nktxwspVpTyS0vdzLx1ffprONEfBY+sMNlNJd358JjnkOH2X8I68Jrx5qHoZCVPYV8fMX0fzcfKQ8HRBFFiYh8EX+NiCCKIGJgTxCvGNLH5Wx8GGLx8lEoI\\\u0022,\\\u0022iv\\\u0022:\\\u0022DhMO7\\\\\\\/mPNNL0u4ZMbh9uZw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c4068ccb53ace6da402e11e121d4f2bed6cca1ad7e6623ff83e6316f79d99fab\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022z5g1C658lQ8JbqbUSRQnIcBCd49RUJ7eVWj4RQy24+\\\\\\\/HGEbHnCHddzYrm4GVvDmrRH\\\\\\\/7o5MZ7aOMwFIcWtkrzgg4an3ykKawVqwW\\\\\\\/3CLSaICEQblBjn7MLicf5ClskEdLtdBPW0XU14Wr8rtftPjklPumVSTe+JI84Ith2Gqot4\\\\\\\/FnbxGX5KFC0IUSBlSaJgGnYQ9+SFZo+OF\\\\\\\/DSGf5+K4cZSrokjDZ8YJ6or0dx1VekezQvlqscmolRpPteNxK3t7CNGyDVTGI3z2N1tkiOw6spS4YD6WX5yjC673gOEs1hgLP\\\\\\\/sfSiRz6TBPZuq7+HFqUnhVNvnuiwHdj91mAfd2bs9\\\\\\\/ml5v56Nsaeh\\\\\\\/auFYajvaS\\\\\\\/6fOhMc5gnGRBTk0oTFpkq99i2+9ivrSpzX493FDK7cZg6pJgW9rNg+Y5F7B90jjsqKz1U\\\\\\\/2dDRI+RcItoHTJ0QocspGgsWEQpiiGWwxiyn5\\\\\\\/ZsXtDgu8h6oFNnf74w\\\\\\\/WZisAFW47E59juxQMRpbKjA011nLYAmQfmH8\\\\\\\/NgoIRMGVGU4NDL\\\\\\\/IomnrkZku8Po6241ZW7lZ3+qrph+vEPQT4d0AbDFBAgdKmkL4A2mvh4UHzljoiQ5npa4mGOQF5uN\\\\\\\/0DfuUyap4ZN79+trX8IFShvUVWyrJwLVuKyEu9Vafsq2uVEc8fGyYV8v+8KiKr3F8\\\\\\\/8cluo+4rZdrmYR4mQpl19r\\\\\\\/amGw233i8GbzeGcywKpoHtYWs2usWcDHPt6hWsNywSD5J1E7sPq5ycGA6cWUsuAyK\\\\\\\/esCRdBF3NrX77AkGaUBDbmeLbTyznBBZruNAzjqe0n7VvXRiby2ru0TFakjBRATahXR+YiPqwPtPZgxppwXQAq+W9bmJsopoAVpttPWltDf\\\\\\\/oxpTJQ7eJoyGbbzOztsUF4GbzWS7KJiggEV8w\\\\\\\/\\\\\\\/2VkUFvIJ\\\\\\\/bj3ff9XL49LpECyh7fqGoZu6js\\\\\\\/GiXTON7mQ1X3mQGG1zqU9ieqZkbHnR7x0rJTWTL5E8LV+KE1Wn4uaA7I5DIc0FiqICSF3VYd30vlTTuCrdnIZvaZ5EjxSQOpbpSqGmLOchP9vvNHjbyhIq+uZSXVLtjeDqMYxCoCDTOc6vlZ30AojG6Np\\\\\\\/snTC4U1M6i1pS+4vRm+uQYqRig\\\\\\\/ChWno1\\\\\\\/ZkiJ+1dwL3lFcXJNRGDYu\\\\\\\/OSpIX1Ou85TroY1dGvNNSjL81BADBPvm83DBTxqlWADiq9EEiL+ZeFPOJmi2O1Mz\\\\\\\/fUwAKky8ztxhuZ6S+TA6JhhwEi86gcV3Qs14LUJYWdG+ZD8HQBzuXIRnQMbye8WHKrZHEl03gyb6ca8I1zeoxpuDeZufTh8mhbO\\\\\\\/4YMgkzKpHQ8+M2WdE3s\\\\\\\/6A3kci27pF7fzxk2UYMRMtseRi57X2uLv\\\\\\\/hoX9rWOShgkm48A4D0P\\\\\\\/U4KRtkwKa3ziBIeBCWfRbXXdAEQNVgc5QQd5bRf0PuepLfvoBDaQ39u8KeRI1+MnTJdBfc0fsj\\\\\\\/NWa7\\\\\\\/FDSjJKNlZHMlHizUTVlpD9d\\\\\\\/o8q898MEOTJOQeepBpDl\\\\\\\/F8g6NKipPQriHIzqV5wodxs0ZlFLkxt1K8UWXtj2KADiKx9LvCYZopuDSq0W6kj+v6hdiIEteeZb\\\\\\\/bFd0ICGO4CGYp6dGdO+4E8oAYhKVgvUZ3YzXrmrdtaIixjzzKMQewoNpdCmkUANZDfE9Vfu+i7cEa8Of9rNA6zi3+nHSgMuiUiZh6iIihieTq3SjcAulLx75HGrV\\\\\\\/yRE+27\\\\\\\/wJ7\\\\\\\/RAP5T89tiTSE3fsKsISzl+SEFqHRyL\\\\\\\/E\\\\\\\/T91fDZJvGRh4LAdv9+onUsJ33QM6+xnhruR5n1eknpicp3qg0LfbLqwyVu\\\\\\\/EqhPMR\\\\\\\/olJ48Z4JQ\\\\\\\/OS\\\\\\\/fMZ\\\\\\\/\\\\\\\/dejfMDY+raYV++Q9JCMN3nQ1P9LO7EwKytug0gT6ZF1cE++lAc0fRat7d3AW9Gqk+UWqfv2XY89aAhVY6bu5cwlVn9mpBJg1uN1E1eI+AA2t4K0tbL6aPeXlO9mNB0edVE3e4zGc9xfcaoQPCov48p2KISv3TOulmMAv5F9tQHHaLXS8D8Ijnmetb8BbZwOzEoIaADk0Lt3zsDrKlcWwqGNoSV\\\\\\\/CtdRKcTp+qAzMR9VAehHkX3DJCkvGqe+kt\\\\\\\/WYBvOIEtYt7HPOsu5Xbosexv\\\\\\\/aFsMsqskMLn3JAi98OyqZpbrI\\\\\\\/iQNb4YaBL\\\\\\\/7JaGHdY+Ji9U3hikMu2\\\\\\\/pTY0lGWitjIf2Ty2lAHfbYoXUrrhH5pNNdDykSYy6OWDB0E2Qoi\\\\\\\/k+QcZ2XFBQJj4iAis7iziZ0Mg4Z3SqnUNBgnda2Xe+Xe+gBjdzz6mqf0+FuYIVbVtH2Iv0xv+rlfilp6tPDOojklOfhWjaYgNgrI9Ky+3AWvFztXMtSYYylzN7uEnhQOolFRtfQrDr7G6QvZ++iuZQANrFUDIuPnWmbDBrWmPPCFR7YnZuZA9hjEt7C1okjy+rzCweoRrwu0W4rrA9Df+2PBtLl1Ja+RUHfzAs2d4RHt3oGU844A3YKLFU+FCZMLuuO6RMoEbaJPQrOPbuBPUtcCMM2BfrTjKsp5MxDEv91DT6xlB+UlOLqR7xpPOnb+BbqcZKii5ycYukItVAKOjj3iUkzQZ0jybrWZnyL\\\\\\\/YyVu+E4OT+mecLyBKG5msh2l8g++bwFFWz+NNN8LBJaC6I72TrV6jfE+4ODRfITN4gQ9AgQHNhjzN84X0nStlRcVuGN0wjnRRjY\\\\\\\/C4e4cAixS73a3oiSd8WKpowfsi4pFrIPUcbF2Sfa4YTy4fB6fqpdhVwpnPe1d3e\\\\\\\/Z2ZNI6\\\\\\\/j0WXj65MjzBhV4KlZrLfl6iuscxJcywhkZPEDrU0WaO3UWqvF3k8Wqv9AJdRp34qGn68v\\\\\\\/SNX98p2rcy0k7X5TwxRyx7KC2idmvkSwrPmiUbLjTdf\\\\\\\/+awY9TZhfifcIl5StGT8w4\\\\\\\/NQ3uq\\\\\\\/eZCOAipab3e0WkCkbMHJfkBGCELNpncwiwup2yvuZTGZFKFK9CJxJr0z0KBq1ww5ovaywBI1aldL\\\\\\\/wEIBAPzp1N\\\\\\\/mKbonO+IyZBBa7V\\\\\\\/z8FDo7\\\\\\\/+ihlLPDDH+vUoVMYb2YeUSwyj7bg2LR4mT51P1Sl30d3v67arxxNx85bwOTqEx+Dy6m4sRBWqQkbM9s50wA1HXceLLevs94K+b+jKZuJNgXt1Xh6ouAtHhskox5I30acFmCZ\\\\\\\/E77nD6O5KUodCF8dDvGiiz7y+OpZ5YF4aHxLYh8Wmqol07oKT0p8zeRbJCPMgWoyMmdAM0owkgk4ikvM+NzFjRWzz691BRdK7TUFUY\\\\\\\/CPrlGCc1zyZWhOhyMX22JwxgqrVUH3KvAfrbQ+dud\\\\\\\/McLo1heauZl7PP1Jwraf2rH4XkyBo3eJyIV8V\\\\\\\/qnN2C1MjlnYwkXUOThWinIbp4zNtMKXgQgQCb8LyfQNa\\\\\\\/JIjIsJV1G3OfyKpm0UOo0Qa7hA5DZMqG6vTW\\\\\\\/NGVOIf1xnVg3JeIh9D3qLZhC0jxszz+XHGisNWiR6Luf5508HxDdk0dqMyd\\\\\\\/it1VkRaD7VaVysF7XU4WUdcJMf6vUD3JSdnB33nRerixUzpA+iPkUxbs6qhnVPAP6frgCwyokn+IfgFAOT62ZIbEMjVrAqrakOgF8ulOWk6OqIKu0UD4Dfxtn0r4rO0vq7aXukgfTsF9aof75dL2VcIjm\\\\\\\/+cbHZL29gIfpL8RkgG6wXQzxOGtXV\\\\\\\/PmdIb4mnNFe1AgzL4m3jyMEfnbixRDXm\\\\\\\/jevXJD8mMRXA+xT\\\\\\\/23TQtTy2UNIArWRCBOqdBexEQ3eA1uRM602\\\\\\\/WkejYtPIox1q6+8cyuQU8SyItBn5DcMLEWpYydxaxbkALEZud5xnT48PFUgJMlzKOT1Cz2Pjlbmep8yJpxGyOXos2Sn0k4eeqi3U0efQzMKaHXdSTWFNNMa3F+QMW1DKmUDpqwZA3AAW75BOj7miWCprvQQ4ylXrTgnUlmYLMmIuSpEWlc6qlJTLIqya7mYwU5AQ48r2T2FsJDSjEcRC3V+rO9ROIMkECEu+J374AZfJjm5hib452N+w6JKDK9RMFNFMgubeIIY2RQqgALuhoy9NhgAJstyFU\\\\\\\/LQOLTDk3sNQ86xNqVTkb4vpcmN98xlwZo\\\\\\\/S2pjkkHRuoanmSYdn2SgRzCEBjHlZzgzTl983nvNeZDBWTTkGtW8sEan\\\\\\\/f67K1EJ7TKXeQlzPyd4UkJXiMrV5j2tT31pLo9bMgToUU8pxhtFcU+dAhnM4Ii3yUQqg6bOZbaOKtF2B8Sdx\\\\\\\/c9Jj2TNVLBtrG6m4AdU9913wR22T5LawT3Lvvc326JH3Dr\\\\\\\/s2nu1cp9BqmyDkIUqjw1G7N0cvW1vZeDuQsg92OLkvixEPZeJtW5JzTs+z1ww0UV3XVVR\\\\\\\/yqAJTScBv2txPK28j26BYijr6BWrLMuwDWs0xf+CXhHfyPOCNwT0iFC01uskbIYq8rURp0KgWGZf5yC9au7feNtIsqimcimc39obAgA7Ht5dWeqj2y8OyGaccBo8+es4N+XhqNNtLozK6gww9viJ3A8ZYJ6rzapCf7q6RPvMJYy4tI2hiwE5b1zBUVk8gu4vpPIt6JXWNQq2Jwhg2ibIr8+qjaX3pEhToJVwdNalo\\\\\\\/NHZpQH0RCPRSAYXA6\\\\\\\/6AMRkn9ZniSexBh7kq+pCf35X+PZC7KK+BXGYrLDPlQ\\\\\\\/aTzIrotMPPh7I23+8HDqbXiqs5muboQye6K5G07oVXUmbxRLSSMtabozJWppaI8eijFzCdwv4NCo89K6qT4mb3taCx33iJHSJIp3\\\\\\\/jxyODNxL7th\\\\\\\/JBN6D4j4ce0l+HJhJe9nVYuR1D61Ne\\\\\\\/tw1+9dzpENUDTcZDiK1QmOhcWApNQ3cLFE4Z8rl812ihYiNXo4BJbQLGUzGEB\\\\\\\/sBn7uTjheDIbe\\\\\\\/jpeXkFp2mF+i2Xw\\\\\\\/FYiRQz6H+oA0DOJnbwRkU9hgvnzSZ46u2v55BiX3r2F6FG45OKuJ7pUMpMpmD8KjS\\\\\\\/9t4+K1f6kyHxcGKwOIv4gZ8knCFhWbVYQHplEuSCLg84dJCuYOlaEFbniBWBbhy1RLqP3ljBNJ2DrZYf0P9v\\\\\\\/ZWkpJ5q9utHzzqHoug3c\\\\\\\/13zuXBDF037LLfQSmr1MTlF2b8Nqy4TLCbYwh\\\\\\\/Pbrwn\\\\\\\/OCU9qUL3Z\\\\\\\/DitAIabCHqzWpqDB6BA9UoQY28Ae\\\\\\\/\\\\\\\/+yn98gEgGZ8FU86V4jblWIZg2Shkgc27GRfprF9xwtcqoVnX9\\\\\\\/5G68HniSOhpsXNdTKh37TskCk2\\\\\\\/vEiW6sqBntYA6XHoKcM1QMPwTiO\\\\\\\/3HufDfppYqnWrfkuIH6YaeWJaYBUkeguCjZj9oBaT1SP2RrsFpbxpd0k8rcKdnkEhYlDQ8D+ry63IQXHZ7ZNiKsgafN4nXaCMugrNDZsXjQGPeMs8MDuFAXh1pXGm3ULNyLS\\\\\\\/mGG\\\\\\\/9NJQycw7Iflez75uVYspj8iDKYiKyz1UtqYi7cbSbPWaIrwGXBQ+JjzwAeE5q29Q3rpU8\\\\\\\/Hmu1Aig3YbfSv7vNCT1cURHx99PJyY4aTUTIguDxm7a1ghO6bZx\\\\\\\/ovvbHiZQeiTWVFX91QXMbQ7a93wrx0WQf00CI5fJ5GCnDJPZFPpkymChCSWNSYihTF4d0OHKL9YA4\\\\\\\/dtIT5n1FGCXGxCBZUGn86ZqNjmYrVD6CFtmwX+zAAbSNtT64Rbzh8\\\\\\\/QCv5keVnRkibUwqvEpyCMpK6tF\\\\\\\/+nDI8usilkyaCYzKuCiEMWhIl4kSpkpt0Pbd8gcDCaBTezTfeqI0Mbe1vkMKsLcpKtb8PlgR528D3Z0WFkxNS66YwyzpAckrwgtxJwmdfyIqGK3gFIpxXOtA3aZgOdcdNvh9lM3we6YOTMf9ovk6RNyHJhclefYV\\\\\\\/9W8j0EEmvGJbWk4gAhKZGgM57Q808qT\\\\\\\/Kb1oBbBK+v6bOj1U4Yj+\\\\\\\/fozMK5o4MBnFIR5km6OQhoAmjD2PWsCHuoduRhs0fAPwQIyE204pcfQdEPWRMgJIv84Ur4tTEFFov7sW88LPyvVenJzTljWat8Klq8vmaPGNM\\\\\\\/UW100PLv20eQOFxXdip9scpRoLVzV2ekeuNchwWnI0tP5VmEcnawVwS8DoB7D\\\\\\\/pJ7sFuVWXa+bcHjVaJVMfEmlDfIOamoTkv3tAfvq1Ii8DL6Wq4t\\\\\\\/+XEkHu6bJrK0uouVQqi7LdFz65+cPmQPwxpJ9PqCPQGPpHEWQT1kUtoPMvHnZdXZ8MS0GDADJAfN2UkkIHcPGte6hi9jpCDX\\\\\\\/PJuUWrA28ukVWxpmfWA1Q9GJcaYTF3YLEFCFXJ+o8BZ1vg+i6jYcarWVTBOUFRxSNutIDUMG08gUupQvDGlLKKoWi5YZJV8zXQFrhWdpuI7FyOTzaYtJ744\\\\\\\/Xtx6WWAs8N9UQ9l0FByGigwT\\\\\\\/nUFXbFc7xNSUL4SpPde2c+b4LJiSLAUrftnarkZDkzX0CgoA4PvzwxnsaR9rV9RmsRC1KoJkOCF7b0\\\\\\\/N3N7F920e3FV5L0wT2shqp0ClXJ+1ASPf52S8cvzbr7busGV52MVcls+ThKAhFiWCLtWhBwCyA2D\\\\\\\/kNwyulgT5fwBfL30a1ruxJdutkcnZOoHlXdv8ZTzBg9SmZec33Z61m9XsINEBMvYKeHtuTW1aQYfqPQt\\\\\\\/ORAIKihtfroO7QwlpbX6VYXsQ25SCL1jLRUIa1UEa9klaZ3K02CW7YK1SeR9wBIDINaOVLP+MPtwnycneOCxTacW9FiZJO\\\\\\\/omqjDSYss9S0PJbaw+\\\\\\\/+Dx7tcR6JRQ+\\\\\\\/GHCPJjUrLh421IIpcUNt7ID90laq6ULP6ftgyjgiNCyAKZ\\\\\\\/pgmWRyjzGP2fBWihBPDLZ4\\\\\\\/bJ33e7IxmdgCjfdD62ZEXXpZeJXUfkCXQ2usqbvRgeAEDSQb0GVcmAid6TwzUucCju+OCAQ\\\\\\\/gOCn3qZEtV2mnPywcokv0ykfE65EYnB5pMzeYb37gYWiMKPOI7Iu0HEbxTOj+tAwa8x\\\\\\\/+xufjjxmpP6MP4rpjofESTBubOLyXyweL\\\\\\\/7A7h8VjaOXZbYXl5PZCdb8d8kosWTAH1GogU0oKbePhhEXlDdppvWFYkNbECzIH\\\\\\\/myDhuABjpiXU9+k6cMH1eNwTQ6m1YqaUYTdc8dr53kHIHXMI3zEOz26oPscy9bw6DslokLDYxwVTxLjZXe98B+8bvM8E326xGsSGlrc\\\\\\\/2bFU3UPruHcJdO9pJz6dlOa1QB60\\\\\\\/Kqj9AEE87aMXik8tHqOOEifAF2ipNDaY5UAtGd7I\\\\\\\/3WNvfx8IG+RJ6PVLEaEnchMlBNZeiJZDVfS46VKMIEH9DfWU4w3w26kKeXp56au23E\\\\\\\/WokeGaRg307wEU6yxj2w0tjaJ2iwPynBV0\\\\\\\/yHfEd5v3ohgU7xOekQk7G\\\\\\\/I5eQdExrzPRZ5xcQRkoSTGBW7gUo0Otq\\\\\\\/\\\\\\\/xvNjIyvP20GA1W7TTt5vK0mCKfkVIL\\\\\\\/Uz2IEVIqwhppVwKdSvPWo7cee3q5H+C7Vi3Kl7CLaNvnFv2eUqY5o0PPH+YWLUL5S\\\\\\\/VteZYdHosjUTXvw3gk\\\\\\\/UbJWHLc1iCRrgInGMQyUbgK1k9\\\\\\\/pNzeJNXYCdn26BWTJCXT+g3LZdsQGmUrsCcyKlCKBo86JjqJNXgGtbCDEELFAxJUJN5yO3pRaH1qwR152VICFadtIj48J\\\\\\\/uFMclWyhhleR2H0S4Gm7m\\\\\\\/+1pr5twXwj3ZzEeqw8PFqouCeZgFgC75VVBOikuFitheU1fHgKUS5Qhgd8I5q2\\\\\\\/OzJClJqzbX570FJVv7hXe4nEIEcf917N8+FA45PZvZOT9kqEqDcmMTEyd2U1na6apaLhgp8LBITqH6eSaWSd2AGPsaIseR5f8OcHyeqUniUzYBHDgS+FGTV+FuhTV0FNul7GmPNbwtwiS3YbJZ1WUkFOcS6UNVp8doOr8jF\\\\\\\/\\\\\\\/PNfcIyMGmzgEY0tprmwgH0xBOmGRev76iS7tihckeGbTjbGgV8sG6lhMYBFVbUEDAl5uiiNgbRL0jOJeyyZ\\\\\\\/30Qc0S4ybsAT4eRhzRxqovS+rVk2t0mSPEq84FBbJLxNrU7joKqzzhVZ+r3Qb9kPQ3ysR498PFDlOYl7YUudRQ4QnmSh668+49ua6ZEvw3JKLT9+3HbGozoTO9viNP3+GsJ0LgpMGDyjGLkc5RH6xhY9POxMjR2291WPbs1pJS1pKJJcsDVPioy2jJclPZ2TVbRZ+Sqtyidwi4Tdk\\\\\\\/txAmn2iJ0ypowEkzooMz1hdTy8a3MWrGKpQnQXHUAZ4bqM3F62Fma4UFVGSu\\\\\\\/FEBER+LvgHq1Sl7tXxCPTU8gg3NbsNnGVmJ1RPUQuMTiwbhB4lcsFgjXcc8S\\\\\\\/pMVifSjOJVkEm4XKhfhEnovjA7hk5ceZBixttbY+Rt0pOyBtDDei3EHugYtotAD8TlDP6XZO+HE7jj3wQaiE7rNQG2Kr5+aZnLdCns1HMkj3gdONdF89iSxWxTPzOD10\\\\\\\/kS\\\\\\\/Qdn2qc9FSwMvKLaEviIEu+ZUBpCtdOBV3HFRmP1VUS7jsAg8qs6Jre1WklXfctkWHCusbGYc7PMoxgn4JONk9ugJA8f\\\\\\\/CTVs4V3\\\\\\\/PdGeFI68+DkRHynSf3PBzv8dLNGblyPZ88tVR7mj2qrjE3PC5IGnqUrukMQWjTEO50Pk25uJ2KcPW6VObZ7BWb64KLkc+2KTtLl50ebbHRRxW1FR9mjP0wjeGqMLYTeQD8nNeCaSsD9HFPuX47DkGEHz6nlDb6BsRqUL7\\\\\\\/b3CTjtiaFmWAGKdY1QpMMfysfW9r7KhagHGPqY2KSMj2l8yeG7myQQKERtG4bVBcPXEWqu10KxgmzZzEXG+FHp5GnTWbea88+bccdJYSWwB0bFy0CkcZ3qO1GZ9fC\\\\\\\/YAA+TKGL8GgCvbC+UgdA9vmwoht\\\\\\\/QlfoAQU6x\\\\\\\/c9FqmGbk7BcEiqWApf2Ytz3cWVDFie5A3kBHO0HJaDYvqUcMovg40vc20bWsvCieIMpyxbzQ02Y+MKLRXzQKZ2pfrLpkObp76hqxJosGmu5D1kffs8PTm4aWBUStTgRijecYS9VZ9GYLsWRTeac\\\\\\\/MNbYZ+NLZHu9xK\\\\\\\/rivqNso2OELK0w3d0YEy4gfOvDgPWl9sbEZ4nJezZ9zf1kSlJTcN3kQu71ZA+iUt1nhvQdbNhaAjZZS\\\\\\\/2oq7FCjUTBD15Y65T3IKRmiuCkeu1v95Iz8TpVzQVyVGzJp+k6eIQ+XBSjaahITsHtn\\\\\\\/4DRdplKrpCL5jOrLXoNd1RRwUkb4qeqLBCkU6J63UKjI1VhmP+iJPSZ7XUs2gqm09xIcgQnq\\\\\\\/9lCzek5rDwbJUx\\\\\\\/si+iZipcrRd5bFZOBzQ9FAvydirO8\\\\\\\/3pzOwtrqkNzV6UKgRweE4luUl8v483OOwg3EMe5aXdCeumOC1IqpE+bae6SCQXg7QvGSP4potrhjW9CdJveV0aRgt6fei9PJXFkxtZ4Ww2Lvy8002tRO\\\\\\\/eFctDGjnDgCv0RizwgiqJLrfzZa\\\\\\\/64TwkJflJ0gFDxjwCzuFpwJjL2e\\\\\\\/L7cWNrw+izHmfIWF78yKzqhfwLSnaKL5LJZ2aToQWPJ7RuYfzhoJVh6kSk5HU7P\\\\\\\/evxyrSkh9oKZwVw657YgnafNsM2EAWswMWbKqLvxPDwRDIQbLJbc3Tw6rEKSJXSTmW6qoy1NPBSFbdyrqEvtNhqyLcQnOAti4vPcMiUaPgo8QIZpUuJyeZH7Qkpi38yq8g5qRMdlACnrA1eDAgmwVCQJ2D55zXSqOUXyvQNw4VaXfeuLW+PrD6MttsKABTXa5sFyZ7ltRy2BLHd2Qi3cPQLSgkzoH89Db3MsOcmAL4SvnicO13fecKuvpcXDMqBkLYqhTKE2UTtfuh20oQ8sqYKJaz2sdWrrUFDf4yke3Hi8VkqtOKIoZ5h5ykMCAE9\\\\\\\/o2D1aWFAFupD0jould3+X4BRVPfPa1SJ09VIf0KP9163LnS1RJkRd4jtKuFuQTtDk2Rn5jaxYUyfJ9PWXCJ2byMaL+gAKJHrJ8ETrWif4EqOL9e+R8ZITSEnehQil\\\\\\\/Ll+GMBXcP8wY8rHxURzvMLYxH2VPWha\\\\\\\/Fzgf8t9sLPuSyyt7lORbDRf0NStAd34g5BTb0BDJ\\\\\\\/6qtzJzzId+yuS\\\\\\\/Xnquwia7myFxXdPzgCgLupRndOL8TqKCP0kShraap7\\\\\\\/VHHVWWj9yZrEn4MLKL7XbJQx9ybPey6YkwIi1qByDDEAkgt8vrtquYkbQ4Us44yRZEICIgovPNzsvu6ly8LUmM10NRXFTW+gpGKVKIdEtoZyNFJpHHKnXw73RTWKT5vlImNJbW7uPh+cYkjNXyJAE2AW9WLU1R0vCSyAZvMe7S0CS1Lus1XvuWG6plWa6SBAPlCMlYdBFAEiXyko9RWCluw6qY4wFXdttFSOO9KFKv9xB7Dd0BHveXH0UpO0AOvX\\\\\\\/CkLh+VAYTFt3Cx06bgaaq522CXKB9FQ\\\\\\\/3y6iSLL+XlpNWSlDAa5BuulNZKvZJZXWMJiWqYHaoKNn430FYUUUSzRpoOs57B081M\\\\\\\/Fd+oxPuLRhzx3lZtzUkGrwjFn3Qvh\\\\\\\/qyM2ehJERRrH6tI\\\\\\\/q+luQD4rEML0F3jGIxREJR7fV7c\\\\\\\/uzArGyUWS3hV4lg+lOXarpe0r3suWAZhL\\\\\\\/qRzPFvVlm7ho4ioR4Ngjd9Y8+QAtlIgGaVWNt3BGJK\\\\\\\/w2zwHzdVKo3DEif40iwt02VMWZ+J+\\\\\\\/I4n1OBfaAgavTJL4nQTgv6EPNTRBBx0gMMNZ7zRUYh8hC40eF7SGrTqiu7hgAA1i53yDClI7KsBaV6kw2fJygWFRMRr44o00Q0E\\\\\\\/tDoeHuEg8Q3XvAvPw2F8OXJCzhf4X4VwrZCLd21\\\\\\\/QwXRk7uxetoOTMZY1fb3tHCXLbY5w4+rB0XtIrC+RMKQvoq9pWmjS\\\\\\\/d2bJRWlByOApgUhTkSngcCm2dClXVwbARgzDt3N6+nq5\\\\\\\/F3C8Mt6i3mBdVyNEd9n3OLa2LOJbIx8F1bYS7D\\\\\\\/nNIgWLzskOA\\\\\\\/wZ6i9Gp6HO\\\\\\\/bldi1e6Hvywh2suDZON9OjviMXf3mVTtgrnmOeASnISGR+bW1brt7CGfV2wQQfzZCDj0msRf0vEVX5ZrcIUuYRfchlUg+ap24txmDAQo+Dq89baEMnjc1MARnlju5gJjpgIS0QMcWjdLgOQaP\\\\\\\/ktTYUQPX9ZIstL8HJauzXPn78mHWvp4kF1mBrYM9xK6hyBb2DjfckE\\\\\\\/MBvNBb2INeJ9soPjLP8CLJ7Uq15k6l2dXufwcTU7w0ciWB1VF4MpKM9rSgekml+M4y7j4kprRRuEBjAFW5zXJ3a9\\\\\\\/K4hTHc0ZFXzdhI8UHKgYVljaMadwsGWJcPcSRqDMhkDSPweai16URZlN2gkQanr1t+03+3oI0UJvz0Q7s3sF1aL+o0jyEViVikvrt\\\\\\\/2tLsYocwesuRB2WvxzSBdW6TuX7ZKHqwHREo049jyXsrqQIr2LKdLy+3K2UuiALIqspBZUc97L1hWNf23TKr+rv6aiZD7+x0TF89saX27aR2KSNWeHEbWdxjOrAWJwGlLXJhHZYaY3Q8AoZhNJlfevR36nIvsc7im2XHDJmTQ3pScjy0ifO+xdCFeKjRiylIVFx2lPTBlwDxz9pM8fFmY3k7zbhJ6k2v0tkjkc9nfWnwe8U8jraRc1ZFhuq7grMefdjhTtQTCIXCvMQNhHT8bP8SDlH24tjTIynGJNsLjU5STSzRlMxsT\\\\\\\/79TkLGmu54yK1kD5frZuIK0I6d\\\\\\\/1FZxequAxgJ7RklMHN4gWWG6yyUy2r9wkNnCownpSZT0y1KX02tgmhTJiPH3xBcQKhvhCvxJsjbnCUmuvGWZ5\\\\\\\/EAmVQcrjcV6ltORM9vXufxJDPzqEgI7gZsTLg5e11PRIzHB2SnpXU3sUIg\\\\\\\/Y1Bw2EWK6XmM47AIWgRj7vIrkykD5cnMh9kvPD2VDnkgwOE2+tbu3RTYrGj4kFFVlyNPy9FonLWhZqWCOzuEUeWgFRI8FGjjGBVbXczRXxLxOR6MKkNORTWIYV1JI4y4a0gF7G2RmhEJAilFQAvWgF9HdMEdYH1YX7kjQzKmyDZONAmsVc8ZbIQi3S8ZhQWYQJqlZLj13pG8gM6i\\\\\\\/EBzB\\\\\\\/49qWLHesDJ6O27V1KabN7zciCyqUE9cF5\\\\\\\/sjOc6RT59ZthKZyHh94jv2x9psSifgn1vxCm5aazlZ7v2fdTY6L9bDSWP3inkGEeOXyFecBYrGLkTUyEqjgphYM9DJoH+5CV\\\\\\\/fAy\\\\\\\/UUKAS+kQ9G9It2wZbKu\\\\\\\/B5iTwy1kPeJ7RJsgcXFzjkph33yFCe7Jrji7GAZJtCARQtZ4\\\\\\\/bLk1R+8fFOZlt06zr21ptXr16nv9j5wNxlRykyhjdi8+u2vFIxVk32mDPleQk\\\\\\\/Ek5nZAS6\\\\\\\/y9m1wU3LmjQOEGKngsyrxCrpKRJ4ihMAE24gz1oJhbJFIfLwVC9MsUmUqdeBA2ojp+LHSRH6xpThd6BKdsZKpazFOV9tQDU4L0fvGk7tviBCnvl6+2CusBTJow18TvyrXEkSrRuvzUcZbIFKCWvEq3vzdFoR721WZ6jmFa4+M\\\\\\\/zhupyrkGeKQ798G3ZnifsZzq3Enhvbe8GFMjYhlHtzMZ\\\\\\\/0OOYQPtYRJNzLctYL\\\\\\\/pkDFpZ9sPLjP9RTkxOzLL7lJmACOp1dSknGJIcTLKewavk\\\\\\\/QcFQafmhB5UPdYodmjBav6M6vEiP9uDHt1hGeiBpklFWo2xIskc5vur0oX9axCimCvakGSEQRoW9nwFeuQ54b2PAx29J11k9LdNqh8qrp0onO\\\\\\\/iiHU0duvwwoEAHCqhIJJDlZIFw1XLexosPd0sue+HhGPRqUqNnAI1ERIdV0ZRFgPjDsK2b3ndNEoIslzZZRycwAY2tBORvbcCTsFPE3gcS8n7\\\\\\\/\\\\\\\/q+ukz+U4zoWz2fqFNsIOVCJqcKMkLRto7rnra2sT1CV2VGdFn1H0GsIcj93Ta6jBSLJEA0dnm3pjSzrJtTWZJNSJR2dZ1dN2oZ6lHaNZszoMAdi2zyy7rr4RhyoqL0iOufWewRBTZIHy9vuecYr5r4qaNsbYzGZEcFNbuN8nbSiHKdaa1tDyV8G5hKeda803\\\\\\\/Z6o78iqnx7xsNM4UH+oDqVzEN+8Cpi6n6JJfhgY\\\\\\\/i3n3T4OPe+tBWbaZV7o8Vc1ZlEQz71XIlOxtpu8Q7gOLS60Bf5gCzI8sveW6Ve+E9ZoF24sVj4NlNQ4T3iayeW+QkmctUt5dyU7sL0bJYdf4FhjEyQxka7Y1KE2ZGQtCqE0+sRHSLisLB+l9MakDWmmmT4exSATL+B\\\\\\\/JY39ZONRhcOF0cKv5D4OjvHiEQV5MZqHEvK+D81FeW7OLos1j1jzeZMxadIG4AOuat994MTuJLBSEpVm68B007HmmNjPbGO5sKV44VdCtMUK\\\\\\\/unn170gw3t7JurqeasGgq6XaZqJDUczGTvPki3KeOcIghB8+QYUQIFhjr+dLwCO0M3SUkV98av47nOHfNoGA9pfYfOHB8cMwNp4cm7qNXB+BNB0uDRt4dfyoYZHrLhuWDuNopK0ZEiuhFTmqeNOSxu\\\\\\\/GMaDyi1YY0LQ7Dc3MbuMzkhhJls+SiFhOtj7Elo3ZLLmLSY5z831yLV81x8UAqDub\\\\\\\/2h+YhX5krBMLQIi4OEBvRKplJhux\\\\\\\/q9hL6\\\\\\\/HopRmTvW\\\\\\\/0oIGYZDI6Rb7a4BENIf3mcGiNJcQQ9yb77rVZQW1MaWXat1E+VxaWFGjATc6K4UxLAQwXs41Xp91CKM\\\\\\\/tr08dQuuyg7PVrXrNXQzOwOeXzwJ0sKhbi1SoYOb7hp8o7+ntayFsrVd0BJ89stfr\\\\\\\/P563+0tMJf2LpLWkC\\\\\\\/zk0+VSaUuPwRh8HfSm\\\\\\\/7WIyJrDKQk7MF+TXud6gg\\\\\\\/WNZXKg5FbWMM8VuyO62BlvwwQvHhl8blZMcBXCdDmgxiT6EGToLGZFnWtTYfebE9M7KSWTZgVrDN3p6W3ryljiG+RVPLDkoEmaOHlyh4NuV71hEdYPzNd4I+lG7f56yjoptE2MTdXWFFEyde1\\\\\\\/BrV0F+UfuAscUxGI+Vbt9rq+Unn\\\\\\\/QsR9jDlk1VDoEvoEgyTmpGj4d\\\\\\\/uZhZZDk2sx6QMkJjsNr8rAV8aBOyM8VnKyeA\\\\\\\/7Xmz+hLO6Zp6TNRWBizNWGUi8JqoVTz8sY37k0+TTyURwVH30CG7ElGcjvxp68TdvLvi\\\\\\\/AzHJzw1g1KoYg7lEvH9+jmiVTKHa3\\\\\\\/0TOYF1Kc38lILMbo2KCZmqVdav234V2axX6psjpSleu+GfSfnphqjcF9dWieVGxXWeP48wXa5OEdV4jYiiPjZqq1tmyxFnqOUuSvW\\\\\\\/lywfyIltF7R5S6wCuoeAH\\\\\\\/1FqypcO3ey3G4zgwPjd5reuoKW7Z2XPEdWlvI3Oif5jil0i5VyEZ38+ZhBUU9oxGIO6Ze8JVjADOZxtaS6KCRu5DyFDBH7eddLeOwydfv7Gt3BGC8tK8Rhccmp3qBoTD2Ve\\\\\\\/cqxND3sTsgBYzDe62A\\\\\\\/di+HioanxRH6Sj5Zdql\\\\\\\/w4WyGw+qyKEwgR3OCxLv7SU9oTt+wWtVHNtF2rfsnxaxLPy2jgf0G7ZHnWTk1bW2cBksH7K5lQcqL+6d1+9+umwN8l8E6XZJo1KkmWv+Zyvf7s2PnCqpyEHiB5wmI4zM3dXoINCGzT1an89Vwr3VCgKDR7jE2OhAlZrPupXmwOXhvRu1MC6LuMFZZA0IZ+NFUVPoqVKISGYg1m3wAkajgowjl99l4HFNtfD8Yf6k29mnxfGXjjIE1f\\\\\\\/5KgPAAC8xVdqq0ZqawiaGyqeBtKWbLmoeDPvEcsmyLQuZW4r7TRDCfzzPCalS4qhW34cMnDvg2faeEell5WGFOXr7CSkVhOLKi9bDxSPlWJcmT5Nsgf+7Bk7uzE955MWj8c7jTh9qFoIRmzKGux8Lq3oIaMYKxZpJOWLB0tpwQQgcODlaSBKhwQskRSEwhEOTittaRUaKC0YwehrfhfQX68ygELmsVpxT78grQ17ofl\\\\\\\/yfznscnGX73Rc\\\\\\\/cdjrL3wn\\\\\\\/ULRDyaXAoYvR+iHSN8VdoqoI7K9r7mDECmQ7rEhZ0wrmQswwPWrz+AI5bHXSpiI7mSEZe2zSbjERcMHpt3B22qQFkp9MXZJqN3IHRQRFa\\\\\\\/x57SWGF6B0umEebLMlYUnm2URZJVrT3ckPp7EsTeuQsyGOYwDHi1GJEG0uXvtwv54AbPd6J\\\\\\\/2shUFx9Do0d5ba+kyBnf4+cEVeu7PFvPcz7rO2MXHAVJdwss6tDZuNRJf7htzcSnCi4knMoapXx629RucvUbEPHZ7bpE28PDhdnoWivzbiah\\\\\\\/legxDnCViIJ3SbOkmcYrArKnsxJgKEFlTnd9Clsnfmjwa\\\\\\\/xu+ZEqQy6fz3BId9BtOYlXhzz3g\\\\\\\/IruEPbQPRThZLs4E64F7LGOUWZv8Zdqo9s1ToN3HOfLBbOlxGTEV\\\\\\\/ZMZRik9AJ6T1omv+C+KfPQ88uvvJ2aQV7MAv2VNTG6Sw28QwdENGa5qFOzssJEWZVNMhpeQOTfEZ5cna2LPcuvWpKKX1iHh7Sc\\\\\\\/DHyiX8Z7KUQQRc0x9BQbuOqBxkT11VDDt\\\\\\\/lXhoDuOx\\\\\\\/DS8V\\\\\\\/bdzrzR1E2dBeLenPgGv5y5H8\\\\\\\/8cZI3Z7r50T+ujj6xLU0\\\\\\\/fFV1ZHcHbId3L3AWrDWr05DxG4YjOO5+J21WNmzcICmyUU16cGHEG5o\\\\\\\/bk4FqPKCAXJnm3oQxiHpUWwLGqVaIA4N7cJqz6gUC4BmBgXX20RAXEt8k2tZr2kgMRp903fOp3hP1lpXWqwOgIhY8Hqs5gwl2frDYzSWxSHY5GG\\\\\\\/OTep4\\\\\\\/jVs6cS9zKOGzZ87RNq2lxHlEq+SvBlNW89+A2e6mLKBegJkgK47wU\\\\\\\/pjxDdHT5ZvxpqvBfA5ZIu\\\\\\\/Vo88+ctES69Q0SuMxfF4B5uBW8UXF38GgVOSCQtXsRcBImkS7dM99UOxIPS6mjGWAvcB+Ol5vsbipdTx+GZHxwPWDkCaKKzC7dKCSLcaVZW0BmfPYhDuUvxoEJLsTjXvqiVdBHIY2l4aacAhdq4juPuoJzz387LUq\\\\\\\/Zk1HglP8o\\\\\\\/iUVGaGkM5CX6IRQW61YZFuxq2xRgP\\\\\\\/P05+bD1ncnWjarLPx91xEczR6jVpltP63vNL0Mm5jnrzsv9oZ0w0yxAs6Wv8dPEeGTQZg5FR+OmVkSQrU0I+z6sSj+mRx7HaDxCdqinjQ8YjVL71DhrNvmW7gfrONAz21d1XqOEyCM7f5JjDMPIgS5grFByEy7Elr4Ff+R6yOwT0zu8dccOdQTBWuEEyq3hW9Mfv6CcsdTOAzbgLXfpHuiDfAo7NAQH9dWoc9GWOBzdK+iI+AblmH8Uq5GpOHpi6ufoPluwEYfcw7S1AHl59HaimwnIZejqTmg1dkniy1AzX1HlvLfv3rTClM1bD3vdRi+YTS5gxEONB\\\\\\\/mAcvCU\\\\\\\/GOExDZxIX4ss+fKd4PvMGA+9BRrV\\\\\\\/3zoxbRbO4tzfYQfMdeSM8ctx8GV66wQUTTsDQb2sQ8\\\\\\\/hrYPWbV5mrnd2LoNKRTvggJJqwVwcCsHsSp5Kr6jXqnbUYT\\\\\\\/Wn4d7Ye7QN0MLv64a3eaHRmxNl6d1sVcJbBHGwXZnFa0Di1flg4HDy0pq1BHf0qeCuH8fURHqENwJ1yTbtU8Rtt\\\\\\\/3Vdh+OY7zemfXIEdyTPLrtMxnVLZnDR+j+A7hVNpXEKhid7oo\\\\\\\/QLABfgVf8XoAIO2ZPbLPULhAGQ8tTJvAy6DIXOibCBpNj5iltzVzS21dm4igowB\\\\\\\/sN1zKDAtKH7V00mgtl1cj8SFmkNve6RzDxK7jci41lyxy3DI4Ze5U00jXmoUo3ghhpcgGbxrA+ft7beI2gbXlI7PgXvuvHjzTWyMlabKROQajjxqRasf1QaXXSCXLvciNKBYJzz0Gw6fGVEj343mDDYUkX0nlO6ktSW+GHNjrsYoC5CVgAq86QdnHe7w7T81UckcPAf1MAeoQJlEeHwSD0zifqPx4WKTWhYO\\\\\\\/0U3Wl2zTdRiBRC8UKeEmdeyCQEwxVKcVI5D0x+4Mz7QWuZZqL6y7YiPqV6TvJsC6vXnM1uXbkxJqzW0XDNKzpqwUgvuVfIouOA9jPkSgQHMmmxd1vq6A0cs3j\\\\\\\/PMl2HHicyr+9uBTWR8xLF2VQj3NIvlToGV30Os\\\\\\\/JSbvFaooNYV35OvekNjMObw8SxxkdyJEDpF45hFkc3HLtoVsBh5cNJmzO2TP+rn2DkWRA8Lkx3t4u4WUfEztwWwbmJMxzhtam8\\\\\\\/bJOcOoDzxDU2T5ocUMZovIceM2WsO61scJ21+PLdk9l4b\\\\\\\/vw5EyKWkAA83o55LKqQ3QcVsTzEKxQgRME4Wfb+3+MgQluJSa6zHOcAHRT4BSKaiXT4O2s23BCF8JFh7B0wJ6opBXZzjrMUEgAb+t+FyozJsWxm0p52r3SdM+HFztypou4JZs+CtxlnJxanOYT77v9OutMWmPpf60Yta5BNEAo9t8otkHCha6cQPyhH9TfECicfkG0+nABdWTdqsKdRaQcy7uxM9TZBuOZ5A46rYTs3GQirGat9CuZ06IwDvhXjKrKgh8SPQdWm4KzuPI3Entzi27Jnhhca8NypJXMiUmgwI9xRVC++6PztHhie\\\\\\\/W26PRjH7p21ksFZ0+60+3GCPSoW\\\\\\\/v0f2W7R0VRpUxfdPClmVWkmjerC1\\\\\\\/MNYEhu64sMvIXilYF4edzLzHbH3xfYCr\\\\\\\/TIG62LK4pN0rCrsLMJyOryH3DSpoTSHlWF9zlkF1QrA5fLPmnUWCJ9SBcpqcf4MlQHb2DYZ3rhqb3oTct7\\\\\\\/Dn\\\\\\\/UEs09FJu0W\\\\\\\/hqYXDPP5oG6TFtJmwrs3EPPZfkhfN2ssNxTubSgULT5THr2iKFdbcmioBFSYDXRTKKbzGaZ1Dknszw6\\\\\\\/H7N\\\\\\\/6fTUP+LhPCq6ALz\\\\\\\/7JbpnbyPQYZReK35tSeHhdnguLXei9bx8ELGBrL4bzERbSMs1LIy9rwYW+6OqkNI5d7K397R6VO7WDixhh+qhVhM4Lgb4QAKhbX3s\\\\\\\/BWw5hc8+Fs7oyPRvyj0d54Ow8tUMcmYuhZvoY37SH4UL8CcauZtAklGMApfZiOieEF\\\\\\\/I+xljy8bYg6Y+zujxl1qRM4xbBe8bScmN\\\\\\\/iwaJKUjS2KUjYYKPGcbg+ausFjGJ252Jznl0+DRzOtOfhCvZEURISyu1n+kSuZlmse8X+uvrbTfP17fR6ESXY82BBYjQHxlqdLX5kqMvmhKSBJl0STbK6i4Swc3GMPJs8nSlBO7llIjCWxNwt4qhoiQcfcF+K+sSfMTUd7tvJB90uYuGv23vNXim\\\\\\\/3Ju\\\\\\\/qTDRDoyekbEojqW2h9jLVgXt36x0KJB5+ITp7ZVxsumKu5sI7HC\\\\\\\/w0FUWQdoHu+Y6HCu1RTXTUvQaEnqRMQ2lTFU0BvrigeVaK4um4StWhaOC5XgjPrCFXhrUokSdpJsCl\\\\\\\/C1h4yVxdMHikNsGO1jQF1wdkTf\\\\\\\/VynX07vtoveHXVIWYOpIYaB10W1p7lYqCKkGzaqh1YkqNg+PbKDlUJHzvKYMSvh\\\\\\\/Fs\\\\\\\/t9EoM5nH2Vrtt64awrWfW+wHEWVsC4QBjgu\\\\\\\/+w\\\\\\\/06AAX2QBN66kqUN4\\\\\\\/qI6nivGWM0ox8SBzbinFGJzVGO1fgCHoBnSLMGcLD7eW2ocqKpt\\\\\\\/dmr1dcjXPdsfZe05OKz\\\\\\\/vEQus5AtX+\\\\\\\/p\\\\\\\/gsfhK5ubuuYa66TRGDyYJd+hp9G1eBuwbkbSLI7\\\\\\\/SLfCfaAqFGw9KXLiO\\\\\\\/GiQUv4cgzQOerTVnfSu5QgGWSJIdrk+nJIGPeSb7LLyc9mj9rRah1i7kkbeU9HhrMLZrYBkXLVMTVY421+AIOjaSVLyhKWF9WzSQ9eMNnfuJ3ZwiT4on4dJ\\\\\\\/76kqB6KNS23gQZqinTHenDya+XUqrIjbcW3lbUSdxBCeriuBi9Z0Z5n\\\\\\\/gaeWVnPfq23gq\\\\\\\/RlfDbGaz0VP3cEyktKT93wgJafyZbSe884JO0kRcx1I2yBHD1i6pImDuFtDUKCIEoSXyMxcUi6mFONUP7Yjz13J7jp4VSagsrG3r4BX7aYlC6NNTB0xaqw7i9KF0gtChEa8PnlzOASDrOOH4l3vfwfP1onV6rDL74Nz+5jTwXS8hhdhyhWNk63L7YAjSzTSjHZl+3Sx2Hkm9B2oQpI8J6Cj+Jws8p\\\\\\\/TqWNhpphcwM4fA7lwPYcXEBPP0+5p7iF9RXEnFVoYLKeaiP8tAdK+PjQ+x5ld6TeMRMkddtVZxRngs7WXfBlVjt7LNHa6BTsXNuqoClQWOIRFQLGWxD0PxFqUrqV6Vd6tsMqXyOgMlxijygm0oCrFWvCIt3cuCZR\\\\\\\/2LkL0IiGkfobh1FiKC+C7gMZHJaSRi7Ih6cPKN7xIbNfMiU4D7uUFXN7Rx2G+ZZ2zYFzRfKU4XVOLF5tbUbJlDHMEhYi7YQeGUDklISL+sBZSvLCLsJOFp58Iu+hk4IURCAAMzsJgsNKCsAbhgiwX3AGEOHCuA3bkP3v6CQ6mXkaMel3KMfokmEv99Rpywpr5mnfoSnLMafy1MpZGtR1nKrd7BC0JNVPpnf2E2HyLBZA+rYdaaibMRge6wXAlwCxXgPGcRRj111Zre3yR4BymrEOwbqsz5hSsVKKHybkxJwxG5hFhibcIzw63L1Lrqg8yMLBgTRZoZVJxGVAt5Zw4BEkoiprBMO93f8YvWMqoiBd8ubfCoG36+FVwGdFAKZyB010mjDRN4ZmeCnJXGJuyPqMhe4WbpVQ9xxUpWJhMTtsYG8NBqfBdTlP03VOCG+b642wv5eOZtaCu1t4G2jpZe5I\\\\\\\/aTr9zRtERGMIcvWVua3RNuhd3IgJwuCtHj1v0Y8vW\\\\\\\/8OEvhrhiN9LO6ND0O+qfmA3WbuEUB3JE8rLTxV8D+BLiMtPUV2O3JWAmsF7yOnMkJcQ+M+cEY7cHizdUEQ0+syflOHVDZinpBhrbQJaiAnQSlJ5dRXpJwgKoziTxA3WvBMJ\\\\\\\/slc2RmRPgcWq2cBGiDpRkj606MUZB5zPV\\\\\\\/KwU2XlRqyOJioxz8Ojr3IYgLOWxz9VIrO25GfcV+lKUvSz41h2LiHE4iUyFOSagQxX0lN2SUzdSI4Z\\\\\\\/BN9EDC2o76gkwd\\\\\\\/ZyscLg9oE7FJxcTkCiRnv7XypN+Z5gMBuyY8JwBTXQMH8ro4tOehKlBuQ3WjmR0Wf8jcr3f9CVFsEVLabYi0kaCZqK2maM+5irNQN9XXhCU3G22NcVjc6d8N075wkMpOZX0GMAT28KHwNBpzKH1pvRQbNsRlsPF2B\\\\\\\/utkJz6UGFszpATnU5bc2z3caOWtAdvSq23q9j5u8+USlu+vVomAbUbUveKPZrVc8Z2eptJWuO0gVmmibCMfUraIxT7cVuwrIC2jIhc\\\\\\\/dwxRotVVQhZX4iDVqG\\\\\\\/x+CDB7xPnYUbXIOnUC5kxwA19p\\\\\\\/+pe\\\\\\\/GzUidmGpn91BJ2OoWr529WysWngz0iEm6ih3y67kf8r7F2gIrxgqapEOpPeXndCSXbCleTxi9JdXagaxT19XKhhmFjG+tjCTEs7GFDKLe+UcnppfMA0oXvAx7mVJWtHaxL8nzZ2pm6UUG6MLHinMNobMH9NUyfvmZbcDm8l3fxYP1HpBH80hbEyt0jBCgfqh26aheYnZ4dIc6joNkAh9pXlYyVU78WOdiwc5qxxNTtphVqSp8lrMe\\\\\\\/uuh+lLUsJBhZ9ByKctige7uRswXQFdqkcNqhR8AcXOwM\\\\\\\/f6oClICx9ps6uybU5CgMKIHcCG8b8VAqU0HARvulFANOeBCvK+Nw5OsLy+bVI0K0+43crUQODirnoy0vpAYovhHwxFuFLErW54n88RpIisC\\\\\\\/hzHmVB11AyUlDrWVD+QyY8GYcmBwhmRIFof762VLKsF4je7AOv+1PHKAgGJrb8uQO3WhvMB2fFve2gJuX7qFL8nFS2rPGEWZntAJwh2wvIKtOpM09GuH6mSKyGDotasHnK+Q7KoRgS8BomeTb9u5BqdvDzWJLTWi0NeV+16IaK6LqPk+9\\\\\\\/xpulD+OmhHJb6h1zgH18zFbLbvxP15ikPZ4JBe7YjAUnTz2NQij1ZTCtAUEDC69UgGQRDLlz0dkA2E1c+Zx52PVkuDRMuzOtfaGpQrLMCWaJnF7y29JbYcngZVfS3AZoPtLLn3Ecdmb3qFtuSJKR3OgvDXMglXjuN7M2ht3sGL\\\\\\\/gvMZHbgcUFb2eNT\\\\\\\/ce7edeew+bl9ORjdtQr1xpFJdnjxqA50IAm1fN3zksOBMikGNuJNISnGe\\\\\\\/JiLTyQuIp5tphOwheQ60NYKKXjBtqV++Fu8WVms2weBN0dtGEQ\\\\\\\/3EDRp1104UGQhXkvH606iojsizFPcWTH71VG9z9\\\\\\\/SK\\\\\\\/0lp1i8iybQholo327xscHj7YbAzgK0Qtj49FvzaMjVsTU5IMa1ci8andO2aUgVV\\\\\\\/rIW3L4ScAqSIO4WLnZSq7OYSLY3aKk3eEMmuCIySYRhwWJmhnYaySvWn0btMX6YKz2NN6tvDR5nN4DXUcGqOSiVXyAc3KuB\\\\\\\/y+lgoy2JOK7ddqUs\\\\\\\/TvV5LgO3p9epNy3McBZFONSHSt+BKxCGWM4HODvuZia6z5G6x1cWdQKfM2j4eZRfDO0w\\\\\\\/pYSLIYMP0gN4f0AE0IMUGiIbWhlbIbHv6sfwcCeryWqhKPLzEnGnL7wQl0vkFWBCeUzIKlkgqQRQrm48hkxhDq5gFyrwVtskZEp2cyvzgSN4uUrAzXUwDDXktkVJLFpoEcwwjBZJQCN4MwFONj5uRuMjBMCVWCHWMIl3uY\\\\\\\/SreZDwUn1ULuD+WRdQHmKTPdn6Cuw0ZigIIJcB9f\\\\\\\/6PI\\\\\\\/nSSxRJRJ3cEcBmdxLXXZpBxl5Xb7XhHxkue0pFt5neRcvCJPQEzslbzJyR7gSKRD1WUk0gPBM7aeJAx0h9rWEco7pnBlo92+X3F2Oi\\\\\\\/xBs8Vh70CmUZFkh8hRNEEvldZwGugGzjnMdD9gvjmj5NQsF2SNLoDugyhRA+X61DLrfQY984Zz2hD8x14ULy4ZsqBIhw1ylLlQ+hr\\\\\\\/KXnLmqT5RzoFizfrto1nzSYNLPHg\\\\\\\/Z7BuJLBCvio4pRLbfFEV6uhUNkHVztNX1mS8jilqhivYpN7NFNvyCyiruHxoqYV4tYqIA6oF+fY4v2ZVqMpfw6tERWvTE7SvAvTWwjt78pbjvEbDGmMZwkwc9AFXJ0UY++c9glBvoTXh7XOAY8aEFQok9piLXPP6IXLzdJNMBqzEMSDUPthLTGqayLRbNcLJKTl6g\\\\\\\/7SUzcMEVKPNOw2UvPbYhOrrdbj8ORqzZDo7yU4jcb+gHArr4eXB6kImLwGEUM\\\\\\\/8sPfDFAVri4bTVcqgisDsjsghcev+Z8XZg4R8W8yAJFTjCSwwPgC3+Gxe4Xr2qQ4U5zXbFfK62tDDriPikUOoRGJ5jK4Zsg2ANIeR2SGWEmrxiGFxv3n4KMjKTNklAVbHHcRS\\\\\\\/8apHxTXtH\\\\\\\/SYZGF0JJk49E+aQeqyxAXTLIUHXleoeeLslYOq1\\\\\\\/xCUpQC1rgLx\\\\\\\/8ARTy4W1tDQGDbtIejoQImO9uQaVzDsIFQ+UWgWjTZ5tQe6sozbI0Cy\\\\\\\/wzMJ1ZgO76nPGYcpz56BCvTU9MrYw+9Xx1hWwOeb1YtUfPcX2\\\\\\\/8RlS0BL3A66bZ5BocC6qmKHw34ZyTRvrA\\\\\\\/WMcAhfjWvB8BfU9LP2o4XKmmOQHO410LHvageKUalATwkDx5mi6E+UjSc2q5JMAHaaW8eItKO4w0gHRT4hNczeo2gi7JGTKP7Dz6tlgJAG\\\\\\\/6Ad2f0WyeORQTj\\\\\\\/mR5PHuyxoPNzdkpQP9pHYQnJNIeM6pa+Gr2qXKshFaUpbIijDlMC+MWkLtUQPWyI\\\\\\\/HPg1JtO7C0C78jKyOzzkfhQGAUPMjFvg2a979hCiV1Iyw+EiTnHJytGWncdUnW8Rd0Ksu+8BM4+kNRwRkz3QvaiX8FCpKWlbCDSWy3siSTWHkPyFwb5R1lNhTfCoKrhwJuujWRQouTMtlSRDWy5j+FDtMssjGNZ8QMEdzEQd4HHP6zQiDUz+l3aN7LsAMtfEwUjs3P3IFwSmV3r01P1QjitPMjllUBfEKPyTFqX60MM6uqfElBWJFcobqIW+I2JqVeiU57czOPkUSoPo7v\\\\\\\/LYLL\\\\\\\/OLl+yBwUeI8rnpefCI1D47PM3oCqBIEBwuNZe6hvWuX8er6arjCk5xhbqpbxtgrx1JntwFkR7DwNV1auNiUuUD63MExMG0LELPGS4ayGW4qtNrAd5EHNvCaWnoXMqUbxyE9ejIuPG7MbKezspvTc3hO8JAGxLetfyM6m5yIlNhI5sGQyBVTwA2u3DZIXiVHX0KWMf7zb764oBb\\\\\\\/AP3HZVJWn1u4ruvrQxsEfXQEy25T8Bh6KPXIB1GvZvNZtcyI9JlUMFLTbn+69CdacGbxUD2uiW9WTOlapeh80rjoktPBuWrH7rg9hj0VxyLAIjpwnoEyToE5IlWvgFTdfMat1k4fk0zN6sVmXJksaxviR3RiqTathrN8XxNK6AmIqM4pajGxyXPzb6QFu2\\\\\\\/jr7BUAKqLKX7IuhjYOysE+aoHTd6pRTtdosFO8S5Qka9JNJikUpRwUw+OTrC12tSCOaG9HXqYBwTbcbK+fMxs2VeM+Q\\\\\\\/Pa9AMS10jGU5lZqU\\\\\\\/f4HbwAyk6YzM+hZ1K+VVK3CfToibTqD6qTaU48EQ1vyJk7\\\\\\\/ZbieYf+C3UqEjEeIPkelGz7a6q1scn47pAyyQ8xXV6Be3sSVRjwE4TYMzZ0KIOAtb8xCJ3JpFNNuFbVexGhsT\\\\\\\/DWwgnqYgvYG4QXnoZh2vpWamZQMJHrdHjYjWbGw\\\\\\\/3pS4RRJzTfA9y4sPfG9zNrPkUt098OhHUYgoiy4OeAFXk16DiiNKUFBMUw1PmCSrkJ802M21fJb8vkmyEXYgdQrVCjstAmMmb38TAZtyJOdyRCbuyHhe4M7gVpSr11Y9RvHbBSyI0OpD0OgKctfgM58+DK96AcXuIkLp4Bf+SjmbSsx87o5pO7A1tmAKj+AjEPKQ3jdxXSfRy8TF4nvkWWpR0vGg4GTX5sPCdEg79y0cZW\\\\\\\/87SmhT4EC8aeo\\\\\\\/FnzfC6d40\\\\\\\/oH0m6bAZPO9S5oB6kmRhpPyqqzdD1oPqNqV1a6C7JcaxjMv+a6W2cGfzLHC1xSCU56rJfx7\\\\\\\/oMP6rzETE3GI3Kw8RXruZfNvSCRV2hGgRjvZIi9oPus0Fw+dRqqmUGeXKCsH4Y3bdVhAlUwSusYWR3wU1yMMKyPcQsG\\\\\\\/DMPCV\\\\\\\/YuhPxxMAVj\\\\\\\/E5qGFhUOj31UBx4zAM51qpUbXREcgs0Uz+61MX0pUxzwP2edkgzeclikyBmnNbpXTaTFz1RJhbN4Y\\\\\\\/zca5WXoFuDKxHaqHatRBbAlb8xupy4MQ4mJ1VpBcvpsE5p4mtI6pBhc3KEPxaGhNReYmLArifVarRYzBBA9Di58ZT8qeeIuf96K9Gr1AjsBmkMS+8iT\\\\\\\/MytUunhZBLlThBFyWxwTtv1BtDNT1bGF1P4RFYal8fI8aXGGmSgblbv1N9AOEWEwFZfTgKUhgUkzMDZEo3jty1hkectQC0ThTXlfH9DyoALZKznAbXD+1+ErLoe6NxJXpacmJ+226IUdKLi3tTJMfcRhgfo67BcrO\\\\\\\/CjHUYj92FeOok\\\\\\\/siJLO+tuGWpsFyfOYs8HqTe8dyMfvWJD4MUdn084Ld5nUKF\\\\\\\/yZCzABEbgoN1MmYppRtS88qQKNgcQpRVePD1YOfMxxnbEbbJY8bjreoS6hm63kDcKRFoUrFJ2Nnl8WarAXyEEmuCagN04rDc\\\\\\\/Kv\\\\\\\/FvNHRVD5ediqi1nDA7yvK5jzJ917gnoqdV2Uu56WQXQUEvAuUet42tMdGz8iTWX74zc3pd1E3lrNnux1CTagpC6HgCTuQByE3YhkfSwNucn8m5QsKkuC6C5wle9l0PMLgisLrkai2BJLGfMjWoTfnrjdxDao3hkREeDxgdiJqBfkAQqVQPCDuBG3\\\\\\\/lfrExw5CWZScRpmBD+1+Vs+l\\\\\\\/odtOeM3tKqiaJUJ0Qcpp6Pa1uSmrWIAW1rfNM5kYc\\\\\\\/PcvzbyMxpE4oI+H+7dXItPD8uKRFUZYoI0iXAETSWIDPqgHDDdJJmKO9IOUx\\\\\\\/eExIzzgt0mUQbRaHXnw2Hwxvt0HFfG1nryvJHPtKJs3WwD22yCaZ8AhGcRofFtquOmy0cMb8kHqsE4zI\\\\\\\/ohFz9MnSt3bWKogn3Eip+a5hKePYy+OSsKQTsjuWo6OPWjoM+GxuWNvAY998NvrhsFuc90yjKsqvDwWSxIf5\\\\\\\/vYrkxTAVeuNyt6ESvTidF2kYSlOyVlYYn9CB5WoRV451DOJ4qoRj6bA710ZUFmUOkQLKX6mzn0SOuUQPn2KxCIySaLGlDrdzIz3liKqhUDfH3GJe6P72fNqWx7aNkw\\\\\\\/5bSTDzDD2lLaaNqvo2t4+\\\\\\\/\\\\\\\/vWvanONTlZuDKRc5sdDuoe6W0+tmFeX\\\\\\\/ccbuViYCqjshaagHLh6UDIsv2H4oAygeCVBXgQ6EggmqlETvJAePvj54xxmtMysu+5xUqjj6UizKLFtXAsbJqIQqhUHVOsEHAd2sdiWgFAFP8m6JCoZRgTAE6bXdQgnBcF0NzqQ3RoIiJTvPcNQFR38cjLM69WlTdAW7lFQWgspAAXYl1ktK\\\\\\\/vaCBlIbfTBJjCDKf6ss7yfU+eScNNqAPlG\\\\\\\/oxBhvhx9+Ezd0ICbMUcrFdQoDvLTEwDrR1LD70uc+z9gmQpXsNpwMAJ9n5W9ST9FRazHYd+hsNQ2bZH0Ye+VD8lbpDDVRLmwEpQ8F+5l20vu+P3EJDvdwaMHA\\\\\\\/W5ddtY2dn1T\\\\\\\/chEedT\\\\\\\/5RTTpFYabqRSbVUsJ3apVXToj\\\\\\\/SSZy6rc5UdgHyvv1mNuwJS3+jyQdPVDMkfySPKRjgyUtPSufbeOff\\\\\\\/jUw3UriXip7d9PtA8mueo\\\\\\\/JPpZCAdAHpluZ31gB8iVMl3lt2kVd5Ck+UaQ1TPgH4SRQFIQj3sE3mliLRuRUEMAQ0Pi4C5yGN80A3AaHkB1e2IlHCmfRxLfAlII84i+q3lJtbisgEdSi5DBNRDHmFGd6J7HRfMFKM7hR6E3ai6L8Po\\\\\\\/5N1KDLbrGspVyx3roqd33WctXFAYGrpnJs5z71dvuhcPp8k3+DB50hm0bMl7+fPVC+52YQRjxEGvI\\\\\\\/qMdHZ1+6ZN6vEPbAnazKoH+v45ZtIshUnmNKbl9+cEKiv\\\\\\\/p8q8G5S1h3LAeF7ptZwaH5TMK8jzFT2lL4\\\\\\\/iOu8oMkT5I4H0vsmUWhPI40BWYBttK4JrLk6PhY5QanZqkhanw4MwNq0DtFyOR\\\\\\\/9bsVP3GUFu+\\\\\\\/RyfAJdWZlHY6uUGer9KxPVMTuSgCo0RoOHVG\\\\\\\/cHa196825z\\\\\\\/RZkgeadransm04IDac3nqsxSFv2JWORyXz7M+weXboABc3EJl1hCePS3SYQQ6ZAvL85uF+0pNs\\\\\\\/tALrhNzaZa9+OmbLViMMXIT+9IUcZGyCywFHNiUZmx7zQWUJsvdSP2Im0IBLK3tD8paWoKdjY3F8EUfGJgmyzuwbEvveh51R4U66wrcXUdWCHft10\\\\\\\/9+ZVAuhKnQthHaM4z9oyPfkgZK1z2WW3A39DgndEbHaaAh0TeG+IqLaUbbSfxeRkhA2Te8q4MOjJHJEDpR8IV\\\\\\\/5F4Cf9efN5V1rH\\\\\\\/\\\\\\\/5GstbsmlTBA+YL11TRtdy9rQ0jlro\\\\\\\/ydgmw9nOsbOmHQ2dum0hOt\\\\\\\/YKepLLjCtbij5qxjbfOVTIk1Qceg4SYiRmSCpbh0iYeTbdSUdJyIidwUcBBzwdE6luu9qOoOLSrJIH06RNeOOvZu2nJyhVl559aCEafuy+iOacnQBbg03U9DQmF5TMYkXd8qv7PiKT7ONmvVBbklrTHVUsSBC2r1fWF06FGe\\\\\\\/Jc80aJYGlVpRryJrYeiEWZhK+1ypunIkH7mBNXOJ9U0il7VTzH1ZkzyrTtJRi3Ec7pOPIRFkIxePod\\\\\\\/EK4aYzPSyezSJwRcP\\\\\\\/O9YAmuXlCcvAx7KM4POOEouDetJQu0W4YLfQO4ME+qgRLhzuWvIeabQfUo4LKPG8awARQ+qnvoJ0iJmvVrz+e8f\\\\\\\/qONrbPxX2WZ13J4nerm5xQCqXJHkfY1vRl0iAr6q3mvyNVOm0KstLRVsno11jyMM9THrW5n16UJiQ5ZrAYWhbzzfKbaQRUbA7WUz45qhDfBvImb8DLScdZhSo+qDYMhIRsitGFN41gs\\\\\\\/UhZ\\\\\\\/gfOz3FyUQDFAK+AemEg5eDqsWsdPmzp2hWBmHZ7H2cXsqv4RYaBaCl9go8ytzgglBZieon\\\\\\\/XQoF+OYYpc9DKEqtGppCQr5J\\\\\\\/3cS22WdWpotiEW5xb0EVpAuqAcEyPJduykZDSxnmi+Xrs4czGNohTwMbp0XM1VuGE3uo\\\\\\\/aVpPBamzImF3iXjDXi+jymCy5fuiI3Ic3bAkyBW0P9fRYdsDRMqiu7iZRJ1cP8KpEEeaD0jJ0BoICr\\\\\\\/sRFU5DuNZ5ZrAX8YcDkHBRa1W2tfGiLQeEjk7dKgJMKThvp86KMUyePHptwCle2GwInGfTsmoymzEPzRr1P8FZ5EIoAOMsMQ573aNhFJTbgsfCwiafm6zHavWnsP7ePI+s830i3bn\\\\\\\/TsqOfgAsnuem5450vtgwUv1Z5QXGqbtG9xFFWnYhdzsWpTWBaMl1FP2ExWW0siIhzQqHoBecolsF+E6sJVHQNJtZLNRI49xuB1VzF7LWGJQ5Dv7Wy2kQCHDemDI03d7eJfYI2N3Kp70lRTP0+qAVZmCiD7BJrvsufbkLOUq1UZ\\\\\\\/cR2f72do2ZL\\\\\\\/RlYSTpmxCosK1jQrcS34NJBvdzv4vyK+KP4w9PsSurgj4Bifddk35fOd3c\\\\\\\/eojKlHyNL28lsmZDVfD9W9VU2FM2PZ7Eq\\\\\\\/ztwfk1eKSRxVXkV5ziTM848dMri7QZOlZ0ndkk9Oh6WxTBnQlVqNTfSkXzuSpRqqahDhz4D\\\\\\\/+nCTN+8Rb3M2svzSDpb1tOV8McvPm9pNRgQYmhDRqKhXaqMWk2E9vjxxZ5SWRaZ9s2zPnX0HcEgA8XsTVFQrj1F6057gx4JQWq7uOABHlRLTzCdWcvfVF3XBDsbS\\\\\\\/+MgfBuDLC\\\\\\\/UHROqyPozoBCHfMnwjc57Xegls8KS5W+j5T19UPl0Cdy5AKjeBSoKu7a5zAKYmW+eI5ECpuHvXNpKzN7QwVvm08AuBfF+A0SMm0stFZP9sh5oAy966ox3Z+eL86YtUQf\\\u0022,\\\u0022iv\\\u0022:\\\u0022rOGa39Opr9N2Kwo35+r+8w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a5b47fe5d9f4aa480380d9afeaf68301e13db87579fa435f218f8b4f62418f21\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Nl5JvDTBmfXFBbE8r4Tniv3okPyFCCoMXvGtB33TiGUBPSftrexCpSMBlfB\\\\\\\/MPGkcyQ3HTnXkfCxSYBe7iIubVUiBj08OuFdB++gWOtcSV6gr3lcFSvh4UWGrWRkJu5GuNMCMzlzt0CBQxM2kkkT\\\\\\\/kQ+kamG+CrpIb6SjhbEFWJxBLAhUjEna4N0ersV+rLK0zCH18b+fvas2Q+kgH3CUIy0nvhK2C5b\\\\\\\/3Mjmk3ypmAqzvrUvPmACY3yfo3dFLEfONphQ2GXwVnq8MSgyG0Q5Jiow4szpbkKvX2PnRZJ7Lnh2zls3nieJMhfteIMf2iGmaLxVZASGmdsxl10n0jwXDmYnT34SggvCRQf6zxPRTVhseUKRr0BEQ0NEE68dCoLo+VmqE9aPgS5naq9TI9X3vOdBkU9bG3ngvBYmx\\\\\\\/zzvHCS+nncfA2l8YaoVJCwskMpQmUi9OVltARuID9QgbRzhW7kcAUhmZZ+QuUAjeEUUQ8M0H0P5L1k67WZxS1LoueHm0Xi279942gSKTumkhB0Eltl4axr456KXNzx5HQYED5TomDHExeHu1zTE7tR8ruzrJxcx+Py7lPyYyC1s1tlG+qqPlPdZP254mdQkMIFRwaYAVybUFa2WOEeUFFSNx88ZUtMpINvddkPuTP1gogCCQh\\\\\\\/ndFVuuJyEj5nmzvxc6AwYSPBaVuHHo75LBogC7mvVuiQsbngJ\\\\\\\/Y+6GbqnYIx8xvw0D8eYfQnLipmSIrhcNBCSObpqbUz88YPUcTUciSsUe2VLzuRl0uQ1SYTiRsD0s\\\\\\\/62FqIxv2fNyWLz0kYaOm0z4+l340Ti8+t1IUb9BckwzuAHyAjMwFIU5\\\\\\\/Xgzc9RPE61HJNsUn0IznJgJVltArRNXCdDjnWnmTRrtRn77Lw4e3gv61hzn753pK9F+iaUdcjv4YPHZ9Ck44luW\\\\\\\/sotQTuEO9rcTtW7KrtwFHRXj+TF5PpCLI7GJHQpR3D6B+SU5fPun9j86moBdS2ji5ZuvxK0D5igYEFvnqC7kY4JhqprEyFn2Flail0aust6cbwtzo5rjo2LTgxezOuRCOvt0zcMMHkAa2xXjrGoQR7lwUatrK6eUKPVK9UsqsZS5ykLXcnP24XFXI7f\\\\\\\/gSbS0Xhc4iAYbEWzNzUBjDKB2bRoEhkKaVAyfCWW1yNP1txojyen1a2IPL4NmuQNHkElXP2EhTjzEXXX2HDrm8BfFQxw9FzLw5xCylHpXozVy6UM1Hd7fncAXfNjC5v8S1zlNcrqrHP\\\\\\\/AaJ86T6ujD56DsUp5Nx4pi7iyaJ33lCEORpX\\\\\\\/POuOlZGVDuy6fdP+DWAD7eZQiB0cnxK6R6Q1+oSzDEpqvHtDOY8SZvZf76+n7+kJgcsUKIUmZZ2Arch\\\\\\\/8MXVgayD2lzvXrt+t2KEp5MSqNlfaJVf5pi\\\\\\\/YFrz\\\\\\\/npXyX1+qc3pzr0NDh+lyOxGySDEMg1IvFcsKjzFl0aubFjLBGvRQAdInyS+tmTVFqB36x56PY8ZCkwbpg3mAwRNsD+WW69tcwqwEgIYGPRXCR2s59yJKygOVjpsGSNfN92mg8hjCyEJzWtwTxbjvyOVpgRrWeqqC6VAwmGSoBIhOdn4XUKdfnC92apn7cQOyXS0LOzT9OiLXrsS\\\\\\\/AWqoKkuAzphFp4AD+Kr\\\\\\\/TgpDIvRXPS6rKGUwN4GKbTcX4qQHSQCH9YWqX5A4XSKGhKe2lXvyv5kKQ3coA8VE4PW+37gwSwBWAQZokG1nSb1TDTz9cBZeFI7ndM6EaQrQGd+B6waqwGylONgJVJFvh+88hm\\\\\\\/m2xWYbEWNUkG7rZGsi7uAqHvgIS5SUPeiuTNE9kCqbG5ZRSH9N2xOxyW9kOrEtPIw3DbxLAIFW9udGknMMLSYpFd22O+aGFvllMxefyFHcFpJ9XRDc\\\\\\\/nuczaGSQ\\\\\\\/QB39S3JUgX6l3FNOWbvayFlBZqAL2IkOewpMW88xgobcRp\\\\\\\/\\\\\\\/qSQwuybDuPlATIqFHny9gtHHr73PmrJ18M2AoGJzF7YXnWzP7f0TiYXpeTrN8UBo4lLokKjdo1LQQ36T9QQ9F4COyCM\\\u0022,\\\u0022iv\\\u0022:\\\u00222QpWK9pCaxzOnAM21DSI6Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022860513ff6975188ca328b434f3f955eeb32b4793eb518bc649311ae9d02e5b0d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459696 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/10/1971" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-l3cw3x3rlH7_fl05cSk0oW2HF4OWS1E0kzFaf0VXrvA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/10/1971" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dgGPQDQDpqn2IBGohrqpL0_kZtvtcxySkM9rNMZgGcA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/10/1963" title="Resolution of Genetic Map Expansion Caused by Excess Heterozygosity in Plant Recombinant Inbred Populations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/10/1981" title="Small RNA Expression from the Human Macrosatellite DXZ4" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459696" id="node459696" data-pisa="ggg;4/10/1971" data-pisa-master="ggg;g3.114.012245" data-apath="/ggg/4/10/1971.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Saturated Genetic Linkage Map of Autotetraploid Alfalfa (<em>Medicago sativa</em> L.) Developed Using Genotyping-by-Sequencing Is Highly Syntenous with the <em>Medicago truncatula</em> Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xuehui Li</span>, <span class="highwire-citation-author" data-delta="1">Yanling Wei</span>, <span class="highwire-citation-author" data-delta="2">Ananta Acharya</span>, <span class="highwire-citation-author" data-delta="3">Qingzhen Jiang</span>, <span class="highwire-citation-author" data-delta="4">Junmei Kang</span> and <span class="highwire-citation-author" data-delta="5">E. Charles Brummer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1971-1979;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012245 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459696" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xuehui Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuehui%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXuehui%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yanling Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanling%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYanling%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ananta Acharya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ananta%2BAcharya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Acharya%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnanta%2BAcharya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Qingzhen Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingzhen%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingzhen%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Junmei Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Institute of Animal Science, Chinese Academy of Agricultural Science, Beijing, China 100193</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junmei%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJunmei%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">E. Charles Brummer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Breeding Center and Department of Plant Sciences, The University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=E.%2BCharles%2BBrummer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brummer%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AE.%2BCharles%2BBrummer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ecbrummer@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">ecbrummer@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/10/1971" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459696" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459696/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/10/1971.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459696" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459696/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/10/1971.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459696" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459696/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/10/1971.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459696" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459696/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/10/1971.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459696" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459696/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">A genetic linkage map is a valuable tool for quantitative trait locus mapping, map-based gene cloning, comparative mapping, and whole-genome assembly. Alfalfa, one of the most important forage crops in the world, is autotetraploid, allogamous, and highly heterozygous, characteristics that have impeded the construction of a high-density linkage map using traditional genetic marker systems. Using genotyping-by-sequencing (GBS), we constructed low-cost, reasonably high-density linkage maps for both maternal and paternal parental genomes of an autotetraploid alfalfa F<sub>1</sub> population. The resulting maps contain 3591 single-nucleotide polymorphism markers on 64 linkage groups across both parents, with an average density of one marker per 1.5 and 1.0 cM for the maternal and paternal haplotype maps, respectively. Chromosome assignments were made based on homology of markers to the <em>M. truncatula</em> genome. Four linkage groups representing the four haplotypes of each alfalfa chromosome were assigned to each of the eight <em>Medicago</em> chromosomes in both the maternal and paternal parents. The alfalfa linkage groups were highly syntenous with <em>M. truncatula</em>, and clearly identified the known translocation between Chromosomes 4 and 8. In addition, a small inversion on Chromosome 1 was identified between <em>M. truncatula</em> and <em>M. sativa</em>. GBS enabled us to develop a saturated linkage map for alfalfa that greatly improved genome coverage relative to previous maps and that will facilitate investigation of genome structure. GBS could be used in breeding populations to accelerate molecular breeding in alfalfa.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-linkage-map" class="hw-term hw-article-keyword hw-article-keyword-genetic-linkage-map" rel="nofollow">genetic linkage map</a></li><li class="kwd"><a href="/keyword/autotetraploid" class="hw-term hw-article-keyword hw-article-keyword-autotetraploid" rel="nofollow">autotetraploid</a></li><li class="kwd"><a href="/keyword/alfalfa" class="hw-term hw-article-keyword hw-article-keyword-alfalfa" rel="nofollow">alfalfa</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li></ul><p id="p-4">Alfalfa is a cool-season forage legume, grown on about 30 million hectares from cold-temperate to subtropical regions throughout the world (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Michaud <em>et al.</em> 1988</a>). Alfalfa germplasm can be differentiated based on its autumn (fall) dormancy (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Teuber <em>et al.</em> 1998</a>). Dormant alfalfa genotypes reduce and/or cease growth in autumn as temperature and photoperiod decrease, but this hardening enables the plant to survive subzero freezing temperatures. Nondormant alfalfa can grow and flower throughout the year in favorable climates, but this germplasm has limited cold tolerance.</p><p id="p-5">Cultivated alfalfa is a tetrasomic tetraploid (2n = 4x = 32) with a basic chromosome number of eight and a genome size of 800−1000 Mbp (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blondon <em>et al.</em> 1994</a>). Genetic and genomic resources have been widely explored and developed (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li and Brummer 2012</a>), including genetic linkage maps for both wild diploid and cultivated tetraploid alfalfa. Most published genetic linkage maps were framework maps populated by, at most, a few hundred molecular markers, typically simple sequence repeats (SSR), which failed to fully saturate all four homologous chromosomes. A high-density linkage map could facilitate quantitative trait locus (QTL) mapping, map-based cloning of genes, and comparative genome analysis and guide the assembly of an alfalfa whole-genome sequence (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Paterson <em>et al.</em> 2009</a>).</p><p id="p-6">Single-nucleotide polymorphism (SNP) discovery and high-throughput SNP array genotyping technologies have led to the development of high-density linkage maps in apple (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Antanaviciute <em>et al.</em> 2012</a>), oilseed rape (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Delourme <em>et al.</em> 2013</a>), pine (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chancerel <em>et al.</em> 2013</a>), potato (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Felcher <em>et al.</em> 2012</a>), sunflower (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bowers <em>et al.</em> 2012</a>), and tomato (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sim <em>et al.</em> 2012</a>). More than 1 million SNPs have been discovered in alfalfa using transcriptome sequencing of 31 genotypes (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Han <em>et al.</em> 2011</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2012</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Yang <em>et al.</em> 2011</a>). An alfalfa Illumina Infinium SNP array with 10,000 SNPs has been developed, of which nearly 8000 produced reliable data in an initial experiment (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li <em>et al.</em> 2014</a>). However, for a biparental mapping population, only markers that are polymorphic within and/or between parents can be used for genetic linkage map construction. For autotetraploid and highly heterozygous cultivated alfalfa, only biallelic SNP markers with certain segregation patterns such as simplex (ABBB×BBBB), duplex (AABB×BBBB), or double-simplex (ABBB×ABBB) can be used effectively for map construction due to tetrasomic inheritance. With TetraploidMap, markers with these three segregation patterns could be mapped, but no more than 800 total markers and no more than 50 markers per linkage group can be mapped at one time (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hackett <em>et al.</em> 2007</a>). With the use of a pseudo-testcross strategy, more markers can be mapped using software designed for diploid species, such as JoinMap (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Van Ooijen 2011</a>), but only markers with simplex segregation patterns can be used. Our preliminary result indicated that only approximately 25% of the 8000 markers on the alfalfa SNP array (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Li <em>et al.</em> 2014</a>) were polymorphic in one of the parents we used in this experiment and only a portion of them would be simplex and useful for map construction using a pseudo-testcross strategy (X. Li and E. C. Brummer, unpublished results). Given the low number of useful markers for a given population and the high development, production, and assay costs, arrays are not ideal for constructing saturated maps in alfalfa or other autopolyploids.</p><p id="p-7">Genotyping-by-sequencing (GBS) is a high-throughput genotyping platform that integrates SNP discovery and genotype calling into one step (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Elshire <em>et al.</em> 2011</a>). By reducing the genome to a subset of regions (<em>e.g.</em>, restriction enzyme recognition sites), many individuals can be sequenced at the same loci, and the sequences themselves can be assembled and serve as a basis for genotype calling. The reduced representation of the genome and the barcoding of each individual being assayed enables multiple samples to be sequenced in one lane, leading to low-cost genotyping of many individuals (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Elshire <em>et al.</em> 2011</a>). Using GBS, high-density linkage maps have been cost-effectively constructed in barley and wheat (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Poland <em>et al.</em> 2012a</a>) and rice (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Spindel <em>et al.</em> 2013</a>).</p><p id="p-8">In this experiment, we show that GBS can be used in an autotetraploid alfalfa F<sub>1</sub> mapping population to quickly and cost-effectively construct saturated, reasonably high-density genetic linkage maps for both parental genomes. We show that marker orders are similar among homologs and that they are largely syntenous with <em>M. truncatula</em>, indicating the robustness of the maps.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mapping population</h3><p id="p-9">Two parental genotypes, DM3 and DM5, were crossed to generate an F<sub>1</sub> mapping population consisting of 384 progenies. DM3 (maternal parent) is a single individual derived from a cross between a genotype from “Maverick” (fall dormancy [FD] score = 1) and a genotype from “UC1465” (FD = 11). DM5 (paternal parent) is a single individual derived from a cross between a genotype from “Ranger” (FD = 3) and a genotype from ABI700 (FD = 6). The cultivars from which the grandparental genotypes were selected are all check cultivars for the standard test for FD (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Teuber <em>et al.</em> 1998</a>). Neither the grandparental plants nor their DNA were available for analysis. The parents and F<sub>1</sub> progenies were grown in the greenhouse of the Samuel Roberts Noble Foundation, Ardmore, OK. Tissue from young leaves was sampled, freeze-dried, ground, and used for DNA extraction.</p></div><div id="sec-3" class="subsection"><h3>DNA isolation and GBS library construction</h3><p id="p-10">DNA was isolated with the Wizard Genomic DNA Purification Kit (A1125; Promega) per the manufacturer’s instructions and quantified with a Quant-iT PicoGreen dsDNA assay kit (P7589; Life Technologies). Three libraries were constructed for the 384 F<sub>1</sub> progenies and the two parents. Library I was a 190-plex including 184 F<sub>1</sub> progenies plus three replications of each parent; Library II and III each were 104-plexes with 100 progenies plus a two replications of each parent. The three libraries were generated based on the protocol of <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Elshire <em>et al.</em> (2011)</a> with minor modifications. In summary, 100 ng of each DNA was digested with <em>Ape</em>KI (R0643L; NEB), and then ligated to a unique barcoded adapter and a common adapter. Equal volumes of the ligated products were pooled and purified with the QIAquick PCR purification kit (28104; QIAGEN) for PCR amplification. For the PCR, 50 ng of template DNA was mixed with NEB 2X Taq Master Mix and two primers (5 nmol each) in a 50 μL of total volume and amplified on a thermocycler for 18 cycles with 10 sec of denaturation at 98°, followed by 30 sec of annealing at 65°, and finally 30 sec extension at 72°. The polymerase chain reaction (PCR) product was cleaned with the QIAquick PCR purification kit. To generate single-end, 100-bp reads, Library I was sequenced on two lanes and Libraries II and III each in one lane on an Illumina HiSequation 2000 at the Genomic Sequencing and Analysis Facility at the University of Texas at Austin, Texas. All sequences were submitted to the National Center for Biotechnology Information Short Read Archive (experiment #SRX529440).</p></div><div id="sec-4" class="subsection"><h3>Sequence analysis and GBS SNP genotype calling</h3><p id="p-11">The Tassel 3.0 Universal Network Enabled Analysis Kit (UNEAK) pipeline (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lu <em>et al.</em> 2013</a>) was used for <em>de novo</em> SNP discovery and genotype calling. We initially used the <em>M. truncatula</em> genome as a reference sequence, but in preliminary analyses, we detected fewer SNP markers than we did when using UNEAK, due in part to the difference in the genome sizes of <em>M. truncatula</em> at ~500 Mbp <em>vs. M. sativa</em> at ~800−1000 Mbp. Because there did not appear to any advantage, and because we wanted to target regions of the alfalfa genome that are not present in <em>Medicago</em>, which may contain genes for alfalfa-specific traits like winter hardiness, autumn dormancy, and others, we used UNEAK for our SNP discovery pipeline.</p><p id="p-12">In summary, the raw 100-bp, single-end reads obtained from the sequencer were first quality filtered and de-multiplexed. All reads that began with one of the expected barcodes that was immediately followed by the expected cut site remnant (CAGC or CTGC for <em>Ape</em>KI) were trimmed to 64 bp (including the cut site remnant but removing the barcode). Identical reads were grouped into one tag. The tags with 10 or more reads across all individuals were retained for pairwise alignment. Pairwise alignment was performed to find pairs of tags that differed at only one nucleotide position. For a SNP marker, the number of reads for each allele in the paired tags in each individual was calculated and used for SNP genotype calling.</p><p id="p-13">For a given SNP (A/B) with a true genotype (G) and a total number of sequencing reads (N) in an autotetraploid individual, the read number of allele “A” (X<sub>A</sub>) follows a binomial distribution:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1971/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1971/embed/mml-math-1.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1971/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1971/embed/mml-math-2.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1971/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1971/embed/mml-math-3.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1971/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1971/embed/mml-math-4.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1971/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1971/embed/mml-math-5.gif"/></noscript></span></span>Where α is defined as sequencing error rate (assuming the probabilities that A is falsely sequenced as B and B is falsely sequenced as A are equal). The sequencing error rate from Illumina Hisequation 2000 is very low after quality filtering (<a href="http://res.illumina.com/documents/products/technotes/technote_q-scores.pdf">http://res.illumina.com/documents/products/technotes/technote_q-scores.pdf</a>).</p><p id="p-14">To limit missing data yet give accurate genotypic calls, we classified SNP genotypes using the following criteria. For a given SNP (A/B), if only a single allele was observed for a given individual, then a minimum of 11 reads was required to call a homozygote (<em>i.e.</em>, AAAA). If fewer than eleven reads were present, we assigned a missing genotype to avoid misclassifying a triplex heterozygote. The probability of miscalling a triplex heterozygote (AAAB) as a homozygote (AAAA) is less than 0.05 if 11 or more reads are present. When both alleles were observed in a given individual, we required a minimum of two reads per allele and a minimum minor allele frequency greater than 0.10 to call a heterozygote; otherwise, a missing genotype call (NA) was assigned. Requiring two reads of the minor allele limits the likelihood that an allele resulted from a sequencing error. However, if a large number of sequencing reads are available for a given locus, multiple sequencing errors might be likely. Therefore, we included the minor allele frequency limit to avoid calling homozygotes as heterozygotes that were obtained solely due to sequencing errors. Reliably discriminating among the three heterozygote genotypes in an autotetraploid would require a read depth of at least 60 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Uitdewilligen <em>et al.</em> 2013</a>). Because only a small percentage of our GBS SNP markers met this criterion, we did not attempt to distinguish among heterozygote genotypes.</p></div><div id="sec-5" class="subsection"><h3>SSR marker genotyping</h3><p id="p-15">SSR markers mapped previously (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Julier <em>et al.</em> 2003</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Robins <em>et al.</em> 2007</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sledge <em>et al.</em> 2005</a>), as well as additional markers from the Samuel Roberts Noble Foundation, Ardmore, OK, were screened on the population (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/012245SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/TableS1.pdf">Table S1</a>). Primers were synthesized by Integrated DNA Technologies (IDT; <a href="http://www.idtdna.com">http://www.idtdna.com</a>), with 18 nucleotides of M13 universal primer sequence added onto the 5′ end of the forward primer (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Schuelke 2000</a>). The M13 universal primer sequence was labeled with blue (6-FAM), green (HEX), or yellow (NED) fluorescent tags and was synthesized by Applied Biosystems (<a href="http://www.appliedbiosystems.com">http://www.appliedbiosystems.com</a>). The PCR steps were as follows: 95° for 2 min, followed by 30 cycles with 30 sec at 95°, 45 sec at 60°, and 45 sec at 72°, plus 10 cycles with 30 sec at 95°, 45 sec at 53°, 45 sec at 72°, and finalized with an elongation step of 7 min at 72°. PCR products from four to eight SSR markers were diluted 10 times and pooled for each individual, mixed with 0.2 μL of GeneScan-500 ROX size standard (401734; ABI), and analyzed on an ABI 3730 DNA analyzer. The data files from the sequencer were analyzed using the Genemarker software (<a href="http://www.softgenetics.com">http://www.softgenetics.com</a>), verified by visual inspection. Each allele of an SSR marker was scored as a dominant marker (present = 1 and absent = 0).</p></div><div id="sec-6" class="subsection"><h3>Construction of the genetic linkage map</h3><p id="p-16">Following the method of <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brouwer and Osborn (1999)</a>, we screened the GBS SNP markers that are either polymorphic only in DM3 (in the parental configuration AB×AA) or polymorphic only in DM5 (in the parental configuration AA×AB) for single-dose alleles (SDAs). The SDAs have an expected segregation ratio of 1:1 (presence:absence) and double-dose alleles have an expected ratio of 5:1 in an autotetraploid F<sub>1</sub> population. A marker that has a segregation ratio of 2.24:1 would be equally likely to have a true genotype of an SDA (<em>i.e.</em>, ABBB) as a double-dose allele (<em>i.e.</em>, AABB). Thus, alleles segregating with a ratio of less than 2:1 were considered as GBS SNP-SDA markers, and those with fewer than 50% missing values among the F<sub>1</sub> progenies were used to construct the genetic linkage maps in this study (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/TableS2.pdf">Table S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/TableS3.pdf">Table S3</a>). We initially used SNP markers with ≤20% missing genotype calls to construct a linkage map, which we then compared with the linkage map constructed using SNP markers with ≤50% missing genotype calls. We did not find any obvious differences in grouping or marker ordering between the two datasets. Consequently, we chose to report the map with more markers, based on the SNP markers with ≤50% missing genotype calls (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FileS1.xls">File S1</a>). Similarly, each allele of the SSR markers present in one parent and absent in the other parent was screened for SDA and SSR-SDA markers were selected for map construction (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FileS1.xls">File S1</a>).</p><p id="p-17">The SDA markers from the maternal and paternal parent were analyzed separately using JoinMap 4.1 following the two-way pseudo-testcross strategy (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Van Ooijen 2011</a>). First, SDA markers were grouped using a minimum logarithm of odds (LOD) score of 14. Second, for each linkage group, markers were ordered using the regression algorithm with the minimum LOD score of 1.0 and maximum recombination frequency of 0.35. Map distances were estimated using the Kosambi mapping function. The linkage maps were drawn with the R package <em>R/qtl</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arends <em>et al.</em> 2010</a>) and MapChart (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Voorrips 2002</a>).</p><p id="p-18">The Basic Local Alignment Search Tool (BLAST) was used to query the consensus sequence of each tag pair containing a SNP against the <em>M. truncatula</em> reference genome Version 4.1. We evaluated physical locations of the GBS-SNP using two best-hit cutoff thresholds of either an E-value &lt; 1 × 10<sup>−5</sup> or &lt; 1 × 10<sup>−20</sup>. The SSR primer sequences also were located on the <em>M. truncatula</em> V4.1 genome using BLAST in a manner analogous to the SNP markers. We also used Bowtie (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Langmead <em>et al.</em> 2009</a>) to query the consensus sequence to locate the physical position of each SNP locus on the <em>M. truncatula</em> V4.1 genome. Synteny between the genetic linkage maps and the <em>M. truncatula</em> physical maps was evaluated visually.</p></div><div id="sec-7" class="subsection"><h3>Segregation distortion regions</h3><p id="p-19">We identified SDA based on a segregation ratio of less than 2:1 (presence:absence) and used a χ<sup>2</sup> test to assess for deviation of the observed allelic distribution from the expected allelic ratio of 1:1. This test is the same as that used previously in the two-way pseudo-testcross for evaluating segregation distortion in F<sub>1</sub> full-sib progenies (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Grattapaglia and Sederoff 1994</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Tavoletti <em>et al.</em> 1996</a>). The log-transformed p-values (–log(p-value)) from the χ<sup>2</sup> tests were plotted along the genetic positions for the mapped markers with locally weighted scatterplot smoothing (LOESS) lines using R package <em>ggplot2</em> (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wickham 2009</a>). We determined segregation distortion regions as those where the smoothed LOESS curve was above LOD = 3 and in which three consecutive markers showed skewing in the same direction at <em>P</em> &lt; 0.001.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Genotyping-by-sequencing</h3><p id="p-20">A total of 890.5 million sequence reads were obtained from the four HiSequation 2000 lanes for the alfalfa mapping population after quality filtering and processing using UNEAK. The average number of reads per F<sub>1</sub> progeny was 2.25 million, ranging from 0.86 to 6.15 million; DM3 had 13.5 million reads, and DM5 had 11.8 million reads. In total 22,956 GBS SNP markers were polymorphic in the two parents with an average of 68.2% missing genotype calls per marker among the F<sub>1</sub> progenies (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Of these markers, 8527 had ≤50% missing genotype calls among the F<sub>1</sub> progenies with an average of 23.7%, and 4069 had ≤20% missing genotype calls with an average of 10.5% (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). We used the set of markers with ≤50% missing calls to build the map reported here.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465085/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465085/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Number of SNP markers and mean percentage of missing genotype calls per marker in datasets with varying cutoff levels for missing genotype calls</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Genetic linkage map</h3><p id="p-23">Of the 8527 GBS SNP markers, 1540 (18.1%) were classified as SDA for DM3 and 2229 (26.1%) as SDA for DM5, giving a total of 3769 SDAs. Of the 3769 GBS SNP-SDA markers, 3132 (83.1%) were aligned to the <em>M. truncatula</em> reference genome under a cutoff of E-value &lt; 1 × 10<sup>−5</sup>. From the 17 SSR markers, 41 alleles were classified as SDA (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/TableS4.pdf">Table S4</a>). The GBS SNP-SDA markers and SSR-SDA markers were grouped using JoinMap 4.1. For each parent, the markers were grouped into 32 linkage groups. Based on the physical locations of the GBS SNP markers, we were able to unambiguously assign the 32 linkage groups to the eight <em>Medicago</em> chromosomes. Four linkage groups representing the four haplotypes (homologous chromosomes) in autotetraploid alfalfa were assigned to each of the eight <em>Medicago</em> chromosomes. All of the mapped SSR markers were grouped onto the same chromosomes to which they had been mapped previously (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Julier <em>et al.</em> 2003</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2011</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Robins <em>et al.</em> 2007</a>). This provided further evidence that the chromosome assignments were all correct.</p><p id="p-24">The DM3 linkage map spanned a total of 2126 cM with a total of 1437 mapped markers and an average density of one marker per1.5 cM on each haplotype map (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). A total of 2154 markers were mapped on the DM5 linkage map with a total length of 2133 cM and an average density of one marker 1.0 cM per haplotype map (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The number of markers varied from 21 to 91 among the 32 linkage groups for DM3 and from 20 to 189 for DM5 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465064/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465064/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Distribution of 3591 GBS SNP and SSR markers among the 32 linkage groups of the DM3 and DM5 parents of the DM35 mapping population and the lengths of each haplotype map</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The 32 linkage groups for the maternal parent DM3 of the DM35 alfalfa mapping population. The positions of markers are shown in Kosambi centiMogan (cM). Each linkage group is named based on M. truncatula, with four homologs ordered A to D based on the number of loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-914020276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The 32 linkage groups for the maternal parent DM3 of the DM35 alfalfa mapping population. The positions of markers are shown in Kosambi centiMogan (cM). Each linkage group is named based on &lt;em&gt;M. truncatula&lt;/em&gt;, with four homologs ordered A to D based on the number of loci.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1971/F1.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/10/1971/F1.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465072" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">The 32 linkage groups for the maternal parent DM3 of the DM35 alfalfa mapping population. The positions of markers are shown in Kosambi centiMogan (cM). Each linkage group is named based on <em>M. truncatula</em>, with four homologs ordered A to D based on the number of loci.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The 32 linkage groups for the paternal parent DM3 of the DM35 alfalfa mapping population. The positions of markers are shown in Kosambi centiMogan (cM). Each linkage group is named based on M. truncatula, with four homologs ordered A to D based on the number of loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-914020276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The 32 linkage groups for the paternal parent DM3 of the DM35 alfalfa mapping population. The positions of markers are shown in Kosambi centiMogan (cM). Each linkage group is named based on &lt;em&gt;M. truncatula&lt;/em&gt;, with four homologs ordered A to D based on the number of loci.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1971/F2.medium.gif" width="440" height="234"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/10/1971/F2.medium.gif" width="440" height="234"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465019" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">The 32 linkage groups for the paternal parent DM3 of the DM35 alfalfa mapping population. The positions of markers are shown in Kosambi centiMogan (cM). Each linkage group is named based on <em>M. truncatula</em>, with four homologs ordered A to D based on the number of loci.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">A high level of synteny was observed between the alfalfa linkage maps and the <em>M. truncatula</em> physical map using a cutoff E-value of &lt; 1 × 10<sup>−5</sup> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS7.pdf">Figure S7</a>). Using either Bowtie or BLAST with a more stringent cutoff E-value &lt; 1 × 10<sup>−20</sup>, resulted in fewer SNP markers aligning to unmatching chromosomes of the <em>M. truncatula</em> reference genome (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). This indicated that most of the noise observed using the less stringent cutoff was from the unreliable alignment of SNP markers to <em>M. truncatula</em> reference genome. A reciprocal translocation between Chromosomes 4 (at 37−39 Mbp) and 8 (at 33−35 Mbp) was identified (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS7.pdf">Figure S7</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/TableS3.pdf">Table S3</a>). An inversion on Chromosome 1 was observed between the <em>M. truncatula</em> physical map and all four haplotype maps of DM5 and two haplotype maps of DM3 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The remaining two DM3 haplotype maps did not have sufficient marker density to identify whether the inversion was present.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dotplot of the positions of markers mapped on the DM3 linkage map relative to their position on the M. truncatula physical map. (A) BLAST with a cutoff of E-value &lt; 1 &#xD7; 10&#x2212;5; (B) a cutoff of E-value &lt; 1 &#xD7; 10&#x2212;20; (C) Bowtie. The dot colors of red, green, blue, and light blue represent the four haplotypes (A, B, C, and D) of each chromosome" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-914020276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dotplot of the positions of markers mapped on the DM3 linkage map relative to their position on the &lt;em&gt;M. truncatula&lt;/em&gt; physical map. (A) BLAST with a cutoff of E-value &lt; 1 &#xD7; 10&lt;sup&gt;&#x2212;5&lt;/sup&gt;; (B) a cutoff of E-value &lt; 1 &#xD7; 10&lt;sup&gt;&#x2212;20&lt;/sup&gt;; (C) Bowtie. The dot colors of red, green, blue, and light blue represent the four haplotypes (A, B, C, and D) of each chromosome&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1971/F3.medium.gif" width="440" height="141"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/10/1971/F3.medium.gif" width="440" height="141"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465014" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Dotplot of the positions of markers mapped on the DM3 linkage map relative to their position on the <em>M. truncatula</em> physical map. (A) BLAST with a cutoff of E-value &lt; 1 × 10<sup>−5</sup>; (B) a cutoff of E-value &lt; 1 × 10<sup>−20</sup>; (C) Bowtie. The dot colors of red, green, blue, and light blue represent the four haplotypes (A, B, C, and D) of each chromosome</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dotplot of the positions of markers mapped on the DM5 linkage map relative to their position on the M. truncatula physical map. (a) BLAST with a cutoff of E-value &lt; 1 &#xD7; 10&#x2212;5; (b) a cutoff of E-value &lt; 1 &#xD7; 10&#x2212;20; (c) Bowtie. The dot colors of red, green, blue, and light blue represent the four haplotypes (A, B, C, and D) of each chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-914020276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dotplot of the positions of markers mapped on the DM5 linkage map relative to their position on the &lt;em&gt;M. truncatula&lt;/em&gt; physical map. (a) BLAST with a cutoff of E-value &lt; 1 &#xD7; 10&lt;sup&gt;&#x2212;5&lt;/sup&gt;; (b) a cutoff of E-value &lt; 1 &#xD7; 10&lt;sup&gt;&#x2212;20&lt;/sup&gt;; (c) Bowtie. The dot colors of red, green, blue, and light blue represent the four haplotypes (A, B, C, and D) of each chromosome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1971/F4.medium.gif" width="440" height="141"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/10/1971/F4.medium.gif" width="440" height="141"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464919" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Dotplot of the positions of markers mapped on the DM5 linkage map relative to their position on the <em>M. truncatula</em> physical map. (a) BLAST with a cutoff of E-value &lt; 1 × 10<sup>−5</sup>; (b) a cutoff of E-value &lt; 1 × 10<sup>−20</sup>; (c) Bowtie. The dot colors of red, green, blue, and light blue represent the four haplotypes (A, B, C, and D) of each chromosome.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of Medicago sativa linkage group 1 maps with the M. truncatula chromosome 1 physical map. The parental alfalfa genetic maps are labeled DM3 and DM5 and the four homologous linkage groups of each parent are labeled A, B, C, or D. One unit on the physical map reflects 5&#xD7;105 bp. The genetic positions of markers are shown in Kosambi centiMogan (cM). Marker names and sequences are found in Table S2 and Table S3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-914020276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of &lt;em&gt;Medicago sativa&lt;/em&gt; linkage group 1 maps with the &lt;em&gt;M. truncatula&lt;/em&gt; chromosome 1 physical map. The parental alfalfa genetic maps are labeled DM3 and DM5 and the four homologous linkage groups of each parent are labeled A, B, C, or D. One unit on the physical map reflects 5&#xD7;10&lt;sup&gt;5&lt;/sup&gt; bp. The genetic positions of markers are shown in Kosambi centiMogan (cM). Marker names and sequences are found in Table S2 and Table S3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1971/F5.medium.gif" width="440" height="164"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/10/1971/F5.medium.gif" width="440" height="164"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1971/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465015" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-31" class="first-child">Comparison of <em>Medicago sativa</em> linkage group 1 maps with the <em>M. truncatula</em> chromosome 1 physical map. The parental alfalfa genetic maps are labeled DM3 and DM5 and the four homologous linkage groups of each parent are labeled A, B, C, or D. One unit on the physical map reflects 5×10<sup>5</sup> bp. The genetic positions of markers are shown in Kosambi centiMogan (cM). Marker names and sequences are found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/TableS2.pdf">Table S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/TableS3.pdf">Table S3</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Segregation distortion</h3><p id="p-32">Of the 1437 markers mapped on the maternal parent DM3 linkage map, 259 (18.0%) were distorted (<em>P</em> &lt; 0.001). Using the criteria of at least three distorted markers within a region where the smoothed LOESS curve was above a LOD = 3, we identified three major distorted regions on LG 4B, 5C, and 6B (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS8.pdf">Figure S8</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/TableS3.pdf">Table S3</a>). For the paternal parent DM5, 351 (16.3%) of the 2154 mapped markers were distorted (<em>P</em> &lt; 0.001), and two clusters of distorted markers were observed on LG 1D and 6B (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS9.pdf">Figure S9</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/TableS3.pdf">Table S3</a>). Part of the scatter of some markers seen in the distortion graphs could be due to missing genotype calls. To evaluate the effect of missing genotype calls on the detection of segregation distortion, we compared distortion on linkage maps constructed with markers that had ≤20% missing genotype calls to that observed above on the maps developed from markers with ≤50% missing calls. For DM3, the cluster of distorted markers on LG 4B and 5C was confirmed and became clearer using the marker set with fewer missing calls, while the cluster on LG6B was not detected in the smaller data set because only a few markers mapped to this group (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS10.pdf">Figure S10</a>). For DM5, the clustered distorted markers on LG 1D was confirmed but not the cluster on LG 6B (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1/FigureS11.pdf">Figure S11</a>).</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>GBS in alfalfa</h3><p id="p-33">SNPs are highly abundant throughout plant genomes and, therefore, widely used in genetic studies and breeding applications. High-throughput SNP genotyping can be done using GBS, which is more flexible and generally more inexpensive than either single marker assays or most array platforms. GBS has been successfully used in genetic linkage map construction (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Poland <em>et al.</em> 2012a</a>), population diversity studies (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Lu <em>et al.</em> 2013</a>), and genomic selection (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Poland <em>et al.</em> 2012b</a>). A substantial number of missing genotype calls has been commonly found when using GBS because of unequal distribution of sequencing reads across SNP loci and among samples. The unbalanced read distribution could be a result of variable initial amounts of DNA and/or PCR efficiency bias due to different sizes and GC contents of fragments.</p><p id="p-34">In cultivated alfalfa, which is a tetrasomic tetraploid and non-inbred, low-read depth at a locus could result in miscalling a heterozygote as a homozygote. Compared with diploid, inbred species (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Spindel <em>et al.</em> 2013</a>), the higher read depth required for genotype calling in tetraploid alfalfa results in fewer loci genotyped within an individual and fewer loci genotyped across individuals in a population for a given depth of sequencing. The number of SNP genotype calls for a given population relies on many factors, such as the genome size of the species, the choice of restriction enzyme (or the number of cut sites), accuracy of DNA quantification, bias of PCR polymerases, etc. The number of SNP calls among diploid soybean inbred lines was increased by 40% using selective primers to achieve a greater genome reduction during library preparation (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sonah <em>et al.</em> 2013</a>). GBS protocols could be optimized for polyploid species by varying factors affecting the evenness of sequencing across loci, such as PCR polymerase bias, size selection of fragments, and choice of enzymes used to generate the libraries.</p></div><div id="sec-14" class="subsection"><h3>Genetic linkage map</h3><p id="p-35">We mapped 3555 GBS SNP-SDA markers and 36 SSR-SDA markers on 64 linkage groups in an autotetraploid alfalfa F<sub>1</sub> population, the highest density linkage map for tetraploid alfalfa to date. The two parental maps were very similar in length, at approximately 2130 cM, or ~530 cM for one genome equivalent, very similar to previous genetic maps in alfalfa (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Li and Brummer 2012</a>). As shown in previous studies (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Choi <em>et al.</em> 2004</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2011</a>), we identified a very high level of synteny between the genetic linkage map of alfalfa and the <em>M. truncatula</em> whole-genome sequence. All SSR markers were mapped to the same chromosomes as previous alfalfa linkage maps, suggesting the GBS based maps are valid. Further, the fact that most markers that aligned to the <em>M. truncatula</em> physical map did so in basically the expected order strongly suggests that the GBS SNP markers were correctly identified and genotyped. Therefore, GBS provides a high-throughput genotyping platform to construct a high-density linkage map for autotetraploid alfalfa and likely for other polyploid species.</p><p id="p-36">Although several genetic linkage maps have been constructed for tetraploid alfalfa, most of them were framework maps with relatively few markers (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Brouwer and Osborn 1999</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Julier <em>et al.</em> 2003</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Khu <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Musial <em>et al.</em> 2007</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Robins <em>et al.</em> 2007</a>). Some QTL related to biomass yield (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Robins <em>et al.</em> 2007</a>), aluminum tolerance (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Khu <em>et al.</em> 2013</a>), and FD and winter survival (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Brouwer and Osborn 1999</a>) have been discovered, but these were generally located to large chromosomal regions. The high-density genetic markers derived from this study would provide enough saturation to use advanced generation populations for mapping, so that the additional recombination compared with F<sub>1</sub> or F<sub>2</sub> populations could further localize QTL to short genomic intervals. The four haplotypes of each of the eight chromosomes were mapped by considering only SDA markers so that mapping software developed for diploid species could be used. Allele interactions within a locus and between loci (epistasis) also could be evaluated. Thus, the high density of GBS markers enables breeders to use only SDA, greatly facilitating genetic dissection of qualitative and quantitative traits in cultivated autotetraploid alfalfa.</p><p id="p-37">One limitation of GBS is the large amount of missing genotypic information for a given marker. Missing marker genotypes can bias ordering of markers during linkage map construction (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hackett and Broadfoot 2003</a>) and reduce power of QTL detection, especially on smaller effect QTL and in smaller population sizes (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Zhang <em>et al.</em> 2010</a>). In this study, the ordering of markers was not biased by the markers with up to 50% missing genotype calls, probably due to the large size of the mapping population. Larger mapping population size and imputation of missing marker genotypes can enhance QTL detection. Another limitation is that that many polymorphic GBS SNP markers not classified as SDA were not used for the haplotype map construction in this study. Advanced software needs to be developed for autotetraploid species so that large numbers of markers with any segregation patterns and with allele dosage information can be used for map construction.</p><p id="p-38">A reciprocal translocation between Chromosomes 4 and 8 was reported in the <em>Medicago truncatula</em> reference accession A17, which served as the basis of the <em>M. truncatula</em> whole-genome sequence (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kamphuis <em>et al.</em> 2007</a>). From the high-density map derived from this study, we could infer that the translocation regions are at about 33−35 Mbp on Chromosome 4 and 37−39 Mbp on Chromosome 8. An inversion on Chromosome 1 also was observed on both maternal and paternal linkage maps. In the same region, we also observed an inversion between <em>M. truncatula</em> and a diploid alfalfa linkage map (X. Li <em>et al.</em> unpublished data). Interestingly, a similar inversion was found between a white clover linkage map and the <em>M. truncatula</em> reference genome (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Griffiths <em>et al.</em> 2013</a>). More dense alfalfa linkage maps and the forthcoming alfalfa whole-genome sequence could provide further clarification of the presence of this inversion between the two <em>Medicago</em> species.</p></div><div id="sec-15" class="subsection"><h3>Segregation distortion</h3><p id="p-39">A substantial number of distorted markers (24–68%) have been commonly found in diploid alfalfa linkage maps (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Li and Brummer 2012</a>). The distorted markers were generally clustered and showed the same skew direction, suggesting that segregation distortion loci or viability genes were causing the distortion of the surrounding markers (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2011</a>). Compared with diploid alfalfa, fewer distorted markers (4–32%) usually were observed in tetraploid alfalfa linkage maps (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Li and Brummer 2012</a>), although this could be biased downward by selection of the markers used for map construction. Distorted markers have been observed along all eight chromosomes, but no consensus segregation distortion regions have been identified (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Brouwer and Osborn 1999</a>; <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Julier <em>et al.</em> 2003</a>; <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Robins <em>et al.</em> 2007</a>).</p><p id="p-40">The complexity of tetrasomic inheritance, including a generally unknown allele dosage at any given locus, challenges the detection of distorted markers and segregation distortion regions in autotetraploid alfalfa. In this study, 17.9% of mapped markers were distorted. However, the amount of distortion was likely underestimated, because we initially selected loci to map based on their segregation profile and markers not meeting that criterion were not included in building the map. Nevertheless, based on the high-density haplotype linkage maps constructed with the SDA markers, several apparent segregation distortion regions could be identified. When we evaluated markers with ≤20% missing genotype calls, we were able to more clearly define segregation distortion regions and minimize the scatter observed with the larger marker set. This suggests that, in some cases, markers can be misclassified as distorted if they have large amounts of missing data. These results in tetraploid alfalfa, together with high-density linkage mapping in diploid alfalfa populations (<em>e.g.</em>, <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2011</a>; X. Li and E. C. Brummer, unpublished results), could enable us to identify viability or fitness genes and (perhaps) self-incompatibility loci in alfalfa.</p><p id="p-41">By using GBS, high-density genetic linkage maps were constructed for an autotetraploid alfalfa F<sub>1</sub> population. The high-density linkage maps could facilitate further applications of QTL mapping, comparative mapping, map-based cloning, and alfalfa whole-genome assembly. GBS can be potentially used for genotyping of unstructured natural populations or breeding populations to facilitate genome-wide association studies and genomic selection.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-42">We are grateful to the Samuel Roberts Noble Foundation and the USDA-DOE Plant Feedstock Genomics for Bioenergy program, award #2009-65504-05809 to E.C.B., for funding. J.K. was supported by China Scholarship Council.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012245/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-43"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-05-20"><span class="received-label">Received </span>May 20, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-08-15"><span class="accepted-label">Accepted </span>August 15, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Li <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.1" data-doi="10.1186/1471-2164-13-203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Antanaviciute</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Fernandez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of a dense SNP-based linkage map of an apple rootstock progeny using the <em>Malus</em> Infinium whole genome genotyping array.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAntanaviciute%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.spage%253D203%26rft.epage%253D203%26rft.atitle%253DDevelopment%2Bof%2Ba%2Bdense%2BSNP-based%2Blinkage%2Bmap%2Bof%2Ban%2Bapple%2Brootstock%2Bprogeny%2Busing%2Bthe%2BMalus%2BInfinium%2Bwhole%2Bgenome%2Bgenotyping%2Barray.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-203%26rft_id%253Dinfo%253Apmid%252F22631220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22631220&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.2" data-doi="10.1093/bioinformatics/btq565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arends</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prins</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">R/qtl: high-throughput multiple QTL mapping.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2990</span>–<span class="cit-lpage">2992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq565%26rft_id%253Dinfo%253Apmid%252F20966004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjMvMjk5MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTAvMTk3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blondon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marie</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondorosi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genome size and base composition in <em>Medicago sativa</em> and <em>M. truncatula</em> species.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">264</span>–<span class="cit-lpage">270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DBlondon%26rft.auinit1%253DF.%26rft.volume%253D37%26rft.issue%253D2%26rft.spage%253D264%26rft.epage%253D270%26rft.atitle%253DGenome%2Bsize%2Band%2Bbase%2Bcomposition%2Bin%2BMedicago%2Bsativa%2Band%2BM.%2Btruncatula%2Bspecies.%26rft_id%253Dinfo%253Apmid%252F18470076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18470076&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachlava</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunick</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of a 10,000 locus genetic map of the sunflower genome based on multiple crosses. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D2%26rft.spage%253D721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.5" data-doi="10.1007/s001220051324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brouwer</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A molecular marker linkage map of tetraploid alfalfa (<em>Medicago sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1194</span>–<span class="cit-lpage">1200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D99%26rft.spage%253D1194%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000084014900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.6" data-doi="10.1186/1741-7007-11-50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chancerel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamy</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesur</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noirot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klopp</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-density linkage mapping in a pine tree reveals a genomic region associated with inbreeding depression and provides clues to the extent and distribution of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol.%26rft.volume%253D11%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-11-50%26rft_id%253Dinfo%253Apmid%252F23597128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-11-50&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23597128&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.7" data-doi="10.1534/genetics.166.3.1463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uhm</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limpens</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A sequence-based genetic map of <em>Medicago truncatula</em> and comparison of marker colinearity with <em>M. sativa</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1463</span>–<span class="cit-lpage">1502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChoi%26rft.auinit1%253DH.-K.%26rft.volume%253D166%26rft.issue%253D3%26rft.spage%253D1463%26rft.epage%253D1502%26rft.atitle%253DA%2BSequence-Based%2BGenetic%2BMap%2Bof%2BMedicago%2Btruncatula%2Band%2BComparison%2Bof%2BMarker%2BColinearity%2Bwith%2BM.%2Bsativa%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.3.1463%26rft_id%253Dinfo%253Apmid%252F15082563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ni8zLzE0NjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE5NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.8" data-doi="10.1186/1471-2164-14-120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delourme</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falentin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fomeju</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boillot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lassalle</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-density SNP-based genetic map development and linkage disequilibrium assessment in <em>Brassica napus</em> L.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDelourme%26rft.auinit1%253DR.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D120%26rft.epage%253D120%26rft.atitle%253DHigh-density%2BSNP-based%2Bgenetic%2Bmap%2Bdevelopment%2Band%2Blinkage%2Bdisequilibrium%2Bassessment%2Bin%2BBrassica%2Bnapus%2BL.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-120%26rft_id%253Dinfo%253Apmid%252F23432809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23432809&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.9" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.10" data-doi="10.1371/journal.pone.0036347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felcher</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombs</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massa</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansey</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Integration of two diploid potato linkage maps with the potato genome sequence.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e36347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFelcher%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De36347%26rft.epage%253De36347%26rft.atitle%253DIntegration%2Bof%2Btwo%2Bdiploid%2Bpotato%2Blinkage%2Bmaps%2Bwith%2Bthe%2Bpotato%2Bgenome%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036347%26rft_id%253Dinfo%253Apmid%252F22558443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22558443&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sederoff</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic linkage maps of <em>Eucalyptus grandis</em> and <em>Eucalyptus urophylla</em> using a pseudo-testcross: mapping strategy and RAPD markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1121</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGrattapaglia%26rft.auinit1%253DD.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1121%26rft.epage%253D1137%26rft.atitle%253DGenetic%2Blinkage%2Bmaps%2Bof%2BEucalyptus%2Bgrandis%2Band%2BEucalyptus%2Burophylla%2Busing%2Ba%2Bpseudo-testcross%253A%2Bmapping%2Bstrategy%2Band%2BRAPD%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F7982566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNy80LzExMjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE5NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.12" data-doi="10.1186/1471-2164-14-388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bickerstaff</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An integrated genetic linkage map for white clover (<em>Trifolium repens</em> L.) with alignment to <em>Medicago</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D388%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-388%26rft_id%253Dinfo%253Apmid%252F23758831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23758831&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.13" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broadfoot</span>  <span class="cit-name-given-names">L. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.14" data-doi="10.1093/jhered/esm086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milne</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TetraploidMap for Windows: linkage map construction and QTL mapping in autotetraploid species.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesm086%26rft_id%253Dinfo%253Apmid%252F17965198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijk4LzcvNzI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xOTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.15" data-doi="10.1186/1471-2164-12-350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torres-Jerez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Town</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide SNP discovery in tetraploid alfalfa using 454 sequencing and high resolution melting analysis.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D12%26rft.spage%253D350%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.16" data-doi="10.1186/1471-2229-3-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Julier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flajoulot</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barre</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardinet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santoni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Construction of two genetic linkage maps in cultivated tetraploid alfalfa (<em>Medicago sativa</em>) using microsatellite and AFLP markers.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DJulier%26rft.auinit1%253DB.%26rft.volume%253D3%26rft.spage%253D9%26rft.epage%253D9%26rft.atitle%253DConstruction%2Bof%2Btwo%2Bgenetic%2Blinkage%2Bmaps%2Bin%2Bcultivated%2Btetraploid%2Balfalfa%2B%2528Medicago%2Bsativa%2529%2Busing%2Bmicrosatellite%2Band%2BAFLP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-3-9%26rft_id%253Dinfo%253Apmid%252F14683527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-3-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14683527&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.17" data-doi="10.1111/j.1469-8137.2007.02039.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamphuis</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Souza</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfaff</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellwood</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The <em>Medicago truncatula</em> reference accession A17 has an aberrant chromosomal configuration.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DKamphuis%26rft.auinit1%253DL.%2BG.%26rft.volume%253D174%26rft.issue%253D2%26rft.spage%253D299%26rft.epage%253D303%26rft.atitle%253DThe%2BMedicago%2Btruncatula%2Breference%2Baccession%2BA17%2Bhas%2Ban%2Baberrant%2Bchromosomal%2Bconfiguration.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2007.02039.x%26rft_id%253Dinfo%253Apmid%252F17388892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2007.02039.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17388892&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245413900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.18" data-doi="10.2135/cropsci2012.03.0181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khu</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyno</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of aluminum tolerance quantitative trait loci in tetraploid alfalfa.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">148</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.03.0181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.03.0181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.19" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.20" data-doi="10.3390/agronomy2010040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Applied genetics and genomics in alfalfa breeding.</span> <abbr class="cit-jnl-abbrev">Agronomy</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%26rft.volume%253D2%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fagronomy2010040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/agronomy2010040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.21" data-doi="10.1007/s00122-011-1617-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prevalence of segregation distortion in diploid alfalfa and its implications for genetics and breeding applications.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">667</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DX.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D667%26rft.epage%253D679%26rft.atitle%253DPrevalence%2Bof%2Bsegregation%2Bdistortion%2Bin%2Bdiploid%2Balfalfa%2Band%2Bits%2Bimplications%2Bfor%2Bgenetics%2Band%2Bbreeding%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1617-5%26rft_id%253Dinfo%253Apmid%252F21625992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1617-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21625992&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.22" data-doi="10.1186/1471-2164-13-568"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farmer</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bharti</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Prevalence of single nucleotide polymorphism among 27 diverse alfalfa genotypes as assessed by transcriptome sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLi%26rft.auinit1%253DX.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D568%26rft.epage%253D568%26rft.atitle%253DPrevalence%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bamong%2B27%2Bdiverse%2Balfalfa%2Bgenotypes%2Bas%2Bassessed%2Bby%2Btranscriptome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-568%26rft_id%253Dinfo%253Apmid%252F23107476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-568&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23107476&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.23" data-doi="10.1371/journal.pone.0084329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farmer</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development of an alfalfa SNP array and its use to evaluate patterns of population structure and linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e84329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D9%26rft.spage%253De84329%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0084329%26rft_id%253Dinfo%253Apmid%252F24416217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0084329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24416217&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.24" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1971.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michaud</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehman</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rumbaugh</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">World distribution and historical development</span>, pp. <span class="cit-fpage">31</span>–<span class="cit-lpage">32</span> in <span class="cit-source">Alfalfa and Alfalfa Improvement</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Hanson</span>  <span class="cit-name-given-names">A.</span></span>. <span class="cit-publ-name">American Society of Agronomy, Crop Science Society of America, Soil Science Society of America</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.26" data-doi="10.1007/s00122-007-0528-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Musial</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackie</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armour</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phan</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellwood</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Identification of QTL for resistance and susceptibility to <em>Stagonospora meliloti</em> in autotetraploid lucerne.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMusial%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D114%26rft.issue%253D8%26rft.spage%253D1427%26rft.epage%253D1435%26rft.atitle%253DIdentification%2Bof%2BQTL%2Bfor%2Bresistance%2Band%2Bsusceptibility%2Bto%2BStagonospora%2Bmeliloti%2Bin%2Bautotetraploid%2Blucerne.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0528-y%26rft_id%253Dinfo%253Apmid%252F17356865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0528-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17356865&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.27" data-doi="10.1038/nature07723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruggmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubchak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sorghum bicolor genome and the diversification of grasses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D457%26rft.issue%253D7229%26rft.spage%253D551%26rft.epage%253D556%26rft.atitle%253DThe%2BSorghum%2Bbicolor%2Bgenome%2Band%2Bthe%2Bdiversification%2Bof%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07723%26rft_id%253Dinfo%253Apmid%252F19189423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19189423&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262852200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.28" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.29" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.30" data-doi="10.2135/cropsci2005.11.0401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robins</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauchan</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic mapping of biomass production in tetraploid alfalfa.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DRobins%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D10%26rft.atitle%253DGenetic%2BMapping%2Bof%2BBiomass%2BProduction%2Bin%2BTetraploid%2BAlfalfa%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.11.0401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.11.0401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000244430100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.31" data-doi="10.1038/72708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schuelke</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">An economic method for the fluorescent labeling of PCR fragments.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DSchuelke%26rft.auinit1%253DM.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D233%26rft.epage%253D234%26rft.atitle%253DAn%2Beconomic%2Bmethod%2Bfor%2Bthe%2Bfluorescent%2Blabeling%2Bof%2BPCR%2Bfragments.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F72708%26rft_id%253Dinfo%253Apmid%252F10657137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/72708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10657137&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085146900036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.32" data-doi="10.1371/journal.pone.0040563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sim</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plieske</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wieseke</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of a large SNP genotyping array and generation of high-density genetic maps in tomato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e40563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSim%26rft.auinit1%253DS.%2BC.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De40563%26rft.epage%253De40563%26rft.atitle%253DDevelopment%2Bof%2Ba%2Blarge%2BSNP%2Bgenotyping%2Barray%2Band%2Bgeneration%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bin%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0040563%26rft_id%253Dinfo%253Apmid%252F22802968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0040563&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22802968&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.33" data-doi="10.1007/s00122-005-0038-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sledge</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">An expressed sequence tag SSR map of tetraploid alfalfa (<em>Medicago sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">980</span>–<span class="cit-lpage">992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSledge%26rft.auinit1%253DM.%2BK.%26rft.volume%253D111%26rft.issue%253D5%26rft.spage%253D980%26rft.epage%253D992%26rft.atitle%253DAn%2Bexpressed%2Bsequence%2Btag%2BSSR%2Bmap%2Bof%2Btetraploid%2Balfalfa%2B%2528Medicago%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0038-8%26rft_id%253Dinfo%253Apmid%252F16075206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0038-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16075206&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232598600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.34" data-doi="10.1371/journal.pone.0054603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastien</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iquira</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tardivel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legare</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e54603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De54603%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054603%26rft_id%253Dinfo%253Apmid%252F23372741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23372741&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.35" data-doi="10.1007/s00122-013-2166-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gage</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2699</span>–<span class="cit-lpage">2716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2699%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2166-x%26rft_id%253Dinfo%253Apmid%252F23918062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2166-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23918062&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.36" data-doi="10.1093/oxfordjournals.jhered.a022978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tavoletti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veronesi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">RFLP linkage map of an alfalfa meiotic mutant based on an F1 population.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DTavoletti%26rft.auinit1%253DS.%26rft.volume%253D87%26rft.issue%253D2%26rft.spage%253D167%26rft.epage%253D170%26rft.atitle%253DRFLP%2BLinkage%2BMap%2Bof%2Ban%2BAlfalfa%2BMeiotic%2BMutant%2BBased%2Bon%2Ban%2BF1%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.jhered.a022978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijg3LzIvMTY3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xOTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.10.1971.37"><div class="cit-metadata unstructured">Teuber, L. R., K. L. Taggard, L. K. Gibbs, M. H. McCaslin, M. A. Peterson <em>et al.</em>, 1998 Fall dormancy, in <em>Standard Tests to Characterize Alfalfa Cultivars</em>, Ed. 3, edited by C. C. Fox, L. Berberet, F. A. Gray, C. R. Grau, D. L. Jessen, <em>et al.</em> North America Alfalfa Improvement Conference, Beltsville, MD. Available at: <a href="http://www.naaic.org/stdtests/Dormancy2.html">http://www.naaic.org/stdtests/Dormancy2.html</a> [Verified 07 September 2014].</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.38" data-doi="10.1371/journal.pone.0062355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolters</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Hoop</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borm</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e62355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62355%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062355%26rft_id%253Dinfo%253Apmid%252F23667470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23667470&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.39" data-doi="10.1017/S0016672311000279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DVAN%2BOoijen%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D93%26rft.issue%253D5%26rft.spage%253D343%26rft.epage%253D349%26rft.atitle%253DMultipoint%2Bmaximum%2Blikelihood%2Bmapping%2Bin%2Ba%2Bfull-sib%2Bfamily%2Bof%2Ban%2Boutbreeding%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672311000279%26rft_id%253Dinfo%253Apmid%252F21878144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672311000279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878144&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.40" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6IjkzLzEvNzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE5NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1971.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>. <span class="cit-source">ggplot2: Elegant Graphics for Data Analysis</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.42" data-doi="10.1186/1471-2164-12-199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamb</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Using RNA-Seq for gene identification, polymorphism detection and transcript profiling in two alfalfa genotypes with divergent cell wall composition in stems.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DYang%26rft.auinit1%253DS.%2BS.%26rft.volume%253D12%26rft.spage%253D199%26rft.epage%253D199%26rft.atitle%253DUsing%2BRNA-Seq%2Bfor%2Bgene%2Bidentification%252C%2Bpolymorphism%2Bdetection%2Band%2Btranscript%2Bprofiling%2Bin%2Btwo%2Balfalfa%2Bgenotypes%2Bwith%2Bdivergent%2Bcell%2Bwall%2Bcomposition%2Bin%2Bstems.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-199%26rft_id%253Dinfo%253Apmid%252F21504589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21504589&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1971.43" data-doi="10.1007/s00122-010-1372-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Effects of missing marker and segregation distortion on QTL mapping in F2 populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">1071</span>–<span class="cit-lpage">1082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DL.%26rft.volume%253D121%26rft.issue%253D6%26rft.spage%253D1071%26rft.epage%253D1082%26rft.atitle%253DEffects%2Bof%2Bmissing%2Bmarker%2Band%2Bsegregation%2Bdistortion%2Bon%2BQTL%2Bmapping%2Bin%2BF2%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1372-z%26rft_id%253Dinfo%253Apmid%252F20535442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1372-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20535442&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1971.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/10/1971.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/10/1963" title="Resolution of Genetic Map Expansion Caused by Excess Heterozygosity in Plant Recombinant Inbred Populations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/10/1981" title="Small RNA Expression from the Human Macrosatellite DXZ4" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/10"><span>Volume 4 Issue 10, October 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/10.cover-source.jpg?itok=f3Vstm56" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/10/1971.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KRrKq1VTAmCY7bLXDjpk0AzFENlplKD16GlsxNXHknM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/10/1971" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/10/1971" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/10/1971" class="active" data-icon-position="" data-hide-link-title="0">A Saturated Genetic Linkage Map of Autotetraploid Alfalfa (Medicago sativa L.) Developed Using Genotyping-by-Sequencing Is Highly Syntenous with the Medicago truncatula Genome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459696" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ugWrxt02_2016GG4GVsolFR8pzulJs7VQGXwbGqQ8gg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/10/1971.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/10/1971" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-zhrIXIJUv8M8Z_5HBPLElbZgwNAWtwHQZIg1JX7VBd8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/10/1971" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/10/1971.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459696" id="node459696--2" data-pisa="ggg;4/10/1971" data-pisa-master="ggg;g3.114.012245" data-apath="/ggg/4/10/1971.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Saturated Genetic Linkage Map of Autotetraploid Alfalfa (<em>Medicago sativa</em> L.) Developed Using Genotyping-by-Sequencing Is Highly Syntenous with the <em>Medicago truncatula</em> Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xuehui Li</span>, <span class="highwire-citation-author" data-delta="1">Yanling Wei</span>, <span class="highwire-citation-author" data-delta="2">Ananta Acharya</span>, <span class="highwire-citation-author" data-delta="3">Qingzhen Jiang</span>, <span class="highwire-citation-author" data-delta="4">Junmei Kang</span> and <span class="highwire-citation-author" data-delta="5">E. Charles Brummer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1971-1979;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012245 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459696--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xuehui Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuehui%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXuehui%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yanling Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanling%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYanling%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ananta Acharya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ananta%2BAcharya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Acharya%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnanta%2BAcharya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Qingzhen Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingzhen%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingzhen%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Junmei Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Institute of Animal Science, Chinese Academy of Agricultural Science, Beijing, China 100193</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junmei%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJunmei%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">E. Charles Brummer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Breeding Center and Department of Plant Sciences, The University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=E.%2BCharles%2BBrummer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brummer%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AE.%2BCharles%2BBrummer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ecbrummer@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">ecbrummer@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/10/1971" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/10/1971&amp;title=A%20Saturated%20Genetic%20Linkage%20Map%20of%20Autotetraploid%20Alfalfa%20%28Medicago%20sativa%20L.%29%20Developed%20Using%20Genotyping-by-Sequencing%20Is%20Highly%20Syntenous%20with%20the%20Medicago%20truncatula%20Genome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/10/1971&amp;title=A%20Saturated%20Genetic%20Linkage%20Map%20of%20Autotetraploid%20Alfalfa%20%28Medicago%20sativa%20L.%29%20Developed%20Using%20Genotyping-by-Sequencing%20Is%20Highly%20Syntenous%20with%20the%20Medicago%20truncatula%20Genome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/10/1971&amp;title=A%20Saturated%20Genetic%20Linkage%20Map%20of%20Autotetraploid%20Alfalfa%20%28Medicago%20sativa%20L.%29%20Developed%20Using%20Genotyping-by-Sequencing%20Is%20Highly%20Syntenous%20with%20the%20Medicago%20truncatula%20Genome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/10/1971&amp;text=A%20Saturated%20Genetic%20Linkage%20Map%20of%20Autotetraploid%20Alfalfa%20%28Medicago%20sativa%20L.%29%20Developed%20Using%20Genotyping-by-Sequencing%20Is%20Highly%20Syntenous%20with%20the%20Medicago%20truncatula%20Genome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/10/1971&amp;title=A%20Saturated%20Genetic%20Linkage%20Map%20of%20Autotetraploid%20Alfalfa%20%28Medicago%20sativa%20L.%29%20Developed%20Using%20Genotyping-by-Sequencing%20Is%20Highly%20Syntenous%20with%20the%20Medicago%20truncatula%20Genome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/10/1971&amp;t=A%20Saturated%20Genetic%20Linkage%20Map%20of%20Autotetraploid%20Alfalfa%20%28Medicago%20sativa%20L.%29%20Developed%20Using%20Genotyping-by-Sequencing%20Is%20Highly%20Syntenous%20with%20the%20Medicago%20truncatula%20Genome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/10/1971&amp;title=A%20Saturated%20Genetic%20Linkage%20Map%20of%20Autotetraploid%20Alfalfa%20%28Medicago%20sativa%20L.%29%20Developed%20Using%20Genotyping-by-Sequencing%20Is%20Highly%20Syntenous%20with%20the%20Medicago%20truncatula%20Genome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/10/1971&amp;title=A%20Saturated%20Genetic%20Linkage%20Map%20of%20Autotetraploid%20Alfalfa%20%28Medicago%20sativa%20L.%29%20Developed%20Using%20Genotyping-by-Sequencing%20Is%20Highly%20Syntenous%20with%20the%20Medicago%20truncatula%20Genome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459696/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459696" id="node459696--3" data-pisa="ggg;4/10/1971" data-pisa-master="ggg;g3.114.012245" data-apath="/ggg/4/10/1971.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Saturated Genetic Linkage Map of Autotetraploid Alfalfa (<em>Medicago sativa</em> L.) Developed Using Genotyping-by-Sequencing Is Highly Syntenous with the <em>Medicago truncatula</em> Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Xuehui Li</span>, <span class="highwire-citation-author" data-delta="1">Yanling Wei</span>, <span class="highwire-citation-author" data-delta="2">Ananta Acharya</span>, <span class="highwire-citation-author" data-delta="3">Qingzhen Jiang</span>, <span class="highwire-citation-author" data-delta="4">Junmei Kang</span> and <span class="highwire-citation-author" data-delta="5">E. Charles Brummer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1971-1979;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012245 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459696--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xuehui Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuehui%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXuehui%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yanling Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanling%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYanling%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ananta Acharya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ananta%2BAcharya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Acharya%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnanta%2BAcharya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Qingzhen Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Forage Improvement Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingzhen%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingzhen%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Junmei Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The Institute of Animal Science, Chinese Academy of Agricultural Science, Beijing, China 100193</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junmei%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJunmei%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">E. Charles Brummer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Breeding Center and Department of Plant Sciences, The University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=E.%2BCharles%2BBrummer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brummer%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AE.%2BCharles%2BBrummer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ecbrummer@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">ecbrummer@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1971&amp;rft.epage=1979&amp;rft.atitle=A%20Saturated%20Genetic%20Linkage%20Map%20of%20Autotetraploid%20Alfalfa%20%28Medicago%20sativa%20L.%29%20Developed%20Using%20Genotyping-by-Sequencing%20Is%20Highly%20Syntenous%20with%20the%20Medicago%20truncatula%20Genome&amp;rft.volume=4&amp;rft.issue=10&amp;rft.date=2014-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Li%2C+Xuehui&amp;rft.au=Wei%2C+Yanling&amp;rft.au=Acharya%2C+Ananta&amp;rft.au=Jiang%2C+Qingzhen&amp;rft.au=Kang%2C+Junmei&amp;rft.au=Brummer%2C+E.+Charles"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459696/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459696/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459696/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459696/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459696/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459696/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459696/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459696/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459696/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459696/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459696/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459696/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/10/1971" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/10/1971.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/10/1971.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/10/1971.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/10/1971.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
