<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/10/1893" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/10/1893.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/10/1893.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/10/1893.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/10/1893.atom" />
<meta name="HW.pisa" content="ggg;4/10/1893" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Transcriptome and Complexity-Reduced, DNA-Based Identification of Intraspecies Single-Nucleotide Polymorphisms in the Polyploid Gossypium hirsutum L." />
<meta name="DC.Identifier" content="10.1534/g3.114.012542" />
<meta name="DC.Date" content="2014-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Zhu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Varietal single nucleotide polymorphisms (SNPs) are the differences within one of the two subgenomes between different tetraploid cotton varieties and have not been practically used in cotton genetics and breeding because they are difficult to identify due to low genetic diversity and very high sequence identity between homeologous genes in cotton. We have used transcriptome and restriction site−associated DNA sequencing to identify varietal SNPs among 18 G. hirsutum varieties based on the rationale that varietal SNPs can be more confidently called when flanked by subgenome-specific SNPs. Using transcriptome data, we successfully identified 37,413 varietal SNPs and, of these, 22,121 did not have an additional varietal SNP within their 20-bp flanking regions so can be used in most SNP genotyping assays. From restriction site−associated DNA sequencing data, we identified an additional 3090 varietal SNPs between two of the varieties. Of the 1583 successful SNP assays achieved using different genotyping platforms, 1363 were verified. Many of the SNPs behaved as dominant markers because of coamplification from homeologous loci, but the number of SNPs acting as codominant markers increased when one or more subgenome-specific SNP(s) were incorporated in their assay primers, giving them greater utility for breeding applications. A G. hirsutum genetic map with 1244 SNP markers was constructed covering 5557.42 centiMorgan and used to map qualitative and quantitative traits. This collection of G. hirsutum varietal SNPs complements existing intra-specific SNPs and provides the cotton community with a valuable marker resource applicable to genetic analyses and breeding programs." />
<meta name="DC.Contributor" content="Qian-Hao Zhu" />
<meta name="DC.Contributor" content="Andrew Spriggs" />
<meta name="DC.Contributor" content="Jennifer M. Taylor" />
<meta name="DC.Contributor" content="Danny Llewellyn" />
<meta name="DC.Contributor" content="Iain Wilson" />
<meta name="article:published_time" content="2014-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Transcriptome and Complexity-Reduced, DNA-Based Identification of Intraspecies Single-Nucleotide Polymorphisms in the Polyploid Gossypium hirsutum L." />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Varietal single nucleotide polymorphisms (SNPs) are the differences within one of the two subgenomes between different tetraploid cotton varieties and have not been practically used in cotton genetics and breeding because they are difficult to identify due to low genetic diversity and very high sequence identity between homeologous genes in cotton. We have used transcriptome and restriction site−associated DNA sequencing to identify varietal SNPs among 18 &lt;i&gt;G. hirsutum&lt;/i&gt; varieties based on the rationale that varietal SNPs can be more confidently called when flanked by subgenome-specific SNPs. Using transcriptome data, we successfully identified 37,413 varietal SNPs and, of these, 22,121 did not have an additional varietal SNP within their 20-bp flanking regions so can be used in most SNP genotyping assays. From restriction site−associated DNA sequencing data, we identified an additional 3090 varietal SNPs between two of the varieties. Of the 1583 successful SNP assays achieved using different genotyping platforms, 1363 were verified. Many of the SNPs behaved as dominant markers because of coamplification from homeologous loci, but the number of SNPs acting as codominant markers increased when one or more subgenome-specific SNP(s) were incorporated in their assay primers, giving them greater utility for breeding applications. A &lt;i&gt;G. hirsutum&lt;/i&gt; genetic map with 1244 SNP markers was constructed covering 5557.42 centiMorgan and used to map qualitative and quantitative traits. This collection of &lt;i&gt;G. hirsutum&lt;/i&gt; varietal SNPs complements existing intra-specific SNPs and provides the cotton community with a valuable marker resource applicable to genetic analyses and breeding programs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/10/01" />
<meta name="citation_mjid" content="ggg;4/10/1893" />
<meta name="citation_id" content="4/10/1893" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/10/1893" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/10/1893.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/10/1893.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/10/1893.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.012542" />
<meta name="citation_pmid" content="25106949" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1893" />
<meta name="citation_lastpage" content="1905" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Qian-Hao Zhu" />
<meta name="citation_author_institution" content="CSIRO Plant Industry, Canberra, ACT 2601, Australia" />
<meta name="citation_author_email" content="Qianhao.Zhu@csiro.au" />
<meta name="citation_author_email" content="Iain.Wilson@csiro.au" />
<meta name="citation_author" content="Andrew Spriggs" />
<meta name="citation_author_institution" content="CSIRO Plant Industry, Canberra, ACT 2601, Australia" />
<meta name="citation_author" content="Jennifer M. Taylor" />
<meta name="citation_author_institution" content="CSIRO Plant Industry, Canberra, ACT 2601, Australia" />
<meta name="citation_author" content="Danny Llewellyn" />
<meta name="citation_author_institution" content="CSIRO Plant Industry, Canberra, ACT 2601, Australia" />
<meta name="citation_author" content="Iain Wilson" />
<meta name="citation_author_institution" content="CSIRO Plant Industry, Canberra, ACT 2601, Australia" />
<meta name="citation_author_email" content="Qianhao.Zhu@csiro.au" />
<meta name="citation_author_email" content="Iain.Wilson@csiro.au" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. L. Adams;citation_author=R. Cronn;citation_author=R. Percifield;citation_author=J. F. Wendel;citation_title=Genes duplicated by polyploidy show unequal contributions to the transcriptome and organ-specific reciprocal silencing.;citation_volume=100;citation_year=2003;citation_pmid=12665616;citation_doi=10.1073/pnas.0630618100" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=A. M. Allen;citation_author=G. L. A. Barker;citation_author=S. T. Berry;citation_author=J. A. Coghill;citation_author=R. Gwilliam;citation_title=Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (Triticum aestivum L.).;citation_pages=1086-1099;citation_volume=9;citation_year=2011;citation_issue=9;citation_pmid=21627760;citation_doi=10.1111/j.1467-7652.2011.00628.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=A. M. Allen;citation_author=G. L. Barker;citation_author=P. Wilkinson;citation_author=A. Burridge;citation_author=M. Winfield;citation_title=Discovery and development of exome-based, co-dominant single nucleotide polymorphism markers in hexaploid wheat (Triticum aestivum L.).;citation_pages=279-295;citation_volume=11;citation_year=2013;citation_pmid=23279710;citation_doi=10.1111/pbi.12009" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=C. An;citation_author=S. Saha;citation_author=J. N. Jenkins;citation_author=B. E. Scheffler;citation_author=T. A. Wilkins;citation_title=Transcriptome profiling, sequence characterization, and SNP-based chromosomal assignment of the EXPANSIN genes in cotton.;citation_pages=539-553;citation_volume=278;citation_year=2007;citation_issue=5;citation_pmid=17724613;citation_doi=10.1007/s00438-007-0270-9" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. An;citation_author=S. Saha;citation_author=J. N. Jenkins;citation_author=D. P. Ma;citation_author=B. E. Scheffler;citation_title=Cotton (Gossypium spp.) R2R3-MYB transcription factors SNP identification, phylogenomic characterization, chromosome localization, and linkage mapping.;citation_pages=1015-1026;citation_volume=116;citation_year=2008;citation_issue=7;citation_pmid=18338155;citation_doi=10.1007/s00122-008-0732-4" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. J. Andres;citation_author=D. T. Bowman;citation_author=B. Kaur;citation_author=V. Kuraparthy;citation_title=Mapping and genomic targeting of the major leaf shape gene (L) in Upland cotton (Gossypium hirsutum L.).;citation_pages=167-177;citation_volume=127;citation_year=2014;citation_pmid=24158249;citation_doi=10.1007/s00122-013-2208-4" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=I. Bancroft;citation_author=C. Morgan;citation_author=F. Fraser;citation_author=J. Higgins;citation_author=R. Wells;citation_title=Dissecting the genome of the polyploid crop oilseed rape by transcriptome sequencing.;citation_pages=762-766;citation_volume=29;citation_year=2011;citation_issue=8;citation_pmid=21804563;citation_doi=10.1038/nbt.1926" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. Barchi;citation_author=S. Lanteri;citation_author=E. Portis;citation_author=A. Acquadro;citation_author=G. Vale;citation_title=Identification of SNP and SSR markers in eggplant using RAD tag sequencing.;citation_pages=304-304;citation_volume=12;citation_year=2011;citation_pmid=21663628;citation_doi=10.1186/1471-2164-12-304" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=G. Barker;citation_author=J. Batley;citation_author=H. O’Sullivan;citation_author=K. J. Edwards;citation_author=D. Edwards;citation_title=Redundancy based detection of sequence polymorphisms in expressed sequence tag data using autoSNP;citation_pages=421-422;citation_volume=19;citation_year=2003;citation_issue=3;citation_pmid=12584131;citation_doi=10.1093/bioinformatics/btf881" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. L. Byers;citation_author=D. B. Harker;citation_author=S. M. Yourstone;citation_author=P. J. Maughan;citation_author=J. A. Udall;citation_title=Development and mapping of SNP assays in allotetraploid cotton.;citation_pages=1201-1214;citation_volume=124;citation_year=2012;citation_issue=7;citation_pmid=22252442;citation_doi=10.1007/s00122-011-1780-8" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences;citation_pages=171-182;citation_volume=1;citation_year=2011;citation_issue=3;citation_pmid=22384329;citation_doi=10.1534/g3.111.000240" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. R. Cavanagh;citation_author=S. Chao;citation_author=S. Wang;citation_author=B. E. Huang;citation_author=S. Stephen;citation_title=Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.;citation_volume=110;citation_year=2013;citation_pmid=23630259;citation_doi=10.1073/pnas.1217133110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Chaudhary;citation_author=L. Flagel;citation_author=R. M. Stupar;citation_author=J. A. Udall;citation_author=N. Verma;citation_title=Reciprocal silencing, transcriptional bias and functional divergence of homeologs in polyploid cotton (Gossypium).;citation_volume=182;citation_year=2009;citation_pmid=19363125;citation_doi=10.1534/genetics.109.102608" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Chen;citation_author=H. He;citation_author=Y. Zou;citation_author=W. Chen;citation_author=R. Yu;citation_title=Development and application of a set of breeder-friendly SNP markers for genetic analyses and molecular breeding of rice (Oryza sativa L.).;citation_pages=869-879;citation_volume=123;citation_year=2011;citation_issue=6;citation_pmid=21681488;citation_doi=10.1007/s00122-011-1633-5" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Plant Physiol.;citation_author=Y. L. Cousins;citation_author=B. R. Lyon;citation_author=D. J. Llewellyn;citation_title=Transformation of an Australian cotton cultivar - prospects for cotton improvement through genetic-engineering.;citation_pages=481-494;citation_volume=18;citation_year=1991;citation_doi=10.1071/PP9910481" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=A. Desai;citation_author=P. W. Chee;citation_author=O. L. May;citation_author=A. H. Paterson;citation_title=Correspondence of trichome mutations in diploid and tetraploid cottons.;citation_volume=99;citation_year=2008;citation_pmid=18296391;citation_doi=10.1093/jhered/esm112" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breeding;citation_author=H. Fang;citation_author=H. P. Zhou;citation_author=S. Sanogo;citation_author=A. E. Lipka;citation_author=D. D. Fang;citation_title=Quantitative trait locus analysis of Verticillium wilt resistance in an introgressed recombinant inbred population of Upland cotton.;citation_pages=709-720;citation_volume=33;citation_year=2014;citation_doi=10.1007/s11032-013-9987-9" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=L. E. Flagel;citation_author=J. F. Wendel;citation_title=Evolutionary rate variation, genomic dominance and duplicate gene expression evolution during allotetraploid cotton speciation.;citation_pages=184-193;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=20002320;citation_doi=10.1111/j.1469-8137.2009.03107.x" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=M. W. Ganal;citation_author=T. Altmann;citation_author=M. S. Roder;citation_title=SNP identification in crop plants.;citation_pages=211-217;citation_volume=12;citation_year=2009;citation_issue=2;citation_pmid=19186095;citation_doi=10.1016/j.pbi.2008.12.009" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=C. Y. Hsu;citation_author=C. An;citation_author=S. Saha;citation_author=D. P. Ma;citation_author=J. N. Jenkins;citation_title=Molecular and SNP characterization of two genome specific transcription factor genes GhMyb8 and GhMyb10 in cotton species.;citation_pages=259-273;citation_volume=159;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Huang;citation_author=L. Deng;citation_author=M. Guan;citation_author=J. Li;citation_author=K. Lu;citation_title=Identification of genome-wide single nucleotide polymorphisms in allopolyploid crop Brassica napus.;citation_pages=717;citation_volume=14;citation_year=2013;citation_pmid=24138473;citation_doi=10.1186/1471-2164-14-717" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Huang;citation_author=Q. Feng;citation_author=Q. Qian;citation_author=Q. Zhao;citation_author=L. Wang;citation_title=High-throughput genotyping by whole-genome resequencing.;citation_volume=19;citation_year=2009;citation_pmid=19420380;citation_doi=10.1101/gr.089516.108" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Huang;citation_author=Y. Zhao;citation_author=X. Wei;citation_author=C. Li;citation_author=A. Wang;citation_title=Genome-wide association study of flowering time and grain yield traits in a worldwide collection of rice germplasm.;citation_pages=32-39;citation_volume=44;citation_year=2012;citation_issue=1;citation_pmid=22138690;citation_doi=10.1038/ng.1018" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Jiang;citation_author=R. J. Wright;citation_author=S. S. Woo;citation_author=T. A. DelMonte;citation_author=A. H. Paterson;citation_title=QTL analysis of leaf morphology in tetraploid Gossypium (cotton).;citation_pages=409-418;citation_volume=100;citation_year=2000;citation_doi=10.1007/s001220050054" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Y. Jiao;citation_author=H. Zhao;citation_author=L. Ren;citation_author=W. Song;citation_author=B. Zeng;citation_title=Genome-wide genetic changes during modern breeding of maize.;citation_pages=812-815;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22660547;citation_doi=10.1038/ng.2312" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=S. Kaur;citation_author=M. G. Francki;citation_author=J. W. Forster;citation_title=Identification, characterization and interpretation of single-nucleotide sequence variation in allopolyploid crop species.;citation_pages=125-138;citation_volume=10;citation_year=2012;citation_issue=2;citation_pmid=21831136;citation_doi=10.1111/j.1467-7652.2011.00644.x" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=J. M. Lacape;citation_author=T. B. Nguyen;citation_title=Mapping quantitative trait loci associated with leaf and stem pubescence in cotton.;citation_volume=96;citation_year=2005;citation_pmid=15829730;citation_doi=10.1093/jhered/esi052" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=J. M. Lacape;citation_author=G. Gawrysiak;citation_author=T. V. Cao;citation_author=C. Viot;citation_author=D. Llewellyn;citation_title=Mapping QTLs for traits related to phenology, morphology and yield components in an inter-specific Gossypium hirsutum x G. barbadense cotton RIL population.;citation_pages=256-267;citation_volume=144;citation_year=2013;citation_doi=10.1016/j.fcr.2013.01.001" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=F. Li;citation_author=G. Fan;citation_author=K. Wang;citation_author=F. Sun;citation_author=Y. Yuan;citation_title=Genome sequence of the cultivated cotton Gossypium arboreum.;citation_pages=567-572;citation_volume=46;citation_year=2014;citation_pmid=24836287;citation_doi=10.1038/ng.2987" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Li;citation_author=G. Ye;citation_author=J. Wang;citation_title=A modified algorithm for the improvement of composite interval mapping.;citation_volume=175;citation_year=2007;citation_pmid=17110476" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. Li;citation_author=Z. Peng;citation_author=X. Yang;citation_author=W. Wang;citation_author=J. Fu;citation_title=Genome-wide association study dissects the genetic architecture of oil biosynthesis in maize kernels.;citation_pages=43-50;citation_volume=45;citation_year=2013;citation_pmid=23242369" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=L. A. B. Lopez-Lavalle;citation_author=V. J. Gillespie;citation_author=W. A. Tate;citation_author=M. H. Ellis;citation_author=W. N. Stiller;citation_title=Molecular mapping of a new source of Fusarium wilt resistance in tetraploid cotton (Gossypium hirsutum L.).;citation_pages=1181-1191;citation_volume=30;citation_year=2012;citation_doi=10.1007/s11032-012-9705-z" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=M. Mascher;citation_author=T. A. Richmond;citation_author=D. J. Gerhardt;citation_author=A. Himmelbach;citation_author=L. Clissold;citation_title=Barley whole exome capture: a tool for genomic research in the genus Hordeum and beyond.;citation_pages=494-505;citation_volume=76;citation_year=2013;citation_pmid=23889683;citation_doi=10.1111/tpj.12294" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=J. T. Page;citation_author=A. R. Gingle;citation_author=J. A. Udall;citation_title=PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms;citation_pages=517-525;citation_volume=3;citation_year=2013;citation_issue=3;citation_pmid=23450226;citation_doi=10.1534/g3.112.005298" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=J. T. Page;citation_author=M. D. Huynh;citation_author=Z. S. Liechty;citation_author=K. Grupp;citation_author=D. Stelly;citation_title=Insights into the evolution of cotton diploids and polyploids from whole-genome re-sequencing.;citation_volume=3;citation_year=2013;citation_pmid=23979935;citation_doi=10.1534/g3.113.007229" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. H. Paterson;citation_author=J. F. Wendel;citation_author=H. Gundlach;citation_author=H. Guo;citation_author=J. Jenkins;citation_title=Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibers.;citation_pages=423-427;citation_volume=492;citation_year=2012;citation_pmid=23257886;citation_doi=10.1038/nature11798" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=M. Pomeranz;citation_author=J. Campbell;citation_author=D. Siegal-Gaskins;citation_author=J. Engelmeier;citation_author=T. Wilson;citation_title=High-resolution computational imaging of leaf hair patterning using polarized light microscopy.;citation_pages=701-708;citation_volume=73;citation_year=2013;citation_pmid=23163919;citation_doi=10.1111/tpj.12075" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. Rahman;citation_author=D. Hussain;citation_author=Y. Zafar;citation_title=Estimation of Genetic Divergence among Elite Cotton Cultivars-Genotypes by DNA Fingerprinting Technology;citation_pages=2137-2144;citation_volume=42;citation_year=2002;citation_issue=6;citation_doi=10.2135/cropsci2002.2137" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=K. M. Rai;citation_author=S. K. Singh;citation_author=A. Bhardwaj;citation_author=V. Kumar;citation_author=D. Lakhwani;citation_title=Large-scale resource development in Gossypium hirsutum L. by 454 sequencing of genic-enriched libraries from six diverse genotypes.;citation_pages=953-963;citation_volume=11;citation_year=2013;citation_pmid=23782852;citation_doi=10.1111/pbi.12088" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=R. A. Rapp;citation_author=J. A. Udall;citation_author=J. F. Wendel;citation_title=Genomic expression dominance in allopolyploids.;citation_pages=18-18;citation_volume=7;citation_year=2009;citation_pmid=19409075;citation_doi=10.1186/1741-7007-7-18" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Riedelsheimer;citation_author=A. Czedik-Eysenberg;citation_author=C. Grieder;citation_author=J. Lisec;citation_author=F. Technow;citation_title=Genomic and metabolic prediction of complex heterotic traits in hybrid maize.;citation_pages=217-220;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22246502;citation_doi=10.1038/ng.1033" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Rong;citation_author=C. Abbey;citation_author=J. E. Bowers;citation_author=C. L. Brubaker;citation_author=C. Chang;citation_title=A 3347-Locus Genetic Recombination Map of Sequence-Tagged Sites Reveals Features of Genome Organization, Transmission and Evolution of Cotton (Gossypium);citation_pages=389-417;citation_volume=166;citation_year=2004;citation_issue=1;citation_pmid=15020432;citation_doi=10.1534/genetics.166.1.389" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=D. Rungis;citation_author=D. Llewellyn;citation_author=E. S. Dennis;citation_author=B. R. Lyon;citation_title=Simple sequence repeat (SSR) markers reveal low levels of polymorphism between cotton (Gossypium hirsutum L.) cultivars.;citation_pages=301-307;citation_volume=56;citation_year=2005;citation_doi=10.1071/AR04190" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=C. Saintenac;citation_author=D. Jiang;citation_author=S. Wang;citation_author=E. Akhunov;citation_title=Sequence-based mapping of the polyploid wheat genome.;citation_volume=3;citation_year=2013;citation_pmid=23665877;citation_doi=10.1534/g3.113.005819" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=A. Salmon;citation_author=L. Flagel;citation_author=B. Ying;citation_author=J. A. Udall;citation_author=J. F. Wendel;citation_title=Homoeologous nonreciprocal recombination in polyploid cotton.;citation_pages=123-134;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=19925554;citation_doi=10.1111/j.1469-8137.2009.03093.x" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=A. Salmon;citation_author=J. A. Udall;citation_author=J. A. Jeddeloh;citation_author=J. Wendel;citation_title=Targeted Capture of Homoeologous Coding and Noncoding Sequence in Polyploid Cotton;citation_pages=921-930;citation_volume=2;citation_year=2012;citation_issue=8;citation_pmid=22908041;citation_doi=10.1534/g3.112.003392" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=R. L. Small;citation_author=J. A. Ryburn;citation_author=J. F. Wendel;citation_title=Low levels of nucleotide diversity at homoeologous Adh loci in allotetraploid cotton (Gossypium L.);citation_pages=491-501;citation_volume=16;citation_year=1999;citation_issue=4;citation_pmid=10331275;citation_doi=10.1093/oxfordjournals.molbev.a026131" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. Trebbi;citation_author=M. Maccaferri;citation_author=P. de Heer;citation_author=A. Sorensen;citation_author=S. Giuliani;citation_title=High-throughput SNP discovery and genotyping in durum wheat (Triticum durum Desf.).;citation_pages=555-569;citation_volume=123;citation_year=2011;citation_issue=4;citation_pmid=21611761;citation_doi=10.1007/s00122-011-1607-7" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=A. Van Deynze;citation_author=K. Stoffel;citation_author=M. Lee;citation_author=T. A. Wilkins;citation_author=A. Kozik;citation_title=Sampling nucleotide diversity in cotton.;citation_pages=125-125;citation_volume=9;citation_year=2009;citation_pmid=19840401;citation_doi=10.1186/1471-2229-9-125" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_journal_title=Analytical biochemistry;citation_journal_abbrev=Anal Biochem;citation_author=C. Y. Wan;citation_author=T. A. Wilkins;citation_title=A modified hot borate method significantly enhances the yield of high-quality RNA from cotton (Gossypium hirsutum L.).;citation_pages=7-12;citation_volume=223;citation_year=1994;citation_issue=1;citation_pmid=7535022;citation_doi=10.1006/abio.1994.1538" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=N. Wang;citation_author=L. Fang;citation_author=H. Xin;citation_author=L. Wang;citation_author=S. Li;citation_title=Construction of a high-density genetic map for grape using next generation restriction-site associated DNA sequencing.;citation_pages=148-148;citation_volume=12;citation_year=2012;citation_issue=1;citation_pmid=22908993;citation_doi=10.1186/1471-2229-12-148" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Wang;citation_author=Z. Wang;citation_author=F. Li;citation_author=W. Ye;citation_author=J. Wang;citation_title=The draft genome of a diploid cotton Gossypium raimondii.;citation_pages=1098-1103;citation_volume=44;citation_year=2012;citation_issue=10;citation_pmid=22922876;citation_doi=10.1038/ng.2371" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=Z. Wang;citation_author=D. Zhang;citation_author=X. Wang;citation_author=X. Tan;citation_author=H. Guo;citation_title=A whole-genome DNA marker map for cotton based on the D-genome sequence of Gossypium raimondii L.;citation_pages=1759-1767;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Adv. Agron.;citation_author=J. F. Wendel;citation_author=R. C. Cronn;citation_title=Polyploidy and the evolutionary history of cotton.;citation_pages=139-186;citation_volume=78;citation_year=2003;citation_doi=10.1016/S0065-2113(02)78004-8" />
<meta name="citation_reference" content="citation_title=Physiology of Cotton;citation_title=The origin and evolution of Gossypium;citation_author=J. F. Wendel;citation_author=C. L. Brubaker;citation_author=T. Seelanan;citation_pages=1-18;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. J. Wright;citation_author=P. M. Thaxton;citation_author=K. H. El-Zik;citation_author=A. H. Paterson;citation_title=Molecular mapping of genes affecting pubescence of cotton;citation_pages=215-219;citation_volume=90;citation_year=1999;citation_issue=1;citation_doi=10.1093/jhered/90.1.215" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=H. Yang;citation_author=Y. Tao;citation_author=Z. Zheng;citation_author=C. Li;citation_author=M. W. Sweetingham;citation_title=Application of next-generation sequencing for rapid marker development in molecular plant breeding: a case study on anthracnose disease resistance in Lupinus angustifolius L.;citation_pages=318-318;citation_volume=13;citation_year=2012;citation_pmid=22805587;citation_doi=10.1186/1471-2164-13-318" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=M. J. Yoo;citation_author=E. Szadkowski;citation_author=J. F. Wendel;citation_title=Homoeolog expression bias and expression level dominance in allopolyploid cotton.;citation_pages=171-180;citation_volume=110;citation_year=2013;citation_pmid=23169565;citation_doi=10.1038/hdy.2012.94" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Transcriptome and Complexity-Reduced, DNA-Based Identification of Intraspecies Single-Nucleotide Polymorphisms in the Polyploid Gossypium hirsutum L." />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10.cover-source.jpg" />
<meta name="twitter:description" content="Varietal single nucleotide polymorphisms (SNPs) are the differences within one of the two subgenomes between different tetraploid cotton varieties and have not been practically used in cotton genetics and breeding because they are difficult to identify due to low genetic diversity and very high sequence identity between homeologous genes in cotton. We have used transcriptome and restriction site−associated DNA sequencing to identify varietal SNPs among 18 G. hirsutum varieties based on the rationale that varietal SNPs can be more confidently called when flanked by subgenome-specific SNPs. Using transcriptome data, we successfully identified 37,413 varietal SNPs and, of these, 22,121 did not have an additional varietal SNP within their 20-bp flanking regions so can be used in most SNP genotyping assays. From restriction site−associated DNA sequencing data, we identified an additional 3090 varietal SNPs between two of the varieties. Of the 1583 successful SNP assays achieved using different genotyping platforms, 1363 were verified. Many of the SNPs behaved as dominant markers because of coamplification from homeologous loci, but the number of SNPs acting as codominant markers increased when one or more subgenome-specific SNP(s) were incorporated in their assay primers, giving them greater utility for breeding applications. A G. hirsutum genetic map with 1244 SNP markers was constructed covering 5557.42 centiMorgan and used to map qualitative and quantitative traits. This collection of G. hirsutum varietal SNPs complements existing intra-specific SNPs and provides the cotton community with a valuable marker resource applicable to genetic analyses and breeding programs." />
<meta name="og-title" property="og:title" content="Transcriptome and Complexity-Reduced, DNA-Based Identification of Intraspecies Single-Nucleotide Polymorphisms in the Polyploid Gossypium hirsutum L." />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/10/1893" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Varietal single nucleotide polymorphisms (SNPs) are the differences within one of the two subgenomes between different tetraploid cotton varieties and have not been practically used in cotton genetics and breeding because they are difficult to identify due to low genetic diversity and very high sequence identity between homeologous genes in cotton. We have used transcriptome and restriction site−associated DNA sequencing to identify varietal SNPs among 18 G. hirsutum varieties based on the rationale that varietal SNPs can be more confidently called when flanked by subgenome-specific SNPs. Using transcriptome data, we successfully identified 37,413 varietal SNPs and, of these, 22,121 did not have an additional varietal SNP within their 20-bp flanking regions so can be used in most SNP genotyping assays. From restriction site−associated DNA sequencing data, we identified an additional 3090 varietal SNPs between two of the varieties. Of the 1583 successful SNP assays achieved using different genotyping platforms, 1363 were verified. Many of the SNPs behaved as dominant markers because of coamplification from homeologous loci, but the number of SNPs acting as codominant markers increased when one or more subgenome-specific SNP(s) were incorporated in their assay primers, giving them greater utility for breeding applications. A G. hirsutum genetic map with 1244 SNP markers was constructed covering 5557.42 centiMorgan and used to map qualitative and quantitative traits. This collection of G. hirsutum varietal SNPs complements existing intra-specific SNPs and provides the cotton community with a valuable marker resource applicable to genetic analyses and breeding programs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10.cover-source.jpg" />
    <title>Transcriptome and Complexity-Reduced, DNA-Based Identification of Intraspecies Single-Nucleotide Polymorphisms in the Polyploid Gossypium hirsutum L. | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"bT-h8Xf0Gp4aRk_5XyU7FxTEWOgfzr2pW1_4MHjAwwE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459681","apath":"\/ggg\/4\/10\/1893.atom","pisa":"ggg;4\/10\/1893","ac":{"\/ggg\/4\/10\/1893.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/10\/1893.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/10\/1893"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/10\/1893"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/10\/1893":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022heHW4WI0ITHJe1Zl0WT5v1vthuQ6aGYal4uxUL1O2Ld8wYnCporFb3hjCEO04iJBh2XTj6GoktD8kjIemq4cM102fikAi87qvmWBF6SMQKpdjMNVJbICbQmNiIQQNTOOzlg8jhp+NQRGctP1biFtXdJ6cg7oQy0DwHwzwdP7kHAv33EfxCChm2IHK1PgXY7ElKco5y9hVm5qV99pU4xrBqJxc+qH1lAmS7JSUP46WVEYCq7yJoj1ss5yzTglX1\\\\\\\/xFM2HaErw98rF9tvmGRJc+4q6DuLlbkQV8SujtpE1mWboeVf2rOmfDa9qexLiw60JFDcFXYU\\\\\\\/ZeZ0eb6UXC3xIEzcY6D36yre5M8E980keQLls9srXSLUPDAV7YS09B31n4rc5poo2326lkzfocYCp81NVENt0VXyBGEKG\\\\\\\/am+0fzDODUTvXnJRpazE\\\\\\\/cdgloxzF6yRBGyzXcf4WgU+\\\\\\\/pBzWFRE4RDozfBz1gv4zdPZ4GHr\\\\\\\/Bia8\\\\\\\/pCPH\\\\\\\/l4fNSXrUZrsFLAr4t7C7wLznMQ\\\\\\\/rI10PX0tJTnO9nQfIKqiDcWHtsrvAIplH+uOvzt32yuudvvQWa8DPwAv5cCcZehGehcVe+70F3IeHosq2C8Kuxl0de9csjUqsXki+S3v75PSZ9flkpzcBb9Ug2tm1nixV1hJteLDAMDWyCdxdGHmSpwLa7ACkkQWkJO9uEKNO2fZ63PqDVLwPdpqrLMklofXGChkRGwB3CUpFGCxYTCNxfZVVxgvYkXpj3FO0kMhj+vJM9GpD33AiDVnbMZ5FMU4R11b8hvBAz8mGOBvxcVMfbUTT2apozVOz8DFYUm5af10ax7pMrmtnE5kTr183g2nYx+rDSWnOtmf8pl3HvDF05zQlsdmhHIDeuQQWzkqeEbDFQTmlsLxgBa3LkqzEVRLkiZ8q\\\\\\\/hMP6Ow2x8e4HNgox8lHFLsaq73lNpCCZkKvOPbN\\\\\\\/zko01XKPjh3TxfMoHFejX6h7H7Fks6BCr90+4g6fyzBdtXkhWDSNO7kAxMpfcSeHa\\\\\\\/CDqOhnWYPKnPiRd6F+XiwLNYvh+oOGS9ONTsBq9xtLIf2\\\\\\\/mPJVot1vFNqeklH6Oh+cp03MckmMTaUbB31X14BU\\\\\\\/Saf19VHn3ISOJXi7IwxUcE4Fi47MWJn25QumGT1Qqxpf4JALaYA7EBWUlUO5GxXNd+P9AyEAjJ13AmdIRCLmx00cBPchLufdw1M6CafHbAqvbepkJK21S7wGtGco2DdMlTfcbxF0r5S4vTlCzW4N7LfeLP0DSiBma3eAv\\\u0022,\\\u0022iv\\\u0022:\\\u0022M7AvIiHifm6EBHuijoJmNw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022863d14d66d36a27dac2ba8de0de9073b2adc74513b804899d7f095d9fb91d98d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+++3ZVM91oiu+1Tj+MM2XR3SK5LLzqvmxQau1hy9oXmAc4IAAuYSDAT2Kp6Y2vZHIhkfa2a2jHOwyYVaA\\\\\\\/r6a3qXJ04ZrWCE971iCI+usriH81YrpBmS4J\\\\\\\/A4VyYp\\\\\\\/6qSh7b9+Pj29Rg0fBHTZW+ifnjoyGrVavL8dawp57CII44EoWePP\\\\\\\/cb1nZ9lIKIqwTxbqdMjzFV0\\\\\\\/YOGwEJ2t1Utv9gNEzxUMeLlI+zajpuYrSqiQ1Wekyf2pTNSZLjl0RigsuSZXnFtaMekhpTumUmiECroiHSAyQH58zk3P7sKxb+gbJ0gVYK49+chLSzJE7f1AOsXXf3IQoq56JoCTq+IJZGdNKIrxzVJxLtvBNnFHq24CpXtltGHnHn2Pz1KYGXu0ysBXNQoJkxMQI1PCUMntmhzLg2TMz3E2\\\\\\\/aeSYsjizOsFAA4SHa95w8sVdbWqZ52xbf4gBdmwAu7xeQGTFQLfGzcxShcmEjUmk5Gf78hXav5dy2auYgfJ1BzTF21ujhXTBp0r7y0o0lz0HaxId+BRS3tPTtLPsy+MVnrowQzi9d83Z2urA1YuX4da9a2mn+UwhXbmvUrD74KoqQD9Y7yWW33PfBrngz73CKu1p58Kp6PN\\\\\\\/6oQZGQBqU9MhnKWjeJQJCScViFQ\\\\\\\/+1uOWLBaBTo+i6Z\\\\\\\/MWtKU5N1b+0voiu30Sazs8FIcT140D7IOqWX\\\\\\\/SQZVhuM1HprxlewgyrZhQdEwdCm55RI\\\\\\\/cw3K6mc3OEMldT2kqB+IR\\\\\\\/R083Cgd2fKZe6unGoZpoBGVIDTsni\\\\\\\/Bkbyem\\\\\\\/+HhMqKMCo302BOcDq3OCry7vQtUPZc5ojj4FcKHiUwVYY0Y43NXnGvRnvdDo\\\\\\\/lrSF6Rjw+2K00LhVxKaZ6HQ\\\\\\\/UO8EV3jvCYjcBybwcnylU66Gf1z6zyzE+upG4ZiZVd7O6Pxotsbw7HnG0HLcgrz7glrHZasHdOUZVwjjE\\\\\\\/yA3Z6pPX+Yxlp19YH1CoT2zq+0EVdBA6473YknEUuwlV\\\\\\\/nn8oV1lMc2LoZuJDRBxo2HLNna7K4\\\\\\\/PQRVzZLpgjKy2w\\\\\\\/Vu+YW0uBcAkjLLJuKfc1hT5JKFfo\\\\\\\/MwuFFLPlKv5dlwscu5KRZvyvcQRh4IlYuo5n+Zw946eweX0vaY229VzSegCCzgJz1DJU2Q+ydGDCWrcTPGzYRiMBYra7gmnlch49XRK+hXuyEjvORIGK8ubTCBccuQ4dGk3nlHvFpZv3J7elXalc2UQDtIuT4qlTE4rZxep7fO32XKUv79IC4UfbNu79x4L+mvyXLPsNrqjW5+rcY9UjsuueUwDeMcd+Ck9jym7rXNS2zZ1PKqVJqjJFp1lcns2th3rRag4tVUdtfuTPgJr3UGM0\\\\\\\/RGNalgrESTlKHrNYstInhADQ7js98X87v+La1cQsgFy8iJBRmiMUkeTnQJ8Hs7lCcwlGmyON5ocQmexUNuj0GSbYMo59x++LJLfbj0c5pu89zaOpYaMbltqd813w6Kuk+3mwuwzPvaH1rfqjaiqa8BzQjziX3I29S1doBEWYuOD\\\\\\\/W\\\\\\\/12OcJJNtwdBmxQLHMmynN8dMAWilXHct6mWH7BJoIn54eXn\\\\\\\/vO3mMHX\\\\\\\/Xi9xdIyMUye7R6zrzIAr5LmZLvDIC105Pk4COy+95hh8QPY3mFg30mWjw+uDn7EKnvSD1dBrH94iTqEOqCG0aFnmeJB022VI0E7y2\\\\\\\/TtnFx6CNm\\\\\\\/cwUwmmgXI56v3VR0J+boik9fmfCH8yQDf5SwKa6VJTXID\\\\\\\/Y41xGUTGB1ogcQXnyJhwVlOhNTGDXxk8lD8o7RbG+gwWHpVEImk1jncozjRGT\\\\\\\/4tu341LNxxvorTKRn3ABhaE\\\\\\\/FPxgCWppcpAiFurG21er3digxGrJimXlM+\\\\\\\/ChRvIxOmZBgsgfoP4cM2m1q+WmDqHEqnEk9cbpGPBey9SKOrI\\\\\\\/aRVPG2vROtdEpUggN3oHj+QNByML4UQ9lNKavY5nnsNykYaAlBq5TB7nld2LAi12TeDeinXNG8XDzSxaXURjWQedtENRrV6ssnYU3U\\\\\\\/fBuuCCTfSPnoh4TdcIAN6856P7DaeSjeWhCqv8u11hcpKXykscZhxUTOtgt98hLcIGLB2JRDcSMw3EvMqMD\\\\\\\/2ZPzfmyTc4ONyMXmC3hDkjy2mJhpS+q4iG7G5tFoUhoZTydLDrDAzowScu6PmKduy9OcP9xDTJldQ8BLR1oZwDsTeOC+ZohqdqCNqRFeBSnWS8\\\\\\\/X2RkrOLoG1LwWV7ygVFrUUJKAaXHgt8l3EAXMfotsxILtwqv50n6octFwrcmcK\\\\\\\/hlJBQyChXQiBPeKDenxRsukHnhl0hNjzd5h8BysPLyR9B8L4kfCCC5ANOUla3dKa4aOMJ7v+2zhV3kvX5dTeUoV1ZfYLhQkPZYwcTmE8dotwkq2EIZavNCVzPiryKwr32bhteVQSt1xCPkMEwu0ahkdLQxoaLvb+BJsklyPsxObxPSK52GiPaY+9lRBYNpF3bUJt2MbMMPvohK5betPYMKAQHvC++DrD8MZSMoFAihz8DhcAfSwxrkbtr7ZXDitU4W9bUwq3mR9KPETCrm3mYusHqCJAgR7slViUDEucUaI0Y48q7XTmfe1EuHZchSk2nmW4eG0i0+IV1m8+j4knzHijsMXQBV+Rizp71629d5h22ZfofSUPXeOkQJDvYGj9N5xLoKLwfXRLGkCS9NtIeUADe\\\\\\\/Jn5vTXphm5sbMEYM60jI7EA6JnAQBSoVQtCZSzwHIvQm8F6nlTZJWzG8umni5owhSy220on8stmvWqtZhOQjDYkC7lBzu443JSD8N5jqbtR+Z1GXAlT\\\\\\\/sdDE2O\\\\\\\/ChJyqUFq1dmfoXjxs3BZS2QCEN2xWtp4N42GFbu1ZXQFSt5SZpRfFMWYPCkugjHPowEDuF0i3s\\\\\\\/ZHS9mDrnvOFK1N0TLavJsSyF\\\\\\\/PTW\\\\\\\/YSX2WMDciW8ragFLDkTnfKcRJs21WXQ47nPWmatsQKUIMaG\\\\\\\/ZTOZaJ1lJ2F1ZaTGfwzkBjMaygU1bhdH5e65Ak168ZGn+MKC9fpl0Q7XIHzzrbkcHfR4ORsz6G9gXQxmL9T2t+WIOX+p0YaA4cIAAlJcOHv78jPc2usCpakuGuWWDBOOpEAUvOQzgm4w0+8kG85WLNhdjVpwsEyr0vIUDHeNrkZwH3zQa\\\\\\\/qPOwUS5ZIfabNhQvtLfQCEiValSvGrkGS4id6E0vlfT2eCdjuPUiB3pqnCYGCyAZy5UmFYKfj2NG5SJMtVxRgOANYyK2kSm2gH03nvz2CAA2aqvw3PUfxjGXNizEASr9u5tV1T90pAGc6xIbHcUogVp3PSwsimeg3dDvfPgA\\\\\\\/VXPFEXRQvHtz0hNLwBQNV5fZEKLUovplCa7iMqiz2dRxYURxeXmsLCuNqnv+lgrHywgUnFtTi5blAi9DBq8PFflt5wEIxAwrA+a1LS22oU6R7cr5nmyWWv40QlVqxta1PQxfTegjCV9eiHvrg5arp1eCnV\\\\\\\/056HRAPpIxAH8AuHdA\\\\\\\/dlsL1AG3xNC4Y5y1HcGS4gmSt\\\\\\\/jFAPGLa46gupgtt51F8aj+Gs+\\\\\\\/u866TLnDeYfprsQLtG1znG9\\\\\\\/1eoXP8GaVr46stIyH132JVoU\\\\\\\/rzhEv+RdRNX7XXiZaGWHkBjtQshIwYW5Hmhw9UosdqTHtnz20Bya7wClNaep\\\\\\\/HtFaX3KjZXzosbnmF52AZdS0GYYZaM5LjIgGpp82r3z4+MRVh80IwQ32s7vsPgKoMUwjvTD2jDku5d0yf9RSMSWTdJTo0OXKE5Z7gbHq95DGk8oJFT5GTl0Ic3PqUFHBxfuUd\\\\\\\/gVpc7o1x0idQr3J9uWXIxK009MhcOGaHg2bAPVGSpNn2DkQ6ua2aRTTHLWXwttiJiYFSsinOc\\\\\\\/FYof7BxSYFTR7zpYOYpUntz83am5lHILHRu5ns4pmQMEdwvIGceV9FKDV87DOw1N1Av1ntaEVsNREeEW4A1ZvfDM9LBtelKUxlZ345nvNZbbUocBqhxFE5FBD2O1cm1X5ec56CDSZp7JPvygqBJHkwYMrwrywVXcYo0+KUgO3vmji3cZ8bgX0kLlOwY22Q2wvgrgAFe7Kj8io4uY\\\\\\\/xFBMqnsAAxWsR1\\\\\\\/LaO4KtDh2D0NMYZduLZeyEZHea90zwjmvylXGGtr5BAQhqgozjkFgemPUqnEbpamkhMAyBvlrY\\\\\\\/dJw\\\\\\\/heGB7aqi7G8TcL7mAJEf9pRF0u7AwypFdXyDJjATV+b514dNLzp0S049ae5jcP2WBKgY8RAbe1oX12CLeJ8Efhev+rtAAWgOsoJzyJrqVbibjVb2naeFwHmHT+xzy9ykyXaJZ4ceJWkDvs5UF9LFiNSi6wpV7szfVhDWRuic0LBiOZMSvp84+RM2R5FEOqYbXlIudmIF0q\\\\\\\/XExgVoMq2e\\\\\\\/Knjy\\\\\\\/We0E1+0bAwhNrL8U5KbDS8yQR57VsbkHZMM\\\\\\\/6KTofchk2y7pcDx1MkLGYKpbchxIAuc7f9B9hEpz2e9w610\\\\\\\/JKOdfd\\\\\\\/NT8TkiQ8Ir4b4cSW4foUlAaUdmy8NIZq\\\\\\\/1dWXFIfZCFHjSsKdY7+kOIdiphIHv8WI1498Vezzwrh4cwUKFyptDsE6UbMil4JJ7qIauYMCfGjJQ1oMrdhHvke0HMJBrMgVYa9gmhPj\\\\\\\/t0lEOOScSzPAEElsQILkTRxCNsZAuDPGqHxomUsb0B78lIGqGMQfs3o34pC0iR0+zo6IopLpEBikTC8YAjnueimm2pqyUT1HxwIbEsY+hl464ie3nyX6SzavdSS9BdAzXafR4gSUgWlc7hdLaokQAF5sObzirsWa5i3c3N\\\\\\\/za8ztgBE6vf7NXKgZx8sZwQ6RkF4VI3G0qlX\\\\\\\/2SqlkFbhh4mSLY9wiRi2fK0bI14hFW3+lEElrbM837KtbpxUHDDMogOP3GNziM417OP1ywZYxk4QxSaVlguG5iTMO8hiVzGIDXPJK1Kh3Bet2uaSd7EBgkKv\\\\\\\/kJSK1V052T7nWrSrnYnWY2OJSj9R2GyomqYlXbuijroRjsPQpDXCEoHnoHnAPLvh26WJNVXHPSCnjMHNldREk9L3yWnY74iP050ra12ujjnitSBhBp28rsk07h1mHTvvsZXdTu2c7ORbOTwrngmrJAbs8h\\\\\\\/kKgP5fP6gTj3QOLU5JeU2MZhjmccdXdDVYg5KB8UdbBGCf64+7hL3JPGmkwUByiEH+X6PKNYfomXyRN7mey1AjwzB2GvxcUiXb5mS7MJfMpxQ989v4wZ75QPv2hIelRWsed72QxX93r1snrT9Ska+yLF\\\\\\\/TuPSvbv3SjJusTTKCMEzfRd\\\\\\\/bju2cCTQVzQMwicvuMTIysP+4ETIVrgv7mqfOQlTwtKiWP\\\\\\\/v1bvJqzEfCDI6NnbkzmTpJX07xALLhGcoV4m37EBZsthX5fv\\\\\\\/pftTELbwawjHs4h3vJY0DJHxPypHYwtvV9ru+nKnB4fjFa0GlgIS\\\\\\\/UoYLjHyWMAOuoGP5TugEyJyXy2dE+YdrfJVjVrbxaTdWi4dRDqEskExzpjjbSvyfszNQB4VH5VzIutU7Pm95BVkyWo91Mpv5vNotgrKZq7hiGmcC+ymMiYlQeG2BB8qzn+ZIUqJX+LQt85AGOdDySl9X2\\\\\\\/YehjPLH5jIXLCVviaVdhQHXOuo2UZO0+aadw4dwEEzR7IVmo6P4yNYiDXgYXGPKSbYJscuSOvS5UtjBtP2DwYkLLDfd1H005Dggntq11NKTaCpiHID4NAqr4GwWtNeJFJAzTJBpnfXlElU\\\\\\\/oL0DNXTGwWhQ2cCsis8ngWSsFhbmsWrdQq9R+SMSGkO7MxNRYw6D8uIs9+lYWtlxWvIsWk+K9L6KWGawnQ\\\\\\\/IQ5CUCMpYWYjNJ7JA1aStjrTsX32c7lHohH6hSTLGM3t4mxb6nih9dDpBArnsgcksRN1AxYaFOc64pmZxRaWylF\\\\\\\/K5TOrZsGRiwL6bjpUo1Rudvy8Xl8cJW9TIYeo0RywJgDe2vAvGNUDaP3T4kTXhSaPR0lRvt7Y9ytUyeqyrupmQMRgqzuWy81O9iXR4SvFjxo8ZlNmYTV5wRFzMlMl2KF9dELcWmxSrOIa1\\\\\\\/vFQsuehZOYJe6mnKnXNLOugGcdcVJhkdRs4zn0uMbJ\\\\\\\/Lp2FMfvvUwd8wZBuP7+FuTR9ZM07wpCCekRhtXi7N40yYV7T1gnWR\\\\\\\/xkzpMPQ\\\\\\\/FplcSdrDtqKJ\\\\\\\/MVD+6ns3ACxHCeWagsPXKDSIYez1ziqRV7ryhO\\\\\\\/3MuIil8gvVqJ8alXjs1\\\\\\\/YUJHahrR2ea8HxxTC8eNNnfRhvBmAmJfUjWdeAo36nozHump0jKY9U9L0n2005uFpSQmlY8H5j9d98iP5YEx3iDWO839gboFh9E2GNZhaw6xf9XylYwhdws5UTNYWueQZBaGJ5oXLO12EJihIucOudg5qnegfT2lb\\\\\\\/kE7a+lYXokGizPWRZ+n6MB2vdoya8MmaFgCkvhc4iFUki1nUaugBPJG+cNYCQT0hyMdbF1EFxZn9e17O05MJnJYB6MDJJyaOMxLXe8DPi0TG0g9tCqK1zoZlzw2q8QEENpa\\\\\\\/OuuGY5a6asuRmtMJter2ABjitzj2SkBmSLCE8vZ75LaTuFk3ecmkeUllii0i+pROzshKN9CsvTh90rB2av+sZcLbpLkGg5ru0kKXFl5VKFjD5FqAZIH1jvZQZRSzKZ9h9cgdObrS3giTkIQqpK34aDzHDlXVsPwkovXq2yJFtr02t2l93gFQYGpArDYs4O17MNl9++DReEFWUHowRWSnScjHjgJs+yF20cfOObaKa6hIoFNBj9DxywwS+Fh2qoS2GLreuFilaPHmQOQmBeLwR1YfGBr7QfGCTCpsEfDnY5HQPjaGZNlhrbqMGpC6vJ+pYvbrAupOO\\\\\\\/eaN1w1sBm4Y828cbYX1mYkxmidJbAkGukqmBjpmtQ9cGPB+HeBHRgofjdOWlV88DViWHKWjNOlKWgYBolKrE09PD8I0B8uO0Kyk9Qo8qq1bEf4cLguuhbCKR3M9TlOLpMkSpoLDFmTqI+lStbFgYdgfb3d051DG\\\\\\\/iaffafQCT9JZB8yFcFfJk+soCfyWH0C0bH8ULRGFFQQt4rrhYR3qDljDmTxil58+3fWZZ02Tnglo5zf0rxROm0xqlOXrguA37NgwW0x7\\\\\\\/bOU3yIsIiQj1rBgc4IMhdY6rZwGoyfpaOw2LFtukIwXfHEcY7SA2pkOtPqMtQcrBsOLrTbeJXv8k7C7iBnUuhEDLmlkF01YBILd51zAlm4DqZBX4y7n3xpysAgH\\\\\\\/gDK2X03Jy25lyc9apQxX3vXsWh7hKSjyrEUpfA6Aj3q8aMN0GyhJ+7DBQZ2c1ko1KAXoXnn8\\\\\\\/+FF+XJhyvBWtoYByr2mNrYbc7HLE8naFTpleNWUU2jKro+N+2zqE1deVfpXAqXA9\\\\\\\/MtGudn9tg\\\\\\\/FC4QsP7zJUTA2YFhbW1j8UMV0kkwoHkCE5Y20tXE\\\\\\\/uet3OtiKuFdVEvhznHnLDS3BGpj9Y6QcLceqVE5bxuOzGDbntLe+GysIkvr88ZU0Kdd4VhWsEs9lqT6eWc\\\\\\\/53KQIud5YmybegypfXKQrTTcQnxs8o9gVKAR5820LlLlYOVbs2BzCd1exEWJKurBBmn\\\\\\\/BQ3+W7UgwYqHgJaWntoZYvFg+UVT9ufk5rKDz+PUmnwtR2j1HavDANT9t8XbgRnm8NDVwMPdpsJ+pg4+b5nEiiqriA+4sVMvZptZigqT1XHEIj1ZzTg\\\\\\\/Orn1nF9pX6L+P58NAnQjT\\\\\\\/wP36pqmFNvOyyFZoanteo4j\\\\\\\/a6hZNMOzriM51mEYMV1y9xP1iz92rUaHXp\\\\\\\/zqD4+vxof7SPELJQQ6Jlu1938p3SNvbsQuJi0z0N7nRtcGbDjoGRpPuEihFILR\\\\\\\/puqFbRkbGAt30+zUgLKziMKtbBnVInEMlXcO1CyfRiKJ5rhT4plaW0VVjpeaWsDE\\\\\\\/QNL9COOhwa38XhXCD+Y051Vkd73GGNTIm2T\\\\\\\/CLOdu5GVrG7HWs7IEDzGlZNxRRmhMDbVkZUGT+D3qjidAw\\\\\\\/Ix4UBpWM0kzDiiXPxi2Aoy4l6Z2c0B646dpq1pJEvzPNcL0RTHl9JAOtQ4bKzAzbjY8l92+zA9mrsSTY+rsgTd4aLK4LFCTuBDU5iPSn\\\\\\\/TJhv3ccSDSUdhp45knRkgjYu5QILq7p\\\\\\\/RZuBN+\\\\\\\/4on2YV\\\\\\\/f4SCp4cxNgL2Enf59pX8inJ4KXDDy\\\\\\\/N1Xc87dqWor5paFNinqQxQD4ulpJ78Mo4Q+FUyMxLIypkGmV5CllSwDnq8yAURBTDNmAu8fiYnHZ44eLR2P\\\\\\\/HvnjeKUSDEHwsbZ7M4\\\\\\\/fhiwsiX9KMuB2o2dXkNrhhhQ5OGgvxFPgdR0JqXqceOzuoMkHA6SYF4oWd2vqHF6BrVQb90gnF2PnGwGSNfFahZ52uznPoC8nCbhTymNxFHdmWfW9NJcPfgehWA0r9qT\\\\\\\/q2JzBcoZxHVa8agNRnw5IwOrbOgzWod5T83HeF\\\\\\\/ur30WudP4YyjO8ytJI+KS4zcYHUVjGzF6afHbn+7w8ZMehQQnvtZsFLOa+KmxkYALkpYu9SAaKX+VMgXTmnJF2JysLjXtvdyFAlqmh6q7ZQlDw1wyPiynw9dv9jBoZzL6NU8xUKJX+kr3clyKmFGICHVu+6gEU28tRoITJW83oRE+1rwNb7dWOvbQyW7UTsXgoX3y\\\\\\\/ax2B3HNOH1E0Go6ivtnwYzixL28G0Gs+mJlpy4ai6W\\\\\\\/1bpEtZ0wNwdTshGtLK6deYO7ACDGeyC9UWX+WRqwJHw3G4K1nKb4NjfvX\\\\\\\/g7nVsFCmsxKWe9aI8k0VmSuG6on+1dfZxScndrQc4N8j0TojtZeMjRSKFlkgLdWmWnUKUtclzh1KhHR2zh1FCpEdQT7UIcbasupEAf0BSrBCgPUF7JJ3pJ9fggj+5NX7DO7d0QQsMXCQc6AFoGbtiy4qMCHIqE9mVRe+qyn6uk58UpEu5Owv5da+I27sMfCspLov8vSLXKS6qpsoc2sPZl7H7XzldGPJj6SG\\\\\\\/w+PNAVgvXUfpBB9Pwt+XpgZc\\\\\\\/LY\\\\\\\/7jot3ZQn898mCZEaKI4ojmPKXh4LSUgaHdCi7YNMNfisD3wE7lHWRUVP\\\\\\\/5tJFAVQGg9DkoO5go0y4QIruaj2\\\\\\\/ZtMfjYc94knTfQNcI9H6Mne6n3OVgyItXcmcffZjgUHbQryKJxJA6waqQ253le3aNwXa06\\\\\\\/r53khYLGw0pjRggnQw+hWhmoHu7+fiNYJ6\\\\\\\/6gTeYVPKIz5SQfGpXc1vofXYmZJI770KOATPSto76MwcnLjto0IP8\\\\\\\/1c+\\\\\\\/o2bi1raqkRjQH60drIGR+1ytEeCFmFv4Bc8+pyWuronqKEQCRpkqgg541thhQkoWWeGW9HD6fUQK1EjxDBf0va2rG4zCfD+Bq24qvUU6Nygdqehql\\\\\\\/nZHByY9LH+bnzWkBHCKIjjPf2j5WXET0Boob23fPS8Qh3zMwgouufp438oRD6lKw8qk64al4Upbclfvve2HhWd2z\\\\\\\/S8rVO3h2LW2hF95m7ueW9lDB+d6+uv9S5FwAg2rliUTlPalkBUiLDb9ZS1oSFO143\\\\\\\/rNSZlVIA0q7BLlmD8ZasJlrsXUROvgbm+9tPbt7HjBcWpHHrADyS6QBCOlpSQ\\\\\\\/gEFs3sjO55hPLclDDcj79Q0ts8B1zxYdwkPlF\\\\\\\/Ihohldv+5QJwnYSBsy\\\\\\\/w0A6ev3+GlP92spzrG5I6ggamRyXI4XYMOyC+EM6Zs97G2hbIMf6OeOPrAror0RFqwqoZh9pF9IszF0K8HhcZGi0b0bBwRFnhUshCANHl5JRlb9hNU\\\\\\\/+nL6D6icUfzqqt5Cr27mYr8SX8NncYn1PkDr5XogFhoMqn7MxxVykzQv65D4x1RNnqqTc3O2ZRUmUXUPTIcbuiZbv0fPrM4rFFQVFezySXmRdmRYMrQyhWRu6w98iG5uc4p3iJl3k\\\\\\\/SIAhcTY1O1GpKilvhXisgwEz38uY3xphhXWXjoCGp0viGMECCmf6n9CbKZyqfLopkG3wx0awlDTFd+u0ljokyYYghLUhouMTIEFsrF\\\\\\\/UDSKuGi66BMnT5JmOHpJuXmAmAg0uQtGR0wJ\\\\\\\/atNily2zp\\\\\\\/lq0c4FjpRlBhPsTGrwS9jZ4GknWby4uanOrRZzfyvR+psKT9G2IrBni+2hTuGrGwhawPHrm3l\\\\\\\/sVzp9aDFOCEfcXvr4yyYJGVeTOJucgOzFcWQhzhX2zeV3pnWfuDGfnoywhn3ID\\\\\\\/7zUnStdEDCfCepIHm4FWzKE51daqiW+nP1NOExDYneutCgkIqSDel4u4uGYulZzykjDCsUaEJRns8C4wzrrFRfK9cNPBfLI\\\\\\\/l9riVXdHsyooc5zFTNQIfVYKn2k6QqWcsWkmC4IqB42\\\\\\\/kMq5cxzdXYi21nv8kbxcDMLmn6q0Gjsg4fNXhp7PAibh30zjl3YTsohI6WAZem4kXB\\\\\\\/Qsf6\\\\\\\/qoxNSVesLrGUJ9sVzk1WdlxhRENE9ge8JGTEi+GdloKuw+VDty5\\\\\\\/2zAbAd2LqVKNUKHsbbDa1PsyC73v0gbNljPEetxoKWPPI09zSEWUHoTzgLPbVPQwgqEBsi5+koBOxPLrUe9KYP0Wf6v128ZmntkjYR4qkMVS7nuWvYxatJ+zogqOTIKYrAWbWmhqNBoP4laSbrjhRC83pyIZBdMojbPBWrjT7X+md9jlC\\\\\\\/D8MOuZYaod0FoRQRCLIBSV0+TBPz34mfplZLvcULtlSDG7O7wCBzHvZYg50o1ypE3SwmsMSXRF0BtKj2\\\\\\\/\\\\\\\/sZ2WyeyP1yX5ol1LsVE3V5aC8t77q89h6BspqAYOZrLo2p\\\\\\\/2pGMSq0y\\\\\\\/mdnkDGFMuiqVlH\\\\\\\/rGv3mVsSlmpE7BVNB1G+9y\\\\\\\/SNlFFleocn3u6fVg3Pwecn+FzeV3oaleG3c+SaUhlxgMX+IR3kop3CFVW1d8OW0JTH11bw5GAg44q1EYPXkYVzpd\\\\\\\/9bWau5CRkSeupk3nMAxcmqKSJtGpXdXjIC274Y6214xjGCfnD9rO7Nwd+6JnqVo9JrEuP7kFA6QYQVd+oL8vjTdq2x1hfrjga\\\\\\\/hXIbMPM09U6sIorxWwrSAXU+k3RZcUf8G9NocLSu+LHyc\\\\\\\/UjEB6uq8SYt9vF+3Kcquzg0zhsYX9vd1b+j8yCXI\\\\\\\/R8wSC7NeIefMhVVYwNdb129dUGlkkslP\\\\\\\/tUsia3AFVLVDBblTHKjEvtcQ1XL59aOscdZ4Dt+bwzRgG0ifc0PBGPSLQFmxGquMsyw6CbIfJQOtDviWCLy0uYZxmkmwZar4jeIV6O2Xhgj1UKJJ0o3Fiv\\\\\\\/ypVP\\\\\\\/q+DQNU2W5RSrslOCi8g0T0jzmJ8dwt6+hEPQu3sqpj7k9RcuAYv1PBwH8i\\\\\\\/JEgLeXb7OuKcIGnBypJK\\\\\\\/E0MT7K2EZPwC1Fwb2iyL+CHwo1irBqehK9QXVIXcUni1k3ozRyiDYtsoWbBGML+7K1WPNbHl+xMTHqAfLVIbzJRDXjN+Mjri6dGAX7CwC\\\\\\\/baUme83d7QQBkDjPr+1Psxo\\\\\\\/pj\\\\\\\/uphn+PUQu2v7Sytr2k9Y7MLGcw6VY2uFobJUFzISljaDG6Rjql0\\\\\\\/blRjVQAmRH3RHyhjZeT75fzSY6HXVvIciEg8rO7l4r2NPncN+KGUWPZVhc9thCWSopXM5JC5eOVb0G481if99\\\\\\\/Hu0RArwSsi4pR2N84kgu60YCEidGoJpYJ51zosxARY7pErJatJzYHwdc0aRKi1e1Btfz0GvkQNbFIe9AInuOABX0TaejgGEZKoQsE4zPlSKwRL\\\\\\\/SiE7zs5TS0qAeQlmPEtO4eo3U9QCfPp\\\\\\\/hqZ4n2rkZioIe818YbokV9USRstOx3eA3UpNlx+Y3Cu10CCHUR\\\\\\\/2A5xKha\\\\\\\/gZj\\\\\\\/LmLOtQWj52wIOnEMjQQr+ZzebyVoEsvYwQqhuAxywfOIHoWYmqhiajfbgiY1sFNYU5yqPi\\\\\\\/XIg7wIwEgS8gw818zBeqLHUU\\\\\\\/bgcr2JphxIvb086BkaJODc4jlJUq9oNMzaWN3ky9hWJ7MFShHfA+1ZYYrfAZtuoqhn+GAl7Ds3OqtwcIoS+bOW\\\\\\\/3qQr0gjh35lJJ+ZH\\\\\\\/UACoY0Eo8w52JDvRhWV9V17GHRiakK5125Zh\\\\\\\/7sjiQAHbeiqgPBUScN+Y8uR0EcuGQ2JG\\\\\\\/mw+SkOrcvak1fhHOChqYhj6YkVBSLwIaJq8WALYN4O2FJyBiujurpxy+H1oVoNWmqOgl6JWuhr7Hp050jqlznPYtu8agk70OFix4S3rRzvz1FE1rimHmLv9hv9TnwxdNUQXRzAE1gQ9\\\\\\\/lf5Kr2ulV7zNgLefU8+RTgmwSKW0a9RDLP0hueHZUKHqdIN47oYMtfTCUYEfh9Yce9wz\\\\\\\/iikTURHNH\\\\\\\/vmDzAFGCskupgePt\\\\\\\/+4KCKgMDANgIIGqxVFoFthW0Q32DgR1gW7MaX6YKkTo9qaVtPC6ODKBNFeyTDAYQLsMqCYNZR9JSUAhln\\\\\\\/zDJ\\\\\\\/jLQc+R+WUddvaN9jcAPz\\\\\\\/IdiVufskyjbQ7SCGVZF8QryqJc\\\\\\\/z3cWMn7nNKUq16tnlOTGV6XaFA5TsruJ4KTwqy1geHdLusqWk26Od44GuE1QUH7yp8ABFCF3DwEvZc+oCeZuYEfYaIv8w7Bao4Dpdc\\\\\\\/7uOGj5eLyS3P74W\\\\\\\/gNwTnN2VfP\\\\\\\/mvfnxp5SSO9lHBCNif06tNotxAoWWEixJkEJIcz1vlS3rYo7LJymT+ES28cf0154JIXbhrAvXr+5uBNgw3sijRoiiLvCwzCF8ObSJpsYxfT\\\\\\\/VF5k0VT3DL7\\\\\\\/41VH6b5G9aRGwlKpDr6Bymjr+aS3e1olseA+qRJjWTX+MBb3YpFiJ4VEjWaCFDKhZEkcz32JLPtcT3T+cvJeznb4sZRY4FIx2oiRypW4YqoWQ0kzqkY5chjD2ZBaMimuupyr8gVglDjsuVJnTMy9qVcGFpxyOrySrmOUxLufg+M\\\\\\\/CcRMHvSk\\\\\\\/\\\\\\\/oM3SyoGXUdxEN396cx9NrqRDBv6JOOkBcrF5Q7qc3Rp0g2PnFYVoihxii5fdkQ4koyhtxMK\\\\\\\/u4okXBTbERn\\\\\\\/g0D4UzL5p6RvWdUpsKN82cv+NiiVgWb7hPrtCYCUX317tuLQmahR1d\\\\\\\/V1JruMRD6Skj8hx63qGUucZ3YVw+fvUQtz\\\\\\\/UD1x+hc6px3+aFjxt66WsjwHJ+ReHZl5poBt6wrzb7AxO5TbaYSHV\\\\\\\/A5Hrlj5\\\\\\\/WiK\\\\\\\/Jx4g890jCLt3I29n\\\\\\\/FloAf1dHsN2Ze6kVlv9WO4aBhuOXKlv\\\\\\\/1SlUxtxQinyCg1kOa0yaCKgRJACvtuOEr2HZUXlXKaktyAhd2IQToWLr85dvTdR12gpiwXT91ymXnsZXO8UXVHQotEFeRqlkT3MvmodbEDsvlnb7I3IkRLun\\\\\\\/eQLXzbqg23dZDffJdA8k8uPw6+FHq10TyvDBRBezczy0aH0J1mnxCRrzNyjXjOurS\\\\\\\/ZET2LgQz9f2C1Lvc0uc8XqXodvaD2yeEFkm1dTuuVKo3CudUuc7InTQTA8BZAncJDeM1siipENP0uyxyzQpKL5I2j4ksWzG589IanasfBT5PxT\\\\\\\/PTI0uNZ03AeS3ByOJwCTjXS5dxQOqnMkDgJHzYN79HZSdZU9u8GFLK4g8e7LtMk2FyarBQpE0cr72mrAipJx23lynV8DmfGFW3LwjiMtSMoQDv13gGkvWjqgdX6jaLVU+vgMWZ09FHDBNVXz0wFdkV10CUnxZCzpnNh2yhD2kaXdC6f27vpgWgcV3IwnjzqQm5HKiaqlqG4+AesGmpXSdv38KnkoJQmu10evcTl4vzG2S4GuvxzvpwJNAFPfV0VXkiBfkaVRSAaUm4ABwXiXNpmgSvMIFsDh6wcmrG+W3fo7o1nkRKP7Cdp79gVGDsZPzJnLsWHzP5wfZpXryEfivL1RFYAEJp4vo7Pz34e\\\\\\\/kvHz8RCJf2asG4Mwxq1UUP68Tgja3UyqhcYA\\\\\\\/aODGPTArmpZfjoiGqL+tfDJaVWYyvl0AFm3IB1jpygk5sXI5DD1p5PTfcqt\\\\\\\/zdz7HNITO9RRj\\\\\\\/ohKHWxGlMF5fWyYGW20Qo0vUwoiV9PxtxznRri1viJeuDvaeHslnvdJVFCOwgtEOw\\\\\\\/L61QxhVvvEx7HZQJgVGYigMomjxAeSJ\\\\\\\/QKsXlT0h8RYNbzeYzaOPOG8VYGZ5CBh9ho2zcOtWNPpdvtJg\\\\\\\/FAp\\\\\\\/wcNNUiAd5qc9IWaJ5XTcvyaF91vjvC7IRsvneQEaOC19Xy3jMao7lz23TJeb9pRxi0G2\\\\\\\/5hd4i43TAM+VZ0fY4v9wvS731dkFTWV7ojk9hVE5RNCQA\\\\\\\/bCo72gWfetQj9SOMBArTjlwnyckjepqorpYrs2+c7HQLZMy84i+8CfW2HScClz7z9kJyCupHm5+bjYXTTRAKngOkw2cWRZAeAI2RLH\\\\\\\/YhImkLtrL0A6PZPRStxvftrouiP9boZosDCMj7mz5n33D925PW12\\\\\\\/O8ZMHMeRvy+FmKsW0XN3nvsRiffATxFfb6lESjb\\\\\\\/r88TYEUJz7G02HzCWtAIiwZ9OvPWrwEskPEhFqR1aBZ458C48SP1uCHP2hMSeuY44DGK7\\\\\\\/gLpH8MMZ77YmdjEzUXjffh+O85aIlNCjIVPhv4fOPPyNqKIQ6qhY1Qa4eHoM6lxsHoTrDlpG21rzfyzMBL3Z7p7XjLhkgWiFVjRMbx4\\\\\\\/A1G5iIjm0U81bcziGfxyLCOmhvUEHelk929qumPPLVGWF3ERMzffzM\\\\\\\/dfEaKoPw5OvVZApXduH2BBrkC7SG+BZ\\\\\\\/50SIwBJID8u+IZX583d7gBCohW8kQ6bvbRuyR8om2I2+5hume93nQH58ffOo3u80yXrIzu7zw5Dbpn8sROmZqtKwk6PdVqUyiwyXe+XIMqW\\\\\\\/QNYa\\\\\\\/tlX8PYBDPEE1oCsUZwux80jVdlSndM7xB8fvU5Ntn1bEhsYZmgnPz7mp1MrJhEt0Bm6oOjYgPCLF1OYZ\\\\\\\/CaEMyrr9BWFcd\\\\\\\/e4Ft5sZd69jPoJ3xA\\\\\\\/ZftmZtgMyz+Gqjc9MRtp5NzodqsgYgApPCBgGikNc1TRe\\\\\\\/NGzmBMNLcPjMKByni1pnVQj+561FkT8cMNNolCeFf5VBftnlW3oBjXPprYrvPB9on1ZGw1DX2lkb+1wYsesQnIm+tBSUpjyrdpFQHud5RK\\\\\\\/TMJINWhlFqLGipa\\\\\\\/mGwv43frwSqu31F2PnRibFZ2Tt38Mg5jHM8WpBsrJR7njY72fAmCsW5r\\\\\\\/8URaAhM3zUnUD5N33gX+zn7EDpY\\\\\\\/nLOOZBLIy9ZvqMVkuy3b0JptiHFU1K6UBLgBdcZGyl5oVo+HRJ6hjHHBogdORXnQJ5Guv6cf1PekFHAia0bMfMR6rrGICK2W8gCjo0QZnfJvpAjYyY141wesjIXWGPT3Aty53xXstBsOEd8Q5NSQk9rem3FMvP+niiuHtWUkyFzYvGIAmYT61uE1UID7OzOw60TB5aWYOVQQx0m1JJGk7QUam\\\\\\\/lKyiNBTbqGeUWvuGqiOSQ6dEnhomB7WCkJV\\\\\\\/AZkv369OnmSYHK9UpsEMU7RDLJu7dB3O4JnHMCuXKACpAWXuVAbFS\\\\\\\/MDqkfYO2eKHoO+IRPYjwk6UlUmILgipPiMMcjfPVkx74CkdTI4zeTesQ1iYxQXhPy0JfHz3SYHro2mTIHpgyUGco7w3uviFSXKZsnRI9HgQDJ5yiCqFOqDePXm54Sbgs8H5OVWC\\\\\\\/mhXcmBvup9doBvFspi0E6PLcAkPuxwoxq1AwjpVVHTBtlqhge3Q6Y0kEBFjKOjy2VaO0KSg6lsoUAvP50KkxnFsjaXW7LkMx55OTUN3st8C8rsTwFPWbHW4Kh0ShoiUdMasa5nq93Uea0AqR+riAVkqPbEuWOrITvjJydlCxRLB6CE2NPUZnpxCLfMmXTjyZFb0FmnpwsA5ZCAoqD4jUNgXhImvZUnpkwvcqZu1BxxgoDQ3COZ80rt\\\\\\\/JL+Mha8PgcN0Sz8lSViLX4fO8WDpuIADKc8xREscqvbpGaftGY\\\\\\\/tyo\\\\\\\/f3ALMl1HuD5jWM8PYLiu0MLKgrENK7h0chRHKqk5e5DZHp40uIIBj3V7pdTSwGfnnCoh+z5y9bgK5Fxpk\\\\\\\/a6sWIzg0QUQaAQ2WMcktNB94M5sIcH4hTJbaYcEcYl+jmOanYyQsYGz5AxTncys6I5KUf3zqAi5pQlEVPESpwb872sw02ag\\\\\\\/+rfGyyo6sLKxKcAedovs0dpkAGpMDcGqpHvFMm8d55IMT9ngxLcftL\\\\\\\/AQjOWewdoM7xVE033k+DowdG08jVcb1j7IgiSJHd6jY1b0Ycj3yK18nMGvPfV3sadAm\\\\\\\/naklxiX3ot4bquV+jBLPA6bJRscQ2TD\\\\\\\/Dd3mTCjarFAN5UEhaQk6Vz3gMtKhhUalyeaW6Zp430TBM5SIA\\\\\\\/0vrqBuTujrWNM9yrUKzm94hs53qOeOhR2yNRqnvp3O5PmAuiNMdYLUwjMqFVVfBTRUeHnfhJBhQp6BKWYOiX+51KeTBsN02t+IyY8Xf3tez962NHZ2NVzeMN3wowWuO3ReEyoc5HvomBtg+7aBSUWBuX9N7Xj6rKTh0Z9dRJG43T30aswRo\\\\\\\/TllbnwVY3K5O5FzAWqEV5wqaFT9Holm+eJxVogS7CDA2G7YGXy+XPdGbamA\\\\\\\/PgfTLxyLPDulcOtrg+qrwf4z3gb3sFOgnmnWa8ei1CER7Z3gabayRM5pPLB9U1iksIQN4GqeymwmXnX4IJOfiXxcuEm10V0PKgIr9JU8+K+RVqK4ws9\\\\\\\/NXGE1Pqa+I2DFUK0qeaaAsXda\\\\\\\/4qPi+Bf72EgGL7teSqdbK+xSavssehCMD5zK6RqXhb7XWySZjxp7xCId7k4pjiowIzuqybg5tB9t3AJ0\\\\\\\/BoInLXBNe9wav7pg0i7Zu2BmKQBCJ10IIGiyrirTg4dcZTr0\\\\\\\/5HOb2+vaWjy4mog2kW4XMfAneqjvCfK2Ow0sZ2mBRSDgg1HYWAywRtW341uIG6Cui\\\\\\\/O9zfXreaE+s8K\\\\\\\/MeNeuQZK9iupb+4aonBTHoVru3aK0ok5qq5nacIAZrv6tcSI9Udo5s9HFQC3Jn+VmWzlCBuzANkb0\\\\\\\/2m3v6eDIpgiZpycAXhzFQx0vwSkz1V7zc7hNSA3EB0PTxbjKxk8nNmtDmAmDl2iHdjghNuInIc6xG74FCs\\\\\\\/edo\\\\\\\/R6Guk3S4SBvlFuBRNyyioQqG9oCxK6t1qal0JW5W0DwwAwZhEHSRgAPU3YzRdIjwsvv3+KEBT2pUfDkPcQlXDq+8EelJ5waKup7siahrA0w8DEl7X7\\\\\\\/hqlEVmZyeT1umIIjku5xc2YVl4Lq86DitF79PKMgVT0FHsqPkQIbgxPui1NeMPQsOitF5ZMoMT9uclnj9GJlkowmCQpkLho+uOCVSqsdknxTo55mYGi\\\\\\\/D7UfJlzKXy8vai2lxZ8u1ZGhDugLJ6TCJUv5ZcrRll2RmzVDHkmEdCM7GoXJg573I2iD05jyGrTwEFxOVGJmybtuyqh7qGT4wHU0HC4szSs6poyo96LuLOMEPgdaqeMUKhxjqjeiSlYWrM+X9PAgGsE0k2k8GDZXEYEIrRdDHuYqBzDsLcNEZ2Q3uU93bSl5lnhZVC\\\\\\\/VxkiyNNgytA+hzcs+c7dfyV6HgaNJXl0gSmodiGG9YGOUrqq15Ha66p9wwjGFWpmFaCcqPlsIulCb3T4s0PlGIYvlzz5I9oE3UPXrA+MHOAfuUuybD\\\\\\\/7tD8FwvVgm1X65SQrqQukC++TRdJynaJDrm6b0\\\\\\\/eSIHgeNLIQxDVByzG5EUo\\\\\\\/+S5QCZecrYLSCYXSCiI+\\\\\\\/bGqgjrRaTxBwbAy1ZYigoyeqCHmP9aMK8U+eU1ELn\\\\\\\/vtS14AeguOoSdn3BSLxAkBYc+X3sPvX826XwNJFftWRMaHmrsjmvA0GfCaykbMUAvsz4siXmNJ1+wzFOHCNtTlpQWjCHKo64o3vl0txrHo5O3fJOVpV7c1cvLxtYNrK8LoZW6tLLOICKuhwY4LtXQlrJyDn74Dw6AthqOE+eKxxAnEbcVHrddby5e6YhVN\\\\\\\/jnEOg9uyWEfxZtiPR5VI6gNr2IfCDiTXAZlAEN40T5bQhhrs9eKRdDHAbcH5EQZEQb5cyQWR\\\\\\\/GHjX2OEC4TAtVp90wztBMh+mPbT4xoP1diQwzjeEKruuM6cAnQc3MeJkqnTdsLlujIaVZ+wox3PNvhob+noiRwRcxLxMUGPSf24WGxMxRrWrRZ2\\\\\\\/3n1+WbLyjx\\\\\\\/LqIxMBhzsXwddVTGrhhiMHh3aBlj5towBZyrW2KCJtuTtEZaVf3Foz0HFUvlHdS7sJUdOqN3pbIRfaSTMXYvIoNuC5YJymfkImPSVoszETXS8pjVkFuPNjCmLbjWv5x0wafEaAT\\\\\\\/wpN3sekv\\\\\\\/yZxx9SDHAzyfODcS\\\\\\\/BZN+DgDe2tGDbHJe+mnpN8TXf3vqDUXtOPhZ63fvw2AaIAMBocJGBXojfTtBArRcQjOxMzM9CVBsHZLfgR7rju3bnNU3WHtzBFoacVu2sUqR\\\\\\\/DhXBrHgW9CyNeqLQtlUNQqwO\\\\\\\/tuPbZaTYTHdWpL\\\\\\\/4qX0hVKleJfI8VKBfMfq7YbcgB0zTmMAIkvyU1vQJJ5RBoNTGLMQhhhX3GjlHf1\\\\\\\/2narm1lBVP9LcKn4VSy+WYQDAGVGOSsw\\\\\\\/+ZRp6M7GFvAw7n7STv8mrj+6hFqndX8c+1seHLbUJLrHpwlspEEyU5ynOkmO7Enh9nVZUTozqKTCjENK1YwgsjuMP48emXp9GKV0KkwslnNcWrGvHG4jmieRgZ1oHoVVE\\\\\\\/f24h5UNKDQtunBUbq908JODG+KaJ3p5wIzIQxKM7gDb4uXVQ8X4Y7nPocuY8WNUEYY\\\\\\\/Uid7SGe4GJsr8uWZXsllhYu7fcqkvVsYJcgv684O0uRSE9pWNryvAIg1KJkcjkktkcLH5Z82h2Clto\\\\\\\/XrqrYRCfr1ryTk9CFztMwCaJSgiNAUSDjhCNPEGWlffjwEs8nv4sHcT1znY6VNoE0Q6cLptScLfuf2+ucztq\\\\\\\/mS4CxEzjUcnUk47HxlU3JANqGYvHl4THQiLQN6NLWp3gugMeIXEoKZwDDSLXSH3GpNsw2qLdR0tHbKBlAXkKPbcmbojbcL1XCILWXaOOMx1EqvUUvzkrKasWb962G+r1AWAwtYvEW7jsdkV0ddGRG2lqimlvkaZdLds\\\\\\\/r7M\\\\\\\/LmSjQMyE3CF2VUiJiDzulRhzlUAtu7raZEyH8Az8GSox4Oc1m3o8z2VcyKCcZRJFRSQT8YvOqDxNOb4jG2AlDQW536u3YvU1DuM4z2zCijlXEqoVPR9kncSDMM82B7Jyqy4B9brpEuJ+nb2rqMYiyHl25qcgfHQeUd5p98DsXNzd\\\\\\\/PK7l+U6UjUINEZWLAbl9WSo0GwQU8+jQkKXpaEuj3cczJQUS7PFHr9U7QjHigLaPnsR9saG4EQchvLx9T3nmZ7vU1NjVCpWPe3wLLO4TZHmri1qfYOkQIqvMFU1Tvxk3WKWhu7+ijZfbl5uuTcRSuA6rUQQJmDW4839Q1t2+QEYyTCCoOWGhuzRNtQ04wDk8OmrX30GMz7ExXeZKTXAbP\\\\\\\/QSxfAuTr0Wmg\\\\\\\/grvWEEkx7oEWejkBUVmd40Vzqt4Lzql1rckGLu8FQxKuI2xMB5VYU54DZtXCcicc9w398FQXNK1qKwjmmpN41WY119MyfhySAQ0ftwHog87JIle4s1ml4Ykj24tqvMj\\\\\\\/wTer4Mg6aI1a\\\\\\\/Gauh\\\\\\\/xjRdLOh8mPKOFkrqvY0OJoyKPr5R3IRsapC5lFXYjOz9pJcI529YNFJhfySop5vUvZ9MrKs2eKH2dVAmw\\\\\\\/vdnAmK\\\\\\\/2rE3vO8A3G7KbHj35UYMPlUDhySqRkLAPh8DfMmA6gYKGGzIgWaJ7LeuaFLWWfqvHavGSxNza66eRO\\\\\\\/SeQVwcBoN2gT0DS145EUJszd1ePZwDrzDPTrnnxlb+iEjWRgu7BUyUcifYDKL81Ot8MummQgF9bQRJd4w8dJlLBkYREhElRSshChX01B1zEEWVd5Gc+46BTUdJUEBT9tmBWVjxFt6LdBAHVGGkJ+Gzs6N2LkhKC8vXSHcQEiQNstyytPQXd+W5LNXatIBReHm+55tIyPcjHHw2O\\\\\\\/azqF1YrFJWLP\\\\\\\/wFsAwzRhYMj8WLjM8xJ9V8StYWvw0\\\\\\\/3zfXSiWEFd3U04PgoGOrLqYq6Gafta+OHwLALsercUCra23KanZQYxm3vez2RjMEyxh4BSIRBGfZWuY\\\\\\\/gIMcVtnlvaj1rCNoyPNs918jGwu+iGDviwWg5ZwhSyLWkCqbQlG9WU\\\\\\\/pAokJprUuzeDcG3Lw8VbJcvi8O88jiBZ9iCsxnAByuZ9ekLVYVSiBUvMtdPwVu350Z0ibwFiKM89AApaWAVySlqL63TkMFOruQTY39i\\\\\\\/yuEIXbLQxm06cGca4FYCqVHvM7ztzJok5XRnRlXHJBBz+KuNr2+e5A4Jcg4FU1FEBr4WpoKugq\\\\\\\/ILopVs929vOSFsE0O999QP6GKFGCvkKnEvwkhSJRwlrGcb6SaPeZJmgKqo+P2xMIl1IQWkCqljUC1HAIlhSXpVhltyrKn1h\\\\\\\/HLBmG1NUk8nZNh4xD8Bx8l9WLCyCmZtPK52QhaZpNFZt7TxUWBhdUvu+sJp7cqZ4PMkrREz7seD1oaTSGc5AwcinN0I76sNJhTSIWqrFR9VYrod3zA899IsGD+JDrvVfY0XW9Y4Q1qpXMUH6lv4x\\\\\\\/0pgd+KpoDhiT1cV8+3KLNP21YT1GS3onNbRBNPmq8XUuJPuZfvdeAukik75rYkzWq5izb7AQRG6ZSO7IMHRrd1+WBN4kiCpFeYjsnxxdbyefBMsBsua0Oo481TNg\\\\\\\/Kbsrrz51D1p3uiUVqFrWN9FZIt8coPeEg5tytc\\\\\\\/H3fatV27+tOv+HyjuDexv1k5pw84xeVlMzrYfbiWvPwrcgX571ihK8AjGnXibERJMV+ULAqTeRiiw6+MfmXEgKUHI0VDRollRCFIwJr9czathVXhDUDNlsT\\\\\\\/eRo89+J7A8GXLRQ\\\\\\\/0fqLlQ0oCFdzGDqoJnH6MqVGzLfufiaI2Gkg6dtZmqQ2939EvgdmGxcI67bjiKbxiLXzt0lBKpb9pETAC393bO0ECBHaYubd6VQz8cd5Y+EEkaHJ+9YOfv5EFnbU\\\\\\\/i1ikozko2ZUA7cepBqpf\\\\\\\/HH5jLzPbHiVP8u5+t5xy983xhwlZELZjS\\\\\\\/nshydcb8DYK1cdv\\\\\\\/yJRd+CvFKurpZ3FdNRAN7GmFLBS5Z5pZ\\\\\\\/qGViHVk8avTIfqWm8mi9rQACGAUtYX4Zf+W0m48Z16jjCwEhOr3jo6UclGDSXk8aQ0GLW75o4\\\\\\\/qAH9\\\\\\\/jcCY7YLgiMCDtLrFnGPcLvF8sxzSzzZ+iUcsVduLlSuNVaM89Q1fwz5xgueJmVGxULhiFbjd5XLG1x+aKrtZ7lbhM5LNTLdadHVcIDHNyRqpij9sAG6BpWMetTpwwAnXd4joxp+1Y\\\\\\\/Zqw9BEAsZGpwmxqb7uwmK1daQBnVvsSRG5NxPcW3U2qUHnzBEEPrdCALnVrcBXFHaM6Rl\\\\\\\/371znJc+z+FkdFba8\\\\\\\/gi\\\\\\\/WsGA1P7jcaoaDJ0jsDuW7QWUjY1nIhsjYVXX7M9qNdSm44fJ0Vuc9As\\\\\\\/YZ\\\\\\\/E73Fkw\\\\\\\/7A+Ydf2AmfnTyXtV3QAWUseVpE+94CgUVdH0XDLUWUanvolKT2oHcCSQRKKoM4i7MrcnazJl1rK7a5+uNkdnyogc0uKkGvdw9IZK8m7B4kJ1BYK6T5MbwCuIZT7CXDvu58mc5Yf67kVXXgXa8ufZQcbcD75h7PZruHl72eWtPqI4F9bd2C47iSWb44F7U4b72\\\\\\\/Eoij6o2iWKjXH6WH48gvMiT6FV2jPTe\\\\\\\/zG6O1Iqh+4yOwgqBT+JC1YQHE8qscQETH7lZnqsU0sOkafV7+EgpI\\\\\\\/34MT3Wq1ZQo+99D9mg8SX8Mv4mAmLGsF72ywSnxfT6RO+4DRtKqZkkADCCqfYnrMKGr2r7ZmJ+LA0uh9ZvaOeFDVOW3tcXS+zZkwu4oy2hpps9H7xCDj\\\\\\\/OxNS9dISOsAlpjxeubDSQcSGhQLri9+WcZM\\\\\\\/kUg1+EASuf4jGUDQVgzwEawJ3k+NrCTGEhb66AWlHOqEgtLtHpkdjQxiXZ\\\\\\\/Tp81ixSQczD5yoMoCYe0y5RS2BCANQhrj9AnlFwrheAXGe\\\\\\\/GQ696QW2J+vQ5w4iEpiezZSHIkeR0IsUHkRkbgixBGVM5eu0QPiVY5g0nHpG0dD0fCk5CDWB0rH9iHgNmIHdnucr17iSsxJ7S2iX\\\\\\\/39prXGJzlcEUgy3yshmV+vp86c2SlKtakzljeaDUJ1pdM\\\\\\\/A7cKkmIjtKAfQ8aU4SJUlHbbVQ4HdhXVyqLkhNiAz+LtOvUqJ17mJMl7MLSSXggkhk4jEi1+b+EPHdXtMNY8EzwIVKf8QEwklBvslF8vfR6sHEmqhE+\\\\\\\/iFIvEPP1tV\\\\\\\/VsHK84Z9N2qQoohM0VLH2JuaeYaWHgZytdD+Jzgr\\\\\\\/bqN+kgwrVKfVE+\\\\\\\/+xXiQojx8SuxNwd8WPTuey\\\\\\\/eGBWBR6NTuevsXk4kk9e8YkcmUaoKxQBvHV+Zz9WveNkxei7xUQhpRN1AX17Jb95dp0XsLAJBf91ypF41RYlz42bhe4PTZaGmRoJ9\\\\\\\/KXf24lGFh9fZEChZ3wrDyw1XOUtPlHs+ZbuzyspK3L6wr6fUBS1\\\\\\\/MrWZVWi1l39GqWuHMQQESc9kkBFfY0R1gY4ym+uX2zdzZSfIazgKaKPJJi8Li6H6ygEcnMfsKKBEXEo2XIK++dwD6gwYTspIP5E0VFx8E9hWB8XEh2\\\\\\\/iPV8G5csiVDObUoRPxZl0ftVtxRef6WiNT8FpZ5cfrVSLaAJhrIzGajYHkIiqM1rBcoDAYnGVuOLq5\\\\\\\/G1n2dRfk2h2L+4tyWCg31047HeHVzZWqQuKGMvcO4rZLLCfgrpixTFwjO+5X1g2Pk7WdzTzqjWiw7l9E5xg9SpFi+8F2vLTDnsF+40hG5gx2D8lvZ0iVFYoB\\\\\\\/du+KjlfNRMjgdKeTWDrfhvjQzljCzwcP+dimeG+D7740SAgY7xJzYvuPfrD0xGTXtCB9W7NpZlqutFAQJkR+rtbc1LQpNfjZEFVEEVf4PB5ZbVTDGZ5sBYXkZ3uQIa+MVEcBo5lQ9YM+Ar8dHJgvvQdORnJ1VmYi+hxOEDYdRsj+\\\\\\\/mSmzS7LDqjEiZTC9sSnLVCE0VKp+cjsiUnWV\\\\\\\/qEfS74lrvOoLu7HPS4cTAbM6iiX1cBnvs1RiXmWCcqr6uGJUxec2gZLghKXpoB\\\\\\\/parLWkUdwFMJQ9hvB3Cq9pI3\\\\\\\/9SiLQ2meNi6PspT+B8DXDN5jk0dAEleQ45bOBU6c0fcZM\\\\\\\/6K46\\\\\\\/b2pI4ZGLmijg9JsfY81iczCqb8i\\\\\\\/267q7X+hdBjr0L+yk9VLNV2ZOutfzf\\\\\\\/UoF+PJWZGn1CsfyTTNUdN4\\\\\\\/Q\\\\\\\/P6\\\\\\\/8ivg+7ihWmMq+F7hxCcSlqG509NSmKbUrtJp9J7ts5adlvVKzyiOn1QgiH4hSvTfdKw2KHpBaHTzCjh5RvPHwGHlQzfqeLiKucX5BynzFoh8avuo7nSRC+DsD12Gz0R1\\\\\\\/Ye6jDNLsJwl6aQOgnr+2uXbQeR49HDS2XO7vZC1nmdEhWUJFQZnYnHGp74NsfGWPKSNRJmdKFqDkxtdeFRgFM6NCGHr11vvT9HNzBbC0Ue1i0LIypYgpxdV4JvQBMMfCPQBECENKSdF1GUhnDvdShrHx2C2yMagh8FjaeyGv6R\\\\\\\/ckoSS7csrQuxk02daGF2UH9SbZif6JrSefPE\\\\\\\/WJ\\\\\\\/LNKjDxJUxVw\\\\\\\/mXfJlahvZAEk0PyuI5O\\\\\\\/Wz8agSchmlQTv6Ow2BWvdFh7TFlQ97GNjaLI5Y8rdWayu7CPdrPfMSw3u8\\\\\\\/BiuXteT1c6VieOoDFY7HUUJINg5SvJA4psZr9kPq+iAzCyn1JH1myXGbYis4P3q0aQxBdclfF7XYBa2IEl4hMrobwZVuALhscihhE3CmmkXCYGaUT4Hmj8+zxmvF4KSvKXw9SbSXc2Cb64CKsq8lhfVfP6UbGO8DuweF+d86SnuMJOGe8vKL8WcMZUHCeRSVM9XwDDyRU9DlDlzBQfXYz9kIw7wBbTwwfB+NegaOtt1A5dbXqEQgoPRwul\\\\\\\/0DfCVYvgrMej7ejGSlgpsMIptOc1dMa6wMsktaQADO2xBoyxmwLbQz\\\\\\\/YP06cHEUFl4FcJGQH75dqW0cCETYvwlVH1Zw9k7nqH+URlelI3vsxXsBjHTVmpqYa+wiRsEIPEpWV\\\\\\\/Nd1AST+TTfqTT7BXUyrRxWVZD5KQ7z+LVmDfOA2FgUEzCh7AMSKMF9sauZvQay0UsGVdGXaN2RdGHidbFFcKN1r5K5TZfL4WT+Kq9BoF9e2mt2\\\\\\\/bgIuGqKsRVnHu7fDqmxC8durWPptISGPKBVTDaGQEvJewfs7DcfMDe5Gc\\\\\\\/F+GZQpVoufWa+7SGOwuSJatkPnJCiWDRkcHkF56qiQEifQRwOsZm5YxOKA7N49GiUZe41TWYJpc5tEGyke45s4\\\\\\\/J2\\\\\\\/r+ywuaf9V0pN6L+B951f2fOkcxcV0ivEyToVMNNeVeWc12S1FpdPq9sOkN5ucBVoGWJM\\\\\\\/I63mdbJPxsSQhfaETanJoC+YRKm7wa\\\\\\\/eulmoAaTfxTZ6LmK24TMg8A\\\\\\\/TOsj7W7QAZM766HHdE7fW6RDvHa59lRCuYgcZpQuZg9c75mM1LHLmZ449S4cn39sKkQal8Q7rdaLCPtPqqyVjU3e8c6Tui\\\\\\\/EdfHqygZaVW\\\\\\\/vtxmgzgIupyZHV\\\\\\\/ynhFsVblZCrTgSL5+M33JltNll5W1oWRY7KKJ987GJ+Nw7GISFSjBjP954L+f5u9X4qubXY3cWffIxkzoDc1qpB5s5uKQEubbgEEr8NJ\\\\\\\/PCqJfgK4kMh42oBHjSqqKgcvD6\\\\\\\/ae403DpWT4CLXK0+DFQTRIceyLILtQCIMzwb+cXMD3mSvz4O9zQh2Szgu5XaW7xT5Xm\\\\\\\/ifNfTx4Ch28ukvgddd398Dd4\\\\\\\/XXqJ12eeWzo3SumJbeN5fMOnD9KY3YRNFaEDMq7luJ9fyaxO0GFeJgSQz5c2V45NiCg5Mekd0nxMfWNImYTkySw3Eabe2PjyjGzrtougojBcd1OFX5dxoeAaN4HNp3YeuIy4VLrm36L+VzbOqX0O4bTPovZSD8qMDj5C5atAUTTrBhBKEb5Ti4cbCuaGINwrrmG5nPhlinj9ophjqizLbUEvpmcFc2AvwURrgO+wz+TQqBCzINOECmiGH4rhhKgnd+wZmvCTvIEWNGdbOD3NX1irD6E87jR0WQE0n4Visylokptf1eNzUx\\\\\\\/gim7kUIfCIo4Htc2X\\\\\\\/kc6K80aJoQH1NO2vg2qGIJa+wpjNdl4s\\\\\\\/e6nuI7PYANSoVzYNPIigLttvwVuWpsxhpiyR+XAYMrrRNV+nRgWHOap\\\\\\\/MI6qveT4fwDm78oIa3S4qZtYYViBX2zEnYooRXanpDPq44aQ4jbmaCEfsDS7zsEUMjoxUVkJ+U6Z+9gUSBxYJCBQoAnipHQs7fI9CHFoXICcZawAaiXrEYsSJswCdV9SD8hfaVEocXojPImp1m5K962QUCVbjLGQgA+JG1XLHX5xuy8bVnp4i4p0ghPI+4t5upQyt6qorPRus3LCkmh\\\\\\\/GKtBTfKod29Xda64Um\\\\\\\/mCPojgZNu1EJ0J7d3u6WM\\\\\\\/1cFpySNP6jcWpe9\\\\\\\/dFzKVfCBiqL1jcb0\\\\\\\/zBh6AqNzSBFAofzlW62PKl5gGV8cJY\\\\\\\/EBSLQ+ScgMF+YcIXt4rGEVaU3Ta0GurKKIhrnRjCg8LMorgxVhj24rLP1yAiHTSZ1WRWKL8pYEti8u5fs3ga4hz09RgT40QDF7c0BHY+UY0QYnnNBRFIf13eg1CtS3GzhO\\\\\\\/sY8ySbHmsVn9MZi2Q55+C8wrs+br+itxpGBUagFxVQqqSy0iiXxgq2Orhk27aCIqHDs5BGkdW8rKXRerTVk1zL0zlIQSsYgc5FE8+xUAIRuT5Vjl3NxCvbr2vhDfx9xG2naz00wQD0sA3wtyvAyqdabGAbOVx41NM9\\\\\\\/4tjT2SDOEXp3mDrYa4JIMAao3knpv+fkEXyoPwpDDjrwJpUHVakcpAXpznPTXTSoyb9FcExxI\\\\\\\/d9evgP9NTmUDGq\\\\\\\/Pz5AebLY8JtTqeQ39xUwA+\\\\\\\/NRShIsS2QJosL7br\\\\\\\/WS8469g8+hbxT9G1olCAN6yOfconvHFh7xL3MiWwU3Q19P0b3xR0Bribvad\\\\\\\/kaf4VJvSNvdvFSP77r4HOaSeJDgMSydFivSIxBrFn6xElQOxxns6fzcGJOLhubPv5DpI2dbKbtZuGXTi9PmoQvBbExCmmILiL0d00hE\\\\\\\/JJu9YsZ2IUk9k7uakhlWfHjzGmQ0Vt8AUwhVRtxp6wEwhZ5by3VlfM0wIbdq3Q4qifV2ZJltZsDtdi85mT1MWO30dAQDEzA1AGK7VB9CPNoxm\\\\\\\/pGiw9USFi6u+\\\\\\\/UAKEj767hzyVehKGy5PnefwdnON6bpNGIbO+FFB5HewyFfVh8N8OjKhvWANAo2Y1mzP8qKBRlPrgSw2MivKXhuVqosF8Q5f0Ozpwa9yo5yr023xNN38as1QCF7XXIH3ak\\\\\\\/NyoIlwli9Dz3wTLcxDFMF0OKCxoLz0OkCi9GNuJft0s3\\\\\\\/DWJWYeuq5UW1HqTDxp2zeV0NpmAkzXN3It86OanMLfuZKmMdx6cW3ndl\\\\\\\/wz9EQtRvOJZDk\\\\\\\/gfMGqCAFY5emynvDHqvIErPddUUdiSS6l98EqOon6rAALdPIsmyERxG\\\\\\\/waZlL4YbbSikaypvH6YZRMyEQAxxOBLDh9Veh57bE5fkYcl2SxbRV3g2wNZb2ZPF9EqOQNuiHxq7zXhfJXLxBOtcTufdFhMZivVxTM51FZGHjnYiA\\\\\\\/GjkkY0MmylA1aM1f+oX8p2Q\\\\\\\/lCcUs3hii1TYxTXaPq3ht168pLmR6YMCNmCz\\\\\\\/0Nz8tuTSvE\\\\\\\/em\\\\\\\/8SeDNXmonbQJ3AVLweVvVQVRxLvQu7+IHrNyOo7MbxL7\\\\\\\/WxqY8d4T8rfsK+Y7gxY5aZTdvVDMmnvZKX5HIYfruX39v+rYgjdk4HBIgfcJLvNZROEeCIaeppR\\\\\\\/keHqIi\\\\\\\/7UiyILNDk5HhofwXG24qN3USqB6zcVc5txyFhgIPHYNovvhQxl74axhsiqvPC9DWdIdWEjuiVdAAk+E4pNnjVWOpsrItM8xMIWtD5dxZcy0lzW647wyfI57fVy5zmZFvi\\\\\\\/CDH9u9w4X3hNkdTSobZtqb6z6FMh3uOyXcshzatbObe9sTtbnn91uJHJTTEWsOwSueE3OT0JkKQXw4d9nl2ZbYlq3CH9mPtKsp1VzNmdaSoNBUl5piCiRTnjGpLlnrsbzCqk\\\\\\\/R4++lfdswusZDB4YUO7jkWbTck0LYO6rYM5Wl5GLZgB0nilkqVFxOYZu18Ct9\\\\\\\/DAVDo5YeISLTaG0F\\\\\\\/2yR0R8QVj4HxnuVhcbV9AJnUC8Fcq0oXzdEIE4Rnlw1Yl1GlYEkAG1U9faUgyrFe43gkdlvXplyHyuvSW+hoP5TeXdV\\\\\\\/U9NKwZRRTvC5VkgZ5AusCwm3fhOXwcAbXpRi6n8eOh0tS+mt4y0LSBYDtqYbkUMbdr632FO93NaDsUFZ5fm1nHj9vM+J5VRbi5946MPKalmvVdSTuBbKpDp0canGLU61VjNcYCcDa64mHSb29fdwtVMBTHVcfXtFNF1kjiDH1ZLElShOGX8VB5hqlFueCSIauHsanaIJZYqJ6bSJB5Ye0a7o\\\\\\\/7gvzeHoUF+Actf3E1WI6hKLWrQrhoE\\\\\\\/fYBLYIpPqJn93sSlasdAWQ7lWLnXYPePhzELiDBqieE3pObV7dkO2Ocxp4jwni2dHaBAKEgLx5rhNWyEEblNhRBySwW+dwc9ui\\\\\\\/IRRFsjFFtvdOEl8ABrT\\\\\\\/ihvYoWNO34MK\\\\\\\/H53f7AwQthBMt7uK1wVAAy\\\\\\\/sUcW8Ht4OMkhO\\\\\\\/6nabRSqjqYIZLyyFah8GFlz+yxZVruKHR7xC+hBJ1rMihSvLJYFPPEDdhwGaPg5E2N6c\\\\\\\/L+OqLbuzqOwNMZzG8LxfdMCj6s8Q48cznuqJvXZcL22alGXtrsDfxZ\\\\\\\/CasjdVlZTg57pHdV03lFkGJiX9ZYkRzaKvvU9HYSYi8Oi0j4b4vZRDaMNiu2C\\\\\\\/DiaRjb3MRzpR\\\\\\\/tWAvjyCZzGQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022FbKBMU8scnWYKHul6rxQQw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225c25859eb732b1b0766493b9918d154260ea01226f2fccf06a700e54f0ae621d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SBGIQOhmc18etHnMkJHTz+SkPuUZ9KQd1ErkwEeOFQR04Au4OWprx6UUQSJ9i5yn52Fh\\\\\\\/xhHoYB6HKiCpWci+xZf2UYhsUiwzcGDt5jiYHMbWYkL1NKQ3i2netqOhC8nOUXFMBxTaRM5lAt1ZVZQkQls+A4mKfeUBu+UqKEJlwlqCxzgJQEMeyG9+4QaDyCp4tF5AT8WvBD+Nri4mcc56YuqObLCDH7kvQF+8IBgwBWKodEhzSjHtfPKGddmovg3tFrUfAsZ\\\\\\\/f62nKC94M5cA2+sNFzgnCZbHn71ERCLc5pvPyk+cjEFYjggEu4BTrWeC1I5LMNXsU9rRztKvTHiD0wAn3z54LL6i86cN+Rba0gXHKWsHqTftMoD1UjoBr5JCiq8fVIb9amzMfA1rXMGPL20L8iDVczKysNFwrqN9wOJQ+BMeXd6hXOCXzk4GqqcZpYYbNSVJE8y8pSqqfJBLCg8UxJgL+G38kqRH8mBfYIv7TloRBPMNMqncd7YWsy+KKCs4dudaa+QcgXSzKDFy\\\\\\\/Z2Lm+LQrYhjIBKVzDNCqmuBQAY2mLmGTDzsZ4GHG8c+roHS6vOpK01f6U4BlEjDy1hhekJcPu8L+Lcpp1jsa4hLRMfTri9ZSKIju9EatkBggfI04zSL8vDXRKBMHDsIXcOnoZhW1jMWTdsPFLZGTNqHBEzdF4NpwPD1ZKjlZK3NHCiiP7El4oznvuoxUKxn01x3sFjLUj+OkpTXIvtXPJpj4UO7eQnauvIx+I4g0gqXIrz887zNl3X3H+tcCQtCRl8PzfKpP0mTLYzXfNRShgHMkwWRI7EXc3cH3lnvOg0YjM\\\\\\\/D8qSYvHk5jxUN9grxYAnipzU62lS2lfXaewZS+aawRWFKe28onTAFOQKwUyNgzEZi5xQQ9EGWEUF58VtKEq8h7WP57tWkgYE1eQFYQsFIdE4R1NZUGcg5lyssYMYvHeg9UYZU+elajE9ZGA6HvDmBAsfQe2vE+ZIdnNyLnGOlVDXEYuhBerr2M9z6XDgtQ4xh+OZqOVq7RDqoFD1UjjVUy90fYfqGaWSeOokhRhMRtj6IrCtoamQtqAgPgheMDIFf4hAcfCZ+e3kewTcYKn\\\\\\\/BxIuHbC\\\\\\\/W6JQspiJyehFaoyS2ibKM74lCZ\\\\\\\/o4nXGgu1YlsC2W9waUJBGai3DAVEEHe6XOibXXP8j0IUJdmIS3dlPKSqFiDJ16k3W6\\\\\\\/mYVWb7ufw+kxBvuexyyq8vRIg7D3Nx1TZNWcOLQxo9h2KQl7Pbid+Ze7\\\\\\\/L3E8uB1NqDk+WZjS+pgidSFIOMXSTewm88Eo6KA4yiCXnFlWuNzi9s4M\\\\\\\/1YgHSil1\\\\\\\/mJVgoCEFbWsUWtddbYUv1k8tcYC1MhpTEx11u+E0EKc9CGRTTOhoSE\\\\\\\/R0Zo7cLAmg8euLlDPGLa2hSZe0dX7xXPr+yngaJ9HIuNNVkfhj1dadRaGT24\\\\\\\/oxGct25MSW2hICx1aJA8iBXVr2PGv2eSrZYmFRL0fX56zZNZGIPKRqazacAFCm4Vt0TtYakO+fAdKZCgkNWWWv7l+6moOmwShcSWiY1sfKtCjNzAIC0639wX6bO6xREto2Ls1B5YZt9ZRcjUMr+GWuSZQbayAVYv5n+3gdV6KmXKiaUQ2UuCn33HRfjGrLwzDmzr9twMTEYYFzv6IbQW2IpwvtlT4G6k7BzPYJM86R3Pzt51dzbKmi3UB26JA+2DjhUdqEtlAmSZSKKs\\\\\\\/6GLiPZCHwxdh7mhnV+ReqVfpD8F4BA9\\\\\\\/waKx4aZEWxB9\\\\\\\/v1HFmozNmVP1OKaCbRgJnyjaF16Y2Ry2OItkv+lbP\\\\\\\/Vn92tNLn2NRCNgn8pWxAuJvAigPCOYCuF3kua8qo5s59hC9SOcE2AixRyY62ZMkuJSdqOuNvaoDWTSCGYV4oxA69H51b\\\\\\\/953uHBpQKxAZ+u2ziAQ3NCHAPl0\\\\\\\/gqXX7k3DfTIpy2D0JB7NCQbSaYeo+TvBB1iipvB671QZUnLW5qBSLZdhcDiAeKlDwPEbIA9w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022+i2b91EexgOYlCEFMiuWMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a993b79c22a5616419346d32853b0a6d6e3697d0c4066736e3eb135e488fb83f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459681 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/10/1893" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qGTLyKJHOMpAZ1qTgu05woXCiE5otnp7i_MVoh9TrpA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/10/1893" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QxDIDR9qJ9hWKBhjEYw-fi1l9eF6Reb_z2bywNMZxaU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/10/1881" title="Endogenous Retrovirus Insertion in the KIT Oncogene Determines White and White spotting in Domestic Cats" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/10/1907" title="Evidence of Hermaphroditism and Sex Ratio Distortion in the Fungal Feeding Nematode Bursaphelenchus okinawaensis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459681" id="node459681" data-pisa="ggg;4/10/1893" data-pisa-master="ggg;g3.114.012542" data-apath="/ggg/4/10/1893.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptome and Complexity-Reduced, DNA-Based Identification of Intraspecies Single-Nucleotide Polymorphisms in the Polyploid <em>Gossypium hirsutum</em> L.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Qian-Hao Zhu</span>, <span class="highwire-citation-author" data-delta="1">Andrew Spriggs</span>, <span class="highwire-citation-author" data-delta="2">Jennifer M. Taylor</span>, <span class="highwire-citation-author" data-delta="3">Danny Llewellyn</span> and <span class="highwire-citation-author" data-delta="4">Iain Wilson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1893-1905;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012542 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459681" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Qian-Hao Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian-Hao%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQian-Hao%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Qianhao.Zhu@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Qianhao.Zhu@csiro.au</a>
<a href="mailto:Iain.Wilson@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Iain.Wilson@csiro.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew Spriggs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BSpriggs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spriggs%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BSpriggs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer M. Taylor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BM.%2BTaylor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taylor%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BM.%2BTaylor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Danny Llewellyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BLlewellyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Llewellyn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanny%2BLlewellyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Iain Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iain%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIain%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Qianhao.Zhu@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Qianhao.Zhu@csiro.au</a>
<a href="mailto:Iain.Wilson@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Iain.Wilson@csiro.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/10/1893" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459681" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459681/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/10/1893.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459681" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459681/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/10/1893.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459681" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459681/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/10/1893.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459681" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459681/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/10/1893.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459681" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459681/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Varietal single nucleotide polymorphisms (SNPs) are the differences within one of the two subgenomes between different tetraploid cotton varieties and have not been practically used in cotton genetics and breeding because they are difficult to identify due to low genetic diversity and very high sequence identity between homeologous genes in cotton. We have used transcriptome and restriction site−associated DNA sequencing to identify varietal SNPs among 18 <em>G. hirsutum</em> varieties based on the rationale that varietal SNPs can be more confidently called when flanked by subgenome-specific SNPs. Using transcriptome data, we successfully identified 37,413 varietal SNPs and, of these, 22,121 did not have an additional varietal SNP within their 20-bp flanking regions so can be used in most SNP genotyping assays. From restriction site−associated DNA sequencing data, we identified an additional 3090 varietal SNPs between two of the varieties. Of the 1583 successful SNP assays achieved using different genotyping platforms, 1363 were verified. Many of the SNPs behaved as dominant markers because of coamplification from homeologous loci, but the number of SNPs acting as codominant markers increased when one or more subgenome-specific SNP(s) were incorporated in their assay primers, giving them greater utility for breeding applications. A <em>G. hirsutum</em> genetic map with 1244 SNP markers was constructed covering 5557.42 centiMorgan and used to map qualitative and quantitative traits. This collection of <em>G. hirsutum</em> varietal SNPs complements existing intra-specific SNPs and provides the cotton community with a valuable marker resource applicable to genetic analyses and breeding programs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/cotton-genomics" class="hw-term hw-article-keyword hw-article-keyword-cotton-genomics" rel="nofollow">cotton genomics</a></li><li class="kwd"><a href="/keyword/qtl-mapping" class="hw-term hw-article-keyword hw-article-keyword-qtl-mapping" rel="nofollow">QTL mapping</a></li><li class="kwd"><a href="/keyword/rna-sequencing" class="hw-term hw-article-keyword hw-article-keyword-rna-sequencing" rel="nofollow">RNA-sequencing</a></li><li class="kwd"><a href="/keyword/snp-detection" class="hw-term hw-article-keyword hw-article-keyword-snp-detection" rel="nofollow">SNP detection</a></li><li class="kwd"><a href="/keyword/snp-genotyping" class="hw-term hw-article-keyword hw-article-keyword-snp-genotyping" rel="nofollow">SNP genotyping</a></li></ul><p id="p-3">Molecular markers genetically linked to agronomically important traits or genes of interest are valuable tools for increasing the efficiency of crop genetic improvement through marker-assisted selection. A number of different types of molecular markers, including restriction fragment length polymorphism, random amplified polymorphic DNA, simple sequence repeat (SSR), diversity arrays technology, amplified fragment length polymorphism, and single-nucleotide polymorphism (SNP), have been developed and used successfully in genetic studies of both simple and complex quantitative traits. SNPs are the most abundant type of molecular markers in plants, even in species that are restricted in their genetic diversity such as many of our crops (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ganal <em>et al.</em> 2009</a>). Before the widespread application of efficient next-generation sequencing (NGS) technologies, large-scale SNP discovery was time-consuming, expensive, and impractical in species without a reference genome. Within the last decade, several different NGS approaches have been applied in large-scale SNP discovery efforts in both model plant species and agriculturally important crop species. These approaches include whole-genome DNA resequencing, transcriptome or RNA sequencing (RNA-seq), complexity-reduced-DNA sequencing (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Davey <em>et al.</em> 2011</a>), and targeted sequence capture and resequencing (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mascher <em>et al.</em> 2013</a>). Among these approaches, RNA-seq, restriction site−associated DNA (RAD) sequencing (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Baird <em>et al.</em> 2008</a>), and genotyping-by-sequencing (GBS; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Poland <em>et al.</em> 2012</a>) are particularly useful in plant species lacking a reference genome (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wang <em>et al.</em> 2012a</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Yang <em>et al.</em> 2012</a>). The advances in SNP discovery using NGS together with the development of high-throughput SNP genotyping technologies have made SNPs the marker of choice in a wide range of plant studies, although their use has lagged behind in polyploid species compared with diploids. These applications of SNPs include, but are not limited to, construction of high-density genetic linkage maps for dissecting quantitative trait loci (QTL; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bancroft <em>et al.</em> 2011</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Wang <em>et al.</em> 2012a</a>), investigation of genetic diversity and population structure of crop germplasm (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chen <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cavanagh <em>et al.</em> 2013</a>), sequence assembly and genome comparison (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bancroft <em>et al.</em> 2011</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Saintenac <em>et al.</em> 2013</a>), determination of recombination breakpoints (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Huang <em>et al.</em> 2009</a>), and genome-wide association studies (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Huang <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jiao <em>et al.</em> 2012</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Riedelsheimer <em>et al.</em> 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2013</a>).</p><p id="p-4">In polyploid species, like wheat, oilseed rape, sugarcane, and cotton, SNP identification still remains a significant analytical challenge, although some progress is being made with new sequencing technologies and new bioinformatic algorithms (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kaur <em>et al.</em> 2012</a>). Tetraploid cotton is the most important fiber crop in the world; however, large-scale identification and use of SNPs in cotton remains in its infancy because of several inherent species-specific limitations. First, <em>Gossypium hirsutum</em> (Upland cotton) and <em>G. barbadense</em> (Sea Island, Pima, or Egyptian cotton), the two cotton species with the largest areas of cultivation worldwide (90% and 6% of global production, respectively) are both allotetraploids. They originated from a relatively recent (1−2 million years ago) interspecific hybridization event between an A-genome−like ancestral African diploid species similar to modern <em>G. arboreum</em> or <em>G. herbaceum</em> and a D-genome−like Central American diploid species similar to modern <em>G. raimondii</em> (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Wendel and Cronn 2003</a>). The two subgenomes (A<sub>t</sub> and D<sub>t</sub>, representing the A and D subgenome of tetraploid cotton, respectively) of tetraploid cotton have a very high (often &gt;95%) sequence conservation between homeologous genes. This means that the genic contents of the two subgenomes often are difficult to distinguish from each other in short read sequences from tetraploid plants. Second, compared with varieties among other major crop species, Upland cotton varieties, including our Australian cotton varieties, have a relatively low DNA sequence diversity. The average frequency of SNPs in Upland cotton was reported to be from less than 0.01–0.04% (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rungis <em>et al.</em> 2005</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Van Deynze <em>et al.</em> 2009</a>). This is probably because the majority of Upland cotton now grown has gone through a number of severe genetic bottlenecks, initially during domestication and then through subsequent breeding, and are now largely generated from repeated use of just a few related genetic backgrounds (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Rahman <em>et al.</em> 2002</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wendel <em>et al.</em> 2010</a>). Finally, a reference A<sub>t</sub>D<sub>t</sub> genome is not yet available, although the draft reference genomes of <em>G. raimondii</em> (D<sub>5</sub>) and <em>G. arboreum</em> (A<sub>2</sub>), the putative extant form of the contributor of the D<sub>t</sub> and A<sub>t</sub> genomes of the cultivated tetraploid cotton species, respectively, have recently been sequenced (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Paterson <em>et al.</em> 2012</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2012b</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li <em>et al.</em> 2014</a>).</p><p id="p-5">Although there have been many reports on the identification of SNPs in cotton, the majority were either interspecific SNPs or, if intraspecific SNPs, were identified from the analysis of sequence data of a single or few genes (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Small <em>et al.</em> 1999</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">An <em>et al.</em> 2007</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hsu <em>et al.</em> 2008</a>). These SNPs have had little utility in breeding because they are generally not polymorphic among intraspecific breeding populations. Large-scale identification of SNPs in <em>G. hirsutum</em> has been relatively recent, and although the CottonGen Database (<a href="https://www.cottongen.org">https://www.cottongen.org</a>) lists some 56,961 <em>G. hirsutum</em> SNPs identified from Public expressed sequence tag (EST) data, no effort was put into identifying varietal variation and most are likely to be subgenome-specific SNPs. Few, if any, of those computationally predicted SNPs have been validated or mapped. The first dedicated effort in the public domain identified SNPs by sequencing a large number of amplicons from prescreened single copy loci using the traditional Sanger sequencing approach. More than a thousand SNPs were identified from a panel of diverse <em>Gossypium</em> germplasm, but only 245 SNPs from 124 loci were found among the 16 <em>G. hirsutum</em> accessions examined (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Van Deynze <em>et al.</em> 2009</a>).</p><p id="p-6">Using a hypomethylated restriction-based genomic enrichment strategy and the 454 pyro-sequencing technology, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rai <em>et al.</em> (2013)</a> recently identified 66,364 potential SNPs (again many likely to be subgenome-specific) among six Indian <em>G. hirsutum</em> lines, but only 30 of those were selected for validation using the Sequenom platform, so their utility remains to be further verified. Using a complexity-reduced, DNA sequencing approach, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Byers <em>et al.</em> (2012)</a> found 11,834 SNPs between a commercial <em>G. hirsutum</em> variety Acala Maxxa and a wild race cotton TX2094 (<em>G. hirsutum</em> race <em>yucatenense</em>) at the extreme of the diversity within <em>G. hirsutum</em>. Only a small proportion (&lt;7%) of a subset consisting of 277 codominant markers assayed on a diverse panel of germplasm were found to be different between other domesticated varieties and Acala Maxxa, so not many of these SNPs are likely to be polymorphic between different breeding lines. A SNP genetic linkage map was constructed using 367 of the total of 1052 SNPs developed from this resource, validating the utility of the SNPs for introgressions from wilder sources of <em>G. hirsutum</em>, but not necessarily for normal intervarietal cotton breeding. <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Salmon <em>et al.</em> (2012)</a> isolated and sequenced 500 pairs of homologous genes from Acala Maxxa and TX2094 using the recently developed targeted sequence capture approach; however, only 31 varietal SNPs were added to those previously identified between these two lines. These existing efforts have provided important early gains in cotton genomics and SNP discovery; however, there remains a strong need to develop robust SNP identification methodologies that will be effective for SNP discovery among elite varieties with relatively narrow or common pedigrees and to have more genuine varietal SNPs available for maker-assisted breeding in <em>G. hirsutum</em>.</p><p id="p-7">In this study, we performed varietal SNP identification using transcriptomes of 18 <em>G. hirsutum</em> varieties and complexity-reduced, DNA sequences from two of those varieties using a novel approach and validated a significant subset (1363 SNPs) using the Sequenom or GoldenGate genotyping platform. This collection of <em>G. hirsutum</em>-specific varietal SNPs provides the cotton community with a valuable marker resource applicable to applied breeding targets and genetic analyses alike. Usage of these SNPs was demonstrated by construction of a <em>G. hirsutum</em> genetic linkage map containing 1244 SNP markers and the mapping of QTL for leaf shape, leaf trichome density, and pollen color.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials</h3><p id="p-8">In total, 18 different <em>G. hirsutum</em> varieties were used for SNP discovery in this study (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). These varieties represent the core parental germplasm used in the Australian cotton breeding program. Apart from varieties developed in Australia, some were introduced from other countries, such as the United States (<em>e.g.</em>, Coker 315), India (<em>e.g.</em>, MCU-5), and China (<em>e.g.</em>, Lumein 14). All 18 varieties were used in transcriptome-based SNP identification, whereas MCU-5 [normal leaf shape, dense leaf trichome (603 ± 117/cm<sup>2</sup>), and yellow pollen color] and Siokra 1-4 [okra leaf shape, sparse leaf trichome (56 ± 9/cm<sup>2</sup>), and creamy pollen color] were also used in RAD-based SNP identification. Seeds of these varieties were provided by the cotton breeders of CSIRO Plant Industry, Narrabri, Australia. An F<sub>7</sub> recombinant inbred line (RIL) population with 244 lines derived from MCU-5 × Siokra 1−4 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lopez-Lavalle <em>et al.</em> 2012</a>) was used in phenotyping for leaf shape, leaf trichome density and pollen color, and QTL mapping.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465040/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465040/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of RNA-sequencing results</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Callus preparation, RNA extraction, and transcriptome sequencing</h3><p id="p-10">Callus induction was performed essentially using the procedures previously described (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cousins <em>et al.</em> 1991</a>) except that the explants used were cotyledons. Total RNA was isolated from callus using the hot borate method (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wan and Wilkins 1994</a>), tested for quality (with an RNA integrity number or RIN score &gt;7) using the Bioanalyzer 2100 (Agilent Technologies), and submitted to Beijing Genomics Institute (BGI Hong Kong) for transcriptome sequencing according to their in-house protocols (RNA normalized using the Duplex-Specific thermostable nuclease enzyme). Sequencing was done using a HiSeq2000 instrument (Illumina) to generate 90-bp paired-end short reads.</p></div><div id="sec-4" class="subsection"><h3>Generation of RAD sequencing libraries</h3><p id="p-11">The RAD sequencing libraries were generated according to the procedures reported by <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Baird <em>et al.</em> (2008)</a> using <em>Eco</em>RI−, <em>Ape</em>kI−, or <em>Sbf</em>I−digested cotton genomic DNA, which was isolated from young leaves of MCU-5 and Siokra 1−4 using the DNeasy Plant Mini Kit (QIAGEN) according to the manufacturer’s instructions. Approximately 500 ng of DNA was used in digestion and ligation. To sequence the six libraries together in a single lane, Adaptor 1 with two different index sequences (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1/TableS1.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1/TableS1.pdf">Table S1</a>) was used for MCU-5 and Siokra 1−4. Single end reads with a length of 100 bp were generated using a HiSeq2000 (Illumina) at the Australian National University (Canberra, Australia).</p></div><div id="sec-5" class="subsection"><h3>SNP discovery</h3><p id="p-12">After adaptor trimming and removal of low-quality reads, RNA-seq reads were stringently aligned against the cotton D<sub>5</sub> genome (<em>G. raimondii</em>; <a href="ftp://ftp.jgi-psf.org/pub/compgen/phytozome/v9.0/Graimondii/">ftp://ftp.jgi-psf.org/pub/compgen/phytozome/v9.0/Graimondii/</a>) using Biokanga (<a href="http://www.biokanga.sourceforge.net">http://www.biokanga.sourceforge.net</a>) with the following settings: ≤4 bp of substitutions, ≤5 bp of microInDels, splice-junction detection for introns up to 1 kb in size, with 5′ and 3′ ends trimmed until edge bases matched the reference, polymerase chain reaction (PCR) differential amplification artifact reduction applied, and no indeterminate bases allowed. Only the reads with a single best unique alignment were used in the following SNP identification processes. The alignment results were written in SAM format.</p><p id="p-13">A custom C++ application, developed in-house, was used to predict bialleic SNPs based on these alignments. The program used a sliding window approach to identify regions with enough read density for SNP calling. The observed SNPs within each variety and all possible permutations of their combinations in these regions were processed in an iterative strategy to identify well-supported, subgenome SNPs. These regions with discriminated genome SNPs were then used in varietal comparisons to identify varietal SNPs. The detailed bioinformatic algorithm will be described in another publication dealing with both the methodology and implementation (A. Spriggs, S. Stephen, Q.-H. Zhu, D. Llewellyn, I. Wilson, J. M. Taylor, unpublished data).</p><p id="p-14">For the RAD sequencing data, after removing adaptor-ligated and low quality reads, we assigned the remaining reads to MCU-5 or Siokra 1−4 based on the index sequences, and the sorted reads were then further separated based on the restriction sites of <em>Eco</em>RI, <em>Ape</em>KI, and <em>Sbf</em>I. After further removing the index sequence, all reads were 3′ trimmed to a length of 78 bp. For each variety, identical short reads were collapsed into a sequence tag, and then the unique sequence tags with a read depth ≥4 (we found that sequencing errors could be effectively removed while keeping a maximum number of informative sequence tags when using this read depth threshold) from each variety were together aligned to the <em>G. raimondii</em> genome using the CLC Genomics Workbench (version 6.0.4; <a href="http://www.clcbio.com/products/clc-genomics-workbench/">http://www.clcbio.com/products/clc-genomics-workbench/</a>) with the following parameter settings: mismatch cost, 2; insertion and deletion cost, 3; length fraction, 0.5; similarity fraction, 0.95; and nonspecifically matched reads ignored.</p><p id="p-15">We found that in the majority of regions with sequence tags mapped, only four tags were aligned, two from each variety, potentially one from the A<sub>t</sub> genome, and another from the D<sub>t</sub> genome. This alignment result was then used to call the potential SNPs using the “quality-based variation detection” model implemented in the CLC Genomics Workbench with the following settings: default read quality filters (<em>i.e.</em>, neighborhood radius, 5; maximum mismatch count, 2; minimum neighborhood quality, 15; minimum central quality, 20); minimum tag coverage, 4; minimum variant frequency, 25%; maximum expected alleles, 2. The results were then filtered using coverage (4) and allele frequency (25% and 75%) to get the first set of potential varietal SNPs. The rationale for using these filters are that the region with a potential varietal SNP should be covered by two sequence tags from each variety and that the potential varietal SNP should have a 1:3 ratio. The alignment status of the filtered SNPs were manually checked to further remove false-positive results and to make sure that a putative varietal SNP is always flanked by at least one genome-specific SNP because this information could not be obtained by filtering. All SNPs reported in this paper are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1/FileS1.xlsx">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1/FileS2.xlsx">File S2</a>, and they have also been submitted to CottonGen (<a href="http://www.cottongen.org/">http://www.cottongen.org/</a>).</p></div><div id="sec-6" class="subsection"><h3>SNP validation</h3><p id="p-16">Selected putative varietal SNPs initially were analyzed by converting them to cleaved amplified polymorphic sequence (CAPS) markers. Primers were designed based on the aligned reads or the genome sequence of <em>G. raimondii</em> to amplify a ~200-bp PCR product using the varieties from which the SNP was identified. For each pair of primers, PCR conditions were optimized to amplify a single band. The PCR products were then digested with an appropriate restriction enzyme for 2−3 hr at the appropriate temperatures and fractionated on a 2% agarose gel to visualize the digestion products.</p><p id="p-17">Sequenom SNP assays were performed using the standard procedure at the Australian Genome Research Facility (Melbourne, Australia). Primers were designed based on 100-bp flanking sequences of the putative varietal SNP and aligned against the <em>G. raimondii</em> genome sequence by Blastn to confirm their uniqueness. Illumina GoldenGate SNP assays were performed by Beijing Genomics Institute (BGI Hong Kong). Then, 100-bp flanking sequences of 1652 putative varietal SNPs (from both transcriptome and complexity-reduced DNA) identified between MCU-5 and Siokra 1−4 and evenly distributed on 13 <em>G. raimondii</em> scaffolds were submitted to Illumina for marker design suitability ranking, before the selection of SNPs to assay. Because the RAD reads were only 78-bp long, <em>G. raimondii</em> genomic sequence was used to extend the flanking sequences (up to 100 bp on each side of the varietal SNP to conform to their required design pipeline). Of these SNPs, 1632 SNPs were selected for Oligo Pool Assay synthesis and used to genotype MCU-5, Siokra 1−4, and 244 F<sub>7</sub> RILs derived from MCU-5 x Siokra 1-4. Of these, 1572 and 60 had a suitability ranking score &gt;0.6 and 0.4−0.6, respectively, and 1521 had a SNP call in ≥95% of the samples analyzed and were kept for further analysis.</p></div><div id="sec-7" class="subsection"><h3>Phenotyping</h3><p id="p-18">The leaf shape of each individual plant of the F<sub>7</sub> RIL population was classified as okra, intermediate, or normal. Pollen color was classified as yellow or creamy. For leaf trichome density, three 57-mm<sup>2</sup> leaf discs were collected from the flanking area of the main vein of the eighth leaf, treated, and observed (the adaxial surface or the upper side of a leaf) as previously described (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Pomeranz <em>et al.</em> 2013</a>) using a Leica MZFL III dissector with an additional polarized light filter. Each branch of a multibranched trichome was counted as a separate trichome. Average number of the three observations from each plant was converted to number of trichome/cm<sup>2</sup> and used in QTL mapping.</p></div><div id="sec-8" class="subsection"><h3>Linkage group construction and QTL analysis</h3><p id="p-19">The software ICIM (<em>i.e.</em>, inclusive composite interval mapping) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2007</a>) was used to construct the genetic linkage map and to perform QTL mapping. The Kosambi mapping function was selected to convert a recombination frequency to genetic distance (cM). Linkage groups and marker orders were determined by using a logarithm of the odds score of 15. Only linkage groups with at least four SNP markers were kept and used in QTL mapping. QTL mapping was performed by using the ICIM-Add method of the program (ICIM) and only significant QTL above the permutation (1000 times) threshold were reported. Graphical representations were generated using MapChart (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Voorrips 2002</a>). Assignment of linkage group to A<sub>t</sub> or D<sub>t</sub> subgenome was based on comparison of sequences containing RNA-seq−derived SNPs with those of the A<sub>2</sub> genome (<em>G. arboretum</em>, our own unpublished callus transcriptome data; data not shown) and the D<sub>5</sub> reference genome (<em>G. raimondii</em>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Paterson <em>et al.</em> 2012</a>).</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Transcriptome-based SNP discovery in tetraploid cotton</h3><p id="p-20">To identify varietal SNPs in <em>G. hirsutum</em>, we sequenced individual transcriptomes of 18 <em>G. hirsutum</em> varieties using messenger RNA isolated from undifferentiated callus derived from cotyledons. This tissue was chosen because a substantial proportion of the genome is transcribed in callus while avoiding the highly expressed genes involved in photosynthesis that would otherwise dominate sequence reads from most vegetative tissues. After trimming and filtering, 45.8−52.7 M high-quality sequence reads (90-bp paired-end) from each of the individual varieties were retained for alignment, except for Sicot F-1, for which 27.2 M processed single-end reads were used in alignment (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Reads were aligned to the <em>G. raimondii</em> reference D<sub>5</sub>-genome sequence using a K-mer Adaptive Next Generation Aligner, Biokanga (<a href="http://www.biokanga.sourceforge.net">http://www.biokanga.sourceforge.net</a>), with the parameters detailed in the section <em>Materials and Methods</em>. Approximately 13.5−22.2 M (36.6–49.8%) reads originating from transcripts of both A<sub>t</sub> and D<sub>t</sub> genomes were uniquely aligned and used in SNP identification.</p><p id="p-21">As an allotetraploid, cotton has two types of SNPs: i) the more abundant subgenome-specific SNPs, which are polymorphisms between homeologous loci from the A<sub>t</sub> and D<sub>t</sub> subgenomes but mostly nonpolymorphic between varieties; and ii) varietal or allelic SNPs (also called hemi-SNPs), which are polymorphisms in only the A<sub>t</sub> or D<sub>t</sub> genome between two different varieties. Accurate identification of varietal SNPs in cotton using RNA-seq data, however, is complicated by: i) the lack of availability of transcriptome or genome reference sequences for both the subgenomes; ii) the presence of genome-specific SNPs at frequencies much higher than varietal SNPs; and iii) the possibility of unequal or differential allelic expression of homeologs from the A<sub>t</sub> and D<sub>t</sub> subgenomes, making it difficult to be certain that all alleles present have been observed for the purposes of robustly assigning a sequence difference to a varietal type.</p><p id="p-22">In a short sequence read alignment pattern looking like that shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, for example, there could be an A/G varietal SNP on the A<sub>t</sub> subgenome between variety 1 and 2, but only if there is definitely a G in this position in both the A<sub>t</sub> and D<sub>t</sub> alleles of variety 2. Alternatively, the nucleotide G detected in variety 2 may simply reflect the sequencing depth being by chance insufficient to detect the A<sub>t</sub> allele or the D<sub>t</sub> allele because of their differential expression and/or technical issues related to library preparation and sequencing. These two scenarios cannot be distinguished with any great confidence, unless the sequencing depth is very large, and would otherwise result in a very high false-positive rate for varietal SNP calls. If there is a genome-specific SNP (<em>e.g.</em>, A/T in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>), however, observed in the flanking region of the putative varietal SNP (A/G in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>), it can be used as a guide to resolve the reads from coexpressed homeologs from the A<sub>t</sub> and D<sub>t</sub> subgenomes and more confidently call the adjacent A/G as a varietal SNP present in the A<sub>t</sub> genome.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Varietal single-nucleotide polymorphisms (SNPs) can be more confidently called when it is flanked by a subgenome-specific SNP. (A) The sequence alignment pattern of two varieties when there is a putative varietal SNP (A/G) located on the At genome but with no subgenome-specific SNP nearby to help assign reads to their correct subgenome. In this case, the consensus sequence of variety 2 could be contributed by reads from both the At and Dt subgenomes with a G at the SNP position or just from one subgenome as a result of differential expression of homeologs or even by chance due to low sequence depth. It is therefore difficult to determine with any confidence whether or not A/G is a true varietal SNP. (B) The sequence alignment pattern of two varieties when there is a subgenome-specific SNP (A/T) flanking the putative varietal SNP (A/G). In this case, A/G is a quite confidently called varietal SNP due to presence of both the At and Dt alleles in sequence reads from both varieties. (C) A/G is a simple SNP and acts as a codominant marker in both homeologs. (D) A/T of the At genome is a candidate codominant SNP marker because of the presence of the flanking subgenome-specific SNPs. D5 represents G. raimondii genome sequence. Bars represent genomic DNA or consensus cDNA sequences derived from RNA-seq reads." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-447823158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Varietal single-nucleotide polymorphisms (SNPs) can be more confidently called when it is flanked by a subgenome-specific SNP. (A) The sequence alignment pattern of two varieties when there is a putative varietal SNP (A/G) located on the A&lt;sub&gt;t&lt;/sub&gt; genome but with no subgenome-specific SNP nearby to help assign reads to their correct subgenome. In this case, the consensus sequence of variety 2 could be contributed by reads from both the A&lt;sub&gt;t&lt;/sub&gt; and D&lt;sub&gt;t&lt;/sub&gt; subgenomes with a G at the SNP position or just from one subgenome as a result of differential expression of homeologs or even by chance due to low sequence depth. It is therefore difficult to determine with any confidence whether or not A/G is a true varietal SNP. (B) The sequence alignment pattern of two varieties when there is a subgenome-specific SNP (A/T) flanking the putative varietal SNP (A/G). In this case, A/G is a quite confidently called varietal SNP due to presence of both the A&lt;sub&gt;t&lt;/sub&gt; and D&lt;sub&gt;t&lt;/sub&gt; alleles in sequence reads from both varieties. (C) A/G is a simple SNP and acts as a codominant marker in both homeologs. (D) A/T of the A&lt;sub&gt;t&lt;/sub&gt; genome is a candidate codominant SNP marker because of the presence of the flanking subgenome-specific SNPs. D&lt;sub&gt;5&lt;/sub&gt; represents &lt;em&gt;G. raimondii&lt;/em&gt; genome sequence. Bars represent genomic DNA or consensus cDNA sequences derived from RNA-seq reads.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1893/F1.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/10/1893/F1.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465052" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Varietal single-nucleotide polymorphisms (SNPs) can be more confidently called when it is flanked by a subgenome-specific SNP. (A) The sequence alignment pattern of two varieties when there is a putative varietal SNP (A/G) located on the A<sub>t</sub> genome but with no subgenome-specific SNP nearby to help assign reads to their correct subgenome. In this case, the consensus sequence of variety 2 could be contributed by reads from both the A<sub>t</sub> and D<sub>t</sub> subgenomes with a G at the SNP position or just from one subgenome as a result of differential expression of homeologs or even by chance due to low sequence depth. It is therefore difficult to determine with any confidence whether or not A/G is a true varietal SNP. (B) The sequence alignment pattern of two varieties when there is a subgenome-specific SNP (A/T) flanking the putative varietal SNP (A/G). In this case, A/G is a quite confidently called varietal SNP due to presence of both the A<sub>t</sub> and D<sub>t</sub> alleles in sequence reads from both varieties. (C) A/G is a simple SNP and acts as a codominant marker in both homeologs. (D) A/T of the A<sub>t</sub> genome is a candidate codominant SNP marker because of the presence of the flanking subgenome-specific SNPs. D<sub>5</sub> represents <em>G. raimondii</em> genome sequence. Bars represent genomic DNA or consensus cDNA sequences derived from RNA-seq reads.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Using this rationale, we deployed an analytical approach to identify only the varietal SNPs among our RNA-seq data that were flanked by at least one subgenome-specific SNP. The approach was first tested using the short reads from four varieties (MCU-5, Siokra 1−4, DeltaOpal and Sicot 70), and this identified 4894 varietal SNPs. Ten of the predicted varietal SNPs and 10 equivocal SNPs without a supporting adjacent subgenome-specific SNP were selected for validation by converting them to CAPS markers. Of the 10 predicted varietal SNPs with flanking subgenome-specific SNP(s), 7 were confirmed to be polymorphic. Two such examples are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. In contrast, of the 10 equivocal SNPs, a polymorphism was confirmed in only one case, supporting our hypothesis. We then extended the analysis to call varietal SNPs by parallel processing the RNA-seq data from all 18 varieties. In total, 37,413 nonredundant varietal SNPs were identified among these <em>G. hirsutum</em> varieties (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1/FileS1.xlsx">File S1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Verification of transcriptome-based predicted varietal SNPs by the CAPS method. The subgenome-specific SNPs are shown in pink and green, and the varietal SNPs are shown in red and blue. The restriction sites used for cleavage of the generated polymerase chain reaction fragments are underlined. The numbers in parentheses after the variety names represent the number of RNA-seq reads with identical sequences to that shown. Lanes 1 and 2 of the agarose gel represent DeltaOpal and Sicot 70, respectively. DNA size markers are indicated in bp. Sub-genome designations (At and Dt) are inferred by comparison to G. raimondii and G. arboreum sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-447823158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Verification of transcriptome-based predicted varietal SNPs by the CAPS method. The subgenome-specific SNPs are shown in pink and green, and the varietal SNPs are shown in red and blue. The restriction sites used for cleavage of the generated polymerase chain reaction fragments are underlined. The numbers in parentheses after the variety names represent the number of RNA-seq reads with identical sequences to that shown. Lanes 1 and 2 of the agarose gel represent DeltaOpal and Sicot 70, respectively. DNA size markers are indicated in bp. Sub-genome designations (A&lt;sub&gt;t&lt;/sub&gt; and D&lt;sub&gt;t&lt;/sub&gt;) are inferred by comparison to &lt;em&gt;G. raimondii&lt;/em&gt; and &lt;em&gt;G. arboreum&lt;/em&gt; sequences.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1893/F2.medium.gif" width="440" height="146"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/10/1893/F2.medium.gif" width="440" height="146"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465156" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Verification of transcriptome-based predicted varietal SNPs by the CAPS method. The subgenome-specific SNPs are shown in pink and green, and the varietal SNPs are shown in red and blue. The restriction sites used for cleavage of the generated polymerase chain reaction fragments are underlined. The numbers in parentheses after the variety names represent the number of RNA-seq reads with identical sequences to that shown. Lanes 1 and 2 of the agarose gel represent DeltaOpal and Sicot 70, respectively. DNA size markers are indicated in bp. Sub-genome designations (A<sub>t</sub> and D<sub>t</sub>) are inferred by comparison to <em>G. raimondii</em> and <em>G. arboreum</em> sequences.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Use of complexity reduced genomic DNA in SNP discovery</h3><p id="p-26">Transcriptome-based varietal SNPs are limited to expressed regions of the genome that are likely to be less polymorphic as they are often constrained by purifying selection. To identify SNPs located within nontranscribed and intronic regions and to test the feasibility of using DNA instead of RNA of tetraploid cotton in SNP identification, we created a RAD sequencing library using genomic DNA isolated from MCU-5 and Siokra 1−4 and digested with <em>Eco</em>RI, <em>Ape</em>KI, or <em>Sbf</em>I. In total, 95.5 M single-end reads (100 bp in length) were generated. After adaptor trimming and removal of low-quality reads, the remaining reads were separated based on the index sequences and restriction sites into seven groups (MCU-5-<em>Eco</em>RI, MCU-5-<em>Ape</em>KI, MCU-5-<em>Sbf</em>I, Siokra 1-4-<em>Eco</em>RI, Siokra 1-4-<em>Ape</em>KI, Siokra 1-4-<em>Sbf</em>I, and a set of others that lacked the index sequence and/or restriction site and so were discarded). Of the reads (76.7 M) with both index and restriction-site information, the majority (95.3%) were from <em>Eco</em>RI−digested DNA, with only 2.2 M (2.8%) and 1.4 M (1.9%) from <em>Ape</em>KI− and <em>Sbf</em>I−digested DNA, respectively. This was probably because <em>Ape</em>KI and <em>Sbf</em>I did not digest cotton DNA well in our hands and <em>Sbf</em>I is a rare cutter restriction enzyme (8-bp cutter). In the following analysis, we used only reads from <em>Eco</em>RI digested DNA.</p><p id="p-27">We used the CLC Genomics Workbench to identify putative varietal SNPs in our RAD sequencing data. We investigated different combinations of the input sequence format and variation detection modules and found that potential varietal SNPs between two tetraploid cotton varieties could be quite accurately called using a nonredundant read set and the “quality-based variation detection” module by following the criteria detailed in the section <em>Materials and Methods</em>. An example is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, where only two types of tags per genotype could be aligned to the <em>G. raimondii</em> genome on both sides of the <em>Eco</em>RI (GAATTC) restriction site. These tags from each genotype are distinguished from each other by the presence of subgenome-specific SNPs on the same sequence reads. The varietal SNP (C/T) shown has a 1:3 ratio among the four tags. Based on this type of alignment pattern and the filters described in Materials and Methods, we identified an additional 3,090 varietal SNPs between MCU-5 and Siokra 1-4 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1/FileS2.xlsx">File S2</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Use of the CLC Genomics Workbench in the identification of restriction-site associated DNA (RAD)-based single-nucleotide polymorphisms (SNPs). A screen shot showing RAD sequence tags aligned to the genome sequence of G. raimondii. The EcoRI restriction site used to generate the sequencing library is boxed. A varietal SNP (at position Chr01-824043) can be quite confidently called due to the presence of a subgenome-specific SNP (at position Chr01-824049) in the same reads." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-447823158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Use of the CLC Genomics Workbench in the identification of restriction-site associated DNA (RAD)-based single-nucleotide polymorphisms (SNPs). A screen shot showing RAD sequence tags aligned to the genome sequence of &lt;em&gt;G. raimondii&lt;/em&gt;. The &lt;em&gt;Eco&lt;/em&gt;RI restriction site used to generate the sequencing library is boxed. A varietal SNP (at position Chr01-824043) can be quite confidently called due to the presence of a subgenome-specific SNP (at position Chr01-824049) in the same reads.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1893/F3.medium.gif" width="440" height="118"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/10/1893/F3.medium.gif" width="440" height="118"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465031" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">Use of the CLC Genomics Workbench in the identification of restriction-site associated DNA (RAD)-based single-nucleotide polymorphisms (SNPs). A screen shot showing RAD sequence tags aligned to the genome sequence of <em>G. raimondii</em>. The <em>Eco</em>RI restriction site used to generate the sequencing library is boxed. A varietal SNP (at position Chr01-824043) can be quite confidently called due to the presence of a subgenome-specific SNP (at position Chr01-824049) in the same reads.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Few varietal SNPs from transcriptome and RAD sequencing are overlapping</h3><p id="p-29">The varietal SNPs identified in this study were distributed across all <em>G. raimondii</em> chromosomes with an average density of 53 SNPs/Mb. Of the 13 chromosomes, Chr02 and Chr09 had the lowest (42 SNPs/Mb) and highest (78 SNPs/Mb) SNP density, respectively (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). To examine the overlap in SNP calls between the transcriptome and RAD sequencing approaches, we used Blast (E value ≤1.0e-50) to align the flanking sequence (100 bp on each side) of all varietal SNPs identified by either approach between MCU-5 and Siokra 1−4 against the full set of predicted transcripts of <em>G. raimondii</em> (<a href="ftp://ftp.jgi-psf.org/pub/compgen/phytozome/v9.0/Graimondii/">ftp://ftp.jgi-psf.org/pub/compgen/phytozome/v9.0/Graimondii/</a>). Flanking sequences of the transcriptome- and RAD-derived varietal SNPs matched <em>G. raimondii</em> transcripts in 92.2% and 19.9% of cases, respectively. Only 43 (or 1.4%) of 3090 RAD-derived SNPs were overlapping with the transcriptome-derived SNPs. In addition, gene density was highly correlated with the density of transcriptome-derived SNPs but not with the density of RAD-derived SNPs that had a much more even distribution within each chromosome than transcriptome-derived SNPs (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). These results suggest that the RAD-derived varietal SNPs were mainly from nontranscribed regions and that these two approaches are complementary because they target different genomic features and that an even distribution of SNPs across the whole genome would be achieved by using both approaches.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465043/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465043/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Distribution of SNPs across all chromosomes of <em>G. raimondii</em></span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SNP and gene density across the 13 G. raimondii chromosomes. The orange peaks represent SNP density, which were generated by counting the number of SNPs in 50-kb sliding windows with a 25-kb overlapping region. Gene density in the same sliding windows was shown by blue peaks and mirrored on a single image. The green lines represent the positions of chromosome starts and/or ends as the graph shows all chromosomes joined together one after another. (A) RNA-seq&#x2212;derived SNPs. (B) Restriction-site associated DNA&#x2212;derived SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-447823158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SNP and gene density across the 13 &lt;em&gt;G. raimondii&lt;/em&gt; chromosomes. The orange peaks represent SNP density, which were generated by counting the number of SNPs in 50-kb sliding windows with a 25-kb overlapping region. Gene density in the same sliding windows was shown by blue peaks and mirrored on a single image. The green lines represent the positions of chromosome starts and/or ends as the graph shows all chromosomes joined together one after another. (A) RNA-seq&#x2212;derived SNPs. (B) Restriction-site associated DNA&#x2212;derived SNPs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1893/F4.medium.gif" width="440" height="288"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/10/1893/F4.medium.gif" width="440" height="288"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465058" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">SNP and gene density across the 13 <em>G. raimondii</em> chromosomes. The orange peaks represent SNP density, which were generated by counting the number of SNPs in 50-kb sliding windows with a 25-kb overlapping region. Gene density in the same sliding windows was shown by blue peaks and mirrored on a single image. The green lines represent the positions of chromosome starts and/or ends as the graph shows all chromosomes joined together one after another. (A) RNA-seq−derived SNPs. (B) Restriction-site associated DNA−derived SNPs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Experimental validation of varietal SNPs</h3><p id="p-32">Some of the initial transcriptome based varietal SNPs were verified by conversion to CAPS markers as indicated previously. We extended the validation to two different high-throughput SNP genotyping platforms to verify a larger number of the predicted SNPs (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). First, 65 transcriptome-derived varietal SNPs identified between Sicot 70 and DeltaOpal were analyzed using the Sequenom platform. Of the 62 that were successfully amplified, 45 (72.6%) were verified to be polymorphic between the two varieties. Second, 513 RAD-derived and 1119 transcriptome-derived varietal SNPs identified between MCU-5 and Siokra 1−4 were genotyped using the GoldenGate platform. A total of 467 RAD-derived and 1054 transcriptome-derived SNPs were successfully genotyped, and of these, 351 (75.2%) and 967 (91.7%), respectively, were confirmed to be polymorphic between the two varieties. These results suggest that a functional SNP assay can be designed for the majority of the varietal SNPs we identified. Although the SNPs used in validation were based on analysis of MCU-5 and Siokra 1−4 (<em>G. hirsutum</em>), 689 (52.3%) of the verified SNPs were also found to be polymorphic between two standard varieties (TM-1: <em>G. hirsutum</em>; 3−79: <em>G. barbadense</em>) commonly used in cotton genetics and generation of a number of mapping populations, suggesting that our SNPs should be useful more broadly in both cotton genetics and breeding.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465075/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465075/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Validation of SNPs</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Use of subgenome-specific SNPs in assay primers to improve the frequency of SNPs behaving as codominant SNP assays</h3><p id="p-34">SNP genotype calling of most currently available SNP genotyping platforms, such as GoldenGate, kompetitive allele-specific PCR <em>(</em>i.e., KASP), and Sequenom, were developed primarily for diploid species. For polyploid species such as cotton, these platforms can be problematic when the SNP assay amplifies both homeologous (A<sub>t</sub> and D<sub>t</sub>) copies of a locus, <em>i.e.</em>, they act as “dominant” SNP markers and are unable to discriminate between a heterozygous plant and one that is homozygous for one of the two possible alleles at that locus being assayed, such as Genotype 1 in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>. All genotyping platforms work well when the SNP assay amplifies only the homeologous copy carrying the SNP, <em>i.e.</em>, they act as codominant SNP assays as they would in a diploid species (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, B and C</a>). Codominant SNP assays should be more useful in cotton breeding programs where it is essential to be able to identify individuals carrying all the homozygous alleles of interest in segregating populations.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Depending on genotyping platforms and sequence context, varietal SNP markers can act as dominant or codominant SNP assays. (A) When subgenome-specific SNPs are not present in the allele-specific primers, both the At and the Dt subgenomes can be amplified and the varietal SNP assay is dominant, i.e., it is unable to distinguish heterozygotes from homozygotes for one of the genotypes (Genotype 1 in this case for a Sequenom assay). (B) When a subgenome-specific SNP is present in the allele-specific primers, only the subgenome (the At subgenome in this case) with the subgenome-specific SNP is amplified; therefore, the varietal SNP assay is codominant, i.e., heterozygotes can be separated from homozygotes in this kompetitive allele-specific PCR assay. (C) A codominant varietal SNP assay (T/C in this case) further enhanced by the presence of multiple subgenome-specific SNPs in the allele-specific primers in this Sequenom assay." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-447823158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Depending on genotyping platforms and sequence context, varietal SNP markers can act as dominant or codominant SNP assays. (A) When subgenome-specific SNPs are not present in the allele-specific primers, both the A&lt;sub&gt;t&lt;/sub&gt; and the D&lt;sub&gt;t&lt;/sub&gt; subgenomes can be amplified and the varietal SNP assay is dominant, &lt;em&gt;i.e.&lt;/em&gt;, it is unable to distinguish heterozygotes from homozygotes for one of the genotypes (Genotype 1 in this case for a Sequenom assay). (B) When a subgenome-specific SNP is present in the allele-specific primers, only the subgenome (the A&lt;sub&gt;t&lt;/sub&gt; subgenome in this case) with the subgenome-specific SNP is amplified; therefore, the varietal SNP assay is codominant, &lt;em&gt;i.e.&lt;/em&gt;, heterozygotes can be separated from homozygotes in this kompetitive allele-specific PCR assay. (C) A codominant varietal SNP assay (T/C in this case) further enhanced by the presence of multiple subgenome-specific SNPs in the allele-specific primers in this Sequenom assay.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1893/F5.medium.gif" width="440" height="190"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/10/1893/F5.medium.gif" width="440" height="190"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464923" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Depending on genotyping platforms and sequence context, varietal SNP markers can act as dominant or codominant SNP assays. (A) When subgenome-specific SNPs are not present in the allele-specific primers, both the A<sub>t</sub> and the D<sub>t</sub> subgenomes can be amplified and the varietal SNP assay is dominant, <em>i.e.</em>, it is unable to distinguish heterozygotes from homozygotes for one of the genotypes (Genotype 1 in this case for a Sequenom assay). (B) When a subgenome-specific SNP is present in the allele-specific primers, only the subgenome (the A<sub>t</sub> subgenome in this case) with the subgenome-specific SNP is amplified; therefore, the varietal SNP assay is codominant, <em>i.e.</em>, heterozygotes can be separated from homozygotes in this kompetitive allele-specific PCR assay. (C) A codominant varietal SNP assay (T/C in this case) further enhanced by the presence of multiple subgenome-specific SNPs in the allele-specific primers in this Sequenom assay.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">There are theoretically two types of effective codominant SNP markers in a tetraploid using current genotyping platforms based on amplification of specific alleles with PCR: one as shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>, where the polymorphic SNP (A/G) between two varieties is a simple SNP identical in each subgenome, but different between varieties, and the other as shown in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>, where the polymorphic SNP (A/T) is only within one subgenome, in this case the A<sub>t</sub> subgenome. In our pool of predicted varietal SNPs, only ~3% were of these types that should behave as codominant SNP assays, the rest potentially behaving as dominant SNP assays because the other subgenome homeologs carry the same nucleotide at the SNP position as one of the alleles (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>). If one or more subgenome-specific SNP(s) were present in the allele-specific and/or the universal primers used in a SNP assay, then a putative “dominant” varietal SNP marker with a sequence alignment pattern as shown in <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1B</a> might behave like a true co-dominant SNP marker (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>) due to the destabilization of primer binding to one of the homeologs. As our varietal SNPs were identified based on the presence of flanking genome-specific SNP(s), some of them would have such genome-specific SNP(s) overlapping with the assay primers, so a portion of these SNPs would, in practice, function as co-dominant SNP markers. Consistent with this expectation, we found that 30.5% and 43.4% of transcriptome- and RAD-based SNPs, respectively, were effectively codominant SNP markers in the GoldenGate SNP genotyping assay.</p><p id="p-37">To investigate the effectiveness of subgenome-specific SNP(s)-containing primers on the conversion of dominant SNP assays to co-dominant SNP assays, we analyzed the number and positions of subgenome-specific SNP(s) in both allele-specific and universal primers for the subset of 258 RAD-based SNPs that had their primers designed based on the 78-bp long RAD reads. The relationship between the number of subgenome-specific SNP(s) in the SNP assay primers and the percentage of assays behaving as co-dominant SNP markers are shown in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. All 18 SNPs without subgenome-specific SNP in both the allele-specific and the universal primers behaved as dominant SNP markers, whereas all four SNPs with four subgenome-specific SNPs in either the allele-specific primer or the universal primer behaved as true codominant SNP markers. All four SNPs with three subgenome-specific SNPs in both the allele-specific and the universal primer also behaved as codominant SNP markers. Generally, although the numbers are small, the percentage of codominant SNP assays increased with an increasing number of subgenome-specific SNPs in the assay primers, irrespective of whether it was the allele-specific or the universal primer (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). In addition, 30 codominant SNP assays and 77 dominant SNP assays had one genome-specific SNP in the allele-specific primer, the universal primer or both, and 50.0% (15/30) and 36.4% (28/77) of these SNPs had their subgenome-specific SNP within the 5-bp region of the 3′ end of the primers (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). These results suggest that the number of genome-specific SNP(s) in the assay primers is positively correlated with the probability of a SNP assay performing as a codominant assay, and that a genome-specific SNP closer to the 3′ end of the assay primers is more useful for ensuring this behavior, but are not always effective.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465006/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465006/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Effect of the number of subgenome-specific SNPs on the codominant behavior of SNP assay</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464963/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464963/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Distribution of single subgenome-specific SNP in assay primers of varietal SNPs behaving as codominant and dominant assays</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">Because most current available SNP genotyping platforms prefer that there be no additional varietal SNP(s) within the 20-bp flanking regions of the targeted nucleotide being assayed, we separated our transcriptome-derived varietal SNPs into three types: type I (15,991), with no additional varietal SNP(s) within the 100-bp regions either side of a varietal SNP; type II (6130), with one or more varietal SNPs within the 21-100-bp flanking regions of a varietal SNP; and type III (15,292), with additional varietal SNP(s) within the 20-bp flanking regions of a varietal SNP (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1/FileS1.xlsx">File S1</a>). Given a large enough number of SNPs to choose from, it should be possible to bias the selection of SNPs to those that will behave as co-dominant assays by choosing those that have flanking genome-specific SNPs within 20 bp.</p></div><div id="sec-15" class="subsection"><h3>Construction of genetic linkage map and QTL mapping</h3><p id="p-43">A <em>G. hirsutum</em> genetic linkage map with 1244 SNP markers (not including 25 redundant SNP markers) was constructed based on an F<sub>7</sub> RIL population derived from MCU-5 × Siokra 1−4 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1/FileS3.pdf">File S3</a>). This map contains 54 linkage groups with a total genetic distance of 5557.42 cM. These linkage groups were first assigned to a corresponding <em>G. raimondii</em> chromosome based on the localization of the majority of SNPs within the linkage group and then assigned to A<sub>t</sub> or D<sub>t</sub> subgenome based on the origin (A<sub>2</sub> or D<sub>5</sub>) of the majority of the RNA-seq derived SNPs of each linkage group. This was determined by comparison of SNP containing reads with the transcriptome data from the A<sub>2</sub> genome (<em>G. arboretum</em>; our own unpublished data) and the published <em>G. raimondii</em> (D<sub>5</sub>) genome sequence (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Paterson <em>et al.</em> 2012</a>). Of the 914 RNA-seq derived SNPs, for which a subgenome (A<sub>t</sub> or D<sub>t</sub>) origin was bioinformatically determined, 905 (99%) were correctly mapped to a corresponding A<sub>t</sub> or D<sub>t</sub> linkage group. Of the 1244 SNP markers, 526 (42.28%) and 718 (57.72%) were mapped to the A<sub>t</sub> and D<sub>t</sub> linkage groups, respectively. Each of the 26 chromosomes of <em>G. hirsutum</em> was represented by 1−4 linkage groups. Generally, colinearity of the SNP markers between <em>G. raimondii</em> and <em>G. hirsutum</em> was observed for the majority of linkage groups although minor intrachromosomal inversions existed in some of the linkage groups. In total, 80 SNP markers (6.43%) did not group with their corresponding chromosome (those highlighted in pink in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1/FileS3.pdf">File S3</a>). For example, SNP marker Chr10_48842108 was mapped to <em>G. raimondii</em> Chr02 [C15(D<sub>t</sub>)_LG04], and SNP markers Chr09_32106785, Chr09_28064206, and Chr09_29035534 were mapped to <em>G. raimondii</em> Chr12 [C04(A<sub>t</sub>)_LG15] (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6, A and D</a>). Some of these could be artifacts of the mapping, but some could be due to chromosome rearrangement. For example, a number of SNP markers from <em>G. raimondii</em> Chr03 were always grouped with those from <em>G. raimondii</em> Chr05 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1/FileS3.pdf">File S3</a>), which is a result of chromosome arm translocation between C02(A<sub>t</sub>) (<em>G. raimondii</em> Chr05) and C03(A<sub>t</sub>) (<em>G. raimondii</em> Chr03) in <em>G. hirsutum</em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wang <em>et al.</em> 2013</a>). Another potential chromosome arm translocation in <em>G. hirsutum</em> was between C04(A<sub>t</sub>) and C05(A<sub>t</sub>) (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Wang <em>et al.</em> 2013</a>). We found that this translocation could in fact be between their counterparts in the D<sub>t</sub> subgenome, <em>i.e.</em>, between C19 (<em>G. raimondii</em> Chr09) and C22 (<em>G. raimondii</em> Chr12) because all RNA-seq derived SNP markers mapped to <em>G. raimondii</em> Chr09 and Chr12 in LG16 were from the D<sub>t</sub> subgenome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1/FileS3.pdf">File S3</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage groups with mapped QTL. For each chromosome, only the linkage group (LG) with a QTL mapped was shown. Each linkage group was named by the chromosome number of G. hirsutum, the corresponding subgenome (At or Dt), and LG in this study. SNPs were named based on the chromosome number of G. raimondii followed by their alignment coordinates on the corresponding G. raimondii chromosome. SNP marker(s) with its origin different from the majority SNPs that defined the linkage group are shown in pink. The QTL regions were indicated by gray rectangles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-447823158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage groups with mapped QTL. For each chromosome, only the linkage group (LG) with a QTL mapped was shown. Each linkage group was named by the chromosome number of &lt;em&gt;G. hirsutum&lt;/em&gt;, the corresponding subgenome (A&lt;sub&gt;t&lt;/sub&gt; or D&lt;sub&gt;t&lt;/sub&gt;), and LG in this study. SNPs were named based on the chromosome number of &lt;em&gt;G. raimondii&lt;/em&gt; followed by their alignment coordinates on the corresponding &lt;em&gt;G. raimondii&lt;/em&gt; chromosome. SNP marker(s) with its origin different from the majority SNPs that defined the linkage group are shown in pink. The QTL regions were indicated by gray rectangles.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1893/F6.medium.gif" width="440" height="430"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/10/1893/F6.medium.gif" width="440" height="430"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1893/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464961" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-44" class="first-child">Linkage groups with mapped QTL. For each chromosome, only the linkage group (LG) with a QTL mapped was shown. Each linkage group was named by the chromosome number of <em>G. hirsutum</em>, the corresponding subgenome (A<sub>t</sub> or D<sub>t</sub>), and LG in this study. SNPs were named based on the chromosome number of <em>G. raimondii</em> followed by their alignment coordinates on the corresponding <em>G. raimondii</em> chromosome. SNP marker(s) with its origin different from the majority SNPs that defined the linkage group are shown in pink. The QTL regions were indicated by gray rectangles.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">Leaf shape in cotton affects yield, insect resistance, lint trash, and the efficacy of foliar chemical application (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Andres <em>et al.</em> 2014</a>). Cotton leaf trichomes serve various functions, including protection against herbivores, insect pests, and drought (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Desai <em>et al.</em> 2008</a>). These two traits have been extensively investigated and QTL related to leaf shape and trichome have been reported in cotton (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wright <em>et al.</em> 1999</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jiang <em>et al.</em> 2000</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lacape and Nguyen 2005</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Desai <em>et al.</em> 2008</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lacape <em>et al.</em> 2013</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Andres <em>et al.</em> 2014</a>). In this study, we chose these two traits together with another morphologic marker trait, pollen color, in QTL analyses to assess the usefulness of the SNP marker-only linkage map in QTL mapping (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). A major leaf shape QTL (<em>qLS1</em>, 62% phenotypic contribution) was mapped to an 8.8-cM interval in linkage group C15(D<sub>t</sub>)_LG04, consistent with previous results achieved in other populations (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Jiang <em>et al.</em> 2000</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Lacape <em>et al.</em> 2013</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Andres <em>et al.</em> 2014</a>). Two major leaf trichome density QTL (<em>qLT1</em> and <em>qLT2</em>, 31% and 28% phenotypic contribution, respectively) were identified in linkage groups <sup>11</sup>C(A<sub>t</sub>)_LG43 and C06(A<sub>t</sub>)_LG21. <em>qLT2</em> has been previously reported in other population (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Wright <em>et al.</em> 1999</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Lacape and Nguyen 2005</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Desai <em>et al.</em> 2008</a>) whereas <em>qLT1</em> is a newly identified QTL in the MCU-5 × Siokra 1−4 population. <em>qPC1</em>, a major QTL (91% phenotypic contribution) for pollen color, was mapped to a 19-cM interval in linkage group C04(A<sub>t</sub>)_LG15. In a recent report, several QTL for pollen color were mapped to C05 (A<sub>t</sub> subgenome) in a region centered on SSR marker CIR253 (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Lacape <em>et al.</em> 2013</a>), which according to <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Wang <em>et al.</em> (2013)</a> was mapped to <em>G. raimondii</em> Chr12 (C04, between 4554023 bp and 4554387 bp) rather than C05 (<em>G. raimondii</em> Chr09), consistent with our result.</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-46">In this study, we showed that the rationale we proposed worked well for the identification of genuine varietal or allelic SNPs in tetraploid cotton as opposed to the mostly subgenome-specific SNPs that have dominated many earlier SNP discovery projects in cotton based on the analysis of cotton ESTs in database collections. The validity of our approach was demonstrated by the high verification rate (72.6–91.7%) obtained with a reasonably large number of the identified varietal SNPs (~1700) and several different SNP genotyping platforms (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Our analyses generated a large number of potential varietal SNPs (~38 K) across a range of commercial varieties that form the base for much of the diversity in commercial cotton breeding in Australia and elsewhere. The international deployment of Australian varieties under the FiberMax brand over the last decade, and their use in a number of major US breeding programs means that the SNPs will have widespread utility and a significant proportion of them have been made available as part of an international cotton community SNP chip now marketed by Illumina (<a href="http://www.illumina.com/applications/agriculture/consortia.ilmn">http://www.illumina.com/applications/agriculture/consortia.ilmn</a>).</p><p id="p-47">To date, only two other published studies have reported large-scale varietal SNP identification in <em>G. hirsutum</em>, and both used NGS and the complexity-reduced DNA sequencing approach (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Byers <em>et al.</em> 2012</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Rai <em>et al.</em> 2013</a>), but neither had the potential breadth of utility for applications to <em>G. hirsutum</em> breeding as those reported here. <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Rai <em>et al.</em> (2013)</a> identified a large number of SNPs among six Indian <em>G. hirsutum</em> accessions, but only a small number (30 SNPs) were selected for validation and it is still unclear what proportion are reliable allelic SNPs rather than just subgenome-specific SNPs. Further verification using more SNPs and a broader range of germplasm is required to establish the reliability of those predicted SNPs. In the second study, where SNPs between a wild and a cultivated cotton were determined, less than 40% of the 1052 SNPs genotyped were found to amplify or segregate in an expected ratio in an F<sub>2</sub> population derived from the two <em>G. hirsutum</em> accessions used in SNP identification (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Byers <em>et al.</em> 2012</a>), so validation rates were low. One of the accessions is a wild race cotton and at the extreme edge of the diversity range in tetraploid <em>G. hirsutum</em>, so even the validated SNPs would not expected to be polymorphic among commercial varieties. In other polyploids, despite a verification rate of 93% being recently reported in <em>Brassica napus</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Huang <em>et al.</em> 2013</a>), most had a verification rate of less than 70% (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allen <em>et al.</em> 2011</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Trebbi <em>et al.</em> 2011</a>). Therefore, this study not only provided the cotton community a significant number of high-quality <em>G. hirsutum</em> SNPs but has contributed a novel way for identification of highly confident varietal SNPs between tetraploid varieties.</p><p id="p-48">Two other strategies have previously been used to identify cotton varietal SNPs from NGS data, although neither has proven to be very satisfactory. One was to <em>de novo</em> assemble high-throughput sequencing reads together from two varieties at a time and then use the automated calling function in AutoSNP (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Barker <em>et al.</em> 2003</a>) to detect varietal SNPs and exclude subgenome-specific SNPs (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Rai <em>et al.</em> 2013</a>). Accurate SNP identification with AutoSNP depends on coalignment of reads from homeologous genes, but avoiding coalignment of reads from paralogous genes. According to <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Salmon <em>et al.</em> (2010)</a>, a 97% identity rate or read mapping stringency was able to separate putative paralogs in cotton and this was the stringency cutoff used by <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Rai <em>et al.</em> (2013)</a>. However, given the sequence depth cutoff (at least three reads per genotype) used by those authors and consequent lack of discrimination between subgenome reads, it is likely that many of their putative varietal SNPs are subgenome-specific SNPs where, by chance or differential expression of homeologs, not all the alleles in one variety were successfully sampled during sequencing (as in a scenario like that shown in <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1A</a>, when only A<sub>t</sub> or D<sub>t</sub> reads were present for variety 2). A second strategy was to separately align the A<sub>t</sub> and D<sub>t</sub> genome reads from two varieties using very stringent alignment parameters or a precharacterized genome-specific SNP index to separate short reads into their subgenome of origin before varietal SNP identification (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Byers <em>et al.</em> 2012</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Salmon <em>et al.</em> 2012</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Page <em>et al.</em> 2013a</a>,<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">b</a>). This is based on the assumption that SNPs in the ancestral diploids (A<sub>2</sub>: <em>G. arboreum</em> and D<sub>5</sub>: <em>G. raimondii</em>) have remained unaltered since polyploidy formation and can be used as diagnostic subgenome-specific SNPs in modern tetraploid cotton (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Salmon <em>et al.</em> 2010</a>). Varietal SNPs can then be separately identified within each subgenome-specific assembly using the approaches and tools used for diploids, such as SAMtools (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2009</a>). However, it was found that ~30% of reads from allopolyploid cotton that mapped to the <em>G. raimondii</em> genome were indistinguishable between the A<sub>t</sub> and D<sub>t</sub> genome (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Page <em>et al.</em> 2013b</a>); therefore, these reads cannot be separated and will still be coaligned.</p><p id="p-49">In addition, although ~76% of the subgenome-specific SNPs found in modern tetraploid cotton were present in the ancestral A and D genomes, the remainder were newly evolved after polyploidization (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Page <em>et al.</em> 2013b</a>), so short reads containing these newly evolved subgenome-specific SNPs cannot be separated using the precharacterized, genome-specific SNP index developed based on the ancestral diploid genomes. Our approach, on the other hand, takes advantage of the <em>G. raimondii</em> (D<sub>5</sub>) genome sequence (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Paterson <em>et al.</em> 2012</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2012b</a>) and used it as the scaffold on which to align the RNA-seq reads allowing sufficient mismatches to capture homeologs but not paralogs. Since the short read sequences are transcript derived, differential expression between homeologs within varieties must also be considered because this may affect sampling of different alleles in the sequence space. Biased expression of A<sub>t</sub> and D<sub>t</sub> homeologs has been well documented in several studies in cotton (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams <em>et al.</em> 2003</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rapp <em>et al.</em> 2009</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Flagel and Wendel 2010</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Yoo <em>et al.</em> 2013</a>), and at least 40% of homeologs were reported to be transcriptionally biased in at least one stage of cotton development (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chaudhary <em>et al.</em> 2009</a>). To address this issue, we used the presence of subgenome-specific SNPs to filter alignments to those known to have both A<sub>t</sub> and D<sub>t</sub> reads represented from each pair of varieties and thence to confidently call adjacent varietal SNPs within the co-aligned reads in a 100-bp sliding window. Our approach avoided the problem associated with the AutoSNP approach, <em>i.e.</em>, false SNP calls caused by differential or biased allelic expression, and adopted the advantage of the separate subgenome alignment approach, <em>i.e.</em>, use of subgenome-specific SNPs to distinguish reads from the two subgenomes, which makes it possible to more confidently call a potential varietal SNP (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-50">More importantly, our approach did not specifically need to distinguish which allele is from the A<sub>t</sub> genome and which allele is from the D<sub>t</sub> genome, just that the SNP was biallelic in both genotypes being compared (although in many cases that should be possible with reference to the genomic resources now available for A<sub>2</sub>- and D<sub>5</sub>-genome cottons). This makes our approach more adaptable in polyploids without reference genome sequences. The constraints we have imposed through our strategy would necessarily underestimate the number of varietal SNPs between genotypes because our approach i) cannot identify varietal SNPs located in regions unique to either the A<sub>t</sub> or D<sub>t</sub> genome, as our SNP calling relies on the presence of both A<sub>t</sub> and D<sub>t</sub> reads; and ii) will miss the genuine varietal SNPs without a flanking subgenome-specific SNP. Nevertheless, in terms of ease of automation and reliability of prediction, our approach is a considerable improvement in SNP detection in the absence of a full tetraploid genome sequence and even when that sequence is available, will still offer some advantages over other automated approaches.</p><p id="p-51">Complexity-reduced DNA-sequencing strategies, such as RAD (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Baird <em>et al.</em> 2008</a>) and GBS (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Poland <em>et al.</em> 2012</a>), have been applied in many species to identify SNPs, particularly in those without a reference genome (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Barchi <em>et al.</em> 2011</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Yang <em>et al.</em> 2012</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Saintenac <em>et al.</em> 2013</a>). A number of bioinformatic tools, such as Stacks (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Catchen <em>et al.</em> 2011</a>) and UNEAK (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lu <em>et al.</em> 2013</a>), have been developed to handle this type of sequencing data for SNP discovery and genotyping. In this study, we used the publically available “off-the-shelf” windows-based tool, the CLC Genomics Workbench, which is accessible to wet-lab biologists, in SNP identification in tetraploid cotton. The software is designed to identify sequence variations between one accession and its reference sequence rather than sequence variations between two varieties but was adapted to this purpose through some preprocessing and filtering of the read data. By changing the format of the input data (sequence tags instead of sequence reads), setting up proper mapping and filtering parameters followed by manual checking, we successfully identified large numbers of varietal SNPs with a high verification rate in two tetraploid cotton varieties using the CLC Genomics Workbench (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>), contributing an alternative way for SNP identification, although again relying on adjacent subgenome-specific SNPs to increase the confidence of the calls. This approach, however, is only applicable for RAD or GBS sequencing data and not for RNA-seq data because each RNA-seq read is unique and cannot be collapsed into sequence tags in the same way as RAD or GBS reads.</p><p id="p-52">When allowing multiple aligned positions, We found that ~70% of the <em>G. hirsutum</em> transcriptome reads could be aligned to the <em>G. raimondii</em> genome, suggesting that the remaining reads could derive from genes either unique to the A<sub>t</sub> genome, or that have diverged significantly from those of their common ancestor with <em>G. raimondii</em>. When allowing only a single best aligned position (≤4 mismatches), 37–50% of <em>G. hirsutum</em> transcriptome reads could still be aligned to <em>G. raimondii</em> (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). In contrast, only about one third of the RAD reads could be aligned to the <em>G. raimondii</em> genome, even allowing up to eight mismatches. These results suggest that, in terms of read alignment and SNP identification, the <em>G. raimondii</em> reference works well for transcriptome derived reads but may not be able to fully support SNP identification from short reads generated from complexity-reduced DNA sequencing, either because of divergence between the D<sub>t</sub> and D<sub>5</sub> genomes or because the reference is just less reliable in those regions where the assembly is not supported by complementary EST or transcriptome data. Alignments will hopefully be improved by using the newly released A<sub>2</sub>-genome sequence or when a high-quality tetraploid cotton genome sequence becomes available, by which stage most cotton marker discovery and assessments will be through direct GBS.</p><p id="p-53">A codominant SNP assay will be more useful in genetic mapping and breeding programs that frequently deal with segregating populations, such as F<sub>2</sub> and backcross populations. In our SNP datasets, only ~3% were predicted to behave as codominant assays, which was significantly lower than that in wheat, where 10–20% of SNPs identified were simple codominant SNPs (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Allen <em>et al.</em> 2013</a>). This could be related to our stringent criterion used in SNP identification but the possibility that a low frequency of such “co-dominant” type SNP markers naturally occurs in cotton could not be ruled out. However, in practice, we found that a potentially dominant acting SNP assay could act as a codominant SNP assay when one or more genome-specific SNP(s) are present in the assay primers such that they amplify only the allele in which the varietal SNP resides. Genome-specific SNP(s) can be included in the allele-specific primers, the universal primer or both (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Where practical, it is better to include as many genome-specific SNP(s) as possible and to have them in both primers to ensure robust allele specificity. SNP assays designed to specifically amplify only the subgenome carrying the SNP has been attempted previously in cotton although with lower than expected success rates (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Byers <em>et al.</em> 2012</a>). Our genotyping data found more codominant SNP assays in the RAD-based SNPs than in the transcriptome-based SNPs, probably because noncoding sequences are more divergent and contain more genome-specific SNPs than coding sequences.</p><p id="p-54">Cotton genetic maps so far used in QTL mapping were mainly constructed using non-SNP markers, mostly SSR markers, although a genetic map with 1104 markers, including 414 SNP markers, has recently been used in mapping of QTL resistant to Verticillum wilt (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fang <em>et al.</em> 2014</a>) and a map with only SNP markers (346 in total) has been constructed (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Byers <em>et al.</em> 2012</a>). In this study we constructed a <em>G. hirsutum</em> genetic linkage map with 1244 of our identified SNP markers that were distributed across all 26 <em>G. hirsutum</em> chromosomes and also used them to map QTL for leaf shape, leaf trichome density, and pollen color on this SNP only map. Although three out of the four major QTL reported in this study were confirmations of earlier studies in different populations (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Wright <em>et al.</em> 1999</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Jiang <em>et al.</em> 2000</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Lacape and Nguyen 2005</a>, <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Desai <em>et al.</em> 2008</a>; <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Lacape <em>et al.</em> 2013</a>; <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Andres <em>et al.</em> 2014</a>), one new QTL was found for leaf trichome density (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>). In addition, the QTL intervals identified in this study were defined by the positions of the SNPs (based on <em>G. raimondii</em>), and the genes annotated in the intervals can then be screened and investigated to single out potential candidate(s) contributing to the QTL. There are 87, 74, 87, and 151 annotated genes in the regions corresponding to <em>qLS1</em>, <em>qLT1</em>, <em>qLT2</em>, and <em>qPC1</em> in <em>G. raimondii</em>, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1/FileS4.xlsx">File S4</a>). Recently, the region corresponding to <em>qLS1</em> has been further narrowed down to containing only 34 annotated genes, from which Gorai.002G244000 and Gorai.002G244200 that encode HD-Zip transcription factors were suggested to be the possible candidates for the leaf shape trait (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Andres <em>et al.</em> 2014</a>). Further investigation is required to confirm this speculation; nevertheless, use of SNP markers with a position designated based on the genome sequence of <em>G. raimondii</em> should be able to speed up the procedure of QTL fine mapping and identification of candidate genes underlying QTL of interest. Our full set of markers will have great utility in mapping more complex traits in cotton such as disease resistance or fiber yield and quality that are often conferred by many genes of small effect and will advance the use of marker-assisted selection in cotton breeding.</p><p id="p-55">Using transcriptome and complexity-reduced-DNA sequencing, we identified a large number of varietal SNPs among 18 <em>G. hirsutum</em> varieties based on a robust protocol that relied on adjacent subgenome-specific SNPs to increase the confidence of SNP assignment to single alleles. A verification rate of 72.6–91.7% was achieved and ~25,000 of these SNPs satisfy the criteria for use in a number of common SNP genotyping platforms. Our pool of SNPs span a range of commercial and elite germplasm of <em>G. hirsutum</em> and so provides valuable marker resource for the cotton community as demonstrated by mapping of QTL for several traits of interest. The SNP identification rationale described here should be applicable to other polyploids.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We thank Tanya Phongkham, Vanessa Gillespie, and Nicki Mossfield for their excellent technical support in callus preparation and DNA extraction and Penghao Wang for helping in generating the graph shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>. This work was supported by Cotton Breeding Australia, a joint venture between CSIRO and Cotton Seed Distributors.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-8"><p id="p-57">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012542/-/DC1</a></p></li><li class="fn" id="fn-9"><p id="p-58"><em>Communicating editor: D. Zamir</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-06-05"><span class="received-label">Received </span>June 5, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-07-31"><span class="accepted-label">Accepted </span>July 31, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Zhu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.1" data-doi="10.1073/pnas.0630618100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Percifield</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genes duplicated by polyploidy show unequal contributions to the transcriptome and organ-specific reciprocal silencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">4649</span>–<span class="cit-lpage">4654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0630618100%26rft_id%253Dinfo%253Apmid%252F12665616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzgvNDY0OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTAvMTg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.2" data-doi="10.1111/j.1467-7652.2011.00628.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">G. L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coghill</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gwilliam</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1086</span>–<span class="cit-lpage">1099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DAllen%26rft.auinit1%253DA.%2BM.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D1086%26rft.epage%253D1099%26rft.atitle%253DTranscript-specific%252C%2Bsingle-nucleotide%2Bpolymorphism%2Bdiscovery%2Band%2Blinkage%2Banalysis%2Bin%2Bhexaploid%2Bbread%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2011.00628.x%26rft_id%253Dinfo%253Apmid%252F21627760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2011.00628.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21627760&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297941700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.3" data-doi="10.1111/pbi.12009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burridge</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winfield</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Discovery and development of exome-based, co-dominant single nucleotide polymorphism markers in hexaploid wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D11%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12009%26rft_id%253Dinfo%253Apmid%252F23279710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23279710&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316825400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.4" data-doi="10.1007/s00438-007-0270-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheffler</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Transcriptome profiling, sequence characterization, and SNP-based chromosomal assignment of the <em>EXPANSIN</em> genes in cotton.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DAn%26rft.auinit1%253DC.%26rft.volume%253D278%26rft.issue%253D5%26rft.spage%253D539%26rft.epage%253D553%26rft.atitle%253DTranscriptome%2Bprofiling%252C%2Bsequence%2Bcharacterization%252C%2Band%2BSNP-based%2Bchromosomal%2Bassignment%2Bof%2Bthe%2BEXPANSIN%2Bgenes%2Bin%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-007-0270-9%26rft_id%253Dinfo%253Apmid%252F17724613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-007-0270-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17724613&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.5" data-doi="10.1007/s00122-008-0732-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheffler</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cotton (<em>Gossypium</em> spp.) R2R3-MYB transcription factors SNP identification, phylogenomic characterization, chromosome localization, and linkage mapping.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">1015</span>–<span class="cit-lpage">1026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAn%26rft.auinit1%253DC.%26rft.volume%253D116%26rft.issue%253D7%26rft.spage%253D1015%26rft.epage%253D1026%26rft.atitle%253DCotton%2B%2528Gossypium%2Bspp.%2529%2BR2R3-MYB%2Btranscription%2Bfactors%2BSNP%2Bidentification%252C%2Bphylogenomic%2Bcharacterization%252C%2Bchromosome%2Blocalization%252C%2Band%2Blinkage%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0732-4%26rft_id%253Dinfo%253Apmid%252F18338155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0732-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18338155&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.6" data-doi="10.1007/s00122-013-2208-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andres</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuraparthy</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mapping and genomic targeting of the major leaf shape gene (<em>L</em>) in Upland cotton (<em>Gossypium hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2208-4%26rft_id%253Dinfo%253Apmid%252F24158249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2208-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24158249&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.7" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.8" data-doi="10.1038/nbt.1926"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bancroft</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Dissecting the genome of the polyploid crop oilseed rape by transcriptome sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DBancroft%26rft.auinit1%253DI.%26rft.volume%253D29%26rft.issue%253D8%26rft.spage%253D762%26rft.epage%253D766%26rft.atitle%253DDissecting%2Bthe%2Bgenome%2Bof%2Bthe%2Bpolyploid%2Bcrop%2Boilseed%2Brape%2Bby%2Btranscriptome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1926%26rft_id%253Dinfo%253Apmid%252F21804563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1926&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21804563&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.9" data-doi="10.1186/1471-2164-12-304"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barchi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanteri</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Portis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acquadro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vale</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of SNP and SSR markers in eggplant using RAD tag sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBarchi%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.spage%253D304%26rft.epage%253D304%26rft.atitle%253DIdentification%2Bof%2BSNP%2Band%2BSSR%2Bmarkers%2Bin%2Beggplant%2Busing%2BRAD%2Btag%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-304%26rft_id%253Dinfo%253Apmid%252F21663628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-304&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21663628&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.10" data-doi="10.1093/bioinformatics/btf881"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Sullivan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Redundancy based detection of sequence polymorphisms in expressed sequence tag data using autoSNP.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBarker%26rft.auinit1%253DG.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D421%26rft.epage%253D422%26rft.atitle%253DRedundancy%2Bbased%2Bdetection%2Bof%2Bsequence%2Bpolymorphisms%2Bin%2Bexpressed%2Bsequence%2Btag%2Bdata%2Busing%2BautoSNP%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtf881%26rft_id%253Dinfo%253Apmid%252F12584131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS8zLzQyMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTAvMTg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.11" data-doi="10.1007/s00122-011-1780-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byers</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harker</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yourstone</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maughan</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development and mapping of SNP assays in allotetraploid cotton.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1201</span>–<span class="cit-lpage">1214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DByers%26rft.auinit1%253DR.%2BL.%26rft.volume%253D124%26rft.issue%253D7%26rft.spage%253D1201%26rft.epage%253D1214%26rft.atitle%253DDevelopment%2Band%2Bmapping%2Bof%2BSNP%2Bassays%2Bin%2Ballotetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1780-8%26rft_id%253Dinfo%253Apmid%252F22252442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1780-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22252442&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1893.12" data-doi="10.1534/g3.111.000240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: building and genotyping Loci de novo from short-read sequences.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DCatchen%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D171%26rft.epage%253D182%26rft.atitle%253DStacks%253A%2BBuilding%2Band%2BGenotyping%2BLoci%2BDe%2BNovo%2BFrom%2BShort-Read%2BSequences%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000240%26rft_id%253Dinfo%253Apmid%252F22384329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjEvMy8xNzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE4OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.13" data-doi="10.1073/pnas.1217133110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">8057</span>–<span class="cit-lpage">8062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1217133110%26rft_id%253Dinfo%253Apmid%252F23630259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIwLzgwNTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE4OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.14" data-doi="10.1534/genetics.109.102608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaudhary</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stupar</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reciprocal silencing, transcriptional bias and functional divergence of homeologs in polyploid cotton (<em>Gossypium</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.102608%26rft_id%253Dinfo%253Apmid%252F19363125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzIvNTAzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.15" data-doi="10.1007/s00122-011-1633-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Development and application of a set of breeder-friendly SNP markers for genetic analyses and molecular breeding of rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">869</span>–<span class="cit-lpage">879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DChen%26rft.auinit1%253DH.%26rft.volume%253D123%26rft.issue%253D6%26rft.spage%253D869%26rft.epage%253D879%26rft.atitle%253DDevelopment%2Band%2Bapplication%2Bof%2Ba%2Bset%2Bof%2Bbreeder-friendly%2BSNP%2Bmarkers%2Bfor%2Bgenetic%2Banalyses%2Band%2Bmolecular%2Bbreeding%2Bof%2Brice%2B%2528Oryza%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1633-5%26rft_id%253Dinfo%253Apmid%252F21681488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1633-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681488&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.16" data-doi="10.1071/PP9910481"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cousins</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyon</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Llewellyn</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Transformation of an Australian cotton cultivar - prospects for cotton improvement through genetic-engineering.</span> <abbr class="cit-jnl-abbrev">Aust. J. Plant Physiol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BPlant%2BPhysiol.%26rft.volume%253D18%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1071%252FPP9910481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/PP9910481&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.17" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.18" data-doi="10.1093/jhered/esm112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chee</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">O. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Correspondence of trichome mutations in diploid and tetraploid cottons.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesm112%26rft_id%253Dinfo%253Apmid%252F18296391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijk5LzIvMTgyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.19" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.20" data-doi="10.1007/s11032-013-9987-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanogo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative trait locus analysis of Verticillium wilt resistance in an introgressed recombinant inbred population of Upland cotton.</span> <abbr class="cit-jnl-abbrev">Mol. Breeding</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">720</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreeding%26rft.volume%253D33%26rft.spage%253D709%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-013-9987-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-013-9987-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.21" data-doi="10.1111/j.1469-8137.2009.03107.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evolutionary rate variation, genomic dominance and duplicate gene expression evolution during allotetraploid cotton speciation.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DFlagel%26rft.auinit1%253DL.%2BE.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D184%26rft.epage%253D193%26rft.atitle%253DEvolutionary%2Brate%2Bvariation%252C%2Bgenomic%2Bdominance%2Band%2Bduplicate%2Bgene%2Bexpression%2Bevolution%2Bduring%2Ballotetraploid%2Bcotton%2Bspeciation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03107.x%26rft_id%253Dinfo%253Apmid%252F20002320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03107.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20002320&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.22" data-doi="10.1016/j.pbi.2008.12.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roder</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SNP identification in crop plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D211%26rft.epage%253D217%26rft.atitle%253DSNP%2Bidentification%2Bin%2Bcrop%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2008.12.009%26rft_id%253Dinfo%253Apmid%252F19186095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2008.12.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19186095&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265115400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular and SNP characterization of two genome specific transcription factor genes <em>GhMyb8</em> and <em>GhMyb10</em> in cotton species.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D159%26rft.spage%253D259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.24" data-doi="10.1186/1471-2164-14-717"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of genome-wide single nucleotide polymorphisms in allopolyploid crop <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D717%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-717%26rft_id%253Dinfo%253Apmid%252F24138473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-717&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24138473&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.25" data-doi="10.1101/gr.089516.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-throughput genotyping by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089516.108%26rft_id%253Dinfo%253Apmid%252F19420380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA2OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTAvMTg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.26" data-doi="10.1038/ng.1018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association study of flowering time and grain yield traits in a worldwide collection of rice germplasm.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D39%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bflowering%2Btime%2Band%2Bgrain%2Byield%2Btraits%2Bin%2Ba%2Bworldwide%2Bcollection%2Bof%2Brice%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1018%26rft_id%253Dinfo%253Apmid%252F22138690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22138690&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.27" data-doi="10.1007/s001220050054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DelMonte</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">QTL analysis of leaf morphology in tetraploid <em>Gossypium</em> (cotton).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D100%26rft.spage%253D409%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000086145800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.28" data-doi="10.1038/ng.2312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide genetic changes during modern breeding of maize.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">812</span>–<span class="cit-lpage">815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DJiao%26rft.auinit1%253DY.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D812%26rft.epage%253D815%26rft.atitle%253DGenome-wide%2Bgenetic%2Bchanges%2Bduring%2Bmodern%2Bbreeding%2Bof%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2312%26rft_id%253Dinfo%253Apmid%252F22660547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660547&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.29" data-doi="10.1111/j.1467-7652.2011.00644.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francki</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification, characterization and interpretation of single-nucleotide sequence variation in allopolyploid crop species.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DKaur%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D125%26rft.epage%253D138%26rft.atitle%253DIdentification%252C%2Bcharacterization%2Band%2Binterpretation%2Bof%2Bsingle-nucleotide%2Bsequence%2Bvariation%2Bin%2Ballopolyploid%2Bcrop%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2011.00644.x%26rft_id%253Dinfo%253Apmid%252F21831136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2011.00644.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21831136&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298781400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.30" data-doi="10.1093/jhered/esi052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mapping quantitative trait loci associated with leaf and stem pubescence in cotton.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesi052%26rft_id%253Dinfo%253Apmid%252F15829730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijk2LzQvNDQxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.31" data-doi="10.1016/j.fcr.2013.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gawrysiak</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">T. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Llewellyn</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mapping QTLs for traits related to phenology, morphology and yield components in an inter-specific <em>Gossypium hirsutum</em> x <em>G. barbadense</em> cotton RIL population.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D144%26rft.spage%253D256%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2013.01.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2013.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.32" data-doi="10.1038/ng.2987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequence of the cultivated cotton <em>Gossypium arboreum</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D567%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2987%26rft_id%253Dinfo%253Apmid%252F24836287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24836287&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A modified algorithm for the improvement of composite interval mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Apmid%252F17110476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzEvMzYxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.34" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTAvMTg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide association study dissects the genetic architecture of oil biosynthesis in maize kernels.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D43%26rft_id%253Dinfo%253Apmid%252F23242369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23242369&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.36" data-doi="10.1007/s11032-012-9705-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Lavalle</span>  <span class="cit-name-given-names">L. A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillespie</span>  <span class="cit-name-given-names">V. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tate</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Molecular mapping of a new source of Fusarium wilt resistance in tetraploid cotton (<em>Gossypium hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D30%26rft.spage%253D1181%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-012-9705-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-012-9705-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.37" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.38" data-doi="10.1111/tpj.12294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerhardt</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Himmelbach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clissold</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Barley whole exome capture: a tool for genomic research in the genus <em>Hordeum</em> and beyond.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">494</span>–<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D76%26rft.spage%253D494%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12294%26rft_id%253Dinfo%253Apmid%252F23889683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23889683&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326246700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1893.39" data-doi="10.1534/g3.112.005298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gingle</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">PolyCat: A resource for genome categorization of sequencing reads from allopolyploid organisms.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DPage%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D517%26rft.epage%253D525%26rft.atitle%253DPolyCat%253A%2BA%2BResource%2Bfor%2BGenome%2BCategorization%2Bof%2BSequencing%2BReads%2BFrom%2BAllopolyploid%2BOrganisms%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.005298%26rft_id%253Dinfo%253Apmid%252F23450226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvMy81MTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE4OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1893.40" data-doi="10.1534/g3.113.007229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huynh</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liechty</span>  <span class="cit-name-given-names">Z. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grupp</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stelly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Insights into the evolution of cotton diploids and polyploids from whole-genome re-sequencing.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">1809</span>–<span class="cit-lpage">1818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007229%26rft_id%253Dinfo%253Apmid%252F23979935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTAvMTgwOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTAvMTg5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.41" data-doi="10.1038/nature11798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundlach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repeated polyploidization of <em>Gossypium</em> genomes and the evolution of spinnable cotton fibers.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">492</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D492%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11798%26rft_id%253Dinfo%253Apmid%252F23257886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23257886&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312488200055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.42" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.43" data-doi="10.1111/tpj.12075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pomeranz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegal-Gaskins</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engelmeier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution computational imaging of leaf hair patterning using polarized light microscopy.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">701</span>–<span class="cit-lpage">708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D73%26rft.spage%253D701%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12075%26rft_id%253Dinfo%253Apmid%252F23163919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23163919&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.44" data-doi="10.2135/cropsci2002.2137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rahman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hussain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zafar</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Estimation of genetic divergence among elite cotton cultivars-genotypes by DNA fingerprinting technology.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">2137</span>–<span class="cit-lpage">2144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DRahman%26rft.auinit1%253DM.%26rft.volume%253D42%26rft.issue%253D6%26rft.spage%253D2137%26rft.epage%253D2144%26rft.atitle%253DEstimation%2Bof%2BGenetic%2BDivergence%2Bamong%2BElite%2BCotton%2BCultivars-Genotypes%2Bby%2BDNA%2BFingerprinting%2BTechnology%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2002.2137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2002.2137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.45" data-doi="10.1111/pbi.12088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rai</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhardwaj</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lakhwani</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Large-scale resource development in <em>Gossypium hirsutum</em> L. by 454 sequencing of genic-enriched libraries from six diverse genotypes.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D11%26rft.spage%253D953%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12088%26rft_id%253Dinfo%253Apmid%252F23782852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23782852&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.46" data-doi="10.1186/1741-7007-7-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rapp</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic expression dominance in allopolyploids.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DRapp%26rft.auinit1%253DR.%2BA.%26rft.volume%253D7%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DGenomic%2Bexpression%2Bdominance%2Bin%2Ballopolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-7-18%26rft_id%253Dinfo%253Apmid%252F19409075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-7-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19409075&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.47" data-doi="10.1038/ng.1033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czedik-Eysenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic and metabolic prediction of complex heterotic traits in hybrid maize.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D220%26rft.atitle%253DGenomic%2Band%2Bmetabolic%2Bprediction%2Bof%2Bcomplex%2Bheterotic%2Btraits%2Bin%2Bhybrid%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1033%26rft_id%253Dinfo%253Apmid%252F22246502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22246502&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.48" data-doi="10.1534/genetics.166.1.389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbey</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A 3347-locus genetic recombination map of sequence-tagged sites reveals features of genome organization, transmission and evolution of cotton (<em>Gossypium</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRong%26rft.auinit1%253DJ.%26rft.volume%253D166%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D417%26rft.atitle%253DA%2B3347-Locus%2BGenetic%2BRecombination%2BMap%2Bof%2BSequence-Tagged%2BSites%2BReveals%2BFeatures%2Bof%2BGenome%2BOrganization%252C%2BTransmission%2Band%2BEvolution%2Bof%2BCotton%2B%2528Gossypium%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.1.389%26rft_id%253Dinfo%253Apmid%252F15020432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY2LzEvMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.49" data-doi="10.1071/AR04190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rungis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Llewellyn</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dennis</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyon</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Simple sequence repeat (SSR) markers reveal low levels of polymorphism between cotton (<em>Gossypium hirsutum</em> L.) cultivars.</span> <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D56%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR04190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR04190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1893.50" data-doi="10.1534/g3.113.005819"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saintenac</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sequence-based mapping of the polyploid wheat genome.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.005819%26rft_id%253Dinfo%253Apmid%252F23665877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjMvNy8xMTA1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.51" data-doi="10.1111/j.1469-8137.2009.03093.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Homoeologous nonreciprocal recombination in polyploid cotton.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSalmon%26rft.auinit1%253DA.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D123%26rft.epage%253D134%26rft.atitle%253DHomoeologous%2Bnonreciprocal%2Brecombination%2Bin%2Bpolyploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03093.x%26rft_id%253Dinfo%253Apmid%252F19925554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03093.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19925554&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1893.52" data-doi="10.1534/g3.112.003392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeddeloh</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Targeted capture of homoeologous coding and noncoding sequence in polyploid cotton.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">2</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DSalmon%26rft.auinit1%253DA.%26rft.volume%253D2%26rft.issue%253D8%26rft.spage%253D921%26rft.epage%253D930%26rft.atitle%253DTargeted%2BCapture%2Bof%2BHomoeologous%2BCoding%2Band%2BNoncoding%2BSequence%2Bin%2BPolyploid%2BCotton%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003392%26rft_id%253Dinfo%253Apmid%252F22908041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvOC85MjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE4OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.53" data-doi="10.1093/oxfordjournals.molbev.a026131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Small</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryburn</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Low levels of nucleotide diversity at homoeologous <em>Adh</em> loci in allotetraploid cotton (<em>Gossypium</em> L.).</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSmall%26rft.auinit1%253DR.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D491%26rft.epage%253D501%26rft.atitle%253DLow%2Blevels%2Bof%2Bnucleotide%2Bdiversity%2Bat%2Bhomoeologous%2BAdh%2Bloci%2Bin%2Ballotetraploid%2Bcotton%2B%2528Gossypium%2BL.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026131%26rft_id%253Dinfo%253Apmid%252F10331275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE2LzQvNDkxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.54" data-doi="10.1007/s00122-011-1607-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trebbi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccaferri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Heer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giuliani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-throughput SNP discovery and genotyping in durum wheat (<em>Triticum durum</em> Desf.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTrebbi%26rft.auinit1%253DD.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D555%26rft.epage%253D569%26rft.atitle%253DHigh-throughput%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bdurum%2Bwheat%2B%2528Triticum%2Bdurum%2BDesf.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1607-7%26rft_id%253Dinfo%253Apmid%252F21611761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1607-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21611761&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.55" data-doi="10.1186/1471-2229-9-125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sampling nucleotide diversity in cotton.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DVan%2BDeynze%26rft.auinit1%253DA.%2BE.%26rft.volume%253D9%26rft.spage%253D125%26rft.epage%253D125%26rft.atitle%253DSampling%2Bnucleotide%2Bdiversity%2Bin%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-125%26rft_id%253Dinfo%253Apmid%252F19840401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19840401&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.56" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: Software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6IjkzLzEvNzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE4OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.57" data-doi="10.1006/abio.1994.1538"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">A modified hot borate method significantly enhances the yield of high-quality RNA from cotton (<em>Gossypium hirsutum</em> L).</span> <abbr class="cit-jnl-abbrev">Anal. Biochem.</abbr> <span class="cit-vol">223</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DWan%26rft.auinit1%253DC.%2BY.%26rft.volume%253D223%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D12%26rft.atitle%253DA%2Bmodified%2Bhot%2Bborate%2Bmethod%2Bsignificantly%2Benhances%2Bthe%2Byield%2Bof%2Bhigh-quality%2BRNA%2Bfrom%2Bcotton%2B%2528Gossypium%2Bhirsutum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fabio.1994.1538%26rft_id%253Dinfo%253Apmid%252F7535022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/abio.1994.1538&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7535022&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PR65400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.58" data-doi="10.1186/1471-2229-12-148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Construction of a high-density genetic map for grape using next generation restriction-site associated DNA sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DWang%26rft.auinit1%253DN.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D148%26rft.epage%253D148%26rft.atitle%253DConstruction%2Bof%2Ba%2Bhigh-density%2Bgenetic%2Bmap%2Bfor%2Bgrape%2Busing%2Bnext%2Bgeneration%2Brestriction-site%2Bassociated%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-12-148%26rft_id%253Dinfo%253Apmid%252F22908993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-12-148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22908993&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.59" data-doi="10.1038/ng.2371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">The draft genome of a diploid cotton <em>Gossypium raimondii</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1098</span>–<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DK.%26rft.volume%253D44%26rft.issue%253D10%26rft.spage%253D1098%26rft.epage%253D1103%26rft.atitle%253DThe%2Bdraft%2Bgenome%2Bof%2Ba%2Bdiploid%2Bcotton%2BGossypium%2Braimondii.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2371%26rft_id%253Dinfo%253Apmid%252F22922876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22922876&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1893.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A whole-genome DNA marker map for cotton based on the D-genome sequence of <em>Gossypium raimondii</em> L.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D1759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.61" data-doi="10.1016/S0065-2113(02)78004-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Polyploidy and the evolutionary history of cotton.</span> <abbr class="cit-jnl-abbrev">Adv. Agron.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D78%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2113%252802%252978004-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2113(02)78004-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1893.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seelanan</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The origin and evolution of <em>Gossypium</em></span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span> in <span class="cit-source">Physiology of Cotton</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Oosterhuis</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Heitholt</span>  <span class="cit-name-given-names">J. J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Mauney</span>  <span class="cit-name-given-names">J. R.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.63" data-doi="10.1093/jhered/90.1.215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaxton</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Zik</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Molecular mapping of genes affecting pubescence of cotton.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DWright%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D215%26rft.epage%253D219%26rft.atitle%253DMolecular%2Bmapping%2Bof%2Bgenes%2Baffecting%2Bpubescence%2Bof%2Bcotton%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F90.1.215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6IjkwLzEvMjE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xODkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.64" data-doi="10.1186/1471-2164-13-318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweetingham</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Application of next-generation sequencing for rapid marker development in molecular plant breeding: a case study on anthracnose disease resistance in <em>Lupinus angustifolius</em> L.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.spage%253D318%26rft.epage%253D318%26rft.atitle%253DApplication%2Bof%2Bnext-generation%2Bsequencing%2Bfor%2Brapid%2Bmarker%2Bdevelopment%2Bin%2Bmolecular%2Bplant%2Bbreeding%253A%2Ba%2Bcase%2Bstudy%2Bon%2Banthracnose%2Bdisease%2Bresistance%2Bin%2BLupinus%2Bangustifolius%2BL.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-318%26rft_id%253Dinfo%253Apmid%252F22805587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22805587&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1893.65" data-doi="10.1038/hdy.2012.94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szadkowski</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Homoeolog expression bias and expression level dominance in allopolyploid cotton.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D110%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.94%26rft_id%253Dinfo%253Apmid%252F23169565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23169565&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1893.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/10/1893.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/10/1881" title="Endogenous Retrovirus Insertion in the KIT Oncogene Determines White and White spotting in Domestic Cats" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/10/1907" title="Evidence of Hermaphroditism and Sex Ratio Distortion in the Fungal Feeding Nematode Bursaphelenchus okinawaensis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/10"><span>Volume 4 Issue 10, October 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/10.cover-source.jpg?itok=f3Vstm56" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/10/1893" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8vpncKuLfysXFNIqR4-TTGX95OlAx_oYRKOA0nK364w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/10/1893" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/10/1893" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/10/1893" class="active" data-icon-position="" data-hide-link-title="0">Transcriptome and Complexity-Reduced, DNA-Based Identification of Intraspecies Single-Nucleotide Polymorphisms in the Polyploid Gossypium hirsutum L.</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459681" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_D6bxxJzgnMhPeIbaHZzXub2e7ntHE1UE-3H_MBoi5U" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/10/1893.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/10/1893" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-KRqYfBK71Wz5MRx3oISqlNd36MJw0C2-6jZix_5RR-o" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/10/1893" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/10/1893.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459681" id="node459681--2" data-pisa="ggg;4/10/1893" data-pisa-master="ggg;g3.114.012542" data-apath="/ggg/4/10/1893.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptome and Complexity-Reduced, DNA-Based Identification of Intraspecies Single-Nucleotide Polymorphisms in the Polyploid <em>Gossypium hirsutum</em> L.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Qian-Hao Zhu</span>, <span class="highwire-citation-author" data-delta="1">Andrew Spriggs</span>, <span class="highwire-citation-author" data-delta="2">Jennifer M. Taylor</span>, <span class="highwire-citation-author" data-delta="3">Danny Llewellyn</span> and <span class="highwire-citation-author" data-delta="4">Iain Wilson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1893-1905;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012542 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459681--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Qian-Hao Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian-Hao%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQian-Hao%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Qianhao.Zhu@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Qianhao.Zhu@csiro.au</a>
<a href="mailto:Iain.Wilson@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Iain.Wilson@csiro.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew Spriggs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BSpriggs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spriggs%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BSpriggs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer M. Taylor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BM.%2BTaylor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taylor%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BM.%2BTaylor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Danny Llewellyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BLlewellyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Llewellyn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanny%2BLlewellyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Iain Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iain%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIain%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Qianhao.Zhu@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Qianhao.Zhu@csiro.au</a>
<a href="mailto:Iain.Wilson@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Iain.Wilson@csiro.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/10/1893" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/10/1893&amp;title=Transcriptome%20and%20Complexity-Reduced%2C%20DNA-Based%20Identification%20of%20Intraspecies%20Single-Nucleotide%20Polymorphisms%20in%20the%20Polyploid%20Gossypium%20hirsutum%20L." id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/10/1893&amp;title=Transcriptome%20and%20Complexity-Reduced%2C%20DNA-Based%20Identification%20of%20Intraspecies%20Single-Nucleotide%20Polymorphisms%20in%20the%20Polyploid%20Gossypium%20hirsutum%20L." id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/10/1893&amp;title=Transcriptome%20and%20Complexity-Reduced%2C%20DNA-Based%20Identification%20of%20Intraspecies%20Single-Nucleotide%20Polymorphisms%20in%20the%20Polyploid%20Gossypium%20hirsutum%20L." id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/10/1893&amp;text=Transcriptome%20and%20Complexity-Reduced%2C%20DNA-Based%20Identification%20of%20Intraspecies%20Single-Nucleotide%20Polymorphisms%20in%20the%20Polyploid%20Gossypium%20hirsutum%20L." id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/10/1893&amp;title=Transcriptome%20and%20Complexity-Reduced%2C%20DNA-Based%20Identification%20of%20Intraspecies%20Single-Nucleotide%20Polymorphisms%20in%20the%20Polyploid%20Gossypium%20hirsutum%20L." id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/10/1893&amp;t=Transcriptome%20and%20Complexity-Reduced%2C%20DNA-Based%20Identification%20of%20Intraspecies%20Single-Nucleotide%20Polymorphisms%20in%20the%20Polyploid%20Gossypium%20hirsutum%20L." id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/10/1893&amp;title=Transcriptome%20and%20Complexity-Reduced%2C%20DNA-Based%20Identification%20of%20Intraspecies%20Single-Nucleotide%20Polymorphisms%20in%20the%20Polyploid%20Gossypium%20hirsutum%20L." id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/10/1893&amp;title=Transcriptome%20and%20Complexity-Reduced%2C%20DNA-Based%20Identification%20of%20Intraspecies%20Single-Nucleotide%20Polymorphisms%20in%20the%20Polyploid%20Gossypium%20hirsutum%20L." id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459681/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459681" id="node459681--3" data-pisa="ggg;4/10/1893" data-pisa-master="ggg;g3.114.012542" data-apath="/ggg/4/10/1893.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptome and Complexity-Reduced, DNA-Based Identification of Intraspecies Single-Nucleotide Polymorphisms in the Polyploid <em>Gossypium hirsutum</em> L.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Qian-Hao Zhu</span>, <span class="highwire-citation-author" data-delta="1">Andrew Spriggs</span>, <span class="highwire-citation-author" data-delta="2">Jennifer M. Taylor</span>, <span class="highwire-citation-author" data-delta="3">Danny Llewellyn</span> and <span class="highwire-citation-author" data-delta="4">Iain Wilson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1893-1905;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012542 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459681--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Qian-Hao Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian-Hao%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQian-Hao%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Qianhao.Zhu@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Qianhao.Zhu@csiro.au</a>
<a href="mailto:Iain.Wilson@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Iain.Wilson@csiro.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew Spriggs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BSpriggs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spriggs%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BSpriggs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer M. Taylor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BM.%2BTaylor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taylor%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BM.%2BTaylor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Danny Llewellyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BLlewellyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Llewellyn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanny%2BLlewellyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Iain Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Plant Industry, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iain%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIain%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Qianhao.Zhu@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Qianhao.Zhu@csiro.au</a>
<a href="mailto:Iain.Wilson@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Iain.Wilson@csiro.au</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1893&amp;rft.epage=1905&amp;rft.atitle=Transcriptome%20and%20Complexity-Reduced%2C%20DNA-Based%20Identification%20of%20Intraspecies%20Single-Nucleotide%20Polymorphisms%20in%20the%20Polyploid%20Gossypium%20hirsutum%20L.&amp;rft.volume=4&amp;rft.issue=10&amp;rft.date=2014-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhu%2C+Qian-Hao&amp;rft.au=Spriggs%2C+Andrew&amp;rft.au=Taylor%2C+Jennifer+M.&amp;rft.au=Llewellyn%2C+Danny&amp;rft.au=Wilson%2C+Iain"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459681/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459681/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459681/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459681/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459681/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459681/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459681/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459681/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459681/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459681/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459681/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459681/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/10/1893" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/10/1893.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/10/1893.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/10/1893.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/10/1893.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
