<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/10/1963" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/10/1963.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/10/1963.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/10/1963.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/10/1963/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/10/1963.atom" />
<meta name="HW.pisa" content="ggg;4/10/1963" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Resolution of Genetic Map Expansion Caused by Excess Heterozygosity in Plant Recombinant Inbred Populations" />
<meta name="DC.Identifier" content="10.1534/g3.114.012468" />
<meta name="DC.Date" content="2014-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Truong et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Recombinant inbred populations of many plant species exhibit more heterozygosity than expected under the Mendelian model of segregation. This segregation distortion causes the overestimation of recombination frequencies and consequent genetic map expansion. Here we build upon existing genetic models of differential zygotic viability to model a heterozygote fitness term and calculate expected genotypic proportions in recombinant inbred populations propagated by selfing. We implement this model using the existing open-source genetic map construction code base for R/qtl to estimate recombination fractions. Finally, we show that accounting for excess heterozygosity in a sorghum recombinant inbred mapping population shrinks the genetic map by 213 cM (a 13% decrease corresponding to 4.26 fewer recombinations per meiosis). More accurate estimates of linkage benefit linkage-based analyses used in the identification and utilization of causal genetic variation." />
<meta name="DC.Contributor" content="Sandra K. Truong" />
<meta name="DC.Contributor" content="Ryan F. McCormick" />
<meta name="DC.Contributor" content="Daryl T. Morishige" />
<meta name="DC.Contributor" content="John E. Mullet" />
<meta name="article:published_time" content="2014-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Resolution of Genetic Map Expansion Caused by Excess Heterozygosity in Plant Recombinant Inbred Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Recombinant inbred populations of many plant species exhibit more heterozygosity than expected under the Mendelian model of segregation. This segregation distortion causes the overestimation of recombination frequencies and consequent genetic map expansion. Here we build upon existing genetic models of differential zygotic viability to model a heterozygote fitness term and calculate expected genotypic proportions in recombinant inbred populations propagated by selfing. We implement this model using the existing open-source genetic map construction code base for R/qtl to estimate recombination fractions. Finally, we show that accounting for excess heterozygosity in a sorghum recombinant inbred mapping population shrinks the genetic map by 213 cM (a 13% decrease corresponding to 4.26 fewer recombinations per meiosis). More accurate estimates of linkage benefit linkage-based analyses used in the identification and utilization of causal genetic variation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/10/01" />
<meta name="citation_mjid" content="ggg;4/10/1963" />
<meta name="citation_id" content="4/10/1963" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/10/1963" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/10/1963.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/10/1963.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/10/1963.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.012468" />
<meta name="citation_pmid" content="25128435" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1963" />
<meta name="citation_lastpage" content="1969" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sandra K. Truong" />
<meta name="citation_author_institution" content="Interdisciplinary Program in Genetics, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_institution" content="Biochemistry &amp; Biophysics Department, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author" content="Ryan F. McCormick" />
<meta name="citation_author_institution" content="Interdisciplinary Program in Genetics, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_institution" content="Biochemistry &amp; Biophysics Department, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author" content="Daryl T. Morishige" />
<meta name="citation_author_institution" content="Biochemistry &amp; Biophysics Department, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author" content="John E. Mullet" />
<meta name="citation_author_institution" content="Interdisciplinary Program in Genetics, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_institution" content="Biochemistry &amp; Biophysics Department, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author_email" content="jmullet@tamu.edu" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. K. Anderson;citation_author=G. G. Doyle;citation_author=B. Brigham;citation_author=J. Carter;citation_author=K. D. Hooker;citation_title=High-Resolution Crossover Maps for Each Bivalent of Zea mays Using Recombination Nodules;citation_pages=849-865;citation_volume=165;citation_year=2003;citation_issue=2;citation_pmid=14573493" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. W. Broman;citation_author=J. L. Weber;citation_title=Characterization of human crossover interference.;citation_pages=1911-1926;citation_volume=66;citation_year=2000;citation_issue=6;citation_pmid=10801387;citation_doi=10.1086/302923" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. W. Broman;citation_author=J. C. Murray;citation_author=V. C. Sheffield;citation_author=R. L. White;citation_author=J. L. Weber;citation_title=Comprehensive human genetic maps: individual and sex-specific variation in recombination.;citation_pages=861-869;citation_volume=63;citation_year=1998;citation_issue=3;citation_pmid=9718341;citation_doi=10.1086/302011" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Ś. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_title=The Mathematical Theory of Quantitative Genetics;citation_author=M. G. Bulmer;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Registrations;citation_author=G. Burow;citation_author=R. Klein;citation_author=C. Franks;citation_author=P. Klein;citation_author=K. Schertz;citation_title=Registration of the BTx623/IS3620C recombinant inbred mapping population of sorghum.;citation_pages=141-145;citation_volume=5;citation_year=2011;citation_doi=10.3198/jpr2010.04.0219crmp" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Cloutier;citation_author=R. Ragupathy;citation_author=E. Miranda;citation_author=N. Radovanovic;citation_author=E. Reimer;citation_title=Integrated consensus genetic and physical maps of flax (Linum usitatissimum L.).;citation_pages=1783-1795;citation_volume=125;citation_year=2012;citation_issue=8;citation_pmid=22890805;citation_doi=10.1007/s00122-012-1953-0" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Dib;citation_author=S. Fauré;citation_author=C. Fizames;citation_author=D. Samson;citation_author=N. Drouot;citation_title=A comprehensive genetic map of the human genome based on 5,264 microsatellites.;citation_pages=152-154;citation_volume=380;citation_year=1996;citation_issue=6570;citation_pmid=8600387;citation_doi=10.1038/380152a0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Haldane;citation_author=C. Waddington;citation_title=INBREEDING AND LINKAGE;citation_pages=357-374;citation_volume=16;citation_year=1931;citation_issue=4;citation_pmid=17246626" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=K. J. Hall;citation_author=J. S. Parker;citation_author=T. H. N. Ellis;citation_title=The relationship between genetic and cytogenetic maps of pea. I. Standard and translocation karyotypes.;citation_pages=744-754;citation_volume=40;citation_year=1997;citation_issue=5;citation_pmid=18464862" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=K. J. Hall;citation_author=J. S. Parker;citation_author=T. H. N. Ellis;citation_author=L. Turner;citation_author=M. R. Knox;citation_title=The relationship between genetic and cytogenetic maps of pea. II. Physical maps of linkage mapping populations.;citation_pages=755-769;citation_volume=40;citation_year=1997;citation_issue=5;citation_pmid=18464863" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. E. Hart;citation_author=K. F. Schertz;citation_author=Y. Peng;citation_author=N. H. Syed;citation_title=Genetic mapping of Sorghum bicolor (L.) Moench QTLs that control variation in tillering and other morphological characters.;citation_pages=1232-1242;citation_volume=103;citation_year=2001;citation_doi=10.1007/s001220100582" />
<meta name="citation_reference" content="citation_journal_title=Comput. Sci. Eng.;citation_author=J. D. Hunter;citation_title=Matplotlib: A 2D graphics environment.;citation_pages=90-95;citation_volume=9;citation_year=2007;citation_doi=10.1109/MCSE.2007.55" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. King;citation_author=L. A. Roberts;citation_author=M. J. Kearsey;citation_author=H. M. Thomas;citation_author=R. N. Jones;citation_title=A Demonstration of a 1:1 Correspondence Between Chiasma Frequency and Recombination Using a Lolium perenne/Festuca pratensis Substitution;citation_pages=307-314;citation_volume=161;citation_year=2002;citation_issue=1;citation_pmid=12019244" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. R. Knox;citation_author=T. H. N. Ellis;citation_title=Excess Heterozygosity Contributes to Genetic Map Expansion in Pea Recombinant Inbred Populations;citation_pages=861-873;citation_volume=162;citation_year=2002;citation_issue=2;citation_pmid=12399396" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=E. S. Lander;citation_author=P. Green;citation_author=J. Abrahamson;citation_author=A. Barlow;citation_author=M. J. Daly;citation_title=MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.;citation_pages=174-181;citation_volume=1;citation_year=1987;citation_issue=2;citation_pmid=3692487;citation_doi=10.1016/0888-7543(87)90010-3" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transform.;citation_volume=26;citation_year=2010;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. Li;citation_author=S. R. Pinson;citation_author=A. H. Paterson;citation_author=W. D. Park;citation_author=J. W. Stansel;citation_title=Genetics of Hybrid Sterility and Hybrid Breakdown in an Intersubspecific Rice (Oryza sativa L.) Population;citation_pages=1139-1148;citation_volume=145;citation_year=1997;citation_issue=4;citation_pmid=9093864" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=S. E. Lincoln;citation_author=E. S. Lander;citation_title=Systematic detection of errors in genetic linkage data.;citation_pages=604-610;citation_volume=14;citation_year=1992;citation_issue=3;citation_pmid=1427888;citation_doi=10.1016/S0888-7543(05)80158-2" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Lorieux;citation_author=B. Goffinet;citation_author=X. Perrier;citation_author=D. G. de León;citation_author=C. Lanaud;citation_title=Maximum-likelihood models for mapping genetic markers showing segregation distortion. 1. Backcross populations.;citation_pages=73-80;citation_volume=90;citation_year=1995;citation_pmid=24173786;citation_doi=10.1007/BF00220998" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Lorieux;citation_author=X. Perrier;citation_author=B. Goffinet;citation_author=C. Lanaud;citation_author=D. G. de León;citation_title=Maximum-likelihood models for mapping genetic markers showing segregation distortion. 2. F2 populations.;citation_pages=81-89;citation_volume=90;citation_year=1995;citation_pmid=24173787;citation_doi=10.1007/BF00220999" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=E. S. Mace;citation_author=J.-F. Rami;citation_author=S. Bouchet;citation_author=P. E. Klein;citation_author=R. R. Klein;citation_title=A consensus genetic map of sorghum that integrates multiple component maps and high-throughput Diversity Array Technology (DArT) markers.;citation_pages=13-13;citation_volume=9;citation_year=2009;citation_pmid=19171067;citation_doi=10.1186/1471-2229-9-13" />
<meta name="citation_reference" content="citation_title=Matlab, version 7.10.0 (R2010a);citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=M. Menz;citation_author=R. Klein;citation_author=J. Mullet;citation_author=J. Obert;citation_author=N. Unruh;citation_title=A high-density genetic map of Sorghum bicolor (L.) Moench based on 2926 AFLP, RFLP and SSR markers.;citation_pages=483-499;citation_volume=48;citation_year=2002;citation_issue=5-6;citation_pmid=11999830;citation_doi=10.1023/A:1014831302392" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. T. Morishige;citation_author=P. E. Klein;citation_author=J. L. Hilley;citation_author=S. M. E. Sahraeian;citation_author=A. Sharma;citation_title=Digital genotyping of sorghum—a diverse plant species with a large repeat-rich genome.;citation_pages=448;citation_volume=14;citation_year=2013;citation_pmid=23829350;citation_doi=10.1186/1471-2164-14-448" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. H. Paterson;citation_author=J. E. Bowers;citation_author=R. Bruggmann;citation_author=I. Dubchak;citation_author=J. Grimwood;citation_title=The Sorghum bicolor genome and the diversification of grasses.;citation_pages=551-556;citation_volume=457;citation_year=2009;citation_issue=7229;citation_pmid=19189423;citation_doi=10.1038/nature07723" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=Y. Peng;citation_author=K. F. Schertz;citation_author=S. Cartinhour;citation_author=G. E. Hart;citation_title=Comparative genome mapping of Sorghum bicolor (L.) Moench using an RFLP map constructed in a population of recombinant inbred lines.;citation_pages=225-235;citation_volume=118;citation_year=1999;citation_doi=10.1046/j.1439-0523.1999.118003225.x" />
<meta name="citation_reference" content="citation_title=Users guide for new BCsFt Tools for R/qtl.;citation_author=L. M. Shannon;citation_author=B. S. Yandell;citation_author=K. W. Broman;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. Sybenga;citation_title=Recombination and chiasmata: few but intriguing discrepancies.;citation_pages=473-484;citation_volume=39;citation_year=1996;citation_issue=3;citation_pmid=18469909" />
<meta name="citation_reference" content="citation_journal_title=The USENIX Magazine;citation_author=O. Tange;citation_title=GNU parallel—the command-line power tool.;login;citation_pages=42-47;citation_volume=36;citation_year=2011" />
<meta name="citation_reference" content="citation_title=From FastQ Data to High-Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline;citation_author=G. A. Van der Auwera;citation_author=M. O. Carneiro;citation_author=C. Hartl;citation_author=R. Poplin;citation_author=G. del Angel;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=C. Wang;citation_author=C. Zhu;citation_author=H. Zhai;citation_author=J. Wan;citation_title=Mapping segregation distortion loci and quantitative trait loci for spikelet sterility in rice ( Oryza sativa L.).;citation_pages=97-106;citation_volume=86;citation_year=2005;citation_issue=2;citation_pmid=16356283;citation_doi=10.1017/S0016672305007779" />
<meta name="citation_reference" content="citation_title=Statistical Genetics of Quantitative Traits: Linkage, Maps and QTL;citation_author=R. Wu;citation_author=C. Ma;citation_author=G. Casella;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_title=Quantitative trait locus mapping can benefit from segregation distortion.;citation_volume=180;citation_year=2008;citation_pmid=18957707;citation_doi=10.1534/genetics.108.090688" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=S. Yang;citation_author=B. D. Weers;citation_author=D. T. Morishige;citation_author=J. E. Mullet;citation_title=CONSTANS is a photoperiod regulated activator of flowering in sorghum.;citation_pages=148;citation_volume=14;citation_year=2014;citation_pmid=24884377;citation_doi=10.1186/1471-2229-14-148" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. Zhu;citation_author=C. Wang;citation_author=Y.-M. Zhang;citation_title=Modeling segregation distortion for viability selection. I. Reconstruction of linkage maps with distorted markers.;citation_pages=295-305;citation_volume=114;citation_year=2007;citation_issue=2;citation_pmid=17119913;citation_doi=10.1007/s00122-006-0432-x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Resolution of Genetic Map Expansion Caused by Excess Heterozygosity in Plant Recombinant Inbred Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10.cover-source.jpg" />
<meta name="twitter:description" content="Recombinant inbred populations of many plant species exhibit more heterozygosity than expected under the Mendelian model of segregation. This segregation distortion causes the overestimation of recombination frequencies and consequent genetic map expansion. Here we build upon existing genetic models of differential zygotic viability to model a heterozygote fitness term and calculate expected genotypic proportions in recombinant inbred populations propagated by selfing. We implement this model using the existing open-source genetic map construction code base for R/qtl to estimate recombination fractions. Finally, we show that accounting for excess heterozygosity in a sorghum recombinant inbred mapping population shrinks the genetic map by 213 cM (a 13% decrease corresponding to 4.26 fewer recombinations per meiosis). More accurate estimates of linkage benefit linkage-based analyses used in the identification and utilization of causal genetic variation." />
<meta name="og-title" property="og:title" content="Resolution of Genetic Map Expansion Caused by Excess Heterozygosity in Plant Recombinant Inbred Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/10/1963" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Recombinant inbred populations of many plant species exhibit more heterozygosity than expected under the Mendelian model of segregation. This segregation distortion causes the overestimation of recombination frequencies and consequent genetic map expansion. Here we build upon existing genetic models of differential zygotic viability to model a heterozygote fitness term and calculate expected genotypic proportions in recombinant inbred populations propagated by selfing. We implement this model using the existing open-source genetic map construction code base for R/qtl to estimate recombination fractions. Finally, we show that accounting for excess heterozygosity in a sorghum recombinant inbred mapping population shrinks the genetic map by 213 cM (a 13% decrease corresponding to 4.26 fewer recombinations per meiosis). More accurate estimates of linkage benefit linkage-based analyses used in the identification and utilization of causal genetic variation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10.cover-source.jpg" />
    <title>Resolution of Genetic Map Expansion Caused by Excess Heterozygosity in Plant Recombinant Inbred Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"wB0V8MkTtj5Ke7c0hQCFe98K6KcDhDsloAYstUqzXEA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459691","apath":"\/ggg\/4\/10\/1963.atom","pisa":"ggg;4\/10\/1963","ac":{"\/ggg\/4\/10\/1963.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/10\/1963.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/10\/1963"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/10\/1963"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/10\/1963":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oZHorq3YVbazABB+lD+E57ZxA2DrnDdksi9KQFaax+N8g99NFwiBg\\\\\\\/VRwNFAkFYn5y3yDRPCCUu0BEA1Q4XHIHGctNsJ9MAkORIfxPetwMAV45K1zA8xUYLtODiSrxAVxUi3em7S7CulWHDW8Mti17mm2dULQS38GRWo6+bxWQD4K4YvCkUcW\\\\\\\/GRuXuvTnC2ph2o4Fykhrk8TWyMUgDndC8cxC8SPeheDDSjMI5DueWcFqH8Lv4qtR1BXiimC+9F2ztefuWaT+zZrPkfvScw9SHgqJ3SBOzZ547I9YeZFaEFPfiBl8KY5OQRyocgv\\\\\\\/bmdeQoICo6uhAJQSu3CL2kosMYM8vpFV\\\\\\\/OOm+QxEJNX5b+JAYIOi5touCGxbpUGVLK0KadenpONb97qsM1KwjhVGjg9R4urPvzKZKwO+ZCY17WFJ6Fv8jQIVbEuvJA+\\\\\\\/HedupH7flFKsrBgA6VCLo5AlmvQSFkr9B0wBrjg8xvezVB5toh\\\\\\\/\\\\\\\/5uYWNmYqGfcOwTZhMeeWBY03YWDgVzwsKRZ+a+prvNG4OHmlc891jslEnM580ygTbOnqWXsQFcDy51+YVLFb9RUW0Rx3diPDVmcyWSrW37oDmfgkTfGVDS9aMOoHq2s7RJu9vmquxoGWfWDqHSeyNW\\\\\\\/3mYfVmddrUUIW02cA0uZFTwT+70Mx7tlCJ2unGTeXm8Zjf9qg8HLvdY5Tzykf3z71gDlZEu6klLAYFuVTKc7B035pUWZSaU\\\\\\\/pbdsUan6WY7ahr7at959T4eu1RNTsf\\\\\\\/vGD20vFE0A60\\\\\\\/aS5nuJs\\\\\\\/XdE6drmZkdUqo0zBsgycNgkNdvcjjbZ3bsY1cJOoqLhPCkybVodYLSxIBaUaqkaehSOGhKInorWpZABG6P0otjiP\\\\\\\/5Ec4VhC5vQxBW2aSU+J\\\\\\\/39x9CV5tzu6zLvkUawJEsiEihboRDK2JSl\\\\\\\/Oa++rnWVjqmtW5TRUoeG9LDpvoSouFxcRJdYufigPo3N5nO5HzW8hA1vCT\\\\\\\/CdC050WkRmMuF+hJojsdPFGdXyvLL423cETrNxeHYzl7B+V0+Wll3P3l+p0xi8PU7cKjat1KAWIcVzGoxDO30ejWN2RF6U5qS2RLWtTyvAQV74WCxIwUcgfdlbS0OJ8XWhk67+DBKFNq7FNpJkl0U5DnYTh59Zl4gO9CtzhDrRV0g5Mb4QwqnAKnr7H1AvpC1XBQyEF9qVwyGtc0XVUZhhJGzx8T26icFLDYmeDWbeB1e2lKTycLdoCfejJhS2WWH3KhUn0BfsbDQu0u9G1f5SYK\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q+6zeHjA52i9EMkcfVDODw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224996b49428d760c58f67d19ea6ae5eb2a3f0116d4b9b2874065928b1c3a43658\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225+MlgbZ+uXuzVXOVRZZJQYMQFpCKipBT5Pso3yGEYQ890799\\\\\\\/uV8A1Cf0QZZqfTvMBUv8rxjaMFJtF5yjgDQtH1QcfZ27tXR9DWa34WC0l5Cx0fRZMWk5VVkc7D0vmNYkzO\\\\\\\/54FaJ5tQ3dMUr3sxBN\\\\\\\/Zsxpg7nk3l8YdsYSsFUC+BustQbRM6FuVY8bZpe8xxUSUzo2iO8NEAZCTnR98UePRDNjDa86xC8LSBhYQ4UfFnUFYnxLMui00ksFXFX\\\\\\\/iWTP2TSB5gmFfBJeUZR7OoVDaNBcNjQxXt+L3Az7AfzdKK5jMlgj0lwzOp5+oLVKCTE6fIRTypXYzePytk1MvCAFc1dyrH1sPGI+N9n789wapgQHndcsigwoEnWbIAdrr1xswOcOFatCrL65U+GUVMFSI9lMsB\\\\\\\/xiEoYIznHmAEtIePjROcWf1M9MfSroBkZGcsGds2Q+K67xu\\\\\\\/7Yy\\\\\\\/ty9pdcLF1O6qDNjlLNu\\\\\\\/o3GH0g8mr6nyKlG61rifTUhkU87GPqNqSLqrQXGaHEEpHUrvKFTI9EUfdKyimG5SsNaTSWiSkmhMiA87BHv3HK6doGFFg4B6U+gr\\\\\\\/aPv\\\\\\\/qiAvNicX5U1V2ocr\\\\\\\/lGU1s61GPJzqk5nG\\\\\\\/3xyRxz71ewd87gtK102MtvPcyGqI7tuQqwDhQH5yoXfpdjr1jOQijoChlD2kwhCY9\\\\\\\/mYsMZFZ\\\\\\\/RJMKYrtC5Lr7+X8jLM6HwM4K2f90MFSj7b6n7y\\\\\\\/2e4RJeInyi4+IasafAdcZBQG7iRTiRKMgCtLMvPSq0iDcei5oQvC7J9O4I3mZ7vXEyHVYAOOHWSyoOYi3ksz7CJu5DTa0jvm6+UNW\\\\\\\/uOKgaI6c9Yf9669fSiyxQkvBs80ecgJgfzX25LQuoo1WYm+u6WADhdoP913tFuMFHvlvRGhxRsK6F4hTk7CXUKo1cHef6Ov8YqNKITvTBe06LGczkR9KZ6jHgVxnZYeSShQ1eVb5uncW1rI61rIfGFF05a04xQXj5y1ydtW9bwAY2SfuuHvuZxPIPsHdp698rBPMwkkCE0V4eRmB0veC49\\\\\\\/+veXajVR71wmVXmS6+rQ51gDmdiExc8kBZZ134fA1sWWm5SRbCM04LD4YdcTRD9XA7hkvTavHQEMwNVLAVh29yxdglzq2\\\\\\\/yH+t4bsAeSQMrmaRORKBovQ5RNFPXBjOPAQOPg\\\\\\\/Zz22MlHYCHTKdmPucXg5+U8CqXWxpJ0zqBgPhHnba2ItNwUgkscxQz\\\\\\\/5Zzj4uxVyMPhjZ+\\\\\\\/riy9viKOOpO8M6UhB93UkeK1siWpAbGoRD82m3ZdMv+DmbbcnOHGpLUoRHjc8mkhwr5S8xUFSwnWdW28\\\\\\\/hRUeoWw9FVaYKUBh63WVCQ83rogirIu0mzlvti\\\\\\\/VB9dR5wj9O5cSI6IZ8gKCDDLwqn6p2UHn1e6QB\\\\\\\/D1q8DZUWB0SFNcvILHYmw07wLLN7hKQqv1lRjZc0Zs\\\\\\\/I26pxRd8L\\\\\\\/sh95\\\\\\\/Eyx6ddn4DI+CmvsEbUdiXkLE97HFRZlLpNTfmtjGtbroB+6qv96uMU6ZrfllY1EAh0JiWYayttnHYuM1SQSCplFXRFb0iyEu2+xlN\\\\\\\/zJvdXb7cidttsU3152GQJrzN7l70Q0eRE1t9dCWLb4bHo7tFxfddgS98rKFSgdlzItZBg6vUQ92JimlD3Dho3Td292Oht0MsSpexzQygLwoNnTOQjeL1h\\\\\\\/RzciQ+JQDIFn01\\\\\\\/BAeLFATjLCUqCC2UbiH7Ql3QRRmWAHOUJ9spyInKmpx4EMGU2q2I+oeEWy7N7TGzO3Ax7wmJL+TC4RzBXxyfHcWxsSEdkIv2y780rMfp3C5hjQnLgF+iLi6Z+l7vogCK5t5a51Gv0FMTIKBzb7ic2V+oCx\\\\\\\/wiArLpopN\\\\\\\/bYKoDd0NOfJpk6Lbg3BqZT8Xf5N\\\\\\\/6EcGIVG56q2VXOWNc8QQSVJIKLn0hOQeIAD9PB60fhsISkjuelF42E12dIkVtmOfDuo\\\\\\\/gjSq5cqcmVPMEK9KD1U5bG7bRYuej3rm1WsyrLgUFIP8sLRINAnsvysKaLsKTsS4tqQl\\\\\\\/dU52IG8b4igSmdCQ4d\\\\\\\/pVrxOmJ8U+0ARWO+GBCYwQiWoLoYWooIguX8RUVmqC280nOuaaragDJZqC\\\\\\\/nVi4gQyX6PkWfu+fQL6mLszqt78d9qbI8aV+86zstWOW8a9wkadRldJivFHFQzCJ6w5kP9O946q8r2I4yGT3hzzEkVrBNmWQpPuQqQr1kf3Alr8YxM4I+bsxsJteRRlaAG2XTUqjRn\\\\\\\/ffqX5wekvjoSs8qywe3FyjhUWSlGtxsmr6HiAxigdpDGbDVRCHdL6FNldYUvoEqj9EMWGYR6qluWQy19mq4gO7qALPqez0W2ft4FTuRwhiNN\\\\\\\/ZR7kXehGakuQKKTMvBMhxTvmIY53LmPTTy0XY59SQ2zK6sc\\\\\\\/d2\\\\\\\/FMXN4dsZml+2idm1tBRBcH1Nr3I41qo471KsrboDRAv1gv8uIeXSfHcSRwvEXxQoH\\\\\\\/fJ5pcp\\\\\\\/wc6TUHhleJl5kOgT9dLRpGaHZKFTD82GPc4g2IKBG0jP\\\\\\\/+WcCrk8iWY6YVRDm9pSpZ7Cu4i9AlZRjYkhruSFuUGIFgcfZZvu31rp2nQzr+OHPam9WFhn\\\\\\\/elavoq9WJLcpBnzURo8hTd9mVLu7VzRyNKRpDYKumuDrlURkLnqMyZrRGXqKEEEgsCDlFGpZvxXfRStiorRDTvtAcT6XxQbXJK+WaniPv1rUskDLcIDZh2X6kLn9Ov1BsQBz6zRDqlC2F6ln7S7EHohg2R+j8CAjwZjUP9wiW738ZLvsrxDqT2\\\\\\\/zaa8zIlriJkMMgxcArd6h0+OF8whs7QsT9P5wgYYE7kOJ5WsWBGtzsQEGmP7nGELwbIQMoJJUZLxwz7CBtPLC25RlSlQzMZvroMZbDhIa4giGXlQTz5hTXg8iQN4P\\\\\\\/Z0DxsyHEhBLGEW9KM+m9od8J1jNfEmbAR780\\\\\\\/GJ6P4TZKT\\\\\\\/CYNLdJ5wW4q4AranWCsHcGgXq\\\\\\\/FagKJfDXu7KwgT8saTVcWj\\\\\\\/yYoS2ftwAKF1AKThAzkukooOV0pObbxjQjvCeJLSh5Yl1GswYGhcJtC4ulFhc\\\\\\\/HJqNEdcA+KsyOaMHPDFL244TKsc0jyeZG5PjC8I1adjnOgzi49qqJN\\\\\\\/qzJ35hyKBRvSXGpLar79TRqX9Ic+L2Pm\\\\\\\/N\\\\\\\/MWqvFGfQ5ZTghupkwFtKEqDu+7t5r6XD9r2oiNJh3XN3tPX+Cbsmt5cvPUePfNXKXiIShfA7jR+TFMuk+c7xHUngrQfj5hVxrSzXFAokiMU49Ze4IWhDux63Wdgg0\\\\\\\/Di11bkdR1Pin8YUwaJnLJTRxgEyyxbIWKCspHqLyS9+Hl4wTjOQPxQjdnC1SRuCqtqeNcE84EBsHr3sepi0nh8V+BheGYvhE8bcCWiad87\\\\\\\/5IY41bha\\\\\\\/6COAcu7D3yY1wsw0qwHev4Vt8N9AWYcFQvJpg1MjBdGtLyhHkzrKd\\\\\\\/yAMixavI+XAj8PdsE8+wiCA9o5ndZKPblErw26wN0Yd2g+IJeumJhBiH17AnpWYqjou0fHGtKmFL0fgrz1KqfEoFDqTHtcVYhcV8eZ4iHxsogIHyKSIU1D+uh8ODKWkxhM3vkvaE89I2d9Q2\\\\\\\/uP2plKHxBmLTqns9dt\\\\\\\/UPDZlQrL5P8G0LmaNqfIQ2gz9SMScNgcGu5BOI48LvtfImOelOnUR6G9NzBI75e9QJm5XYBIj+cXIRQyEXJJJlizsPLYs4qqUG9Ba5r+iqZRueCcCXWxWLseve9aMb\\\\\\\/jsMpFch3indtJnU2G1\\\\\\\/JhBtCj5nVFfBshdHMhOwME9uFQFB+OU\\\\\\\/w07Ju9ImERVvj23HldKxM3Goetd1m7HaIk5hMPcZxLGMs3hLb1o2KQJObcccXezssTcTV1fvqse7NHvyEuheI6e1Ivra24Zv3r6m8bFccjTzGigpxhz5s\\\\\\\/\\\\\\\/M\\\\\\\/vXdIw6fuQnO2x0fPXjQcjf9Fsgld9TVYertaO8dLpdt5ZlZL6EXf0VcbcIINHIf2Jlg2fSwEljxT7b7UZ4eNaVSKAVbWZCSQ\\\\\\\/3yvinB5l8uNOJQNtjPVeYZhZ3xrT7rNIlpf\\\\\\\/4zR\\\\\\\/sguefsd1OJdtYmBkaxxn+sd5L2VyGdJfuToc9S4hs95C18eQ4CxoZNAaNF91dxK7ac7v\\\\\\\/mG1UXz7tD3mzfhEjOd5JVB8PlG\\\\\\\/rtpNInTaUAqkU\\\\\\\/fQ\\\\\\\/Q3l4oAo8MRQB5Pxb+kKJi\\\\\\\/T3blvDCHaXdVVZPWWz7R2pQv6Md79K2dqbNvZddXtZRBOLLCy+IcbMVaPm7dm8hX7vx3VkihqB1mg1FBIt6voRwcGCeK2+jOR2TyRv6LYSzC+QKkb90CC6xWkbZWu+PzF4O3276LjI1t7GaOCsTYdkIvpg+xLPWgaE6SIIVflmGTCGxIeuGbguY88BGzCEbFZV0SAzbG4KKL+qRp7hO9BnfTy9GY3mOENRBU6ozD5YkGsAKum7N13Mvqfum\\\\\\\/JQ6uySOpGVlPK\\\\\\\/Aig9tH1KNWJxVzGojQsjOhFUksCO9lqmsv\\\\\\\/Dcp1P6Wiy0ZC3CcZcJAdcY2W5QWXf1xob1o9\\\\\\\/2G0fbeCq8LhxG7pERtQSqtt\\\\\\\/YpcIkZHH8wnQ72M8tVH8ssKjXtebPgmZuCUTMts2fLxgATDJ4VTU7KESfpzpCR6lh6EoKF2sBWhIBLD\\\\\\\/fD7L5QsaS1ylVIdanFq22ZcKECNFsf0yvk3pWYufUKVE+al6UEFpKHLE4a2GAhSJN\\\\\\\/cOOUC09+COBvR2zgNDVCz\\\\\\\/V8KDrf2FqLZfumGs3ZtwsIeZhirm9nuUSytPdo4QXvQE\\\\\\\/wGzE34MDE\\\\\\\/I2jGNzv0BWLdP2QZbjLTa0Ro80Gq8kqXp+fwb9xjxDbS8hinPnfVmuqrcJkakU6o74keZ4bEa31GdJrvGboKY7KiU68xSrq7L1ZyvEPRdRyKk3uZlBZx2bBY2vIxL5r0ugDOfwj4H+2F+TExZjBS4CI304DEXtA+KDyfzJ3IHF1oeijFjStkr7D4BujoBVZM40TMsPJb6+lYwjNYZhyfR4PbK0RYfsav4ACS3LE62\\\\\\\/CuVlVOZHrn\\\\\\\/5xslDfl5ImQC3MlljXH\\\\\\\/7fgtpeMmxK2ASZfQRNOM\\\\\\\/sf9iFMRQgeUsmrM0F71Y\\\\\\\/08r0xVrdwxt7XdAg6K+MA4RBipX6cMFSCMmLRCk77YJMM4t8LNa\\\\\\\/du9D\\\\\\\/PgZATUQFArPKIV7d\\\\\\\/XxY1\\\\\\\/ea1JzRzGvRxdx0OVS3y4XEmXUcfLTxokagiH1i2b\\\\\\\/rWK5tc8hxVcekdwWOnft6c9vnV4GivFORK1XMwmno2M6kzRyT5QDg1wmqTicZFD2YpC5SNa1ORveq08tiwRXanK5Hw7XCPAptTyDb7p1hHSC5qSNe7WFekEGn1cTN14Kj0g+1OVa+WzWqUsU7U3kas2cEIgqy9nfHWTXXgHfeEhSXx084niRs5ChQWo1d\\\\\\\/w6ZCCwTmL1IWqDmyPiEkj0521uQ1amc2ZvpPE1B6kuDGHFgJbR26oUvBBoomj7LMEMN0UtMFJLkEAdxfklLC9V879mzpIGHA4KJFW0539gmHvky0N8JRGHIyJESaO3lD8J+onXG+39E7qp1Sfg2OPCYGKaF3fVc9LrSCPfejKampYanmGg2L8wuQasua1uAja9bVnFeqc856lRddGS776ZKvKuocHuxFyQbnCz3htYvi7BQxeTBupWbOWP9TtfI6KW39l6Nf02GKK3y4IkcpW9MM5jfpRJj7t7H4t56FFS90x\\\\\\\/CFwS9aYefAkcZobh6Sgq25AMCAiSlxHYM0ix5cgl\\\\\\\/AHP7FVY8Lo79Ko7R+LVhf6N+RvWo5yEB8FfnE4djIkU7KHpF9JpEP7PtcUaIVBTT9lBRH5Q0oer8jdwNW1iRJS6jPDtMWvmRi+f92VWnrh\\\\\\\/5649HNVKveZ2Hkx1JA5HNsyey\\\\\\\/dyC9c9efCGNKjrjSoh46yp2Th6hQm8Xrrqc+18n4ccoN2En1HVuzq5p26eDpm3WWH41CmQqLu5RRUtlcgevqX8Z4sB5Z9dakis7pitT5vNYHO3JVxNKd1RPj0bIJveQ4UagCCZu4NRGfMJJXie8R+g6zsfmp4p+bbW6D7\\\\\\\/1EfZgrfxPy0OWtJvckrxQ6OuikgKwpx2OhXw7JRpxcVEyoX2mEcawkzfIkjKPjKJWFxIDLAS8vP1COvFbHPZvSk60ClsQzg6beHJfh28H91z9uEms0OVcjZEqTIx5js8jeEno1uLcp2lbOumwhbq9TAUyDp4zHrwB02VckziufY3Q4mJDpqkXYtt1LizTnaXkonT3Qa05nbWOBzV\\\\\\\/rVxUekIYlKV3v7+mm6SZRUHDhl54ohi8T3KruyDYihwJ\\\\\\\/3xEc9XE45zsX\\\\\\\/iPWMoNSRgv80y9WXPEOla\\\\\\\/jcFV1iVdaoCLMSAb6pkYv1NRQWWvG4X0cDo7MCO2EQuTTBLwJhTWgEX4WNd+tWkzjYxKTcXn91WO0H+BlCGQF9\\\\\\\/qJc+qlBQuEWyqC3wVdtE7zDQJnv6ncdJ6Hv8X\\\\\\\/2BsSz0wKodhKeXJ6xAQyULg9UYNkLpXQJWqfWdJZ2OVqoOt4xvpJCejLPsymSNxEuARrtjYG12ijhuDvFTKb1BeavywMRrRyrqTQTxB3VFy4fcvU2w0Bq0R9HaQ+vCRjASIqzraewW8fzBk2kYhPht46A3ky3km03Yiiz0KKRfbzQMQBiJwQGH\\\\\\\/BE7eBywSruXDpiKPBi1NXLk7\\\\\\\/gFjW+XFE9I4NSY3mxrJ5nsCWj5LXN\\\\\\\/Q+EIdevsp0n5dnrQpSbCOlCuF7LK51CS3nhBA13rkesq6GQHYYoF4pfMfog5NH+5UjvVLL8DebJTVnGOILLzLxHhushvuLoc6YQZdNT+KNHg4CvrWTfxWdtswp\\\\\\\/Mz59UOauJvVhE55pmaeCAObeMYtskLpf32uJZmDJlZIrva9LTMgdLBZEt7GilfNicrrdt7zviaOOhlkAhVwD8umkpIEExIEw3rBMYdSQBt\\\\\\\/OEev9ocbtGZIWOQn7vK14LRhtfBMFNTNbZ8fK0AcBFHzSQ6SwMGJBIxyYabcfHvlH\\\\\\\/APCZR0JdP5dIacdoTzwSU0L6Cqc8XAMDG4RIHASiEOzEA3iFdVUGIsmUgiUAMwtwF2eu\\\\\\\/oZq\\\\\\\/n9Spj4Vk\\\\\\\/gDzX93rPFYFFJ\\\\\\\/1O766b1D2A1I3EendIZL\\\\\\\/8domeXjGzpAQiEBx2awtuJtMIS+FjSkl6UdCwd8YJe+vCC9B4axHHDTXa9IVU+BuwnMVvn+\\\\\\\/8hEcNp+wyYFIFtBv2WMur32TA2fmSw6\\\\\\\/BAEcTjzFeQWMe5mI3AfO1m0i3gSxOufpL7MDp7rI+pRbzEgYUy5BQo1fu0quwKlMWMZ8dh6+FHEMnKCVRLscxsPSeQ63tQKtD6VGyc6OwVM4k5zbHNBLi+pU2Rlfxkt2spnZjkqe92UmG6LjW3RdPJP6RFndueNg2NiT6ok1tiJrWfJRo03iVrdyoPG2fk99oBg2+E+RB37aqLDy0DhM3DqFmMXm4by9s7UzpZm642hx5RKSAthAuvbdLgzxliZWjpmFpDt3KRogOAjZYmlq6nlWI+rHRWoYBk\\\\\\\/4Y6VqdVwt2ODzh3AgGIvaGXMCewhmBHFxAa70TIy6np6Gk5eU4DYOENP2o\\\\\\\/YGzAeito6P1oeAG\\\\\\\/GitfahN5lzgwQxLqnIA9bBIROGZbAKxtLq3EuX3185roWxbaq4D\\\\\\\/86YrQmK+tRXO+sDf9vHoJ\\\\\\\/D62oTSFecrA\\\\\\\/TtvH8uBM3xg\\\\\\\/77po9UmiGZyehJNfZym8vISNsMeKSOOKFhhNrkx91uG\\\\\\\/m9HWpKP6L\\\\\\\/ljRnK7edE4Y\\\\\\\/E9Tr9OMbOp7mDmvBVc8mmGdHW9cfNHXZuu\\\\\\\/3a8pnZXY8AGHj0Prq+3WmxyL1DvJPnWEl9Br0iNekL2qqra13liMm3seiGfs7qT5KISXc2qjb6NFzcNm7Bo1ZixhMQaL5y6Dei9i7pb6NjmJjkZTuR9fiGzYzP09SA2xtjGwQvKnpYhGTblsVG5KWqEctvgRBt8gSCbVJjxqjvDKWTxBMU1xnnzup8GVtOciqqe7ZYQEwvtBZ7i\\\\\\\/N2NOezLSpMBdBVLOsGxpSsTJPe9lf9fTHBapnMyrxPU7ucDG7VijRtHNAOGQSg86XFGC\\\\\\\/crZRcM8GomxO3MXHRUkDrCc1aWsXJxPkpmA+L5so2J7h0wLyepYZG5tgECofV23LvAJg9\\\\\\\/EFtDtUnntOBQlbKBgIDuC1mG3ffNUQBDI\\\\\\\/c\\\\\\\/D8OEaNkSAUXwlWtwWzrWuDXIyt\\\\\\\/z1xOQKI3Cc7blFzrto5dUuV6jsAJyzZVhwyR9pP\\\\\\\/5RIATTHrc6bA\\\\\\\/F6BDSj9ZyxeL1swgf8MLPX9vVzoNwdlPF1RG4BdSfk2InQBVUNd+TS2bqyntGCQudloI1r37CHr6AJO3r6Nh7GbFjxXMWOpfR3Bg5t\\\\\\\/yJtyE1nKbIyUcaMfUH0mShg1U5obIdFqWlxNJQ9T5WlfSVf\\\\\\\/JsVC5gt8SbUVb5BPoAvzwXJZsG0tUkW82pDWrIh9m\\\\\\\/3Pn72FWSvRHD09PdMKidqYWnpxUcXSp1dGTOOsRv3DnzU0YR5b6HN4CJKeoLBZfsqTTgrrye6LD7DYO+OBugeUa23vJgLLh629Xk1ix3gfKCKlwMTxJhrll4xyR7JGNbKCevodSfceNzxUMq7ADSN3quG99tRpWN60YThO3tWd4160ew4MkmcbB7s0tm2M+P0XAHWaJcimt4JtYIFfhOvHxnHe5eSIwNK7i5XsdrpBq2Sy2ODA4nPp2QAsKAOQwyibOpwH3u3EP52npSnQz0B1YVNLhApgsJqtnaGyh4\\\\\\\/W6owlD+ltP4SLGuIvTnzti6lvFTsGePpvRUM6CVKZvBUssOO8RLKT0yiIx6ik8JZySmt3h9YwzbnPMl\\\\\\\/SoSrXDlQvywErahokYOHGuNKeQphsqwftS\\\\\\\/gh9nwJe5Ep1rJAVhtHiTxn+xlh\\\\\\\/dMAiBuysB+3fvSxlEZdcNYD8\\\\\\\/foUyPkLkQIAHHij7Zk8RXemE7nDLdyNQgzUE27b8mMZf5vKRQm6AvAlgqCn3+jKzL4gQXru3CDoXOfAusq\\\\\\\/sSztjarcbQC5XKJFEEIJrxzqV+6TJD+Hribnz8sGy00OL6qqeFR1BKqyYaXTOb78kRM3QiS2UC1zUfH1cmYvaQyXR3I4nnHeoAse3+uMj8uHcm\\\\\\\/rd5sjDLPzCebjDTUoNxUsTwt328fXMmfE7DMY4Callz3ofOM7BizR\\\\\\\/BD1WWXfmVlNvcp95uy+lv6iw7Qo211TStSo1nqsKENgVMiCaMAx6mVV7Y9ucOSCt405k3YrpVFfE1uXQWCsZXhl3O64mZkYGqZ++\\\\\\\/GzIhoJ00Yetdb6CjyBa0jRKn3mytHJyUvqFpiAcPZVvYX79p3ifbJbfEe4XhagTPILaIBbz4UC0AZaiqF8GRDUqeY4MaJ2AUnscj9mlm2FnPPaLMocVrI\\\\\\\/A59XaHoQ\\\\\\\/1qRsYRltMiXz3a6utAiTILy91hOH5\\\\\\\/IQyJuPpkKkrVk0sRTbd\\\\\\\/YD73J1aA6KPlUxQ8boq7gAJzXjMreXP1X5mDlDzGals3vNGCwo+1rsCoS5jmq0OJJWtPyF3\\\\\\\/vWD+WjYbqmwT84Do8USCIahx7eFf\\\\\\\/f3qMp\\\\\\\/d2iz0LKKcQAvouU1zQVoOUM7xV2Ds3rxEk+Wd42fF35CiKK8\\\\\\\/23JauslMj44ByUI\\\\\\\/AxWBx2JKhT8BE+Cm1E\\\\\\\/LkVhiRcCKyAgCMT1wLL5JMYVYoYZsB2CAO9FvAIBp6nuT1QtIwMR\\\\\\\/rUKw8UUPtbYL+odInp6Kab7BMgFJ2wOOw5J\\\\\\\/gk57ZKD0j7HwUWkcVs8YzCpTQ27nBIEOflX0GAqa5PF\\\\\\\/TkDq+nlgdgjyZWADJxsaL1jiYOikpmjZIgO57YMdiy4Yjj42y0ZRicl0xOMdAPsdzbfPD\\\\\\\/Slr5Z95\\\\\\\/6N5Oj0T\\\\\\\/J0jun36Ns5k59fvzzx7Ble2EnJS3KvjM8stuXqrWazc7AEN3PI9Q0CBh4oUTCeMMtDCwGiYSUUrPN0QYPwOEPtXva1YUvO+if6XyznuIUNFNh+mOmnMkyhpE4JNttFd6U7w8JTidyJc2s8vC8PB+akNKiFXEOVpbDgP8FkRzTfsJRMUudFzm5yo72OJErZ1Q0YBzLXdC3OPBLckU5Fdk+PVDqYudDtoyN2l1FISieQspLwoW3OOPUCu+Ecc\\\\\\\/dLQ7NDgWxcRmvtPfTF6aYRkJuzYAdlmeYksRm7zssS\\\\\\\/0HSZylBXf\\\\\\\/au+odkEZlXTS0ViibuTkC+VQrmnAdwSloWbU82Rx+ViINhhBTFEjWdRFjhZc+2aWuVaAMlUFx6VMRPQ9YtmV5WZk1sCyBtgCAyylw18Z4wt4UL\\\\\\\/23pEwJDIpe01PDCj469hQMwr9HO9wppQd18ao2aSguU817P+Wz83cYbg2pAMkYn3REtJ9CKfRYl4DIb0STioKskH8BysQ42xJMEb3iAN3do24Bo\\\\\\\/aktP5vm2MWZFOudXkhyPFHQjBBNn21d3BLyzgyqDtz8KFxI4dO5HBOqwxPKcYwzaV3MnucS0IZ+iwgU\\\\\\\/cNG90384kIHrDjf4XH\\\\\\\/zu9fISsWxGPLuoeXZ\\\\\\\/bQ9yzu+qNx9Puc+VhfVKV5aILt98M2Y7vL7SMJ+qoKRPT\\\\\\\/amL3RJyIuGNx8qLGlljbz4ycKflttfBs3aJ8Jg5Z7B3p9Ak8XBev7Pch6VjP+TjMudi2QNeeC9pVT59CwO5eS8KAjg97bE4FM3lqh6p1v6ZL3ufm+oCzOqL6hENDfkFwc+4og\\\\\\\/zZ6ehyEPiOcLgzfNrnAM\\\\\\\/ETG8vajYf8GaXZY7mmsS5BuOm3S3A9qAaqVMU\\\\\\\/vEcqCzQlRJpg7hEUmXK7vDaj9mYizRLlp8o7oGA5WVsTuM4v9T6\\\\\\\/x66EXDuCrPPSTFHFu8RcXJK+UkC0IT1c4jmYP3p2wHf0Y2Lrcb9q56eEEYD0\\\\\\\/b4fhqUSvD6B+gri\\\\\\\/WF\\\\\\\/am\\\\\\\/EplfYR\\\\\\\/yWfW4Pj1VACm4bjfTQmAVVSg8i0ZwWPpSMJeQxMt9hT6I7hhXV4klKLxAq64OuQ5C5AdZkXG8yX\\\\\\\/Kr742KbxSbjojaXZewvb7KUncdPheXufpycNntBM15L6cq256izumG4QsNRNpiM4EAImOHFeyF3RM0364Z\\\\\\\/0Rym+i6Npg82XTGg2nDpnUiaPDQ4QUVEHKMWbs776ZnCjfcysbAYFWidFAVJ93UadkZWBlNa0q13uJfZOjjqEiJ3KtrLMmg+1cZTZKkXIzPn3rSmh\\\\\\\/PjwjD5xtKiW205Ghh4DB+KM7+Up7FAGHkSpkpSRfSXtSkBv82Sz8edZF6pRNyv8t2YW5gueyXvV\\\\\\\/9kyGr\\\\\\\/uTrUGNBPOQucak\\\\\\\/8KuCio3dNnugQl1fRqO6FDefocf4\\\\\\\/TDEgCsHJzC+arjsQ7OW1hQ+tak1KWO9bdYPnONNkZ7JV5ntIb1URcq7xZgUqHgP\\\\\\\/G7zn51JTdrD77VDroEPEj4mv7t9ysHy8bKJSQVAA5LZ3m+7dCnn15QTTVdhjPe9CdflI\\\\\\\/GiLmNkqCSghSBRplDfEkovIc9WZxhH63ghPYmCgveJz4rdQnXmbfyNedPWOvJAfv5yeTXD3tz9GdMNBhOtrwLwHPrup2siMgegM2AFD6Bh65Cl9kcUWxsuEOuHbVrCDgL\\\\\\\/q6EVtjIMkHf\\\\\\\/fs3mmHHnDxbNufn6Dbp5\\\\\\\/6jemUoZE9pAyyjMqS8s0nfmH3XLKte3fzawmDWYaRD2BmVkPtlNXtkGtVA9GpDtrwxKsKi7SsNDfR\\\\\\\/jk4zGCLq+eTgULWxaDbyRwR4UltzN\\\\\\\/rAKsg93iJi8vnKJ8hSkS8meA7fQ\\\\\\\/vOwRYivIlD7EgUF3nSp03dQ+CyG9sPyDGoGpYi4DHgxuQdYLTfUcSrHaALXMfI9zhUdWmwWwwgzguNjo\\\\\\\/ceVhKAaSFvtlAVOOwUINbsjo0TXCxonIO\\\\\\\/sNiJDXu7LOZKPpuhH2NenA\\\\\\\/16iqIr4kuMsP8m5GB4k78ucPo+DfPCct92K+ahX0rG+na8cDHUy5lf5jMFG6lm2UhJy3YBKjfcrwLSKmKQJEYyH9XS5k8MfLk5WVVG3ywlsc6J3LPd2v8gE8JHKWj6O1zvq75Ikx6R3N\\\\\\\/iNCD80s8dT2zS12xfsgxuSv0kqhdMRv0gU7FV\\\\\\\/iS9N73ttSk0ZCvY4fyScXBqjl4M8BDWcobRL\\\\\\\/KMDrkL935BKQRigfkQWIoHObyj1ijZ03pz9YjgRifXrCAskZW3s7e5TazYjpQ\\\\\\\/Vcowx7ZF9\\\\\\\/ES2Nxj9POBySRi3O5oHwnI48JksKryrbzYoYDczcpOMHFVPKmgSkApkzeyn4ufH6bY+h16Cu+WqjEHNaV4h8sErED5lcXW7SavRPm6ujusc8vGPzizpITpS+8mwF\\\\\\\/nz+y6lA5pGFicCjAlpzW7ysJjK+dMpdOfaoCxyXM2gugmT+fq4MdWDoOkO5FsKMAyX+it6gZHZBw4FZ5WOgLiVBt2kpvlZ3A1bmR7B+r4krZ6+OpeNTHPNtPYoiY\\\\\\\/BgbjIs3p5NUUnIkQhXfeReqvIS9WhzBijxEaMUHFNyMhmuuC75fSOm7uRwmjOda6Cj9aoWXxklrpqJjSaXWvJYhwqdWAdO9oVcsZ5qGPD46EmCySRRLleTOHA8+NKFwujL\\\\\\\/PUReWnprSe8EmChl1SUe\\\\\\\/ylhCoohrxPfZuFUO+bqOqCIUMGaskvW6S42Qo97RVt2+HdbzhsP\\\\\\\/IYd9lkJSbaKNB4Tx5zDdKSAl6BkG7gHiaDF\\\\\\\/b0\\\\\\\/Nlue5xVlkVtBrjf2cZwrpulxghLmQT7OS9LytJ\\\\\\\/ZpvBOPBwxEwcvu7mmhWLbKr9oAtFOFdmBP2yzYR6TezGxkqPaUXfGeBEh12HxdE914U1gr+5DEmBDleLcacRyTbekTkH4mn7iUp2G4HETjIhwHytQxRIEzqGZrPgyg9EtjhgjzoyhsoA7VVo5vXh5Thiq2dohfZVeRbk9qmH8vXeQNDDdpz8VyYLFsMOkZIF61JtxVGd5Q7WxEP1zH8xFOTh8HnRcD8\\\\\\\/UWXDAkwvdb1LwQL9lCHdUh4PDx+fFgT9bnO3nPxjH8rWBOdgdSD\\\\\\\/3k5H\\\\\\\/Po3Ni4PttEDvfUYXnLcuS7qYCUax2I9pFFApuEhcxtLm4ec0mOhKM64oVl8uu5s\\\\\\\/55pB\\\\\\\/z7Vh09HhvtjOQ1q4xRNvotH5LJljBzJe8RkkDvAo57KebuHJbJm6JhtnDsum3GpzR5I45I8N2\\\\\\\/H9LAVThhIFfZAT\\\\\\\/OWR7qx94upIaqqZ4UK0mLbK9DoieI2pTjrQkklLKYhk0eWXI9gxbYIAkKjOdVA+EGRSeBTB87jfTMq7CVPG4Ns+pY3TtzXb803AC65OBa4H\\\\\\\/4YKZF7IXbmJtoSM9VOLgr73wNL1pj++07Y6SXCSzxNrSomfBvk1\\\\\\\/Ihecj1E0jBERZCZUTbWMm4Kx1fUVRJybIPAGR+Xz\\\\\\\/KJopwFeAuny5axdXiordG\\\\\\\/ksgC1NuqxocSZMUPhLwYc4MgTmAONvHwbQIEjfw+povseEJ3C7WLZv4uslef++ynN1zAgpZq0kvHBcrAzsX2pR3ifY2gX6pjS4ssayVMNRVf2ADJHRG\\\\\\\/rWGfV9AflZOoIOyHWO73CFQa6uV6N3zF7AK2sHkt5CiAqatGWW7xvYMv1JgZOHWzMnYq4FQ5TEb7lNjDFRMQRZNpqWYCHYqs2+iWbfOHIrM+pHJ4H5B+PwyeBAnTVoRbfN\\\\\\\/Rk7ELhp3mlzI4fchgxbEoJceSOJ2IfpzBbj5YDhUmJ8X40medLCQZV1ztSuTHww8Fp40jRoJ6Vk7k7\\\\\\\/6TUvPt7kGdr39eXzQ6NoEg3\\\\\\\/u1oq3kqUTNo7HDDq7lTSz9RBbsFReXvgGgSfLCGTCTEM3jF18TSi8Y3rABNcC1uaRuCyotyI9cZTmmBHAh4hLpDzzEbkKEiN4XMDllWjIYpRddxIBDwmARdnjAEK\\\\\\\/ilehI0kyn72BiyoYH7nVOtFWudwLViVd\\\\\\\/ehGkCfItfnXJ9aB4oJNSe2O88beNE6ZcwT0nanghrqVVQSCPR2XgQH3lkAbP8OwxspzTtnAxgzkwJLsEJoW1BRmuH7f8qHjhFXK4WOaPyMub79uP2wpqcT1TQiHWd8fzuY3KMX6xmscX61iwXELQJLy4uJJzmsHZfpKh4YH1YSKm1KICr664+YqYkxJ8r38QSpChdk25jWB+Z4cVPY\\\\\\\/vueBXNCP+X2em8ZUt11P+he8I2PXUN\\\\\\\/kfOFV\\\\\\\/aMekef0zdT42KYByXAxFFffEdhXMoRlyTJ+gGfHgRtKLwoIbO8U1rlCt2\\\\\\\/i2RCESYPQqphWv8+BR9j5DqVfHxofgfUs3xji+bk9bJGeMvob3FY01nE36UOMGK2l4y\\\\\\\/fJpqXsbiRfxhoVhpQJBfSY9LuNYcsOjSpqSicPvVwFB+1G5VGzz5zxjg1OOi7uTZKLpPWoRLCeKyoKKQilw7yqUaG0ENioBMfPWMu\\\\\\\/YOZ9Efd7Rfx+35HVews7BhqESW1ehcKz9Pzob1u5Nvz8QKv8mTA5GWm8nFprGjXJt0pH\\\\\\\/\\\\\\\/OTgitNm2Xynfg4lETCcwRiUHfF686xuSESag0EhrjzvRxDXsISZWJqpusqECLRGXkfu8YmV6FhCOAaL+DgyneDpp5icHQ0fbRTMaTbSttqMiELWgUqCQ0NKXnaBkAhF7fcMhha5uYr0dWXyE0Lm1U98yyAbhN3cY0yylMgfWdmwGz48KNBA2A5xd80cZf5nqiocrJaMRVwO30HLvEUukEXnrPAaRs74wexaqGvoNW9hkH8X5y33JxBfrBqizImGxAU2g9mHGrgHy4pGE4AiOQJX7kcK56Dg0ZIQTVZtWNULl\\\\\\\/We0\\\\\\\/8\\\\\\\/BQcNxDgx6WD2Sn0JEydylCg4hbvv5TXyMHEFidDjoclzxpgOUl+2H7OXs4WxJ9u\\\\\\\/yVsatWk003\\\\\\\/hJQc14DVB1aRNi08YmkqAkMHjiYDryhu7I+wPGm5XsT71j6UfjXQa1aKFQEfclPQqyZSCIVqXSBb7mjomY4hMtufoATY4di4T8E9sNx0QikmVwBrJ2JcfAjSdXQR38x3CVQcgHSW50l6D3RqZuHTXQ7mub9uFr9sdGvqkm7yu7EtvgzEqdlp\\\\\\\/0QUeqFcvndDax0ivpNbz3xpBvCkRe+em0bv0JpsbKT4+DwLYcXouM4gu0wrbhhGiKNDuawaYqSFXJ\\\\\\\/1tUDY7EqzXwv7e9tGmJ1HHAM9rLg9ce\\\\\\\/oE13JsUOiqzf7\\\\\\\/brNbVTXCZIjGmQiuDpnuRyjaiZEQGbv+RX8X4nunhwia0gDGRQw3STa6FlaaUHIbXQWES+z7sZcKVsUb4HWGQ7eo45BJD11LOWrJjrdAVLc35w3qg\\\\\\\/BOD1taUhAJkM7t168V+AoKFrnt49Ebbn0ZIMgciwmxRBA7dq\\\\\\\/AK547W8PdBWFMuFmIqXS6PR9lZe8f5IkdopfYQDkg6CpaVv8jNOimyqeMYGXix6pxChYRsqtIcmtK1ZqMFMStFU+itcCl\\\\\\\/3J3R5HbiDsUtCTAURpC1nNSLjAy1wW8sjntrXgLKXu6f9T0qw\\\\\\\/uZpSAmu5hMBjoBMAg4hN1k4PLOmtIjZpOysCi7zbzJwnxOoAoOK7MBCjr6ODdUiALtdEXGJwqJjDHb6C0BmCXozJUG7inH9Dcpnc+PuFw9hW5HZRevMfy07SYQwA1gOmtzc+oPnJlKvJh1drCtQ4B9zefys6YkoiCvz9\\\\\\\/t3gUF1zZ4Y8atyN\\\\\\\/NrT0ZPVs49C49jx\\\\\\\/luS5Wg4knMAY7ksY4lmGb5CnyiqNLpRpREJxcfnAQ+VI3jnrn+9CYNqScvtSJo94Qo5rF7ewPC94DsiRynjw9943LkI8s1X2W3ILefd3iUXJx6YKMzJlSVdA0+skeN2bufOehOzY40QCJ57hoBo724pPZpdLymOcOdRNcpy0iDC2buVyhkYMmoqTq+bxXcsu\\\\\\\/BTpwqDZJ9nZEEvU7KuaiT0xRFYmFD4ho7s+F9H48+XIrYYHQL+YeIz\\\\\\\/6Wf4CBlAA\\\\\\\/iDzBTdXoiKwNCS9aO\\\\\\\/sKh9KbsGQepWXbLkMj\\\\\\\/nAuD02DGUgZ0VjMA6DraqAbX0w7n72ied4SsWUqx5+tOypFwbT0iSWOCiaY1T0RSotL2gufXHOvC2ZgpRU1kDpV3kQE\\\\\\\/OC61duA+3\\\\\\\/f7\\\\\\\/L7tbbWKUOABlTe39gH6c7krP2Hsokft4BTvHamNKiQbbBWKso5c0KFtNBtJcIaK9CcTYfdO7yv8q9UsD9sCTOZG4Td5qOUWmO+kmqpc8Wsbm1whmVgnTetEX9utT2D1li9SBzYEH\\\\\\\/q7dw4FGvrZvjfBoxuyfDb0Vuy3m0qmroSV1gYgOsK1bQaCDZ2YtGlcuhmdWzQ4r2JTdBBEGvN\\\\\\\/SW2gwREiGggs1COGdmPgqCX5+deeGEyi00UpXLUoJPRmiQp77X2deyF4fVLU\\\\\\\/vxMyU7aZ1BXRBsomndmersi0v5Bbdo\\\\\\\/Qtx2Ww5bmZhTb6X3hxWO\\\\\\\/QYHIMu4IkldYNYMEq0X4kRqlcKxNslNIJTXElcm7tgBggNuM1nl3znkwfzD8v6hxYMiDI9ghVviB9ngNGOB2Znsm8Jy5tWtOo36UCY729lmXFsWEPfRM10Y21Axsnr5QM1+FLE3nrJ1LEsVQBnPipMfUjkO2Pd6AAcutSdeu9vBfZzGEui\\\\\\\/ZOGCKkGUhUBzm+8hoqg8a8E9FBO36yuNWy9O0FesTGZPvgw77fA6sBhr9swmGwbbYmWxExZ+NMGG+L0C9fsqxGrDLF7+JScpAbDSgyIfGCbzx+iTM4wAOCW+Zrz65rPsOSeASEJzui1993v\\\\\\\/2PlEVnT7L6fILhwltRrJYC3sX+AAZPDZSyWX336NC2WPBQwpYEynRFAaKBtJ6zz6R+ryuCg5rNpD5HcF4RTmIzvMxoBx7viEqxAY6tzrcOxQRtY9AuEIucW4z03eWCwiARI\\\\\\\/ThDETQCGZ+PJG1Q9af1Agg75BWTfwPL8znKObCehlX2MiA2s7Sp2TbLnrnzRIQtvuFLBdy+IfOznm0xSm3Nt\\\\\\\/fuyFPGDkqU8TkEpIfEHiIqBk25e4E\\\\\\\/yDWcKOUITR2XX3LKLSaPh9fa2aPG\\\\\\\/nFKZDkZfoCFHOrof\\\\\\\/PbNb7WwYxS36j+fL4REeOi85rnITWLm4Vl+G5dcTR8GmbZdyGqkAGrp7tkdkOwcAQP3TWdQCW58xjyaD4kK0T1ehkUN1xEW1NMrEye7zZv8MObjC0\\\\\\\/RyG0s9wuoAHhKbV\\\\\\\/ET2JazBV7z+qtaGdFbyBA8mEIwQ9CUSwvMxLDYezxdOvMex6ywqBwTt8XKH8iaX5HL8g2qdyJScxPEbTPJwMSNlSmSUb7MZ0\\\\\\\/IuRqwLjTXMSa5W\\\\\\\/gd8FrQgzfAoewjiGER2DMTSU+cX4HdJt+gXApa1C4rtM7AmKRiEJ55LPuBTxXyLN\\\\\\\/DCp55iljTPH3\\\\\\\/pwAiuDQYz\\\\\\\/HcHsAEwgAxBjQr3FrUlxbBYN49lLUvHBgJGD\\\\\\\/ck13AF\\\\\\\/VCYjiNfeFu4+xhSnPCGIl3SMc5rfHUlotetdgzatI3yYavXUK838VF\\\\\\\/CMGlJ03bXOv5yAOdLm4\\\\\\\/LAeeOm7t2PZ5rhTU8EBg5MbKjUCs\\\\\\\/iwWqfJdBwN4+s7ILJ8Ez5cT8Z1XAi3koQqEDVAzLMVP9PSDoFzVz4DasJ7vzE74brtFxlUbQkpaKh9fsnVDcqqIMbj1vNkV5bVGcu37XB1RrCANTBytNnNmU\\\\\\\/gZGYc\\\\\\\/YaS8JBIpRmC8g\\\\\\\/ZJ48YNdZghOc+\\\\\\\/ctYQqN6SDqRZNojvJdoC4sWOa67rjW0TNTKbupE8vie5\\\\\\\/s3u45XxHJRCLoP1IzGqYf1T68K2HYZfgg57AYagczYlhu1AIRI93SF5qZ12wHVbRVgt8YeeDsOJmNq0HrhrFigq+vcsvxFzDAvfqjZPRJFD9iuD5HeIXQ+qhr144S2jBDPT\\\\\\\/sD\\\\\\\/atA4PJJdYtWzwB3yxaRnu0KvZUUySGSLUNe5pZeyu+TnFUAhkG9kfOvv9J3ucorNU1Upn5oQpI56nqdwU+1q+9+mBjGEA50ElVCDAT15\\\\\\\/y3C80tHoO3N3GWHEMaAxMdygzoIcOsAIvFIcrHfumG+bldcFAL+CbYxJC\\\\\\\/dzGSl1brvPRlI4gql8+jb0YZdmI3Q9a81MV6wm9VPndQdSW4gMcNzJNVB7ihqbXdUa2Px6FvLRQ45dWB8tCBqPb7NAxkJIJqJXXIIbzJihKt5Bcty\\\\\\\/D04XR4Q+20LU8wH1kXJQjImp8gK4kExICzDILFxmSxdUGKuP6dijCQyEB8C4yK2Pkao4JQv\\\\\\\/9hM+vOGyvckrKwLEWgkh8kvmWr9XnzhbrRBG8hnVaePCu5dKYBRwWsbq+N2v+LTydX\\\\\\\/UtVQv1tV8KIZhZaeGEtSyaK9\\\\\\\/86hVYdvIGClyAdAYdjIpogwQnC38j0DKU\\\\\\\/YE4ZNOuNAL1KK3x3MUb+EeGNhPlt8gErOMTL29WKfJNYFCdr44g0MF5YXqG4Lg\\\\\\\/WbyOtCufCoY9U4e+L8X\\\\\\\/CWo7TR6wSm\\\\\\\/eGjNIeMNClpQjC9MSeT449QI3O\\\\\\\/yWziHJdL5vPr4bQS+qSacQCSgRFbmLY1RA5R2SWkMBTtq7l3Uug9AWA0FaB6JAhnm0vqXEAbCwCP+Vlu3JInBdyg5YEM+1\\\\\\\/GgGWSgDoadtp1ykpgVLTBIgvDPBeCl9yRsBA4Q5KY\\\\\\\/UIB1Yp3Lmh+UdULLkzm7cGnaSeENdUjBbGpjuGchjP+7Fy46Atc7MyjG\\\\\\\/D8FIVBR+aKn\\\\\\\/WGNhYh8ayikUfJqCd6iCqWZ0ltvM9jcpY31NMNswnS2oXYQNdMNTnhAsngP4gCuDE5apP65KJVCVZv2dwTWBhxejJXzh7MIa18y4FNuyutuqu4KdrtbJlTD3zyeRjP65UeshK3qrNfzXEh8j9Xdin8ApQgzVD68iy0vIasyw9RwBWffVsz1YbqUZg6Ya64u4739fuSiMFChMZowDhvZp\\\\\\\/wBE6i6+6VLWR20NIQRQ\\\\\\\/QnMpYXHFVifX6fAj35ErVONJUI7dj9QYQAiTiRyEMc3RUVKSVSQhn0slYrBcPwD5uNDyC\\\\\\\/J3Wz+x9GUXKubsnEeFUgcszLEU+32VbiaP3InbzlNy6CmQohkaeVMsOy9EsVci2PXtPEfGGOpkOMN7DPVj7zWhnTzy2jVlq\\\\\\\/SOgd5at1yYbx6B1WWi6EI+pv2EFdwGu6vzM4gvT3O2M63BlI2ZTM0\\\\\\\/jasAzfhOjRAUqwFhJXVm81YcGQePd77qLtVEFy5mt9w8bn+fM6C1QuCZkbHAKvQNPI\\\\\\\/IUDF3aKUawT9NT6rK8rAFIgDxqdBjImMFcZbZYos88pYC6BCj9E53YNxfAq261nrcGfbK02CWtiGWYvS2iVSTic3df1n8F4LRs4dxqsQT5noyWM4Wk6TI68D2yE\\\\\\\/\\\\\\\/8zSMzZHosNAgQrML+tZu4PZZDoWQn8exT1AaKT+VbtKdKLZY2rWKH7\\\\\\\/aOpzXRw67guyTxABRDBZ\\\\\\\/tm\\\\\\\/3yqK3ePZJqytg577Z8jkGvx669oUjPZe3ufq9TmRV\\\\\\\/ss\\\\\\\/iq7BQ+Yk1+CaQCbhMo0k9AcnzxqCGEhQzSWwCHeZ1r\\\\\\\/YYNbEfgRHBn8iuLv8lFrebGY8xvAfeyYPiHIl6i07puem7kxrExkOyYH8a8V0WNyaYKvQZFsfV6M8w7BWQuk9g5AAS\\\\\\\/0zTjlJqbKHsWyscxXQxZrLeSfJ5cOeWRRmsMSpud0THXfjL0xyw54H\\\\\\\/13aKdWfQZwwq\\\\\\\/aEcxfrwUyD7SZQoLtMfdimqLwS+thw9R7fRNDxkMnV8EMXMg77y13b1aGu3TJJGJWhKAvbbCBAQqC1t++RQ9vNXKeYwHCxFrvzac661wUiF3niMPvMzZVlo0SsauuLSMIfDrOgSpuFas4JG+jeCuslB8O8owP5Z8XtuPYOSRaOoxyc5V0Rqm1a4w+tpKxpRjy\\\\\\\/2rksx+tUgKjyOEdpEiE3\\\\\\\/srGY6RQ7+Keyv9bply7ATpwY1pWGOd06n4fzCSFRmvVqiF+iPiCRP1629nIPuDmNbtIisu1\\\\\\\/952a\\\\\\\/sQnUu245tuHN\\\\\\\/GYGrmDOMvZv2G7iUt6F7s64mcRir83XaVx6JfPhvxYVuJTM907j3b3+nGD5iY2dvA\\\\\\\/pl8bee9RA+5ZHtpp\\\\\\\/zVu92sGeUO8dL9UFU2fzXLx3CTgef+621op8ttvLMW\\\\\\\/RFPqC\\\\\\\/mQ6io2w2XDBqxeYOPKzSuWwRaiGkfu3f2WTu6Vdsu\\\\\\\/5EasMJfPsHL8sfyJW6ktBACRvvWg22hVI2FW7J2ZjNeMavFH8\\\\\\\/91ezqIDmpFPNKTRC\\\\\\\/YQiZ7MQCS9qL5quxnZGuCTJd39e\\\\\\\/qxNRBkoY1+2rFBq0FE1\\\\\\\/FuBnCmUT+PErGZUCaRJq5Wagzcp81INDT7WoTKkR6HazhOxw+CkFBip9NeB0IWVtdrcy5zlUJ2KIua08J7qZacv1wvJLEbTKm0IYqx3cnCOU7ylzt\\\\\\\/c4XdGfsW6GFT49WoApBRjuqUuGTw+S6Ci7Z1HJ2UOQI\\\\\\\/LHtayF4SZKHxdGnbX3++pySQ4yAP6OYISjVyc9ZTvVZxURGoxb6uO6UhlWmj7DtFdzjzIEVvxb4hHvI21e34OHZpn6PvCOqxcUXhw2NBnhjAl0977QJGxjUN7Jpk70c4QX7vQVTjm+ziYVQgiZr6ejwdLrB5xMpngydZuDxaT3GlNnAg+76BxFNCdb6gEMNsgUerihbXz8erSGwjfuqZxSuoHkwI\\\\\\\/dWba5RoNw+MMzTJw5d5xQSYLBNmjK09cuXUw7H6X5X0E+QigTzo5vo87th0wU78\\\\\\\/wVomyAa5cLMYtVSMIVxtI\\\\\\\/taCPOwMbt4M5Qs5mHs53olx7PIOhV0Tk5k9Jzh9SLsOda7m+S4qcC7CLUeeE0cHNyx\\\\\\\/RqlvNqrin1BJ+c0874r4P4oP24OX4DmplKl19XcJlKHmMem1J01pSypG5eJuNE2nOcZmqt7DiZpBdkKRA6h5VJWB6C1TDiIu5\\\\\\\/8nMRmwUM66tQ4P+kRRrZiRfp6iA0IkPIRsSnk1iFZ\\\\\\\/dnLPzrOifwC9WKuaH6W5mtRZjgKd\\\\\\\/acLPmnyvOHIjp7iwXyC1bSS0pdgZPszXdWR3T\\\\\\\/Gphe8Jl3VcBf4MDySW48UsVWUyi2NGvmrbPOtBffYGVLuz7PDko8mpqaFPMxjkQwEiChMWt3O3kgd\\\\\\\/X51UGAePX6yhj7b6Gi+ACIhoScaNCVxW8s+dh0mC2r7IrRoqNBjcQqMNvfuWKl6VRXEZfgjeuU55xMK2ju9CpB4yCxLtH+rNSZEh5bMc6mUyWw5a7uV+gtLDiYwWiOui2cHflt9aEjiUi5piYcLTtL+lFiR4OYPgTJnn5itMcwf0y2ExnWzxiWeZPUJqvediMsSmj8JMs+GS3qX40rFaEgHvS2Ngq14sGXm3aGTu2FxUINvw2ryWYX3JBEm+XbXKkLs2233OzkvnF1rRxhp9wiDFu\\\\\\\/ynU+u7yqW6NVMkYZvLiQauiD8INQ\\\\\\\/FNSIE+ZOo5trOhkeLHCb161Dm+mcUqatvhKRoTGKkvQIuV8kQwQkmJJ86DJ49\\\\\\\/2DMKzED9AeNtE1KrvDldiOGCLm9LPH8250qW91cY08U78jqTnKiH+qT3dGY2SMoaNwFiQJBiDvd6Cz3IEBpWicNd77lAl90xMrTggAfzrlLnPSpvV56d45yqKfxa29H\\\\\\\/XdhRPpyLYHnHvaaoQyQ4tB6JXkFGnnBPDiHUxOnRhlCVmn9BMR6JOus6fvjEcV5\\\\\\\/b9mG6YgElSX6wuLJkL\\\\\\\/4N9a5fwGU65RBEG4sMsWI0z8mDx7GiMOlDS62nlWJnn7xz+i0DubZZVSC4IEn5b8kWiuAO\\\\\\\/kEAtiKcRAkBuRoMhphQ6dQPkDXdRjMguUM0zOz4246pgkKfB\\\\\\\/yljQlGkYU74atTJLLmo9ebW7\\\\\\\/ZsC7Xp9iXAVTiBcyRT80XH2RApVzLalfnYQt6BH1pR5rvH8k9V4ajuDQH+mct761cnnVDMjFANGRS6J7ERCVq3fwhtRIwPlhJLDt+SQ7KzirN9sS8Ye5PbaiGZojNM5y7wZzbqVR\\\\\\\/nJz3CMD5tGsxA6OfPmXgjRq+dMiOX5Q5\\\\\\\/H1JJ1rR5U9dT4j8LLrMMeEm1RqqiO97qHoJvJbL8rknuPXbob9kys4EY9hNYz05djiJo3NhTTiWk14Q\\\\\\\/B6nlVLpZSG5SjyuMSFB4sLIAfn5vaPLmdbfLNGLr6vFdE+mnMSTP9SabWQTL01vTHFbxuMM25LGc1QDDKqPh9IN\\\\\\\/q4D5+eixYYxo9f6VMoyrnNJ6r\\\\\\\/sdPvqRihjbF6\\\\\\\/zQyiFn40xpfX2mTQrJ2PkVPj8hqCIoDL0OvvZSxxpi7UprGWxTa6TBcHYrcsYJ7ixsN0Lchf1RLKc0m8RH+D2CT\\\\\\\/Z3pv340PNlTUl20yRiAthjJh4WrBzH1e\\\\\\\/tXKbt6bAGZaTq\\\\\\\/nxcHATv6lCYTTuDkCN6+0gPuszeLO3T8IdVcycBw9rIOl0ssqBS1ruo5G5ZBzWN0PeY9Q9w7nN497LBpHcL2DRdjgoMQnRY4mr\\\\\\\/hfG9fx\\\\\\\/e3eZfNzGoj8Ey4PcsgDF46NVySUV4zgsQ7aj4IsAWfvMRsM24LiTRF7ZfLdE6ICvGtGeOjF4NB3rLeJtD8Cl22TlKaC7U6hq0oVDc2q2YN45xrYv3kvtlPT7RxXz1VMPz3FHUIK2wCbJuamHaO1csXYfCgDysA9E3CTczuonBr+2xlDwRXOaqP2U0q7qxfasSNSCphkm3ZyeyNh2KupIh\\\\\\\/09Q73YKO7qHedZ7yhEPNTk3w\\\\\\\/G1TW7nKj92qa88MAdFz8qGvUsqcKFdD6F8Wk9gU2fW\\\\\\\/uH1Ubn+1dsbGbr\\\\\\\/GRJEkvhI0oKz6HRqqrvajM+NsLHMhYAX\\\\\\\/TYmiQz5KvWcU+kCIyIs5obMK3TU3n\\\\\\\/\\\\\\\/1nlSSh\\\\\\\/O2UD5tBrJF7SBV8yPdZLTfvhDxfr7VmlRphYJteL\\\\\\\/FFV\\\\\\\/WCTy7hXbGrWgRUzbVMILD1ZjDmBFjqtpamPV\\\\\\\/zKnAC2+G7Mf17+2b5bgTVN0\\\\\\\/\\\\\\\/0KFonNOWqPNOTzu3u\\\\\\\/5MAVPsN1wyYQVcXDrKSevPAfgYfegmIev4uQYfJt6FxEnfSqxnDvXyA2mPO4EW6CJ0CBQKNtZ5w51oGEn7Hxpah43hnq159buCYaW8F7VYF5pmQ5AL8uPxZXMZeEFersyvbWmHPL3e+1mT1sSJUWFHPWfVldyHtYetcBNYKtss1DAnt3hyxx7AQrccw0D8GiBQG81YfUZz\\\\\\\/GOXD5+An5CQGNguIVPZ\\\\\\\/NNS1QD2sS+wjeLpfX9lEr+95k35BjfaijO\\\\\\\/jMwNsNIM9UBTBjzM9OluQUzd\\\\\\\/U53g86vn3r9gPILlqfrtlNQdPqTrV0xASrF3YRrBOegQIO0xQHH2tAnH1RSK1c5RHWMD9FSWlXyoDpmpmWXEeb\\\\\\\/vGB3zYrEbvp65diikwjqo3bKbnc3g6Tsehep3dRucXkrHchdpp919rz3LsvXwUVWmOB8VPgoMg2+aSkpAjNGRxzcFZo6qSeQXc6R6Z0CnpxuDw5RriuU\\\\\\\/cOaSb8qQcCZ0yHStKi16upsvnkLYyRGWPwtWvo1lgWxLYO27aUrBAA99qeG3227z4seniCU9spQAl9QbD\\\\\\\/q277Q04H\\\\\\\/Wqt9O7cfgQj1GDOegtHSeZZxaK3mUWPw8Gppl5il85Grqc\\\\\\\/g6aKl3p9yIDKfYii35FFhjbmmyn3Bp4if2cP1gPUfNF69Qp6tc439KpYFr0fdJo\\\\\\\/f33myySQaN8N\\\\\\\/Lgkv7T24X0odCE9zg6o7HzNGgqwxJiqQGg6NEhq2vmIUrlCV\\\\\\\/zhaP83Qw03VwF9yA93qCuyg06W+U2KwlpdW7d8Hiq9e4xlY8JDD3rS8DTfMbRpC7n7fC\\\\\\\/4EVPPP4dW8E4UwZHuBVO\\\\\\\/ugiDQ5ctykTDCH9JvPEeeK1WKPFGNZLH8ImrnSu\\\\\\\/\\\\\\\/bA6mwKlkaPM7SrYdIb925Hx3tGyA9Or0Uv7EzT6MrnT3qXUZkwlEbuVMLtcHca+ZGP6Qc3B7XZSSCsehofuprYCONalbsQaQ3SzYJbkNrzxEuNQmhEwIrAjzwtBz6LrIQYt2qQr4\\\\\\\/IE6Nkb8EWJsBvMB12IUX2AAbuh3EoydViGmD3NFExqiquc+fYIoy\\\\\\\/bwc6u2MbIU7Vi6PGFAyMAPwwOl9PLfOuV2YsI2FtfEm\\\\\\\/uP0X9BbCiL4BHaJeqQpnI4D1acnSTN1EE4ixTiw2auoOu\\\\\\\/nBorxFQs42xsOfx0XrdMDaySiYed\\\\\\\/ffM7pyCAnr39i8GesxJRisO1xShLNfhOCjXNQN97R4oNTcIgcOChhq08RdnpOgfMb+XMaXPQcO6ajHck8AuQXJrCNpXqhgYf3aNhfTKnx5Rwo0Fr2wrOr\\\\\\\/EJaHMeDZ4rm0P9G+ROQnuGIXWH0YjIAAmiho5Bw7TX5chv4vY+iZFpBcbTdVeZUOQC54hdu7cSolwZHcetYid7XH7zvadUUJKIvwygYG8CffT98p949xAyW4n3ZYy52PQGbLJeHNz\\\\\\\/Cb01DGlp5SVGJXS+2xJsCKRRlrSbw4rVTAvrQ064qmL\\\\\\\/TphETLW4Mhf\\\\\\\/iQ7c95NQmF9NnKtcUjwXbqHhC+Kkh7Km+GDuym31mpDUUj0fIt30+2\\\\\\\/iC9PfABbjPJpP9tKGvMwibH29sf6t6zzAuGrmjRUUE8A5S2h7woDjTlc0bfLWrKo1vUG\\\\\\\/t7cKLveybXvzCEIN2GFwiMMwUIU5kydOwBiC78KXOz0q+3uei9xuZXA7pEsQJ26xsK6PtRwl8qmc5\\\\\\\/84uH5e\\\\\\\/EUIqtCGdwPhBUUcYE8PU7YFgl7OLYwdaSqkP+Rh6E7lYpgUwWcJIcDvdMihpB5sZJnoeAnJbiyk955aesANSa0OmvlCe8ooR2jRubBCV5sLFHn7X\\\\\\\/L47dsfPC1C\\\\\\\/fqBRmXk7LHja7XkQ5epW7scahfgjan6FMH3FCwCy8FoUEn8hRMclUEn+ASMIbcCNVHRDU6VkhBEFpmRtnWOaLyekp9U7BOiL15N6j41Ykvgi8d7hLcxGWPyvUuwq+20h0frKsSWRac6v9nAowWJ81SSTBBg0xYkTN5tiOarCdAGRiL+IbuXE1VPYmVonHfOVX2cTA1ZrR01ggLkCb70XG4060XHoouwYwzPj5MK0IcmpYUIlCWVJZr1KKOGGEZ4iczTiD\\\\\\\/Jt583M+4U\\\\\\\/DUhwYsHD56odvRGjwSb6LLRMQM+FtP\\\\\\\/Ta9lfJ+xHSgSCKjzxrbho7QBFj0MN3hG4UeB31j5kJzsJeKPIH6jY23bxq1EL4kYXKKmxLScYXPpoDvfzZi5xYLZCmRs\\\\\\\/Q0B9rI5tbzlh8A5FUZ3Z1zb6ieHr\\\\\\\/HHYnc0xZQy37s1vOTQtjCtoi06S4iYQza2aNx7PhPOwzgVaypzC2FIPmuyNEq1AfHJwkixy1ZGLXL4KQACVcIdG\\\\\\\/JzE+uPCK\\\\\\\/3DVTaGvQtO+b1V6kHaOKEtl5ZZr8B4bZAxXSFZlwSA3J2LmV05CG\\\\\\\/FXRO7Q5WRVRCk2wFqZcoCoN60aVTrIuuNn4JQw84sckPgPSvZvl2NEJ+nezlCWMcKc867YW5CFb87VzrSy83wPGL7z3bOhPxwAl+TbtRFo4y6WyVyh6\\\\\\\/P2G0O\\\\\\\/Wwa2\\\\\\\/aocvxAPS71iWYyQoxwmIw8HnYLI9KZUoVhiV4ibi1ZOSAmJwivgff86btN5XaEQSvOtt2TB2DYFLTeO4bAIoX88KBh9V6n47UhXw2e\\\\\\\/utFnzGTn33HgdoBAHdOyI4OaJUuWfLt5zNrV6aGxQ2oKX8cES38tZNlsKogUSHt6vflmxxWcek4Axx1EFHcsyDI+gawdwN95G49moC4W9834fpkyRA5LHUf7FT5avY9mmMHXdFmMp0pRl6CcDP7NmkA88f2v7XTcndnOAdsKZOUmtcIn7n1iMRQce\\\\\\\/q8jA+JoTIKXuxq1b6sGS9pXhpRC85v9r+7kk3ea3d7fAmWiRwTwIkp+Vn+UEnrR5S+F5Ypoxnhy2awInB3URias2Zig\\\\\\\/3iWbFlUcBs0RfGc42DGHm2Hk4Q11RzVNHjajHuBKllV7LiKcLv8gVVAgvtG2tHX8NQlVviPZp9ZWZFHZUVzH7qjNskyLQfNFOp4Xrn9N5VeY8DgUz4l5Ae7OMPynD2dKuAvakfiu0fNlM\\\\\\\/NKRAbIu3qGzvCbWjXIm5+oaQcRsbX4FYKfImrxwP69S7HrRr14UVejp6BpLyoY4hkksaLrBOgFq3x8NpfEZcmh2+J0L8zyGjiaHCJhBbasUanxmRavusOmoabeL+8fB37pos8GaOU3W0+nwFoEaVRB2KMBgIfl7y5YTPzEiR6olGGJwhrU8qmKv77rlTdxTP1YYB2YifosQb3bgA\\\\\\\/tOh+ysH3uu7BnMtwQlXWme\\\\\\\/rTVp7EzZPCj99vm+9zsXCLGjm0NfvPELj1zGSUJKqEJDW\\\\\\\/wfjqTcy+1UL6zpymSdv2ansyhd0j0FRjoeiX4XGfLNJoS466i0L++mSGFoTitCYEYsB\\\\\\\/shn2k8sWImESvOGxZv5VTFP6HJCPsCvai8kZXbSiWe3IUeLcCCTBH\\\\\\\/wy1msPG8MMhhZ2leeehvKX0c9bUqh0lx55CYenFBI6WrzVn5Dqflqq4jONo\\\\\\\/be4Fcxi2PjBA8HNMq0qMar0LvWPxzuqv+iBspfnT0E+MwWaoEauz3bzaUG+vvdTZ9fxREOGcR9CTtlBq71SDEc6ND6D0ivv\\\\\\\/fEkBwzE62zNGKtiUJrtq+pTu0zFrf4Kt8R+HFlq1IhkEtPp5AMntk6rGcKEfLDX7+hC+0pceGsNwV3TYDJmW07vZY2j\\\\\\\/mzjNAfB6HnzZQdkieVbqcaUswSVDqbiwTvZkTXYfMsjmpjpkB7WFjEo3dfr1MFCpSjWeiZHCbYVbEqgYmRAIVMN2wtRZkz5i\\\\\\\/vfZLV8xSZdHoqfD5\\\\\\\/qTFb48rkXE6ozHIo3MI1jjsmdPhQ6KgrIPr7au\\\\\\\/SVraVHZY3KJvdWt9TvJatID\\\\\\\/Q9z03RpXX3UobSQRskgqiDJS3OQfbzwkzq6BpKhgTuWBwu4KKKvcjOpxe2NxXx+L0zuRBqvp+0aY0CdkK0ISizK\\\\\\\/HvNiqCwcLPeGoaZC7YYujn3pnqSW\\\\\\\/SNwWMsirqYfuY2GTt3yezUQXnQk2sXil0QVs44muGMWAX5y6KhbLc0c2POrin6DKdNnjWxPCjZNmdygLq88r3FYnIayxyNyUciFtQQAmIrQ6K+I\\\\\\\/4Y9TIv5qyghHsO7Oq\\\\\\\/AJVj1t20bA2w4daxS0Dv8QIO8RliN3M708DNbLy0fCUSY3aSSZJ0JbABqIsfZSU1K3Gc7FInnTL7TJBTPENCbOWodpRSnNn4zP\\\\\\\/R7pAsdCailaWtGXXXw\\\\\\\/aBuhTu7et2p0pruD\\\\\\\/r5cSMKLRGr2lJBVEzSWXuimAsaW7FqYlACm2uBoYtZGSR7QOyESkTwPYoNNH54GhF4x2DqY2ZDLKi246wfxHF1MLGtDIge5zdnCv0tq9\\\\\\\/z\\\\\\\/8ASqD+Bvvf8E3fdrVH2JevDYc25tIeJPvzQlKaLfmZZJkHNCtkuZ1Ny3rdhQYBHj0igJCACNCeM7IW0J2mHGRIRExjDAWR8a2uvWXwdrgu6ugMJ6e\\\\\\\/dyy45\\\\\\\/\\\\\\\/vxOcIHF+dH6vi64A8XEyq0GN6QAYuLk5YWmkKPtFLZE\\\\\\\/i88t8YEpFxOJumP115\\\\\\\/NWrpeB3yebSDullzbwxPRXryGD3ksn24ywegTCSafiagUmPvaLTlgGsk4Qn3XAhrruoE\\\\\\\/o7CpUPObnONG7G5\\\\\\\/Gb998ueJAoeDQj6dGMlObZL6b2HJ2mCS6AWrTz4EHXIuMAs7xnSMpKIwF47+wSMMY5Ea9OmHpnmnBCUY1mI1xCwrEAIvqpF\\\\\\\/ifl1WNYqYJm1f\\\\\\\/4gpjktWjTdxlqvYztFSmz\\\\\\\/33+MTGWhKvjia4tkBoE0lTa4p8HyJ\\\\\\\/jr1SDjNtb7HSU6b75fs7Ht15IOkTrxNXw72wtoWaB4j41n\\\\\\\/LJa4LZ51PxWcfOLKm+tkr8aoEbWtgPyNspkipBlIko+gXHawJfaeGOkwG9Do7vZatwzJ87pzLCdTTvisMnLiEuyoT\\\\\\\/Iri5aiSE2hsrW1ZePpVAxMC4+h0TUhW5n21lusDXXolxVaExq1QviSmk1w8GOeQVunG0V\\\\\\\/BxLqbdma7xVCsMawS6rGLe1edqvJwG+cVJN3wJyQwD7i3vnPWk\\\\\\\/rFoc74aET5KBJKgnzSQAm+2N60Rl5Dp6RT3V+Zbdoggzdbi8pNu8QS7MnK7zWWVPSo67Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022V4RIZ46lPK9+Jb+V2beDhQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225f319b3fb807f9b61ea418d8536653040ee32eed2e2f5055da44fc12fdaea533\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eOzWj0x1HqgfipLhYp5GtOI\\\\\\\/TI0wNkvZS4LVVVJLdh9qIMyJMcARXkfJwSa2Vt9+m\\\\\\\/1SFxc+DcLwlgM7mr2vVrNKUZGp6+900wNMTtYeflo\\\\\\\/OJg\\\\\\\/c\\\\\\\/j3\\\\\\\/IOW0HgqDIqmZAHu5R0\\\\\\\/Paiy62ew2t+41ceZ4ywdoI1ricgcUtBymTkVEPEn+GsL0EsqFWOf5wcR0ZudOsRKPsImhZd0CAzfBmgzZlkSjlfjZJV65B6A4rL+dyuzpjJqK2k4twHC27WWDqRuo6V8BK\\\\\\\/M4S+8uQwlU0uV98pwsra8tsMJFZzI10\\\\\\\/nvj7qzfmd8oGJfxlT5yNOvMymdBj5iMphzJPnUmz2RxERd+ARfET1JBolsTL9qts8slETkK0xG7EoYClBFS4+cnyprD6fUeJ4QL1h3w9JKwQ\\\\\\\/Sz8evN0+RZD3YtTEUQCGgSlfWlwowZQnw94kWRK5eHAP1WQ2li6LrVpOq+A\\\\\\\/bzP7JTMA+zPhnxwsus\\\\\\\/rmY+BGAq8Pn5jIYVsM5YOKLS45QnlaLVQr2YLxw2mLPB9Jimx4\\\\\\\/\\\\\\\/reqFG42lrbwer3\\\\\\\/0IAWVHKfSgQ+TDHV2\\\\\\\/ynTfQw2qr\\\\\\\/6iWVGILHx+saIkCLZTF7zak\\\\\\\/2gup5j4KNXOVB3SBmSaD5OIh1ffsH+bKyQeVRvesKx8XETw1EpUfqowRoe1TYBvCqU0eITivzATPIv\\\\\\\/NkBtJbZvE0K3gHnWlCrUGphxcWvoo1y6rjmwaQsp666lS\\\\\\\/GTwuY15ih1PSJ45f9efhzPxGMWD9RS9vXS0kkeM+fKcR7eTbbn+zdUEteuVigepld273tT70wkd2vtpvqqB6OIFRKh4mzS7fQd+O2qqLZe6ViikXzgoq7d2nzqHW8cnCuGGUw+Vm81gKPuHDYrdJYesLeL2BiErl+sPNxj8\\\\\\\/ud3YbodCHZhq2tmNJdh4t15p\\\\\\\/NIvdhCq6hpR\\\\\\\/uEb9UA6t2Sq7gHC4wk8ErgsdWCYlBLJLFTsLwQe5YDrgQdcSHnk8OqrrbtARhLBsWDJesFVHbBysyqWDS2TPFf+ydjxJThwUB0EJ6+6NeEHyS7dBCzauYQTJb2ECeYw9qhNjBdfWlRoNLyaaNSsyMKs5eQJCsCc\\\\\\\/1Az8YBBbVfCIDnxsnOk+rp2tF+UWOclUYj5V2rTfHeTsJaYyiEA\\\\\\\/tl0Or9QcKmUqotEaORO8ZjyphGqtQXCRnO+cPziRqHfiwNHEU\\\\\\\/OWF+JbPAHwKPzLv93vFSzigmZFGqrEoZ+62h713HaK5q0C5aMCOD7bw9XTOe5vx5a0Ufrfs2eyg1DCa7sZpBcdybj94VOVG9qPJyNKP2zYYGXD8LRfErJozMsjRmbV1XL5GQRsqpjMcRCd6yqE8vKWv41sCoam2h8Gt2RTUk4x09FB7XVbw1dMU8HZ+mkNQeEBLoonUBPJWKHRxvkCi2YIKV6QJyaFKXFERwG8j6H23w\\\\\\\/G3YqA4rbsqCPLCME4fS\\\\\\\/nk2tAiz4QndV5v6E7sNjmA9T0kHFJB2xjRnP4gM8Cq\\\\\\\/hHs9vi6MgRME7Lt5B9tXfzSOzzUKuxgmrXmRPuyQJ7RILkEJVKLH7CQMWDsyRz6yQM7PoDGhaqL\\\\\\\/uTFx52O\\\\\\\/xcnDhAufWtgyPuxvaHzbPhXYyn30wdqhllon6u12E9dVhTjUlE4U\\\\\\\/7DSbwrQmh09LJo2U+aOtqfxZBLL9OYdC8kLJZ7Btvdr39hbu05Np9FDIHk8gWNZafhurNogGyTs8BHuGqyhtlsz5u0P0xvjsQhgNxDqqtfWMrlF0aSbHr8Du4qz+WM5kXyZ3wGEuvG\\\\\\\/yW4x1I2Q95BXV5680W+JLjmSmZWKX9fSjv05TlCj2hBsQZJg9M2+L9u9BPiEIrn4IH8yV0nIrj73gb8srwjAUDDfV4\\\\\\\/siDlz1xQRY\\\\\\\/n+KLSDuHOomoWaiKFjAHtjPKttjiUMOGpXksyo6Xvr\\\\\\\/4Tm7DFiPXwppGFOvEWPhRIg3IMRqqzchpjNCz3Gc1FQMH19REPh2xwddj2FZTwHLwNLcRJj5gTWYKNyA1QQCKPw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022UXDvdPkSndvj4FTr4w7iUQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022719774495175b9da3b4e27841029eeb5e41b317a96a27a2eb62f1b6414741ea7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459691 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/10/1963" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BUt6GQCbWxKMzAeSMgCfmQNiyRi42cI2kVeRS6QbMv4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/10/1963" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YhNzqM2uWYyBHqGsOshLsI0X7uZom4sBnpnLLelppLU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/10/1955" title="Performance of the Cas9 Nickase System in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/10/1971" title="A Saturated Genetic Linkage Map of Autotetraploid Alfalfa (Medicago sativa L.) Developed Using Genotyping-by-Sequencing Is Highly Syntenous with the Medicago truncatula Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459691" id="node459691" data-pisa="ggg;4/10/1963" data-pisa-master="ggg;g3.114.012468" data-apath="/ggg/4/10/1963.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resolution of Genetic Map Expansion Caused by Excess Heterozygosity in Plant Recombinant Inbred Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sandra K. Truong</span>, <span class="highwire-citation-author" data-delta="1">Ryan F. McCormick</span>, <span class="highwire-citation-author" data-delta="2">Daryl T. Morishige</span> and <span class="highwire-citation-author" data-delta="3">John E. Mullet</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1963-1969;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012468 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459691" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sandra K. Truong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BK.%2BTruong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truong%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BK.%2BTruong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ryan F. McCormick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BF.%2BMcCormick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCormick%20RF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BF.%2BMcCormick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daryl T. Morishige </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daryl%2BT.%2BMorishige%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morishige%20DT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaryl%2BT.%2BMorishige%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">John E. Mullet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BE.%2BMullet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mullet%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BE.%2BMullet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmullet@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">jmullet@tamu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/10/1963" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459691" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459691/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/10/1963.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459691" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459691/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/10/1963.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459691" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459691/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/10/1963.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459691" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459691/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/10/1963.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459691" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459691/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Recombinant inbred populations of many plant species exhibit more heterozygosity than expected under the Mendelian model of segregation. This segregation distortion causes the overestimation of recombination frequencies and consequent genetic map expansion. Here we build upon existing genetic models of differential zygotic viability to model a heterozygote fitness term and calculate expected genotypic proportions in recombinant inbred populations propagated by selfing. We implement this model using the existing open-source genetic map construction code base for R/qtl to estimate recombination fractions. Finally, we show that accounting for excess heterozygosity in a sorghum recombinant inbred mapping population shrinks the genetic map by 213 cM (a 13% decrease corresponding to 4.26 fewer recombinations per meiosis). More accurate estimates of linkage benefit linkage-based analyses used in the identification and utilization of causal genetic variation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/plant-recombinant-inbred-lines" class="hw-term hw-article-keyword hw-article-keyword-plant-recombinant-inbred-lines" rel="nofollow">plant recombinant inbred lines</a></li><li class="kwd"><a href="/keyword/excess-heterozygosity" class="hw-term hw-article-keyword hw-article-keyword-excess-heterozygosity" rel="nofollow">excess heterozygosity</a></li><li class="kwd"><a href="/keyword/genetic-map-construction" class="hw-term hw-article-keyword hw-article-keyword-genetic-map-construction" rel="nofollow">genetic map construction</a></li><li class="kwd"><a href="/keyword/genetic-map-expansion" class="hw-term hw-article-keyword hw-article-keyword-genetic-map-expansion" rel="nofollow">genetic map expansion</a></li><li class="kwd"><a href="/keyword/rqtl" class="hw-term hw-article-keyword hw-article-keyword-r-qtl" rel="nofollow">R/qtl</a></li></ul><p id="p-4">Linkage maps, or genetic maps, are the relative ordering of and distance between genetic loci in terms of the frequency of recombination between them. Knowledge of the linkage between loci is useful for the identification and use of causal genetic variation using techniques like map-based cloning, marker-assisted selection, and quantitative trait locus (QTL) mapping. Using this linkage information, the genotype of an observable locus (<em>i.e.</em>, a marker) can be used to predict the genotype at proximal loci (<em>e.g.</em>, a QTL), and the correct relative ordering of markers can be inferred for applications like genome assembly. As such, accurately calculating the linkage between markers is of practical importance.</p><p id="p-5">For some plant species, including maize and pea, discrepancies in recombination frequencies exist between genetic maps calculated using recombination frequency estimates between markers and cytological maps calculated by observing cytological manifestations of recombination events with microscopy (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hall <em>et al.</em> 1997a</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">b</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anderson <em>et al.</em> 2003</a>). In general, the genetic maps predict more recombination events per meiosis than the cytological maps observe, and cytological maps are considered to more accurately represent true recombination rates (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">King <em>et al.</em> 2002</a>). Two of the major factors contributing to this disparity include tight double recombination events and segregation distortion found in marker data (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Sybenga 1996</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Knox and Ellis 2002</a>). Tight double recombinations are observed when an allele is found in a phase opposite to that of alleles from adjacent markers within a relatively short genetic distance (<em>e.g.</em>, &lt;5 cM). The source of tight double recombinations is still an open question; they could arise from biological phenomena such as mutations or gene conversions, or they could be (and experiments have shown that they are generally are) genotyping errors (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lincoln and Lander 1992</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dib <em>et al.</em> 1996</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Broman <em>et al.</em> 1998</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Broman and Weber 2000</a>). However, the frequency with which they are observed, even with disparate genotyping technologies, suggest that there may be an underlying biological process responsible for some of these tight double recombinations (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Sybenga 1996</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Broman <em>et al.</em> 1998</a>). Regardless, because of the assumptions implicit in genetic map construction, tight double recombinations in marker data greatly expand genetic maps. For the purposes of this work, due to the dramatic map expansion caused by tight double recombinations and our current inability to conclusively identify their origins, we treat tight double recombinations as genotyping errors and set them to missing as commonly practiced in the literature (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Lincoln and Lander 1992</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Dib <em>et al.</em> 1996</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Broman <em>et al.</em> 1998</a>).</p><p id="p-6">Similarly, segregation distortion is a commonly observed phenomenon and can also affect the estimation of recombination frequencies. Segregation distortion is the observed deviation of a locus from the expected segregation ratio under the model of Mendelian inheritance, and it generally occurs as a consequence of unequal gametic or zygotic fitness (<em>e.g.</em>, artificial selection, meiotic drive, etc.), or as a consequence of an error prone marker. Although one solution for distorted markers is their removal, the removal of markers reduces genome coverage, and techniques have been developed to account for distorted markers by (i) integrating repeated observations in multiple populations (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2005</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cloutier <em>et al.</em> 2012</a>) or (ii) modeling the differential viability of gametes or zygotes (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lorieux <em>et al.</em> 1995a</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">b</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wu <em>et al.</em> 2007</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Zhu <em>et al.</em> 2007</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Xu 2008</a>). In addition, information on segregation distortion can be used to aid in the identification of selection and QTL mapping (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Xu 2008</a>).</p><p id="p-7">Multiple reports have documented extensive segregation distortion in plant recombinant inbred populations manifesting mostly as excess heterozygosity, and occasionally as reduced heterozygosity (<a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/012468SI.tif">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/TableS1.tif">Table S1</a>) (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Knox and Ellis 2002</a>). Although the source of distortion is not conclusively known, it is hypothesized to be the result of a general selective advantage (or disadvantage) of heterozygote genotypes. Despite the prevalence of heterozygosity in plant recombinant inbred lines (RILs), the techniques developed to incorporate distorted markers are not commonly used. In the case of retaining markers based on multiple observations, this technique necessitates multiple RIL populations (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cloutier <em>et al.</em> 2012</a>) which may be too high a barrier given some plant generation intervals. Existing methods to model the viability of each genotype differentially treats each marker pair, and so may suffer from the overfitting of large data sets without specific biological models; these have also only been shown for <em>BC</em><sub>1</sub> and <em>F</em><sub>2</sub> populations (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Lorieux <em>et al.</em> 1995a</a>,<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">b</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Zhu <em>et al.</em> 2007</a>). In general, plant geneticists have constructed genetic maps of <em>F<sub>t</sub></em> populations, where <em>t</em> is the generation interval, by fitting observations to the expected genotype frequencies of a Mendelian fixed RIL model that relies on assumptions of complete fixation, no selection, and no mutation; this model is unable to account for proportions of heterozygosity maintained per generation other than 0.5. If the recombinant inbred population is treated as though all loci are fixed (as <em>t</em> → ∞), yet more heterozygosity was maintained per generation than expected by Mendelian segregation on the way to fixation, then the recombination frequencies will be artificially overestimated; not accounting for excess heterozygosity underestimates the number of informative meioses that can occur prior to fixation. In addition, treating RIL populations that have not yet reached fixation as fixed RILs results in the loss of genotypic information and makes incorrect assumptions when calculating recombination fractions. Finally, in cases in which the distortion occurs across the entire genome, such as for the sorghum mapping population used here, removal of distorted markers under a Mendelian <em>F<sub>t</sub></em> model would remove the majority of typed markers, causing a dramatic loss of genetic information.</p><p id="p-8">Here, we build off an existing model of differential zygotic viability to incorporate a heterozygosity maintenance term for plant recombinant inbred populations and find a new solution for the genotype probabilities used to calculate recombination frequencies. We incorporate this calculation for expected genotype frequencies to account for different proportions of heterozygosity maintained per generation (other than 0.5) using the open-source genetic map construction code base from R/qtl (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Broman <em>et al.</em> 2003</a>) and report its efficacy in a simulated RIL population and a sorghum mapping population.</p><p id="p-9">This modeling allows more accurate generation of genetic maps and retention of more genetic information by accounting for the biological phenomenon of differential fitness of heterozygous loci. More accurate estimations of recombination fractions, and thus linkage, will improve the accuracy of methods that use linkage information to detect and use causal genetic variants.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Derivation and implementation of genetic model</h3><p id="p-10">The model for genetic map construction from genetic markers of a population with known pedigree and markers ordered on the basis of physical position with a reference genome is simplified into calculating recombination fractions between pairs of markers. Here we derive the quantitative genetic theory underlying the expected genotypes of a selfed population, <em>F<sub>t</sub></em>, given a proportion of heterozygosity retained that deviates from Mendelian segregation assumptions. To derive these equations we simultaneously extend and incorporate two models: (i) Bulmer’s general solutions for genotype frequencies of self-fertilized populations based on the work of Haldane and Waddington at two linked loci (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Haldane and Waddington 1931</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bulmer <em>et al.</em> 1980</a>) and (ii) a model for zygotic differential viability, where each genotype is assigned a fitness (that may confer an advantage/disadvantage) in the <em>F</em><sub>2</sub> progeny (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Wu <em>et al.</em> 2007</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Zhu <em>et al.</em> 2007</a>).</p><div id="sec-3" class="subsection"><h4>Genotype frequencies of selfing populations:</h4><p id="p-11">Before we model heterozygosity maintenance into estimating genotype frequencies, we will set up the familiar framework used to estimate genotype frequencies in a traditional RIL. Consider two linked loci (or markers) <em>α</em> and <em>β</em>. Locus <em>α</em> has alleles A and a and locus <em>β</em> has alleles B and b. Suppose that the initial parental mating was <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-1.gif"/></noscript></span></span>, then in subsequent generations, <em>F<sub>t</sub></em> where <em>t</em> ∈ ℕ is the generation interval, the family of individuals will contain a distribution of ten different genotypes, and for the initial condition of <em>t</em> = 1, all genotypes in the <em>F</em><sub>1</sub> generation ∈ <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-2.gif"/></noscript></span></span> genotype. Furthermore, due to the symmetry of genotypes under self-fertilization, the genotype probabilities are reduced into <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-3.gif"/></noscript></span></span> as five genotype classes (class <em>i</em>, <em>i</em> ∈ [1, 2, 3, 4, 5]) as described by <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Haldane and Waddington (1931)</a>, where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-4.gif"/></noscript></span></span>.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-5.gif"/></noscript></span></span>The transition from one class to another each generation is a Markov chain and is described through the transition probability matrix, <strong>T</strong>, that takes into consideration the gametic outputs of each class for each meiosis event, such that for generation <em>t</em> and the initial condition <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-6.gif"/></noscript></span></span> we satisfy<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-7.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>Thus far, we’ve introduced the common modeling of genotype probabilities for the case of self-fertilized populations. Solving for the genotype probabilities is dependent on defining the transition probability matrix, <strong>T</strong>. Under the assumption of Mendelian segregation, <strong>T</strong> is defined and so <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-8.gif"/></noscript></span></span> has a general solution which has been implemented (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bulmer <em>et al.</em> 1980</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lander <em>et al.</em> 1987</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Broman <em>et al.</em> 2003</a>). Furthermore, for the case of differential zygotic viability, <strong>T</strong> has been modeled and solved for an <em>F</em><sub>2</sub> (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Zhu <em>et al.</em> 2007</a>).</p></div><div id="sec-4" class="subsection"><h4>Modeling heterozygosity:</h4><p id="p-12">Here we model a heterozygosity maintenance term for selfed recombinant inbred populations to account for viabilities of heterozygote genotypes that deviate from Mendelian segregation (where <em>h</em> = 0.5). To construct <strong>T</strong> that accounts for the proportion of heterozygosity maintained each generation, <em>h</em>, we will examine each class’ expected transition from generation <em>t</em> to generation <em>t</em> + 1 under a potential deviation of <em>h</em> from 0.5.</p><div id="sec-5" class="subsection"><h5>Transition from class 1 and class 2 is fixed:</h5><p id="p-13">Class 1 and 2 are the ultimate absorption states as <em>t</em> → ∞. For example, the probability of class <em>i</em> ≠ 1 in generation <em>t</em> + 1 given that the marker pair was in class 1 in generation <em>t</em> is zero. Once a marker pair is in either class 1 or class 2, it will remain there.</p></div><div id="sec-6" class="subsection"><h5>Transition from class 3 depends on h:</h5><p id="p-14">Class 3 requires consideration of the segregation of only one marker that is heterozygous in generation <em>t</em> as the other marker will be homozygous and thus fixed in any subsequent generation after <em>t</em>.</p><p id="p-15">Let <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-9.gif"/></noscript></span></span> be the proportion of heterozygosity observed for all markers of an <em>F<sub>t</sub></em> family and assume that the amount of heterozygosity maintained in all markers, <em>h</em>, is constant each generation. Then we can solve for <em>h</em> through the following relationship <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-10.gif"/></noscript></span></span>. <em>h</em> will be modeled into the transition probability matrix as a modifier of expected segregation. For data <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-11.gif"/></noscript></span></span>,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-12.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>For the heterozygous marker <em>α</em> and <em>h</em> (in class 3) in generation <em>F<sub>t</sub></em>, the genotype probabilities for generation <em>F<sub>t</sub></em><sub>+1</sub> will be dependent on the expected segregation of the alleles of marker <em>α</em>, class 1: 2: 3, which is <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-13.gif"/></noscript></span></span>.</p></div><div id="sec-7" class="subsection"><h5>Transition from class 4 and 5 depends on h and r:</h5><p id="p-16">Class 4 and 5 requires consideration of the segregation of two markers that are heterozygous at generation <em>t</em> and the recombination frequency, <em>r</em>, between the two markers.</p><p id="p-17">Similar to treatment of heterozygosity for one marker, we now apply the same heterozygosity term to both markers <em>α</em> and <em>β</em>. We model this within the context of zygotic differential viability, as shown in (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Lorieux <em>et al.</em> 1995b</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Wu <em>et al.</em> 2007</a>). Assume for marker <em>α</em> that the viability of genotype Aa relative to AA or aa is <em>u</em> and the same <em>u</em> applies to the alleles of marker <em>β</em>. Then, the genotype probabilities in generation <em>F<sub>t</sub></em><sub>+1</sub> from class 4, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-14.gif"/></noscript></span></span>, or class 5, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-15.gif"/></noscript></span></span>, in generation <em>F<sub>t</sub></em> is dependent on the segregation of alleles of marker <em>α</em> and <em>β</em>, class 1 : 2 : 3 : 4 : 5. From class 4 this ratio will be <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-16.gif"/></noscript></span></span> : <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-17.gif"/></noscript></span></span> : <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-18.gif"/></noscript></span></span> : <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-19.gif"/></noscript></span></span> : <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-20.gif"/></noscript></span></span>, and from class 5 this ratio will be <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-21.gif"/></noscript></span></span> : <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-22.gif"/></noscript></span></span> : <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-23.gif"/></noscript></span></span> : <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-24.gif"/></noscript></span></span> : <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-25.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-26.gif"/></noscript></span></span>.</p><p id="p-18">To model the amount of heterozygosity retained in generation <em>t</em> for a marker pair of class <em>j</em>, for <em>j</em> ∈ [4, 5], in the previous generation <em>t</em> − 1 we model <em>h</em>, calculated by equation 2 as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-27.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>such that we can calculate <em>u</em> with variable <em>r</em> and subsequently <em>d</em>.</p></div><div id="sec-8" class="subsection"><h5>Transition probability matrix, <strong>T:</strong></h5><p id="p-19">Incorporating the transition from a given class to all classes in every generation, we now have a transition probability matrix,</p><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-28.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span><p id="p-20">With <strong>T</strong> defined, we solve for the general solution of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-29.gif"/></noscript></span></span> by equation 1 and initial condition <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-30.gif"/></noscript></span></span> and use <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-31.gif"/></noscript></span></span> to fit recombination fractions (see Supplemental Materials for calculations and solution). When the expectations of segregation are in fact Mendelian, <em>h</em> = 0.5, then as expected the solution for genotype frequencies will reduce to the same ones solved for by <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Haldane and Waddington (1931)</a>.</p></div></div><div id="sec-9" class="subsection"><h4>Genotype frequencies with heterozygosity model:</h4><p id="p-21">Given the theory derived for <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-32.gif"/></noscript></span></span> that is altered with a heterozygosity model, we solved for the general solution of <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-33.gif"/></noscript></span></span>, genotype frequencies, using <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Matlab (2010)</a>, and an M-file is provided in <a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/FileS1.pdf">File S1</a> to document all variables defined and calculations.</p></div><div id="sec-10" class="subsection"><h4>Implementation and simulation:</h4><p id="p-22">Calculations of the genotype frequencies for proportions of heterozygosity maintained, <em>h</em>, other than 0.5 were implemented in C within a fork of the R/qtl v1.28.19 code base (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Broman <em>et al.</em> 2003</a>). Specifically we used the golden section search algorithm as implemented in the R/qtl <em>BC<sub>s</sub>F<sub>t</sub></em> tools (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Shannon <em>et al.</em> 2013</a>) to estimate recombination fractions given genotype data for a marker pair. Map distances were calculated using the Haldane mapping function given the recombination fractions estimated from the golden section search.</p><p id="p-23">The source code is available on GitHub as a forked R/qtl repository at <a href="https://github.com/MulletLab/qtl">https://github.com/MulletLab/qtl</a>. The hetexp branch contains the new functions, including est.rf.exHet() that can be called from R similar to the existing est.rf() but with a heterozygosity term, <em>h</em>, passed to it. The est.rf.exHet() function can also estimate <em>h</em> on the basis of H for each linkage group. Example usage can be found at <a href="https://github.com/MulletLab/exHet_Supplement">https://github.com/MulletLab/exHet_Supplement</a>.</p><p id="p-24">Genotypes for a 200-cM linkage group genotyped for 1000 individuals at 1000 markers were simulated under the derived heterozygosity model both (i) without errors or missing data, and (ii) with 1% errors and 5% missing data. The code used to generate the datasets, the simulated datasets, and their respective results can be found at <a href="https://github.com/MulletLab/exHet_Supplement">https://github.com/MulletLab/exHet_Supplement</a>.</p></div></div><div id="sec-11" class="subsection"><h3>Plant materials and genotyping</h3><p id="p-25">The sorghum recombinant inbred mapping population, BT×623 × IS3620C, were made available by the USDA-ARS Plant Genetic Resource and Conservation Unit, Griffin, GA (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burow <em>et al.</em> 2011</a>). These <em>F</em><sub>7–9</sub> individuals were planted in fields in College Station, TX, in the summer of 2013. DNA was extracted from leaf tissue of 10−12 plants from seed stock of each RIL and prepared by digital genotyping with restriction endonuclease <em>NgoMIV</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Morishige <em>et al.</em> 2013</a>). The digital genotyping templates were sequenced on Illumina HiSequation 2500 with 72 (or fewer) samples per lane.</p><p id="p-26">Genotypes were generated from the sequenced reads of the recombinant inbred lines and their parents, BT×623 and IS3620C. The sequence reads were delivered already sorted on sample barcode, and they were checked for restriction sites using awk; where applicable, preprocessing was parallelized using GNU parallel (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Tange 2011</a>). Reads were aligned to the sorghum reference genome (Sbi1) with BWA mem (v 0.7.5a) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Paterson <em>et al.</em> 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li and Durbin 2010</a>). Aligned reads were realigned around indels using the Genome Analysis Toolkit (GATK v3.1-1) and the Queue framework with IndelRealigner; individual GVCFs were generated using the HaplotypeCaller; and joint genotyping was performed using GenotypeGVCFs (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Depristo <em>et al.</em> 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Van Der Auwera <em>et al.</em> 2013</a>). Variants were hard filtered using VariantFiltration under the following criteria: DP &lt; 10; QD &lt; 5.0; MQ &lt; 30.0; MQRankSum &lt; ×10.0; BaseQRankSum &lt; −10.0. The remaining variants were filtered to keep only biallelic variants for which the two parents, BT×623 and IS3620C, were each homozygous for different alleles and to keep only variants that were genotyped with a GQ score ≥ 20 in ≥ 25% of the samples. For these genotypes, the median depth of reads that passed the HaplotypeCaller’s internal quality control metrics (<em>i.e.</em>, the median sample-level DP annotation) was 17 reads. Genotypes with a GQ score &lt;20 were set to missing, and those remaining were screened for tight double recombinations occurring within 2 kbp; genotypes involved in a tight double recombination were set to missing. These variants and genotypes were used as the initial input for genetic map construction in R/qtl.</p></div><div id="sec-12" class="subsection"><h3>Genetic map construction</h3><p id="p-27">Genetic map construction was performed as an iterative process in R/qtl, starting with 424 individuals (RILs) genotyped at 12,836 single-nucleotide polymorphisms and indel markers. Two individuals and 1340 markers were removed due to high missingness levels (≥60%), seven individuals were removed due to sharing ≥90% of genotypes with another individual, 703 markers were removed for being uninformative due to close proximity, and 17 individuals were removed for having genotypic proportions far outside the distribution of most of the population members. The remaining 398 individuals and 10,793 markers had an overall 7.4% heterozygous genotypes. The percentage 7.4% was used as an initial <em>H</em><sub>0</sub> to test for segregation distortion. Markers that deviated largely from a 463:74:463 ratio (homozygous parent 1:heterozygous:homozygous parent 2) by a χ<sup>2</sup> test (<em>P</em> &lt; 1 × 10<sup>−15</sup>) were excluded; due to extreme segregation distortion for one parental allele on chromosome 1 caused by artificial selection of a known flowering time QTL, we dramatically relaxed the distortion threshold (<em>P</em> &lt; 1 × 10<sup>−30</sup>) for chromosome 1, although a large gap still remains due to failing to retain markers in the region of the most severe distortion (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Yang <em>et al.</em> 2014</a>). After filtration, 398 individuals and 10,090 markers remained with 7.5% heterozygous genotypes. 7.5% was considered to better represent the true percentage of heterozygous genotypes in the dataset, and so we applied the segregation distortion test for the 10,793 markers with the updated <em>H</em><sub>1</sub> of 7.5%; this retained 398 individuals typed at 10,091 markers. Of note, at our <em>P</em>-value thresholds, the expected Mendelian ratio (<em>H</em> = 1.6%) retained only 4512 markers, whereas the excess heterozygosity model (<em>H</em> = 7.5%) retained 10,091 markers.</p><p id="p-28">With the 398 individuals typed at 10,091 markers, we then constructed an initial genetic map by estimating recombination fractions calculated under the excess heterozygosity model and R/qtl’s implemented Haldane mapping function with markers grouped and ordered by their physical position on the Sbi1 reference genome. Ten markers on chromosome 6 were removed due to their incorrect placement on the Sbi1 reference assembly, as indicated by inspection of recombination fractions and previous work (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Morishige <em>et al.</em> 2013</a>). The genetic map was then re-estimated, and tight double recombinations less than or equal to 2.0 cM were removed. The proportion of heterozygosity at this point, <em>H</em> = 6.7%, was used to estimate <em>h</em> for use in the final map estimation under the excess heterozygosity model; the same markers and genotypes were used for map estimation under the Mendelian model. Genetic maps were estimated directly from calculated pairwise recombination fractions for adjacent markers using R/qtl’s implemented est.rf() and our implemented est.rf.exHet().</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Excess heterozygosity generally causes overestimation of recombination frequencies</h3><p id="p-29">To demonstrate how excess heterozygosity expands genetic maps, we plotted the estimated recombination frequency, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-34.gif"/></noscript></span></span>, given genotype counts expected under conditions of excess heterozygosity for different recombination frequencies, <em>r</em>, estimated using the Mendelian model and using the derived heterozygosity model (see the section <em>Materials and Methods</em> for the derivation and implementation). When the genotype counts for the two markers arise from an excess heterozygosity model for an <em>F</em><sub>7</sub> RIL population, accounting for the excess heterozygosity when calculating <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-35.gif"/></noscript></span></span> correctly estimates the recombination frequency, <em>r</em>, underlying the data (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). However, use of the Mendelian model to estimate <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-36.gif"/></noscript></span></span> results in an overestimation relative to the recombination frequency underlying the data. Overestimation of <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-37.gif"/></noscript></span></span> decreases as linkage increases (<em>r</em> ≤ 0.3), and even these small overestimations between many pairs of markers lead to map expansion proportional to the genetic distance of the region with excess heterozygosity.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1963/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated recombination frequencies, , under excess heterozygosity and Mendelian models. Recombination frequencies estimated from genotype frequencies under Mendelian expectations (h = 0.5) vs. under modeling a global heterozygosity advantage (h = 0.6373) at generation t = 7 of a selfing population. This shows that if the population was retaining excess heterozygosity (at a rate of 63.73% each generation as opposed to the Mendelian 50%), then estimating recombination fractions under Mendelian expectations would lead to overestimation of the recombination frequency underlying the data and subsequent map expansion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1855613920" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Estimated recombination frequencies, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-38&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, under excess heterozygosity and Mendelian models. Recombination frequencies estimated from genotype frequencies under Mendelian expectations (&lt;em&gt;h&lt;/em&gt; = 0.5) &lt;em&gt;vs.&lt;/em&gt; under modeling a global heterozygosity advantage (&lt;em&gt;h&lt;/em&gt; = 0.6373) at generation &lt;em&gt;t&lt;/em&gt; = 7 of a selfing population. This shows that if the population was retaining excess heterozygosity (at a rate of 63.73% each generation as opposed to the Mendelian 50%), then estimating recombination fractions under Mendelian expectations would lead to overestimation of the recombination frequency underlying the data and subsequent map expansion.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1963/F1.medium.gif" width="440" height="345"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/10/1963/F1.medium.gif" width="440" height="345"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1963/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1963/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464990" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-30" class="first-child">Estimated recombination frequencies, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/F1/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/F1/embed/mml-math-38.gif"/></noscript></span></span>, under excess heterozygosity and Mendelian models. Recombination frequencies estimated from genotype frequencies under Mendelian expectations (<em>h</em> = 0.5) <em>vs.</em> under modeling a global heterozygosity advantage (<em>h</em> = 0.6373) at generation <em>t</em> = 7 of a selfing population. This shows that if the population was retaining excess heterozygosity (at a rate of 63.73% each generation as opposed to the Mendelian 50%), then estimating recombination fractions under Mendelian expectations would lead to overestimation of the recombination frequency underlying the data and subsequent map expansion.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">To further demonstrate the effects of excess heterozygosity, we simulated an <em>F</em><sub>7</sub> RIL population of 1000 individuals with a 200-cM linkage group covered by 1000 markers under conditions of excess heterozygosity maintained per generation (<em>h</em> = 0.6373). Estimating recombination frequencies under a Mendelian model (<em>h</em> = 0.5) overestimates the map by 18.0% (236.0 cM), whereas accounting for excess heterozygosity in the genetic model yields a genetic map that differs from the simulated distance by only 2.5% (204.9 cM) (<a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/FigureS5.tif">Figure S5</a>).</p></div><div id="sec-15" class="subsection"><h3>Incorporation of a heterozygosity term into the genetic model shrinks a sorghum genetic map</h3><p id="p-32">To demonstrate that accounting for excess heterozygosity can shrink the genetic map of a plant recombinant inbred population (as postulated by <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Knox and Ellis 2002</a>), we applied our method to a sorghum recombinant inbred population displaying excess heterozygosity (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Burow <em>et al.</em> 2011</a>). The members of the population ranged from <em>F</em><sub>7</sub> to <em>F</em><sub>9</sub> and exhibit more than a 300% increase in heterozygosity relative to the expected heterozygosity given a Mendelian model: 6.7% observed after our quality control steps <em>vs.</em> 1.6% given a Mendelian model for <em>t</em> = 7 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Heterozygosity was present at elevated levels throughout the genome relative to expectations under a Mendelian model, although some regions deviated notably from the average (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/FigureS2.pdf">Figure S2</a>, and <a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/FigureS3.tif">Figure S3</a>). Previous reports estimating the genetic map as an RIL that has gone to fixation for this sorghum population range from 1279 cM to 1713 cM, a difference of 8.48 recombinations per meiosis (<a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/TableS2.tif">Table S2</a>) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Peng <em>et al.</em> 1999</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hart <em>et al.</em> 2001</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Menz <em>et al.</em> 2002</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mace <em>et al.</em> 2009</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Burow <em>et al.</em> 2011</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1963/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Excess heterozygosity in a sorghum mapping population. Box plot of genotype frequencies of 398 individuals of the BT&#xD7;623 &#xD7; IS3620C recombinant inbred population. Each individual has a percentage of its genotypes that are homozygous or heterozygous for a BT&#xD7;623 parental allele, A, and IS3620C parental allele, a. The dashed red lines represent the expected genotype frequencies under the assumptions of Mendelian segregation. The expected heterozygous frequency is lower than the median observed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1855613920" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Excess heterozygosity in a sorghum mapping population. Box plot of genotype frequencies of 398 individuals of the BT&#xD7;623 &#xD7; IS3620C recombinant inbred population. Each individual has a percentage of its genotypes that are homozygous or heterozygous for a BT&#xD7;623 parental allele, A, and IS3620C parental allele, a. The dashed red lines represent the expected genotype frequencies under the assumptions of Mendelian segregation. The expected heterozygous frequency is lower than the median observed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1963/F2.medium.gif" width="265" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/10/1963/F2.medium.gif" width="265" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1963/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1963/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465097" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-33" class="first-child">Excess heterozygosity in a sorghum mapping population. Box plot of genotype frequencies of 398 individuals of the BT×623 × IS3620C recombinant inbred population. Each individual has a percentage of its genotypes that are homozygous or heterozygous for a BT×623 parental allele, A, and IS3620C parental allele, a. The dashed red lines represent the expected genotype frequencies under the assumptions of Mendelian segregation. The expected heterozygous frequency is lower than the median observed.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/10/1963/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accounting for excess heterozygosity shrinks the sorghum genetic map. This plot shows the genetic position of 10,081 markers for two genetic maps. For each chromosome, the genetic map on the left is calculated under the Mendelian segregation model. The genetic map on the right is calculated under the excess heterozygosity model. For all chromosomes (#1&#x2212;10), the map shrinks by accounting for excess heterozygosity. The coloring of the markers correspond to the percentage of heterozygosity at that locus (no heterozygosity, white, to high (&gt;11%) heterozygosity, purple). The expected heterozygosity of an F7 RIL population is 1.6% and the observed heterozygosity in the BT&#xD7;623 &#xD7; IS3620C population was 6.7% as depicted on the color bar. Faint gray lines connect a marker&#x2019;s position in one map with its corresponding position in the other map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1855613920" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Accounting for excess heterozygosity shrinks the sorghum genetic map. This plot shows the genetic position of 10,081 markers for two genetic maps. For each chromosome, the genetic map on the left is calculated under the Mendelian segregation model. The genetic map on the right is calculated under the excess heterozygosity model. For all chromosomes (#1&#x2212;10), the map shrinks by accounting for excess heterozygosity. The coloring of the markers correspond to the percentage of heterozygosity at that locus (no heterozygosity, white, to high (&gt;11%) heterozygosity, purple). The expected heterozygosity of an &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;7&lt;/sub&gt; RIL population is 1.6% and the observed heterozygosity in the BT&#xD7;623 &#xD7; IS3620C population was 6.7% as depicted on the color bar. Faint gray lines connect a marker&#x2019;s position in one map with its corresponding position in the other map.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/10/1963/F3.medium.gif" width="440" height="254"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/10/1963/F3.medium.gif" width="440" height="254"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/10/1963/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/10/1963/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465056" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Accounting for excess heterozygosity shrinks the sorghum genetic map. This plot shows the genetic position of 10,081 markers for two genetic maps. For each chromosome, the genetic map on the left is calculated under the Mendelian segregation model. The genetic map on the right is calculated under the excess heterozygosity model. For all chromosomes (#1−10), the map shrinks by accounting for excess heterozygosity. The coloring of the markers correspond to the percentage of heterozygosity at that locus (no heterozygosity, white, to high (&gt;11%) heterozygosity, purple). The expected heterozygosity of an <em>F</em><sub>7</sub> RIL population is 1.6% and the observed heterozygosity in the BT×623 × IS3620C population was 6.7% as depicted on the color bar. Faint gray lines connect a marker’s position in one map with its corresponding position in the other map.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">The genotype calls for this population were used to parameterize the heterozygosity term, <em>h</em>, by treating the population as an <em>F</em><sub>7</sub> such that <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-39.gif"/></noscript></span></span>= 0.067 and <em>h</em> = 0.6373 by equation 2; <em>u</em> and <em>d</em> were subsequently found by equation 3 (<em>Materials and Methods</em>). <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> compares the genetic maps of the sorghum recombinant inbred population estimated as an <em>F</em><sub>7</sub> under Mendelian expectations (<em>h</em> = 0.5) on the left and estimated under the excess heterozygosity model (<em>h</em> = 0.6373) on the right for each chromosome. Once excess heterozygosity is accounted for, the genetic map shrinks from 1603.8 cM to 1390.6 cM, a 213.2-cM difference corresponding to a 13% decrease, or 4.26 recombinations fewer recombinations per meiosis. As expected, the derived heterozygosity model behaves identically to the Mendelian model when <em>h</em> = 0.5 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465016/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465016/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genetic maps estimated from the BT×623 × IS3620C sorghum recombinant inbred mapping population</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-37">Observations that deviate from a model’s expectations, such as segregation distortion caused by excess heterozygosity, will generally cause the model to generate inaccurate estimations; unsurprisingly, excess heterozygosity leads to unexpected map lengths when the genetic map is estimated under the assumptions of Mendelian segregation. Here we have shown that the excess heterozygosity present in a sorghum recombinant inbred population caused map expansion under Mendelian expectations. However there is no theoretical reason why excess heterozygosity could not also shrink the genetic map under certain conditions. If we had observed this recombinant inbred population in its <em>F</em><sub>3</sub> stage and parametrized an <em>h</em> = 0.6373 (the same amount of heterozygosity maintained each generation), excess heterozygosity in this case would cause recombination frequencies to be underestimated under the Mendelian model (<a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/FigureS1.pdf">Figure S1</a>). Although this result does not agree with the idea that excess heterozygosity always causes map expansion (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Knox and Ellis 2002</a>), it is not an unexpected one; our assumptions are dependent on a RIL approaching fixation (<em>t</em> → ∞), in which case the longer maintenance of heterozygous loci provides more opportunities for recombination at the given loci. In other words, in the context of the genetic model derived in the <em>Materials and Methods</em>, as <em>t</em> → ∞, the proportion of class 2 genotypes (<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-40.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-41.gif"/></noscript></span></span>) will be larger for populations with excess heterozygosity than those following Mendelian expectations. Under our model and moderate values of <em>h</em> (<em>e.g.</em>, <em>h</em> = 0.6373), excess heterozygosity is predicted to cause map shrinkage for small generation values (<em>e.g.</em>, <em>t</em> = 3), and map expansion for larger generation values (<em>e.g.</em>, <em>t</em> = 7). The general case is simply that, when the observed genotype frequencies deviate from those predicted by the model, the estimated recombination fractions, <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-42.gif"/></noscript></span></span>, will be inaccurate.</p><p id="p-38">Our modeling was done under the assumption that the amount of heterozygosity maintained by each generation is evenly distributed among markers. While our data for this population show that excess heterozygosity is present throughout the genome, there is also local variation (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/FigureS2.pdf">Figure S2</a>). This finding agrees with previous work showing that hybrid advantage and/or disadvantage can localize to specific loci in the genome (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li <em>et al.</em> 1997</a>), and in these cases it may be more appropriate to obtain an <em>h</em><sub>marker pair</sub> from data <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/10/1963/embed/mml-math-43.gif"/></noscript></span></span> for each marker pair (which we derive in <a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/FileS1.pdf">File S1</a>). However we chose not to implement this method for our mapping population to avoid overfitting the data and to maintain an expected global value with which the genetic map could be curated. We have additionally implemented an option to parameterize <em>h</em> for each linkage group based on the <em>H</em> of the genotypes for the linkage group, though this yielded little difference for our use case (<a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/FigureS5.tif">Figure S5</a>). Ultimately, our solution strikes a balance between an <em>a priori</em> model based on Mendelian segregation and parameterizing the model based entirely on each marker pair. Future work may explore an intermediate approach based on estimating regional heterozygosity levels to paramaterize <em>h</em> for groups of markers; an examination of the biological mechanisms underlying these regional deviations from the global level of maintained heterozygosity is also merited (<a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/FigureS3.tif">Figure S3</a>, and <a href="http://www.g3journal.org/content/suppl/2014/08/15/g3.114.012468.DC1/FigureS4.tif">Figure S4</a>).</p><p id="p-39">We have made the implementation of this method available as a fork of the R/qtl code base at <a href="https://github.com/MulletLab/qtl">https://github.com/MulletLab/qtl</a>, and provide examples for its use at <a href="https://github.com/MulletLab/exHet_Supplement">https://github.com/MulletLab/exHet_Supplement</a> so that it can be used where appropriate for conditions of excess heterozygosity; we are grateful to the R/qtl developers for making their code base accessible to be built upon (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Broman <em>et al.</em> 2003</a>). We believe this work serves as an example of when to extend a genetic model to fit observations of biological phenomena that deviate from traditional expectations, and that the differential zygotic viability model (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Zhu <em>et al.</em> 2007</a>) will serve as a useful base to modify as the mechanisms underlying segregation distortion become better understood. As genotyping technologies continue to improve, so too should the models we use to interpret the phenomena underlying the data. Deviations from traditional models, including segregation distortion and tight double recombinations, will need to be corrected to generate genetic maps that have reasonable agreement with the cytological maps calculated using microscopy to observe indicators of recombination events. More accurate genetic maps will improve linkage based analyses such as map-based cloning, marker-assisted selection, and QTL mapping, as well as assist marker ordering for genome assembly and provide better estimates of how recombination is distributed in the genome.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">We thank the matplotlib developers and contributors for the open-source Python module that we used to generate the figures (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hunter 2007</a>). This work was supported in part by Great Lakes Bioenergy Research Center and the Perry Adkisson Chair (to J.E.M.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-41"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-06-02"><span class="received-label">Received </span>June 2, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-08-12"><span class="accepted-label">Accepted </span>August 12, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Truong <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brigham</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooker</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">High-resolution crossover maps for each bivalent of <em>Zea mays</em> using recombination nodules.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAnderson%26rft.auinit1%253DL.%2BK.%26rft.volume%253D165%26rft.issue%253D2%26rft.spage%253D849%26rft.epage%253D865%26rft.atitle%253DHigh-Resolution%2BCrossover%2BMaps%2Bfor%2BEach%2BBivalent%2Bof%2BZea%2Bmays%2BUsing%2BRecombination%2BNodules%26rft_id%253Dinfo%253Apmid%252F14573493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY1LzIvODQ5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xOTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.2" data-doi="10.1086/302923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Characterization of human crossover interference.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">1911</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D66%26rft.issue%253D6%26rft.spage%253D1911%26rft.epage%253D1926%26rft.atitle%253DCharacterization%2Bof%2Bhuman%2Bcrossover%2Binterference.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F302923%26rft_id%253Dinfo%253Apmid%252F10801387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/302923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10801387&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088373800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.3" data-doi="10.1086/302011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheffield</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Comprehensive human genetic maps: individual and sex-specific variation in recombination.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">861</span>–<span class="cit-lpage">869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D63%26rft.issue%253D3%26rft.spage%253D861%26rft.epage%253D869%26rft.atitle%253DComprehensive%2Bhuman%2Bgenetic%2Bmaps%253A%2Bindividual%2Band%2Bsex-specific%2Bvariation%2Bin%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F302011%26rft_id%253Dinfo%253Apmid%252F9718341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/302011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9718341&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075919000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.4" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTAvMTk2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1963.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-source">The Mathematical Theory of Quantitative Genetics</span>. <span class="cit-publ-name">Clarendon Press</span>, <span class="cit-publ-loc">Chicago</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.6" data-doi="10.3198/jpr2010.04.0219crmp"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franks</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schertz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Registration of the BTx623/IS3620C recombinant inbred mapping population of sorghum.</span> <abbr class="cit-jnl-abbrev">J. Plant Registrations</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BRegistrations%26rft.volume%253D5%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.3198%252Fjpr2010.04.0219crmp%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3198/jpr2010.04.0219crmp&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.7" data-doi="10.1007/s00122-012-1953-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cloutier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ragupathy</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miranda</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radovanovic</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reimer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Integrated consensus genetic and physical maps of flax (<em>Linum usitatissimum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1783</span>–<span class="cit-lpage">1795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DCloutier%26rft.auinit1%253DS.%26rft.volume%253D125%26rft.issue%253D8%26rft.spage%253D1783%26rft.epage%253D1795%26rft.atitle%253DIntegrated%2Bconsensus%2Bgenetic%2Band%2Bphysical%2Bmaps%2Bof%2Bflax%2B%2528Linum%2Busitatissimum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1953-0%26rft_id%253Dinfo%253Apmid%252F22890805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1953-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22890805&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.8" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.9" data-doi="10.1038/380152a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dib</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fauré</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fizames</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drouot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A comprehensive genetic map of the human genome based on 5,264 microsatellites.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">380</span>: <span class="cit-fpage">152</span>–<span class="cit-lpage">154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDib%26rft.auinit1%253DC.%26rft.volume%253D380%26rft.issue%253D6570%26rft.spage%253D152%26rft.epage%253D154%26rft.atitle%253DA%2Bcomprehensive%2Bgenetic%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome%2Bbased%2Bon%2B5%252C264%2Bmicrosatellites.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F380152a0%26rft_id%253Dinfo%253Apmid%252F8600387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/380152a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8600387&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TZ97800048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waddington</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1931</span> <span class="cit-article-title">Inbreeding and linkage.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHaldane%26rft.auinit1%253DJ.%2BB.%2BS.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D374%26rft.atitle%253DINBREEDING%2BAND%2BLINKAGE%26rft_id%253Dinfo%253Apmid%252F17246626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIxNi80LzM1NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTAvMTk2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">T. H. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span><span class="cit-comment">a</span> <span class="cit-article-title">The relationship between genetic and cytogenetic maps of pea. I. Standard and translocation karyotypes.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">744</span>–<span class="cit-lpage">754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DHall%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D744%26rft.epage%253D754%26rft.atitle%253DThe%2Brelationship%2Bbetween%2Bgenetic%2Band%2Bcytogenetic%2Bmaps%2Bof%2Bpea.%2BI.%2BStandard%2Band%2Btranslocation%2Bkaryotypes.%26rft_id%253Dinfo%253Apmid%252F18464862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18464862&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">T. H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knox</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span><span class="cit-comment">b</span> <span class="cit-article-title">The relationship between genetic and cytogenetic maps of pea. II. Physical maps of linkage mapping populations.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">755</span>–<span class="cit-lpage">769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DHall%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D755%26rft.epage%253D769%26rft.atitle%253DThe%2Brelationship%2Bbetween%2Bgenetic%2Band%2Bcytogenetic%2Bmaps%2Bof%2Bpea.%2BII.%2BPhysical%2Bmaps%2Bof%2Blinkage%2Bmapping%2Bpopulations.%26rft_id%253Dinfo%253Apmid%252F18464863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18464863&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.13" data-doi="10.1007/s001220100582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schertz</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Syed</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic mapping of <em>Sorghum bicolor</em> (L.) Moench QTLs that control variation in tillering and other morphological characters.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">1232</span>–<span class="cit-lpage">1242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D103%26rft.spage%253D1232%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220100582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220100582&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.14" data-doi="10.1109/MCSE.2007.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Matplotlib: A 2D graphics environment.</span> <abbr class="cit-jnl-abbrev">Comput. Sci. Eng.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">90</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BSci.%2BEng.%26rft.volume%253D9%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMCSE.2007.55%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/MCSE.2007.55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kearsey</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A demonstration of a 1:1 correspondence between chiasma frequency and recombination using a <em>Lolium perenne</em>/<em>Festuca pratensis</em> substitution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKing%26rft.auinit1%253DJ.%26rft.volume%253D161%26rft.issue%253D1%26rft.spage%253D307%26rft.epage%253D314%26rft.atitle%253DA%2BDemonstration%2Bof%2Ba%2B1%253A1%2BCorrespondence%2BBetween%2BChiasma%2BFrequency%2Band%2BRecombination%2BUsing%2Ba%2BLolium%2Bperenne%252FFestuca%2Bpratensis%2BSubstitution%26rft_id%253Dinfo%253Apmid%252F12019244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYxLzEvMzA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xOTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knox</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">T. H. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Excess heterozygosity contributes to genetic map expansion in pea recombinant inbred populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">861</span>–<span class="cit-lpage">873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKnox%26rft.auinit1%253DM.%2BR.%26rft.volume%253D162%26rft.issue%253D2%26rft.spage%253D861%26rft.epage%253D873%26rft.atitle%253DExcess%2BHeterozygosity%2BContributes%2Bto%2BGenetic%2BMap%2BExpansion%2Bin%2BPea%2BRecombinant%2BInbred%2BPopulations%26rft_id%253Dinfo%253Apmid%252F12399396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYyLzIvODYxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNC8xMC8xOTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.17" data-doi="10.1016/0888-7543(87)90010-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrahamson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barlow</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Mapmaker: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D181%26rft.atitle%253DMAPMAKER%253A%2Ban%2Binteractive%2Bcomputer%2Bpackage%2Bfor%2Bconstructing%2Bprimary%2Bgenetic%2Blinkage%2Bmaps%2Bof%2Bexperimental%2Band%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0888-7543%252887%252990010-3%26rft_id%253Dinfo%253Apmid%252F3692487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0888-7543(87)90010-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3692487&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.18" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long-read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi81LzU4OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTAvMTk2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinson</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stansel</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Genetics of hybrid sterility and hybrid breakdown in an intersubspecific rice (<em>Oryza sativa</em> L.) population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">1139</span>–<span class="cit-lpage">1148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLi%26rft.auinit1%253DZ.%26rft.volume%253D145%26rft.issue%253D4%26rft.spage%253D1139%26rft.epage%253D1148%26rft.atitle%253DGenetics%2Bof%2BHybrid%2BSterility%2Band%2BHybrid%2BBreakdown%2Bin%2Ban%2BIntersubspecific%2BRice%2B%2528Oryza%2Bsativa%2BL.%2529%2BPopulation%26rft_id%253Dinfo%253Apmid%252F9093864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NS80LzExMzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE5NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.20" data-doi="10.1016/S0888-7543(05)80158-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lincoln</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Systematic detection of errors in genetic linkage data.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">604</span>–<span class="cit-lpage">610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLincoln%26rft.auinit1%253DS.%2BE.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D604%26rft.epage%253D610%26rft.atitle%253DSystematic%2Bdetection%2Bof%2Berrors%2Bin%2Bgenetic%2Blinkage%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0888-7543%252805%252980158-2%26rft_id%253Dinfo%253Apmid%252F1427888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0888-7543(05)80158-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1427888&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JW35200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.21" data-doi="10.1007/BF00220998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorieux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrier</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de León</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanaud</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span><span class="cit-comment">a</span> <span class="cit-article-title">Maximum-likelihood models for mapping genetic markers showing segregation distortion. 1. Backcross populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D90%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220998%26rft_id%253Dinfo%253Apmid%252F24173786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220998&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24173786&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QF11000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.22" data-doi="10.1007/BF00220999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorieux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrier</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanaud</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de León</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span><span class="cit-comment">b</span> <span class="cit-article-title">Maximum-likelihood models for mapping genetic markers showing segregation distortion. 2. F2 populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D90%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220999%26rft_id%253Dinfo%253Apmid%252F24173787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24173787&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QF11000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.23" data-doi="10.1186/1471-2229-9-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rami</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A consensus genetic map of sorghum that integrates multiple component maps and high-throughput Diversity Array Technology (DArT) markers.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DMace%26rft.auinit1%253DE.%2BS.%26rft.volume%253D9%26rft.spage%253D13%26rft.epage%253D13%26rft.atitle%253DA%2Bconsensus%2Bgenetic%2Bmap%2Bof%2Bsorghum%2Bthat%2Bintegrates%2Bmultiple%2Bcomponent%2Bmaps%2Band%2Bhigh-throughput%2BDiversity%2BArray%2BTechnology%2B%2528DArT%2529%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-13%26rft_id%253Dinfo%253Apmid%252F19171067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19171067&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1963.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">MATLAB</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Matlab, version 7.10.0 (R2010a)</span>. <span class="cit-publ-name">The MathWorks Inc</span>., <span class="cit-publ-loc">Natick, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.25" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzQvMTAvMTk2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.26" data-doi="10.1023/A:1014831302392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Menz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Unruh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A high-density genetic map of <em>Sorghum bicolor</em> (L.) Moench based on 2926 AFLP, RFLP and SSR markers.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DMenz%26rft.auinit1%253DM.%2BA.%26rft.volume%253D48%26rft.issue%253D5-6%26rft.spage%253D483%26rft.epage%253D499%26rft.atitle%253DA%2Bhigh-density%2Bgenetic%2Bmap%2Bof%2BSorghum%2Bbicolor%2B%2528L.%2529%2BMoench%2Bbased%2Bon%2B2926%2BAFLP%252C%2BRFLP%2Band%2BSSR%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1014831302392%26rft_id%253Dinfo%253Apmid%252F11999830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1014831302392&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11999830&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174606400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.27" data-doi="10.1186/1471-2164-14-448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morishige</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hilley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahraeian</span>  <span class="cit-name-given-names">S. M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Digital genotyping of sorghum—a diverse plant species with a large repeat-rich genome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D448%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-448%26rft_id%253Dinfo%253Apmid%252F23829350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23829350&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.28" data-doi="10.1038/nature07723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruggmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubchak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The <em>Sorghum bicolor</em> genome and the diversification of grasses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D457%26rft.issue%253D7229%26rft.spage%253D551%26rft.epage%253D556%26rft.atitle%253DThe%2BSorghum%2Bbicolor%2Bgenome%2Band%2Bthe%2Bdiversification%2Bof%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07723%26rft_id%253Dinfo%253Apmid%252F19189423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19189423&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262852200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.29" data-doi="10.1046/j.1439-0523.1999.118003225.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schertz</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cartinhour</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Comparative genome mapping of <em>Sorghum bicolor</em> (L.) Moench using an RFLP map constructed in a population of recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D118%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1439-0523.1999.118003225.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1439-0523.1999.118003225.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Users guide for new <em>BC<sub>s</sub>F<sub>t</sub></em> Tools for R/qtl.</span> Available from: <a href="http://cran.r-project.org/web/packages/qtl/vignettes/bcsft.pdf">http://cran.r-project.org/web/packages/qtl/vignettes/bcsft.pdf</a>. Accessed August 27, 2014.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sybenga</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Recombination and chiasmata: few but intriguing discrepancies.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DSybenga%26rft.auinit1%253DJ.%26rft.volume%253D39%26rft.issue%253D3%26rft.spage%253D473%26rft.epage%253D484%26rft.atitle%253DRecombination%2Band%2Bchiasmata%253A%2Bfew%2Bbut%2Bintriguing%2Bdiscrepancies.%26rft_id%253Dinfo%253Apmid%252F18469909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18469909&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tange</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GNU parallel—the command-line power tool.;login</span>. <abbr class="cit-jnl-abbrev">The USENIX Magazine</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">42</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BUSENIX%2BMagazine%26rft.volume%253D36%26rft.spage%253D42%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1963.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Auwera</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">del Angel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">From FastQ Data to High-Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline</span>. <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.34" data-doi="10.1017/S0016672305007779"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mapping segregation distortion loci and quantitative trait loci for spikelet sterility in rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWang%26rft.auinit1%253DC.%26rft.volume%253D86%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D106%26rft.atitle%253DMapping%2Bsegregation%2Bdistortion%2Bloci%2Band%2Bquantitative%2Btrait%2Bloci%2Bfor%2Bspikelet%2Bsterility%2Bin%2Brice%2B%2528%2BOryza%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672305007779%26rft_id%253Dinfo%253Apmid%252F16356283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672305007779&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16356283&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-4.10.1963.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source">Statistical Genetics of Quantitative Traits: Linkage, Maps and QTL</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.36" data-doi="10.1534/genetics.108.090688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Quantitative trait locus mapping can benefit from segregation distortion.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">2201</span>–<span class="cit-lpage">2208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090688%26rft_id%253Dinfo%253Apmid%252F18957707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC80LzIyMDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy80LzEwLzE5NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.37" data-doi="10.1186/1471-2229-14-148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weers</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morishige</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullet</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CONSTANS is a photoperiod regulated activator of flowering in sorghum.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D14%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-14-148%26rft_id%253Dinfo%253Apmid%252F24884377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-14-148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24884377&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.10.1963.38" data-doi="10.1007/s00122-006-0432-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Modeling segregation distortion for viability selection I. reconstruction of linkage maps with distorted markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhu%26rft.auinit1%253DC.%26rft.volume%253D114%26rft.issue%253D2%26rft.spage%253D295%26rft.epage%253D305%26rft.atitle%253DModeling%2Bsegregation%2Bdistortion%2Bfor%2Bviability%2Bselection.%2BI.%2BReconstruction%2Bof%2Blinkage%2Bmaps%2Bwith%2Bdistorted%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0432-x%26rft_id%253Dinfo%253Apmid%252F17119913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0432-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17119913&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F10%2F1963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/10/1963.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/10/1955" title="Performance of the Cas9 Nickase System in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/10/1971" title="A Saturated Genetic Linkage Map of Autotetraploid Alfalfa (Medicago sativa L.) Developed Using Genotyping-by-Sequencing Is Highly Syntenous with the Medicago truncatula Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/10"><span>Volume 4 Issue 10, October 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/10.cover-source.jpg?itok=f3Vstm56" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/10/1963" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ScJJsBg4A2WGnb4F3Ip1R1ubxpzwS_kOwuzcr-O7EpQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/10/1963" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/10/1963" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/10/1963" class="active" data-icon-position="" data-hide-link-title="0">Resolution of Genetic Map Expansion Caused by Excess Heterozygosity in Plant Recombinant Inbred Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459691" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-UD2Ztax05DG_S3SPmmeMUUk1VTimXvToCZjUmcR6l2M" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/10/1963.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/10/1963" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-q1waX8qm3bPGwJxsofmp-IeZkDkASTs4itGf07THbHA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/10/1963" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/10/1963.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459691" id="node459691--2" data-pisa="ggg;4/10/1963" data-pisa-master="ggg;g3.114.012468" data-apath="/ggg/4/10/1963.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resolution of Genetic Map Expansion Caused by Excess Heterozygosity in Plant Recombinant Inbred Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sandra K. Truong</span>, <span class="highwire-citation-author" data-delta="1">Ryan F. McCormick</span>, <span class="highwire-citation-author" data-delta="2">Daryl T. Morishige</span> and <span class="highwire-citation-author" data-delta="3">John E. Mullet</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1963-1969;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012468 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459691--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sandra K. Truong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BK.%2BTruong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truong%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BK.%2BTruong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ryan F. McCormick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BF.%2BMcCormick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCormick%20RF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BF.%2BMcCormick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daryl T. Morishige </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daryl%2BT.%2BMorishige%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morishige%20DT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaryl%2BT.%2BMorishige%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">John E. Mullet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BE.%2BMullet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mullet%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BE.%2BMullet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmullet@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">jmullet@tamu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/10/1963" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/10/1963&amp;title=Resolution%20of%20Genetic%20Map%20Expansion%20Caused%20by%20Excess%20Heterozygosity%20in%20Plant%20Recombinant%20Inbred%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/10/1963&amp;title=Resolution%20of%20Genetic%20Map%20Expansion%20Caused%20by%20Excess%20Heterozygosity%20in%20Plant%20Recombinant%20Inbred%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/10/1963&amp;title=Resolution%20of%20Genetic%20Map%20Expansion%20Caused%20by%20Excess%20Heterozygosity%20in%20Plant%20Recombinant%20Inbred%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/10/1963&amp;text=Resolution%20of%20Genetic%20Map%20Expansion%20Caused%20by%20Excess%20Heterozygosity%20in%20Plant%20Recombinant%20Inbred%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/10/1963&amp;title=Resolution%20of%20Genetic%20Map%20Expansion%20Caused%20by%20Excess%20Heterozygosity%20in%20Plant%20Recombinant%20Inbred%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/10/1963&amp;t=Resolution%20of%20Genetic%20Map%20Expansion%20Caused%20by%20Excess%20Heterozygosity%20in%20Plant%20Recombinant%20Inbred%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/10/1963&amp;title=Resolution%20of%20Genetic%20Map%20Expansion%20Caused%20by%20Excess%20Heterozygosity%20in%20Plant%20Recombinant%20Inbred%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/10/1963&amp;title=Resolution%20of%20Genetic%20Map%20Expansion%20Caused%20by%20Excess%20Heterozygosity%20in%20Plant%20Recombinant%20Inbred%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459691/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459691" id="node459691--3" data-pisa="ggg;4/10/1963" data-pisa-master="ggg;g3.114.012468" data-apath="/ggg/4/10/1963.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resolution of Genetic Map Expansion Caused by Excess Heterozygosity in Plant Recombinant Inbred Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sandra K. Truong</span>, <span class="highwire-citation-author" data-delta="1">Ryan F. McCormick</span>, <span class="highwire-citation-author" data-delta="2">Daryl T. Morishige</span> and <span class="highwire-citation-author" data-delta="3">John E. Mullet</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1963-1969;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012468 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459691--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sandra K. Truong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BK.%2BTruong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truong%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BK.%2BTruong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ryan F. McCormick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BF.%2BMcCormick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCormick%20RF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BF.%2BMcCormick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daryl T. Morishige </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daryl%2BT.%2BMorishige%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morishige%20DT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaryl%2BT.%2BMorishige%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">John E. Mullet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Interdisciplinary Program in Genetics, Texas A&M University, College Station, Texas 77843</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Biochemistry & Biophysics Department, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BE.%2BMullet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mullet%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BE.%2BMullet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmullet@tamu.edu" class="" data-icon-position="" data-hide-link-title="0">jmullet@tamu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1963&amp;rft.epage=1969&amp;rft.atitle=Resolution%20of%20Genetic%20Map%20Expansion%20Caused%20by%20Excess%20Heterozygosity%20in%20Plant%20Recombinant%20Inbred%20Populations&amp;rft.volume=4&amp;rft.issue=10&amp;rft.date=2014-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Truong%2C+Sandra+K.&amp;rft.au=McCormick%2C+Ryan+F.&amp;rft.au=Morishige%2C+Daryl+T.&amp;rft.au=Mullet%2C+John+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459691/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459691/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459691/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459691/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459691/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459691/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459691/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459691/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459691/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459691/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459691/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459691/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/10/1963" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/10/1963.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/10/1963.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/10/1963.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/10/1963.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
