<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/5/1563" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/5/1563.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/5/1563.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/5/1563.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/5/1563.atom" />
<meta name="HW.pisa" content="ggg;7/5/1563" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (Pinus lambertiana)" />
<meta name="DC.Identifier" content="10.1534/g3.117.040055" />
<meta name="DC.Date" content="2017-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Crepeau et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We investigate the utility and scalability of new read cloud technologies to improve the draft genome assemblies of the colossal, and largely repetitive, genomes of conifers. Synthetic long read technologies have existed in various forms as a means of reducing complexity and resolving repeats since the outset of genome assembly. Recently, technologies that combine subhaploid pools of high molecular weight DNA with barcoding on a massive scale have brought new efficiencies to sample preparation and data generation. When combined with inexpensive light shotgun sequencing, the resulting data can be used to scaffold large genomes. The protocol is efficient enough to consider routinely for even the largest genomes. Conifers represent the largest reference genome projects executed to date. The largest of these is that of the conifer Pinus lambertiana (sugar pine), with a genome size of 31 billion bp. In this paper, we report on the molecular and computational protocols for scaffolding the P. lambertiana genome using the library technology from 10× Genomics. At 247,000 bp, the NG50 of the existing reference sequence is the highest scaffold contiguity among the currently published conifer assemblies; this new assembly’s NG50 is 1.94 million bp, an eightfold increase." />
<meta name="DC.Contributor" content="Marc W. Crepeau" />
<meta name="DC.Contributor" content="Charles H. Langley" />
<meta name="DC.Contributor" content="Kristian A. Stevens" />
<meta name="article:published_time" content="2017-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (Pinus lambertiana)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We investigate the utility and scalability of new read cloud technologies to improve the draft genome assemblies of the colossal, and largely repetitive, genomes of conifers. Synthetic long read technologies have existed in various forms as a means of reducing complexity and resolving repeats since the outset of genome assembly. Recently, technologies that combine subhaploid pools of high molecular weight DNA with barcoding on a massive scale have brought new efficiencies to sample preparation and data generation. When combined with inexpensive light shotgun sequencing, the resulting data can be used to scaffold large genomes. The protocol is efficient enough to consider routinely for even the largest genomes. Conifers represent the largest reference genome projects executed to date. The largest of these is that of the conifer &lt;i&gt;Pinus lambertiana&lt;/i&gt; (sugar pine), with a genome size of 31&amp;nbsp;billion&amp;nbsp;bp. In this paper, we report on the molecular and computational protocols for scaffolding the &lt;i&gt;P. lambertiana&lt;/i&gt; genome using the library technology from 10× Genomics. At 247,000&amp;nbsp;bp, the NG50 of the existing reference sequence is the highest scaffold contiguity among the currently published conifer assemblies; this new assembly’s NG50 is 1.94&amp;nbsp;million&amp;nbsp;bp, an eightfold increase.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/05/01" />
<meta name="citation_mjid" content="ggg;7/5/1563" />
<meta name="citation_id" content="7/5/1563" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/5/1563" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/5/1563.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/5/1563.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/5/1563.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.040055" />
<meta name="citation_pmid" content="28341701" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1563" />
<meta name="citation_lastpage" content="1568" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marc W. Crepeau" />
<meta name="citation_author_institution" content="Department of Evolution and Ecology, University of California, Davis, California 95616" />
<meta name="citation_author" content="Charles H. Langley" />
<meta name="citation_author_institution" content="Department of Evolution and Ecology, University of California, Davis, California 95616" />
<meta name="citation_author" content="Kristian A. Stevens" />
<meta name="citation_author_institution" content="Department of Evolution and Ecology, University of California, Davis, California 95616" />
<meta name="citation_author_email" content="kastevens@ucdavis.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Adey;citation_author=J. O. Kitzman;citation_author=J. N. Burton;citation_author=R. Daza;citation_author=A. Kumar;citation_title=In vitro, long-range sequence information for de novo genome assembly via transposase contiguity.;citation_volume=24;citation_year=2014;citation_pmid=25327137;citation_doi=10.1101/gr.178319.114" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. Alexeyenko;citation_author=B. Nystedt;citation_author=F. Vezzi;citation_author=E. Sherwood;citation_author=R. Ye;citation_title=Efficient de novo assembly of large and complex genomes by massively parallel sequencing of fosmid pools.;citation_pages=439;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. Amini;citation_author=D. Pushkarev;citation_author=L. Christiansen;citation_author=E. Kostem;citation_author=T. Royce;citation_title=Haplotype-resolved whole-genome sequencing by contiguity-preserving transposition and combinatorial indexing.;citation_pages=1343-1351;citation_volume=46;citation_year=2014;citation_pmid=25326703;citation_doi=10.1038/ng.3119" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=I. Birol;citation_author=A. Raymond;citation_author=S. D. Jackman;citation_author=S. Pleasance;citation_author=R. Coope;citation_title=Assembling the 20 Gb white spruce (Picea glauca) genome from whole-genome shotgun sequencing data.;citation_volume=29;citation_year=2013;citation_pmid=23698863;citation_doi=10.1093/bioinformatics/btt178" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Duitama;citation_author=G. K. McEwen;citation_author=T. Huebsch;citation_author=S. Palczewski;citation_author=S. Schulz;citation_title=Fosmid-based whole genome haplotyping of a HapMap trio child: evaluation of single individual haplotyping techniques.;citation_volume=40;citation_year=2012;citation_pmid=22102577;citation_doi=10.1093/nar/gkr1042" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=V. Kuleshov;citation_author=M. P. Snyder;citation_author=S. Batzoglou;citation_title=Genome assembly from synthetic long read clouds.;citation_volume=32;citation_year=2016;citation_pmid=27307620;citation_doi=10.1093/bioinformatics/btw267" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=S. Kurtz;citation_author=A. Phillippy;citation_author=A. L. Delcher;citation_author=M. Smoot;citation_author=M. Shumway;citation_title=Versatile and open software for comparing large genomes.;citation_pages=1;citation_volume=5;citation_year=2004;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long-read alignment with Burrows–Wheeler transform.;citation_volume=26;citation_year=2010;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=Y. Mostovoy;citation_author=M. Levy-Sakin;citation_author=J. Lam;citation_author=E. T. Lam;citation_author=A. R. Hastie;citation_title=A hybrid approach for de novo human genome sequence assembly and phasing.;citation_pages=587-590;citation_volume=13;citation_year=2016;citation_pmid=27159086;citation_doi=10.1038/nmeth.3865" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=E. W. Myers;citation_title=The fragment assembly string graph;citation_pages=ii79-ii85;citation_volume=21;citation_year=2005;citation_issue=suppl_2;citation_pmid=16204131;citation_doi=10.1093/bioinformatics/bti1114" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. B. Neale;citation_author=J. L. Wegrzyn;citation_author=K. A. Stevens;citation_author=A. V. Zimin;citation_author=D. Puiu;citation_title=DECODING THE MASSIVE GENOME OF LOBLOLLY PINE USING HAPLOID DNA AND NOVEL ASSEMBLY STRATEGIES.;citation_pages=R59;citation_volume=15;citation_year=2014;citation_pmid=24647006;citation_doi=10.1186/gb-2014-15-3-r59" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. Nystedt;citation_author=N. R. Street;citation_author=A. Wetterbom;citation_author=A. Zuccolo;citation_author=Y.-C. Lin;citation_title=The Norway spruce genome sequence and conifer genome evolution.;citation_pages=579-584;citation_volume=497;citation_year=2013;citation_pmid=23698360;citation_doi=10.1038/nature12211" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. A. Peters;citation_author=B. G. Kermani;citation_author=A. B. Sparks;citation_author=O. A. Alferov;citation_author=P. Hong;citation_title=Accurate whole-genome sequencing and haplotyping from 10 to 20 human cells.;citation_pages=190-195;citation_volume=487;citation_year=2012;citation_issue=7406;citation_pmid=22785314;citation_doi=10.1038/nature11236" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. A. Stevens;citation_author=J. L. Wegrzyn;citation_author=A. Zimin;citation_author=D. Puiu;citation_author=M. Crepeau;citation_title=Sequence of the sugar pine megagenome;citation_volume=204;citation_year=2016;citation_pmid=27794028;citation_doi=10.1534/genetics.116.193227" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_author=A. Voskoboynik;citation_author=N. F. Neff;citation_author=D. Sahoo;citation_author=A. M. Newman;citation_author=D. Pushkarev;citation_title=The genome sequence of the colonial chordate, Botryllus schlosseri.;citation_volume=2;citation_year=2013;citation_pmid=23840927;citation_doi=10.7554/eLife.00569" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=R. L. Warren;citation_author=C. I. Keeling;citation_author=M. M. S. Yuen;citation_author=A. Raymond;citation_author=G. A. Taylor;citation_title=Improved white spruce (Picea glauca) genome assemblies and annotation of large gene families of conifer terpenoid and phenolic defense metabolism.;citation_pages=189-212;citation_volume=83;citation_year=2015;citation_pmid=26017574;citation_doi=10.1111/tpj.12886" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Zhang;citation_author=X. Fang;citation_author=X. Guo;citation_author=L. Li;citation_author=R. Luo;citation_title=The oyster genome reveals stress adaptation and complexity of shell formation.;citation_pages=49-54;citation_volume=490;citation_year=2012;citation_issue=7418;citation_pmid=22992520;citation_doi=10.1038/nature11413" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Zimin;citation_author=K. A. Stevens;citation_author=M. W. Crepeau;citation_author=A. Holtz-Morris;citation_author=M. Koriabine;citation_title=Sequencing and Assembly of the 22-Gb Loblolly Pine Genome;citation_pages=875-890;citation_volume=196;citation_year=2014;citation_issue=3;citation_pmid=24653210;citation_doi=10.1534/genetics.113.159715" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=A. Zimin;citation_author=K. A. Stevens;citation_author=M. W. Crepeau;citation_author=D. Puiu;citation_author=J. L. Wegrzyn;citation_title=An improved assembly of the loblolly pine mega-genome using long-read single-molecule sequencing;citation_pages=1-4;citation_volume=6;citation_year=2017" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (Pinus lambertiana)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="twitter:description" content="We investigate the utility and scalability of new read cloud technologies to improve the draft genome assemblies of the colossal, and largely repetitive, genomes of conifers. Synthetic long read technologies have existed in various forms as a means of reducing complexity and resolving repeats since the outset of genome assembly. Recently, technologies that combine subhaploid pools of high molecular weight DNA with barcoding on a massive scale have brought new efficiencies to sample preparation and data generation. When combined with inexpensive light shotgun sequencing, the resulting data can be used to scaffold large genomes. The protocol is efficient enough to consider routinely for even the largest genomes. Conifers represent the largest reference genome projects executed to date. The largest of these is that of the conifer Pinus lambertiana (sugar pine), with a genome size of 31 billion bp. In this paper, we report on the molecular and computational protocols for scaffolding the P. lambertiana genome using the library technology from 10× Genomics. At 247,000 bp, the NG50 of the existing reference sequence is the highest scaffold contiguity among the currently published conifer assemblies; this new assembly’s NG50 is 1.94 million bp, an eightfold increase." />
<meta name="og-title" property="og:title" content="From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (Pinus lambertiana)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/5/1563" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We investigate the utility and scalability of new read cloud technologies to improve the draft genome assemblies of the colossal, and largely repetitive, genomes of conifers. Synthetic long read technologies have existed in various forms as a means of reducing complexity and resolving repeats since the outset of genome assembly. Recently, technologies that combine subhaploid pools of high molecular weight DNA with barcoding on a massive scale have brought new efficiencies to sample preparation and data generation. When combined with inexpensive light shotgun sequencing, the resulting data can be used to scaffold large genomes. The protocol is efficient enough to consider routinely for even the largest genomes. Conifers represent the largest reference genome projects executed to date. The largest of these is that of the conifer Pinus lambertiana (sugar pine), with a genome size of 31 billion bp. In this paper, we report on the molecular and computational protocols for scaffolding the P. lambertiana genome using the library technology from 10× Genomics. At 247,000 bp, the NG50 of the existing reference sequence is the highest scaffold contiguity among the currently published conifer assemblies; this new assembly’s NG50 is 1.94 million bp, an eightfold increase." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
    <title>From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (Pinus lambertiana) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mo37JnbzT264ETh3-CZ7IMyeUf0-ZHQJ_qOxiQL1-YA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484257","apath":"\/ggg\/7\/5\/1563.atom","pisa":"ggg;7\/5\/1563","ac":{"\/ggg\/7\/5\/1563.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/5\/1563.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1563"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1563"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/5\/1563.figures-only":true,"\/content\/7\/5\/1563":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gNeeXA1bImO76BO0rcJGelfxtCxwoLcENkxsMUJeTU55UlDMSt5qBEY2lNkzHJ+M0qQoUix2NmOgmO1KAiug5gt14PHpiEYInId3hDBRZKOb7NhN8XQY9mMs0OVfefLwnS\\\\\\\/M3BsY37X4QusZDSNZBF7o8cxYEoofQ6tnxRy6PBW9YKF0USafRxjwAkFRBwklkC4xEK1wn5rIc20pGGcSmlrf5ysEyCj+TltFVE9Gy7Rz7L48D4kiV95mj49fdFq15Pe+L\\\\\\\/EQ\\\\\\\/qjK3xg\\\\\\\/J8KLqpy1BNBdVNizl62H8tknC\\\\\\\/yqjt\\\\\\\/I9tBBqkOLGdA5Wpp3TdFioooCNL\\\\\\\/QS4TnavNGDXznKd9Dtt0OWu3SiB579wXUF5zXCYH0u2Pg5dSZSvZAzpjPlGeNWBGqgp9X\\\\\\\/sfMFHBSs0HR6w8pgLl77IrK\\\\\\\/95B8Hsp2ilR0wbO4pBmYo\\\\\\\/1Oodh1k9wtvyO9bY1ppZ1qYFJKSfTO4OlF83SfewoP9OKED7lY2KbdymriNGBeR5xrgOA3bwrEZ7voLiKawtTdJJLmy0RSE0A7tzC03W7v\\\\\\\/UU0NzOUGzpf0iHbIOCMtE3ph\\\\\\\/\\\\\\\/8LQuTMAZ82OOtk\\\\\\\/burRA0oVvIHJUo5QwZ02M+7UDQkDc9BYhUKdx2B0qGeRJB3lo1\\\\\\\/oipZrrsmp79rPUV4Tv4Eqc9w7nUYz4PMCGeqRGYRFNqODeM5MPuswOXCqWpjlR\\\\\\\/J97\\\\\\\/4VB1gxkepEf7USf8kyOlSQjgPsL0+f6heQr5SV4rbqQ925jZaLkRx\\\\\\\/tE+Pj8L2cI5Ro32eG9jUbKdZXpCqXlvl\\\\\\\/WSAKGiZJ18NSgvpyMsxia8BMPIyZa2uVuh\\\\\\\/glikVor24x852qvqYMUHEv830vV50U\\\\\\\/hqYmozvTM1Rlb0vezjhE6cj4yP4b8+RB1EYDUY1cMK8C+ZtTj2j9iTY\\\\\\\/PhWJO7lvFUUQiVW+ApaRo4RXwmhNXD\\\\\\\/xCVWyMsV8SKLG1mkgHzNl\\\\\\\/lXc8mXDXv9acnD0eN1koRd2QiZ0A\\\\\\\/6LIJRnvBfEHulrfECddIVjqAT1j00qNlOfDqDZGmaOBkNtOTTSjs0uXA3PnX6d0nu1rL2ph0VJ\\\\\\\/yVaBasyvAOAuWR7nl4gPB59xa\\\\\\\/UDWe6BVAn4dsKidIlZo2IUJga\\\\\\\/qVHti0MdOkCnhA6Cs69X47BrynB7YNoWjpABJ+NKcetLyazulqqV7oGf1atwHRDSGHimZ29501PUnJZR3fcMvXdHpKVeb7tqd2TCpbMCscNL7uB+JZg9FUb8wqhfVowz4iDzHSR00XmKMZ9z9wqjV\\\u0022,\\\u0022iv\\\u0022:\\\u0022AxvrjC6cIu9vbT0m0Dmf3Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228f21d914dc424075c1cce94f795ccfcd9e12abff60c297669b0daa3e02c0e403\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226jRnxYVVr282IJEBMVcRdH5pGLeHFSX5dKB4Ar\\\\\\\/\\\\\\\/ZLepTV65KnPyvoLK9F5SR2ydBkDg0Tr778koe\\\\\\\/Oz\\\\\\\/5Dce0H6RaJMvn1NqdOhPbuhJfbgGzhe96\\\\\\\/XVaV2e5aCJWBItiu6a46F9Ug9IBZlOdX3izujgBuvFrC02+g58uEr3SuYfTdDG1Veuv97G9PVSjhaYGyJtBEC53EXTCD+vOo3Cctp6uZUWprVYW0ebw99BCeaZ1FkL7fAKPYlhETULAab5gSB+SAZHC2AIDooTBGXH\\\\\\\/NH2Zzp2L4BSnnLkhqgind3iAyBck6e2eik73At+ikmJS7h1WWqScl4ScoPJ6yU+PppXM42WevpPw7RvPi\\\\\\\/YoV+ZkCd1ZPVYd8qvmE\\\\\\\/9SedU2NizuIEtcXPfhJvh9SVHSUl9LRmuUbmKi33AnzcIz6RweN1pSJeINbuT0k7Xu3gFICAAb+5k4Ns02Hg0PlMHOF9euG94PbFcbU7+7q9OjqqZt6wq8BLhkLDriv02z9IquntWRR1goZ6bwfZHkUeU+e0aqHjFowa+x51FIDLwGNONcVaQp0u3R6SqiZRErLndo1IRQZJZHHTbfRlehWX\\\\\\\/rBeG2V\\\\\\\/uaAWn\\\\\\\/tXt\\\\\\\/J9kx+QedmKthC6UyD8WiQhIMS3NEvn9uH4EJbVmH5vrJewBHSsOLcIpNCCvvl3sARmuP+OzX4y5GfwiSVjdIosHX90+ZKOg4JQ5guHoC\\\\\\\/mdNKFOrh1m\\\\\\\/SgL65Qy8gAJHWfx3iS3bitS+H64WH5fq4FFzQmGXNIXrn9VIIxwiORqhfcJQ7EoQG\\\\\\\/JO9zqWCT6mYOdMNmAH2qey67VJSFUxsnv56+jvCAjtd2KU\\\\\\\/BNRPPDUAlbQ\\\\\\\/St1BWlK5HuPugJf6OqB+0uN+l7AD6SmpaZQ1Z6B\\\\\\\/xx4FRLoC+kKiLAQVxHzju2OUQ39k8nAvUb+NTl7SMjsDd3R4OoqOxyhJjWcDZK6ELWctpFSoQYfhUda5lAmBEr8YJFFh8Pq3Vp6xn4BGOJwSsNLE11FIWDbuMdxtont6D\\\\\\\/HX\\\\\\\/SLdWaGttWfg6GfqOYNBpZj9HZdfTj05+Zzdb64JcEPj\\\\\\\/QU98X+JeITPr+fyCA3FAnpOksR8C7iJWIwVP3WHACUDUAmPwahG4SIMSlNNhoa8\\\\\\\/Z04mjdZrNBobkalMmAEOhkwoUxgsN6yTydh5v8Ylgo+oVjE24vhqafhgGDfNGfwVEw18OhW16KKoNk3QEjWaoJ04QhIf9HsJwDYIF\\\\\\\/6XEkMkpbn\\\\\\\/FMEnAxaItpo9wf6rdngoCW7h1qpQXquFqBogryPpidUCHAAfVdvSRtW46bAwmOr\\\\\\\/oCXSdhM3Z7kmQtqBWbl38Lpf\\\\\\\/KU8F34NLq+VFSddO5+xZip+JbRi1wIA+kJNtEyoz5S97TXHjbX5DqMraHnLtyOwHVfoOFNTkmXdgo15a8gklJJ4xscnzQQ67XobtzKB8Au0obUF8B+aWoh7EmsdB5BdDsv+2P5reN9JqiuEr+Jlt7RPXYHz5vZfv4UISn1jDx4jt6UY6ekBDl3VJgL+3qoHJkTqTwIELtk6mEgq+H6LgLtJd6qmoSYLfgGkX72K\\\\\\\/cP6On4f\\\\\\\/YLmpCIpm6jqktkA+Fm7ywZGrtWaXhz1Xm9IGY5D\\\\\\\/qLjLv4r3YZSvddBX+7exyuriiXlTUoPoQVcTp6s7+YToEcCEz7Lb2Cid0GzTeZ9Ge0WyP3gjp8y4aiijCPoZpVQgjMsUwyoCTLoSc7vlILtN5HN0csgvksZmU790W73O2sgrkpUvDzsgDStp+vnhZJPEpqTYvD4RgKeSrEb3K6nGDvJoEfyKP60S7LccwxQxetfVnrWQpT4U3pVI6ryxu4PyHS8RbCEQdrn2zAX8wR7ZddiFoyKnMJiz\\\\\\\/nOFbJCnu9rcJw6YhZyIM4ool7QJrpxes0jpUqTEzcnW+ST+C09i3CacAL8TbDpd883wUhIGtmqfriCl5c0BznW337zYfaD54Gnyyz9hviRBe8xPhGvyLgRzXzkFWn9n39sggl+WGLofGUvzB2bb30ZSEt6TWfwB+rZKJ93uKJMMBPVmpKl6dYZACQNH5H6RL1At4MJqzkjfRDE\\\\\\\/IXlmhoiC4sE5yxXgz9cnAkBVtdlSRSt\\\\\\\/q7ynwa8Y7CJAMvhvM7copkcf4amg5bgOn\\\\\\\/Tt4R1q4MBVwHHdkUPjgKN6d3YKityjjmbY63CftbJ\\\\\\\/FOVx\\\\\\\/ZMxCWGTICDTRkbt1qCOAz4ErtEOWXC4i7rhSXpcoSBupXj+7CPY6gP0hkInC\\\\\\\/OM2Hwi9SeyDTDtOw9HsLgGmtS3ruT0ls4\\\\\\\/sl+H76m4jVzFlUuie5uTwgXamZgiqGipK1Puxi1FFbPu8CPoUz23nwqcllqI\\\\\\\/n54V4Pko5NBtX9Cls93tIGgy8RbfBcwVJjAvlFTmwomcw\\\\\\\/aqqP7XFEh9HmMIwsWzaYujKpeY1Bm9qGSENkyTCmStaxFqNIpPriXe1a5tJbjUFZWggoMLPR0P11XUIJIU1tBqaz7RMzjbJKySzd7YZ3aEvo7h3d\\\\\\\/WaFdxVRsnyrxS27BRDHDH55NrBgCI6pdIrzlpLLhM7xegwAjniflJErRibIdUTnny9RazzoyVXALoSSEIfN9pert9B0biv\\\\\\\/40ojWgkENEvOzZ23DxaBabNhopIr1Y\\\\\\\/8rcCrtjQPG5k8unUaR2IcaMkT0wQ1gwSXkJEbjXItCChFJj4H4cozZs7iBR5xtIlm6ZWPDKL\\\\\\\/ztLbOH0KHLvm0DGkwt3aa9YyNWXIan7KSGAPfoZYimzS+rIzRa+ssklmCnfk0XICNREexDzCQnmLBQ1wv6mudyYjPii5BDsHwYzraDExgAYuqcwoiAB\\\\\\\/zdARGAZ1QMnP\\\\\\\/rB59XHyAB0fgVvfSeq8NZ\\\\\\\/vtsyv6R17fM8nqrhbxRUC2pU8We6hTq+vodtqXivGdHm3QUBiyM+0+Zx6gPCvj\\\\\\\/z7oWAv9yk+\\\\\\\/6THfiny+CFfU73kmrrw0c2pRgjopNSX2hgud0d2LYFDmcdTmMakfp12DaR1sAevaxo1w3\\\\\\\/ksSWti8xRtUNHl+SPUw\\\\\\\/\\\\\\\/snuQN49V+oANwODPJCmBNepbM8oeAyGC6+pH9nyKSh\\\\\\\/DjcYUBbXxG+TNOfAxLbXyxrIM+DBz9t27Wr6AbsGL3S2z6XQ1KnYZQfXCmgOlWPFHmwmFztUjriTslSyOh2bYhHN9+E070UxEm4HMkhwB+8VlL6YxC+YPJ6WN2bqoZ7aZgQNkqZnqWDIMFXTkcPnxx7+yATgR6OOcnXA5C5LTrFSia5667\\\\\\\/QV\\\\\\\/Kp9+MnwK9JkBaQt+FLVL5dgfx\\\\\\\/nmHrXqjLojH\\\\\\\/kuYplRONYVGWb4VVMFmm4kRI2x4AKlonmpPt9Oy67BNeoxKOAQlDPSEFIXJEpoX6kpXGE0PLXYIC02zu7OnmrmtAJW8238Q8W39OCzhQCqbddaT\\\\\\\/Lz2MNi7\\\\\\\/CuUBQtOLesVTtlKFBz6VYe20nSuUieTmi9A8XHXUSy0Kku9HwkbI0BVjvlWTeukMP18qRMDcpLPToLm9INLDbhFm8XrCJoIGzljd\\\\\\\/jxWDJesUHqp2xH6k\\\\\\\/N0LXhwCn7g7rd8it1EAFV1Q\\\\\\\/dQGmjuO0CTwiu0uIyRqMMEoywA85cRjDiBhQAlcpl3tJif2zWyMbZOtiu2YvF4xZqSSV\\\\\\\/WE2rTm89OGOsxR\\\\\\\/6V8awbYDKeVO0VbiX8NdkWUhyuVaANxNHMwJNs6oB50xnG4uf9HaXp+XGlmx5b2FxEx3Kac3ieokZmKxtrcXIKYAPYuYMek\\\\\\\/MCHx0b7+MsKFYRYMnDRBNrMviD+6la5yQ3d6pXjwRO0zVBo88vUBPrkNYh58AnBVYCadu3x8yhQq2grJ3o\\\\\\\/x9GPzldnXhjmE3wvKBcyRK+uHsL\\\\\\\/6LEHkNQ\\\\\\\/CFAOLKaTZz5pz5BnJOx5gx4VU4ePLYULaPb85gGOpc4h15u\\\\\\\/jy9MDFxmBLWvnnWOUjBievIlEBmY7r4IGLGrYDgME2sWhDv0Cp6t9VA2\\\\\\\/DxCiwrFsuwfelnsIk8r6Gtq6zZzPm85GMwGKjzWVk+ujCePubUcJlYfY+1TP0GskJoYZd5ulKamuzxpmFsMTSrKec2GdzPKx488CyzJ7lrg3ov7yXOFemsv+AMry79NH8IXoDDyEhHnWVi19u8od88HQJHl5EeW12VmHE2ZwSsbw4gy0oLSosLIzNj5Wq3JeFJp4LPBTVNerbug0Zu\\\\\\\/Fyp84gkfmqzO02ZSh4vWoiSAfRixfXCpKuGiZargHKWffeuwvYt9PVyRlr4zwJlkrsXXc5xN6d1wxSk8mr3hbt9Or7vsvpY\\\\\\\/Zwah9xdZgraprmxCrUVjq48TRkLxUZ5CoNPYy+v7n7qkFwAVasYzLC2TXTXy+WLwR2M52z5nFIyGs1FNC8Jxa9dWT16O8+9+umYaARI2iKxDKxUtZBzNVghgpoOhLKk1WECZa2M8\\\\\\\/O998g5yI5+pNQxND1+AE8eChp2TKxB8BTgohMksGF0smw8\\\\\\\/5FTvxDUncB4edflFVfCjRvAGq2RvSj4YD6tfgUjs1yB7nsQ1U\\\\\\\/SGI3G8\\\\\\\/f2u6Y+FwI1jtahQTmKmUsE4oWWDblfnJfw+Ge6iHXfKkIOjSDJpHd0yHjeDCb2fn984fUGIGq7fiX2E5poeiMEBuILJ\\\\\\\/m0lEHokckm4T7DOQU2dyDaHWLTPSWevhARtPtkeziWS+vxT\\\\\\\/xBflemR854IGuz+9g+iqGq+BOzBs0Jmm18bnf1OnItiDXXxld3vH+YQ0CEQeBJGiCdDod7Xp54qB2ZcHsbxVfDHU8FcdvYrhAI7L\\\\\\\/WgC+b2PWQhgO5apz+ohz62zjrk9iuQlArOFjUmCdRMdGpqpEaPT49wnc8pb7UkpYTSpvlOC7VLV2Yll5ZNpYt4tNX+BjZxeLEhs42XtpO9IazvGWsrdVEkGVAztTkHvPajdjktjhGzRCGsUIjmvLMoDK5n3WsK6i+jKESYYu8wyjcnF9gtXbdV9KoVzJCuNFxHIDs1UdqbsraIg9bP6JViifXPXA\\\\\\\/74TR47w2TVGEzUERF3JLNuiJsmFWVx+onhM0zlk9dg0e+UOWeTpqiNHVlsC3scE009PczgyV\\\\\\\/5B4WQvY9PZ4qZ0rnz4ll9niEX7iI4lZLP9owperdt+Ml7wse7OIRjbpAZ0htSjWb3pvnO4a2nB2zJZ7QVJhrmYSdhuZw6KvjqaA3gPfL05\\\\\\\/hPlgLa3vZpKpFVBL+QA6obPiDHLBeZKSekGsygeOdsG4UL7tJwZG2j8bVMYv7\\\\\\\/gWGBC+PpV9S739Eg4XBoAYEQTq7lQKH+ZNPBLXCYrcRPEJyrgEj8JoNFq2GLfZgoXwufhxjXFYuW\\\\\\\/7J\\\\\\\/qWrF8KQKE3g+o2d\\\\\\\/lirLudfe11HaBoj9xz5ADBiDbVCxqfe6qRA2MwKfHZAY2j3y3lWNLkk8ffRdkqhjteg56v7H4oacvaNDB1m82dboZ0yIZZRvoOE8Ltgv37Yg3sLAwwZhGv2Pkiqajiq8Orkh9RC4ftDTBDoEuYrzdUkBW1J4IXcU0FNQzjPIHZ\\\\\\\/yeAI2hN9cvdtWYaR4KF4HLsMUdj8mqbWBkJRM8X5zfDblRPW4q\\\\\\\/OILBmvudW6Dx\\\\\\\/lJ9T1NF1Sz3UC8SL1Ax9oeZpNmV\\\\\\\/fMSZHupUP1FSrpsImacQh\\\\\\\/VY1F9ROQskxUa7iP03qczA169QWLis0H+9Ek6bXXGq8Lk7G3Fkj\\\\\\\/RTveT0Fn0jtD9PnVO85d4baEHOt2FEJmXw7p2ozCci78ggeg6RSzyJRG1Xe7JjkDk37TcizE8yvtNKV3Yo7xv6EikkkBioycgHJ0Ii8Ir1PoGCd2E4\\\\\\\/8E7ukc6pUp\\\\\\\/FkZ8+shT3es+zTMKTGrJRd8rG4C8qgDFcRt0JdDywRePu7H9pmfoG1nCnM5ZjKmxhFwZNP3ZUk74lM1VHwRXthDq85vHjzFTbGM0ueq4GldO4qKic38gr1S\\\\\\\/1Un5D5TRRK2z0tjNJVkDCuT+csz8YDHdTovvNELsT6FLHpAzI9mLKTK+yXcXwf2K6apxAJ6buh\\\\\\\/XIUWxEpFbQyBL+ocwuu7ic9YAXNPoQF5aZfF49cxcOc7iEhRvyBJusdrI5poms6Gc4NfqTj\\\\\\\/aZM\\\\\\\/HxQFLciKl\\\\\\\/HQQAUAKNK7fMd3PQ0EoL5FyQ\\\\\\\/rsQhQnhSfpRHhUjqTg3zz3SIteWQyiJNrUMNsayvdzK8dB9xUHtOFvMxEMghVSxiTSC6\\\\\\\/iUVhTtdd6v56dx5gMjRs6wKSaadJg0aOCT4MA+QcZe1qEnkLYe3wpKCW02LIU2C70MoqF9H2OIL908TzlgAaFCkh0wl3qZ3okxJFuaryN8SQvbawUN6MybsZmBcCtfIb7a+ASDCxJ68umNpkxVTRNdkzT1m644bbepzQa1i+dyW\\\\\\\/hu1PUzha7mmQEddLECZoSAzq7ZsJu8YNHqRR7wYWoZcHFo4I6Z2L2Sw+oU0XlHsSbg2\\\\\\\/UEINEInT5LJovpz9Y9DUrQq1RQn9CvwCXOOY\\\\\\\/YwkB18E3V\\\\\\\/0wij5XUGtxaG3ByhokCJtasbP0I0W2MA78kPIwzN63bZaxpzkb0UPMUGFXa\\\\\\\/t5Ftm9vtHFUq4S9L\\\\\\\/zTNh7quoGeW5Y5jc3Tp12ZBjAL4AgUPkJIJniQRmPSCOLzHDItiZp1\\\\\\\/EyWP8j6ZHwfqhD4RPzNw5x1BknVx7JO+coIYho30n1NeBT9yQrrLAxEtEok\\\\\\\/hBBy5Xxwgc2RBPNDQASwv3+EoKI3w+XQbSYObaHPH1Z8Y482x92P+nPrMRC2krcobCWO2P2tbHOBMYcu4AtSAdlS6n22lozIDkgutOJ5AZ5xUgaVjy0cqXaJlzK4fpCEqwzZ3nILWCQfibLa8aNt+u67rATmnECcZjn5vbJtqFookzZt5hebuOMzcAY4NTG0JHVv1AtN0VpNES4Iy8xyqFU6Num9V9RHEM7NMXIaBaUeYUX2fjUNX6PkFD7u0bTLzyPf\\\\\\\/7\\\\\\\/6z9ucCylID1\\\\\\\/LIvYvVqrKxOlMQtHFl0W0CGDanqthY6T8WuTqVx627P\\\\\\\/hs\\\\\\\/QgOUTYw0WzCoz1HScEVbLqMSSt2vfRtaEcqFl2q5OGGoLsCmgbU3+PLWRL0zOhO1tHbPXJGSVUvil2rZyuvi5b5OEHl1gNT+yasiulZEjWTmPuhWcHAlhhmBXm5PyNh4b\\\\\\\/yKWJTsjE4A8xNEqYmskIGZeL5vfuk+JAQkwONwO0fNQ2sMI6GaBBMyL\\\\\\\/NOujfB8Ufi1a3L1aXPAQIlY9khCbkS\\\\\\\/IvdaAEulBmdAaSpBOZDT9kzwQkmkIafE4lx+dBeqthSlu1iRIvmrA5Tix1HVg\\\\\\\/WkbINwaA2\\\\\\\/fpURy+nYxGD\\\\\\\/5AtAmuscwpDu2CPwC0oXOrapVQ6cDTlC6TWcvj7eGejCQF\\\\\\\/azrAzwJqSVealE1NXo5RY9h4cPY6xjb4CgZVz\\\\\\\/g9BcXvYfqdNruoxTv2Z3DxQLw6NYtFILxvOdlfmoAX3u2IfeNRRmIE6VIAEhVZPYRGS471JwZCl2odkRgfbyLc7D1Ph\\\\\\\/OHZLM1FvTuWEoBd9fIIHoLAeyAjEPJBqsTjFypOJkBrZaO8pq5qo0SQS0EEdsrSI6ie5vJMvzQ0X6NC8FkmCDkK22enmMy9wMVV2Qnx3qgmT4UTinrQnvIKLD87l8GWnM1WP+U\\\\\\\/Y+4vSt9wa+WAFG6H1kUJbSdGEsf7xc7gTQ0Ne0IIREjxAZcbq9wIlCLxC9OS+UjWqZ9YaKDPiQ0fH+QuYUWmJ0qbvtUEH3oTs1O9zyE52o0U+qRXDItke6wij8pyfm7QTbPxFChNd8rpfFE+kQl8f2jPgcTBsaWIEaaZNOaMIWc4gnljsIcHwiRQ99eaH+AU+bd3ySCTM+G03+jfEdTU2vzlhSh4PIyiQHjSc5LhygcUw\\\\\\\/XplePyQIEL89nFFTDh7Zp1j3x9OK89GMoJZh1QIAuOnHSZE4+ejyJfxeSP0GXI1rg52S5szVaW+a9rZFvK7ZGoB5BGjg1UGKRiat5jgO\\\\\\\/eGTt98NidBEX\\\\\\\/3pT7crblLNqbCJ4SW5xM0FRP0oXuL+jUZZzv3vS\\\\\\\/Gmh83iCJLIExLb3N6QL8kOgnbu9JMTy7BH8BrC+zKMdervhce77HwT0N+jAnUD+JgMRJ+U2QG1d2oaW8fckrNmdaUd4ZfLVcseg1wjsoW6RAQH3riSLvGgnhhcoT4KaCeDjF\\\\\\\/c56QNs3b8cSi0HWRAzRDyaVUuCNlpFqtvYvgfLCLk1jG13hG9\\\\\\\/Bau024mr4VktD6\\\\\\\/6k\\\\\\\/KreodAXkRN9VIutFL+stJA51Beb4prkcdyBdGEk0\\\\\\\/\\\\\\\/koTilfmX2xhcvdZIJX7ZJGQh7yB1Ij4VtgMehd+zTRVbo7NDSOu\\\\\\\/kC396e3MrR3oBumQmmTbmzL++duusfXqvh9dE8r0cY1pniiH8lxNfHsXBU4vRr16CisZkh1XmPjMv+so61\\\\\\\/Ynt0u\\\\\\\/nbxMJh2Cw5Bkz7rIDv6xWqJz7MyB1+A8qwJdJLVUgTebCk7AM5RCLVTUWpE6ZWb3LTq7sUc2xlIZCbTWxx72aGjfOKGxahMNspyYYyrNRRyz5sGHcLFKGGDaqyDkVrRR7zWuyGV1G6lmWp5u6IBO6SfHUZICGvXecRxRLI2xX8tK0IgeJ9E5xNoXLi\\\\\\\/boxfD+oIb+sUUCU+OaudA+1k8SNKuU+sT03tTwn+AZ4VZq\\\\\\\/4Xwws2dAbMp14TlkOD0i6tjyf6l\\\\\\\/eOfInwuJVi8oM9vZfDiovaZl02xJYvytQJAf7qIIY99pLPzvWXyqr6npArAkjNaB3LqPEyI4pO4CJOLqBLUagf2EbNRFbR\\\\\\\/0A\\\\\\\/CIdJMQdJ5f6k+GqRLPN\\\\\\\/Vp6NsqeghfBU3sTSavdhCvA9RGGua00+7WBqsOs3mlvRhfvVeMN4dwJcRPvzFCH0t\\\\\\\/Si2M6odZnZ1rvWzO+fzC\\\\\\\/T8jmF7rj6aJ113RYEXjsIxlioLl26vUpBpljEMUyDND5C9OBzpLVQEsGuXigubDlbLpB5AFf9Teg0sniirdwh2253tF6zqMTvxiFSmSz\\\\\\\/3nrDWUFCfeP5VC0uw7hHgneqvX7haofcY8\\\\\\\/Ap+EkJbJqQt\\\\\\\/bkudZEQ0Jk+Q4xe8YuMCCUB7axF73NRwM5LumjvFl8JEfm4bLDryVoEDFhhoxr8vsfGL5yXtk0mrn63AUiChk43S5wCyWe+nsrlEBQqtymroVzX0R0sAIfvejH3QTJEH9MMXF9MLePX1Hr9IIDjGWa18EJ3rawvDo26GxYss9FkCzORvWBLiV6R7PkgSFsOZTOnkXDEBBWEeOp8293+iVwXl7BeqpTawyS5NuHPf+7OO8Ja6av6s5aBGttwtQSMXfHO6Ty4Q0U4tX15USYFwsSTzfIo1DmIfP06krg9HZXrCJq5v2mftG4mj7uX1y9lpjqwYri+mZGBHlfK4UuasxeIzgo4a0hHAABTNsZ2QD1KrfLT26LTSbV4Rk9kb5kZCqut8I2aBY4hq0eqsCZ4o9OGUTIll9g5f4Y8qsQTo6YspmsNdDZkWoyeEHyBGhdQb5xzupsLQhl1VH+iiQm3ACkgRWcTLsAAZy0+M9Goqh0gI8NYsm4OkyGp3XCam1J1nt+ZQu6V+suLulQ8Ab3AjI7nSfla+OLoA156hjKwMR4D0KPp9eDceZp1qoOaL8emE0o6Ucdtd47Z8xPUAGFS717AVqiKRG45ms2jZzZcn629PFw4cc79+FnKuCE+e+85BvlOHJj7XslTaMFKOVAp+thv2tcLg8JI6ZNZgGPNJO0y36zjP5es3s\\\\\\\/SQe\\\\\\\/ZCGVG6qYd\\\\\\\/ExUUa+QRdnd5OzYyei6sfHmuznaVo\\\\\\\/t50z7sUvatz8oI\\\\\\\/NCdOnWq7hRMkV9uoC+P\\\\\\\/zf+IUSg3Y\\\\\\\/otoxcJbP4bP\\\\\\\/B43cfdCdeZz0QBnMSypLu+ZlM3CsHjH0cp7PN8BJ7XIW0kBoLm6k9WbIc5Pm3+Jv39XbP\\\\\\\/hH\\\\\\\/l0o88IIfvpRKibuuntmNOEG9Poc\\\\\\\/vvlpBQZoBPxldf9Zw87AqrbkGVoYNzSrTdogOZM5I0CGLwpah7dj2FX6kspKO11q6BYlfrnNjDo1DvEddqukMmnSHp3iVDOwZ\\\\\\\/PaZDB7XqOOFd\\\\\\\/GswJKpUGWL+NzJHp9aOhyX8U0sdxTynlfI2WlsbsIhUjcECx3OaWpyeA7PrX7b6bWDgoNLTPazOrZYchaSQi99j6cXTRaJFVMNW4fgYAw8kIOL1iXlNeOGzaJSX9T6TfIPcV+R+ikWaGWlWIMTFzZE\\\\\\\/Lw1+m7wY80zkDp4+Kt16zhHi06iO8YAiX1QlY4L+IIcZI3NEVYqfu\\\\\\\/bHc9F8V\\\\\\\/uYa4JBJPztlChE29E5Aa6akUFnVAHiOFRfNjNz04kWxFUHoYf5Se\\\\\\\/kD0\\\\\\\/WJQoZ4RZolSZibWXaF91KaLzTE1KK3t+TQqKszFt6NPfxOTKdTF+yn\\\\\\\/tY1fgUqd5WloPRrMQhZO+D+tulYCdP2a5V5TjR7CB7PvKGG8tKd5sTTy7yK+bJhV9dxgmFKaHM+UDkj4nDw0XdfkWglnnCtXeINBDsKx9TmTTjT6rpl2IJKYGIFTJe2C9FI\\\\\\\/8zBQ05X+fJyt3JFpVyjb8gYo8WwywiDwZh942a51cXvfZTIffB+8QlWW4dsNZvGp9uUv\\\\\\\/FfNjcJlseDhY8NSRuZQXVtB6n2XeVdJ9jvflxPCEJeYSLoBZNSQavLalOm5PEZyFO\\\\\\\/K8X9UImeXCVIGlT45mAvNvrtjwHQI7lqQcAyaDoxCAktuLJHFCpd\\\\\\\/k0dBrS\\\\\\\/IA3PA5GhQ3JO0lw+Jhm8A9jycxn\\\\\\\/qK\\\\\\\/hTdJUWbItbe5naoOAN+vxrQkPpUtmVFjSH7YzxKmS4zLMG9XitutNZwwvr6i7YyDPA5GehjSIFxn52bRjK633k8lO7xV2HNrxGdBqvwKIqRp1lpBBHknM86Ldj\\\\\\\/W1KszV+8lLMfH5Itl76ggkE+Z+\\\\\\\/yQgAAKNCOEjeB2LQspjgtVuusZMy61tNrm83WWRp3CxZlo6c2TSTVVJqZjJ4H3O2UqN+AaR8hqkdOGpi9Q5vdol7MHpF83LbudnskEpAwqA\\\\\\\/AJnFgDuQT2ONhciWMtv1JO7Rg42nsRBo5CA\\\\\\\/Yfq8CmkKF4SxucCVIzQ2yIfvyZelqFYmmKtCsA+gFkxG9IwzXULQzc4GS12MGqTYmHL10CvN2CJSkSMyHLdDIzSqIAUnjZ+x9haE29TyUbvLh\\\\\\\/E8z2tKsuPKLUqpdSZacl2FYvrHhTY590OpBH3lthCi9SKVQ5JSscNSnvGaJX3kivEJ2dL9rqyywqkSIF+ngFgpYnx63ki6IP07Gh51Dh87PU3M1BDiwpb8EhcLQSX\\\\\\\/8ygPhIw6\\\\\\\/fvEgRAPihm+\\\\\\\/\\\\\\\/VXbj0e7DXWcS3T8kcbL4iN7wnlx8xwJjNz7TyM\\\\\\\/MjXk8SC1jOQNuuDJYwhjQJ80vFXx2AluKZFOopDJnnND8kROKQWlTbE7L9QMmokw\\\\\\\/DCIWjz6w9KL62UohbS75Mp3q1UwAOUWWzSMEI+Mtq5vCTEB7NGc2Ol60C1LspCXhtXt\\\\\\\/iUZlQaXex58nmDgJpI4UTsLQtGlcpykekQvKgvz9zcASeB8Ci8uD1+QdgLEqdWPwNZ1MiaAzz0i6qXxQj8BRUret69DBNJXQ\\\\\\\/JVAK8mS1w1fN1IKI6vnF3mp0\\\\\\\/BMNLNddgctrksnwHxcFS6r9AV93X\\\\\\\/6Cexd9eF1DzxY3Q7XWxL2jB0uh6n6zl9BeK6bBmbbDZ3fLyFOPVFveORNQX2fJm2L0SAvDI9n2r+uJPSyFavZjHEqK9\\\\\\\/8PskxguGD1pVG3vGvCRJWzYNcFAWVkFkoJXbdwx8G+6luMlnEYUhOwbWPGu75ox2fhNuffpqp53SxVlXLpqvo9RHFPe9tjcXX+OpfQZbGrGmX46bcJW0nIPclbI\\\\\\\/Tn760YK4CpNJcMGxf96M7GC\\\\\\\/FBu6nBrezRKWOFhr731I3Jo36aa5mhDrxetfJCzkk6xzhqlbimdR\\\\\\\/0MfE3V+aNyEmxurTFB3hmubh\\\\\\\/WCPCXYl0wCT3heoEC1y\\\\\\\/JZ6zp\\\\\\\/eIYo+sncgUrFMMPvKe8EYaEy4mos8zM9A4ZlkM33xu+4Xh8cki5GCBsVQwipR3NYQgtgH3UWO7Z1o8z2NCZwRpnoL2ukG3TGcu3oY7pxRXITOFBA2zFxXC3gocYmMHs\\\\\\\/3PNbUL21o0pidtoujL08Wh\\\\\\\/7cvvNTA3UEIUUWXwNXaqiJOfRtyu3BwXN+6GnPwIzIczM39ZbAQU5\\\\\\\/vPqI5dLKq6FXyixtJ7iWr5FuVVIh2W\\\\\\\/7QLLCMrOrSytA69QxNRZ2BnYE3ofNRURcQH2ckjv36m9rvg+saJ7FkQHZ3H1ozRTIJeKJBa+RMQ95YCRV672+oHO2hAmzjOKuzOAsZJJ3gfFkhJm3VnYmAsuwogP3mglZ2coGr6PN\\\\\\\/mNi+FXqnzjTFNRL+k5kRSE42Guc8CVNbyBxnLtg7SvqU2iEW\\\\\\\/TPHgqocJPdVz60dY+P+Dd6N8gNkkTFPjtXnMSQgg02Sl3lrN2W9VQLJmjtz7buK0IG3YEJj22wN4\\\\\\\/7azgjLNPiMwNHc1NqGBv77ja0HE5WOAFYb3czCTiZyW9nfUj+foUTeI3xSHdceXThykIBSTclJ54gO1n0iyRVJdS67P7Dx8hiVRfLKu+oYshiY7r8PTPRQvUK6qx4MQLNE2tPA9nBclW581FN7S0RO5LWZq9hlqnFWZCsqAUgtD\\\\\\\/4nsVIqp2gzC4mL0uHOi3wYUw1414jaW\\\\\\\/5Kib4ZSJCdoucJ4zBzQ7QLdpYZ4SoIHUBo7NthoVSlkEH3MFnNQemgrxS9sxgzPQu1sB2T3nlqcwWrboZUjDi41nUSvVD0i3Z5\\\\\\\/xiuAKLTdKk8LY4SaEh7Pe9rW3fq4mbqQZ9jqJg5ge+YCivxjBcbtm7S98W0i0XqgXXoc6B67\\\\\\\/4bINp6I89kwK1xaSDL9qtdcst5gRzEEnxzmNceFTFSZNSsp3iL6regYVmzzYD44CSGgdXACNJcRkj5Kz+vh\\\\\\\/mfebpuWBd61Ks33FWWFFeDjUWhrjF8MKUfzUFeUntm6u7WUMFnMqWt2iih+xXaOTcnWpFX6kPlkrtlbEk1laSXVx\\\\\\\/D3cs+3Urf7ya6ym3ZNd1FrJMJVjaNWoVUky3bvR+Cnmd5hCCOPdpEv5CBW8EHHA94FlVJX6i5zHFNJaTUWuHHjd9k5XbBNRa8NKJJdlf+5rIoyn8sURB+GRaLI40U9z1DgVjTjyzXTupq+OePgtY0lzC6bwgrXLBq\\\\\\\/JR0THnpB0KpRasYXojpP3VMqDVM\\\\\\\/QDO3IJG3PeAFkbALuzTwCv2+j4z29+unMY0DdMtRiMRu2NOU15bex6zSx6\\\\\\\/rwkIP\\\\\\\/GVmGD+nJhGWE\\\\\\\/LPTl27bXxC5UwUM5Z31daj6LHCXISUp\\\\\\\/b71fySHFlBGLUKZ5IaI+ifiB9w+xf2Mf3uGebtJGxs011+JLE\\\\\\\/k4zoenRmhHvTUavc7WjStZ3KITnnlwAsZwUjLM+Dqy39hGqPXMRwX5CIew1N2wuJD5aAnsAyplk0NzeUU2a6XCej11ba4WTuk7bVwf\\\\\\\/zDrGNFQsn67Irffjx\\\\\\\/klbgRRXlzp7xV5NmjVxMofp+WFYrGC+lmi+YEbBpqbupoYzNUYX\\\\\\\/7nKEt\\\\\\\/io3dEwUu6dia793nBh5QmGdE\\\\\\\/m8k8a\\\\\\\/RBrH2cXz8eZ5Gh7g9jjoQITp5EnV6p6o6YCzAwa95x2WccA7X\\\\\\\/774+PTvns+4KyS85qHdHbIKAAyH7Z5YRFQ1s7S1i9uAW+2x93e29NbyJ6vt\\\\\\\/yl883JpXPGEclIKVmlkqIcmLSx67DtN6Wnwg\\\\\\\/EVG199sf35mM36zLlr8qiu8SeHM9MDoOAVJ7zx7FmI\\\\\\\/5qnW\\\\\\\/xemdidZGLJ4ca+Gt7PR6BP4xiSQciouYGEly38xLqI5b1MvzMe1jOjzKxKWLT2g+ctFurSriQDKs\\\\\\\/PYIt8FDD6BKZxu8hCCuFfYmHJdr4AdrAPQ5fv3ubWjFwUW4fUDcn\\\\\\\/v7FUjVXwLLcwPkHnCHPYrWSvZ+pFiklARdNGe41mJtDBwKMQyuRskZgZ8ZGtHGqQUI+x\\\\\\\/yAH5bBCfDlllkp+246Zg5wL2boByOFt+qkJRAB+xk34Qx7t8iJZpe0TobsxQzJxRmrYAIcSHkhkGlyYdEgbipZ6BqMRXYm0yVrbfdPBiLJCAPLsSmAOuZGsddOvjQIeJR3yNr8eRbp1iqHNl3WdWKD73dWPBIdbVBY1CmQBBoCiqLITziZoLtu1aIeWIhTr6ZoVR2\\\\\\\/K2GQYp6\\\\\\\/fut0nTT44gsfmhOIQMaV2iWCn7is\\\\\\\/Kx7iQfQXJlKkHOXY02vR89wYc51wO7BZ2\\\\\\\/OveDYHFGMR7DkrQ53TBDlkicEPgeD5x\\\\\\\/923WJGojmItiNcs1j+1u99pc7cRZNSNgBRra6nSOUYJHdv5jAUyizbdsuP55Sht0w3ehisnEeNLb2RDFV4RMBoOwqPPfVDmA+D94ltNwBqIZcYKYTCHr8Ki2LOMlIa6aLSkYzAtAZq6m7CvzcvKHb5WIKIMZsueXzDZ91YkV9tVlwKrqghZ8+1\\\\\\\/RoCCS\\\\\\\/4\\\\\\\/B1Np0q9Pt46YGbTdqHLvUuoRWJHwDL0Fh3GqCq47Zvt6vF94+vs6A6veRSnIqPHqIn7IqjxMlhzTmj0RpQYnGbJR43P+VY0j6ETSSNSg+HtC9F3O+R3n032TvdUSbuFMyOm7D14wTCR\\\\\\\/0j0jZqDoJ2t4W1DkmyACN\\\\\\\/zrWKXs2JzyTt49hRl7pH7C7t9AKnCUYjSpDd7p5OVFLcIUy156oeAT+poaKtsV54SALDoxakgBuVCl0RDvPOXp4A8JEr3dYo3HdHXO10nBvrTPYV1bROsH7HL9FL14kPMTozA+YlmOi80RUY63sWMBfnJMG4ALC7d4Ym2BokmC1ZFB\\\\\\\/7FBx9IeacUTupLZUiNZyzRm+aYlojd+cE6Yr5hGvpTSSBxrVmPisxy5AvL2Npt9JSsgjSUizkOMmYWvOCpOuOgQdeA6JEB52M7Rahm\\\\\\\/ciQBXnXUYtwoYyJ8oz7KhIeUEi4VoqBYTO0rS0UJliefODAQhjntJgjri\\\\\\\/hrjZb1zQTLPeC1548sj2McLk9vV\\\\\\\/ZuADWuSAuB3N8\\\\\\\/BqXj55HspHELbY61l1wlPOGaNmkCumt5ZVawdKf5aQdjaUGar1W2608DH20LkFrALoqkbNcO1B\\\\\\\/IWCccDJunw44o7XWut4sn\\\\\\\/gH7SOylYijGbf2G+CbayGz8mGCrX3b9Ty5zwWYLqY61xprMQrvNlG5Xu1ou4jUCyxk27w1zlpwpPeG8dYuFHsNzS8LDABa7a2TaVReQhJzCxZW+jqUp6QvbQ0AJ2Ti\\\\\\\/6mGcZQzxVVCdYbFAix8vJ+QfCEpYpgdEuUd0E3tgzGhIwqUaFMDJDOIc3+99o1o8D8tH1NvZMSXCfey6YMLIo5ihWBI\\\\\\\/JOYfJW1rw2IY8FlGpzA7VbWXffDMtlvlzMe8o90przggZhM06\\\\\\\/zNNwFuQEaM35huumXyjkmilwFbWn+aQBEDgKea32XNracMDAMlxBAn1zu9WVdiUq5xvx73b\\\\\\\/xve9zvpYXCYnLY280wZo\\\\\\\/\\\\\\\/x2cMwYC2Za0iSG\\\\\\\/TZqjJuR1iZGOkAzp+GoQifSRIW3S11Lt08bzji4NvW\\\\\\\/\\\\\\\/cUWaxAMSLRhwsyaTuCHDfTmeoNwGjgXbfzdfnCe7T5SFYyTewn3SsywxvuYKVxxew8WUQIVTRS4h8SLtVBo76oR9PMIk\\\\\\\/pdEEDsCQmzTnOWlDh4M2aMKFOBh\\\\\\\/jl4yb6NqDdBynXlxr\\\\\\\/bRHdWyPMHYMZ0An9bd6ZPrZ649rMG792yTNqNACRAxupyQ9m2jxDZBtxLACa9Q\\\\\\\/EogoycXGNAyBLj5Tf9swurVJ3mhz1d87Ic0Z4WdkQTzeaOqmH3A87049w8AKVYsLYDDbpuB8NbQjdJU4QCtV7o3qmVwymHE437UaU9F5R8O9JqHUDjmKwCqEZ6b8oR30hY7wGH7jK7hWZloLZavUH6pmVXQupOiwqVbBWk5Gs+iY80bvGFynmSZjhe8wIghRpFKaKs56oqXqFWmR1D7gx8CRI5dJ3W9576SzMwH0idPM6ngAK4BdHhItnlVnhPuABXDl4miM65nkFHtgr5v+X24+rSo4t9VniVuCKX5m3IopM4EuS7fRo+tfeemOvQeTd0hDJ6JuESQKvWZYadszogb5kK5feHdgV5YrA1B3NiPV23bnUeILMeWJQP19b+VX6BtlEonBJBkR4jHZmvHhVGu82E97CBP09v0R6wydIiaDZER\\\\\\\/0M1DoPzbJRBvIHicjBGEnlQuxnRs0KdqFJLozr9MdqShY1twaPmrkc2joxWG7a4wAPAJoimUJEYiaZGSWZkzxqhPuiXheOAezFgBfcrsM6ojrj5s\\\\\\\/XcbtAx7XR80ltjL37lzZJyzQziIOLvlL6G5Y0HMrGTNm81+qHYOKPbkkHsk\\\\\\\/ywcaDbZ1fb9p+WudNo8GuRFi7WLwH6WmNI6cIIY2wsEIdpcG9D62KVDywLuCqTN1OR+IJilUCKoYl9Eq4OyQmeRnPQep42jjI9WCC2g2edSq4guCEEIT5SoJzYYBv2ItzIkfMRHikBCaUbrRmpqRQ7FPqi6k1EK27OnUju9FSgPbnisO1IS\\\\\\\/OLHOJM4V6fTmD4b29Wp2qnzbTfJ9zDeCUhK13WxlAIG05Jms0BqEPd8gHzo7tDoSbVcsXXKCyWT+MLm7FcCqVlyAP1QgbTybQLco9EU3pqAwC9eXxBh8PnqUfcsMDwHTrCx1WnM2Vs\\\\\\\/czbXTsJGP7r6ClkJbwvNr9l7Qkb\\\\\\\/Y\\\\\\\/pqGccrfC+pmD+bcsn7siDeHzc9d00GTLLJjehR5EDk+3Y2QeomiOhTr11x+RrikKObX7uesMpYsZmfrWRjHmRr1I\\\\\\\/EmC4QwNxOh8x4OmmrN+rnik0LQaPkuHAmgDu7\\\\\\\/OOfeHt\\\\\\\/nO4ykmI3846q+zKnJlZz0wHofvJPxPCX0Mol7QDU12LUl7aYm1ouS6aeL\\\\\\\/v7\\\\\\\/6DOMZ5GduhtV2MBhAn2HTH3UVrPJGODPOlsRtiqz2hJZKLavJ0FEzyyuJSn5QiMVTuYkqN\\\\\\\/kR6wlMyEAlABwTBfoG2aRABOtfAYc7IyTI0Tsr6+PO2T5HAT68PfZM2seTODzXPOlaW1DgT7gj8i62XW9M1sj7gvW4jf+rnZs8+Yy+RerBoY\\\\\\\/mcw10rqUMORRqs\\\\\\\/+OXucYe3M8tqwtlvPIkXNPvUqVf\\\\\\\/WeaInoo+43UWnJvG7pW5EBqkSe+kCFVXgDdL51z8\\\\\\\/2OCrgZ6YihsOroeBGK8ZEAWwI5shAxwK7z7FFflpkouR5SWL2IqxdRBnYnRVTJguqnw8NDHGA\\\\\\\/hP211AfOb4bdpGrDkiUqIlqoh4\\\\\\\/b9DKT5zCsbjCWggqU5esCqxKPHd\\\\\\\/xLs3qiE8lOqxO4JmUcQJK\\\\\\\/0Odra\\\\\\\/K0AiqhPnvsf60h3wkLGW+SvZUFE8jIn76+OllFSMmfQdWjHWSEwZfeiv9q5Za5L5wd+VDZsifEH9Ulf\\\\\\\/D7TNaSyCWSRLZHcwM3ygci3d3yBmW9TaUfEiUjiP5tYgUAAp2CB+ZqBL4TFdKVU\\\\\\\/ybo7FHtkRvdjFtNWjj+bjOs\\\\\\\/hO2s8iXFSCpnmJSO7f2b+18qBifg+1y0e5pmki16pXMHgaCs51sUD91bQ16OQMmF7yBe7dWAYmDp6OvRYnBzan3tt\\\\\\\/OvQqpmYC2UedDlGowgu0jUJY\\\\\\\/SzN2wjGJFdjrtqivxCLQQajI02YTkl6pFAP+cCpPf35E8eLyBitLBat+f7r1\\\\\\\/ethsLR5jCEARRBNBDLexAG0PqIWYCpRy84HYY8+9ePHsMmEAmY06KsmXJlXTeK6yH0eXZLBJrGicgvab+Q28anTiKRCFo24ok5ybLcVVlSfvQrCAVeYg3s9Kp0+IelbNvz8rivyA+shaCZtIcQfsqBxvyk5lvE5Ig+0pwn9myYvC6bxbXzJOKbri+bh\\\\\\\/1SgTOdIE\\\\\\\/yme930MocUA80BQtvgZiwDy4VrK0LifrmCHEkB659cywZdBm3KMzx0MfUm1atc\\\\\\\/4Ryr5+vXwdQOehq28LVX3\\\\\\\/yjEmmVRH2hMvf7H8KlwPHS\\\\\\\/Pme3H5IQdWdcOIF8HQXx8TCfNDIiEBbtTyaCB6qE8YdnS73HR3mAQ\\\\\\\/dexW2Dzxnp8sRX0VB3pnmO2tO7IAs+PyfvycZ7B\\\\\\\/5MO59yMdXH0F8dz4VykncisoFw26z3RwA8oYXZFZYlYi7puYLJUE\\\\\\\/ck5WGvlfvgjtMRybfhWRA7CfGtEF9H38Y8HzhTtFhM7eIdCwK1sxJfQ\\\\\\\/FulbInMHERAFso3Dd0yc2iNf+pn7G9WVhNYmaONsgeIan+KwjgJGvq+UgGIOuy79DeeMxar1Dkmpi9xhQJCBhZ8uB3aIjAy9C8C87RvQ0ea5dSMaN0qxiA1jo5QkF60Qft4eaHye137SmdY0h3E1japRwWbssmxKfYJc14Yc8Fzpo2\\\\\\\/gJVcBQVhS+\\\\\\\/hPvZcuVhmJzfL5nkjPk3eoKEU7U6A7T5E5wbwA5Z6JvOREJw1DVGfFS7n3qw1nD3TGUaH0TBgOnC4kvTH4XHJHPwRPJpGT2Yg0A1ZjMifuY3bskRM69gyCl8tvu\\\\\\\/Llb2eQ31Gyfy6GiaT6tNSKvqev6lxbi9tUv7ZHou4ZBAm5UrojYDNbsEIjiSJL3iWy8fvBapjnsQ4mjFi8fZ37Rx+vSCojGLVbGDfByxLOiuDvc2L3QHen5TLRZGq\\\\\\\/QpKYKvh3D8x6g7vZyB0VfltvoWI4YykN\\\\\\\/hZN+ktV6o3syuan7hUQerqlW6bQRBCeWeW3pBoxZg2f6Wuk2yrZqFpXRy\\\\\\\/LKh7z+dJBOyqHDZHbeAWtvnjdFDYfsr0sYz+uA6VXT1nKX6CrhjMNKn9a9PJ+ABn6b14aEYUdrGL0AUP2SfQfpTbkYYE9yaMLBgaanr7clY7VfMLpzKN\\\\\\\/By6TyD0RjVOkTYkMf2naz1ji7JzzVSN9w2f9MPYQg1pk61Ahqe8+fzKtdaDBsrxJnefu3S2gPM2sqg9zS6b6F\\\\\\\/oLp2ewMJHjIMAAHyJIxFhPrUjYkCmAkOR1nk+KKhnzjoxi6tZsrwCQxzS9xkc6YumiIegvrldGWXKHilxjN6UNmxSJ34N4dnYKatd7BkKQHvsioYebc3HeJbu3yBVWhIBkUVtvepzFzyfz0ltuVzb5STCwGATPE46LE\\\\\\\/5JbmlHZrAgV7M8H6WZiMoebkqVo5XiuM\\\\\\\/9Cd3fLeXu8qpkL1Q0RUVx1hw9i0N+YCYG8Vd72Zs3vR5pJDyqYop6fPind1BzGIxXSHy\\\\\\\/YvpHvt6+ElL6VdIeEjdx4OtTCosKpzAm4YVZcatXi5kDQGHCfMj93ijg9PbuEawq82AKi6cyGOpjkHgQcha8UDnHZ2kprkuaN8XPc2rQ0S7U8X20HoXh4QFGMSToIK6bo6TpaBdu8PTYZt4UNjaz3msJXy9ICMFH3p0EtuztTh6cV2lCnO5pc2TFoyepsVrzgq6707XrkoO6wGTcKEX3\\\\\\\/L2JilOrCWo9BKE7s0Ar\\\\\\\/VrAzxXQu39GGMazGctq+7i7K2btSFe2qWmp9o8eS52\\\\\\\/53n1rQHOT3klOwCvljYkrvhMzO9miDHDYufHV28hhHhR6eKrdSTkUN2t19FR12fLmcqzqOX5T5TpEgF3\\\\\\\/GNL9hYmbaLW6xxiNjsL7ms+x2RZCw1ZxiGbDDNigHhZl8GLjg6DpDjQ6XjZrKLP+ANAHBiVHf7vG+p7FYGKyZfDlwvqSHvKZ5Z1G7eiJJubwJUurGSHrradf8n7I5v7alUxgl0Q3q+vL3fo8ED6VVQpBgdrKlsVER8n81CwpBkrk\\\\\\\/AW0BVCC+qi2aOadKDUx007TcN8\\\\\\\/M+HNVmUZw\\\\\\\/FPBYYDJKkoM7W\\\\\\\/45y13CyrKNaux1Qf+pONEuCr6mQcCcbOUesGbvTjqnjor5GSrXl3JcYmDqj+kSC+EWggMOIJ\\\\\\\/P0qOQGD\\\\\\\/rhBhUvk1V+Ob\\\\\\\/VIW5xrDsvb9ctVYDtc6EqTLB\\\\\\\/zAwEtvNXy2YxEjnVTk\\\\\\\/7\\\\\\\/\\\\\\\/pRjk6h2k3t44gFIvMZNlTNjscR+ehm2LA\\\\\\\/KqBeTtTpp0YLWt9awFq+TMgX8aeWPqKMKyw6QdKt3Qin6BEGIpgfQXEMjdAxOtqVAa1hRHxnmkMo8jqAwic7nTEO54ky1j27wb0w53c0FbTthTh0SGskw2zCpHdbMqniKedJKK70tWSgVoDLSC5c6\\\\\\\/QygyOQoKTNNh3WToztzqRp9\\\\\\\/EZlLtVswWJlapMR27RjxBenbj4q6nNO+nHrWUNrAXHRPFevcxfE9QmP5Ek8Ubrcr\\\\\\\/b4FHXNMNweN8g1QT5PFcmAMJWbQkr3hjyJ92suRsPAZS1ZFhf5gYCN0YNqS5L8yu6dpS4ih\\\\\\\/Hekm9xWWPzLTpuzxGsokLP5R4cmbSmL68JSMoNLHFWKyXX2sN0kvlJqonfhMHkNlDOwnU5f\\\\\\\/hmwtjA7iBeJYTir\\\\\\\/bjht162QDlAPWouPWUyzUubR6ZsUXH13h8EMLIgc3C11r\\\\\\\/nUiGQWWpuV3SCU6ZoeOzMClhdPN5f7GSJvoMxUkuFECjguWzZFood3AgeKTeOpjMfhbAihG97dDaJwZeAwU+rHmq9GkBbxavsWgK1nhMOuHjhOqkFhy8PD6LKNfFGIM4Vfzd4qiMvuvtDEQ5\\\\\\\/L46ECQ0Y2BB+vqW4QMgtuRiiYGPoxf8VHT2VmhpojCBMOxfJzGloUav1NQn56CmcKJFgukBiS1TVwiYy8yHYc+q+qeRSUtoNkJYgjexbNN7FcpYX\\\\\\\/47CeFDuCTkLe6XYCsl4pBX19HKL+ih2Gcb4PjEfUMHZBYbts0zQ0sJGnvfAe+9fUqMZl8YG9nXEBPAodPbJdmaVdch5rzgklvlZbDRSh0eSh8mewrMQgIp3stwnc8mf6JneG\\\\\\\/qnI2sOleniT2YaWeILQ\\\\\\\/g7Nl4kMwxAT2vgvuiC8VMijM0hV+FI+g5CZFW6PVjPfljUo+bAwesr2Ayyk\\\\\\\/zV\\\\\\\/YG94saMdMda3qGKt4LyRZkCSbjS6meBWKOCaQgy5w3aXfcit3H6NqhLhiuqz1ZvQa3jifI5wOemp4kTexrmAES\\\\\\\/DK4JsENaN2rHGj\\\\\\\/kfbjSXmU0lIznaBGhPS2u1mEKOqlvRXxY2KxMS5KaL5V2l2PPCaMX18UDV7gmWzg7lemBXmIdcHoIE\\\\\\\/rhW0+O0xuP8tYc94BvRunivI7BNP7k15cSQNKOT4LuuNpKAb31mI431EVF6gkjecuvnRg\\\\\\\/HOeZkgYiF+Jf6YD\\\\\\\/RFwPNuUSBr4N2osVX4S\\\\\\\/Z3pY6HsdovyjOVMBw\\\\\\\/J717lOsyGPIcL5vBYseGx2NiWa8b5p2JEw4i6M9T9bM0kjBnw1brG4b0+Q9Xmi15\\\\\\\/e1E9wKdRqDQsogwh5ksOx7EaSnubVwzU9YEIMnqEWlN3BLnAf+vIqvIv\\\\\\\/5qQ4IAPJDzV9lN5g+4PnHDHZc\\\\\\\/bM91qD0tFVTVdD5dcPFRlsRWBBg0UVJQxRNovSCa8NHuGQasfFIJ1BuM5BKW+hu6bgdVxa95YAnoozWwHcgQrqHIW1X\\\\\\\/xxI2I4CTvYxSyQrhZ2z4lZxeyL+5o0Zgg6EO\\\\\\\/2\\\\\\\/heaCzpSiaYP56YSqrZlQYD6gMJpLzNfWx9JrxQmLqnMQCFSqjJerGeXw\\\\\\\/XC04h+p2vdIH9BAzMUesFIv4u78ym0S9B8+z+iNexcy27S9RauOL+FocvU9G6QrP97Docm6Lv8o1e03yYbWS1SK\\\\\\\/EyeH0N7pSX2apiZezq9zLmaJX73uA3XhAymMKetIem1B4c8hBqzkwRua21NVeCNfm+TLY8873InAlhNm6Aq0AK9YhVR7bZ8DFSH6XumNZ6qIAUgOCiBA2IUFGBKO8wbDTNumKXg8dVsaQFOK0NJMNcQH30546PkOnVPAa0c0N+3rbXN0o1t2a5V4ueeIlt40DoJP153d40dFZCpm71nsUoM\\\\\\\/XudZiaCdRuSI2xQbZ6gIpOjqpCo8lJNkP+IPLP3HqZoXVBjAwi7D\\\\\\\/djXvKdGDZ9rUnwvgEp0AUG6K+OCAyYH9B2qhvN7F1mR870xtrVfEdlpycmywkq8zni\\\\\\\/aVjMia4bzk8WvHmccSbHjrM8kPCBZNMd4m9fBDlR3HMiK1DbMxiBOJ8CNcMw7wyJ3uOcIAoa2DtKhW9DId4DCDU\\\\\\\/SSSkmRImyrv2fY\\\\\\\/MEpvZn3CUsxo6VgKm9MVz8AVQNPd14R9qo4It8p6Q6BjADoI1Leo2pR1fzQ7MQ7qygNSeDKruwDjk+0W2Hz\\\\\\\/tdN21d++4Y9ixf\\\\\\\/CmAukxrdZU4IIoWYUwk6E\\\\\\\/J93til5aHwQ\\\\\\\/n2+6eEus8pacL3SFdAUv3Kj5ln0pN3I2h2WxLQp70QGdg\\\\\\\/tyUiFUNr7E3XayI2pk+KO\\\\\\\/XaAEvvm2ZjZU3yosnOFMDvAtFo7RfN98M45KlFdH0XIjKwfxy0tBZBrZvjJKqZrLmGfEaQIPxYOz79dw6XwkhPucpZj5fFUqZrAvWYCZyJQcwmD8qBizjJCLztnwvVHvIZjiRSTNYz\\\\\\\/rKmdS0\\\\\\\/sYqCthMxnbSjhIsDPOmuOGxj6xTnw95pbJmOpcM9Jt\\\\\\\/6Bs7mguJPEHzjD7E2glsIKDDKa4B7+a967apmlcyrOtrfcX84JviRDK0LHTxI8OEQN3DM\\\\\\\/RCp+4+nMd71wrYTQY9j5dMrAcuoJe2zkJY+8JLnpAAVtvtjHftLU2PGW8pDT2F8VJmxlCeegsrBu+OkCmDLxV9Xb\\\\\\\/vc9wBsRsyZRdmjaCminY8jTKtRGQLiByj0mU2DPMWgBWCh9JyhH97F+w0f+CKbMo8AbXye83niphfiPjwDS1v+XiAkuJmxHqUmbxn+CAPfg9rdrnViCmiIQGZYYT+rzEJJtev86MrUqIazhi5MKly3fi7DJqkABwvQuHt2KYMTBMEMfc4gUWwtyQmuNRpcM8nbdtaFjU1lLlpnBld4jjcsVBbTs3Dm0Q2pceAs5Px0Q+blI10le\\\\\\\/IHbPRYwh6CQa8YIXzNv+hvTpKstadpV8Q6tD85zCbvVQjmRGdfb+911i\\\\\\\/\\\\\\\/t85nbw6FT3248GtPu3CDFSjbtSKbkj3iNwKLZLQ4z6iGCDak3Lgrv2eDEGZhh4kq91FajiqiLo2Fj1UMd5eOzkQwhro\\\\\\\/Y72eyW4zkSXxdUVIWwH57eEhMLSKzViSeMdC5AD6nGqrtmPdPfPS8tPT9Bj0Q1YGUrdykmekOhUtXRKS1OdTrC8HVMzTiwwttfgZkJYEx13WjuwlcjjgmhAPaLSgPCpIZ3RZAO15pyp2aoiKwueWctBwmTVvsTsJ+N0QvlIHBP+hLM6lY+s7P2JJcZo3r1f4sZsxdUfqly6kfhvfGXFoxb58WebR3hvdbMpL9xwjeqngDPfLQMFWo7rFbWjsRNr0S1sRBf4d+heMZ\\\\\\\/yFHlDrZYZk+FTRzLFEI6KH\\\\\\\/NbHKtR0jqGnC2aNRtA589O6jWfeJk2\\\\\\\/vjhoR7qDx0g33G1fYKMA3gzO+ASTXEsxz8myeOYCnNIIipDBA1swF7V8uxgAdw7tBJSoHn1QenZB04BQZhx85buDG6np9ZfsvtqjPO9Bh7dOGIzLFYIz58cicof9TVjIFC852AnYKn25YIp97d+T5LaCOjPT25gzdi8OVeezOsQgFFJfII+eMegpMKS2ORwIbToxOiZ0QvzDRjCffD9IteXfB8gn+PwEz\\\\\\\/uE2ToX7sEvwBTRVJeUeQdg7PuPCF3k3YEvZAUPSq6R53FDGkAb91XbQTjw5BvUiR1hk6usmfOYUMasAfvzvlD0CDf5qdvcNZFCT3VDKjbk4lj23bpIx8bNJltPg\\\\\\\/cBi9V3vsOmtq9F9drtg7xmnB99jEnQfMm55WPj7PO8a4as\\\\\\\/pOO6Fd08M1luZjyWSVztqzbejeBFM6vRJgeLHDAGEZVw67bTlwSt\\\\\\\/AOtDTk8lJ9qmNzPKEFoU6\\\\\\\/53kaHev1jX62L5nQaFBQB\\\\\\\/4QpkSHgErLX\\\\\\\/FNl7ip9PNfI\\\\\\\/3Z0\\\\\\\/WBU8vqUaU4OImlyItVFWxq+QJd5iheXGjdSJUszVDk\\\\\\\/iHLYal+vk+nS7VkPt1\\\\\\\/7ticQo14+NznRaM6V5215hNUxGml+M7A6k4sI1uLa+r17oxCejaaxF0l4pgaaJ1\\\\\\\/n7WfUojb+9\\\\\\\/JImOoFO3CVLMmGSo6gcr8qj\\\\\\\/fZpPOliGdiwcTFzb+SDi2KMO3rmPLAvPeGDH2CE2rvXeAdqrWiuMydsG87ltFxtncT1WBLv\\\\\\\/VpM1Z8G9uFq3BvTkrtvGj2YVx8prCxo6KX2RbvwxBqAwr0MD58JznOe2vmu8ZXVmm7LSxzjv0nWaNpJBZKT1x5pgKlH\\\\\\\/Am8\\\\\\\/8Yn1VHJ2Mjnljk6E0av77Os9atVLJl5qct4gIvAirgWylQOFJ\\\\\\\/uOe426zIapTh2i4vlGxbB2hXRFlcf7jjPKorMnuHIBh3Nzl9fRis0ZJpAYglhlBMgHUfqidE8JCuwtmdbShZRnn3nasimgK10LJdZqtvdYJtj6XzcePfXiSLQzMSb3Jr+pA6JVSOe29GYYZuZnx0hfOk6TLUwd60oITNgcYfmD+7Bn77KWwFVjWi5O8xH24Ne7rJTqHmdi8z7kZ5GDVnY9OOa+8nAClA0IjwI2Ad9q8UwJYpXJHd1dsufZ\\\\\\\/1h0mKl34AuT7zDvyo\\\\\\\/9C3CFqd13ZgzpgvGtrsFM7zd4S72h4xMC3ywdY8IuGRIhySrZheDzHzhFr7nLduWUiUdmdT0I5OZrtn5Tqgx1MehvOfFTeJlMxcYoG1arDo1I7+oGH9BxJPoLTnflgRt2l3HVjXx9iM3tESH6n0IcsVX1JuGuUSK7fHADiCRRBz8ybto8QhkiZPkBooRO0ZepSsbEulvOf66d\\\\\\\/0Hn\\\\\\\/7fOXwU2gyk34814Fvh8CKjYvaPgv+N3OIpjyinDsj1MESbpPRqjoV+0h6HmDKWaOU25zWF2E5Z0rGIufPitycO8eAdMEqJOFgEiSO2G+7\\\\\\\/5fdjedYaFzBtfD6WgiAPozMM4usPv9ZOS6Vx5X1OmgK8PoC3EzEBgzHcT4aBFcXUIUnoVLMW\\\\\\\/FXiP6vCWit\\\\\\\/Nyx8wqDeFVLcWCC\\\\\\\/8FhDDlIxFfHJKfRu\\\\\\\/x5GGxRktO9a5R7AVYvZnk3m10E7MRZS58iQnX\\\\\\\/Pw4weGVO\\\\\\\/YhxtB0sK5tTOd5FU3EO+AhdJsy1F1OsBXuH8cSbkn6UGLOtyWC5AuAKflJR1+RnPlSGgjtFT\\\\\\\/uXRyS2Vbnd6N+sOzIJ2XiteJSBhjxGP732rsKSRR4FHlxpQmhZFDfIKssLiUQ\\\\\\\/eIt0S5CKRp3C35M7fNVRZpmPTrSP2xsLCQxqkOi1TYmTz8nPEYxODpXh6KbxNNE7x0\\\\\\\/pE5g9CQcbA+JQY5mmvBdS7tuIHRbVGvfLUDvvrEe6pm4P4l\\\\\\\/IGUl5hh8IiqzjnoQSDgTMB4t00tm71mEN3qstmlyPrqg64aAUy4KBEOF1xAe1HIG4BNf1NxtH0WIHKFPZ2rTOY4xLEDrUCzGA2XjjafPl8mh7N0CK1GbwzXxdKEmb1\\\\\\\/szjt2Zy0pIX6nXSwi2L8HKYJG2RS0aN5npCYPAAWe2jg3elrsiRVwAvzcVY+igmpn80gCo3w12\\\\\\\/0OnpYTLOQ4lCo5s87Fgiahj63yHncC7uQ0J9DhjDyG3rPAVGMb8sbzxBkqMPSJMZ+mcLP6jnLJVYFE+mQkUprTcsHEMvFaa8wU523tFEstWJxvLkEfrvR4s4L3OemjMcZWyBpO9FzQffn6wWctz7fPtlpwS+XDOnL\\\\\\\/hPw\\\\\\\/aORI+jdKe7VbyHoLz2cgl8+26zW8ExQAPuaXcGqPHbVwfItRfts7RhcWspZXJVW7bvaEopv7HPBykknewLDrTAifXns1tZ77XE+3Rty1rtyXwLgHUUjCDPikcOEHh9TH++PgUsb75WQoPrOPS8u0s1+1gdPLEVzyiOCfmjytOifSrWZyCZ4EBoURRVj4mmOZNl0hmd4zh5PZkq2GK2221SX1snbu5fR4dBjsMrpKYnJQ0HJYTaHFqX92bdNsDwAvF1t\\\\\\\/3XdB79ezebD+ng8B1Tl0vlVw4GsdM54wec7WuSS\\\\\\\/jnj7qSmAYdEBo8GifqWjIyydE8QsMzObbZRSMojcBU4PFI+3g0ByZNxzrzx6540WeobJujMCocif4duzVo2qcWhjJlhk8roSdypTpg4UFAgz3iRvHCWBUP13iEu8C\\\\\\\/ILkDdRwVfqroQzTWj1aZuoKfCFmSGvMdZenO0eXajFvbR7F8SRLXl0XsEuTgNM8v6qGyevFmLiPph5v51ty0Mdx08i9\\\\\\\/N1TgnGKz73vV3N306rXpAiL6YAJsTQbJqWlL6gvIBC1\\\\\\\/zRNd3hAQtsCby+HHoMv3PFytAyj5SSSGihJiDNPSRbSCKbVjg0yDEVXZl3Wy1Xl9K7uiZxP1Q9gBHHiG1cxA5tqo9jp9koG8XTqacV3Umzy5cJAaIHfUSoem6MqdbaWH1tcSqrZ8j35E6glZo2w0SC\\\\\\\/eO2IOn2t09amLFeYAFV1gnFkPpykpdEOS3F3XfIvVBZ1\\\\\\\/sZjX3Uyt\\\\\\\/niSjsCjfW3cwnPAzeGdrI94kESHM\\\\\\\/ZHFkKDflTumKRgC34mqQsgTKvsC9rvGUz9IqyYCGjzVqgBuQrQaY+GrIor3nD+dX3l68GRrWDttm+nU5sZUxqCKIMpcKgbq5DTYrCbb7D4JpOym98BenvV0iqoTRJNvuFEIzhICr05sYbU+yqFQtd79h29+DhlnUGA+dwDH53VefepFbfu7Lr1tkp4xLkQMx00rzsWpyiSqQIGQYZIkl7dtCXXhgAGqV9Xx9j0m8sUMusA67PZ4m4jOjlUTBf9J1HYQmWoRz69xb2nsqi2fwn5zxvvW7sHUUWn57fDu9vu7U0RCQ0InDydJCxEX+clUREGqcjyswtinrpkuqVXqf\\\\\\\/T3G3p3iUNy0BG6+oMrzJVrZO37nCUUQ7Xq5AaA3RVRpkehyJ79fuG8Thra08y8+cnxuyZ4fWyQqU3L9iPY22y\\\\\\\/juyemtLxKeV7GuR28ANxS1bZrCAQpBCTb0Y7xnjg8KexddLu49c8i3DNxhOYCm4ipr\\\\\\\/dgfQ\\\\\\\/Zy0wsyosMTOblNmsIdM\\\\\\\/awgz9nAZl+L+ZkyTp+QVwSux5RIWV97osdhmGfg6G2fsln6SkmRzKhZ5JrBctpG7SSVnYVM\\\\\\\/1k\\\\\\\/n7QICspeGBj8PUfp2pwebMIhvLdVhQ4R3Ja5HvaKhZhwU4OuIp9BmaGWHzBRipHib7ajMuvTFCCB8Xx4YjZORS2vQqO4m8GFfe9dzEObbHdqGR9He1Mvxb2usYjTzSqXJrM7j6\\\\\\\/UW0DULbAZya6ce97r24+gzUphyXVrleDop8nMzYGVoVaP23RhjGzwXFiD\\\\\\\/lXDxRowA9Dans3U\\\\\\\/KmI05loyba\\\\\\\/csMII5SrYHtWUl4C0AGE87dAO\\\\\\\/x2f8owJcfY7oz7Of68Lc0JloXPVKMP929C3pL1tJE786t9WdjPoN9VXaEAm\\\\\\\/otifYuL2fgBJbxxrs7k06zmS4l3k4cCItzzMkskv0avlhWVxm4d+oEKOrGC6YGIcBZsfMWF9jmznISnfXyPanKVmp0n7avSZWkuPTWTCqAK67G4szBM4oSMttdiIRM9PlwE18N65hoV0mWhw0MhurXR0JRVNEpFRqT09ufxi6bCOlVyxHEBKpwTsu+TfioSeIjZELEPb+viqKioSqT747U3aF2wuQClHtZr7JIBWJ\\\\\\\/Qw8mdpbjnWf\\\\\\\/sjGqFKwEuJhWG77z9MQJOtHCl7eq9iLPIk2+tGDMT7D6MTzecq0fhcBAyx0RSVBO4Uq7c2rQGFSN4k+7USdltlw9f3pMUvoKiZHJa1qMRcwKBVBSoxk5jT7mwaKOy6Zv1uODRBuYulp8eJxNLDapA+KAvAU2mfC\\\\\\\/I1JeztUWxWsTcPp3bF1kCJY36Pb9KmxSWJS4a2FRyOu1iZ0s97H+lF7othlxhkFGhK2nRVcbDhPgUe\\\\\\\/RlAtX8voRs7VKK89494xtk90HAjTQhpmvWhGvkTG+ZUqI8qdluUVUhciKHpBnGMuFC\\\\\\\/rPIey9cJFMqSVKsey7e6rEoFzWrtj8M7c0PbBtf8mRjb824QKtqOvC4a0diiFDKlzJ5fVnrOdxJvGiaSk0eh4tyvu5t2ttRpqTuO2PiCcUVupmb4JQxh9v9K\\\\\\\/lPAJD4XGdRwCrdcjuvEToStRLZlcawomXhrTZcz56CFSB+VU45rMHec25C95MnmI6dNMdUiHwOZ3gRtCEpF+Vbl2buNTtoO7BCfn0XMOew2ZiHgzwXjpwsRoznPYXg9HrH\\\u0022,\\\u0022iv\\\u0022:\\\u0022zsUyQBFSSl6\\\\\\\/EnNvGDQvug==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227fb77f9c78f59ce80832dfbf3968f37a5a951d2d892484556340f34a1fc752f0\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xFlSTzQM11n5WxvPW\\\\\\\/YZ1anz9AheE3JS6vYox6j1LaTDk6P9fL\\\\\\\/mcQycqoFSRzLTfGYkSqApP7biO5V34+mcZqVXxKlq\\\\\\\/iA02TX1QC7WgP6t4\\\\\\\/FWrKe5Gohbo1UDO3Kzg2o4FcRDsNZQXhPNA7gVlB3eCBCZyMhQ30jMzSBfLTt2zxujImpLobrUo4Arg2fIhl\\\\\\\/AlfTg1\\\\\\\/gXbm3pMtpr0dmYckncxnX5eZh31GnYM8OIHehz2ckS+6ptR8BqZNZS4Q5VVCzTgpvG3Zd9CYUaxI20ZpREsMvBTCTFjsj8zYFdEe+OovFziaMekUWsZC6UjLOcmCjX7kuVAqzEd92WKbgtP3ItvDt3Jxe\\\\\\\/\\\\\\\/1TDoI51mylYw2KOwQws+msa\\\\\\\/oqGUPCcln+0G8vmR98QwB\\\\\\\/k1xscyj8RlJYIZX4RY6V7p1Ozi9muaIYCSuboEDZyRZzF5LGLAiLQvxNNfQ\\\\\\\/+zUpyhVX6uWfVKxtrzv1UOvTNhNyw139KSBFkbomYO4zOrQQDwR94Jq2q\\\\\\\/O041t4W9NqCi1tOc5n5BDHl9TdYlQIYQPFQnmGN28Uap75YePw0NmtTyFfz1xs3mNkJZ\\\\\\\/uBoQviBlaHYIQr\\\\\\\/m2DOedLrGXUyiObByON4kb2Dk6T5U8sTsOgDc3wNLYqBn1PylmhXeoUq5+VsRPJGfgUMCRRffpjr\\\\\\\/QbB9XzqxMcnrABLl+LXf0W77VosVgeZzKJYiyAznbGNkzIhFcQ9LMpev4vDHADwKCQL1APTIOkDub52FVB42CS0RyG2z1N3NaTMwOd5mKgFUomyZtI1XolljJqXxSAaSoDShP6SjRi5ncFOMCRaYe3KVvVucx3QsZ9rALWPQsQyfkBah9ayw7fIA1vYkZ0Gk7Q8pehU5h2aroRLKor1VFQ9xmyqpbVbhjOQa0iB4qz8uQc2Ft\\\\\\\/Ewq6PwC\\\\\\\/zC028q+KHk9s1YO\\\\\\\/Qeg7LsXdpM48yuclzAxOlVRy30Akff\\\\\\\/oCzR3mM5RTuT+ZvYe+mLUXJ9gn0Pgy6hZCYeqd2VHL1lYMq3oScSL4YAC7lcm1ekrlNa\\\\\\\/GPPyr84yQOpjU+SiSNCfKK1OfTdjS6mz3Tx+yyj6z7\\\\\\\/HqK11yMCU64MHqkVHglr8189YXsrUaPaMQzsku0M\\\\\\\/yFNtKmn6+mBo9udhsxxUOn9M14ajuf8lk7J3uDt18bTl9eIZaZo+5\\\\\\\/NJK0M4iTG6Ak5UbqLOeNucW24WnjuORpZ99AoUzg+QbkX4\\\\\\\/jVRzTdCOk2cdloNaed416y5alRNQ00HRTFSxmJMvsgBU8ZjjauWUxllF1lt8qzS2BpumoRdxu2g5Ig60GPPJ3ccvZoid7Yzk7ipD\\\\\\\/LIv7QyJf9SX8hOMHVo1pCy5WkI4uxTM95tnpq4qy\\\\\\\/aVyiVWmc6C28JBBCIJjSDJKWjzFRvoZ87evq8xeFJRFCPX6LFb2kY5yOMuDThMVWPEdjZJTA\\\\\\\/b3EMvIGEF4TRl7cImU45dD58DeaZruFl8qOtKuhUsWH\\\\\\\/nl\\\\\\\/7O6ycjpl0EXxgCIyHbdWznKAiyESB4ElE6nJ0IT1YhUnLQ5PUUFUZcW8og1JJ8QMI0FMCqY2BK9xLbsr6RnGcV9Ve6YQ+Vo6Nw7XDGKF2H9Kjq+LSxM9W4ytZMGIPIphOMUEja\\\\\\\/RNz\\\\\\\/HhPdl+c7abcMK4CxEqWM5mQvnF33AcfVcoi4JU1+QI5GxwAka4b7rQohUyHcNhx+7getp3Y6rNuYg\\\\\\\/FBFSpY6rZMk0PUjE88C4x2SRtIOhJNPaf8HHgBwfo1jrjk1j\\\\\\\/BHYAfaNKQG3WzjvBnicKruWlBqnV37SRuXHcvB\\\\\\\/sKYSw2OzE7VFcYhYJXbw5L6BgPNdnz8eEs4V\\\\\\\/C8pFu8qA0qtWbMw95VwNKB+O+6Tgfb43Q2pYpXA3S9QbVmHDZbQlVq2PHuI4odXDopj8UXTWtNXEXlV\\\\\\\/kRWFl\\\\\\\/kTZ33WJxRs7cWkRdtHy5YvJBg\\\\\\\/0KizH5PzRPYvefniLR3XZWheHC39Y5eJPKmIswU9N8i1iyh58L578FmLJrCqkSTufK5W7z+cmhoBYrq8S6LUGtXnZTmB2ZkVNii\\\u0022,\\\u0022iv\\\u0022:\\\u0022TfIQM5HKPy+2qUknfYXudA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dc1248a57751c6224de9789b2f19f2fdc5e8e385c9e7e7444301a466d539dbaa\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484257 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1563" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5st66Kh9INC4k2peTcXDQPm50DWakuOiyXdp3KI6fuE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1563" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zp26n3SrRfhX3bXBdcdiAc7nOmY7kQl-2wP7IgwmIdo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1551" title="Genotyping-by-Sequencing Facilitates a High-Density Consensus Linkage Map for Aegilops umbellulata, a Wild Relative of Cultivated Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1569" title="asymptoticMK: A Web-Based Tool for the Asymptotic McDonald–Kreitman Test" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484257" id="node484257" data-pisa="ggg;7/5/1563" data-pisa-master="ggg;g3.117.040055" data-apath="/ggg/7/5/1563.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (<em>Pinus lambertiana</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marc W. Crepeau</span>, <span class="highwire-citation-author" data-delta="1">Charles H. Langley</span> and <span class="highwire-citation-author" data-delta="2">Kristian A. Stevens</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1563-1568;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040055 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484257" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marc W. Crepeau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BW.%2BCrepeau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crepeau%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarc%2BW.%2BCrepeau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Charles H. Langley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BH.%2BLangley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langley%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BH.%2BLangley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kristian A. Stevens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristian%2BA.%2BStevens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stevens%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKristian%2BA.%2BStevens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kastevens@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">kastevens@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/5/1563" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484257" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484257/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1563.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484257" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484257/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1563.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484257" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484257/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/5/1563.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484257" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484257/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">We investigate the utility and scalability of new read cloud technologies to improve the draft genome assemblies of the colossal, and largely repetitive, genomes of conifers. Synthetic long read technologies have existed in various forms as a means of reducing complexity and resolving repeats since the outset of genome assembly. Recently, technologies that combine subhaploid pools of high molecular weight DNA with barcoding on a massive scale have brought new efficiencies to sample preparation and data generation. When combined with inexpensive light shotgun sequencing, the resulting data can be used to scaffold large genomes. The protocol is efficient enough to consider routinely for even the largest genomes. Conifers represent the largest reference genome projects executed to date. The largest of these is that of the conifer <em>Pinus lambertiana</em> (sugar pine), with a genome size of 31 billion bp. In this paper, we report on the molecular and computational protocols for scaffolding the <em>P. lambertiana</em> genome using the library technology from 10× Genomics. At 247,000 bp, the NG50 of the existing reference sequence is the highest scaffold contiguity among the currently published conifer assemblies; this new assembly’s NG50 is 1.94 million bp, an eightfold increase.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/10%C3%97-genomics" class="hw-term hw-article-keyword hw-article-keyword-10&#xD7;-genomics" rel="nofollow">10× genomics</a></li><li class="kwd"><a href="/keyword/conifer-genomes" class="hw-term hw-article-keyword hw-article-keyword-conifer-genomes" rel="nofollow">conifer genomes</a></li><li class="kwd"><a href="/keyword/genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-genome-assembly" rel="nofollow">genome assembly</a></li><li class="kwd"><a href="/keyword/sugar-pine" class="hw-term hw-article-keyword hw-article-keyword-sugar-pine" rel="nofollow">sugar pine</a></li></ul><p id="p-3">As a class, conifer genome reference sequences represent the largest genome assemblies accomplished to date. Recent and active conifer genome projects have target genomes far exceeding that of the human genome (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Birol <em>et al.</em> 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Nystedt <em>et al.</em> 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Neale <em>et al.</em> 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Warren <em>et al.</em> 2015</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Stevens <em>et al.</em> 2016</a>). Most of the size of conifer genomes derives from insertions of transposable elements (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Nystedt <em>et al.</em> 2013</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Stevens <em>et al.</em> 2016</a>). Their large scale and repetitive nature make them an important assembly benchmark, and a target for technological improvements. The largest genome project to date is that of the white pine <em>Pinus lambertiana</em> (sugar pine), with a genome size of 31 Gbp (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Stevens <em>et al.</em> 2016</a>).</p><p id="p-4">Complexity reduction played an important role in the initial assembly of <em>P. lambertiana</em>. A gymnosperm megagametophyte (MGP) is maternally derived tissue found within each seed containing the same haploid genome that is contributed to the diploid zygote (embryo). The haploid MGP contains enough DNA for creation of a set of size-selected paired end Illumina libraries. These can be sequenced to sufficient depth to support the creation of a high quality, error corrected, read set forming the basis for <em>de novo</em> assemblies (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Zimin <em>et al.</em> 2014</a>; <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Stevens <em>et al.</em> 2016</a>). In this paper, we present a protocol for scaffolding a conifer genome applying the new library technology, described in detail below, from 10× genomics (<a href="http://www.10xgenomics.com">www.10xgenomics.com</a>). Because DNA requirements are small, the protocol can be implemented using DNA from a single conifer megagametophyte. Application of this new protocol to <em>P. lambertiana</em> results in an eightfold improvement to the scaffold contiguity of the colossal genome. For most MGPs, extracted DNA will be of sufficient quality and quantity that the protocol presented here can be used in parallel with methods for generating standard size selected paired end libraries (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Zimin <em>et al.</em> 2014</a>).</p><p id="p-5">The idea of reducing the complexity of the assembly problem by simplifying the target has been around since the early genome projects. The shotgun sequencing and assembly of cloned DNA was the original method of constructing “synthetic long reads”. On a larger scale, the shotgun sequencing and assembly of fosmid pools to create “synthetic long reads” has also been used effectively for the assembly of complex genomes (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Duitama <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Zhang <em>et al.</em> 2012</a>) and proposed for conifers (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexeyenko <em>et al.</em> 2014</a>). Alternatively, Illumina’s TruSeq Synthetic Long Read technology (formerly Moleculo) uses shotgun sequencing and assembly of amplified high molecular weight (HMW) DNA to construct synthetic long reads (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Voskoboynik <em>et al.</em> 2013</a>). These protocols suffer a common practical limitation on the number of independent subhaploid pools that can be generated. Contiguity information is extracted from each pool by deep sequencing and assembly to generate synthetic long reads, which are useful as inputs to another assembly.</p><p id="p-6">Recently, it has become practical to replace the deep sequence coverage requirement of synthetic long reads, with a deep subhaploid pool coverage. We follow <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Kuleshov <em>et al.</em> (2016)</a> and refer to the resulting data as synthetic long read “clouds.” When aligned to a genome, each long DNA fragment is represented by a low-density cloud of clustered reads. Currently, few methods exist for implementing deep pool coverage.</p><p id="p-7">The GemCode system from 10× Genomics (<a href="http://www.10xgenomics.com">www.10xgenomics.com</a>) utilizes a combination of proprietary hardware and reagents to produce Illumina sequencing libraries that preserve the long-range linkage information present in large DNA molecules. The two key components of their platform are (1) a microfluidic device designed to produce emulsified droplets each containing a small “pool” of HMW DNA fragments, and (2) distinct indices drawn from a massive set (∼750,000) of 14-bp oligonucleotide “barcodes” (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Each library begins with ∼1 ng of input DNA, which is partitioned into ∼200,000 barcoded pools. These droplet partitions are proprietarily named GEMs, for Gel bead in EMulsion. Because the amount of partitioned DNA is small (&lt;10 fg), for genomes of sufficient size, each GEM will contain a collection of genomic fragments comprising a small fraction of the haploid genome. Multiple libraries can be used to further increase the total number of barcoded pools generated. High quality linkage information is contingent upon HMW input DNA. One advantage of the 10× Genomics protocol, in this regard, is a relatively small amount of input DNA (∼1 ng) recommended.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1563/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(Top) Construction of synthetic long read clouds with 10&#xD7; Genomics technology. (A) HMW DNA is prepared from a single haploid sugar pine MGP. (B) Within the instrument emulsion droplets are used to pool HMW DNA. A barcode containing bead in each droplet is used for indexing the pools. During subsequent thermal cycling the bead dissolves and the input DNA fragments act as templates for primer extension by the barcoded random hexamers (blue). All extension products within a droplet contain the same 14&#xA0;bp barcode (magenta). After completion of primer extension cycles the emulsion is &#x201C;broken,&#x201D; pooled extension products are physically sheared to subkilobase fragments, and p7 and p5 adapters are added to the barcoded terminal fragments by ligation and enrichment PCR. Final molecules have the standard configuration and adapter sequences of Illumina dual-index, paired-end libraries, except that i5 is 14&#xA0;bp instead of 8&#xA0;bp. The random hexamer sequence (Nmer) is trimmed from the start of read one during data processing. (C) The oligo-containing gel bead (colored dots) within each droplet contains a single 14&#xA0;bp barcode and multiple long DNA fragments (wavy lines) that serve as templates for generation of library fragments and then reads (short bicolored bars). Different droplets may contain identical gel beads increasing the effective size of the pool. All reads with the same barcode form an effective pool. (D) Alignments to a contig will assign the contig to one or more pools. Overlap graph nodes are defined in windows at the contig ends. Because there are a large number of distinct pools (barcodes), any two nodes are unlikely to belong to the same two (or more) pools by chance. However, one shared barcode is common. (Bottom) Scaffolding with synthetic long read clouds and fragScaff. (E) For each node, the read group assignments for that node are compared with the read group assignments for every other node and the fraction of read groups shared between them (their &#x201C;shared fraction&#x201D;) is calculated. (F) A typical distribution of shared fraction for a node in our data. For each node, a normal distribution is fitted to the observed shared fraction data. Link scores for each ordered pair of nodes are computed by taking the negative log10 probability of the observed shared fraction under the fitted normal distribution. After all pairwise link scores are calculated, a global link score threshold is determined. In our example, the pair (n, n&#x2032;) shared one barcode and are below the threshold, while the pair (n, n&#x2032;&#x2032;) shared &gt;1 and are above. (F) Pairs of nodes with a score exceeding the global link score threshold become edges in an overlap graph with a weight corresponding to the score. (G) Layout linearizes the weighted graph and is accomplished by greedy algorithm described in (Adey et al. 2014). From each linearized subgraph the consensus scaffold is determined by concatenating component scaffolds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1456740389" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(Top) Construction of synthetic long read clouds with 10&#xD7; Genomics technology. (A) HMW DNA is prepared from a single haploid sugar pine MGP. (B) Within the instrument emulsion droplets are used to pool HMW DNA. A barcode containing bead in each droplet is used for indexing the pools. During subsequent thermal cycling the bead dissolves and the input DNA fragments act as templates for primer extension by the barcoded random hexamers (blue). All extension products within a droplet contain the same 14&#xA0;bp barcode (magenta). After completion of primer extension cycles the emulsion is &#x201C;broken,&#x201D; pooled extension products are physically sheared to subkilobase fragments, and p7 and p5 adapters are added to the barcoded terminal fragments by ligation and enrichment PCR. Final molecules have the standard configuration and adapter sequences of Illumina dual-index, paired-end libraries, except that i5 is 14&#xA0;bp instead of 8&#xA0;bp. The random hexamer sequence (Nmer) is trimmed from the start of read one during data processing. (C) The oligo-containing gel bead (colored dots) within each droplet contains a single 14&#xA0;bp barcode and multiple long DNA fragments (wavy lines) that serve as templates for generation of library fragments and then reads (short bicolored bars). Different droplets may contain identical gel beads increasing the effective size of the pool. All reads with the same barcode form an effective pool. (D) Alignments to a contig will assign the contig to one or more pools. Overlap graph nodes are defined in windows at the contig ends. Because there are a large number of distinct pools (barcodes), any two nodes are unlikely to belong to the same two (or more) pools by chance. However, one shared barcode is common. (Bottom) Scaffolding with synthetic long read clouds and fragScaff. (E) For each node, the read group assignments for that node are compared with the read group assignments for every other node and the fraction of read groups shared between them (their &#x201C;shared fraction&#x201D;) is calculated. (F) A typical distribution of shared fraction for a node in our data. For each node, a normal distribution is fitted to the observed shared fraction data. Link scores for each ordered pair of nodes are computed by taking the negative log&lt;sub&gt;10&lt;/sub&gt; probability of the observed shared fraction under the fitted normal distribution. After all pairwise link scores are calculated, a global link score threshold is determined. In our example, the pair (n, n&#x2032;) shared one barcode and are below the threshold, while the pair (n, n&#x2032;&#x2032;) shared &gt;1 and are above. (F) Pairs of nodes with a score exceeding the global link score threshold become edges in an overlap graph with a weight corresponding to the score. (G) Layout linearizes the weighted graph and is accomplished by greedy algorithm described in (Adey &lt;em&gt;et al.&lt;/em&gt; 2014). From each linearized subgraph the consensus scaffold is determined by concatenating component scaffolds.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1563/F1.medium.gif" width="326" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/5/1563/F1.medium.gif" width="326" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1563/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1563/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484306" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">(Top) Construction of synthetic long read clouds with 10× Genomics technology. (A) HMW DNA is prepared from a single haploid sugar pine MGP. (B) Within the instrument emulsion droplets are used to pool HMW DNA. A barcode containing bead in each droplet is used for indexing the pools. During subsequent thermal cycling the bead dissolves and the input DNA fragments act as templates for primer extension by the barcoded random hexamers (blue). All extension products within a droplet contain the same 14 bp barcode (magenta). After completion of primer extension cycles the emulsion is “broken,” pooled extension products are physically sheared to subkilobase fragments, and p7 and p5 adapters are added to the barcoded terminal fragments by ligation and enrichment PCR. Final molecules have the standard configuration and adapter sequences of Illumina dual-index, paired-end libraries, except that i5 is 14 bp instead of 8 bp. The random hexamer sequence (Nmer) is trimmed from the start of read one during data processing. (C) The oligo-containing gel bead (colored dots) within each droplet contains a single 14 bp barcode and multiple long DNA fragments (wavy lines) that serve as templates for generation of library fragments and then reads (short bicolored bars). Different droplets may contain identical gel beads increasing the effective size of the pool. All reads with the same barcode form an effective pool. (D) Alignments to a contig will assign the contig to one or more pools. Overlap graph nodes are defined in windows at the contig ends. Because there are a large number of distinct pools (barcodes), any two nodes are unlikely to belong to the same two (or more) pools by chance. However, one shared barcode is common. (Bottom) Scaffolding with synthetic long read clouds and fragScaff. (E) For each node, the read group assignments for that node are compared with the read group assignments for every other node and the fraction of read groups shared between them (their “shared fraction”) is calculated. (F) A typical distribution of shared fraction for a node in our data. For each node, a normal distribution is fitted to the observed shared fraction data. Link scores for each ordered pair of nodes are computed by taking the negative log<sub>10</sub> probability of the observed shared fraction under the fitted normal distribution. After all pairwise link scores are calculated, a global link score threshold is determined. In our example, the pair (n, n′) shared one barcode and are below the threshold, while the pair (n, n′′) shared &gt;1 and are above. (F) Pairs of nodes with a score exceeding the global link score threshold become edges in an overlap graph with a weight corresponding to the score. (G) Layout linearizes the weighted graph and is accomplished by greedy algorithm described in (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adey <em>et al.</em> 2014</a>). From each linearized subgraph the consensus scaffold is determined by concatenating component scaffolds.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">The first application for this data was haplotype phasing, which relies on a reference genome alignment to directly demarcate the long (linking) DNA fragments from aligned reads. This follows similar implementations by others using related (but lower-plexity) data types (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Peters <em>et al.</em> 2012</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amini <em>et al.</em> 2014</a>). Utilizing read clouds for assembly relies on a property of the interactions between &gt;1 barcode. For the subhaploid dilution achieved, multiple DNA fragments have the same barcode, hence any given barcode is associated with many parts of the genome. However, the dilution into pools is of sufficient magnitude to ensure that coincidental overlap between two barcodes is rare. This implies that two (or more) barcodes will intersect at a unique location. Similarly, if two or more barcodes are shared between unlinked sequences, they are likely in close proximity to each other. Few programs exist that will utilize this information for genome assembly (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Adey <em>et al.</em> 2014</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Kuleshov <em>et al.</em> 2016</a>). To create our new assembly we used the <em>fragScaff</em> program from <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Adey <em>et al.</em> (2014)</a>. <em>fragScaff</em> implements a modification of the classic overlap–layout–consensus paradigm (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Myers 2005</a>). Specifically, <em>fragScaff</em> uses a link score to define overlaps and build an overlap graph. Each edge in this graph has a weight. The layout step is accomplished by finding a new simpler graph, which, when traversed, defines a scaffold. The consensus step concatenates component scaffold sequences with spacers.</p><p id="p-10">The primary input to <em>fragScaff</em> is a bam formatted alignment file. In the case of our data, each aligned read is annotated with a 14 bp barcode. Sequencing may be single read, for our scaffolding protocol, the additional read in paired-end sequencing is used only to help resolve alignment location. The other important input is a bed formatted annotation of repetitive regions of the genome, which is used to establish the subset of trusted alignments. We address methods for creating this annotation as well as setting the most important assembly parameters in this paper.</p><p id="p-11">Our results build on the results of <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Mostovoy <em>et al.</em> (2016)</a> for the human genome. We demonstrate the utility and scalability of this approach, and establish a protocol for the much larger, more repetitive, and more fragmented genomes of conifers. We validate our approach using empirical estimates of the type 1 and type 2 error rates in linking.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Library construction and sequencing</h3><p id="p-12">DNA was extracted from a single sugar pine MGP using the method previously reported in <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Zimin <em>et al.</em> (2014)</a> for obtaining HMW DNA from pine needles. In total, 3.1 µg of DNA was obtained. Size selection was performed using an aliquot of 1.1 µg run on a PAC30KB BluePippin cassette (Sage Science, Inc.) with a 40 kb cutoff, yielding 200 ng of size-selected DNA. Library SPX1 was constructed to evaluate non-size-selected DNA while libraries SPX2 through SPX5 used the size-selected DNA. All libraries began with 1.2 ng of DNA, and followed the protocol in the version B GemCode user guide with the shearing regime adapted to the Covaris E220 (Covaris, Inc.). GemCode DNA was brought to a final volume of 130 µl and sheared in a Covaris eight microTUBE strip at the Covaris E220 recommended settings for 800 bp fragments. DNA was then concentrated to 50 µl using 1.8× SPRI-select beads (Beckman-Coulter, Inc.).</p><p id="p-13">For evaluation purposes, we initially sequenced one non-size-selected library (SPX1), and one size-selected library (SPX2) on two lanes of a HiSeq2500 Rapid Run flowcell with a dual-index, pair-end run regime of 98+8+14+98 cycles. The size-selected library showed a substantial improvement in the overall distribution of fragment sizes and physical coverage (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Subsequent effort focused on SPX2, and three additional size-selected libraries (SPX3, SPX4, and SPX5). These were sequenced on two lanes of a HiSeq4000 flowcell with a dual-index, pair-end run regime of 101+8+14+101 cycles.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484345/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484345/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Library and sequencing results</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Sequence processing and genome alignment</h3><p id="p-15">The Illumina BCL files for all sequencing runs were translated to FASTQ using <em>bclprocessor</em> (10× Genomics). The resulting FASTQ was demultiplexed (both sample indices and pool barcodes) using the BASIC pipeline (10× Genomics). Demultiplexed reads are labeled by a pool barcode: 14 bp barcode concatenated to a sample designator. A readgroups.tsv file was also produced containing all pool barcodes with 50 or more reads. Only barcodes contained this file were considered for scaffolding. Demultiplexed FASTQ files were aligned to the genome to be scaffolded using BWA (Li <em>et al.</em> 2010) specifying that read group identifiers, when present, be appended to each record (bwa mem -h readgroups.tsv). This output was then filtered with <em>samtools</em> to remove low quality alignments, PCR duplicates, unmapped reads, unpaired reads, and supplemental alignments (samtools view -q 10 -f 2 –F 3084).</p></div><div id="sec-4" class="subsection"><h3>Read cloud length and physical coverage estimation</h3><p id="p-16">We used the read cloud length distribution as a proxy for the molecular weight distribution of the fragments used for library construction. We demarcated read clouds for purposes of estimating a length distribution. Reads aligning to the larger <em>P. lambertiana</em> v1.0 scaffolds (&gt;500 kb) were isolated and the alignments were subsequently binned into 5 kb windows across each scaffold. Consecutive runs of windows with the same barcode were interpreted as read clouds. We expect, on average, the read clouds containing multiple windows will be padded by one half window on either end. To compensate for this we subtracted one window width from all our estimates. To estimate density, the total number of reads contained in the read clouds defined for each library was divided by the sum of the read cloud lengths in kilobase for that library to determine the aligned reads per kilobase. To estimate genome wide physical coverage, the observed physical coverage from the subset of larger scaffolds was scaled to the complete genome using the ratio of all aligned reads to reads aligned to the larger scaffolds. This scaling factor was consistently estimated as 3.1.</p></div><div id="sec-5" class="subsection"><h3>Repeat annotation</h3><p id="p-17"><em>fragScaff</em> employs BED formatted files to ignore repeats and ambiguous bases in the definition of nodes. A RepeatMasker exclusion file was prepared using the <em>P. lambertiana</em> v1.0 repeat annotation described in <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Stevens <em>et al.</em> (2016)</a>. A mapping-quality-based exclusion file was prepared by converting low quality alignments (MAPQ ≤ 10) into BED formatted genomic intervals. These intervals were subsequently unioned with <em>bedtools</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Quinlan and Hall 2010</a>) to form a nonredundant mask. Intervals were unioned if they were separated by &lt;40 bp. The BED file annotating the coordinates of all intervals of Ns in the reference genome was created using a Python script.</p></div><div id="sec-6" class="subsection"><h3>Calibration runs</h3><p id="p-18">To construct a dataset appropriate for calibration and validation, we split <em>P. lambertiana</em> v1.0 scaffolds at least 500 kb in length into windows that were 50 kb in length (excluding Ns) with the exception of the final window. This process generated 157,873 new <em>child</em> scaffolds from 10,403 <em>parent</em> scaffolds. These new <em>child</em> scaffolds were used as a validation dataset, replacing their parent scaffolds as the input data for <em>fragScaff</em>. Using a Python script, the bam and bed files were also transformed to the new coordinate system. Reads were reassigned to scaffolds based on their leftmost coordinate. Calibration run results were analyzed to compare the order of the reassembled child scaffolds with their ordering in the original <em>P. lambertiana</em> v1.0 parent scaffolds. The joins made by fragScaff and the potential joins were classified into the following mutually exclusive categories:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-19"><strong>Correct:</strong> A join between two adjacent child scaffolds originating from the same parent scaffold.</p></li><li id="list-item-2"><p id="p-20"><strong>Type 1 error:</strong> A join between two nonadjacent child scaffolds. If from different parents, only one child can be the first or last (<em>i.e.</em>, terminal) piece of its parent scaffold. Joins between nonterminal child scaffolds and smaller un-split scaffolds were also counted.</p></li><li id="list-item-3"><p id="p-21"><strong>Type 2 error</strong>: Two unjoined child scaffolds that are adjacent in a parent scaffold.</p></li><li id="list-item-4"><p id="p-22"><strong>Ignored:</strong> A join of two child scaffolds where both are terminal pieces from different parent scaffolds. These terminal joins between parents were considered unverifiable.</p></li></ul></div><div id="sec-7" class="subsection"><h3>Production run</h3><p id="p-23">Following calibration, we assembled original unmodified scaffolds using selected parameters. The <em>fragScaff</em> program was run with an updated terminal window size (–<em>E</em> = 10,000), clipping the top 1.5% of shared fraction distribution (–<em>D</em> = 0.985), and our custom repeat mask based on analysis of mapping quality. Furthermore, to improve tractability we limited <em>fragScaff</em> to the scaffolds 5 kb and longer (–<em>m</em> = 5000) representing 74.8% of the genome. The sparse coverage meant that many of the shortest genome elements would not have enough read group assignments to supply the statistical power necessary for inclusion in the assembly.</p></div><div id="sec-8" class="subsection"><h3>Fosmid pool comparison</h3><p id="p-24">The program nucmer (nucmer–maxmatch –l 100 –c 100; delta-filter –r –i 90; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Kurtz <em>et al.</em> 2004</a>) was also used to align our assembly (<em>P. lambertiana</em> v1.0) to the collection of PacBio sequenced and assembled fosmids reported in <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Stevens <em>et al.</em> (2016)</a>. We expected a few fosmids to overlap joins made here in constructing <em>P. lambertiana</em> v1.5. Alignments involving joined scaffolds were inspected for evidence supporting or contradicting the joins.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-25">The sequence data used for this study was deposited in the NCBI trace archive under BioProject 174450 and accession number SRX2629912. The assembly is available at the Pine Reference Sequence project site <a href="http://dendrome.ucdavis.edu/ftp/Genome_Data/genome/pinerefseq/Pila/v1.5/">http://dendrome.ucdavis.edu/ftp/Genome_Data/genome/pinerefseq/Pila/v1.5/</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Libraries and sequencing</h3><p id="p-26">We prepared five GemCode libraries from DNA extracted from a single <em>P. lambertiana</em> megagametophyte (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The megagametophyte used was from the same mother tree (tree 5038; <a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Stevens <em>et al.</em> 2016</a>) as the <em>P. lambertiana</em> v1.0 reference genome assembly. The haplotype captured in our libraries is a sibling meiotic haplotype to the one used for <em>P. lambertiana</em> v1.0. We expect that 50% of the content will be the identical genotype.</p><p id="p-27">In total, we sequenced 1,385,363,072 reads from barcoded libraries, representing 4.25× raw genome coverage. Subsequent alignment to the genome yielded 1,226,556,973 aligned reads containing valid barcodes. After Samtools filtering (see <em>Materials and Methods</em>), the total number of aligned reads with valid barcodes was further reduced to 760,034,806 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). For each library, more than half a million valid pool barcodes were present among the filtered reads; however, we only utilized alignments from barcodes present 50 or more times, which reduced the number of “read groups” (<em>i.e.</em>, distinct barcodes) per library to less than a quarter million (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The distribution of filtered alignments per pool was consistent across libraries with a negligible tail.</p><p id="p-28">Across all size selected libraries, our estimated median length of read clouds was consistently ∼50 kb (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) for the size-selected libraries. The <em>N</em>50 weighted averages for HMW DNA fragment length were also consistent, with half of the coverage coming from fragments &gt; 80 kb (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). The average number of alignments per kilobase within each read cloud varied from a low of 0.87 reads per kilobase for SPX3 to a high of 1.5 reads per kilobase for SPX1, with a median of 1.0 for SPX5. While read coverage was sparse (4.5×), total physical coverage in HMW DNA fragments was deeper (23.8×) for the haploid 31 Gbp genome.</p></div><div id="sec-12" class="subsection"><h3>Parameter calibration and optimization</h3><p id="p-29">We deemed it critical to empirically validate our assembly results. We opted to use the most contiguous elements of our existing genome assembly as the internal control, and to generate a corresponding test dataset <em>in silico</em>. Specifically, we began with all 10,403 elements whose total length (including gaps) was ≥500 kb, and broke these into 157,873 pieces that were exactly 50 kb in length (excluding Ns) with the exception of the 3′ terminal pieces. These fragmented scaffolds then became the input data for <em>fragScaff</em>. The original order and orientation of the 50 kb pieces allowed us to evaluate the performance of our scaffolding procedure under different parameter settings. We refer to these assemblies as “calibration runs” since we used them to evaluate parameters for our final assembly of unmodified data.</p><p id="p-30">This methodology relies on two plausible assumptions: (1) the existing assembly for elements &gt;500 kb in length is essentially correct and representative; (2) the performance of the methodology for joining 50 kb elements will be indicative of the performance for the overall genome. It is also noteworthy that, because the haploid genome of our megagametophyte is not identical to the reference genome sequence, there is the possibility that genuine structural variation is a source of error.</p><p id="p-31">Runs were performed in four stages: (1) bam file parsing, which included definition of nodes and barcode hits to each node; (2) shared fraction calculations, which was the most computationally intensive step; and (3) graph based layout and (4) consensus. We focused our efforts on evaluating and optimizing the exposed <em>fragScaff</em> parameters and procedures that directly effect the definition of nodes and edges in initial graph construction.</p><div id="sec-13" class="subsection"><h4>Optimizing repeat exclusion:</h4><p id="p-32">The <em>P. lambertiana</em> genome is highly repetitive. Reads originating from repetitive regions obfuscate scaffolding because their genomic locations are ambiguous. Hence, the fragScaff protocol recommends the specification of an exclusion file in BED format. Regions specified in this file will be excluded from nodes so that reads aligning within them are not considered. For the <em>P. lambertiana</em> genome, a repeat mask file masks 79% of the genome (<a id="xref-ref-15-8" class="xref-bibr" href="#ref-15">Stevens <em>et al.</em> 2016</a>). We ran a parameter-calibration run using this exclusion file with default parameters and estimated a type I error of 5.84%, and an even higher type II error of 25.73%, <em>i.e.</em>, only 74.27% of expected joins were made.</p><p id="p-33">An alternative approach was to rely on BWA mapping quality estimates to annotate repetitive regions. For each alignment, BWA estimates a confidence in the location, in the form of a Phred transformed probability that the locus is correct (mapping quality or MAPQ). By default, the program only considers alignments with a mapping quality &gt;10. We created an additional exclusion file based on the mapping quality estimates. This was done by collecting all alignments with MAPQ ≤ 10, converting the alignment coordinates to BED format (see <em>Materials and Methods</em>). This produced a less restrictive (more targeted) exclusion file. Similarly, using this exclusion file in a parameter-calibration run, we found that it improved type I error to 4.92%, while the type II error rate and contiguity were substantially improved: 84.80% of expected joins were made.</p></div><div id="sec-14" class="subsection"><h4>Optimizing the window size for defining graph nodes:</h4><p id="p-34">We evaluated two settings for the number of unmasked bases defining an overlap graph node (–<em>E</em> parameter), the default value of 5000 bp, and larger value of 10,000 bp, which would incorporate more barcodes. In both cases the maximum node length (–<em>o</em>) was kept at 50,000 bp, and the minimum input element length (–<em>m</em>) was set to 5000 bp. The median number of barcodes per node increased from 26 to 39. This increase improved the number of valid links made. Upon updating the value of –<em>E</em> from 5000 to 10,000, our estimated type II error rate dropped from 15.20 to 13.52%. This modest change has a substantial impact on the <em>N</em>50 value of the calibration run, increasing it by 25%. Our type I error rate increased slightly from 4.92 to 5.19%.</p></div><div id="sec-15" class="subsection"><h4>Optimizing overlap graph construction:</h4><p id="p-35">For each node, fragScaff generates a histogram of shared valid pool barcodes with all other nodes. This distribution forms the basis of the link score calculation, and ultimately what edges get included in the overlap graph. By default, the outlying 5% tails of this distribution is completely ignored. We noticed that many more false negatives than false positives fell in the right tail of this distribution. By examining the results of our parameter calibration runs, we were able to modify the value of the (–<em>D</em>) parameter leading to improved type II error rates relative to its default setting. Using calibration data, we estimated the likely impact of adjustments to the type I error and type II error rates. This led to updating of the parameter to filter the top 1.5% of the distribution, rejecting nodes with 98 or more read groups. Comparing similar calibration runs before and after parameter update, we observe a reduction in type II error rate from 13.52 to 8.69%, while type I error rate stayed relatively contained (5.19% <em>vs.</em> 4.85%, respectively).</p></div></div><div id="sec-16" class="subsection"><h3>Final assembly and validation</h3><p id="p-36">The production fragScaff assembly of all <em>P. lambertiana</em> v1.0 elements ≥5 kb resulted in an eightfold improvement in scaffold <em>N</em>50 relative to the input (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Merging this product with the remaining (&lt;5 kb) elements of <em>P. lambertiana</em> v1.0, we obtain a final <em>P. lambertiana</em> v1.5 reference genome. Scaling by a genome size of 31 billion bp, we obtain a scaffold NG50 of 1.94 million bp, this is also an eightfold improvement over the original NG50 scaffold of 247 kb.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484301/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484301/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Assembly statistics before and after rescaffolding with long read clouds using fragScaff</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">The final estimates of the type I and type II linking error rates correspond to the values of 4.85 and 8.69% reported for our final calibration run. We also validated links using an independent PacBio sequenced and HGAP assembled pool of 40 kb fosmid clones from the same diploid mother. We found 14 instances where nucmer results (see <em>Materials and Methods</em>) supported the overlap of two whole genome shotgun (WGS) elements by a single fosmid (although the highly repetitive content of the genome causes frequent spurious alignments that add uncertainty to such interpretations). Since the linking range of our GemCode data easily spans the length of a fosmid clone, we expected pairs of WGS elements contained within a fosmid to be ideal candidates for joining, and, indeed, 7 of the 14 were found to be joined in the <em>P. lambertiana</em> v1.5 assembly.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-39"><em>P. lambertiana</em> is the largest genome sequenced and assembled to date. Among conifers, this reference genome assembly already stands out as having the highest scaffold contiguity (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Nystedt <em>et al.</em> 2013</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Neale <em>et al.</em> 2014</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Warren <em>et al.</em> 2015</a>; <a id="xref-ref-15-9" class="xref-bibr" href="#ref-15">Stevens <em>et al.</em> 2016</a>). Like most initial assemblies of megagenomes, large-scale scaffolding is the obvious next step toward a more contiguous and serviceable resource. In this paper, we report the generation of deep physical coverage from subhaploid pools using a single molecule sequencing technology (10× Genomics). Relatively light sequencing of the resulting libraries was needed to obtain a substantial eightfold increase to the weighted scaffold contiguity.</p><p id="p-40">System performance with this technology is highly dependent on the quality, especially the size distribution, of the input DNA. In particular, the range of the linkage information produced is contingent upon the distribution of molecular weight of the starting DNA. We demonstrated that sufficient HMW DNA could be obtained from a single conifer megagametophyte for this scaffolding. What was surprising was that the majority (nearly 99%) of the DNA extracted from the single megagametophyte was not actually needed since the input requirements of library construction are so extremely modest. Thus this remaining DNA could be used for additional single-molecular physical coverage. Alternatively it could be used as input to the protocol followed in the original sequencing and assembly of <em>P. lambertiana</em> (<a id="xref-ref-15-10" class="xref-bibr" href="#ref-15">Stevens <em>et al.</em> 2016</a>). The implication is that a highly “scaffolded” genome assembly using <em>only</em> the DNA from a single MGP is achievable for many conifer species. The only requirement is that the extraction method be optimized for recovering HMW DNA.</p><p id="p-41">Previously, a haploid conifer MGP has been used for robust error correction, cleaning, and compression of both Illumina and PacBio sequences (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Zimin <em>et al.</em> 2014</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2017</a>; <a id="xref-ref-15-11" class="xref-bibr" href="#ref-15">Stevens <em>et al.</em> 2016</a>) to form the basis of a high quality genome assembly. The powerful protocol described here augments and extends that approach, and promises much better reference sequences in future applications.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-42">We gratefully acknowledge the management and staff of the DNA Technologies Core Facility at the University of California Davis Genome Center for performing the Illumina sequencing, and Nicole Rapicavoli, Joseph Aman, Michael Schnall-Levin, and David Jaffe at 10× Genomics for advice and technical assistance. We thank our colleagues in the PineRefSeq project for their many contributions to our appreciation of conifer genomics. We also want to acknowledge the support of the United States Department of Agriculture/National Institute of Food and Agriculture (USDA/NIFA) (2011-67009-30030).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-43"><em>Communicating editor: S. A. Jackson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-30"><span class="received-label">Received </span>January 30, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-03-21"><span class="accepted-label">Accepted </span>March 21, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Crepeau <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.1" data-doi="10.1101/gr.178319.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitzman</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daza</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">In vitro, long-range sequence information for de novo genome assembly via transposase contiguity.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2041</span>–<span class="cit-lpage">2049</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.178319.114%26rft_id%253Dinfo%253Apmid%252F25327137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMi8yMDQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexeyenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nystedt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vezzi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherwood</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient de novo assembly of large and complex genomes by massively parallel sequencing of fosmid pools.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.3" data-doi="10.1038/ng.3119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amini</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pushkarev</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christiansen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kostem</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Royce</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Haplotype-resolved whole-genome sequencing by contiguity-preserving transposition and combinatorial indexing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1343</span>–<span class="cit-lpage">1351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D1343%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3119%26rft_id%253Dinfo%253Apmid%252F25326703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25326703&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1563.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.4" data-doi="10.1093/bioinformatics/btt178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackman</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pleasance</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coope</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assembling the 20 Gb white spruce (<em>Picea glauca</em>) genome from whole-genome shotgun sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1492</span>–<span class="cit-lpage">1497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt178%26rft_id%253Dinfo%253Apmid%252F23698863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjkvMTIvMTQ5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.5" data-doi="10.1093/nar/gkr1042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duitama</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEwen</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huebsch</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palczewski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fosmid-based whole genome haplotyping of a HapMap trio child: evaluation of single individual haplotyping techniques.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">2041</span>–<span class="cit-lpage">2053</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1042%26rft_id%253Dinfo%253Apmid%252F22102577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQwLzUvMjA0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.6" data-doi="10.1093/bioinformatics/btw267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuleshov</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batzoglou</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome assembly from synthetic long read clouds.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">i216</span>–<span class="cit-lpage">i224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw267%26rft_id%253Dinfo%253Apmid%252F27307620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzIvMTIvaTIxNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smoot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shumway</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Versatile and open software for comparing large genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D5%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.5.1563.8" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long-read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi81LzU4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.9" data-doi="10.1038/nmeth.3865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mostovoy</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy-Sakin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A hybrid approach for <em>de novo</em> human genome sequence assembly and phasing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D13%26rft.spage%253D587%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3865%26rft_id%253Dinfo%253Apmid%252F27159086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27159086&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1563.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.10" data-doi="10.1093/bioinformatics/bti1114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The fragment assembly string graph.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>(<span class="cit-supplement">Suppl. 2</span>): <span class="cit-fpage">ii79</span>–<span class="cit-lpage">ii85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DMyers%26rft.auinit1%253DE.%2BW.%26rft.volume%253D21%26rft.issue%253Dsuppl_2%26rft.spage%253Dii79%26rft.epage%253Dii85%26rft.atitle%253DThe%2Bfragment%2Bassembly%2Bstring%2Bgraph%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti1114%26rft_id%253Dinfo%253Apmid%252F16204131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxNToiMjEvc3VwcGxfMi9paTc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.11" data-doi="10.1186/gb-2014-15-3-r59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Decoding the massive genome of loblolly pine using haploid DNA and novel assembly strategies.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253DR59%26rft.atitle%253DDECODING%2BTHE%2BMASSIVE%2BGENOME%2BOF%2BLOBLOLLY%2BPINE%2BUSING%2BHAPLOID%2BDNA%2BAND%2BNOVEL%2BASSEMBLY%2BSTRATEGIES.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-3-r59%26rft_id%253Dinfo%253Apmid%252F24647006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-3-r59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24647006&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1563.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.12" data-doi="10.1038/nature12211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nystedt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wetterbom</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuccolo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Norway spruce genome sequence and conifer genome evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">497</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12211%26rft_id%253Dinfo%253Apmid%252F23698360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23698360&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1563.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319556100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.13" data-doi="10.1038/nature11236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kermani</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sparks</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alferov</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accurate whole-genome sequencing and haplotyping from 10 to 20 human cells.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">487</span>: <span class="cit-fpage">190</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPeters%26rft.auinit1%253DB.%2BA.%26rft.volume%253D487%26rft.issue%253D7406%26rft.spage%253D190%26rft.epage%253D195%26rft.atitle%253DAccurate%2Bwhole-genome%2Bsequencing%2Band%2Bhaplotyping%2Bfrom%2B10%2Bto%2B20%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11236%26rft_id%253Dinfo%253Apmid%252F22785314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22785314&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1563.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306278900030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.14" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.15" data-doi="10.1534/genetics.116.193227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crepeau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sequence of the sugar pine megagenome</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">1613</span>–<span class="cit-lpage">1626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.193227%26rft_id%253Dinfo%253Apmid%252F27794028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNC80LzE2MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTU2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.16" data-doi="10.7554/eLife.00569"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voskoboynik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neff</span>  <span class="cit-name-given-names">N. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahoo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pushkarev</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genome sequence of the colonial chordate, <em>Botryllus schlosseri</em>.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e00569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.00569%26rft_id%253Dinfo%253Apmid%252F23840927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjIvMC9lMDA1NjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTU2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.17" data-doi="10.1111/tpj.12886"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeling</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">M. M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Improved white spruce (<em>Picea glauca</em>) genome assemblies and annotation of large gene families of conifer terpenoid and phenolic defense metabolism.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D83%26rft.spage%253D189%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12886%26rft_id%253Dinfo%253Apmid%252F26017574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12886&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26017574&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1563.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.18" data-doi="10.1038/nature11413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The oyster genome reveals stress adaptation and complexity of shell formation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">490</span>(<span class="cit-issue">7418</span>): <span class="cit-fpage">49</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZhang%26rft.auinit1%253DG.%26rft.volume%253D490%26rft.issue%253D7418%26rft.spage%253D49%26rft.epage%253D54%26rft.atitle%253DThe%2Boyster%2Bgenome%2Breveals%2Bstress%2Badaptation%2Band%2Bcomplexity%2Bof%2Bshell%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11413%26rft_id%253Dinfo%253Apmid%252F22992520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22992520&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1563.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309446800030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.19" data-doi="10.1534/genetics.113.159715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crepeau</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holtz-Morris</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koriabine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing and assembly of the 22-Gb loblolly pine genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZimin%26rft.auinit1%253DA.%26rft.volume%253D196%26rft.issue%253D3%26rft.spage%253D875%26rft.epage%253D890%26rft.atitle%253DSequencing%2Band%2BAssembly%2Bof%2Bthe%2B22-Gb%2BLoblolly%2BPine%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159715%26rft_id%253Dinfo%253Apmid%252F24653210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1563.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crepeau</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">An improved assembly of the loblolly pine mega-genome using long-read single-molecule sequencing</span>. <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/5/1563.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1551" title="Genotyping-by-Sequencing Facilitates a High-Density Consensus Linkage Map for Aegilops umbellulata, a Wild Relative of Cultivated Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1569" title="asymptoticMK: A Web-Based Tool for the Asymptotic McDonald–Kreitman Test" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/5"><span>Volume 7 Issue 5, May 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/5.cover-source.jpg?itok=VWLvGstM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1563.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VumO3Zkt6aiop-wXpU73lY3Efv8rrKliF_lRKvQRuE8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/5/1563" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/5/1563" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/5/1563" class="active" data-icon-position="" data-hide-link-title="0">From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (Pinus lambertiana)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484257" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-stQxsRC-GWVIDqTirV6R9Vs0vSLO8QPZuXoVRmUfP1Q" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1563.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/5/1563" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-RtyxsFkpjzgGCHH6qmbe7_5opkME95qB6oYxN2vfpNg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/5/1563" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1563.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484257" id="node484257--2" data-pisa="ggg;7/5/1563" data-pisa-master="ggg;g3.117.040055" data-apath="/ggg/7/5/1563.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (<em>Pinus lambertiana</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marc W. Crepeau</span>, <span class="highwire-citation-author" data-delta="1">Charles H. Langley</span> and <span class="highwire-citation-author" data-delta="2">Kristian A. Stevens</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1563-1568;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040055 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484257--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marc W. Crepeau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BW.%2BCrepeau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crepeau%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarc%2BW.%2BCrepeau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Charles H. Langley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BH.%2BLangley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langley%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BH.%2BLangley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kristian A. Stevens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristian%2BA.%2BStevens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stevens%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKristian%2BA.%2BStevens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kastevens@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">kastevens@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/5/1563" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/5/1563&amp;title=From%20Pine%20Cones%20to%20Read%20Clouds%3A%20Rescaffolding%20the%20Megagenome%20of%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/5/1563&amp;title=From%20Pine%20Cones%20to%20Read%20Clouds%3A%20Rescaffolding%20the%20Megagenome%20of%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/5/1563&amp;title=From%20Pine%20Cones%20to%20Read%20Clouds%3A%20Rescaffolding%20the%20Megagenome%20of%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/5/1563&amp;text=From%20Pine%20Cones%20to%20Read%20Clouds%3A%20Rescaffolding%20the%20Megagenome%20of%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/5/1563&amp;title=From%20Pine%20Cones%20to%20Read%20Clouds%3A%20Rescaffolding%20the%20Megagenome%20of%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/5/1563&amp;t=From%20Pine%20Cones%20to%20Read%20Clouds%3A%20Rescaffolding%20the%20Megagenome%20of%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/5/1563&amp;title=From%20Pine%20Cones%20to%20Read%20Clouds%3A%20Rescaffolding%20the%20Megagenome%20of%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/5/1563&amp;title=From%20Pine%20Cones%20to%20Read%20Clouds%3A%20Rescaffolding%20the%20Megagenome%20of%20Sugar%20Pine%20%28Pinus%20lambertiana%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484257/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484257" id="node484257--3" data-pisa="ggg;7/5/1563" data-pisa-master="ggg;g3.117.040055" data-apath="/ggg/7/5/1563.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (<em>Pinus lambertiana</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marc W. Crepeau</span>, <span class="highwire-citation-author" data-delta="1">Charles H. Langley</span> and <span class="highwire-citation-author" data-delta="2">Kristian A. Stevens</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1563-1568;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040055 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484257--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marc W. Crepeau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BW.%2BCrepeau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crepeau%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarc%2BW.%2BCrepeau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Charles H. Langley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BH.%2BLangley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langley%20CH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BH.%2BLangley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kristian A. Stevens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Evolution and Ecology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristian%2BA.%2BStevens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stevens%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKristian%2BA.%2BStevens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kastevens@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">kastevens@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1563&amp;rft.epage=1568&amp;rft.atitle=From%20Pine%20Cones%20to%20Read%20Clouds%3A%20Rescaffolding%20the%20Megagenome%20of%20Sugar%20Pine%20%28Pinus%20lambertiana%29&amp;rft.volume=7&amp;rft.issue=5&amp;rft.date=2017-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Crepeau%2C+Marc+W.&amp;rft.au=Langley%2C+Charles+H.&amp;rft.au=Stevens%2C+Kristian+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484257/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484257/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484257/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484257/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484257/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484257/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484257/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484257/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484257/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484257/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484257/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484257/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/5/1563" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/5/1563.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/5/1563.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/7/5/1563.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
