<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/5/1551" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/5/1551.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/5/1551.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/5/1551.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/5/1551.atom" />
<meta name="HW.pisa" content="ggg;7/5/1551" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genotyping-by-Sequencing Facilitates a High-Density Consensus Linkage Map for Aegilops umbellulata, a Wild Relative of Cultivated Wheat" />
<meta name="DC.Identifier" content="10.1534/g3.117.039966" />
<meta name="DC.Date" content="2017-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Edae et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="High-density genetic maps are useful to precisely localize QTL or genes that might be used to improve traits of nutritional and/or economical importance in crops. However, high-density genetic maps are lacking for most wild relatives of crop species, including wheat. Aegilops umbellulata is a wild relative of wheat known for its potential as a source of biotic and abiotic stress resistance genes. In this work, we have developed a framework consensus genetic map using two biparental populations derived from accessions PI 298905, PI 542369, PI 5422375, and PI 554395. The framework map comprised 3009 genotype-by-sequence SNPs with a total map size of 948.72 cM. On average, there were three SNPs per centimorgan for each chromosome. Chromosome 1U was the shortest (66.5 cM), with only 81 SNPs, whereas the remaining chromosomes had between 391 and 591 SNP markers. A total of 2395 unmapped SNPs were added to the linkage maps through a recombination frequency approach, and increased the number of SNPs placed on the consensus map to a total of 5404 markers. Segregation distortion was disproportionally high for chromosome 1U for both populations used to construct component linkage maps, and thus segregation distortion could be one of the probable reasons for the exceptionally reduced linkage size for chromosome 1U. From comparative analysis, Ae . umbellulata chromosomes except 4U showed moderate to strong collinearity with corresponding homeologous chromosomes of hexaploid wheat and barley. The present consensus map may serve as a reference map in QTL mapping and validation projects, and also in genome assembly to develop a reference genome sequence for Ae. umbellulata ." />
<meta name="DC.Contributor" content="Erena A. Edae" />
<meta name="DC.Contributor" content="Pablo D. Olivera" />
<meta name="DC.Contributor" content="Yue Jin" />
<meta name="DC.Contributor" content="Matthew N. Rouse" />
<meta name="article:published_time" content="2017-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genotyping-by-Sequencing Facilitates a High-Density Consensus Linkage Map for Aegilops umbellulata, a Wild Relative of Cultivated Wheat" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;High-density genetic maps are useful to precisely localize QTL or genes that might be used to improve traits of nutritional and/or economical importance in crops. However, high-density genetic maps are lacking for most wild relatives of crop species, including wheat. &lt;i&gt;Aegilops umbellulata&lt;/i&gt; is a wild relative of wheat known for its potential as a source of biotic and abiotic stress resistance genes. In this work, we have developed a framework consensus genetic map using two biparental populations derived from accessions PI 298905, PI 542369, PI 5422375, and PI 554395. The framework map comprised 3009 genotype-by-sequence SNPs with a total map size of 948.72&amp;nbsp;cM. On average, there were three SNPs per centimorgan for each chromosome. Chromosome 1U was the shortest (66.5&amp;nbsp;cM), with only 81 SNPs, whereas the remaining chromosomes had between 391 and 591 SNP markers. A total of 2395 unmapped SNPs were added to the linkage maps through a recombination frequency approach, and increased the number of SNPs placed on the consensus map to a total of 5404 markers. Segregation distortion was disproportionally high for chromosome 1U for both populations used to construct component linkage maps, and thus segregation distortion could be one of the probable reasons for the exceptionally reduced linkage size for chromosome 1U. From comparative analysis, &lt;i&gt;Ae&lt;/i&gt;. &lt;i&gt;umbellulata&lt;/i&gt; chromosomes except 4U showed moderate to strong collinearity with corresponding homeologous chromosomes of hexaploid wheat and barley. The present consensus map may serve as a reference map in QTL mapping and validation projects, and also in genome assembly to develop a reference genome sequence for &lt;i&gt;Ae. umbellulata&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/05/01" />
<meta name="citation_mjid" content="ggg;7/5/1551" />
<meta name="citation_id" content="7/5/1551" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/5/1551" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/5/1551.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/5/1551.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/5/1551.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.039966" />
<meta name="citation_pmid" content="28364036" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1551" />
<meta name="citation_lastpage" content="1561" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Erena A. Edae" />
<meta name="citation_author_institution" content="United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Cereal Disease Laboratory, St. Paul, Minnesota 55108" />
<meta name="citation_author_email" content="erena.edae@ars.usda.gov" />
<meta name="citation_author_email" content="matthew.rouse@ars.usda.gov" />
<meta name="citation_author" content="Pablo D. Olivera" />
<meta name="citation_author_institution" content="Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author" content="Yue Jin" />
<meta name="citation_author_institution" content="United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Cereal Disease Laboratory, St. Paul, Minnesota 55108" />
<meta name="citation_author_institution" content="Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author" content="Matthew N. Rouse" />
<meta name="citation_author_institution" content="United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Cereal Disease Laboratory, St. Paul, Minnesota 55108" />
<meta name="citation_author_institution" content="Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author_email" content="erena.edae@ars.usda.gov" />
<meta name="citation_author_email" content="matthew.rouse@ars.usda.gov" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=K. V. Alheit;citation_author=J. C. Reif;citation_author=H. P. Maurer;citation_author=V. Hahn;citation_author=E. A. Weissmann;citation_title=Detection of segregation distortion loci in triticale (x Triticosecale Wittmack) based on a high-density DArT marker consensus genetic linkage map.;citation_pages=380-380;citation_volume=12;citation_year=2011;citation_pmid=21798064;citation_doi=10.1186/1471-2164-12-380" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=T. W. M. Balsalobre;citation_author=G. D. S. Pereira;citation_author=G. R. A. Margarido;citation_author=R. Gazaffi;citation_author=F. Z. Barreto;citation_title=GBS-based single dosage markers for linkage and QTL mapping allow gene mining for yield-related traits in sugarcane.;citation_pages=72;citation_volume=18;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. Belanger;citation_author=I. Clermont;citation_author=P. Esteves;citation_author=F. Belzile;citation_title=Extent and overlap of segregation distortion regions in 12 barley crosses determined via a pool-GBS approach.;citation_pages=1393-1404;citation_volume=129;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=C. Bodenes;citation_author=E. Chancerel;citation_author=F. Ehrenmann;citation_author=A. Kremer;citation_author=C. Plomion;citation_title=High-density linkage mapping and distribution of segregation distortion regions in the oak genome.;citation_volume=23;citation_year=2016;citation_issue=2;citation_pmid=27013549;citation_doi=10.1093/dnares/dsw001" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. W. Doerge;citation_title=Mapping and analysis of quantitative trait loci in experimental populations.;citation_pages=43-52;citation_volume=3;citation_year=2002;citation_issue=1;citation_pmid=11823790;citation_doi=10.1038/nrg703" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. M. Devos;citation_author=M. D. Atkinson;citation_author=C. N. Chinoy;citation_author=R. L. Harcourt;citation_author=R. M. D. Koebner;citation_title=Chromosomal rearrangements in the rye genome relative to that of wheat.;citation_pages=673-680;citation_volume=85;citation_year=1993;citation_pmid=24196035;citation_doi=10.1007/BF00225004" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. M. Devos;citation_author=M. D. Atkinson;citation_author=C. N. Chinoy;citation_author=H. A. Francis;citation_author=R. L. Harcourt;citation_title=Chromosomal rearrangements in the rye genome relative to that of wheat.;citation_pages=673-680;citation_volume=85;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=K. M. Devos;citation_author=M. D. Gale;citation_title=Genome Relationships: The Grass Model in Current Research;citation_pages=637-646;citation_volume=12;citation_year=2000;citation_issue=5;citation_pmid=10810140;citation_doi=10.1105/tpc.12.5.637" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. E. Edae;citation_author=P. D. Olivera;citation_author=Y. Jin;citation_author=J. A. Poland;citation_author=M. N. Rouse;citation_title=Genotype-by-sequencing facilitates genetic mapping of a stem rust resistance locus in Aegilops umbellulata, a wild relative of cultivated wheat.;citation_pages=1039;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=C. J. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Endelman;citation_author=C. Plomion;citation_title=LPmerge: an R package for merging genetic maps by linear programming.;citation_volume=30;citation_year=2014;citation_pmid=24532720;citation_doi=10.1093/bioinformatics/btu091" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. D. Faris;citation_author=B. Laddomada;citation_author=B. S. Gill;citation_title=Molecular Mapping of Segregation Distortion Loci in Aegilops tauschii;citation_pages=319-327;citation_volume=149;citation_year=1998;citation_issue=1;citation_pmid=9584106" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=B. Friebe;citation_author=J. Jiang;citation_author=R. P. Raup;citation_author=R. A. McIntosh;citation_author=B. S. Gill;citation_title=Characterization of wheat alien translocations conferring resistance to diseases and pests: current status.;citation_pages=59-87;citation_volume=71;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=B. S. Gill;citation_author=H. C. Sharma;citation_author=W. J. Raup;citation_author=L. E. Browder;citation_author=J. Hatchet;citation_title=Evaluation of Aegilops species for resistance to wheat powdery mildew, wheat leaf rust and Hessian fly and green bug.;citation_pages=314-316;citation_volume=69;citation_year=1985" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=W. Gong;citation_author=G. Li;citation_author=J. Zhou;citation_author=G. Li;citation_author=C. Liu;citation_title=Cytogenetic and molecular markers in a wheat background.;citation_pages=489-497;citation_volume=57;citation_year=2014" />
<meta name="citation_reference" content="citation_title=Bulg. J. Agric. Sci.;citation_author=B. Hadzhiivanova;citation_author=V. Bozhanova;citation_author=D. Dechev;citation_title=Interspecific hybridization between durum wheat and Ae. umbellulata(Zhuk).;citation_pages=713-721;citation_volume=18;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=R. Hajjar;citation_author=T. Hodgkin;citation_title=The use of wild relatives in crop improvement: a survey of developments over the last 20 years.;citation_pages=1-13;citation_volume=156;citation_year=2007;citation_doi=10.1007/s10681-007-9363-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Huang;citation_author=S. Brooks;citation_author=W. Li;citation_author=J. Fellers;citation_author=H. Trick;citation_title=Map-Based Cloning of Leaf Rust Resistance Gene Lr21 From the Large and Polyploid Genome of Bread Wheat;citation_pages=655-664;citation_volume=164;citation_year=2003;citation_issue=2;citation_pmid=12807786" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Huang;citation_author=F. Qi;citation_author=Q. Qian;citation_author=Z. Qiang;citation_author=W. Lu;citation_title=High-throughput genotyping by whole-genome resequencing.;citation_volume=19;citation_year=2009;citation_pmid=19420380;citation_doi=10.1101/gr.089516.108" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. Huang;citation_author=J. A. Poland;citation_author=C. P. Wight;citation_author=E. W. Jackson;citation_author=N. A. Tinker;citation_title=Using Genotype-by-sequencing (GBS) for genomic discovery in cultivated oat.;citation_pages=e102448;citation_volume=9;citation_year=2014;citation_issue=7;citation_pmid=25047601;citation_doi=10.1371/journal.pone.0102448" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=H. Iwata;citation_author=S. Ninomiya;citation_title=AntMap: constructing genetic linkage maps using an ant colony optimization algorithm.;citation_pages=371-377;citation_volume=56;citation_year=2006;citation_doi=10.1270/jsbbs.56.371" />
<meta name="citation_reference" content="citation_title=Science;citation_title=A chromosome-based draft sequence of the hexaploid bread wheat (Triticum aestivum) genome.;citation_pages=286;citation_volume=345;citation_year=2014;citation_issue=6194" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=J. Jiang;citation_author=B. Friebe;citation_author=B. S. Gill;citation_title=Recent advances in alien gene transfer in wheat.;citation_pages=199-212;citation_volume=73;citation_year=1994;citation_doi=10.1007/BF00036700" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. A. Khan;citation_author=Y. Han;citation_author=Y. F. Zhao;citation_author=M. Troggio;citation_author=S. S. Korban;citation_title=A Multi-Population Consensus Genetic Map Reveals Inconsistent Marker Order among Maps Likely Attributed to Structural Variations in the Apple Genome.;citation_pages=e47864-e47864;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23144832;citation_doi=10.1371/journal.pone.0047864" />
<meta name="citation_reference" content="citation_journal_title=Gen Res Camb;citation_author=G. Kimber;citation_title=The addition of the chromosomes of Ae. umbellulata to Triticum aestivum var. Chinese Spring.;citation_pages=111-114;citation_volume=9;citation_year=1967;citation_doi=10.1017/S0016672300010351" />
<meta name="citation_reference" content="citation_title=Wild Crop Relatives: Genomic and breeding resources;citation_title=Aegilops;citation_author=B. Kilian;citation_author=K. Mammen;citation_author=E. Millet;citation_author=R. Sharma;citation_author=A. Graner;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=R. D. Koebner;citation_author=K. W. Shepherd;citation_title=Allosyndetic recombination between a chromosome of Ae. umbellulata and wheat chromosomes.;citation_pages=33-45;citation_volume=59;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=T. J. Y. Kono;citation_author=K. Seth;citation_author=J. A. Poland;citation_author=P. L. Morrel;citation_title=SNPMeta: SNP annotation and SNP metadata collection without a reference genome.;citation_pages=419-425;citation_volume=14;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=V. Korzun;citation_author=S. Malyshev;citation_author=A. V. Voylokov;citation_author=A. Börner;citation_title=A genetic map of rye (Secale cereale L.) combining RFLP, isozyme, protein, microsatellite and gene loci.;citation_pages=709-717;citation_volume=102;citation_year=2001;citation_doi=10.1007/s001220051701" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=A. Kujur;citation_author=H. D. Upadhyaya;citation_author=D. Bajaj;citation_author=C. L. Gowda;citation_author=S. Sharma;citation_title=Identification of candidate genes and natural allelic variants for QTLs governing plant height in chickpea.;citation_pages=27968;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=S. Kumar;citation_author=B. S. Gill;citation_author=J. D. Faris;citation_title=Identification and characterization of segregation distortion loci along chromosome 5B in tetraploid wheat.;citation_pages=187-196;citation_volume=278;citation_year=2007;citation_issue=2;citation_pmid=17520291;citation_doi=10.1007/s00438-007-0248-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Biomed Res. Int.;citation_author=C. Li;citation_author=G. Bai;citation_author=S. Chao;citation_author=Z. Wang;citation_title=A high-density SNP and SSR consensus map reveals segregation distortion regions in wheat.;citation_pages=830618;citation_volume=2015;citation_year=2015;citation_doi=10.1155/2015/830618" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=H. Li;citation_author=A. Kilian;citation_author=M. Zhou;citation_author=P. Wenzi;citation_author=E. Huttner;citation_title=Construction of a high-density composite map and comparative mapping of segregation distortion regions in barley.;citation_pages=319-331;citation_volume=284;citation_year=2010;citation_issue=5;citation_pmid=20803217;citation_doi=10.1007/s00438-010-0570-3" />
<meta name="citation_reference" content="citation_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_issue=1;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=F. Lu;citation_author=M. C. Romay;citation_author=J. C. Glaubitz;citation_author=P. J. Bradbury;citation_author=R. J. Elshire;citation_title=High-resolution genetic mapping of maize pan-genome sequence anchors.;citation_pages=6914;citation_volume=6;citation_year=2015;citation_pmid=25881062;citation_doi=10.1038/ncomms7914" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=A. J. Lukaszewski;citation_author=K. Rybka;citation_author=V. Korzun;citation_author=S. V. Malyshev;citation_author=B. Lapinski;citation_title=Genetic and physical mapping of homoeologous recombination points involving wheat chromosome 2B and rye chromosome 2R.;citation_pages=36-45;citation_volume=47;citation_year=2004;citation_issue=1;citation_pmid=15060600" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=E. S. Mace;citation_author=J.-F. Rami;citation_author=S. Bouchet;citation_author=P. Klein;citation_author=R. R. Klein;citation_title=A consensus genetic map of sorghum that integrates multiple component maps and high-throughput Diversity Array Technology (DArT) markers.;citation_pages=13-13;citation_volume=9;citation_year=2009;citation_pmid=19171067;citation_doi=10.1186/1471-2229-9-13" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=M. Mascher;citation_author=G. Muehlbauer;citation_author=D. Rokhsar;citation_author=J. Chapman;citation_author=J. Schmutz;citation_title=Anchoring and ordering NGS contig assemblies by population sequencing (POPSEQ).;citation_pages=718-727;citation_volume=76;citation_year=2013;citation_pmid=23998490;citation_doi=10.1111/tpj.12319" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Maccaferri;citation_author=M. A. Cane;citation_author=M. C. Sanguineti;citation_author=S. Salvi;citation_author=M. C. Colalong;citation_title=A consensus framework map of durum wheat (Triticum durum Desf.) suitable for linkage disequilibrium analysis and genome-wide association mapping.;citation_pages=873;citation_volume=15;citation_year=2014;citation_pmid=25293821;citation_doi=10.1186/1471-2164-15-873" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=M. Maccaferri;citation_author=A. Ricci;citation_author=S. Salvi;citation_author=S. G. Milner;citation_author=E. Noli;citation_title=A high-density, SNP-based consensus map of tetraploid wheat as a bridge to integrate durum and bread wheat genomics and breeding.;citation_pages=648-663;citation_volume=13;citation_year=2015;citation_pmid=25424506;citation_doi=10.1111/pbi.12288" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Mayer;citation_author=R. Waugh;citation_author=J. Wang;citation_author=A. Schulman;citation_author=P. Langridge;citation_title=A physical, genetic and functional sequence assembly of the barley genome.;citation_pages=711-716;citation_volume=491;citation_year=2012;citation_issue=7426;citation_pmid=23075845" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. R. McCough;citation_author=M. H. Wright;citation_author=C. W. Tung;citation_author=L. G. Maron;citation_author=K. L. McNally;citation_title=Open access resources for genome-wide association mapping in rice.;citation_pages=10532;citation_volume=7;citation_year=2016;citation_pmid=26842267;citation_doi=10.1038/ncomms10532" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=P. Milczarski;citation_author=H. Bolibok-Bragoszewska;citation_author=B. Myskow;citation_author=S. Stojalowski;citation_author=K. Heller-Uszynska;citation_title=A high density consensus map of rye (Secale cereale L.) based on DArT markers.;citation_pages=e28495-e28495;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22163026;citation_doi=10.1371/journal.pone.0028495" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=I. Molnar;citation_author=H. Imkova;citation_author=M. Leverington-Waite;citation_author=R. Goram;citation_author=A. Cseh;citation_title=Syntenic relationships between the U and M genomes of Aegilops, wheat and the model species Brachypodium and rice as revealed by COS markers.;citation_pages=e70844;citation_volume=8;citation_year=2013;citation_issue=8;citation_pmid=23940651;citation_doi=10.1371/journal.pone.0070844" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=I. Molnar;citation_author=J. Vrana;citation_author=V. Buresova;citation_author=P. Capal;citation_author=A. Farkas;citation_author=E. Dark;citation_title=Dissecting the U, M, S and C genomes of wild relatives of bread wheat (Aegilops spp.) into chromosomes and exploring their synteny with wheat.;citation_pages=452-467;citation_volume=88;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=M. Ozgen;citation_author=M. Yildiz;citation_author=H. Ulukan;citation_author=N. Koyuncu;citation_title=Association of gliadin protein pattern and rust resistance derived from Aegilops ubmellulata Zhuk. In winter Triticum durum Desf.;citation_pages=287-290;citation_volume=54;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Poland;citation_author=P. Brown;citation_author=M. Sorrells;citation_author=J. L. Jannink;citation_title=Development of high density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_author=P. Pootakham;citation_author=N. Jomchai;citation_author=P. Ruang-areerate;citation_author=J. R. Shearman;citation_author=C. Sonthirod;citation_title=Genome-wide SNP discovery and identification of QTL associated with agronomic traits in oil palm using genotyping-by-sequencing (GBS).;citation_pages=288-295;citation_volume=105;citation_year=2015" />
<meta name="citation_reference" content="citation_title=Genes from the Wild: Using Wild Genetic Resources for Food and Raw materials;citation_author=C. Prescott-Allen;citation_author=R. Prescott-Allen;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Cereal Res. Comm.;citation_author=S. M. Reader;citation_author=T. E. Miller;citation_title=The simultaneous substitution of two pairs of chromosomes from two alien species in Triticum aestivum cv. Chinese Spring.;citation_pages=39-42;citation_volume=15;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=C. Saintenac;citation_author=W. Zhang;citation_author=A. Salcedo;citation_author=M. Rouse;citation_author=H. Trick;citation_title=Identification of wheat gene Sr35 that confers resistance to Ug99 stem rust race group.;citation_volume=341;citation_year=2013;citation_pmid=23811222;citation_doi=10.1126/science.1239022" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=A. Schneider;citation_author=I. Molnar;citation_author=M. Molnar-Lang;citation_title=Utilization of Aegilops (goatgrass) species to widen the genetic diversity of cultivated wheat.;citation_pages=1-19;citation_volume=163;citation_year=2008;citation_doi=10.1007/s10681-007-9624-y" />
<meta name="citation_reference" content="citation_journal_title=Brookhaven Symp. Biol.;citation_author=E. R. Sears;citation_title=The transfer of leaf rust resistance from Aegilops umbellulata to wheat.;citation_pages=1-21;citation_volume=9;citation_year=1956" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=P. Senft;citation_author=G. Wricke;citation_title=An extended genetic map of rye (Secale cereale L.).;citation_pages=508-510;citation_volume=115;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=N. Sharopova;citation_author=M. D. McMullen;citation_author=L. Schultz;citation_author=S. Schroeder;citation_author=H. Sanchez-Villeda;citation_title=Development and mapping of SSR markers for maize.;citation_pages=463-481;citation_volume=48;citation_year=2002;citation_issue=5-6;citation_pmid=12004892;citation_doi=10.1023/A:1014868625533" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. J. Somers;citation_author=P. Isaac;citation_author=K. Edwards;citation_title=A high-density microsatellite consensus map for bread wheat (Triticum aestivum L.).;citation_pages=1105-1114;citation_volume=109;citation_year=2004;citation_issue=6;citation_pmid=15490101;citation_doi=10.1007/s00122-004-1740-7" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Tanksley;citation_author=S. McCouch;citation_title=Seed Banks and Molecular Maps: Unlocking Genetic Potential from the Wild;citation_pages=1063-1066;citation_volume=277;citation_year=1997;citation_issue=5329;citation_pmid=9262467;citation_doi=10.1126/science.277.5329.1063" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=D. Tayler;citation_author=P. K. Ingvarsson;citation_title=Common features of segregation distortion in plants and animals.;citation_pages=27-35;citation_volume=117;citation_year=2003;citation_issue=1;citation_pmid=12656570;citation_doi=10.1023/A:1022308414864" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Van Os;citation_author=S. Andrzejewski;citation_author=E. Bakker;citation_author=I. Barrena;citation_author=G. J. Bryan;citation_title=Construction of a 10,000-marker ultradense genetic recombination map of potato: providing a framework for accelerated gene isolation and a genome-wide physical map.;citation_volume=173;citation_year=2006;citation_pmid=16582432;citation_doi=10.1534/genetics.106.055871" />
<meta name="citation_reference" content="citation_title=Wild wheats: A Monograph of Aegilops L. and Amblyopyrum (Jaub. &amp; Spach) Eig (Poaceae): A Revision Of All Taxa Closely Related to Wheat, Excluding Wild Triticum Species, With Notes on Other Genera in the Tribe Triticcae, Especially Triticum. Wageningen Agricultural University, Wageningen, The Netherlands;citation_author=M. W. Van Slageren;citation_pages=94-97;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=S. Wang;citation_author=D. Wong;citation_author=K. Forrest;citation_author=A. Allen;citation_author=S. Chao;citation_title=Characterization of polyploid wheat genomic diversity using a high-density 90 000 single nucleotide polymorphism array.;citation_pages=787-796;citation_volume=12;citation_year=2014;citation_issue=6;citation_pmid=24646323;citation_doi=10.1111/pbi.12183" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=J. W. Westbrook;citation_author=C. E. Vikram;citation_author=W. Le-Shin;citation_author=C. Srikar;citation_author=N. G. Leandro;citation_title=A consensus genetic map for Pinus taeda and Pinus elliottii and extent of linkage disequilibrium in two genotype-phenotype discovery populations of Pinus taeda.;citation_pages=1685-1694;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Wu;citation_author=P. R. Bhat;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.;citation_pages=e1000212-e1000212;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18846212;citation_doi=10.1371/journal.pgen.1000212" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=B. Wulff;citation_author=M. Moscou;citation_title=Strategies for transferring resistance into wheat: from wide crosses to GM cassettes.;citation_pages=1-11;citation_volume=5;citation_year=2014;citation_doi=10.3389/fpls.2014.00692" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. Zhang;citation_author=J. Jia;citation_author=M. Gale;citation_author=K. Devos;citation_title=Relationships between the chromosomes of Aegilops umbellulata and wheat.;citation_pages=69-75;citation_volume=96;citation_year=1998;citation_doi=10.1007/s001220050710" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. Zaharieva;citation_author=P. Monneveux;citation_title=Spontaneous Hybridization between Bread Wheat (Triticum aestivum L.) and Its Wild Relatives in Europe;citation_pages=512-527;citation_volume=46;citation_year=2006;citation_issue=2;citation_doi=10.2135/cropsci2005.0023" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genotyping-by-Sequencing Facilitates a High-Density Consensus Linkage Map for Aegilops umbellulata, a Wild Relative of Cultivated Wheat" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="twitter:description" content="High-density genetic maps are useful to precisely localize QTL or genes that might be used to improve traits of nutritional and/or economical importance in crops. However, high-density genetic maps are lacking for most wild relatives of crop species, including wheat. Aegilops umbellulata is a wild relative of wheat known for its potential as a source of biotic and abiotic stress resistance genes. In this work, we have developed a framework consensus genetic map using two biparental populations derived from accessions PI 298905, PI 542369, PI 5422375, and PI 554395. The framework map comprised 3009 genotype-by-sequence SNPs with a total map size of 948.72 cM. On average, there were three SNPs per centimorgan for each chromosome. Chromosome 1U was the shortest (66.5 cM), with only 81 SNPs, whereas the remaining chromosomes had between 391 and 591 SNP markers. A total of 2395 unmapped SNPs were added to the linkage maps through a recombination frequency approach, and increased the number of SNPs placed on the consensus map to a total of 5404 markers. Segregation distortion was disproportionally high for chromosome 1U for both populations used to construct component linkage maps, and thus segregation distortion could be one of the probable reasons for the exceptionally reduced linkage size for chromosome 1U. From comparative analysis, Ae . umbellulata chromosomes except 4U showed moderate to strong collinearity with corresponding homeologous chromosomes of hexaploid wheat and barley. The present consensus map may serve as a reference map in QTL mapping and validation projects, and also in genome assembly to develop a reference genome sequence for Ae. umbellulata ." />
<meta name="og-title" property="og:title" content="Genotyping-by-Sequencing Facilitates a High-Density Consensus Linkage Map for Aegilops umbellulata, a Wild Relative of Cultivated Wheat" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/5/1551" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="High-density genetic maps are useful to precisely localize QTL or genes that might be used to improve traits of nutritional and/or economical importance in crops. However, high-density genetic maps are lacking for most wild relatives of crop species, including wheat. Aegilops umbellulata is a wild relative of wheat known for its potential as a source of biotic and abiotic stress resistance genes. In this work, we have developed a framework consensus genetic map using two biparental populations derived from accessions PI 298905, PI 542369, PI 5422375, and PI 554395. The framework map comprised 3009 genotype-by-sequence SNPs with a total map size of 948.72 cM. On average, there were three SNPs per centimorgan for each chromosome. Chromosome 1U was the shortest (66.5 cM), with only 81 SNPs, whereas the remaining chromosomes had between 391 and 591 SNP markers. A total of 2395 unmapped SNPs were added to the linkage maps through a recombination frequency approach, and increased the number of SNPs placed on the consensus map to a total of 5404 markers. Segregation distortion was disproportionally high for chromosome 1U for both populations used to construct component linkage maps, and thus segregation distortion could be one of the probable reasons for the exceptionally reduced linkage size for chromosome 1U. From comparative analysis, Ae . umbellulata chromosomes except 4U showed moderate to strong collinearity with corresponding homeologous chromosomes of hexaploid wheat and barley. The present consensus map may serve as a reference map in QTL mapping and validation projects, and also in genome assembly to develop a reference genome sequence for Ae. umbellulata ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
    <title>Genotyping-by-Sequencing Facilitates a High-Density Consensus Linkage Map for Aegilops umbellulata, a Wild Relative of Cultivated Wheat | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__6uPNahEPJnOkpurvaRxri2qBNHoapqvM8y1Wtdb7u_Q__sE1YcwwFB0K8aMYGFpNWEgt3sH5t9Ld9QAF34gSvqPY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__EjT-6YMPvgLl8LSJ78FRw-tAFtIqvvC0dKPDt3xx2Ec__naWIXWhSBwaYF2DqFTeJJxk8gTTEkdfCwGZ2o8_HgIw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__WoUUhFHu9IZRJChlxfU_hFiUTBhKim997BgtM6rQv_0__WtsG9gDKkyL0Zcl619hpmtR4jbV5vkG5UnoAmrjf3jM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8QInhxP5W_phykIfGiQtpfmpoJ9hFOjgawJZPmtyTuI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484262","apath":"\/ggg\/7\/5\/1551.atom","pisa":"ggg;7\/5\/1551","ac":{"\/ggg\/7\/5\/1551.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/5\/1551.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1551"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1551"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/5\/1551":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zONR8EW9r6MyBfKNztPUYIr3iwdAS1BDx3wuJ7L+W3T84cd0RVsYbnI6UWpID4ubODF6hlle\\\\\\\/JWhD2ejlBqUh7aqw7an6Y0wU\\\\\\\/LMD4RoDB6aJhKFq3pv7Izv\\\\\\\/ukuHnUTa7yOTodr0Lwr8kngTK3wtQKxRT6\\\\\\\/OrDNnKP2QKxqKXAmQo0HlQDNAT6wekkwOpThLdh4DeaTytnMb7FpGSbGtHTBC4Kr44lgOLbgqJntro\\\\\\\/zB8cu\\\\\\\/67rwp+82H9\\\\\\\/9dawqi1nJL4Zls\\\\\\\/Jj\\\\\\\/sL8fGIEwHncrBjT9\\\\\\\/+cT61Wicu\\\\\\\/RumwWgoIQsoMD9M01OYuB\\\\\\\/Hgv+6Imiu\\\\\\\/pUIrJ1JvSpZICThs+4+OWRCto\\\\\\\/7hvEoqvNPLUJBt6csq\\\\\\\/LvhSUgGiNltgD42mb6ZqcZkf2hT3HYLiOUqxniC3nWV8pZl69zOttJ5M44g\\\\\\\/DJLREPa605ODlAKazcMMjHGGqs7D4Rtq9Kji+OxJzeEN8773xcVFUmRAVYj23I21G+SL9qfH0F0IKJwSguN7dPsrP74hm4HOYD+HxFb6Qe9Dv23u2OndAgEM+\\\\\\\/t5PvCdboHIF23UzIm7KrkFaJ2j9HAW9INFblnryamA\\\\\\\/NOdmkdI\\\\\\\/WgXMeY2o7KNRH+fDC+SaGwN0F3bx+LYHqNqTOANSDxtlJfNKCIPLgLwleNJVJwpNzJvAgXUnldizKR7J8oxw\\\\\\\/PTTDjP1fJFHrBEDcl0h\\\\\\\/0N\\\\\\\/dgdSS8YdF0qTPD2o\\\\\\\/OlG7B07K4ufizdP2jmeaOi4fagf4IaYlJzOWDNbSXBeNeicqxWeveFvaKd06EIG9r5kVntya+GnFt9HliaZmlhvJQ21qQhEHsPubvuDeeZBI9QUcnVjFD+MwYtVtXRoRp59e+IyaC1Ts7cG05Jy4ZL9QteZA3z2KyknQFLUkn1LK2k15DcrpxnQcUhe9cqlqBFReANodzlKFAyllUv0FXH+T3R7BAhhMo2UX7mG5zCVgsVReknx0uZSQKpXHTUu\\\\\\\/Sx8UFS7PfhD6bN0WhRmfUap1ycX8WfaWnfc6\\\\\\\/t+EblF1ft5WE9BPgSi0l7c3ROYGhUemoPp5g0MQZ1aoD9HTzjJ+QvpcRT39amOdAyq+50kVKHyQN\\\\\\\/fhamBYorb6rF7TX6+SXCR1gjXc72fwiYn9LBIV9lPGHqyz6bJELhyo2BWAplw2v0mM53nt\\\\\\\/tnJpaowKg9kQZsO7CziFSipIueFzqe9hfy40EjXqyzk+vra9QdTvh1WA+pguwXVGkSM9mW5S1snF0eJ7LeiMoZnjnj5NwBJQF8Fb93OUc2W\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZMGHDQ1W4riYZ8wS6DXjJQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002231100fedc4851562b2e869f1ceb59efa5873246c0894a8a5c4a72da2ac6ce91e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022P\\\\\\\/gMJjv\\\\\\\/R4WpQrK3SJtkTA\\\\\\\/Hn1Of+652RnC8owFLkaaKlh0e\\\\\\\/JAAH\\\\\\\/GqUgPXlVPoIqM+0PsIQ2hRdLWWStHwzmbazQ\\\\\\\/WMFoC\\\\\\\/9fSst7uYTN6NvbW+cDHP8spkWT2hPKsD4CnfK0fzQeWmG92NFuXSvpz94b6lZE6iGeNoz98vkU7aNbzKxRpEH+4QvLyaJCaB8FybQh9jrCQmHldVcRp4N5+9gEPGmYXDjvGI9polE+hdveqMaVKUBBQiu0CAse\\\\\\\/lCBExaaYCIm4WO29Ol16zPm0smokxV1v2w3LoOh4H3Npb7QZK19HP1DU6j3fypMjsE+ltzyw2F1sWB5tDxIzxIOt7\\\\\\\/dZXwamhniNzgOZ4PUWNcRpvVMWKIzYsvvWTsKRA4JkDrE4H53L+QnYgnTbzE1ZS3zALRblIwvqnJQ+3P9LhirQ1G4rJusOGK0o0xFPVB4J9VvVl\\\\\\\/Q3P\\\\\\\/he+cn2XHVEfDK3GzOERax0PEjAtvabhhWNl7thqDlYDoIm8NHu+2UFtocYbHu0SSPWIAZ\\\\\\\/gpbBgtyWb+jx53R1HZDaucEPCh6c5TD8gC+Wgnozg3R7EQ8aZgoBvxPHG4IER+68ZWVGGP7ynbZa8LaWNPrhsJojcxZDZ59hOjgTVga7dM12YW3AFfH4Tv1uz+raxz4pwsgWgxJFBLFTMgh+XxV49C0FfWTE+gA241Iw+4y6cB8NOgYGtizYH7aavo7KOMV808OZYc\\\\\\\/YRsiVEiNjkVyE871Z7IenGayYOfvCCuPCPMYKbZbei6ocY7HFbmHxAtVJO+bIj8TWp6nPvQBFpmUM5g+kIv9ckfzFbJR1nY2RPANTIgJf6yr6vOMFYZYt1tpHqKfEdECtoHKjzCJfPeXETbFKeJ03gSa49dRX9C0+rCDjJecLkkoQ8xYMIyqMQm0DJQGSsQ8HfsVQNbrDlc6HkGBJ49FvlvQPLn3OLlArSVXv\\\\\\\/IceelnnzLI48+CIvLyU67ym8VHgMMHkD6BctTZ0bxqewmoOXZ\\\\\\\/dzF\\\\\\\/jOtCtw6lYTHhzGhLsdvKrLSCPuAeHfm3dU3MmCDLUFhW8X0YzOit7OZLQ0NfXKavJzjLx1+mshVkK85X6zaB5woumOrUQsJGhfg9LJ+Y0DjoIv8sE9SpC\\\\\\\/AZ+PadPduWTk6Y0lRhPnNWdB7FNEcvWJkJGSDBccgKl8uiqOjwBZVzcW4UwhP7AjhUB5RJO+hQO3N4er8dMFXH3BloM76GLDtJHTWb+EUhlfSCUDTsaPn\\\\\\\/Tqf45HvAm46rrrecpurTu8HTgz\\\\\\\/vwMHkUF4RVPYM8uES4CmzpYYBftPmp7NAnpUOBrkTMxnQN8VYaPoPmxz3mUHR9s\\\\\\\/rhyE5iRtFmTr0eCksIPijb+wNUAslgwW2SR1XnxxrVBv1D\\\\\\\/uMuaZ6ZPbHOJYI6e0wniOtwwCYw2d\\\\\\\/v4nC7v+gfBnsShP+pAhwoA6bZw4\\\\\\\/peSLhoEOvGSlJm9+YdNy+\\\\\\\/Ox0YRCQ0Frq22aBG9q1LSRhznMsMd4CvG9qzzl7QzBOgVVDyQ0UjMCEbxAG1KHWalZftDFEdmNmP+DIFnkrQOU0VACRSSfBYvvvgHqR611uc3hFwm+H4NIBQ0HVkftIXAoMKleooqpCTy0H\\\\\\\/7w\\\\\\\/61406f6Xnfn1zRp34jpqxNs4HRfMqfAwhCGSxo3rJ+eJV9VhH6to3Iv5UdFRaZ2dP0HuKWcfaizhvU2PJwZ5\\\\\\\/dCPENpW6gN6Xjgwe7rSO89nd+BgtKAvB0mcfdCwCO8HHXnGoJFLPNERu7PtnqXNciwssruD90Et9Jp22L4t78B65X1yCIarmVoghw6OtL0+zgE2p4Bt4wu5Wa7m28T7KTgodB1JD+DQ5j9kYQIXuem8iuxaYrj9c0unC5561ofIPEc8WF\\\\\\\/CTbZrW\\\\\\\/AxsGdIMStfyCrllhewAF\\\\\\\/Zm+3W26LP2ueHSaQ76PF4Z86l2yiC05+yqPfefQTh80XLRFcIp7Nvaci5uCn9TXx+OwceeZ3+QemCkBu3ja0keGfQM\\\\\\\/a9Hb+lOA\\\\\\\/tAMd6X0Q36omJdb9NZPmiwIlq\\\\\\\/HVB3WtETe8QSOdeN1jdpbRLYpmcz49Ms\\\\\\\/QVWYXTrVn2NMkyEplplsRyjt0+j5RGyffBnFT2YBr2oJ1ZSCnEpn2snGLtsMfiJii9+1r0cqcRYmEQfS0vnfXpTsUV8E\\\\\\\/rkikp+6fAiene9QjSq6DbaRAlHB6d959\\\\\\\/z5NbqjrMD1MYbnFgLINid\\\\\\\/CxaaScZbVLFwpeoBXCwqqu9v8bVyp3bBDUpCEu3RSVkrOHEQPuoGXfYRn+W2yjEBfe\\\\\\\/BQ2t2CP0GbwVl8Hnn9w3pNbrin3HnLjmK4ZPcoV9zL+i0ZdvNcnwuCJstTu7DKiXKW4V\\\\\\\/tEnACKDTiUxdsZT2lxjyUR9VrecC\\\\\\\/wUpMfKvxRu8t+4HRdtB4AlGqZY8KLnmICvtcAit14iuAVKmCLQUscP0bdCjanhIQOwqto02JqHlODx\\\\\\\/cI4SQfU+UQQBqL61yRa67D07Bt8T5X2Ibp5Y6zhwmJz3\\\\\\\/VYr4SLVXvS\\\\\\\/f45LRpFZiEUrUcyfKIQ1Is4HzhjBtB+S0LsfAzml2PHbB9FwTacbfsEg1em0cE+QMJS+2J7cGkbI8xTlEXHtILQHl\\\\\\\/hoyOiU2jSD6\\\\\\\/S0DPp2bAp3aN14jxuUgZqJqoH6Yrgq2JN7jgVQk20+vDhoRaEX\\\\\\\/NC36mWrpFQoWjuZi8SmNG5VbufXQA9JmqOLXZAms41lbDzk7NPh4PGLEoCyoOuZIS0JOXtkIHgSIKI6MpFsWJZVmKUjsRJYwPhYm4LGxg3QvDQ8L3hY1hJ38aUsVildWtOW28AO\\\\\\\/oZk\\\\\\\/dFK9s\\\\\\\/x39WLSM61atkx5BHqX8nR3vsuIAl4Qga8Z+OzSSnrvMpMQ8iFlBD1+QwNHHtN2iiTbe9vR36e+JwMV1c7UNF1IDEjoPjGH6hpC5jqECxbJb0FDnfxGNBzRBTSOF+xAmgC4lHoRzMwv\\\\\\\/Awwu5rnVrNYGhNUcLVRMdaIW9Ywsyro4jS1pGLVRdiHblbEbrWvZl2BQd6WyRIQO1oNSZoHW5HMYpxZ++LIIK6x+WrHKdkxnm846pWjbmLSBYqVEJ2cDHdGaAVRzv+6U8jMGCaV96xFBLXTLSThWkhPUb8mLdiBlDXKHMfAD148UmkT+y79HenW45sc5Gzdiy+kHuU1nNowDNtzvxM2eJ57Fya2AVSeuRv385lSUCSDgFnSiUYrTfJL6UOvmK\\\\\\\/5ua1gWKyC4sNiSpOPeY3sTgK5I9zINM4wO8hIs5vWIZ6ZvyjaHlP5+paSyYmYw2etPjhEjo6SVG3AaIUTmj8GUE54KRT3v73X4MqewsNQpH97FZ+d+NqVaTtKvSg0JQ1SV6uxwVRrBIO16e5Hcq428ly8DcUh4wUdNS9yTpDQTULBW6OYmCOzOWl69bJJtSv2iqBABsuncD249d0gjNG5W34ziD0ZugiCcrWozO7dI0iuUHrU1OFj\\\\\\\/hbZSSTeVzh5HdSFKzF+uIBs52HnP6gtltA\\\\\\\/jqKjYr8T80c2nCsMo3fNMxpq4OSUV+f29cU1UL0jB2G7PlQATt8wUlxW4STu1Kcuwu8PhglDUq7eu\\\\\\\/P2+AGxuRE5FlNZfz+O4FftIZzxwSlpriut5DOxqBltX0XebvHcSZOiLHnlZegCKBnwkNV+vmsu6aEh5ORajPL2yP2TSgT9J+uEiHy0ZHbckhWzOU4jz9xftAlB5NsU\\\\\\\/4qjWhRa9j0oOp+wSoRcbkcu45kerrq99Qm6p6vAePaU\\\\\\\/KhKkiUXbi0lVIEoa6kvvlLsDte1a\\\\\\\/+lDua04FhvzCl8DAOmKApYeSMwh59o7uyN9mkVW8jbzdDbahPaK5FyWgC8G5ImeXwMyI5jJZvFhlNlxV6SuRSZ1UsPbDSaaDg4OzI\\\\\\\/+32uvI9BUjWEqtYxUVDmp9WjYi7yRb2hA6pgYX7gNE32d+jAPtu+4LWWRmKTli+oWD985WTQbsEUot60VcvIiUzD1xSwd87KOxOTE64q0EC\\\\\\\/SDJf3ieF+KKZUWPZM4zIRpyn+lH01uWZIXr9TYnIiqkPGwoMG1KZvbbr4WSfhN9BVIJuOJCEJtR29wXHf\\\\\\\/04EfuJH\\\\\\\/WGpqUg9DTUp+zSbF3kCENTT3tiESFRFfFtxtIZDbwCbAECvzaI5CtIQ9Zt9bX+rEUyEX+uAg97e4cDkjkkPGV4IBVV92Etf\\\\\\\/S4e8yG0v27KZOLQlapJ5SEJcYI4Rmso8XKO8fu5wqF66qBMnhoSuQBLxivTdFt5YUeqdO7qOtOvShZBM0E+gcYLi4wbwA3Mm6Del2em85rG8P5tmIGCbzbB2r4TrhXgDk1c+91os9f6uptpO8BK5hfQ5PnsvJhTrw7+mYgdfTcz6A1wHWRhdByOHG3Z26+aVJd+3hfRjL2DmLEMMdj75a0wVV2g7Wrcfg6bBDDsYHkQCma2I5v0hUKWTS5HVhGPT2cxE1znCHd5prsu1ncR9shtth+MBlLeO+GhL4PYK9tMNDWrj5IRxj1BS36FIiXHT9s2e8sgSL1CdKaYky8Zdr494AX5rUAf3BE0\\\\\\\/byZfmfDZTIjk+dXTkk0ixO4PynS4IzjCt5XlKN260ootxTICFgguHshg0qN4l4+cMB1ui4tECohUQ8UO48OsSzFRNenL6+Jo\\\\\\\/GO7z2BdRrSnsu9dvuvi8u1ywASdpQcV\\\\\\\/cwTynAYzfb0vywSEoMBgApUeeButjLpTk7WdfMUl8N8I48zs91NaMqyr0+wcxav+vwoMc3najHajRdDkn2dKTElTpBTkvI3H0cgiQC6Px7wR2QQa4oFPXIOQrmoWQU6NPg5c1\\\\\\\/\\\\\\\/VJB\\\\\\\/KR4jp64K5yFAJv6gIRDWxX8EzR\\\\\\\/hjhbbX4g9f0lVvP65tPxl2Xcj01LjVcJeDsR0kJOgWLyWvrheFLSm837TXjjVVc4f\\\\\\\/6EcX6Ej+\\\\\\\/t7T2vOeTvMMW51fb\\\\\\\/DClgcGDwrDwyE4ATlo4\\\\\\\/B13lrJVqANF6jwUpapPgPC7ojVjWvYfSacD1536D8n9MaL6wa\\\\\\\/Hh7bL9rJmZU0o9FJBcGfo10haawNzsqPOtrmsBfDJTr+W1unHcJvDcSEdzyiNh3SnxeKHNxXORUbj9VkqJf7+YyxrOOUX9Tvb94s5iitMB4pmVqUKerc+4fdgH2\\\\\\\/95J9T\\\\\\\/+SIJnMHGu1jp0FrKeyzzkZ1Thf3f+qK42BZLHN+q\\\\\\\/e\\\\\\\/X17CFuX57U8B4s0JH7emvxXIPm\\\\\\\/nLzwoajj9Jyhacu3QZMoz+evcWRB7k89mMPjCytCSUkwtkct1zPa4wxlQc3knJ+sIMgCYuPe6MwODvLGLCiyY68hj7V+qh4+UCPYt5mCO\\\\\\\/qTaqGvJxybouM0\\\\\\\/A5\\\\\\\/rcQoNadP8+pNlMXwrJxizEUvCfZ+bbZEc9fjZbYz\\\\\\\/OuX3ewxw1D1rnvz\\\\\\\/beBGo3XdYkCsWRLdGuTQ1uZdqqPsnNj0PvqOT\\\\\\\/i2U07m9UF3KWmeqeDDfDq63N9\\\\\\\/LvbYWPzrB+jR1x81FcO7tH3BPVge26GV0NnhJ6hcAXHYgfMGAHxvNy303NcJrSY1SnxBy3JFPht2E+q5drLw0bK2MiOibtVECLYpV3CMbxRTBqf471GGQVV8zAlsvnoEzD0h+vES7I+fqnUuyyn7iKNE8D4jYQRhdYu35Psb5JYldbBwHdk9o9Xou\\\\\\\/gzokJiGRxcdmskubtA3KZ0zeYs4eauWZRy3ZCsaTBj+ZiLYQhznzqe0Vh9NraWQNXQGGBKbkEtLJeyl3NxLuUMVrFbM0qR+otZP9VcyPnmJbz+GF003gMiYBQCrvIMTNfwrObVo49k1u7oDF8W9BYnmAZB1D56lRure4uOT94PBuIZAlBLF8hI9h6vGV5vq2ZsmpQHrn8mUYEzPv\\\\\\\/qJI39hhiwKD37D6fAnJ35+CB8vX4AL8beOoGX93kI3D4Q25jlzQAbnPQCr\\\\\\\/yMA1gd4wZCGwmGyuKQhdpQCXZh+61TDOblnZCMo8Zh\\\\\\\/MDdcj3dvGYdqPfwjUeVcuKZRbZx8xeHDWrIt7PGXawloaTogmx8E1I61qBI6PAHI5bZXUHKvKlTavG99opRfA7in\\\\\\\/ASMGFc451Tdz8cPJ0BoKLQ+ybk8GvAPHddiEzMuRcoIi4MUMvVg7pC1wtDuxmlQ2Wijq7yLnGPi4Z+SEtbbYlZ6+Klr+xnpXfBlYwJdjkSge6pax3e+BCfBrgFn7KW7HcHTklSOEl6pE0JSOHrtaquvxv59Vu\\\\\\\/V7HwvB6LW7+6EFsQZQKAc+T0WSkKMovmACi3AYb1l51Bjcn5q5glHz3Zq1WqtWNnsN45vpRu1aSj3QoRTSt4g+0LuthXeN3zMZo72g55x9Qvl48v3qxvLshHzEbuOv7umxCVXQ8x8J\\\\\\\/W6pmrDHwb9xKjkzKtDKgAiM70crVFOumQJacpeQQBXfzdBzVnZc75btui7l5gqT7fbdBHrRMSyOPBuxlG\\\\\\\/b+0HFpNcXcpqGXe\\\\\\\/auy5fOejaS1rpyoBhyAjAAfRUEyMqcuYPSWmDQpCpptlEtCgLPP9BFYHTmF3AvY\\\\\\\/SsMbzwwnAn4H4LCyMb2MWcmtISn7pYDUCWtLbE8O7eCqXzu+W\\\\\\\/lRmuQ6JxkYrxvEHGNhkns6c8HUDpo4HeEn0FPFSI7yLw+kRKCVNFqUFODaq5gn9Lh4+oP3qluwqN6MOQEoGuEDt4tkVYKCPtW3Wi1YeF3IZFCvCGFfSbuupoulA2EwQhSBv83FHRArlYRiWzqREm\\\\\\\/Oz\\\\\\\/QjmhNNcE5DDzZgFxSB2CCBcCrj3yhiyuyFMBGkycwn2XpFpxetDva41sSxp7chw0JfDM4xRiohDKe2s0LWex2f\\\\\\\/TT+UhHLwiVIJoyVcOTyLVZ9gRHzWGe3HHQdeYFyMvIhuwzbA5uAc+Xm7jdQhp+JVzLVU6VQd8QXLmYu7vm3XCuWFztMsvw\\\\\\\/JmFJT1bhlx+Mcf9sQ\\\\\\\/s3hVA2zg7Q7semXnBD\\\\\\\/u+J\\\\\\\/vbDp7ry0cwQHJauAHEL8VrY8YjrIRoVeJGLcXZL0WHJ22GUWt2wvJJkM\\\\\\\/WbiMRTRp\\\\\\\/k7URp+Ib56IE7\\\\\\\/HUDsK\\\\\\\/0XvzAfzrGQZTG2u6I4vO8Oq4LnsMjVj22Wf2Vw4EnQZsgmrlX69RUA+70aJkHjDmayM0Ik1JLo4vQrSu1ohBfypEoA7ddr7R4swUc2jUagnYn5zBf4tAYh\\\\\\\/4RlcdiiNdEOBjZk7uvqc97g5i22EHWj0xnqAkf19adI3PdofBvrWxVLvVatA4UNHp7\\\\\\\/4CBAatYD4BkDIq65U6BRqCRLts5bpA+mUqumSMMX00pvu99NNAIuSi9e22PO\\\\\\\/wzmfafAIZ3\\\\\\\/IJD4jjoNoFCj5DQE2oskZJJVyJDWkjRctLV3BSRr9UNFJyUauItt+e8Jd8nSFzybaohYiFU6nnvBksfZXujHwjHwVgHQEGn6D1t+nKLDp1Hgd3oePcjSZQUP8s\\\\\\\/\\\\\\\/HcyrfIt+h0MVjI97XBCVFA0zlilQQasWlhO7O13MXLwhLzY\\\\\\\/Z7D5vpN4H1a9cMjoI5gsClC1DiYu51X4BAQN\\\\\\\/s2H5IP7xJzopr\\\\\\\/iJMdiS32litUv9rlvtIMjDqpfMgGPzZ4vf+SnCv9\\\\\\\/t5wUk4D5I9BEJEwCe1Tsp\\\\\\\/2+65intjNgNhgGf7w7Pd1kThHOn85KVB+PMucZWYx8JS9gz6\\\\\\\/KH8aw0v1qeHDytCPVfIDgfNDKHYY\\\\\\\/lS6W+ICMS1OXTA29kaEp1s90Ks8FVpzaSzueKgxllBuDdexnl9r8On1CUFxt98F8Fzd89Al+VUvIC5\\\\\\\/A5I2E0EUJdtcNPhnP0RVYX2oWGjI7iVt3HucpjKcgFEA2mq\\\\\\\/C+UmI6hhpblbOx6b8ooFudIYgdnUF4fHwT+9ntChBuqrRIbWKSEcxz34aRQ2OYUFR7DkepVZAcxgua+LHtLftdggxXbhkoeKsqN+3x0gohYDX1RsAcr7JSwusSHlmZ33VPv6CRVgnGnM05FD1F4Ls7cL5OzVIs\\\\\\\/h11mIJKm3aAqy1wW3IlBVQDiy221GctVY26tNVfNwdhaQqr7yiYtJzX0UxjZI\\\\\\\/N6xK4eobdp5PA86OKvUcvNAjCZBVbcqzvGJzkQNiYKshN7ukUK9NUEiV54YlL395phLuJ+KXF9TjGfZ3eiTusA8mjZ1nLMuzkK7ZOOBrgl7bgEcfkRi5kBBUyvOqdzMCsHNnw7Gwo07eWmF9QYA1xjybLnYDN+c0pvrNmnvkbEVYTN5ENQqbzrW1f5HTe6ow8igkA6SrNUf+YxMgmsn29j\\\\\\\/MNUApS+5prQTddE1x0jZMFFJJj3MeS21jTpS+kxlleu+YD66rPMzjum6NuRqRU7KQzBy0BHbRUcQTx51mDn+c\\\\\\\/J5oOeiFmCmr089GYeRkCNglJr5ntnHH64kIuhB0QADpRhG+4MKgXWjZFMcCRr2uehIdyW3z5zH9ekkZBgfdLMt7ze1PzFh2o3pH6709m\\\\\\\/3bR3+JnVAioG2c8jE4A13i+XxAEehFf6jZyGVhZlK7v47NQYo2NTJvezqkTYow1zxQoRjZGlmfoKUU077z0Elkh4msN4ly9jGOs0JMo2irk9JOL0auymTZ2gHy0VN2Uq9V8f6asltoQKgxEmCOX8CMHW0EQsGTNxP+\\\\\\\/jwd6DGEawOpexu4PJk9DC8zd9JAPtJ9nXfyLmKAUP8ExOWCWvcblpSbUeqvjIfmqrJN\\\\\\\/J0xPNjSoJkViMaMs11ocbKRoGMlycKbhPSGsKw3f2AnM+w5g\\\\\\\/ARYJxro7E\\\\\\\/0R6MY0c8xurVPFehe89b\\\\\\\/0hcWSSrvPGxGNAWndSt0JHWrP5xbAMkOHxeCxsSK7UeuWiwZZJEtUybWpPwRxNz2zAG1sDcFxHatOy6Xo2bghsdy5UuPFSYQaDStXE0RquSynV6BQR8X24YTlkBobH4lpsupIw6G6CXrGeoVi+DT9qrV8FtiDK7d6g+Y9Ed4E68\\\\\\\/BZcBxeSrwEjexL9A8mH3P4Zrbeb6YJdxRp9J0Pa+tbTLg50tF\\\\\\\/CV\\\\\\\/xMPd0n5qGV0OvdTaolwUOQTf6G5GsA7LXerD4WzLE6xUIvwax\\\\\\\/Y5MSRqQ0\\\\\\\/7TzCxaKUHZfWtHDOwtkquALka45uWDSFV4nvAjR9lY+WTIfy2EW7jicBN8izjljivRIicfl\\\\\\\/sDbwrqYHnw9DGboOlD3vKsELgXGtIOP\\\\\\\/gk5qvY\\\\\\\/vjh8h+mFHFgZRRQCSdPYFdKeaOBjSLH+rurBRH5E0gxDROLqqoQ28SgMMD+8tWbqmPZDEPw\\\\\\\/BAO+Uq8dAF7K9HlHKuIIOdmxLLZX\\\\\\\/cm+315VgXrmIfkudzrdPJDb4IIXCGdLgdGM3ruqkqfnlmNO8rjwi93BjjdoPYa12+PNHvQGeyqx6vp16pOS93TB+1FdDlUkWoYxzk\\\\\\\/T5+R2dKVrEFVFI+MOhfaLTl4i5dj3SOy2oJufyxRiCtwZvFww6PphHaGDtjyjo9EYJ8lZpfoSsZPZZWMWs7K37qg\\\\\\\/kNUDNPm46FpeNozQjHAhJl+KUY1J2wlWaflTy2qoZA3zFgmNuDC2Yoso\\\\\\\/AxJJ6A+wleZnzXt4qc5rbeqKNOdvz\\\\\\\/1R2sy8\\\\\\\/pUa5epCV+k7G2VNzLm7+VeGEuO57RJNHOEQum0udgypOvRHcSbCl8CS1f6VDONpwTvvpbu5GyPlAPArb6Prs5unc\\\\\\\/21NF76Y7oYrKNj75DJZqTSI97ITgNzhvma7+6SRj3Sc4LESBtOt4CD35Yoru8u\\\\\\\/s+40EuqaqBDwwEsmX6oRhkavMMQmJbzidRERyKlzDtsUD3UTxxuQ60cbgksSUEDQCgpVxjkG36RyNIp8qQ5D8Tx10obP5nBmijU7KfqeC66GjBDlx1ulaeAmkYq\\\\\\\/Sf2AK3tUg8D8RvSdO3858nXed\\\\\\\/J+gKbtzx\\\\\\\/fdMeD9hVdhuJ8nNoOGLSzzgQpDpIAODbm4+HtNHaQhHBCslwmf+Jm2wWoys8cPOVewp6WkOx2OJbKKSanZZtgfu4\\\\\\\/YfL9zwqKWt86TJlYMfJxJHYJf0TgRBmsqmuKeiZ69i06cbdhtyzHF8X2gZVEmMNhq\\\\\\\/ntcU8P7zqUUsln4UyboljT9i0ieXF8+wRXop9EzvrBKvFSA6soGKKDwfkK+mh6mF118kGFPnMfwXfKYvC6QcMwvg4YNCaDt2\\\\\\\/SMU2pME2qZg08yC3B+zw042xysveLnGZceWPzqveddHTunqAaWICx+cVLV3\\\\\\\/yYdbEHJp35UjIMUoiJHYKT1dPrRM55yogFBd0x1t2\\\\\\\/dsmdvHsbEjTZNSndiB6cwZyoes7GZL9ltzp7fZf+yT17mzqZTDcyNZz\\\\\\\/dU3IhEqCEf7SiNPjrnDvGc1\\\\\\\/MhRJbK86wQKC7z1BzsFBTCs8Obdb\\\\\\\/HJW0NXDZdMn9RT9tL5RMw++\\\\\\\/pClqAxPYtOttK4Ar3\\\\\\\/1feG9PSW0mGPHkfDDSUhWhcNHrPpTZ+jg3GijOaRT29ATus3qij\\\\\\\/ESd0KjRRMD5QcynS9N71m7zGhgsdX2XoRQQQ0lcIzPzlymXRYAGkHLFyQc4tZUgiyJ8j6SgKjEb0mM9k3ysjpgYAvcahQb4CYmx2XPrecaSpEPvJBeuSY9YBjzDUMDj6\\\\\\\/h0yEyP2XjRDIIN6BuuAOnhlklB5DBxrAm9MKIXjrobVsdScusYpbf5UYnHqnGyrZ33t28jqjoy97559Ia6VKejny1CU5Sffzkook1ayA5rDWFCDWQC5BydyUCoX+v018dbIlV5c515OpZUjR+4QzQd6SjmeWaFn1GUTJJTS+cR8fPqdQliYtF+ttAmZqiqJ2NHzwxqOCUmaKWUSu7\\\\\\\/onCtdi\\\\\\\/zF5ercRUx\\\\\\\/nQGdMLS\\\\\\\/mOCgPnPx8Wn2lRr7UT710FXnrIV\\\\\\\/Xv+sLAWEOwr+l1lfhb0YcjvK34q3Azlhb1BAoTAK2URLENh1SkkCNQ\\\\\\\/1RYOVUucuIKvGaXTyP5w0CVQ91lxsni\\\\\\\/4rbb3VXOfNIzEva0ZNOUHDND4K8gd3\\\\\\\/R2jWNA3njLCnGrdvBEC0qVEdUWQwOXsVwjo0bQX\\\\\\\/znaXvIg2XqBKmDxah6s\\\\\\\/Z\\\\\\\/g5uwqB4vVHcuZ7Z84BfnCcUJNF67JJ60DofrTX9jZh4aiR7rmcZdnxHMdoxQoHdQjhah2mTu4\\\\\\\/yN6Ep7NTowI6i4Cb5Kxi1jxyE95sqs8h3nrtQGhRSJ+d8JBxsf3Dru18GKruIwQ+q1s5T\\\\\\\/FcpDOS7cvXVlloJhrGFcGcuaZfhgU2XSUU04tRssc\\\\\\\/6tyuTa6uiKtgBuxGiGukFNExS9UuPfc0pgUXBtj75wGNKVUXgPuZi8RTzpsIfhUW3Roguj6LAsN1R\\\\\\\/pBxIziqnfJ62loHxZfrVZv0kcn2nAhRoHhHdeMdkfiwYKTGBtLcyBWPxo7Agwb9zjazD+bSR2h6cs0TcRUGyjuQkfygahmj4tRKaneTcwucNhbWSm9X8MPtnxDfH1ZBX35IVWLzYw6yn09zJWSW5PxMqq+zOgW0WHTysgmlSU5aBRJmbqx+mkh4AYeUtyd4XaiABGwYfizf3llFSsdZh2h7QC89GitvF9+WDrVe02QJ5kvpLf2yHIphAAqzB0TXF6\\\\\\\/f0aT\\\\\\\/eEEFD3lLh9O+wXLjNV2U4te3uRR6pFAZIPToof95KVdcbNb9yELRdb3D3sFhwVHrJSjDRKd2zPkqJIB1rB8NuW+C4+wfOdCGq5XaxG3TLne\\\\\\\/FaTfAz5kb9F9DoLMVNstD\\\\\\\/eYZlYhdZ73vwHycwF8Ka9lr2rWy3k4Q92lVf6vBTF28+9QSNbm9LOb37q0bLIaCR4JemQhiQ5+MxLnwbuzUEJdxWdTjQfNvcQKY2EiTR4t8VydZDxCyXgxprJ+6PI0u\\\\\\\/f1S884O9gD8bcmMlCknDDPuAC+BD\\\\\\\/P8AD7uDllM2A1Pq\\\\\\\/ng17lmFTS3KNMXTsqoHCA5\\\\\\\/f9i2PJR513KAJYcve0n13swGdqDkidImLp888Xcbg1qHxJzVvLEFMg6p39hhskXohQj17DWFyown0BNYNJhzYDGUi02vV4gV1otn+J\\\\\\\/DO7Vd1BqiSTLagNXy7cc4\\\\\\\/OyryjiOTXJW0M7vk2TXbnnM5Jw3xjk6GqWF3qNlV9aZZ2Rs46Quo1JtK52fVOlHmTFQZ6jV2Kil1tAJFXOyVmJG2U8JR1VQChAy2Vpka+LQFnS4m9Xj8C1L3u+dnKdjUYPl22PkuPAWIAphPvrdLLw0G\\\\\\\/SCL2zAzPQkds0XQQp6VmipLu9z587ZQNOPrT2In5vjyJMageL64635y5VSNfr7zaVMZ7YuySlZgCyMdPP9FUJQtunJ++KDQ1TTOmLff6XMASTDvFbaRJ1sqSrBq0yeP7xmGHDzZOOsb+Emau5ZTgjsFnTtZkMtZRbJu9l\\\\\\\/21FLnW7xbVld3l6DuTkUslkn9RPW262ktYc+3AiriYwt4t3+0oMbBp8UDH07GqA2nodpXORAW8bv7XNWzj58ldVAt14fmOErtR1bQa76+1OX4s0wATMBLUsq1GJz0XLH5taR3HPAjQQYBo+zx+fig0uXg2kH\\\\\\\/cM+OhPF34toam60SMJQo7DYAto3T0dT15hTlLfTAA2T4yISgv\\\\\\\/R9iu0Rrepl1Ysl4rJcOzvHxsibS4wfNU8AHXmiNdEO1Ol+AnT736SCcPR1DwurWDVnjKN9ip50ISgUWqFGWUYBSI2Dm+VXVj0ROb7dGZqyEyqwJgR+FnUZQEXL2Pbn1\\\\\\\/gD6BwRG9x3ZwdEzhVsGRH0XsIeH1b6J2rPnSSFWKV1tJDcbCtDiOPXKMzQGtQ26M6KNwMXUmqriaJP1fDlqvXCvrrcQMdwAcYGWhGtRPetT0dXeDLxNSORp0li3\\\\\\\/rIoIdyHnJJHN2H+I\\\\\\\/nopj2g+LHWAEew9rmUayn\\\\\\\/97h+C95ukqPg5XRIycwCwUFGw0u3NLlyZJ4j8F3QNeOHLDANMOtDsK0LIZ4qZPc\\\\\\\/Kc4KUZEakoNgarYGpFtoLFSRmWwfLXpAoQgdnjYufgF4f0tgz9QacYNXwPp3ZtCiKuWwJwKBFSCmXAVj3gSuvFLmdKPXMwQ9+JpTaNQmJERYyOvREaYlfSq7XQEyyPh\\\\\\\/5bYnIjPACXl7qv1sk0B\\\\\\\/idVe7HKgLFKIlee6huyVDetlRuqb5Ij1gvsl7LGVG5Q8nHcPnqYYTxFn9iHsuw523Q7zae0nPhIVdbZ3InlkbgUBsd0GaTTN+8kTE6WNyINcNGGoJSG4U6Jr+qeME7O6WgAL+KLKquG31VjHbAusmwkDYZEpjmDTMAwZ460bEwXKrOgiOaqBxhxuLZvjWRJLHmW0Y17K0x14bLeHZUyRgBZEcuo0xm6X\\\\\\\/0tBIzbLYw3QoYgL7TRysoWKF0t6ukj2URrM8DTgKXOPrTlIMNrL+Q5JGdS3Jyaf+cuA1O6M3A4+L\\\\\\\/G1dMGxUbRBqK\\\\\\\/76VReoQEQD8y8JmQCqWyUT8+0zrg3IEDgW7B6AHRj0Bq6r4lrW\\\\\\\/TeB5ehHDh4wj20y3IBXPJ7lMtT1Ghn6uQVx4RKpvKWVJMJXIpnQPd37wbSrlqi6dMAKugpp9AylhiYGpygtJmaM4lCICT2qiv4HMAZm\\\\\\\/ojhcMPXlTndu7EuQsVFiErvQzc4b\\\\\\\/raiZ5K1bEuu+E9utf2N6zyzgn4GRLwH4mr3jTdAIdr+heg7pa1jd3EgS5U8eZz8lYbdl0wFZrgdhSyGKRYPHxd+A\\\\\\\/L2ZA6HCBs+Goy7HGTfcd8JItSYx5VtbYlSW9TiSoeZgCcewiTghgMauMFpMW3vtwzdYL8bJZGV8PQmXauDyiNmvHXQiP5yhNWDgRrCWXuNLuxHty7yv2ImuDrFGYe1pFN9uEF0RhBx5rJE3AF\\\\\\\/i+i8PlwmG1\\\\\\\/ModxIlSgOQWMbDPp1qfT1SxKtRIX3d3SJiUul25g60ne3ALNtEauswhq1D3puZB06jxZJ7snZm8B3FXzmGe9LXdART3l+Ge1I3CBzjtjcuGamJEic1E5VyxWGOUjWydvYSTsRBVXl51a7OMBpd\\\\\\\/Ecl9i1AP31eHEEee0+e+osjXcEBz2z0nnPHn6z8vDJmCbHbL9SJKkFo7wirYBUllS3yjNmB7CG+jcRO9vVaoAyk91FEbOQz4XCfQqcjkAYoeHU\\\\\\\/l1n7nPFFn1jQGrBzUIaMkcnfXjs3ob7zgGUR5aEdsTphH\\\\\\\/Y5Vw3Y7DLQ4+FKPYrh00zKrj6tENPPV0naGj9daWhQTF54FSSVCLNmQHmzt4BHzxBHyjO8cYP78Tk0CyS8SzTy8DNGdg+xQyC8d4z9kvNrQxIBOufsDSz7YNq3RRfajt2jh0u010BeL5\\\\\\\/wITbQOMv08WyRnHP53KNl8MnYMb61YtHOmzu1ve5Cr4+BrGDMdjrbAlarGXqq\\\\\\\/GZ4JJNwvq5RWlEZjSdFlydKuD4Z7D+3CnwOs8Qd3USwKLXvkwQ9D8UcP\\\\\\\/pCvSUClT\\\\\\\/6Au8DB0hXshttNrAXSzfVMCxpUx0pVWC3Fxm2jZ1br6nNH\\\\\\\/CfjxLfF0a3Fm7LN8Udl6TzX4Ur\\\\\\\/y7zlt2VWug6140CwTpaXjGnF+dsDQZsfDhpjtq620pkhvb+v65d8AAc9sVznjkGt9PNVtEY+nIpv9k34CY5bsOlqwT6E+lm52l1mOWMhy4aRnSnfnkXgLxn+BEhOZyhaBFDSpPTQji925oHbUW2jCH6qhj1iKxQBXo1vysJtgN5DsRSEFYNEseIkal+Y68jeo5GJKN3oyCs1eA4MKe2uK+AgovZeuRDm3p7yUaYA7LiYSR8WUb1WKwcU98SUpzFugnU6VIed+6LJ76azp6osyuAgXqeCDTOgMc4wT4tz\\\\\\\/u4jTOlBM4lnp3igSc43cZNtEEjkwMjaW\\\\\\\/DS4GF8E6i5\\\\\\\/5jidBGT+QNpu17uMsbX4M7ZxIjpGfIMAsFQqXyrFigS0UBpC7gl3Dsyzgyxuu03zLk49uITUDIR3iXxz6Ndxq+Y4dhTSyi\\\\\\\/js0APMNeLmEjpoX8rtg4EIIAvX2a+WfvOLbtvgBiJtidBSRhRe4D8VbggI1HPCZeOVpc\\\\\\\/CGNt2BOT6qrJWXw+Hi+nTvTcqVUZoMHgDoy\\\\\\\/JVX+pMHbvNQAX1pfaQ3qhMWprF1GxIv5d4rdakW29Fb5koHUYXYvH0JQZTi70oXR7uBpOFIeCnH5XMnIS242Cfn+6PxPozvVu++XNgZBchsaEClcMf8hkf0mn\\\\\\\/CtutRiaP3IKjZUO\\\\\\\/Tdz3dNZH6+YezQXy\\\\\\\/AHBtTX42z6zEFjrwPzoB3UTekylRz8H7JwDoxml\\\\\\\/shwknyxbG8cKDAedlWerX094OIn8Ls\\\\\\\/xkRUZ8Prl+fAwyAqdQLATyOPwiEa\\\\\\\/gnoFTA6yyKm0XWUJw\\\\\\\/zKivTdJ0sH\\\\\\\/8bqlZTaf+Le9WMZAumWg7JCAuzPlazHHzBZMKZobs8hgXTAPrLyMTrx4hVAOI7k4a1Ir4\\\\\\\/bFxO1L5BkY\\\\\\\/h5opbkck4agq0mpVyuPa1pINiFATbZCq8UBG0sZVrOTE85O0Y64cyMembhOkxE+L5Ikz3AOwbxR\\\\\\\/6RWNfeMu1pgsD6uhr90K\\\\\\\/npI7QKhedkBrvVJllBAmQDGgocoSq0If9UfTnWnYY\\\\\\\/WFEKczUVfyLJam6Bg72STgqsS4JYh+ho87riT58dKEuL15\\\\\\\/G3TL3VZ4qtEYT2Jf+wEnJ7IjTPCiOks\\\\\\\/vcWFFyIB70VYqHhZ0rDZCmUpIWp+CX5JJ51fhBPDNOTqYG\\\\\\\/Hi29kbLcKEdLNvsgnho8EZrhxUGZlHpFazNFsOVecGAl5VAUo6vrH3l6et+1PZn6o8h9Xjb2KfBqSJzTvL\\\\\\\/RUkHXGrUHSTWOgC4WGkrurC5+GFoRXyhIWt5ixC\\\\\\\/umZhv5ME0y+xgPcRvb\\\\\\\/I1MnbLyjtDbct7\\\\\\\/9qVFqyafptbzF+NOaBRpFrcxyMKTMkaNjwoemmTVXq4mIbFYkYl3+8bcWgUUjRqwrfDyvdtfQTdenZSMD7ALKnzMNgBpWhPtpZpE8vlB+NG0m8rY3k0UaiBNCSQ9rDIq0Vwv\\\\\\\/y18peFv+2K90g6KTxR6e9KFcrItlS39K2cDdZHkMilGsh\\\\\\\/tzHztLcypxG4RbgaA+VHX2fg09nYdxt2tMY3xR5LRxY8QBnnqKe7nb1dJdDtfcWwVWQgrKf71XLnmpBrkehlQsoJZgpGCMZ5Ucnh+7cmloL2z4EfUYDJYClBnG2nMZ5iAuj9234\\\\\\\/ZGYUMiPundAB9IxXyqw2wkHX3vyTIz2hQUjXirw2EAgJcghfT+e+jBj0KrzCTHLLijAwWxsLjW5db22E568i4\\\\\\\/wW8KUTgo+66\\\\\\\/NynPd6K5jDyzcfsE4GLWr4yxKARsZD1fYS21eiM6ImPvnebzDpVHocQnceVDxPZypK+P5kbmQSmFQ3YzaGzsQn+yMYTg2QcToEXhgwCXAMdeJVUKKoU3shUWJx\\\\\\\/VfNhLJ5Z6CAh\\\\\\\/POo8cLBoQtfjBzCKNkkY7+NRFNZ5C5NTVkQ79\\\\\\\/v5SSbSdpt+R7Tgbl\\\\\\\/I9MLtxlW0YkqEJ73x+5b7B6gyaPkHuORjhPwGVla5AVQYbn12mgGN9vPDfV18gaEvSysAXeix\\\\\\\/luwMVXEBsXVlCCLZagwoTEW6IMPaS20UzX24g3gurlLTokkJVzLEGP+NHdTStRNzPwHds1yHYCGADfFN1ZSSUVH5OCe1kmPD2MuNtncxqHIlc1YnlgWgubkTBfCFGcx5T\\\\\\\/HCSxiMQlvfidhizryBsWnhq3xyidsG9hhCC9Gntg1xdIXXtDSnOEkIWhW1aTOQh6ZO6y9jgP5wfuWNZnSTaUCacoCuRklZA\\\\\\\/CRXqXAXT0xIlMzmFQynBpn9ySQZXyiSzY1h2FG4c3HDapBl0TA9hwVFiZNtToVQdV\\\\\\\/QBte02r+t\\\\\\\/0MOqW2Ht3RYrnM9s5Si4vCHwIuxBtwzOYJWi\\\\\\\/vieCPNh\\\\\\\/y2w+WOEZHLwLK6xGqVOrIMG6WirLAeefb9AebZqsJVo+pHnTSMvWORM1xW8LaHNbzqW5ZrYxHXtUBtTK9LOrDlHwGsm2GtdA2\\\\\\\/iAaOP7CHY0cfnNeHhyb6QylEQ8hPJ0SatV9gZCU2hWeZNMxA8UwmMQXP\\\\\\\/QomjxJqa2dmBDc7RBUKvgYWQmtKF9pZdlUUKvLKPPMGWo8W09BW9sfy9ttty8IH3QXIYnrdFKvAUFXyhxfCWdWb7IFux5RETiyrGg\\\\\\\/Tkm1cwl8vs72Jlte3ZCV6dm+nwsz2cP23FeIQpEP03q5uUV\\\\\\\/0DBMHFvIgJCOFY5u2q6Ko7MO5efwPtNFg6b92wtjr1GT00bT\\\\\\\/m9YK6Ba8hFufwhReZ\\\\\\\/1+PJto6PhSaZ0x4nBp8SrHd1KA27PuiHVSdp1RJ+kwsCTLIAUio5BR7MO9rFNDlog3Kuc05SCBtC8oyu\\\\\\\/8REjUp5q6PiNWUlkBvDe3vq\\\\\\\/rnyyUJjONDvNM9aXgnroh6Au0\\\\\\\/dJNyp9uf8buVT7ZmvngwKYXC+YPDZfWRAiKOTdFH6T614gGGistVhb+a\\\\\\\/6nTvr9wrCjUqllaJ6AfXiI9ZhIbMOPa\\\\\\\/zJiE3k+mmgY+S8ifKdd6SExI7wL6AquZjlhEv9QqhOOvp3paNwoLjroAt5XWz\\\\\\\/AKs1a98E5YuHLHW+Ny07IRDGgHyoHPPzkQ+XKEDCkcu0LqlnJUmLKpLeSe8yDNB1Ac5KSRP0EObBi9PuQ3s7o590ZJdMiaMOWM0xWLJ5pcomwN+4s8V6Pqp8m71VkOIl8G0n0EFwMjNmNH6S9gyrIZhnSt84f2hp8Wls5LgSsoEqToao\\\\\\\/MWdNEBxQv2odbWeTNwxlq1O\\\\\\\/vbVxZhLiMYguWeG4itzYFv2uoIlU1lZwFgCg454LgxpVMlga+8KwnCa1kPWye\\\\\\\/VtBKrXNqQRqByLCLoMxweVbn9G83L8fIVozrllNWvVPFM5vtGrGKkOD++opUJ+x6dFfcysC+3z1yC7gfThfbt0IjqR5j5XTSF3o+hFleRI8UJ404gEa17eGGOgSa1dTZh7GsxqncecqwlltlYU9cOz0MdDoN6Cw9akaLIC9WuMfctGxppiiMamy0Q4ImX84VXdQROxHgozfnR\\\\\\\/iSBU2C7otd526K1kBPSbOW20LZHneRndtnVAPXr6xCv6aIAu+im25o3slmrd+4IaR0Nwf7+1uc0ODPNYeMDP697pNlrs13ux3cpI9SaAl0YqcDYIxEPI8Js+6N3L+hb9m1OLY6hsNkIA+o5pTKSoPRRuJ0CPMOheooE1BqQqZeoIAoppgiQivYcFgjSFwUraYV\\\\\\\/JEf8Q40bpLnYVOGcJw1xPNJEP1JPbCihApcdEjsO+hiyU5ITshn7Pcji6Ll8n5cW1aSdITTxRW+ra\\\\\\\/+v83roOWOBwk9I3wmLvax+7Q35XFZpbUWA1lIXyT1\\\\\\\/d8HL9\\\\\\\/4Iz9+oYPKLXmjt4nTfaaHX9HNAFJKThk1mF2r4sxOM\\\\\\\/isV2Hx19nIgIcYFQsOxc0n56nvz3\\\\\\\/71B7xyC6sy9grb6GMZ831ypxGu6Fr1SOBQCxfMcobWTzfFnJUGdIQ7ooNO3OPqLUhSkv6EaM1kJZP7wyxKbUnLsBfkpntarwvbVkppKbByahX2mmaCGcHNeHMRWsrZVjdb2qoSqTuPrkQ14lBYMvaN6TFeCTCqdUpzOZkwcSTldw2dKK+DLpesbzpY1cCSUAQScVAWpjOJsDSMs0cjiKcL7m2OLeww0SDfgKuG7mDwrSY9UvpAhkV0RZOTkfVn7XHB9bhRPwVyEoat\\\\\\\/mBLpZOqU43EN5xsB0ZGBk\\\\\\\/mOWu5omUo11oC3mQbcQtMAiTRrAwgELNaKWMwpS65PEEoWApY06iQaDohFCL0eadZa1t0Fd6i186Uw36KyzizXae3Lb3ryuLBHnYxX8LE6+bNCLjbaBEJCqSBaRW6ITw3DQjEQbdxNq94mer2KMyokwS88zs2dryA7c83Sm00YAEyBM6xyBZFKOSLlDhDAcaCYuf38NGcXaBr9l2KlrIEp9RUj3AgG50NCvHS9OGVwRVcm2HUdxIUALkYOt+e4Z5AVLigVSNDbC6fr5zCT1cZahU5D2MUfZrFD5ZKtR5gYQXxpy2Qf3AYKUtIPt+BECFu9z\\\\\\\/v3XjJKyd6o2AAwKl7w\\\\\\\/mh\\\\\\\/eCFVWD9c+oCQ71yebClMH3bknH55AmT4+MgRlEAR1vDfx9mMSSEjfuZC5MpXoPpBJNTRUDbSIXkQDqAKHRGnOePCl2nmmlornl1sLmWq2uDJG0qZnwwwU1XfIJIUVaYQx1QLfWV\\\\\\\/6Q3fLVYtfdqOIHaiIH9tCEvymrYO25zpcslTNYMQdKJI+LgQnNxXgv\\\\\\\/XtasvD9C8b2U4OZF7iVU9AgysFd1vDa771\\\\\\\/O9e50apzyy4OoqBB\\\\\\\/HrNyLlYAZsfOTX6UG5f7fgIdNz3RD+EyuARfWEhvDiU8VjN+UsLawKKsj8iwiazOPePcFH7yM13riGibaf9SBCQuCys95b05t55dDxdDK+FDpnLta0xoBRLi1D0Y04eQH+n0X\\\\\\\/7qMwqv6se3yEWSgzs1DiNAL4bef7aHYNQZN4IQ1cMhXEPPSsXWfJyp8MUp+gA\\\\\\\/\\\\\\\/pRq1EJyYn5Ab6b7xCmvOaJP\\\\\\\/A3pY4B9PoCUJU5wQw9oy7FxoMUdQWwjtcp+B38ItnJ1APtoUW1zJvlDG6JbAUpQWU25ljqOTEN1jb7nOF+J8f6dEHxSr00y1oczMRqAGYNPRqYu4ueCEoQGd8ueAj2qmjDqAck8Uno3oG8cuAoFvVR6Z9b9txESSz1k167Qdbq9WFPgViTfhpoJ4jgptwh97eO6+7uHWerayt6EmU8irscHkim8j9RjXcTC0AoB6+WxZK9NVJ7NAqimDtvQ+OERUKpM9jzdnv9LeZ7nnivXBsCz1AXUMT1uRmJc1e7Bxd3l25B9qAd69lWoydbz3ZHT0LgwmyKFt1t9JU7ojpKdmWN1XeHAHR\\\\\\\/JaHMV5oYWegDxjMeoW5Bf6JgQ3AnmyOggdhittzEbi4RWGVz9\\\\\\\/H+0\\\\\\\/Gay7vsMmZu\\\\\\\/JfA2kmuoel3TBJhwzzEKmC5YF90xFETwpFTorsAph+3XiKBqUfRVFb7Vq5NeT0+etTBNIeJz1UDUnTkw2PvOPJWV47zq+tH0455OG7pFtoyhwHS9MXzLCfYG1MPddVm+61CtKpvnQPyFtx9yC8RLRMfmziQypxclTHO7LQ6luVN42cFPY09Gp8L7syKjOjQrhrNfH5il\\\\\\\/MfRLZs9CxEcJVFrJXP\\\\\\\/YXcBweNxqoS5kpZzQyndb8IjMkK55bn5bS2sUdb3kiBMA+R6Lp+DJem2ecYTNGjGv8gEzCBzfxpOO0sd1BZ5K3DkeFfrEzPDVTxhukihg7PLIZlpnY8mjgOIgUNtCZBXley0l56Y+s+IMW4nk8yP3yU+tq3xgNSnXzfvCepZsUIjaMRrnY+p5fqvLdsbSjHFx\\\\\\\/n4n8T3HJr52w\\\\\\\/jyvSUp7u0WQTzGl6UFnGgDhl9y5VO+HFOcWvYVC22nWbFaVSjsnHsp0cqZXD\\\\\\\/jOLEayQ4ESwq7Y1ToJc4DItpZRJ0sEaN469TP1IAzK3xXEZIJ6zTx5epU1KZgIYFpROpmaj4XCrI1npOW71EI1\\\\\\\/kZU+8GLAzDsq42Edn8x7BpsnO0nwdpa7Aa\\\\\\\/1n1zkm3eG3GDY4hN+KUFoKmHpX37EKl6kiICvmPeZ8Iu06y5H8BZWwNAWjCCQciubWTuGXs5Mc\\\\\\\/KEpVgT2w4ENHi4Jzqhd1Se6gU474OCNUpqWsJYkOXaiCetPq1BtWqfyTdyP6bKGvAREp0UejEXRkMATWK3tgvjplGHCt+AeuXkfyrYzzcXxHoYkMIvhI3MzkSu6o3MqjVr1zfdAnpwU51JfuhTP+m45RjSpGkINd7PZxkcoAuvbs5eMHMakmFNZhw4R4pQeWd6imhId0bQZu0QQCPUn8A1e7JtD8ODvEIZx+R8zJs8LQUQmcCF8uTVvfa6e+IlYQ4tMD3rXzwDQWRpseqN3fzI2MLpz1c3JdyajupEbL+3EDrBfwJyX5WOvU4oqgYcrrliWSai17CItClGdos+kfjon4RSPA7GAiZHdcGPfhlpgabbsOMum3bufcfPEJPCvc1RQd0p5q2a56qhheFnah36PvvZxB9360LHb8\\\\\\\/vZoqCEda8mlTTne9qiq9SEczdl+RhzGecNHV8lsd8Hohz5Ub9eNrqjU0jihalHo7MDxvE7VHYYp9dmN5BjBlAh8GafTFTkn3IoMmTlRE4T3aNbfoIsroPj5nhgfpc5zkWjTRrt4nDRXL3K6i0LNBKhJmJIqaqH9RfAoL0uQ2RhjLv\\\\\\\/Y7e4Wmok7ThmfFhew26xddFjG5YpHwetxnuzxy+KM8hpPE9kkwY0ZsK5bGH8OpG93UCdRHpnadT2yxesQtX+CNhAFNmacT\\\\\\\/Ecpb7XS1QtyYq3f\\\\\\\/UV7IqFGzYq89sUxJGVdQ4BVIehYOyeuGeCuWZDdQnPH+VVlXhll1dWLR0ufDqxdWlKpzhU6SW6QLreHDBN8IcC3pez56uRP+jFtTpySnNEQvYam8X79iYNCPujbzkUxrd9zwlbTrUpQNwZ990oLAxKozLvRXfaPHLnyXxtu6Y76Dqw9NUbquLJXKWmV+bfAFY+e0fXFq\\\\\\\/7p3Je\\\\\\\/Yas7mvGfXBRrFygsvI0AEbD8MSIvvHwdvn9zYeX9RFjm\\\\\\\/MrQUMZ3\\\\\\\/slRcBPHp4qxToekZ\\\\\\\/Zf5O59usZTFb\\\\\\\/WJIsSwJBGjPn3NeAYJTPuSWYYIgYolru0wjI6GXvpMFoUO\\\\\\\/Lji7BtkG0CVa9JH2Ohd6ENw0lyfzmqx8JNpTiHskrXRYd8Ac8BnkQiXwbfPQ4L3Tuy0CLLEVRh+I6OShEPxR3PXi+erY7thN5WDnFdQQ1K4t9cu2k65Q533qugflwH79vMPD8df0Jstre5TQ0ldt1GUoW1xID6QbYbRkCPwsHBLTLia9tlZdJysxzYmD0KJ2SkmFgN2FOsov1DzLaJlcYKqAqUITbxtANo1qvjKDZ8yFFailFdwMMmQSA56H5XIZ2KJUpIYcJ3ZpndBpCpYptAHlMiwmRUUgLSu6VW3KrKJr67xBifQikAG9D30S69Pe50bDVal30p8v0N17YsYm0yx\\\\\\\/Rzk2\\\\\\\/ILZqakpI5zS55IvPWT2l9j02bno7osFrn4mUiWqrf+Y\\\\\\\/At2D9SoNBWJ3gLu39JCwdLU+Z9N1Hbi\\\\\\\/Z45wE+mYz8m0pnx7IfU3JE7IEG9xURWhbF6uDhiP9326jHBTx+dx2Vu+yhhXaa2siDVFoVqZrmAdPz8JEV\\\\\\\/4jbV0ufOBk6vgO9R0vra2vEWUYZMsUjIQf\\\\\\\/WiIYim7Z0Nv1+J9EYKfDNs61BRq0LiMnYnrElHCzDiLkDNSj1wWDd+6VeGjdb4nuXlQ3CK0OWG\\\\\\\/Z1j+VBeCC43khy3jk2y76qjGGViIQRxrY0jtOw9SmFJ8rdZuJmti0aOuJZQw32k4d8ZCymTK5y3O+3LHDgu\\\\\\\/RNrF9dQtyu+HrTiIxvm6DwLsgsFerAs1cBHVGtWUFMlRVdB2NFS6k2A3N3OM65hGRbUgt5yHQSWqc1Br4kzpqTKgy+Rn0VqsEw0Vh4WOLGnAYFOdOE6UHTEte8A9OhFA7oNoV57OWDx1A17JimWmMJha3QJwanqs3LYckBV8FtiUBfYMTdJmJGNS6dxNerYU+nu4ZkmFEsZj6KcP59VO3RbQss4e\\\\\\\/QnEg2VBhI9lBlRrz4ZxvfQVWyhvxhQdNFrHqCCpN98vwBOhXB0\\\\\\\/OPRcBumRht6aPLQTnP5UbNXhepj0yZaOAsAjI+xQNvw68G57rSihERD\\\\\\\/rhc3EryXZKGuCVMDO87RbPCu17mioti3SXCpgAujWTZg+YAHxYubdN0+YINZLgdDyNeFS77fXsCbsrecYqZn+nPNZx7cuV16K0vKD5rXazrW0JUzIioZI9JizlUkgLn+oAEN78Nl8qY9QAZL9quU79wH7y2v3keof22Ok1AByloKZIc1hMJrtC0MgOqzxdtW1uBTpKlTvT7ScPvc8opUJrDBZQqQO\\\\\\\/f6beGmcrRHvQyJScs\\\\\\\/uylDuzq4\\\\\\\/8bXakBigL3wOGP3vJMAsPpUkau18uY4XMU4Y9nQ+lGEzM7px96p1o0cYv30rXT3iuNxn+U0hY8EdjOmm0UfU4u98+sETip3OKgNNZ6wQem66M23xSE387UCWamERfkdwhUY7XeNLF04QgPR1JX\\\\\\\/2YRqDWNSHTMaUScmmgDmzc8+N\\\\\\\/81znbssmhxzapKC27ETAJi4\\\\\\\/pjzDx6NzFPBHPNAvy5YtER\\\\\\\/Ry9583\\\\\\\/FvpHUiUeRL56BTJJJEII0JsFLK8+4Fg7zi7uBlSZ\\\\\\\/g4G\\\\\\\/pZWp4z0Hyuuw8Id8421dIysr1B2Q3nZqmCjLA9G08Bxw6noz404i9XJ6GPwJQjIVbLKrZbrqLcdzrY+ZTVri0GmFiJt6b6HB3nSpih1Xr0+kwP23hEdHXt\\\\\\\/kCS8HUudbvg1QyGVnU80P\\\\\\\/3s4WYqx7boPZYBM\\\\\\\/D0rXGtXVytqRBamqhRNWuNvpfMcq99nJGoCfmHpMA7vZQzfUjd0QY\\\\\\\/OfgSLMAMcBBVUwKPxcqz9H9PQmN5TPN1M49iNIxZRNXz8v63aFNUEjzTDH5LNY6zcxUM4ViA3IgGysT6IkHGp8fwct8o+8B8zE2oYUi8xyMmCAAkOEuGc7iHuJtl1ifvP5EGGqohwuAjW8ATfnEelGk0rg6lFuJmYrmYDOlw\\\\\\\/C7nLpbZvxQffoo6Pzg5CneqY5Mu38rNyXKlHF8RHCmAhZOv9gzP+kokgMPvmzO1l8Wv2hhfu4YZNhnUmpN27AEfLp\\\\\\\/ExMqPsfxdFqR+UWsbcef+LHc\\\\\\\/YewcFxO1e6jKzxgEtzyjSVECDRCncfjCyVD94jHZ0cuFSzBC+Fhs5xqJc1lEOOK0WSQfeobgEZ\\\\\\\/BQtKU2cJgccZt\\\\\\\/T7VT9ultPU6k+AeiIS9NolioKRGcygp2NKDlAxDP7NHY5TQCyk2r354R7nhYtg6N\\\\\\\/c8dpb8beZudHbfhUD+XT8fSxGDlBmMYAaWnmPjFeskfs37aGtOvGK1FGeBtCeaV5iEy3e0xILptp6LQa\\\\\\\/MmyYaT01MDn7QcSc3Odr66QNVQQ7thWWE7+6ohSFruHSoJ1TPMc8aHL3LdPhmG539A0zE1p2pkWiqPbN2KYETVdDwVH9Ljw+Ip4o+E0WM789Lyl1TSxl4u\\\\\\\/aY8sYKdyu+2r7eoRO5FlCBMrEZ7UWM9H0xfdzDJCmBZwLRu\\\\\\\/mTdzJZEZB3j4J3tWj6Nkeb3p6VDjfQFqOHDrSyfN8bgx724MNU05G\\\\\\\/3VklLi6MhwoklEXqbEFxTh61jG6NQaMlz110jXR24E5LtY8WVtg3QpaFxJMAIRZakPDzkH49jrsR5CVTu4pXBEvruHoxr5N\\\\\\\/qRVh39nsAhPQIjC4vv+gPL4CW\\\\\\\/uQ4yruttJzIi0bSFSN328xEAp6WsapZjxxdAVgC5C5M6yTZH2+l8o1j6OowaxlUfmeByMUJftQYR4yt5Ep5btImjxE5NlxWpRzu7sfCe214xzgN8+59VxoETbd086Lhaxx+x8qgJy0TwxCPYLWwSbtsIAWrWpUwlbwmT7gV6xOe3kHJZZIlfcHqq7+xudZ5stQnpVYOShvDDjyS9Nny9fm66ODlKat8iZUT+X38vzram8Ex\\\\\\\/JCoqD21mbDdLHceiasSlp8pV0\\\\\\\/KkT\\\\\\\/9jv+VMrHGBee\\\\\\\/DeeFJrzQS3OEAY3oMYn50Y183woMQ96F3r65E+SwgkRh2Wt0XMpiucK3\\\\\\\/aPzz\\\\\\\/1ptR6Q5gfCyFxCLOAfV7lnZd0SouHeV95C3+0OL+vW0I4x9fO3NfRCFIflIeKbQMfGIswzi0J6YQ6QoYeSpeeFpnPq4H8ZQltjLfBfsSYZ9F7C20VP8yfFq3XigV5xeUMgVjciSMLrqCKjGRX2UcKO0NYJhEAHvO\\\\\\\/y3oMp8xm3TvztKTrHkN2PGnzmR5hX9MZcdGwkNOV2It+8MsoNERZLcqYmkjxBp+VTrjUukpFMr6qbLSt\\\\\\\/A\\\\\\\/vNffoK65tn9l3kngLn4LOGVauWS\\\\\\\/rw9vSUd9aGAmb4kq37K3yfO1t9TO4NmRoWqrGC4ihXzFfS\\\\\\\/dvxx\\\\\\\/zplT6ufK\\\\\\\/AHpmpXdmHSvZx2R107A8MFSJJDjQ3Yx6U7\\\\\\\/jgUiLGN7Kky4rLWfsWtHmQDduQDishkFGe6LjUib\\\\\\\/xOGZF2MSEOFPoEnD+3nZaFkWGVVHlB1\\\\\\\/BZYTIjuIUH4ZfdG+SoScotgHudAUHPYmRoQIxNffid6fsJCikKRAFu3DoHe6ADWxiV42uMVUMFgCXPxPKmqiLzCSotKN3B5FOz83Vlv\\\\\\\/4ZQ1YHO2+5YmE7glwMa7SsAJoNNJgEAjO8FWnRJycG289jTc4XoaM6pmK+aHtot8zZmsa1NHlLgp6bz+ngUnmfD0Xo9xp1vP5F5ukrkU3oEezggKzifbdgB1C9VhT9HwiVSnJGvArn7m7NlP1DCa8dzrgK1BqV1rMKhVonLEN7pN0krnshDERONuuZtNqEUpPyWQEZmRbSfzy6vGPqdrwq24caePg7QP3XTDYOwXmT0ZgQmW7IjJ6rn7SJzG1V+DWJVRTiRdl\\\\\\\/B3xUcPNk0Imah1xByl\\\\\\\/4ZVnUA7tvMpG7sDxyhji1uorGW2qKC9kwSen+wkpGwjAXlOPGRha8u9QDbsJqsP3cNDrVsHkDRr2iM87g3EV8d8t0fQkPkhzQz1J7HtW2TtQeb8\\\\\\\/xQQTKQKmTUEb0rXExlm9qKj4Z64640l6OOrn6BT3BHH43YCvuhg06CSyt10XV+FhttgiNMf50qwITMrB\\\\\\\/hhTyN2OduQeAXh2+GYFcqmfXnaVYka\\\\\\\/eP0zlJ5Lb1xk3BFC2fmsEsSJvYXetTIIiXFoUrpBV36uQYEFvWtyJhN3yTzJTpqVMy7jtJRRLpwvp83kpcGDah8FTLZWicOmEOHUttuYp24rGApFntmsyxUoruYyC4TvEYP69iUfUN0Ib18JVWgKMd6LRdCCAM7kBenBNARCQnWXjrnFBWDJwSJlKfOb7HrJXojqp6VOdqNrByY\\\\\\\/0QyeB4jaIdRbOWvJSmZ6eTiWXcrIRQiHVvprndjh8vG3Er11PhIJ4ahnSsZ+6OSez5k4qOLqahOBGSzUKZ5vN+RJTC8M3efChbdB6lfX1UT34E35SVsnWsqI9NDgq9Zawhc2lJ6+PG+7DKFrpr27\\\\\\\/Fx06O5i77mWMheIYjg7SUzLpJ1ygQFT3t21H6gE4Kh1egbQkhmNRQjdnIRnMHZkgroPqwfY\\\\\\\/xUKp1mBg0pMgh055a5tGQSPv+wo3LUAwwkq27FOyt5ETxjnaxJyB7Xx7YEaYKhjB6Hs3R3SYz52aMYzwi+0w1IBnokp4HxeS2KjXu8hSwf6OxJvdEsPE80OSyXLiQkBbRp18gklk\\\\\\\/1SR8z1t1ucd3fk6N0PgRcaXncsMkr3P8ET8uLKSTbOTjmuaFDssHQzk66AvTBvpme6ft0YgTtD4Jb4gX5TltyrWU59mHOD9pJlGTIriSXs6lYLOZ+obqynXC9zNULJNP1GWBdQYriGWOFRR5fq7\\\\\\\/+MpIKOvrwfCEb0+X4f\\\\\\\/rL\\\\\\\/2Vj3RJZ5N93B+6QXcEpOlN+FvFDQvOpRHlX2NtUBGnwse6K\\\\\\\/upCubKebZd\\\\\\\/adhzTb7ZnCYsT7AmcpKN5LnHPDSJZeh9siuWmAlQ+\\\\\\\/uexuHgiqpTYNfrO9uTggrojFAl0R1Hy56sqIMMvQUSV7vf4+Oe0+erKudy\\\\\\\/128hL2S2MpvjWS2BmwlffAcy57EeOXo\\\\\\\/2ZDKLKUp3jXCduDsBGXWNq5BCdQWsrJZX91hxsKNPsqu1pQJFZN54RR2m2EjkdxX30lXqUJNv39\\\\\\\/5IMRtCw5ZAKfNlY010PbwnfsDQpw2PmrRGUBQhxz\\\\\\\/JyY02pmZqBfznO\\\\\\\/uwadiaCIOKKFbKdikbGVt59w1YKTfdow4J4L+3bawlaQeRin1foXig7yuuQ0cq8KtlbT13StN7Ge3e4rtUw8+m0brB5wXiDw\\\\\\\/5akbRP0p\\\\\\\/xSiZzEzkvLSV34o5jHDJfQBS7PYNLz0G\\\\\\\/pTKmgvmK07HQzhz\\\\\\\/t1gkOlYe1FvMAVmaj\\\\\\\/\\\\\\\/CCdPAnej9qngohdT04BdXFbpjDvDoI\\\\\\\/xJqn+IF1O9pVjKQK90rXveXrJV2zJa+I6kVl+VOxSizYdJEKasduFn0F24kPLRwQIwFDcPNnGgScsn5DqF43XxgdlCV0qD9q2ejzgkamYnoijFfZl2lpcmUjknmG8mTn9NWBN8yjAaLV5uY+mtdXNGdPTHlBsHQPU2DLYNms3o4Mb7DV6eaGrBi+iy2mYo6f0GPWcD7x7RPyWa+CNj6cMQ25GOaP8BCwvUXAoPPBc6S1lDCfAVpXqDvdwgYd7Jdch3hGjk+aQyBs+uMvov8Vg\\\\\\\/\\\\\\\/Ge3TNCRTQ3+DzQMGDMstDNsQwixq2EP3iZAgyY9C39PeFq9ZEXuSnXsApeKCk7ddgVyd1d2irdJnl7QUJpQo0mBwiXUmthMcKU4lPkiVHpbzyrWBRePB9PSHmaSbGhalCUZ\\\\\\\/mIOYoaLgdopj7JrAGEFB71VzHlxjzMpBk0wP16jbQpnGFLkKeAqeyr9IAYV4JLWK5oZstPJ0hbgR\\\\\\\/mVN6QdeKf1\\\\\\\/XTn8TfvAsEkAg9BYIGTOdT6m1sBW4PZJpu\\\\\\\/gTIzDHkfn6EsWAPeml84zYe1oTHpWY9vDXxo2B\\\\\\\/ztKRhEL7l18GrmC5NE7Ra+fzEdJnYciraO5aNGxzpU1AOb31trBlqVqkAMhlK94cMEXroW12TO8uGJ\\\\\\\/15FyKWWFUBExNZXtonfADJ2roToqE9Ll5SXf2rxg\\\\\\\/fT9PnOUhWuKYxBg2rHU6KyW6mFkIXfYMiwAHHwcBt89TGtSLwBcUTCxBLzst16gV3LUk0Xxdn5EmaGcSu1MapEXEl6bG0b7lLiK487EJEZZHpMlvNbVoov0vPNQSy8F\\\\\\\/dq8p8TxqUw8hG21ZrdxNSh\\\\\\\/iID+ouLdLByVl7OlVy13ipoQCtV6Sg7XsPHhO41kfLRd93rDv3iyDyfiL+bkVnFcit68n7H9y+Hyl32XlrKPwrBqXgYyUXp\\\\\\\/hEfPp7fou4cM8v3czZlH7\\\\\\\/IAb0jpv+E9+fYkmtfFZJsWPwPuXSp2RhQOA3adFR4vZxJh5mXVLEZg9sAVN\\\\\\\/DOHKv1Y\\\\\\\/0V9MuF5\\\\\\\/RC69IrghSVMSXiYZNRzgg+SfCeoZUoDggsTdsudj6JPtDAMwjZhWRZF7MmvHAzQ5la5ImqEjbzanbfslgyf16YZ9f19sW2sXx9l3KWnqsTA3LdwxNo5rfVuP7bdYHHbvVaqoXyBL0RQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00224jOQy\\\\\\\/hy+pymtmwX4lsWfw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dcf7b1fcf5fd278944afcfb77929586de0bf5b6ed6fe26d93412dd346e8899ed\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224T2IvXWx8xrDuFDbZxRKd0z8T5CSJGDOApTlB7\\\\\\\/pnjLOzV3wKek3i2zixQRpcGXmSQAbh5MfhhKvtUykLHUJa6DuHtbGheceYWeIIL8ye+Ql+8XQbYw4gKaAUkMZKjgyGYiecH2982RJGGuoS+FojB64pG\\\\\\\/7GDjYe6hpZil7FwzH44eQTEXham\\\\\\\/V7xizCdsBNnKXFQ8ys+zd+4dXrFFQ+hGC3HNV5SGqVmW58tdMFoFi+9xF8hFSJORD0b1iu5m5uKKP+TRyCzr8g3UnWcY7VOrzshPzdoigzHNQTXmgTDKK2STSF0udXr59PeRbccILf78ufFNuwhplETXulnSlQpTRuQcf76hlYNl2EAs0X+NQgAzldx2Khur\\\\\\\/+LPkFHegMGvYmhbP9tpRmCCGQ1EbQCTe9Xf7h+slyuJVutEsBD+guittOqJr\\\\\\\/9qaUfk9tL3yxuZtDp4W8ALaa6Mvo6zLFiWZXqCbBuJvLxZ+mw4KEbPnirljOGyulYJmaZqflmjfxY6pFPozQjHXIFitgk8g7qajQpKu1Lw0ZbWdtg3seq3FfCN2onKfGkAqGmn7bTxnyDB9WMKJzepifMtJ4Iyw25wjSRN\\\\\\\/gaCmQdNRd9iSEupBpVljXRwB1T4qQIxIUvn0JeiU7Czb1LrMKD\\\\\\\/MQiZE7jgVURaMv88T2P4ab5IRddLt7lBDfRbx+nInud0IC3FJPVRD0s2urgJp+lIxBq4bViYWHs6A3AEZvhNGWAD2CKml1o3rANWMEjGQzaYITtjF5J27r7JKMo9Xin0GF9pzFaklOlMp35jvKQQRtErThwJhJvOtEnD7dd7hGaBY1twHCTcLWORjU9IKchYWPL1SPCpEoyrPkldB7QAILjbRKiFQ77rVKb7mJWMdIQjcVY7jVFDQNvbZEo7DQQoYvyCB0ayjGfu7\\\\\\\/IvS\\\\\\\/nRIx91Sr7uIb\\\\\\\/P42c\\\\\\\/YM\\\\\\\/lI\\\\\\\/BQmkQIVA3InJ1mMNvO1dPAOVW9m1QGs3YKxTq\\\\\\\/G00ZecZo7nAX1JkvdqYd5v5QpdaWZww\\\\\\\/vFrX0fSW7hAlhctepn7yTdkZP3V+x543YA7AwB2pt0+a5Ru5WRSmblmLcZ+qjSIt5DzaWXNd2F2BdaDwcO\\\\\\\/JcHAkkC\\\\\\\/vmpeNLsWgZL06uwQ9lqkMm31eriGK2BRwa8NJ\\\\\\\/XXdLxWngY3Uj80y2lTNeJ45vMsnHkTSeh6JUjgD6HmgDI+YGAElUCXpfqFhZ3Mz8\\\\\\\/2D8cAU5Rc0pPfx3O0W58+62MWpXna4d7vF23fOKqVaWmci5d+D3YtLrFYM\\\\\\\/fzdZoc5SzGogWjn1nVP9raAwUY5WQoin2QTHaxmUAxtbBkIw2nuJYHlua5CGFBUgv1Umda66ZczwRWrlqU416Ub9uWryPxZ8lXERIVwBHFxFQh6amvDevOf\\\\\\\/nRxRj6FbOhAIiFR1E5yimBexBAxzLNkrRYldDgiSlM+52xY5C74YGduxGZOD7yM8XZqq9xQ9gPplO\\\\\\\/5FTA0ejnmi+k6StD8zncauliGkh11ztsGNNUqza9CyoHhJi1Jh84sGOtQWXfiBgYJdaKlhbZP\\\\\\\/0DI6Mr2jXwB81LZ4IfKKzbaUuYiznbprpnqwFoFAzs9Y4Aur6qZtCLysmPsbLr\\\\\\\/VkIJNI85jJWhJYlZJiPCz69BMfVoRuqC6mqJctk\\\\\\\/VSnl6okZvuXOOFF0GPJDPYF2P2wDSxIm3xCJs2Qm5OAuc7J8LxZVMnMkO5Kb\\\\\\\/bxn9Bozd5uHEyTg2b\\\\\\\/nAomixIeLQkT+sUkB4nr4dQ6l5hm0zPAkyISICwPFsmw38PatSmxddneMjn4qrl1mQJUPo2sh92q\\\\\\\/7oY69DQ061ehQXvQBLWY8QOj9UT1JYbEebodcXMzU0QB3bKgDMrDUmB8h9x45F11CWCSqVkpqdF8gZEhjMWjs5wtN8fvrXszjr2yfI8d7EQAOFesZYnSLwroYNld6v6yTRRdagjM+0l7CnOBMTHaISAt0q7\\\\\\\/OnUXIkoCRsp6TBve56pHL0WQhj81rAv20W61YG2LkwA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022OZV2OH\\\\\\\/yB1VXH5UX0ut+ww==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bf4d913cc9beb1c528cd3a411437e359472b88d24bac5d641a63c64a8bfdfb9b\u0022}"},"ajax":{"edit-submit--5":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484262 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1551" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-skYydcuTgPSB1m5emBE1F-yNY94R75VgetqlJMEC06E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1551" method="post" id="highwire-search-quicksearch-form-1" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--3">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--3" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--7" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ELzphZFclhmr6rLPRmjrBZ-TXKcpSKnsiviPs8F6xNI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_1" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-3" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1539" title="TheCellMap.org: A Web-Accessible Database for Visualizing and Mining the Global Yeast Genetic Interaction Network" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1563" title="From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (Pinus lambertiana)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484262" id="node484262" data-pisa="ggg;7/5/1551" data-pisa-master="ggg;g3.117.039966" data-apath="/ggg/7/5/1551.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotyping-by-Sequencing Facilitates a High-Density Consensus Linkage Map for <em>Aegilops umbellulata</em>, a Wild Relative of Cultivated Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Erena A. Edae</span>, <span class="highwire-citation-author" data-delta="1">Pablo D. Olivera</span>, <span class="highwire-citation-author" data-delta="2">Yue Jin</span> and <span class="highwire-citation-author" data-delta="3">Matthew N. Rouse</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1551-1561;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039966 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484262" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erena A. Edae </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Cereal Disease Laboratory, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erena%2BA.%2BEdae%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edae%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErena%2BA.%2BEdae%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erena.edae@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">erena.edae@ars.usda.gov</a>
<a href="mailto:matthew.rouse@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">matthew.rouse@ars.usda.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pablo D. Olivera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pablo%2BD.%2BOlivera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Olivera%20PD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APablo%2BD.%2BOlivera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yue Jin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Cereal Disease Laboratory, St. Paul, Minnesota 55108</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yue%2BJin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYue%2BJin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew N. Rouse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Cereal Disease Laboratory, St. Paul, Minnesota 55108</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BN.%2BRouse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rouse%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BN.%2BRouse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erena.edae@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">erena.edae@ars.usda.gov</a>
<a href="mailto:matthew.rouse@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">matthew.rouse@ars.usda.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/5/1551" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484262" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484262/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1551.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484262" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484262/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1551.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484262" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484262/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1551.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484262" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484262/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/5/1551.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484262" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484262/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">High-density genetic maps are useful to precisely localize QTL or genes that might be used to improve traits of nutritional and/or economical importance in crops. However, high-density genetic maps are lacking for most wild relatives of crop species, including wheat. <em>Aegilops umbellulata</em> is a wild relative of wheat known for its potential as a source of biotic and abiotic stress resistance genes. In this work, we have developed a framework consensus genetic map using two biparental populations derived from accessions PI 298905, PI 542369, PI 5422375, and PI 554395. The framework map comprised 3009 genotype-by-sequence SNPs with a total map size of 948.72 cM. On average, there were three SNPs per centimorgan for each chromosome. Chromosome 1U was the shortest (66.5 cM), with only 81 SNPs, whereas the remaining chromosomes had between 391 and 591 SNP markers. A total of 2395 unmapped SNPs were added to the linkage maps through a recombination frequency approach, and increased the number of SNPs placed on the consensus map to a total of 5404 markers. Segregation distortion was disproportionally high for chromosome 1U for both populations used to construct component linkage maps, and thus segregation distortion could be one of the probable reasons for the exceptionally reduced linkage size for chromosome 1U. From comparative analysis, <em>Ae</em>. <em>umbellulata</em> chromosomes except 4U showed moderate to strong collinearity with corresponding homeologous chromosomes of hexaploid wheat and barley. The present consensus map may serve as a reference map in QTL mapping and validation projects, and also in genome assembly to develop a reference genome sequence for <em>Ae. umbellulata</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/consensus-map" class="hw-term hw-article-keyword hw-article-keyword-consensus-map" rel="nofollow">consensus map</a></li><li class="kwd"><a href="/keyword/gbs" class="hw-term hw-article-keyword hw-article-keyword-gbs" rel="nofollow">GBS</a></li><li class="kwd"><a href="/keyword/high-density-linkage-map" class="hw-term hw-article-keyword hw-article-keyword-high-density-linkage-map" rel="nofollow">high-density linkage map</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/aegilops-umbellulata" class="hw-term hw-article-keyword hw-article-keyword-aegilops-umbellulata" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Aegilops umbellulata</em></a></li><li class="kwd"><a href="/keyword/wild-relatives" class="hw-term hw-article-keyword hw-article-keyword-wild-relatives" rel="nofollow">wild relatives</a></li></ul><p id="p-3">Wild relatives of many crop species have been used as sources of useful genes in crop improvement programs (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hajjar and Hodgkin 2007</a>). Several genes derived from wild relatives have already been deployed in cultivated wheat varieties over the past decades, and have played a significant role in wheat improvement worldwide (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Prescott-Allen and Prescott-Allen 1988</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Wulff and Moscou 2014</a>). Species of the genus <em>Aegilops</em> have been used successfully in wheat wide-crossing programs (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zaharieva and Monneveux 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gong <em>et al.</em> 2014</a>), and in the production of several substitution, addition, and translocation lines containing chromosome introgressions from diploid <em>Aegilops</em> species such as <em>Ae. umbellulata</em>, <em>Ae. comosa</em>, <em>Ae. speltoides</em>, and <em>Ae. markgrafii</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jiang <em>et al.</em> 1994</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Friebe <em>et al.</em> 1996</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Schneider <em>et al.</em> 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kilian <em>et al.</em> 2011</a>). <em>Aegilops</em> species are annual grasses, and most of them are considered as self-fertile, with an outcrossing rate similar to that of wheat (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Zaharieva and Monneveux 2006</a>). <em>Aegilops umbellulata</em> (2<em>n</em> = 2<em>x</em> = 14, UU genome), a Mediterranean and western Asiatic grass, is one of the 11 diploid species in the <em>Aegilops</em> genera (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Van Slageren 1994</a>) that possesses seven pairs of chromosomes. <em>Ae. umbellulata</em> has been identified as a source of resistance to stem rust (caused by <em>Puccinia graminis</em> f. sp. <em>tritici</em>) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ozgen <em>et al.</em> 2004</a>), powdery mildew (caused by <em>Blumeria graminis</em> f. sp. <em>tritici</em>), Hessian fly (<em>Mayetiola destuctor</em>), and greenbug (<em>Schizaphis graminum</em>) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gill <em>et al.</em> 1985</a>). It was the species from which leaf rust resistance gene <em>Lr9</em> was transferred to cultivated hexaploid wheat (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sears 1956</a>). Hybridization barriers such as reduced embryo development and stunted growth, or sterility of F<sub>1</sub> hybrids between <em>Ae. umbellulata</em> and <em>Triticum</em> sp. have been overcome through techniques such as embryo rescue and bridge crossing (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Sears 1956</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hadzhiivanova <em>et al.</em> 2012</a>). Consequently, <em>Triticum aestivum–Ae. umbellulata</em> addition lines (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kimber <em>et al.</em> 1967</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Friebe <em>et al.</em> 1996</a>), substitution lines (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Reader and Miller 1987</a>), and translocation lines (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Koebner and Shepherd 1987</a>) have been produced. However, as with all wild relatives of cultivated crops, genomic resources for <em>Ae. umbellulata</em> have been limited, and, until now, suitable molecular markers have not been available for this species (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Molnar <em>et al.</em> 2013</a>). Specifically, <em>Ae. umbellulata</em> is lacking genomic resources such as genetic linkage maps, genome-specific SNP markers, and a reference sequence.</p><p id="p-4">Genetic linkage maps have wide applications in genetics and genomics research. Availability of a genetic linkage map is a prerequisite for QTL mapping (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lander and Bosttein 1989</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Doerge 2002</a>) and map-based gene cloning (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2003</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Saintenac <em>et al.</em> 2013</a>). The availability of linkage maps also allows efficient utilization of the genetic diversity in wild relatives of cultivated crops (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Tanksley and McCouch 1997</a>). Consensus linkage maps are useful to compare locations of QTL identified in independent populations (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Westbrook <em>et al.</em> 2015</a>). There is a wide gap between domesticated crop species and their wild relatives in terms of available genetic resources. Multiple genetic linkage maps have already been developed for cereal crop species, and consensus genetic maps have been constructed for hexaploid wheat (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Somers <em>et al.</em> 2004</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wang <em>et al.</em> 2014</a>), durum wheat (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Maccaferri <em>et al.</em> 2014</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">2015</a>), barley (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2010</a>), and rye (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Milczarski <em>et al.</em> 2011</a>).</p><p id="p-5">Recent advances in high-throughput DNA sequencing technologies have enabled the discovery of a large number of SNPs in major crops such as wheat (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Poland <em>et al.</em> 2012</a>), rice (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">McCough <em>et al.</em> 2016</a>), and maize (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lu <em>et al.</em> 2015</a>). Genotype-by-sequencing (GBS), a complexity-reduction sequencing method for large genomes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elshire <em>et al.</em> 2011</a>), is becoming commonplace in <em>de novo</em> genetic map construction in many species. GBS typically provides a large number of SNP markers per mapping population. High resolution genetic maps that can be created with a large numbers of SNP markers are useful not only to precisely map the position of QTL/genes but also to order scaffolds in a genome assembly as done for hexaploid wheat (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">IWGSC 2014</a>) and barley (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Mayer <em>et al.</em> 2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mascher <em>et al.</em> 2013</a>) for developing high-quality reference genome sequences. Although genome reference sequences are available for many major crops species, orphan crops and wild relatives of the cultivated crops are still lacking ordered reference genome sequences. Here, we report a GBS-based high-density consensus linkage map constructed for <em>Ae. umbellulata</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mapping populations</h3><p id="p-6">Two F<sub>2</sub> <em>Ae. umbellulata</em> populations were used for linkage map construction. The first population (hereafter <em>Aeupop1</em>) consisted of a total of 140 F<sub>2</sub> individuals that were derived from a cross made between two accessions of <em>Ae. umbellulata</em>: PI 298905, originally collected from Manisa, Turkey, and resistant to wheat stem rust pathogen <em>P. graminis</em> f. sp. <em>tritici</em> race TTTTF (isolate 01MN84A-1-2), and PI 542369, originally collected from Usak, Turkey, and susceptible to race TTTTF. Similarly, the second population (hereafter <em>Aeupop2</em>) was derived from a cross made between accessions PI 542375 (originally collected from Denizli, Turkey, and resistant to race TTTTF) and PI 554395 (originally collected from Diyarbakir, Turkey, and susceptible to race TTTTF). There were a total of 154 F<sub>2</sub> individuals derived from this cross.</p></div><div id="sec-3" class="subsection"><h3>GBS library preparation and sequencing</h3><p id="p-7">Leaf tissue was collected from each F<sub>2</sub> individual and the four parents at the seedling stage, and DNA was extracted for each individual following the BioSprint protocol (<a href="https://www.qiagen.com/us/shop/sample-technologies/dna/genomic-dna/biosprint-96-dna-plant-kit/">https://www.qiagen.com/us/shop/sample-technologies/dna/genomic-dna/biosprint-96-dna-plant-kit/</a>). A total of two GBS libraries was constructed from a single pool of samples per population [a pool of 142 samples for <em>Aeupop1</em>, and a pool of 156 samples for <em>Aeupop2</em> following a GBS protocol with the two restriction enzymes <em>Pst</em>I (CTGCAG) and <em>Msp</em>I (CCGG) (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Poland <em>et al.</em> 2012</a>)]. Two bar-coded adaptors were used for each sample. The four parents were sequenced to a depth of 6× compared to that of the F<sub>2</sub> individuals. The library was sequenced on the Illumina HiSequation 2000 platform.</p></div><div id="sec-4" class="subsection"><h3>GBS SNP calling</h3><p id="p-8">Raw sequence data of the two populations were processed together for SNPs discovery with the UNEAK algorithm (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lu <em>et al.</em> 2013</a>) implemented in TASSEL 3.0 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bradbury <em>et al.</em> 2007</a>). The UNEAK parameters were set as follows: maximum number of expected reads per sequence file at 3,000,000,000; restriction enzymes used for library construction were Pst<em>I</em>–<em>Msp</em>I; minimum number of tags required for output was 16; maximum tag number in the merged tag counts was 200,000,000; option to merge multiple sample per line was “yes;” error tolerance rate at 0.01; minimum/maximum minor allele frequencies (MAF) were 0.01 and 0.5; and minimum/maximum call rates were 0 and 1. SNPs with up to 80% missing data points were retained for subsequent data analysis. The discovered SNP data were managed separately for each population for downstream analysis. SNP alleles of the two parents were used to further filter the data where SNPs were required to be homozygous, polymorphic (opposite allele calls in parents), and no missing data points for the two parents of each population. Only SNPs that passed these filtering criteria were converted into parental genotypes.</p></div><div id="sec-5" class="subsection"><h3>Linkage map construction</h3><p id="p-9">SNP data were first converted into parental genotypes for polymorphic SNPs with no missing or heterozygous genotypes for both parents. SNPs with minor allele frequency (MAF) &lt;20%, percent heterozygosity &gt;80%, and proportion of missing data points &gt;10% were also removed. SNPs with distorted segregation at p &lt;0.01 were also removed. A total of 1841 and 1403 SNPs for <em>Aeupop1</em> and <em>Aeupop2</em> passed these filtering criteria, respectively. These quality-filtered SNP markers were retained for framework linkage map construction for each population and consensus map construction. Linkage map construction was done with MSTMap algorithm (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wu <em>et al.</em> 2008</a>) with the following parameters: Distance function Kosambi, cut_off_p_value set at 10<sup>−8</sup>, no_map_dist set at 15, no_map_size set at 0, missing_threshold set at 0.10, estimate_before_clustering set at “no,” detect_bad_data set at “yes,” and objective function set at “ML” for both populations, except that cut_off_p_value was set at 10<sup>−9</sup> for <em>Aeupop1</em>. An inflated linkage map size from the first round of mapping was stabilized using a sliding window approach (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huang <em>et al.</em> 2009</a>) to correct genotypic error. The genotypic data were processed, with modification, with Python written scripts that were implemented in Genotype-Corrector software (Miao, C., J. Fang, P. Liang, X. Zhang, and H. Tang, unpublished data). Linkage maps were also constructed using AntMap software (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Iwata and Ninomiya 2006</a>) using the corrected data sets to confirm the consistency of marker order and genetic positions.</p></div><div id="sec-6" class="subsection"><h3>GBS SNP annotation With gene models and transcripts</h3><p id="p-10">For the purpose of determining the number of genic GBS SNPs, sequence similarity searches were completed for all 41,933 SNP tags of <em>Ae. umbellulata</em> in the local databases created for gene models of hexaploid wheat and barley, and cDNA assemblies of <em>Ae. tauschii</em>, <em>Triticum urartu</em>, and <em>Ae. umbellulata</em>. For all species, maximum <em>e</em>-value of 10<sup>−10</sup> and percent of identity &gt;80% were used as thresholds to define the best hit for each GBS SNP tag. Only one best hit per SNP tag was retained. The retrieved best hits of each species were merged with the SNPs on the consensus map.</p></div><div id="sec-7" class="subsection"><h3>Consensus linkage map construction</h3><p id="p-11">Composite maps from the two populations were constructed for each chromosome using R package LPmerge (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Endelman and Plomion 2014</a>), which uses marker names and genetic distances as input instead of recombination frequencies. With this approach, the component linkage maps are integrated into one map by minimizing the mean absolute error between composite map and component maps using linear programming. A maximum interval size (<em>K</em>) between bins ranging from <em>K</em> = 1 to <em>K</em> = 4 were tested, and the consensus map at each <em>K</em> value was evaluated based on the minimum root mean-squared error (RMSE) and SD between the consensus map and the component linkage maps. Population sizes (139 and 149 after removing individuals with &lt;45% missing marker data) were used to weigh in the objective function. A consensus map at K with minimum mean RMSE and SD was selected for each chromosome. Two consensus maps were constructed: a consensus map constructed using SNP markers with known genetic map position (<em>hereafter framework consensus map</em>), and a consensus map consisting of both markers with known genetic map position and SNP markers placed on the linkage maps using minimum recombination frequency values (hereafter high-density map). For the construction of the high-density consensus map, initially, recombination frequencies were calculated among markers that passed filtering criteria ≤40% missing data points, polymorphic between the parents, and no missing data for both parents. The calculation of recombination frequencies was done separately for each mapping population. For each unmapped marker (SNP with unknown genetic map position), recombination frequency values corresponding to all other mapped markers (SNPs with known genetic map position) were extracted and sorted from minimum to maximum recombination frequency values. Only recombination frequency values ≤0.1 were used to assign the unmapped markers to the closest markers on a linkage map. Each unmapped marker was assigned to the genetic map position of the marker with which it had the smallest recombination frequency value. In cases where two mapped markers had a tie in minimum recombination frequency, the unmapped marker was placed between the two mapped markers using their average genetic distance.</p></div><div id="sec-8" class="subsection"><h3>Assessing the syntenic relationship between Ae. umbellulata and both hexaploid wheat and barley</h3><p id="p-12">Chromosome assignment of the linkage groups of the framework map was accomplished utilizing the draft genome assembly of hexaploid wheat v0.4 and the barley genome assembly (<a href="ftp://ftpmips.helmholtz-muenchen.de/plants/barley/public_data/">ftp://ftpmips.helmholtz-muenchen.de/plants/barley/public_data/</a>). A total of 41,933 <em>Ae. umbellulata</em> SNP tags were similarity-searched using “blastn” in the locally created database for both species assemblies. Only best hits significant at e value of 10<sup>–5</sup> were retrieved for subsequent analysis. Linkage groups were assigned to respective chromosomes of Ae. umbellulata comparing the best hits found for each wheat chromosome. The degree of collinearity of Ae. umbellulata with hexaploid wheat and barley was determined by calculating Spearman’s rank correlation coefficients (<em>ρ</em>) based on the following formula:</p><p id="p-13"><em>ρ</em> = 1− 6∑<em>d<sub>i</sub></em><sup>2</sup>/[<em>n</em>(<em>n</em><sup>2</sup>−1)], where <em>ρ</em> is Spearman’s rank correlation, ∑<em>d<sub>i</sub></em><sup>2</sup> is the summation of rank differences, and <em>n</em> is the number pairs to be ranked.</p></div><div id="sec-9" class="subsection"><h3>Segregation distortion analysis</h3><p id="p-14">Significant deviation from the expected Mendelian genotypic frequencies (1:2:1 genotypic ratio for F<sub>2</sub> population) was tested using chi-square tests (<em>α</em> ≤0.01). We used SNP markers added to the framework map with a recombination frequency approach. The patterns of segregation distortion across chromosome regions were assessed by plotting negative logarithms of p-values [−log<sub>10</sub> (p-values)] <em>vs.</em> consensus genetic distance (centimorgan). The proportion of distorted markers was also calculated for each chromosome. The frequencies of the three genotypic classes were plotted against genetic distance to assess the direction of segregation distortion. Distorted regions were also identified for each chromosome. Distorted regions (DR) were defined as regions of a chromosome with two or more significantly distorted markers (<em>α</em> ≤0.01).</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-15">All necessary data we used to draw conclusions in this article are represented either within the article or within Supplemental Material. Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/TableS1.xlsx">Table S1</a> consists of linkage maps constructed for both populations using MSTMap and AntMap softwares. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/TableS2.xlsx">Table S2</a> contains the high density consensus map, framework map, and physical positions of SNPs in Chinese Spring wheat assembly, and rank correlation coefficient data are also available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/TableS3.docx">Table S3</a>.</p></div></div><div class="section results-discussion" id="sec-11"><h2 class="">Results and Discussion</h2><div id="sec-12" class="subsection"><h3>Component maps and framework map construction</h3><p id="p-16">In this study we report a consensus linkage map constructed based on individual maps of two F<sub>2</sub> populations of <em>Ae. umbellulata</em> using GBS platform. First, the component maps of the two mapping populations were constructed with MSTMap software using quality-filtered genotypic datasets except that the weakly distorted SNPs (<em>α</em> &gt;0.01) were kept in the datasets as segregation distortion is a common phenomenon in many crop species such as maize (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Sharopova <em>et al.</em> 2002</a>), sorghum (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mace <em>et al.</em> 2009</a>), potato (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Van Os <em>et al.</em> 2006</a>), rye (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Senft and Wricke 1996</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Korzun <em>et al.</em> 2001</a>), and triticale (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alheit <em>et al.</em> 2011</a>). By removing SNP markers that were highly distorted (<em>α</em> &lt;0.01), we successfully constructed component linkage maps using two populations for all chromosomes.</p><p id="p-17">For the purpose of assessing the accuracy of the individual maps, we confirmed the consistency of marker order and genetic positions through remapping with an independent mapping algorithm implemented in AntMap software, and found strong agreement between corresponding linkage groups for both populations (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/TableS1.xlsx">Table S1</a>). Then a framework consensus map comprised of 3009 GBS markers of two populations was constructed (<em>Aeupop1</em> and <em>Aeupop2</em>) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of linkage maps constructed with MSTMap and Antmap algorithms for Aeupop1. The x-axis indicates the genetic distance centimeter based on Antmap software, whereas the y-axis indicates the genetic distance centimeter based on MSTMap software for each chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-60292066" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of linkage maps constructed with MSTMap and Antmap algorithms for &lt;em&gt;Aeupop1&lt;/em&gt;. The &lt;em&gt;x-axis&lt;/em&gt; indicates the genetic distance centimeter based on Antmap software, whereas the &lt;em&gt;y-axis&lt;/em&gt; indicates the genetic distance centimeter based on MSTMap software for each chromosome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1551/F1.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/5/1551/F1.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484411" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Comparison of linkage maps constructed with MSTMap and Antmap algorithms for <em>Aeupop1</em>. The <em>x-axis</em> indicates the genetic distance centimeter based on Antmap software, whereas the <em>y-axis</em> indicates the genetic distance centimeter based on MSTMap software for each chromosome.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of linkage maps constructed with MSTMap and Antmap algorithms for Aeupop2. The x-axis indicates the genetic distance (centimorgan) based on Antmap software, whereas the y-axis indicates the genetic distance (centimorgan) based on MSTMap software for each chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-60292066" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of linkage maps constructed with MSTMap and Antmap algorithms for &lt;em&gt;Aeupop2&lt;/em&gt;. The &lt;em&gt;x-axis&lt;/em&gt; indicates the genetic distance (centimorgan) based on Antmap software, whereas the &lt;em&gt;y-axis&lt;/em&gt; indicates the genetic distance (centimorgan) based on MSTMap software for each chromosome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1551/F2.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/5/1551/F2.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484343" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Comparison of linkage maps constructed with MSTMap and Antmap algorithms for <em>Aeupop2</em>. The <em>x-axis</em> indicates the genetic distance (centimorgan) based on Antmap software, whereas the <em>y-axis</em> indicates the genetic distance (centimorgan) based on MSTMap software for each chromosome.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484278/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484278/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Distribution of markers and lengths of the framework consensus genetic map constructed for <em>Ae. umbellulata</em> using two biparental populations</span><div class="sb-div caption-clear"></div></div></div><p id="p-20">The framework consensus linkage map’s size per chromosome was in the range of 66.6 cM (1U)–181 cM (5U), with a total map size of 948.7 cM. Chromosome 1U was consistently found as the shortest of all chromosomes across the mapping populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/TableS1.xlsx">Table S1</a>). It had also the least number of markers with 1.2 SNPs/cM, whereas chromosome 7U had the largest number of markers (3.3 SNPs/cM). The relationship of the framework consensus map with the component linkage maps was assessed by comparing the genetic position of each marker on individual linkage maps with the genetic position of the framework consensus map for each population (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Spearman rank correlation (<em>ρ</em>) values of marker order between consensus map and component maps were calculated for each chromosome for both populations. The result showed that there was strong agreement between component maps and framework consensus maps in marker order for all chromosomes with <em>ρ</em> &gt;0.999 for all chromosomes. Such very high rank correlation coefficients have also been reported for the robust framework consensus map constructed for durum wheat (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Maccaferri <em>et al.</em> 2014</a>). However, some inconsistencies in marker order were detected in each chromosome in the current study, and the corresponding markers were excluded by the algorithm implemented in the LPmerge software during consensus map construction. The probable cause for marker order conflicts may be due to biological factors such as chromosomal segment rearrangements, segmental duplications, and differences in recombination frequencies caused by genomic structural variations between the two mapping populations, as observed in different consensus map construction projects for crop species such as apple (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Khan <em>et al.</em> 2012</a>), sorghum (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Mace <em>et al.</em> 2009</a>), and durum wheat (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Maccaferri <em>et al.</em> 2014</a>). Among the nonbiological factors, missing data and genotypic errors may have contributed to the marker order conflicts observed in the current work.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between Aeupop1 map and consensus map. The x-axis shows the genetic distance (centimorgan) linkage map from Aeupop1, and the y-axis is the genetic distance (centimorgan) of the consensus map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-60292066" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between &lt;em&gt;Aeupop1&lt;/em&gt; map and consensus map. The &lt;em&gt;x-axis&lt;/em&gt; shows the genetic distance (centimorgan) linkage map from &lt;em&gt;Aeupop1&lt;/em&gt;, and the &lt;em&gt;y-axis&lt;/em&gt; is the genetic distance (centimorgan) of the consensus map.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1551/F3.medium.gif" width="440" height="259"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/5/1551/F3.medium.gif" width="440" height="259"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484341" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Relationship between <em>Aeupop1</em> map and consensus map. The <em>x-axis</em> shows the genetic distance (centimorgan) linkage map from <em>Aeupop1</em>, and the <em>y-axis</em> is the genetic distance (centimorgan) of the consensus map.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between Aeupop2 map and consensus map. The x-axis shows the genetic distance (centimorgan) linkage map from Aeupop2, and the y-axis is genetic distance (centimorgan) of the consensus map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-60292066" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between &lt;em&gt;Aeupop2&lt;/em&gt; map and consensus map. The &lt;em&gt;x-axis&lt;/em&gt; shows the genetic distance (centimorgan) linkage map from &lt;em&gt;Aeupop2&lt;/em&gt;, and the &lt;em&gt;y-axis&lt;/em&gt; is genetic distance (centimorgan) of the consensus map.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1551/F4.medium.gif" width="440" height="259"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/5/1551/F4.medium.gif" width="440" height="259"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484350" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-22" class="first-child">Relationship between <em>Aeupop2</em> map and consensus map. The <em>x-axis</em> shows the genetic distance (centimorgan) linkage map from <em>Aeupop2</em>, and the <em>y-axis</em> is genetic distance (centimorgan) of the consensus map.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>High-density consensus map construction and segregation distortion analysis</h3><p id="p-23">Although the framework consensus map was constructed with quality-filtered data [missing data points ≤10%, weak segregation distortion (<em>α</em> &gt; 0.01) and no missing values for the parents], the number of SNPs placed on the framework consensus map (3009 SNPs) represented only 32.67% of the total 9210 SNPs markers that had ≤40% missing data points and were polymorphic in the <em>Aeupop1</em> or/and <em>Aeupop2</em> populations. The framework consensus map had 209 SNPs shared between the two populations, whereas the high-density consensus map was constructed based on 506 SNPs that were common between the two mapping populations. Thus, to leverage the resolution of the consensus map, we were able to place an additional 2395 markers on the component maps of the two populations using the recombination frequency approach (see <em>Materials and Methods</em> section), and the final high-density consensus map comprised a total of 5404 (58.68%) SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/TableS2.xlsx">Table S2</a>). Moreover, these markers added to the linkage maps based on the recombination frequency method enabled us to assess the pattern of segregation distortion for each chromosome of <em>Ae. umbellulata</em>, and also to determine the syntenic relationship between <em>Ae. umbellulata</em> and other species.</p><p id="p-24">The segregation distortion analysis indicated that the proportion of distorted (<em>α</em> ≤0.01) markers was high for chromosome 1U (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), and the extremely distorted markers were located on the proximal end of the chromosome (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Generally, over 60% of the SNPs placed on the high density map via recombination frequency values were significantly distorted for chromosome 1U, indicating segregation distortion was the major factor for the elimination of several SNP markers during filtering steps for linkage map construction for this chromosome. The remaining six chromosomes had &lt;50% distorted markers. The patterns of segregation distortion for all chromosomes are reported in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS5.pdf">Figure S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS6.pdf">Figure S6</a>. With regard to regions with highly distorted markers, with the exception of chromosomes 1U and 7U, which each had a single highly distorted region on the short arm, all other chromosomes possessed two to five distorted regions. The nonrandom distributions of segregation distortion along chromosomal regions was previously reported for hexaploid wheat (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2015</a>), tetraploid wheat (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kumar <em>et al.</em> 2007</a>), barley (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2010</a>), <em>Aegilops tauschii</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Faris <em>et al.</em> 1998</a>), and oak (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bodenes <em>et al.</em> 2016</a>), as well as many other plant species. Segregation distortion can occur either due to nonbiological factors such as sampling error, missing data points, and genotypic error, or due to biological factors such as gamete competition, hybrid incompatibility, deleterious alleles (genetic load), and chromosome loss or rearrangements (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Tayler and Ingvarsson 2003</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alheit <em>et al.</em> 2011</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bodenes <em>et al.</em> 2016</a>). Although the effect of technical factors cannot be ruled out, especially for weakly distorted markers that were randomly distributed across the genome, clusters of significantly distorted markers in the previously reported distorted regions for many grass species are most likely caused by genetic factors. Chromosome 1U, with the most severe segregation distortion, is homeologous to wheat group 1 chromosomes (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Zhang <em>et al.</em> 1998</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Molnar <em>et al.</em> 2016</a>) and barley chromosome 1H (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Edae <em>et al.</em> 2016</a>). Segregation distortion regions have been reported on the short arms of wheat group 1 chromosome (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2015</a>), <em>Ae. tauschii</em> 1DS (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Faris <em>et al.</em> 1998</a>), and on barley 1H chromosome (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2010</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Belanger <em>et al.</em> 2016</a>). Although the current study was not designed to study mechanisms that cause segregation distortion, the analysis conducted here is rather to determine the effect of segregation distortion regions on linkage map construction. As expected, linkage groups with severe segregation distortion (<em>e.g.</em>, 1U) had fewer markers, were shorter in size, and displayed inconsistent marker distribution. We also conducted similar analysis for missing data, but we have not observed noticeable effects of missing data amount on quality of linkage maps (data not shown). As to the direction of segregation distortion, in most cases, the frequency of heterozygous genotypes was either lower than the frequency of homozygous genotypes, or between the two parental frequencies (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). This could be partly due to the presence of heterozygous undercalling due to absence of deeper sequence coverage to sample both alleles in heterozygous individuals in the data sets of both mapping populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS7.pdf">Figure S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS8.pdf">Figure S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS9.pdf">Figure S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS10.pdf">Figure S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS11.pdf">Figure S11</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS12.pdf">Figure S12</a>). Overall, although it is difficult to distinguish the type of biological factors involved in segregation distortion from this study, the analysis elucidated that segregation distortion is one of the factors that adversely affected the linkage map for chromosome 1U, which was found to be the shortest chromosome with the least number of markers.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of significantly distorted markers (p&#xA0;&#x2264;&#xA0;0.01) of SNP markers placed on the high density consensus map for each chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-60292066" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Proportion of significantly distorted markers (p&#xA0;&#x2264;&#xA0;0.01) of SNP markers placed on the high density consensus map for each chromosome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1551/F5.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/5/1551/F5.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484266" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-25" class="first-child">Proportion of significantly distorted markers (p ≤ 0.01) of SNP markers placed on the high density consensus map for each chromosome.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pattern of segregation distortion across regions of chromosome&#xA0;1&#xA0;U. The y-axis indicates the segregation distortion p-value (in its negative logarithm form) for the corresponding genetic position on the x-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-60292066" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pattern of segregation distortion across regions of chromosome&#xA0;1&#xA0;U. The &lt;em&gt;y-axis&lt;/em&gt; indicates the segregation distortion p-value (in its negative logarithm form) for the corresponding genetic position on the &lt;em&gt;x-axis&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1551/F6.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/5/1551/F6.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484363" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-26" class="first-child">Pattern of segregation distortion across regions of chromosome 1 U. The <em>y-axis</em> indicates the segregation distortion p-value (in its negative logarithm form) for the corresponding genetic position on the <em>x-axis</em>.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pattern of genotypic frequencies across regions of chromosome&#xA0;1U. The y-axis indicates the genotypic frequencies of the three genotypic classes (AA, AB, and BB) for the corresponding genetic position on the x-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-60292066" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pattern of genotypic frequencies across regions of chromosome&#xA0;1U. The &lt;em&gt;y-axis&lt;/em&gt; indicates the genotypic frequencies of the three genotypic classes (AA, AB, and BB) for the corresponding genetic position on the &lt;em&gt;x-axis&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1551/F7.medium.gif" width="440" height="395"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/5/1551/F7.medium.gif" width="440" height="395"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484289" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-27" class="first-child">Pattern of genotypic frequencies across regions of chromosome 1U. The <em>y-axis</em> indicates the genotypic frequencies of the three genotypic classes (AA, AB, and BB) for the corresponding genetic position on the <em>x-axis</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Chromosome collinearity and synteny across Ae. umbellulata, wheat, and barley</h3><p id="p-28">The draft sequence of hexaploid wheat has greatly facilitated the assignment of linkage groups into chromosomes. The SNP tags for the markers in the high-density map (5404 markers) were aligned against the Chinese Spring assembly v0.4 (<a href="https://wheat-urgi.versailles.inra.fr/Seq-Repository/Assemblies">https://wheat-urgi.versailles.inra.fr/Seq-Repository/Assemblies</a>). With this approach, we were able to assess broadly the syntenic relationship between <em>Ae. umbellulata</em> and hexaploid wheat, and also assigned all linkage groups into <em>Ae. umbellulata</em> chromosomes unambiguously using hexaploid wheat-anchored markers. A total of 3649 (67.52%) of the SNPs on the <em>Ae. umbellulata</em> high-density consensus map had unique best hits in the hexaploid genome scaffolds. Out of these SNPs, 59.69% were mapped to the expected homeologous hexaploid wheat chromosomes, <em>i.e.</em>, they showed syntentic relationships with the corresponding group of chromosomes (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). With the exception of chromosome 4U, all <em>Ae. umbellulata</em> chromosomes showed strong collinearity with corresponding homeologous chromosomes of wheat. However, chromosome 4U consisted mainly of segments of group 1, group 6, and group 7 chromosomes (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>). Similarly, the SNP tags of the markers in the high-density map were integrated again into the barley genome assembly (<a href="ftp://ftpmips.helmholtz-muenchen.de/plants/barley/public_data/">ftp://ftpmips.helmholtz-muenchen.de/plants/barley/public_data/</a>). A total of 802 (14.84%) SNPs had unique best hits in the barley genome assembly, and 510 (63.59%) of them were found to be syntenous. Broadly, the comparison of <em>Ae. umbellulata</em> with barley chromosomes revealed a similar syntenic pattern with that of <em>Ae. umbellulata</em> and hexaploid wheat (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS13.pdf">Figure S13</a>). Chromosomal structural rearrangements are common in <em>Ae. umbellulata</em>, and at least 11 rearrangements have been identified with an RFLP-based genetic map between the wheat D genome and <em>Ae. umbellulata</em> chromosomes (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Zhang <em>et al.</em> 1998</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Devos and Gale 2000</a>). We also detected the majority of those previously reported chromosome translocations and inversions. The major ones are translocations of short arms of 3U to 7U, the similarity of 4U with 5U, 6U, and 7U, and the chromosome inversion that spans the long arm of 2U. In addition, it appears that chromosome 6U is composed of large segments of chromosomal translocations from both 4U and 5U (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>). For chromosomes 5U and 7U, we have also noticed the presence of chromosome inversions that have not been reported previously for <em>Ae. umbellulata</em>. Although we detected numerous relationships between nonhomeologous chromosomes based on single markers, we rather focused on regions that were supported by several markers as true indicators of chromosomal rearrangements as single marker-based relationship may be due to technical errors during linkage map construction and/or sequence alignment.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Syntenic relationship between the homeologous chromosomes of Ae. umbellulata and hexaploid wheat. The left y-axis shows Chinese Spring wheat physical map (in million base pair), and the bottom x-axis identifies genetic distance (centimorgan) of Ae. umbellulata chromosomes. The upper x-axis represents chromosomes of Chinese Spring wheat, whereas the left y-axis represents chromosomes of Ae. umbellulata." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-60292066" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Syntenic relationship between the homeologous chromosomes of &lt;em&gt;Ae. umbellulata&lt;/em&gt; and hexaploid wheat. The left &lt;em&gt;y-axis&lt;/em&gt; shows Chinese Spring wheat physical map (in million base pair), and the bottom &lt;em&gt;x&lt;/em&gt;-axis identifies genetic distance (centimorgan) of &lt;em&gt;Ae. umbellulata&lt;/em&gt; chromosomes. The upper &lt;em&gt;x-axis&lt;/em&gt; represents chromosomes of Chinese Spring wheat, whereas the left &lt;em&gt;y-axis&lt;/em&gt; represents chromosomes of &lt;em&gt;Ae. umbellulata&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1551/F8.medium.gif" width="410" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/5/1551/F8.medium.gif" width="410" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1551/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484311" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-29" class="first-child">Syntenic relationship between the homeologous chromosomes of <em>Ae. umbellulata</em> and hexaploid wheat. The left <em>y-axis</em> shows Chinese Spring wheat physical map (in million base pair), and the bottom <em>x</em>-axis identifies genetic distance (centimorgan) of <em>Ae. umbellulata</em> chromosomes. The upper <em>x-axis</em> represents chromosomes of Chinese Spring wheat, whereas the left <em>y-axis</em> represents chromosomes of <em>Ae. umbellulata</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">The use of alien segments that carry novel genes depends on the extent of chromosome collinearity and meiotic pairing of alien chromosomes with homeologous chromosomes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Devos <em>et al.</em> 1993</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lukaszewski <em>et al.</em> 2004</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Devos et al. 2007</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Molnar <em>et al.</em> 2016</a>). In the current study, the level of collinearity of <em>Ae. umbellulata</em> chromosomes with homeologous chromosomes of hexaploid wheat and barley was assessed using Spearman’s rank correlation coefficients. In this line, we observed a wide range of variations among pairs of homeologous chromosomes both for hexaploid wheat and barley. Chromosomes 3U and Group 3 of hexaploid wheat (3A, 3B, and 3D) had very strong collinearity with rank correlation coefficient, <em>ρ</em> &gt;0.9, followed by 1U and group 1 of hexaploid wheat, 6U, and group 6 of hexaploid wheat, with <em>ρ</em> values ranging from 0.6 to 0.9. Hexaploid wheat group 5 and 7 chromosomes showed low to moderated collinearity with <em>Ae. umbellulata</em> counterpart chromosomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS13.pdf">Figure S13</a>). Using rank correlation, a similar pattern of collinearity was also observed between homeologous chromosomes of <em>Ae. umbellulata</em> and barley. Chromosomes 4U has no relationship with its homeologous chromosomes of hexaploid wheat as it shared multiple segments with other nonhomeologous chromosomes such as group 1, 6, and 7 of hexaploid wheat, and this was confirmed again with its chromosome collinearity with 1H, 6H, and 7H of barley (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1/FigureS13.pdf">Figure S13</a>). The current results also agree with our previous analysis based on the data from a single population (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Edae <em>et al.</em> 2016</a>).</p></div><div id="sec-15" class="subsection"><h3>GBS SNP annotation with gene models and transcripts</h3><p id="p-31">Similarity search of GBS SNP tags in gene models and cDNA libraries of other cereal crops revealed that ∼6% of the <em>Ae. umbellulata</em> SNPs were in genic regions (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). However, through integration of GBS tags into transcript assembly of <em>Ae. umbellulata</em> itself indicated that ∼20% of GBS SNPs are genic SNPs. GBS genic SNP estimation has also been estimated for oat (5%) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Huang <em>et al.</em> 2014</a>) and barley (10.2%) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kono <em>et al.</em> 2013</a>). In the current work, we found similar results as to the integration of GBS SNPs into other species, but the higher genic GBS SNPs from <em>Ae. umbellulata</em> transcripts is likely because tissue sources for the transcript assemblies of the latter were from two parents of the four parents of the mapping populations used in this study. Nonetheless, the presence of GBS SNPs in the expressed portion of a genome implies that there is a possibility to identify functional SNPs, especially with high density SNPs, through QTL mapping and association mapping approaches using GBS technology.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484385/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484385/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Similarity of <em>Ae. umbellulata</em> SNP tags with gene models and cDNA transcripts of <em>Ae. umbellulata</em>, hexaploid wheat, barley, <em>Ae. tauschii</em>, and <em>T. urartu</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-32">The use of two mapping populations allowed us to place a high number of GBS SNPs on a single consensus map. To date this is the only high-density consensus map constructed for <em>Ae. umbellulata</em>. The consensus map is developed with the belief that its availability will greatly facilitate genomic/genetics studies including the identification of novel QTL, refining QTL positions, gene cloning and genome assembly in <em>Ae. umbellulata</em>. Specifically, the mapping of novel stem rust resistance from this species has already started with the first population (<em>Aeupop1</em>) and a major QTL for two stem rust races has been detected on chromosome 2U (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Edae <em>et al.</em> 2016</a>). Therefore, mapping resistance QTL for the same races will be done using the second population (<em>Aeupop2</em>). We hope that this consensus map will help to confirm and refine the genetic position of the mapped QTL as the use of GBS-based linkage maps for precise QTL mapping and facilitation of gene cloning for different traits has been shown in several previous studies (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kujur <em>et al.</em> 2016</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Pootakham <em>et al.</em> 2015</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Balsalobre <em>et al.</em> 2017</a>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-33">Computational resources for this project were accessed through the Minnesota Supercomputing Institute (MSI) high performance computing (HPC) at the University of Minnesota. Mention of trade names or commercial products in this publication is solely for the purpose of providing specific information, and does not imply recommendation or endorsement by the United States Department of Agriculture (USDA). USDA is an equal opportunity provider and employer.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-34">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.039966/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-35"><em>Communicating editor: A. H. Paterson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-25"><span class="received-label">Received </span>January 25, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-03-20"><span class="accepted-label">Accepted </span>March 20, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Edae <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.1" data-doi="10.1186/1471-2164-12-380"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alheit</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissmann</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Detection of segregation distortion loci in <em>triticale</em> (x <em>Triticosecale Wittmack</em>) based on a high-density DArT marker consensus genetic linkage map.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAlheit%26rft.auinit1%253DK.%2BV.%26rft.volume%253D12%26rft.spage%253D380%26rft.epage%253D380%26rft.atitle%253DDetection%2Bof%2Bsegregation%2Bdistortion%2Bloci%2Bin%2Btriticale%2B%2528x%2BTriticosecale%2BWittmack%2529%2Bbased%2Bon%2Ba%2Bhigh-density%2BDArT%2Bmarker%2Bconsensus%2Bgenetic%2Blinkage%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-380%26rft_id%253Dinfo%253Apmid%252F21798064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-380&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21798064&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balsalobre</span>  <span class="cit-name-given-names">T. W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">G. D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margarido</span>  <span class="cit-name-given-names">G. R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gazaffi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barreto</span>  <span class="cit-name-given-names">F. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">GBS-based single dosage markers for linkage and QTL mapping allow gene mining for yield-related traits in sugarcane.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D72%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belanger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clermont</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esteves</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belzile</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Extent and overlap of segregation distortion regions in 12 barley crosses determined via a pool-GBS approach.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1393</span>–<span class="cit-lpage">1404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.4" data-doi="10.1093/dnares/dsw001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bodenes</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chancerel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenmann</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">High-density linkage mapping and distribution of segregation distortion regions in the oak genome.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">115</span>–<span class="cit-lpage">124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsw001%26rft_id%253Dinfo%253Apmid%252F27013549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIzLzIvMTE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.5" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTkvMjYzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.6" data-doi="10.1038/nrg703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mapping and analysis of quantitative trait loci in experimental populations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">43</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDoerge%26rft.auinit1%253DR.%2BW.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D52%26rft.atitle%253DMapping%2Band%2Banalysis%2Bof%2Bquantitative%2Btrait%2Bloci%2Bin%2Bexperimental%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg703%26rft_id%253Dinfo%253Apmid%252F11823790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11823790&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173267200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.7" data-doi="10.1007/BF00225004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atkinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chinoy</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harcourt</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koebner</span>  <span class="cit-name-given-names">R. M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Chromosomal rearrangements in the rye genome relative to that of wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D85%26rft.spage%253D673%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00225004%26rft_id%253Dinfo%253Apmid%252F24196035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00225004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24196035&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KP83700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atkinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chinoy</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francis</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harcourt</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Chromosomal rearrangements in the rye genome relative to that of wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D85%26rft.spage%253D673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.9" data-doi="10.1105/tpc.12.5.637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genome relationships: The grass model in current research.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DDevos%26rft.auinit1%253DK.%2BM.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D637%26rft.epage%253D646%26rft.atitle%253DGenome%2BRelationships%253A%2BThe%2BGrass%2BModel%2Bin%2BCurrent%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.12.5.637%26rft_id%253Dinfo%253Apmid%252F10810140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjEyLzUvNjM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edae</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olivera</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouse</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genotype-by-sequencing facilitates genetic mapping of a stem rust resistance locus in <em>Aegilops umbellulata</em>, a wild relative of cultivated wheat.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D1039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.11" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotype-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.12" data-doi="10.1093/bioinformatics/btu091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">LPmerge: an R package for merging genetic maps by linear programming.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu091%26rft_id%253Dinfo%253Apmid%252F24532720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTEvMTYyMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laddomada</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Molecular mapping of segregation distortion loci in <em>Aegilops tauchii</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFaris%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D149%26rft.issue%253D1%26rft.spage%253D319%26rft.epage%253D327%26rft.atitle%253DMolecular%2BMapping%2Bof%2BSegregation%2BDistortion%2BLoci%2Bin%2BAegilops%2Btauschii%26rft_id%253Dinfo%253Apmid%252F9584106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ5LzEvMzE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friebe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raup</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntosh</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Characterization of wheat alien translocations conferring resistance to diseases and pests: current status.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D71%26rft.spage%253D59%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raup</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browder</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatchet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Evaluation of <em>Aegilops</em> species for resistance to wheat powdery mildew, wheat leaf rust and Hessian fly and green bug.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">314</span>–<span class="cit-lpage">316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D69%26rft.spage%253D314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cytogenetic and molecular markers in a wheat background.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">489</span>–<span class="cit-lpage">497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D57%26rft.spage%253D489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1551.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hadzhiivanova</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bozhanova</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dechev</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Interspecific hybridization between durum wheat and Ae. umbellulata(Zhuk).</span> <span class="cit-source">Bulg. J. Agric. Sci.</span> <span class="cit-vol">18</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">721</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulg.%2BJ.%2BAgric.%2BSci.%26rft.volume%253D18%26rft.spage%253D713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.18" data-doi="10.1007/s10681-007-9363-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hajjar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodgkin</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The use of wild relatives in crop improvement: a survey of developments over the last 20 years.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D156%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-007-9363-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-007-9363-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000246750400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fellers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trick</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Map-based cloning of leaf rust resistance gene <em>Lr21</em> from the large and polyploid genome of bread wheat.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">655</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHuang%26rft.auinit1%253DL.%26rft.volume%253D164%26rft.issue%253D2%26rft.spage%253D655%26rft.epage%253D664%26rft.atitle%253DMap-Based%2BCloning%2Bof%2BLeaf%2BRust%2BResistance%2BGene%2BLr21%2BFrom%2Bthe%2BLarge%2Band%2BPolyploid%2BGenome%2Bof%2BBread%2BWheat%26rft_id%253Dinfo%253Apmid%252F12807786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzIvNjU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.20" data-doi="10.1101/gr.089516.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-throughput genotyping by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089516.108%26rft_id%253Dinfo%253Apmid%252F19420380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA2OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.21" data-doi="10.1371/journal.pone.0102448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wight</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Using Genotype-by-sequencing (GBS) for genomic discovery in cultivated oat.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">e102448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De102448%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0102448%26rft_id%253Dinfo%253Apmid%252F25047601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0102448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25047601&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.22" data-doi="10.1270/jsbbs.56.371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ninomiya</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">AntMap: constructing genetic linkage maps using an ant colony optimization algorithm.</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D56%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1270%252Fjsbbs.56.371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1270/jsbbs.56.371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000243982400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-other" id="cit-7.5.1551.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">IWGSC, The International Wheat Genome Sequencing Consortium</span></li></ol><cite>. <span class="cit-pub-date">2014</span> <span class="cit-article-title">A chromosome-based draft sequence of the hexaploid bread wheat (<em>Triticum aestivum</em>) genome.</span> <span class="cit-source">Science</span> <span class="cit-vol">345</span>(<span class="cit-issue">6194</span>): <span class="cit-fpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D345%26rft.spage%253D286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.24" data-doi="10.1007/BF00036700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friebe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Recent advances in alien gene transfer in wheat.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D73%26rft.spage%253D199%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00036700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00036700&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994NK38500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1551.25" data-doi="10.1371/journal.pone.0047864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korban</span>  <span class="cit-name-given-names">S. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A multi-population consensus genetic map reveals inconsistent marker order among maps likely attributed to structural variations in the apple genome.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">7</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e47864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKhan%26rft.auinit1%253DM.%2BA.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De47864%26rft.epage%253De47864%26rft.atitle%253DA%2BMulti-Population%2BConsensus%2BGenetic%2BMap%2BReveals%2BInconsistent%2BMarker%2BOrder%2Bamong%2BMaps%2BLikely%2BAttributed%2Bto%2BStructural%2BVariations%2Bin%2Bthe%2BApple%2BGenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0047864%26rft_id%253Dinfo%253Apmid%252F23144832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0047864&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23144832&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.26" data-doi="10.1017/S0016672300010351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimber</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">The addition of the chromosomes of <em>Ae. umbellulata</em> to <em>Triticum aestivum</em> var. Chinese Spring.</span> <abbr class="cit-jnl-abbrev">Gen Res Camb</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGen%2BRes%2BCamb%26rft.volume%253D9%26rft.spage%253D111%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1551.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mammen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millet</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Aegilops</span>, pp. 1–76 in <span class="cit-source">Wild Crop Relatives: Genomic and breeding resources</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kole</span>  <span class="cit-name-given-names">C.</span></span>. <span class="cit-publ-name">Spring-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koebner</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shepherd</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Allosyndetic recombination between a chromosome of <em>Ae. umbellulata</em> and wheat chromosomes.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D59%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kono</span>  <span class="cit-name-given-names">T. J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seth</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrel</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SNPMeta: SNP annotation and SNP metadata collection without a reference genome.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D14%26rft.spage%253D419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.30" data-doi="10.1007/s001220051701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malyshev</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voylokov</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Börner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A genetic map of rye (<em>Secale cereale</em> L.) combining RFLP, isozyme, protein, microsatellite and gene loci.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D102%26rft.spage%253D709%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000168403700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kujur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bajaj</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identification of candidate genes and natural allelic variants for QTLs governing plant height in chickpea.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">27968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D27968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.32" data-doi="10.1007/s00438-007-0248-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Identification and characterization of segregation distortion loci along chromosome 5B in tetraploid wheat.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DKumar%26rft.auinit1%253DS.%26rft.volume%253D278%26rft.issue%253D2%26rft.spage%253D187%26rft.epage%253D196%26rft.atitle%253DIdentification%2Band%2Bcharacterization%2Bof%2Bsegregation%2Bdistortion%2Bloci%2Balong%2Bchromosome%2B5B%2Bin%2Btetraploid%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-007-0248-7%26rft_id%253Dinfo%253Apmid%252F17520291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-007-0248-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17520291&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping Mendelian Factors Underlying Quantitative Traits Using RFLP Linkage Maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.34" data-doi="10.1155/2015/830618"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A high-density SNP and SSR consensus map reveals segregation distortion regions in wheat.</span> <abbr class="cit-jnl-abbrev">Biomed Res. Int.</abbr> <span class="cit-vol">2015</span>: <span class="cit-fpage">830618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1155/2015/830618</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiomed%2BRes.%2BInt.%26rft.volume%253D2015%26rft.spage%253D830618%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2015%252F830618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2015/830618&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.35" data-doi="10.1007/s00438-010-0570-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wenzi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huttner</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Construction of a high-density composite mapping of segregation distortion regions in barley.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D284%26rft.issue%253D5%26rft.spage%253D319%26rft.epage%253D331%26rft.atitle%253DConstruction%2Bof%2Ba%2Bhigh-density%2Bcomposite%2Bmap%2Band%2Bcomparative%2Bmapping%2Bof%2Bsegregation%2Bdistortion%2Bregions%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-010-0570-3%26rft_id%253Dinfo%253Apmid%252F20803217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-010-0570-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20803217&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-web" id="cit-7.5.1551.36" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.</span> <span class="cit-source">PLoS Genet.</span> <span class="cit-vol">9</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e1003215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.37" data-doi="10.1038/ncomms7914"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-resolution genetic mapping of maize pan-genome sequence anchors.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6914</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D6914%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms7914%26rft_id%253Dinfo%253Apmid%252F25881062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms7914&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25881062&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lukaszewski</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rybka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malyshev</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapinski</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic and physical mapping of homoeologous recombination points involving wheat chromosome 2B and rye chromosome 2R.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLukaszewski%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D45%26rft.atitle%253DGenetic%2Band%2Bphysical%2Bmapping%2Bof%2Bhomoeologous%2Brecombination%2Bpoints%2Binvolving%2Bwheat%2Bchromosome%2B2B%2Band%2Brye%2Bchromosome%2B2R.%26rft_id%253Dinfo%253Apmid%252F15060600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15060600&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.39" data-doi="10.1186/1471-2229-9-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rami</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A consensus genetic map of sorghum that integrates multiple component maps and high throughput Diversity Array Technology (DArT) markers.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DMace%26rft.auinit1%253DE.%2BS.%26rft.volume%253D9%26rft.spage%253D13%26rft.epage%253D13%26rft.atitle%253DA%2Bconsensus%2Bgenetic%2Bmap%2Bof%2Bsorghum%2Bthat%2Bintegrates%2Bmultiple%2Bcomponent%2Bmaps%2Band%2Bhigh-throughput%2BDiversity%2BArray%2BTechnology%2B%2528DArT%2529%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-13%26rft_id%253Dinfo%253Apmid%252F19171067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19171067&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.40" data-doi="10.1111/tpj.12319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muehlbauer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokhsar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Anchoring and ordering NGS contig assemblies by population sequencing (POPSEQ).</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">718</span>–<span class="cit-lpage">727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D76%26rft.spage%253D718%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12319%26rft_id%253Dinfo%253Apmid%252F23998490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23998490&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326138100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.41" data-doi="10.1186/1471-2164-15-873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maccaferri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cane</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanguineti</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salvi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colalong</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A consensus framework map of durum wheat (<em>Triticum durum</em> Desf.) suitable for linkage disequilibrium analysis and genome-wide association mapping.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D873%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-873%26rft_id%253Dinfo%253Apmid%252F25293821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-873&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25293821&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.42" data-doi="10.1111/pbi.12288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maccaferri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricci</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salvi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milner</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A high-density, SNP-based consensus map of tetraploid wheat as a bridge to integrate durum and bread wheat genomics and breeding.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D13%26rft.spage%253D648%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12288%26rft_id%253Dinfo%253Apmid%252F25424506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25424506&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waugh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Physical, genetic and functional sequence assembly of the barley genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMayer%26rft.auinit1%253DK.%2BF.%26rft.volume%253D491%26rft.issue%253D7426%26rft.spage%253D711%26rft.epage%253D716%26rft.atitle%253DA%2Bphysical%252C%2Bgenetic%2Band%2Bfunctional%2Bsequence%2Bassembly%2Bof%2Bthe%2Bbarley%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F23075845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23075845&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311606000035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.44" data-doi="10.1038/ncomms10532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCough</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maron</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNally</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Open access resources for genome-wide association mapping in rice.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">10532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D10532%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms10532%26rft_id%253Dinfo%253Apmid%252F26842267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms10532&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26842267&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.45" data-doi="10.1371/journal.pone.0028495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milczarski</span>  <span class="cit-name-given-names">P</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolibok-Bragoszewska</span>  <span class="cit-name-given-names">H</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myskow</span>  <span class="cit-name-given-names">B</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stojalowski</span>  <span class="cit-name-given-names">S</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heller-Uszynska</span>  <span class="cit-name-given-names">K</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A high density consensus map of rye (<em>Secale cereale</em> L.) based on DArT markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e28495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMilczarski%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28495%26rft.epage%253De28495%26rft.atitle%253DA%2Bhigh%2Bdensity%2Bconsensus%2Bmap%2Bof%2Brye%2B%2528Secale%2Bcereale%2BL.%2529%2Bbased%2Bon%2BDArT%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028495%26rft_id%253Dinfo%253Apmid%252F22163026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22163026&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.46" data-doi="10.1371/journal.pone.0070844"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Molnar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imkova</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leverington-Waite</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goram</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cseh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Syntenic relationships between the U and M genomes of Aegilops, wheat and the model species <em>Brachypodium</em> and rice as revealed by COS markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">e70844</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De70844%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0070844%26rft_id%253Dinfo%253Apmid%252F23940651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0070844&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23940651&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Molnar</span>  <span class="cit-name-given-names">I</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vrana</span>  <span class="cit-name-given-names">J</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buresova</span>  <span class="cit-name-given-names">V</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capal</span>  <span class="cit-name-given-names">P</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farkas</span>  <span class="cit-name-given-names">A</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dark</span>  <span class="cit-name-given-names">E</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Dissecting the U, M, S and C genomes of wild relatives of bread wheat (Aegilops spp.) into chromosomes and exploring their synteny with wheat.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D88%26rft.spage%253D452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozgen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yildiz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulukan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koyuncu</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Association of gliadin protein pattern and rust resistance derived from <em>Aegilops ubmellulata</em> Zhuk. In winter <em>Triticum durum</em> Desf.</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D54%26rft.spage%253D287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D2%26rft.spage%253De32253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pootakham</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jomchai</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruang-areerate</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shearman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonthirod</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide SNP discovery and identification of QTL associated with agronomic traits in oil palm using genotyping-by-sequencing (GBS).</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D105%26rft.spage%253D288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-other" id="cit-7.5.1551.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prescott-Allen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prescott-Allen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-source">Genes from the Wild: Using Wild Genetic Resources for Food and Raw materials</span>. <span class="cit-publ-name">International Institute for Environment and Development</span>, <span class="cit-publ-loc">London.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reader</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">T. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The simultaneous substitution of two pairs of chromosomes from two alien species in <em>Triticum aestivum</em> cv. Chinese Spring.</span> <abbr class="cit-jnl-abbrev">Cereal Res. Comm.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCereal%2BRes.%2BComm.%26rft.volume%253D15%26rft.spage%253D39%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.53" data-doi="10.1126/science.1239022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saintenac</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salcedo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trick</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of wheat gene Sr35 that confers resistance to Ug99 stem rust race group.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">341</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1239022%26rft_id%253Dinfo%253Apmid%252F23811222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDEvNjE0Ny83ODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTU1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.54" data-doi="10.1007/s10681-007-9624-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molnar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molnar-Lang</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Utilization of <em>Aegilops</em> (goatgrass) species to widen the genetic diversity of cultivated wheat.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D163%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-007-9624-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-007-9624-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000257921500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sears</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1956</span> <span class="cit-article-title">The transfer of leaf rust resistance from <em>Aegilops umbellulata</em> to wheat.</span> <abbr class="cit-jnl-abbrev">Brookhaven Symp. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrookhaven%2BSymp.%2BBiol.%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Senft</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wricke</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">An extended genetic map of rye (<em>Secale cereale</em> L.).</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">508</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D115%26rft.spage%253D508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.57" data-doi="10.1023/A:1014868625533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharopova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Villeda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Development and mapping of SSR markers for maize.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DSharopova%26rft.auinit1%253DN.%26rft.volume%253D48%26rft.issue%253D5-6%26rft.spage%253D463%26rft.epage%253D481%26rft.atitle%253DDevelopment%2Band%2Bmapping%2Bof%2BSSR%2Bmarkers%2Bfor%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1014868625533%26rft_id%253Dinfo%253Apmid%252F12004892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1014868625533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12004892&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174606400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.58" data-doi="10.1007/s00122-004-1740-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Somers</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isaac</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A high-density microsatellite consensus map of bread wheat (<em>Triticum aestivum</em>).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSomers%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D109%26rft.issue%253D6%26rft.spage%253D1105%26rft.epage%253D1114%26rft.atitle%253DA%2Bhigh-density%2Bmicrosatellite%2Bconsensus%2Bmap%2Bfor%2Bbread%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1740-7%26rft_id%253Dinfo%253Apmid%252F15490101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1740-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15490101&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224614100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.59" data-doi="10.1126/science.277.5329.1063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Seed banks and molecular maps: unlocking genetic potential from the wild.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">1063</span>–<span class="cit-lpage">1066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTanksley%26rft.auinit1%253DS.%2BD.%26rft.volume%253D277%26rft.issue%253D5329%26rft.spage%253D1063%26rft.epage%253D1066%26rft.atitle%253DSeed%2BBanks%2Band%2BMolecular%2BMaps%253A%2BUnlocking%2BGenetic%2BPotential%2Bfrom%2Bthe%2BWild%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.277.5329.1063%26rft_id%253Dinfo%253Apmid%252F9262467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzcvNTMyOS8xMDYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE1NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.60" data-doi="10.1023/A:1022308414864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tayler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Common features of segregation distortion in plants and animals.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DTaylor%26rft.auinit1%253DD.%2BR.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D35%26rft.atitle%253DCommon%2Bfeatures%2Bof%2Bsegregation%2Bdistortion%2Bin%2Bplants%2Band%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1022308414864%26rft_id%253Dinfo%253Apmid%252F12656570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1022308414864&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12656570&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181010300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.61" data-doi="10.1534/genetics.106.055871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Os</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrzejewski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakker</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrena</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Construction of a 10,000-marker ultradense genetic recombination map of potato: providing a framework for accelerated gene isolation and a genome-wide physical map.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1075</span>–<span class="cit-lpage">1087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.055871%26rft_id%253Dinfo%253Apmid%252F16582432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8yLzEwNzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTU1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1551.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Slageren</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-source"><em>Wild wheats: A Monograph of Aegilops L. and Amblyopyrum (Jaub</em>. &amp;<em> Spach) Eig (Poaceae): A Revision Of All Taxa Closely Related to Wheat</em>, <em>Excluding Wild Triticum Species</em>, <em>With Notes on Other Genera in the Tribe Triticcae</em>, <em>Especially Triticum. Wageningen Agricultural University</em>, <em>Wageningen</em>, <em>The Netherlands</em></span>, pp. <span class="cit-fpage">94</span>–<span class="cit-lpage">97</span> <span class="cit-publ-name">International Center for Agricultural Research in Dry Areas</span>, <span class="cit-publ-loc">Aleppo, Syria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.63" data-doi="10.1111/pbi.12183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterization of polyploid wheat genomic diversity using a high-density 90 000 single nucleotide polymorphism array.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">787</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D12%26rft.spage%253D787%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12183%26rft_id%253Dinfo%253Apmid%252F24646323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24646323&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339488200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westbrook</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vikram</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le-Shin</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srikar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leandro</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A consensus genetic map for <em>Pinus taeda</em> and <em>Pinus elliottii</em> and extent of linkage disequilibrium in two genotype-phenotype discovery populations of <em>Pinus taeda</em>.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1685</span>–<span class="cit-lpage">1694</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D5%26rft.spage%253D1685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.65" data-doi="10.1371/journal.pgen.1000212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000212%26rft.epage%253De1000212%26rft.atitle%253DEfficient%2Band%2Baccurate%2Bconstruction%2Bof%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bthe%2Bminimum%2Bspanning%2Btree%2Bof%2Ba%2Bgraph.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000212%26rft_id%253Dinfo%253Apmid%252F18846212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846212&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1551.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.66" data-doi="10.3389/fpls.2014.00692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wulff</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moscou</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Strategies for transferring resistance into wheat: from wide crosses to GM cassettes.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2014.00692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2014.00692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2014.00692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.67" data-doi="10.1007/s001220050710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Relationships between the chromosomes of Aegilops umbellulata and wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D96%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050710&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000071845200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1551.68" data-doi="10.2135/cropsci2005.0023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zaharieva</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monneveux</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Spontaneous hybridization between bread wheat <em>Triticum aestivum</em> L.) and its wild relatives in Europe.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">512</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DZaharieva%26rft.auinit1%253DM.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D512%26rft.epage%253D527%26rft.atitle%253DSpontaneous%2BHybridization%2Bbetween%2BBread%2BWheat%2B%2528Triticum%2Baestivum%2BL.%2529%2Band%2BIts%2BWild%2BRelatives%2Bin%2BEurope%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.0023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.0023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000235991100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/5/1551.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1539" title="TheCellMap.org: A Web-Accessible Database for Visualizing and Mining the Global Yeast Genetic Interaction Network" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1563" title="From Pine Cones to Read Clouds: Rescaffolding the Megagenome of Sugar Pine (Pinus lambertiana)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/5"><span>Volume 7 Issue 5, May 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/5.cover-source.jpg?itok=VWLvGstM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1551" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aJs4jN2PFP4vIyvzlJlT9wbnd2S5iEyQc8ayPX4V7PA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/5/1551" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/5/1551" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/5/1551" class="active" data-icon-position="" data-hide-link-title="0">Genotyping-by-Sequencing Facilitates a High-Density Consensus Linkage Map for Aegilops umbellulata, a Wild Relative of Cultivated Wheat</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484262" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-EZYBXbT-pZI3ZRo9IwYiRbv2BKGP_P-5iGFUK1ZjfKg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit--2" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1551.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/5/1551" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--5" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-RCFRL6O07VH-dbNTQiHmdOVuqiCwPLASgTawZB6ABAs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/5/1551" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1551.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484262" id="node484262--2" data-pisa="ggg;7/5/1551" data-pisa-master="ggg;g3.117.039966" data-apath="/ggg/7/5/1551.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotyping-by-Sequencing Facilitates a High-Density Consensus Linkage Map for <em>Aegilops umbellulata</em>, a Wild Relative of Cultivated Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Erena A. Edae</span>, <span class="highwire-citation-author" data-delta="1">Pablo D. Olivera</span>, <span class="highwire-citation-author" data-delta="2">Yue Jin</span> and <span class="highwire-citation-author" data-delta="3">Matthew N. Rouse</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1551-1561;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039966 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484262--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erena A. Edae </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Cereal Disease Laboratory, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erena%2BA.%2BEdae%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edae%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErena%2BA.%2BEdae%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erena.edae@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">erena.edae@ars.usda.gov</a>
<a href="mailto:matthew.rouse@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">matthew.rouse@ars.usda.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pablo D. Olivera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pablo%2BD.%2BOlivera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Olivera%20PD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APablo%2BD.%2BOlivera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yue Jin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Cereal Disease Laboratory, St. Paul, Minnesota 55108</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yue%2BJin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYue%2BJin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew N. Rouse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Cereal Disease Laboratory, St. Paul, Minnesota 55108</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BN.%2BRouse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rouse%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BN.%2BRouse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erena.edae@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">erena.edae@ars.usda.gov</a>
<a href="mailto:matthew.rouse@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">matthew.rouse@ars.usda.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/5/1551" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/5/1551&amp;title=Genotyping-by-Sequencing%20Facilitates%20a%20High-Density%20Consensus%20Linkage%20Map%20for%20Aegilops%20umbellulata%2C%20a%20Wild%20Relative%20of%20Cultivated%20Wheat" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/5/1551&amp;title=Genotyping-by-Sequencing%20Facilitates%20a%20High-Density%20Consensus%20Linkage%20Map%20for%20Aegilops%20umbellulata%2C%20a%20Wild%20Relative%20of%20Cultivated%20Wheat" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/5/1551&amp;title=Genotyping-by-Sequencing%20Facilitates%20a%20High-Density%20Consensus%20Linkage%20Map%20for%20Aegilops%20umbellulata%2C%20a%20Wild%20Relative%20of%20Cultivated%20Wheat" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/5/1551&amp;text=Genotyping-by-Sequencing%20Facilitates%20a%20High-Density%20Consensus%20Linkage%20Map%20for%20Aegilops%20umbellulata%2C%20a%20Wild%20Relative%20of%20Cultivated%20Wheat" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/5/1551&amp;title=Genotyping-by-Sequencing%20Facilitates%20a%20High-Density%20Consensus%20Linkage%20Map%20for%20Aegilops%20umbellulata%2C%20a%20Wild%20Relative%20of%20Cultivated%20Wheat" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/5/1551&amp;t=Genotyping-by-Sequencing%20Facilitates%20a%20High-Density%20Consensus%20Linkage%20Map%20for%20Aegilops%20umbellulata%2C%20a%20Wild%20Relative%20of%20Cultivated%20Wheat" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/5/1551&amp;title=Genotyping-by-Sequencing%20Facilitates%20a%20High-Density%20Consensus%20Linkage%20Map%20for%20Aegilops%20umbellulata%2C%20a%20Wild%20Relative%20of%20Cultivated%20Wheat" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/5/1551&amp;title=Genotyping-by-Sequencing%20Facilitates%20a%20High-Density%20Consensus%20Linkage%20Map%20for%20Aegilops%20umbellulata%2C%20a%20Wild%20Relative%20of%20Cultivated%20Wheat" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484262/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484262" id="node484262--3" data-pisa="ggg;7/5/1551" data-pisa-master="ggg;g3.117.039966" data-apath="/ggg/7/5/1551.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotyping-by-Sequencing Facilitates a High-Density Consensus Linkage Map for <em>Aegilops umbellulata</em>, a Wild Relative of Cultivated Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Erena A. Edae</span>, <span class="highwire-citation-author" data-delta="1">Pablo D. Olivera</span>, <span class="highwire-citation-author" data-delta="2">Yue Jin</span> and <span class="highwire-citation-author" data-delta="3">Matthew N. Rouse</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1551-1561;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.039966 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484262--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erena A. Edae </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Cereal Disease Laboratory, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erena%2BA.%2BEdae%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edae%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErena%2BA.%2BEdae%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erena.edae@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">erena.edae@ars.usda.gov</a>
<a href="mailto:matthew.rouse@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">matthew.rouse@ars.usda.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pablo D. Olivera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pablo%2BD.%2BOlivera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Olivera%20PD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APablo%2BD.%2BOlivera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yue Jin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Cereal Disease Laboratory, St. Paul, Minnesota 55108</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yue%2BJin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYue%2BJin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew N. Rouse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Cereal Disease Laboratory, St. Paul, Minnesota 55108</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BN.%2BRouse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rouse%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BN.%2BRouse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:erena.edae@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">erena.edae@ars.usda.gov</a>
<a href="mailto:matthew.rouse@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">matthew.rouse@ars.usda.gov</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1551&amp;rft.epage=1561&amp;rft.atitle=Genotyping-by-Sequencing%20Facilitates%20a%20High-Density%20Consensus%20Linkage%20Map%20for%20Aegilops%20umbellulata%2C%20a%20Wild%20Relative%20of%20Cultivated%20Wheat&amp;rft.volume=7&amp;rft.issue=5&amp;rft.date=2017-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Edae%2C+Erena+A.&amp;rft.au=Olivera%2C+Pablo+D.&amp;rft.au=Jin%2C+Yue&amp;rft.au=Rouse%2C+Matthew+N."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484262/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484262/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484262/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484262/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484262/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484262/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484262/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484262/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484262/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484262/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484262/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484262/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/5/1551" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/5/1551.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/5/1551.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/5/1551.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/5/1551.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
