<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/5/1463" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/5/1463.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/5/1463.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/5/1463.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/5/1463.atom" />
<meta name="HW.pisa" content="ggg;7/5/1463" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Schizosaccharomyces pombe MutSα and MutLα Maintain Stability of Tetra-Nucleotide Repeats and Msh3 of Hepta-Nucleotide Repeats" />
<meta name="DC.Identifier" content="10.1534/g3.117.040816" />
<meta name="DC.Date" content="2017-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Villahermosa et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Defective mismatch repair (MMR) in humans is associated with colon cancer and instability of microsatellites, that is, DNA sequences with one or several nucleotides repeated. Key factors of eukaryotic MMR are the heterodimers MutSα (Msh2-Msh6), which recognizes base-base mismatches and unpaired nucleotides in DNA, and MutLα (Mlh1-Pms1), which facilitates downstream steps. In addition, MutSβ (Msh2-Msh3) recognizes DNA loops of various sizes, although our previous data and the data presented here suggest that Msh3 of Schizosaccharomyces pombe does not play a role in MMR. To test microsatellite stability in S. pombe and hence DNA loop repair, we have inserted tetra-, penta-, and hepta-nucleotide repeats in the ade6 gene and determined their Ade+ reversion rates and spectra in wild type and various mutants. Our data indicate that loops with four unpaired nucleotides in the nascent and the template strand are the upper limit of MutSα- and MutLα-mediated MMR in S. pombe . Stability of hepta-nucleotide repeats requires Msh3 and Exo1 in MMR-independent processes as well as the DNA repair proteins Rad50, Rad51, and Rad2FEN1. Most strikingly, mutation rates in the double mutants msh3 exo1 and msh3 rad51 were decreased when compared to respective single mutants, indicating that Msh3 prevents error prone processes carried out by Exo1 and Rad51. We conclude that Msh3 has no obvious function in MMR in S. pombe , but contributes to DNA repeat stability in MMR-independent processes." />
<meta name="DC.Contributor" content="Desirée Villahermosa" />
<meta name="DC.Contributor" content="Olaf Christensen" />
<meta name="DC.Contributor" content="Karen Knapp" />
<meta name="DC.Contributor" content="Oliver Fleck" />
<meta name="article:published_time" content="2017-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Schizosaccharomyces pombe MutSα and MutLα Maintain Stability of Tetra-Nucleotide Repeats and Msh3 of Hepta-Nucleotide Repeats" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Defective mismatch repair (MMR) in humans is associated with colon cancer and instability of microsatellites, that is, DNA sequences with one or several nucleotides repeated. Key factors of eukaryotic MMR are the heterodimers MutSα (Msh2-Msh6), which recognizes base-base mismatches and unpaired nucleotides in DNA, and MutLα (Mlh1-Pms1), which facilitates downstream steps. In addition, MutSβ (Msh2-Msh3) recognizes DNA loops of various sizes, although our previous data and the data presented here suggest that Msh3 of &lt;i&gt;Schizosaccharomyces pombe&lt;/i&gt; does not play a role in MMR. To test microsatellite stability in &lt;i&gt;S. pombe&lt;/i&gt; and hence DNA loop repair, we have inserted tetra-, penta-, and hepta-nucleotide repeats in the &lt;i&gt;ade6&lt;/i&gt; gene and determined their Ade&lt;sup&gt;+&lt;/sup&gt; reversion rates and spectra in wild type and various mutants. Our data indicate that loops with four unpaired nucleotides in the nascent and the template strand are the upper limit of MutSα- and MutLα-mediated MMR in &lt;i&gt;S. pombe&lt;/i&gt;. Stability of hepta-nucleotide repeats requires Msh3 and Exo1 in MMR-independent processes as well as the DNA repair proteins Rad50, Rad51, and Rad2&lt;sup&gt;FEN1&lt;/sup&gt;. Most strikingly, mutation rates in the double mutants &lt;i&gt;msh3 exo1&lt;/i&gt; and &lt;i&gt;msh3 rad51&lt;/i&gt; were decreased when compared to respective single mutants, indicating that Msh3 prevents error prone processes carried out by Exo1 and Rad51. We conclude that Msh3 has no obvious function in MMR in &lt;i&gt;S. pombe&lt;/i&gt;, but contributes to DNA repeat stability in MMR-independent processes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/05/01" />
<meta name="citation_mjid" content="ggg;7/5/1463" />
<meta name="citation_id" content="7/5/1463" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/5/1463" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/5/1463.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/5/1463.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/5/1463.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.040816" />
<meta name="citation_pmid" content="28341698" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1463" />
<meta name="citation_lastpage" content="1473" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Desirée Villahermosa" />
<meta name="citation_author_institution" content="North West Cancer Research Institute, School of Medical Sciences, Bangor University, LL57 2UW, UK" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4072-6159" />
<meta name="citation_author" content="Olaf Christensen" />
<meta name="citation_author_institution" content="Strain Development LBPE38, Lonza, CH-3930 Visp, Switzerland" />
<meta name="citation_author" content="Karen Knapp" />
<meta name="citation_author_institution" content="North West Cancer Research Institute, School of Medical Sciences, Bangor University, LL57 2UW, UK" />
<meta name="citation_author" content="Oliver Fleck" />
<meta name="citation_author_institution" content="North West Cancer Research Institute, School of Medical Sciences, Bangor University, LL57 2UW, UK" />
<meta name="citation_author_email" content="o.fleck@bangor.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0550-6513" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=E. Alani;citation_title=The Saccharomyces cerevisiae Msh2 and Msh6 proteins form a complex that specifically binds to duplex oligonucleotides containing mismatched DNA base pairs;citation_pages=5604-5615;citation_volume=16;citation_year=1996;citation_issue=10;citation_pmid=8816473;citation_doi=10.1128/MCB.16.10.5604" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=N. S. Amin;citation_author=M. N. Nguyen;citation_author=S. Oh;citation_author=R. D. Kolodner;citation_title=exo1-Dependent Mutator Mutations: Model System for Studying Functional Interactions in Mismatch Repair;citation_pages=5142-5155;citation_volume=21;citation_year=2001;citation_issue=15;citation_pmid=11438669;citation_doi=10.1128/MCB.21.15.5142-5155.2001" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=R. Bhargava;citation_author=D. O. Onyango;citation_author=J. M. Stark;citation_title=Regulation of single-strand annealing and its role in genome maintenance.;citation_pages=566-575;citation_volume=32;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. Bowen;citation_author=C. E. Smith;citation_author=A. Srivatsan;citation_author=S. Willcox;citation_author=J. D. Griffith;citation_title=Reconstitution of long and short patch mismatch repair reactions using Saccharomyces cerevisiae proteins.;citation_volume=110;citation_year=2013;citation_pmid=24187148;citation_doi=10.1073/pnas.1318971110" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=P. E. Burby;citation_author=L. A. Simmons;citation_title=MutS2 promotes homologous recombination in Bacillus subtilis.;citation_volume=199;citation_year=2017;citation_pmid=27799325;citation_doi=10.1128/JB.00682-16" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=P. Cejka;citation_title=DNA end resection: nucleases team up with the right partners to initiate homologous recombination.;citation_volume=290;citation_year=2015;citation_pmid=26231213;citation_doi=10.1074/jbc.R115.675942" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=U. Chakraborty;citation_author=E. Alani;citation_title=Understanding how mismatch repair proteins participate in the repair/anti-recombination decision.;citation_volume=16;citation_year=2016;citation_pmid=27573382;citation_doi=10.1093/femsyr/fow071" />
<meta name="citation_reference" content="citation_journal_title=Anticancer Research;citation_journal_abbrev=Anticancer Res;citation_author=F. C. Da Silva;citation_author=P. Wernhoff;citation_author=C. Dominguez-Barrera;citation_author=M. Dominguez-Valentin;citation_title=Update on Hereditary Colorectal Cancer;citation_pages=4399-4405;citation_volume=36;citation_year=2016;citation_issue=9;citation_pmid=27630275;citation_doi=10.21873/anticanres.10983" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. T. Drummond;citation_author=J. Genschel;citation_author=E. Wolf;citation_author=P. Modrich;citation_title=DHFR/MSH3 amplification in methotrexate-resistant cells alters the hMutS{alpha}/hMutS{beta} ratio and reduces the efficiency of base-base mismatch repair;citation_pages=10144-10149;citation_volume=94;citation_year=1997;citation_issue=19;citation_pmid=9294177;citation_doi=10.1073/pnas.94.19.10144" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=P. Fiorentini;citation_author=K. N. Huang;citation_author=D. X. Tishkoff;citation_author=R. D. Kolodner;citation_author=L. S. Symington;citation_title=Exonuclease I of Saccharomyces cerevisiae functions in mitotic recombination in vivo and in vitro;citation_pages=2764-2773;citation_volume=17;citation_year=1997;citation_issue=5;citation_pmid=9111347;citation_doi=10.1128/MCB.17.5.2764" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=O. Fleck;citation_author=H. Michael;citation_author=L. Heim;citation_title=The swi4+ gene of Schizosaccharomyces pombe encodes a homologue of mismatch repair enzymes;citation_pages=2271-2278;citation_volume=20;citation_year=1992;citation_issue=9;citation_pmid=1317550;citation_doi=10.1093/nar/20.9.2271" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Flores-Rozas;citation_author=D. Clark;citation_author=R. D. Kolodner;citation_title=Proliferating cell nuclear antigen and Msh2p-Msh6p interact to form an active mispair recognition complex.;citation_pages=375-378;citation_volume=26;citation_year=2000;citation_issue=3;citation_pmid=11062484;citation_doi=10.1038/81708" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. H. Freudenreich;citation_author=S. M. Kantrow;citation_author=V. A. Zakian;citation_title=Expansion and Length-Dependent Fragility of CTG Repeats in Yeast;citation_pages=853-856;citation_volume=279;citation_year=1998;citation_issue=5352;citation_pmid=9452383;citation_doi=10.1126/science.279.5352.853" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=J. Genschel;citation_author=S. J. Littman;citation_author=J. T. Drummond;citation_author=P. Modrich;citation_title=Isolation of MutS{beta} from Human Cells and Comparison of the Mismatch Repair Specificities of MutS{beta} and MutS{alpha};citation_pages=19895-19901;citation_volume=273;citation_year=1998;citation_issue=31;citation_pmid=9677427;citation_doi=10.1074/jbc.273.31.19895" />
<meta name="citation_reference" content="citation_journal_title=EMBO reports;citation_journal_abbrev=EMBO Rep.;citation_author=G. Giannini;citation_author=E. Ristori;citation_author=F. Cerignoli;citation_author=C. Rinaldi;citation_author=M. Zani;citation_title=Human MRE11 is inactivated in mismatch repair-deficient cancers;citation_pages=248-254;citation_volume=3;citation_year=2002;citation_issue=3;citation_pmid=11850399;citation_doi=10.1093/embo-reports/kvf044" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=E. M. Goellner;citation_author=C. E. Smith;citation_author=C. S. Campbell;citation_author=H. Hombauer;citation_author=A. Desai;citation_title=PCNA and Msh2-Msh6 activate an Mlh1-Pms1 endonuclease pathway required for Exo1-independent mismatch repair.;citation_pages=291-304;citation_volume=55;citation_year=2014;citation_pmid=24981171;citation_doi=10.1016/j.molcel.2014.04.034" />
<meta name="citation_reference" content="citation_journal_title=DNA Repair (Amst.);citation_author=E. M. Goellner;citation_author=C. D. Putnam;citation_author=R. D. Kolodner;citation_title=Exonuclease 1-dependent and independent mismatch repair.;citation_pages=24-32;citation_volume=32;citation_year=2015;citation_pmid=25956862;citation_doi=10.1016/j.dnarep.2015.04.010" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. N. Greene;citation_author=S. Jinks-Robertson;citation_title=Frameshift intermediates in homopolymer runs are removed efficiently by yeast mismatch repair proteins;citation_pages=2844-2850;citation_volume=17;citation_year=1997;citation_issue=5;citation_pmid=9111356;citation_doi=10.1128/MCB.17.5.2844" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Grimm;citation_author=J. Bähler;citation_author=J. Kohli;citation_title=M26 recombinational hotspot and physical conversion tract analysis in the ade6 gene of Schizosaccharomyces pombe.;citation_pages=41-51;citation_volume=136;citation_year=1994;citation_issue=1;citation_pmid=7908005" />
<meta name="citation_reference" content="citation_journal_title=Nat. Struct. Mol. Biol.;citation_author=S. Gupta;citation_author=M. Gellert;citation_author=W. Yang;citation_title=Mechanism of mismatch recognition revealed by human MutSβ bound to unpaired DNA loops.;citation_pages=72-78;citation_volume=19;citation_year=2011;citation_pmid=22179786;citation_doi=10.1038/nsmb.2175" />
<meta name="citation_reference" content="citation_title=Handbook of Genetics;citation_author=H. Gutz;citation_author=H. Heslot;citation_author=U. Leupold;citation_author=N. Loprieno;citation_title=Schizosaccharomyces pombe;citation_pages=395-446;citation_volume=Vol. 1;citation_year=1974" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=Y. Habraken;citation_author=P. Sung;citation_author=L. Prakash;citation_author=S. Prakash;citation_title=Binding of insertion/deletion DNA mismatches by the heterodimer of yeast mismatch repair proteins MSH2 and MSH3.;citation_pages=1185-1187;citation_volume=6;citation_year=1996;citation_issue=9;citation_pmid=8805366;citation_doi=10.1016/S0960-9822(02)70686-6" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=J. M. Harrington;citation_author=R. D. Kolodner;citation_title=Saccharomyces cerevisiae Msh2-Msh3 acts in repair of base-base mispairs.;citation_volume=27;citation_year=2007;citation_pmid=17636021;citation_doi=10.1128/MCB.00855-07" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=E. Hartsuiker;citation_author=E. Vaessen;citation_author=A. M. Carr;citation_author=J. Kohli;citation_title=Fission yeast Rad50 stimulates sister chromatid recombination and links cohesion with repair;citation_pages=6660-6671;citation_volume=20;citation_year=2001;citation_issue=23;citation_pmid=11726502;citation_doi=10.1093/emboj/20.23.6660" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=P. Hentges;citation_author=B. Van Driessche;citation_author=L. Tafforeau;citation_author=J. Vandenhaute;citation_author=A. M. Carr;citation_title=Three novel antibiotic marker cassettes for gene disruption and marker switching in Schizosaccharomyces pombe.;citation_pages=1013-1019;citation_volume=22;citation_year=2005;citation_issue=13;citation_pmid=16200533;citation_doi=10.1002/yea.1291" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=H. Ito;citation_author=Y. Fukuda;citation_author=K. Murata;citation_author=A. Kimura;citation_title=Transformation of intact yeast cells treated with alkali cations.;citation_pages=163-168;citation_volume=153;citation_year=1983;citation_issue=1;citation_pmid=6336730" />
<meta name="citation_reference" content="citation_journal_title=Chemical Reviews;citation_journal_abbrev=Chemical Reviews;citation_author=R. R. Iyer;citation_author=A. Pluciennik;citation_author=V. Burdett;citation_author=P. L. Modrich;citation_title=DNA mismatch repair: functions and mechanisms.;citation_pages=302-323;citation_volume=106;citation_year=2006;citation_issue=2;citation_pmid=16464007;citation_doi=10.1021/cr0404794" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=R. R. Iyer;citation_author=A. Pluciennik;citation_author=J. Genschel;citation_author=M. S. Tsai;citation_author=L. S. Beese;citation_title=MutLα and proliferating cell nuclear antigen share binding sites on MutSβ.;citation_volume=285;citation_year=2010;citation_pmid=20154325;citation_doi=10.1074/jbc.M110.104125" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Perspectives in Biology;citation_journal_abbrev=Cold Spring Harb. Perspect. Biol.;citation_author=J. Jiricny;citation_title=Postreplicative Mismatch Repair;citation_pages=a012633-a012633;citation_volume=5;citation_year=2013;citation_issue=4;citation_pmid=23545421;citation_doi=10.1101/cshperspect.a012633" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. E. Johnson;citation_author=G. K. Kovvali;citation_author=L. Prakash;citation_author=S. Prakash;citation_title=Requirement of the yeast RTH1 5' to 3' exonuclease for the stability of simple repetitive DNA;citation_pages=238-240;citation_volume=269;citation_year=1995;citation_issue=5221;citation_pmid=7618086;citation_doi=10.1126/science.7618086" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=R. E. Johnson;citation_author=G. K. Kovvali;citation_author=L. Prakash;citation_author=S. Prakash;citation_title=Requirement of the Yeast MSH3 and MSH6 Genes for MSH2-dependent Genomic Stability;citation_pages=7285-7288;citation_volume=271;citation_year=1996;citation_issue=13;citation_pmid=8631743;citation_doi=10.1074/jbc.271.13.7285" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. A. Kadyrov;citation_author=L. Dzantiev;citation_author=N. Constantin;citation_author=P. Modrich;citation_title=Endonucleolytic function of MutLalpha in human mismatch repair.;citation_pages=297-308;citation_volume=126;citation_year=2006;citation_issue=2;citation_pmid=16873062;citation_doi=10.1016/j.cell.2006.05.039" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=F. A. Kadyrov;citation_author=S. F. Holmes;citation_author=M. E. Arana;citation_author=O. A. Lukianova;citation_author=M. O’Donnell;citation_title=Saccharomyces cerevisiae MutLα is a mismatch repair endonuclease.;citation_volume=282;citation_year=2007;citation_pmid=17951253;citation_doi=10.1074/jbc.M707617200" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=H. Karaoglu;citation_author=C. M. Lee;citation_author=W. Meyer;citation_title=Survey of simple sequence repeats in completed fungal genomes.;citation_volume=22;citation_year=2005;citation_pmid=15563717;citation_doi=10.1093/molbev/msi057" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=D. U. Kim;citation_author=J. Hayles;citation_author=D. Kim;citation_author=V. Wood;citation_author=H. O. Park;citation_title=Analysis of a genome-wide set of gene deletions in the fission yeast Schizosaccharomyces pombe.;citation_pages=617-623;citation_volume=28;citation_year=2010;citation_issue=6;citation_pmid=20473289;citation_doi=10.1038/nbt.1628" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Kirchner;citation_author=H. Tran;citation_author=M. A. Resnick;citation_title=A DNA Polymerase {epsilon} Mutant That Specifically Causes +1 Frameshift Mutations Within Homonucleotide Runs in Yeast;citation_pages=1623-1632;citation_volume=155;citation_year=2000;citation_issue=4;citation_pmid=10924461" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. T. Kirkpatrick;citation_author=J. R. Ferguson;citation_author=T. D. Petes;citation_author=L. S. Symington;citation_title=Decreased Meiotic Intergenic Recombination and Increased Meiosis I Nondisjunction in exo1 Mutants of Saccharomyces cerevisiae;citation_pages=1549-1557;citation_volume=156;citation_year=2000;citation_issue=4;citation_pmid=11102356" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=H. E. Kleczkowska;citation_author=G. Marra;citation_author=T. Lettieri;citation_author=J. Jiricny;citation_title=hMSH3 and hMSH6 interact with PCNA and colocalize with it to replication foci;citation_pages=724-736;citation_volume=15;citation_year=2001;citation_issue=6;citation_pmid=11274057;citation_doi=10.1101/gad.191201" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. S. Lander;citation_author=L. M. Linton;citation_author=B. Birren;citation_author=C. Nusbaum;citation_author=M. C. Zody;citation_title=Initial sequencing and analysis of the human genome.;citation_pages=860-921;citation_volume=409;citation_year=2001;citation_issue=6822;citation_pmid=11237011;citation_doi=10.1038/35057062" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. D. Lee;citation_author=E. Alani;citation_title=Analysis of interactions between mismatch repair initiation factors and the replication processivity factor PCNA.;citation_pages=175-184;citation_volume=355;citation_year=2006;citation_issue=2;citation_pmid=16303135;citation_doi=10.1016/j.jmb.2005.10.059" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. D. Lee;citation_author=J. A. Surtees;citation_author=E. Alani;citation_title=Saccharomyces cerevisiae MSH2-MSH3 and MSH2-MSH6 complexes display distinct requirements for DNA binding domain I in mismatch recognition.;citation_pages=53-66;citation_volume=366;citation_year=2007;citation_issue=1;citation_pmid=17157869;citation_doi=10.1016/j.jmb.2006.10.099" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=Y. Liu;citation_author=S. H. Wilson;citation_title=DNA base excision repair: a mechanism of trinucleotide repeat expansion.;citation_pages=162-172;citation_volume=37;citation_year=2012;citation_issue=4;citation_pmid=22285516;citation_doi=10.1016/j.tibs.2011.12.002" />
<meta name="citation_reference" content="citation_journal_title=Clinical genetics;citation_journal_abbrev=Clin Genet;citation_author=H. T. Lynch;citation_author=P. M. Lynch;citation_author=S. J. Lanspa;citation_author=C. L. Snyder;citation_author=J. F. Lynch;citation_title=Review of the Lynch syndrome: history, molecular genetics, screening, differential diagnosis, and medicolegal ramifications.;citation_pages=1-18;citation_volume=76;citation_year=2009;citation_issue=1;citation_pmid=19659756" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. A. Mansour;citation_author=C. Tornier;citation_author=E. Lehmann;citation_author=M. Darmon;citation_author=O. Fleck;citation_title=Control of GT Repeat Stability in Schizosaccharomyces pombe by Mismatch Repair Factors;citation_pages=77-85;citation_volume=158;citation_year=2001;citation_issue=1;citation_pmid=11333219" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=G. Marra;citation_author=I. Iaccarino;citation_author=T. Lettieri;citation_author=G. Roscilli;citation_author=P. Delmastro;citation_title=Mismatch repair deficiency associated with overexpression of the MSH3 gene;citation_pages=8568-8573;citation_volume=95;citation_year=1998;citation_issue=15;citation_pmid=9671718;citation_doi=10.1073/pnas.95.15.8568" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=G. T. Marsischky;citation_author=N. Filosi;citation_author=M. F. Kane;citation_author=R. Kolodner;citation_title=Redundancy of Saccharomyces cerevisiae MSH3 and MSH6 in MSH2-dependent mismatch repair.;citation_pages=407-420;citation_volume=10;citation_year=1996;citation_issue=4;citation_pmid=8600025;citation_doi=10.1101/gad.10.4.407" />
<meta name="citation_reference" content="citation_journal_title=Cellular and molecular life sciences :  CMLS;citation_journal_abbrev=Cell Mol Life Sci;citation_author=T. M. Marti;citation_author=O. Fleck;citation_title=DNA repair nucleases.;citation_pages=336-354;citation_volume=61;citation_year=2004;citation_issue=3;citation_pmid=14770297;citation_doi=10.1007/s00018-003-3223-4" />
<meta name="citation_reference" content="citation_journal_title=Journal of cellular physiology;citation_journal_abbrev=J Cell Physiol;citation_author=T. M. Marti;citation_author=C. Kunz;citation_author=O. Fleck;citation_title=DNA mismatch repair and mutation avoidance pathways.;citation_pages=28-41;citation_volume=191;citation_year=2002;citation_issue=1;citation_pmid=11920679;citation_doi=10.1002/jcp.10077" />
<meta name="citation_reference" content="citation_journal_title=DNA repair;citation_journal_abbrev=DNA Repair (Amst);citation_author=T. M. Marti;citation_author=A. A. Mansour;citation_author=E. Lehmann;citation_author=O. Fleck;citation_title=Different frameshift mutation spectra in non-repetitive DNA of MutSalpha- and MutLalpha-deficient fission yeast cells.;citation_pages=571-580;citation_volume=2;citation_year=2003;citation_issue=5;citation_pmid=12713814" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. M. Murray;citation_author=M. Tavassoli;citation_author=R. al-Harithy;citation_author=K. S. Sheldrick;citation_author=A. R. Lehmann;citation_title=Structural and functional conservation of the human homolog of the Schizosaccharomyces pombe rad2 gene, which is required for chromosome segregation and recovery from DNA damage.;citation_pages=4878-4888;citation_volume=14;citation_year=1994;citation_issue=7;citation_pmid=8007985;citation_doi=10.1128/MCB.14.7.4878" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=J. Padeken;citation_author=P. Zeller;citation_author=S. M. Gasser;citation_title=Repeat DNA in genome organization and stability.;citation_pages=12-19;citation_volume=31;citation_year=2015;citation_doi=10.1016/j.gde.2015.03.009" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. V. Pinto;citation_author=A. Mathieu;citation_author=S. Marsin;citation_author=X. Veaute;citation_author=L. Ielpi;citation_title=Suppression of homologous and homeologous recombination by the bacterial MutS2 protein.;citation_pages=113-120;citation_volume=17;citation_year=2005;citation_issue=1;citation_pmid=15629722;citation_doi=10.1016/j.molcel.2004.11.035" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Pluciennik;citation_author=L. Dzantiev;citation_author=R. R. Iyer;citation_author=N. Constantin;citation_author=F. A. Kadyrov;citation_title=PCNA function in the activation and strand direction of MutLα endonuclease in mismatch repair.;citation_volume=107;citation_year=2010;citation_pmid=20713735;citation_doi=10.1073/pnas.1010662107" />
<meta name="citation_reference" content="citation_journal_title=Clinical Cancer Research;citation_journal_abbrev=Clin. Cancer Res.;citation_author=R. Rodriguez;citation_author=L. T. Hansen;citation_author=G. Phear;citation_author=J. Scorah;citation_author=M. Spang-Thomsen;citation_title=Thymidine Selectively Enhances Growth Suppressive Effects of Camptothecin/Irinotecan in MSI+ Cells and Tumors Containing a Mutation of MRE11;citation_pages=5476-5483;citation_volume=14;citation_year=2008;citation_issue=17;citation_pmid=18765539;citation_doi=10.1158/1078-0432.CCR-08-0274" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=C. Rudolph;citation_author=O. Fleck;citation_author=J. Kohli;citation_title=Schizosaccharomyces pombe exo1 is involved in the same mismatch repair pathway as msh2 and pms1.;citation_pages=343-350;citation_volume=34;citation_year=1998;citation_issue=5;citation_pmid=9871115;citation_doi=10.1007/s002940050405" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. Rudolph;citation_author=C. Kunz;citation_author=S. Parisi;citation_author=E. Lehmann;citation_author=E. Hartsuiker;citation_title=The msh2 Gene of Schizosaccharomyces pombe Is Involved in Mismatch Repair, Mating-Type Switching, and Meiotic Chromosome Organization;citation_pages=241-250;citation_volume=19;citation_year=1999;citation_issue=1;citation_pmid=9858548;citation_doi=10.1128/MCB.19.1.241" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. Schär;citation_author=M. Baur;citation_author=C. Schneider;citation_author=J. Kohli;citation_title=Mismatch Repair in Schizosaccharomyces pombe Requires the mutL Homologous Gene pms1: Molecular Cloning and Functional Analysis;citation_pages=1275-1286;citation_volume=146;citation_year=1997;citation_issue=4;citation_pmid=9258673" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. S. Shell;citation_author=C. D. Putnam;citation_author=R. D. Kolodner;citation_title=Chimeric Saccharomyces cerevisiae Msh6 protein with an Msh3 mispair-binding domain combines properties of both proteins.;citation_volume=104;citation_year=2007;citation_pmid=17573527;citation_doi=10.1073/pnas.0704148104" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=E. A. Sia;citation_author=R. J. Kokoska;citation_author=M. Dominska;citation_author=P. Greenwell;citation_author=T. D. Petes;citation_title=Microsatellite instability in yeast: dependence on repeat unit size and DNA mismatch repair genes;citation_pages=2851-2858;citation_volume=17;citation_year=1997;citation_issue=5;citation_pmid=9111357;citation_doi=10.1128/MCB.17.5.2851" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. E. Smith;citation_author=M. L. Mendillo;citation_author=N. Bowen;citation_author=H. Hombauer;citation_author=C. S. Campbell;citation_title=Dominant mutations in S. cerevisiae PMS1 identify the Mlh1-Pms1 endonuclease active site and an exonuclease 1-independent mismatch repair pathway.;citation_pages=e1003869;citation_volume=9;citation_year=2013;citation_pmid=24204293;citation_doi=10.1371/journal.pgen.1003869" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Sommer;citation_author=C. M. Stith;citation_author=P. M. Burgers;citation_author=R. S. Lahue;citation_title=Partial reconstitution of DNA large loop repair with purified proteins from Saccharomyces cerevisiae.;citation_volume=36;citation_year=2008;citation_pmid=18628298;citation_doi=10.1093/nar/gkn446" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=N. Sugawara;citation_author=F. Pâques;citation_author=M. Colaiácovo;citation_author=J. E. Haber;citation_title=Role of Saccharomyces cerevisiae Msh2 and Msh3 repair proteins in double-strand break-induced recombination;citation_pages=9214-9219;citation_volume=94;citation_year=1997;citation_issue=17;citation_pmid=9256462;citation_doi=10.1073/pnas.94.17.9214" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Sundararajan;citation_author=L. Gellon;citation_author=R. M. Zunder;citation_author=C. H. Freudenreich;citation_title=Double-strand break repair pathways protect against CAG/CTG repeat expansions, contractions and repeat-mediated chromosomal fragility in Saccharomyces cerevisiae.;citation_volume=184;citation_year=2010;citation_pmid=19901069;citation_doi=10.1534/genetics.109.111039" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. Szankasi;citation_author=G. R. Smith;citation_title=A role for exonuclease I from S. pombe in mutation avoidance and mismatch correction;citation_pages=1166-1169;citation_volume=267;citation_year=1995;citation_issue=5201;citation_pmid=7855597;citation_doi=10.1126/science.7855597" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=P. Szankasi;citation_author=W. D. Heyer;citation_author=P. Schuchert;citation_author=J. Kohli;citation_title=DNA sequence analysis of the ade6 gene of Schizosaccharomyces pombe. Wild-type and mutant alleles including the recombination host spot allele ade6-M26.;citation_pages=917-925;citation_volume=204;citation_year=1988;citation_issue=4;citation_pmid=3221399;citation_doi=10.1016/0022-2836(88)90051-4" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. X. Tishkoff;citation_author=N. Filosi;citation_author=G. M. Gaida;citation_author=R. D. Kolodner;citation_title=A novel mutation avoidance mechanism dependent on S. cerevisiae RAD27 is distinct from DNA mismatch repair.;citation_pages=253-263;citation_volume=88;citation_year=1997;citation_issue=2;citation_pmid=9008166;citation_doi=10.1016/S0092-8674(00)81846-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Tornier;citation_author=S. Bessone;citation_author=I. Varlet;citation_author=C. Rudolph;citation_author=M. Darmon;citation_title=Requirement for Msh6, but Not for Swi4 (Msh3), in Msh2-Dependent Repair of Base-Base Mismatches and Mononucleotide Loops in Schizosaccharomyces pombe;citation_pages=65-75;citation_volume=158;citation_year=2001;citation_issue=1;citation_pmid=11333218" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=H. Tsubouchi;citation_author=H. Ogawa;citation_title=Exo1 Roles for Repair of DNA Double-Strand Breaks and Meiotic Crossing Over in Saccharomyces cerevisiae;citation_pages=2221-2233;citation_volume=11;citation_year=2000;citation_issue=7;citation_pmid=10888664;citation_doi=10.1091/mbc.11.7.2221" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. J. Warren;citation_author=T. J. Pohlhaus;citation_author=A. Changela;citation_author=R. R. Iyer;citation_author=P. L. Modrich;citation_title=Structure of the human MutSalpha DNA lesion recognition complex.;citation_pages=579-592;citation_volume=26;citation_year=2007;citation_issue=4;citation_pmid=17531815;citation_doi=10.1016/j.molcel.2007.04.018" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=A. T. Watson;citation_author=V. Garcia;citation_author=N. Bone;citation_author=A. M. Carr;citation_author=J. Armstrong;citation_title=Gene tagging and gene replacement using recombinase-mediated cassette exchange in Schizosaccharomyces pombe.;citation_pages=63-74;citation_volume=407;citation_year=2008;citation_issue=1-2;citation_pmid=18054176;citation_doi=10.1016/j.gene.2007.09.024" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Schizosaccharomyces pombe MutSα and MutLα Maintain Stability of Tetra-Nucleotide Repeats and Msh3 of Hepta-Nucleotide Repeats" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
<meta name="twitter:description" content="Defective mismatch repair (MMR) in humans is associated with colon cancer and instability of microsatellites, that is, DNA sequences with one or several nucleotides repeated. Key factors of eukaryotic MMR are the heterodimers MutSα (Msh2-Msh6), which recognizes base-base mismatches and unpaired nucleotides in DNA, and MutLα (Mlh1-Pms1), which facilitates downstream steps. In addition, MutSβ (Msh2-Msh3) recognizes DNA loops of various sizes, although our previous data and the data presented here suggest that Msh3 of Schizosaccharomyces pombe does not play a role in MMR. To test microsatellite stability in S. pombe and hence DNA loop repair, we have inserted tetra-, penta-, and hepta-nucleotide repeats in the ade6 gene and determined their Ade+ reversion rates and spectra in wild type and various mutants. Our data indicate that loops with four unpaired nucleotides in the nascent and the template strand are the upper limit of MutSα- and MutLα-mediated MMR in S. pombe . Stability of hepta-nucleotide repeats requires Msh3 and Exo1 in MMR-independent processes as well as the DNA repair proteins Rad50, Rad51, and Rad2FEN1. Most strikingly, mutation rates in the double mutants msh3 exo1 and msh3 rad51 were decreased when compared to respective single mutants, indicating that Msh3 prevents error prone processes carried out by Exo1 and Rad51. We conclude that Msh3 has no obvious function in MMR in S. pombe , but contributes to DNA repeat stability in MMR-independent processes." />
<meta name="og-title" property="og:title" content="Schizosaccharomyces pombe MutSα and MutLα Maintain Stability of Tetra-Nucleotide Repeats and Msh3 of Hepta-Nucleotide Repeats" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/5/1463" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Defective mismatch repair (MMR) in humans is associated with colon cancer and instability of microsatellites, that is, DNA sequences with one or several nucleotides repeated. Key factors of eukaryotic MMR are the heterodimers MutSα (Msh2-Msh6), which recognizes base-base mismatches and unpaired nucleotides in DNA, and MutLα (Mlh1-Pms1), which facilitates downstream steps. In addition, MutSβ (Msh2-Msh3) recognizes DNA loops of various sizes, although our previous data and the data presented here suggest that Msh3 of Schizosaccharomyces pombe does not play a role in MMR. To test microsatellite stability in S. pombe and hence DNA loop repair, we have inserted tetra-, penta-, and hepta-nucleotide repeats in the ade6 gene and determined their Ade+ reversion rates and spectra in wild type and various mutants. Our data indicate that loops with four unpaired nucleotides in the nascent and the template strand are the upper limit of MutSα- and MutLα-mediated MMR in S. pombe . Stability of hepta-nucleotide repeats requires Msh3 and Exo1 in MMR-independent processes as well as the DNA repair proteins Rad50, Rad51, and Rad2FEN1. Most strikingly, mutation rates in the double mutants msh3 exo1 and msh3 rad51 were decreased when compared to respective single mutants, indicating that Msh3 prevents error prone processes carried out by Exo1 and Rad51. We conclude that Msh3 has no obvious function in MMR in S. pombe , but contributes to DNA repeat stability in MMR-independent processes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/5.cover-source.jpg" />
    <title>Schizosaccharomyces pombe MutSα and MutLα Maintain Stability of Tetra-Nucleotide Repeats and Msh3 of Hepta-Nucleotide Repeats | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__6uPNahEPJnOkpurvaRxri2qBNHoapqvM8y1Wtdb7u_Q__sE1YcwwFB0K8aMYGFpNWEgt3sH5t9Ld9QAF34gSvqPY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__EjT-6YMPvgLl8LSJ78FRw-tAFtIqvvC0dKPDt3xx2Ec__naWIXWhSBwaYF2DqFTeJJxk8gTTEkdfCwGZ2o8_HgIw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__WoUUhFHu9IZRJChlxfU_hFiUTBhKim997BgtM6rQv_0__WtsG9gDKkyL0Zcl619hpmtR4jbV5vkG5UnoAmrjf3jM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"SIb5wAPPPFQkPSKUrrWbcT_kzzNIHW4GPBsk19s0T1A","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484260","apath":"\/ggg\/7\/5\/1463.atom","pisa":"ggg;7\/5\/1463","ac":{"\/ggg\/7\/5\/1463.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/5\/1463.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1463"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/5\/1463"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/5\/1463":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wK2g1yiuVjgsnyN4OuYjtHrUmy38wkbJCi9Hb1JFbWfFiD1TXVOjtPScfRvtHZswzlHSlF9aukrwWCoK9JzbJpbl2JOBUuZKSUKcVDveWhyi7W824mp6C58Y6ytwc4bma0Cl2PfrD+KdtY5QciKlEPKiLVgiC7EyBC4dnDa9zu1JT+YhoR3zBOC5C77i830bMkyCyRi70QEh8nr9L5hv8fGu7L5+sPg9CNJ8FbyJpCtwaggs5A1POFilIvhGwO1gw+5KYFG\\\\\\\/ajkOC3MxITVr9qj9eFuREwNRzevQV5iwi1YK85WeL94PikcmXTge2iDjSFfQ9T55Wmr31F0x1xLDsJhHpg6ZzOyzI\\\\\\\/ZAPUbElzrUhmn9nlWmQbKsrC0V9rt3dVKoFjub0N5Sxxqrq6IG4nRJgMWwxScLEPdI66j2Pa3H20caF3I2yeulJf+p+J8c2kG4B7wPGovoHMYovVrl845KBHXWeBSYObVDgaM4wg8k7k5SGz5YFR3fld8p3UzEkbnwGMCwa1TE44zGASxItrRmETD6LFihznqz0q6aB+hmLSzYyx8AWkyBQ7tNtD4CFP2hVib+XKwbgENge3elhqCGaYv\\\\\\\/EPPCw6p7k+89Bhs1iNEV+M8AWQziH1MvqlL9tG3eQImrUP6UFl9RdgUSvz2Gd7\\\\\\\/TpYZ5Grs0xuahWMwv+mn3mOi\\\\\\\/5QI+fG3v5ZOk60XvXpXzCqAJi86qGWQ8cUKXWBVTd1zu4UU80l0xcNHiclrP2126pq7P9L758EbQeTA+8e\\\\\\\/NkXs+EiJECwEEiBg6Yi5Aq1pKS+EABD0z8Nr5iWiw+5c+eF1VLxhIFQMGYZHgp0b\\\\\\\/hCFuQ4xyMsE29dOxYr9OV4JhaJOprtph9zvDQlBsktA4gOWfl6QmiJR1UYoJ5o4zvajd9WWsOqoRQjJJhQxiJGKZ\\\\\\\/4In7Mu5+ax4gfhbUjVmB7ywfBrPIdx3756b7vDqhuz\\\\\\\/\\\\\\\/dZx\\\\\\\/tDxDo5muJWlq92iCEsm4fLWiUFnsBTIlRE0\\\\\\\/Ql5aEWCNWg10TVIwdWeThI3DzOGe4SwLwXPFo4BZFwIFvStzXLZmCDZdZOp2EIfjJwb4JRmlOdd9QEnQQkaSc6wH59FunrHIO9A9uvrfHZNQhXmyNSHBSsqkjiVxu6NloBezPlJ16iJLf2byL6bnGfAXabxYgibSYYmWiNlObgNoDkGX7HrVP32wOCW9Lq\\\\\\\/vssBb+VmGlB09MWJY2M8GHUoXM8xPEJP4dtkapBluPAYH2ApNc0Z9OW33dg7KcVhbqbFHsVrVoEa\\\u0022,\\\u0022iv\\\u0022:\\\u00220AaXJ5uXzWlxJl4HXRHA2A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a1c53773121e7755182d4d54c30119e20ab1d313d7d346505da12d1ac79fb2c8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228+\\\\\\\/CdMiHfz0dYQrcfnlY\\\\\\\/cdA7ETkk2aV0+KRhlitEYj3j7MMH0JrbzX2Koi5Y+SqX90InK8rEhaCWplN0NjDkdBb9nR0AEnDkot4Yzl6pqPiAEg98IhTJChj6v28UX1REfYlUi0kA4oG7Vv0LSdpPGzCGlHPhxkW2Gx\\\\\\\/9RlnZkOPRj8LYro\\\\\\\/\\\\\\\/Loml5NbhU4qBrWPtqNgcwv72TkTTalSiK1cjltul2Fq8C163IwdAY2TcHtxk6Kmw3zbHz2v4IWnGTH53iD1vWHEBG2dkWr+aFMyqRkIQX0IHWRFi2pmYnyJAfO4A3EjIBBqOwZT4Yd6K8mEp\\\\\\\/jM+ZSaPpn6a9aJbEPduEzRAk0reS3PK0ujiUUpoWKqFcQQucWEK36jY82icRC+Ay3ORyT6N5gtX7+oal7TjckeLoDYeePk6WdB\\\\\\\/A8zTkvojeC4tLB+5dlNVyjSieowIfLV2iaj8pLgiHcKTTJu7oliKKiVHF990e00X3xT\\\\\\\/QKQRBIDSaCfDn7v\\\\\\\/UwUBUlN6B3rtxAuNWLFcSFHA65rl2nrvydkmX+HNEVK8PnwAbGXQ\\\\\\\/vaGhNZ504c3F1U0z+JFwlBE3UYNR4AGpGMo6nFjFeBP\\\\\\\/awJp\\\\\\\/eGt7cauOhhRfkPSfhCZzf+H3apSPNNAQbPHdotYudh5VbJee0tnOtyxqQ8Xs+rQptM5HiJQTrWdwRsUn7OHUGYu+A9twOweksi+mZMIFkn0tV+j2xT2lTDykTkqAufZTR0e8U4NZM08bXwWwxDxM1n4I1ZGDd13\\\\\\\/VzpWX+FfMJsWKFtfyoM6yZyTF\\\\\\\/5IvyTRRjpUSrvzkWNtm96A4u35gUWoAAuTMg\\\\\\\/YtGw8zHT2YuAMrJ1WcNeuD7SUaSwDnHgl072WXdjzBdfw80cJteBH7FT7qBBqEDEPLmRdUPqUNoTEBHGIJnPkvPxR81lLo0EKh5AjQp6M2L8vGb3QlzxRsTD2uZcpD6qo\\\\\\\/\\\\\\\/Z17GWrIi3dcJdtCPbD1uGPAHn1nJ5AXBZZmcA3TaJ8aYXBSKpoEJUech\\\\\\\/irKA+32wgawAO4ESU\\\\\\\/G+Z+ViBXKJWov9XsXUU0gqie+c86\\\\\\\/QVTBVn24xKWjzNHaz25gL\\\\\\\/7qOs1QhVjqSs5c3U+IC2trm+3Ud4U3QwXM495F5hOCHj2ZKMOgEfCWaFIwfzSYJslavfXRGMSS2DTYbR3cTAvUSCQ873AXr6UO1PnBARPDwUQ8lPEqBw0Loii2Z8aftEFkhzaL7EaMH0BKEF\\\\\\\/JyJoUSEPjHnsXsp+kCyh1iNPUHctl4iJDl+V4kNxBBRV1YN\\\\\\\/0TFehOoSMSyiozXrrfJLvMhl+nNTrVeBPj5UipRx+lsNp8Fk75FbhB9JirufpZcy7NcLqoFuAHqS5M9wyMWTvurIES8kamQ6XwpJHKFS10rLYCK9y5jqO7kzArL2H+J4GgBYqrfcF1sbf2Hplnw8emBwLTSnKS0bP3xnADzadEq4uSCpj6frwMy2HbZ9RAH2wfuVWMuV4ICxVlllEAosKZj4zdfJS2Pwk\\\\\\\/3e\\\\\\\/vgVQ9diVj5yqWtfkyGjzL0dYZeGHgMjrHdioDtJahYwf0yLoEBSubh5p4jE7xJjz1MONbKTJokneF1ZfJWzRZCgIrzreYR5ilBIUkPcakBrf9g+dG9Tg0cvPONkwt16G2XAgBQF0r7+L6ln228pxYI1h\\\\\\\/LwgoJnkzXemygMRhVMTvnVrtiyItb1COuGMxQmOAkFGm1iv5fF9bvt1wY6U31i\\\\\\\/Duy4G9JG27EYfGRdTiPlXLh3ZPZgAZjWMfNhB0cJnbe7MD1EuxHhKw7cu4unOmIZ8\\\\\\\/Ea2h4uSAnMXekytvTkGnesVDK3VW5h0L5wYhHxx3gX0F15USh36G7S\\\\\\\/XLsaBNaul8yuPIYQt6\\\\\\\/l9o6PH0I+OuoaCxASdkuHx8teLezTpo\\\\\\\/jJhUwN0do8nxIEeLRwbtL1uih+WHyH+Wkc3E6kBlpHtVxg2jiRRPv3rZ+naJtMd1naJDoOa5CJkeKokZWJdwOlf87OTASKKVvFULfqaam\\\\\\\/oV9\\\\\\\/UeqSzAXk8RcVa9OH94zNweBXK1ii63Ui5AO8WhRt7lYtM3h+N2SXwvAwkwnjuwQPkr3Ud3mTA\\\\\\\/KrWiZ5w5rZg96B+YLgRJq3MbyLkeyGm7AxP1\\\\\\\/m9neo8kfEytMiLzSU4j\\\\\\\/yUXbzZpfgIihuuVXNIOIyaVAjlRiD0nFpK\\\\\\\/wBSp0qVHCQuXTdqkzp5ik4COvgVnmhRIrXMyveN9z5n3GDK3rNbuvvnST1aUnCJsychfOpuR7JCsZGN4AioeceZ3QlFycKTj8wy6JctQeVj1WPVpdcs3qQNZKFAmhRwVJecXK\\\\\\\/U1otcEpM51MsapNklXyJxQKyPwkE0KpRUgMOj17\\\\\\\/3hSaWTEbykNeaejZNimhmbIj8NLvOyTFxCWOu+Jgim\\\\\\\/gjCSM4DYgKlleX5+XJIBjXE4bvIHIzmeRI66JQGXxOJehapZLWvO8LyrljMsI9n9hJzdEkuHSnx33w8ptt7HcStT3Y8t4GOzFD7cDwK0PmLMlBXCyw6bw92T\\\\\\\/2+XUvX8gRH4COsUCx7E7v2skx7Y\\\\\\\/LQRn9ayDfq2qxwnQ\\\\\\\/qmE3Ct1IEzn0\\\\\\\/me+xeT6rhwLFr0oC43DaoJfUKWTUsJC039zxSD7QhHO9jX7Yts9jhejnPh\\\\\\\/8OZqe83B8XEl0TKRs1EWZHfO05FpFltENjZUQfPiKa2iaY3vjZW18nlz+qOxInoleZpz8+PuTfnhLwNFaYkLVRAuRtMlodlFIzYnfjr0z2vQnmHLZtx+eNBPLKa9VTtufhEgaWxSFJXQXq0uHNLPwXWZY65R\\\\\\\/ekYrdlPPR8wFpSylXgEoeWlbyz0rjkWwLGeaNiIAS0Vx\\\\\\\/bOZCATH1iwCCEg65TZJpmd7kr+nbiC5NRWHppAG\\\\\\\/0R4abAuDwuxoUiAPeGmM0o5RdO9GtCEeGgxCDgzAIikkoEA+D+d2qH8n+8TQLufmLC6Lp5wn9aSN3zrUTqhzCCb2K6rUfbWCaezy0x9865qntRzxAIUVXJNKtkeXDwi5Lu1Z4eqtuIE3PgVpqt+iVTqmWgexEEVvbz9gv3GXyztsFVKmAoRccwTUjPyuEI0U0+YRy9xXYKbVU\\\\\\\/KbQ6Bxlg\\\\\\\/0WknQRrX69cfGNAaFxQNUYxhUDvbkUyJAU5gfQlaZE3XmwSF7sBrWLkc4XFYlKUADsH89M1xhRQBp6\\\\\\\/2HcZBNA+8ypLH8kuI1M29YseZc\\\\\\\/WaCxmkvPUatX0VbJrLqW4x\\\\\\\/sX+lqkwUoiRLvLng7zX0yew6Hk75N4C2j0EoaWkc5qYMvPyEa\\\\\\\/jouJAnFN9LsQaSgViOUOyyWvG1hbByrePEwoei7xrwQgEhKkuB69sflgcA0NTpz6yNLMKn5O9vVCFJv8Zi7PfZrLbtuxGdm6vckwPC9BS0lKXcCALLHEqO7zfr+B3LXxIqsSRzHWFnzw0fG\\\\\\\/kVWvgSBBQxJZVyqz0Q\\\\\\\/EhUm2IuXSkdPNp0gMO7Sih6jdOovdDXbnWs1+Mo5pz98QYfd0fUWEPBDkggSXIUaG4TTmVhlg85lzVvcIZlRRmv5InJxx06CoMb+eu9LzVg3nDGpDgeUEYqi+M\\\\\\\/BuIm0PvZIFqHqaZiGxCrk5n6TYr+9oE45qkJHINuYazx9JzV0lkJDQLKYSai8e58HHOYdTN7zoJ28vKbhwRdxAB\\\\\\\/ZCZVuOb7yKjUag2Izy03hax0Hu5VFfYUnLFJvUwK8XQGfUhCwnV+M0GJBOiFLaekZecLAP7eT8ZD0s4WN9yeWqn0fYu4PiMjYb53osDZBI3jDGhNRSSv4k003B1NK7E20065lPDtV948bNY5I2iUBV1hIOu+\\\\\\\/SaxNV2HwBkNRWIuq1MvyxO1I6Zc8aKGpSYY19EUOxa9OvAL7bvZ2DCDjtWm7oI7THBXNOPbJoq428pJomiY+\\\\\\\/qEHMK2\\\\\\\/z5rLO72CzEm4TRruoKPeHSOsKLDXpWzPibnqyWxjtJS4jynqiyPav7low3Z8HNOIliPy8pJgXXi+Ao6EgosJ7esWALaxdMEvkI\\\\\\\/zNbcuiEd0YCJFigslHxsLaOONwFa6ljRF21z0PKMLEvFXEe9BSg2wHLdGTJaM2N1ry7K7ILDM2oBOHs597gNbKXKgFHS\\\\\\\/yrLXpUmmdFZfkzFNhP14r0N1vy+nC\\\\\\\/lzxkejxmS5U32p3zI3zbItVPaQTBCvMEckXIIhoF\\\\\\\/TmyAwuIBkxAE3DNGSM0AwqHoJ8YMETgzngU57OKjx2r+xPIXNB3cY8Z5iyc5fanSONDhhWjtkqsL0\\\\\\\/QQN12KRI9NnhYmNk\\\\\\\/aMExI2tFNpEVzUq6GAqGI6GyW4Xt5TcOFu8d7mz6kXEn+v92rrO39cdGNZqBDskUT09i7\\\\\\\/cDgDIs4TNG1nvsK5gFYyzNWRCxUweHsFAYxT1\\\\\\\/cApradKfHIJ0wtPXINvAFlUiibKHXt0SAkRzHALEIqBkvrK2ckS3jdrOaYQDcm20s7bPl5\\\\\\\/d6t21mbKBDWhlwe1IQRJy+dNQkCN0RdhciUpz3zZjVC07rF7c02y4vB6eawritsG9pQgVQZ0FTHUDUIUWgKbr4m6WiKf1N3HOpU3XGUaOrh\\\\\\\/BSF7g8XANzBAUzAARwb5s+g8p79tI3TyWA8\\\\\\\/K6A6bbYsAs+5ueIbLxXx\\\\\\\/JLee8FDSn+YQ9j6Ii\\\\\\\/GaMEY3ulEVy8h0gGQAvG1JPVuQ\\\\\\\/CJ5NxcHav\\\\\\\/2QvAaPQ5AhhmYe\\\\\\\/27NFFu853SAMk4BX1DWpwuqNQXSqibR7HaAdc2LH3Q\\\\\\\/XrtVtW\\\\\\\/ABTd9DjjA0n0v\\\\\\\/sTJOEgmWqRrXs0W8ozD1TBcCzR4aTVpc1h\\\\\\\/QcRUycdQ8sQwQWPiPnXP41tPD678zhMLDIlBc+MzgpzbczjTGb4bn3ljbwtRrLYAdFi4G4szx5Btz2uol4U7BVE3GEHSR2PqiA0Hjqq6T7ILfgEswTPDT7zVp64HSJSZeSbfy9+AC7CkxIngz4pkF8H5n8DbMKESVdx7PAGpXrYZ9XkPKMYIQOISYt0N9TXDjBu2d0qWco4uMPZh\\\\\\\/66gAVzR+jEJTRReNVYJaiEIRjqyMaqsUaQH6YmKsoH9lkoIPQG663kUIVe1rmmPNOewX4fuxH3zW5ekPnGU\\\\\\\/YP+YIsydaZFfsebzitLqXyOOFGM8FZledaYM8luORQwg2zYMpZ9UMjTiGVdqPjsABol5REm6f\\\\\\\/oUOQHtBsgRQAcrgZ9tGHcOa3iqxlHfTS1tbAber59klycFY0au51kuwKdAzhg\\\\\\\/7OzY9D6YibIvYGNGFBYn+IljyEIkZ6bgh6q++ebVaG8FgsuFVlCUhFI\\\\\\\/lRkjc4GoamXhLtRqAV3IucF7nWC6ytMevYmbPzhbwHdgGbNrN3ugLR0izRnVCAwIjDyWS\\\\\\\/3QjWlzMNvHaLsGnBZMxmUhWnMbjXimuh1C2ssQaqHLaqUooWAs2Lhlt343dKR+SsJ2niCfVxeEA4DfsKsMFvGqyngnPoQdkUzbAFaqd1qQS1jeAAvOuVBm3HOaPfmJ7vCFKmJvsdscmbSBquBoX73A8T5nO7GgUlyvL82RYrXd\\\\\\\/fUpzheX60CltaayBR7g+zaPls9mrtZzkfqzMQfxcseCjvq4N54T9n3bOXUPobdGpqIsz08FYdlFcF+CU0nEkLpzIBwoWq+qmrExP6FRiTYGPgn4YWATCRfMJHcpTXSORhyV4QgQs6rgLhc+FgFZYDJKASj9fRpUNGCDNNu34ERcuynIIj\\\\\\\/5FrAkVKeuVxziTr01am1uB\\\\\\\/RrOup\\\\\\\/sKPm5JhVIeEijDhwa+NQOx0GcIk3wZA1Js+ux98reQKhyJW6ER+b9S1P16lS0hjVCbYSx\\\\\\\/5DKzvhd5VJOADCQhIREGSt6fXPzBR1YAz1AXpIT6h951ZKoe+e+\\\\\\\/pAR+cYIyP3Fy23yg9mOvGGX4nJC50mXeMP2QCru7r5i3vWGJeVHZlGRWHlz55g3B2NCRBt5Kc39NwbVL1gkalOrRsNKMAcTEZXCZFBwWjZwnjlGr0GaL2DK\\\\\\\/\\\\\\\/XpgXPRpjXk7XCMyB7PyBydPHxaEUNXgnlCL+yrpre0OHCUfwcHoxLDrsCQT2KGsnEvxwvN6AwgM6Kt3tN8Zbb1CPznNqyTDmaTBn3qlCWD7Gzj9+eDirSeo67CfzJrgGOKKox+hvWyW9\\\\\\\/w\\\\\\\/Eo7wZcqfzCGbUBpsmXyshxW2tQtLx5Ue8N8TMjLsLZFnfGsWg886elZJKwuRtwC0mTfdx0UatD8QmocBhxOKvg68bb2+cJK2ssMcCDUuuejSX\\\\\\\/sKxYUT9LLM+Lg9+PRIHYvTSgUhgHTl1meVsWraaxgPivjKGRv01Ti7\\\\\\\/qz\\\\\\\/FOolYW9XVk7jiHkYNGKurvhyvQudlF2fcO0RsJ26dNu+OUJZFA9mSOV\\\\\\\/PPkjUjclEHb2I6oO8+vfWTr50\\\\\\\/sxaNLNtcAnf7QAKk4brp\\\\\\\/RQYSlbbgbjhZN4iWlVyM3kNBdZy1rBfEDU7rKQNMx6c0z1e9tIlOzyAiG0tnZz2oZCKw\\\\\\\/G7CU1BeJq7KYpVIq3Hb8Vj2OIsQRKgo9mbQ7F\\\\\\\/30xenzo+oCrjlmksFkeDCG7eRZLRLfnXC0HWBtwIWfhfmznrwsGdtkxi+3G7JmUd9aU2KiG+UNZhh4sAZI48bp6d59fmfzlybfPwQSZknd13pUd9KGpbdUhGZn+y+Lu954Wn31hH6\\\\\\\/cR9LMYkdfehV7j4BtiW33XF5siug7Y9832\\\\\\\/IQ5Y9aBjJnkHnCP3kZbHlgHYCJxmCWNbvAApm4x7vgZ5A04O9vC8ppB8Au0wS5QqSe8krFr58Wcoz8g\\\\\\\/XpnyAczuEYzq5KsPAbN1vht8kuOCTDhfEuPfD33uLKjRPR4ityPQgpva8LKcm\\\\\\\/NMOV9q2Iu2VUYZAqaz39Ofi08irJvTUZoc4pP9O0WG1BhmuiqFn6oI404ee8Qa+B7fusCuH3UiEijtEYroUw+MLkOx3svCjPWdkC3dYaBhN\\\\\\\/9IJEotja3QIOSw547c6WrN8Uidr9EizEVb16aPKcjgEB5nB5\\\\\\\/vcrPUNdrKp4BEWe2MrCOORxLm1ZRlkaEQRWZxSTJuWBJZWQY3tenC0yipXGxTno3xs\\\\\\\/cIOOluqEH\\\\\\\/z23xn5c8r1HhBWLhpLf2sH30u5y9b98VH2UCFpBhPS9jbAkGXuMP8PyZ4ConET48c8cfJq9n1pXNOe9+TnUl5ydp1AQaRVEYvmcxegz2mIXomlNQtiQ53kWwrVLmR85IzkfdBGOaeQupXq4R7uruJfK8yasgKWZhvEasnCyJP7Mb60TtKYDh8UgknnabLNQnU9syt8aR2tKMsA\\\\\\\/LIfslyZbZ5GPEQvCEqnhmt7YgsuzAaJscdiRTaTptYPwFjH5m2\\\\\\\/LVDDjdKDd\\\\\\\/qLXNXhoJwvVulppEZPSixtgQEkxuo1Y7u01WByTSronwsgRQYS05HEYuyUokNyiaNwtKqOJerzxheoYC\\\\\\\/IJ3jU\\\\\\\/Z+RcoyccGoAZNZccwq6+qgZcuJByzwpBl\\\\\\\/oz0W7HS6dcmcb+pTMo7OgnWn1ISmLpGp3fU0qPdK+utM3vS9DfZSO6Wa95hsd1hAAoP0SuO4O9Y+I+m5nueVUGwHugTtcGhV1aVDEAAXrV8QE8qRfOaaFuJXikn5uLy2k7Dy2wZE5oh0q3CQpBC3F4+iEPt06rB1rLF8RRQEsmXTp07CAb8LXG8CXLfKrW2XOzKv\\\\\\\/AoM1GGpNUcf8nqt8NtLlxHbhIVmVfao0JctLq0efyg920DcEnjDTF9y+RL4xjWejctLbZqaSsAceAZD9oDFQvGIgSC+QCRbn7zhB1+WPuKeShVFpcAeGGIhBlOIZJRa72DrFUmxpgA+ejqFvclARM697f\\\\\\\/5ZkAedY0hJElT1W9xjzHJmKjluf5xZmvcKYbFSzGFJJ9ZHCV\\\\\\\/kdad+9h0bx+F+nyRQy08Gww5PnJcW1YtAYxVjBUCPZdEe\\\\\\\/KDOYseguoJkrKO+c7uXKJxNr6D8dZnxLSNsBUCrRpjWdkYOxmcHKhV4OyRdhzy7E0njTELHWOMnssV6UKwYuY+EbM7nHEypL2AvsfhO5dV5I7NgQP\\\\\\\/pzpcLBIuZ2LulXRA4plykfphRHl4G+wzZdkcAO38jMwASa2wSlHTQnSDGZlW067pPk\\\\\\\/qUT5g4oaUpm\\\\\\\/OqeajMjGQ9JSlkD9wIF6DRnJtjhIgC13xhEObDfFdn+W8FgG2JtKBo9EUqux2xSGkkVeGTeFvfh0fLf2UY91CDrfH5DoNC6zpCqdIkTcyNB6+XkRL6rLjjmpYnWCbGC5C3Zv56mVADQMHRKoIMN9FH2ewcRPoVL9W93vsrQcAOujkaYA\\\\\\\/ntAnjXTfdvgHfWy8vUYjRXumfaM8t9dDSzM282oHN+O+sZz+UOthehLatPSNb+OMA3qs6qvvNLt6n5IkzIdWB8wAfZPgMGEQjk2Jk1\\\\\\\/UYyDJh9SCC4KBF457YZu6PiyenRHUVqFh+2HEr7g90+b1j58STnLvfbZZxHQ0\\\\\\\/os85CNHkXN0CFNVOk9jFzmghvAUvnOcsjypyflsdwwWCYiXsEYr7xPO5dQJk97A95oJNd6S+T7wTU5fO5mNR1x3PzhGqAs7U+je+ZbDhXTWutqluvGfppINJk+iJmkizXaLilyMtz3CtA5OwPAkHDnTJ2gav8SJUkaXY\\\\\\\/PUXs9TwB770kmyh5YikEXjF2jj+r5WlXCRlw0cErP8Zg2WbL7PIILN9Q3Chqn7Et3fq49FvDcKqhH34UPJYcD2GmDteFnlXGNCl5fAV+hGDbhJNc9fs1dXw2fznjGOL8NYHVB+3l4GIypRsGkvCFogVm7DhrBWlRVB02Tzg0UStomZk\\\\\\\/dxRSelvucMr2khS1+ytB7wW+HOVUMRPmbn6JP+NQCwgsTPCkzewaBmuQMRuLWNvVoRA9kWajVyR+7izy6xf7hGm4JqKJu824SvUoBlyTkbSrw6m9SFgJXm3kq09k0fYFcRJZieYro7O9llvtyL3jGR\\\\\\\/05SbicL9dW69DD6z\\\\\\\/NxL4LBEhPwfDIaMoT\\\\\\\/lhOdbNNYJTQcTnzqS\\\\\\\/xSN9O2rEk6EbNoccm7a\\\\\\\/grdePhN34T0LoqB4meE\\\\\\\/dNIJerIyi51lT\\\\\\\/ikejVvmLzySRbeTMyI\\\\\\\/yZH3x6fdNB0gLmewruvMmzcVKRDAyLbOW83oqFrryluQvI+OrtaaYBNW9VS934\\\\\\\/7akVl\\\\\\\/kyKL2IEvd2zgatHZTggAO4UYfT7Uo\\\\\\\/bZD+RQ82nW\\\\\\\/w740VuKxi5Di33Toe8PrhsSxZNBRWgOOvsFWlZc58F2U4B3zaVF0i\\\\\\\/gq6X6YVOU54VClqD0PSpLPpdWKgAVkNTHeSL9bKCew8DjLXiXrNpKJFaso2tB\\\\\\\/O\\\\\\\/+++hc37nQe8ZT0NIqgcWRB\\\\\\\/dVQHha03xMwvLFt+vBWnDfJHyjiKhJ05akmp0zAmh4ZH02MazkzMbiP5w7dj05sHaPx8HBpH1IU+Zf6bGejuZk72licb+ZYv64eV6dDBX4ZOCSRnQSWFq63U2dHNupK2RTI8AFFvD\\\\\\\/8F1JztzgN+44s7Se8BATzjuRcWXe8vnBG64SEQ7MAZqJEj2EPMXavy9\\\\\\\/TRcM9AZ0HxTK3maS2Ma8j2SU4A4xbTHdgYg2\\\\\\\/STCYcA0fOTGqWzzHDuNBLSyqBHXnHhu4Mzzchaf3JJFlQN5AOLe2CHPYXl5r6Q+wr\\\\\\\/wtYkjQDTMGwBtT5ywCzwSOwdYtXb91fadH4rAZwJIBTgZ\\\\\\\/vscrVXpXbKTzikUIK2VJEp3VdTObNxCQH8\\\\\\\/JtMqTh+M4K+T\\\\\\\/PCNG6VZkZ9sjTYnzWs+wf1c+YXn6B+xTrF42WCA+QNUG\\\\\\\/jMeZaeLOT2j5mLX0n\\\\\\\/GzWAGXgTgcJKPRU7XdhmXsFs2vbej7lCQR6F4ORcHConJhKx\\\\\\\/WdmsgOsFz9MRHG1O9VhJmNVW7VeJ8N8mrKSYDC8c9RlweGTfLWzmC0TR0H7mK5lRjoHPhnhl7JuXrP782lKTtGBMkZbXm\\\\\\\/dASVSk3IOi95QiPi7grvUmCvz9CIWA+57YiVVDKzZN+X8nA6wTA2COq9OJmjJsfQQLj3vfJJOprWVFkdsHh6vuuyQOVLpqzsg6OQ9nVog3iE6kMqsbGDVyarFnuMixgu49tKN+JNx9cvuVXaXrffY\\\\\\\/xUCroluEIuEhYiS6wj08CyS038+3LvHgwSDseMQ8ZgnvxcOI8PvAe42CYC81V11i9zz6XfJ4lfnKDcNUopmsStNvzzkUyG7Kf5MH4hn7pKWC0aoiBFbYni\\\\\\\/BgvNyrQiA6MD4ODZk8PVB77Kbrx4GryPe1JTdJnuYyVYz1R5wvLioC55nAw65yyAeK4p02MeZiErY0C4Fo9lFSYTHH9LjNHDVvzZNnua1a8988LSoZh0T8\\\\\\\/9ynU1FyyUQCWL9IyDTl6lgrVDJUTSU1HBs\\\\\\\/kPuYyHrDWd2Cpm7Y\\\\\\\/A\\\\\\\/JjLNJGc+iEmpKhINhfT0tlTfLcSJ1oYvvsOyxLQc5wGNcI4GJXpfR6EE9WdCi0b6iRprAwnPlQM8btfn4sv0Wg9ymC1iHE3U8EfUse+48top8jOHEm632Pysy6UwVAfco8jsJNarYyMgl97ZLk0a6tmDCI0qFbpJkMAqLkjGSlIeZoSMhxNjsDEugVAhgKjDxEIC8GfFU\\\\\\\/oMoI51Cg47HxPl0cB\\\\\\\/YQpmMsCC493ki8ufBsd3yv3nPcH1eR9YE2aSoyAWjua\\\\\\\/0SCmgOwhpVYk0tHUsXEP6l+NvZceMS+sJHsnU+AKjVa3YyJmIe8NubabacDhEPHwMhNHuvBObVqzqAaFgBAEjBSXZ56ad72FvJK4ooL6nm+8Jrd6k2Xe13+yUYCkon5O2LYGK0BpJcL0TP\\\\\\\/kTCRDD79bwH47aYHUsELmgVa4mEzsZpAd3a6RArnkSUibgUKBeJFGZVYGbDekRD3eOC2iWVKqzQXawzASXFvzIJ2VSGJLnN6QvCN4E9rY96N54XCVdS4hZ+gI2Bvo8WUqiNgEuFOOt0yWFlD\\\\\\\/xaho\\\\\\\/q2HWWcQGgYEdKcD6M\\\\\\\/SE1wsZ4m3x1VSzyjvVxV3\\\\\\\/8CZzlNBuhMqBtoC9tzHoBg6i3Mt\\\\\\\/jE6SfM0J3okkcmLutzRR5KEXDPcXHHOtldAgy8ptjFdQwNCT7aghPbq38RzpCHPrnNWWJoK5q\\\\\\\/7QTxX9edQMVf5DCDuOJ8HeOFYmJWbrUApWm\\\\\\\/fzPweFqKw00RYDbqle61TcjVkiPt3YtmrdiXJuJ3+h6Pmpc8xutraPNHThDyYKV8932ThtCoWc2mw7dYxc1C3rOQM80aF0+NuQC8zzufLBUMw5E4idrApvnIeUQfzBBbfJ\\\\\\\/tPTu\\\\\\\/0PPgN\\\\\\\/uZ2VziH1trijFqzOThoVBeTAoCAj9wu2vHP\\\\\\\/gOEzH9wen45mSxRWUsC6UxuoQfjNBHaTiNu5k5NOos9ipzcTV8n9NqTmtI\\\\\\\/mId7CGfbZDdZCT1j7ZNyy9KxTbGA6VVi09ghJDAwadir2bnb\\\\\\\/Zdcn2s\\\\\\\/INXC9lXz\\\\\\\/iQv1gUE9xCygJ6cizt\\\\\\\/rjkPO3sT8AYTqaRpAgXrn+6VafAGpXUSlodwXNJyVxhreNq2XFeDebOnw\\\\\\\/MGD1m8nAbXVH1sdIroPE9m4T7B1ErKSD\\\\\\\/nEHvFrCJ9pYOTI7BOKhG9CMOg6Fqe8WE10mNqUYJQZrmqNrFaUGfyadeAzj25ab0GsEyBBNJ4ASYqeJFPWJOuz7folhOaiXq4aVGVzL2oMLJcsUbDo1MH1cK0osKWcn1dGHKC5AsHpv+i65sD3UvGB1L7FSmh\\\\\\\/gAvtFLA8G8dGAUzzl3pNpI0LMtpq2KmUZD8SRm8c1MzT3RMYS0yQwBasUr26jV7wcanOEu54k3SFJKEtB80r56oKs+zDStxwGA+hhVdFBNK+05jhh0hpBf6nKwev7c671Qp+0ajfJV2nVQuiPUWBpeE1EL+WS+XLxcCSJqh\\\\\\\/ShuHydk6IRkX+zr4f1qGFWJyQKulNueGa2f0lHtNDK6TuXvbLkj0aTGtI+9PEaItLy0HhjCiGLVimjXNW8bXNx16+IYpxEx2OcxKyV8MdYvObrJFCoH14HAa2NGLZl3gpJCGGeQS9PO+M+deFds4DXmoR\\\\\\\/JOQprx5ZOHG5ZiYVJxm2gOa1lOgKyf5rv1lN2qG22rB9v4zLGFGX\\\\\\\/IujmTuujoispwrq7nfZwcb2\\\\\\\/lMZ0SeOWsD\\\\\\\/lIflgO1bcvRhBfNEMvBKWbNuvgwJ1N6O2z\\\\\\\/6p5FB1yXsHgG25RweGhsm\\\\\\\/MOjg2EJASMOk5VDv237JbEHT2HQQ72Pr4BbazS8rh6P7BkRW1Fwjmvf\\\\\\\/zF395Cj0VTxcPgZ5zqE2BV7VTtOB7tfv3Um4VQS959zp3IcrRETb2pLhCfrY\\\\\\\/VR94cAtXCpUVgTaa74vdxXqoTyUsZwldCvVku\\\\\\\/klyMj+X\\\\\\\/qhw0KVnfgfFBIQ8dw6eyuDGWVm1zvl61wRXXSH182r7rxDsLIKIEPZGOv05mp14WLA\\\\\\\/P8gCRptRuTQoRoOcvwwSuCvlTxphG6oqhlEF0fNefzt75U1hxZL0kLBrjWg\\\\\\\/R86XaLprY1kDSjbEjOMBJ0i\\\\\\\/kdlFOo4cWiYIYhnELEMQKAVfH+VbCdEjcEwihIIBCjdgUDhJb4ZqPSh7szJqmn\\\\\\\/urManyvvOuzpAJ9KXqkwuoBqUmJKR9tYA5HXKmP3aV76Uv6ncdKx6OqrQgoIkLv56e2gTMDqbgaN4dLPcKBUtG7NZeC+BA8utvLt8lo729yveFTnk+T2SRKaAgnRPOLnQQFYH\\\\\\\/59tZA+0AxwxCPiAFxRNVfiLoyiyXfSOFUdLvngllsoeC6MBGvX7rqc+yRNCG9Vw14ki6AFN8K4UMeya0QaFf8MX\\\\\\\/kRWZwGo2MWmyG8eJVmHEvb6q3x9OFdLoDnIcyzanBmXZINuPDmtmddf4cr42oy6oAkAdtO9LMPLSXB56XGID9Y1KltcmiKm121Bxul7TW5EEa1jLiPm3hCXo1lZvWRPlLlWICJNsKBUUkuB6KgP+wRtxrowvnFF4UcWIR1mEy4f4sVO9ZJM+WPPR4coDPI5kZqLTsInHX81TI9KGHFHe+xnqvCy1DkGsLVUF1RYVsNfEUhy86H8uHu17LNhFp9FNOoo94YuTe4NN6EIcrymRrno78c2ZS9aONQht2rq0Ph2HYIOXEUwnsAEyhK6ixm59rB0SX6WSxYxw4GdS8kvYNeH1khzM\\\\\\\/eStm9p4Ns9XlxIy\\\\\\\/TBcQ+zKU4TDO\\\\\\\/rvNaNtgW+MGhexezt+i9QwUfOirzfHZx85zmTpfrQUY92O3C2H8fCs8borxfnYH+b5v9UHQW9SQ8a5YMkyl5DEr0c7vCpKuY4LTu3va4ElYvc01CKJx8Hj5xW455jKMjVOaKdzp0kKonF+3PxmCXFFWQbJvEH\\\\\\\/o6poEuv3EaB+v\\\\\\\/\\\\\\\/yQhfqIHSaR6npEDGx+H\\\\\\\/iAFa1X8uXFGUEJA+HAqICc\\\\\\\/VAyxN+Gt87RN8pLSXOQUPgHDMkFjL1HkxAmqaqBkNOOcSzbSNMD00Kaz1pK5UJM8j5rXImsJeoNTY48Z71ZBEoaQbLsIt6VvpBsGyNYIpLqldg6gcH5BoHa3h9hsBEzs4\\\\\\\/sdzh\\\\\\\/jkbAIhMp4ad6B+Pkb4MW7vQBHMoARG4S8Q\\\\\\\/1qEVQZTZEH1aIlnugpMAf0xjvGysx7QRLlwRww4RonEGeWFFWGof32Hkyd5w+cBMFXkWHChoIL82+ARSFYQ0SngnP7uljDriCZ7yMtGUQt5VynZM5oMp7XHm+AXXr+zyeIiE2FvM4S+wQi2HFnRc73mEpor5dHBvzSabGd6HQhb\\\\\\\/+1iSLnZWi4y30IPiQgbVuESbcIxczgY0fuRLPAi0CkHc2jxV6PvVbOXuBftiucmx9gysWPPa+m7vF0Sh0Cw9uG9gk15L2o+J0fnu5iBNz4dMwLJuPQYzEHHZZhY4akt80282AZu0By2qBAngmSo+8F2YeKWL2iRSnc7Qc2ySKUUM\\\\\\\/vUmbf1pBbGqyWgYJOe2Z7kGlMdLpSew+D\\\\\\\/zN5yc41QSAyGTjYWdyA76YSmCyo09tJh2Ep2TdaRPBvvPOu08ibPxiBmfjP0paAvExbrGa8DxmZMEeF5rny\\\\\\\/Mbxyac\\\\\\\/BXY5xYe1atW0MOPEZMNnhCETfHwOrtzku10YKDLHBPpDDtqGVjjrTDCMvuxIFZVEVbditEUiK5xCDcSl5DEEOaylKOY7LzCGbTZ0VuxISv5rRdjahP+Ekjr5\\\\\\\/4s85FGhsLJKk2lZTt+9BonFu5oXUW8fyQpdYhTPLZ7G20MSJWKmSC1J+H+\\\\\\\/WIDi4ttD+0co1sjMw0oWP4Ih6p2n4sAy9QLfDCrFh14TLvAvPhNbzhjYIGfbbB9xkl6P23alJrtsDuTOrx4fw1EpJXH1flAl2hwcQ6yzCSN3ziKafZrerOmHDAl8+JYhjGeMS2wHyBTI\\\\\\\/ojjwVcqUrA8xLfLHkM2c6WPSZNOeVS+ALlc1bdx8vX9yZLK+9TkF9pUQGoMwlnEbsZ2JFdGihcQGsycsEvRz6qoqe+BU4fEO6iT5mt1\\\\\\\/7IiE999SNy7N\\\\\\\/litMh6QPDOYuwBFkABJr8DwDBzLKGWJYRNi7JuyR0NAor4JRd4\\\\\\\/E8SeJ0EPsneOR3eJvwxmenae6xKMcPDbGLLKCaWe6Hr\\\\\\\/u3bOe2gkyvl32qfMKZrZCgAk6iOlK2BG5HOUTfNghquLovSs190Sdoe5kktVjqDmxK\\\\\\\/GMi7D9stbFk5xAVBvKF3GxVkjOf32IPIf4K79MrNF9uXrWnGPs5FzYZ4AoPXeSqTg4H5PmrlvAHfhhMWHMACNKinOmXR+ddWYqpJBBp5Wviyl\\\\\\\/mhJSFocaJYE\\\\\\\/ICSYAftpgo6jGm3SiHpZrEOCG0ObAVaUuyFvNeZnJZSl7SMts0PGKdeS1nfB4fS9FUshtXwrPuDTTzXKzli39Ge0EPfCh+AKLOcrx5IN5ZeL3d94d4HnJ0+InMSnofXT3KikVylTmPYofWt9\\\\\\\/QHuQYRdrb4GHxyKcwa8EeTCZaWJ0oDJT5lGvHC5CqHwpbJJzpUD+sJZiFCyq00ZFln2HwOaHm++2PsV1JrmL5OhlR4woNAAPmSgWCz1CdyXO\\\\\\\/jlvns7cXOExmpKnm5GNFQJnUqsf3dcmREThtx2nzDK30hBS0bEcmk+9307NuN49J\\\\\\\/CYM0fgtTrlilQn+3ymxAG1k\\\\\\\/gD\\\\\\\/L6RDfIMXLv0cZhfChaqxaP8ZvzWSPJpSWoNUWtBeAPp3S5WawNbdqpu0Le1UrYmbGO67ZCiXlnihPsZN+KIGsc0Fh5oY6ad2rnpDXyk9V+NJEG14q\\\\\\\/qec+CXJZOTLo8jv9OuUt6he4RqqK6h+X+a8W+Y+VBrnKYf0u1BZkBA87J7\\\\\\\/m8b0xapQ7Jcx9wPVvi0kTbknMZrgCD\\\\\\\/mXilkBxeCRHErfSgbZ3ex\\\\\\\/HnZKdgBZjemHS\\\\\\\/QzNmY0JT4hYLDZJJZ3a3pdREmK0cr2f\\\\\\\/lonpm8bZ59rcV310ZKXLtYu18ElCN\\\\\\\/oKCxK\\\\\\\/0rMFxR9+uUaedaCJadPyiY1dABEiY2bWwQ\\\\\\\/7Hqoqv0Ymd0tFWw1LHMaAVSd1lIIyXx\\\\\\\/tl\\\\\\\/ZWOEY6KRlJeq\\\\\\\/I4L1g7DGLsnYR4GhyNVQei2zVAvecBsis7Xt4GxHOy3l2q2tegR8KTOhdfqx9w0lt2cXXlwJKn6hr4MlsuWrlGkhAj4YMHFMsVF3xUkzGUVs8GDSR4dU9TOscO+SgQgpCxnpfGSF4fW2bfPTZa+YV1Fs9FhB7JnBmXwiCwREi2tTNuoJ0Yksvzy9PNAqeUMI92LpaI39svkpyn4dui3hQt8\\\\\\\/U8hjedUSk+RYR+oXQvKYYBpdw3dik0bZjJXMIgnP8yPqboJPiUjBx85sqXSRMqq4hNUZmZ7IuSgVwJt4BYmH4K\\\\\\\/wT1c6eNOnENeQeDii4OG4jsG5n69i9wFm\\\\\\\/0p8f1ZqFyylTSQXkbPu9JKQoYMKoB\\\\\\\/0lOZa1xG7YlFpdhtVd0Uvh3qDkGOY4iYPCfEo97MGZyRG3a85tQi69UUt2kHr+RnAXp7OVpjzV0AoGoI87R+xxcTlij33mgrMkN2I36uURrxp4DAozk4WTjUmuweMLED8X1M\\\\\\\/l+VYDFpyr8VzI6AT1qsLwqaLN8t+WLkAKe7isTPXGGkA3zBOeHj5HrDgx2yocSgapA61tyHaKv6tZLZdvEkm9GDXr+t96RyJyllXBfQtQ1puW11Do4dPIzaLHK9Y7rtH3vsDT6A2v\\\\\\\/mSvuB0MqhvOs1zTuCdrR2zOkBeZ66EnNgn2pTwf2WJxP6nqPovQ0+Q2R6v7pa3s1xlW4Pqp0fzFEit0M63DdIxU0DLSWZwqZQEQ\\\\\\\/yoZa9m5a7hfRdICdxNOoy9VX4mkePpGATWpv5IW\\\\\\\/2GaPDC35CUOYQ4uU5\\\\\\\/Dcr8ED6MX40mkma9kczwhj3VYzLWdDiVpfABvFcSW53FS9U+4uoGi4tiozTd\\\\\\\/nVwoF7Hk159x4ijuWZ3RfF5d28W3XnHs\\\\\\\/vHOKt\\\\\\\/CeCeACCmVHGPdR4KtIk8WIpRf08cQ6+uoXaE62kwR8scP1DgdEvtc2X3SZqwHZ0nltykxLZU9M6et4WEJjlRg9NN3Fdqe088BFJHvqLqmr5r08fsEpz41kfYEmjFFmsGWr8COY2pptnaOPbfqvf76NjJZuxzox2q+rxmj4LklBa6l4RNDK+JXu3Ai1RdxXBadTXsAyA+mKjdZUcVWYkHZbO7y1OpujS6QxH+tua7hIwdd12vtQLX0OxnZ75A7j438XWeLO40NY7g7Gq6CxIgWaGBig1v9Rn9\\\\\\\/A7ZILjluUhgWzZW6F3l8ACzwqyWIGbVDWBm\\\\\\\/AGdtXpOhfoft0m\\\\\\\/PRia+bf+wcdxVwD4wv9aT7nSd0clqXLK+S0aIBSdN3pXDjt4L\\\\\\\/Nphuv1HV5BabdDbP2VBnCEFmpZYph+f9A7hQ97x9n5qp8LP9KLmrab\\\\\\\/v7GBPSkuNzH40\\\\\\\/DwlbQMC6CPWKvAwYBtqsiFpoeMCexSRMSUbpIJtHCcjAYg5V2tM\\\\\\\/HmrB\\\\\\\/1uapht2IPB9YPp25HZ8xjIr9B2BiYeZJQFv0933t9eUFB5dHBkOgUyeNO35Yco137zf0pcvOKbXlE2jaKHO8d6S5FXfjIJnF11CvFlj2fDnN6M\\\\\\\/b1knVPvE1QAxEDFnHwBVZkA3AhWOPzkWmSEpF9\\\\\\\/7l6UE1z3vQtSPl13QX2BInLuiX95qIPezguDA3DnebltDonS1yHKKNfGNU27hbjqUWtPlgCsPcu9gO4RrIKZJFubt4YVhWKKZcyfCGXILhawP04dIi81kU1fvtxo5GLWoAYvjTlBXrbqhQpMFwNakN5YKtU67rDkLCzzl8bzL0JM6YCpcZzOgK\\\\\\\/QGyMSm0UzwQkCxGCAzK8U6ZBxBXd5Wyf6Oc\\\\\\\/s5XXzFeQP5qgyMYvEpDacUjvjDD3zrlMMTTGq3QNhTQD\\\\\\\/rgGn\\\\\\\/fc46QmITbbL3Kjrws4SlocZNuTGT3Hxudkf937jz0bsfqfet9x\\\\\\\/IXN3Ltp9\\\\\\\/URgjQX7xYmmTHWe7TfSUTUPmUc4zrQQUsJ0a2StFoxtvnixyv1FFFwgmo\\\\\\\/VtdPzcvZMRfL45\\\\\\\/wxSfMK6BN4qY0TB+Dmx88+uC6yZbOpbm6EVk0Q4V0lSba2pswr5DLrO2KUnBYXTiE\\\\\\\/jIuNcoXKu+TSf8omfo68b6mMc8xlXKrS5p6x8zuAepg7O3EY4Ex9oAjPfkgeSAy1m3a1PtsK4pHVNRoKFj0rveMK0nGXeZ0Dumc3G94t+ID970LKuuAYdKLoZ94Bk9AUtDyDZF+bun0bzuqA6qpbvb+IQ1mEvZgikW48ewLCmhEPEPGS2sJt7RwIv6QeRsWChRvSSinPQfUsAMowSQmiXjkZ7klwnDg0I9AjcJfV4+tmjaUXfawWKo0AgSBcsWGVeByKfxiqSF631yLY6eumQkZ4l32+N9agf5\\\\\\\/rrHaMELhBju27oofsMoG2o686GLzpR7Ylzqqo79v2OAZvZ9lTeeW8Rd21JpTLs6FRKtF3i4O46w2mZy24gYQm\\\\\\\/ixPAU0WjWgFjkdjexh0w4PMoki3JaxV56Lrn23Mjqef+o9EyMBeAlh3rjLjZpgLYzIKS15R5CbNGcPwQunFTUsJhNAQcECvvcGiqYApebao+7vJ+lGpx79FNwAVCJNZR6rJuoqOLypzRddp25TkLuK\\\\\\\/KgGC1EIRBH6eBT2DdijCbLsbFA4M6B+qk359OQ3bUuMaqIG8E+nzcsTyIwIf4xa3DvoqTPm96z0wB14zwsZR0rbtqYL4dRb8UE289+UDe70pufIJ0gi3adL1xEsq1N0hejz1bWalxIS4u82xu8jEsMLTztFR89ItIO1ExfMZJGvVQl\\\\\\\/hr6Q+6alz6BIrNMWSk4FfAeJuniMH+LO\\\\\\\/vPPcmUt0doXfSedZEtxP8+0aYmd3H2vakgnJspUXCgDfMKKpIV3U5+6Xd89Q9zrd9GlDAhEA1F7pAetTY4FdTiHLRfMFxmIoP7Eh8LgYrebKR7o8Ub7hD+u\\\\\\\/Kl7+vsG35TCvsqKhYncpfrK\\\\\\\/J6IR6mfAb2qfQWSymyEDVXJyBDGqlIm63SdSCTyg+AvL\\\\\\\/EjdGLgdkIsB5mVm0NlMCneLm\\\\\\\/r65Z+H7AaB+ZOtyd4ATrMuWmIaWPXwTZGxj2Okc1ENoffR\\\\\\\/qv90Wxilj8nKkKuCS\\\\\\\/8qYyDwY2hFE6PCOLXNevp++uCM10pRxvyQ6AqO0RfIVZ2Mo\\\\\\\/UORQYtA9YZZ+QbvkKsbFZMmxjNnl5hPKsmFF8GvN1B9hi\\\\\\\/6\\\\\\\/WZwK\\\\\\\/nTevmkG4D4d+6l24ZPUH3sT5ZmF+TI18VVwYvX+zeG\\\\\\\/k\\\\\\\/l9dvCyHtCrNIO30rveJp4Ec1Sypk7S7XOVNQtdn4EH+K76Oey+kbOdz3paLoLk1IXwf1STg2A7lKu6y7y5WAPNF1tDnB0sULTrAVbT5dJzYl90hxQ1QkXexRrIuAHfzdCgu1NgrfzIgMrAZoK+ebu+h6K8RHn\\\\\\\/G4v\\\\\\\/wfXxQ9xFbDr2wDXCNAMGUbIC6\\\\\\\/wcT2FAZfltz9MLp\\\\\\\/7rSU2A6k7Y7enlUq4vVttxo36JOHwKTAbMwh6CS+d\\\\\\\/\\\\\\\/p9Fng87VCw9Z+CHKFjdT+OVsS\\\\\\\/id+AO8IAiz2c88bxbjfMfiv\\\\\\\/kC\\\\\\\/kICVGSsUpNxcub69hrVcukdExEEhm+dJFNk7oXhUR+XBgME0UHBn2Jek3gBUCPoDlXnDSMfP0wCdPRLdcRwNIziIWD8F+COxmxO8r4oNlleNPoiiuW8TKW6+kN1Zf8FAbHOKNPaviXTbh5ETrwNf\\\\\\\/P7xwxZxmpO0aOMPxVhSxA4wh89bO9V0CTz+ZxTamfMOVOtluFYAawL7Bru0gCanfCM1ahy+hC3zFiOjO1QO1IuNYlqPgHx8w2pbk14eqFx9vIPsck6UHy9wFkOdfuKAR4qSHJRDJMLiwfNREX4EMCN6mwRt6+2Rh2WjGnG2CRzNfYQL0r1jVGtfPFW4IEQgvmcWv5Jo4djg5pMenpC\\\\\\\/VEF1YofIiGvvpG7Mh90qSBlSlqkzHG5LnnhoMj27xUyhS4sx0Cjq5c7c2yQzgMABhb8e\\\\\\\/09l1k16aw7a2N7h+HR2cCMMA8nWt57C7fy4Amd8OWvZ0yJ4opDIfxkniihmpHhslDCz7Kvu2Wd560gqf6pADBOATct9alMD84K9qUjQgB3buUtra43Uab+N8L5NfQyBC\\\\\\\/NFaDavYdozyfIqiVmYRXN+25CWAGoeytg19Z+9l65bqlx38ho2GW2byV0KyHbAc5RUXA+M8NkFYxQfQHZ9t9M7ZBOxNkngi7lvk9rkzx63PJJDPsHbA2r8DAtXI+TJQk8lJKn6Tfw8uNh7YbkTpmkDZi1eoBEMCktuA26+ZaPmZngzTU\\\\\\\/mHWgAblXu0KTeaObkunoFUsNI\\\\\\\/wdiD8Ques4jk77x7NMM+9XdFwFsLkPzqScEPGcw09XMn23VG4BNxpw5ed3FeLHjCT\\\\\\\/CF2rELO02phXyqjX0DAlKlQ75eR\\\\\\\/2exakR4nKaWuHBvZ4EuejGwdG4lxFkjDz977uouNXJlzrNLhG0A2V1sxGXjgF1ykakctZ+VUwwcvMI7FJT6oE0V9bmsURx5mwek3t3zgy6Mv7v\\\\\\\/aj+hZ7hkTkY1906Fj5YEnqGcWBiauC3p2rzOs6vR7jE++7Y9gKFjLZw5UAiVWfW83pJQpmMg4GbzaL0VZB+SSZ9lB8ZPcK6TpyEncqT96qQLelJs12kYMqm7KhQD+Nuaz3K2036CNg0Ga44NujSvl67YVTj0T+chUajMryfZ\\\\\\\/Jtq7Kgc8mSL\\\\\\\/aIxERlcTaxfcVllDI6ys7qxd0MsXJiFlSR8e0ZI3tzxochA\\\\\\\/7j7jbT4KUR3oYfg49fmv8Z2xprZ9jy0BiyrWwV6MXq+hMEhTTtOOsfFDcfupiyb0tWthNpSbAVx1o3m4qKOs633LMI3v4gw6Aq7GJ3iguRr7hz5qlnyJMUhB+ixJ8LIuicmJ9+Epp1+mCRdY7SXeXTIPUbKCvohVjxeWip8KFR6VdVwfJrl3R3MTjHZsL3CKDjUwX2tjAhoyiXNVlYLNNMVerUD1O8NBXLCR\\\\\\\/4fdxIgqsuU\\\\\\\/Fso+qDnGtEKXj6ZHIOhKegNQid1r74YJjxtB0Jp6eG\\\\\\\/jCuztaC+l0UAe6zczBN4wcAb5CnbnXw4Sg3S3SCYNte01k8pJpsnH\\\\\\\/l3OSfQnwiMHi2cEeePSYo0aCsEadTdOJR+N1wLLF125ilFhfAfKs4uNKPCAcUIdMZOCX90thnP+HXJfbvWbqwJjKZin5eh2++57AxFwLUHiOT26hWnTd8LwnSwppzJLW\\\\\\\/5+My2d3KCxYzLA1oLgZi0H506DauV2DOi3DRJMyEqe0N7P8zcpB\\\\\\\/2eYsFEigFK0gEEIDc3\\\\\\\/OGcKp1xygQv385zveLDupWJCH5bhOnp3N6Px60peM\\\\\\\/u3c3sCvy18HRLoUri6Ui7fD8O4uzkNe02e1c9x4g6YId6qersNLMs7GJQ4w9GFi+Jh+Ao3D7RMdgDi6z\\\\\\\/ImZdUPRVoSFDbGeBks71q4bYVB5EE4nOseDYhyp7Y4J38NuH3EXafq1fcGRhe+L9FZ2h7zW8oZDru4mssFAJC2jDEVU2Cgz2Ndctuvrg0fmNZHgX21twdm1zhYNZ6x4qhl1KqeZ9lKx6LNLzPrlqUo85tJx5CHPtnu\\\\\\\/reUuv1cSRl+lJHP3ua1fVOMUjrq\\\\\\\/d+ikiURjdh1iESUtZDCCn488IXPM7lc1\\\\\\\/kJj9VzuBQfViM\\\\\\\/wUinBRQfey07fqBf6bVbNRHD4KjT41QCKU7\\\\\\\/jRYChTfVeirsv1Kzy6+GHM8G5bpgiWZj5Ko\\\\\\\/yG1U7ErTRh7t2AAA5kgE1lLgP2qtOVbk48KzpPiQQfSIfrj3NeIpWb03TlZPSZhR\\\\\\\/p6i1gYmrLb1133ocvTCPVVmfIHH8P5OJWKretvfrwvjBRQCQkF+4YOJrregYT7G4JSqw0s8CzwMU3J8y5I\\\\\\\/x7\\\\\\\/i7uI3deTG9lHzkbtu+EfifAfrmI2o6kBHxTquymHKg7oVT4ihSJQ0IdfL6BFWFLLb0reKyC+s083\\\\\\\/hKBlx6nrcDaGfWudOyPcjvrw9aMRlQD5yxBUTHrVCpRLeAiyz7QThQsmZAC6OAPwf1StQ2cxmuwfSf3PuKSX0OOt0+6BWxGo7HF259R3gpAzrJPjz77jL5qWWC1iQC2hdrXUpYA+tSKGSeeBHW0gZ27G6Lz7UsoAQEwPOd1hUzs3qHZscMlH7wVWE3\\\\\\\/ohGAjTkpcoyP9TeSStJq2xm+vKQNCKE5o8CSNTR6sNIgG98mqHQoTHzUQqN5GVWpzaEAZ3ahwjsPPomL3GeVhVNXf+Mbuw58DfrDL7UFx3BNWvymXRek\\\\\\\/Laa5jGS67EaXvqK1e3zktPIREhYCo1M3FkYp9l9EXGBlz5UWV5dHnOOyvgEQMBfTHO0VVkDmc\\\\\\\/GIv9wQnbDmlt5Q1lfmWqCgBWVKAenX\\\\\\\/ggVOhzTxLtdWwicygmbw0FWZn3xLb9gJX9v3yZJXmiFoPjRq2loghUNx3lXWPC6jQFIl97IINC5K3MVrtnHAaVPsGW\\\\\\\/1ltfMzmSVwjt\\\\\\\/ah4Kom3efLeDPdYqMakYbEiftQQ9fb59S+ALEYVvxW5NYl+D6ttrMuTkkedjya95bl0S+ZSMmyExzrlBfoZuzj\\\\\\\/lQOnito6lhlRPQVRZeBT5NwI0izoaPUBLOHyYpKIDbyCQwRpYTuOZWwbvHKqfpTDN58rcD9z8BPIJyl2g70LHj05diEA49o0VP9eQyi+6eRV679wmt\\\\\\\/n1gWkL\\\\\\\/2EKiSpDzbyiMmfTcfpFm4V3wQ\\\\\\\/tlRvEPUaf0mdYx2Ti2Ulz6KZDUQit060+jmJEyZdVzKTb4zdEAqf6vT5m5jomvyMKH1EGen1cforEasYnbsg+bZpMYJnfb78DJQHVidEPtQV4gmwC6mjy+ZA9h46VGTGiY6cj+9h6b+ciZQi8BDvRzQgB5JIt7a75nWVhybUsYyfhcpn6+EkZ4MX\\\\\\\/qKX7DkkTp6iWsHIdU2iAhT2zzT8mVEq0wAYkuUt\\\\\\\/mSpoFC0wC5Sc8Kn\\\\\\\/xIUaaqHUzxEgQD3Z2Cq\\\\\\\/hkEdlbXHv6Yob1\\\\\\\/oKlkldHJb9dAdTvs0zplNwFGwQocmP3s3W2Hh3HSc2N5P\\\\\\\/gUCUNGb+Mk1ezbNGybCO13zkofmfNd33nJvyF6IZXFJ\\\\\\\/cHHUv2jsYBdYUQx\\\\\\\/Cpfvn7K+0OTPwaRulOiTn+1l6vq4wpJVJgi5JeMQch7zPZ+Fff\\\\\\\/bQeF33xMwylOUSvqid40dhGDY9IW+NpaZD8+JsX1goiiToxQBjen8W6hZf53Wp8xLqxhtB05wbDIB4ByuUyrYMq3WO+3vCN2KAT8qJe46TkNSOzcSX56Ijqo1+0rg3Uq2BVv9BhHylNv1rxiyw5zeLlxRTdTirzwrsIKv\\\\\\\/RaS+BH\\\\\\\/zTiGpj0tbCNCMMtPcYeXjNiHS5Pwqi6KryUIilIcurQ6yUaSwMYXMgQw1gP3qvlHHfB6n7OUjaHFo1uF9cUj6pXsLoJQOIUzgAeoJbzsnG\\\\\\\/1x+7R77nJ1BJAENA\\\\\\\/xpPDTWWLFHagv8Qg561d5OvPeyV9w9crQW5sL6XttjUCFKimjd+b7xXlL9jutgKMUGI2BO+z38bAwEJgd+RIolBp9WvJjY67yp4spDXlij\\\\\\\/6uUGkKOjXjAMk5\\\\\\\/XXUFBHh\\\\\\\/lBd1T+N7xaDI9c7UsaYDRQuUmd+JCcZ2trW9XxNtOSunoQvGkv9Wxl83UgH7wfeg+XW0I90diI75iUbGALPghEVlgUPrjlZ\\\\\\\/McoWyQiHI1Qry0GdFgJHAYGfoKwB0yEyKMmSSe9EilUJhGrM2IkiaO3JhvKNc5WW2+f8djGCqNRx71PbHRn6LjLoxY9hnZbTp81WhLE8\\\\\\\/XbfogHPzVrITbIIqXU0LuNIrYzDyJrRiaRdpBdcEHZU50ixVURojOslxHFMGoOoQI21QzjTGPTgh3UFeCms5vb5X4\\\\\\\/3RDDX8IHStvdWCyK\\\\\\\/MVcV513Cl5fPhv1Ut+7uOO4Dt8VEfUY+ylqkA0NbQkyqJIim2dxqPa6Cd5WmdMeo5ztzjqBMSBGSYT7\\\\\\\/N7e7oUvhy\\\\\\\/Nc55iBk5xh\\\\\\\/+IrFxEoMSxlk9xJ6u3gMO2tIwIHTQihuHTfUfhhgYj4k9iE6f\\\\\\\/RR6REKRF2wAfeD6pjeAvlgijZ96Zc7YLgRWfoXqz6yVidHGeEcKIwCNZb5F8a6M3P0K2Sldf9VyNpK58nBde366jetiO5lyLx+nVUcdALGF\\\\\\\/G7mLD8JUcZ8zGRQVDCZH8bukJOYGKHRaYXmaAph45o7Xu1zDB+pW3tOR2tWsvdHTvBW83IgXyinjKpGv7gXlzwMWBtZmS2ZtzRdGslUT+L30\\\\\\\/K1plxZr3z6J\\\\\\\/AmRcIa0wylG8p7a0j\\\\\\\/Fz2EtY8CsGmvk1\\\\\\\/sc7OHAqnAjGWHe9J21Tax4WpYRNRA6aqp1nmxOGSXosXIwTrtXt0swcAo7ynUBEjG2y2MoFPCTYGD5UzLmkT\\\\\\\/LYVocq5FYFk+pmx+AfVQ2tIMO+iGlQ+08xcNRc+md5PLoBHOY0mxMDC4dGI1WdczrtcKlmU80ihmg5AbFWT\\\\\\\/6Ld7kNYK1DOklfZn3ilrhRq7eX+blHEKeJVvBZ8keQuXR\\\\\\\/85iO57OEP+XEjmMMqEb3SbDImhVZN5tppTFIE5eFTACjVP1DqaOqBTiJKDf9x\\\\\\\/N5QSZp\\\\\\\/Va6HVxVWM1jj3WSqTB8p\\\\\\\/O9pYhLUQsDjSOW6STUKd8uZ5SH3nCo\\\\\\\/VEMTnDNPCt1RE0QaYTXXt41CWAelut1M1LiHs3x7hg78PVkKrdvGsd8RdbvTW7Pxlc2meH8L\\\\\\\/UxBaDUWmhtdM7Q\\\\\\\/ypjWRV7dZuXzENEDqO3VxVUDCZ+7hXu9W2t83h1+Yn+plHwkvhxghGZTnz4SdVJuHUJtVJbl4KQdhI+AYZXVxKsl85KBj8a3lk5H35ZriaptYX7\\\\\\\/gpLi1ycfUgwmk42UbBczshYl6cDPY6ysZMBPYG2zLmxD8xQO\\\\\\\/EJNG3MHQUnJ6SNU7XIqwlHKAz\\\\\\\/TcxphooG2ai+iUTT9rhskkdr9seHGOc+MqAqWWy7egBj6JLsbAApBKF6+adgQI9a4qxLPBDZf7cYKtPMhtYelmd5GEjrPEy+WPpxsRKPsPc9O\\\\\\\/ly24hW99qxNQ\\\\\\\/8QqedDwk3r4IGvqaDhgpyLAw16W4\\\\\\\/MzPh3rBsJythaR2P1IKDIamuHyb+UI\\\\\\\/VOSJQuzgNR2lYsrGPaKnXTNLegB2TghJRApzUtVVB9cjQUP4qH\\\\\\\/l9tuFjfIjgoUDYY0TFcTKcW7EezTmefOCrtJ9z0z50utaOQiZ5+S1\\\\\\\/Tjs7CVbpIAFxbFmJ9s+v+Px3y3nP\\\\\\\/Q6+D8oeduuuE5oosRW6fpls4+kaEA1NKl6Q8PHOemEf53svgIXOpl8bzFYz0leyd9AxSkPBRpwn0vf7zSiUIGd\\\\\\\/j3Ebx4Dd71bb06k5O9XAPNOujx2mbgGj\\\\\\\/EYLXwuUcam1agMVwZs\\\\\\\/bhE7WSdKdvyb+KLEEAe+zhuTBXoxt6+32ZCtiKepnwHpG8ot5qnYfPDR9ErwZTL8gVfBnM5WNO+hfqXIoxWTeRQYSdg33CAQdVjJK7pGHvc6TUcx21PrW1tSRBd\\\\\\\/AH53LyKKXKAu8AWBZ8SFvll+ttJ7qBi04zQZO1md4zY\\\\\\\/wB+cbA7r2EA7VGwNeSLLPQEgY1Vnjt3qwfvhWN3PhNUkYEf800IiFGvCq\\\\\\\/wq0N8Vz99ZZFOu0Tb8nokxuSYRn8X1pObBJkTRcmo20\\\\\\\/VDHoXEhzfQYTuVDoJrOXECSEsKXucsBRj\\\\\\\/fWy3p6LNQ1lLWbEUE\\\\\\\/zuUoAhoP0s2oa\\\\\\\/KfrPP8vq0+WVXXAwOiFeXqhtbR\\\\\\\/O+G3e\\\\\\\/ow9YlBufWE\\\\\\\/OfbRS4lPrXz1zsXtmwG0tL5Q7TsnnxRpKqBMLcnYKODHrfnzdP82Dj2mwz19KyPIkPQly3cUUvOolluCAwwiBIbVKBeU73U4kH0V96Vj3EC4eacWG4K2hZeccSi21RZYdOdwwPNs5IPwOCWUyOzekjYSQlH6yqR7eExy2SHFi1GaSgTSka7MhREBdvYgl+eAE7PA71LhdizdZcF+Pfgp79qipX8lXYoZ1Qn6kqFTOTTE9d2en39d\\\\\\\/m53uLuGKJ052WabCKmLKnU2UnsV\\\\\\\/dT1zyT1aA6HKbF0GeEbgUxe1wYqzzIBBs9nvMPegAxk9KCxQ07YvMHkDypaB18MeI0fRqs0DQr1jE\\\\\\\/Ewvg7fbVEcAe05pOl8+Tdpf7UhtXHEA3PfOzObZ3nQbgfTe+yuvQhJ3Oi4p8EQUNWMiMfOvMyIgJGLuftgdfkwseQfCV9CVCnxNvzCZcDnIX8O2jYvFezbbmN8VDvXrVK\\\\\\\/t59Y0FEN8JukDW0TMHsglDzq6bILZXGgmemjH1jXzuUJB8gsSVbcX8vxreT4X+YnqryYpLsdeGJ0M\\\\\\\/6xeGbinyljB2pZjo5MJye2pjJYat0JUZdQq091iqyZq8bV6DNu9IjvHDWi99lsSCf74BzcYlqxoTM36xQPJeFX2jmkrQHN6iF\\\\\\\/zQm1Sbf\\\\\\\/suHGCFsqjj9JaneRwLDaExs6AE6jdsLXYL3+RWlPxCN4GGBP7Mur4k9\\\\\\\/M1W7klFhT1RRDxlPIlwrABWlN7rXfMscloZOOYO3Oc69roo0iAG\\\\\\\/4vPNqOuMHhih+My4szA7ly6J8N57i7FoyulNNCQNlPIYBTnUZ\\\\\\\/zRPommX0XynRlOVqsylFEjdjPzLXsOMaP294BiQmjmka6i5SOUOvMJvYM7TtCVgzjc17LesDlBiNaS7VNhnCWB6V5YZ06IwBAZPn0864yz7QlP0ydhGgub\\\\\\\/7fdB6BfZyYArYG4d51BafFlC\\\\\\\/9AM9RSng7mZ89Ft6rOxN0viowTJ\\\\\\\/ZBp+HKurazbCs97JVpP727iGZgmGE3duHVUElKeYwzaoqfxoP3xGXiG16+GR7D+3KM5AMNHTqklgjU+6LmkKmACWc337yroBso86cG\\\\\\\/SBVqod8iqnU2\\\\\\\/Sidrkt7XUG5wdWu5diVsvjNpnZQtMcVgibjGZGjzLlzAVoEXt3DdOSIblzmFJ3iNxHfSNi5JfJOQHgy5eAizVwr6nWd2V\\\\\\\/2GOYX4gBV6o+7qZdqd1J14m7Rf4Gbgd1bnYcByru10hhKMSpf7MmGofYfdBseOWplkeN8wiDpYyvGr1X\\\\\\\/8nwqpT8cb66l8MJTBl5dGFf6D4z5xGandR4VMWH3xg3z7EXKwFK3RYvAsO4N\\\\\\\/ITR7ffqFD7aGwRYTLPIKld8rWRB34\\\\\\\/FtvXbqdene2TLZEf\\\\\\\/\\\\\\\/XonhaWp6DMosELPwvx0Xe+u2oTvZPzrhvhQf\\\\\\\/3e9rK6YBnPg3qvpI5FHTiD6r78lxSPfmvCWGPGGqWNIZxh7CJDWDlfCCyYU6xMuX4rVfgoBd3ygi1TkYD7GvuG5imcOZmUIv\\\\\\\/MkgZNNIxgo7qKAHa4OVgDYK99fJdhQ8iseLxG2TkHIo7xQ1O70rzYtgGWRwnYN98who+MAJsNuv3DelJF+In1qJAGlSu9OGG5zSPAxS6qZoswYFSeT5gaUDUjcXiUZl0MFypz7m9nrRU+sxxt95CCJ6b54E+PXuxusrc57VSTwVHSEXKkMC0UXUaxFsdZH+wRVaKdbGeQilReQ44Rxui1TrjO5pJm3TblossNwCyM\\\\\\\/dRnZfqM3PNTby05UDfzuTdAhKl70cl3BFUBsYO\\\\\\\/Wl6aVU2w5555wpm9MybqRI4mPXnxiuwPEdfZBQUcvUHcg3WSCwEYGaONryxYeXBdoXNvrng3Lq6\\\\\\\/QyKKd8PLdFjdJe3RCVzofPZ3ywix7ydSh7w95MEHCuvjAchaw9tEvcFhfpKEsm3gwTaOBT43x0mgRpIg01aYJtPt8qSltjE\\\\\\\/4ZJatEldAfFxlqcG3VPkCZ8mz2OyihzpoA1GddjJYdiqP3a5ApZjgG8w8f3cuUR04zhASXyOiVaN9z9XUs7CTbMH6Nhh+fz7mo3oHOvzblhmc9gADBRQS36Qope+lxC6QqtF+fvJlzHsWhYEKXBfPi5IXAJLgUElB22SgefznStf6Gu9F0LiWgCE4PCvhcdkWXznaD3XOybkUHkUVrgzip9BTzwgHWe6rP2i012SAwOUF4FsLHiryu\\\\\\\/LfFRcO+Xsj\\\\\\\/PAV6e1zxix0m6Vz2BtdsFUurOxjkRj4aueUJFljopjXduHJ3Vk12w7RsW2rvHP8oakieR02NN7gp3nMdcsIp1uNzbjv\\\\\\\/qz2xdF9iNCnKqE8vpwT6axuNSldvMPo1jnyxnYe2YU5xzaLVqzhv\\\\\\\/NeUOLeFPWmch8ZcI8QIpd\\\\\\\/yG3twOt4Gtf1znHraYHRw6mLlEuJTVTy7Q9Csp\\\\\\\/7O8MptD92qJgGAXh+IPqPWPGx9cQ9OULOdn7JsH7BwODgVMWkruc39Kxs1lPVvASbsSWtYm3Zy4HejIUG92g4PEENOtUJFskPrL0XffFChDLQ54yzjlY3cs5AXhwQntNYDWM8hK5\\\\\\\/uHQ95vdapUNY1dwHnZR3jMZ\\\\\\\/Z39drTAOSKbo3X5RkKpuY\\\\\\\/ewoGiEosBLZutYewN0jYOGHq+t2v4vG6GGuS37BFuJK4uV7gfiwsGVdg57KmVcN10+hhZy9OemoAuo41opkouGLyOYEA5o3K6iakM4tYYgXT5zvXI53B9SNuzDQbkDusnWRxihb21Jy5gIyjfWNjWyBX5dnCV0KAKLDnbfOxnFmQWDJCudljldRy7Sn5+3ceO7uDoLRcjyR7zpNxmtMBdApsBicGqg+e3oScW4g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xU+Te8ovWq9IwTgFg6OpRA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bdea921a45e108573b46fd2256e09ec1b023b57b3bb96d67af933b094f913c02\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BFOUlj2Ks7Pbf87vYcMxzGeNR2UWOuq8ovhZXT1YXKiMj76Fb4oqNiA1vvzHwwouNA74IFAVEjZhGDkWvcZh+VVnTB53z5nM9QKdFhWntCQkY1n3AZAq2jWsJtTK4PIJOpsKbC4JEYTDpHYy3JyBNiefUNN1ALCwdMl8Zcroq7QkEMnswYZF1xRK1HWvuXjeNYbUCbT\\\\\\\/HNweWuwFHaeHva8LG0UD26Qp7vH78Gxan49dJErUmQ626LKCpVM2gt2BGAWL+piSMVesHG0p7v0QTgHDIVMq4xqGrMebvcEsmMWSfseExMGeRNx4igHrlNcllEBmkBzJkacFsFWEWrsMCHfP1PwHjq2LbvknVFGQ9IqTI+ANACB120HoZIXp0ogoURwW8TatC2y333LaSJsVbt9SwDYWxV6cpUhGtoLbbxTSD9FbqhR5RsLQGGdnfWVeiDZm4PR7D8imItMGDNIpOdswhtd0LSC\\\\\\\/gmmrdnaW2e6ycxJeOO5UBUM2FWAiYqZhQ4CDb\\\\\\\/Ps0irC0xuCYl4AQ0LvyiFWLAJAPXi6CzSLDGgzYCEVZft7+16NuQIVAxnsfjqleyO9NlHoCiEFMjTT0vX+a2V1HmAxm2ZQ2LQ95H\\\\\\\/xxhYTCVi0J5u8bcFJZ9equZTPYY8fAa8am5hpiowXGp7bS8359+mYJozirXoXhy+dAwJkplAvJQPDMDAR5II2B4EI8aCiWIJJnTL0f5lesgP69RhkSYz7qeQupQVvaZSp+un+G6Z\\\\\\\/luDO8Cjnm+g28WZCzEH+Jdqj41zYIHZqDVmzBy+ZdNfR\\\\\\\/NNfvc6wdwdFaypie1uvsUrOCsT5ObxDoOfl84WyIdKC58\\\\\\\/yfxrcYHo6VmwrT140qrHBezpksK97QJrtU3DlC9Q6SKVmscLK9UBzrFkUA53X1BI7qbfuc3tkWLT3Fb2gOmiNE\\\\\\\/NdLLvObejFR2v3VE3ObvXQINITUBtrPiou86igVoNxY3xxY8HiWAQ7jdHnCDCgEr8oEwBZFm+l+++dTXe8WQnBrR+cdNToMCg47d8qp8XY\\\\\\\/\\\\\\\/SoNnLzik0YkTICyyT+Y0dq\\\\\\\/UAG8NMEvt9T9Wfc+FAN+2y1jtgqurWMuO\\\\\\\/P64c9smnm8pOo9USFtc\\\\\\\/8K7jcwxmuMa44Kh7c8p9W0qZs6nrxpZwEEH\\\\\\\/vTtOzyvP9AuiARQN3HvaoffSKaJod8HbAjpP5F0+hBdsQy2TFB7euWsrxKLmlhc+WG3ME\\\\\\\/v\\\\\\\/FDMKB+WB1WyxvTdsY77Uzh2e\\\\\\\/uES1Ny2H7bez1sM0Yfkap\\\\\\\/70sPuAHSrdBmxSg9Ez9AKE4FhbZI5oDda6Bhpt5EKC8Gkc84VdN9SgPEJ\\\\\\\/aQdlNElLqiW7ZJWZ9pIIXi1suC5N31fN2iiD5skdX+93bJ8T7CYfF+qGadTvd1dOu8RZhOuYB7w+akpSLi7hlB8Twz2VjZMTXyof5cQF05yFZcRke4te9thRzvTitu\\\\\\\/RBh6\\\\\\\/LQNKz9dv+ZgFUok8UDpAmsq24IEG2Fnf2AWBa2Dg840FMVq4KMuKS2FlfIk5+xBgVG6JaGF9g3sG4kircro+EbfjCa5RrgkKhBnaqDlv5RllB4pz8L3L05IKpj\\\\\\\/pe1SgBQ\\\\\\\/yoIVg\\\\\\\/OWzHA1ivy47MPescj8z8ReV2bzlnpEWEpSYdJIZcG53hHO66RohvZRtWCtexWOTWOBx5XQRRJQ6wFJ7MIy\\\\\\\/Wwa09596m0nYd\\\\\\\/UHb4iypNeZHr3gnD7pUZe5N6ibd7KevA3O+CdckBqvNmCQPmOPsvrY1qRxSDeO7I9+gsyXfvjKA5q5lhyNCS8Lv\\\\\\\/FMzS5eniqvxi2zAhwD1W76ebinVW21B\\\\\\\/wD3CpX\\\\\\\/jVZm5RMO7aJusnTXJU7OCSifdoVH+jzea8h0JcZMNbauCjGwCscWdsVdbKVAGXPnEQSd4SbeO46hRVvf3o4PIBZM+M7mkrkNw\\\\\\\/oCxBF3JrVF7RaWNgv+C4RK85oJilQUoYnROaNkr3FJo4K2aOh\\\\\\\/JN9wH94WTZPGDo422Ge6uKDbzZWw0iAcw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BrBfOzbJ+1rJcZl4eq7HFg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e781de1943cf7d2fe565771b4dae13616da01b9bf8c6c552992fa5ee8185339f\u0022}"},"ajax":{"edit-submit--5":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484260 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1463" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-21jAjAz8h8a0XzdRdJGe_exapElBue1M_BMvlrELw_M" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1463" method="post" id="highwire-search-quicksearch-form-1" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--3">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--3" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--7" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NCQBT8FZsMI8ueGGDDKp0ISJo-BQ6WMlWXVYLpfge-M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_1" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-3" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1451" title="Aging Effects of Caenorhabditis elegans Ryanodine Receptor Variants Corresponding to Human Myopathic Mutations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1475" title="Extensive Copy Number Variation in Fermentation-Related Genes Among Saccharomyces cerevisiae Wine Strains" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484260" id="node484260" data-pisa="ggg;7/5/1463" data-pisa-master="ggg;g3.117.040816" data-apath="/ggg/7/5/1463.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>Schizosaccharomyces pombe</em> MutSα and MutLα Maintain Stability of Tetra-Nucleotide Repeats and Msh3 of Hepta-Nucleotide Repeats</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4072-6159" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Desirée Villahermosa</span>, <span class="highwire-citation-author" data-delta="1">Olaf Christensen</span>, <span class="highwire-citation-author" data-delta="2">Karen Knapp</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0550-6513" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Oliver Fleck</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1463-1473;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040816 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484260" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Desirée Villahermosa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>North West Cancer Research Institute, School of Medical Sciences, Bangor University, LL57 2UW, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Desir%C3%A9e%2BVillahermosa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Villahermosa%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADesir%25C3%25A9e%2BVillahermosa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4072-6159" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Desirée Villahermosa</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Olaf Christensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Strain Development LBPE38, Lonza, CH-3930 Visp, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olaf%2BChristensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Christensen%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOlaf%2BChristensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karen Knapp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>North West Cancer Research Institute, School of Medical Sciences, Bangor University, LL57 2UW, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BKnapp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Knapp%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BKnapp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Oliver Fleck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>North West Cancer Research Institute, School of Medical Sciences, Bangor University, LL57 2UW, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oliver%2BFleck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fleck%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOliver%2BFleck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0550-6513" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Oliver Fleck</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:o.fleck@bangor.ac.uk" class="" data-icon-position="" data-hide-link-title="0">o.fleck@bangor.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/5/1463" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484260" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484260/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1463.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484260" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484260/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/5/1463.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484260" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484260/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/5/1463.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484260" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484260/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Defective mismatch repair (MMR) in humans is associated with colon cancer and instability of microsatellites, that is, DNA sequences with one or several nucleotides repeated. Key factors of eukaryotic MMR are the heterodimers MutSα (Msh2-Msh6), which recognizes base-base mismatches and unpaired nucleotides in DNA, and MutLα (Mlh1-Pms1), which facilitates downstream steps. In addition, MutSβ (Msh2-Msh3) recognizes DNA loops of various sizes, although our previous data and the data presented here suggest that Msh3 of <em>Schizosaccharomyces pombe</em> does not play a role in MMR. To test microsatellite stability in <em>S. pombe</em> and hence DNA loop repair, we have inserted tetra-, penta-, and hepta-nucleotide repeats in the <em>ade6</em> gene and determined their Ade<sup>+</sup> reversion rates and spectra in wild type and various mutants. Our data indicate that loops with four unpaired nucleotides in the nascent and the template strand are the upper limit of MutSα- and MutLα-mediated MMR in <em>S. pombe</em>. Stability of hepta-nucleotide repeats requires Msh3 and Exo1 in MMR-independent processes as well as the DNA repair proteins Rad50, Rad51, and Rad2<sup>FEN1</sup>. Most strikingly, mutation rates in the double mutants <em>msh3 exo1</em> and <em>msh3 rad51</em> were decreased when compared to respective single mutants, indicating that Msh3 prevents error prone processes carried out by Exo1 and Rad51. We conclude that Msh3 has no obvious function in MMR in <em>S. pombe</em>, but contributes to DNA repeat stability in MMR-independent processes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/mismatch-repair" class="hw-term hw-article-keyword hw-article-keyword-mismatch-repair" rel="nofollow">mismatch repair</a></li><li class="kwd"><a href="/keyword/microsatellite-instability" class="hw-term hw-article-keyword hw-article-keyword-microsatellite-instability" rel="nofollow">microsatellite instability</a></li><li class="kwd"><a href="/keyword/homologous-recombination-repair" class="hw-term hw-article-keyword hw-article-keyword-homologous-recombination-repair" rel="nofollow">homologous recombination repair</a></li><li class="kwd"><a href="/keyword/repetitive-dna" class="hw-term hw-article-keyword hw-article-keyword-repetitive-dna" rel="nofollow">repetitive DNA</a></li><li class="kwd"><a href="/keyword/fen1" class="hw-term hw-article-keyword hw-article-keyword-fen1" rel="nofollow">FEN1</a></li></ul><p id="p-4">Repetitive DNA elements are widespread in genomes. They are located in centromeres, telomeres, rDNA genes, transposons, and intergenic regions (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Padeken <em>et al.</em> 2015</a>). One class of repetitive DNA comprises microsatellites, which are short tandem repeats of one to several nucleotides. Their lengths often differ between individuals of a given species, but are identical in cells of an individual organism. During DNA replication, repeat units can slip and misalign with repeats of the opposite strand. Unrecognized slippage in the template strand leads to deletions, and in the nascent strand to insertions after the next round of replication. The major pathway for maintaining repeat tract lengths of microsatellites is mismatch repair (MMR). Eukaryotic MMR is initiated either by Msh2-Msh6 (MutSα) or by Msh2-Msh3 (MutSβ) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alani 1996</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Habraken <em>et al.</em> 1996</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Johnson <em>et al.</em> 1996</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Marsischky <em>et al.</em> 1996</a>). Depending on the eukaryote, some differences exist in the recognition spectrum of the two complexes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Marti <em>et al.</em> 2002</a>). Generally, MutSα is responsible for repair of base-base mismatches and loops, and MutSβ for repair of loops. After mismatch binding, MutL heterodimers are recruited, with Mlh1-Pms1 (MutLα) (termed Mlh1-Pms2 in humans) being the major factor for MMR (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jiricny 2013</a>). The newly synthesized strand with the erroneous nucleotide(s) is subsequently degraded either by the 5′-exonuclease activity of Exo1 or by the endonuclease activity of MutLα (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kadyrov <em>et al.</em> 2006</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2007</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Smith <em>et al.</em> 2013</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Goellner <em>et al.</em> 2014</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">2015</a>). The homotrimeric processivity factor PCNA binds to MutSα, MutSβ, and MutLα and coordinates correct incision of the newly synthesized strand (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Flores-Rozas <em>et al.</em> 2000</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kleczkowska <em>et al.</em> 2001</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lee and Alani 2006</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Iyer <em>et al.</em> 2010</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Pluciennik <em>et al.</em> 2010</a>). After excision beyond the mismatch, a replicative DNA polymerase fills the resulting gap and the remaining nick is ligated.</p><p id="p-5">A defect in the human MMR genes <em>MSH2</em> and <em>MLH1</em> causes microsatellite instability and a predisposition to colon and other types of cancer (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lynch <em>et al.</em> 2009</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Da Silva <em>et al.</em> 2016</a>). Mutations in other MMR genes are rarely correlated with cancer, probably due to functional redundancy. In contrast to other types of repetitive elements, microsatellites are often situated in genes, which is a critical factor for tumor development and important for the choice of drugs for treatment of cancer patients. For example, deletions in a T<sub>11</sub> repeat in intron 4 of <em>MRE11</em> causes aberrant splicing, and as a consequence, a truncated MRE11 protein (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Giannini <em>et al.</em> 2002</a>). This sensitizes cancer cells to combined treatment with camptothecin and thymidine (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Rodriguez <em>et al.</em> 2008</a>).</p><p id="p-6">Recombination processes can alter tract lengths of repetitive DNA either by unequal crossover between repeats or through secondary structures formed between repeats in the same strand. The Mre11-Rad50-Nbs1 (MRN) complex has single-stranded 3′-exonuclease and endonuclease activities as well as structural functions in recombination processes (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cejka 2015</a>). After 5′–3′ resection of DNA double-strand breaks or ends by Exo1 or other 5′-exonucleases, Rad51-dependent homologous recombination (HR) can occur by invasion of the 3′-ended single strand into a complimentary DNA molecule. Rad51-independent single-strand annealing (SSA) can also occur between two repeats and leads to deletion of the intervening sequence. SSA requires the nucleotide excision repair factors Rad1-Rad10 of <em>Saccharomyces cerevisiae</em> (XPF-ERCC1 in human) and MutSβ (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bhargava <em>et al.</em> 2016</a>). FEN1 is a flap endonuclease with multiple roles in DNA metabolisms. FEN1 is involved in processing of Okazaki fragments during replication, in long-patch base excision repair and in other processes (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Marti and Fleck 2004</a>). <em>S. cerevisiae</em>, <em>rad27</em> (a FEN1 homolog) mutants exhibit instability of mono- and dinucleotide repeats and generate duplications of sequences flanked by repeats (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Johnson <em>et al.</em> 1995</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Tishkoff <em>et al.</em> 1997</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kirchner <em>et al.</em> 2000</a>). In addition, FEN1 has been implicated in trinucleotide repeat stability (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Freudenreich <em>et al.</em> 1998</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Liu and Wilson 2012</a>) and repair of large loops with up to 216 unpaired nucleotides (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sommer <em>et al.</em> 2008</a>).</p><p id="p-7">The genome of fission yeast <em>Schizosaccharomyces pombe</em> encodes the MutS homologs Msh1, Msh2, Msh3, and Msh6, the MutL homologs Mlh1 and Pms1, and the exonuclease Exo1. Based on homology with <em>S. cerevisiae</em> Msh1, <em>S. pombe</em> Msh1 likely acts in MMR of mitochondrial DNA. <em>S. pombe</em> Msh2, Msh6, Mlh1, and Pms1 are indispensable for repair of base-base mismatches and small loops with one or two nucleotides (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Schär <em>et al.</em> 1997</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rudolph <em>et al.</em> 1999</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mansour <em>et al.</em> 2001</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Tornier <em>et al.</em> 2001</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Marti <em>et al.</em> 2003</a>). In contrast, Msh3 seems to have no, or a minor and rather MMR-independent, function in repair of base-base mismatches and small loops. Exo1 appears to be involved in MMR of base-base mismatches but has a rather MMR-independent function in repair of small loops (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Rudolph <em>et al.</em> 1998</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Mansour <em>et al.</em> 2001</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Marti <em>et al.</em> 2003</a>).</p><p id="p-8">In the present study, we tested stability of tetra-, penta-, and hepta-nucleotide repeats in <em>S. pombe</em>. Our aim was to analyze whether stability of such repeats is dependent on MMR, and if so, whether MutSα, MutSβ, or both are involved. In addition, we analyzed <em>mlh1</em>, and thus MutLα-deficient strains as well as <em>exo1</em> mutants.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>General yeast genetic methods, media, and S. pombe strains</h3><p id="p-9">The <em>S. pombe</em> media minimal medium agar (MMA), yeast extract agar (YEA), and yeast extract liquid (YEL), and general genetic methods were used as described (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gutz <em>et al.</em> 1974</a>). <em>S. pombe</em> strains used in this study were derived from Ru39 <em>h<sup>−</sup> msh2</em>::<em>his3 his3-D1</em> (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Rudolph <em>et al.</em> 1999</a>); KK83 <em>smt-0 msh3</em>::<em>loxP-ura4-loxM leu1-32 ura4-D18</em> (D. Villahermosa, K. Knapp, and O. Fleck, unpublished data); OL937 <em>h<sup>−</sup> mlh1</em>::<em>kanMX his3-D1 ura4-D18</em> (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Marti <em>et al.</em> 2003</a>); Ru42 <em>h<sup>−</sup> exo1</em>::<em>ura4 ura4-D18</em> (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Szankasi and Smith 1995</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Rudolph <em>et al.</em> 1998</a>); sp217 <em>h<sup>−</sup> rad2</em>::<em>ura4 ade6-704 leu1-32 ura4-D18</em> (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Murray <em>et al.</em> 1994</a>); EH65 <em>smt-0 rad50</em>::<em>kanMX ura4-D18</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hartsuiker <em>et al.</em> 2001</a>), and SPAC644.14c <em>h<sup>+</sup> rad51</em>::<em>kanMX ade6 leu1-32 ura4-D18</em> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kim <em>et al.</em> 2010</a>).</p><p id="p-10">Strains KK42 <em>h<sup>−</sup> msh2</em>::<em>hphMX ura4-D18</em> and KK37 <em>h<sup>−</sup> msh6</em>::<em>hphMX ura4-D18</em> originated from transformations of OL2137 (<em>h<sup>−</sup> ura4-D18</em>), with DNA fragments obtained by fusion PCRs. pFA6a-hphMX (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hentges <em>et al.</em> 2005</a>) was used as template to amplify the hygromycin resistance cassette. Genomic DNA of strain RO144 (<em>smt-0</em>) was used to amplify either 450 bp of the 3′ UTR of the <em>msh2</em> locus or 500 bp of the 5′ UTR of the <em>msh6</em> locus. Primers for the <em>msh2</em> disruption were msh2_For 5′-GAGGTTTTTTATTTATCCTTTTTGAGGACTTAACTGTGGCAAGGAGTTTCTTCTCCTGTTTTATACATTTCGCGTTCGCGCTTTAGAACATTCAATCAATCGGATCCCCGGGTTAATTAA; msh2_Rev 5′-TTTCCTCGTTTTAGTAAAAAATTATTTTATTCATAAAATGCGCTTCCAAAAAACATGTACCTTGGTTGAATTCTTTCAATTAGTACCTTGCTCACATTCTGAATTCGAGCTCGTTTAAAC; msh2_For3 5′-TTGAAAGAATTCAACCAAGG; and msh2_Rev3 5′-GCTAAAACAAAATTATGCCG. Primers for the <em>msh6</em> disruption were msh6_For 5′-TATATATGTTATTTTGTGCTCTCATGTTAGCTTTGTTTACTATTAGAATGCTGCTTTTTGTAAATAACTGAACTTAGCCAAAACCAACACTTGTTCCAGTCGGATCCCCGGGTTAATTAA; msh6_Rev 5′-ATAACGTAAGTAAATGGTAAATAAAAGCAAGCTTCCGCTTGCCAGCAAACGAAAGATATTGCTTTGAATAGTCATAAAACTGATAGAGTGTTGACAGTTAGAATTCGAGCTCGTTTAAAC; msh6_For2 5′-CTCATCTTACCTAAACTCTC; and msh6_Rev2 5′-GAACAAGTGTTGGTTTTGGC.</p></div><div id="sec-3" class="subsection"><h3>Construction of (GACC)<sub>n</sub> repeats in the ade6 gene</h3><p id="p-11">In pAN-K, a pUC18 derivative, the kanamycin resistance gene was replaced by a 340-bp <em>Dra</em>III-<em>Hin</em>dIII fragment of the <em>ade6</em> gene containing a (GACC)<sub>7</sub> repeat near the <em>Dra</em>III site (underlined in primer ade6-GACC7; see below). The fragment was obtained by PCR with primers ade6-GACC7 5′-TCC<span class="underline">CACTTGGTG</span>ACCGACCGACCGACCGACCGACCGACCTTTATGTTGAAAAGTTCGTTC-3′ and ade6-H 5′-GGGCAAGCTTCAATGGTGTA-3′, and subsequent digestion with <em>Dra</em>III and <em>Hin</em>dIII. PCR was performed under standard conditions using pCG162 as template (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Grimm <em>et al.</em> 1994</a>). Plasmids with the desired insert were identified by restriction digests and DNA sequencing. In addition, a plasmid was identified in which one T in a T<sub>3</sub> run immediately 3′ to the (GACC)<sub>7</sub> repeat was deleted. The 1.7-kb <em>Xho</em>I-<em>Eco</em>RI fragments of the plasmids containing either (GACC)<sub>7</sub> or (GACC)<sub>7</sub>ΔT were transformed into the <em>S. pombe</em> strain AM1 (<em>h<sup>–</sup> ade6</em>::<em>ura4 his3-D1 ura4-D18 leu1-32</em>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Mansour <em>et al.</em> 2001</a>). Correct integration was confirmed by PCR and DNA sequencing. The resulting strains were <em>h<sup>–</sup> ade6-(GACC)<sub>7</sub> his3-D1 leu1-32 ura4-D18</em> and <em>h<sup>–</sup> ade6-(GACC)<sub>7</sub></em>Δ<em>T his3-D1 leu1-32 ura4-D18</em>. A strain with a (GACC)<sub>8</sub> repeat was isolated during a fluctuation test with an <em>h</em><sup>+</sup> <em>msh2</em>::<em>his3 ade6-(GACC)<sub>7</sub></em> mutant.</p></div><div id="sec-4" class="subsection"><h3>Construction of the ade6::ura4 strain DE1</h3><p id="p-12">The strain DE1 (<em>h<sup>−</sup> ade6</em>::<em>ura4 ura4-D18</em>) was constructed by transformation of strain OL2137 (<em>h<sup>−</sup> ura4-D18</em>) with a PCR fragment obtained with primers ade6_d_ura4_F 5′-TCCTTTTGTACTGAAAAGTAAAACATTGGCTTACGACGGTCGTGGAAATTACGTTGTTCATCAACCATCTGAGATTCCTACTGCCATCAAAGCACTTGGTagcttagctacaaatcccac and ade6_d_ura4_R 5′-GAATGGTCTCAGTTGTAGGATAAGCATAAACTTTTCCGTCTAAACTGCGTACTACCATCACTGCAATTTCCATGGAGAAAGGAACGAACTTTTCAACATagcttgtgatattgacgaaac, and pAW1 as template (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Watson <em>et al.</em> 2008</a>). Nucleotides in lowercase letters were derived from the <em>ura4</em> marker gene; nucleotides in capital letters were derived from <em>ade6</em>. In this way, 13 nucleotides (5′-GATCGTCCGCTTT) in <em>ade6</em> were deleted and replaced by <em>ura4</em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1463/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of the position and nature of the various repeats. In the ade6::ura4 disruption strain, 13 nucleotides have been deleted and replaced by the ura4 marker. This strain was transformed with DNA fragments to produce ade6 mutants with the indicated repeats (highlighted by blue arrows). Nucleotides within the repeats that differ from the wild-type sequence are shown in red. Integration of the repeats caused frameshifts with nearby located stop codons. The numbers of net inserted nucleotides and of the major deletion/insertion events that lead to Ade+ reversions are given on the right. (+14) indicates that, although this event restores the open reading frame, it was not found among the 30 ade6-(ATCGTCC)5&#x394;T revertants sequenced (Table 6). In (GACC)7&#x394;T and (ATCGTCC)5&#x394;T, a T in a T3 stretch immediately downstream of the repeats is deleted. This T deletion allows detecting different types of deletions and insertions within respective repeats in comparison to (GACC)8 and (ATCGTCC)5, as indicated on the right." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-96982063" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic of the position and nature of the various repeats. In the &lt;em&gt;ade6&lt;/em&gt;::&lt;em&gt;ura4&lt;/em&gt; disruption strain, 13 nucleotides have been deleted and replaced by the &lt;em&gt;ura4&lt;/em&gt; marker. This strain was transformed with DNA fragments to produce &lt;em&gt;ade6&lt;/em&gt; mutants with the indicated repeats (highlighted by blue arrows). Nucleotides within the repeats that differ from the wild-type sequence are shown in red. Integration of the repeats caused frameshifts with nearby located stop codons. The numbers of net inserted nucleotides and of the major deletion/insertion events that lead to Ade&lt;sup&gt;+&lt;/sup&gt; reversions are given on the right. (+14) indicates that, although this event restores the open reading frame, it was not found among the 30 &lt;em&gt;ade6-(ATCGTCC)&lt;sub&gt;5&lt;/sub&gt;&lt;/em&gt;&#x394;&lt;em&gt;T&lt;/em&gt; revertants sequenced (Table 6). In (GACC)&lt;sub&gt;7&lt;/sub&gt;&#x394;T and (ATCGTCC)&lt;sub&gt;5&lt;/sub&gt;&#x394;T, a T in a T&lt;sub&gt;3&lt;/sub&gt; stretch immediately downstream of the repeats is deleted. This T deletion allows detecting different types of deletions and insertions within respective repeats in comparison to (GACC)&lt;sub&gt;8&lt;/sub&gt; and (ATCGTCC)&lt;sub&gt;5&lt;/sub&gt;, as indicated on the right.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1463/F1.medium.gif" width="440" height="169"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/5/1463/F1.medium.gif" width="440" height="169"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1463/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1463/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484349" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Schematic of the position and nature of the various repeats. In the <em>ade6</em>::<em>ura4</em> disruption strain, 13 nucleotides have been deleted and replaced by the <em>ura4</em> marker. This strain was transformed with DNA fragments to produce <em>ade6</em> mutants with the indicated repeats (highlighted by blue arrows). Nucleotides within the repeats that differ from the wild-type sequence are shown in red. Integration of the repeats caused frameshifts with nearby located stop codons. The numbers of net inserted nucleotides and of the major deletion/insertion events that lead to Ade<sup>+</sup> reversions are given on the right. (+14) indicates that, although this event restores the open reading frame, it was not found among the 30 <em>ade6-(ATCGTCC)<sub>5</sub></em>Δ<em>T</em> revertants sequenced (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). In (GACC)<sub>7</sub>ΔT and (ATCGTCC)<sub>5</sub>ΔT, a T in a T<sub>3</sub> stretch immediately downstream of the repeats is deleted. This T deletion allows detecting different types of deletions and insertions within respective repeats in comparison to (GACC)<sub>8</sub> and (ATCGTCC)<sub>5</sub>, as indicated on the right.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Construction of penta- and hepta-nucleotide repeats in the ade6 gene</h3><p id="p-14">We subjected 100 pmol of each of two 120-nucleotide-long oligonucleotides, with ∼40 nucleotides complementary to each other at their 3′ ends, to primer extension in 50 μl reactions containing 1 U Go<em>Taq</em> polymerase, 2.5 mM MgCl<sub>2</sub>, 50 μM each dNTP in Colorless Go<em>Taq</em> Flexi buffer (Promega). After initial denaturation for 1 min at 94°, we applied two cycles with 30 sec at 94°, 30 sec at 45°, and 30 sec at 72°, followed by five cycles with 30 sec at 94°, 30 sec at 55°, and 30 sec at 72°. Reaction samples were transformed into strain DE1 using the method of <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ito <em>et al.</em> (1983)</a> with some modifications (D. Villahermosa, K. Knapp, and O. Fleck, unpublished data). Primers Penta2-F 5′-TAAAACATTGGCTTACGACGGTCGTGGAAATTACGTTGTTCATCAACCATCTGAGATTCCTACTGCCATCAAAGCACTTGGTGCTGCCCTGCCCTGCCCTGCCCTGCCCTGCCTATGTTG and Penta2-R 5′-GGATAAGCATAAACTTTTCCGTCTAAACTGCGTACTACCATCACTGCAATTTCCATGGAGAAAGGAACGAACTTTTCAACATAGGCAGGGCAGGGCAGGGCAGGGCAGGGCAGCACCAAG were used to construct <em>ade6-(CTGCC)<sub>6</sub></em>. Primers Hepta-F 5′-ACATTGGCTTACGACGGTCGTGGAAATTACGTTGTTCATCAACCATCTGAGATTCCTACTGCCATCAAAGCACTTGGTGATCGTCCATCGTCCATCGTCCATCGTCC ATCGTCCTTTATG and Hepta-R 5′-AAGCATAAACTTTTCCGTCTAAACTGCGTACTACCATCACTGCAATTTCCATGGAGAAAGGAACGAACTTTTCAACATAAAGGACGATGGACGATGGACGATGGACGATGGACGATCACC were used for <em>ade6-(ATCGTCC)<sub>5</sub></em>. Primers HeptadT-F 5′-AACATTGGCTTACGACGGTCGTGGAAATTACGTTGTTCATCAACCATCTGAGATTCCTACTGCCATCAAAGCACTTGGTGATCGTCCATCGTCCATCGTCCATCGTCCATCGTCCTTATG and HeptadT-R 5′-TAAGCATAAACTTTTCCGTCTAAACTGCGTACTACCATCACTGCAATTTCCATGGAGAAAGGAACGAACTTTTCAACATAAGGACGATGGACGATGGACGATGGACGATGGACGATCACC were used for <em>ade6-(ATCGTCC)<sub>5</sub></em>Δ<em>T</em>. Here, a single T immediately downstream of the repeats has been deleted (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p></div><div id="sec-6" class="subsection"><h3>Determination of mutation rates and spectra</h3><p id="p-15">Mutation rates were determined by fluctuation tests as described (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Mansour <em>et al.</em> 2001</a>). In brief, seven tubes containing 2 ml YEL were each inoculated with a single small colony and incubated at 30° until cultures were grown to stationary phase. Appropriate dilutions were plated on YEA for cell titer determination and on MMA for selection of Ade<sup>+</sup> revertants. Colonies were counted after 5 d of growth at 30°, except for strains with <em>rad50</em> or <em>rad51</em> background, where colonies were counted after 6 d to compensate for their slow growth. Reversion rates were calculated from at least three independent fluctuation tests. Statistical significance was calculated with a two-tailed Student’s <em>t</em>-test.</p><p id="p-16">The nature of mutations was determined by sequencing of PCR products from genomic DNA using primers ade6_F 5′-ATTAACACTGATGCCTTGGC and ade6_R 5′-ACAGAGAACGTTTAGCGATC. In the case of <em>ade6-(GACC)<sub>7</sub></em>Δ<em>T</em>, repeat tract changes were also analyzed by inspection of the color of Ade<sup>+</sup> revertants (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The color was best determined when revertants were restreaked on YEA without supplemented adenine. The proportion of white and pink revertants was determined after 1–2 d of growth at 30°. Final averages and SDs were calculated from averages of at least three independent fluctuation tests, each with seven cultures, and up to 20 random revertants (where available) per culture.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/5/1463/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of deletions and insertions in the ade6-(GACC)7&#x394;T repeat. (A) Deletions and insertions within the (GACC)7&#x394;T repeat that lead to Ade+ can be distinguished by their color. ade6-(GACC)7&#x394;T mutants form red colonies on medium with limited amount of adenine due to a defective ade6 gene. They can revert to Ade+ by deletion of two or five repeat units, producing white Ade+, or by insertion of one or four repeat units, producing pink Ade+. (B) Percentage of deletions in the various strain backgrounds. Wild type, msh3, and exo1 mainly reverted to Ade+ by deletions, while msh2, msh6, and mlh1 mutants mainly reverted by insertions. Significantly different to wild type: ** p &lt; 0.01; *** p &lt; 0.001. Shown are average values with SDs. Reversion spectra are also presented as pie charts, with the proportion of deletions and insertions indicated in white and pink, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-96982063" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of deletions and insertions in the &lt;em&gt;ade6-(GACC)&lt;sub&gt;7&lt;/sub&gt;&lt;/em&gt;&#x394;&lt;em&gt;T&lt;/em&gt; repeat. (A) Deletions and insertions within the (GACC)&lt;sub&gt;7&lt;/sub&gt;&#x394;T repeat that lead to Ade&lt;sup&gt;+&lt;/sup&gt; can be distinguished by their color. &lt;em&gt;ade6-(GACC)&lt;sub&gt;7&lt;/sub&gt;&lt;/em&gt;&#x394;&lt;em&gt;T&lt;/em&gt; mutants form red colonies on medium with limited amount of adenine due to a defective &lt;em&gt;ade6&lt;/em&gt; gene. They can revert to Ade&lt;sup&gt;+&lt;/sup&gt; by deletion of two or five repeat units, producing white Ade&lt;sup&gt;+&lt;/sup&gt;, or by insertion of one or four repeat units, producing pink Ade&lt;sup&gt;+&lt;/sup&gt;. (B) Percentage of deletions in the various strain backgrounds. Wild type, &lt;em&gt;msh3&lt;/em&gt;, and &lt;em&gt;exo1&lt;/em&gt; mainly reverted to Ade&lt;sup&gt;+&lt;/sup&gt; by deletions, while &lt;em&gt;msh2&lt;/em&gt;, &lt;em&gt;msh6&lt;/em&gt;, and &lt;em&gt;mlh1&lt;/em&gt; mutants mainly reverted by insertions. Significantly different to wild type: ** p &lt; 0.01; *** p &lt; 0.001. Shown are average values with SDs. Reversion spectra are also presented as pie charts, with the proportion of deletions and insertions indicated in white and pink, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/5/1463/F2.medium.gif" width="406" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/5/1463/F2.medium.gif" width="406" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/5/1463/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/5/1463/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484390" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Distribution of deletions and insertions in the <em>ade6-(GACC)<sub>7</sub></em>Δ<em>T</em> repeat. (A) Deletions and insertions within the (GACC)<sub>7</sub>ΔT repeat that lead to Ade<sup>+</sup> can be distinguished by their color. <em>ade6-(GACC)<sub>7</sub></em>Δ<em>T</em> mutants form red colonies on medium with limited amount of adenine due to a defective <em>ade6</em> gene. They can revert to Ade<sup>+</sup> by deletion of two or five repeat units, producing white Ade<sup>+</sup>, or by insertion of one or four repeat units, producing pink Ade<sup>+</sup>. (B) Percentage of deletions in the various strain backgrounds. Wild type, <em>msh3</em>, and <em>exo1</em> mainly reverted to Ade<sup>+</sup> by deletions, while <em>msh2</em>, <em>msh6</em>, and <em>mlh1</em> mutants mainly reverted by insertions. Significantly different to wild type: ** p &lt; 0.01; *** p &lt; 0.001. Shown are average values with SDs. Reversion spectra are also presented as pie charts, with the proportion of deletions and insertions indicated in white and pink, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-18"><em>S. pombe</em> strains are available on request. The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Genetic assay for microsatellite instability in S. pombe</h3><p id="p-19">We have previously reported that in-frame nucleotide insertions and codon changes in a region around nucleotide 1397 of the <em>ade6</em> gene [the ATG start codon is at 875 as defined by <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Szankasi <em>et al.</em> (1988)</a>] in <em>S. pombe</em> did not disrupt or only slightly disrupted its functionality (<a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Mansour <em>et al.</em> 2001</a>; <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Marti <em>et al.</em> 2003</a>). Here, we inserted tetra-, penta-, and hepta-nucleotide repeats in this part of <em>ade6</em> in order to analyze microsatellite instability in wild type and MMR mutants (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Insertion of the repeats caused frameshifts, which rendered cells auxotrophic for adenine due to nearby located stop codons. Such <em>ade6</em> mutants required a supply of adenine for growth and turned red on the medium with a limited amount of adenine due to accumulation of a red pigment. Reversions of such strains to Ade<sup>+</sup> can occur by deletions or insertions of repeats, when these events restore the open reading frame. With (GACC)<sub>7</sub>ΔT, (CTGCC)<sub>6</sub>, and (ATCGTCC)<sub>7</sub>, insertions of one repeat unit and deletions of two repeat units are the major events detectable. The opposite is the case with (GACC)<sub>8</sub> and (ATCGTCC)<sub>7</sub>ΔT, where deletions of one repeat unit and insertions of two repeat units are the principal events that can be detected, although in the case of (ATCGTCC)<sub>7</sub>ΔT, we identified exclusive deletions as described below.</p><p id="p-20">In the case <em>of ade6-(GACC)<sub>7</sub></em>Δ<em>T</em>, deletions and insertion can be distinguished by the color of revertants (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). Deletions caused white Ade<sup>+</sup>, while insertions caused pink Ade<sup>+</sup>. The pink color likely reflects that <em>ade6</em> is not fully functional and thus that the red pigment is produced in low quantities. Sequencing of 18 Ade<sup>+</sup> revertants of the <em>(GACC)<sub>7</sub></em>Δ<em>T</em> repeat in the various genetic backgrounds revealed deletions and sequencing of 23 <em>(GACC)<sub>7</sub></em>Δ<em>T</em> revertants revealed insertions (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). All revertants with deletions were white and all revertants with insertions were pink. Thus, the occurrence of deletions and insertions in the <em>(GACC)<sub>7</sub></em>Δ<em>T</em> repeat can be easily determined from a large number of revertants. Analyzing revertants of the other <em>ade6</em> repeats did not allow a distinction by color, probably because Ade<sup>+</sup> originating from deletions and insertions were both not fully functional. In these cases, PCR products of independent Ade<sup>+</sup> revertants were subjected to sequencing to identify the number of repeats.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484326/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484326/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Reversion spectra of tetra-nucleotide repeats</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>GACC tetra-nucleotide repeats were unstable in msh2, msh6, and mlh1 mutants</h3><p id="p-23">We first analyzed stability of GACC tetra-nucleotide repeats in wild type, and in mutants deleted for either <em>msh2</em>, <em>msh3</em>, <em>msh6</em>, <em>mlh1</em>, or <em>exo1</em>. In wild type, <em>ade6-(GACC)<sub>8</sub></em> and <em>ade6-(GACC)<sub>7</sub></em>Δ<em>T</em> reverted to Ade<sup>+</sup> with 1.2 × 10<sup>−5</sup> and 2.1 × 10<sup>−5</sup> reversions per cell division, respectively (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The <em>ade6-(GACC)<sub>8</sub></em> reversion rates increased 39–44-fold in <em>msh2</em>, <em>msh6</em>, and <em>mlh1</em> mutants and slightly decreased in <em>msh3</em> and <em>exo1</em> mutants. Similarly, <em>ade6-(GACC)<sub>7</sub></em>Δ<em>T</em> reversion rates increased 17–22-fold in <em>msh2</em>, <em>msh6</em>, and <em>mlh1</em> mutants and decreased 3–5-fold in <em>msh3</em> and <em>exo1</em> mutants. None of the differences between <em>msh3</em>, <em>exo1</em>, and wild type was statistically significant.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484388/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484388/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Reversion rates of GACC tetra-nucleotide repeats</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">Repeat tract changes in <em>ade6-(GACC)<sub>7</sub></em>Δ<em>T</em> can be determined by colony color as described above and in <em>Materials and Methods</em>, and as illustrated in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>. In wild type, <em>msh3</em>, and <em>exo1</em> strains, reversions mainly occurred by deletions (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). Most were deletions of two repeat units (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). In contrast, <em>msh2</em>, <em>msh6</em>, and <em>mlh1</em> mutants mainly reverted to Ade<sup>+</sup> by insertion of one repeat (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Sequencing of Ade<sup>+</sup> revertants of <em>ade6-(GACC)<sub>8</sub></em> revealed that in wild type, all 10 had two repeat insertions (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). In contrast, <em>msh2</em> and <em>msh6</em> reverted mainly by deletion of one repeat. The spectrum for <em>msh3</em> was more heterogeneous, with some preference for insertions. Importantly, the occurrence of deletions <em>vs.</em> insertions was significantly different to wild type (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-11" class="subsection"><h3>Increased penta-nucleotide repeat stability in the msh3 mutant</h3><p id="p-28">The (CTGCC)<sub>6</sub> repeat reverted in wild type with a rate of 3.4 × 10<sup>−5</sup> to Ade<sup>+</sup> (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Rates were not significantly different in <em>msh2</em>, <em>msh6</em>, <em>mlh1</em>, and <em>exo1</em> mutants. In contrast, the (CTGCC)<sub>6</sub> repeat appeared to be more stable in the <em>msh3</em> mutant. It reverted mostly through gain of one repeat unit in all strain backgrounds, with the possible exception of <em>mlh1</em> (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). We conclude that MutSα cannot repair loops with five unpaired nucleotides in the (CTGCC)<sub>6</sub> context, and that Msh3 has some function in stability of the penta-nucleotide repeat.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484386/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484386/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Reversion rates of the (CTGCC)<sub>6</sub> repeat</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484399/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484399/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Reversion spectra of the (CTGCC)<sub>6</sub> repeat</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Hepta-nucleotide repeat stability was not affected in msh2, msh6, and mlh1 and was slightly decreased in msh3 and exo1 mutants</h3><p id="p-34">In wild type, the hepta-nucleotide repeats <em>ade6-(ATCGTCC)<sub>5</sub></em>Δ<em>T</em> and <em>ade6-(ATCGTCC)<sub>5</sub></em> reverted to Ade<sup>+</sup> at rates of 7.2 × 10<sup>−6</sup> and 7.5 × 10<sup>−6</sup>, respectively (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Rates were not significantly different in <em>msh2</em>, <em>msh6</em>, and <em>mlh1</em> mutants, suggesting that loops with seven unpaired nucleotides are not repaired by MMR in <em>S. pombe</em>. In <em>msh3</em> and <em>exo1</em> mutants, reversion rates were increased 1.7–1.9-fold for <em>ade6-(ATCGTCC)<sub>5</sub></em>Δ<em>T</em> (not significant) and 2.8–2.9-fold for <em>ade6-(ATCGTCC)<sub>5</sub></em> (significant) (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). All sequenced <em>ade6-(ATCGTCC)<sub>5</sub></em>Δ<em>T</em> revertants contained four hepta-nucleotide repeats and thus originated from deletion of one repeat (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>). The fact that insertions were not found at all could be either due to a relative low sample size or that insertion of 14 nucleotides, leading to a total of 39 additional nucleotides in <em>ade6</em> (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>), does not render cells Ade<sup>+</sup>. <em>ade6-(ATCGTCC)<sub>5</sub></em> reverted mainly by insertion of one repeat and less frequently by deletion of two repeats without any significant differences between the spectra of wild type and any of the mutants (<a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484389/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484389/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Reversion rates of hepta-nucleotide repeats</span><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484275/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484275/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Reversion spectra of hepta-nucleotide repeats</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Hepta-nucleotide repeat stability in rad2<sup>FEN1</sup>, rad50, and rad51 mutants</h3><p id="p-39">The <em>msh3</em> and <em>exo1</em> mutants, but not the mutants <em>msh2</em>, <em>msh6</em>, and <em>mlh1</em>, exhibited increased instability of the (ATCGTCC)<sub>5</sub> repeat (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). Since defective <em>msh2</em> or <em>mlh1</em> is generally considered to completely inactivate MMR, the (ATCGTCC)<sub>5</sub> repeat instability in <em>msh3</em> and <em>exo1</em> is not due to a defect in MMR. We therefore wanted to analyze the genetic context of <em>msh3</em> and <em>exo1</em> defects in microsatellite stability. To do this, we measured reversion rates of the hepta-nucleotide repeat in <em>rad2<sup>FEN1</sup></em>, <em>rad50</em>, and <em>rad51</em> mutants and in various double mutants (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). FEN1 and HR have been implicated in repeat stability in <em>S. cerevisiae</em> (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Johnson <em>et al.</em> 1995</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Tishkoff <em>et al.</em> 1997</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Freudenreich <em>et al.</em> 1998</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Kirchner <em>et al.</em> 2000</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Sundararajan <em>et al.</em> 2010</a>). We found that the <em>rad2<sup>FEN1</sup></em>, <em>rad50</em>, and <em>rad51</em> single mutants had ∼2-fold increased reversion rates (<a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a>). The <em>msh3 exo1</em> double mutant reverted significantly less frequently to Ade<sup>+</sup> than either single mutant. Mutation rates remained about the same in <em>msh3 rad50</em>, <em>msh3 rad2<sup>FEN1</sup></em>, and <em>exo1 rad50</em> but decreased in the <em>msh3 rad51</em> and <em>exo1 rad51</em> double mutants when compared to respective single mutants (<a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7</a>). Like wild type and the other single mutants, <em>rad50</em> and <em>rad51</em> strains mainly reverted to Ade<sup>+</sup> by insertion of one repeat unit (<a id="xref-table-wrap-6-4" class="xref-table" href="#T6">Table 6</a>). In the <em>rad2<sup>FEN1</sup></em> mutant, 50% of the reversions were due to deletion of two repeats, which was not significantly different to wild type. A reduction of reversion rates in the <em>msh3 rad51</em> and <em>exo1 rad51</em> double mutants indicates that Exo1 and Rad51 act error prone on the (ATCGTCC)<sub>5</sub> repeat when <em>msh3</em> is mutated.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484325/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484325/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Reversion rates of (ATCGTCC)<sub>5</sub> repeats in <em>msh3</em>, <em>exo1</em>, and <em>rad</em> mutants</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><div id="sec-15" class="subsection"><h3>MutSα and MutLα are essential for S. pombe MMR, which is limited to loops with up to four nucleotides</h3><p id="p-43">Eukaryotic MMR is initiated by MutSα for repair of base-base mismatches and loops or by MutSβ for repair of loops (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Marti <em>et al.</em> 2002</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jiricny 2013</a>). In <em>S. cerevisiae</em>, <em>msh3</em> and <em>msh6</em> mutants show little to moderate increases of mutation rates in mono- and dinucleotide repeats (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Johnson <em>et al.</em> 1996</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Marsischky <em>et al.</em> 1996</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Greene and Jinks-Robertson 1997</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Sia <em>et al.</em> 1997</a>). On the other hand, such repeats are highly unstable in <em>msh3 msh6</em> double mutants and within the range of the <em>msh2</em> instability, indicating redundancy of MutSα and MutSβ for small loops in this organism. In humans, MutSα is the major factor for recognition of base-base mismatches and loops, while MutSβ rather serves as a backup (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Drummond <em>et al.</em> 1997</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Genschel <em>et al.</em> 1998</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Marra <em>et al.</em> 1998</a>).</p><p id="p-44">In <em>S. pombe</em>, we knew to this date that MMR is able to repair base-base mismatches and loops with up to two nucleotides (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Schär <em>et al.</em> 1997</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Rudolph <em>et al.</em> 1999</a>; <a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Mansour <em>et al.</em> 2001</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Tornier <em>et al.</em> 2001</a>; <a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Marti <em>et al.</em> 2003</a>). This requires MutSα and MutLα but not MutSβ. In the present study, we expanded analysis of loop repair in <em>S. pombe</em> to four to seven unpaired nucleotides. Our aim was to determine the contributions of Msh2, Msh3, Msh6, Mlh1, and Exo1, and particularly the relative roles of MutSα and MutSβ in stability of repeats with four or more iterated nucleotides in this model organism. The microsatellites tested were (GACC)<sub>7</sub> and (GACC)<sub>8</sub> tetra-, (CTGCC)<sub>6</sub> penta-, and (ATCGTCC)<sub>5</sub> hepta-nucleotide repeats (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). All such insertions caused a frameshift, rendering cells defective in <em>ade6</em>. Reversions to Ade<sup>+</sup> occurred when deletions or insertions of repeats restored the open reading frame.</p><p id="p-45">Inactivated Msh2, Msh6, and Mlh1 caused instability of the tetra-nucleotide repeats, while defective Msh3 and Exo1 rather made the repeats slightly more stable, although this was not significantly different to wild type (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The (GACC)<sub>8</sub> repeat reverted in wild type by insertions of eight nucleotides, while in <em>msh2</em> and <em>msh6</em> mutants, mainly deletions of four nucleotides occurred (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). Thus, MutSα of <em>S. pombe</em> is capable to initiate MMR of loops with four unpaired nucleotides, whereas MutSβ is not.</p><p id="p-46">The assay with the (GACC)<sub>7</sub>ΔT repeat allowed distinguishing deletions from insertions by the color of Ade<sup>+</sup> revertants (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>). We found that wild type, <em>msh3</em>, and <em>exo1</em> mainly reverted by eight-nucleotide deletions, and <em>msh2</em>, <em>msh6</em>, and <em>mlh1</em> mainly by four-nucleotide insertions (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a> and <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). Thus, this assay also revealed that MutSα but not MutSβ initiates MMR of loops with four nucleotides. In addition, Mlh1, and by extrapolation MutLα, is involved in removal of four-nucleotide loops. Since four-nucleotide deletions, detectable with (GACC)<sub>8</sub>, and four-nucleotide insertions, detectable with (GACC)<sub>7</sub>ΔT, were the predominant reversion events in <em>msh2</em> and <em>msh6</em> mutants, slippage of one repeat can occur in the template and in the nascent strand during replication, and both types of events are corrected by MMR mediated by MutSα and MutLα.</p><p id="p-47">Mutation rates of the penta-nucleotide repeat (CTGCC)<sub>6</sub> in <em>msh2</em>, <em>msh6</em>, <em>mlh1</em>, and <em>exo1</em> mutants were similar to that of wild type, but decreased in <em>msh3</em> (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). All strains preferentially reverted to Ade<sup>+</sup> by insertion of one repeat (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Inactivation of <em>msh2</em>, <em>msh6</em>, and <em>mlh1</em> did not affect reversion rates or spectra of the hepta-nucleotide repeats (ATCGTCC)<sub>5</sub>ΔT and (ATCGTCC)<sub>5</sub> (<a id="xref-table-wrap-5-5" class="xref-table" href="#T5">Table 5</a> and <a id="xref-table-wrap-6-5" class="xref-table" href="#T6">Table 6</a>). Like wild type, the mutants reverted by deletion of one repeat in (ATCGTCC)<sub>5</sub>ΔT and mainly by insertion of one repeat in (ATCGTCC)<sub>5</sub>. We conclude that loops in penta- and hepta-nucleotide repeats are not substrates of MutSα and MutLα in <em>S. pombe</em>.</p></div><div id="sec-16" class="subsection"><h3>Msh3 has an MMR-independent function in repeat stability</h3><p id="p-48">Our previous data showed that <em>msh3</em> mutants had no significant defects in repair of base-base mismatches and of loops with one unpaired nucleotide in a T<sub>6</sub> repeat and in nonrepetitive DNA (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Tornier <em>et al.</em> 2001</a>; <a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Marti <em>et al.</em> 2003</a>). <em>msh3</em> mutations caused some instability of a (GT)<sub>8</sub> dinucleotide repeat, which was mostly evident by a reversion spectrum different to wild type (<a id="xref-ref-44-7" class="xref-bibr" href="#ref-44">Mansour <em>et al.</em> 2001</a>). Wild type reverted mainly by four-nucleotide insertions, whereas <em>msh3</em> mainly reverted by two-nucleotide deletions in the (GT)<sub>8</sub> repeat. However, this was clearly less frequent than in <em>msh2</em>, <em>msh6</em>, and <em>pms1</em> mutants (<a id="xref-ref-44-8" class="xref-bibr" href="#ref-44">Mansour <em>et al.</em> 2001</a>). In the present study, we also found that the spectrum for the (GACC)<sub>8</sub> repeat was different to wild type and to <em>msh2</em> and <em>msh6</em> (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). Wild type exclusively reverted via eight-nucleotide insertions (10 out of 10 revertants analyzed), and <em>msh2</em> and <em>msh6</em> mostly by four-nucleotide deletions. In contrast, four out of 11 revertants in <em>msh3</em> background were due to four-nucleotide deletions, six originated from eight-nucleotide insertions, and one by an insertion of 20 nucleotides (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>). Thus, Msh3 appears to have a function in tetra-nucleotide repeat stability, which is different to Msh2 and Msh6.</p><p id="p-49">In the case of the penta-nucleotide repeat, we found that the <em>msh3</em> mutant showed a lower mutation rate than wild type (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>), indicating a role of Msh3 in supporting tract length changes in this repeat. Intriguingly, the <em>msh3</em> mutant exhibited increased instability of the hepta-nucleotide repeats (<a id="xref-table-wrap-5-6" class="xref-table" href="#T5">Table 5</a>). Thus, Msh3 has a function in maintaining stability of such repeats. Since <em>msh2</em>, <em>msh6</em>, and <em>mlh1</em> did not show instability of hepta-nucleotide repeats, the Msh3 function appears to be MMR independent. We have found a genetic link to Rad50, Rad51, and Exo1, indicating that the Msh3 function is related to recombinational processes. Such a function is well known for <em>S. cerevisiae</em> MutSβ, which participates in SSA where repeats flank a double-strand break (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Sugawara <em>et al.</em> 1997</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chakraborty and Alani 2016</a>). MutSβ of <em>S. pombe</em> likely acts similarly, as both Msh2 and Msh3 have a function in the recombinational process of mating-type switching (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fleck <em>et al.</em> 1992</a>; <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Rudolph <em>et al.</em> 1999</a>). However, genetic data presented here and in our previous studies suggest that the functions of Msh3 in repeat stability and recombination is likely independent of Msh2 (<a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">Tornier <em>et al.</em> 2001</a>; <a id="xref-ref-44-9" class="xref-bibr" href="#ref-44">Mansour <em>et al.</em> 2001</a>; <a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">Marti <em>et al.</em> 2003</a>). The Msh2 independent role of <em>S. pombe</em> Msh3 in recombination may relate to that of bacterial MutS2 proteins, which act in recombination and antirecombination mechanisms but not in MMR (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Pinto <em>et al.</em> 2005</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Burby and Simmons 2017</a>). MutS2 of <em>Helicobacter pylori</em> binds to DNA structures that resemble recombination intermediates and inhibits strand exchange <em>in vitro</em> (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Pinto <em>et al.</em> 2005</a>). In this regard, <em>S. pombe</em> Msh3 may be functionally similar, although structurally, it lacks the endonuclease domain of MutS2, and homology of its amino acid sequence clearly allocates it to the group of eukaryotic Msh3 proteins.</p><p id="p-50">Structural studies with human MutS heterodimers showed that mismatch binding largely occurs by Msh3 or Msh6, while Msh2 has few contacts with the DNA backbone of correctly paired nucleotides in the vicinity (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Warren <em>et al.</em> 2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gupta <em>et al.</em> 2011</a>). The human Msh6 protein interacts directly with mismatched bases via a phenylalanine, which is conserved in eukaryotic Msh6 orthologs and bacterial MutS. In contrast, human Msh3 lacks this residue and instead interacts with phosphate groups of the unpaired nucleotides (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gupta <em>et al.</em> 2011</a>). Work by <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lee <em>et al.</em> (2007)</a> demonstrated that deletion of the mismatch binding domain of <em>S. cerevisiae</em> Msh2 causes loss of MutSβ−dependent MMR activity and revealed that the domain in Msh2 is required for general DNA binding, and in Msh3 for binding to DNA loops. In complex with Msh2, a chimeric Msh6 protein of <em>S. cerevisiae</em> containing the mismatch binding domain of Msh3 showed substrate specificity of Msh3, <em>i.e.</em>, high affinity to loops with one to four unpaired nucleotides (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Shell <em>et al.</em> 2007</a>). The amino acid sequence within the mismatch binding domain of <em>S. pombe</em> Msh3 is very similar to that of human Msh3. However, in contrast to <em>S. cerevisiae</em> and human Msh3 and orthologs of other eukaryotes, <em>S. pombe</em> Msh3 lacks a canonical PIP box, which mediates interaction with PCNA. The PIP box of human Msh3 overlaps with the Mlh1 binding domain (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Iyer <em>et al.</em> 2010</a>). Thus, it is also conceivable that amino acid residues required for interaction with Mlh1 are not present in <em>S. pombe</em> Msh3. It is currently not known whether <em>S. pombe</em> Msh3 can interact with PCNA or MutLα. If it does not, this may explain that it does not participate in MMR.</p></div><div id="sec-17" class="subsection"><h3>Role of MutSα and MutSβ in MMR</h3><p id="p-51">The <em>Escherichia coli</em> homodimer MutS enables repair of base-base mismatches and loops with up to four nucleotides (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Iyer <em>et al.</em> 2006</a>). Reconstituted MMR with <em>S. cerevisiae</em> proteins <em>in vitro</em> revealed that both MutSα and MutSβ could initiate repair of base-base mismatches and of loops with one, two, or four nucleotides (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bowen <em>et al.</em> 2013</a>). Human MutSα binds to base-base mismatches and to loops with up to eight nucleotides, while MutSβ allows repair of loops with two to about eight nucleotides (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Genschel <em>et al.</em> 1998</a>). Our genetic data imply that <em>S. pombe</em> Msh6 as part of the MutSα heterodimer is able to bind to loops with up to four unpaired nucleotides. Thus, the substrate spectrum of bacterial MutS and MutSα of <em>S. pombe</em> appears to be similar. During evolution, the spectrum had been extended to enable recognition of larger loops in humans. On the other hand, the substrate spectrum of MutSβ considerably differs between species. Human MutSβ supports repair of loops with two to eight nucleotides (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Genschel <em>et al.</em> 1998</a>), whereas <em>S. cerevisiae</em> MutSβ is also involved in repair of some base-base mismatches besides loop repair (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Harrington and Kolodner 2007</a>), and <em>S. pombe</em> MutSβ apparently does not have a function in MMR. In addition, some eukaryotes, such as <em>Caenorhabditis elegans</em> and <em>Drosophila melanogaster</em> do not have an Msh3 ortholog (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Marti <em>et al.</em> 2002</a>) and likely carry out MMR with MutSα and MutLα and no other MutS and MutL heterodimers, like <em>S. pombe</em> does.</p><p id="p-52"><a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Harrington and Kolodner (2007)</a> interpreted mutation spectra of base substitutions in <em>S. cerevisiae msh3</em> mutants that were different to wild type as a role of MutSβ in MMR of base-base mismatches. We observed differences of <em>msh3</em> in mutation spectra for a (GT)<sub>8</sub> repeat (<a id="xref-ref-44-10" class="xref-bibr" href="#ref-44">Mansour <em>et al.</em> 2001</a>), a reduction of recombination events (<a id="xref-ref-67-5" class="xref-bibr" href="#ref-67">Tornier <em>et al.</em> 2001</a>), and of reversion rates at the (CTGCC)<sub>6</sub> repeat (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>), an altered reversion spectrum for (GACC)<sub>8</sub> (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>) and repeat instability of (ATCGTCC)<sub>5</sub> (<a id="xref-table-wrap-5-7" class="xref-table" href="#T5">Table 5</a>). We interpret these differences as phenotypes caused by loss of MMR-independent functions of Msh3.</p></div><div id="sec-18" class="subsection"><h3>Does S. pombe Exo1 have a function in MMR?</h3><p id="p-53">Exo1 of <em>S. pombe</em> was the first eukaryotic exonuclease to be identified as having a function in repair of mismatches (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Szankasi and Smith 1995</a>). Further studies with <em>S. pombe</em> showed that Exo1 contributes to MMR of base-base mismatches (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Rudolph <em>et al.</em> 1998</a>), modulates MMR of two-nucleotide loops in nonrepetitive DNA (<a id="xref-ref-49-8" class="xref-bibr" href="#ref-49">Marti <em>et al.</em> 2003</a>), and has an MMR-independent function in dinucleotide repeat stability (<a id="xref-ref-44-11" class="xref-bibr" href="#ref-44">Mansour <em>et al.</em> 2001</a>). In the present study, we did not found any evidence for a role of Exo1 in tetra- and penta-nucleotide repeat stability (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). However, we observed that loss of Exo1 caused instability of hepta-nucleotide repeats, in contrast to the MMR mutants <em>msh2</em>, <em>msh6</em>, and <em>mlh1</em> (<a id="xref-table-wrap-5-8" class="xref-table" href="#T5">Table 5</a>). Exo1 also acts in recombination and double-strand break repair (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fiorentini <em>et al.</em> 1997</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Tsubouchi and Ogawa 2000</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kirkpatrick <em>et al.</em> 2000</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cejka 2015</a>). Thus, a defect in a recombination mechanism might cause hepta-nucleotide instability of the <em>S. pombe exo1</em> mutant rather than MMR deficiency, as discussed below.</p><p id="p-54">Although a nuclease is essential for removal of unpaired nucleotides during MMR, Exo1 does seem to be dispensable for MMR-mediated loop repair in <em>S. pombe</em>. This may be attributed to redundancy with other nucleases. MutLα of <em>S. cerevisiae</em> and humans has endonuclease activity, which is sufficient for completing MMR in the absence of Exo1 (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Kadyrov <em>et al.</em> 2006</a>, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">2007</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Smith <em>et al.</em> 2013</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Goellner <em>et al.</em> 2014</a>, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">2015</a>). Thus, it is also likely that Exo1 of <em>S. pombe</em> participates in MMR, but that MutLα and maybe other nucleases can replace its function. In fact, the amino acids required for MutLα nuclease activity are all highly conserved between eukaryotes, including <em>S. pombe</em> (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Smith <em>et al.</em> 2013</a>), supporting the idea that having endonuclease activity is a general feature of eukaryotic MutLα. In <em>S. cerevisiae</em>, <em>exo1</em> mutants exhibit weak defects in MMR (<a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Tishkoff <em>et al.</em> 1997</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Amin <em>et al.</em> 2001</a>; <a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Smith <em>et al.</em> 2013</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Goellner <em>et al.</em> 2014</a>), likely because MutLα and Exo1 nuclease activities are largely redundant. <em>exo1</em> deletion strains and <em>pms1</em> strains with mutations causing endonuclease deficiency generally showed subtle increases of mutation rates, which strongly increased when both mutations were combined (<a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Smith <em>et al.</em> 2013</a>).</p></div><div id="sec-19" class="subsection"><h3>Rad51 and Exo1 are involved in error prone repair at (ATCGTCC)<sub>5</sub> repeats in msh3 mutants</h3><p id="p-55">Stability of the hepta-nucleotide repeat (ATCGTCC)<sub>5</sub> was influenced by processes involving Msh3, Exo1, Rad2<sup>FEN1</sup>, Rad50, and Rad51. Deletions of any of the genes caused an ∼2–3-fold increase of reversion rates, which was predominantly due to expansions by one repeat unit and therefore by insertions in the nascent strand (<a id="xref-table-wrap-6-6" class="xref-table" href="#T6">Table 6</a> and <a id="xref-table-wrap-7-4" class="xref-table" href="#T7">Table 7</a>). Rates were not further increased in the <em>msh3 rad2<sup>FEN1</sup></em>, <em>msh3 rad50</em>, and <em>exo1 rad50</em> double mutants. Instead, the <em>msh3 exo1</em>, <em>msh3 rad51</em>, and <em>exo1 rad51</em> double mutants had lower rates than the respective single mutants. In <em>S. cerevisiae</em>, CAG trinucleotide repeats were unstable in <em>rad51</em>, <em>rad52</em>, and <em>mre11</em> single mutants (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Sundararajan <em>et al.</em> 2010</a>). However, increased rates of repeat expansions in <em>mre11</em> were largely suppressed by additional mutation of <em>rad52</em>. These data suggest that the MRN complex plays a role in maintaining repeat stability, and that downstream steps of HR in <em>mre11</em>, but not in wild-type background, can carry out error prone recombination at repeats (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Sundararajan <em>et al.</em> 2010</a>). In summary, the (ATCGTCC)<sub>5</sub> repeat analyzed in our study might be stabilized by Msh3 and slipped-out loops correctly processed by HR requiring Rad50, Exo1, and Rad51, thereby preventing aberrant events. In the absence of Msh3, the Exo1 and Rad51 proteins might carry out error prone processes, such as misalignment of repeats after strand resection catalyzed by Exo1 and during strand invasion mediated by Rad51.</p></div><div id="sec-20" class="subsection"><h3>Conclusions</h3><p id="p-56">We conclude from our studies that <em>S. pombe</em> Msh6, as part of MutSα, recognizes base-base mismatches and loops with one to four unpaired nucleotides, while Msh3 does not play a significant role in MMR, but rather maintains repeat stability independently of MMR. Consequently, <em>S. pombe</em> MMR cannot repair loops with five or more nucleotides, in contrast to human MMR (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Genschel <em>et al.</em> 1998</a>). Microsatellites with five or six iterated nucleotides are rare in <em>S. pombe</em> (hepta-nucleotide repeats were not analyzed) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Karaoglu <em>et al.</em> 2005</a>), but are relatively abundant in the human genome (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lander <em>et al.</em> 2001</a>). Thus, to ensure genome stability, humans require repair of larger loops that occur by strand slippage in microsatellites, while larger loops may be formed rarely in <em>S. pombe</em> microsatellites. It is therefore critical for humans, but not for <em>S. pombe</em>, to have an MMR system that can deal with larger loops.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">We thank Harri Burford, Huw Gibbard, and David Jones for constructing <em>S. pombe</em> strains with hepta-nucleotide repeats during their third-year projects and Edgar Hartsuiker for critical reading of the manuscript. This work was supported by North West Cancer Research grant CR947 and the National Institute of Social Care and Health Research-Cancer Genetics Biomedical Research Unit.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-19"><p id="p-58"><em>Communicating editor: C. S. Hoffman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-11-25"><span class="received-label">Received </span>November 25, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-03-07"><span class="accepted-label">Accepted </span>March 7, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Villahermosa <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.1" data-doi="10.1128/MCB.16.10.5604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The <em>Saccharomyces cerevisiae</em> Msh2 and Msh6 proteins form a complex that specifically binds to duplex oligonucleotides containing mismatched DNA base pairs.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">5604</span>–<span class="cit-lpage">5615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DAlani%26rft.auinit1%253DE.%26rft.volume%253D16%26rft.issue%253D10%26rft.spage%253D5604%26rft.epage%253D5615%26rft.atitle%253DThe%2BSaccharomyces%2Bcerevisiae%2BMsh2%2Band%2BMsh6%2Bproteins%2Bform%2Ba%2Bcomplex%2Bthat%2Bspecifically%2Bbinds%2Bto%2Bduplex%2Boligonucleotides%2Bcontaining%2Bmismatched%2BDNA%2Bbase%2Bpairs%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.16.10.5604%26rft_id%253Dinfo%253Apmid%252F8816473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMC81NjA0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.2" data-doi="10.1128/MCB.21.15.5142-5155.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amin</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title"><em>exo1</em>-Dependent mutator mutations: model system for studying functional interactions in mismatch repair.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">5142</span>–<span class="cit-lpage">5155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DAmin%26rft.auinit1%253DN.%2BS.%26rft.volume%253D21%26rft.issue%253D15%26rft.spage%253D5142%26rft.epage%253D5155%26rft.atitle%253Dexo1-Dependent%2BMutator%2BMutations%253A%2BModel%2BSystem%2Bfor%2BStudying%2BFunctional%2BInteractions%2Bin%2BMismatch%2BRepair%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.21.15.5142-5155.2001%26rft_id%253Dinfo%253Apmid%252F11438669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xNS81MTQyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhargava</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Onyango</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Regulation of single-strand annealing and its role in genome maintenance.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">566</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D32%26rft.spage%253D566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.4" data-doi="10.1073/pnas.1318971110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivatsan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willcox</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffith</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reconstitution of long and short patch mismatch repair reactions using <em>Saccharomyces cerevisiae</em> proteins.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">18472</span>–<span class="cit-lpage">18477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1318971110%26rft_id%253Dinfo%253Apmid%252F24187148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzQ2LzE4NDcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.5" data-doi="10.1128/JB.00682-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burby</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simmons</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">MutS2 promotes homologous recombination in <em>Bacillus subtilis</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">e00682-16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00682-16%26rft_id%253Dinfo%253Apmid%252F27799325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTU6IjE5OS8yL2UwMDY4Mi0xNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.6" data-doi="10.1074/jbc.R115.675942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cejka</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">DNA end resection: nucleases team up with the right partners to initiate homologous recombination.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">22931</span>–<span class="cit-lpage">22938</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.R115.675942%26rft_id%253Dinfo%253Apmid%252F26231213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTAvMzgvMjI5MzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.7" data-doi="10.1093/femsyr/fow071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Understanding how mismatch repair proteins participate in the repair/anti-recombination decision.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">fow071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffow071%26rft_id%253Dinfo%253Apmid%252F27573382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjExOiIxNi82L2ZvdzA3MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.8" data-doi="10.21873/anticanres.10983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Da Silva</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wernhoff</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominguez-Barrera</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominguez-Valentin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Update on hereditary colorectal cancer.</span> <abbr class="cit-jnl-abbrev">Anticancer Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">4399</span>–<span class="cit-lpage">4405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnticancer%2BResearch%26rft.stitle%253DAnticancer%2BRes%26rft.aulast%253DDA%2BSILVA%26rft.auinit1%253DF.%2BC.%26rft.volume%253D36%26rft.issue%253D9%26rft.spage%253D4399%26rft.epage%253D4405%26rft.atitle%253DUpdate%2Bon%2BHereditary%2BColorectal%2BCancer%26rft_id%253Dinfo%253Adoi%252F10.21873%252Fanticanres.10983%26rft_id%253Dinfo%253Apmid%252F27630275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImFudGljYW5yZXMiO3M6NToicmVzaWQiO3M6OToiMzYvOS80Mzk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.9" data-doi="10.1073/pnas.94.19.10144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Genschel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modrich</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title"><em>DHFR/MSH3</em> amplification in methotrexate-resistant cells alters the hMutSα/hMutSβ ratio and reduces the efficiency of base-base mismatch repair.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">10144</span>–<span class="cit-lpage">10149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDrummond%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D94%26rft.issue%253D19%26rft.spage%253D10144%26rft.epage%253D10149%26rft.atitle%253DDHFR%252FMSH3%2Bamplification%2Bin%2Bmethotrexate-resistant%2Bcells%2Balters%2Bthe%2BhMutS%257Balpha%257D%252FhMutS%257Bbeta%257D%2Bratio%2Band%2Breduces%2Bthe%2Befficiency%2Bof%2Bbase-base%2Bmismatch%2Brepair%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.19.10144%26rft_id%253Dinfo%253Apmid%252F9294177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTQvMTkvMTAxNDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.10" data-doi="10.1128/MCB.17.5.2764"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fiorentini</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">D. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symington</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Exonuclease I of <em>Saccharomyces cerevisiae</em> functions in mitotic recombination in vivo and in vitro.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2764</span>–<span class="cit-lpage">2773</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DFiorentini%26rft.auinit1%253DP.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D2764%26rft.epage%253D2773%26rft.atitle%253DExonuclease%2BI%2Bof%2BSaccharomyces%2Bcerevisiae%2Bfunctions%2Bin%2Bmitotic%2Brecombination%2Bin%2Bvivo%2Band%2Bin%2Bvitro%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.17.5.2764%26rft_id%253Dinfo%253Apmid%252F9111347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE3LzUvMjc2NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.11" data-doi="10.1093/nar/20.9.2271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fleck</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heim</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The <em>swi4<sup>+</sup></em> gene of <em>Schizosaccharomyces pombe</em> encodes a homologue of mismatch repair enzymes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2271</span>–<span class="cit-lpage">2278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DFleck%26rft.auinit1%253DO.%26rft.volume%253D20%26rft.issue%253D9%26rft.spage%253D2271%26rft.epage%253D2278%26rft.atitle%253DThe%2Bswi4%252B%2Bgene%2Bof%2BSchizosaccharomyces%2Bpombe%2Bencodes%2Ba%2Bhomologue%2Bof%2Bmismatch%2Brepair%2Benzymes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F20.9.2271%26rft_id%253Dinfo%253Apmid%252F1317550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMjI3MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.12" data-doi="10.1038/81708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flores-Rozas</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Proliferating cell nuclear antigen and Msh2p-Msh6p interact to form an active mispair recognition complex.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DFlores-Rozas%26rft.auinit1%253DH.%26rft.volume%253D26%26rft.issue%253D3%26rft.spage%253D375%26rft.epage%253D378%26rft.atitle%253DProliferating%2Bcell%2Bnuclear%2Bantigen%2Band%2BMsh2p-Msh6p%2Binteract%2Bto%2Bform%2Ban%2Bactive%2Bmispair%2Brecognition%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F81708%26rft_id%253Dinfo%253Apmid%252F11062484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/81708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11062484&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165176500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.13" data-doi="10.1126/science.279.5352.853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freudenreich</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kantrow</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakian</span>  <span class="cit-name-given-names">V. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Expansion and length-dependent fragility of CTG repeats in yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">856</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFreudenreich%26rft.auinit1%253DC.%2BH.%26rft.volume%253D279%26rft.issue%253D5352%26rft.spage%253D853%26rft.epage%253D856%26rft.atitle%253DExpansion%2Band%2BLength-Dependent%2BFragility%2Bof%2BCTG%2BRepeats%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.279.5352.853%26rft_id%253Dinfo%253Apmid%252F9452383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzkvNTM1Mi84NTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.14" data-doi="10.1074/jbc.273.31.19895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Genschel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Littman</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modrich</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Isolation of MutSβ from human cells and comparison of the mismatch repair specificities of MutSβ and MutSα.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">19895</span>–<span class="cit-lpage">19901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DGenschel%26rft.auinit1%253DJ.%26rft.volume%253D273%26rft.issue%253D31%26rft.spage%253D19895%26rft.epage%253D19901%26rft.atitle%253DIsolation%2Bof%2BMutS%257Bbeta%257D%2Bfrom%2BHuman%2BCells%2Band%2BComparison%2Bof%2Bthe%2BMismatch%2BRepair%2BSpecificities%2Bof%2BMutS%257Bbeta%257D%2Band%2BMutS%257Balpha%257D%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.273.31.19895%26rft_id%253Dinfo%253Apmid%252F9677427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzMvMzEvMTk4OTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.15" data-doi="10.1093/embo-reports/kvf044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giannini</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ristori</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cerignoli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinaldi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zani</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Human MRE11 is inactivated in mismatch repair-deficient cancers.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">248</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2Breports%26rft.stitle%253DEMBO%2BRep.%26rft.aulast%253DGiannini%26rft.auinit1%253DG.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D248%26rft.epage%253D254%26rft.atitle%253DHuman%2BMRE11%2Bis%2Binactivated%2Bin%2Bmismatch%2Brepair-deficient%2Bcancers%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fembo-reports%252Fkvf044%26rft_id%253Dinfo%253Apmid%252F11850399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6NzoiMy8zLzI0OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.16" data-doi="10.1016/j.molcel.2014.04.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goellner</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hombauer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">PCNA and Msh2-Msh6 activate an Mlh1-Pms1 endonuclease pathway required for Exo1-independent mismatch repair.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D55%26rft.spage%253D291%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.04.034%26rft_id%253Dinfo%253Apmid%252F24981171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.04.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24981171&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.17" data-doi="10.1016/j.dnarep.2015.04.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goellner</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Putnam</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Exonuclease 1-dependent and independent mismatch repair.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRepair%2B%2528Amst.%2529%26rft.volume%253D32%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2015.04.010%26rft_id%253Dinfo%253Apmid%252F25956862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2015.04.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25956862&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.18" data-doi="10.1128/MCB.17.5.2844"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jinks-Robertson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Frameshift intermediates in homopolymer runs are removed efficiently by yeast mismatch repair proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2844</span>–<span class="cit-lpage">2850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DGreene%26rft.auinit1%253DC.%2BN.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D2844%26rft.epage%253D2850%26rft.atitle%253DFrameshift%2Bintermediates%2Bin%2Bhomopolymer%2Bruns%2Bare%2Bremoved%2Befficiently%2Bby%2Byeast%2Bmismatch%2Brepair%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.17.5.2844%26rft_id%253Dinfo%253Apmid%252F9111356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE3LzUvMjg0NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grimm</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bähler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohli</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title"><em>M26</em> recombinational hotspot and physical conversion tract analysis in the <em>ade6</em> gene of <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGrimm%26rft.auinit1%253DC.%26rft.volume%253D136%26rft.issue%253D1%26rft.spage%253D41%26rft.epage%253D51%26rft.atitle%253DM26%2Brecombinational%2Bhotspot%2Band%2Bphysical%2Bconversion%2Btract%2Banalysis%2Bin%2Bthe%2Bade6%2Bgene%2Bof%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Apmid%252F7908005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTM2LzEvNDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.20" data-doi="10.1038/nsmb.2175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gellert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mechanism of mismatch recognition revealed by human MutSβ bound to unpaired DNA loops.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">72</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D19%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2175%26rft_id%253Dinfo%253Apmid%252F22179786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2175&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22179786&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-7.5.1463.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutz</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leupold</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loprieno</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title"><em>Schizosaccharomyces pombe</em></span>, pp. <span class="cit-fpage">395</span>–<span class="cit-lpage">446</span> in <span class="cit-source"><em>Handbook of Genetics</em></span>, <span class="cit-vol">Vol. 1</span>, edited by <span class="cit-ed"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">R. C.</span></span>. <span class="cit-publ-name">Plenum Press</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHandbook%2Bof%2BGenetics%26rft.volume%253D1%26rft.spage%253D395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.22" data-doi="10.1016/S0960-9822(02)70686-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habraken</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Binding of insertion/deletion DNA mismatches by the heterodimer of yeast mismatch repair proteins MSH2 and MSH3.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1185</span>–<span class="cit-lpage">1187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DHabraken%26rft.auinit1%253DY.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D1185%26rft.epage%253D1187%26rft.atitle%253DBinding%2Bof%2Binsertion%252Fdeletion%2BDNA%2Bmismatches%2Bby%2Bthe%2Bheterodimer%2Bof%2Byeast%2Bmismatch%2Brepair%2Bproteins%2BMSH2%2Band%2BMSH3.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252802%252970686-6%26rft_id%253Dinfo%253Apmid%252F8805366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(02)70686-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8805366&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VH27200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.23" data-doi="10.1128/MCB.00855-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrington</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> Msh2-Msh3 acts in repair of base-base mispairs.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">6546</span>–<span class="cit-lpage">6554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00855-07%26rft_id%253Dinfo%253Apmid%252F17636021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xOC82NTQ2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.24" data-doi="10.1093/emboj/20.23.6660"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartsuiker</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaessen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohli</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Fission yeast Rad50 stimulates sister chromatid recombination and links cohesion with repair.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">6660</span>–<span class="cit-lpage">6671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DHartsuiker%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D23%26rft.spage%253D6660%26rft.epage%253D6671%26rft.atitle%253DFission%2Byeast%2BRad50%2Bstimulates%2Bsister%2Bchromatid%2Brecombination%2Band%2Blinks%2Bcohesion%2Bwith%2Brepair%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F20.23.6660%26rft_id%253Dinfo%253Apmid%252F11726502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjAvMjMvNjY2MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.25" data-doi="10.1002/yea.1291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hentges</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Driessche</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tafforeau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vandenhaute</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Three novel antibiotic marker cassettes for gene disruption and marker switching in <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1013</span>–<span class="cit-lpage">1019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DVallazza-Deschamps%26rft.auinit1%253DG.%26rft.volume%253D22%26rft.issue%253D13%26rft.spage%253D1013%26rft.epage%253D1019%26rft.atitle%253DThree%2Bnovel%2Bantibiotic%2Bmarker%2Bcassettes%2Bfor%2Bgene%2Bdisruption%2Band%2Bmarker%2Bswitching%2Bin%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1291%26rft_id%253Dinfo%253Apmid%252F16200533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1291&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16200533&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232785800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuda</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murata</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Transformation of intact yeast cells treated with alkali cations.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DIto%26rft.auinit1%253DH%26rft.volume%253D153%26rft.issue%253D1%26rft.spage%253D163%26rft.epage%253D168%26rft.atitle%253DTransformation%2Bof%2Bintact%2Byeast%2Bcells%2Btreated%2Bwith%2Balkali%2Bcations.%26rft_id%253Dinfo%253Apmid%252F6336730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTUzLzEvMTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.27" data-doi="10.1021/cr0404794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pluciennik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burdett</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modrich</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">DNA mismatch repair: functions and mechanisms.</span> <abbr class="cit-jnl-abbrev">Chem. Rev.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">302</span>–<span class="cit-lpage">323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemical%2BReviews%26rft.stitle%253DChemical%2BReviews%26rft.aulast%253DIyer%26rft.auinit1%253DR.%2BR.%26rft.volume%253D106%26rft.issue%253D2%26rft.spage%253D302%26rft.epage%253D323%26rft.atitle%253DDNA%2Bmismatch%2Brepair%253A%2Bfunctions%2Band%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fcr0404794%26rft_id%253Dinfo%253Apmid%252F16464007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/cr0404794&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16464007&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235347300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.28" data-doi="10.1074/jbc.M110.104125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pluciennik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Genschel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beese</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MutLα and proliferating cell nuclear antigen share binding sites on MutSβ.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">11730</span>–<span class="cit-lpage">11739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.104125%26rft_id%253Dinfo%253Apmid%252F20154325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvMTUvMTE3MzAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.29" data-doi="10.1101/cshperspect.a012633"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiricny</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Postreplicative mismatch repair.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">a012633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BPerspectives%2Bin%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft.aulast%253DJiricny%26rft.auinit1%253DJ.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253Da012633%26rft.epage%253Da012633%26rft.atitle%253DPostreplicative%2BMismatch%2BRepair%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a012633%26rft_id%253Dinfo%253Apmid%252F23545421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI1LzQvYTAxMjYzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.30" data-doi="10.1126/science.7618086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovvali</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Requirement of the yeast RTH1 5′ to 3′ exonuclease for the stability of simple repetitive DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">238</span>–<span class="cit-lpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJohnson%26rft.auinit1%253DR.%26rft.volume%253D269%26rft.issue%253D5221%26rft.spage%253D238%26rft.epage%253D240%26rft.atitle%253DRequirement%2Bof%2Bthe%2Byeast%2BRTH1%2B5%2527%2Bto%2B3%2527%2Bexonuclease%2Bfor%2Bthe%2Bstability%2Bof%2Bsimple%2Brepetitive%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7618086%26rft_id%253Dinfo%253Apmid%252F7618086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjkvNTIyMS8yMzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.31" data-doi="10.1074/jbc.271.13.7285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovvali</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Requirement of the yeast <em>MSH3</em> and <em>MSH6</em> genes for <em>MSH2</em>-dependent genomic stability.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">7285</span>–<span class="cit-lpage">7288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DJohnson%26rft.auinit1%253DR.%2BE.%26rft.volume%253D271%26rft.issue%253D13%26rft.spage%253D7285%26rft.epage%253D7288%26rft.atitle%253DRequirement%2Bof%2Bthe%2BYeast%2BMSH3%2Band%2BMSH6%2BGenes%2Bfor%2BMSH2-dependent%2BGenomic%2BStability%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.271.13.7285%26rft_id%253Dinfo%253Apmid%252F8631743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNzEvMTMvNzI4NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.32" data-doi="10.1016/j.cell.2006.05.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kadyrov</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dzantiev</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Constantin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modrich</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Endonucleolytic function of MutLα in human mismatch repair.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKadyrov%26rft.auinit1%253DF.%2BA.%26rft.volume%253D126%26rft.issue%253D2%26rft.spage%253D297%26rft.epage%253D308%26rft.atitle%253DEndonucleolytic%2Bfunction%2Bof%2BMutLalpha%2Bin%2Bhuman%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.05.039%26rft_id%253Dinfo%253Apmid%252F16873062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.05.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16873062&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239552600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.33" data-doi="10.1074/jbc.M707617200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kadyrov</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arana</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lukianova</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> MutLα is a mismatch repair endonuclease.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">37181</span>–<span class="cit-lpage">37190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M707617200%26rft_id%253Dinfo%253Apmid%252F17951253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvNTEvMzcxODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.34" data-doi="10.1093/molbev/msi057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karaoglu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Survey of simple sequence repeats in completed fungal genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi057%26rft_id%253Dinfo%253Apmid%252F15563717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNjM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.35" data-doi="10.1038/nbt.1628"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayles</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">H. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis of a genome-wide set of gene deletions in the fission yeast <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKim%26rft.auinit1%253DD.%2BU.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D617%26rft.epage%253D623%26rft.atitle%253DAnalysis%2Bof%2Ba%2Bgenome-wide%2Bset%2Bof%2Bgene%2Bdeletions%2Bin%2Bthe%2Bfission%2Byeast%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1628%26rft_id%253Dinfo%253Apmid%252F20473289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1628&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20473289&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278820200027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirchner</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A DNA polymerase epsilon mutant that specifically causes +1 frameshift mutations within homonucleotide runs in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKirchner%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D155%26rft.issue%253D4%26rft.spage%253D1623%26rft.epage%253D1632%26rft.atitle%253DA%2BDNA%2BPolymerase%2B%257Bepsilon%257D%2BMutant%2BThat%2BSpecifically%2BCauses%2B%252B1%2BFrameshift%2BMutations%2BWithin%2BHomonucleotide%2BRuns%2Bin%2BYeast%26rft_id%253Dinfo%253Apmid%252F10924461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS80LzE2MjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symington</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Decreased meiotic intergenic recombination and increased meiosis I nondisjunction in <em>exo1</em> mutants of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1549</span>–<span class="cit-lpage">1557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKirkpatrick%26rft.auinit1%253DD.%2BT.%26rft.volume%253D156%26rft.issue%253D4%26rft.spage%253D1549%26rft.epage%253D1557%26rft.atitle%253DDecreased%2BMeiotic%2BIntergenic%2BRecombination%2Band%2BIncreased%2BMeiosis%2BI%2BNondisjunction%2Bin%2Bexo1%2BMutants%2Bof%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F11102356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni80LzE1NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.38" data-doi="10.1101/gad.191201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kleczkowska</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lettieri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiricny</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">hMSH3 and hMSH6 interact with PCNA and colocalize with it to replication foci.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">724</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKleczkowska%26rft.auinit1%253DH.%2BE.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253D724%26rft.epage%253D736%26rft.atitle%253DhMSH3%2Band%2BhMSH6%2Binteract%2Bwith%2BPCNA%2Band%2Bcolocalize%2Bwith%2Bit%2Bto%2Breplication%2Bfoci%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.191201%26rft_id%253Dinfo%253Apmid%252F11274057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTUvNi83MjQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.39" data-doi="10.1038/35057062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linton</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nusbaum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Initial sequencing and analysis of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">860</span>–<span class="cit-lpage">921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D409%26rft.issue%253D6822%26rft.spage%253D860%26rft.epage%253D921%26rft.atitle%253DInitial%2Bsequencing%2Band%2Banalysis%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35057062%26rft_id%253Dinfo%253Apmid%252F11237011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35057062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11237011&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166938800058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.40" data-doi="10.1016/j.jmb.2005.10.059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Analysis of interactions between mismatch repair initiation factors and the replication processivity factor PCNA.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">355</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DLee%26rft.auinit1%253DS.%2BD.%26rft.volume%253D355%26rft.issue%253D2%26rft.spage%253D175%26rft.epage%253D184%26rft.atitle%253DAnalysis%2Bof%2Binteractions%2Bbetween%2Bmismatch%2Brepair%2Binitiation%2Bfactors%2Band%2Bthe%2Breplication%2Bprocessivity%2Bfactor%2BPCNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2005.10.059%26rft_id%253Dinfo%253Apmid%252F16303135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2005.10.059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16303135&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234371900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.41" data-doi="10.1016/j.jmb.2006.10.099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Surtees</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> MSH2–MSH3 and MSH2–MSH6 complexes display distinct requirements for DNA binding domain I in mismatch recognition.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">366</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DLee%26rft.auinit1%253DS.%2BD.%26rft.volume%253D366%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D66%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BMSH2-MSH3%2Band%2BMSH2-MSH6%2Bcomplexes%2Bdisplay%2Bdistinct%2Brequirements%2Bfor%2BDNA%2Bbinding%2Bdomain%2BI%2Bin%2Bmismatch%2Brecognition.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2006.10.099%26rft_id%253Dinfo%253Apmid%252F17157869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2006.10.099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17157869&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244161200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.42" data-doi="10.1016/j.tibs.2011.12.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">S. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">DNA base excision repair: a mechanism of trinucleotide repeat expansion.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">162</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DLiu%26rft.auinit1%253DY.%26rft.volume%253D37%26rft.issue%253D4%26rft.spage%253D162%26rft.epage%253D172%26rft.atitle%253DDNA%2Bbase%2Bexcision%2Brepair%253A%2Ba%2Bmechanism%2Bof%2Btrinucleotide%2Brepeat%2Bexpansion.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2011.12.002%26rft_id%253Dinfo%253Apmid%252F22285516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2011.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22285516&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303087600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanspa</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Review of the Lynch syndrome: history, molecular genetics, screening, differential diagnosis, and medicolegal ramifications.</span> <abbr class="cit-jnl-abbrev">Clin. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Bgenetics%26rft.stitle%253DClin%2BGenet%26rft.aulast%253DLynch%26rft.auinit1%253DH.%2BT.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D18%26rft.atitle%253DReview%2Bof%2Bthe%2BLynch%2Bsyndrome%253A%2Bhistory%252C%2Bmolecular%2Bgenetics%252C%2Bscreening%252C%2Bdifferential%2Bdiagnosis%252C%2Band%2Bmedicolegal%2Bramifications.%26rft_id%253Dinfo%253Apmid%252F19659756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19659756&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268051600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mansour</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tornier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darmon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleck</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Control of GT repeat stability in <em>Schizosaccharomyces pombe</em> by mismatch repair factors.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMansour%26rft.auinit1%253DA.%2BA.%26rft.volume%253D158%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D85%26rft.atitle%253DControl%2Bof%2BGT%2BRepeat%2BStability%2Bin%2BSchizosaccharomyces%2Bpombe%2Bby%2BMismatch%2BRepair%2BFactors%26rft_id%253Dinfo%253Apmid%252F11333219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTU4LzEvNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.45" data-doi="10.1073/pnas.95.15.8568"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iaccarino</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lettieri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roscilli</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delmastro</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Mismatch repair deficiency associated with overexpression of the <em>MSH3</em> gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">8568</span>–<span class="cit-lpage">8573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMarra%26rft.auinit1%253DG.%26rft.volume%253D95%26rft.issue%253D15%26rft.spage%253D8568%26rft.epage%253D8573%26rft.atitle%253DMismatch%2Brepair%2Bdeficiency%2Bassociated%2Bwith%2Boverexpression%2Bof%2Bthe%2BMSH3%2Bgene%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.15.8568%26rft_id%253Dinfo%253Apmid%252F9671718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTUvMTUvODU2OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.46" data-doi="10.1101/gad.10.4.407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marsischky</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filosi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Redundancy of <em>Saccharomyces cerevisiae MSH3</em> and <em>MSH6</em> in <em>MSH2</em>-dependent mismatch repair.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMarsischky%26rft.auinit1%253DG%2BT%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D407%26rft.epage%253D420%26rft.atitle%253DRedundancy%2Bof%2BSaccharomyces%2Bcerevisiae%2BMSH3%2Band%2BMSH6%2Bin%2BMSH2-dependent%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.4.407%26rft_id%253Dinfo%253Apmid%252F8600025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTAvNC80MDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.47" data-doi="10.1007/s00018-003-3223-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marti</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleck</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">DNA repair nucleases.</span> <abbr class="cit-jnl-abbrev">Cell. Mol. Life Sci.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">336</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Band%2Bmolecular%2Blife%2Bsciences%2B%253A%2B%2BCMLS%26rft.stitle%253DCell%2BMol%2BLife%2BSci%26rft.aulast%253DMarti%26rft.auinit1%253DT.%2BM.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D336%26rft.epage%253D354%26rft.atitle%253DDNA%2Brepair%2Bnucleases.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-003-3223-4%26rft_id%253Dinfo%253Apmid%252F14770297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-003-3223-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14770297&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189145600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.48" data-doi="10.1002/jcp.10077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marti</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleck</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">DNA mismatch repair and mutation avoidance pathways.</span> <abbr class="cit-jnl-abbrev">J. Cell. Physiol.</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcellular%2Bphysiology%26rft.stitle%253DJ%2BCell%2BPhysiol%26rft.aulast%253DMarti%26rft.auinit1%253DT.%2BM.%26rft.volume%253D191%26rft.issue%253D1%26rft.spage%253D28%26rft.epage%253D41%26rft.atitle%253DDNA%2Bmismatch%2Brepair%2Band%2Bmutation%2Bavoidance%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcp.10077%26rft_id%253Dinfo%253Apmid%252F11920679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcp.10077&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11920679&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174166700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marti</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mansour</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleck</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Different frameshift mutation spectra in non-repetitive DNA of MutSα- and MutLα-deficient fission yeast cells.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2Brepair%26rft.stitle%253DDNA%2BRepair%2B%2528Amst%2529%26rft.aulast%253DMarti%26rft.auinit1%253DT.%2BM.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D571%26rft.epage%253D580%26rft.atitle%253DDifferent%2Bframeshift%2Bmutation%2Bspectra%2Bin%2Bnon-repetitive%2BDNA%2Bof%2BMutSalpha-%2Band%2BMutLalpha-deficient%2Bfission%2Byeast%2Bcells.%26rft_id%253Dinfo%253Apmid%252F12713814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12713814&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.50" data-doi="10.1128/MCB.14.7.4878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavassoli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">al-Harithy</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheldrick</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Structural and functional conservation of the human homolog of the <em>Schizosaccharomyces pombe rad2</em> gene, which is required for chromosome segregation and recovery from DNA damage.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">4878</span>–<span class="cit-lpage">4888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMurray%26rft.auinit1%253DJ%2BM%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D4878%26rft.epage%253D4888%26rft.atitle%253DStructural%2Band%2Bfunctional%2Bconservation%2Bof%2Bthe%2Bhuman%2Bhomolog%2Bof%2Bthe%2BSchizosaccharomyces%2Bpombe%2Brad2%2Bgene%252C%2Bwhich%2Bis%2Brequired%2Bfor%2Bchromosome%2Bsegregation%2Band%2Brecovery%2Bfrom%2BDNA%2Bdamage.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.14.7.4878%26rft_id%253Dinfo%253Apmid%252F8007985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE0LzcvNDg3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.51" data-doi="10.1016/j.gde.2015.03.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Padeken</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeller</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasser</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Repeat DNA in genome organization and stability.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D31%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2015.03.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2015.03.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.52" data-doi="10.1016/j.molcel.2004.11.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathieu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marsin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veaute</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ielpi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Suppression of homologous and homeologous recombination by the bacterial MutS2 protein.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DPinto%26rft.auinit1%253DA.%2BV.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D113%26rft.epage%253D120%26rft.atitle%253DSuppression%2Bof%2Bhomologous%2Band%2Bhomeologous%2Brecombination%2Bby%2Bthe%2Bbacterial%2BMutS2%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.11.035%26rft_id%253Dinfo%253Apmid%252F15629722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.11.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15629722&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226371100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.53" data-doi="10.1073/pnas.1010662107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pluciennik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dzantiev</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Constantin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadyrov</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">PCNA function in the activation and strand direction of MutLα endonuclease in mismatch repair.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">16066</span>–<span class="cit-lpage">16071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1010662107%26rft_id%253Dinfo%253Apmid%252F20713735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzM3LzE2MDY2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.54" data-doi="10.1158/1078-0432.CCR-08-0274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phear</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scorah</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spang-Thomsen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Thymidine selectively enhances growth suppressive effects of camptothecin/irinotecan in MSI+ cells and tumors containing a mutation of <em>MRE11</em>.</span> <abbr class="cit-jnl-abbrev">Clin. Cancer Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">5476</span>–<span class="cit-lpage">5483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BCancer%2BResearch%26rft.stitle%253DClin.%2BCancer%2BRes.%26rft.aulast%253DRodriguez%26rft.auinit1%253DR.%26rft.volume%253D14%26rft.issue%253D17%26rft.spage%253D5476%26rft.epage%253D5483%26rft.atitle%253DThymidine%2BSelectively%2BEnhances%2BGrowth%2BSuppressive%2BEffects%2Bof%2BCamptothecin%252FIrinotecan%2Bin%2BMSI%252B%2BCells%2Band%2BTumors%2BContaining%2Ba%2BMutation%2Bof%2BMRE11%26rft_id%253Dinfo%253Adoi%252F10.1158%252F1078-0432.CCR-08-0274%26rft_id%253Dinfo%253Apmid%252F18765539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImNsaW5jYW5yZXMiO3M6NToicmVzaWQiO3M6MTA6IjE0LzE3LzU0NzYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.55" data-doi="10.1007/s002940050405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rudolph</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleck</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohli</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title"><em>Schizosaccharomyces pombe exo1</em> is involved in the same mismatch repair pathway as <em>msh2</em> and <em>pms1</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DRudolph%26rft.auinit1%253DC.%26rft.volume%253D34%26rft.issue%253D5%26rft.spage%253D343%26rft.epage%253D350%26rft.atitle%253DSchizosaccharomyces%2Bpombe%2Bexo1%2Bis%2Binvolved%2Bin%2Bthe%2Bsame%2Bmismatch%2Brepair%2Bpathway%2Bas%2Bmsh2%2Band%2Bpms1.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002940050405%26rft_id%253Dinfo%253Apmid%252F9871115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002940050405&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9871115&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077833600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.56" data-doi="10.1128/MCB.19.1.241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rudolph</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parisi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartsuiker</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The <em>msh2</em> gene of <em>Schizosaccharomyces pombe</em> is involved in mismatch repair, mating-type switching, and meiotic chromosome organization.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DRudolph%26rft.auinit1%253DC.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D241%26rft.epage%253D250%26rft.atitle%253DThe%2Bmsh2%2BGene%2Bof%2BSchizosaccharomyces%2Bpombe%2BIs%2BInvolved%2Bin%2BMismatch%2BRepair%252C%2BMating-Type%2BSwitching%252C%2Band%2BMeiotic%2BChromosome%2BOrganization%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.1.241%26rft_id%253Dinfo%253Apmid%252F9858548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMjQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schär</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baur</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohli</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mismatch repair in <em>Schizosaccharomyces pombe</em> requires the <em>mutL</em> homologous gene <em>pms1</em>: molecular cloning and functional analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">1275</span>–<span class="cit-lpage">1286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchar%26rft.auinit1%253DP.%26rft.volume%253D146%26rft.issue%253D4%26rft.spage%253D1275%26rft.epage%253D1286%26rft.atitle%253DMismatch%2BRepair%2Bin%2BSchizosaccharomyces%2Bpombe%2BRequires%2Bthe%2BmutL%2BHomologous%2BGene%2Bpms1%253A%2BMolecular%2BCloning%2Band%2BFunctional%2BAnalysis%26rft_id%253Dinfo%253Apmid%252F9258673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0Ni80LzEyNzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.58" data-doi="10.1073/pnas.0704148104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shell</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Putnam</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Chimeric <em>Saccharomyces cerevisiae</em> Msh6 protein with an Msh3 mispair-binding domain combines properties of both proteins.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">10956</span>–<span class="cit-lpage">10961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0704148104%26rft_id%253Dinfo%253Apmid%252F17573527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzI2LzEwOTU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.59" data-doi="10.1128/MCB.17.5.2851"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sia</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kokoska</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominska</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenwell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Microsatellite instability in yeast: dependence on repeat unit size and DNA mismatch repair genes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2851</span>–<span class="cit-lpage">2858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSia%26rft.auinit1%253DE.%2BA.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D2851%26rft.epage%253D2858%26rft.atitle%253DMicrosatellite%2Binstability%2Bin%2Byeast%253A%2Bdependence%2Bon%2Brepeat%2Bunit%2Bsize%2Band%2BDNA%2Bmismatch%2Brepair%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.17.5.2851%26rft_id%253Dinfo%253Apmid%252F9111357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE3LzUvMjg1MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.60" data-doi="10.1371/journal.pgen.1003869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendillo</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hombauer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dominant mutations in <em>S. cerevisiae PMS1</em> identify the Mlh1-Pms1 endonuclease active site and an exonuclease 1-independent mismatch repair pathway.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003869%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003869%26rft_id%253Dinfo%253Apmid%252F24204293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003869&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24204293&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.61" data-doi="10.1093/nar/gkn446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stith</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgers</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lahue</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Partial reconstitution of DNA large loop repair with purified proteins from <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">4699</span>–<span class="cit-lpage">4707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn446%26rft_id%253Dinfo%253Apmid%252F18628298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNi8xNC80Njk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.62" data-doi="10.1073/pnas.94.17.9214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sugawara</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pâques</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colaiácovo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Role of <em>Saccharomyces cerevisiae</em> Msh2 and Msh3 repair proteins in double-strand break-induced recombination.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">9214</span>–<span class="cit-lpage">9219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSugawara%26rft.auinit1%253DN.%26rft.volume%253D94%26rft.issue%253D17%26rft.spage%253D9214%26rft.epage%253D9219%26rft.atitle%253DRole%2Bof%2BSaccharomyces%2Bcerevisiae%2BMsh2%2Band%2BMsh3%2Brepair%2Bproteins%2Bin%2Bdouble-strand%2Bbreak-induced%2Brecombination%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.17.9214%26rft_id%253Dinfo%253Apmid%252F9256462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTcvOTIxNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.63" data-doi="10.1534/genetics.109.111039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sundararajan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gellon</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zunder</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freudenreich</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Double-strand break repair pathways protect against CAG/CTG repeat expansions, contractions and repeat-mediated chromosomal fragility in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.111039%26rft_id%253Dinfo%253Apmid%252F19901069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTg0LzEvNjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.64" data-doi="10.1126/science.7855597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szankasi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">A role for exonuclease I from <em>S. pombe</em> in mutation avoidance and mismatch correction.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">267</span>: <span class="cit-fpage">1166</span>–<span class="cit-lpage">1169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSzankasi%26rft.auinit1%253DP.%26rft.volume%253D267%26rft.issue%253D5201%26rft.spage%253D1166%26rft.epage%253D1169%26rft.atitle%253DA%2Brole%2Bfor%2Bexonuclease%2BI%2Bfrom%2BS.%2Bpombe%2Bin%2Bmutation%2Bavoidance%2Band%2Bmismatch%2Bcorrection%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7855597%26rft_id%253Dinfo%253Apmid%252F7855597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNjcvNTIwMS8xMTY2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy81LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.65" data-doi="10.1016/0022-2836(88)90051-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szankasi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heyer</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuchert</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohli</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">DNA sequence analysis of the <em>ade6</em> gene of <em>Schizosaccharomyces pombe</em>. Wild-type and mutant alleles including the recombination host spot allele <em>ade6–M26</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">917</span>–<span class="cit-lpage">925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DSzankasi%26rft.auinit1%253DP.%26rft.volume%253D204%26rft.issue%253D4%26rft.spage%253D917%26rft.epage%253D925%26rft.atitle%253DDNA%2Bsequence%2Banalysis%2Bof%2Bthe%2Bade6%2Bgene%2Bof%2BSchizosaccharomyces%2Bpombe.%2BWild-type%2Band%2Bmutant%2Balleles%2Bincluding%2Bthe%2Brecombination%2Bhost%2Bspot%2Ballele%2Bade6-M26.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252888%252990051-4%26rft_id%253Dinfo%253Apmid%252F3221399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(88)90051-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3221399&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988R913200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.66" data-doi="10.1016/S0092-8674(00)81846-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">D. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filosi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaida</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A novel mutation avoidance mechanism dependent on <em>S. cerevisiae RAD27</em> is distinct from DNA mismatch repair.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTishkoff%26rft.auinit1%253DD.%2BX.%26rft.volume%253D88%26rft.issue%253D2%26rft.spage%253D253%26rft.epage%253D263%26rft.atitle%253DA%2Bnovel%2Bmutation%2Bavoidance%2Bmechanism%2Bdependent%2Bon%2BS.%2Bcerevisiae%2BRAD27%2Bis%2Bdistinct%2Bfrom%2BDNA%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981846-2%26rft_id%253Dinfo%253Apmid%252F9008166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81846-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9008166&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WE97000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tornier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bessone</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varlet</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudolph</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darmon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Requirement for Msh6, but not for Swi4 (Msh3), in Msh2-dependent repair of base-base mismatches and mononucleotide loops in <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTornier%26rft.auinit1%253DC.%26rft.volume%253D158%26rft.issue%253D1%26rft.spage%253D65%26rft.epage%253D75%26rft.atitle%253DRequirement%2Bfor%2BMsh6%252C%2Bbut%2BNot%2Bfor%2BSwi4%2B%2528Msh3%2529%252C%2Bin%2BMsh2-Dependent%2BRepair%2Bof%2BBase-Base%2BMismatches%2Band%2BMononucleotide%2BLoops%2Bin%2BSchizosaccharomyces%2Bpombe%26rft_id%253Dinfo%253Apmid%252F11333218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTU4LzEvNjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzUvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.68" data-doi="10.1091/mbc.11.7.2221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Exo1 roles for repair of DNA double-strand breaks and meiotic crossing over in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2221</span>–<span class="cit-lpage">2233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DTsubouchi%26rft.auinit1%253DH.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D2221%26rft.epage%253D2233%26rft.atitle%253DExo1%2BRoles%2Bfor%2BRepair%2Bof%2BDNA%2BDouble-Strand%2BBreaks%2Band%2BMeiotic%2BCrossing%2BOver%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.11.7.2221%26rft_id%253Dinfo%253Apmid%252F10888664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjExLzcvMjIyMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNS8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.69" data-doi="10.1016/j.molcel.2007.04.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pohlhaus</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Changela</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modrich</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Structure of the human MutSα DNA lesion recognition complex.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DWarren%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D26%26rft.issue%253D4%26rft.spage%253D579%26rft.epage%253D592%26rft.atitle%253DStructure%2Bof%2Bthe%2Bhuman%2BMutSalpha%2BDNA%2Blesion%2Brecognition%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2007.04.018%26rft_id%253Dinfo%253Apmid%252F17531815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2007.04.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17531815&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246975500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.5.1463.70" data-doi="10.1016/j.gene.2007.09.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bone</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armstrong</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene tagging and gene replacement using recombinase-mediated cassette exchange in <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">407</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DWatson%26rft.auinit1%253DA.%2BT.%26rft.volume%253D407%26rft.issue%253D1-2%26rft.spage%253D63%26rft.epage%253D74%26rft.atitle%253DGene%2Btagging%2Band%2Bgene%2Breplacement%2Busing%2Brecombinase-mediated%2Bcassette%2Bexchange%2Bin%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2007.09.024%26rft_id%253Dinfo%253Apmid%252F18054176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2007.09.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18054176&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F5%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252917700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/5/1463.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/5/1451" title="Aging Effects of Caenorhabditis elegans Ryanodine Receptor Variants Corresponding to Human Myopathic Mutations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/5/1475" title="Extensive Copy Number Variation in Fermentation-Related Genes Among Saccharomyces cerevisiae Wine Strains" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/5"><span>Volume 7 Issue 5, May 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/5.cover-source.jpg?itok=VWLvGstM" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/5/1463" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RJXqjfU-zsSec6HYuX4gRLjUyw05KP3L3oG-D0A0yUI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/5/1463" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/5/1463" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/5/1463" class="active" data-icon-position="" data-hide-link-title="0">Schizosaccharomyces pombe MutSα and MutLα Maintain Stability of Tetra-Nucleotide Repeats and Msh3 of Hepta-Nucleotide Repeats</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484260" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-EOpur9rb8sgiiJPjUnsJRusk_sBdxyCC4D6M5I1xNhI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit--2" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1463.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/5/1463" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--5" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-1xBY0eu5rKowxARD9F3kT7eTRNFLpp6MDiz3pi766Zo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/5/1463" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/5/1463.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484260" id="node484260--2" data-pisa="ggg;7/5/1463" data-pisa-master="ggg;g3.117.040816" data-apath="/ggg/7/5/1463.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>Schizosaccharomyces pombe</em> MutSα and MutLα Maintain Stability of Tetra-Nucleotide Repeats and Msh3 of Hepta-Nucleotide Repeats</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4072-6159" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Desirée Villahermosa</span>, <span class="highwire-citation-author" data-delta="1">Olaf Christensen</span>, <span class="highwire-citation-author" data-delta="2">Karen Knapp</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0550-6513" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Oliver Fleck</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1463-1473;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040816 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484260--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Desirée Villahermosa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>North West Cancer Research Institute, School of Medical Sciences, Bangor University, LL57 2UW, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Desir%C3%A9e%2BVillahermosa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Villahermosa%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADesir%25C3%25A9e%2BVillahermosa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4072-6159" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Desirée Villahermosa</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Olaf Christensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Strain Development LBPE38, Lonza, CH-3930 Visp, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olaf%2BChristensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Christensen%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOlaf%2BChristensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karen Knapp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>North West Cancer Research Institute, School of Medical Sciences, Bangor University, LL57 2UW, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BKnapp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Knapp%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BKnapp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Oliver Fleck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>North West Cancer Research Institute, School of Medical Sciences, Bangor University, LL57 2UW, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oliver%2BFleck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fleck%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOliver%2BFleck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0550-6513" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Oliver Fleck</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:o.fleck@bangor.ac.uk" class="" data-icon-position="" data-hide-link-title="0">o.fleck@bangor.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/5/1463" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/5/1463&amp;title=Schizosaccharomyces%20pombe%20MutS%CE%B1%20and%20MutL%CE%B1%20Maintain%20Stability%20of%20Tetra-Nucleotide%20Repeats%20and%20Msh3%20of%20Hepta-Nucleotide%20Repeats" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/5/1463&amp;title=Schizosaccharomyces%20pombe%20MutS%CE%B1%20and%20MutL%CE%B1%20Maintain%20Stability%20of%20Tetra-Nucleotide%20Repeats%20and%20Msh3%20of%20Hepta-Nucleotide%20Repeats" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/5/1463&amp;title=Schizosaccharomyces%20pombe%20MutS%CE%B1%20and%20MutL%CE%B1%20Maintain%20Stability%20of%20Tetra-Nucleotide%20Repeats%20and%20Msh3%20of%20Hepta-Nucleotide%20Repeats" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/5/1463&amp;text=Schizosaccharomyces%20pombe%20MutS%CE%B1%20and%20MutL%CE%B1%20Maintain%20Stability%20of%20Tetra-Nucleotide%20Repeats%20and%20Msh3%20of%20Hepta-Nucleotide%20Repeats" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/5/1463&amp;title=Schizosaccharomyces%20pombe%20MutS%CE%B1%20and%20MutL%CE%B1%20Maintain%20Stability%20of%20Tetra-Nucleotide%20Repeats%20and%20Msh3%20of%20Hepta-Nucleotide%20Repeats" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/5/1463&amp;t=Schizosaccharomyces%20pombe%20MutS%CE%B1%20and%20MutL%CE%B1%20Maintain%20Stability%20of%20Tetra-Nucleotide%20Repeats%20and%20Msh3%20of%20Hepta-Nucleotide%20Repeats" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/5/1463&amp;title=Schizosaccharomyces%20pombe%20MutS%CE%B1%20and%20MutL%CE%B1%20Maintain%20Stability%20of%20Tetra-Nucleotide%20Repeats%20and%20Msh3%20of%20Hepta-Nucleotide%20Repeats" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/5/1463&amp;title=Schizosaccharomyces%20pombe%20MutS%CE%B1%20and%20MutL%CE%B1%20Maintain%20Stability%20of%20Tetra-Nucleotide%20Repeats%20and%20Msh3%20of%20Hepta-Nucleotide%20Repeats" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484260/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484260" id="node484260--3" data-pisa="ggg;7/5/1463" data-pisa-master="ggg;g3.117.040816" data-apath="/ggg/7/5/1463.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>Schizosaccharomyces pombe</em> MutSα and MutLα Maintain Stability of Tetra-Nucleotide Repeats and Msh3 of Hepta-Nucleotide Repeats</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4072-6159" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Desirée Villahermosa</span>, <span class="highwire-citation-author" data-delta="1">Olaf Christensen</span>, <span class="highwire-citation-author" data-delta="2">Karen Knapp</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0550-6513" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Oliver Fleck</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1463-1473;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.040816 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484260--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Desirée Villahermosa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>North West Cancer Research Institute, School of Medical Sciences, Bangor University, LL57 2UW, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Desir%C3%A9e%2BVillahermosa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Villahermosa%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADesir%25C3%25A9e%2BVillahermosa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4072-6159" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Desirée Villahermosa</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Olaf Christensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Strain Development LBPE38, Lonza, CH-3930 Visp, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olaf%2BChristensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Christensen%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOlaf%2BChristensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karen Knapp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>North West Cancer Research Institute, School of Medical Sciences, Bangor University, LL57 2UW, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BKnapp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Knapp%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BKnapp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Oliver Fleck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>North West Cancer Research Institute, School of Medical Sciences, Bangor University, LL57 2UW, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oliver%2BFleck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fleck%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOliver%2BFleck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0550-6513" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Oliver Fleck</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:o.fleck@bangor.ac.uk" class="" data-icon-position="" data-hide-link-title="0">o.fleck@bangor.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1463&amp;rft.epage=1473&amp;rft.atitle=Schizosaccharomyces%20pombe%20MutS%CE%B1%20and%20MutL%CE%B1%20Maintain%20Stability%20of%20Tetra-Nucleotide%20Repeats%20and%20Msh3%20of%20Hepta-Nucleotide%20Repeats&amp;rft.volume=7&amp;rft.issue=5&amp;rft.date=2017-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Villahermosa%2C+Desir%C3%A9e&amp;rft.au=Christensen%2C+Olaf&amp;rft.au=Knapp%2C+Karen&amp;rft.au=Fleck%2C+Oliver"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484260/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484260/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484260/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484260/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484260/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484260/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484260/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484260/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484260/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484260/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484260/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484260/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/5/1463" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/5/1463.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/5/1463.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/7/5/1463.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
